

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/6/685/2754553","@type":"ScholarlyArticle","name":"Solution Hybrid Selection Capture for the Recovery of Functional\nFull-Length Eukaryotic cDNAs From Complex Environmental Samples","datePublished":"2014-10-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2014-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu030","keywords":["metatranscriptomics","soil RNA","soil eukaryotes","sequence capture","glycoside hydrolase family GH11"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/6/685/2754553","author":[{"name":"Bragalini, Claudia","affiliation":"1  Department of Life Sciences and Systems Biology ,  University of Turin ,  viale Mattioli 25, Turin 10125 ,  Italy    2  Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364 ,  Université de Lyon, Université Lyon 1 ,  Villeurbanne 69622 ,  France","@type":"Person"},{"name":"Ribière, Céline","affiliation":"3  EA 4678 CIDAM, BP 10448 ,  Clermont Université, Université d'Auvergne ,  Clermont-Ferrand F-63001 ,  France","@type":"Person"},{"name":"Parisot, Nicolas","affiliation":"3  EA 4678 CIDAM, BP 10448 ,  Clermont Université, Université d'Auvergne ,  Clermont-Ferrand F-63001 ,  France","@type":"Person"},{"name":"Vallon, Laurent","affiliation":"2  Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364 ,  Université de Lyon, Université Lyon 1 ,  Villeurbanne 69622 ,  France","@type":"Person"},{"name":"Prudent, Elsa","affiliation":"2  Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364 ,  Université de Lyon, Université Lyon 1 ,  Villeurbanne 69622 ,  France","@type":"Person"},{"name":"Peyretaillade, Eric","affiliation":"3  EA 4678 CIDAM, BP 10448 ,  Clermont Université, Université d'Auvergne ,  Clermont-Ferrand F-63001 ,  France","@type":"Person"},{"name":"Girlanda, Mariangela","affiliation":"2  Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364 ,  Université de Lyon, Université Lyon 1 ,  Villeurbanne 69622 ,  France    4  Istituto per la Protezione Sostenibile delle Piante (IPSP), Consiglio Nazionale delle Ricerche ,  Viale Mattioli 25, Turin 10125 ,  Italy","@type":"Person"},{"name":"Peyret, Pierre","affiliation":"3  EA 4678 CIDAM, BP 10448 ,  Clermont Université, Université d'Auvergne ,  Clermont-Ferrand F-63001 ,  France","@type":"Person"},{"name":"Marmeisse, Roland","affiliation":"1  Department of Life Sciences and Systems Biology ,  University of Turin ,  viale Mattioli 25, Turin 10125 ,  Italy    2  Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364 ,  Université de Lyon, Université Lyon 1 ,  Villeurbanne 69622 ,  France","@type":"Person"},{"name":"Luis, Patricia","affiliation":"2  Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364 ,  Université de Lyon, Université Lyon 1 ,  Villeurbanne 69622 ,  France","@type":"Person"}],"description":"Eukaryotic microbial communities play key functional roles in soil biology and potentially represent a rich source of natural products including biocatalysts. C","pageStart":"685","pageEnd":"694","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/6/1/m_cover.gif?Expires=2147483647&Signature=NlMFHVRI1l1rW9qSwHlYw5oIbjXSqExnWRK4iEjBMZviRiHHauyZm8~g6MAuWSmBaY0~VScPROJjmkyKfZlDCJxwLSn0SLlEV3yXpjY3bZBPj-1qn2Aa5SIlmi51b00TUU4jvZRdwWUFIC54tMGsKuH~dAy4gBDUSIUXAPIdXD7OOYdr4Z-7FqtC5yf-hOhzAIYwuxOLZtaP7T3KIRVyHt4FkBjwVqN5mQx8H7zlYi~z~OF7ZuYB~vZ5tToG9q8yqYB1hVQC69tlDb5jZlqfp~pPUu~WPqA0aKh4xT5xse2XvAKV0yd5SScfSoD3tq2D4tv8DZpsiTb9UtAWz1LQPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Solution Hybrid Selection Capture for the Recovery of Functional\nFull-Length Eukaryotic cDNAs From Complex Environmental Samples","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/6/1/m_cover.gif?Expires=2147483647&Signature=NlMFHVRI1l1rW9qSwHlYw5oIbjXSqExnWRK4iEjBMZviRiHHauyZm8~g6MAuWSmBaY0~VScPROJjmkyKfZlDCJxwLSn0SLlEV3yXpjY3bZBPj-1qn2Aa5SIlmi51b00TUU4jvZRdwWUFIC54tMGsKuH~dAy4gBDUSIUXAPIdXD7OOYdr4Z-7FqtC5yf-hOhzAIYwuxOLZtaP7T3KIRVyHt4FkBjwVqN5mQx8H7zlYi~z~OF7ZuYB~vZ5tToG9q8yqYB1hVQC69tlDb5jZlqfp~pPUu~WPqA0aKh4xT5xse2XvAKV0yd5SScfSoD3tq2D4tv8DZpsiTb9UtAWz1LQPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Solution Hybrid Selection Capture for the Recovery of Functional
Full-Length Eukaryotic cDNAs From Complex Environmental Samples" />
    <meta property="og:description" content="Eukaryotic microbial communities play key functional roles in soil biology and potentially represent a rich source of natural products including biocatalysts. C" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/6/685/2754553" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-10-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Bragalini, Claudia" /><meta name="citation_author_institution" content="Department of Life Sciences and Systems Biology, University of Turin, viale Mattioli 25, Turin 10125, Italy" /><meta name="citation_author_institution" content="Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364, Université de Lyon, Université Lyon 1, Villeurbanne 69622, France" /><meta name="citation_author" content="Ribière, Céline" /><meta name="citation_author_institution" content="EA 4678 CIDAM, BP 10448, Clermont Université, Université d'Auvergne, Clermont-Ferrand F-63001, France" /><meta name="citation_author" content="Parisot, Nicolas" /><meta name="citation_author_institution" content="EA 4678 CIDAM, BP 10448, Clermont Université, Université d'Auvergne, Clermont-Ferrand F-63001, France" /><meta name="citation_author" content="Vallon, Laurent" /><meta name="citation_author_institution" content="Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364, Université de Lyon, Université Lyon 1, Villeurbanne 69622, France" /><meta name="citation_author" content="Prudent, Elsa" /><meta name="citation_author_institution" content="Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364, Université de Lyon, Université Lyon 1, Villeurbanne 69622, France" /><meta name="citation_author" content="Peyretaillade, Eric" /><meta name="citation_author_institution" content="EA 4678 CIDAM, BP 10448, Clermont Université, Université d'Auvergne, Clermont-Ferrand F-63001, France" /><meta name="citation_author" content="Girlanda, Mariangela" /><meta name="citation_author_institution" content="Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364, Université de Lyon, Université Lyon 1, Villeurbanne 69622, France" /><meta name="citation_author_institution" content="Istituto per la Protezione Sostenibile delle Piante (IPSP), Consiglio Nazionale delle Ricerche, Viale Mattioli 25, Turin 10125, Italy" /><meta name="citation_author" content="Peyret, Pierre" /><meta name="citation_author_institution" content="EA 4678 CIDAM, BP 10448, Clermont Université, Université d'Auvergne, Clermont-Ferrand F-63001, France" /><meta name="citation_author" content="Marmeisse, Roland" /><meta name="citation_author_institution" content="Department of Life Sciences and Systems Biology, University of Turin, viale Mattioli 25, Turin 10125, Italy" /><meta name="citation_author_institution" content="Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364, Université de Lyon, Université Lyon 1, Villeurbanne 69622, France" /><meta name="citation_author" content="Luis, Patricia" /><meta name="citation_author_institution" content="Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364, Université de Lyon, Université Lyon 1, Villeurbanne 69622, France" /><meta name="citation_title" content="Solution Hybrid Selection Capture for the Recovery of Functional
Full-Length Eukaryotic cDNAs From Complex Environmental Samples" /><meta name="citation_firstpage" content="685" /><meta name="citation_lastpage" content="694" /><meta name="citation_doi" content="10.1093/dnares/dsu030" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2014/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Metagenomics: DNA sequencing of environmental samples;  citation_journal_title=Nat. Rev. Genet.;  citation_year=2005;  citation_volume=6;  citation_pages=805-14; " /><meta name="citation_reference" content="citation_title=Metagenomic analyses: past and future trends;  citation_journal_title=Appl. Environ. Microbiol.;  citation_year=2011;  citation_volume=77;  citation_pages=1153-61; " /><meta name="citation_reference" content="citation_title=Tackling soil diversity with the assembly of large, complex metagenomes;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2014;  citation_volume=111;  citation_pages=4904-9; " /><meta name="citation_reference" content="citation_title=Fungal community on decomposing leaf litter undergoes rapid successional changes;  citation_journal_title=ISME J.;  citation_year=2013;  citation_volume=7;  citation_pages=477-86; " /><meta name="citation_reference" content="citation_title=Patchiness and spatial distribution of laccase genes of ectomycorrhizal, saprotrophic, and unknown basidiomycetes in the upper horizons of a mixed forest cambisol;  citation_journal_title=Microb. Ecol.;  citation_year=2005;  citation_volume=50;  citation_pages=570-9; " /><meta name="citation_reference" content="citation_title=Widespread occurrence of expressed fungal secretory peroxidases in forest soils;  citation_journal_title=PLoS ONE;  citation_year=2014;  citation_volume=9;  citation_pages=e95557" /><meta name="citation_reference" content="citation_title=Fungi unearthed: transcripts encoding lignocellulolytic and chitinolytic enzymes in forest soil;  citation_journal_title=PLoS ONE;  citation_year=2010;  citation_volume=5;  citation_pages=e10971" /><meta name="citation_reference" content="citation_title=Polymerase chain reaction primers miss half of rRNA microbial diversity;  citation_journal_title=ISME J.;  citation_year=2009;  citation_volume=3;  citation_pages=1365-73; " /><meta name="citation_reference" content="citation_title=Gene capture coupled to high-throughput sequencing as a strategy for targeted metagenome exploration DNA;  citation_journal_title=DNA Res.;  citation_year=2013;  citation_volume=20;  citation_pages=185-96; " /><meta name="citation_reference" content="citation_title=HiSpOD: probe design for functional DNA microarrays;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=641-8; " /><meta name="citation_reference" content="citation_title=KASpOD-a web service for highly specific and explorative oligonucleotide design;  citation_journal_title=Bioinformatics;  citation_year=2012;  citation_volume=28;  citation_pages=3161-2; " /><meta name="citation_reference" content="citation_title=Soil eukaryotic functional diversity, a metatranscriptomic approach;  citation_journal_title=ISME J.;  citation_year=2007;  citation_volume=1;  citation_pages=632-42; " /><meta name="citation_reference" content="citation_title=Screening of a soil metatranscriptomic library by functional complementation of Saccharomyces cerevisiae mutants;  citation_journal_title=Microbiol. Res.;  citation_year=2011;  citation_volume=166;  citation_pages=360-8; " /><meta name="citation_reference" content="citation_title=A novel fungal family of oligopeptide transporters identified by functional metatranscriptomics of soil eukaryotes;  citation_journal_title=ISME J.;  citation_year=2011;  citation_volume=5;  citation_pages=1871-80; " /><meta name="citation_reference" content="citation_title=Soil metatranscriptomics for mining eukaryotic heavy metal resistance genes;  citation_journal_title=Environ. Microbiol.;  citation_year=2013;  citation_volume=15;  citation_pages=2829-40; " /><meta name="citation_reference" content="citation_title=Discovery of glycoside hydrolase enzymes in an avicel-adapted forest soil fungal community by a metatranscriptomic approach;  citation_journal_title=PLoS ONE;  citation_year=2013;  citation_volume=8;  citation_pages=e55485" /><meta name="citation_reference" content="citation_title=Global biogeography of highly diverse protistan communities in soil;  citation_journal_title=ISME J.;  citation_year=2013;  citation_volume=7;  citation_pages=652-9; " /><meta name="citation_reference" content="citation_title=A first comprehensive census of fungi in soil reveals both hyperdiversity and fine-scale niche partitioning;  citation_journal_title=Ecol. Monogr.;  citation_year=2014;  citation_volume=84;  citation_pages=3-20; " /><meta name="citation_reference" content="citation_title=Who is who in litter decomposition? Metaproteomics reveals major microbial players and their biogeochemical functions;  citation_journal_title=ISME J.;  citation_year=2012;  citation_volume=6;  citation_pages=1749-62; " /><meta name="citation_reference" content="citation_title=Cellulose utilization in forest litter and soil: identification of bacterial and fungal decomposers;  citation_journal_title=FEMS Microbiol. Ecol.;  citation_year=2012;  citation_volume=80;  citation_pages=735-46; " /><meta name="citation_reference" content="citation_title=The business of biotechnology;  citation_journal_title=Ind. Biotechnol.;  citation_year=2007;  citation_volume=3;  citation_pages=269-83; " /><meta name="citation_reference" content="citation_title=Metatranscriptomics reveals the diversity of genes expressed by eukaryotes in forest soils;  citation_journal_title=PloS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e28967" /><meta name="citation_reference" content="citation_title=MycoCosm portal: gearing up for 1000 fungal genomes;  citation_journal_title=Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=D699-704; " /><meta name="citation_reference" content="citation_title=The Carbohydrate-active enzymes database (CAZy) in 2013;  citation_journal_title=Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=D490-495; " /><meta name="citation_reference" content="citation_title=Long term monitoring of soil microbiological activities in two forest sites in South tyrol in the italian alps;  citation_journal_title=Microb. Environ.;  citation_year=2014;  citation_volume=29;  citation_pages=277-85; " /><meta name="citation_reference" content="citation_title=GH11 xylanases: structure/function/properties relationships and applications;  citation_journal_title=Biotechnol. Adv.;  citation_year=2012;  citation_volume=30;  citation_pages=564-92; " /><meta name="citation_reference" content="citation_title=Structural and functional variation in soil fungal communities associated with litter bags containing maize leaf;  citation_journal_title=FEMS Microbiol. Ecol.;  citation_year=2013;  citation_volume=84;  citation_pages=519-31; " /><meta name="citation_reference" content="citation_title=A molecular method to evaluate Basidiomycete laccase gene expression in forest soils;  citation_journal_title=Geoderma;  citation_year=2005;  citation_volume=128;  citation_pages=18-27; " /><meta name="citation_reference" content="citation_title=Performance of the COX1 gene as a marker for the study of metabolically active Pezizomycotina and Agaricomycetes fungal communities from the analysis of soil RNA;  citation_journal_title=FEMS Microbiol. Ecol.;  citation_year=2010;  citation_volume=74;  citation_pages=693-705; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool;  citation_journal_title=J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing;  citation_journal_title=Nat. Biotechnol.;  citation_year=2009;  citation_volume=27;  citation_pages=182-9; " /><meta name="citation_reference" content="citation_title=PANDAseq: paired-end assembler for illumina sequences;  citation_journal_title=BMC Bioinformatics;  citation_year=2012;  citation_volume=13;  citation_pages=31" /><meta name="citation_reference" content="citation_title=Introducing mothur: open-source, platform-independent, community-supported software for describing and comparing microbial communities;  citation_journal_title=Appl. Environ. Microbiol.;  citation_year=2009;  citation_volume=75;  citation_pages=7537-41; " /><meta name="citation_reference" content="citation_title=UCHIME improves sensitivity and speed of chimera detection;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2194-200; " /><meta name="citation_reference" content="citation_title=The sequence manipulation suite: JavaScript programs for analyzing and formatting protein and DNA sequences;  citation_journal_title=Biotechniques;  citation_year=2000;  citation_volume=28;  citation_pages=1102-4; " /><meta name="citation_reference" content="citation_title=NTR1 encodes a high affinity oligopeptide transporter in Arabidopsis;  citation_journal_title=FEBS Lett.;  citation_year=1995;  citation_volume=370;  citation_pages=264-8; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY;  citation_title=Molecular cloning: a laboratory manual;  citation_year=2001; " /><meta name="citation_reference" content="citation_publisher=Cold Spring Harbor Laboratory Press, Cold Spring Harbor, NY;  citation_title=Methods in yeast genetics: a laboratory course manual;  citation_year=1990; " /><meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;  citation_journal_title=Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=1792-7; " /><meta name="citation_reference" content="citation_title=SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2010;  citation_volume=27;  citation_pages=221-4; " /><meta name="citation_reference" content="citation_title=MEGA6: Molecular Evolutionary Genetics Analysis Version 6.0;  citation_journal_title=Mol. Biol. Evol.;  citation_year=2013;  citation_volume=30;  citation_pages=2725-9; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/6/685/17463038/dsu030.pdf" /><meta name="description" content="Eukaryotic microbial communities play key functional roles in soil biology and potentially represent a rich source of natural products including biocatalysts. C" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/6/685/2754553" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/6/685/2754553" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/6/685/2754553");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu030';
    </script>
<title>
	Solution Hybrid Selection Capture for the Recovery of Functional
Full-Length Eukaryotic cDNAs From Complex Environmental Samples | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2754553" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbpuPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCcmFnYWxpbmksIENsYXVkaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIExpZmUgU2NpZW5jZXMgYW5kIFN5c3RlbXMgQmlvbG9neSwgVW5pdmVyc2l0eSBvZiBUdXJpbiwgdmlhbGUgTWF0dGlvbGkgMjUsIFR1cmluIDEwMTI1LCBJdGFseSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkVjb2xvZ2llIE1pY3JvYmllbm5lLCBVTVIgQ05SUyA1NTU3LCBVU0MgSU5SQSAxMzY0LCBVbml2ZXJzaXTDqSBkZSBMeW9uLCBVbml2ZXJzaXTDqSBMeW9uIDEsIFZpbGxldXJiYW5uZSA2OTYyMiwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmliacOocmUsIEPDqWxpbmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJFQSA0Njc4IENJREFNLCBCUCAxMDQ0OCwgQ2xlcm1vbnQgVW5pdmVyc2l0w6ksIFVuaXZlcnNpdMOpIGQnQXV2ZXJnbmUsIENsZXJtb250LUZlcnJhbmQgRi02MzAwMSwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGFyaXNvdCwgTmljb2xhcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkVBIDQ2NzggQ0lEQU0sIEJQIDEwNDQ4LCBDbGVybW9udCBVbml2ZXJzaXTDqSwgVW5pdmVyc2l0w6kgZCdBdXZlcmduZSwgQ2xlcm1vbnQtRmVycmFuZCBGLTYzMDAxLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJWYWxsb24sIExhdXJlbnQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJFY29sb2dpZSBNaWNyb2JpZW5uZSwgVU1SIENOUlMgNTU1NywgVVNDIElOUkEgMTM2NCwgVW5pdmVyc2l0w6kgZGUgTHlvbiwgVW5pdmVyc2l0w6kgTHlvbiAxLCBWaWxsZXVyYmFubmUgNjk2MjIsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBydWRlbnQsIEVsc2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJFY29sb2dpZSBNaWNyb2JpZW5uZSwgVU1SIENOUlMgNTU1NywgVVNDIElOUkEgMTM2NCwgVW5pdmVyc2l0w6kgZGUgTHlvbiwgVW5pdmVyc2l0w6kgTHlvbiAxLCBWaWxsZXVyYmFubmUgNjk2MjIsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBleXJldGFpbGxhZGUsIEVyaWMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJFQSA0Njc4IENJREFNLCBCUCAxMDQ0OCwgQ2xlcm1vbnQgVW5pdmVyc2l0w6ksIFVuaXZlcnNpdMOpIGQnQXV2ZXJnbmUsIENsZXJtb250LUZlcnJhbmQgRi02MzAwMSwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR2lybGFuZGEsIE1hcmlhbmdlbGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJFY29sb2dpZSBNaWNyb2JpZW5uZSwgVU1SIENOUlMgNTU1NywgVVNDIElOUkEgMTM2NCwgVW5pdmVyc2l0w6kgZGUgTHlvbiwgVW5pdmVyc2l0w6kgTHlvbiAxLCBWaWxsZXVyYmFubmUgNjk2MjIsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IklzdGl0dXRvIHBlciBsYSBQcm90ZXppb25lIFNvc3RlbmliaWxlIGRlbGxlIFBpYW50ZSAoSVBTUCksIENvbnNpZ2xpbyBOYXppb25hbGUgZGVsbGUgUmljZXJjaGUsIFZpYWxlIE1hdHRpb2xpIDI1LCBUdXJpbiAxMDEyNSwgSXRhbHkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQZXlyZXQsIFBpZXJyZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkVBIDQ2NzggQ0lEQU0sIEJQIDEwNDQ4LCBDbGVybW9udCBVbml2ZXJzaXTDqSwgVW5pdmVyc2l0w6kgZCdBdXZlcmduZSwgQ2xlcm1vbnQtRmVycmFuZCBGLTYzMDAxLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYXJtZWlzc2UsIFJvbGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTGlmZSBTY2llbmNlcyBhbmQgU3lzdGVtcyBCaW9sb2d5LCBVbml2ZXJzaXR5IG9mIFR1cmluLCB2aWFsZSBNYXR0aW9saSAyNSwgVHVyaW4gMTAxMjUsIEl0YWx5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRWNvbG9naWUgTWljcm9iaWVubmUsIFVNUiBDTlJTIDU1NTcsIFVTQyBJTlJBIDEzNjQsIFVuaXZlcnNpdMOpIGRlIEx5b24sIFVuaXZlcnNpdMOpIEx5b24gMSwgVmlsbGV1cmJhbm5lIDY5NjIyLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMdWlzLCBQYXRyaWNpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkVjb2xvZ2llIE1pY3JvYmllbm5lLCBVTVIgQ05SUyA1NTU3LCBVU0MgSU5SQSAxMzY0LCBVbml2ZXJzaXTDqSBkZSBMeW9uLCBVbml2ZXJzaXTDqSBMeW9uIDEsIFZpbGxldXJiYW5uZSA2OTYyMiwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJTb2x1dGlvbiBIeWJyaWQgU2VsZWN0aW9uIENhcHR1cmUgZm9yIHRoZSBSZWNvdmVyeSBvZiBGdW5jdGlvbmFsCkZ1bGwtTGVuZ3RoIEV1a2FyeW90aWMgY0ROQXMgRnJvbSBDb21wbGV4IEVudmlyb25tZW50YWwgU2FtcGxlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjY4NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNjk0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN1MDMwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTQvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRhZ2Vub21pY3M6IEROQSBzZXF1ZW5jaW5nIG9mIGVudmlyb25tZW50YWwgc2FtcGxlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ODA1LTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRhZ2Vub21pYyBhbmFseXNlczogcGFzdCBhbmQgZnV0dXJlIHRyZW5kczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03NzsgIGNpdGF0aW9uX3BhZ2VzPTExNTMtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRhY2tsaW5nIHNvaWwgZGl2ZXJzaXR5IHdpdGggdGhlIGFzc2VtYmx5IG9mIGxhcmdlLCBjb21wbGV4IG1ldGFnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTE7ICBjaXRhdGlvbl9wYWdlcz00OTA0LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmdhbCBjb21tdW5pdHkgb24gZGVjb21wb3NpbmcgbGVhZiBsaXR0ZXIgdW5kZXJnb2VzIHJhcGlkIHN1Y2Nlc3Npb25hbCBjaGFuZ2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JU01FIEouOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz00NzctODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdGNoaW5lc3MgYW5kIHNwYXRpYWwgZGlzdHJpYnV0aW9uIG9mIGxhY2Nhc2UgZ2VuZXMgb2YgZWN0b215Y29ycmhpemFsLCBzYXByb3Ryb3BoaWMsIGFuZCB1bmtub3duIGJhc2lkaW9teWNldGVzIGluIHRoZSB1cHBlciBob3Jpem9ucyBvZiBhIG1peGVkIGZvcmVzdCBjYW1iaXNvbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWljcm9iLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01MDsgIGNpdGF0aW9uX3BhZ2VzPTU3MC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaWRlc3ByZWFkIG9jY3VycmVuY2Ugb2YgZXhwcmVzc2VkIGZ1bmdhbCBzZWNyZXRvcnkgcGVyb3hpZGFzZXMgaW4gZm9yZXN0IHNvaWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTk1NTU3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuZ2kgdW5lYXJ0aGVkOiB0cmFuc2NyaXB0cyBlbmNvZGluZyBsaWdub2NlbGx1bG9seXRpYyBhbmQgY2hpdGlub2x5dGljIGVuenltZXMgaW4gZm9yZXN0IHNvaWw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTA5NzEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb2x5bWVyYXNlIGNoYWluIHJlYWN0aW9uIHByaW1lcnMgbWlzcyBoYWxmIG9mIHJSTkEgbWljcm9iaWFsIGRpdmVyc2l0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SVNNRSBKLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTM2NS03MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZSBjYXB0dXJlIGNvdXBsZWQgdG8gaGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmcgYXMgYSBzdHJhdGVneSBmb3IgdGFyZ2V0ZWQgbWV0YWdlbm9tZSBleHBsb3JhdGlvbiBETkE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTg1LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaVNwT0Q6IHByb2JlIGRlc2lnbiBmb3IgZnVuY3Rpb25hbCBETkEgbWljcm9hcnJheXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9NjQxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUtBU3BPRC1hIHdlYiBzZXJ2aWNlIGZvciBoaWdobHkgc3BlY2lmaWMgYW5kIGV4cGxvcmF0aXZlIG9saWdvbnVjbGVvdGlkZSBkZXNpZ247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MzE2MS0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Tb2lsIGV1a2FyeW90aWMgZnVuY3Rpb25hbCBkaXZlcnNpdHksIGEgbWV0YXRyYW5zY3JpcHRvbWljIGFwcHJvYWNoOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JU01FIEouOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz02MzItNDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNjcmVlbmluZyBvZiBhIHNvaWwgbWV0YXRyYW5zY3JpcHRvbWljIGxpYnJhcnkgYnkgZnVuY3Rpb25hbCBjb21wbGVtZW50YXRpb24gb2YgU2FjY2hhcm9teWNlcyBjZXJldmlzaWFlIG11dGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYmlvbC4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjY7ICBjaXRhdGlvbl9wYWdlcz0zNjAtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBub3ZlbCBmdW5nYWwgZmFtaWx5IG9mIG9saWdvcGVwdGlkZSB0cmFuc3BvcnRlcnMgaWRlbnRpZmllZCBieSBmdW5jdGlvbmFsIG1ldGF0cmFuc2NyaXB0b21pY3Mgb2Ygc29pbCBldWthcnlvdGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JU01FIEouOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xODcxLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Tb2lsIG1ldGF0cmFuc2NyaXB0b21pY3MgZm9yIG1pbmluZyBldWthcnlvdGljIGhlYXZ5IG1ldGFsIHJlc2lzdGFuY2UgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0yODI5LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXNjb3Zlcnkgb2YgZ2x5Y29zaWRlIGh5ZHJvbGFzZSBlbnp5bWVzIGluIGFuIGF2aWNlbC1hZGFwdGVkIGZvcmVzdCBzb2lsIGZ1bmdhbCBjb21tdW5pdHkgYnkgYSBtZXRhdHJhbnNjcmlwdG9taWMgYXBwcm9hY2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNTU0ODUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgYmlvZ2VvZ3JhcGh5IG9mIGhpZ2hseSBkaXZlcnNlIHByb3Rpc3RhbiBjb21tdW5pdGllcyBpbiBzb2lsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JU01FIEouOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz02NTItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBmaXJzdCBjb21wcmVoZW5zaXZlIGNlbnN1cyBvZiBmdW5naSBpbiBzb2lsIHJldmVhbHMgYm90aCBoeXBlcmRpdmVyc2l0eSBhbmQgZmluZS1zY2FsZSBuaWNoZSBwYXJ0aXRpb25pbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVjb2wuIE1vbm9nci47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9ODQ7ICBjaXRhdGlvbl9wYWdlcz0zLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG8gaXMgd2hvIGluIGxpdHRlciBkZWNvbXBvc2l0aW9uPyBNZXRhcHJvdGVvbWljcyByZXZlYWxzIG1ham9yIG1pY3JvYmlhbCBwbGF5ZXJzIGFuZCB0aGVpciBiaW9nZW9jaGVtaWNhbCBmdW5jdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUlTTUUgSi47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTE3NDktNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNlbGx1bG9zZSB1dGlsaXphdGlvbiBpbiBmb3Jlc3QgbGl0dGVyIGFuZCBzb2lsOiBpZGVudGlmaWNhdGlvbiBvZiBiYWN0ZXJpYWwgYW5kIGZ1bmdhbCBkZWNvbXBvc2VyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVNUyBNaWNyb2Jpb2wuIEVjb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTgwOyAgY2l0YXRpb25fcGFnZXM9NzM1LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYnVzaW5lc3Mgb2YgYmlvdGVjaG5vbG9neTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW5kLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MjY5LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRhdHJhbnNjcmlwdG9taWNzIHJldmVhbHMgdGhlIGRpdmVyc2l0eSBvZiBnZW5lcyBleHByZXNzZWQgYnkgZXVrYXJ5b3RlcyBpbiBmb3Jlc3Qgc29pbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMjg5NjciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NeWNvQ29zbSBwb3J0YWw6IGdlYXJpbmcgdXAgZm9yIDEwMDAgZnVuZ2FsIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPUQ2OTktNzA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQ2FyYm9oeWRyYXRlLWFjdGl2ZSBlbnp5bWVzIGRhdGFiYXNlIChDQVp5KSBpbiAyMDEzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz1ENDkwLTQ5NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9uZyB0ZXJtIG1vbml0b3Jpbmcgb2Ygc29pbCBtaWNyb2Jpb2xvZ2ljYWwgYWN0aXZpdGllcyBpbiB0d28gZm9yZXN0IHNpdGVzIGluIFNvdXRoIHR5cm9sIGluIHRoZSBpdGFsaWFuIGFscHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYi4gRW52aXJvbi47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0yNzctODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdIMTEgeHlsYW5hc2VzOiBzdHJ1Y3R1cmUvZnVuY3Rpb24vcHJvcGVydGllcyByZWxhdGlvbnNoaXBzIGFuZCBhcHBsaWNhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb3RlY2hub2wuIEFkdi47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz01NjQtOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyYWwgYW5kIGZ1bmN0aW9uYWwgdmFyaWF0aW9uIGluIHNvaWwgZnVuZ2FsIGNvbW11bml0aWVzIGFzc29jaWF0ZWQgd2l0aCBsaXR0ZXIgYmFncyBjb250YWluaW5nIG1haXplIGxlYWY7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFTVMgTWljcm9iaW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTUxOS0zMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBtb2xlY3VsYXIgbWV0aG9kIHRvIGV2YWx1YXRlIEJhc2lkaW9teWNldGUgbGFjY2FzZSBnZW5lIGV4cHJlc3Npb24gaW4gZm9yZXN0IHNvaWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW9kZXJtYTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjg7ICBjaXRhdGlvbl9wYWdlcz0xOC0yNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGVyZm9ybWFuY2Ugb2YgdGhlIENPWDEgZ2VuZSBhcyBhIG1hcmtlciBmb3IgdGhlIHN0dWR5IG9mIG1ldGFib2xpY2FsbHkgYWN0aXZlIFBleml6b215Y290aW5hIGFuZCBBZ2FyaWNvbXljZXRlcyBmdW5nYWwgY29tbXVuaXRpZXMgZnJvbSB0aGUgYW5hbHlzaXMgb2Ygc29pbCBSTkE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFTVMgTWljcm9iaW9sLiBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT03NDsgIGNpdGF0aW9uX3BhZ2VzPTY5My03MDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2ljIGxvY2FsIGFsaWdubWVudCBzZWFyY2ggdG9vbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTU7ICBjaXRhdGlvbl9wYWdlcz00MDMtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNvbHV0aW9uIGh5YnJpZCBzZWxlY3Rpb24gd2l0aCB1bHRyYS1sb25nIG9saWdvbnVjbGVvdGlkZXMgZm9yIG1hc3NpdmVseSBwYXJhbGxlbCB0YXJnZXRlZCBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTgyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBBTkRBc2VxOiBwYWlyZWQtZW5kIGFzc2VtYmxlciBmb3IgaWxsdW1pbmEgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0zMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJvZHVjaW5nIG1vdGh1cjogb3Blbi1zb3VyY2UsIHBsYXRmb3JtLWluZGVwZW5kZW50LCBjb21tdW5pdHktc3VwcG9ydGVkIHNvZnR3YXJlIGZvciBkZXNjcmliaW5nIGFuZCBjb21wYXJpbmcgbWljcm9iaWFsIGNvbW11bml0aWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTc1OyAgY2l0YXRpb25fcGFnZXM9NzUzNy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VUNISU1FIGltcHJvdmVzIHNlbnNpdGl2aXR5IGFuZCBzcGVlZCBvZiBjaGltZXJhIGRldGVjdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yMTk0LTIwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNlcXVlbmNlIG1hbmlwdWxhdGlvbiBzdWl0ZTogSmF2YVNjcmlwdCBwcm9ncmFtcyBmb3IgYW5hbHl6aW5nIGFuZCBmb3JtYXR0aW5nIHByb3RlaW4gYW5kIEROQSBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb3RlY2huaXF1ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0xMTAyLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5UUjEgZW5jb2RlcyBhIGhpZ2ggYWZmaW5pdHkgb2xpZ29wZXB0aWRlIHRyYW5zcG9ydGVyIGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRUJTIExldHQuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTM3MDsgIGNpdGF0aW9uX3BhZ2VzPTI2NC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q29sZCBTcHJpbmcgSGFyYm9yIExhYm9yYXRvcnkgUHJlc3MsIENvbGQgU3ByaW5nIEhhcmJvciwgTlk7ICBjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgY2xvbmluZzogYSBsYWJvcmF0b3J5IG1hbnVhbDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUNvbGQgU3ByaW5nIEhhcmJvciBMYWJvcmF0b3J5IFByZXNzLCBDb2xkIFNwcmluZyBIYXJib3IsIE5ZOyAgY2l0YXRpb25fdGl0bGU9TWV0aG9kcyBpbiB5ZWFzdCBnZW5ldGljczogYSBsYWJvcmF0b3J5IGNvdXJzZSBtYW51YWw7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1VU0NMRTogbXVsdGlwbGUgc2VxdWVuY2UgYWxpZ25tZW50IHdpdGggaGlnaCBhY2N1cmFjeSBhbmQgaGlnaCB0aHJvdWdocHV0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0xNzkyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlYVZpZXcgdmVyc2lvbiA0OiBhIG11bHRpcGxhdGZvcm0gZ3JhcGhpY2FsIHVzZXIgaW50ZXJmYWNlIGZvciBzZXF1ZW5jZSBhbGlnbm1lbnQgYW5kIHBoeWxvZ2VuZXRpYyB0cmVlIGJ1aWxkaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MjIxLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1FR0E2OiBNb2xlY3VsYXIgRXZvbHV0aW9uYXJ5IEdlbmV0aWNzIEFuYWx5c2lzIFZlcnNpb24gNi4wOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MjcyNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjEvNi82ODUvMTc0NjMwMzgvZHN1MDMwLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iRXVrYXJ5b3RpYyBtaWNyb2JpYWwgY29tbXVuaXRpZXMgcGxheSBrZXkgZnVuY3Rpb25hbCByb2xlcyBpbiBzb2lsIGJpb2xvZ3kgYW5kIHBvdGVudGlhbGx5IHJlcHJlc2VudCBhIHJpY2ggc291cmNlIG9mIG5hdHVyYWwgcHJvZHVjdHMgaW5jbHVkaW5nIGJpb2NhdGFseXN0cy4gQyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9Mjc1NDU1MyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTShYVrwwzOUasjoKDPMB2XvUmubA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f685%2f2754553" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f685%2f2754553" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=22047" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/6/1/m_cover.gif?Expires=2147483647&amp;Signature=NlMFHVRI1l1rW9qSwHlYw5oIbjXSqExnWRK4iEjBMZviRiHHauyZm8~g6MAuWSmBaY0~VScPROJjmkyKfZlDCJxwLSn0SLlEV3yXpjY3bZBPj-1qn2Aa5SIlmi51b00TUU4jvZRdwWUFIC54tMGsKuH~dAy4gBDUSIUXAPIdXD7OOYdr4Z-7FqtC5yf-hOhzAIYwuxOLZtaP7T3KIRVyHt4FkBjwVqN5mQx8H7zlYi~z~OF7ZuYB~vZ5tToG9q8yqYB1hVQC69tlDb5jZlqfp~pPUu~WPqA0aKh4xT5xse2XvAKV0yd5SScfSoD3tq2D4tv8DZpsiTb9UtAWz1LQPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/6">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87198550">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198550">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198568">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198568">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198626">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198626">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198638">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198638">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198643">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198643">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198645">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198645">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87198647">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87198647">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/6/673/2754551">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/21/6/695/2754554">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Solution Hybrid Selection Capture for the Recovery of Functional
Full-Length Eukaryotic cDNAs From Complex Environmental Samples
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Claudia Bragalini</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Life Sciences and Systems Biology</div>, <div class="insititution">University of Turin</div>, <div class="addr-line">viale Mattioli 25, Turin 10125</div>, <div class="country">Italy</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364</div>, <div class="insititution">Université de Lyon, Université Lyon 1</div>, <div class="addr-line">Villeurbanne 69622</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Claudia+Bragalini">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bragalini C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bragalini C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Claudia Bragalini</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">C&#233;line Ribi&#232;re</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">EA 4678 CIDAM, BP 10448</div>, <div class="insititution">Clermont Université, Université d'Auvergne</div>, <div class="addr-line">Clermont-Ferrand F-63001</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=C%c3%a9line+Ribi%c3%a8re">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ribi&#232;re C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ribi&#232;re C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Céline Ribière</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nicolas Parisot</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">EA 4678 CIDAM, BP 10448</div>, <div class="insititution">Clermont Université, Université d'Auvergne</div>, <div class="addr-line">Clermont-Ferrand F-63001</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nicolas+Parisot">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Parisot N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Parisot N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nicolas Parisot</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Laurent Vallon</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364</div>, <div class="insititution">Université de Lyon, Université Lyon 1</div>, <div class="addr-line">Villeurbanne 69622</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Laurent+Vallon">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Vallon L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Vallon L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Laurent Vallon</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Elsa Prudent</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364</div>, <div class="insititution">Université de Lyon, Université Lyon 1</div>, <div class="addr-line">Villeurbanne 69622</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Elsa+Prudent">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Prudent E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Prudent E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Elsa Prudent</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eric Peyretaillade</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">EA 4678 CIDAM, BP 10448</div>, <div class="insititution">Clermont Université, Université d'Auvergne</div>, <div class="addr-line">Clermont-Ferrand F-63001</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eric+Peyretaillade">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Peyretaillade E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Peyretaillade E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Eric Peyretaillade</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mariangela Girlanda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364</div>, <div class="insititution">Université de Lyon, Université Lyon 1</div>, <div class="addr-line">Villeurbanne 69622</div>, <div class="country">France</div></div><div class="aff"><div class="label -label">4</div><div class="insititution">Istituto per la Protezione Sostenibile delle Piante (IPSP), Consiglio Nazionale delle Ricerche</div>, <div class="addr-line">Viale Mattioli 25, Turin 10125</div>, <div class="country">Italy</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mariangela+Girlanda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Girlanda M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Girlanda M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mariangela Girlanda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Pierre Peyret</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">EA 4678 CIDAM, BP 10448</div>, <div class="insititution">Clermont Université, Université d'Auvergne</div>, <div class="addr-line">Clermont-Ferrand F-63001</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pierre+Peyret">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Peyret P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Peyret P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Pierre Peyret</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Roland Marmeisse</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Life Sciences and Systems Biology</div>, <div class="insititution">University of Turin</div>, <div class="addr-line">viale Mattioli 25, Turin 10125</div>, <div class="country">Italy</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364</div>, <div class="insititution">Université de Lyon, Université Lyon 1</div>, <div class="addr-line">Villeurbanne 69622</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Roland+Marmeisse">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Marmeisse R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Marmeisse R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Roland Marmeisse</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Patricia Luis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Ecologie Microbienne, UMR CNRS 5557, USC INRA 1364</div>, <div class="insititution">Université de Lyon, Université Lyon 1</div>, <div class="addr-line">Villeurbanne 69622</div>, <div class="country">France</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +33 472431050. Fax. +33 47 2431643. Email: <a href="mailto:patricia.luis@univ-lyon1.fr" target="_blank">patricia.luis@univ-lyon1.fr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Patricia+Luis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Luis P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Luis P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Patricia Luis</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Prof. Takashi Ito</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 6, 1 December 2014, Pages 685–694, <a href='https://doi.org/10.1093/dnares/dsu030'>https://doi.org/10.1093/dnares/dsu030</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 October 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">16 May 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">04 September 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2754553" href="/dnaresearch/article-pdf/21/6/685/17463038/dsu030.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Claudia Bragalini, Céline Ribière, Nicolas Parisot, Laurent Vallon, Elsa Prudent, Eric Peyretaillade, Mariangela Girlanda, Pierre Peyret, Roland Marmeisse, Patricia Luis;  Solution Hybrid Selection Capture for the Recovery of Functional
Full-Length Eukaryotic cDNAs From Complex Environmental Samples, <em>DNA Research</em>, Volume 21, Issue 6, 1 December 2014, Pages 685–694, <a href='https://doi.org/10.1093/dnares/dsu030'>https://doi.org/10.1093/dnares/dsu030</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2754553?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754553?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754553?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754553?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754553?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754553?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754553?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Solution%20Hybrid%20Selection%20Capture%20for%20the%20Recovery%20of%20Functional%0AFull-Length%20Eukaryotic%20cDNAs%20From%20Complex%20Environmental%20Samples&amp;publicationDate=2014-10-03&amp;volumeNum=21&amp;issueNum=6&amp;author=Bragalini%2C%20Claudia%3B%20Ribi%C3%A8re%2C%20C%C3%A9line&amp;startPage=685&amp;endPage=694&amp;contentId=10.1093%2Fdnares%2Fdsu030&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f685%2f2754553" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





<section class="abstract"><p>Eukaryotic microbial communities play key functional roles in soil biology and potentially represent a rich source of natural products including biocatalysts. Culture-independent molecular methods are powerful tools to isolate functional genes from uncultured microorganisms. However, none of the methods used in environmental genomics allow for a rapid isolation of numerous functional genes from eukaryotic microbial communities. We developed an original adaptation of the solution hybrid selection (SHS) for an efficient recovery of functional complementary DNAs (cDNAs) synthesized from soil-extracted polyadenylated mRNAs. This protocol was tested on the Glycoside Hydrolase 11 gene family encoding <em>endo</em>-xylanases for which we designed 35 explorative 31-mers capture probes. SHS was implemented on four soil eukaryotic cDNA pools. After two successive rounds of capture, &gt;90% of the resulting cDNAs were GH11 sequences, of which 70% (38 among 53 sequenced genes) were full length. Between 1.5 and 25% of the cloned captured sequences were expressed in <em>Saccharomyces cerevisiae</em>. Sequencing of polymerase chain reaction-amplified GH11 gene fragments from the captured sequences highlighted hundreds of phylogenetically diverse sequences that were not yet described, in public databases. This protocol offers the possibility of performing exhaustive exploration of eukaryotic gene families within microbial communities thriving in any type of environment.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="metatranscriptomics">metatranscriptomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;soil RNA&quot;">soil RNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;soil eukaryotes&quot;">soil eukaryotes</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;sequence capture&quot;">sequence capture</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;glycoside hydrolase family GH11&quot;">glycoside hydrolase family GH11</a></div>                            <h2 scrollto-destination=87198550 id="87198550" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>A common objective of many studies in the field of environmental microbiology is to evaluate the functional diversity of the complex microbial communities colonizing natural or man-made environments, fresh or marine waters, sediments, soils, digestive tracts or food products. This diversity can be apprehended through the systematic sequencing and functional annotation of DNA (metagenomics) or RNA (metatranscriptomics) molecules directly extracted from environmental samples.<sup><a reveal-id="DSU030C1" data-open="DSU030C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSU030C2" data-open="DSU030C2" class="link link-ref link-reveal xref-bibr">2</a></sup> However, as a result of the extreme taxonomic richness of most microbial communities, high-throughput shotgun sequencing of environmental nucleic acids is far from covering their full gene repertoire.<sup><a reveal-id="DSU030C3" data-open="DSU030C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Alternatively, many studies focus on specific environmental processes which, for some of them, are controlled by a limited and defined set of genes encoding key enzymes. The diversity of the corresponding gene families and of the organisms that possess and express them is classically evaluated by the systematic sequencing and taxonomic annotation of polymerase chain reaction (PCR)-amplified gene fragments from environmental DNA or RNA (metabarcoding).<sup><a reveal-id="DSU030C4" data-open="DSU030C4" class="link link-ref link-reveal xref-bibr">4</a>–<a reveal-id="DSU030C7" data-open="DSU030C7" class="link link-ref link-reveal xref-bibr">7</a></sup> This latter approach has itself well-documented limitations. One of the limitations is that the use of a single pair of degenerate primers, designed to hybridize to internal gene consensus sequences, usually fails to amplify all homologous sequences present in an environmental sample.<sup><a reveal-id="DSU030C8" data-open="DSU030C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Another, often underestimated limitation is that metabarcoding does not allow amplification of full-length functional genes. Besides limiting the number of phylogenetically informative nucleotide positions for precise phylogenetic assignment of environmental sequences, obtaining partial sequences also prevents their functional study by expression in a heterologous microbial host. Full-length functional genes are yet of importance (i) in ecology to establish potential relationships between enzyme catalytic properties (substrate range, sensitivity to physicochemical parameters) and prevailing environmental conditions and (ii) in environmental biotechnology to isolate novel biocatalysts for industrial purpose.</p><p>Recently, Denonfoux <em>et al</em>.<sup><a reveal-id="DSU030C9" data-open="DSU030C9" class="link link-ref link-reveal xref-bibr">9</a></sup> developed an alternative strategy to explore microbial communities from complex environments. Based on solution hybrid selection (hereafter referred to as SHS), this method allows for the specific recovery of large DNA fragments harbouring biomarkers of interest even from rare or unknown microorganisms. Indeed, SHS is based on the design of several oligonucleotide probes which can cover the whole gene of interest as opposed to PCR strategies targeting internal regions. Moreover, explorative probe design strategies using appropriate software such as HiSpOD<sup><a reveal-id="DSU030C10" data-open="DSU030C10" class="link link-ref link-reveal xref-bibr">10</a></sup> or KASpOD<sup><a reveal-id="DSU030C11" data-open="DSU030C11" class="link link-ref link-reveal xref-bibr">11</a></sup> allow recovering not yet described homologous sequences.<sup><a reveal-id="DSU030C9" data-open="DSU030C9" class="link link-ref link-reveal xref-bibr">9</a></sup> These probes are synthesized as biotinylated RNA oligonucleotides and hybridized, in solution, to the target gene sequences diluted among a majority of non-target DNA fragments. The hybrid molecules (biotinylated probes + target sequences) are then specifically captured by affinity binding on streptavidin-coated paramagnetic beads. SHS can be repeated several times successively to increase the enrichment in desired sequences by a factor of up to 1.7 × 10<sup>5</sup> times.<sup><a reveal-id="DSU030C9" data-open="DSU030C9" class="link link-ref link-reveal xref-bibr">9</a></sup> In environmental microbiology, the captured DNA fragments can be subjected to high-throughput sequencing. <em>In silico</em> assembly of the reads not only leads to the reconstruction of the full-length sequences of the different members of the targeted gene family, but also of their genomic environment and could therefore facilitate operon reconstructions.<sup><a reveal-id="DSU030C9" data-open="DSU030C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>In microbial ecology, SHS has thus far been successfully used to capture archaeal protein-coding genes from environmental DNA.<sup><a reveal-id="DSU030C9" data-open="DSU030C9" class="link link-ref link-reveal xref-bibr">9</a></sup> As previously discussed,<sup><a reveal-id="DSU030C12" data-open="DSU030C12" class="link link-ref link-reveal xref-bibr">12</a></sup> environmental DNA is however not the most appropriate matrix to recover full-length functional genes of eukaryotic origin, which could be easily expressed in a heterologous microbial host. Environmental polyadenylated messenger RNAs, devoid of introns, represent a better source of eukaryotic genes which, following their conversion into complementary DNAs (cDNAs), can be expressed in either bacteria or yeasts.<sup><a reveal-id="DSU030C12" data-open="DSU030C12" class="link link-ref link-reveal xref-bibr">12</a>–<a reveal-id="DSU030C16" data-open="DSU030C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p><p>Soil eukaryotes such as fungi are highly diverse,<sup><a reveal-id="DSU030C17" data-open="DSU030C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSU030C18" data-open="DSU030C18" class="link link-ref link-reveal xref-bibr">18</a></sup> play essential roles in soil biology as, for example, the main agents in plant organic matter degradation<sup><a reveal-id="DSU030C19" data-open="DSU030C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSU030C20" data-open="DSU030C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and represent a rich source of enzymes and biomolecules used in industry.<sup><a reveal-id="DSU030C21" data-open="DSU030C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Despite these obvious interests, very few environmental genomics studies specifically focus on soil eukaryote functional diversity.<sup><a reveal-id="DSU030C22" data-open="DSU030C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p><p>To promote such studies, we developed and evaluated in the present report an original adaptation of the SHS for the efficient recovery of full-length functional fungal cDNAs synthesized from soil RNA. Successful development of this technique was favoured by the ever increasing number of available fungal genomes that provide a correspondingly large number of members of specific gene families for the design of hybridization probes.<sup><a reveal-id="DSU030C23" data-open="DSU030C23" class="link link-ref link-reveal xref-bibr">23</a></sup> The fungal gene family targeted in this study is the Glycoside Hydrolase 11 (GH11) family which encode <em>endo</em>-β-1,4-xylanases (E.C. 3.2.1.8) (CAZY Carbohydrate Active Enzymes database, <a class="link link-uri" href="http://www.cazy.org">http://www.cazy.org</a>).<sup><a reveal-id="DSU030C24" data-open="DSU030C24" class="link link-ref link-reveal xref-bibr">24</a></sup> As xylan is the second most abundant polysaccharide in nature and one of the major structural polysaccharide in the plant cell wall, such enzymes have an obvious importance for soil ecology and for the degradation of plant hemicelluloses. A recent study also suggested that fungi contributed to most xylanase activity in soils.<sup><a reveal-id="DSU030C25" data-open="DSU030C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Furthermore, GH11 enzymes are also abundantly used in different industrial processes.<sup><a reveal-id="DSU030C26" data-open="DSU030C26" class="link link-ref link-reveal xref-bibr">26</a></sup> GH11 genes are present in the genomes of numerous fungi, mainly Ascomycota and Basidiomycota, and at the start of this study, &gt;300 sequences were publicly available. Furthermore, in a random shotgun sequencing of forest soil eukaryotic polyA-mRNAs, it was shown that GH11 transcripts occurred at a low frequency ranging from 0 to 1 per 10<sup>4</sup> sequences obtained.<sup><a reveal-id="DSU030C22" data-open="DSU030C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSU030C27" data-open="DSU030C27" class="link link-ref link-reveal xref-bibr">27</a></sup></p>                            <h2 scrollto-destination=87198568 id="87198568" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87198576 id="87198576" class="section-title" data-legacy-id=s2a>2.1. Soil RNA extraction and cDNA synthesis</h3>
<p>Four different forest soils from France and Italy were used in this study (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> for sites and soils characteristics). At each site, between 30 (BEW) and 60 (BRH) sieved (2 mm) soil cores were mixed together to constitute composite samples which were stored at −75°C prior to RNA extraction. RNA was extracted from 4 to 48 g of soil using protocols adapted to each soil. RNA from the Puéchabon (PUE) sample was extracted according to Luis <em>et al</em>.<sup><a reveal-id="DSU030C28" data-open="DSU030C28" class="link link-ref link-reveal xref-bibr">28</a></sup> RNA from the Breuil Spruce (BRE) and Breuil Beech (BRH) samples was extracted according to Damon <em>et al</em>.<sup><a reveal-id="DSU030C29" data-open="DSU030C29" class="link link-ref link-reveal xref-bibr">29</a></sup> RNA from the Berchidda (BEW) sample was extracted using the PowerSoil<sup>®</sup> Total RNA Isolation Kit (Mo Bio Laboratories), according to the manufacturer's instructions. All RNA samples were treated with RNase-free DNase I to remove residual DNA contaminations and quantified by spectrophotometry (ND-1000 NanoDrop<sup>®</sup>, Thermo Scientific).</p><p>Eukaryotic cDNAs were synthesized from 2 µg of total soil RNA by using the Mint-2 cDNA synthesis and amplification kit according to the manufacturer's instructions (Evrogen). First-strand synthesis was initiated at the RNA 3′ poly-A end using a modified poly-dT primer (CDS-4M). The number of PCR cycles (between 22 and 30) necessary for optimal synthesis of the double-stranded cDNA (dscDNA) was evaluated for each cDNA sample. As a result of using the Mint-2 kit, all amplified cDNAs were bordered at their 5′ end by the M1 sequence (AAGCAGTGGTATCAACGCAGAGT) and the <em>Sfi</em>IA restriction site (GGCCATTACGGCC) while, at their 3′ end, they were bordered by the <em>Sfi</em>IB restriction site (GGCCGAGGCGGCC) and the M1 sequence. dscDNA was purified by phenol–chloroform extraction, precipitated by 2.5 volume of ethanol and 0.1 volume of sodium acetate, resuspended in ultrapure water and quantified.</p>                            <h3 scrollto-destination=87198587 id="87198587" class="section-title" data-legacy-id=s2b>2.2. Capture probe design and synthesis</h3>
<p>As in July 2012, all publicly available GH11 DNA coding sequences of eukaryotic origin were identified by BLAST searches<sup><a reveal-id="DSU030C30" data-open="DSU030C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and collected from GenBank (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/genbank/">http://www.ncbi.nlm.nih.gov/genbank/</a>), the Joint Genome Institute database (<a class="link link-uri" href="http://jgi.doe.gov/">http://jgi.doe.gov/</a>), the Broad Institute genome database (<a class="link link-uri" href="http://www.broadinstitute.org/">http://www.broadinstitute.org/</a>) and CAZy (<a class="link link-uri" href="http://www.cazy.org/">http://www.cazy.org/</a>). A set of thirty-five 31-mers, degenerate capture probes, targeting the catalytic domain of the encoded proteins (pfam no. PF00457, ∼540 nucleotide long; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), was designed from a collection of 342 coding DNA sequences using the KASpOD software.<sup><a reveal-id="DSU030C11" data-open="DSU030C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Individual probe coverage ranged from 7 to 54% of the 342 sequences, leading to a probe set coverage of 90% (four allowed mismatches).</p><p>The 35 oligonucleotide probes included the specific sequences (X)<sub>31</sub> targeting cDNAs encoding GH11 and adaptor sequences at each extremities for PCR amplification: ATCGCACCAGCGTGT-(X)<sub>31</sub>-CACTGCGGCTCCTCA (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Biotinylated RNA capture probes were prepared according to the two-step procedure of Gnirke <em>et al</em>.<sup><a reveal-id="DSU030C31" data-open="DSU030C31" class="link link-ref link-reveal xref-bibr">31</a></sup> In the first step, each single-stranded DNA probe was amplified by PCR using primers complementary to the 5′ and 3′ adaptors to allow double-strand DNA formation. In the second step, agarose gel-purified double-stranded DNA probes were converted into biotinylated RNA probes by <em>in vitro</em> transcription using the MEGAScript<sup>®</sup>T7 kit (Ambion) and biotin-dUTP (TeBu Bio). RNA probes were then mixed together in equimolar amounts.</p>                            <h3 scrollto-destination=87198593 id="87198593" class="section-title" data-legacy-id=s2c>2.3. cDNA capture</h3>
<p>cDNA capture was carried out as described by Denonfoux <em>et al</em>.<sup><a reveal-id="DSU030C9" data-open="DSU030C9" class="link link-ref link-reveal xref-bibr">9</a></sup> and summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>. Briefly, 500 ng of heat denatured PCR-amplified cDNAs were hybridized to the equimolar mix of biotinylated RNA probes (500 ng) for 24 h at 65°C. Probe/cDNA hybrids were trapped by streptavidin-coated paramagnetic beads (Dynabeads<sup>®</sup> M-280 Streptavidin, Invitrogen). After different washing steps to remove unbound cDNAs, the captured cDNAs were eluted from the beads using 50 µl of 0.1 M NaOH at room temperature, neutralized with 70 µl of 1 M Tris–HCl, pH 7.5, and purified using the Qiaquick PCR purification kit (Qiagen).</p><p>Captured cDNAs were PCR amplified using primer M1 that binds at both 5′ and 3′ ends of the cDNAs. PCRs were set up using 5 µl of eluate, 200 µM of deoxynucleotides (dNTPs), 400 nM primer M1, 5 µl of reaction buffer 10× (Evrogen) and 1 µl of 50× Encyclo DNA polymerase (Evrogen) in a final volume of 50 µl. After an initial denaturation at 95°C for 1 min, cDNAs were amplified for 25 cycles comprising 15 s at 95°C, 20 s at 66°C and 3 min at 72°C. Ten independent amplifications were conducted for each sample. PCR products of the same sample were purified on QIAquick columns (Qiagen) and pooled. A second round of hybridization and PCR amplification was performed using each of the amplified cDNA samples obtained after the first hybridization capture. Purified products originating from the same cDNA sample were pooled together and quantified by spectrophotometry (NanoDrop™ 2000, Thermo Scientific). The DNA quality and size distribution of captured cDNA were assessed on an Agilent 2100 Bioanalyzer DNA 12000 chip (Agilent Technologies).</p>                            <h3 scrollto-destination=87198597 id="87198597" class="section-title" data-legacy-id=s2d>2.4. Semi-quantitative PCR</h3>
<p>Enrichment in GH11 sequences at each step of the capture protocol was evaluated by semi-quantitative PCR using different quantities of cDNAs and GH11-fungal-specific degenerate primers GH11-F (GGVAAGGGITGGAAYCNNGG) and GH11-R (TGKCGRACIGACCARTAYTG) amplifying a ±281-bp fragment (Luis P. <em>et al</em>., unpublished). PCRs were performed using 10, 1, 0.1 or 0.01 ng cDNAs obtained before and after one or two cycles of hybridization capture. Twenty-five microlitres of PCR mixes contained 1 µl of template cDNA, 2.5 µl of 10× PCR buffer without Mg (Invitrogen), 1.5 mM MgCl<sub>2</sub>, 0.8 mM of each dNTP, 0.5 µM of each primer and 1 U of <em>Taq</em> DNA polymerase (Invitrogen). After an initial denaturation at 94°C for 3 min, GH11 gene fragments were amplified for 45 cycles comprising 45 s at 94°C, 45 s at 50°C and 2 min at 72°C. After a final elongation at 72°C for 10 min, 10 µl of PCR products were run in a 1.5% ethidium bromide-stained agarose gel.</p>                            <h3 scrollto-destination=87198602 id="87198602" class="section-title" data-legacy-id=s2e>2.5. High-throughput sequencing</h3>
<p>Diversity of GH11 sequences at each step of the capture protocol was evaluated by high-throughput sequencing of GH11 PCR products obtained, as described above, using primers GH11-F and GH11-R. PCRs were performed using cDNAs obtained before and after one or two cycles of hybridization capture. Twenty-five microlitres of PCR mixes contained 10 ng of template cDNA, 2.5 µl of 10× PCR buffer without Mg (Invitrogen), 1.5 mM MgCl<sub>2</sub>, 0.8 mM of each dNTP, 0.5 µM of each primer and 1.25 U of DNA polymerase (a 24:1 mix of Invitrogen <em>Taq</em> DNA polymerase and Biorad iProof polymerase). PCR cycling conditions were as described above. Five different PCRs were prepared and run in parallel for each cDNA sample. PCR products were first checked on 1.5% agarose gel before pooling together the five replicates and purification using the QIAquick PCR purification kit (Qiagen). Paired-end sequencing (2 × 250 bp) was carried out on an Illumina MiSeq sequencer (Fasteris, Switzerland).</p><p>Paired-end reads were assembled using PandaSeq v.2.5,<sup><a reveal-id="DSU030C32" data-open="DSU030C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and all sequences containing unidentified nucleotide positions (‘N’) were filtered out. Primers and barcodes were removed using MOTHUR v.1.30.2.<sup><a reveal-id="DSU030C33" data-open="DSU030C33" class="link link-ref link-reveal xref-bibr">33</a></sup> UCHIME<sup><a reveal-id="DSU030C34" data-open="DSU030C34" class="link link-ref link-reveal xref-bibr">34</a></sup> was used for chimera detection, and sequence clusters were constructed at a 95% nucleotide sequence identity threshold. The most abundant representative sequence of each of the most abundant clusters, altogether encompassing &gt;90% of the sequences, was translated into amino acid sequence using the ORF Finder tool of the Sequence Manipulation Suite<sup><a reveal-id="DSU030C35" data-open="DSU030C35" class="link link-ref link-reveal xref-bibr">35</a></sup> (<a class="link link-uri" href="http://www.bioinformatics.org/sms2/">http://www.bioinformatics.org/sms2/</a>). Shannon diversity indices (H′) were calculated after rarefying the different data sets from the same soil to the same sequencing depth (i.e. the lowest sequencing depth of the three samples of each soil, Table <a reveal-id="DSU030TB2" data-open="DSU030TB2" class="link link-reveal link-table xref-fig">2</a>). Venn diagrams were drawn using the BioVenn tool (<a class="link link-uri" href="http://www.cmbi.ru.nl/cdd/biovenn/">http://www.cmbi.ru.nl/cdd/biovenn/</a>).</p>                            <h3 scrollto-destination=87198609 id="87198609" class="section-title" data-legacy-id=s2f>2.6. Full-length cDNA cloning and sequencing</h3>
<p>Amplified cDNAs obtained after two rounds of hybridization capture were digested by <em>Sfi</em>I (Fermentas), which recognizes two distinct <em>Sfi</em>IA and <em>Sfi</em>IB sites located at the 5′ and 3′ ends of the cDNAs, respectively. Digested cDNAs were then ligated to the <em>Sfi</em>I-digested pDR196-SfiI-Kan yeast expression vector<sup><a reveal-id="DSU030C36" data-open="DSU030C36" class="link link-ref link-reveal xref-bibr">36</a></sup> modified to contain two <em>Sfi</em>IA and <em>Sfi</em>IB sites, downstream of the <em>Saccharomyces cerevisiae PMA1</em> promoter, thus allowing the directional cloning and potential constitutive expression of the cDNAs in yeast.</p><p>Several transformed, kanamycin-resistant <em>Escherichia coli</em> (One Shot<sup>®</sup> TOP10 strain, Invitrogen) colonies from each sample were first randomly selected and subjected to colony PCR using the GH11-F and GH11-R primers to detect the presence of a GH11 cDNA insert. cDNA inserts from PCR-positive bacterial colonies were entirely sequenced by BIOFIDAL (Villeurbanne, France) using a PMA1 primer (CTCTCTTTTATACACACATTC) and additional internal primers when necessary.</p>                            <h3 scrollto-destination=87198616 id="87198616" class="section-title" data-legacy-id=s2g>2.7. Plasmid library construction, yeast transformation and functional screening</h3>
<p>For each cDNA sample, a minimum of 2,000 independent kanamycin-resistant transformed <em>E. coli</em> colonies were pooled together for plasmid extraction using the alkaline lysis method.<sup><a reveal-id="DSU030C37" data-open="DSU030C37" class="link link-ref link-reveal xref-bibr">37</a></sup> Aliquot samples of each plasmid library were used to transform the <em>S. cerevisiae</em> strain DSY-5 (<em>MATα leu2 trp1 ura3-52 his3::PGAL1-GAL4 pep4 prb1-1122</em>; Dualsystems Biotech) using a standard lithium acetate protocol.<sup><a reveal-id="DSU030C38" data-open="DSU030C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Transformed yeasts were selected on a solid yeast nitrogen base (YNB) minimal medium supplemented with glucose (2%) and amino acids, but lacking uracil. YNB agar plates were overlaid by a thin layer of the same medium containing 4 mg l<sup>−1</sup> of AZCL-xylan (Megazyme), a substrate specific for <em>endo</em>-xylanases. Plates were incubated at 30°C. Yeast colonies producing a secreted <em>endo</em>-xylanase were surrounded by a dark blue halo resulting from the hydrolysis of AZCL-xylan.</p><p>For each sample, several yeast colonies positive for <em>endo</em>-xylanase activity were picked, lysed at 95°C for 10 min in 3 µl of 20 mM NaOH and the pDR196 insert amplified by PCR using primers PMA1 and ADH (GCGAATTTCTTATGATTTATG). PCR products were sequenced by BIOFIDAL using the PMA1 primer.</p>                            <h3 scrollto-destination=87198621 id="87198621" class="section-title" data-legacy-id=s2h>2.8. Phylogenetic analyses</h3>
<p>Sequences obtained from plasmid inserts were manually edited and corrected. Deduced amino acid sequences were aligned using MUSCLE<sup><a reveal-id="DSU030C39" data-open="DSU030C39" class="link link-ref link-reveal xref-bibr">39</a></sup> to GH11 amino acid sequences obtained from public databases. Maximum likelihood phylogeny analyses were generated with the PhyML 3.0 program using the WAG substitution model as implemented in SeaView v. 4.<sup><a reveal-id="DSU030C40" data-open="DSU030C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Phylogenetic trees were drawn in MEGA v. 6.<sup><a reveal-id="DSU030C41" data-open="DSU030C41" class="link link-ref link-reveal xref-bibr">41</a></sup></p>                            <h3 scrollto-destination=87198624 id="87198624" class="section-title" data-legacy-id=s2i>2.9. Sequence accessibility</h3>
<p>Sequences from plasmid inserts are available in the EBI/DDJB/GenBank databases under accession Nos. LK932029-LK932091. Illumina MiSeq sequence reads have been deposited in the Sequence Read Archive of the EBI database under study no. PRJEB6672.</p>                            <h2 scrollto-destination=87198626 id="87198626" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87198627 id="87198627" class="section-title" data-legacy-id=s3a>3.1. GH11 cDNA capture</h3>
<p>As in July 2012, we identified and collected 342 full-length eukaryotic GH11 DNA coding sequences from public databases, from 113 fungal species and from 2 non-fungal ones. Seventy-two percent of these sequences were from Ascomycotina (85 species), 20% from Basidiomycotina (26 species) and 7% from other taxonomic groups. Prevalence of sequences from Ascomycotina is likely to reflect a greater genome sequencing effort in this taxonomic group, rather than a higher occurrence of the GH11 family among Ascomycotina.<sup><a reveal-id="DSU030C23" data-open="DSU030C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Among the publicly available sequences, those putatively full-length sequences ranged in size from 639 to 2,099 bp. Occurrence of carbohydrate-binding motives or of C-terminal, non-catalytic extensions in the encoded polypeptides accounted for most of these size variations. The 35 degenerate capture probes were exclusively designed on the shared ca. 540-bp-long conserved catalytic domain and were susceptible to hybridize to 90% of the collected sequences.</p><div class="block-child-p">SHS was performed on cDNAs synthesized from polyadenylated mRNAs extracted from four different forest soils. Electrophoregrams of all cDNAs recovered after two successive rounds of capture were characterized by a background smear of which emerged discrete bands ranging in size from 300 to 1,500 bp (Fig. <a reveal-id="DSU030F1" data-open="DSU030F1" class="link link-reveal link-table xref-fig">1</a>). <div data-id="DSU030F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/m_dsu03001.jpeg?Expires=2147483647&Signature=db43KaEdUJh2K6cMH2SCx2b1PiAkYXoonJ6DuTNEpdm~p5m10V-rjF2pHtnAI-XSgbS0zNKB0SyFwQn71129Aom1fYrmKwfxIt26TgWNvAeHBHIpVlAlIpL--oJ3ElU6NWAVQFrgPNJXUehPoYiPTEErYJBIGXGA7WXsr6S583hRNlPfEoPZ-v5Z4iBlip1bkaqjt2C~aMHVCIFCLX2YuA2Ui-9WTorjPN6oZdCUamuUX7xLxi-GZVkl25XHb5yOyJ4tfA05agtfGToDwORlXK2Z51xm5qj9NaKOadmmn7SUDimCb-3D36YZWxJn2FsVwDSu-oj0n6RkIVzkbrBUJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Electrophoretic separation of cDNAs obtained following two consecutive solution hybridization selection. Captured cDNAs from the four soil samples PUE, BRH, BRE and BEW were run on an Agilent DNA 12000 microfluidic chip. Each band could encompass one or several unique but abundant GH11 cDNAs." path-from-xml="dsu03001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198629/dsu03001.jpeg" path-from-xml="dsu03001.jpeg" target="_blank">View large</a><a section="87198629" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu03001.jpeg?Expires=2147483647&Signature=os71w8pGKuIyj~jBvEMk6c-po0qmnOE7jO5v4b94Vh0JzTwXqSjNf06JdJyHFNP2f44hxGNyvcsSGX~oQwDKhJAQYwBYa-0wPWBhMks4PLE4QNC-gQtLWjJIFRVIm-jGSJyiYFGLqy~DpHS9cVZ1zvAj3C4Uw6VVILEBNDhhpMLwJLbttFkHGTLYc9IX6S71mPHY-OqE812acF2ZPuO1XinU1tc2ErAlLLTtrOdsbF~AEdbRZeZaWsmMoQyMYg0Xo-kGoHbjsSrr28cOM9akbZZfsU7BliufGTfJ-EvlukmnvpFVNR6~6~I42L1NHk6pz0F70R1lo4limTsGwDpsCw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198629&ar=2754553&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Electrophoretic separation of cDNAs obtained following two consecutive solution hybridization selection. Captured cDNAs from the four soil samples PUE, BRH, BRE and BEW were run on an Agilent DNA 12000 microfluidic chip. Each band could encompass one or several unique but abundant GH11 cDNAs.</p></div></div><div content-id="DSU030F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/m_dsu03001.jpeg?Expires=2147483647&Signature=db43KaEdUJh2K6cMH2SCx2b1PiAkYXoonJ6DuTNEpdm~p5m10V-rjF2pHtnAI-XSgbS0zNKB0SyFwQn71129Aom1fYrmKwfxIt26TgWNvAeHBHIpVlAlIpL--oJ3ElU6NWAVQFrgPNJXUehPoYiPTEErYJBIGXGA7WXsr6S583hRNlPfEoPZ-v5Z4iBlip1bkaqjt2C~aMHVCIFCLX2YuA2Ui-9WTorjPN6oZdCUamuUX7xLxi-GZVkl25XHb5yOyJ4tfA05agtfGToDwORlXK2Z51xm5qj9NaKOadmmn7SUDimCb-3D36YZWxJn2FsVwDSu-oj0n6RkIVzkbrBUJQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Electrophoretic separation of cDNAs obtained following two consecutive solution hybridization selection. Captured cDNAs from the four soil samples PUE, BRH, BRE and BEW were run on an Agilent DNA 12000 microfluidic chip. Each band could encompass one or several unique but abundant GH11 cDNAs." path-from-xml="dsu03001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198629/dsu03001.jpeg" path-from-xml="dsu03001.jpeg" target="_blank">View large</a><a section="87198629" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu03001.jpeg?Expires=2147483647&Signature=os71w8pGKuIyj~jBvEMk6c-po0qmnOE7jO5v4b94Vh0JzTwXqSjNf06JdJyHFNP2f44hxGNyvcsSGX~oQwDKhJAQYwBYa-0wPWBhMks4PLE4QNC-gQtLWjJIFRVIm-jGSJyiYFGLqy~DpHS9cVZ1zvAj3C4Uw6VVILEBNDhhpMLwJLbttFkHGTLYc9IX6S71mPHY-OqE812acF2ZPuO1XinU1tc2ErAlLLTtrOdsbF~AEdbRZeZaWsmMoQyMYg0Xo-kGoHbjsSrr28cOM9akbZZfsU7BliufGTfJ-EvlukmnvpFVNR6~6~I42L1NHk6pz0F70R1lo4limTsGwDpsCw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87198629&ar=2754553&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu03001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Electrophoretic separation of cDNAs obtained following two consecutive solution hybridization selection. Captured cDNAs from the four soil samples PUE, BRH, BRE and BEW were run on an Agilent DNA 12000 microfluidic chip. Each band could encompass one or several unique but abundant GH11 cDNAs.</p></div></div></div><div class="block-child-p">Successful enrichment in GH11 sequences along the capture protocol was demonstrated by semi-quantitative PCR using GH11-specific PCR primers and different quantities of cDNA in the PCRs (from 10 to 0.01 ng). As illustrated in Fig. <a reveal-id="DSU030F2" data-open="DSU030F2" class="link link-reveal link-table xref-fig">2</a> for the Breuil beech forest (BRH sample) and for the other soil samples discussed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&amp;Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>, clear positive amplification of a GH11 fragment after two rounds of capture was always obtained using the lowest quantity of cDNA (0.01 ng), whereas no amplification could be observed for the same amount of cDNA prior to SHS. <div data-id="DSU030F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/m_dsu03002.jpeg?Expires=2147483647&amp;Signature=rH-hC68n~ayo87v7aA9IJ6M-VP0IWMHa357MhkHpQBx9n7iFo31uerDua3eyCmm3tynjy8XErBuolqpuvX0szuTMb-OZ2xvz1-stWWOlLX2LyUVKA8KkjwzZPQk4YIJADT5NQkq-yxYt4nSfn5EWWNaRxvhuuxtbIOhmwx9vOH8yD3NQoq6VwTv27oH1w7o7xi-DjAw2xNvNOa8GKrfpvnlzzDVHnvhhAJnK134yj-HHkcyOovv4AazYobZtLjMNiNTr3x8TBkuMJD3lPDrqyDduJ1dss70MbslIaIRQwyBH2qZYUq9cdacfzyTPswxAyWOIxK13owg1zKKg9PXdiw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Semi-quantitative PCR amplification of a 281-bp GH11 fragment using different quantities (from 10 to 0.01 ng) of BRH cDNA obtained before (H0) and after one (H1) or two (H2) cycles of hybridization. Before capture, PCR products could only be obtained using 10 ng of input cDNA. Amplifications of the PUE, BRE and BEW samples are illustrated in Supplementary Data." path-from-xml="dsu03002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198630/dsu03002.jpeg" path-from-xml="dsu03002.jpeg" target="_blank">View large</a><a section="87198630" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu03002.jpeg?Expires=2147483647&amp;Signature=ELL2~bTfCgLM1vsUpsgqQjDT27Xh6Ogr6lEbYHIX6rA1jfm~HIJSgfw50SQsdfi2j0BWdBWB23wbn-UA1yc~OcaMSRyzQNLZ8AK4SXiSGcbtOln0KsdEVYgrmYAg7MwfUf68j98D5eHvBXkCCrPFE~qMnpBFFtG-c7kVQ2rvLFOrFu50EK86~iXGENsdiiVkQCeoR1KJ~SKka6WpWWOKubKL5lP-UzQgN2hxWzdJBFLlMKoyeUsVirxm~UWTNa1G600CU1MrImupMANZJ-m5y3K-BjUxAjKlcBL81YJGgoK1UPSUPCubD8HQzcp0j-C~vWpmDIxU3gD5prh4kANxrA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87198630&amp;ar=2754553&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Semi-quantitative PCR amplification of a 281-bp GH11 fragment using different quantities (from 10 to 0.01 ng) of BRH cDNA obtained before (H0) and after one (H1) or two (H2) cycles of hybridization. Before capture, PCR products could only be obtained using 10 ng of input cDNA. Amplifications of the PUE, BRE and BEW samples are illustrated in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&amp;Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p></div></div><div content-id="DSU030F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/m_dsu03002.jpeg?Expires=2147483647&amp;Signature=rH-hC68n~ayo87v7aA9IJ6M-VP0IWMHa357MhkHpQBx9n7iFo31uerDua3eyCmm3tynjy8XErBuolqpuvX0szuTMb-OZ2xvz1-stWWOlLX2LyUVKA8KkjwzZPQk4YIJADT5NQkq-yxYt4nSfn5EWWNaRxvhuuxtbIOhmwx9vOH8yD3NQoq6VwTv27oH1w7o7xi-DjAw2xNvNOa8GKrfpvnlzzDVHnvhhAJnK134yj-HHkcyOovv4AazYobZtLjMNiNTr3x8TBkuMJD3lPDrqyDduJ1dss70MbslIaIRQwyBH2qZYUq9cdacfzyTPswxAyWOIxK13owg1zKKg9PXdiw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Semi-quantitative PCR amplification of a 281-bp GH11 fragment using different quantities (from 10 to 0.01 ng) of BRH cDNA obtained before (H0) and after one (H1) or two (H2) cycles of hybridization. Before capture, PCR products could only be obtained using 10 ng of input cDNA. Amplifications of the PUE, BRE and BEW samples are illustrated in Supplementary Data." path-from-xml="dsu03002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198630/dsu03002.jpeg" path-from-xml="dsu03002.jpeg" target="_blank">View large</a><a section="87198630" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu03002.jpeg?Expires=2147483647&amp;Signature=ELL2~bTfCgLM1vsUpsgqQjDT27Xh6Ogr6lEbYHIX6rA1jfm~HIJSgfw50SQsdfi2j0BWdBWB23wbn-UA1yc~OcaMSRyzQNLZ8AK4SXiSGcbtOln0KsdEVYgrmYAg7MwfUf68j98D5eHvBXkCCrPFE~qMnpBFFtG-c7kVQ2rvLFOrFu50EK86~iXGENsdiiVkQCeoR1KJ~SKka6WpWWOKubKL5lP-UzQgN2hxWzdJBFLlMKoyeUsVirxm~UWTNa1G600CU1MrImupMANZJ-m5y3K-BjUxAjKlcBL81YJGgoK1UPSUPCubD8HQzcp0j-C~vWpmDIxU3gD5prh4kANxrA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87198630&amp;ar=2754553&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Semi-quantitative PCR amplification of a 281-bp GH11 fragment using different quantities (from 10 to 0.01 ng) of BRH cDNA obtained before (H0) and after one (H1) or two (H2) cycles of hybridization. Before capture, PCR products could only be obtained using 10 ng of input cDNA. Amplifications of the PUE, BRE and BEW samples are illustrated in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&amp;Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p></div></div></div>                            <h3 scrollto-destination=87198631 id="87198631" class="section-title" data-legacy-id=s3b>3.2. Cloning, sequencing and heterologous expression
of captured cDNA</h3>
<p>Captured cDNAs in the range of 700–1,500 bp were cloned into the pDR196 <em>E. coli</em>/<em>S. cerevisiae</em> shuttle expression vector to constitute four soil-specific GH11-enriched plasmid libraries (Table <a reveal-id="DSU030TB1" data-open="DSU030TB1" class="link link-reveal link-table xref-fig">1</a>). Forty recombinant colonies per library were randomly screened by PCR using GH11-specific primers to evaluate the percentage of GH11-containing recombinant plasmids. Efficient enrichment occurred for all libraries with 80 to &gt;90% of positive clones (Table <a reveal-id="DSU030TB1" data-open="DSU030TB1" class="link link-reveal link-table xref-fig">1</a>). Among the 55 fully sequenced plasmid inserts from PCR- positive colonies, all but two indeed corresponded to GH11 sequences (Table <a reveal-id="DSU030TB1" data-open="DSU030TB1" class="link link-reveal link-table xref-fig">1</a>). Seventy-two percent of the sequences encoded putatively full-length GH11 polypeptides based on alignment length to known GH11 polypeptides and the presence of in-frame putative start and stop codons. Out of them, 15% were characterized by the presence of a family 1 carbohydrate-binding domain (CBM1) in a C-terminal position. <div content-id="DSU030TB1" class="table-modal"><div class="table-wrap"><div id="DSU030TB1" data-id="DSU030TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Cloning and characterization of captured GH11 cDNAs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Samples </th><th>PUE </th><th>BRH </th><th>BRE </th><th>BEW </th></tr></thead><tbody><tr><td>No. of captured cDNAs cloned in <em>Escherichia coli</em> </td><td>6,770 </td><td>2,020 </td><td>5,720 </td><td>5,880 </td></tr><tr><td>No. of <em>E. coli</em> colonies screened by PCR </td><td>40 </td><td>40 </td><td>40 </td><td>40 </td></tr><tr><td>Positive amplification of a GH11 fragment (%) </td><td>37 (92.5) </td><td>33 (82.5) </td><td>35 (87.5) </td><td>36 (90) </td></tr><tr><td>No. of inserts sequenced </td><td>12 </td><td>13 </td><td>16 </td><td>14 </td></tr><tr><td>No. of GH11 inserts (%) </td><td>11 (92) </td><td>12 (92) </td><td>16 (100) </td><td>14 (100) </td></tr><tr><td>No. of putative full-length GH11 (%) </td><td>9 (82) </td><td>9 (75) </td><td>11 (69) </td><td>9 (64) </td></tr><tr><td>% of <em>endo</em>-xylanase-positive yeast colonies </td><td>1.5 </td><td>25 </td><td>12 </td><td>6 </td></tr></tbody></table></div><div content-id="DSU030TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Samples </th><th>PUE </th><th>BRH </th><th>BRE </th><th>BEW </th></tr></thead><tbody><tr><td>No. of captured cDNAs cloned in <em>Escherichia coli</em> </td><td>6,770 </td><td>2,020 </td><td>5,720 </td><td>5,880 </td></tr><tr><td>No. of <em>E. coli</em> colonies screened by PCR </td><td>40 </td><td>40 </td><td>40 </td><td>40 </td></tr><tr><td>Positive amplification of a GH11 fragment (%) </td><td>37 (92.5) </td><td>33 (82.5) </td><td>35 (87.5) </td><td>36 (90) </td></tr><tr><td>No. of inserts sequenced </td><td>12 </td><td>13 </td><td>16 </td><td>14 </td></tr><tr><td>No. of GH11 inserts (%) </td><td>11 (92) </td><td>12 (92) </td><td>16 (100) </td><td>14 (100) </td></tr><tr><td>No. of putative full-length GH11 (%) </td><td>9 (82) </td><td>9 (75) </td><td>11 (69) </td><td>9 (64) </td></tr><tr><td>% of <em>endo</em>-xylanase-positive yeast colonies </td><td>1.5 </td><td>25 </td><td>12 </td><td>6 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="DSU030TB1" data-id="DSU030TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Cloning and characterization of captured GH11 cDNAs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Samples </th><th>PUE </th><th>BRH </th><th>BRE </th><th>BEW </th></tr></thead><tbody><tr><td>No. of captured cDNAs cloned in <em>Escherichia coli</em> </td><td>6,770 </td><td>2,020 </td><td>5,720 </td><td>5,880 </td></tr><tr><td>No. of <em>E. coli</em> colonies screened by PCR </td><td>40 </td><td>40 </td><td>40 </td><td>40 </td></tr><tr><td>Positive amplification of a GH11 fragment (%) </td><td>37 (92.5) </td><td>33 (82.5) </td><td>35 (87.5) </td><td>36 (90) </td></tr><tr><td>No. of inserts sequenced </td><td>12 </td><td>13 </td><td>16 </td><td>14 </td></tr><tr><td>No. of GH11 inserts (%) </td><td>11 (92) </td><td>12 (92) </td><td>16 (100) </td><td>14 (100) </td></tr><tr><td>No. of putative full-length GH11 (%) </td><td>9 (82) </td><td>9 (75) </td><td>11 (69) </td><td>9 (64) </td></tr><tr><td>% of <em>endo</em>-xylanase-positive yeast colonies </td><td>1.5 </td><td>25 </td><td>12 </td><td>6 </td></tr></tbody></table></div><div content-id="DSU030TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Samples </th><th>PUE </th><th>BRH </th><th>BRE </th><th>BEW </th></tr></thead><tbody><tr><td>No. of captured cDNAs cloned in <em>Escherichia coli</em> </td><td>6,770 </td><td>2,020 </td><td>5,720 </td><td>5,880 </td></tr><tr><td>No. of <em>E. coli</em> colonies screened by PCR </td><td>40 </td><td>40 </td><td>40 </td><td>40 </td></tr><tr><td>Positive amplification of a GH11 fragment (%) </td><td>37 (92.5) </td><td>33 (82.5) </td><td>35 (87.5) </td><td>36 (90) </td></tr><tr><td>No. of inserts sequenced </td><td>12 </td><td>13 </td><td>16 </td><td>14 </td></tr><tr><td>No. of GH11 inserts (%) </td><td>11 (92) </td><td>12 (92) </td><td>16 (100) </td><td>14 (100) </td></tr><tr><td>No. of putative full-length GH11 (%) </td><td>9 (82) </td><td>9 (75) </td><td>11 (69) </td><td>9 (64) </td></tr><tr><td>% of <em>endo</em>-xylanase-positive yeast colonies </td><td>1.5 </td><td>25 </td><td>12 </td><td>6 </td></tr></tbody></table></div></div></p><p>Functional screening using <em>S. cerevisiae</em> was conducted on the four GH11-enriched plasmid libraries by plating the recombinant yeasts onto a medium supplemented with an <em>endo</em>-xylanase-specific colour reagent (AZCL-xylan). Depending on the library, between 1.5 (sample PUE) and 25% (sample BRH) of the transformed yeast colonies developed a dark blue halo demonstrating secretion of a functional <em>endo</em>-xylanase (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). All 11 sequenced plasmid inserts from these xylanase-positive yeast colonies encoded GH11 proteins (ranged between 221 and 289 amino acids in length); 5 of them had already been identified among sequences obtained from bacterial colonies and 4 had a C-terminal CBM1 domain. The percent sequence identity between the catalytic domain of the selected functional proteins and the catalytic domain of their closest Blastp hits in GenBank ranged between 69% (81% similarity) and 87% (94% similarity).</p>                            <h3 scrollto-destination=87198634 id="87198634" class="section-title" data-legacy-id=s3c>3.3. Selectivity of the SHS GH11 capture</h3>
<div class="block-child-p">To evaluate the diversity of GH11 sequences at each step of the capture protocol, we performed a high-throughput Illumina MiSeq sequencing of GH11 amplicons obtained from all four cDNA samples, prior (H0) and after one (H1) or two (H2) cycles of SHS capture. Paired-end sequence reads were assembled to reconstitute the ca. 281-bp-long amplicons. Altogether, the total data set contained 334,161 full-length amplicon sequences that were clustered at a 95% nucleotide sequence identity threshold to produce a total number of 1,458 clusters, of which 1,001 (69%) were singletons (data summarized in Table <a reveal-id="DSU030TB2" data-open="DSU030TB2" class="link link-reveal link-table xref-fig">2</a> for each sample). Each of the 12 sequence data sets (4 cDNA samples × the 3 steps of the SHS) was characterized by few dominant clusters encompassing most of the sequences and a large number of clusters each containing a few, or even a single, sequences (illustrated in Fig. <a reveal-id="DSU030F3" data-open="DSU030F3" class="link link-reveal link-table xref-fig">3</a>A for the PUE sample). None of the sequences obtained were identical to sequences deposited in databases. Only 17 of the sequence clusters, of which 14 exclusively from the BEW site, were &gt;90% identical (maximum value of 97.5%) at the nucleotide level over their entire length to GH11 genes from either the Basidiomycota <em>Tulasnella calospora</em> or the Ascomycota <em>Nectria haematococca</em> and <em>Pyrenophora teres</em>. <div content-id="DSU030TB2" class="table-modal"><div class="table-wrap"><div id="DSU030TB2" data-id="DSU030TB2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Summary statistics from Illumina MiSeq sequencing of GH11 PCR fragments amplified, for each four cDNA samples, before (H0) or after one (H1) or two (H2) hybridization capture</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample </th><th>Total no. of
sequences </th><th>Total no. of
clusters<sup>a</sup> (95%) </th><th>No. of clusters encompassing
≥90% of the sequences </th><th>Shannon diversity
index (H′)<sup>b</sup> </th><th>No. of shared clusters
between H0–H1–H2<sup>b</sup> </th></tr></thead><tbody><tr><td>PUE_H0 </td><td>12,960 </td><td>298 </td><td>52 (17%) </td><td>3.819 </td><td rowspan="3">70 (11%) </td></tr><tr><td>PUE_H1 </td><td>24,565 </td><td>227 </td><td>51 (22%) </td><td>4.015 </td></tr><tr><td>PUE_H2 </td><td>25,053 </td><td>291 </td><td>46 (16%) </td><td>3.912 </td></tr><tr><td>BRE_H0 </td><td>13,538 </td><td>87 </td><td>9 (10%) </td><td>2.254 </td><td rowspan="3">11 (5%) </td></tr><tr><td>BRE_H1 </td><td>42,000 </td><td>140 </td><td>5 (4%) </td><td>1.651 </td></tr><tr><td>BRE_H2 </td><td>46,626 </td><td>112 </td><td>6 (5%) </td><td>1.73 </td></tr><tr><td>BRH_H0 </td><td>2,765 </td><td>26 </td><td>3 (12%) </td><td>1.061 </td><td rowspan="3">5 (4%) </td></tr><tr><td>BRH_H1 </td><td>28,366 </td><td>51 </td><td>3 (6%) </td><td>1.234 </td></tr><tr><td>BRH_H2 </td><td>17,322 </td><td>159 </td><td>18 (11%) </td><td>2.135 </td></tr><tr><td>BEW_H0 </td><td>41,799 </td><td>214 </td><td>15 (7%) </td><td>2.761 </td><td rowspan="3">38 (6%) </td></tr><tr><td>BEW_H1 </td><td>42,308 </td><td>249 </td><td>10 (4%) </td><td>2.496 </td></tr><tr><td>BEW_H2 </td><td>36,859 </td><td>205 </td><td>6 (3%) </td><td>2.196 </td></tr></tbody></table></div><div content-id="DSU030TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample </th><th>Total no. of
sequences </th><th>Total no. of
clusters<sup>a</sup> (95%) </th><th>No. of clusters encompassing
≥90% of the sequences </th><th>Shannon diversity
index (H′)<sup>b</sup> </th><th>No. of shared clusters
between H0–H1–H2<sup>b</sup> </th></tr></thead><tbody><tr><td>PUE_H0 </td><td>12,960 </td><td>298 </td><td>52 (17%) </td><td>3.819 </td><td rowspan="3">70 (11%) </td></tr><tr><td>PUE_H1 </td><td>24,565 </td><td>227 </td><td>51 (22%) </td><td>4.015 </td></tr><tr><td>PUE_H2 </td><td>25,053 </td><td>291 </td><td>46 (16%) </td><td>3.912 </td></tr><tr><td>BRE_H0 </td><td>13,538 </td><td>87 </td><td>9 (10%) </td><td>2.254 </td><td rowspan="3">11 (5%) </td></tr><tr><td>BRE_H1 </td><td>42,000 </td><td>140 </td><td>5 (4%) </td><td>1.651 </td></tr><tr><td>BRE_H2 </td><td>46,626 </td><td>112 </td><td>6 (5%) </td><td>1.73 </td></tr><tr><td>BRH_H0 </td><td>2,765 </td><td>26 </td><td>3 (12%) </td><td>1.061 </td><td rowspan="3">5 (4%) </td></tr><tr><td>BRH_H1 </td><td>28,366 </td><td>51 </td><td>3 (6%) </td><td>1.234 </td></tr><tr><td>BRH_H2 </td><td>17,322 </td><td>159 </td><td>18 (11%) </td><td>2.135 </td></tr><tr><td>BEW_H0 </td><td>41,799 </td><td>214 </td><td>15 (7%) </td><td>2.761 </td><td rowspan="3">38 (6%) </td></tr><tr><td>BEW_H1 </td><td>42,308 </td><td>249 </td><td>10 (4%) </td><td>2.496 </td></tr><tr><td>BEW_H2 </td><td>36,859 </td><td>205 </td><td>6 (3%) </td><td>2.196 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p><sup>a</sup>Including singletons.</p></div></div><div content-id="TN2"><div class="fn"><p><sup>b</sup>Shannon diversity indices and shared clusters were calculated after rarefying the different data sets from the same soil to the same sequencing depth of 12,960, 13,538, 2,765 and 36,859 sequences for PUE, BRE, BRH and BEW, respectively.</p></div></div></div></div></div><div class="table-wrap"><div id="DSU030TB2" data-id="DSU030TB2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Summary statistics from Illumina MiSeq sequencing of GH11 PCR fragments amplified, for each four cDNA samples, before (H0) or after one (H1) or two (H2) hybridization capture</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Sample </th><th>Total no. of
sequences </th><th>Total no. of
clusters<sup>a</sup> (95%) </th><th>No. of clusters encompassing
≥90% of the sequences </th><th>Shannon diversity
index (H′)<sup>b</sup> </th><th>No. of shared clusters
between H0–H1–H2<sup>b</sup> </th></tr></thead><tbody><tr><td>PUE_H0 </td><td>12,960 </td><td>298 </td><td>52 (17%) </td><td>3.819 </td><td rowspan="3">70 (11%) </td></tr><tr><td>PUE_H1 </td><td>24,565 </td><td>227 </td><td>51 (22%) </td><td>4.015 </td></tr><tr><td>PUE_H2 </td><td>25,053 </td><td>291 </td><td>46 (16%) </td><td>3.912 </td></tr><tr><td>BRE_H0 </td><td>13,538 </td><td>87 </td><td>9 (10%) </td><td>2.254 </td><td rowspan="3">11 (5%) </td></tr><tr><td>BRE_H1 </td><td>42,000 </td><td>140 </td><td>5 (4%) </td><td>1.651 </td></tr><tr><td>BRE_H2 </td><td>46,626 </td><td>112 </td><td>6 (5%) </td><td>1.73 </td></tr><tr><td>BRH_H0 </td><td>2,765 </td><td>26 </td><td>3 (12%) </td><td>1.061 </td><td rowspan="3">5 (4%) </td></tr><tr><td>BRH_H1 </td><td>28,366 </td><td>51 </td><td>3 (6%) </td><td>1.234 </td></tr><tr><td>BRH_H2 </td><td>17,322 </td><td>159 </td><td>18 (11%) </td><td>2.135 </td></tr><tr><td>BEW_H0 </td><td>41,799 </td><td>214 </td><td>15 (7%) </td><td>2.761 </td><td rowspan="3">38 (6%) </td></tr><tr><td>BEW_H1 </td><td>42,308 </td><td>249 </td><td>10 (4%) </td><td>2.496 </td></tr><tr><td>BEW_H2 </td><td>36,859 </td><td>205 </td><td>6 (3%) </td><td>2.196 </td></tr></tbody></table></div><div content-id="DSU030TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Sample </th><th>Total no. of
sequences </th><th>Total no. of
clusters<sup>a</sup> (95%) </th><th>No. of clusters encompassing
≥90% of the sequences </th><th>Shannon diversity
index (H′)<sup>b</sup> </th><th>No. of shared clusters
between H0–H1–H2<sup>b</sup> </th></tr></thead><tbody><tr><td>PUE_H0 </td><td>12,960 </td><td>298 </td><td>52 (17%) </td><td>3.819 </td><td rowspan="3">70 (11%) </td></tr><tr><td>PUE_H1 </td><td>24,565 </td><td>227 </td><td>51 (22%) </td><td>4.015 </td></tr><tr><td>PUE_H2 </td><td>25,053 </td><td>291 </td><td>46 (16%) </td><td>3.912 </td></tr><tr><td>BRE_H0 </td><td>13,538 </td><td>87 </td><td>9 (10%) </td><td>2.254 </td><td rowspan="3">11 (5%) </td></tr><tr><td>BRE_H1 </td><td>42,000 </td><td>140 </td><td>5 (4%) </td><td>1.651 </td></tr><tr><td>BRE_H2 </td><td>46,626 </td><td>112 </td><td>6 (5%) </td><td>1.73 </td></tr><tr><td>BRH_H0 </td><td>2,765 </td><td>26 </td><td>3 (12%) </td><td>1.061 </td><td rowspan="3">5 (4%) </td></tr><tr><td>BRH_H1 </td><td>28,366 </td><td>51 </td><td>3 (6%) </td><td>1.234 </td></tr><tr><td>BRH_H2 </td><td>17,322 </td><td>159 </td><td>18 (11%) </td><td>2.135 </td></tr><tr><td>BEW_H0 </td><td>41,799 </td><td>214 </td><td>15 (7%) </td><td>2.761 </td><td rowspan="3">38 (6%) </td></tr><tr><td>BEW_H1 </td><td>42,308 </td><td>249 </td><td>10 (4%) </td><td>2.496 </td></tr><tr><td>BEW_H2 </td><td>36,859 </td><td>205 </td><td>6 (3%) </td><td>2.196 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="TN1"><div class="fn"><p><sup>a</sup>Including singletons.</p></div></div><div content-id="TN2"><div class="fn"><p><sup>b</sup>Shannon diversity indices and shared clusters were calculated after rarefying the different data sets from the same soil to the same sequencing depth of 12,960, 13,538, 2,765 and 36,859 sequences for PUE, BRE, BRH and BEW, respectively.</p></div></div></div></div><div data-id="DSU030F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/m_dsu03003.jpeg?Expires=2147483647&amp;Signature=haJb90shEV2tcXiA-HZuMeHkm2xETJfgKqi0rpmWIE6EiQTDxLlRBC6N8vED1I63oGxU5XWP0gR3Vnb54FvIXvEQLWuPEHx3gZIltCQUuh~uU9m9FDkgD62G8fSPPM1dCE47ea0og5X5xcu0zAVXyYsmNTA01SOlIlbOwJgMEnKV1mOpH7Q76dRGUivNhS91mWNYT~THwwtN7Fn~4KE-5jd6YXGy5ChB7hTGANGPRebhe86G1qAQ2bFYx3Q5XWRJkqGX-xA9spywFgjRHYdhqCoKuy~eoGj~kdT3bEwT3BuYXp7CGQMbY4CgZeSZIYF6oqCDQ2yQaOxVuHrbMHqWyA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Selectivity of the SHS capture. (A) Rank-abundance distribution of the most abundant GH11 nucleotide sequence clusters identified before (H0), or after one (H1) or two (H2) cycles of hybridization on the PUE cDNAs. Only clusters encompassing 80% of the sequences in the H0, H1 or H2 samples are shown. ‘C’ or ‘Y’ letters above bars indicate sequences obtained by random sequencing of plasmid inserts or which could be functionally expressed in yeast, respectively. (B) Venn diagram showing the number of unique or shared GH11 sequence clusters, before (H0), or after one (H1) or two (H2) cycles of hybridization on the PUE cDNAs. As in (A), only the most abundant clusters, encompassing 90% of the sequences, were used for the calculation. GH11 PCR sequences were clustered using a nucleotide sequence identity threshold of 95%. Similar Venn diagrams for the BRH, BRE and BEW samples are illustrated in Supplementary Data." path-from-xml="dsu03003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198635/dsu03003.jpeg" path-from-xml="dsu03003.jpeg" target="_blank">View large</a><a section="87198635" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu03003.jpeg?Expires=2147483647&amp;Signature=WoWhCVJUF0hILXo8SJ6PrYx0Qh9mJwwVsLfTlgeN5X15MNTMWxkkpbV-UAbV9FgUdSnbnQiPxbwKtvhuy3PpPm7KQTEj2RlfyPOrWJj3QubvAYX-~SAT0ygGMkCSIr1WxOvrpodgT0ElI6BK4s8v3rUpJb0o0y26viUf1XhFA569ICaqK82PKaUSAuhg4fLZZqHma7baNxgQ-lmB3PWZy-bG8V0izL9f4BfjRFF8VU8-AKgrn~Sugo2ytKAP1BBbAtvC8phIU8ERsnqsEP22K-j07viME90dqHgZCAJ8XEIYkFnApqhJZ6HMbDRJiKFtrIv7LWSqapQbXFuQ99YmrQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87198635&amp;ar=2754553&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Selectivity of the SHS capture. (A) Rank-abundance distribution of the most abundant GH11 nucleotide sequence clusters identified before (H0), or after one (H1) or two (H2) cycles of hybridization on the PUE cDNAs. Only clusters encompassing 80% of the sequences in the H0, H1 or H2 samples are shown. ‘C’ or ‘Y’ letters above bars indicate sequences obtained by random sequencing of plasmid inserts or which could be functionally expressed in yeast, respectively. (B) Venn diagram showing the number of unique or shared GH11 sequence clusters, before (H0), or after one (H1) or two (H2) cycles of hybridization on the PUE cDNAs. As in (A), only the most abundant clusters, encompassing 90% of the sequences, were used for the calculation. GH11 PCR sequences were clustered using a nucleotide sequence identity threshold of 95%. Similar Venn diagrams for the BRH, BRE and BEW samples are illustrated in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&amp;Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p></div></div><div content-id="DSU030F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/m_dsu03003.jpeg?Expires=2147483647&amp;Signature=haJb90shEV2tcXiA-HZuMeHkm2xETJfgKqi0rpmWIE6EiQTDxLlRBC6N8vED1I63oGxU5XWP0gR3Vnb54FvIXvEQLWuPEHx3gZIltCQUuh~uU9m9FDkgD62G8fSPPM1dCE47ea0og5X5xcu0zAVXyYsmNTA01SOlIlbOwJgMEnKV1mOpH7Q76dRGUivNhS91mWNYT~THwwtN7Fn~4KE-5jd6YXGy5ChB7hTGANGPRebhe86G1qAQ2bFYx3Q5XWRJkqGX-xA9spywFgjRHYdhqCoKuy~eoGj~kdT3bEwT3BuYXp7CGQMbY4CgZeSZIYF6oqCDQ2yQaOxVuHrbMHqWyA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Selectivity of the SHS capture. (A) Rank-abundance distribution of the most abundant GH11 nucleotide sequence clusters identified before (H0), or after one (H1) or two (H2) cycles of hybridization on the PUE cDNAs. Only clusters encompassing 80% of the sequences in the H0, H1 or H2 samples are shown. ‘C’ or ‘Y’ letters above bars indicate sequences obtained by random sequencing of plasmid inserts or which could be functionally expressed in yeast, respectively. (B) Venn diagram showing the number of unique or shared GH11 sequence clusters, before (H0), or after one (H1) or two (H2) cycles of hybridization on the PUE cDNAs. As in (A), only the most abundant clusters, encompassing 90% of the sequences, were used for the calculation. GH11 PCR sequences were clustered using a nucleotide sequence identity threshold of 95%. Similar Venn diagrams for the BRH, BRE and BEW samples are illustrated in Supplementary Data." path-from-xml="dsu03003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198635/dsu03003.jpeg" path-from-xml="dsu03003.jpeg" target="_blank">View large</a><a section="87198635" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu03003.jpeg?Expires=2147483647&amp;Signature=WoWhCVJUF0hILXo8SJ6PrYx0Qh9mJwwVsLfTlgeN5X15MNTMWxkkpbV-UAbV9FgUdSnbnQiPxbwKtvhuy3PpPm7KQTEj2RlfyPOrWJj3QubvAYX-~SAT0ygGMkCSIr1WxOvrpodgT0ElI6BK4s8v3rUpJb0o0y26viUf1XhFA569ICaqK82PKaUSAuhg4fLZZqHma7baNxgQ-lmB3PWZy-bG8V0izL9f4BfjRFF8VU8-AKgrn~Sugo2ytKAP1BBbAtvC8phIU8ERsnqsEP22K-j07viME90dqHgZCAJ8XEIYkFnApqhJZ6HMbDRJiKFtrIv7LWSqapQbXFuQ99YmrQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87198635&amp;ar=2754553&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Selectivity of the SHS capture. (A) Rank-abundance distribution of the most abundant GH11 nucleotide sequence clusters identified before (H0), or after one (H1) or two (H2) cycles of hybridization on the PUE cDNAs. Only clusters encompassing 80% of the sequences in the H0, H1 or H2 samples are shown. ‘C’ or ‘Y’ letters above bars indicate sequences obtained by random sequencing of plasmid inserts or which could be functionally expressed in yeast, respectively. (B) Venn diagram showing the number of unique or shared GH11 sequence clusters, before (H0), or after one (H1) or two (H2) cycles of hybridization on the PUE cDNAs. As in (A), only the most abundant clusters, encompassing 90% of the sequences, were used for the calculation. GH11 PCR sequences were clustered using a nucleotide sequence identity threshold of 95%. Similar Venn diagrams for the BRH, BRE and BEW samples are illustrated in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&amp;Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>.</p></div></div></div><p>Figure <a reveal-id="DSU030F3" data-open="DSU030F3" class="link link-reveal link-table xref-fig">3</a> also showed that the most abundant sequence clusters obtained after one (H1) and two (H2) cycles of capture did not, for a majority of them, correspond to the most abundant clusters present before capture (H0). Venn diagrams drawn using only these most prominent sequence clusters, encompassing altogether 90–93% of sample sequences, showed that there existed a larger overlap between the post-capture samples H1 and H2 than between the pre-capture samples H0 and H1 or H2 (Fig. <a reveal-id="DSU030F3" data-open="DSU030F3" class="link link-reveal link-table xref-fig">3</a>B). This trend was observed, to some extent, for samples BEW, BRE and PUE, but not for the BRH one which differed from the others by the dominance of only three clusters in the H0 cDNA pool which encompassed 90% of the sequenced reads (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Despite these apparent differences in sequence distribution between the pre-capture H0 and the post-capture H1 and H2 samples, sequence diversity indices, such as the Shannon index, did not differ between the pre- and post-capture sequence pools (Table <a reveal-id="DSU030TB2" data-open="DSU030TB2" class="link link-reveal link-table xref-fig">2</a>, with the exception of the BRH sample). Between 2.7% (BRE and BEW) and 15% (PUE and BRH) of the sequence clusters were shared between two sites. Eight sequence clusters were identified in all four studied sites.</p><div class="block-child-p">To address the phylogenetic diversity of the captured sequences, we first produced an amino acid sequence alignment of 62 known GH11 proteins representative of the phylogenetic diversity of this gene family. To this alignment, we added the GH11 sequences obtained by the random sequencing of plasmid inserts, the sequences producing a functional enzyme in yeast and the sequences representative of the most abundant Illumina sequence clusters before (H0) or after (H1 and H2) SHS capture. The GH11 family is a highly diversified and fast-evolving gene family and phylogenies based either on full-length protein sequence alignments or on partial alignments, as in the present case, clearly do not reflect the species phylogenies and comprise very few well-supported internal branches (Fig. <a reveal-id="DSU030F4" data-open="DSU030F4" class="link link-reveal link-table xref-fig">4</a>). Phylogenetic trees obtained for sequences from the four studied soils (Fig. <a reveal-id="DSU030F4" data-open="DSU030F4" class="link link-reveal link-table xref-fig">4</a>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&amp;Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>) all clearly showed that the captured sequences were distributed over the entire reference tree. <div data-id="DSU030F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/m_dsu03004.jpeg?Expires=2147483647&amp;Signature=0TI0zLqFbRBN3uC5UYFxlUlZn-1CIGDSv48pAR6-LFsfyst65UJvQwUj4dh6XXRPq8YP8ivrZw~JX4jIeC6fErwQSD94Z2eAKBvm1nk4IIcPJlvS0XNXWElPopnsNplADmZ53OZF6fjBoiXtNr2w7H3QFZAhpUjkoavcRTGTmu9Eiw9pCC~LlZzkhUZAos9G~o763EYzQ2Y9ysOWZzP1I7av0Wnkzyv4JhFZZy70FLpAn-NqB6oX45-~o9yLXCDF3s4G7DziP0XHh8UGFLQBsofWUToUX~TTk28EXTg-2Nl89aBUAzryJesH33xGMb1vS~9ithufo0C0TK8Rs1H~aQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic diversity of the GH11 partial amino acid sequences obtained from PUE cDNA samples. 0, 1 and 2 translated PCR sequences obtained before or after one or two cycles of hybridization. PUE sequences are scattered over the entire tree that includes representative reference sequences from Ascomycota and Basidiomycota. c, sequences obtained from Escherichia coli clones; y, sequences functionally expressed in yeast clones. PhyML tree calculation was based on an alignment of ca. 80-amino-acid-long GH11 partial sequences. Thicker internal black branches indicate bootstrap value ≥60% (1,000 replications). Full species names and accession numbers of the reference sequences are given in Supplementary DataA. Similar trees drawn using the sequences from sites BRE, BRH and BEW are illustrated in Supplementary Data B, C and D, respectively. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu03004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198637/dsu03004.jpeg" path-from-xml="dsu03004.jpeg" target="_blank">View large</a><a section="87198637" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu03004.jpeg?Expires=2147483647&amp;Signature=czzDKHVJbY23wgUomIMszy~MAEImSp0FYg9GA0ThEzGrAw65vKMf3swSA1b9Ypbr1-si6FM3ehA97d89Hk-qhFZ7-2HFX0xbTxgR6cV3n81cinDgNwlUPjYA~3jiqz2QZSJ35VsSEy1abEctoVWlHoshhV00lO~lUP8CMRdsNajDV6t3B~ztkS~5BoOU0nYVnKCpqLjFMx7N7JYdXr7EFTCuM9cj0B2odBVyPu5QkO-SJZCk4I4i747msbsyqmDAoKuPY57ZZ-KBPXFdnCSc3YI-oRUNkbb-HvAJJQapldxvzsBRa24JrGoUPHq5J4xylNbwW-~WZdoiwwXTL8q7Vg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87198637&amp;ar=2754553&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic diversity of the GH11 partial amino acid sequences obtained from PUE cDNA samples. 0, 1 and 2 translated PCR sequences obtained before or after one or two cycles of hybridization. PUE sequences are scattered over the entire tree that includes representative reference sequences from Ascomycota and Basidiomycota. c, sequences obtained from <em>Escherichia coli</em> clones; y, sequences functionally expressed in yeast clones. PhyML tree calculation was based on an alignment of ca. 80-amino-acid-long GH11 partial sequences. Thicker internal black branches indicate bootstrap value ≥60% (1,000 replications). Full species names and accession numbers of the reference sequences are given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&amp;Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>A. Similar trees drawn using the sequences from sites BRE, BRH and BEW are illustrated in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&amp;Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> B, C and D, respectively. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div><div content-id="DSU030F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/m_dsu03004.jpeg?Expires=2147483647&amp;Signature=0TI0zLqFbRBN3uC5UYFxlUlZn-1CIGDSv48pAR6-LFsfyst65UJvQwUj4dh6XXRPq8YP8ivrZw~JX4jIeC6fErwQSD94Z2eAKBvm1nk4IIcPJlvS0XNXWElPopnsNplADmZ53OZF6fjBoiXtNr2w7H3QFZAhpUjkoavcRTGTmu9Eiw9pCC~LlZzkhUZAos9G~o763EYzQ2Y9ysOWZzP1I7av0Wnkzyv4JhFZZy70FLpAn-NqB6oX45-~o9yLXCDF3s4G7DziP0XHh8UGFLQBsofWUToUX~TTk28EXTg-2Nl89aBUAzryJesH33xGMb1vS~9ithufo0C0TK8Rs1H~aQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic diversity of the GH11 partial amino acid sequences obtained from PUE cDNA samples. 0, 1 and 2 translated PCR sequences obtained before or after one or two cycles of hybridization. PUE sequences are scattered over the entire tree that includes representative reference sequences from Ascomycota and Basidiomycota. c, sequences obtained from Escherichia coli clones; y, sequences functionally expressed in yeast clones. PhyML tree calculation was based on an alignment of ca. 80-amino-acid-long GH11 partial sequences. Thicker internal black branches indicate bootstrap value ≥60% (1,000 replications). Full species names and accession numbers of the reference sequences are given in Supplementary DataA. Similar trees drawn using the sequences from sites BRE, BRH and BEW are illustrated in Supplementary Data B, C and D, respectively. This figure appears in colour in the online version of DNA Research." path-from-xml="dsu03004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87198637/dsu03004.jpeg" path-from-xml="dsu03004.jpeg" target="_blank">View large</a><a section="87198637" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu03004.jpeg?Expires=2147483647&amp;Signature=czzDKHVJbY23wgUomIMszy~MAEImSp0FYg9GA0ThEzGrAw65vKMf3swSA1b9Ypbr1-si6FM3ehA97d89Hk-qhFZ7-2HFX0xbTxgR6cV3n81cinDgNwlUPjYA~3jiqz2QZSJ35VsSEy1abEctoVWlHoshhV00lO~lUP8CMRdsNajDV6t3B~ztkS~5BoOU0nYVnKCpqLjFMx7N7JYdXr7EFTCuM9cj0B2odBVyPu5QkO-SJZCk4I4i747msbsyqmDAoKuPY57ZZ-KBPXFdnCSc3YI-oRUNkbb-HvAJJQapldxvzsBRa24JrGoUPHq5J4xylNbwW-~WZdoiwwXTL8q7Vg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87198637&amp;ar=2754553&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu03004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic diversity of the GH11 partial amino acid sequences obtained from PUE cDNA samples. 0, 1 and 2 translated PCR sequences obtained before or after one or two cycles of hybridization. PUE sequences are scattered over the entire tree that includes representative reference sequences from Ascomycota and Basidiomycota. c, sequences obtained from <em>Escherichia coli</em> clones; y, sequences functionally expressed in yeast clones. PhyML tree calculation was based on an alignment of ca. 80-amino-acid-long GH11 partial sequences. Thicker internal black branches indicate bootstrap value ≥60% (1,000 replications). Full species names and accession numbers of the reference sequences are given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&amp;Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>A. Similar trees drawn using the sequences from sites BRE, BRH and BEW are illustrated in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&amp;Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> B, C and D, respectively. This figure appears in colour in the online version of <em>DNA Research</em>.</p></div></div></div>                            <h2 scrollto-destination=87198638 id="87198638" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The results obtained clearly demonstrate that SHS represents a powerful strategy to select full-length cDNAs, representative of a specific gene family, originally diluted in a highly complex metatranscriptomic sequence pool. This protocol was successfully implemented on four different forest soil RNA samples. Based on previous estimates of the frequency of GH11 sequences among eukaryotic cDNA for two of the soils used in this study (BRE and BRH),<sup><a reveal-id="DSU030C22" data-open="DSU030C22" class="link link-ref link-reveal xref-bibr">22</a></sup> two successive cycles of SHS have the potential to enrich specific cDNA sequences by a factor of at least 10<sup><a reveal-id="DSU030C4" data-open="DSU030C4" class="link link-ref link-reveal xref-bibr">4</a></sup>. As suggested by the results of the semi-quantitative PCR, in some cases (e.g. the PUE sample, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), one cycle of capture may be sufficient to get a maximum level of enrichment, while in other cases two cycles seem required (e.g. the BRH sample, Fig. <a reveal-id="DSU030F2" data-open="DSU030F2" class="link link-reveal link-table xref-fig">2</a>).</p><p>Sequence analysis of PCR fragments amplified from pre- or post-capture cDNAs demonstrated that capture succeeded in selecting both a large number and phylogenetically diverse representatives of the selected gene family. Furthermore, none of the captured sequences appeared to be identical to already known ones which we originally used for probe design. Capture could however preferentially select sequences that were not necessarily among the most abundant in the original cDNA pool. This should be evaluated in the future by quantitative PCR. Despite explorative probe design strategy, publicly available homologous sequences at the start of the study greatly influence the capture selectivity. Probe sets utilized to capture a given biomarker should therefore be upgraded regularly, taking into account newly deposited sequences.</p><p>Thanks to the ever increasing number of published fungal genomes, representative of the phylogenetic diversity of this taxonomic group; explorative probe design strategies could be carried out to unravel the metabolic capacities of these microorganisms within different ecosystems. Besides GH11 sequences, SHS capture can be implemented for any other gene family of interest, allowing a comprehensive taxonomic or functional description of the studied microbial community. As mentioned in the introduction, sequence capture presents the advantage over PCR to give access to the full-length gene sequence, including facultative modules, not always associated to the studied catalytic domain. This was indeed the case for the GH11, for which we estimated that 72% of the captured sequences were full length and that 15% of them processed a C-terminal, fungal-specific, CBM1 module (see the CAZy database, <a class="link link-uri" href="http://www.cazy.org">http://www.cazy.org</a>). A discrepancy however existed between the estimated fraction of full-length captured GH11 cDNA and the systematically lower fraction of cDNAs which produced a functional enzyme upon expression in <em>S. cerevisiae</em>. The absence of expression in yeast can be attributed to a number of independent factors such as bias in codon usage, non-recognition by <em>S. cerevisiae</em> of the protein signal peptide necessary for correct secretion, protein misfolding or hyperglycosylation. Some of these problems could be addressed by using expression plasmids including a yeast signal peptide downstream of the cloning site and/or by using a different yeast species for protein production.</p><p>Sequencing of PCR fragments amplified from captured cDNAs also indicate that altogether the four captured cDNA samples obtained in this single study encompass a greater number of novel and different GH11 sequences than have been deposited and are available in public databases over several decades. This observation should promote the use of cDNA sequence capture (i) as a complementary approach to PCR to explore and quantify the extent of eukaryotic functional diversity in complex environments, but also (ii) as a powerful tool in environmental biotechnology to efficiently screen for enzyme variants with novel biochemical properties.</p>                            <h2 scrollto-destination=87198643 id="87198643" class="section-title" data-legacy-id=s6>Funding</h2>
<p>C.B. was supported by the University of Turin and the région Rhône-Alpes (CMIRA program); C.R. received a graduate grant from the Ministère de l'Enseignement Supérieur et de la Recherche; N.P. was funded by the Direction Générale de l'Armement and E.P. by the Agence Nationale pour la Recherche. Work was financed by the CNRS-INSU ECCO Microbien program, the INRA métaprogramme M2E (project Metascreen), project ANR 09-GENM-033-001 (Eumetasol); EU-project ‘EcoFINDERS' No. 264465 and local funding by the University of Turin (ex-60%). Funding to pay the Open Access publication charges for this article was provided by the Centre national de la recherche scientifique (CNRS).</p>                            <h2 scrollto-destination=87198645 id="87198645" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We would like to thank Richard Joffre, Jacques Ranger and Alberto Orgiazzi for soil sampling at the Puéchabon, Breuil and Berchidda sites, respectively. Audrey Dubost and Stefano Ghignone contributed to bioinformatic analyses and Jérémie Denonfoux to gene capture. We acknowledge the JGI of the US Department of Energy and the Broad Institute for making available genome data prior to their publication.</p>                            <h2 scrollto-destination=87198647 id="87198647" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU030C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tringe</div>
               <div class="given-names">S.G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Rubin</div>
               <div class="given-names">E.M.</div>
            </div>
          </div>
          <div class="article-title">Metagenomics: DNA sequencing of environmental samples</div>
          <div class="source">Nat. Rev. Genet.</div> 
          <div class="year">2005</div>
          <div class="volume">6</div>
          <div class="fpage">805</div>
          <div class="lpage">14</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomics%3A%20DNA%20sequencing%20of%20environmental%20samples&amp;author=S.G.%20Tringe&amp;author=E.M.%20Rubin&amp;publication_year=2005&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=6&amp;pages=805-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1709" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1709" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1709"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16304596" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Simon</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Daniel</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="article-title">Metagenomic analyses: past and future trends</div>
          <div class="source">Appl. Environ. Microbiol.</div> 
          <div class="year">2011</div>
          <div class="volume">77</div>
          <div class="fpage">1153</div>
          <div class="lpage">61</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomic%20analyses%3A%20past%20and%20future%20trends&amp;author=C.%20Simon&amp;author=R.%20Daniel&amp;publication_year=2011&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=77&amp;pages=1153-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.02345-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.02345-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.02345-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21169428" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Howe</div>
               <div class="given-names">A.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jansson</div>
               <div class="given-names">J.K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Malfatti</div>
               <div class="given-names">S.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tringe</div>
               <div class="given-names">S.G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Tiedje</div>
               <div class="given-names">J.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Brown</div>
               <div class="given-names">C.T.</div>
            </div>
          </div>
          <div class="article-title">Tackling soil diversity with the assembly of large, complex metagenomes</div>
          <div class="source">Proc. Natl. Acad. Sci. USA</div> 
          <div class="year">2014</div>
          <div class="volume">111</div>
          <div class="fpage">4904</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tackling%20soil%20diversity%20with%20the%20assembly%20of%20large%2C%20complex%20metagenomes&amp;author=A.C.%20Howe&amp;author=J.K.%20Jansson&amp;author=S.A.%20Malfatti&amp;author=S.G.%20Tringe&amp;author=J.M.%20Tiedje&amp;author=C.T.%20Brown&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=111&amp;pages=4904-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1402564111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1402564111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1402564111"> </span></div> </div></div></div></div></div><div content-id="DSU030C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Voříšková</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Baldrian</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="article-title">Fungal community on decomposing leaf litter undergoes rapid successional changes</div>
          <div class="source">ISME J.</div> 
          <div class="year">2013</div>
          <div class="volume">7</div>
          <div class="fpage">477</div>
          <div class="lpage">86</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fungal%20community%20on%20decomposing%20leaf%20litter%20undergoes%20rapid%20successional%20changes&amp;author=J.%20Vo%C5%99%C3%AD%C5%A1kov%C3%A1&amp;author=P.%20Baldrian&amp;publication_year=2013&amp;journal=ISME%20J.&amp;volume=7&amp;pages=477-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2012.116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2012.116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2012.116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23051693" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Luis</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kellner</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zimdars</div>
               <div class="given-names">B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Langer</div>
               <div class="given-names">U.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Martin</div>
               <div class="given-names">F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Buscot</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="article-title">Patchiness and spatial distribution of laccase genes of ectomycorrhizal, saprotrophic, and unknown basidiomycetes in the upper horizons of a mixed forest cambisol</div>
          <div class="source">Microb. Ecol.</div> 
          <div class="year">2005</div>
          <div class="volume">50</div>
          <div class="fpage">570</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Patchiness%20and%20spatial%20distribution%20of%20laccase%20genes%20of%20ectomycorrhizal%2C%20saprotrophic%2C%20and%20unknown%20basidiomycetes%20in%20the%20upper%20horizons%20of%20a%20mixed%20forest%20cambisol&amp;author=P.%20Luis&amp;author=H.%20Kellner&amp;author=B.%20Zimdars&amp;author=U.%20Langer&amp;author=F.%20Martin&amp;author=F.%20Buscot&amp;publication_year=2005&amp;journal=Microb.%20Ecol.&amp;volume=50&amp;pages=570-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00248-005-5047-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00248-005-5047-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00248-005-5047-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16341831" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kellner</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Luis</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Pecyna</div>
               <div class="given-names">M.J.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Widespread occurrence of expressed fungal secretory peroxidases in forest soils</div>
          <div class="source">PLoS ONE</div> 
          <div class="year">2014</div>
          <div class="volume">9</div>
          <div class="fpage">e95557</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20occurrence%20of%20expressed%20fungal%20secretory%20peroxidases%20in%20forest%20soils&amp;author=H.%20Kellner&amp;author=P.%20Luis&amp;author=M.J.%20Pecyna&amp;publication_year=2014&amp;journal=PLoS%20ONE&amp;volume=9&amp;pages=e95557" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0095557" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0095557" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0095557"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24763280" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kellner</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zak</div>
               <div class="given-names">D.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Vandenbol</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="article-title">Fungi unearthed: transcripts encoding lignocellulolytic and chitinolytic enzymes in forest soil</div>
          <div class="source">PLoS ONE</div> 
          <div class="year">2010</div>
          <div class="volume">5</div>
          <div class="fpage">e10971</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fungi%20unearthed%3A%20transcripts%20encoding%20lignocellulolytic%20and%20chitinolytic%20enzymes%20in%20forest%20soil&amp;author=H.%20Kellner&amp;author=D.R.%20Zak&amp;author=M.%20Vandenbol&amp;publication_year=2010&amp;journal=PLoS%20ONE&amp;volume=5&amp;pages=e10971" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0010971" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0010971" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0010971"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20532045" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hong</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bunge</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Leslin</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Jeon</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Epstein</div>
               <div class="given-names">S.S.</div>
            </div>
          </div>
          <div class="article-title">Polymerase chain reaction primers miss half of rRNA microbial diversity</div>
          <div class="source">ISME J.</div> 
          <div class="year">2009</div>
          <div class="volume">3</div>
          <div class="fpage">1365</div>
          <div class="lpage">73</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Polymerase%20chain%20reaction%20primers%20miss%20half%20of%20rRNA%20microbial%20diversity&amp;author=S.%20Hong&amp;author=J.%20Bunge&amp;author=C.%20Leslin&amp;author=S.%20Jeon&amp;author=S.S.%20Epstein&amp;publication_year=2009&amp;journal=ISME%20J.&amp;volume=3&amp;pages=1365-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2009.89" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2009.89" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2009.89"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19693101" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Denonfoux</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Parisot</div>
               <div class="given-names">N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Dugat-Bony</div>
               <div class="given-names">E.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Gene capture coupled to high-throughput sequencing as a strategy for targeted metagenome exploration DNA</div>
          <div class="source">DNA Res.</div> 
          <div class="year">2013</div>
          <div class="volume">20</div>
          <div class="fpage">185</div>
          <div class="lpage">96</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20capture%20coupled%20to%20high-throughput%20sequencing%20as%20a%20strategy%20for%20targeted%20metagenome%20exploration%20DNA&amp;author=J.%20Denonfoux&amp;author=N.%20Parisot&amp;author=E.%20Dugat-Bony&amp;publication_year=2013&amp;journal=DNA%20Res.&amp;volume=20&amp;pages=185-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23364577" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dugat-Bony</div>
               <div class="given-names">E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Missaoui</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Peyretaillade</div>
               <div class="given-names">E.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">HiSpOD: probe design for functional DNA microarrays</div>
          <div class="source">Bioinformatics</div> 
          <div class="year">2011</div>
          <div class="volume">27</div>
          <div class="fpage">641</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HiSpOD%3A%20probe%20design%20for%20functional%20DNA%20microarrays&amp;author=E.%20Dugat-Bony&amp;author=M.%20Missaoui&amp;author=E.%20Peyretaillade&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=641-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq712" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq712" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq712"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21216777" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Parisot</div>
               <div class="given-names">N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Denonfoux</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Dugat-Bony</div>
               <div class="given-names">E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Peyret</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Peyretaillade</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="article-title">KASpOD-a web service for highly specific and explorative oligonucleotide design</div>
          <div class="source">Bioinformatics</div> 
          <div class="year">2012</div>
          <div class="volume">28</div>
          <div class="fpage">3161</div>
          <div class="lpage">2</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KASpOD-a%20web%20service%20for%20highly%20specific%20and%20explorative%20oligonucleotide%20design&amp;author=N.%20Parisot&amp;author=J.%20Denonfoux&amp;author=E.%20Dugat-Bony&amp;author=P.%20Peyret&amp;author=E.%20Peyretaillade&amp;publication_year=2012&amp;journal=Bioinformatics&amp;volume=28&amp;pages=3161-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts597" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts597" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts597"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23047560" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bailly</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Fraissinet-Tachet</div>
               <div class="given-names">L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Verner</div>
               <div class="given-names">M.C.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Soil eukaryotic functional diversity, a metatranscriptomic approach</div>
          <div class="source">ISME J.</div> 
          <div class="year">2007</div>
          <div class="volume">1</div>
          <div class="fpage">632</div>
          <div class="lpage">42</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Soil%20eukaryotic%20functional%20diversity%2C%20a%20metatranscriptomic%20approach&amp;author=J.%20Bailly&amp;author=L.%20Fraissinet-Tachet&amp;author=M.C.%20Verner&amp;publication_year=2007&amp;journal=ISME%20J.&amp;volume=1&amp;pages=632-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2007.68" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2007.68" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2007.68"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18043670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kellner</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Luis</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Portetelle</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Vandenbol</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="article-title">Screening of a soil metatranscriptomic library by functional complementation of <em>Saccharomyces cerevisiae</em> mutants</div>
          <div class="source">Microbiol. Res.</div> 
          <div class="year">2011</div>
          <div class="volume">166</div>
          <div class="fpage">360</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Screening%20of%20a%20soil%20metatranscriptomic%20library%20by%20functional%20complementation%20of%20Saccharomyces%20cerevisiae%20mutants&amp;author=H.%20Kellner&amp;author=P.%20Luis&amp;author=D.%20Portetelle&amp;author=M.%20Vandenbol&amp;publication_year=2011&amp;journal=Microbiol.%20Res.&amp;volume=166&amp;pages=360-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.micres.2010.07.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.micres.2010.07.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.micres.2010.07.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20869217" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Damon</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Vallon</div>
               <div class="given-names">L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zimmermann</div>
               <div class="given-names">S.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">A novel fungal family of oligopeptide transporters identified by functional metatranscriptomics of soil eukaryotes</div>
          <div class="source">ISME J.</div> 
          <div class="year">2011</div>
          <div class="volume">5</div>
          <div class="fpage">1871</div>
          <div class="lpage">80</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20fungal%20family%20of%20oligopeptide%20transporters%20identified%20by%20functional%20metatranscriptomics%20of%20soil%20eukaryotes&amp;author=C.%20Damon&amp;author=L.%20Vallon&amp;author=S.%20Zimmermann&amp;publication_year=2011&amp;journal=ISME%20J.&amp;volume=5&amp;pages=1871-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2011.67" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2011.67" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2011.67"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21654847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lehembre</div>
               <div class="given-names">F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Doillon</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">David</div>
               <div class="given-names">E.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Soil metatranscriptomics for mining eukaryotic heavy metal resistance genes</div>
          <div class="source">Environ. Microbiol.</div> 
          <div class="year">2013</div>
          <div class="volume">15</div>
          <div class="fpage">2829</div>
          <div class="lpage">40</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Soil%20metatranscriptomics%20for%20mining%20eukaryotic%20heavy%20metal%20resistance%20genes&amp;author=F.%20Lehembre&amp;author=D.%20Doillon&amp;author=E.%20David&amp;publication_year=2013&amp;journal=Environ.%20Microbiol.&amp;volume=15&amp;pages=2829-40" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23663419" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Takasaki</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Miura</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kanno</div>
               <div class="given-names">M.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Discovery of glycoside hydrolase enzymes in an avicel-adapted forest soil fungal community by a metatranscriptomic approach</div>
          <div class="source">PLoS ONE</div> 
          <div class="year">2013</div>
          <div class="volume">8</div>
          <div class="fpage">e55485</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20glycoside%20hydrolase%20enzymes%20in%20an%20avicel-adapted%20forest%20soil%20fungal%20community%20by%20a%20metatranscriptomic%20approach&amp;author=K.%20Takasaki&amp;author=T.%20Miura&amp;author=M.%20Kanno&amp;publication_year=2013&amp;journal=PLoS%20ONE&amp;volume=8&amp;pages=e55485" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0055485" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0055485" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0055485"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23393585" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bates</div>
               <div class="given-names">S.T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Clemente</div>
               <div class="given-names">J.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Flores</div>
               <div class="given-names">G.E.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Global biogeography of highly diverse protistan communities in soil</div>
          <div class="source">ISME J.</div> 
          <div class="year">2013</div>
          <div class="volume">7</div>
          <div class="fpage">652</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20biogeography%20of%20highly%20diverse%20protistan%20communities%20in%20soil&amp;author=S.T.%20Bates&amp;author=J.C.%20Clemente&amp;author=G.E.%20Flores&amp;publication_year=2013&amp;journal=ISME%20J.&amp;volume=7&amp;pages=652-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2012.147" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2012.147" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2012.147"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23235291" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Taylor</div>
               <div class="given-names">D.L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hollingsworth</div>
               <div class="given-names">T.N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">McFarland</div>
               <div class="given-names">J.W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lennon</div>
               <div class="given-names">N.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nusbaum</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ruess</div>
               <div class="given-names">R.W.</div>
            </div>
          </div>
          <div class="article-title">A first comprehensive census of fungi in soil reveals both hyperdiversity and fine-scale niche partitioning</div>
          <div class="source">Ecol. Monogr.</div> 
          <div class="year">2014</div>
          <div class="volume">84</div>
          <div class="fpage">3</div>
          <div class="lpage">20</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20first%20comprehensive%20census%20of%20fungi%20in%20soil%20reveals%20both%20hyperdiversity%20and%20fine-scale%20niche%20partitioning&amp;author=D.L.%20Taylor&amp;author=T.N.%20Hollingsworth&amp;author=J.W.%20McFarland&amp;author=N.J.%20Lennon&amp;author=C.%20Nusbaum&amp;author=R.W.%20Ruess&amp;publication_year=2014&amp;journal=Ecol.%20Monogr.&amp;volume=84&amp;pages=3-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1890/12-1693.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1890%2F12-1693.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1890%2F12-1693.1"> </span></div> </div></div></div></div></div><div content-id="DSU030C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schneider</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Keiblinger</div>
               <div class="given-names">K.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Schmid</div>
               <div class="given-names">E.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Who is who in litter decomposition? Metaproteomics reveals major microbial players and their biogeochemical functions</div>
          <div class="source">ISME J.</div> 
          <div class="year">2012</div>
          <div class="volume">6</div>
          <div class="fpage">1749</div>
          <div class="lpage">62</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Who%20is%20who%20in%20litter%20decomposition%3F%20Metaproteomics%20reveals%20major%20microbial%20players%20and%20their%20biogeochemical%20functions&amp;author=T.%20Schneider&amp;author=K.M.%20Keiblinger&amp;author=E.%20Schmid&amp;publication_year=2012&amp;journal=ISME%20J.&amp;volume=6&amp;pages=1749-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ismej.2012.11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fismej.2012.11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fismej.2012.11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22402400" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stursová</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zifčáková</div>
               <div class="given-names">L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Leigh</div>
               <div class="given-names">M.B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Burgess</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Baldrian</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="article-title">Cellulose utilization in forest litter and soil: identification of bacterial and fungal decomposers</div>
          <div class="source">FEMS Microbiol. Ecol.</div> 
          <div class="year">2012</div>
          <div class="volume">80</div>
          <div class="fpage">735</div>
          <div class="lpage">46</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cellulose%20utilization%20in%20forest%20litter%20and%20soil%3A%20identification%20of%20bacterial%20and%20fungal%20decomposers&amp;author=M.%20Stursov%C3%A1&amp;author=L.%20Zif%C4%8D%C3%A1kov%C3%A1&amp;author=M.B.%20Leigh&amp;author=R.%20Burgess&amp;author=P.%20Baldrian&amp;publication_year=2012&amp;journal=FEMS%20Microbiol.%20Ecol.&amp;volume=80&amp;pages=735-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fem.2012.80.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffem.2012.80.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffem.2012.80.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22379979" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Demain</div>
               <div class="given-names">A.L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Dana</div>
               <div class="given-names">C.A.</div>
            </div>
          </div>
          <div class="article-title">The business of biotechnology</div>
          <div class="source">Ind. Biotechnol.</div> 
          <div class="year">2007</div>
          <div class="volume">3</div>
          <div class="fpage">269</div>
          <div class="lpage">83</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20business%20of%20biotechnology&amp;author=A.L.%20Demain&amp;author=C.A.%20Dana&amp;publication_year=2007&amp;journal=Ind.%20Biotechnol.&amp;volume=3&amp;pages=269-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/ind.2007.3.269" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Find.2007.3.269" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Find.2007.3.269"> </span></div> </div></div></div></div></div><div content-id="DSU030C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Damon</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lehembre</div>
               <div class="given-names">F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Oger-Desfeux</div>
               <div class="given-names">C.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Metatranscriptomics reveals the diversity of genes expressed by eukaryotes in forest soils</div>
          <div class="source">PloS ONE</div> 
          <div class="year">2012</div>
          <div class="volume">7</div>
          <div class="fpage">e28967</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metatranscriptomics%20reveals%20the%20diversity%20of%20genes%20expressed%20by%20eukaryotes%20in%20forest%20soils&amp;author=C.%20Damon&amp;author=F.%20Lehembre&amp;author=C.%20Oger-Desfeux&amp;publication_year=2012&amp;journal=PloS%20ONE&amp;volume=7&amp;pages=e28967" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0028967" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0028967" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0028967"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22238585" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Grigoriev</div>
               <div class="given-names">I.V.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nikitin</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Haridas</div>
               <div class="given-names">S.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">MycoCosm portal: gearing up for 1000 fungal genomes</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2014</div>
          <div class="volume">42</div>
          <div class="fpage">D699</div>
          <div class="lpage">704</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MycoCosm%20portal%3A%20gearing%20up%20for%201000%20fungal%20genomes&amp;author=I.V.%20Grigoriev&amp;author=R.%20Nikitin&amp;author=S.%20Haridas&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=D699-704" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1183" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1183" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1183"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24297253" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lombard</div>
               <div class="given-names">V.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Golaconda Ramulu</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Drula</div>
               <div class="given-names">E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Coutinho</div>
               <div class="given-names">P.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Henrissat</div>
               <div class="given-names">B.</div>
            </div>
          </div>
          <div class="article-title">The Carbohydrate-active enzymes database (CAZy) in 2013</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2014</div>
          <div class="volume">42</div>
          <div class="fpage">D490</div>
          <div class="lpage">495</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Carbohydrate-active%20enzymes%20database%20(CAZy)%20in%202013&amp;author=V.%20Lombard&amp;author=H.%20Golaconda%20Ramulu&amp;author=E.%20Drula&amp;author=P.M.%20Coutinho&amp;author=B.%20Henrissat&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=D490-495" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1178" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1178" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1178"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24270786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Margesin</div>
               <div class="given-names">R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Minerbi</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Schinner</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="article-title">Long term monitoring of soil microbiological activities in two forest sites in South tyrol in the italian alps</div>
          <div class="source">Microb. Environ.</div> 
          <div class="year">2014</div>
          <div class="volume">29</div>
          <div class="fpage">277</div>
          <div class="lpage">85</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long%20term%20monitoring%20of%20soil%20microbiological%20activities%20in%20two%20forest%20sites%20in%20South%20tyrol%20in%20the%20italian%20alps&amp;author=R.%20Margesin&amp;author=S.%20Minerbi&amp;author=F.%20Schinner&amp;publication_year=2014&amp;journal=Microb.%20Environ.&amp;volume=29&amp;pages=277-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1264/jsme2.ME14050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1264%2Fjsme2.ME14050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1264%2Fjsme2.ME14050"> </span></div> </div></div></div></div></div><div content-id="DSU030C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Paës</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Berrin</div>
               <div class="given-names">J.G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Beaugrand</div>
               <div class="given-names">J.</div>
            </div>
          </div>
          <div class="article-title">GH11 xylanases: structure/function/properties relationships and applications</div>
          <div class="source">Biotechnol. Adv.</div> 
          <div class="year">2012</div>
          <div class="volume">30</div>
          <div class="fpage">564</div>
          <div class="lpage">92</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GH11%20xylanases%3A%20structure%2Ffunction%2Fproperties%20relationships%20and%20applications&amp;author=G.%20Pa%C3%ABs&amp;author=J.G.%20Berrin&amp;author=J.%20Beaugrand&amp;publication_year=2012&amp;journal=Biotechnol.%20Adv.&amp;volume=30&amp;pages=564-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biotechadv.2011.10.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biotechadv.2011.10.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biotechadv.2011.10.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22067746" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kuramae</div>
               <div class="given-names">E.E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hillekens</div>
               <div class="given-names">R.H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">de Hollander</div>
               <div class="given-names">M.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Structural and functional variation in soil fungal communities associated with litter bags containing maize leaf</div>
          <div class="source">FEMS Microbiol. Ecol.</div> 
          <div class="year">2013</div>
          <div class="volume">84</div>
          <div class="fpage">519</div>
          <div class="lpage">31</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20and%20functional%20variation%20in%20soil%20fungal%20communities%20associated%20with%20litter%20bags%20containing%20maize%20leaf&amp;author=E.E.%20Kuramae&amp;author=R.H.%20Hillekens&amp;author=M.%20de%20Hollander&amp;publication_year=2013&amp;journal=FEMS%20Microbiol.%20Ecol.&amp;volume=84&amp;pages=519-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/femsec.2013.84.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffemsec.2013.84.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffemsec.2013.84.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23360493" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Luis</div>
               <div class="given-names">P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kellner</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Martin</div>
               <div class="given-names">F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Buscot</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="article-title">A molecular method to evaluate Basidiomycete laccase gene expression in forest soils</div>
          <div class="source">Geoderma</div> 
          <div class="year">2005</div>
          <div class="volume">128</div>
          <div class="fpage">18</div>
          <div class="lpage">27</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20molecular%20method%20to%20evaluate%20Basidiomycete%20laccase%20gene%20expression%20in%20forest%20soils&amp;author=P.%20Luis&amp;author=H.%20Kellner&amp;author=F.%20Martin&amp;author=F.%20Buscot&amp;publication_year=2005&amp;journal=Geoderma&amp;volume=128&amp;pages=18-27" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.geoderma.2004.12.023" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.geoderma.2004.12.023" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.geoderma.2004.12.023"> </span></div> </div></div></div></div></div><div content-id="DSU030C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Damon</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Barroso</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Férandon</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ranger</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Fraissinet-Tachet</div>
               <div class="given-names">L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Marmeisse</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="article-title">Performance of the COX1 gene as a marker for the study of metabolically active Pezizomycotina and Agaricomycetes fungal communities from the analysis of soil RNA</div>
          <div class="source">FEMS Microbiol. Ecol.</div> 
          <div class="year">2010</div>
          <div class="volume">74</div>
          <div class="fpage">693</div>
          <div class="lpage">705</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Performance%20of%20the%20COX1%20gene%20as%20a%20marker%20for%20the%20study%20of%20metabolically%20active%20Pezizomycotina%20and%20Agaricomycetes%20fungal%20communities%20from%20the%20analysis%20of%20soil%20RNA&amp;author=C.%20Damon&amp;author=G.%20Barroso&amp;author=C.%20F%C3%A9randon&amp;author=J.%20Ranger&amp;author=L.%20Fraissinet-Tachet&amp;author=R.%20Marmeisse&amp;publication_year=2010&amp;journal=FEMS%20Microbiol.%20Ecol.&amp;volume=74&amp;pages=693-705" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6941.2010.00983.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6941.2010.00983.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6941.2010.00983.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20977493" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Altschul</div>
               <div class="given-names">S.F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gish</div>
               <div class="given-names">W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Miller</div>
               <div class="given-names">W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Myers</div>
               <div class="given-names">E.W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lipman</div>
               <div class="given-names">D.J.</div>
            </div>
          </div>
          <div class="article-title">Basic local alignment search tool</div>
          <div class="source">J. Mol. Biol.</div> 
          <div class="year">1990</div>
          <div class="volume">215</div>
          <div class="fpage">403</div>
          <div class="lpage">10</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gnirke</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Melnikov</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Maguire</div>
               <div class="given-names">J.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Solution hybrid selection with ultra-long oligonucleotides for massively parallel targeted sequencing</div>
          <div class="source">Nat. Biotechnol.</div> 
          <div class="year">2009</div>
          <div class="volume">27</div>
          <div class="fpage">182</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Solution%20hybrid%20selection%20with%20ultra-long%20oligonucleotides%20for%20massively%20parallel%20targeted%20sequencing&amp;author=A.%20Gnirke&amp;author=A.%20Melnikov&amp;author=J.%20Maguire&amp;publication_year=2009&amp;journal=Nat.%20Biotechnol.&amp;volume=27&amp;pages=182-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1523"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19182786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Masella</div>
               <div class="given-names">A.P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bartram</div>
               <div class="given-names">A.K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Truszkowski</div>
               <div class="given-names">J.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Brown</div>
               <div class="given-names">D.G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Neufeld</div>
               <div class="given-names">J.D.</div>
            </div>
          </div>
          <div class="article-title">PANDAseq: paired-end assembler for illumina sequences</div>
          <div class="source">BMC Bioinformatics</div> 
          <div class="year">2012</div>
          <div class="volume">13</div>
          <div class="fpage">31</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PANDAseq%3A%20paired-end%20assembler%20for%20illumina%20sequences&amp;author=A.P.%20Masella&amp;author=A.K.%20Bartram&amp;author=J.M.%20Truszkowski&amp;author=D.G.%20Brown&amp;author=J.D.%20Neufeld&amp;publication_year=2012&amp;journal=BMC%20Bioinformatics&amp;volume=13&amp;pages=31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-13-31" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-13-31" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-13-31"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22333067" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Schloss</div>
               <div class="given-names">P.D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Westcott</div>
               <div class="given-names">S.L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Ryabin</div>
               <div class="given-names">T.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Introducing mothur: open-source, platform-independent, community-supported software for describing and comparing microbial communities</div>
          <div class="source">Appl. Environ. Microbiol.</div> 
          <div class="year">2009</div>
          <div class="volume">75</div>
          <div class="fpage">7537</div>
          <div class="lpage">41</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Introducing%20mothur%3A%20open-source%2C%20platform-independent%2C%20community-supported%20software%20for%20describing%20and%20comparing%20microbial%20communities&amp;author=P.D.%20Schloss&amp;author=S.L.%20Westcott&amp;author=T.%20Ryabin&amp;publication_year=2009&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=75&amp;pages=7537-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.01541-09" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01541-09" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.01541-09"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19801464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Edgar</div>
               <div class="given-names">R.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Haas</div>
               <div class="given-names">B.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Clemente</div>
               <div class="given-names">J.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Quince</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Knight</div>
               <div class="given-names">R.</div>
            </div>
          </div>
          <div class="article-title">UCHIME improves sensitivity and speed of chimera detection</div>
          <div class="source">Bioinformatics</div> 
          <div class="year">2011</div>
          <div class="volume">27</div>
          <div class="fpage">2194</div>
          <div class="lpage">200</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=UCHIME%20improves%20sensitivity%20and%20speed%20of%20chimera%20detection&amp;author=R.C.%20Edgar&amp;author=B.J.%20Haas&amp;author=J.C.%20Clemente&amp;author=C.%20Quince&amp;author=R.%20Knight&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2194-200" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr381" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr381" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr381"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21700674" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stothard</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="article-title">The sequence manipulation suite: JavaScript programs for analyzing and formatting protein and DNA sequences</div>
          <div class="source">Biotechniques</div> 
          <div class="year">2000</div>
          <div class="volume">28</div>
          <div class="fpage">1102</div>
          <div class="lpage">4</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20manipulation%20suite%3A%20JavaScript%20programs%20for%20analyzing%20and%20formatting%20protein%20and%20DNA%20sequences&amp;author=P.%20Stothard&amp;publication_year=2000&amp;journal=Biotechniques&amp;volume=28&amp;pages=1102-4" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10868275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rentsch</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Laloi</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Rouhara</div>
               <div class="given-names">I.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Schmelzer</div>
               <div class="given-names">E.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Delrot</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Frommer</div>
               <div class="given-names">W.B.</div>
            </div>
          </div>
          <div class="article-title">NTR1 encodes a high affinity oligopeptide transporter in Arabidopsis</div>
          <div class="source">FEBS Lett.</div> 
          <div class="year">1995</div>
          <div class="volume">370</div>
          <div class="fpage">264</div>
          <div class="lpage">8</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NTR1%20encodes%20a%20high%20affinity%20oligopeptide%20transporter%20in%20Arabidopsis&amp;author=D.%20Rentsch&amp;author=M.%20Laloi&amp;author=I.%20Rouhara&amp;author=E.%20Schmelzer&amp;author=S.%20Delrot&amp;author=W.B.%20Frommer&amp;publication_year=1995&amp;journal=FEBS%20Lett.&amp;volume=370&amp;pages=264-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-5793(95)00853-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-5793(95)00853-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-5793(95)00853-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7656990" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sambrook</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Russell</div>
               <div class="given-names">D.W.</div>
            </div>
          </div>
          <div class="source">Molecular cloning: a laboratory manual</div> 
          <div class="year">2001</div>
          <div class="edition">3rd edition</div>
          <div class="publisher-loc">Cold Spring Harbor, NY</div>
          <div class="publisher-name">Cold Spring Harbor Laboratory Press</div>
        </div></div></div></div><div content-id="DSU030C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Rose</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Winston</div>
               <div class="given-names">F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hieter</div>
               <div class="given-names">P.</div>
            </div>
          </div>
          <div class="source">Methods in yeast genetics: a laboratory course manual</div> 
          <div class="year">1990</div>
          <div class="publisher-loc">Cold Spring Harbor, NY</div>
          <div class="publisher-name">Cold Spring Harbor Laboratory Press</div>
        </div></div></div></div><div content-id="DSU030C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Edgar</div>
               <div class="given-names">R.C.</div>
            </div>
          </div>
          <div class="article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2004</div>
          <div class="volume">32</div>
          <div class="fpage">1792</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;author=R.C.%20Edgar&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=1792-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh340" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh340"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gouy</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Guindon</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gascuel</div>
               <div class="given-names">O.</div>
            </div>
          </div>
          <div class="article-title">SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building</div>
          <div class="source">Mol. Biol. Evol.</div> 
          <div class="year">2010</div>
          <div class="volume">27</div>
          <div class="fpage">221</div>
          <div class="lpage">4</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SeaView%20version%204%3A%20a%20multiplatform%20graphical%20user%20interface%20for%20sequence%20alignment%20and%20phylogenetic%20tree%20building&amp;author=M.%20Gouy&amp;author=S.%20Guindon&amp;author=O.%20Gascuel&amp;publication_year=2010&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=27&amp;pages=221-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msp259" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsp259" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsp259"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19854763" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU030C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU030C41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tamura</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Stecher</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Peterson</div>
               <div class="given-names">D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Filipski</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kumar</div>
               <div class="given-names">S.</div>
            </div>
          </div>
          <div class="article-title">MEGA6: Molecular Evolutionary Genetics Analysis Version 6.0</div>
          <div class="source">Mol. Biol. Evol.</div> 
          <div class="year">2013</div>
          <div class="volume">30</div>
          <div class="fpage">2725</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA6%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20Version%206.0&amp;author=K.%20Tamura&amp;author=G.%20Stecher&amp;author=D.%20Peterson&amp;author=A.%20Filipski&amp;author=S.%20Kumar&amp;publication_year=2013&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=30&amp;pages=2725-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/mst197" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmst197" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmst197"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24132122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2754553&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/6/10.1093_dnares_dsu030/2/dsu030_Supplementary_Data.zip?Expires=2147483647&amp;Signature=kNv3roRxsMc3RXXJPA7Jewe8ZXBjN8YH2a~9N70EkkfKgk-0OU~0KDaLWqn9F~mTMp7rkfHvEYa5nEkkdWazh63vjPJiYChxPZRN-ktvXbHC2ewuyXR~-~6y2navEdHkxQLQr9kikJtCEx9nsE8g9VDLHtPh7325QTPutxUlc9yo5lAdHqlifl43VuMa9LOy~2FSo~T9S0e-ngb2ISLWbzUK16-VkJyNwTTFE1kuliWZoKI208N16PkYfdVv~N3nIlmCamP94eLzI25qWi9szB1kn-hfpcPfY0kQZh2fJSXDalGCiU8wS2wfLS5h5ZdB98zdF4Mi4IoNPonpXUnJEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">336</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000347101100010&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">11</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu030"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2754553"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000347101100010&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f6%2f685%2f2754553">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000347101100010&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (11)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F6%2F685%2F2754553" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2754553'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Solution Hybrid Selection Capture for the Recovery of Functional
Full-Length Eukaryotic cDNAs From Complex Environmental Samples" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu030");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f6%2f685%2f2754553">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

