

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/3/279/2972857","@type":"ScholarlyArticle","name":"Comparative analyses of the  major royal jelly protein  gene cluster in three  Apis  species with long amplicon sequencing","datePublished":"2017-02-07","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2017-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw064","keywords":["MinION™","gene duplication","<italic>Apis mellifera</italic>","<italic>Apis florea</italic>","<italic>Apis dorsata</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/3/279/2972857","author":[{"name":"Helbing, Sophie","affiliation":"1 Institut für Biologie, Molekulare Ökologie, Martin-Luther-Universität Halle-Wittenberg, Halle (Saale), Germany","@type":"Person"},{"name":"Lattorff, H. Michael G.","affiliation":"1 Institut für Biologie, Molekulare Ökologie, Martin-Luther-Universität Halle-Wittenberg, Halle (Saale), Germany   2 German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig, Leipzig, Germany","@type":"Person"},{"name":"Moritz, Robin F.A.","affiliation":"1 Institut für Biologie, Molekulare Ökologie, Martin-Luther-Universität Halle-Wittenberg, Halle (Saale), Germany   2 German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig, Leipzig, Germany   3 Department of Zoology and Entomology, University of Pretoria,Pretoria, South Africa","@type":"Person"},{"name":"Buttstedt, Anja","affiliation":"1 Institut für Biologie, Molekulare Ökologie, Martin-Luther-Universität Halle-Wittenberg, Halle (Saale), Germany","@type":"Person"}],"description":"Abstract.  The western honeybee, Apis mellifera is a prominent model organism in the field of sociogenomics and a recent upgrade substantially improved annotati","pageStart":"279","pageEnd":"287","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Comparative analyses of the  major royal jelly protein  gene cluster in three  Apis  species with long amplicon sequencing","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Comparative analyses of the  major royal jelly protein  gene cluster in three  Apis  species with long amplicon sequencing" />
    <meta property="og:description" content="Abstract.  The western honeybee, Apis mellifera is a prominent model organism in the field of sociogenomics and a recent upgrade substantially improved annotati" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/3/279/2972857" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-02-07" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Helbing, Sophie" /><meta name="citation_author_institution" content="Institut für Biologie, Molekulare Ökologie, Martin-Luther-Universität Halle-Wittenberg, Halle (Saale), Germany" /><meta name="citation_author" content="Lattorff, H. Michael G." /><meta name="citation_author_institution" content="Institut für Biologie, Molekulare Ökologie, Martin-Luther-Universität Halle-Wittenberg, Halle (Saale), Germany" /><meta name="citation_author_institution" content="German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig, Leipzig, Germany" /><meta name="citation_author" content="Moritz, Robin F.A." /><meta name="citation_author_institution" content="Institut für Biologie, Molekulare Ökologie, Martin-Luther-Universität Halle-Wittenberg, Halle (Saale), Germany" /><meta name="citation_author_institution" content="German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig, Leipzig, Germany" /><meta name="citation_author_institution" content="Department of Zoology and Entomology, University of Pretoria,Pretoria, South Africa" /><meta name="citation_author" content="Buttstedt, Anja" /><meta name="citation_author_institution" content="Institut für Biologie, Molekulare Ökologie, Martin-Luther-Universität Halle-Wittenberg, Halle (Saale), Germany" /><meta name="citation_title" content="Comparative analyses of the major royal jelly protein gene cluster in three Apis species with long amplicon sequencing" /><meta name="citation_firstpage" content="279" /><meta name="citation_lastpage" content="287" /><meta name="citation_doi" content="10.1093/dnares/dsw064" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2017/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Beware of mis-assembled genomes;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=4320-1; " /><meta name="citation_reference" content="citation_title=Detection and correction of false segmental duplications caused by genome mis-assembly;  citation_journal_title=Genome Biol;  citation_year=2010;  citation_volume=11;  citation_pages=R28" /><meta name="citation_reference" content="citation_title=Assembly errors cause false tandem duplicate regions in the chicken (Gallus gallus) genome sequence;  citation_journal_title=Chromosoma;  citation_year=2013;  citation_volume=123;  citation_pages=165-8; " /><meta name="citation_reference" content="citation_title=Evolution of the Yellow/Major Royal Jelly Protein family and the emergence of social behavior in honey bees;  citation_journal_title=Genome Res.;  citation_year=2006;  citation_pages=1385-94; " /><meta name="citation_reference" content="citation_title=Insights into social insects from the genome of the honeybee Apis mellifera; citation_author=Honeybee Genome Sequencing Consortium;  citation_journal_title=Nature;  citation_year=2006;  citation_volume=443;  citation_pages=931-49; " /><meta name="citation_reference" content="citation_title=A family of major royal jelly proteins of the honeybee Apis mellifera L;  citation_journal_title=Cell. Mol. Life Sci. C;  citation_year=1998;  citation_volume=54;  citation_pages=1020-30; " /><meta name="citation_reference" content="citation_title=The MRJP/YELLOW protein family of Apis mellifera: Identification of new members in the EST library;  citation_journal_title=J. Insect Physiol;  citation_year=2004;  citation_volume=50;  citation_pages=51-9; " /><meta name="citation_reference" content="citation_title=Origin and function of the major royal jelly proteins of the honeybee (Apis mellifera) as members of the yellow gene family;  citation_journal_title=Biol. Rev;  citation_year=2014;  citation_volume=89;  citation_pages=255-69; " /><meta name="citation_reference" content="citation_title=Identification and partial characterization of the major royal jelly protein of the honey bee (Apis mellifera L.);  citation_journal_title=J. Apic. Res;  citation_year=1992;  citation_volume=31;  citation_pages=22-6; " /><meta name="citation_reference" content="citation_title=Comprehensive identification of novel proteins and N-glycosylation sites in royal jelly;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=135" /><meta name="citation_reference" content="citation_publisher=McGraw-Hill Book Company, New York;  citation_title=Anatomy and physiology of the honey bee;  citation_year=1925; " /><meta name="citation_reference" content="citation_title=Royalactin is not a royal making of a queen;  citation_journal_title=Nature;  citation_year=2016;  citation_volume=537;  citation_pages=E10-E12; " /><meta name="citation_reference" content="citation_title=More than royal food – major royal jelly protein genes in sexuals and workers of the honeybee Apis mellifera;  citation_journal_title=Front. Zool;  citation_year=2013;  citation_volume=10;  citation_pages=72" /><meta name="citation_reference" content="citation_title=Characterization of major royal jelly protein-like DNA sequences in Apis dorsata;  citation_journal_title=J. Apic. Res;  citation_year=2002;  citation_volume=41;  citation_pages=75-82; " /><meta name="citation_reference" content="citation_title=Organization and potential function of the mrjp3 locus in four honeybee species;  citation_journal_title=J. Agric. Food Chem;  citation_year=2005;  citation_volume=53;  citation_pages=8075-81; " /><meta name="citation_reference" content="citation_title=Cloning, expression and genomic organization of genes encoding major royal jelly protein 1 and 2 of the honey bee (Apis cerana);  citation_journal_title=J. Biochem. Mol. Biol;  citation_year=2005;  citation_volume=38;  citation_pages=49-57; " /><meta name="citation_reference" content="citation_title=Molecular cloning and analysis of four cDNAs from the heads of Apis cerana cerana nurse honeybees coding for major royal jelly proteins;  citation_journal_title=Apidologie;  citation_year=2005;  citation_volume=36;  citation_pages=389-401; " /><meta name="citation_reference" content="citation_title=Reconstructing complex regions of genomes using long-read sequencing technology;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=688-96; " /><meta name="citation_reference" content="citation_title=Resolving the complexity of the human genome using single-molecule sequencing;  citation_journal_title=Nature;  citation_year=2015;  citation_volume=517;  citation_pages=608-11; " /><meta name="citation_reference" content=" citation_journal_title=PLoS One;  citation_year=2016;  citation_volume=11;  citation_issue=6;  citation_pages=e0157600" /><meta name="citation_reference" content="citation_title=Primer3—new capabilities and interfaces;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_issue=15;  citation_pages=e11" /><meta name="citation_reference" content="citation_title=Assessing the performance of the Oxford Nanopore Technologies MinION;  citation_journal_title=Biomol. Detect. Quantif;  citation_year=2015;  citation_volume=3;  citation_pages=1-8; " /><meta name="citation_reference" content="citation_title=Genome assembly using nanopore-guided long and error-free DNA reads;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=327" /><meta name="citation_reference" content="citation_title=poRe: an R package for the visualization and analysis of nanopore sequencing data;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=114-115; " /><meta name="citation_reference" content="citation_title=MEGA5: Molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods;  citation_journal_title=Mol. Biol. Evol;  citation_year=2011;  citation_volume=28;  citation_pages=2731-9; " /><meta name="citation_reference" content="citation_title=NGS-eval: NGS Error analysis and novel sequence VAriant detection tooL;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=W301-W305; " /><meta name="citation_reference" content="citation_title=Finding the missing honey bee genes: lessons learned from a genome upgrade;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=1-29; " /><meta name="citation_reference" content="citation_title=A worldwide survey of genome sequence variation provides insight into the evolutionary history of the honeybee Apis mellifera;  citation_journal_title=Nat. Genet;  citation_year=2014;  citation_volume=46;  citation_pages=1081-8; " /><meta name="citation_reference" content="citation_title=Genomic signatures of evolutionary transitions from solitary to group living;  citation_journal_title=Science (80-.);  citation_year=2015;  citation_volume=348;  citation_pages=1139-43; " /><meta name="citation_reference" content="citation_title=The genomes of two key bumblebee species with primitive eusocial organization;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=76" /><meta name="citation_reference" content="citation_title=A dietary phytochemical alters caste-associated gene expression in honey bees;  citation_journal_title=Sci. Adv;  citation_year=2015;  citation_volume=1;  citation_pages=e1500795" /><meta name="citation_reference" content="citation_title=The transcriptomic and evolutionary signature of social interactions regulating honey bee caste development;  citation_journal_title=Ecol. Evol;  citation_year=2015;  citation_volume=5;  citation_pages=4795-807; " /><meta name="citation_reference" content="citation_title=Population genomics of the honey bee reveals strong signatures of positive selection on worker traits;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2014;  citation_volume=111;  citation_issue=7;  citation_pages=2614-9; " /><meta name="citation_reference" content="citation_title=A worldwide survey of genome sequence variation provides insight into the evolutionary history of the honeybee Apis mellifera;  citation_journal_title=Nature Genetics;  citation_year=2014;  citation_volume=46;  citation_pages=1081-8; " /><meta name="citation_reference" content="citation_title=A molecular phylogeny of the genus Apis suggests that the Giant Honey Bee of the Philippines, A. breviligula Maa, and the Plains Honey Bee of southern India, A. indica Fabricius, are valid species;  citation_journal_title=Syst. Entomol;  citation_year=2010;  citation_volume=35;  citation_pages=226-33; " /><meta name="citation_reference" content="citation_publisher=Springer-Verlag, Heidelberg, Germany;  citation_title=Evolution by gene duplication;  citation_year=1970; " /><meta name="citation_reference" content="citation_publisher=Sinauer Associates, Inc. Publishers, Sunderland, MA;  citation_title=The origins of genome architecture;  citation_year=2007; " /><meta name="citation_reference" content="citation_title=The evolution of gene duplications: classifying and distinguishing between models;  citation_journal_title=Nat. Rev. Genet;  citation_year=2010;  citation_volume=11;  citation_pages=97-108; " /><meta name="citation_reference" content="citation_title=Reduced expression of major royal jelly protein 1 gene in the mushroom bodies of worker honeybees with reduced learning ability;  citation_journal_title=Apidologie;  citation_year=2010;  citation_volume=41;  citation_pages=194-202; " /><meta name="citation_reference" content="citation_title=Genes with social effects are expected to harbor more sequence variation within and between species;  citation_journal_title=Evolution (N. Y);  citation_year=2009;  citation_volume=63;  citation_pages=1685-96; " /><meta name="citation_reference" content="citation_title=Genes involved in convergent evolution of eusociality in bees;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2011;  citation_volume=108;  citation_pages=7472-7477; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/3/279/17743956/dsw064.pdf" /><meta name="description" content="Abstract.  The western honeybee, Apis mellifera is a prominent model organism in the field of sociogenomics and a recent upgrade substantially improved annotati" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/3/279/2972857" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/3/279/2972857" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/3/279/2972857");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw064';
    </script>
<title>
	Comparative analyses of the major royal jelly protein gene cluster in three Apis species with long amplicon sequencing | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2972857" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeBjPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIZWxiaW5nLCBTb3BoaWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dCBmw7xyIEJpb2xvZ2llLCBNb2xla3VsYXJlIMOWa29sb2dpZSwgTWFydGluLUx1dGhlci1Vbml2ZXJzaXTDpHQgSGFsbGUtV2l0dGVuYmVyZywgSGFsbGUgKFNhYWxlKSwgR2VybWFueSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxhdHRvcmZmLCBILiBNaWNoYWVsIEcuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXQgZsO8ciBCaW9sb2dpZSwgTW9sZWt1bGFyZSDDlmtvbG9naWUsIE1hcnRpbi1MdXRoZXItVW5pdmVyc2l0w6R0IEhhbGxlLVdpdHRlbmJlcmcsIEhhbGxlIChTYWFsZSksIEdlcm1hbnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHZXJtYW4gQ2VudHJlIGZvciBJbnRlZ3JhdGl2ZSBCaW9kaXZlcnNpdHkgUmVzZWFyY2ggKGlEaXYpIEhhbGxlLUplbmEtTGVpcHppZywgTGVpcHppZywgR2VybWFueSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1vcml0eiwgUm9iaW4gRi5BLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0IGbDvHIgQmlvbG9naWUsIE1vbGVrdWxhcmUgw5Zrb2xvZ2llLCBNYXJ0aW4tTHV0aGVyLVVuaXZlcnNpdMOkdCBIYWxsZS1XaXR0ZW5iZXJnLCBIYWxsZSAoU2FhbGUpLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR2VybWFuIENlbnRyZSBmb3IgSW50ZWdyYXRpdmUgQmlvZGl2ZXJzaXR5IFJlc2VhcmNoIChpRGl2KSBIYWxsZS1KZW5hLUxlaXB6aWcsIExlaXB6aWcsIEdlcm1hbnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIFpvb2xvZ3kgYW5kIEVudG9tb2xvZ3ksIFVuaXZlcnNpdHkgb2YgUHJldG9yaWEsUHJldG9yaWEsIFNvdXRoIEFmcmljYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJ1dHRzdGVkdCwgQW5qYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0IGbDvHIgQmlvbG9naWUsIE1vbGVrdWxhcmUgw5Zrb2xvZ2llLCBNYXJ0aW4tTHV0aGVyLVVuaXZlcnNpdMOkdCBIYWxsZS1XaXR0ZW5iZXJnLCBIYWxsZSAoU2FhbGUpLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJDb21wYXJhdGl2ZSBhbmFseXNlcyBvZiB0aGUgbWFqb3Igcm95YWwgamVsbHkgcHJvdGVpbiBnZW5lIGNsdXN0ZXIgaW4gdGhyZWUgQXBpcyBzcGVjaWVzIHdpdGggbG9uZyBhbXBsaWNvbiBzZXF1ZW5jaW5nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjc5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyODciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwNjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJld2FyZSBvZiBtaXMtYXNzZW1ibGVkIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NDMyMC0xOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gYW5kIGNvcnJlY3Rpb24gb2YgZmFsc2Ugc2VnbWVudGFsIGR1cGxpY2F0aW9ucyBjYXVzZWQgYnkgZ2Vub21lIG1pcy1hc3NlbWJseTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz1SMjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3NlbWJseSBlcnJvcnMgY2F1c2UgZmFsc2UgdGFuZGVtIGR1cGxpY2F0ZSByZWdpb25zIGluIHRoZSBjaGlja2VuIChHYWxsdXMgZ2FsbHVzKSBnZW5vbWUgc2VxdWVuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNocm9tb3NvbWE7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTIzOyAgY2l0YXRpb25fcGFnZXM9MTY1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiB0aGUgWWVsbG93L01ham9yIFJveWFsIEplbGx5IFByb3RlaW4gZmFtaWx5IGFuZCB0aGUgZW1lcmdlbmNlIG9mIHNvY2lhbCBiZWhhdmlvciBpbiBob25leSBiZWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3BhZ2VzPTEzODUtOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc2lnaHRzIGludG8gc29jaWFsIGluc2VjdHMgZnJvbSB0aGUgZ2Vub21lIG9mIHRoZSBob25leWJlZSBBcGlzIG1lbGxpZmVyYTsgY2l0YXRpb25fYXV0aG9yPUhvbmV5YmVlIEdlbm9tZSBTZXF1ZW5jaW5nIENvbnNvcnRpdW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDM7ICBjaXRhdGlvbl9wYWdlcz05MzEtNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZmFtaWx5IG9mIG1ham9yIHJveWFsIGplbGx5IHByb3RlaW5zIG9mIHRoZSBob25leWJlZSBBcGlzIG1lbGxpZmVyYSBMOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsLiBNb2wuIExpZmUgU2NpLiBDOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9MTAyMC0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE1SSlAvWUVMTE9XIHByb3RlaW4gZmFtaWx5IG9mIEFwaXMgbWVsbGlmZXJhOiBJZGVudGlmaWNhdGlvbiBvZiBuZXcgbWVtYmVycyBpbiB0aGUgRVNUIGxpYnJhcnk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEluc2VjdCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9NTEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpZ2luIGFuZCBmdW5jdGlvbiBvZiB0aGUgbWFqb3Igcm95YWwgamVsbHkgcHJvdGVpbnMgb2YgdGhlIGhvbmV5YmVlIChBcGlzIG1lbGxpZmVyYSkgYXMgbWVtYmVycyBvZiB0aGUgeWVsbG93IGdlbmUgZmFtaWx5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9sLiBSZXY7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9ODk7ICBjaXRhdGlvbl9wYWdlcz0yNTUtNjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCBwYXJ0aWFsIGNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIG1ham9yIHJveWFsIGplbGx5IHByb3RlaW4gb2YgdGhlIGhvbmV5IGJlZSAoQXBpcyBtZWxsaWZlcmEgTC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBcGljLiBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0yMi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIGlkZW50aWZpY2F0aW9uIG9mIG5vdmVsIHByb3RlaW5zIGFuZCBOLWdseWNvc3lsYXRpb24gc2l0ZXMgaW4gcm95YWwgamVsbHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1NY0dyYXctSGlsbCBCb29rIENvbXBhbnksIE5ldyBZb3JrOyAgY2l0YXRpb25fdGl0bGU9QW5hdG9teSBhbmQgcGh5c2lvbG9neSBvZiB0aGUgaG9uZXkgYmVlOyAgY2l0YXRpb25feWVhcj0xOTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb3lhbGFjdGluIGlzIG5vdCBhIHJveWFsIG1ha2luZyBvZiBhIHF1ZWVuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NTM3OyAgY2l0YXRpb25fcGFnZXM9RTEwLUUxMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9yZSB0aGFuIHJveWFsIGZvb2Qg4oCTIG1ham9yIHJveWFsIGplbGx5IHByb3RlaW4gZ2VuZXMgaW4gc2V4dWFscyBhbmQgd29ya2VycyBvZiB0aGUgaG9uZXliZWUgQXBpcyBtZWxsaWZlcmE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBab29sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NzIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIG1ham9yIHJveWFsIGplbGx5IHByb3RlaW4tbGlrZSBETkEgc2VxdWVuY2VzIGluIEFwaXMgZG9yc2F0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQXBpYy4gUmVzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9NzUtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9yZ2FuaXphdGlvbiBhbmQgcG90ZW50aWFsIGZ1bmN0aW9uIG9mIHRoZSBtcmpwMyBsb2N1cyBpbiBmb3VyIGhvbmV5YmVlIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEFncmljLiBGb29kIENoZW07ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz04MDc1LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbG9uaW5nLCBleHByZXNzaW9uIGFuZCBnZW5vbWljIG9yZ2FuaXphdGlvbiBvZiBnZW5lcyBlbmNvZGluZyBtYWpvciByb3lhbCBqZWxseSBwcm90ZWluIDEgYW5kIDIgb2YgdGhlIGhvbmV5IGJlZSAoQXBpcyBjZXJhbmEpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9jaGVtLiBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz00OS01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNsb25pbmcgYW5kIGFuYWx5c2lzIG9mIGZvdXIgY0ROQXMgZnJvbSB0aGUgaGVhZHMgb2YgQXBpcyBjZXJhbmEgY2VyYW5hIG51cnNlIGhvbmV5YmVlcyBjb2RpbmcgZm9yIG1ham9yIHJveWFsIGplbGx5IHByb3RlaW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BcGlkb2xvZ2llOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9Mzg5LTQwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVjb25zdHJ1Y3RpbmcgY29tcGxleCByZWdpb25zIG9mIGdlbm9tZXMgdXNpbmcgbG9uZy1yZWFkIHNlcXVlbmNpbmcgdGVjaG5vbG9neTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTY4OC05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzb2x2aW5nIHRoZSBjb21wbGV4aXR5IG9mIHRoZSBodW1hbiBnZW5vbWUgdXNpbmcgc2luZ2xlLW1vbGVjdWxlIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01MTc7ICBjaXRhdGlvbl9wYWdlcz02MDgtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IiBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25faXNzdWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUwMTU3NjAwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJpbWVyM+KAlG5ldyBjYXBhYmlsaXRpZXMgYW5kIGludGVyZmFjZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25faXNzdWU9MTU7ICBjaXRhdGlvbl9wYWdlcz1lMTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3Nlc3NpbmcgdGhlIHBlcmZvcm1hbmNlIG9mIHRoZSBPeGZvcmQgTmFub3BvcmUgVGVjaG5vbG9naWVzIE1pbklPTjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvbW9sLiBEZXRlY3QuIFF1YW50aWY7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGFzc2VtYmx5IHVzaW5nIG5hbm9wb3JlLWd1aWRlZCBsb25nIGFuZCBlcnJvci1mcmVlIEROQSByZWFkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MzI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9cG9SZTogYW4gUiBwYWNrYWdlIGZvciB0aGUgdmlzdWFsaXphdGlvbiBhbmQgYW5hbHlzaXMgb2YgbmFub3BvcmUgc2VxdWVuY2luZyBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTExNC0xMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1FR0E1OiBNb2xlY3VsYXIgZXZvbHV0aW9uYXJ5IGdlbmV0aWNzIGFuYWx5c2lzIHVzaW5nIG1heGltdW0gbGlrZWxpaG9vZCwgZXZvbHV0aW9uYXJ5IGRpc3RhbmNlLCBhbmQgbWF4aW11bSBwYXJzaW1vbnkgbWV0aG9kczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MjczMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OR1MtZXZhbDogTkdTIEVycm9yIGFuYWx5c2lzIGFuZCBub3ZlbCBzZXF1ZW5jZSBWQXJpYW50IGRldGVjdGlvbiB0b29MOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPVczMDEtVzMwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmluZGluZyB0aGUgbWlzc2luZyBob25leSBiZWUgZ2VuZXM6IGxlc3NvbnMgbGVhcm5lZCBmcm9tIGEgZ2Vub21lIHVwZ3JhZGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEtMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgd29ybGR3aWRlIHN1cnZleSBvZiBnZW5vbWUgc2VxdWVuY2UgdmFyaWF0aW9uIHByb3ZpZGVzIGluc2lnaHQgaW50byB0aGUgZXZvbHV0aW9uYXJ5IGhpc3Rvcnkgb2YgdGhlIGhvbmV5YmVlIEFwaXMgbWVsbGlmZXJhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MTA4MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIHNpZ25hdHVyZXMgb2YgZXZvbHV0aW9uYXJ5IHRyYW5zaXRpb25zIGZyb20gc29saXRhcnkgdG8gZ3JvdXAgbGl2aW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlICg4MC0uKTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zNDg7ICBjaXRhdGlvbl9wYWdlcz0xMTM5LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lcyBvZiB0d28ga2V5IGJ1bWJsZWJlZSBzcGVjaWVzIHdpdGggcHJpbWl0aXZlIGV1c29jaWFsIG9yZ2FuaXphdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz03NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZGlldGFyeSBwaHl0b2NoZW1pY2FsIGFsdGVycyBjYXN0ZS1hc3NvY2lhdGVkIGdlbmUgZXhwcmVzc2lvbiBpbiBob25leSBiZWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIEFkdjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9ZTE1MDA3OTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdHJhbnNjcmlwdG9taWMgYW5kIGV2b2x1dGlvbmFyeSBzaWduYXR1cmUgb2Ygc29jaWFsIGludGVyYWN0aW9ucyByZWd1bGF0aW5nIGhvbmV5IGJlZSBjYXN0ZSBkZXZlbG9wbWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RWNvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDc5NS04MDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcHVsYXRpb24gZ2Vub21pY3Mgb2YgdGhlIGhvbmV5IGJlZSByZXZlYWxzIHN0cm9uZyBzaWduYXR1cmVzIG9mIHBvc2l0aXZlIHNlbGVjdGlvbiBvbiB3b3JrZXIgdHJhaXRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Qcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTE7ICBjaXRhdGlvbl9pc3N1ZT03OyAgY2l0YXRpb25fcGFnZXM9MjYxNC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHdvcmxkd2lkZSBzdXJ2ZXkgb2YgZ2Vub21lIHNlcXVlbmNlIHZhcmlhdGlvbiBwcm92aWRlcyBpbnNpZ2h0IGludG8gdGhlIGV2b2x1dGlvbmFyeSBoaXN0b3J5IG9mIHRoZSBob25leWJlZSBBcGlzIG1lbGxpZmVyYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlIEdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQ2OyAgY2l0YXRpb25fcGFnZXM9MTA4MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG1vbGVjdWxhciBwaHlsb2dlbnkgb2YgdGhlIGdlbnVzIEFwaXMgc3VnZ2VzdHMgdGhhdCB0aGUgR2lhbnQgSG9uZXkgQmVlIG9mIHRoZSBQaGlsaXBwaW5lcywgQS4gYnJldmlsaWd1bGEgTWFhLCBhbmQgdGhlIFBsYWlucyBIb25leSBCZWUgb2Ygc291dGhlcm4gSW5kaWEsIEEuIGluZGljYSBGYWJyaWNpdXMsIGFyZSB2YWxpZCBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeXN0LiBFbnRvbW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9MjI2LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9U3ByaW5nZXItVmVybGFnLCBIZWlkZWxiZXJnLCBHZXJtYW55OyAgY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIGJ5IGdlbmUgZHVwbGljYXRpb247ICBjaXRhdGlvbl95ZWFyPTE5NzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1TaW5hdWVyIEFzc29jaWF0ZXMsIEluYy4gUHVibGlzaGVycywgU3VuZGVybGFuZCwgTUE7ICBjaXRhdGlvbl90aXRsZT1UaGUgb3JpZ2lucyBvZiBnZW5vbWUgYXJjaGl0ZWN0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZXZvbHV0aW9uIG9mIGdlbmUgZHVwbGljYXRpb25zOiBjbGFzc2lmeWluZyBhbmQgZGlzdGluZ3Vpc2hpbmcgYmV0d2VlbiBtb2RlbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTk3LTEwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVkdWNlZCBleHByZXNzaW9uIG9mIG1ham9yIHJveWFsIGplbGx5IHByb3RlaW4gMSBnZW5lIGluIHRoZSBtdXNocm9vbSBib2RpZXMgb2Ygd29ya2VyIGhvbmV5YmVlcyB3aXRoIHJlZHVjZWQgbGVhcm5pbmcgYWJpbGl0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBpZG9sb2dpZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPTE5NC0yMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVzIHdpdGggc29jaWFsIGVmZmVjdHMgYXJlIGV4cGVjdGVkIHRvIGhhcmJvciBtb3JlIHNlcXVlbmNlIHZhcmlhdGlvbiB3aXRoaW4gYW5kIGJldHdlZW4gc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXZvbHV0aW9uIChOLiBZKTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT02MzsgIGNpdGF0aW9uX3BhZ2VzPTE2ODUtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVzIGludm9sdmVkIGluIGNvbnZlcmdlbnQgZXZvbHV0aW9uIG9mIGV1c29jaWFsaXR5IGluIGJlZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTc0NzItNzQ3NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI0LzMvMjc5LzE3NzQzOTU2L2RzdzA2NC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlIHdlc3Rlcm4gaG9uZXliZWUsIEFwaXMgbWVsbGlmZXJhIGlzIGEgcHJvbWluZW50IG1vZGVsIG9yZ2FuaXNtIGluIHRoZSBmaWVsZCBvZiBzb2Npb2dlbm9taWNzIGFuZCBhIHJlY2VudCB1cGdyYWRlIHN1YnN0YW50aWFsbHkgaW1wcm92ZWQgYW5ub3RhdGkiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI5NzI4NTcmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkueTYkjyQ7oXVAoLPMPOrZ8xio0w=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f279%2f2972857" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f279%2f2972857" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97549" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/3/1/m_dnares_24_3cover.png?Expires=2147483647&amp;Signature=eoXc11KAMpHEsaqW0C6T2VF4hIzrR2AQ2xJtZ-I1SzuN8Ozcm3pfcXhSESBXVMgST8o0VjwPqT85KWv95Ep~BtVLeJis8VsXRFBVpMHhb~g9dvz54ejvU8kSReDsueZzPSdTwZuEswyhmEGVK~9fmtkzww9NnyhhDFzj14yR9QQPP3D5RCa40I2kb6S3cosdNYyeAUAcZmppIJGZquGgABfHKwfV6R3Ffo5Kcif39o76ejsFOOlH0X60OYvM4Yme1Vyjd2eLo4R5~a0mH9tf6MZMrQpFP4iVZGwKwAheMFv-ISdx3BcAb4Re6~N24GwZYpxXAywMLeRm98GmIDGpbg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/3">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="88473457">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473457">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473459">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473459">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473464">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473464">2. Material and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473480">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473480">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473497">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473497">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473505">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473505">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="88473515">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#88473515">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/3/271/2965793">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/3/289/2960056">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Comparative analyses of the <em>major royal jelly protein</em> gene cluster in three <em>Apis</em> species with long amplicon sequencing
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sophie Helbing</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Institut für Biologie, Molekulare Ökologie, Martin-Luther-Universität Halle-Wittenberg, Halle (Saale), Germany</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw064-cor1" data-open="dsw064-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: 0049 345 5526394. Fax: 0049 345 5527500. Email: <a href="mailto:sophie.helbing@zoologie.uni-halle.de" target="_blank">sophie.helbing@zoologie.uni-halle.de</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sophie+Helbing">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Helbing S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Helbing S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sophie Helbing</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">H. Michael G. Lattorff</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Institut für Biologie, Molekulare Ökologie, Martin-Luther-Universität Halle-Wittenberg, Halle (Saale), Germany</div><div class="aff"><div class="label -label">2</div>German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig, Leipzig, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=H.+Michael+G.+Lattorff">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lattorff H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lattorff H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">H. Michael G. Lattorff</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Robin F.A. Moritz</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Institut für Biologie, Molekulare Ökologie, Martin-Luther-Universität Halle-Wittenberg, Halle (Saale), Germany</div><div class="aff"><div class="label -label">2</div>German Centre for Integrative Biodiversity Research (iDiv) Halle-Jena-Leipzig, Leipzig, Germany</div><div class="aff"><div class="label -label">3</div>Department of Zoology and Entomology, University of Pretoria,Pretoria, South Africa</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Robin+F.A.+Moritz">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Moritz R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Moritz R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Robin F.A. Moritz</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Anja Buttstedt</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Institut für Biologie, Molekulare Ökologie, Martin-Luther-Universität Halle-Wittenberg, Halle (Saale), Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Anja+Buttstedt">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Buttstedt A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Buttstedt A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Anja Buttstedt</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw064-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Yuji Kohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 3, 1 June 2017, Pages 279–287, <a href='https://doi.org/10.1093/dnares/dsw064'>https://doi.org/10.1093/dnares/dsw064</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">07 February 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">06 April 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">20 December 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2972857" href="/dnaresearch/article-pdf/24/3/279/17743956/dsw064.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Sophie Helbing, H. Michael G. Lattorff, Robin F.A. Moritz, Anja Buttstedt;  Comparative analyses of the <em>major royal jelly protein</em> gene cluster in three <em>Apis</em> species with long amplicon sequencing, <em>DNA Research</em>, Volume 24, Issue 3, 1 June 2017, Pages 279–287, <a href='https://doi.org/10.1093/dnares/dsw064'>https://doi.org/10.1093/dnares/dsw064</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2972857?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2972857?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2972857?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2972857?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2972857?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2972857?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2972857?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Comparative%20analyses%20of%20the%20%26lt%3Bem%26gt%3Bmajor%20royal%20jelly%20protein%26lt%3B%2Fem%26gt%3B%20gene%20cluster%20in%20three%20%26lt%3Bem%26gt%3BApis%26lt%3B%2Fem%26gt%3B%20species%20with%20long%20amplicon%20sequencing&amp;publicationDate=2017-02-07&amp;volumeNum=24&amp;issueNum=3&amp;author=Helbing%2C%20Sophie%3B%20Lattorff%2C%20H.%20Michael%20G.&amp;startPage=279&amp;endPage=287&amp;contentId=10.1093%2Fdnares%2Fdsw064&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f279%2f2972857" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=88473457 id="88473457" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The western honeybee, <em>Apis mellifera</em> is a prominent model organism in the field of sociogenomics and a recent upgrade substantially improved annotations of the reference genome. Nevertheless, genome assemblies based on short-sequencing reads suffer from problems in regions comprising e.g. multi-copy genes. We used single-molecule nanopore-based sequencing with extensive read-lengths to reconstruct the organization of the <em>major royal jelly protein</em> (<em>mrjp</em>) region in three species of the genus <em>Apis</em>. Long-amplicon sequencing provides evidence for lineage-specific evolutionary fates of <em>Apis mrjp</em>s. Whereas the most basal species, <em>A. florea</em>, seems to encode ten <em>mrjp</em>s, different patterns of gene loss and retention were observed for <em>A. mellifera</em> and <em>A. dorsata</em>. Furthermore, we show that a previously reported pseudogene in <em>A. mellifera</em>, <em>mrjp2-like</em>, is an assembly artefact arising from short read sequencing.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="MinION™">MinION™</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene duplication&quot;">gene duplication</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Apis mellifera&quot;"><em>Apis mellifera</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Apis florea&quot;"><em>Apis florea</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Apis dorsata&quot;"><em>Apis dorsata</em></a></div>                            <h2 scrollto-destination=88473459 id="88473459" class="section-title" >1. Introduction</h2>
<p>The emergence of second-generation sequencing technologies has led to an accumulation of genome sequence data providing broad access to reference genomes and thus, depicting important resources for addressing evolutionary questions. However, the quality of reference genome assemblies often suffers from short-sequencing reads and even sophisticated assembly algorithms can struggle with repetitive and duplicated regions, either not detecting specific stretches or artificially duplicating polymorphic regions.<a reveal-id="dsw064-B1 dsw064-B2 dsw064-B3" data-open="dsw064-B1 dsw064-B2 dsw064-B3" class="link link-ref link-reveal xref-bibr"><sup>1–3</sup></a></p><p>In the western honeybee <em>Apis mellifera</em>, the <em>major royal jelly protein</em> (<em>mrjp</em>) gene cluster provides an excellent system to test for assembly problems in regions comprising multi-copy genes. The gene cluster is located on chromosome 11 between two <em>yellow</em> genes (<em>y-e3</em> and <em>y-h</em>)<a reveal-id="dsw064-B4" data-open="dsw064-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsw064-B5" data-open="dsw064-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> consisting of nine protein coding genes <em>mrjp1</em> to <em>9</em> and one pseudogene – <em>mrjp-</em>Ψ, recently renamed into <em>mrjp</em>2<em>-like</em><em>.</em><a reveal-id="dsw064-B4 dsw064-B5 dsw064-B6 dsw064-B7" data-open="dsw064-B4 dsw064-B5 dsw064-B6 dsw064-B7" class="link link-ref link-reveal xref-bibr"><sup>4–7</sup></a> All major royal jelly proteins (MRJP) show a remarkable pairwise identity ranging from 47% to 74%<a reveal-id="dsw064-B8" data-open="dsw064-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> and have mostly been identified in royal jelly (RJ),<a reveal-id="dsw064-B9" data-open="dsw064-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsw064-B10" data-open="dsw064-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> a hypopharyngeal gland secretion produced by nurse bees feeding the brood.<a reveal-id="dsw064-B11" data-open="dsw064-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> In <em>A. mellifera</em>, the RJ-feeding regime determines the prospective caste (queen or worker) of developing female larvae and although it has been under discussion that a specific MRJP in RJ might interfere with queen determination there is according to newer investigations no indication that MRJPs transcend the long proposed nutritional value in RJ.<a reveal-id="dsw064-B6" data-open="dsw064-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsw064-B12" data-open="dsw064-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a></p><p>Whereas the nine <em>mrjps</em> found in the genome have been confirmed on mRNA and protein level in <em>A. mellifera</em>,<a reveal-id="dsw064-B10" data-open="dsw064-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsw064-B13" data-open="dsw064-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> similar information about the <em>mrjps</em> in other honeybee species is missing and only available for some <em>mrjp</em> homologues.<a reveal-id="dsw064-B14 dsw064-B15 dsw064-B16 dsw064-B17" data-open="dsw064-B14 dsw064-B15 dsw064-B16 dsw064-B17" class="link link-ref link-reveal xref-bibr"><sup>14–17</sup></a> Based on draft genome sequences, first insights in the gene cluster organization of the most basal species <em>A. florea</em> suggested a highly conserved gene order except for the position of the pseudogene (<em>mrjp-Ψ/2-like</em>).<a reveal-id="dsw064-B8" data-open="dsw064-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> However, because <em>mrjps</em> have all evolved from the same <em>yellow</em> gene, they share high-sequence similarity and structural resemblance<a reveal-id="dsw064-B4" data-open="dsw064-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a>: a challenge for an accurate assembly of this complex region based on short sequence reads. And indeed, in <em>A. mellifera</em> certain indications for local mis-assemblies are given: nearly identical sequence stretches of two adjacently located genes, the pseudogene <em>mrjp2-like</em> (GB55211) and <em>mrjp2</em> (GB55212), provide evidence for artificially duplicated regions present in the current gene cluster assembly (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). In addition, Sanger sequencing of <em>mrjp2</em> revealed the absence of a duplicated 430 bp region (GenBank: KX951418; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). These hints indicate that the currently available <em>mrjp</em> cluster sequence might have suffered from short-sequencing read assembly mistakes due to repetitive and duplicated regions.</p><p>Sequencing technologies that allow for extensive read lengths, as e.g. PacBio<sup>®</sup> or Illumina<sup>®</sup> TruSeq Synthetic Long-Read, provide unprecedented power to improve genome assemblies<a reveal-id="dsw064-B18" data-open="dsw064-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a><sup>,</sup><a reveal-id="dsw064-B19" data-open="dsw064-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> and to solve assembly problems even in highly repetitive and/or duplicated regions. An alternative to these established long-read sequencing technologies is the MinION<sup>TM</sup> sequencer (Oxford Nanopore Technologies Ltd), which uses a very different sequencing strategy. This technique facilitates sequencing through pore-forming proteins (so-called nanopores) that naturally act as ion or small molecule channels, embedded in a polymer membrane. Upon applying voltage across the membrane, the nanopore is passed by ionic current which is in turn subject to characteristic interruptions dependent on the DNA base crossing the pore. In contrast to next-generation sequencing usually based on DNA fragments of 200–400 consecutive bases which are later on assembled, nanopores allow for sequencing of long DNA strands in one piece currently limited to a length of approximately 200,000 bases.<a reveal-id="dsw064-B20" data-open="dsw064-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Thus, assembly caused problems do not even occur in the first place and if amplicon sequencing is performed read lengths are rather limited by PCR restrictions than by the sequencing itself. We here validate and reconstruct the genomic organization of the <em>mrjp</em> region by combining long-range PCR and long-read nanopore sequencing to investigate synteny of <em>mrjp</em> genes between the three honeybee species, <em>Apis mellifera</em>, <em>Apis florea</em> and <em>Apis dorsata</em>.</p>                            <h2 scrollto-destination=88473464 id="88473464" class="section-title" >2. Material and methods</h2>
                            <h3 scrollto-destination=88473465 id="88473465" class="section-title" >2.1. Samples and DNA extraction</h3>
<p>Gene order conservation of the <em>mrjp</em> gene cluster was analyzed in three species of the genus <em>Apis</em>: <em>A. mellifera carnica</em> (Halle/Saale, Germany, 2014), <em>A. dorsata</em> (Manaol, Phillipines, 2013) and <em>A. florea</em> (Dien Bien, Vietnam, 2013), each represented by a single worker. Genomic DNA was extracted from thoraces using phenol/chloroform extraction. DNA purity was determined using the NanoDrop ND-1000 (Peqlab, Erlangen, Germany).</p>                            <h3 scrollto-destination=88473467 id="88473467" class="section-title" >2.2. PCR/purification</h3>
<p>DNA primers (Metabion, Planegg, Germany) were designed using Primer3<a reveal-id="dsw064-B21" data-open="dsw064-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> spanning at least two neighbouring genes. Primer sequences are reported in <a reveal-id="dsw064-T1" data-open="dsw064-T1" class="link link-reveal link-table xref-fig">Table 1</a>. PCR conditions were 94 °C for 1 min; 35 cycles at 98 °C for 10 s; 66 °C for 15 s; 68 °C for 1 min/kb; with a final elongation at 68 °C for 16 min. Each 10 µl reaction contained 8.0 mM dNTPs, 0.3 µM of each gene-specific forward and reverse primer, 0.25 U of PrimeSTAR GXL DNA Polymerase (Takara Bio Europe/S.A.S., Saint-Germain-en-Laye, France) and 1 µL DNA. Fragment sizes were inspected on a 0.7% TAE agarose gel. PCR products were purified using Agencourt AMPure XP (Beckman Coulter GmbH, Krefeld, Germany). Quantity of purified amplicons was determined using ND-1000 (Peqlab, Erlangen, Germany). For some primer combinations, PCR amplification yielded an additional 2.5 – 3.0 kb PCR product (see <a reveal-id="dsw064-T1" data-open="dsw064-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Thus, we set a minimum read length of 6.5 kb to exclude reads possibly produced from these additional PCR products (see below). <div content-id="dsw064-T1" class="table-modal"><div class="table-wrap"><div id="dsw064-T1" data-id="dsw064-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Primers used for PCR and summary of MinION<sup>TM</sup> read count</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Amplicon </th><th colspan="2">Primer (sequence 5'–3') </th><th>Size (kb) </th><th>Selection criteria </th><th>Read count </th></tr></thead><tbody><tr><td>afAmp1 </td><td>F: </td><td>AAAGTGGTTGTTGCTGGTGG </td><td char=".">10.2<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="12">Length: 0.7
Similarity: 0.6 </td><td char=".">38 </td></tr><tr><td> </td><td>R: </td><td>CCTGATGTGATTTAGCTTTCGC </td><td> </td><td> </td></tr><tr><td>afAmp2 </td><td>F: </td><td>GCTGAAGCGAGACACAAACA </td><td char=".">10.2 </td><td char=".">32 </td></tr><tr><td> </td><td>R: </td><td>TTGACCTTGGCCGCTTTTAG </td><td> </td><td> </td></tr><tr><td>afAmp3 </td><td>F: </td><td>ACTCGCGCATATATGTCACG </td><td char=".">9.8 </td><td char=".">100 </td></tr><tr><td> </td><td>R: </td><td>TGCAGACTGTTTTGTTTAACTCC </td><td> </td><td> </td></tr><tr><td>afAmp4 </td><td>F: </td><td>TGTTGCAGTTCACATTCACTCT </td><td char=".">9.5 </td><td char=".">140 </td></tr><tr><td> </td><td>R: </td><td>CGTCACGTGTCATAATGGTTACT </td><td> </td><td> </td></tr><tr><td>afAmp5 </td><td>F: </td><td>TCAGATCAATTTGCAAAGGAACA </td><td char=".">8.8 </td><td char=".">89 </td></tr><tr><td> </td><td>R: </td><td>TGAAGGCCATCGAAACTGAT </td><td> </td><td> </td></tr><tr><td>afAmp6 </td><td>F: </td><td>TTGCCCAACCGTGACAATTC </td><td char=".">8.0 </td><td char=".">99 </td></tr><tr><td> </td><td>R: </td><td>CGATTAAGCAGCAGTAAAATCCA </td><td>(exp. 12.3) </td><td> </td></tr><tr><td colspan="6"><hr /> </td></tr><tr><td>amAmp1 </td><td>F: </td><td>AAAGTGGTTGTTGCTGGTGG </td><td char=".">9.1<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="12">Length: 0.7
Similarity: 0.6 </td><td char=".">89 </td></tr><tr><td> </td><td>R: </td><td>TATCGGGATTTTCGCAACGA </td><td> </td><td> </td></tr><tr><td>amAmp2 </td><td>F: </td><td>ATCAAAGAGTGGCGTCCTCT </td><td char=".">10.7<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">79 </td></tr><tr><td> </td><td>R: </td><td>CAACTAGCGATACGGGTCCT </td><td> </td><td> </td></tr><tr><td>amAmp3 </td><td>F: </td><td>TGTGAAGTACACGTCAATACCT </td><td char=".">9.5 </td><td char=".">61 </td></tr><tr><td> </td><td>R: </td><td>TCATGATGCCATGCTTAATTTTTGT </td><td> </td><td> </td></tr><tr><td>amAmp4 </td><td>F: </td><td>TGAAGTACACGTCATTATCGCCT </td><td char=".">14.0 </td><td char=".">32 </td></tr><tr><td> </td><td>R: </td><td>CACGTGACACGATGATACTCGAT </td><td>(exp.22.0) </td><td> </td></tr><tr><td>amAmp5 </td><td>F: </td><td>GCTCCAAAGTTGCACGTCTT </td><td char=".">10.8 </td><td char=".">79 </td></tr><tr><td> </td><td>R: </td><td>AGGAGTTGTGGAGGGCATAC </td><td> </td><td> </td></tr><tr><td>amAmp6 </td><td>F: </td><td>CTCTTCTGGCACCGTATCCT </td><td char=".">12.1 </td><td char="."><strong>16</strong> </td></tr><tr><td> </td><td>R: </td><td>TTGATTTACAGGCGCATTCAAA </td><td> </td><td> </td></tr><tr><td colspan="6"><hr /> </td></tr><tr><td>adAmp1 </td><td>F: </td><td>AAAGTGGTTGTTGCTGGTGG </td><td char=".">10.7<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="4">Length: 0.7
Similarity: 0.6 </td><td char=".">60 </td></tr><tr><td> </td><td>R: </td><td colspan="2">GAAGCAAGTGAAGTGTGATATGATT </td><td> </td></tr><tr><td>adAmp2 </td><td>F: </td><td>TATTTGGCCGATCGTTGGTC </td><td char=".">10.9<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">25 </td></tr><tr><td> </td><td>R: </td><td>TTCTGCCTGCATTTAATGTTTCA </td><td> </td><td> </td></tr><tr><td>adAmp3 </td><td>F: </td><td>ACACGCGCATTTATGTCACG </td><td char=".">11.6 </td><td rowspan="2">Length: 0.5
Similarity: 0.5 </td><td char=".">37 </td></tr><tr><td> </td><td>R: </td><td>GGAATACTTACTCGTGGCTGAT </td><td> </td><td> </td></tr><tr><td>adAmp4 </td><td>F: </td><td>ACCGACCGTCAATCAAAATTCA </td><td char=".">10.5 </td><td rowspan="4">Length: 0.7
Similarity: 0.6 </td><td char=".">24 </td></tr><tr><td> </td><td>R: </td><td>AAGTGAAGAGATGTGACAGAGAT </td><td> </td><td> </td></tr><tr><td>adAmp5 </td><td>F: </td><td>ACACGTCATTACCGCCTATATAT </td><td char=".">10.3 </td><td char=".">76 </td></tr><tr><td> </td><td>R: </td><td>TCAAGTGTCATGATACGATGCT </td><td> </td><td> </td></tr><tr><td>adAmp6 </td><td>F: </td><td>AGGTGTGACAAAGCTGACAA </td><td char=".">12.0 </td><td rowspan="4">Length: 0.5
Similarity: 0.5 </td><td char=".">40 </td></tr><tr><td> </td><td>R: </td><td>GCCCGCTTAATTATCATACAGTT </td><td>(exp.14.9) </td><td> </td></tr><tr><td>adAmp7 </td><td>F: </td><td>TTGCCCAACCGTGACAATTC </td><td char=".">12.0 </td><td char=".">42 </td></tr><tr><td> </td><td>R: </td><td>ATTCGGAAATGCAAGACGGC </td><td> </td><td> </td></tr></tbody></table></div><div content-id="dsw064-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Amplicon </th><th colspan="2">Primer (sequence 5'–3') </th><th>Size (kb) </th><th>Selection criteria </th><th>Read count </th></tr></thead><tbody><tr><td>afAmp1 </td><td>F: </td><td>AAAGTGGTTGTTGCTGGTGG </td><td char=".">10.2<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="12">Length: 0.7
Similarity: 0.6 </td><td char=".">38 </td></tr><tr><td> </td><td>R: </td><td>CCTGATGTGATTTAGCTTTCGC </td><td> </td><td> </td></tr><tr><td>afAmp2 </td><td>F: </td><td>GCTGAAGCGAGACACAAACA </td><td char=".">10.2 </td><td char=".">32 </td></tr><tr><td> </td><td>R: </td><td>TTGACCTTGGCCGCTTTTAG </td><td> </td><td> </td></tr><tr><td>afAmp3 </td><td>F: </td><td>ACTCGCGCATATATGTCACG </td><td char=".">9.8 </td><td char=".">100 </td></tr><tr><td> </td><td>R: </td><td>TGCAGACTGTTTTGTTTAACTCC </td><td> </td><td> </td></tr><tr><td>afAmp4 </td><td>F: </td><td>TGTTGCAGTTCACATTCACTCT </td><td char=".">9.5 </td><td char=".">140 </td></tr><tr><td> </td><td>R: </td><td>CGTCACGTGTCATAATGGTTACT </td><td> </td><td> </td></tr><tr><td>afAmp5 </td><td>F: </td><td>TCAGATCAATTTGCAAAGGAACA </td><td char=".">8.8 </td><td char=".">89 </td></tr><tr><td> </td><td>R: </td><td>TGAAGGCCATCGAAACTGAT </td><td> </td><td> </td></tr><tr><td>afAmp6 </td><td>F: </td><td>TTGCCCAACCGTGACAATTC </td><td char=".">8.0 </td><td char=".">99 </td></tr><tr><td> </td><td>R: </td><td>CGATTAAGCAGCAGTAAAATCCA </td><td>(exp. 12.3) </td><td> </td></tr><tr><td colspan="6"><hr /> </td></tr><tr><td>amAmp1 </td><td>F: </td><td>AAAGTGGTTGTTGCTGGTGG </td><td char=".">9.1<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="12">Length: 0.7
Similarity: 0.6 </td><td char=".">89 </td></tr><tr><td> </td><td>R: </td><td>TATCGGGATTTTCGCAACGA </td><td> </td><td> </td></tr><tr><td>amAmp2 </td><td>F: </td><td>ATCAAAGAGTGGCGTCCTCT </td><td char=".">10.7<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">79 </td></tr><tr><td> </td><td>R: </td><td>CAACTAGCGATACGGGTCCT </td><td> </td><td> </td></tr><tr><td>amAmp3 </td><td>F: </td><td>TGTGAAGTACACGTCAATACCT </td><td char=".">9.5 </td><td char=".">61 </td></tr><tr><td> </td><td>R: </td><td>TCATGATGCCATGCTTAATTTTTGT </td><td> </td><td> </td></tr><tr><td>amAmp4 </td><td>F: </td><td>TGAAGTACACGTCATTATCGCCT </td><td char=".">14.0 </td><td char=".">32 </td></tr><tr><td> </td><td>R: </td><td>CACGTGACACGATGATACTCGAT </td><td>(exp.22.0) </td><td> </td></tr><tr><td>amAmp5 </td><td>F: </td><td>GCTCCAAAGTTGCACGTCTT </td><td char=".">10.8 </td><td char=".">79 </td></tr><tr><td> </td><td>R: </td><td>AGGAGTTGTGGAGGGCATAC </td><td> </td><td> </td></tr><tr><td>amAmp6 </td><td>F: </td><td>CTCTTCTGGCACCGTATCCT </td><td char=".">12.1 </td><td char="."><strong>16</strong> </td></tr><tr><td> </td><td>R: </td><td>TTGATTTACAGGCGCATTCAAA </td><td> </td><td> </td></tr><tr><td colspan="6"><hr /> </td></tr><tr><td>adAmp1 </td><td>F: </td><td>AAAGTGGTTGTTGCTGGTGG </td><td char=".">10.7<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="4">Length: 0.7
Similarity: 0.6 </td><td char=".">60 </td></tr><tr><td> </td><td>R: </td><td colspan="2">GAAGCAAGTGAAGTGTGATATGATT </td><td> </td></tr><tr><td>adAmp2 </td><td>F: </td><td>TATTTGGCCGATCGTTGGTC </td><td char=".">10.9<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">25 </td></tr><tr><td> </td><td>R: </td><td>TTCTGCCTGCATTTAATGTTTCA </td><td> </td><td> </td></tr><tr><td>adAmp3 </td><td>F: </td><td>ACACGCGCATTTATGTCACG </td><td char=".">11.6 </td><td rowspan="2">Length: 0.5
Similarity: 0.5 </td><td char=".">37 </td></tr><tr><td> </td><td>R: </td><td>GGAATACTTACTCGTGGCTGAT </td><td> </td><td> </td></tr><tr><td>adAmp4 </td><td>F: </td><td>ACCGACCGTCAATCAAAATTCA </td><td char=".">10.5 </td><td rowspan="4">Length: 0.7
Similarity: 0.6 </td><td char=".">24 </td></tr><tr><td> </td><td>R: </td><td>AAGTGAAGAGATGTGACAGAGAT </td><td> </td><td> </td></tr><tr><td>adAmp5 </td><td>F: </td><td>ACACGTCATTACCGCCTATATAT </td><td char=".">10.3 </td><td char=".">76 </td></tr><tr><td> </td><td>R: </td><td>TCAAGTGTCATGATACGATGCT </td><td> </td><td> </td></tr><tr><td>adAmp6 </td><td>F: </td><td>AGGTGTGACAAAGCTGACAA </td><td char=".">12.0 </td><td rowspan="4">Length: 0.5
Similarity: 0.5 </td><td char=".">40 </td></tr><tr><td> </td><td>R: </td><td>GCCCGCTTAATTATCATACAGTT </td><td>(exp.14.9) </td><td> </td></tr><tr><td>adAmp7 </td><td>F: </td><td>TTGCCCAACCGTGACAATTC </td><td char=".">12.0 </td><td char=".">42 </td></tr><tr><td> </td><td>R: </td><td>ATTCGGAAATGCAAGACGGC </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>For the location of the respective amplicons within the cluster see <a reveal-id="dsw064-F3" data-open="dsw064-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>. Expected sizes were calculated based on the reference genomic sequences. The last column refers to the number of reads that mapped using the selection criteria given in the previous column. Due to assembly errors or sequencing gaps in reference sequences, selection criteria for adAmp3, adAmp6 and adAmp7 were less stringent compared to the other amplicons. Consensus calling was adjusted to the minimum number of reads (amAmp6: 16 reads).</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>a</sup>Amplification yielded an additional smaller PCR product (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>).</p></div></div></div></div></div><div class="table-wrap"><div id="dsw064-T1" data-id="dsw064-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Primers used for PCR and summary of MinION<sup>TM</sup> read count</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Amplicon </th><th colspan="2">Primer (sequence 5'–3') </th><th>Size (kb) </th><th>Selection criteria </th><th>Read count </th></tr></thead><tbody><tr><td>afAmp1 </td><td>F: </td><td>AAAGTGGTTGTTGCTGGTGG </td><td char=".">10.2<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="12">Length: 0.7
Similarity: 0.6 </td><td char=".">38 </td></tr><tr><td> </td><td>R: </td><td>CCTGATGTGATTTAGCTTTCGC </td><td> </td><td> </td></tr><tr><td>afAmp2 </td><td>F: </td><td>GCTGAAGCGAGACACAAACA </td><td char=".">10.2 </td><td char=".">32 </td></tr><tr><td> </td><td>R: </td><td>TTGACCTTGGCCGCTTTTAG </td><td> </td><td> </td></tr><tr><td>afAmp3 </td><td>F: </td><td>ACTCGCGCATATATGTCACG </td><td char=".">9.8 </td><td char=".">100 </td></tr><tr><td> </td><td>R: </td><td>TGCAGACTGTTTTGTTTAACTCC </td><td> </td><td> </td></tr><tr><td>afAmp4 </td><td>F: </td><td>TGTTGCAGTTCACATTCACTCT </td><td char=".">9.5 </td><td char=".">140 </td></tr><tr><td> </td><td>R: </td><td>CGTCACGTGTCATAATGGTTACT </td><td> </td><td> </td></tr><tr><td>afAmp5 </td><td>F: </td><td>TCAGATCAATTTGCAAAGGAACA </td><td char=".">8.8 </td><td char=".">89 </td></tr><tr><td> </td><td>R: </td><td>TGAAGGCCATCGAAACTGAT </td><td> </td><td> </td></tr><tr><td>afAmp6 </td><td>F: </td><td>TTGCCCAACCGTGACAATTC </td><td char=".">8.0 </td><td char=".">99 </td></tr><tr><td> </td><td>R: </td><td>CGATTAAGCAGCAGTAAAATCCA </td><td>(exp. 12.3) </td><td> </td></tr><tr><td colspan="6"><hr /> </td></tr><tr><td>amAmp1 </td><td>F: </td><td>AAAGTGGTTGTTGCTGGTGG </td><td char=".">9.1<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="12">Length: 0.7
Similarity: 0.6 </td><td char=".">89 </td></tr><tr><td> </td><td>R: </td><td>TATCGGGATTTTCGCAACGA </td><td> </td><td> </td></tr><tr><td>amAmp2 </td><td>F: </td><td>ATCAAAGAGTGGCGTCCTCT </td><td char=".">10.7<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">79 </td></tr><tr><td> </td><td>R: </td><td>CAACTAGCGATACGGGTCCT </td><td> </td><td> </td></tr><tr><td>amAmp3 </td><td>F: </td><td>TGTGAAGTACACGTCAATACCT </td><td char=".">9.5 </td><td char=".">61 </td></tr><tr><td> </td><td>R: </td><td>TCATGATGCCATGCTTAATTTTTGT </td><td> </td><td> </td></tr><tr><td>amAmp4 </td><td>F: </td><td>TGAAGTACACGTCATTATCGCCT </td><td char=".">14.0 </td><td char=".">32 </td></tr><tr><td> </td><td>R: </td><td>CACGTGACACGATGATACTCGAT </td><td>(exp.22.0) </td><td> </td></tr><tr><td>amAmp5 </td><td>F: </td><td>GCTCCAAAGTTGCACGTCTT </td><td char=".">10.8 </td><td char=".">79 </td></tr><tr><td> </td><td>R: </td><td>AGGAGTTGTGGAGGGCATAC </td><td> </td><td> </td></tr><tr><td>amAmp6 </td><td>F: </td><td>CTCTTCTGGCACCGTATCCT </td><td char=".">12.1 </td><td char="."><strong>16</strong> </td></tr><tr><td> </td><td>R: </td><td>TTGATTTACAGGCGCATTCAAA </td><td> </td><td> </td></tr><tr><td colspan="6"><hr /> </td></tr><tr><td>adAmp1 </td><td>F: </td><td>AAAGTGGTTGTTGCTGGTGG </td><td char=".">10.7<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="4">Length: 0.7
Similarity: 0.6 </td><td char=".">60 </td></tr><tr><td> </td><td>R: </td><td colspan="2">GAAGCAAGTGAAGTGTGATATGATT </td><td> </td></tr><tr><td>adAmp2 </td><td>F: </td><td>TATTTGGCCGATCGTTGGTC </td><td char=".">10.9<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">25 </td></tr><tr><td> </td><td>R: </td><td>TTCTGCCTGCATTTAATGTTTCA </td><td> </td><td> </td></tr><tr><td>adAmp3 </td><td>F: </td><td>ACACGCGCATTTATGTCACG </td><td char=".">11.6 </td><td rowspan="2">Length: 0.5
Similarity: 0.5 </td><td char=".">37 </td></tr><tr><td> </td><td>R: </td><td>GGAATACTTACTCGTGGCTGAT </td><td> </td><td> </td></tr><tr><td>adAmp4 </td><td>F: </td><td>ACCGACCGTCAATCAAAATTCA </td><td char=".">10.5 </td><td rowspan="4">Length: 0.7
Similarity: 0.6 </td><td char=".">24 </td></tr><tr><td> </td><td>R: </td><td>AAGTGAAGAGATGTGACAGAGAT </td><td> </td><td> </td></tr><tr><td>adAmp5 </td><td>F: </td><td>ACACGTCATTACCGCCTATATAT </td><td char=".">10.3 </td><td char=".">76 </td></tr><tr><td> </td><td>R: </td><td>TCAAGTGTCATGATACGATGCT </td><td> </td><td> </td></tr><tr><td>adAmp6 </td><td>F: </td><td>AGGTGTGACAAAGCTGACAA </td><td char=".">12.0 </td><td rowspan="4">Length: 0.5
Similarity: 0.5 </td><td char=".">40 </td></tr><tr><td> </td><td>R: </td><td>GCCCGCTTAATTATCATACAGTT </td><td>(exp.14.9) </td><td> </td></tr><tr><td>adAmp7 </td><td>F: </td><td>TTGCCCAACCGTGACAATTC </td><td char=".">12.0 </td><td char=".">42 </td></tr><tr><td> </td><td>R: </td><td>ATTCGGAAATGCAAGACGGC </td><td> </td><td> </td></tr></tbody></table></div><div content-id="dsw064-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Amplicon </th><th colspan="2">Primer (sequence 5'–3') </th><th>Size (kb) </th><th>Selection criteria </th><th>Read count </th></tr></thead><tbody><tr><td>afAmp1 </td><td>F: </td><td>AAAGTGGTTGTTGCTGGTGG </td><td char=".">10.2<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="12">Length: 0.7
Similarity: 0.6 </td><td char=".">38 </td></tr><tr><td> </td><td>R: </td><td>CCTGATGTGATTTAGCTTTCGC </td><td> </td><td> </td></tr><tr><td>afAmp2 </td><td>F: </td><td>GCTGAAGCGAGACACAAACA </td><td char=".">10.2 </td><td char=".">32 </td></tr><tr><td> </td><td>R: </td><td>TTGACCTTGGCCGCTTTTAG </td><td> </td><td> </td></tr><tr><td>afAmp3 </td><td>F: </td><td>ACTCGCGCATATATGTCACG </td><td char=".">9.8 </td><td char=".">100 </td></tr><tr><td> </td><td>R: </td><td>TGCAGACTGTTTTGTTTAACTCC </td><td> </td><td> </td></tr><tr><td>afAmp4 </td><td>F: </td><td>TGTTGCAGTTCACATTCACTCT </td><td char=".">9.5 </td><td char=".">140 </td></tr><tr><td> </td><td>R: </td><td>CGTCACGTGTCATAATGGTTACT </td><td> </td><td> </td></tr><tr><td>afAmp5 </td><td>F: </td><td>TCAGATCAATTTGCAAAGGAACA </td><td char=".">8.8 </td><td char=".">89 </td></tr><tr><td> </td><td>R: </td><td>TGAAGGCCATCGAAACTGAT </td><td> </td><td> </td></tr><tr><td>afAmp6 </td><td>F: </td><td>TTGCCCAACCGTGACAATTC </td><td char=".">8.0 </td><td char=".">99 </td></tr><tr><td> </td><td>R: </td><td>CGATTAAGCAGCAGTAAAATCCA </td><td>(exp. 12.3) </td><td> </td></tr><tr><td colspan="6"><hr /> </td></tr><tr><td>amAmp1 </td><td>F: </td><td>AAAGTGGTTGTTGCTGGTGG </td><td char=".">9.1<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="12">Length: 0.7
Similarity: 0.6 </td><td char=".">89 </td></tr><tr><td> </td><td>R: </td><td>TATCGGGATTTTCGCAACGA </td><td> </td><td> </td></tr><tr><td>amAmp2 </td><td>F: </td><td>ATCAAAGAGTGGCGTCCTCT </td><td char=".">10.7<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">79 </td></tr><tr><td> </td><td>R: </td><td>CAACTAGCGATACGGGTCCT </td><td> </td><td> </td></tr><tr><td>amAmp3 </td><td>F: </td><td>TGTGAAGTACACGTCAATACCT </td><td char=".">9.5 </td><td char=".">61 </td></tr><tr><td> </td><td>R: </td><td>TCATGATGCCATGCTTAATTTTTGT </td><td> </td><td> </td></tr><tr><td>amAmp4 </td><td>F: </td><td>TGAAGTACACGTCATTATCGCCT </td><td char=".">14.0 </td><td char=".">32 </td></tr><tr><td> </td><td>R: </td><td>CACGTGACACGATGATACTCGAT </td><td>(exp.22.0) </td><td> </td></tr><tr><td>amAmp5 </td><td>F: </td><td>GCTCCAAAGTTGCACGTCTT </td><td char=".">10.8 </td><td char=".">79 </td></tr><tr><td> </td><td>R: </td><td>AGGAGTTGTGGAGGGCATAC </td><td> </td><td> </td></tr><tr><td>amAmp6 </td><td>F: </td><td>CTCTTCTGGCACCGTATCCT </td><td char=".">12.1 </td><td char="."><strong>16</strong> </td></tr><tr><td> </td><td>R: </td><td>TTGATTTACAGGCGCATTCAAA </td><td> </td><td> </td></tr><tr><td colspan="6"><hr /> </td></tr><tr><td>adAmp1 </td><td>F: </td><td>AAAGTGGTTGTTGCTGGTGG </td><td char=".">10.7<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="4">Length: 0.7
Similarity: 0.6 </td><td char=".">60 </td></tr><tr><td> </td><td>R: </td><td colspan="2">GAAGCAAGTGAAGTGTGATATGATT </td><td> </td></tr><tr><td>adAmp2 </td><td>F: </td><td>TATTTGGCCGATCGTTGGTC </td><td char=".">10.9<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">25 </td></tr><tr><td> </td><td>R: </td><td>TTCTGCCTGCATTTAATGTTTCA </td><td> </td><td> </td></tr><tr><td>adAmp3 </td><td>F: </td><td>ACACGCGCATTTATGTCACG </td><td char=".">11.6 </td><td rowspan="2">Length: 0.5
Similarity: 0.5 </td><td char=".">37 </td></tr><tr><td> </td><td>R: </td><td>GGAATACTTACTCGTGGCTGAT </td><td> </td><td> </td></tr><tr><td>adAmp4 </td><td>F: </td><td>ACCGACCGTCAATCAAAATTCA </td><td char=".">10.5 </td><td rowspan="4">Length: 0.7
Similarity: 0.6 </td><td char=".">24 </td></tr><tr><td> </td><td>R: </td><td>AAGTGAAGAGATGTGACAGAGAT </td><td> </td><td> </td></tr><tr><td>adAmp5 </td><td>F: </td><td>ACACGTCATTACCGCCTATATAT </td><td char=".">10.3 </td><td char=".">76 </td></tr><tr><td> </td><td>R: </td><td>TCAAGTGTCATGATACGATGCT </td><td> </td><td> </td></tr><tr><td>adAmp6 </td><td>F: </td><td>AGGTGTGACAAAGCTGACAA </td><td char=".">12.0 </td><td rowspan="4">Length: 0.5
Similarity: 0.5 </td><td char=".">40 </td></tr><tr><td> </td><td>R: </td><td>GCCCGCTTAATTATCATACAGTT </td><td>(exp.14.9) </td><td> </td></tr><tr><td>adAmp7 </td><td>F: </td><td>TTGCCCAACCGTGACAATTC </td><td char=".">12.0 </td><td char=".">42 </td></tr><tr><td> </td><td>R: </td><td>ATTCGGAAATGCAAGACGGC </td><td> </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>For the location of the respective amplicons within the cluster see <a reveal-id="dsw064-F3" data-open="dsw064-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>. Expected sizes were calculated based on the reference genomic sequences. The last column refers to the number of reads that mapped using the selection criteria given in the previous column. Due to assembly errors or sequencing gaps in reference sequences, selection criteria for adAmp3, adAmp6 and adAmp7 were less stringent compared to the other amplicons. Consensus calling was adjusted to the minimum number of reads (amAmp6: 16 reads).</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>a</sup>Amplification yielded an additional smaller PCR product (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>).</p></div></div></div></div></p>                            <h3 scrollto-destination=88473469 id="88473469" class="section-title" >2.3. Sequencing</h3>
<p>Long-amplicon sequencing was performed with MinION (Oxford Nanopore Technologies Ltd)—a nanopore-based sequencing technology capable of producing read length s of up to 200 kb<a reveal-id="dsw064-B20" data-open="dsw064-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsw064-B22" data-open="dsw064-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw064-B23" data-open="dsw064-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> facilitating continuous sequencing across the entire <em>mrjp</em> cluster region.</p><p>Prior to MinION library preparation, amplicons were pooled in equimolar amounts. The amplicon pool was end-repaired using the NEBNext End Repair Module (NEB, Ipswich, MA, USA), followed by Agencourt AMPure XP (Beckman Coulter GmbH, Krefeld, Germany) purification and dA-tailing for 10 min at 37 °C using NEBNextdA-Tailing Module. dA-tailed DNA was purified as described above. The dA-tailed DNA was further processed using the SQK-MAP006 sequencing kit (Oxford Nanopore) following manufacturer’s instructions. Finally 150–195 ng of the pre-sequencing mix were loaded into the flow cells (<em>N</em> = 2). The 48 h sequencing protocol was chosen. The sequencing protocol was run for 2 h (MinKNOW<sup>TM</sup> software 0.50.2). Base calling was performed using the Metrichor<sup>TM</sup> Agent (Oxford Nanopore). Metrichor produces the fast5 file format containing fastq files, which were extracted using poRe.<a reveal-id="dsw064-B24" data-open="dsw064-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Oxford Nanopore raw reads have been uploaded to the Sequence Read Archive (Accession number: SRR4426288). Only 2D reads (comprising information from both DNA strands) with a minimum read length of 6.5 kb (see above) were included in subsequent analyses (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). Reads were first mapped against the three reference genomes (<a class="link link-uri" href="http://ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA_000469605.1_Apis_dorsata_1.3" target="_blank">ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA_000469605.1_Apis_dorsata_1.3</a>; <a class="link link-uri" href="http://ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA_000184785.1_Aflo_1.0; Amel4.5" target="_blank">ftp://ftp.ncbi.nlm.nih.gov/genomes/all/GCA_000184785.1_Aflo_1.0 (11 January 2017, date last accessed); Amel4.5</a>) in order to assign reads to the corresponding amplicon (by species and genomic target). Read mapping was performed using CLC Genomics Workbench 8.0 (CLC Bio, Aarhus, Denmark). With regard to the high error rate less stringent mapping parameter settings were used: mismatch cost of 2; insertion/deletion costs of 2 and 1, respectively; insertion/deletion open and extend cost of 6 and 1, respectively; length fraction of 0.7/0.5; a similarity fraction of 0.6/0.5 (see below).</p>                            <h3 scrollto-destination=88473472 id="88473472" class="section-title" >2.4. Data analyses</h3>
<p>A graphical presentation of the data analysis pipeline is provided in <a reveal-id="dsw064-F1" data-open="dsw064-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>. Sequencing yielded 8468 2D reads, with a median length of 2814 bp (range: 165–15,889 bp) and a mean sequence quality of 10 (Phred score) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). For subsequent analyses, only reads ≥ 6.5 kbp were included resulting in 2261 reads that matched these size requirements.</p>                        <a id="88473474" scrollto-destination="88473474"></a>
<div id="dsw064-F1" data-id="dsw064-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/m_dsw064f1.png?Expires=2147483647&Signature=uEILoIdNpTGjNm2b~t7GOQj~XpcWShh~uAs9LeVmADmMr~KgLtkaCDioM0GV12LoYpn1~1Cg2Y3vU0euF~FWPM7~hHHhpANatqJKOdcWMPzPLN0LKZgvBfEddYFDYO2DMt-Y5tX5tCZUMovMZ8O0hZCcxTDI-DrMKpLl9w5OGHPKqFJWrm9RLPXcsVxa3oqMQuPrkv7WKolG15jQ2BOKcqp~KLjYM8lL4SMw8vGm9Tt~5EQvrnSbdwHZPMPsDl-Mo0jBsoFZJe6o2P9GCSQtGS5-699FSocwFG403s2vzI09vX5vZYDc6zMGSeavBp9RF7JOH5MWvWIylA0FiUtuqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Graphical presentation of the data analysis pipeline. 2D raw reads (1) were size-selected (minimum read length of 6.5 kb) (2) and mapped against the three reference genomes, in order to assign the reads by species and genomic target (3). Only those reads that matched our quality filters (similarity fraction: 0.6 [0.5 for adAmp3, 6 and 7], length fraction: 0.7 [0.5 for adAmp3, 6 and 7]) were included in further analyses (4). Per amplicon sixteen reads (minimum number of reads that mapped to an amplicon—amAmp6) were selected and aligned to each other independent of a reference sequence to build the nanopore-derived consensus sequence (5). Finally, the consensus sequence and the reference sequence were aligned (6). In order to correct the genomic reference sequences of the mrjp gene cluster of A. mellifera, A. florea and A. dorsata, assembly gaps (N) and local mis-assemblies were identified based on this consensus/reference sequence alignment. Assembly gaps (N) in the reference sequence were replaced with the consensus sequence and mis-assemblies were either discarded (when only present in the reference but not in the consensus sequence) or included (when only present in the consensus but not in the reference sequence)." path-from-xml="dsw064f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473474/dsw064f1.tif" path-from-xml="dsw064f1.tif" target="_blank">View large</a><a section="88473474" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064f1.png?Expires=2147483647&Signature=z2Wr7LgjpDfA5Y8DCF9DTWfzoLYqPQGFT8yL3m~SiJfKqwHlcXVsNxH9~Oe2Qwyvt2mqZ9CqXSP9LVPVvWhrygeq8Cg03nZrMHDu5grihrQ8sb0S5zzvMdHfROafyhQIR7NVNF5p6usErXfKkF8CsEg5EDU-m3EcAqAOEVvgN-UCP8dQLZEfDxmXv1cu372UKQx7MLxYylVN~1l85ETT8e35RMxpwcCF2iBFmjbg3I-37u2EXGptbzGcELVBFtIpSqPZjqKosn~oMSBCe44rJfkVzpu~hxYK2IJ7p8Va53FR6AJRcjYm2T5AqJRecfL08b8k0eBy0w-MoWOaIn9y0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473474&ar=2972857&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw064f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Graphical presentation of the data analysis pipeline. 2D raw reads (1) were size-selected (minimum read length of 6.5 kb) (2) and mapped against the three reference genomes, in order to assign the reads by species and genomic target (3). Only those reads that matched our quality filters (similarity fraction: 0.6 [0.5 for adAmp3, 6 and 7], length fraction: 0.7 [0.5 for adAmp3, 6 and 7]) were included in further analyses (4). Per amplicon sixteen reads (minimum number of reads that mapped to an amplicon—amAmp6) were selected and aligned to each other independent of a reference sequence to build the nanopore-derived consensus sequence (5). Finally, the consensus sequence and the reference sequence were aligned (6). In order to correct the genomic reference sequences of the <em>mrjp</em> gene cluster of <em>A. mellifera</em>, <em>A. florea</em> and <em>A. dorsata</em>, assembly gaps (N) and local mis-assemblies were identified based on this consensus/reference sequence alignment. Assembly gaps (N) in the reference sequence were replaced with the consensus sequence and mis-assemblies were either discarded (when only present in the reference but not in the consensus sequence) or included (when only present in the consensus but not in the reference sequence).</p></div></div><div content-id="dsw064-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/m_dsw064f1.png?Expires=2147483647&Signature=uEILoIdNpTGjNm2b~t7GOQj~XpcWShh~uAs9LeVmADmMr~KgLtkaCDioM0GV12LoYpn1~1Cg2Y3vU0euF~FWPM7~hHHhpANatqJKOdcWMPzPLN0LKZgvBfEddYFDYO2DMt-Y5tX5tCZUMovMZ8O0hZCcxTDI-DrMKpLl9w5OGHPKqFJWrm9RLPXcsVxa3oqMQuPrkv7WKolG15jQ2BOKcqp~KLjYM8lL4SMw8vGm9Tt~5EQvrnSbdwHZPMPsDl-Mo0jBsoFZJe6o2P9GCSQtGS5-699FSocwFG403s2vzI09vX5vZYDc6zMGSeavBp9RF7JOH5MWvWIylA0FiUtuqg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Graphical presentation of the data analysis pipeline. 2D raw reads (1) were size-selected (minimum read length of 6.5 kb) (2) and mapped against the three reference genomes, in order to assign the reads by species and genomic target (3). Only those reads that matched our quality filters (similarity fraction: 0.6 [0.5 for adAmp3, 6 and 7], length fraction: 0.7 [0.5 for adAmp3, 6 and 7]) were included in further analyses (4). Per amplicon sixteen reads (minimum number of reads that mapped to an amplicon—amAmp6) were selected and aligned to each other independent of a reference sequence to build the nanopore-derived consensus sequence (5). Finally, the consensus sequence and the reference sequence were aligned (6). In order to correct the genomic reference sequences of the mrjp gene cluster of A. mellifera, A. florea and A. dorsata, assembly gaps (N) and local mis-assemblies were identified based on this consensus/reference sequence alignment. Assembly gaps (N) in the reference sequence were replaced with the consensus sequence and mis-assemblies were either discarded (when only present in the reference but not in the consensus sequence) or included (when only present in the consensus but not in the reference sequence)." path-from-xml="dsw064f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473474/dsw064f1.tif" path-from-xml="dsw064f1.tif" target="_blank">View large</a><a section="88473474" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064f1.png?Expires=2147483647&Signature=z2Wr7LgjpDfA5Y8DCF9DTWfzoLYqPQGFT8yL3m~SiJfKqwHlcXVsNxH9~Oe2Qwyvt2mqZ9CqXSP9LVPVvWhrygeq8Cg03nZrMHDu5grihrQ8sb0S5zzvMdHfROafyhQIR7NVNF5p6usErXfKkF8CsEg5EDU-m3EcAqAOEVvgN-UCP8dQLZEfDxmXv1cu372UKQx7MLxYylVN~1l85ETT8e35RMxpwcCF2iBFmjbg3I-37u2EXGptbzGcELVBFtIpSqPZjqKosn~oMSBCe44rJfkVzpu~hxYK2IJ7p8Va53FR6AJRcjYm2T5AqJRecfL08b8k0eBy0w-MoWOaIn9y0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473474&ar=2972857&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw064f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Graphical presentation of the data analysis pipeline. 2D raw reads (1) were size-selected (minimum read length of 6.5 kb) (2) and mapped against the three reference genomes, in order to assign the reads by species and genomic target (3). Only those reads that matched our quality filters (similarity fraction: 0.6 [0.5 for adAmp3, 6 and 7], length fraction: 0.7 [0.5 for adAmp3, 6 and 7]) were included in further analyses (4). Per amplicon sixteen reads (minimum number of reads that mapped to an amplicon—amAmp6) were selected and aligned to each other independent of a reference sequence to build the nanopore-derived consensus sequence (5). Finally, the consensus sequence and the reference sequence were aligned (6). In order to correct the genomic reference sequences of the <em>mrjp</em> gene cluster of <em>A. mellifera</em>, <em>A. florea</em> and <em>A. dorsata</em>, assembly gaps (N) and local mis-assemblies were identified based on this consensus/reference sequence alignment. Assembly gaps (N) in the reference sequence were replaced with the consensus sequence and mis-assemblies were either discarded (when only present in the reference but not in the consensus sequence) or included (when only present in the consensus but not in the reference sequence).</p></div></div><p>Only reads where at least 70% of the alignment matched the reference sequence and displayed at least 60% identity were included in the analyses. These selection criteria were chosen as others reported on MinION error rates between 30% and 40%.<a reveal-id="dsw064-B22" data-open="dsw064-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw064-B23" data-open="dsw064-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Applying the stringent quality filters (similarity fraction: 0.6; length fraction: 0.7), only 46% of the 2261 reads mapped to the reference sequences (<em>A. mellifera</em> 356, <em>A. florea</em> 498, <em>A. dorsata</em> 185; <a reveal-id="dsw064-T1" data-open="dsw064-T1" class="link link-reveal link-table xref-fig">Table 1</a>). These selection criteria did only result in 21 reads mapping to the amplicons adAmp3, adAmp6 and adAmp7 as assembly errors or sequencing gaps in the reference sequences caused reduced mappability. Therefore, read selection criteria for these amplicons were reduced (similarity fraction: 0.5; length fraction: 0.5) leading to 98 additional reads mapping to adAmp3, adAmp6 and adAmp7. As 16 reads was the minimum number of reads that mapped to an amplicon (amAmp6; <a reveal-id="dsw064-T1" data-open="dsw064-T1" class="link link-reveal link-table xref-fig">Table 1</a>), for each amplicon, a subset of 16 random reads were selected, aligned to each other (gap open and extend cost 5.0 and 1.0, respectively) and a Nanopore-derived consensus sequence was built. We used for each amplicon the same number of reads in order to account for amplicon coverage-bias. However, the accuracy of an inferred consensus sequence is always dependent on various parameters such as the alignment program, their parameters, thresholds and the depth of the aligned reads and thus using different analysis programs might lead to slightly different consensus sequences.</p><p>The nanopore-derived consensus and the corresponding genomic reference sequences were aligned (MAFFT Version 7) in order to close assembly gaps and identify misassembled regions in the reference sequences (<a reveal-id="dsw064-F1" data-open="dsw064-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Assembly gaps are stretches of undefined nucleotides (N) of a predicted size in reference sequences. In regions comprising assembly gaps, the alignments of the gap-spanning nanopore consensus sequence to gap-flanking regions were used to infer the actual gap size and nucleotide sequence. To identify and correct misassembled regions alignment discrepancies encompassing &gt;250 bp were considered as indication for a local misassembly. Stretches of unaligned bases in the reference genome without counterpart in the consensus sequence were considered as ‘foreign’, whereas unaligned bases present in the consensus sequence but absent from the reference were scored as ‘missing’. The genomic reference sequences were accordingly corrected (gap closure; elimination of foreign sequence stretches) and are available as <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7- S9</a></span>.</p><p>The MRJP amino acid sequences were derived from the corrected genome sequences and used for a subsequent phylogenetic analysis. Amino acid sequences of <em>Bombus terrestris</em> yellow e-3 (GenBank accession: ADW82101.1) and <em>Bombus impatiens</em> yellow-like (GenBank accession: XP_003493839.1) were used as an outgroup. We constructed a maximum-likelihood tree based on multiple sequence alignment using MEGA version 5,<a reveal-id="dsw064-B25" data-open="dsw064-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> to infer the evolutionary relationships among <em>Apis</em> MRJP proteins (model: Jones-Taylor-Thornton + Gamma distribution; bootstrap analysis: 500 replications).</p>                            <h3 scrollto-destination=88473478 id="88473478" class="section-title" >2.5. Sanger sequencing</h3>
<p>In order to validate the presence of <em>mrjp10</em> in <em>A. dorsata</em> and <em>A. florea</em> and for the purpose of re-sequencing <em>mrjp2</em> in <em>A. mellifera</em>, these genes were sequenced using traditional Sanger sequencing. Therefore, amplification was performed in a thermocycler, with denaturation at 94 °C for 1 min; 35 cycles at 98 °C for 10 s; 60 °C for 15 s; 68 °C for 6 min and a final elongation step for 10 min at 68 °C. The PCR reaction setup was as described above. Sequencing of <em>afmrjp10</em> and <em>admrjp10</em> has been performed by LGC genomics (Berlin, Germany). The initial sequencing step has been performed using the amplification primer. In order to cover the full-length sequence, nested (internal) sequencing primers were used. Amplification- and sequencing-primers are given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. Sanger sequences are deposited on GenBank (GenBank accession numbers: KX925848 (<em>A. dorsata mrjp10</em>), KX925849 (<em>A. florea mrjp10</em>) and KX951418 (<em>A. mellifera mrjp2</em>).</p>                            <h2 scrollto-destination=88473480 id="88473480" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=88473481 id="88473481" class="section-title" >3.1. Error rates of MinION<sup>TM</sup> sequencing</h3>
<p>To analyze error patterns of the MinION sequencing, one gene, <em>admrjp10</em>, independently sequenced using Sanger sequencing (GenBank accession number: KX925848) was used to exemplarily infer the error rate and patterns of individual sequencing reads for the aligned region (≈ 3200 bp) using NGS-eval.<a reveal-id="dsw064-B26" data-open="dsw064-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> In total, 21 individual sequencing reads were used to exemplarily infer the error patterns. For an individual read, the most prominent error type is substitution (average substitution per aligned base: 0.098). Average insertions and deletions per aligned base are 0.053 and 0.066, respectively (<a reveal-id="dsw064-F2" data-open="dsw064-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). The highest substitution rates were observed for adenine (A) and the lowest substitution rates for cytosine (C) (<a reveal-id="dsw064-F2" data-open="dsw064-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). Adenine and thymine got frequently substituted by guanine and cytosine. This pattern most likely reflects the base composition of the target sequence, which is A/T rich. In order to describe the relationship between error rates and sequencing depth, several consensus sequences were built averaging sequence information from multiple reads. For this purpose, consensus sequences (2–4 replications each) were generated using 3, 5, 7, 11 or 21 reads. Here, substitutions, ambiguous nucleotides and insertions per aligned base decreased with increasing sequencing depth, while deletions were almost unaffected by sequence coverage (<a reveal-id="dsw064-F2" data-open="dsw064-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). This suggests that the occurrence of deletions is rather a systemic error. Indeed, deletions primarily occur in regions comprising homopolymer stretches (85.7%), while the remaining deletions occurred in other repetitive regions like dinucleotide motifs (<a reveal-id="dsw064-F2" data-open="dsw064-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). Furthermore, the number of deleted bases is directly related to the length of a homopolymer stretch (<a reveal-id="dsw064-F2" data-open="dsw064-F2" class="link link-reveal link-table xref-fig">Fig. 2D</a>).</p>                        <a id="88473483" scrollto-destination="88473483"></a>
<div id="dsw064-F2" data-id="dsw064-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/m_dsw064f2.png?Expires=2147483647&Signature=E7mZnkaNZSGE9caDspExVJl1AQ1TmSjr9V6C5yP52uCxDutj-vxr5dNM~NKHiJ1OCJ7jdpjAJaYQiR0xGB5OsOhv3Fz~531ZCQXxYBc7BVkYi~tlqHL99YGHDF0O8wX1bmZxFqsrsFsOqB2G~0nMR~kmSS4u8FX3laSdJR6xGE6CE5MAMRTlp~kvMe1IQ2cIw106XcOWTVze052ZlqeR0SEDmMl0b3pUmeDkM1bZRpbtiKTY~mlRi9hGKP3TKjDH54XfT7uwUTtW7ANIfOZSMNkldnYFyqOfyQg~xWX7EQWfcLkBOTTPFxIOURr5VA~VE0zpG8iUxMXhWOAbn9yb0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Error patterns of sequencing reads. (A) Average (mean ±SE) insertion, deletion and substitution per aligned base for individual reads (n = 21), consensus sequences based on three reads (n = 4), five reads (n = 4), seven reads (n = 4), 11 reads (n = 3) and 21 reads (n = 2). Ambiguous refers to unknown nucleotides (N) in consensus sequence. (B) Substitution matrix. (C) Distribution of deletions according to the underlying local sequence characteristics: homopolymer stretches, dinucleotide repeats, other repeat types or no patterns. 86% of deletions occur in homopolymer stretches. (D) Dependency of the number of deleted nucleotides on the length of the homopolymer stretch." path-from-xml="dsw064f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473483/dsw064f2.tif" path-from-xml="dsw064f2.tif" target="_blank">View large</a><a section="88473483" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064f2.png?Expires=2147483647&Signature=S7LEZBxKsarJazWccvwgyG2y3ObE~2BzqJBCQXIkZ5wdMmnl8B8yqy6HNC93IoLwKlAuBbzj8-2MkFyzCEAltEJ4ZXMyg-73Q5vF4lZGo982ynxJyGArNpQ4A3E6T0ydRGm~DySUI18DYCzRVnA2yhnBlcwQHrWruBNm9DaCQ74SLFXNElEMwD2a6P43SZxTsqY6NqUxOrqcUiqgl0NYkcHE06wDtFLD~GBQTAx88pkBD7p5ASQItc6AnFSe26PPEAswJWyI-bb0xP-3GwmrU6HrB0Xq1b3YVzl0eUKtX5mEfEMM6iw5mcvAqKOGvS3TQVN7Jf7WvhfAWrjPySuNfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473483&ar=2972857&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw064f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Error patterns of sequencing reads. (A<strong>)</strong> Average (mean ±SE) insertion, deletion and substitution per aligned base for individual reads (<em>n</em> = 21), consensus sequences based on three reads (<em>n</em> = 4), five reads (<em>n</em> = 4), seven reads (<em>n</em> = 4), 11 reads (<em>n</em> = 3) and 21 reads (<em>n</em> = 2). Ambiguous refers to unknown nucleotides (N) in consensus sequence. (B) Substitution matrix. (C) Distribution of deletions according to the underlying local sequence characteristics: homopolymer stretches, dinucleotide repeats, other repeat types or no patterns. 86% of deletions occur in homopolymer stretches. (D) Dependency of the number of deleted nucleotides on the length of the homopolymer stretch.</p></div></div><div content-id="dsw064-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/m_dsw064f2.png?Expires=2147483647&Signature=E7mZnkaNZSGE9caDspExVJl1AQ1TmSjr9V6C5yP52uCxDutj-vxr5dNM~NKHiJ1OCJ7jdpjAJaYQiR0xGB5OsOhv3Fz~531ZCQXxYBc7BVkYi~tlqHL99YGHDF0O8wX1bmZxFqsrsFsOqB2G~0nMR~kmSS4u8FX3laSdJR6xGE6CE5MAMRTlp~kvMe1IQ2cIw106XcOWTVze052ZlqeR0SEDmMl0b3pUmeDkM1bZRpbtiKTY~mlRi9hGKP3TKjDH54XfT7uwUTtW7ANIfOZSMNkldnYFyqOfyQg~xWX7EQWfcLkBOTTPFxIOURr5VA~VE0zpG8iUxMXhWOAbn9yb0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Error patterns of sequencing reads. (A) Average (mean ±SE) insertion, deletion and substitution per aligned base for individual reads (n = 21), consensus sequences based on three reads (n = 4), five reads (n = 4), seven reads (n = 4), 11 reads (n = 3) and 21 reads (n = 2). Ambiguous refers to unknown nucleotides (N) in consensus sequence. (B) Substitution matrix. (C) Distribution of deletions according to the underlying local sequence characteristics: homopolymer stretches, dinucleotide repeats, other repeat types or no patterns. 86% of deletions occur in homopolymer stretches. (D) Dependency of the number of deleted nucleotides on the length of the homopolymer stretch." path-from-xml="dsw064f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473483/dsw064f2.tif" path-from-xml="dsw064f2.tif" target="_blank">View large</a><a section="88473483" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064f2.png?Expires=2147483647&Signature=S7LEZBxKsarJazWccvwgyG2y3ObE~2BzqJBCQXIkZ5wdMmnl8B8yqy6HNC93IoLwKlAuBbzj8-2MkFyzCEAltEJ4ZXMyg-73Q5vF4lZGo982ynxJyGArNpQ4A3E6T0ydRGm~DySUI18DYCzRVnA2yhnBlcwQHrWruBNm9DaCQ74SLFXNElEMwD2a6P43SZxTsqY6NqUxOrqcUiqgl0NYkcHE06wDtFLD~GBQTAx88pkBD7p5ASQItc6AnFSe26PPEAswJWyI-bb0xP-3GwmrU6HrB0Xq1b3YVzl0eUKtX5mEfEMM6iw5mcvAqKOGvS3TQVN7Jf7WvhfAWrjPySuNfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473483&ar=2972857&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw064f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Error patterns of sequencing reads. (A<strong>)</strong> Average (mean ±SE) insertion, deletion and substitution per aligned base for individual reads (<em>n</em> = 21), consensus sequences based on three reads (<em>n</em> = 4), five reads (<em>n</em> = 4), seven reads (<em>n</em> = 4), 11 reads (<em>n</em> = 3) and 21 reads (<em>n</em> = 2). Ambiguous refers to unknown nucleotides (N) in consensus sequence. (B) Substitution matrix. (C) Distribution of deletions according to the underlying local sequence characteristics: homopolymer stretches, dinucleotide repeats, other repeat types or no patterns. 86% of deletions occur in homopolymer stretches. (D) Dependency of the number of deleted nucleotides on the length of the homopolymer stretch.</p></div></div>                            <h3 scrollto-destination=88473484 id="88473484" class="section-title" >3.2. Apis mellifera</h3>
<p>For <em>A. mellifera</em>, six overlapping PCR products were sequenced spanning the entire <em>mrjp</em> gene cluster comprising ≈ 57 kb (<a reveal-id="dsw064-F3" data-open="dsw064-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>; GenBank accession: KY087961) broadly confirming the original structure of the cluster<a reveal-id="dsw064-B4" data-open="dsw064-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsw064-B5" data-open="dsw064-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> with <em>mrjp3</em>, <em>1</em>, <em>4</em>, <em>6</em> and <em>5</em> located at the 5′ end of the cluster. In the reference genome, <em>mrjp5</em> is than followed by a partial <em>mrjp</em>-<em>like</em> sequence (LOC102654393), the pseudogene <em>mrjp2-like</em> and <em>mrjp2</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). However, we observed an 8 kb-fragment size variation in the region spanning <em>mrjp5</em> to <em>2</em> (amAmp4) (expected: ≈ 22 kb; observed: ≈ 14 kb). In contrast to the reference genome, the sequence of amplicon amAmp4 did neither include the pseudogene <em>mrjp2-like</em> nor its flanking region (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). The high-sequence identity between <em>mrjp2-like</em> (GB55211) and <em>mrjp2</em> (GB55212) suggests that this sequence stretch is a segmental duplication resulting from a short read sequencing assembly error (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Thus <em>mrjp2-like</em> (GB55211) is actually an assembly artefact and not a pseudogene within the cluster. In order to confirm our findings, we amplified the region spanning <em>mrjp5</em>–<em>mrjp7</em> using two different primer combinations. For both primer pairs observed and expected amplicon size differed by ≈ 8 kb (I. primer: expected: ≈27.5 kb; observed ≈19.5 kb; II. primer pair: expected: ≈25.0 kb; observed ≈17.0 kb) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). In addition, the amAmp4 consensus sequence is not in line with the current <em>mrjp2</em> (GB55212) annotation (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). The original 430 bp duplication in GB55212 (comprising the first two introns, as well as exon2, OGSv3.2) was absent in our amAmp4 sequence, probably also representing an assembly artefact in Amel_4.5. Sanger sequencing of am<em>mrjp2</em> confirmed the absence of this sequence stretch (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). However, our sequence of <em>mrjp2</em> matched the NCBI mRNA prediction (NM_001011580.1). A partial <em>mrjp</em>-like sequence (LOC102654393), symbolized by the black destructed arrow in <a reveal-id="dsw064-F3" data-open="dsw064-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>, located between <em>mrjp5</em> and <em>2</em> (Amel_4.5)<sup>27</sup> could be verified (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>).</p>                        <a id="88473486" scrollto-destination="88473486"></a>
<div id="dsw064-F3" data-id="dsw064-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/m_dsw064f3.png?Expires=2147483647&Signature=NtssqCSZravMvFHIGvE-7URkCtgof-1leDqux~KVeI1-IOt4qdM5Hvi2OBY-tuclXhU~sL9RAgDD5DaPM7Aku9Th0vnjvRIQc42OsfUrzqici-j9hbVdq9FvFFe1eI2RWIn5Su87fqznw28aHiKbCl5VKFrengFlLY5j0OY1nkFAXJ4rw9GSJ55GZqAoF7aEqq233gvUJ9UXzv1vzSatOJU~-GggBVnxJPxzI5Jef8kchpd92MZ05RvOEc8famCvhQpcAwTVaNvAoJO1BKWktSgjnNn5eMi~nsQmNFUL-ZokXmyfgEejuDg~BIZlYdDxLYfQu-~2obbEGB9GRiCTAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic organization of the major royal jelly protein gene cluster across three species of the genus Apis. To emphasize the structural organization of mrjps, exons are illustrated in black. Grey arrows refer to putative mrjps based on draft genome sequences. Black destructed arrows illustrate pseudogenized genes. Location of the respective amplicons (adAmp 1–7; afAmp 1–6; amAmp 1–6) within the cluster is also illustrated; with dashed lines referring to fragments deviating from expected product sizes. ad, A. dorsata; am, A. mellifera; af, A. florea. Gene names are the corrected ones after the phylogenetic analysis." path-from-xml="dsw064f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473486/dsw064f3.tif" path-from-xml="dsw064f3.tif" target="_blank">View large</a><a section="88473486" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064f3.png?Expires=2147483647&Signature=R~-v6hsXEUESXnBXKfyU~M9cPE2yYSIBTiagLIv1xK~rH2-MYgFgGjEZFNinPmvdNfwP34bAden11vQcFodVZiXxHXCGbOvZBFFS481Rc-FKpyaH2~52p8RAgzHX3DArB4GxzY7n4sU-X~L50Ip9FNdFNNI1uXxIzsVHkx5tbC9Eoj18JRjK08XyIpKKmGhq2Mtx8IlsbIjYynjj784pGjVPxi~y7McwJcTz8n5PEdf4YueWxbNDhyIOPeZV9E8TO-BG6wZCpICWIkPHzpRN~QGEEekKWN9SB4txJnXBqN-UGr7JCzfPQh7h~us7U0xzIu6RVIamCbPJfMSAnQcLiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473486&ar=2972857&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw064f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic organization of the <em>major royal jelly protein</em> gene cluster across three species of the genus <em>Apis.</em> To emphasize the structural organization of <em>mrjps</em>, exons are illustrated in black. Grey arrows refer to putative <em>mrjps</em> based on draft genome sequences. Black destructed arrows illustrate pseudogenized genes. Location of the respective amplicons (adAmp 1–7; afAmp 1–6; amAmp 1–6) within the cluster is also illustrated; with dashed lines referring to fragments deviating from expected product sizes. ad, <em>A. dorsata</em>; am, <em>A. mellifera</em>; af, <em>A. florea</em>. Gene names are the corrected ones after the phylogenetic analysis.</p></div></div><div content-id="dsw064-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/m_dsw064f3.png?Expires=2147483647&Signature=NtssqCSZravMvFHIGvE-7URkCtgof-1leDqux~KVeI1-IOt4qdM5Hvi2OBY-tuclXhU~sL9RAgDD5DaPM7Aku9Th0vnjvRIQc42OsfUrzqici-j9hbVdq9FvFFe1eI2RWIn5Su87fqznw28aHiKbCl5VKFrengFlLY5j0OY1nkFAXJ4rw9GSJ55GZqAoF7aEqq233gvUJ9UXzv1vzSatOJU~-GggBVnxJPxzI5Jef8kchpd92MZ05RvOEc8famCvhQpcAwTVaNvAoJO1BKWktSgjnNn5eMi~nsQmNFUL-ZokXmyfgEejuDg~BIZlYdDxLYfQu-~2obbEGB9GRiCTAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic organization of the major royal jelly protein gene cluster across three species of the genus Apis. To emphasize the structural organization of mrjps, exons are illustrated in black. Grey arrows refer to putative mrjps based on draft genome sequences. Black destructed arrows illustrate pseudogenized genes. Location of the respective amplicons (adAmp 1–7; afAmp 1–6; amAmp 1–6) within the cluster is also illustrated; with dashed lines referring to fragments deviating from expected product sizes. ad, A. dorsata; am, A. mellifera; af, A. florea. Gene names are the corrected ones after the phylogenetic analysis." path-from-xml="dsw064f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473486/dsw064f3.tif" path-from-xml="dsw064f3.tif" target="_blank">View large</a><a section="88473486" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064f3.png?Expires=2147483647&Signature=R~-v6hsXEUESXnBXKfyU~M9cPE2yYSIBTiagLIv1xK~rH2-MYgFgGjEZFNinPmvdNfwP34bAden11vQcFodVZiXxHXCGbOvZBFFS481Rc-FKpyaH2~52p8RAgzHX3DArB4GxzY7n4sU-X~L50Ip9FNdFNNI1uXxIzsVHkx5tbC9Eoj18JRjK08XyIpKKmGhq2Mtx8IlsbIjYynjj784pGjVPxi~y7McwJcTz8n5PEdf4YueWxbNDhyIOPeZV9E8TO-BG6wZCpICWIkPHzpRN~QGEEekKWN9SB4txJnXBqN-UGr7JCzfPQh7h~us7U0xzIu6RVIamCbPJfMSAnQcLiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473486&ar=2972857&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw064f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic organization of the <em>major royal jelly protein</em> gene cluster across three species of the genus <em>Apis.</em> To emphasize the structural organization of <em>mrjps</em>, exons are illustrated in black. Grey arrows refer to putative <em>mrjps</em> based on draft genome sequences. Black destructed arrows illustrate pseudogenized genes. Location of the respective amplicons (adAmp 1–7; afAmp 1–6; amAmp 1–6) within the cluster is also illustrated; with dashed lines referring to fragments deviating from expected product sizes. ad, <em>A. dorsata</em>; am, <em>A. mellifera</em>; af, <em>A. florea</em>. Gene names are the corrected ones after the phylogenetic analysis.</p></div></div><p>To independently validate our findings, we used <em>A. m. carnica</em> short sequencing data<a reveal-id="dsw064-B28" data-open="dsw064-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> to analyze coverage variation between the original <em>mrjp</em> gene cluster (GC) and our long-read guided assembly (lacking segmental duplications). Whereas sequence coverage for an unchanged control region (<em>mrjp1</em>) is similar (72×) for both gene cluster assemblies, coverage of <em>mrjp2</em> considerably increased when mapped against our long-read guided assembly (original GC: 16×; new GC: 64×).</p><p>The genomic organization of the genes <em>mrjp7</em>, <em>mrjp8</em> and <em>mrjp9</em> located downstream of <em>mrjp2</em> could be verified (<a reveal-id="dsw064-F3" data-open="dsw064-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>).</p>                            <h3 scrollto-destination=88473489 id="88473489" class="section-title" >3.3. Apis florea</h3>
<p>First insights in the gene cluster of <em>A. florea</em> suggested a similar gene order to <em>A. mellifera</em> (<em>mrjp3</em>, <em>1</em>, <em>4</em>, <em>6</em>, <em>5</em>, <em>2</em>, <em>Ψ</em>, <em>7</em>, <em>8</em> and <em>9</em>), with a switch in position of the so far predicted pseudogene (<em>mrjp-Ψ/2-like)</em> and <em>mrjp2</em> (for more information see <a reveal-id="dsw064-F2" data-open="dsw064-F2" class="link link-reveal link-table xref-fig">Fig. 2</a> in Buttstedt et al.<a reveal-id="dsw064-B8" data-open="dsw064-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a>). The amplicon sizes for <em>A. florea</em> were in the expected range, except for afAmp6 (expected: ≈ 10.8 kb; observed: ≈ 7.8 kb; see below). Sequencing four overlapping PCR products at the 5′prime end of the gene cluster (<a reveal-id="dsw064-F3" data-open="dsw064-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>), confirmed the proposed gene order<a reveal-id="dsw064-B8" data-open="dsw064-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> for five <em>mrjps</em> (af<em>mrjp3</em>–<em>5)</em>. We were not able to amplify the region between <em>mrjp5</em> and <em>7</em> which was supposed to harbour af<em>mrjp2</em> and af<em>mrjp-Ψ</em><sup>8</sup><em>.</em> However, analyzing the reference genomic sequence revealed indeed two full-length genes to be located between <em>mrjp5</em> and <em>mrjp7</em> (<a reveal-id="dsw064-F3" data-open="dsw064-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). This is in contrast to <em>A. mellifera</em>, where <em>mrjp5</em> is followed by one partial <em>mrjp-like</em> sequence and a full-length <em>mrjp</em> (<em>mrjp2</em>). Amplicons afAmp5 and 6 confirmed the presence of <em>mrjp7</em> and <em>8</em> but not <em>mrjp9</em>. Instead we identified an additional afAmp6-R primer-binding site on an unplaced genomic scaffold04303 (1909 bp). This scaffold showed a ≈550bp sequence overlap with the 3′prime end of afAmp6, whereas a part (the last 60 bp) of the remaining sequence is highly identical to the 5′prime region of <em>mrjp9</em>. Thus, scaffold04303 should be replaced between <em>mrjp8</em> and <em>mrjp9</em>, to close the gap in the current assembly. Assemblies based on MinION<sup>TM</sup> consensus sequences have been deposited on Genbank (Accession nos. KY087959 and KY087960).</p>                            <h3 scrollto-destination=88473491 id="88473491" class="section-title" >3.4. Apis dorsata</h3>
<p>For <em>A. dorsata</em>, up to now no <em>mrjp</em> gene order has been proposed. Here, seven overlapping PCR products were sequenced spanning two contiguous parts of the <em>mrjp</em> gene cluster (<a reveal-id="dsw064-F3" data-open="dsw064-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>; GenBank accession nos. KY087957 and KY087958). The observed amplicon sizes were in the expected range, except for adAmp6 (expected: ≈ 14.9 kb; observed: ≈ 12.5 kb). The majority of the reads mapped to the unplaced genomic scaffold_260 (KI273360.1) comprising ad<em>mrjp3</em>–<em>8</em>, whereas a minor proportion mapped to the unplaced genomic scaffold_444 (KI273484.1) containing <em>admrjp9</em>. The <em>mrjp</em> region contained several gaps within the partial reference sequences, which could be resolved based on the nanopore-derived consensus sequence. Our sequence analysis confirmed the general arrangement of <em>mrjp3</em>–<em>5</em>. One full-length <em>mrjp-like</em> gene, located downstream of ad<em>mrjp5</em> (amplicon adAmp5) (<a reveal-id="dsw064-F3" data-open="dsw064-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>), could not unambiguously assigned to an <em>A.</em><em>mellifera</em> homologue. Furthermore <em>a</em> ≈ 1.4 kb sequence stretch, present in the partial genome sequence between ad<em>mrjp2</em> and ad<em>mrjp8</em>, was absent in our adAmp6 sequence, probably representing an assembly artefact. Whereas amplicon adAmp6 confirmed the presence of <em>admrjp2</em>, we only found a truncated <em>A. dorsata</em> homologue of <em>mrjp7</em> at the supposed position in <em>A. mellifera</em> and <em>A. florea</em>. The location of <em>mrjp8</em> and <em>9</em> at the 3′ end of the cluster was verified by amplicon adAmp7.</p>                            <h3 scrollto-destination=88473493 id="88473493" class="section-title" >3.5. A new member of the mrjp family—mrjp10</h3>
<p>In all three Apis species, <em>mrjp3</em>, <em>1</em>, <em>4</em>, <em>6</em> and <em>5</em> showed an identical location at the 5′ end of the cluster and <em>mrjp8</em> and <em>9</em> were always found at the 3′ end (<a reveal-id="dsw064-F3" data-open="dsw064-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Between <em>mrjp5</em> and <em>8</em>, the gene order seems to be less clear: In <em>A. mellifera</em>, a partial <em>mrjp-like</em> sequence is followed by am<em>mrjp2</em> and <em>7;</em> in <em>A florea</em>, two full-length af<em>mrjps</em> (which have previously been described as af<em>mrjp2</em> and af<em>mrjp-Ψ</em><sup>8</sup>) are followed by af<em>mrjp7</em> and in <em>A. dorsata</em> an unknown full-length ad<em>mrjp</em> is followed by ad<em>mrjp2</em> and a partial ad<em>mrjp</em> (<a reveal-id="dsw064-F3" data-open="dsw064-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Phylogenetic reconstruction based on MRJP amino acid sequences suggest that <em>A. dorsata</em> and <em>A. florea</em> seem to encode a previously uncharacterized member of the gene cluster, located directly downstream of <em>mrjp5</em>, which we propose to be called <em>mrjp10</em> (<a reveal-id="dsw064-F4" data-open="dsw064-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Thus, the full-length <em>mrjp</em> of <em>A. florea</em> found downstream of af<em>mrjp5</em> is not as formerly predicted af<em>mrjp2</em><sup>8</sup> but indeed af<em>mrjp10</em>. In our analyses, afMRJP10 clustered together with adMRJP10, missing a homologous protein in <em>A. mellifera</em> where multiple sequence alignment suggests a truncated am<em>mrjp</em>-like gene (corresponding to LOC102654393) located between am<em>mrjp5</em>and am<em>mrjp2</em>. Evidence for a full-length <em>mrjp10</em> homologue in <em>A. mellifera</em> is missing due to the absence of the first three exons. The occurrence of <em>mrjp10</em> in <em>A. florea</em> and <em>A. dorsata</em> has in addition been validated by Sanger sequencing (GenBank: KX925848 and KX925849).</p>                        <a id="88473495" scrollto-destination="88473495"></a>
<div id="dsw064-F4" data-id="dsw064-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/m_dsw064f4.png?Expires=2147483647&Signature=kKHtRSRPO29F1i3tNv0kRADfrTpoAAL3TkLS3DUaEfo1VGknZYz39u6KdLvu0sPv19OR1gWjqqC0ADyD~3BKdi-tHOK6sdLXiqry0eQhFx99bWAZv5FcbGrhq~tTLX2zggg49Vg1qyOM1T9Z6CxPipG7fmbcI-6MWDGQoQRmaTqSZF4PL2kEVXG0cu4HBVwJwOZp8MycnBK1JMpYAvS-0vxrIN7nNRdkvjq9yXSzVwMAWkJOvhCD0GA2Uj0ZAgjapwVmojjSh0ZACZvl~sSmZwv8HEhAkzEE4tT0nVwwRz1Oqo5BBLIcpJw~AIZ3rSFn--X3UKfiy64Rsy-d9mN0ZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogeny of Apis major royal jelly proteins (MRJP). The maximum-likelihood tree was reconstructed from aligned amino acid sequences using Jones–Taylor–Thornton, including bootstrapping (500 replications). A discrete Gamma distribution was used to model evolutionary rate differences among sites (five categories, G parameter = 2.1115). All positions containing gaps and missing data were eliminated. There were a total of 365 positions in the final dataset. Model selection was performed using MEGA version 5 (Tamura et al.25). Sequences for afMRJP10 (old = afMRJP2), afMRJP2 (old = afMRJPψ) and afMRJP9 were derived from the draft genome (scaffold 1824). Ad, A. dorsata; am, A. mellifera; af, A. florea." path-from-xml="dsw064f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473495/dsw064f4.tif" path-from-xml="dsw064f4.tif" target="_blank">View large</a><a section="88473495" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064f4.png?Expires=2147483647&Signature=eNRjlyhachApkZEp-xOcRFpgwRSblrovlFtx7~muow2hJ-Syj-~kndlq9zexbrp74YIwn1CmCvAYKwAjwEPyJGi5yTBaYXb0ffxdZTpSYgb-2CSqGcgu2gF3a8LUZgE7sLQdycYlgg1G-8MPPW6kvkb4m7w5hYN4ct2pNoX5MtJefNvMzh0RwBWMjgYfWFsLEsNrkzLy9LLsiWNvge7RJ4SdgzNu-K8rZaSTl-wr1nzB0eLBW6jTQh0dN7rAqalQtp49gkb3y7bbC1-eDey-1k201bB2NcpvQqwxOjcy7YL33aUGETAqiTwwmEIkrobKoHpdYSsPmS5nQkkyIMZ8fw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473495&ar=2972857&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw064f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogeny of <em>Apis</em> major royal jelly proteins (MRJP). The maximum-likelihood tree was reconstructed from aligned amino acid sequences using Jones–Taylor–Thornton, including bootstrapping (500 replications). A discrete Gamma distribution was used to model evolutionary rate differences among sites (five categories, G parameter = 2.1115). All positions containing gaps and missing data were eliminated. There were a total of 365 positions in the final dataset. Model selection was performed using MEGA version 5 (Tamura et al.<a reveal-id="dsw064-B25" data-open="dsw064-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a>). Sequences for afMRJP10 (old = afMRJP2), afMRJP2 (old = afMRJPψ) and afMRJP9 were derived from the draft genome (scaffold 1824). Ad, <em>A. dorsata</em>; am, <em>A. mellifera</em>; af, <em>A. florea</em>.</p></div></div><div content-id="dsw064-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/m_dsw064f4.png?Expires=2147483647&Signature=kKHtRSRPO29F1i3tNv0kRADfrTpoAAL3TkLS3DUaEfo1VGknZYz39u6KdLvu0sPv19OR1gWjqqC0ADyD~3BKdi-tHOK6sdLXiqry0eQhFx99bWAZv5FcbGrhq~tTLX2zggg49Vg1qyOM1T9Z6CxPipG7fmbcI-6MWDGQoQRmaTqSZF4PL2kEVXG0cu4HBVwJwOZp8MycnBK1JMpYAvS-0vxrIN7nNRdkvjq9yXSzVwMAWkJOvhCD0GA2Uj0ZAgjapwVmojjSh0ZACZvl~sSmZwv8HEhAkzEE4tT0nVwwRz1Oqo5BBLIcpJw~AIZ3rSFn--X3UKfiy64Rsy-d9mN0ZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogeny of Apis major royal jelly proteins (MRJP). The maximum-likelihood tree was reconstructed from aligned amino acid sequences using Jones–Taylor–Thornton, including bootstrapping (500 replications). A discrete Gamma distribution was used to model evolutionary rate differences among sites (five categories, G parameter = 2.1115). All positions containing gaps and missing data were eliminated. There were a total of 365 positions in the final dataset. Model selection was performed using MEGA version 5 (Tamura et al.25). Sequences for afMRJP10 (old = afMRJP2), afMRJP2 (old = afMRJPψ) and afMRJP9 were derived from the draft genome (scaffold 1824). Ad, A. dorsata; am, A. mellifera; af, A. florea." path-from-xml="dsw064f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/88473495/dsw064f4.tif" path-from-xml="dsw064f4.tif" target="_blank">View large</a><a section="88473495" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064f4.png?Expires=2147483647&Signature=eNRjlyhachApkZEp-xOcRFpgwRSblrovlFtx7~muow2hJ-Syj-~kndlq9zexbrp74YIwn1CmCvAYKwAjwEPyJGi5yTBaYXb0ffxdZTpSYgb-2CSqGcgu2gF3a8LUZgE7sLQdycYlgg1G-8MPPW6kvkb4m7w5hYN4ct2pNoX5MtJefNvMzh0RwBWMjgYfWFsLEsNrkzLy9LLsiWNvge7RJ4SdgzNu-K8rZaSTl-wr1nzB0eLBW6jTQh0dN7rAqalQtp49gkb3y7bbC1-eDey-1k201bB2NcpvQqwxOjcy7YL33aUGETAqiTwwmEIkrobKoHpdYSsPmS5nQkkyIMZ8fw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=88473495&ar=2972857&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw064f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogeny of <em>Apis</em> major royal jelly proteins (MRJP). The maximum-likelihood tree was reconstructed from aligned amino acid sequences using Jones–Taylor–Thornton, including bootstrapping (500 replications). A discrete Gamma distribution was used to model evolutionary rate differences among sites (five categories, G parameter = 2.1115). All positions containing gaps and missing data were eliminated. There were a total of 365 positions in the final dataset. Model selection was performed using MEGA version 5 (Tamura et al.<a reveal-id="dsw064-B25" data-open="dsw064-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a>). Sequences for afMRJP10 (old = afMRJP2), afMRJP2 (old = afMRJPψ) and afMRJP9 were derived from the draft genome (scaffold 1824). Ad, <em>A. dorsata</em>; am, <em>A. mellifera</em>; af, <em>A. florea</em>.</p></div></div><p>The protein sequence inferred from the gene located downstream of af<em>mrjp10</em>, formerly predicted as af<em>mrjp-Ψ</em>,<a reveal-id="dsw064-B8" data-open="dsw064-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> formed a separate clade together with amMRJP2 and adMRJP2 (<a reveal-id="dsw064-F4" data-open="dsw064-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Thus, all three <em>Apis</em> species possess a full-length <em>mrjp2</em>. In contrast, amMRJP7 clustered together with afMRJP7 but a full length homologue in <em>A. dorsata</em> was missing (<a reveal-id="dsw064-F4" data-open="dsw064-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>).</p>                            <h2 scrollto-destination=88473497 id="88473497" class="section-title" >4. Conclusions</h2>
<p>In the era of sociogenomics, comparative analyses are key to understand features linked to advanced social organization (e.g.<a reveal-id="dsw064-B29" data-open="dsw064-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a><sup>,</sup><a reveal-id="dsw064-B30" data-open="dsw064-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a>)—presuming high-quality (draft) genomes and reliable gene predictions. However, even well-studied genomes are not error-free<a reveal-id="dsw064-B2" data-open="dsw064-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsw064-B3" data-open="dsw064-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> and this might affect downstream dependencies. Technical innovations including improved sequencing strategies, assembly algorithms and gene prediction methods, have been shown to substantially enhance the initial <em>A. mellifera</em> reference genome.<a reveal-id="dsw064-B27" data-open="dsw064-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> However, our results provide evidence that even additional incorporation of alternative sequencing strategies (ABI SOLiD and Roche 454 paired-end sequencing) was insufficient to fully resolve a complex region.</p><p>In <em>A. mellifera</em>, gene order within a multi-copy gene cluster has been previously determined by Drapeau et al. as <em>y-e3</em>, am<em>mrjp3</em>, am<em>mrjp1</em>, am<em>mrjp4</em>, am<em>mrjp6</em>, am<em>mrjp5</em>, a truncated RJ gene am<em>mrjp-Ψ</em>, am<em>mrjp2</em>, am<em>mrjp7</em>, am<em>mrjp8</em>, am<em>mrjp9</em> and <em>y-h</em><em>.</em><a reveal-id="dsw064-B4" data-open="dsw064-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsw064-B5" data-open="dsw064-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> In the current NCBI annotation, am<em>mrjp5</em> is followed by a truncated <em>mrjp</em> (probably <em>mrjp-Ψ</em> in the publication by Drapeau et al.<a reveal-id="dsw064-B4" data-open="dsw064-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a>) and a full-length <em>mrjp</em> named am<em>mrjp2-like</em> (in recent publications referred to as pseudogene). The proposed am<em>mrjp2-like</em> gene (pseudo; GB55211) located between am<em>mrjp5</em> and am<em>mrjp2</em> appears to be an assembly artefact arising from short read sequencing as we show here by long-amplicon sequencing. The truncated variant already described by Drapeau et al.<a reveal-id="dsw064-B4" data-open="dsw064-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> as pseudogene was also found in our study. The honeybee genome sequence has repeatedly been used as a framework studying key mechanisms of eusociality. However, sequence assembly errors will affect reference genome-guided analyses: e.g. variant calling (polymorphisms) or skew large-scale gene expression analyses. For example Mao et al.<a reveal-id="dsw064-B31" data-open="dsw064-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> and Vojvodic et al.<a reveal-id="dsw064-B32" data-open="dsw064-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> reported on the expression of <em>mrjp2-like</em> (pseudo; GB55211) in <em>A. mellifera</em>. However, these reads most likely belonged to am<em>mrjp2</em>, thus biasing the gene expression estimates of am<em>mrjp2</em>. Evolutionary rate estimates<a reveal-id="dsw064-B33" data-open="dsw064-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsw064-B34" data-open="dsw064-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> might be biased as well. Moreover, long-amplicon reads are also suitable to solve inter-scaffold gaps as shown for <em>A. florea</em>. For example scaffold04303 between <em>mrjp8</em> and <em>mrjp9</em>, replaced the gap in the current assembly. Thus, even in spite of ∼10% nucleotide-sequencing errors, MinION reads were most suitable to reconstruct the organization of extensive gene clusters. The combination of long-range PCR with third-generation sequencing overcame the intrinsic problems of short-sequencing reads assemblies.</p><p>Our results suggest a similar gene cluster organization among the three <em>Apis</em> species. However, the number of full-length homologues is different. Whereas the <em>A. florea</em> genome possesses 10 full-length <em>mrjp</em> genes, <em>A. dorsata</em> and <em>A. mellifera</em> comprise only nine. In that regard, our results indicate lineage-specific evolutionary fates of <em>mrjp</em> duplicates located between <em>mrjp5</em> and <em>8.</em> In <em>A. florea</em>, the region between <em>mrjp5</em> and <em>8</em> encodes three full-length <em>mrjps</em>: af<em>mrjp10</em>—a previously uncharacterized member of the <em>mrjp</em>/<em>yellow</em> family, af<em>mrjp2</em> and af<em>mrjp7</em>. A full-length copy of <em>mrjp10</em> is also existing in <em>A. dorsata</em> but only partially present in <em>A. mellifera</em> (LOC102654393). In contrast, <em>mrjp7</em> is present in <em>A. mellifera</em> and <em>A. florea</em>, but a full-length homolog in <em>A. dorsata</em> is missing.</p><p>A phylogenetic reconstruction based on the MRJP amino acid sequences revealed 10 terminal <em>MRJP</em> clades (<a reveal-id="dsw064-F4" data-open="dsw064-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). However, the various orthologous <em>mrjp</em> genes are more similar among species than the within-species paralog genes, suggesting a common ancestral gene cluster structure for all studied species. Taking into account recent phylogenetic findings, indicating that <em>A. florea</em> is basal to the cavity nesting and giant honeybees,<a reveal-id="dsw064-B35" data-open="dsw064-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> we conclude that the common ancestor likely encoded 10 <em>mrjps</em>. This is further supported by the presence of incomplete gene copies in the more derived species, <em>A. mellifera</em> (<em>mrjp10</em>) and <em>A. dorsata</em> (<em>mrjp7</em>). The region from <em>mrjp5</em>–<em>8</em> thus might have experienced a period of reduced selective constraint accompanied with accumulation of non-synonymous deleterious mutations, facilitating independent gene loss in <em>A. mellifera</em> and <em>A. dorsata</em> after their divergence.</p><p>Temporal functional redundancy following gene duplication is assumed to reduce selective constraint on one copy und thus provides a prerequisite for pseudogenization, neo- and/or sub-functionalization,<a reveal-id="dsw064-B36 dsw064-B37 dsw064-B38" data-open="dsw064-B36 dsw064-B37 dsw064-B38" class="link link-ref link-reveal xref-bibr"><sup>36–38</sup></a> ultimately determining the evolutionary fate of gene copies (preservation or loss). In case of the <em>Apis mrjp</em> genes, neo- and subfunctionalization might synergistically contribute to their accumulation and preservation. Expression patterns for the ancestral <em>mrjp8</em> and <em>9</em> differ remarkably from the remaining <em>mrjp</em>s. Almost ubiquitous transcript abundances of <em>mrjp8</em> and <em>9</em> across various body sections and independent of caste suggest a more general physiological than a nutritive function for both gene products.<a reveal-id="dsw064-B4" data-open="dsw064-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsw064-B13" data-open="dsw064-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Hence, subfunctionalization might explain preservation of this ancestral gene pair <em>mrjp8</em> and <em>9.</em> In contrast, worker- and tissue-biased expression of <em>mrjp1</em><em>–</em><em>7</em> indicates spatial and functional separation from <em>mrjp8</em> and <em>9</em>, suggesting neofunctionalization of an ancestral duplicate. All worker-<em>mrjps</em> are predominantly expressed in heads, although their spatial expression patterns indicate further differentiation–with preferential expression in hypopharyngeal glands (for those genes encoding proteins with nutritional value) and/or the brain.<a reveal-id="dsw064-B39" data-open="dsw064-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Moreover, elevated transcript abundances of some <em>mrjps</em> in brood-raising nurse bees compared with foragers (and <em>vice versa</em>) indicate more specialized functions related to division of labor.<a reveal-id="dsw064-B13" data-open="dsw064-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsw064-B32" data-open="dsw064-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> Phylogenetic trajectories revealed different complements of worker-<em>mrjp</em> genes within <em>Apis</em> and in this regard, each species might express a slightly different set of <em>mrjps</em>, whose conditional expression is assumed to impact their evolution. Genes almost exclusively expressed in sterile workers (in <em>A. mellifera</em>: <em>mrjp1</em>–<em>7</em>) but with a phenotypic effect in another individual, are considered as indirect effect genes and are thus expected to evolve under relaxed purifying selection.<a reveal-id="dsw064-B40" data-open="dsw064-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> In contrast, <em>mrjp8</em> and <em>9</em> are both evenly expressed across all body sections and irrespective of social state,<a reveal-id="dsw064-B13" data-open="dsw064-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> thus exert a direct effect in the individual itself. Therefore, different evolutionary modes of <em>mrjps</em> might be a consequence of species-specific conditional expression of worker-<em>mrjps</em>. However, currently we lack information concerning timing and tissue specificity of expression even for the shared set of worker-<em>mrjps</em> for <em>A. florea</em> and <em>A. dorsata</em>. Nevertheless, at least we can draw on expression data for <em>A.</em><em>florea.</em><a reveal-id="dsw064-B41" data-open="dsw064-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Using the TSA BLAST database from NCBI, we were able to identify transcripts supporting the expression of the af<em>mrjps</em> including transcripts matching af<em>mrjp2</em> and af<em>mrjp10</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). However, using the <em>A. florea</em> antennal transcriptome (Bioproject PRJNA325192), we identified reads supporting the expression of all <em>mrjps</em> except <em>mrjp10</em>.</p><p>Also the cause of pseudogenization/loss of <em>mrjp7</em> in <em>A. dorsata</em> and <em>mrjp10</em> in <em>A. mellifera</em>, while preservation in the respectively other species and conservation of both duplicates in <em>A. florea</em>, remains obscure. Whereas increased transcript abundance of <em>mrjp7</em> in the mushroom bodies of <em>A. mellifera</em> are indicative for a function in the honeybee brain,<a reveal-id="dsw064-B13" data-open="dsw064-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsw064-B39" data-open="dsw064-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> absolutely nothing is known about the expression/function of <em>mrjp10</em> in <em>A. dorsata</em> or <em>A. florea</em>. Maybe both gene products exhibit at least partial functional redundancy and reciprocal gene loss is a consequence of dosage-sensitivity in the respective lineages. However, different patterns of <em>mrjp-like</em> gene loss and retention are common in Hymenoptera<a reveal-id="dsw064-B8" data-open="dsw064-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> and may be a consequence of lineage specific adaptation.</p><p>Characterization and comparative analysis of <em>mrjp</em> expression pattern in <em>A. florea</em> and <em>A. dorsata</em> would help to infer the extent of gene expression diversity or might indicate functional redundancy between lost/retained genes. In addition, comparative analyses of the <em>mrjp</em> gene cluster organization in other honeybee species belonging to the three subgenera (<em>Micrapis</em>, <em>Megapis</em> and <em>Apis</em>) might also help to unravel the evolutionary history of this multi-gene family.</p>                            <h2 scrollto-destination=88473505 id="88473505" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Alexis Beaurepaire for travelling around the world collecting bees and Eduard Schwarz for sequencing <em>mrjp2</em> in <em>A. mellifera</em>. We would like to thank G. Robinson, O. Rueppell, C. Elsik and the Baylor College of Medicine Human Genome Sequencing Center for making the genome data of <em>A. florea</em> and <em>A. dorsata</em> available pre-publication. This project was supported by the German Research Foundation (Deutsche Forschungsgemeinschaft—DFG, Grant MO 373/32-1 to R.F.A.M.).</p>                            <h3 scrollto-destination=88473507 id="88473507" class="section-title" >Conflict of interest</h3>
<p>The authors declare that they have no competing interests. H.M.G.L. is part of the MinION<sup>TM</sup> Access Programme (MAP). Oxford Nanopore Technologies Ltd contributed reagents free of charge and had no role in study design, data collection and analysis, or preparation of the manuscript.</p>                            <h3 scrollto-destination=88473509 id="88473509" class="section-title" >Accession number</h3>
<p>KY087957, KY087958, KY087959, KY087960, KY087961, KX925848, KX925849, KX951418, SRR4426288.</p>                            <h3 scrollto-destination=88473511 id="88473511" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=88473513 id="88473513" class="section-title" >Funding</h3>
<p>This project was supported by the German Research Foundation (Deutsche Forschungsgemeinschaft - DFG, Grant MO 373/32-1 to RFAM).</p>                            <h2 scrollto-destination=88473515 id="88473515" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw064-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div>, <div class="name string-name"><div class="surname">Yorke</div> <div class="given-names">J. A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Beware of mis-assembled genomes</div>. <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">4320</div>–<div class="lpage">1</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Beware%20of%20mis-assembled%20genomes&amp;author=S.%20L.%20Salzberg&amp;author=J.%20A.%20Yorke&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=4320-1" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti769" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti769" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti769"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16332717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kelley</div> <div class="given-names">D. R.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Detection and correction of false segmental duplications caused by genome mis-assembly</div>. <div class="source">Genome Biol</div> ., <div class="volume">11</div>, <div class="fpage">R28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20and%20correction%20of%20false%20segmental%20duplications%20caused%20by%20genome%20mis-assembly&amp;author=D.%20R.%20Kelley&amp;author=S.%20L.%20Salzberg&amp;publication_year=2010&amp;journal=Genome%20Biol&amp;volume=11&amp;pages=R28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-3-r28" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-3-r28" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-3-r28"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20219098" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Backström</div> <div class="given-names">N.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Assembly errors cause false tandem duplicate regions in the chicken (<em>Gallus gallus</em>) genome sequence</div>. <div class="source">Chromosoma</div> , <div class="volume">123</div>, <div class="fpage">165</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assembly%20errors%20cause%20false%20tandem%20duplicate%20regions%20in%20the%20chicken%20(Gallus%20gallus)%20genome%20sequence&amp;author=Q.%20Zhang&amp;author=N.%20Backstr%C3%B6m&amp;publication_year=2013&amp;journal=Chromosoma&amp;volume=123&amp;pages=165-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00412-013-0443-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00412-013-0443-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00412-013-0443-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24213641" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Drapeau</div> <div class="given-names">M. D.</div></div>, <div class="name string-name"><div class="surname">Albert</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kucharski</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Prusko</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Maleszka</div> <div class="given-names">R.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Evolution of the Yellow/Major Royal Jelly Protein family and the emergence of social behavior in honey bees</div>. <div class="source">Genome Res.</div> , <div class="fpage">1385</div>–<div class="lpage">94</div>.</div></div></div></div><div content-id="dsw064-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="collab">Honeybee Genome Sequencing Consortium</div>. <div class="year">2006</div>, 
<div class="article-title">Insights into social insects from the genome of the honeybee Apis mellifera</div>. <div class="source">Nature</div> , <div class="volume">443</div>, <div class="fpage">931</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05260" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05260" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05260"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17073008" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schmitzová</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Klaudiny</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Albert</div> <div class="given-names">Š.</div></div></div>, et al.   <div class="year">1998</div>, 
<div class="article-title">A family of major royal jelly proteins of the honeybee Apis mellifera L</div>. <div class="source">Cell. Mol. Life Sci. C</div> ., <div class="volume">54</div>, <div class="fpage">1020</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20family%20of%20major%20royal%20jelly%20proteins%20of%20the%20honeybee%20Apis%20mellifera%20L&amp;author=J.%20Schmitzov%C3%A1&amp;author=J.%20Klaudiny&amp;author=%C5%A0.%20Albert&amp;publication_year=1998&amp;journal=Cell.%20Mol.%20Life%20Sci.%20C&amp;volume=54&amp;pages=1020-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s000180050229" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs000180050229" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs000180050229"> </span></div> </div></div></div></div></div><div content-id="dsw064-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Albert</div> <div class="given-names">Š.</div></div>, <div class="name string-name"><div class="surname">Klaudiny</div> <div class="given-names">J.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">The MRJP/YELLOW protein family of Apis mellifera: Identification of new members in the EST library</div>. <div class="source">J. Insect Physiol</div> ., <div class="volume">50</div>, <div class="fpage">51</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20MRJP%2FYELLOW%20protein%20family%20of%20Apis%20mellifera%3A%20Identification%20of%20new%20members%20in%20the%20EST%20library&amp;author=%C5%A0.%20Albert&amp;author=J.%20Klaudiny&amp;publication_year=2004&amp;journal=J.%20Insect%20Physiol&amp;volume=50&amp;pages=51-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jinsphys.2003.09.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jinsphys.2003.09.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jinsphys.2003.09.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15037093" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Buttstedt</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Moritz</div> <div class="given-names">R. F. A.</div></div>, <div class="name string-name"><div class="surname">Erler</div> <div class="given-names">S.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Origin and function of the major royal jelly proteins of the honeybee (Apis mellifera) as members of the yellow gene family</div>. <div class="source">Biol. Rev</div> ., <div class="volume">89</div>, <div class="fpage">255</div>–<div class="lpage">69</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20and%20function%20of%20the%20major%20royal%20jelly%20proteins%20of%20the%20honeybee%20(Apis%20mellifera)%20as%20members%20of%20the%20yellow%20gene%20family&amp;author=A.%20Buttstedt&amp;author=R.%20F.%20A.%20Moritz&amp;author=S.%20Erler&amp;publication_year=2014&amp;journal=Biol.%20Rev&amp;volume=89&amp;pages=255-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/brv.2014.89.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fbrv.2014.89.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fbrv.2014.89.issue-2"> </span></div> </div></div></div></div></div><div content-id="dsw064-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hanes</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Šimúth</div> <div class="given-names">J.</div></div></div> <div class="year">1992</div>, 
<div class="article-title">Identification and partial characterization of the major royal jelly protein of the honey bee (Apis mellifera L.)</div>. <div class="source">J. Apic. Res</div> ., <div class="volume">31</div>, <div class="fpage">22</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20partial%20characterization%20of%20the%20major%20royal%20jelly%20protein%20of%20the%20honey%20bee%20(Apis%20mellifera%20L.)&amp;author=J.%20Hanes&amp;author=J.%20%C5%A0im%C3%BAth&amp;publication_year=1992&amp;journal=J.%20Apic.%20Res&amp;volume=31&amp;pages=22-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/00218839.1992.11101256" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F00218839.1992.11101256" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F00218839.1992.11101256"> </span></div> </div></div></div></div></div><div content-id="dsw064-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Comprehensive identification of novel proteins and N-glycosylation sites in royal jelly</div>. <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">135</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20identification%20of%20novel%20proteins%20and%20N-glycosylation%20sites%20in%20royal%20jelly&amp;author=L.%20Zhang&amp;author=B.%20Han&amp;author=R.%20Li&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=135" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-135" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-135" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-135"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24529077" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Snodgrass</div> <div class="given-names">R. E.</div></div></div> <div class="year">1925</div>, <div class="source">Anatomy and physiology of the honey bee</div> . 
<div class="publisher-name">McGraw-Hill Book Company</div>: 
<div class="publisher-loc">New York</div>.</div></div></div></div><div content-id="dsw064-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Buttstedt</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ihling</div> <div class="given-names">C. I.</div></div>, <div class="name string-name"><div class="surname">Pietzsch</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Moritz</div> <div class="given-names">R. F. A.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Royalactin is not a royal making of a queen</div>. <div class="source">Nature</div> , <div class="volume">537</div>, <div class="fpage">E10</div>–<div class="lpage">E12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Royalactin%20is%20not%20a%20royal%20making%20of%20a%20queen&amp;author=A.%20Buttstedt&amp;author=C.%20I.%20Ihling&amp;author=M.%20Pietzsch&amp;author=R.%20F.%20A.%20Moritz&amp;publication_year=2016&amp;journal=Nature&amp;volume=537&amp;pages=E10-E12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature19349" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature19349" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature19349"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27652566" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Buttstedt</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Moritz</div> <div class="given-names">R. F.A</div></div>, <div class="name string-name"><div class="surname">Erler</div> <div class="given-names">S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">More than royal food – major royal jelly protein genes in sexuals and workers of the honeybee Apis mellifera</div>. <div class="source">Front. Zool</div> ., <div class="volume">10</div>, <div class="fpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=More%20than%20royal%20food%20%E2%80%93%20major%20royal%20jelly%20protein%20genes%20in%20sexuals%20and%20workers%20of%20the%20honeybee%20Apis%20mellifera&amp;author=A.%20Buttstedt&amp;author=R.%20F.A%20Moritz&amp;author=S.%20Erler&amp;publication_year=2013&amp;journal=Front.%20Zool&amp;volume=10&amp;pages=72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1742-9994-10-72" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1742-9994-10-72" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1742-9994-10-72"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24279675" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Albert</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Schmitz</div> <div class="given-names">J.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Characterization of major royal jelly protein-like DNA sequences in <em>Apis dorsata</em></div>. <div class="source">J. Apic. Res</div> ., <div class="volume">41</div>, <div class="fpage">75</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20major%20royal%20jelly%20protein-like%20DNA%20sequences%20in%20Apis%20dorsata&amp;author=S.%20Albert&amp;author=J.%20Schmitz&amp;publication_year=2002&amp;journal=J.%20Apic.%20Res&amp;volume=41&amp;pages=75-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/00218839.2002.11101072" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F00218839.2002.11101072" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F00218839.2002.11101072"> </span></div> </div></div></div></div></div><div content-id="dsw064-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Albertová</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Su</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Brockmann</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Gadau</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Albert</div> <div class="given-names">S.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Organization and potential function of the mrjp3 locus in four honeybee species</div>. <div class="source">J. Agric. Food Chem</div> ., <div class="volume">53</div>, <div class="fpage">8075</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Organization%20and%20potential%20function%20of%20the%20mrjp3%20locus%20in%20four%20honeybee%20species&amp;author=V.%20Albertov%C3%A1&amp;author=S.%20Su&amp;author=A.%20Brockmann&amp;author=J.%20Gadau&amp;author=S.%20Albert&amp;publication_year=2005&amp;journal=J.%20Agric.%20Food%20Chem&amp;volume=53&amp;pages=8075-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/jf051417x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fjf051417x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fjf051417x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16190673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Imjongjirak</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Klinbunga</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sittipraneed</div> <div class="given-names">S.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Cloning, expression and genomic organization of genes encoding major royal jelly protein 1 and 2 of the honey bee (Apis cerana)</div>. <div class="source">J. Biochem. Mol. Biol</div> ., <div class="volume">38</div>, <div class="fpage">49</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%2C%20expression%20and%20genomic%20organization%20of%20genes%20encoding%20major%20royal%20jelly%20protein%201%20and%202%20of%20the%20honey%20bee%20(Apis%20cerana)&amp;author=C.%20Imjongjirak&amp;author=S.%20Klinbunga&amp;author=S.%20Sittipraneed&amp;publication_year=2005&amp;journal=J.%20Biochem.%20Mol.%20Biol&amp;volume=38&amp;pages=49-57" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15715946" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Su</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Albert</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhong</div> <div class="given-names">B.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Molecular cloning and analysis of four cDNAs from the heads of <em>Apis cerana cerana</em> nurse honeybees coding for major royal jelly proteins</div>. <div class="source">Apidologie</div> , <div class="volume">36</div>, <div class="fpage">389</div>–<div class="lpage">401</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%20and%20analysis%20of%20four%20cDNAs%20from%20the%20heads%20of%20Apis%20cerana%20cerana%20nurse%20honeybees%20coding%20for%20major%20royal%20jelly%20proteins&amp;author=S.%20Su&amp;author=S.%20Albert&amp;author=S.%20Chen&amp;author=B.%20Zhong&amp;publication_year=2005&amp;journal=Apidologie&amp;volume=36&amp;pages=389-401" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1051/apido:2005026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1051%2Fapido:2005026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1051%2Fapido:2005026"> </span></div> </div></div></div></div></div><div content-id="dsw064-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huddleston</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ranade</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Malig</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Reconstructing complex regions of genomes using long-read sequencing technology</div>. <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">688</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reconstructing%20complex%20regions%20of%20genomes%20using%20long-read%20sequencing%20technology&amp;author=J.%20Huddleston&amp;author=S.%20Ranade&amp;author=M.%20Malig&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=688-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.168450.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.168450.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.168450.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24418700" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chaisson</div> <div class="given-names">M. J. P.</div></div>, <div class="name string-name"><div class="surname">Huddleston</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Dennis</div> <div class="given-names">M. Y.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Resolving the complexity of the human genome using single-molecule sequencing</div>. <div class="source">Nature</div> , <div class="volume">517</div>, <div class="fpage">608</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resolving%20the%20complexity%20of%20the%20human%20genome%20using%20single-molecule%20sequencing&amp;author=M.%20J.%20P.%20Chaisson&amp;author=J.%20Huddleston&amp;author=M.%20Y.%20Dennis&amp;publication_year=2015&amp;journal=Nature&amp;volume=517&amp;pages=608-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature13907" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature13907" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature13907"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25383537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Karamitros</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Harrison</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Piorkowska</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Katzourakis</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Magiorkinis</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Mbisa</div> <div class="given-names">J. L.</div></div></div> <div class="year">2016</div>, De Novo Assembly of Human Herpes Virus Type 1 (HHV-1) genome, Mining of Non-Canonical Structures and Detection of Novel Drug-Resistance Mutations Using Short- and Long-Read Next Generation Sequencing Technologies. <div class="source">PLoS One</div> . <div class="volume">11</div>(<div class="issue">6</div>):<div class="fpage">e0157600</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0157600" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0157600" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0157600"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27309375" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Untergasser</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Cutcutache</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Koressaar</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Primer3—new capabilities and interfaces</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>(<div class="issue">15</div>):<div class="fpage">e11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primer3%E2%80%94new%20capabilities%20and%20interfaces&amp;author=A.%20Untergasser&amp;author=I.%20Cutcutache&amp;author=T.%20Koressaar&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=e11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks596" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks596" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks596"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22102584" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Laver</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Harrison</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">O’Neill</div> <div class="given-names">P. A.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Assessing the performance of the Oxford Nanopore Technologies MinION</div>. <div class="source">Biomol. Detect. Quantif</div> ., <div class="volume">3</div>, <div class="fpage">1</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessing%20the%20performance%20of%20the%20Oxford%20Nanopore%20Technologies%20MinION&amp;author=T.%20Laver&amp;author=J.%20Harrison&amp;author=P.%20A.%20O%E2%80%99Neill&amp;publication_year=2015&amp;journal=Biomol.%20Detect.%20Quantif&amp;volume=3&amp;pages=1-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bdq.2015.02.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bdq.2015.02.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bdq.2015.02.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26753127" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Madoui</div> <div class="given-names">M.-A.</div></div>, <div class="name string-name"><div class="surname">Engelen</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Cruaud</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Genome assembly using nanopore-guided long and error-free DNA reads</div>. <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">327</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20assembly%20using%20nanopore-guided%20long%20and%20error-free%20DNA%20reads&amp;author=M.-A.%20Madoui&amp;author=S.%20Engelen&amp;author=C.%20Cruaud&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=327" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1519-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1519-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1519-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25927464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Watson</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Thomson</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Risse</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Talbot</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Santoyo-Lopez</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Gharbi</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Blaxter</div> <div class="given-names">M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">poRe: an R package for the visualization and analysis of nanopore sequencing data</div>. <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">114</div>–<div class="lpage">115</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=poRe%3A%20an%20R%20package%20for%20the%20visualization%20and%20analysis%20of%20nanopore%20sequencing%20data&amp;author=M.%20Watson&amp;author=M.%20Thomson&amp;author=J.%20Risse&amp;author=R.%20Talbot&amp;author=J.%20Santoyo-Lopez&amp;author=K.%20Gharbi&amp;author=M.%20Blaxter&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=114-115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu590" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu590" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu590"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25173419" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Peterson</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Stecher</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">MEGA5: Molecular evolutionary genetics analysis using maximum likelihood, evolutionary distance, and maximum parsimony methods</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">28</div>, <div class="fpage">2731</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA5%3A%20Molecular%20evolutionary%20genetics%20analysis%20using%20maximum%20likelihood%2C%20evolutionary%20distance%2C%20and%20maximum%20parsimony%20methods&amp;author=K.%20Tamura&amp;author=D.%20Peterson&amp;author=N.%20Peterson&amp;author=G.%20Stecher&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol&amp;volume=28&amp;pages=2731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21546353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">May</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Abeln</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Buijs</div> <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Heringa</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Crielaard</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Brandt</div> <div class="given-names">B.W.</div></div></div> (<div class="year">2015</div>). 
<div class="article-title">NGS-eval: NGS Error analysis and novel sequence VAriant detection tooL</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>:<div class="fpage">W301</div>–<div class="lpage">W305</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NGS-eval%3A%20NGS%20Error%20analysis%20and%20novel%20sequence%20VAriant%20detection%20tooL&amp;author=A.%20May&amp;author=S.%20Abeln&amp;author=M.J.%20Buijs&amp;author=J.%20Heringa&amp;author=W.%20Crielaard&amp;author=B.W.%20Brandt&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=W301-W305" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv346" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv346" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv346"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25878034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Elsik</div> <div class="given-names">C. G.</div></div>, <div class="name string-name"><div class="surname">Worley</div> <div class="given-names">K. C.</div></div>, <div class="name string-name"><div class="surname">Bennett</div> <div class="given-names">A. K.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Finding the missing honey bee genes: lessons learned from a genome upgrade</div>. <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">1</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Finding%20the%20missing%20honey%20bee%20genes%3A%20lessons%20learned%20from%20a%20genome%20upgrade&amp;author=C.%20G.%20Elsik&amp;author=K.%20C.%20Worley&amp;author=A.%20K.%20Bennett&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=1-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-86" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-86" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-86"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24382143" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wallberg</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Wellhagen</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">A worldwide survey of genome sequence variation provides insight into the evolutionary history of the honeybee <em>Apis mellifera</em></div>. <div class="source">Nat. Genet</div> ., <div class="volume">46</div>, <div class="fpage">1081</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20worldwide%20survey%20of%20genome%20sequence%20variation%20provides%20insight%20into%20the%20evolutionary%20history%20of%20the%20honeybee%20Apis%20mellifera&amp;author=A.%20Wallberg&amp;author=F.%20Han&amp;author=G.%20Wellhagen&amp;publication_year=2014&amp;journal=Nat.%20Genet&amp;volume=46&amp;pages=1081-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3077" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3077" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3077"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25151355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kapheim</div> <div class="given-names">K. M.</div></div>, <div class="name string-name"><div class="surname">Pan</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Genomic signatures of evolutionary transitions from solitary to group living</div>. <div class="source">Science (80-.)</div> ., <div class="volume">348</div>, <div class="fpage">1139</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20signatures%20of%20evolutionary%20transitions%20from%20solitary%20to%20group%20living&amp;author=K.%20M.%20Kapheim&amp;author=H.%20Pan&amp;author=C.%20Li&amp;publication_year=2015&amp;journal=Science%20(80-.)&amp;volume=348&amp;pages=1139-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.aaa4788" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aaa4788" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.aaa4788"> </span></div> </div></div></div></div></div><div content-id="dsw064-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sadd</div> <div class="given-names">B. M.</div></div>, <div class="name string-name"><div class="surname">Barribeau</div> <div class="given-names">S. M.</div></div>, <div class="name string-name"><div class="surname">Bloch</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">The genomes of two key bumblebee species with primitive eusocial organization</div>. <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genomes%20of%20two%20key%20bumblebee%20species%20with%20primitive%20eusocial%20organization&amp;author=B.%20M.%20Sadd&amp;author=S.%20M.%20Barribeau&amp;author=G.%20Bloch&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0623-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0623-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0623-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25908251" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mao</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Schuler</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Berenbaum</div> <div class="given-names">M. R.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">A dietary phytochemical alters caste-associated gene expression in honey bees</div>. <div class="source">Sci. Adv</div> ., <div class="volume">1</div>, <div class="fpage">e1500795</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20dietary%20phytochemical%20alters%20caste-associated%20gene%20expression%20in%20honey%20bees&amp;author=W.%20Mao&amp;author=M.%20A.%20Schuler&amp;author=M.%20R.%20Berenbaum&amp;publication_year=2015&amp;journal=Sci.%20Adv&amp;volume=1&amp;pages=e1500795" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/sciadv.1500795" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fsciadv.1500795" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fsciadv.1500795"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26601244" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vojvodic</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Johnson</div> <div class="given-names">B. R.</div></div>, <div class="name string-name"><div class="surname">Harpur</div> <div class="given-names">B. A.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">The transcriptomic and evolutionary signature of social interactions regulating honey bee caste development</div>. <div class="source">Ecol. Evol</div> ., <div class="volume">5</div>, <div class="fpage">4795</div>–<div class="lpage">807</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcriptomic%20and%20evolutionary%20signature%20of%20social%20interactions%20regulating%20honey%20bee%20caste%20development&amp;author=S.%20Vojvodic&amp;author=B.%20R.%20Johnson&amp;author=B.%20A.%20Harpur&amp;publication_year=2015&amp;journal=Ecol.%20Evol&amp;volume=5&amp;pages=4795-807" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/ece3.1720" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fece3.1720" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fece3.1720"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26640660" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Harpur</div> <div class="given-names">B.A.</div></div>, <div class="name string-name"><div class="surname">Kent</div> <div class="given-names">C.F.</div></div>, <div class="name string-name"><div class="surname">Molodtsova</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Lebon</div> <div class="given-names">J.M.D.</div></div>, <div class="name string-name"><div class="surname">Alqarni</div> <div class="given-names">A.S.</div></div>, <div class="name string-name"><div class="surname">Owayss</div> <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Zayed</div> <div class="given-names">A.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Population genomics of the honey bee reveals strong signatures of positive selection on worker traits</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> ., <div class="volume">111</div>(<div class="issue">7</div>), <div class="fpage">2614</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population%20genomics%20of%20the%20honey%20bee%20reveals%20strong%20signatures%20of%20positive%20selection%20on%20worker%20traits&amp;author=B.A.%20Harpur&amp;author=C.F.%20Kent&amp;author=D.%20Molodtsova&amp;author=J.M.D.%20Lebon&amp;author=A.S.%20Alqarni&amp;author=A.A.%20Owayss&amp;author=A.%20Zayed&amp;publication_year=2014&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=111&amp;pages=2614-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1315506111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1315506111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1315506111"> </span></div> </div></div></div></div></div><div content-id="dsw064-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wallberg</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Han</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Wellhagen</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Dahle</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Kawata</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Haddad</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Paulino Simoes</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Allsopp</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kandemir</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">De la Rua</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Pirk</div> <div class="given-names">C.W.</div></div>, <div class="name string-name"><div class="surname">Webster</div> <div class="given-names">M.T.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">A worldwide survey of genome sequence variation provides insight into the evolutionary history of the honeybee <em>Apis mellifera</em></div>. <div class="source">Nature Genetics</div> , <div class="volume">46</div>, <div class="fpage">1081</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20worldwide%20survey%20of%20genome%20sequence%20variation%20provides%20insight%20into%20the%20evolutionary%20history%20of%20the%20honeybee%20Apis%20mellifera&amp;author=A.%20Wallberg&amp;author=F.%20Han&amp;author=G.%20Wellhagen&amp;author=B.%20Dahle&amp;author=M.%20Kawata&amp;author=N.%20Haddad&amp;author=Z.%20Paulino%20Simoes&amp;author=M.%20Allsopp&amp;author=I.%20Kandemir&amp;author=P.%20De%20la%20Rua&amp;author=C.W.%20Pirk&amp;author=M.T.%20Webster&amp;publication_year=2014&amp;journal=Nature%20Genetics&amp;volume=46&amp;pages=1081-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3077" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3077" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3077"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25151355" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lo</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Gloag</div> <div class="given-names">R. S.</div></div>, <div class="name string-name"><div class="surname">Anderson</div> <div class="given-names">D. L.</div></div>, <div class="name string-name"><div class="surname">Oldroyd</div> <div class="given-names">B. P.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">A molecular phylogeny of the genus Apis suggests that the Giant Honey Bee of the Philippines, <em>A. breviligula</em> Maa, and the Plains Honey Bee of southern India, <em>A. indica</em> Fabricius, are valid species</div>. <div class="source">Syst. Entomol</div> ., <div class="volume">35</div>, <div class="fpage">226</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20molecular%20phylogeny%20of%20the%20genus%20Apis%20suggests%20that%20the%20Giant%20Honey%20Bee%20of%20the%20Philippines%2C%20A.%20breviligula%20Maa%2C%20and%20the%20Plains%20Honey%20Bee%20of%20southern%20India%2C%20A.%20indica%20Fabricius%2C%20are%20valid%20species&amp;author=N.%20Lo&amp;author=R.%20S.%20Gloag&amp;author=D.%20L.%20Anderson&amp;author=B.%20P.%20Oldroyd&amp;publication_year=2010&amp;journal=Syst.%20Entomol&amp;volume=35&amp;pages=226-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-3113.2009.00504.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-3113.2009.00504.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-3113.2009.00504.x"> </span></div> </div></div></div></div></div><div content-id="dsw064-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohno</div> <div class="given-names">S.</div></div></div> <div class="year">1970</div>, <div class="source">Evolution by gene duplication</div> . 
<div class="publisher-name">Springer-Verlag</div>, 
<div class="publisher-loc">Heidelberg, Germany</div>.</div></div></div></div><div content-id="dsw064-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lynch</div> <div class="given-names">M.</div></div></div> <div class="year">2007</div>, <div class="source">The origins of genome architecture</div> . 
<div class="publisher-name">Sinauer Associates, Inc. Publishers</div>. 
<div class="publisher-loc">Sunderland, MA</div>.</div></div></div></div><div content-id="dsw064-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Innan</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kondrashov</div> <div class="given-names">F.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">The evolution of gene duplications: classifying and distinguishing between models</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">11</div>, <div class="fpage">97</div>–<div class="lpage">108</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolution%20of%20gene%20duplications%3A%20classifying%20and%20distinguishing%20between%20models&amp;author=H.%20Innan&amp;author=F.%20Kondrashov&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet&amp;volume=11&amp;pages=97-108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2689" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2689" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2689"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20051986" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw064-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hojo</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kagami</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Sasaki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Nakamura</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Sasaki</div> <div class="given-names">M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Reduced expression of major royal jelly protein 1 gene in the mushroom bodies of worker honeybees with reduced learning ability</div>. <div class="source">Apidologie</div> , <div class="volume">41</div>, <div class="fpage">194</div>–<div class="lpage">202</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reduced%20expression%20of%20major%20royal%20jelly%20protein%201%20gene%20in%20the%20mushroom%20bodies%20of%20worker%20honeybees%20with%20reduced%20learning%20ability&amp;author=M.%20Hojo&amp;author=T.%20Kagami&amp;author=T.%20Sasaki&amp;author=J.%20Nakamura&amp;author=M.%20Sasaki&amp;publication_year=2010&amp;journal=Apidologie&amp;volume=41&amp;pages=194-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1051/apido/2009075" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1051%2Fapido%2F2009075" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1051%2Fapido%2F2009075"> </span></div> </div></div></div></div></div><div content-id="dsw064-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Linksvayer</div> <div class="given-names">T. A.</div></div>, <div class="name string-name"><div class="surname">Wade</div> <div class="given-names">M. J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Genes with social effects are expected to harbor more sequence variation within and between species</div>. <div class="source">Evolution (N. Y)</div> ., <div class="volume">63</div>, <div class="fpage">1685</div>–<div class="lpage">96</div>.</div></div></div></div><div content-id="dsw064-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw064-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Woodard</div> <div class="given-names">S.H.</div></div>, <div class="name string-name"><div class="surname">Fischman</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Venkat</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hudson</div> <div class="given-names">M.E.</div></div>, <div class="name string-name"><div class="surname">Varala</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Cameron</div> <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Clark</div> <div class="given-names">A.G.</div></div>, <div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">G.E.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Genes involved in convergent evolution of eusociality in bees</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> ., <div class="volume">108</div>, <div class="fpage">7472</div>–<div class="lpage">7477</div>. 10.1073/pnas.1103457108.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20involved%20in%20convergent%20evolution%20of%20eusociality%20in%20bees&amp;author=S.H.%20Woodard&amp;author=B.J.%20Fischman&amp;author=A.%20Venkat&amp;author=M.E.%20Hudson&amp;author=K.%20Varala&amp;author=S.A.%20Cameron&amp;author=A.G.%20Clark&amp;author=G.E.%20Robinson&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=108&amp;pages=7472-7477" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1103457108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1103457108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1103457108"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2972857&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/3/10.1093_dnares_dsw064/1/dsw064_Supp.zip?Expires=2147483647&amp;Signature=jK8ewsS3EVehsrKADusY3RpwLrebGQjL~TIHArA0hikA4q~ydkpLAluT4w91nh96dY7b9cpqJhKDLJrDOGII~Llo7nWNNmcWbRKP0VTRIAmXZOzmayghgZiaB5SHJjGXH6~i06R2Eaghrqm1Go0WLQa0n8oZm8QES9cXReWhglsNTg5zF17DlqAnt-fKvxuj1uxeAzFOKx6NjB5jtS7r7MujmrswYYGU31QSOKNV0AIl-5P~BLEn2sTMP-BuBaAzwkpjPWB3gDMBwJbu5u~48ds7fi-N~j5FLDg9FxzL1TloSMIJZN7n65dxycvJNzxe7NZeq25UkZ4xufdxSR1QKg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">792</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000404439600006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw064"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2972857"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000404439600006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f3%2f279%2f2972857">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000404439600006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F3%2F279%2F2972857" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2972857'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Comparative analyses of the major royal jelly protein gene cluster in three Apis species with long amplicon sequencing" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw064");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f3%2f279%2f2972857">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

