

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/2/205/2738679","@type":"ScholarlyArticle","name":"Alternative splicing in tomato pollen in response to heat stress  †  ","datePublished":"2016-12-24","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2017-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw051","keywords":["tomato pollen","RNA-seq","alternative splicing","heat stress","transcriptome","assembly"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/2/205/2738679","author":[{"name":"Keller, Mario","affiliation":"1 Department of Biosciences, Molecular Cell Biology of Plants","@type":"Person"},{"name":"Hu, Yangjie","affiliation":"1 Department of Biosciences, Molecular Cell Biology of Plants","@type":"Person"},{"name":"Mesihovic, Anida","affiliation":"1 Department of Biosciences, Molecular Cell Biology of Plants","@type":"Person"},{"name":"Fragkostefanakis, Sotirios","affiliation":"1 Department of Biosciences, Molecular Cell Biology of Plants","@type":"Person"},{"name":"Schleiff, Enrico","affiliation":"1 Department of Biosciences, Molecular Cell Biology of Plants   2 Cluster of Excellence Frankfurt   3 Buchmann Institute for Molecular Life Sciences (BMLS), Goethe University, D-60438 Frankfurt am Main, Germany","@type":"Person"},{"name":"Simm, Stefan","affiliation":"1 Department of Biosciences, Molecular Cell Biology of Plants   2 Cluster of Excellence Frankfurt","@type":"Person"}],"description":"Abstract.  Alternative splicing (AS) is a key control mechanism influencing signal response cascades in different developmental stages and under stress conditio","pageStart":"205","pageEnd":"217","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Alternative splicing in tomato pollen in response to heat stress  †  ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Alternative splicing in tomato pollen in response to heat stress  †  " />
    <meta property="og:description" content="Abstract.  Alternative splicing (AS) is a key control mechanism influencing signal response cascades in different developmental stages and under stress conditio" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/2/205/2738679" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-12-24" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Keller, Mario" /><meta name="citation_author_institution" content="Department of Biosciences, Molecular Cell Biology of Plants" /><meta name="citation_author" content="Hu, Yangjie" /><meta name="citation_author_institution" content="Department of Biosciences, Molecular Cell Biology of Plants" /><meta name="citation_author" content="Mesihovic, Anida" /><meta name="citation_author_institution" content="Department of Biosciences, Molecular Cell Biology of Plants" /><meta name="citation_author" content="Fragkostefanakis, Sotirios" /><meta name="citation_author_institution" content="Department of Biosciences, Molecular Cell Biology of Plants" /><meta name="citation_author" content="Schleiff, Enrico" /><meta name="citation_author_institution" content="Department of Biosciences, Molecular Cell Biology of Plants" /><meta name="citation_author_institution" content="Cluster of Excellence Frankfurt" /><meta name="citation_author_institution" content="Buchmann Institute for Molecular Life Sciences (BMLS), Goethe University, D-60438 Frankfurt am Main, Germany" /><meta name="citation_author" content="Simm, Stefan" /><meta name="citation_author_institution" content="Department of Biosciences, Molecular Cell Biology of Plants" /><meta name="citation_author_institution" content="Cluster of Excellence Frankfurt" /><meta name="citation_title" content="Alternative splicing in tomato pollen in response to heat stress" /><meta name="citation_firstpage" content="205" /><meta name="citation_lastpage" content="217" /><meta name="citation_doi" content="10.1093/dnares/dsw051" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2017/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Meta-analysis of heat- and chemically upregulated chaperone genes in plant and human cells;  citation_journal_title=Cell Stress Chaperones;  citation_year=2011;  citation_volume=16;  citation_pages=15-31; " /><meta name="citation_reference" content="citation_title=Chaperone network composition in Solanum lycopersicum explored by transcriptome profiling and microarray meta-analysis;  citation_journal_title=Plant Cell Environ;  citation_year=2015;  citation_volume=38;  citation_pages=693-709; " /><meta name="citation_reference" content="citation_title=Genome-wide identification and analysis of early heat stress responsive genes in rice;  citation_journal_title=J. Plant Biol;  citation_year=2012;  citation_volume=55;  citation_pages=10" /><meta name="citation_reference" content="citation_title=Heat stress phenotypes of Arabidopsis mutants implicate multiple signaling pathways in the acquisition of thermotolerance;  citation_journal_title=Plant Physiol;  citation_year=2005;  citation_volume=138;  citation_pages=882-897; " /><meta name="citation_reference" content="citation_title=Transcriptome analysis of heat stress response in switchgrass (Panicum virgatum L.);  citation_journal_title=BMC Plant Biol;  citation_year=2013;  citation_volume=13;  citation_pages=153" /><meta name="citation_reference" content="citation_title=Coexpression network analysis associated with call of rice seedlings for encountering heat stress;  citation_journal_title=Plant Mol. Biol;  citation_year=2014;  citation_volume=84;  citation_pages=125-143; " /><meta name="citation_reference" content="citation_title=NnHSP17.5, a cytosolic class II small heat shock protein gene from Nelumbo nucifera, contributes to seed germination vigor and seedling thermotolerance in transgenic Arabidopsis;  citation_journal_title=Plant Cell Rep;  citation_year=2012;  citation_volume=31;  citation_pages=379-89; " /><meta name="citation_reference" content="citation_title=Physiological and molecular assessment of altered expression of Hsc70-1 in Arabidopsis. Evidence for pleiotropic consequences;  citation_journal_title=Plant Physiol;  citation_year=2003;  citation_volume=132;  citation_pages=979-87; " /><meta name="citation_reference" content="citation_title=Transgenic expression of the Trichoderma harzianum hsp70 gene increases Arabidopsis resistance to heat and other abiotic stresses;  citation_journal_title=J. Plant Physiol;  citation_year=2010;  citation_volume=167;  citation_pages=659-65; " /><meta name="citation_reference" content="citation_title=Altered expression of cytosolic/nuclear HSC70-1 molecular chaperone affects development and abiotic stress tolerance in Arabidopsis thaliana;  citation_journal_title=J. Exp. Bot;  citation_year=2009;  citation_volume=60;  citation_pages=2653-64; " /><meta name="citation_reference" content="citation_title=The plant heat stress transcription factor (Hsf) family: structure, function and evolution;  citation_journal_title=Biochim. Biophys. Acta;  citation_year=2012;  citation_volume=1819;  citation_pages=104-19; " /><meta name="citation_reference" content="citation_title=Post-transcriptional and post-translational regulations of drought and heat response in plants: a spider’s web of mechanisms;  citation_journal_title=Front. Plant Science;  citation_year=2015;  citation_volume=6; " /><meta name="citation_reference" content="citation_title=Alternative splicing: a pivotal step between eukaryotic transcription and translation;  citation_journal_title=Nat. Rev. Mol. Cell Biol;  citation_year=2013;  citation_volume=14;  citation_pages=153-65; " /><meta name="citation_reference" content="citation_title=Pre-mRNA processing reaches back to transcription and ahead to translation;  citation_journal_title=Cell;  citation_year=2009;  citation_volume=136;  citation_pages=688-700; " /><meta name="citation_reference" content="citation_title=Alternative splicing at the intersection of biological timing, development, and stress responses;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=3640-56; " /><meta name="citation_reference" content="citation_title=Alternative splicing of pre-messenger RNAs in plants in the genomic era;  citation_journal_title=Annu. Rev. Plant Biol;  citation_year=2007;  citation_volume=58;  citation_pages=267-94; " /><meta name="citation_reference" content="citation_title=Splicing regulation: from a parts list of regulatory elements to an integrated splicing code;  citation_journal_title=RNA;  citation_year=2008;  citation_volume=14;  citation_pages=802-13; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of heat-sensitive alternative splicing in Physcomitrella patens;  citation_journal_title=Plant Physiol;  citation_year=2014;  citation_volume=165;  citation_pages=826-40; " /><meta name="citation_reference" content="citation_title=Identification of alternative splicing events by RNA sequencing in early growth tomato fruits;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=948" /><meta name="citation_reference" content="citation_title=Comparisons of computational methods for differential alternative splicing detection using RNA-seq in plant systems;  citation_journal_title=BMC Bioinform;  citation_year=2014;  citation_volume=15; " /><meta name="citation_reference" content="citation_title=Alternative splicing in plants–coming of age;  citation_journal_title=Trends Plant Sci;  citation_year=2012;  citation_volume=17;  citation_pages=616-23; " /><meta name="citation_reference" content="citation_title=Alternative splicing and nonsense-mediated decay modulate expression of important regulatory genes in Arabidopsis;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=2454-69; " /><meta name="citation_reference" content="citation_title=Alternative splicing: enhancing ability to cope with stress via transcriptome plasticity;  citation_journal_title=Plant Sci;  citation_year=2012;  citation_volume=185-186;  citation_pages=40-49; " /><meta name="citation_reference" content="citation_title=Complexity of the alternative splicing landscape in plants;  citation_journal_title=Plant Cell;  citation_year=2013;  citation_volume=25;  citation_pages=3657-83; " /><meta name="citation_reference" content="citation_title=Genome-wide mapping of alternative splicing in Arabidopsis thaliana;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=45-58; " /><meta name="citation_reference" content="citation_title=Intron retention is a major phenomenon in alternative splicing in Arabidopsis;  citation_journal_title=Plant J;  citation_year=2004;  citation_volume=39;  citation_pages=877-85; " /><meta name="citation_reference" content="citation_title=Genome-wide cataloging and analysis of alternatively spliced genes in cereal crops;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=721" /><meta name="citation_reference" content="citation_title=RNA-seq of arabidopsis pollen uncovers novel transcription and alternative splicing;  citation_journal_title=Plant Physiol;  citation_year=2013;  citation_volume=162;  citation_pages=1092-109; " /><meta name="citation_reference" content="citation_title=Proteomic analyses of Oryza sativa mature pollen reveal novel proteins associated with pollen germination and tube growth;  citation_journal_title=Proteomics;  citation_year=2006;  citation_volume=6;  citation_pages=2504-29; " /><meta name="citation_reference" content="citation_title=Perspectives on deciphering mechanisms underlying plant heat stress response and thermotolerance; citation_author=Solanaceae Pollen Thermotolerance Initial Training Network, C;  citation_journal_title=Front. Plant Sci;  citation_year=2013;  citation_volume=4;  citation_pages=315" /><meta name="citation_reference" content="citation_title=Ensuring reproduction at high temperatures: the heat stress response during anther and pollen development;  citation_journal_title=Plants;  citation_year=2013;  citation_volume=2;  citation_pages=17" /><meta name="citation_reference" content="citation_title=The effect of heat stress on tomato pollen characteristics is associated with changes in carbohydrate concentration in the developing anthers;  citation_journal_title=Ann. Bot;  citation_year=2002;  citation_volume=90;  citation_pages=631-6; " /><meta name="citation_reference" content="citation_title=A decade of pollen transcriptomics;  citation_journal_title=Plant Reprod;  citation_year=2015;  citation_volume=28;  citation_pages=73-89; " /><meta name="citation_reference" content="citation_title=HsfA2 controls the activity of developmentally and stress-regulated heat stress protection mechanisms in tomato male reproductive tissues;  citation_journal_title=Plant Physiol;  citation_year=2016; " /><meta name="citation_reference" content="citation_title=The membrane proteome of male gametophyte in Solanum lycopersicum;  citation_journal_title=J. Proteomics;  citation_year=2016;  citation_volume=131;  citation_pages=48-60; " /><meta name="citation_reference" content="citation_title=The Sol Genomics Network (solgenomics.net): growing tomatoes using Perl;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=D1149-55; " /><meta name="citation_reference" content="citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions;  citation_journal_title=Genome Biol;  citation_year=2013;  citation_volume=14;  citation_pages=R36" /><meta name="citation_reference" content="citation_title=The tomato genome sequence provides insights into fleshy fruit evolution;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=485;  citation_pages=635-41; " /><meta name="citation_reference" content="citation_title=HTSeq–a Python framework to work with high-throughput sequencing data;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=166-9; " /><meta name="citation_reference" content="citation_title=Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method;  citation_journal_title=Methods;  citation_year=2001;  citation_volume=25;  citation_pages=402-8; " /><meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology. The Gene Ontology Consortium;  citation_journal_title=Nat. Genet;  citation_year=2000;  citation_volume=25;  citation_pages=25-9; " /><meta name="citation_reference" content="citation_title=InterProScan 5: genome-scale protein function classification;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=1236-40; " /><meta name="citation_reference" content="citation_title=Cytoscape: a software environment for integrated models of biomolecular interaction networks;  citation_journal_title=Genome Res;  citation_year=2003;  citation_volume=13;  citation_pages=2498-504; " /><meta name="citation_reference" content="citation_title=BiNGO: a Cytoscape plugin to assess overrepresentation of gene ontology categories in biological networks;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3448-9; " /><meta name="citation_reference" content="citation_title=Controlling the false discovery rate - a practical and powerful approach to multiple testing;  citation_journal_title=J. R. Stat. Soc. B. Methods;  citation_year=1995;  citation_volume=57;  citation_pages=289-300; " /><meta name="citation_reference" content="citation_title=Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks;  citation_journal_title=Nat. Protoc;  citation_year=2012;  citation_volume=7;  citation_pages=562-78; " /><meta name="citation_reference" content="citation_title=Pfam: the protein families database;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=D222-30; " /><meta name="citation_reference" content="citation_title=Hidden Markov model speed heuristic and iterative HMM search procedure;  citation_journal_title=BMC Bioinformatics;  citation_year=2010;  citation_volume=11;  citation_pages=431" /><meta name="citation_reference" content="citation_title=Gene Expression Omnibus: NCBI gene expression and hybridization array data repository;  citation_journal_title=Nucleic Acids Res;  citation_year=2002;  citation_volume=30;  citation_pages=207-10; " /><meta name="citation_reference" content="citation_title=Landscape of the lipidome and transcriptome under heat stress in Arabidopsis thaliana;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=10533" /><meta name="citation_reference" content="citation_title=Genome-wide analysis of alternative splicing in Zea mays: landscape and genetic regulation;  citation_journal_title=Plant Cell;  citation_year=2014;  citation_volume=26;  citation_pages=3472-87; " /><meta name="citation_reference" content="citation_title=A comparison of the low temperature transcriptomes of two tomato genotypes that differ in freezing tolerance: Solanum lycopersicum and Solanum habrochaites;  citation_journal_title=BMC Plant Biol;  citation_year=2015;  citation_volume=15;  citation_pages=132" /><meta name="citation_reference" content="citation_title=Widespread intron retention in mammals functionally tunes transcriptomes;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1774-86; " /><meta name="citation_reference" content="citation_title=Stressed out! Effects of environmental stress on mRNA metabolism;  citation_journal_title=FEMS Yeast Res;  citation_year=2006;  citation_volume=6;  citation_pages=160-70; " /><meta name="citation_reference" content="citation_title=A splicing factor that is inactivated during invivo heat-shock is functionally equivalent to the [U4/U6.U5] Triple Snrnp-Specific Proteins;  citation_journal_title=Genes Dev;  citation_year=1992;  citation_volume=6;  citation_pages=631-41; " /><meta name="citation_reference" content="citation_title=An alternatively spliced heat shock transcription factor, OsHSFA2dI, functions in the heat stress-induced unfolded protein response in rice;  citation_journal_title=Plant Biol;  citation_year=2015;  citation_volume=17;  citation_pages=419-29; " /><meta name="citation_reference" content="citation_title=An autoregulatory loop controlling Arabidopsis HsfA2 expression: role of heat shock-induced alternative splicing;  citation_journal_title=Plant Physiol;  citation_year=2013;  citation_volume=162;  citation_pages=512-521; " /><meta name="citation_reference" content="citation_title=The Cytosolic protein response as a subcomponent of the wider heat shock response in Arabidopsis;  citation_journal_title=Plant Cell;  citation_year=2009;  citation_volume=21;  citation_pages=642-54; " /><meta name="citation_reference" content="citation_title=Molecular and functional characterization of HSC54, a novel variant of human heat-shock cognate protein 70;  citation_journal_title=Mol. Pharmacol;  citation_year=2000;  citation_volume=58;  citation_pages=1257-263; " /><meta name="citation_reference" content="citation_title=Identification of a novel splice variant of heat shock cognate protein 70 after chronic antidepressant treatment in rat frontal cortex;  citation_journal_title=Biochem. Biophys. Res. Commun;  citation_year=1999;  citation_volume=261;  citation_pages=541-5; " /><meta name="citation_reference" content="citation_title=Alternative 5’ splice site selection induced by heat shock;  citation_journal_title=Mol. Cell. Biol;  citation_year=1994;  citation_volume=14;  citation_pages=567-75; " /><meta name="citation_reference" content="citation_title=Widespread intron retention diversifies most cancer transcriptomes;  citation_journal_title=Genome Med;  citation_year=2015;  citation_volume=7;  citation_pages=45" /><meta name="citation_reference" content="citation_title=Noisy splicing drives mRNA isoform diversity in human cells;  citation_journal_title=PLoS Genet;  citation_year=2010;  citation_volume=6;  citation_pages=e1001236" /><meta name="citation_reference" content="citation_title=Error prevention and mitigation as forces in the evolution of genes and genomes;  citation_journal_title=Nat. Rev. Genet;  citation_year=2011;  citation_volume=12;  citation_pages=875-81; " /><meta name="citation_reference" content="citation_publisher=Springer, Berlin; New York;  citation_title=Plant Proteomics;  citation_year=2007; " /><meta name="citation_reference" content="citation_title=Heat-stress-dependency and developmental modulation of gene expression: the potential of house-keeping genes as internal standards in mRNA expression profiling using real-time RT-PCR;  citation_journal_title=J. Exp. Bot;  citation_year=2003;  citation_volume=54;  citation_pages=2343-9; " /><meta name="citation_reference" content="citation_title=Regulation of protein function by ‘microProteins’;  citation_journal_title=EMBO Rep;  citation_year=2011;  citation_volume=12;  citation_pages=35-42; " /><meta name="citation_reference" content="citation_title=Competitive inhibition of transcription factors by small interfering peptides;  citation_journal_title=Trends Plant Sci;  citation_year=2011;  citation_volume=16;  citation_pages=541-9; " /><meta name="citation_reference" content="citation_title=Domain atrophy creates rare cases of functional partial protein domains;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=88" /><meta name="citation_reference" content="citation_title=Transcriptional profiling of Arabidopsis heat shock proteins and transcription factors reveals extensive overlap between heat and non-heat stress response pathways;  citation_journal_title=BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=125" /><meta name="citation_reference" content="citation_title=Temperature stress differentially modulates transcription in meiotic anthers of heat-tolerant and heat-sensitive tomato plants;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=384" /><meta name="citation_reference" content="citation_title=The cullin protein family;  citation_journal_title=Genome Biol;  citation_year=2011;  citation_volume=12;  citation_pages=220" /><meta name="citation_reference" content="citation_title=Nedd8 on cullin: building an expressway to protein destruction;  citation_journal_title=Oncogene;  citation_year=2004;  citation_volume=23;  citation_pages=1985-97; " /><meta name="citation_reference" content="citation_title=Cullin-RING ubiquitin ligase family in plant abiotic stress pathways(F);  citation_journal_title=J. Integr. Plant Biol;  citation_year=2013;  citation_volume=55;  citation_pages=21-30; " /><meta name="citation_reference" content="citation_title=A dominant-negative form of the E3 ubiquitin ligase Cullin-1 disrupts the correct allocation of cell fate in the neural crest lineage;  citation_journal_title=Development;  citation_year=2006;  citation_volume=133;  citation_pages=559-68; " /><meta name="citation_reference" content="citation_title=Multifaceted roles of Arabidopsis PP6 phosphatase in regulating cellular signaling and plant development;  citation_journal_title=Plant Signal. Behav;  citation_year=2013;  citation_volume=8;  citation_pages=e22508" /><meta name="citation_reference" content="citation_title=Serine/threonine protein phosphatase 5 (PP5) interacts with substrate under heat stress conditions and forms protein complex in Arabidopsis;  citation_journal_title=Plant Signal. Behav;  citation_year=2012;  citation_volume=7;  citation_pages=535-8; " /><meta name="citation_reference" content="citation_title=Mapping of protein phosphatase-6 association with its SAPS domain regulatory subunit using a model of helical repeats;  citation_journal_title=BMC Biochem;  citation_year=2009;  citation_volume=10;  citation_pages=24" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/2/205/13688871/dsw051.pdf" /><meta name="description" content="Abstract.  Alternative splicing (AS) is a key control mechanism influencing signal response cascades in different developmental stages and under stress conditio" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/2/205/2738679" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/2/205/2738679" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/2/205/2738679");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw051';
    </script>
<title>
	Alternative splicing in tomato pollen in response to heat stress† | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2738679" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfKwATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2VsbGVyLCBNYXJpbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvc2NpZW5jZXMsIE1vbGVjdWxhciBDZWxsIEJpb2xvZ3kgb2YgUGxhbnRzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSHUsIFlhbmdqaWUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3NjaWVuY2VzLCBNb2xlY3VsYXIgQ2VsbCBCaW9sb2d5IG9mIFBsYW50cyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1lc2lob3ZpYywgQW5pZGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb3NjaWVuY2VzLCBNb2xlY3VsYXIgQ2VsbCBCaW9sb2d5IG9mIFBsYW50cyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZyYWdrb3N0ZWZhbmFraXMsIFNvdGlyaW9zIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlcywgTW9sZWN1bGFyIENlbGwgQmlvbG9neSBvZiBQbGFudHMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTY2hsZWlmZiwgRW5yaWNvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlcywgTW9sZWN1bGFyIENlbGwgQmlvbG9neSBvZiBQbGFudHMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDbHVzdGVyIG9mIEV4Y2VsbGVuY2UgRnJhbmtmdXJ0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQnVjaG1hbm4gSW5zdGl0dXRlIGZvciBNb2xlY3VsYXIgTGlmZSBTY2llbmNlcyAoQk1MUyksIEdvZXRoZSBVbml2ZXJzaXR5LCBELTYwNDM4IEZyYW5rZnVydCBhbSBNYWluLCBHZXJtYW55IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2ltbSwgU3RlZmFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9zY2llbmNlcywgTW9sZWN1bGFyIENlbGwgQmlvbG9neSBvZiBQbGFudHMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDbHVzdGVyIG9mIEV4Y2VsbGVuY2UgRnJhbmtmdXJ0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJBbHRlcm5hdGl2ZSBzcGxpY2luZyBpbiB0b21hdG8gcG9sbGVuIGluIHJlc3BvbnNlIHRvIGhlYXQgc3RyZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjA1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyMTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3cwNTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNy8wNC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ldGEtYW5hbHlzaXMgb2YgaGVhdC0gYW5kIGNoZW1pY2FsbHkgdXByZWd1bGF0ZWQgY2hhcGVyb25lIGdlbmVzIGluIHBsYW50IGFuZCBodW1hbiBjZWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbCBTdHJlc3MgQ2hhcGVyb25lczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTE1LTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFwZXJvbmUgbmV0d29yayBjb21wb3NpdGlvbiBpbiBTb2xhbnVtIGx5Y29wZXJzaWN1bSBleHBsb3JlZCBieSB0cmFuc2NyaXB0b21lIHByb2ZpbGluZyBhbmQgbWljcm9hcnJheSBtZXRhLWFuYWx5c2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBDZWxsIEVudmlyb247ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz02OTMtNzA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBpZGVudGlmaWNhdGlvbiBhbmQgYW5hbHlzaXMgb2YgZWFybHkgaGVhdCBzdHJlc3MgcmVzcG9uc2l2ZSBnZW5lcyBpbiByaWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTU1OyAgY2l0YXRpb25fcGFnZXM9MTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IZWF0IHN0cmVzcyBwaGVub3R5cGVzIG9mIEFyYWJpZG9wc2lzIG11dGFudHMgaW1wbGljYXRlIG11bHRpcGxlIHNpZ25hbGluZyBwYXRod2F5cyBpbiB0aGUgYWNxdWlzaXRpb24gb2YgdGhlcm1vdG9sZXJhbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEzODsgIGNpdGF0aW9uX3BhZ2VzPTg4Mi04OTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgYW5hbHlzaXMgb2YgaGVhdCBzdHJlc3MgcmVzcG9uc2UgaW4gc3dpdGNoZ3Jhc3MgKFBhbmljdW0gdmlyZ2F0dW0gTC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgUGxhbnQgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTE1MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZXhwcmVzc2lvbiBuZXR3b3JrIGFuYWx5c2lzIGFzc29jaWF0ZWQgd2l0aCBjYWxsIG9mIHJpY2Ugc2VlZGxpbmdzIGZvciBlbmNvdW50ZXJpbmcgaGVhdCBzdHJlc3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT04NDsgIGNpdGF0aW9uX3BhZ2VzPTEyNS0xNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5uSFNQMTcuNSwgYSBjeXRvc29saWMgY2xhc3MgSUkgc21hbGwgaGVhdCBzaG9jayBwcm90ZWluIGdlbmUgZnJvbSBOZWx1bWJvIG51Y2lmZXJhLCBjb250cmlidXRlcyB0byBzZWVkIGdlcm1pbmF0aW9uIHZpZ29yIGFuZCBzZWVkbGluZyB0aGVybW90b2xlcmFuY2UgaW4gdHJhbnNnZW5pYyBBcmFiaWRvcHNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbCBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zNzktODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeXNpb2xvZ2ljYWwgYW5kIG1vbGVjdWxhciBhc3Nlc3NtZW50IG9mIGFsdGVyZWQgZXhwcmVzc2lvbiBvZiBIc2M3MC0xIGluIEFyYWJpZG9wc2lzLiBFdmlkZW5jZSBmb3IgcGxlaW90cm9waWMgY29uc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzMjsgIGNpdGF0aW9uX3BhZ2VzPTk3OS04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNnZW5pYyBleHByZXNzaW9uIG9mIHRoZSBUcmljaG9kZXJtYSBoYXJ6aWFudW0gaHNwNzAgZ2VuZSBpbmNyZWFzZXMgQXJhYmlkb3BzaXMgcmVzaXN0YW5jZSB0byBoZWF0IGFuZCBvdGhlciBhYmlvdGljIHN0cmVzc2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBQbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE2NzsgIGNpdGF0aW9uX3BhZ2VzPTY1OS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJlZCBleHByZXNzaW9uIG9mIGN5dG9zb2xpYy9udWNsZWFyIEhTQzcwLTEgbW9sZWN1bGFyIGNoYXBlcm9uZSBhZmZlY3RzIGRldmVsb3BtZW50IGFuZCBhYmlvdGljIHN0cmVzcyB0b2xlcmFuY2UgaW4gQXJhYmlkb3BzaXMgdGhhbGlhbmE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV4cC4gQm90OyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTYwOyAgY2l0YXRpb25fcGFnZXM9MjY1My02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHBsYW50IGhlYXQgc3RyZXNzIHRyYW5zY3JpcHRpb24gZmFjdG9yIChIc2YpIGZhbWlseTogc3RydWN0dXJlLCBmdW5jdGlvbiBhbmQgZXZvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGltLiBCaW9waHlzLiBBY3RhOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE4MTk7ICBjaXRhdGlvbl9wYWdlcz0xMDQtMTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvc3QtdHJhbnNjcmlwdGlvbmFsIGFuZCBwb3N0LXRyYW5zbGF0aW9uYWwgcmVndWxhdGlvbnMgb2YgZHJvdWdodCBhbmQgaGVhdCByZXNwb25zZSBpbiBwbGFudHM6IGEgc3BpZGVy4oCZcyB3ZWIgb2YgbWVjaGFuaXNtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIFBsYW50IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWx0ZXJuYXRpdmUgc3BsaWNpbmc6IGEgcGl2b3RhbCBzdGVwIGJldHdlZW4gZXVrYXJ5b3RpYyB0cmFuc2NyaXB0aW9uIGFuZCB0cmFuc2xhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIE1vbC4gQ2VsbCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTUzLTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmUtbVJOQSBwcm9jZXNzaW5nIHJlYWNoZXMgYmFjayB0byB0cmFuc2NyaXB0aW9uIGFuZCBhaGVhZCB0byB0cmFuc2xhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzY7ICBjaXRhdGlvbl9wYWdlcz02ODgtNzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSBzcGxpY2luZyBhdCB0aGUgaW50ZXJzZWN0aW9uIG9mIGJpb2xvZ2ljYWwgdGltaW5nLCBkZXZlbG9wbWVudCwgYW5kIHN0cmVzcyByZXNwb25zZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zNjQwLTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSBzcGxpY2luZyBvZiBwcmUtbWVzc2VuZ2VyIFJOQXMgaW4gcGxhbnRzIGluIHRoZSBnZW5vbWljIGVyYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5udS4gUmV2LiBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9MjY3LTk0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TcGxpY2luZyByZWd1bGF0aW9uOiBmcm9tIGEgcGFydHMgbGlzdCBvZiByZWd1bGF0b3J5IGVsZW1lbnRzIHRvIGFuIGludGVncmF0ZWQgc3BsaWNpbmcgY29kZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Uk5BOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9ODAyLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBoZWF0LXNlbnNpdGl2ZSBhbHRlcm5hdGl2ZSBzcGxpY2luZyBpbiBQaHlzY29taXRyZWxsYSBwYXRlbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTY1OyAgY2l0YXRpb25fcGFnZXM9ODI2LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBhbHRlcm5hdGl2ZSBzcGxpY2luZyBldmVudHMgYnkgUk5BIHNlcXVlbmNpbmcgaW4gZWFybHkgZ3Jvd3RoIHRvbWF0byBmcnVpdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTk0OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb25zIG9mIGNvbXB1dGF0aW9uYWwgbWV0aG9kcyBmb3IgZGlmZmVyZW50aWFsIGFsdGVybmF0aXZlIHNwbGljaW5nIGRldGVjdGlvbiB1c2luZyBSTkEtc2VxIGluIHBsYW50IHN5c3RlbXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm07ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIHNwbGljaW5nIGluIHBsYW50c+KAk2NvbWluZyBvZiBhZ2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBQbGFudCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz02MTYtMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIHNwbGljaW5nIGFuZCBub25zZW5zZS1tZWRpYXRlZCBkZWNheSBtb2R1bGF0ZSBleHByZXNzaW9uIG9mIGltcG9ydGFudCByZWd1bGF0b3J5IGdlbmVzIGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTI0NTQtNjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIHNwbGljaW5nOiBlbmhhbmNpbmcgYWJpbGl0eSB0byBjb3BlIHdpdGggc3RyZXNzIHZpYSB0cmFuc2NyaXB0b21lIHBsYXN0aWNpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xODUtMTg2OyAgY2l0YXRpb25fcGFnZXM9NDAtNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXhpdHkgb2YgdGhlIGFsdGVybmF0aXZlIHNwbGljaW5nIGxhbmRzY2FwZSBpbiBwbGFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zNjU3LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBtYXBwaW5nIG9mIGFsdGVybmF0aXZlIHNwbGljaW5nIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NDUtNTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJvbiByZXRlbnRpb24gaXMgYSBtYWpvciBwaGVub21lbm9uIGluIGFsdGVybmF0aXZlIHNwbGljaW5nIGluIEFyYWJpZG9wc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9ODc3LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBjYXRhbG9naW5nIGFuZCBhbmFseXNpcyBvZiBhbHRlcm5hdGl2ZWx5IHNwbGljZWQgZ2VuZXMgaW4gY2VyZWFsIGNyb3BzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz03MjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEtc2VxIG9mIGFyYWJpZG9wc2lzIHBvbGxlbiB1bmNvdmVycyBub3ZlbCB0cmFuc2NyaXB0aW9uIGFuZCBhbHRlcm5hdGl2ZSBzcGxpY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNjI7ICBjaXRhdGlvbl9wYWdlcz0xMDkyLTEwOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVvbWljIGFuYWx5c2VzIG9mIE9yeXphIHNhdGl2YSBtYXR1cmUgcG9sbGVuIHJldmVhbCBub3ZlbCBwcm90ZWlucyBhc3NvY2lhdGVkIHdpdGggcG9sbGVuIGdlcm1pbmF0aW9uIGFuZCB0dWJlIGdyb3d0aDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvdGVvbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MjUwNC0yOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGVyc3BlY3RpdmVzIG9uIGRlY2lwaGVyaW5nIG1lY2hhbmlzbXMgdW5kZXJseWluZyBwbGFudCBoZWF0IHN0cmVzcyByZXNwb25zZSBhbmQgdGhlcm1vdG9sZXJhbmNlOyBjaXRhdGlvbl9hdXRob3I9U29sYW5hY2VhZSBQb2xsZW4gVGhlcm1vdG9sZXJhbmNlIEluaXRpYWwgVHJhaW5pbmcgTmV0d29yaywgQzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MzE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5zdXJpbmcgcmVwcm9kdWN0aW9uIGF0IGhpZ2ggdGVtcGVyYXR1cmVzOiB0aGUgaGVhdCBzdHJlc3MgcmVzcG9uc2UgZHVyaW5nIGFudGhlciBhbmQgcG9sbGVuIGRldmVsb3BtZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudHM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGVmZmVjdCBvZiBoZWF0IHN0cmVzcyBvbiB0b21hdG8gcG9sbGVuIGNoYXJhY3RlcmlzdGljcyBpcyBhc3NvY2lhdGVkIHdpdGggY2hhbmdlcyBpbiBjYXJib2h5ZHJhdGUgY29uY2VudHJhdGlvbiBpbiB0aGUgZGV2ZWxvcGluZyBhbnRoZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIEJvdDsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT05MDsgIGNpdGF0aW9uX3BhZ2VzPTYzMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGRlY2FkZSBvZiBwb2xsZW4gdHJhbnNjcmlwdG9taWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBSZXByb2Q7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz03My04OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SHNmQTIgY29udHJvbHMgdGhlIGFjdGl2aXR5IG9mIGRldmVsb3BtZW50YWxseSBhbmQgc3RyZXNzLXJlZ3VsYXRlZCBoZWF0IHN0cmVzcyBwcm90ZWN0aW9uIG1lY2hhbmlzbXMgaW4gdG9tYXRvIG1hbGUgcmVwcm9kdWN0aXZlIHRpc3N1ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtZW1icmFuZSBwcm90ZW9tZSBvZiBtYWxlIGdhbWV0b3BoeXRlIGluIFNvbGFudW0gbHljb3BlcnNpY3VtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBQcm90ZW9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTEzMTsgIGNpdGF0aW9uX3BhZ2VzPTQ4LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgU29sIEdlbm9taWNzIE5ldHdvcmsgKHNvbGdlbm9taWNzLm5ldCk6IGdyb3dpbmcgdG9tYXRvZXMgdXNpbmcgUGVybDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz1EMTE0OS01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9wSGF0MjogYWNjdXJhdGUgYWxpZ25tZW50IG9mIHRyYW5zY3JpcHRvbWVzIGluIHRoZSBwcmVzZW5jZSBvZiBpbnNlcnRpb25zLCBkZWxldGlvbnMgYW5kIGdlbmUgZnVzaW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz1SMzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgdG9tYXRvIGdlbm9tZSBzZXF1ZW5jZSBwcm92aWRlcyBpbnNpZ2h0cyBpbnRvIGZsZXNoeSBmcnVpdCBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODU7ICBjaXRhdGlvbl9wYWdlcz02MzUtNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhUU2Vx4oCTYSBQeXRob24gZnJhbWV3b3JrIHRvIHdvcmsgd2l0aCBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZyBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTE2Ni05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiByZWxhdGl2ZSBnZW5lIGV4cHJlc3Npb24gZGF0YSB1c2luZyByZWFsLXRpbWUgcXVhbnRpdGF0aXZlIFBDUiBhbmQgdGhlIDIoLURlbHRhIERlbHRhIEMoVCkpIE1ldGhvZDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTQwMi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIG9udG9sb2d5OiB0b29sIGZvciB0aGUgdW5pZmljYXRpb24gb2YgYmlvbG9neS4gVGhlIEdlbmUgT250b2xvZ3kgQ29uc29ydGl1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTI1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyUHJvU2NhbiA1OiBnZW5vbWUtc2NhbGUgcHJvdGVpbiBmdW5jdGlvbiBjbGFzc2lmaWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xMjM2LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DeXRvc2NhcGU6IGEgc29mdHdhcmUgZW52aXJvbm1lbnQgZm9yIGludGVncmF0ZWQgbW9kZWxzIG9mIGJpb21vbGVjdWxhciBpbnRlcmFjdGlvbiBuZXR3b3JrczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTI0OTgtNTA0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaU5HTzogYSBDeXRvc2NhcGUgcGx1Z2luIHRvIGFzc2VzcyBvdmVycmVwcmVzZW50YXRpb24gb2YgZ2VuZSBvbnRvbG9neSBjYXRlZ29yaWVzIGluIGJpb2xvZ2ljYWwgbmV0d29ya3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MzQ0OC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db250cm9sbGluZyB0aGUgZmFsc2UgZGlzY292ZXJ5IHJhdGUgLSBhIHByYWN0aWNhbCBhbmQgcG93ZXJmdWwgYXBwcm9hY2ggdG8gbXVsdGlwbGUgdGVzdGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gUi4gU3RhdC4gU29jLiBCLiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9Mjg5LTMwMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlmZmVyZW50aWFsIGdlbmUgYW5kIHRyYW5zY3JpcHQgZXhwcmVzc2lvbiBhbmFseXNpcyBvZiBSTkEtc2VxIGV4cGVyaW1lbnRzIHdpdGggVG9wSGF0IGFuZCBDdWZmbGlua3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUHJvdG9jOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz01NjItNzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBmYW06IHRoZSBwcm90ZWluIGZhbWlsaWVzIGRhdGFiYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPUQyMjItMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZGRlbiBNYXJrb3YgbW9kZWwgc3BlZWQgaGV1cmlzdGljIGFuZCBpdGVyYXRpdmUgSE1NIHNlYXJjaCBwcm9jZWR1cmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTQzMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgRXhwcmVzc2lvbiBPbW5pYnVzOiBOQ0JJIGdlbmUgZXhwcmVzc2lvbiBhbmQgaHlicmlkaXphdGlvbiBhcnJheSBkYXRhIHJlcG9zaXRvcnk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MjA3LTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYW5kc2NhcGUgb2YgdGhlIGxpcGlkb21lIGFuZCB0cmFuc2NyaXB0b21lIHVuZGVyIGhlYXQgc3RyZXNzIGluIEFyYWJpZG9wc2lzIHRoYWxpYW5hOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MTA1MzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBhbHRlcm5hdGl2ZSBzcGxpY2luZyBpbiBaZWEgbWF5czogbGFuZHNjYXBlIGFuZCBnZW5ldGljIHJlZ3VsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0zNDcyLTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvbXBhcmlzb24gb2YgdGhlIGxvdyB0ZW1wZXJhdHVyZSB0cmFuc2NyaXB0b21lcyBvZiB0d28gdG9tYXRvIGdlbm90eXBlcyB0aGF0IGRpZmZlciBpbiBmcmVlemluZyB0b2xlcmFuY2U6IFNvbGFudW0gbHljb3BlcnNpY3VtIGFuZCBTb2xhbnVtIGhhYnJvY2hhaXRlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIFBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMzIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaWRlc3ByZWFkIGludHJvbiByZXRlbnRpb24gaW4gbWFtbWFscyBmdW5jdGlvbmFsbHkgdHVuZXMgdHJhbnNjcmlwdG9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNzc0LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJlc3NlZCBvdXQhIEVmZmVjdHMgb2YgZW52aXJvbm1lbnRhbCBzdHJlc3Mgb24gbVJOQSBtZXRhYm9saXNtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GRU1TIFllYXN0IFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MTYwLTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNwbGljaW5nIGZhY3RvciB0aGF0IGlzIGluYWN0aXZhdGVkIGR1cmluZyBpbnZpdm8gaGVhdC1zaG9jayBpcyBmdW5jdGlvbmFsbHkgZXF1aXZhbGVudCB0byB0aGUgW1U0L1U2LlU1XSBUcmlwbGUgU25ybnAtU3BlY2lmaWMgUHJvdGVpbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmVzIERldjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NjMxLTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhbHRlcm5hdGl2ZWx5IHNwbGljZWQgaGVhdCBzaG9jayB0cmFuc2NyaXB0aW9uIGZhY3RvciwgT3NIU0ZBMmRJLCBmdW5jdGlvbnMgaW4gdGhlIGhlYXQgc3RyZXNzLWluZHVjZWQgdW5mb2xkZWQgcHJvdGVpbiByZXNwb25zZSBpbiByaWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9NDE5LTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhdXRvcmVndWxhdG9yeSBsb29wIGNvbnRyb2xsaW5nIEFyYWJpZG9wc2lzIEhzZkEyIGV4cHJlc3Npb246IHJvbGUgb2YgaGVhdCBzaG9jay1pbmR1Y2VkIGFsdGVybmF0aXZlIHNwbGljaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE2MjsgIGNpdGF0aW9uX3BhZ2VzPTUxMi01MjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBDeXRvc29saWMgcHJvdGVpbiByZXNwb25zZSBhcyBhIHN1YmNvbXBvbmVudCBvZiB0aGUgd2lkZXIgaGVhdCBzaG9jayByZXNwb25zZSBpbiBBcmFiaWRvcHNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTY0Mi01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGFuZCBmdW5jdGlvbmFsIGNoYXJhY3Rlcml6YXRpb24gb2YgSFNDNTQsIGEgbm92ZWwgdmFyaWFudCBvZiBodW1hbiBoZWF0LXNob2NrIGNvZ25hdGUgcHJvdGVpbiA3MDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQaGFybWFjb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NTg7ICBjaXRhdGlvbl9wYWdlcz0xMjU3LTI2MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgYSBub3ZlbCBzcGxpY2UgdmFyaWFudCBvZiBoZWF0IHNob2NrIGNvZ25hdGUgcHJvdGVpbiA3MCBhZnRlciBjaHJvbmljIGFudGlkZXByZXNzYW50IHRyZWF0bWVudCBpbiByYXQgZnJvbnRhbCBjb3J0ZXg7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoZW0uIEJpb3BoeXMuIFJlcy4gQ29tbXVuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI2MTsgIGNpdGF0aW9uX3BhZ2VzPTU0MS01OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbHRlcm5hdGl2ZSA14oCZIHNwbGljZSBzaXRlIHNlbGVjdGlvbiBpbmR1Y2VkIGJ5IGhlYXQgc2hvY2s7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQ2VsbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTU2Ny03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9V2lkZXNwcmVhZCBpbnRyb24gcmV0ZW50aW9uIGRpdmVyc2lmaWVzIG1vc3QgY2FuY2VyIHRyYW5zY3JpcHRvbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgTWVkOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz00NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5vaXN5IHNwbGljaW5nIGRyaXZlcyBtUk5BIGlzb2Zvcm0gZGl2ZXJzaXR5IGluIGh1bWFuIGNlbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMTAwMTIzNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVycm9yIHByZXZlbnRpb24gYW5kIG1pdGlnYXRpb24gYXMgZm9yY2VzIGluIHRoZSBldm9sdXRpb24gb2YgZ2VuZXMgYW5kIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTg3NS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPVNwcmluZ2VyLCBCZXJsaW47IE5ldyBZb3JrOyAgY2l0YXRpb25fdGl0bGU9UGxhbnQgUHJvdGVvbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGVhdC1zdHJlc3MtZGVwZW5kZW5jeSBhbmQgZGV2ZWxvcG1lbnRhbCBtb2R1bGF0aW9uIG9mIGdlbmUgZXhwcmVzc2lvbjogdGhlIHBvdGVudGlhbCBvZiBob3VzZS1rZWVwaW5nIGdlbmVzIGFzIGludGVybmFsIHN0YW5kYXJkcyBpbiBtUk5BIGV4cHJlc3Npb24gcHJvZmlsaW5nIHVzaW5nIHJlYWwtdGltZSBSVC1QQ1I7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV4cC4gQm90OyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9MjM0My05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWd1bGF0aW9uIG9mIHByb3RlaW4gZnVuY3Rpb24gYnkg4oCYbWljcm9Qcm90ZWluc+KAmTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RU1CTyBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0zNS00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGV0aXRpdmUgaW5oaWJpdGlvbiBvZiB0cmFuc2NyaXB0aW9uIGZhY3RvcnMgYnkgc21hbGwgaW50ZXJmZXJpbmcgcGVwdGlkZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBQbGFudCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz01NDEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG9tYWluIGF0cm9waHkgY3JlYXRlcyByYXJlIGNhc2VzIG9mIGZ1bmN0aW9uYWwgcGFydGlhbCBwcm90ZWluIGRvbWFpbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9ODgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgcHJvZmlsaW5nIG9mIEFyYWJpZG9wc2lzIGhlYXQgc2hvY2sgcHJvdGVpbnMgYW5kIHRyYW5zY3JpcHRpb24gZmFjdG9ycyByZXZlYWxzIGV4dGVuc2l2ZSBvdmVybGFwIGJldHdlZW4gaGVhdCBhbmQgbm9uLWhlYXQgc3RyZXNzIHJlc3BvbnNlIHBhdGh3YXlzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTEyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRlbXBlcmF0dXJlIHN0cmVzcyBkaWZmZXJlbnRpYWxseSBtb2R1bGF0ZXMgdHJhbnNjcmlwdGlvbiBpbiBtZWlvdGljIGFudGhlcnMgb2YgaGVhdC10b2xlcmFudCBhbmQgaGVhdC1zZW5zaXRpdmUgdG9tYXRvIHBsYW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9Mzg0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGN1bGxpbiBwcm90ZWluIGZhbWlseTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0yMjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZWRkOCBvbiBjdWxsaW46IGJ1aWxkaW5nIGFuIGV4cHJlc3N3YXkgdG8gcHJvdGVpbiBkZXN0cnVjdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9T25jb2dlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xOTg1LTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DdWxsaW4tUklORyB1YmlxdWl0aW4gbGlnYXNlIGZhbWlseSBpbiBwbGFudCBhYmlvdGljIHN0cmVzcyBwYXRod2F5cyhGKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gSW50ZWdyLiBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTU1OyAgY2l0YXRpb25fcGFnZXM9MjEtMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZG9taW5hbnQtbmVnYXRpdmUgZm9ybSBvZiB0aGUgRTMgdWJpcXVpdGluIGxpZ2FzZSBDdWxsaW4tMSBkaXNydXB0cyB0aGUgY29ycmVjdCBhbGxvY2F0aW9uIG9mIGNlbGwgZmF0ZSBpbiB0aGUgbmV1cmFsIGNyZXN0IGxpbmVhZ2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPURldmVsb3BtZW50OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEzMzsgIGNpdGF0aW9uX3BhZ2VzPTU1OS02ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlmYWNldGVkIHJvbGVzIG9mIEFyYWJpZG9wc2lzIFBQNiBwaG9zcGhhdGFzZSBpbiByZWd1bGF0aW5nIGNlbGx1bGFyIHNpZ25hbGluZyBhbmQgcGxhbnQgZGV2ZWxvcG1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFNpZ25hbC4gQmVoYXY7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWUyMjUwOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcmluZS90aHJlb25pbmUgcHJvdGVpbiBwaG9zcGhhdGFzZSA1IChQUDUpIGludGVyYWN0cyB3aXRoIHN1YnN0cmF0ZSB1bmRlciBoZWF0IHN0cmVzcyBjb25kaXRpb25zIGFuZCBmb3JtcyBwcm90ZWluIGNvbXBsZXggaW4gQXJhYmlkb3BzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFNpZ25hbC4gQmVoYXY7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTUzNS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIG9mIHByb3RlaW4gcGhvc3BoYXRhc2UtNiBhc3NvY2lhdGlvbiB3aXRoIGl0cyBTQVBTIGRvbWFpbiByZWd1bGF0b3J5IHN1YnVuaXQgdXNpbmcgYSBtb2RlbCBvZiBoZWxpY2FsIHJlcGVhdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBCaW9jaGVtOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjQvMi8yMDUvMTM2ODg4NzEvZHN3MDUxLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBBbHRlcm5hdGl2ZSBzcGxpY2luZyAoQVMpIGlzIGEga2V5IGNvbnRyb2wgbWVjaGFuaXNtIGluZmx1ZW5jaW5nIHNpZ25hbCByZXNwb25zZSBjYXNjYWRlcyBpbiBkaWZmZXJlbnQgZGV2ZWxvcG1lbnRhbCBzdGFnZXMgYW5kIHVuZGVyIHN0cmVzcyBjb25kaXRpbyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MjczODY3OSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQ8BJsf9VmHr3DjI2RjlI0pMqYpWA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f205%2f2738679" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f205%2f2738679" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97104" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&amp;Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/2">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="78054887">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78054887">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78054889">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78054889">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78054895">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78054895">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78054924">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78054924">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78054945">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78054945">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78054957">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78054957">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78054965">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78054965">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/2/193/2731026">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/2/219/3744603">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Alternative splicing in tomato pollen in response to heat stress<a reveal-id="dsw051-FM2" data-open="dsw051-FM2" class="link link-ref link-reveal xref-fn"><sup>†</sup></a>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mario Keller</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Biosciences, Molecular Cell Biology of Plants</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mario+Keller">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Keller M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Keller M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mario Keller</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yangjie Hu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Biosciences, Molecular Cell Biology of Plants</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yangjie+Hu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hu Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hu Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yangjie Hu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Anida Mesihovic</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Biosciences, Molecular Cell Biology of Plants</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Anida+Mesihovic">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mesihovic A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mesihovic A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Anida Mesihovic</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sotirios Fragkostefanakis</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Biosciences, Molecular Cell Biology of Plants</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sotirios+Fragkostefanakis">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fragkostefanakis S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fragkostefanakis S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sotirios Fragkostefanakis</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Enrico Schleiff</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Biosciences, Molecular Cell Biology of Plants</div><div class="aff"><div class="label -label">2</div>Cluster of Excellence Frankfurt</div><div class="aff"><div class="label -label">3</div>Buchmann Institute for Molecular Life Sciences (BMLS), Goethe University, D-60438 Frankfurt am Main, Germany</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw051-cor1" data-open="dsw051-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: 00496979829287. E-mail: <a href="mailto:schleiff@bio.uni-frankfurt.de" target="_blank">schleiff@bio.uni-frankfurt.de</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Enrico+Schleiff">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Schleiff E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Schleiff E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Enrico Schleiff</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Stefan Simm</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Biosciences, Molecular Cell Biology of Plants</div><div class="aff"><div class="label -label">2</div>Cluster of Excellence Frankfurt</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Stefan+Simm">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Simm S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Simm S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Stefan Simm</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw051-FM2"><div class="fn"><div class="footnote-content"><sup>†</sup>This manuscript is a contribution of SPOT-ITN.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw051-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Doil Choi</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 205–217, <a href='https://doi.org/10.1093/dnares/dsw051'>https://doi.org/10.1093/dnares/dsw051</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">24 December 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">23 June 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">26 October 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2738679" href="/dnaresearch/article-pdf/24/2/205/13688871/dsw051.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Mario Keller, Yangjie Hu, Anida Mesihovic, Sotirios Fragkostefanakis, Enrico Schleiff, Stefan Simm;  Alternative splicing in tomato pollen in response to heat stress, <em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 205–217, <a href='https://doi.org/10.1093/dnares/dsw051'>https://doi.org/10.1093/dnares/dsw051</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2738679?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2738679?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2738679?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2738679?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2738679?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2738679?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2738679?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Alternative%20splicing%20in%20tomato%20pollen%20in%20response%20to%20heat%20stress%26lt%3Ba%20reveal-id%3D%26quot%3Bdsw051-FM2%26quot%3B%20data-open%3D%26quot%3Bdsw051-FM2%26quot%3B%20class%3D%26quot%3Blink%20link-ref%20link-reveal%20xref-fn%26quot%3B%26gt%3B%26lt%3Bsup%26gt%3B%E2%80%A0%26lt%3B%2Fsup%26gt%3B%26lt%3B%2Fa%26gt%3B&amp;publicationDate=2016-12-24&amp;volumeNum=24&amp;issueNum=2&amp;author=Keller%2C%20Mario%3B%20Hu%2C%20Yangjie&amp;startPage=205&amp;endPage=217&amp;contentId=10.1093%2Fdnares%2Fdsw051&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f205%2f2738679" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=78054887 id="78054887" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Alternative splicing (AS) is a key control mechanism influencing signal response cascades in different developmental stages and under stress conditions. In this study, we examined heat stress (HS)-induced AS in the heat sensitive pollen tissue of two tomato cultivars. To obtain the entire spectrum of HS-related AS, samples taken directly after HS and after recovery were combined and analysed by RNA-seq. For nearly 9,200 genes per cultivar, we observed at least one AS event under HS. In comparison to control, for one cultivar we observed 76% more genes with intron retention (IR) or exon skipping (ES) under HS. Furthermore, 2,343 genes had at least one transcript with IR or ES accumulated under HS in both cultivars. These genes are involved in biological processes like protein folding, gene expression and heat response. Transcriptome assembly of these genes revealed that most of the alternative spliced transcripts possess truncated coding sequences resulting in partial or total loss of functional domains. Moreover, 141 HS specific and 22 HS repressed transcripts were identified. Further on, we propose AS as layer of stress response regulating constitutively expressed genes under HS by isoform abundance.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;tomato pollen&quot;">tomato pollen</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="RNA-seq">RNA-seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;alternative splicing&quot;">alternative splicing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;heat stress&quot;">heat stress</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="assembly">assembly</a></div>                            <h2 scrollto-destination=78054889 id="78054889" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>The response of sessile lifeforms like plants to environmental alterations like heat stress (HS) is dependent on a complex network of distinct and interconnected pathways to maintain protein homeostasis and minimize cellular damage. Heat shock proteins (Hsps) are examples for HS-induced genes in various plant species,<a reveal-id="dsw051-B1 dsw051-B2 dsw051-B3 dsw051-B4 dsw051-B5 dsw051-B6" data-open="dsw051-B1 dsw051-B2 dsw051-B3 dsw051-B4 dsw051-B5 dsw051-B6" class="link link-ref link-reveal xref-bibr"><sup>1–6</sup></a> which contribute to thermotolerance as shown by genetic studies.<a reveal-id="dsw051-B7 dsw051-B8 dsw051-B9 dsw051-B10" data-open="dsw051-B7 dsw051-B8 dsw051-B9 dsw051-B10" class="link link-ref link-reveal xref-bibr"><sup>7–10</sup></a> The majority of these HS-induced genes are under the control of HS transcription factors (Hsfs). Although metazoans and fungi contain only one, plant genomes code for a large repertoire of Hsfs, which are of major importance due to their ability to rapidly activate the transcription of essential genes for stress response and survival.<a reveal-id="dsw051-B11" data-open="dsw051-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a></p><p>In addition to transcriptional changes, pre-mRNA alternative splicing (AS) is another important mechanism involved in heat stress response (HSR).<a reveal-id="dsw051-B12" data-open="dsw051-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Splicing is a central process for pre-mRNA maturation of intron-containing genes in all eukaryotic organisms.<a reveal-id="dsw051-B13" data-open="dsw051-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsw051-B14" data-open="dsw051-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> In response to environmental or developmental stimuli, AS of a single pre-mRNA can lead to various transcripts that may code for different functional protein variants.<a reveal-id="dsw051-B15" data-open="dsw051-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> AS is a key mechanism for eukaryotes to increase proteome diversity from a defined number of pre-mRNAs.<a reveal-id="dsw051-B16" data-open="dsw051-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsw051-B17" data-open="dsw051-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> Consequently, AS is a common regulatory mechanism and the enhanced transcriptome diversity is required for stress adaptation<a reveal-id="dsw051-B18" data-open="dsw051-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and definition of plant tissue identity.<a reveal-id="dsw051-B19" data-open="dsw051-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> In line, in plants about 61% of multi-exon genes are alternative spliced.<a reveal-id="dsw051-B20" data-open="dsw051-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a></p><p>At stage, at least four different modes of AS are annotated as intron retention (IR), exon skipping (ES), alternative acceptor (AA) and alternative donor (AD) site splicing.<a reveal-id="dsw051-B20" data-open="dsw051-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> IR occurs when an intron is not removed from the pre-mRNA and remains in the mature mRNA. ES describes the event where an exon is spliced out of the pre-mRNA and AA and AD describe the usage of alternative 3′ and 5′ splice sites by the spliceosome, respectively.<a reveal-id="dsw051-B21" data-open="dsw051-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> The various modes of AS can lead to premature termination codons (PTCs) resulting either in degradation of the mRNA through the nonsense-mediated decay (NMD) pathway or translation of a truncated protein.<a reveal-id="dsw051-B22" data-open="dsw051-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsw051-B23" data-open="dsw051-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> In addition, AS can result in changes or complete removal of functional domains, which may lead to proteins with novel functions.<a reveal-id="dsw051-B24" data-open="dsw051-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> The prevalent AS event in <em>Arabidopsis thaliana</em> is IR that leads to PTCs.<a reveal-id="dsw051-B25" data-open="dsw051-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsw051-B26" data-open="dsw051-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a></p><p>Therefore, the analysis of genome-wide AS especially in crop plants is of major interest to describe the mechanisms involved in stress response<a reveal-id="dsw051-B27" data-open="dsw051-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and pollen development.<a reveal-id="dsw051-B28" data-open="dsw051-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> Based on this, we analysed the transcriptome of tomato (<em>Solanum lycopersicum</em>) pollen as model for a reduced tissue (two or three celled structure)<a reveal-id="dsw051-B29" data-open="dsw051-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a>. Pollen development is characterized by a remarkable sensitivity to high temperatures.<a reveal-id="dsw051-B30" data-open="dsw051-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsw051-B31" data-open="dsw051-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> For example, plants exposed to HS show decreased pollen viability,<a reveal-id="dsw051-B32" data-open="dsw051-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> which can cause sterility and in an agricultural aspect lead to crop damage.<a reveal-id="dsw051-B33" data-open="dsw051-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a></p><p>We used pollen from two different tomato cultivars, namely Moneymaker and Red Setter to analyse the changes induced by AS in response to HS using RNA-seq datasets. The application of two cultivars enabled us to exclude genotype specific AS and focus on HS induced AS. We focused on changes induced by IR, ES, AA and AD. Additionally, HS-dependent accumulation and reduction of AS events in specific genes was observed and these genes used to identify biological processes affected by AS under HS. In the end, transcriptome assembly led to the identification of 141 HS-specific and 22 HS-repressed transcripts. HS-specific and repressed putative protein isoforms were analysed concerning length differences in general and functional changes due to loss, gain or change of functional domains. We observed transcripts with steady expression levels under control conditions and HS showing a clear difference in the occurrence of specific isoforms partially or completely lacking functional domains. The latter exemplifies that post transcriptional AS leads to synthesis of transcripts putatively encoding for alternative protein isoforms, which might be required for HSR.</p>                            <h2 scrollto-destination=78054895 id="78054895" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=78054896 id="78054896" class="section-title" >2.1. Plant material, stress treatment and pollen analysis</h3>
<p>Tomato plants (<em>S</em><em>.</em><em>lycopersicum</em> cv. Moneymaker and Red Setter) were grown under controlled greenhouse conditions (24 °C/18 °C, day/night temperature; 60% humidity). Eight-week old flowering plants were transferred to a climate chamber and exposed to a 39°C HS treatment for 1 h and then allowed to recover till anthesis. Flowers that received the stress 9–12 days before anthesis, corresponding to pollen meiosis and tetrads stages of development were analysed for pollen viability and number as previously described.<a reveal-id="dsw051-B34" data-open="dsw051-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> In total 15–20 plants were used for the analysis for each genotype and treatment, and the experiment was repeated three independent times.</p><p>For RNAseq analysis flowering plants received the same treatment and buds were collected directly after HS or allowed to recover for 1.5 h at 24 °C. Untreated/control plants were kept at 24°C for the same time. Anthers from four flower buds were sampled in 500 µl of germination solution (2 mM boric acid, 2 mM calcium nitrate, 2 mM magnesium sulphate, 1 mM potassium nitrate) and squeezed open with slight mechanical stress, vortexed and filtered through muslin cloth. The collected pollen grains were harvested by centrifugation at 10,000× g for 15 min at 4 °C. The pelleted pollen were washed in 200 µl germination solution and centrifuged at 1,000× g for 1 min at 4 °C. This step was repeated a couple of times to remove the unwanted material. In total, more than 100 flower buds were used for pollen isolation. Purity of pollen was inspected under a light microscope. After isolation of pollen, all developmental pollen stages (microspore mother cell, tetrads, microspores and mature pollen) were pooled together for each of the four conditions: cv. Moneymaker-CO, cv. Moneymaker-HS, cv. Red Setter-CO and cv. Red Setter-HS. For each condition, two biological replicates were used for RNA-seq analysis, and another three independent biological replicates were used for qPCR analysis.</p>                            <h3 scrollto-destination=78054899 id="78054899" class="section-title" >2.2. RNA isolation and mRNA sequencing</h3>
<p>Total RNA was extracted from pollen of all eight samples using the E.Z.N.A. Plant RNA Kit (Omega Bio-Tek, Norcross, GA, USA). Total RNA was used for the preparation of mRNA-seq libraries (GenXPro GmbH, Frankfurt Main, Germany) by performing rRNA depletion and enrichment of mRNA via polyadenylation (polyA) selection and purification.<a reveal-id="dsw051-B35" data-open="dsw051-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> Constructed libraries from each replicate of cv. Moneymaker and cv. Red Setter were strand-specific sequenced (paired-end) with a read length of 135 bases on a HiSeq2500 (GenXPro GmbH, Frankfurt Main). The mRNA-seq datasets discussed in this publication have been deposited in NCBI Gene Expression Omnibus (Edgar <em>et al.</em>, 2002) and are accessible through GEO Series accession number GSE80556 (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE80556" target="_blank">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE80556</a>).</p>                            <h3 scrollto-destination=78054901 id="78054901" class="section-title" >2.3. Read alignment and identification of AS events</h3>
<p>The eight FASTQ files were aligned in paired-end mode to the <em>S. lycopersicum</em> genome (version ITAG2.4, cv. Heinz) available from the Sol Genomics Network.<a reveal-id="dsw051-B36" data-open="dsw051-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> The alignment of the RNA-seq reads onto the reference genome was done via TopHat2 (version 2.0.13)<a reveal-id="dsw051-B37" data-open="dsw051-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> using the following alterations of default parameters: –max-multihits 1; –min-intron-length 36; –max-intron-length 50,000; –mate-inner-dist 300; –mate-std-dev 100. The parameters were chosen due to the fact that the wild form of tomato <em>S</em><em>olanum</em><em>pimpinellifolium</em> has only a nucleotide divergence of 0.6%<a reveal-id="dsw051-B38" data-open="dsw051-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> and by this our cultivars should have no more divergence.</p><p>Next, the Pearson correlation coefficient of read counts was computed pairwise between all samples based on computed read counts of all genes. Read counts were obtained by counting the number of reads (NuR) aligned on a specific gene. Due to the high correlation (&gt;0.99) and small biological variation between biological replicates, the TopHat2 alignments of biological replicates were merged into a single alignment file resulting in four final alignment files (Moneymaker CO, Moneymaker HS, Red Setter CO and Red Setter HS).</p><p>Next, TopHat2 alignments were used for the identification of AS events, namely IR, ES, AA and AD. For this purpose, the coordinates of all annotated introns and exons were extracted from the ITAG2.4 corresponding Generic feature format version 3 (GFF3) file of <em>S. lycopersicum</em>. Introns were assumed to be retained (IR) if all positions of the intron were covered by at least one read. The other three exon specific events (ES, AA and AD) were considered to be present if the NuR witnessing the particular event exceeded the average genome coverage per position of each library.</p>                            <h3 scrollto-destination=78054905 id="78054905" class="section-title" >2.4. FPKM calculation and verification</h3>
<p>As cDNA fragments were sequenced from both ends (paired-end reads), quantification was done by counting paired reads as one fragment. Assignment of fragments was preceded to annotated exons of gene models (gene fragments). The assignment of fragment counts was performed with htseq-count of the High-Throughput Sequencing python framework (HTSeq).<a reveal-id="dsw051-B39" data-open="dsw051-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> Afterwards, for each gene the number of gene fragments was normalized by the size of the library and the length of all exons within the gene. Finally, multiplication by one billion (to counteract the division by kilobase and by million mapped reads) resulted in FPKM (fragments per kilo base of exon per million fragments mapped) values for genes.</p><p>FPKM calculation was verified by qRT-PCR. cDNA synthesis was performed with 1 μg of total RNA using the Revert Aid reverse transcriptase (Thermo Scientific) following the manufacturer’s instructions. The expression of selected genes was investigated by real-time PCR on a Stratagene Mx3000P cycler (Agilent Technologies). The qRT-PCR reaction (10 μl) included gene specific primers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S1</a></span>), PerfeCTa SYBR Green FastMix Low ROX™ (Quanta Biosciencies) and the appropriate cDNA template. Thermal cycling conditions were 95°C/3 min followed by 95°C/15 s, 60°C/30 s, 72°C/30 s for 40 cycles. Gene primers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S1</a></span>) were designed using PRIMER3 (<a class="link link-uri" href="http://www-genome.wi.mit.edu/cgi-bin/primer/primer3.cgi/" target="_blank">http://www-genome.wi.mit.edu/cgi-bin/primer/primer3.cgi/</a>; September 2016). Data were analysed by standard methods<a reveal-id="dsw051-B40" data-open="dsw051-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and presented as relative levels of gene expression using ubiquitin (UBI; Solyc07g064130) and EF1α (Solyc06g005060) genes as internal standards. Each qRT-PCR reaction was repeated three times on the basis of three independent biological replicates. The RNA used for qRT-PCR analysis was independent from the RNA used for RNA-seq.</p>                            <h3 scrollto-destination=78054908 id="78054908" class="section-title" >2.5. Detection of HS-dependent IR and ES events</h3>
<div class="block-child-p">At first, the frequency of all IR and ES was determined by normalizing the NuR witnessing the event to the size of the respective library. For determination of HS-dependent accumulation or reduction of the IR and ES events, the frequencies of the events were first normalized to the expression of the underlying gene by dividing it by the FPKM value of the gene. Next, log<sub>2</sub> fold changes between the ratios of the HS libraries and the CO libraries were computed for each cultivar (Formula 1): <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mtext>HS induction=</mtext><mo> </mo><mtext>log</mtext><mo> </mo></mrow><mrow><mn>2</mn></mrow></msub><msub xmlns=""><mrow><mfenced separators="|"><mrow><mtext> </mtext><mfrac><mrow><msub><mrow><mfenced separators="|"><mrow><mtext> </mtext><mfrac><mrow><msub><mrow><mtext>Frequency</mtext></mrow><mrow><mtext>Event</mtext></mrow></msub></mrow><mrow><mtext>FPKM</mtext></mrow></mfrac><mtext> </mtext></mrow></mfenced></mrow><mrow><mtext>HS</mtext></mrow></msub></mrow><mrow><msub><mrow><mfenced separators="|"><mrow><mtext> </mtext><mfrac><mrow><msub><mrow><mtext>Frequency</mtext></mrow><mrow><mtext>Event</mtext></mrow></msub></mrow><mrow><mtext>FPKM</mtext></mrow></mfrac><mtext> </mtext></mrow></mfenced></mrow><mrow><mtext>CO</mtext></mrow></msub></mrow></mfrac><mtext> </mtext></mrow></mfenced></mrow><mrow></mrow></msub></math></div></div></div></div><p>Afterwards, genes were discriminated by this in sets of (I) HS-dependent accumulation of AS and (II) HS-dependent reduction of AS. Regarding the first set (I), IR and ES events of the upper 5% quantiles of the log<sub>2</sub> fold change distributions and exclusively occurring under HS were determined. In the next step IR and ES events were assigned to genes. Finally, only genes showing HS-dependent accumulation of at least one common AS event (IR or ES) in both cultivars were extracted. Regarding the second set (II), in a similar approach genes showing HS-dependent reduction of AS were identified by extracting those IR and ES events of the lower 5% quantile and those exclusively present under CO.</p>                            <h3 scrollto-destination=78054911 id="78054911" class="section-title" >2.6. Gene set enrichment analysis</h3>
<p>Gene ontology (GO)<a reveal-id="dsw051-B41" data-open="dsw051-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> terms were assigned to all annotated genes (ITAG 2.4) via InterProScan (version 5.14-53.0).<a reveal-id="dsw051-B42" data-open="dsw051-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> Next, genes alternatively spliced under HS and CO conditions were analysed for enrichment of GO terms related to the biological process category. The enrichment analysis was performed with the Cytoscape (version 3.2.1)<a reveal-id="dsw051-B43" data-open="dsw051-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> plugin BiNGO (version 3.0.3)<a reveal-id="dsw051-B44" data-open="dsw051-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> by using the Benjamini and Hochberg false discovery rate (FDR) correction<a reveal-id="dsw051-B45" data-open="dsw051-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> and applying an adjusted <em>P</em>-value cut-off of 0.01 as threshold. In addition, visual inspection of enriched GO terms was performed to exclude unspecific terms (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S2</a></span>; bold terms). Terms were excluded if their function could be more precisely defined by a child-term (e.g. transport and intracellular transport).</p>                            <h3 scrollto-destination=78054913 id="78054913" class="section-title" >2.7. HS-dependent transcript assembly</h3>
<p>All mapped reads located in a range from 1 kilobase pair (kbp) downstream to 1 kbp upstream of genes showing HS-dependent accumulation or reduction of AS were extracted from TopHat2 Sequence Alignment/Map files of all four runs and used as input for reference based transcript assembly via Cufflinks (version 2.2.1).<a reveal-id="dsw051-B46" data-open="dsw051-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Cufflinks was executed with default parameters except for the following modifications: –min-isoform-fraction 0.05; –pre-mRNA-fraction 0.05; –small-anchor-fraction 0.05.</p><p>Transcripts assembled by Cufflinks were afterwards assigned to annotated tomato genes of the ITAG2.4 and renamed concerning the identifier (e.g. SolycXXgYYYYYY) of the gene. Next, all transcripts assigned to the same gene identifier were compared between HS and CO condition and those present only under HS for both cultivars determined as HS-specific transcripts and those only present under CO for both cultivars as HS-repressed transcripts. Transcripts were assumed to be similar if all underlying exon coordinates were identical except for the start coordinate of the first and the end coordinate of the last exon as such differences are expected to lead to alternative untranslated regions (UTRs) without an effect on the coding sequence (CDS).</p>                            <h3 scrollto-destination=78054916 id="78054916" class="section-title" >2.8. Identification of AS events in HS-specific and repressed transcripts</h3>
<p>HS-specific and -repressed transcripts were compared with the transcripts annotated in the GFF3 of the ITAG2.4. In a first step, transcripts, whose exterior exons were not overlapping the exterior exons of the ITAG2.4 (inconsistent length) were excluded. In a second step, transcripts covering more than one annotated tomato gene (multisolyc transcripts) or containing no stop codon (transcripts with no CDS) were excluded. Next, the remaining transcripts were sub-divided into five groups based on the presence and absence of AS events: transcripts with only IR (i, only IR), transcripts with only ES (ii, only ES), transcripts with mixtures of IR and ES (iii, IR + ES), transcripts with other events than only IR and ES like AA and AD events as well as not annotated introns and exons (iv, others) and transcripts showing no difference in their splicing pattern to the ITAG2.4 annotation (v, perfectly).</p>                            <h3 scrollto-destination=78054918 id="78054918" class="section-title" >2.9. Confirmation of HS-specific isoforms via RT-PCR</h3>
<p>Transcripts assembled via Cufflinks and TopHat2 based on RNA-seq data were verified via RT-PCR. One microgram of total RNA was used for cDNA synthesis with Revert Aid reverse transcriptase (Thermo Scientific) following the manufacturer’s protocol. Splicing of selected genes was monitored by RT-PCR using specific primers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S1</a></span>). Four transcripts were chosen, one serves as control with no AS isoform in all four samples (Solyc08g068350). The other three transcripts contain HS-specific IR (Solyc09g074680), ES (Solyc03g121800) or both (Solyc10g084400) in the HS datasets.</p>                            <h3 scrollto-destination=78054920 id="78054920" class="section-title" >2.10. Functional domain analysis</h3>
<p>As a first step, the CDS within the HS-specific and repressed transcripts was determined. The CDS search was performed by taking the start codon annotated in the ITAG2.4 as starting point and searching for the first stop codon (TAA, TAG or TGA). The obtained CDSs of all transcripts were then translated into protein sequences and together with the corresponding annotated protein sequences of tomato ITAG2.4 analysed for functional domains. For domain detection the protein families database (PFAM)<a reveal-id="dsw051-B47" data-open="dsw051-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> and HMMER3<a reveal-id="dsw051-B48" data-open="dsw051-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> were used executing hmmscan with default parameters. After that, the predicted domains were compared between the predicted and annotated proteins to identify gain, loss and changes in the domain architecture.</p>                            <h3 scrollto-destination=78054922 id="78054922" class="section-title" >2.11. Large datasets</h3>
<p>The mRNA-seq datasets discussed in this publication have been deposited in NCBI’s Gene Expression Omnibus<a reveal-id="dsw051-B49" data-open="dsw051-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> and are accessible through GEO Series accession number GSE80556 (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE80556" target="_blank">http://www.ncbi.nlm.nih.gov/geo/query/acc.cgi?acc=GSE80556</a>). Sixteen paired-end fastq files for eight samples are deposited. These eight samples are from tow tomato cultivars (Moneymaker and Red Setter) and treated with HS (39 °C) or under control conditions (25 °C) in two replicates each.</p>                            <h2 scrollto-destination=78054924 id="78054924" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=78054925 id="78054925" class="section-title" >3.1. AS events in tomato pollen in response to HS</h3>
<p>To analyse the effect of HS on the transcriptional fate of tomato pollen, total RNA of pooled developmental stages of pollen was extracted from Moneymaker and Red Setter tomato plants. Phenotypic analysis revealed that the short (1 h) HS at 39°C did not affect the viability of Moneymaker pollen but caused a 21% reduction in Red Setter (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S3</a></span>). The analysis was done for the earlier developmental stages, which are considered as more sensitive to elevated temperatures. Subsequently, the selection of these two cultivars allowed us to analyse changes occurring in two genotypes with slight pollen thermotolerance differences. In a similar manner, for genome-wide transcriptome analysis, flowering plants were exposed to 39 °C for 1 h and pollen was isolated directly after the treatment as well as after recovery of flowers at 25 °C for 1.5 h. Both pollen samples were combined. By this way we were able to monitor the AS events occurring during HS and during the recovery from HS, as both represent important phases of stress response with distinct transcriptional signatures.<a reveal-id="dsw051-B50" data-open="dsw051-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> To analyse the HSR, pollen from plants kept at 25 °C were isolated and analysed. This sample is annotated as control (CO).</p><p>RNA-seq libraries from two biological replicates were constructed for HS and CO conditions for both cultivars. After sequencing, all libraries were aligned to the tomato genome of cultivar Heinz with the spliced aligner TopHat2.<a reveal-id="dsw051-B37" data-open="dsw051-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> In contrast to <em>de novo</em> or reference based transcriptome assembly,<a reveal-id="dsw051-B35" data-open="dsw051-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> the alignment to a reference genome allows the expression analysis and the detection of AS events for annotated genes. A high Pearson correlation between the read counts of biological replicates (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S4</a></span>), allowed us to merge their alignments into a single file leading to four samples to be analysed (Moneymaker CO, Moneymaker HS, Red Setter CO and Red Setter HS) with a total amount of more than 280 million reads each (<a reveal-id="dsw051-T1" data-open="dsw051-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The rate of read alignment slightly differed between the four libraries ranging from 42% for Moneymaker-CO to 61% for Red Setter-HS. Reads witnessing an IR, ES, AA or AD event were traced in the alignment files. <div content-id="dsw051-T1" class="table-modal"><div class="table-wrap"><div id="dsw051-T1" data-id="dsw051-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p> Statistics of mRNA-Seq libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="2">Moneymaker<hr /> </th><th colspan="2">Red Setter<hr /> </th></tr><tr><th>CO </th><th>HS </th><th>CO </th><th>HS </th></tr></thead><tbody><tr><td><strong>NuR</strong> </td><td>285,468,115 </td><td>329,165,337 </td><td>280,177,153 </td><td>351,372,546 </td></tr><tr><td><strong>AR</strong> </td><td>120,969,297 </td><td>192,550,586 </td><td>160,831,268 </td><td>213,563,554 </td></tr><tr><td><strong>%</strong> </td><td>42.4 </td><td>58.5 </td><td>57.4 </td><td>60.8 </td></tr></tbody></table></div><div content-id="dsw051-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="2">Moneymaker<hr /> </th><th colspan="2">Red Setter<hr /> </th></tr><tr><th>CO </th><th>HS </th><th>CO </th><th>HS </th></tr></thead><tbody><tr><td><strong>NuR</strong> </td><td>285,468,115 </td><td>329,165,337 </td><td>280,177,153 </td><td>351,372,546 </td></tr><tr><td><strong>AR</strong> </td><td>120,969,297 </td><td>192,550,586 </td><td>160,831,268 </td><td>213,563,554 </td></tr><tr><td><strong>%</strong> </td><td>42.4 </td><td>58.5 </td><td>57.4 </td><td>60.8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Given is the total number of reads (NuR), of aligned reads (AR) and the percentage ARs (%) for Moneymaker and Red Setter HS and CO libraries.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw051-T1" data-id="dsw051-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p> Statistics of mRNA-Seq libraries</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="2">Moneymaker<hr /> </th><th colspan="2">Red Setter<hr /> </th></tr><tr><th>CO </th><th>HS </th><th>CO </th><th>HS </th></tr></thead><tbody><tr><td><strong>NuR</strong> </td><td>285,468,115 </td><td>329,165,337 </td><td>280,177,153 </td><td>351,372,546 </td></tr><tr><td><strong>AR</strong> </td><td>120,969,297 </td><td>192,550,586 </td><td>160,831,268 </td><td>213,563,554 </td></tr><tr><td><strong>%</strong> </td><td>42.4 </td><td>58.5 </td><td>57.4 </td><td>60.8 </td></tr></tbody></table></div><div content-id="dsw051-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="2">Moneymaker<hr /> </th><th colspan="2">Red Setter<hr /> </th></tr><tr><th>CO </th><th>HS </th><th>CO </th><th>HS </th></tr></thead><tbody><tr><td><strong>NuR</strong> </td><td>285,468,115 </td><td>329,165,337 </td><td>280,177,153 </td><td>351,372,546 </td></tr><tr><td><strong>AR</strong> </td><td>120,969,297 </td><td>192,550,586 </td><td>160,831,268 </td><td>213,563,554 </td></tr><tr><td><strong>%</strong> </td><td>42.4 </td><td>58.5 </td><td>57.4 </td><td>60.8 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Given is the total number of reads (NuR), of aligned reads (AR) and the percentage ARs (%) for Moneymaker and Red Setter HS and CO libraries.</p></div></div></div></div></p><div class="block-child-p">We compared the alignment position with the exon-intron structures of the gene annotation in the <em>S</em><em>.</em><em>lycopersicum</em> cv. Heinz genome (ITAG2.4; <a class="link link-uri" href="http://solgenomics.net/" target="_blank">http://solgenomics.net/</a>; cv. Heinz 1706) (<a reveal-id="dsw051-F1" data-open="dsw051-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). An IR was assumed to be present if the respective intron was covered at each position by at least one read. An ES was assigned if a read was aligned to two exons excluding another exon or even more exons in between. In the same line, we annotated an AA and AD in case that a read was aligned to two exons, but either the donor or acceptor site differed from that deposited one in the database. An ES, AD and AA event was considered significant when the NuR witnessing the AS event exceeded the average genome coverage of the library. <div data-id="dsw051-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051f1.png?Expires=2147483647&Signature=w6e~4H-ENgcLAbIVgOvreHpS53s5Ad3tQGV921dqNGXf590TD6uV-fnt7nelk~4cKcNMn-fXe6Xt3VZlQeMTzqsBy1caihpGIngArR4AipsT7H2DnuN7f5qbA-TaIoe6zUIngw1ZiRExCbKwTB3~ITWsRL8US0vuX5HfFIjjjJj6f7f1V0O9GzA2FVko4HT6liZNih27BvUuaxtkvFeyOkqHe-ulQxvLNuoAM35NLYaWjZGvjNsnfFt3tfydoPXlSHFL7LbGrSZUlY~labiWmlaZtWPMLEZQUBe1awkMXWBtt3~XuIptJA3VdSZidr4lDMiQDymh0NF2Z2J0BG1c~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="AS events in pollen of different tomato cultivars exposed to HS. (A) Schematic representation for detection of AS events using reads witnessing a particular event. Detected events are IRs, ESs, AD sites and AA sites. (B and C) representing the number of genes showing particular AS events (IR, ES, AA and AD) under CO, HS or both conditions (Overlap) for the cultivars Moneymaker (B) and Red Setter (C)." path-from-xml="dsw051f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78054928/dsw051f1.tif" path-from-xml="dsw051f1.tif" target="_blank">View large</a><a section="78054928" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051f1.png?Expires=2147483647&Signature=CvXRxjIZ9bOr8dWpSTTK1V-TH-P4jy0fgqeeTfToPj-dFxPlpETc~Abn8pvSz5CA9GVQxZWJE-B7onvs7EAucEBt-W7GV0z~4cxbszw~AvcroVTI0m6Ml95~N7fHwSqECpcc84xZfpYxSvID8JxVx3QOCR1~Qz~xFbO3rNZ0-L-MBk2VcuViP1pdMOtRddxTqrQWapv1Wa66b5ZS7NFOxZ~15SpbJLU60MzZze-wECyZz8dCENdgy4VtPlUEfu1pvOFN0fJFCKpDAntWI-wpxm5NS65ERF8fk76Qp64-V6b0AquujbXyqtPxO3gCiAX8SQDC91brsnRtoe-iKvwAmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78054928&ar=2738679&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw051f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>AS events in pollen of different tomato cultivars exposed to HS. <strong>(A)</strong> Schematic representation for detection of AS events using reads witnessing a particular event. Detected events are IRs, ESs, AD sites and AA sites. <strong>(B and C)</strong> representing the number of genes showing particular AS events (IR, ES, AA and AD) under CO, HS or both conditions (Overlap) for the cultivars Moneymaker (<strong>B</strong>) and Red Setter (<strong>C</strong>).</p></div></div><div content-id="dsw051-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051f1.png?Expires=2147483647&Signature=w6e~4H-ENgcLAbIVgOvreHpS53s5Ad3tQGV921dqNGXf590TD6uV-fnt7nelk~4cKcNMn-fXe6Xt3VZlQeMTzqsBy1caihpGIngArR4AipsT7H2DnuN7f5qbA-TaIoe6zUIngw1ZiRExCbKwTB3~ITWsRL8US0vuX5HfFIjjjJj6f7f1V0O9GzA2FVko4HT6liZNih27BvUuaxtkvFeyOkqHe-ulQxvLNuoAM35NLYaWjZGvjNsnfFt3tfydoPXlSHFL7LbGrSZUlY~labiWmlaZtWPMLEZQUBe1awkMXWBtt3~XuIptJA3VdSZidr4lDMiQDymh0NF2Z2J0BG1c~w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="AS events in pollen of different tomato cultivars exposed to HS. (A) Schematic representation for detection of AS events using reads witnessing a particular event. Detected events are IRs, ESs, AD sites and AA sites. (B and C) representing the number of genes showing particular AS events (IR, ES, AA and AD) under CO, HS or both conditions (Overlap) for the cultivars Moneymaker (B) and Red Setter (C)." path-from-xml="dsw051f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78054928/dsw051f1.tif" path-from-xml="dsw051f1.tif" target="_blank">View large</a><a section="78054928" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051f1.png?Expires=2147483647&Signature=CvXRxjIZ9bOr8dWpSTTK1V-TH-P4jy0fgqeeTfToPj-dFxPlpETc~Abn8pvSz5CA9GVQxZWJE-B7onvs7EAucEBt-W7GV0z~4cxbszw~AvcroVTI0m6Ml95~N7fHwSqECpcc84xZfpYxSvID8JxVx3QOCR1~Qz~xFbO3rNZ0-L-MBk2VcuViP1pdMOtRddxTqrQWapv1Wa66b5ZS7NFOxZ~15SpbJLU60MzZze-wECyZz8dCENdgy4VtPlUEfu1pvOFN0fJFCKpDAntWI-wpxm5NS65ERF8fk76Qp64-V6b0AquujbXyqtPxO3gCiAX8SQDC91brsnRtoe-iKvwAmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78054928&ar=2738679&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw051f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>AS events in pollen of different tomato cultivars exposed to HS. <strong>(A)</strong> Schematic representation for detection of AS events using reads witnessing a particular event. Detected events are IRs, ESs, AD sites and AA sites. <strong>(B and C)</strong> representing the number of genes showing particular AS events (IR, ES, AA and AD) under CO, HS or both conditions (Overlap) for the cultivars Moneymaker (<strong>B</strong>) and Red Setter (<strong>C</strong>).</p></div></div></div><p>For each of the four AS events the number of genes showing this particular event was determined, where multiple occurrences of the event in a single gene were counted only once. The number of genes with a particular IR and ES event is clearly different between the CO and HS conditions. We observed 8,415 genes with IR under HS, and 4,578 genes with IR under CO in Moneymaker (<a reveal-id="dsw051-F1" data-open="dsw051-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). About 87% of the genes with IR found at CO also showed an IR under HS (<a reveal-id="dsw051-F1" data-open="dsw051-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>, grey bar). In Red Setter we observed only a slight increase of genes with IR at HS (8,795 genes) when compared with CO (8,394 genes), while the total number of genes with IR under HS conditions is rather comparable in both cultivars. We realized that IR is by far the most common AS event (<a reveal-id="dsw051-F1" data-open="dsw051-F1" class="link link-reveal link-table xref-fig">Fig. 1B and C</a>). Furthermore, an increase of genes with ES from 684/612 under CO to 1,225/901 genes under HS was observed in Moneymaker and Red Setter, respectively. In contrast the number of genes with AA and AD did not increase upon HS in the two cultivars. Remarkably, 54 and 61% of all genes with ES under CO and HS occur in both cultivars. Further, 81/89% of all genes with IR in Moneymaker were also present in Red Setter under control or HS conditions. These findings demonstrate that AS is a general temperature-dependent response, while cultivar-specific AS exists depending on the AS event and condition (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S5</a></span>).</p>                            <h3 scrollto-destination=78054930 id="78054930" class="section-title" >3.2. HS-dependent IR and ES events in tomato pollen</h3>
<p>The increase of genes with IR and ES events in tomato pollen upon HS might indicate heat-sensitive IR and ES. Next, we checked whether the different AS events are related with the abundance of expression of the respective genes. For this purpose we determined the FPKM normalized expression of all genes and categorized them with respect to their AS behavior, meaning AS under both, one or none of the conditions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S6</a></span>). In total, 23,868 (Moneymaker) and 25,314 (Red Setter) genes showed an FPKM &gt; 1 in both conditions (CO and HS), respectively. For the majority of these genes (58.7 and 56.8%) we did not identify reads that would indicate AS events. In turn, for 22.3 and 28.8% of all genes reads were identified, which suggest the existence of alternatively spliced transcripts under both conditions in Moneymaker and Red Setter, respectively. Differences between cultivars were observed for genes showing AS under a single condition. For Moneymaker we found with 15.7% a five times higher amount of genes with alternatively spliced transcripts under HS compared with CO (3.25%). For Red Setter we realized nearly similar amounts of alternatively spliced transcripts under HS (7.6%) and CO (6.7%). Notably, the identification of AS was not dependent on FPKM expression of a certain gene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S6</a></span>). For verification of FPKM expression levels, these were correlated with expression levels obtained from qPCR. Results revealed for both Moneymaker and Red Setter correlation higher than 0.97 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S7</a></span>). Furthermore, we aimed to rule out a potential dependency of detected IR events on the number of introns comprised in a gene. Therefore, genes were examined for their intron density and the amount of retained introns (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S8</a></span>). Although a small number of individual genes showed a higher degree of retained introns with up to 12 IR events (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S8</a></span>, white scatter plot), the general tendency revealed an accumulation of IR between zero and three IR events. This observation was irrespective of the number of introns contained in a gene, where Moneymaker CO showed the lowest abundance with the majority of genes having only up to one IR event (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S8</a></span>, contour plot). In addition, we compared these observations to a model supporting a dependency between the amount of IR events and the number of introns in a gene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S8</a></span>, red scatter plot), but no relationship could be detected. These results confirm that detection of AS is neither dependent on the expression of the underlying gene nor on the composition of the gene and by this shows that AS is a controlled mechanism and no side effect.</p><div class="block-child-p">Consequently, we aimed to quantify the alteration of IR and ES between the two conditions at intron and exon resolution, respectively. For this purpose, the IR frequency of each intron and the ES frequency of each exon were normalized to the library size and the expression of the underlying gene (Formula 1). The log<sub>2</sub> value of the normalized frequency under HS divided by the normalized frequency under CO serves as a measure of HS-dependent accumulation (positive log<sub>2</sub> ratio) or reduction (negative log<sub>2</sub> ratio) for IR and ES (<a reveal-id="dsw051-F2" data-open="dsw051-F2" class="link link-reveal link-table xref-fig">Fig. 2A and B</a>). Interestingly, the fold change distribution of IR (light and dark blue) and ES frequency (light and dark orange) for both cultivars represents a normal distribution. In total, 4,331/8,275 IR and 553/618 ES events could be detected in both conditions for Moneymaker (light color) and Red Setter (dark color), respectively. Thus, we defined for each cultivar the IR and ES events of the upper (accumulation) and lower (reduction) 5% quantile of the respective IR and ES distribution as HS-dependent (<a reveal-id="dsw051-F2" data-open="dsw051-F2" class="link link-reveal link-table xref-fig">Fig. 2A and B</a>). In addition, 10,882 and 7,334 IR events as well as 1,178 and 735 ES events solely occur under HS in Moneymaker or Red Setter, respectively (<a reveal-id="dsw051-F2" data-open="dsw051-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). These events were added to the respective set of IR and ES events accumulating under HS (in total: 11,099 IR events in Moneymaker and 7,748 IR events in Red Setter; 1,206 ES events in Moneymaker and 766 ES events in Red Setter). Similarly, the 1,625/5,224 IR and 309/243 ES events exclusively found under CO condition in Moneymaker or Red Setter were added to the respective set of IR and ES events reduced under HS (in total: 1,842 IR events in Moneymaker and 5,638 IR events in Red Setter; 337 ES events in Moneymaker and 274 ES events in Red Setter). This demonstrates that induction of AS events like IR and ES is a more abundant regulatory mode in HSR than the reduction of AS events, irrespective of the cultivar analysed. <div data-id="dsw051-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051f2.png?Expires=2147483647&Signature=2zZNdlJI2S3WeKCR1SJe0Jwahmh50fTWQlVDyE~su7M0tXbiarDTI2nerwXsNekecYerxcleBYsW7wkUey-AnoLK7fGkCo4oF44C6FU0r6RA5RfQ5QOxy5lUBXZR1jNTpnSsdvokOSe-~jsn35F4xziGvTA-2tpzoSyv8MvNUL1l9eLXw-IqexI-dM0XQhdv6BCnaPKfB51uz7hBW9Q08QOlTBn41jaHxSEoMm0C9S43qwmSCE7Vy2SY-NZlQSlMV2sbqb8z2yVMmbieHitfHFusGWKgv5Dtn4XSI9Zf05bpsVMyAfOouaDIFcc0B55Mqx0eAoYgj5IRUKQtNpkmQQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of IR and ES events accumulated or reduced under HS. (A and B) show the distribution of HS-dependent accumulated (log2 ratio &gt; 0) and HS-dependent reduced (log2 ratio &lt; 0) IR (light and dark blue) and ES (light and dark orange) events. The log2 ratio is based on normalized event frequencies between HS and CO samples in Moneymaker (A) and Red Setter (B). Given in C is the number of IR and ES solely detected under HS or CO condition for both Moneymaker and Red Setter." path-from-xml="dsw051f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78054932/dsw051f2.tif" path-from-xml="dsw051f2.tif" target="_blank">View large</a><a section="78054932" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051f2.png?Expires=2147483647&Signature=szBt1KYfaC-2bMProWlLUCiiBp1hN4qUKcDxZTC7n8P3B~Z-EmMS5Baa7CidwEY898tt~9vzDiJkwY5dF4aun1PpNXgLjpq9Gau-GkCJQoSvQ3p0HrsY3hAOLxgH8rdJCIJ2sf1JvJR6e9J02eJIhA1~pDvkdkl6lD4brFQq055VK0-wPmgOtqByI7A3vMKpJJneu37cq3mOtSnPiCZp2fNaN64V5--3kkjGGku4-dauaW4EVRWPrI5EC3KxOjym6kr3LBorqkrZFAXeSYGs0N-1YN-MBeEbOeUW31Rz6p47G9KHi9tYzTR5VkPrPOxpV3l4xN1sVuF86RKe08d32Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78054932&ar=2738679&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw051f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of IR and ES events accumulated or reduced under HS. <strong>(A and B)</strong> show the distribution of HS-dependent accumulated (log2 ratio &gt; 0) and HS-dependent reduced (log2 ratio &lt; 0) IR (light and dark blue) and ES (light and dark orange) events. The log2 ratio is based on normalized event frequencies between HS and CO samples in Moneymaker (A) and Red Setter (B). Given in C is the number of IR and ES solely detected under HS or CO condition for both Moneymaker and Red Setter.</p></div></div><div content-id="dsw051-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051f2.png?Expires=2147483647&Signature=2zZNdlJI2S3WeKCR1SJe0Jwahmh50fTWQlVDyE~su7M0tXbiarDTI2nerwXsNekecYerxcleBYsW7wkUey-AnoLK7fGkCo4oF44C6FU0r6RA5RfQ5QOxy5lUBXZR1jNTpnSsdvokOSe-~jsn35F4xziGvTA-2tpzoSyv8MvNUL1l9eLXw-IqexI-dM0XQhdv6BCnaPKfB51uz7hBW9Q08QOlTBn41jaHxSEoMm0C9S43qwmSCE7Vy2SY-NZlQSlMV2sbqb8z2yVMmbieHitfHFusGWKgv5Dtn4XSI9Zf05bpsVMyAfOouaDIFcc0B55Mqx0eAoYgj5IRUKQtNpkmQQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of IR and ES events accumulated or reduced under HS. (A and B) show the distribution of HS-dependent accumulated (log2 ratio &gt; 0) and HS-dependent reduced (log2 ratio &lt; 0) IR (light and dark blue) and ES (light and dark orange) events. The log2 ratio is based on normalized event frequencies between HS and CO samples in Moneymaker (A) and Red Setter (B). Given in C is the number of IR and ES solely detected under HS or CO condition for both Moneymaker and Red Setter." path-from-xml="dsw051f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78054932/dsw051f2.tif" path-from-xml="dsw051f2.tif" target="_blank">View large</a><a section="78054932" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051f2.png?Expires=2147483647&Signature=szBt1KYfaC-2bMProWlLUCiiBp1hN4qUKcDxZTC7n8P3B~Z-EmMS5Baa7CidwEY898tt~9vzDiJkwY5dF4aun1PpNXgLjpq9Gau-GkCJQoSvQ3p0HrsY3hAOLxgH8rdJCIJ2sf1JvJR6e9J02eJIhA1~pDvkdkl6lD4brFQq055VK0-wPmgOtqByI7A3vMKpJJneu37cq3mOtSnPiCZp2fNaN64V5--3kkjGGku4-dauaW4EVRWPrI5EC3KxOjym6kr3LBorqkrZFAXeSYGs0N-1YN-MBeEbOeUW31Rz6p47G9KHi9tYzTR5VkPrPOxpV3l4xN1sVuF86RKe08d32Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78054932&ar=2738679&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw051f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of IR and ES events accumulated or reduced under HS. <strong>(A and B)</strong> show the distribution of HS-dependent accumulated (log2 ratio &gt; 0) and HS-dependent reduced (log2 ratio &lt; 0) IR (light and dark blue) and ES (light and dark orange) events. The log2 ratio is based on normalized event frequencies between HS and CO samples in Moneymaker (A) and Red Setter (B). Given in C is the number of IR and ES solely detected under HS or CO condition for both Moneymaker and Red Setter.</p></div></div></div>                            <h3 scrollto-destination=78054933 id="78054933" class="section-title" >3.3. Processes affected by HS-dependent IR and ES events in tomato pollen</h3>
<div class="block-child-p">After the global analysis of splicing events we aimed at the definition of the global network of genes regulated by AS under HS irrespective of genotype specificities. Thus, we searched for genes regulated by the same type of AS event (IR or ES) in both cultivars under HS or CO conditions, respectively. To this end, we collected all genes for which at least one IR or ES event shows HS-dependent accumulation or HS-dependent reduction and determined the overlap between both cultivars. Approximately 7,500 genes with HS-dependent accumulation of IR and ES were detected (<a reveal-id="dsw051-F3" data-open="dsw051-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). For ∼3,400 genes at least one IR or one ES event occurred in both cultivars (<a reveal-id="dsw051-F3" data-open="dsw051-F3" class="link link-reveal link-table xref-fig">Fig. 3A;</a> white numerals). The majority of these genes (2,731) contained only IR events in both cultivars. HS-dependent reduction of IR and/or ES was found in 4,846 genes (<a reveal-id="dsw051-F3" data-open="dsw051-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). Most genes contained an IR solely in Red Setter (2,974 genes). In contrast, only 19% of the genes shared at least the same AS event type in both cultivars (<a reveal-id="dsw051-F3" data-open="dsw051-F3" class="link link-reveal link-table xref-fig">Fig. 3B;</a> white numerals). Thus, we postulate that the HS-dependent accumulation of IR and ES appears to be more global, while the reduction seems to be more cultivar specific (<a reveal-id="dsw051-F3" data-open="dsw051-F3" class="link link-reveal link-table xref-fig">Fig. 3A and B</a>). <div data-id="dsw051-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051f3.png?Expires=2147483647&Signature=bJMWo9YTPdDiw3CUuz53t-RoeXw2SX~WCrLlIDLOLJ5TQ~YuYmqg7kiStYZWPmoNsIxEXk-BEA2KxSxWke436I9s9sz8QRvzvMAvGXd1MBz8S5ufG0G0FBp~nHw1c1GfB0yCsPT8S~ECYTdh-toWk9S~8OGotLJbMm3Y8Mz9qejA7VK9MDIo1-j4El41RkfVThOfAj2HhI6Q959gqwIqts7GepIKTgUQEaUhzysJxDok1zwxp~3IK0F9cYjXPcpLKfpXHWD4bQ~pTS9n~J~JrlFx6KDnvtpJN2LFEYVgOkAiFDNeEi02gG~T8BLVNlkQDkHZPLRoMdihLMIIDJXEvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genes showing HS-dependent accumulation or reduction of AS in pollen. (A) and (B) display Venn diagrams representing the overlap of IR and ES containing genes showing HS-dependent accumulation (A) or reduction (B). Genes featuring IR (light and dark blue) and/or ES (light and dark orange) for the cultivars Moneymaker and Red Setter are shown. Numbers in white and bold lines indicate genes showing at least one common IR or ES event in both cultivars." path-from-xml="dsw051f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78054934/dsw051f3.tif" path-from-xml="dsw051f3.tif" target="_blank">View large</a><a section="78054934" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051f3.png?Expires=2147483647&Signature=OmDHkuB10NOqIw9KigDPHVkZEVdYMsuwh-lFDazXASc5hN3nLatB12reWuFCAf4Vs6zfTabDUpeZl12hktVYYTY9F2tGgunjoKXCijOC5OjLrFTwOe4YONTo-qdc-cXyqS9UNCNKUEbQO-jBGJWHjVjjEATpgMx78y0wrBrZq9IgbofPwmKjSvJCRh6HvBmSzOxu9xnnUN4JMpIQ1-pGWFS19dWX8mz~lGn2wSJcsX6cWPLQNVbNqhJ0S~A4~NjHcg~LKp~I76T6dv73OFzVYczEH7AYTaenAiEncBY6gi7GPCjH5~0Z12ipYySer3Y4xipKsNQwHtYaGEkailYNRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78054934&ar=2738679&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw051f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genes showing HS-dependent accumulation or reduction of AS in pollen. <strong>(A)</strong> and <strong>(B)</strong> display Venn diagrams representing the overlap of IR and ES containing genes showing HS-dependent accumulation (A) or reduction (B). Genes featuring IR (light and dark blue) and/or ES (light and dark orange) for the cultivars Moneymaker and Red Setter are shown. Numbers in white and bold lines indicate genes showing at least one common IR or ES event in both cultivars.</p></div></div><div content-id="dsw051-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051f3.png?Expires=2147483647&Signature=bJMWo9YTPdDiw3CUuz53t-RoeXw2SX~WCrLlIDLOLJ5TQ~YuYmqg7kiStYZWPmoNsIxEXk-BEA2KxSxWke436I9s9sz8QRvzvMAvGXd1MBz8S5ufG0G0FBp~nHw1c1GfB0yCsPT8S~ECYTdh-toWk9S~8OGotLJbMm3Y8Mz9qejA7VK9MDIo1-j4El41RkfVThOfAj2HhI6Q959gqwIqts7GepIKTgUQEaUhzysJxDok1zwxp~3IK0F9cYjXPcpLKfpXHWD4bQ~pTS9n~J~JrlFx6KDnvtpJN2LFEYVgOkAiFDNeEi02gG~T8BLVNlkQDkHZPLRoMdihLMIIDJXEvQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genes showing HS-dependent accumulation or reduction of AS in pollen. (A) and (B) display Venn diagrams representing the overlap of IR and ES containing genes showing HS-dependent accumulation (A) or reduction (B). Genes featuring IR (light and dark blue) and/or ES (light and dark orange) for the cultivars Moneymaker and Red Setter are shown. Numbers in white and bold lines indicate genes showing at least one common IR or ES event in both cultivars." path-from-xml="dsw051f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78054934/dsw051f3.tif" path-from-xml="dsw051f3.tif" target="_blank">View large</a><a section="78054934" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051f3.png?Expires=2147483647&Signature=OmDHkuB10NOqIw9KigDPHVkZEVdYMsuwh-lFDazXASc5hN3nLatB12reWuFCAf4Vs6zfTabDUpeZl12hktVYYTY9F2tGgunjoKXCijOC5OjLrFTwOe4YONTo-qdc-cXyqS9UNCNKUEbQO-jBGJWHjVjjEATpgMx78y0wrBrZq9IgbofPwmKjSvJCRh6HvBmSzOxu9xnnUN4JMpIQ1-pGWFS19dWX8mz~lGn2wSJcsX6cWPLQNVbNqhJ0S~A4~NjHcg~LKp~I76T6dv73OFzVYczEH7AYTaenAiEncBY6gi7GPCjH5~0Z12ipYySer3Y4xipKsNQwHtYaGEkailYNRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78054934&ar=2738679&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw051f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genes showing HS-dependent accumulation or reduction of AS in pollen. <strong>(A)</strong> and <strong>(B)</strong> display Venn diagrams representing the overlap of IR and ES containing genes showing HS-dependent accumulation (A) or reduction (B). Genes featuring IR (light and dark blue) and/or ES (light and dark orange) for the cultivars Moneymaker and Red Setter are shown. Numbers in white and bold lines indicate genes showing at least one common IR or ES event in both cultivars.</p></div></div></div><div class="block-child-p">Finally, we inspected for all genes found to be modulated by IR or ES whether at least one event targets the same intron or exon in both cultivars. In total, we observed 2,343 genes for which the same AS event shows HS-dependent accumulation and 451 genes with the same HS-dependent AS reduction event in both cultivars. Subsequently, enrichment of GO terms was performed to identify the cellular functions that are affected by alterations of IR or ES events in response to HS (<a reveal-id="dsw051-F4" data-open="dsw051-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Interestingly, the analysis of the 451 genes with HS-dependent IR or ES reduction did not lead to the enrichment of any biological category annotated in the GO term database. <div data-id="dsw051-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051f4.png?Expires=2147483647&Signature=iL9uxqz2FoQM0VwPadLc4XbY8zZau9XyrKXYDjBKP11nwkm8J2-uaruCzXnSn7W3oOBsn9eH1CloCb~~2Iwx6uyA8tulwCy9OTE1VOQzvt6vt5bOvcv-nyfJ~RKDb3cfxPgwt-UQNNGQvpE0spMSW~NFcwW-hbBamc4a8bcwuGCp7IkXEtnJ2M1xrANikx8aj4~nCigS8EC03dJnSxRyvudZAJGZ6von1gE7d8pOONm7RA62F7ApyYF91RDrXm0DxFhK-xGMbPm-K673YiEJvaDy3vJsPzQZqRa985yCI9vSo4kZ2nFoa0TDPRl7nt-2ETSw7R4S8ppUoQcfIOT~2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Enriched biological processes due to HS-dependent accumulation of AS. Enriched GO terms (biological process) within the 2,343 genes showing HS-dependent accumulation of AS. Character sizes are dependent on the adjusted P-values (Benjamini and Hochberg FDR correction with a &lt; 0.01 significance level) of the enrichment analysis, whereby bigger character sizes indicate lower p-values. Coloring of the biological process terms is related to their affiliation to functional categories, namely heat response (orange); gene expression (red), biosynthetic processes (green) as well as transport and localization (blue)." path-from-xml="dsw051f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78054935/dsw051f4.tif" path-from-xml="dsw051f4.tif" target="_blank">View large</a><a section="78054935" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051f4.png?Expires=2147483647&Signature=4H69Dub-vwUm1jxqraQTEneiECh-0h2HgHD77hC5dYJRlsre~-swv23u1M9G34ygwHhhsPL86xC3A0uyuXUw1XyOpHYH3C89JSyW6FjXWIyd~~1swvPv~4MiLnZreVAh025OYO1J9yeuVwW6p0gOwN~6K1aTqkR41vXNz3FKtZZAiAtk6BZkOzcLeYqDuTFdfhvYCqq2zimXocxevLb2sMoBOyOgdW2ISgDvKQgK7ef-g-IBYrt9~Pyo9ReIo-wzGoFnSOOAU2PdwUXHrztm1bfePjRHomlJ2jmSsKzWr1-7hqYgRnqx2krAFK0qwc5w-RaxyNchoOoa51uNaUydUg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78054935&ar=2738679&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw051f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Enriched biological processes due to HS-dependent accumulation of AS. Enriched GO terms (biological process) within the 2,343 genes showing HS-dependent accumulation of AS. Character sizes are dependent on the adjusted <em>P</em>-values (Benjamini and Hochberg FDR correction with a &lt; 0.01 significance level) of the enrichment analysis, whereby bigger character sizes indicate lower p-values. Coloring of the biological process terms is related to their affiliation to functional categories, namely heat response (orange); gene expression (red), biosynthetic processes (green) as well as transport and localization (blue).</p></div></div><div content-id="dsw051-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051f4.png?Expires=2147483647&Signature=iL9uxqz2FoQM0VwPadLc4XbY8zZau9XyrKXYDjBKP11nwkm8J2-uaruCzXnSn7W3oOBsn9eH1CloCb~~2Iwx6uyA8tulwCy9OTE1VOQzvt6vt5bOvcv-nyfJ~RKDb3cfxPgwt-UQNNGQvpE0spMSW~NFcwW-hbBamc4a8bcwuGCp7IkXEtnJ2M1xrANikx8aj4~nCigS8EC03dJnSxRyvudZAJGZ6von1gE7d8pOONm7RA62F7ApyYF91RDrXm0DxFhK-xGMbPm-K673YiEJvaDy3vJsPzQZqRa985yCI9vSo4kZ2nFoa0TDPRl7nt-2ETSw7R4S8ppUoQcfIOT~2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Enriched biological processes due to HS-dependent accumulation of AS. Enriched GO terms (biological process) within the 2,343 genes showing HS-dependent accumulation of AS. Character sizes are dependent on the adjusted P-values (Benjamini and Hochberg FDR correction with a &lt; 0.01 significance level) of the enrichment analysis, whereby bigger character sizes indicate lower p-values. Coloring of the biological process terms is related to their affiliation to functional categories, namely heat response (orange); gene expression (red), biosynthetic processes (green) as well as transport and localization (blue)." path-from-xml="dsw051f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78054935/dsw051f4.tif" path-from-xml="dsw051f4.tif" target="_blank">View large</a><a section="78054935" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051f4.png?Expires=2147483647&Signature=4H69Dub-vwUm1jxqraQTEneiECh-0h2HgHD77hC5dYJRlsre~-swv23u1M9G34ygwHhhsPL86xC3A0uyuXUw1XyOpHYH3C89JSyW6FjXWIyd~~1swvPv~4MiLnZreVAh025OYO1J9yeuVwW6p0gOwN~6K1aTqkR41vXNz3FKtZZAiAtk6BZkOzcLeYqDuTFdfhvYCqq2zimXocxevLb2sMoBOyOgdW2ISgDvKQgK7ef-g-IBYrt9~Pyo9ReIo-wzGoFnSOOAU2PdwUXHrztm1bfePjRHomlJ2jmSsKzWr1-7hqYgRnqx2krAFK0qwc5w-RaxyNchoOoa51uNaUydUg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78054935&ar=2738679&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw051f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Enriched biological processes due to HS-dependent accumulation of AS. Enriched GO terms (biological process) within the 2,343 genes showing HS-dependent accumulation of AS. Character sizes are dependent on the adjusted <em>P</em>-values (Benjamini and Hochberg FDR correction with a &lt; 0.01 significance level) of the enrichment analysis, whereby bigger character sizes indicate lower p-values. Coloring of the biological process terms is related to their affiliation to functional categories, namely heat response (orange); gene expression (red), biosynthetic processes (green) as well as transport and localization (blue).</p></div></div></div><p>HS-dependent IR or ES accumulation was found to be enriched in genes coding for proteins represented by 33 GO terms (<a reveal-id="dsw051-F4" data-open="dsw051-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>), which fall in four general functional categories: (i) gene expression, RNA modification, translation and protein folding (<a reveal-id="dsw051-F4" data-open="dsw051-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, red), (ii) localization and transport (<a reveal-id="dsw051-F4" data-open="dsw051-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, blue), (iii) heat response (<a reveal-id="dsw051-F4" data-open="dsw051-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, orange) and (iv) metabolic related processes (<a reveal-id="dsw051-F4" data-open="dsw051-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, green). One of the most enriched GO terms is protein folding and gene expression. In the category protein folding we identified genes coding for Hsps, namely seven genes coding for Hsp90, four for Hsp70, eight for HSP60, seven for Hsp40 and three for HSP10 (<a reveal-id="dsw051-T2" data-open="dsw051-T2" class="link link-reveal link-table xref-fig">Table 2</a>). With this, we found IR or ES events for all Hsp90 coding genes, 19% of all Hsp70 coding genes, 44% of all HSP60 coding genes, 7% of all Hsp40 coding genes and 75% of all HSP10 coding genes.<a reveal-id="dsw051-B2" data-open="dsw051-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> The category gene expression comprised many transcription factors including six Hsfs (<a reveal-id="dsw051-T2" data-open="dsw051-T2" class="link link-reveal link-table xref-fig">Table 2</a>).<a reveal-id="dsw051-B11" data-open="dsw051-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> The latter is remarkable, as 27 Hsf coding genes exist in the genome of tomato<a reveal-id="dsw051-B11" data-open="dsw051-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> and thus, for 22% a HS-dependent accumulation of a splice variant is found. This observation suggests that AS is involved in the complex regulatory network of HSR.<a reveal-id="dsw051-B12" data-open="dsw051-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><div content-id="dsw051-T2" class="table-modal"><div class="table-wrap"><div id="dsw051-T2" data-id="dsw051-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Genes involved in protein folding and gene expression altered by HS-dependent IR or ES accumulation</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>GO-term </th><th>Protein family </th><th>Protein </th><th>Gene acc. number </th></tr></thead><tbody><tr><td rowspan="29">protein folding </td><td rowspan="7">Hsp90 </td><td>Hsp90-1 </td><td>Solyc12g015880 </td></tr><tr><td>Hsp90-2 </td><td>Solyc07g065840 </td></tr><tr><td>Hsp90-3 </td><td>Solyc06g036290 </td></tr><tr><td>Hsp90-4 </td><td>Solyc03g007890 </td></tr><tr><td>Hsp90-5 </td><td>Solyc05g010670 </td></tr><tr><td>Hsp90-6 </td><td>Solyc07g047790 </td></tr><tr><td>Hsp90-7 </td><td>Solyc04g081570 </td></tr><tr><td rowspan="4">Hsp70 </td><td>Hsp70-9 </td><td>Solyc11g020040 </td></tr><tr><td>Hsp70-11 </td><td>Solyc01g103450 </td></tr><tr><td>Hsp70-12 </td><td>Solyc01g106210 </td></tr><tr><td>Hsp70-13 </td><td>Solyc01g106260 </td></tr><tr><td rowspan="8">HSP60 </td><td>Cpn60 (1) </td><td>Solyc05g053470 </td></tr><tr><td>Cpn60 (2) </td><td>Solyc09g091180 </td></tr><tr><td>Cpn60 (3) </td><td>Solyc03g121640 </td></tr><tr><td>Cpn60-α (2) </td><td>Solyc11g069790 </td></tr><tr><td>Cpn60-β (1) </td><td>Solyc01g028810 </td></tr><tr><td>Cpn60-β (2) </td><td>Solyc03g120850 </td></tr><tr><td>CCT-γ </td><td>Solyc05g056310 </td></tr><tr><td>CCT-ε </td><td>Solyc05g013990 </td></tr><tr><td rowspan="7">Hsp40 </td><td>DnaJ-A </td><td>Solyc01g105340 </td></tr><tr><td>DnaJ-A </td><td>Solyc01g086740 </td></tr><tr><td>DnaJ-A </td><td>Solyc11g071830 </td></tr><tr><td>DnaJ-A </td><td>Solyc03g118500 </td></tr><tr><td>DnaJ-A </td><td>Solyc01g090550 </td></tr><tr><td>DnaJ-B </td><td>Solyc02g077670 </td></tr><tr><td>DnaJ-B </td><td>Solyc01g079610 </td></tr><tr><td rowspan="3">HSP10 </td><td>Cpn10-P </td><td>Solyc01g088610 </td></tr><tr><td>Cpn10 (1)-M </td><td>Solyc05g056390 </td></tr><tr><td>Cpn21-P </td><td>Solyc07g042250 </td></tr><tr><td rowspan="6">transcription factor </td><td rowspan="6">Hsfs </td><td>HsfA1b </td><td>Solyc03g097120 </td></tr><tr><td>HsfA2 </td><td>Solyc08g062960 </td></tr><tr><td>HsfA3 </td><td>Solyc09g009100 </td></tr><tr><td>HsfA7 </td><td>Solyc09g065660 </td></tr><tr><td>HsfB2a </td><td>Solyc03g026020 </td></tr><tr><td>HsfB1 </td><td>Solyc02g090820 </td></tr></tbody></table></div><div content-id="dsw051-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>GO-term </th><th>Protein family </th><th>Protein </th><th>Gene acc. number </th></tr></thead><tbody><tr><td rowspan="29">protein folding </td><td rowspan="7">Hsp90 </td><td>Hsp90-1 </td><td>Solyc12g015880 </td></tr><tr><td>Hsp90-2 </td><td>Solyc07g065840 </td></tr><tr><td>Hsp90-3 </td><td>Solyc06g036290 </td></tr><tr><td>Hsp90-4 </td><td>Solyc03g007890 </td></tr><tr><td>Hsp90-5 </td><td>Solyc05g010670 </td></tr><tr><td>Hsp90-6 </td><td>Solyc07g047790 </td></tr><tr><td>Hsp90-7 </td><td>Solyc04g081570 </td></tr><tr><td rowspan="4">Hsp70 </td><td>Hsp70-9 </td><td>Solyc11g020040 </td></tr><tr><td>Hsp70-11 </td><td>Solyc01g103450 </td></tr><tr><td>Hsp70-12 </td><td>Solyc01g106210 </td></tr><tr><td>Hsp70-13 </td><td>Solyc01g106260 </td></tr><tr><td rowspan="8">HSP60 </td><td>Cpn60 (1) </td><td>Solyc05g053470 </td></tr><tr><td>Cpn60 (2) </td><td>Solyc09g091180 </td></tr><tr><td>Cpn60 (3) </td><td>Solyc03g121640 </td></tr><tr><td>Cpn60-α (2) </td><td>Solyc11g069790 </td></tr><tr><td>Cpn60-β (1) </td><td>Solyc01g028810 </td></tr><tr><td>Cpn60-β (2) </td><td>Solyc03g120850 </td></tr><tr><td>CCT-γ </td><td>Solyc05g056310 </td></tr><tr><td>CCT-ε </td><td>Solyc05g013990 </td></tr><tr><td rowspan="7">Hsp40 </td><td>DnaJ-A </td><td>Solyc01g105340 </td></tr><tr><td>DnaJ-A </td><td>Solyc01g086740 </td></tr><tr><td>DnaJ-A </td><td>Solyc11g071830 </td></tr><tr><td>DnaJ-A </td><td>Solyc03g118500 </td></tr><tr><td>DnaJ-A </td><td>Solyc01g090550 </td></tr><tr><td>DnaJ-B </td><td>Solyc02g077670 </td></tr><tr><td>DnaJ-B </td><td>Solyc01g079610 </td></tr><tr><td rowspan="3">HSP10 </td><td>Cpn10-P </td><td>Solyc01g088610 </td></tr><tr><td>Cpn10 (1)-M </td><td>Solyc05g056390 </td></tr><tr><td>Cpn21-P </td><td>Solyc07g042250 </td></tr><tr><td rowspan="6">transcription factor </td><td rowspan="6">Hsfs </td><td>HsfA1b </td><td>Solyc03g097120 </td></tr><tr><td>HsfA2 </td><td>Solyc08g062960 </td></tr><tr><td>HsfA3 </td><td>Solyc09g009100 </td></tr><tr><td>HsfA7 </td><td>Solyc09g065660 </td></tr><tr><td>HsfB2a </td><td>Solyc03g026020 </td></tr><tr><td>HsfB1 </td><td>Solyc02g090820 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsw051-T2" data-id="dsw051-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Genes involved in protein folding and gene expression altered by HS-dependent IR or ES accumulation</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>GO-term </th><th>Protein family </th><th>Protein </th><th>Gene acc. number </th></tr></thead><tbody><tr><td rowspan="29">protein folding </td><td rowspan="7">Hsp90 </td><td>Hsp90-1 </td><td>Solyc12g015880 </td></tr><tr><td>Hsp90-2 </td><td>Solyc07g065840 </td></tr><tr><td>Hsp90-3 </td><td>Solyc06g036290 </td></tr><tr><td>Hsp90-4 </td><td>Solyc03g007890 </td></tr><tr><td>Hsp90-5 </td><td>Solyc05g010670 </td></tr><tr><td>Hsp90-6 </td><td>Solyc07g047790 </td></tr><tr><td>Hsp90-7 </td><td>Solyc04g081570 </td></tr><tr><td rowspan="4">Hsp70 </td><td>Hsp70-9 </td><td>Solyc11g020040 </td></tr><tr><td>Hsp70-11 </td><td>Solyc01g103450 </td></tr><tr><td>Hsp70-12 </td><td>Solyc01g106210 </td></tr><tr><td>Hsp70-13 </td><td>Solyc01g106260 </td></tr><tr><td rowspan="8">HSP60 </td><td>Cpn60 (1) </td><td>Solyc05g053470 </td></tr><tr><td>Cpn60 (2) </td><td>Solyc09g091180 </td></tr><tr><td>Cpn60 (3) </td><td>Solyc03g121640 </td></tr><tr><td>Cpn60-α (2) </td><td>Solyc11g069790 </td></tr><tr><td>Cpn60-β (1) </td><td>Solyc01g028810 </td></tr><tr><td>Cpn60-β (2) </td><td>Solyc03g120850 </td></tr><tr><td>CCT-γ </td><td>Solyc05g056310 </td></tr><tr><td>CCT-ε </td><td>Solyc05g013990 </td></tr><tr><td rowspan="7">Hsp40 </td><td>DnaJ-A </td><td>Solyc01g105340 </td></tr><tr><td>DnaJ-A </td><td>Solyc01g086740 </td></tr><tr><td>DnaJ-A </td><td>Solyc11g071830 </td></tr><tr><td>DnaJ-A </td><td>Solyc03g118500 </td></tr><tr><td>DnaJ-A </td><td>Solyc01g090550 </td></tr><tr><td>DnaJ-B </td><td>Solyc02g077670 </td></tr><tr><td>DnaJ-B </td><td>Solyc01g079610 </td></tr><tr><td rowspan="3">HSP10 </td><td>Cpn10-P </td><td>Solyc01g088610 </td></tr><tr><td>Cpn10 (1)-M </td><td>Solyc05g056390 </td></tr><tr><td>Cpn21-P </td><td>Solyc07g042250 </td></tr><tr><td rowspan="6">transcription factor </td><td rowspan="6">Hsfs </td><td>HsfA1b </td><td>Solyc03g097120 </td></tr><tr><td>HsfA2 </td><td>Solyc08g062960 </td></tr><tr><td>HsfA3 </td><td>Solyc09g009100 </td></tr><tr><td>HsfA7 </td><td>Solyc09g065660 </td></tr><tr><td>HsfB2a </td><td>Solyc03g026020 </td></tr><tr><td>HsfB1 </td><td>Solyc02g090820 </td></tr></tbody></table></div><div content-id="dsw051-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>GO-term </th><th>Protein family </th><th>Protein </th><th>Gene acc. number </th></tr></thead><tbody><tr><td rowspan="29">protein folding </td><td rowspan="7">Hsp90 </td><td>Hsp90-1 </td><td>Solyc12g015880 </td></tr><tr><td>Hsp90-2 </td><td>Solyc07g065840 </td></tr><tr><td>Hsp90-3 </td><td>Solyc06g036290 </td></tr><tr><td>Hsp90-4 </td><td>Solyc03g007890 </td></tr><tr><td>Hsp90-5 </td><td>Solyc05g010670 </td></tr><tr><td>Hsp90-6 </td><td>Solyc07g047790 </td></tr><tr><td>Hsp90-7 </td><td>Solyc04g081570 </td></tr><tr><td rowspan="4">Hsp70 </td><td>Hsp70-9 </td><td>Solyc11g020040 </td></tr><tr><td>Hsp70-11 </td><td>Solyc01g103450 </td></tr><tr><td>Hsp70-12 </td><td>Solyc01g106210 </td></tr><tr><td>Hsp70-13 </td><td>Solyc01g106260 </td></tr><tr><td rowspan="8">HSP60 </td><td>Cpn60 (1) </td><td>Solyc05g053470 </td></tr><tr><td>Cpn60 (2) </td><td>Solyc09g091180 </td></tr><tr><td>Cpn60 (3) </td><td>Solyc03g121640 </td></tr><tr><td>Cpn60-α (2) </td><td>Solyc11g069790 </td></tr><tr><td>Cpn60-β (1) </td><td>Solyc01g028810 </td></tr><tr><td>Cpn60-β (2) </td><td>Solyc03g120850 </td></tr><tr><td>CCT-γ </td><td>Solyc05g056310 </td></tr><tr><td>CCT-ε </td><td>Solyc05g013990 </td></tr><tr><td rowspan="7">Hsp40 </td><td>DnaJ-A </td><td>Solyc01g105340 </td></tr><tr><td>DnaJ-A </td><td>Solyc01g086740 </td></tr><tr><td>DnaJ-A </td><td>Solyc11g071830 </td></tr><tr><td>DnaJ-A </td><td>Solyc03g118500 </td></tr><tr><td>DnaJ-A </td><td>Solyc01g090550 </td></tr><tr><td>DnaJ-B </td><td>Solyc02g077670 </td></tr><tr><td>DnaJ-B </td><td>Solyc01g079610 </td></tr><tr><td rowspan="3">HSP10 </td><td>Cpn10-P </td><td>Solyc01g088610 </td></tr><tr><td>Cpn10 (1)-M </td><td>Solyc05g056390 </td></tr><tr><td>Cpn21-P </td><td>Solyc07g042250 </td></tr><tr><td rowspan="6">transcription factor </td><td rowspan="6">Hsfs </td><td>HsfA1b </td><td>Solyc03g097120 </td></tr><tr><td>HsfA2 </td><td>Solyc08g062960 </td></tr><tr><td>HsfA3 </td><td>Solyc09g009100 </td></tr><tr><td>HsfA7 </td><td>Solyc09g065660 </td></tr><tr><td>HsfB2a </td><td>Solyc03g026020 </td></tr><tr><td>HsfB1 </td><td>Solyc02g090820 </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=78054937 id="78054937" class="section-title" >3.4. Occurrence of HS-related alternative transcripts</h3>
<p>So far, AS events were analysed independently for each intron and exon without drawing conclusions concerning putative transcripts. Therefore, we performed reference based transcriptome assembly for the 2,343 and 451 genes showing HS-dependent accumulation or reduction of IR and ES events, respectively, in both cultivars (see ‘Materials and methods’ section). In brief, reads aligned to the genomic sequence in the range of 1 kb downstream to 1 kb upstream of the respective genes were collected for all four RNA-seq libraries (Moneymaker CO &amp; HS and Red Setter CO &amp; HS). These reads were used for each individual library as input for the transcriptome assembly. All assembled transcripts were annotated according to the annotation of the Heinz genome (ITAG2.4). For the 451 genes with HS-dependent reduction of AS about 1,400–1,900 transcripts were assembled in the various pools. More than 90% of these transcripts are assigned to gene identifiers (<a reveal-id="dsw051-T3" data-open="dsw051-T3" class="link link-reveal link-table xref-fig">Table 3</a>). The remaining transcripts without assignment to gene identifiers result from reads aligned in proximity to the genes of interest, but are not derived from these. For genes with HS-dependent accumulation of AS ∼6,900 to 9,100 transcripts were established, and again &gt;90% of the transcripts could be assigned to at least one gene of the reference genome (<a reveal-id="dsw051-T3" data-open="dsw051-T3" class="link link-reveal link-table xref-fig">Table 3</a>). <div content-id="dsw051-T3" class="table-modal"><div class="table-wrap"><div id="dsw051-T3" data-id="dsw051-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Transcriptome assembly descriptive statistics</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene set </th><th colspan="4">HS-dependent reduction<hr /> </th><th colspan="4">HS-dependent accumulation<hr /> </th></tr><tr><th rowspan="2"><strong>Library</strong> </th><th colspan="2"><strong>Moneymaker</strong> </th><th colspan="2"><strong>Red setter</strong> </th><th colspan="2"><strong>Moneymaker</strong> </th><th colspan="2"><strong>Red setter</strong> </th></tr><tr><th><strong>CO</strong> </th><th><strong>HS</strong> </th><th><strong>CO</strong> </th><th><strong>HS</strong> </th><th><strong>CO</strong> </th><th><strong>HS</strong> </th><th><strong>CO</strong> </th><th><strong>HS</strong> </th></tr></thead><tbody><tr><td><strong>Assembled transcripts</strong> </td><td>1,868 </td><td>1,639 </td><td>1,597 </td><td>1,481 </td><td>8,347 </td><td>9,088 </td><td>6,910 </td><td>8,057 </td></tr><tr><td><strong>Transcripts with SolycID</strong> </td><td>1,746 </td><td>1,552 </td><td>1,522 </td><td>1,400 </td><td>7,868 </td><td>8,652 </td><td>6,636 </td><td>7,716 </td></tr></tbody></table></div><div content-id="dsw051-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene set </th><th colspan="4">HS-dependent reduction<hr /> </th><th colspan="4">HS-dependent accumulation<hr /> </th></tr><tr><th rowspan="2"><strong>Library</strong> </th><th colspan="2"><strong>Moneymaker</strong> </th><th colspan="2"><strong>Red setter</strong> </th><th colspan="2"><strong>Moneymaker</strong> </th><th colspan="2"><strong>Red setter</strong> </th></tr><tr><th><strong>CO</strong> </th><th><strong>HS</strong> </th><th><strong>CO</strong> </th><th><strong>HS</strong> </th><th><strong>CO</strong> </th><th><strong>HS</strong> </th><th><strong>CO</strong> </th><th><strong>HS</strong> </th></tr></thead><tbody><tr><td><strong>Assembled transcripts</strong> </td><td>1,868 </td><td>1,639 </td><td>1,597 </td><td>1,481 </td><td>8,347 </td><td>9,088 </td><td>6,910 </td><td>8,057 </td></tr><tr><td><strong>Transcripts with SolycID</strong> </td><td>1,746 </td><td>1,552 </td><td>1,522 </td><td>1,400 </td><td>7,868 </td><td>8,652 </td><td>6,636 </td><td>7,716 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>Given is the number of assembled transcripts and assembled transcripts with assigned SolycID for the Moneymaker and Red Setter CO and HS libraries with HS-dependent reduction or accumulation of alternative spliced transcripts.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw051-T3" data-id="dsw051-T3" class="table-wrap-title"><div class="label -label">Table 3.</div><div class="caption"><p>Transcriptome assembly descriptive statistics</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene set </th><th colspan="4">HS-dependent reduction<hr /> </th><th colspan="4">HS-dependent accumulation<hr /> </th></tr><tr><th rowspan="2"><strong>Library</strong> </th><th colspan="2"><strong>Moneymaker</strong> </th><th colspan="2"><strong>Red setter</strong> </th><th colspan="2"><strong>Moneymaker</strong> </th><th colspan="2"><strong>Red setter</strong> </th></tr><tr><th><strong>CO</strong> </th><th><strong>HS</strong> </th><th><strong>CO</strong> </th><th><strong>HS</strong> </th><th><strong>CO</strong> </th><th><strong>HS</strong> </th><th><strong>CO</strong> </th><th><strong>HS</strong> </th></tr></thead><tbody><tr><td><strong>Assembled transcripts</strong> </td><td>1,868 </td><td>1,639 </td><td>1,597 </td><td>1,481 </td><td>8,347 </td><td>9,088 </td><td>6,910 </td><td>8,057 </td></tr><tr><td><strong>Transcripts with SolycID</strong> </td><td>1,746 </td><td>1,552 </td><td>1,522 </td><td>1,400 </td><td>7,868 </td><td>8,652 </td><td>6,636 </td><td>7,716 </td></tr></tbody></table></div><div content-id="dsw051-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene set </th><th colspan="4">HS-dependent reduction<hr /> </th><th colspan="4">HS-dependent accumulation<hr /> </th></tr><tr><th rowspan="2"><strong>Library</strong> </th><th colspan="2"><strong>Moneymaker</strong> </th><th colspan="2"><strong>Red setter</strong> </th><th colspan="2"><strong>Moneymaker</strong> </th><th colspan="2"><strong>Red setter</strong> </th></tr><tr><th><strong>CO</strong> </th><th><strong>HS</strong> </th><th><strong>CO</strong> </th><th><strong>HS</strong> </th><th><strong>CO</strong> </th><th><strong>HS</strong> </th><th><strong>CO</strong> </th><th><strong>HS</strong> </th></tr></thead><tbody><tr><td><strong>Assembled transcripts</strong> </td><td>1,868 </td><td>1,639 </td><td>1,597 </td><td>1,481 </td><td>8,347 </td><td>9,088 </td><td>6,910 </td><td>8,057 </td></tr><tr><td><strong>Transcripts with SolycID</strong> </td><td>1,746 </td><td>1,552 </td><td>1,522 </td><td>1,400 </td><td>7,868 </td><td>8,652 </td><td>6,636 </td><td>7,716 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>Given is the number of assembled transcripts and assembled transcripts with assigned SolycID for the Moneymaker and Red Setter CO and HS libraries with HS-dependent reduction or accumulation of alternative spliced transcripts.</p></div></div></div></div></p><div class="block-child-p">In depth analysis yielded 332 transcripts occurring only after HS (HS-specific transcripts) and 70 transcripts only under CO conditions (HS-repressed transcripts) in both cultivars. At first we confirmed that all of those transcripts are assigned to at least one gene of the reference genome. Second, we removed transcripts with inconsistent length, lack of CDS or assignment to more than one identifier as described in methods. Removal was performed due to the fact that these transcripts could comprise non-fully assembled transcripts and might represent misannotations in the ITAG2.4, which were not in the focus of our study. 58 and 69% of all HS-specific and HS-repressed transcripts, respectively, were excluded by this procedure from further analyses. The remaining 141 HS-specific and 22 HS-repressed transcripts were classified in five categories according to the AS event observed. Category ‘i’ unifies transcripts with IR only (i, only IR), category ‘ii’ transcripts with ES only (ii, only ES) and category ‘iii’ transcripts with IR and ES only (iii, IR + ES). Transcripts with other AS events in addition to or even in the absence of IR or ES (iv, others) were annotated as category ‘iv’ and transcripts without AS when compared with the ITAG2.4 as category ‘v’ (v, perfectly; <a reveal-id="dsw051-F5" data-open="dsw051-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). The latter two occurred if other events than IR or ES arise (i.e. AA, AD, not annotated introns or not annotated exons) or if the transcript annotated in the ITAG2.4 is only detected after HS and CO, respectively. Most transcripts show only IR (i: 36% HS-specific, 50% HS-repressed), while another large fraction represents perfectly spliced transcripts (v: 26% HS-specific, 23% HS-repressed). Interestingly, 26% of the HS-specific and of the 23% HS-repressed transcripts show modifications of the transcript distinct from IR or ES (iv). The results of the reference based transcriptome assembly were confirmed by RT-PCR analysis of three randomly selected genes with HS-specific transcripts and one gene constitutively spliced under all conditions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&amp;Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S9</a></span>). <div data-id="dsw051-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051f5.png?Expires=2147483647&amp;Signature=VXjWX0Md-e5BZq0XWTmrSwSzEhSFUaYBYf8yVpZZgjpvKV90c2OHa~u6V7K0rZ3jYRTXeq7iZ03Q84iAYoxRBErswjclxHz~jcjHtasTsZ5NP8ZakLbA10p6TCHWmEaQc4qOQdGNC9JI5S51R-OrKkbP0XFoABRG-WD4cb1fAzdhqlfnv-K4eBSNv~fBnycmfcnfr~fMowTlnfvkO5VUDIcAfek4X9XhjQ894iHeFyHDxdjx6ZdcfhEcuGxUGPojZrxdz1~qN1OTRSMRU0u4JFuiUVMbL-f5umIeemqvOn3ujF~N--4QIUGdMBciRnffiaq6X68J54DYf3NqyxZcbg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CDS length alterations of transcripts due to AS under HS in pollen. Shown are changes in CDS length for HS-repressed (dark grey; left) and HS-specific transcripts (light grey; right) due to AS. Transcripts were classified in five categories with respect to the AS events observed: Only IR events (only IR); only ES events (only ES); IR and ES events (IR + ES); other AS events than IR and ES (others); transcripts perfectly spliced (perfectly). Numbers above the bars indicate the amount of transcripts assigned to the different categories." path-from-xml="dsw051f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78054939/dsw051f5.tif" path-from-xml="dsw051f5.tif" target="_blank">View large</a><a section="78054939" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051f5.png?Expires=2147483647&amp;Signature=mtVe-tyzwiVKLS-vvgNJnvl826fQYKTNSHXjLKFd8CyZFkJ6i9VNeDCFm12e0GdxzAJ0KBw78jBi7ne5~6dZOR~oZfhz9YJHjADV40j1MTcBzePxfPHQ-M6E6kvAVfpAJZTkg8yKsdZzF0vLrcSV5CUuYRC9dsXe8yyqDtP4IDIhO0g7EDkvEZqFVyWRgK83xukctWYIQXzC-znV-G-Y2uLjLN-2EXUROE~dpEMeaWUNFHpSJql60Uyrv-XnvCuH7qkN6NXxsLf-VzNtOIeQtGj8HY~f334koNXXhgPc0LpdZL-U9rWFsoKHXnkx4wcYHbgsdQYBaJKYAgRPbqRf8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78054939&amp;ar=2738679&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw051f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CDS length alterations of transcripts due to AS under HS in pollen. Shown are changes in CDS length for HS-repressed (dark grey; left) and HS-specific transcripts (light grey; right) due to AS. Transcripts were classified in five categories with respect to the AS events observed: Only IR events (only IR); only ES events (only ES); IR and ES events (IR + ES); other AS events than IR and ES (others); transcripts perfectly spliced (perfectly). Numbers above the bars indicate the amount of transcripts assigned to the different categories.</p></div></div><div content-id="dsw051-F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051f5.png?Expires=2147483647&amp;Signature=VXjWX0Md-e5BZq0XWTmrSwSzEhSFUaYBYf8yVpZZgjpvKV90c2OHa~u6V7K0rZ3jYRTXeq7iZ03Q84iAYoxRBErswjclxHz~jcjHtasTsZ5NP8ZakLbA10p6TCHWmEaQc4qOQdGNC9JI5S51R-OrKkbP0XFoABRG-WD4cb1fAzdhqlfnv-K4eBSNv~fBnycmfcnfr~fMowTlnfvkO5VUDIcAfek4X9XhjQ894iHeFyHDxdjx6ZdcfhEcuGxUGPojZrxdz1~qN1OTRSMRU0u4JFuiUVMbL-f5umIeemqvOn3ujF~N--4QIUGdMBciRnffiaq6X68J54DYf3NqyxZcbg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CDS length alterations of transcripts due to AS under HS in pollen. Shown are changes in CDS length for HS-repressed (dark grey; left) and HS-specific transcripts (light grey; right) due to AS. Transcripts were classified in five categories with respect to the AS events observed: Only IR events (only IR); only ES events (only ES); IR and ES events (IR + ES); other AS events than IR and ES (others); transcripts perfectly spliced (perfectly). Numbers above the bars indicate the amount of transcripts assigned to the different categories." path-from-xml="dsw051f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78054939/dsw051f5.tif" path-from-xml="dsw051f5.tif" target="_blank">View large</a><a section="78054939" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051f5.png?Expires=2147483647&amp;Signature=mtVe-tyzwiVKLS-vvgNJnvl826fQYKTNSHXjLKFd8CyZFkJ6i9VNeDCFm12e0GdxzAJ0KBw78jBi7ne5~6dZOR~oZfhz9YJHjADV40j1MTcBzePxfPHQ-M6E6kvAVfpAJZTkg8yKsdZzF0vLrcSV5CUuYRC9dsXe8yyqDtP4IDIhO0g7EDkvEZqFVyWRgK83xukctWYIQXzC-znV-G-Y2uLjLN-2EXUROE~dpEMeaWUNFHpSJql60Uyrv-XnvCuH7qkN6NXxsLf-VzNtOIeQtGj8HY~f334koNXXhgPc0LpdZL-U9rWFsoKHXnkx4wcYHbgsdQYBaJKYAgRPbqRf8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78054939&amp;ar=2738679&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw051f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CDS length alterations of transcripts due to AS under HS in pollen. Shown are changes in CDS length for HS-repressed (dark grey; left) and HS-specific transcripts (light grey; right) due to AS. Transcripts were classified in five categories with respect to the AS events observed: Only IR events (only IR); only ES events (only ES); IR and ES events (IR + ES); other AS events than IR and ES (others); transcripts perfectly spliced (perfectly). Numbers above the bars indicate the amount of transcripts assigned to the different categories.</p></div></div></div><p>Afterwards the length of the CDS represented by the HS-specific and repressed transcripts was compared with the annotated CDSs (ITAG2.4). As expected no alterations of CDS length was observed for all perfectly spliced transcripts (perfectly; <a reveal-id="dsw051-F5" data-open="dsw051-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). In turn, the CDS encoded by the transcripts of category ‘i’–‘iv’ showed generally a length reduction (<a reveal-id="dsw051-F5" data-open="dsw051-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Although this was expected for transcripts with ES, the shortened CDS for transcripts of category ‘i’ and ‘iv’ suggests the occurrence of PTCs. Therefore, alterations in the architecture of functional domains can be expected as well as the occurrence of PTCs.</p>                            <h3 scrollto-destination=78054941 id="78054941" class="section-title" >3.5. Transcripts coding for proteins with altered domain architecture upon HS</h3>
<p>We analysed the domain architecture of the proteins putatively translated from the CDS of the HS-repressed or specific transcripts. Each CDS was translated into a protein sequence followed by a domain search with HMMER (hmmsearch).<a reveal-id="dsw051-B48" data-open="dsw051-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> Predicted domains were compared with domains annotated for the proteins encoded by the CDS of the ITAG2.4. For both, the HS-repressed and specific proteins, at least 54% of the proteins had the same domain architecture as predicted for the annotated gene (No change; <a reveal-id="dsw051-T4" data-open="dsw051-T4" class="link link-reveal link-table xref-fig">Table 4</a>). Only five HS-specific proteins showed a gain of functional domains (Gain and Gain + Loss; <a reveal-id="dsw051-T4" data-open="dsw051-T4" class="link link-reveal link-table xref-fig">Table 4</a>). Nevertheless, only for the F-box/LRR-repeat protein (Solyc10g080020) a new F-box domain was detected going along with loss of a Fist_C domain. The remaining proteins showed either an AS-induced domain split, which led to the detection of two sub-domains, or an AS-induced transition to a similar domain. For example, the Calpain-2 catalytic subunit (Solyc11g068460) showed an IR induced switch from EF_hand_5 to left-hand domain. In turn, loss of domains constitutes for all of the alterations of the HS-repressed proteins and 88% of the alterations of the HS-specific proteins (Loss, <a reveal-id="dsw051-T4" data-open="dsw051-T4" class="link link-reveal link-table xref-fig">Table 4</a>). To analyse this dominant effect of domain loss in more detail, all proteins with partial domain loss and translated from transcripts containing only IR and ES were inspected in more detail (<a reveal-id="dsw051-T5" data-open="dsw051-T5" class="link link-reveal link-table xref-fig">Table 5</a>). <div content-id="dsw051-T4" class="table-modal"><div class="table-wrap"><div id="dsw051-T4" data-id="dsw051-T4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>Domain alterations of HS-repressed and specific proteins</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>Gain </th><th>Loss </th><th>Gain+Loss </th><th>No change </th></tr></thead><tbody><tr><td rowspan="5"><strong>HS avoi. proteins</strong> </td><td><strong>only ES</strong> </td><td>— </td><td>1 </td><td>— </td><td>— </td></tr><tr><td><strong>only IR</strong> </td><td>— </td><td>7 </td><td>— </td><td>4 </td></tr><tr><td><strong>IR+ES</strong> </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td><strong>others</strong> </td><td>— </td><td>2 </td><td>— </td><td>3 </td></tr><tr><td><strong>perfectly</strong> </td><td>— </td><td>– </td><td>— </td><td>5 </td></tr><tr><td rowspan="5"><strong>HS occ. proteins</strong> </td><td><strong>only ES</strong> </td><td>1 </td><td>7 </td><td>— </td><td>6 </td></tr><tr><td><strong>only IR</strong> </td><td>— </td><td>17 </td><td>2 </td><td>32 </td></tr><tr><td><strong>IR+ES</strong> </td><td>— </td><td>3 </td><td>— </td><td>1 </td></tr><tr><td><strong>others</strong> </td><td>— </td><td>12 </td><td>2 </td><td>21 </td></tr><tr><td><strong>perfectly</strong> </td><td>— </td><td>— </td><td>— </td><td>37 </td></tr></tbody></table></div><div content-id="dsw051-T4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>Gain </th><th>Loss </th><th>Gain+Loss </th><th>No change </th></tr></thead><tbody><tr><td rowspan="5"><strong>HS avoi. proteins</strong> </td><td><strong>only ES</strong> </td><td>— </td><td>1 </td><td>— </td><td>— </td></tr><tr><td><strong>only IR</strong> </td><td>— </td><td>7 </td><td>— </td><td>4 </td></tr><tr><td><strong>IR+ES</strong> </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td><strong>others</strong> </td><td>— </td><td>2 </td><td>— </td><td>3 </td></tr><tr><td><strong>perfectly</strong> </td><td>— </td><td>– </td><td>— </td><td>5 </td></tr><tr><td rowspan="5"><strong>HS occ. proteins</strong> </td><td><strong>only ES</strong> </td><td>1 </td><td>7 </td><td>— </td><td>6 </td></tr><tr><td><strong>only IR</strong> </td><td>— </td><td>17 </td><td>2 </td><td>32 </td></tr><tr><td><strong>IR+ES</strong> </td><td>— </td><td>3 </td><td>— </td><td>1 </td></tr><tr><td><strong>others</strong> </td><td>— </td><td>12 </td><td>2 </td><td>21 </td></tr><tr><td><strong>perfectly</strong> </td><td>— </td><td>— </td><td>— </td><td>37 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p>Given is the number of HS-repressed and specific proteins with alterations in domain architecture; either domain gain, loss, gain and loss, or no changes in domain architecture with respect to predicted domains for annotated ITAG2.4 proteins.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw051-T4" data-id="dsw051-T4" class="table-wrap-title"><div class="label -label">Table 4.</div><div class="caption"><p>Domain alterations of HS-repressed and specific proteins</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>Gain </th><th>Loss </th><th>Gain+Loss </th><th>No change </th></tr></thead><tbody><tr><td rowspan="5"><strong>HS avoi. proteins</strong> </td><td><strong>only ES</strong> </td><td>— </td><td>1 </td><td>— </td><td>— </td></tr><tr><td><strong>only IR</strong> </td><td>— </td><td>7 </td><td>— </td><td>4 </td></tr><tr><td><strong>IR+ES</strong> </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td><strong>others</strong> </td><td>— </td><td>2 </td><td>— </td><td>3 </td></tr><tr><td><strong>perfectly</strong> </td><td>— </td><td>– </td><td>— </td><td>5 </td></tr><tr><td rowspan="5"><strong>HS occ. proteins</strong> </td><td><strong>only ES</strong> </td><td>1 </td><td>7 </td><td>— </td><td>6 </td></tr><tr><td><strong>only IR</strong> </td><td>— </td><td>17 </td><td>2 </td><td>32 </td></tr><tr><td><strong>IR+ES</strong> </td><td>— </td><td>3 </td><td>— </td><td>1 </td></tr><tr><td><strong>others</strong> </td><td>— </td><td>12 </td><td>2 </td><td>21 </td></tr><tr><td><strong>perfectly</strong> </td><td>— </td><td>— </td><td>— </td><td>37 </td></tr></tbody></table></div><div content-id="dsw051-T4" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>Gain </th><th>Loss </th><th>Gain+Loss </th><th>No change </th></tr></thead><tbody><tr><td rowspan="5"><strong>HS avoi. proteins</strong> </td><td><strong>only ES</strong> </td><td>— </td><td>1 </td><td>— </td><td>— </td></tr><tr><td><strong>only IR</strong> </td><td>— </td><td>7 </td><td>— </td><td>4 </td></tr><tr><td><strong>IR+ES</strong> </td><td>— </td><td>— </td><td>— </td><td>— </td></tr><tr><td><strong>others</strong> </td><td>— </td><td>2 </td><td>— </td><td>3 </td></tr><tr><td><strong>perfectly</strong> </td><td>— </td><td>– </td><td>— </td><td>5 </td></tr><tr><td rowspan="5"><strong>HS occ. proteins</strong> </td><td><strong>only ES</strong> </td><td>1 </td><td>7 </td><td>— </td><td>6 </td></tr><tr><td><strong>only IR</strong> </td><td>— </td><td>17 </td><td>2 </td><td>32 </td></tr><tr><td><strong>IR+ES</strong> </td><td>— </td><td>3 </td><td>— </td><td>1 </td></tr><tr><td><strong>others</strong> </td><td>— </td><td>12 </td><td>2 </td><td>21 </td></tr><tr><td><strong>perfectly</strong> </td><td>— </td><td>— </td><td>— </td><td>37 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p>Given is the number of HS-repressed and specific proteins with alterations in domain architecture; either domain gain, loss, gain and loss, or no changes in domain architecture with respect to predicted domains for annotated ITAG2.4 proteins.</p></div></div></div></div><div content-id="dsw051-T5" class="table-modal"><div class="table-wrap"><div id="dsw051-T5" data-id="dsw051-T5" class="table-wrap-title"><div class="label -label">Table 5.</div><div class="caption"><p>Proteins with partial domain loss encoded by alternatively spliced isoforms</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>Solyc </th><th>Function </th><th>CDS </th><th>Domain alterations </th></tr></thead><tbody><tr><td rowspan="3"><strong>HS avoioded proteins</strong> </td><td>IR </td><td>Solyc10g079360.5 </td><td>transcription initiation factor IIB </td><td>49.67 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf1.png?Expires=2147483647&amp;Signature=cf9zAuifD5hJQob8Dd68ozzIENE9UcT7~btZY-3kMrInoNN5Qjd~7705fX1XR39jcC-uZ74W6gzGYGNJegx36jYWllc~EhY8GjeX3a0Zkqnc719omngu2mZ8Bny5m81bMjHfyn3kM9DnIWVSlnvkaGv9wiT0hobz5Vk7BFhHtw0ZgJ06UatHd6HLAZat~oqkRo0gwxGscanLt~X9TTzbpJlJZ~-nkkbtFvFIOlEnMH~7ONxgKNqdvdW7EvUO8J6-kLuaGO18O2uvQF30CGBe-1iYng20R~PegY62HNuHS4sIfeel1EYHiupsjcuKzTao6Q~SXpplWWM8d62GZKBhBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf1.tif" /> </td></tr><tr><td> </td><td>Solyc10g081600.1 </td><td>Rhomboid protease GlpG </td><td>64.54 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf2.png?Expires=2147483647&amp;Signature=MF~UINjN2c0RLxKV2E3j1jyvGQ6XxOW~zGUVIFah~rAe1YVf6CLOwmOXewGjLE6frbDYyo7hLzFi6lIjoP0k488cKLRtfGnbojigghCaJtDoSMJl~LIouYu3wCB6gsMMqeMlUF0ydP1QpgngMfn6o-P1kUCqLbcde91TeUVpntBMIpsOhxqhxQnO5OO6h~MSMSWuFM6m96tUvypixFmUw9CyHvhBMdMYSEoUA8s9jHbyJHHw5miiZ0qPdKTOM9JpjrEtSQ3RFrjnAXxtLzBOUhVV368FeNgo~K8waNSqqRfoP4dPJnQ2VFbuc6RLQrxCFChhARAh1lNGcvJjT7iP6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf2.tif" /> </td></tr><tr><td> </td><td>Solyc12g098930.1 </td><td>pyruvate dehydrogenase kinase </td><td>72.82 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf3.png?Expires=2147483647&amp;Signature=N1a1OuAYQiBsmYPMzZlAzuTg8MqjflSIC2XNpFuUFE1XHIvKMjdH0zpE0ewVQAMyj8kDvp1svhhXymVprmU0ufAvA26NgBZZK~PQzaAxKfHZd9C6hgEVGgHQeyciziCM6hpAlPyk1~bBt8NtxI0WGrIHLztYoYWkrDai18-XAcJzEw8HN0bZb4WzdxHeGlw8CcvVlQe8qQRktWXMWqk8mRlDGL2s5HHD6pTRTg3PyaeRgkCjyISNU0ZPFw6kSqCpE-fbAB6QNNyI5F-RmaH6c03l5UcdBuyegFVTS1dpwwRaqnTYhGde0C5EQEohtowGh~8~DHmVuFG32KzZKOsF-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf3.tif" /> </td></tr><tr><td rowspan="7"><strong>HS-specific prtoteins</strong> </td><td>IR </td><td>Solyc01g065490.1 </td><td>Putative Aldo/keto reductase </td><td>45.00 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf4.png?Expires=2147483647&amp;Signature=hqBdhXT4Y8z7-e6fl0-KZVHyYIhWDXuHWE4EA0ObQgDgNocXhmkilPC5lV~gWQl4~l~8wJ3evcQoWcyXKJZndy83cwPr8Kg30MPrfZUM2SL6AMr8fKSrK8mSTyGkutNbC9bpCjjQeBCybBXneQtB-loI8NZEOL5kcrR6TTm99YZQQxgK4G1ubLZ4ez3jOu00UvLovtZqXROO-cN~EaHSso93uGbYjsZ80KdcTfmDiycYhD1C80ekSn8rf9XExzp43D7FrM2oTYPyhwnr5MlY4p2~bkg4JAwPyDIp4JGFBJ9ADHCVb1qDnJLoUt8~RRppEF1XlNV1lESj77JP9J2-Pw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf4.tif" /> </td></tr><tr><td> </td><td>Solyc04g078050.2 </td><td>transcription initiation factor </td><td>26.63 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf5.png?Expires=2147483647&amp;Signature=uUgXnHoO8O30AeVw4hgTq-gVA-XUAuTPj1E-KtQ-JPwrFntGgSm0Vox-tP5RU7kzK1uL4vFALlEHutoKB76f~mHpXfEJuwojm1XcwD0aVjh5VI2dK8W3mbpQOE~sXtWOGveq1CXpUa3dnmeB8~JWBoCc6cPVESDo6OC~s0MMiTELCEsM2S98bnAdeJsVbPpwsap~OnnUu0F3Rg7n16AsTJnhBmIK0D~gjHMjShLsT2g4RRYpLDXhf-X8vsXHcMPy93dPwzQrwiVRU2QQ8kg-eHFm68v1zrIRKGwmpIwwEV3LdE~7Rd7YND4BXEWMbRbtr4HcCEas2hE-2s9CC0XSOA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf5.tif" /> </td></tr><tr><td> </td><td>Solyc05g053190.1 </td><td>Putative tRNA pseudouridine synthase </td><td>37.19 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf6.png?Expires=2147483647&amp;Signature=P~K5g79Q2TToWO~0Hnb4G-fdFq7MeLRSpWIYv69M5of9-9nIpx2IpVI7j8igUrU-bN41F8htcJa82lu~etJ9ehw7yuWZd5CkNeNa16sxpcORlPOa8mVK4wJp-EhtlVUeyCg-dMN0Ejds8~L-B-Ja2UMn3mV8xn~RclUIReL3IJFTyHVoZNjRcLbw9gQo6nZfTErJSFBYq71kjp0KY9~P-88q6m8E0~6Fi0MvRseCVbjpsgMBSVxb1l7WSFZZNgs1x77CDxZptmYlvoEaTusECYu-tPX3W9CMmGaw9lb7WUJ0nrDMfO8kxNV2yf5aAbz2G79LbSQvCLQe8w1w1qTLmw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf6.tif" /> </td></tr><tr><td> </td><td>Solyc08g081070.3 </td><td>Mitochondrial carrier family Mitoferrin-1 </td><td>62.13 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf7.png?Expires=2147483647&amp;Signature=n9zTIjCKqEvArnSiL2uHWvWqqq2jJ3f7KT~vqN3yquqZYFxa55ofFaRjR33eXv2GGBKBvNr3celMHjbWRW7JF~ZJqxSBqFfKiBiI-AhLhmTtv3dzQsHtw17y7qT49F5dDvNNzQgr9BycySOz2leVdCTxcLqIUIQ5Uae4ON6pbKmVPOWoaH6VlirwWYPbRhBwqWxKNodI6dm3Ip3Gt2waZecO-MHy~7mc~eEeEKhZqHeIk1JswA7Owv-g~PME9dCVHdBn9LjZa1bJX9Ia4HsMiFD9yEz~F4KlH2irMX~snQOKgtw6U~Ggcj0FCapvkx2WslW9njKB2fJDN8q2yHdLhA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf7.tif" /> </td></tr><tr><td> </td><td>Solyc09g074680.1 </td><td>Putative CRL complex subunit </td><td>83.53 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf8.png?Expires=2147483647&amp;Signature=kbKDzcKF-r6jtVRVNV955g6QngIKv6zDjmu2eEpbC4ND4HzMqM6mEe3E~0IA0~OXuWZwLiK6mCCfXioYIlzCg7qIllGxwSeHbXyoOtitHBSriv8kAHx7-J8YdTiY27c2ZnmPSd8ysZAjBQjQuC-namnvYZOD4NRQTFOJ66WA0t9Vg7v4p7eoSfyEvzesnYkQ194rI8xs~1H421Pedg8P2wHZh16r1Dpt-OUrlOy7LV3ASmdZ54rwNGuLLjVEXjo6gVw79hWDt9w0EzARoRXv2KuXOXor7DO9zaRA9UztCvtcjUt7JrPMAMXB4ykVGMtHLJV0ZxWx6z9BU~B6Mj23Hg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf8.tif" /> </td></tr><tr><td> </td><td>Solyc11g069490.1 </td><td>Serine/threonine-protein phosphatase 6 regulatory subunit 3 </td><td>43.98 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf9.png?Expires=2147483647&amp;Signature=q3Kzukvsz6oAPHcX6G0bNUvRNtW6Mw5JZ9bcYv1hyF7pOxyfBG3Wj9hi~t-D8V2P7Yrphx54yo-davl8NhFcPqp0vwf2pLNTmW2UmANIAS1b8B4VMXm5vFIYBJ7jbSqiDFzlNB937dm0f6BgTrKlcuzfHQBHpHU~5UpoxhVqdj66Vmzi8H2gM2g5H-m6HfEFQMID~-9mw7xqKuHKdeuKt6vZad7qU0wvZ-7U9K2xiLrM3uPBfrEcwuMyyqGozaqp2JVpXEh3dqvOwEakeL3ONxnhwbUYwC41PC6ClM3FI9br240k~u8HvuQF35h2zrAvB5M8TtpEIWJ6w3bXflk~nA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf9.tif" /> </td></tr><tr><td>ES </td><td>Solyc10g009070.2 </td><td>RNA helicase DEAD32 </td><td>65.52 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf10.png?Expires=2147483647&amp;Signature=pmtmj~pvHw57Qsn0Vo76c9PJ5hkXMOvThDLaZOLvLm~Vp5V9NhzJAxP~NUwnoe3iuha2bgUxo7kAe2Vh6Pb5vmkxcV2zwv80b39GYkeFNFBc~Cv-F8jlqyp3t3BQMLPHFnnYH0d5CEjaHAED~J9W5UwMQyIQYYWBg4V9bgiCydN9qmt9bV4h~fs73-c3kwdiKQlBA-cwG3xWlNUiYwpzRncNhKQYz8AhPN0qACXsZQDwOvl~fCneRP-gqeqtuJd0TiVQH1FE8-yGPMrk4yyvT9aYWytgQhCIDqEJEqAOzaDEcVk2rpPWTd9tnwjDWh49zjjWPCX7nXYybGIbF6LF8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf10.tif" /> </td></tr></tbody></table></div><div content-id="dsw051-T5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>Solyc </th><th>Function </th><th>CDS </th><th>Domain alterations </th></tr></thead><tbody><tr><td rowspan="3"><strong>HS avoioded proteins</strong> </td><td>IR </td><td>Solyc10g079360.5 </td><td>transcription initiation factor IIB </td><td>49.67 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf1.png?Expires=2147483647&amp;Signature=cf9zAuifD5hJQob8Dd68ozzIENE9UcT7~btZY-3kMrInoNN5Qjd~7705fX1XR39jcC-uZ74W6gzGYGNJegx36jYWllc~EhY8GjeX3a0Zkqnc719omngu2mZ8Bny5m81bMjHfyn3kM9DnIWVSlnvkaGv9wiT0hobz5Vk7BFhHtw0ZgJ06UatHd6HLAZat~oqkRo0gwxGscanLt~X9TTzbpJlJZ~-nkkbtFvFIOlEnMH~7ONxgKNqdvdW7EvUO8J6-kLuaGO18O2uvQF30CGBe-1iYng20R~PegY62HNuHS4sIfeel1EYHiupsjcuKzTao6Q~SXpplWWM8d62GZKBhBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf1.tif" /> </td></tr><tr><td> </td><td>Solyc10g081600.1 </td><td>Rhomboid protease GlpG </td><td>64.54 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf2.png?Expires=2147483647&amp;Signature=MF~UINjN2c0RLxKV2E3j1jyvGQ6XxOW~zGUVIFah~rAe1YVf6CLOwmOXewGjLE6frbDYyo7hLzFi6lIjoP0k488cKLRtfGnbojigghCaJtDoSMJl~LIouYu3wCB6gsMMqeMlUF0ydP1QpgngMfn6o-P1kUCqLbcde91TeUVpntBMIpsOhxqhxQnO5OO6h~MSMSWuFM6m96tUvypixFmUw9CyHvhBMdMYSEoUA8s9jHbyJHHw5miiZ0qPdKTOM9JpjrEtSQ3RFrjnAXxtLzBOUhVV368FeNgo~K8waNSqqRfoP4dPJnQ2VFbuc6RLQrxCFChhARAh1lNGcvJjT7iP6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf2.tif" /> </td></tr><tr><td> </td><td>Solyc12g098930.1 </td><td>pyruvate dehydrogenase kinase </td><td>72.82 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf3.png?Expires=2147483647&amp;Signature=N1a1OuAYQiBsmYPMzZlAzuTg8MqjflSIC2XNpFuUFE1XHIvKMjdH0zpE0ewVQAMyj8kDvp1svhhXymVprmU0ufAvA26NgBZZK~PQzaAxKfHZd9C6hgEVGgHQeyciziCM6hpAlPyk1~bBt8NtxI0WGrIHLztYoYWkrDai18-XAcJzEw8HN0bZb4WzdxHeGlw8CcvVlQe8qQRktWXMWqk8mRlDGL2s5HHD6pTRTg3PyaeRgkCjyISNU0ZPFw6kSqCpE-fbAB6QNNyI5F-RmaH6c03l5UcdBuyegFVTS1dpwwRaqnTYhGde0C5EQEohtowGh~8~DHmVuFG32KzZKOsF-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf3.tif" /> </td></tr><tr><td rowspan="7"><strong>HS-specific prtoteins</strong> </td><td>IR </td><td>Solyc01g065490.1 </td><td>Putative Aldo/keto reductase </td><td>45.00 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf4.png?Expires=2147483647&amp;Signature=hqBdhXT4Y8z7-e6fl0-KZVHyYIhWDXuHWE4EA0ObQgDgNocXhmkilPC5lV~gWQl4~l~8wJ3evcQoWcyXKJZndy83cwPr8Kg30MPrfZUM2SL6AMr8fKSrK8mSTyGkutNbC9bpCjjQeBCybBXneQtB-loI8NZEOL5kcrR6TTm99YZQQxgK4G1ubLZ4ez3jOu00UvLovtZqXROO-cN~EaHSso93uGbYjsZ80KdcTfmDiycYhD1C80ekSn8rf9XExzp43D7FrM2oTYPyhwnr5MlY4p2~bkg4JAwPyDIp4JGFBJ9ADHCVb1qDnJLoUt8~RRppEF1XlNV1lESj77JP9J2-Pw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf4.tif" /> </td></tr><tr><td> </td><td>Solyc04g078050.2 </td><td>transcription initiation factor </td><td>26.63 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf5.png?Expires=2147483647&amp;Signature=uUgXnHoO8O30AeVw4hgTq-gVA-XUAuTPj1E-KtQ-JPwrFntGgSm0Vox-tP5RU7kzK1uL4vFALlEHutoKB76f~mHpXfEJuwojm1XcwD0aVjh5VI2dK8W3mbpQOE~sXtWOGveq1CXpUa3dnmeB8~JWBoCc6cPVESDo6OC~s0MMiTELCEsM2S98bnAdeJsVbPpwsap~OnnUu0F3Rg7n16AsTJnhBmIK0D~gjHMjShLsT2g4RRYpLDXhf-X8vsXHcMPy93dPwzQrwiVRU2QQ8kg-eHFm68v1zrIRKGwmpIwwEV3LdE~7Rd7YND4BXEWMbRbtr4HcCEas2hE-2s9CC0XSOA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf5.tif" /> </td></tr><tr><td> </td><td>Solyc05g053190.1 </td><td>Putative tRNA pseudouridine synthase </td><td>37.19 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf6.png?Expires=2147483647&amp;Signature=P~K5g79Q2TToWO~0Hnb4G-fdFq7MeLRSpWIYv69M5of9-9nIpx2IpVI7j8igUrU-bN41F8htcJa82lu~etJ9ehw7yuWZd5CkNeNa16sxpcORlPOa8mVK4wJp-EhtlVUeyCg-dMN0Ejds8~L-B-Ja2UMn3mV8xn~RclUIReL3IJFTyHVoZNjRcLbw9gQo6nZfTErJSFBYq71kjp0KY9~P-88q6m8E0~6Fi0MvRseCVbjpsgMBSVxb1l7WSFZZNgs1x77CDxZptmYlvoEaTusECYu-tPX3W9CMmGaw9lb7WUJ0nrDMfO8kxNV2yf5aAbz2G79LbSQvCLQe8w1w1qTLmw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf6.tif" /> </td></tr><tr><td> </td><td>Solyc08g081070.3 </td><td>Mitochondrial carrier family Mitoferrin-1 </td><td>62.13 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf7.png?Expires=2147483647&amp;Signature=n9zTIjCKqEvArnSiL2uHWvWqqq2jJ3f7KT~vqN3yquqZYFxa55ofFaRjR33eXv2GGBKBvNr3celMHjbWRW7JF~ZJqxSBqFfKiBiI-AhLhmTtv3dzQsHtw17y7qT49F5dDvNNzQgr9BycySOz2leVdCTxcLqIUIQ5Uae4ON6pbKmVPOWoaH6VlirwWYPbRhBwqWxKNodI6dm3Ip3Gt2waZecO-MHy~7mc~eEeEKhZqHeIk1JswA7Owv-g~PME9dCVHdBn9LjZa1bJX9Ia4HsMiFD9yEz~F4KlH2irMX~snQOKgtw6U~Ggcj0FCapvkx2WslW9njKB2fJDN8q2yHdLhA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf7.tif" /> </td></tr><tr><td> </td><td>Solyc09g074680.1 </td><td>Putative CRL complex subunit </td><td>83.53 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf8.png?Expires=2147483647&amp;Signature=kbKDzcKF-r6jtVRVNV955g6QngIKv6zDjmu2eEpbC4ND4HzMqM6mEe3E~0IA0~OXuWZwLiK6mCCfXioYIlzCg7qIllGxwSeHbXyoOtitHBSriv8kAHx7-J8YdTiY27c2ZnmPSd8ysZAjBQjQuC-namnvYZOD4NRQTFOJ66WA0t9Vg7v4p7eoSfyEvzesnYkQ194rI8xs~1H421Pedg8P2wHZh16r1Dpt-OUrlOy7LV3ASmdZ54rwNGuLLjVEXjo6gVw79hWDt9w0EzARoRXv2KuXOXor7DO9zaRA9UztCvtcjUt7JrPMAMXB4ykVGMtHLJV0ZxWx6z9BU~B6Mj23Hg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf8.tif" /> </td></tr><tr><td> </td><td>Solyc11g069490.1 </td><td>Serine/threonine-protein phosphatase 6 regulatory subunit 3 </td><td>43.98 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf9.png?Expires=2147483647&amp;Signature=q3Kzukvsz6oAPHcX6G0bNUvRNtW6Mw5JZ9bcYv1hyF7pOxyfBG3Wj9hi~t-D8V2P7Yrphx54yo-davl8NhFcPqp0vwf2pLNTmW2UmANIAS1b8B4VMXm5vFIYBJ7jbSqiDFzlNB937dm0f6BgTrKlcuzfHQBHpHU~5UpoxhVqdj66Vmzi8H2gM2g5H-m6HfEFQMID~-9mw7xqKuHKdeuKt6vZad7qU0wvZ-7U9K2xiLrM3uPBfrEcwuMyyqGozaqp2JVpXEh3dqvOwEakeL3ONxnhwbUYwC41PC6ClM3FI9br240k~u8HvuQF35h2zrAvB5M8TtpEIWJ6w3bXflk~nA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf9.tif" /> </td></tr><tr><td>ES </td><td>Solyc10g009070.2 </td><td>RNA helicase DEAD32 </td><td>65.52 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf10.png?Expires=2147483647&amp;Signature=pmtmj~pvHw57Qsn0Vo76c9PJ5hkXMOvThDLaZOLvLm~Vp5V9NhzJAxP~NUwnoe3iuha2bgUxo7kAe2Vh6Pb5vmkxcV2zwv80b39GYkeFNFBc~Cv-F8jlqyp3t3BQMLPHFnnYH0d5CEjaHAED~J9W5UwMQyIQYYWBg4V9bgiCydN9qmt9bV4h~fs73-c3kwdiKQlBA-cwG3xWlNUiYwpzRncNhKQYz8AhPN0qACXsZQDwOvl~fCneRP-gqeqtuJd0TiVQH1FE8-yGPMrk4yyvT9aYWytgQhCIDqEJEqAOzaDEcVk2rpPWTd9tnwjDWh49zjjWPCX7nXYybGIbF6LF8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf10.tif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p>Listed are the category (HS-repressed/specific), the AS process leading to the isoform, the accession number, the putative function of the protein, percentage of CDS length compared with the ITAG2.4 and domains present (black), completely (grey) or partially absent (black to grey color gradient).</p></div></div></div></div></div><div class="table-wrap"><div id="dsw051-T5" data-id="dsw051-T5" class="table-wrap-title"><div class="label -label">Table 5.</div><div class="caption"><p>Proteins with partial domain loss encoded by alternatively spliced isoforms</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>Solyc </th><th>Function </th><th>CDS </th><th>Domain alterations </th></tr></thead><tbody><tr><td rowspan="3"><strong>HS avoioded proteins</strong> </td><td>IR </td><td>Solyc10g079360.5 </td><td>transcription initiation factor IIB </td><td>49.67 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf1.png?Expires=2147483647&amp;Signature=cf9zAuifD5hJQob8Dd68ozzIENE9UcT7~btZY-3kMrInoNN5Qjd~7705fX1XR39jcC-uZ74W6gzGYGNJegx36jYWllc~EhY8GjeX3a0Zkqnc719omngu2mZ8Bny5m81bMjHfyn3kM9DnIWVSlnvkaGv9wiT0hobz5Vk7BFhHtw0ZgJ06UatHd6HLAZat~oqkRo0gwxGscanLt~X9TTzbpJlJZ~-nkkbtFvFIOlEnMH~7ONxgKNqdvdW7EvUO8J6-kLuaGO18O2uvQF30CGBe-1iYng20R~PegY62HNuHS4sIfeel1EYHiupsjcuKzTao6Q~SXpplWWM8d62GZKBhBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf1.tif" /> </td></tr><tr><td> </td><td>Solyc10g081600.1 </td><td>Rhomboid protease GlpG </td><td>64.54 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf2.png?Expires=2147483647&amp;Signature=MF~UINjN2c0RLxKV2E3j1jyvGQ6XxOW~zGUVIFah~rAe1YVf6CLOwmOXewGjLE6frbDYyo7hLzFi6lIjoP0k488cKLRtfGnbojigghCaJtDoSMJl~LIouYu3wCB6gsMMqeMlUF0ydP1QpgngMfn6o-P1kUCqLbcde91TeUVpntBMIpsOhxqhxQnO5OO6h~MSMSWuFM6m96tUvypixFmUw9CyHvhBMdMYSEoUA8s9jHbyJHHw5miiZ0qPdKTOM9JpjrEtSQ3RFrjnAXxtLzBOUhVV368FeNgo~K8waNSqqRfoP4dPJnQ2VFbuc6RLQrxCFChhARAh1lNGcvJjT7iP6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf2.tif" /> </td></tr><tr><td> </td><td>Solyc12g098930.1 </td><td>pyruvate dehydrogenase kinase </td><td>72.82 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf3.png?Expires=2147483647&amp;Signature=N1a1OuAYQiBsmYPMzZlAzuTg8MqjflSIC2XNpFuUFE1XHIvKMjdH0zpE0ewVQAMyj8kDvp1svhhXymVprmU0ufAvA26NgBZZK~PQzaAxKfHZd9C6hgEVGgHQeyciziCM6hpAlPyk1~bBt8NtxI0WGrIHLztYoYWkrDai18-XAcJzEw8HN0bZb4WzdxHeGlw8CcvVlQe8qQRktWXMWqk8mRlDGL2s5HHD6pTRTg3PyaeRgkCjyISNU0ZPFw6kSqCpE-fbAB6QNNyI5F-RmaH6c03l5UcdBuyegFVTS1dpwwRaqnTYhGde0C5EQEohtowGh~8~DHmVuFG32KzZKOsF-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf3.tif" /> </td></tr><tr><td rowspan="7"><strong>HS-specific prtoteins</strong> </td><td>IR </td><td>Solyc01g065490.1 </td><td>Putative Aldo/keto reductase </td><td>45.00 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf4.png?Expires=2147483647&amp;Signature=hqBdhXT4Y8z7-e6fl0-KZVHyYIhWDXuHWE4EA0ObQgDgNocXhmkilPC5lV~gWQl4~l~8wJ3evcQoWcyXKJZndy83cwPr8Kg30MPrfZUM2SL6AMr8fKSrK8mSTyGkutNbC9bpCjjQeBCybBXneQtB-loI8NZEOL5kcrR6TTm99YZQQxgK4G1ubLZ4ez3jOu00UvLovtZqXROO-cN~EaHSso93uGbYjsZ80KdcTfmDiycYhD1C80ekSn8rf9XExzp43D7FrM2oTYPyhwnr5MlY4p2~bkg4JAwPyDIp4JGFBJ9ADHCVb1qDnJLoUt8~RRppEF1XlNV1lESj77JP9J2-Pw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf4.tif" /> </td></tr><tr><td> </td><td>Solyc04g078050.2 </td><td>transcription initiation factor </td><td>26.63 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf5.png?Expires=2147483647&amp;Signature=uUgXnHoO8O30AeVw4hgTq-gVA-XUAuTPj1E-KtQ-JPwrFntGgSm0Vox-tP5RU7kzK1uL4vFALlEHutoKB76f~mHpXfEJuwojm1XcwD0aVjh5VI2dK8W3mbpQOE~sXtWOGveq1CXpUa3dnmeB8~JWBoCc6cPVESDo6OC~s0MMiTELCEsM2S98bnAdeJsVbPpwsap~OnnUu0F3Rg7n16AsTJnhBmIK0D~gjHMjShLsT2g4RRYpLDXhf-X8vsXHcMPy93dPwzQrwiVRU2QQ8kg-eHFm68v1zrIRKGwmpIwwEV3LdE~7Rd7YND4BXEWMbRbtr4HcCEas2hE-2s9CC0XSOA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf5.tif" /> </td></tr><tr><td> </td><td>Solyc05g053190.1 </td><td>Putative tRNA pseudouridine synthase </td><td>37.19 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf6.png?Expires=2147483647&amp;Signature=P~K5g79Q2TToWO~0Hnb4G-fdFq7MeLRSpWIYv69M5of9-9nIpx2IpVI7j8igUrU-bN41F8htcJa82lu~etJ9ehw7yuWZd5CkNeNa16sxpcORlPOa8mVK4wJp-EhtlVUeyCg-dMN0Ejds8~L-B-Ja2UMn3mV8xn~RclUIReL3IJFTyHVoZNjRcLbw9gQo6nZfTErJSFBYq71kjp0KY9~P-88q6m8E0~6Fi0MvRseCVbjpsgMBSVxb1l7WSFZZNgs1x77CDxZptmYlvoEaTusECYu-tPX3W9CMmGaw9lb7WUJ0nrDMfO8kxNV2yf5aAbz2G79LbSQvCLQe8w1w1qTLmw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf6.tif" /> </td></tr><tr><td> </td><td>Solyc08g081070.3 </td><td>Mitochondrial carrier family Mitoferrin-1 </td><td>62.13 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf7.png?Expires=2147483647&amp;Signature=n9zTIjCKqEvArnSiL2uHWvWqqq2jJ3f7KT~vqN3yquqZYFxa55ofFaRjR33eXv2GGBKBvNr3celMHjbWRW7JF~ZJqxSBqFfKiBiI-AhLhmTtv3dzQsHtw17y7qT49F5dDvNNzQgr9BycySOz2leVdCTxcLqIUIQ5Uae4ON6pbKmVPOWoaH6VlirwWYPbRhBwqWxKNodI6dm3Ip3Gt2waZecO-MHy~7mc~eEeEKhZqHeIk1JswA7Owv-g~PME9dCVHdBn9LjZa1bJX9Ia4HsMiFD9yEz~F4KlH2irMX~snQOKgtw6U~Ggcj0FCapvkx2WslW9njKB2fJDN8q2yHdLhA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf7.tif" /> </td></tr><tr><td> </td><td>Solyc09g074680.1 </td><td>Putative CRL complex subunit </td><td>83.53 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf8.png?Expires=2147483647&amp;Signature=kbKDzcKF-r6jtVRVNV955g6QngIKv6zDjmu2eEpbC4ND4HzMqM6mEe3E~0IA0~OXuWZwLiK6mCCfXioYIlzCg7qIllGxwSeHbXyoOtitHBSriv8kAHx7-J8YdTiY27c2ZnmPSd8ysZAjBQjQuC-namnvYZOD4NRQTFOJ66WA0t9Vg7v4p7eoSfyEvzesnYkQ194rI8xs~1H421Pedg8P2wHZh16r1Dpt-OUrlOy7LV3ASmdZ54rwNGuLLjVEXjo6gVw79hWDt9w0EzARoRXv2KuXOXor7DO9zaRA9UztCvtcjUt7JrPMAMXB4ykVGMtHLJV0ZxWx6z9BU~B6Mj23Hg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf8.tif" /> </td></tr><tr><td> </td><td>Solyc11g069490.1 </td><td>Serine/threonine-protein phosphatase 6 regulatory subunit 3 </td><td>43.98 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf9.png?Expires=2147483647&amp;Signature=q3Kzukvsz6oAPHcX6G0bNUvRNtW6Mw5JZ9bcYv1hyF7pOxyfBG3Wj9hi~t-D8V2P7Yrphx54yo-davl8NhFcPqp0vwf2pLNTmW2UmANIAS1b8B4VMXm5vFIYBJ7jbSqiDFzlNB937dm0f6BgTrKlcuzfHQBHpHU~5UpoxhVqdj66Vmzi8H2gM2g5H-m6HfEFQMID~-9mw7xqKuHKdeuKt6vZad7qU0wvZ-7U9K2xiLrM3uPBfrEcwuMyyqGozaqp2JVpXEh3dqvOwEakeL3ONxnhwbUYwC41PC6ClM3FI9br240k~u8HvuQF35h2zrAvB5M8TtpEIWJ6w3bXflk~nA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf9.tif" /> </td></tr><tr><td>ES </td><td>Solyc10g009070.2 </td><td>RNA helicase DEAD32 </td><td>65.52 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf10.png?Expires=2147483647&amp;Signature=pmtmj~pvHw57Qsn0Vo76c9PJ5hkXMOvThDLaZOLvLm~Vp5V9NhzJAxP~NUwnoe3iuha2bgUxo7kAe2Vh6Pb5vmkxcV2zwv80b39GYkeFNFBc~Cv-F8jlqyp3t3BQMLPHFnnYH0d5CEjaHAED~J9W5UwMQyIQYYWBg4V9bgiCydN9qmt9bV4h~fs73-c3kwdiKQlBA-cwG3xWlNUiYwpzRncNhKQYz8AhPN0qACXsZQDwOvl~fCneRP-gqeqtuJd0TiVQH1FE8-yGPMrk4yyvT9aYWytgQhCIDqEJEqAOzaDEcVk2rpPWTd9tnwjDWh49zjjWPCX7nXYybGIbF6LF8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf10.tif" /> </td></tr></tbody></table></div><div content-id="dsw051-T5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>Solyc </th><th>Function </th><th>CDS </th><th>Domain alterations </th></tr></thead><tbody><tr><td rowspan="3"><strong>HS avoioded proteins</strong> </td><td>IR </td><td>Solyc10g079360.5 </td><td>transcription initiation factor IIB </td><td>49.67 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf1.png?Expires=2147483647&amp;Signature=cf9zAuifD5hJQob8Dd68ozzIENE9UcT7~btZY-3kMrInoNN5Qjd~7705fX1XR39jcC-uZ74W6gzGYGNJegx36jYWllc~EhY8GjeX3a0Zkqnc719omngu2mZ8Bny5m81bMjHfyn3kM9DnIWVSlnvkaGv9wiT0hobz5Vk7BFhHtw0ZgJ06UatHd6HLAZat~oqkRo0gwxGscanLt~X9TTzbpJlJZ~-nkkbtFvFIOlEnMH~7ONxgKNqdvdW7EvUO8J6-kLuaGO18O2uvQF30CGBe-1iYng20R~PegY62HNuHS4sIfeel1EYHiupsjcuKzTao6Q~SXpplWWM8d62GZKBhBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf1.tif" /> </td></tr><tr><td> </td><td>Solyc10g081600.1 </td><td>Rhomboid protease GlpG </td><td>64.54 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf2.png?Expires=2147483647&amp;Signature=MF~UINjN2c0RLxKV2E3j1jyvGQ6XxOW~zGUVIFah~rAe1YVf6CLOwmOXewGjLE6frbDYyo7hLzFi6lIjoP0k488cKLRtfGnbojigghCaJtDoSMJl~LIouYu3wCB6gsMMqeMlUF0ydP1QpgngMfn6o-P1kUCqLbcde91TeUVpntBMIpsOhxqhxQnO5OO6h~MSMSWuFM6m96tUvypixFmUw9CyHvhBMdMYSEoUA8s9jHbyJHHw5miiZ0qPdKTOM9JpjrEtSQ3RFrjnAXxtLzBOUhVV368FeNgo~K8waNSqqRfoP4dPJnQ2VFbuc6RLQrxCFChhARAh1lNGcvJjT7iP6g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf2.tif" /> </td></tr><tr><td> </td><td>Solyc12g098930.1 </td><td>pyruvate dehydrogenase kinase </td><td>72.82 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf3.png?Expires=2147483647&amp;Signature=N1a1OuAYQiBsmYPMzZlAzuTg8MqjflSIC2XNpFuUFE1XHIvKMjdH0zpE0ewVQAMyj8kDvp1svhhXymVprmU0ufAvA26NgBZZK~PQzaAxKfHZd9C6hgEVGgHQeyciziCM6hpAlPyk1~bBt8NtxI0WGrIHLztYoYWkrDai18-XAcJzEw8HN0bZb4WzdxHeGlw8CcvVlQe8qQRktWXMWqk8mRlDGL2s5HHD6pTRTg3PyaeRgkCjyISNU0ZPFw6kSqCpE-fbAB6QNNyI5F-RmaH6c03l5UcdBuyegFVTS1dpwwRaqnTYhGde0C5EQEohtowGh~8~DHmVuFG32KzZKOsF-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf3.tif" /> </td></tr><tr><td rowspan="7"><strong>HS-specific prtoteins</strong> </td><td>IR </td><td>Solyc01g065490.1 </td><td>Putative Aldo/keto reductase </td><td>45.00 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf4.png?Expires=2147483647&amp;Signature=hqBdhXT4Y8z7-e6fl0-KZVHyYIhWDXuHWE4EA0ObQgDgNocXhmkilPC5lV~gWQl4~l~8wJ3evcQoWcyXKJZndy83cwPr8Kg30MPrfZUM2SL6AMr8fKSrK8mSTyGkutNbC9bpCjjQeBCybBXneQtB-loI8NZEOL5kcrR6TTm99YZQQxgK4G1ubLZ4ez3jOu00UvLovtZqXROO-cN~EaHSso93uGbYjsZ80KdcTfmDiycYhD1C80ekSn8rf9XExzp43D7FrM2oTYPyhwnr5MlY4p2~bkg4JAwPyDIp4JGFBJ9ADHCVb1qDnJLoUt8~RRppEF1XlNV1lESj77JP9J2-Pw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf4.tif" /> </td></tr><tr><td> </td><td>Solyc04g078050.2 </td><td>transcription initiation factor </td><td>26.63 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf5.png?Expires=2147483647&amp;Signature=uUgXnHoO8O30AeVw4hgTq-gVA-XUAuTPj1E-KtQ-JPwrFntGgSm0Vox-tP5RU7kzK1uL4vFALlEHutoKB76f~mHpXfEJuwojm1XcwD0aVjh5VI2dK8W3mbpQOE~sXtWOGveq1CXpUa3dnmeB8~JWBoCc6cPVESDo6OC~s0MMiTELCEsM2S98bnAdeJsVbPpwsap~OnnUu0F3Rg7n16AsTJnhBmIK0D~gjHMjShLsT2g4RRYpLDXhf-X8vsXHcMPy93dPwzQrwiVRU2QQ8kg-eHFm68v1zrIRKGwmpIwwEV3LdE~7Rd7YND4BXEWMbRbtr4HcCEas2hE-2s9CC0XSOA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf5.tif" /> </td></tr><tr><td> </td><td>Solyc05g053190.1 </td><td>Putative tRNA pseudouridine synthase </td><td>37.19 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf6.png?Expires=2147483647&amp;Signature=P~K5g79Q2TToWO~0Hnb4G-fdFq7MeLRSpWIYv69M5of9-9nIpx2IpVI7j8igUrU-bN41F8htcJa82lu~etJ9ehw7yuWZd5CkNeNa16sxpcORlPOa8mVK4wJp-EhtlVUeyCg-dMN0Ejds8~L-B-Ja2UMn3mV8xn~RclUIReL3IJFTyHVoZNjRcLbw9gQo6nZfTErJSFBYq71kjp0KY9~P-88q6m8E0~6Fi0MvRseCVbjpsgMBSVxb1l7WSFZZNgs1x77CDxZptmYlvoEaTusECYu-tPX3W9CMmGaw9lb7WUJ0nrDMfO8kxNV2yf5aAbz2G79LbSQvCLQe8w1w1qTLmw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf6.tif" /> </td></tr><tr><td> </td><td>Solyc08g081070.3 </td><td>Mitochondrial carrier family Mitoferrin-1 </td><td>62.13 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf7.png?Expires=2147483647&amp;Signature=n9zTIjCKqEvArnSiL2uHWvWqqq2jJ3f7KT~vqN3yquqZYFxa55ofFaRjR33eXv2GGBKBvNr3celMHjbWRW7JF~ZJqxSBqFfKiBiI-AhLhmTtv3dzQsHtw17y7qT49F5dDvNNzQgr9BycySOz2leVdCTxcLqIUIQ5Uae4ON6pbKmVPOWoaH6VlirwWYPbRhBwqWxKNodI6dm3Ip3Gt2waZecO-MHy~7mc~eEeEKhZqHeIk1JswA7Owv-g~PME9dCVHdBn9LjZa1bJX9Ia4HsMiFD9yEz~F4KlH2irMX~snQOKgtw6U~Ggcj0FCapvkx2WslW9njKB2fJDN8q2yHdLhA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf7.tif" /> </td></tr><tr><td> </td><td>Solyc09g074680.1 </td><td>Putative CRL complex subunit </td><td>83.53 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf8.png?Expires=2147483647&amp;Signature=kbKDzcKF-r6jtVRVNV955g6QngIKv6zDjmu2eEpbC4ND4HzMqM6mEe3E~0IA0~OXuWZwLiK6mCCfXioYIlzCg7qIllGxwSeHbXyoOtitHBSriv8kAHx7-J8YdTiY27c2ZnmPSd8ysZAjBQjQuC-namnvYZOD4NRQTFOJ66WA0t9Vg7v4p7eoSfyEvzesnYkQ194rI8xs~1H421Pedg8P2wHZh16r1Dpt-OUrlOy7LV3ASmdZ54rwNGuLLjVEXjo6gVw79hWDt9w0EzARoRXv2KuXOXor7DO9zaRA9UztCvtcjUt7JrPMAMXB4ykVGMtHLJV0ZxWx6z9BU~B6Mj23Hg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf8.tif" /> </td></tr><tr><td> </td><td>Solyc11g069490.1 </td><td>Serine/threonine-protein phosphatase 6 regulatory subunit 3 </td><td>43.98 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf9.png?Expires=2147483647&amp;Signature=q3Kzukvsz6oAPHcX6G0bNUvRNtW6Mw5JZ9bcYv1hyF7pOxyfBG3Wj9hi~t-D8V2P7Yrphx54yo-davl8NhFcPqp0vwf2pLNTmW2UmANIAS1b8B4VMXm5vFIYBJ7jbSqiDFzlNB937dm0f6BgTrKlcuzfHQBHpHU~5UpoxhVqdj66Vmzi8H2gM2g5H-m6HfEFQMID~-9mw7xqKuHKdeuKt6vZad7qU0wvZ-7U9K2xiLrM3uPBfrEcwuMyyqGozaqp2JVpXEh3dqvOwEakeL3ONxnhwbUYwC41PC6ClM3FI9br240k~u8HvuQF35h2zrAvB5M8TtpEIWJ6w3bXflk~nA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf9.tif" /> </td></tr><tr><td>ES </td><td>Solyc10g009070.2 </td><td>RNA helicase DEAD32 </td><td>65.52 </td><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051ilf10.png?Expires=2147483647&amp;Signature=pmtmj~pvHw57Qsn0Vo76c9PJ5hkXMOvThDLaZOLvLm~Vp5V9NhzJAxP~NUwnoe3iuha2bgUxo7kAe2Vh6Pb5vmkxcV2zwv80b39GYkeFNFBc~Cv-F8jlqyp3t3BQMLPHFnnYH0d5CEjaHAED~J9W5UwMQyIQYYWBg4V9bgiCydN9qmt9bV4h~fs73-c3kwdiKQlBA-cwG3xWlNUiYwpzRncNhKQYz8AhPN0qACXsZQDwOvl~fCneRP-gqeqtuJd0TiVQH1FE8-yGPMrk4yyvT9aYWytgQhCIDqEJEqAOzaDEcVk2rpPWTd9tnwjDWh49zjjWPCX7nXYybGIbF6LF8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dsw051ilf10.tif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p>Listed are the category (HS-repressed/specific), the AS process leading to the isoform, the accession number, the putative function of the protein, percentage of CDS length compared with the ITAG2.4 and domains present (black), completely (grey) or partially absent (black to grey color gradient).</p></div></div></div></div></p><p>We detected three HS-repressed transcripts that coded for proteins with the loss of at least one functional domain. We observed a transcript for the transcription initiation factor IIB (Solyc10g079360.5) coding for the protein without the two TFIIB repeats (TFIIB). The rhomboid protease GlpG (Solyc10g081600.1) encoded by the AS transcript does not contain the ubiquitin associated domain. Moreover, the AS containing transcript for pyruvate dehydrogenase kinase (Solyc12g098930.1) does not code for the ATPase domain (HATPase_c).</p><p>In turn, seven proteins translated from HS-specific transcripts with at least one completely lost domain were detected. For four of the seven proteins (Solyc01g065490.1; Solyc05g053190.1; Solyc08g081070.3; Solyc11g069490.1) lost only repeated domains or motifs due to IR or ES. The remaining three proteins are the putative cullin-RING ubiquitin ligase (CRL) complex subunit (Solyc09g074680.1), which is lacking its neddylation site, a transcription initiation factor (Solyc04g078050.2) missing the TAF4 domain and the RNA helicase DEAD32 (Solyc10g009070.2) lacking the DBP10CT domain. The partial domain loss observed for these proteins raises the question whether the proteins would still be functional with either reduced or alternative function or if the underlying transcripts are triggered for NMD.</p>                            <h2 scrollto-destination=78054945 id="78054945" class="section-title" >4. Discussion</h2>
                            <h3 scrollto-destination=78054946 id="78054946" class="section-title" >4.1. The degree of AS in response to HS is cultivar specific</h3>
<p>AS is a central regulatory mechanism to adjust transcript abundance or expand the diversity of the protein content.<a reveal-id="dsw051-B15 dsw051-B16 dsw051-B17" data-open="dsw051-B15 dsw051-B16 dsw051-B17" class="link link-ref link-reveal xref-bibr"><sup>15–17</sup></a> The observed number of AS events and the ratio between IR, ES, AD and AA events in tomato pollen (<a reveal-id="dsw051-F1" data-open="dsw051-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>) are in agreement with recent studies on young tomato fruits.<a reveal-id="dsw051-B19" data-open="dsw051-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Furthermore, similar AS profiles have been observed in cereals<a reveal-id="dsw051-B27" data-open="dsw051-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and different tissues of <em>Zea mays</em>.<a reveal-id="dsw051-B51" data-open="dsw051-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> Both studies demonstrated that IR is the most dominant AS event, followed by AA, AD and ES, which is the general tendency in plants.<a reveal-id="dsw051-B20" data-open="dsw051-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Moreover, we report an increase of the number of genes for which transcripts with IR or ES are identified under HS conditions (<a reveal-id="dsw051-F1" data-open="dsw051-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Although this observation stands in contrast to the reported AS alteration in response to HS in <em>Physcomitrella patens</em>,<a reveal-id="dsw051-B18" data-open="dsw051-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> our findings are in line with the observed enhanced number of IR events in response to salt and cold stress in <em>A</em><em>.</em><em>thaliana</em> and <em>S</em><em>.</em><em>lycopersicum</em>.<a reveal-id="dsw051-B52" data-open="dsw051-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a><sup>,</sup><a reveal-id="dsw051-B53" data-open="dsw051-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> Consequently, these studies and our findings suggest that IR is a tissue independent general stress response mechanism.</p><p>In pollen, AS is a general regulatory process, independent of transctipt abundance or regulation under HS (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S6</a></span>), or even with the number of introns of a gene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S8</a></span>). The amount of observed IR and ES events after HS application is comparable between both tested cultivars (<a reveal-id="dsw051-F1" data-open="dsw051-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>) and many HS-dependent accumulated or reduced IR and ES events were commonly detected in both cultivars (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S5</a></span>). For example, about 80% of all genes with transcripts carrying a HS dependent accumulated IR were found in both, Moneymaker and Red Setter. However, the number of genes with HS-dependent accumulated AS (2,343) exceeds the number of genes with HS reduced AS by a factor of five (451). This difference might be due to the reduced functionality of the spliceosome under HS, which is supported by the high number of HS-induced IR events.<a reveal-id="dsw051-B54" data-open="dsw051-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a><sup>,</sup><a reveal-id="dsw051-B55" data-open="dsw051-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> However, the process of stress regulated AS is at least in parts specific. On the one hand, while many of alternative spliced transcripts carry NMD features, it was observed in <em>A. thaliana</em> that this does not necessarily lead to NMD.<a reveal-id="dsw051-B22" data-open="dsw051-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> On the other hand, we observed a decline of specific AS events in response to HS (<a reveal-id="dsw051-F3" data-open="dsw051-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>), which might be related to a reprogramming of splicing. The large overlap of observed AS events after HS between the two cultivars might even suggest that HS-induced AS is global regulatory mechanism rather than a cultivar specific event.</p><p>Although the degree of AS after HS is comparable between both cultivars (<a reveal-id="dsw051-F1" data-open="dsw051-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>), differences in the amount of IR and ES events were observed in the transcriptome of non-stressed pollen (<a reveal-id="dsw051-F1" data-open="dsw051-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Although the number of AS events is largely increased in pollen of Moneymaker after HS treatment, a high overlap (78%) of genes with transcripts with IR found under CO and HS conditions was observed for Red Setter. One possible interpretation could be that Red Setter exhibits higher sensitivity against HS than Moneymaker and HS-induced alteration already occur at conditions used as control. However, irrespective of the reason for the difference found for Red Setter and Moneymaker, due to the high amount of IR under control conditions Red Setter seems to have a lower capacity for alterations of the transcriptome by AS under HS than Moneymaker.</p>                            <h3 scrollto-destination=78054950 id="78054950" class="section-title" >4.2. HS-related AS targets genes involved in HSR</h3>
<p>Commonly identified HS induced AS events in both cultivars mainly target genes coding for factors involved in protein folding and gene expression among other processes (<a reveal-id="dsw051-F4" data-open="dsw051-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Within these functional categories, we realized an enrichment of AS events in HS related genes like Hsps and Hsfs exclusively under HS (<a reveal-id="dsw051-T2" data-open="dsw051-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Therefore, we aimed to analyse these two protein families in more detail with respect to HS-dependent accumulation of IR and ES events as not all members of these families were represented in the protein folding and gene expression category (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S10</a></span>). Within the set of Hsfs we observed up to four HS accumulated AS events in at least one cultivar, whereby the majority of these Hsfs were up-regulated upon HS like for example HsfA2 (Solyc08g062960) for both Moneymaker and Red Setter. In Hsp families 90, 70, 60 and 40 on the other side, the majority of Hsps were constitutively expressed but both HS induced and constitutively expressed Hsps showed a high degree of HS accumulated AS. For example, the two HS induced Hsp90s (Solyc06g036290 and Solyc03g007890) showed both two IR events and the constitutively expressed Hsp70-12 (Solyc01g106210) four IR events in both cultivars. A similar pattern was observed for the Hsp100 family, where the HS induced ClpB3 (Solyc02g088610) comprised seven HS accumulated IR events in Red Setter and the constitutively expressed ClpC1 (Solyc03g118340) two IR events in both cultivars.</p><p>The AS of Hsfs has been described in a variety of plant species, such as <em>Medicago sativa</em>, <em>A</em><em>.</em><em>thaliana</em> or <em>Oryza sativa</em><a reveal-id="dsw051-B56" data-open="dsw051-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a><sup>,</sup><a reveal-id="dsw051-B57" data-open="dsw051-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> and for different members of this family, including the HS induced HsfA2 in <em>A</em><em>.</em><em>thaliana</em>.<a reveal-id="dsw051-B12" data-open="dsw051-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsw051-B58" data-open="dsw051-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> Many of these Hsf splice variants were shown to be subjected to NMD; however, for the HsfA2-III splice variant of <em>A</em><em>.</em><em>thaliana</em> a truncated translation product was detected.<a reveal-id="dsw051-B57" data-open="dsw051-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> In contrast, for Hsps less about AS events is known. But for instance in human and rat, splice variants of heat-shock cognate protein 70 were identified, which are translated into truncated proteins lacking about 150 amino acid residues of the full-length protein.<a reveal-id="dsw051-B59" data-open="dsw051-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a><sup>,</sup><a reveal-id="dsw051-B60" data-open="dsw051-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> In addition, AS of mouse HSP47 was shown to be induced by heat shock leading to an extension of the 5′UTR by 169 nucleotides and by this to higher translation efficiency.<a reveal-id="dsw051-B61" data-open="dsw051-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> The HS accumulated IR events we observed for the different constitutively expressed and HS induced Hsps were all introducing PTCs. One possible reason might be that although we excluded a global dependency of AS detection from expression of the underlying gene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S6</a></span>), some of the IR events and especially the low abundant might be related to noisy splicing introduced by errors in the splicing process instead of a targeted process.<a reveal-id="dsw051-B62 dsw051-B63 dsw051-B64" data-open="dsw051-B62 dsw051-B63 dsw051-B64" class="link link-ref link-reveal xref-bibr"><sup>62–64</sup></a> On the other hand, the PTCs could lead to degradation of Hsp mRNAs by the NMD pathway if the mRNAs are NMD-sensitive.<a reveal-id="dsw051-B22" data-open="dsw051-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> As we used pooled samples extracted directly after HS and after a recovery period, the IR events observed for the constitutively expressed Hsps might be associated to the sample extracted directly after HS, where transcript levels of these Hsps should be reduced with the purpose of a higher translation rate of HS induced Hsps.<a reveal-id="dsw051-B65" data-open="dsw051-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> Similarly, IR events of HS induced Hsps might be related to the recovery period, where HS induced Hsps are no longer required and reduced by degradation of the respective PTC containing mRNAs.<a reveal-id="dsw051-B66" data-open="dsw051-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> This mechanism has been described in mammals where targeted introduction of PTCs by retained introns led to degradation of transcripts not required for the physiology of the cell.<a reveal-id="dsw051-B53" data-open="dsw051-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a></p>                            <h3 scrollto-destination=78054953 id="78054953" class="section-title" >4.3. HS-related AS results in alteration of protein sequences</h3>
<p>By reference based transcriptome assembly we detected various isoforms of pre-selected genes containing AS events accumulated or reduced dependent on HS compared with the annotation of tomato cultivar Heinz (ITAG 2.4).<a reveal-id="dsw051-B38" data-open="dsw051-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Within the 141 HS-specific and the 22 with HS-repressed transcripts, we observed an AS-induced length reduction of the CDS mainly due to PTC occurrence (<a reveal-id="dsw051-F5" data-open="dsw051-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Although most of these transcripts carry classical NMD features, it was recently shown that at least for some genes IR transcripts possessing NMD features are not sensitive to NMD.<a reveal-id="dsw051-B22" data-open="dsw051-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> This suggests that at least 36% of HS-specific transcripts (<a reveal-id="dsw051-F5" data-open="dsw051-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>, only IR) are putatively translated and not degraded. Among these proteins translated, six show a loss of at least one functional domain. The lack of one or more functional domains was shown to be a negative regulatory mechanism, where truncated isoforms may lack important domains but still form nonfunctional dimers that may compete with functional dimers.<a reveal-id="dsw051-B24" data-open="dsw051-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a><sup>,</sup><a reveal-id="dsw051-B67" data-open="dsw051-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> This negative regulation was for example shown for different transcription factors<a reveal-id="dsw051-B68" data-open="dsw051-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> and in our study we could also detect two transcription initiation factors (Solyc10g079360 and Solyc04g078050) having a putative competing isoform under HS. In addition, the observed C-terminal truncation of protein domains has been described recently.<a reveal-id="dsw051-B69" data-open="dsw051-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> Such terminal domain truncation, also known as domain atrophy, may be introduced by PTCs but contrary to the expectation leaves the domain often functional, albeit reduced. Next to the proteins showing AS-induced domain loss, 26% of HS-specific transcripts were perfectly spliced (<a reveal-id="dsw051-F5" data-open="dsw051-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>; perfectly). This means that under HS the normal protein function is maintained, whereas under CO either an alternative functionality of the protein or a degradation of the transcripts may occur.</p><div class="block-child-p">Up to now, the main focus of studies dealing with cellular response to HS was focused on genes showing expression changes upon HS<a reveal-id="dsw051-B70" data-open="dsw051-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a><sup>,</sup><a reveal-id="dsw051-B71" data-open="dsw051-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> as these are likely candidates with important roles in stress-regulatory networks.<a reveal-id="dsw051-B68" data-open="dsw051-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> However, this procedure may have underestimated the importance of genes with constant expression. These genes may not contribute to the cellular stress response by expression changes but AS transcripts, which offers a new layer of stress response. To verify whether such a regulation exists, we first extracted HS-specific transcripts (<a reveal-id="dsw051-T4" data-open="dsw051-T4" class="link link-reveal link-table xref-fig">Table 4</a>) and divided them in two pools comprised of (i) 14 transcripts containing an AS event inducing domain gain and/or loss (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&amp;Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S11</a></span>, red) and (ii) 37 perfectly spliced transcripts (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&amp;Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S11</a></span>, green). Second, expression levels of the underlying genes were compared for HS and CO of both cultivars and by this constantly expressed genes under both conditions could be identified (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&amp;Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S11</a></span>). The general tendency of expression levels was comparable between CO and HS for both pools supporting evidence for the proposed new layer of stress response (<a reveal-id="dsw051-F6" data-open="dsw051-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). Beside HSR regulated by up- or down-regulation of specific genes (<a reveal-id="dsw051-F6" data-open="dsw051-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>; 1 and 3) we propose HSR by AS in the case of constant gene expression (<a reveal-id="dsw051-F6" data-open="dsw051-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>; 2). The AS driven HSR can take place either under HS (<a reveal-id="dsw051-F6" data-open="dsw051-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>; 2A) leading to a competing isoform or under CO (<a reveal-id="dsw051-F6" data-open="dsw051-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>; 2B), at which the constitutively spliced isoform is expressed under HS. The transcript levels of selected genes supporting this new regulatory concept were verified by qPCR (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&amp;Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials S12</a></span>), which revealed constant expression between CO and HS samples for both Moneymaker and Red Setter. <div data-id="dsw051-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051f6.png?Expires=2147483647&amp;Signature=3CWHFulB6crwQ5SLosZ1GTXruvoXpg04XZFOi9dbR1ni1SoFv3NbzRkcjJKvssoWVDxb7UoyFw~mHDLq96-3GIlcimJ3vWG-OxuIMGRJm3PUyIRVyBvi~dNO6eyPSHeps8y785nqxTfnR2IlSmuEOlufGpLHvY0UVHASV0OS9IMP5vXpJBxjduql3u9gcAtV2E4VI3Iuocz4jppDsXSwZ0umKNXZ1WaqbbEFiCiMkPkryJZ7SYuifNAMUrZqOupdSd3u6ZZQazo8JHr6gNT7u~X1YCmG8O2z-dkqSF38jL7GdVk-DY5AFpJMjfBFhqiffJRyrq0X3uh632sOZaDFVg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="New putative regulatory pathways via AS due to HSR in pollen. The model shows two stages of regulating HSR: I. Transcript levels may be altered by reduction (1) or induction (3) of expression.; II. Independent of expression (2). For II. the HSR is regulated by AS creating isoforms leading to reduced/altered functionality. The concept of this regulation for HS (2A; Solyc09g074680; Solyc11g069490) and CO (2B; Solyc09g064920; Solyc06g069370) dependent AS is shown. Gene models for the annotated ITAG2.4 isoform and the HS or CO specific isoforms are provided as well as the domain architecture of the translated protein." path-from-xml="dsw051f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78054955/dsw051f6.tif" path-from-xml="dsw051f6.tif" target="_blank">View large</a><a section="78054955" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051f6.png?Expires=2147483647&amp;Signature=WnBkZ4FWOSuMOyqL8bVGm4AS~A8bjfiy7XcNxLk9G6UizGPTwnB-gsNwZeWgLrYMrAArer-VN7Z0gSvqKCcapU3aDqFFWeXOUfym9fLxnFeT2r31weYEbOONXmR--4-oyVAx2LfNtKgEsV9eY605iadCoHXSkl-GJli1SJ5Xp~hwTyktj6aMiV9PjRHO-X6~q8NyLLByohJvLfpOjo1Uv2u6uEpHQm31rvSUzGxmi2kK5fP5GcnR1I9dslZcDxWMLcYE2b~x8ov1Z9A2HjfcevrStpkMa8wF5cvmrM8S3UTbals~RcGi7oM6O16m7b8PKkzr73DYijITKsp0fBc52w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78054955&amp;ar=2738679&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw051f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>New putative regulatory pathways via AS due to HSR in pollen. The model shows two stages of regulating HSR: I. Transcript levels may be altered by reduction (1) or induction (3) of expression.; II. Independent of expression (2). For II. the HSR is regulated by AS creating isoforms leading to reduced/altered functionality. The concept of this regulation for HS (2A; Solyc09g074680; Solyc11g069490) and CO (2B; Solyc09g064920; Solyc06g069370) dependent AS is shown. Gene models for the annotated ITAG2.4 isoform and the HS or CO specific isoforms are provided as well as the domain architecture of the translated protein.</p></div></div><div content-id="dsw051-F6" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/m_dsw051f6.png?Expires=2147483647&amp;Signature=3CWHFulB6crwQ5SLosZ1GTXruvoXpg04XZFOi9dbR1ni1SoFv3NbzRkcjJKvssoWVDxb7UoyFw~mHDLq96-3GIlcimJ3vWG-OxuIMGRJm3PUyIRVyBvi~dNO6eyPSHeps8y785nqxTfnR2IlSmuEOlufGpLHvY0UVHASV0OS9IMP5vXpJBxjduql3u9gcAtV2E4VI3Iuocz4jppDsXSwZ0umKNXZ1WaqbbEFiCiMkPkryJZ7SYuifNAMUrZqOupdSd3u6ZZQazo8JHr6gNT7u~X1YCmG8O2z-dkqSF38jL7GdVk-DY5AFpJMjfBFhqiffJRyrq0X3uh632sOZaDFVg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="New putative regulatory pathways via AS due to HSR in pollen. The model shows two stages of regulating HSR: I. Transcript levels may be altered by reduction (1) or induction (3) of expression.; II. Independent of expression (2). For II. the HSR is regulated by AS creating isoforms leading to reduced/altered functionality. The concept of this regulation for HS (2A; Solyc09g074680; Solyc11g069490) and CO (2B; Solyc09g064920; Solyc06g069370) dependent AS is shown. Gene models for the annotated ITAG2.4 isoform and the HS or CO specific isoforms are provided as well as the domain architecture of the translated protein." path-from-xml="dsw051f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78054955/dsw051f6.tif" path-from-xml="dsw051f6.tif" target="_blank">View large</a><a section="78054955" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051f6.png?Expires=2147483647&amp;Signature=WnBkZ4FWOSuMOyqL8bVGm4AS~A8bjfiy7XcNxLk9G6UizGPTwnB-gsNwZeWgLrYMrAArer-VN7Z0gSvqKCcapU3aDqFFWeXOUfym9fLxnFeT2r31weYEbOONXmR--4-oyVAx2LfNtKgEsV9eY605iadCoHXSkl-GJli1SJ5Xp~hwTyktj6aMiV9PjRHO-X6~q8NyLLByohJvLfpOjo1Uv2u6uEpHQm31rvSUzGxmi2kK5fP5GcnR1I9dslZcDxWMLcYE2b~x8ov1Z9A2HjfcevrStpkMa8wF5cvmrM8S3UTbals~RcGi7oM6O16m7b8PKkzr73DYijITKsp0fBc52w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78054955&amp;ar=2738679&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw051f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>New putative regulatory pathways via AS due to HSR in pollen. The model shows two stages of regulating HSR: I. Transcript levels may be altered by reduction (1) or induction (3) of expression.; II. Independent of expression (2). For II. the HSR is regulated by AS creating isoforms leading to reduced/altered functionality. The concept of this regulation for HS (2A; Solyc09g074680; Solyc11g069490) and CO (2B; Solyc09g064920; Solyc06g069370) dependent AS is shown. Gene models for the annotated ITAG2.4 isoform and the HS or CO specific isoforms are provided as well as the domain architecture of the translated protein.</p></div></div></div><p>To exemplify regulation under HS by AS, we chose two candidates of the first pool (i), namely the ubiquitin ligase complex subunit Cullin 1B (Solyc09g074680) and the serine/threonine-protein phosphatase 6 regulatory subunit 3 (PP6R3, Solyc11g069490). Cullin proteins are evolutionary conserved proteins acting as scaffold in multi-subunit CRL complexes required for ubiquitin-dependent protein degradation.<a reveal-id="dsw051-B72" data-open="dsw051-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> Cullin proteins are composed of a conserved cullin domain and a neddylation site.<a reveal-id="dsw051-B73" data-open="dsw051-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a> Conjugation of the ubiquitine-like molecule Nedd8 to the neddylation site leads to activation of the CRL complex.<a reveal-id="dsw051-B72" data-open="dsw051-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> In addition, CRL complexes were shown to be involved in plant abiotic stress pathways including first indications for their involvement in temperature stress response.<a reveal-id="dsw051-B74" data-open="dsw051-B74" class="link link-ref link-reveal xref-bibr"><sup>74</sup></a> The isoform only detected under HS revealed retention of intron 17 leading to a PTC and a C-terminal truncation of 16.47%. The truncation comprised the entire neddylation site (Cullin_Nedd8) and a small portion of the cullin domain (Cullin). A possible effect of the truncation might be that the CRL complex is not fully assembled or remains in an inactive state due to the inability for neddylation. This assumption is supported by results in <em>Xaneopus laevis</em>, where C-terminally truncated <em>Cullin-1</em> constructs were shown to have negative effects on ubiquitination by titrating out endogeneous components into inactive complexes, leading to an increase of degradation targets.<a reveal-id="dsw051-B75" data-open="dsw051-B75" class="link link-ref link-reveal xref-bibr"><sup>75</sup></a> The second protein exemplifying response by AS under HS is PP6R3. PP6R3 forms together with the catalytic subunit PP6c and an ankyrin repeat-containing protein the heterotrimeric PP6 holoenzyme complex, which is required for regulation of different signaling pathways.<a reveal-id="dsw051-B76" data-open="dsw051-B76" class="link link-ref link-reveal xref-bibr"><sup>76</sup></a> The involvement of serine/threonine phosphatases in HSR was shown by overexpression of the AtPP5, which led to enhanced heat shock resistance of <em>A</em><em>.</em><em>thaliana</em>.<a reveal-id="dsw051-B77" data-open="dsw051-B77" class="link link-ref link-reveal xref-bibr"><sup>77</sup></a> The HS-specific PP6R3 isoform showed retention of intron 12 comprising a PTC leading to a truncation including about 51% of the SAPS domain. If the truncated SAPS domain is still functional, like in the case of domain atrophy,<a reveal-id="dsw051-B69" data-open="dsw051-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a> the binding to PP6c might still be feasible, as the SAPS domain is sufficient for binding. However, the interaction of PP6R3 with the ankyrin-repeat subunit would not be feasible anymore as the C-terminal region is required for interaction.<a reveal-id="dsw051-B78" data-open="dsw051-B78" class="link link-ref link-reveal xref-bibr"><sup>78</sup></a> This may suggest that like in the case of Cullin 1B, the observed PP6R3 isoform leads to inactive PP6 complexes having a contrary effect on signal transduction. In the end, these results exemplify our proposed model, at which the transcript abundance is constant but different AS isoforms serve as alternative layer of HSR.</p>                            <h2 scrollto-destination=78054957 id="78054957" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We would like to thank Daniela Bublak for excellent technical assistance and members of the SPOT-ITN consortium for helpful comments during the manuscript preparation. We are grateful to Holger Schranz and Gerald Kircher for maintenance of plants in the greenhouse. We acknowledge the contribution of all SPOT-ITN members in discussion of the results. The article is dedicated to two of our wonderful colleagues that have contributed to the field of HSR of tomato of the last four decades, a work that has inspired us in our research: Prof Dr Lutz Nover and Dr Klaus-Dieter Scharf.</p>                            <h3 scrollto-destination=78054959 id="78054959" class="section-title" >Conflict of interests</h3>
<p>None declared.</p>                            <h3 scrollto-destination=78054961 id="78054961" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=78054963 id="78054963" class="section-title" >Funding</h3>
<p>The work was supported by grants of the Deutsche Forschungsgemeinschaft (Schleiff: CRC-SFB902 B9; EXC-Cluster of excellence macromolecular complexes) and from the EU/Marie Curie (project: Pollen thermotolerance and crop fertility, acronym: SPOT-ITN, grant agreement No. 289220).</p>                            <h2 scrollto-destination=78054965 id="78054965" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw051-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finka</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Mattoo</div> <div class="given-names">R. U.</div></div>, <div class="name string-name"><div class="surname">Goloubinoff</div> <div class="given-names">P.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Meta-analysis of heat- and chemically upregulated chaperone genes in plant and human cells</div>. <div class="source">Cell Stress Chaperones</div> , <div class="volume">16</div>, <div class="fpage">15</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Meta-analysis%20of%20heat-%20and%20chemically%20upregulated%20chaperone%20genes%20in%20plant%20and%20human%20cells&amp;author=A.%20Finka&amp;author=R.%20U.%20Mattoo&amp;author=P.%20Goloubinoff&amp;publication_year=2011&amp;journal=Cell%20Stress%20Chaperones&amp;volume=16&amp;pages=15-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12192-010-0216-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12192-010-0216-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12192-010-0216-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20694844" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fragkostefanakis</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Simm</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Paul</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Bublak</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Scharf</div> <div class="given-names">K. D.</div></div>, <div class="name string-name"><div class="surname">Schleiff</div> <div class="given-names">E.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Chaperone network composition in Solanum lycopersicum explored by transcriptome profiling and microarray meta-analysis</div>. <div class="source">Plant Cell Environ</div> ., <div class="volume">38</div>, <div class="fpage">693</div>–<div class="lpage">709</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chaperone%20network%20composition%20in%20Solanum%20lycopersicum%20explored%20by%20transcriptome%20profiling%20and%20microarray%20meta-analysis&amp;author=S.%20Fragkostefanakis&amp;author=S.%20Simm&amp;author=P.%20Paul&amp;author=D.%20Bublak&amp;author=K.%20D.%20Scharf&amp;author=E.%20Schleiff&amp;publication_year=2015&amp;journal=Plant%20Cell%20Environ&amp;volume=38&amp;pages=693-709" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pce.2015.38.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpce.2015.38.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpce.2015.38.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25124075" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jung</div> <div class="given-names">K.H.</div></div>, <div class="name string-name"><div class="surname">Ko</div> <div class="given-names">H.J.</div></div>, <div class="name string-name"><div class="surname">Nguyen</div> <div class="given-names">M.X.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S.R.</div></div>, <div class="name string-name"><div class="surname">Ronald</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">An</div> <div class="given-names">G.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Genome-wide identification and analysis of early heat stress responsive genes in rice</div>. <div class="source">J. Plant Biol</div> ., <div class="volume">55</div>, <div class="fpage">10</div>.</div></div></div></div><div content-id="dsw051-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Larkindale</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Hall</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Knight</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Vierling</div> <div class="given-names">E.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Heat stress phenotypes of Arabidopsis mutants implicate multiple signaling pathways in the acquisition of thermotolerance</div>. <div class="source">Plant Physiol</div> ., <div class="volume">138</div>, <div class="fpage">882</div>–<div class="lpage">897</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heat%20stress%20phenotypes%20of%20Arabidopsis%20mutants%20implicate%20multiple%20signaling%20pathways%20in%20the%20acquisition%20of%20thermotolerance&amp;author=J.%20Larkindale&amp;author=J.D.%20Hall&amp;author=M.R.%20Knight&amp;author=E.%20Vierling&amp;publication_year=2005&amp;journal=Plant%20Physiol&amp;volume=138&amp;pages=882-897" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.105.062257" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.105.062257" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.105.062257"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15923322" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.F.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kakani</div> <div class="given-names">V.G.</div></div>, <div class="name string-name"><div class="surname">Mahalingam</div> <div class="given-names">R.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Transcriptome analysis of heat stress response in switchgrass (Panicum virgatum L.)</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">13</div>, <div class="fpage">153</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20analysis%20of%20heat%20stress%20response%20in%20switchgrass%20(Panicum%20virgatum%20L.)&amp;author=Y.F.%20Li&amp;author=Y.%20Wang&amp;author=Y.%20Tang&amp;author=V.G.%20Kakani&amp;author=R.%20Mahalingam&amp;publication_year=2013&amp;journal=BMC%20Plant%20Biol&amp;volume=13&amp;pages=153" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-13-153" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-13-153" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-13-153"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24093800" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sarkar</div> <div class="given-names">N.K.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">Y.K.</div></div>, <div class="name string-name"><div class="surname">Grover</div> <div class="given-names">A.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Coexpression network analysis associated with call of rice seedlings for encountering heat stress</div>. <div class="source">Plant Mol. Biol</div> ., <div class="volume">84</div>, <div class="fpage">125</div>–<div class="lpage">143</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coexpression%20network%20analysis%20associated%20with%20call%20of%20rice%20seedlings%20for%20encountering%20heat%20stress&amp;author=N.K.%20Sarkar&amp;author=Y.K.%20Kim&amp;author=A.%20Grover&amp;publication_year=2014&amp;journal=Plant%20Mol.%20Biol&amp;volume=84&amp;pages=125-143" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-013-0123-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-013-0123-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-013-0123-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23975147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Chu</div> <div class="given-names">P.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">NnHSP17.5, a cytosolic class II small heat shock protein gene from Nelumbo nucifera, contributes to seed germination vigor and seedling thermotolerance in transgenic Arabidopsis</div>. <div class="source">Plant Cell Rep</div> ., <div class="volume">31</div>, <div class="fpage">379</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NnHSP17.5%2C%20a%20cytosolic%20class%20II%20small%20heat%20shock%20protein%20gene%20from%20Nelumbo%20nucifera%2C%20contributes%20to%20seed%20germination%20vigor%20and%20seedling%20thermotolerance%20in%20transgenic%20Arabidopsis&amp;author=Y.%20Zhou&amp;author=H.%20Chen&amp;author=P.%20Chu&amp;publication_year=2012&amp;journal=Plant%20Cell%20Rep&amp;volume=31&amp;pages=379-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00299-011-1173-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00299-011-1173-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00299-011-1173-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22009054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sung</div> <div class="given-names">D. Y.</div></div>, <div class="name string-name"><div class="surname">Guy</div> <div class="given-names">C. L.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Physiological and molecular assessment of altered expression of Hsc70-1 in Arabidopsis. Evidence for pleiotropic consequences</div>. <div class="source">Plant Physiol</div> ., <div class="volume">132</div>, <div class="fpage">979</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Physiological%20and%20molecular%20assessment%20of%20altered%20expression%20of%20Hsc70-1%20in%20Arabidopsis.%20Evidence%20for%20pleiotropic%20consequences&amp;author=D.%20Y.%20Sung&amp;author=C.%20L.%20Guy&amp;publication_year=2003&amp;journal=Plant%20Physiol&amp;volume=132&amp;pages=979-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.102.019398" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.102.019398" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.102.019398"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12805626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Montero-Barrientos</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hermosa</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Cardoza</div> <div class="given-names">R.E.</div></div>, <div class="name string-name"><div class="surname">Gutierrez</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Nicolas</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Monte</div> <div class="given-names">E.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Transgenic expression of the Trichoderma harzianum hsp70 gene increases Arabidopsis resistance to heat and other abiotic stresses</div>. <div class="source">J. Plant Physiol</div> ., <div class="volume">167</div>, <div class="fpage">659</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transgenic%20expression%20of%20the%20Trichoderma%20harzianum%20hsp70%20gene%20increases%20Arabidopsis%20resistance%20to%20heat%20and%20other%20abiotic%20stresses&amp;author=M.%20Montero-Barrientos&amp;author=R.%20Hermosa&amp;author=R.E.%20Cardoza&amp;author=S.%20Gutierrez&amp;author=C.%20Nicolas&amp;author=E.%20Monte&amp;publication_year=2010&amp;journal=J.%20Plant%20Physiol&amp;volume=167&amp;pages=659-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jplph.2009.11.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jplph.2009.11.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jplph.2009.11.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20080316" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cazale</div> <div class="given-names">A. C.</div></div>, <div class="name string-name"><div class="surname">Clement</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Chiarenza</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2009</div>, 
<div class="article-title">Altered expression of cytosolic/nuclear HSC70-1 molecular chaperone affects development and abiotic stress tolerance in Arabidopsis thaliana</div>. <div class="source">J. Exp. Bot</div> ., <div class="volume">60</div>, <div class="fpage">2653</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Altered%20expression%20of%20cytosolic%2Fnuclear%20HSC70-1%20molecular%20chaperone%20affects%20development%20and%20abiotic%20stress%20tolerance%20in%20Arabidopsis%20thaliana&amp;author=A.%20C.%20Cazale&amp;author=M.%20Clement&amp;author=S.%20Chiarenza&amp;publication_year=2009&amp;journal=J.%20Exp.%20Bot&amp;volume=60&amp;pages=2653-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erp109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferp109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferp109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19443614" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Scharf</div> <div class="given-names">K. D.</div></div>, <div class="name string-name"><div class="surname">Berberich</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ebersberger</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Nover</div> <div class="given-names">L.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">The plant heat stress transcription factor (Hsf) family: structure, function and evolution</div>. <div class="source">Biochim. Biophys. Acta</div> , <div class="volume">1819</div>, <div class="fpage">104</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20plant%20heat%20stress%20transcription%20factor%20(Hsf)%20family%3A%20structure%2C%20function%20and%20evolution&amp;author=K.%20D.%20Scharf&amp;author=T.%20Berberich&amp;author=I.%20Ebersberger&amp;author=L.%20Nover&amp;publication_year=2012&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=1819&amp;pages=104-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbagrm.2011.10.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbagrm.2011.10.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbagrm.2011.10.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22033015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guerra</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Crosatti</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Khoshro</div> <div class="given-names">H.H.</div></div>, <div class="name string-name"><div class="surname">Mastrangelo</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Mica</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Mazzucotelli</div> <div class="given-names">E.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Post-transcriptional and post-translational regulations of drought and heat response in plants: a spider’s web of mechanisms</div>. <div class="source">Front. Plant Science</div> , <div class="volume">6</div>, 57.</div></div></div></div><div content-id="dsw051-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kornblihtt</div> <div class="given-names">A.R.</div></div>, <div class="name string-name"><div class="surname">Schor</div> <div class="given-names">I.E.</div></div>, <div class="name string-name"><div class="surname">Allo</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Dujardin</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Petrillo</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Munoz</div> <div class="given-names">M.J.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Alternative splicing: a pivotal step between eukaryotic transcription and translation</div>. <div class="source">Nat. Rev. Mol. Cell Biol</div> ., <div class="volume">14</div>, <div class="fpage">153</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%3A%20a%20pivotal%20step%20between%20eukaryotic%20transcription%20and%20translation&amp;author=A.R.%20Kornblihtt&amp;author=I.E.%20Schor&amp;author=M.%20Allo&amp;author=G.%20Dujardin&amp;author=E.%20Petrillo&amp;author=M.J.%20Munoz&amp;publication_year=2013&amp;journal=Nat.%20Rev.%20Mol.%20Cell%20Biol&amp;volume=14&amp;pages=153-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrm3525" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrm3525" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrm3525"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23385723" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Moore</div> <div class="given-names">M. J.</div></div>, <div class="name string-name"><div class="surname">Proudfoot</div> <div class="given-names">N. J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Pre-mRNA processing reaches back to transcription and ahead to translation</div>. <div class="source">Cell</div> , <div class="volume">136</div>, <div class="fpage">688</div>–<div class="lpage">700</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pre-mRNA%20processing%20reaches%20back%20to%20transcription%20and%20ahead%20to%20translation&amp;author=M.%20J.%20Moore&amp;author=N.%20J.%20Proudfoot&amp;publication_year=2009&amp;journal=Cell&amp;volume=136&amp;pages=688-700" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2009.02.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2009.02.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2009.02.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19239889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Staiger</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">J. W.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Alternative splicing at the intersection of biological timing, development, and stress responses</div>. <div class="source">Plant Cell</div> , <div class="volume">25</div>, <div class="fpage">3640</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20at%20the%20intersection%20of%20biological%20timing%2C%20development%2C%20and%20stress%20responses&amp;author=D.%20Staiger&amp;author=J.%20W.%20Brown&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=3640-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.113803" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.113803" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.113803"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24179132" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Reddy</div> <div class="given-names">A.S.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Alternative splicing of pre-messenger RNAs in plants in the genomic era</div>. <div class="source">Annu. Rev. Plant Biol</div> ., <div class="volume">58</div>, <div class="fpage">267</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20of%20pre-messenger%20RNAs%20in%20plants%20in%20the%20genomic%20era&amp;author=A.S.%20Reddy&amp;publication_year=2007&amp;journal=Annu.%20Rev.%20Plant%20Biol&amp;volume=58&amp;pages=267-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.58.032806.103754" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.58.032806.103754" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.58.032806.103754"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17222076" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Burge</div> <div class="given-names">C.B.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Splicing regulation: from a parts list of regulatory elements to an integrated splicing code</div>. <div class="source">RNA</div> , <div class="volume">14</div>, <div class="fpage">802</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Splicing%20regulation%3A%20from%20a%20parts%20list%20of%20regulatory%20elements%20to%20an%20integrated%20splicing%20code&amp;author=Z.%20Wang&amp;author=C.B.%20Burge&amp;publication_year=2008&amp;journal=RNA&amp;volume=14&amp;pages=802-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.876308" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.876308" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.876308"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18369186" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chang</div> <div class="given-names">C. Y.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">W. D.</div></div>, <div class="name string-name"><div class="surname">Tu</div> <div class="given-names">S. L.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Genome-wide analysis of heat-sensitive alternative splicing in Physcomitrella patens</div>. <div class="source">Plant Physiol</div> ., <div class="volume">165</div>, <div class="fpage">826</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20heat-sensitive%20alternative%20splicing%20in%20Physcomitrella%20patens&amp;author=C.%20Y.%20Chang&amp;author=W.%20D.%20Lin&amp;author=S.%20L.%20Tu&amp;publication_year=2014&amp;journal=Plant%20Physiol&amp;volume=165&amp;pages=826-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.113.230540" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.113.230540" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.113.230540"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24777346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sun</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Xiao</div> <div class="given-names">H.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Identification of alternative splicing events by RNA sequencing in early growth tomato fruits</div>. <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">948</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20alternative%20splicing%20events%20by%20RNA%20sequencing%20in%20early%20growth%20tomato%20fruits&amp;author=Y.%20Sun&amp;author=H.%20Xiao&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=948" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-2128-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-2128-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-2128-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26573826" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Loraine</div> <div class="given-names">A.E.</div></div>, <div class="name string-name"><div class="surname">Dickerson</div> <div class="given-names">J.A.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Comparisons of computational methods for differential alternative splicing detection using RNA-seq in plant systems</div>. <div class="source">BMC Bioinform</div> ., <div class="volume">15</div>.</div></div></div></div><div content-id="dsw051-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Syed</div> <div class="given-names">N.H.</div></div>, <div class="name string-name"><div class="surname">Kalyna</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Marquez</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Barta</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">J.W.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Alternative splicing in plants–coming of age</div>. <div class="source">Trends Plant Sci</div> ., <div class="volume">17</div>, <div class="fpage">616</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20in%20plants%E2%80%93coming%20of%20age&amp;author=N.H.%20Syed&amp;author=M.%20Kalyna&amp;author=Y.%20Marquez&amp;author=A.%20Barta&amp;author=J.W.%20Brown&amp;publication_year=2012&amp;journal=Trends%20Plant%20Sci&amp;volume=17&amp;pages=616-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2012.06.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2012.06.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2012.06.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22743067" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kalyna</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Simpson</div> <div class="given-names">C.G.</div></div>, <div class="name string-name"><div class="surname">Syed</div> <div class="given-names">N.H.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Alternative splicing and nonsense-mediated decay modulate expression of important regulatory genes in Arabidopsis</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">2454</div>–<div class="lpage">69</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%20and%20nonsense-mediated%20decay%20modulate%20expression%20of%20important%20regulatory%20genes%20in%20Arabidopsis&amp;author=M.%20Kalyna&amp;author=C.G.%20Simpson&amp;author=N.H.%20Syed&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=2454-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr932" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr932" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr932"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22127866" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mastrangelo</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Marone</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Laido</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">De Leonardis</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">De Vita</div> <div class="given-names">P.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Alternative splicing: enhancing ability to cope with stress via transcriptome plasticity</div>. <div class="source">Plant Sci</div> ., <div class="volume">185-186</div>, <div class="fpage">40</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20splicing%3A%20enhancing%20ability%20to%20cope%20with%20stress%20via%20transcriptome%20plasticity&amp;author=A.M.%20Mastrangelo&amp;author=D.%20Marone&amp;author=G.%20Laido&amp;author=A.M.%20De%20Leonardis&amp;author=P.%20De%20Vita&amp;publication_year=2012&amp;journal=Plant%20Sci&amp;volume=185-186&amp;pages=40-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plantsci.2011.09.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2011.09.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plantsci.2011.09.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22325865" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Reddy</div> <div class="given-names">A.S.</div></div>, <div class="name string-name"><div class="surname">Marquez</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kalyna</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Barta</div> <div class="given-names">A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Complexity of the alternative splicing landscape in plants</div>. <div class="source">Plant Cell</div> , <div class="volume">25</div>, <div class="fpage">3657</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complexity%20of%20the%20alternative%20splicing%20landscape%20in%20plants&amp;author=A.S.%20Reddy&amp;author=Y.%20Marquez&amp;author=M.%20Kalyna&amp;author=A.%20Barta&amp;publication_year=2013&amp;journal=Plant%20Cell&amp;volume=25&amp;pages=3657-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.113.117523" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.113.117523" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.113.117523"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24179125" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Filichkin</div> <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Priest</div> <div class="given-names">H.D.</div></div>, <div class="name string-name"><div class="surname">Givan</div> <div class="given-names">S.A.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Genome-wide mapping of alternative splicing in Arabidopsis thaliana</div>. <div class="source">Genome Res</div> ., <div class="volume">20</div>, <div class="fpage">45</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20mapping%20of%20alternative%20splicing%20in%20Arabidopsis%20thaliana&amp;author=S.A.%20Filichkin&amp;author=H.D.%20Priest&amp;author=S.A.%20Givan&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=45-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.093302.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.093302.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.093302.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19858364" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ner-Gaon</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Halachmi</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Savaldi-Goldstein</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Rubin</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Ophir</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Fluhr</div> <div class="given-names">R.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Intron retention is a major phenomenon in alternative splicing in Arabidopsis</div>. <div class="source">Plant J</div> ., <div class="volume">39</div>, <div class="fpage">877</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intron%20retention%20is%20a%20major%20phenomenon%20in%20alternative%20splicing%20in%20Arabidopsis&amp;author=H.%20Ner-Gaon&amp;author=R.%20Halachmi&amp;author=S.%20Savaldi-Goldstein&amp;author=E.%20Rubin&amp;author=R.%20Ophir&amp;author=R.%20Fluhr&amp;publication_year=2004&amp;journal=Plant%20J&amp;volume=39&amp;pages=877-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2004.39.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2004.39.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2004.39.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15341630" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Min</div> <div class="given-names">X.J.</div></div>, <div class="name string-name"><div class="surname">Powell</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Braessler</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Meinken</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Sablok</div> <div class="given-names">G.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Genome-wide cataloging and analysis of alternatively spliced genes in cereal crops</div>. <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">721</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20cataloging%20and%20analysis%20of%20alternatively%20spliced%20genes%20in%20cereal%20crops&amp;author=X.J.%20Min&amp;author=B.%20Powell&amp;author=J.%20Braessler&amp;author=J.%20Meinken&amp;author=F.%20Yu&amp;author=G.%20Sablok&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=721" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1914-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1914-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1914-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26391769" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Loraine</div> <div class="given-names">A.E.</div></div>, <div class="name string-name"><div class="surname">McCormick</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Estrada</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Patel</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Qin</div> <div class="given-names">P.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">RNA-seq of arabidopsis pollen uncovers novel transcription and alternative splicing</div>. <div class="source">Plant Physiol</div> ., <div class="volume">162</div>, <div class="fpage">1092</div>–<div class="lpage">109</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-seq%20of%20arabidopsis%20pollen%20uncovers%20novel%20transcription%20and%20alternative%20splicing&amp;author=A.E.%20Loraine&amp;author=S.%20McCormick&amp;author=A.%20Estrada&amp;author=K.%20Patel&amp;author=P.%20Qin&amp;publication_year=2013&amp;journal=Plant%20Physiol&amp;volume=162&amp;pages=1092-109" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.112.211441" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.112.211441" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.112.211441"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23590974" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dai</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Chong</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Xue</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">T.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Proteomic analyses of Oryza sativa mature pollen reveal novel proteins associated with pollen germination and tube growth</div>. <div class="source">Proteomics</div> , <div class="volume">6</div>, <div class="fpage">2504</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteomic%20analyses%20of%20Oryza%20sativa%20mature%20pollen%20reveal%20novel%20proteins%20associated%20with%20pollen%20germination%20and%20tube%20growth&amp;author=S.%20Dai&amp;author=L.%20Li&amp;author=T.%20Chen&amp;author=K.%20Chong&amp;author=Y.%20Xue&amp;author=T.%20Wang&amp;publication_year=2006&amp;journal=Proteomics&amp;volume=6&amp;pages=2504-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1615-9861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1615-9861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1615-9861"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16548068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bokszczanin</div> <div class="given-names">K.L.</div></div><div class="collab">Solanaceae Pollen Thermotolerance Initial Training Network, C</div><div class="name string-name"><div class="surname">Fragkostefanakis</div> <div class="given-names">S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Perspectives on deciphering mechanisms underlying plant heat stress response and thermotolerance</div>. <div class="source">Front. Plant Sci</div> ., <div class="volume">4</div>, <div class="fpage">315</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Perspectives%20on%20deciphering%20mechanisms%20underlying%20plant%20heat%20stress%20response%20and%20thermotolerance&amp;author=K.L.%20Bokszczanin&amp;author=S.%20Fragkostefanakis&amp;publication_year=2013&amp;journal=Front.%20Plant%20Sci&amp;volume=4&amp;pages=315" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2013.00315" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2013.00315" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2013.00315"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23986766" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Giorno</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Wolters-Arts</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Mariani</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Rieu</div> <div class="given-names">I.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Ensuring reproduction at high temperatures: the heat stress response during anther and pollen development</div>. <div class="source">Plants</div> , <div class="volume">2</div>, <div class="fpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ensuring%20reproduction%20at%20high%20temperatures%3A%20the%20heat%20stress%20response%20during%20anther%20and%20pollen%20development&amp;author=F.%20Giorno&amp;author=M.%20Wolters-Arts&amp;author=C.%20Mariani&amp;author=I.%20Rieu&amp;publication_year=2013&amp;journal=Plants&amp;volume=2&amp;pages=17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3390/plants2030489" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3390%2Fplants2030489" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3390%2Fplants2030489"> </span></div> </div></div></div></div></div><div content-id="dsw051-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pressman</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Peet</div> <div class="given-names">M.M.</div></div>, <div class="name string-name"><div class="surname">Pharr</div> <div class="given-names">D.M.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">The effect of heat stress on tomato pollen characteristics is associated with changes in carbohydrate concentration in the developing anthers</div>. <div class="source">Ann. Bot</div> ., <div class="volume">90</div>, <div class="fpage">631</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20effect%20of%20heat%20stress%20on%20tomato%20pollen%20characteristics%20is%20associated%20with%20changes%20in%20carbohydrate%20concentration%20in%20the%20developing%20anthers&amp;author=E.%20Pressman&amp;author=M.M.%20Peet&amp;author=D.M.%20Pharr&amp;publication_year=2002&amp;journal=Ann.%20Bot&amp;volume=90&amp;pages=631-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/aob/mcf240" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Faob%2Fmcf240" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Faob%2Fmcf240"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12466104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rutley</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Twell</div> <div class="given-names">D.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">A decade of pollen transcriptomics</div>. <div class="source">Plant Reprod</div> ., <div class="volume">28</div>, <div class="fpage">73</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20decade%20of%20pollen%20transcriptomics&amp;author=N.%20Rutley&amp;author=D.%20Twell&amp;publication_year=2015&amp;journal=Plant%20Reprod&amp;volume=28&amp;pages=73-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00497-015-0261-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00497-015-0261-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00497-015-0261-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25761645" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fragkostefanakis</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Mesihovic</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Simm</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">HsfA2 controls the activity of developmentally and stress-regulated heat stress protection mechanisms in tomato male reproductive tissues</div>. <div class="source">Plant Physiol</div> ., <strong>170</strong>, 2461–77.</div></div></div></div><div content-id="dsw051-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Paul</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Chaturvedi</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Selymesi</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">The membrane proteome of male gametophyte in Solanum lycopersicum</div>. <div class="source">J. Proteomics</div> , <div class="volume">131</div>, <div class="fpage">48</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20membrane%20proteome%20of%20male%20gametophyte%20in%20Solanum%20lycopersicum&amp;author=P.%20Paul&amp;author=P.%20Chaturvedi&amp;author=M.%20Selymesi&amp;publication_year=2016&amp;journal=J.%20Proteomics&amp;volume=131&amp;pages=48-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jprot.2015.10.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jprot.2015.10.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jprot.2015.10.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26455813" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bombarely</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Menda</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Tecle</div> <div class="given-names">I.Y.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">The Sol Genomics Network (solgenomics.net): growing tomatoes using Perl</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">D1149</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sol%20Genomics%20Network%20(solgenomics.net)%3A%20growing%20tomatoes%20using%20Perl&amp;author=A.%20Bombarely&amp;author=N.%20Menda&amp;author=I.Y.%20Tecle&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=D1149-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq866" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq866" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq866"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20935049" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pimentel</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kelley</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions</div>. <div class="source">Genome Biol</div> ., <div class="volume">14</div>, <div class="fpage">R36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat2%3A%20accurate%20alignment%20of%20transcriptomes%20in%20the%20presence%20of%20insertions%2C%20deletions%20and%20gene%20fusions&amp;author=D.%20Kim&amp;author=G.%20Pertea&amp;author=C.%20Trapnell&amp;author=H.%20Pimentel&amp;author=R.%20Kelley&amp;author=S.L.%20Salzberg&amp;publication_year=2013&amp;journal=Genome%20Biol&amp;volume=14&amp;pages=R36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-4-r36" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-4-r36" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-4-r36"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23618408" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tomato Genome</div> <div class="given-names">C.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">The tomato genome sequence provides insights into fleshy fruit evolution</div>. <div class="source">Nature</div> , <div class="volume">485</div>, <div class="fpage">635</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20tomato%20genome%20sequence%20provides%20insights%20into%20fleshy%20fruit%20evolution&amp;author=C.%20Tomato%20Genome&amp;publication_year=2012&amp;journal=Nature&amp;volume=485&amp;pages=635-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22660326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Anders</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Pyl</div> <div class="given-names">P.T.</div></div>, <div class="name string-name"><div class="surname">Huber</div> <div class="given-names">W.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">HTSeq–a Python framework to work with high-throughput sequencing data</div>. <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">166</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HTSeq%E2%80%93a%20Python%20framework%20to%20work%20with%20high-throughput%20sequencing%20data&amp;author=S.%20Anders&amp;author=P.T.%20Pyl&amp;author=W.%20Huber&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=166-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu638" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu638" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu638"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25260700" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Livak</div> <div class="given-names">K.J.</div></div>, <div class="name string-name"><div class="surname">Schmittgen</div> <div class="given-names">T.D.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Analysis of relative gene expression data using real-time quantitative PCR and the 2(-Delta Delta C(T)) Method</div>. <div class="source">Methods</div> , <div class="volume">25</div>, <div class="fpage">402</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20relative%20gene%20expression%20data%20using%20real-time%20quantitative%20PCR%20and%20the%202(-Delta%20Delta%20C(T))%20Method&amp;author=K.J.%20Livak&amp;author=T.D.%20Schmittgen&amp;publication_year=2001&amp;journal=Methods&amp;volume=25&amp;pages=402-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/meth.2001.1262" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fmeth.2001.1262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fmeth.2001.1262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11846609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ashburner</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ball</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">Blake</div> <div class="given-names">J.A.</div></div></div>, et al.  . <div class="year">2000</div>, 
<div class="article-title">Gene ontology: tool for the unification of biology. The Gene Ontology Consortium</div>. <div class="source">Nat. Genet</div> ., <div class="volume">25</div>, <div class="fpage">25</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20ontology%3A%20tool%20for%20the%20unification%20of%20biology.%20The%20Gene%20Ontology%20Consortium&amp;author=M.%20Ashburner&amp;author=C.A.%20Ball&amp;author=J.A.%20Blake&amp;publication_year=2000&amp;journal=Nat.%20Genet&amp;volume=25&amp;pages=25-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/75556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F75556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jones</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Binns</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">H.Y.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">InterProScan 5: genome-scale protein function classification</div>. <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">1236</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%205%3A%20genome-scale%20protein%20function%20classification&amp;author=P.%20Jones&amp;author=D.%20Binns&amp;author=H.Y.%20Chang&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=1236-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24451626" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shannon</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Markiel</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ozier</div> <div class="given-names">O.</div></div></div>, et al.  . <div class="year">2003</div>, 
<div class="article-title">Cytoscape: a software environment for integrated models of biomolecular interaction networks</div>. <div class="source">Genome Res</div> ., <div class="volume">13</div>, <div class="fpage">2498</div>–<div class="lpage">504</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytoscape%3A%20a%20software%20environment%20for%20integrated%20models%20of%20biomolecular%20interaction%20networks&amp;author=P.%20Shannon&amp;author=A.%20Markiel&amp;author=O.%20Ozier&amp;publication_year=2003&amp;journal=Genome%20Res&amp;volume=13&amp;pages=2498-504" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1239303" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1239303" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1239303"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14597658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maere</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Heymans</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kuiper</div> <div class="given-names">M.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">BiNGO: a Cytoscape plugin to assess overrepresentation of gene ontology categories in biological networks</div>. <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">3448</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BiNGO%3A%20a%20Cytoscape%20plugin%20to%20assess%20overrepresentation%20of%20gene%20ontology%20categories%20in%20biological%20networks&amp;author=S.%20Maere&amp;author=K.%20Heymans&amp;author=M.%20Kuiper&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3448-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti551" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti551" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti551"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15972284" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Benjamini</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Hochberg</div> <div class="given-names">Y.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Controlling the false discovery rate - a practical and powerful approach to multiple testing</div>. <div class="source">J. R. Stat. Soc. B. Methods</div> , <div class="volume">57</div>, <div class="fpage">289</div>–<div class="lpage">300</div>.</div></div></div></div><div content-id="dsw051-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Goff</div> <div class="given-names">L.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks</div>. <div class="source">Nat. Protoc</div> ., <div class="volume">7</div>, <div class="fpage">562</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20gene%20and%20transcript%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20TopHat%20and%20Cufflinks&amp;author=C.%20Trapnell&amp;author=A.%20Roberts&amp;author=L.%20Goff&amp;publication_year=2012&amp;journal=Nat.%20Protoc&amp;volume=7&amp;pages=562-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2012.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2012.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2012.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22383036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R.D.</div></div>, <div class="name string-name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Clements</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Pfam: the protein families database</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">D222</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pfam%3A%20the%20protein%20families%20database&amp;author=R.D.%20Finn&amp;author=A.%20Bateman&amp;author=J.%20Clements&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=D222-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1223" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1223"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24288371" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Johnson</div> <div class="given-names">L.S.</div></div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div>, <div class="name string-name"><div class="surname">Portugaly</div> <div class="given-names">E.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Hidden Markov model speed heuristic and iterative HMM search procedure</div>. <div class="source">BMC Bioinformatics</div> , <div class="volume">11</div>, <div class="fpage">431</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hidden%20Markov%20model%20speed%20heuristic%20and%20iterative%20HMM%20search%20procedure&amp;author=L.S.%20Johnson&amp;author=S.R.%20Eddy&amp;author=E.%20Portugaly&amp;publication_year=2010&amp;journal=BMC%20Bioinformatics&amp;volume=11&amp;pages=431" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-11-431" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-11-431" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-11-431"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20718988" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Edgar</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Domrachev</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Lash</div> <div class="given-names">A.E.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Gene Expression Omnibus: NCBI gene expression and hybridization array data repository</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">30</div>, <div class="fpage">207</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20Expression%20Omnibus%3A%20NCBI%20gene%20expression%20and%20hybridization%20array%20data%20repository&amp;author=R.%20Edgar&amp;author=M.%20Domrachev&amp;author=A.E.%20Lash&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res&amp;volume=30&amp;pages=207-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.1.207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.1.207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.1.207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11752295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Higashi</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Okazaki</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Myouga</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Shinozaki</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Saito</div> <div class="given-names">K.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Landscape of the lipidome and transcriptome under heat stress in Arabidopsis thaliana</div>. <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">10533</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Landscape%20of%20the%20lipidome%20and%20transcriptome%20under%20heat%20stress%20in%20Arabidopsis%20thaliana&amp;author=Y.%20Higashi&amp;author=Y.%20Okazaki&amp;author=F.%20Myouga&amp;author=K.%20Shinozaki&amp;author=K.%20Saito&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=10533" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep10533" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep10533" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep10533"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26013835" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Thatcher</div> <div class="given-names">S.R.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Leonard</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Genome-wide analysis of alternative splicing in Zea mays: landscape and genetic regulation</div>. <div class="source">Plant Cell</div> , <div class="volume">26</div>, <div class="fpage">3472</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20alternative%20splicing%20in%20Zea%20mays%3A%20landscape%20and%20genetic%20regulation&amp;author=S.R.%20Thatcher&amp;author=W.%20Zhou&amp;author=A.%20Leonard&amp;publication_year=2014&amp;journal=Plant%20Cell&amp;volume=26&amp;pages=3472-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.114.130773" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.114.130773" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.114.130773"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25248552" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">A.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">A comparison of the low temperature transcriptomes of two tomato genotypes that differ in freezing tolerance: Solanum lycopersicum and Solanum habrochaites</div>. <div class="source">BMC Plant Biol</div> ., <div class="volume">15</div>, <div class="fpage">132</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20comparison%20of%20the%20low%20temperature%20transcriptomes%20of%20two%20tomato%20genotypes%20that%20differ%20in%20freezing%20tolerance%3A%20Solanum%20lycopersicum%20and%20Solanum%20habrochaites&amp;author=H.%20Chen&amp;author=X.%20Chen&amp;author=D.%20Chen&amp;author=J.%20Li&amp;author=Y.%20Zhang&amp;author=A.%20Wang&amp;publication_year=2015&amp;journal=BMC%20Plant%20Biol&amp;volume=15&amp;pages=132" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12870-015-0521-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12870-015-0521-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12870-015-0521-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26048292" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Braunschweig</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Barbosa-Morais</div> <div class="given-names">N.L.</div></div>, <div class="name string-name"><div class="surname">Pan</div> <div class="given-names">Q.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Widespread intron retention in mammals functionally tunes transcriptomes</div>. <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">1774</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20intron%20retention%20in%20mammals%20functionally%20tunes%20transcriptomes&amp;author=U.%20Braunschweig&amp;author=N.L.%20Barbosa-Morais&amp;author=Q.%20Pan&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1774-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.177790.114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.177790.114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.177790.114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25258385" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bond</div> <div class="given-names">U.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Stressed out! Effects of environmental stress on mRNA metabolism</div>. <div class="source">FEMS Yeast Res</div> ., <div class="volume">6</div>, <div class="fpage">160</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Stressed%20out!%20Effects%20of%20environmental%20stress%20on%20mRNA%20metabolism&amp;author=U.%20Bond&amp;publication_year=2006&amp;journal=FEMS%20Yeast%20Res&amp;volume=6&amp;pages=160-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1567-1364.2006.00032.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1567-1364.2006.00032.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1567-1364.2006.00032.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16487339" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Utans</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Behrens</div> <div class="given-names">S.E.</div></div>, <div class="name string-name"><div class="surname">Luhrmann</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Kole</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Kramer</div> <div class="given-names">A.</div></div></div> <div class="year">1992</div>, 
<div class="article-title">A splicing factor that is inactivated during invivo heat-shock is functionally equivalent to the [U4/U6.U5] Triple Snrnp-Specific Proteins</div>. <div class="source">Genes Dev</div> ., <div class="volume">6</div>, <div class="fpage">631</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20splicing%20factor%20that%20is%20inactivated%20during%20invivo%20heat-shock%20is%20functionally%20equivalent%20to%20the%20%5BU4%2FU6.U5%5D%20Triple%20Snrnp-Specific%20Proteins&amp;author=U.%20Utans&amp;author=S.E.%20Behrens&amp;author=R.%20Luhrmann&amp;author=R.%20Kole&amp;author=A.%20Kramer&amp;publication_year=1992&amp;journal=Genes%20Dev&amp;volume=6&amp;pages=631-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.6.4.631" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.6.4.631" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.6.4.631"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1532785" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">Z.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">An alternatively spliced heat shock transcription factor, OsHSFA2dI, functions in the heat stress-induced unfolded protein response in rice</div>. <div class="source">Plant Biol</div> ., <div class="volume">17</div>, <div class="fpage">419</div>–<div class="lpage">29</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20alternatively%20spliced%20heat%20shock%20transcription%20factor%2C%20OsHSFA2dI%2C%20functions%20in%20the%20heat%20stress-induced%20unfolded%20protein%20response%20in%20rice&amp;author=Q.%20Cheng&amp;author=Y.%20Zhou&amp;author=Z.%20Liu&amp;publication_year=2015&amp;journal=Plant%20Biol&amp;volume=17&amp;pages=419-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/plb.12267" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fplb.12267" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fplb.12267"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25255693" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">An autoregulatory loop controlling Arabidopsis HsfA2 expression: role of heat shock-induced alternative splicing</div>. <div class="source">Plant Physiol</div> ., <div class="volume">162</div>, <div class="fpage">512</div>–<div class="lpage">521</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20autoregulatory%20loop%20controlling%20Arabidopsis%20HsfA2%20expression%3A%20role%20of%20heat%20shock-induced%20alternative%20splicing&amp;author=J.%20Liu&amp;author=N.%20Sun&amp;author=M.%20Liu&amp;publication_year=2013&amp;journal=Plant%20Physiol&amp;volume=162&amp;pages=512-521" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.112.205864" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.112.205864" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.112.205864"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23503691" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sugio</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Dreos</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Aparicio</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Maule</div> <div class="given-names">A.J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">The Cytosolic protein response as a subcomponent of the wider heat shock response in Arabidopsis</div>. <div class="source">Plant Cell</div> , <div class="volume">21</div>, <div class="fpage">642</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Cytosolic%20protein%20response%20as%20a%20subcomponent%20of%20the%20wider%20heat%20shock%20response%20in%20Arabidopsis&amp;author=A.%20Sugio&amp;author=R.%20Dreos&amp;author=F.%20Aparicio&amp;author=A.J.%20Maule&amp;publication_year=2009&amp;journal=Plant%20Cell&amp;volume=21&amp;pages=642-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.108.062596" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.108.062596" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.108.062596"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19244141" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tsukahara</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Yoshioka</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Muraki</div> <div class="given-names">T.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Molecular and functional characterization of HSC54, a novel variant of human heat-shock cognate protein 70</div>. <div class="source">Mol. Pharmacol</div> ., <div class="volume">58</div>, <div class="fpage">1257</div>–<div class="lpage">263</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20and%20functional%20characterization%20of%20HSC54%2C%20a%20novel%20variant%20of%20human%20heat-shock%20cognate%20protein%2070&amp;author=F.%20Tsukahara&amp;author=T.%20Yoshioka&amp;author=T.%20Muraki&amp;publication_year=2000&amp;journal=Mol.%20Pharmacol&amp;volume=58&amp;pages=1257-263" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11093761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamada</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yamada</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kiuchi</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">1999</div>, 
<div class="article-title">Identification of a novel splice variant of heat shock cognate protein 70 after chronic antidepressant treatment in rat frontal cortex</div>. <div class="source">Biochem. Biophys. Res. Commun</div> ., <div class="volume">261</div>, <div class="fpage">541</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20novel%20splice%20variant%20of%20heat%20shock%20cognate%20protein%2070%20after%20chronic%20antidepressant%20treatment%20in%20rat%20frontal%20cortex&amp;author=M.%20Yamada&amp;author=M.%20Yamada&amp;author=Y.%20Kiuchi&amp;publication_year=1999&amp;journal=Biochem.%20Biophys.%20Res.%20Commun&amp;volume=261&amp;pages=541-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/bbrc.1999.1067" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fbbrc.1999.1067" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fbbrc.1999.1067"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10425221" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Takechi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Hosokawa</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Hirayoshi</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">K.</div></div></div> <div class="year">1994</div>, 
<div class="article-title">Alternative 5’ splice site selection induced by heat shock</div>. <div class="source">Mol. Cell. Biol</div> ., <div class="volume">14</div>, <div class="fpage">567</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%205%E2%80%99%20splice%20site%20selection%20induced%20by%20heat%20shock&amp;author=H.%20Takechi&amp;author=N.%20Hosokawa&amp;author=K.%20Hirayoshi&amp;author=K.%20Nagata&amp;publication_year=1994&amp;journal=Mol.%20Cell.%20Biol&amp;volume=14&amp;pages=567-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.14.1.567" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.14.1.567" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.14.1.567"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8264624" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dvinge</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Bradley</div> <div class="given-names">R.K.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Widespread intron retention diversifies most cancer transcriptomes</div>. <div class="source">Genome Med</div> ., <div class="volume">7</div>, <div class="fpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Widespread%20intron%20retention%20diversifies%20most%20cancer%20transcriptomes&amp;author=H.%20Dvinge&amp;author=R.K.%20Bradley&amp;publication_year=2015&amp;journal=Genome%20Med&amp;volume=7&amp;pages=45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13073-015-0168-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13073-015-0168-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13073-015-0168-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26113877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pickrell</div> <div class="given-names">J.K.</div></div>, <div class="name string-name"><div class="surname">Pai</div> <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Gilad</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Pritchard</div> <div class="given-names">J.K.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Noisy splicing drives mRNA isoform diversity in human cells</div>. <div class="source">PLoS Genet</div> ., <div class="volume">6</div>, <div class="fpage">e1001236</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Noisy%20splicing%20drives%20mRNA%20isoform%20diversity%20in%20human%20cells&amp;author=J.K.%20Pickrell&amp;author=A.A.%20Pai&amp;author=Y.%20Gilad&amp;author=J.K.%20Pritchard&amp;publication_year=2010&amp;journal=PLoS%20Genet&amp;volume=6&amp;pages=e1001236" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1001236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1001236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1001236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21151575" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Warnecke</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Hurst</div> <div class="given-names">L.D.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Error prevention and mitigation as forces in the evolution of genes and genomes</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">12</div>, <div class="fpage">875</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Error%20prevention%20and%20mitigation%20as%20forces%20in%20the%20evolution%20of%20genes%20and%20genomes&amp;author=T.%20Warnecke&amp;author=L.D.%20Hurst&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet&amp;volume=12&amp;pages=875-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22094950" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Šamaj</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Thelen</div> <div class="given-names">J.J.</div></div></div> <div class="year">2007</div>, <div class="source">Plant Proteomics</div> . 
<div class="publisher-name">Springer</div>: 
<div class="publisher-loc">Berlin; New York</div>.</div></div></div></div><div content-id="dsw051-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Volkov</div> <div class="given-names">R.A.</div></div>, <div class="name string-name"><div class="surname">Panchuk</div> <div class="given-names">II</div></div>, <div class="name string-name"><div class="surname">Schoffl</div> <div class="given-names">F.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Heat-stress-dependency and developmental modulation of gene expression: the potential of house-keeping genes as internal standards in mRNA expression profiling using real-time RT-PCR</div>. <div class="source">J. Exp. Bot</div> ., <div class="volume">54</div>, <div class="fpage">2343</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heat-stress-dependency%20and%20developmental%20modulation%20of%20gene%20expression%3A%20the%20potential%20of%20house-keeping%20genes%20as%20internal%20standards%20in%20mRNA%20expression%20profiling%20using%20real-time%20RT-PCR&amp;author=R.A.%20Volkov&amp;author=II%20Panchuk&amp;author=F.%20Schoffl&amp;publication_year=2003&amp;journal=J.%20Exp.%20Bot&amp;volume=54&amp;pages=2343-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erg244" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferg244" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferg244"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14504302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Staudt</div> <div class="given-names">A.C.</div></div>, <div class="name string-name"><div class="surname">Wenkel</div> <div class="given-names">S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Regulation of protein function by ‘microProteins’</div>. <div class="source">EMBO Rep</div> ., <div class="volume">12</div>, <div class="fpage">35</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulation%20of%20protein%20function%20by%20%E2%80%98microProteins%E2%80%99&amp;author=A.C.%20Staudt&amp;author=S.%20Wenkel&amp;publication_year=2011&amp;journal=EMBO%20Rep&amp;volume=12&amp;pages=35-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/embor.2010.196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fembor.2010.196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fembor.2010.196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21151039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Seo</div> <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Hong</div> <div class="given-names">S.Y.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S.G.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">C.M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Competitive inhibition of transcription factors by small interfering peptides</div>. <div class="source">Trends Plant Sci</div> ., <div class="volume">16</div>, <div class="fpage">541</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Competitive%20inhibition%20of%20transcription%20factors%20by%20small%20interfering%20peptides&amp;author=P.J.%20Seo&amp;author=S.Y.%20Hong&amp;author=S.G.%20Kim&amp;author=C.M.%20Park&amp;publication_year=2011&amp;journal=Trends%20Plant%20Sci&amp;volume=16&amp;pages=541-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tplants.2011.06.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tplants.2011.06.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tplants.2011.06.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21723179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Prakash</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Domain atrophy creates rare cases of functional partial protein domains</div>. <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Domain%20atrophy%20creates%20rare%20cases%20of%20functional%20partial%20protein%20domains&amp;author=A.%20Prakash&amp;author=A.%20Bateman&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0655-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0655-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0655-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25924720" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Swindell</div> <div class="given-names">W.R.</div></div>, <div class="name string-name"><div class="surname">Huebner</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Weber</div> <div class="given-names">A.P.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Transcriptional profiling of Arabidopsis heat shock proteins and transcription factors reveals extensive overlap between heat and non-heat stress response pathways</div>. <div class="source">BMC Genomics</div> , <div class="volume">8</div>, <div class="fpage">125</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20profiling%20of%20Arabidopsis%20heat%20shock%20proteins%20and%20transcription%20factors%20reveals%20extensive%20overlap%20between%20heat%20and%20non-heat%20stress%20response%20pathways&amp;author=W.R.%20Swindell&amp;author=M.%20Huebner&amp;author=A.P.%20Weber&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=125" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-125" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-125" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-125"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17519032" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bita</div> <div class="given-names">C.E.</div></div>, <div class="name string-name"><div class="surname">Zenoni</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Vriezen</div> <div class="given-names">W.H.</div></div>, <div class="name string-name"><div class="surname">Mariani</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pezzotti</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Gerats</div> <div class="given-names">T.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Temperature stress differentially modulates transcription in meiotic anthers of heat-tolerant and heat-sensitive tomato plants</div>. <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">384</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Temperature%20stress%20differentially%20modulates%20transcription%20in%20meiotic%20anthers%20of%20heat-tolerant%20and%20heat-sensitive%20tomato%20plants&amp;author=C.E.%20Bita&amp;author=S.%20Zenoni&amp;author=W.H.%20Vriezen&amp;author=C.%20Mariani&amp;author=M.%20Pezzotti&amp;author=T.%20Gerats&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=384" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-384" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-384" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-384"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21801454" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sarikas</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hartmann</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Pan</div> <div class="given-names">Z.Q.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">The cullin protein family</div>. <div class="source">Genome Biol</div> ., <div class="volume">12</div>, <div class="fpage">220</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20cullin%20protein%20family&amp;author=A.%20Sarikas&amp;author=T.%20Hartmann&amp;author=Z.Q.%20Pan&amp;publication_year=2011&amp;journal=Genome%20Biol&amp;volume=12&amp;pages=220" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2011-12-4-220" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2011-12-4-220" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2011-12-4-220"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21554755" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pan</div> <div class="given-names">Z.Q.</div></div>, <div class="name string-name"><div class="surname">Kentsis</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Dias</div> <div class="given-names">D.C.</div></div>, <div class="name string-name"><div class="surname">Yamoah</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">K.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Nedd8 on cullin: building an expressway to protein destruction</div>. <div class="source">Oncogene</div> , <div class="volume">23</div>, <div class="fpage">1985</div>–<div class="lpage">97</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nedd8%20on%20cullin%3A%20building%20an%20expressway%20to%20protein%20destruction&amp;author=Z.Q.%20Pan&amp;author=A.%20Kentsis&amp;author=D.C.%20Dias&amp;author=K.%20Yamoah&amp;author=K.%20Wu&amp;publication_year=2004&amp;journal=Oncogene&amp;volume=23&amp;pages=1985-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.onc.1207414" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.onc.1207414" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.onc.1207414"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15021886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guo</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Nezames</div> <div class="given-names">C.D.</div></div>, <div class="name string-name"><div class="surname">Sheng</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Deng</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wei</div> <div class="given-names">N.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Cullin-RING ubiquitin ligase family in plant abiotic stress pathways(F)</div>. <div class="source">J. Integr. Plant Biol</div> ., <div class="volume">55</div>, <div class="fpage">21</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cullin-RING%20ubiquitin%20ligase%20family%20in%20plant%20abiotic%20stress%20pathways(F)&amp;author=L.%20Guo&amp;author=C.D.%20Nezames&amp;author=L.%20Sheng&amp;author=X.%20Deng&amp;author=N.%20Wei&amp;publication_year=2013&amp;journal=J.%20Integr.%20Plant%20Biol&amp;volume=55&amp;pages=21-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jipb.2013.55.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjipb.2013.55.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjipb.2013.55.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23206256" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Voigt</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Papalopulu</div> <div class="given-names">N.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">A dominant-negative form of the E3 ubiquitin ligase Cullin-1 disrupts the correct allocation of cell fate in the neural crest lineage</div>. <div class="source">Development</div> , <div class="volume">133</div>, <div class="fpage">559</div>–<div class="lpage">68</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20dominant-negative%20form%20of%20the%20E3%20ubiquitin%20ligase%20Cullin-1%20disrupts%20the%20correct%20allocation%20of%20cell%20fate%20in%20the%20neural%20crest%20lineage&amp;author=J.%20Voigt&amp;author=N.%20Papalopulu&amp;publication_year=2006&amp;journal=Development&amp;volume=133&amp;pages=559-68" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/dev.02201" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fdev.02201" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fdev.02201"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16396913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B76" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B76" class="ref-content "><div class="label -label">76</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dai</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Terzaghi</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">H.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Multifaceted roles of Arabidopsis PP6 phosphatase in regulating cellular signaling and plant development</div>. <div class="source">Plant Signal. Behav</div> ., <div class="volume">8</div>, <div class="fpage">e22508</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multifaceted%20roles%20of%20Arabidopsis%20PP6%20phosphatase%20in%20regulating%20cellular%20signaling%20and%20plant%20development&amp;author=M.%20Dai&amp;author=W.%20Terzaghi&amp;author=H.%20Wang&amp;publication_year=2013&amp;journal=Plant%20Signal.%20Behav&amp;volume=8&amp;pages=e22508" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/psb.22508" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fpsb.22508" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fpsb.22508"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23104112" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B77" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B77" class="ref-content "><div class="label -label">77</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Park</div> <div class="given-names">J.H.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">W.Y.</div></div>, <div class="name string-name"><div class="surname">Chae</div> <div class="given-names">H.B.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">M.G.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">S.Y.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Serine/threonine protein phosphatase 5 (PP5) interacts with substrate under heat stress conditions and forms protein complex in Arabidopsis</div>. <div class="source">Plant Signal. Behav</div> ., <div class="volume">7</div>, <div class="fpage">535</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Serine%2Fthreonine%20protein%20phosphatase%205%20(PP5)%20interacts%20with%20substrate%20under%20heat%20stress%20conditions%20and%20forms%20protein%20complex%20in%20Arabidopsis&amp;author=J.H.%20Park&amp;author=W.Y.%20Kim&amp;author=H.B.%20Chae&amp;author=M.G.%20Kim&amp;author=S.Y.%20Lee&amp;publication_year=2012&amp;journal=Plant%20Signal.%20Behav&amp;volume=7&amp;pages=535-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/psb.19699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fpsb.19699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fpsb.19699"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22516824" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw051-B78" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw051-B78" class="ref-content "><div class="label -label">78</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guergnon</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Derewenda</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Edelson</div> <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">Brautigan</div> <div class="given-names">D.L.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Mapping of protein phosphatase-6 association with its SAPS domain regulatory subunit using a model of helical repeats</div>. <div class="source">BMC Biochem</div> ., <div class="volume">10</div>, <div class="fpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20of%20protein%20phosphatase-6%20association%20with%20its%20SAPS%20domain%20regulatory%20subunit%20using%20a%20model%20of%20helical%20repeats&amp;author=J.%20Guergnon&amp;author=U.%20Derewenda&amp;author=J.R.%20Edelson&amp;author=D.L.%20Brautigan&amp;publication_year=2009&amp;journal=BMC%20Biochem&amp;volume=10&amp;pages=24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2091-10-24" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2091-10-24" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2091-10-24"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19835610" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p"> This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2738679&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw051/2/dsw051_Supp.zip?Expires=2147483647&amp;Signature=lYNXuE06JFrdb6v1~lM7t2xKpl579FU4QPDVTY3zA7PTlHJLitoZu0a-Blykwzh0ogUddQR2Z0yRedubJ5FclTJjmkOVSy-rzshABrgC1QWtGQnQGCUSBjWgD~dwHu4j3LWh~M~859KdA2A8-dejHHYjBhC~brRXTSYDVxD0-XqBIdp4nd9HJrk~s8yDDjUFEbm-zgRar116qRbbbF-mWV5NcM3TKci96o52UTD8pSBZMrRc38kuhGIyF33rSsiUpXd4AhllLGaEmLXgRpV7GH6TvoukfAbomSFL-B-AA7OwunN7jxcL4khjkJvM4kW80cuG3S1BUzluxgJiBjooWQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,264</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000400913000009&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">5</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw051"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2738679"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000400913000009&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f2%2f205%2f2738679">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000400913000009&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (5)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F2%2F205%2F2738679" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2738679'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Alternative splicing in tomato pollen in response to heat stress†" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw051");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f205%2f2738679">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

