

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/6/609/3904549","@type":"ScholarlyArticle","name":"A convenient method to pre-screen candidate guide RNAs for CRISPR/Cas9 gene editing by NHEJ-mediated integration of a ‘self-cleaving’ GFP-expression plasmid","datePublished":"2017-06-30","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2017-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx029","keywords":["Cas9","CRISPR","gRNA testing","reporter assay","NHEJ-cloning"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/6/609/3904549","author":[{"name":"Tálas, András","affiliation":"School of Ph.D. Studies, Semmelweis University, Budapest, Hungary  Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary","@type":"Person"},{"name":"Kulcsár, Péter István","affiliation":"Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary  Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary  University of Szeged, Szeged, Hungary","@type":"Person"},{"name":"Weinhardt, Nóra","affiliation":"Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary  Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary  University of Szeged, Szeged, Hungary","@type":"Person"},{"name":"Borsy, Adrienn","affiliation":"Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary","@type":"Person"},{"name":"Tóth, Eszter","affiliation":"Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary  Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary","@type":"Person"},{"name":"Szebényi, Kornélia","affiliation":"Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary","@type":"Person"},{"name":"Krausz, Sarah Laura","affiliation":"School of Ph.D. Studies, Semmelweis University, Budapest, Hungary  Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary","@type":"Person"},{"name":"Huszár, Krisztina","affiliation":"Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary  Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary","@type":"Person"},{"name":"Vida, István","affiliation":"Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary  Institute of Organic Chemistry, Eötvös Loránd University, Budapest, Hungary","@type":"Person"},{"name":"Sturm, Ádám","affiliation":"Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary","@type":"Person"},{"name":"Gordos, Bianka","affiliation":"Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary","@type":"Person"},{"name":"Hoffmann, Orsolya Ivett","affiliation":"Animal Biotechnology Section, Ruminant Genome Biology Group, NARIC Agricultural Biotechnology Institute, Gödöllő, Hungary","@type":"Person"},{"name":"Bencsura, Petra","affiliation":"Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary  Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary","@type":"Person"},{"name":"Nyeste, Antal","affiliation":"Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary  Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary","@type":"Person"},{"name":"Ligeti, Zoltán","affiliation":"Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary","@type":"Person"},{"name":"Fodor, Elfrieda","affiliation":"Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary","@type":"Person"},{"name":"Welker, Ervin","affiliation":"Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary  Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary","@type":"Person"}],"description":"Abstract.  The efficacies of guide RNAs (gRNAs), the short RNA molecules that bind to and determine the sequence specificity of the Streptococcus pyogenes Cas9 ","pageStart":"609","pageEnd":"621","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A convenient method to pre-screen candidate guide RNAs for CRISPR/Cas9 gene editing by NHEJ-mediated integration of a ‘self-cleaving’ GFP-expression plasmid","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="convenient method to pre-screen candidate guide RNAs for CRISPR/Cas9 gene editing by NHEJ-mediated integration of a ‘self-cleaving’ GFP-expression plasmid" />
    <meta property="og:description" content="Abstract.  The efficacies of guide RNAs (gRNAs), the short RNA molecules that bind to and determine the sequence specificity of the Streptococcus pyogenes Cas9 " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/6/609/3904549" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-06-30" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Tálas, András" /><meta name="citation_author_institution" content="School of Ph.D. Studies, Semmelweis University, Budapest, Hungary" /><meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary" /><meta name="citation_author" content="Kulcsár, Péter István" /><meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary" /><meta name="citation_author_institution" content="Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary" /><meta name="citation_author_institution" content="University of Szeged, Szeged, Hungary" /><meta name="citation_author" content="Weinhardt, Nóra" /><meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary" /><meta name="citation_author_institution" content="Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary" /><meta name="citation_author_institution" content="University of Szeged, Szeged, Hungary" /><meta name="citation_author" content="Borsy, Adrienn" /><meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary" /><meta name="citation_author" content="Tóth, Eszter" /><meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary" /><meta name="citation_author_institution" content="Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary" /><meta name="citation_author" content="Szebényi, Kornélia" /><meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary" /><meta name="citation_author" content="Krausz, Sarah Laura" /><meta name="citation_author_institution" content="School of Ph.D. Studies, Semmelweis University, Budapest, Hungary" /><meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary" /><meta name="citation_author" content="Huszár, Krisztina" /><meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary" /><meta name="citation_author_institution" content="Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary" /><meta name="citation_author" content="Vida, István" /><meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary" /><meta name="citation_author_institution" content="Institute of Organic Chemistry, Eötvös Loránd University, Budapest, Hungary" /><meta name="citation_author" content="Sturm, Ádám" /><meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary" /><meta name="citation_author" content="Gordos, Bianka" /><meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary" /><meta name="citation_author" content="Hoffmann, Orsolya Ivett" /><meta name="citation_author_institution" content="Animal Biotechnology Section, Ruminant Genome Biology Group, NARIC Agricultural Biotechnology Institute, Gödöllő, Hungary" /><meta name="citation_author" content="Bencsura, Petra" /><meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary" /><meta name="citation_author_institution" content="Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary" /><meta name="citation_author" content="Nyeste, Antal" /><meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary" /><meta name="citation_author_institution" content="Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary" /><meta name="citation_author" content="Ligeti, Zoltán" /><meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary" /><meta name="citation_author" content="Fodor, Elfrieda" /><meta name="citation_author_institution" content="Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary" /><meta name="citation_author" content="Welker, Ervin" /><meta name="citation_author_institution" content="Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary" /><meta name="citation_author_institution" content="Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary" /><meta name="citation_title" content="A convenient method to pre-screen candidate guide RNAs for CRISPR/Cas9 gene editing by NHEJ-mediated integration of a ‘self-cleaving’ GFP-expression plasmid" /><meta name="citation_firstpage" content="609" /><meta name="citation_lastpage" content="621" /><meta name="citation_doi" content="10.1093/dnares/dsx029" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2017/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity;  citation_journal_title=Science;  citation_year=2012;  citation_volume=337;  citation_pages=816-21; " /><meta name="citation_reference" content="citation_title=RNA-guided human genome engineering via Cas9;  citation_journal_title=Science;  citation_year=2013;  citation_volume=339;  citation_pages=823-26; " /><meta name="citation_reference" content="citation_title=Multiplex genome engineering using CRISPR/Cas systems;  citation_journal_title=Science;  citation_year=2013;  citation_volume=339;  citation_pages=819-23; " /><meta name="citation_reference" content="citation_title=RNA-programmed genome editing in human cells;  citation_journal_title=Elife;  citation_year=2013;  citation_volume=2;  citation_pages=e00471" /><meta name="citation_reference" content="citation_title=CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes;  citation_journal_title=Cell;  citation_year=2013;  citation_volume=154;  citation_pages=442-51; " /><meta name="citation_reference" content="citation_title=Genome-scale CRISPR-Cas9 knockout screening in human cells;  citation_journal_title=Science;  citation_year=2014;  citation_volume=343;  citation_pages=84-7; " /><meta name="citation_reference" content="citation_title=Engineering complex synthetic transcriptional programs with CRISPR RNA scaffolds;  citation_journal_title=Cell;  citation_year=2015;  citation_volume=160;  citation_pages=339-50; " /><meta name="citation_reference" content="citation_title=Genome-scale CRISPR-mediated control of gene repression and activation;  citation_journal_title=Cell;  citation_year=2014;  citation_volume=159;  citation_pages=647-61; " /><meta name="citation_reference" content="citation_title=Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression;  citation_journal_title=Cell;  citation_year=2013;  citation_volume=152;  citation_pages=1173-83; " /><meta name="citation_reference" content="citation_title=Gene disruption by cell-penetrating peptide-mediated delivery of Cas9 protein and guide RNA;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1020-7; " /><meta name="citation_reference" content="citation_title=CAS9 transcriptional activators for target specificity screening and paired nickases for cooperative genome engineering;  citation_journal_title=Nat. Biotechnol;  citation_year=2013;  citation_volume=31;  citation_pages=833-8; " /><meta name="citation_reference" content="citation_title=Double nicking by RNA-guided CRISPR Cas9 for enhanced genome editing specificity;  citation_journal_title=Cell;  citation_year=2013;  citation_volume=154;  citation_pages=1380-9; " /><meta name="citation_reference" content="citation_title=Modeling Disease In Vivo With CRISPR/Cas9;  citation_journal_title=Trends Mol. Med;  citation_year=2015;  citation_volume=21;  citation_pages=609-21; " /><meta name="citation_reference" content="citation_title=A guide to genome engineering with programmable nucleases;  citation_journal_title=Nat. Rev. Genet;  citation_year=2014;  citation_volume=15;  citation_pages=321-34; " /><meta name="citation_reference" content="citation_title=Surrogate reporter-based enrichment of cells containing RNA-guided Cas9 nuclease-induced mutations;  citation_journal_title=Nat. Commun;  citation_year=2014;  citation_volume=5;  citation_pages=3378" /><meta name="citation_reference" content="citation_title=Rational design of highly active sgRNAs for CRISPR-Cas9-mediated gene inactivation;  citation_journal_title=Nat. Biotechnol;  citation_year=2014;  citation_volume=32;  citation_pages=1262-7; " /><meta name="citation_reference" content="citation_title=Genetic screens in human cells using the CRISPR-Cas9 system;  citation_journal_title=Science;  citation_year=2014;  citation_volume=343;  citation_pages=80-4; " /><meta name="citation_reference" content="citation_title=Unraveling CRISPR-Cas9 genome engineering parameters via a library-on-library approach;  citation_journal_title=Nat. Methods;  citation_year=2015;  citation_volume=12;  citation_pages=823-6; " /><meta name="citation_reference" content="citation_title=CRISPRscan: designing highly efficient sgRNAs for CRISPR-Cas9 targeting in vivo;  citation_journal_title=Nat. Methods;  citation_year=2015;  citation_volume=12;  citation_pages=982-8; " /><meta name="citation_reference" content="citation_title=WU-CRISPR: characteristics of functional guide RNAs for the CRISPR/Cas9 system;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=1-8; " /><meta name="citation_reference" content="citation_title=Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9;  citation_journal_title=Nat. Biotechnol;  citation_year=2016;  citation_volume=34;  citation_pages=184-91; " /><meta name="citation_reference" content="citation_title=Nucleosomes inhibit Cas9 endonuclease activity in vitro;  citation_journal_title=Biochemistry;  citation_year=2015;  citation_volume=54;  citation_pages=7063-6; " /><meta name="citation_reference" content="citation_title=Nucleosomes impede Cas9 access to DNA in vivo and in vitro;  citation_journal_title=Elife;  citation_year=2016;  citation_volume=5;  citation_pages=e12677" /><meta name="citation_reference" content="citation_title=Quantifying on-and off-target genome editing;  citation_journal_title=Trends Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=132-40; " /><meta name="citation_reference" content="citation_title=Targeted germline modifications in rats using CRISPR/Cas9 and spermatogonial stem cells;  citation_journal_title=Cell Rep;  citation_year=2015;  citation_volume=10;  citation_pages=1828-35; " /><meta name="citation_reference" content="citation_title=Inducible in vivo genome editing with CRISPR-Cas9;  citation_journal_title=Nature Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=390-4; " /><meta name="citation_reference" content="citation_title=Genome engineering using the CRISPR-Cas9 system;  citation_journal_title=Nat. Protoc;  citation_year=2013;  citation_volume=8;  citation_pages=2281-308; " /><meta name="citation_reference" content="citation_title=Targeted genome engineering in human cells with the Cas9 RNA-guided endonuclease;  citation_journal_title=Nat. Biotechnol;  citation_year=2013;  citation_volume=31;  citation_pages=230-2; " /><meta name="citation_reference" content="citation_title=Mutation detection using Surveyor™ nuclease;  citation_journal_title=Biotechniques;  citation_year=2004;  citation_volume=36;  citation_pages=702-7; " /><meta name="citation_reference" content="citation_title=Simple methods for generating and detecting locus-specific mutations induced with TALENs in the zebrafish genome;  citation_journal_title=PLoS Genet;  citation_year=2012;  citation_volume=8;  citation_pages=e1002861" /><meta name="citation_reference" content="citation_title=Unbiased detection of off-target cleavage by CRISPR-Cas9 and TALENs using integrase-defective lentiviral vectors;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=175-8; " /><meta name="citation_reference" content="citation_title=High-throughput genotyping of CRISPR/Cas9-mediated mutants using fluorescent PCR-capillary gel electrophoresis;  citation_journal_title=Sci. Rep;  citation_year=2015;  citation_volume=5;  citation_pages=15587" /><meta name="citation_reference" content="citation_title=Easy quantitative assessment of genome editing by sequence trace decomposition;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=e168" /><meta name="citation_reference" content="citation_title=Optimization of scarless human stem cell genome editing;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=9049-61; " /><meta name="citation_reference" content="citation_title=Highly efficient endogenous human gene correction using designed zinc-finger nucleases;  citation_journal_title=Nature;  citation_year=2005;  citation_volume=435;  citation_pages=646-51; " /><meta name="citation_reference" content="citation_title=Efficient CRISPR-Cas9-mediated genome editing in Plasmodium falciparum;  citation_journal_title=Nat. Methods;  citation_year=2014;  citation_volume=11;  citation_pages=915-8; " /><meta name="citation_reference" content="citation_title=One-step generation of mice carrying mutations in multiple genes by CRISPR/Cas-mediated genome engineering;  citation_journal_title=Cell;  citation_year=2013;  citation_volume=153;  citation_pages=910-8; " /><meta name="citation_reference" content="citation_title=Highly efficient CRISPR/Cas9-mediated knock-in in zebrafish by homology-independent DNA repair;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=142-53; " /><meta name="citation_reference" content="citation_title=Tracking genome engineering outcome at individual DNA breakpoints;  citation_journal_title=Nat. Methods;  citation_year=2011;  citation_volume=8;  citation_pages=671-6; " /><meta name="citation_reference" content="citation_title=Generation of mutant mice by pronuclear injection of circular plasmid expressing Cas9 and single guided RNA;  citation_journal_title=Sci. Rep;  citation_year=2013;  citation_volume=3;  citation_pages=3355" /><meta name="citation_reference" content="citation_title=Efficient targeting of expressed and silent genes in human ESCs and iPSCs using zinc-finger nucleases;  citation_journal_title=Nat. Biotechnol;  citation_year=2009;  citation_volume=27;  citation_pages=851-7; " /><meta name="citation_reference" content="citation_title=Surrogate reporters for enrichment of cells with nuclease-induced mutations;  citation_journal_title=Nat. Methods;  citation_year=2011;  citation_volume=8;  citation_pages=941-3; " /><meta name="citation_reference" content="citation_title=Zinc-finger nuclease-driven targeted integration into mammalian genomes using donors with limited chromosomal homology;  citation_journal_title=Nucleic Acids Res;  citation_year=2010;  citation_volume=38;  citation_pages=e152" /><meta name="citation_reference" content="citation_title=In vivo cleavage of transgene donors promotes nuclease‐mediated targeted integration;  citation_journal_title=Biotechnol. Bioeng;  citation_year=2013;  citation_volume=110;  citation_pages=871-80; " /><meta name="citation_reference" content="citation_title=CRISPR‐Cas targeted plasmid integration into mammalian cells via non‐homologous end joining;  citation_journal_title=Biotechnol. Bioeng;  citation_year=2015;  citation_volume=112;  citation_pages=2154-62; " /><meta name="citation_reference" content="citation_title=A generic strategy for CRISPR-Cas9-mediated gene tagging;  citation_journal_title=Nat. Commun;  citation_year=2015;  citation_volume=6;  citation_pages=10237" /><meta name="citation_reference" content="citation_title=Multicolor CRISPR labeling of chromosomal loci in human cells;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2015;  citation_volume=112;  citation_pages=3002-7; " /><meta name="citation_reference" content="citation_title=Restriction enzyme body doubles and PCR cloning: on the general use of type IIs restriction enzymes for cloning;  citation_journal_title=PloS One;  citation_year=2014;  citation_volume=9;  citation_pages=e90896" /><meta name="citation_reference" content="citation_title=Lentivirus-delivered stable gene silencing by RNAi in primary cells;  citation_journal_title=RNA;  citation_year=2003;  citation_volume=9;  citation_pages=493-501; " /><meta name="citation_reference" content="citation_title=A general chemical method to regulate protein stability in the mammalian central nervous system;  citation_journal_title=Chem. Biol;  citation_year=2010;  citation_volume=17;  citation_pages=981-8; " /><meta name="citation_reference" content="citation_title=Gene targeting to the ROSA26 locus directed by engineered zinc finger nucleases;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=3741-52; " /><meta name="citation_reference" content="citation_title=Obligate ligation-gated recombination (ObLiGaRe): custom-designed nuclease-mediated targeted integration through nonhomologous end joining;  citation_journal_title=Genome Res;  citation_year=2013;  citation_volume=23;  citation_pages=539-46; " /><meta name="citation_reference" content="citation_title=E-CRISP: fast CRISPR target site identification;  citation_journal_title=Nat. Methods;  citation_year=2014;  citation_volume=11;  citation_pages=122-3; " /><meta name="citation_reference" content="Aach J., Mali P., Church G. M.  2014 , CasFinder: Flexible algorithm for identifying specific Cas9 targets in genomes. bioRxiv , 005074. " /><meta name="citation_reference" content="citation_title=CRISPR-mediated direct mutation of cancer genes in the mouse liver;  citation_journal_title=Nature;  citation_year=2014;  citation_volume=514;  citation_pages=380-4; " /><meta name="citation_reference" content="citation_title=Efficient generation of large-scale genome-modified mice using gRNA and CAS9 endonuclease;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=e187" /><meta name="citation_reference" content="citation_title=Nuclear domain ‘knock-in’screen for the evaluation and identification of small molecule enhancers of CRISPR-based genome editing;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=9379-92; " /><meta name="citation_reference" content="Li K., Wang G., Andersen T., Zhou P., Pu W. T.  2014 , Optimization of genome engineering approaches with the CRISPR/Cas9 system. Plos One., 9 , e105779. " /><meta name="citation_reference" content="Geisinger J. M., Turan S., Hernandez S., Spector L. P., Calos M. P.  2016 , In vivo blunt-end cloning through CRISPR/Cas9-facilitated non-homologous end-joining. Nucleic Acids Res. , 44 , e76 – e76. " /><meta name="citation_reference" content="citation_title=Rationally engineered Cas9 nucleases with improved specificity;  citation_journal_title=Science;  citation_year=2016;  citation_volume=351;  citation_pages=84-8; " /><meta name="citation_reference" content="citation_title=High-fidelity CRISPR–Cas9 nucleases with no detectable genome-wide off-target effects;  citation_journal_title=Nature;  citation_year=2016;  citation_volume=529;  citation_pages=490-5; " /><meta name="citation_reference" content="citation_title=Induction and repair of DNA double strand breaks: the increasing spectrum of non-homologous end joining pathways;  citation_journal_title=Mutat. Res;  citation_year=2011;  citation_volume=711;  citation_pages=61-72; " /><meta name="citation_reference" content="citation_title=Microhomology-based choice of Cas9 nuclease target sites;  citation_journal_title=Nat. Methods;  citation_year=2014;  citation_volume=11;  citation_pages=705-6; " /><meta name="citation_reference" content="citation_title=MMEJ repair of double-strand breaks (director’s cut): deleted sequences and alternative endings;  citation_journal_title=Trends Genet;  citation_year=2008;  citation_volume=24;  citation_pages=529-38; " /><meta name="citation_reference" content="citation_title=Comparison of T7E1 and surveyor mismatch cleavage assays to detect mutations triggered by engineered nucleases;  citation_journal_title=G3;  citation_year=2015;  citation_volume=5;  citation_pages=407-15; " /><meta name="citation_reference" content="citation_title=Microchip electrophoresis coupled with on‐line magnetic separation and chemiluminescence detection for multiplexed immunoassay;  citation_journal_title=Electrophoresis;  citation_year=2012;  citation_volume=33;  citation_pages=1198-204; " /><meta name="citation_reference" content="citation_title=Front-line of genome editing technology for animal cell engineering;  citation_journal_title=BMC Proceed.,;  citation_year=2015;  citation_pages=O1" /><meta name="citation_reference" content="citation_title=Development of novel visual-plus quantitative analysis systems for studying DNA double-strand break repairs in zebrafish;  citation_journal_title=J. Genet. Genomics;  citation_year=2012;  citation_volume=39;  citation_pages=489-502; " /><meta name="citation_reference" content="citation_title=, Non-homologous end joining plays a key role in transgene concatemer formation in transgenic zebrafish embryos;  citation_journal_title=Int. J. Biol. Sci;  citation_year=2010;  citation_volume=6;  citation_pages=756" /><meta name="citation_reference" content="citation_title=Homologous recombination and DNA-end joining reactions in zygotes and early embryos of zebrafish (Danio rerio) and Drosophila melanogaster;  citation_journal_title=Biol. Chem;  citation_year=1998;  citation_volume=379;  citation_pages=673-82; " /><meta name="citation_reference" content="citation_title=In vivo genome editing via CRISPR/Cas9 mediated homology-independent targeted integration;  citation_journal_title=Nature;  citation_year=2016;  citation_volume=540;  citation_pages=144-9; " /><meta name="citation_reference" content="citation_title=Microhomology-mediated end-joining-dependent integration of donor DNA in cells and animals using TALENs and CRISPR/Cas9;  citation_journal_title=Nat. Commun;  citation_year=2014;  citation_volume=5;  citation_pages=5560" /><meta name="citation_reference" content="citation_title=CRISPaint allows modular base-specific gene tagging using a ligase-4-dependent mechanism;  citation_journal_title=Nat. Commun;  citation_year=2016;  citation_volume=7;  citation_pages=12338" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/6/609/22295781/dsx029.pdf" /><meta name="description" content="Abstract.  The efficacies of guide RNAs (gRNAs), the short RNA molecules that bind to and determine the sequence specificity of the Streptococcus pyogenes Cas9 " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/6/609/3904549" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/6/609/3904549" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/6/609/3904549");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx029';
    </script>
<title>
	convenient method to pre-screen candidate guide RNAs for CRISPR/Cas9 gene editing by NHEJ-mediated integration of a ‘self-cleaving’ GFP-expression plasmid | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3904549" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZrAATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVMOhbGFzLCBBbmRyw6FzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU2Nob29sIG9mIFBoLkQuIFN0dWRpZXMsIFNlbW1lbHdlaXMgVW5pdmVyc2l0eSwgQnVkYXBlc3QsIEh1bmdhcnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgRW56eW1vbG9neSwgUmVzZWFyY2ggQ2VudHJlIGZvciBOYXR1cmFsIFNjaWVuY2VzIG9mIHRoZSBIdW5nYXJpYW4gQWNhZGVteSBvZiBTY2llbmNlcywgQnVkYXBlc3QsIEh1bmdhcnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLdWxjc8OhciwgUMOpdGVyIElzdHbDoW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgRW56eW1vbG9neSwgUmVzZWFyY2ggQ2VudHJlIGZvciBOYXR1cmFsIFNjaWVuY2VzIG9mIHRoZSBIdW5nYXJpYW4gQWNhZGVteSBvZiBTY2llbmNlcywgQnVkYXBlc3QsIEh1bmdhcnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQmlvY2hlbWlzdHJ5LCBCaW9sb2dpY2FsIFJlc2VhcmNoIENlbnRyZSBvZiB0aGUgSHVuZ2FyaWFuIEFjYWRlbXkgb2YgU2NpZW5jZXMsIFN6ZWdlZCwgSHVuZ2FyeSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlVuaXZlcnNpdHkgb2YgU3plZ2VkLCBTemVnZWQsIEh1bmdhcnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXZWluaGFyZHQsIE7Ds3JhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEVuenltb2xvZ3ksIFJlc2VhcmNoIENlbnRyZSBmb3IgTmF0dXJhbCBTY2llbmNlcyBvZiB0aGUgSHVuZ2FyaWFuIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEJ1ZGFwZXN0LCBIdW5nYXJ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2NoZW1pc3RyeSwgQmlvbG9naWNhbCBSZXNlYXJjaCBDZW50cmUgb2YgdGhlIEh1bmdhcmlhbiBBY2FkZW15IG9mIFNjaWVuY2VzLCBTemVnZWQsIEh1bmdhcnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVbml2ZXJzaXR5IG9mIFN6ZWdlZCwgU3plZ2VkLCBIdW5nYXJ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQm9yc3ksIEFkcmllbm4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgRW56eW1vbG9neSwgUmVzZWFyY2ggQ2VudHJlIGZvciBOYXR1cmFsIFNjaWVuY2VzIG9mIHRoZSBIdW5nYXJpYW4gQWNhZGVteSBvZiBTY2llbmNlcywgQnVkYXBlc3QsIEh1bmdhcnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUw7N0aCwgRXN6dGVyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEVuenltb2xvZ3ksIFJlc2VhcmNoIENlbnRyZSBmb3IgTmF0dXJhbCBTY2llbmNlcyBvZiB0aGUgSHVuZ2FyaWFuIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEJ1ZGFwZXN0LCBIdW5nYXJ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2NoZW1pc3RyeSwgQmlvbG9naWNhbCBSZXNlYXJjaCBDZW50cmUgb2YgdGhlIEh1bmdhcmlhbiBBY2FkZW15IG9mIFNjaWVuY2VzLCBTemVnZWQsIEh1bmdhcnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTemViw6lueWksIEtvcm7DqWxpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBFbnp5bW9sb2d5LCBSZXNlYXJjaCBDZW50cmUgZm9yIE5hdHVyYWwgU2NpZW5jZXMgb2YgdGhlIEh1bmdhcmlhbiBBY2FkZW15IG9mIFNjaWVuY2VzLCBCdWRhcGVzdCwgSHVuZ2FyeSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktyYXVzeiwgU2FyYWggTGF1cmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTY2hvb2wgb2YgUGguRC4gU3R1ZGllcywgU2VtbWVsd2VpcyBVbml2ZXJzaXR5LCBCdWRhcGVzdCwgSHVuZ2FyeSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBFbnp5bW9sb2d5LCBSZXNlYXJjaCBDZW50cmUgZm9yIE5hdHVyYWwgU2NpZW5jZXMgb2YgdGhlIEh1bmdhcmlhbiBBY2FkZW15IG9mIFNjaWVuY2VzLCBCdWRhcGVzdCwgSHVuZ2FyeSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikh1c3rDoXIsIEtyaXN6dGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBFbnp5bW9sb2d5LCBSZXNlYXJjaCBDZW50cmUgZm9yIE5hdHVyYWwgU2NpZW5jZXMgb2YgdGhlIEh1bmdhcmlhbiBBY2FkZW15IG9mIFNjaWVuY2VzLCBCdWRhcGVzdCwgSHVuZ2FyeSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCaW9jaGVtaXN0cnksIEJpb2xvZ2ljYWwgUmVzZWFyY2ggQ2VudHJlIG9mIHRoZSBIdW5nYXJpYW4gQWNhZGVteSBvZiBTY2llbmNlcywgU3plZ2VkLCBIdW5nYXJ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVmlkYSwgSXN0dsOhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBFbnp5bW9sb2d5LCBSZXNlYXJjaCBDZW50cmUgZm9yIE5hdHVyYWwgU2NpZW5jZXMgb2YgdGhlIEh1bmdhcmlhbiBBY2FkZW15IG9mIFNjaWVuY2VzLCBCdWRhcGVzdCwgSHVuZ2FyeSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBPcmdhbmljIENoZW1pc3RyeSwgRcO2dHbDtnMgTG9yw6FuZCBVbml2ZXJzaXR5LCBCdWRhcGVzdCwgSHVuZ2FyeSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlN0dXJtLCDDgWTDoW0iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgRW56eW1vbG9neSwgUmVzZWFyY2ggQ2VudHJlIGZvciBOYXR1cmFsIFNjaWVuY2VzIG9mIHRoZSBIdW5nYXJpYW4gQWNhZGVteSBvZiBTY2llbmNlcywgQnVkYXBlc3QsIEh1bmdhcnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHb3Jkb3MsIEJpYW5rYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBFbnp5bW9sb2d5LCBSZXNlYXJjaCBDZW50cmUgZm9yIE5hdHVyYWwgU2NpZW5jZXMgb2YgdGhlIEh1bmdhcmlhbiBBY2FkZW15IG9mIFNjaWVuY2VzLCBCdWRhcGVzdCwgSHVuZ2FyeSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhvZmZtYW5uLCBPcnNvbHlhIEl2ZXR0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQW5pbWFsIEJpb3RlY2hub2xvZ3kgU2VjdGlvbiwgUnVtaW5hbnQgR2Vub21lIEJpb2xvZ3kgR3JvdXAsIE5BUklDIEFncmljdWx0dXJhbCBCaW90ZWNobm9sb2d5IEluc3RpdHV0ZSwgR8O2ZMO2bGzFkSwgSHVuZ2FyeSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJlbmNzdXJhLCBQZXRyYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBFbnp5bW9sb2d5LCBSZXNlYXJjaCBDZW50cmUgZm9yIE5hdHVyYWwgU2NpZW5jZXMgb2YgdGhlIEh1bmdhcmlhbiBBY2FkZW15IG9mIFNjaWVuY2VzLCBCdWRhcGVzdCwgSHVuZ2FyeSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCaW9jaGVtaXN0cnksIEJpb2xvZ2ljYWwgUmVzZWFyY2ggQ2VudHJlIG9mIHRoZSBIdW5nYXJpYW4gQWNhZGVteSBvZiBTY2llbmNlcywgU3plZ2VkLCBIdW5nYXJ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTnllc3RlLCBBbnRhbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBFbnp5bW9sb2d5LCBSZXNlYXJjaCBDZW50cmUgZm9yIE5hdHVyYWwgU2NpZW5jZXMgb2YgdGhlIEh1bmdhcmlhbiBBY2FkZW15IG9mIFNjaWVuY2VzLCBCdWRhcGVzdCwgSHVuZ2FyeSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0ZSBvZiBCaW9jaGVtaXN0cnksIEJpb2xvZ2ljYWwgUmVzZWFyY2ggQ2VudHJlIG9mIHRoZSBIdW5nYXJpYW4gQWNhZGVteSBvZiBTY2llbmNlcywgU3plZ2VkLCBIdW5nYXJ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGlnZXRpLCBab2x0w6FuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEVuenltb2xvZ3ksIFJlc2VhcmNoIENlbnRyZSBmb3IgTmF0dXJhbCBTY2llbmNlcyBvZiB0aGUgSHVuZ2FyaWFuIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEJ1ZGFwZXN0LCBIdW5nYXJ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRm9kb3IsIEVsZnJpZWRhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2NoZW1pc3RyeSwgQmlvbG9naWNhbCBSZXNlYXJjaCBDZW50cmUgb2YgdGhlIEh1bmdhcmlhbiBBY2FkZW15IG9mIFNjaWVuY2VzLCBTemVnZWQsIEh1bmdhcnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJXZWxrZXIsIEVydmluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEVuenltb2xvZ3ksIFJlc2VhcmNoIENlbnRyZSBmb3IgTmF0dXJhbCBTY2llbmNlcyBvZiB0aGUgSHVuZ2FyaWFuIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEJ1ZGFwZXN0LCBIdW5nYXJ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIEJpb2NoZW1pc3RyeSwgQmlvbG9naWNhbCBSZXNlYXJjaCBDZW50cmUgb2YgdGhlIEh1bmdhcmlhbiBBY2FkZW15IG9mIFNjaWVuY2VzLCBTemVnZWQsIEh1bmdhcnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkEgY29udmVuaWVudCBtZXRob2QgdG8gcHJlLXNjcmVlbiBjYW5kaWRhdGUgZ3VpZGUgUk5BcyBmb3IgQ1JJU1BSL0NhczkgZ2VuZSBlZGl0aW5nIGJ5IE5IRUotbWVkaWF0ZWQgaW50ZWdyYXRpb24gb2YgYSDigJhzZWxmLWNsZWF2aW5n4oCZIEdGUC1leHByZXNzaW9uIHBsYXNtaWQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI2MDkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjYyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeDAyOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE3LzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwcm9ncmFtbWFibGUgZHVhbC1STkEtZ3VpZGVkIEROQSBlbmRvbnVjbGVhc2UgaW4gYWRhcHRpdmUgYmFjdGVyaWFsIGltbXVuaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMzNzsgIGNpdGF0aW9uX3BhZ2VzPTgxNi0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BLWd1aWRlZCBodW1hbiBnZW5vbWUgZW5naW5lZXJpbmcgdmlhIENhczk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzM5OyAgY2l0YXRpb25fcGFnZXM9ODIzLTI2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZXggZ2Vub21lIGVuZ2luZWVyaW5nIHVzaW5nIENSSVNQUi9DYXMgc3lzdGVtczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMzk7ICBjaXRhdGlvbl9wYWdlcz04MTktMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQS1wcm9ncmFtbWVkIGdlbm9tZSBlZGl0aW5nIGluIGh1bWFuIGNlbGxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FbGlmZTsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9ZTAwNDcxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q1JJU1BSLW1lZGlhdGVkIG1vZHVsYXIgUk5BLWd1aWRlZCByZWd1bGF0aW9uIG9mIHRyYW5zY3JpcHRpb24gaW4gZXVrYXJ5b3RlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTQ7ICBjaXRhdGlvbl9wYWdlcz00NDItNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS1zY2FsZSBDUklTUFItQ2FzOSBrbm9ja291dCBzY3JlZW5pbmcgaW4gaHVtYW4gY2VsbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MzQzOyAgY2l0YXRpb25fcGFnZXM9ODQtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5naW5lZXJpbmcgY29tcGxleCBzeW50aGV0aWMgdHJhbnNjcmlwdGlvbmFsIHByb2dyYW1zIHdpdGggQ1JJU1BSIFJOQSBzY2FmZm9sZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTYwOyAgY2l0YXRpb25fcGFnZXM9MzM5LTUwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtc2NhbGUgQ1JJU1BSLW1lZGlhdGVkIGNvbnRyb2wgb2YgZ2VuZSByZXByZXNzaW9uIGFuZCBhY3RpdmF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OTsgIGNpdGF0aW9uX3BhZ2VzPTY0Ny02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwdXJwb3NpbmcgQ1JJU1BSIGFzIGFuIFJOQS1ndWlkZWQgcGxhdGZvcm0gZm9yIHNlcXVlbmNlLXNwZWNpZmljIGNvbnRyb2wgb2YgZ2VuZSBleHByZXNzaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE1MjsgIGNpdGF0aW9uX3BhZ2VzPTExNzMtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZGlzcnVwdGlvbiBieSBjZWxsLXBlbmV0cmF0aW5nIHBlcHRpZGUtbWVkaWF0ZWQgZGVsaXZlcnkgb2YgQ2FzOSBwcm90ZWluIGFuZCBndWlkZSBSTkE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xMDIwLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNBUzkgdHJhbnNjcmlwdGlvbmFsIGFjdGl2YXRvcnMgZm9yIHRhcmdldCBzcGVjaWZpY2l0eSBzY3JlZW5pbmcgYW5kIHBhaXJlZCBuaWNrYXNlcyBmb3IgY29vcGVyYXRpdmUgZ2Vub21lIGVuZ2luZWVyaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz04MzMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG91YmxlIG5pY2tpbmcgYnkgUk5BLWd1aWRlZCBDUklTUFIgQ2FzOSBmb3IgZW5oYW5jZWQgZ2Vub21lIGVkaXRpbmcgc3BlY2lmaWNpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTU0OyAgY2l0YXRpb25fcGFnZXM9MTM4MC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2RlbGluZyBEaXNlYXNlIEluIFZpdm8gV2l0aCBDUklTUFIvQ2FzOTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIE1vbC4gTWVkOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NjA5LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGd1aWRlIHRvIGdlbm9tZSBlbmdpbmVlcmluZyB3aXRoIHByb2dyYW1tYWJsZSBudWNsZWFzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTMyMS0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3Vycm9nYXRlIHJlcG9ydGVyLWJhc2VkIGVucmljaG1lbnQgb2YgY2VsbHMgY29udGFpbmluZyBSTkEtZ3VpZGVkIENhczkgbnVjbGVhc2UtaW5kdWNlZCBtdXRhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQ29tbXVuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0zMzc4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmF0aW9uYWwgZGVzaWduIG9mIGhpZ2hseSBhY3RpdmUgc2dSTkFzIGZvciBDUklTUFItQ2FzOS1tZWRpYXRlZCBnZW5lIGluYWN0aXZhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9MTI2Mi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHNjcmVlbnMgaW4gaHVtYW4gY2VsbHMgdXNpbmcgdGhlIENSSVNQUi1DYXM5IHN5c3RlbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zNDM7ICBjaXRhdGlvbl9wYWdlcz04MC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbnJhdmVsaW5nIENSSVNQUi1DYXM5IGdlbm9tZSBlbmdpbmVlcmluZyBwYXJhbWV0ZXJzIHZpYSBhIGxpYnJhcnktb24tbGlicmFyeSBhcHByb2FjaDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9ODIzLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNSSVNQUnNjYW46IGRlc2lnbmluZyBoaWdobHkgZWZmaWNpZW50IHNnUk5BcyBmb3IgQ1JJU1BSLUNhczkgdGFyZ2V0aW5nIGluIHZpdm87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTk4Mi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XVS1DUklTUFI6IGNoYXJhY3RlcmlzdGljcyBvZiBmdW5jdGlvbmFsIGd1aWRlIFJOQXMgZm9yIHRoZSBDUklTUFIvQ2FzOSBzeXN0ZW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcHRpbWl6ZWQgc2dSTkEgZGVzaWduIHRvIG1heGltaXplIGFjdGl2aXR5IGFuZCBtaW5pbWl6ZSBvZmYtdGFyZ2V0IGVmZmVjdHMgb2YgQ1JJU1BSLUNhczk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTE4NC05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVvc29tZXMgaW5oaWJpdCBDYXM5IGVuZG9udWNsZWFzZSBhY3Rpdml0eSBpbiB2aXRybzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvY2hlbWlzdHJ5OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9NzA2My02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZW9zb21lcyBpbXBlZGUgQ2FzOSBhY2Nlc3MgdG8gRE5BIGluIHZpdm8gYW5kIGluIHZpdHJvOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FbGlmZTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEyNjc3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpZnlpbmcgb24tYW5kIG9mZi10YXJnZXQgZ2Vub21lIGVkaXRpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MTMyLTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXJnZXRlZCBnZXJtbGluZSBtb2RpZmljYXRpb25zIGluIHJhdHMgdXNpbmcgQ1JJU1BSL0NhczkgYW5kIHNwZXJtYXRvZ29uaWFsIHN0ZW0gY2VsbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGwgUmVwOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTgyOC0zNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5kdWNpYmxlIGluIHZpdm8gZ2Vub21lIGVkaXRpbmcgd2l0aCBDUklTUFItQ2FzOTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0zOTAtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIGVuZ2luZWVyaW5nIHVzaW5nIHRoZSBDUklTUFItQ2FzOSBzeXN0ZW07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUHJvdG9jOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0yMjgxLTMwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0ZWQgZ2Vub21lIGVuZ2luZWVyaW5nIGluIGh1bWFuIGNlbGxzIHdpdGggdGhlIENhczkgUk5BLWd1aWRlZCBlbmRvbnVjbGVhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTIzMC0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdXRhdGlvbiBkZXRlY3Rpb24gdXNpbmcgU3VydmV5b3LihKIgbnVjbGVhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb3RlY2huaXF1ZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz03MDItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2ltcGxlIG1ldGhvZHMgZm9yIGdlbmVyYXRpbmcgYW5kIGRldGVjdGluZyBsb2N1cy1zcGVjaWZpYyBtdXRhdGlvbnMgaW5kdWNlZCB3aXRoIFRBTEVOcyBpbiB0aGUgemVicmFmaXNoIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDI4NjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbmJpYXNlZCBkZXRlY3Rpb24gb2Ygb2ZmLXRhcmdldCBjbGVhdmFnZSBieSBDUklTUFItQ2FzOSBhbmQgVEFMRU5zIHVzaW5nIGludGVncmFzZS1kZWZlY3RpdmUgbGVudGl2aXJhbCB2ZWN0b3JzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0xNzUtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IGdlbm90eXBpbmcgb2YgQ1JJU1BSL0NhczktbWVkaWF0ZWQgbXV0YW50cyB1c2luZyBmbHVvcmVzY2VudCBQQ1ItY2FwaWxsYXJ5IGdlbCBlbGVjdHJvcGhvcmVzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaS4gUmVwOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0xNTU4NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVhc3kgcXVhbnRpdGF0aXZlIGFzc2Vzc21lbnQgb2YgZ2Vub21lIGVkaXRpbmcgYnkgc2VxdWVuY2UgdHJhY2UgZGVjb21wb3NpdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz1lMTY4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3B0aW1pemF0aW9uIG9mIHNjYXJsZXNzIGh1bWFuIHN0ZW0gY2VsbCBnZW5vbWUgZWRpdGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz05MDQ5LTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdobHkgZWZmaWNpZW50IGVuZG9nZW5vdXMgaHVtYW4gZ2VuZSBjb3JyZWN0aW9uIHVzaW5nIGRlc2lnbmVkIHppbmMtZmluZ2VyIG51Y2xlYXNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTQzNTsgIGNpdGF0aW9uX3BhZ2VzPTY0Ni01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmaWNpZW50IENSSVNQUi1DYXM5LW1lZGlhdGVkIGdlbm9tZSBlZGl0aW5nIGluIFBsYXNtb2RpdW0gZmFsY2lwYXJ1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9OTE1LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9uZS1zdGVwIGdlbmVyYXRpb24gb2YgbWljZSBjYXJyeWluZyBtdXRhdGlvbnMgaW4gbXVsdGlwbGUgZ2VuZXMgYnkgQ1JJU1BSL0Nhcy1tZWRpYXRlZCBnZW5vbWUgZW5naW5lZXJpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTUzOyAgY2l0YXRpb25fcGFnZXM9OTEwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2hseSBlZmZpY2llbnQgQ1JJU1BSL0NhczktbWVkaWF0ZWQga25vY2staW4gaW4gemVicmFmaXNoIGJ5IGhvbW9sb2d5LWluZGVwZW5kZW50IEROQSByZXBhaXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNDItNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYWNraW5nIGdlbm9tZSBlbmdpbmVlcmluZyBvdXRjb21lIGF0IGluZGl2aWR1YWwgRE5BIGJyZWFrcG9pbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTY3MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lcmF0aW9uIG9mIG11dGFudCBtaWNlIGJ5IHByb251Y2xlYXIgaW5qZWN0aW9uIG9mIGNpcmN1bGFyIHBsYXNtaWQgZXhwcmVzc2luZyBDYXM5IGFuZCBzaW5nbGUgZ3VpZGVkIFJOQTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTMzNTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZpY2llbnQgdGFyZ2V0aW5nIG9mIGV4cHJlc3NlZCBhbmQgc2lsZW50IGdlbmVzIGluIGh1bWFuIEVTQ3MgYW5kIGlQU0NzIHVzaW5nIHppbmMtZmluZ2VyIG51Y2xlYXNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9ODUxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN1cnJvZ2F0ZSByZXBvcnRlcnMgZm9yIGVucmljaG1lbnQgb2YgY2VsbHMgd2l0aCBudWNsZWFzZS1pbmR1Y2VkIG11dGF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz05NDEtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9WmluYy1maW5nZXIgbnVjbGVhc2UtZHJpdmVuIHRhcmdldGVkIGludGVncmF0aW9uIGludG8gbWFtbWFsaWFuIGdlbm9tZXMgdXNpbmcgZG9ub3JzIHdpdGggbGltaXRlZCBjaHJvbW9zb21hbCBob21vbG9neTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz1lMTUyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW4gdml2byBjbGVhdmFnZSBvZiB0cmFuc2dlbmUgZG9ub3JzIHByb21vdGVzIG51Y2xlYXNl4oCQbWVkaWF0ZWQgdGFyZ2V0ZWQgaW50ZWdyYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb3RlY2hub2wuIEJpb2VuZzsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTA7ICBjaXRhdGlvbl9wYWdlcz04NzEtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNSSVNQUuKAkENhcyB0YXJnZXRlZCBwbGFzbWlkIGludGVncmF0aW9uIGludG8gbWFtbWFsaWFuIGNlbGxzIHZpYSBub27igJBob21vbG9nb3VzIGVuZCBqb2luaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW90ZWNobm9sLiBCaW9lbmc7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTEyOyAgY2l0YXRpb25fcGFnZXM9MjE1NC02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5lcmljIHN0cmF0ZWd5IGZvciBDUklTUFItQ2FzOS1tZWRpYXRlZCBnZW5lIHRhZ2dpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQ29tbXVuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xMDIzNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU11bHRpY29sb3IgQ1JJU1BSIGxhYmVsaW5nIG9mIGNocm9tb3NvbWFsIGxvY2kgaW4gaHVtYW4gY2VsbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTExMjsgIGNpdGF0aW9uX3BhZ2VzPTMwMDItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVzdHJpY3Rpb24gZW56eW1lIGJvZHkgZG91YmxlcyBhbmQgUENSIGNsb25pbmc6IG9uIHRoZSBnZW5lcmFsIHVzZSBvZiB0eXBlIElJcyByZXN0cmljdGlvbiBlbnp5bWVzIGZvciBjbG9uaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTkwODk2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGVudGl2aXJ1cy1kZWxpdmVyZWQgc3RhYmxlIGdlbmUgc2lsZW5jaW5nIGJ5IFJOQWkgaW4gcHJpbWFyeSBjZWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Uk5BOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz00OTMtNTAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGdlbmVyYWwgY2hlbWljYWwgbWV0aG9kIHRvIHJlZ3VsYXRlIHByb3RlaW4gc3RhYmlsaXR5IGluIHRoZSBtYW1tYWxpYW4gY2VudHJhbCBuZXJ2b3VzIHN5c3RlbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2hlbS4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTk4MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIHRhcmdldGluZyB0byB0aGUgUk9TQTI2IGxvY3VzIGRpcmVjdGVkIGJ5IGVuZ2luZWVyZWQgemluYyBmaW5nZXIgbnVjbGVhc2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTM3NDEtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9ibGlnYXRlIGxpZ2F0aW9uLWdhdGVkIHJlY29tYmluYXRpb24gKE9iTGlHYVJlKTogY3VzdG9tLWRlc2lnbmVkIG51Y2xlYXNlLW1lZGlhdGVkIHRhcmdldGVkIGludGVncmF0aW9uIHRocm91Z2ggbm9uaG9tb2xvZ291cyBlbmQgam9pbmluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTUzOS00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RS1DUklTUDogZmFzdCBDUklTUFIgdGFyZ2V0IHNpdGUgaWRlbnRpZmljYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTEyMi0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJBYWNoIEouLCBNYWxpIFAuLCBDaHVyY2ggRy4gTS4gIDIwMTQgLCBDYXNGaW5kZXI6IEZsZXhpYmxlIGFsZ29yaXRobSBmb3IgaWRlbnRpZnlpbmcgc3BlY2lmaWMgQ2FzOSB0YXJnZXRzIGluIGdlbm9tZXMuIGJpb1J4aXYgLCAwMDUwNzQuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNSSVNQUi1tZWRpYXRlZCBkaXJlY3QgbXV0YXRpb24gb2YgY2FuY2VyIGdlbmVzIGluIHRoZSBtb3VzZSBsaXZlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTUxNDsgIGNpdGF0aW9uX3BhZ2VzPTM4MC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZpY2llbnQgZ2VuZXJhdGlvbiBvZiBsYXJnZS1zY2FsZSBnZW5vbWUtbW9kaWZpZWQgbWljZSB1c2luZyBnUk5BIGFuZCBDQVM5IGVuZG9udWNsZWFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz1lMTg3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TnVjbGVhciBkb21haW4g4oCYa25vY2staW7igJlzY3JlZW4gZm9yIHRoZSBldmFsdWF0aW9uIGFuZCBpZGVudGlmaWNhdGlvbiBvZiBzbWFsbCBtb2xlY3VsZSBlbmhhbmNlcnMgb2YgQ1JJU1BSLWJhc2VkIGdlbm9tZSBlZGl0aW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTkzNzktOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IkxpIEsuLCBXYW5nIEcuLCBBbmRlcnNlbiBULiwgWmhvdSBQLiwgUHUgVy4gVC4gIDIwMTQgLCBPcHRpbWl6YXRpb24gb2YgZ2Vub21lIGVuZ2luZWVyaW5nIGFwcHJvYWNoZXMgd2l0aCB0aGUgQ1JJU1BSL0Nhczkgc3lzdGVtLiBQbG9zIE9uZS4sIDkgLCBlMTA1Nzc5LiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJHZWlzaW5nZXIgSi4gTS4sIFR1cmFuIFMuLCBIZXJuYW5kZXogUy4sIFNwZWN0b3IgTC4gUC4sIENhbG9zIE0uIFAuICAyMDE2ICwgSW4gdml2byBibHVudC1lbmQgY2xvbmluZyB0aHJvdWdoIENSSVNQUi9DYXM5LWZhY2lsaXRhdGVkIG5vbi1ob21vbG9nb3VzIGVuZC1qb2luaW5nLiBOdWNsZWljIEFjaWRzIFJlcy4gLCA0NCAsIGU3NiDigJMgZTc2LiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXRpb25hbGx5IGVuZ2luZWVyZWQgQ2FzOSBudWNsZWFzZXMgd2l0aCBpbXByb3ZlZCBzcGVjaWZpY2l0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNTE7ICBjaXRhdGlvbl9wYWdlcz04NC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWdoLWZpZGVsaXR5IENSSVNQUuKAk0NhczkgbnVjbGVhc2VzIHdpdGggbm8gZGV0ZWN0YWJsZSBnZW5vbWUtd2lkZSBvZmYtdGFyZ2V0IGVmZmVjdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT01Mjk7ICBjaXRhdGlvbl9wYWdlcz00OTAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5kdWN0aW9uIGFuZCByZXBhaXIgb2YgRE5BIGRvdWJsZSBzdHJhbmQgYnJlYWtzOiB0aGUgaW5jcmVhc2luZyBzcGVjdHJ1bSBvZiBub24taG9tb2xvZ291cyBlbmQgam9pbmluZyBwYXRod2F5czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TXV0YXQuIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03MTE7ICBjaXRhdGlvbl9wYWdlcz02MS03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9ob21vbG9neS1iYXNlZCBjaG9pY2Ugb2YgQ2FzOSBudWNsZWFzZSB0YXJnZXQgc2l0ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTcwNS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NTUVKIHJlcGFpciBvZiBkb3VibGUtc3RyYW5kIGJyZWFrcyAoZGlyZWN0b3LigJlzIGN1dCk6IGRlbGV0ZWQgc2VxdWVuY2VzIGFuZCBhbHRlcm5hdGl2ZSBlbmRpbmdzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz01MjktMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmlzb24gb2YgVDdFMSBhbmQgc3VydmV5b3IgbWlzbWF0Y2ggY2xlYXZhZ2UgYXNzYXlzIHRvIGRldGVjdCBtdXRhdGlvbnMgdHJpZ2dlcmVkIGJ5IGVuZ2luZWVyZWQgbnVjbGVhc2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HMzsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9NDA3LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb2NoaXAgZWxlY3Ryb3Bob3Jlc2lzIGNvdXBsZWQgd2l0aCBvbuKAkGxpbmUgbWFnbmV0aWMgc2VwYXJhdGlvbiBhbmQgY2hlbWlsdW1pbmVzY2VuY2UgZGV0ZWN0aW9uIGZvciBtdWx0aXBsZXhlZCBpbW11bm9hc3NheTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RWxlY3Ryb3Bob3Jlc2lzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MTE5OC0yMDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZyb250LWxpbmUgb2YgZ2Vub21lIGVkaXRpbmcgdGVjaG5vbG9neSBmb3IgYW5pbWFsIGNlbGwgZW5naW5lZXJpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBQcm9jZWVkLiw7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl9wYWdlcz1PMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIG5vdmVsIHZpc3VhbC1wbHVzIHF1YW50aXRhdGl2ZSBhbmFseXNpcyBzeXN0ZW1zIGZvciBzdHVkeWluZyBETkEgZG91YmxlLXN0cmFuZCBicmVhayByZXBhaXJzIGluIHplYnJhZmlzaDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gR2VuZXQuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9NDg5LTUwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9LCBOb24taG9tb2xvZ291cyBlbmQgam9pbmluZyBwbGF5cyBhIGtleSByb2xlIGluIHRyYW5zZ2VuZSBjb25jYXRlbWVyIGZvcm1hdGlvbiBpbiB0cmFuc2dlbmljIHplYnJhZmlzaCBlbWJyeW9zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIEJpb2wuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NzU2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG9tb2xvZ291cyByZWNvbWJpbmF0aW9uIGFuZCBETkEtZW5kIGpvaW5pbmcgcmVhY3Rpb25zIGluIHp5Z290ZXMgYW5kIGVhcmx5IGVtYnJ5b3Mgb2YgemVicmFmaXNoIChEYW5pbyByZXJpbykgYW5kIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9sLiBDaGVtOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTM3OTsgIGNpdGF0aW9uX3BhZ2VzPTY3My04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW4gdml2byBnZW5vbWUgZWRpdGluZyB2aWEgQ1JJU1BSL0NhczkgbWVkaWF0ZWQgaG9tb2xvZ3ktaW5kZXBlbmRlbnQgdGFyZ2V0ZWQgaW50ZWdyYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT01NDA7ICBjaXRhdGlvbl9wYWdlcz0xNDQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9ob21vbG9neS1tZWRpYXRlZCBlbmQtam9pbmluZy1kZXBlbmRlbnQgaW50ZWdyYXRpb24gb2YgZG9ub3IgRE5BIGluIGNlbGxzIGFuZCBhbmltYWxzIHVzaW5nIFRBTEVOcyBhbmQgQ1JJU1BSL0Nhczk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQ29tbXVuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz01NTYwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q1JJU1BhaW50IGFsbG93cyBtb2R1bGFyIGJhc2Utc3BlY2lmaWMgZ2VuZSB0YWdnaW5nIHVzaW5nIGEgbGlnYXNlLTQtZGVwZW5kZW50IG1lY2hhbmlzbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBDb21tdW47ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTEyMzM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI0LzYvNjA5LzIyMjk1NzgxL2RzeDAyOS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlIGVmZmljYWNpZXMgb2YgZ3VpZGUgUk5BcyAoZ1JOQXMpLCB0aGUgc2hvcnQgUk5BIG1vbGVjdWxlcyB0aGF0IGJpbmQgdG8gYW5kIGRldGVybWluZSB0aGUgc2VxdWVuY2Ugc3BlY2lmaWNpdHkgb2YgdGhlIFN0cmVwdG9jb2NjdXMgcHlvZ2VuZXMgQ2FzOSAiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM5MDQ1NDkmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkyNoPMhfPFwzuFMHyKk/aUQM3/7c=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f609%2f3904549" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f609%2f3904549" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=103821" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&amp;Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/6">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="104086353">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086353">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086355">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086355">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086360">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086360">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086384">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086384">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086425">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086425">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086438">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086438">5. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086440">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086440">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086454">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086454">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/6/597/3902984">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/6/623/3953933">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A convenient method to pre-screen candidate guide RNAs for CRISPR/Cas9 gene editing by NHEJ-mediated integration of a ‘self-cleaving’ GFP-expression plasmid
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Andr&#225;s T&#225;las</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Ph.D. Studies, Semmelweis University, Budapest, Hungary</div><div class="aff">Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Andr%c3%a1s+T%c3%a1las">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=T&#225;las A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;T&#225;las A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">András Tálas</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">P&#233;ter Istv&#225;n Kulcs&#225;r</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary</div><div class="aff">Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary</div><div class="aff">University of Szeged, Szeged, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=P%c3%a9ter+Istv%c3%a1n+Kulcs%c3%a1r">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kulcs&#225;r P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kulcs&#225;r P&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Péter István Kulcsár</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">N&#243;ra Weinhardt</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary</div><div class="aff">Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary</div><div class="aff">University of Szeged, Szeged, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=N%c3%b3ra+Weinhardt">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Weinhardt N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Weinhardt N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Nóra Weinhardt</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Adrienn Borsy</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Adrienn+Borsy">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Borsy A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Borsy A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Adrienn Borsy</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eszter T&#243;th</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary</div><div class="aff">Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eszter+T%c3%b3th">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=T&#243;th E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;T&#243;th E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Eszter Tóth</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Korn&#233;lia Szeb&#233;nyi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Korn%c3%a9lia+Szeb%c3%a9nyi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Szeb&#233;nyi K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Szeb&#233;nyi K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kornélia Szebényi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sarah Laura Krausz</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">School of Ph.D. Studies, Semmelweis University, Budapest, Hungary</div><div class="aff">Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sarah+Laura+Krausz">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Krausz S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Krausz S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Sarah Laura Krausz</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Krisztina Husz&#225;r</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary</div><div class="aff">Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Krisztina+Husz%c3%a1r">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Husz&#225;r K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Husz&#225;r K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Krisztina Huszár</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Istv&#225;n Vida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary</div><div class="aff">Institute of Organic Chemistry, Eötvös Loránd University, Budapest, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Istv%c3%a1n+Vida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Vida I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Vida I&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">István Vida</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">&#193;d&#225;m Sturm</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=%c3%81d%c3%a1m+Sturm">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sturm &#193;">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sturm &#193;&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Ádám Sturm</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bianka Gordos</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bianka+Gordos">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gordos B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gordos B&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Bianka Gordos</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Orsolya Ivett Hoffmann</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Animal Biotechnology Section, Ruminant Genome Biology Group, NARIC Agricultural Biotechnology Institute, Gödöllő, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Orsolya+Ivett+Hoffmann">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hoffmann O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hoffmann O&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Orsolya Ivett Hoffmann</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Petra Bencsura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary</div><div class="aff">Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Petra+Bencsura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bencsura P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bencsura P&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Petra Bencsura</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Antal Nyeste</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary</div><div class="aff">Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Antal+Nyeste">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nyeste A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nyeste A&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Antal Nyeste</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zolt&#225;n Ligeti</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zolt%c3%a1n+Ligeti">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ligeti Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ligeti Z&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Zoltán Ligeti</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Elfrieda Fodor</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Elfrieda+Fodor">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fodor E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fodor E&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Elfrieda Fodor</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ervin Welker</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Enzymology, Research Centre for Natural Sciences of the Hungarian Academy of Sciences, Budapest, Hungary</div><div class="aff">Institute of Biochemistry, Biological Research Centre of the Hungarian Academy of Sciences, Szeged, Hungary</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx029-cor1" data-open="dsx029-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +361 382 6610. Email: <a href="mailto:welker.ervin@ttk.mta.hu" target="_blank">welker.ervin@ttk.mta.hu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ervin+Welker">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Welker E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Welker E&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Ervin Welker</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx029-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Toshihiko Shiroishi</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 609–621, <a href='https://doi.org/10.1093/dnares/dsx029'>https://doi.org/10.1093/dnares/dsx029</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">30 June 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">29 November 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">07 June 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3904549" href="/dnaresearch/article-pdf/24/6/609/22295781/dsx029.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>András Tálas, Péter István Kulcsár, Nóra Weinhardt, Adrienn Borsy, Eszter Tóth, Kornélia Szebényi, Sarah Laura Krausz, Krisztina Huszár, István Vida, Ádám Sturm, Bianka Gordos, Orsolya Ivett Hoffmann, Petra Bencsura, Antal Nyeste, Zoltán Ligeti, Elfrieda Fodor, Ervin Welker;  A convenient method to pre-screen candidate guide RNAs for CRISPR/Cas9 gene editing by NHEJ-mediated integration of a ‘self-cleaving’ GFP-expression plasmid, <em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 609–621, <a href='https://doi.org/10.1093/dnares/dsx029'>https://doi.org/10.1093/dnares/dsx029</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3904549?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3904549?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3904549?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3904549?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3904549?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3904549?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3904549?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20convenient%20method%20to%20pre-screen%20candidate%20guide%20RNAs%20for%20CRISPR%2FCas9%20gene%20editing%20by%20NHEJ-mediated%20integration%20of%20a%20%E2%80%98self-cleaving%E2%80%99%20GFP-expression%20plasmid&amp;publicationDate=2017-06-30&amp;volumeNum=24&amp;issueNum=6&amp;author=T%C3%A1las%2C%20Andr%C3%A1s%3B%20Kulcs%C3%A1r%2C%20P%C3%A9ter%20Istv%C3%A1n&amp;startPage=609&amp;endPage=621&amp;contentId=10.1093%2Fdnares%2Fdsx029&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f609%2f3904549" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=104086353 id="104086353" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The efficacies of guide RNAs (gRNAs), the short RNA molecules that bind to and determine the sequence specificity of the <em>Streptococcus pyogenes</em> Cas9 nuclease, to mediate DNA cleavage vary dramatically. Thus, the selection of appropriate target sites, and hence spacer sequence, is critical for most applications. Here, we describe a simple, unparalleled method for experimentally pre-testing the efficiencies of various gRNAs targeting a gene. The method explores NHEJ-cloning, genomic integration of a GFP-expressing plasmid without homologous arms and linearized in-cell. The use of ‘self-cleaving’ GFP-plasmids containing universal gRNAs and corresponding targets alleviates cloning burdens when this method is applied. These universal gRNAs mediate efficient plasmid cleavage and are designed to avoid genomic targets in several model species. The method combines the advantages of the straightforward FACS detection provided by applying fluorescent reporter systems and of the PCR-based approaches being capable of testing targets in their genomic context, without necessitating any extra cloning steps. Additionally, we show that NHEJ-cloning can also be used in mammalian cells for targeted integration of donor plasmids up to 10 kb in size, with up to 30% efficiency, without any selection or enrichment.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="Cas9">Cas9</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="CRISPR">CRISPR</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gRNA testing&quot;">gRNA testing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;reporter assay&quot;">reporter assay</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="NHEJ-cloning">NHEJ-cloning</a></div>                            <h2 scrollto-destination=104086355 id="104086355" class="section-title" >1. Introduction</h2>
<p>The use of RNA-guided nucleases revolutionized our ability to modify complex genomes and to control gene expression networks, in the same manner that PCR revolutionized molecular biology at the end of the 20th century.<a reveal-id="dsx029-B1 dsx029-B2 dsx029-B3 dsx029-B4 dsx029-B5 dsx029-B6 dsx029-B7 dsx029-B8 dsx029-B9 dsx029-B10" data-open="dsx029-B1 dsx029-B2 dsx029-B3 dsx029-B4 dsx029-B5 dsx029-B6 dsx029-B7 dsx029-B8 dsx029-B9 dsx029-B10" class="link link-ref link-reveal xref-bibr"><sup>1–10</sup></a> The specificity of Cas9 nucleases is determined in part by the so-called spacer sequence of the guide RNA (gRNA) associated to the protein and responsible for target recognition. This makes the Cas9 proteins easily reprogrammable by simply varying the sequence of the spacer.<a reveal-id="dsx029-B1" data-open="dsx029-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> This attractive feature of the Cas9 nucleases has been exploited for genome editing and gene expression modulation, as well as for generating disease and therapeutic models.<a reveal-id="dsx029-B7" data-open="dsx029-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx029-B9" data-open="dsx029-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsx029-B11 dsx029-B12 dsx029-B13 dsx029-B14 dsx029-B15" data-open="dsx029-B11 dsx029-B12 dsx029-B13 dsx029-B14 dsx029-B15" class="link link-ref link-reveal xref-bibr"><sup>11–15</sup></a></p><p>The efficiencies of various gRNA spacers to result targeted DNA cleavage by Cas9 vary widely. While several sequence determinants of efficient target recognition for <em>Streptococcus pyogenes</em> Cas9 (SpCas9), have been revealed,<a reveal-id="dsx029-B16 dsx029-B17 dsx029-B18 dsx029-B19 dsx029-B20 dsx029-B21" data-open="dsx029-B16 dsx029-B17 dsx029-B18 dsx029-B19 dsx029-B20 dsx029-B21" class="link link-ref link-reveal xref-bibr"><sup>16–21</sup></a> an adequate prediction of the efficiency of a given spacer sequence has not been achieved, partially because the activity of a gRNA may be modulated by the genomic context of the target: a factor difficult to predict.<a reveal-id="dsx029-B22" data-open="dsx029-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a><sup>,</sup><a reveal-id="dsx029-B23" data-open="dsx029-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Generally, several candidate spacer sequences are tested before choosing the appropriate ones for performing the desired genetic modifications. Existing methods to test the efficiency of a given spacer sequence commonly measure the sequence alterations acquired during the repair of the cleaved DNA.<a reveal-id="dsx029-B24" data-open="dsx029-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Breaks in the DNA involving one or both strands are effectively repaired by the cell using one of the two main repair pathways: the homologous recombination (HR) or the more error-prone non-homologous end-joining (NHEJ).<a reveal-id="dsx029-B25" data-open="dsx029-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> NHEJ-dependent repair of double strand breaks frequently leads to small insertions or deletions (indels), the frequency of which can be explored to monitor the efficiency of Cas9 cleavage.<a reveal-id="dsx029-B26 dsx029-B27 dsx029-B28" data-open="dsx029-B26 dsx029-B27 dsx029-B28" class="link link-ref link-reveal xref-bibr"><sup>26–28</sup></a> Accordingly, many methods rely on the assessment of indel frequencies that are usually determined by PCR amplification of the corresponding region followed by Surveyor/T7E1 nuclease assays,<a reveal-id="dsx029-B29" data-open="dsx029-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> high resolution melt analysis,<a reveal-id="dsx029-B30" data-open="dsx029-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a><sup>,</sup><a reveal-id="dsx029-B31" data-open="dsx029-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> fluorescent PCR-capillary gel electrophoresis<a reveal-id="dsx029-B32" data-open="dsx029-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> and direct<a reveal-id="dsx029-B33" data-open="dsx029-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> or deep<a reveal-id="dsx029-B11" data-open="dsx029-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsx029-B12" data-open="dsx029-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><sup>,</sup><a reveal-id="dsx029-B34" data-open="dsx029-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> sequencing. Alternatively, a large number of clones can be sequenced upon the PCR amplification of the regions of interest. The occurrence of HR events as a result of HR repair can also be monitored by analysing clones using PCR followed by restriction digestion<a reveal-id="dsx029-B35" data-open="dsx029-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> or by sequencing,<a reveal-id="dsx029-B34" data-open="dsx029-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> as well as by Southern blotting<a reveal-id="dsx029-B36 dsx029-B37 dsx029-B38" data-open="dsx029-B36 dsx029-B37 dsx029-B38" class="link link-ref link-reveal xref-bibr"><sup>36–38</sup></a> of genomic DNA. As another approach, reporter assays are explored for detecting HR events: (I) where a fluorescent signal is measured as a result of recombination events correcting a truncated fluorescent protein;<a reveal-id="dsx029-B39" data-open="dsx029-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsx029-B40" data-open="dsx029-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> (II) or where a fluorescent protein expression cassette is incorporated at the cleavage site - at the cost of the laborious construction of homologous arms.<a reveal-id="dsx029-B41" data-open="dsx029-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> In case of exploiting NHEJ repair, reporter assays are also frequently employed for monitoring indel events that alter the reading frame of a fluorescent protein, resulting in either loss or recovery of the fluorescent signal.<a reveal-id="dsx029-B35" data-open="dsx029-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><a reveal-id="dsx029-B39" data-open="dsx029-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsx029-B42" data-open="dsx029-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> Reporter assays that enable the use of fluorescence activated cell sorting (FACS) to monitor the repair of double strand breaks are convenient, making genomic DNA isolation and PCR amplification -including the sometimes very tedious condition/primer optimization- unnecessary. They also provide more accurate estimations than Surveyor/T7E1 assays and are much less expensive than deep sequencing.<a reveal-id="dsx029-B24" data-open="dsx029-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> However, these approaches generally require extra cloning steps to become applicable to a particular task and lose the advantage of PCR-based methods of being capable of monitoring spacer efficiency in the genomic context of the particular targets.</p><p>Here, we introduce a reporter assay, which requires no additional cloning steps and is capable of testing spacer efficiencies on targets within their genomic context. The method explores NHEJ-cloning (NHEJ repair mediated integration) of a GFP-expression cassette to a target site that is cleaved by SpCas9. The key feature of this approach is the use of a ‘self-cleaving’ plasmid that enhances targeted integration.</p><p>Using SpCas9 and a self-cleaving plasmid, we also demonstrate a very convenient and effective way of inserting relatively large DNA fragments into the mammalian genome. NHEJ-cloning has been described in zebrafish, using transcription activator-like effector nuclease (TALEN) and SpCas9, and in mammalian cells, using zinc-finger nuclease (ZFN) and TALEN.<a reveal-id="dsx029-B38" data-open="dsx029-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a><sup>,</sup><a reveal-id="dsx029-B43" data-open="dsx029-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsx029-B44" data-open="dsx029-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> There have been a few attempts to use SpCas9 for NHEJ-cloning of larger DNA cassettes in mammalian cells as well,<a reveal-id="dsx029-B45" data-open="dsx029-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><sup>,</sup><a reveal-id="dsx029-B46" data-open="dsx029-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> however, they achieved integrations at a relatively low frequency, typically under 1% without selection or FACS-enrichment. Here, we achieved a frequency that is greater by over an order of magnitude. Thus, this approach facilitates the implementation of several genome engineering tasks in mammalian systems, making them less labor-intensive, especially so regarding the insertion of expression cassettes to safe-harbour locations in contrast with the use of random integration.</p>                            <h2 scrollto-destination=104086360 id="104086360" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=104086361 id="104086361" class="section-title" >2.1. Materials</h3>
<p>All restriction enzymes, Klenow polymerase and T4 ligase were acquired from Thermo Scientific, Q5 and Phusion polymerases T7 endonuclease I, and Gibson Assembly Master Mix were from New England Biolabs Inc. Oligonucleotides were purchased from Microsynth AG and Sigma-Aldrich Co. and their sequences are listed in the <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_data_dsx029.xlsx?Expires=2147483647&Signature=tvN8l6Nr13c-J7Z5ueuYtmqjQDRRDKGeMZZ4zP2xfTAnHe5~chW-J73UkVFgA2WKJsBA02QbIMp6AMjSQiHSwMi0vHIVHEbihM53zO2GWQvuc4s2FTp1W9zVQUk9sLiwr5wcXiiABvFYaNZHZUtIk4XHVLMRF7WB-X7LD~F1Kvfd7WpSBIrtxhybwx0b1m2jkT6FrEL5~IBFX0ygmG~VV1Mnqudots2x7TJaZbc6aOeoBKvcy9vW8rzF6Vlr8ElemUAIh~IO~SFhI4KvLxWuzc14LBQoyV6OmOXBQL0M1rsXEwQYXq189aSoITXXGt2Lp8RRmBOJ1IL4zERV9~H8WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>. Dulbecco’s modified Eagle’s Medium, foetal bovine serum, Turbofect, Lipofectamine 2000, GlutaMAX, penicillin, streptomycin and puromycin were acquired from Thermo Fisher Scientific, and trimethoprim (TMP) from Sigma-Aldrich Co. (T7883).</p>                            <h3 scrollto-destination=104086363 id="104086363" class="section-title" >2.2. Vector constructions</h3>
<p>Briefly, SpCas9-gRNA expressing plasmids (targeting human and mouse genes and also containing TL1 and TL2 spacers) were generated from the px330 vector (Addgene # 42230<a reveal-id="dsx029-B3" data-open="dsx029-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a>), and cloning spacers into pmCherry_gRNA plasmid was conducted according to Hanhui Ma et al.<a reveal-id="dsx029-B47" data-open="dsx029-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> The construction of pmCherry_gRNA, pX330-Flag-ehSpCas9 and pX330-Flag-wtSpCas9 is described in (Kulcsár et al. 2017 submitted). The nuclease inactive SpCas9 (dCas9) expressing plasmid was created from px335 (Addgene #42335<a reveal-id="dsx029-B3" data-open="dsx029-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a>) by using Body Double cloning method.<a reveal-id="dsx029-B48" data-open="dsx029-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> The self-cleaving plasmids were created by using Gibson assembly method. The self-cleaving plasmid containing a puromycin cassette (pSc1-puro) was constructed by using pLKO.1puro plasmid (Addgene #8453<a reveal-id="dsx029-B49" data-open="dsx029-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a>). The self-cleaving plasmid with a degradation domain (pSc1-DD) was created from pBMN DHFR(DD)-YFP plasmid (Addgene #29325<a reveal-id="dsx029-B50" data-open="dsx029-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a>). The <em>Prnp</em> gene targeting homologous recombination donor plasmid was created by using nested PCR. The <em>Rosa26</em> locus homologous recombination donor plasmid was created by using pDonor-MCS-rosa26 plasmid [Addgene #37200<a reveal-id="dsx029-B51" data-open="dsx029-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a>]. For more detailed information see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Materials and Methods</a></span>. The sequences of all plasmid constructs were confirmed by Sanger sequencing (Microsynth AG).</p>                            <h3 scrollto-destination=104086365 id="104086365" class="section-title" >2.3. Cell culture and transfection</h3>
<p>All cell lines, Neuro-2a (CCL-131), HeLa (CCL-2), NIH/3T3 (CRL-1658), and HEK293 (CRL-1573)—all from ATCC, were maintained in Dulbecco’s modified Eagle’s Medium supplemented with 10% fetal bovine serum, 2 mM GlutaMAX, 100 U/ml penicillin, and 100 μg/ml streptomycin at 37 °C in a 5% CO2 in air and humidified atmosphere. All cell lines were checked regularly and before the experiments for mycoplasma infection.</p>                            <h3 scrollto-destination=104086367 id="104086367" class="section-title" >2.4. Experiments with pSc1, pSc2 plasmids, and HDR assays</h3>
<p>Cells were seeded at 1.5×10<sup>5</sup> cells/well in the case of HeLa, Neuro-2a (N2a) and HEK293 cells, and at 3×10<sup>5</sup> cells/well density for NIH/3T3 cells, on 6-well plates 1 day prior to transfection. Cells were transfected by 2 µg total DNA/well, combining 1 µg Cas9-gRNA vector and 1 µg GFP vector DNAs (pSc1, pSc2, pHR-PRNP, pHR-ROSA), and using either Turbofect at 3 µl/µg total DNA (in the case of HeLa, N2a, and HEK293 cells) or Lipofectamine 2000 at 4.5 µl/µg total DNA (in the case of NIH/3T3 cells). Transfected cells were passaged at a 1:10 ratio until confluency reached 80–100% and the excess cells were analysed by FACS. The passaging continued until the transient expression disappeared (typically 12–15 days post transfection). Transfection efficiency was determined on the second day after transfection by measuring the fraction of GFP positive cells (the values obtained were considered 100%) and was later used to normalize the corresponding sample data. All transfections were made in triplicates.</p>                            <h3 scrollto-destination=104086369 id="104086369" class="section-title" >2.5. Coupled integration of the GFP and puromycin cassettes</h3>
<p>Before performing the experiments, the lowest antibiotic concentration that kills all non-transfected N2a cells was determined by applying increasing concentrations of antibiotic, up to 3.1 µg/ml puromycin. A concentration of 0.7 µg/ml was the lowest that killed all cells after 2 weeks of selection. This concentration was used further for selection. N2a cells were co-transfected in four replicates with pSc1-puro and pten specific Cas9-gRNA plasmid, using the previously described protocol with the pSc1 plasmid. As a control for random integration, pSc1-puro was co-transfected with a plasmid harbouring an inactive SpCas9 cassette (dCas9). Non-transfected control cells were also used to monitor selection effectiveness. Puromycin selection was started 2 days after transfection and on days 2 and 12 post-transfection the number of GFP positive cells were measured by FACS.</p>                            <h3 scrollto-destination=104086371 id="104086371" class="section-title" >2.6. Experiments with pSc1-DD</h3>
<p>Transfection was performed similarly as above with pSc1 plasmid, except now 1 µg Cas9-gRNA, 300 ng pSc1-DD, and 300 ng mCherry expressing plasmids were used per well for transfection on 6-well plates. Transfection efficiencies were determined 2 days after transfection by measuring the fraction of mCherry positive cells (the values obtained were considered 100%) and was later used to normalize the corresponding sample data. TMP (1 µM) was added to the media 24 h before FACS analysis.</p>                            <h3 scrollto-destination=104086373 id="104086373" class="section-title" >2.7. Experiments with eSpCas9</h3>
<p>A gRNA and mCherry coding plasmid (pmCherry_gRNA) was co-transfected with a wtSpCas9 or eSpCas9 coding plasmid (pX330-Flag-ehSpCas9 or pX330-Flag-wtSpCas9, respectively) and with the pSc1 self-cleaving plasmid into N2a cells. Transfection was performed similarly as with the pSc1 plasmid above, except now 760 ng Cas9 coding, 760 ng pSc1 and 480 ng pmCherry_gRNA plasmid was used. Transfection efficiencies were determined 2 days after transfection by measuring the fraction of GFP positive cells (the values obtained were considered 100%) and was later used to normalize the corresponding sample data.</p>                            <h3 scrollto-destination=104086375 id="104086375" class="section-title" >2.8. Isolation of genomic DNA and genomic PCR</h3>
<p>Genomic DNA from 1×10<sup>6</sup> cells was isolated after stable GFP-cassette integration (typically 12–15 days after transfection) using PureLink™ Genomic DNA Mini Kit (Invitrogen) following the manufacturer’s protocol or by following Gentra DNA purification protocol (Gentra Puregen Handbook, Qiagen). From the isolated genomic DNA PCR was conducted with Q5 polymerase (for PCR primers and conditions, see <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_data_dsx029.xlsx?Expires=2147483647&Signature=tvN8l6Nr13c-J7Z5ueuYtmqjQDRRDKGeMZZ4zP2xfTAnHe5~chW-J73UkVFgA2WKJsBA02QbIMp6AMjSQiHSwMi0vHIVHEbihM53zO2GWQvuc4s2FTp1W9zVQUk9sLiwr5wcXiiABvFYaNZHZUtIk4XHVLMRF7WB-X7LD~F1Kvfd7WpSBIrtxhybwx0b1m2jkT6FrEL5~IBFX0ygmG~VV1Mnqudots2x7TJaZbc6aOeoBKvcy9vW8rzF6Vlr8ElemUAIh~IO~SFhI4KvLxWuzc14LBQoyV6OmOXBQL0M1rsXEwQYXq189aSoITXXGt2Lp8RRmBOJ1IL4zERV9~H8WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>).</p>                            <h3 scrollto-destination=104086377 id="104086377" class="section-title" >2.9. TIDE and T7 endonuclease I assay</h3>
<p>Genomic PCR products were gel excised, purified via NucleoSpin® Gel and PCR Clean-up kit and were Sanger sequenced by Microsynth AG. Indel efficiencies were analysed by TIDE web tool<a reveal-id="dsx029-B33" data-open="dsx029-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> (<a class="link link-uri" href="https://tide.nki.nl/" target="_blank">https://tide.nki.nl/</a>) by comparing Cas9 treated and control samples.</p><p>Samples of 200 ng from the same PCR reactions that were analysed by TIDE were digested with five units of T7 endonuclease I for 2 h at 37 °C and were then analysed by agarose gel electrophoresis. Densitometry assessment of cleaved and uncleaved bands were conducted with ImageJ software and the following formula was used to determine InDel efficiency: InDel = 100*(1 − (1 − (b + c)/(a + b + c))<sup>0.5</sup>), where <em>b</em> and <em>c</em> represent the cleaved products and <em>a</em> represents the uncleaved band.</p>                            <h3 scrollto-destination=104086380 id="104086380" class="section-title" >2.10. Statistics</h3>
<p>Differences between groups were tested for statistical significance in each experiment using one-way ANOVA test with Bonferroni’s post-hoc test (IBM SPSS Statistics v.21).</p>                            <h3 scrollto-destination=104086382 id="104086382" class="section-title" >2.11. Flow cytometry</h3>
<p>Flow cytometry analysis was carried out on an Attune Acoustic Focusing Cytometer (Applied Biosystems by Life Technologies). Attune Cytometric Software was used for data analysis. In all experiments, a total of 10,000 events were acquired and viable single cells were gated based on side and forward light-scatter parameters. The GFP signal was detected using a 488 nm diode laser for excitation and a 530/30 nm filter for emission and the mCherry fluorescent signal was detected using a 488 nm diode laser for excitation and a 640LP filter for emission.</p>                            <h2 scrollto-destination=104086384 id="104086384" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=104086385 id="104086385" class="section-title" >3.1. In-cell cleavage of a GFP-plasmid facilitates monitoring of gRNA efficiencies</h3>
<p>We aimed to compare the DNA double strand break mediating efficiencies of different gRNAs with SpCas9. To this end, we measured the fluorescence intensities associated with GFP expression cassettes captured by the NHEJ repair system at the corresponding sites of the double strand breaks. Here, our main concern was whether NHEJ-mediated integration would sufficiently be efficient to be discernible by fluorescence-detection from the random integration background of the GFP cassette. To test this, first, we co-transfected N2a mouse neuroblastoma cells with a circular GFP expression vector (pEGFP-C1 - referred to as GFP-plasmid) along with the SpCas9 and gRNA expressing vectors, and monitored the integration events by the fluorescence of GFP after transfection. As a control, a nuclease inactive SpCas9 (dCas9) expressing plasmid was employed that allowed to monitor the decay of the fluorescent signal originating from transient expression and facilitated determination of targeted integration as being the signal detected above the background, non-targeted, random integration of the expression cassette.</p><p>Five spacer sequences targeting three genes in the mouse genome, three for <em>Prnp</em>, one for <em>Sprn</em> and one for <em>Piwil4</em> genes, were selected and the corresponding expression constructs with SpCas9 and gRNAs were co-transfected with the GFP-plasmid into N2a cells. Cells displaying GFP fluorescence were monitored by FACS until the signal derived from transient expression has decayed and the fluorescence stabilized. This occurred over a 12–15-day period in all experiments (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>), depending on factors such as transfection efficiency, which could be determined by measuring the GFP fluorescence on the second day after transfection. Given that the level of stabilized fluorescence varies from experiment to experiment, all samples to be compared were tested side by side in these studies. In these experiments, however, after the transient expression decayed, none of the samples could be clearly distinguished from the dCas9 controls representing random integration (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2A</a></span>).</p><p>Since we expected linearized plasmids to be more readily integrated, next, we linearized the GFP-plasmid with a restriction enzyme and tested its integration similarly in N2a cells, and also, with an overlapping set of gRNAs in NIH/3T3 mouse fibroblast cells. Since the linearized DNA is more vulnerable to cellular nucleases, in order to decrease the possibility of digestion of the expression cassette before the targeted integration would take place, more than 1000 bp-long extra sequences were left at both sides of the GFP expression cassette when linearized by the restriction enzyme. The use of a linearized plasmid resulted in higher integrations in both cell lines (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2B and C</a></span>), however, similarly to the previous results using circular plasmid (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2A</a></span>), none of the samples provided signals over twice as large as the level of random integrations. This is in spite of that the majority of the employed gRNAs supported HR mediated targeted integration in N2a cells (unpublished results). These data, in line with the general perception, suggest that the efficiencies of capturing circular or pre-linearized exogenous DNA by NHEJ-repair at the sites of targeted cleavage are not generally sufficient to be clearly detectable above the level of random integration.</p><p>The efficiency of targeted integration of exogenous DNA by NHEJ was reported to be somewhat higher when the donor plasmid was linearized inside the cell by designer nucleases.<a reveal-id="dsx029-B38" data-open="dsx029-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a><sup>,</sup><a reveal-id="dsx029-B44" data-open="dsx029-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsx029-B52" data-open="dsx029-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> To exploit this possibility, we picked two efficient-gRNA expressing plasmids that were available in our laboratory (targeting <em>Ttn</em> and <em>Rbl2</em> genes—referred to as Ttn and Rb, respectively), and cloned each corresponding protospacer into the GFP-plasmid either preceding, or both preceding and following the GFP-cassette (<a reveal-id="dsx029-F1" data-open="dsx029-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). These plasmids and the control (protospacerless) GFP-plasmid were each co-transfected with a vector containing an active SpCas9 and the corresponding gRNA (Rb or Ttn) or a vector containing inactive SpCas9. The numbers of fluorescent cells were counted as previously. The in-cell cleavage of plasmids resulted targeted integrations four to six-fold above the background level of random integration measured with circular, protospacerless GFP-plasmid and active SpCas9 (<a reveal-id="dsx029-F1" data-open="dsx029-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). </p>                        <a id="104086390" scrollto-destination="104086390"></a>
<div id="dsx029-F1" data-id="dsx029-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/m_dsx029f1.png?Expires=2147483647&Signature=0BUwiQLyVMtuhwwXTK5ddFfEbl7NxhXRUx7BXIKh7wODhbilOqAbbcbqfjwdMhbzXXBss9zWXjPU1KI4yWnXVbb89ZzNm8gkOMwmpdskbeA9q-tnSbUDqB4EGeSh9gTI4pqyPv5-OYLiW2m6abmF~~18ayuTPwEm~ZXq2oG6baIhms7qwe7hWNqzsetQEI3QfX-QHx9blC7~ul1oXeer8c-7iQf3xh1QvRaPlJVxp6--pey8Emsl~wE3fXFyzpvvdReqvTCn4LoJgUQS6nalwCvVfNCKTCM-9TheAi2kkqxWWDB~ZJPhr9JgiQGsojIW2tSemsUVWYBSQMPBTeQlMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="in-cell cleavage of the GFP-plasmid facilitates targeted integration. (A) Scheme of the genomic integration of an in-cell cleavable plasmid. The target sequence of the gRNA (magenta) from the plasmid containing the SpCas9 is present both in the genome (magenta target in the grey double helix) and in the GFP-cassette containing plasmid - either preceding or at both ends (the latter case is shown in the figure by two magenta boxes) of the GFP-cassette (green). SpCas9 cleaves both the genome and the GFP-plasmid. During repair of the broken genomic DNA, the opened plasmid will efficiently integrate into the genome at the cleavage site even in the absence of homologous arms. The protospacer on the GFP-plasmid has to be redesigned and re-cloned for every new genomic target in order to achieve opening of the plasmid. (B) in-cell linearization of the circular plasmid increases the efficiency of the genomic integration. Two different genes, Rbl2 and Ttn were chosen and targeted by corresponding gRNAs (Rb and Ttn, respectively), to test the effect of in-cell linearization of the GFP-plasmid. Bars show the percentages of cells harbouring stably integrated GFP-cassette after co-transfection by an expression vector for both SpCas9 and either Rb or Ttn gRNAs and a circular GFP-plasmid that either contains one (Rb 1x, Ttn 1x) or - as on scheme A - two (Rb 2x, Ttn 2x) copies of the respective targets (light grey bars). As controls (dark grey bars), an inactive SpCas9 with the GFP plasmid (dead) and to measure the backgrounds without in-cell cleavage, an active SpCas9 with Rb or Ttn targets combined with the circular plasmids were used (Rb -, Ttn -). The values were compared to their respective circular control with the active nuclease. Bars show the means ± S.D. of percentages measured in n = 3 independent transfections, which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 87 and 91%). ***P&lt;0.001." path-from-xml="dsx029f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086390/dsx029f1.tif" path-from-xml="dsx029f1.tif" target="_blank">View large</a><a section="104086390" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/dsx029f1.png?Expires=2147483647&Signature=uLpOcvKCEKtPZiu92yhjYt5VaTrhqERVVkqJHKSneQup4Pjjl6gPjGCPmN7Md6LRSYBv-ckzbRBmcUjlOfSER~jJJ5cOTKUtB1SvoFpwgujn7-vFS~W2zD8LCk2JVp~LyAlILlXe-35yUJjivJh1i7cT4gszcubI5iRVBkqdEqdsM8Eq30Vel8XwtCyQpl28B2WIKxoYsxqzPdZYmompYFGm0P7~90hHoVSb-TuYk~Dihb6RJUORaCU3heWQEm8-4Qu~E4sr~XotZB7kmemYMuToVk5DBcNS4yF4UPNK3irGpc6ftEvnDVzC6X6djXPCCsgP73PmxByaf0BzxwxZXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086390&ar=3904549&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx029f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>in-cell cleavage of the GFP-plasmid facilitates targeted integration. (A) Scheme of the genomic integration of an in-cell cleavable plasmid. The target sequence of the gRNA (magenta) from the plasmid containing the SpCas9 is present both in the genome (magenta target in the grey double helix) and in the GFP-cassette containing plasmid - either preceding or at both ends (the latter case is shown in the figure by two magenta boxes) of the GFP-cassette (green). SpCas9 cleaves both the genome and the GFP-plasmid. During repair of the broken genomic DNA, the opened plasmid will efficiently integrate into the genome at the cleavage site even in the absence of homologous arms. The protospacer on the GFP-plasmid has to be redesigned and re-cloned for every new genomic target in order to achieve opening of the plasmid. (B) in-cell linearization of the circular plasmid increases the efficiency of the genomic integration. Two different genes, <em>Rbl2</em> and <em>Ttn</em> were chosen and targeted by corresponding gRNAs (Rb and Ttn, respectively), to test the effect of in-cell linearization of the GFP-plasmid. Bars show the percentages of cells harbouring stably integrated GFP-cassette after co-transfection by an expression vector for both SpCas9 and either Rb or Ttn gRNAs and a circular GFP-plasmid that either contains one (Rb 1x, Ttn 1x) or - as on scheme A - two (Rb 2x, Ttn 2x) copies of the respective targets (light grey bars). As controls (dark grey bars), an inactive SpCas9 with the GFP plasmid (dead) and to measure the backgrounds without in-cell cleavage, an active SpCas9 with Rb or Ttn targets combined with the circular plasmids were used (Rb -, Ttn -). The values were compared to their respective circular control with the active nuclease. Bars show the means ± S.D. of percentages measured in <em>n</em> = 3 independent transfections, which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 87 and 91%). ***<em>P</em>&lt;0.001.</p></div></div><div content-id="dsx029-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/m_dsx029f1.png?Expires=2147483647&Signature=0BUwiQLyVMtuhwwXTK5ddFfEbl7NxhXRUx7BXIKh7wODhbilOqAbbcbqfjwdMhbzXXBss9zWXjPU1KI4yWnXVbb89ZzNm8gkOMwmpdskbeA9q-tnSbUDqB4EGeSh9gTI4pqyPv5-OYLiW2m6abmF~~18ayuTPwEm~ZXq2oG6baIhms7qwe7hWNqzsetQEI3QfX-QHx9blC7~ul1oXeer8c-7iQf3xh1QvRaPlJVxp6--pey8Emsl~wE3fXFyzpvvdReqvTCn4LoJgUQS6nalwCvVfNCKTCM-9TheAi2kkqxWWDB~ZJPhr9JgiQGsojIW2tSemsUVWYBSQMPBTeQlMQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="in-cell cleavage of the GFP-plasmid facilitates targeted integration. (A) Scheme of the genomic integration of an in-cell cleavable plasmid. The target sequence of the gRNA (magenta) from the plasmid containing the SpCas9 is present both in the genome (magenta target in the grey double helix) and in the GFP-cassette containing plasmid - either preceding or at both ends (the latter case is shown in the figure by two magenta boxes) of the GFP-cassette (green). SpCas9 cleaves both the genome and the GFP-plasmid. During repair of the broken genomic DNA, the opened plasmid will efficiently integrate into the genome at the cleavage site even in the absence of homologous arms. The protospacer on the GFP-plasmid has to be redesigned and re-cloned for every new genomic target in order to achieve opening of the plasmid. (B) in-cell linearization of the circular plasmid increases the efficiency of the genomic integration. Two different genes, Rbl2 and Ttn were chosen and targeted by corresponding gRNAs (Rb and Ttn, respectively), to test the effect of in-cell linearization of the GFP-plasmid. Bars show the percentages of cells harbouring stably integrated GFP-cassette after co-transfection by an expression vector for both SpCas9 and either Rb or Ttn gRNAs and a circular GFP-plasmid that either contains one (Rb 1x, Ttn 1x) or - as on scheme A - two (Rb 2x, Ttn 2x) copies of the respective targets (light grey bars). As controls (dark grey bars), an inactive SpCas9 with the GFP plasmid (dead) and to measure the backgrounds without in-cell cleavage, an active SpCas9 with Rb or Ttn targets combined with the circular plasmids were used (Rb -, Ttn -). The values were compared to their respective circular control with the active nuclease. Bars show the means ± S.D. of percentages measured in n = 3 independent transfections, which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 87 and 91%). ***P&lt;0.001." path-from-xml="dsx029f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086390/dsx029f1.tif" path-from-xml="dsx029f1.tif" target="_blank">View large</a><a section="104086390" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/dsx029f1.png?Expires=2147483647&Signature=uLpOcvKCEKtPZiu92yhjYt5VaTrhqERVVkqJHKSneQup4Pjjl6gPjGCPmN7Md6LRSYBv-ckzbRBmcUjlOfSER~jJJ5cOTKUtB1SvoFpwgujn7-vFS~W2zD8LCk2JVp~LyAlILlXe-35yUJjivJh1i7cT4gszcubI5iRVBkqdEqdsM8Eq30Vel8XwtCyQpl28B2WIKxoYsxqzPdZYmompYFGm0P7~90hHoVSb-TuYk~Dihb6RJUORaCU3heWQEm8-4Qu~E4sr~XotZB7kmemYMuToVk5DBcNS4yF4UPNK3irGpc6ftEvnDVzC6X6djXPCCsgP73PmxByaf0BzxwxZXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086390&ar=3904549&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx029f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>in-cell cleavage of the GFP-plasmid facilitates targeted integration. (A) Scheme of the genomic integration of an in-cell cleavable plasmid. The target sequence of the gRNA (magenta) from the plasmid containing the SpCas9 is present both in the genome (magenta target in the grey double helix) and in the GFP-cassette containing plasmid - either preceding or at both ends (the latter case is shown in the figure by two magenta boxes) of the GFP-cassette (green). SpCas9 cleaves both the genome and the GFP-plasmid. During repair of the broken genomic DNA, the opened plasmid will efficiently integrate into the genome at the cleavage site even in the absence of homologous arms. The protospacer on the GFP-plasmid has to be redesigned and re-cloned for every new genomic target in order to achieve opening of the plasmid. (B) in-cell linearization of the circular plasmid increases the efficiency of the genomic integration. Two different genes, <em>Rbl2</em> and <em>Ttn</em> were chosen and targeted by corresponding gRNAs (Rb and Ttn, respectively), to test the effect of in-cell linearization of the GFP-plasmid. Bars show the percentages of cells harbouring stably integrated GFP-cassette after co-transfection by an expression vector for both SpCas9 and either Rb or Ttn gRNAs and a circular GFP-plasmid that either contains one (Rb 1x, Ttn 1x) or - as on scheme A - two (Rb 2x, Ttn 2x) copies of the respective targets (light grey bars). As controls (dark grey bars), an inactive SpCas9 with the GFP plasmid (dead) and to measure the backgrounds without in-cell cleavage, an active SpCas9 with Rb or Ttn targets combined with the circular plasmids were used (Rb -, Ttn -). The values were compared to their respective circular control with the active nuclease. Bars show the means ± S.D. of percentages measured in <em>n</em> = 3 independent transfections, which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 87 and 91%). ***<em>P</em>&lt;0.001.</p></div></div><p>The difference was less than two-fold between the two controls, when cleaving the genome without in-cell linearization and the random integration of a circular GFP-plasmid (<a reveal-id="dsx029-F1" data-open="dsx029-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>), similarly as we found earlier using different spacers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2A</a></span>). Cleaving at both sides of the GFP-cassette may lead to more in-cell linearized plasmids, and so it may be expected to result in higher integrations. However, we detected no increased fluorescence with the plasmids that contain two targets thus, for the further experiments we cleaved only at one site of the plasmids.</p>                            <h3 scrollto-destination=104086392 id="104086392" class="section-title" >3.2. Construction of self-cleaving plasmids with universal target-spacer pairs</h3>
<p>Although the above results are encouraging and indicate that the integration of in-cell linearized plasmids without homology arms are effective in these cells, such a design does not allow the measurement of the actual integration to the SpCas9 cleaved genomic sites. This is because the same gRNA mediates the cleavage of the genome and of the plasmid, and thus, the two events are coupled and it is not possible to assess the background originating only from the random integration of the in-cell linearized plasmid independently of the genome cleavage.</p><p>Furthermore, the efficiency of targeted integration when using the same target sequence both in the genome and in the GFP-plasmid (<a reveal-id="dsx029-F1" data-open="dsx029-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>) depends on both, possibly differing, effectiveness of cleaving the target in its genomic context and out of its genomic context on the GFP-plasmid. To allow a more comparable and adequate estimation of cleavage activities for various gRNAs on their targets in their genomic context, the use of one common GFP-plasmid that contains the same target sequence ensuring identical in-cell cleavage for all gRNAs to be tested, and which is also independent of the tested targets, would be the most adequate. In addition, such an approach would eliminate the need for the laborious cloning of each corresponding protospacer to the GFP-plasmid for each spacer to be tested. The practicality of such generally applicable plasmid would be further extended if the corresponding common gRNA for the in-cell cleavage of the plasmid was also placed into the GFP-plasmid. Further we refer to plasmids constructed in this way as ‘self-cleaving’, since such plasmids, expressing the gRNA with its own target, are programmed to initiate their own cleavage (<a reveal-id="dsx029-F2" data-open="dsx029-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). </p>                        <a id="104086395" scrollto-destination="104086395"></a>
<div id="dsx029-F2" data-id="dsx029-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/m_dsx029f2.png?Expires=2147483647&Signature=iZCS1AlMx8TzcK~7XKj9ECI2lcDGqnbdsYaCgKz8yKVQsvILw6dXxljE35gnwRJqpJ2oA7HvQboD4XRrPPt1lY9~qpkmtvhqvWUg7020~xyt2WhrHoafUpMeiEzJXbG2fOiS1yGumAOMsU-eW9spSObm~wecCOY7rLeqAGo45ZY-Fiv0xD8qHMduSj86YztCD6IFRXbcy1f-vRjLIey6pewogtzITxogHWa7Jzl6WNI~zvkS6FLW67kReST4FxGDHrQMgqUasfuRgtNFTabI-VW1wk4XiFwLwTjQ6hB4Vh-u3kkiRY0oknFwFxc~rmnfPFe9VgSFaE5tIuW~IZ6GtA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Scheme of the genomic integration of a self-cleaving plasmid. The target sequence of the gRNA (magenta) expressed from the SpCas9 plasmid is present only in the genome (magenta target in the grey double helix). The circular plasmid that contains a GFP-box (green) harbours a genomic-targetless gRNA (TL gRNA) and its target sequence (blue boxes). During repair of the broken genomic DNA, the opened plasmid will efficiently integrate into the genome at the cleavage site even in the absence of homologous arms. Such construction of the GFP-plasmid does not require retailoring the protospacer for every new genomic target and can be universally used. (B) The efficiency of targeted integration when using self-cleaving GFP-plasmid in N2a cells. The percentages of the cells harbouring a stably integrated GFP-cassette are shown. The Rbl2 or Ttn genes were targeted by the corresponding gRNAs, Rb or Ttn, respectively. Cells were co-transfected by the expression vector for both wild type SpCas9 and a respective gRNA, and either of the self-cleaving circular GFP-plasmids pSc1 or pSc2 or the circular GFP-plasmid containing the Rb or Ttn protospacer (Rb-InCell, Ttn-InCell) or without containing any target sequence (Rb or Ttn, Circ). To measure the random integration background of the self-cleaving plasmids, they were co-transfected with a vector expressing active SpCas9 and the corresponding TL gRNAs; thus, the genome is not targeted, but the self-cleaving plasmids are linearized (TL1, TL2). As an additional negative control, an inactive SpCas9 was used along with either of the two self-cleaving plasmids (dead). Bars show the means ± S.D. of percentages measured in n = 3 independent transfections, which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 94 and 96%)." path-from-xml="dsx029f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086395/dsx029f2.tif" path-from-xml="dsx029f2.tif" target="_blank">View large</a><a section="104086395" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/dsx029f2.png?Expires=2147483647&Signature=R9kI1hN8JGHrEuxhrX4~pKTKZAGmUqygJ~u2UiY7V7zUx4ypIc2idmOpxu8wXj-zxHjaCX5ERQDwPWx4tUdLh0V2WhAvJnlD4ZysEdUImnd1ZHZpKYT2lKQuurQuVar2HigSDWr6zBt1MTJue2qvOA0iF561HS5MqPTk1DrLOrO3XodMK51ZJ4QF73Yx4kJwT9W-WEvw3QcsTO3WFR5qvtJDhwIZq2cA61ixz8Z9hrBATdAAx-u6sDZ1eRmRpI8PywGYu-V2akhKULBHK0vj58255rZP27sDJpen1MtZgutkb8n2OiQCmZzIS9Uzz6qukYWbC7~ryZR1fhPEnVWbkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086395&ar=3904549&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx029f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Scheme of the genomic integration of a self-cleaving plasmid. The target sequence of the gRNA (magenta) expressed from the SpCas9 plasmid is present only in the genome (magenta target in the grey double helix). The circular plasmid that contains a GFP-box (green) harbours a genomic-targetless gRNA (TL gRNA) and its target sequence (blue boxes). During repair of the broken genomic DNA, the opened plasmid will efficiently integrate into the genome at the cleavage site even in the absence of homologous arms. Such construction of the GFP-plasmid does not require retailoring the protospacer for every new genomic target and can be universally used. (B) The efficiency of targeted integration when using self-cleaving GFP-plasmid in N2a cells. The percentages of the cells harbouring a stably integrated GFP-cassette are shown. The <em>Rbl2</em> or <em>Ttn</em> genes were targeted by the corresponding gRNAs, Rb or Ttn, respectively. Cells were co-transfected by the expression vector for both wild type SpCas9 and a respective gRNA, and either of the self-cleaving circular GFP-plasmids pSc1 or pSc2 or the circular GFP-plasmid containing the Rb or Ttn protospacer (Rb-InCell, Ttn-InCell) or without containing any target sequence (Rb or Ttn, Circ). To measure the random integration background of the self-cleaving plasmids, they were co-transfected with a vector expressing active SpCas9 and the corresponding TL gRNAs; thus, the genome is not targeted, but the self-cleaving plasmids are linearized (TL1, TL2). As an additional negative control, an inactive SpCas9 was used along with either of the two self-cleaving plasmids (dead). Bars show the means ± S.D. of percentages measured in <em>n</em> = 3 independent transfections, which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 94 and 96%).</p></div></div><div content-id="dsx029-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/m_dsx029f2.png?Expires=2147483647&Signature=iZCS1AlMx8TzcK~7XKj9ECI2lcDGqnbdsYaCgKz8yKVQsvILw6dXxljE35gnwRJqpJ2oA7HvQboD4XRrPPt1lY9~qpkmtvhqvWUg7020~xyt2WhrHoafUpMeiEzJXbG2fOiS1yGumAOMsU-eW9spSObm~wecCOY7rLeqAGo45ZY-Fiv0xD8qHMduSj86YztCD6IFRXbcy1f-vRjLIey6pewogtzITxogHWa7Jzl6WNI~zvkS6FLW67kReST4FxGDHrQMgqUasfuRgtNFTabI-VW1wk4XiFwLwTjQ6hB4Vh-u3kkiRY0oknFwFxc~rmnfPFe9VgSFaE5tIuW~IZ6GtA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Scheme of the genomic integration of a self-cleaving plasmid. The target sequence of the gRNA (magenta) expressed from the SpCas9 plasmid is present only in the genome (magenta target in the grey double helix). The circular plasmid that contains a GFP-box (green) harbours a genomic-targetless gRNA (TL gRNA) and its target sequence (blue boxes). During repair of the broken genomic DNA, the opened plasmid will efficiently integrate into the genome at the cleavage site even in the absence of homologous arms. Such construction of the GFP-plasmid does not require retailoring the protospacer for every new genomic target and can be universally used. (B) The efficiency of targeted integration when using self-cleaving GFP-plasmid in N2a cells. The percentages of the cells harbouring a stably integrated GFP-cassette are shown. The Rbl2 or Ttn genes were targeted by the corresponding gRNAs, Rb or Ttn, respectively. Cells were co-transfected by the expression vector for both wild type SpCas9 and a respective gRNA, and either of the self-cleaving circular GFP-plasmids pSc1 or pSc2 or the circular GFP-plasmid containing the Rb or Ttn protospacer (Rb-InCell, Ttn-InCell) or without containing any target sequence (Rb or Ttn, Circ). To measure the random integration background of the self-cleaving plasmids, they were co-transfected with a vector expressing active SpCas9 and the corresponding TL gRNAs; thus, the genome is not targeted, but the self-cleaving plasmids are linearized (TL1, TL2). As an additional negative control, an inactive SpCas9 was used along with either of the two self-cleaving plasmids (dead). Bars show the means ± S.D. of percentages measured in n = 3 independent transfections, which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 94 and 96%)." path-from-xml="dsx029f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086395/dsx029f2.tif" path-from-xml="dsx029f2.tif" target="_blank">View large</a><a section="104086395" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/dsx029f2.png?Expires=2147483647&Signature=R9kI1hN8JGHrEuxhrX4~pKTKZAGmUqygJ~u2UiY7V7zUx4ypIc2idmOpxu8wXj-zxHjaCX5ERQDwPWx4tUdLh0V2WhAvJnlD4ZysEdUImnd1ZHZpKYT2lKQuurQuVar2HigSDWr6zBt1MTJue2qvOA0iF561HS5MqPTk1DrLOrO3XodMK51ZJ4QF73Yx4kJwT9W-WEvw3QcsTO3WFR5qvtJDhwIZq2cA61ixz8Z9hrBATdAAx-u6sDZ1eRmRpI8PywGYu-V2akhKULBHK0vj58255rZP27sDJpen1MtZgutkb8n2OiQCmZzIS9Uzz6qukYWbC7~ryZR1fhPEnVWbkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086395&ar=3904549&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx029f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Scheme of the genomic integration of a self-cleaving plasmid. The target sequence of the gRNA (magenta) expressed from the SpCas9 plasmid is present only in the genome (magenta target in the grey double helix). The circular plasmid that contains a GFP-box (green) harbours a genomic-targetless gRNA (TL gRNA) and its target sequence (blue boxes). During repair of the broken genomic DNA, the opened plasmid will efficiently integrate into the genome at the cleavage site even in the absence of homologous arms. Such construction of the GFP-plasmid does not require retailoring the protospacer for every new genomic target and can be universally used. (B) The efficiency of targeted integration when using self-cleaving GFP-plasmid in N2a cells. The percentages of the cells harbouring a stably integrated GFP-cassette are shown. The <em>Rbl2</em> or <em>Ttn</em> genes were targeted by the corresponding gRNAs, Rb or Ttn, respectively. Cells were co-transfected by the expression vector for both wild type SpCas9 and a respective gRNA, and either of the self-cleaving circular GFP-plasmids pSc1 or pSc2 or the circular GFP-plasmid containing the Rb or Ttn protospacer (Rb-InCell, Ttn-InCell) or without containing any target sequence (Rb or Ttn, Circ). To measure the random integration background of the self-cleaving plasmids, they were co-transfected with a vector expressing active SpCas9 and the corresponding TL gRNAs; thus, the genome is not targeted, but the self-cleaving plasmids are linearized (TL1, TL2). As an additional negative control, an inactive SpCas9 was used along with either of the two self-cleaving plasmids (dead). Bars show the means ± S.D. of percentages measured in <em>n</em> = 3 independent transfections, which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 94 and 96%).</p></div></div><p>To prepare such effective GFP-plasmids for general use, we computer-generated a pool of 30,000 random spacer sequences and picked the ones that have the least predicted off-targets in the mouse genome as assessed by e-CRISP Cas9 construct designer<a reveal-id="dsx029-B53" data-open="dsx029-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> and had the best predicted efficiency by sgRNA Designer.<a reveal-id="dsx029-B16" data-open="dsx029-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsx029-B21" data-open="dsx029-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> From these, two candidates were picked that according to the CasFinder<a reveal-id="dsx029-B54" data-open="dsx029-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> have no targets in the human genome and in 11 frequently employed model organisms (mouse, rat, cat, dog, pig, cow, chicken, zebrafish, <em>C. elegans</em>, <em>Drosophila</em>, and <em>S. cerevisiae</em>) thus, reserving the potential to be used more generally for self-cleaving GFP-plasmids in different cell lines/organisms (although, we do not explore these possibilities here). We refer to these as TL (<strong><span class="underline">t</span></strong>arget<strong><span class="underline">l</span></strong>ess) spacers, TL1 and TL2; these two spacer sequences were used to generate two ‘self-cleaving’ GFP-plasmids (referred to as pSc1 and pSc2, respectively - Addgene #80436, #80437) containing both the TL spacer/gRNA and the corresponding protospacer on one vector (<a reveal-id="dsx029-F2" data-open="dsx029-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>).</p><p>We tested the targeted integration of the two self-cleaving plasmids in combination with either the Ttn or Rb-gRNA in comparison with the Ttn and Rb cleaved in-cell linearized plasmid (<a reveal-id="dsx029-F2" data-open="dsx029-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>, indicated as Rb-, Ttn-InCell), as earlier. As a control, TL-gRNA bearing plasmids were co-transfected with either of the two self-cleaving plasmids; thus, the GFP-plasmid is linearized in-cell without the genome being targeted. These experiments demonstrate that the self-cleaving plasmids achieve comparable integrations with the two in-cell cleaved GFP-plasmids in case of both genome-targeting (Ttn or Rb) gRNAs tested. This suggests that both gRNAs containing TL spacers (TL-gRNAs) support as efficient plasmid DNA cleavage as the Rb or Ttn spacers do. Thus, these self-cleaving plasmids seem to facilitate efficient integration, while providing a more adequate assessment of efficiencies of different gRNAs.</p><p>The two self-cleaving GFP-plasmids pSc1 and pSc2 were further used to test 11 spacers, targeting 7 different mouse genes [<em>Pten</em>,<a reveal-id="dsx029-B55" data-open="dsx029-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><em>Rbl2</em>, <em>Tp53</em>,<a reveal-id="dsx029-B55" data-open="dsx029-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><em>Prnp</em>, <em>Sprn</em>, <em>Piwil2</em>, and <em>Piwil4</em> (<a reveal-id="dsx029-F3" data-open="dsx029-F3" class="link link-reveal link-table xref-fig">Fig. 3A and B</a>)]. Targeted integration is assessed here as the fluorescence <em>above</em> the background level that derives from the random integration of the in-cell cleaved self-cleaving plasmid. For this control sample the gRNA, which is identical to the corresponding TL-gRNA of the self-cleaving plasmid is placed to the SpCas9 plasmid; thus, the GFP-plasmid is linearized in-cell without the genome being targeted. The random integration of these in-cell-cleaved GFP cassettes is usually about 4–5%, which is higher than the random integration of a circular (1–2%) or even a linear (2–3.5%) plasmid. The nearly 2% increment likely come from an increased propensity of the in-cell cleaved plasmid for random integration, although a contribution also from off-target cleavage activity of the TL guides cannot be ruled out. Nevertheless, these values—although, may vary among cell types—are sufficiently low to allow the detection of targeted integration above this background level fluorescence. </p>                        <a id="104086399" scrollto-destination="104086399"></a>
<div id="dsx029-F3" data-id="dsx029-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/m_dsx029f3.png?Expires=2147483647&Signature=WvFrNx-YU5awzzl5Pr89wSFz7tRNbKT2wKadSfjQpFY9iFkfISdv5qQhObEEhIkp-x8kAJJ9Z2DFABHBfNaeLkndwC8CVWU6vlVvBYZkQ2sEAkUO~wpS2fR5uMo77DYJnMacTTnSuGxCyrG7VEhWxDs4JNJvSyL6C~OC2nCdncSW8mad1w0UaTNOyqU2DWi9KIRrfTvojXldoGYNjn75Oe2S55a4M3~5kqcSwg5F5KB7upTfIBAnVJlXEM0h3zga9pftnJm1fJU4wlK-xNKmgWIfbFA0rnMx74cIRT9yNhKEY0BSnZEnURljghCVoU-MZ2WGNP5Rx31bTGCrpbNybg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The efficiency of different spacers to mediate targeted integration when used with self-cleaving GFP-plasmids in N2a cells. The efficiencies of different spacers in targeted integration using the pSc1 (A) or pSc2 (B) self-cleaving GFP-plasmid are assessed. The percentages of the cells harbouring stably integrated GFP-cassette were measured for 11 different spacers (pten, Rb, p53, prp1-5, sho, piwi1-2) targeting seven genes (Pten, Rbl2, Tp53, Prnp, Sprn, Piwil4, Piwil2) as indicated by the light grey bars, when co-transfecting with either the circular pSc1 (A) or pSc2 (B) self-cleaving GFP-plasmids. To measure the random integration background of the self-cleaving plasmids, they were co-transfected with a vector expressing active SpCas9 and the corresponding TL gRNA; thus, the genome is not targeted, but the self-cleaving plasmids are linearized (genome not targeted, Sc1 in-cell cleaved; A and B, indicated by the dashed lines). Bars show the means ± S.D. of percentages measured in n = 3 independent transfections, which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 94 and 98%). Values obtained are compared to the control where the linearized self-cleaving GFP-plasmid (genome not targeted, pSc1 in-cell cleaved) was used. ***P&lt;0.001." path-from-xml="dsx029f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086399/dsx029f3.tif" path-from-xml="dsx029f3.tif" target="_blank">View large</a><a section="104086399" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/dsx029f3.png?Expires=2147483647&Signature=xlOd6qeqIGJ074jCXxDHZ2rifJAJjCzTp5cVmSG~Brr-F9e2wdv-2vi-1Bntyl6-0AjehqiuIbEJCoTsjdmQXL~p53MFLVIBkP1A1-twQvH0lnmrZTLhH6x9YNDvSdzyrgfUpzjt2hR9JxO-VmFQ~8KJ8fqBg5IDcTukj7l~tV0zAzTxNBfvXGneeOLBViLPRznnFifJjO3b8es9kvbHO276KmhyAAiXKfuhbB80HxZ2j2Zn1HFpqDFFqcavrpkjzEkhOxci7KDAtBw4NqwkZAPkSid04RKmd0XuayhDuPdlKlyT2Ry-~njWMIwaqztSO8dZnmb0AkTfAtyfH8z9Yg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086399&ar=3904549&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx029f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The efficiency of different spacers to mediate targeted integration when used with self-cleaving GFP-plasmids in N2a cells. The efficiencies of different spacers in targeted integration using the pSc1 (A) or pSc2 (B) self-cleaving GFP-plasmid are assessed. The percentages of the cells harbouring stably integrated GFP-cassette were measured for 11 different spacers (pten, Rb, p53, prp1-5, sho, piwi1-2) targeting seven genes (<em>Pten</em>, <em>Rbl2</em>, <em>Tp53</em>, <em>Prnp</em>, <em>Sprn</em>, <em>Piwil4</em>, <em>Piwil2</em>) as indicated by the light grey bars, when co-transfecting with either the circular pSc1 (A) or pSc2 (B) self-cleaving GFP-plasmids. To measure the random integration background of the self-cleaving plasmids, they were co-transfected with a vector expressing active SpCas9 and the corresponding TL gRNA; thus, the genome is not targeted, but the self-cleaving plasmids are linearized (genome not targeted, Sc1 in-cell cleaved; A and B, indicated by the dashed lines). Bars show the means ± S.D. of percentages measured in <em>n</em> = 3 independent transfections, which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 94 and 98%). Values obtained are compared to the control where the linearized self-cleaving GFP-plasmid (genome not targeted, pSc1 in-cell cleaved) was used. ***<em>P</em>&lt;0.001.</p></div></div><div content-id="dsx029-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/m_dsx029f3.png?Expires=2147483647&Signature=WvFrNx-YU5awzzl5Pr89wSFz7tRNbKT2wKadSfjQpFY9iFkfISdv5qQhObEEhIkp-x8kAJJ9Z2DFABHBfNaeLkndwC8CVWU6vlVvBYZkQ2sEAkUO~wpS2fR5uMo77DYJnMacTTnSuGxCyrG7VEhWxDs4JNJvSyL6C~OC2nCdncSW8mad1w0UaTNOyqU2DWi9KIRrfTvojXldoGYNjn75Oe2S55a4M3~5kqcSwg5F5KB7upTfIBAnVJlXEM0h3zga9pftnJm1fJU4wlK-xNKmgWIfbFA0rnMx74cIRT9yNhKEY0BSnZEnURljghCVoU-MZ2WGNP5Rx31bTGCrpbNybg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The efficiency of different spacers to mediate targeted integration when used with self-cleaving GFP-plasmids in N2a cells. The efficiencies of different spacers in targeted integration using the pSc1 (A) or pSc2 (B) self-cleaving GFP-plasmid are assessed. The percentages of the cells harbouring stably integrated GFP-cassette were measured for 11 different spacers (pten, Rb, p53, prp1-5, sho, piwi1-2) targeting seven genes (Pten, Rbl2, Tp53, Prnp, Sprn, Piwil4, Piwil2) as indicated by the light grey bars, when co-transfecting with either the circular pSc1 (A) or pSc2 (B) self-cleaving GFP-plasmids. To measure the random integration background of the self-cleaving plasmids, they were co-transfected with a vector expressing active SpCas9 and the corresponding TL gRNA; thus, the genome is not targeted, but the self-cleaving plasmids are linearized (genome not targeted, Sc1 in-cell cleaved; A and B, indicated by the dashed lines). Bars show the means ± S.D. of percentages measured in n = 3 independent transfections, which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 94 and 98%). Values obtained are compared to the control where the linearized self-cleaving GFP-plasmid (genome not targeted, pSc1 in-cell cleaved) was used. ***P&lt;0.001." path-from-xml="dsx029f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086399/dsx029f3.tif" path-from-xml="dsx029f3.tif" target="_blank">View large</a><a section="104086399" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/dsx029f3.png?Expires=2147483647&Signature=xlOd6qeqIGJ074jCXxDHZ2rifJAJjCzTp5cVmSG~Brr-F9e2wdv-2vi-1Bntyl6-0AjehqiuIbEJCoTsjdmQXL~p53MFLVIBkP1A1-twQvH0lnmrZTLhH6x9YNDvSdzyrgfUpzjt2hR9JxO-VmFQ~8KJ8fqBg5IDcTukj7l~tV0zAzTxNBfvXGneeOLBViLPRznnFifJjO3b8es9kvbHO276KmhyAAiXKfuhbB80HxZ2j2Zn1HFpqDFFqcavrpkjzEkhOxci7KDAtBw4NqwkZAPkSid04RKmd0XuayhDuPdlKlyT2Ry-~njWMIwaqztSO8dZnmb0AkTfAtyfH8z9Yg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086399&ar=3904549&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx029f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The efficiency of different spacers to mediate targeted integration when used with self-cleaving GFP-plasmids in N2a cells. The efficiencies of different spacers in targeted integration using the pSc1 (A) or pSc2 (B) self-cleaving GFP-plasmid are assessed. The percentages of the cells harbouring stably integrated GFP-cassette were measured for 11 different spacers (pten, Rb, p53, prp1-5, sho, piwi1-2) targeting seven genes (<em>Pten</em>, <em>Rbl2</em>, <em>Tp53</em>, <em>Prnp</em>, <em>Sprn</em>, <em>Piwil4</em>, <em>Piwil2</em>) as indicated by the light grey bars, when co-transfecting with either the circular pSc1 (A) or pSc2 (B) self-cleaving GFP-plasmids. To measure the random integration background of the self-cleaving plasmids, they were co-transfected with a vector expressing active SpCas9 and the corresponding TL gRNA; thus, the genome is not targeted, but the self-cleaving plasmids are linearized (genome not targeted, Sc1 in-cell cleaved; A and B, indicated by the dashed lines). Bars show the means ± S.D. of percentages measured in <em>n</em> = 3 independent transfections, which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 94 and 98%). Values obtained are compared to the control where the linearized self-cleaving GFP-plasmid (genome not targeted, pSc1 in-cell cleaved) was used. ***<em>P</em>&lt;0.001.</p></div></div><p>Several spacers gave rise to targeted integration (above the random levels of 4%) in combination with either of the two self-cleaving plasmids, pSc1, or pSc2 (<a reveal-id="dsx029-F3" data-open="dsx029-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a> or <a reveal-id="dsx029-B3" data-open="dsx029-B3" class="link link-ref link-reveal xref-bibr">3</a>B, respectively). Experiments with the least effective spacer piwi2, (indicated as ‘piwi2’, <a reveal-id="dsx029-F3" data-open="dsx029-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>) resulted in no clearly detectable targeted integrations that would exceed random integration of an in-cell cleaved plasmid. By contrast, the pten spacer (indicated as ‘pten’, <a reveal-id="dsx029-F3" data-open="dsx029-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>) was by far the most effective with both self-cleaving plasmids mediating targeted integration of around 25%. The two self-cleaving GFP-plasmids supported targeted integrations in a similar fashion in combination with the 11 spacers. To test the presence of targeted integrations, we performed PCR using one primer complementary to the end of the plasmid and another one complementary to the genomic locus, on DNA samples purified from the stably transfected cell populations. Testing on the <em>Pten</em> locus agarose gel electrophoresis analysis revealed that the plasmids were incorporated at both orientations to the target site (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Direct sequencing of the PCR products in case of integration at a different locus (<em>Tp53</em>), demonstrated that a considerable fraction of the integrations took place by error-free NHEJ repair (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>) as it had also been reported earlier for blunt ended NHEJ-cloning.<a reveal-id="dsx029-B38" data-open="dsx029-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a><sup>,</sup><a reveal-id="dsx029-B46" data-open="dsx029-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a></p><p>In another experiment, additional seven gRNAs (targeting the mouse <em>Vim</em> and <em>Dcun1d2</em> genes) were tested along with those targeting <em>Pten</em>, <em>Rbl2</em>, and <em>Ttn</em>, with both the wild type and the enhanced SpCas9 (eSpCas9) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). While for Rb and Ttn both nucleases showed comparable efficiencies and for four other gRNAs (all that target <em>Dcun1d2</em> and with vim1 spacer) neither of the nucleases showed efficient cleavage compared to their controls, with another four wtSpCas9 produced higher rates of targeted integrations while eSpCas9 showed low or no activity, which most probably is attributable in part to the higher specificity of eSpCas9.</p>                            <h3 scrollto-destination=104086402 id="104086402" class="section-title" >3.3. The universal self-cleaving GFP-plasmid approach compares to an HR-based fluorescence reporter method and it is also effective in NIH/3T3, HeLa, and HEK293 cells</h3>
<p>We compared the relative efficiencies of four prion protein gene-targeting spacers to result either HR or NHEJ mediated targeted integration of a GFP-cassette flanked by either 1 kb-long homology arms or no homologous sequences, respectively. The latter is present in the pSc1 plasmid (<a reveal-id="dsx029-F4" data-open="dsx029-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). We selected genomic targets whose cleavage sites are situated within a few base pairs in order to minimize the effect of the distances between cleavage sites of SpCas9 and the beginning of the homologous region, on the efficiency of HR-mediated targeted integration. Two of the cleavage sites (prp2, prp3) were situated in a few nucleotide distances from the end of the upstream and two other (prp4, prp5) from the beginning of the downstream HR arms, respectively. Both HR and NHEJ mediated integrations indicated that both target pairs can be efficiently cleaved by SpCas9. Interestingly, in case of the upstream site gRNAs (prp2 and -3) prp2 seems to be more effective than prp3 when assessed by NHEJ-mediated integration that is not apparent in the HR-mediated assay. </p>                        <a id="104086404" scrollto-destination="104086404"></a>
<div id="dsx029-F4" data-id="dsx029-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/m_dsx029f4.png?Expires=2147483647&Signature=Rb4-EJYtcJN6WdAxih08GXHb0ioSljpHqvZyuhXA7hjiqM3gYEAlS6-w2vw7h0qGpDXLjTOzoOLizXFeD1tYxnmmudV9xDN~eDIC-avvcwkP2fkF4HvtOHz5rHUDCeqftdwLMDcGFvRS1FTDVEoDadmAvAg4z1j14HovW2skSc~cUFS3Bp38CLpncW2K-~NKxSM~xtQfOsutwOsdlUSnKBRyoW94Q~9jvuCaYzHYHb~sLNHr~XMLYrXOLZO1MUqplb6RV-ozyX0-pjwEMayXoUYvujCwn9VDIhGeqjhRkLy5yzi8LgUugH6g0ZRHYw2Z8eeSqe0XDD8LGn3eHy85GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Efficiency of integration of the self-cleaving plasmid to the genome is comparable to that of a HR-mediated integration. The percentages of the cells harbouring a stably integrated GFP-cassette (without normalization to the transfection efficiency) are shown for four different spacers targeting the Prnp gene (prp2-5). Cells were co-transfected by an active SpCas9 and the respective gRNA expressing vector along with either of the following GFP-plasmids: a homologous recombination (HR) donor GFP-plasmid (empty bars) containing a promoter-less GFP gene with 1 kb homologous arms for the Prnp gene at both sides of the GFP-cassette; or the self-cleaving GFP-plasmid pSc1 (grey bars). As a control for the HR mediated integrations, the HR donor plasmid was co-transfected with an inactive SpCas9 expression vector (dead Cas9). The values obtained for this control (0.1%) indicated as random integration background with HR, was subtracted from all corresponding sample values to determine the extent of targeted integrations. For experiments with the self-cleaving plasmid, the control used was the self-cleaving plasmid pSc1 co-transfected with a vector expressing active SpCas9 and the TL1 gRNA and thus, the pSc1 plasmid is linearized in-cell but the genome is not targeted (indicated as random integration background with pSc1 donor). The value obtained for this control (5%) was subtracted from all corresponding sample values to determine the extent of net targeted integrations. Values were not normalized for transfection efficiency in either experiments. Bars show the means ± S.D. of percentages measured in n = 3 independent transfections." path-from-xml="dsx029f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086404/dsx029f4.tif" path-from-xml="dsx029f4.tif" target="_blank">View large</a><a section="104086404" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/dsx029f4.png?Expires=2147483647&Signature=Y3xPXptt2UiIKsNptV-QRgsU7b4kbM-M-2cl3TmqCyh7LhB28kAwAQpkxWaQJW2Ji26cTPVplYObWfmfh3YFSfLLKHimiq0-c5bimsqGb-eX2rk1WE~B0IzveZ2QFPiaXXn1VZ1pUrxY~Vi~XzgPt85zARttsQNKpEwmArLkCyubDdYd2VsI5rXJ6q9nwycU5ww~-JskRJZaIsyts0O8eAOqlqX88ZU3AwRs8MAbhFVw~K66ihjZb-giRQDlgjRRJMPVQkDxfJu8Yp~Ek0TqRKuEEWlUglZ9L8bGyhhjDcULktVDOgUM-WRjVDUF64LJhZjrjmZeDu-0tEk69Jo2nA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086404&ar=3904549&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx029f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Efficiency of integration of the self-cleaving plasmid to the genome is comparable to that of a HR-mediated integration. The percentages of the cells harbouring a stably integrated GFP-cassette (without normalization to the transfection efficiency) are shown for four different spacers targeting the <em>Prnp</em> gene (prp2-5). Cells were co-transfected by an active SpCas9 and the respective gRNA expressing vector along with either of the following GFP-plasmids: a homologous recombination (HR) donor GFP-plasmid (empty bars) containing a promoter-less GFP gene with 1 kb homologous arms for the <em>Prnp</em> gene at both sides of the GFP-cassette; or the self-cleaving GFP-plasmid pSc1 (grey bars). As a control for the HR mediated integrations, the HR donor plasmid was co-transfected with an inactive SpCas9 expression vector (dead Cas9). The values obtained for this control (0.1%) indicated as <em>random integration background with HR</em>, was subtracted from all corresponding sample values to determine the extent of targeted integrations. For experiments with the self-cleaving plasmid, the control used was the self-cleaving plasmid pSc1 co-transfected with a vector expressing active SpCas9 and the TL1 gRNA and thus, the pSc1 plasmid is linearized in-cell but the genome is not targeted (indicated as <em>random integration background with pSc1 donor</em>). The value obtained for this control (5%) was subtracted from all corresponding sample values to determine the extent of net targeted integrations. Values were not normalized for transfection efficiency in either experiments. Bars show the means ± S.D. of percentages measured in <em>n</em> = 3 independent transfections.</p></div></div><div content-id="dsx029-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/m_dsx029f4.png?Expires=2147483647&Signature=Rb4-EJYtcJN6WdAxih08GXHb0ioSljpHqvZyuhXA7hjiqM3gYEAlS6-w2vw7h0qGpDXLjTOzoOLizXFeD1tYxnmmudV9xDN~eDIC-avvcwkP2fkF4HvtOHz5rHUDCeqftdwLMDcGFvRS1FTDVEoDadmAvAg4z1j14HovW2skSc~cUFS3Bp38CLpncW2K-~NKxSM~xtQfOsutwOsdlUSnKBRyoW94Q~9jvuCaYzHYHb~sLNHr~XMLYrXOLZO1MUqplb6RV-ozyX0-pjwEMayXoUYvujCwn9VDIhGeqjhRkLy5yzi8LgUugH6g0ZRHYw2Z8eeSqe0XDD8LGn3eHy85GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Efficiency of integration of the self-cleaving plasmid to the genome is comparable to that of a HR-mediated integration. The percentages of the cells harbouring a stably integrated GFP-cassette (without normalization to the transfection efficiency) are shown for four different spacers targeting the Prnp gene (prp2-5). Cells were co-transfected by an active SpCas9 and the respective gRNA expressing vector along with either of the following GFP-plasmids: a homologous recombination (HR) donor GFP-plasmid (empty bars) containing a promoter-less GFP gene with 1 kb homologous arms for the Prnp gene at both sides of the GFP-cassette; or the self-cleaving GFP-plasmid pSc1 (grey bars). As a control for the HR mediated integrations, the HR donor plasmid was co-transfected with an inactive SpCas9 expression vector (dead Cas9). The values obtained for this control (0.1%) indicated as random integration background with HR, was subtracted from all corresponding sample values to determine the extent of targeted integrations. For experiments with the self-cleaving plasmid, the control used was the self-cleaving plasmid pSc1 co-transfected with a vector expressing active SpCas9 and the TL1 gRNA and thus, the pSc1 plasmid is linearized in-cell but the genome is not targeted (indicated as random integration background with pSc1 donor). The value obtained for this control (5%) was subtracted from all corresponding sample values to determine the extent of net targeted integrations. Values were not normalized for transfection efficiency in either experiments. Bars show the means ± S.D. of percentages measured in n = 3 independent transfections." path-from-xml="dsx029f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086404/dsx029f4.tif" path-from-xml="dsx029f4.tif" target="_blank">View large</a><a section="104086404" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/dsx029f4.png?Expires=2147483647&Signature=Y3xPXptt2UiIKsNptV-QRgsU7b4kbM-M-2cl3TmqCyh7LhB28kAwAQpkxWaQJW2Ji26cTPVplYObWfmfh3YFSfLLKHimiq0-c5bimsqGb-eX2rk1WE~B0IzveZ2QFPiaXXn1VZ1pUrxY~Vi~XzgPt85zARttsQNKpEwmArLkCyubDdYd2VsI5rXJ6q9nwycU5ww~-JskRJZaIsyts0O8eAOqlqX88ZU3AwRs8MAbhFVw~K66ihjZb-giRQDlgjRRJMPVQkDxfJu8Yp~Ek0TqRKuEEWlUglZ9L8bGyhhjDcULktVDOgUM-WRjVDUF64LJhZjrjmZeDu-0tEk69Jo2nA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086404&ar=3904549&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx029f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Efficiency of integration of the self-cleaving plasmid to the genome is comparable to that of a HR-mediated integration. The percentages of the cells harbouring a stably integrated GFP-cassette (without normalization to the transfection efficiency) are shown for four different spacers targeting the <em>Prnp</em> gene (prp2-5). Cells were co-transfected by an active SpCas9 and the respective gRNA expressing vector along with either of the following GFP-plasmids: a homologous recombination (HR) donor GFP-plasmid (empty bars) containing a promoter-less GFP gene with 1 kb homologous arms for the <em>Prnp</em> gene at both sides of the GFP-cassette; or the self-cleaving GFP-plasmid pSc1 (grey bars). As a control for the HR mediated integrations, the HR donor plasmid was co-transfected with an inactive SpCas9 expression vector (dead Cas9). The values obtained for this control (0.1%) indicated as <em>random integration background with HR</em>, was subtracted from all corresponding sample values to determine the extent of targeted integrations. For experiments with the self-cleaving plasmid, the control used was the self-cleaving plasmid pSc1 co-transfected with a vector expressing active SpCas9 and the TL1 gRNA and thus, the pSc1 plasmid is linearized in-cell but the genome is not targeted (indicated as <em>random integration background with pSc1 donor</em>). The value obtained for this control (5%) was subtracted from all corresponding sample values to determine the extent of net targeted integrations. Values were not normalized for transfection efficiency in either experiments. Bars show the means ± S.D. of percentages measured in <em>n</em> = 3 independent transfections.</p></div></div><p>In another experiment, we selected and cloned 5 spacers targeting the <em>Rosa26</em> locus in the mouse genome (one of them, rosa5 spacer previously reported in ref.<a reveal-id="dsx029-B56" data-open="dsx029-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a>). The five gRNAs seem to mediate comparable cleavage efficiencies (about 10–20%) by both approaches (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>), although the target sites were not so carefully selected as in case of <a reveal-id="dsx029-F4" data-open="dsx029-F4" class="link link-reveal link-table xref-fig">Figure 4</a> and thus, the relative position of the targets may have influenced their efficiency in the HR-mediated assay.</p><p>HR-mediated integration is currently the only approach used in fluorescence-based methods that are capable of testing the genomic targets in their genomic context (GFP-disruption and analogous assays are restricted to targets aimed at the GFP sequences). However, the HR-mediated method is much more laborious due to the need to construct the homologous arms. Therefore, it is only worth to apply it for screening gRNA-efficiencies if the eventual application is to employ the same homologous arms. Thus, the advantages of our approach over other fluorescence-based methods for gRNA testing are apparent and it may be more relevant to compare to PCR-based methods what we present later in the manuscript.</p><p>Further experimentation on NIH/3T3 mouse fibroblast cells with one of the self-cleaving plasmids and seven spacers of the set previously tested in N2a cells, targeting five genes (<em>Pten</em>, <em>Rbl2</em>, Tp53, <em>Prnp</em>, and <em>Piwil4</em>) resulted in sufficiently high targeted integrations (i.e. for the majority of the spacers clearly measurable above the background 5%) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7A</a></span>). The most effective spacer, pten resulted in close to 15% targeted integration, whereas piwi2 showed no detectable targeted integration. Thus, these results recapitulate those seen earlier with these spacers in N2a cells and suggest that our approach is applicable in NIH/3T3 mouse cells as well.</p><p>The TL spacers have neither targets nor predicted off-targets in the human genome, allowing us therefore, to test the applicability of our approach on human cells as well. The commonly used human cell line HeLa was selected, using one of the two self-cleaving plasmids (pSc1). We could not use most of the mouse gene-targeting spacers tested earlier in N2a cells on the corresponding human genomic sites; however, we recognized that the pten spacer also targets the human <em>PTEN</em> gene and <em>PTENP1</em> pseudogene. In addition, two other spacers that target the human <em>PRNP</em> gene were designed. While the in-cell linearized plasmid background remained at 1.4%, the two prion protein targeting spacers and the pten spacer mediate targeted integrations (i.e. above background level) from 5% to 15%, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7B</a></span>). Additionally, four previously reported active gRNAs<a reveal-id="dsx029-B57" data-open="dsx029-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> were tested in HEK293 cells targeting the <em>LMNA</em> and <em>PML</em> genes along with the gRNA targeting the <em>PTEN</em> gene. All spacers mediated effective targeted integrations (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7C</a></span>). Altogether, these results on different cell lines (N2a, NIH/3T3, HeLa, and HEK293) suggest that NHEJ-mediated targeted integration in combination with these self-cleaving plasmids is sufficiently high above random integration to exploit it for testing gRNA efficiency more generally.</p>                            <h3 scrollto-destination=104086409 id="104086409" class="section-title" >3.4. Shortening detection time with a degradation domain fused to GFP</h3>
<p>Although these experiments demonstrate the effectiveness of our approach requiring little effort and cost, the long time-frame necessary for obtaining the readout limits its use. Our experiments suggested that the long decay time of the transient signal originates from the high stability of GFP protein molecules that are present after the disappearance of the plasmid DNA. To avoid this effect, ideally, GFP expression should be induced after the degradation of the transfected plasmid DNA in the cell, ensuring that the GFP expression originates only from the integrated copies of the expression cassette. We achieved a similar outcome by fusing a degradation domain (DD), a destabilized mutant of <em>E. coli</em> dihydrofolate reductase (ecDHFR) protein to GFP that drives faster degradation of the fluorescent protein. The degradation domain is stabilized in the presence of trimethoprim (TMP), preventing enhanced degradation.<a reveal-id="dsx029-B50" data-open="dsx029-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> Adding TMP to the cells after the transfected plasmid DNA has disappeared allows the detection of the GFP signal originating only from proteins expressed from the integrated copies.</p><p>Fusing the degradation domain either to the N- or to the C- terminus of GFP resulted similar expression profiles (data not shown), thus, we selected the N-terminal fusion for further experiments and placed it to a self-cleaving plasmid (pSc1-DD, Addgene #80439). Three gRNAs tested previously, Rb, p53, and piwi2 (the first two representing efficient, whereas the third poor gRNAs) were exploited to adjust the condition of the modified assay employing pSc1-DD. The gRNA-Cas9 plasmids were co-transfected with both the pSc1-DD plasmid and an mCherry coding plasmid to N2a cells. The latter allows for monitoring the transfection efficiencies. To determine the earliest time point that is appropriate for distinguishing between poor and active gRNA-s, GFP positive cells were measured daily from the 2nd to the 10th day post transfection (data not shown). TMP was added to all samples 24 h before FACS analysis. The gRNAs tested previously with the pSc1 and pSc2 plasmids (<a reveal-id="dsx029-F3" data-open="dsx029-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>), now with pSc1-DD showed similar activities. This approach resulted in the successful reduction of the assay time from 12–15 to 6–8 days (<a reveal-id="dsx029-F5" data-open="dsx029-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). The activities of the gRNAs are clearly distinct (i.e. the poor and good gRNAs can be easily distinguished) on the 6th day post transfection, although the fluorescence stabilizes only from the 8th day (data not shown). </p>                        <a id="104086412" scrollto-destination="104086412"></a>
<div id="dsx029-F5" data-id="dsx029-F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/m_dsx029f5.png?Expires=2147483647&Signature=xEd7dFvOU3t67I0FVIHD-xr9YSEaXB6tIW5v7RRbOz144xahjtDs3bGhfaLzWALYCmRYP85LvZNiHo9RfXJ6-joKyFJTlho8~kSExGAsI9OA~tNj-gPuktcauAjiuvaEfZ7yhsHcTzRwNzFnFEDYm1w~aACWqZCgJAW0jt5SWLT-ULDhSeUo5foN8XWcHdgoJWTVqjZnFfZFXjlNhXZph9RN04vm8smqmeLo7Ip~tKZkP8mhjqGsfSxWJ-ushNhtLZV6fFXG6bl~bZES3EuN21cBA4htdDAuX8ynjHXrJU8OwiAHHGVPkUJKIjtEDEFA33GusZUhQwo9-PulaGtv3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Shortening detection time with a degradation domain fused to GFP. The percentages of the cells harbouring stably integrated GFP-cassette were measured between 6 and 8 days after transfection when a self-cleaving plasmid with a degradation domain-fused GFP (pSc1-DD) was co-transfected with three formerly used (e.g. Fig. 3) SpCas9-gRNA coding plasmids (targeting the Rbl2, Tp53 and Piwil4 genes) and an mCherry coding plasmid (to track transfection efficiency) to N2a cells. To stabilize the degradation domain, trimethoprim was added to the cells 24 h before FACS analysis. As controls, the pSc1-DD plasmid was co-transfected with either (I) a nuclease inactive SpCas9 coding plasmid (dead); or (II) with an active SpCas9 and the TL1 gRNA expressing vector and thus, the pSc1 plasmid is linearized in-cell but the genome is not targeted (indicated as genome not targeted pSc1-DD in cell cut). Bars show the means ± S.D. of percentages measured in n = 3 independent transfections, which were normalized to the transfection efficiency [% mCherry positive cells (ranging between 44 and 52%) measured on the 2nd day after transfection]." path-from-xml="dsx029f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086412/dsx029f5.tif" path-from-xml="dsx029f5.tif" target="_blank">View large</a><a section="104086412" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/dsx029f5.png?Expires=2147483647&Signature=Fi~tmkHzNArYfkr99qgRH-qtwgj~xpDOGSqCmpnSoWsrr19L9PLcz1kLspVAZuyyVxN~Z396UZjyzkzqv7leTwGQNzVz2zXHunNEWJmrvE6h3E9xa1CjbLP7iXhtRVrifEOCTHreHD9nLogERuc1c24W191cqzTkGCIz9bbZPi4JhgUN7-iN8tFt1jbb16A-YrdRuhbmFXizO~SbzQYy381oFwBr61DgDOU7io0Zw3vZ3Ejj592BrcHasooqyB7ZI2dP~Uhf0v39GKbV7~DSOJhRoBJT~BPu77i4v5sLQM4Jk-A03TKN80l6BfSi0AJZKsJGnBLrQb3RTXTl~TA9Cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086412&ar=3904549&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx029f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Shortening detection time with a degradation domain fused to GFP. The percentages of the cells harbouring stably integrated GFP-cassette were measured between 6 and 8 days after transfection when a self-cleaving plasmid with a degradation domain-fused GFP (pSc1-DD) was co-transfected with three formerly used (e.g. <a reveal-id="dsx029-F3" data-open="dsx029-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>) SpCas9-gRNA coding plasmids (targeting the <em>Rbl2</em>, <em>Tp53</em> and <em>Piwil4</em> genes) and an mCherry coding plasmid (to track transfection efficiency) to N2a cells. To stabilize the degradation domain, trimethoprim was added to the cells 24 h before FACS analysis. As controls, the pSc1-DD plasmid was co-transfected with either (I) a nuclease inactive SpCas9 coding plasmid (dead); or (II) with an active SpCas9 and the TL1 gRNA expressing vector and thus, the pSc1 plasmid is linearized in-cell but the genome is not targeted (indicated as <em>genome not targeted pSc1-DD in cell cut</em>). Bars show the means ± S.D. of percentages measured in <em>n</em> = 3 independent transfections, which were normalized to the transfection efficiency [% mCherry positive cells (ranging between 44 and 52%) measured on the 2nd day after transfection].</p></div></div><div content-id="dsx029-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/m_dsx029f5.png?Expires=2147483647&Signature=xEd7dFvOU3t67I0FVIHD-xr9YSEaXB6tIW5v7RRbOz144xahjtDs3bGhfaLzWALYCmRYP85LvZNiHo9RfXJ6-joKyFJTlho8~kSExGAsI9OA~tNj-gPuktcauAjiuvaEfZ7yhsHcTzRwNzFnFEDYm1w~aACWqZCgJAW0jt5SWLT-ULDhSeUo5foN8XWcHdgoJWTVqjZnFfZFXjlNhXZph9RN04vm8smqmeLo7Ip~tKZkP8mhjqGsfSxWJ-ushNhtLZV6fFXG6bl~bZES3EuN21cBA4htdDAuX8ynjHXrJU8OwiAHHGVPkUJKIjtEDEFA33GusZUhQwo9-PulaGtv3g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Shortening detection time with a degradation domain fused to GFP. The percentages of the cells harbouring stably integrated GFP-cassette were measured between 6 and 8 days after transfection when a self-cleaving plasmid with a degradation domain-fused GFP (pSc1-DD) was co-transfected with three formerly used (e.g. Fig. 3) SpCas9-gRNA coding plasmids (targeting the Rbl2, Tp53 and Piwil4 genes) and an mCherry coding plasmid (to track transfection efficiency) to N2a cells. To stabilize the degradation domain, trimethoprim was added to the cells 24 h before FACS analysis. As controls, the pSc1-DD plasmid was co-transfected with either (I) a nuclease inactive SpCas9 coding plasmid (dead); or (II) with an active SpCas9 and the TL1 gRNA expressing vector and thus, the pSc1 plasmid is linearized in-cell but the genome is not targeted (indicated as genome not targeted pSc1-DD in cell cut). Bars show the means ± S.D. of percentages measured in n = 3 independent transfections, which were normalized to the transfection efficiency [% mCherry positive cells (ranging between 44 and 52%) measured on the 2nd day after transfection]." path-from-xml="dsx029f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086412/dsx029f5.tif" path-from-xml="dsx029f5.tif" target="_blank">View large</a><a section="104086412" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/dsx029f5.png?Expires=2147483647&Signature=Fi~tmkHzNArYfkr99qgRH-qtwgj~xpDOGSqCmpnSoWsrr19L9PLcz1kLspVAZuyyVxN~Z396UZjyzkzqv7leTwGQNzVz2zXHunNEWJmrvE6h3E9xa1CjbLP7iXhtRVrifEOCTHreHD9nLogERuc1c24W191cqzTkGCIz9bbZPi4JhgUN7-iN8tFt1jbb16A-YrdRuhbmFXizO~SbzQYy381oFwBr61DgDOU7io0Zw3vZ3Ejj592BrcHasooqyB7ZI2dP~Uhf0v39GKbV7~DSOJhRoBJT~BPu77i4v5sLQM4Jk-A03TKN80l6BfSi0AJZKsJGnBLrQb3RTXTl~TA9Cw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086412&ar=3904549&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx029f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Shortening detection time with a degradation domain fused to GFP. The percentages of the cells harbouring stably integrated GFP-cassette were measured between 6 and 8 days after transfection when a self-cleaving plasmid with a degradation domain-fused GFP (pSc1-DD) was co-transfected with three formerly used (e.g. <a reveal-id="dsx029-F3" data-open="dsx029-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>) SpCas9-gRNA coding plasmids (targeting the <em>Rbl2</em>, <em>Tp53</em> and <em>Piwil4</em> genes) and an mCherry coding plasmid (to track transfection efficiency) to N2a cells. To stabilize the degradation domain, trimethoprim was added to the cells 24 h before FACS analysis. As controls, the pSc1-DD plasmid was co-transfected with either (I) a nuclease inactive SpCas9 coding plasmid (dead); or (II) with an active SpCas9 and the TL1 gRNA expressing vector and thus, the pSc1 plasmid is linearized in-cell but the genome is not targeted (indicated as <em>genome not targeted pSc1-DD in cell cut</em>). Bars show the means ± S.D. of percentages measured in <em>n</em> = 3 independent transfections, which were normalized to the transfection efficiency [% mCherry positive cells (ranging between 44 and 52%) measured on the 2nd day after transfection].</p></div></div>                            <h3 scrollto-destination=104086413 id="104086413" class="section-title" >3.5. The approach is also validated by two PCR-based methods: TIDE and T7 endonuclease I assay</h3>
<p>Ten spacer sequences targeting six genes in the mouse genome (four for <em>Prnp</em>, two for <em>Pten</em>, and one for <em>Dcun1d2, Rbl2</em>, <em>Tp53</em>, and <em>Piwil4</em> genes), were selected for a comparison of this approach and indel detection methods such as Tracking Indels by Decomposition (TIDE<a reveal-id="dsx029-B33" data-open="dsx029-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a>) and T7 endonuclease I assay. These 10 gRNA-Cas9 plasmids were used for co-transfection with the pSc1-DD plasmid into N2a cells. Genomic DNA from the FACS-analysed cells was extracted on the 8th day post transfection and the SpCas9 targeted sites were PCR amplified and indels were detected by both TIDE and T7 endonuclease I assay. All approaches, this reporter assay, TIDE and T7 endonuclease I assay identified the same five gRNAs as being the more efficient ones (pten, pten2, p53, Rb, and prp2) and the same five gRNAs to result low or no cleavage activities (prp8, prp9, prp10, piwi2, and dcun1d2) (<a reveal-id="dsx029-F6" data-open="dsx029-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). However, as expected, the relative efficiencies among the gRNAs slightly differed, likely reflecting the differences in the outcomes of the competing repair events they monitor. </p>                        <a id="104086415" scrollto-destination="104086415"></a>
<div id="dsx029-F6" data-id="dsx029-F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/m_dsx029f6.png?Expires=2147483647&amp;Signature=eL8dRWm0Xj833pinfDk5tW38jxclWaG5MBVh8uxoVB6qvbUeTeSLrGsMcKyaFE421qGF-bNg3posKKL2ZAgJsZOCAmlOuzXxjoIQoEkOjWa3iTN8visYk8K6XATY7lpN0~2mSnyGqWMZlPqKXOnIBdKr4dXuPM2J7pZ1e9n-gWwyWjvjls5Vbo8895MhqRmkLYqNMAa60XnmlfJNVZDLkytRqbkDRfE9AWEq~7LQoc5qLh1uqrSx3TYTObF9eHMf5x~~Tgvv2Gsxoa2d8NNPw5ONzLTRXcMSnFNVykzpVSdSCHFZ021Y10OtaLkzn4MsFTPYLLb51-eXW3F9EtbV7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Validation of the method by TIDE and T7 endonuclease I assay. The activities of 10 gRNA-s are shown that were measured with the degradation domain-fused GFP bearing self-cleaving plasmid (pSc1-DD, dark grey boxes) and with indels that were detected in the same samples with TIDE (light grey boxes) and T7 endonuclease I assay (medium grey boxes). The percentages of the cells harbouring a stably integrated GFP-cassette are shown for 10 different spacers targeting the (Prnp, Pten, Dcun1d2, Rbl2, Tp53, and Piwil4 genes). Cells were co-transfected by an active SpCas9 and the respective gRNA expressing vector along with the degradation domain bearing GFP- self-cleaving plasmid, pSc1-DD (dark grey bars). As a control the pSc1-DD was co-transfected with an active SpCas9 and the TL1 gRNA expressing vector and thus, the pSc1-DD plasmid is linearized in-cell but the genome is not targeted (indicated as genome not targeted, pSc1-DD in-cell cut). The GFP positive cells were analysed by FACS 8 days after transfection. Bars show the means ± S.D. of percentages measured in n = 3 independent transfections which were normalized to the transfection efficiency [% mCherry positive cells (ranging between 44 and 52%) measured on the 2nd day after transfection]. After FACS measurements, the three parallel samples were mixed, genomic DNA was isolated and three parallel PCR reactions were carried out from each sample. Indels were measured by both TIDE and T7 endonuclease I assay on each PCR reaction. The agarose gel electrophoresis analysis of T7EI assays are shown in Supplementary Fig. S8." path-from-xml="dsx029f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086415/dsx029f6.tif" path-from-xml="dsx029f6.tif" target="_blank">View large</a><a section="104086415" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/dsx029f6.png?Expires=2147483647&amp;Signature=rjsz6LH72v1ctLRHrF3NDRHUF4jJaGmEO2YNSr0149QdJNI8KacfqjPMd7-o9LdklasMhH3KvI3ZH0qS-YRfEexYd2ODVH1VVSR9PhyvTHM-l1xgiaARmy2EUYO2Aon8pzYGchp7KNZKTjDrkH6eNES6IfPtDVkuO7IkuvkW1RM8~on78eCjE9-le8exiz9NrXv4bXo8aCXEQanjtb8LFf7MNiJ3wlM7T7Kleh2glOJjl5WIPBJp0K5OkZoqv24Q7yycCmbB6369p92o-n6vfSmxMAorg3piqBNW8koZAQpEiRY~8Whz7T9piGc7lcGjnULvLWmVPzL5ztXm-wJkGw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=104086415&amp;ar=3904549&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx029f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Validation of the method by TIDE and T7 endonuclease I assay. The activities of 10 gRNA-s are shown that were measured with the degradation domain-fused GFP bearing self-cleaving plasmid (pSc1-DD, dark grey boxes) and with indels that were detected in the same samples with TIDE (light grey boxes) and T7 endonuclease I assay (medium grey boxes). The percentages of the cells harbouring a stably integrated GFP-cassette are shown for 10 different spacers targeting the (<em>Prnp</em>, <em>Pten, Dcun1d2, Rbl2</em>, <em>Tp53</em>, and <em>Piwil4</em> genes). Cells were co-transfected by an active SpCas9 and the respective gRNA expressing vector along with the degradation domain bearing GFP- self-cleaving plasmid, pSc1-DD (dark grey bars). As a control the pSc1-DD was co-transfected with an active SpCas9 and the TL1 gRNA expressing vector and thus, the pSc1-DD plasmid is linearized in-cell but the genome is not targeted (indicated as <em>genome not targeted, pSc1-DD in-cell cut</em>). The GFP positive cells were analysed by FACS 8 days after transfection. Bars show the means ± S.D. of percentages measured in <em>n</em> = 3 independent transfections which were normalized to the transfection efficiency [% mCherry positive cells (ranging between 44 and 52%) measured on the 2nd day after transfection]. After FACS measurements, the three parallel samples were mixed, genomic DNA was isolated and three parallel PCR reactions were carried out from each sample. Indels were measured by both TIDE and T7 endonuclease I assay on each PCR reaction. The agarose gel electrophoresis analysis of T7EI assays are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&amp;Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>.</p></div></div><div content-id="dsx029-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/m_dsx029f6.png?Expires=2147483647&amp;Signature=eL8dRWm0Xj833pinfDk5tW38jxclWaG5MBVh8uxoVB6qvbUeTeSLrGsMcKyaFE421qGF-bNg3posKKL2ZAgJsZOCAmlOuzXxjoIQoEkOjWa3iTN8visYk8K6XATY7lpN0~2mSnyGqWMZlPqKXOnIBdKr4dXuPM2J7pZ1e9n-gWwyWjvjls5Vbo8895MhqRmkLYqNMAa60XnmlfJNVZDLkytRqbkDRfE9AWEq~7LQoc5qLh1uqrSx3TYTObF9eHMf5x~~Tgvv2Gsxoa2d8NNPw5ONzLTRXcMSnFNVykzpVSdSCHFZ021Y10OtaLkzn4MsFTPYLLb51-eXW3F9EtbV7g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Validation of the method by TIDE and T7 endonuclease I assay. The activities of 10 gRNA-s are shown that were measured with the degradation domain-fused GFP bearing self-cleaving plasmid (pSc1-DD, dark grey boxes) and with indels that were detected in the same samples with TIDE (light grey boxes) and T7 endonuclease I assay (medium grey boxes). The percentages of the cells harbouring a stably integrated GFP-cassette are shown for 10 different spacers targeting the (Prnp, Pten, Dcun1d2, Rbl2, Tp53, and Piwil4 genes). Cells were co-transfected by an active SpCas9 and the respective gRNA expressing vector along with the degradation domain bearing GFP- self-cleaving plasmid, pSc1-DD (dark grey bars). As a control the pSc1-DD was co-transfected with an active SpCas9 and the TL1 gRNA expressing vector and thus, the pSc1-DD plasmid is linearized in-cell but the genome is not targeted (indicated as genome not targeted, pSc1-DD in-cell cut). The GFP positive cells were analysed by FACS 8 days after transfection. Bars show the means ± S.D. of percentages measured in n = 3 independent transfections which were normalized to the transfection efficiency [% mCherry positive cells (ranging between 44 and 52%) measured on the 2nd day after transfection]. After FACS measurements, the three parallel samples were mixed, genomic DNA was isolated and three parallel PCR reactions were carried out from each sample. Indels were measured by both TIDE and T7 endonuclease I assay on each PCR reaction. The agarose gel electrophoresis analysis of T7EI assays are shown in Supplementary Fig. S8." path-from-xml="dsx029f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086415/dsx029f6.tif" path-from-xml="dsx029f6.tif" target="_blank">View large</a><a section="104086415" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/dsx029f6.png?Expires=2147483647&amp;Signature=rjsz6LH72v1ctLRHrF3NDRHUF4jJaGmEO2YNSr0149QdJNI8KacfqjPMd7-o9LdklasMhH3KvI3ZH0qS-YRfEexYd2ODVH1VVSR9PhyvTHM-l1xgiaARmy2EUYO2Aon8pzYGchp7KNZKTjDrkH6eNES6IfPtDVkuO7IkuvkW1RM8~on78eCjE9-le8exiz9NrXv4bXo8aCXEQanjtb8LFf7MNiJ3wlM7T7Kleh2glOJjl5WIPBJp0K5OkZoqv24Q7yycCmbB6369p92o-n6vfSmxMAorg3piqBNW8koZAQpEiRY~8Whz7T9piGc7lcGjnULvLWmVPzL5ztXm-wJkGw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=104086415&amp;ar=3904549&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx029f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Validation of the method by TIDE and T7 endonuclease I assay. The activities of 10 gRNA-s are shown that were measured with the degradation domain-fused GFP bearing self-cleaving plasmid (pSc1-DD, dark grey boxes) and with indels that were detected in the same samples with TIDE (light grey boxes) and T7 endonuclease I assay (medium grey boxes). The percentages of the cells harbouring a stably integrated GFP-cassette are shown for 10 different spacers targeting the (<em>Prnp</em>, <em>Pten, Dcun1d2, Rbl2</em>, <em>Tp53</em>, and <em>Piwil4</em> genes). Cells were co-transfected by an active SpCas9 and the respective gRNA expressing vector along with the degradation domain bearing GFP- self-cleaving plasmid, pSc1-DD (dark grey bars). As a control the pSc1-DD was co-transfected with an active SpCas9 and the TL1 gRNA expressing vector and thus, the pSc1-DD plasmid is linearized in-cell but the genome is not targeted (indicated as <em>genome not targeted, pSc1-DD in-cell cut</em>). The GFP positive cells were analysed by FACS 8 days after transfection. Bars show the means ± S.D. of percentages measured in <em>n</em> = 3 independent transfections which were normalized to the transfection efficiency [% mCherry positive cells (ranging between 44 and 52%) measured on the 2nd day after transfection]. After FACS measurements, the three parallel samples were mixed, genomic DNA was isolated and three parallel PCR reactions were carried out from each sample. Indels were measured by both TIDE and T7 endonuclease I assay on each PCR reaction. The agarose gel electrophoresis analysis of T7EI assays are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&amp;Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>.</p></div></div>                            <h3 scrollto-destination=104086416 id="104086416" class="section-title" >3.6. In-cell cleavage of a linear plasmid does not recapitulate the same level of integration as in-cell cleavage of a circular plasmid</h3>
<p>It is not clear why in-cell cleavage of circular plasmids promotes more efficient integration as compared to transfection of linear, i.e.: <em>out-of-cell cleaved</em> ones (we refer to this as pre-linearized plasmids). One explanation could be that linear DNA is more prone to degradation and a large proportion of the transfected DNA is digested before its targeted integration can take place. Alternatively, the end of the linear molecule might become integration incompetent with time, for example by concatemerization. It is also possible that NHEJ ligation may preferentially take place among DNA ends that are liberated at the same time. To get more insight into this problem we compared NHEJ-mediated targeted integration of pre-linearized plasmids either cleaving or not cleaving them again in-cell. in-cell cleavage of the pre-linearized plasmids apparently does not increase targeted integration whereas in-cell linearization of the circular plasmid (resulting in one fragment) facilitated twice as many targeted integrations as pre-linearization (<a reveal-id="dsx029-F7" data-open="dsx029-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). </p>                        <a id="104086418" scrollto-destination="104086418"></a>
<div id="dsx029-F7" data-id="dsx029-F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/m_dsx029f7.png?Expires=2147483647&Signature=3W3dSCQtZlB63~UEfvUOp~aEcKfmBo9LICEJVHZwBLBJ97D8AJHYBHCQpK6THCpaQnHQMTZBeGWzbuksSEhAl6aQ9YuG1P5Pem4ENg9Ps02G-4RCd4SKTTjM57yF9Lr8QlnlsI6un7ZSZOtcHg~5xj4cUZR4WupLDkE8~EFvfwoT78sWwKBWm3nG15JWN5EENdzXoZJHVsp9JvD6c9yae2Xk9GLT4e1UHqxpd~JkLGga8qyPaqstsfhIpoha1AktyKH6kUIk1ZjDc~QF5a7DFZfEy3Dmg3tP2-rWu0JdXUt3L7YYNz69GVeJsiPGigWxWqB56h3q89tffaf63VPwuw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Integration efficiency of the in-cell cleaved pre-linearized plasmid is lower than that of the in-cell cleaved circular plasmid in N2a cells. The percentages of the cells harbouring stably integrated GFP-cassette are shown when targeting two different genes, Ttn or Rbl2 with the spacers Ttn or Rb, respectively, as indicated. In case of either of the targets, the cells were co-transfected by an expression vector for nuclease active SpCas9 (+) and the respective gRNA (Rb or Ttn) along with: (I) a circular GFP-plasmid containing two matching protospacers ( ++) before and after the GFP-cassette (dark green bars; as seen formerly in Fig. 1A); (II) a pre-linearized GFP-plasmid containing two matching protospacers ( ++) before and after the GFP-cassette (light green bars); (III) a pre-linearized GFP-plasmid containing no protospacers (magenta bars). For both spacers as a negative control a nuclease inactive SpCas9 (-) and gRNA expressing plasmid was co-transfected with a pre-linearized GFP-plasmid (grey bars). For the in-cell linearized plasmid, as a control, a circular plasmid with no matching protospacer was co-transfected with Rb spacer and SpCas9 expressing vector, therefore, the plasmid cannot be linearized in the cell (sky blue bar). As an additional control a circular GFP-plasmid was co-transfected with inactive SpCas9 nuclease expressing vector (orange). Bars show the mean ± standard deviation (S.D.) of percentages measured in n = 3 independent transfections which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 95 and 99%)." path-from-xml="dsx029f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086418/dsx029f7.tif" path-from-xml="dsx029f7.tif" target="_blank">View large</a><a section="104086418" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/dsx029f7.png?Expires=2147483647&Signature=YeKtz2PhzbcxdK9Oxl1La7LtkKU6Bz7v-tnBRY5fEhEdfjggz7TnV0JhmXE9ZxGZIsK6mH~tlm-2O~CRuvP7mRs-Q~UjIkuhw152SaEj-rYnosvWBryuKtTeL0Yz~HLcAXeqtQP5KYimOLnQUygFYJyq6Bfs5StcgoXE1Xm0oj-bRL6Rkv8qIxKJt9gbSnsoEddXd4mV8EqK58g0aXcMphHl1ULGHp8GAytgHlWaTNILEW19jsw8q15etTHBVxMBN1XNXcn3DuPoYDfJ9nMhFsdnLjNAuWUwAnUgL3yijhrB426ygWuJARTtC4-w~G3yHhptlg0XOPnPIL2XKMWvFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086418&ar=3904549&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx029f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Integration efficiency of the in-cell cleaved pre-linearized plasmid is lower than that of the in-cell cleaved circular plasmid in N2a cells. The percentages of the cells harbouring stably integrated GFP-cassette are shown when targeting two different genes, <em>Ttn</em> or <em>Rbl2</em> with the spacers Ttn or Rb, respectively, as indicated. In case of either of the targets, the cells were co-transfected by an expression vector for nuclease active SpCas9 (+) and the respective gRNA (Rb or Ttn) along with: (I) a circular GFP-plasmid containing two matching protospacers ( ++) before and after the GFP-cassette (dark green bars; as seen formerly in <a reveal-id="dsx029-F1" data-open="dsx029-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>); (II) a pre-linearized GFP-plasmid containing two matching protospacers ( ++) before and after the GFP-cassette (light green bars); (III) a pre-linearized GFP-plasmid containing no protospacers (magenta bars). For both spacers as a negative control a nuclease inactive SpCas9 (-) and gRNA expressing plasmid was co-transfected with a pre-linearized GFP-plasmid (grey bars). For the in-cell linearized plasmid, as a control, a circular plasmid with no matching protospacer was co-transfected with Rb spacer and SpCas9 expressing vector, therefore, the plasmid cannot be linearized in the cell (sky blue bar). As an additional control a circular GFP-plasmid was co-transfected with inactive SpCas9 nuclease expressing vector (orange). Bars show the mean ± standard deviation (S.D.) of percentages measured in <em>n</em> = 3 independent transfections which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 95 and 99%).</p></div></div><div content-id="dsx029-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/m_dsx029f7.png?Expires=2147483647&Signature=3W3dSCQtZlB63~UEfvUOp~aEcKfmBo9LICEJVHZwBLBJ97D8AJHYBHCQpK6THCpaQnHQMTZBeGWzbuksSEhAl6aQ9YuG1P5Pem4ENg9Ps02G-4RCd4SKTTjM57yF9Lr8QlnlsI6un7ZSZOtcHg~5xj4cUZR4WupLDkE8~EFvfwoT78sWwKBWm3nG15JWN5EENdzXoZJHVsp9JvD6c9yae2Xk9GLT4e1UHqxpd~JkLGga8qyPaqstsfhIpoha1AktyKH6kUIk1ZjDc~QF5a7DFZfEy3Dmg3tP2-rWu0JdXUt3L7YYNz69GVeJsiPGigWxWqB56h3q89tffaf63VPwuw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Integration efficiency of the in-cell cleaved pre-linearized plasmid is lower than that of the in-cell cleaved circular plasmid in N2a cells. The percentages of the cells harbouring stably integrated GFP-cassette are shown when targeting two different genes, Ttn or Rbl2 with the spacers Ttn or Rb, respectively, as indicated. In case of either of the targets, the cells were co-transfected by an expression vector for nuclease active SpCas9 (+) and the respective gRNA (Rb or Ttn) along with: (I) a circular GFP-plasmid containing two matching protospacers ( ++) before and after the GFP-cassette (dark green bars; as seen formerly in Fig. 1A); (II) a pre-linearized GFP-plasmid containing two matching protospacers ( ++) before and after the GFP-cassette (light green bars); (III) a pre-linearized GFP-plasmid containing no protospacers (magenta bars). For both spacers as a negative control a nuclease inactive SpCas9 (-) and gRNA expressing plasmid was co-transfected with a pre-linearized GFP-plasmid (grey bars). For the in-cell linearized plasmid, as a control, a circular plasmid with no matching protospacer was co-transfected with Rb spacer and SpCas9 expressing vector, therefore, the plasmid cannot be linearized in the cell (sky blue bar). As an additional control a circular GFP-plasmid was co-transfected with inactive SpCas9 nuclease expressing vector (orange). Bars show the mean ± standard deviation (S.D.) of percentages measured in n = 3 independent transfections which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 95 and 99%)." path-from-xml="dsx029f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086418/dsx029f7.tif" path-from-xml="dsx029f7.tif" target="_blank">View large</a><a section="104086418" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/dsx029f7.png?Expires=2147483647&Signature=YeKtz2PhzbcxdK9Oxl1La7LtkKU6Bz7v-tnBRY5fEhEdfjggz7TnV0JhmXE9ZxGZIsK6mH~tlm-2O~CRuvP7mRs-Q~UjIkuhw152SaEj-rYnosvWBryuKtTeL0Yz~HLcAXeqtQP5KYimOLnQUygFYJyq6Bfs5StcgoXE1Xm0oj-bRL6Rkv8qIxKJt9gbSnsoEddXd4mV8EqK58g0aXcMphHl1ULGHp8GAytgHlWaTNILEW19jsw8q15etTHBVxMBN1XNXcn3DuPoYDfJ9nMhFsdnLjNAuWUwAnUgL3yijhrB426ygWuJARTtC4-w~G3yHhptlg0XOPnPIL2XKMWvFw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086418&ar=3904549&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx029f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Integration efficiency of the in-cell cleaved pre-linearized plasmid is lower than that of the in-cell cleaved circular plasmid in N2a cells. The percentages of the cells harbouring stably integrated GFP-cassette are shown when targeting two different genes, <em>Ttn</em> or <em>Rbl2</em> with the spacers Ttn or Rb, respectively, as indicated. In case of either of the targets, the cells were co-transfected by an expression vector for nuclease active SpCas9 (+) and the respective gRNA (Rb or Ttn) along with: (I) a circular GFP-plasmid containing two matching protospacers ( ++) before and after the GFP-cassette (dark green bars; as seen formerly in <a reveal-id="dsx029-F1" data-open="dsx029-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>); (II) a pre-linearized GFP-plasmid containing two matching protospacers ( ++) before and after the GFP-cassette (light green bars); (III) a pre-linearized GFP-plasmid containing no protospacers (magenta bars). For both spacers as a negative control a nuclease inactive SpCas9 (-) and gRNA expressing plasmid was co-transfected with a pre-linearized GFP-plasmid (grey bars). For the in-cell linearized plasmid, as a control, a circular plasmid with no matching protospacer was co-transfected with Rb spacer and SpCas9 expressing vector, therefore, the plasmid cannot be linearized in the cell (sky blue bar). As an additional control a circular GFP-plasmid was co-transfected with inactive SpCas9 nuclease expressing vector (orange). Bars show the mean ± standard deviation (S.D.) of percentages measured in <em>n</em> = 3 independent transfections which were normalized to the transfection efficiency (% GFP positive cells measured on 2nd day after transfection: ranging between 95 and 99%).</p></div></div><p>Since in-cell double cleavage (where the restriction enzyme linearized plasmid is cleaved both upstream and downstream the GFP-cassette), may effectively triple the number of DNA molecules with free ends, it might adversely affect the integration efficiency of the fragment containing the GFP-cassette having to compete for the available genomic DNA breaks. To account for this effect, we repeated the experiments, now using controls of pre-linearized plasmids digested into one, two or three pieces. The linear plasmids were also cleaved again once or twice in-cell generating either two or three fragments, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). Pre-linearization of the plasmids into two or three pieces before transfection decreased targeted integration, as expected. in-cell cleavage of the pre-linearized plasmid into two or three pieces facilitated targeted integration twice as much as pre-linearization of the plasmids into two or three pieces before transfection. However, no significant differences were seen when the pre-linearized plasmid was cut once or twice in-cell (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). Thus, in-cell cleavage of pre-linearized plasmid failed to increase targeted integration to a similar level as the in-cell cleavage of a circular DNA. These experiments suggest that the major reason for the increased targeted integration of in-cell cleaved circular plasmids in N2a cells is the higher resistance to nucleases, although these experiments do not rule out a positive contribution from a synchronized in-cell cleavage.</p>                            <h3 scrollto-destination=104086420 id="104086420" class="section-title" >3.7. NHEJ-cloning employing SpCas9 and a self-cleaving plasmid is superior to random integration</h3>
<p>Targeted integration of plasmids as large as 5 kb by NHEJ-cloning using SpCas9, had also been demonstrated in mammalian cells, although, with limited efficiency.<a reveal-id="dsx029-B45" data-open="dsx029-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> We used a self-cleaving plasmid to assess if NHEJ-cloning is capable of targeted integration of 10 kb plasmids with sufficient efficiency. A 5 kb cassette was cloned to the self-cleaving plasmid (resulting in 10805 bp final size) and its integration was tested with the most effective spacer (pten) used in these studies. We found 30% and 18% targeted integration (without normalization for the transfection efficiency i.e.: without upscaling to 100%) of the 10 kb plasmid harbouring a GFP-cassette in N2a and HeLa cells, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10A and B</a></span>). These experiments suggest that the size of the plasmid in this range is not a limiting factor for NHEJ-cloning in these cell lines.</p><p>The efficiencies we achieved here for targeted integrations are almost an order of magnitude higher than those reported earlier in HEK293 cells without using a self-cleaving plasmid.<a reveal-id="dsx029-B45" data-open="dsx029-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> To see if the differences are attributable only to the different cell lines (N2a, NIH/3T3, HeLa vs. HEK293, and CHO) used, we tested our approach on HEK293 cells as well. The experiments presented on <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10C</a></span> demonstrate that by using self-cleaving plasmids (either 5 or 10 kb large), efficient integrations are achievable in HEK293 cells, showing more than an order of magnitude higher efficiencies than seen in earlier studies [7% targeted integration here (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10C</a></span>) vs. 0.17% in Ref. <a reveal-id="dsx029-B45" data-open="dsx029-B45" class="link link-ref link-reveal xref-bibr">45</a>]. These results were confirmed later exploiting more targets (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7C</a></span>).</p><p>Our experiments suggest that NHEJ-cloning using SpCas9 nuclease with a self-cleaving plasmid could be a superior alternative to random integration of DNA cassettes. To establish an easy way for integration of DNA cassettes to safe-harbour locations, we identified 5 spacers targeting the <em>Rosa26</em> locus in the mouse genome, as discussed earlier (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>).</p><p>We also examined whether NHEJ-cloning of a self-cleaving plasmid would result in coupled integration of two expression cassettes harboured on the same plasmid. This is a major concern exploiting random integration where selection for an antibiotic does not always result in expression of the target gene located on the same plasmid. N2a cells were co-transfected with two plasmids: one is a self-cleaving plasmid containing both a GFP and a puromycin expression cassette (pSc1-puro, Addgene #80438) and the other is a plasmid for SpCas9 and gRNA expression targeting the <em>PTEN</em> gene. The fraction of cells exhibiting GFP fluorescence was counted by FACS after 10 days of puromycin selection when the transient GFP expression had been decayed. These experiments demonstrated about 90% coupled integration of the puromycin and the GFP expression cassettes by NHEJ-cloning of a self-cleaving plasmid. By contrast, this value is 50% for coupled random integration in the same experimental setup but without cleaving the plasmid and the genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>). This latter experiment resulted in two orders of magnitude less resistant cells indicating the higher efficiency of NHEJ-cloning.</p>                            <h2 scrollto-destination=104086425 id="104086425" class="section-title" >4. Discussion</h2>
<p>In the absence of accurate prediction tools for the efficiency of Cas9 spacers, suitable gRNAs can only be selected reliably by experimental testing. Although several approaches exist to measure the activity of SpCas9-gRNAs,<a reveal-id="dsx029-B24" data-open="dsx029-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> there is an absence of simple fluorescence-based methods that would be able to assess the effectivity of gRNAs on targets in their genomic context while ensuring an easy and reliable detection by FACS. Here, we introduced a method that fulfils this gap and offers a simple alternative to existing approaches for testing gRNA efficiencies prior to executing more complex applications like generating specifically modified cell lines, transgenic animals or performing multiplex genome engineering tasks. As it requires only the ready gRNAs, our method can be applied by simply co-transfecting the Cas9-gRNA plasmids with a self-cleaving reporter plasmid. Thus, it does not require extra cloning steps for testing a particular gRNA in contrast to other fluorescent reporter-based methods.<a reveal-id="dsx029-B35" data-open="dsx029-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><a reveal-id="dsx029-B39" data-open="dsx029-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsx029-B40" data-open="dsx029-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a><sup>,</sup><a reveal-id="dsx029-B42" data-open="dsx029-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> In <a reveal-id="dsx029-T1" data-open="dsx029-T1" class="link link-reveal link-table xref-fig">Table 1</a>, we compared the self-cleaving plasmid reporter method to other FACS- and PCR-based methods. Although, the exact numbers for some of the parameters may vary from lab-to-lab making an absolute comparison difficult, considering optimal instances for each, the self-cleaving method presents numerous advantages over the other methods. FACS-based methods are inherently less labor-consuming than PCR based methods. As far as optimization is concerned, here only a new cell line may present a need for optimization, which for our technique refers to adjusting the parameters of a FACS device. However, if the transfection efficiency is also measured when a PCR-based approach is applied (generally and most easily by exploiting a fluorescent protein and FACS), no extra optimization is necessary, in this sense, for our method. By contrast, PCR-based approaches require some optimization of the primers and PCR condition for each target region selected. Thus, it seems to be a favourable alternative approach for those who do not mind the longer time-frame in exchange for spending less effort on gRNA testing. <div content-id="dsx029-T1" class="table-modal"><div class="table-wrap"><div id="dsx029-T1" data-id="dsx029-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Comparison of methods that can detect Cas9 nuclease induced DNA mutations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="3">PCR based methods<hr /> </th><th colspan="3">FACS-based assays<hr /> </th></tr><tr><th>TIDE<sup>33</sup> </th><th>T7E1 </th><th>NGS </th><th>GFP disruption </th><th>EGxxFP<sup>40</sup> </th><th>self-cleaving plasmid </th></tr></thead><tbody><tr><td>Readout time </td><td>2–5 days </td><td>2–3 days </td><td>7–21 days </td><td>3–7 days </td><td>2 days </td><td>6–8 days </td></tr><tr><td>Hands-on time/sample </td><td>3 h </td><td>3 h </td><td>3 h </td><td>40 min </td><td>30 min </td><td>40 min </td></tr><tr><td>Optimization needed </td><td>Yes </td><td>Yes </td><td>Yes </td><td>No </td><td>No </td><td>No </td></tr><tr><td>Genomic context </td><td>Yes </td><td>Yes </td><td>Yes </td><td>Yes </td><td>No </td><td>Yes </td></tr><tr><td>Any target </td><td>Yes </td><td>Yes </td><td>Yes </td><td>GFP targets only </td><td>Yes </td><td>Yes </td></tr><tr><td>Extra cloning steps </td><td>No </td><td>No </td><td>No </td><td>No </td><td>Yes </td><td>No </td></tr><tr><td>Cost/sample </td><td>$25 </td><td>$18 </td><td>$70 </td><td>$7 </td><td>$6 </td><td>$7 </td></tr></tbody></table></div><div content-id="dsx029-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="3">PCR based methods<hr /> </th><th colspan="3">FACS-based assays<hr /> </th></tr><tr><th>TIDE<sup>33</sup> </th><th>T7E1 </th><th>NGS </th><th>GFP disruption </th><th>EGxxFP<sup>40</sup> </th><th>self-cleaving plasmid </th></tr></thead><tbody><tr><td>Readout time </td><td>2–5 days </td><td>2–3 days </td><td>7–21 days </td><td>3–7 days </td><td>2 days </td><td>6–8 days </td></tr><tr><td>Hands-on time/sample </td><td>3 h </td><td>3 h </td><td>3 h </td><td>40 min </td><td>30 min </td><td>40 min </td></tr><tr><td>Optimization needed </td><td>Yes </td><td>Yes </td><td>Yes </td><td>No </td><td>No </td><td>No </td></tr><tr><td>Genomic context </td><td>Yes </td><td>Yes </td><td>Yes </td><td>Yes </td><td>No </td><td>Yes </td></tr><tr><td>Any target </td><td>Yes </td><td>Yes </td><td>Yes </td><td>GFP targets only </td><td>Yes </td><td>Yes </td></tr><tr><td>Extra cloning steps </td><td>No </td><td>No </td><td>No </td><td>No </td><td>Yes </td><td>No </td></tr><tr><td>Cost/sample </td><td>$25 </td><td>$18 </td><td>$70 </td><td>$7 </td><td>$6 </td><td>$7 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The self-cleaving <em>plasmid</em> method is compared to some of the PCR- and FACS-based assays’ most relevant attributes, which can detect Cas9 nuclease induced mutations. The cost and time calculations are available in detail in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_data_dsx029.xlsx?Expires=2147483647&Signature=tvN8l6Nr13c-J7Z5ueuYtmqjQDRRDKGeMZZ4zP2xfTAnHe5~chW-J73UkVFgA2WKJsBA02QbIMp6AMjSQiHSwMi0vHIVHEbihM53zO2GWQvuc4s2FTp1W9zVQUk9sLiwr5wcXiiABvFYaNZHZUtIk4XHVLMRF7WB-X7LD~F1Kvfd7WpSBIrtxhybwx0b1m2jkT6FrEL5~IBFX0ygmG~VV1Mnqudots2x7TJaZbc6aOeoBKvcy9vW8rzF6Vlr8ElemUAIh~IO~SFhI4KvLxWuzc14LBQoyV6OmOXBQL0M1rsXEwQYXq189aSoITXXGt2Lp8RRmBOJ1IL4zERV9~H8WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>. Favourable conditions are marked with white and less favourable conditions are marked with grey backgrounds.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx029-T1" data-id="dsx029-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Comparison of methods that can detect Cas9 nuclease induced DNA mutations</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="3">PCR based methods<hr /> </th><th colspan="3">FACS-based assays<hr /> </th></tr><tr><th>TIDE<sup>33</sup> </th><th>T7E1 </th><th>NGS </th><th>GFP disruption </th><th>EGxxFP<sup>40</sup> </th><th>self-cleaving plasmid </th></tr></thead><tbody><tr><td>Readout time </td><td>2–5 days </td><td>2–3 days </td><td>7–21 days </td><td>3–7 days </td><td>2 days </td><td>6–8 days </td></tr><tr><td>Hands-on time/sample </td><td>3 h </td><td>3 h </td><td>3 h </td><td>40 min </td><td>30 min </td><td>40 min </td></tr><tr><td>Optimization needed </td><td>Yes </td><td>Yes </td><td>Yes </td><td>No </td><td>No </td><td>No </td></tr><tr><td>Genomic context </td><td>Yes </td><td>Yes </td><td>Yes </td><td>Yes </td><td>No </td><td>Yes </td></tr><tr><td>Any target </td><td>Yes </td><td>Yes </td><td>Yes </td><td>GFP targets only </td><td>Yes </td><td>Yes </td></tr><tr><td>Extra cloning steps </td><td>No </td><td>No </td><td>No </td><td>No </td><td>Yes </td><td>No </td></tr><tr><td>Cost/sample </td><td>$25 </td><td>$18 </td><td>$70 </td><td>$7 </td><td>$6 </td><td>$7 </td></tr></tbody></table></div><div content-id="dsx029-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"> </th><th colspan="3">PCR based methods<hr /> </th><th colspan="3">FACS-based assays<hr /> </th></tr><tr><th>TIDE<sup>33</sup> </th><th>T7E1 </th><th>NGS </th><th>GFP disruption </th><th>EGxxFP<sup>40</sup> </th><th>self-cleaving plasmid </th></tr></thead><tbody><tr><td>Readout time </td><td>2–5 days </td><td>2–3 days </td><td>7–21 days </td><td>3–7 days </td><td>2 days </td><td>6–8 days </td></tr><tr><td>Hands-on time/sample </td><td>3 h </td><td>3 h </td><td>3 h </td><td>40 min </td><td>30 min </td><td>40 min </td></tr><tr><td>Optimization needed </td><td>Yes </td><td>Yes </td><td>Yes </td><td>No </td><td>No </td><td>No </td></tr><tr><td>Genomic context </td><td>Yes </td><td>Yes </td><td>Yes </td><td>Yes </td><td>No </td><td>Yes </td></tr><tr><td>Any target </td><td>Yes </td><td>Yes </td><td>Yes </td><td>GFP targets only </td><td>Yes </td><td>Yes </td></tr><tr><td>Extra cloning steps </td><td>No </td><td>No </td><td>No </td><td>No </td><td>Yes </td><td>No </td></tr><tr><td>Cost/sample </td><td>$25 </td><td>$18 </td><td>$70 </td><td>$7 </td><td>$6 </td><td>$7 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The self-cleaving <em>plasmid</em> method is compared to some of the PCR- and FACS-based assays’ most relevant attributes, which can detect Cas9 nuclease induced mutations. The cost and time calculations are available in detail in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_data_dsx029.xlsx?Expires=2147483647&Signature=tvN8l6Nr13c-J7Z5ueuYtmqjQDRRDKGeMZZ4zP2xfTAnHe5~chW-J73UkVFgA2WKJsBA02QbIMp6AMjSQiHSwMi0vHIVHEbihM53zO2GWQvuc4s2FTp1W9zVQUk9sLiwr5wcXiiABvFYaNZHZUtIk4XHVLMRF7WB-X7LD~F1Kvfd7WpSBIrtxhybwx0b1m2jkT6FrEL5~IBFX0ygmG~VV1Mnqudots2x7TJaZbc6aOeoBKvcy9vW8rzF6Vlr8ElemUAIh~IO~SFhI4KvLxWuzc14LBQoyV6OmOXBQL0M1rsXEwQYXq189aSoITXXGt2Lp8RRmBOJ1IL4zERV9~H8WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>. Favourable conditions are marked with white and less favourable conditions are marked with grey backgrounds.</p></div></div></div></div></p><p>Another built-in advantage of the method is that the transfection efficiencies can easily be determined by simply measuring the GFP fluorescence of a portion of each population on the second day post transfection. While the transfection efficiency influences the readout of any gRNA-testing method to a great extent, it is generally not monitored in most of the studies using either PCR- or fluorescence-based methods.<a reveal-id="dsx029-B58" data-open="dsx029-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a><sup>,</sup><a reveal-id="dsx029-B59" data-open="dsx029-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a></p><p>The targeted integration of the GFP-cassette can be assessed only after the transient expression has decayed. The use of an inducible degradation domain fused to GFP decreased this time to 6–8 days, a time-period that is less than what is usually needed for deep sequencing, but it is slightly longer than that required for measuring indel frequency by Surveyor/T7E1 assay (<a reveal-id="dsx029-T1" data-open="dsx029-T1" class="link link-reveal link-table xref-fig">Table 1</a>).<a reveal-id="dsx029-B24" data-open="dsx029-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> The extent of the random integration also seems to vary with the condition of the cells; this makes necessary the use of appropriate negative controls such as non-targeted integration of the in-cell cleaved plasmid, in each experiment.</p><p>The method measures the targeted integration to sites of genomic cleavage identified by the gRNA. Thus, it does not distinguish between on-target and off-target cleavage. However, off-target cleavage can generally be kept low with the careful design of the spacer sequence and using improved SpCas9 nucleases with reduced off-target effect, such as eSpCas9 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>) or SpCas9-HF.<a reveal-id="dsx029-B60" data-open="dsx029-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a><sup>,</sup><a reveal-id="dsx029-B61" data-open="dsx029-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a></p><p>The extent and nature of the correlation between generating double strand breaks and integrating exogenous DNA by NHEJ repair is difficult to assess since there is no easily available method to directly measure the formation of double strand breaks in the cells. Most approaches are based on detecting those outcomes of the repairs of double strand DNA breaks that result in altered sequences.<a reveal-id="dsx029-B24" data-open="dsx029-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> However, the exact correlation between the number of double strand breaks and detectable repair events (i.e. repairs resulting in altered sequences) are not known for any of the approaches designed to monitor it. As such, they depend on many factors, for instance, the actual DNA sequence at the break and the position and presence of microhomologies that are thought to greatly influence the ratio of error-free to non-error-free NHEJ repair.<a reveal-id="dsx029-B62 dsx029-B63 dsx029-B64" data-open="dsx029-B62 dsx029-B63 dsx029-B64" class="link link-ref link-reveal xref-bibr"><sup>62–64</sup></a> The conditions and factors that influence the ratio of HR to NHEJ mediated repair are also difficult to account for.<a reveal-id="dsx029-B25" data-open="dsx029-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> In addition, both main repair systems are capable of capturing foreign DNA at the break points, although the factors that may influence this are also poorly understood, for instance the ratio of NHEJ-mediated integration and indel generating NHEJ repair. In addition, the actual methods used might not accurately measure the frequency of mutations generated by the repair system. For example, it is not fully understood how the distribution and nature of the mutations in the indel population affect the outcome of the Surveyor/T7E1 nucleases assays.<a reveal-id="dsx029-B65" data-open="dsx029-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a><sup>,</sup><a reveal-id="dsx029-B66" data-open="dsx029-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> Furthermore, PCR-based methods generally ignore larger deletions or insertions of which frequencies might also influence the number of smaller indels detected.<a reveal-id="dsx029-B24" data-open="dsx029-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Thus, it is not apparent that the relative efficiencies of spacers would be identical when assessed with two different methods. Nevertheless, all approaches are likely to be suitable for a rough estimation to distinguish very poor from efficient spacers.</p><p>Here, we measured NHEJ mediated and HR mediated targeted integration using the same sets of gRNAs (<a reveal-id="dsx029-F4" data-open="dsx029-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). The gRNAs were designed to minimize the positional effects of the target position in relation to the homologous arms. Although there might be several other factors difficult to account for, this allowed a more direct comparison of the gRNA efficiencies to induce DNA repair by the two repair systems. The method was also compared to both TIDE and T7 endonuclease I assays that measure indel frequencies induced by SpCas9 cleavage employing an identical set of gRNAs. These experiments confirmed that our approach is capable of distinguishing poor and efficient spacer sequences and is a useful, effortless alternative for screening for more efficient gRNAs.</p><p>The method absolutely requires in-cell cleavage of the GFP-plasmid to be integrated. Occasionally, we were able to detect targeted integration of pre-linearized plasmids above the background random integration with some spacers (<a reveal-id="dsx029-F7" data-open="dsx029-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>); however, these were neither reproducible nor robust enough to base a method on. The underlying exact mechanism for the integration of the in-cell cleaved plasmids is not well understood. To distinguish between alternative scenarios, we in-cell re-cleaved the pre-linearized plasmid (<a reveal-id="dsx029-F7" data-open="dsx029-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). These experiments seem to indicate that the major factor for the difference is that circular DNAs are more resistant to cellular DNases than the linear ones are. However, it is still not clear that by increasing the linear DNA pool in the cell, available for integration to the competing double strand breaks, why the in-cell cleaved plasmid is preferred to be integrated (at proportionally higher rates) to nuclease-generated over to endogenous DSBs (<a reveal-id="dsx029-F1" data-open="dsx029-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>) when one compares this to circular or pre-linearized plasmids (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>).</p><p>The key step of the method is to use a self-cleaving plasmid that provides both controlled and standardized in-cell cleavage and enhanced integration of the plasmid. Another important feature lies in the use of targetless, low-background and effective spacers. The former characteristic was achieved by exploring some 30000 spacers and it is necessary for the low background we observed, as well as for general applicability. The latter probably increases the sensitivity of our method, that is in accord with Auer et al. showing that more efficient cleavage of the donor plasmid results in higher targeted integration in zebrafish.<a reveal-id="dsx029-B38" data-open="dsx029-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a></p><p>Our work also demonstrates an efficient way for targeted integration of DNA cassettes in mammalian cells without laborious construction of homologous arms although without base pair precision. Based on earlier studies it was not clear if NHEJ-cloning is also effective in mammalian cells following SpCas9 cleavage that generates blunt ends. In attempts to enhance its efficiency, a few approaches used compatible overhangs or edited microhomology (8 bases-long) to the DSB site.<a reveal-id="dsx029-B43" data-open="dsx029-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a><sup>,</sup><a reveal-id="dsx029-B44" data-open="dsx029-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsx029-B52" data-open="dsx029-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a><sup>,</sup><a reveal-id="dsx029-B67" data-open="dsx029-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> Auer et al., building on the high activity of NHEJ in zebrafish embryos,<a reveal-id="dsx029-B68 dsx029-B69 dsx029-B70" data-open="dsx029-B68 dsx029-B69 dsx029-B70" class="link link-ref link-reveal xref-bibr"><sup>68–70</sup></a> demonstrated high efficiency of NHEJ-cloning without any overhangs or edited microhomology.<a reveal-id="dsx029-B38" data-open="dsx029-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> However, when SpCas9 was used in mammalian cells for NHEJ-cloning of a donor plasmid without any overhangs or edited microhomology, none of the attempts demonstrated highly efficient targeted integration (0.17% and 0.45% or about 1%, in HEK and CHO or HAP1 cells, respectively).<a reveal-id="dsx029-B45" data-open="dsx029-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a><sup>,</sup><a reveal-id="dsx029-B46" data-open="dsx029-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> Here, we successfully demonstrated that NHEJ-cloning coupled with a self-cleaving donor plasmid without any pre-selection or FACS-enrichments can be effectively applied (with up to 20–30% efficiency of integration) to mammalian cells as well.</p><p>A recent study demonstrated targeted integration of expression cassettes amplified by PCR, and reached targeted integration of mCherry-containing cassettes between 1 and 5%.<a reveal-id="dsx029-B59" data-open="dsx029-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a> this is in agreement with our results presented here using linear plasmids (<a reveal-id="dsx029-F7" data-open="dsx029-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S2 and S9</a></span>). Interestingly, using the smaller and brighter protein Clover, about 20% integrations were reported suggesting that transfection of a linear donor DNA less than 2 kilobase, might integrate more efficiently. While it is very promising, in the absence of measuring the actual transfection efficiencies, the widely different amounts of DNA used for transfections of samples and controls, and in the absence of monitoring the decay of the transient fluorescence with an appropriate inactive dCas9 control, it is difficult to assess the real impact of these results. Another work demonstrated homology independent integration, donor DNA knock-in not only in cell culture model systems but <em>in vivo</em> as well.<a reveal-id="dsx029-B71" data-open="dsx029-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a></p><p>NHEJ-cloning is a useful tool for targeted integration when there is no need for absolute precise joining of the DNA ends or when a selection can be applied for the proper joining. Since the DNA fragment can be inserted in two orientations and in three frames, only some of the targeted integration events would result from in frame incorporation of the donor DNA, depending on whether precise NHEJ dominates over error-prone NHEJ or whether the actual sequences prefer certain deletions/insertion over others. Thus, it seems that it is generally not efficient enough for easy protein tagging without selection. Even so, a few studies reported successful application of NHEJ-cloning to tag endogenous proteins using fluorescent protein or other selectable markers.<a reveal-id="dsx029-B46" data-open="dsx029-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a><sup>,</sup><a reveal-id="dsx029-B67" data-open="dsx029-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a><sup>,</sup><a reveal-id="dsx029-B72" data-open="dsx029-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> Nevertheless, it might be more suitable for IRES tagging that is without much sensitivity for base pair-precise insertion and thus, about every second targeted integration (one of the two orientations of the inserted DNA) is appropriate for proper tagging, provided that sufficiently high activity gRNAs are employed. It is important to note that, using only one site for in-cell linearization also results in the incorporation of bacterial, ‘junk’ DNA. In applications where avoiding the integration of ‘junk’ DNA is preferable, a more precise integration of the target gene/DNA cassette can also be achieved by employing the design on <a reveal-id="dsx029-F1" data-open="dsx029-F1" class="link link-reveal link-table xref-fig">Figure 1A</a>, using two cleavage sites as also demonstrated by Nakade et al. and Lackner et al.<a reveal-id="dsx029-B46" data-open="dsx029-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a><sup>,</sup><a reveal-id="dsx029-B72" data-open="dsx029-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> A specifically attractive approach, CRISPaint was developed recently, by which both the integration of the plasmid-cleaving gRNA into the target genomic site and, by applying DNA minicircles, the insertion of bacterial DNA can be avoided.<a reveal-id="dsx029-B73" data-open="dsx029-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a></p><p>NHEJ-cloning is especially attractive for its potential as a superior substitute for random integration, to incorporate DNA cassettes to safe-harbour sites in the genome—avoiding the positional effects of the insertion—frequently encountered using random integration and the laborious work for constructing homologous arms. It can be more effective by at least one order of magnitude than random integration and provides stronger coupling between target and marker genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>), facilitating a more reliable identification of the proper clones. Another attractive application of it is the creation of loss-of-function alleles by the easy targeting of DNA at endogenous loci. Using self-cleaving plasmids to disrupt genes is comparably effective to out-of-frame indel-generation methods with the added advantage of the invalidated alleles being labeled.</p>                            <h2 scrollto-destination=104086438 id="104086438" class="section-title" >5. Conclusion</h2>
<p>We demonstrated a versatile, yet simple approach for the identification of the effective gRNAs using a fluorescent reporter assay with self-cleaving plasmids. The method is capable of testing targets in their genomic context and requires no extra cloning steps. While it needs about 4 days longer to get the results, the method requires considerably less hands-on time and it certainly will be favoured by researchers who wish to avoid the use of the ‘gold-standard’ T7/Surveyor assays. This method might generally be used for other RNA-guided designer nucleases as well, such as Cas9-s from other species and Cpf1 nucleases. Additionally, we showed that NHEJ-cloning can also be used in mammalian cells for targeted integration of donor plasmids up to 10 kb in size, with up to 30% efficiency, without any selection or enrichment. Its further advantages as compared to random integrations include being much more effective and being able to provide stronger coupling of two DNA cassettes, such as the one to be integrated and a selection marker.</p>                            <h2 scrollto-destination=104086440 id="104086440" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We would like to thank György Várady and Izabel Patik for helping with the FACS measurements, Lucia Illy, Judit Tálas and Zsófia Molnár for critical reading and suggestions to the manuscript and Ildikó Pulinka Szűcsné, Judit Szűcs and Luca Xénia Turgyán for technical assistance. This work was supported by the Hungarian Scientific Research Fund [K-82090, PD-111964].</p>                            <h3 scrollto-destination=104086442 id="104086442" class="section-title" >Competing interests</h3>
<p>The authors declare no competing interests.</p>                            <h3 scrollto-destination=104086444 id="104086444" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=104086446 id="104086446" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at DNARES online.</p>                            <h3 scrollto-destination=104086448 id="104086448" class="section-title" >Funding</h3>
<p>This work was supported by the Hungarian Scientific Research Fund [K-82090, PD-111964].</p>                            <h3 scrollto-destination=104086450 id="104086450" class="section-title" >Availability</h3>
<p>The self-cleaving plasmids used in this study are available at Addgene: pSc1 (#80436), pSc2 (#80437), pSc1-puro (#80438) and pSc1-DD (#80439). Sequences of all other constructs are listed in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_data_dsx029.xlsx?Expires=2147483647&Signature=tvN8l6Nr13c-J7Z5ueuYtmqjQDRRDKGeMZZ4zP2xfTAnHe5~chW-J73UkVFgA2WKJsBA02QbIMp6AMjSQiHSwMi0vHIVHEbihM53zO2GWQvuc4s2FTp1W9zVQUk9sLiwr5wcXiiABvFYaNZHZUtIk4XHVLMRF7WB-X7LD~F1Kvfd7WpSBIrtxhybwx0b1m2jkT6FrEL5~IBFX0ygmG~VV1Mnqudots2x7TJaZbc6aOeoBKvcy9vW8rzF6Vlr8ElemUAIh~IO~SFhI4KvLxWuzc14LBQoyV6OmOXBQL0M1rsXEwQYXq189aSoITXXGt2Lp8RRmBOJ1IL4zERV9~H8WA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span> and are available upon request.</p>                            <h3 scrollto-destination=104086452 id="104086452" class="section-title" >Author’s contributions</h3>
<p>A.T. and E.W. conceived and designed the experiments, A.T., P.I.K., N.W., A.B, K.S., E.T., S.L.K., B.G., A.S., O.I.H, Z.L., performed the experiments and all authors analysed the data. A. T., E. F. and E.W. wrote the manuscript.</p>                            <h2 scrollto-destination=104086454 id="104086454" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx029-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jinek</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Chylinski</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Fonfara</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Hauer</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Doudna</div> <div class="given-names">J. A.</div></div>, <div class="name string-name"><div class="surname">Charpentier</div> <div class="given-names">E.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">A programmable dual-RNA-guided DNA endonuclease in adaptive bacterial immunity</div>. <div class="source">Science</div> , <div class="volume">337</div>, <div class="fpage">816</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20programmable%20dual-RNA-guided%20DNA%20endonuclease%20in%20adaptive%20bacterial%20immunity&amp;author=M.%20Jinek&amp;author=K.%20Chylinski&amp;author=I.%20Fonfara&amp;author=M.%20Hauer&amp;author=J.%20A.%20Doudna&amp;author=E.%20Charpentier&amp;publication_year=2012&amp;journal=Science&amp;volume=337&amp;pages=816-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1225829" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1225829" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1225829"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22745249" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mali</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Esvelt</div> <div class="given-names">K. M.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">RNA-guided human genome engineering via Cas9</div>. <div class="source">Science</div> , <div class="volume">339</div>, <div class="fpage">823</div>–<div class="lpage">26</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-guided%20human%20genome%20engineering%20via%20Cas9&amp;author=P.%20Mali&amp;author=L.%20Yang&amp;author=K.%20M.%20Esvelt&amp;publication_year=2013&amp;journal=Science&amp;volume=339&amp;pages=823-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1232033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1232033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1232033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23287722" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cong</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Ran</div> <div class="given-names">F. A.</div></div>, <div class="name string-name"><div class="surname">Cox</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Multiplex genome engineering using CRISPR/Cas systems</div>. <div class="source">Science</div> , <div class="volume">339</div>, <div class="fpage">819</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiplex%20genome%20engineering%20using%20CRISPR%2FCas%20systems&amp;author=L.%20Cong&amp;author=F.%20A.%20Ran&amp;author=D.%20Cox&amp;publication_year=2013&amp;journal=Science&amp;volume=339&amp;pages=819-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1231143" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1231143" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1231143"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23287718" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jinek</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">East</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Doudna</div> <div class="given-names">J.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">RNA-programmed genome editing in human cells</div>. <div class="source">Elife</div> , <div class="volume">2</div>, <div class="fpage">e00471</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-programmed%20genome%20editing%20in%20human%20cells&amp;author=M.%20Jinek&amp;author=A.%20East&amp;author=A.%20Cheng&amp;author=S.%20Lin&amp;author=E.%20Ma&amp;author=J.%20Doudna&amp;publication_year=2013&amp;journal=Elife&amp;volume=2&amp;pages=e00471" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7554/eLife.00471" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7554%2FeLife.00471" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7554%2FeLife.00471"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23386978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gilbert</div> <div class="given-names">L. A.</div></div>, <div class="name string-name"><div class="surname">Larson</div> <div class="given-names">M. H.</div></div>, <div class="name string-name"><div class="surname">Morsut</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">CRISPR-mediated modular RNA-guided regulation of transcription in eukaryotes</div>. <div class="source">Cell</div> , <div class="volume">154</div>, <div class="fpage">442</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPR-mediated%20modular%20RNA-guided%20regulation%20of%20transcription%20in%20eukaryotes&amp;author=L.%20A.%20Gilbert&amp;author=M.%20H.%20Larson&amp;author=L.%20Morsut&amp;publication_year=2013&amp;journal=Cell&amp;volume=154&amp;pages=442-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2013.06.044" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2013.06.044" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2013.06.044"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23849981" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shalem</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Sanjana</div> <div class="given-names">N. E.</div></div>, <div class="name string-name"><div class="surname">Hartenian</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Genome-scale CRISPR-Cas9 knockout screening in human cells</div>. <div class="source">Science</div> , <div class="volume">343</div>, <div class="fpage">84</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-scale%20CRISPR-Cas9%20knockout%20screening%20in%20human%20cells&amp;author=O.%20Shalem&amp;author=N.%20E.%20Sanjana&amp;author=E.%20Hartenian&amp;publication_year=2014&amp;journal=Science&amp;volume=343&amp;pages=84-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1247005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1247005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1247005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24336571" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zalatan</div> <div class="given-names">J. G.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">M. E.</div></div>, <div class="name string-name"><div class="surname">Almeida</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Engineering complex synthetic transcriptional programs with CRISPR RNA scaffolds</div>. <div class="source">Cell</div> , <div class="volume">160</div>, <div class="fpage">339</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Engineering%20complex%20synthetic%20transcriptional%20programs%20with%20CRISPR%20RNA%20scaffolds&amp;author=J.%20G.%20Zalatan&amp;author=M.%20E.%20Lee&amp;author=R.%20Almeida&amp;publication_year=2015&amp;journal=Cell&amp;volume=160&amp;pages=339-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2014.11.052" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2014.11.052" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2014.11.052"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25533786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gilbert</div> <div class="given-names">L. A.</div></div>, <div class="name string-name"><div class="surname">Horlbeck</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Adamson</div> <div class="given-names">B.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Genome-scale CRISPR-mediated control of gene repression and activation</div>. <div class="source">Cell</div> , <div class="volume">159</div>, <div class="fpage">647</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-scale%20CRISPR-mediated%20control%20of%20gene%20repression%20and%20activation&amp;author=L.%20A.%20Gilbert&amp;author=M.%20A.%20Horlbeck&amp;author=B.%20Adamson&amp;publication_year=2014&amp;journal=Cell&amp;volume=159&amp;pages=647-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2014.09.029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2014.09.029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2014.09.029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25307932" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Qi</div> <div class="given-names">L. S.</div></div>, <div class="name string-name"><div class="surname">Larson</div> <div class="given-names">M. H.</div></div>, <div class="name string-name"><div class="surname">Gilbert</div> <div class="given-names">L. A.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Repurposing CRISPR as an RNA-guided platform for sequence-specific control of gene expression</div>. <div class="source">Cell</div> , <div class="volume">152</div>, <div class="fpage">1173</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Repurposing%20CRISPR%20as%20an%20RNA-guided%20platform%20for%20sequence-specific%20control%20of%20gene%20expression&amp;author=L.%20S.%20Qi&amp;author=M.%20H.%20Larson&amp;author=L.%20A.%20Gilbert&amp;publication_year=2013&amp;journal=Cell&amp;volume=152&amp;pages=1173-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2013.02.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2013.02.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2013.02.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23452860" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ramakrishna</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Dad</div> <div class="given-names">A.-B. K.</div></div>, <div class="name string-name"><div class="surname">Beloor</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Gopalappa</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">S.-K.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">H.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Gene disruption by cell-penetrating peptide-mediated delivery of Cas9 protein and guide RNA</div>. <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">1020</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20disruption%20by%20cell-penetrating%20peptide-mediated%20delivery%20of%20Cas9%20protein%20and%20guide%20RNA&amp;author=S.%20Ramakrishna&amp;author=A.-B.%20K.%20Dad&amp;author=J.%20Beloor&amp;author=R.%20Gopalappa&amp;author=S.-K.%20Lee&amp;author=H.%20Kim&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1020-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.171264.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.171264.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.171264.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24696462" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mali</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Aach</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Stranges</div> <div class="given-names">P. B.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">CAS9 transcriptional activators for target specificity screening and paired nickases for cooperative genome engineering</div>. <div class="source">Nat. Biotechnol</div> ., <div class="volume">31</div>, <div class="fpage">833</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CAS9%20transcriptional%20activators%20for%20target%20specificity%20screening%20and%20paired%20nickases%20for%20cooperative%20genome%20engineering&amp;author=P.%20Mali&amp;author=J.%20Aach&amp;author=P.%20B.%20Stranges&amp;publication_year=2013&amp;journal=Nat.%20Biotechnol&amp;volume=31&amp;pages=833-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2675" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2675" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2675"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23907171" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ran</div> <div class="given-names">F. A.</div></div>, <div class="name string-name"><div class="surname">Hsu</div> <div class="given-names">P. D.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">C.-Y.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Double nicking by RNA-guided CRISPR Cas9 for enhanced genome editing specificity</div>. <div class="source">Cell</div> , <div class="volume">154</div>, <div class="fpage">1380</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Double%20nicking%20by%20RNA-guided%20CRISPR%20Cas9%20for%20enhanced%20genome%20editing%20specificity&amp;author=F.%20A.%20Ran&amp;author=P.%20D.%20Hsu&amp;author=C.-Y.%20Lin&amp;publication_year=2013&amp;journal=Cell&amp;volume=154&amp;pages=1380-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2013.08.021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2013.08.021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2013.08.021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23992846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dow</div> <div class="given-names">L. E.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Modeling Disease In Vivo With CRISPR/Cas9</div>. <div class="source">Trends Mol. Med</div> ., <div class="volume">21</div>, <div class="fpage">609</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modeling%20Disease%20In%20Vivo%20With%20CRISPR%2FCas9&amp;author=L.%20E.%20Dow&amp;publication_year=2015&amp;journal=Trends%20Mol.%20Med&amp;volume=21&amp;pages=609-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molmed.2015.07.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molmed.2015.07.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molmed.2015.07.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26432018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">J.-S.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">A guide to genome engineering with programmable nucleases</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">15</div>, <div class="fpage">321</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20guide%20to%20genome%20engineering%20with%20programmable%20nucleases&amp;author=H.%20Kim&amp;author=J.-S.%20Kim&amp;publication_year=2014&amp;journal=Nat.%20Rev.%20Genet&amp;volume=15&amp;pages=321-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3686" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3686" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3686"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24690881" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ramakrishna</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Cho</div> <div class="given-names">S. W.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Surrogate reporter-based enrichment of cells containing RNA-guided Cas9 nuclease-induced mutations</div>. <div class="source">Nat. Commun</div> ., <div class="volume">5</div>, <div class="fpage">3378</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Surrogate%20reporter-based%20enrichment%20of%20cells%20containing%20RNA-guided%20Cas9%20nuclease-induced%20mutations&amp;author=S.%20Ramakrishna&amp;author=S.%20W.%20Cho&amp;author=S.%20Kim&amp;publication_year=2014&amp;journal=Nat.%20Commun&amp;volume=5&amp;pages=3378" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms4378" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms4378" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms4378"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24569644" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Doench</div> <div class="given-names">J. G.</div></div>, <div class="name string-name"><div class="surname">Hartenian</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Graham</div> <div class="given-names">D. B.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Rational design of highly active sgRNAs for CRISPR-Cas9-mediated gene inactivation</div>. <div class="source">Nat. Biotechnol</div>  <div class="volume">32</div>, <div class="fpage">1262</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rational%20design%20of%20highly%20active%20sgRNAs%20for%20CRISPR-Cas9-mediated%20gene%20inactivation&amp;author=J.%20G.%20Doench&amp;author=E.%20Hartenian&amp;author=D.%20B.%20Graham&amp;publication_year=2014&amp;journal=Nat.%20Biotechnol&amp;volume=32&amp;pages=1262-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25184501" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Wei</div> <div class="given-names">J. J.</div></div>, <div class="name string-name"><div class="surname">Sabatini</div> <div class="given-names">D. M.</div></div>, <div class="name string-name"><div class="surname">Lander</div> <div class="given-names">E. S.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Genetic screens in human cells using the CRISPR-Cas9 system</div>. <div class="source">Science</div> , <div class="volume">343</div>, <div class="fpage">80</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20screens%20in%20human%20cells%20using%20the%20CRISPR-Cas9%20system&amp;author=T.%20Wang&amp;author=J.%20J.%20Wei&amp;author=D.%20M.%20Sabatini&amp;author=E.%20S.%20Lander&amp;publication_year=2014&amp;journal=Science&amp;volume=343&amp;pages=80-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1246981" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1246981" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1246981"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24336569" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chari</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Mali</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Moosburner</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Church</div> <div class="given-names">G. M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Unraveling CRISPR-Cas9 genome engineering parameters via a library-on-library approach</div>. <div class="source">Nat. Methods</div> , <div class="volume">12</div>, <div class="fpage">823</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unraveling%20CRISPR-Cas9%20genome%20engineering%20parameters%20via%20a%20library-on-library%20approach&amp;author=R.%20Chari&amp;author=P.%20Mali&amp;author=M.%20Moosburner&amp;author=G.%20M.%20Church&amp;publication_year=2015&amp;journal=Nat.%20Methods&amp;volume=12&amp;pages=823-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.3473" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.3473" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.3473"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26167643" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Moreno-Mateos</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Vejnar</div> <div class="given-names">C. E.</div></div>, <div class="name string-name"><div class="surname">Beaudoin</div> <div class="given-names">J.-D.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">CRISPRscan: designing highly efficient sgRNAs for CRISPR-Cas9 targeting in vivo</div>. <div class="source">Nat. Methods</div> , <div class="volume">12</div>, <div class="fpage">982</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPRscan%3A%20designing%20highly%20efficient%20sgRNAs%20for%20CRISPR-Cas9%20targeting%20in%20vivo&amp;author=M.%20A.%20Moreno-Mateos&amp;author=C.%20E.%20Vejnar&amp;author=J.-D.%20Beaudoin&amp;publication_year=2015&amp;journal=Nat.%20Methods&amp;volume=12&amp;pages=982-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.3543" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.3543" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.3543"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26322839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wong</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">WU-CRISPR: characteristics of functional guide RNAs for the CRISPR/Cas9 system</div>. <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">1</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WU-CRISPR%3A%20characteristics%20of%20functional%20guide%20RNAs%20for%20the%20CRISPR%2FCas9%20system&amp;author=N.%20Wong&amp;author=W.%20Liu&amp;author=X.%20Wang&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=1-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0784-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0784-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0784-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25583448" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Doench</div> <div class="given-names">J. G.</div></div>, <div class="name string-name"><div class="surname">Fusi</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Sullender</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Optimized sgRNA design to maximize activity and minimize off-target effects of CRISPR-Cas9</div>. <div class="source">Nat. Biotechnol</div>  <div class="volume">34</div>, <div class="fpage">184</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimized%20sgRNA%20design%20to%20maximize%20activity%20and%20minimize%20off-target%20effects%20of%20CRISPR-Cas9&amp;author=J.%20G.%20Doench&amp;author=N.%20Fusi&amp;author=M.%20Sullender&amp;publication_year=2016&amp;journal=Nat.%20Biotechnol&amp;volume=34&amp;pages=184-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3437" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3437" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3437"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26780180" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hinz</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Laughery</div> <div class="given-names">M. F.</div></div>, <div class="name string-name"><div class="surname">Wyrick</div> <div class="given-names">J. J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Nucleosomes inhibit Cas9 endonuclease activity in vitro</div>. <div class="source">Biochemistry</div> , <div class="volume">54</div>, <div class="fpage">7063</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleosomes%20inhibit%20Cas9%20endonuclease%20activity%20in%20vitro&amp;author=J.%20M.%20Hinz&amp;author=M.%20F.%20Laughery&amp;author=J.%20J.%20Wyrick&amp;publication_year=2015&amp;journal=Biochemistry&amp;volume=54&amp;pages=7063-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/acs.biochem.5b01108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Facs.biochem.5b01108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Facs.biochem.5b01108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26579937" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Horlbeck</div> <div class="given-names">M. A.</div></div>, <div class="name string-name"><div class="surname">Witkowsky</div> <div class="given-names">L. B.</div></div>, <div class="name string-name"><div class="surname">Guglielmi</div> <div class="given-names">B.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Nucleosomes impede Cas9 access to DNA in vivo and in vitro</div>. <div class="source">Elife</div> , <div class="volume">5</div>, <div class="fpage">e12677</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleosomes%20impede%20Cas9%20access%20to%20DNA%20in%20vivo%20and%20in%20vitro&amp;author=M.%20A.%20Horlbeck&amp;author=L.%20B.%20Witkowsky&amp;author=B.%20Guglielmi&amp;publication_year=2016&amp;journal=Elife&amp;volume=5&amp;pages=e12677" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26987018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hendel</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Fine</div> <div class="given-names">E. J.</div></div>, <div class="name string-name"><div class="surname">Bao</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Porteus</div> <div class="given-names">M. H.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Quantifying on-and off-target genome editing</div>. <div class="source">Trends Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">132</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantifying%20on-and%20off-target%20genome%20editing&amp;author=A.%20Hendel&amp;author=E.%20J.%20Fine&amp;author=G.%20Bao&amp;author=M.%20H.%20Porteus&amp;publication_year=2015&amp;journal=Trends%20Biotechnol&amp;volume=33&amp;pages=132-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2014.12.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2014.12.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2014.12.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25595557" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chapman</div> <div class="given-names">K. M.</div></div>, <div class="name string-name"><div class="surname">Medrano</div> <div class="given-names">G. A.</div></div>, <div class="name string-name"><div class="surname">Jaichander</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Targeted germline modifications in rats using CRISPR/Cas9 and spermatogonial stem cells</div>. <div class="source">Cell Rep</div> ., <div class="volume">10</div>, <div class="fpage">1828</div>–<div class="lpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20germline%20modifications%20in%20rats%20using%20CRISPR%2FCas9%20and%20spermatogonial%20stem%20cells&amp;author=K.%20M.%20Chapman&amp;author=G.%20A.%20Medrano&amp;author=P.%20Jaichander&amp;publication_year=2015&amp;journal=Cell%20Rep&amp;volume=10&amp;pages=1828-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.celrep.2015.02.040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.celrep.2015.02.040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.celrep.2015.02.040"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25772367" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dow</div> <div class="given-names">L. E.</div></div>, <div class="name string-name"><div class="surname">Fisher</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">O'Rourke</div> <div class="given-names">K. P.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Inducible in vivo genome editing with CRISPR-Cas9</div>. <div class="source">Nature Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">390</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Inducible%20in%20vivo%20genome%20editing%20with%20CRISPR-Cas9&amp;author=L.%20E.%20Dow&amp;author=J.%20Fisher&amp;author=K.%20P.%20O'Rourke&amp;publication_year=2015&amp;journal=Nature%20Biotechnol&amp;volume=33&amp;pages=390-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3155" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3155" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3155"> </span></div> </div></div></div></div></div><div content-id="dsx029-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ran</div> <div class="given-names">F. A.</div></div>, <div class="name string-name"><div class="surname">Hsu</div> <div class="given-names">P. D.</div></div>, <div class="name string-name"><div class="surname">Wright</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Agarwala</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Scott</div> <div class="given-names">D. A.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">F.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Genome engineering using the CRISPR-Cas9 system</div>. <div class="source">Nat. Protoc</div> ., <div class="volume">8</div>, <div class="fpage">2281</div>–<div class="lpage">308</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20engineering%20using%20the%20CRISPR-Cas9%20system&amp;author=F.%20A.%20Ran&amp;author=P.%20D.%20Hsu&amp;author=J.%20Wright&amp;author=V.%20Agarwala&amp;author=D.%20A.%20Scott&amp;author=F.%20Zhang&amp;publication_year=2013&amp;journal=Nat.%20Protoc&amp;volume=8&amp;pages=2281-308" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2013.143" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2013.143" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2013.143"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24157548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cho</div> <div class="given-names">S. W.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">J.-S.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Targeted genome engineering in human cells with the Cas9 RNA-guided endonuclease</div>. <div class="source">Nat. Biotechnol</div> ., <div class="volume">31</div>, <div class="fpage">230</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeted%20genome%20engineering%20in%20human%20cells%20with%20the%20Cas9%20RNA-guided%20endonuclease&amp;author=S.%20W.%20Cho&amp;author=S.%20Kim&amp;author=J.%20M.%20Kim&amp;author=J.-S.%20Kim&amp;publication_year=2013&amp;journal=Nat.%20Biotechnol&amp;volume=31&amp;pages=230-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2507" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2507" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2507"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23360966" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Qiu</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Shandilya</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">D Alessio</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">O Connor</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Durocher</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Gerard</div> <div class="given-names">G. F.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Mutation detection using Surveyor™ nuclease</div>. <div class="source">Biotechniques</div> , <div class="volume">36</div>, <div class="fpage">702</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation%20detection%20using%20Surveyor%E2%84%A2%20nuclease&amp;author=P.%20Qiu&amp;author=H.%20Shandilya&amp;author=J.%20M.%20D%20Alessio&amp;author=K.%20O%20Connor&amp;author=J.%20Durocher&amp;author=G.%20F.%20Gerard&amp;publication_year=2004&amp;journal=Biotechniques&amp;volume=36&amp;pages=702-7" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15088388" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dahlem</div> <div class="given-names">T. J.</div></div>, <div class="name string-name"><div class="surname">Hoshijima</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Jurynec</div> <div class="given-names">M. J.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Simple methods for generating and detecting locus-specific mutations induced with TALENs in the zebrafish genome</div>. <div class="source">PLoS Genet</div> ., <div class="volume">8</div>, <div class="fpage">e1002861</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Simple%20methods%20for%20generating%20and%20detecting%20locus-specific%20mutations%20induced%20with%20TALENs%20in%20the%20zebrafish%20genome&amp;author=T.%20J.%20Dahlem&amp;author=K.%20Hoshijima&amp;author=M.%20J.%20Jurynec&amp;publication_year=2012&amp;journal=PLoS%20Genet&amp;volume=8&amp;pages=e1002861" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1002861" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1002861" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1002861"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22916025" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Unbiased detection of off-target cleavage by CRISPR-Cas9 and TALENs using integrase-defective lentiviral vectors</div>. <div class="source">Nat. Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">175</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unbiased%20detection%20of%20off-target%20cleavage%20by%20CRISPR-Cas9%20and%20TALENs%20using%20integrase-defective%20lentiviral%20vectors&amp;author=X.%20Wang&amp;author=Y.%20Wang&amp;author=X.%20Wu&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=175-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25599175" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ramlee</div> <div class="given-names">M. K.</div></div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Cheung</div> <div class="given-names">A. M.</div></div>, <div class="name string-name"><div class="surname">Chuah</div> <div class="given-names">C. T.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">S.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">High-throughput genotyping of CRISPR/Cas9-mediated mutants using fluorescent PCR-capillary gel electrophoresis</div>. <div class="source">Sci. Rep</div> ., <div class="volume">5</div>, <div class="fpage">15587</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20genotyping%20of%20CRISPR%2FCas9-mediated%20mutants%20using%20fluorescent%20PCR-capillary%20gel%20electrophoresis&amp;author=M.%20K.%20Ramlee&amp;author=T.%20Yan&amp;author=A.%20M.%20Cheung&amp;author=C.%20T.%20Chuah&amp;author=S.%20Li&amp;publication_year=2015&amp;journal=Sci.%20Rep&amp;volume=5&amp;pages=15587" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep15587" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep15587" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep15587"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26498861" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brinkman</div> <div class="given-names">E. K.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Amendola</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">van Steensel</div> <div class="given-names">B.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Easy quantitative assessment of genome editing by sequence trace decomposition</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">e168</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Easy%20quantitative%20assessment%20of%20genome%20editing%20by%20sequence%20trace%20decomposition&amp;author=E.%20K.%20Brinkman&amp;author=T.%20Chen&amp;author=M.%20Amendola&amp;author=B.%20van%20Steensel&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=e168" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku936" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku936" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku936"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25300484" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Guell</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Byrne</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Optimization of scarless human stem cell genome editing</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">9049</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimization%20of%20scarless%20human%20stem%20cell%20genome%20editing&amp;author=L.%20Yang&amp;author=M.%20Guell&amp;author=S.%20Byrne&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=9049-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt555" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt555" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt555"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23907390" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Urnov</div> <div class="given-names">F. D.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">J. C.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">Y.-L.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Highly efficient endogenous human gene correction using designed zinc-finger nucleases</div>. <div class="source">Nature</div> , <div class="volume">435</div>, <div class="fpage">646</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20efficient%20endogenous%20human%20gene%20correction%20using%20designed%20zinc-finger%20nucleases&amp;author=F.%20D.%20Urnov&amp;author=J.%20C.%20Miller&amp;author=Y.-L.%20Lee&amp;publication_year=2005&amp;journal=Nature&amp;volume=435&amp;pages=646-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15806097" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wagner</div> <div class="given-names">J. C.</div></div>, <div class="name string-name"><div class="surname">Platt</div> <div class="given-names">R. J.</div></div>, <div class="name string-name"><div class="surname">Goldfless</div> <div class="given-names">S. J.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Niles</div> <div class="given-names">J. C.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Efficient CRISPR-Cas9-mediated genome editing in <em>Plasmodium falciparum</em></div>. <div class="source">Nat. Methods</div>  <div class="volume">11</div>, <div class="fpage">915</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20CRISPR-Cas9-mediated%20genome%20editing%20in%20Plasmodium%20falciparum&amp;author=J.%20C.%20Wagner&amp;author=R.%20J.%20Platt&amp;author=S.%20J.%20Goldfless&amp;author=F.%20Zhang&amp;author=J.%20C.%20Niles&amp;publication_year=2014&amp;journal=Nat.%20Methods&amp;volume=11&amp;pages=915-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.3063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.3063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.3063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25108687" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Shivalila</div> <div class="given-names">C. S.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">One-step generation of mice carrying mutations in multiple genes by CRISPR/Cas-mediated genome engineering</div>. <div class="source">Cell</div> , <div class="volume">153</div>, <div class="fpage">910</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=One-step%20generation%20of%20mice%20carrying%20mutations%20in%20multiple%20genes%20by%20CRISPR%2FCas-mediated%20genome%20engineering&amp;author=H.%20Wang&amp;author=H.%20Yang&amp;author=C.%20S.%20Shivalila&amp;publication_year=2013&amp;journal=Cell&amp;volume=153&amp;pages=910-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2013.04.025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2013.04.025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2013.04.025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23643243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Auer</div> <div class="given-names">T. O.</div></div>, <div class="name string-name"><div class="surname">Duroure</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">De Cian</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Concordet</div> <div class="given-names">J.-P.</div></div>, <div class="name string-name"><div class="surname">Del Bene</div> <div class="given-names">F.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Highly efficient CRISPR/Cas9-mediated knock-in in zebrafish by homology-independent DNA repair</div>. <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">142</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Highly%20efficient%20CRISPR%2FCas9-mediated%20knock-in%20in%20zebrafish%20by%20homology-independent%20DNA%20repair&amp;author=T.%20O.%20Auer&amp;author=K.%20Duroure&amp;author=A.%20De%20Cian&amp;author=J.-P.%20Concordet&amp;author=F.%20Del%20Bene&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=142-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.161638.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.161638.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.161638.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24179142" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Certo</div> <div class="given-names">M. T.</div></div>, <div class="name string-name"><div class="surname">Ryu</div> <div class="given-names">B. Y.</div></div>, <div class="name string-name"><div class="surname">Annis</div> <div class="given-names">J. E.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Tracking genome engineering outcome at individual DNA breakpoints</div>. <div class="source">Nat. Methods</div> , <div class="volume">8</div>, <div class="fpage">671</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tracking%20genome%20engineering%20outcome%20at%20individual%20DNA%20breakpoints&amp;author=M.%20T.%20Certo&amp;author=B.%20Y.%20Ryu&amp;author=J.%20E.%20Annis&amp;publication_year=2011&amp;journal=Nat.%20Methods&amp;volume=8&amp;pages=671-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1648" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1648" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1648"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21743461" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mashiko</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Fujihara</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Satouh</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Miyata</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Isotani</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Ikawa</div> <div class="given-names">M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Generation of mutant mice by pronuclear injection of circular plasmid expressing Cas9 and single guided RNA</div>. <div class="source">Sci. Rep</div> ., <div class="volume">3</div>, <div class="fpage">3355</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20of%20mutant%20mice%20by%20pronuclear%20injection%20of%20circular%20plasmid%20expressing%20Cas9%20and%20single%20guided%20RNA&amp;author=D.%20Mashiko&amp;author=Y.%20Fujihara&amp;author=Y.%20Satouh&amp;author=H.%20Miyata&amp;author=A.%20Isotani&amp;author=M.%20Ikawa&amp;publication_year=2013&amp;journal=Sci.%20Rep&amp;volume=3&amp;pages=3355" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep03355" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep03355" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep03355"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24284873" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hockemeyer</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Soldner</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Beard</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Efficient targeting of expressed and silent genes in human ESCs and iPSCs using zinc-finger nucleases</div>. <div class="source">Nat. Biotechnol</div> ., <div class="volume">27</div>, <div class="fpage">851</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20targeting%20of%20expressed%20and%20silent%20genes%20in%20human%20ESCs%20and%20iPSCs%20using%20zinc-finger%20nucleases&amp;author=D.%20Hockemeyer&amp;author=F.%20Soldner&amp;author=C.%20Beard&amp;publication_year=2009&amp;journal=Nat.%20Biotechnol&amp;volume=27&amp;pages=851-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1562" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1562" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1562"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19680244" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kim</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Um</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Cho</div> <div class="given-names">S.-R.</div></div>, <div class="name string-name"><div class="surname">Jung</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">J.-S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Surrogate reporters for enrichment of cells with nuclease-induced mutations</div>. <div class="source">Nat. Methods</div> , <div class="volume">8</div>, <div class="fpage">941</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Surrogate%20reporters%20for%20enrichment%20of%20cells%20with%20nuclease-induced%20mutations&amp;author=H.%20Kim&amp;author=E.%20Um&amp;author=S.-R.%20Cho&amp;author=C.%20Jung&amp;author=H.%20Kim&amp;author=J.-S.%20Kim&amp;publication_year=2011&amp;journal=Nat.%20Methods&amp;volume=8&amp;pages=941-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1733" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1733" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1733"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21983922" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Orlando</div> <div class="given-names">S. J.</div></div>, <div class="name string-name"><div class="surname">Santiago</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">DeKelver</div> <div class="given-names">R. C.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Zinc-finger nuclease-driven targeted integration into mammalian genomes using donors with limited chromosomal homology</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">38</div>, <div class="fpage">e152</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Zinc-finger%20nuclease-driven%20targeted%20integration%20into%20mammalian%20genomes%20using%20donors%20with%20limited%20chromosomal%20homology&amp;author=S.%20J.%20Orlando&amp;author=Y.%20Santiago&amp;author=R.%20C.%20DeKelver&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res&amp;volume=38&amp;pages=e152" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq512" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq512" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq512"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20530528" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cristea</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Freyvert</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Santiago</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">In vivo cleavage of transgene donors promotes nuclease‐mediated targeted integration</div>. <div class="source">Biotechnol. Bioeng</div> ., <div class="volume">110</div>, <div class="fpage">871</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vivo%20cleavage%20of%20transgene%20donors%20promotes%20nuclease%E2%80%90mediated%20targeted%20integration&amp;author=S.%20Cristea&amp;author=Y.%20Freyvert&amp;author=Y.%20Santiago&amp;publication_year=2013&amp;journal=Biotechnol.%20Bioeng&amp;volume=110&amp;pages=871-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/bit.24733" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fbit.24733" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fbit.24733"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23042119" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bachu</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Bergareche</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Chasin</div> <div class="given-names">L. A.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">CRISPR‐Cas targeted plasmid integration into mammalian cells via non‐homologous end joining</div>. <div class="source">Biotechnol. Bioeng</div>  <div class="volume">112</div>, <div class="fpage">2154</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPR%E2%80%90Cas%20targeted%20plasmid%20integration%20into%20mammalian%20cells%20via%20non%E2%80%90homologous%20end%20joining&amp;author=R.%20Bachu&amp;author=I.%20Bergareche&amp;author=L.%20A.%20Chasin&amp;publication_year=2015&amp;journal=Biotechnol.%20Bioeng&amp;volume=112&amp;pages=2154-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/bit.v112.10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fbit.v112.10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fbit.v112.10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25943095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lackner</div> <div class="given-names">D. H.</div></div>, <div class="name string-name"><div class="surname">Carré</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Guzzardo</div> <div class="given-names">P. M.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">A generic strategy for CRISPR-Cas9-mediated gene tagging</div>. <div class="source">Nat. Commun</div> ., <div class="volume">6</div>, <div class="fpage">10237</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20generic%20strategy%20for%20CRISPR-Cas9-mediated%20gene%20tagging&amp;author=D.%20H.%20Lackner&amp;author=A.%20Carr%C3%A9&amp;author=P.%20M.%20Guzzardo&amp;publication_year=2015&amp;journal=Nat.%20Commun&amp;volume=6&amp;pages=10237" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms10237" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms10237" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms10237"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26674669" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ma</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Naseri</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Reyes-Gutierrez</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">S. A.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Pederson</div> <div class="given-names">T.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Multicolor CRISPR labeling of chromosomal loci in human cells</div>. <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">112</div>, <div class="fpage">3002</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multicolor%20CRISPR%20labeling%20of%20chromosomal%20loci%20in%20human%20cells&amp;author=H.%20Ma&amp;author=A.%20Naseri&amp;author=P.%20Reyes-Gutierrez&amp;author=S.%20A.%20Wolfe&amp;author=S.%20Zhang&amp;author=T.%20Pederson&amp;publication_year=2015&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=112&amp;pages=3002-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1420024112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1420024112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1420024112"> </span></div> </div></div></div></div></div><div content-id="dsx029-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tóth</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Huszár</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Bencsura</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Restriction enzyme body doubles and PCR cloning: on the general use of type IIs restriction enzymes for cloning</div>. <div class="source">PloS One</div> , <div class="volume">9</div>, <div class="fpage">e90896</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Restriction%20enzyme%20body%20doubles%20and%20PCR%20cloning%3A%20on%20the%20general%20use%20of%20type%20IIs%20restriction%20enzymes%20for%20cloning&amp;author=E.%20T%C3%B3th&amp;author=K.%20Husz%C3%A1r&amp;author=P.%20Bencsura&amp;publication_year=2014&amp;journal=PloS%20One&amp;volume=9&amp;pages=e90896" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0090896" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0090896" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0090896"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24618593" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stewart</div> <div class="given-names">S. A.</div></div>, <div class="name string-name"><div class="surname">Dykxhoorn</div> <div class="given-names">D. M.</div></div>, <div class="name string-name"><div class="surname">Palliser</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2003</div>, 
<div class="article-title">Lentivirus-delivered stable gene silencing by RNAi in primary cells</div>. <div class="source">RNA</div> , <div class="volume">9</div>, <div class="fpage">493</div>–<div class="lpage">501</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lentivirus-delivered%20stable%20gene%20silencing%20by%20RNAi%20in%20primary%20cells&amp;author=S.%20A.%20Stewart&amp;author=D.%20M.%20Dykxhoorn&amp;author=D.%20Palliser&amp;publication_year=2003&amp;journal=RNA&amp;volume=9&amp;pages=493-501" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.2192803" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.2192803" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.2192803"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12649500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Iwamoto</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Björklund</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Lundberg</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kirik</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Wandless</div> <div class="given-names">T. J.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">A general chemical method to regulate protein stability in the mammalian central nervous system</div>. <div class="source">Chem. Biol</div> ., <div class="volume">17</div>, <div class="fpage">981</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20general%20chemical%20method%20to%20regulate%20protein%20stability%20in%20the%20mammalian%20central%20nervous%20system&amp;author=M.%20Iwamoto&amp;author=T.%20Bj%C3%B6rklund&amp;author=C.%20Lundberg&amp;author=D.%20Kirik&amp;author=T.%20J.%20Wandless&amp;publication_year=2010&amp;journal=Chem.%20Biol&amp;volume=17&amp;pages=981-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.chembiol.2010.07.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.chembiol.2010.07.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.chembiol.2010.07.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20851347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Perez-Pinera</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Ousterout</div> <div class="given-names">D. G.</div></div>, <div class="name string-name"><div class="surname">Brown</div> <div class="given-names">M. T.</div></div>, <div class="name string-name"><div class="surname">Gersbach</div> <div class="given-names">C. A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Gene targeting to the ROSA26 locus directed by engineered zinc finger nucleases</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">3741</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20targeting%20to%20the%20ROSA26%20locus%20directed%20by%20engineered%20zinc%20finger%20nucleases&amp;author=P.%20Perez-Pinera&amp;author=D.%20G.%20Ousterout&amp;author=M.%20T.%20Brown&amp;author=C.%20A.%20Gersbach&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=3741-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1214" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1214" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1214"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22169954" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Maresca</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">V. G.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Y.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Obligate ligation-gated recombination (ObLiGaRe): custom-designed nuclease-mediated targeted integration through nonhomologous end joining</div>. <div class="source">Genome Res</div> ., <div class="volume">23</div>, <div class="fpage">539</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Obligate%20ligation-gated%20recombination%20(ObLiGaRe)%3A%20custom-designed%20nuclease-mediated%20targeted%20integration%20through%20nonhomologous%20end%20joining&amp;author=M.%20Maresca&amp;author=V.%20G.%20Lin&amp;author=N.%20Guo&amp;author=Y.%20Yang&amp;publication_year=2013&amp;journal=Genome%20Res&amp;volume=23&amp;pages=539-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.145441.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.145441.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.145441.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23152450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Heigwer</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Kerr</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Boutros</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">E-CRISP: fast CRISPR target site identification</div>. <div class="source">Nat. Methods</div> , <div class="volume">11</div>, <div class="fpage">122</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=E-CRISP%3A%20fast%20CRISPR%20target%20site%20identification&amp;author=F.%20Heigwer&amp;author=G.%20Kerr&amp;author=M.%20Boutros&amp;publication_year=2014&amp;journal=Nat.%20Methods&amp;volume=11&amp;pages=122-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2812" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2812" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2812"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24481216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aach</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Mali</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Church</div> <div class="given-names">G. M.</div></div></div> <div class="year">2014</div>, CasFinder: Flexible algorithm for identifying specific Cas9 targets in genomes. <em>bioRxiv</em>, 005074.</div></div></div></div><div content-id="dsx029-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xue</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yin</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">CRISPR-mediated direct mutation of cancer genes in the mouse liver</div>. <div class="source">Nature</div>  <div class="volume">514</div>, <div class="fpage">380</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPR-mediated%20direct%20mutation%20of%20cancer%20genes%20in%20the%20mouse%20liver&amp;author=W.%20Xue&amp;author=S.%20Chen&amp;author=H.%20Yin&amp;publication_year=2014&amp;journal=Nature&amp;volume=514&amp;pages=380-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature13589" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature13589" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature13589"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25119044" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fujii</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Kawasaki</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Sugiura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Naito</div> <div class="given-names">K.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Efficient generation of large-scale genome-modified mice using gRNA and CAS9 endonuclease</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">e187</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20generation%20of%20large-scale%20genome-modified%20mice%20using%20gRNA%20and%20CAS9%20endonuclease&amp;author=W.%20Fujii&amp;author=K.%20Kawasaki&amp;author=K.%20Sugiura&amp;author=K.%20Naito&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=e187" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt772" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt772" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt772"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23997119" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pinder</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Salsman</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Dellaire</div> <div class="given-names">G.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Nuclear domain ‘knock-in’screen for the evaluation and identification of small molecule enhancers of CRISPR-based genome editing</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">9379</div>–<div class="lpage">92</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20domain%20%E2%80%98knock-in%E2%80%99screen%20for%20the%20evaluation%20and%20identification%20of%20small%20molecule%20enhancers%20of%20CRISPR-based%20genome%20editing&amp;author=J.%20Pinder&amp;author=J.%20Salsman&amp;author=G.%20Dellaire&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=9379-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv993" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv993" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv993"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26429972" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Andersen</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Pu</div> <div class="given-names">W. T.</div></div></div> <div class="year">2014</div>, Optimization of genome engineering approaches with the CRISPR/Cas9 system. <div class="source">Plos One</div> ., <div class="volume">9</div>, <div class="fpage">e105779</div>.</div></div></div></div><div content-id="dsx029-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Geisinger</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Turan</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hernandez</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Spector</div> <div class="given-names">L. P.</div></div>, <div class="name string-name"><div class="surname">Calos</div> <div class="given-names">M. P.</div></div></div> <div class="year">2016</div>, In vivo blunt-end cloning through CRISPR/Cas9-facilitated non-homologous end-joining. <em>Nucleic Acids Res.</em>, <div class="volume">44</div>, <div class="fpage">e76</div>–<div class="lpage">e76</div>.</div></div></div></div><div content-id="dsx029-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Slaymaker</div> <div class="given-names">I. M.</div></div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zetsche</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Scott</div> <div class="given-names">D. A.</div></div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">W. X.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">F.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Rationally engineered Cas9 nucleases with improved specificity</div>. <div class="source">Science</div> , <div class="volume">351</div>, <div class="fpage">84</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rationally%20engineered%20Cas9%20nucleases%20with%20improved%20specificity&amp;author=I.%20M.%20Slaymaker&amp;author=L.%20Gao&amp;author=B.%20Zetsche&amp;author=D.%20A.%20Scott&amp;author=W.%20X.%20Yan&amp;author=F.%20Zhang&amp;publication_year=2016&amp;journal=Science&amp;volume=351&amp;pages=84-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.aad5227" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aad5227" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.aad5227"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26628643" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kleinstiver</div> <div class="given-names">B. P.</div></div>, <div class="name string-name"><div class="surname">Pattanayak</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Prew</div> <div class="given-names">M. S.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">High-fidelity CRISPR–Cas9 nucleases with no detectable genome-wide off-target effects</div>. <div class="source">Nature</div>  <div class="volume">529</div>, <div class="fpage">490</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-fidelity%20CRISPR%E2%80%93Cas9%20nucleases%20with%20no%20detectable%20genome-wide%20off-target%20effects&amp;author=B.%20P.%20Kleinstiver&amp;author=V.%20Pattanayak&amp;author=M.%20S.%20Prew&amp;publication_year=2016&amp;journal=Nature&amp;volume=529&amp;pages=490-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature16526" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature16526" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature16526"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26735016" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mladenov</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Iliakis</div> <div class="given-names">G.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Induction and repair of DNA double strand breaks: the increasing spectrum of non-homologous end joining pathways</div>. <div class="source">Mutat. Res</div> ., <div class="volume">711</div>, <div class="fpage">61</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Induction%20and%20repair%20of%20DNA%20double%20strand%20breaks%3A%20the%20increasing%20spectrum%20of%20non-homologous%20end%20joining%20pathways&amp;author=E.%20Mladenov&amp;author=G.%20Iliakis&amp;publication_year=2011&amp;journal=Mutat.%20Res&amp;volume=711&amp;pages=61-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mrfmmm.2011.02.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mrfmmm.2011.02.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mrfmmm.2011.02.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21329706" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bae</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kweon</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">H. S.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">J.-S.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Microhomology-based choice of Cas9 nuclease target sites</div>. <div class="source">Nat. Methods</div> , <div class="volume">11</div>, <div class="fpage">705</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microhomology-based%20choice%20of%20Cas9%20nuclease%20target%20sites&amp;author=S.%20Bae&amp;author=J.%20Kweon&amp;author=H.%20S.%20Kim&amp;author=J.-S.%20Kim&amp;publication_year=2014&amp;journal=Nat.%20Methods&amp;volume=11&amp;pages=705-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.3015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.3015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.3015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24972169" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McVey</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">S. E.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">MMEJ repair of double-strand breaks (director’s cut): deleted sequences and alternative endings</div>. <div class="source">Trends Genet</div> ., <div class="volume">24</div>, <div class="fpage">529</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MMEJ%20repair%20of%20double-strand%20breaks%20(director%E2%80%99s%20cut)%3A%20deleted%20sequences%20and%20alternative%20endings&amp;author=M.%20McVey&amp;author=S.%20E.%20Lee&amp;publication_year=2008&amp;journal=Trends%20Genet&amp;volume=24&amp;pages=529-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2008.08.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2008.08.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2008.08.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18809224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vouillot</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Thélie</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Pollet</div> <div class="given-names">N.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Comparison of T7E1 and surveyor mismatch cleavage assays to detect mutations triggered by engineered nucleases</div>. <div class="source">G3</div> , <div class="volume">5</div>, <div class="fpage">407</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20T7E1%20and%20surveyor%20mismatch%20cleavage%20assays%20to%20detect%20mutations%20triggered%20by%20engineered%20nucleases&amp;author=L.%20Vouillot&amp;author=A.%20Th%C3%A9lie&amp;author=N.%20Pollet&amp;publication_year=2015&amp;journal=G3&amp;volume=5&amp;pages=407-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.114.015834" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.114.015834" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.114.015834"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25566793" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Shi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Liang</div> <div class="given-names">H.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Microchip electrophoresis coupled with on‐line magnetic separation and chemiluminescence detection for multiplexed immunoassay</div>. <div class="source">Electrophoresis</div> , <div class="volume">33</div>, <div class="fpage">1198</div>–<div class="lpage">204</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microchip%20electrophoresis%20coupled%20with%20on%E2%80%90line%20magnetic%20separation%20and%20chemiluminescence%20detection%20for%20multiplexed%20immunoassay&amp;author=Y.%20Huang&amp;author=S.%20Zhao&amp;author=M.%20Shi&amp;author=J.%20Liu&amp;author=H.%20Liang&amp;publication_year=2012&amp;journal=Electrophoresis&amp;volume=33&amp;pages=1198-204" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/elps.201100543" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Felps.201100543" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Felps.201100543"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22539323" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sakuma</div> <div class="given-names">T.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Front-line of genome editing technology for animal cell engineering</div>. <div class="source">BMC Proceed.,</div>  <strong>9</strong>, <div class="fpage">O1</div>.</div></div></div></div><div content-id="dsx029-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Gong</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Chang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Peng</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Development of novel visual-plus quantitative analysis systems for studying DNA double-strand break repairs in zebrafish</div>. <div class="source">J. Genet. Genomics</div> , <div class="volume">39</div>, <div class="fpage">489</div>–<div class="lpage">502</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20novel%20visual-plus%20quantitative%20analysis%20systems%20for%20studying%20DNA%20double-strand%20break%20repairs%20in%20zebrafish&amp;author=J.%20Liu&amp;author=L.%20Gong&amp;author=C.%20Chang&amp;author=C.%20Liu&amp;author=J.%20Peng&amp;author=J.%20Chen&amp;publication_year=2012&amp;journal=J.%20Genet.%20Genomics&amp;volume=39&amp;pages=489-502" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jgg.2012.07.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jgg.2012.07.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jgg.2012.07.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23021549" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dai</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Cui</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">W.</div></div></div> <div class="year">2010</div>
<div class="article-title">, Non-homologous end joining plays a key role in transgene concatemer formation in transgenic zebrafish embryos</div>. <div class="source">Int. J. Biol. Sci</div> ., <div class="volume">6</div>, <div class="fpage">756</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=%2C%20Non-homologous%20end%20joining%20plays%20a%20key%20role%20in%20transgene%20concatemer%20formation%20in%20transgenic%20zebrafish%20embryos&amp;author=J.%20Dai&amp;author=X.%20Cui&amp;author=Z.%20Zhu&amp;author=W.%20Hu&amp;publication_year=2010&amp;journal=Int.%20J.%20Biol.%20Sci&amp;volume=6&amp;pages=756" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7150/ijbs.6.756" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7150%2Fijbs.6.756" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7150%2Fijbs.6.756"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21152116" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hagmann</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bruggmann</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Xue</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">1998</div>, 
<div class="article-title">Homologous recombination and DNA-end joining reactions in zygotes and early embryos of zebrafish (<em>Danio rerio</em>) and <em>Drosophila melanogaster</em></div>. <div class="source">Biol. Chem</div> ., <div class="volume">379</div>, <div class="fpage">673</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homologous%20recombination%20and%20DNA-end%20joining%20reactions%20in%20zygotes%20and%20early%20embryos%20of%20zebrafish%20(Danio%20rerio)%20and%20Drosophila%20melanogaster&amp;author=M.%20Hagmann&amp;author=R.%20Bruggmann&amp;author=L.%20Xue&amp;publication_year=1998&amp;journal=Biol.%20Chem&amp;volume=379&amp;pages=673-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1515/bchm.1998.379.6.673" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1515%2Fbchm.1998.379.6.673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1515%2Fbchm.1998.379.6.673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9687016" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Tsunekawa</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Hernandez-Benitez</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">In vivo genome editing via CRISPR/Cas9 mediated homology-independent targeted integration</div>. <div class="source">Nature</div> , <div class="volume">540</div>, <div class="fpage">144</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20vivo%20genome%20editing%20via%20CRISPR%2FCas9%20mediated%20homology-independent%20targeted%20integration&amp;author=K.%20Suzuki&amp;author=Y.%20Tsunekawa&amp;author=R.%20Hernandez-Benitez&amp;publication_year=2016&amp;journal=Nature&amp;volume=540&amp;pages=144-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature20565" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature20565" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature20565"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27851729" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nakade</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Tsubota</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Sakane</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Microhomology-mediated end-joining-dependent integration of donor DNA in cells and animals using TALENs and CRISPR/Cas9</div>. <div class="source">Nat. Commun</div> ., <div class="volume">5</div>, <div class="fpage">5560</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microhomology-mediated%20end-joining-dependent%20integration%20of%20donor%20DNA%20in%20cells%20and%20animals%20using%20TALENs%20and%20CRISPR%2FCas9&amp;author=S.%20Nakade&amp;author=T.%20Tsubota&amp;author=Y.%20Sakane&amp;publication_year=2014&amp;journal=Nat.%20Commun&amp;volume=5&amp;pages=5560" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms6560" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms6560" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms6560"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25410609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx029-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx029-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schmid-Burgk</div> <div class="given-names">J. L.</div></div>, <div class="name string-name"><div class="surname">Höning</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Ebert</div> <div class="given-names">T. S.</div></div>, <div class="name string-name"><div class="surname">Hornung</div> <div class="given-names">V.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">CRISPaint allows modular base-specific gene tagging using a ligase-4-dependent mechanism</div>. <div class="source">Nat. Commun</div> ., <div class="volume">7</div>, <div class="fpage">12338</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRISPaint%20allows%20modular%20base-specific%20gene%20tagging%20using%20a%20ligase-4-dependent%20mechanism&amp;author=J.%20L.%20Schmid-Burgk&amp;author=K.%20H%C3%B6ning&amp;author=T.%20S.%20Ebert&amp;author=V.%20Hornung&amp;publication_year=2016&amp;journal=Nat.%20Commun&amp;volume=7&amp;pages=12338" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms12338" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms12338" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms12338"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27465542" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3904549&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_figures_and_methods_dsx029.pdf?Expires=2147483647&amp;Signature=SHcW7I5km-vphwmN4lJ0pbEDxMtrSRhaCvBgLLDWtUUKfE~j~ummAKQofQgn4GBhfVesVJk8-tfOA4Nvh8NDN0vRLYTMby2s7sClOOZigawXB7i1dXX0lj6ZyiEGAjm-iFpIBfF1HhSYadIjjxdwKxh74zVo159bHhMC22G-ZZN4bqENM9nrcNr82zK4FikflxP8jUoCRJLkGjOVy9bIOR0~bjMRJ70vO4L1RA1qxakyeg~5dDyq6jQlfwwlulD1BphHtEc0ClevhSxymhoIhtxZtyVNidtJwLkmYcHT9piVjkkZ8fD0DZVK-qLGU9KHJ~u35nTym9yC7~ce~4LmMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figures and Methods</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx029/1/supplementary_data_dsx029.xlsx?Expires=2147483647&amp;Signature=tvN8l6Nr13c-J7Z5ueuYtmqjQDRRDKGeMZZ4zP2xfTAnHe5~chW-J73UkVFgA2WKJsBA02QbIMp6AMjSQiHSwMi0vHIVHEbihM53zO2GWQvuc4s2FTp1W9zVQUk9sLiwr5wcXiiABvFYaNZHZUtIk4XHVLMRF7WB-X7LD~F1Kvfd7WpSBIrtxhybwx0b1m2jkT6FrEL5~IBFX0ygmG~VV1Mnqudots2x7TJaZbc6aOeoBKvcy9vW8rzF6Vlr8ElemUAIh~IO~SFhI4KvLxWuzc14LBQoyV6OmOXBQL0M1rsXEwQYXq189aSoITXXGt2Lp8RRmBOJ1IL4zERV9~H8WA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,878</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000418420400006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx029"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3904549"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000418420400006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f6%2f609%2f3904549">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000418420400006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F6%2F609%2F3904549" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3904549'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="convenient method to pre-screen candidate guide RNAs for CRISPR/Cas9 gene editing by NHEJ-mediated integration of a ‘self-cleaving’ GFP-expression plasmid" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx029");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f609%2f3904549">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

