

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/6/387/512877","@type":"ScholarlyArticle","name":"MetaGeneAnnotator: Detecting Species-Specific Patterns of Ribosomal Binding Site for Precise Gene Prediction in Anonymous Prokaryotic and Phage Genomes","datePublished":"2008-10-21","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2008-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsn027","keywords":["bioinformatics","gene-finding","prokaryote","phage","ribosomal binding site"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/6/387/512877","author":[{"name":"Noguchi, Hideki","affiliation":"Advanced Science and Technology Research Group ,  Mitsubishi Research Institute, Inc. ,  2-3-6 Otemachi, Chiyoda-ku, Tokyo 100-8141 ,  Japan","@type":"Person"},{"name":"Taniguchi, Takeaki","affiliation":"Advanced Science and Technology Research Group ,  Mitsubishi Research Institute, Inc. ,  2-3-6 Otemachi, Chiyoda-ku, Tokyo 100-8141 ,  Japan","@type":"Person"},{"name":"Itoh, Takehiko","affiliation":"Advanced Science and Technology Research Group ,  Mitsubishi Research Institute, Inc. ,  2-3-6 Otemachi, Chiyoda-ku, Tokyo 100-8141 ,  Japan","@type":"Person"}],"description":"Abstract.  Recent advances in DNA sequencers are accelerating genome sequencing, especially in microbes, and complete and draft genomes from various species hav","pageStart":"387","pageEnd":"396","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/6/1/m_cover.gif?Expires=2147483647&Signature=cbcaSTnXmSAtcWAqInhnHikjrANtVVQT69rwVkS2N2-Rvvq~3o4NF54oja~GIUWptdFQr454edog~WN90FgAfPnwCCNM9uZ3DuEnj-WTN3JyklWsccNFHC6w0Mg4O1XsO2YSzFEaGsd~LWUMMgT-JRIdN1RQUYUQEdvCgKjLmwjDjkQDpQkgPjsYyRLj1QLXlpM4IsivMdAnqm9LmZlXnvMUUp5UVsqaQBEzVLg9CZ1N~86MZnaIY5LkZ1lOjOrEUjr78DNfXaNPxd2goUtJ24j8goSO16w~p5Co8ZBl8FnseAHBtje4N~aGVfSKUlSXEIVKmxpcUZsLAtBrli2ACg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"MetaGeneAnnotator: Detecting Species-Specific Patterns of Ribosomal Binding Site for Precise Gene Prediction in Anonymous Prokaryotic and Phage Genomes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/6/1/m_cover.gif?Expires=2147483647&Signature=cbcaSTnXmSAtcWAqInhnHikjrANtVVQT69rwVkS2N2-Rvvq~3o4NF54oja~GIUWptdFQr454edog~WN90FgAfPnwCCNM9uZ3DuEnj-WTN3JyklWsccNFHC6w0Mg4O1XsO2YSzFEaGsd~LWUMMgT-JRIdN1RQUYUQEdvCgKjLmwjDjkQDpQkgPjsYyRLj1QLXlpM4IsivMdAnqm9LmZlXnvMUUp5UVsqaQBEzVLg9CZ1N~86MZnaIY5LkZ1lOjOrEUjr78DNfXaNPxd2goUtJ24j8goSO16w~p5Co8ZBl8FnseAHBtje4N~aGVfSKUlSXEIVKmxpcUZsLAtBrli2ACg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="MetaGeneAnnotator: Detecting Species-Specific Patterns of Ribosomal Binding Site for Precise Gene Prediction in Anonymous Prokaryotic and Phage Genomes" />
    <meta property="og:description" content="Abstract.  Recent advances in DNA sequencers are accelerating genome sequencing, especially in microbes, and complete and draft genomes from various species hav" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/6/387/512877" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-10-21" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Noguchi, Hideki" /><meta name="citation_author_institution" content="Advanced Science and Technology Research Group, Mitsubishi Research Institute, Inc., 2-3-6 Otemachi, Chiyoda-ku, Tokyo 100-8141, Japan" /><meta name="citation_author" content="Taniguchi, Takeaki" /><meta name="citation_author_institution" content="Advanced Science and Technology Research Group, Mitsubishi Research Institute, Inc., 2-3-6 Otemachi, Chiyoda-ku, Tokyo 100-8141, Japan" /><meta name="citation_author" content="Itoh, Takehiko" /><meta name="citation_author_institution" content="Advanced Science and Technology Research Group, Mitsubishi Research Institute, Inc., 2-3-6 Otemachi, Chiyoda-ku, Tokyo 100-8141, Japan" /><meta name="citation_title" content="MetaGeneAnnotator: Detecting Species-Specific Patterns of Ribosomal Binding Site for Precise Gene Prediction in Anonymous Prokaryotic and Phage Genomes" /><meta name="citation_firstpage" content="387" /><meta name="citation_lastpage" content="396" /><meta name="citation_doi" content="10.1093/dnares/dsn027" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2008/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Recognition of protein coding regions in DNA sequences; Nucleic Acids Res.;  citation_year=1981;  citation_volume=10;  citation_pages=5303-5318; " /><meta name="citation_reference" content="citation_title=The codon preference plot: graphic analysis of protein coding sequences and prediction of gene expression; Nucleic Acids Res.;  citation_year=1984;  citation_volume=12;  citation_pages=539-549; " /><meta name="citation_reference" content="citation_title=Measurements of the effects of that coding for a protein has on a DNA sequence and their use for finding genes; Nucleic Acids Res.;  citation_year=1984;  citation_volume=12;  citation_pages=551-567; " /><meta name="citation_reference" content="citation_title=Statistical patterns in primary structures of functional regions in the E. coli genome: III. Computer recognition of coding regions; Mol. Biol.;  citation_year=1986;  citation_volume=20;  citation_pages=1145-1150; " /><meta name="citation_reference" content="citation_title=GeneMark: parallel gene recognition for both DNA strands; Comput. Chem.;  citation_year=1993;  citation_volume=17;  citation_pages=123-153; " /><meta name="citation_reference" content="citation_title=A hidden Markov model that finds genes in E.coli DNA; Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=4768-4778; " /><meta name="citation_reference" content="citation_title=Microbial gene identification using interpolated Markov model; Nucleic Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=544-548; " /><meta name="citation_reference" content="citation_title=GeneMark.hmm: new solutions for gene finding; Nucleic Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=1107-1115; " /><meta name="citation_reference" content="citation_title=Improved microbial gene identification with GLIMMER; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=4636-4641; " /><meta name="citation_reference" content="citation_title=Modeling and predicting transcriptional units of Escherichia coli genes using hidden Markov model; Bioinformatics;  citation_year=1999;  citation_volume=15;  citation_pages=987-993; " /><meta name="citation_reference" content="citation_title=A novel bacterial gene-finding system with improved accuracy in locating start codons; DNA Res.;  citation_year=2001;  citation_volume=8;  citation_pages=97-106; " /><meta name="citation_reference" content="citation_title=How to interpret an anonymous bacterial genome: machine learning approach to gene identification; Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=1154-1171; " /><meta name="citation_reference" content="citation_title=Self-identification of protein-coding regions in microbial genomes; Proc. Natl. Acad. Sci. U. S. A.;  citation_year=1998;  citation_volume=95;  citation_pages=10026-10031; " /><meta name="citation_reference" content="citation_title=On the convergence of a clustering algorithm for protein-coding regions in microbial genomes; Bioinformatics;  citation_year=2000;  citation_volume=16;  citation_pages=367-371; " /><meta name="citation_reference" content="citation_title=GeneMarkS: a self-training method for prediction of gene starts in microbial genomes; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29;  citation_pages=2607-2618; " /><meta name="citation_reference" content="citation_title=Identifying bacterial genes and endosymbiont DNA with Glimmer; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=673-679; " /><meta name="citation_reference" content="citation_title=Next-generation sequencing transforms today’s biology; Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=16-18; " /><meta name="citation_reference" content="citation_title=Advanced sequencing technologies and their wider impact in microbiology; J. Exp. Biol.;  citation_year=2007;  citation_volume=210;  citation_pages=1518-1525; " /><meta name="citation_reference" content="citation_title=SHARCGS, a fast and highly accurate short-read assembly algorithm for de novo genomic sequencing; Genome Res.;  citation_year=2007;  citation_volume=17;  citation_pages=1697" /><meta name="citation_reference" content="citation_title=De novo bacterial genome sequencing: millions of very short reads assembled on a desktop computer; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=802" /><meta name="citation_reference" content="citation_title=ALLPATHS: de novo assembly of whole-genome shotgun microreads; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=810" /><meta name="citation_reference" content="citation_title=Velvet: algorithms for de novo short read assembly using de Bruijn graph; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=821" /><meta name="citation_reference" content="citation_title=MetaGene: prokaryotic gene finding from environmental genome shotgun sequences; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=5623-5630; " /><meta name="citation_reference" content="citation_title=Comparative metagenomics revealed commonly enriched gene sets in human gut microbiomes; DNA Res.;  citation_year=2007;  citation_volume=14;  citation_pages=169-181; " /><meta name="citation_reference" content="citation_title=Get the most out of your metagenome: computational analysis of environmental sequence data; Curr. Opin. Microbiol.;  citation_year=2007;  citation_volume=10;  citation_pages=490-498; " /><meta name="citation_reference" content="citation_title=Metagenomics, biotechnology with non-culturable microbes; Appl. Microbiol. Biotechnol.;  citation_year=2007;  citation_volume=75;  citation_pages=955-962; " /><meta name="citation_reference" content="citation_title=Bioinformatics challenges of new sequencing technology; Trends Genet.;  citation_year=2008;  citation_volume=24;  citation_pages=142-149; " /><meta name="citation_reference" content="citation_title=NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins; Nucleic Acids Res.;  citation_year=2007;  citation_volume=35;  citation_pages=D61-65; " /><meta name="citation_reference" content="citation_title=The neighbor-joining method: a new method for reconstructing phylogenetic trees; Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=406-425; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of enterohemorrhagic Escherichia coli O157:H7 and genomic comparison with a laboratory strain K-12; DNA Res.;  citation_year=2001;  citation_volume=8;  citation_pages=11-22; " /><meta name="citation_reference" content="citation_title=Diversification of Escherichia coli genomes: are bacteriophages the major contributors?; Trens Microbiol.;  citation_year=2001;  citation_volume=9;  citation_pages=481-485; " /><meta name="citation_reference" content="citation_title=Heuristic approach to deriving models for gene finding; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=3911-3920; " /><meta name="citation_reference" content="citation_title=The 3′-terminal sequence of Escherichia coli 16S ribosomal RNA: Complementary to nonsense triplets and ribosome binding sites; Proc. Natl. Acad. Sci. U. S. A.;  citation_year=1974;  citation_volume=71;  citation_pages=1342-1346; " /><meta name="citation_reference" content="citation_title=Accurate phylogenetic classification of variable-length DNA fragments; Nat. Methods;  citation_year=2007;  citation_volume=4;  citation_pages=63-72; " /><meta name="citation_reference" content="citation_title=Phylogenetic classification of short environmental DNA fragments; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=2230-2239; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/6/387/1376547/dsn027.pdf" /><meta name="description" content="Abstract.  Recent advances in DNA sequencers are accelerating genome sequencing, especially in microbes, and complete and draft genomes from various species hav" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/6/387/512877" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/6/387/512877" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/6/387/512877");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsn027';
    </script>
<title>
	MetaGeneAnnotator: Detecting Species-Specific Patterns of Ribosomal Binding Site for Precise Gene Prediction in Anonymous Prokaryotic and Phage Genomes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./512877" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZVMPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOb2d1Y2hpLCBIaWRla2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBZHZhbmNlZCBTY2llbmNlIGFuZCBUZWNobm9sb2d5IFJlc2VhcmNoIEdyb3VwLCBNaXRzdWJpc2hpIFJlc2VhcmNoIEluc3RpdHV0ZSwgSW5jLiwgMi0zLTYgT3RlbWFjaGksIENoaXlvZGEta3UsIFRva3lvIDEwMC04MTQxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhbmlndWNoaSwgVGFrZWFraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkFkdmFuY2VkIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgUmVzZWFyY2ggR3JvdXAsIE1pdHN1YmlzaGkgUmVzZWFyY2ggSW5zdGl0dXRlLCBJbmMuLCAyLTMtNiBPdGVtYWNoaSwgQ2hpeW9kYS1rdSwgVG9reW8gMTAwLTgxNDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXRvaCwgVGFrZWhpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBZHZhbmNlZCBTY2llbmNlIGFuZCBUZWNobm9sb2d5IFJlc2VhcmNoIEdyb3VwLCBNaXRzdWJpc2hpIFJlc2VhcmNoIEluc3RpdHV0ZSwgSW5jLiwgMi0zLTYgT3RlbWFjaGksIENoaXlvZGEta3UsIFRva3lvIDEwMC04MTQxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iTWV0YUdlbmVBbm5vdGF0b3I6IERldGVjdGluZyBTcGVjaWVzLVNwZWNpZmljIFBhdHRlcm5zIG9mIFJpYm9zb21hbCBCaW5kaW5nIFNpdGUgZm9yIFByZWNpc2UgR2VuZSBQcmVkaWN0aW9uIGluIEFub255bW91cyBQcm9rYXJ5b3RpYyBhbmQgUGhhZ2UgR2Vub21lcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjM4NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMzk2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNuMDI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMDgvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWNvZ25pdGlvbiBvZiBwcm90ZWluIGNvZGluZyByZWdpb25zIGluIEROQSBzZXF1ZW5jZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTUzMDMtNTMxODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvZG9uIHByZWZlcmVuY2UgcGxvdDogZ3JhcGhpYyBhbmFseXNpcyBvZiBwcm90ZWluIGNvZGluZyBzZXF1ZW5jZXMgYW5kIHByZWRpY3Rpb24gb2YgZ2VuZSBleHByZXNzaW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODQ7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz01MzktNTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZWFzdXJlbWVudHMgb2YgdGhlIGVmZmVjdHMgb2YgdGhhdCBjb2RpbmcgZm9yIGEgcHJvdGVpbiBoYXMgb24gYSBETkEgc2VxdWVuY2UgYW5kIHRoZWlyIHVzZSBmb3IgZmluZGluZyBnZW5lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9NTUxLTU2NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RhdGlzdGljYWwgcGF0dGVybnMgaW4gcHJpbWFyeSBzdHJ1Y3R1cmVzIG9mIGZ1bmN0aW9uYWwgcmVnaW9ucyBpbiB0aGUgRS4gY29saSBnZW5vbWU6IElJSS4gQ29tcHV0ZXIgcmVjb2duaXRpb24gb2YgY29kaW5nIHJlZ2lvbnM7IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xMTQ1LTExNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVNYXJrOiBwYXJhbGxlbCBnZW5lIHJlY29nbml0aW9uIGZvciBib3RoIEROQSBzdHJhbmRzOyBDb21wdXQuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTIzLTE1MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWRkZW4gTWFya292IG1vZGVsIHRoYXQgZmluZHMgZ2VuZXMgaW4gRS5jb2xpIEROQTsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9NDc2OC00Nzc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb2JpYWwgZ2VuZSBpZGVudGlmaWNhdGlvbiB1c2luZyBpbnRlcnBvbGF0ZWQgTWFya292IG1vZGVsOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz01NDQtNTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lTWFyay5obW06IG5ldyBzb2x1dGlvbnMgZm9yIGdlbmUgZmluZGluZzsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MTEwNy0xMTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZlZCBtaWNyb2JpYWwgZ2VuZSBpZGVudGlmaWNhdGlvbiB3aXRoIEdMSU1NRVI7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTQ2MzYtNDY0MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9kZWxpbmcgYW5kIHByZWRpY3RpbmcgdHJhbnNjcmlwdGlvbmFsIHVuaXRzIG9mIEVzY2hlcmljaGlhIGNvbGkgZ2VuZXMgdXNpbmcgaGlkZGVuIE1hcmtvdiBtb2RlbDsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz05ODctOTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIGJhY3RlcmlhbCBnZW5lLWZpbmRpbmcgc3lzdGVtIHdpdGggaW1wcm92ZWQgYWNjdXJhY3kgaW4gbG9jYXRpbmcgc3RhcnQgY29kb25zOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9OTctMTA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3cgdG8gaW50ZXJwcmV0IGFuIGFub255bW91cyBiYWN0ZXJpYWwgZ2Vub21lOiBtYWNoaW5lIGxlYXJuaW5nIGFwcHJvYWNoIHRvIGdlbmUgaWRlbnRpZmljYXRpb247IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xMTU0LTExNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGYtaWRlbnRpZmljYXRpb24gb2YgcHJvdGVpbi1jb2RpbmcgcmVnaW9ucyBpbiBtaWNyb2JpYWwgZ2Vub21lczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTEwMDI2LTEwMDMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbiB0aGUgY29udmVyZ2VuY2Ugb2YgYSBjbHVzdGVyaW5nIGFsZ29yaXRobSBmb3IgcHJvdGVpbi1jb2RpbmcgcmVnaW9ucyBpbiBtaWNyb2JpYWwgZ2Vub21lczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz0zNjctMzcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lTWFya1M6IGEgc2VsZi10cmFpbmluZyBtZXRob2QgZm9yIHByZWRpY3Rpb24gb2YgZ2VuZSBzdGFydHMgaW4gbWljcm9iaWFsIGdlbm9tZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTI2MDctMjYxODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZnlpbmcgYmFjdGVyaWFsIGdlbmVzIGFuZCBlbmRvc3ltYmlvbnQgRE5BIHdpdGggR2xpbW1lcjsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz02NzMtNjc5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyB0cmFuc2Zvcm1zIHRvZGF54oCZcyBiaW9sb2d5OyBOYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTE2LTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZHZhbmNlZCBzZXF1ZW5jaW5nIHRlY2hub2xvZ2llcyBhbmQgdGhlaXIgd2lkZXIgaW1wYWN0IGluIG1pY3JvYmlvbG9neTsgSi4gRXhwLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMTA7ICBjaXRhdGlvbl9wYWdlcz0xNTE4LTE1MjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNIQVJDR1MsIGEgZmFzdCBhbmQgaGlnaGx5IGFjY3VyYXRlIHNob3J0LXJlYWQgYXNzZW1ibHkgYWxnb3JpdGhtIGZvciBkZSBub3ZvIGdlbm9taWMgc2VxdWVuY2luZzsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0xNjk3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byBiYWN0ZXJpYWwgZ2Vub21lIHNlcXVlbmNpbmc6IG1pbGxpb25zIG9mIHZlcnkgc2hvcnQgcmVhZHMgYXNzZW1ibGVkIG9uIGEgZGVza3RvcCBjb21wdXRlcjsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz04MDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BTExQQVRIUzogZGUgbm92byBhc3NlbWJseSBvZiB3aG9sZS1nZW5vbWUgc2hvdGd1biBtaWNyb3JlYWRzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTgxMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZlbHZldDogYWxnb3JpdGhtcyBmb3IgZGUgbm92byBzaG9ydCByZWFkIGFzc2VtYmx5IHVzaW5nIGRlIEJydWlqbiBncmFwaDsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz04MjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRhR2VuZTogcHJva2FyeW90aWMgZ2VuZSBmaW5kaW5nIGZyb20gZW52aXJvbm1lbnRhbCBnZW5vbWUgc2hvdGd1biBzZXF1ZW5jZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTU2MjMtNTYzMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgbWV0YWdlbm9taWNzIHJldmVhbGVkIGNvbW1vbmx5IGVucmljaGVkIGdlbmUgc2V0cyBpbiBodW1hbiBndXQgbWljcm9iaW9tZXM7IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MTY5LTE4MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2V0IHRoZSBtb3N0IG91dCBvZiB5b3VyIG1ldGFnZW5vbWU6IGNvbXB1dGF0aW9uYWwgYW5hbHlzaXMgb2YgZW52aXJvbm1lbnRhbCBzZXF1ZW5jZSBkYXRhOyBDdXJyLiBPcGluLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NDkwLTQ5ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0YWdlbm9taWNzLCBiaW90ZWNobm9sb2d5IHdpdGggbm9uLWN1bHR1cmFibGUgbWljcm9iZXM7IEFwcGwuIE1pY3JvYmlvbC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NzU7ICBjaXRhdGlvbl9wYWdlcz05NTUtOTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9pbmZvcm1hdGljcyBjaGFsbGVuZ2VzIG9mIG5ldyBzZXF1ZW5jaW5nIHRlY2hub2xvZ3k7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNDItMTQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OQ0JJIFJlZmVyZW5jZSBTZXF1ZW5jZSAoUmVmU2VxKTogYSBjdXJhdGVkIG5vbi1yZWR1bmRhbnQgc2VxdWVuY2UgZGF0YWJhc2Ugb2YgZ2Vub21lcywgdHJhbnNjcmlwdHMgYW5kIHByb3RlaW5zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1ENjEtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBuZWlnaGJvci1qb2luaW5nIG1ldGhvZDogYSBuZXcgbWV0aG9kIGZvciByZWNvbnN0cnVjdGluZyBwaHlsb2dlbmV0aWMgdHJlZXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQwNi00MjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiBlbnRlcm9oZW1vcnJoYWdpYyBFc2NoZXJpY2hpYSBjb2xpIE8xNTc6SDcgYW5kIGdlbm9taWMgY29tcGFyaXNvbiB3aXRoIGEgbGFib3JhdG9yeSBzdHJhaW4gSy0xMjsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTExLTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaXZlcnNpZmljYXRpb24gb2YgRXNjaGVyaWNoaWEgY29saSBnZW5vbWVzOiBhcmUgYmFjdGVyaW9waGFnZXMgdGhlIG1ham9yIGNvbnRyaWJ1dG9ycz87IFRyZW5zIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTQ4MS00ODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhldXJpc3RpYyBhcHByb2FjaCB0byBkZXJpdmluZyBtb2RlbHMgZm9yIGdlbmUgZmluZGluZzsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MzkxMS0zOTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgM+KAsi10ZXJtaW5hbCBzZXF1ZW5jZSBvZiBFc2NoZXJpY2hpYSBjb2xpIDE2UyByaWJvc29tYWwgUk5BOiBDb21wbGVtZW50YXJ5IHRvIG5vbnNlbnNlIHRyaXBsZXRzIGFuZCByaWJvc29tZSBiaW5kaW5nIHNpdGVzOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFUuIFMuIEEuOyAgY2l0YXRpb25feWVhcj0xOTc0OyAgY2l0YXRpb25fdm9sdW1lPTcxOyAgY2l0YXRpb25fcGFnZXM9MTM0Mi0xMzQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2N1cmF0ZSBwaHlsb2dlbmV0aWMgY2xhc3NpZmljYXRpb24gb2YgdmFyaWFibGUtbGVuZ3RoIEROQSBmcmFnbWVudHM7IE5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NjMtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VuZXRpYyBjbGFzc2lmaWNhdGlvbiBvZiBzaG9ydCBlbnZpcm9ubWVudGFsIEROQSBmcmFnbWVudHM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTIyMzAtMjIzOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE1LzYvMzg3LzEzNzY1NDcvZHNuMDI3LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBSZWNlbnQgYWR2YW5jZXMgaW4gRE5BIHNlcXVlbmNlcnMgYXJlIGFjY2VsZXJhdGluZyBnZW5vbWUgc2VxdWVuY2luZywgZXNwZWNpYWxseSBpbiBtaWNyb2JlcywgYW5kIGNvbXBsZXRlIGFuZCBkcmFmdCBnZW5vbWVzIGZyb20gdmFyaW91cyBzcGVjaWVzIGhhdiIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NTEyODc3JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZEc1FM7nKqqDSx297YWd/EAFK2zM" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f6%2f387%2f512877" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f6%2f387%2f512877" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=20280" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/6/1/m_cover.gif?Expires=2147483647&amp;Signature=cbcaSTnXmSAtcWAqInhnHikjrANtVVQT69rwVkS2N2-Rvvq~3o4NF54oja~GIUWptdFQr454edog~WN90FgAfPnwCCNM9uZ3DuEnj-WTN3JyklWsccNFHC6w0Mg4O1XsO2YSzFEaGsd~LWUMMgT-JRIdN1RQUYUQEdvCgKjLmwjDjkQDpQkgPjsYyRLj1QLXlpM4IsivMdAnqm9LmZlXnvMUUp5UVsqaQBEzVLg9CZ1N~86MZnaIY5LkZ1lOjOrEUjr78DNfXaNPxd2goUtJ24j8goSO16w~p5Co8ZBl8FnseAHBtje4N~aGVfSKUlSXEIVKmxpcUZsLAtBrli2ACg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/6">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7969097">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7969097">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7969099">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7969099">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7969103">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7969103">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7969116">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7969116">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7969146">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7969146">Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7969148">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7969148">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7969150">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7969150">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7969152">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7969152">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/6/375/512736">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">MetaGeneAnnotator: Detecting Species-Specific Patterns of Ribosomal Binding Site for Precise Gene Prediction in Anonymous Prokaryotic and Phage Genomes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Noguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Advanced Science and Technology Research Group</div>, <div class="insititution">Mitsubishi Research Institute, Inc.</div>, <div class="addr-line">2-3-6 Otemachi, Chiyoda-ku, Tokyo 100-8141</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 3-3277-0556. Fax. <div class="fax">+81 3-3277-0568</div>. E-mail: <a href="mailto:nog@mri.co.jp" target="_blank">nog@mri.co.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Noguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Noguchi H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Noguchi H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideki Noguchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takeaki Taniguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Advanced Science and Technology Research Group</div>, <div class="insititution">Mitsubishi Research Institute, Inc.</div>, <div class="addr-line">2-3-6 Otemachi, Chiyoda-ku, Tokyo 100-8141</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takeaki+Taniguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Taniguchi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Taniguchi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takeaki Taniguchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takehiko Itoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Advanced Science and Technology Research Group</div>, <div class="insititution">Mitsubishi Research Institute, Inc.</div>, <div class="addr-line">2-3-6 Otemachi, Chiyoda-ku, Tokyo 100-8141</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takehiko+Itoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itoh T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Itoh T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takehiko Itoh</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Masahira Hattori</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 6, 1 December 2008, Pages 387–396, <a href='https://doi.org/10.1093/dnares/dsn027'>https://doi.org/10.1093/dnares/dsn027</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">21 October 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">19 August 2008</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">24 September 2008</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="512877" href="/dnaresearch/article-pdf/15/6/387/1376547/dsn027.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Hideki Noguchi, Takeaki Taniguchi, Takehiko Itoh;  MetaGeneAnnotator: Detecting Species-Specific Patterns of Ribosomal Binding Site for Precise Gene Prediction in Anonymous Prokaryotic and Phage Genomes, <em>DNA Research</em>, Volume 15, Issue 6, 1 December 2008, Pages 387–396, <a href='https://doi.org/10.1093/dnares/dsn027'>https://doi.org/10.1093/dnares/dsn027</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/512877?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/512877?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/512877?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/512877?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/512877?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/512877?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/512877?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=MetaGeneAnnotator%3A%20Detecting%20Species-Specific%20Patterns%20of%20Ribosomal%20Binding%20Site%20for%20Precise%20Gene%20Prediction%20in%20Anonymous%20Prokaryotic%20and%20Phage%20Genomes&amp;publicationDate=2008-10-21&amp;volumeNum=15&amp;issueNum=6&amp;author=Noguchi%2C%20Hideki%3B%20Taniguchi%2C%20Takeaki&amp;startPage=387&amp;endPage=396&amp;contentId=10.1093%2Fdnares%2Fdsn027&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f6%2f387%2f512877" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7969097 id="7969097" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Recent advances in DNA sequencers are accelerating genome sequencing, especially in microbes, and complete and draft genomes from various species have been sequenced in rapid succession. Here, we present a comprehensive gene prediction tool, the MetaGeneAnnotator (MGA), which precisely predicts all kinds of prokaryotic genes from a single or a set of anonymous genomic sequences having a variety of lengths. The MGA integrates statistical models of prophage genes, in addition to those of bacterial and archaeal genes, and also uses a self-training model from input sequences for predictions. As a result, the MGA sensitively detects not only typical genes but also atypical genes, such as horizontally transferred and prophage genes in a prokaryotic genome. In this paper, we also propose a novel approach for analyzing the ribosomal binding site (RBS), which enables us to detect species-specific patterns of the RBSs. The MGA has the ingenious RBS model based on this approach, and precisely predicts translation starts of genes. The MGA also succeeds in improving prediction accuracies for short sequences by using the adapted RBS models (96% sensitivity and 93% specificity for 700 bp fragments). These features of the MGA expedite wide ranges of microbial genome studies, such as genome annotations and metagenome analyses.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="bioinformatics">bioinformatics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="gene-finding">gene-finding</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="prokaryote">prokaryote</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="phage">phage</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;ribosomal binding site&quot;">ribosomal binding site</a></div>                            <h2 scrollto-destination=7969099 id="7969099" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Identification of genes on genomic sequences is the indispensable first step in every genome analysis, including individual genome analysis of a single organism and metagenomic analyses. Sequence similarity-based methods of gene predictions enable us to detect reliably the genes if their DNA or amino acid sequences have strong similarities to those of known genes. However, a significant portion of genes has no sequence similarities to known genes, and <em>ab initio</em> gene-finding methods are necessary for identifying all genes on newly sequenced microbial genomes, particularly those of uncharacterized or poorly characterized species. Computational gene finding from genomic sequences has a long history<sup><a reveal-id="DSN027C1  DSN027C3" data-open="DSN027C1  DSN027C3" class="link link-ref link-reveal xref-bibr">1–3</a></sup>, and a number of tools have been developed for predicting prokaryotic genes. These gene-finding tools have been widely used for annotation processes of prokaryotic genomes.</p><p>Although conventional gene-finding tools have achieved extremely high prediction performances, they have some critical limitations. Most conventional tools require predetermined statistical models of the known genes of a target species<sup><a reveal-id="DSN027C4  DSN027C11" data-open="DSN027C4  DSN027C11" class="link link-ref link-reveal xref-bibr">4–11</a></sup> or a long enough input sequence for statistical models to perform self-training<sup><a reveal-id="DSN027C12  DSN027C16" data-open="DSN027C12  DSN027C16" class="link link-ref link-reveal xref-bibr">12–16</a></sup>. This is because the tools are designed to predict genes on complete genomes having several million base pairs. However, a target genomic sequence is not always long enough. For example, second-generation DNA sequencers, which have put high throughput sequencing into practice, especially those of microbial genomes<sup><a reveal-id="DSN027C17" data-open="DSN027C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSN027C18" data-open="DSN027C18" class="link link-ref link-reveal xref-bibr">18</a></sup>, produce vast amounts of very short sequence reads. The short reads are assembled into some longer contig sequences, but the contigs are usually still short [far shorter than 1 mega bases (Mb)]<sup><a reveal-id="DSN027C19  DSN027C22" data-open="DSN027C19  DSN027C22" class="link link-ref link-reveal xref-bibr">19–22</a></sup>. A fosmid clone, which has ∼40 kb in insert length, is another example of a short genomic sequence. Moreover, metagenomic analyses produce large amounts of short sequences derived from multiple species’ genomes. Most of the conventional gene-finding tools cannot be applied to such sequences. MetaGene<sup><a reveal-id="DSN027C23" data-open="DSN027C23" class="link link-ref link-reveal xref-bibr">23</a></sup> (MG) is one of the new tools that is applicable to gene prediction on such short anonymous sequences.</p><p>MG is a gene-finding program originally developed for metagenomic sequence data, which is a mixture of (short) sequences derived from various prokaryotic genomes. MG assumes correlations between the GC content and the di-codon frequencies of an input sequence, and enables us to predict genes accurately on short anonymous sequences without any training. MG can be successfully applied to wide varieties of prokaryotic genomic sequences<sup><a reveal-id="DSN027C24  DSN027C27" data-open="DSN027C24  DSN027C27" class="link link-ref link-reveal xref-bibr">24–27</a></sup>, but two major limitations exist: one is the lack of a ribosomal binding site (RBS) model, and the other is less sensitivity to atypical genes, whose codon usages are different from those of typical genes. When MG is applied to very short sequences containing one or two partial genes, these limitations are not significant. However, such limitations are undesirable when MG is applied to longer genomic sequences for precise annotations. To overcome these limitations and to improve the usability of the program, we developed a new version of the MG, the MetaGeneAnnotator (MGA). The MGA has statistical models of prophage genes and can automatically detect them in addition to chromosome backbone genes even when input genomic sequences have mosaic structures attributed to lateral gene transfers and/or phage infections. The MGA also has an adaptable the RBS model based on complementary sequences of the 3′ tail of 16S ribosomal RNA, and precisely predicts translation starts of genes even when input genomic sequences are short and anonymous sequences. These features of the MGA remarkably improve prediction accuracies of genes on a wide range of prokaryotic genomes. Here, we report the results of a performance test of the MGA applied to various types of genomic sequences, such as complete genomes, plasmids and their subsequences of various lengths, under conditions of anonymity.</p>                            <h2 scrollto-destination=7969103 id="7969103" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7969104 id="7969104" class="section-title" data-legacy-id=s2a>2.1. Construction of prophage gene model</h3>
<p>In addition to the bacterial and archaeal gene models of MG<sup><a reveal-id="DSN027C23" data-open="DSN027C23" class="link link-ref link-reveal xref-bibr">23</a></sup>, prophage models were constructed as follows. Genomic sequences and their annotations for 439 phages were obtained from the RefSeq database<sup><a reveal-id="DSN027C28" data-open="DSN027C28" class="link link-ref link-reveal xref-bibr">28</a></sup> (release 27). As a preprocessing, a mono-codon usage was calculated from each phage genome, and the Euclidean distances of all pairs of the codon usages were calculated. When the distance between two phages’ usages was &lt;0.02, one of them was removed from the dataset because they might have been related (or identical) phages. Then, the codon frequencies of the remaining 244 phages were plotted against their GC contents, and we confirmed that the codon frequencies of phage were highly correlated with their GC contents, as seen in bacterial and archaeal genomes. For gene prediction, the MGA used di-codon frequencies that represent conditional probabilities of codon occurrences providing a previous codon (61 × 61 frequencies). Because each phage did not have enough genes to calculate di-codon frequencies, phage genomes having about the same range of GC contents were treated as a unit, and then di-codon frequencies were calculated from all genes annotated in the grouped genomes. Finally, a logistic regression analysis was performed in the same manner as the prokaryotic di-codon model construction in MG.</p>                            <h3 scrollto-destination=7969106 id="7969106" class="section-title" data-legacy-id=s2b>2.2. Procedures for predicting typical and atypical genes</h3>
<div class="block-child-p">The self-training model for typical genes is constructed as follows. Initially, genes are predicted using an optimal set of the di-codon regression models (bacterial, archaeal or prophage models). Then, these predicted genes are used for the self-training of the di-codon statistics of typical genes. The self-training model is defined as the weighted averages of the di-codon frequencies derived from the predicted genes, and from the regression models used for the initial prediction. A di-codon frequency of the self-training model, <em>f</em><sub>self</sub>, is defined by the frequency of the predicted genes, <em>f</em><sub>pred</sub>, and of the regression models, <em>f</em><sub>reg</sub>, as follows: <a id="DSN027M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN027M1"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn02707.gif?Expires=2147483647&Signature=XoAT0g89SPU00YPNIU79s0lq7V~ysD~FpZ8OIIlG3xlkIDrGT1~whiX9QuGT~iJl~OYIN2kIuzTxY-mL0CBFAW6omY2jbdbTRBngq3BNs0BTh-RZ0NeHB3F38RRTwp14QXSCJThET~cUGV6JiWG0Lg3-419kn4o35vZvZiTmVWAXgkz6vsF1DWbAsCKGVkYYq-pN5KDT5df3vw5F3D9VWo2S7ARrVwz5fgbctUncQn-oSnl72AXRerlH4zmj6Ltv-uRPnO2DDmyASRltdHltu3baCh5WGZLOM0HcQ0NQdoOy0xbESgrurzOcPojugsCyPOJn3ukD1VGSseKv36tJZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsn02707.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/m_dsn02707.gif?Expires=2147483647&Signature=hnJlkfexMwocSTzWRyFgQqe99yZqkv8yOFrs-LcNkg9ZSUjJ4~pTUkixBfbKaPND1E83fLd~5CQXuJHVl9I1cRXW4Xy68XGv1l3C0~38X8PCBuzXmrnYs-mp13jUWluQ7ZIcM70VlYys59De5m52To5GcKv7qXRuz5TSklplVEqBUIgAmz9gulkI-Rca909k-gSETHep6Nb5yJLcVy~7SiBgzqhh4-CENLb2rob1UjrFm55y80dp5xnf9XW6WrJsvvm3AljVdxRruHJUAEYZG6ArEYCJsGFJfq7EtBGihRc5yxvnCUV0yGi9H49Dm4XHbY9i~rjlOWTayyxNxCHUuQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn02707.gif" /></a></div></div><div class="label">1</div></div> where <em>a</em> and <em>b</em> are codons, <em>k</em> and <em>l</em> are the numbers of di-codons used to calculate <em>f</em><sub>reg</sub> and <em>f</em><sub>pred</sub>, respectively. In the MGA, <em>k</em> was heuristically set to 30 000, which corresponds approximately to the number of di-codons in a 100 kb genomic sequence. The value was determined by testing prediction performances on the training data of MG<sup><a reveal-id="DSN027C23" data-open="DSN027C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and was meant to be enough to avoid overfitting of the self-training model to a few genes on a short input sequence. If a significant number of di-codons are extracted from the predicted genes, the self-training model is nearly equal to the di-codon frequencies of the predicted genes. If not, the self-training model comes closer to the di-codon frequencies derived from the regression models.</div><p>After training, four sets (self, bacteria, archaea and prophage) of di-codon frequencies are applied for scoring candidate genes. Unlike the original MG algorithm, each open-reading frame (ORF) is individually scored according to its own GC content in this step to detect atypical genes. Typical genes are expected to score the highest mark with the self-training model, and atypical genes to score the highest mark with one of the other models. Then, a maximal scoring combination of genes is calculated as the definitive prediction. While this procedure (ORF-by-ORF) is sensitive to atypical genes, some more false-positives are included in the prediction. So, the ORF-by-ORF procedure is applied only to the sequences longer than 5000 bp (containing multiple genes). For shorter sequences, the conventional procedure, in which all ORFs are scored by one of the four sets of the di-codon models according to the GC content of the input sequence, is applied.</p>                            <h3 scrollto-destination=7969111 id="7969111" class="section-title" data-legacy-id=s2c>2.3. The RBS map analysis and the RBS model construction</h3>
<p>We defined nine hexamers derived from the following sequence, which was complementary to a tail of 16S rRNA, as the potential RBS motifs: G(A/T)(A/T)AGGAGGT(G/A)ATC. Starting from the left, the motifs were named Motif-1, …, Motif-9 [e.g. Motif-3 is ‘(A/T)AGGAG’]. An exact match or one-base mismatch sequence of the motifs was sought against an upstream region of a start codon, and the best match motif and location were determined. In the RBS map analysis (see below), upstream sequences of the annotated start codons range from −2 to −21 were used for analysis. In the RBS prediction model, upstream sequences of the predicted start codons (in the previous step) range from −3 to −19 were used for model construction and prediction. The detected sequences were considered to be representative RBSs of the species, and the proportion of genes having representative RBSs (an RBS ratio, <em>w</em><sub>RBS</sub>) was stored for the use in scoring RBSs. Then, a two-dimensional frequency distribution of the representative RBSs was calculated to construct the RBS map. For the analysis, distances between the constructed RBS maps were defined by the Euclidean distance, and the neighbor joining method<sup><a reveal-id="DSN027C29" data-open="DSN027C29" class="link link-ref link-reveal xref-bibr">29</a></sup> was applied to make clusters of the RBS maps. This RBS map analysis was performed using 591 annotated microbial genomes obtained from the RefSeq database (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). As the RBS prediction model, a position weight matrix (PWM) for each motif was constructed using the representative RBS sequences detected earlier. In the prediction process, the RBS scores for all candidate genes were calculated using the constructed PWMs and the frequency distributions of the positions. Here, the RBS score, <em>S</em><sub>RBS</sub>, was heuristically weighted using a frequency of a motif <em>m</em>, <em>w</em><sub><em>m</em></sub>, and the RBS ratio (<em>w</em><sub>RBS</sub>) to reduce noise in less frequently used motifs.</p><div class="block-child-p"><a id="DSN027M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN027M2"><div class="fig-graphic"><a class="jumplink-placeholder" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn02708.gif?Expires=2147483647&Signature=D3Ppc9oGqTK1oNG28aWX5VaW5MRCXQ8OK6gTuzEVnVqB0r3oZEYdE36CIcplrG7P5cf7FrlcE7dbdrrGL59w37sgYbKJFIEF-7SfQcaLFqjpP5ty67i9UcvHMdVGAoLwvQ5KU6USljDq1s0XL5m5xZv3-yqvG7vqRvNzZoIpAtubVh0gSaOCQb-FRl80UnPEPnfjaKhdVvdbXZ-bYZqwNFO5Ch-S35MHKmmVvizxWckmdW~48mGmV5q7heH~ZfjofGm9qhltWqEOQFCsMMmiUCK3Yx-untRvWhLy8DqQFKtS~r-JYOKaYSArcvRMmbPIElwKrrakg0igvSZeb~yr5Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" id="dsn02708.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/m_dsn02708.gif?Expires=2147483647&Signature=3V2FopenR6hVRTIPhWwbJt3jcTY2G5Ap0c6iJNBbBuPae8~LkxTntEyb8CrzUARgW9QIBFDYQDZoNV2cvDaGxIwWVvz7fCvLj-YUJQJ6~AoHOlxBjnANFq6A7W-A1hVzRNFQow5ATDtOdPMRavqU01otlknFfwmdtKrMFRI45C5iMlPP6OM8L~jfjdCjE65fi6lPbRN-ZqC487ROBZtFyZhW~V5uyUyjN7~Yj6RtUqbhU8Gn3skciXKPCcwRJF7~I1ePLxx8-0xYKCaJ~mSLUFDD~Os64y8QKQXpD6uu-Kjkw2S2KMJSV5mnGzciHleNzoOC-R4i1cG234qd2cFElA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn02708.gif" /></a></div></div><div class="label">2</div></div> where <em>x</em><sub><em>i,j</em></sub> is an <em>i</em>th nucleotide of a hexamer <em>j</em> appeared in an upstream of a gene, <em>p</em><sub><em>m</em></sub> (<em>x</em><sub><em>i,j</em></sub>) is a frequency of <em>x</em><sub><em>i,j</em></sub> at a position <em>i</em> of a PWM for a motif <em>m</em>, and <em>q</em>(<em>x</em><sub><em>i,j</em></sub>) is a background frequency of <em>x</em><sub><em>i,j</em></sub> calculated from a GC content of an input sequence. A value of <em>w</em><sub><em>m</em></sub> was standardized, and was 1 when a motif <em>m</em> was the most frequently used. For each gene, the best motif, which marked the highest RBS score, was selected, and the RBS score was added to the score of the genes. Then, the optimal combination of genes with the recalculated scores was estimated by the dynamic programming procedure used in MG<sup><a reveal-id="DSN027C23" data-open="DSN027C23" class="link link-ref link-reveal xref-bibr">23</a></sup>. All of these steps were iterated until the prediction results stopped changing.</div>                            <h3 scrollto-destination=7969114 id="7969114" class="section-title" data-legacy-id=s2d>2.4. Performance evaluation</h3>
<p>Prediction performances of gene-finders were evaluated using datasets, including the MetaGene dataset<sup><a reveal-id="DSN027C23" data-open="DSN027C23" class="link link-ref link-reveal xref-bibr">23</a></sup>. The MetaGene dataset consists of nine bacterial and three archaeal genomic sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). In addition to these complete sequences, their subsequences (1 Mb, 500, 100, 40, 10, 5, 3 and 1 kb, 700 and 100 bp sequences) having 1× genome coverage (i.e. the total length of the subsequences is equal to the complete genome size) were also used for the evaluation. These sequences were not used for constructing statistical models of the MGA. The ratios of true-positives, including partially matching predictions with correct reading frames, relative to all annotated genes (sensitivity) and to all predicted genes (specificity) were used as indices for the evaluation. In addition, sensitivity to the start codons, in which only exactly matching predictions were counted as true-positives, was also utilized.</p>                            <h2 scrollto-destination=7969116 id="7969116" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=7969117 id="7969117" class="section-title" data-legacy-id=s3a>3.1. Predicting prophage genes</h3>
<p>The MGA is based on the algorithm of the MG and utilizes logistic regression models of the GC content and the di-codon frequencies<sup><a reveal-id="DSN027C23" data-open="DSN027C23" class="link link-ref link-reveal xref-bibr">23</a></sup> (di-codon models). In addition to the bacterial and archaeal di-codon models of MG, prophage models are constructed and integrated into the MGA (Fig. <a reveal-id="DSN027F1" data-open="DSN027F1" class="link link-reveal link-table xref-fig">1</a>A). Although the proportions of prophage genes in the prokaryotic genomes are ordinarily not so large, they usually have biologically important functions, such as pathogenicity and niche adaptation, in the organisms. Therefore, detecting prophage genes is fundamental to understanding the genetic background of an organism.</p>                        <a id="7969119" scrollto-destination="7969119"></a>
<div id="DSN027F1" data-id="DSN027F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/m_dsn02701.jpeg?Expires=2147483647&Signature=XIeKmnAU3YNkkF4lpJDhBpazpSluEB8b1Lz8gaB725IHgL2WYg-Gxw8~a49JlYI-6~JF8pRH6FjlOa9xJ~50DdFvgTWkKd30X3veLWH22dxtBVHB8QX~kD4yrYNt-fH7-oFZ77E0yPWdtZQigYoJNZ3f8Vd~3Qdw-PJmlHagZs8Rqk4PF01my7tOArq0oclf40FQkzoUgFhVEQhkQ9NxN~R2LjJ7YuoC7cSdepFQTSV4vqpB1QNt7md9bTwKFdZ8SMdI0-ve5kiE6utrChW~QHoaRlG8j0~4ytr1w5-s-lNVEjy7H88bvN2wFTCB0c84md8KnapN2~VAzDXcOfgoEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic diagram of the MGA algorithm. (A) Prediction protocol of the MGA. (B) ORF-by-ORF procedure." path-from-xml="dsn02701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7969119/dsn02701.jpeg" path-from-xml="dsn02701.jpeg" target="_blank">View large</a><a section="7969119" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn02701.jpeg?Expires=2147483647&Signature=AckHum6Go5-u0cbzM2IRT0j74yGyZ-KjsLepZxcZsUFAjLQq87SlgcbtvtvvwbDxYK1pYTZp8zG-7EjbjyT8OYPJ0lkaGvA0yEy~KDg6JMcYgrFqUk84PT~hQDoNDEsOU6vBfwSkF2ydmIsNOqQwBoPrT1JYCJ7sA5246fRnJEu42GwynFbV9sC-KjqmhQAEHx~MP4YZ-oqK5k7QLa6eX~8iIrY76FCcqFqaJ5TDbHawJdujjlz18cbDlu16JaSGd1jOQ4M96YoQz4gpBcnjWtYZtNijFoid14NcRODQtAon~52tYZdTPUXaFfTqze6EIG5czs32H~TGXZGrE6dFpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7969119&ar=512877&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic diagram of the MGA algorithm. (<strong>A</strong>) Prediction protocol of the MGA. (<strong>B</strong>) ORF-by-ORF procedure.</p></div></div><div content-id="DSN027F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/m_dsn02701.jpeg?Expires=2147483647&Signature=XIeKmnAU3YNkkF4lpJDhBpazpSluEB8b1Lz8gaB725IHgL2WYg-Gxw8~a49JlYI-6~JF8pRH6FjlOa9xJ~50DdFvgTWkKd30X3veLWH22dxtBVHB8QX~kD4yrYNt-fH7-oFZ77E0yPWdtZQigYoJNZ3f8Vd~3Qdw-PJmlHagZs8Rqk4PF01my7tOArq0oclf40FQkzoUgFhVEQhkQ9NxN~R2LjJ7YuoC7cSdepFQTSV4vqpB1QNt7md9bTwKFdZ8SMdI0-ve5kiE6utrChW~QHoaRlG8j0~4ytr1w5-s-lNVEjy7H88bvN2wFTCB0c84md8KnapN2~VAzDXcOfgoEg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A schematic diagram of the MGA algorithm. (A) Prediction protocol of the MGA. (B) ORF-by-ORF procedure." path-from-xml="dsn02701.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7969119/dsn02701.jpeg" path-from-xml="dsn02701.jpeg" target="_blank">View large</a><a section="7969119" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn02701.jpeg?Expires=2147483647&Signature=AckHum6Go5-u0cbzM2IRT0j74yGyZ-KjsLepZxcZsUFAjLQq87SlgcbtvtvvwbDxYK1pYTZp8zG-7EjbjyT8OYPJ0lkaGvA0yEy~KDg6JMcYgrFqUk84PT~hQDoNDEsOU6vBfwSkF2ydmIsNOqQwBoPrT1JYCJ7sA5246fRnJEu42GwynFbV9sC-KjqmhQAEHx~MP4YZ-oqK5k7QLa6eX~8iIrY76FCcqFqaJ5TDbHawJdujjlz18cbDlu16JaSGd1jOQ4M96YoQz4gpBcnjWtYZtNijFoid14NcRODQtAon~52tYZdTPUXaFfTqze6EIG5czs32H~TGXZGrE6dFpA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7969119&ar=512877&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02701.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A schematic diagram of the MGA algorithm. (<strong>A</strong>) Prediction protocol of the MGA. (<strong>B</strong>) ORF-by-ORF procedure.</p></div></div><p>Because most prophage genes have codon frequencies similar to those of bacteria and archaea, MG (and probably other prokaryotic gene finders as well) can predict prophage genes with relatively high accuracies (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). However, Fig. <a reveal-id="DSN027F2" data-open="DSN027F2" class="link link-reveal link-table xref-fig">2</a>A and B shows that certain other (non-codon) properties of prophage genes are different from those of prokaryotic genes: prophage genes are generally shorter (∼660 bp in average) than bacterial and archaeal genes (∼940 bp in average), and most genes are organized in tandem (&gt;90%). This means that gene densities are higher in prophage genomes than in prokaryotic genomes, and most genes are packed in a few operons. These observations and statistics, in addition to the prophage di-codon models, are utilized to predict prophage genes. As a result, the sensitivities of the MGA to prophage genes are remarkably improved (from 88 to 93%) without any decrease in specificity (90%) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>).</p>                        <a id="7969121" scrollto-destination="7969121"></a>
<div id="DSN027F2" data-id="DSN027F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/m_dsn02702.jpeg?Expires=2147483647&Signature=JZLPYcFwGseby-CUS2XYmqviWkyGEKIACkTuJKVe~t1u7n5~bZve2gp2Cf6xE8yCo853KpQqe2gM6K8Moo-ixKXtutzNGfubVgNfWMgaGXUDFDdNYbTMNc1HYDArUnRRzJV~gINNo4~4DD9SGKbno6rYE~nZzxh3om4R3stZ87VjPbCf89TGamKDNfJIvcBtf2wjo-zGImoaSFWAjTVnB2Iey8ne6yTFKxHZox3WG6X7h~eFvx2FpGDKk6ToRLL-KB6JyBnmFZkzzzmA-aYF4EdJ3xqOZFgJ0lMuf6CaoOD1Rd8dD7JwflXcxWULhZm0kQnhk9UqIH~oSgEq-u6cSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Statistics of prophage genes. (A) Frequency distributions of gene lengths in prokaryote and prophage. (B) Proportions of the consecutive gene arrangements in prokaryote and prophage." path-from-xml="dsn02702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7969121/dsn02702.jpeg" path-from-xml="dsn02702.jpeg" target="_blank">View large</a><a section="7969121" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn02702.jpeg?Expires=2147483647&Signature=cP1JfYcs1XeVaASeY3UHnHOInQy7rpY9SHBwjtniQSUB~aEs1Qt6ucsIg2-LI1BMrGZSIUhiHvPyEu29RQwOXQ2ZuHcLbz0ESCb4gpeQnxa7w3yUnmG2UnCt7MeGkfpjmA2Q1wr8HUxwLI0ltLZcyC55UXyQomwM9-5u-xQH3oBO8hlVEwQTkwczyJ~aH0yBMrxlTMajdJluS521SScOQmAd5agKNJZRVMZ-lVSkobJd~KL0Rg00feX4oHNQY9npkxHjn4Z4YUB05kbTwaqXebeuEgBFG9oB6PVxaCK7IQb83vh7q0gsbIQ7D-tHLB4f8heqEkvmRNeUYM6WFQnntg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7969121&ar=512877&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Statistics of prophage genes. (<strong>A</strong>) Frequency distributions of gene lengths in prokaryote and prophage. (<strong>B</strong>) Proportions of the consecutive gene arrangements in prokaryote and prophage.</p></div></div><div content-id="DSN027F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/m_dsn02702.jpeg?Expires=2147483647&Signature=JZLPYcFwGseby-CUS2XYmqviWkyGEKIACkTuJKVe~t1u7n5~bZve2gp2Cf6xE8yCo853KpQqe2gM6K8Moo-ixKXtutzNGfubVgNfWMgaGXUDFDdNYbTMNc1HYDArUnRRzJV~gINNo4~4DD9SGKbno6rYE~nZzxh3om4R3stZ87VjPbCf89TGamKDNfJIvcBtf2wjo-zGImoaSFWAjTVnB2Iey8ne6yTFKxHZox3WG6X7h~eFvx2FpGDKk6ToRLL-KB6JyBnmFZkzzzmA-aYF4EdJ3xqOZFgJ0lMuf6CaoOD1Rd8dD7JwflXcxWULhZm0kQnhk9UqIH~oSgEq-u6cSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Statistics of prophage genes. (A) Frequency distributions of gene lengths in prokaryote and prophage. (B) Proportions of the consecutive gene arrangements in prokaryote and prophage." path-from-xml="dsn02702.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7969121/dsn02702.jpeg" path-from-xml="dsn02702.jpeg" target="_blank">View large</a><a section="7969121" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn02702.jpeg?Expires=2147483647&Signature=cP1JfYcs1XeVaASeY3UHnHOInQy7rpY9SHBwjtniQSUB~aEs1Qt6ucsIg2-LI1BMrGZSIUhiHvPyEu29RQwOXQ2ZuHcLbz0ESCb4gpeQnxa7w3yUnmG2UnCt7MeGkfpjmA2Q1wr8HUxwLI0ltLZcyC55UXyQomwM9-5u-xQH3oBO8hlVEwQTkwczyJ~aH0yBMrxlTMajdJluS521SScOQmAd5agKNJZRVMZ-lVSkobJd~KL0Rg00feX4oHNQY9npkxHjn4Z4YUB05kbTwaqXebeuEgBFG9oB6PVxaCK7IQb83vh7q0gsbIQ7D-tHLB4f8heqEkvmRNeUYM6WFQnntg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7969121&ar=512877&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02702.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Statistics of prophage genes. (<strong>A</strong>) Frequency distributions of gene lengths in prokaryote and prophage. (<strong>B</strong>) Proportions of the consecutive gene arrangements in prokaryote and prophage.</p></div></div>                            <h3 scrollto-destination=7969122 id="7969122" class="section-title" data-legacy-id=s3b>3.2. Predicting atypical genes</h3>
<p>MG predicts genes using the di-codon frequencies (and other parameters) estimated by the GC content of an input genomic sequence. That is to say, all genes in the same genomic sequence are predicted by the same set of di-codon frequencies. In this procedure, typical genes can be accurately and specifically predicted, but atypical genes, such as horizontally transferred and prophage genes, cannot be detected because their di-codon frequencies are different from those of typical genes. To overcome this limitation, we employ an ORF-by-ORF procedure, in which each candidate ORF is treated as an individual anonymous sequence (Fig. <a reveal-id="DSN027F1" data-open="DSN027F1" class="link link-reveal link-table xref-fig">1</a>B). This procedure assumes that every ORF has a potentially different origin and contributes to improving the sensitivities of the MGA to atypical genes.</p><p>To predict properly the typical genes under the ORF-by-ORF procedure, we arranged a self-training model of di-codon frequencies in addition to the logistic regression models (Fig. <a reveal-id="DSN027F1" data-open="DSN027F1" class="link link-reveal link-table xref-fig">1</a>A). In the self-training model, di-codon frequencies are calculated from the initially predicted genes using the conventional scoring procedure of the MG, and then the weighted averages of di-codon frequencies derived from the predicted genes and from the regression models are calculated as the di-codon frequencies of typical genes. The self-training model fits well to typical genes compared with the regression models, and improves both sensitivity and specificity of the MGA to typical genes.</p><p>To evaluate the effectiveness of these procedures, prediction performances were tested on the chromosome and plasmid of enterohemorrhagic <em>Escherichia coli</em> O157:H7 strain Sakai<sup><a reveal-id="DSN027C30" data-open="DSN027C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSN027C31" data-open="DSN027C31" class="link link-ref link-reveal xref-bibr">31</a></sup> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Sensitivities of the MGA are extremely higher than those of the MG, especially in S-loops, which are O157:H7 strain-specific regions identified from comparisons with the <em>E. coli</em> K12 genome and that contain many horizontally acquired virulence-related genes. Higher sensitivities are also observed for a large virulence plasmid (pO157). Specificities of the MGA are slightly lower than those of MG, but are still higher than those of GeneMarkS<sup><a reveal-id="DSN027C16" data-open="DSN027C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and GeneMark.heuristics<sup><a reveal-id="DSN027C32" data-open="DSN027C32" class="link link-ref link-reveal xref-bibr">32</a></sup>. These results indicate that our ORF-by-ORF procedure works well for predicting atypical genes and can be applied to genomes having mosaic structures with high specificity.</p>                            <h3 scrollto-destination=7969126 id="7969126" class="section-title" data-legacy-id=s3c>3.3. Analyzing species-specific patterns of the RBS</h3>
<p>The other notable feature of the MGA is an adaptable model of the RBS. An RBS, which is also known as the Shine-Dalgarno (SD) sequence<sup><a reveal-id="DSN027C33" data-open="DSN027C33" class="link link-ref link-reveal xref-bibr">33</a></sup>, is located on the 5′ flanking region of the start codon, and interacts with a part of the 3′ end of 16S ribosomal RNA (rRNA) to control translation initiations of the gene. Although RBSs are complementary to the 3′ tail of the 16S rRNA in every organism, their sequences (motifs) and preferred locations relative to start codons (or ‘spacer’ lengths) differ slightly from organism to organism. In gene-finding programs, the Gibbs sampling algorithm is widely used for training the motifs and the spacer length distribution of the target species’ RBSs<sup><a reveal-id="DSN027C16" data-open="DSN027C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSN027C17" data-open="DSN027C17" class="link link-ref link-reveal xref-bibr">17</a></sup>, although this algorithm takes no thought for the observation that the RBSs are complementary to the tail of the 16S rRNA. This approach basically assumes one motif and one frequency distribution of the spacer lengths in each species. However, our analysis suggests that this assumption is not appropriate for most species.</p><p>We examined the upstream sequences of annotated genes from 229 prokaryotic genomes and constructed RBS maps that show a two-dimensional frequency distribution of the best match motif (out of the nine candidate motifs we suggested) and the spacer lengths of the RBSs for each species. The average RBS map (Fig. <a reveal-id="DSN027F3" data-open="DSN027F3" class="link link-reveal link-table xref-fig">3</a>) shows that Motif-3 is most frequently used, but all nine motifs are potential RBSs. The higher the motif number, the shorter the spacer lengths. This is reasonable because it means that the position of the main body of the 16S rRNA is fixed even if the hybridization position of 16S rRNA tail is moved.</p>                        <a id="7969129" scrollto-destination="7969129"></a>
<div id="DSN027F3" data-id="DSN027F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/m_dsn02703.jpeg?Expires=2147483647&Signature=tAnhnHYz45BQN4FbErhRZCX-Hx7TuSgIODu3JMYt08pLomuZ1aQEkBJmJ-wWpm9tAL3OA0TFQ-XymUZiopEmgfP7~XRHlMw-CNN17gLlQMc4EGQ-yBuXzmuvKT5SjH~mm9ysQ0bwUlulE-1WnFKx0TxmKSN3rHzm7mHonOJ3x5Rcc~MidiUk2BXr92vPIxTSwadUYNwh4-1L1i90wqERrX48yiEepKIf~SKYDbXFmGkDnY0rswfthmmR9w-QTi7Br53iJ-u1qO6DkY5HNBOXbZ72fjs6cNs032dkm-CH4qITPvpExgbh0psmR~Zm1GownPgyI0830yvU4U5STN8q6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The average RBS map. The horizontal axis represents relative positions from the start codons [equal to –(spacer length+1)], and the vertical axis represents motif numbers." path-from-xml="dsn02703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7969129/dsn02703.jpeg" path-from-xml="dsn02703.jpeg" target="_blank">View large</a><a section="7969129" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn02703.jpeg?Expires=2147483647&Signature=kvBBFBwbUCzYq8WsDeH8-nbPB3GbCKGBS4bnV8-lrhKZqAgYyxKJuRJHr9BQJo4S7ixY0JGTEkvkqaiIQImuNa5IaRY-7CYZz~xGDoq6JMkldv1e94xenzVGq941RjRe3IWwV4KL4JVpnCT7Ui3Fmb5vLW5ECMBTY9Kr5ciMVtXu3vTHM1Vv71OjBjG3hdJ4hTHd0QqaDo4FLLuOqpcUGkOIp9-J7Mq0z-Oz0OxmmRjpQPhuBAS1RmUxyUt9HPP78DfDu6nnRoOFC2pgUm2nu7sPrP7NxAucWYE~nGVXuo5fZ4AD7LK0CdoftV8zWe1ANvMN38OgwcWg6U9iwxdJgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7969129&ar=512877&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The average RBS map. The horizontal axis represents relative positions from the start codons [equal to –(spacer length+1)], and the vertical axis represents motif numbers.</p></div></div><div content-id="DSN027F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/m_dsn02703.jpeg?Expires=2147483647&Signature=tAnhnHYz45BQN4FbErhRZCX-Hx7TuSgIODu3JMYt08pLomuZ1aQEkBJmJ-wWpm9tAL3OA0TFQ-XymUZiopEmgfP7~XRHlMw-CNN17gLlQMc4EGQ-yBuXzmuvKT5SjH~mm9ysQ0bwUlulE-1WnFKx0TxmKSN3rHzm7mHonOJ3x5Rcc~MidiUk2BXr92vPIxTSwadUYNwh4-1L1i90wqERrX48yiEepKIf~SKYDbXFmGkDnY0rswfthmmR9w-QTi7Br53iJ-u1qO6DkY5HNBOXbZ72fjs6cNs032dkm-CH4qITPvpExgbh0psmR~Zm1GownPgyI0830yvU4U5STN8q6g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The average RBS map. The horizontal axis represents relative positions from the start codons [equal to –(spacer length+1)], and the vertical axis represents motif numbers." path-from-xml="dsn02703.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7969129/dsn02703.jpeg" path-from-xml="dsn02703.jpeg" target="_blank">View large</a><a section="7969129" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn02703.jpeg?Expires=2147483647&Signature=kvBBFBwbUCzYq8WsDeH8-nbPB3GbCKGBS4bnV8-lrhKZqAgYyxKJuRJHr9BQJo4S7ixY0JGTEkvkqaiIQImuNa5IaRY-7CYZz~xGDoq6JMkldv1e94xenzVGq941RjRe3IWwV4KL4JVpnCT7Ui3Fmb5vLW5ECMBTY9Kr5ciMVtXu3vTHM1Vv71OjBjG3hdJ4hTHd0QqaDo4FLLuOqpcUGkOIp9-J7Mq0z-Oz0OxmmRjpQPhuBAS1RmUxyUt9HPP78DfDu6nnRoOFC2pgUm2nu7sPrP7NxAucWYE~nGVXuo5fZ4AD7LK0CdoftV8zWe1ANvMN38OgwcWg6U9iwxdJgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7969129&ar=512877&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02703.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The average RBS map. The horizontal axis represents relative positions from the start codons [equal to –(spacer length+1)], and the vertical axis represents motif numbers.</p></div></div><p>The observed patterns of the RBS maps vary from organism to organism, while phylogenetically related species show similar patterns (Figs <a reveal-id="DSN027F4" data-open="DSN027F4" class="link link-reveal link-table xref-fig">4</a> and <a reveal-id="DSN027F5" data-open="DSN027F5" class="link link-reveal link-table xref-fig">5</a>). Although some species such as <em>Helicobacter pylori</em> (Fig. <a reveal-id="DSN027F5" data-open="DSN027F5" class="link link-reveal link-table xref-fig">5</a>A) and <em>Buchnera aphidicola</em>, predominantly use Motif-2 and -3 and are therefore congruous with the one motif assumption described earlier many other species show broader distributions. For example, some <em>Firmicutes</em>, including <em>Clostridium</em> (Fig. <a reveal-id="DSN027F5" data-open="DSN027F5" class="link link-reveal link-table xref-fig">5</a>B), and <em>Thermotogae</em> indicate broad and clear patterns of the RBS maps. Some archaea, including methanogens (Fig. <a reveal-id="DSN027F5" data-open="DSN027F5" class="link link-reveal link-table xref-fig">5</a>C), also indicate broad patterns, but the preferred motifs are different between these bacteria and archaea (e.g. <em>Clostridium acetobutylicum</em> prefers Motif-3 and -4, but <em>Methanobrevibacter smithii</em> prefers Motif-8.). Overall, bacterial species tend to prefer motifs of 3′ side of a tail of 16S rRNA, while archaeal ones tend to prefer motifs of 5′ side of the tail. Only very weak signals of the RBS motifs are found in some species belonging to <em>Bacteroidetes</em> and <em>Cyanobacteria</em> (Fig. <a reveal-id="DSN027F5" data-open="DSN027F5" class="link link-reveal link-table xref-fig">5</a>D). In these species, no other significant motif is found. These results suggest that our RBS map with nine fixed motifs is effective for capturing the species-specific pattern of the RBSs. Hence, we used this two-dimensional frequency distribution and the PWMs of the nine RBS motifs as an RBS model of the MGA. Parameters of the RBS model are estimated from upstream sequences of predicted genes. To predict the RBSs on very short input sequences (having no training data), a general model of the RBS was manually constructed, based on the average RBS map and was integrated into the MGA (Fig. <a reveal-id="DSN027F1" data-open="DSN027F1" class="link link-reveal link-table xref-fig">1</a>A).</p>                        <a id="7969131" scrollto-destination="7969131"></a>
<div id="DSN027F4" data-id="DSN027F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/m_dsn02704.jpeg?Expires=2147483647&Signature=ik5ApfYGCIz4zBRui~cLx5bFxBiWNYnMPYvPLTbxpTssmF-9-XDg5yGrEfspeOWgw4WZAmi56ppe8S3nZqQwbqTnV7mI7PSp95qqLiUTCAH1pcj-z-H2g1SQkc9p4pN7n-JbbiljE1UTHcx8jUZpc3WRYcdcNUYJrM7DWXI9tg3aK3v9JACdAT42wxBkUEaAIF0kq0V657xEqtZtCa2fCFc0Ldhv8CB1iWvWcOpOLfoGmMaHvfUEyd4S-YRjPV1lH9N5IFQy7Egt-dU0oIM6SCFHGeYE2aDmeJyNIG5vI4sAIh2kah3Mkn42YHd8Ly1xB3jMRpWg0isKBIF0aKXtYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The clustering result of the RBS maps derived from 229 of 591 prokaryotic genomes (one species per genus)." path-from-xml="dsn02704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7969131/dsn02704.jpeg" path-from-xml="dsn02704.jpeg" target="_blank">View large</a><a section="7969131" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn02704.jpeg?Expires=2147483647&Signature=ezRMk6fKzcKH~odWkz3VdXBVzsAVvb61FLOwlVvoeXP0SnubwwAJSezGbga4tcfU-iZIo6p3UdWIwIvf53x7IHtGC1kdYkVDQxzM6BHaLsD5Ett2I9f5ClOrVy3ZJLN2XGE4GH8N6i~jUNSznYGzfxEVw7POkaUo6vzxiNWhw~FNK-zU41E97JRUGfo6Pcg2mzuKftMJeIoVLA1lPWTIZUnjck53hzpap4~V8iY2AlQGGGrmNP9H9Ezu7JTr8yVBczrubjkt0ysd0kxKs6FAM5UXRXfV3xjf9DNz65BGHF8YM12ZJaTUqJsa6vDwzAKeuE1jLghqTjId7dbQOcVmug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7969131&ar=512877&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The clustering result of the RBS maps derived from 229 of 591 prokaryotic genomes (one species per genus).</p></div></div><div content-id="DSN027F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/m_dsn02704.jpeg?Expires=2147483647&Signature=ik5ApfYGCIz4zBRui~cLx5bFxBiWNYnMPYvPLTbxpTssmF-9-XDg5yGrEfspeOWgw4WZAmi56ppe8S3nZqQwbqTnV7mI7PSp95qqLiUTCAH1pcj-z-H2g1SQkc9p4pN7n-JbbiljE1UTHcx8jUZpc3WRYcdcNUYJrM7DWXI9tg3aK3v9JACdAT42wxBkUEaAIF0kq0V657xEqtZtCa2fCFc0Ldhv8CB1iWvWcOpOLfoGmMaHvfUEyd4S-YRjPV1lH9N5IFQy7Egt-dU0oIM6SCFHGeYE2aDmeJyNIG5vI4sAIh2kah3Mkn42YHd8Ly1xB3jMRpWg0isKBIF0aKXtYg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The clustering result of the RBS maps derived from 229 of 591 prokaryotic genomes (one species per genus)." path-from-xml="dsn02704.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7969131/dsn02704.jpeg" path-from-xml="dsn02704.jpeg" target="_blank">View large</a><a section="7969131" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn02704.jpeg?Expires=2147483647&Signature=ezRMk6fKzcKH~odWkz3VdXBVzsAVvb61FLOwlVvoeXP0SnubwwAJSezGbga4tcfU-iZIo6p3UdWIwIvf53x7IHtGC1kdYkVDQxzM6BHaLsD5Ett2I9f5ClOrVy3ZJLN2XGE4GH8N6i~jUNSznYGzfxEVw7POkaUo6vzxiNWhw~FNK-zU41E97JRUGfo6Pcg2mzuKftMJeIoVLA1lPWTIZUnjck53hzpap4~V8iY2AlQGGGrmNP9H9Ezu7JTr8yVBczrubjkt0ysd0kxKs6FAM5UXRXfV3xjf9DNz65BGHF8YM12ZJaTUqJsa6vDwzAKeuE1jLghqTjId7dbQOcVmug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7969131&ar=512877&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02704.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The clustering result of the RBS maps derived from 229 of 591 prokaryotic genomes (one species per genus).</p></div></div>                        <a id="7969132" scrollto-destination="7969132"></a>
<div id="DSN027F5" data-id="DSN027F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/m_dsn02705.jpeg?Expires=2147483647&Signature=TB2RF~qgDDrVoJsVQT4mpX9wfT2RFhJD7vKWn8UbOWNLA~DoFeN8z96WABkTey~8B~5dEyIW70222fipsSme-2REMyJ1H3ygXV6vEO5H8BI2GhYmCH443GLPr4f-Cj~8-8EYdklWj0SIJxggU9GcvJQm1riuMi3Ok-112ZJ1Frp0O74EBxcUpochB4pjnpH1htIGQAUDN9N~0AsAnTm~97YoOvrEoOtLwI7xr8IorLHR93vQv9F6cmQDuHA-lT683JmjWGgM8gbh0OWT9KkYVNy93OsCcBD1QHI-JsuUROrgZ8FQLIzVIouLyEpDR-1tr50TkMxZV2wFlpR4QmL2AQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The RBS maps for four species. (A) Helicobacter pylori (B) Clostridium acetobutylicum (C) Methanobrevibacter smithii (D) Prochlorococcus marinus." path-from-xml="dsn02705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7969132/dsn02705.jpeg" path-from-xml="dsn02705.jpeg" target="_blank">View large</a><a section="7969132" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn02705.jpeg?Expires=2147483647&Signature=dnzoLMddvROCZvcBEB58r5cvLfKFj~wGAonD8XoT74ZHYzFzqFJSDLCC20pGEjwtRRkC6b-ovRNLJtAK8tMhd18nKXqWpQKaSDR9vshpKpJ0pPQ3N4eyXUY0lAL8c~Ifsw3Z2zXUkoVDD4aHB~HuhfEuZfF3lj7WM2P3wdjU~jgwbzJCPvGWsD7E99eDMNGsgnOwG2DK1gU244YYeXhsNA43ykUFMW1Te-fJleAGF14AaTl0nT31C~kfF9goeV~dMJ4b27AMV4dNt~2f~cSJbcrRys7tvbArXYD2xD4qaf9ZQY5kiT~x0FKk0kHlaBRkEuv~eLypzz64NyerALs4sw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7969132&ar=512877&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The RBS maps for four species. (<strong>A</strong>) <em>Helicobacter pylori</em> (<strong>B</strong>) <em>Clostridium acetobutylicum</em> (<strong>C</strong>) <em>Methanobrevibacter smithii</em> (<strong>D</strong>) <em>Prochlorococcus marinus</em>.</p></div></div><div content-id="DSN027F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/m_dsn02705.jpeg?Expires=2147483647&Signature=TB2RF~qgDDrVoJsVQT4mpX9wfT2RFhJD7vKWn8UbOWNLA~DoFeN8z96WABkTey~8B~5dEyIW70222fipsSme-2REMyJ1H3ygXV6vEO5H8BI2GhYmCH443GLPr4f-Cj~8-8EYdklWj0SIJxggU9GcvJQm1riuMi3Ok-112ZJ1Frp0O74EBxcUpochB4pjnpH1htIGQAUDN9N~0AsAnTm~97YoOvrEoOtLwI7xr8IorLHR93vQv9F6cmQDuHA-lT683JmjWGgM8gbh0OWT9KkYVNy93OsCcBD1QHI-JsuUROrgZ8FQLIzVIouLyEpDR-1tr50TkMxZV2wFlpR4QmL2AQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The RBS maps for four species. (A) Helicobacter pylori (B) Clostridium acetobutylicum (C) Methanobrevibacter smithii (D) Prochlorococcus marinus." path-from-xml="dsn02705.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7969132/dsn02705.jpeg" path-from-xml="dsn02705.jpeg" target="_blank">View large</a><a section="7969132" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn02705.jpeg?Expires=2147483647&Signature=dnzoLMddvROCZvcBEB58r5cvLfKFj~wGAonD8XoT74ZHYzFzqFJSDLCC20pGEjwtRRkC6b-ovRNLJtAK8tMhd18nKXqWpQKaSDR9vshpKpJ0pPQ3N4eyXUY0lAL8c~Ifsw3Z2zXUkoVDD4aHB~HuhfEuZfF3lj7WM2P3wdjU~jgwbzJCPvGWsD7E99eDMNGsgnOwG2DK1gU244YYeXhsNA43ykUFMW1Te-fJleAGF14AaTl0nT31C~kfF9goeV~dMJ4b27AMV4dNt~2f~cSJbcrRys7tvbArXYD2xD4qaf9ZQY5kiT~x0FKk0kHlaBRkEuv~eLypzz64NyerALs4sw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7969132&ar=512877&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02705.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The RBS maps for four species. (<strong>A</strong>) <em>Helicobacter pylori</em> (<strong>B</strong>) <em>Clostridium acetobutylicum</em> (<strong>C</strong>) <em>Methanobrevibacter smithii</em> (<strong>D</strong>) <em>Prochlorococcus marinus</em>.</p></div></div>                            <h3 scrollto-destination=7969133 id="7969133" class="section-title" data-legacy-id=s3d>3.4. Prediction performances on long genomic sequences</h3>
<p>The prediction performances of the MGA and conventional gene-finding tools based on unsupervised learning, such as GeneMarkS<sup><a reveal-id="DSN027C16" data-open="DSN027C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and Glimmer3<sup><a reveal-id="DSN027C17" data-open="DSN027C17" class="link link-ref link-reveal xref-bibr">17</a></sup>, were evaluated on various datasets. Fig. <a reveal-id="DSN027F6" data-open="DSN027F6" class="link link-reveal link-table xref-fig">6</a>A shows the prediction accuracies on the MetaGene dataset, which consists of nine bacterial and three archaeal genomic sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>) and their subsequences having 1× genome coverage (i.e. the total length of the subsequences is equal to the complete genome size).</p>                        <a id="7969135" scrollto-destination="7969135"></a>
<div id="DSN027F6" data-id="DSN027F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/m_dsn02706.jpeg?Expires=2147483647&Signature=geFD5SP8iMN22luR7085nOfFEM06tItXg80NpTutk9l56ZZjtVpqccrO52T4tTeM6clVy0YmJ13ZAnTKjJvVzq1Hf3~v0CblPgTNXdDEPfqFzgLSUzGFhNckTcD8xWPoZ3BBtmt7YIVheaeRkXtzSGcJDaIV8t1VtH1dDfW4dOnnB0wAKmCWwd1XD7NLyUF912lUOVUoioCzx2x721TH7V2RZJBB0JUoxjpQK89RFPUj-OKsqwAo5ZlMZD3jf7~7pyE5a~5yz-Xvty9inxHBAUulxQwTOEEcpQzb1OreuoFD~EVpqRbDmyoyC6bkDcjCr5mm9YaZGMG2~OAbTaJNzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Prediction performances of gene finders on the MetaGene dataset. (A) Accuracy comparisons of the MGA, GeneMarkS and Glimmer3. In the Glimmer3 prediction, a script ‘g3-iterated.csh’ is used. (B) Accuracy comparisons of the MGA and MG. In the MGA prediction, two different running options, which treat multiple input sequences individually (MGA) or as a unit (MGA-s), are used. (C) Relationship between accuracies and number of 40 kb-sequences in the MGA-s prediction. Sn, exact and Sp indicate sensitivity, sensitivity to start codons and specificity, respectively." path-from-xml="dsn02706.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7969135/dsn02706.jpeg" path-from-xml="dsn02706.jpeg" target="_blank">View large</a><a section="7969135" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn02706.jpeg?Expires=2147483647&Signature=f0yQxsxbYjeYYczLuPJjbYpvxKzoEuNf9~fkZMT-Dh5Jilfia0txQ7InrFbsBpELWJunHDjZNTQqQME6SeMRtTSrgc--y0CIEUv76GYEENgPEGm6hJHLnhZ0UlfWSEpav33ALlLhHzFjK-lDT2-0sdAhVO2cGXV2sP50wr3-0s1KEGmPRGtYNPaPtiNVaq60~hyXMhB6ZfP65h15F2FqA4XrV4himaWkKURbD1MdQ7kD7OfibXRHESOxtaOo1z4RNql32XqYVwbXTJGNFdMBF1vAEGK5zhu21woHpthWZRGgwmbEE4yTxMh07MbDHanMOGPsT4l5DfFCkYpQW2~yvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7969135&ar=512877&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02706.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Prediction performances of gene finders on the MetaGene dataset. (<strong>A</strong>) Accuracy comparisons of the MGA, GeneMarkS and Glimmer3. In the Glimmer3 prediction, a script ‘g3-iterated.csh’ is used. (<strong>B</strong>) Accuracy comparisons of the MGA and MG. In the MGA prediction, two different running options, which treat multiple input sequences individually (MGA) or as a unit (MGA-s), are used. (<strong>C</strong>) Relationship between accuracies and number of 40 kb-sequences in the MGA-s prediction. Sn, exact and Sp indicate sensitivity, sensitivity to start codons and specificity, respectively.</p></div></div><div content-id="DSN027F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/m_dsn02706.jpeg?Expires=2147483647&Signature=geFD5SP8iMN22luR7085nOfFEM06tItXg80NpTutk9l56ZZjtVpqccrO52T4tTeM6clVy0YmJ13ZAnTKjJvVzq1Hf3~v0CblPgTNXdDEPfqFzgLSUzGFhNckTcD8xWPoZ3BBtmt7YIVheaeRkXtzSGcJDaIV8t1VtH1dDfW4dOnnB0wAKmCWwd1XD7NLyUF912lUOVUoioCzx2x721TH7V2RZJBB0JUoxjpQK89RFPUj-OKsqwAo5ZlMZD3jf7~7pyE5a~5yz-Xvty9inxHBAUulxQwTOEEcpQzb1OreuoFD~EVpqRbDmyoyC6bkDcjCr5mm9YaZGMG2~OAbTaJNzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Prediction performances of gene finders on the MetaGene dataset. (A) Accuracy comparisons of the MGA, GeneMarkS and Glimmer3. In the Glimmer3 prediction, a script ‘g3-iterated.csh’ is used. (B) Accuracy comparisons of the MGA and MG. In the MGA prediction, two different running options, which treat multiple input sequences individually (MGA) or as a unit (MGA-s), are used. (C) Relationship between accuracies and number of 40 kb-sequences in the MGA-s prediction. Sn, exact and Sp indicate sensitivity, sensitivity to start codons and specificity, respectively." path-from-xml="dsn02706.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7969135/dsn02706.jpeg" path-from-xml="dsn02706.jpeg" target="_blank">View large</a><a section="7969135" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn02706.jpeg?Expires=2147483647&Signature=f0yQxsxbYjeYYczLuPJjbYpvxKzoEuNf9~fkZMT-Dh5Jilfia0txQ7InrFbsBpELWJunHDjZNTQqQME6SeMRtTSrgc--y0CIEUv76GYEENgPEGm6hJHLnhZ0UlfWSEpav33ALlLhHzFjK-lDT2-0sdAhVO2cGXV2sP50wr3-0s1KEGmPRGtYNPaPtiNVaq60~hyXMhB6ZfP65h15F2FqA4XrV4himaWkKURbD1MdQ7kD7OfibXRHESOxtaOo1z4RNql32XqYVwbXTJGNFdMBF1vAEGK5zhu21woHpthWZRGgwmbEE4yTxMh07MbDHanMOGPsT4l5DfFCkYpQW2~yvg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7969135&ar=512877&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02706.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Prediction performances of gene finders on the MetaGene dataset. (<strong>A</strong>) Accuracy comparisons of the MGA, GeneMarkS and Glimmer3. In the Glimmer3 prediction, a script ‘g3-iterated.csh’ is used. (<strong>B</strong>) Accuracy comparisons of the MGA and MG. In the MGA prediction, two different running options, which treat multiple input sequences individually (MGA) or as a unit (MGA-s), are used. (<strong>C</strong>) Relationship between accuracies and number of 40 kb-sequences in the MGA-s prediction. Sn, exact and Sp indicate sensitivity, sensitivity to start codons and specificity, respectively.</p></div></div><p>For complete genomes and 1 Mb subsequences, all prediction tools indicate almost identical sensitivities (∼97%), while specificity is significantly higher in the MGA (93%) compared with the others (90% in GeneMarkS and 86–87% in Glimmer3). In other words, the sensitivities of the MGA are potentially higher than the others at the same specificity level. Sensitivities to start codons are also identical in the MGA (78%) and GeneMarkS (77%), but Glimmer3 shows lower values (72–75%), although both GeneMarkS and Glimmer3 utilize the Gibbs sampling procedure to train their RBS models. In contrast, the mean sensitivity to start codons in Glimmer3 is better than that in GeneMarkS on the other dataset (Table <a reveal-id="DSN027TB1" data-open="DSN027TB1" class="link link-reveal link-table xref-fig">1</a>), which consists of six complete genomes (one archaea and five bacteria) having relatively broad distributions of the RBS maps. The performance of the MGA is stable and exceeds the others also in this dataset, especially in <em>Clostridium acetobutylicum</em>. In comparison with the original MG (Fig. <a reveal-id="DSN027F6" data-open="DSN027F6" class="link link-reveal link-table xref-fig">6</a>B), the MGA remarkably improves sensitivities to both genes and start codons without reducing specificities. These results indicate that our simple RBS model works well for detecting various types of the RBS.</p>                        <a id="7969137" scrollto-destination="7969137"></a>
<div content-id="DSN027TB1" class="table-modal"><div class="table-wrap"><div id="DSN027TB1" class="table-wrap-title" data-id="DSN027TB1"><div class="label -label">Table 1</div><div class="caption"><p>Prediction performances on the complete genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Species </th><th rowspan="2">GC% </th><th rowspan="2">RBS% </th><th colspan="2">MGA<hr /> </th><th colspan="2">GeneMarkS<hr /> </th><th colspan="2">Glimmer3<hr /> </th></tr><tr><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th></tr></thead><tbody><tr><td><em>S. marinus</em> </td><td>35.7 </td><td>85.4 </td><td>99.4 (87.8) </td><td>94.5 </td><td>99.6 (87.2) </td><td>92.5 </td><td>99.8 (87.6) </td><td>90.8 </td></tr><tr><td><em>C. acetobutylicum</em> </td><td>30.9 </td><td>93.7 </td><td>98.3 (92.1) </td><td>96.1 </td><td>98.5 (74.1) </td><td>92.8 </td><td>98.0 (90.9) </td><td>94.5 </td></tr><tr><td><em>F. nodosum</em> </td><td>35.0 </td><td>90.2 </td><td>99.6 (91.2) </td><td>94.8 </td><td>99.8 (90.6) </td><td>92.8 </td><td>99.7 (91.1) </td><td>94.0 </td></tr><tr><td><em>L. lactis</em> </td><td>35.3 </td><td>81.1 </td><td>98.5 (88.0) </td><td>95.1 </td><td>98.9 (88.4) </td><td>92.7 </td><td>98.2 (86.2) </td><td>93.2 </td></tr><tr><td><em>D. radiodurans</em> </td><td>67.0 </td><td>47.9 </td><td>97.8 (63.5) </td><td>93.6 </td><td>96.3 (56.7) </td><td>93.1 </td><td>96.5 (58.3) </td><td>92.1 </td></tr><tr><td><em>A. caulinodans</em> </td><td>67.3 </td><td>64.8 </td><td>99.2 (66.2) </td><td>95.4 </td><td>98.8 (61.5) </td><td>95.8 </td><td>98.6 (63.6) </td><td>93.6 </td></tr><tr><td>Average </td><td> </td><td> </td><td>98.7 (80.2) </td><td>95.0 </td><td>98.5 (74.3) </td><td>93.4 </td><td>98.2 (78.0) </td><td>93.5 </td></tr></tbody></table></div><div content-id="DSN027TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Species </th><th rowspan="2">GC% </th><th rowspan="2">RBS% </th><th colspan="2">MGA<hr /> </th><th colspan="2">GeneMarkS<hr /> </th><th colspan="2">Glimmer3<hr /> </th></tr><tr><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th></tr></thead><tbody><tr><td><em>S. marinus</em> </td><td>35.7 </td><td>85.4 </td><td>99.4 (87.8) </td><td>94.5 </td><td>99.6 (87.2) </td><td>92.5 </td><td>99.8 (87.6) </td><td>90.8 </td></tr><tr><td><em>C. acetobutylicum</em> </td><td>30.9 </td><td>93.7 </td><td>98.3 (92.1) </td><td>96.1 </td><td>98.5 (74.1) </td><td>92.8 </td><td>98.0 (90.9) </td><td>94.5 </td></tr><tr><td><em>F. nodosum</em> </td><td>35.0 </td><td>90.2 </td><td>99.6 (91.2) </td><td>94.8 </td><td>99.8 (90.6) </td><td>92.8 </td><td>99.7 (91.1) </td><td>94.0 </td></tr><tr><td><em>L. lactis</em> </td><td>35.3 </td><td>81.1 </td><td>98.5 (88.0) </td><td>95.1 </td><td>98.9 (88.4) </td><td>92.7 </td><td>98.2 (86.2) </td><td>93.2 </td></tr><tr><td><em>D. radiodurans</em> </td><td>67.0 </td><td>47.9 </td><td>97.8 (63.5) </td><td>93.6 </td><td>96.3 (56.7) </td><td>93.1 </td><td>96.5 (58.3) </td><td>92.1 </td></tr><tr><td><em>A. caulinodans</em> </td><td>67.3 </td><td>64.8 </td><td>99.2 (66.2) </td><td>95.4 </td><td>98.8 (61.5) </td><td>95.8 </td><td>98.6 (63.6) </td><td>93.6 </td></tr><tr><td>Average </td><td> </td><td> </td><td>98.7 (80.2) </td><td>95.0 </td><td>98.5 (74.3) </td><td>93.4 </td><td>98.2 (78.0) </td><td>93.5 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>RBS%, the RBS ratio (the proportion of genes having representative RBSs); Sn, sensitivity to genes; (exact), sensitivity to start codons; Sp, specificity.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7969137" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN027TB1" class="table-wrap-title" data-id="DSN027TB1"><div class="label -label">Table 1</div><div class="caption"><p>Prediction performances on the complete genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Species </th><th rowspan="2">GC% </th><th rowspan="2">RBS% </th><th colspan="2">MGA<hr /> </th><th colspan="2">GeneMarkS<hr /> </th><th colspan="2">Glimmer3<hr /> </th></tr><tr><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th></tr></thead><tbody><tr><td><em>S. marinus</em> </td><td>35.7 </td><td>85.4 </td><td>99.4 (87.8) </td><td>94.5 </td><td>99.6 (87.2) </td><td>92.5 </td><td>99.8 (87.6) </td><td>90.8 </td></tr><tr><td><em>C. acetobutylicum</em> </td><td>30.9 </td><td>93.7 </td><td>98.3 (92.1) </td><td>96.1 </td><td>98.5 (74.1) </td><td>92.8 </td><td>98.0 (90.9) </td><td>94.5 </td></tr><tr><td><em>F. nodosum</em> </td><td>35.0 </td><td>90.2 </td><td>99.6 (91.2) </td><td>94.8 </td><td>99.8 (90.6) </td><td>92.8 </td><td>99.7 (91.1) </td><td>94.0 </td></tr><tr><td><em>L. lactis</em> </td><td>35.3 </td><td>81.1 </td><td>98.5 (88.0) </td><td>95.1 </td><td>98.9 (88.4) </td><td>92.7 </td><td>98.2 (86.2) </td><td>93.2 </td></tr><tr><td><em>D. radiodurans</em> </td><td>67.0 </td><td>47.9 </td><td>97.8 (63.5) </td><td>93.6 </td><td>96.3 (56.7) </td><td>93.1 </td><td>96.5 (58.3) </td><td>92.1 </td></tr><tr><td><em>A. caulinodans</em> </td><td>67.3 </td><td>64.8 </td><td>99.2 (66.2) </td><td>95.4 </td><td>98.8 (61.5) </td><td>95.8 </td><td>98.6 (63.6) </td><td>93.6 </td></tr><tr><td>Average </td><td> </td><td> </td><td>98.7 (80.2) </td><td>95.0 </td><td>98.5 (74.3) </td><td>93.4 </td><td>98.2 (78.0) </td><td>93.5 </td></tr></tbody></table></div><div content-id="DSN027TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Species </th><th rowspan="2">GC% </th><th rowspan="2">RBS% </th><th colspan="2">MGA<hr /> </th><th colspan="2">GeneMarkS<hr /> </th><th colspan="2">Glimmer3<hr /> </th></tr><tr><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th></tr></thead><tbody><tr><td><em>S. marinus</em> </td><td>35.7 </td><td>85.4 </td><td>99.4 (87.8) </td><td>94.5 </td><td>99.6 (87.2) </td><td>92.5 </td><td>99.8 (87.6) </td><td>90.8 </td></tr><tr><td><em>C. acetobutylicum</em> </td><td>30.9 </td><td>93.7 </td><td>98.3 (92.1) </td><td>96.1 </td><td>98.5 (74.1) </td><td>92.8 </td><td>98.0 (90.9) </td><td>94.5 </td></tr><tr><td><em>F. nodosum</em> </td><td>35.0 </td><td>90.2 </td><td>99.6 (91.2) </td><td>94.8 </td><td>99.8 (90.6) </td><td>92.8 </td><td>99.7 (91.1) </td><td>94.0 </td></tr><tr><td><em>L. lactis</em> </td><td>35.3 </td><td>81.1 </td><td>98.5 (88.0) </td><td>95.1 </td><td>98.9 (88.4) </td><td>92.7 </td><td>98.2 (86.2) </td><td>93.2 </td></tr><tr><td><em>D. radiodurans</em> </td><td>67.0 </td><td>47.9 </td><td>97.8 (63.5) </td><td>93.6 </td><td>96.3 (56.7) </td><td>93.1 </td><td>96.5 (58.3) </td><td>92.1 </td></tr><tr><td><em>A. caulinodans</em> </td><td>67.3 </td><td>64.8 </td><td>99.2 (66.2) </td><td>95.4 </td><td>98.8 (61.5) </td><td>95.8 </td><td>98.6 (63.6) </td><td>93.6 </td></tr><tr><td>Average </td><td> </td><td> </td><td>98.7 (80.2) </td><td>95.0 </td><td>98.5 (74.3) </td><td>93.4 </td><td>98.2 (78.0) </td><td>93.5 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>RBS%, the RBS ratio (the proportion of genes having representative RBSs); Sn, sensitivity to genes; (exact), sensitivity to start codons; Sp, specificity.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7969137" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7969138 id="7969138" class="section-title" data-legacy-id=s3e>3.5. Prediction performances on short genomic sequences</h3>
<p>For sequences shorter than 1 Mb, the MGA retained high accuracies in every index (Fig. <a reveal-id="DSN027F6" data-open="DSN027F6" class="link link-reveal link-table xref-fig">6</a>A). Both sensitivities and specificities of Glimmer3 are relatively high when input sequences are longer than 40 kb, but the performance of the start codon prediction is rapidly degraded as the input sequences become shorter. This is because the Gibbs sampling algorithm requires a significant number of positive (RBS) sequences to detect the correct motif. A 40 kb-sequence has &lt;40 genes (or RBSs) on average, and the sensitivity to start codons declines to 57% in Glimmer3. GeneMarkS does not accept a shorter input sequence than 1 Mb, probably because it has the same weak point as Glimmer3. Unlike the RBS models of these tools, the MGA assumes only nine hexamers as candidate's RBSs, and relatively few sequences are needed to estimate the parameters of the RBS model. As a result, the MGA requires just 500 kb (or ∼500 genes) to adapt the RBS model fully to the input sequence, and its sensitivity to start codons is sufficiently high (75%) even in 40 kb sequences. Furthermore, Fig. <a reveal-id="DSN027F6" data-open="DSN027F6" class="link link-reveal link-table xref-fig">6</a>B and Table <a reveal-id="DSN027TB2" data-open="DSN027TB2" class="link link-reveal link-table xref-fig">2</a> show that the general RBS model of the MGA also works well for predicting the start codons of genes on very short sequences. Although most genes on 700 bp-subsequences lack their 5′ sequences (including start codon and RBSs), the results also indicate that the RBS model contributes to improving prediction specificities by deselecting false-positive translation starts.</p>                        <a id="7969140" scrollto-destination="7969140"></a>
<div content-id="DSN027TB2" class="table-modal"><div class="table-wrap"><div id="DSN027TB2" class="table-wrap-title" data-id="DSN027TB2"><div class="label -label">Table 2</div><div class="caption"><p>Prediction performances on 700 bp subsequences (1× genome coverage)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Species </th><th rowspan="2">GC% </th><th rowspan="2">RBS% </th><th colspan="2">MGA-s<hr /> </th><th colspan="2">MGA<hr /> </th><th colspan="2">MG<hr /> </th></tr><tr><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th></tr></thead><tbody><tr><td><em>M. jannaschii</em> </td><td>31.4 </td><td>87.6 </td><td>98.3 (79.3) </td><td>95.8 </td><td>97.7 (80.3) </td><td>94.1 </td><td>97.8 (82.4) </td><td>94.1 </td></tr><tr><td><em>A. fulgidus</em> </td><td>48.6 </td><td>61.7 </td><td>96.7 (82.9) </td><td>94.1 </td><td>95.7 (81.7) </td><td>93.5 </td><td>95.8 (81.5) </td><td>93.7 </td></tr><tr><td><em>N. pharaonis</em> </td><td>63.4 </td><td>39.6 </td><td>97.4 (88.3) </td><td>97.1 </td><td>97.1 (87.1) </td><td>94.5 </td><td>97.1 (86.2) </td><td>93.0 </td></tr><tr><td><em>B. aphidicola</em> </td><td>26.3 </td><td>60.9 </td><td>98.4 (91.5) </td><td>93.6 </td><td>98.6 (91.7) </td><td>93.2 </td><td>98.2 (90.9) </td><td>92.7 </td></tr><tr><td><em>P. marinus</em> </td><td>31.2 </td><td>21.0 </td><td>95.2 (88.8) </td><td>93.0 </td><td>94.9 (87.4) </td><td>92.3 </td><td>95.5 (87.6) </td><td>92.7 </td></tr><tr><td><em>W. endosymbiont</em> </td><td>34.2 </td><td>40.1 </td><td>93.8 (85.8) </td><td>74.5 </td><td>93.6 (82.9) </td><td>72.7 </td><td>93.1 (80.8) </td><td>76.0 </td></tr><tr><td><em>H. pylori</em> </td><td>39.2 </td><td>78.3 </td><td>96.8 (88.1) </td><td>95.1 </td><td>93.5 (82.9) </td><td>92.4 </td><td>92.6 (77.7) </td><td>92.7 </td></tr><tr><td><em>B. subtillis</em> </td><td>43.5 </td><td>92.6 </td><td>97.3 (88.8) </td><td>94.5 </td><td>93.9 (82.2) </td><td>92.9 </td><td>92.3 (73.5) </td><td>92.5 </td></tr><tr><td><em>E. coli</em> </td><td>50.8 </td><td>77.6 </td><td>96.4 (83.5) </td><td>94.6 </td><td>95.0 (82.9) </td><td>94.0 </td><td>95.3 (81.2) </td><td>93.2 </td></tr><tr><td><em>C. tepidum</em> </td><td>56.5 </td><td>45.4 </td><td>88.8 (75.3) </td><td>93.5 </td><td>87.6 (73.7) </td><td>90.6 </td><td>88.1 (73.2) </td><td>89.6 </td></tr><tr><td><em>C. jeikeium</em> </td><td>61.4 </td><td>72.8 </td><td>95.7 (83.9) </td><td>95.1 </td><td>94.9 (82.8) </td><td>93.3 </td><td>94.0 (78.5) </td><td>91.4 </td></tr><tr><td><em>B. pseudomallei 1</em> </td><td>67.7 </td><td>56.2 </td><td>96.6 (83.1) </td><td>93.7 </td><td>96.9 (82.9) </td><td>90.6 </td><td>96.8 (81.2) </td><td>87.9 </td></tr><tr><td><em>B. pseudomallei 2</em> </td><td>68.5 </td><td>56.3 </td><td>96.2 (83.9) </td><td>91.6 </td><td>96.4 (82.8) </td><td>89.0 </td><td>96.6 (81.2) </td><td>85.7 </td></tr><tr><td> </td><td> </td><td> </td><td>96.0 (84.9) </td><td>92.8 </td><td>95.1 (83.2) </td><td>91.0 </td><td>94.9 (81.2) </td><td>90.4 </td></tr></tbody></table></div><div content-id="DSN027TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Species </th><th rowspan="2">GC% </th><th rowspan="2">RBS% </th><th colspan="2">MGA-s<hr /> </th><th colspan="2">MGA<hr /> </th><th colspan="2">MG<hr /> </th></tr><tr><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th></tr></thead><tbody><tr><td><em>M. jannaschii</em> </td><td>31.4 </td><td>87.6 </td><td>98.3 (79.3) </td><td>95.8 </td><td>97.7 (80.3) </td><td>94.1 </td><td>97.8 (82.4) </td><td>94.1 </td></tr><tr><td><em>A. fulgidus</em> </td><td>48.6 </td><td>61.7 </td><td>96.7 (82.9) </td><td>94.1 </td><td>95.7 (81.7) </td><td>93.5 </td><td>95.8 (81.5) </td><td>93.7 </td></tr><tr><td><em>N. pharaonis</em> </td><td>63.4 </td><td>39.6 </td><td>97.4 (88.3) </td><td>97.1 </td><td>97.1 (87.1) </td><td>94.5 </td><td>97.1 (86.2) </td><td>93.0 </td></tr><tr><td><em>B. aphidicola</em> </td><td>26.3 </td><td>60.9 </td><td>98.4 (91.5) </td><td>93.6 </td><td>98.6 (91.7) </td><td>93.2 </td><td>98.2 (90.9) </td><td>92.7 </td></tr><tr><td><em>P. marinus</em> </td><td>31.2 </td><td>21.0 </td><td>95.2 (88.8) </td><td>93.0 </td><td>94.9 (87.4) </td><td>92.3 </td><td>95.5 (87.6) </td><td>92.7 </td></tr><tr><td><em>W. endosymbiont</em> </td><td>34.2 </td><td>40.1 </td><td>93.8 (85.8) </td><td>74.5 </td><td>93.6 (82.9) </td><td>72.7 </td><td>93.1 (80.8) </td><td>76.0 </td></tr><tr><td><em>H. pylori</em> </td><td>39.2 </td><td>78.3 </td><td>96.8 (88.1) </td><td>95.1 </td><td>93.5 (82.9) </td><td>92.4 </td><td>92.6 (77.7) </td><td>92.7 </td></tr><tr><td><em>B. subtillis</em> </td><td>43.5 </td><td>92.6 </td><td>97.3 (88.8) </td><td>94.5 </td><td>93.9 (82.2) </td><td>92.9 </td><td>92.3 (73.5) </td><td>92.5 </td></tr><tr><td><em>E. coli</em> </td><td>50.8 </td><td>77.6 </td><td>96.4 (83.5) </td><td>94.6 </td><td>95.0 (82.9) </td><td>94.0 </td><td>95.3 (81.2) </td><td>93.2 </td></tr><tr><td><em>C. tepidum</em> </td><td>56.5 </td><td>45.4 </td><td>88.8 (75.3) </td><td>93.5 </td><td>87.6 (73.7) </td><td>90.6 </td><td>88.1 (73.2) </td><td>89.6 </td></tr><tr><td><em>C. jeikeium</em> </td><td>61.4 </td><td>72.8 </td><td>95.7 (83.9) </td><td>95.1 </td><td>94.9 (82.8) </td><td>93.3 </td><td>94.0 (78.5) </td><td>91.4 </td></tr><tr><td><em>B. pseudomallei 1</em> </td><td>67.7 </td><td>56.2 </td><td>96.6 (83.1) </td><td>93.7 </td><td>96.9 (82.9) </td><td>90.6 </td><td>96.8 (81.2) </td><td>87.9 </td></tr><tr><td><em>B. pseudomallei 2</em> </td><td>68.5 </td><td>56.3 </td><td>96.2 (83.9) </td><td>91.6 </td><td>96.4 (82.8) </td><td>89.0 </td><td>96.6 (81.2) </td><td>85.7 </td></tr><tr><td> </td><td> </td><td> </td><td>96.0 (84.9) </td><td>92.8 </td><td>95.1 (83.2) </td><td>91.0 </td><td>94.9 (81.2) </td><td>90.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>RBS%, the RBS ratio (the proportion of genes having representative RBSs); Sn, sensitivity to genes; (exact), sensitivity to start codons; Sp, specificity; MGA-s, MGA with –s option in which multiple sequences are treated as a unit.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7969140" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN027TB2" class="table-wrap-title" data-id="DSN027TB2"><div class="label -label">Table 2</div><div class="caption"><p>Prediction performances on 700 bp subsequences (1× genome coverage)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Species </th><th rowspan="2">GC% </th><th rowspan="2">RBS% </th><th colspan="2">MGA-s<hr /> </th><th colspan="2">MGA<hr /> </th><th colspan="2">MG<hr /> </th></tr><tr><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th></tr></thead><tbody><tr><td><em>M. jannaschii</em> </td><td>31.4 </td><td>87.6 </td><td>98.3 (79.3) </td><td>95.8 </td><td>97.7 (80.3) </td><td>94.1 </td><td>97.8 (82.4) </td><td>94.1 </td></tr><tr><td><em>A. fulgidus</em> </td><td>48.6 </td><td>61.7 </td><td>96.7 (82.9) </td><td>94.1 </td><td>95.7 (81.7) </td><td>93.5 </td><td>95.8 (81.5) </td><td>93.7 </td></tr><tr><td><em>N. pharaonis</em> </td><td>63.4 </td><td>39.6 </td><td>97.4 (88.3) </td><td>97.1 </td><td>97.1 (87.1) </td><td>94.5 </td><td>97.1 (86.2) </td><td>93.0 </td></tr><tr><td><em>B. aphidicola</em> </td><td>26.3 </td><td>60.9 </td><td>98.4 (91.5) </td><td>93.6 </td><td>98.6 (91.7) </td><td>93.2 </td><td>98.2 (90.9) </td><td>92.7 </td></tr><tr><td><em>P. marinus</em> </td><td>31.2 </td><td>21.0 </td><td>95.2 (88.8) </td><td>93.0 </td><td>94.9 (87.4) </td><td>92.3 </td><td>95.5 (87.6) </td><td>92.7 </td></tr><tr><td><em>W. endosymbiont</em> </td><td>34.2 </td><td>40.1 </td><td>93.8 (85.8) </td><td>74.5 </td><td>93.6 (82.9) </td><td>72.7 </td><td>93.1 (80.8) </td><td>76.0 </td></tr><tr><td><em>H. pylori</em> </td><td>39.2 </td><td>78.3 </td><td>96.8 (88.1) </td><td>95.1 </td><td>93.5 (82.9) </td><td>92.4 </td><td>92.6 (77.7) </td><td>92.7 </td></tr><tr><td><em>B. subtillis</em> </td><td>43.5 </td><td>92.6 </td><td>97.3 (88.8) </td><td>94.5 </td><td>93.9 (82.2) </td><td>92.9 </td><td>92.3 (73.5) </td><td>92.5 </td></tr><tr><td><em>E. coli</em> </td><td>50.8 </td><td>77.6 </td><td>96.4 (83.5) </td><td>94.6 </td><td>95.0 (82.9) </td><td>94.0 </td><td>95.3 (81.2) </td><td>93.2 </td></tr><tr><td><em>C. tepidum</em> </td><td>56.5 </td><td>45.4 </td><td>88.8 (75.3) </td><td>93.5 </td><td>87.6 (73.7) </td><td>90.6 </td><td>88.1 (73.2) </td><td>89.6 </td></tr><tr><td><em>C. jeikeium</em> </td><td>61.4 </td><td>72.8 </td><td>95.7 (83.9) </td><td>95.1 </td><td>94.9 (82.8) </td><td>93.3 </td><td>94.0 (78.5) </td><td>91.4 </td></tr><tr><td><em>B. pseudomallei 1</em> </td><td>67.7 </td><td>56.2 </td><td>96.6 (83.1) </td><td>93.7 </td><td>96.9 (82.9) </td><td>90.6 </td><td>96.8 (81.2) </td><td>87.9 </td></tr><tr><td><em>B. pseudomallei 2</em> </td><td>68.5 </td><td>56.3 </td><td>96.2 (83.9) </td><td>91.6 </td><td>96.4 (82.8) </td><td>89.0 </td><td>96.6 (81.2) </td><td>85.7 </td></tr><tr><td> </td><td> </td><td> </td><td>96.0 (84.9) </td><td>92.8 </td><td>95.1 (83.2) </td><td>91.0 </td><td>94.9 (81.2) </td><td>90.4 </td></tr></tbody></table></div><div content-id="DSN027TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Species </th><th rowspan="2">GC% </th><th rowspan="2">RBS% </th><th colspan="2">MGA-s<hr /> </th><th colspan="2">MGA<hr /> </th><th colspan="2">MG<hr /> </th></tr><tr><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th><th>Sn (exact) (%) </th><th>Sp (%) </th></tr></thead><tbody><tr><td><em>M. jannaschii</em> </td><td>31.4 </td><td>87.6 </td><td>98.3 (79.3) </td><td>95.8 </td><td>97.7 (80.3) </td><td>94.1 </td><td>97.8 (82.4) </td><td>94.1 </td></tr><tr><td><em>A. fulgidus</em> </td><td>48.6 </td><td>61.7 </td><td>96.7 (82.9) </td><td>94.1 </td><td>95.7 (81.7) </td><td>93.5 </td><td>95.8 (81.5) </td><td>93.7 </td></tr><tr><td><em>N. pharaonis</em> </td><td>63.4 </td><td>39.6 </td><td>97.4 (88.3) </td><td>97.1 </td><td>97.1 (87.1) </td><td>94.5 </td><td>97.1 (86.2) </td><td>93.0 </td></tr><tr><td><em>B. aphidicola</em> </td><td>26.3 </td><td>60.9 </td><td>98.4 (91.5) </td><td>93.6 </td><td>98.6 (91.7) </td><td>93.2 </td><td>98.2 (90.9) </td><td>92.7 </td></tr><tr><td><em>P. marinus</em> </td><td>31.2 </td><td>21.0 </td><td>95.2 (88.8) </td><td>93.0 </td><td>94.9 (87.4) </td><td>92.3 </td><td>95.5 (87.6) </td><td>92.7 </td></tr><tr><td><em>W. endosymbiont</em> </td><td>34.2 </td><td>40.1 </td><td>93.8 (85.8) </td><td>74.5 </td><td>93.6 (82.9) </td><td>72.7 </td><td>93.1 (80.8) </td><td>76.0 </td></tr><tr><td><em>H. pylori</em> </td><td>39.2 </td><td>78.3 </td><td>96.8 (88.1) </td><td>95.1 </td><td>93.5 (82.9) </td><td>92.4 </td><td>92.6 (77.7) </td><td>92.7 </td></tr><tr><td><em>B. subtillis</em> </td><td>43.5 </td><td>92.6 </td><td>97.3 (88.8) </td><td>94.5 </td><td>93.9 (82.2) </td><td>92.9 </td><td>92.3 (73.5) </td><td>92.5 </td></tr><tr><td><em>E. coli</em> </td><td>50.8 </td><td>77.6 </td><td>96.4 (83.5) </td><td>94.6 </td><td>95.0 (82.9) </td><td>94.0 </td><td>95.3 (81.2) </td><td>93.2 </td></tr><tr><td><em>C. tepidum</em> </td><td>56.5 </td><td>45.4 </td><td>88.8 (75.3) </td><td>93.5 </td><td>87.6 (73.7) </td><td>90.6 </td><td>88.1 (73.2) </td><td>89.6 </td></tr><tr><td><em>C. jeikeium</em> </td><td>61.4 </td><td>72.8 </td><td>95.7 (83.9) </td><td>95.1 </td><td>94.9 (82.8) </td><td>93.3 </td><td>94.0 (78.5) </td><td>91.4 </td></tr><tr><td><em>B. pseudomallei 1</em> </td><td>67.7 </td><td>56.2 </td><td>96.6 (83.1) </td><td>93.7 </td><td>96.9 (82.9) </td><td>90.6 </td><td>96.8 (81.2) </td><td>87.9 </td></tr><tr><td><em>B. pseudomallei 2</em> </td><td>68.5 </td><td>56.3 </td><td>96.2 (83.9) </td><td>91.6 </td><td>96.4 (82.8) </td><td>89.0 </td><td>96.6 (81.2) </td><td>85.7 </td></tr><tr><td> </td><td> </td><td> </td><td>96.0 (84.9) </td><td>92.8 </td><td>95.1 (83.2) </td><td>91.0 </td><td>94.9 (81.2) </td><td>90.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>RBS%, the RBS ratio (the proportion of genes having representative RBSs); Sn, sensitivity to genes; (exact), sensitivity to start codons; Sp, specificity; MGA-s, MGA with –s option in which multiple sequences are treated as a unit.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7969140" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7969141 id="7969141" class="section-title" data-legacy-id=s3f>3.6. Advantage of self-training using a set of genomic sequences</h3>
<p>If multiple (short) input sequences can be assumed as the genomic sequences of the same species, prediction accuracies on the sequences are improved by self-training of the models as well as on a long-genomic sequence (the MGA-s in Fig. <a reveal-id="DSN027F6" data-open="DSN027F6" class="link link-reveal link-table xref-fig">6</a>B and C). Fig. <a reveal-id="DSN027F6" data-open="DSN027F6" class="link link-reveal link-table xref-fig">6</a>C shows the relationships between prediction accuracies and the number of 40 kb-sequences treated as a unit. Fig. <a reveal-id="DSN027F6" data-open="DSN027F6" class="link link-reveal link-table xref-fig">6</a>C also suggests that a total of about 500 kb (10–20×40 kb) are needed for full adaptation of the RBS model, but the prediction accuracies steadily improve if the number of input sequences are increased. When a sufficient amount of sequences are available, the MGA provides prediction performances comparable to the complete genome analyses, even if each sequence is very short (Table <a reveal-id="DSN027TB2" data-open="DSN027TB2" class="link link-reveal link-table xref-fig">2</a>). So, if multiple contig sequences are obtained by sequencing a single species’ genome, or if metagenomic sequences are phylogenetically classified into groups using some classification methods<sup><a reveal-id="DSN027C34" data-open="DSN027C34" class="link link-ref link-reveal xref-bibr">34</a>,<a reveal-id="DSN027C35" data-open="DSN027C35" class="link link-ref link-reveal xref-bibr">35</a></sup>, genes on the sequences can be more precisely predicted by the MGA.</p>                            <h3 scrollto-destination=7969143 id="7969143" class="section-title" data-legacy-id=s3g>3.7. Conclusion</h3>
<p>As mentioned, the MGA successfully overcomes the limitations of the MG, and archives high prediction accuracies especially in the start codon predictions. Although some gene-finding tools advocating high sensitivity to start codons, such as GeneMarkS and Glimmer3 tend to sacrifice specificities for improving sensitivities, the RBS model of the MGA enables the sensitive detection of start codons without reducing specificities. Our RBS model is based on previous knowledge about the RBS and 16S rRNA, and requires little training data for estimating its parameters. As a result, the MGA can precisely predict genes even on short genomic sequences unlike the other tools. Both typical and atypical genes can be sensitively and precisely detected while keeping high specificity. The MGA can detect not only chromosome backbone genes but also prophage genes and provides a complete set of genes on a genomic sequence. The MGA also provides information about the selected di-codon model (bacteria, archaea, prophage or self) for predicting each gene, and the information is helpful for further analyses of genes because it reflects statistical differences among the genes.</p><p>In addition to the precise prediction ability of the MGA, the RBS map analysis proposed here is helpful for genome annotations and is useful for analyzing translation initiation mechanisms and their evolutions. It is important for annotators to comprehend a specific RBS pattern of a target species and its related species. The MGA can automatically extract the pattern, and outputs information on RBSs in addition to location information on genes. We believe that the MGA accelerates not only metagenomic analyses but also the annotation processes of all kinds of prokaryotic and phage genomes.</p>                            <h2 scrollto-destination=7969146 id="7969146" class="section-title" data-legacy-id=s4>Availability</h2>
<p>MetaGeneAnnotator are freely available at <a class="link link-uri" href="http://metagene.cb.k.u-tokyo.ac.jp">http://metagene.cb.k.u-tokyo.ac.jp</a>.</p>                            <h2 scrollto-destination=7969148 id="7969148" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data is available online at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7969150 id="7969150" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The original MetaGene was developed at Toshihisa Takagi laboratory (University of Tokyo). We thank Prof Tetsuya Hayashi (Miyazaki University) and Prof Ken Kurokawa (Tokyo Institute of Technology) for stimulating discussions.</p>                            <h2 scrollto-destination=7969152 id="7969152" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSN027C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fickett</div> <div class="given-names">J. W.</div></div>. </div><div class="article-title">Recognition of protein coding regions in DNA sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1981</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">5303</div>-<div class="lpage">5318</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recognition%20of%20protein%20coding%20regions%20in%20DNA%20sequences&amp;author=J.%20W.%20Fickett&amp;publication_year=1981&amp;journal=Nucleic%20Acids%20Res.&amp;volume=10&amp;pages=5303-5318" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/10.17.5303" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F10.17.5303" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F10.17.5303"> </span></div> </div></div></div></div></div><div content-id="DSN027C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gribskov</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Devereux</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Burgess</div> <div class="given-names">R. R.</div></div>. </div><div class="article-title">The codon preference plot: graphic analysis of protein coding sequences and prediction of gene expression</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1984</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">539</div>-<div class="lpage">549</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20codon%20preference%20plot%3A%20graphic%20analysis%20of%20protein%20coding%20sequences%20and%20prediction%20of%20gene%20expression&amp;author=M.%20Gribskov&amp;author=J.%20Devereux&amp;author=R.%20R.%20Burgess&amp;publication_year=1984&amp;journal=Nucleic%20Acids%20Res.&amp;volume=12&amp;pages=539-549" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/12.1Part2.539" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F12.1Part2.539" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F12.1Part2.539"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6694906" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Staden</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Measurements of the effects of that coding for a protein has on a DNA sequence and their use for finding genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1984</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">551</div>-<div class="lpage">567</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Measurements%20of%20the%20effects%20of%20that%20coding%20for%20a%20protein%20has%20on%20a%20DNA%20sequence%20and%20their%20use%20for%20finding%20genes&amp;author=R.%20Staden&amp;publication_year=1984&amp;journal=Nucleic%20Acids%20Res.&amp;volume=12&amp;pages=551-567" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/12.1Part2.551" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F12.1Part2.551" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F12.1Part2.551"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6364041" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M. Y.</div></div>,  <div class="name"><div class="surname">Sprizhitskii</div> <div class="given-names">Y. A.</div></div>,  <div class="name"><div class="surname">Golovanov</div> <div class="given-names">E. I.</div></div>,  <div class="name"><div class="surname">Aleksandrov</div> <div class="given-names">A. A.</div></div>. </div><div class="article-title">Statistical patterns in primary structures of functional regions in the <em>E. coli</em> genome: III. Computer recognition of coding regions</div>, <div class="source">Mol. Biol.</div> , <div class="year">1986</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1145</div>-<div class="lpage">1150</div>)</div></div></div></div><div content-id="DSN027C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M. Y.</div></div>,  <div class="name"><div class="surname">McIninch</div> <div class="given-names">J. D.</div></div>. </div><div class="article-title">GeneMark: parallel gene recognition for both DNA strands</div>, <div class="source">Comput. Chem.</div> , <div class="year">1993</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">123</div>-<div class="lpage">153</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneMark%3A%20parallel%20gene%20recognition%20for%20both%20DNA%20strands&amp;author=M.%20Y.%20Borodovsky&amp;author=J.%20D.%20McIninch&amp;publication_year=1993&amp;journal=Comput.%20Chem.&amp;volume=17&amp;pages=123-153" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0097-8485(93)85004-V" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0097-8485(93)85004-V" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0097-8485(93)85004-V"> </span></div> </div></div></div></div></div><div content-id="DSN027C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krogh</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mian</div> <div class="given-names">I. S.</div></div>,  <div class="name"><div class="surname">Haussler</div> <div class="given-names">D.</div></div>. </div><div class="article-title">A hidden Markov model that finds genes in <em>E.coli</em> DNA</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1994</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">4768</div>-<div class="lpage">4778</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20hidden%20Markov%20model%20that%20finds%20genes%20in%20E.coli%20DNA&amp;author=A.%20Krogh&amp;author=I.%20S.%20Mian&amp;author=D.%20Haussler&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=22&amp;pages=4768-4778" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.22.4768" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.22.4768" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.22.4768"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7984429" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div>,  <div class="name"><div class="surname">Delcher</div> <div class="given-names">A. L.</div></div>,  <div class="name"><div class="surname">Kasif</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">O.</div></div>. </div><div class="article-title">Microbial gene identification using interpolated Markov model</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">544</div>-<div class="lpage">548</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microbial%20gene%20identification%20using%20interpolated%20Markov%20model&amp;author=S.%20L.%20Salzberg&amp;author=A.%20L.%20Delcher&amp;author=S.%20Kasif&amp;author=O.%20White&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res.&amp;volume=26&amp;pages=544-548" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.2.544" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.2.544" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.2.544"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9421513" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lukashin</div> <div class="given-names">A. V.</div></div>,  <div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>. </div><div class="article-title">GeneMark.hmm: new solutions for gene finding</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1107</div>-<div class="lpage">1115</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneMark.hmm%3A%20new%20solutions%20for%20gene%20finding&amp;author=A.%20V.%20Lukashin&amp;author=M.%20Borodovsky&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res.&amp;volume=26&amp;pages=1107-1115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.4.1107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.4.1107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.4.1107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9461475" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Delcher</div> <div class="given-names">A. L.</div></div>,  <div class="name"><div class="surname">Harmon</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Kasif</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div>. </div><div class="article-title">Improved microbial gene identification with GLIMMER</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">4636</div>-<div class="lpage">4641</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20microbial%20gene%20identification%20with%20GLIMMER&amp;author=A.%20L.%20Delcher&amp;author=D.%20Harmon&amp;author=S.%20Kasif&amp;author=O.%20White&amp;author=S.%20L.%20Salzberg&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=4636-4641" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.23.4636" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.23.4636" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.23.4636"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10556321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yada</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakao</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Totoki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Modeling and predicting transcriptional units of <em>Escherichia coli</em> genes using hidden Markov model</div>, <div class="source">Bioinformatics</div> , <div class="year">1999</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">987</div>-<div class="lpage">993</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modeling%20and%20predicting%20transcriptional%20units%20of%20Escherichia%20coli%20genes%20using%20hidden%20Markov%20model&amp;author=T.%20Yada&amp;author=M.%20Nakao&amp;author=Y.%20Totoki&amp;author=K.%20Nakai&amp;publication_year=1999&amp;journal=Bioinformatics&amp;volume=15&amp;pages=987-993" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/15.12.987" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F15.12.987" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F15.12.987"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10745988" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yada</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Totoki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Takagi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div>. </div><div class="article-title">A novel bacterial gene-finding system with improved accuracy in locating start codons</div>, <div class="source">DNA Res.</div> , <div class="year">2001</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">97</div>-<div class="lpage">106</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20bacterial%20gene-finding%20system%20with%20improved%20accuracy%20in%20locating%20start%20codons&amp;author=T.%20Yada&amp;author=Y.%20Totoki&amp;author=T.%20Takagi&amp;author=K.%20Nakai&amp;publication_year=2001&amp;journal=DNA%20Res.&amp;volume=8&amp;pages=97-106" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.3.97" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.3.97" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.3.97"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11475327" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayes</div> <div class="given-names">W. S.</div></div>,  <div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>. </div><div class="article-title">How to interpret an anonymous bacterial genome: machine learning approach to gene identification</div>, <div class="source">Genome Res.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">1154</div>-<div class="lpage">1171</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=How%20to%20interpret%20an%20anonymous%20bacterial%20genome%3A%20machine%20learning%20approach%20to%20gene%20identification&amp;author=W.%20S.%20Hayes&amp;author=M.%20Borodovsky&amp;publication_year=1998&amp;journal=Genome%20Res.&amp;volume=8&amp;pages=1154-1171" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9847079" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Audic</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Claverie</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title">Self-identification of protein-coding regions in microbial genomes</div>, <div class="source">Proc. Natl. Acad. Sci. U. S. A.</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">10026</div>-<div class="lpage">10031</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Self-identification%20of%20protein-coding%20regions%20in%20microbial%20genomes&amp;author=S.%20Audic&amp;author=J.%20M.%20Claverie&amp;publication_year=1998&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;volume=95&amp;pages=10026-10031" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.17.10026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.17.10026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.17.10026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9707594" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Baldi</div> <div class="given-names">P.</div></div>. </div><div class="article-title">On the convergence of a clustering algorithm for protein-coding regions in microbial genomes</div>, <div class="source">Bioinformatics</div> , <div class="year">2000</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">367</div>-<div class="lpage">371</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20convergence%20of%20a%20clustering%20algorithm%20for%20protein-coding%20regions%20in%20microbial%20genomes&amp;author=P.%20Baldi&amp;publication_year=2000&amp;journal=Bioinformatics&amp;volume=16&amp;pages=367-371" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/16.4.367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F16.4.367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F16.4.367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10869034" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Besemer</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lomsadze</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>. </div><div class="article-title">GeneMarkS: a self-training method for prediction of gene starts in microbial genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> (pg. <div class="fpage">2607</div>-<div class="lpage">2618</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneMarkS%3A%20a%20self-training%20method%20for%20prediction%20of%20gene%20starts%20in%20microbial%20genomes&amp;author=J.%20Besemer&amp;author=A.%20Lomsadze&amp;author=M.%20Borodovsky&amp;publication_year=2001&amp;journal=Nucleic%20Acids%20Res.&amp;volume=29&amp;pages=2607-2618" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/29.12.2607" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F29.12.2607" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F29.12.2607"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11410670" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Delcher</div> <div class="given-names">A. L.</div></div>,  <div class="name"><div class="surname">Bratke</div> <div class="given-names">K. A.</div></div>,  <div class="name"><div class="surname">Powers</div> <div class="given-names">E. C.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div>. </div><div class="article-title">Identifying bacterial genes and endosymbiont DNA with Glimmer</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">673</div>-<div class="lpage">679</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identifying%20bacterial%20genes%20and%20endosymbiont%20DNA%20with%20Glimmer&amp;author=A.%20L.%20Delcher&amp;author=K.%20A.%20Bratke&amp;author=E.%20C.%20Powers&amp;author=S.%20L.%20Salzberg&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=673-679" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17237039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schuster</div> <div class="given-names">S. C.</div></div>. </div><div class="article-title">Next-generation sequencing transforms today’s biology</div>, <div class="source">Nat. Methods</div> , <div class="year">2008</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">16</div>-<div class="lpage">18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20sequencing%20transforms%20today%E2%80%99s%20biology&amp;author=S.%20C.%20Schuster&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=16-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth1156" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth1156" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth1156"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18165802" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hall</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Advanced sequencing technologies and their wider impact in microbiology</div>, <div class="source">J. Exp. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">210</div> (pg. <div class="fpage">1518</div>-<div class="lpage">1525</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advanced%20sequencing%20technologies%20and%20their%20wider%20impact%20in%20microbiology&amp;author=N.%20Hall&amp;publication_year=2007&amp;journal=J.%20Exp.%20Biol.&amp;volume=210&amp;pages=1518-1525" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/jeb.001370" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fjeb.001370" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fjeb.001370"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17449817" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dohm</div> <div class="given-names">J. C.</div></div>,  <div class="name"><div class="surname">Lottaz</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Borodina</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Himmelbauer</div> <div class="given-names">H.</div></div>. </div><div class="article-title">SHARCGS, a fast and highly accurate short-read assembly algorithm for de novo genomic sequencing</div>, <div class="source">Genome Res.</div> , <div class="year">2007</div>, vol. <div class="volume">17</div> pg. <div class="fpage">1697</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SHARCGS%2C%20a%20fast%20and%20highly%20accurate%20short-read%20assembly%20algorithm%20for%20de%20novo%20genomic%20sequencing&amp;author=J.%20C.%20Dohm&amp;author=C.%20Lottaz&amp;author=T.%20Borodina&amp;author=H.%20Himmelbauer&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=1697" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.6435207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.6435207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.6435207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17908823" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hernandez</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">François</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Farinelli</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Østerås</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Schrenzel</div> <div class="given-names">J.</div></div>. </div><div class="article-title">De novo bacterial genome sequencing: millions of very short reads assembled on a desktop computer</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> pg. <div class="fpage">802</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20bacterial%20genome%20sequencing%3A%20millions%20of%20very%20short%20reads%20assembled%20on%20a%20desktop%20computer&amp;author=D.%20Hernandez&amp;author=P.%20Fran%C3%A7ois&amp;author=P.%20Farinelli&amp;author=M.%20%C3%98ster%C3%A5s&amp;author=J.%20Schrenzel&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=802" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.072033.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.072033.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.072033.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18332092" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Butler</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">MacCallum</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Kleber</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">ALLPATHS: de novo assembly of whole-genome shotgun microreads</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> pg. <div class="fpage">810</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ALLPATHS%3A%20de%20novo%20assembly%20of%20whole-genome%20shotgun%20microreads&amp;author=J.%20Butler&amp;author=I.%20MacCallum&amp;author=M.%20Kleber&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=810" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.7337908" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.7337908" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.7337908"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18340039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zerbino</div> <div class="given-names">B. R.</div></div>,  <div class="name"><div class="surname">Birney</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Velvet: algorithms for de novo short read assembly using de Bruijn graph</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> pg. <div class="fpage">821</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Velvet%3A%20algorithms%20for%20de%20novo%20short%20read%20assembly%20using%20de%20Bruijn%20graph&amp;author=B.%20R.%20Zerbino&amp;author=E.%20Birney&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=821" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.074492.107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.074492.107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.074492.107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18349386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Park</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Takagi</div> <div class="given-names">T.</div></div>. </div><div class="article-title">MetaGene: prokaryotic gene finding from environmental genome shotgun sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">5623</div>-<div class="lpage">5630</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MetaGene%3A%20prokaryotic%20gene%20finding%20from%20environmental%20genome%20shotgun%20sequences&amp;author=H.%20Noguchi&amp;author=J.%20Park&amp;author=T.%20Takagi&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=5623-5630" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl723" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl723" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl723"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17028096" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kurokawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Itoh</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kuwahara</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Comparative metagenomics revealed commonly enriched gene sets in human gut microbiomes</div>, <div class="source">DNA Res.</div> , <div class="year">2007</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">169</div>-<div class="lpage">181</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20metagenomics%20revealed%20commonly%20enriched%20gene%20sets%20in%20human%20gut%20microbiomes&amp;author=K.%20Kurokawa&amp;author=T.%20Itoh&amp;author=T.%20Kuwahara&amp;publication_year=2007&amp;journal=DNA%20Res.&amp;volume=14&amp;pages=169-181" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsm018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsm018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsm018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17916580" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Raes</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Foerstner</div> <div class="given-names">K. U.</div></div>,  <div class="name"><div class="surname">Bork</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Get the most out of your metagenome: computational analysis of environmental sequence data</div>, <div class="source">Curr. Opin. Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">490</div>-<div class="lpage">498</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Get%20the%20most%20out%20of%20your%20metagenome%3A%20computational%20analysis%20of%20environmental%20sequence%20data&amp;author=J.%20Raes&amp;author=K.%20U.%20Foerstner&amp;author=P.%20Bork&amp;publication_year=2007&amp;journal=Curr.%20Opin.%20Microbiol.&amp;volume=10&amp;pages=490-498" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2007.09.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2007.09.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2007.09.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17936679" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmeisser</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Steele</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Streit</div> <div class="given-names">W. R.</div></div>. </div><div class="article-title">Metagenomics, biotechnology with non-culturable microbes</div>, <div class="source">Appl. Microbiol. Biotechnol.</div> , <div class="year">2007</div>, vol. <div class="volume">75</div> (pg. <div class="fpage">955</div>-<div class="lpage">962</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metagenomics%2C%20biotechnology%20with%20non-culturable%20microbes&amp;author=C.%20Schmeisser&amp;author=H.%20Steele&amp;author=W.%20R.%20Streit&amp;publication_year=2007&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=75&amp;pages=955-962" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-007-0945-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-007-0945-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-007-0945-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17396253" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div>. </div><div class="article-title">Bioinformatics challenges of new sequencing technology</div>, <div class="source">Trends Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">142</div>-<div class="lpage">149</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioinformatics%20challenges%20of%20new%20sequencing%20technology&amp;author=M.%20Pop&amp;author=S.%20L.%20Salzberg&amp;publication_year=2008&amp;journal=Trends%20Genet.&amp;volume=24&amp;pages=142-149" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2007.12.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2007.12.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2007.12.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18262676" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pruitt</div> <div class="given-names">K. D.</div></div>,  <div class="name"><div class="surname">Tatusova</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Maglott</div> <div class="given-names">D. R.</div></div>. </div><div class="article-title">NCBI Reference Sequence (RefSeq): a curated non-redundant sequence database of genomes, transcripts and proteins</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D61</div>-<div class="lpage">65</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NCBI%20Reference%20Sequence%20(RefSeq)%3A%20a%20curated%20non-redundant%20sequence%20database%20of%20genomes%2C%20transcripts%20and%20proteins&amp;author=K.%20D.%20Pruitt&amp;author=T.%20Tatusova&amp;author=D.%20R.%20Maglott&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res.&amp;volume=35&amp;pages=D61-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl842" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl842" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl842"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17130148" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Saitou</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div><div class="article-title">The neighbor-joining method: a new method for reconstructing phylogenetic trees</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1987</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">406</div>-<div class="lpage">425</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20neighbor-joining%20method%3A%20a%20new%20method%20for%20reconstructing%20phylogenetic%20trees&amp;author=N.%20Saitou&amp;author=M.%20Nei&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=406-425" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3447015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hayashi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Makino</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ohnishi</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Complete genome sequence of enterohemorrhagic <em>Escherichia coli</em> O157:H7 and genomic comparison with a laboratory strain K-12</div>, <div class="source">DNA Res.</div> , <div class="year">2001</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">11</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20enterohemorrhagic%20Escherichia%20coli%20O157%3AH7%20and%20genomic%20comparison%20with%20a%20laboratory%20strain%20K-12&amp;author=T.%20Hayashi&amp;author=K.%20Makino&amp;author=M.%20Ohnishi&amp;publication_year=2001&amp;journal=DNA%20Res.&amp;volume=8&amp;pages=11-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/8.1.11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F8.1.11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F8.1.11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11258796" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ohnishi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kurokawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hayashi</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Diversification of <em>Escherichia coli</em> genomes: are bacteriophages the major contributors?</div>, <div class="source">Trens Microbiol.</div> , <div class="year">2001</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">481</div>-<div class="lpage">485</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversification%20of%20Escherichia%20coli%20genomes%3A%20are%20bacteriophages%20the%20major%20contributors%3F&amp;author=M.%20Ohnishi&amp;author=K.%20Kurokawa&amp;author=T.%20Hayashi&amp;publication_year=2001&amp;journal=Trens%20Microbiol.&amp;volume=9&amp;pages=481-485" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0966-842X(01)02173-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0966-842X(01)02173-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0966-842X(01)02173-4"> </span></div> </div></div></div></div></div><div content-id="DSN027C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Besemer</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Heuristic approach to deriving models for gene finding</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">3911</div>-<div class="lpage">3920</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heuristic%20approach%20to%20deriving%20models%20for%20gene%20finding&amp;author=J.%20Besemer&amp;author=M.%20Borodovsky&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=3911-3920" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.19.3911" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.19.3911" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.19.3911"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10481031" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shine</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Dalgarno</div> <div class="given-names">L.</div></div>. </div><div class="article-title">The 3′-terminal sequence of Escherichia coli 16S ribosomal RNA: Complementary to nonsense triplets and ribosome binding sites</div>, <div class="source">Proc. Natl. Acad. Sci. U. S. A.</div> , <div class="year">1974</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">1342</div>-<div class="lpage">1346</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%203%E2%80%B2-terminal%20sequence%20of%20Escherichia%20coli%2016S%20ribosomal%20RNA%3A%20Complementary%20to%20nonsense%20triplets%20and%20ribosome%20binding%20sites&amp;author=J.%20Shine&amp;author=L.%20Dalgarno&amp;publication_year=1974&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A.&amp;volume=71&amp;pages=1342-1346" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.71.4.1342" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.71.4.1342" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.71.4.1342"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4598299" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McHardy</div> <div class="given-names">A. C.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">H. G.</div></div>,  <div class="name"><div class="surname">Tsirigos</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hugenholtz</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Rigoutsos</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Accurate phylogenetic classification of variable-length DNA fragments</div>, <div class="source">Nat. Methods</div> , <div class="year">2007</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">63</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20phylogenetic%20classification%20of%20variable-length%20DNA%20fragments&amp;author=A.%20C.%20McHardy&amp;author=H.%20G.%20Martin&amp;author=A.%20Tsirigos&amp;author=P.%20Hugenholtz&amp;author=I.%20Rigoutsos&amp;publication_year=2007&amp;journal=Nat.%20Methods&amp;volume=4&amp;pages=63-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth976" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth976" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth976"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17179938" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN027C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN027C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krause</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Diaz</div> <div class="given-names">N. N.</div></div>,  <div class="name"><div class="surname">Goesmann</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Phylogenetic classification of short environmental DNA fragments</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">2230</div>-<div class="lpage">2239</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20classification%20of%20short%20environmental%20DNA%20fragments&amp;author=L.%20Krause&amp;author=N.%20N.%20Diaz&amp;author=A.%20Goesmann&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=2230-2239" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18285365" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=512877&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn027-dsn027supp_Table4.xls?Expires=2147483647&amp;Signature=5DGzK57oKVTOtG1VDYa0UKSmEei8-QGiLo12G8DkfCth3oR4d9TqIIKG2cL6B0r6hev-meMZRqxYW3lJJmppwKqonBNNZ99q~xe38q0NDYp0Q9Polw2r79Gw1dDrK~2eNFAXseuvucsTmqbCxFPvbFe24SXAtMrTwAaBGpjsjjq6duL-5XWNfqaO8k7yK453hRW8mkQVWfAaXE6wQnmz-JLbIL6KtQ5BaNXhua~70seKSOMAGLvrxTw9MOIlzzCHCv8gtxHrG7KqqYN4JRmVmHqXatnuKN9ODAjR~Bzqib~7VYYohrGqEy3XJjXohzKOK2Oob1nq8ZBoOQcv9dzVCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn027-dsn027supp_Table3.xls?Expires=2147483647&amp;Signature=ksXHrg5JsEvowKEUdcz5iUQyW2zVX8WtUCx3XUP7EjRtrx~p3jzwk3v9rXBWIE22CwKhl8yoZaJz1D-fICE2QOTlrsk~VtY~gcowlE3hG4gN4QzK3ZkkxVLZrDYYpMhZQf3BvAX36VX8Ne6Gd5vBSIE5FT4V1pTBS8tV8UmQVCw2u12J3PsDW7bIGJsvckz8zf04tNF84HKsLwYAowJcEO5~l90Gsygz51XSHR-4FVUh0dJYlCzgoaCA5-x52MAlu9UwsyH1ZL2ckuwiUsnOzQ9VitHrQQu8828~0dykG4bzgpdKi3ys30o~FlRl5VD6bEg0-zgHPw4a6saMTYTZ7w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn027-dsn027supp_Table2.xls?Expires=2147483647&amp;Signature=w8SLI8hN3HKey3PflBEpJCt33mehMqhprThdd9yZ3wg7s~h8dcgjAsEYLiH0cbdwD4-yk96aPBRZbBPfEDQQgAtarC~dcatgKQgh6H1wvukwaw-lc0xPR1RUS6s21F-T5-B3SB1SdC1RtYwkBMKB723zl63lR8E-V9M23zKJwiW3KpCfp1ypTPrQMUoLzSUhSyZadtn8nbIDbMoUKtKbJN71kuG-55pe1cjEFJ9CJggy-LpnyvfuPFfyZr2IopHISOjncoH3JgK0XWwKNI3C5uw8y1~3-7If5SJJNtcEJHTvuJLuX0d2woF9OblpN2317e8bLivoPeCBM853rewz3w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn027/2/dsn027-dsn027supp_Table1.xls?Expires=2147483647&amp;Signature=zQsGmS6A8UJ1~14SEtHHriR6rs6RyH7K9Z1Cr1EPl3Vaapwf-9SnzQIC-b5LJ2rpwqhWhvAajO9cPDo8ST4YX9mMDirnh5rAt3wGWd4andrm0rdb8pKqi4HXMC7WniB8zZBlD6OB3GauExcf02koHcukA-qVt5elXQHWHA0e4fcmD-CNjb9Xy1wStbRxpxJJdBfsEau2MJ2snA19cfqoGDjpE~JUY9tzNZfnxFHVMLjeqESSUC64DlGRqW04O-8PNAo127XIWwR2qax3QfGucXotXUJTyGkXZPFKTWiEMVnteyCLEbv5zaw-76CRb81kB4E8l6N4-e4b0H4FZdWwAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">987</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000262588400006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">192</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsn027"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="512877"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000262588400006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f6%2f387%2f512877">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000262588400006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (192)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F6%2F387%2F512877" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/512877'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="MetaGeneAnnotator: Detecting Species-Specific Patterns of Ribosomal Binding Site for Precise Gene Prediction in Anonymous Prokaryotic and Phage Genomes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsn027");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f6%2f387%2f512877">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

