

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/2/169/2870503","@type":"ScholarlyArticle","name":"Complete genome sequence and expression profile of the commercial lytic enzyme producer  Lysobacter enzymogenes  M497-1","datePublished":"2017-01-08","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2017-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw055","keywords":["<italic>Lysobacter enzymogenes</italic>","genome sequence","lytic enzyme","RNA sequencing","metabolic potential"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/2/169/2870503","author":[{"name":"Takami, Hideto","affiliation":"1 Microbial Genome Research Group, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa, Japan","@type":"Person"},{"name":"Toyoda, Atsushi","affiliation":"2 Comparative Genomics Laboratory, National Institute of Genetics, Mishima, Japan","@type":"Person"},{"name":"Uchiyama, Ikuo","affiliation":"3 Laboratory of Genome Informatics, National Institute for Basic Biology, National Institutes of Natural Sciences, Aichi, Japan","@type":"Person"},{"name":"Itoh, Takehiko","affiliation":"4 Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, Meguro-ku, Tokyo, Japan","@type":"Person"},{"name":"Takaki, Yoshihiro","affiliation":"1 Microbial Genome Research Group, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa, Japan","@type":"Person"},{"name":"Arai, Wataru","affiliation":"1 Microbial Genome Research Group, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa, Japan","@type":"Person"},{"name":"Nishi, Shinro","affiliation":"1 Microbial Genome Research Group, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa, Japan","@type":"Person"},{"name":"Kawai, Mikihiko","affiliation":"1 Microbial Genome Research Group, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa, Japan","@type":"Person"},{"name":"Shin-ya, Kazuo","affiliation":"5 National Institute of Advanced Industrial Science and Technology, Koto-ku, Tokyo, Japan","@type":"Person"},{"name":"Ikeda, Haruo","affiliation":"6 Kitasato Institute for Life Sciences, Kitasato University, Sagamihara, Kanagawa, Japan","@type":"Person"}],"description":"Abstract.  Lysobacter enzymogenes M497-1 is a producer of commercialized achromopeptidase and is expected to harbour genes encoding various other antimicrobial ","pageStart":"169","pageEnd":"177","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Complete genome sequence and expression profile of the commercial lytic enzyme producer  Lysobacter enzymogenes  M497-1","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Complete genome sequence and expression profile of the commercial lytic enzyme producer  Lysobacter enzymogenes  M497-1" />
    <meta property="og:description" content="Abstract.  Lysobacter enzymogenes M497-1 is a producer of commercialized achromopeptidase and is expected to harbour genes encoding various other antimicrobial " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/2/169/2870503" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-01-08" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Takami, Hideto" /><meta name="citation_author_institution" content="Microbial Genome Research Group, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa, Japan" /><meta name="citation_author" content="Toyoda, Atsushi" /><meta name="citation_author_institution" content="Comparative Genomics Laboratory, National Institute of Genetics, Mishima, Japan" /><meta name="citation_author" content="Uchiyama, Ikuo" /><meta name="citation_author_institution" content="Laboratory of Genome Informatics, National Institute for Basic Biology, National Institutes of Natural Sciences, Aichi, Japan" /><meta name="citation_author" content="Itoh, Takehiko" /><meta name="citation_author_institution" content="Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, Meguro-ku, Tokyo, Japan" /><meta name="citation_author" content="Takaki, Yoshihiro" /><meta name="citation_author_institution" content="Microbial Genome Research Group, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa, Japan" /><meta name="citation_author" content="Arai, Wataru" /><meta name="citation_author_institution" content="Microbial Genome Research Group, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa, Japan" /><meta name="citation_author" content="Nishi, Shinro" /><meta name="citation_author_institution" content="Microbial Genome Research Group, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa, Japan" /><meta name="citation_author" content="Kawai, Mikihiko" /><meta name="citation_author_institution" content="Microbial Genome Research Group, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa, Japan" /><meta name="citation_author" content="Shin-ya, Kazuo" /><meta name="citation_author_institution" content="National Institute of Advanced Industrial Science and Technology, Koto-ku, Tokyo, Japan" /><meta name="citation_author" content="Ikeda, Haruo" /><meta name="citation_author_institution" content="Kitasato Institute for Life Sciences, Kitasato University, Sagamihara, Kanagawa, Japan" /><meta name="citation_title" content="Complete genome sequence and expression profile of the commercial lytic enzyme producer Lysobacter enzymogenes M497-1" /><meta name="citation_firstpage" content="169" /><meta name="citation_lastpage" content="177" /><meta name="citation_doi" content="10.1093/dnares/dsw055" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2017/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="Isono M., Takahashi T., Yamazaki Y.  1972 , Bacteriolytic enzyme and process for the production thereof. US  Patent  3,649,454. " /><meta name="citation_reference" content="citation_title=A New proteolytic enzyme from Achromobacter lyticus M497-1;  citation_journal_title=Agric. Biol. Chem;  citation_year=1978;  citation_volume=42;  citation_pages=1443-5; " /><meta name="citation_reference" content="citation_title=Studies on a new proteolytic enzyme from Achromobacter lyticus M497-1. I. Purification and some enzymatic properties;  citation_journal_title=Biochim. Biophys. Acta;  citation_year=1981;  citation_volume=660;  citation_pages=44-50; " /><meta name="citation_reference" content="citation_title=Studies on a new proteolytic enzyme from Achromobacter lyticus M497-1. II. Specificity and inhibition studies of Achromobacter protease I;  citation_journal_title=Biochim. Biophys. Acta;  citation_year=1981;  citation_volume=660;  citation_pages=51-5; " /><meta name="citation_reference" content="citation_title=Purification and some properties of Achromobacter protease Ia from Achromobacter lyticus M497-1;  citation_journal_title=Agric. Biol. Chem.;  citation_year=1986;  citation_volume=50;  citation_pages=3087-91; " /><meta name="citation_reference" content="citation_title=Cloning, nucleotide sequence, and expression of Achromobacter protease I gene;  citation_journal_title=J. Biol. Chem.;  citation_year=1989;  citation_volume=264;  citation_pages=20625-31; " /><meta name="citation_reference" content="citation_title=Achromopeptidase for rapid lysis of oral anaerobic Gram-positive rods;  citation_journal_title=Oral Microbiol. Immunol;  citation_year=1988;  citation_volume=3;  citation_pages=86-8; " /><meta name="citation_reference" content="citation_title=Lysis of Micrococcus radiodurans;  citation_journal_title=Agric. Biol. Chem;  citation_year=1973;  citation_volume=37;  citation_pages=2667-8; " /><meta name="citation_reference" content="citation_title=A new isolation method of plasmid deoxyribonucleic acid from Staphylococcus aureus using a lytic enzyme of Achromobacter lyticus;  citation_journal_title=Agric. Biol. Chem;  citation_year=1977;  citation_volume=41;  citation_pages=2487-9; " /><meta name="citation_reference" content="citation_title=Achromopeptidase for lysis of anaerobic Gram-positive cocci;  citation_journal_title=J. Clinic. Microbiol.;  citation_year=1982;  citation_volume=16;  citation_pages=844-6; " /><meta name="citation_reference" content="citation_title=An improved DNA Isolation method for metagenomic analysis of the microbial flora of the human intestine;  citation_journal_title=Microbes Environ.;  citation_year=2007;  citation_volume=22;  citation_pages=214-22; " /><meta name="citation_reference" content="citation_title=Molecular cloning and nucleotide sequence of the β-lytic protease gene from Achromobacter lyticus;  citation_journal_title=J. Bacteriol.;  citation_year=1990;  citation_volume=172;  citation_pages=6506-11; " /><meta name="citation_reference" content="citation_title=Purification, Staphylolytic activity, and cleavage sites of α-lytic protease from Achromobacter lyticus;  citation_journal_title=J. Biochem.;  citation_year=1997;  citation_volume=122;  citation_pages=772-8; " /><meta name="citation_reference" content="citation_title=Bacteriolytic activity and specificity of Achromobacter β-lytic protease;  citation_journal_title=J. Biochem.;  citation_year=1998;  citation_volume=124;  citation_pages=332-9; " /><meta name="citation_reference" content="citation_title=The β-lytic protease of Myxobacter 495: aspects of its structure;  citation_journal_title=Proc. Can. Fed. Biol. Soc;  citation_year=1972;  citation_volume=15;  citation_pages=27" /><meta name="citation_reference" content="citation_title=Lysis of bacterial cell walls by an enzyme isolated from a Myxobacter;  citation_journal_title=J. Bacteriol.;  citation_year=1965;  citation_volume=90;  citation_pages=395-401; " /><meta name="citation_reference" content="citation_title=Characterization of a small proteolytic enzyme which lyses bacterial cell walls;  citation_journal_title=J. Bacteriol.;  citation_year=1966;  citation_volume=91;  citation_pages=524-34; " /><meta name="citation_reference" content="citation_title=Lytic enzymes of Sorangium sp., isolation and properties of the α-lytic and β-lytic proteases;  citation_journal_title=Can J. Biochem.;  citation_year=1965;  citation_volume=43;  citation_pages=1935-54; " /><meta name="citation_reference" content="citation_title=Biological control of Bipolaris sorokiniana on tall fescue by Stenotrophomonas maltophila strain C3;  citation_journal_title=Phytopathology;  citation_year=1999;  citation_volume=89;  citation_pages=817-22; " /><meta name="citation_reference" content="citation_title=Effect of culture fluids and preincubation of chitinase production on biocontrol of Bipolaris leaf spot by Stenotrophomonas maltophila C3;  citation_journal_title=Biol. Control;  citation_year=2000;  citation_volume=18;  citation_pages=277-86; " /><meta name="citation_reference" content="citation_title=Lysobacter, a new genus of nonfruiting, gliding bacteria with high base ratio;  citation_journal_title=Int. J. Syst. Bacteriol;  citation_year=1978;  citation_volume=28;  citation_pages=367-93; " /><meta name="citation_reference" content="citation_title=Lysobacter enzymogenes subsp. enzymogenes Christensen and Cook 1978, L. enzymogenes subsp. cookii Christensen 1978 and Streptococcus casseliflavus (Mundt and Graham 1968) Vaughan et al. 1978 should have been cited in the approved lists of bacterial names. Request for an opinion;  citation_journal_title=Int. J. Syst. Evolv. Microbiol;  citation_year=2006;  citation_volume=56;  citation_pages=2707-9; " /><meta name="citation_reference" content="citation_title=The α-lytic protease gene of Lysobacter enzymogenes. The nucleotide sequence predicts a large prepro-peptide with homology to pro-peptides of other chymotrypsin-like enzymes;  citation_journal_title=J. Biol. Chem;  citation_year=1988;  citation_volume=263;  citation_pages=16586-90; " /><meta name="citation_reference" content="citation_title=Lysobacter strain with high lysyl endopeptidase production;  citation_journal_title=FEMS Microbiol. Lett.;  citation_year=2002;  citation_volume=213;  citation_pages=13-20; " /><meta name="citation_reference" content="citation_title=A second lysine-specific serine protease from Lysobacter sp. strain IB-9374;  citation_journal_title=J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=5093-100; " /><meta name="citation_reference" content="citation_title=Bean rust biological control using bacterial agents;  citation_journal_title=Crop Protect.;  citation_year=2001;  citation_volume=20;  citation_pages=395-402; " /><meta name="citation_reference" content="citation_title=Characterization of Lysobacter enzymogenes (Christensen and Cook 1978) strain 3.1T8, a powerful antagonist of fungal diseases of cucumber;  citation_journal_title=Microbiol. Res;  citation_year=2003;  citation_volume=158;  citation_pages=107-15; " /><meta name="citation_reference" content="citation_title=Influence of Lysobacter enzymogenes strain C3 on Nematodes;  citation_journal_title=J. Nematol;  citation_year=2006;  citation_volume=38;  citation_pages=233-9; " /><meta name="citation_reference" content="citation_title=Effect of plant and microbial chitinases on the eggs and juveniles of Meloidogyne hapla Chitiwood;  citation_journal_title=Nematologica;  citation_year=1992;  citation_volume=38;  citation_pages=227-36; " /><meta name="citation_reference" content="citation_title=The role of chitinase production by Stenotrophomonas maltophilia strain C3 in biological control of Bipolaris sorokiniana;  citation_journal_title=Phytopathology;  citation_year=2000;  citation_volume=90;  citation_pages=384-9; " /><meta name="citation_reference" content="citation_title=Chitinases from the plant disease biocontrol agent, Stenotrophomonas maltophilia C3;  citation_journal_title=Phytopathology;  citation_year=2001;  citation_volume=91;  citation_pages=204-10; " /><meta name="citation_reference" content="citation_title=Selection of bacterial antagonists using enrichment cultures for the control of summer patch disease in Kentucky Bluegrass;  citation_journal_title=Cur. Microbiol.;  citation_year=1996;  citation_volume=32;  citation_pages=106-11; " /><meta name="citation_reference" content="citation_title=Molecular characterization and expression in Escherichia coli of three β-1,3-glucanase genes from Lysobacter enzymogenes strain N4-7;  citation_journal_title=J. Bacteriol;  citation_year=2003;  citation_volume=185;  citation_pages=4362-70; " /><meta name="citation_reference" content="citation_title=Tripropeptins, novel antimicrobial agents produced by Lysobacter sp., I. Taxonomy, isolation and biological activities;  citation_journal_title=J. Antibiotics;  citation_year=2001;  citation_volume=54;  citation_pages=1054-9; " /><meta name="citation_reference" content="citation_title=A new type of tripropeptin with anteiso-branched chain fatty acid from Lysobacter sp. BMK333-48F3;  citation_journal_title=J. Antibiotics;  citation_year=2008;  citation_volume=61;  citation_pages=577-82; " /><meta name="citation_reference" content="citation_title=Lysocin E is a new antibiotic that targets menaquinone in the bacterial membrane;  citation_journal_title=Nat. Chem. Biol.;  citation_year=2015;  citation_volume=11;  citation_pages=127-33; " /><meta name="citation_reference" content="citation_title=Comparative genomics and metabolic profiling of the genus Lysobacter;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=991" /><meta name="citation_reference" content="citation_title=Automated finishing with autofinish;  citation_journal_title=Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=614-25; " /><meta name="citation_reference" content="citation_title=PePPER: a webserver for prediction of prokaryote promoter elements and regulons;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=299" /><meta name="citation_reference" content="citation_title=MEROPS: the database of proteolytic enzymes, their substrates and inhibitors;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=D503-509; " /><meta name="citation_reference" content="citation_title=Hierarchical clustering algorithm for comprehensive orthologous-domain classification in multiple genomes;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=647-58; " /><meta name="citation_reference" content="citation_title=MBGD update 2015: microbial genome database for flexible ortholog analysis utilizing a diverse set of genomic data;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=D270-6; " /><meta name="citation_reference" content="citation_title=Multiple genome alignment for identifying the core structure among moderately related microbial genomes;  citation_journal_title=BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=515" /><meta name="citation_reference" content="citation_title=NRPS-PKS: a knowledge-based resource for analysis of NRPS/PKS megasynthases;  citation_journal_title=Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=W405-13; " /><meta name="citation_reference" content="citation_title=ASMPKS: an analysis system for modular polyketide synthases;  citation_journal_title=BMC Bioinformatics.;  citation_year=2007;  citation_volume=8;  citation_pages=327" /><meta name="citation_reference" content="citation_title=Methods for in silico prediction of microbial secondary metabolic pathways from DNA sequence data;  citation_journal_title=Methods Enzymol;  citation_year=2009;  citation_volume=458;  citation_pages=181-217; " /><meta name="citation_reference" content="citation_title=antiSMASH: rapid identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genome sequences;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=W339-46; " /><meta name="citation_reference" content="citation_title=An automated system for evaluation of the potential functionome: MAPLE version 2.1.0;  citation_journal_title=DNA Res;  citation_year=2016; " /><meta name="citation_reference" content="citation_title=Evaluation method for the potential functionome harbored in the genome and metagenome;  citation_journal_title=BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=699" /><meta name="citation_reference" content="citation_title=Avoidance of iron toxicity through regulation of bacterial iron transport;  citation_journal_title=Biol. Chem.;  citation_year=1997;  citation_volume=378;  citation_pages=779-786; " /><meta name="citation_reference" content="citation_title=TonB-dependent transporters: regulation, structure, and function;  citation_journal_title=Annu. Rev. Microbiol.;  citation_year=2010;  citation_volume=64;  citation_pages=43-60; " /><meta name="citation_reference" content="citation_publisher=ASM Press, Washington, DC;  citation_title=Bacillus subtilis and Other Gram-Positive Bacteria; " /><meta name="citation_reference" content="citation_publisher=ASM Press, Washington, DC;  citation_title=Antibiotics; Actions, Origins, Resistance;  citation_year=2003; " /><meta name="citation_reference" content="citation_title=Bioactive natural products from Lysobacter;  citation_journal_title=Nat. Prod. Pep;  citation_year=2012;  citation_volume=29;  citation_pages=1277-87; " /><meta name="citation_reference" content="citation_title=Identification and characterization of the lysobactin biosynthetic gene cluster reveals mechanistic insights into an unusual termination module architecture;  citation_journal_title=Chem. Biol.;  citation_year=2011;  citation_volume=18;  citation_pages=655-64; " /><meta name="citation_reference" content="citation_title=A clp gene homologue belonging to the crp gene family globally regulates lytic enzyme production, antimicrobial activity, and biological control activity expressed by Lysobacter enzymogenes strain C3;  citation_journal_title=Appl. Environ. Microbiol.;  citation_year=2005;  citation_volume=71;  citation_pages=261-9; " /><meta name="citation_reference" content="citation_title=Transcriptomic analysis reveals new regulatory roles of Clp signaling in secondary metabolite synthesis and surface motility in Lysobacter enzymogenes OH11;  citation_journal_title=Appl. Microbiol. Biotechnol;  citation_year=2014;  citation_volume=98;  citation_pages=9009-20; " /><meta name="citation_reference" content="citation_title=Identification of a small molecule signaling factor that regulates the biosynthesis of the antifungal polycyclic tetramate macrolactam HSAF in Lysobacter enzymogenes;  citation_journal_title=Appl. Microbiol. Biotechnol.;  citation_year=2015;  citation_volume=99;  citation_pages=801-11; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/2/169/13688910/dsw055.pdf" /><meta name="description" content="Abstract.  Lysobacter enzymogenes M497-1 is a producer of commercialized achromopeptidase and is expected to harbour genes encoding various other antimicrobial " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/2/169/2870503" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/2/169/2870503" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/2/169/2870503");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw055';
    </script>
<title>
	Complete genome sequence and expression profile of the commercial lytic enzyme producer Lysobacter enzymogenes M497-1 | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2870503" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BcGQATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFrYW1pLCBIaWRldG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNaWNyb2JpYWwgR2Vub21lIFJlc2VhcmNoIEdyb3VwLCBKYXBhbiBBZ2VuY3kgZm9yIE1hcmluZS1FYXJ0aCBTY2llbmNlIGFuZCBUZWNobm9sb2d5IChKQU1TVEVDKSwgWW9rb2hhbWEsIEthbmFnYXdhLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRveW9kYSwgQXRzdXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNvbXBhcmF0aXZlIEdlbm9taWNzIExhYm9yYXRvcnksIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBHZW5ldGljcywgTWlzaGltYSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJVY2hpeWFtYSwgSWt1byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYm9yYXRvcnkgb2YgR2Vub21lIEluZm9ybWF0aWNzLCBOYXRpb25hbCBJbnN0aXR1dGUgZm9yIEJhc2ljIEJpb2xvZ3ksIE5hdGlvbmFsIEluc3RpdHV0ZXMgb2YgTmF0dXJhbCBTY2llbmNlcywgQWljaGksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXRvaCwgVGFrZWhpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvc2NpZW5jZSBhbmQgQmlvdGVjaG5vbG9neSwgVG9reW8gSW5zdGl0dXRlIG9mIFRlY2hub2xvZ3ksIE1lZ3Vyby1rdSwgVG9reW8sIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFrYWtpLCBZb3NoaWhpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNaWNyb2JpYWwgR2Vub21lIFJlc2VhcmNoIEdyb3VwLCBKYXBhbiBBZ2VuY3kgZm9yIE1hcmluZS1FYXJ0aCBTY2llbmNlIGFuZCBUZWNobm9sb2d5IChKQU1TVEVDKSwgWW9rb2hhbWEsIEthbmFnYXdhLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFyYWksIFdhdGFydSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1pY3JvYmlhbCBHZW5vbWUgUmVzZWFyY2ggR3JvdXAsIEphcGFuIEFnZW5jeSBmb3IgTWFyaW5lLUVhcnRoIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgKEpBTVNURUMpLCBZb2tvaGFtYSwgS2FuYWdhd2EsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmlzaGksIFNoaW5ybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1pY3JvYmlhbCBHZW5vbWUgUmVzZWFyY2ggR3JvdXAsIEphcGFuIEFnZW5jeSBmb3IgTWFyaW5lLUVhcnRoIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgKEpBTVNURUMpLCBZb2tvaGFtYSwgS2FuYWdhd2EsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F3YWksIE1pa2loaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTWljcm9iaWFsIEdlbm9tZSBSZXNlYXJjaCBHcm91cCwgSmFwYW4gQWdlbmN5IGZvciBNYXJpbmUtRWFydGggU2NpZW5jZSBhbmQgVGVjaG5vbG9neSAoSkFNU1RFQyksIFlva29oYW1hLCBLYW5hZ2F3YSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTaGluLXlhLCBLYXp1byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZHZhbmNlZCBJbmR1c3RyaWFsIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3ksIEtvdG8ta3UsIFRva3lvLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IklrZWRhLCBIYXJ1byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IktpdGFzYXRvIEluc3RpdHV0ZSBmb3IgTGlmZSBTY2llbmNlcywgS2l0YXNhdG8gVW5pdmVyc2l0eSwgU2FnYW1paGFyYSwgS2FuYWdhd2EsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJDb21wbGV0ZSBnZW5vbWUgc2VxdWVuY2UgYW5kIGV4cHJlc3Npb24gcHJvZmlsZSBvZiB0aGUgY29tbWVyY2lhbCBseXRpYyBlbnp5bWUgcHJvZHVjZXIgTHlzb2JhY3RlciBlbnp5bW9nZW5lcyBNNDk3LTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxNjkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjE3NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdzA1NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE3LzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iSXNvbm8gTS4sIFRha2FoYXNoaSBULiwgWWFtYXpha2kgWS4gIDE5NzIgLCBCYWN0ZXJpb2x5dGljIGVuenltZSBhbmQgcHJvY2VzcyBmb3IgdGhlIHByb2R1Y3Rpb24gdGhlcmVvZi4gVVMgIFBhdGVudCAgMyw2NDksNDU0LiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIE5ldyBwcm90ZW9seXRpYyBlbnp5bWUgZnJvbSBBY2hyb21vYmFjdGVyIGx5dGljdXMgTTQ5Ny0xOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BZ3JpYy4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MTk3ODsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTE0NDMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3R1ZGllcyBvbiBhIG5ldyBwcm90ZW9seXRpYyBlbnp5bWUgZnJvbSBBY2hyb21vYmFjdGVyIGx5dGljdXMgTTQ5Ny0xLiBJLiBQdXJpZmljYXRpb24gYW5kIHNvbWUgZW56eW1hdGljIHByb3BlcnRpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoaW0uIEJpb3BoeXMuIEFjdGE7ICBjaXRhdGlvbl95ZWFyPTE5ODE7ICBjaXRhdGlvbl92b2x1bWU9NjYwOyAgY2l0YXRpb25fcGFnZXM9NDQtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0dWRpZXMgb24gYSBuZXcgcHJvdGVvbHl0aWMgZW56eW1lIGZyb20gQWNocm9tb2JhY3RlciBseXRpY3VzIE00OTctMS4gSUkuIFNwZWNpZmljaXR5IGFuZCBpbmhpYml0aW9uIHN0dWRpZXMgb2YgQWNocm9tb2JhY3RlciBwcm90ZWFzZSBJOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9jaGltLiBCaW9waHlzLiBBY3RhOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTY2MDsgIGNpdGF0aW9uX3BhZ2VzPTUxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVB1cmlmaWNhdGlvbiBhbmQgc29tZSBwcm9wZXJ0aWVzIG9mIEFjaHJvbW9iYWN0ZXIgcHJvdGVhc2UgSWEgZnJvbSBBY2hyb21vYmFjdGVyIGx5dGljdXMgTTQ5Ny0xOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BZ3JpYy4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9wYWdlcz0zMDg3LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbG9uaW5nLCBudWNsZW90aWRlIHNlcXVlbmNlLCBhbmQgZXhwcmVzc2lvbiBvZiBBY2hyb21vYmFjdGVyIHByb3RlYXNlIEkgZ2VuZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl92b2x1bWU9MjY0OyAgY2l0YXRpb25fcGFnZXM9MjA2MjUtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjaHJvbW9wZXB0aWRhc2UgZm9yIHJhcGlkIGx5c2lzIG9mIG9yYWwgYW5hZXJvYmljIEdyYW0tcG9zaXRpdmUgcm9kczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9T3JhbCBNaWNyb2Jpb2wuIEltbXVub2w7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTg2LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUx5c2lzIG9mIE1pY3JvY29jY3VzIHJhZGlvZHVyYW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BZ3JpYy4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MTk3MzsgIGNpdGF0aW9uX3ZvbHVtZT0zNzsgIGNpdGF0aW9uX3BhZ2VzPTI2NjctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgaXNvbGF0aW9uIG1ldGhvZCBvZiBwbGFzbWlkIGRlb3h5cmlib251Y2xlaWMgYWNpZCBmcm9tIFN0YXBoeWxvY29jY3VzIGF1cmV1cyB1c2luZyBhIGx5dGljIGVuenltZSBvZiBBY2hyb21vYmFjdGVyIGx5dGljdXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFncmljLiBCaW9sLiBDaGVtOyAgY2l0YXRpb25feWVhcj0xOTc3OyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9MjQ4Ny05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2hyb21vcGVwdGlkYXNlIGZvciBseXNpcyBvZiBhbmFlcm9iaWMgR3JhbS1wb3NpdGl2ZSBjb2NjaTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQ2xpbmljLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0xOTgyOyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9ODQ0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGltcHJvdmVkIEROQSBJc29sYXRpb24gbWV0aG9kIGZvciBtZXRhZ2Vub21pYyBhbmFseXNpcyBvZiB0aGUgbWljcm9iaWFsIGZsb3JhIG9mIHRoZSBodW1hbiBpbnRlc3RpbmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYmVzIEVudmlyb24uOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MjE0LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgY2xvbmluZyBhbmQgbnVjbGVvdGlkZSBzZXF1ZW5jZSBvZiB0aGUgzrItbHl0aWMgcHJvdGVhc2UgZ2VuZSBmcm9tIEFjaHJvbW9iYWN0ZXIgbHl0aWN1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzI7ICBjaXRhdGlvbl9wYWdlcz02NTA2LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QdXJpZmljYXRpb24sIFN0YXBoeWxvbHl0aWMgYWN0aXZpdHksIGFuZCBjbGVhdmFnZSBzaXRlcyBvZiDOsS1seXRpYyBwcm90ZWFzZSBmcm9tIEFjaHJvbW9iYWN0ZXIgbHl0aWN1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTIyOyAgY2l0YXRpb25fcGFnZXM9NzcyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhY3RlcmlvbHl0aWMgYWN0aXZpdHkgYW5kIHNwZWNpZmljaXR5IG9mIEFjaHJvbW9iYWN0ZXIgzrItbHl0aWMgcHJvdGVhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTEyNDsgIGNpdGF0aW9uX3BhZ2VzPTMzMi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgzrItbHl0aWMgcHJvdGVhc2Ugb2YgTXl4b2JhY3RlciA0OTU6IGFzcGVjdHMgb2YgaXRzIHN0cnVjdHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gQ2FuLiBGZWQuIEJpb2wuIFNvYzsgIGNpdGF0aW9uX3llYXI9MTk3MjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9THlzaXMgb2YgYmFjdGVyaWFsIGNlbGwgd2FsbHMgYnkgYW4gZW56eW1lIGlzb2xhdGVkIGZyb20gYSBNeXhvYmFjdGVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTY1OyAgY2l0YXRpb25fdm9sdW1lPTkwOyAgY2l0YXRpb25fcGFnZXM9Mzk1LTQwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbiBvZiBhIHNtYWxsIHByb3Rlb2x5dGljIGVuenltZSB3aGljaCBseXNlcyBiYWN0ZXJpYWwgY2VsbCB3YWxsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk2NjsgIGNpdGF0aW9uX3ZvbHVtZT05MTsgIGNpdGF0aW9uX3BhZ2VzPTUyNC0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9THl0aWMgZW56eW1lcyBvZiBTb3JhbmdpdW0gc3AuLCBpc29sYXRpb24gYW5kIHByb3BlcnRpZXMgb2YgdGhlIM6xLWx5dGljIGFuZCDOsi1seXRpYyBwcm90ZWFzZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNhbiBKLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk2NTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTE5MzUtNTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb2xvZ2ljYWwgY29udHJvbCBvZiBCaXBvbGFyaXMgc29yb2tpbmlhbmEgb24gdGFsbCBmZXNjdWUgYnkgU3Rlbm90cm9waG9tb25hcyBtYWx0b3BoaWxhIHN0cmFpbiBDMzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5dG9wYXRob2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9ODk7ICBjaXRhdGlvbl9wYWdlcz04MTctMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmVjdCBvZiBjdWx0dXJlIGZsdWlkcyBhbmQgcHJlaW5jdWJhdGlvbiBvZiBjaGl0aW5hc2UgcHJvZHVjdGlvbiBvbiBiaW9jb250cm9sIG9mIEJpcG9sYXJpcyBsZWFmIHNwb3QgYnkgU3Rlbm90cm9waG9tb25hcyBtYWx0b3BoaWxhIEMzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9sLiBDb250cm9sOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9Mjc3LTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MeXNvYmFjdGVyLCBhIG5ldyBnZW51cyBvZiBub25mcnVpdGluZywgZ2xpZGluZyBiYWN0ZXJpYSB3aXRoIGhpZ2ggYmFzZSByYXRpbzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBKLiBTeXN0LiBCYWN0ZXJpb2w7ICBjaXRhdGlvbl95ZWFyPTE5Nzg7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0zNjctOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUx5c29iYWN0ZXIgZW56eW1vZ2VuZXMgc3Vic3AuIGVuenltb2dlbmVzIENocmlzdGVuc2VuIGFuZCBDb29rIDE5NzgsIEwuIGVuenltb2dlbmVzIHN1YnNwLiBjb29raWkgQ2hyaXN0ZW5zZW4gMTk3OCBhbmQgU3RyZXB0b2NvY2N1cyBjYXNzZWxpZmxhdnVzIChNdW5kdCBhbmQgR3JhaGFtIDE5NjgpIFZhdWdoYW4gZXQgYWwuIDE5Nzggc2hvdWxkIGhhdmUgYmVlbiBjaXRlZCBpbiB0aGUgYXBwcm92ZWQgbGlzdHMgb2YgYmFjdGVyaWFsIG5hbWVzLiBSZXF1ZXN0IGZvciBhbiBvcGluaW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIFN5c3QuIEV2b2x2LiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NTY7ICBjaXRhdGlvbl9wYWdlcz0yNzA3LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSDOsS1seXRpYyBwcm90ZWFzZSBnZW5lIG9mIEx5c29iYWN0ZXIgZW56eW1vZ2VuZXMuIFRoZSBudWNsZW90aWRlIHNlcXVlbmNlIHByZWRpY3RzIGEgbGFyZ2UgcHJlcHJvLXBlcHRpZGUgd2l0aCBob21vbG9neSB0byBwcm8tcGVwdGlkZXMgb2Ygb3RoZXIgY2h5bW90cnlwc2luLWxpa2UgZW56eW1lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmlvbC4gQ2hlbTsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNjM7ICBjaXRhdGlvbl9wYWdlcz0xNjU4Ni05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9THlzb2JhY3RlciBzdHJhaW4gd2l0aCBoaWdoIGx5c3lsIGVuZG9wZXB0aWRhc2UgcHJvZHVjdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVNUyBNaWNyb2Jpb2wuIExldHQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTIxMzsgIGNpdGF0aW9uX3BhZ2VzPTEzLTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNlY29uZCBseXNpbmUtc3BlY2lmaWMgc2VyaW5lIHByb3RlYXNlIGZyb20gTHlzb2JhY3RlciBzcC4gc3RyYWluIElCLTkzNzQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTg2OyAgY2l0YXRpb25fcGFnZXM9NTA5My0xMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJlYW4gcnVzdCBiaW9sb2dpY2FsIGNvbnRyb2wgdXNpbmcgYmFjdGVyaWFsIGFnZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3JvcCBQcm90ZWN0LjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTM5NS00MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgTHlzb2JhY3RlciBlbnp5bW9nZW5lcyAoQ2hyaXN0ZW5zZW4gYW5kIENvb2sgMTk3OCkgc3RyYWluIDMuMVQ4LCBhIHBvd2VyZnVsIGFudGFnb25pc3Qgb2YgZnVuZ2FsIGRpc2Vhc2VzIG9mIGN1Y3VtYmVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NaWNyb2Jpb2wuIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTg7ICBjaXRhdGlvbl9wYWdlcz0xMDctMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluZmx1ZW5jZSBvZiBMeXNvYmFjdGVyIGVuenltb2dlbmVzIHN0cmFpbiBDMyBvbiBOZW1hdG9kZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE5lbWF0b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9Mzg7ICBjaXRhdGlvbl9wYWdlcz0yMzMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWZmZWN0IG9mIHBsYW50IGFuZCBtaWNyb2JpYWwgY2hpdGluYXNlcyBvbiB0aGUgZWdncyBhbmQganV2ZW5pbGVzIG9mIE1lbG9pZG9neW5lIGhhcGxhIENoaXRpd29vZDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmVtYXRvbG9naWNhOyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9MjI3LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcm9sZSBvZiBjaGl0aW5hc2UgcHJvZHVjdGlvbiBieSBTdGVub3Ryb3Bob21vbmFzIG1hbHRvcGhpbGlhIHN0cmFpbiBDMyBpbiBiaW9sb2dpY2FsIGNvbnRyb2wgb2YgQmlwb2xhcmlzIHNvcm9raW5pYW5hOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QaHl0b3BhdGhvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT05MDsgIGNpdGF0aW9uX3BhZ2VzPTM4NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGl0aW5hc2VzIGZyb20gdGhlIHBsYW50IGRpc2Vhc2UgYmlvY29udHJvbCBhZ2VudCwgU3Rlbm90cm9waG9tb25hcyBtYWx0b3BoaWxpYSBDMzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5dG9wYXRob2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz0yMDQtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGVjdGlvbiBvZiBiYWN0ZXJpYWwgYW50YWdvbmlzdHMgdXNpbmcgZW5yaWNobWVudCBjdWx0dXJlcyBmb3IgdGhlIGNvbnRyb2wgb2Ygc3VtbWVyIHBhdGNoIGRpc2Vhc2UgaW4gS2VudHVja3kgQmx1ZWdyYXNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXIuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0xMDYtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjaGFyYWN0ZXJpemF0aW9uIGFuZCBleHByZXNzaW9uIGluIEVzY2hlcmljaGlhIGNvbGkgb2YgdGhyZWUgzrItMSwzLWdsdWNhbmFzZSBnZW5lcyBmcm9tIEx5c29iYWN0ZXIgZW56eW1vZ2VuZXMgc3RyYWluIE40LTc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xODU7ICBjaXRhdGlvbl9wYWdlcz00MzYyLTcwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ucmlwcm9wZXB0aW5zLCBub3ZlbCBhbnRpbWljcm9iaWFsIGFnZW50cyBwcm9kdWNlZCBieSBMeXNvYmFjdGVyIHNwLiwgSS4gVGF4b25vbXksIGlzb2xhdGlvbiBhbmQgYmlvbG9naWNhbCBhY3Rpdml0aWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBBbnRpYmlvdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTEwNTQtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgdHlwZSBvZiB0cmlwcm9wZXB0aW4gd2l0aCBhbnRlaXNvLWJyYW5jaGVkIGNoYWluIGZhdHR5IGFjaWQgZnJvbSBMeXNvYmFjdGVyIHNwLiBCTUszMzMtNDhGMzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQW50aWJpb3RpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NjE7ICBjaXRhdGlvbl9wYWdlcz01NzctODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUx5c29jaW4gRSBpcyBhIG5ldyBhbnRpYmlvdGljIHRoYXQgdGFyZ2V0cyBtZW5hcXVpbm9uZSBpbiB0aGUgYmFjdGVyaWFsIG1lbWJyYW5lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENoZW0uIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9MTI3LTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5vbWljcyBhbmQgbWV0YWJvbGljIHByb2ZpbGluZyBvZiB0aGUgZ2VudXMgTHlzb2JhY3RlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9OTkxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b21hdGVkIGZpbmlzaGluZyB3aXRoIGF1dG9maW5pc2g7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NjE0LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZVBQRVI6IGEgd2Vic2VydmVyIGZvciBwcmVkaWN0aW9uIG9mIHByb2thcnlvdGUgcHJvbW90ZXIgZWxlbWVudHMgYW5kIHJlZ3Vsb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yOTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRVJPUFM6IHRoZSBkYXRhYmFzZSBvZiBwcm90ZW9seXRpYyBlbnp5bWVzLCB0aGVpciBzdWJzdHJhdGVzIGFuZCBpbmhpYml0b3JzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPUQ1MDMtNTA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1IaWVyYXJjaGljYWwgY2x1c3RlcmluZyBhbGdvcml0aG0gZm9yIGNvbXByZWhlbnNpdmUgb3J0aG9sb2dvdXMtZG9tYWluIGNsYXNzaWZpY2F0aW9uIGluIG11bHRpcGxlIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9NjQ3LTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQkdEIHVwZGF0ZSAyMDE1OiBtaWNyb2JpYWwgZ2Vub21lIGRhdGFiYXNlIGZvciBmbGV4aWJsZSBvcnRob2xvZyBhbmFseXNpcyB1dGlsaXppbmcgYSBkaXZlcnNlIHNldCBvZiBnZW5vbWljIGRhdGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9RDI3MC02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZSBnZW5vbWUgYWxpZ25tZW50IGZvciBpZGVudGlmeWluZyB0aGUgY29yZSBzdHJ1Y3R1cmUgYW1vbmcgbW9kZXJhdGVseSByZWxhdGVkIG1pY3JvYmlhbCBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTUxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5SUFMtUEtTOiBhIGtub3dsZWRnZS1iYXNlZCByZXNvdXJjZSBmb3IgYW5hbHlzaXMgb2YgTlJQUy9QS1MgbWVnYXN5bnRoYXNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz1XNDA1LTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BU01QS1M6IGFuIGFuYWx5c2lzIHN5c3RlbSBmb3IgbW9kdWxhciBwb2x5a2V0aWRlIHN5bnRoYXNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybWF0aWNzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MzI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWV0aG9kcyBmb3IgaW4gc2lsaWNvIHByZWRpY3Rpb24gb2YgbWljcm9iaWFsIHNlY29uZGFyeSBtZXRhYm9saWMgcGF0aHdheXMgZnJvbSBETkEgc2VxdWVuY2UgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWV0aG9kcyBFbnp5bW9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTQ1ODsgIGNpdGF0aW9uX3BhZ2VzPTE4MS0yMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWFudGlTTUFTSDogcmFwaWQgaWRlbnRpZmljYXRpb24sIGFubm90YXRpb24gYW5kIGFuYWx5c2lzIG9mIHNlY29uZGFyeSBtZXRhYm9saXRlIGJpb3N5bnRoZXNpcyBnZW5lIGNsdXN0ZXJzIGluIGJhY3RlcmlhbCBhbmQgZnVuZ2FsIGdlbm9tZSBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM5OyAgY2l0YXRpb25fcGFnZXM9VzMzOS00NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYXV0b21hdGVkIHN5c3RlbSBmb3IgZXZhbHVhdGlvbiBvZiB0aGUgcG90ZW50aWFsIGZ1bmN0aW9ub21lOiBNQVBMRSB2ZXJzaW9uIDIuMS4wOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmFsdWF0aW9uIG1ldGhvZCBmb3IgdGhlIHBvdGVudGlhbCBmdW5jdGlvbm9tZSBoYXJib3JlZCBpbiB0aGUgZ2Vub21lIGFuZCBtZXRhZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz02OTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bdm9pZGFuY2Ugb2YgaXJvbiB0b3hpY2l0eSB0aHJvdWdoIHJlZ3VsYXRpb24gb2YgYmFjdGVyaWFsIGlyb24gdHJhbnNwb3J0OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0zNzg7ICBjaXRhdGlvbl9wYWdlcz03NzktNzg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub25CLWRlcGVuZGVudCB0cmFuc3BvcnRlcnM6IHJlZ3VsYXRpb24sIHN0cnVjdHVyZSwgYW5kIGZ1bmN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm51LiBSZXYuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjQ7ICBjaXRhdGlvbl9wYWdlcz00My02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUFTTSBQcmVzcywgV2FzaGluZ3RvbiwgREM7ICBjaXRhdGlvbl90aXRsZT1CYWNpbGx1cyBzdWJ0aWxpcyBhbmQgT3RoZXIgR3JhbS1Qb3NpdGl2ZSBCYWN0ZXJpYTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUFTTSBQcmVzcywgV2FzaGluZ3RvbiwgREM7ICBjaXRhdGlvbl90aXRsZT1BbnRpYmlvdGljczsgQWN0aW9ucywgT3JpZ2lucywgUmVzaXN0YW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmlvYWN0aXZlIG5hdHVyYWwgcHJvZHVjdHMgZnJvbSBMeXNvYmFjdGVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFByb2QuIFBlcDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTEyNzctODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHRoZSBseXNvYmFjdGluIGJpb3N5bnRoZXRpYyBnZW5lIGNsdXN0ZXIgcmV2ZWFscyBtZWNoYW5pc3RpYyBpbnNpZ2h0cyBpbnRvIGFuIHVudXN1YWwgdGVybWluYXRpb24gbW9kdWxlIGFyY2hpdGVjdHVyZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2hlbS4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz02NTUtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY2xwIGdlbmUgaG9tb2xvZ3VlIGJlbG9uZ2luZyB0byB0aGUgY3JwIGdlbmUgZmFtaWx5IGdsb2JhbGx5IHJlZ3VsYXRlcyBseXRpYyBlbnp5bWUgcHJvZHVjdGlvbiwgYW50aW1pY3JvYmlhbCBhY3Rpdml0eSwgYW5kIGJpb2xvZ2ljYWwgY29udHJvbCBhY3Rpdml0eSBleHByZXNzZWQgYnkgTHlzb2JhY3RlciBlbnp5bW9nZW5lcyBzdHJhaW4gQzM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIEVudmlyb24uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NzE7ICBjaXRhdGlvbl9wYWdlcz0yNjEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9taWMgYW5hbHlzaXMgcmV2ZWFscyBuZXcgcmVndWxhdG9yeSByb2xlcyBvZiBDbHAgc2lnbmFsaW5nIGluIHNlY29uZGFyeSBtZXRhYm9saXRlIHN5bnRoZXNpcyBhbmQgc3VyZmFjZSBtb3RpbGl0eSBpbiBMeXNvYmFjdGVyIGVuenltb2dlbmVzIE9IMTE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIE1pY3JvYmlvbC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTkwMDktMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGEgc21hbGwgbW9sZWN1bGUgc2lnbmFsaW5nIGZhY3RvciB0aGF0IHJlZ3VsYXRlcyB0aGUgYmlvc3ludGhlc2lzIG9mIHRoZSBhbnRpZnVuZ2FsIHBvbHljeWNsaWMgdGV0cmFtYXRlIG1hY3JvbGFjdGFtIEhTQUYgaW4gTHlzb2JhY3RlciBlbnp5bW9nZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QXBwbC4gTWljcm9iaW9sLiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTgwMS0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI0LzIvMTY5LzEzNjg4OTEwL2RzdzA1NS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgTHlzb2JhY3RlciBlbnp5bW9nZW5lcyBNNDk3LTEgaXMgYSBwcm9kdWNlciBvZiBjb21tZXJjaWFsaXplZCBhY2hyb21vcGVwdGlkYXNlIGFuZCBpcyBleHBlY3RlZCB0byBoYXJib3VyIGdlbmVzIGVuY29kaW5nIHZhcmlvdXMgb3RoZXIgYW50aW1pY3JvYmlhbCAiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI4NzA1MDMmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkGuw8sazwCesuO9r9M752JjzAW04=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f169%2f2870503" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f169%2f2870503" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97104" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&amp;Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/2">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="78055106">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055106">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055108">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055108">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055113">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055113">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055125">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055125">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055146">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055146">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055148">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055148">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055156">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055156">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055158">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055158">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/2/159/2960055">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/2/179/2960054">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Complete genome sequence and expression profile of the commercial lytic enzyme producer <em>Lysobacter enzymogenes</em> M497-1
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideto Takami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Microbial Genome Research Group, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw055-cor1" data-open="dsw055-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: +81-45-778-5459. Fax: +81-45-778-5588. E-mail: <a href="mailto:takamih@jamstec.go.jp" target="_blank">takamih@jamstec.go.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideto+Takami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takami H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takami H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideto Takami</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi Toyoda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Comparative Genomics Laboratory, National Institute of Genetics, Mishima, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsushi+Toyoda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Toyoda A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Toyoda A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Atsushi Toyoda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ikuo Uchiyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Laboratory of Genome Informatics, National Institute for Basic Biology, National Institutes of Natural Sciences, Aichi, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ikuo+Uchiyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Uchiyama I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Uchiyama I&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ikuo Uchiyama</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takehiko Itoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div>Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, Meguro-ku, Tokyo, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takehiko+Itoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itoh T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Itoh T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takehiko Itoh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshihiro Takaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Microbial Genome Research Group, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshihiro+Takaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takaki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takaki Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshihiro Takaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wataru Arai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Microbial Genome Research Group, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wataru+Arai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Arai W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Arai W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wataru Arai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shinro Nishi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Microbial Genome Research Group, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shinro+Nishi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nishi S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nishi S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shinro Nishi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mikihiko Kawai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Microbial Genome Research Group, Japan Agency for Marine-Earth Science and Technology (JAMSTEC), Yokohama, Kanagawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mikihiko+Kawai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawai M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawai M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mikihiko Kawai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kazuo Shin-ya</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div>National Institute of Advanced Industrial Science and Technology, Koto-ku, Tokyo, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kazuo+Shin-ya">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shin-ya K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shin-ya K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kazuo Shin-ya</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Haruo Ikeda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div>Kitasato Institute for Life Sciences, Kitasato University, Sagamihara, Kanagawa, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Haruo+Ikeda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ikeda H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ikeda H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Haruo Ikeda</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw055-FM1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: Graduate School of Bioscience and Biotechnology, Tokyo Institute of Technology, Meguro-ku, Tokyo, Japan</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw055-FM2"><div class="fn"><div class="footnote-content">Edited by Dr. Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 169–177, <a href='https://doi.org/10.1093/dnares/dsw055'>https://doi.org/10.1093/dnares/dsw055</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">08 January 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">14 June 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">19 November 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2870503" href="/dnaresearch/article-pdf/24/2/169/13688910/dsw055.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Hideto Takami, Atsushi Toyoda, Ikuo Uchiyama, Takehiko Itoh, Yoshihiro Takaki, Wataru Arai, Shinro Nishi, Mikihiko Kawai, Kazuo Shin-ya, Haruo Ikeda;  Complete genome sequence and expression profile of the commercial lytic enzyme producer <em>Lysobacter enzymogenes</em> M497-1, <em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 169–177, <a href='https://doi.org/10.1093/dnares/dsw055'>https://doi.org/10.1093/dnares/dsw055</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2870503?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2870503?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2870503?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2870503?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2870503?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2870503?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2870503?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Complete%20genome%20sequence%20and%20expression%20profile%20of%20the%20commercial%20lytic%20enzyme%20producer%20%26lt%3Bem%26gt%3BLysobacter%20enzymogenes%26lt%3B%2Fem%26gt%3B%20M497-1&amp;publicationDate=2017-01-08&amp;volumeNum=24&amp;issueNum=2&amp;author=Takami%2C%20Hideto%3B%20Toyoda%2C%20Atsushi&amp;startPage=169&amp;endPage=177&amp;contentId=10.1093%2Fdnares%2Fdsw055&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f169%2f2870503" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=78055106 id="78055106" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Lysobacter enzymogenes</em> M497-1 is a producer of commercialized achromopeptidase and is expected to harbour genes encoding various other antimicrobial enzymes. Here, we present the complete sequence of the genome of M497-1 and the expression profiles of the genes for various antimicrobial enzymes. Of the 117 peptidase-encoding genes found in the 6.1-Mb genome of M497-1, 15 genes (aside from the gene encoding the achromopeptidase) were expressed at a level higher than that of the average ribosomal protein genes in the 24-h culture. Thus, the strain was found more valuable than hitherto considered. In addition, M497-1 harbours 98 genes involved in the biosynthesis of various natural products, 16 of which are M497-1-specific across 4 <em>Lysobacter</em> species. A gene cluster starting at LEN_2603 through LEN_2673 among the 98 genes closely resembled the lysobactin biosynthesis gene cluster of <em>Lysobacter</em> sp. ATCC 53042. It is likely that M497-1 may produce lysobactin or related antibacterial compounds. Furthermore, comparative genomic analysis of M497-1 and four other <em>Lysobacter</em> species revealed that their core genome structure comprises 3,737 orthologous groups. Our findings are expected to advance further biotechnological application of <em>Lysobacter</em> spp. as a promising source of natural bioactive compounds.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Lysobacter enzymogenes&quot;"><em>Lysobacter enzymogenes</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequence&quot;">genome sequence</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;lytic enzyme&quot;">lytic enzyme</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;RNA sequencing&quot;">RNA sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;metabolic potential&quot;">metabolic potential</a></div>                            <h2 scrollto-destination=78055108 id="78055108" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p><em>Lysobacter enzymogenes</em> M497-1 (formerly <em>Achromobacter lyticus</em>) was originally isolated from a Japanese soil sample and identified as <em>A</em><em>.</em><em>lyticus</em> within the class <em>Betaproteobacteria</em> by Isono <em>et al.</em><a reveal-id="dsw055-B1" data-open="dsw055-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> It was registered as a producer of the bacteriolytic enzyme lysyl endopeptidase (US patent no. 3,649,454) in 1972.<a reveal-id="dsw055-B1" data-open="dsw055-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> The achromopeptidase (lysyl endopeptidase) produced by this strain<a reveal-id="dsw055-B2 dsw055-B3 dsw055-B4 dsw055-B5 dsw055-B6" data-open="dsw055-B2 dsw055-B3 dsw055-B4 dsw055-B5 dsw055-B6" class="link link-ref link-reveal xref-bibr"><sup>2–6</sup></a> has broad bacteriolytic specificity and has been used for the lysis of anaerobic gram-positive rods, including various <em>Actinomycetales</em> and <em>Bifidobacterium</em> species,<a reveal-id="dsw055-B7" data-open="dsw055-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> and gram-positive cocci, such as <em>Micrococcus radiodurans</em>,<a reveal-id="dsw055-B8" data-open="dsw055-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a>, <em>Staphylococcus aureus</em>,<a reveal-id="dsw055-B9" data-open="dsw055-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> and <em>Peptococcus saccharolyticus.</em><a reveal-id="dsw055-B10" data-open="dsw055-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> The commercialized achromopeptidase is widely employed in metagenomic analyses of the human gut microbiome, which contains anaerobic cocci and rods insensitive to lysozyme, because it enables stable and highly efficient lysis (&gt;90%) of human intestinal microorganisms.<a reveal-id="dsw055-B11" data-open="dsw055-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> Other types of proteolytic enzymes produced by strain M497-1 include α- and β-lytic proteases.<a reveal-id="dsw055-B12" data-open="dsw055-B12" class="link link-ref link-reveal xref-bibr"><sup>12–14</sup></a></p><p><em>Lysobacter</em> species share some properties, including the ability to lyse prokaryotic and eukaryotic microbes, high genomic G + C content (65–70%), and gliding motility, with plant-associated myxobacteria, although <em>Lysobacter</em> species have never been observed to form fruiting bodies, which are typical for myxobacteria. Consequently, <em>Lysobacte</em>r species have been occasionally misidentified as myxobacteria in the older literature,<a reveal-id="dsw055-B15" data-open="dsw055-B15" class="link link-ref link-reveal xref-bibr"><sup>15–20</sup></a> but this taxonomic problem was resolved in 1978 by creating a new genus, <em>Lysobacter</em>, within the class <em>Gammaproteobacteria</em>.<a reveal-id="dsw055-B21" data-open="dsw055-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsw055-B22" data-open="dsw055-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> To clarify the taxonomic position of <em>A. lyticus</em> M497-1, which produces lytic enzymes similar to those of <em>Lysobacter</em> species,<a reveal-id="dsw055-B23 dsw055-B24 dsw055-B25" data-open="dsw055-B23 dsw055-B24 dsw055-B25" class="link link-ref link-reveal xref-bibr"><sup>23–25</sup></a> we conducted a sequence homology search based on the 16S rRNA gene amplified by PCR from M497-1 DNA remaining in the commercial crude enzyme preparation, and we found 99.9% identity with the 16S rRNA of <em>L. enzymogenes</em> (formerly <em>Myxobacter</em>) strain 495.<a reveal-id="dsw055-B15" data-open="dsw055-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a></p><p><em>L. enzymogenes</em> is a versatile bacterium producing various types of lytic enzymes specific not only for bacteria<a reveal-id="dsw055-B7 dsw055-B8 dsw055-B9 dsw055-B10" data-open="dsw055-B7 dsw055-B8 dsw055-B9 dsw055-B10" class="link link-ref link-reveal xref-bibr"><sup>7–10</sup></a><sup>,</sup><a reveal-id="dsw055-B12 dsw055-B13 dsw055-B14" data-open="dsw055-B12 dsw055-B13 dsw055-B14" class="link link-ref link-reveal xref-bibr"><sup>12–14</sup></a> but also for eukaryotic cells. Consequently, several <em>L</em>. <em>enzymogenes</em> strains have been characterized as biological control agents against plant diseases caused by fungi such as <em>Bipolaris sorokiniana</em><a reveal-id="dsw055-B19" data-open="dsw055-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsw055-B20" data-open="dsw055-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> and <em>Uromyces appendiculatus</em>,<a reveal-id="dsw055-B26" data-open="dsw055-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> filamentous protists such as <em>Pythium aphanidermatum</em> (class <em>Oomycota</em>),<a reveal-id="dsw055-B27" data-open="dsw055-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and roundworms<a reveal-id="dsw055-B28" data-open="dsw055-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> such as the sugar-beet cyst nematode <em>Heterodera schachtii</em> and the root-knot nematode <em>Meloidogyne javanica. L. enzymogenes</em>-derived chitinases can inhibit conidial germination of plant pathogens and degrade egg shells and hatched juveniles of nematodes<a reveal-id="dsw055-B20" data-open="dsw055-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a><sup>,</sup><a reveal-id="dsw055-B28" data-open="dsw055-B28" class="link link-ref link-reveal xref-bibr"><sup>28–31</sup></a>, while β-1,3-glucanases from the bacterium can lyse the cell walls of pathogenic fungi.<a reveal-id="dsw055-B32" data-open="dsw055-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsw055-B33" data-open="dsw055-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> Furthermore, <em>Lysobacter</em> species are known to synthesize other antimicrobial agents such as tripropeptins<a reveal-id="dsw055-B34" data-open="dsw055-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a><sup>,</sup><a reveal-id="dsw055-B35" data-open="dsw055-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and lycosin E,<a reveal-id="dsw055-B36" data-open="dsw055-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> which are active against methicillin- and vancomycin-resistant <em>S. aureus</em>, respectively.</p><p>Over 40 species within the genus <em>Lysobacter</em> have been registered in the NCBI taxonomy database, and recently, the complete genomes of 4 species (<em>L. antibioticus, L. capsici, L. enzymogenes</em>, and <em>L. gummosus</em>) have been sequenced.<a reveal-id="dsw055-B37" data-open="dsw055-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> The genome sequence data are very useful to elucidate the intragenus as well as intraspecies genomic diversity. Here, we compared the genomes of M497-1 and 4 other <em>Lysobacter</em> species to determine the core genomic structure of the genus <em>Lysobacter</em>, and explored the genetic, metabolic, and physiological potential of M497-1, which could be exploited in biotechnological applications.</p>                            <h2 scrollto-destination=78055113 id="78055113" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=78055114 id="78055114" class="section-title" >2.1. Strain and cultivation</h3>
<p><em>L. enzymogenes</em> strain M497-1 was kindly supplied by Wako Pure Chemical Industries (Osaka, Japan), Ltd. This strain was deposited to the American Type Culture Collection as <em>Lysobacter</em> sp. (ATCC 21456). M497-1 was aerobically grown at 30 °C in 500-ml flasks containing 100 ml of medium for 24 or 48 h. For DNA analysis, the bacteria were grown in CY medium containing 0.5% casitone and 0.1% yeast extract, and for RNA expression analysis, they were cultured in medium containing 1.5% glycerol, 0.3% NaCl, 0.5%, <span class="small-caps">l</span>-glutamate monohydrate (pH 7.4), and 1.5% cotton seed meal (Traders Protein, Southern Cotton Oil Company, Memphis, TN, USA).</p>                            <h3 scrollto-destination=78055116 id="78055116" class="section-title" >2.2. DNA and RNA purification</h3>
<p>Genomic DNA was extracted and purified using the NucleoSpin Tissue kit (Macherey-Nagel GmbH &amp; Co. KG, Düren, Germany) according to the manufacturer’s protocol. RNA was extracted and purified from 24-h and 48-h M497-1 cultures using the RNeasy Protect Bacteria Mini kit and QIAzol Lysis Reagent (Qiagen, Tokyo, Japan) according to the manufacturer’s protocol. DNase I-treated RNA was re-purified using the RNeasy Mini Cleanup Kit (Qiagen) and rRNA was removed using the Ribo-Zero rRNA Removal Kit (Gram-Negative Bacteria) (Epicentre, Madison, WI, USA).</p>                            <h3 scrollto-destination=78055118 id="78055118" class="section-title" >2.3. Sequencing, assembly, and data analyses</h3>
<p>For whole-genome shotgun sequencing, we combined the Roche GS FLX (Roche Diagnostics K.K., Tokyo, Japan) and Sanger sequencing technologies, which produced ∼53- and 23-fold genome coverage, respectively. First, GS FLX reads with an average length of 231 bp were assembled using the Newbler programme version 2.3, and 385 contigs with an N50 length of 33,738 bp were generated. Next, we performed end-sequencing of 77,952 and 1,920 clones from 2–3 to 35–40 kb (fosmid) insert-size libraries, respectively, using an ABI 3730xl sequencer (Thermo Fisher Scientific K.K., Yokohama, Japan), and hybrid assembly was carried out by the KB Basecaller and Phrap/Consed programmes.<a reveal-id="dsw055-B38" data-open="dsw055-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> The remaining gaps between contigs were filled in using PCR and Sanger sequencing of amplified products. In addition, low-quality genomic regions were sequenced to increase the genome coverage.</p><p>RNA-seq libraries were prepared from 24- to 48-h cultures according to the standard Illumina protocol and cDNA libraries were checked for quality and quantity using the DNA-100 kit (Agilent Technologies Japan, Ltd., Tokyo, Japan) and a 2100 Bioanalyzer. Each library was sequenced with the Illumina Sequencing Kit v2 on one lane of a MiSeq desktop sequencer (Illumina K.K., Tokyo, Japan) to obtain 150-bp average paired-end reads. The value of Reads Per Kilobase per Million mapped reads (RPKM) was calculated according to the standard method. The RPKM ratio was calculated by dividing the RPKM of each gene by the mean RPKM of all ribosomal proteins, and was used to determine relative gene expression levels. Promoter-like sequences were detected with GENETYX-MAC version 18 (Tokyo, Japan) and PePPER<a reveal-id="dsw055-B39" data-open="dsw055-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> (<a class="link link-uri" href="http://pepper.molgenrug.nl/index.php/prokaryote-promoters" target="_blank">http://pepper.molgenrug.nl/index.php/prokaryote-promoters</a>). Peptidase genes were identified by BLAST searches and from the MEROPS database of proteolytic enzymes; MEROPS identifiers were assigned on the basis of a phylogenetic tree: an identifier was assigned to all sequences derived from the same node as a holotype.<a reveal-id="dsw055-B40" data-open="dsw055-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a></p>                            <h3 scrollto-destination=78055121 id="78055121" class="section-title" >2.4. Orthologous analysis and estimation of gene clusters for biosynthesis of natural product</h3>
<p>Orthologous groups (OGs) for M497-1 and 4 other <em>Lysobacter</em> species<a reveal-id="dsw055-B37" data-open="dsw055-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> were constructed using the rapid classification programme DomClust<a reveal-id="dsw055-B41" data-open="dsw055-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> on the Microbial Genome Database (MBGD) server.<a reveal-id="dsw055-B42" data-open="dsw055-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> A core genome is defined as a set of genes (OGs) syntenically conserved in at least half of the compared strains. In our study, a set of genes in the syntenic regions shared by at least three strains was defined as the core genome for the five analysed <em>Lysobacter</em> strains. A set of syntenic regions and the consensus order of OGs in these regions designated ‘core genome structure’ were created using the CoreAligner programme<a reveal-id="dsw055-B43" data-open="dsw055-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> based on conserved linkages between orthologous genes in each chromosome. Among the OGs in the core genome, those present in all the genomes were called ‘universal core genome’. Non-ribosomal protein synthase (NRPS)- and polyketide synthase (PKS)-encoding genes were manually annotated using web-based prediction programmes.<a reveal-id="dsw055-B44" data-open="dsw055-B44" class="link link-ref link-reveal xref-bibr"><sup>44–47</sup></a> Gene clusters related to biosynthesis of bioactive natural product were estimated based on gene direction and continuity in the M497-1 genome and compared with previously identified gene clusters.</p>                            <h3 scrollto-destination=78055123 id="78055123" class="section-title" >2.5. Evaluation of the metabolic and physiological potential</h3>
<p>The metabolic and physiological potential of <em>L. enzymogenes</em> M497-1 was investigated using <span class="underline">M</span>etabolic <span class="underline">a</span>nd <span class="underline">P</span>hysiological Potentia<span class="underline">l E</span>valuator (MAPLE) ver. 2.1.<a reveal-id="dsw055-B48" data-open="dsw055-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> Genes were mapped to functional modules defined by the Kyoto Encyclopedia of Genes and Genomes (KEGG) (pathways: 255, complexes: 284, functional sets: 149, signatures: 40) and the module completion ratio (MCR) was calculated according to a Boolean algebra-like equation described previously.<a reveal-id="dsw055-B49" data-open="dsw055-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> The MCR patterns of M497-1 were compared with those of other <em>Lysobacter</em> species.</p>                            <h2 scrollto-destination=78055125 id="78055125" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=78055126 id="78055126" class="section-title" >3.1. General features of the <em>L. enzymogenes</em> M497-1 genome</h3>
<div class="block-child-p">The genome of <em>L. enzymogenes</em> M497-1 consists of a single circular chromosome (6,096,022 bp) with a mean G + C content of 69.4% (<a reveal-id="dsw055-F1" data-open="dsw055-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). We identified 4,891 protein-coding sequences (CDSs), 55 transfer RNA genes, and 6 rRNA genes comprising 2 rRNA operons (Table S1). Although there was no significant difference in genome size between <em>L. enzymogenes</em> strains M497-1 and C3, the former has 638 CDSs fewer than the latter. M497-1 and C3 shared 4,043 OGs, while they had 488 and 1,022 unique genes, respectively. Among 485 M497-1 genes unique to the 5 <em>Lysobacter</em> spp. (<em>L. enzymogenes</em> M497-1 and C3, <em>L. antibioticus</em> 76, <em>L. capsici</em> 55, and <em>L. gummosus</em> 3.2.11), 8 encoded peptidases and 16 encoded NRPSs and PKSs, while the function of 338 unique genes (70%) is not known. Among the functionally unknown genes, 11 showed an RPKM ratio higher than 2 in 24-h culture; LEN_1053 had the highest RPKM ratio (186) among all genes identified in the M497-1 genome. The original RPKM values and RPKM ratios for all genes are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&amp;Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. <div data-id="dsw055-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/m_dsw055f1.png?Expires=2147483647&amp;Signature=kq7urNTZVs4E4H0axsbrI8RUSr4VZWTwBN0IFKaQ~H1mGKnDRW1Sf4td0kkxFcuWSEZMwpUwq9B4sENFT5d0wO34yi9M-WAW~9eBbRlen3ifZV~CRw7Htpd2l4tNwFiSaIbkAc55Bw-2EIYq6PeTogZxvRROEpCuJAKlqHexQPBR1vsLNESgi08fr9IIWe0OlCVKEHc4aoZy6pRewfK3hg0XbisbRA2Jeiufb4EiYbeaFW0eInSERmgvsF9DbHe8KLQfUvMxSraoBDIiqIhessi0DsdMX6Ktr2LTHcdwBANe3cw2hrU1hv3KFMyRAjs7W~M4RDm5LPBjt7nHmpSLoQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic structure of Lysobacter sp. (A) Genome map of L. enzymogenes M497-1. The distribution of CDSs is indicated by coloured boxes according to the KEGG functional modules and transcription direction; the first two outer circles represent sense and antisense strands, respectively, and the third and fourth circles indicate the distribution of tRNAs and rRNAs, respectively. The innermost circle (in green) represents variations in the G + C content (%) of the genome. Red, energy metabolism (pathway); pink, carbohydrate and lipid metabolism (pathway); orange, nucleotide and amino acid metabolism (pathway); magenta, secondary metabolism (pathway); blue, energy metabolism (complex); magenta and cyan, genetic information processing; soft blue, environmental information processing (complex); yellow green, metabolism (functional set); light green, environmental information processing (functional set); dark green, cellular processes (functional set); yellow, gene set (signature); light gray, hypothetical conserved protein; dark gray, hypothetical protein; tan, KEGG Orthology not assigned to modules. (B) Genes in the universal core genome structure for five Lysobacter species constructed on the basis of the consensus arrangement of conserved orthologs. Gene location in each chromosome is indicated; ortholog groups are denoted by coloured lines; only universally conserved ortholog groups with one-to-one correspondence are shown. To visualize chromosomal rearrangement, a colour gradient scale (red to yellow to green) is used according to the location on the M497-1 chromosome. Origins of replication are at the center." path-from-xml="dsw055f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055127/dsw055f1.tif" path-from-xml="dsw055f1.tif" target="_blank">View large</a><a section="78055127" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055f1.png?Expires=2147483647&amp;Signature=RfYzeTKMbg9qc93eo9ul401AIa5USO63jGOCG9aEZJi8VXhDzy7SMCGogmArRdccer0WDIEeKK-CDsvlPh7tN~CRI5m-Tsdvdcsw2ydoiEQyi4IxiBYYVpAWr~YN~vPOKRq3ehxILjbxbCjS1thJW7-tOFNs3T8DtrX0ABsEZ6VaFarSN7fNZLDYe6VxF6jfbgZqg7WbUnFiMzFOcM24w61lMIJrk2ppQkgtLbT15FRSSRxsSg5ZfC2l7tZpP250pmsw3glZ~7qINkjumkcSXZp88pzxOV3~UpWv5x0QjQgO3iEDWDS7CJMQ4KPpB-VL~ENtWwbJ-~yvnbMBRMB5vQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055127&amp;ar=2870503&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw055f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic structure of <em>Lysobacter</em> sp. (<strong>A)</strong> Genome map of <em>L. enzymogenes</em> M497-1. The distribution of CDSs is indicated by coloured boxes according to the KEGG functional modules and transcription direction; the first two outer circles represent sense and antisense strands, respectively, and the third and fourth circles indicate the distribution of tRNAs and rRNAs, respectively. The innermost circle (in green) represents variations in the G + C content (%) of the genome. Red, energy metabolism (pathway); pink, carbohydrate and lipid metabolism (pathway); orange, nucleotide and amino acid metabolism (pathway); magenta, secondary metabolism (pathway); blue, energy metabolism (complex); magenta and cyan, genetic information processing; soft blue, environmental information processing (complex); yellow green, metabolism (functional set); light green, environmental information processing (functional set); dark green, cellular processes (functional set); yellow, gene set (signature); light gray, hypothetical conserved protein; dark gray, hypothetical protein; tan, KEGG Orthology not assigned to modules. (<strong>B)</strong> Genes in the universal core genome structure for five <em>Lysobacter</em> species constructed on the basis of the consensus arrangement of conserved orthologs. Gene location in each chromosome is indicated; ortholog groups are denoted by coloured lines; only universally conserved ortholog groups with one-to-one correspondence are shown. To visualize chromosomal rearrangement, a colour gradient scale (red to yellow to green) is used according to the location on the M497-1 chromosome. Origins of replication are at the center.</p></div></div><div content-id="dsw055-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/m_dsw055f1.png?Expires=2147483647&amp;Signature=kq7urNTZVs4E4H0axsbrI8RUSr4VZWTwBN0IFKaQ~H1mGKnDRW1Sf4td0kkxFcuWSEZMwpUwq9B4sENFT5d0wO34yi9M-WAW~9eBbRlen3ifZV~CRw7Htpd2l4tNwFiSaIbkAc55Bw-2EIYq6PeTogZxvRROEpCuJAKlqHexQPBR1vsLNESgi08fr9IIWe0OlCVKEHc4aoZy6pRewfK3hg0XbisbRA2Jeiufb4EiYbeaFW0eInSERmgvsF9DbHe8KLQfUvMxSraoBDIiqIhessi0DsdMX6Ktr2LTHcdwBANe3cw2hrU1hv3KFMyRAjs7W~M4RDm5LPBjt7nHmpSLoQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genomic structure of Lysobacter sp. (A) Genome map of L. enzymogenes M497-1. The distribution of CDSs is indicated by coloured boxes according to the KEGG functional modules and transcription direction; the first two outer circles represent sense and antisense strands, respectively, and the third and fourth circles indicate the distribution of tRNAs and rRNAs, respectively. The innermost circle (in green) represents variations in the G + C content (%) of the genome. Red, energy metabolism (pathway); pink, carbohydrate and lipid metabolism (pathway); orange, nucleotide and amino acid metabolism (pathway); magenta, secondary metabolism (pathway); blue, energy metabolism (complex); magenta and cyan, genetic information processing; soft blue, environmental information processing (complex); yellow green, metabolism (functional set); light green, environmental information processing (functional set); dark green, cellular processes (functional set); yellow, gene set (signature); light gray, hypothetical conserved protein; dark gray, hypothetical protein; tan, KEGG Orthology not assigned to modules. (B) Genes in the universal core genome structure for five Lysobacter species constructed on the basis of the consensus arrangement of conserved orthologs. Gene location in each chromosome is indicated; ortholog groups are denoted by coloured lines; only universally conserved ortholog groups with one-to-one correspondence are shown. To visualize chromosomal rearrangement, a colour gradient scale (red to yellow to green) is used according to the location on the M497-1 chromosome. Origins of replication are at the center." path-from-xml="dsw055f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055127/dsw055f1.tif" path-from-xml="dsw055f1.tif" target="_blank">View large</a><a section="78055127" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055f1.png?Expires=2147483647&amp;Signature=RfYzeTKMbg9qc93eo9ul401AIa5USO63jGOCG9aEZJi8VXhDzy7SMCGogmArRdccer0WDIEeKK-CDsvlPh7tN~CRI5m-Tsdvdcsw2ydoiEQyi4IxiBYYVpAWr~YN~vPOKRq3ehxILjbxbCjS1thJW7-tOFNs3T8DtrX0ABsEZ6VaFarSN7fNZLDYe6VxF6jfbgZqg7WbUnFiMzFOcM24w61lMIJrk2ppQkgtLbT15FRSSRxsSg5ZfC2l7tZpP250pmsw3glZ~7qINkjumkcSXZp88pzxOV3~UpWv5x0QjQgO3iEDWDS7CJMQ4KPpB-VL~ENtWwbJ-~yvnbMBRMB5vQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055127&amp;ar=2870503&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw055f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genomic structure of <em>Lysobacter</em> sp. (<strong>A)</strong> Genome map of <em>L. enzymogenes</em> M497-1. The distribution of CDSs is indicated by coloured boxes according to the KEGG functional modules and transcription direction; the first two outer circles represent sense and antisense strands, respectively, and the third and fourth circles indicate the distribution of tRNAs and rRNAs, respectively. The innermost circle (in green) represents variations in the G + C content (%) of the genome. Red, energy metabolism (pathway); pink, carbohydrate and lipid metabolism (pathway); orange, nucleotide and amino acid metabolism (pathway); magenta, secondary metabolism (pathway); blue, energy metabolism (complex); magenta and cyan, genetic information processing; soft blue, environmental information processing (complex); yellow green, metabolism (functional set); light green, environmental information processing (functional set); dark green, cellular processes (functional set); yellow, gene set (signature); light gray, hypothetical conserved protein; dark gray, hypothetical protein; tan, KEGG Orthology not assigned to modules. (<strong>B)</strong> Genes in the universal core genome structure for five <em>Lysobacter</em> species constructed on the basis of the consensus arrangement of conserved orthologs. Gene location in each chromosome is indicated; ortholog groups are denoted by coloured lines; only universally conserved ortholog groups with one-to-one correspondence are shown. To visualize chromosomal rearrangement, a colour gradient scale (red to yellow to green) is used according to the location on the M497-1 chromosome. Origins of replication are at the center.</p></div></div></div><p>The core genome structure of the five <em>Lysobacter</em> strains was defined using CoreAligner<a reveal-id="dsw055-B43" data-open="dsw055-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a>; it contains a set of well-conserved syntenic OGs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Out of 4,891 CDSs in the M497-1 genome, 3,593 were included in the core genome comprising 3,737 OGs; among them, 3,081 were shared by all 5 strains and thus, represented universal core gene clusters. In total, 3,046 OGs show one-to-one correspondence (<a reveal-id="dsw055-F1" data-open="dsw055-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). Types III and IV secretion pathways are highly conserved in the M497-1 genome as well as in those of other <em>Lysobacter</em> species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>).<a reveal-id="dsw055-B37" data-open="dsw055-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> All genes of <em>L. enzymogenes</em> M497-1 related to type III secretion (LEN_4800, LEN_4801, LEN_4803, and LEN_4806–4815) were included in the universal core gene cluster (Table S2). However, three genes related to the type IV secretion pathway (LEN_2216–2218) were not included in the universal core as they were not identified as orthologs in the <em>L. antibioticus</em> 76 genome by the MBGD server, although the gene organization of the adjacent region was well conserved. The RPKM ratios of genes related to type IV secretion in the M497 genome varied; 6 out of 10 genes (LEN_2207, LEN_2209, LEN_2211, and LEN_2213–2215) demonstrated almost equal or higher expression than the average level for ribosomal proteins in 24-h culture. In particular, LEN_2213 and LEN_2214 showed RPKM ratios of ∼9–10; however, the expression level dramatically decreased in 48-h culture, indicating that type IV secretion is active in the exponential growth phase. In contrast to the type IV secretion genes, most genes related to the type III secretion pathway showed 5- to 10-fold lower expression levels than ribosomal proteins.</p><p>Iron uptake is vital for almost all bacteria because iron is used as a cofactor in key metabolic processes, including deoxyribonucleotide synthesis, oxidative phosphorylation, and electron transport. To survive under iron-limiting conditions, bacteria have developed sophisticated iron acquisition strategies based on the secretion of iron-chelating siderophores.<a reveal-id="dsw055-B50" data-open="dsw055-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> TonB-dependent transporters are bacterial outer membrane receptor proteins that bind and translocate siderophores as well as vitamin B<sub>12</sub>, nickel complexes, and carbohydrates.<a reveal-id="dsw055-B51" data-open="dsw055-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> We found that the <em>Lysobacter</em> strains possess a remarkable number of genes encoding the TonB-dependent family of proteins, of which 52 and 41 were included in the core and universal core gene clusters, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Considering that <em>Lysobacter</em> spp. are inhabitants of soil and water, which are characterized by low bioavailability of iron, the high number of TonB-dependent proteins should be helpful for iron acquisition. In fact, seven genes included in the core OGs (LEN_0009, LEN_0351, LEN_3594, LEN_3807, LEN_4109, LEN_4624, and LEN_4786) in the M497-1 genome showed an expression level equal to or higher than that of ribosomal proteins in 24-h culture; among them, the highest RPKM ratio (more than 10) was observed for LEN_4109.</p>                            <h3 scrollto-destination=78055130 id="78055130" class="section-title" >3.2. Bacteriolytic enzymes</h3>
<div class="block-child-p">One hundred seventeen genes encoding peptidases, including the commercialized lysyl endopeptidase (LEN_1308), were identified in the M497-1 genome. These peptidase genes were classified into 56 families according to the MEROPS peptidase database.<a reveal-id="dsw055-B40" data-open="dsw055-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> The largest family, S08A, comprised 21 peptidases, and the S01A family contained LEN_1308 together with 2 other peptidases, LEN_1307 and LEN_1765 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&amp;Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Among all peptidase genes, 80 were included in the universal core gene clusters (<a reveal-id="dsw055-F2" data-open="dsw055-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&amp;Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>) and 16 showed an expression level equal to or higher than the average level of ribosomal proteins in 24-h culture (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&amp;Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>); LEN_4116 (β-lytic protease) of the M23A family showed an RPKM ratio of ∼30. However, the expression level of these 16 genes dramatically decreased after 48 h (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&amp;Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). In gram-positive bacilli, among which are various industrial enzyme producers, protease production usually starts from the early stationary phase of cultivation (20–24 h), and the productivity is maintained even at the late stationary phase (48 h).<a reveal-id="dsw055-B52" data-open="dsw055-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> In contrast to bacilli, <em>L. enzymogenes</em> M497-1 easily lyse after reaching the stationary phase; cell lysis was actually observed after 24 h of cultivation (data not shown), likely due to autolysis by autoproduced peptidases. <div data-id="dsw055-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/m_dsw055f2.png?Expires=2147483647&amp;Signature=VanwHlkuRg6whn3W20-CUmj1DKSJ5NEKHN5kLSie0iOee~yLzblQupZ7Fb0JEyBdxokxYe~qXOYdx9Rd3tgsKwM-ekf3jaHz9YpI8EYiAPGDXMBiMYIZMJN0CnrRa2Fw-ibrLV9XFs3Ld~pTF3UshuvDYW0kWTp~Yh73cKkxWxMWOeeNTEFDqW16oRweS07YCnenxQKLocqyJBrd~ijP7~5lg52xRIygvsgU-ovb2DcGxh5EBhbBKI-HmHhJUtVPSae5m65vqivEDxNj8uVK~TCdMKiZXPSWZ2FrmTbbDy2G~TTqNpe~bf~3B6lrxYyDi8QZTAyOdK9N45EkBgYVSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution and expression of peptidase genes in Lysobacter strains. (A) Distribution of peptidase genes in the genomes of five Lysobacter strains. The backbone of each genome is composed of 3,081 universal core OGs, which are represented as gray lines. Red and blue lines represent peptidase genes included and not included in the universal core genome structure, respectively. The triangle shows the position of peptidase islands in the M497-1 genome, detailed gene organization of which is shown in B. (B) Gene organization in the peptidase island of the M497-1 genome. Characters in parentheses indicate family names assigned by MEROPS40, and P indicates the position of promoter-like sequences. (C) Expression patterns (RPKM ratios) of the genes in the peptidase island in 24-h (black) and 48-h (grey) M497-1 cultures (see also Supplementary Table S2)." path-from-xml="dsw055f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055131/dsw055f2.tif" path-from-xml="dsw055f2.tif" target="_blank">View large</a><a section="78055131" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055f2.png?Expires=2147483647&amp;Signature=1v8EEp1YPNKi4XpMNmVC0qsBZ4462UPilqxJ3dGdML~iCvBhFXvi-54wfN2hlJGH3zj3QTVxcEy-3uKMqzhmjLJES8-4A-A81UMXYrOqQty4N1VcjYMaANq1i5iRHr9IwnCkAZuFl5y23X1nEP0RmBvzhVriU9ZhUaB45gagKoLvhk~tK4f7MR7aBS7aAVM6GrI1Vdj-Gr3~UHxV9T5J3f11aXDu-I6m00zoc9fMnGdKnaMie2TjdUOYAw~E3olCtYShMZ1y2M~ZrKcO-kj02UqxyodDUOf8poUY-P7UBfThequYlvzirfUDf2fwnS0DwWozlKYs21xdl-2ppzHL8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055131&amp;ar=2870503&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw055f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution and expression of peptidase genes in <em>Lysobacter</em> strains. (<strong>A)</strong> Distribution of peptidase genes in the genomes of five <em>Lysobacter</em> strains. The backbone of each genome is composed of 3,081 universal core OGs, which are represented as gray lines. Red and blue lines represent peptidase genes included and not included in the universal core genome structure, respectively. The triangle shows the position of peptidase islands in the M497-1 genome, detailed gene organization of which is shown in B. <strong>(B)</strong> Gene organization in the peptidase island of the M497-1 genome. Characters in parentheses indicate family names assigned by MEROPS<a reveal-id="dsw055-B40" data-open="dsw055-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a>, and P indicates the position of promoter-like sequences. <strong>(C)</strong> Expression patterns (RPKM ratios) of the genes in the peptidase island in 24-h (black) and 48-h (grey) M497-1 cultures (see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&amp;Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>).</p></div></div><div content-id="dsw055-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/m_dsw055f2.png?Expires=2147483647&amp;Signature=VanwHlkuRg6whn3W20-CUmj1DKSJ5NEKHN5kLSie0iOee~yLzblQupZ7Fb0JEyBdxokxYe~qXOYdx9Rd3tgsKwM-ekf3jaHz9YpI8EYiAPGDXMBiMYIZMJN0CnrRa2Fw-ibrLV9XFs3Ld~pTF3UshuvDYW0kWTp~Yh73cKkxWxMWOeeNTEFDqW16oRweS07YCnenxQKLocqyJBrd~ijP7~5lg52xRIygvsgU-ovb2DcGxh5EBhbBKI-HmHhJUtVPSae5m65vqivEDxNj8uVK~TCdMKiZXPSWZ2FrmTbbDy2G~TTqNpe~bf~3B6lrxYyDi8QZTAyOdK9N45EkBgYVSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution and expression of peptidase genes in Lysobacter strains. (A) Distribution of peptidase genes in the genomes of five Lysobacter strains. The backbone of each genome is composed of 3,081 universal core OGs, which are represented as gray lines. Red and blue lines represent peptidase genes included and not included in the universal core genome structure, respectively. The triangle shows the position of peptidase islands in the M497-1 genome, detailed gene organization of which is shown in B. (B) Gene organization in the peptidase island of the M497-1 genome. Characters in parentheses indicate family names assigned by MEROPS40, and P indicates the position of promoter-like sequences. (C) Expression patterns (RPKM ratios) of the genes in the peptidase island in 24-h (black) and 48-h (grey) M497-1 cultures (see also Supplementary Table S2)." path-from-xml="dsw055f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055131/dsw055f2.tif" path-from-xml="dsw055f2.tif" target="_blank">View large</a><a section="78055131" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055f2.png?Expires=2147483647&amp;Signature=1v8EEp1YPNKi4XpMNmVC0qsBZ4462UPilqxJ3dGdML~iCvBhFXvi-54wfN2hlJGH3zj3QTVxcEy-3uKMqzhmjLJES8-4A-A81UMXYrOqQty4N1VcjYMaANq1i5iRHr9IwnCkAZuFl5y23X1nEP0RmBvzhVriU9ZhUaB45gagKoLvhk~tK4f7MR7aBS7aAVM6GrI1Vdj-Gr3~UHxV9T5J3f11aXDu-I6m00zoc9fMnGdKnaMie2TjdUOYAw~E3olCtYShMZ1y2M~ZrKcO-kj02UqxyodDUOf8poUY-P7UBfThequYlvzirfUDf2fwnS0DwWozlKYs21xdl-2ppzHL8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055131&amp;ar=2870503&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw055f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution and expression of peptidase genes in <em>Lysobacter</em> strains. (<strong>A)</strong> Distribution of peptidase genes in the genomes of five <em>Lysobacter</em> strains. The backbone of each genome is composed of 3,081 universal core OGs, which are represented as gray lines. Red and blue lines represent peptidase genes included and not included in the universal core genome structure, respectively. The triangle shows the position of peptidase islands in the M497-1 genome, detailed gene organization of which is shown in B. <strong>(B)</strong> Gene organization in the peptidase island of the M497-1 genome. Characters in parentheses indicate family names assigned by MEROPS<a reveal-id="dsw055-B40" data-open="dsw055-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a>, and P indicates the position of promoter-like sequences. <strong>(C)</strong> Expression patterns (RPKM ratios) of the genes in the peptidase island in 24-h (black) and 48-h (grey) M497-1 cultures (see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&amp;Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>).</p></div></div></div><p>LEN_1308, located in a peptidase island comprising nine continuous peptidase genes, showed RPKM ratios of 14 and 1.6 in the 24- and 48-h cultures, respectively, and the expression of the neighbouring LEN_1309 and LEN_1310 genes also exceeded the average level of ribosomal proteins (<a reveal-id="dsw055-F2" data-open="dsw055-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>); moreover, potential σ<sup>54</sup>-dependent promoter sequences were identified upstream of these genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Among the other genes in the peptidase island (LEN_1301–1306), another σ<sup>54</sup>-dependent promoter-like region was detected for LEN_1301, suggesting polycistronic expression of the LEN_1301–1306 region; however, expression was very low compared with the average level of ribosomal proteins (<a reveal-id="dsw055-F2" data-open="dsw055-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). On the other hand, expression of 4 peptidase genes (LEN_1590, LEN_1765, LEN_1799, and LEN_3019) was detected, while no putative promoter-like sequences were identified (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Our manual search for other types of promoter-like regions was not successful either, probably because promoters depending on different σ factors are likely to have significant sequence variability.</p>                            <h3 scrollto-destination=78055133 id="78055133" class="section-title" >3.3. Lytic enzymes having activity against eukaryotic cells</h3>
<div class="block-child-p"><em>L</em>. <em>enzymogenes</em> is known to protect plant from diseases caused by various fungi, filamentous protists, and nematodes. Chitinase is a hydrolytic enzyme cleaving glycosidic bonds in chitin, a component of fungal cell walls and of exoskeleton of lower animals, including nematodes. <em>L. enzymogenes</em> M497-1 has 3 chitinase-encoding genes belonging to glycoside hydrolase families 18 and 19; all of these genes were included in the universal core OGs (<a reveal-id="dsw055-F3" data-open="dsw055-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). Among them, LEN_2961 demonstrated significant expression in 24-h culture, as evidenced by its RPKM ratio of 3.9, while the other two genes (LEN_3788 and LEN_4456) were expressed at levels below the average of ribosomal proteins; upstream of each gene, putative σ<sup>54</sup>-dependent promoter-like sequences were detected (<a reveal-id="dsw055-F3" data-open="dsw055-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). <div data-id="dsw055-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/m_dsw055f3.png?Expires=2147483647&Signature=moEo289QIQw8xrsZZfKpj7mivNnDkbLiK0WX5qpff9mncLW4dgQYLyvNOpJvSthone14dcgzNIioqXx1L4uFXXeva8OJSB7Oc2Els-lvycQDlR3ZjILcFB9dHwlQCgfBwgVwbPRb1PS~K95gYnN8on754ajJ-WMPOsqioRfa0tU1V5Xw-Xb2SYjVJcz-HorG03TsAHZ3oS3-D0u1iRdDGFDuTAG--na1N-TpamqJ5gVgZHOXGQku7U-2pmuhbgB-qgkJuQJyCoHYmLdLgR13hGkvei-7msUaMm-Q-J2zR9XRsqR84VAoOKMba5idOQzQQBfwsPpw7qtl8vO-CjUklg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Lysobacter genes encoding lytic enzymes active against eukaryotic cells. (A) Distribution of the genes encoding endoglucanase/β-1,3-glucanase and chitinase (genome backbones are the same as in Fig. 2); red and blue lines indicate the lytic enzyme genes included and not included in the universal core genome structure, respectively. (B) Orthologous relationships of lytic enzymes among five Lysobacter strains and expression patterns of the orthologs identified in M497-1. Red and blue boxes correspond to red and blue lines, respectively, in A; green and yellow bars show RPKM ratios in 24- and 48-h cultures, respectively. Promoter-like sequences detected only by PePPER39 are shown in blue, and those identified by both PePPER and GENETYX-MAC are shown in black. n.d., not detected." path-from-xml="dsw055f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055134/dsw055f3.tif" path-from-xml="dsw055f3.tif" target="_blank">View large</a><a section="78055134" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055f3.png?Expires=2147483647&Signature=c9xyIZymP2MErYmfBrd1V02yfyFk8yDBq61yE7No0jSHVDOlelanVjLRAXPou0abgmywmUvBfCqnJ5fNpoUTwKcSmGVpcyd0LQIRqVhTQlV7LWczoZOnqjZ8aP0xWi2OglxDP0EKTCyU6rmsK~8j2yDMwtBvl-I6EZ6WoUD4SngiiP885zXdKmGDPyiyw5ncDgmXcaBqy2ntw1Bpn2JX9jcYyAYPIt7v54ytGwi9WcY4bWZCCSCc5nmb34sjp5TN3GDyVfPlK7eW7HCPJdMYYNm-CHr0ZOJyppt-wekYzKs7idoIV7ht-1YWZA8f4ZfnlGfX-P80CC4ON5z0He9ghA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055134&ar=2870503&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw055f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Lysobacter</em> genes encoding lytic enzymes active against eukaryotic cells. <strong>(A)</strong> Distribution of the genes encoding endoglucanase/β-1,3-glucanase and chitinase (genome backbones are the same as in <a reveal-id="dsw055-F2" data-open="dsw055-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>); red and blue lines indicate the lytic enzyme genes included and not included in the universal core genome structure, respectively. <strong>(B)</strong> Orthologous relationships of lytic enzymes among five <em>Lysobacter</em> strains and expression patterns of the orthologs identified in M497-1. Red and blue boxes correspond to red and blue lines, respectively, in A; green and yellow bars show RPKM ratios in 24- and 48-h cultures, respectively. Promoter-like sequences detected only by PePPER<sup>39</sup> are shown in blue, and those identified by both PePPER and GENETYX-MAC are shown in black. n.d., not detected.</p></div></div><div content-id="dsw055-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/m_dsw055f3.png?Expires=2147483647&Signature=moEo289QIQw8xrsZZfKpj7mivNnDkbLiK0WX5qpff9mncLW4dgQYLyvNOpJvSthone14dcgzNIioqXx1L4uFXXeva8OJSB7Oc2Els-lvycQDlR3ZjILcFB9dHwlQCgfBwgVwbPRb1PS~K95gYnN8on754ajJ-WMPOsqioRfa0tU1V5Xw-Xb2SYjVJcz-HorG03TsAHZ3oS3-D0u1iRdDGFDuTAG--na1N-TpamqJ5gVgZHOXGQku7U-2pmuhbgB-qgkJuQJyCoHYmLdLgR13hGkvei-7msUaMm-Q-J2zR9XRsqR84VAoOKMba5idOQzQQBfwsPpw7qtl8vO-CjUklg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Lysobacter genes encoding lytic enzymes active against eukaryotic cells. (A) Distribution of the genes encoding endoglucanase/β-1,3-glucanase and chitinase (genome backbones are the same as in Fig. 2); red and blue lines indicate the lytic enzyme genes included and not included in the universal core genome structure, respectively. (B) Orthologous relationships of lytic enzymes among five Lysobacter strains and expression patterns of the orthologs identified in M497-1. Red and blue boxes correspond to red and blue lines, respectively, in A; green and yellow bars show RPKM ratios in 24- and 48-h cultures, respectively. Promoter-like sequences detected only by PePPER39 are shown in blue, and those identified by both PePPER and GENETYX-MAC are shown in black. n.d., not detected." path-from-xml="dsw055f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055134/dsw055f3.tif" path-from-xml="dsw055f3.tif" target="_blank">View large</a><a section="78055134" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055f3.png?Expires=2147483647&Signature=c9xyIZymP2MErYmfBrd1V02yfyFk8yDBq61yE7No0jSHVDOlelanVjLRAXPou0abgmywmUvBfCqnJ5fNpoUTwKcSmGVpcyd0LQIRqVhTQlV7LWczoZOnqjZ8aP0xWi2OglxDP0EKTCyU6rmsK~8j2yDMwtBvl-I6EZ6WoUD4SngiiP885zXdKmGDPyiyw5ncDgmXcaBqy2ntw1Bpn2JX9jcYyAYPIt7v54ytGwi9WcY4bWZCCSCc5nmb34sjp5TN3GDyVfPlK7eW7HCPJdMYYNm-CHr0ZOJyppt-wekYzKs7idoIV7ht-1YWZA8f4ZfnlGfX-P80CC4ON5z0He9ghA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055134&ar=2870503&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw055f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Lysobacter</em> genes encoding lytic enzymes active against eukaryotic cells. <strong>(A)</strong> Distribution of the genes encoding endoglucanase/β-1,3-glucanase and chitinase (genome backbones are the same as in <a reveal-id="dsw055-F2" data-open="dsw055-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>); red and blue lines indicate the lytic enzyme genes included and not included in the universal core genome structure, respectively. <strong>(B)</strong> Orthologous relationships of lytic enzymes among five <em>Lysobacter</em> strains and expression patterns of the orthologs identified in M497-1. Red and blue boxes correspond to red and blue lines, respectively, in A; green and yellow bars show RPKM ratios in 24- and 48-h cultures, respectively. Promoter-like sequences detected only by PePPER<sup>39</sup> are shown in blue, and those identified by both PePPER and GENETYX-MAC are shown in black. n.d., not detected.</p></div></div></div><p>The M497-1 genome also contains 4 endoglucanase/β-1,3-glucanase-encoding genes belonging to glycoside hydrolase families 6, 8, 16 and 64; 2 of these (LEN_0845 and LEN_3546) were included in the universal core OGs (<a reveal-id="dsw055-F3" data-open="dsw055-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). <em>L. enzymogenes</em> C3 lacks an orthologous gene corresponding to LEN_1176 (family 6), while <em>L. antibioticus</em> 76 and <em>L gummosus</em> 3.2.11 do not have orthologs of LEN_4406 (family 16) (<a reveal-id="dsw055-F3" data-open="dsw055-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). Except for LEN_3546, endoglucanase/β-1,3-glucanase-encoding genes were significantly expressed (relative RPKM ratio of 1.5 and higher in 24-h culture); the highest levels were observed for LEN_0845 (family 8), which demonstrated RPKM ratios of almost 10 and 2 in 24-h and 48-h cultures, respectively. However, no σ<sup>54</sup>-dependent promoter-like sequences could be detected for these genes.</p>                            <h3 scrollto-destination=78055136 id="78055136" class="section-title" >3.4. Natural products</h3>
<div class="block-child-p">Although natural products from gram-positive actinomycetes have been extensively studied,<a reveal-id="dsw055-B53" data-open="dsw055-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> those from gram-negative bacteria have received little attention. However, a number of <em>Lysobacter</em> spp. are currently recognized as promising antibiotic producers.<a reveal-id="dsw055-B54" data-open="dsw055-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> In the <em>L. enzymogenes</em> M497-1 genome, we found 98 genes related to the biosynthesis of natural products such as lantibiotics, non-ribosomal peptide and polyketide compounds, and siderophores (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&amp;Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>); to the best of our knowledge, this is the highest number of such genes identified in <em>Lysobacter</em> genomes. However, none of these genes, except for LEN_4535, was included in the universal core OGs, although some of them had orthologous relationships to those of other <em>Lysobacter</em> strains. Among the 98 genes, 31 encoded NRPSs, type I PKSs, and NRPS/PKS hybrids comprising various catalytic domains, and 12 of them, LEN_2075-2077, LEN_2515–2517 and LEN_2548–2551 (NRPS/PKS), LEN_2803 (type I PKS), and LEN_4535 (NRPS) were well conserved at similar positions in the genomes of <em>L. enzymogenes</em> stains M497-1 and C3 (<a reveal-id="dsw055-F4" data-open="dsw055-F4" class="link link-reveal link-table xref-fig">Fig. 4</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&amp;Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). On the other hand, M497-1 was found to have many unique genes, especially for type I PKS and NRPS/PKS hybrids, which have not yet been identified in other fully sequenced <em>Lysobacter</em> spp. (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&amp;Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). Furthermore, 9 out of 20 genes responsible for NRPS were also unique in M497-1. These unique genes might have been acquired through horizontal gene transfer because they showed 40–60% sequence identity with those of several species within phylogenetically distant genera <em>Rhodococcus</em>, <em>Polyangium, Bradyrhizobium</em>, except for <em>Xanthomonas</em>, within the same family. Unfortunately, significant expression was observed only for the NRPS cluster comprising LEN_4535 and LEN_4536, which showed RPKM ratios of 4–6 in 24-h culture and 2.5 in 48-h culture although the product was not identified in the culture medium. Although genes encoding types II and III PKSs (which are involved in the biosynthesis of aromatic compounds) and genes involved in the biosynthesis of terpene compounds have been frequently found in <em>Actinomycetales</em> genomes, such genes were not identified in the M497-1 genome. Interestingly, the region LEN_2603–LEN_2673 (ca. 145 kb) was extremely similar to that containing a gene cluster for lysobactin biosynthesis of <em>Lysobacter</em> sp. ATCC 53042, which is a patent strain (accession no. JF412274). The lysobactin biosynthesis cluster comprising LG3211_2475, 2476 is also present in the <em>L. gummosus</em> 3.2.11 genome,<a reveal-id="dsw055-B37" data-open="dsw055-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> and these genes showed significant identities (&gt;70%) to LEN_2615 and LEN_2613. Through the analysis of a series of genes involved in lysobactin biosynthesis in <em>Lysobacter</em> sp. ATCC 53042, it was suggested that the region LEN_2608–LEN_2615 is responsible for the biosynthesis of lysobactin or related compounds.<a reveal-id="dsw055-B55" data-open="dsw055-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Although 2 NRPSs (LybA and LybB) of <em>Lysobacter</em> sp. ATCC 53042 are involved in the formation of the peptide backbone of lysobactin, 3 NRPSs (LEN_2613–2615) identified in the <em>L. enzymogenes</em> M497-1 genome were found to correspond to LybA and LybB; LEN_2615 corresponds to LybA, while LybB is divided into two NRPSs, LEN_2614 and LEN_2613. Thus, <em>L. enzymogenes</em> M497-1 is presumably able to produce lysobactin or related antibacterials. Notably, gene clusters encoding other antibiotics identified in <em>Lysobacter</em> species, such as heat-stable antifungal factor (HSAF)/dihydromaltophilin, WAP-8294A2 (anti-MRSA), and phenazine,<a reveal-id="dsw055-B37" data-open="dsw055-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> were not detected in M497-1. <div data-id="dsw055-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/m_dsw055f4.png?Expires=2147483647&amp;Signature=bhKETe4u7w-FCp-t8LWmRmoWzHCY4OskWdTKOU8LTzTyYKZHH8rSwUmbPeWlRBS3bG3QPdFfwzs-QkaNQyp8nOJx61M0jvl71~kqIfVfG1JNbob2ri35oWCEbBDCDB7rk9TI0F~ANfwQCc9gp9P2oSqcswgZpN-LIow5gjAhaDzwIqZ7Oqj8xxUtTbf-J4pnD9o52~JNRlIX05GgVntRbsJF-r~wVWtegu~HYji6gsviWfmXgwULvUNWzcTumMJnrqHKaoY8s8GdxTininmiUwy-I2yAGEqc6jsEag4bU8vAsQ73VQBJLIqcDemPrSikPFzv~itfCQh-4mABPFDStA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of genes encoding NRPS and PKS with core catalytic domains for secondary metabolite biosynthesis. Gene sets with the same ID number (1–9) are orthologous; gene sets unique for each strain (no ID number) are indicated. The detailed core domain structure of each gene is shown in Supplementary Table S5." path-from-xml="dsw055f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055137/dsw055f4.tif" path-from-xml="dsw055f4.tif" target="_blank">View large</a><a section="78055137" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055f4.png?Expires=2147483647&amp;Signature=29CgOcMFVEsAceZzB9RR4nDpjzuDF7a~z7KDxoF~88ifKhIOmR3e-EgaLRcQAmnOt4-756rYoPzfrz4gtnPW0ppKosnn-cxRna4u5xqFxs5knpwD5bjlKq1AP8791jMY0CxRMZq07j~tWKueS~duybhi~JYnQadP~tlGj5oxHYLGmiCyurKQQym8Gkh-2~gr5jiwBwnhimXgvDzaccwP1Ovb-ofvSzNCBCcMAjvIOQ5QN6uDf6-l6Y-PSNgnljXLHC41i-Gk5B5An9~shYkTgLjDgSXCgFXQt1NiY3PCX68Deb2~8y8vGguTLRlhe-DelHboM~ETO1CVD3Fv32rskw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055137&amp;ar=2870503&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw055f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of genes encoding NRPS and PKS with core catalytic domains for secondary metabolite biosynthesis. Gene sets with the same ID number (1–9) are orthologous; gene sets unique for each strain (no ID number) are indicated. The detailed core domain structure of each gene is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&amp;Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>.</p></div></div><div content-id="dsw055-F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/m_dsw055f4.png?Expires=2147483647&amp;Signature=bhKETe4u7w-FCp-t8LWmRmoWzHCY4OskWdTKOU8LTzTyYKZHH8rSwUmbPeWlRBS3bG3QPdFfwzs-QkaNQyp8nOJx61M0jvl71~kqIfVfG1JNbob2ri35oWCEbBDCDB7rk9TI0F~ANfwQCc9gp9P2oSqcswgZpN-LIow5gjAhaDzwIqZ7Oqj8xxUtTbf-J4pnD9o52~JNRlIX05GgVntRbsJF-r~wVWtegu~HYji6gsviWfmXgwULvUNWzcTumMJnrqHKaoY8s8GdxTininmiUwy-I2yAGEqc6jsEag4bU8vAsQ73VQBJLIqcDemPrSikPFzv~itfCQh-4mABPFDStA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of genes encoding NRPS and PKS with core catalytic domains for secondary metabolite biosynthesis. Gene sets with the same ID number (1–9) are orthologous; gene sets unique for each strain (no ID number) are indicated. The detailed core domain structure of each gene is shown in Supplementary Table S5." path-from-xml="dsw055f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055137/dsw055f4.tif" path-from-xml="dsw055f4.tif" target="_blank">View large</a><a section="78055137" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055f4.png?Expires=2147483647&amp;Signature=29CgOcMFVEsAceZzB9RR4nDpjzuDF7a~z7KDxoF~88ifKhIOmR3e-EgaLRcQAmnOt4-756rYoPzfrz4gtnPW0ppKosnn-cxRna4u5xqFxs5knpwD5bjlKq1AP8791jMY0CxRMZq07j~tWKueS~duybhi~JYnQadP~tlGj5oxHYLGmiCyurKQQym8Gkh-2~gr5jiwBwnhimXgvDzaccwP1Ovb-ofvSzNCBCcMAjvIOQ5QN6uDf6-l6Y-PSNgnljXLHC41i-Gk5B5An9~shYkTgLjDgSXCgFXQt1NiY3PCX68Deb2~8y8vGguTLRlhe-DelHboM~ETO1CVD3Fv32rskw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055137&amp;ar=2870503&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw055f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of genes encoding NRPS and PKS with core catalytic domains for secondary metabolite biosynthesis. Gene sets with the same ID number (1–9) are orthologous; gene sets unique for each strain (no ID number) are indicated. The detailed core domain structure of each gene is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&amp;Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>.</p></div></div></div>                            <h3 scrollto-destination=78055138 id="78055138" class="section-title" >3.5. Metabolic and physiological potential</h3>
<div class="block-child-p">To compare the metabolic and physiological potential of M497-1 and other <em>Lysobacter</em> strains, the completion ratios of the four types of KEGG functional modules (pathway, structural complex, functional set and signature modules) were calculated with MAPLE (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&amp;Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>).<a reveal-id="dsw055-B48" data-open="dsw055-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> Most MCR patterns were similar among the 5 <em>Lysobacter</em> strains (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&amp;Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>); however, several modules demonstrated species-specific differences (<a reveal-id="dsw055-F5" data-open="dsw055-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Among them, there were seven pathway modules related to amino acid metabolism, nitrogen metabolism, carbohydrate metabolism, and terpenoid backbone biosynthesis. In the ornithine biosynthesis module comprising 5 reaction steps, <em>L. enzymogenes</em> M497-1 and <em>L. gummosus</em> 3.2.11 lacked amino-acid N-acetyltransferase (<em>argA</em>) assigned to the first reaction step. The module for dissimilatory nitrate reduction comprising two reaction steps was complete only in <em>L. antibioticus</em> 76, while the enzymes for each reaction step, nitrate reductase and nitrite reductase, were absent in the other 4 species. Except for <em>L. enzymogenes</em> strains, the other three <em>Lysobacter</em> spp. completed the module for <span class="small-caps">d</span>-galactonate degradation, which is one of the rare modules, i.e. completed by less than 10% of prokaryotes.<a reveal-id="dsw055-B49" data-open="dsw055-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a><div data-id="dsw055-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/m_dsw055f5.png?Expires=2147483647&amp;Signature=4bhWhtXkhSMyA-DYqiFh1L836S6xPCnyFGA3CIywnK9atghq6WUMpDED6b60nAqR-HfO7FColIB3bdO-ZrXZFUzU~~nfOSUlOOaUKpDiIKeqEJU~TA0fQN-c5LyBs9FIigLHqo54Nfu0CqEDh2mmFpEO1NAysfcpdO9m65~Tt8vDiLl5yS7G8pmn8SztvkWmLE35Hm1SQgIjLqoD6Iy4kCYgRVNBHUX9F4pPnDVpwvrcX2WxD~cfkn4EGVQJUNAsW1Q3MWjq3WJEG1jESi1PERbsDaf-QuloHbYwp2Kg0erOwX2tqW3XxERkzNPNxx9um50s2iM8iT2W5cpaMQWHmw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Major differences in metabolic and physiological potential among Lysobacter strains. The MCR of the KEGG functional modules was calculated using MAPLE48; modules with different MCR patterns among Lysobacter strains are highlighted. First 7 modules, pathways; next 10 modules, structural complexes; last 7 modules, functional sets. Modules defined as ‘rare’ are indicated with an asterisk.49" path-from-xml="dsw055f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055139/dsw055f5.tif" path-from-xml="dsw055f5.tif" target="_blank">View large</a><a section="78055139" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055f5.png?Expires=2147483647&amp;Signature=INsoIdzjcAG8~b0yf9nlHK3kFRfiijiYvR-OfAq8N7s1szKPOwXWuajZP6yZJDVoXBoZBQMSokrh0-VSPkxtJfRjaxasTBuODaJwKBdIeoNf2loJbk3yfx2HVE4PdPe3Jhp~3hBWiZCydewUbaCqdYq6uvbE8LccPgSECSgzPUBEbZ045SFnmWG4JatOKqkeFq0INB~GISFOpBz3iuBB9Odyk91eR6N1vQFaDwnDKpakGl~p4F5zd-C2sCWcF0XEBn56xZJhWwcxE75m49rwkWHyX-wNF8fuv7MbK4s~egmMruCMiuy9TSefZ9qeD8g~FbfhVJVACNvFzvkJDN1Uqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055139&amp;ar=2870503&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw055f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Major differences in metabolic and physiological potential among <em>Lysobacter</em> strains. The MCR of the KEGG functional modules was calculated using MAPLE<sup>48</sup>; modules with different MCR patterns among <em>Lysobacter</em> strains are highlighted. First 7 modules, pathways; next 10 modules, structural complexes; last 7 modules, functional sets. Modules defined as ‘rare’ are indicated with an asterisk.<a reveal-id="dsw055-B49" data-open="dsw055-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a></p></div></div><div content-id="dsw055-F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/m_dsw055f5.png?Expires=2147483647&amp;Signature=4bhWhtXkhSMyA-DYqiFh1L836S6xPCnyFGA3CIywnK9atghq6WUMpDED6b60nAqR-HfO7FColIB3bdO-ZrXZFUzU~~nfOSUlOOaUKpDiIKeqEJU~TA0fQN-c5LyBs9FIigLHqo54Nfu0CqEDh2mmFpEO1NAysfcpdO9m65~Tt8vDiLl5yS7G8pmn8SztvkWmLE35Hm1SQgIjLqoD6Iy4kCYgRVNBHUX9F4pPnDVpwvrcX2WxD~cfkn4EGVQJUNAsW1Q3MWjq3WJEG1jESi1PERbsDaf-QuloHbYwp2Kg0erOwX2tqW3XxERkzNPNxx9um50s2iM8iT2W5cpaMQWHmw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Major differences in metabolic and physiological potential among Lysobacter strains. The MCR of the KEGG functional modules was calculated using MAPLE48; modules with different MCR patterns among Lysobacter strains are highlighted. First 7 modules, pathways; next 10 modules, structural complexes; last 7 modules, functional sets. Modules defined as ‘rare’ are indicated with an asterisk.49" path-from-xml="dsw055f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055139/dsw055f5.tif" path-from-xml="dsw055f5.tif" target="_blank">View large</a><a section="78055139" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055f5.png?Expires=2147483647&amp;Signature=INsoIdzjcAG8~b0yf9nlHK3kFRfiijiYvR-OfAq8N7s1szKPOwXWuajZP6yZJDVoXBoZBQMSokrh0-VSPkxtJfRjaxasTBuODaJwKBdIeoNf2loJbk3yfx2HVE4PdPe3Jhp~3hBWiZCydewUbaCqdYq6uvbE8LccPgSECSgzPUBEbZ045SFnmWG4JatOKqkeFq0INB~GISFOpBz3iuBB9Odyk91eR6N1vQFaDwnDKpakGl~p4F5zd-C2sCWcF0XEBn56xZJhWwcxE75m49rwkWHyX-wNF8fuv7MbK4s~egmMruCMiuy9TSefZ9qeD8g~FbfhVJVACNvFzvkJDN1Uqw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055139&amp;ar=2870503&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw055f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Major differences in metabolic and physiological potential among <em>Lysobacter</em> strains. The MCR of the KEGG functional modules was calculated using MAPLE<sup>48</sup>; modules with different MCR patterns among <em>Lysobacter</em> strains are highlighted. First 7 modules, pathways; next 10 modules, structural complexes; last 7 modules, functional sets. Modules defined as ‘rare’ are indicated with an asterisk.<a reveal-id="dsw055-B49" data-open="dsw055-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a></p></div></div></div><p>The differences in MCR patterns among <em>Lysobacter</em> strains were also observed in structural-complex and functional-set modules. As a conspicuous example, only <em>L. antibioticus</em> 76 completed the modules for RTX (repeats in toxin) transport belonging to the type I secretion system, putrescine transport, phospholipid transport, aminoglycoside-resistant protease HtpX, and the osmotic stress response (EnvZ-OmpR) two-component regulatory system (<a reveal-id="dsw055-F5" data-open="dsw055-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). In addition, the modules for type VI secretion and simple sugar transport were completed only by <em>L. enzymogenes</em> C3 and <em>L. capsici</em> 55, respectively. Moreover, strain-specific differences between M497-1 and C3 were detected in 3 rare modules: the α-hemolysin/cyclolysin transport system and two-component regulatory systems associated with central carbon metabolism (BarA-UvrY) and chemosensing (WspE-WspRF). Thus, we could detect exact differences in the metabolic and physiological potential among five closely related <em>Lysobacter</em> species.</p>                            <h3 scrollto-destination=78055141 id="78055141" class="section-title" >3.6. Regulatory factors</h3>
<p>The production of bioactive products in <em>Lysobacter</em> strains is regulated by intracellular signaling mediated by cyclic adenosine monophosphate-receptor-like protein (Clp) and the diffusible signal factor (DSF)-dependent system.<a reveal-id="dsw055-B56" data-open="dsw055-B56" class="link link-ref link-reveal xref-bibr"><sup>56–58</sup></a> The Clp-encoding gene was identified in all <em>Lysobacter</em> genomes, including M497-1 (LEN_0938), and the deduced gene products were 99–100% identical to each other. LEN_0938 was highly expressed as evidenced by its RPKM ratio of 6.6, and the expression of various Clp-regulated genes, including those encoding lytic enzymes such as chitinase, endoglucanase/β-1,3-glucanase, and peptidase, was also markedly induced (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). These data are consistent with previous reports that Clp upregulated the expression of chitinase, β-1,3-glucanse, peptidase, and antifungal factor in <em>L. enzymogenes</em> C3 and OH11.<a reveal-id="dsw055-B57" data-open="dsw055-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a><sup>,</sup><a reveal-id="dsw055-B58" data-open="dsw055-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a></p><p>Genes encoding components of the DSF-dependent system (<em>rpfG</em>, <em>rpfC</em>, <em>rpfF</em>, and <em>rpfB</em>), which is widely conserved among <em>Lysobacter</em> species (with the exception of <em>L. capsici</em>), were also identified in the M497-1 genome (LEN_2756–2759). A previous study did not detect genes corresponding to <em>rpfC</em> and <em>rpfF</em> in <em>L. capsici</em> 55; however, two genes (LC55x_5419 and LC55x_5418) were assigned to the <em>rpfF</em> and <em>rpfC</em> OGs, respectively, by MBGD analysis.<a reveal-id="dsw055-B37" data-open="dsw055-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> These genes showed 30–40% sequence similarity to phylogenetically distant species within different classes such as <em>Betaproteobacteria</em> and <em>Alphaproteobacteria</em>, suggesting that they might have been acquired by <em>L. capsici</em> 55 through horizontal gene transfer. <em>L. enzymogenes</em> OH11 produces a DSF-like protein, LeDSF3, which positively regulates HSAF biosynthesis through NRPS and the RpfC/RpfG-Clp pathway.<a reveal-id="dsw055-B58" data-open="dsw055-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> However, in the M497-1 genome, the genes responsible for the DSF-dependent system (LEN_2756–2759) showed very low expression when compared with Clp (RPKM ratio 0.2–0.6 versus 6.6, respectively), and genes related to HSAF biosynthesis regulated by the DSF-dependent system were not identified.</p><p>With regard to antibiotic production pathways, we found 16 genes related to type I PKS and NRPS to be unique to M497-1 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). Additionally, all of them were silent, although three other orthologous genes (LEN_4535, LEN_2614, and LEN_2615) showed an expression level greater than half the average expression level of ribosomal proteins in a 24-h culture (&gt;0.58). Further studies focusing on the regulatory mechanisms underlying antibiotic production in <em>L. enzymogenes</em> M497-1 will be required with regard to not only unique genes but also other orthologous genes to identify novel antibiotics synthesized by this strain.</p><p>In conclusion, <em>L. enzymogenes</em> M497-1 genome sequencing and expression profiling performed in this study revealed 16 unique genes for biosynthesis of natural product and 15 highly expressed genes encoding peptidases except for commercialized achromopeptidase. It also represented the possibility to produce lysobactin or related antimicrobial compounds. Thus, our findings are expected to promote further biotechnological application of <em>Lysobacter</em> spp. as a promising source of natural, bioactive compounds.</p>                            <h2 scrollto-destination=78055146 id="78055146" class="section-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=78055148 id="78055148" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Mr T. Hara of Wako Pure Chemical Industries, Ltd. for his valuable contribution to this work.</p>                            <h3 scrollto-destination=78055150 id="78055150" class="section-title" >Data availability</h3>
<p>The nucleotide sequences determined in this study were deposited in DDBJ/EBI/NCBI database. The accession number for <em>L. enzymogenes</em> M497-1 whole genome sequence is AP014940 and that for RNA sequences (RNA-seq) of 24- and 48-h cultures is DRA003922.</p>                            <h3 scrollto-destination=78055152 id="78055152" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=78055154 id="78055154" class="section-title" >Accession numbers</h3>
<p>AP014940</p>                            <h2 scrollto-destination=78055156 id="78055156" class="backsection-title" >Funding</h2>
<p>This work was supported in part by a Grant-in-Aid for Scientific Research from the Ministry of Education, Culture, Sports, Science and Technology of Japan to H.T. (No. 20310124 and 26550053) and by a grant from the Cross-ministerial Strategic Innovation Promotion Programme to HT and WA. This work was also supported in part by grants from the collaborative research programme of National Institute for Basic Biology (NIBB) to H.T. and I.U. (No. 13-360, 14-359, and 15-355). Computational resources were provided partly by the Data Integration and Analysis Facility of NIBB.</p>                            <h2 scrollto-destination=78055158 id="78055158" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw055-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Isono</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Takahashi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Yamazaki</div> <div class="given-names">Y.</div></div></div> <div class="year">1972</div>, <div class="source">Bacteriolytic enzyme and process for the production thereof</div> . US Patent 3,649,454.</div></div></div></div><div content-id="dsw055-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Masaki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Nakamura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Isono</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Soejima</div> <div class="given-names">M.</div></div></div> <div class="year">1978</div>, 
<div class="article-title">A New proteolytic enzyme from <em>Achromobacter lyticus</em> M497-1</div>. <div class="source">Agric. Biol. Chem</div> ., <div class="volume">42</div>, <div class="fpage">1443</div>–<div class="lpage">5</div>.</div></div></div></div><div content-id="dsw055-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Masaki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Tanabe</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Nakamura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Soejima</div> <div class="given-names">M.</div></div></div> <div class="year">1981</div>, 
<div class="article-title">Studies on a new proteolytic enzyme from <em>Achromobacter lyticus</em> M497-1. I. Purification and some enzymatic properties</div>. <div class="source">Biochim. Biophys. Acta</div> , <div class="volume">660</div>, <div class="fpage">44</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Studies%20on%20a%20new%20proteolytic%20enzyme%20from%20Achromobacter%20lyticus%20M497-1.%20I.%20Purification%20and%20some%20enzymatic%20properties&amp;author=T.%20Masaki&amp;author=M.%20Tanabe&amp;author=K.%20Nakamura&amp;author=M.%20Soejima&amp;publication_year=1981&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=660&amp;pages=44-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0005-2744(81)90106-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0005-2744(81)90106-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0005-2744(81)90106-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6791693" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Masaki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Tanabe</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Nakamura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Soejima</div> <div class="given-names">M.</div></div></div> <div class="year">1981</div>, 
<div class="article-title">Studies on a new proteolytic enzyme from <em>Achromobacter lyticus</em> M497-1. II. Specificity and inhibition studies of <em>Achromobacter</em> protease I</div>. <div class="source">Biochim. Biophys. Acta</div> , <div class="volume">660</div>, <div class="fpage">51</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Studies%20on%20a%20new%20proteolytic%20enzyme%20from%20Achromobacter%20lyticus%20M497-1.%20II.%20Specificity%20and%20inhibition%20studies%20of%20Achromobacter%20protease%20I&amp;author=T.%20Masaki&amp;author=M.%20Tanabe&amp;author=K.%20Nakamura&amp;author=M.%20Soejima&amp;publication_year=1981&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=660&amp;pages=51-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0005-2744(81)90107-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0005-2744(81)90107-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0005-2744(81)90107-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6168293" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Masaki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Soejima</div> <div class="given-names">M.</div></div></div> <div class="year">1986</div>, 
<div class="article-title">Purification and some properties of <em>Achromobacter</em> protease Ia from <em>Achromobacter lyticus</em> M497-1</div>. <div class="source">Agric. Biol. Chem.</div>  <div class="volume">50</div>, <div class="fpage">3087</div>–<div class="lpage">91</div>.</div></div></div></div><div content-id="dsw055-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ohara</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Makino</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Shinagawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Nakata</div> <div class="given-names">A</div></div>, <div class="name string-name"><div class="surname">Norioka</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sakiyama</div> <div class="given-names">F.</div></div></div> <div class="year">1989</div>, 
<div class="article-title">Cloning, nucleotide sequence, and expression of <em>Achromobacter</em> protease I gene</div>. <div class="source">J. Biol. Chem.</div>  <div class="volume">264</div>, <div class="fpage">20625</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%2C%20nucleotide%20sequence%2C%20and%20expression%20of%20Achromobacter%20protease%20I%20gene&amp;author=T.%20Ohara&amp;author=K.%20Makino&amp;author=H.%20Shinagawa&amp;author=A%20Nakata&amp;author=S.%20Norioka&amp;author=F.%20Sakiyama&amp;publication_year=1989&amp;journal=J.%20Biol.%20Chem.&amp;volume=264&amp;pages=20625-31" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2684982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barsotti</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Morrier</div> <div class="given-names">J. J.</div></div>, <div class="name string-name"><div class="surname">Freney</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">1988</div>, 
<div class="article-title">Achromopeptidase for rapid lysis of oral anaerobic Gram-positive rods</div>. <div class="source">Oral Microbiol. Immunol</div> ., <div class="volume">3</div>, <div class="fpage">86</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Achromopeptidase%20for%20rapid%20lysis%20of%20oral%20anaerobic%20Gram-positive%20rods&amp;author=O.%20Barsotti&amp;author=J.%20J.%20Morrier&amp;author=J.%20Freney&amp;publication_year=1988&amp;journal=Oral%20Microbiol.%20Immunol&amp;volume=3&amp;pages=86-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/omi.1988.3.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fomi.1988.3.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fomi.1988.3.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3268757" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nakamura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Okazawa</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Soejima</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Masaki</div> <div class="given-names">T.</div></div></div> <div class="year">1973</div>, 
<div class="article-title">Lysis of <em>Micrococcus radiodurans</em></div>. <div class="source">Agric. Biol. Chem</div> ., <div class="volume">37</div>, <div class="fpage">2667</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lysis%20of%20Micrococcus%20radiodurans&amp;author=K.%20Nakamura&amp;author=Y.%20Okazawa&amp;author=M.%20Soejima&amp;author=T.%20Masaki&amp;publication_year=1973&amp;journal=Agric.%20Biol.%20Chem&amp;volume=37&amp;pages=2667-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/00021369.1973.10861056" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F00021369.1973.10861056" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F00021369.1973.10861056"> </span></div> </div></div></div></div></div><div content-id="dsw055-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Horinouchi</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Uozumi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Beppu</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Arima</div> <div class="given-names">K.</div></div></div> <div class="year">1977</div>, 
<div class="article-title">A new isolation method of plasmid deoxyribonucleic acid from <em>Staphylococcus aureus</em> using a lytic enzyme of <em>Achromobacter lyticus</em></div>. <div class="source">Agric. Biol. Chem</div> ., <div class="volume">41</div>, <div class="fpage">2487</div>–<div class="lpage">9</div>.</div></div></div></div><div content-id="dsw055-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ezaki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">S.</div></div></div> <div class="year">1982</div>, 
<div class="article-title">Achromopeptidase for lysis of anaerobic Gram-positive cocci</div>. <div class="source">J. Clinic. Microbiol.</div>  <div class="volume">16</div>, <div class="fpage">844</div>–<div class="lpage">6</div>.</div></div></div></div><div content-id="dsw055-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Morita</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kuwahara</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ohshima</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">An improved DNA Isolation method for metagenomic analysis of the microbial flora of the human intestine</div>. <div class="source">Microbes Environ.</div>  <div class="volume">22</div>, <div class="fpage">214</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20improved%20DNA%20Isolation%20method%20for%20metagenomic%20analysis%20of%20the%20microbial%20flora%20of%20the%20human%20intestine&amp;author=H.%20Morita&amp;author=T.%20Kuwahara&amp;author=K.%20Ohshima&amp;publication_year=2007&amp;journal=Microbes%20Environ.&amp;volume=22&amp;pages=214-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1264/jsme2.22.214" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1264%2Fjsme2.22.214" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1264%2Fjsme2.22.214"> </span></div> </div></div></div></div></div><div content-id="dsw055-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Norioka</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sakiyama</div> <div class="given-names">F.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Molecular cloning and nucleotide sequence of the β-lytic protease gene from <em>Achromobacter lyticus</em></div>. <div class="source">J. Bacteriol.</div>  <div class="volume">172</div>, <div class="fpage">6506</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%20and%20nucleotide%20sequence%20of%20the%20%CE%B2-lytic%20protease%20gene%20from%20Achromobacter%20lyticus&amp;author=S.%20Li&amp;author=S.%20Norioka&amp;author=F.%20Sakiyama&amp;publication_year=1990&amp;journal=J.%20Bacteriol.&amp;volume=172&amp;pages=6506-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/jb.172.11.6506-6511.1990" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2Fjb.172.11.6506-6511.1990" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2Fjb.172.11.6506-6511.1990"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2228973" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Norioka</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sakiyama</div> <div class="given-names">F.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Purification, Staphylolytic activity, and cleavage sites of α-lytic protease from <em>Achromobacter lyticus</em></div>. <div class="source">J. Biochem.</div>  <div class="volume">122</div>, <div class="fpage">772</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Purification%2C%20Staphylolytic%20activity%2C%20and%20cleavage%20sites%20of%20%CE%B1-lytic%20protease%20from%20Achromobacter%20lyticus&amp;author=S.%20Li&amp;author=S.%20Norioka&amp;author=F.%20Sakiyama&amp;publication_year=1997&amp;journal=J.%20Biochem.&amp;volume=122&amp;pages=772-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.jbchem.a021822" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.jbchem.a021822" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.jbchem.a021822"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9399581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Norioka</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Sakiyama</div> <div class="given-names">F.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Bacteriolytic activity and specificity of <em>Achromobacter</em> β-lytic protease</div>. <div class="source">J. Biochem.</div>  <div class="volume">124</div>, <div class="fpage">332</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacteriolytic%20activity%20and%20specificity%20of%20Achromobacter%20%CE%B2-lytic%20protease&amp;author=S.%20Li&amp;author=S.%20Norioka&amp;author=F.%20Sakiyama&amp;publication_year=1998&amp;journal=J.%20Biochem.&amp;volume=124&amp;pages=332-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.jbchem.a022116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.jbchem.a022116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.jbchem.a022116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9685723" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Damoglou</div> <div class="given-names">A. P.</div></div>, <div class="name string-name"><div class="surname">Allen</div> <div class="given-names">L. C.</div></div>, <div class="name string-name"><div class="surname">Roy</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Whitaker</div> <div class="given-names">D. R.</div></div></div> <div class="year">1972</div>, 
<div class="article-title">The β-lytic protease of <em>Myxobacter</em> 495: aspects of its structure</div>. <div class="source">Proc. Can. Fed. Biol. Soc</div> . <div class="volume">15</div>, <div class="fpage">27</div>.</div></div></div></div><div content-id="dsw055-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ensign</div> <div class="given-names">J. C.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">R. S.</div></div></div> <div class="year">1965</div>, 
<div class="article-title">Lysis of bacterial cell walls by an enzyme isolated from a <em>Myxobacter</em></div>. <div class="source">J. Bacteriol.</div>  <div class="volume">90</div>, <div class="fpage">395</div>–<div class="lpage">401</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lysis%20of%20bacterial%20cell%20walls%20by%20an%20enzyme%20isolated%20from%20a%20Myxobacter&amp;author=J.%20C.%20Ensign&amp;author=R.%20S.%20Wolfe&amp;publication_year=1965&amp;journal=J.%20Bacteriol.&amp;volume=90&amp;pages=395-401" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14330733" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ensign</div> <div class="given-names">J. C.</div></div>, <div class="name string-name"><div class="surname">Wolfe</div> <div class="given-names">R. S.</div></div></div> <div class="year">1966</div>, 
<div class="article-title">Characterization of a small proteolytic enzyme which lyses bacterial cell walls</div>. <div class="source">J. Bacteriol.</div>  <div class="volume">91</div>, <div class="fpage">524</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20small%20proteolytic%20enzyme%20which%20lyses%20bacterial%20cell%20walls&amp;author=J.%20C.%20Ensign&amp;author=R.%20S.%20Wolfe&amp;publication_year=1966&amp;journal=J.%20Bacteriol.&amp;volume=91&amp;pages=524-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5935339" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Whitaker</div> <div class="given-names">D. R.</div></div></div> <div class="year">1965</div>. 
<div class="article-title">Lytic enzymes of <em>Sorangium</em> sp., isolation and properties of the α-lytic and β-lytic proteases</div>. <div class="source">Can J. Biochem.</div>  <div class="volume">43</div>, <div class="fpage">1935</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lytic%20enzymes%20of%20Sorangium%20sp.%2C%20isolation%20and%20properties%20of%20the%20%CE%B1-lytic%20and%20%CE%B2-lytic%20proteases&amp;author=D.%20R.%20Whitaker&amp;publication_year=1965&amp;journal=Can%20J.%20Biochem.&amp;volume=43&amp;pages=1935-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/o65-217" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fo65-217" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fo65-217"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/5326246" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhan</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Yuen</div> <div class="given-names">G. Y.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Biological control of <em>Bipolaris sorokiniana</em> on tall fescue by <em>Stenotrophomonas maltophila</em> strain C3</div>. <div class="source">Phytopathology</div>  <div class="volume">89</div>, <div class="fpage">817</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biological%20control%20of%20Bipolaris%20sorokiniana%20on%20tall%20fescue%20by%20Stenotrophomonas%20maltophila%20strain%20C3&amp;author=Z.%20Zhan&amp;author=G.%20Y.%20Yuen&amp;publication_year=1999&amp;journal=Phytopathology&amp;volume=89&amp;pages=817-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/PHYTO.1999.89.9.817" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FPHYTO.1999.89.9.817" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FPHYTO.1999.89.9.817"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18944711" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Yuen</div> <div class="given-names">G. Y.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Effect of culture fluids and preincubation of chitinase production on biocontrol of <em>Bipolaris</em> leaf spot by <em>Stenotrophomonas maltophila</em> C3</div>. <div class="source">Biol. Control</div>  <div class="volume">18</div>, <div class="fpage">277</div>–<div class="lpage">86</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effect%20of%20culture%20fluids%20and%20preincubation%20of%20chitinase%20production%20on%20biocontrol%20of%20Bipolaris%20leaf%20spot%20by%20Stenotrophomonas%20maltophila%20C3&amp;author=Z.%20Zhang&amp;author=G.%20Y.%20Yuen&amp;publication_year=2000&amp;journal=Biol.%20Control&amp;volume=18&amp;pages=277-86" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/bcon.2000.0834" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fbcon.2000.0834" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fbcon.2000.0834"> </span></div> </div></div></div></div></div><div content-id="dsw055-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Christensen</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Cook</div> <div class="given-names">F. D.</div></div></div> <div class="year">1978</div>, 
<div class="article-title"><em>Lysobacter</em>, a new genus of nonfruiting, gliding bacteria with high base ratio</div>. <div class="source">Int. J. Syst. Bacteriol</div> ., <div class="volume">28</div>, <div class="fpage">367</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lysobacter%2C%20a%20new%20genus%20of%20nonfruiting%2C%20gliding%20bacteria%20with%20high%20base%20ratio&amp;author=P.%20Christensen&amp;author=F.%20D.%20Cook&amp;publication_year=1978&amp;journal=Int.%20J.%20Syst.%20Bacteriol&amp;volume=28&amp;pages=367-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00207713-28-3-367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00207713-28-3-367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00207713-28-3-367"> </span></div> </div></div></div></div></div><div content-id="dsw055-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tindall</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Euzéby</div> <div class="given-names">J. P.</div></div></div> <div class="year">2006</div>, 
<div class="article-title"><em>Lysobacter enzymogenes</em> subsp. <em>enzymogenes</em> Christensen and Cook 1978, <em>L. enzymogenes</em> subsp. <em>cookii</em> Christensen 1978 and <em>Streptococcus casseliflavus</em> (Mundt and Graham 1968) Vaughan <em>et al.</em> 1978 should have been cited in the approved lists of bacterial names. Request for an opinion</div>. <div class="source">Int. J. Syst. Evolv. Microbiol</div> ., <div class="volume">56</div>, <div class="fpage">2707</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lysobacter%20enzymogenes%20subsp.%20enzymogenes%20Christensen%20and%20Cook%201978%2C%20L.%20enzymogenes%20subsp.%20cookii%20Christensen%201978%20and%20Streptococcus%20casseliflavus%20(Mundt%20and%20Graham%201968)%20Vaughan%20et%20al.%201978%20should%20have%20been%20cited%20in%20the%20approved%20lists%20of%20bacterial%20names.%20Request%20for%20an%20opinion&amp;author=B.%20J.%20Tindall&amp;author=J.%20P.%20Euz%C3%A9by&amp;publication_year=2006&amp;journal=Int.%20J.%20Syst.%20Evolv.%20Microbiol&amp;volume=56&amp;pages=2707-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/ijs.0.64339-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fijs.0.64339-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fijs.0.64339-0"> </span></div> </div></div></div></div></div><div content-id="dsw055-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Epstein</div> <div class="given-names">D. M.</div></div>, <div class="name string-name"><div class="surname">Wensink</div> <div class="given-names">P. C.</div></div></div> <div class="year">1988</div>, 
<div class="article-title">The α-lytic protease gene of <em>Lysobacter enzymogenes</em>. The nucleotide sequence predicts a large prepro-peptide with homology to pro-peptides of other chymotrypsin-like enzymes</div>. <div class="source">J. Biol. Chem</div> . <div class="volume">263</div>, <div class="fpage">16586</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20%CE%B1-lytic%20protease%20gene%20of%20Lysobacter%20enzymogenes.%20The%20nucleotide%20sequence%20predicts%20a%20large%20prepro-peptide%20with%20homology%20to%20pro-peptides%20of%20other%20chymotrypsin-like%20enzymes&amp;author=D.%20M.%20Epstein&amp;author=P.%20C.%20Wensink&amp;publication_year=1988&amp;journal=J.%20Biol.%20Chem&amp;volume=263&amp;pages=16586-90" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3053694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chohnan</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Nonaka</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Teramoto</div> <div class="given-names">K.</div></div></div> et al.  . <div class="year">2002</div>, 
<div class="article-title"><em>Lysobacter</em> strain with high lysyl endopeptidase production</div>. <div class="source">FEMS Microbiol. Lett.</div>  <div class="volume">213</div>, <div class="fpage">13</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lysobacter%20strain%20with%20high%20lysyl%20endopeptidase%20production&amp;author=S.%20Chohnan&amp;author=J.%20Nonaka&amp;author=K.%20Teramoto&amp;publication_year=2002&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=213&amp;pages=13-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2002.213.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2002.213.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2002.213.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12127482" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chohnan</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Shiraki</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Yokoya</div> <div class="given-names">K.</div></div></div> et al.  . <div class="year">2004</div>, 
<div class="article-title">A second lysine-specific serine protease from <em>Lysobacter</em> sp. strain IB-9374</div>. <div class="source">J. Bacteriol.</div>  <div class="volume">186</div>, <div class="fpage">5093</div>–<div class="lpage">100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20second%20lysine-specific%20serine%20protease%20from%20Lysobacter%20sp.%20strain%20IB-9374&amp;author=S.%20Chohnan&amp;author=K.%20Shiraki&amp;author=K.%20Yokoya&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=5093-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.15.5093-5100.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.15.5093-5100.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.15.5093-5100.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15262946" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yuen</div> <div class="given-names">G. Y.</div></div>, <div class="name string-name"><div class="surname">Steadman</div> <div class="given-names">J. R.</div></div>, <div class="name string-name"><div class="surname">Lindgren</div> <div class="given-names">D.T.</div></div>, <div class="name string-name"><div class="surname">Schaff</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Jochum</div> <div class="given-names">C.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Bean rust biological control using bacterial agents</div>. <div class="source">Crop Protect.</div>  <div class="volume">20</div>, <div class="fpage">395</div>–<div class="lpage">402</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bean%20rust%20biological%20control%20using%20bacterial%20agents&amp;author=G.%20Y.%20Yuen&amp;author=J.%20R.%20Steadman&amp;author=D.T.%20Lindgren&amp;author=D.%20Schaff&amp;author=C.%20Jochum&amp;publication_year=2001&amp;journal=Crop%20Protect.&amp;volume=20&amp;pages=395-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0261-2194(00)00154-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0261-2194(00)00154-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0261-2194(00)00154-X"> </span></div> </div></div></div></div></div><div content-id="dsw055-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Folman</div> <div class="given-names">L. B.</div></div>, <div class="name string-name"><div class="surname">Postma</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">van Veen</div> <div class="given-names">J. A.</div></div></div>, <div class="year">2003</div>, 
<div class="article-title">Characterization of <em>Lysobacter enzymogenes</em> (Christensen and Cook 1978) strain 3.1T8, a powerful antagonist of fungal diseases of cucumber</div>. <div class="source">Microbiol. Res</div> ., <div class="volume">158</div>, <div class="fpage">107</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20Lysobacter%20enzymogenes%20(Christensen%20and%20Cook%201978)%20strain%203.1T8%2C%20a%20powerful%20antagonist%20of%20fungal%20diseases%20of%20cucumber&amp;author=L.%20B.%20Folman&amp;author=J.%20Postma&amp;author=J.%20A.%20van%20Veen&amp;publication_year=2003&amp;journal=Microbiol.%20Res&amp;volume=158&amp;pages=107-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1078/0944-5013-00185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1078%2F0944-5013-00185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1078%2F0944-5013-00185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12906383" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Moore</div> <div class="given-names">W. H.</div></div>, <div class="name string-name"><div class="surname">Yuen</div> <div class="given-names">G. Y.</div></div>, <div class="name string-name"><div class="surname">Kobayashi</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Caswell-Chen</div> <div class="given-names">E. P.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Influence of <em>Lysobacter enzymogenes</em> strain C3 on Nematodes</div>. <div class="source">J. Nematol</div> ., <div class="volume">38</div>, <div class="fpage">233</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Influence%20of%20Lysobacter%20enzymogenes%20strain%20C3%20on%20Nematodes&amp;author=J.%20Chen&amp;author=W.%20H.%20Moore&amp;author=G.%20Y.%20Yuen&amp;author=D.%20Kobayashi&amp;author=E.%20P.%20Caswell-Chen&amp;publication_year=2006&amp;journal=J.%20Nematol&amp;volume=38&amp;pages=233-9" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19259452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mercer</div> <div class="given-names">C. F.</div></div>, <div class="name string-name"><div class="surname">Greenwood</div> <div class="given-names">D. R.</div></div>, <div class="name string-name"><div class="surname">Grant</div> <div class="given-names">J. L.</div></div></div> <div class="year">1992</div>. 
<div class="article-title">Effect of plant and microbial chitinases on the eggs and juveniles of <em>Meloidogyne hapla</em> Chitiwood</div>. <div class="source">Nematologica</div>  <div class="volume">38</div>, <div class="fpage">227</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effect%20of%20plant%20and%20microbial%20chitinases%20on%20the%20eggs%20and%20juveniles%20of%20Meloidogyne%20hapla%20Chitiwood&amp;author=C.%20F.%20Mercer&amp;author=D.%20R.%20Greenwood&amp;author=J.%20L.%20Grant&amp;publication_year=1992&amp;journal=Nematologica&amp;volume=38&amp;pages=227-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1163/187529292X00199" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1163%2F187529292X00199" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1163%2F187529292X00199"> </span></div> </div></div></div></div></div><div content-id="dsw055-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Yuen</div> <div class="given-names">G. Y.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">The role of chitinase production by <em>Stenotrophomonas maltophilia</em> strain C3 in biological control of <em>Bipolaris sorokiniana</em></div>. <div class="source">Phytopathology</div>  <div class="volume">90</div>, <div class="fpage">384</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20chitinase%20production%20by%20Stenotrophomonas%20maltophilia%20strain%20C3%20in%20biological%20control%20of%20Bipolaris%20sorokiniana&amp;author=Z.%20Zhang&amp;author=G.%20Y.%20Yuen&amp;publication_year=2000&amp;journal=Phytopathology&amp;volume=90&amp;pages=384-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/PHYTO.2000.90.4.384" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FPHYTO.2000.90.4.384" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FPHYTO.2000.90.4.384"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18944588" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Yuen</div> <div class="given-names">G. Y.</div></div>, <div class="name string-name"><div class="surname">Sarath</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Penheiter</div> <div class="given-names">A.R.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Chitinases from the plant disease biocontrol agent, <em>Stenotrophomonas maltophilia</em> C3</div>. <div class="source">Phytopathology</div>  <div class="volume">91</div>, <div class="fpage">204</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chitinases%20from%20the%20plant%20disease%20biocontrol%20agent%2C%20Stenotrophomonas%20maltophilia%20C3&amp;author=Z.%20Zhang&amp;author=G.%20Y.%20Yuen&amp;author=G.%20Sarath&amp;author=A.R.%20Penheiter&amp;publication_year=2001&amp;journal=Phytopathology&amp;volume=91&amp;pages=204-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1094/PHYTO.2001.91.2.204" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1094%2FPHYTO.2001.91.2.204" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1094%2FPHYTO.2001.91.2.204"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18944395" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kobayashi</div> <div class="given-names">D.Y.</div></div>, <div class="name string-name"><div class="surname">El-Barrad</div> <div class="given-names">N.E.-H.</div></div></div> <div class="year">1996</div>, 
<div class="article-title">Selection of bacterial antagonists using enrichment cultures for the control of summer patch disease in Kentucky Bluegrass</div>. <div class="source">Cur. Microbiol.</div>  <div class="volume">32</div>, <div class="fpage">106</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20of%20bacterial%20antagonists%20using%20enrichment%20cultures%20for%20the%20control%20of%20summer%20patch%20disease%20in%20Kentucky%20Bluegrass&amp;author=D.Y.%20Kobayashi&amp;author=N.E.-H.%20El-Barrad&amp;publication_year=1996&amp;journal=Cur.%20Microbiol.&amp;volume=32&amp;pages=106-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002849900019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002849900019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002849900019"> </span></div> </div></div></div></div></div><div content-id="dsw055-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Palmubo</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Sullivan</div> <div class="given-names">R.F.</div></div>, <div class="name string-name"><div class="surname">Kobayashi</div> <div class="given-names">D.Y.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Molecular characterization and expression in <em>Escherichia coli</em> of three β-1,3-glucanase genes from <em>Lysobacter enzymogenes</em> strain N4-7</div>. <div class="source">J. Bacteriol</div> ., <div class="volume">185</div>, <div class="fpage">4362</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20and%20expression%20in%20Escherichia%20coli%20of%20three%20%CE%B2-1%2C3-glucanase%20genes%20from%20Lysobacter%20enzymogenes%20strain%20N4-7&amp;author=J.D.%20Palmubo&amp;author=R.F.%20Sullivan&amp;author=D.Y.%20Kobayashi&amp;publication_year=2003&amp;journal=J.%20Bacteriol&amp;volume=185&amp;pages=4362-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.185.15.4362-4370.2003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.185.15.4362-4370.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.185.15.4362-4370.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12867444" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hashizume</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Igarashi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hattori</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hori</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hamada</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Takeuchi</div> <div class="given-names">T.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Tripropeptins, novel antimicrobial agents produced by <em>Lysobacter</em> sp., I. Taxonomy, isolation and biological activities</div>. <div class="source">J. Antibiotics</div>  <div class="volume">54</div>, <div class="fpage">1054</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tripropeptins%2C%20novel%20antimicrobial%20agents%20produced%20by%20Lysobacter%20sp.%2C%20I.%20Taxonomy%2C%20isolation%20and%20biological%20activities&amp;author=H.%20Hashizume&amp;author=M.%20Igarashi&amp;author=S.%20Hattori&amp;author=M.%20Hori&amp;author=M.%20Hamada&amp;author=T.%20Takeuchi&amp;publication_year=2001&amp;journal=J.%20Antibiotics&amp;volume=54&amp;pages=1054-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.7164/antibiotics.54.1054" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.7164%2Fantibiotics.54.1054" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.7164%2Fantibiotics.54.1054"> </span></div> </div></div></div></div></div><div content-id="dsw055-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hashizume</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Igarashi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Sawa</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Adachi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Nishimura</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Akamatsu</div> <div class="given-names">Y.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">A new type of tripropeptin with anteiso-branched chain fatty acid from <em>Lysobacter</em> sp. BMK333-48F3</div>. <div class="source">J. Antibiotics</div>  <div class="volume">61</div>, <div class="fpage">577</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20type%20of%20tripropeptin%20with%20anteiso-branched%20chain%20fatty%20acid%20from%20Lysobacter%20sp.%20BMK333-48F3&amp;author=H.%20Hashizume&amp;author=M.%20Igarashi&amp;author=R.%20Sawa&amp;author=H.%20Adachi&amp;author=Y.%20Nishimura&amp;author=Y.%20Akamatsu&amp;publication_year=2008&amp;journal=J.%20Antibiotics&amp;volume=61&amp;pages=577-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ja.2008.78" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fja.2008.78" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fja.2008.78"> </span></div> </div></div></div></div></div><div content-id="dsw055-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hamamoto</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Urai</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ishii</div> <div class="given-names">K.</div></div></div>, et al.  . <div class="year">2015</div>. 
<div class="article-title">Lysocin E is a new antibiotic that targets menaquinone in the bacterial membrane</div>. <div class="source">Nat. Chem. Biol.</div>  <div class="volume">11</div>, <div class="fpage">127</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lysocin%20E%20is%20a%20new%20antibiotic%20that%20targets%20menaquinone%20in%20the%20bacterial%20membrane&amp;author=H.%20Hamamoto&amp;author=M.%20Urai&amp;author=K.%20Ishii&amp;publication_year=2015&amp;journal=Nat.%20Chem.%20Biol.&amp;volume=11&amp;pages=127-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nchembio.1710" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnchembio.1710" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnchembio.1710"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25485686" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">de Brujin</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">de Jager</div> <div class="given-names">V.</div></div></div>, et al.  . <div class="year">2015</div>. 
<div class="article-title">Comparative genomics and metabolic profiling of the genus <em>Lysobacter</em></div>. <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">991</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20and%20metabolic%20profiling%20of%20the%20genus%20Lysobacter&amp;author=I.%20de%20Brujin&amp;author=X.%20Cheng&amp;author=V.%20de%20Jager&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=991" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-2191-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-2191-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-2191-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26597042" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gordon</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Desmarais</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Green</div> <div class="given-names">P.</div></div></div> <div class="year">2001</div>. 
<div class="article-title">Automated finishing with autofinish</div>. <div class="source">Genome Res.</div>  <div class="volume">11</div>, <div class="fpage">614</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20finishing%20with%20autofinish&amp;author=D.%20Gordon&amp;author=C.%20Desmarais&amp;author=P.%20Green&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=614-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.171401" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.171401" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.171401"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11282977" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jong</div> <div class="given-names">A. D.</div></div>, <div class="name string-name"><div class="surname">Pietersma</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Cordes</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kuipers</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Kok</div> <div class="given-names">J.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">PePPER: a webserver for prediction of prokaryote promoter elements and regulons</div>. <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">299</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PePPER%3A%20a%20webserver%20for%20prediction%20of%20prokaryote%20promoter%20elements%20and%20regulons&amp;author=A.%20D.%20Jong&amp;author=H.%20Pietersma&amp;author=M.%20Cordes&amp;author=O.%20Kuipers&amp;author=J.%20Kok&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=299" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-299" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-299" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-299"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22747501" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rawlings</div> <div class="given-names">N. D.</div></div>, <div class="name string-name"><div class="surname">Waller</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Barrett</div> <div class="given-names">A. J.</div></div>, <div class="name string-name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div></div> <div class="year">2014</div>, 
<div class="article-title"><em>MEROPS</em>: the database of proteolytic enzymes, their substrates and inhibitors</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">D503</div>–<div class="lpage">509</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEROPS%3A%20the%20database%20of%20proteolytic%20enzymes%2C%20their%20substrates%20and%20inhibitors&amp;author=N.%20D.%20Rawlings&amp;author=M.%20Waller&amp;author=A.%20J.%20Barrett&amp;author=A.%20Bateman&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=D503-509" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt953" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt953" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt953"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24157837" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Uchiyama</div> <div class="given-names">I.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Hierarchical clustering algorithm for comprehensive orthologous-domain classification in multiple genomes</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">647</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hierarchical%20clustering%20algorithm%20for%20comprehensive%20orthologous-domain%20classification%20in%20multiple%20genomes&amp;author=I.%20Uchiyama&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=647-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj448" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj448" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj448"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16436801" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Uchiyama</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Mihara</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Nishide</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Chiba</div> <div class="given-names">H.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">MBGD update 2015: microbial genome database for flexible ortholog analysis utilizing a diverse set of genomic data</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">D270</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MBGD%20update%202015%3A%20microbial%20genome%20database%20for%20flexible%20ortholog%20analysis%20utilizing%20a%20diverse%20set%20of%20genomic%20data&amp;author=I.%20Uchiyama&amp;author=M.%20Mihara&amp;author=H.%20Nishide&amp;author=H.%20Chiba&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=D270-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25398900" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Uchiyama</div> <div class="given-names">I.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Multiple genome alignment for identifying the core structure among moderately related microbial genomes</div>. <div class="source">BMC Genomics</div>  <div class="volume">9</div>, <div class="fpage">515</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20genome%20alignment%20for%20identifying%20the%20core%20structure%20among%20moderately%20related%20microbial%20genomes&amp;author=I.%20Uchiyama&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=515" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-515" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-515" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-515"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18976470" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ansari</div> <div class="given-names">M.Z.</div></div>, <div class="name string-name"><div class="surname">Yadav</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Gokhale</div> <div class="given-names">R.S.</div></div>, <div class="name string-name"><div class="surname">Mohanty</div> <div class="given-names">D.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">NRPS-PKS: a knowledge-based resource for analysis of NRPS/PKS megasynthases</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">32</div>, <div class="fpage">W405</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NRPS-PKS%3A%20a%20knowledge-based%20resource%20for%20analysis%20of%20NRPS%2FPKS%20megasynthases&amp;author=M.Z.%20Ansari&amp;author=G.%20Yadav&amp;author=R.S.%20Gokhale&amp;author=D.%20Mohanty&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=W405-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh359" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh359" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh359"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15215420" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tae</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kong</div> <div class="given-names">E.B.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">K.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">ASMPKS: an analysis system for modular polyketide synthases</div>. <div class="source">BMC Bioinformatics.</div>  <div class="volume">8</div>, <div class="fpage">327</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ASMPKS%3A%20an%20analysis%20system%20for%20modular%20polyketide%20synthases&amp;author=H.%20Tae&amp;author=E.B.%20Kong&amp;author=K.%20Park&amp;publication_year=2007&amp;journal=BMC%20Bioinformatics.&amp;volume=8&amp;pages=327" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-8-327" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-8-327" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-8-327"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17764579" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bachmann</div> <div class="given-names">B.O.</div></div>, <div class="name string-name"><div class="surname">Ravel</div> <div class="given-names">J.</div></div></div>. <div class="year">2009</div>, 
<div class="article-title">Methods for <em>in silico</em> prediction of microbial secondary metabolic pathways from DNA sequence data</div>. <div class="source">Methods Enzymol</div> ., <div class="volume">458</div>, <div class="fpage">181</div>–<div class="lpage">217</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Methods%20for%20in%20silico%20prediction%20of%20microbial%20secondary%20metabolic%20pathways%20from%20DNA%20sequence%20data&amp;author=B.O.%20Bachmann&amp;author=J.%20Ravel&amp;publication_year=2009&amp;journal=Methods%20Enzymol&amp;volume=458&amp;pages=181-217" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19374984" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Medema</div> <div class="given-names">M.H.</div></div>, <div class="name string-name"><div class="surname">Blin</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Cimermancic</div> <div class="given-names">P.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">antiSMASH: rapid identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genome sequences</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">W339</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=antiSMASH%3A%20rapid%20identification%2C%20annotation%20and%20analysis%20of%20secondary%20metabolite%20biosynthesis%20gene%20clusters%20in%20bacterial%20and%20fungal%20genome%20sequences&amp;author=M.H.%20Medema&amp;author=K.%20Blin&amp;author=P.%20Cimermancic&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=W339-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr466" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr466" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr466"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21672958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Takami</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Taniguchi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Arai</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Takemoto</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Moriya</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Goto</div> <div class="given-names">S.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">An automated system for evaluation of the potential functionome: MAPLE version 2.1.0</div>. <div class="source">DNA Res</div> ., <strong>23</strong>, 467–475.</div></div></div></div><div content-id="dsw055-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Takami</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Taniguchi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Moriya</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kuwahara</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Goto</div> <div class="given-names">S.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Evaluation method for the potential functionome harbored in the genome and metagenome</div>. <div class="source">BMC Genomics</div> , <div class="volume">13</div>, <div class="fpage">699</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20method%20for%20the%20potential%20functionome%20harbored%20in%20the%20genome%20and%20metagenome&amp;author=H.%20Takami&amp;author=T.%20Taniguchi&amp;author=Y.%20Moriya&amp;author=T.%20Kuwahara&amp;author=M.%20Kanehisa&amp;author=S.%20Goto&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=699" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-699"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23234305" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Braun</div> <div class="given-names">V.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Avoidance of iron toxicity through regulation of bacterial iron transport</div>. <div class="source">Biol. Chem.</div>  <div class="volume">378</div>, <div class="fpage">779</div>–<div class="lpage">786</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Avoidance%20of%20iron%20toxicity%20through%20regulation%20of%20bacterial%20iron%20transport&amp;author=V.%20Braun&amp;publication_year=1997&amp;journal=Biol.%20Chem.&amp;volume=378&amp;pages=779-786" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9377472" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Noinaji</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Guillier</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Barnard</div> <div class="given-names">T.J.</div></div>, <div class="name string-name"><div class="surname">Buchanan</div> <div class="given-names">S.K.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">TonB-dependent transporters: regulation, structure, and function</div>. <div class="source">Annu. Rev. Microbiol.</div>  <div class="volume">64</div>, <div class="fpage">43</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TonB-dependent%20transporters%3A%20regulation%2C%20structure%2C%20and%20function&amp;author=N.%20Noinaji&amp;author=M.%20Guillier&amp;author=T.J.%20Barnard&amp;author=S.K.%20Buchanan&amp;publication_year=2010&amp;journal=Annu.%20Rev.%20Microbiol.&amp;volume=64&amp;pages=43-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.micro.112408.134247" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.micro.112408.134247" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.micro.112408.134247"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20420522" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sonenshein</div> <div class="given-names">A.L.</div></div>, <div class="name string-name"><div class="surname">Hoch</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Losic</div> <div class="given-names">R</div></div></div> (eds.), <div class="source">Bacillus subtilis and Other Gram-Positive Bacteria</div> . 
<div class="publisher-name">ASM Press</div>, 
<div class="publisher-loc">Washington, DC</div>.</div></div></div></div><div content-id="dsw055-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Walsh</div> <div class="given-names">C.T.</div></div></div> <div class="year">2003</div>, <div class="source">Antibiotics; Actions, Origins, Resistance</div> . 
<div class="publisher-name">ASM Press</div>, 
<div class="publisher-loc">Washington, DC</div>.</div></div></div></div><div content-id="dsw055-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xie</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wright</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Shen</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Du</div> <div class="given-names">L.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Bioactive natural products from <em>Lysobacter</em></div>. <div class="source">Nat. Prod. Pep</div> . <div class="volume">29</div>, <div class="fpage">1277</div>–<div class="lpage">87</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioactive%20natural%20products%20from%20Lysobacter&amp;author=Y.%20Xie&amp;author=S.%20Wright&amp;author=Y.%20Shen&amp;author=L.%20Du&amp;publication_year=2012&amp;journal=Nat.%20Prod.%20Pep&amp;volume=29&amp;pages=1277-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1039/c2np20064c" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1039%2Fc2np20064c" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1039%2Fc2np20064c"> </span></div> </div></div></div></div></div><div content-id="dsw055-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hou</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Robbel</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Marahiel</div> <div class="given-names">M.A.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Identification and characterization of the lysobactin biosynthetic gene cluster reveals mechanistic insights into an unusual termination module architecture</div>. <div class="source">Chem. Biol.</div>  <div class="volume">18</div>, <div class="fpage">655</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20the%20lysobactin%20biosynthetic%20gene%20cluster%20reveals%20mechanistic%20insights%20into%20an%20unusual%20termination%20module%20architecture&amp;author=J.%20Hou&amp;author=L.%20Robbel&amp;author=M.A.%20Marahiel&amp;publication_year=2011&amp;journal=Chem.%20Biol.&amp;volume=18&amp;pages=655-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.chembiol.2011.02.012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.chembiol.2011.02.012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.chembiol.2011.02.012"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21609846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kobayashi</div> <div class="given-names">D.Y.</div></div>, <div class="name string-name"><div class="surname">Reedy</div> <div class="given-names">R.M.</div></div>, <div class="name string-name"><div class="surname">Palumbo</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">J.-M.</div></div>, <div class="name string-name"><div class="surname">Yuen</div> <div class="given-names">G.Y.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">A <em>clp</em> gene homologue belonging to the <em>crp</em> gene family globally regulates lytic enzyme production, antimicrobial activity, and biological control activity expressed by <em>Lysobacter enzymogenes</em> strain C3</div>. <div class="source">Appl. Environ. Microbiol.</div>  <div class="volume">71</div>, <div class="fpage">261</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20clp%20gene%20homologue%20belonging%20to%20the%20crp%20gene%20family%20globally%20regulates%20lytic%20enzyme%20production%2C%20antimicrobial%20activity%2C%20and%20biological%20control%20activity%20expressed%20by%20Lysobacter%20enzymogenes%20strain%20C3&amp;author=D.Y.%20Kobayashi&amp;author=R.M.%20Reedy&amp;author=J.D.%20Palumbo&amp;author=J.-M.%20Zhou&amp;author=G.Y.%20Yuen&amp;publication_year=2005&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=71&amp;pages=261-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.71.1.261-269.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.71.1.261-269.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.71.1.261-269.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15640196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhan</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">Transcriptomic analysis reveals new regulatory roles of Clp signaling in secondary metabolite synthesis and surface motility in <em>Lysobacter enzymogenes</em> OH11</div>. <div class="source">Appl. Microbiol. Biotechnol</div> ., <div class="volume">98</div>, <div class="fpage">9009</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptomic%20analysis%20reveals%20new%20regulatory%20roles%20of%20Clp%20signaling%20in%20secondary%20metabolite%20synthesis%20and%20surface%20motility%20in%20Lysobacter%20enzymogenes%20OH11&amp;author=Y.%20Wang&amp;author=Y.%20Zhao&amp;author=J.%20Zhan&amp;publication_year=2014&amp;journal=Appl.%20Microbiol.%20Biotechnol&amp;volume=98&amp;pages=9009-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-014-6072-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-014-6072-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-014-6072-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25236801" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw055-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw055-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Han</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tombosa</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Identification of a small molecule signaling factor that regulates the biosynthesis of the antifungal polycyclic tetramate macrolactam HSAF in <em>Lysobacter enzymogenes</em></div>. <div class="source">Appl. Microbiol. Biotechnol.</div>  <div class="volume">99</div>, <div class="fpage">801</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20small%20molecule%20signaling%20factor%20that%20regulates%20the%20biosynthesis%20of%20the%20antifungal%20polycyclic%20tetramate%20macrolactam%20HSAF%20in%20Lysobacter%20enzymogenes&amp;author=Y.%20Han&amp;author=Y.%20Wang&amp;author=S.%20Tombosa&amp;publication_year=2015&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=99&amp;pages=801-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00253-014-6120-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00253-014-6120-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00253-014-6120-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25301587" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2870503&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw055/2/dsw055_Supp.zip?Expires=2147483647&amp;Signature=zejfsocCkzML6xmBUfNWFjbzLdlcD0TFh5dpCn6Vw~Y1pfybaLcVtbrgcdTfgXdM9WAOCgidWmRgqcgpxCFsbYnhCdRaP~A4GTqwTyzSrbm1caa-aoSwJEyxGvddSpk0HcWHO3J4eOyVWDGlY5Q9EfK4ZtrMFkEfEb6ay9721yKeRS7okLm3TswOv~LylUdLVa1dj15gcmLX9PsY23XElJkaZjw7vo1kLteLq2PZWyjqulYHktmG4yxJjo-juSCyIMgvt--T04-YYFd6jcH8rcmp-EofM-5MRTuGiX7QOfzT6VxVHI7hu2XxmgSeeMluHJQqSnxP9EgTbSSOT4MKVQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">661</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw055"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2870503"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f2%2f169%2f2870503">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F2%2F169%2F2870503" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2870503'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Complete genome sequence and expression profile of the commercial lytic enzyme producer Lysobacter enzymogenes M497-1" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw055");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f169%2f2870503">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

