

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/6/357/513030","@type":"ScholarlyArticle","name":"Comparison of Correspondence Analysis Methods for Synonymous Codon Usage in Bacteria","datePublished":"2008-10-21","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2008-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsn028","keywords":["correspondence analysis","synonymous codon usage","horizontal gene transfer","strand-specific mutational bias","translational selection"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/6/357/513030","author":[{"name":"Suzuki, Haruo","affiliation":"Department of Biological Sciences and Initiative for Bioinformatics and Evolutionary Studies ,  University of Idaho ,  PO Box 443051, Moscow, Idaho 83844-3051","@type":"Person"},{"name":"Brown, Celeste J.","affiliation":"Department of Biological Sciences and Initiative for Bioinformatics and Evolutionary Studies ,  University of Idaho ,  PO Box 443051, Moscow, Idaho 83844-3051","@type":"Person"},{"name":"Forney, Larry J.","affiliation":"Department of Biological Sciences and Initiative for Bioinformatics and Evolutionary Studies ,  University of Idaho ,  PO Box 443051, Moscow, Idaho 83844-3051","@type":"Person"},{"name":"Top, Eva M.","affiliation":"Department of Biological Sciences and Initiative for Bioinformatics and Evolutionary Studies ,  University of Idaho ,  PO Box 443051, Moscow, Idaho 83844-3051","@type":"Person"}],"description":"Abstract.  Synonymous codon usage varies both between organisms and among genes within a genome, and arises due to differences in G + C content, replication str","pageStart":"357","pageEnd":"365","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/6/1/m_cover.gif?Expires=2147483647&Signature=cbcaSTnXmSAtcWAqInhnHikjrANtVVQT69rwVkS2N2-Rvvq~3o4NF54oja~GIUWptdFQr454edog~WN90FgAfPnwCCNM9uZ3DuEnj-WTN3JyklWsccNFHC6w0Mg4O1XsO2YSzFEaGsd~LWUMMgT-JRIdN1RQUYUQEdvCgKjLmwjDjkQDpQkgPjsYyRLj1QLXlpM4IsivMdAnqm9LmZlXnvMUUp5UVsqaQBEzVLg9CZ1N~86MZnaIY5LkZ1lOjOrEUjr78DNfXaNPxd2goUtJ24j8goSO16w~p5Co8ZBl8FnseAHBtje4N~aGVfSKUlSXEIVKmxpcUZsLAtBrli2ACg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Comparison of Correspondence Analysis Methods for Synonymous Codon Usage in Bacteria","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/6/1/m_cover.gif?Expires=2147483647&Signature=cbcaSTnXmSAtcWAqInhnHikjrANtVVQT69rwVkS2N2-Rvvq~3o4NF54oja~GIUWptdFQr454edog~WN90FgAfPnwCCNM9uZ3DuEnj-WTN3JyklWsccNFHC6w0Mg4O1XsO2YSzFEaGsd~LWUMMgT-JRIdN1RQUYUQEdvCgKjLmwjDjkQDpQkgPjsYyRLj1QLXlpM4IsivMdAnqm9LmZlXnvMUUp5UVsqaQBEzVLg9CZ1N~86MZnaIY5LkZ1lOjOrEUjr78DNfXaNPxd2goUtJ24j8goSO16w~p5Co8ZBl8FnseAHBtje4N~aGVfSKUlSXEIVKmxpcUZsLAtBrli2ACg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Comparison of Correspondence Analysis Methods for Synonymous Codon Usage in Bacteria" />
    <meta property="og:description" content="Abstract.  Synonymous codon usage varies both between organisms and among genes within a genome, and arises due to differences in G + C content, replication str" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/6/357/513030" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-10-21" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Suzuki, Haruo" /><meta name="citation_author_institution" content="Department of Biological Sciences and Initiative for Bioinformatics and Evolutionary Studies, University of Idaho, PO Box 443051, Moscow, Idaho 83844-3051" /><meta name="citation_author" content="Brown, Celeste J." /><meta name="citation_author_institution" content="Department of Biological Sciences and Initiative for Bioinformatics and Evolutionary Studies, University of Idaho, PO Box 443051, Moscow, Idaho 83844-3051" /><meta name="citation_author" content="Forney, Larry J." /><meta name="citation_author_institution" content="Department of Biological Sciences and Initiative for Bioinformatics and Evolutionary Studies, University of Idaho, PO Box 443051, Moscow, Idaho 83844-3051" /><meta name="citation_author" content="Top, Eva M." /><meta name="citation_author_institution" content="Department of Biological Sciences and Initiative for Bioinformatics and Evolutionary Studies, University of Idaho, PO Box 443051, Moscow, Idaho 83844-3051" /><meta name="citation_title" content="Comparison of Correspondence Analysis Methods for Synonymous Codon Usage in Bacteria" /><meta name="citation_firstpage" content="357" /><meta name="citation_lastpage" content="365" /><meta name="citation_doi" content="10.1093/dnares/dsn028" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2008/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Codon usage patterns in Escherichia coli, Bacillus subtilis, Saccharomyces cerevisiae, Schizosaccharomyces pombe, Drosophila melanogaster and Homo sapiens; a review of the considerable within-species diversity; Nucleic Acids Res.;  citation_year=1988;  citation_volume=16;  citation_pages=8207-8211; " /><meta name="citation_reference" content="citation_title=Detecting anomalous gene clusters and pathogenicity islands in diverse bacterial genomes; Trends Microbiol.;  citation_year=2001;  citation_volume=9;  citation_pages=335-343; " /><meta name="citation_reference" content="citation_title=Amelioration of bacterial genomes: rates of change and exchange; J. Mol. Evol.;  citation_year=1997;  citation_volume=44;  citation_pages=383-397; " /><meta name="citation_reference" content="citation_title=Lateral gene transfer and the nature of bacterial innovation; Nature;  citation_year=2000;  citation_volume=405;  citation_pages=299-304; " /><meta name="citation_reference" content="citation_title=Asymmetric substitution patterns in the two DNA strands of bacteria; Mol. Biol. Evol.;  citation_year=1996;  citation_volume=13;  citation_pages=660-665; " /><meta name="citation_reference" content="citation_title=Base composition skews, replication orientation, and gene orientation in 12 prokaryote genomes; J. Mol. Evol.;  citation_year=1998;  citation_volume=47;  citation_pages=691-696; " /><meta name="citation_reference" content="citation_title=Synonymous codon bias is related to gene length in Escherichia coli: selection for translational accuracy?; Mol Biol Evol;  citation_year=1996;  citation_volume=13;  citation_pages=864-872; " /><meta name="citation_reference" content="citation_title=Codon usage and tRNA content in unicellular and multicellular organisms; Mol Biol Evol;  citation_year=1985;  citation_volume=2;  citation_pages=13-34; " /><meta name="citation_reference" content="citation_title=Codon bias signatures, organization of microorganisms in codon space, and lifestyle; Mol. Biol. Evol.;  citation_year=2005;  citation_volume=22;  citation_pages=547-561; " /><meta name="citation_reference" content="citation_title=Variation in the strength of selected codon usage bias among bacteria; Nucleic Acids Res.;  citation_year=2005;  citation_volume=33;  citation_pages=1141-1153; " /><meta name="citation_reference" content="citation_title=An environmental signature for 323 microbial genomes based on codon adaptation indices; Genome Biol.;  citation_year=2006;  citation_volume=7;  citation_pages=R114" /><meta name="citation_reference" content="citation_title=HGT-DB: a database of putative horizontally transferred genes in prokaryotic complete genomes; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=187-189; " /><meta name="citation_reference" content="citation_title=Codon bias and base composition are poor indicators of horizontally transferred genes; Mol. Biol. Evol.;  citation_year=2001;  citation_volume=18;  citation_pages=404-412; " /><meta name="citation_reference" content="citation_title=Detecting alien genes in bacterial genomes; Ann. N Y Acad. Sci.;  citation_year=1999;  citation_volume=870;  citation_pages=314-329; " /><meta name="citation_reference" content="citation_title=Predicting gene expression level from codon usage bias; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=10-12; " /><meta name="citation_reference" content="citation_title=Characterizations of highly expressed genes of four fast-growing bacteria; J. Bacteriol.;  citation_year=2001;  citation_volume=183;  citation_pages=5025-5040; " /><meta name="citation_reference" content="citation_title=HEG-DB: a database of predicted highly expressed genes in prokaryotic complete genomes under translational selection; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D524-D527; " /><meta name="citation_reference" content="citation_title=The codon adaptation index – a measure of directional synonymous codon usage bias, and its potential applications; Nucleic Acids Res.;  citation_year=1987;  citation_volume=15;  citation_pages=1281-1295; " /><meta name="citation_reference" content="citation_title=Correspondence analysis applied to microarray data; Proc. Natl Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=10781-10786; " /><meta name="citation_reference" content="citation_title=Codon catalog usage and the genome hypothesis; Nucleic Acids Res.;  citation_year=1980;  citation_volume=8;  citation_pages=r49-r62; " /><meta name="citation_reference" content="citation_title=Use and misuse of correspondence analysis in codon usage studies; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=4548-4555; " /><meta name="citation_reference" content="citation_title=Trends in codon and amino acid usage in Thermotoga maritima; J. Mol. Evol.;  citation_year=2002;  citation_volume=54;  citation_pages=563-568; " /><meta name="citation_reference" content="citation_title=Unexpected correlations between gene expression and codon usage bias from microarray data for the whole Escherichia coli K-12 genome; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=6976-6985; " /><meta name="citation_reference" content="citation_title=Codon usage and the origin of P elements; Mol. Biol. Evol.;  citation_year=2000;  citation_volume=17;  citation_pages=467-468; " /><meta name="citation_reference" content="citation_title=Online synonymous codon usage analyses with the ade4 and seqinR packages; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=545-547; " /><meta name="citation_reference" content="citation_title=Codon and amino acid usage in two major human pathogens of genus Bartonella – optimization between replicational–transcriptional selection, translational control and cost minimization; DNA Res.;  citation_year=2005;  citation_volume=12;  citation_pages=91-102; " /><meta name="citation_reference" content="citation_title=Temperature adaptation of synonymous codon usage in different functional categories of genes: a comparative study between homologous genes of Methanococcus jannaschii and Methanococcus maripaludis; FEBS Lett.;  citation_year=2006;  citation_volume=580;  citation_pages=3895-3899; " /><meta name="citation_reference" content="citation_title=Analysis of Nanoarchaeum equitans genome and proteome composition: indications for hyperthermophilic and parasitic adaptation; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=186" /><meta name="citation_reference" content="citation_title=Evolutionary constraints on codon and amino acid usage in two strains of human pathogenic actinobacteria Tropheryma whipplei; J. Mol. Evol.;  citation_year=2006;  citation_volume=62;  citation_pages=645-658; " /><meta name="citation_reference" content="citation_title=Synonymous codon usage in adenoviruses: influence of mutation, selection and protein hydropathy; Virus Res.;  citation_year=2006;  citation_volume=117;  citation_pages=227-236; " /><meta name="citation_reference" content="citation_title=Comparative study of the hemagglutinin and neuraminidase genes of influenza A virus H3N2, H9N2, and H5N1 subtypes using bioinformatics techniques; Can. J. Microbiol.;  citation_year=2007;  citation_volume=53;  citation_pages=830-839; " /><meta name="citation_reference" content="citation_title=Evaluation of codon bias perspectives in phage therapy of Mycobacterium tuberculosis by multivariate analysis; In Silico Biol.;  citation_year=2007;  citation_volume=7;  citation_pages=0030" /><meta name="citation_reference" content="citation_title=Studies on synonymous codon and amino acid usage biases in the broad-host range bacteriophage KVP40; J. Microbiol.;  citation_year=2007;  citation_volume=45;  citation_pages=58-63; " /><meta name="citation_reference" content="citation_title=Analysis of codon usage patterns and predicted highly expressed genes for six phytopathogenic Xanthomonas genomes shows a high degree of conservation; In Silico Biol.;  citation_year=2007;  citation_volume=7;  citation_pages=547-558; " /><meta name="citation_reference" content="citation_title=Rapid divergence of codon usage patterns within the rice genome; BMC Evol. Biol.;  citation_year=2007;  citation_volume=7;  citation_pages=S6" /><meta name="citation_reference" content="citation_title=Mutation pressure shapes codon usage in the GC-rich genome of foot-and-mouth disease virus; Virus Genes;  citation_year=2007;  citation_volume=35;  citation_pages=767-776; " /><meta name="citation_reference" content="citation_title=Analysis of synonymous codon usage in 11 Human Bocavirus isolates; Biosystems;  citation_year=2008;  citation_volume=92;  citation_pages=207-214; " /><meta name="citation_reference" content="citation_title=GenBank; Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=D25-D30; " /><meta name="citation_reference" content="citation_title=Proteome composition and codon usage in spirochaetes: species-specific and DNA strand-specific mutational biases; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=1642-1649; " /><meta name="citation_reference" content="citation_title=Replicational and transcriptional selection on codon usage in Borrelia burgdorferi; Proc. Natl Acad. Sci. USA;  citation_year=1998;  citation_volume=95;  citation_pages=10698-106703; " /><meta name="citation_reference" content="citation_title=Codon usage in Chlamydia trachomatis is the result of strand-specific mutational biases and a complex pattern of selective forces; Nucleic Acids Res.;  citation_year=2000;  citation_volume=28;  citation_pages=2084-2090; " /><meta name="citation_reference" content="citation_title=Translational selection is operative for synonymous codon usage in Clostridium perfringens and Clostridium acetobutylicum; Microbiology;  citation_year=2003;  citation_volume=149;  citation_pages=855-863; " /><meta name="citation_reference" content="citation_title=Evidence for horizontal gene transfer in Escherichia coli speciation; J. Mol. Biol.;  citation_year=1991;  citation_volume=222;  citation_pages=851-856; " /><meta name="citation_reference" content="citation_title=On-line tools for sequence retrieval and multivariate statistics in molecular biology; Comput. Appl. Biosci.;  citation_year=1996;  citation_volume=12;  citation_pages=63-69; " /><meta name="citation_reference" content="citation_title=Absence of translationally selected synonymous codon usage bias in Helicobacter pylori; Microbiology;  citation_year=2000;  citation_volume=146;  citation_issue=Pt 4;  citation_pages=851-860; " /><meta name="citation_reference" content="citation_title=Prokaryotic genome evolution as assessed by multivariate analysis of codon usage patterns; Microbiol Comp. Genomics;  citation_year=1997;  citation_volume=2;  citation_pages=89-97; " /><meta name="citation_reference" content="citation_title=Codon usage and base composition in Rickettsia prowazekii; J. Mol. Evol.;  citation_year=1996;  citation_volume=42;  citation_pages=525-536; " /><meta name="citation_reference" content="citation_title=Mutational and selective pressures on codon and amino acid usage in Buchnera, endosymbiotic bacteria of aphids; Genome Res.;  citation_year=2004;  citation_volume=14;  citation_pages=44-53; " /><meta name="citation_reference" content="citation_title=Codon usage in yeast: cluster analysis clearly differentiates highly and lowly expressed genes; Nucleic Acids Res.;  citation_year=1986;  citation_volume=14;  citation_pages=5125-5143; " /><meta name="citation_reference" content="citation_title=ADE-4: a multivariate analysis and graphical display software; Stat. Comput.;  citation_year=1997;  citation_volume=7;  citation_pages=75-83; " /><meta name="citation_reference" content="citation_title=R: a language and environment for statistical computing, R Foundation for Statistical Computing; citation_author=R Development Core Team.; citation_publisher=Vienna, Austria;  citation_year=2007; " /><meta name="citation_reference" content="citation_title=Synonymous codon usage in Pseudomonas aeruginosa PA01; Gene;  citation_year=2002;  citation_volume=289;  citation_pages=131-139; " /><meta name="citation_reference" content="citation_title=A simple method for displaying the hydropathic character of a protein; J. Mol. Biol.;  citation_year=1982;  citation_volume=157;  citation_pages=105-132; " /><meta name="citation_reference" content="citation_title=A problem in multivariate analysis of codon usage data and a possible solution; FEBS Lett;  citation_year=2005;  citation_volume=579;  citation_pages=6499-504; " /><meta name="citation_reference" content="citation_title=Hydrophobicity, expressivity and aromaticity are the major trends of amino-acid usage in 999 Escherichia coli chromosome-encoded genes; Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=3174-3180; " /><meta name="citation_reference" content="citation_title=Global analysis of protein expression in yeast; Nature;  citation_year=2003;  citation_volume=425;  citation_pages=737-741; " /><meta name="citation_reference" content="citation_title=Context-dependent codon bias and messenger RNA longevity in the yeast transcriptome; Mol. Biol. Evol.;  citation_year=2005;  citation_volume=22;  citation_pages=1403-1411; " /><meta name="citation_reference" content="citation_title=Systematic base composition variation around the genome of Mycoplasma genitalium, but not Mycoplasma pneumoniae; Mol. Microbiol.;  citation_year=1997;  citation_volume=25;  citation_pages=1177-1179; " /><meta name="citation_reference" content="citation_title=G-language genome analysis environment: a workbench for nucleotide sequence data mining; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=305-306; " /><meta name="citation_reference" content="citation_title=G-language system as a platform for large-scale analysis of high-throughput omics data; J. Pesticide Sci.;  citation_year=2006;  citation_volume=31;  citation_pages=282-288; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/6/357/1376886/dsn028.pdf" /><meta name="description" content="Abstract.  Synonymous codon usage varies both between organisms and among genes within a genome, and arises due to differences in G + C content, replication str" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/6/357/513030" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/6/357/513030" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/6/357/513030");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsn028';
    </script>
<title>
	Comparison of Correspondence Analysis Methods for Synonymous Codon Usage in Bacteria | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./513030" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbV/PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdXp1a2ksIEhhcnVvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzIGFuZCBJbml0aWF0aXZlIGZvciBCaW9pbmZvcm1hdGljcyBhbmQgRXZvbHV0aW9uYXJ5IFN0dWRpZXMsIFVuaXZlcnNpdHkgb2YgSWRhaG8sIFBPIEJveCA0NDMwNTEsIE1vc2NvdywgSWRhaG8gODM4NDQtMzA1MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJyb3duLCBDZWxlc3RlIEouIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW9sb2dpY2FsIFNjaWVuY2VzIGFuZCBJbml0aWF0aXZlIGZvciBCaW9pbmZvcm1hdGljcyBhbmQgRXZvbHV0aW9uYXJ5IFN0dWRpZXMsIFVuaXZlcnNpdHkgb2YgSWRhaG8sIFBPIEJveCA0NDMwNTEsIE1vc2NvdywgSWRhaG8gODM4NDQtMzA1MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZvcm5leSwgTGFycnkgSi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIEluaXRpYXRpdmUgZm9yIEJpb2luZm9ybWF0aWNzIGFuZCBFdm9sdXRpb25hcnkgU3R1ZGllcywgVW5pdmVyc2l0eSBvZiBJZGFobywgUE8gQm94IDQ0MzA1MSwgTW9zY293LCBJZGFobyA4Mzg0NC0zMDUxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVG9wLCBFdmEgTS4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMgYW5kIEluaXRpYXRpdmUgZm9yIEJpb2luZm9ybWF0aWNzIGFuZCBFdm9sdXRpb25hcnkgU3R1ZGllcywgVW5pdmVyc2l0eSBvZiBJZGFobywgUE8gQm94IDQ0MzA1MSwgTW9zY293LCBJZGFobyA4Mzg0NC0zMDUxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJDb21wYXJpc29uIG9mIENvcnJlc3BvbmRlbmNlIEFuYWx5c2lzIE1ldGhvZHMgZm9yIFN5bm9ueW1vdXMgQ29kb24gVXNhZ2UgaW4gQmFjdGVyaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzNTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjM2NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbjAyOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA4LzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgcGF0dGVybnMgaW4gRXNjaGVyaWNoaWEgY29saSwgQmFjaWxsdXMgc3VidGlsaXMsIFNhY2NoYXJvbXljZXMgY2VyZXZpc2lhZSwgU2NoaXpvc2FjY2hhcm9teWNlcyBwb21iZSwgRHJvc29waGlsYSBtZWxhbm9nYXN0ZXIgYW5kIEhvbW8gc2FwaWVuczsgYSByZXZpZXcgb2YgdGhlIGNvbnNpZGVyYWJsZSB3aXRoaW4tc3BlY2llcyBkaXZlcnNpdHk7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTgyMDctODIxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW5nIGFub21hbG91cyBnZW5lIGNsdXN0ZXJzIGFuZCBwYXRob2dlbmljaXR5IGlzbGFuZHMgaW4gZGl2ZXJzZSBiYWN0ZXJpYWwgZ2Vub21lczsgVHJlbmRzIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTMzNS0zNDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFtZWxpb3JhdGlvbiBvZiBiYWN0ZXJpYWwgZ2Vub21lczogcmF0ZXMgb2YgY2hhbmdlIGFuZCBleGNoYW5nZTsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTM4My0zOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhdGVyYWwgZ2VuZSB0cmFuc2ZlciBhbmQgdGhlIG5hdHVyZSBvZiBiYWN0ZXJpYWwgaW5ub3ZhdGlvbjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTQwNTsgIGNpdGF0aW9uX3BhZ2VzPTI5OS0zMDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzeW1tZXRyaWMgc3Vic3RpdHV0aW9uIHBhdHRlcm5zIGluIHRoZSB0d28gRE5BIHN0cmFuZHMgb2YgYmFjdGVyaWE7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz02NjAtNjY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNlIGNvbXBvc2l0aW9uIHNrZXdzLCByZXBsaWNhdGlvbiBvcmllbnRhdGlvbiwgYW5kIGdlbmUgb3JpZW50YXRpb24gaW4gMTIgcHJva2FyeW90ZSBnZW5vbWVzOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9NjkxLTY5NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lub255bW91cyBjb2RvbiBiaWFzIGlzIHJlbGF0ZWQgdG8gZ2VuZSBsZW5ndGggaW4gRXNjaGVyaWNoaWEgY29saTogc2VsZWN0aW9uIGZvciB0cmFuc2xhdGlvbmFsIGFjY3VyYWN5PzsgTW9sIEJpb2wgRXZvbDsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTg2NC04NzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGFuZCB0Uk5BIGNvbnRlbnQgaW4gdW5pY2VsbHVsYXIgYW5kIG11bHRpY2VsbHVsYXIgb3JnYW5pc21zOyBNb2wgQmlvbCBFdm9sOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0xMy0zNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gYmlhcyBzaWduYXR1cmVzLCBvcmdhbml6YXRpb24gb2YgbWljcm9vcmdhbmlzbXMgaW4gY29kb24gc3BhY2UsIGFuZCBsaWZlc3R5bGU7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz01NDctNTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WYXJpYXRpb24gaW4gdGhlIHN0cmVuZ3RoIG9mIHNlbGVjdGVkIGNvZG9uIHVzYWdlIGJpYXMgYW1vbmcgYmFjdGVyaWE7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPTExNDEtMTE1MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gZW52aXJvbm1lbnRhbCBzaWduYXR1cmUgZm9yIDMyMyBtaWNyb2JpYWwgZ2Vub21lcyBiYXNlZCBvbiBjb2RvbiBhZGFwdGF0aW9uIGluZGljZXM7IEdlbm9tZSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9UjExNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhHVC1EQjogYSBkYXRhYmFzZSBvZiBwdXRhdGl2ZSBob3Jpem9udGFsbHkgdHJhbnNmZXJyZWQgZ2VuZXMgaW4gcHJva2FyeW90aWMgY29tcGxldGUgZ2Vub21lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MTg3LTE4OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gYmlhcyBhbmQgYmFzZSBjb21wb3NpdGlvbiBhcmUgcG9vciBpbmRpY2F0b3JzIG9mIGhvcml6b250YWxseSB0cmFuc2ZlcnJlZCBnZW5lczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTQwNC00MTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVjdGluZyBhbGllbiBnZW5lcyBpbiBiYWN0ZXJpYWwgZ2Vub21lczsgQW5uLiBOIFkgQWNhZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT04NzA7ICBjaXRhdGlvbl9wYWdlcz0zMTQtMzI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW5nIGdlbmUgZXhwcmVzc2lvbiBsZXZlbCBmcm9tIGNvZG9uIHVzYWdlIGJpYXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xMC0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hhcmFjdGVyaXphdGlvbnMgb2YgaGlnaGx5IGV4cHJlc3NlZCBnZW5lcyBvZiBmb3VyIGZhc3QtZ3Jvd2luZyBiYWN0ZXJpYTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODM7ICBjaXRhdGlvbl9wYWdlcz01MDI1LTUwNDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhFRy1EQjogYSBkYXRhYmFzZSBvZiBwcmVkaWN0ZWQgaGlnaGx5IGV4cHJlc3NlZCBnZW5lcyBpbiBwcm9rYXJ5b3RpYyBjb21wbGV0ZSBnZW5vbWVzIHVuZGVyIHRyYW5zbGF0aW9uYWwgc2VsZWN0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz1ENTI0LUQ1Mjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb2RvbiBhZGFwdGF0aW9uIGluZGV4IOKAkyBhIG1lYXN1cmUgb2YgZGlyZWN0aW9uYWwgc3lub255bW91cyBjb2RvbiB1c2FnZSBiaWFzLCBhbmQgaXRzIHBvdGVudGlhbCBhcHBsaWNhdGlvbnM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTEyODEtMTI5NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29ycmVzcG9uZGVuY2UgYW5hbHlzaXMgYXBwbGllZCB0byBtaWNyb2FycmF5IGRhdGE7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9OTg7ICBjaXRhdGlvbl9wYWdlcz0xMDc4MS0xMDc4NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gY2F0YWxvZyB1c2FnZSBhbmQgdGhlIGdlbm9tZSBoeXBvdGhlc2lzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPXI0OS1yNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzZSBhbmQgbWlzdXNlIG9mIGNvcnJlc3BvbmRlbmNlIGFuYWx5c2lzIGluIGNvZG9uIHVzYWdlIHN0dWRpZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTQ1NDgtNDU1NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJlbmRzIGluIGNvZG9uIGFuZCBhbWlubyBhY2lkIHVzYWdlIGluIFRoZXJtb3RvZ2EgbWFyaXRpbWE7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz01NjMtNTY4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbmV4cGVjdGVkIGNvcnJlbGF0aW9ucyBiZXR3ZWVuIGdlbmUgZXhwcmVzc2lvbiBhbmQgY29kb24gdXNhZ2UgYmlhcyBmcm9tIG1pY3JvYXJyYXkgZGF0YSBmb3IgdGhlIHdob2xlIEVzY2hlcmljaGlhIGNvbGkgSy0xMiBnZW5vbWU7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTY5NzYtNjk4NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgYW5kIHRoZSBvcmlnaW4gb2YgUCBlbGVtZW50czsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTQ2Ny00Njg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9ubGluZSBzeW5vbnltb3VzIGNvZG9uIHVzYWdlIGFuYWx5c2VzIHdpdGggdGhlIGFkZTQgYW5kIHNlcWluUiBwYWNrYWdlczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz01NDUtNTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiBhbmQgYW1pbm8gYWNpZCB1c2FnZSBpbiB0d28gbWFqb3IgaHVtYW4gcGF0aG9nZW5zIG9mIGdlbnVzIEJhcnRvbmVsbGEg4oCTIG9wdGltaXphdGlvbiBiZXR3ZWVuIHJlcGxpY2F0aW9uYWzigJN0cmFuc2NyaXB0aW9uYWwgc2VsZWN0aW9uLCB0cmFuc2xhdGlvbmFsIGNvbnRyb2wgYW5kIGNvc3QgbWluaW1pemF0aW9uOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTkxLTEwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGVtcGVyYXR1cmUgYWRhcHRhdGlvbiBvZiBzeW5vbnltb3VzIGNvZG9uIHVzYWdlIGluIGRpZmZlcmVudCBmdW5jdGlvbmFsIGNhdGVnb3JpZXMgb2YgZ2VuZXM6IGEgY29tcGFyYXRpdmUgc3R1ZHkgYmV0d2VlbiBob21vbG9nb3VzIGdlbmVzIG9mIE1ldGhhbm9jb2NjdXMgamFubmFzY2hpaSBhbmQgTWV0aGFub2NvY2N1cyBtYXJpcGFsdWRpczsgRkVCUyBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT01ODA7ICBjaXRhdGlvbl9wYWdlcz0zODk1LTM4OTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIE5hbm9hcmNoYWV1bSBlcXVpdGFucyBnZW5vbWUgYW5kIHByb3Rlb21lIGNvbXBvc2l0aW9uOiBpbmRpY2F0aW9ucyBmb3IgaHlwZXJ0aGVybW9waGlsaWMgYW5kIHBhcmFzaXRpYyBhZGFwdGF0aW9uOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTE4NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBjb25zdHJhaW50cyBvbiBjb2RvbiBhbmQgYW1pbm8gYWNpZCB1c2FnZSBpbiB0d28gc3RyYWlucyBvZiBodW1hbiBwYXRob2dlbmljIGFjdGlub2JhY3RlcmlhIFRyb3BoZXJ5bWEgd2hpcHBsZWk7IEouIE1vbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz02NDUtNjU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5vbnltb3VzIGNvZG9uIHVzYWdlIGluIGFkZW5vdmlydXNlczogaW5mbHVlbmNlIG9mIG11dGF0aW9uLCBzZWxlY3Rpb24gYW5kIHByb3RlaW4gaHlkcm9wYXRoeTsgVmlydXMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTc7ICBjaXRhdGlvbl9wYWdlcz0yMjctMjM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBzdHVkeSBvZiB0aGUgaGVtYWdnbHV0aW5pbiBhbmQgbmV1cmFtaW5pZGFzZSBnZW5lcyBvZiBpbmZsdWVuemEgQSB2aXJ1cyBIM04yLCBIOU4yLCBhbmQgSDVOMSBzdWJ0eXBlcyB1c2luZyBiaW9pbmZvcm1hdGljcyB0ZWNobmlxdWVzOyBDYW4uIEouIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz04MzAtODM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmFsdWF0aW9uIG9mIGNvZG9uIGJpYXMgcGVyc3BlY3RpdmVzIGluIHBoYWdlIHRoZXJhcHkgb2YgTXljb2JhY3Rlcml1bSB0dWJlcmN1bG9zaXMgYnkgbXVsdGl2YXJpYXRlIGFuYWx5c2lzOyBJbiBTaWxpY28gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTAwMzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHVkaWVzIG9uIHN5bm9ueW1vdXMgY29kb24gYW5kIGFtaW5vIGFjaWQgdXNhZ2UgYmlhc2VzIGluIHRoZSBicm9hZC1ob3N0IHJhbmdlIGJhY3RlcmlvcGhhZ2UgS1ZQNDA7IEouIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz01OC02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgY29kb24gdXNhZ2UgcGF0dGVybnMgYW5kIHByZWRpY3RlZCBoaWdobHkgZXhwcmVzc2VkIGdlbmVzIGZvciBzaXggcGh5dG9wYXRob2dlbmljIFhhbnRob21vbmFzIGdlbm9tZXMgc2hvd3MgYSBoaWdoIGRlZ3JlZSBvZiBjb25zZXJ2YXRpb247IEluIFNpbGljbyBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NTQ3LTU1ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFwaWQgZGl2ZXJnZW5jZSBvZiBjb2RvbiB1c2FnZSBwYXR0ZXJucyB3aXRoaW4gdGhlIHJpY2UgZ2Vub21lOyBCTUMgRXZvbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPVM2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb24gcHJlc3N1cmUgc2hhcGVzIGNvZG9uIHVzYWdlIGluIHRoZSBHQy1yaWNoIGdlbm9tZSBvZiBmb290LWFuZC1tb3V0aCBkaXNlYXNlIHZpcnVzOyBWaXJ1cyBHZW5lczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPTc2Ny03NzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHN5bm9ueW1vdXMgY29kb24gdXNhZ2UgaW4gMTEgSHVtYW4gQm9jYXZpcnVzIGlzb2xhdGVzOyBCaW9zeXN0ZW1zOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTkyOyAgY2l0YXRpb25fcGFnZXM9MjA3LTIxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuQmFuazsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM2OyAgY2l0YXRpb25fcGFnZXM9RDI1LUQzMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVvbWUgY29tcG9zaXRpb24gYW5kIGNvZG9uIHVzYWdlIGluIHNwaXJvY2hhZXRlczogc3BlY2llcy1zcGVjaWZpYyBhbmQgRE5BIHN0cmFuZC1zcGVjaWZpYyBtdXRhdGlvbmFsIGJpYXNlczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9MTY0Mi0xNjQ5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXBsaWNhdGlvbmFsIGFuZCB0cmFuc2NyaXB0aW9uYWwgc2VsZWN0aW9uIG9uIGNvZG9uIHVzYWdlIGluIEJvcnJlbGlhIGJ1cmdkb3JmZXJpOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTk1OyAgY2l0YXRpb25fcGFnZXM9MTA2OTgtMTA2NzAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiB1c2FnZSBpbiBDaGxhbXlkaWEgdHJhY2hvbWF0aXMgaXMgdGhlIHJlc3VsdCBvZiBzdHJhbmQtc3BlY2lmaWMgbXV0YXRpb25hbCBiaWFzZXMgYW5kIGEgY29tcGxleCBwYXR0ZXJuIG9mIHNlbGVjdGl2ZSBmb3JjZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTIwODQtMjA5MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNsYXRpb25hbCBzZWxlY3Rpb24gaXMgb3BlcmF0aXZlIGZvciBzeW5vbnltb3VzIGNvZG9uIHVzYWdlIGluIENsb3N0cmlkaXVtIHBlcmZyaW5nZW5zIGFuZCBDbG9zdHJpZGl1bSBhY2V0b2J1dHlsaWN1bTsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE0OTsgIGNpdGF0aW9uX3BhZ2VzPTg1NS04NjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciBob3Jpem9udGFsIGdlbmUgdHJhbnNmZXIgaW4gRXNjaGVyaWNoaWEgY29saSBzcGVjaWF0aW9uOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTIyMjsgIGNpdGF0aW9uX3BhZ2VzPTg1MS04NTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9uLWxpbmUgdG9vbHMgZm9yIHNlcXVlbmNlIHJldHJpZXZhbCBhbmQgbXVsdGl2YXJpYXRlIHN0YXRpc3RpY3MgaW4gbW9sZWN1bGFyIGJpb2xvZ3k7IENvbXB1dC4gQXBwbC4gQmlvc2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTYzLTY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BYnNlbmNlIG9mIHRyYW5zbGF0aW9uYWxseSBzZWxlY3RlZCBzeW5vbnltb3VzIGNvZG9uIHVzYWdlIGJpYXMgaW4gSGVsaWNvYmFjdGVyIHB5bG9yaTsgTWljcm9iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE0NjsgIGNpdGF0aW9uX2lzc3VlPVB0IDQ7ICBjaXRhdGlvbl9wYWdlcz04NTEtODYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qcm9rYXJ5b3RpYyBnZW5vbWUgZXZvbHV0aW9uIGFzIGFzc2Vzc2VkIGJ5IG11bHRpdmFyaWF0ZSBhbmFseXNpcyBvZiBjb2RvbiB1c2FnZSBwYXR0ZXJuczsgTWljcm9iaW9sIENvbXAuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz04OS05NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgYW5kIGJhc2UgY29tcG9zaXRpb24gaW4gUmlja2V0dHNpYSBwcm93YXpla2lpOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9NTI1LTUzNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXV0YXRpb25hbCBhbmQgc2VsZWN0aXZlIHByZXNzdXJlcyBvbiBjb2RvbiBhbmQgYW1pbm8gYWNpZCB1c2FnZSBpbiBCdWNobmVyYSwgZW5kb3N5bWJpb3RpYyBiYWN0ZXJpYSBvZiBhcGhpZHM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NDQtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGluIHllYXN0OiBjbHVzdGVyIGFuYWx5c2lzIGNsZWFybHkgZGlmZmVyZW50aWF0ZXMgaGlnaGx5IGFuZCBsb3dseSBleHByZXNzZWQgZ2VuZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk4NjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTUxMjUtNTE0MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QURFLTQ6IGEgbXVsdGl2YXJpYXRlIGFuYWx5c2lzIGFuZCBncmFwaGljYWwgZGlzcGxheSBzb2Z0d2FyZTsgU3RhdC4gQ29tcHV0LjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NzUtODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVI6IGEgbGFuZ3VhZ2UgYW5kIGVudmlyb25tZW50IGZvciBzdGF0aXN0aWNhbCBjb21wdXRpbmcsIFIgRm91bmRhdGlvbiBmb3IgU3RhdGlzdGljYWwgQ29tcHV0aW5nOyBjaXRhdGlvbl9hdXRob3I9UiBEZXZlbG9wbWVudCBDb3JlIFRlYW0uOyBjaXRhdGlvbl9wdWJsaXNoZXI9Vmllbm5hLCBBdXN0cmlhOyAgY2l0YXRpb25feWVhcj0yMDA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5vbnltb3VzIGNvZG9uIHVzYWdlIGluIFBzZXVkb21vbmFzIGFlcnVnaW5vc2EgUEEwMTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yODk7ICBjaXRhdGlvbl9wYWdlcz0xMzEtMTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHNpbXBsZSBtZXRob2QgZm9yIGRpc3BsYXlpbmcgdGhlIGh5ZHJvcGF0aGljIGNoYXJhY3RlciBvZiBhIHByb3RlaW47IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODI7ICBjaXRhdGlvbl92b2x1bWU9MTU3OyAgY2l0YXRpb25fcGFnZXM9MTA1LTEzMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwcm9ibGVtIGluIG11bHRpdmFyaWF0ZSBhbmFseXNpcyBvZiBjb2RvbiB1c2FnZSBkYXRhIGFuZCBhIHBvc3NpYmxlIHNvbHV0aW9uOyBGRUJTIExldHQ7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NTc5OyAgY2l0YXRpb25fcGFnZXM9NjQ5OS01MDQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh5ZHJvcGhvYmljaXR5LCBleHByZXNzaXZpdHkgYW5kIGFyb21hdGljaXR5IGFyZSB0aGUgbWFqb3IgdHJlbmRzIG9mIGFtaW5vLWFjaWQgdXNhZ2UgaW4gOTk5IEVzY2hlcmljaGlhIGNvbGkgY2hyb21vc29tZS1lbmNvZGVkIGdlbmVzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz0zMTc0LTMxODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdsb2JhbCBhbmFseXNpcyBvZiBwcm90ZWluIGV4cHJlc3Npb24gaW4geWVhc3Q7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00MjU7ICBjaXRhdGlvbl9wYWdlcz03MzctNzQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db250ZXh0LWRlcGVuZGVudCBjb2RvbiBiaWFzIGFuZCBtZXNzZW5nZXIgUk5BIGxvbmdldml0eSBpbiB0aGUgeWVhc3QgdHJhbnNjcmlwdG9tZTsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTE0MDMtMTQxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3lzdGVtYXRpYyBiYXNlIGNvbXBvc2l0aW9uIHZhcmlhdGlvbiBhcm91bmQgdGhlIGdlbm9tZSBvZiBNeWNvcGxhc21hIGdlbml0YWxpdW0sIGJ1dCBub3QgTXljb3BsYXNtYSBwbmV1bW9uaWFlOyBNb2wuIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xMTc3LTExNzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUctbGFuZ3VhZ2UgZ2Vub21lIGFuYWx5c2lzIGVudmlyb25tZW50OiBhIHdvcmtiZW5jaCBmb3IgbnVjbGVvdGlkZSBzZXF1ZW5jZSBkYXRhIG1pbmluZzsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0zMDUtMzA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HLWxhbmd1YWdlIHN5c3RlbSBhcyBhIHBsYXRmb3JtIGZvciBsYXJnZS1zY2FsZSBhbmFseXNpcyBvZiBoaWdoLXRocm91Z2hwdXQgb21pY3MgZGF0YTsgSi4gUGVzdGljaWRlIFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0yODItMjg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTUvNi8zNTcvMTM3Njg4Ni9kc24wMjgucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFN5bm9ueW1vdXMgY29kb24gdXNhZ2UgdmFyaWVzIGJvdGggYmV0d2VlbiBvcmdhbmlzbXMgYW5kIGFtb25nIGdlbmVzIHdpdGhpbiBhIGdlbm9tZSwgYW5kIGFyaXNlcyBkdWUgdG8gZGlmZmVyZW5jZXMgaW4gRyArIEMgY29udGVudCwgcmVwbGljYXRpb24gc3RyIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj01MTMwMzAmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkKampwX8QHEiHBl/rxdLDRgsEqaQ=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f6%2f357%2f513030" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f6%2f357%2f513030" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=20280" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/6/1/m_cover.gif?Expires=2147483647&amp;Signature=cbcaSTnXmSAtcWAqInhnHikjrANtVVQT69rwVkS2N2-Rvvq~3o4NF54oja~GIUWptdFQr454edog~WN90FgAfPnwCCNM9uZ3DuEnj-WTN3JyklWsccNFHC6w0Mg4O1XsO2YSzFEaGsd~LWUMMgT-JRIdN1RQUYUQEdvCgKjLmwjDjkQDpQkgPjsYyRLj1QLXlpM4IsivMdAnqm9LmZlXnvMUUp5UVsqaQBEzVLg9CZ1N~86MZnaIY5LkZ1lOjOrEUjr78DNfXaNPxd2goUtJ24j8goSO16w~p5Co8ZBl8FnseAHBtje4N~aGVfSKUlSXEIVKmxpcUZsLAtBrli2ACg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/6">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7971547">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7971547">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7971549">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7971549">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7971554">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7971554">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7971569">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7971569">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7971598">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7971598">4. Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7971600">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7971600">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7971602">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7971602">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7971604">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7971604">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7971606">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7971606">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/6/347/512140">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/15/6/367/512658">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Comparison of Correspondence Analysis Methods for Synonymous Codon Usage in Bacteria
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Haruo Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biological Sciences and Initiative for Bioinformatics and Evolutionary Studies</div>, <div class="insititution">University of Idaho</div>, <div class="addr-line">PO Box 443051, Moscow, Idaho 83844-3051</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +1 208-885-8858. Fax. <div class="fax">+1 208-885-7905</div>. E-mail: <a href="mailto:hsuzuki@uidaho.edu" target="_blank">hsuzuki@uidaho.edu</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Haruo+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Haruo Suzuki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Celeste J. Brown</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biological Sciences and Initiative for Bioinformatics and Evolutionary Studies</div>, <div class="insititution">University of Idaho</div>, <div class="addr-line">PO Box 443051, Moscow, Idaho 83844-3051</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Celeste+J.+Brown">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Brown C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Brown C&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Celeste J. Brown</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Larry J. Forney</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biological Sciences and Initiative for Bioinformatics and Evolutionary Studies</div>, <div class="insititution">University of Idaho</div>, <div class="addr-line">PO Box 443051, Moscow, Idaho 83844-3051</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Larry+J.+Forney">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Forney L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Forney L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Larry J. Forney</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eva M. Top</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">Department of Biological Sciences and Initiative for Bioinformatics and Evolutionary Studies</div>, <div class="insititution">University of Idaho</div>, <div class="addr-line">PO Box 443051, Moscow, Idaho 83844-3051</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eva+M.+Top">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Top E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Top E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Eva M. Top</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Hiroyuki Toh</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 6, 1 December 2008, Pages 357–365, <a href='https://doi.org/10.1093/dnares/dsn028'>https://doi.org/10.1093/dnares/dsn028</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">21 October 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">21 June 2008</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">24 September 2008</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="513030" href="/dnaresearch/article-pdf/15/6/357/1376886/dsn028.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Haruo Suzuki, Celeste J. Brown, Larry J. Forney, Eva M. Top;  Comparison of Correspondence Analysis Methods for Synonymous Codon Usage in Bacteria, <em>DNA Research</em>, Volume 15, Issue 6, 1 December 2008, Pages 357–365, <a href='https://doi.org/10.1093/dnares/dsn028'>https://doi.org/10.1093/dnares/dsn028</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/513030?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/513030?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/513030?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/513030?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/513030?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/513030?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/513030?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Comparison%20of%20Correspondence%20Analysis%20Methods%20for%20Synonymous%20Codon%20Usage%20in%20Bacteria&amp;publicationDate=2008-10-21&amp;volumeNum=15&amp;issueNum=6&amp;author=Suzuki%2C%20Haruo%3B%20Brown%2C%20Celeste%20J.&amp;startPage=357&amp;endPage=365&amp;contentId=10.1093%2Fdnares%2Fdsn028&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f6%2f357%2f513030" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7971547 id="7971547" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Synonymous codon usage varies both between organisms and among genes within a genome, and arises due to differences in G + C content, replication strand skew, or gene expression levels. Correspondence analysis (CA) is widely used to identify major sources of variation in synonymous codon usage among genes and provides a way to identify horizontally transferred or highly expressed genes. Four methods of CA have been developed based on three kinds of input data: absolute codon frequency, relative codon frequency, and relative synonymous codon usage (RSCU) as well as within-group CA (WCA). Although different CA methods have been used in the past, no comprehensive comparative study has been performed to evaluate their effectiveness. Here, the four CA methods were evaluated by applying them to 241 bacterial genome sequences. The results indicate that WCA is more effective than the other three methods in generating axes that reflect variations in synonymous codon usage. Furthermore, WCA reveals sources that were previously unnoticed in some genomes; e.g. synonymous codon usage related to replication strand skew was detected in <em>Rickettsia prowazekii</em>. Though CA based on RSCU is widely used, our evaluation indicates that this method does not perform as well as WCA.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;correspondence analysis&quot;">correspondence analysis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;synonymous codon usage&quot;">synonymous codon usage</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;horizontal gene transfer&quot;">horizontal gene transfer</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;strand-specific mutational bias&quot;">strand-specific mutational bias</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;translational selection&quot;">translational selection</a></div>                            <h2 scrollto-destination=7971549 id="7971549" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Most amino acids are encoded by more than one codon, and these synonymous codons usually differ by one nucleotide in the third position. Generally, alternative synonymous codons are not used with equal frequency; their usage varies among different species, and often among genes within the same genome.<sup><a reveal-id="DSN028C1" data-open="DSN028C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Three principal factors have been proposed to account for the intragenomic variation in synonymous codon usage. First, intragenomic variation in G + C content is mostly related to the existence of regions with unusual base composition, so-called genomic islands, that may be the result of recent horizontal DNA transfer.<sup><a reveal-id="DSN028C2  DSN028C4" data-open="DSN028C2  DSN028C4" class="link link-ref link-reveal xref-bibr">2–4</a></sup> Secondly, the excess of G over C in the leading strand of DNA replication relative to the lagging strand is observed in many bacteria, and this is thought to reflect strand-specific mutational bias.<sup><a reveal-id="DSN028C5" data-open="DSN028C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSN028C6" data-open="DSN028C6" class="link link-ref link-reveal xref-bibr">6</a></sup> Thirdly, genes expressed at high levels in fast-growing bacteria tend to preferentially use translationally optimal codons that are recognized by the most abundant tRNAs. This presumably reflects natural selection for synonymous codons that are translated more efficiently and accurately.<sup><a reveal-id="DSN028C7" data-open="DSN028C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSN028C8" data-open="DSN028C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Thus, the use of synonymous codons in any gene can be the result of a mixture of these different evolutionary factors, and their relative contributions may vary among different species depending on their life history.<sup><a reveal-id="DSN028C9  DSN028C11" data-open="DSN028C9  DSN028C11" class="link link-ref link-reveal xref-bibr">9–11</a></sup> It follows that information on synonymous codon usage can be used to identify certain kinds of genes, e.g. those that have been horizontally transferred<sup><a reveal-id="DSN028C12  DSN028C14" data-open="DSN028C12  DSN028C14" class="link link-ref link-reveal xref-bibr">12–14</a></sup> or are highly expressed.<sup><a reveal-id="DSN028C15  DSN028C18" data-open="DSN028C15  DSN028C18" class="link link-ref link-reveal xref-bibr">15–18</a></sup></p><p>To reliably detect and quantify synonymous codon usage patterns, it is necessary to employ appropriate statistical methods. One such method is correspondence analysis (CA), a multivariate statistical method that can be used to summarize high dimensional data, such as codon counts, by reducing them to a limited number of variables, called axes.<sup><a reveal-id="DSN028C19" data-open="DSN028C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSN028C20" data-open="DSN028C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The axes retain much of the information about the variability in codon usage among the genes, but in a way that makes those differences easier to understand. This method is widely used to identify major sources of variation in synonymous codon usage among genes.</p><p>A common issue in synonymous codon usage analysis is that variation in amino acid composition among proteins is a confounding factor in assessing variation in synonymous codon usage among nucleotide sequences. Different approaches have been taken to remove such amino acid composition effects. Most commonly, CA is performed on modified codon usage data that have been adjusted for the frequency of the amino acids they encode. The resulting relative codon frequency (RF) and relative synonymous codon usage (RSCU) are used instead of the original codon count data, which is also referred to as the absolute codon frequency (AF). However, previous studies showed that for some genomes the use of RF and RSCU to remove amino acid composition effects introduced a bias associated with the low frequency of cysteine in proteins.<sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C22" data-open="DSN028C22" class="link link-ref link-reveal xref-bibr">22</a></sup> To validate findings, some researchers compared the results of CA using different input data (termed here CA-AF, CA-RF, and CA-RSCU).<sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C23" data-open="DSN028C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSN028C24" data-open="DSN028C24" class="link link-ref link-reveal xref-bibr">24</a></sup> The within-group CA (WCA) has been proposed as an alternative method to dissociate the effects of different amino acid compositions from the effects directly related to synonymous codon usage.<sup><a reveal-id="DSN028C25" data-open="DSN028C25" class="link link-ref link-reveal xref-bibr">25</a></sup> This method adjusts the value for each codon by the average value of all the codons encoding for the same amino acid using a different method than CA-RF or CA-RSCU. These four different CA methods have all been used for studying synonymous codon usage, but it remains unclear which one is the most effective. In spite of the lack of rigorous testing, CA-RSCU remains the most popular method.<sup><a reveal-id="DSN028C26  DSN028C37" data-open="DSN028C26  DSN028C37" class="link link-ref link-reveal xref-bibr">26–37</a></sup></p><p>In this paper, we have evaluated and compared four CA methods for the analysis of synonymous codon usage (CA-AF, CA-RF, CA-RSCU and WCA) by applying them to 241 bacterial genomes for which complete genome sequences were available. Our results indicate that WCA is more effective than the other three methods in generating axes corresponding to variation in synonymous codon usage.</p>                            <h2 scrollto-destination=7971554 id="7971554" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7971555 id="7971555" class="section-title" data-legacy-id=s2a>2.1. Sequences</h3>
<p>Complete genome sequences of bacterial species in GenBank format<sup><a reveal-id="DSN028C38" data-open="DSN028C38" class="link link-ref link-reveal xref-bibr">38</a></sup> were retrieved from the NCBI FTP site (ftp://ftp.ncbi.nih.gov/genomes/Bacteria). In the case of species for which multiple strains have been sequenced, only one representative was randomly selected. An exception was made for the genomes of the following 10 strains, which were specifically selected as species representatives because they have been previously analyzed by CA: <em>Borrelia burgdorferi</em> B31 (<em>B. burgdorferi</em> B31),<sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C39" data-open="DSN028C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSN028C40" data-open="DSN028C40" class="link link-ref link-reveal xref-bibr">40</a></sup><em>Chlamydia trachomatis</em> D/UW-3/CX (<em>C. trachomatis</em> D/UW-3/CX),<sup><a reveal-id="DSN028C41" data-open="DSN028C41" class="link link-ref link-reveal xref-bibr">41</a></sup><em>Clostridium perfringens</em> 13 (<em>C</em>. <em>perfringens</em> 13),<sup><a reveal-id="DSN028C42" data-open="DSN028C42" class="link link-ref link-reveal xref-bibr">42</a></sup><em>Escherichia coli</em> K12 MG1655 (<em>E</em>. <em>coli</em> K12 MG1655),<sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C23" data-open="DSN028C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSN028C43" data-open="DSN028C43" class="link link-ref link-reveal xref-bibr">43</a></sup><em>Haemophilus influenzae</em> Rd KW20 (<em>H</em>. <em>influenzae</em> Rd KW20),<sup><a reveal-id="DSN028C44" data-open="DSN028C44" class="link link-ref link-reveal xref-bibr">44</a></sup><em>Helicobacter pylori</em> 26695 (<em>H. pylori</em> 26695),<sup><a reveal-id="DSN028C45" data-open="DSN028C45" class="link link-ref link-reveal xref-bibr">45</a></sup><em>Mycoplasma genitalium</em> G37 (<em>M</em>. <em>genitalium</em> G37),<sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C46" data-open="DSN028C46" class="link link-ref link-reveal xref-bibr">46</a></sup><em>Rickettsia prowazekii</em> Madrid E (<em>R. prowazekii</em> Madrid E),<sup><a reveal-id="DSN028C47" data-open="DSN028C47" class="link link-ref link-reveal xref-bibr">47</a></sup><em>Thermotoga maritima</em> MSB8 (<em>T</em>. <em>maritima</em> MSB8)<sup><a reveal-id="DSN028C22" data-open="DSN028C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and <em>Treponema pallidum</em> Nichols (<em>T</em>. <em>pallidum</em> Nichols).<sup><a reveal-id="DSN028C39" data-open="DSN028C39" class="link link-ref link-reveal xref-bibr">39</a></sup> Moreover, genomes were excluded when genes used in the analysis (Section 2.4) were missing. The final data set included 241 genomes (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1 or S2</a> for a comprehensive list). All protein-coding sequences, except those containing letters other than A, C, G, or T were included in the analysis. Because methionine and tryptophan are generally encoded by only a single codon, the codons for methionine and tryptophan were excluded. Start and stop codons were also eliminated.</p>                            <h3 scrollto-destination=7971557 id="7971557" class="section-title" data-legacy-id=s2b>2.2. Definitions of codon usage data</h3>
<div class="block-child-p">We computed original codon count data, i.e. the AF, and two kinds of modified codon usage data that have been normalized for each individual amino acid. The latter included the RF, which is defined as the ratio of the number of occurrences of a codon to the sum of all synonymous codons<sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C48" data-open="DSN028C48" class="link link-ref link-reveal xref-bibr">48</a></sup> and the RSCU, which is defined as the ratio of the observed number of occurrences of a codon to the number expected if all synonymous codons were used with equal frequency.<sup><a reveal-id="DSN028C49" data-open="DSN028C49" class="link link-ref link-reveal xref-bibr">49</a></sup> The values of AF, RF and RSCU of the <em>c</em>th codon for the <em>a</em>th amino acid (AF<sub><em>ac</em></sub>, RF<sub><em>ac</em></sub>, and RSCU<sub><em>ac</em></sub>, respectively) were calculated as follows: <a id="DSN028M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN028M1"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn02803.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/m_dsn02803.gif?Expires=2147483647&amp;Signature=mNdm6jmnEhpgCch4HA76zvNjRfOTRaS0Hpehmj0i5fW52C-vD96bSEn5m7wK9uTDYLa0TskfwdQmfxvvv~eteiPxh6uNHaxX-UAQjLg5qoXjZTw4GWUskpLd~ytXthA6lgep3~f~lTKwhsekdv-3dlxDzATV5dqoBa86prl5PB1Se-Ulx5I7PkvzL0pPspsmU5A0Tyq0gb7GTKbOalTbLY48LOJZOq8Wi7kaCMff8hnu4j-vdxLJ-H4qSdA4xchMypivRL7ywQs7oj4eR64YEToRKliDG4Z9On6BCnXUfxlxomncBNenHM3dXSAx0hKhXi0s~XkiPYcsPtxg0uuisw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn02803.gif" /></a></div></div><div class="label">1</div></div><a id="DSN028M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN028M2"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn02804.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/m_dsn02804.gif?Expires=2147483647&amp;Signature=VZdIGC07QO0Y0OBF5E0AHpRrX7nGa-6mXDEZ7PEqZFuZGCxJku~1cjizi9y8dp771pdwkdyLyQoQQOaIth~zYpCxuid42Fry3t-SSIf~SaHc1J-h2O50aOzvk40BmvBjdrEu5KVrioo8HGuw2pvWnNN~tzSfFIKaNlr~9H4-qBXnFpK7J7~T3HKN2kn9SZTaDcwARwERt29Y1REesTmHdj5tYmlYDVt3IFOb9HflntoFgOxmzpLsX2zBS-6PeNwBc3rupX2i9hQ-4JyrJkV2lm8VDl3o~52c~mP7GNUcvRPK0BAQi~wzQUiEHtTEWV-kd2-nZYYv~pmcGMmus1dlMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn02804.gif" /></a></div></div><div class="label">2</div></div><a id="DSN028M3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN028M3"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn02805.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/m_dsn02805.gif?Expires=2147483647&amp;Signature=Yh13giB1Sujy6qCg3ZfCuGGjVPy3HrUjPP2pGw5BLO32mAKP0PuHqyK36OtZTpfF41NCRt0vDqTXcXQrA5MAJyqRlLzbHKT4s4IQSxxjUaX1B-VQQk~sHQayAqJiv7fn5NBAy2bAMJiYQwkiiROeQKHBKpvWkrP-Zh-Pb3a2jb7aKZXjr7-inU5x4PoKG3G~6gukk5V1erBsLitrqUBmHY2sIQx~-SLHl-CSDd0lXPWOg0dRiPjGrVKNw10Ph0H~pLyFrnYFbU86~pYNo2NYwE7kRO8EAcpTF3aLmgrLfVUEB8n7rq9SEZ6dVOGhT73UDOQIY-J-Y5zC7bpVg70yMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn02805.gif" /></a></div></div><div class="label">3</div></div> where <em>n</em><sub><em>ac</em></sub> is the number of occurrences of the <em>c</em>th codon for the <em>a</em>th amino acid, and <em>d</em><sub><em>a</em></sub> the degree of codon degeneracy for the <em>a</em>th amino acid. RF<sub><em>ac</em></sub> equals 1/<em>d</em><sub><em>a</em></sub> (e.g. 1/2 for cysteine and 1/6 for arginine) when alternative synonymous codons are used with equal frequency, and reaches the maximum value of 1 when only one of synonymous codons is used and all others are not present with value of 0. RSCU<sub><em>ac</em></sub> equals 1 when alternative synonymous codons are used with equal frequency, and attains its maximum value of <em>d</em><sub><em>a</em></sub> (e.g. 2 for cysteine and 6 for arginine) when only one of synonymous codons is used for the amino acid.</div>                            <h3 scrollto-destination=7971559 id="7971559" class="section-title" data-legacy-id=s2c>2.3. Implementation of CA</h3>
<p>CA was implemented using the ‘dudi.coa’ and ‘within’ functions in the ‘ade4’<sup><a reveal-id="DSN028C50" data-open="DSN028C50" class="link link-ref link-reveal xref-bibr">50</a></sup> library of R.<sup><a reveal-id="DSN028C51" data-open="DSN028C51" class="link link-ref link-reveal xref-bibr">51</a></sup> CA takes multivariate data and combines them into a small number of variables (axes) that explains most of the variation among the original variables.<sup><a reveal-id="DSN028C19" data-open="DSN028C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C25" data-open="DSN028C25" class="link link-ref link-reveal xref-bibr">25</a></sup> In our study our variables are the 59 codons for each gene in a genome, and the result of the CA yields the coordinates of each gene on each new axis. A matrix is created in which the rows correspond to the genes on one bacterial genome and the columns to the 59 codons, such that each row has the codon usage information for a specific gene. For the different CA methods, CA-AF, CA-RF, CA-RSCU, or WCA, the cells contain AF, RF, RSCU, or AF values, respectively, for each gene and codon.</p><p>We provide a brief explanation of our implementation of CA for analyzing synonymous codon usage. For each genome, the matrix <em>X</em> = [<em>x</em><sub><em>ij</em></sub>] is an input data table with <em>N</em> genes (rows) and 59 codons (columns). We denote the sum of values for the <em>i</em>th gene of <em>X</em> as <em>x</em><sub><em>i</em>+</sub> and the <em>j</em>th codon as <em>x</em><sub>+<em>j</em></sub>. We denote the sum of all of the data in <em>X</em> as <em>x</em><sub>++</sub>. The weight of the <em>i</em>th gene is defined as <em>p</em><sub><em>i</em>+</sub> = <em>x</em><sub><em>i</em></sub><sub>+</sub>/<em>x</em><sub>++</sub>, that of the <em>j</em>th codon is defined as <em>p</em><sub>+<em>j</em></sub> = <em>x</em><sub>+<em>j</em></sub>/<em>x</em><sub>++</sub>. The matrix <em>Y</em> has elements <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/m_dsn02806.gif?Expires=2147483647&amp;Signature=BxYtfbefd0vSY61IwhAdiCzoTjc3lByjBh~m~Ukx1Vqh3TgUiysm4fZ4TXZPaZ9ku0bsMUY3Q2G9idzKDxWcDKRTd9WW3vww~7UJAzGLvguopiDohy1RFb2FYV2bnvUvf2jM746419O49Dk8zvaMCUzstyZfjxUGegpaDSJYbR4g83GVfbXf6oApEaHJcu9vCZThz85CxaAIcRMdqCpQITbU9AXNE2d4MPyjZKcy2so7vkDqX74~yO~CKP6xMcQV-aVkUMgDgbSn6OEQ0LRBAC9qntGWJW1u-089k60qYotuQjA7mUBPSuQuO3Tbat-sY99XXXxoPqGnSHymCccCoQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsn02806.gif" /></span> where <em>p</em><sub><em>ij</em></sub> is the weight of each cell <em>p</em><sub><em>ij</em></sub> = <em>x</em><sub><em>ij</em></sub>/<em>x</em><sub>++</sub>. The matrix <em>Y</em> for WCA is obtained by replacing the elements <em>y</em><sub><em>ij</em></sub> in the matrix <em>Y</em> for CA-AF by <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/m_dsn02807.gif?Expires=2147483647&amp;Signature=v~KTxmAVSMcl0Bjw7GpPFYYePta0TeAzhOjhn9qIKZ~Znp4d8VcSFYeiywl0bXFbap1mtv6ybvwXXuO53u1zWKOf~gKS8rPHJQz0is6awblDA6pRJGwr6SMTfZ8GKNyfj31RLf3b6T21276AGxe-~zgcqbMGq-8u3HaLycNLsgSj17ycf3Mn3hQ78Py9rvsZMCMvaMQvj9FF5ciDVYhMVmqB14sfY9nmnmPTYAl-ORE~omTtJuKBfJtgOaCRhrm4OsV0sMbsSd9RkqpLH5n7Zs9uB3ImYkdxQRkBr4rknsRSJ4eiyPWykNT8AXj3VOn3xHZ39fdn0z2h-C4p-C0SvQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsn02807.gif" />⁠</span>, where the sum extends over all codons <em>j</em> encoding amino acid <em>a</em>. This subtraction centers the data in each cell based upon the value of the codons that encode a particular amino acid. In other words, the <em>y</em><sub><em>ij</em></sub> values for WCA become the difference between the <em>y</em><sub><em>ij</em></sub> values for CA-AF and their adjusted average.</p><div class="block-child-p">The matrix <em>Z</em> with elements <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/m_dsn02808.gif?Expires=2147483647&amp;Signature=yK40yepD9CpTnkcnq0qw2OWKv5dSLCzAWU18RKprZerLMTQbsqcpq3VCRq0MjNLWZHoqAtgiNnb9DjBYubr~Qo74nRLcywGwYn8gCWF0aZjOjrBolQbzR1R7aRbzzq7NQDyG~JqH0nDpwVpzScyl18iSDEe~4L8dWIa8ppjyde9WyqGmJzribxMOejQpnfB164wCl6bMv7P0l2ABwqucx2POeOxwv1FNOuSWCO6-46IPB3E0Ga~bl334JlnRhLGjq9RTohwskFpW6ysA80FWtfHs9lAV5QGZCzbYKMkfydfzsSib1nPSkY9uCuS7UvtIMoqNX2ZfvwQwvDsD-x5z3g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsn02808.gif" /></span> is submitted to singular value decomposition, producing three matrices: <em>Z</em> = <em>USV</em><sup>t</sup>. <em>S</em> is a diagonal matrix whose diagonal elements <em>s</em><sub><em>k</em></sub> are singular values, the matrices <em>U</em> and <em>V</em> have elements <em>u</em><sub><em>ik</em></sub> and <em>v</em><sub><em>jk</em></sub>, respectively (the superscript t is the transposition operator). The coordinates for the <em>i</em>th gene or the <em>j</em>th codon in the <em>k</em>th axis (<em>g</em><sub><em>ik</em></sub> and <em>c</em><sub><em>jk</em></sub>, respectively) are calculated as follows: <a id="DSN028M4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN028M4"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn02809.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/m_dsn02809.gif?Expires=2147483647&amp;Signature=qPIZuV3-P0G28VwmNmuUOHyxUrxu0jxEqkO7BPbFu5tJmfvqXIWcdfyVoh-DCFe9xDuLhlXe~nI6xUMei~IB~kOs8A6qfypQh4IMajhtLoa4aJUi1gPxjT2y1BrNSE4veSAA9c05GfQh2Cy5BtJO~-xkYLznTsAGqoJckrDxvUhwaqrzO~mi4lhKR9CfP457bYOIb4hKff3qeMAcizQyeiidMrAHW9fbyS70XZweCoe~uSLH9F~mhpe4CmczULV4tkIJ3bwW3jix1ZsEqc6QINtwXrn0HpbcSLwsCU~BPjmrRKbTnADT2QezIVMRIk4oENbtuu239o68x-0G4DJc2Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn02809.gif" /></a></div></div><div class="label">4</div></div><a id="DSN028M5" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSN028M5"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsn02810.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/m_dsn02810.gif?Expires=2147483647&amp;Signature=wYoHKL7bYneb4g4oXpnSHjwfw8-w72ielv~9cGnpezarUiHyhKDtZndSuQLUJHj6gJOol8b29vhEyR0xgcxteTX0KjwBBBpb367eZjAXDu5n~ZNYLic5SeEjMK7m0Ej~McuPYnsYSpOB0dFSH69W3yP4EEx2lvMscN5PPc6-nen-Mz-d72AV~9sK3DHIk8mHeY8Pg9LICOQScwXNCUzjoI~4m7EKE0KgKFT5m6T~8dBDE6FenQ-ei1eZ99Td7fBaR1ohOG0eFeUhC6z6iIP4jw6w1ZTDNovjLzgSjSo96GP2vE7-F~79OZ5FPyc4jT9yRfV4W4yU-hfVcYdmnbtgUQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsn02810.gif" /></a></div></div><div class="label">5</div></div></div><p>The <em>g</em><sub><em>ik</em></sub> scores are the values that are correlated with other gene features in the subsequent analyses (see Section 2.4).</p><p>The contribution of the <em>j</em>th codon to the <em>k</em>th axis is given by <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/m_dsn02811.gif?Expires=2147483647&amp;Signature=awXNdFHRQN~YeLx4WtccG3-tlcsXCa0qtDQXDP8p0fEi6P2~BboqWLKMHZ4i7BjfXuxgpW3q2k31H5O2aweA-VnbEomQk3Jab291VTcnK7A1Fgb18DmV5BS1vbnyyTY1KQg8YZpeN2T96dL8JbV1Ryld~Ot9Iave7eOdtX95zDvb6BcWI7rWHbVxHo5ssODKqCXdcPzuSJzihs1gClVs~nFj-evHCm-D5c1f0tpC8K~xfytpha2pPWK9q-f8-qt66cmVuf-So~8dZBfderrujRCXT3D2fTnLaryFv3VOBvWeMJEoK30tCVyg0ibji6XpcZhqNNdvY0gL6n9BrojdFQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsn02811.gif" />⁠</span>. The sum of the contributions of all 59 codons to each axis is one; that is, <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/m_dsn02812.gif?Expires=2147483647&amp;Signature=DZ-1aezzzN0uwmIb96G2ugdPPuthxEx9jWmRyc6z9FS2kTwD1sjqLBOhdsgwatOi1P0EPd9WKoJ60qiUOrkneaKSV4Q2c7InnHycHnS1wgrt2L9ndkJj-LqMiEjJrKlgipw0NXMH2fijlsOpIZJhxT~mWCu~bmZMJo1Ri4pb~zRqpdkhYLxSiMr50h3hcZYeZgGaSjNboOyUBBg2H6QJ6ckcCDX6DOk1eeKUzXPwTOTkDdiwk3H98RxSe0fIVz-T9Po70HCUHUDdT3tuvmvaPHbn5NQR624xk42lR-GHH0tafyyiMEQAqQllDPTclETHCzYkNCuKzYfZPBtMiLpzNw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsn02812.gif" />⁠</span>. We compared the sum of the contributions of 18 codons with twofold degeneracy (those coding for asparagine, aspartic acid, cysteine, glutamic acid, glutamine, histidine, lysine, phenylalanine, and tyrosine) and the sum of the contributions of 18 codons with sixfold degeneracy (those coding arginine, leucine, and serine).</p><p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a> shows the percentage of total variance explained by the first 10 axes, as generated by these four CA methods for 241 bacterial genomes. Because the percentage of variance explained by axes &gt;3 was small overall, our subsequent analyses were focused on the first three axes.</p>                            <h3 scrollto-destination=7971566 id="7971566" class="section-title" data-legacy-id=s2d>2.4. Interpretation of axes generated by CA</h3>
<p>To identify major sources of variation among genes on the axes generated by CA of codon usage data, we conducted two analyses that considered four commonly used features of protein-coding genes: <em>GRAVY</em>, <em>GC3content</em>, <em>GC3skew</em>, and <em>Expression</em>.<sup><a reveal-id="DSN028C22" data-open="DSN028C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSN028C52" data-open="DSN028C52" class="link link-ref link-reveal xref-bibr">52</a></sup> First, we tested for the correlation between scores of each of three axes [Equation (<a href="#DSN028M4" class="sectionLink xref-sec">4</a>)] and values of <em>GRAVY</em>, <em>GC3content</em>, or <em>GC3skew. GRAVY</em> is the mean of the sum of the hydropathic index of each amino acid in the protein, and thus reflects amino acid composition.<sup><a reveal-id="DSN028C53" data-open="DSN028C53" class="link link-ref link-reveal xref-bibr">53</a></sup><em>GC3content</em> is the relative frequency of guanine and cytosine, (G + C)/(A + T + G + C), at the third codon position in the nucleotide sequence, and <em>GC3skew</em> is the deviation from equal amounts of guanine and cytosine, (G − C)/(G + C), at the third codon position in the nucleotide sequence. Pearson’s product moment correlation coefficient (<em>r</em>) between the axis scores and gene feature values was calculated. The square of <em>r</em> measures the percentage of variance; e.g. the square of 0.70 indicates that 49% of the variance in the axis scores is explained by the variance in the gene feature values. For each axis, the gene feature with an absolute <em>r</em> value (|<em>r</em>|) &gt;0.70 was identified as the main source of variation among genes on the axis. At lower threshold |<em>r</em>| values, different gene features were detected on the same axis and/or the same gene feature was detected on more than one axis, and thus the interpretation of the axes becomes quite difficult. Additionally, low |<em>r</em>| values may be statistically significantly different from zero due to very large sample sizes, but weak correlations may have no biological meaning.</p><p>Secondly, to analyze the correlation between scores of each of the three axes [Equation (<a href="#DSN028M4" class="sectionLink xref-sec">4</a>)] and levels of gene expression (<em>Expression</em>), we tested for the distribution of the axis scores for 40 genes expected to be expressed constitutively at high levels.<sup><a reveal-id="DSN028C10" data-open="DSN028C10" class="link link-ref link-reveal xref-bibr">10</a></sup> This set included the genes encoding translation elongation factors Tu (<em>tuf</em>), Ts (<em>tsf</em>) and G (<em>fus</em>), and 37 of the larger ribosomal proteins (encoded by genes <em>rplA-rplF</em>, <em>rplI-rplT</em>, and <em>rpsB-rpsT</em>). In each axis, the score for each gene was standardized by subtracting the mean and dividing by the standard deviation of scores for all protein genes. For each axis, <em>Expression</em> was detected as the main source of variation among genes on the axis when the mean absolute standard score for the 40 highly expressed genes was &gt;1.644854 (an interval in which theoretically only 5% of all protein genes are included).</p>                            <h2 scrollto-destination=7971569 id="7971569" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=7971570 id="7971570" class="section-title" data-legacy-id=s3a>3.1. Performance of different CA methods</h3>
<p>CA summarizes high dimensional data, such as codon counts, by reducing them to a limited number of variables (axes). We tested the ability of the four CA methods, CA-AF, CA-RF, CA-RSCU, and WCA, to generate axes that correspond to variation in synonymous codon usage. We considered two commonly used gene features: <em>GC3content</em> is the G + C content at the third codon position, and <em>GC3skew</em> that reflects the bias in G over C content at the third codon position. We investigated how often these two gene features were correlated with one of the first three axes in 241 bacterial genomes (Table <a reveal-id="DSN028TB1" data-open="DSN028TB1" class="link link-reveal link-table xref-fig">1</a>). To illustrate our method, Fig. <a reveal-id="DSN028F1" data-open="DSN028F1" class="link link-reveal link-table xref-fig">1</a> shows scatter plots of axis 1 scores obtained by the four methods, plotted against <em>GC3skew</em> for <em>R</em>. <em>prowazekii</em> Madrid E genes. At the threshold |<em>r</em>| value of 0.70, <em>GC3skew</em> values were significantly correlated with axis 1 scores of WCA (|<em>r</em>| = 0.84), but not with those of CA-AF (|<em>r</em>| = 0.46), CA-RF (|<em>r</em>| = 0.32), and CA-RSCU (|<em>r</em>| = 0.04). Thus, in <em>R</em>. <em>prowazekii</em> Madrid E, <em>GC3skew</em> was detected on axis 1 of WCA, but not on axis 1 of CA-AF, CA-RF, and CA-RSCU. <em>GC3content</em> was detected in 191 genomes when the WCA method was used, which was more than when CA-AF (150), CA-RF (143), or CA-RSCU (145) were used (Table <a reveal-id="DSN028TB1" data-open="DSN028TB1" class="link link-reveal link-table xref-fig">1</a>A). Likewise, the total number of genomes where <em>GC3skew</em> was detected (108) was also greater when WCA was used than when CA-AF (46), CA-RF (30), and CA-RSCU (53) were used (Table <a reveal-id="DSN028TB1" data-open="DSN028TB1" class="link link-reveal link-table xref-fig">1</a>B). Thus, WCA detected <em>GC3content</em> and <em>GC3skew</em> more often than CA-AF, CA-RF, and CA-RSCU.</p>                        <a id="7971572" scrollto-destination="7971572"></a>
<div id="DSN028F1" data-id="DSN028F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/m_dsn02801.jpeg?Expires=2147483647&Signature=HK01IvEceHBFPQ8vh03xthm0S52r~LsYQSMe0N6l8g79fQl8IKGkMcNJPpanzL~Cne9MVuwVuQIEBeTp96FRl9wDIaxCuF2SFvkQ4eo-uCkPVL1TbxAw-e0thEeciA40t~ZI4HBYHexeyTJS39iqPlOIDoWyKgPLOW71ArX6lyge99Jog8tdxz-hOJw6Pz1qYGLCWMxr6FIhGkAwUmwuXSDd~Dp4CrWQ1sD04VxC-YDxQmbc7-4AmyvXc~DS1N9vW4GW4OdUWtQqsQRm8wMRbkEULjZLj4h2kWu6CTtDulO3oEmFz1vLi~rGfzVFZIairdGy4tS-mOryk3urX71Asg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plot showing axis 1 scores obtained by different CA methods, CA-AF (A), CA-RF (B), CA-RSCU (C), and WCA (D), plotted against GC3skew for R. prowazekii Madrid E genes. Each point represents a gene." path-from-xml="dsn02801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7971572/dsn02801.jpeg" path-from-xml="dsn02801.jpeg" target="_blank">View large</a><a section="7971572" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/dsn02801.jpeg?Expires=2147483647&Signature=UuFZIXgevf9pGa3p8hpJJuGZvEwMlxORqvFbYuBRtBtpCcJKt3RIUZqHHVM-1HzjLrnXIYIoa0wW~wVsiGWT4I67LaBwhbsUpb85lgw3Zt54mwYKRPuqwZTEAVDzYjstdnSFjZ~0q4oyxeY0CjO-RILFqkeYlyPVPAXcXEKn0zvx8X47hHtrWw06X4KPY2asdP55CzIc6hNWE9QznoQg4kZJeqMpXZPbZqr7tMS0tpQm170jY4mcyPRzsjdd5O1NWNKIKQla74D~wwIm0u~ADUO2UmiXzqZ1fbT0T6WoLI~v9c~QyUmnrrQpIAFL3zrxwf6WeSM8UuU8MpiynnTiSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7971572&ar=513030&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plot showing axis 1 scores obtained by different CA methods, CA-AF (<strong>A</strong>), CA-RF (<strong>B</strong>), CA-RSCU (<strong>C</strong>), and WCA (<strong>D</strong>), plotted against <em>GC3skew</em> for <em>R. prowazekii</em> Madrid E genes. Each point represents a gene.</p></div></div><div content-id="DSN028F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/m_dsn02801.jpeg?Expires=2147483647&Signature=HK01IvEceHBFPQ8vh03xthm0S52r~LsYQSMe0N6l8g79fQl8IKGkMcNJPpanzL~Cne9MVuwVuQIEBeTp96FRl9wDIaxCuF2SFvkQ4eo-uCkPVL1TbxAw-e0thEeciA40t~ZI4HBYHexeyTJS39iqPlOIDoWyKgPLOW71ArX6lyge99Jog8tdxz-hOJw6Pz1qYGLCWMxr6FIhGkAwUmwuXSDd~Dp4CrWQ1sD04VxC-YDxQmbc7-4AmyvXc~DS1N9vW4GW4OdUWtQqsQRm8wMRbkEULjZLj4h2kWu6CTtDulO3oEmFz1vLi~rGfzVFZIairdGy4tS-mOryk3urX71Asg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Scatter plot showing axis 1 scores obtained by different CA methods, CA-AF (A), CA-RF (B), CA-RSCU (C), and WCA (D), plotted against GC3skew for R. prowazekii Madrid E genes. Each point represents a gene." path-from-xml="dsn02801.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7971572/dsn02801.jpeg" path-from-xml="dsn02801.jpeg" target="_blank">View large</a><a section="7971572" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/dsn02801.jpeg?Expires=2147483647&Signature=UuFZIXgevf9pGa3p8hpJJuGZvEwMlxORqvFbYuBRtBtpCcJKt3RIUZqHHVM-1HzjLrnXIYIoa0wW~wVsiGWT4I67LaBwhbsUpb85lgw3Zt54mwYKRPuqwZTEAVDzYjstdnSFjZ~0q4oyxeY0CjO-RILFqkeYlyPVPAXcXEKn0zvx8X47hHtrWw06X4KPY2asdP55CzIc6hNWE9QznoQg4kZJeqMpXZPbZqr7tMS0tpQm170jY4mcyPRzsjdd5O1NWNKIKQla74D~wwIm0u~ADUO2UmiXzqZ1fbT0T6WoLI~v9c~QyUmnrrQpIAFL3zrxwf6WeSM8UuU8MpiynnTiSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7971572&ar=513030&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02801.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Scatter plot showing axis 1 scores obtained by different CA methods, CA-AF (<strong>A</strong>), CA-RF (<strong>B</strong>), CA-RSCU (<strong>C</strong>), and WCA (<strong>D</strong>), plotted against <em>GC3skew</em> for <em>R. prowazekii</em> Madrid E genes. Each point represents a gene.</p></div></div>                        <a id="7971573" scrollto-destination="7971573"></a>
<div content-id="DSN028TB1" class="table-modal"><div class="table-wrap"><div id="DSN028TB1" class="table-wrap-title" data-id="DSN028TB1"><div class="label -label">Table 1</div><div class="caption"><p>Numbers of genomes where the gene feature <em>GC3content</em>, <em>GC3skew</em>, or <em>GRAVY</em> was significantly correlated with one of three axes generated by different CA methods, CA-AF, CA-RF, CA-RSCU, and WCA, in 241 bacterial genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Method </th><th>Axis 1 </th><th>Axis 2 </th><th>Axis 3 </th></tr></thead><tbody><tr><td colspan="4">A. <em>GC3content</em> </td></tr><tr><td> CA-AF </td><td>121 </td><td>17 </td><td>12 </td></tr><tr><td> CA-RF </td><td>129 </td><td>9 </td><td>5 </td></tr><tr><td> CA-RSCU </td><td>134 </td><td>11 </td><td>0 </td></tr><tr><td> WCA </td><td>150 </td><td>34 </td><td>7 </td></tr><tr><td colspan="4">B. <em>GC3skew</em> </td></tr><tr><td> CA-AF </td><td>26 </td><td>7 </td><td>13 </td></tr><tr><td> CA-RF </td><td>26 </td><td>4 </td><td>0 </td></tr><tr><td> CA-RSCU </td><td>25 </td><td>25 </td><td>3 </td></tr><tr><td> WCA </td><td>38 </td><td>57 </td><td>13 </td></tr><tr><td colspan="4">C. <em>GRAVY</em> </td></tr><tr><td> CA-AF </td><td>20 </td><td>69 </td><td>55 </td></tr><tr><td> CA-RF </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td> CA-RSCU </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td> WCA </td><td>0 </td><td>0 </td><td>0 </td></tr></tbody></table></div><div content-id="DSN028TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Method </th><th>Axis 1 </th><th>Axis 2 </th><th>Axis 3 </th></tr></thead><tbody><tr><td colspan="4">A. <em>GC3content</em> </td></tr><tr><td> CA-AF </td><td>121 </td><td>17 </td><td>12 </td></tr><tr><td> CA-RF </td><td>129 </td><td>9 </td><td>5 </td></tr><tr><td> CA-RSCU </td><td>134 </td><td>11 </td><td>0 </td></tr><tr><td> WCA </td><td>150 </td><td>34 </td><td>7 </td></tr><tr><td colspan="4">B. <em>GC3skew</em> </td></tr><tr><td> CA-AF </td><td>26 </td><td>7 </td><td>13 </td></tr><tr><td> CA-RF </td><td>26 </td><td>4 </td><td>0 </td></tr><tr><td> CA-RSCU </td><td>25 </td><td>25 </td><td>3 </td></tr><tr><td> WCA </td><td>38 </td><td>57 </td><td>13 </td></tr><tr><td colspan="4">C. <em>GRAVY</em> </td></tr><tr><td> CA-AF </td><td>20 </td><td>69 </td><td>55 </td></tr><tr><td> CA-RF </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td> CA-RSCU </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td> WCA </td><td>0 </td><td>0 </td><td>0 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7971573" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN028TB1" class="table-wrap-title" data-id="DSN028TB1"><div class="label -label">Table 1</div><div class="caption"><p>Numbers of genomes where the gene feature <em>GC3content</em>, <em>GC3skew</em>, or <em>GRAVY</em> was significantly correlated with one of three axes generated by different CA methods, CA-AF, CA-RF, CA-RSCU, and WCA, in 241 bacterial genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Method </th><th>Axis 1 </th><th>Axis 2 </th><th>Axis 3 </th></tr></thead><tbody><tr><td colspan="4">A. <em>GC3content</em> </td></tr><tr><td> CA-AF </td><td>121 </td><td>17 </td><td>12 </td></tr><tr><td> CA-RF </td><td>129 </td><td>9 </td><td>5 </td></tr><tr><td> CA-RSCU </td><td>134 </td><td>11 </td><td>0 </td></tr><tr><td> WCA </td><td>150 </td><td>34 </td><td>7 </td></tr><tr><td colspan="4">B. <em>GC3skew</em> </td></tr><tr><td> CA-AF </td><td>26 </td><td>7 </td><td>13 </td></tr><tr><td> CA-RF </td><td>26 </td><td>4 </td><td>0 </td></tr><tr><td> CA-RSCU </td><td>25 </td><td>25 </td><td>3 </td></tr><tr><td> WCA </td><td>38 </td><td>57 </td><td>13 </td></tr><tr><td colspan="4">C. <em>GRAVY</em> </td></tr><tr><td> CA-AF </td><td>20 </td><td>69 </td><td>55 </td></tr><tr><td> CA-RF </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td> CA-RSCU </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td> WCA </td><td>0 </td><td>0 </td><td>0 </td></tr></tbody></table></div><div content-id="DSN028TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Method </th><th>Axis 1 </th><th>Axis 2 </th><th>Axis 3 </th></tr></thead><tbody><tr><td colspan="4">A. <em>GC3content</em> </td></tr><tr><td> CA-AF </td><td>121 </td><td>17 </td><td>12 </td></tr><tr><td> CA-RF </td><td>129 </td><td>9 </td><td>5 </td></tr><tr><td> CA-RSCU </td><td>134 </td><td>11 </td><td>0 </td></tr><tr><td> WCA </td><td>150 </td><td>34 </td><td>7 </td></tr><tr><td colspan="4">B. <em>GC3skew</em> </td></tr><tr><td> CA-AF </td><td>26 </td><td>7 </td><td>13 </td></tr><tr><td> CA-RF </td><td>26 </td><td>4 </td><td>0 </td></tr><tr><td> CA-RSCU </td><td>25 </td><td>25 </td><td>3 </td></tr><tr><td> WCA </td><td>38 </td><td>57 </td><td>13 </td></tr><tr><td colspan="4">C. <em>GRAVY</em> </td></tr><tr><td> CA-AF </td><td>20 </td><td>69 </td><td>55 </td></tr><tr><td> CA-RF </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td> CA-RSCU </td><td>0 </td><td>0 </td><td>0 </td></tr><tr><td> WCA </td><td>0 </td><td>0 </td><td>0 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7971573" target="_blank">View Large</a></div></div><p>It is important to note that these results remained similar when all complete bacterial genomic sequences available from the NCBI repository on August 2008 were included (data not shown). Similar results were obtained when only long sequences with &gt;300 codons were used (data not shown). We also verified the consistency of the results when using detection thresholds below |<em>r</em>| = 0.70 (data not shown). Thus we conclude that WCA is more effective than the other three methods in generating axes that correspond to variation in synonymous codon usage, regardless of the data sets and statistical criteria used.</p><p>WCA may have performed best because it does not mask variation in synonymous codon usage caused by amino acid composition and codon degeneracy. CA-AF may have performed worse because it is confounded by amino acid composition. CA-RF and CA-RSCU did not perform as well as WCA possibly because their input data depend on the degree of codon degeneracy, which differs among amino acids [<em>d</em><sub><em>a</em></sub> in Equations (<a href="#DSN028M2" class="sectionLink xref-sec">2</a>) and (<a href="#DSN028M3" class="sectionLink xref-sec">3</a>) in Section 2.2].<sup><a reveal-id="DSN028C54" data-open="DSN028C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Later, we demonstrate these effects on the four CA methods.</p>                            <h3 scrollto-destination=7971576 id="7971576" class="section-title" data-legacy-id=s3b>3.2. Effect of amino acid composition and codon degeneracy in different CA methods</h3>
<p>To determine the effect of amino acid composition, we tested the ability of the four CA methods, CA-AF, CA-RF, CA-RSCU, and WCA, to generate axes that correspond to variation in amino acid composition. The protein feature <em>GRAVY</em>, which represents the global hydrophobicity of proteins, can be used to measure the variation in amino acid composition among proteins.<sup><a reveal-id="DSN028C55" data-open="DSN028C55" class="link link-ref link-reveal xref-bibr">55</a></sup> We investigated how often <em>GRAVY</em> was correlated with one of the first three axes in 241 bacterial genomes. CA-AF detected the correlation between <em>GRAVY</em> and one of the first three axes in 144 genomes, whereas CA-RF, CA-RSCU, and WCA did not detect it (Table <a reveal-id="DSN028TB1" data-open="DSN028TB1" class="link link-reveal link-table xref-fig">1</a>C). This result suggests that CA-AF can generate axes corresponding to variation in amino acid composition as well as synonymous codon usage, whereas CA-RF, CA-RSCU, and WCA never generate such axes because they compensate for differences in amino acid composition.</p><p>The use of RF and RSCU to remove the confounding effects of amino acid composition introduces other effects associated with the degree of codon degeneracy, which may be pronounced for rare amino acids. To determine the effect of the difference in the degree of codon degeneracy between amino acids, we compared the contributions to axis 1 of nine amino acids with low (twofold) degeneracy and three amino acids with high (sixfold) degeneracy, totaling 18 codons each. This was done for the four CA methods, CA-AF, CA-RF, CA-RSCU, and WCA. Fig. <a reveal-id="DSN028F2" data-open="DSN028F2" class="link link-reveal link-table xref-fig">2</a> shows scatter plots of the contribution of twofold degenerate codons (<em>y</em>-axis) plotted against that of sixfold degenerate codons (<em>x</em>-axis) for 241 bacterial genomes. The scatter plots for CA-AF and WCA (Fig. <a reveal-id="DSN028F2" data-open="DSN028F2" class="link link-reveal link-table xref-fig">2</a>A and D) displayed genome distributions less biased toward twofold or sixfold degenerate codons than the scatter plots for CA-RF and CA-RSCU (Fig. <a reveal-id="DSN028F2" data-open="DSN028F2" class="link link-reveal link-table xref-fig">2</a>B and C). For CA-RF, 208 (86%) of the 241 genomes fell above the line <em>y</em> = <em>x</em>, indicating that twofold degenerate codons contributed more to the axis than sixfold degenerate codons in most genomes (Fig. <a reveal-id="DSN028F2" data-open="DSN028F2" class="link link-reveal link-table xref-fig">2</a>B). For CA-RSCU, 238 (99%) of the 241 genomes were below the line <em>y</em> = <em>x</em>, indicating that sixfold degenerate codons contributed more to the axis than twofold degenerate codons in most genomes (Fig. <a reveal-id="DSN028F2" data-open="DSN028F2" class="link link-reveal link-table xref-fig">2</a>C). Thus, CA-RF and CA-RSCU tend to generate axes corresponding to variation in low (twofold) and high (sixfold) degenerate codons, respectively. This observation can be explained by the dependence of their input data on the degree of codon degeneracy [<em>d</em><sub><em>a</em></sub> in Equations (<a href="#DSN028M2" class="sectionLink xref-sec">2</a>) and (<a href="#DSN028M3" class="sectionLink xref-sec">3</a>) in Section 2.2]. Thus, the use of RF and RSCU to remove effects of amino acid usage introduces other effects associated with the degree of codon degeneracy, whereas WCA does not. In spite of these shortcomings, these methods, in particular CA-RSCU, are still frequently used.<sup><a reveal-id="DSN028C26" data-open="DSN028C26" class="link link-ref link-reveal xref-bibr">26</a>–<a reveal-id="DSN028C37" data-open="DSN028C37" class="link link-ref link-reveal xref-bibr">37</a></sup> We recommend using WCA for analyzing synonymous codon usage.</p>                        <a id="7971581" scrollto-destination="7971581"></a>
<div id="DSN028F2" data-id="DSN028F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/m_dsn02802.jpeg?Expires=2147483647&Signature=GBrpRd-3TlVQNFm18D2VQ9GswITQka5S2t10jVl8rqDDVzVYt7AkkAP3w5te-4uMvz6SVT82plOINWhpsSacARxiFMcXbGCdqexThDjbYz4w3r37f9Gcuh9Zu0hEoq-7N2o1kIEMgPU8QoKgJNZFqjj4U6sWMUKS0R1UT5ZD3sZ3ZRW1dTSTmIQaf4UxuRGhRCW2YTHciJ2Dq8Osndcz26h75DETIihx~mI5i2aRYFn2jqhXgfmwgNejGIVqfIW29ZxlP1vXgIGxeIAg4dbMep99VpLqo59iKS95h9wJaRBshCWhIaM1nKxMoNyvNwRvAlvbULAwiwtxvfjDsQ7RWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Contributions of twofold and sixfold degenerate codons to axis 1, obtained by different CA methods, CA-AF (A), CA-RF (B), CA-RSCU (C), and WCA (D), for 241 bacterial genomes. Each point represents a genome." path-from-xml="dsn02802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7971581/dsn02802.jpeg" path-from-xml="dsn02802.jpeg" target="_blank">View large</a><a section="7971581" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/dsn02802.jpeg?Expires=2147483647&Signature=RGUQALCFMh58vw78czZJWqIgJtlmQ9sKSI2ACe3Tf68~CyWzfTkLTrlWQj9Ly23VmkGaNPr2wfSJqFv8Bl5ggBSbJzwehwho5MTTpNctL0jhnHooj6Zoo-fTMlaXL8rixYaUqQ9XuXAjo3aBHRZX5dXwhzNewH7XZX2LayLa~hnNPJ7IAakPiTxjCWAjwUGsc7HezOt6w99au6SJK~7qZf6SSsMKm~0xcbYIkQcgOanZUvlacp2sf1BBcYXg4QM0Dk1YGdp9jlKBR7Tjpb-Tt3fC6I492Hkl1rK-HOaH9BnBtXHxdN5xII6ps4LxTAU6CasO44lGw3uflzVHqAF6AA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7971581&ar=513030&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Contributions of twofold and sixfold degenerate codons to axis 1, obtained by different CA methods, CA-AF (<strong>A</strong>), CA-RF (<strong>B</strong>), CA-RSCU (<strong>C</strong>), and WCA (<strong>D</strong>), for 241 bacterial genomes. Each point represents a genome.</p></div></div><div content-id="DSN028F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/m_dsn02802.jpeg?Expires=2147483647&Signature=GBrpRd-3TlVQNFm18D2VQ9GswITQka5S2t10jVl8rqDDVzVYt7AkkAP3w5te-4uMvz6SVT82plOINWhpsSacARxiFMcXbGCdqexThDjbYz4w3r37f9Gcuh9Zu0hEoq-7N2o1kIEMgPU8QoKgJNZFqjj4U6sWMUKS0R1UT5ZD3sZ3ZRW1dTSTmIQaf4UxuRGhRCW2YTHciJ2Dq8Osndcz26h75DETIihx~mI5i2aRYFn2jqhXgfmwgNejGIVqfIW29ZxlP1vXgIGxeIAg4dbMep99VpLqo59iKS95h9wJaRBshCWhIaM1nKxMoNyvNwRvAlvbULAwiwtxvfjDsQ7RWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Contributions of twofold and sixfold degenerate codons to axis 1, obtained by different CA methods, CA-AF (A), CA-RF (B), CA-RSCU (C), and WCA (D), for 241 bacterial genomes. Each point represents a genome." path-from-xml="dsn02802.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7971581/dsn02802.jpeg" path-from-xml="dsn02802.jpeg" target="_blank">View large</a><a section="7971581" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/dsn02802.jpeg?Expires=2147483647&Signature=RGUQALCFMh58vw78czZJWqIgJtlmQ9sKSI2ACe3Tf68~CyWzfTkLTrlWQj9Ly23VmkGaNPr2wfSJqFv8Bl5ggBSbJzwehwho5MTTpNctL0jhnHooj6Zoo-fTMlaXL8rixYaUqQ9XuXAjo3aBHRZX5dXwhzNewH7XZX2LayLa~hnNPJ7IAakPiTxjCWAjwUGsc7HezOt6w99au6SJK~7qZf6SSsMKm~0xcbYIkQcgOanZUvlacp2sf1BBcYXg4QM0Dk1YGdp9jlKBR7Tjpb-Tt3fC6I492Hkl1rK-HOaH9BnBtXHxdN5xII6ps4LxTAU6CasO44lGw3uflzVHqAF6AA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7971581&ar=513030&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn02802.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Contributions of twofold and sixfold degenerate codons to axis 1, obtained by different CA methods, CA-AF (<strong>A</strong>), CA-RF (<strong>B</strong>), CA-RSCU (<strong>C</strong>), and WCA (<strong>D</strong>), for 241 bacterial genomes. Each point represents a genome.</p></div></div>                            <h3 scrollto-destination=7971585 id="7971585" class="section-title" data-legacy-id=s3c>3.3. Sources of intragenomic variation in synonymous codon usage among genes</h3>
<p>We applied WCA to the genomes of 241 bacterial species to identify major sources of intragenomic variation in synonymous codon usage among genes. In addition to the two gene features described earlier (<em>GC3content</em> and <em>GC3skew</em>), gene expression level (<em>Expression</em>) was also considered. In 57 genomes, WCA detected one of the three gene features, <em>GC3content</em>, <em>GC3skew</em>, and <em>Expression</em> on axis 1 but none of the features on axes 2 and 3 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). In 97 other genomes, WCA detected two of the three gene features on axes 1 and 2 but none of the gene features on axis 3. All three features were detected on the first three axes of 40 genomes, and only in nine genomes were no gene features detected on the first three axes. The results demonstrate that the three gene features can contribute to intragenomic variations in synonymous codon usage among genes, and that their relative contributions vary among different genomes.</p><p>CA of codon usage data generated axes on which no gene feature was detected. There are three possible explanations for this observation. First, in some cases, the axis was moderately correlated with one of the gene features considered here, but the correlation was not strong enough to reach the detection threshold. For example in <em>Shewanella putrefaciens</em> CN-32, the |<em>r</em>| value between axis 1 of WCA and <em>GC3content</em> (0.68) was below the threshold |<em>r</em>| value of 0.70. Secondly, although the axis was not correlated with any of the gene features considered here, it may be correlated with other relevant gene features that can be determined computationally or experimentally; e.g. protein abundance<sup><a reveal-id="DSN028C56" data-open="DSN028C56" class="link link-ref link-reveal xref-bibr">56</a></sup> and mRNA half-life.<sup><a reveal-id="DSN028C57" data-open="DSN028C57" class="link link-ref link-reveal xref-bibr">57</a></sup> Thirdly, variation among genes on the axis, even if the axis accounts for the largest fractions of the total variation among genes, may have no biological meaning. These possibilities should be kept in mind when interpreting the axes generated by CA of codon usage data.</p><p>For 10 genomes in our study that were previously analyzed by CA (Table <a reveal-id="DSN028TB2" data-open="DSN028TB2" class="link link-reveal link-table xref-fig">2</a>), we compared our findings with previous conclusions. First, <em>GC3content</em> was detected as a primary source of synonymous codon usage variation among genes in <em>E</em>. <em>coli</em> K12 MG1655, <em>M</em>. <em>genitalium</em> G37, <em>T</em>. <em>maritima</em> MSB8, and <em>H</em>. <em>pylori</em> 26695. G + C content was previously detected in these first three genomes (previous analysis for <em>H</em>. <em>pylori</em> is not directly comparable). Intragenomic variation in G + C content mostly reflects the existence of regions with anomalous nucleotide composition, putatively acquired by horizontal transfer.<sup><a reveal-id="DSN028C2" data-open="DSN028C2" class="link link-ref link-reveal xref-bibr">2</a></sup> The exception to this is <em>M</em>. <em>genitalium</em>, in which intragenomic G + C variation is continuous along the genome.<sup><a reveal-id="DSN028C58" data-open="DSN028C58" class="link link-ref link-reveal xref-bibr">58</a></sup> Thus if the WCA axis clearly separates anomalous gene clusters from other genes, the axis scores can be used to predict genes that have recently transferred.</p>                        <a id="7971593" scrollto-destination="7971593"></a>
<div content-id="DSN028TB2" class="table-modal"><div class="table-wrap"><div id="DSN028TB2" class="table-wrap-title" data-id="DSN028TB2"><div class="label -label">Table 2</div><div class="caption"><p>Gene features that are significantly correlated with one of three axes generated by WCA in 10 bacterial genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Bacterial strain </th><th>Axis 1 </th><th>Axis 2 </th><th>Axis 3 </th><th>References<sup>a</sup> </th></tr></thead><tbody><tr><td><em>B. burgdorferi</em> B31 </td><td><em>GC3skew</em> </td><td>nd<sup>b</sup> </td><td>nd </td><td><sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C39" data-open="DSN028C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSN028C40" data-open="DSN028C40" class="link link-ref link-reveal xref-bibr">40</a></sup> </td></tr><tr><td><em>C. trachomatis</em> D/UW-3/CX </td><td><em>GC3skew</em> </td><td><em>Expression</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C41" data-open="DSN028C41" class="link link-ref link-reveal xref-bibr">41</a></sup> </td></tr><tr><td><em>C. perfringens</em> 13 </td><td><em>Expression</em> </td><td>nd </td><td>nd </td><td><sup><a reveal-id="DSN028C42" data-open="DSN028C42" class="link link-ref link-reveal xref-bibr">42</a></sup> </td></tr><tr><td><em>E. coli</em> K12 MG1655 </td><td><em>GC3content</em> </td><td><em>Expression</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C23" data-open="DSN028C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSN028C43" data-open="DSN028C43" class="link link-ref link-reveal xref-bibr">43</a></sup> </td></tr><tr><td><em>H. influenzae</em> Rd KW20 </td><td><em>Expression</em> </td><td><em>GC3content</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C44" data-open="DSN028C44" class="link link-ref link-reveal xref-bibr">44</a></sup> </td></tr><tr><td><em>H. pylori</em> 26695 </td><td><em>GC3content</em> </td><td><em>GC3skew</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C45" data-open="DSN028C45" class="link link-ref link-reveal xref-bibr">45</a></sup> </td></tr><tr><td><em>M. genitalium</em> G37 </td><td><em>GC3content</em> </td><td>nd </td><td>nd </td><td><sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C46" data-open="DSN028C46" class="link link-ref link-reveal xref-bibr">46</a></sup> </td></tr><tr><td><em>R. prowazekii</em> Madrid E </td><td><em>GC3skew</em> </td><td><em>GC3content</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C47" data-open="DSN028C47" class="link link-ref link-reveal xref-bibr">47</a></sup> </td></tr><tr><td><em>T. maritima</em> MSB8 </td><td><em>GC3content</em> </td><td>nd </td><td>nd </td><td><sup><a reveal-id="DSN028C22" data-open="DSN028C22" class="link link-ref link-reveal xref-bibr">22</a></sup> </td></tr><tr><td><em>T. pallidum</em> Nichols </td><td><em>GC3skew</em> </td><td><em>GC3content</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C39" data-open="DSN028C39" class="link link-ref link-reveal xref-bibr">39</a></sup> </td></tr></tbody></table></div><div content-id="DSN028TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Bacterial strain </th><th>Axis 1 </th><th>Axis 2 </th><th>Axis 3 </th><th>References<sup>a</sup> </th></tr></thead><tbody><tr><td><em>B. burgdorferi</em> B31 </td><td><em>GC3skew</em> </td><td>nd<sup>b</sup> </td><td>nd </td><td><sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C39" data-open="DSN028C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSN028C40" data-open="DSN028C40" class="link link-ref link-reveal xref-bibr">40</a></sup> </td></tr><tr><td><em>C. trachomatis</em> D/UW-3/CX </td><td><em>GC3skew</em> </td><td><em>Expression</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C41" data-open="DSN028C41" class="link link-ref link-reveal xref-bibr">41</a></sup> </td></tr><tr><td><em>C. perfringens</em> 13 </td><td><em>Expression</em> </td><td>nd </td><td>nd </td><td><sup><a reveal-id="DSN028C42" data-open="DSN028C42" class="link link-ref link-reveal xref-bibr">42</a></sup> </td></tr><tr><td><em>E. coli</em> K12 MG1655 </td><td><em>GC3content</em> </td><td><em>Expression</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C23" data-open="DSN028C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSN028C43" data-open="DSN028C43" class="link link-ref link-reveal xref-bibr">43</a></sup> </td></tr><tr><td><em>H. influenzae</em> Rd KW20 </td><td><em>Expression</em> </td><td><em>GC3content</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C44" data-open="DSN028C44" class="link link-ref link-reveal xref-bibr">44</a></sup> </td></tr><tr><td><em>H. pylori</em> 26695 </td><td><em>GC3content</em> </td><td><em>GC3skew</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C45" data-open="DSN028C45" class="link link-ref link-reveal xref-bibr">45</a></sup> </td></tr><tr><td><em>M. genitalium</em> G37 </td><td><em>GC3content</em> </td><td>nd </td><td>nd </td><td><sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C46" data-open="DSN028C46" class="link link-ref link-reveal xref-bibr">46</a></sup> </td></tr><tr><td><em>R. prowazekii</em> Madrid E </td><td><em>GC3skew</em> </td><td><em>GC3content</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C47" data-open="DSN028C47" class="link link-ref link-reveal xref-bibr">47</a></sup> </td></tr><tr><td><em>T. maritima</em> MSB8 </td><td><em>GC3content</em> </td><td>nd </td><td>nd </td><td><sup><a reveal-id="DSN028C22" data-open="DSN028C22" class="link link-ref link-reveal xref-bibr">22</a></sup> </td></tr><tr><td><em>T. pallidum</em> Nichols </td><td><em>GC3skew</em> </td><td><em>GC3content</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C39" data-open="DSN028C39" class="link link-ref link-reveal xref-bibr">39</a></sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Previous studies, whose results do not necessarily agree with those shown here. See Section 3.3 for conflicts.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>nd, none of the gene features considered here were detected.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7971593" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN028TB2" class="table-wrap-title" data-id="DSN028TB2"><div class="label -label">Table 2</div><div class="caption"><p>Gene features that are significantly correlated with one of three axes generated by WCA in 10 bacterial genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Bacterial strain </th><th>Axis 1 </th><th>Axis 2 </th><th>Axis 3 </th><th>References<sup>a</sup> </th></tr></thead><tbody><tr><td><em>B. burgdorferi</em> B31 </td><td><em>GC3skew</em> </td><td>nd<sup>b</sup> </td><td>nd </td><td><sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C39" data-open="DSN028C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSN028C40" data-open="DSN028C40" class="link link-ref link-reveal xref-bibr">40</a></sup> </td></tr><tr><td><em>C. trachomatis</em> D/UW-3/CX </td><td><em>GC3skew</em> </td><td><em>Expression</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C41" data-open="DSN028C41" class="link link-ref link-reveal xref-bibr">41</a></sup> </td></tr><tr><td><em>C. perfringens</em> 13 </td><td><em>Expression</em> </td><td>nd </td><td>nd </td><td><sup><a reveal-id="DSN028C42" data-open="DSN028C42" class="link link-ref link-reveal xref-bibr">42</a></sup> </td></tr><tr><td><em>E. coli</em> K12 MG1655 </td><td><em>GC3content</em> </td><td><em>Expression</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C23" data-open="DSN028C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSN028C43" data-open="DSN028C43" class="link link-ref link-reveal xref-bibr">43</a></sup> </td></tr><tr><td><em>H. influenzae</em> Rd KW20 </td><td><em>Expression</em> </td><td><em>GC3content</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C44" data-open="DSN028C44" class="link link-ref link-reveal xref-bibr">44</a></sup> </td></tr><tr><td><em>H. pylori</em> 26695 </td><td><em>GC3content</em> </td><td><em>GC3skew</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C45" data-open="DSN028C45" class="link link-ref link-reveal xref-bibr">45</a></sup> </td></tr><tr><td><em>M. genitalium</em> G37 </td><td><em>GC3content</em> </td><td>nd </td><td>nd </td><td><sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C46" data-open="DSN028C46" class="link link-ref link-reveal xref-bibr">46</a></sup> </td></tr><tr><td><em>R. prowazekii</em> Madrid E </td><td><em>GC3skew</em> </td><td><em>GC3content</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C47" data-open="DSN028C47" class="link link-ref link-reveal xref-bibr">47</a></sup> </td></tr><tr><td><em>T. maritima</em> MSB8 </td><td><em>GC3content</em> </td><td>nd </td><td>nd </td><td><sup><a reveal-id="DSN028C22" data-open="DSN028C22" class="link link-ref link-reveal xref-bibr">22</a></sup> </td></tr><tr><td><em>T. pallidum</em> Nichols </td><td><em>GC3skew</em> </td><td><em>GC3content</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C39" data-open="DSN028C39" class="link link-ref link-reveal xref-bibr">39</a></sup> </td></tr></tbody></table></div><div content-id="DSN028TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Bacterial strain </th><th>Axis 1 </th><th>Axis 2 </th><th>Axis 3 </th><th>References<sup>a</sup> </th></tr></thead><tbody><tr><td><em>B. burgdorferi</em> B31 </td><td><em>GC3skew</em> </td><td>nd<sup>b</sup> </td><td>nd </td><td><sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C39" data-open="DSN028C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DSN028C40" data-open="DSN028C40" class="link link-ref link-reveal xref-bibr">40</a></sup> </td></tr><tr><td><em>C. trachomatis</em> D/UW-3/CX </td><td><em>GC3skew</em> </td><td><em>Expression</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C41" data-open="DSN028C41" class="link link-ref link-reveal xref-bibr">41</a></sup> </td></tr><tr><td><em>C. perfringens</em> 13 </td><td><em>Expression</em> </td><td>nd </td><td>nd </td><td><sup><a reveal-id="DSN028C42" data-open="DSN028C42" class="link link-ref link-reveal xref-bibr">42</a></sup> </td></tr><tr><td><em>E. coli</em> K12 MG1655 </td><td><em>GC3content</em> </td><td><em>Expression</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C23" data-open="DSN028C23" class="link link-ref link-reveal xref-bibr">23</a>,<a reveal-id="DSN028C43" data-open="DSN028C43" class="link link-ref link-reveal xref-bibr">43</a></sup> </td></tr><tr><td><em>H. influenzae</em> Rd KW20 </td><td><em>Expression</em> </td><td><em>GC3content</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C44" data-open="DSN028C44" class="link link-ref link-reveal xref-bibr">44</a></sup> </td></tr><tr><td><em>H. pylori</em> 26695 </td><td><em>GC3content</em> </td><td><em>GC3skew</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C45" data-open="DSN028C45" class="link link-ref link-reveal xref-bibr">45</a></sup> </td></tr><tr><td><em>M. genitalium</em> G37 </td><td><em>GC3content</em> </td><td>nd </td><td>nd </td><td><sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSN028C46" data-open="DSN028C46" class="link link-ref link-reveal xref-bibr">46</a></sup> </td></tr><tr><td><em>R. prowazekii</em> Madrid E </td><td><em>GC3skew</em> </td><td><em>GC3content</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C47" data-open="DSN028C47" class="link link-ref link-reveal xref-bibr">47</a></sup> </td></tr><tr><td><em>T. maritima</em> MSB8 </td><td><em>GC3content</em> </td><td>nd </td><td>nd </td><td><sup><a reveal-id="DSN028C22" data-open="DSN028C22" class="link link-ref link-reveal xref-bibr">22</a></sup> </td></tr><tr><td><em>T. pallidum</em> Nichols </td><td><em>GC3skew</em> </td><td><em>GC3content</em> </td><td>nd </td><td><sup><a reveal-id="DSN028C39" data-open="DSN028C39" class="link link-ref link-reveal xref-bibr">39</a></sup> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Previous studies, whose results do not necessarily agree with those shown here. See Section 3.3 for conflicts.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>nd, none of the gene features considered here were detected.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7971593" target="_blank">View Large</a></div></div><p>The second feature, <em>GC3skew</em> was detected as a primary source of synonymous codon usage variation among genes in <em>B</em>. <em>burgdorferi</em> B31, <em>C</em>. <em>trachomatis</em> D/UW-3/CX, <em>R</em>. <em>prowazekii</em> Madrid E, and <em>T</em>. <em>pallidum</em> Nichols (Table <a reveal-id="DSN028TB2" data-open="DSN028TB2" class="link link-reveal link-table xref-fig">2</a> and Fig. <a reveal-id="DSN028F1" data-open="DSN028F1" class="link link-reveal link-table xref-fig">1</a>). Intragenomic variation in <em>GC3skew</em> presumably reflects differences in mutational bias between the leading and lagging strands of replication.<sup><a reveal-id="DSN028C5" data-open="DSN028C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSN028C6" data-open="DSN028C6" class="link link-ref link-reveal xref-bibr">6</a></sup> This mutational bias was previously detected in each of these genomes, except <em>R</em>. <em>prowazekii</em>.<sup><a reveal-id="DSN028C47" data-open="DSN028C47" class="link link-ref link-reveal xref-bibr">47</a></sup> Thus in genomes where <em>GC3skew</em> is detected on axis 1 of WCA, the axis scores can be used to predict whether the gene is located on the leading or lagging strands.</p><p>The third feature, <em>Expression</em>, was detected as a major source of synonymous codon usage variation among genes in <em>C</em>. <em>trachomatis</em> D/UW-3/CX, <em>C</em>. <em>perfringens</em> 13, <em>E</em>. <em>coli</em> K12 MG1655 and <em>H</em>. <em>influenzae</em> Rd KW20, which is consistent with previous findings (Table <a reveal-id="DSN028TB2" data-open="DSN028TB2" class="link link-reveal link-table xref-fig">2</a>). The relative contribution of <em>Expression</em> varies among different genomes; e.g. <em>Expression</em> is a primary source in <em>H</em>. <em>influenzae</em>, while it is a secondary source in <em>E</em>. <em>coli</em>. The anomalous codon usage of highly expressed genes presumably reflects natural selection for optimal codons that are translated more efficiently and accurately; so-called translational selection.<sup><a reveal-id="DSN028C7" data-open="DSN028C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSN028C8" data-open="DSN028C8" class="link link-ref link-reveal xref-bibr">8</a></sup> In <em>B</em>. <em>burgdorferi</em> and <em>M</em>. <em>genitalium</em>, conflicting conclusions regarding the presence or absence of translational selection on synonymous codon usage have been reported.<sup><a reveal-id="DSN028C21" data-open="DSN028C21" class="link link-ref link-reveal xref-bibr">21</a></sup> In the present analysis, <em>Expression</em> was not detected in these two genomes, suggesting there is no evidence for translational selection. This is in agreement with conclusions drawn using a different statistical method.<sup><a reveal-id="DSN028C10" data-open="DSN028C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Thus in genomes where <em>Expression</em> is detected by WCA, the axis scores can be used to predict gene expression level and compared with experimental expression data obtained by DNA microarray (transcriptomes) and 2D gel electrophoresis (proteomes).</p>                            <h3 scrollto-destination=7971596 id="7971596" class="section-title" data-legacy-id=s3d>3.4. Conclusion</h3>
<p>Of the four CA methods, WCA was found to be most useful for the analysis of synonymous codon usage. Using WCA, it may be possible to find new factors that can explain variation in synonymous codon usage among genes, and improve the accuracy of identifying genes that have been horizontally transferred or are highly expressed.</p>                            <h2 scrollto-destination=7971598 id="7971598" class="section-title" data-legacy-id=s4>4. Availability</h2>
<p>All analyses are implemented using G-language Genome Analysis Environment version 1.8.3,<sup><a reveal-id="DSN028C59" data-open="DSN028C59" class="link link-ref link-reveal xref-bibr">59</a>,<a reveal-id="DSN028C60" data-open="DSN028C60" class="link link-ref link-reveal xref-bibr">60</a></sup> available at <a class="link link-uri" href="http://www.g-language.org/">http://www.g-language.org/</a>.</p>                            <h2 scrollto-destination=7971600 id="7971600" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=7971602 id="7971602" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This project was supported by the Microbial Genome Sequencing Program of the National Science Foundation (EF-0627988), and by the National Institutes of Health grant R01 GM073821 from the National Institute of General Medical Sciences, and COBRE and INBRE grants P20RR016454 and P20RR16448 from the National Center for Research Resources, National Institutes of Health.</p>                            <h2 scrollto-destination=7971604 id="7971604" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Kazuharu Arakawa (Institute for Advanced Biosciences, Keio University) for his technical advice on the G-language Genome Analysis Environment and Christopher J. Williams (Department of Statistics, University of Idaho) for statistical advice.</p>                            <h2 scrollto-destination=7971606 id="7971606" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSN028C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>,  <div class="name"><div class="surname">Cowe</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D. G.</div></div>,  <div class="name"><div class="surname">Shields</div> <div class="given-names">D. C.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K. H.</div></div>,  <div class="name"><div class="surname">Wright</div> <div class="given-names">F.</div></div>. </div><div class="article-title">Codon usage patterns in <em>Escherichia coli</em>, <em>Bacillus subtilis</em>, <em>Saccharomyces cerevisiae</em>, <em>Schizosaccharomyces pombe</em>, <em>Drosophila melanogaster</em> and <em>Homo sapiens</em>; a review of the considerable within-species diversity</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1988</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">8207</div>-<div class="lpage">8211</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20patterns%20in%20Escherichia%20coli%2C%20Bacillus%20subtilis%2C%20Saccharomyces%20cerevisiae%2C%20Schizosaccharomyces%20pombe%2C%20Drosophila%20melanogaster%20and%20Homo%20sapiens%3B%20a%20review%20of%20the%20considerable%20within-species%20diversity&amp;author=P.%20M.%20Sharp&amp;author=E.%20Cowe&amp;author=D.%20G.%20Higgins&amp;author=D.%20C.%20Shields&amp;author=K.%20H.%20Wolfe&amp;author=F.%20Wright&amp;publication_year=1988&amp;journal=Nucleic%20Acids%20Res.&amp;volume=16&amp;pages=8207-8211" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/16.17.8207" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F16.17.8207" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F16.17.8207"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3138659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Detecting anomalous gene clusters and pathogenicity islands in diverse bacterial genomes</div>, <div class="source">Trends Microbiol.</div> , <div class="year">2001</div>, vol. <div class="volume">9</div> (pg. <div class="fpage">335</div>-<div class="lpage">343</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20anomalous%20gene%20clusters%20and%20pathogenicity%20islands%20in%20diverse%20bacterial%20genomes&amp;author=S.%20Karlin&amp;publication_year=2001&amp;journal=Trends%20Microbiol.&amp;volume=9&amp;pages=335-343" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0966-842X(01)02079-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0966-842X(01)02079-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0966-842X(01)02079-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11435108" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lawrence</div> <div class="given-names">J. G.</div></div>,  <div class="name"><div class="surname">Ochman</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Amelioration of bacterial genomes: rates of change and exchange</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1997</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">383</div>-<div class="lpage">397</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amelioration%20of%20bacterial%20genomes%3A%20rates%20of%20change%20and%20exchange&amp;author=J.%20G.%20Lawrence&amp;author=H.%20Ochman&amp;publication_year=1997&amp;journal=J.%20Mol.%20Evol.&amp;volume=44&amp;pages=383-397" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9089078" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ochman</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Lawrence</div> <div class="given-names">J. G.</div></div>,  <div class="name"><div class="surname">Groisman</div> <div class="given-names">E. A.</div></div>. </div><div class="article-title">Lateral gene transfer and the nature of bacterial innovation</div>, <div class="source">Nature</div> , <div class="year">2000</div>, vol. <div class="volume">405</div> (pg. <div class="fpage">299</div>-<div class="lpage">304</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lateral%20gene%20transfer%20and%20the%20nature%20of%20bacterial%20innovation&amp;author=H.%20Ochman&amp;author=J.%20G.%20Lawrence&amp;author=E.%20A.%20Groisman&amp;publication_year=2000&amp;journal=Nature&amp;volume=405&amp;pages=299-304" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35012500" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35012500" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35012500"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10830951" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lobry</div> <div class="given-names">J. R.</div></div>. </div><div class="article-title">Asymmetric substitution patterns in the two DNA strands of bacteria</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1996</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">660</div>-<div class="lpage">665</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Asymmetric%20substitution%20patterns%20in%20the%20two%20DNA%20strands%20of%20bacteria&amp;author=J.%20R.%20Lobry&amp;publication_year=1996&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=13&amp;pages=660-665" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025626" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025626" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a025626"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8676740" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McLean</div> <div class="given-names">M. J.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K. H.</div></div>,  <div class="name"><div class="surname">Devine</div> <div class="given-names">K. M.</div></div>. </div><div class="article-title">Base composition skews, replication orientation, and gene orientation in 12 prokaryote genomes</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1998</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">691</div>-<div class="lpage">696</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base%20composition%20skews%2C%20replication%20orientation%2C%20and%20gene%20orientation%20in%2012%20prokaryote%20genomes&amp;author=M.%20J.%20McLean&amp;author=K.%20H.%20Wolfe&amp;author=K.%20M.%20Devine&amp;publication_year=1998&amp;journal=J.%20Mol.%20Evol.&amp;volume=47&amp;pages=691-696" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006428" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006428" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006428"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9847411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eyre-Walker</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Synonymous codon bias is related to gene length in <em>Escherichia coli</em>: selection for translational accuracy?</div>, <div class="source">Mol Biol Evol</div> , <div class="year">1996</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">864</div>-<div class="lpage">872</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20bias%20is%20related%20to%20gene%20length%20in%20Escherichia%20coli%3A%20selection%20for%20translational%20accuracy%3F&amp;author=A.%20Eyre-Walker&amp;publication_year=1996&amp;journal=Mol%20Biol%20Evol&amp;volume=13&amp;pages=864-872" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a025646" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a025646" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a025646"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8754221" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Codon usage and tRNA content in unicellular and multicellular organisms</div>, <div class="source">Mol Biol Evol</div> , <div class="year">1985</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">13</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20tRNA%20content%20in%20unicellular%20and%20multicellular%20organisms&amp;author=T.%20Ikemura&amp;publication_year=1985&amp;journal=Mol%20Biol%20Evol&amp;volume=2&amp;pages=13-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3916708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carbone</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kepes</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Zinovyev</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Codon bias signatures, organization of microorganisms in codon space, and lifestyle</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2005</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">547</div>-<div class="lpage">561</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20bias%20signatures%2C%20organization%20of%20microorganisms%20in%20codon%20space%2C%20and%20lifestyle&amp;author=A.%20Carbone&amp;author=F.%20Kepes&amp;author=A.%20Zinovyev&amp;publication_year=2005&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=22&amp;pages=547-561" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msi040" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsi040" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsi040"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15537809" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>,  <div class="name"><div class="surname">Bailes</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Grocock</div> <div class="given-names">R. J.</div></div>,  <div class="name"><div class="surname">Peden</div> <div class="given-names">J. F.</div></div>,  <div class="name"><div class="surname">Sockett</div> <div class="given-names">R. E.</div></div>. </div><div class="article-title">Variation in the strength of selected codon usage bias among bacteria</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2005</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">1141</div>-<div class="lpage">1153</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Variation%20in%20the%20strength%20of%20selected%20codon%20usage%20bias%20among%20bacteria&amp;author=P.%20M.%20Sharp&amp;author=E.%20Bailes&amp;author=R.%20J.%20Grocock&amp;author=J.%20F.%20Peden&amp;author=R.%20E.%20Sockett&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res.&amp;volume=33&amp;pages=1141-1153" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki242" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki242" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki242"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Willenbrock</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Friis</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Friis</div> <div class="given-names">A. S.</div></div>,  <div class="name"><div class="surname">Ussery</div> <div class="given-names">D. W.</div></div>. </div><div class="article-title">An environmental signature for 323 microbial genomes based on codon adaptation indices</div>, <div class="source">Genome Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">R114</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20environmental%20signature%20for%20323%20microbial%20genomes%20based%20on%20codon%20adaptation%20indices&amp;author=H.%20Willenbrock&amp;author=C.%20Friis&amp;author=A.%20S.%20Friis&amp;author=D.%20W.%20Ussery&amp;publication_year=2006&amp;journal=Genome%20Biol.&amp;volume=7&amp;pages=R114" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2006-7-12-r114" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2006-7-12-r114" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2006-7-12-r114"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17156429" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garcia-Vallve</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Guzman</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Montero</div> <div class="given-names">M. A.</div></div>,  <div class="name"><div class="surname">Romeu</div> <div class="given-names">A.</div></div>. </div><div class="article-title">HGT-DB: a database of putative horizontally transferred genes in prokaryotic complete genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">187</div>-<div class="lpage">189</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HGT-DB%3A%20a%20database%20of%20putative%20horizontally%20transferred%20genes%20in%20prokaryotic%20complete%20genomes&amp;author=S.%20Garcia-Vallve&amp;author=E.%20Guzman&amp;author=M.%20A.%20Montero&amp;author=A.%20Romeu&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=187-189" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koski</div> <div class="given-names">L. B.</div></div>,  <div class="name"><div class="surname">Morton</div> <div class="given-names">R. A.</div></div>,  <div class="name"><div class="surname">Golding</div> <div class="given-names">G. B.</div></div>. </div><div class="article-title">Codon bias and base composition are poor indicators of horizontally transferred genes</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2001</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">404</div>-<div class="lpage">412</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20bias%20and%20base%20composition%20are%20poor%20indicators%20of%20horizontally%20transferred%20genes&amp;author=L.%20B.%20Koski&amp;author=R.%20A.%20Morton&amp;author=G.%20B.%20Golding&amp;publication_year=2001&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=18&amp;pages=404-412" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a003816" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a003816" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a003816"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11230541" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mrazek</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Detecting alien genes in bacterial genomes</div>, <div class="source">Ann. N Y Acad. Sci.</div> , <div class="year">1999</div>, vol. <div class="volume">870</div> (pg. <div class="fpage">314</div>-<div class="lpage">329</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detecting%20alien%20genes%20in%20bacterial%20genomes&amp;author=J.%20Mrazek&amp;author=S.%20Karlin&amp;publication_year=1999&amp;journal=Ann.%20N%20Y%20Acad.%20Sci.&amp;volume=870&amp;pages=314-329" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1749-6632.1999.tb08893.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1749-6632.1999.tb08893.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1749-6632.1999.tb08893.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10415493" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Henry</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>. </div><div class="article-title">Predicting gene expression level from codon usage bias</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">10</div>-<div class="lpage">12</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predicting%20gene%20expression%20level%20from%20codon%20usage%20bias&amp;author=I.%20Henry&amp;author=P.%20M.%20Sharp&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=10-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msl148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsl148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsl148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17038449" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karlin</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mrazek</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Campbell</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Kaiser</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Characterizations of highly expressed genes of four fast-growing bacteria</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2001</div>, vol. <div class="volume">183</div> (pg. <div class="fpage">5025</div>-<div class="lpage">5040</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterizations%20of%20highly%20expressed%20genes%20of%20four%20fast-growing%20bacteria&amp;author=S.%20Karlin&amp;author=J.%20Mrazek&amp;author=A.%20Campbell&amp;author=D.%20Kaiser&amp;publication_year=2001&amp;journal=J.%20Bacteriol.&amp;volume=183&amp;pages=5025-5040" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.183.17.5025-5040.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.183.17.5025-5040.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.183.17.5025-5040.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11489855" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Puigbo</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Romeu</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Garcia-Vallve</div> <div class="given-names">S.</div></div>. </div><div class="article-title">HEG-DB: a database of predicted highly expressed genes in prokaryotic complete genomes under translational selection</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D524</div>-<div class="lpage">D527</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HEG-DB%3A%20a%20database%20of%20predicted%20highly%20expressed%20genes%20in%20prokaryotic%20complete%20genomes%20under%20translational%20selection&amp;author=P.%20Puigbo&amp;author=A.%20Romeu&amp;author=S.%20Garcia-Vallve&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D524-D527" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm831" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm831" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm831"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17933767" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W. H.</div></div>. </div><div class="article-title">The codon adaptation index – a measure of directional synonymous codon usage bias, and its potential applications</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1987</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">1281</div>-<div class="lpage">1295</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20codon%20adaptation%20index%20%E2%80%93%20a%20measure%20of%20directional%20synonymous%20codon%20usage%20bias%2C%20and%20its%20potential%20applications&amp;author=P.%20M.%20Sharp&amp;author=W.%20H.%20Li&amp;publication_year=1987&amp;journal=Nucleic%20Acids%20Res.&amp;volume=15&amp;pages=1281-1295" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/15.3.1281" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F15.3.1281" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F15.3.1281"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3547335" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fellenberg</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hauser</div> <div class="given-names">N. C.</div></div>,  <div class="name"><div class="surname">Brors</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Neutzner</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hoheisel</div> <div class="given-names">J. D.</div></div>,  <div class="name"><div class="surname">Vingron</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Correspondence analysis applied to microarray data</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">10781</div>-<div class="lpage">10786</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correspondence%20analysis%20applied%20to%20microarray%20data&amp;author=K.%20Fellenberg&amp;author=N.%20C.%20Hauser&amp;author=B.%20Brors&amp;author=A.%20Neutzner&amp;author=J.%20D.%20Hoheisel&amp;author=M.%20Vingron&amp;publication_year=2001&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=10781-10786" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.181597298" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.181597298" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.181597298"> </span></div> </div></div></div></div></div><div content-id="DSN028C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grantham</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Gautier</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Mercier</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Pave</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Codon catalog usage and the genome hypothesis</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1980</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">r49</div>-<div class="lpage">r62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20catalog%20usage%20and%20the%20genome%20hypothesis&amp;author=R.%20Grantham&amp;author=C.%20Gautier&amp;author=M.%20Gouy&amp;author=R.%20Mercier&amp;author=A.%20Pave&amp;publication_year=1980&amp;journal=Nucleic%20Acids%20Res.&amp;volume=8&amp;pages=r49-r62" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6986610" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perriere</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Thioulouse</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Use and misuse of correspondence analysis in codon usage studies</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">4548</div>-<div class="lpage">4555</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20and%20misuse%20of%20correspondence%20analysis%20in%20codon%20usage%20studies&amp;author=G.%20Perriere&amp;author=J.%20Thioulouse&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=4548-4555" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkf565" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkf565" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkf565"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12384602" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zavala</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Naya</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Romero</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Musto</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Trends in codon and amino acid usage in <em>Thermotoga maritima</em></div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2002</div>, vol. <div class="volume">54</div> (pg. <div class="fpage">563</div>-<div class="lpage">568</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trends%20in%20codon%20and%20amino%20acid%20usage%20in%20Thermotoga%20maritima&amp;author=A.%20Zavala&amp;author=H.%20Naya&amp;author=H.%20Romero&amp;author=H.%20Musto&amp;publication_year=2002&amp;journal=J.%20Mol.%20Evol.&amp;volume=54&amp;pages=563-568" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-001-0040-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-001-0040-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-001-0040-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11965430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">dos Reis</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wernisch</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Savva</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Unexpected correlations between gene expression and codon usage bias from microarray data for the whole <em>Escherichia coli</em> K-12 genome</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">6976</div>-<div class="lpage">6985</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unexpected%20correlations%20between%20gene%20expression%20and%20codon%20usage%20bias%20from%20microarray%20data%20for%20the%20whole%20Escherichia%20coli%20K-12%20genome&amp;author=M.%20dos%20Reis&amp;author=L.%20Wernisch&amp;author=R.%20Savva&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=6976-6985" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg897" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg897" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg897"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14627830" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lerat</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Biemont</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Capy</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Codon usage and the origin of <em>P</em> elements</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2000</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">467</div>-<div class="lpage">468</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20the%20origin%20of%20P%20elements&amp;author=E.%20Lerat&amp;author=C.%20Biemont&amp;author=P.%20Capy&amp;publication_year=2000&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=17&amp;pages=467-468" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026326" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026326" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026326"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10723747" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Charif</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Thioulouse</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Lobry</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Perriere</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Online synonymous codon usage analyses with the ade4 and seqinR packages</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">545</div>-<div class="lpage">547</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Online%20synonymous%20codon%20usage%20analyses%20with%20the%20ade4%20and%20seqinR%20packages&amp;author=D.%20Charif&amp;author=J.%20Thioulouse&amp;author=J.%20R.%20Lobry&amp;author=G.%20Perriere&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=545-547" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15374859" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Das</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Paul</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Chatterjee</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Dutta</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Codon and amino acid usage in two major human pathogens of genus <em>Bartonella</em> – optimization between replicational–transcriptional selection, translational control and cost minimization</div>, <div class="source">DNA Res.</div> , <div class="year">2005</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">91</div>-<div class="lpage">102</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20and%20amino%20acid%20usage%20in%20two%20major%20human%20pathogens%20of%20genus%20Bartonella%20%E2%80%93%20optimization%20between%20replicational%E2%80%93transcriptional%20selection%2C%20translational%20control%20and%20cost%20minimization&amp;author=S.%20Das&amp;author=S.%20Paul&amp;author=S.%20Chatterjee&amp;author=C.%20Dutta&amp;publication_year=2005&amp;journal=DNA%20Res.&amp;volume=12&amp;pages=91-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/12.2.91" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F12.2.91" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F12.2.91"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16303741" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Basak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ghosh</div> <div class="given-names">T. C.</div></div>. </div><div class="article-title">Temperature adaptation of synonymous codon usage in different functional categories of genes: a comparative study between homologous genes of <em>Methanococcus jannaschii</em> and <em>Methanococcus maripaludis</em></div>, <div class="source">FEBS Lett.</div> , <div class="year">2006</div>, vol. <div class="volume">580</div> (pg. <div class="fpage">3895</div>-<div class="lpage">3899</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Temperature%20adaptation%20of%20synonymous%20codon%20usage%20in%20different%20functional%20categories%20of%20genes%3A%20a%20comparative%20study%20between%20homologous%20genes%20of%20Methanococcus%20jannaschii%20and%20Methanococcus%20maripaludis&amp;author=S.%20Basak&amp;author=T.%20C.%20Ghosh&amp;publication_year=2006&amp;journal=FEBS%20Lett.&amp;volume=580&amp;pages=3895-3899" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2006.06.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2006.06.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2006.06.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16797014" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Das</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Paul</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bag</div> <div class="given-names">S. K.</div></div>,  <div class="name"><div class="surname">Dutta</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Analysis of <em>Nanoarchaeum equitans</em> genome and proteome composition: indications for hyperthermophilic and parasitic adaptation</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">186</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20Nanoarchaeum%20equitans%20genome%20and%20proteome%20composition%3A%20indications%20for%20hyperthermophilic%20and%20parasitic%20adaptation&amp;author=S.%20Das&amp;author=S.%20Paul&amp;author=S.%20K.%20Bag&amp;author=C.%20Dutta&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=186" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-186" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-186" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-186"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16869956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Das</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Paul</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Dutta</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Evolutionary constraints on codon and amino acid usage in two strains of human pathogenic actinobacteria <em>Tropheryma whipplei</em></div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2006</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">645</div>-<div class="lpage">658</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20constraints%20on%20codon%20and%20amino%20acid%20usage%20in%20two%20strains%20of%20human%20pathogenic%20actinobacteria%20Tropheryma%20whipplei&amp;author=S.%20Das&amp;author=S.%20Paul&amp;author=C.%20Dutta&amp;publication_year=2006&amp;journal=J.%20Mol.%20Evol.&amp;volume=62&amp;pages=645-658" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-005-0164-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-005-0164-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-005-0164-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16557339" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Das</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Paul</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Dutta</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Synonymous codon usage in adenoviruses: influence of mutation, selection and protein hydropathy</div>, <div class="source">Virus Res.</div> , <div class="year">2006</div>, vol. <div class="volume">117</div> (pg. <div class="fpage">227</div>-<div class="lpage">236</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20usage%20in%20adenoviruses%3A%20influence%20of%20mutation%2C%20selection%20and%20protein%20hydropathy&amp;author=S.%20Das&amp;author=S.%20Paul&amp;author=C.%20Dutta&amp;publication_year=2006&amp;journal=Virus%20Res.&amp;volume=117&amp;pages=227-236" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.virusres.2005.10.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.virusres.2005.10.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.virusres.2005.10.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16307819" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ahn</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Son</div> <div class="given-names">H. S.</div></div>. </div><div class="article-title">Comparative study of the hemagglutinin and neuraminidase genes of influenza A virus H3N2, H9N2, and H5N1 subtypes using bioinformatics techniques</div>, <div class="source">Can. J. Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">830</div>-<div class="lpage">839</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20study%20of%20the%20hemagglutinin%20and%20neuraminidase%20genes%20of%20influenza%20A%20virus%20H3N2%2C%20H9N2%2C%20and%20H5N1%20subtypes%20using%20bioinformatics%20techniques&amp;author=I.%20Ahn&amp;author=H.%20S.%20Son&amp;publication_year=2007&amp;journal=Can.%20J.%20Microbiol.&amp;volume=53&amp;pages=830-839" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/W07-044" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2FW07-044" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2FW07-044"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17898838" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ranjan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Vidyarthi</div> <div class="given-names">A. S.</div></div>,  <div class="name"><div class="surname">Poddar</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Evaluation of codon bias perspectives in phage therapy of <em>Mycobacterium tuberculosis</em> by multivariate analysis</div>, <div class="source">In Silico Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">7</div> pg. <div class="fpage">0030</div> </div></div></div></div><div content-id="DSN028C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sau</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Gupta</div> <div class="given-names">S. K.</div></div>,  <div class="name"><div class="surname">Sau</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Mandal</div> <div class="given-names">S. C.</div></div>,  <div class="name"><div class="surname">Ghosh</div> <div class="given-names">T. C.</div></div>. </div><div class="article-title">Studies on synonymous codon and amino acid usage biases in the broad-host range bacteriophage KVP40</div>, <div class="source">J. Microbiol.</div> , <div class="year">2007</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">58</div>-<div class="lpage">63</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Studies%20on%20synonymous%20codon%20and%20amino%20acid%20usage%20biases%20in%20the%20broad-host%20range%20bacteriophage%20KVP40&amp;author=K.%20Sau&amp;author=S.%20K.%20Gupta&amp;author=S.%20Sau&amp;author=S.%20C.%20Mandal&amp;author=T.%20C.%20Ghosh&amp;publication_year=2007&amp;journal=J.%20Microbiol.&amp;volume=45&amp;pages=58-63" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17342057" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sen</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Sur</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bose</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Analysis of codon usage patterns and predicted highly expressed genes for six phytopathogenic <em>Xanthomonas</em> genomes shows a high degree of conservation</div>, <div class="source">In Silico Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">547</div>-<div class="lpage">558</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20codon%20usage%20patterns%20and%20predicted%20highly%20expressed%20genes%20for%20six%20phytopathogenic%20Xanthomonas%20genomes%20shows%20a%20high%20degree%20of%20conservation&amp;author=G.%20Sen&amp;author=S.%20Sur&amp;author=D.%20Bose&amp;publication_year=2007&amp;journal=In%20Silico%20Biol.&amp;volume=7&amp;pages=547-558" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18391244" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">H. C.</div></div>,  <div class="name"><div class="surname">Hickey</div> <div class="given-names">D. A.</div></div>. </div><div class="article-title">Rapid divergence of codon usage patterns within the rice genome</div>, <div class="source">BMC Evol. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">7</div> <div class="supplement">Suppl 1</div>pg. <div class="fpage">S6</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20divergence%20of%20codon%20usage%20patterns%20within%20the%20rice%20genome&amp;author=H.%20C.%20Wang&amp;author=D.%20A.%20Hickey&amp;publication_year=2007&amp;journal=BMC%20Evol.%20Biol.&amp;volume=7&amp;pages=S6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-7-S1-S6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-7-S1-S6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-7-S1-S6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17288579" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhong</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Zhao</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>. </div><div class="article-title">Mutation pressure shapes codon usage in the GC-rich genome of foot-and-mouth disease virus</div>, <div class="source">Virus Genes</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">767</div>-<div class="lpage">776</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutation%20pressure%20shapes%20codon%20usage%20in%20the%20GC-rich%20genome%20of%20foot-and-mouth%20disease%20virus&amp;author=J.%20Zhong&amp;author=Y.%20Li&amp;author=S.%20Zhao&amp;author=S.%20Liu&amp;author=Z.%20Zhang&amp;publication_year=2007&amp;journal=Virus%20Genes&amp;volume=35&amp;pages=767-776" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11262-007-0159-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11262-007-0159-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11262-007-0159-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17768673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhao</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Analysis of synonymous codon usage in 11 Human Bocavirus isolates</div>, <div class="source">Biosystems</div> , <div class="year">2008</div>, vol. <div class="volume">92</div> (pg. <div class="fpage">207</div>-<div class="lpage">214</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20synonymous%20codon%20usage%20in%2011%20Human%20Bocavirus%20isolates&amp;author=S.%20Zhao&amp;author=Q.%20Zhang&amp;author=X.%20Liu&amp;publication_year=2008&amp;journal=Biosystems&amp;volume=92&amp;pages=207-214" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biosystems.2008.01.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biosystems.2008.01.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biosystems.2008.01.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18378386" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benson</div> <div class="given-names">D. A.</div></div>,  <div class="name"><div class="surname">Karsch-Mizrachi</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div>,  <div class="name"><div class="surname">Ostell</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wheeler</div> <div class="given-names">D. L.</div></div>. </div><div class="article-title">GenBank</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">D25</div>-<div class="lpage">D30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GenBank&amp;author=D.%20A.%20Benson&amp;author=I.%20Karsch-Mizrachi&amp;author=D.%20J.%20Lipman&amp;author=J.%20Ostell&amp;author=D.%20L.%20Wheeler&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=D25-D30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm929" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm929" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm929"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18073190" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lafay</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lloyd</div> <div class="given-names">A. T.</div></div>,  <div class="name"><div class="surname">McLean</div> <div class="given-names">M. J.</div></div>,  <div class="name"><div class="surname">Devine</div> <div class="given-names">K. M.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>,  <div class="name"><div class="surname">Wolfe</div> <div class="given-names">K. H.</div></div>. </div><div class="article-title">Proteome composition and codon usage in spirochaetes: species-specific and DNA strand-specific mutational biases</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">1642</div>-<div class="lpage">1649</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Proteome%20composition%20and%20codon%20usage%20in%20spirochaetes%3A%20species-specific%20and%20DNA%20strand-specific%20mutational%20biases&amp;author=B.%20Lafay&amp;author=A.%20T.%20Lloyd&amp;author=M.%20J.%20McLean&amp;author=K.%20M.%20Devine&amp;author=P.%20M.%20Sharp&amp;author=K.%20H.%20Wolfe&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=1642-1649" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.7.1642" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.7.1642" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.7.1642"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10075995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McInerney</div> <div class="given-names">J. O.</div></div>. </div><div class="article-title">Replicational and transcriptional selection on codon usage in <em>Borrelia burgdorferi</em></div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1998</div>, vol. <div class="volume">95</div> (pg. <div class="fpage">10698</div>-<div class="lpage">106703</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Replicational%20and%20transcriptional%20selection%20on%20codon%20usage%20in%20Borrelia%20burgdorferi&amp;author=J.%20O.%20McInerney&amp;publication_year=1998&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=95&amp;pages=10698-106703" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.95.18.10698" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.95.18.10698" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.95.18.10698"> </span></div> </div></div></div></div></div><div content-id="DSN028C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Romero</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Zavala</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Musto</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Codon usage in <em>Chlamydia trachomatis</em> is the result of strand-specific mutational biases and a complex pattern of selective forces</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2000</div>, vol. <div class="volume">28</div> (pg. <div class="fpage">2084</div>-<div class="lpage">2090</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20Chlamydia%20trachomatis%20is%20the%20result%20of%20strand-specific%20mutational%20biases%20and%20a%20complex%20pattern%20of%20selective%20forces&amp;author=H.%20Romero&amp;author=A.%20Zavala&amp;author=H.%20Musto&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res.&amp;volume=28&amp;pages=2084-2090" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.10.2084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.10.2084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.10.2084"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10773076" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Musto</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Romero</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Zavala</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Translational selection is operative for synonymous codon usage in <em>Clostridium perfringens</em> and <em>Clostridium acetobutylicum</em></div>, <div class="source">Microbiology</div> , <div class="year">2003</div>, vol. <div class="volume">149</div> (pg. <div class="fpage">855</div>-<div class="lpage">863</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20selection%20is%20operative%20for%20synonymous%20codon%20usage%20in%20Clostridium%20perfringens%20and%20Clostridium%20acetobutylicum&amp;author=H.%20Musto&amp;author=H.%20Romero&amp;author=A.%20Zavala&amp;publication_year=2003&amp;journal=Microbiology&amp;volume=149&amp;pages=855-863" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.26063-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.26063-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.26063-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12686628" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Medigue</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Rouxel</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Vigier</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Henaut</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Evidence for horizontal gene transfer in <em>Escherichia coli</em> speciation</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1991</div>, vol. <div class="volume">222</div> (pg. <div class="fpage">851</div>-<div class="lpage">856</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20horizontal%20gene%20transfer%20in%20Escherichia%20coli%20speciation&amp;author=C.%20Medigue&amp;author=T.%20Rouxel&amp;author=P.%20Vigier&amp;author=A.%20Henaut&amp;author=A.%20Danchin&amp;publication_year=1991&amp;journal=J.%20Mol.%20Biol.&amp;volume=222&amp;pages=851-856" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(91)90575-Q" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(91)90575-Q" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(91)90575-Q"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1762151" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perriere</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Thioulouse</div> <div class="given-names">J.</div></div>. </div><div class="article-title">On-line tools for sequence retrieval and multivariate statistics in molecular biology</div>, <div class="source">Comput. Appl. Biosci.</div> , <div class="year">1996</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">63</div>-<div class="lpage">69</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On-line%20tools%20for%20sequence%20retrieval%20and%20multivariate%20statistics%20in%20molecular%20biology&amp;author=G.%20Perriere&amp;author=J.%20Thioulouse&amp;publication_year=1996&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=12&amp;pages=63-69" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8670621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lafay</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Atherton</div> <div class="given-names">J. C.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>. </div><div class="article-title">Absence of translationally selected synonymous codon usage bias in <em>Helicobacter pylori</em></div>, <div class="source">Microbiology</div> , <div class="year">2000</div>, vol. <div class="volume">146</div> <div class="issue">Pt 4</div>(pg. <div class="fpage">851</div>-<div class="lpage">860</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Absence%20of%20translationally%20selected%20synonymous%20codon%20usage%20bias%20in%20Helicobacter%20pylori&amp;author=B.%20Lafay&amp;author=J.%20C.%20Atherton&amp;author=P.%20M.%20Sharp&amp;publication_year=2000&amp;journal=Microbiology&amp;volume=146&amp;pages=851-860" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/00221287-146-4-851" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F00221287-146-4-851" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F00221287-146-4-851"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10784043" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">McInerney</div> <div class="given-names">J. O.</div></div>. </div><div class="article-title">Prokaryotic genome evolution as assessed by multivariate analysis of codon usage patterns</div>, <div class="source">Microbiol Comp. Genomics</div> , <div class="year">1997</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">89</div>-<div class="lpage">97</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prokaryotic%20genome%20evolution%20as%20assessed%20by%20multivariate%20analysis%20of%20codon%20usage%20patterns&amp;author=J.%20O.%20McInerney&amp;publication_year=1997&amp;journal=Microbiol%20Comp.%20Genomics&amp;volume=2&amp;pages=89-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/omi.1.1997.2.89" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fomi.1.1997.2.89" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fomi.1.1997.2.89"> </span></div> </div></div></div></div></div><div content-id="DSN028C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Andersson</div> <div class="given-names">S. G.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>. </div><div class="article-title">Codon usage and base composition in <em>Rickettsia prowazekii</em></div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1996</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">525</div>-<div class="lpage">536</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20base%20composition%20in%20Rickettsia%20prowazekii&amp;author=S.%20G.%20Andersson&amp;author=P.%20M.%20Sharp&amp;publication_year=1996&amp;journal=J.%20Mol.%20Evol.&amp;volume=42&amp;pages=525-536" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02352282" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02352282" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02352282"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8662004" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rispe</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Delmotte</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">van Ham</div> <div class="given-names">R. C.</div></div>,  <div class="name"><div class="surname">Moya</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Mutational and selective pressures on codon and amino acid usage in <em>Buchnera</em>, endosymbiotic bacteria of aphids</div>, <div class="source">Genome Res.</div> , <div class="year">2004</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">44</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mutational%20and%20selective%20pressures%20on%20codon%20and%20amino%20acid%20usage%20in%20Buchnera%2C%20endosymbiotic%20bacteria%20of%20aphids&amp;author=C.%20Rispe&amp;author=F.%20Delmotte&amp;author=R.%20C.%20van%20Ham&amp;author=A.%20Moya&amp;publication_year=2004&amp;journal=Genome%20Res.&amp;volume=14&amp;pages=44-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1358104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1358104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1358104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14672975" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>,  <div class="name"><div class="surname">Tuohy</div> <div class="given-names">T. M.</div></div>,  <div class="name"><div class="surname">Mosurski</div> <div class="given-names">K. R.</div></div>. </div><div class="article-title">Codon usage in yeast: cluster analysis clearly differentiates highly and lowly expressed genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1986</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">5125</div>-<div class="lpage">5143</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20in%20yeast%3A%20cluster%20analysis%20clearly%20differentiates%20highly%20and%20lowly%20expressed%20genes&amp;author=P.%20M.%20Sharp&amp;author=T.%20M.%20Tuohy&amp;author=K.%20R.%20Mosurski&amp;publication_year=1986&amp;journal=Nucleic%20Acids%20Res.&amp;volume=14&amp;pages=5125-5143" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/14.13.5125" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F14.13.5125" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F14.13.5125"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3526280" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thioulouse</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Chessel</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Dolédec</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Olivier</div> <div class="given-names">J. M.</div></div>. </div><div class="article-title">ADE-4: a multivariate analysis and graphical display software</div>, <div class="source">Stat. Comput.</div> , <div class="year">1997</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">75</div>-<div class="lpage">83</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ADE-4%3A%20a%20multivariate%20analysis%20and%20graphical%20display%20software&amp;author=J.%20Thioulouse&amp;author=D.%20Chessel&amp;author=S.%20Dol%C3%A9dec&amp;author=J.%20M.%20Olivier&amp;publication_year=1997&amp;journal=Stat.%20Comput.&amp;volume=7&amp;pages=75-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1018513530268" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1018513530268" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1018513530268"> </span></div> </div></div></div></div></div><div content-id="DSN028C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="collab">R Development Core Team.</div><div class="article-title">R: a language and environment for statistical computing, R Foundation for Statistical Computing</div>, <div class="year">2007</div><div class="publisher-loc">Vienna, Austria</div></div></div></div></div><div content-id="DSN028C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grocock</div> <div class="given-names">R. J.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>. </div><div class="article-title">Synonymous codon usage in <em>Pseudomonas aeruginosa</em> PA01</div>, <div class="source">Gene</div> , <div class="year">2002</div>, vol. <div class="volume">289</div> (pg. <div class="fpage">131</div>-<div class="lpage">139</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20usage%20in%20Pseudomonas%20aeruginosa%20PA01&amp;author=R.%20J.%20Grocock&amp;author=P.%20M.%20Sharp&amp;publication_year=2002&amp;journal=Gene&amp;volume=289&amp;pages=131-139" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(02)00503-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(02)00503-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(02)00503-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12036591" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kyte</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Doolittle</div> <div class="given-names">R. F.</div></div>. </div><div class="article-title">A simple method for displaying the hydropathic character of a protein</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1982</div>, vol. <div class="volume">157</div> (pg. <div class="fpage">105</div>-<div class="lpage">132</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20method%20for%20displaying%20the%20hydropathic%20character%20of%20a%20protein&amp;author=J.%20Kyte&amp;author=R.%20F.%20Doolittle&amp;publication_year=1982&amp;journal=J.%20Mol.%20Biol.&amp;volume=157&amp;pages=105-132" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(82)90515-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(82)90515-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(82)90515-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7108955" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Suzuki</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Saito</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Tomita</div> <div class="given-names">M.</div></div>. </div><div class="article-title">A problem in multivariate analysis of codon usage data and a possible solution</div>, <div class="source">FEBS Lett</div> , <div class="year">2005</div>, vol. <div class="volume">579</div> (pg. <div class="fpage">6499</div>-<div class="lpage">504</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20problem%20in%20multivariate%20analysis%20of%20codon%20usage%20data%20and%20a%20possible%20solution&amp;author=H.%20Suzuki&amp;author=R.%20Saito&amp;author=M.%20Tomita&amp;publication_year=2005&amp;journal=FEBS%20Lett&amp;volume=579&amp;pages=6499-504" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2005.10.032" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2005.10.032" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2005.10.032"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16289058" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lobry</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Gautier</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Hydrophobicity, expressivity and aromaticity are the major trends of amino-acid usage in 999 <em>Escherichia coli</em> chromosome-encoded genes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1994</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">3174</div>-<div class="lpage">3180</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Hydrophobicity%2C%20expressivity%20and%20aromaticity%20are%20the%20major%20trends%20of%20amino-acid%20usage%20in%20999%20Escherichia%20coli%20chromosome-encoded%20genes&amp;author=J.%20R.%20Lobry&amp;author=C.%20Gautier&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=22&amp;pages=3174-3180" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.15.3174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.15.3174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.15.3174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8065933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ghaemmaghami</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Huh</div> <div class="given-names">W. K.</div></div>,  <div class="name"><div class="surname">Bower</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Global analysis of protein expression in yeast</div>, <div class="source">Nature</div> , <div class="year">2003</div>, vol. <div class="volume">425</div> (pg. <div class="fpage">737</div>-<div class="lpage">741</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20analysis%20of%20protein%20expression%20in%20yeast&amp;author=S.%20Ghaemmaghami&amp;author=W.%20K.%20Huh&amp;author=K.%20Bower&amp;publication_year=2003&amp;journal=Nature&amp;volume=425&amp;pages=737-741" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02046" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02046" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02046"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14562106" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C57" class="ref-content "><div class="label -label">57</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Carlini</div> <div class="given-names">D. B.</div></div>. </div><div class="article-title">Context-dependent codon bias and messenger RNA longevity in the yeast transcriptome</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">2005</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1403</div>-<div class="lpage">1411</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Context-dependent%20codon%20bias%20and%20messenger%20RNA%20longevity%20in%20the%20yeast%20transcriptome&amp;author=D.%20B.%20Carlini&amp;publication_year=2005&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=22&amp;pages=1403-1411" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msi135" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsi135" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsi135"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15772378" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C58" class="ref-content "><div class="label -label">58</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kerr</div> <div class="given-names">A. R.</div></div>,  <div class="name"><div class="surname">Peden</div> <div class="given-names">J. F.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P. M.</div></div>. </div><div class="article-title">Systematic base composition variation around the genome of <em>Mycoplasma genitalium</em>, but not <em>Mycoplasma pneumoniae</em></div>, <div class="source">Mol. Microbiol.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1177</div>-<div class="lpage">1179</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20base%20composition%20variation%20around%20the%20genome%20of%20Mycoplasma%20genitalium%2C%20but%20not%20Mycoplasma%20pneumoniae&amp;author=A.%20R.%20Kerr&amp;author=J.%20F.%20Peden&amp;author=P.%20M.%20Sharp&amp;publication_year=1997&amp;journal=Mol.%20Microbiol.&amp;volume=25&amp;pages=1177-1179" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.1997.5461902.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.1997.5461902.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.1997.5461902.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9350873" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C59" class="ref-content "><div class="label -label">59</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arakawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Mori</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Ikeda</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Matsuzaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tomita</div> <div class="given-names">M.</div></div>. </div><div class="article-title">G-language genome analysis environment: a workbench for nucleotide sequence data mining</div>, <div class="source">Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">305</div>-<div class="lpage">306</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=G-language%20genome%20analysis%20environment%3A%20a%20workbench%20for%20nucleotide%20sequence%20data%20mining&amp;author=K.%20Arakawa&amp;author=K.%20Mori&amp;author=K.%20Ikeda&amp;author=T.%20Matsuzaki&amp;author=Y.%20Kobayashi&amp;author=M.%20Tomita&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=305-306" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/19.2.305" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F19.2.305" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F19.2.305"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12538262" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN028C60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN028C60" class="ref-content "><div class="label -label">60</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Arakawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tomita</div> <div class="given-names">M.</div></div>. </div><div class="article-title">G-language system as a platform for large-scale analysis of high-throughput omics data</div>, <div class="source">J. Pesticide Sci.</div> , <div class="year">2006</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">282</div>-<div class="lpage">288</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=G-language%20system%20as%20a%20platform%20for%20large-scale%20analysis%20of%20high-throughput%20omics%20data&amp;author=K.%20Arakawa&amp;author=M.%20Tomita&amp;publication_year=2006&amp;journal=J.%20Pesticide%20Sci.&amp;volume=31&amp;pages=282-288" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1584/jpestics.31.282" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1584%2Fjpestics.31.282" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1584%2Fjpestics.31.282"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=513030&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/dsn028-dsn028supp_Table2.xls?Expires=2147483647&amp;Signature=3GcVkDTn~r3jgz5YxH-bGj9hrvSI5MA50QSCRpGSDbTB5tC~nyjn5VMoAlk9UrrIY-C~DZ64xevesmxfM6WgSN-i36TEinovSXoyL9tyGnMaqoN6xUMm6tG7ha3aFyeYBYfn6dgIcKhmQxG0C~Gx0uk~FAUurR8qCGc2NzlcMGfSFJc4T2kx1BaTgIj8KMmg4QF7XFU96McffPO5RR1JNw4J97TpOn1a6K5hmoOkoLvGjzmHJYyqhQDZJ35moaD14y~m8HgKMmhe5XpQhTaKwhq~BRlreyUq6I4Mh0kljSz3J3N0j6HFGZWeq41NjrQB1Irg4XrH8EyuSa17A6ZJlQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/6/10.1093/dnares/dsn028/2/dsn028-dsn028supp_Table1.xls?Expires=2147483647&amp;Signature=jzM6Fh442ZzxP4ekxfNQ8AkXVRHrIPLMbrd7YSl22DduSfStR9lVWmfEWTZkSPGeur-rOMZLjnN-PNx~Sx-alUP-CazBOteBciPj7wTOSCHHBIGQh5EDIN9kk7B3~Q0yTTSJVCbBZsQ4cGuzKORANevoyommyxx07-4DU5sSXHRbwokqQM9GsjNXUYclDFZsWONlrX8xV7rB6zxwtL--Hz~7gRcBbyoNvS2oXaN6jawFAabTenxqTStRIWjxIoqc1v~WeuPo3alJTcgHowLPniRuhpl-9mH8naN4VowAwyiljHlr-c33qE44VAM7ANGU3nVj66AtjsL1bcCQY8Sb8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">354</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000262588400003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">42</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsn028"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="513030"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000262588400003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f6%2f357%2f513030">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000262588400003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (42)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F6%2F357%2F513030" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/513030'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Comparison of Correspondence Analysis Methods for Synonymous Codon Usage in Bacteria" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsn028");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f6%2f357%2f513030">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

