

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/5/499/5045548","@type":"ScholarlyArticle","name":"The genome of tapeworm  Taenia multiceps  sheds light on understanding parasitic mechanism and control of coenurosis disease","datePublished":"2018-06-25","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2018-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy020","keywords":["<italic>Taenia multiceps</italic>","coenurosis","genome","parasitism"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/5/499/5045548","author":[{"name":"Li, Wenhui","affiliation":"State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China","@type":"Person"},{"name":"Liu, Bo","affiliation":"Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China","@type":"Person"},{"name":"Yang, Yang","affiliation":"State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China","@type":"Person"},{"name":"Ren, Yuwei","affiliation":"Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China","@type":"Person"},{"name":"Wang, Shuai","affiliation":"State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China","@type":"Person"},{"name":"Liu, Conghui","affiliation":"Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China","@type":"Person"},{"name":"Zhang, Nianzhang","affiliation":"State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China","@type":"Person"},{"name":"Qu, Zigang","affiliation":"State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China","@type":"Person"},{"name":"Yang, Wanxu","affiliation":"Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China","@type":"Person"},{"name":"Zhang, Yan","affiliation":"Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China","@type":"Person"},{"name":"Yan, Hongbing","affiliation":"State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China","@type":"Person"},{"name":"Jiang, Fan","affiliation":"Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China","@type":"Person"},{"name":"Li, Li","affiliation":"State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China","@type":"Person"},{"name":"Li, Shuqu","affiliation":"Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China","@type":"Person"},{"name":"Jia, Wanzhong","affiliation":"State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China  Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Disease, Yangzhou, Jiangsu, China","@type":"Person"},{"name":"Yin, Hong","affiliation":"State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China  Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Disease, Yangzhou, Jiangsu, China","@type":"Person"},{"name":"Cai, Xuepeng","affiliation":"State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China","@type":"Person"},{"name":"Liu, Tao","affiliation":"Annoroad Gene Tech. (Beijing) Co Ltd, Beijing, China","@type":"Person"},{"name":"McManus, Donald P","affiliation":"Molecular Parasitology Laboratory, QIMR Berghofer Medical Research Institute, Brisbane, Australia","@type":"Person"},{"name":"Fan, Wei","affiliation":"Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China","@type":"Person"},{"name":"Fu, Baoquan","affiliation":"State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China  Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Disease, Yangzhou, Jiangsu, China","@type":"Person"}],"description":"Abstract.  Coenurosis, caused by the larval coenurus of the tapeworm Taenia multiceps, is a fatal central nervous system disease in both sheep and humans. Thoug","pageStart":"499","pageEnd":"510","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"The genome of tapeworm  Taenia multiceps  sheds light on understanding parasitic mechanism and control of coenurosis disease","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="genome of tapeworm  Taenia multiceps  sheds light on understanding parasitic mechanism and control of coenurosis disease" />
    <meta property="og:description" content="Abstract.  Coenurosis, caused by the larval coenurus of the tapeworm Taenia multiceps, is a fatal central nervous system disease in both sheep and humans. Thoug" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/5/499/5045548" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-06-25" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Li, Wenhui" /><meta name="citation_author_institution" content="State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China" /><meta name="citation_author" content="Liu, Bo" /><meta name="citation_author_institution" content="Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China" /><meta name="citation_author" content="Yang, Yang" /><meta name="citation_author_institution" content="State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China" /><meta name="citation_author" content="Ren, Yuwei" /><meta name="citation_author_institution" content="Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China" /><meta name="citation_author" content="Wang, Shuai" /><meta name="citation_author_institution" content="State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China" /><meta name="citation_author" content="Liu, Conghui" /><meta name="citation_author_institution" content="Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China" /><meta name="citation_author" content="Zhang, Nianzhang" /><meta name="citation_author_institution" content="State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China" /><meta name="citation_author" content="Qu, Zigang" /><meta name="citation_author_institution" content="State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China" /><meta name="citation_author" content="Yang, Wanxu" /><meta name="citation_author_institution" content="Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China" /><meta name="citation_author" content="Zhang, Yan" /><meta name="citation_author_institution" content="Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China" /><meta name="citation_author" content="Yan, Hongbing" /><meta name="citation_author_institution" content="State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China" /><meta name="citation_author" content="Jiang, Fan" /><meta name="citation_author_institution" content="Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China" /><meta name="citation_author" content="Li, Li" /><meta name="citation_author_institution" content="State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China" /><meta name="citation_author" content="Li, Shuqu" /><meta name="citation_author_institution" content="Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China" /><meta name="citation_author" content="Jia, Wanzhong" /><meta name="citation_author_institution" content="State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China" /><meta name="citation_author_institution" content="Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Disease, Yangzhou, Jiangsu, China" /><meta name="citation_author" content="Yin, Hong" /><meta name="citation_author_institution" content="State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China" /><meta name="citation_author_institution" content="Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Disease, Yangzhou, Jiangsu, China" /><meta name="citation_author" content="Cai, Xuepeng" /><meta name="citation_author_institution" content="State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China" /><meta name="citation_author" content="Liu, Tao" /><meta name="citation_author_institution" content="Annoroad Gene Tech. (Beijing) Co Ltd, Beijing, China" /><meta name="citation_author" content="McManus, Donald P" /><meta name="citation_author_institution" content="Molecular Parasitology Laboratory, QIMR Berghofer Medical Research Institute, Brisbane, Australia" /><meta name="citation_author" content="Fan, Wei" /><meta name="citation_author_institution" content="Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China" /><meta name="citation_author" content="Fu, Baoquan" /><meta name="citation_author_institution" content="State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China" /><meta name="citation_author_institution" content="Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Disease, Yangzhou, Jiangsu, China" /><meta name="citation_title" content="The genome of tapeworm Taenia multiceps sheds light on understanding parasitic mechanism and control of coenurosis disease" /><meta name="citation_firstpage" content="499" /><meta name="citation_lastpage" content="510" /><meta name="citation_doi" content="10.1093/dnares/dsy020" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2018/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Cerebral and non-cerebral coenurosis in small ruminants;  citation_journal_title=Trop. Biomed;  citation_year=2014;  citation_volume=31;  citation_pages=1-16; " /><meta name="citation_reference" content="citation_title=Detailed transcriptome description of the neglected cestode Taenia multiceps;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e45830" /><meta name="citation_reference" content="citation_title=Updates on morphobiology, epidemiology and molecular characterization of coenurosis in sheep;  citation_journal_title=Parassitologia;  citation_year=2006;  citation_volume=48;  citation_pages=61-3; " /><meta name="citation_reference" content="citation_title=Posterior paralysis in a lamb caused by a coeneurus cerebralis cyst in the lumbar spinal cord;  citation_journal_title=Vet. Rec;  citation_year=1997;  citation_volume=140;  citation_pages=560" /><meta name="citation_reference" content="citation_title=Coenurus cerebralis infection in Ethiopian highland sheep: incidence and observations on pathogenesis and clinical signs;  citation_journal_title=Trop. Anim. Health Prod;  citation_year=1999;  citation_volume=31;  citation_pages=15-24; " /><meta name="citation_reference" content="citation_title=Genetic characterisation of Taenia multiceps cysts from ruminants in Greece;  citation_journal_title=Infect. Genet. Evol;  citation_year=2016;  citation_volume=38;  citation_pages=110-6; " /><meta name="citation_reference" content="citation_title=Coenurosis status in Afro-Asian region: a review;  citation_journal_title=Small Ruminant Res;  citation_year=2006;  citation_volume=64;  citation_pages=197-202; " /><meta name="citation_reference" content="citation_title=Taenia multiceps: a rare human cestode infection in Israel;  citation_journal_title=Vet. Ital;  citation_year=2008;  citation_volume=44;  citation_pages=621-31; " /><meta name="citation_reference" content="citation_title=The organization, ontogeny, and orientation of the Cestoda;  citation_journal_title=Quart. Rev. Biol;  citation_year=1962;  citation_volume=37;  citation_pages=23-34; " /><meta name="citation_reference" content="Roberts L. S. S., Janovy G. D.  2009 , Gerald D. Schmidt &amp; Larry S. Roberts’ Foundations of parasitology. " /><meta name="citation_reference" content="citation_title=Long-term follow-up of the surgical treatment of intracranial coenurosis;  citation_journal_title=Br. J. Neurosurg;  citation_year=1990;  citation_volume=4;  citation_pages=39-43; " /><meta name="citation_reference" content="citation_title=Computed tomography and magnetic resonance imaging of cerebral coenurosis;  citation_journal_title=Surg. Neurol;  citation_year=1987;  citation_volume=27;  citation_pages=548-52; " /><meta name="citation_reference" content="citation_title=Vaccination with recombinant oncosphere antigens reduces the susceptibility of sheep to infection with Taenia multiceps;  citation_journal_title=Int. J. Parasitol;  citation_year=2008;  citation_volume=38;  citation_pages=1041-50; " /><meta name="citation_reference" content="citation_title=Preliminary field trial of a vaccine against coenurosis caused by Taenia multiceps;  citation_journal_title=Vet. Parasitol;  citation_year=2009;  citation_volume=162;  citation_pages=285-9; " /><meta name="citation_reference" content="citation_title=Cestode vaccines: origins, current status and future prospects;  citation_journal_title=Parasitology;  citation_year=2006;  citation_volume=133;  citation_pages=S27-42; " /><meta name="citation_reference" content="citation_title=The genomes of four tapeworm species reveal adaptations to parasitism;  citation_journal_title=Nature;  citation_year=2013;  citation_volume=496;  citation_pages=57-63; " /><meta name="citation_reference" content="citation_title=Comparative genomics reveals adaptive evolution of Asian tapeworm in switching to a new intermediate host;  citation_journal_title=Nat. Commun;  citation_year=2016;  citation_volume=7;  citation_pages=12845" /><meta name="citation_reference" content="citation_title=The genome of the hydatid tapeworm Echinococcus granulosus;  citation_journal_title=Nat. Genet;  citation_year=2013;  citation_volume=45;  citation_pages=1168-75; " /><meta name="citation_reference" content="citation_title=Identification of neglected cestode Taenia multiceps microRNAs by illumina sequencing and bioinformatic analysis;  citation_journal_title=BMC Vet. Res;  citation_year=2013;  citation_volume=9;  citation_pages=162" /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement;  citation_journal_title=PloS One;  citation_year=2014;  citation_volume=9;  citation_pages=e112963" /><meta name="citation_reference" content="citation_title=Chromosome-scale scaffolding of de novo genome assemblies based on chromatin interactions;  citation_journal_title=Nat. Biotechnol;  citation_year=2013;  citation_volume=31;  citation_pages=1119-25; " /><meta name="citation_reference" content="citation_title=Automated eukaryotic gene structure annotation using EVidenceModeler and the program to assemble spliced alignments;  citation_journal_title=Genome Biol;  citation_year=2008;  citation_volume=9;  citation_pages=R7" /><meta name="citation_reference" content="citation_title=The Universal Protein Resource (UniProt): an expanding universe of protein information;  citation_journal_title=Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=D187-91; " /><meta name="citation_reference" content="citation_title=InterProScan: protein domains identifier;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=W116-20; " /><meta name="citation_reference" content="citation_title=The KEGG resource for deciphering the genome;  citation_journal_title=Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=D277-80; " /><meta name="citation_reference" content="citation_title=Tandem repeats finder: a program to analyze DNA sequences;  citation_journal_title=Nucleic Acids Res;  citation_year=1999;  citation_volume=27;  citation_pages=573-80; " /><meta name="citation_reference" content="citation_title=Perspective - synteny and collinearity in plant genomes;  citation_journal_title=Science;  citation_year=2008;  citation_volume=320;  citation_pages=486-8; " /><meta name="citation_reference" content="citation_title=OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=14" /><meta name="citation_reference" content="citation_title=MUSCLE: multiple sequence alignment with high accuracy and high throughput;  citation_journal_title=Nucleic Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=1792-7; " /><meta name="citation_reference" content="citation_title=New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0;  citation_journal_title=Syst. Biol;  citation_year=2010;  citation_volume=59;  citation_pages=307-21; " /><meta name="citation_reference" content="citation_title=ProtTest 3: fast selection of best-fit models of protein evolution;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=1164-5; " /><meta name="citation_reference" content="citation_title=PAML 4: phylogenetic analysis by maximum likelihood;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=1586-91; " /><meta name="citation_reference" content="citation_title=Phylogenetic relationships within Echinococcus and Taenia tapeworms (Cestoda: taeniidae): An inference from nuclear protein-coding genes;  citation_journal_title=Mol. Phylogenet. Evol;  citation_year=2011;  citation_volume=61;  citation_pages=628-38; " /><meta name="citation_reference" content="citation_title=KaKs_Calculator 2.0: a toolkit incorporating gamma-series methods and sliding window strategies;  citation_journal_title=Genomics Proteomics Bioinformatics;  citation_year=2010;  citation_volume=8;  citation_pages=77-80; " /><meta name="citation_reference" content="citation_title=The HMMTOP transmembrane topology prediction server;  citation_journal_title=Bioinformatics;  citation_year=2001;  citation_volume=17;  citation_pages=849-50; " /><meta name="citation_reference" content="citation_title=Assembling large genomes with single-molecule sequencing and locality-sensitive hashing;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=623-30; " /><meta name="citation_reference" content="citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=3210-2; " /><meta name="citation_reference" content="citation_title=Plant transposable elements: where genetics meets genomics;  citation_journal_title=Nat. Rev. Genet;  citation_year=2002;  citation_volume=3;  citation_pages=329-41; " /><meta name="citation_reference" content="citation_title=A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood;  citation_journal_title=Syst. Biol;  citation_year=2003;  citation_volume=52;  citation_pages=696-704; " /><meta name="citation_reference" content="citation_title=Approximate likelihood calculation on a phylogeny for bayesian estimation of divergence times;  citation_journal_title=Mol. Biol. Evol;  citation_year=2011;  citation_volume=28;  citation_pages=2161-72; " /><meta name="citation_reference" content="Marr M.  2011 , Developmental Regulatory Genes in Parasitic Flatworms. Doctoral dissertation, Department of Life Sciences and the Natural History Museum, Imperial College London. " /><meta name="citation_reference" content="citation_title=The T-box gene family: emerging roles in development, stem cells and cancer;  citation_journal_title=Development;  citation_year=2014;  citation_volume=141;  citation_pages=3819-33; " /><meta name="citation_reference" content="Kanamori K.  2017 , In vivo N-15 MRS study of glutamate metabolism in the rat brain. Anal. Biochem., 529 , 179 – 192. " /><meta name="citation_reference" content="citation_title=Clinical aspects of urea cycle dysfunction and altered brain energy metabolism on modulation of glutamate receptors and transporters in acute and chronic hyperammonemia;  citation_journal_title=Biomed. Pharmacother;  citation_year=2016;  citation_volume=81;  citation_pages=192-202; " /><meta name="citation_reference" content="citation_title=Mechanisms of glutamate toxicity in multiple sclerosis: biomarker and therapeutic opportunities;  citation_journal_title=LancetNeurol;  citation_year=2016;  citation_volume=15;  citation_pages=1089-102; " /><meta name="citation_reference" content="citation_title=Global extent of horizontal gene transfer;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2007;  citation_volume=104;  citation_pages=4489-94; " /><meta name="citation_reference" content="citation_title=A role for host-parasite interactions in the horizontal transfer of transposons across phyla;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=464;  citation_pages=1347" /><meta name="citation_reference" content="citation_title=Functional and ecological impacts of horizontal gene transfer in eukaryotes;  citation_journal_title=Curr. Opin. Genet. Dev;  citation_year=2009;  citation_volume=19;  citation_pages=613-9; " /><meta name="citation_reference" content="citation_title=The concise guide to pharmacology 2013/14: enzymes;  citation_journal_title=Br. J. Pharmacol;  citation_year=2013;  citation_volume=170;  citation_pages=1797-867; " /><meta name="citation_reference" content="citation_title=Parasite neuropeptide biology: seeding rational drug target selection?;  citation_journal_title=Int. J. Parasitol. Drug;  citation_year=2012;  citation_volume=2;  citation_pages=76-91; " /><meta name="citation_reference" content="citation_title=PacBio sequencing and its applications;  citation_journal_title=Genomics. Proteomics Bioinformatics;  citation_year=2015;  citation_volume=13;  citation_pages=278-89; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/5/499/26076973/dsy020.pdf" /><meta name="description" content="Abstract.  Coenurosis, caused by the larval coenurus of the tapeworm Taenia multiceps, is a fatal central nervous system disease in both sheep and humans. Thoug" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/5/499/5045548" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/5/499/5045548" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/5/499/5045548");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy020';
    </script>
<title>
	genome of tapeworm Taenia multiceps sheds light on understanding parasitic mechanism and control of coenurosis disease | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./5045548" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZuaATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGksIFdlbmh1aSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIFZldGVyaW5hcnkgRXRpb2xvZ2ljYWwgQmlvbG9neSwgS2V5IExhYm9yYXRvcnkgb2YgVmV0ZXJpbmFyeSBQYXJhc2l0b2xvZ3kgb2YgR2Fuc3UgUHJvdmluY2UsIExhbnpob3UgVmV0ZXJpbmFyeSBSZXNlYXJjaCBJbnN0aXR1dGUsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIExhbnpob3UsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGl1LCBCbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkFncmljdWx0dXJhbCBHZW5vbWljIEluc3RpdHV0ZSwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgU2hlbnpoZW4sIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFuZywgWWFuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIFZldGVyaW5hcnkgRXRpb2xvZ2ljYWwgQmlvbG9neSwgS2V5IExhYm9yYXRvcnkgb2YgVmV0ZXJpbmFyeSBQYXJhc2l0b2xvZ3kgb2YgR2Fuc3UgUHJvdmluY2UsIExhbnpob3UgVmV0ZXJpbmFyeSBSZXNlYXJjaCBJbnN0aXR1dGUsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIExhbnpob3UsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUmVuLCBZdXdlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkFncmljdWx0dXJhbCBHZW5vbWljIEluc3RpdHV0ZSwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgU2hlbnpoZW4sIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iV2FuZywgU2h1YWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBWZXRlcmluYXJ5IEV0aW9sb2dpY2FsIEJpb2xvZ3ksIEtleSBMYWJvcmF0b3J5IG9mIFZldGVyaW5hcnkgUGFyYXNpdG9sb2d5IG9mIEdhbnN1IFByb3ZpbmNlLCBMYW56aG91IFZldGVyaW5hcnkgUmVzZWFyY2ggSW5zdGl0dXRlLCBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBMYW56aG91LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpdSwgQ29uZ2h1aSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkFncmljdWx0dXJhbCBHZW5vbWljIEluc3RpdHV0ZSwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgU2hlbnpoZW4sIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhhbmcsIE5pYW56aGFuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIFZldGVyaW5hcnkgRXRpb2xvZ2ljYWwgQmlvbG9neSwgS2V5IExhYm9yYXRvcnkgb2YgVmV0ZXJpbmFyeSBQYXJhc2l0b2xvZ3kgb2YgR2Fuc3UgUHJvdmluY2UsIExhbnpob3UgVmV0ZXJpbmFyeSBSZXNlYXJjaCBJbnN0aXR1dGUsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIExhbnpob3UsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUXUsIFppZ2FuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIFZldGVyaW5hcnkgRXRpb2xvZ2ljYWwgQmlvbG9neSwgS2V5IExhYm9yYXRvcnkgb2YgVmV0ZXJpbmFyeSBQYXJhc2l0b2xvZ3kgb2YgR2Fuc3UgUHJvdmluY2UsIExhbnpob3UgVmV0ZXJpbmFyeSBSZXNlYXJjaCBJbnN0aXR1dGUsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIExhbnpob3UsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFuZywgV2FueHUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBZ3JpY3VsdHVyYWwgR2Vub21pYyBJbnN0aXR1dGUsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIFNoZW56aGVuLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpoYW5nLCBZYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBZ3JpY3VsdHVyYWwgR2Vub21pYyBJbnN0aXR1dGUsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIFNoZW56aGVuLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbiwgSG9uZ2JpbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBWZXRlcmluYXJ5IEV0aW9sb2dpY2FsIEJpb2xvZ3ksIEtleSBMYWJvcmF0b3J5IG9mIFZldGVyaW5hcnkgUGFyYXNpdG9sb2d5IG9mIEdhbnN1IFByb3ZpbmNlLCBMYW56aG91IFZldGVyaW5hcnkgUmVzZWFyY2ggSW5zdGl0dXRlLCBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBMYW56aG91LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkppYW5nLCBGYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBZ3JpY3VsdHVyYWwgR2Vub21pYyBJbnN0aXR1dGUsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIFNoZW56aGVuLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxpLCBMaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIFZldGVyaW5hcnkgRXRpb2xvZ2ljYWwgQmlvbG9neSwgS2V5IExhYm9yYXRvcnkgb2YgVmV0ZXJpbmFyeSBQYXJhc2l0b2xvZ3kgb2YgR2Fuc3UgUHJvdmluY2UsIExhbnpob3UgVmV0ZXJpbmFyeSBSZXNlYXJjaCBJbnN0aXR1dGUsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIExhbnpob3UsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGksIFNodXF1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQWdyaWN1bHR1cmFsIEdlbm9taWMgSW5zdGl0dXRlLCBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBTaGVuemhlbiwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKaWEsIFdhbnpob25nIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3RhdGUgS2V5IExhYm9yYXRvcnkgb2YgVmV0ZXJpbmFyeSBFdGlvbG9naWNhbCBCaW9sb2d5LCBLZXkgTGFib3JhdG9yeSBvZiBWZXRlcmluYXJ5IFBhcmFzaXRvbG9neSBvZiBHYW5zdSBQcm92aW5jZSwgTGFuemhvdSBWZXRlcmluYXJ5IFJlc2VhcmNoIEluc3RpdHV0ZSwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgTGFuemhvdSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJKaWFuZ3N1IENvLWlubm92YXRpb24gQ2VudGVyIGZvciBQcmV2ZW50aW9uIGFuZCBDb250cm9sIG9mIEltcG9ydGFudCBBbmltYWwgSW5mZWN0aW91cyBEaXNlYXNlLCBZYW5nemhvdSwgSmlhbmdzdSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZaW4sIEhvbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdGF0ZSBLZXkgTGFib3JhdG9yeSBvZiBWZXRlcmluYXJ5IEV0aW9sb2dpY2FsIEJpb2xvZ3ksIEtleSBMYWJvcmF0b3J5IG9mIFZldGVyaW5hcnkgUGFyYXNpdG9sb2d5IG9mIEdhbnN1IFByb3ZpbmNlLCBMYW56aG91IFZldGVyaW5hcnkgUmVzZWFyY2ggSW5zdGl0dXRlLCBDaGluZXNlIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBMYW56aG91LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkppYW5nc3UgQ28taW5ub3ZhdGlvbiBDZW50ZXIgZm9yIFByZXZlbnRpb24gYW5kIENvbnRyb2wgb2YgSW1wb3J0YW50IEFuaW1hbCBJbmZlY3Rpb3VzIERpc2Vhc2UsIFlhbmd6aG91LCBKaWFuZ3N1LCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkNhaSwgWHVlcGVuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN0YXRlIEtleSBMYWJvcmF0b3J5IG9mIFZldGVyaW5hcnkgRXRpb2xvZ2ljYWwgQmlvbG9neSwgS2V5IExhYm9yYXRvcnkgb2YgVmV0ZXJpbmFyeSBQYXJhc2l0b2xvZ3kgb2YgR2Fuc3UgUHJvdmluY2UsIExhbnpob3UgVmV0ZXJpbmFyeSBSZXNlYXJjaCBJbnN0aXR1dGUsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIExhbnpob3UsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGl1LCBUYW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBbm5vcm9hZCBHZW5lIFRlY2guIChCZWlqaW5nKSBDbyBMdGQsIEJlaWppbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWNNYW51cywgRG9uYWxkIFAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNb2xlY3VsYXIgUGFyYXNpdG9sb2d5IExhYm9yYXRvcnksIFFJTVIgQmVyZ2hvZmVyIE1lZGljYWwgUmVzZWFyY2ggSW5zdGl0dXRlLCBCcmlzYmFuZSwgQXVzdHJhbGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRmFuLCBXZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJBZ3JpY3VsdHVyYWwgR2Vub21pYyBJbnN0aXR1dGUsIENoaW5lc2UgQWNhZGVteSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIFNoZW56aGVuLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkZ1LCBCYW9xdWFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3RhdGUgS2V5IExhYm9yYXRvcnkgb2YgVmV0ZXJpbmFyeSBFdGlvbG9naWNhbCBCaW9sb2d5LCBLZXkgTGFib3JhdG9yeSBvZiBWZXRlcmluYXJ5IFBhcmFzaXRvbG9neSBvZiBHYW5zdSBQcm92aW5jZSwgTGFuemhvdSBWZXRlcmluYXJ5IFJlc2VhcmNoIEluc3RpdHV0ZSwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgTGFuemhvdSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJKaWFuZ3N1IENvLWlubm92YXRpb24gQ2VudGVyIGZvciBQcmV2ZW50aW9uIGFuZCBDb250cm9sIG9mIEltcG9ydGFudCBBbmltYWwgSW5mZWN0aW91cyBEaXNlYXNlLCBZYW5nemhvdSwgSmlhbmdzdSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlRoZSBnZW5vbWUgb2YgdGFwZXdvcm0gVGFlbmlhIG11bHRpY2VwcyBzaGVkcyBsaWdodCBvbiB1bmRlcnN0YW5kaW5nIHBhcmFzaXRpYyBtZWNoYW5pc20gYW5kIGNvbnRyb2wgb2YgY29lbnVyb3NpcyBkaXNlYXNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNDk5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI1MTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3kwMjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxOC8xMC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNlcmVicmFsIGFuZCBub24tY2VyZWJyYWwgY29lbnVyb3NpcyBpbiBzbWFsbCBydW1pbmFudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyb3AuIEJpb21lZDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTEtMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGFpbGVkIHRyYW5zY3JpcHRvbWUgZGVzY3JpcHRpb24gb2YgdGhlIG5lZ2xlY3RlZCBjZXN0b2RlIFRhZW5pYSBtdWx0aWNlcHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lNDU4MzAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VcGRhdGVzIG9uIG1vcnBob2Jpb2xvZ3ksIGVwaWRlbWlvbG9neSBhbmQgbW9sZWN1bGFyIGNoYXJhY3Rlcml6YXRpb24gb2YgY29lbnVyb3NpcyBpbiBzaGVlcDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGFyYXNzaXRvbG9naWE7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz02MS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3N0ZXJpb3IgcGFyYWx5c2lzIGluIGEgbGFtYiBjYXVzZWQgYnkgYSBjb2VuZXVydXMgY2VyZWJyYWxpcyBjeXN0IGluIHRoZSBsdW1iYXIgc3BpbmFsIGNvcmQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVZldC4gUmVjOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTE0MDsgIGNpdGF0aW9uX3BhZ2VzPTU2MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZW51cnVzIGNlcmVicmFsaXMgaW5mZWN0aW9uIGluIEV0aGlvcGlhbiBoaWdobGFuZCBzaGVlcDogaW5jaWRlbmNlIGFuZCBvYnNlcnZhdGlvbnMgb24gcGF0aG9nZW5lc2lzIGFuZCBjbGluaWNhbCBzaWduczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJvcC4gQW5pbS4gSGVhbHRoIFByb2Q7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0xNS0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBjaGFyYWN0ZXJpc2F0aW9uIG9mIFRhZW5pYSBtdWx0aWNlcHMgY3lzdHMgZnJvbSBydW1pbmFudHMgaW4gR3JlZWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbmZlY3QuIEdlbmV0LiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9MTEwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZW51cm9zaXMgc3RhdHVzIGluIEFmcm8tQXNpYW4gcmVnaW9uOiBhIHJldmlldzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U21hbGwgUnVtaW5hbnQgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTY0OyAgY2l0YXRpb25fcGFnZXM9MTk3LTIwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFlbmlhIG11bHRpY2VwczogYSByYXJlIGh1bWFuIGNlc3RvZGUgaW5mZWN0aW9uIGluIElzcmFlbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VmV0LiBJdGFsOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9NjIxLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgb3JnYW5pemF0aW9uLCBvbnRvZ2VueSwgYW5kIG9yaWVudGF0aW9uIG9mIHRoZSBDZXN0b2RhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1RdWFydC4gUmV2LiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTYyOyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MjMtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IlJvYmVydHMgTC4gUy4gUy4sIEphbm92eSBHLiBELiAgMjAwOSAsIEdlcmFsZCBELiBTY2htaWR0ICZhbXA7IExhcnJ5IFMuIFJvYmVydHPigJkgRm91bmRhdGlvbnMgb2YgcGFyYXNpdG9sb2d5LiAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Mb25nLXRlcm0gZm9sbG93LXVwIG9mIHRoZSBzdXJnaWNhbCB0cmVhdG1lbnQgb2YgaW50cmFjcmFuaWFsIGNvZW51cm9zaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJyLiBKLiBOZXVyb3N1cmc7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTM5LTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wdXRlZCB0b21vZ3JhcGh5IGFuZCBtYWduZXRpYyByZXNvbmFuY2UgaW1hZ2luZyBvZiBjZXJlYnJhbCBjb2VudXJvc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TdXJnLiBOZXVyb2w7ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz01NDgtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZhY2NpbmF0aW9uIHdpdGggcmVjb21iaW5hbnQgb25jb3NwaGVyZSBhbnRpZ2VucyByZWR1Y2VzIHRoZSBzdXNjZXB0aWJpbGl0eSBvZiBzaGVlcCB0byBpbmZlY3Rpb24gd2l0aCBUYWVuaWEgbXVsdGljZXBzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIFBhcmFzaXRvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPTEwNDEtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWxpbWluYXJ5IGZpZWxkIHRyaWFsIG9mIGEgdmFjY2luZSBhZ2FpbnN0IGNvZW51cm9zaXMgY2F1c2VkIGJ5IFRhZW5pYSBtdWx0aWNlcHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVZldC4gUGFyYXNpdG9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE2MjsgIGNpdGF0aW9uX3BhZ2VzPTI4NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DZXN0b2RlIHZhY2NpbmVzOiBvcmlnaW5zLCBjdXJyZW50IHN0YXR1cyBhbmQgZnV0dXJlIHByb3NwZWN0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGFyYXNpdG9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEzMzsgIGNpdGF0aW9uX3BhZ2VzPVMyNy00MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZXMgb2YgZm91ciB0YXBld29ybSBzcGVjaWVzIHJldmVhbCBhZGFwdGF0aW9ucyB0byBwYXJhc2l0aXNtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDk2OyAgY2l0YXRpb25fcGFnZXM9NTctNjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbm9taWNzIHJldmVhbHMgYWRhcHRpdmUgZXZvbHV0aW9uIG9mIEFzaWFuIHRhcGV3b3JtIGluIHN3aXRjaGluZyB0byBhIG5ldyBpbnRlcm1lZGlhdGUgaG9zdDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBDb21tdW47ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTEyODQ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbm9tZSBvZiB0aGUgaHlkYXRpZCB0YXBld29ybSBFY2hpbm9jb2NjdXMgZ3JhbnVsb3N1czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTExNjgtNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIG5lZ2xlY3RlZCBjZXN0b2RlIFRhZW5pYSBtdWx0aWNlcHMgbWljcm9STkFzIGJ5IGlsbHVtaW5hIHNlcXVlbmNpbmcgYW5kIGJpb2luZm9ybWF0aWMgYW5hbHlzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBWZXQuIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTYyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBhbmQgYWNjdXJhdGUgc2hvcnQgcmVhZCBhbGlnbm1lbnQgd2l0aCBCdXJyb3dzLVdoZWVsZXIgdHJhbnNmb3JtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTE3NTQtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBpbG9uOiBhbiBpbnRlZ3JhdGVkIHRvb2wgZm9yIGNvbXByZWhlbnNpdmUgbWljcm9iaWFsIHZhcmlhbnQgZGV0ZWN0aW9uIGFuZCBnZW5vbWUgYXNzZW1ibHkgaW1wcm92ZW1lbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1lMTEyOTYzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21vc29tZS1zY2FsZSBzY2FmZm9sZGluZyBvZiBkZSBub3ZvIGdlbm9tZSBhc3NlbWJsaWVzIGJhc2VkIG9uIGNocm9tYXRpbiBpbnRlcmFjdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTExMTktMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUF1dG9tYXRlZCBldWthcnlvdGljIGdlbmUgc3RydWN0dXJlIGFubm90YXRpb24gdXNpbmcgRVZpZGVuY2VNb2RlbGVyIGFuZCB0aGUgcHJvZ3JhbSB0byBhc3NlbWJsZSBzcGxpY2VkIGFsaWdubWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz1SNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBVbml2ZXJzYWwgUHJvdGVpbiBSZXNvdXJjZSAoVW5pUHJvdCk6IGFuIGV4cGFuZGluZyB1bml2ZXJzZSBvZiBwcm90ZWluIGluZm9ybWF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPUQxODctOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyUHJvU2NhbjogcHJvdGVpbiBkb21haW5zIGlkZW50aWZpZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9VzExNi0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEtFR0cgcmVzb3VyY2UgZm9yIGRlY2lwaGVyaW5nIHRoZSBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9RDI3Ny04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFuZGVtIHJlcGVhdHMgZmluZGVyOiBhIHByb2dyYW0gdG8gYW5hbHl6ZSBETkEgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTU3My04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGVyc3BlY3RpdmUgLSBzeW50ZW55IGFuZCBjb2xsaW5lYXJpdHkgaW4gcGxhbnQgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zMjA7ICBjaXRhdGlvbl9wYWdlcz00ODYtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3J0aG9GaW5kZXI6IHNvbHZpbmcgZnVuZGFtZW50YWwgYmlhc2VzIGluIHdob2xlIGdlbm9tZSBjb21wYXJpc29ucyBkcmFtYXRpY2FsbHkgaW1wcm92ZXMgb3J0aG9ncm91cCBpbmZlcmVuY2UgYWNjdXJhY3k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NVVNDTEU6IG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCB3aXRoIGhpZ2ggYWNjdXJhY3kgYW5kIGhpZ2ggdGhyb3VnaHB1dDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0xNzkyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5ldyBhbGdvcml0aG1zIGFuZCBtZXRob2RzIHRvIGVzdGltYXRlIG1heGltdW0tbGlrZWxpaG9vZCBwaHlsb2dlbmllczogYXNzZXNzaW5nIHRoZSBwZXJmb3JtYW5jZSBvZiBQaHlNTCAzLjA7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVN5c3QuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTk7ICBjaXRhdGlvbl9wYWdlcz0zMDctMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3RUZXN0IDM6IGZhc3Qgc2VsZWN0aW9uIG9mIGJlc3QtZml0IG1vZGVscyBvZiBwcm90ZWluIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0xMTY0LTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBBTUwgNDogcGh5bG9nZW5ldGljIGFuYWx5c2lzIGJ5IG1heGltdW0gbGlrZWxpaG9vZDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTU4Ni05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5ldGljIHJlbGF0aW9uc2hpcHMgd2l0aGluIEVjaGlub2NvY2N1cyBhbmQgVGFlbmlhIHRhcGV3b3JtcyAoQ2VzdG9kYTogdGFlbmlpZGFlKTogQW4gaW5mZXJlbmNlIGZyb20gbnVjbGVhciBwcm90ZWluLWNvZGluZyBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBQaHlsb2dlbmV0LiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9NjI4LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1LYUtzX0NhbGN1bGF0b3IgMi4wOiBhIHRvb2xraXQgaW5jb3Jwb3JhdGluZyBnYW1tYS1zZXJpZXMgbWV0aG9kcyBhbmQgc2xpZGluZyB3aW5kb3cgc3RyYXRlZ2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21pY3MgUHJvdGVvbWljcyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NzctODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBITU1UT1AgdHJhbnNtZW1icmFuZSB0b3BvbG9neSBwcmVkaWN0aW9uIHNlcnZlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz04NDktNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFzc2VtYmxpbmcgbGFyZ2UgZ2Vub21lcyB3aXRoIHNpbmdsZS1tb2xlY3VsZSBzZXF1ZW5jaW5nIGFuZCBsb2NhbGl0eS1zZW5zaXRpdmUgaGFzaGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NjIzLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CVVNDTzogYXNzZXNzaW5nIGdlbm9tZSBhc3NlbWJseSBhbmQgYW5ub3RhdGlvbiBjb21wbGV0ZW5lc3Mgd2l0aCBzaW5nbGUtY29weSBvcnRob2xvZ3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9MzIxMC0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCB0cmFuc3Bvc2FibGUgZWxlbWVudHM6IHdoZXJlIGdlbmV0aWNzIG1lZXRzIGdlbm9taWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTMyOS00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzaW1wbGUsIGZhc3QsIGFuZCBhY2N1cmF0ZSBhbGdvcml0aG0gdG8gZXN0aW1hdGUgbGFyZ2UgcGh5bG9nZW5pZXMgYnkgbWF4aW11bSBsaWtlbGlob29kOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeXN0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTUyOyAgY2l0YXRpb25fcGFnZXM9Njk2LTcwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXBwcm94aW1hdGUgbGlrZWxpaG9vZCBjYWxjdWxhdGlvbiBvbiBhIHBoeWxvZ2VueSBmb3IgYmF5ZXNpYW4gZXN0aW1hdGlvbiBvZiBkaXZlcmdlbmNlIHRpbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz0yMTYxLTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJNYXJyIE0uICAyMDExICwgRGV2ZWxvcG1lbnRhbCBSZWd1bGF0b3J5IEdlbmVzIGluIFBhcmFzaXRpYyBGbGF0d29ybXMuIERvY3RvcmFsIGRpc3NlcnRhdGlvbiwgRGVwYXJ0bWVudCBvZiBMaWZlIFNjaWVuY2VzIGFuZCB0aGUgTmF0dXJhbCBIaXN0b3J5IE11c2V1bSwgSW1wZXJpYWwgQ29sbGVnZSBMb25kb24uICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBULWJveCBnZW5lIGZhbWlseTogZW1lcmdpbmcgcm9sZXMgaW4gZGV2ZWxvcG1lbnQsIHN0ZW0gY2VsbHMgYW5kIGNhbmNlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RGV2ZWxvcG1lbnQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTQxOyAgY2l0YXRpb25fcGFnZXM9MzgxOS0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iS2FuYW1vcmkgSy4gIDIwMTcgLCBJbiB2aXZvIE4tMTUgTVJTIHN0dWR5IG9mIGdsdXRhbWF0ZSBtZXRhYm9saXNtIGluIHRoZSByYXQgYnJhaW4uIEFuYWwuIEJpb2NoZW0uLCA1MjkgLCAxNzkg4oCTIDE5Mi4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xpbmljYWwgYXNwZWN0cyBvZiB1cmVhIGN5Y2xlIGR5c2Z1bmN0aW9uIGFuZCBhbHRlcmVkIGJyYWluIGVuZXJneSBtZXRhYm9saXNtIG9uIG1vZHVsYXRpb24gb2YgZ2x1dGFtYXRlIHJlY2VwdG9ycyBhbmQgdHJhbnNwb3J0ZXJzIGluIGFjdXRlIGFuZCBjaHJvbmljIGh5cGVyYW1tb25lbWlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9tZWQuIFBoYXJtYWNvdGhlcjsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT04MTsgIGNpdGF0aW9uX3BhZ2VzPTE5Mi0yMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1lY2hhbmlzbXMgb2YgZ2x1dGFtYXRlIHRveGljaXR5IGluIG11bHRpcGxlIHNjbGVyb3NpczogYmlvbWFya2VyIGFuZCB0aGVyYXBldXRpYyBvcHBvcnR1bml0aWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1MYW5jZXROZXVyb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0xMDg5LTEwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIGV4dGVudCBvZiBob3Jpem9udGFsIGdlbmUgdHJhbnNmZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS5TLkE7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9NDQ4OS05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSByb2xlIGZvciBob3N0LXBhcmFzaXRlIGludGVyYWN0aW9ucyBpbiB0aGUgaG9yaXpvbnRhbCB0cmFuc2ZlciBvZiB0cmFuc3Bvc29ucyBhY3Jvc3MgcGh5bGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjQ7ICBjaXRhdGlvbl9wYWdlcz0xMzQ3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBhbmQgZWNvbG9naWNhbCBpbXBhY3RzIG9mIGhvcml6b250YWwgZ2VuZSB0cmFuc2ZlciBpbiBldWthcnlvdGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBHZW5ldC4gRGV2OyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NjEzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb25jaXNlIGd1aWRlIHRvIHBoYXJtYWNvbG9neSAyMDEzLzE0OiBlbnp5bWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Cci4gSi4gUGhhcm1hY29sOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE3MDsgIGNpdGF0aW9uX3BhZ2VzPTE3OTctODY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QYXJhc2l0ZSBuZXVyb3BlcHRpZGUgYmlvbG9neTogc2VlZGluZyByYXRpb25hbCBkcnVnIHRhcmdldCBzZWxlY3Rpb24/OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JbnQuIEouIFBhcmFzaXRvbC4gRHJ1ZzsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9NzYtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhY0JpbyBzZXF1ZW5jaW5nIGFuZCBpdHMgYXBwbGljYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWljcy4gUHJvdGVvbWljcyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTI3OC04OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI1LzUvNDk5LzI2MDc2OTczL2RzeTAyMC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQ29lbnVyb3NpcywgY2F1c2VkIGJ5IHRoZSBsYXJ2YWwgY29lbnVydXMgb2YgdGhlIHRhcGV3b3JtIFRhZW5pYSBtdWx0aWNlcHMsIGlzIGEgZmF0YWwgY2VudHJhbCBuZXJ2b3VzIHN5c3RlbSBkaXNlYXNlIGluIGJvdGggc2hlZXAgYW5kIGh1bWFucy4gVGhvdWciIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTUwNDU1NDgmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkX+njab+padCjGayOiBV6HoeHYSg=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f499%2f5045548" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f499%2f5045548" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=110919" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/5">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="122671172">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671172">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671174">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671174">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671179">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671179">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671194">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671194">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671228">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671228">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671231">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671231">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="122671239">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#122671239">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/5/489/5036360">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/5/511/5047107">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">The genome of tapeworm <em>Taenia multiceps</em> sheds light on understanding parasitic mechanism and control of coenurosis disease
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wenhui Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wenhui+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Wenhui Li</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bo Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bo+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu B&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Bo Liu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yang Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yang+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yang Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yang Yang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuwei Ren</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuwei+Ren">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ren Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ren Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yuwei Ren</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shuai Wang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shuai+Wang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Wang S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Wang S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Shuai Wang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Conghui Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Conghui+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu C">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu C&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Conghui Liu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nianzhang Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nianzhang+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Nianzhang Zhang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Zigang Qu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Zigang+Qu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Qu Z">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Qu Z&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Zigang Qu</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wanxu Yang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wanxu+Yang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yang W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yang W&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Wanxu Yang</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yan Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yan+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang Y&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Yan Zhang</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hongbing Yan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hongbing+Yan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yan H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yan H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hongbing Yan</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fan Jiang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fan+Jiang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jiang F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jiang F&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Fan Jiang</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Li Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Li+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li L&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Li Li</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shuqu Li</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shuqu+Li">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Li S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Li S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Shuqu Li</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wanzhong Jia</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China</div><div class="aff">Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Disease, Yangzhou, Jiangsu, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wanzhong+Jia">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jia W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jia W&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Wanzhong Jia</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hong Yin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China</div><div class="aff">Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Disease, Yangzhou, Jiangsu, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hong+Yin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yin H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yin H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hong Yin</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xuepeng Cai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xuepeng+Cai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Cai X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Cai X&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Xuepeng Cai</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tao Liu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Annoroad Gene Tech. (Beijing) Co Ltd, Beijing, China</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tao+Liu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liu T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liu T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tao Liu</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Donald P McManus</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Molecular Parasitology Laboratory, QIMR Berghofer Medical Research Institute, Brisbane, Australia</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy020-cor1" data-open="dsy020-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +86 0931 8342 675. Email: <a href="mailto:don.mcmanus@qimrberghofer.edu.au" target="_blank">don.mcmanus@qimrberghofer.edu.au</a> (D.P.M.); <a href="mailto:fanweiagis@126.com" target="_blank">fanweiagis@126.com</a> (W.F.); <a href="mailto:fubaoquan@caas.cn" target="_blank">fubaoquan@caas.cn</a> (B.F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Donald+P+McManus">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=McManus D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;McManus D&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Donald P McManus</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wei Fan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Agricultural Genomic Institute, Chinese Academy of Agricultural Sciences, Shenzhen, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy020-cor1" data-open="dsy020-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +86 0931 8342 675. Email: <a href="mailto:don.mcmanus@qimrberghofer.edu.au" target="_blank">don.mcmanus@qimrberghofer.edu.au</a> (D.P.M.); <a href="mailto:fanweiagis@126.com" target="_blank">fanweiagis@126.com</a> (W.F.); <a href="mailto:fubaoquan@caas.cn" target="_blank">fubaoquan@caas.cn</a> (B.F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wei+Fan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fan W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fan W&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Wei Fan</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Baoquan Fu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">State Key Laboratory of Veterinary Etiological Biology, Key Laboratory of Veterinary Parasitology of Gansu Province, Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences, Lanzhou, China</div><div class="aff">Jiangsu Co-innovation Center for Prevention and Control of Important Animal Infectious Disease, Yangzhou, Jiangsu, China</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy020-cor1" data-open="dsy020-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +86 0931 8342 675. Email: <a href="mailto:don.mcmanus@qimrberghofer.edu.au" target="_blank">don.mcmanus@qimrberghofer.edu.au</a> (D.P.M.); <a href="mailto:fanweiagis@126.com" target="_blank">fanweiagis@126.com</a> (W.F.); <a href="mailto:fubaoquan@caas.cn" target="_blank">fubaoquan@caas.cn</a> (B.F.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Baoquan+Fu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fu B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fu B&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Baoquan Fu</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy020-FM2"><div class="fn"><div class="footnote-content">Wenhui Li, Bo Liu, Yang Yang, Yuwei Ren, Shuai Wang and Conghui Liu authors contributed equally to this work.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 5, 1 October 2018, Pages 499–510, <a href='https://doi.org/10.1093/dnares/dsy020'>https://doi.org/10.1093/dnares/dsy020</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">25 June 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">03 November 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">30 May 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="5045548" href="/dnaresearch/article-pdf/25/5/499/26076973/dsy020.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Wenhui Li, Bo Liu, Yang Yang, Yuwei Ren, Shuai Wang, Conghui Liu, Nianzhang Zhang, Zigang Qu, Wanxu Yang, Yan Zhang, Hongbing Yan, Fan Jiang, Li Li, Shuqu Li, Wanzhong Jia, Hong Yin, Xuepeng Cai, Tao Liu, Donald P McManus, Wei Fan, Baoquan Fu;  The genome of tapeworm <em>Taenia multiceps</em> sheds light on understanding parasitic mechanism and control of coenurosis disease, <em>DNA Research</em>, Volume 25, Issue 5, 1 October 2018, Pages 499–510, <a href='https://doi.org/10.1093/dnares/dsy020'>https://doi.org/10.1093/dnares/dsy020</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/5045548?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/5045548?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/5045548?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/5045548?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/5045548?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/5045548?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/5045548?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=The%20genome%20of%20tapeworm%20%26lt%3Bem%26gt%3BTaenia%20multiceps%26lt%3B%2Fem%26gt%3B%20sheds%20light%20on%20understanding%20parasitic%20mechanism%20and%20control%20of%20coenurosis%20disease&amp;publicationDate=2018-06-25&amp;volumeNum=25&amp;issueNum=5&amp;author=Li%2C%20Wenhui%3B%20Liu%2C%20Bo&amp;startPage=499&amp;endPage=510&amp;contentId=10.1093%2Fdnares%2Fdsy020&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f499%2f5045548" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=122671172 id="122671172" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Coenurosis, caused by the larval coenurus of the tapeworm <em>Taenia multiceps</em>, is a fatal central nervous system disease in both sheep and humans. Though treatment and prevention options are available, the control of coenurosis still faces presents great challenges. Here, we present a high-quality genome sequence of <em>T. multiceps</em> in which 240 Mb (96%) of the genome has been successfully assembled using Pacbio single-molecule real-time (SMRT) and Hi-C data with a N50 length of 44.8 Mb. In total, 49.5 Mb (20.6%) repeat sequences and 13, 013 gene models were identified. We found that <em>Taenia</em> spp. have an expansion of transposable elements and recent small-scale gene duplications following the divergence of <em>Taenia</em> from <em>Echinococcus</em>, but not in <em>Echinococcus</em> genomes, and the genes underlying environmental adaptability and dosage effect tend to be over-retained in the <em>T. multiceps</em> genome. Moreover, we identified several genes encoding proteins involved in proglottid formation and interactions with the host central nervous system, which may contribute to the adaption of <em>T. multiceps</em> to its parasitic life style. Our study not only provides insights into the biology and evolution of <em>T. multiceps</em>, but also identifies a set of species-specific gene targets for developing novel treatment and control tools for coenurosis.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Taenia multiceps&quot;"><em>Taenia multiceps</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="coenurosis">coenurosis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="genome">genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="parasitism">parasitism</a></div>                            <h2 scrollto-destination=122671174 id="122671174" class="section-title" >1. Introduction</h2>
<p>Coenurosis is an often fatal central nervous system (CNS) parasitic disease mainly in sheep and other ungulates, this frequently leads to their death and huge socioeconomic losses, especially in developing countries. This parasite disease is caused by the larval stage (<em>Coenurus cerebralis</em>) of the tapeworm <em>Taenia multiceps</em><a reveal-id="dsy020-B1" data-open="dsy020-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> (Phylum Platyhelminthes). The parasite can also cause zoonotic infections in humans, but has been largely neglected due to a lack of diagnostic techniques and studies.<a reveal-id="dsy020-B2" data-open="dsy020-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Two hosts are required to complete its life cycle. The adult tapeworm inhabits the small intestine of the definitive host (dog, wolf and fox), while the larval coenurus stage parasitizes the brain or spinal cord of the intermediate host (sheep, goat and cattle),<a reveal-id="dsy020-B3" data-open="dsy020-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> causing a disease called ‘staggers’ or ‘gid’ resulting in pronounced intracranial pressure, leading to headache, ataxia, hypermetria, head deviation, blindness, stumbling, paralysis and even death.<a reveal-id="dsy020-B4 dsy020-B5 dsy020-B6" data-open="dsy020-B4 dsy020-B5 dsy020-B6" class="link link-ref link-reveal xref-bibr"><sup>4–6</sup></a> Coenurosis has spread across most regions of the world, especially in developing countries in Africa and southeast Asia that are involved in the husbandry of sheep or goats, resulting in serious economic losses.<a reveal-id="dsy020-B7" data-open="dsy020-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Notably, coenurosis is also a zoonosis which can lead to serious pathological conditions in humans, and appears to be more common than previously assumed.<a reveal-id="dsy020-B8" data-open="dsy020-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a></p><p><em>T. multiceps</em> has evolved several strategies to adapt to a parasitic life style including the loss of respiratory organs and a digestive tract so that andnutrients are absorbed through its unique external tegument.<a reveal-id="dsy020-B9" data-open="dsy020-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> It is monoecious, producing diploid eggs by self-fertilization which give rise to infective oncospheres.<a reveal-id="dsy020-B10" data-open="dsy020-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Further, the germinative neck region of the adult worm of <em>T. multiceps</em> can produce hundreds of proglottids (segments), each of which has a set of complete reproductive organs and contains tens of thousands of oncospheres (eggs) after maturation.<a reveal-id="dsy020-B9" data-open="dsy020-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> In addition, one oncosphere can develop into a cysticercus which possesses unusual powers of asexual multiplication, forming a bladder or coenurus which can give rise to hundreds of daughter protoscoleces, each of which has the potential to grow into an adult worm, if ingested by a definitive host. The characteristics of nutrient absorption and reproduction have contributed to the wide-spread, global distribution of <em>T. multiceps.</em></p><p>Though treatment and preventative measures are available, the control of coenurosis is still challenging. Burning or burying infected intermediate host offal is the most simple and effective way to block the transmission of <em>T. multiceps</em> to canine definitive hosts, but these procedures result in considerable economic loss. Surgery after general anaesthesia of an infected animal has been used for the removal of the coenurus cyst,<a reveal-id="dsy020-B11" data-open="dsy020-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsy020-B12" data-open="dsy020-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> but the procedure is limited under field conditions due to economic constraints. Antiparasitic drugs, such as a combination of fenbendazole, praziquantel and albendazole, are effective against the migrating larvae and represent a more practicable method for expelling <em>T. multiceps</em>, but they are often associated with side effects in the host. Development of an effective vaccine can complement interventions for the prevention of coenurosis, and in recent years, the recombinant proteins, Tm16 and Tm18, have been developed as vaccine candidates, having been shown to provide some protection in sheep against experimental infection with <em>T. multiceps</em> eggs.<a reveal-id="dsy020-B13" data-open="dsy020-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsy020-B14" data-open="dsy020-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> However, the vaccine efficacy is comparatively modest compared with other recombinant anti-cestode vaccines that have been developed.<a reveal-id="dsy020-B15" data-open="dsy020-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Therefore, the development of more effective and safe interventions to control coenurosis is still an urgent need.</p><p>The recent sequencing of tapeworm genomes has facilitated a better understanding of their genetic makeup, providing new insight on the functional biology and mechanisms of pathogenesis in cestodes as well as providing new targets for control tools leading to improved prevention and treatment of cestodiases. To date, several tapeworm genomes, including two <em>Echinococcus</em> species (<em>E. multilocularis</em> and <em>E. granulosu</em>s) and three <em>Taenia</em> species (<em>T. solium</em>, <em>T. saginata</em> and <em>T. asiatica</em>) have been sequenced, with assembled genome sizes ranging from 114 to 169 Mb.<a reveal-id="dsy020-B16 dsy020-B17 dsy020-B18" data-open="dsy020-B16 dsy020-B17 dsy020-B18" class="link link-ref link-reveal xref-bibr"><sup>16–18</sup></a> However, there is limited genetic information reported for <em>T. multiceps</em>, apart from some limited transcriptome and microRNA data.<a reveal-id="dsy020-B2" data-open="dsy020-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a><sup>,</sup><a reveal-id="dsy020-B19" data-open="dsy020-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Here, we present a high-quality genome assembly for <em>T. multiceps</em> together with transcriptomic information for several of its developmental stages. Our study not only provides insights into the biology and evolution of <em>T. multiceps</em>, but also identifies a set of potentially novel drug and vaccine targets, which we anticipate will contribute to the development of much needed new treatment and control measures for coenurosis.</p>                            <h2 scrollto-destination=122671179 id="122671179" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=122671180 id="122671180" class="section-title" >2.1. Samples collection and sequencing</h3>
<p>Adult worms of <em>T. multiceps</em> were collected 68 days after infection from a dog experimentally infected with protoscoleces. The protoscoleces and cyst wall were separated from the brain of a naturally infected sheep around 3 months infection collected in Gansu province, China. According to the Animal Ethics Procedures and Guidelines of the People’s Republic of China, animals used in this study were cared in accordance with good animal practice and the study was permitted by the Institutional Committee for the Care and Use of Experimental Animals of Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences (no. LVRIAEC 2010-002). Scolex-neck proglottids, immature-mature proglottids and gravid proglottids were obtained by cutting the whole adult worms into three parts based on the tapeworm proglottids characteristics.<a reveal-id="dsy020-B10" data-open="dsy020-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Oncospheres were hatched by 0.75% sodium hypochlorite and activated under artificial intestinal fluid conditions in vitro. Genomic DNA was extracted from a single adult <em>T. multiceps</em> for constructing four Illumina 400-bp insert libraries and sequenced on Illumina HiSeq 2500 platforms. Two PacBio 20-kb insert library were constructed and sequenced on RSII and Sequel, respectively. Total mRNA was extracted from the seven <em>T. multiceps</em> materials (activated oncospheres, protoscolices, cyst wall and adult, scolex-neck proglottids, immature-mature proglottids and gravid proglottids) for constructing cDNA libraries (insert 350-bp) (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Methods</a></span> and Results section), and sequenced on an Illumina HiSeq 2500 sequencer.</p>                            <h3 scrollto-destination=122671182 id="122671182" class="section-title" >2.2. Genome assembly and annotation</h3>
<p>The Illumina raw reads were filtered by trimming the adapter sequence and low-quality part, resulting in a clean and high-quality reads data with average error rate &lt; 0.001. For the PacBio raw data, the short subreads (&lt;2 kb) and low-quality (error rate &gt; 0.2) subreads were filtered out, and only one representative subread was retained for each PacBio read. The clean PacBio reads were assembled by the software canu (<a class="link link-uri" href="http://canu.readthedocs.io/en/stable/index.html#" target="_blank">http://canu.readthedocs.io/en/stable/index.html#</a>), then Illumina reads were aligned to the contigs by Burrows-Wheeler Aligner-MEM (BWA-MEM),<a reveal-id="dsy020-B20" data-open="dsy020-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> and single base errors in the contigs were corrected by Pilon (v1.16)<a reveal-id="dsy020-B21" data-open="dsy020-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Methods</a></span> and Results section).</p><p>For the Hi-C-based proximity-guided assembly, we removed duplications and kept reads that uniquely mapped to the reference genome. The assembly package, Lachesis was applied to do clustering, ordering and orienting. Based on the agglomerative hierarchical clustering algorithm,<a reveal-id="dsy020-B22" data-open="dsy020-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> we clustered the scaffolds into 100 N. Then the longest acyclic spanning tree which be called ‘trunk’ was build according to the relations between the normalized HiC interactions and the scaffolds that were excluded from the trunk were sites that maximized the amount of linkage between adjacent scaffolds. For each chromosome cluster, we got an exact scaffold order of the internal groups and traversed all the directions of the scaffolds through a weighted directed acyclic graph to predict orientation for each of the scaffolds.</p><p>The gene models in <em>T. multiceps</em> genome were predicted by EVidence Modeler v1.1.1,<a reveal-id="dsy020-B23" data-open="dsy020-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> integrating evidences from <em>ab initio</em> predictions, homology-based searches and RNA-seq alignments (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Methods</a></span> and Results section). Then, the protein-coding sequences were mapped by RNA-seq data and functionally annotated using UniProt<a reveal-id="dsy020-B24" data-open="dsy020-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> and InterProScan (5.16-55.0) databases.<a reveal-id="dsy020-B25" data-open="dsy020-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Finally, the gene models were retained if they had at least one supporting evidence from UniProt database, InterProScan domain and RNA-seq data. Gene functional annotation was performed by aligning the protein sequences to NCBI NR, UniProt, eggNOG and KEGG databases using BLASTP v2.3.0+ with E-value cut-off of 10<sup>−5</sup>. The pathway analysis and functional classification were conducted based on KEGG database.<a reveal-id="dsy020-B26" data-open="dsy020-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> InterProScan<a reveal-id="dsy020-B25" data-open="dsy020-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> was used to assign preliminary gene ontology (GO) terms, Pfam domains and IPR domains to the gene models.</p><p>A <em>de novo</em> repeat library for <em>T. multiceps</em> was constructed by RepeatModeler (v1.0.4; <a class="link link-uri" href="http://www.repeatmasker.org/RepeatModeler.html" target="_blank">http://www.repeatmasker.org/RepeatModeler.html</a>). transposable elements (TEs) in the <em>T. multiceps</em> genome were also identified by RepeatMasker (v4.0.6; <a class="link link-uri" href="http://www.repeatmasker.org/" target="_blank">http://www.repeatmasker.org/</a>) using both Repbase library and the <em>de novo</em> library (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Methods</a></span> and Results section). Tandem repeats were predicted using Tandem Repeats Finder v4.07b.<a reveal-id="dsy020-B27" data-open="dsy020-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> The divergence rates of TEs were calculated between the identified TE elements in the genome and their consensus sequence at the repeat family level.</p>                            <h3 scrollto-destination=122671187 id="122671187" class="section-title" >2.3. Evolution analysis</h3>
<p>Duplicated genomic fragments were identified by MCscanX,<a reveal-id="dsy020-B28" data-open="dsy020-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> requiring at least 10 paralogous gene pairs per collinear block, and the duplicate_gene_classifier in MCscanX was implemented to classify the origins of the duplicate genes into different types. Orthologous and paralogous gene families were assigned from 14 species (<em>T. multiceps</em>, <em>T. saginata</em>, <em>T. asiatica</em>, <em>T. solium</em>, <em>E. granulosus</em>, <em>E. multilocularis</em>, <em>Hymenolepis microstoma</em>, <em>Brugia malayi</em>, <em>Caenorhabditis elegans</em>, <em>Clonorchis sinensis</em>, <em>Pristionchus pacificus</em>, <em>Schistosoma mansoni</em>, <em>Fasciola hepatica</em> and <em>Trichinella spiralis</em>) by OrthoFinder<a reveal-id="dsy020-B29" data-open="dsy020-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> with default parameters.</p><p>Gene families that contain only one gene for each species were selected to construct the phylogenetic tree. The protein sequences of each gene family was independently aligned by muscle v3.8.31<a reveal-id="dsy020-B30" data-open="dsy020-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> and then concatenated into one super-sequence. The phylogenetic tree was constructed by maximum likelihood (ML) using PhyML v3.0<a reveal-id="dsy020-B31" data-open="dsy020-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> with best-fit model (MtMAN) that was estimated by ProtTest3.<a reveal-id="dsy020-B32" data-open="dsy020-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> The Bayesian-Relaxed Molecular Clock approach was adopted to estimate the neutral evolutionary rate and species divergence time using the programme MCMCTree, implemented in PAML v4.9 package.<a reveal-id="dsy020-B33" data-open="dsy020-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> The calibration time (fossil record time) interval (27–29 Mya) of <em>Echinococcus</em> species was adopted from previous results.<a reveal-id="dsy020-B34" data-open="dsy020-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a></p><p>Paralogous gene pairs with a length of more than 100 amino acids and E value of 1 e<sup>−10</sup> were used to calculate synonymous mutation rate (<em>Ks</em>) by KaKs_Calculator 2.0 with default parameter.<a reveal-id="dsy020-B35" data-open="dsy020-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><em>Ks</em> values between two species were calculated using the syntenic gene pairs.</p><p>The positively selected genes (PSGs) were identified based on the 3,316 one-to-one orthologous gene groups from genomes of the six tapeworms from the results by OrthoFinder.<a reveal-id="dsy020-B29" data-open="dsy020-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Multiple alignments of protein-coding DNA sequence were generated as referenced by the protein alignments using ParaAT (v1.0) and MAFFT (v7.147b). The gaps in the alignment results were removed, and likelihood ratio tests (LRTs) for selection (<em>P</em> &lt; 0.05) on the <em>T. multiceps</em> lineage of the phylogenetic tree were performed using Codeml with a modified branch-site model A (model = 2, <em>N</em> sites = 2) implemented in the PAML package (v4.8).</p>                            <h3 scrollto-destination=122671192 id="122671192" class="section-title" >2.4. Identification of potential drug targets</h3>
<p>The enzyme drug targets from US Food and Drug Administration (FDA) were used to identify homologues in <em>T. multiceps</em> based on NR and UniProt databases. Furthermore, the G-protein coupled receptors (GPCRs) proteins were identified based on Pfam annotation. The transmembrane domain of these GPCR proteins in <em>T. multiceps</em> was re-annotated by Prediction of transmembrane helices and topology of proteins (HMMTOP).<a reveal-id="dsy020-B36" data-open="dsy020-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Then, For the GPCR proteins containing more than two transmembrane domains were selected as potential drug targets.<a reveal-id="dsy020-B16" data-open="dsy020-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> On the other hand, the genes of <em>T. multiceps</em> involved in serotonin (5-HT) and acetylcholine (Ach) synaptic metabolic pathway were also selected as potential drug targets.<a reveal-id="dsy020-B16" data-open="dsy020-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Besides, previous report antigens, including antigen B/secreted antigen Ts8B1, EMY162, Tsol15, oncosphere protein, STARP (sporozoite threonine and asparagine-rich protein), cytoplasmic antigen 1, cytoplasmic antigen 1, GP50 and 8 kDa antigen, in other tapeworm species<a reveal-id="dsy020-B16" data-open="dsy020-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> were used to align against the proteins in <em>T. multiceps</em> by BLASTP with <em>E</em>-value &lt;1 e<sup>−5</sup>. Finally, to reduce the side-effect of potential drug target for host, the selected drug target genes were aligned to the host protein sequences, including dog, sheep and human respectively, by BLASTP with <em>E</em>-value &lt; 1 e<sup>−5</sup>. The species-specific drug target genes were identified based on the BLASTP hit.</p>                            <h2 scrollto-destination=122671194 id="122671194" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=122671195 id="122671195" class="section-title" >3.1. High-quality genome assembly and gene annotation</h3>
<p>We generated 31 Gigabase (Gb) PacBio Single-molecule real-time (SMRT) sequences with an average read length of 9.5 kb and 51 Gb Illumina paired-end sequences with a read length of 250 bp using DNA extracted from a single diploid adult worm of <em>T. multiceps</em> (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&amp;Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_information.doc?Expires=2147483647&amp;Signature=jIuQ--s09O7g7OkKSDE1O6ogPCl1b6MtP4mi6d7RJAntqreB1qTWQWUlm25XduzK8XPeMIpI9slbCO~Tt7hW0lLxrorAPEV1MyNrh2LKa~6MLocgWnyoz3XVKDYw9MtexuAaehtYtAAzFYyrAiUuvb7XQJxXgL66apVa9cQMe4WnTpNT33vcevI-vStbA3n26CMDRoNH6RGZU6VK~e0xOUDSSEYrA94PrX79Oxhv3Z5LjCEHAfkmsHDpHhEGutz2Go~EafD5a17R3mmRMksViphdFI6In0FGjuhLyKdRIvFVPi9qk93xtho9wtmqK6K2TV4rI8xfuDq5HpsOVijGMw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). The PacBio SMRT sequences were assembled by Canu,<a reveal-id="dsy020-B37" data-open="dsy020-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> giving rise to an assembly of 2,050 contigs with a total length of 240 Mb, a N50 length of 756 kb, and a N90 length of 34 kb. The GC content of the genome is 43.7%, which is similar to those of other reported Taeniidae species ranging from 41.9 to 43.2%<a reveal-id="dsy020-B16 dsy020-B17 dsy020-B18" data-open="dsy020-B16 dsy020-B17 dsy020-B18" class="link link-ref link-reveal xref-bibr"><sup>16–18</sup></a> (<a reveal-id="dsy020-T1" data-open="dsy020-T1" class="link link-reveal link-table xref-fig">Table 1</a>), but higher than that of <em>H. microstoma</em> (35.9%). Estimated from the k-mer frequency distribution, the genome sizes of sequenced <em>Taenia</em> species, including <em>T. saginata</em>,<a reveal-id="dsy020-B17" data-open="dsy020-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><em>T. asiatica</em><a reveal-id="dsy020-B17" data-open="dsy020-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> and <em>T. multiceps</em>, are all nearly 250 Mb, larger than those of <em>Echinococcus</em> species estimated from 154 to 195 Mb (<a reveal-id="dsy020-T1" data-open="dsy020-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <a reveal-id="dsy020-F1" data-open="dsy020-F1" class="link link-reveal link-table xref-fig">Fig. 1a</a>; <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&amp;Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). The long continuity information resulting from the PacBio SMRT data indicated that ∼96% of the genome has been successfully assembled in <em>T. multiceps</em>, which is much higher than those from the Illumina/454 based assembly of other tapeworm species that range from 60 to 75% (<a reveal-id="dsy020-T1" data-open="dsy020-T1" class="link link-reveal link-table xref-fig">Table 1</a>). Based on Hi-C technology, a total of 217.0 Mb (90.4%) assembled contigs were anchored and oriented to the seven linkage groups (<a reveal-id="dsy020-F1" data-open="dsy020-F1" class="link link-reveal link-table xref-fig">Fig. 1b</a>), with the longest 80.1 and shortest 8.5 Mb, respectively, suggesting that the assembly quality was much better than the published genomes of <em>Taenia</em> sepecies so far (<a reveal-id="dsy020-T1" data-open="dsy020-T1" class="link link-reveal link-table xref-fig">Table 1</a>). <div content-id="dsy020-T1" class="table-modal"><div class="table-wrap"><div id="dsy020-T1" data-id="dsy020-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Summary of assembly and annotation of tapeworm genomes</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Assembly feature </th><th><em>T. multiceps</em> </th><th><em>T. solium</em> </th><th><em>T. saginata</em> </th><th><em>T. asiatica</em> </th><th><em>E. multilocularis</em> </th><th><em>E. granulosus</em> </th><th><em>H. microstoma</em> </th></tr></thead><tbody><tr><td>Estimated genome size (Mb) </td><td>250 </td><td>— </td><td>260<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>260<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>154 </td><td>195 </td><td>— </td></tr><tr><td>Assembled sequences (Mb) </td><td>240 </td><td>122 </td><td>169 </td><td>168 </td><td>115 </td><td>114 </td><td>182 </td></tr><tr><td>Assembly coverage (%) </td><td>96 </td><td>— </td><td>65 </td><td>65 </td><td>75 </td><td>60 </td><td>— </td></tr><tr><td>Gaps Ratios (%) </td><td>0.05 </td><td>0.1 </td><td>1.6 </td><td>2.5 </td><td>0.3 </td><td>2.4 </td><td>10.5 </td></tr><tr><td>Longest scaffold size (Mb) </td><td>10.5 </td><td>0.7 </td><td>7.3 </td><td>4.2 </td><td>20.1 </td><td>15.9 </td><td>22.3 </td></tr><tr><td>N50 size of scaffold (Mb) </td><td>44.8 </td><td>0.1 </td><td>0.6 </td><td>0.3 </td><td>13.8 </td><td>5.2 </td><td>7.6 </td></tr><tr><td>N90 size of scaffold (kb) </td><td>8,527.5 </td><td>5.3 </td><td>29.4 </td><td>14.3 </td><td>2,924.3 </td><td>213.5 </td><td>40.7 </td></tr><tr><td>GC content in genome (%) </td><td>43.7 </td><td>42.9 </td><td>43.2 </td><td>43.1 </td><td>42.2 </td><td>41.9 </td><td>35.9 </td></tr><tr><td>Gene annotation </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Number of gene models </td><td>13,013 </td><td>12,481 </td><td>13,161 </td><td>13,323 </td><td>10,663 </td><td>10,245 </td><td>12,368 </td></tr><tr><td>BUSCO complete gene (ratio) </td><td>351 (81.8%) </td><td>382 (89.0%) </td><td>364 (84.8%) </td><td>360 (83.9%) </td><td>393 (91.6%) </td><td>388 (90.4%) </td><td>381 (88.8%) </td></tr><tr><td>Coding sequence size (Mb) </td><td>18.5 </td><td>15.5 </td><td>13.3 </td><td>13.5 </td><td>15.7 </td><td>15.2 </td><td>16.9 </td></tr><tr><td>Average CDS size (bp) </td><td>1,424 </td><td>1,242 </td><td>1,011 </td><td>1,013 </td><td>1,472 </td><td>1, 484 </td><td>1, 366 </td></tr><tr><td>Average exon number </td><td>6.6 </td><td>5.6 </td><td>4.4 </td><td>4.3 </td><td>6.7 </td><td>6.8 </td><td>6.0 </td></tr><tr><td>Average exon size (bp) </td><td>215 </td><td>222 </td><td>232 </td><td>234 </td><td>218 </td><td>218 </td><td>228 </td></tr><tr><td>GC content in coding region (%) </td><td>50.9 </td><td>50.1 </td><td>50.1 </td><td>50.1 </td><td>50.0 </td><td>50.0 </td><td>44.4 </td></tr></tbody></table></div><div content-id="dsy020-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Assembly feature </th><th><em>T. multiceps</em> </th><th><em>T. solium</em> </th><th><em>T. saginata</em> </th><th><em>T. asiatica</em> </th><th><em>E. multilocularis</em> </th><th><em>E. granulosus</em> </th><th><em>H. microstoma</em> </th></tr></thead><tbody><tr><td>Estimated genome size (Mb) </td><td>250 </td><td>— </td><td>260<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>260<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>154 </td><td>195 </td><td>— </td></tr><tr><td>Assembled sequences (Mb) </td><td>240 </td><td>122 </td><td>169 </td><td>168 </td><td>115 </td><td>114 </td><td>182 </td></tr><tr><td>Assembly coverage (%) </td><td>96 </td><td>— </td><td>65 </td><td>65 </td><td>75 </td><td>60 </td><td>— </td></tr><tr><td>Gaps Ratios (%) </td><td>0.05 </td><td>0.1 </td><td>1.6 </td><td>2.5 </td><td>0.3 </td><td>2.4 </td><td>10.5 </td></tr><tr><td>Longest scaffold size (Mb) </td><td>10.5 </td><td>0.7 </td><td>7.3 </td><td>4.2 </td><td>20.1 </td><td>15.9 </td><td>22.3 </td></tr><tr><td>N50 size of scaffold (Mb) </td><td>44.8 </td><td>0.1 </td><td>0.6 </td><td>0.3 </td><td>13.8 </td><td>5.2 </td><td>7.6 </td></tr><tr><td>N90 size of scaffold (kb) </td><td>8,527.5 </td><td>5.3 </td><td>29.4 </td><td>14.3 </td><td>2,924.3 </td><td>213.5 </td><td>40.7 </td></tr><tr><td>GC content in genome (%) </td><td>43.7 </td><td>42.9 </td><td>43.2 </td><td>43.1 </td><td>42.2 </td><td>41.9 </td><td>35.9 </td></tr><tr><td>Gene annotation </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Number of gene models </td><td>13,013 </td><td>12,481 </td><td>13,161 </td><td>13,323 </td><td>10,663 </td><td>10,245 </td><td>12,368 </td></tr><tr><td>BUSCO complete gene (ratio) </td><td>351 (81.8%) </td><td>382 (89.0%) </td><td>364 (84.8%) </td><td>360 (83.9%) </td><td>393 (91.6%) </td><td>388 (90.4%) </td><td>381 (88.8%) </td></tr><tr><td>Coding sequence size (Mb) </td><td>18.5 </td><td>15.5 </td><td>13.3 </td><td>13.5 </td><td>15.7 </td><td>15.2 </td><td>16.9 </td></tr><tr><td>Average CDS size (bp) </td><td>1,424 </td><td>1,242 </td><td>1,011 </td><td>1,013 </td><td>1,472 </td><td>1, 484 </td><td>1, 366 </td></tr><tr><td>Average exon number </td><td>6.6 </td><td>5.6 </td><td>4.4 </td><td>4.3 </td><td>6.7 </td><td>6.8 </td><td>6.0 </td></tr><tr><td>Average exon size (bp) </td><td>215 </td><td>222 </td><td>232 </td><td>234 </td><td>218 </td><td>218 </td><td>228 </td></tr><tr><td>GC content in coding region (%) </td><td>50.9 </td><td>50.1 </td><td>50.1 </td><td>50.1 </td><td>50.0 </td><td>50.0 </td><td>44.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The genome data of six other tapeworm species (<em>T. solium</em>, <em>T. saginata</em>, <em>T. asiatica</em>, <em>E. multilocularis</em>, <em>E. granulosus</em> and <em>H. microstoma</em>) were downloaded from WormBase and the NCBI database. The N50 and N90 sizes were calculated based on the assembled genome size.</p></div></div><div content-id="tblfn2"><div class="fn"><div class="label -label">a</div><p>The estimated genome sizes were reported in Wang <em>et al.</em> (2016); For the <em>E. multilocularis</em> and <em>E. granulosus</em>, the estimated genome sizes were used the Illumina reads by distribution of kmer frequency (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&amp;Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). We did not estimate the genome sizes of <em>T. solium</em> and <em>H. microstoma</em>, because of Illumina reads not be found.</p></div></div></div></div></div><div class="table-wrap"><div id="dsy020-T1" data-id="dsy020-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Summary of assembly and annotation of tapeworm genomes</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Assembly feature </th><th><em>T. multiceps</em> </th><th><em>T. solium</em> </th><th><em>T. saginata</em> </th><th><em>T. asiatica</em> </th><th><em>E. multilocularis</em> </th><th><em>E. granulosus</em> </th><th><em>H. microstoma</em> </th></tr></thead><tbody><tr><td>Estimated genome size (Mb) </td><td>250 </td><td>— </td><td>260<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>260<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>154 </td><td>195 </td><td>— </td></tr><tr><td>Assembled sequences (Mb) </td><td>240 </td><td>122 </td><td>169 </td><td>168 </td><td>115 </td><td>114 </td><td>182 </td></tr><tr><td>Assembly coverage (%) </td><td>96 </td><td>— </td><td>65 </td><td>65 </td><td>75 </td><td>60 </td><td>— </td></tr><tr><td>Gaps Ratios (%) </td><td>0.05 </td><td>0.1 </td><td>1.6 </td><td>2.5 </td><td>0.3 </td><td>2.4 </td><td>10.5 </td></tr><tr><td>Longest scaffold size (Mb) </td><td>10.5 </td><td>0.7 </td><td>7.3 </td><td>4.2 </td><td>20.1 </td><td>15.9 </td><td>22.3 </td></tr><tr><td>N50 size of scaffold (Mb) </td><td>44.8 </td><td>0.1 </td><td>0.6 </td><td>0.3 </td><td>13.8 </td><td>5.2 </td><td>7.6 </td></tr><tr><td>N90 size of scaffold (kb) </td><td>8,527.5 </td><td>5.3 </td><td>29.4 </td><td>14.3 </td><td>2,924.3 </td><td>213.5 </td><td>40.7 </td></tr><tr><td>GC content in genome (%) </td><td>43.7 </td><td>42.9 </td><td>43.2 </td><td>43.1 </td><td>42.2 </td><td>41.9 </td><td>35.9 </td></tr><tr><td>Gene annotation </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Number of gene models </td><td>13,013 </td><td>12,481 </td><td>13,161 </td><td>13,323 </td><td>10,663 </td><td>10,245 </td><td>12,368 </td></tr><tr><td>BUSCO complete gene (ratio) </td><td>351 (81.8%) </td><td>382 (89.0%) </td><td>364 (84.8%) </td><td>360 (83.9%) </td><td>393 (91.6%) </td><td>388 (90.4%) </td><td>381 (88.8%) </td></tr><tr><td>Coding sequence size (Mb) </td><td>18.5 </td><td>15.5 </td><td>13.3 </td><td>13.5 </td><td>15.7 </td><td>15.2 </td><td>16.9 </td></tr><tr><td>Average CDS size (bp) </td><td>1,424 </td><td>1,242 </td><td>1,011 </td><td>1,013 </td><td>1,472 </td><td>1, 484 </td><td>1, 366 </td></tr><tr><td>Average exon number </td><td>6.6 </td><td>5.6 </td><td>4.4 </td><td>4.3 </td><td>6.7 </td><td>6.8 </td><td>6.0 </td></tr><tr><td>Average exon size (bp) </td><td>215 </td><td>222 </td><td>232 </td><td>234 </td><td>218 </td><td>218 </td><td>228 </td></tr><tr><td>GC content in coding region (%) </td><td>50.9 </td><td>50.1 </td><td>50.1 </td><td>50.1 </td><td>50.0 </td><td>50.0 </td><td>44.4 </td></tr></tbody></table></div><div content-id="dsy020-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Assembly feature </th><th><em>T. multiceps</em> </th><th><em>T. solium</em> </th><th><em>T. saginata</em> </th><th><em>T. asiatica</em> </th><th><em>E. multilocularis</em> </th><th><em>E. granulosus</em> </th><th><em>H. microstoma</em> </th></tr></thead><tbody><tr><td>Estimated genome size (Mb) </td><td>250 </td><td>— </td><td>260<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>260<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>154 </td><td>195 </td><td>— </td></tr><tr><td>Assembled sequences (Mb) </td><td>240 </td><td>122 </td><td>169 </td><td>168 </td><td>115 </td><td>114 </td><td>182 </td></tr><tr><td>Assembly coverage (%) </td><td>96 </td><td>— </td><td>65 </td><td>65 </td><td>75 </td><td>60 </td><td>— </td></tr><tr><td>Gaps Ratios (%) </td><td>0.05 </td><td>0.1 </td><td>1.6 </td><td>2.5 </td><td>0.3 </td><td>2.4 </td><td>10.5 </td></tr><tr><td>Longest scaffold size (Mb) </td><td>10.5 </td><td>0.7 </td><td>7.3 </td><td>4.2 </td><td>20.1 </td><td>15.9 </td><td>22.3 </td></tr><tr><td>N50 size of scaffold (Mb) </td><td>44.8 </td><td>0.1 </td><td>0.6 </td><td>0.3 </td><td>13.8 </td><td>5.2 </td><td>7.6 </td></tr><tr><td>N90 size of scaffold (kb) </td><td>8,527.5 </td><td>5.3 </td><td>29.4 </td><td>14.3 </td><td>2,924.3 </td><td>213.5 </td><td>40.7 </td></tr><tr><td>GC content in genome (%) </td><td>43.7 </td><td>42.9 </td><td>43.2 </td><td>43.1 </td><td>42.2 </td><td>41.9 </td><td>35.9 </td></tr><tr><td>Gene annotation </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td><td> </td></tr><tr><td>Number of gene models </td><td>13,013 </td><td>12,481 </td><td>13,161 </td><td>13,323 </td><td>10,663 </td><td>10,245 </td><td>12,368 </td></tr><tr><td>BUSCO complete gene (ratio) </td><td>351 (81.8%) </td><td>382 (89.0%) </td><td>364 (84.8%) </td><td>360 (83.9%) </td><td>393 (91.6%) </td><td>388 (90.4%) </td><td>381 (88.8%) </td></tr><tr><td>Coding sequence size (Mb) </td><td>18.5 </td><td>15.5 </td><td>13.3 </td><td>13.5 </td><td>15.7 </td><td>15.2 </td><td>16.9 </td></tr><tr><td>Average CDS size (bp) </td><td>1,424 </td><td>1,242 </td><td>1,011 </td><td>1,013 </td><td>1,472 </td><td>1, 484 </td><td>1, 366 </td></tr><tr><td>Average exon number </td><td>6.6 </td><td>5.6 </td><td>4.4 </td><td>4.3 </td><td>6.7 </td><td>6.8 </td><td>6.0 </td></tr><tr><td>Average exon size (bp) </td><td>215 </td><td>222 </td><td>232 </td><td>234 </td><td>218 </td><td>218 </td><td>228 </td></tr><tr><td>GC content in coding region (%) </td><td>50.9 </td><td>50.1 </td><td>50.1 </td><td>50.1 </td><td>50.0 </td><td>50.0 </td><td>44.4 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The genome data of six other tapeworm species (<em>T. solium</em>, <em>T. saginata</em>, <em>T. asiatica</em>, <em>E. multilocularis</em>, <em>E. granulosus</em> and <em>H. microstoma</em>) were downloaded from WormBase and the NCBI database. The N50 and N90 sizes were calculated based on the assembled genome size.</p></div></div><div content-id="tblfn2"><div class="fn"><div class="label -label">a</div><p>The estimated genome sizes were reported in Wang <em>et al.</em> (2016); For the <em>E. multilocularis</em> and <em>E. granulosus</em>, the estimated genome sizes were used the Illumina reads by distribution of kmer frequency (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&amp;Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). We did not estimate the genome sizes of <em>T. solium</em> and <em>H. microstoma</em>, because of Illumina reads not be found.</p></div></div></div></div></p>                        <a id="122671197" scrollto-destination="122671197"></a>
<div id="dsy020-F1" data-id="dsy020-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/m_dsy020f1.png?Expires=2147483647&Signature=qSTnCltDA7gRr~ZPXh54wB4iYrYsgOgr9yuE8Qdqj96p3sG0wdVr56Mj9n6V~O6ZdDanXlXk33dVRee8b5ZLC4FIId8ZKLwpOI9Vbt6Q8uRdjLqTBCg0O92j~zv5qG5PFlWp1yb5-rh1I7Yj6h6ByGQvt1DBX6g9LozWblxhhqxEGdV35flAxEgFRPsh7ceKPQaL4UNY5A6Ii5RdUXa7Yq6f94ew6ijZomHc69611HilWuRUpXwK0EQbCXe4tY7N4pBO1aGdlsLCEdCDFdf-p2pWObvQMwC7d42rT4x0U96va5LLb~1Un~PPAT3nigp1O2ZcWTZEczczBYOyX3S6yQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The genome characteristics of T. multiceps. (a) Distribution of 19-mer frequency. Error corrected Illumina reads were used to calculate kmer frequency. (b) Hi-C produces a genome-wide contact matrix with 500 kb window between in seven-linkage group (LG). (c) A Venn diagram showing the unique and shared size." path-from-xml="dsy020f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671197/dsy020f1.tif" path-from-xml="dsy020f1.tif" target="_blank">View large</a><a section="122671197" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020f1.png?Expires=2147483647&Signature=zMo1j5VBeYEDV075-3-y1f1chyPKAemzG~s6CUX8~2tIzqf31l1dHwEDaM0J7R45MbzZi~spv1yiZ2cHSWO4M1ik-eND8pOZ4~jVoifEebGKg9O1xqlreVJ90i16nE2TrxUFCC~kgjIJG4olYBmcLk3sqnxAu9aPIliLdXxockj7esog6yz~b0FQWdAPloWzOHdW8UjnezwID4VUw9Sx2gU1xg7Vvauqh6Aa5WCzS-Gt4fYjhggMtuQA4pYIddGgZj4yKb6Sv2vhiBzwV2l9h~tLerhvD-IK2MRWlgtPdEPfKoCPZxlgfJqgL6vMM8AlYx7tPH~SirttYg48~-ZnjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671197&ar=5045548&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy020f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The genome characteristics of <em>T. multiceps.</em> (a) Distribution of 19-mer frequency. Error corrected Illumina reads were used to calculate kmer frequency. (b) Hi-C produces a genome-wide contact matrix with 500 kb window between in seven-linkage group (LG). (c) A Venn diagram showing the unique and shared size.</p></div></div><div content-id="dsy020-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/m_dsy020f1.png?Expires=2147483647&Signature=qSTnCltDA7gRr~ZPXh54wB4iYrYsgOgr9yuE8Qdqj96p3sG0wdVr56Mj9n6V~O6ZdDanXlXk33dVRee8b5ZLC4FIId8ZKLwpOI9Vbt6Q8uRdjLqTBCg0O92j~zv5qG5PFlWp1yb5-rh1I7Yj6h6ByGQvt1DBX6g9LozWblxhhqxEGdV35flAxEgFRPsh7ceKPQaL4UNY5A6Ii5RdUXa7Yq6f94ew6ijZomHc69611HilWuRUpXwK0EQbCXe4tY7N4pBO1aGdlsLCEdCDFdf-p2pWObvQMwC7d42rT4x0U96va5LLb~1Un~PPAT3nigp1O2ZcWTZEczczBYOyX3S6yQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The genome characteristics of T. multiceps. (a) Distribution of 19-mer frequency. Error corrected Illumina reads were used to calculate kmer frequency. (b) Hi-C produces a genome-wide contact matrix with 500 kb window between in seven-linkage group (LG). (c) A Venn diagram showing the unique and shared size." path-from-xml="dsy020f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671197/dsy020f1.tif" path-from-xml="dsy020f1.tif" target="_blank">View large</a><a section="122671197" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020f1.png?Expires=2147483647&Signature=zMo1j5VBeYEDV075-3-y1f1chyPKAemzG~s6CUX8~2tIzqf31l1dHwEDaM0J7R45MbzZi~spv1yiZ2cHSWO4M1ik-eND8pOZ4~jVoifEebGKg9O1xqlreVJ90i16nE2TrxUFCC~kgjIJG4olYBmcLk3sqnxAu9aPIliLdXxockj7esog6yz~b0FQWdAPloWzOHdW8UjnezwID4VUw9Sx2gU1xg7Vvauqh6Aa5WCzS-Gt4fYjhggMtuQA4pYIddGgZj4yKb6Sv2vhiBzwV2l9h~tLerhvD-IK2MRWlgtPdEPfKoCPZxlgfJqgL6vMM8AlYx7tPH~SirttYg48~-ZnjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671197&ar=5045548&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy020f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The genome characteristics of <em>T. multiceps.</em> (a) Distribution of 19-mer frequency. Error corrected Illumina reads were used to calculate kmer frequency. (b) Hi-C produces a genome-wide contact matrix with 500 kb window between in seven-linkage group (LG). (c) A Venn diagram showing the unique and shared size.</p></div></div><p>The accuracy and completeness of the assembly were evaluated by mapping the Illumina shotgun reads to the assembled reference genome. Significantly, on average 99.8 and 98.2% of the genome-derived and transcriptome-derived reads could be aligned to the reference genome (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s1.xlsx?Expires=2147483647&amp;Signature=KLjFb7BUTE5n1H-iUmV5Hf0xggueqoIj1voQ4a6nNQD9ndEqpMjOZSH4WnDukwAoG19jetEaISSjtoq~lnLi9RO7LQrrmFigdeNSkUKRRO-VCQeay790oTo5hhYfJ5Ctuqi2Qp5CzjgS~zOElwIJOlpYq0QEGV0S3rm5lJPWfO5tTRhrEJ3T8mzk6Nz~m3qUwhKo2Oz9pl0sttAIwkJnN6njYFI45oXBc1QQfSt0b455qGWZcT4vy6-XR~5jTquh7PJjsXwjPWlH~u~u59~qk2C8qeG9AFhzT-VI3NWuYkmww1qcxnpfjSE3z5MdG4a2El1znx6HU1OjJzmpYQEjRw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), respectively, indicating that the majority of the genomic sequence is contained in the current assembly. Furthermore, ∼98% of Illumina assembled contigs were mapped to ∼50% of the reference genome with 99.2% identity, emphasizing the considerable advantage of using the long-reads assembly (<a reveal-id="dsy020-F1" data-open="dsy020-F1" class="link link-reveal link-table xref-fig">Fig. 1c</a>). Moreover, a total of 13 telomeres were identified located on one end of the contigs with an average size of 10 kb, compared with 12 identified telomeres in <em>E. multilocularis</em> and only 1–3 telomeres in other tapeworm genomes, further supporting the high physical coverage of the current assembly (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&amp;Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>).</p><p>The protein-coding genes were predicted on the reference genome by evidence modeler,<a reveal-id="dsy020-B23" data-open="dsy020-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> integrating evidence from <em>de novo</em> prediction, transcriptome and homology data. In total, 13,013 gene models were predicted as the reference gene set, with coding regions spanning ∼18.5 Mb (7.7%) of the genome (<a reveal-id="dsy020-T1" data-open="dsy020-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The distribution of exon number and CDS length in <em>T. multiceps</em> is similar to the closely related <em>E. multilocularis</em> and <em>E. granulosus</em> (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). 81.8% of eukaryote core genes from OrthoDB (<a class="link link-uri" href="http://www.orthodb.org" target="_blank">http://www.orthodb.org</a>) were identified as complete in the reference gene set by BUSCO<a reveal-id="dsy020-B38" data-open="dsy020-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> (<a reveal-id="dsy020-T1" data-open="dsy020-T1" class="link link-reveal link-table xref-fig">Table 1</a>). For functional annotation, a total of 11,914 (92%) coding proteins were annotated by functional databases (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). On the other hand, a total of 521 non-coding RNA genes (379 tRNA and 142 rRNA) were identified in the <em>T. multiceps</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). Together, these demonstrate that the accuracy of gene annotation we achieved is comparable to or better than the other published tapeworm genomes.<a reveal-id="dsy020-B16" data-open="dsy020-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsy020-B17" data-open="dsy020-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a></p>                            <h3 scrollto-destination=122671200 id="122671200" class="section-title" >3.2. Recent expansion of transposable elements in <em>Taenia</em> species</h3>
<p>The high assembly coverage we obtained enabled a comprehensive analysis of TEs which have multiple roles in driving genome evolution in eukaryotes.<a reveal-id="dsy020-B39" data-open="dsy020-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> In total, we identified 28.9 Mb (12.0%) TE sequences in the assembled <em>T. multiceps</em> genome (<a reveal-id="dsy020-F2" data-open="dsy020-F2" class="link link-reveal link-table xref-fig">Fig. 2a</a> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). Notably, the most abundant LTR retrotransposons present in <em>T. multiceps</em> are <em>Gypsy</em> elements, accounting for 49.7% of the LTR elements, followed by <em>Copia</em> (12.2%) and <em>ERV1</em> (3.7%) (<a reveal-id="dsy020-F2" data-open="dsy020-F2" class="link link-reveal link-table xref-fig">Fig. 2b</a>). Furthermore, we used the same strategy to annotate and compare TEs in the published <em>T. saginata</em>, <em>T. asiatica</em>, <em>E. multilocularis</em> and <em>E. granulosus</em> genomes. The total TE content in the <em>Taenia</em> species is almost twice that found in the <em>Echinococcus</em> species, but the overall TE content and classification are similar within the former species (<a reveal-id="dsy020-F2" data-open="dsy020-F2" class="link link-reveal link-table xref-fig">Fig. 2a</a> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). </p>                        <a id="122671202" scrollto-destination="122671202"></a>
<div id="dsy020-F2" data-id="dsy020-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/m_dsy020f2.png?Expires=2147483647&Signature=w1YzyCFpAcxQHEl~4TfjBSG7ER9wmAPK3KvJcoK8zTxT0d2JCjg5pfJy3opYhXGonsHDRyqLSLo8LFt04UYs1v0hpmhFTasLgwIxDpzbKbKOJYaARso8guw38OccC-KaZNuNuYnTo7ta8VasGKH4k5z2UnNIMasCJa-8xD-t4HxLY1y7lUmMfMNa-brc4PLGb5l8MaD0yzHPz570qa1xSkWB~HN7nsq3Y7t4D-fO8PLiu7WkoH9KBsDArK0CMeICWhnW9F4MhoaZd17iUWAgjv2XaQFluwzPo59k~SoeS5Bvvy3T0KQU5qR814PpMnsbg1j59J7RsgMqSVfS7jfK0A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Repetitive sequences content and TE divergence rate in tapeworm genomes. (a) Classification and contents of repetitive sequences in T. multiceps compared with T. saginata, T. asiatica, T. solium, E. granulosus and E. multilocularis. (b) The ratio of top 10 abundant TEs families in T. multiceps genome. (c) Distribution of TE divergence rate in tapeworm genomes." path-from-xml="dsy020f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671202/dsy020f2.tif" path-from-xml="dsy020f2.tif" target="_blank">View large</a><a section="122671202" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020f2.png?Expires=2147483647&Signature=O8DN4wtxi1WFIE4wxSkkY5w~1GJ-tz~pqf23hLA0EE~aoX5dnHsUazsTvmNh-HwBn6ASfYwkYMeWLyUBOxLpjCo6U7bzHF3FNc-NHiZiFRI8~gYExyYfXHNvkmRCrN57ZOOdFvRSGbw2dxjHu8g4PQb~9NJ1V0bokHPAab0fcVi2JGzjfLkt1Q1p8Pvs9ryyX4zVQBswNVRaPlROWgvzuM0wk2Fgiw4lQa4~lZ97fRRHvSXvtR7mg8yt11H8t3fJgKXOgOULeuUZCIDJwuC8hUkKToMcK~SmaeJ0ynvS0q7JS1-fvrNYHo28WY8f~lUVKZAvbE4pLQ5vb8ZaF8~Gjw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671202&ar=5045548&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy020f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Repetitive sequences content and TE divergence rate in tapeworm genomes. (a) Classification and contents of repetitive sequences in <em>T. multiceps</em> compared with <em>T. saginata</em>, <em>T. asiatica</em>, <em>T. solium</em>, <em>E. granulosus</em> and <em>E. multilocularis</em>. (b) The ratio of top 10 abundant TEs families in <em>T. multiceps</em> genome. (c) Distribution of TE divergence rate in tapeworm genomes.</p></div></div><div content-id="dsy020-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/m_dsy020f2.png?Expires=2147483647&Signature=w1YzyCFpAcxQHEl~4TfjBSG7ER9wmAPK3KvJcoK8zTxT0d2JCjg5pfJy3opYhXGonsHDRyqLSLo8LFt04UYs1v0hpmhFTasLgwIxDpzbKbKOJYaARso8guw38OccC-KaZNuNuYnTo7ta8VasGKH4k5z2UnNIMasCJa-8xD-t4HxLY1y7lUmMfMNa-brc4PLGb5l8MaD0yzHPz570qa1xSkWB~HN7nsq3Y7t4D-fO8PLiu7WkoH9KBsDArK0CMeICWhnW9F4MhoaZd17iUWAgjv2XaQFluwzPo59k~SoeS5Bvvy3T0KQU5qR814PpMnsbg1j59J7RsgMqSVfS7jfK0A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Repetitive sequences content and TE divergence rate in tapeworm genomes. (a) Classification and contents of repetitive sequences in T. multiceps compared with T. saginata, T. asiatica, T. solium, E. granulosus and E. multilocularis. (b) The ratio of top 10 abundant TEs families in T. multiceps genome. (c) Distribution of TE divergence rate in tapeworm genomes." path-from-xml="dsy020f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671202/dsy020f2.tif" path-from-xml="dsy020f2.tif" target="_blank">View large</a><a section="122671202" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020f2.png?Expires=2147483647&Signature=O8DN4wtxi1WFIE4wxSkkY5w~1GJ-tz~pqf23hLA0EE~aoX5dnHsUazsTvmNh-HwBn6ASfYwkYMeWLyUBOxLpjCo6U7bzHF3FNc-NHiZiFRI8~gYExyYfXHNvkmRCrN57ZOOdFvRSGbw2dxjHu8g4PQb~9NJ1V0bokHPAab0fcVi2JGzjfLkt1Q1p8Pvs9ryyX4zVQBswNVRaPlROWgvzuM0wk2Fgiw4lQa4~lZ97fRRHvSXvtR7mg8yt11H8t3fJgKXOgOULeuUZCIDJwuC8hUkKToMcK~SmaeJ0ynvS0q7JS1-fvrNYHo28WY8f~lUVKZAvbE4pLQ5vb8ZaF8~Gjw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671202&ar=5045548&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy020f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Repetitive sequences content and TE divergence rate in tapeworm genomes. (a) Classification and contents of repetitive sequences in <em>T. multiceps</em> compared with <em>T. saginata</em>, <em>T. asiatica</em>, <em>T. solium</em>, <em>E. granulosus</em> and <em>E. multilocularis</em>. (b) The ratio of top 10 abundant TEs families in <em>T. multiceps</em> genome. (c) Distribution of TE divergence rate in tapeworm genomes.</p></div></div><p>Next, we analysed the divergence rate of TEs among the available sequenced tapeworm genomes. The divergence rate was calculated by comparing all TE sequences in each subfamily to its corresponding consensus sequence. The results showed that each <em>Taenia</em> species had a peak at ∼20% divergence rate, containing approximately half of TE copies. In contrast, the <em>Echinococcus</em> species had a peak at ∼36% divergence rate (<a reveal-id="dsy020-F2" data-open="dsy020-F2" class="link link-reveal link-table xref-fig">Fig. 2c</a> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>).</p><p>Tandem repeat is another kind of repetitive sequence abundant in the eukaryote genome, which plays an important role in maintaining the genome structure. For the sequenced tapeworms, most of the tandem repeats were not correctly assembled and thus missing from the current reference sequences based on Illumina sequencing. In contrast, in the reference assembly of <em>T. multiceps</em>, we successfully annotated a total of 20.5 Mb (8.5%) tandem repeat sequences (<a reveal-id="dsy020-F2" data-open="dsy020-F2" class="link link-reveal link-table xref-fig">Fig. 2a</a>), ∼92% of which locate in the fragmentally assembled gene-free sequences. These assembled tandem repeat sequences provide a valuable resource for studying the structure of centromere, telomere, as well as heterochromatic regions.</p>                            <h3 scrollto-destination=122671205 id="122671205" class="section-title" >3.3. Gene expansions mostly derived from small-scale gene duplications</h3>
<p>To gain insights into an evolutionary perspective for <em>T. multiceps</em>, we built a phylogenetic tree based on 271 high-confidence single copy orthologs of 14 helminth worm species by PhyML<a reveal-id="dsy020-B40" data-open="dsy020-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> and estimated divergence time using mcmctree<a reveal-id="dsy020-B41" data-open="dsy020-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> (<a reveal-id="dsy020-F3" data-open="dsy020-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a>). The results indicate that <em>T. multiceps</em> diverged from <em>T. saginata</em> and <em>T. asiatica</em> 4–11 million years ago (Mya) (95% CI, and from <em>T. solium</em> 13–20 Mya. Analysis of gene families in the high-confidence gene set and 14 sequenced helminth worm genomes identified 122 <em>T. multiceps</em>-specific gene families (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). </p>                        <a id="122671207" scrollto-destination="122671207"></a>
<div id="dsy020-F3" data-id="dsy020-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/m_dsy020f3.png?Expires=2147483647&Signature=OCOcRV5pWCiq8t4CrUwOMU3uD-KTH6UTyFn6E0~ESLZWrt4yXC2WDJG5xugn5B4bQxvDgnJQpuguAY5585hpOo6oIJHBQLFj3~bgcX6JJ~4TTT6lI1OA5m19cKni8GlQyMeZMBK7NzoDV4oThXmF49tKhcQGWTixwFlN2Hhz-wbXpkwfTdpRjveXM2d76QIZRjqazDRD2wSxsfm-bTvig8tOxwuhcl9X7BcWM6JJhehwpUWS8ahpxIXjw2YlLDt5PnLbBeGFI~YtnzSmNGKh0uaCKiTn28sEvdF-AeSdFxNtEGFF7R6iqNHsyDZS5RmccoQ0jspmixp3B460ogmrQg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evolution of the T. multiceps genome. (a) Dated tree for 14 species. The age of each node is indicated by 95% CI. (b) Distribution of different types of gene duplication in tapeworm genomes. WGD means whole genome duplication. (c) Distribution of synonymous mutation rate (Ks) values for paralogous gene pairs in each tapeworm genome. Ks values between T. multiceps and E. multilocularis are calculated using the syntenic ortholog gene pairs." path-from-xml="dsy020f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671207/dsy020f3.tif" path-from-xml="dsy020f3.tif" target="_blank">View large</a><a section="122671207" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020f3.png?Expires=2147483647&Signature=J7HL8mOtY7kcOYD~e54jMKZ6eyZL4KVkKAcJ2iUIwbIhDEJ38Dq6ItlTbwHgN10uz2IdiH8aJkIBVVDT0jIZgzqAs7h1V-URXy6sOc9HMFtx3lLrX4XdvMXMOW~0ULLayValakU5FJlWnU9Mxs9mEO8Fj~cFRoZv-bYhD-9NPw8ZO28Ksau2JgjfariyFEj2hMwfRZm5Pht6VIpqefKYM4yS6PkD-Me7ep3YSAfoUXdknNNTHG~FIVn9hvIQUmfBsBpXXPuL0A54UqRgwCeGgKTP-EomuBa5Cv3iRdbDXdEjF5ZTwE7OZpk8mmzoiDceYYk4UHfvmGGRKnt6Jy3jOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671207&ar=5045548&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy020f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evolution of the <em>T. multiceps</em> genome. (a) Dated tree for 14 species. The age of each node is indicated by 95% CI. (b) Distribution of different types of gene duplication in tapeworm genomes. WGD means whole genome duplication. (c) Distribution of synonymous mutation rate (<em>K</em><sub>s</sub>) values for paralogous gene pairs in each tapeworm genome. <em>K</em><sub>s</sub> values between <em>T. multiceps</em> and <em>E. multilocularis</em> are calculated using the syntenic ortholog gene pairs.</p></div></div><div content-id="dsy020-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/m_dsy020f3.png?Expires=2147483647&Signature=OCOcRV5pWCiq8t4CrUwOMU3uD-KTH6UTyFn6E0~ESLZWrt4yXC2WDJG5xugn5B4bQxvDgnJQpuguAY5585hpOo6oIJHBQLFj3~bgcX6JJ~4TTT6lI1OA5m19cKni8GlQyMeZMBK7NzoDV4oThXmF49tKhcQGWTixwFlN2Hhz-wbXpkwfTdpRjveXM2d76QIZRjqazDRD2wSxsfm-bTvig8tOxwuhcl9X7BcWM6JJhehwpUWS8ahpxIXjw2YlLDt5PnLbBeGFI~YtnzSmNGKh0uaCKiTn28sEvdF-AeSdFxNtEGFF7R6iqNHsyDZS5RmccoQ0jspmixp3B460ogmrQg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Evolution of the T. multiceps genome. (a) Dated tree for 14 species. The age of each node is indicated by 95% CI. (b) Distribution of different types of gene duplication in tapeworm genomes. WGD means whole genome duplication. (c) Distribution of synonymous mutation rate (Ks) values for paralogous gene pairs in each tapeworm genome. Ks values between T. multiceps and E. multilocularis are calculated using the syntenic ortholog gene pairs." path-from-xml="dsy020f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671207/dsy020f3.tif" path-from-xml="dsy020f3.tif" target="_blank">View large</a><a section="122671207" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020f3.png?Expires=2147483647&Signature=J7HL8mOtY7kcOYD~e54jMKZ6eyZL4KVkKAcJ2iUIwbIhDEJ38Dq6ItlTbwHgN10uz2IdiH8aJkIBVVDT0jIZgzqAs7h1V-URXy6sOc9HMFtx3lLrX4XdvMXMOW~0ULLayValakU5FJlWnU9Mxs9mEO8Fj~cFRoZv-bYhD-9NPw8ZO28Ksau2JgjfariyFEj2hMwfRZm5Pht6VIpqefKYM4yS6PkD-Me7ep3YSAfoUXdknNNTHG~FIVn9hvIQUmfBsBpXXPuL0A54UqRgwCeGgKTP-EomuBa5Cv3iRdbDXdEjF5ZTwE7OZpk8mmzoiDceYYk4UHfvmGGRKnt6Jy3jOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671207&ar=5045548&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy020f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Evolution of the <em>T. multiceps</em> genome. (a) Dated tree for 14 species. The age of each node is indicated by 95% CI. (b) Distribution of different types of gene duplication in tapeworm genomes. WGD means whole genome duplication. (c) Distribution of synonymous mutation rate (<em>K</em><sub>s</sub>) values for paralogous gene pairs in each tapeworm genome. <em>K</em><sub>s</sub> values between <em>T. multiceps</em> and <em>E. multilocularis</em> are calculated using the syntenic ortholog gene pairs.</p></div></div><p>To investigate the evolutionary dynamics of genome structure, we performed comparative genomic analysis among <em>T. multiceps</em>, <em>T. saginata</em> and <em>E. multilocularis</em>, using proteins as markers to identify syntenic genes by MCScan pipeline with parameter –w 10. In the <em>T. multiceps</em> genome, 42.5% genes were identified synteny relationships with <em>T. saginata</em>; whereas 5.15% genes were identified synteny relationships with <em>E. multilocularis</em>. Scrutiny of the local scale synteny, we found that the ratio of syntenic genes between <em>T. multiceps</em> and <em>T. saginata</em> was ∼71, versus 68% between <em>T. multiceps</em> and <em>E. multilocularis</em>, which is reasonable considering their phylogenetic distance (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>).</p><p>In <em>T. multiceps</em>, there were 6,300 dispersed duplicated genes (48.4% of total genes), which were much more than proximal duplicated genes 1,446 (11.1%) and tandem duplicated genes 1,128 (8.7%). The proportion of dispersed duplicated genes in the <em>T. multiceps</em> was similar to that in the <em>T. saginata</em> and <em>T. asiatica</em> (45.8 and 48.1%, respectively), but was significantly higher than that in the <em>E. multilocularis</em> and <em>E. granulosus</em> (32.2 and 30.9%, respectively; <em>P</em> &lt; 0.001; <a reveal-id="dsy020-F3" data-open="dsy020-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>), consistent with previously reports for the <em>T. saginata</em> and <em>T. asiatica</em> genomes.<a reveal-id="dsy020-B17" data-open="dsy020-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> These results suggest that gene expansions in <em>Taenia</em> species are mostly derived from small-scale gene duplications, instead of whole genome duplication.</p><p>Using the data obtained for the duplicated genes, we analysed the synonymous mutation rate (<em>K</em><sub>s</sub>) of these paralogous gene pairs. No obvious <em>K</em><sub>s</sub> peaks were observed for any of the five tapeworms, reinforcing our inference of no recent whole genome duplication having occurred. The duplicated genes were likely generated gradually over time with relatively more recent duplication events in the <em>Taenia spp.</em> compared with <em>Echinococcus</em> (<a reveal-id="dsy020-F3" data-open="dsy020-F3" class="link link-reveal link-table xref-fig">Fig. 3c</a>). In total, 3,697 genes with <em>K</em><sub>s</sub> &lt; 0.3 were used to analyse the gene function. Results showed that the Hsp70, fibronectin type III, transcription factor families (T-box, homeobox and zinc finger) were significantly enriched in these duplicated genes of <em>T. multiceps</em> (<em>P</em> &lt; 0.001), and the genes involved in the MAPK signalling pathway (map04010) were significantly enriched (<em>P</em> = 0.0071). These observations indicate that genes underlying environmental adaptability and dosage effect tend to be conserved in the <em>T. multiceps</em> genome.</p>                            <h3 scrollto-destination=122671211 id="122671211" class="section-title" >3.4. PSGs imply extended evolutionary pressures in the <em>Taenia</em> lineage</h3>
<p>Positive selection provides evolutionary innovation during adaptation to a new environment. We evaluated the role of positive selection in the evolution of <em>T. multiceps</em> by branch-site model analysis in PAML<a reveal-id="dsy020-B33" data-open="dsy020-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> based on orthologous genes from the Taeniidae tapeworms, and identified 204 PSGs (LRT, false discovery rate &lt; 0.05) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_information.doc?Expires=2147483647&Signature=jIuQ--s09O7g7OkKSDE1O6ogPCl1b6MtP4mi6d7RJAntqreB1qTWQWUlm25XduzK8XPeMIpI9slbCO~Tt7hW0lLxrorAPEV1MyNrh2LKa~6MLocgWnyoz3XVKDYw9MtexuAaehtYtAAzFYyrAiUuvb7XQJxXgL66apVa9cQMe4WnTpNT33vcevI-vStbA3n26CMDRoNH6RGZU6VK~e0xOUDSSEYrA94PrX79Oxhv3Z5LjCEHAfkmsHDpHhEGutz2Go~EafD5a17R3mmRMksViphdFI6In0FGjuhLyKdRIvFVPi9qk93xtho9wtmqK6K2TV4rI8xfuDq5HpsOVijGMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S1). In fact, we found PSGs of essential genes in many fundamental cellular processes, particularly those containing molecular functions of the DNA replication and repairing-related genes, transcription regulation, translation, and protein kinase (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_information.doc?Expires=2147483647&Signature=jIuQ--s09O7g7OkKSDE1O6ogPCl1b6MtP4mi6d7RJAntqreB1qTWQWUlm25XduzK8XPeMIpI9slbCO~Tt7hW0lLxrorAPEV1MyNrh2LKa~6MLocgWnyoz3XVKDYw9MtexuAaehtYtAAzFYyrAiUuvb7XQJxXgL66apVa9cQMe4WnTpNT33vcevI-vStbA3n26CMDRoNH6RGZU6VK~e0xOUDSSEYrA94PrX79Oxhv3Z5LjCEHAfkmsHDpHhEGutz2Go~EafD5a17R3mmRMksViphdFI6In0FGjuhLyKdRIvFVPi9qk93xtho9wtmqK6K2TV4rI8xfuDq5HpsOVijGMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S1). <em>T. multiceps</em> has a life cycle involving a tissue tropism in the brain of ungulates, which, in terms of nutrient availability, represents a different environment compared with most of the other <em>Taenia</em> tapeworms. Strong positive selection signals were observed in energy-metabolism related genes, such as genes involved in carbohydrate metabolism and uptake, including Glucosidase II beta subunit, glucan (1, 4 alpha), branching enzyme 1 and nicotinamide adenine dinucleotid ubiquinone oxidoreductase sgdh subunit, guanosine diphosphate fucose transporter. Notably, homologous genes probably involved in the speciation of <em>T. asiatica</em><a reveal-id="dsy020-B17" data-open="dsy020-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> were also included in the PSGs of <em>T. multiceps</em> (e.g. carbonic anhydrase, amiloride sensitive cation channel 4A, dynein light chain), suggesting the frequent involvement of these genes in adaptation to a new host or tissue environment during tapeworm evolution.</p>                            <h3 scrollto-destination=122671213 id="122671213" class="section-title" >3.5. Expansion and up-regulation of the T-box 6 genes produce more proglottids in strobilization</h3>
<p>Both asexual and sexual reproduction occurs in <em>T. multiceps</em>. Strobilization is a process of asexual reproduction whereby the adult tapeworm buds off new segments from the undifferentiated and germinative neck region, forming a series of proglottids which when mature give rise sexually to ovoid eggs.<a reveal-id="dsy020-B42" data-open="dsy020-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> The number of proglottids varies among the different tapeworms, with hundreds of segments produced by <em>Taenia</em> species but with only three to five being produced by in the <em>Echinococcus</em> species.<a reveal-id="dsy020-B10" data-open="dsy020-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a></p><p>The T-box family of transcription factors shows broad participation in the development of all metazoans; the subfamily T-box 6 (Tbx6), especially plays a central role in somite polarization in the mouse.<a reveal-id="dsy020-B43" data-open="dsy020-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> In this study, the number of T-box transcription factors was shown to be significantly expanded in the Taeniidae, representing on average 31 genes in the 4 <em>Taenia</em> species and 5 genes in the 2 <em>Echinococcus</em> species (<em>P</em> = 1.02 e<sup>−4</sup>) (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S11</a></span>). Specifically, there were 31, 23, 20 and 12 Tbx6 genes in <em>T. multiceps</em>, <em>T. asiatica</em>, <em>T. saginata</em> and <em>T. solium</em>, respectively, significantly more than in <em>E. granulosus</em> and in <em>E. multilocularis</em>, (each having only one Tbx6 gene (<em>P</em> &lt; 0.001). Based on a phylogenetic analysis of the Tbx6 genes in the Taeniidae, we found that the genes have been expanded in the ancestors of the <em>Taenia</em> species after their divergence from <em>Echinococcus</em>. Furthermore, the sequences of the Tbx6 genes have been divergent along the <em>T. multiceps</em> lineage which contained different motifs on each clade (<a reveal-id="dsy020-F4" data-open="dsy020-F4" class="link link-reveal link-table xref-fig">Fig. 4a and b</a>). These results imply that the Tbx6 genes might gain rapid functional divergence after gene duplication. On the other hand, to investigate the gene expression pattern of Tbx6, we analysed genes expressed with a cut off fragments per kilobase per million (FPKM) &gt; one in the four samples of three different <em>T. multiceps</em>, life cycle stages (one from oncospheres, two from the coenurus and one from the adult). It was surprising that 24 of 31 Tbx6 genes were expressed in the adult stage, whereas no or very limited expression was apparent in the other stages (<a reveal-id="dsy020-F4" data-open="dsy020-F4" class="link link-reveal link-table xref-fig">Fig. 4c</a>). Overall, the different number of proglottids produced in the <em>Taenia</em> species compared with the <em>Echinococcus</em> species may be as a result of the expansion of the Tbx6 genes in the former group which may play an important role in the strobilization process in adult tapeworms. </p>                        <a id="122671216" scrollto-destination="122671216"></a>
<div id="dsy020-F4" data-id="dsy020-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/m_dsy020f4.png?Expires=2147483647&Signature=RftmmukUIRMa4pOnhdOQYi9s6PxhsxLpkjxve5z9ZsgTSSoL8TJJbgJ~oW6FO-vqoOEuoiaCaLncniI4rOFxe-Zr5c7V5kVYRacj2n6vKtGJPc15aaDJ8rf8ENu1VFgI2shH-thfyExBjj8-oEdyKGpqCsVbPpUiE5KbLdHudmF-O3Z-~l4aHuPlzX6gMx4kSZBs0-JKAN2CY4LyvRF0c91czdMkQ2LMNmIKZJspDAixGfb4sl~xRkbbjID039jbZ~ata3auVzu7fmW1KqsSOYdo2qt2Fc5~vkVqQyQmxCwPa-OuKKDNOEKGCzp5XOZln5uPw2DzPzcg-NVtsNJd1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expansion and divergence of the Tbx6 subfamily in T. multiceps. (a) ML Phylogenetic tree of Tbx6 genes in T. multiceps. Two Tbx6 genes of Echinococcus were used as outgroup. (b) Motifs of Tbx6 genes in T. multiceps. Each colour of rectangle represents a motif. (c) Expression patterns of Tbx6 genes in stages (Onc, Pro, Cyst and Adu) of T. multiceps." path-from-xml="dsy020f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671216/dsy020f4.tif" path-from-xml="dsy020f4.tif" target="_blank">View large</a><a section="122671216" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020f4.png?Expires=2147483647&Signature=WCUOAB9qvALbi-g6zXC19JepWPUq2qNMK9s89-2vNzhCPU6gEfgaTxn780X7MXM17cQsoUDsCSvzcexJ~xTm9kdfvI60OlR6K4ioaoS7Whz~dEkvtoGZp~4mPkKEtufwMktJ4XIM9QaCDLuYtLC6LJ6vU~baJXGJppvbaJPcnsHpo0DucK50GqxTbMqA4y3ztPQWk1yVDrVels2dKzI177I-ew0i~LnnYd~g9YpMwbYB70tibForrehLHBjacvYYKvbMB76pxJWT-pkHsnrMONOVpx9Cp0eWxbyxgq2UkR-YhNFOHVbidpGXv-st4LcyVD1cYL~4Rp9uTE3PlSxdkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671216&ar=5045548&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy020f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expansion and divergence of the Tbx6 subfamily in <em>T. multiceps.</em> (a) ML Phylogenetic tree of Tbx6 genes in <em>T. multiceps</em>. Two Tbx6 genes of <em>Echinococcus</em> were used as outgroup. (b) Motifs of Tbx6 genes in <em>T. multiceps</em>. Each colour of rectangle represents a motif. (c) Expression patterns of Tbx6 genes in stages (Onc, Pro, Cyst and Adu) of <em>T. multiceps</em>.</p></div></div><div content-id="dsy020-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/m_dsy020f4.png?Expires=2147483647&Signature=RftmmukUIRMa4pOnhdOQYi9s6PxhsxLpkjxve5z9ZsgTSSoL8TJJbgJ~oW6FO-vqoOEuoiaCaLncniI4rOFxe-Zr5c7V5kVYRacj2n6vKtGJPc15aaDJ8rf8ENu1VFgI2shH-thfyExBjj8-oEdyKGpqCsVbPpUiE5KbLdHudmF-O3Z-~l4aHuPlzX6gMx4kSZBs0-JKAN2CY4LyvRF0c91czdMkQ2LMNmIKZJspDAixGfb4sl~xRkbbjID039jbZ~ata3auVzu7fmW1KqsSOYdo2qt2Fc5~vkVqQyQmxCwPa-OuKKDNOEKGCzp5XOZln5uPw2DzPzcg-NVtsNJd1A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expansion and divergence of the Tbx6 subfamily in T. multiceps. (a) ML Phylogenetic tree of Tbx6 genes in T. multiceps. Two Tbx6 genes of Echinococcus were used as outgroup. (b) Motifs of Tbx6 genes in T. multiceps. Each colour of rectangle represents a motif. (c) Expression patterns of Tbx6 genes in stages (Onc, Pro, Cyst and Adu) of T. multiceps." path-from-xml="dsy020f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671216/dsy020f4.tif" path-from-xml="dsy020f4.tif" target="_blank">View large</a><a section="122671216" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020f4.png?Expires=2147483647&Signature=WCUOAB9qvALbi-g6zXC19JepWPUq2qNMK9s89-2vNzhCPU6gEfgaTxn780X7MXM17cQsoUDsCSvzcexJ~xTm9kdfvI60OlR6K4ioaoS7Whz~dEkvtoGZp~4mPkKEtufwMktJ4XIM9QaCDLuYtLC6LJ6vU~baJXGJppvbaJPcnsHpo0DucK50GqxTbMqA4y3ztPQWk1yVDrVels2dKzI177I-ew0i~LnnYd~g9YpMwbYB70tibForrehLHBjacvYYKvbMB76pxJWT-pkHsnrMONOVpx9Cp0eWxbyxgq2UkR-YhNFOHVbidpGXv-st4LcyVD1cYL~4Rp9uTE3PlSxdkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671216&ar=5045548&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy020f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expansion and divergence of the Tbx6 subfamily in <em>T. multiceps.</em> (a) ML Phylogenetic tree of Tbx6 genes in <em>T. multiceps</em>. Two Tbx6 genes of <em>Echinococcus</em> were used as outgroup. (b) Motifs of Tbx6 genes in <em>T. multiceps</em>. Each colour of rectangle represents a motif. (c) Expression patterns of Tbx6 genes in stages (Onc, Pro, Cyst and Adu) of <em>T. multiceps</em>.</p></div></div>                            <h3 scrollto-destination=122671217 id="122671217" class="section-title" >3.6. Coenurus occupy nutrient and release toxicity to the host CNS</h3>
<p>Coenurosis, as a parasitic zoonosis, could cause potential health threat to the human beings and serious economic losses in sheep farms.<a reveal-id="dsy020-B7" data-open="dsy020-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsy020-B8" data-open="dsy020-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> It is a fatal CNS disease to the intermediate host sheep. The coenurus cyst contains hundreds of protoscolices and can grow to the size of a hen’s egg, caused coenurosis by pressing the brain to induce neural lesion (<a reveal-id="dsy020-F5" data-open="dsy020-F5" class="link link-reveal link-table xref-fig">Fig. 5a</a>). Moreover, the nervous system toxicity related with wastes from metabolic process and neurodegenerative disorder protein of parasite (<a reveal-id="dsy020-F5" data-open="dsy020-F5" class="link link-reveal link-table xref-fig">Fig. 5b</a>), may also occupy an important position in the causation of sheep CNS disease. </p>                        <a id="122671219" scrollto-destination="122671219"></a>
<div id="dsy020-F5" data-id="dsy020-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/m_dsy020f5.png?Expires=2147483647&Signature=a92qRuc3WyIXJ2tVYDad8mtHMvOZFQuTjRUF5A9eISVoVsSKZtcUbpw9YnkUV4P4yu74ORKDKZ3KTgKdYM5sVo5Th4AjTNCNP2INfYKQo9olZCe-kT9gzAe9fcqp7LANiXGJI~fZyyFXm3Wr2Yzpyec4Z58rfnhmAH19sL~WKp9EV1fdBmz35RHGJwrLHbFIgA8OEGYgytsgS89otcqMAjSXyuZ-yxo0378s8fuD0FqgMOfpUHW~wnfSa~cW4xsvVr~TD~ju594OZyfqnSznNhRp5624GWfU6Artjey8wu8ze78CuIf0GcuD5V6V6wHpQrC8lz1TXx6fUQeCnQ20SA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Effect of the coenurus on the sheep CNS. (a) The diagrammatic map of T. multiceps life cycle. The enlarged photo is a growing coenurus cyst with clusters of protoscoleces. (b) Possible mechanisms of the effects of the coenurus on the intermediate host brain. Nutrients and waste materials, are transported outside and inside. The word ‘Partial enlargement’ means the big circle at the right of the arrow is the enlarged model of protoscoleces. Various shapes exist in the figure, and the same shape with the same colour refers to the same protein or ion, and two squares drawn with a red dotted line indicate the TCA cycle and neurotoxic materials, respectively. The two-way arrows with thickened black line show the double-direction transmission of proteins and ions, whereas the single-way arrow indicates one direction; the single-way arrow with a dotted line points to the name of the shape." path-from-xml="dsy020f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671219/dsy020f5.tif" path-from-xml="dsy020f5.tif" target="_blank">View large</a><a section="122671219" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020f5.png?Expires=2147483647&Signature=FfHQ0yQwIePkoTC806Ovtz~3fx-LFWKyIdjqaAmFMYOxOs1JDDytB28y7c2xPT2x-ZYeeb0LIzBpF5j8CNkOM0qRjfRJK8lDiIM98gFK~35ne-qMxnuzuoxgqSCzjCv2thj~lsppgWE0UlGGQ80ChhIc8EGKJsB2JhV3jP88G6jRqpDXjxVVX64NJRfztq7Sq0dOLFWoNp61A5WLWLht1XofjofkmXBr6LjxPw0TB~lFKvMpcjY7c8pRHCSQIq0xVu5xZ7Sdu9WV2-jKHAXPiKlS~7VTiwtcYrsMl9s6kJvCgiX3PYudI3itVdMIxdZV188hvYdbJ40EG1uMSTG2jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671219&ar=5045548&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy020f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Effect of the coenurus on the sheep CNS. (a) The diagrammatic map of <em>T. multiceps</em> life cycle. The enlarged photo is a growing coenurus cyst with clusters of protoscoleces. (b) Possible mechanisms of the effects of the coenurus on the intermediate host brain. Nutrients and waste materials, are transported outside and inside. The word ‘Partial enlargement’ means the big circle at the right of the arrow is the enlarged model of protoscoleces. Various shapes exist in the figure, and the same shape with the same colour refers to the same protein or ion, and two squares drawn with a red dotted line indicate the TCA cycle and neurotoxic materials, respectively. The two-way arrows with thickened black line show the double-direction transmission of proteins and ions, whereas the single-way arrow indicates one direction; the single-way arrow with a dotted line points to the name of the shape.</p></div></div><div content-id="dsy020-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/m_dsy020f5.png?Expires=2147483647&Signature=a92qRuc3WyIXJ2tVYDad8mtHMvOZFQuTjRUF5A9eISVoVsSKZtcUbpw9YnkUV4P4yu74ORKDKZ3KTgKdYM5sVo5Th4AjTNCNP2INfYKQo9olZCe-kT9gzAe9fcqp7LANiXGJI~fZyyFXm3Wr2Yzpyec4Z58rfnhmAH19sL~WKp9EV1fdBmz35RHGJwrLHbFIgA8OEGYgytsgS89otcqMAjSXyuZ-yxo0378s8fuD0FqgMOfpUHW~wnfSa~cW4xsvVr~TD~ju594OZyfqnSznNhRp5624GWfU6Artjey8wu8ze78CuIf0GcuD5V6V6wHpQrC8lz1TXx6fUQeCnQ20SA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Effect of the coenurus on the sheep CNS. (a) The diagrammatic map of T. multiceps life cycle. The enlarged photo is a growing coenurus cyst with clusters of protoscoleces. (b) Possible mechanisms of the effects of the coenurus on the intermediate host brain. Nutrients and waste materials, are transported outside and inside. The word ‘Partial enlargement’ means the big circle at the right of the arrow is the enlarged model of protoscoleces. Various shapes exist in the figure, and the same shape with the same colour refers to the same protein or ion, and two squares drawn with a red dotted line indicate the TCA cycle and neurotoxic materials, respectively. The two-way arrows with thickened black line show the double-direction transmission of proteins and ions, whereas the single-way arrow indicates one direction; the single-way arrow with a dotted line points to the name of the shape." path-from-xml="dsy020f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671219/dsy020f5.tif" path-from-xml="dsy020f5.tif" target="_blank">View large</a><a section="122671219" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020f5.png?Expires=2147483647&Signature=FfHQ0yQwIePkoTC806Ovtz~3fx-LFWKyIdjqaAmFMYOxOs1JDDytB28y7c2xPT2x-ZYeeb0LIzBpF5j8CNkOM0qRjfRJK8lDiIM98gFK~35ne-qMxnuzuoxgqSCzjCv2thj~lsppgWE0UlGGQ80ChhIc8EGKJsB2JhV3jP88G6jRqpDXjxVVX64NJRfztq7Sq0dOLFWoNp61A5WLWLht1XofjofkmXBr6LjxPw0TB~lFKvMpcjY7c8pRHCSQIq0xVu5xZ7Sdu9WV2-jKHAXPiKlS~7VTiwtcYrsMl9s6kJvCgiX3PYudI3itVdMIxdZV188hvYdbJ40EG1uMSTG2jg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671219&ar=5045548&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy020f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Effect of the coenurus on the sheep CNS. (a) The diagrammatic map of <em>T. multiceps</em> life cycle. The enlarged photo is a growing coenurus cyst with clusters of protoscoleces. (b) Possible mechanisms of the effects of the coenurus on the intermediate host brain. Nutrients and waste materials, are transported outside and inside. The word ‘Partial enlargement’ means the big circle at the right of the arrow is the enlarged model of protoscoleces. Various shapes exist in the figure, and the same shape with the same colour refers to the same protein or ion, and two squares drawn with a red dotted line indicate the TCA cycle and neurotoxic materials, respectively. The two-way arrows with thickened black line show the double-direction transmission of proteins and ions, whereas the single-way arrow indicates one direction; the single-way arrow with a dotted line points to the name of the shape.</p></div></div><p>To proliferate and grow, the coenurus larva needs to exchange nutrients including glucose, amino acids, lipids and other materials between the cyst wall and host brain. In this respect, glutamate is not only able to provide metabolic energy but also acts as a neurotransmitter to regulate signal transmissions between the synaptosomes and neurons.<a reveal-id="dsy020-B44" data-open="dsy020-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a><sup>,</sup><a reveal-id="dsy020-B45" data-open="dsy020-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> Several members of the SLC1A glutamate transporter family have been reported to be involved in glutamate transmission.<a reveal-id="dsy020-B46" data-open="dsy020-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> We identified 7 SLC1A gene family members in <em>T. multiceps</em>, which was considerably more than found present in the five other tapeworms (mean 4.4; median 4; <em>P</em> &lt; 0.05), suggesting an increased glutamate transmission capability. Among the SLC1A genes, SLC1A4 composed a major percentage of 50%, which showed a specific expression in different stages (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s1.xlsx?Expires=2147483647&Signature=KLjFb7BUTE5n1H-iUmV5Hf0xggueqoIj1voQ4a6nNQD9ndEqpMjOZSH4WnDukwAoG19jetEaISSjtoq~lnLi9RO7LQrrmFigdeNSkUKRRO-VCQeay790oTo5hhYfJ5Ctuqi2Qp5CzjgS~zOElwIJOlpYq0QEGV0S3rm5lJPWfO5tTRhrEJ3T8mzk6Nz~m3qUwhKo2Oz9pl0sttAIwkJnN6njYFI45oXBc1QQfSt0b455qGWZcT4vy6-XR~5jTquh7PJjsXwjPWlH~u~u59~qk2C8qeG9AFhzT-VI3NWuYkmww1qcxnpfjSE3z5MdG4a2El1znx6HU1OjJzmpYQEjRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S2). It is noteworthy that three members of SLC1A4 were abundantly expressed in the coenurus, indicating that the SLC1A4 genes might be functional divergent in <em>T. multiceps.</em> Sugars are also centrally involved in metabolism and energy production and storage, and we identified in the <em>T. multiceps</em> genome one glucose and four sugar transporters having a high level of expression in the cyst wall or protoscoleces. Moreover, two important genes mediating glucose metabolism and energy production (glucose-6-phosphate-dehydrogenase, Tm1G004355 and glucose-6-phosphate isomerase, Tm3G008866), were highly expressed in protoscoleces (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s1.xlsx?Expires=2147483647&Signature=KLjFb7BUTE5n1H-iUmV5Hf0xggueqoIj1voQ4a6nNQD9ndEqpMjOZSH4WnDukwAoG19jetEaISSjtoq~lnLi9RO7LQrrmFigdeNSkUKRRO-VCQeay790oTo5hhYfJ5Ctuqi2Qp5CzjgS~zOElwIJOlpYq0QEGV0S3rm5lJPWfO5tTRhrEJ3T8mzk6Nz~m3qUwhKo2Oz9pl0sttAIwkJnN6njYFI45oXBc1QQfSt0b455qGWZcT4vy6-XR~5jTquh7PJjsXwjPWlH~u~u59~qk2C8qeG9AFhzT-VI3NWuYkmww1qcxnpfjSE3z5MdG4a2El1znx6HU1OjJzmpYQEjRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S2). Besides obtaining nutrients from the mammalian host, the coenurus stage causes neurologic dysfunction by the production and release of neurotoxic substances. During metabolism, the consumption of glucose and oxygen by protoscoleces results in the production of carbon dioxide that is released outside the cyst wall, resulting in host cell toxicity. Additionally, there is high expression of sodium/potassium-transporting ATPase (Tm1G003772) and calcium pump (Tm1G002964) in protoscoleces and this may impact on ion homeostasis in the host neuron microenvironment. It has been shown that the imbalance of glutamate diffusing over synaptosomes could change synaptic function and interfere with signal transmission, causing toxicity in the host nervous system.<a reveal-id="dsy020-B46" data-open="dsy020-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> The expression of glutamate dehydrogenase 1 (Tm3G009148; FPKM = 577) in protoscoleces was significantly higher than in the other stages (<em>P</em> &lt; 0.001), and this could result in an imbalance in the concentration of glutamate in the host brain, resulting in synaptic dysfunction. Moreover, two gene homologues of neurodegenerative disorder were highly expressed in protoscoleces (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s1.xlsx?Expires=2147483647&Signature=KLjFb7BUTE5n1H-iUmV5Hf0xggueqoIj1voQ4a6nNQD9ndEqpMjOZSH4WnDukwAoG19jetEaISSjtoq~lnLi9RO7LQrrmFigdeNSkUKRRO-VCQeay790oTo5hhYfJ5Ctuqi2Qp5CzjgS~zOElwIJOlpYq0QEGV0S3rm5lJPWfO5tTRhrEJ3T8mzk6Nz~m3qUwhKo2Oz9pl0sttAIwkJnN6njYFI45oXBc1QQfSt0b455qGWZcT4vy6-XR~5jTquh7PJjsXwjPWlH~u~u59~qk2C8qeG9AFhzT-VI3NWuYkmww1qcxnpfjSE3z5MdG4a2El1znx6HU1OjJzmpYQEjRw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S2), and these may also contribute to nervous system toxicity.</p>                            <h3 scrollto-destination=122671221 id="122671221" class="section-title" >3.7. Environment adaptation and immune evasion contributed by horizontal gene transfer</h3>
<p>Horizontal gene transfer (HGT), or the passage of genetic material between non-mating species, is a ubiquitous process that increases the divergence and improves the physiological metabolism of recipient organisms.<a reveal-id="dsy020-B47" data-open="dsy020-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a><sup>,</sup><a reveal-id="dsy020-B48" data-open="dsy020-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> We identified HGT genes in <em>T. multiceps</em> obtained from the host using a published determination pipeline with modifications.<a reveal-id="dsy020-B49" data-open="dsy020-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a></p><p>Eleven genes were identified as HGT candidates in the <em>T. multiceps</em> genome from three hosts, including sheep (<em>Ovis aries</em>), dog (<em>Canis lupus familiaris</em>) and goat (<em>Capra hircus</em>) (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S12</a></span>). In phylogenetic analysis, the 11 candidate genes displayed a close relationship with host orthologs but were much more distant from the orthologs of species close to <em>T. multiceps</em> (<a reveal-id="dsy020-F6" data-open="dsy020-F6" class="link link-reveal link-table xref-fig">Fig. 6a</a> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7a</a></span>–j). Among the 11 HGT candidates, all have high identity (&gt;70%) and coverage (&gt;80%) with the orthologs from mammalian hosts, such as Tm1G003541 (<a reveal-id="dsy020-F6" data-open="dsy020-F6" class="link link-reveal link-table xref-fig">Fig. 6b</a>). To confirm that the HGT genes were transferred from the host rather than due to assembly error, the Illumina pair-end reads were mapped to the region flanking a HGT candidate, Tm1G003541. The results showed that the pair-end reads could cross the HGT candidate region continuously in which the flanking genes represented syntenic relationships with <em>E. multilocularis</em>, excepting the HGT candidate (<a reveal-id="dsy020-F6" data-open="dsy020-F6" class="link link-reveal link-table xref-fig">Fig. 6c</a>). These results indicated that the HGT candidate (Tm1G003541), which might bind to the MHCII from the host and regulate the release of Type I interferon, was probably transferred from host and play important role to interfere in the immune recognition and following immune responses. Furthermore, 8 of 11 candidates had unigene evidence from a previous shotgun assembled transcriptome of <em>T. multiceps.</em><a reveal-id="dsy020-B2" data-open="dsy020-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> In our transcriptome data on stages and tissues of <em>T. multiceps</em>, the expression of HGT candidates showed that a protein related to translational elongation (Tm1G005508) and a Hsp90 (Tm1G003541) had high expression levels, especially in the cyst wall and protoscoleces, whereas five Hsp70 proteins showed low expression patterns (<a reveal-id="dsy020-F6" data-open="dsy020-F6" class="link link-reveal link-table xref-fig">Fig. 6d</a>). The GO analysis of the 11 candidates in <em>T. multiceps</em> showed that the HGT could be clustered into 4 pools, including protein modification, adaptation, reproduction and heat shock (<a reveal-id="dsy020-F6" data-open="dsy020-F6" class="link link-reveal link-table xref-fig">Fig. 6e</a>). In summary, these results suggest that the <em>T. multiceps</em> genes transferred from hosts may play important roles in environment adaptation and immune evasion. </p>                        <a id="122671224" scrollto-destination="122671224"></a>
<div id="dsy020-F6" data-id="dsy020-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/m_dsy020f6.png?Expires=2147483647&Signature=lXgb2l5eBC4JDot2xIn3nzoHwxqhf90QzyGgI7lsAsh5zMdqMdkQDEeE2mC6fg7V4kT2ZUJd21ezxolWlfzmta7AiiHaBefE4cYNe0fSvWSw-pOIITlZb8zvgiT2wbsBIAjN5co2Dcu~kG9v8CHU7zM9fgddm6Q6QfFc70Siqv826bcQRhEAzkMPVDDABgrWYHf8WSkr6iQDanQ1zL3MUwEA5GAfBajoqXIFsfhFoXYFyqvSZ8~a1FgQmHwDMqNIaettVkUSZjde4XIUQnG3HalhIxPncCXrgncdH7B0WfPrLelFQMAQx~lWQXxyU6TLDLxSw1RaCuGj6FUaogG5lw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Predicted host-origin HTGs in T. multiceps. (a) Phylogenetic tree of Tm1G003541 and its homologues. The clades highlighted in yellow, green and blue represent mammals, lower vertebrates and invertebrates, respectively, while the font in red represents tapeworm and dog. (b) The characteristic of sequence alignment between HTG candidate (Tm1G003541). (c) The local pair-end reads mapping of the flanking region of HTG candidate (Tm1G003541). The red box indicates the HTG gene. (d) Expression patterns of T. multiceps HTGs based on transcriptome analysis. (e) GO annotation of the HGT candidates." path-from-xml="dsy020f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671224/dsy020f6.tif" path-from-xml="dsy020f6.tif" target="_blank">View large</a><a section="122671224" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020f6.png?Expires=2147483647&Signature=cSloqNUBNoEPBVo~PelpRKkOFwIiW~AIvIiWmH94UXiKoYnTbZK4HQ2BROKbCBYA6UYN9f8yCEwGvRSn7ttflI~NxYim9qCTbQB06RLFSdrH~I7OIMLf570B-6OjmS8cZaPdnV48K3LBqNsNDSP4kQ9J7czO0ICDE9QpJ15LmJGl45Y2p0OGDa~EpqWbU9xjNmvZ88IxrE7aMTIje2yUTke26FgSz9SDxHLumxhghP0hhRuEb-AckboIU4hbPlNqk2gbB4yIseD6qxhz-0WJpYSVF7oZFV3U03jCynbm9Q7~YvboQZ90GcAl9fWKg7l4GFgk7pKYrXZl8oJLBeu56g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671224&ar=5045548&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy020f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Predicted host-origin HTGs in <em>T. multiceps</em>. (a) Phylogenetic tree of Tm1G003541 and its homologues. The clades highlighted in yellow, green and blue represent mammals, lower vertebrates and invertebrates, respectively, while the font in red represents tapeworm and dog. (b) The characteristic of sequence alignment between HTG candidate (Tm1G003541). (c) The local pair-end reads mapping of the flanking region of HTG candidate (Tm1G003541). The red box indicates the HTG gene. (d) Expression patterns of <em>T. multiceps</em> HTGs based on transcriptome analysis. (e) GO annotation of the HGT candidates.</p></div></div><div content-id="dsy020-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/m_dsy020f6.png?Expires=2147483647&Signature=lXgb2l5eBC4JDot2xIn3nzoHwxqhf90QzyGgI7lsAsh5zMdqMdkQDEeE2mC6fg7V4kT2ZUJd21ezxolWlfzmta7AiiHaBefE4cYNe0fSvWSw-pOIITlZb8zvgiT2wbsBIAjN5co2Dcu~kG9v8CHU7zM9fgddm6Q6QfFc70Siqv826bcQRhEAzkMPVDDABgrWYHf8WSkr6iQDanQ1zL3MUwEA5GAfBajoqXIFsfhFoXYFyqvSZ8~a1FgQmHwDMqNIaettVkUSZjde4XIUQnG3HalhIxPncCXrgncdH7B0WfPrLelFQMAQx~lWQXxyU6TLDLxSw1RaCuGj6FUaogG5lw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Predicted host-origin HTGs in T. multiceps. (a) Phylogenetic tree of Tm1G003541 and its homologues. The clades highlighted in yellow, green and blue represent mammals, lower vertebrates and invertebrates, respectively, while the font in red represents tapeworm and dog. (b) The characteristic of sequence alignment between HTG candidate (Tm1G003541). (c) The local pair-end reads mapping of the flanking region of HTG candidate (Tm1G003541). The red box indicates the HTG gene. (d) Expression patterns of T. multiceps HTGs based on transcriptome analysis. (e) GO annotation of the HGT candidates." path-from-xml="dsy020f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/122671224/dsy020f6.tif" path-from-xml="dsy020f6.tif" target="_blank">View large</a><a section="122671224" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020f6.png?Expires=2147483647&Signature=cSloqNUBNoEPBVo~PelpRKkOFwIiW~AIvIiWmH94UXiKoYnTbZK4HQ2BROKbCBYA6UYN9f8yCEwGvRSn7ttflI~NxYim9qCTbQB06RLFSdrH~I7OIMLf570B-6OjmS8cZaPdnV48K3LBqNsNDSP4kQ9J7czO0ICDE9QpJ15LmJGl45Y2p0OGDa~EpqWbU9xjNmvZ88IxrE7aMTIje2yUTke26FgSz9SDxHLumxhghP0hhRuEb-AckboIU4hbPlNqk2gbB4yIseD6qxhz-0WJpYSVF7oZFV3U03jCynbm9Q7~YvboQZ90GcAl9fWKg7l4GFgk7pKYrXZl8oJLBeu56g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=122671224&ar=5045548&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy020f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Predicted host-origin HTGs in <em>T. multiceps</em>. (a) Phylogenetic tree of Tm1G003541 and its homologues. The clades highlighted in yellow, green and blue represent mammals, lower vertebrates and invertebrates, respectively, while the font in red represents tapeworm and dog. (b) The characteristic of sequence alignment between HTG candidate (Tm1G003541). (c) The local pair-end reads mapping of the flanking region of HTG candidate (Tm1G003541). The red box indicates the HTG gene. (d) Expression patterns of <em>T. multiceps</em> HTGs based on transcriptome analysis. (e) GO annotation of the HGT candidates.</p></div></div>                            <h3 scrollto-destination=122671225 id="122671225" class="section-title" >3.8. Genome-based new molecular targets for drug intervention</h3>
<p>Coenurosis is a devastating disease and difficult to treat and the <em>T. multiceps</em> genome project makes feasible the identification of novel targets for drug discovery. Historically, enzymes have been a key target group for drug discovery.<a reveal-id="dsy020-B50" data-open="dsy020-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> Based on Food and Drug Administration (FDA) and clinical studies, we identified a total of 210 known drug target genes involved with 56 enzyme systems in <em>T. multiceps</em> (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S13</a></span>). In addition to enzymes, kinases, GPCRs and ligand-gated ion channels (LGICs) may also be important drug targets for the treatment of <em>T. multiceps</em>. We identified a total of 284 kinases and 38 GPCRs in <em>T. multiceps</em> (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S14 and S15</a></span>) and 45 members of four major LGIC families. Several effective anthelminthic drugs interfere with one of the multiple forms of neural control and communication.<a reveal-id="dsy020-B51" data-open="dsy020-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> Therefore, we searched the signalling pathways of neurotransmitters and identified 11 and 8 conserved genes in the Ach and serotonin pathways respectively (<span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S16 and S17</a></span>), which may serve as broad-spectrum drug targets for treating coenuriasis and other cestodiases.</p><p>To identify species-specific genes in <em>T. multiceps</em>, we compared the above potential targets to host proteins by BLASTP (cutoff: 1 <em>e</em><sup>−5</sup>). In total, no homologues of 218 specific targets genes in <em>T. multiceps</em> were found in humans, canines and sheep (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s2.xlsx?Expires=2147483647&Signature=e-mgmcNxLNQBfk3vrQ6r~zeURjyzFQEjtaKxuIWcjwuiefGnPmT-1gQJ4g~AhNHDUZy2L4U8PMiYnvUeMV4pDQWrZFvqVBRdK4rOwZsN4ivUc5Oc4Ske76CyfSj8mWi92k2zaRC-3TTIcMDfZMucgP8xosAzDKhaVQ22xxssPyVZmx684AEcKxM7XCCp854H-WN~SIlGBYq5n3icLQCri~LhIlrBq1Q49sEm8i-j8a6mtVlJcU0YnacodaZ7xWIoFH0lYFVC73MMuGZcH6gAKCHS6rZjJDzch-jdwmCz05CCrLSlOddIHpL7atAH6kZ3pBmO2PfzSx94N9QGTr8Z1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span> S3), indicating that these species-specific targets not only have potential for developing as diagnostics or as vaccines but may also have limited side effects on mammalian hosts.</p>                            <h2 scrollto-destination=122671228 id="122671228" class="section-title" >4. Discussion</h2>
<p>SMRT sequencing offers longer length reads than second-generation sequencing technologies, making it well suited for unsolved problems in genome research.<a reveal-id="dsy020-B52" data-open="dsy020-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a><sup>,</sup><em>De novo</em> assemblies using PacBio sequencing, as applied here, can close gaps in current reference assemblies, which are mostly composed of large repeat sequences.<a reveal-id="dsy020-B52" data-open="dsy020-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> Using this high coverage genome assembly, we have identified much more TEs and tandem repeat sequences in <em>T. multiceps</em> than in the previously published <em>Taenia</em> genomes, enabling a comprehensive analysis of the content of each type of repeat sequence as well as telomere structures. The high-quality assembly allows fine annotation of gene models, facilitating evolutionary and functional analysis.</p><p>The genome and transcriptome data we generated provide a valuable resource for better understanding the molecular biology, physiology, phylogeny and mechanisms of pathogenesis in <em>T. multiceps</em> and coenurosis. In this study, we found expansion of Tbx6 genes and their up-regulation in the adult stage of <em>T. multiceps</em>, which may be associated with the increased number of proglottids. We also found several genes involved in interaction with the host CNS, and several genes likely derived from horizontal transfer, as a result of adaption to the parasitic life style. Furthermore, the species-specific gene targets identified in <em>T. multiceps</em> provide candidatesfor development as novel immunological diagnostic markers, as well as preventative vaccines and therapeutic drugs. This high-quality <em>T. multiceps</em> reference genome provides not only a comprehensive foundation for deeper understanding of the genetic and evolution of tapeworms, but also an invaluable resource to accelerate the development of novel treatments and tools for the control of coenurosis.</p>                            <h2 scrollto-destination=122671231 id="122671231" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank W.J. Tian, G. Yao and F.S. Cai from Lanzhou Veterinary Research Institute, CAAS for helping us to collect parasite materials. We also thank J. Ruan and D.M. Fang from Agricultural Genomic Institute at Shenzhen, CAAS for providing suggestions on genome assembly.</p>                            <h3 scrollto-destination=122671233 id="122671233" class="section-title" >Accession numbers</h3>
<p>The assembly and raw data have been deposited in GenBank with the following accession numbers: <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/MRSW00000000" target="_blank">MRSW00000000</a> for Whole Genome Shotgun project of <em>T. multiceps</em> under Bio Project accession <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/PRJNA307624" target="_blank">PRJNA307624</a>; The version described in this paper is version <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/MRSW02000000" target="_blank">MRSW02000000</a>; <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRR3103080" target="_blank">SRR3103080</a>, <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRR3219054" target="_blank">SRR3219054</a>, <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRR3219055" target="_blank">SRR3219055</a> and <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRR3219056" target="_blank">SRR3219056</a> for the <em>T. multiceps</em> genomic Illumina DNA data; <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRX1531851" target="_blank">SRX1531851</a>, <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRX3851533" target="_blank">SRX3851533</a>, <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRX3851532" target="_blank">SRX3851532</a> and <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRX3851531" target="_blank">SRX3851531</a> for the <em>T. multiceps</em> genomic PacBio SMRT data; <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRR5032913" target="_blank">SRR5032913</a>, <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRR5032914" target="_blank">SRR5032914</a>, <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRR5032918" target="_blank">SRR5032918</a>, <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRR5032916" target="_blank">SRR5032916</a>, <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRR5032912" target="_blank">SRR5032912</a>, <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRR5032915" target="_blank">SRR5032915</a> and <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRR5032917" target="_blank">SRR5032917</a> for the <em>T. multiceps</em> RNA-seq data. <a class="link link-gen" href="http://www.ncbi.nlm.nih.gov/nuccore/SRX3856587" target="_blank">SRX3856587</a> for the <em>T. multiceps</em> genomic Hi-C data.</p>                            <h3 scrollto-destination=122671235 id="122671235" class="section-title" >Funding</h3>
<p>The study was funded by the State Key Laboratory of Veterinary Etiological Biology of Lanzhou Veterinary Research Institute, Chinese Academy of Agricultural Sciences; Agricultural Science and Technology Innovation Program (ASTIP) of Chinese Academy of Agricultural Sciences (CAAS); and was also partially supported by the National Natural Science Foundation of China (Grant No. 31702228); the National Beef Cattle and Yak Industrial Technology System, NBCITS, MOA (No. CARS-37); the Elite Youth Program of Chinese Academy of Agricultural Sciences and National Key Research and Development Program of China (2016YFC1200600); Shenzhen science and technology program (JCYJ20150630165133395).</p>                            <h3 scrollto-destination=122671237 id="122671237" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=122671239 id="122671239" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy020-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Oryan</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Akbari</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Moazeni</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Amrabadi</div> <div class="given-names">O. R.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Cerebral and non-cerebral coenurosis in small ruminants</div>, <div class="source">Trop. Biomed</div> ., <div class="volume">31</div>, <div class="fpage">1</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cerebral%20and%20non-cerebral%20coenurosis%20in%20small%20ruminants&amp;author=A.%20Oryan&amp;author=M.%20Akbari&amp;author=M.%20Moazeni&amp;author=O.%20R.%20Amrabadi&amp;publication_year=2014&amp;journal=Trop.%20Biomed&amp;volume=31&amp;pages=1-16" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24862039" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Fu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Detailed transcriptome description of the neglected cestode <em>Taenia multiceps</em></div>, <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e45830</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detailed%20transcriptome%20description%20of%20the%20neglected%20cestode%20Taenia%20multiceps&amp;author=X.%20Wu&amp;author=Y.%20Fu&amp;author=D.%20Yang&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e45830" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0045830" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0045830" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0045830"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23049872" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Scala</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Varcasia</div> <div class="given-names">A.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Updates on morphobiology, epidemiology and molecular characterization of coenurosis in sheep</div>, <div class="source">Parassitologia</div> , <div class="volume">48</div>, <div class="fpage">61</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Updates%20on%20morphobiology%2C%20epidemiology%20and%20molecular%20characterization%20of%20coenurosis%20in%20sheep&amp;author=A.%20Scala&amp;author=A.%20Varcasia&amp;publication_year=2006&amp;journal=Parassitologia&amp;volume=48&amp;pages=61-3" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16881398" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bussell</div> <div class="given-names">K. M.</div></div>, <div class="name string-name"><div class="surname">Kinder</div> <div class="given-names">A. E.</div></div>, <div class="name string-name"><div class="surname">Scott</div> <div class="given-names">P. R.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Posterior paralysis in a lamb caused by a coeneurus cerebralis cyst in the lumbar spinal cord</div>, <div class="source">Vet. Rec</div> ., <div class="volume">140</div>, <div class="fpage">560</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Posterior%20paralysis%20in%20a%20lamb%20caused%20by%20a%20coeneurus%20cerebralis%20cyst%20in%20the%20lumbar%20spinal%20cord&amp;author=K.%20M.%20Bussell&amp;author=A.%20E.%20Kinder&amp;author=P.%20R.%20Scott&amp;publication_year=1997&amp;journal=Vet.%20Rec&amp;volume=140&amp;pages=560" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1136/vr.140.21.560" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1136%2Fvr.140.21.560" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1136%2Fvr.140.21.560"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9185315" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Achenef</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Markos</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Feseha</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Hibret</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Tembely</div> <div class="given-names">S.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Coenurus cerebralis infection in Ethiopian highland sheep: incidence and observations on pathogenesis and clinical signs</div>, <div class="source">Trop. Anim. Health Prod</div> ., <div class="volume">31</div>, <div class="fpage">15</div>–<div class="lpage">24</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coenurus%20cerebralis%20infection%20in%20Ethiopian%20highland%20sheep%3A%20incidence%20and%20observations%20on%20pathogenesis%20and%20clinical%20signs&amp;author=M.%20Achenef&amp;author=T.%20Markos&amp;author=G.%20Feseha&amp;author=A.%20Hibret&amp;author=S.%20Tembely&amp;publication_year=1999&amp;journal=Trop.%20Anim.%20Health%20Prod&amp;volume=31&amp;pages=15-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1005125316275" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1005125316275" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1005125316275"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10399813" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Al-Riyami</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ioannidou</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Koehler</div> <div class="given-names">A. V.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Genetic characterisation of <em>Taenia multiceps</em> cysts from ruminants in Greece</div>. <div class="source">Infect. Genet. Evol</div> ., <div class="volume">38</div>, <div class="fpage">110</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20characterisation%20of%20Taenia%20multiceps%20cysts%20from%20ruminants%20in%20Greece&amp;author=S.%20Al-Riyami&amp;author=E.%20Ioannidou&amp;author=A.%20V.%20Koehler&amp;publication_year=2016&amp;journal=Infect.%20Genet.%20Evol&amp;volume=38&amp;pages=110-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.meegid.2015.12.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.meegid.2015.12.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.meegid.2015.12.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26688203" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">D. K.</div></div>, <div class="name string-name"><div class="surname">Chauhan</div> <div class="given-names">P. P. S.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Coenurosis status in Afro-Asian region: a review</div>, <div class="source">Small Ruminant Res</div> ., <div class="volume">64</div>, <div class="fpage">197</div>–<div class="lpage">202</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coenurosis%20status%20in%20Afro-Asian%20region%3A%20a%20review&amp;author=D.%20K.%20Sharma&amp;author=P.%20P.%20S.%20Chauhan&amp;publication_year=2006&amp;journal=Small%20Ruminant%20Res&amp;volume=64&amp;pages=197-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.smallrumres.2005.05.021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.smallrumres.2005.05.021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.smallrumres.2005.05.021"> </span></div> </div></div></div></div></div><div content-id="dsy020-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">El-On</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Shelef</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Cagnano</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Benifla</div> <div class="given-names">M.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Taenia multiceps: a rare human cestode infection in Israel</div>, <div class="source">Vet. Ital</div> ., <div class="volume">44</div>, <div class="fpage">621</div>–<div class="lpage">31</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Taenia%20multiceps%3A%20a%20rare%20human%20cestode%20infection%20in%20Israel&amp;author=J.%20El-On&amp;author=I.%20Shelef&amp;author=E.%20Cagnano&amp;author=M.%20Benifla&amp;publication_year=2008&amp;journal=Vet.%20Ital&amp;volume=44&amp;pages=621-31" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20411489" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stunkard</div> <div class="given-names">H. W.</div></div></div> <div class="year">1962</div>, 
<div class="article-title">The organization, ontogeny, and orientation of the Cestoda</div>, <div class="source">Quart. Rev. Biol</div> ., <div class="volume">37</div>, <div class="fpage">23</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20organization%2C%20ontogeny%2C%20and%20orientation%20of%20the%20Cestoda&amp;author=H.%20W.%20Stunkard&amp;publication_year=1962&amp;journal=Quart.%20Rev.%20Biol&amp;volume=37&amp;pages=23-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/403568" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F403568" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F403568"> </span></div> </div></div></div></div></div><div content-id="dsy020-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">L. S. S.</div></div>, <div class="name string-name"><div class="surname">Janovy</div> <div class="given-names">G. D.</div></div></div> <div class="year">2009</div>, <em>Gerald D. Schmidt &amp; Larry S. Roberts’ Foundations of parasitology</em>.</div></div></div></div><div content-id="dsy020-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pau</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Perria</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Turtas</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Brambilla</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Viale</div> <div class="given-names">G.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Long-term follow-up of the surgical treatment of intracranial coenurosis</div>, <div class="source">Br. J. Neurosurg</div> ., <div class="volume">4</div>, <div class="fpage">39</div>–<div class="lpage">43</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long-term%20follow-up%20of%20the%20surgical%20treatment%20of%20intracranial%20coenurosis&amp;author=A.%20Pau&amp;author=C.%20Perria&amp;author=S.%20Turtas&amp;author=M.%20Brambilla&amp;author=G.%20Viale&amp;publication_year=1990&amp;journal=Br.%20J.%20Neurosurg&amp;volume=4&amp;pages=39-43" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3109/02688699009000680" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3109%2F02688699009000680" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3109%2F02688699009000680"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2334526" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pau</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Turtas</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Brambilla</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Leoni</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Rosa</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Viale</div> <div class="given-names">G. L.</div></div></div> <div class="year">1987</div>, 
<div class="article-title">Computed tomography and magnetic resonance imaging of cerebral coenurosis</div>, <div class="source">Surg. Neurol</div> ., <div class="volume">27</div>, <div class="fpage">548</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computed%20tomography%20and%20magnetic%20resonance%20imaging%20of%20cerebral%20coenurosis&amp;author=A.%20Pau&amp;author=S.%20Turtas&amp;author=M.%20Brambilla&amp;author=A.%20Leoni&amp;author=M.%20Rosa&amp;author=G.%20L.%20Viale&amp;publication_year=1987&amp;journal=Surg.%20Neurol&amp;volume=27&amp;pages=548-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0090-3019(87)90153-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0090-3019(87)90153-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0090-3019(87)90153-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3576430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gauci</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Vural</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Öncel</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Vaccination with recombinant oncosphere antigens reduces the susceptibility of sheep to infection with <em>Taenia multiceps</em></div>, <div class="source">Int. J. Parasitol</div> ., <div class="volume">38</div>, <div class="fpage">1041</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Vaccination%20with%20recombinant%20oncosphere%20antigens%20reduces%20the%20susceptibility%20of%20sheep%20to%20infection%20with%20Taenia%20multiceps&amp;author=C.%20Gauci&amp;author=G.%20Vural&amp;author=T.%20%C3%96ncel&amp;publication_year=2008&amp;journal=Int.%20J.%20Parasitol&amp;volume=38&amp;pages=1041-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ijpara.2007.11.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijpara.2007.11.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ijpara.2007.11.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18160069" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Varcasia</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Tosciri</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Coccone</div> <div class="given-names">G. N.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Preliminary field trial of a vaccine against coenurosis caused by <em>Taenia multiceps</em></div>, <div class="source">Vet. Parasitol</div> ., <div class="volume">162</div>, <div class="fpage">285</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Preliminary%20field%20trial%20of%20a%20vaccine%20against%20coenurosis%20caused%20by%20Taenia%20multiceps&amp;author=A.%20Varcasia&amp;author=G.%20Tosciri&amp;author=G.%20N.%20Coccone&amp;publication_year=2009&amp;journal=Vet.%20Parasitol&amp;volume=162&amp;pages=285-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.vetpar.2009.03.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.vetpar.2009.03.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.vetpar.2009.03.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19345506" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lightowlers</div> <div class="given-names">M.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Cestode vaccines: origins, current status and future prospects</div>, <div class="source">Parasitology</div> , <div class="volume">133</div>, <div class="fpage">S27</div>–<div class="lpage">42</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cestode%20vaccines%3A%20origins%2C%20current%20status%20and%20future%20prospects&amp;author=M.%20Lightowlers&amp;publication_year=2006&amp;journal=Parasitology&amp;volume=133&amp;pages=S27-42" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S003118200600179X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS003118200600179X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS003118200600179X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17274847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tsai</div> <div class="given-names">I. J.</div></div>, <div class="name string-name"><div class="surname">Zarowiecki</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Holroyd</div> <div class="given-names">N.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The genomes of four tapeworm species reveal adaptations to parasitism</div>, <div class="source">Nature</div> , <div class="volume">496</div>, <div class="fpage">57</div>–<div class="lpage">63</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genomes%20of%20four%20tapeworm%20species%20reveal%20adaptations%20to%20parasitism&amp;author=I.%20J.%20Tsai&amp;author=M.%20Zarowiecki&amp;author=N.%20Holroyd&amp;publication_year=2013&amp;journal=Nature&amp;volume=496&amp;pages=57-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature12031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature12031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature12031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23485966" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Comparative genomics reveals adaptive evolution of Asian tapeworm in switching to a new intermediate host</div>, <div class="source">Nat. Commun</div> ., <div class="volume">7</div>, <div class="fpage">12845</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20reveals%20adaptive%20evolution%20of%20Asian%20tapeworm%20in%20switching%20to%20a%20new%20intermediate%20host&amp;author=S.%20Wang&amp;author=S.%20Wang&amp;author=Y.%20Luo&amp;publication_year=2016&amp;journal=Nat.%20Commun&amp;volume=7&amp;pages=12845" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms12845" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms12845" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms12845"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27653464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zheng</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The genome of the hydatid tapeworm <em>Echinococcus granulosus</em></div>, <div class="source">Nat. Genet</div> ., <div class="volume">45</div>, <div class="fpage">1168</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20hydatid%20tapeworm%20Echinococcus%20granulosus&amp;author=H.%20Zheng&amp;author=W.%20Zhang&amp;author=L.%20Zhang&amp;publication_year=2013&amp;journal=Nat.%20Genet&amp;volume=45&amp;pages=1168-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2757" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2757" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2757"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24013640" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Fu</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Identification of neglected cestode <em>Taenia multiceps</em> microRNAs by illumina sequencing and bioinformatic analysis</div>, <div class="source">BMC Vet. Res</div> ., <div class="volume">9</div>, <div class="fpage">162</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20neglected%20cestode%20Taenia%20multiceps%20microRNAs%20by%20illumina%20sequencing%20and%20bioinformatic%20analysis&amp;author=X.%20Wu&amp;author=Y.%20Fu&amp;author=D.%20Yang&amp;publication_year=2013&amp;journal=BMC%20Vet.%20Res&amp;volume=9&amp;pages=162" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1746-6148-9-162" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1746-6148-9-162" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1746-6148-9-162"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23941076" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Walker</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Abeel</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Shea</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Pilon: an integrated tool for comprehensive microbial variant detection and genome assembly improvement</div>, <div class="source">PloS One</div> ., <div class="volume">9</div>, <div class="fpage">e112963</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pilon%3A%20an%20integrated%20tool%20for%20comprehensive%20microbial%20variant%20detection%20and%20genome%20assembly%20improvement&amp;author=B.%20J.%20Walker&amp;author=T.%20Abeel&amp;author=T.%20Shea&amp;publication_year=2014&amp;journal=PloS%20One&amp;volume=9&amp;pages=e112963" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0112963" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0112963" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0112963"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25409509" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Burton</div> <div class="given-names">J. N.</div></div>, <div class="name string-name"><div class="surname">Adey</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Patwardhan</div> <div class="given-names">R. P.</div></div>, <div class="name string-name"><div class="surname">Qiu</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Kitzman</div> <div class="given-names">J. O.</div></div>, <div class="name string-name"><div class="surname">Shendure</div> <div class="given-names">J.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Chromosome-scale scaffolding of de novo genome assemblies based on chromatin interactions</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">31</div>, <div class="fpage">1119</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosome-scale%20scaffolding%20of%20de%20novo%20genome%20assemblies%20based%20on%20chromatin%20interactions&amp;author=J.%20N.%20Burton&amp;author=A.%20Adey&amp;author=R.%20P.%20Patwardhan&amp;author=R.%20Qiu&amp;author=J.%20O.%20Kitzman&amp;author=J.%20Shendure&amp;publication_year=2013&amp;journal=Nat.%20Biotechnol&amp;volume=31&amp;pages=1119-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2727" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2727" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2727"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24185095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B. J.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Automated eukaryotic gene structure annotation using EVidenceModeler and the program to assemble spliced alignments</div>, <div class="source">Genome Biol</div> ., <div class="volume">9</div>, <div class="fpage">R7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20eukaryotic%20gene%20structure%20annotation%20using%20EVidenceModeler%20and%20the%20program%20to%20assemble%20spliced%20alignments&amp;author=B.%20J.%20Haas&amp;author=S.%20L.%20Salzberg&amp;author=W.%20Zhu&amp;publication_year=2008&amp;journal=Genome%20Biol&amp;volume=9&amp;pages=R7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2008-9-1-r7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2008-9-1-r7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2008-9-1-r7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18190707" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">C. H.</div></div>, <div class="name string-name"><div class="surname">Apweiler</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Bairoch</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">The Universal Protein Resource (UniProt): an expanding universe of protein information</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">34</div>, <div class="fpage">D187</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Universal%20Protein%20Resource%20(UniProt)%3A%20an%20expanding%20universe%20of%20protein%20information&amp;author=C.%20H.%20Wu&amp;author=R.%20Apweiler&amp;author=A.%20Bairoch&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=D187-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj161" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj161" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj161"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381842" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quevillon</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Silventoinen</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Pillai</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">InterProScan: protein domains identifier</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">33</div>, <div class="fpage">W116</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%3A%20protein%20domains%20identifier&amp;author=E.%20Quevillon&amp;author=V.%20Silventoinen&amp;author=S.%20Pillai&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=W116-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Goto</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kawashima</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Okuno</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Hattori</div> <div class="given-names">M.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">The KEGG resource for deciphering the genome</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">32</div>, <div class="fpage">D277</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20KEGG%20resource%20for%20deciphering%20the%20genome&amp;author=M.%20Kanehisa&amp;author=S.%20Goto&amp;author=S.%20Kawashima&amp;author=Y.%20Okuno&amp;author=M.%20Hattori&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=D277-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh063" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh063" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh063"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Benson</div> <div class="given-names">G.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Tandem repeats finder: a program to analyze DNA sequences</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">27</div>, <div class="fpage">573</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences&amp;author=G.%20Benson&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res&amp;volume=27&amp;pages=573-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.2.573" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.2.573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.2.573"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9862982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tang</div> <div class="given-names">H. B.</div></div>, <div class="name string-name"><div class="surname">Bowers</div> <div class="given-names">J. E.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">X. Y.</div></div>, <div class="name string-name"><div class="surname">Ming</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Alam</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Paterson</div> <div class="given-names">A. H.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Perspective - synteny and collinearity in plant genomes</div>, <div class="source">Science</div> , <div class="volume">320</div>, <div class="fpage">486</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Perspective%20-%20synteny%20and%20collinearity%20in%20plant%20genomes&amp;author=H.%20B.%20Tang&amp;author=J.%20E.%20Bowers&amp;author=X.%20Y.%20Wang&amp;author=R.%20Ming&amp;author=M.%20Alam&amp;author=A.%20H.%20Paterson&amp;publication_year=2008&amp;journal=Science&amp;volume=320&amp;pages=486-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1153917" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1153917" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1153917"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18436778" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Emms</div> <div class="given-names">D. M.</div></div>, <div class="name string-name"><div class="surname">Kelly</div> <div class="given-names">S.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">OrthoFinder: solving fundamental biases in whole genome comparisons dramatically improves orthogroup inference accuracy</div>, <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OrthoFinder%3A%20solving%20fundamental%20biases%20in%20whole%20genome%20comparisons%20dramatically%20improves%20orthogroup%20inference%20accuracy&amp;author=D.%20M.%20Emms&amp;author=S.%20Kelly&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0721-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0721-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0721-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25616342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Edgar</div> <div class="given-names">R. C.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">MUSCLE: multiple sequence alignment with high accuracy and high throughput</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">32</div>, <div class="fpage">1792</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MUSCLE%3A%20multiple%20sequence%20alignment%20with%20high%20accuracy%20and%20high%20throughput&amp;author=R.%20C.%20Edgar&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res&amp;volume=32&amp;pages=1792-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh340" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh340" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh340"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15034147" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guindon</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Dufayard</div> <div class="given-names">J. F.</div></div>, <div class="name string-name"><div class="surname">Lefort</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Anisimova</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hordijk</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">New algorithms and methods to estimate maximum-likelihood phylogenies: assessing the performance of PhyML 3.0</div>, <div class="source">Syst. Biol</div> ., <div class="volume">59</div>, <div class="fpage">307</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20algorithms%20and%20methods%20to%20estimate%20maximum-likelihood%20phylogenies%3A%20assessing%20the%20performance%20of%20PhyML%203.0&amp;author=S.%20Guindon&amp;author=J.%20F.%20Dufayard&amp;author=V.%20Lefort&amp;author=M.%20Anisimova&amp;author=W.%20Hordijk&amp;author=O.%20Gascuel&amp;publication_year=2010&amp;journal=Syst.%20Biol&amp;volume=59&amp;pages=307-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/syq010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsyq010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fsysbio%2Fsyq010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20525638" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Darriba</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Taboada</div> <div class="given-names">G. L.</div></div>, <div class="name string-name"><div class="surname">Doallo</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Posada</div> <div class="given-names">D.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">ProtTest 3: fast selection of best-fit models of protein evolution</div>, <div class="source">Bioinformatics</div>  , <div class="volume">27</div>, <div class="fpage">1164</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ProtTest%203%3A%20fast%20selection%20of%20best-fit%20models%20of%20protein%20evolution&amp;author=D.%20Darriba&amp;author=G.%20L.%20Taboada&amp;author=R.%20Doallo&amp;author=D.%20Posada&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=1164-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21335321" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z. H.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">PAML 4: phylogenetic analysis by maximum likelihood</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">1586</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PAML%204%3A%20phylogenetic%20analysis%20by%20maximum%20likelihood&amp;author=Z.%20H.%20Yang&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=1586-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17483113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Knapp</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Nakao</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yanagida</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Phylogenetic relationships within <em>Echinococcus</em> and <em>Taenia</em> tapeworms (Cestoda: taeniidae): An inference from nuclear protein-coding genes</div>, <div class="source">Mol. Phylogenet. Evol</div> ., <div class="volume">61</div>, <div class="fpage">628</div>–<div class="lpage">38</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20relationships%20within%20Echinococcus%20and%20Taenia%20tapeworms%20(Cestoda%3A%20taeniidae)%3A%20An%20inference%20from%20nuclear%20protein-coding%20genes&amp;author=J.%20Knapp&amp;author=M.%20Nakao&amp;author=T.%20Yanagida&amp;publication_year=2011&amp;journal=Mol.%20Phylogenet.%20Evol&amp;volume=61&amp;pages=628-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2011.07.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2011.07.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2011.07.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21907295" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wang</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">J.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">KaKs_Calculator 2.0: a toolkit incorporating gamma-series methods and sliding window strategies</div>, <div class="source">Genomics Proteomics Bioinformatics</div> , <div class="volume">8</div>, <div class="fpage">77</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KaKs_Calculator%202.0%3A%20a%20toolkit%20incorporating%20gamma-series%20methods%20and%20sliding%20window%20strategies&amp;author=D.%20Wang&amp;author=Y.%20Zhang&amp;author=Z.%20Zhang&amp;author=J.%20Zhu&amp;author=J.%20Yu&amp;publication_year=2010&amp;journal=Genomics%20Proteomics%20Bioinformatics&amp;volume=8&amp;pages=77-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1672-0229(10)60008-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1672-0229(10)60008-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1672-0229(10)60008-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20451164" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tusnady</div> <div class="given-names">G. E.</div></div>, <div class="name string-name"><div class="surname">Simon</div> <div class="given-names">I.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">The HMMTOP transmembrane topology prediction server</div>, <div class="source">Bioinformatics</div>  , <div class="volume">17</div>, <div class="fpage">849</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20HMMTOP%20transmembrane%20topology%20prediction%20server&amp;author=G.%20E.%20Tusnady&amp;author=I.%20Simon&amp;publication_year=2001&amp;journal=Bioinformatics&amp;volume=17&amp;pages=849-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/17.9.849" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F17.9.849" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F17.9.849"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11590105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Berlin</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Koren</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Chin</div> <div class="given-names">C.S.</div></div>, <div class="name string-name"><div class="surname">Drake</div> <div class="given-names">J.P.</div></div>, <div class="name string-name"><div class="surname">Landolin</div> <div class="given-names">J.M.</div></div>, <div class="name string-name"><div class="surname">Phillippy</div> <div class="given-names">A.M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Assembling large genomes with single-molecule sequencing and locality-sensitive hashing</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">623</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assembling%20large%20genomes%20with%20single-molecule%20sequencing%20and%20locality-sensitive%20hashing&amp;author=K.%20Berlin&amp;author=S.%20Koren&amp;author=C.S.%20Chin&amp;author=J.P.%20Drake&amp;author=J.M.%20Landolin&amp;author=A.M.%20Phillippy&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=623-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3238" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3238" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3238"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26006009" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simao</div> <div class="given-names">F. A.</div></div>, <div class="name string-name"><div class="surname">Waterhouse</div> <div class="given-names">R. M.</div></div>, <div class="name string-name"><div class="surname">Ioannidis</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kriventseva</div> <div class="given-names">E. V.</div></div>, <div class="name string-name"><div class="surname">Zdobnov</div> <div class="given-names">E. M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</div>, <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">3210</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BUSCO%3A%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs&amp;author=F.%20A.%20Simao&amp;author=R.%20M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.%20V.%20Kriventseva&amp;author=E.%20M.%20Zdobnov&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3210-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26059717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Feschotte</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Wessler</div> <div class="given-names">S. R.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Plant transposable elements: where genetics meets genomics</div>, <div class="source">Nat. Rev. Genet</div> ., <div class="volume">3</div>, <div class="fpage">329</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20transposable%20elements%3A%20where%20genetics%20meets%20genomics&amp;author=C.%20Feschotte&amp;author=N.%20Jiang&amp;author=S.%20R.%20Wessler&amp;publication_year=2002&amp;journal=Nat.%20Rev.%20Genet&amp;volume=3&amp;pages=329-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg793" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg793" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg793"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11988759" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Guindon</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">A simple, fast, and accurate algorithm to estimate large phylogenies by maximum likelihood</div>, <div class="source">Syst. Biol</div> ., <div class="volume">52</div>, <div class="fpage">696</div>–<div class="lpage">704</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%2C%20fast%2C%20and%20accurate%20algorithm%20to%20estimate%20large%20phylogenies%20by%20maximum%20likelihood&amp;author=S.%20Guindon&amp;author=O.%20Gascuel&amp;publication_year=2003&amp;journal=Syst.%20Biol&amp;volume=52&amp;pages=696-704" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150390235520" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150390235520" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F10635150390235520"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14530136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">dos Reis</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z. H.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Approximate likelihood calculation on a phylogeny for bayesian estimation of divergence times</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">28</div>, <div class="fpage">2161</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Approximate%20likelihood%20calculation%20on%20a%20phylogeny%20for%20bayesian%20estimation%20of%20divergence%20times&amp;author=M.%20dos%20Reis&amp;author=Z.%20H.%20Yang&amp;publication_year=2011&amp;journal=Mol.%20Biol.%20Evol&amp;volume=28&amp;pages=2161-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msr045" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsr045" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsr045"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21310946" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marr</div> <div class="given-names">M.</div></div></div> <div class="year">2011</div>, <em>Developmental Regulatory Genes in Parasitic Flatworms</em>. Doctoral dissertation, Department of Life Sciences and the Natural History Museum, Imperial College London.</div></div></div></div><div content-id="dsy020-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Papaioannou</div> <div class="given-names">V. E.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">The T-box gene family: emerging roles in development, stem cells and cancer</div>, <div class="source">Development</div> , <div class="volume">141</div>, <div class="fpage">3819</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20T-box%20gene%20family%3A%20emerging%20roles%20in%20development%2C%20stem%20cells%20and%20cancer&amp;author=V.%20E.%20Papaioannou&amp;publication_year=2014&amp;journal=Development&amp;volume=141&amp;pages=3819-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/dev.104471" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fdev.104471" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fdev.104471"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25294936" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kanamori</div> <div class="given-names">K.</div></div></div> <div class="year">2017</div>, In vivo N-15 MRS study of glutamate metabolism in the rat brain. <em>Anal. Biochem</em>., <div class="volume">529</div>, <div class="fpage">179</div>–<div class="lpage">192</div>.</div></div></div></div><div content-id="dsy020-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Natesan</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Mani</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Arumugam</div> <div class="given-names">R.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Clinical aspects of urea cycle dysfunction and altered brain energy metabolism on modulation of glutamate receptors and transporters in acute and chronic hyperammonemia</div>, <div class="source">Biomed. Pharmacother</div> ., <div class="volume">81</div>, <div class="fpage">192</div>–<div class="lpage">202</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clinical%20aspects%20of%20urea%20cycle%20dysfunction%20and%20altered%20brain%20energy%20metabolism%20on%20modulation%20of%20glutamate%20receptors%20and%20transporters%20in%20acute%20and%20chronic%20hyperammonemia&amp;author=V.%20Natesan&amp;author=R.%20Mani&amp;author=R.%20Arumugam&amp;publication_year=2016&amp;journal=Biomed.%20Pharmacother&amp;volume=81&amp;pages=192-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biopha.2016.04.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biopha.2016.04.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biopha.2016.04.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27261594" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Macrez</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Stys</div> <div class="given-names">P. K.</div></div>, <div class="name string-name"><div class="surname">Vivien</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Lipton</div> <div class="given-names">S. A.</div></div>, <div class="name string-name"><div class="surname">Docagne</div> <div class="given-names">F.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Mechanisms of glutamate toxicity in multiple sclerosis: biomarker and therapeutic opportunities</div>, <div class="source">LancetNeurol</div> ., <div class="volume">15</div>, <div class="fpage">1089</div>–<div class="lpage">102</div>.</div></div></div></div><div content-id="dsy020-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Choi</div> <div class="given-names">I. G.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">S. H.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Global extent of horizontal gene transfer</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">104</div>, <div class="fpage">4489</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20extent%20of%20horizontal%20gene%20transfer&amp;author=I.%20G.%20Choi&amp;author=S.%20H.%20Kim&amp;publication_year=2007&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=104&amp;pages=4489-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0611557104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0611557104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0611557104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17360551" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gilbert</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Schaack</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Pace</div> <div class="given-names">J. K.</div></div>, <div class="name string-name"><div class="surname">Brindley</div> <div class="given-names">P. J.</div></div>, <div class="name string-name"><div class="surname">Feschotte</div> <div class="given-names">C.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">A role for host-parasite interactions in the horizontal transfer of transposons across phyla</div>, <div class="source">Nature</div> , <div class="volume">464</div>, <div class="fpage">1347</div>., 1347-U1344.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20role%20for%20host-parasite%20interactions%20in%20the%20horizontal%20transfer%20of%20transposons%20across%20phyla&amp;author=C.%20Gilbert&amp;author=S.%20Schaack&amp;author=J.%20K.%20Pace&amp;author=P.%20J.%20Brindley&amp;author=C.%20Feschotte&amp;publication_year=2010&amp;journal=Nature&amp;volume=464&amp;pages=1347" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08939" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08939" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08939"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20428170" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Keeling</div> <div class="given-names">P. J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Functional and ecological impacts of horizontal gene transfer in eukaryotes</div>, <div class="source">Curr. Opin. Genet. Dev</div> ., <div class="volume">19</div>, <div class="fpage">613</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20and%20ecological%20impacts%20of%20horizontal%20gene%20transfer%20in%20eukaryotes&amp;author=P.%20J.%20Keeling&amp;publication_year=2009&amp;journal=Curr.%20Opin.%20Genet.%20Dev&amp;volume=19&amp;pages=613-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gde.2009.10.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gde.2009.10.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gde.2009.10.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19897356" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Alexander</div> <div class="given-names">S. P. H.</div></div>, <div class="name string-name"><div class="surname">Benson</div> <div class="given-names">H. E.</div></div>, <div class="name string-name"><div class="surname">Faccenda</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The concise guide to pharmacology 2013/14: enzymes</div>, <div class="source">Br. J. Pharmacol</div> ., <div class="volume">170</div>, <div class="fpage">1797</div>–<div class="lpage">867</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20concise%20guide%20to%20pharmacology%202013%2F14%3A%20enzymes&amp;author=S.%20P.%20H.%20Alexander&amp;author=H.%20E.%20Benson&amp;author=E.%20Faccenda&amp;publication_year=2013&amp;journal=Br.%20J.%20Pharmacol&amp;volume=170&amp;pages=1797-867" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/bph.12451" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fbph.12451" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fbph.12451"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24528243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy020-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McVeigh</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Atkinson</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Marks</div> <div class="given-names">N. J.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Parasite neuropeptide biology: seeding rational drug target selection?</div> <div class="source">Int. J. Parasitol. Drug</div> , <div class="volume">2</div>, <div class="fpage">76</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Parasite%20neuropeptide%20biology%3A%20seeding%20rational%20drug%20target%20selection%3F&amp;author=P.%20McVeigh&amp;author=L.%20Atkinson&amp;author=N.%20J.%20Marks&amp;publication_year=2012&amp;journal=Int.%20J.%20Parasitol.%20Drug&amp;volume=2&amp;pages=76-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ijpddr.2011.10.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijpddr.2011.10.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ijpddr.2011.10.004"> </span></div> </div></div></div></div></div><div content-id="dsy020-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy020-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rhoads</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Au</div> <div class="given-names">K. F.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">PacBio sequencing and its applications</div>, <div class="source">Genomics. Proteomics Bioinformatics</div> ., <div class="volume">13</div>, <div class="fpage">278</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PacBio%20sequencing%20and%20its%20applications&amp;author=A.%20Rhoads&amp;author=K.%20F.%20Au&amp;publication_year=2015&amp;journal=Genomics.%20Proteomics%20Bioinformatics&amp;volume=13&amp;pages=278-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gpb.2015.08.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gpb.2015.08.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gpb.2015.08.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26542840" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        
        <div class="wi-editors">
            <div class="al-authors-list">

                    <div class="editor-tag-label">
Editor:
                            <span class="al-author-name">
                                <span id="info-18743851" class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_Article_EditorInformation_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Ito</div>
            <div class="info-card-role">
                Editor
            </div>
    </div>






        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Ito">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ito T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ito T&quot;">Google Scholar</a>
            </div>

</div>

                                </span>
                                <a class="linked-name" href="javascript:;">Takashi Ito</a>
                            </span>
                        <br/>
                    </div>
            </div>
        </div>

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=5045548&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_information.doc?Expires=2147483647&amp;Signature=jIuQ--s09O7g7OkKSDE1O6ogPCl1b6MtP4mi6d7RJAntqreB1qTWQWUlm25XduzK8XPeMIpI9slbCO~Tt7hW0lLxrorAPEV1MyNrh2LKa~6MLocgWnyoz3XVKDYw9MtexuAaehtYtAAzFYyrAiUuvb7XQJxXgL66apVa9cQMe4WnTpNT33vcevI-vStbA3n26CMDRoNH6RGZU6VK~e0xOUDSSEYrA94PrX79Oxhv3Z5LjCEHAfkmsHDpHhEGutz2Go~EafD5a17R3mmRMksViphdFI6In0FGjuhLyKdRIvFVPi9qk93xtho9wtmqK6K2TV4rI8xfuDq5HpsOVijGMw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Information</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s1.xlsx?Expires=2147483647&amp;Signature=KLjFb7BUTE5n1H-iUmV5Hf0xggueqoIj1voQ4a6nNQD9ndEqpMjOZSH4WnDukwAoG19jetEaISSjtoq~lnLi9RO7LQrrmFigdeNSkUKRRO-VCQeay790oTo5hhYfJ5Ctuqi2Qp5CzjgS~zOElwIJOlpYq0QEGV0S3rm5lJPWfO5tTRhrEJ3T8mzk6Nz~m3qUwhKo2Oz9pl0sttAIwkJnN6njYFI45oXBc1QQfSt0b455qGWZcT4vy6-XR~5jTquh7PJjsXwjPWlH~u~u59~qk2C8qeG9AFhzT-VI3NWuYkmww1qcxnpfjSE3z5MdG4a2El1znx6HU1OjJzmpYQEjRw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data S1</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s2.xlsx?Expires=2147483647&amp;Signature=e-mgmcNxLNQBfk3vrQ6r~zeURjyzFQEjtaKxuIWcjwuiefGnPmT-1gQJ4g~AhNHDUZy2L4U8PMiYnvUeMV4pDQWrZFvqVBRdK4rOwZsN4ivUc5Oc4Ske76CyfSj8mWi92k2zaRC-3TTIcMDfZMucgP8xosAzDKhaVQ22xxssPyVZmx684AEcKxM7XCCp854H-WN~SIlGBYq5n3icLQCri~LhIlrBq1Q49sEm8i-j8a6mtVlJcU0YnacodaZ7xWIoFH0lYFVC73MMuGZcH6gAKCHS6rZjJDzch-jdwmCz05CCrLSlOddIHpL7atAH6kZ3pBmO2PfzSx94N9QGTr8Z1Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data S2</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/5/10.1093_dnares_dsy020/1/dsy020_supplementary_data_s3.xlsx?Expires=2147483647&amp;Signature=YIKMobuP~Zs-yxZosItZX-6R0jRFmHtsfXCqlSw~V~kSXHy5-ztG0lCcQAXlV1EH8sNx65pDt7Yyr4Ugr0ZQd2EgOW36O5i2a-Jp9F9ONcLGstBF-2DVpJkDgVXDwXoExqoPoSDFAxo4xBEtspUYEdGST7wX0qAMQyi3kgr11ZeElS1QgKIc0fo-ldMsvSi3yU5bnPwYas-UOD00m0CFZsvLGRpyoPKlBUc7LpU3qHroq541FhZYdBZuOvPrc2de8evbqt1DU7LgBEJV-LfaUU7o2LPkMDWMmUom25-A7y~LPIieWFq3pI3BoezZTXaOiOczECPomLfmiAQJj9~Xhg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data S3</div></a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">703</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
0                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy020"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="5045548"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f5%2f499%2f5045548">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F5%2F499%2F5045548" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/5045548'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="genome of tapeworm Taenia multiceps sheds light on understanding parasitic mechanism and control of coenurosis disease" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy020");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f5%2f499%2f5045548">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

