

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/1/11/2462413","@type":"ScholarlyArticle","name":"The complete genome sequencing of  Prevotella intermedia  strain OMA14 and a subsequent fine-scale, intra-species genomic comparison reveal an unusual amplification of conjugative and mobile transposons and identify a novel  Prevotella- lineage-specific repeat","datePublished":"2015-12-08","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2016-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv032","keywords":["<italic>Prevotella intermedia</italic>","complete genome sequence","conjugative transposon","mobilizable transposon","repeat sequence"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/1/11/2462413","author":[{"name":"Naito, Mariko","affiliation":"1  Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunity ,  Nagasaki University Graduate School of Biomedical Sciences ,  Nagasaki 852-8588 ,  Japan","@type":"Person"},{"name":"Ogura, Yoshitoshi","affiliation":"2  Department of Bacteriology, Faculty of Medical Sciences ,  Kyushu University ,  Higashi-ku, Fukuoka 812-8582 ,  Japan","@type":"Person"},{"name":"Itoh, Takehiko","affiliation":"3  Department of Biological Information ,  Tokyo Institute of Technology ,  Meguro-ku, Tokyo 152-8550 ,  Japan","@type":"Person"},{"name":"Shoji, Mikio","affiliation":"1  Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunity ,  Nagasaki University Graduate School of Biomedical Sciences ,  Nagasaki 852-8588 ,  Japan","@type":"Person"},{"name":"Okamoto, Masaaki","affiliation":"4  Department of Oral Microbiology ,  Tsurumi University, School of Dental Medicine ,  Tsurumi-ku, Yokohama 230-8501 ,  Japan","@type":"Person"},{"name":"Hayashi, Tetsuya","affiliation":"2  Department of Bacteriology, Faculty of Medical Sciences ,  Kyushu University ,  Higashi-ku, Fukuoka 812-8582 ,  Japan","@type":"Person"},{"name":"Nakayama, Koji","affiliation":"1  Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunity ,  Nagasaki University Graduate School of Biomedical Sciences ,  Nagasaki 852-8588 ,  Japan","@type":"Person"}],"description":"Abstract.  Prevotella intermedia is a pathogenic bacterium involved in periodontal diseases. Here, we present the complete genome sequence of a clinical strain,","pageStart":"11","pageEnd":"19","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/1/1/m_cover.gif?Expires=2147483647&Signature=KARGGT3434AhJvd0livqaaljLmMbBUo3zfp1TqsESiO3YlY~tkN7g3APCzu-WSs6e8Nl3TqpcrmHNHH0dQCdcITSOeE~aAPITj~eX0Yge-o3-8ykKxYb~OvG6Tl68sbE8kb9MDj8-DVfaGgSNHoeCfxrY9GfQxxlAheNwGlc1dEbYs6TpQ1qqETGqeV3fGp~5ePhsu4hP22hUcJ1Z06TJ5-2W4IJPC8QUHrse0jdAO7iAQAiMxLNxFuiLgIQ4xVlRKAVyu8pGxiw2RvdR2ib~ISf7YQpT2ikZjdevDSe5J5xnrmr4aL9l8xCPdqY1Y38fIiBcp0jIC7kA0-uvGAJYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"The complete genome sequencing of  Prevotella intermedia  strain OMA14 and a subsequent fine-scale, intra-species genomic comparison reveal an unusual amplification of conjugative and mobile transposons and identify a novel  Prevotella- lineage-specific repeat","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/1/1/m_cover.gif?Expires=2147483647&Signature=KARGGT3434AhJvd0livqaaljLmMbBUo3zfp1TqsESiO3YlY~tkN7g3APCzu-WSs6e8Nl3TqpcrmHNHH0dQCdcITSOeE~aAPITj~eX0Yge-o3-8ykKxYb~OvG6Tl68sbE8kb9MDj8-DVfaGgSNHoeCfxrY9GfQxxlAheNwGlc1dEbYs6TpQ1qqETGqeV3fGp~5ePhsu4hP22hUcJ1Z06TJ5-2W4IJPC8QUHrse0jdAO7iAQAiMxLNxFuiLgIQ4xVlRKAVyu8pGxiw2RvdR2ib~ISf7YQpT2ikZjdevDSe5J5xnrmr4aL9l8xCPdqY1Y38fIiBcp0jIC7kA0-uvGAJYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="complete genome sequencing of  Prevotella intermedia  strain OMA14 and a subsequent fine-scale, intra-species genomic comparison reveal an unusual amplification of conjugative and mobile transposons and identify a novel  Prevotella- lineage-specific repeat" />
    <meta property="og:description" content="Abstract.  Prevotella intermedia is a pathogenic bacterium involved in periodontal diseases. Here, we present the complete genome sequence of a clinical strain," />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/1/11/2462413" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2015-12-08" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Naito, Mariko" /><meta name="citation_author_institution" content="Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunity, Nagasaki University Graduate School of Biomedical Sciences, Nagasaki 852-8588, Japan" /><meta name="citation_author" content="Ogura, Yoshitoshi" /><meta name="citation_author_institution" content="Department of Bacteriology, Faculty of Medical Sciences, Kyushu University, Higashi-ku, Fukuoka 812-8582, Japan" /><meta name="citation_author" content="Itoh, Takehiko" /><meta name="citation_author_institution" content="Department of Biological Information, Tokyo Institute of Technology, Meguro-ku, Tokyo 152-8550, Japan" /><meta name="citation_author" content="Shoji, Mikio" /><meta name="citation_author_institution" content="Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunity, Nagasaki University Graduate School of Biomedical Sciences, Nagasaki 852-8588, Japan" /><meta name="citation_author" content="Okamoto, Masaaki" /><meta name="citation_author_institution" content="Department of Oral Microbiology, Tsurumi University, School of Dental Medicine, Tsurumi-ku, Yokohama 230-8501, Japan" /><meta name="citation_author" content="Hayashi, Tetsuya" /><meta name="citation_author_institution" content="Department of Bacteriology, Faculty of Medical Sciences, Kyushu University, Higashi-ku, Fukuoka 812-8582, Japan" /><meta name="citation_author" content="Nakayama, Koji" /><meta name="citation_author_institution" content="Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunity, Nagasaki University Graduate School of Biomedical Sciences, Nagasaki 852-8588, Japan" /><meta name="citation_title" content="The complete genome sequencing of Prevotella intermedia strain OMA14 and a subsequent fine-scale, intra-species genomic comparison reveal an unusual amplification of conjugative and mobile transposons and identify a novel Prevotella-lineage-specific repeat" /><meta name="citation_firstpage" content="11" /><meta name="citation_lastpage" content="19" /><meta name="citation_doi" content="10.1093/dnares/dsv032" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2016/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Dental biofilms: difficult therapeutic targets,;  citation_journal_title=Periodontol.;  citation_year=2002;  citation_volume=28;  citation_pages=12-55; " /><meta name="citation_reference" content="citation_title=Localization and identification of root canal bacteria in clinically asymptomatic periapical pathosis;  citation_journal_title=J. Endod.;  citation_year=1990;  citation_volume=16;  citation_pages=534-8; " /><meta name="citation_reference" content="citation_title=Association of black-pigmented bacteria with endodontic infections;  citation_journal_title=J. Endod.;  citation_year=1999;  citation_volume=25;  citation_pages=413-5; " /><meta name="citation_reference" content="citation_title=Experimental gingivitis during pregnancy and post-partum: clinical, endocrinological, and microbiological aspects;  citation_journal_title=J. Clin. Periodontol.;  citation_year=1994;  citation_volume=21;  citation_pages=549-58; " /><meta name="citation_reference" content="citation_title=The bacteriology of acute necrotizing ulcerative gingivitis;  citation_journal_title=J. Periodontol.;  citation_year=1982;  citation_volume=53;  citation_pages=223-30; " /><meta name="citation_reference" content="citation_title=Associations between periodontal disease and risk for atherosclerosis, cardiovascular disease, and stroke. A systematic review;  citation_journal_title=Ann. Periodontol.;  citation_year=2003;  citation_volume=8;  citation_pages=38-53; " /><meta name="citation_reference" content="citation_title=Maternal periodontitis and prematurity. Part I: obstetric outcome of prematurity and growth restriction;  citation_journal_title=Ann. Periodontol.;  citation_year=2001;  citation_volume=6;  citation_pages=164-74; " /><meta name="citation_reference" content="citation_title=Identification of periodontal pathogens in atheromatous plaques;  citation_journal_title=J. Periodontol.;  citation_year=2000;  citation_volume=71;  citation_pages=1554-60; " /><meta name="citation_reference" content="citation_title=Maternal periodontitis and prematurity. Part II: maternal infection and fetal exposure;  citation_journal_title=Ann. Periodontol.;  citation_year=2001;  citation_volume=6;  citation_pages=175-82; " /><meta name="citation_reference" content="citation_title=AdpC is a Prevotella intermedia 17 leucine-rich repeat internalin-like protein;  citation_journal_title=Infect. Immun.;  citation_year=2010;  citation_volume=78;  citation_pages=2385-96; " /><meta name="citation_reference" content="citation_title=Interaction of Prevotella intermedia strain 17 leucine-rich repeat domain protein AdpF with eukaryotic cells promotes bacterial internalization;  citation_journal_title=Infect. Immun.;  citation_year=2014;  citation_volume=82;  citation_pages=2637-48; " /><meta name="citation_reference" content="citation_title=A new autocatalytic activation mechanism for cysteine proteases revealed by Prevotella intermedia interpain A;  citation_journal_title=J. Biol. Chem.;  citation_year=2008;  citation_volume=283;  citation_pages=2871-82; " /><meta name="citation_reference" content="citation_title=Chemical structure and immunobiological activity of lipid A from Prevotella intermedia ATCC 25611 lipopolysaccharide;  citation_journal_title=FEBS Lett.;  citation_year=2003;  citation_volume=543;  citation_pages=98-102; " /><meta name="citation_reference" content="citation_title=Comparative genome analysis of Prevotella intermedia strain isolated from infected root canal reveals features related to pathogenicity and adaptation;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=122" /><meta name="citation_reference" content="citation_title=Phenotypic characteristics and DNA relatedness in Prevotella intermedia and similar organisms;  citation_journal_title=Oral Microbiol. Immunol.;  citation_year=1992;  citation_volume=7;  citation_pages=60-4; " /><meta name="citation_reference" content="citation_title=Characterization of the Porphyromonas gingivalis conjugative transposon CTnPg1: determination of the integration site and the genes essential for conjugal transfer;  citation_journal_title=Microbiology;  citation_year=2011;  citation_volume=157;  citation_pages=2022-32; " /><meta name="citation_reference" content="citation_title=Microbial Genome Annotation Pipeline (MiGAP) for diverse users; citation_publisher=Japan;  citation_year=2009; " /><meta name="citation_reference" content="citation_title=Pfam: the protein families database;  citation_journal_title=Nucleic Acids Res.;  citation_year=2014;  citation_volume=42;  citation_pages=D222-30; " /><meta name="citation_reference" content="citation_title=Ori-Finder: a web-based system for finding oriCs in unannotated bacterial genomes;  citation_journal_title=BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=79" /><meta name="citation_reference" content="citation_title=GenomeMatcher: a graphical user interface for DNA sequence comparison;  citation_journal_title=BMC Bioinformatics;  citation_year=2008;  citation_volume=9;  citation_pages=376" /><meta name="citation_reference" content="citation_title=CD-HIT Suite: a web server for clustering and comparing biological sequences;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=680-2; " /><meta name="citation_reference" content="citation_title=DoriC: a database of oriC regions in bacterial genomes;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1866-7; " /><meta name="citation_reference" content="citation_title=ISfinder: the reference centre for bacterial insertion sequences;  citation_journal_title=Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=D32-6; " /><meta name="citation_reference" content="citation_title=When phage, plasmids, and transposons collide: genomic islands, and conjugative- and mobilizable-transposons as a mosaic continuum;  citation_journal_title=Plasmid;  citation_year=2002;  citation_volume=48;  citation_pages=202-12; " /><meta name="citation_reference" content="citation_title=New genetic element carrying the erythromycin resistance determinant erm(TR) in Streptococcus pneumoniae;  citation_journal_title=Antimicrob. Agents Chemother.;  citation_year=2008;  citation_volume=52;  citation_pages=619-25; " /><meta name="citation_reference" content="citation_title=The Bacteroides mobilizable insertion element, NBU1, integrates into the 3′ end of a Leu-tRNA gene and has an integrase that is a member of the lambda integrase family;  citation_journal_title=J. Bacteriol.;  citation_year=1996;  citation_volume=178;  citation_pages=3594-600; " /><meta name="citation_reference" content="citation_title=Characterization of a Bacteroides mobilizable transposon, NBU2, which carries a functional lincomycin resistance gene;  citation_journal_title=J. Bacteriol.;  citation_year=2000;  citation_volume=182;  citation_pages=3559-71; " /><meta name="citation_reference" content="citation_title=Molecular basis of indole production catalyzed by tryptophanase in the genus Prevotella;  citation_journal_title=FEMS Microbiol. Lett.;  citation_year=2011;  citation_volume=322;  citation_pages=51-9; " /><meta name="citation_reference" content="citation_title=Development of Prevotella intermedia-specific PCR primers based on the nucleotide sequences of a DNA probe Pig27,;  citation_journal_title=J. Microbiol. Methods;  citation_year=2011;  citation_volume=84;  citation_pages=394-7; " /><meta name="citation_reference" content="citation_title=Prevotella aurantiaca sp. nov., isolated from the human oral cavity;  citation_journal_title=Int. J. Syst. Evol. Microbiol.;  citation_year=2010;  citation_volume=60;  citation_pages=500-3; " /><meta name="citation_reference" content="citation_title=Prevotella falsenii sp. nov., a Prevotella intermedia-like organism isolated from monkey dental plaque;  citation_journal_title=Int. J. Syst. Evol. Microbiol.;  citation_year=2009;  citation_volume=59;  citation_pages=319-22; " /><meta name="citation_reference" content="citation_title=A protein secretion system linked to bacteroidete gliding motility and pathogenesis;  citation_journal_title=Proc. Natl Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=276-81; " /><meta name="citation_reference" content="citation_title=Porphyromonas gingivalis and related bacteria: from colonial pigmentation to the type IX secretion system and gliding motility;  citation_journal_title=J. Periodontal. Res.;  citation_year=2015;  citation_volume=50;  citation_pages=1-8; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/1/11/17462811/dsv032.pdf" /><meta name="description" content="Abstract.  Prevotella intermedia is a pathogenic bacterium involved in periodontal diseases. Here, we present the complete genome sequence of a clinical strain," />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/1/11/2462413" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/1/11/2462413" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/1/11/2462413");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv032';
    </script>
<title>
	complete genome sequencing of Prevotella intermedia strain OMA14 and a subsequent fine-scale, intra-species genomic comparison reveal an unusual amplification of conjugative and mobile transposons and identify a novel Prevotella-lineage-specific repeat | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2462413" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZpbPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWl0bywgTWFyaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgTWljcm9iaW9sb2d5IGFuZCBPcmFsIEluZmVjdGlvbiwgRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgTWljcm9iaW9sb2d5IGFuZCBJbW11bml0eSwgTmFnYXNha2kgVW5pdmVyc2l0eSBHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvbWVkaWNhbCBTY2llbmNlcywgTmFnYXNha2kgODUyLTg1ODgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2d1cmEsIFlvc2hpdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJhY3RlcmlvbG9neSwgRmFjdWx0eSBvZiBNZWRpY2FsIFNjaWVuY2VzLCBLeXVzaHUgVW5pdmVyc2l0eSwgSGlnYXNoaS1rdSwgRnVrdW9rYSA4MTItODU4MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJdG9oLCBUYWtlaGlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9naWNhbCBJbmZvcm1hdGlvbiwgVG9reW8gSW5zdGl0dXRlIG9mIFRlY2hub2xvZ3ksIE1lZ3Vyby1rdSwgVG9reW8gMTUyLTg1NTAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hvamksIE1pa2lvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgTWljcm9iaW9sb2d5IGFuZCBPcmFsIEluZmVjdGlvbiwgRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgTWljcm9iaW9sb2d5IGFuZCBJbW11bml0eSwgTmFnYXNha2kgVW5pdmVyc2l0eSBHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvbWVkaWNhbCBTY2llbmNlcywgTmFnYXNha2kgODUyLTg1ODgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iT2thbW90bywgTWFzYWFraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgT3JhbCBNaWNyb2Jpb2xvZ3ksIFRzdXJ1bWkgVW5pdmVyc2l0eSwgU2Nob29sIG9mIERlbnRhbCBNZWRpY2luZSwgVHN1cnVtaS1rdSwgWW9rb2hhbWEgMjMwLTg1MDEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGF5YXNoaSwgVGV0c3V5YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmFjdGVyaW9sb2d5LCBGYWN1bHR5IG9mIE1lZGljYWwgU2NpZW5jZXMsIEt5dXNodSBVbml2ZXJzaXR5LCBIaWdhc2hpLWt1LCBGdWt1b2thIDgxMi04NTgyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5ha2F5YW1hLCBLb2ppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgTWljcm9iaW9sb2d5IGFuZCBPcmFsIEluZmVjdGlvbiwgRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgTWljcm9iaW9sb2d5IGFuZCBJbW11bml0eSwgTmFnYXNha2kgVW5pdmVyc2l0eSBHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvbWVkaWNhbCBTY2llbmNlcywgTmFnYXNha2kgODUyLTg1ODgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJUaGUgY29tcGxldGUgZ2Vub21lIHNlcXVlbmNpbmcgb2YgUHJldm90ZWxsYSBpbnRlcm1lZGlhIHN0cmFpbiBPTUExNCBhbmQgYSBzdWJzZXF1ZW50IGZpbmUtc2NhbGUsIGludHJhLXNwZWNpZXMgZ2Vub21pYyBjb21wYXJpc29uIHJldmVhbCBhbiB1bnVzdWFsIGFtcGxpZmljYXRpb24gb2YgY29uanVnYXRpdmUgYW5kIG1vYmlsZSB0cmFuc3Bvc29ucyBhbmQgaWRlbnRpZnkgYSBub3ZlbCBQcmV2b3RlbGxhLWxpbmVhZ2Utc3BlY2lmaWMgcmVwZWF0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDMyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTYvMDIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZW50YWwgYmlvZmlsbXM6IGRpZmZpY3VsdCB0aGVyYXBldXRpYyB0YXJnZXRzLDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGVyaW9kb250b2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9MTItNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvY2FsaXphdGlvbiBhbmQgaWRlbnRpZmljYXRpb24gb2Ygcm9vdCBjYW5hbCBiYWN0ZXJpYSBpbiBjbGluaWNhbGx5IGFzeW1wdG9tYXRpYyBwZXJpYXBpY2FsIHBhdGhvc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBFbmRvZC47ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz01MzQtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzb2NpYXRpb24gb2YgYmxhY2stcGlnbWVudGVkIGJhY3RlcmlhIHdpdGggZW5kb2RvbnRpYyBpbmZlY3Rpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBFbmRvZC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz00MTMtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwZXJpbWVudGFsIGdpbmdpdml0aXMgZHVyaW5nIHByZWduYW5jeSBhbmQgcG9zdC1wYXJ0dW06IGNsaW5pY2FsLCBlbmRvY3Jpbm9sb2dpY2FsLCBhbmQgbWljcm9iaW9sb2dpY2FsIGFzcGVjdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIENsaW4uIFBlcmlvZG9udG9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTU0OS01ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGJhY3RlcmlvbG9neSBvZiBhY3V0ZSBuZWNyb3RpemluZyB1bGNlcmF0aXZlIGdpbmdpdml0aXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFBlcmlvZG9udG9sLjsgIGNpdGF0aW9uX3llYXI9MTk4MjsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTIyMy0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzb2NpYXRpb25zIGJldHdlZW4gcGVyaW9kb250YWwgZGlzZWFzZSBhbmQgcmlzayBmb3IgYXRoZXJvc2NsZXJvc2lzLCBjYXJkaW92YXNjdWxhciBkaXNlYXNlLCBhbmQgc3Ryb2tlLiBBIHN5c3RlbWF0aWMgcmV2aWV3OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Bbm4uIFBlcmlvZG9udG9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MzgtNTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hdGVybmFsIHBlcmlvZG9udGl0aXMgYW5kIHByZW1hdHVyaXR5LiBQYXJ0IEk6IG9ic3RldHJpYyBvdXRjb21lIG9mIHByZW1hdHVyaXR5IGFuZCBncm93dGggcmVzdHJpY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubi4gUGVyaW9kb250b2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xNjQtNzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIHBlcmlvZG9udGFsIHBhdGhvZ2VucyBpbiBhdGhlcm9tYXRvdXMgcGxhcXVlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gUGVyaW9kb250b2wuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTcxOyAgY2l0YXRpb25fcGFnZXM9MTU1NC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWF0ZXJuYWwgcGVyaW9kb250aXRpcyBhbmQgcHJlbWF0dXJpdHkuIFBhcnQgSUk6IG1hdGVybmFsIGluZmVjdGlvbiBhbmQgZmV0YWwgZXhwb3N1cmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFubi4gUGVyaW9kb250b2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0xNzUtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkcEMgaXMgYSBQcmV2b3RlbGxhIGludGVybWVkaWEgMTcgbGV1Y2luZS1yaWNoIHJlcGVhdCBpbnRlcm5hbGluLWxpa2UgcHJvdGVpbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW5mZWN0LiBJbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9Nzg7ICBjaXRhdGlvbl9wYWdlcz0yMzg1LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlcmFjdGlvbiBvZiBQcmV2b3RlbGxhIGludGVybWVkaWEgc3RyYWluIDE3IGxldWNpbmUtcmljaCByZXBlYXQgZG9tYWluIHByb3RlaW4gQWRwRiB3aXRoIGV1a2FyeW90aWMgY2VsbHMgcHJvbW90ZXMgYmFjdGVyaWFsIGludGVybmFsaXphdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW5mZWN0LiBJbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9ODI7ICBjaXRhdGlvbl9wYWdlcz0yNjM3LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBhdXRvY2F0YWx5dGljIGFjdGl2YXRpb24gbWVjaGFuaXNtIGZvciBjeXN0ZWluZSBwcm90ZWFzZXMgcmV2ZWFsZWQgYnkgUHJldm90ZWxsYSBpbnRlcm1lZGlhIGludGVycGFpbiBBOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yODM7ICBjaXRhdGlvbl9wYWdlcz0yODcxLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGVtaWNhbCBzdHJ1Y3R1cmUgYW5kIGltbXVub2Jpb2xvZ2ljYWwgYWN0aXZpdHkgb2YgbGlwaWQgQSBmcm9tIFByZXZvdGVsbGEgaW50ZXJtZWRpYSBBVENDIDI1NjExIGxpcG9wb2x5c2FjY2hhcmlkZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RkVCUyBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT01NDM7ICBjaXRhdGlvbl9wYWdlcz05OC0xMDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbm9tZSBhbmFseXNpcyBvZiBQcmV2b3RlbGxhIGludGVybWVkaWEgc3RyYWluIGlzb2xhdGVkIGZyb20gaW5mZWN0ZWQgcm9vdCBjYW5hbCByZXZlYWxzIGZlYXR1cmVzIHJlbGF0ZWQgdG8gcGF0aG9nZW5pY2l0eSBhbmQgYWRhcHRhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGhlbm90eXBpYyBjaGFyYWN0ZXJpc3RpY3MgYW5kIEROQSByZWxhdGVkbmVzcyBpbiBQcmV2b3RlbGxhIGludGVybWVkaWEgYW5kIHNpbWlsYXIgb3JnYW5pc21zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1PcmFsIE1pY3JvYmlvbC4gSW1tdW5vbC47ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTYwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgdGhlIFBvcnBoeXJvbW9uYXMgZ2luZ2l2YWxpcyBjb25qdWdhdGl2ZSB0cmFuc3Bvc29uIENUblBnMTogZGV0ZXJtaW5hdGlvbiBvZiB0aGUgaW50ZWdyYXRpb24gc2l0ZSBhbmQgdGhlIGdlbmVzIGVzc2VudGlhbCBmb3IgY29uanVnYWwgdHJhbnNmZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTc7ICBjaXRhdGlvbl9wYWdlcz0yMDIyLTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb2JpYWwgR2Vub21lIEFubm90YXRpb24gUGlwZWxpbmUgKE1pR0FQKSBmb3IgZGl2ZXJzZSB1c2VyczsgY2l0YXRpb25fcHVibGlzaGVyPUphcGFuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZmFtOiB0aGUgcHJvdGVpbiBmYW1pbGllcyBkYXRhYmFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9RDIyMi0zMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpLUZpbmRlcjogYSB3ZWItYmFzZWQgc3lzdGVtIGZvciBmaW5kaW5nIG9yaUNzIGluIHVuYW5ub3RhdGVkIGJhY3RlcmlhbCBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTc5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lTWF0Y2hlcjogYSBncmFwaGljYWwgdXNlciBpbnRlcmZhY2UgZm9yIEROQSBzZXF1ZW5jZSBjb21wYXJpc29uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTM3NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNELUhJVCBTdWl0ZTogYSB3ZWIgc2VydmVyIGZvciBjbHVzdGVyaW5nIGFuZCBjb21wYXJpbmcgYmlvbG9naWNhbCBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NjgwLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURvcmlDOiBhIGRhdGFiYXNlIG9mIG9yaUMgcmVnaW9ucyBpbiBiYWN0ZXJpYWwgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xODY2LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlTZmluZGVyOiB0aGUgcmVmZXJlbmNlIGNlbnRyZSBmb3IgYmFjdGVyaWFsIGluc2VydGlvbiBzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPUQzMi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaGVuIHBoYWdlLCBwbGFzbWlkcywgYW5kIHRyYW5zcG9zb25zIGNvbGxpZGU6IGdlbm9taWMgaXNsYW5kcywgYW5kIGNvbmp1Z2F0aXZlLSBhbmQgbW9iaWxpemFibGUtdHJhbnNwb3NvbnMgYXMgYSBtb3NhaWMgY29udGludXVtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFzbWlkOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTQ4OyAgY2l0YXRpb25fcGFnZXM9MjAyLTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OZXcgZ2VuZXRpYyBlbGVtZW50IGNhcnJ5aW5nIHRoZSBlcnl0aHJvbXljaW4gcmVzaXN0YW5jZSBkZXRlcm1pbmFudCBlcm0oVFIpIGluIFN0cmVwdG9jb2NjdXMgcG5ldW1vbmlhZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW50aW1pY3JvYi4gQWdlbnRzIENoZW1vdGhlci47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTI7ICBjaXRhdGlvbl9wYWdlcz02MTktMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBCYWN0ZXJvaWRlcyBtb2JpbGl6YWJsZSBpbnNlcnRpb24gZWxlbWVudCwgTkJVMSwgaW50ZWdyYXRlcyBpbnRvIHRoZSAz4oCyIGVuZCBvZiBhIExldS10Uk5BIGdlbmUgYW5kIGhhcyBhbiBpbnRlZ3Jhc2UgdGhhdCBpcyBhIG1lbWJlciBvZiB0aGUgbGFtYmRhIGludGVncmFzZSBmYW1pbHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTc4OyAgY2l0YXRpb25fcGFnZXM9MzU5NC02MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgYSBCYWN0ZXJvaWRlcyBtb2JpbGl6YWJsZSB0cmFuc3Bvc29uLCBOQlUyLCB3aGljaCBjYXJyaWVzIGEgZnVuY3Rpb25hbCBsaW5jb215Y2luIHJlc2lzdGFuY2UgZ2VuZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xODI7ICBjaXRhdGlvbl9wYWdlcz0zNTU5LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgYmFzaXMgb2YgaW5kb2xlIHByb2R1Y3Rpb24gY2F0YWx5emVkIGJ5IHRyeXB0b3BoYW5hc2UgaW4gdGhlIGdlbnVzIFByZXZvdGVsbGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFTVMgTWljcm9iaW9sLiBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMjI7ICBjaXRhdGlvbl9wYWdlcz01MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXZlbG9wbWVudCBvZiBQcmV2b3RlbGxhIGludGVybWVkaWEtc3BlY2lmaWMgUENSIHByaW1lcnMgYmFzZWQgb24gdGhlIG51Y2xlb3RpZGUgc2VxdWVuY2VzIG9mIGEgRE5BIHByb2JlIFBpZzI3LDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTWljcm9iaW9sLiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTg0OyAgY2l0YXRpb25fcGFnZXM9Mzk0LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZXZvdGVsbGEgYXVyYW50aWFjYSBzcC4gbm92LiwgaXNvbGF0ZWQgZnJvbSB0aGUgaHVtYW4gb3JhbCBjYXZpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gSi4gU3lzdC4gRXZvbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT02MDsgIGNpdGF0aW9uX3BhZ2VzPTUwMC0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmV2b3RlbGxhIGZhbHNlbmlpIHNwLiBub3YuLCBhIFByZXZvdGVsbGEgaW50ZXJtZWRpYS1saWtlIG9yZ2FuaXNtIGlzb2xhdGVkIGZyb20gbW9ua2V5IGRlbnRhbCBwbGFxdWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUludC4gSi4gU3lzdC4gRXZvbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OTsgIGNpdGF0aW9uX3BhZ2VzPTMxOS0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwcm90ZWluIHNlY3JldGlvbiBzeXN0ZW0gbGlua2VkIHRvIGJhY3Rlcm9pZGV0ZSBnbGlkaW5nIG1vdGlsaXR5IGFuZCBwYXRob2dlbmVzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9Mjc2LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Qb3JwaHlyb21vbmFzIGdpbmdpdmFsaXMgYW5kIHJlbGF0ZWQgYmFjdGVyaWE6IGZyb20gY29sb25pYWwgcGlnbWVudGF0aW9uIHRvIHRoZSB0eXBlIElYIHNlY3JldGlvbiBzeXN0ZW0gYW5kIGdsaWRpbmcgbW90aWxpdHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFBlcmlvZG9udGFsLiBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjMvMS8xMS8xNzQ2MjgxMS9kc3YwMzIucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFByZXZvdGVsbGEgaW50ZXJtZWRpYSBpcyBhIHBhdGhvZ2VuaWMgYmFjdGVyaXVtIGludm9sdmVkIGluIHBlcmlvZG9udGFsIGRpc2Vhc2VzLiBIZXJlLCB3ZSBwcmVzZW50IHRoZSBjb21wbGV0ZSBnZW5vbWUgc2VxdWVuY2Ugb2YgYSBjbGluaWNhbCBzdHJhaW4sIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0yNDYyNDEzJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZLOOAsw5JZJz7Jmp8y0JOS+WIenk" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f11%2f2462413" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f11%2f2462413" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=83835" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/1/1/m_cover.gif?Expires=2147483647&amp;Signature=KARGGT3434AhJvd0livqaaljLmMbBUo3zfp1TqsESiO3YlY~tkN7g3APCzu-WSs6e8Nl3TqpcrmHNHH0dQCdcITSOeE~aAPITj~eX0Yge-o3-8ykKxYb~OvG6Tl68sbE8kb9MDj8-DVfaGgSNHoeCfxrY9GfQxxlAheNwGlc1dEbYs6TpQ1qqETGqeV3fGp~5ePhsu4hP22hUcJ1Z06TJ5-2W4IJPC8QUHrse0jdAO7iAQAiMxLNxFuiLgIQ4xVlRKAVyu8pGxiw2RvdR2ib~ISf7YQpT2ikZjdevDSe5J5xnrmr4aL9l8xCPdqY1Y38fIiBcp0jIC7kA0-uvGAJYQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/1">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87196607">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196607">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196613">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196613">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196627">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196627">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196638">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196638">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196723">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196723">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196728">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196728">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/1/1/2462410">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/1/21/2462418">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
<span class="article-flag"><a href="https://academic.oup.com/dnaresearch/search-results?f_OUPSeries=Editor's+Choice">Editor's Choice</a></span>            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">The complete genome sequencing of <em>Prevotella intermedia</em> strain OMA14 and a subsequent fine-scale, intra-species genomic comparison reveal an unusual amplification of conjugative and mobile transposons and identify a novel <em>Prevotella-</em>lineage-specific repeat
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mariko Naito</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunity</div>, <div class="insititution">Nagasaki University Graduate School of Biomedical Sciences</div>, <div class="addr-line">Nagasaki 852-8588</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 95-849-7649. Fax. +81 95-849-7650. E-mail: <a href="mailto:mnaito@nagasaki-u.ac.jp" target="_blank">mnaito@nagasaki-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mariko+Naito">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Naito M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Naito M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mariko Naito</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshitoshi Ogura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bacteriology, Faculty of Medical Sciences</div>, <div class="insititution">Kyushu University</div>, <div class="addr-line">Higashi-ku, Fukuoka 812-8582</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshitoshi+Ogura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogura Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshitoshi Ogura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takehiko Itoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Biological Information</div>, <div class="insititution">Tokyo Institute of Technology</div>, <div class="addr-line">Meguro-ku, Tokyo 152-8550</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takehiko+Itoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Itoh T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Itoh T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takehiko Itoh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mikio Shoji</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunity</div>, <div class="insititution">Nagasaki University Graduate School of Biomedical Sciences</div>, <div class="addr-line">Nagasaki 852-8588</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mikio+Shoji">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shoji M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shoji M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mikio Shoji</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masaaki Okamoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Oral Microbiology</div>, <div class="insititution">Tsurumi University, School of Dental Medicine</div>, <div class="addr-line">Tsurumi-ku, Yokohama 230-8501</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masaaki+Okamoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Okamoto M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Okamoto M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masaaki Okamoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuya Hayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bacteriology, Faculty of Medical Sciences</div>, <div class="insititution">Kyushu University</div>, <div class="addr-line">Higashi-ku, Fukuoka 812-8582</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuya+Hayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hayashi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hayashi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tetsuya Hayashi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Koji Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunity</div>, <div class="insititution">Nagasaki University Graduate School of Biomedical Sciences</div>, <div class="addr-line">Nagasaki 852-8588</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Koji+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Koji Nakayama</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 1, 1 February 2016, Pages 11–19, <a href='https://doi.org/10.1093/dnares/dsv032'>https://doi.org/10.1093/dnares/dsv032</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">08 December 2015</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 July 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">23 October 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2462413" href="/dnaresearch/article-pdf/23/1/11/17462811/dsv032.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Mariko Naito, Yoshitoshi Ogura, Takehiko Itoh, Mikio Shoji, Masaaki Okamoto, Tetsuya Hayashi, Koji Nakayama;  The complete genome sequencing of <em>Prevotella intermedia</em> strain OMA14 and a subsequent fine-scale, intra-species genomic comparison reveal an unusual amplification of conjugative and mobile transposons and identify a novel <em>Prevotella-</em>lineage-specific repeat, <em>DNA Research</em>, Volume 23, Issue 1, 1 February 2016, Pages 11–19, <a href='https://doi.org/10.1093/dnares/dsv032'>https://doi.org/10.1093/dnares/dsv032</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2462413?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462413?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462413?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462413?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462413?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462413?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2462413?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=The%20complete%20genome%20sequencing%20of%20%26lt%3Bem%26gt%3BPrevotella%20intermedia%26lt%3B%2Fem%26gt%3B%20strain%20OMA14%20and%20a%20subsequent%20fine-scale%2C%20intra-species%20genomic%20comparison%20reveal%20an%20unusual%20amplification%20of%20conjugative%20and%20mobile%20transposons%20and%20identify%20a%20novel%20%26lt%3Bem%26gt%3BPrevotella-%26lt%3B%2Fem%26gt%3Blineage-specific%20repeat&amp;publicationDate=2015-12-08&amp;volumeNum=23&amp;issueNum=1&amp;author=Naito%2C%20Mariko%3B%20Ogura%2C%20Yoshitoshi&amp;startPage=11&amp;endPage=19&amp;contentId=10.1093%2Fdnares%2Fdsv032&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f11%2f2462413" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87196607 id="87196607" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Prevotella intermedia</em> is a pathogenic bacterium involved in periodontal diseases. Here, we present the complete genome sequence of a clinical strain, OMA14, of this bacterium along with the results of comparative genome analysis with strain 17 of the same species whose genome has also been sequenced, but not fully analysed yet. The genomes of both strains consist of two circular chromosomes: the larger chromosomes are similar in size and exhibit a high overall linearity of gene organizations, whereas the smaller chromosomes show a significant size variation and have undergone remarkable genome rearrangements. Unique features of the <em>Pre. intermedia</em> genomes are the presence of a remarkable number of essential genes on the second chromosomes and the abundance of conjugative and mobilizable transposons (CTns and MTns). The CTns/MTns are particularly abundant in the second chromosomes, involved in its extensive genome rearrangement, and have introduced a number of strain-specific genes into each strain. We also found a novel 188-bp repeat sequence that has been highly amplified in <em>Pre. intermedia</em> and are specifically distributed among the <em>Pre. intermedia</em>-related species. These findings expand our understanding of the genetic features of <em>Pre. intermedia</em> and the roles of CTns and MTns in the evolution of bacteria.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Prevotella intermedia&quot;"><em>Prevotella intermedia</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;complete genome sequence&quot;">complete genome sequence</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;conjugative transposon&quot;">conjugative transposon</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;mobilizable transposon&quot;">mobilizable transposon</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;repeat sequence&quot;">repeat sequence</a></div>                            <h2 scrollto-destination=87196613 id="87196613" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p><em>Prevotella intermedia</em> is a Gram-negative, black-pigmented anaerobic bacterium that is classified into the genus <em>Prevotella</em> belonging to the phylum <em>Bacteroidetes</em>. This bacterium is frequently found in subgingival plaque from patients with periodontal diseases and is considered one of the periodontal pathogens (a member of the so-called ‘orange complex’).<sup><a reveal-id="DSV032C1" data-open="DSV032C1" class="link link-ref link-reveal xref-bibr">1</a></sup><em>Prevotella</em> is phylogenetically close to other periodontal pathogens, namely <em>Porphyromonas gingivalis</em> and <em>Tannerella forsythia,</em> which are classified into the same phylum<em>. Prevotella intermedia</em> has also been associated with other oral infections, including endodontic infections,<sup><a reveal-id="DSV032C2" data-open="DSV032C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSV032C3" data-open="DSV032C3" class="link link-ref link-reveal xref-bibr">3</a></sup> pregnancy gingivitis<sup><a reveal-id="DSV032C4" data-open="DSV032C4" class="link link-ref link-reveal xref-bibr">4</a></sup> and acute necrotizing ulcerative gingivitis.<sup><a reveal-id="DSV032C5" data-open="DSV032C5" class="link link-ref link-reveal xref-bibr">5</a></sup> An association of periodontitis with systemic diseases, such as cardiovascular diseases<sup><a reveal-id="DSV032C6" data-open="DSV032C6" class="link link-ref link-reveal xref-bibr">6</a></sup> and preterm birth,<sup><a reveal-id="DSV032C7" data-open="DSV032C7" class="link link-ref link-reveal xref-bibr">7</a></sup> has also been suggested, because the DNA of periodontal pathogens, including <em>Pre. intermedia</em>, has been detected in atherosclerotic plaques.<sup><a reveal-id="DSV032C8" data-open="DSV032C8" class="link link-ref link-reveal xref-bibr">8</a></sup> In addition, the anti-<em>Pre. intermedia</em> IgM titre in umbilical cord blood is significantly higher in preterm infants than in full-term infants.<sup><a reveal-id="DSV032C9" data-open="DSV032C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>The virulence factors of <em>Pre. intermedia</em>, such as adhesins,<sup><a reveal-id="DSV032C10" data-open="DSV032C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSV032C11" data-open="DSV032C11" class="link link-ref link-reveal xref-bibr">11</a></sup> proteolytic enzymes<sup><a reveal-id="DSV032C12" data-open="DSV032C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and lipopolysaccharides,<sup><a reveal-id="DSV032C13" data-open="DSV032C13" class="link link-ref link-reveal xref-bibr">13</a></sup> have been investigated, but its pathogenicity has not yet been fully elucidated. Three <em>Pre. intermedia</em> genome sequences, namely those from strains 17, ATCC25611 and ZT, are available, and an intra- and inter-species genomic comparison was previously performed to obtain information regarding the conservation and variation in the gene contents.<sup><a reveal-id="DSV032C14" data-open="DSV032C14" class="link link-ref link-reveal xref-bibr">14</a></sup> However, the complete genome sequence, which may provide more precise information on the genome structure, gene content, mobile genetic elements (MGEs) ,and repetitive elements and therefore a deeper understanding of the evolutionary process of bacteria, has only been determined for strain 17.</p><p>In this study, we determined the complete sequences of two chromosomes in <em>Pre. intermedia</em> strain OMA14 and performed a fine comparison of the fully sequenced genomes of strains OMA14 and 17. Our analysis revealed an unusual amplification of conjugative and mobilizable transposons (CTns and MTns) and identified a novel repeat specific to a sublineage of the genus <em>Prevotella</em>, which was named <em>‘<span class="underline">P</span>re. <span class="underline">i</span>ntermedia–<span class="underline">n</span>igrescens group-</em><span class="underline">s</span>pecific <span class="underline">r</span>epeat’. These MGEs and repeats were found to be deeply involved in the genomic rearrangements and the generation of the variability in the gene contents detected between the two genomes, particularly those associated with the second chromosome. These data will expand our understanding of the genetic features of <em>Pre. intermedia</em> and provide novel insights into the evolutionary process through which this bacterium has adapted to and survive in the stressful and variable oral habitat.</p>                            <h2 scrollto-destination=87196627 id="87196627" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87196628 id="87196628" class="section-title" data-legacy-id=s2a>2.1. Genome sequencing and sequence analysis</h3>
<p>The strains used in this study are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. <em>Prevotella intermedia</em> OMA14 was isolated from a periodontal pocket of a Japanese patient with periodontitis, and strain 17 was also isolated from a human periodontal pocket.<sup><a reveal-id="DSV032C15" data-open="DSV032C15" class="link link-ref link-reveal xref-bibr">15</a></sup> The <em>Prevotella</em> strains were cultured anaerobically (10% CO<sub>2</sub>, 10% H<sub>2</sub>, and 80% N<sub>2</sub>) on trypto-soya agar (Nissui, Tokyo, Japan) supplemented with 0.5% brain–heart infusion, 0.1% cysteine, 5 μg/ml hemin, and 0.5 μg/ml menadione. Cells were harvested after 1 day of culture on trypto-soya agar plates to prepare genomic DNA as described previously.<sup><a reveal-id="DSV032C16" data-open="DSV032C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p><p>The <em>Pre. intermedia</em> OMA14 genome was sequenced using the Roche 454 GS FLX titanium platform (Roche Diagnostics, Indianapolis, IN, USA). In total, 363,377 single-end reads (150 Mb) were assembled into 85 contigs using the GS Assembler software version 2.6. For gap closing, a fosmid library was constructed using a Copy Control Fosmid Library Production kit (Epicentre, Madison, WI, USA) according to the manufacturer's instructions. Gaps were closed through the sequencing of gap-spanning PCR products and fosmid clones using an ABI3130xl DNA sequencer (Applied Biosystems, Foster City, CA, USA). To correct sequencing errors, the OMA14 genome was resequenced using the SOLiD (Applied Biosystems) and MiSeq (Illumina, San Diego, CA, USA) platforms. The candidate sequence errors were then confirmed by Sanger sequencing. The accuracy of sequence assembly was confirmed by optical mapping (Hitachi Solutions, Ltd, Tokyo, Japan).</p><p>Protein-coding sequences (CDSs) were identified and annotated using the Microbial Genome Annotation Pipeline (<a class="link link-uri" href="http://www.migap.org/">http://www.migap.org/</a>)<sup><a reveal-id="DSV032C17" data-open="DSV032C17" class="link link-ref link-reveal xref-bibr">17</a></sup> followed by manual curation using additional information obtained by a search of the Pfam protein family database (pfam.xfam.org).<sup><a reveal-id="DSV032C18" data-open="DSV032C18" class="link link-ref link-reveal xref-bibr">18</a></sup> CDS numbers headed by ‘PIOMA14_I_’ or ‘PIOMA14_II_’ were assigned to each CDS on the large and small chromosomes, respectively. The Ori-Finder program<sup><a reveal-id="DSV032C19" data-open="DSV032C19" class="link link-ref link-reveal xref-bibr">19</a></sup> was used to find <em>oriC</em> using the <em>Escherichia coli</em> DnaA boxes and one unmatch site option. The GenomeMatcher program<sup><a reveal-id="DSV032C20" data-open="DSV032C20" class="link link-ref link-reveal xref-bibr">20</a></sup> was used to define the conserved genomic regions, inversions, and translocations between the two genomes using the bla2seq comparison at sensitivity 500 and 95% nucleotide identity threshold. The CD-HIT Suite for Biological Sequence Clustering and Comparison (<a class="link link-uri" href="http://weizhong-lab.ucsd.edu/cdhit_suite/cgi-bin/index.cgi">http://weizhong-lab.ucsd.edu/cdhit_suite/cgi-bin/index.cgi</a>)<sup><a reveal-id="DSV032C21" data-open="DSV032C21" class="link link-ref link-reveal xref-bibr">21</a></sup> was used to identify orthologous CDS clusters (with cut-offs of &gt;60% minimal alignment coverage and &gt;90% sequence identity). The annotated genome sequences of strain OMA14 have been deposited into the DDBJ/NCBI/EMBL database under the Accession Numbers AP014597 and AP014598.</p>                            <h3 scrollto-destination=87196633 id="87196633" class="section-title" data-legacy-id=s2b>2.2. PCR assay for the excision of CTns and MTns</h3>
<p>Primer pairs were designed to amplify the attachment site-containing regions of the excised and circularized forms of each CTn and MTn (we refer to attachment sites on CTns and MTns as ‘<em>attTn</em>’ in this manuscript) and the <em>attB</em> regions of each cast-off genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). PCR amplification was performed with 100 ng of genomic DNA and an LA Taq PCR kit (Takara Bio, Otsu, Japan) according to the manufacturer's instructions using the following temperature program: 94°C for 1 min and 30 cycles of 94°C for 20 s, 55°C for 30 s, and 68°C for 2 min. The amplified fragments were subjected to direct sequencing on an ABI PRISM 3130xl sequencer to determine the nucleotide sequences of the <em>attTn-</em> or <em>attB</em>-flanking regions.</p>                            <h3 scrollto-destination=87196635 id="87196635" class="section-title" data-legacy-id=s2c>2.3. <em>Prevotella intermedia/nigrescens</em> group-specific repeat-specific PCR</h3>
<p>The <em>Pre. <span class="underline">i</span>ntermedia/<span class="underline">n</span>igrescens</em> group-<span class="underline">s</span>pecific <span class="underline">r</span>epeat (PINSR)-specific primers PINSR-F and PINSR-R (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>) were designed from the consensus sequence of PINSR. The expected product length was 149 bp. PCR amplification was performed with 50 ng of genomic DNA and an Ex Taq PCR kit (Takara Bio) using the following temperature program: 94°C for 1 min and 30 cycles of 94°C for 20 s, 51°C for 30 s, and 72°C for 30 s. The amplified products were detected by agarose gel electrophoresis followed by ethidium bromide straining. As a control, the 16S-rRNA gene was amplified using the primers 16S-rRNA-10F and 16S-rRNA-800 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>).</p>                            <h2 scrollto-destination=87196638 id="87196638" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87196640 id="87196640" class="section-title" data-legacy-id=s3a>3.1. General features of the <em>Pre. intermedia</em> genomes</h3>
<p>Similar to the genome of <em>Pre. intermedia</em> strain 17 and the other fully sequenced genomes of <em>Prevotella</em> species (<em>Prevotella denticola</em> and <em>Preotella melaninogenica</em>), the genome of OMA14 consisted of two circular chromosomes with sizes of 2,280,262 and 867,855 bp (Fig. <a reveal-id="DSV032F1" data-open="DSV032F1" class="link link-reveal link-table xref-fig">1</a> and Table <a reveal-id="DSV032TB1" data-open="DSV032TB1" class="link link-reveal link-table xref-fig">1</a>). The large and small chromosomes of strain 17 were designated Chromosome II and Chromosome I (Accession No. NC_017861, NC_017860), respectively. However, to avoid confusion, we refer to the large and small chromosomes of both strains in this manuscript as the first and second chromosomes, respectively. The first nucleotide positions of the first and second chromosomes were assigned to the first nucleotides of the <em>dnaA</em> gene (PIOMA14_I_0001) and the initiation factor 3 gene (PIOMA14_II_0001), respectively. The first chromosome of OMA14 is larger than that of strain 17 (2,280 kb versus. 2,013 kb), and the second chromosome showed a more striking variation in size, namely 868 kb in OMA14 and 579 kb in strain 17 (Table <a reveal-id="DSV032TB1" data-open="DSV032TB1" class="link link-reveal link-table xref-fig">1</a> and Fig. <a reveal-id="DSV032F1" data-open="DSV032F1" class="link link-reveal link-table xref-fig">1</a>). This variation is mainly due to the insertion of more MGEs into the second chromosome of OMA14 than into that of strain 17, as described in the next section. In both strains, although clear GC skew transitions were not detected in the second chromosome, clear transitions were observed at two loci located at opposite sites on the first chromosome. These two loci are located at analogous regions between the two strains. One of the loci (positions 1,498,452–1,498,927 in OMA14) contains multiple DnaA boxes and was predicted to be <em>oriC</em> of the first chromosome by Ori-Finder. Thus, this locus and the opposite locus most likely represent the replication origin and terminus of the first chromosome, respectively, but the <em>dnaA</em> gene is not located near the former locus. We have confirmed in the phylum <em>Bacteroidetes</em>, the <em>oriC</em> region is commonly located far from the <em>dnaA</em> gene (DoriC: an updated database of bacterial and archaeal replication origins, <a class="link link-uri" href="http://tubic.tju.edu.cn/doric/">http://tubic.tju.edu.cn/doric/</a>).<sup><a reveal-id="DSV032C22" data-open="DSV032C22" class="link link-ref link-reveal xref-bibr">22</a></sup> PIOMA14_II_0001 on the second chromosome encodes a protein belonging to the Rep_3 superfamily. This may suggest a possibility that the second chromosome has a plasmid-type replication system. But, we did not find any genes related to plasmid maintenance and iteron-like repeat sequences in the vicinity of this CDS. Therefore, the replication origin and terminus of the second chromosome are currently unknown. </p>                        <a id="87196643" scrollto-destination="87196643"></a>
<div content-id="DSV032TB1" class="table-modal"><div class="table-wrap"><div id="DSV032TB1" class="table-wrap-title" data-id="DSV032TB1"><div class="label -label">Table 1.</div><div class="caption"><p>General genomic features of <em>Prevotella intermedia</em> strains OMA14 and 17</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain </th><th colspan="3">OMA14<hr /> </th><th colspan="3">17<hr /> </th></tr><tr><th>Chromosome </th><th>First </th><th>Second </th><th>Total </th><th>First </th><th>Second </th><th>Total </th></tr></thead><tbody><tr><td>Size (bp) </td><td>2,280,262 </td><td>867,855 </td><td>3,148,117 </td><td>2,119,790 </td><td>579,647 </td><td>2,699,437 </td></tr><tr><td>CDSs (strain-specific CDSs) </td><td>1,996 (24.7%) </td><td>801 (54.4%) </td><td>2,797 (33.2%) </td><td>1,796 (18.7%) </td><td>470 (27.2%) </td><td>2,266 (20.4%) </td></tr><tr><td>rRNA operons </td><td>3 </td><td>1 </td><td>4 </td><td>3 </td><td>1 </td><td>4 </td></tr><tr><td>tRNA genes </td><td>37 </td><td>11 </td><td>48 </td><td>38 </td><td>12 </td><td>50 </td></tr><tr><td>IS elements </td><td>8 (2) </td><td>6 (3) </td><td>14 (5) </td><td>17 </td><td>10 (2) </td><td>27 (2) </td></tr><tr><td>CTns </td><td>2 </td><td>4 (2) </td><td>6 (2) </td><td>1 </td><td>1 (1) </td><td>2 (1) </td></tr><tr><td>MTns </td><td>10 (2) </td><td>5 (2) </td><td>15 (4) </td><td>5 (2) </td><td>2 (1) </td><td>7 (3) </td></tr><tr><td>PINSRs </td><td>120 </td><td>57 </td><td>177 </td><td>122 </td><td>62 </td><td>184 </td></tr></tbody></table></div><div content-id="DSV032TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain </th><th colspan="3">OMA14<hr /> </th><th colspan="3">17<hr /> </th></tr><tr><th>Chromosome </th><th>First </th><th>Second </th><th>Total </th><th>First </th><th>Second </th><th>Total </th></tr></thead><tbody><tr><td>Size (bp) </td><td>2,280,262 </td><td>867,855 </td><td>3,148,117 </td><td>2,119,790 </td><td>579,647 </td><td>2,699,437 </td></tr><tr><td>CDSs (strain-specific CDSs) </td><td>1,996 (24.7%) </td><td>801 (54.4%) </td><td>2,797 (33.2%) </td><td>1,796 (18.7%) </td><td>470 (27.2%) </td><td>2,266 (20.4%) </td></tr><tr><td>rRNA operons </td><td>3 </td><td>1 </td><td>4 </td><td>3 </td><td>1 </td><td>4 </td></tr><tr><td>tRNA genes </td><td>37 </td><td>11 </td><td>48 </td><td>38 </td><td>12 </td><td>50 </td></tr><tr><td>IS elements </td><td>8 (2) </td><td>6 (3) </td><td>14 (5) </td><td>17 </td><td>10 (2) </td><td>27 (2) </td></tr><tr><td>CTns </td><td>2 </td><td>4 (2) </td><td>6 (2) </td><td>1 </td><td>1 (1) </td><td>2 (1) </td></tr><tr><td>MTns </td><td>10 (2) </td><td>5 (2) </td><td>15 (4) </td><td>5 (2) </td><td>2 (1) </td><td>7 (3) </td></tr><tr><td>PINSRs </td><td>120 </td><td>57 </td><td>177 </td><td>122 </td><td>62 </td><td>184 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The numbers in parentheses indicate the copy numbers of partial elements.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196643" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV032TB1" class="table-wrap-title" data-id="DSV032TB1"><div class="label -label">Table 1.</div><div class="caption"><p>General genomic features of <em>Prevotella intermedia</em> strains OMA14 and 17</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain </th><th colspan="3">OMA14<hr /> </th><th colspan="3">17<hr /> </th></tr><tr><th>Chromosome </th><th>First </th><th>Second </th><th>Total </th><th>First </th><th>Second </th><th>Total </th></tr></thead><tbody><tr><td>Size (bp) </td><td>2,280,262 </td><td>867,855 </td><td>3,148,117 </td><td>2,119,790 </td><td>579,647 </td><td>2,699,437 </td></tr><tr><td>CDSs (strain-specific CDSs) </td><td>1,996 (24.7%) </td><td>801 (54.4%) </td><td>2,797 (33.2%) </td><td>1,796 (18.7%) </td><td>470 (27.2%) </td><td>2,266 (20.4%) </td></tr><tr><td>rRNA operons </td><td>3 </td><td>1 </td><td>4 </td><td>3 </td><td>1 </td><td>4 </td></tr><tr><td>tRNA genes </td><td>37 </td><td>11 </td><td>48 </td><td>38 </td><td>12 </td><td>50 </td></tr><tr><td>IS elements </td><td>8 (2) </td><td>6 (3) </td><td>14 (5) </td><td>17 </td><td>10 (2) </td><td>27 (2) </td></tr><tr><td>CTns </td><td>2 </td><td>4 (2) </td><td>6 (2) </td><td>1 </td><td>1 (1) </td><td>2 (1) </td></tr><tr><td>MTns </td><td>10 (2) </td><td>5 (2) </td><td>15 (4) </td><td>5 (2) </td><td>2 (1) </td><td>7 (3) </td></tr><tr><td>PINSRs </td><td>120 </td><td>57 </td><td>177 </td><td>122 </td><td>62 </td><td>184 </td></tr></tbody></table></div><div content-id="DSV032TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain </th><th colspan="3">OMA14<hr /> </th><th colspan="3">17<hr /> </th></tr><tr><th>Chromosome </th><th>First </th><th>Second </th><th>Total </th><th>First </th><th>Second </th><th>Total </th></tr></thead><tbody><tr><td>Size (bp) </td><td>2,280,262 </td><td>867,855 </td><td>3,148,117 </td><td>2,119,790 </td><td>579,647 </td><td>2,699,437 </td></tr><tr><td>CDSs (strain-specific CDSs) </td><td>1,996 (24.7%) </td><td>801 (54.4%) </td><td>2,797 (33.2%) </td><td>1,796 (18.7%) </td><td>470 (27.2%) </td><td>2,266 (20.4%) </td></tr><tr><td>rRNA operons </td><td>3 </td><td>1 </td><td>4 </td><td>3 </td><td>1 </td><td>4 </td></tr><tr><td>tRNA genes </td><td>37 </td><td>11 </td><td>48 </td><td>38 </td><td>12 </td><td>50 </td></tr><tr><td>IS elements </td><td>8 (2) </td><td>6 (3) </td><td>14 (5) </td><td>17 </td><td>10 (2) </td><td>27 (2) </td></tr><tr><td>CTns </td><td>2 </td><td>4 (2) </td><td>6 (2) </td><td>1 </td><td>1 (1) </td><td>2 (1) </td></tr><tr><td>MTns </td><td>10 (2) </td><td>5 (2) </td><td>15 (4) </td><td>5 (2) </td><td>2 (1) </td><td>7 (3) </td></tr><tr><td>PINSRs </td><td>120 </td><td>57 </td><td>177 </td><td>122 </td><td>62 </td><td>184 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>The numbers in parentheses indicate the copy numbers of partial elements.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87196643" target="_blank">View Large</a></div></div>                        <a id="87196645" scrollto-destination="87196645"></a>
<div id="DSV032F1" data-id="DSV032F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/m_dsv03201.png?Expires=2147483647&Signature=niPWClsoGADnHLWXcY4vgkI8QhcqifLSdLyekHf-D9TzmHyWgkdrPnbwXm7yuc79Mbm0pLZt85PMvt~pES2eMoGRcpul1mmbk4euKIr3nHSqH214b9Teb7DNCP-MzSPZ~wwdVmtml0nfbYEMHuK5e-htiA8t8IaK1lTBY~UDbR1gZQAhgaIlHhZ0f8LCHE2yydM6t57-sI7P7aI~WsQcR9AH6sWK4YZynbwn1xa6uGtMtyjjC87H~5w51jv7ceqdH3fzsx2Q4kdMITwzOCICauNGNaRQyT8ntFUev~hj6QO0ByEMWoFCOsc1sdlRS-nXrKMBbtEOY~64fuh0-kmmPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circular maps of the chromosomes of Prevotella intermedia strains OMA14 and 17. The chromosomes of Pre. intermedia strains OMA14 and 17 are shown in A and B, respectively. The first and second circles (counted from the outside in) indicate CDSs on the plus and minus strands, respectively. The conserved and strain-specific CDSs are indicated in blue and red, respectively. The third to fifth circles show rRNA operons (red), tRNA genes (black), and MGEs (green, IS elements; purple, CTn; orange, MTn), respectively. In the sixth circle, large strain-specific segments are indicated in grey (&gt;10 kb). The seventh circle shows PINSR (black). The GC skew [(G − C)/(G + C)] and GC content are shown in the eighth and ninth circles, respectively. Note that the dnaA genes of both strains are placed at the top of the first chromosome circle so that the three rrn operons of the two strains are located at analogous positions in this circular map presentation." path-from-xml="dsv03201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196645/dsv03201.tiff" path-from-xml="dsv03201.tiff" target="_blank">View large</a><a section="87196645" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv03201.png?Expires=2147483647&Signature=oAW9wBQzJC6eEBt0j25cDMhAWz2JNqLUnuk5sF-GatIzsL5L-ekjoKgI5WIhDvWs0yfurL87T5JuBvtPqC5FvL1H7DiCR4NqeeUvRsR~q618eLEBWq1euFMrSKgj5FYMUjnRY5X9RNZaqFaSXppBuq7C85zxVsXkfyjHcwlG35t113U0fd7QW1W4woenhKYqQ99cBJZ9uI2mOZ69lV9ufEhbblMQoF5qDtIIeNNZH4S2RqParg8Wu4Y-4W1B76HREF0x15Bqcw~aoK6FE0UPn-nHDUFZGZ9kxYPl0tt-uNSGDe2VBZhHxoXdKmxzQfmHmS0tOuvBbEzqXRkCho6LOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196645&ar=2462413&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circular maps of the chromosomes of <em>Prevotella intermedia</em> strains OMA14 and 17. The chromosomes of <em>Pre. intermedia</em> strains OMA14 and 17 are shown in A and B, respectively. The first and second circles (counted from the outside in) indicate CDSs on the plus and minus strands, respectively. The conserved and strain-specific CDSs are indicated in blue and red, respectively. The third to fifth circles show rRNA operons (red), tRNA genes (black), and MGEs (green, IS elements; purple, CTn; orange, MTn), respectively. In the sixth circle, large strain-specific segments are indicated in grey (&gt;10 kb). The seventh circle shows PINSR (black). The GC skew [(G − C)/(G + C)] and GC content are shown in the eighth and ninth circles, respectively. Note that the <em>dnaA</em> genes of both strains are placed at the top of the first chromosome circle so that the three <em>rrn</em> operons of the two strains are located at analogous positions in this circular map presentation.</p></div></div><div content-id="DSV032F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/m_dsv03201.png?Expires=2147483647&Signature=niPWClsoGADnHLWXcY4vgkI8QhcqifLSdLyekHf-D9TzmHyWgkdrPnbwXm7yuc79Mbm0pLZt85PMvt~pES2eMoGRcpul1mmbk4euKIr3nHSqH214b9Teb7DNCP-MzSPZ~wwdVmtml0nfbYEMHuK5e-htiA8t8IaK1lTBY~UDbR1gZQAhgaIlHhZ0f8LCHE2yydM6t57-sI7P7aI~WsQcR9AH6sWK4YZynbwn1xa6uGtMtyjjC87H~5w51jv7ceqdH3fzsx2Q4kdMITwzOCICauNGNaRQyT8ntFUev~hj6QO0ByEMWoFCOsc1sdlRS-nXrKMBbtEOY~64fuh0-kmmPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circular maps of the chromosomes of Prevotella intermedia strains OMA14 and 17. The chromosomes of Pre. intermedia strains OMA14 and 17 are shown in A and B, respectively. The first and second circles (counted from the outside in) indicate CDSs on the plus and minus strands, respectively. The conserved and strain-specific CDSs are indicated in blue and red, respectively. The third to fifth circles show rRNA operons (red), tRNA genes (black), and MGEs (green, IS elements; purple, CTn; orange, MTn), respectively. In the sixth circle, large strain-specific segments are indicated in grey (&gt;10 kb). The seventh circle shows PINSR (black). The GC skew [(G − C)/(G + C)] and GC content are shown in the eighth and ninth circles, respectively. Note that the dnaA genes of both strains are placed at the top of the first chromosome circle so that the three rrn operons of the two strains are located at analogous positions in this circular map presentation." path-from-xml="dsv03201.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196645/dsv03201.tiff" path-from-xml="dsv03201.tiff" target="_blank">View large</a><a section="87196645" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv03201.png?Expires=2147483647&Signature=oAW9wBQzJC6eEBt0j25cDMhAWz2JNqLUnuk5sF-GatIzsL5L-ekjoKgI5WIhDvWs0yfurL87T5JuBvtPqC5FvL1H7DiCR4NqeeUvRsR~q618eLEBWq1euFMrSKgj5FYMUjnRY5X9RNZaqFaSXppBuq7C85zxVsXkfyjHcwlG35t113U0fd7QW1W4woenhKYqQ99cBJZ9uI2mOZ69lV9ufEhbblMQoF5qDtIIeNNZH4S2RqParg8Wu4Y-4W1B76HREF0x15Bqcw~aoK6FE0UPn-nHDUFZGZ9kxYPl0tt-uNSGDe2VBZhHxoXdKmxzQfmHmS0tOuvBbEzqXRkCho6LOA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196645&ar=2462413&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03201.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circular maps of the chromosomes of <em>Prevotella intermedia</em> strains OMA14 and 17. The chromosomes of <em>Pre. intermedia</em> strains OMA14 and 17 are shown in A and B, respectively. The first and second circles (counted from the outside in) indicate CDSs on the plus and minus strands, respectively. The conserved and strain-specific CDSs are indicated in blue and red, respectively. The third to fifth circles show rRNA operons (red), tRNA genes (black), and MGEs (green, IS elements; purple, CTn; orange, MTn), respectively. In the sixth circle, large strain-specific segments are indicated in grey (&gt;10 kb). The seventh circle shows PINSR (black). The GC skew [(G − C)/(G + C)] and GC content are shown in the eighth and ninth circles, respectively. Note that the <em>dnaA</em> genes of both strains are placed at the top of the first chromosome circle so that the three <em>rrn</em> operons of the two strains are located at analogous positions in this circular map presentation.</p></div></div><p>The OMA14 genome contains 2,797 CDSs, which is a greater number than that found in strain 17, reflecting the difference in genome size observed between the strains. In both strains, three and one rRNA operons are present in the first and second chromosomes, respectively. A full set of tRNA genes specifying all 20 amino acids was identified in both strains, i.e. 48 and 50 genes in strains OMA14 and 17, respectively. These tRNA genes are distributed to both chromosomes, but the tRNA<sup>Ser</sup> genes (two copies) are encoded only on the second chromosome in both strains. A similar, small chromosome-specific distribution of specific tRNA genes has been observed in other <em>Prevotella</em> species with two chromosomes: tRNA<sup>Cys</sup> and tRNA<sup>Lys</sup> in <em>Pre. dentalis</em> (Accession No. NC_019960, NC_019968) and tRNA<sup>Asp</sup>, tRNA<sup>Glu</sup>, and tRNA<sup>Ser</sup> in <em>Pre. melaninogenica</em> (Accession No. NC_014370, NC_014371). In addition to the rRNA operons and tRNA genes, the second chromosome of <em>Pre. ntermedia</em> contains many CDSs for house-keeping functions, including multiple ribosomal proteins (S1, L27, L28, and L33), multiple DNA polymerase III subunits (beta, gamma/tau, and epsilon), the RecA and RecN proteins, NADH dehydrogenase, NADH : ubiquinone oxidoreductase, prolyl-tRNA synthetase, heat-shock protein 90, and transcription termination factor Rho. Our preliminary analysis of other fully sequenced <em>Prevotella</em> species, <em>Pre. dentalis and Pre. melaninogenica</em>, indicate that many essential house-keeping genes are located in their second (smaller) chromosomes (ribosomal protein genes, chaperon protein genes, and many others). These data indicate that in <em>Prevotella</em> species, genes that are essentially required for cell growth are distributed in both the large and small chromosomes.</p><p>Clustered regularly interspaced short palindromic repeats (CRISPRs) were also found in both chromosomes but only the locus on the first chromosome contains CRISPR-associated genes. In OMA14, the numbers of repeats in the repeat/space regions of the CRISPR-47-29 locus on the first chromosome and CRISPR 36-30 locus on the second chromosome were 33 and 6, respectively, whereas the analogous regions in strain 17 contain 38 and 17 repeats, respectively. Notably the <em>cas1</em> gene of strain OMA14 has been disrupted by the insertion of a 34.2-kb phage-related segment, resulting in the apparent inactivation of the CRISPR system in OMA14. Smaller numbers of repeats in both loci of strain OMA14 (compared with strain 17) may be associated with the inactivation of the CRISPR system in this strain.</p>                            <h3 scrollto-destination=87196652 id="87196652" class="section-title" data-legacy-id=s3b>3.2. Mobile genetic elements</h3>
<p>One of the notable genomic features of <em>Pre. intermedia</em> is the abundance of MGEs, particularly CTns and <em>MTns</em> (Table <a reveal-id="DSV032TB1" data-open="DSV032TB1" class="link link-reveal link-table xref-fig">1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3–S5</a></span>). Thus, we conducted a detailed analysis of these MGEs in strains OMA14 and 17.</p>                            <h4 scrollto-destination=87196655 id="87196655" class="section-title" data-legacy-id=s3b1>3.2.1. IS elements</h4>
<p>In total, 14 and 27 IS elements (including partial copies) were identified in strains OMA14 and 17, respectively. These elements were classified into two types, namely IS<em>Pi1</em> and IS<em>Pi2,</em> and both types are distributed to both chromosomes and are more abundant in strain 17 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3 and S4</a></span>). IS<em>Pi1</em> is closely related to IS<em>Pg3</em> found in <em>Por. gingivalis</em>, sharing the same inverted repeat sequences and generating the same length of target sequence duplication (7 bp) upon transposition. IS<em>Pi2</em> was assigned to the IS<em>4</em> family by IS finder (<a class="link link-uri" href="https://www-is.biotoul.fr/">https://www-is.biotoul.fr/</a>)<sup><a reveal-id="DSV032C23" data-open="DSV032C23" class="link link-ref link-reveal xref-bibr">23</a></sup> using the tblastx program and default settings.</p>                            <h4 scrollto-destination=87196660 id="87196660" class="section-title" data-legacy-id=s3b2>3.2.2. CTn and MTn</h4>
<p>OMA14 contains 6 CTns and 15 MTns, whereas strain 17 possesses 2 CTns and 7 MTns (including highly degraded MTns; Fig. <a reveal-id="DSV032F2" data-open="DSV032F2" class="link link-reveal link-table xref-fig">2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3 and S5</a></span>). These transposons account for 14.6 and 4.8% of the total genomes of OMA14 and strain 17, respectively. In addition, CTns and MTns occupy large portions (20.5%) of the second chromosome, particularly in OMA14, which is one of the main driving forces responsible for the marked difference in the size of the second chromosomes between the two strains. The CTns and MTns identified were classified into four (CTnPi1–CTnPi4) and 10 types (MTnPi1–MTnPi10), respectively. Among these, one type of CTn and two types of MTns are shared by the two strains, suggesting that these CTn and MTns may be widely distributed among <em>Pre. intermedia</em> strains. Several copies found in OMA14 (CTnPi2-a and -b; MTnPi1-a and -b; MTnPi2-a, -b, -c, and -d; and MTnPi3-a, -b, and -c) are nearly identical and thus appear to have been very recently duplicated (Fig. <a reveal-id="DSV032F2" data-open="DSV032F2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="87196664" scrollto-destination="87196664"></a>
<div id="DSV032F2" data-id="DSV032F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/m_dsv03202.png?Expires=2147483647&amp;Signature=K-yMULgyePC3fa5f8yxw2OVtZXkd22ZTtgofc9huBmpdVvvvOVZIsgG8wFy8wZalJDN8ne7JPfy5uk34RS6elt7wAd86P~oq~EpmZfU5RhlG-4Euh1pkrGI76BYhMu4Yw1iriP74dC~3tPN6mNnIhO3jJJGJj0Vw29q7gAnT5-5N8LO6JD0Jx-wGbtuA-M~ss8I9osg9uS8ewrZqa48RTVkMsGQ~N0bo-J1NJtGxULhVIsjs65TECN5nemvRO56xRj-H2NS8RcdMt-asBvY~IKTGpnvrHmMjuVzd9amqObe3a-sW64k~QBo9kEck4MG5Ur-V~A4Oh8s2Sb~2dN9CQg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CTns and MTns identified in the genomes of strains OMA14 and 17. The gene organizations of all CTn and MTn copies identified in the two strains are shown (CTns in A and MTns in B). Some CTns or MTns (indicated by underlining) were found in strain 17, and others were found in strain OMA14. The CDSs are depicted by arrows (striped arrow, genes related to conjugation or mobilization; black arrow, integrase genes; grey arrow, other functionally annotated CDSs; and white arrow, genes for hypothetical proteins). The black triangles indicate the att sites of each CTn and MTn. Three att sequences (attL of MTnPi6-a, attL of MTnPi7, and attR of MTnPi9) were predicted based on the sequence homology to the consensus att core sequence of MTnPi1, MTnPi2, and MTnPi3 (see Supplementary Table S3). CTn and MTn transposons lacking either one of att sequences can be regarded as the remnants. The grey shading indicates homologous CDSs with &gt;70% amino acid sequence identity. The CDS groups indicated by asterisks, open circles, and black circles exhibited &gt;50% identity among the group. T4CP, coupling protein T4PC; rel, relaxase; pri, primase family protein; exc, excisionase; tetR, TetR family transcriptional regulator; Lux, LuxR family transcriptional regulator. A double line indicates a degraded integrase gene." path-from-xml="dsv03202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196664/dsv03202.tiff" path-from-xml="dsv03202.tiff" target="_blank">View large</a><a section="87196664" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv03202.png?Expires=2147483647&amp;Signature=ripim1ILmU9Joe5a-p6jIvAmvwK4p0spbljS~XEMef-5OBy0apnRtgYgRDbjvmq0xWOyqcDu0f1iPlYWfYkOzsnBZUWNfpng2OIwA7Km267gpN1v2dSxCfkTDme4u4MOyRsnMmLX8vw2KB7uPRtp-~rLL35s2it-j8zAZmmm1aJLJVI~~rTHouTibCJs~y-R~SgH-DRKz0eGhkwa3OOX4WulOxr9WA0Ce5pIlTyS6v3-gNQxQ7nZ999E~Ow6TzIdkTvpM2uB7AoFw7bRaCcXbs9JCzltxVmEg55eCeKsiFHGihusf7xXj0GfHnWAlFuXCcG6KOVqZYsHb~Sg~zGwwQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196664&amp;ar=2462413&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsv03202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CTns and MTns identified in the genomes of strains OMA14 and 17. The gene organizations of all CTn and MTn copies identified in the two strains are shown (CTns in A and MTns in B). Some CTns or MTns (indicated by underlining) were found in strain 17, and others were found in strain OMA14. The CDSs are depicted by arrows (striped arrow, genes related to conjugation or mobilization; black arrow, integrase genes; grey arrow, other functionally annotated CDSs; and white arrow, genes for hypothetical proteins). The black triangles indicate the <em>att</em> sites of each CTn and MTn. Three <em>att</em> sequences (<em>attL</em> of MTnPi6-a, <em>attL</em> of MTnPi7, and <em>attR</em> of MTnPi9) were predicted based on the sequence homology to the consensus <em>att</em> core sequence of MTnPi1, MTnPi2, and MTnPi3 (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). CTn and MTn transposons lacking either one of <em>att</em> sequences can be regarded as the remnants. The grey shading indicates homologous CDSs with &gt;70% amino acid sequence identity. The CDS groups indicated by asterisks, open circles, and black circles exhibited &gt;50% identity among the group. T4CP, coupling protein T4PC; rel, relaxase; pri, primase family protein; exc, excisionase; tetR, TetR family transcriptional regulator; Lux, LuxR family transcriptional regulator. A double line indicates a degraded integrase gene.</p></div></div><div content-id="DSV032F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/m_dsv03202.png?Expires=2147483647&amp;Signature=K-yMULgyePC3fa5f8yxw2OVtZXkd22ZTtgofc9huBmpdVvvvOVZIsgG8wFy8wZalJDN8ne7JPfy5uk34RS6elt7wAd86P~oq~EpmZfU5RhlG-4Euh1pkrGI76BYhMu4Yw1iriP74dC~3tPN6mNnIhO3jJJGJj0Vw29q7gAnT5-5N8LO6JD0Jx-wGbtuA-M~ss8I9osg9uS8ewrZqa48RTVkMsGQ~N0bo-J1NJtGxULhVIsjs65TECN5nemvRO56xRj-H2NS8RcdMt-asBvY~IKTGpnvrHmMjuVzd9amqObe3a-sW64k~QBo9kEck4MG5Ur-V~A4Oh8s2Sb~2dN9CQg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="CTns and MTns identified in the genomes of strains OMA14 and 17. The gene organizations of all CTn and MTn copies identified in the two strains are shown (CTns in A and MTns in B). Some CTns or MTns (indicated by underlining) were found in strain 17, and others were found in strain OMA14. The CDSs are depicted by arrows (striped arrow, genes related to conjugation or mobilization; black arrow, integrase genes; grey arrow, other functionally annotated CDSs; and white arrow, genes for hypothetical proteins). The black triangles indicate the att sites of each CTn and MTn. Three att sequences (attL of MTnPi6-a, attL of MTnPi7, and attR of MTnPi9) were predicted based on the sequence homology to the consensus att core sequence of MTnPi1, MTnPi2, and MTnPi3 (see Supplementary Table S3). CTn and MTn transposons lacking either one of att sequences can be regarded as the remnants. The grey shading indicates homologous CDSs with &gt;70% amino acid sequence identity. The CDS groups indicated by asterisks, open circles, and black circles exhibited &gt;50% identity among the group. T4CP, coupling protein T4PC; rel, relaxase; pri, primase family protein; exc, excisionase; tetR, TetR family transcriptional regulator; Lux, LuxR family transcriptional regulator. A double line indicates a degraded integrase gene." path-from-xml="dsv03202.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196664/dsv03202.tiff" path-from-xml="dsv03202.tiff" target="_blank">View large</a><a section="87196664" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv03202.png?Expires=2147483647&amp;Signature=ripim1ILmU9Joe5a-p6jIvAmvwK4p0spbljS~XEMef-5OBy0apnRtgYgRDbjvmq0xWOyqcDu0f1iPlYWfYkOzsnBZUWNfpng2OIwA7Km267gpN1v2dSxCfkTDme4u4MOyRsnMmLX8vw2KB7uPRtp-~rLL35s2it-j8zAZmmm1aJLJVI~~rTHouTibCJs~y-R~SgH-DRKz0eGhkwa3OOX4WulOxr9WA0Ce5pIlTyS6v3-gNQxQ7nZ999E~Ow6TzIdkTvpM2uB7AoFw7bRaCcXbs9JCzltxVmEg55eCeKsiFHGihusf7xXj0GfHnWAlFuXCcG6KOVqZYsHb~Sg~zGwwQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196664&amp;ar=2462413&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsv03202.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>CTns and MTns identified in the genomes of strains OMA14 and 17. The gene organizations of all CTn and MTn copies identified in the two strains are shown (CTns in A and MTns in B). Some CTns or MTns (indicated by underlining) were found in strain 17, and others were found in strain OMA14. The CDSs are depicted by arrows (striped arrow, genes related to conjugation or mobilization; black arrow, integrase genes; grey arrow, other functionally annotated CDSs; and white arrow, genes for hypothetical proteins). The black triangles indicate the <em>att</em> sites of each CTn and MTn. Three <em>att</em> sequences (<em>attL</em> of MTnPi6-a, <em>attL</em> of MTnPi7, and <em>attR</em> of MTnPi9) were predicted based on the sequence homology to the consensus <em>att</em> core sequence of MTnPi1, MTnPi2, and MTnPi3 (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). CTn and MTn transposons lacking either one of <em>att</em> sequences can be regarded as the remnants. The grey shading indicates homologous CDSs with &gt;70% amino acid sequence identity. The CDS groups indicated by asterisks, open circles, and black circles exhibited &gt;50% identity among the group. T4CP, coupling protein T4PC; rel, relaxase; pri, primase family protein; exc, excisionase; tetR, TetR family transcriptional regulator; Lux, LuxR family transcriptional regulator. A double line indicates a degraded integrase gene.</p></div></div><p>Both CTnPi1 (found in OMA14) and CTnPi4 (strain 17) belong to the CTnPg1 family, which was originally identified in <em>Por. gingivalis</em>,<sup><a reveal-id="DSV032C16" data-open="DSV032C16" class="link link-ref link-reveal xref-bibr">16</a></sup> and share a set of backbone genes for integration- and conjugal transfer-related functions with other family members, which have been identified in <em>Prevotella</em>, <em>Porphyromonas</em>, and <em>Bacteroides</em> species, all belonging to the order <em>Bacteroidales</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Both CTns, particularly CTnPi4, carry various accessory genes. Some of these genes are distributed in other CTnPg1 family members, e.g. genes for an ABC transporter component and a glycine C-acetyltransferase (CTnPi1) in the family members found in <em>Por. gingivalis</em> TDC60 and <em>Pre. endodontalis</em> ATCC 35406 and genes encoding an ABC transporter component, a TonB-dependent receptor, and several other proteins (CTnPi4) in the members found in <em>Pre. nigrescens</em> ATCC 33563 and <em>Prevotella buccae</em> D17. These results suggest that accessory genes are frequently exchanged between CTnPg1 family members.</p><p>CTnPi2 and CTnPi3, both of which are found in OMA14, are closely related to one another, sharing a conserved backbone encoding integration- and conjugal transfer-related functions, which are partially homologous to those of CTnPi1 and CTnPi4 (Fig. <a reveal-id="DSV032F2" data-open="DSV032F2" class="link link-reveal link-table xref-fig">2</a>A). CTnPi2 and CTnPi3 contain multiple integrase genes in internal regions. There are no putative <em>att</em> sequences just upstream of those internally residing integrase genes, but these integrases contain the motifs conserved in functional tyrosine recombinases. It is possible that the two CTns were generated by the insertion of at least one integrative element similar to MTns, i.e. one element in CTnPi2 and two or three elements in CTnPi3 with one shared by the two CTns. As accessory genes, CTnPi3 carries genes for a type I restriction-modification system and a tetratricopeptide repeat domain-containing protein as well as genes that are shared with CTnPi2 (encoding an ABC transporter component and a TonB-dependent receptor).</p><p>MTns are a group of transposons that contain an <em>oriT</em> sequence and can thus be mobilized by a co-resident conjugative plasmid or transposon.<sup><a reveal-id="DSV032C24" data-open="DSV032C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Among the 10 types of MTns identified in the two strains, only degraded copies were found for several types (MTnPi5, 6, 7, 8, 9, and 10). However, genes for relaxase, a primase family protein, and integrase, which are required for MTn function, were found in most types (Fig. <a reveal-id="DSV032F2" data-open="DSV032F2" class="link link-reveal link-table xref-fig">2</a>B). Because the integrases of all MTns are tyrosine recombinases, they all belong to type I MTn.<sup><a reveal-id="DSV032C24" data-open="DSV032C24" class="link link-ref link-reveal xref-bibr">24</a></sup> MTnPi1, 2, 3, 6, 7, and 9 appear to be related to one another and share similar relaxases, primase family proteins, and integrases. MTnPi4 and 8 and MTnPi5 and 10 appear to form different groups, respectively. A unique feature of this group is the presence of two integrase genes located in tandem, although the second gene (a downstream copy) is missing or degraded in some MTn copies. The two integrase genes in each MTn show no significant homology to one another (see the next subsection for a discussion of the functions of these two integrases). It was previously reported that Tn1806 of <em>Streptococcus pneumonia</em> AP200 also contains two <em>int</em> genes in tandem.<sup><a reveal-id="DSV032C25" data-open="DSV032C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The other four MTns may be classified into two groups: one including ‘MTnPi4 (found in OMA14) and MTnPi8 (strain 17)’, and the other including ‘MTnPi5 (OMA14) and MTnPi-10 (strain 17)’, because each pair shares similar genes for a primase family protein, relaxase, or integrase. Most MTns also carry one or more accessory genes, such as those encoding an ABC transporter component, a LuxR family transcriptional regulator, a subtilase-like protease, a β-lactamase family protein, and an arginase.</p>                            <h4 scrollto-destination=87196681 id="87196681" class="section-title" data-legacy-id=s3b3>3.2.3. Excision assay of CTns and MTns and identification of attachment sequences</h4>
<p>In general, CTns/MTns are integrated into a specific target site (attachment site; <em>att</em>) that contains a short sequence specific to each CTn or MTn. Upon integration, the <em>att</em> sequences are duplicated and form chromosome/CTn (or MTn) boundaries (<em>attL</em> and <em>attR</em>), which are good landmarks for the definition of each CTn/MTn. However, it is not always easy to identify <em>att</em> sequences. Therefore, we used PCR assays to detect the excised and circularized intermediates of each CTn and MTn and thereby examined whether the identified CTns and MTns are excised from the chromosome (Fig. <a reveal-id="DSV032F3" data-open="DSV032F3" class="link link-reveal link-table xref-fig">3</a>A). In addition, through a sequencing analysis of the PCR products, we determined the <em>att</em> sequences of each CTn/MTn (<em>attTn</em> on the excised and circularized forms of transposon and <em>attL</em> and <em>attR</em> at chromosome/transposon boundaries). Through this series of analyses, we found that CTnPi1, 2, 3, and 4 and MTnPi1, 2, 3, and 4 are constitutively excised from the chromosome to form a circular intermediate (Fig. <a reveal-id="DSV032F3" data-open="DSV032F3" class="link link-reveal link-table xref-fig">3</a>B), and we identified their <em>att</em> sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). Among these CTns and MTns, multiple copies of CTnPi2 and MTnPi1, 2, and 3 were found. Because the sequences of <em>attTn</em>-flanking regions in the different copies are identical, it was impossible to determine which copy is excised in each type of transposon. We therefore performed an additional PCR assay to detect the chromosomal attachment sites (<em>attB</em>) of each copy from which CTn/MTn has been excised (Fig. <a reveal-id="DSV032F3" data-open="DSV032F3" class="link link-reveal link-table xref-fig">3</a>A and C). This analysis confirmed that all of the copies, with the exception of one copy of MTnPi1 and one copy of MTnPi3, were excised from the chromosome. These results suggest that these CTns and MTns are likely active. We speculate that they can potentially transfer (or be transferred) to other bacteria, but conjugation assays for each transposon need to confirm their activities. We were unable to detect the excision of the remaining MTns (data not shown), which was expected for six of the MTns (MTnPi5–MTnPi10) due to their high degree of degradation. </p>                        <a id="87196688" scrollto-destination="87196688"></a>
<div id="DSV032F3" data-id="DSV032F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/m_dsv03203.png?Expires=2147483647&Signature=4MxEPCIlMHqIti4Jnsat~RMVl2VIYPN4z5brMFX7tJ1KHQOl8nhe0BokKsW5ISzB4ag~fUcUR1MLyJkAkSudrT~E5YH5KlX9TYbl093FisN7jpT5OsKMvRs43GlPg0P7~rSbOsz~TK4B5UeC8ChIIj1kvWfItSnLHDQwa4hrjKNvv3uzMZFil-ruEpBXL5eTva7bGgCjWYdmiBL5q7SZUL7AQmu~aT8HHTXVM0OdtYAR5osUVqacoX0kZ-n4Bep9Q0kZIkPbWuGtYpL2oPXP6~z1NMl8oM5AsOPv4EwKMCRwkZMpIrAS~6fnnvXrOxEVCxgLeUfl-xyFNEB-kuab~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Excision of CTns and MTns. (A) Schematic representation of the excision of CTns and MTns from the chromosome. Black triangles indicate the att sites. The PCR primers are indicated by arrows. Agarose gel electrophoresis of the PCR products obtained by the primer pairs left/right (B) and up/down (C). The attTn-containing regions in CTnPi-4 and MTnPi1-c were amplified from strain 17. The other regions were amplified from OMA14. attTn indicates the att sequence on the excised and circularized form of transposon." path-from-xml="dsv03203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196688/dsv03203.tiff" path-from-xml="dsv03203.tiff" target="_blank">View large</a><a section="87196688" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv03203.png?Expires=2147483647&Signature=1Ta4UIkiLtyeIwJ7QZn6VhKHRR4IFNFg3Z7iB8Z29TaJdSRpV8Aj5yXvMojIUH1RC~SkLJHrRtdJLsswb5UzLg27o5yG~ACTBb~AawUCInJKib~9KCHd53UgVKyG0d4DuUPjheaDy0ItU7IBTxnFPiPUnffS8Y59XxhXY2RkdNHU3GWdhXQOLKWElxvCZ~6QJIQyxehFvoP8pg72qilYQ~vK912aTBaUislRAlUD03IXNq9YsVS2QiJFi8Oc2KHLPLEGqhgmtoNC4BBhhPAj0lG98Y0YQri0XrOpsddX4qUklpFtol~FivTOByfZQTukJf9hQ85r1lMtAm0~cYHm6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196688&ar=2462413&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Excision of CTns and MTns. (A) Schematic representation of the excision of CTns and MTns from the chromosome. Black triangles indicate the <em>att</em> sites. The PCR primers are indicated by arrows. Agarose gel electrophoresis of the PCR products obtained by the primer pairs left/right (B) and up/down (C). The <em>attTn</em>-containing regions in CTnPi-4 and MTnPi1-c were amplified from strain 17. The other regions were amplified from OMA14. <em>attTn</em> indicates the <em>att</em> sequence on the excised and circularized form of transposon.</p></div></div><div content-id="DSV032F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/m_dsv03203.png?Expires=2147483647&Signature=4MxEPCIlMHqIti4Jnsat~RMVl2VIYPN4z5brMFX7tJ1KHQOl8nhe0BokKsW5ISzB4ag~fUcUR1MLyJkAkSudrT~E5YH5KlX9TYbl093FisN7jpT5OsKMvRs43GlPg0P7~rSbOsz~TK4B5UeC8ChIIj1kvWfItSnLHDQwa4hrjKNvv3uzMZFil-ruEpBXL5eTva7bGgCjWYdmiBL5q7SZUL7AQmu~aT8HHTXVM0OdtYAR5osUVqacoX0kZ-n4Bep9Q0kZIkPbWuGtYpL2oPXP6~z1NMl8oM5AsOPv4EwKMCRwkZMpIrAS~6fnnvXrOxEVCxgLeUfl-xyFNEB-kuab~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Excision of CTns and MTns. (A) Schematic representation of the excision of CTns and MTns from the chromosome. Black triangles indicate the att sites. The PCR primers are indicated by arrows. Agarose gel electrophoresis of the PCR products obtained by the primer pairs left/right (B) and up/down (C). The attTn-containing regions in CTnPi-4 and MTnPi1-c were amplified from strain 17. The other regions were amplified from OMA14. attTn indicates the att sequence on the excised and circularized form of transposon." path-from-xml="dsv03203.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196688/dsv03203.tiff" path-from-xml="dsv03203.tiff" target="_blank">View large</a><a section="87196688" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv03203.png?Expires=2147483647&Signature=1Ta4UIkiLtyeIwJ7QZn6VhKHRR4IFNFg3Z7iB8Z29TaJdSRpV8Aj5yXvMojIUH1RC~SkLJHrRtdJLsswb5UzLg27o5yG~ACTBb~AawUCInJKib~9KCHd53UgVKyG0d4DuUPjheaDy0ItU7IBTxnFPiPUnffS8Y59XxhXY2RkdNHU3GWdhXQOLKWElxvCZ~6QJIQyxehFvoP8pg72qilYQ~vK912aTBaUislRAlUD03IXNq9YsVS2QiJFi8Oc2KHLPLEGqhgmtoNC4BBhhPAj0lG98Y0YQri0XrOpsddX4qUklpFtol~FivTOByfZQTukJf9hQ85r1lMtAm0~cYHm6A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196688&ar=2462413&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03203.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Excision of CTns and MTns. (A) Schematic representation of the excision of CTns and MTns from the chromosome. Black triangles indicate the <em>att</em> sites. The PCR primers are indicated by arrows. Agarose gel electrophoresis of the PCR products obtained by the primer pairs left/right (B) and up/down (C). The <em>attTn</em>-containing regions in CTnPi-4 and MTnPi1-c were amplified from strain 17. The other regions were amplified from OMA14. <em>attTn</em> indicates the <em>att</em> sequence on the excised and circularized form of transposon.</p></div></div><p>Notably, MTnPi1-c was not excised (Fig. <a reveal-id="DSV032F3" data-open="DSV032F3" class="link link-reveal link-table xref-fig">3</a>B), which was rather unexpected, because this element contains all of the genes found in other MTnPi1 copies except for the second (downstream) integrase (Fig. <a reveal-id="DSV032F2" data-open="DSV032F2" class="link link-reveal link-table xref-fig">2</a>). The integrase of MTnPi1-c has been significantly diverged in sequence from the upstream integrase of MTnPi1-a and MTnPi1-b (92.7% amino acid sequence identity). This makes a sharp contrast to the fact that the upstream integrases of MTnPi1-a and MTnPi1-b are identical in amino acid sequence. Although the motif for tyrosine recombinase is conserved in the integrase of MTnPi1-c, amino acid substitutions have accumulated in the C-terminal region. Inactivation of the integrase by these amino acid substitutions may be a possible reason for the failure of excision of MTnPi1-c. However, the apparent and specific deletion of the downstream integrase gene from MTnPi1-c suggests another possibility that both integrases may be required for their function, at least for the excision of MTnPi1. If this is true, two integrase genes in MTnPi2 and MTnPi3 would also be required for the excision of these MTns, because their integrases show high sequence similarities to those of MTnPi1 (Fig. <a reveal-id="DSV032F2" data-open="DSV032F2" class="link link-reveal link-table xref-fig">2</a>).</p><p>In addition to allowing the exact definition of chromosome/CTn (or MTn) boundaries, the analysis of the identified <em>att</em> sequences/sites yielded several important findings. CTnPi1 and CTnPi4 use the same 13-bp sequence (TTTTCNNNNAAAA) that used by other CTnPg1 family members as its attachment site, supporting our conclusion regarding the relationship of these two CTns to the CTnPg1 family obtained from the previous structural comparison (Fig. <a reveal-id="DSV032F2" data-open="DSV032F2" class="link link-reveal link-table xref-fig">2</a>). Second, CTnPi2-b, CTnPi3, and MTnPi4 are inserted into tRNA genes, similar to several other CTns<sup><a reveal-id="DSV032C24" data-open="DSV032C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and MTns.<sup><a reveal-id="DSV032C26" data-open="DSV032C26" class="link link-ref link-reveal xref-bibr">26</a>,<a reveal-id="DSV032C27" data-open="DSV032C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Another copy of CTnPi2-a is inserted into an integrase gene (II_0226) that may form part of a degraded MTn similar to MTnPi1 and MTnPi2. Notably, MTnPi1, 2, and 3 use similar 11-bp sequences as the <em>att</em> site, supporting the close relatedness of these MTns that was inferred from the previous structural comparison (Fig. <a reveal-id="DSV032F2" data-open="DSV032F2" class="link link-reveal link-table xref-fig">2</a>).</p>                            <h3 scrollto-destination=87196695 id="87196695" class="section-title" data-legacy-id=s3c>3.4. <em>Prevotella intermedia/nigrescens</em> group-specific repeat</h3>
<p>We found a 188-bp incomplete repeat sequence located in intergenic regions in the genomes of both strains OMA14 and 17 (Fig. <a reveal-id="DSV032F4" data-open="DSV032F4" class="link link-reveal link-table xref-fig">4</a>A). This repeat sequence was expanded to 177 copies in OMA14 and 184 copies in strain 17 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). A database search for homologous sequences confirmed that this 188-bp repeat is also present in the downstream region of the <em>tnaA</em> gene<sup><a reveal-id="DSV032C28" data-open="DSV032C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and DNA probe Pig27<sup><a reveal-id="DSV032C29" data-open="DSV032C29" class="link link-ref link-reveal xref-bibr">29</a></sup> of <em>Pre. intermedia</em> ATCC 25611 and the draft genomes of <em>Pre. nigrescens</em> ATCC 33563 and F103. We further examined 18 <em>Prevotella</em> species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) for the presence of this repeat by PCR using primers specific to the repeat, which were designed based on the consensus sequence deduced from all copies found in strains OMA14 and 17 (Fig. <a reveal-id="DSV032F4" data-open="DSV032F4" class="link link-reveal link-table xref-fig">4</a>A). This analysis revealed that, in addition to <em>Pre. intermedia</em> and <em>Pre. nigrescens</em>, <em>Prevotella falsenii</em> and <em>Prevotella disiens</em> also contain the repeat (Fig. <a reveal-id="DSV032F4" data-open="DSV032F4" class="link link-reveal link-table xref-fig">4</a>B). These four species are closely related and form a clade in the genus <em>Prevotella</em> (Fig. <a reveal-id="DSV032F4" data-open="DSV032F4" class="link link-reveal link-table xref-fig">4</a>C).<sup><a reveal-id="DSV032C30" data-open="DSV032C30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="DSV032C31" data-open="DSV032C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Although some species in this clade lack the repeat, none of the <em>Prevotella</em> species belonging to other clades apparently possess the 188-bp repeat. We therefore named this repeat PINSR (<em><span class="underline">P</span>revotella <span class="underline">i</span>ntermedia/<span class="underline">n</span>igrescens</em> group-<span class="underline">s</span>pecific <span class="underline">r</span>epeat). The selective distribution of PINSR in the <em>Pre. intermedia/nigrescens</em> group suggests that PINSR was acquired by a common ancestor of this group and then amplified in some species (particularly <em>Pre. intermedia</em>) or deleted (or diverged in sequence) from other species following speciation. PINSR-specific PCR would thus be useful for identifying <em>Pre. intermedia</em> and the three other PINSR-possessing <em>Prevotella</em> species. </p>                        <a id="87196697" scrollto-destination="87196697"></a>
<div id="DSV032F4" data-id="DSV032F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/m_dsv03204.png?Expires=2147483647&Signature=e4tfhSTDlhIPQm-m79dll6Npg7fa1GK9cplZt~7Dj3XYcBnn0Bm~SXvOGKF2WoA~ociq4ZoP~iKtRB2MRn3vQQl1qa2g2aOrK95e85FxX3MHnetaOrFG-0jQjvlJJM46aOPydmEQDH8MIuymaCu-j4Z0as3gSenHXwRsaZHfmK6Tc4Dd~weUxvUM6aVLnqGEmlYwq927mh8vG4vxWBOkjiDKB53f~Omwa7lc~IFyWYSm26BPrprup-ezknTsRFNPuF3HNjguymytF0fFj-iYwwbVzz~AJD-o6uX1PPaneBvpSx-GeqShLFPEpqtEwpZFCtOzUEhGeq4NpOoboRgwug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Prevotella intermedia–nigrescens group-specific repeat (PINSR). (A) The 188-bp consensus sequence denoted PINSR was deduced from the alignment of all PINSR copies found in strains OMA14 and 17. The nucleotides conserved in &gt;90% and &gt;50% copies are indicated in upper case and lower case, respectively. The target sequences of the designed PCR primers are underlined. (B) PCR detection of PINSR sequences (upper) and 16S-rRNA sequences (lower) in various Prevotella species. Lane 1, Pre. intermedia OMA14; Lane 2, Pre. intermedia strain 17; Lane 3, Pre. aurantiaca OMA31; Lane 4, Pre. falsenii JCM15124T; Lane 5, Pre. pallens AHN10371T; Lane 6, Pre. nigrescens ATCC33563; Lane 7, Pre. nigrescens ATCC25261; Lane 8, Pre. disiens ATCC29426; Lane 9, Pre. corporis ATCC33547; Lane 10, Pre. paludivivens JCM 13650T; Lane 11, Pre. amnii CCUG53648; Lane 12, Pre. oulorum ATCC43324; Lane 13, Pre. oris ATCC33573; Lane 14, Pre. veroralis ATCC33779; Lane 15, Pre. albensis DSM 11370; Lane 16, Pre. denticola ATCC35308; Lane 17, Pre. marshii DSM16973; Lane 18, Pre. loesheii ATCC15930; Lane 19, Pre. pleuritidis JCM 14110 T; Lane 20, Pre. nanceiensis JCM 15639T; and Lane 21, distilled water (negative control). (C) Distribution of PINSR within the genus Prevotella. The presence (closed triangles) or absence (open triangles) of PINSR is shown in the 16S-rRNA gene sequence-based phylogenetic tree of the genus Prevotella (adapted from Sakamoto et al.30). Species, the complete genome sequences of which are available in public databases, are indicated by open circles. Only the species indicated by triangles were examined for the presence of PINSR by PCR." path-from-xml="dsv03204.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196697/dsv03204.tiff" path-from-xml="dsv03204.tiff" target="_blank">View large</a><a section="87196697" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv03204.png?Expires=2147483647&Signature=HxIGcJiVyJpc~ATmydAzRSIzVLU~NdQVakmHP~iX2cJylbFZI-MHMmNHhWTmMGsjr0~GMmIFIJiTz0mRUWVS4pWxV1ygIZluLMcuREBZ9JoanJsQ7YFFJVK~mVKgZOtAf5y8sDetJbIOdGgLh4bFy7ycYK~QTXn~ZK2J7IQyeICoNcrAgWvYZJoeJKGuDcT235TG-si7gucpdwjUZL9rDTkfQPnebLrjrw0sTJ1knRrJPmOT3G~BJopDjficeAT2mwwqFLBXI3kZNWC98qWQeyLjN6z2g3mgZhmN1X5F6b~Ye-kijsZRjKf2eRyPpaoqWAW2n5HIeE~6Qy4PtfgTzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196697&ar=2462413&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03204.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Prevotella intermedia–nigrescens</em> group-specific repeat (PINSR). (A) The 188-bp consensus sequence denoted PINSR was deduced from the alignment of all PINSR copies found in strains OMA14 and 17. The nucleotides conserved in &gt;90% and &gt;50% copies are indicated in upper case and lower case, respectively. The target sequences of the designed PCR primers are underlined. (B) PCR detection of PINSR sequences (upper) and 16S-rRNA sequences (lower) in various <em>Prevotella</em> species. Lane 1<em>, Pre. intermedia</em> OMA14; Lane 2, <em>Pre. intermedia</em> strain 17; Lane 3, <em>Pre. aurantiaca</em> OMA31; Lane 4, <em>Pre. falsenii</em> JCM15124T; Lane 5, <em>Pre. pallens</em> AHN10371T; Lane 6, <em>Pre. nigrescens</em> ATCC33563; Lane 7, <em>Pre. nigrescens</em> ATCC25261; Lane 8, <em>Pre. disiens</em> ATCC29426; Lane 9, <em>Pre. corporis</em> ATCC33547; Lane 10, <em>Pre. paludivivens</em> JCM 13650T; Lane 11, <em>Pre. amnii</em> CCUG53648; Lane 12, <em>Pre. oulorum</em> ATCC43324; Lane 13, <em>Pre. oris</em> ATCC33573; Lane 14, <em>Pre. veroralis</em> ATCC33779; Lane 15, <em>Pre. albensis</em> DSM 11370; Lane 16, <em>Pre. denticola</em> ATCC35308; Lane 17, <em>Pre. marshii</em> DSM16973; Lane 18, <em>Pre. loesheii</em> ATCC15930; Lane 19, <em>Pre. pleuritidis</em> JCM 14110 T; Lane 20, <em>Pre. nanceiensis</em> JCM 15639T; and Lane 21, distilled water (negative control). (C) Distribution of PINSR within the genus <em>Prevotella</em>. The presence (closed triangles) or absence (open triangles) of PINSR is shown in the 16S-rRNA gene sequence-based phylogenetic tree of the genus <em>Prevotella</em> (adapted from Sakamoto et al.<sup><a reveal-id="DSV032C30" data-open="DSV032C30" class="link link-ref link-reveal xref-bibr">30</a></sup>). Species, the complete genome sequences of which are available in public databases, are indicated by open circles. Only the species indicated by triangles were examined for the presence of PINSR by PCR.</p></div></div><div content-id="DSV032F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/m_dsv03204.png?Expires=2147483647&Signature=e4tfhSTDlhIPQm-m79dll6Npg7fa1GK9cplZt~7Dj3XYcBnn0Bm~SXvOGKF2WoA~ociq4ZoP~iKtRB2MRn3vQQl1qa2g2aOrK95e85FxX3MHnetaOrFG-0jQjvlJJM46aOPydmEQDH8MIuymaCu-j4Z0as3gSenHXwRsaZHfmK6Tc4Dd~weUxvUM6aVLnqGEmlYwq927mh8vG4vxWBOkjiDKB53f~Omwa7lc~IFyWYSm26BPrprup-ezknTsRFNPuF3HNjguymytF0fFj-iYwwbVzz~AJD-o6uX1PPaneBvpSx-GeqShLFPEpqtEwpZFCtOzUEhGeq4NpOoboRgwug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Prevotella intermedia–nigrescens group-specific repeat (PINSR). (A) The 188-bp consensus sequence denoted PINSR was deduced from the alignment of all PINSR copies found in strains OMA14 and 17. The nucleotides conserved in &gt;90% and &gt;50% copies are indicated in upper case and lower case, respectively. The target sequences of the designed PCR primers are underlined. (B) PCR detection of PINSR sequences (upper) and 16S-rRNA sequences (lower) in various Prevotella species. Lane 1, Pre. intermedia OMA14; Lane 2, Pre. intermedia strain 17; Lane 3, Pre. aurantiaca OMA31; Lane 4, Pre. falsenii JCM15124T; Lane 5, Pre. pallens AHN10371T; Lane 6, Pre. nigrescens ATCC33563; Lane 7, Pre. nigrescens ATCC25261; Lane 8, Pre. disiens ATCC29426; Lane 9, Pre. corporis ATCC33547; Lane 10, Pre. paludivivens JCM 13650T; Lane 11, Pre. amnii CCUG53648; Lane 12, Pre. oulorum ATCC43324; Lane 13, Pre. oris ATCC33573; Lane 14, Pre. veroralis ATCC33779; Lane 15, Pre. albensis DSM 11370; Lane 16, Pre. denticola ATCC35308; Lane 17, Pre. marshii DSM16973; Lane 18, Pre. loesheii ATCC15930; Lane 19, Pre. pleuritidis JCM 14110 T; Lane 20, Pre. nanceiensis JCM 15639T; and Lane 21, distilled water (negative control). (C) Distribution of PINSR within the genus Prevotella. The presence (closed triangles) or absence (open triangles) of PINSR is shown in the 16S-rRNA gene sequence-based phylogenetic tree of the genus Prevotella (adapted from Sakamoto et al.30). Species, the complete genome sequences of which are available in public databases, are indicated by open circles. Only the species indicated by triangles were examined for the presence of PINSR by PCR." path-from-xml="dsv03204.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196697/dsv03204.tiff" path-from-xml="dsv03204.tiff" target="_blank">View large</a><a section="87196697" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv03204.png?Expires=2147483647&Signature=HxIGcJiVyJpc~ATmydAzRSIzVLU~NdQVakmHP~iX2cJylbFZI-MHMmNHhWTmMGsjr0~GMmIFIJiTz0mRUWVS4pWxV1ygIZluLMcuREBZ9JoanJsQ7YFFJVK~mVKgZOtAf5y8sDetJbIOdGgLh4bFy7ycYK~QTXn~ZK2J7IQyeICoNcrAgWvYZJoeJKGuDcT235TG-si7gucpdwjUZL9rDTkfQPnebLrjrw0sTJ1knRrJPmOT3G~BJopDjficeAT2mwwqFLBXI3kZNWC98qWQeyLjN6z2g3mgZhmN1X5F6b~Ye-kijsZRjKf2eRyPpaoqWAW2n5HIeE~6Qy4PtfgTzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196697&ar=2462413&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03204.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>Prevotella intermedia–nigrescens</em> group-specific repeat (PINSR). (A) The 188-bp consensus sequence denoted PINSR was deduced from the alignment of all PINSR copies found in strains OMA14 and 17. The nucleotides conserved in &gt;90% and &gt;50% copies are indicated in upper case and lower case, respectively. The target sequences of the designed PCR primers are underlined. (B) PCR detection of PINSR sequences (upper) and 16S-rRNA sequences (lower) in various <em>Prevotella</em> species. Lane 1<em>, Pre. intermedia</em> OMA14; Lane 2, <em>Pre. intermedia</em> strain 17; Lane 3, <em>Pre. aurantiaca</em> OMA31; Lane 4, <em>Pre. falsenii</em> JCM15124T; Lane 5, <em>Pre. pallens</em> AHN10371T; Lane 6, <em>Pre. nigrescens</em> ATCC33563; Lane 7, <em>Pre. nigrescens</em> ATCC25261; Lane 8, <em>Pre. disiens</em> ATCC29426; Lane 9, <em>Pre. corporis</em> ATCC33547; Lane 10, <em>Pre. paludivivens</em> JCM 13650T; Lane 11, <em>Pre. amnii</em> CCUG53648; Lane 12, <em>Pre. oulorum</em> ATCC43324; Lane 13, <em>Pre. oris</em> ATCC33573; Lane 14, <em>Pre. veroralis</em> ATCC33779; Lane 15, <em>Pre. albensis</em> DSM 11370; Lane 16, <em>Pre. denticola</em> ATCC35308; Lane 17, <em>Pre. marshii</em> DSM16973; Lane 18, <em>Pre. loesheii</em> ATCC15930; Lane 19, <em>Pre. pleuritidis</em> JCM 14110 T; Lane 20, <em>Pre. nanceiensis</em> JCM 15639T; and Lane 21, distilled water (negative control). (C) Distribution of PINSR within the genus <em>Prevotella</em>. The presence (closed triangles) or absence (open triangles) of PINSR is shown in the 16S-rRNA gene sequence-based phylogenetic tree of the genus <em>Prevotella</em> (adapted from Sakamoto et al.<sup><a reveal-id="DSV032C30" data-open="DSV032C30" class="link link-ref link-reveal xref-bibr">30</a></sup>). Species, the complete genome sequences of which are available in public databases, are indicated by open circles. Only the species indicated by triangles were examined for the presence of PINSR by PCR.</p></div></div><p>In addition, small CDSs that overlap PINSR were annotated in the genome sequence of strain 17 (Accession No. NC_017860, NC_017861). However, clear ribosome-binding sequences are not present in most cases, and no significant homology to known proteins was detected. The sizes of these small CDSs are highly variable. Therefore, it is most likely that PINSR does not contain protein-coding sequences. PINSR also does not contain terminal-inverted repeat sequences. Therefore, PINSR is distinct from the miniature-inverted repeat transposable element (MITE) that is transposable by the action of the transposase provided <em>in trans</em> from a cognate IS element. Intriguingly, approximately three-quarters of the PINSR-containing regions identified in strains OMA14 and 17 are conserved between the two strains, suggesting that PINSR may have been amplified prior to the separation of the two strains. This hypothesis is supported by the finding that most strain-specific regions, such as CTns and MTns, in both strains do not contain PINSR. Although PINSR apparently played some roles in the genome rearrangement that occurred between the two strains, as described in the next section, further studies are necessary to elucidate the biological functions and source of PINSR (if present).</p>                            <h3 scrollto-destination=87196708 id="87196708" class="section-title" data-legacy-id=s3d>3.5. Genome rearrangement</h3>
<p>A whole-genome sequence alignment analysis revealed that extensive genomic rearrangements have occurred between strains OMA14 and 17. We identified 26 inversions/translocations and &gt;100 simple insertions/deletions (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Approximately two-thirds of the identified inversions/translocations are associated with MGEs, and at least one IS, CTn, or MTn element is present at each inversion/translocation site. The remaining inversions/translocations occurred between PINSRs, <em>rrn</em> operons, or tRNA genes. This analysis also identified strain-specific segments (&gt;10 kb) in both genomes, namely Segments A-W in OMA14 and Segments A-I in strain 17 (the sixth circle in Fig. <a reveal-id="DSV032F1" data-open="DSV032F1" class="link link-reveal link-table xref-fig">1</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S7 and S8</a></span>). Approximately half of these segments simply consist of a single CTn or MTn (10 of the 23 segments in stain OMA14 and 4 of the 9 segments in strain 17). However, several segments, particularly larger ones, exhibit composite natures containing multiple MGTs. Furthermore, several segments contain phage-related genes and thus likely represent prophages or degraded prophages. These features further support the hypothesis that MGEs played key roles in generating the genomic diversity of <em>Pre. intermedia</em>.</p>                            <h3 scrollto-destination=87196712 id="87196712" class="section-title" data-legacy-id=s3e>3.6. Gene repertoire comparison between the two <em>Pre. intermedia</em> strains</h3>
<p>To compare the gene repertoires of strains OMA14 and 17, we performed a clustering analysis of all CDSs identified in the two strains using the CD-HIT program, which generated 2,884 CDS clusters. Among these, 1,744 were found to be conserved in both strains, whereas 686 and 454 are specific to strains OMA14 and 17, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). This number of conserved genes is somewhat larger than that recently revealed by the comparison of the genome of strain 17 and draft sequences of two <em>Pre. intermedia</em> strains (∼1,600).<sup><a reveal-id="DSV032C14" data-open="DSV032C14" class="link link-ref link-reveal xref-bibr">14</a></sup> This difference may be due to the difference in the number of strains analysed and to the difference in the quality of the sequences or annotations (complete versus. draft sequences). Therefore, the core genome of <em>Pre. intermedia</em> most likely consists of 1,600–1,700 genes and encodes most virulence genes that have been identified in <em>Pre. intermedia</em>, such as proteolytic enzyme interpain A (II_0221)<sup><a reveal-id="DSV032C12" data-open="DSV032C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and the adhesins AdpC (I_1236)<sup><a reveal-id="DSV032C10" data-open="DSV032C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and AdpF (I_1057).<sup><a reveal-id="DSV032C11" data-open="DSV032C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The gene set encoding the type IX secretion system (T9SS), which was originally identified in <em>Por. gingivalis</em>,<sup><a reveal-id="DSV032C32" data-open="DSV032C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSV032C33" data-open="DSV032C33" class="link link-ref link-reveal xref-bibr">33</a></sup> was also found in the core genome.</p><p>The strain-specific CDSs listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S9 and S10</a></span> include many genes in CTns and MTns, reflecting the difference in the transposon repertoires between the two strains. In particular, as many as 54.4% of the CDSs on the second chromosome of strain OMA14, where CTns and MTns are most densely integrated, are strain-specific CDSs. Notably, each strain contains a large number of strain-specific genes related to DNA restriction modification. Another noteworthy finding is the presence of strain-specific glycosyltransferase gene clusters in each strain (Segment D in OMA14 and Segment G in strain 17), which suggests that the two strains may produce different extracellular polysaccharides.</p>                            <h3 scrollto-destination=87196718 id="87196718" class="section-title" data-legacy-id=s3f>3.7. Conclusions</h3>
<p>In this study, we determined the complete genome sequence of <em>Pre. intermedia</em> OMA14 and compared it with the completely sequenced <em>Pre. intermedia</em> strain 17. Our analysis revealed that the genomes of <em>Pre. intermedia</em> are unusually enriched in CTns and MTns, which account for large parts of the strain-specific sequences observed in each <em>Pre. intermedia</em> genome. Together with IS elements, these CTns and MTns are also involved in the extensive genomic rearrangements observed between the two strains. In addition, we found a novel repeat element (named PINSR) that has been extensively amplified in <em>Pre. intermedia</em> and several <em>Prevotella</em> species belonging to the <em>Pre. intermedia</em>–<em>nigrescens</em> group. These results expand our understanding of the genetic features and genomic diversity of <em>Pre. intermedia</em> and provide novel insights into the roles of MGEs, particularly CTns and MTns, in the evolution of bacteria.</p>                            <h2 scrollto-destination=87196723 id="87196723" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by a Grant-in-Aid for Scientific Research on Innovative Areas ‘Genome Science’ from the Ministry of Education, Culture, Sports, Science and Technology (No. 221S0002), and by grants from the Japan Society for the Promotion of Science (No. 20249073 and No. 24659817) to K.N. Funding to pay the Open Access publication charges for this article was provided by grants from the Japan Society for the Promotion of Science (No. 26670804) to K.N.</p>                            <h2 scrollto-destination=87196728 id="87196728" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV032C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Socransky</div>
               <div class="given-names">S.S.</div>
            </div>, <div class="name string-name"><div class="surname">Haffajee</div> <div class="given-names">A.D.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">Dental biofilms: difficult therapeutic targets,</div><div class="source">Periodontol.</div> , <div class="volume">28</div>, <div class="fpage">12</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dental%20biofilms%3A%20difficult%20therapeutic%20targets%2C&amp;author=S.S.%20Socransky&amp;author=A.D.%20Haffajee&amp;publication_year=2002&amp;journal=Periodontol.&amp;volume=28&amp;pages=12-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1034/j.1600-0757.2002.280102.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1034%2Fj.1600-0757.2002.280102.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1034%2Fj.1600-0757.2002.280102.x"> </span></div> </div></div></div></div></div><div content-id="DSV032C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fukushima</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Yamamoto</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Hirohata</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Sagawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Leung</div> <div class="given-names">K.P.</div></div>, <div class="name string-name"><div class="surname">Walker</div> <div class="given-names">C.B.</div></div></div>
          <div class="year">1990</div>, <div class="article-title">Localization and identification of root canal bacteria in clinically asymptomatic periapical pathosis</div>, <div class="source">J. Endod.</div> , <div class="volume">16</div>, <div class="fpage">534</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Localization%20and%20identification%20of%20root%20canal%20bacteria%20in%20clinically%20asymptomatic%20periapical%20pathosis&amp;author=H.%20Fukushima&amp;author=K.%20Yamamoto&amp;author=K.%20Hirohata&amp;author=H.%20Sagawa&amp;author=K.P.%20Leung&amp;author=C.B.%20Walker&amp;publication_year=1990&amp;journal=J.%20Endod.&amp;volume=16&amp;pages=534-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0099-2399(07)80216-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0099-2399(07)80216-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0099-2399(07)80216-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2084210" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Baumgartner</div>
               <div class="given-names">J.C.</div>
            </div>, <div class="name string-name"><div class="surname">Watkins</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Bae</div> <div class="given-names">K.S.</div></div>, <div class="name string-name"><div class="surname">Xia</div> <div class="given-names">T.</div></div></div>
          <div class="year">1999</div>, <div class="article-title">Association of black-pigmented bacteria with endodontic infections</div>, <div class="source">J. Endod.</div> , <div class="volume">25</div>, <div class="fpage">413</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Association%20of%20black-pigmented%20bacteria%20with%20endodontic%20infections&amp;author=J.C.%20Baumgartner&amp;author=B.J.%20Watkins&amp;author=K.S.%20Bae&amp;author=T.%20Xia&amp;publication_year=1999&amp;journal=J.%20Endod.&amp;volume=25&amp;pages=413-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0099-2399(99)80268-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0099-2399(99)80268-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0099-2399(99)80268-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10530240" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Raber-Durlacher</div>
               <div class="given-names">J.E.</div>
            </div>, <div class="name string-name"><div class="surname">van Steenbergen</div> <div class="given-names">T.J.</div></div>, <div class="name string-name"><div class="surname">Van der Velden</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">de Graaff</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Abraham-Inpijn</div> <div class="given-names">L.</div></div></div>
          <div class="year">1994</div>, <div class="article-title">Experimental gingivitis during pregnancy and post-partum: clinical, endocrinological, and microbiological aspects</div>, <div class="source">J. Clin. Periodontol.</div> , <div class="volume">21</div>, <div class="fpage">549</div>–<div class="lpage">58</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20gingivitis%20during%20pregnancy%20and%20post-partum%3A%20clinical%2C%20endocrinological%2C%20and%20microbiological%20aspects&amp;author=J.E.%20Raber-Durlacher&amp;author=T.J.%20van%20Steenbergen&amp;author=U.%20Van%20der%20Velden&amp;author=J.%20de%20Graaff&amp;author=L.%20Abraham-Inpijn&amp;publication_year=1994&amp;journal=J.%20Clin.%20Periodontol.&amp;volume=21&amp;pages=549-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/cpe.1994.21.issue-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fcpe.1994.21.issue-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fcpe.1994.21.issue-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7989619" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Loesche</div>
               <div class="given-names">W.J.</div>
            </div>, <div class="name string-name"><div class="surname">Syed</div> <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Laughon</div> <div class="given-names">B.E.</div></div>, <div class="name string-name"><div class="surname">Stoll</div> <div class="given-names">J.</div></div></div>
          <div class="year">1982</div>, <div class="article-title">The bacteriology of acute necrotizing ulcerative gingivitis</div>, <div class="source">J. Periodontol.</div> , <div class="volume">53</div>, <div class="fpage">223</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20bacteriology%20of%20acute%20necrotizing%20ulcerative%20gingivitis&amp;author=W.J.%20Loesche&amp;author=S.A.%20Syed&amp;author=B.E.%20Laughon&amp;author=J.%20Stoll&amp;publication_year=1982&amp;journal=J.%20Periodontol.&amp;volume=53&amp;pages=223-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1902/jop.1982.53.4.223" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1902%2Fjop.1982.53.4.223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1902%2Fjop.1982.53.4.223"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6122728" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Scannapieco</div>
               <div class="given-names">F.A.</div>
            </div>, <div class="name string-name"><div class="surname">Bush</div> <div class="given-names">R.B.</div></div>, <div class="name string-name"><div class="surname">Paju</div> <div class="given-names">S.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Associations between periodontal disease and risk for atherosclerosis, cardiovascular disease, and stroke. A systematic review</div>, <div class="source">Ann. Periodontol.</div> , <div class="volume">8</div>, <div class="fpage">38</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Associations%20between%20periodontal%20disease%20and%20risk%20for%20atherosclerosis%2C%20cardiovascular%20disease%2C%20and%20stroke.%20A%20systematic%20review&amp;author=F.A.%20Scannapieco&amp;author=R.B.%20Bush&amp;author=S.%20Paju&amp;publication_year=2003&amp;journal=Ann.%20Periodontol.&amp;volume=8&amp;pages=38-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1902/annals.2003.8.1.38" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1902%2Fannals.2003.8.1.38" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1902%2Fannals.2003.8.1.38"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14971247" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Offenbacher</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Lieff</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Boggess</div> <div class="given-names">K.A.</div></div>et al.  </div>. <div class="year">2001</div>, <div class="article-title">Maternal periodontitis and prematurity. Part I: obstetric outcome of prematurity and growth restriction</div>, <div class="source">Ann. Periodontol.</div> , <div class="volume">6</div>, <div class="fpage">164</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maternal%20periodontitis%20and%20prematurity.%20Part%20I%3A%20obstetric%20outcome%20of%20prematurity%20and%20growth%20restriction&amp;author=S.%20Offenbacher&amp;author=S.%20Lieff&amp;author=K.A.%20Boggess&amp;publication_year=2001&amp;journal=Ann.%20Periodontol.&amp;volume=6&amp;pages=164-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1902/annals.2001.6.1.164" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1902%2Fannals.2001.6.1.164" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1902%2Fannals.2001.6.1.164"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11887460" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Haraszthy</div>
               <div class="given-names">V.I.</div>
            </div>, <div class="name string-name"><div class="surname">Zambon</div> <div class="given-names">J.J.</div></div>, <div class="name string-name"><div class="surname">Trevisan</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Zeid</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Genco</div> <div class="given-names">R.J.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Identification of periodontal pathogens in atheromatous plaques</div>, <div class="source">J. Periodontol.</div> , <div class="volume">71</div>, <div class="fpage">1554</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20periodontal%20pathogens%20in%20atheromatous%20plaques&amp;author=V.I.%20Haraszthy&amp;author=J.J.%20Zambon&amp;author=M.%20Trevisan&amp;author=M.%20Zeid&amp;author=R.J.%20Genco&amp;publication_year=2000&amp;journal=J.%20Periodontol.&amp;volume=71&amp;pages=1554-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1902/jop.2000.71.10.1554" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1902%2Fjop.2000.71.10.1554" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1902%2Fjop.2000.71.10.1554"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11063387" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Madianos</div>
               <div class="given-names">P.N.</div>
            </div>, <div class="name string-name"><div class="surname">Lieff</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Murtha</div> <div class="given-names">A.P.</div></div>et al.  </div>. <div class="year">2001</div>, <div class="article-title">Maternal periodontitis and prematurity. Part II: maternal infection and fetal exposure</div>, <div class="source">Ann. Periodontol.</div> , <div class="volume">6</div>, <div class="fpage">175</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Maternal%20periodontitis%20and%20prematurity.%20Part%20II%3A%20maternal%20infection%20and%20fetal%20exposure&amp;author=P.N.%20Madianos&amp;author=S.%20Lieff&amp;author=A.P.%20Murtha&amp;publication_year=2001&amp;journal=Ann.%20Periodontol.&amp;volume=6&amp;pages=175-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1902/annals.2001.6.1.175" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1902%2Fannals.2001.6.1.175" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1902%2Fannals.2001.6.1.175"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11887461" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Iyer</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Anaya-Bergman</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">K.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">AdpC is a <em>Prevotella intermedia</em> 17 leucine-rich repeat internalin-like protein</div>, <div class="source">Infect. Immun.</div> , <div class="volume">78</div>, <div class="fpage">2385</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AdpC%20is%20a%20Prevotella%20intermedia%2017%20leucine-rich%20repeat%20internalin-like%20protein&amp;author=D.%20Iyer&amp;author=C.%20Anaya-Bergman&amp;author=K.%20Jones&amp;publication_year=2010&amp;journal=Infect.%20Immun.&amp;volume=78&amp;pages=2385-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.00510-09" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.00510-09" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.00510-09"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20308299" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sengupta</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Kang</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Anaya-Bergman</div> <div class="given-names">C.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Interaction of Prevotella intermedia strain 17 leucine-rich repeat domain protein AdpF with eukaryotic cells promotes bacterial internalization</div>, <div class="source">Infect. Immun.</div> , <div class="volume">82</div>, <div class="fpage">2637</div>–<div class="lpage">48</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Interaction%20of%20Prevotella%20intermedia%20strain%2017%20leucine-rich%20repeat%20domain%20protein%20AdpF%20with%20eukaryotic%20cells%20promotes%20bacterial%20internalization&amp;author=D.%20Sengupta&amp;author=D.J.%20Kang&amp;author=C.%20Anaya-Bergman&amp;publication_year=2014&amp;journal=Infect.%20Immun.&amp;volume=82&amp;pages=2637-48" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.01361-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.01361-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.01361-13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24711565" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mallorqui-Fernandez</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Manandhar</div> <div class="given-names">S.P.</div></div>, <div class="name string-name"><div class="surname">Mallorqui-Fernandez</div> <div class="given-names">G.</div></div>et al.  </div>. <div class="year">2008</div>, <div class="article-title">A new autocatalytic activation mechanism for cysteine proteases revealed by Prevotella intermedia interpain A</div>, <div class="source">J. Biol. Chem.</div> , <div class="volume">283</div>, <div class="fpage">2871</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20autocatalytic%20activation%20mechanism%20for%20cysteine%20proteases%20revealed%20by%20Prevotella%20intermedia%20interpain%20A&amp;author=N.%20Mallorqui-Fernandez&amp;author=S.P.%20Manandhar&amp;author=G.%20Mallorqui-Fernandez&amp;publication_year=2008&amp;journal=J.%20Biol.%20Chem.&amp;volume=283&amp;pages=2871-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M708481200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M708481200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M708481200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17993455" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hashimoto</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Asai</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tamai</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Jinno</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Umatani</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Ogawa</div> <div class="given-names">T.</div></div></div>
          <div class="year">2003</div>, <div class="article-title">Chemical structure and immunobiological activity of lipid A from <em>Prevotella intermedia</em> ATCC 25611 lipopolysaccharide</div>, <div class="source">FEBS Lett.</div> , <div class="volume">543</div>, <div class="fpage">98</div>–<div class="lpage">102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chemical%20structure%20and%20immunobiological%20activity%20of%20lipid%20A%20from%20Prevotella%20intermedia%20ATCC%2025611%20lipopolysaccharide&amp;author=M.%20Hashimoto&amp;author=Y.%20Asai&amp;author=R.%20Tamai&amp;author=T.%20Jinno&amp;author=K.%20Umatani&amp;author=T.%20Ogawa&amp;publication_year=2003&amp;journal=FEBS%20Lett.&amp;volume=543&amp;pages=98-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0014-5793(03)00414-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0014-5793(03)00414-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0014-5793(03)00414-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12753913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ruan</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Shen</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Zou</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2015</div>, <div class="article-title">Comparative genome analysis of <em>Prevotella intermedia</em> strain isolated from infected root canal reveals features related to pathogenicity and adaptation</div>, <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">122</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genome%20analysis%20of%20Prevotella%20intermedia%20strain%20isolated%20from%20infected%20root%20canal%20reveals%20features%20related%20to%20pathogenicity%20and%20adaptation&amp;author=Y.%20Ruan&amp;author=L.%20Shen&amp;author=Y.%20Zou&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=122" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1272-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1272-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1272-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25765460" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Fukushima</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Moroi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Inoue</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">1992</div>, <div class="article-title">Phenotypic characteristics and DNA relatedness in <em>Prevotella intermedia</em> and similar organisms</div>, <div class="source">Oral Microbiol. Immunol.</div> , <div class="volume">7</div>, <div class="fpage">60</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phenotypic%20characteristics%20and%20DNA%20relatedness%20in%20Prevotella%20intermedia%20and%20similar%20organisms&amp;author=H.%20Fukushima&amp;author=H.%20Moroi&amp;author=J.%20Inoue&amp;publication_year=1992&amp;journal=Oral%20Microbiol.%20Immunol.&amp;volume=7&amp;pages=60-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/omi.1992.7.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fomi.1992.7.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fomi.1992.7.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1528627" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Naito</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Shoji</div> <div class="given-names">M.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Characterization of the <em>Porphyromonas gingivalis</em> conjugative transposon CTnPg1: determination of the integration site and the genes essential for conjugal transfer</div>, <div class="source">Microbiology</div> , <div class="volume">157</div>, <div class="fpage">2022</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20Porphyromonas%20gingivalis%20conjugative%20transposon%20CTnPg1%3A%20determination%20of%20the%20integration%20site%20and%20the%20genes%20essential%20for%20conjugal%20transfer&amp;author=M.%20Naito&amp;author=K.%20Sato&amp;author=M.%20Shoji&amp;publication_year=2011&amp;journal=Microbiology&amp;volume=157&amp;pages=2022-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.047803-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.047803-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.047803-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21527470" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sugawara</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Ohyama</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Mori</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kurokawa</div> <div class="given-names">K.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Microbial Genome Annotation Pipeline (MiGAP) for diverse users</div>. In: <div class="conf-name">The 20th International Conference on Genome Informatics (GIW2009)</div>, <div class="conf-loc">Yokohama,</div><div class="publisher-loc">Japan</div>, pp. <div class="comment">S001-001002</div>.</div></div></div></div><div content-id="DSV032C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Finn</div>
               <div class="given-names">R.D.</div>
            </div>, <div class="name string-name"><div class="surname">Bateman</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Clements</div> <div class="given-names">J.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Pfam: the protein families database</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">42</div>, <div class="fpage">D222</div>–<div class="lpage">30</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pfam%3A%20the%20protein%20families%20database&amp;author=R.D.%20Finn&amp;author=A.%20Bateman&amp;author=J.%20Clements&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res.&amp;volume=42&amp;pages=D222-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt1223" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt1223" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt1223"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24288371" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gao</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">Ori-Finder: a web-based system for finding oriCs in unannotated bacterial genomes</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">9</div>, <div class="fpage">79</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ori-Finder%3A%20a%20web-based%20system%20for%20finding%20oriCs%20in%20unannotated%20bacterial%20genomes&amp;author=F.%20Gao&amp;author=C.T.%20Zhang&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=79" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-79" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-79" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-79"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18237442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ohtsubo</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Ikeda-Ohtsubo</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Nagata</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tsuda</div> <div class="given-names">M.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">GenomeMatcher: a graphical user interface for DNA sequence comparison</div>, <div class="source">BMC Bioinformatics</div> , <div class="volume">9</div>, <div class="fpage">376</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GenomeMatcher%3A%20a%20graphical%20user%20interface%20for%20DNA%20sequence%20comparison&amp;author=Y.%20Ohtsubo&amp;author=W.%20Ikeda-Ohtsubo&amp;author=Y.%20Nagata&amp;author=M.%20Tsuda&amp;publication_year=2008&amp;journal=BMC%20Bioinformatics&amp;volume=9&amp;pages=376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-9-376" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-9-376" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-9-376"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18793444" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Huang</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Niu</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Fu</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">CD-HIT Suite: a web server for clustering and comparing biological sequences</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">680</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CD-HIT%20Suite%3A%20a%20web%20server%20for%20clustering%20and%20comparing%20biological%20sequences&amp;author=Y.%20Huang&amp;author=B.%20Niu&amp;author=Y.%20Gao&amp;author=L.%20Fu&amp;author=W.%20Li&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=680-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20053844" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gao</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">C.T.</div></div></div>
          <div class="year">2007</div>, <div class="article-title">DoriC: a database of oriC regions in bacterial genomes</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">1866</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DoriC%3A%20a%20database%20of%20oriC%20regions%20in%20bacterial%20genomes&amp;author=F.%20Gao&amp;author=C.T.%20Zhang&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1866-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm255" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm255" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm255"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17496319" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Siguier</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Perochon</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Lestrade</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Mahillon</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Chandler</div> <div class="given-names">M.</div></div></div>
          <div class="year">2006</div>, <div class="article-title">ISfinder: the reference centre for bacterial insertion sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">34</div>, <div class="fpage">D32</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ISfinder%3A%20the%20reference%20centre%20for%20bacterial%20insertion%20sequences&amp;author=P.%20Siguier&amp;author=J.%20Perochon&amp;author=L.%20Lestrade&amp;author=J.%20Mahillon&amp;author=M.%20Chandler&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=D32-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkj014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkj014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkj014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16381877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Osborn</div>
               <div class="given-names">A.M.</div>
            </div>, <div class="name string-name"><div class="surname">Boltner</div> <div class="given-names">D.</div></div></div>
          <div class="year">2002</div>, <div class="article-title">When phage, plasmids, and transposons collide: genomic islands, and conjugative- and mobilizable-transposons as a mosaic continuum</div>, <div class="source">Plasmid</div> , <div class="volume">48</div>, <div class="fpage">202</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=When%20phage%2C%20plasmids%2C%20and%20transposons%20collide%3A%20genomic%20islands%2C%20and%20conjugative-%20and%20mobilizable-transposons%20as%20a%20mosaic%20continuum&amp;author=A.M.%20Osborn&amp;author=D.%20Boltner&amp;publication_year=2002&amp;journal=Plasmid&amp;volume=48&amp;pages=202-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0147-619X(02)00117-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0147-619X(02)00117-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0147-619X(02)00117-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12460536" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Camilli</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Del Grosso</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Iannelli</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Pantosti</div> <div class="given-names">A.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">New genetic element carrying the erythromycin resistance determinant erm(TR) in Streptococcus pneumoniae</div>, <div class="source">Antimicrob. Agents Chemother.</div> , <div class="volume">52</div>, <div class="fpage">619</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=New%20genetic%20element%20carrying%20the%20erythromycin%20resistance%20determinant%20erm(TR)%20in%20Streptococcus%20pneumoniae&amp;author=R.%20Camilli&amp;author=M.%20Del%20Grosso&amp;author=F.%20Iannelli&amp;author=A.%20Pantosti&amp;publication_year=2008&amp;journal=Antimicrob.%20Agents%20Chemother.&amp;volume=52&amp;pages=619-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AAC.01081-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAAC.01081-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAAC.01081-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18070957" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shoemaker</div>
               <div class="given-names">N.B.</div>
            </div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">G.R.</div></div>, <div class="name string-name"><div class="surname">Salyers</div> <div class="given-names">A.A.</div></div></div>
          <div class="year">1996</div>, <div class="article-title">The Bacteroides mobilizable insertion element, NBU1, integrates into the 3′ end of a Leu-tRNA gene and has an integrase that is a member of the lambda integrase family</div>, <div class="source">J. Bacteriol.</div> , <div class="volume">178</div>, <div class="fpage">3594</div>–<div class="lpage">600</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Bacteroides%20mobilizable%20insertion%20element%2C%20NBU1%2C%20integrates%20into%20the%203%E2%80%B2%20end%20of%20a%20Leu-tRNA%20gene%20and%20has%20an%20integrase%20that%20is%20a%20member%20of%20the%20lambda%20integrase%20family&amp;author=N.B.%20Shoemaker&amp;author=G.R.%20Wang&amp;author=A.A.%20Salyers&amp;publication_year=1996&amp;journal=J.%20Bacteriol.&amp;volume=178&amp;pages=3594-600" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/jb.178.12.3594-3600.1996" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2Fjb.178.12.3594-3600.1996" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2Fjb.178.12.3594-3600.1996"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8655559" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Shoemaker</div> <div class="given-names">N.B.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">G.R.</div></div>, <div class="name string-name"><div class="surname">Salyers</div> <div class="given-names">A.A.</div></div></div>
          <div class="year">2000</div>, <div class="article-title">Characterization of a Bacteroides mobilizable transposon, NBU2, which carries a functional lincomycin resistance gene</div>, <div class="source">J. Bacteriol.</div> , <div class="volume">182</div>, <div class="fpage">3559</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20Bacteroides%20mobilizable%20transposon%2C%20NBU2%2C%20which%20carries%20a%20functional%20lincomycin%20resistance%20gene&amp;author=J.%20Wang&amp;author=N.B.%20Shoemaker&amp;author=G.R.%20Wang&amp;author=A.A.%20Salyers&amp;publication_year=2000&amp;journal=J.%20Bacteriol.&amp;volume=182&amp;pages=3559-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.182.12.3559-3571.2000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.182.12.3559-3571.2000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.182.12.3559-3571.2000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10852890" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sasaki-Imamura</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Yoshida</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Suwabe</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Yoshimura</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Kato</div> <div class="given-names">H.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Molecular basis of indole production catalyzed by tryptophanase in the genus Prevotella</div>, <div class="source">FEMS Microbiol. Lett.</div> , <div class="volume">322</div>, <div class="fpage">51</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20basis%20of%20indole%20production%20catalyzed%20by%20tryptophanase%20in%20the%20genus%20Prevotella&amp;author=T.%20Sasaki-Imamura&amp;author=Y.%20Yoshida&amp;author=K.%20Suwabe&amp;author=F.%20Yoshimura&amp;author=H.%20Kato&amp;publication_year=2011&amp;journal=FEMS%20Microbiol.%20Lett.&amp;volume=322&amp;pages=51-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/fml.2011.322.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ffml.2011.322.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ffml.2011.322.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21658104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kim</div>
               <div class="given-names">M.J.</div>
            </div>, <div class="name string-name"><div class="surname">Hwang</div> <div class="given-names">K.H.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">Y.S.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">J.Y.</div></div>, <div class="name string-name"><div class="surname">Kook</div> <div class="given-names">J.K.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Development of <em>Prevotella intermedia</em>-specific PCR primers based on the nucleotide sequences of a DNA probe Pig27,</div><div class="source">J. Microbiol. Methods</div> , <div class="volume">84</div>, <div class="fpage">394</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20Prevotella%20intermedia-specific%20PCR%20primers%20based%20on%20the%20nucleotide%20sequences%20of%20a%20DNA%20probe%20Pig27%2C&amp;author=M.J.%20Kim&amp;author=K.H.%20Hwang&amp;author=Y.S.%20Lee&amp;author=J.Y.%20Park&amp;author=J.K.%20Kook&amp;publication_year=2011&amp;journal=J.%20Microbiol.%20Methods&amp;volume=84&amp;pages=394-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mimet.2010.12.019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mimet.2010.12.019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mimet.2010.12.019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21192988" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sakamoto</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Okamoto</div> <div class="given-names">M.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Prevotella aurantiaca sp. nov., isolated from the human oral cavity</div>, <div class="source">Int. J. Syst. Evol. Microbiol.</div> , <div class="volume">60</div>, <div class="fpage">500</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prevotella%20aurantiaca%20sp.%20nov.%2C%20isolated%20from%20the%20human%20oral%20cavity&amp;author=M.%20Sakamoto&amp;author=N.%20Suzuki&amp;author=M.%20Okamoto&amp;publication_year=2010&amp;journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&amp;volume=60&amp;pages=500-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/ijs.0.012831-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fijs.0.012831-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fijs.0.012831-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19654360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sakamoto</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Kumada</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Hamada</div> <div class="given-names">N.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">Prevotella falsenii sp. nov., a <em>Prevotella intermedia</em>-like organism isolated from monkey dental plaque</div>, <div class="source">Int. J. Syst. Evol. Microbiol.</div> , <div class="volume">59</div>, <div class="fpage">319</div>–<div class="lpage">22</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prevotella%20falsenii%20sp.%20nov.%2C%20a%20Prevotella%20intermedia-like%20organism%20isolated%20from%20monkey%20dental%20plaque&amp;author=M.%20Sakamoto&amp;author=H.%20Kumada&amp;author=N.%20Hamada&amp;publication_year=2009&amp;journal=Int.%20J.%20Syst.%20Evol.%20Microbiol.&amp;volume=59&amp;pages=319-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/ijs.0.002626-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fijs.0.002626-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fijs.0.002626-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19196771" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV032C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sato</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Naito</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Yukitake</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">A protein secretion system linked to bacteroidete gliding motility and pathogenesis</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="volume">107</div>, <div class="fpage">276</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20protein%20secretion%20system%20linked%20to%20bacteroidete%20gliding%20motility%20and%20pathogenesis&amp;author=K.%20Sato&amp;author=M.%20Naito&amp;author=H.%20Yukitake&amp;publication_year=2010&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=276-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0912010107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0912010107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0912010107"> </span></div> </div></div></div></div></div><div content-id="DSV032C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV032C33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Nakayama</div>
               <div class="given-names">K.</div>
            </div>
          </div>
          <div class="year">2015</div>, <div class="article-title">Porphyromonas gingivalis and related bacteria: from colonial pigmentation to the type IX secretion system and gliding motility</div>, <div class="source">J. Periodontal. Res.</div> , <div class="volume">50</div>, <div class="fpage">1</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Porphyromonas%20gingivalis%20and%20related%20bacteria%3A%20from%20colonial%20pigmentation%20to%20the%20type%20IX%20secretion%20system%20and%20gliding%20motility&amp;author=K.%20Nakayama&amp;publication_year=2015&amp;journal=J.%20Periodontal.%20Res.&amp;volume=50&amp;pages=1-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/jre.2014.50.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fjre.2014.50.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fjre.2014.50.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25546073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2015. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2462413&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/1/10.1093_dnares_dsv032/1/dsv032_Supplementary_Data.zip?Expires=2147483647&amp;Signature=nTSkUWX8S9fiFWPMHFA4IY5CuT8DzmkH4uPQyjP8aW5v4TAvSd33fIRURL1M5kK9MTrW28MfveBWwO~2qzPLXtyxslcsB6gzVeV88RoD6sGvNeVZ4~Mq64xGXoY2WSMKDUfCyejpK4OUAucJT0XkAfzFz87Bz~jd2JnPmsaAO-aryBXIcnOWF~ifDFVW6apeiVMBZcOrRwwa1VBMfgGfsuhy9gdoqNaNPuhBiY2-D3JHLEmNJSBHtLn9UjCScYgad-9-QM2SDHQ8axZ-NHSF7m1Be9j92aJ6p-r6vJ7bgeVNa3FQXAb0OoaIDVSLNUXvsN75g19Ct2oHURy9ERk2mA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">467</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000370973100002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">4</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv032"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2462413"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000370973100002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f1%2f11%2f2462413">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000370973100002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (4)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F1%2F11%2F2462413" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2462413'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="complete genome sequencing of Prevotella intermedia strain OMA14 and a subsequent fine-scale, intra-species genomic comparison reveal an unusual amplification of conjugative and mobile transposons and identify a novel Prevotella-lineage-specific repeat" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv032");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f1%2f11%2f2462413">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

