

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/4/333/3058514","@type":"ScholarlyArticle","name":"Evidence of translation efficiency adaptation of the coding regions of the bacteriophage lambda","datePublished":"2017-02-28","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2017-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx005","keywords":["decoding rates","bacteriophage genome","bacteriophage fitness","coding regions evolution","ribosomes and tRNAs"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/4/333/3058514","author":[{"name":"Goz, Eli","affiliation":"1 Department of Biomedical Engineering, Tel-Aviv University, Ramat Aviv 69978, Israel   2 SynVaccine Ltd Ramat Hachayal, Tel Aviv 6971039, Israel","@type":"Person"},{"name":"Mioduser, Oriah","affiliation":"1 Department of Biomedical Engineering, Tel-Aviv University, Ramat Aviv 69978, Israel","@type":"Person"},{"name":"Diament, Alon","affiliation":"1 Department of Biomedical Engineering, Tel-Aviv University, Ramat Aviv 69978, Israel","@type":"Person"},{"name":"Tuller, Tamir","affiliation":"1 Department of Biomedical Engineering, Tel-Aviv University, Ramat Aviv 69978, Israel   2 SynVaccine Ltd Ramat Hachayal, Tel Aviv 6971039, Israel   3 Sagol School of Neuroscience, Tel-Aviv University, Ramat Aviv 69978, Israel","@type":"Person"}],"description":"Abstract.  Deciphering the way gene expression regulatory aspects are encoded in viral genomes is a challenging mission with ramifications related to all biomed","pageStart":"333","pageEnd":"342","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/4/1/m_dnares_24_4cover.png?Expires=2147483647&Signature=Bn68nrHzMnl3Hf0TBxDUSpIiE3ALYvws4lUuOsXgZQeZw~qHN97KSJf53QRVRMw9UIEwsJhU~oIqlbc~bNDrFxzLTkzmyUjnAZ7sS1Ope7v8mL4P5X8GIjyJTBx96xx3yzaFdQ9im8bglRhXw7RCd3PHFDaTYDb24rkCGDLkkLUBVi~BntzWxZrnH-LI5Yr~cYB3mJnIO~ISuJYAH~ZtK9rwdBnTT~rg4DIDb~I6Y2wsrkFjsIdVm-sIz91ZlsrHMWugLRT8lXkXDxrnR3~Cwmkt8-yfu-WWcXNP8U4bmwXopL-J4wp6s4jITl~IopCYMRAqzqwzkZxKkX6zWcYGMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Evidence of translation efficiency adaptation of the coding regions of the bacteriophage lambda","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/4/1/m_dnares_24_4cover.png?Expires=2147483647&Signature=Bn68nrHzMnl3Hf0TBxDUSpIiE3ALYvws4lUuOsXgZQeZw~qHN97KSJf53QRVRMw9UIEwsJhU~oIqlbc~bNDrFxzLTkzmyUjnAZ7sS1Ope7v8mL4P5X8GIjyJTBx96xx3yzaFdQ9im8bglRhXw7RCd3PHFDaTYDb24rkCGDLkkLUBVi~BntzWxZrnH-LI5Yr~cYB3mJnIO~ISuJYAH~ZtK9rwdBnTT~rg4DIDb~I6Y2wsrkFjsIdVm-sIz91ZlsrHMWugLRT8lXkXDxrnR3~Cwmkt8-yfu-WWcXNP8U4bmwXopL-J4wp6s4jITl~IopCYMRAqzqwzkZxKkX6zWcYGMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Evidence of translation efficiency adaptation of the coding regions of the bacteriophage lambda" />
    <meta property="og:description" content="Abstract.  Deciphering the way gene expression regulatory aspects are encoded in viral genomes is a challenging mission with ramifications related to all biomed" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/4/333/3058514" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-02-28" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Goz, Eli" /><meta name="citation_author_institution" content="Department of Biomedical Engineering, Tel-Aviv University, Ramat Aviv 69978, Israel" /><meta name="citation_author_institution" content="SynVaccine Ltd Ramat Hachayal, Tel Aviv 6971039, Israel" /><meta name="citation_author" content="Mioduser, Oriah" /><meta name="citation_author_institution" content="Department of Biomedical Engineering, Tel-Aviv University, Ramat Aviv 69978, Israel" /><meta name="citation_author" content="Diament, Alon" /><meta name="citation_author_institution" content="Department of Biomedical Engineering, Tel-Aviv University, Ramat Aviv 69978, Israel" /><meta name="citation_author" content="Tuller, Tamir" /><meta name="citation_author_institution" content="Department of Biomedical Engineering, Tel-Aviv University, Ramat Aviv 69978, Israel" /><meta name="citation_author_institution" content="SynVaccine Ltd Ramat Hachayal, Tel Aviv 6971039, Israel" /><meta name="citation_author_institution" content="Sagol School of Neuroscience, Tel-Aviv University, Ramat Aviv 69978, Israel" /><meta name="citation_title" content="Evidence of translation efficiency adaptation of the coding regions of the bacteriophage lambda" /><meta name="citation_firstpage" content="333" /><meta name="citation_lastpage" content="342" /><meta name="citation_doi" content="10.1093/dnares/dsx005" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2017/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Bacteriophages and their implications on future biotechnology: a review;  citation_journal_title=Virol. J;  citation_year=2012;  citation_volume=9; " /><meta name="citation_reference" content="citation_title=Bacteriophages and biotechnology: vaccines, gene therapy and antibacterials;  citation_journal_title=Trends Biotechnol;  citation_year=2006;  citation_volume=24;  citation_pages=212-218; " /><meta name="citation_reference" content="citation_title=Taking aim on bacterial pathogens: from phage therapy to enzybiotics;  citation_journal_title=Curr. Opin. Microbiol;  citation_year=2007;  citation_volume=10;  citation_pages=461-472; " /><meta name="citation_reference" content="citation_title=Experimental molecular evolution of bacteriophage-T7;  citation_journal_title=Evolution;  citation_year=1993;  citation_volume=47;  citation_pages=993-1007; " /><meta name="citation_reference" content="citation_title=KSHV 2.0: a comprehensive annotation of the Kaposi's sarcoma-associated herpesvirus genome using nextgeneration sequencing reveals novel genomic and functional features;  citation_journal_title=Plos Pathog;  citation_year=2014;  citation_volume=10; " /><meta name="citation_reference" content="citation_title=Deciphering poxvirus gene expression by RNA sequencing and ribosome profiling;  citation_journal_title=J. Virol;  citation_year=2015;  citation_volume=89;  citation_pages=6874-6886; " /><meta name="citation_reference" content="  Stern-Ginossar N., Ingolia N.T.  2015 , Ribosome profiling as a tool to decipher viral complexity. In: L.W., Enquist, (ed.), Annual Review of Virology, Vol. 2 , pp. 335 – 349. " /><meta name="citation_reference" content="citation_title=High-resolution analysis of coronavirus gene expression by RNA sequencing and ribosome profiling;  citation_journal_title=Plos Pathog;  citation_year=2016;  citation_volume=12; " /><meta name="citation_reference" content="citation_title=High-resolution view of bacteriophage lambda gene expression by ribosome profiling;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2013;  citation_volume=110;  citation_pages=11928-11933; " /><meta name="citation_reference" content="citation_title=Codon bias is a major factor explaining phage evolution in translationally biased hosts;  citation_journal_title=J. Mol. Evol;  citation_year=2008;  citation_volume=66;  citation_pages=210-223; " /><meta name="citation_reference" content="citation_title=Virus attenuation by genome-scale changes in codon pair bias;  citation_journal_title=Science;  citation_year=2008;  citation_volume=320;  citation_pages=1784-1787; " /><meta name="citation_reference" content="citation_title=Evolution of base composition and codon usage bias in the genus Flavivirus;  citation_journal_title=J. Mol. Evol;  citation_year=2001;  citation_volume=52;  citation_pages=383-390; " /><meta name="citation_reference" content="citation_title=Evolutionary basis of codon usage and nucleotide composition bias in vertebrate DNA viruses;  citation_journal_title=J. Mol. Evol;  citation_year=2006;  citation_volume=62;  citation_pages=551-563; " /><meta name="citation_reference" content="citation_title=Control of gene-expression in bacteriophage-lambda;  citation_journal_title=Ann. Rev. Genet;  citation_year=1973;  citation_volume=7;  citation_pages=289-324; " /><meta name="citation_reference" content="citation_title=Bacteriophage lambda: early pioneer and still relevant;  citation_journal_title=Virology;  citation_year=2015;  citation_volume=479;  citation_pages=310-330; " /><meta name="citation_reference" content="citation_title=Evidence of a direct evolutionary selection for strong folding and mutational robustness within HIV coding regions;  citation_journal_title=J. Comput. Biol;  citation_year=2016;  citation_volume=23; " /><meta name="citation_reference" content="citation_title=Widespread signatures of local mRNA folding structure selection in four Dengue virus serotypes;  citation_journal_title=BMC Genom;  citation_year=2015;  citation_volume=16; " /><meta name="citation_reference" content="citation_title=Evidence of a novel RNA secondary structure in the coding region of HIV-1 pol gene;  citation_journal_title=RNA-Publ. RNA Soc;  citation_year=2008;  citation_volume=14;  citation_pages=2478-2488; " /><meta name="citation_reference" content="citation_title=Role of RNA structures present at the 3ʹ UTR of dengue virus on translation, RNA synthesis, and viral replication;  citation_journal_title=Virology;  citation_year=2005;  citation_volume=339;  citation_pages=200-212; " /><meta name="citation_reference" content="citation_title=variation of the hepatitis-C virus 5ʹ noncoding region – implications for secondary structure, virus detection and typing;  citation_journal_title=J. Gener. Virol;  citation_year=1995;  citation_volume=76;  citation_pages=1749-1761; " /><meta name="citation_reference" content="citation_title=The extent of codon usage bias in human RNA viruses and its evolutionary origin;  citation_journal_title=Virus Res;  citation_year=2003;  citation_volume=92;  citation_pages=1-7; " /><meta name="citation_reference" content="citation_title=selection pressures on codon usage in the complete genome of bacteriophage-T7;  citation_journal_title=J. Mol. Evol;  citation_year=1985;  citation_volume=21;  citation_pages=150-160; " /><meta name="citation_reference" content="citation_title=Comparative analysis of the base composition and codon usages in fourteen mycobacteriophage genomes;  citation_journal_title=J. Biomol. Struct. Dyn;  citation_year=2005;  citation_volume=23;  citation_pages=63-71; " /><meta name="citation_reference" content="citation_title=Synonymous codon usage bias in 16 Staphylococcus aureus phages: implication in phage therapy;  citation_journal_title=Virus Res;  citation_year=2005;  citation_volume=113;  citation_pages=123-131; " /><meta name="citation_reference" content="citation_title=Comparison of synonymous codon distribution patterns of bacteriophage and host genomes;  citation_journal_title=DNA Res;  citation_year=1998;  citation_volume=5;  citation_pages=319-326; " /><meta name="citation_reference" content="citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads;  citation_journal_title=EMBnet J;  citation_year=2011;  citation_volume=17;  citation_pages=2226-6089; " /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome;  citation_journal_title=Genome Biol;  citation_year=2009;  citation_volume=10; " /><meta name="citation_reference" content="citation_title=SPARCS: a web server to analyze (un)structured regions in coding RNA sequences;  citation_journal_title=Nucl. Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=W480-W485; " /><meta name="citation_reference" content="citation_title=Solving the riddle of codon usage preferences: a test for translational selection;  citation_journal_title=Nucl. Acids Res;  citation_year=2004;  citation_volume=32;  citation_pages=5036-5044; " /><meta name="citation_reference" content="citation_title=The effect of tRNA levels on decoding times of mRNA codons;  citation_journal_title=Nucl. Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=9171-9181; " /><meta name="citation_reference" content="citation_title=ViennaRNA package 2.0;  citation_journal_title=Algorithms Mol. Biol;  citation_year=2011;  citation_volume=6; " /><meta name="citation_reference" content="citation_title=Translation efficiency is determined by both codon bias and folding energy;  citation_journal_title=Proc. Natl. Acad. Sci. U. S. A;  citation_year=2010;  citation_volume=107;  citation_pages=3645-3650; " /><meta name="citation_reference" content="citation_title=Optimal computer folding of large RNA sequences using thermodynamics and auxiliary information;  citation_journal_title=Nucl. Acids Res;  citation_year=1981;  citation_volume=9;  citation_pages=133-148; " /><meta name="citation_reference" content="citation_title=Multiple roles of the coding sequence 5ʹ end in gene expression regulation;  citation_journal_title=Nucl. Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=13-28; " /><meta name="citation_reference" content="citation_title=Exploiting hidden information interleaved in the redundancy of the genetic code without prior knowledge;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=1161-1168; " /><meta name="citation_reference" content="citation_title=An evolutionarily conserved mechanism for controlling the efficiency of protein translation;  citation_journal_title=Cell;  citation_year=2010;  citation_volume=141;  citation_pages=344-354; " /><meta name="citation_reference" content="citation_title=Conservation of the relative tRNA composition in healthy and cancerous tissues;  citation_journal_title=RNA-Publ. RNA Soc;  citation_year=2012;  citation_volume=18;  citation_pages=640-652; " /><meta name="citation_reference" content="citation_title=Codon usage influences the local rate of translation elongation to regulate co-translational protein folding;  citation_journal_title=Mol. Cell;  citation_year=2015;  citation_volume=59;  citation_pages=744-754; " /><meta name="citation_reference" content="citation_title=Modelling the efficiency of codon–tRNA interactions based on codon usage bias;  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=511-525; " /><meta name="citation_reference" content="citation_title=Estimating translational selection in eukaryotic genomes;  citation_journal_title=Mol. Biol. Evol;  citation_year=2009;  citation_volume=26;  citation_pages=451-461; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/4/333/22892815/dsx005.pdf" /><meta name="description" content="Abstract.  Deciphering the way gene expression regulatory aspects are encoded in viral genomes is a challenging mission with ramifications related to all biomed" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/4/333/3058514" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/4/333/3058514" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/4/333/3058514");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx005';
    </script>
<title>
	Evidence of translation efficiency adaptation of the coding regions of the bacteriophage lambda | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3058514" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BehdPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHb3osIEVsaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbWVkaWNhbCBFbmdpbmVlcmluZywgVGVsLUF2aXYgVW5pdmVyc2l0eSwgUmFtYXQgQXZpdiA2OTk3OCwgSXNyYWVsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3luVmFjY2luZSBMdGQgUmFtYXQgSGFjaGF5YWwsIFRlbCBBdml2IDY5NzEwMzksIElzcmFlbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1pb2R1c2VyLCBPcmlhaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbWVkaWNhbCBFbmdpbmVlcmluZywgVGVsLUF2aXYgVW5pdmVyc2l0eSwgUmFtYXQgQXZpdiA2OTk3OCwgSXNyYWVsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRGlhbWVudCwgQWxvbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbWVkaWNhbCBFbmdpbmVlcmluZywgVGVsLUF2aXYgVW5pdmVyc2l0eSwgUmFtYXQgQXZpdiA2OTk3OCwgSXNyYWVsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVHVsbGVyLCBUYW1pciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbWVkaWNhbCBFbmdpbmVlcmluZywgVGVsLUF2aXYgVW5pdmVyc2l0eSwgUmFtYXQgQXZpdiA2OTk3OCwgSXNyYWVsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3luVmFjY2luZSBMdGQgUmFtYXQgSGFjaGF5YWwsIFRlbCBBdml2IDY5NzEwMzksIElzcmFlbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNhZ29sIFNjaG9vbCBvZiBOZXVyb3NjaWVuY2UsIFRlbC1Bdml2IFVuaXZlcnNpdHksIFJhbWF0IEF2aXYgNjk5NzgsIElzcmFlbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRXZpZGVuY2Ugb2YgdHJhbnNsYXRpb24gZWZmaWNpZW5jeSBhZGFwdGF0aW9uIG9mIHRoZSBjb2RpbmcgcmVnaW9ucyBvZiB0aGUgYmFjdGVyaW9waGFnZSBsYW1iZGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzMzMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjM0MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeDAwNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE3LzA4LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjdGVyaW9waGFnZXMgYW5kIHRoZWlyIGltcGxpY2F0aW9ucyBvbiBmdXR1cmUgYmlvdGVjaG5vbG9neTogYSByZXZpZXc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVZpcm9sLiBKOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhY3RlcmlvcGhhZ2VzIGFuZCBiaW90ZWNobm9sb2d5OiB2YWNjaW5lcywgZ2VuZSB0aGVyYXB5IGFuZCBhbnRpYmFjdGVyaWFsczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0yMTItMjE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYWtpbmcgYWltIG9uIGJhY3RlcmlhbCBwYXRob2dlbnM6IGZyb20gcGhhZ2UgdGhlcmFweSB0byBlbnp5YmlvdGljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3Vyci4gT3Bpbi4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NDYxLTQ3MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwZXJpbWVudGFsIG1vbGVjdWxhciBldm9sdXRpb24gb2YgYmFjdGVyaW9waGFnZS1UNzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RXZvbHV0aW9uOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9OTkzLTEwMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUtTSFYgMi4wOiBhIGNvbXByZWhlbnNpdmUgYW5ub3RhdGlvbiBvZiB0aGUgS2Fwb3NpJ3Mgc2FyY29tYS1hc3NvY2lhdGVkIGhlcnBlc3ZpcnVzIGdlbm9tZSB1c2luZyBuZXh0Z2VuZXJhdGlvbiBzZXF1ZW5jaW5nIHJldmVhbHMgbm92ZWwgZ2Vub21pYyBhbmQgZnVuY3Rpb25hbCBmZWF0dXJlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxvcyBQYXRob2c7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlY2lwaGVyaW5nIHBveHZpcnVzIGdlbmUgZXhwcmVzc2lvbiBieSBSTkEgc2VxdWVuY2luZyBhbmQgcmlib3NvbWUgcHJvZmlsaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBWaXJvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT04OTsgIGNpdGF0aW9uX3BhZ2VzPTY4NzQtNjg4NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iICBTdGVybi1HaW5vc3NhciBOLiwgSW5nb2xpYSBOLlQuICAyMDE1ICwgUmlib3NvbWUgcHJvZmlsaW5nIGFzIGEgdG9vbCB0byBkZWNpcGhlciB2aXJhbCBjb21wbGV4aXR5LiBJbjogTC5XLiwgRW5xdWlzdCwgKGVkLiksIEFubnVhbCBSZXZpZXcgb2YgVmlyb2xvZ3ksIFZvbC4gMiAsIHBwLiAzMzUg4oCTIDM0OS4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1yZXNvbHV0aW9uIGFuYWx5c2lzIG9mIGNvcm9uYXZpcnVzIGdlbmUgZXhwcmVzc2lvbiBieSBSTkEgc2VxdWVuY2luZyBhbmQgcmlib3NvbWUgcHJvZmlsaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbG9zIFBhdGhvZzsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1yZXNvbHV0aW9uIHZpZXcgb2YgYmFjdGVyaW9waGFnZSBsYW1iZGEgZ2VuZSBleHByZXNzaW9uIGJ5IHJpYm9zb21lIHByb2ZpbGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTExMDsgIGNpdGF0aW9uX3BhZ2VzPTExOTI4LTExOTMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiBiaWFzIGlzIGEgbWFqb3IgZmFjdG9yIGV4cGxhaW5pbmcgcGhhZ2UgZXZvbHV0aW9uIGluIHRyYW5zbGF0aW9uYWxseSBiaWFzZWQgaG9zdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIE1vbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT02NjsgIGNpdGF0aW9uX3BhZ2VzPTIxMC0yMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZpcnVzIGF0dGVudWF0aW9uIGJ5IGdlbm9tZS1zY2FsZSBjaGFuZ2VzIGluIGNvZG9uIHBhaXIgYmlhczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0zMjA7ICBjaXRhdGlvbl9wYWdlcz0xNzg0LTE3ODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBiYXNlIGNvbXBvc2l0aW9uIGFuZCBjb2RvbiB1c2FnZSBiaWFzIGluIHRoZSBnZW51cyBGbGF2aXZpcnVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NTI7ICBjaXRhdGlvbl9wYWdlcz0zODMtMzkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgYmFzaXMgb2YgY29kb24gdXNhZ2UgYW5kIG51Y2xlb3RpZGUgY29tcG9zaXRpb24gYmlhcyBpbiB2ZXJ0ZWJyYXRlIEROQSB2aXJ1c2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz01NTEtNTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db250cm9sIG9mIGdlbmUtZXhwcmVzc2lvbiBpbiBiYWN0ZXJpb3BoYWdlLWxhbWJkYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW5uLiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0xOTczOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0yODktMzI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYWN0ZXJpb3BoYWdlIGxhbWJkYTogZWFybHkgcGlvbmVlciBhbmQgc3RpbGwgcmVsZXZhbnQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVZpcm9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTQ3OTsgIGNpdGF0aW9uX3BhZ2VzPTMxMC0zMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIG9mIGEgZGlyZWN0IGV2b2x1dGlvbmFyeSBzZWxlY3Rpb24gZm9yIHN0cm9uZyBmb2xkaW5nIGFuZCBtdXRhdGlvbmFsIHJvYnVzdG5lc3Mgd2l0aGluIEhJViBjb2RpbmcgcmVnaW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQ29tcHV0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaWRlc3ByZWFkIHNpZ25hdHVyZXMgb2YgbG9jYWwgbVJOQSBmb2xkaW5nIHN0cnVjdHVyZSBzZWxlY3Rpb24gaW4gZm91ciBEZW5ndWUgdmlydXMgc2Vyb3R5cGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub207ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIG9mIGEgbm92ZWwgUk5BIHNlY29uZGFyeSBzdHJ1Y3R1cmUgaW4gdGhlIGNvZGluZyByZWdpb24gb2YgSElWLTEgcG9sIGdlbmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVJOQS1QdWJsLiBSTkEgU29jOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9MjQ3OC0yNDg4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb2xlIG9mIFJOQSBzdHJ1Y3R1cmVzIHByZXNlbnQgYXQgdGhlIDPKuSBVVFIgb2YgZGVuZ3VlIHZpcnVzIG9uIHRyYW5zbGF0aW9uLCBSTkEgc3ludGhlc2lzLCBhbmQgdmlyYWwgcmVwbGljYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVZpcm9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMzOTsgIGNpdGF0aW9uX3BhZ2VzPTIwMC0yMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPXZhcmlhdGlvbiBvZiB0aGUgaGVwYXRpdGlzLUMgdmlydXMgNcq5IG5vbmNvZGluZyByZWdpb24g4oCTIGltcGxpY2F0aW9ucyBmb3Igc2Vjb25kYXJ5IHN0cnVjdHVyZSwgdmlydXMgZGV0ZWN0aW9uIGFuZCB0eXBpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEdlbmVyLiBWaXJvbDsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT03NjsgIGNpdGF0aW9uX3BhZ2VzPTE3NDktMTc2MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGV4dGVudCBvZiBjb2RvbiB1c2FnZSBiaWFzIGluIGh1bWFuIFJOQSB2aXJ1c2VzIGFuZCBpdHMgZXZvbHV0aW9uYXJ5IG9yaWdpbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VmlydXMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTkyOyAgY2l0YXRpb25fcGFnZXM9MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1zZWxlY3Rpb24gcHJlc3N1cmVzIG9uIGNvZG9uIHVzYWdlIGluIHRoZSBjb21wbGV0ZSBnZW5vbWUgb2YgYmFjdGVyaW9waGFnZS1UNzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MTUwLTE2MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgYW5hbHlzaXMgb2YgdGhlIGJhc2UgY29tcG9zaXRpb24gYW5kIGNvZG9uIHVzYWdlcyBpbiBmb3VydGVlbiBteWNvYmFjdGVyaW9waGFnZSBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCaW9tb2wuIFN0cnVjdC4gRHluOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9NjMtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5bm9ueW1vdXMgY29kb24gdXNhZ2UgYmlhcyBpbiAxNiBTdGFwaHlsb2NvY2N1cyBhdXJldXMgcGhhZ2VzOiBpbXBsaWNhdGlvbiBpbiBwaGFnZSB0aGVyYXB5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1WaXJ1cyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9MTIzLTEzMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBvZiBzeW5vbnltb3VzIGNvZG9uIGRpc3RyaWJ1dGlvbiBwYXR0ZXJucyBvZiBiYWN0ZXJpb3BoYWdlIGFuZCBob3N0IGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTMxOS0zMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN1dGFkYXB0IHJlbW92ZXMgYWRhcHRlciBzZXF1ZW5jZXMgZnJvbSBoaWdoLXRocm91Z2hwdXQgc2VxdWVuY2luZyByZWFkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RU1CbmV0IEo7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0yMjI2LTYwODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVsdHJhZmFzdCBhbmQgbWVtb3J5LWVmZmljaWVudCBhbGlnbm1lbnQgb2Ygc2hvcnQgRE5BIHNlcXVlbmNlcyB0byB0aGUgaHVtYW4gZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U1BBUkNTOiBhIHdlYiBzZXJ2ZXIgdG8gYW5hbHl6ZSAodW4pc3RydWN0dXJlZCByZWdpb25zIGluIGNvZGluZyBSTkEgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsLiBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz1XNDgwLVc0ODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNvbHZpbmcgdGhlIHJpZGRsZSBvZiBjb2RvbiB1c2FnZSBwcmVmZXJlbmNlczogYSB0ZXN0IGZvciB0cmFuc2xhdGlvbmFsIHNlbGVjdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbC4gQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9NTAzNi01MDQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZWZmZWN0IG9mIHRSTkEgbGV2ZWxzIG9uIGRlY29kaW5nIHRpbWVzIG9mIG1STkEgY29kb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsLiBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz05MTcxLTkxODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZpZW5uYVJOQSBwYWNrYWdlIDIuMDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWxnb3JpdGhtcyBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNsYXRpb24gZWZmaWNpZW5jeSBpcyBkZXRlcm1pbmVkIGJ5IGJvdGggY29kb24gYmlhcyBhbmQgZm9sZGluZyBlbmVyZ3k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS4gUy4gQTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDc7ICBjaXRhdGlvbl9wYWdlcz0zNjQ1LTM2NTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU9wdGltYWwgY29tcHV0ZXIgZm9sZGluZyBvZiBsYXJnZSBSTkEgc2VxdWVuY2VzIHVzaW5nIHRoZXJtb2R5bmFtaWNzIGFuZCBhdXhpbGlhcnkgaW5mb3JtYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2wuIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTMzLTE0ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TXVsdGlwbGUgcm9sZXMgb2YgdGhlIGNvZGluZyBzZXF1ZW5jZSA1yrkgZW5kIGluIGdlbmUgZXhwcmVzc2lvbiByZWd1bGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsLiBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xMy0yODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwbG9pdGluZyBoaWRkZW4gaW5mb3JtYXRpb24gaW50ZXJsZWF2ZWQgaW4gdGhlIHJlZHVuZGFuY3kgb2YgdGhlIGdlbmV0aWMgY29kZSB3aXRob3V0IHByaW9yIGtub3dsZWRnZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0xMTYxLTExNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGV2b2x1dGlvbmFyaWx5IGNvbnNlcnZlZCBtZWNoYW5pc20gZm9yIGNvbnRyb2xsaW5nIHRoZSBlZmZpY2llbmN5IG9mIHByb3RlaW4gdHJhbnNsYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTQxOyAgY2l0YXRpb25fcGFnZXM9MzQ0LTM1NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29uc2VydmF0aW9uIG9mIHRoZSByZWxhdGl2ZSB0Uk5BIGNvbXBvc2l0aW9uIGluIGhlYWx0aHkgYW5kIGNhbmNlcm91cyB0aXNzdWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1STkEtUHVibC4gUk5BIFNvYzsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTY0MC02NTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGluZmx1ZW5jZXMgdGhlIGxvY2FsIHJhdGUgb2YgdHJhbnNsYXRpb24gZWxvbmdhdGlvbiB0byByZWd1bGF0ZSBjby10cmFuc2xhdGlvbmFsIHByb3RlaW4gZm9sZGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU5OyAgY2l0YXRpb25fcGFnZXM9NzQ0LTc1NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9kZWxsaW5nIHRoZSBlZmZpY2llbmN5IG9mIGNvZG9u4oCTdFJOQSBpbnRlcmFjdGlvbnMgYmFzZWQgb24gY29kb24gdXNhZ2UgYmlhczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTUxMS01MjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVzdGltYXRpbmcgdHJhbnNsYXRpb25hbCBzZWxlY3Rpb24gaW4gZXVrYXJ5b3RpYyBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz00NTEtNDYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjQvNC8zMzMvMjI4OTI4MTUvZHN4MDA1LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBEZWNpcGhlcmluZyB0aGUgd2F5IGdlbmUgZXhwcmVzc2lvbiByZWd1bGF0b3J5IGFzcGVjdHMgYXJlIGVuY29kZWQgaW4gdmlyYWwgZ2Vub21lcyBpcyBhIGNoYWxsZW5naW5nIG1pc3Npb24gd2l0aCByYW1pZmljYXRpb25zIHJlbGF0ZWQgdG8gYWxsIGJpb21lZCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzA1ODUxNCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQ6KlkEuUmUE+cjoiMIBdYItuOTZA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f333%2f3058514" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f333%2f3058514" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=98073" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/4/1/m_dnares_24_4cover.png?Expires=2147483647&amp;Signature=Bn68nrHzMnl3Hf0TBxDUSpIiE3ALYvws4lUuOsXgZQeZw~qHN97KSJf53QRVRMw9UIEwsJhU~oIqlbc~bNDrFxzLTkzmyUjnAZ7sS1Ope7v8mL4P5X8GIjyJTBx96xx3yzaFdQ9im8bglRhXw7RCd3PHFDaTYDb24rkCGDLkkLUBVi~BntzWxZrnH-LI5Yr~cYB3mJnIO~ISuJYAH~ZtK9rwdBnTT~rg4DIDb~I6Y2wsrkFjsIdVm-sIz91ZlsrHMWugLRT8lXkXDxrnR3~Cwmkt8-yfu-WWcXNP8U4bmwXopL-J4wp6s4jITl~IopCYMRAqzqwzkZxKkX6zWcYGMA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/4">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="106354678">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106354678">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106354680">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106354680">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106354686">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106354686">2. Material and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106354730">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106354730">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106354756">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106354756">4. Conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106354765">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106354765">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106354767">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106354767">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106354771">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106354771">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/24/4/343/3056574">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Evidence of translation efficiency adaptation of the coding regions of the bacteriophage lambda
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Eli Goz</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Biomedical Engineering, Tel-Aviv University, Ramat Aviv 69978, Israel</div><div class="aff"><div class="label -label">2</div>SynVaccine Ltd Ramat Hachayal, Tel Aviv 6971039, Israel</div>
        </div>

                <div class="info-card-note">
                    <p><sup>†</sup>These authors contributed equally to this study. Edited by Prof. Kenta Nakai</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Eli+Goz">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Goz E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Goz E&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Eli Goz</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Oriah Mioduser</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Biomedical Engineering, Tel-Aviv University, Ramat Aviv 69978, Israel</div>
        </div>

                <div class="info-card-note">
                    <p><sup>†</sup>These authors contributed equally to this study. Edited by Prof. Kenta Nakai</p>
                </div>




        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Oriah+Mioduser">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mioduser O">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mioduser O&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Oriah Mioduser</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Alon Diament</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Biomedical Engineering, Tel-Aviv University, Ramat Aviv 69978, Israel</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Alon+Diament">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Diament A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Diament A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Alon Diament</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tamir Tuller</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Department of Biomedical Engineering, Tel-Aviv University, Ramat Aviv 69978, Israel</div><div class="aff"><div class="label -label">2</div>SynVaccine Ltd Ramat Hachayal, Tel Aviv 6971039, Israel</div><div class="aff"><div class="label -label">3</div>Sagol School of Neuroscience, Tel-Aviv University, Ramat Aviv 69978, Israel</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx005-cor1" data-open="dsx005-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. 972 - 3-6405839. Fax. 972 - 3-6407308. Email: <a href="mailto:tamirtul@post.tau.ac.il" target="_blank">tamirtul@post.tau.ac.il</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tamir+Tuller">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tuller T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tuller T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tamir Tuller</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx005-FM1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this study. Edited by Prof. Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 4, 1 August 2017, Pages 333–342, <a href='https://doi.org/10.1093/dnares/dsx005'>https://doi.org/10.1093/dnares/dsx005</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">28 February 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">06 October 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">23 January 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">01 February 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3058514" href="/dnaresearch/article-pdf/24/4/333/22892815/dsx005.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Eli Goz, Oriah Mioduser, Alon Diament, Tamir Tuller;  Evidence of translation efficiency adaptation of the coding regions of the bacteriophage lambda, <em>DNA Research</em>, Volume 24, Issue 4, 1 August 2017, Pages 333–342, <a href='https://doi.org/10.1093/dnares/dsx005'>https://doi.org/10.1093/dnares/dsx005</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3058514?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3058514?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3058514?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3058514?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3058514?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3058514?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3058514?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Evidence%20of%20translation%20efficiency%20adaptation%20of%20the%20coding%20regions%20of%20the%20bacteriophage%20lambda&amp;publicationDate=2017-02-28&amp;volumeNum=24&amp;issueNum=4&amp;author=Goz%2C%20Eli%3B%20Mioduser%2C%20Oriah&amp;startPage=333&amp;endPage=342&amp;contentId=10.1093%2Fdnares%2Fdsx005&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f333%2f3058514" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=106354678 id="106354678" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Deciphering the way gene expression regulatory aspects are encoded in viral genomes is a challenging mission with ramifications related to all biomedical disciplines. Here, we aimed to understand how the evolution shapes the bacteriophage lambda genes by performing a high resolution analysis of ribosomal profiling data and gene expression related synonymous/silent information encoded in bacteriophage coding regions.</p><p>We demonstrated evidence of selection for distinct compositions of synonymous codons in early and late viral genes related to the adaptation of translation efficiency to different bacteriophage developmental stages. Specifically, we showed that evolution of viral coding regions is driven, among others, by selection for codons with higher decoding rates; during the initial/progressive stages of infection the decoding rates in early/late genes were found to be superior to those in late/early genes, respectively. Moreover, we argued that selection for translation efficiency could be partially explained by adaptation to <em>Escherichia coli</em> tRNA pool and the fact that it can change during the bacteriophage life cycle.</p><p>An analysis of additional aspects related to the expression of viral genes, such as mRNA folding and more complex/longer regulatory signals in the coding regions, is also reported. The reported conclusions are likely to be relevant also to additional viruses.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;decoding rates&quot;">decoding rates</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;bacteriophage genome&quot;">bacteriophage genome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;bacteriophage fitness&quot;">bacteriophage fitness</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;coding regions evolution&quot;">coding regions evolution</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;ribosomes and tRNAs&quot;">ribosomes and tRNAs</a></div>                            <h2 scrollto-destination=106354680 id="106354680" class="section-title" >1. Introduction</h2>
<p>Because of a major advantage of being a diverse group of easily manipulated viruses, bacteriophages have various potential uses both in fundamental research and in various biotechnological and biomedical applications. For example, they are used as vehicles for vaccines (both DNA and protein), for the detection of pathogenic bacterial strains, and as a display system for many proteins and antibodies.<a reveal-id="dsx005-B1" data-open="dsx005-B1" class="link link-ref link-reveal xref-bibr"><sup>1–4</sup></a> Furthermore, phages were also suggested to be used as biocontrol agents in agriculture and petroleum industry, and as alternatives to antibiotics in the case of antibiotic resistant bacterial strains. In addition, they often serve as model organisms in molecular evolution studies.<a reveal-id="dsx005-B1" data-open="dsx005-B1" class="link link-ref link-reveal xref-bibr"><sup>1–4</sup></a> Therefore, understanding the way the viral fitness is encoded in the genetic material of bacteriophages (or other type of viruses) is an important and challenging mission that may potentially contribute to all biomedical disciplines.<a reveal-id="dsx005-B1" data-open="dsx005-B1" class="link link-ref link-reveal xref-bibr"><sup>1–4</sup></a></p><p>Deciphering the regulatory information encoded in the genomes of phages or other viruses, and the relation between the nucleotides composition of the coding regions and the viral fitness is a very challenging mission, which was tackled, in the recent years, by various researchers.<a reveal-id="dsx005-B5" data-open="dsx005-B5" class="link link-ref link-reveal xref-bibr"><sup>5–13</sup></a> Among others, it was suggested that ribosome profiling, which enables the large <em>in vivo</em> genome wide monitoring of ribosome state at a resolution of single nucleotide, is a very useful tool for deciphering the coding complexity of viral (and other organisms) genomes. Specifically, it was shown that ribosome profiling enables detecting novel (possibly very short) coding regions and estimate the translation status of various open reading frames.<a reveal-id="dsx005-B5" data-open="dsx005-B5" class="link link-ref link-reveal xref-bibr"><sup>5–9</sup></a></p><p>Here, we focus on the Bacteriophage lambda which is a well-known and studied member of the Siphoviridae family of double-stranded DNA viruses in the Caudovirales order (also known as ‘tailed bacteriophages’ due to their characteristic form). During its lifecycle this phage either resides within the genome of its <em>Escherichia coli</em> host through lysogeny or enters into a lytic phase (which lasts ∼25 min) during which it produces progeny viral particles, and kills and lyses the cell (see for example<a reveal-id="dsx005-B14" data-open="dsx005-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsx005-B15" data-open="dsx005-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a>). The genome size of the bacteriophage lambda is ∼50 kb nt and includes 66 known genes that were analysed in this study. These genes can be divided into two groups, ‘early’ and ‘late’ according to the stages in the lytic phase when their expression is dominant.<a reveal-id="dsx005-B9" data-open="dsx005-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a></p><p>The specific aim of this study is at exploring the way in which translation efficiency related information is encoded on a synonymous level in the coding regions of genes that are expressed during different bacteriophage lambda development stages (i.e. 0–20 min after the beginning of the lytic phase). Some previous studies shed some light on specific elements related to this topic. For example, a recent study by Liu et al.<a reveal-id="dsx005-B9" data-open="dsx005-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> employed ribosome profiling to study the progress of bacteriophage lambda gene expression during phage development and showed that the known genes are expressed in a predictable fashion; in addition many previously unknown potential open reading frames were detected. Other studies focused on different aspects of viral translation (and lifecycle in general) regulation that has a significant effect on shaping viral genomes, such as: secondary structures within viral transcripts (mainly in untranslated regions<a reveal-id="dsx005-B16" data-open="dsx005-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsx005-B17" data-open="dsx005-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> but also within coding regions<a reveal-id="dsx005-B18" data-open="dsx005-B18" class="link link-ref link-reveal xref-bibr"><sup>18–20</sup></a>) and evolutionary pressure on synonymous codons usage bias.<a reveal-id="dsx005-B21" data-open="dsx005-B21" class="link link-ref link-reveal xref-bibr"><sup>21–25</sup></a></p><p>However, to the best of our knowledge, this is the first study aiming to perform a comprehensive, large scale analysis of different types of genomic synonymous information related to regulation of translation efficiency in all coding regions of a bacteriophage lambda, during different stages of its development. In particular, basing on the analysis of ribosome profiling data,<a reveal-id="dsx005-B9" data-open="dsx005-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> we demonstrated, for the first time, the condition specific adaptation of the bacteriophage codons in early/late genes to the intra-cellular <em>E. coli</em> environment during the different stages of phage development.</p>                            <h2 scrollto-destination=106354686 id="106354686" class="section-title" >2. Material and methods</h2>
                            <h3 scrollto-destination=106354687 id="106354687" class="section-title" >2.1. Data</h3>
<p>Ribosome profiling was applied to the process of lytic growth of Bacteriophage lambda by Liu et al.<a reveal-id="dsx005-B9" data-open="dsx005-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> They chose temperature induction of the classic cI857 repressor mutant of Bacteriophage lambda in a lysogen of <em>E. coli</em> MG1655 to synchronize the lytic process, sampling the lysogen and control non-lysogen both before and 2, 5, 10, and 20 min after shifting the temperature from 32 °C to 42 °C. Transcript sequences were obtained from EnsEMBL for <em>E. coli</em> (K-12 MG1655 release 121, accessed 28/07/15) and from NCBI for the lambda phage (J02459, accessed 07/12/15). There were 4,319 genes of <em>E. coli</em> and 66 genes of lambda phage in the obtained sequences.</p>                            <h4 scrollto-destination=106354689 id="106354689" class="section-title" >2.1.1. Ribo-seq reads mapping</h4>
<p>Ribosome footprint sequences were obtained from<a reveal-id="dsx005-B9" data-open="dsx005-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> (GSE47509, induction 0–20 min). We trimmed the poly-A adaptors from the reads using Cutadapt<a reveal-id="dsx005-B26" data-open="dsx005-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> (version 1.8.3), and utilized Bowtie<a reveal-id="dsx005-B27" data-open="dsx005-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> (version 1.1.1) to map them to the <em>E. coli</em>-lambda transcriptome. The location of the A-site was approximated by an 11-nt shift from the 5ʹ end of the aligned read. This shift maximized the correlation between MTDR (described below) and the observed read densities per <em>E. coli</em> gene. Further details related to the ribo-seq processing appear in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/supplemental_material_final_dsx005.pdf?Expires=2147483647&Signature=JfV351g5JZJlaQLNJvevlyaEXZLTzH-T39nHddiYwSmsEsgAFanrPc8EVN7JnK2D7oFgOQCwnRY9DWiP-056DkUChg74QKm8urRjrXYZrKtNzVdxIPxRg8t0pqmRljaU18jUVZDV~pYPzgSUZjesvalmlTY0dYZotLEUh3AAaBQ8Iz-2311-Vc5o9CNPIq7GuWPQ-wk4FwA3f-qDwOyYFvrsaeUGkwaumZDALzopR28H1ROQz75jGGJfxATVapbAA5w7rvo22w3GnseL82lIad-gT-ArjTI-zLtIQ0jBvuiQKGwg7FWf01zQSyuBSoXZDiM0B6juYPwtYfqH3sZn7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary methods</a></span>.</p>                            <h3 scrollto-destination=106354691 id="106354691" class="section-title" >2.2. Randomization models</h3>
<p>We considered two randomization models: (i) To preserve both <em>the amino acids order and content</em> and the frequencies distribution of 16 possible pairs of adjacent nucleotides (<em>dinucleotides</em>), a model based on a multivariate Boltzmann sampling scheme was used.<a reveal-id="dsx005-B28" data-open="dsx005-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> This model was initially introduced in the context of enumerative combinatorics and was used by us before for studying other viruses.<a reveal-id="dsx005-B16" data-open="dsx005-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsx005-B17" data-open="dsx005-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> We used the original source code which can be found in <a class="link link-uri" href="http://csb.cs.mcgill.ca/sparcs" target="_blank">http://csb.cs.mcgill.ca/sparcs</a> (7 February 2017, date last accessed). (ii) To preserve both <em>the amino acids order and content</em> and the codon usage bias, synonymous positions in codon sequences were randomly permuted.</p>                            <h4 scrollto-destination=106354693 id="106354693" class="section-title" >2.2.1. tRNA adaptation index (<em>tAI</em>).<a reveal-id="dsx005-B29" data-open="dsx005-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a></h4>
<p>Quantifies the adaptation of the codons of a coding region to the tRNA pool. Technical details regarding this measure appear in the supplementary.</p>                            <h4 scrollto-destination=106354695 id="106354695" class="section-title" >2.2.2. Ribosome profiling data normalization</h4>
<div class="block-child-p">We began this analysis by reconstructing ribosome profiles for <em>E. coli</em> and Bacteriophage Lambda expressed genes and performing normalization described in the <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/supplemental_material_final_dsx005.pdf?Expires=2147483647&Signature=JfV351g5JZJlaQLNJvevlyaEXZLTzH-T39nHddiYwSmsEsgAFanrPc8EVN7JnK2D7oFgOQCwnRY9DWiP-056DkUChg74QKm8urRjrXYZrKtNzVdxIPxRg8t0pqmRljaU18jUVZDV~pYPzgSUZjesvalmlTY0dYZotLEUh3AAaBQ8Iz-2311-Vc5o9CNPIq7GuWPQ-wk4FwA3f-qDwOyYFvrsaeUGkwaumZDALzopR28H1ROQz75jGGJfxATVapbAA5w7rvo22w3GnseL82lIad-gT-ArjTI-zLtIQ0jBvuiQKGwg7FWf01zQSyuBSoXZDiM0B6juYPwtYfqH3sZn7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">supplementary data</a></span>. The normalization enables measuring the relative time a ribosome spends translating each codon in a specific gene relative to other codons in it, whilst considering the total number of codons in the gene, resulting in its normalized footprint count (<em>NFC</em>)<a reveal-id="dsx005-B30" data-open="dsx005-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a>: <a id="E1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E1"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">N</mi><mi xmlns="">F</mi><msub xmlns=""><mrow><mi>C</mi></mrow><mrow><mi>j</mi></mrow></msub><mo xmlns="">=</mo><mfrac xmlns=""><mrow><mi>R</mi><msub><mrow><mi>C</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow><mrow><mrow><mrow><mo>(</mo><mfrac><mrow><mn>1</mn></mrow><mrow><mi>J</mi><mo>-</mo><mn>40</mn></mrow></mfrac><mo>)</mo></mrow></mrow><mfenced separators="|"><mrow><mi>R</mi><msub><mrow><mi>C</mi></mrow><mrow><mn>21</mn></mrow></msub><mo>+</mo><mi>R</mi><msub><mrow><mi>C</mi></mrow><mrow><mn>22</mn></mrow></msub><mo>+</mo><mo>…</mo><mo>+</mo><mi>R</mi><msub><mrow><mi>C</mi></mrow><mrow><mi>J</mi><mo>-</mo><mn>20</mn></mrow></msub></mrow></mfenced></mrow></mfrac><mo xmlns="">,</mo></math></div></div></div><a id="E2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E2"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">j</mi><mo xmlns="">=</mo><mn xmlns="">21</mn><mo xmlns="">…</mo><mi xmlns="">J</mi><mo xmlns="">-</mo><mn xmlns="">20</mn><mo xmlns="">,</mo></math></div></div></div> where <em>J</em> is the number of codons in the gene and <em>j</em> is the index of a codon.</div><p>We generate histogram of <em>NFC</em> for each codon. Each <em>NFC</em> distribution describes the probability (Pi) (<em>y</em>-axis) of observing each of the codon’s <em>NFC</em> values (<em>x</em>-axis) in the ORFs of the analysed organism.</p>                            <h3 scrollto-destination=106354698 id="106354698" class="section-title" >2.3. Codon typical decoding rate (TDR)</h3>
<div class="block-child-p">To estimate the typical decoding time of each codon based on <em>NFC</em> distributions, we used a novel statistical model,<a reveal-id="dsx005-B17" data-open="dsx005-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> which takes into consideration the skewed nature of the <em>NFC</em> distribution. The aim is to describe the <em>NFC</em> histogram of each codon as an output of a random variable which is a sum of two random variables: a normal and an exponential variable. Thus, the distribution of this new random variable includes three parameters, and is called <em>EMG</em> distribution.<a reveal-id="dsx005-B30" data-open="dsx005-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> In this model, the typical codon decoding time was described by the normal distribution with two parameters: mean (<span class="inline-formula">⁠<math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">μ</mi></math>⁠</span>) and standard deviation (<span class="inline-formula">⁠<math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">σ</mi></math>⁠</span>); the <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">μ</mi></math></span> parameter represents the location of the mean of the theoretical Gaussian component that should be obtained if there are no phenomena such as pauses/biases/ribosomal traffic jams; <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">σ</mi></math></span> represents the width of the Gaussian component. The exponential distribution has one parameter <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">λ</mi></math></span> which represents the skewness of the <em>NFC</em> distribution due to reasons such as ribosomal jamming caused by codons with different decoding times, extreme pauses, incomplete halting of the ribosomes and biases in the experiment. The <em>EMG</em> is defined as follows: <a id="E3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E3"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">f</mi><mfenced separators="|" xmlns=""><mrow><mi>x</mi><mo>;</mo><mi>μ</mi><mo>,</mo><mi>σ</mi><mo>,</mo><mi>λ</mi></mrow></mfenced><mo xmlns="">=</mo><mfrac xmlns=""><mrow><mi>λ</mi></mrow><mrow><mn>2</mn></mrow></mfrac><msup xmlns=""><mrow><mi>e</mi></mrow><mrow><mfrac><mrow><mi>λ</mi></mrow><mrow><mn>2</mn></mrow></mfrac><mfenced separators="|"><mrow><mn>2</mn><mi>μ</mi><mo>+</mo><mi>λ</mi><msup><mrow><mi>σ</mi></mrow><mrow><mn>2</mn></mrow></msup><mo>-</mo><mn>2</mn><mi>x</mi></mrow></mfenced></mrow></msup><mi xmlns="">e</mi><mi xmlns="">r</mi><mi xmlns="">f</mi><mi xmlns="">c</mi><mfenced separators="|" xmlns=""><mrow><mfrac><mrow><mi>μ</mi><mo>+</mo><mi>λ</mi><msup><mrow><mi>σ</mi></mrow><mrow><mn>2</mn></mrow></msup><mo>-</mo><mi>x</mi></mrow><mrow><msqrt><mn>2</mn></msqrt><mi>σ</mi></mrow></mfrac></mrow></mfenced><mo xmlns="">,</mo></math></div></div></div><a id="E4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E4"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">e</mi><mi xmlns="">r</mi><mi xmlns="">f</mi><mi xmlns="">c</mi><mfenced separators="|" xmlns=""><mrow><mi>x</mi></mrow></mfenced><mo xmlns="">=</mo><mn xmlns="">1</mn><mo xmlns="">-</mo><mrow xmlns=""><mrow><mi mathvariant="italic">erf</mi></mrow><mo>⁡</mo><mrow><mfenced separators="|"><mrow><mi>x</mi></mrow></mfenced></mrow></mrow><mo xmlns="">=</mo><mi mathvariant="normal" xmlns=""> </mi><mrow xmlns=""><msubsup><mo stretchy="false">∫</mo><mrow><mi>x</mi></mrow><mrow><mi mathvariant="normal">∞</mi></mrow></msubsup><mrow><msup><mrow><mi>e</mi></mrow><mrow><mo>-</mo><msup><mrow><mi>t</mi></mrow><mrow><mn>2</mn></mrow></msup></mrow></msup><mi>d</mi><mi>t</mi></mrow></mrow><mo xmlns="">.</mo></math></div></div></div></div><p>We used the maximum likelihood criterion to estimate these three parameters for each codon based on <em>E. coli</em> ribosome profiling data by fitting the suggested model to the <em>NFC</em> distribution. <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mn xmlns="">1</mn><mo xmlns="">/</mo><mi xmlns="">μ</mi></math></span> was defined to be the <em>TDR</em> of each codon.</p><p>In order to optimize the <em>TDR</em> to <em>E. coli</em>’s read counts in every time condition; we removed outliers from the <em>NFC</em> distribution of each codon in the following way: for every codon (in every time condition), and for each <em>NFCi</em> point related to the codon, we calculated the <em>Pi</em> to see value larger or equal to <em>NFCi</em> based on the pdf fitted to the codon (<em>EMG</em> distribution). Let <em>Ni</em> denote the number of measurements of the codon <em>NFC</em> based on the data; we removed points in which the result of <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>p</mi></mrow><mrow><mi>i</mi></mrow></msub><mi xmlns="">*</mi><msub xmlns=""><mrow><mi>N</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> was lower than 0.001.</p>                            <h3 scrollto-destination=106354702 id="106354702" class="section-title" >2.4. Synonymous codons usage analysis</h3>
<div class="block-child-p">Synonymous codons composition of a coding sequence was represented by a 61-dimensional vector of relative synonymous codons frequencies (<em>RSCF</em>) of each one of 61 coding codons (stop codons were excluded): <a id="E5" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E5"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">R</mi><mi xmlns="">S</mi><mi xmlns="">C</mi><mi xmlns="">F</mi><mo xmlns="">=</mo><mfenced separators="|" xmlns=""><mrow><mi>R</mi><mi>S</mi><mi>C</mi><mi>F</mi><mfenced open="[" close="]" separators="|"><mrow><mn>1</mn></mrow></mfenced><mo>,</mo><mo>…</mo><mo>,</mo><mi>R</mi><mi>S</mi><mi>C</mi><mi>F</mi><mfenced open="[" close="]" separators="|"><mrow><mn>61</mn></mrow></mfenced></mrow></mfenced><mo xmlns="">,</mo></math></div></div></div><a id="E6" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E6"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">R</mi><mi xmlns="">S</mi><mi xmlns="">C</mi><mi xmlns="">F</mi><mfenced open="[" close="]" separators="|" xmlns=""><mrow><mi>i</mi></mrow></mfenced><mo xmlns="">=</mo><mi mathvariant="normal" xmlns=""> </mi><mfrac xmlns=""><mrow><msub><mrow><mi>q</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mrow><mrow><munder><mo stretchy="false">∑</mo><mrow><mi>j</mi><mo>∈</mo><mi>s</mi><mi>y</mi><mi>n</mi><mfenced open="[" close="]" separators="|"><mrow><mi>i</mi></mrow></mfenced></mrow></munder><mrow><msub><mrow><mi>q</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow></mrow></mrow></mfrac><mo xmlns="">,</mo><mi mathvariant="normal" xmlns=""> </mi><mrow xmlns=""><munder><mo stretchy="false">∑</mo><mrow><mi>j</mi><mo>∈</mo><mi>s</mi><mi>y</mi><mi>n</mi><mfenced open="[" close="]" separators="|"><mrow><mi>i</mi></mrow></mfenced></mrow></munder><mrow><mi>R</mi><mi>S</mi><mi>C</mi><mi>F</mi><mfenced open="[" close="]" separators="|"><mrow><mi>j</mi></mrow></mfenced><mo>=</mo><mn>1</mn></mrow></mrow><mo xmlns="">,</mo></math></div></div></div> where <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>q</mi></mrow><mrow><mi>i</mi></mrow></msub></math></span> is the number of appearances of codon <em>i</em> in a sequence, <em>syn</em>[<em>i</em>] is a subset of indexes in <em>RSCF</em> pointing at codons synonymous to codon <em>i</em>. Therefore, each of the 61 coding codons was assigned a number between 0 and 1 according to its frequency relatively to the other codons coding for the same among acid.</div><p>Clustering analysis was performed on <em>RSCF</em> vectors of all coding sequences. In order to exclude biases due to a possible absence of specific amino acids in some of the sequences (missing amino acids), the relative synonymous frequency of a codon corresponding to a missing amino acid was set to the average relative synonymous frequency of this codon over all sequences in which at least one such amino acid is present.</p><div class="block-child-p">Each viral sequence was assigned a group label corresponding to its temporary expression stage (early/late) (according to the classification known in the literature). The tendency of sequences to cluster according to the codons usage in two different clusters corresponding to their temporary expression stages (early/late) was measured using the Davies–Bouldin score (<em>DBS</em>). This score is based on a ratio of within-cluster and between-cluster distances and is defined as: <a id="E7" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E7"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">D</mi><mi xmlns="">B</mi><mo xmlns="">=</mo><mi mathvariant="normal" xmlns=""> </mi><mfrac xmlns=""><mrow><mn>1</mn></mrow><mrow><mi>k</mi></mrow></mfrac><mrow xmlns=""><munderover><mo stretchy="false">∑</mo><mrow><mi>i</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi mathvariant="normal">k</mi></mrow></munderover><mrow><mrow><mrow><munder><mrow><mi mathvariant="normal">max</mi></mrow><mrow><mi>j</mi><mo>≠</mo><mi>i</mi></mrow></munder></mrow><mo>⁡</mo><mrow><mo>{</mo><mrow><msub><mrow><mi>D</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mrow><mo>}</mo><mo>,</mo></mrow></mrow></mrow><msub xmlns=""><mrow><mi>D</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mo xmlns="">=</mo><mi mathvariant="normal" xmlns=""> </mi><mfrac xmlns=""><mrow><mover accent="true"><mrow><msub><mrow><mi>d</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mo>-</mo></mover><mo>+</mo><mrow><mover accent="true"><mrow><msub><mrow><mi>d</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow><mo>-</mo></mover></mrow></mrow><mrow><msub><mrow><mi>d</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></mrow></mfrac><mo xmlns="">,</mo></math></div></div></div> where <em>k</em> is the number of evaluated clusters, <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>D</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></math></span> is the within-to-between cluster distance ratio for the <em>i</em>th and <em>j</em>th clusters; <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mover accent="true"><mrow><msub><mrow><mi>d</mi></mrow><mrow><mi>i</mi></mrow></msub></mrow><mo>-</mo></mover></mrow><mi xmlns=""> </mi></math></span>is the average Euclidian distance between each point in the <em>i</em>th cluster and the centroid of the <em>i</em>th cluster; <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mover accent="true"><mrow><msub><mrow><mi>d</mi></mrow><mrow><mi>j</mi></mrow></msub></mrow><mo>-</mo></mover></mrow></math></span> is the average Euclidian distance between each point in the <em>j</em>th cluster and the centroid of the <em>j</em>th cluster; <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>d</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub><mi mathvariant="normal" xmlns=""> </mi></math></span>is the Euclidean distance between the centroids of the <em>i</em>th and <em>j</em>th clusters. The maximum value of <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msub xmlns=""><mrow><mi>D</mi></mrow><mrow><mi>i</mi><mi>j</mi></mrow></msub></math></span> represents the worst-case within-to-between cluster ratio for cluster <em>i</em>. The optimal clustering solution has the smallest Davies–Bouldin score value.</div><p>The significance of cluster separation was assessed by comparing the <em>DBS</em> of the wildtype sequences to the randomized scores obtained from 100 permutations of gene group labels (early/late).</p><p>In order to visualize the clustering, a principal component analysis (<em>PCA</em>) was applied to project the <em>RSCF</em> vectors to a plane spanned by their first two principal components. In order to visualize the separation between clusters a maximum margin separation line—a line for which the distance between it and the nearest point from either of the groups is maximized, was calculated and plotted.</p><p>In the same manner, analysis of clustering between early and late viral groups and a set of top 50 host genes with the highest protein abundance can be performed.</p>                            <h4 scrollto-destination=106354709 id="106354709" class="section-title" >2.4.1. Mean typical decoding rate (<em>MTDR</em>)</h4>
<div class="block-child-p">A measure which estimates the translation elongation efficiency of the entire gene as a geometric average of TDRs of its codons: <a id="E15" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E15"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">M</mi><mi xmlns="">T</mi><mi xmlns="">D</mi><mi xmlns="">R</mi><mo xmlns="">=</mo><msup xmlns=""><mrow><mi>e</mi></mrow><mrow><mo>(</mo><mn>1</mn><mo>/</mo><mi>L</mi><mo>)</mo><mrow><munderover><mo stretchy="false">∑</mo><mrow><mi>l</mi><mo>=</mo><mn>1</mn></mrow><mrow><mi>L</mi></mrow></munderover><mrow><mo> </mo><mtext>log</mtext><mo> </mo><mo>⁡</mo><mo>(</mo><msub><mrow><mi>T</mi><mi>D</mi><mi>R</mi></mrow><mrow><mi>i</mi></mrow></msub><mfenced separators="|"><mrow><mi>l</mi></mrow></mfenced><mo>)</mo></mrow></mrow></mrow></msup><mo xmlns="">,</mo></math></div></div></div></div><p>where <em>i</em> is an index of a codon and <em>L</em> is the gene length in codon unit.</p><p>We computed the <em>MTDR</em> based on the ribosome profiling data of the <em>E. coli</em> (TDRs are effected by translation factors which are common to the host and the bacteriophage). Thus, it should not be sensitive to the low number of bacteriophage reads at the first time points.</p>                            <h4 scrollto-destination=106354713 id="106354713" class="section-title" >2.4.2. Relative translation elongation efficiency coefficient (RTEC)</h4>
<p>Quantifies the relative differences in mean <em>MTDR</em> values of early and late gene: <em>RTEC = </em><span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mfrac xmlns=""><mrow><mfenced separators="|"><mrow><mi>m</mi><mi>e</mi><mi>a</mi><mi>n</mi><mi> </mi><mi>M</mi><mi>T</mi><mi>D</mi><msub><mrow><mi>R</mi></mrow><mrow><mi>E</mi><mi> </mi></mrow></msub><mo>-</mo><mo></mo><mi>m</mi><mi>e</mi><mi>a</mi><mi>n</mi><mi> </mi><mi>M</mi><mi>T</mi><mi>D</mi><msub><mrow><mi>R</mi></mrow><mrow><mi>L</mi></mrow></msub></mrow></mfenced></mrow><mrow><mo>(</mo><mi>m</mi><mi>e</mi><mi>a</mi><mi>n</mi><mi> </mi><mi>M</mi><mi>T</mi><mi>D</mi><msub><mrow><mi>R</mi></mrow><mrow><mi>E</mi><mi> </mi></mrow></msub><mo>+</mo><mo></mo><mi>m</mi><mi>e</mi><mi>a</mi><mi>n</mi><mi> </mi><mi>M</mi><mi>T</mi><mi>D</mi><msub><mrow><mi>R</mi></mrow><mrow><mi>L</mi></mrow></msub><mo>)</mo></mrow></mfrac></math></span> where <em>E</em> and <em>L</em> sigh for early and late genes.</p>                            <h4 scrollto-destination=106354715 id="106354715" class="section-title" >2.4.3. Folding energy analysis</h4>
<p>Minimum free folding energy (<em>MFE</em>) is a thermodynamic energy involved in maintaining a secondary structure available to perform physical work whilst being released, and thus is characterized by non-positive values. mRNA secondary structure is believed to be in the most stable conformation when minimum amount of free energy is exerted (the <em>MFE</em> obtains the most negative value). The local <em>MFE</em>-profiles were constructed by applying a 39 nt length sliding window to a genomic sequence: in each step the <em>MFE</em> of a local subsequence enclosed by the corresponding window was calculated by Vienna (v. 2.1.9) package RNAfold<a reveal-id="dsx005-B31" data-open="dsx005-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> function with default parameters (see, also<a reveal-id="dsx005-B16" data-open="dsx005-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsx005-B32" data-open="dsx005-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a>). This function predicts the MFE and the associated secondary structure for the input RNA sequence using a dynamic programming based on the thermodynamic nearest-neighbor approach (the Zucker algorithm).<a reveal-id="dsx005-B33" data-open="dsx005-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a></p><p>First, all the genes in the bacteriophage genome were lined up according to their start codon and <em>MFE</em>-profiles were calculated for each coding region together with 40-nt up-stream the start codon sequence from the 5ʹ UTR. Then, all the genes in the genome were lined up according to their end codon and <em>MFE</em> profiles were calculated for each coding region together with 40-nt down-stream the end codon from the 3ʹ UTR.</p><p>For each gene 100 randomized <em>MFE</em>-profiles variants were computed basing on randomized sequences generated by the dinucleotide preserving and codon preserving randomization models (both preserving also the encoded proteins, see section above).<a reveal-id="dsx005-B16" data-open="dsx005-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsx005-B17" data-open="dsx005-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsx005-B28" data-open="dsx005-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> We did not change the UTRs in the randomization as in this study we are interested in the coding regions.</p><p>The mean <em>MFE</em>-profile was obtained by averaging the <em>MFE</em>-profiles of all genes (in a position wise manner). In a similar manner, 100 randomized mean <em>MFE</em>-profiles were computed by grouping the randomized <em>MFE</em>-profiles of all genes in 100 groups, each group contains a different variant for each gene, and then averaging the profiles in each group in a position-wise manner.</p><p>In order to assess the statistical significance of the folding strength at a particular position in a sequence, we compared the mean <em>MFE</em> values at this position with the mean <em>MFE</em> values in the corresponding position in each one of the randomized variants by calculating an empiric <em>P</em>-value—a proportion of the randomized values as extreme as in the wild type. Positions with <em>MFE</em> related <em>P</em>-value &lt;0.05 were defined as selected for strong/weak folding.</p><p>In addition, we computed mean <em>MFE</em> values for each gene over all windows (by averaging the values in the corresponding <em>MFE</em>-profiles) and compared them to the mean <em>MFE</em> values obtained from the corresponding 100 randomized profiles. For each gene we calculated its mean <em>MFE</em> value and an average of 100 mean <em>MFE</em> values from its randomized variants; the distributions of the wildtype and randomized mean <em>MFE</em> values of different genes were compared using Wilcoxon signed-rank test. Early and late genes were analysed separately.</p>                            <h4 scrollto-destination=106354722 id="106354722" class="section-title" >2.4.4. Average Repetitive Substring (<em>ARS</em>) index</h4>
<p>This measure is based on the assumption that evolution shapes the organismal coding sequences (and other part of the gene) to improve their interaction with the intra-cellular gene expression machinery. Since these interactions are mediated via binding of the gene expression machinery (e.g. translation/transcription factors, RNAP, ribosomes, RNA binding proteins, etc.) to the genetic material (DNA or RNA), the genetic material tend to have optimized binding sites (which are sub-sequences of nucleotides). We also expect that binding sites will appear in many coding regions and that more optimal binding sites will tend to appear more times in the genome. Thus, if longer substrings of a genome tend to appear in a certain organism’s coding sequence, it suggests that this coding sequence is more optimized to the intra-cellular gene expression machinery and thus it is probably more highly expressed. Here, we computed the <em>ARS</em> index for each bacteriophage gene in comparison to the host (<em>E. coli</em>) and in comparison to the rest of the viral genes.</p><p>The algorithm of <em>ARS index</em> is based on the following steps: (i) For each position <em>i</em> in the coding sequence <em>S</em> find the longest substring<span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns=""> </mi><msubsup xmlns=""><mrow><mi>S</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi>j</mi></mrow></msubsup></math></span> that starts in that position, and also appears in at least one of the coding sequences of the reference genome (<em>E. coli</em>/viral). (ii) Let |<em>S</em>| denote the length of a sequence <em>S</em>; the <em>ARS index</em> of <em>S</em> is the mean length of all the substrings <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><msubsup xmlns=""><mrow><mi>S</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi>j</mi></mrow></msubsup><mo xmlns="">:</mo><mi xmlns="">A</mi><mi xmlns="">R</mi><mi xmlns="">S</mi><mo xmlns="">=</mo><mo xmlns=""></mo><mfrac xmlns=""><mrow><mrow><mo stretchy="false">∑</mo><mrow><msubsup><mrow><mi>s</mi></mrow><mrow><mi>i</mi></mrow><mrow><mi>j</mi></mrow></msubsup></mrow></mrow></mrow><mrow><mo>|</mo><mi>s</mi><mo>|</mo></mrow></mfrac><mo xmlns="">.</mo></math></span></p>                            <h4 scrollto-destination=106354725 id="106354725" class="section-title" >2.4.5. Rare codons analysis</h4>
<p>Rare codons in a reference set of coding sequences were defined as codons with the relative synonymous frequency &lt;0.2. Three reference sets were used: <em>E. coli</em> coding sequences, bacteriophage early coding sequences, bacteriophage late coding sequences.</p><div class="block-child-p">A rare codons score (<em>RCS</em>) for a specific early/late/<em>E. coli</em> coding sequence with respect to a reference set of all early/all late/all <em>E. coli</em> coding sequences is defined as a percentage of amino acids in that sequence encoded by a rare codon out of all amino acids that are encoded by at least one rare codon in the corresponding reference set (if an AA is not encoded by codons that are rare in the reference set we exclude it from the analysis): <a id="E8" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="E8"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">R</mi><mi xmlns="">C</mi><mi xmlns="">S</mi><mo xmlns="">=</mo><mi mathvariant="normal" xmlns=""> </mi><mfrac xmlns=""><mrow><mn>1</mn></mrow><mrow><mi>N</mi></mrow></mfrac><mrow xmlns=""><munder><mo stretchy="false">∑</mo><mrow><mi>c</mi></mrow></munder><mrow><msub><mrow><mi>I</mi></mrow><mrow><mi>c</mi></mrow></msub></mrow></mrow><mo xmlns="">,</mo><mi mathvariant="normal" xmlns=""> </mi><msub xmlns=""><mrow><mi>I</mi></mrow><mrow><mi>c</mi></mrow></msub><mo xmlns="">=</mo><mi mathvariant="normal" xmlns=""> </mi><mfenced open="{" close="" separators="|" xmlns=""><mrow><mtable><mtr><mtd><mrow><maligngroup></maligngroup><mn>1</mn><mo>,</mo><mi> </mi><mi>R</mi><mi>S</mi><mi>C</mi><mi>F</mi><mfenced separators="|"><mrow><mi>c</mi></mrow></mfenced><mo>&lt;</mo><mn>0.2</mn></mrow></mtd></mtr><mtr><mtd><mrow><maligngroup></maligngroup><mn>0</mn><mo>,</mo><mi> </mi><mi>o</mi><mi>t</mi><mi>h</mi><mi>e</mi><mi>r</mi><mi>w</mi><mi>i</mi><mi>s</mi><mi>e</mi><mi> </mi></mrow></mtd></mtr></mtable></mrow></mfenced><mo xmlns="">,</mo></math></div></div></div> where the sum is over all codons <em>c</em> that have at least one rare synonymous codon that appears in the reference set and <em>N</em> is the total number of such codons.</div>                            <h4 scrollto-destination=106354728 id="106354728" class="section-title" >2.4.6. Late genes sampling</h4>
<p>In order to control the influence of the difference of genes length between early and late groups we sampled the late genes so that the average length of both genes groups is the same. The sampling was of random contiguous blocks of codons from the late genes and according to distributions of early genes.</p>                            <h2 scrollto-destination=106354730 id="106354730" class="section-title" >3. Results and discussion</h2>
<p>The research outline of the study is described in <a reveal-id="dsx005-F1" data-open="dsx005-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>. Our analysis was based on the genome (mainly the coding sequences) of the <em>E. coli</em> host (A), the genome of bacteriophage lambda (B) and the ribo-seq measurements of these two (C). To assess the statistical significance of the signals found in the analysed viral genes and to exclude the possibility that these signals are un-direct consequences of other genomic properties, we compared them to signals expected by chance in the corresponding randomized variants (D); two different randomization models were employed: one that maintains the encoded proteins and the frequencies of pairs of adjacent nucleotides (dinucleotides), and the other that maintains the encoded proteins and the frequencies of synonymous codons (codon usage bias). Basing on the ribo-seq data, the expression levels of each gene at each time point (E), the relative decoding rate of each codon (F), and the classification of the bacteriophage genes to early and late (with respect to the beginning of the lytic phase) (G) were derived. On the basis of A, B, D, G, we performed synonymous codons usage analysis of coding regions in viral and <em>E. coli</em> genes (H) using the <em>RSCF</em> (I) and tRNA adaptation indexes (<em>tAI</em>) (J). On the basis of E, F, G, we analysed codons decoding rates for early/late genes at different stages of the viral development (L) on gene/genomic (M) and per-codon (N) levels. In addition, based on A, B, D, G, we studied the local and global signals of evolutionary selection for strong/weak mRNA folding (K) and for higher order synonymous information encoded in repetitive sequence motifs that are longer/more complex than single codons in the coding regions of the bacteriophage genes (O). </p>                        <a id="106354732" scrollto-destination="106354732"></a>
<div id="dsx005-F1" data-id="dsx005-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/m_dsx005f1.png?Expires=2147483647&Signature=A8A0fF4jQ8H1WsQL6t98W7TrpnGZ9O4GN4o1b~s0iWCgqTT~WeKHxWiJtxuow1jsGO4GdDIUszWZEVNhfWC10ursb1mW2s5D6QW1Oes2AxJXV13LtE9o4T1pX0KhXxOPdmXxU4kUoDkadQJIhrowSWG8LU7Jbx1xv6lpTyF9zJnGtz9AKAbJuqVrRMb582sQ2Tjult4mxSQXdq0ijDWV9nV6egFOm2tdzWWasrYrX35Hg0oihrpGJA-uu2GSefqRPeFz9VBBIDGhIn-8lCte5zR-8Ses7OswqqZUBZZwIlPKt7jGsc~YGzcJAIHEFx5k-z-Tp-7ATD~9J4QttGJTtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) A flow diagram and illustration of the study (see details in the main text). (B) Relative expression level of each of the gene groups (early/late) in read count per nucleotide. " path-from-xml="dsx005f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106354732/dsx005f1.png" path-from-xml="dsx005f1.png" target="_blank">View large</a><a section="106354732" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/dsx005f1.png?Expires=2147483647&Signature=Kf778zH39wD4xd9pmnrwk4yYqqbSMAUwmKlbyhi9tKrSJHXpWxe1-9ubvXMycAW5MPJm5zXe~2ljatZdjYG0V4f5fxQb2fxMo8M2FdHpIqmlhgi~HZi5faGFLjO3AY9kFHIPgBftuU~SShZLpL0uA6Z8GTXPXHpaQUvhh-EvlyqTnXEzxcXsYmSkXnk-8~34IxdcfvASO78WYrglCHy~Pg2QzsY2IKM150xT8bK4t7YFD6l99b6vStR-3l96DzrsrBO8IgLCHW3NiWbU-8cgCCtINAe3WFWMOVdMBNnXIo5EgY2CinWYJFA7WO-kcs7e4KwQ7PLHPS0t1cBObpd4Ng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106354732&ar=3058514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx005f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) A flow diagram and illustration of the study (see details in the main text). (B) Relative expression level of each of the gene groups (early/late) in read count per nucleotide. </p></div></div><div content-id="dsx005-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/m_dsx005f1.png?Expires=2147483647&Signature=A8A0fF4jQ8H1WsQL6t98W7TrpnGZ9O4GN4o1b~s0iWCgqTT~WeKHxWiJtxuow1jsGO4GdDIUszWZEVNhfWC10ursb1mW2s5D6QW1Oes2AxJXV13LtE9o4T1pX0KhXxOPdmXxU4kUoDkadQJIhrowSWG8LU7Jbx1xv6lpTyF9zJnGtz9AKAbJuqVrRMb582sQ2Tjult4mxSQXdq0ijDWV9nV6egFOm2tdzWWasrYrX35Hg0oihrpGJA-uu2GSefqRPeFz9VBBIDGhIn-8lCte5zR-8Ses7OswqqZUBZZwIlPKt7jGsc~YGzcJAIHEFx5k-z-Tp-7ATD~9J4QttGJTtg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) A flow diagram and illustration of the study (see details in the main text). (B) Relative expression level of each of the gene groups (early/late) in read count per nucleotide. " path-from-xml="dsx005f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106354732/dsx005f1.png" path-from-xml="dsx005f1.png" target="_blank">View large</a><a section="106354732" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/dsx005f1.png?Expires=2147483647&Signature=Kf778zH39wD4xd9pmnrwk4yYqqbSMAUwmKlbyhi9tKrSJHXpWxe1-9ubvXMycAW5MPJm5zXe~2ljatZdjYG0V4f5fxQb2fxMo8M2FdHpIqmlhgi~HZi5faGFLjO3AY9kFHIPgBftuU~SShZLpL0uA6Z8GTXPXHpaQUvhh-EvlyqTnXEzxcXsYmSkXnk-8~34IxdcfvASO78WYrglCHy~Pg2QzsY2IKM150xT8bK4t7YFD6l99b6vStR-3l96DzrsrBO8IgLCHW3NiWbU-8cgCCtINAe3WFWMOVdMBNnXIo5EgY2CinWYJFA7WO-kcs7e4KwQ7PLHPS0t1cBObpd4Ng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106354732&ar=3058514&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx005f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) A flow diagram and illustration of the study (see details in the main text). (B) Relative expression level of each of the gene groups (early/late) in read count per nucleotide. </p></div></div><p>The major aim of this study is to compare the properties of coding regions of bacteriophage early and late genes; thus, we start with a brief description of the expression pattern of these two gene groups. The analysis of ribo-seq read count per nucleotide for early and late groups of bacteriophage genes appears in <a reveal-id="dsx005-F1" data-open="dsx005-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>. As can be seen, at the first time point the read count of both groups is very low. Afterwards the expression levels of both gene groups increase; whilst the expression of the early gene group is dominant during minutes 1–10, the expression levels of the late group become dominant towards the 20th minute. Specifically, 32 of the bacteriophage genes are defined as early genes as their expression levels increase from 5 to 10 min, and then decrease by minute 20 after the beginning of the lytic process<a reveal-id="dsx005-B9" data-open="dsx005-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a>; the rest of the genes (34 genes) are define as late genes as their expression levels become significant only at minute 10 from the beginning of the lytic process, and then increase considerably by minute 20 (<a reveal-id="dsx005-F1" data-open="dsx005-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a> and<a reveal-id="dsx005-B9" data-open="dsx005-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a>).</p>                            <h3 scrollto-destination=106354734 id="106354734" class="section-title" >3.1. Evidence of selection for different codons in early and late genes</h3>
<p>At the first step (<a reveal-id="dsx005-F2" data-open="dsx005-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>), we aimed at comparing the synonymous codons usage in <em>E. coli</em> and bacteriophage early and late coding sequences. To this end, for each coding sequence, we computed its relative synonymous codon frequencies (<em>RSCF</em>)—a 61-dimensional vector representing each codon (except the stop codons) by its frequency in that sequence normalized relative to the frequencies of other synonymous codons coding for the same amino acid (Methods). Our analysis demonstrated that the early and late genes tend to be clustered into two significantly separated (<em>P</em>-value &lt;0.01) clusters according to their synonymous codons usage. In addition synonymous codon usage in both groups of viral genes was found to be significantly different (<em>P</em>-value &lt;0.01) from that of <em>E. coli</em> (<a reveal-id="dsx005-F2" data-open="dsx005-F2" class="link link-reveal link-table xref-fig">Fig. 2A;</a> Methods). These results provide evidence that different sets of synonymous codons for early vs. late genes are selected in the course of viral evolution and may be related to the optimization of the viral fitness. </p>                        <a id="106354736" scrollto-destination="106354736"></a>
<div id="dsx005-F2" data-id="dsx005-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/m_dsx005f2.png?Expires=2147483647&amp;Signature=gG8Uiau~s3WKaSwJzTn8GrW~2olRNf4uoE-D9xYsGYDfZl85W16g16Mm05a02ZLJrKQpY5q8jnVK56FWTPf93uxztL2IIWZgs7pgoqaf7e-aFxyQgJso-Gc0DZH8Ji9MPnTp5QB7Qw8786fdmWfnck3v8Mu6eOOgTecFoQu-nKHy~k~rRjQAWz58VRgv~ukSjY-~NeM5dmxh6km1zGP6EAnoUFTJXMJzZYHboLDVSLj1BduQ-kCO75Fox56SfoGuEWraj1ktB57zAMm6jA~HlyBsNBCmEHh3R~rEQrcWQOGPCm5gcEx76I9fYLfFF12P-~mMGAsId8-CtbfXVHDjFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Selection for different codons in early and late genes. A plot of two principal components of RSCF vectors of all bacteriophage and E. coli genes. Bicteriophage early (blue) late (red) and E. coli (green) genes tend to be clustered into two distinct groups according to their synonymous codons compositions. The separation between the groups was measured by Davies–Bouldin index and found to be significantly higher than expected in random (P &lt; 0.01; see methods). The separation between the groups of early and late viral genes was visualized by a maximum margin separation line—a line for which the distance between it and the nearest point from either of the groups is maximized. (B) Selection for translation elongations efficiency in bacteriophage coding regions. At each time point: average MTDR values of wildtype early/late genes (vertical bars) were compared with MTDR values of 100 corresponding randomized variants (histograms). Average wildtype MTDR values of each group are significantly higher (P &lt; 0.05) than expected in random. The late genes were sampled to control the length factor (see methods; see also Supplementary Fig. S1). (C) Adaptation of translation elongation efficiency in early and late genes to different bacteriophage development stages genes. Relative translation elongation efficiency coefficient, RTEC = mean MTDRE -mean MTDRL/(mean MTDRE +mean MTDRL), as a function of time from the beginning of the lytic stage (0–20 min), where MTDRE and MTDRL sigh for the MTDR of early and late genes, respectively. We can see that the RTEC of early genes is higher at the beginning and become lower with time (as expected); the first point (t = 0), when there are no measurements of expression, is ignored (see also Fig. 1B). (D) Correlation between codons typical decoding rates (TDR) and relative synonymous codons frequencies (RSCF) at different time conditions for all, early and late viral genes and all E. coli genes. Time points with significant correlations (Spearman P-values &lt;0.05) are marked by asterix. For early genes, the correlation is higher than for late and E. coli genes and is significant (P-value &lt;0.05) for the firsttime points. No significant correlation can be seen for late genes except the first time point. The correlation in the case of the E. coli is significant up to 10 min (except at 2 min). (E) Selection for adaptation to E. coli tRNA pool in both early and late genes. Average tAI values of wildtype early (blue)/late (red) genes (vertical bars) were compared with tAI values of 1,000 corresponding randomized variants (histograms). Average wildtype tAI values of each group are significantly higher (P &lt; 0.001) than expected in random. The late genes were sampled to control the length factor (see Methods). (F) Correlation between TDR and tAI values for each codons at different type points is significant (P value &lt; 0.05)." path-from-xml="dsx005f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106354736/dsx005f2.png" path-from-xml="dsx005f2.png" target="_blank">View large</a><a section="106354736" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/dsx005f2.png?Expires=2147483647&amp;Signature=YktEGLCevHSjJrJ2psZcSLuz44OEJDIxSOVXNqiXIjPZDd0E8jO~nQkB9SSEMEhvL1QwanVXfG0IgMmrmUHS~WJKOTtugd5Z5N5bkTuXKT3UnC0GPYCBmZaJfdHu-tV0QbXCeyNfr-TfYKK-J~q7hyPT82rnHLSAqOH9EDox78elSQ3q33TEkVQ~CqSEv6tlW76sNE7u7Y-5rmBiM3P3FlckMsqEgtr51cf6Ji2Ef3lHJgEqLAn9RvCi~c1CqEaQWp2yp3nnQ24ErRr3zOMwbN8N2M05FVzPcnjYVYvlroRGrUIr3O8zhMkcbjXNg18Jcxgjm7kF-Jl8161N7yamEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106354736&amp;ar=3058514&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx005f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Selection for different codons in early and late genes. A plot of two principal components of RSCF vectors of all bacteriophage and <em>E. coli</em> genes. Bicteriophage early (blue) late (red) and <em>E. coli</em> (green) genes tend to be clustered into two distinct groups according to their synonymous codons compositions. The separation between the groups was measured by Davies–Bouldin index and found to be significantly higher than expected in random (<em>P</em> &lt; 0.01; see methods). The separation between the groups of early and late viral genes was visualized by a maximum margin separation line—a line for which the distance between it and the nearest point from either of the groups is maximized. (B) Selection for translation elongations efficiency in bacteriophage coding regions. At each time point: average MTDR values of wildtype early/late genes (vertical bars) were compared with MTDR values of 100 corresponding randomized variants (histograms). Average wildtype MTDR values of each group are significantly higher (<em>P</em> &lt; 0.05) than expected in random. The late genes were sampled to control the length factor (see methods; see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/supplemental_material_final_dsx005.pdf?Expires=2147483647&amp;Signature=JfV351g5JZJlaQLNJvevlyaEXZLTzH-T39nHddiYwSmsEsgAFanrPc8EVN7JnK2D7oFgOQCwnRY9DWiP-056DkUChg74QKm8urRjrXYZrKtNzVdxIPxRg8t0pqmRljaU18jUVZDV~pYPzgSUZjesvalmlTY0dYZotLEUh3AAaBQ8Iz-2311-Vc5o9CNPIq7GuWPQ-wk4FwA3f-qDwOyYFvrsaeUGkwaumZDALzopR28H1ROQz75jGGJfxATVapbAA5w7rvo22w3GnseL82lIad-gT-ArjTI-zLtIQ0jBvuiQKGwg7FWf01zQSyuBSoXZDiM0B6juYPwtYfqH3sZn7w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). (C) Adaptation of translation elongation efficiency in early and late genes to different bacteriophage development stages genes. Relative translation elongation efficiency coefficient, RTEC = <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mfenced separators="|" xmlns=""><mrow><mi>m</mi><mi>e</mi><mi>a</mi><mi>n</mi><mi mathvariant="normal"> </mi><mi>M</mi><mi>T</mi><mi>D</mi><msub><mrow><mi>R</mi></mrow><mrow><mi>E</mi><mi mathvariant="normal"> </mi></mrow></msub><mo>-</mo><mo></mo><mi>m</mi><mi>e</mi><mi>a</mi><mi>n</mi><mi mathvariant="normal"> </mi><mi>M</mi><mi>T</mi><mi>D</mi><msub><mrow><mi>R</mi></mrow><mrow><mi>L</mi></mrow></msub></mrow></mfenced><mo xmlns="">/</mo><mo xmlns="">(</mo><mi xmlns="">m</mi><mi xmlns="">e</mi><mi xmlns="">a</mi><mi xmlns="">n</mi><mi mathvariant="normal" xmlns=""> </mi><mi xmlns="">M</mi><mi xmlns="">T</mi><mi xmlns="">D</mi><msub xmlns=""><mrow><mi>R</mi></mrow><mrow><mi>E</mi><mi mathvariant="normal"> </mi></mrow></msub><mo xmlns="">+</mo><mo xmlns=""></mo><mi xmlns="">m</mi><mi xmlns="">e</mi><mi xmlns="">a</mi><mi xmlns="">n</mi><mi mathvariant="normal" xmlns=""> </mi><mi xmlns="">M</mi><mi xmlns="">T</mi><mi xmlns="">D</mi><msub xmlns=""><mrow><mi>R</mi></mrow><mrow><mi>L</mi></mrow></msub><mo xmlns="">)</mo></math>⁠</span>, as a function of time from the beginning of the lytic stage (0–20 min), where <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">M</mi><mi xmlns="">T</mi><mi xmlns="">D</mi><msub xmlns=""><mrow><mi>R</mi></mrow><mrow><mi>E</mi><mi mathvariant="normal"> </mi></mrow></msub></math></span>and<span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns=""> </mi><mi xmlns="">M</mi><mi xmlns="">T</mi><mi xmlns="">D</mi><msub xmlns=""><mrow><mi>R</mi></mrow><mrow><mi>L</mi></mrow></msub></math></span> sigh for the MTDR of early and late genes, respectively. We can see that the RTEC of early genes is higher at the beginning and become lower with time (as expected); the first point (<em>t</em> = 0), when there are no measurements of expression, is ignored (see also <a reveal-id="dsx005-F1" data-open="dsx005-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). (D) Correlation between codons typical decoding rates (TDR) and relative synonymous codons frequencies (RSCF) at different time conditions for all, early and late viral genes and all <em>E. coli</em> genes. Time points with significant correlations (Spearman <em>P</em>-values &lt;0.05) are marked by asterix. For early genes, the correlation is higher than for late and <em>E. coli</em> genes and is significant (<em>P</em>-value &lt;0.05) for the firsttime points. No significant correlation can be seen for late genes except the first time point. The correlation in the case of the <em>E. coli</em> is significant up to 10 min (except at 2 min). (E) Selection for adaptation to <em>E. coli</em> tRNA pool in both early and late genes. Average tAI values of wildtype early (blue)/late (red) genes (vertical bars) were compared with tAI values of 1,000 corresponding randomized variants (histograms). Average wildtype tAI values of each group are significantly higher (<em>P</em> &lt; 0.001) than expected in random. The late genes were sampled to control the length factor (see Methods). (F) Correlation between TDR and tAI values for each codons at different type points is significant (<em>P</em> value &lt; 0.05).</p></div></div><div content-id="dsx005-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/m_dsx005f2.png?Expires=2147483647&amp;Signature=gG8Uiau~s3WKaSwJzTn8GrW~2olRNf4uoE-D9xYsGYDfZl85W16g16Mm05a02ZLJrKQpY5q8jnVK56FWTPf93uxztL2IIWZgs7pgoqaf7e-aFxyQgJso-Gc0DZH8Ji9MPnTp5QB7Qw8786fdmWfnck3v8Mu6eOOgTecFoQu-nKHy~k~rRjQAWz58VRgv~ukSjY-~NeM5dmxh6km1zGP6EAnoUFTJXMJzZYHboLDVSLj1BduQ-kCO75Fox56SfoGuEWraj1ktB57zAMm6jA~HlyBsNBCmEHh3R~rEQrcWQOGPCm5gcEx76I9fYLfFF12P-~mMGAsId8-CtbfXVHDjFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Selection for different codons in early and late genes. A plot of two principal components of RSCF vectors of all bacteriophage and E. coli genes. Bicteriophage early (blue) late (red) and E. coli (green) genes tend to be clustered into two distinct groups according to their synonymous codons compositions. The separation between the groups was measured by Davies–Bouldin index and found to be significantly higher than expected in random (P &lt; 0.01; see methods). The separation between the groups of early and late viral genes was visualized by a maximum margin separation line—a line for which the distance between it and the nearest point from either of the groups is maximized. (B) Selection for translation elongations efficiency in bacteriophage coding regions. At each time point: average MTDR values of wildtype early/late genes (vertical bars) were compared with MTDR values of 100 corresponding randomized variants (histograms). Average wildtype MTDR values of each group are significantly higher (P &lt; 0.05) than expected in random. The late genes were sampled to control the length factor (see methods; see also Supplementary Fig. S1). (C) Adaptation of translation elongation efficiency in early and late genes to different bacteriophage development stages genes. Relative translation elongation efficiency coefficient, RTEC = mean MTDRE -mean MTDRL/(mean MTDRE +mean MTDRL), as a function of time from the beginning of the lytic stage (0–20 min), where MTDRE and MTDRL sigh for the MTDR of early and late genes, respectively. We can see that the RTEC of early genes is higher at the beginning and become lower with time (as expected); the first point (t = 0), when there are no measurements of expression, is ignored (see also Fig. 1B). (D) Correlation between codons typical decoding rates (TDR) and relative synonymous codons frequencies (RSCF) at different time conditions for all, early and late viral genes and all E. coli genes. Time points with significant correlations (Spearman P-values &lt;0.05) are marked by asterix. For early genes, the correlation is higher than for late and E. coli genes and is significant (P-value &lt;0.05) for the firsttime points. No significant correlation can be seen for late genes except the first time point. The correlation in the case of the E. coli is significant up to 10 min (except at 2 min). (E) Selection for adaptation to E. coli tRNA pool in both early and late genes. Average tAI values of wildtype early (blue)/late (red) genes (vertical bars) were compared with tAI values of 1,000 corresponding randomized variants (histograms). Average wildtype tAI values of each group are significantly higher (P &lt; 0.001) than expected in random. The late genes were sampled to control the length factor (see Methods). (F) Correlation between TDR and tAI values for each codons at different type points is significant (P value &lt; 0.05)." path-from-xml="dsx005f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106354736/dsx005f2.png" path-from-xml="dsx005f2.png" target="_blank">View large</a><a section="106354736" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/dsx005f2.png?Expires=2147483647&amp;Signature=YktEGLCevHSjJrJ2psZcSLuz44OEJDIxSOVXNqiXIjPZDd0E8jO~nQkB9SSEMEhvL1QwanVXfG0IgMmrmUHS~WJKOTtugd5Z5N5bkTuXKT3UnC0GPYCBmZaJfdHu-tV0QbXCeyNfr-TfYKK-J~q7hyPT82rnHLSAqOH9EDox78elSQ3q33TEkVQ~CqSEv6tlW76sNE7u7Y-5rmBiM3P3FlckMsqEgtr51cf6Ji2Ef3lHJgEqLAn9RvCi~c1CqEaQWp2yp3nnQ24ErRr3zOMwbN8N2M05FVzPcnjYVYvlroRGrUIr3O8zhMkcbjXNg18Jcxgjm7kF-Jl8161N7yamEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106354736&amp;ar=3058514&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx005f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Selection for different codons in early and late genes. A plot of two principal components of RSCF vectors of all bacteriophage and <em>E. coli</em> genes. Bicteriophage early (blue) late (red) and <em>E. coli</em> (green) genes tend to be clustered into two distinct groups according to their synonymous codons compositions. The separation between the groups was measured by Davies–Bouldin index and found to be significantly higher than expected in random (<em>P</em> &lt; 0.01; see methods). The separation between the groups of early and late viral genes was visualized by a maximum margin separation line—a line for which the distance between it and the nearest point from either of the groups is maximized. (B) Selection for translation elongations efficiency in bacteriophage coding regions. At each time point: average MTDR values of wildtype early/late genes (vertical bars) were compared with MTDR values of 100 corresponding randomized variants (histograms). Average wildtype MTDR values of each group are significantly higher (<em>P</em> &lt; 0.05) than expected in random. The late genes were sampled to control the length factor (see methods; see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/supplemental_material_final_dsx005.pdf?Expires=2147483647&amp;Signature=JfV351g5JZJlaQLNJvevlyaEXZLTzH-T39nHddiYwSmsEsgAFanrPc8EVN7JnK2D7oFgOQCwnRY9DWiP-056DkUChg74QKm8urRjrXYZrKtNzVdxIPxRg8t0pqmRljaU18jUVZDV~pYPzgSUZjesvalmlTY0dYZotLEUh3AAaBQ8Iz-2311-Vc5o9CNPIq7GuWPQ-wk4FwA3f-qDwOyYFvrsaeUGkwaumZDALzopR28H1ROQz75jGGJfxATVapbAA5w7rvo22w3GnseL82lIad-gT-ArjTI-zLtIQ0jBvuiQKGwg7FWf01zQSyuBSoXZDiM0B6juYPwtYfqH3sZn7w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). (C) Adaptation of translation elongation efficiency in early and late genes to different bacteriophage development stages genes. Relative translation elongation efficiency coefficient, RTEC = <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mfenced separators="|" xmlns=""><mrow><mi>m</mi><mi>e</mi><mi>a</mi><mi>n</mi><mi mathvariant="normal"> </mi><mi>M</mi><mi>T</mi><mi>D</mi><msub><mrow><mi>R</mi></mrow><mrow><mi>E</mi><mi mathvariant="normal"> </mi></mrow></msub><mo>-</mo><mo></mo><mi>m</mi><mi>e</mi><mi>a</mi><mi>n</mi><mi mathvariant="normal"> </mi><mi>M</mi><mi>T</mi><mi>D</mi><msub><mrow><mi>R</mi></mrow><mrow><mi>L</mi></mrow></msub></mrow></mfenced><mo xmlns="">/</mo><mo xmlns="">(</mo><mi xmlns="">m</mi><mi xmlns="">e</mi><mi xmlns="">a</mi><mi xmlns="">n</mi><mi mathvariant="normal" xmlns=""> </mi><mi xmlns="">M</mi><mi xmlns="">T</mi><mi xmlns="">D</mi><msub xmlns=""><mrow><mi>R</mi></mrow><mrow><mi>E</mi><mi mathvariant="normal"> </mi></mrow></msub><mo xmlns="">+</mo><mo xmlns=""></mo><mi xmlns="">m</mi><mi xmlns="">e</mi><mi xmlns="">a</mi><mi xmlns="">n</mi><mi mathvariant="normal" xmlns=""> </mi><mi xmlns="">M</mi><mi xmlns="">T</mi><mi xmlns="">D</mi><msub xmlns=""><mrow><mi>R</mi></mrow><mrow><mi>L</mi></mrow></msub><mo xmlns="">)</mo></math>⁠</span>, as a function of time from the beginning of the lytic stage (0–20 min), where <span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">M</mi><mi xmlns="">T</mi><mi xmlns="">D</mi><msub xmlns=""><mrow><mi>R</mi></mrow><mrow><mi>E</mi><mi mathvariant="normal"> </mi></mrow></msub></math></span>and<span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi mathvariant="normal" xmlns=""> </mi><mi xmlns="">M</mi><mi xmlns="">T</mi><mi xmlns="">D</mi><msub xmlns=""><mrow><mi>R</mi></mrow><mrow><mi>L</mi></mrow></msub></math></span> sigh for the MTDR of early and late genes, respectively. We can see that the RTEC of early genes is higher at the beginning and become lower with time (as expected); the first point (<em>t</em> = 0), when there are no measurements of expression, is ignored (see also <a reveal-id="dsx005-F1" data-open="dsx005-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>). (D) Correlation between codons typical decoding rates (TDR) and relative synonymous codons frequencies (RSCF) at different time conditions for all, early and late viral genes and all <em>E. coli</em> genes. Time points with significant correlations (Spearman <em>P</em>-values &lt;0.05) are marked by asterix. For early genes, the correlation is higher than for late and <em>E. coli</em> genes and is significant (<em>P</em>-value &lt;0.05) for the firsttime points. No significant correlation can be seen for late genes except the first time point. The correlation in the case of the <em>E. coli</em> is significant up to 10 min (except at 2 min). (E) Selection for adaptation to <em>E. coli</em> tRNA pool in both early and late genes. Average tAI values of wildtype early (blue)/late (red) genes (vertical bars) were compared with tAI values of 1,000 corresponding randomized variants (histograms). Average wildtype tAI values of each group are significantly higher (<em>P</em> &lt; 0.001) than expected in random. The late genes were sampled to control the length factor (see Methods). (F) Correlation between TDR and tAI values for each codons at different type points is significant (<em>P</em> value &lt; 0.05).</p></div></div>                            <h3 scrollto-destination=106354737 id="106354737" class="section-title" >3.2. Differential codon usage in early and late genes can be partially explained by adaptation of translation elongation efficiency to different bacteriophage development stages</h3>
<p>Having shown that early and late viral genes have a significantly different composition of synonymous codons which may be associated with various features of their expression, we would like to focus one such feature, and understand the translation elongation efficiency of bacteriophage coding regions and how it behaves in different stages of the viral lytic cycle.</p><p>To this aim we employed a <em>condition</em> specific measure of translation elongation<a reveal-id="dsx005-B30" data-open="dsx005-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> to study the elongation speed of viral codons/genes during the different steps of phage development. This measure, called <em>MTDR</em>, is based on the estimation of a typical codon decoding rate (<em>TDR</em>) of each codon at each time point<a reveal-id="dsx005-B30" data-open="dsx005-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> based on the ribosome profiling data<a reveal-id="dsx005-B9" data-open="dsx005-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> and enables ranking codons and coding regions according to their elongation rate whilst controlling for other factors, such as initiation rates and mRNA levels (see details in the Methods section).</p><p>At the first step, we wanted to check whether the bacteriophage coding regions undergo any selection for optimizing translation elongation. To this end, at every time condition we computed two average <em>MTDR</em> values, for early and late genes separately, and compared them to the average <em>MTDR</em> values obtained for corresponding randomized variants that maintain the wild type amino acid content and the dinucleotide distribution (Methods). As can be seen in <a reveal-id="dsx005-F2" data-open="dsx005-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/supplemental_material_final_dsx005.pdf?Expires=2147483647&Signature=JfV351g5JZJlaQLNJvevlyaEXZLTzH-T39nHddiYwSmsEsgAFanrPc8EVN7JnK2D7oFgOQCwnRY9DWiP-056DkUChg74QKm8urRjrXYZrKtNzVdxIPxRg8t0pqmRljaU18jUVZDV~pYPzgSUZjesvalmlTY0dYZotLEUh3AAaBQ8Iz-2311-Vc5o9CNPIq7GuWPQ-wk4FwA3f-qDwOyYFvrsaeUGkwaumZDALzopR28H1ROQz75jGGJfxATVapbAA5w7rvo22w3GnseL82lIad-gT-ArjTI-zLtIQ0jBvuiQKGwg7FWf01zQSyuBSoXZDiM0B6juYPwtYfqH3sZn7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>, the average <em>MTDR</em> of both groups is significantly higher than expected from the random model in all time points (early: <em>p</em> &lt;0.01; late: <em>p</em> &lt;= 0.04). These results suggest that, indeed, translation elongation efficiency is maintained along the lytic cycle of infection and may be a factor that drives codon evolution in both groups of the bacteriophage genes.</p><p>At the next step, to compare the translation elongation efficiency between early and late genes, we looked at the <em>RTEC</em> which quantifies the relative differences in average <em>MTDR</em> values of two gene groups: more positive <em>RTEC</em> values mean that early genes are more efficient than the late genes and vice versa, more negative <em>RTEC</em> values mean that late genes are more efficient than the early ones; <em>RTEC</em> values close to zero mean that the two groups of genes have a similar translational efficiency (Methods). <a reveal-id="dsx005-F2" data-open="dsx005-F2" class="link link-reveal link-table xref-fig">Figure 2C</a> describes the <em>RTEC</em> as a function of time (0–20 min). As can be seen, the relative efficiency of elongation of early genes (in comparison to the late genes) is high at the beginning and become lower with time (<em>P</em>-value = 0.04; based on spearman correlation). These results demonstrate that translation elongation efficiency of the early genes is relatively higher at the early stages of the bacteriophage development (when they are expressed) and the translation elongation efficiency of the late genes is relatively higher at the late stages of the bacteriophage development (when they are expressed).</p><p><a reveal-id="dsx005-F2" data-open="dsx005-F2" class="link link-reveal link-table xref-fig">Figure 2D</a> describes the per codon correlation between the <em>TDR</em> and <em>RSCF</em> for the two bacteriophage gene groups (early and late) and the <em>E. coli</em> genes at different time points. As can be seen, the correlation is higher and significant for the first time points in the case of the <em>E. coli</em> and early bacteriophage genes. For the late genes, the correlation is significant only at the initial points. The fact that the correlation between <em>RSCF</em> and <em>TDR</em> in the case of the <em>early viral genes is</em> significant at the initial points supports the conjecture that the relative codon decoding times change during the viral development; this is probably related, among others, to the fact that the bacteriophage affects extensively the gene expression in the cell.</p><p>The lower correlations at time 2 does not seems to be relate to trivial biases/problems with the experiment as the number of reads in the <em>E. coli</em> (used for inferring the <em>MTDR</em>) is similar to the number of read in the different time points; in addition, the number of reads mapped to the viral genes is higher than in time 0. Thus, it is possible that the lower correlation is related to a biological phenomenon: e.g. it is possible that in this time point there is (strong) deviation (which is possibly short in time) of the concentration levels of the translation factors in the cell related to the other points; whilst the codon distributions were shaped to fit the other (longer) periods of the bacteriophage development.</p>                            <h3 scrollto-destination=106354744 id="106354744" class="section-title" >3.3. Selection for translation efficiency in bacteriophage genes may be partially explained by adaptation to the<em> E. coli</em> tRNA pool, and the fact that it changes during the bacteriophage development</h3>
<p>Previous studies demonstrated that that the codon decoding times may be directly influenced by the tRNA levels in the cell (see, for example<a reveal-id="dsx005-B30" data-open="dsx005-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a>).</p><p>In <a reveal-id="dsx005-F2" data-open="dsx005-F2" class="link link-reveal link-table xref-fig">Fig. 2E</a>, we analysed the adaptation of the viral codons to the genomic tRNA copy number in the host at natural conditions (Methods) and found it to be significant in comparison to the randomized variants that maintain the dinucleotides distribution for both early and late gene groups. However, as can be seen in <a reveal-id="dsx005-F2" data-open="dsx005-F2" class="link link-reveal link-table xref-fig">Fig. 2F</a>, the correlation between the <em>TDR</em> and the <em>tAI</em> of different codons is significant but <em>decreases</em> during the viral development stages. These results may suggest that among others the tRNA levels change during the viral development stages, affecting the codon decoding times.</p><p>In addition, we analysed the tendency of early/late bacterial coding sequences to use rare synonymous codons with respect to early/late/bacterial gene groups, respectively (RCS). <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/supplemental_material_final_dsx005.pdf?Expires=2147483647&Signature=JfV351g5JZJlaQLNJvevlyaEXZLTzH-T39nHddiYwSmsEsgAFanrPc8EVN7JnK2D7oFgOQCwnRY9DWiP-056DkUChg74QKm8urRjrXYZrKtNzVdxIPxRg8t0pqmRljaU18jUVZDV~pYPzgSUZjesvalmlTY0dYZotLEUh3AAaBQ8Iz-2311-Vc5o9CNPIq7GuWPQ-wk4FwA3f-qDwOyYFvrsaeUGkwaumZDALzopR28H1ROQz75jGGJfxATVapbAA5w7rvo22w3GnseL82lIad-gT-ArjTI-zLtIQ0jBvuiQKGwg7FWf01zQSyuBSoXZDiM0B6juYPwtYfqH3sZn7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span> describes the per condition partial correlation (controlling for gene length) between RCS and mean read counts for the two viral gene groups (early and late) and the <em>E. coli</em> genes. As can be seen, the correlation decreases in the case of the early genes and <em>E. coli</em> genes and increases for late genes. Our analysis demonstrates that early/late genes with rare early/late genes codons tend to be lowly expressed at the early/late stages, respectively.</p><p>The results reported in this section support the conjecture that some of the differences between the early and the late genes are related to the adaptation of viral codons to the intracellular environments in different stages of the phage development. Specifically, we suggest that such adaptation may be result of the fact that the typical decoding times (possibly due to changes in tRNA levels) change during the bacteriophage development.</p>                            <h3 scrollto-destination=106354749 id="106354749" class="section-title" >3.4. Additional constraints that shape the codon content of the bacteriophage</h3>
<p>In the previous section, we emphasized the importance of the translation elongation efficiency on shaping the codon composition of the Bacteriophage coding regions. In this section, we demonstrate that additional gene expression aspects are also encoded (in parallel) in the coding sequences.</p><p>First, we examined whether there is a selection for strong/weak local folding in different regions along the genomes (coding regions and flanking UTRs) (<a reveal-id="dsx005-F3" data-open="dsx005-F3" class="link link-reveal link-table xref-fig">Fig. 3A–C</a>). To this end, all early/late genes were aligned to the start codon and average values of a minimal folding energy (<em>MFE</em>) over all genes in each group were predicted within all possible 39-nt length local window. These average <em>MFE</em> values where compared in a position wise manner (<a reveal-id="dsx005-F3" data-open="dsx005-F3" class="link link-reveal link-table xref-fig">Fig. 3A and B</a>) and in average over all positions (<a reveal-id="dsx005-F3" data-open="dsx005-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/supplemental_material_final_dsx005.pdf?Expires=2147483647&Signature=JfV351g5JZJlaQLNJvevlyaEXZLTzH-T39nHddiYwSmsEsgAFanrPc8EVN7JnK2D7oFgOQCwnRY9DWiP-056DkUChg74QKm8urRjrXYZrKtNzVdxIPxRg8t0pqmRljaU18jUVZDV~pYPzgSUZjesvalmlTY0dYZotLEUh3AAaBQ8Iz-2311-Vc5o9CNPIq7GuWPQ-wk4FwA3f-qDwOyYFvrsaeUGkwaumZDALzopR28H1ROQz75jGGJfxATVapbAA5w7rvo22w3GnseL82lIad-gT-ArjTI-zLtIQ0jBvuiQKGwg7FWf01zQSyuBSoXZDiM0B6juYPwtYfqH3sZn7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>) to the <em>MFE</em> values expected in random. Positions with significantly strong (more negative <em>MFE</em>) or weak (more positive <em>MFE</em>) folding where identified (<a reveal-id="dsx005-F3" data-open="dsx005-F3" class="link link-reveal link-table xref-fig">Fig. 3A and B</a>) and also global <em>P</em>-value related to the mean average <em>MFE</em> values was computed (<a reveal-id="dsx005-F3" data-open="dsx005-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>). To make sure that the obtained folding signals were not mainly an indirect consequence of codon usage bias and/or selection for specific dinucleotide contents not related to mRNA folding we employed two randomization models, one designed to maintain both the encoded protein and the distribution of dinucleotides and the other designed to maintain both the encoded protein and the codon usage bias (see the Methods section for more details). </p>                        <a id="106354752" scrollto-destination="106354752"></a>
<div id="dsx005-F3" data-id="dsx005-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/m_dsx005f3.png?Expires=2147483647&amp;Signature=4wlNOCMsTfVYVY0xBsxXZrnRkmqPhs3w2GlydcQFgs051n8hFLZAu5o5nfftB7ZlxD4pj3LSLwc-g6xZgvsAPkQs2EZhyU~Ga9xGYC8f5IspbNSNKL3dMxcldpeLI0VaaUa9HXQU-ZC-U1UnAOKAdBGd-KjNGDyGa6Ty4CqQV~oIdkl4GMrpzQuwOGaZjclC~r9pegXPw0ZfMgn4zEaANO6lBEtsCRiFzihVespoKXzQjAZhsHaR7VRsbyqCO-pQfbrG9QWFKm3T94yMr7vsnpTnrQRQ7X9GIyKp-c8AWLPOvnNger1x5f1OHk-RuuuYF1HGhK5-K5W0Rz2~TmT3-w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A and B) Profiles of folding energy (average MFE in all windows of 39-nt length) across the bacteriophage genome (blue) vs. an averaged profile corresponding to 100 randomized variants (black) based on dinucleotide preserving randomization; the window index denotes the distance (in nucleotides) from the beginning of the ORF to the beginning of the window. Regions where the folding energy of the wild type genome is significantly higher (red) or lower (green) than in randomized variants are marked at the bottom of the figure. (A) The profiles include the 5′-UTR near the beginning of the ORF (negative window indexes). (B) The profiles include the 3ʹ-UTR near the ending of the ORF (positive window indexes). (C) Histograms of mean local folding energies (folding energies averaged over all the windows of each gene) compared with randomized mean local folding energies obtained from two models: (i) protein + dinucleotides preserving and (ii) protein + codon usage bias preserving (see also Supplementary Fig. S3). (D) Histograms of log[ARS index]. Eight analyses were performed: two types of reference genomes; bacterial and viral, two type of randomizations; dinucleotide and codons, two groups of genes; early and late. In each histogram, the wild type distribution is compared with the mean random distribution (1,000 random genomes). The P-values were calculated according to Wilcoxon signed-rank test." path-from-xml="dsx005f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106354752/dsx005f3.png" path-from-xml="dsx005f3.png" target="_blank">View large</a><a section="106354752" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/dsx005f3.png?Expires=2147483647&amp;Signature=zWtF3yMPYAL82lcm38zHubjb4bx5sF4VbNtxETCEK79R05~gJu69rmPiC5ZPU8a5xRg6SuPHiCwRM~8MCHhHRqqadRIJQRW8GCgP4XnvjCP0DR4XMdjuPdeMSS~wSXXkwzD-iWtFZXzEWAmO7RyKBx8i4QcdKIHeStmyTUdVhoDS7EDMGMdR96WP7CVgjd2m4tN024bY-sQ9upANRlM9nbGmjRyY9YbnsJDneGI2NnHHDbyvw-We0mRyvNUWMhnDwtjCtILeuxq1zUU94QtHeeNUtCejnadTVBXmJ4MaW01GdTADQuo6XVK6PTeIUDkoKwNe018NtKp99BjC10eXkQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106354752&amp;ar=3058514&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx005f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A and B) Profiles of folding energy (average MFE in all windows of 39-nt length) across the bacteriophage genome (blue) vs. an averaged profile corresponding to 100 randomized variants (black) based on dinucleotide preserving randomization; the window index denotes the distance (in nucleotides) from the beginning of the ORF to the beginning of the window. Regions where the folding energy of the wild type genome is significantly higher (red) or lower (green) than in randomized variants are marked at the bottom of the figure. (A) The profiles include the 5′-UTR near the beginning of the ORF (negative window indexes). (B) The profiles include the 3ʹ-UTR near the ending of the ORF (positive window indexes). (C) Histograms of mean local folding energies (folding energies averaged over all the windows of each gene) compared with randomized mean local folding energies obtained from two models: (i) protein + dinucleotides preserving and (ii) protein + codon usage bias preserving (see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/supplemental_material_final_dsx005.pdf?Expires=2147483647&amp;Signature=JfV351g5JZJlaQLNJvevlyaEXZLTzH-T39nHddiYwSmsEsgAFanrPc8EVN7JnK2D7oFgOQCwnRY9DWiP-056DkUChg74QKm8urRjrXYZrKtNzVdxIPxRg8t0pqmRljaU18jUVZDV~pYPzgSUZjesvalmlTY0dYZotLEUh3AAaBQ8Iz-2311-Vc5o9CNPIq7GuWPQ-wk4FwA3f-qDwOyYFvrsaeUGkwaumZDALzopR28H1ROQz75jGGJfxATVapbAA5w7rvo22w3GnseL82lIad-gT-ArjTI-zLtIQ0jBvuiQKGwg7FWf01zQSyuBSoXZDiM0B6juYPwtYfqH3sZn7w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). (D) Histograms of log[ARS index]. Eight analyses were performed: two types of reference genomes; bacterial and viral, two type of randomizations; dinucleotide and codons, two groups of genes; early and late. In each histogram, the wild type distribution is compared with the mean random distribution (1,000 random genomes). The <em>P</em>-values were calculated according to Wilcoxon signed-rank test.</p></div></div><div content-id="dsx005-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/m_dsx005f3.png?Expires=2147483647&amp;Signature=4wlNOCMsTfVYVY0xBsxXZrnRkmqPhs3w2GlydcQFgs051n8hFLZAu5o5nfftB7ZlxD4pj3LSLwc-g6xZgvsAPkQs2EZhyU~Ga9xGYC8f5IspbNSNKL3dMxcldpeLI0VaaUa9HXQU-ZC-U1UnAOKAdBGd-KjNGDyGa6Ty4CqQV~oIdkl4GMrpzQuwOGaZjclC~r9pegXPw0ZfMgn4zEaANO6lBEtsCRiFzihVespoKXzQjAZhsHaR7VRsbyqCO-pQfbrG9QWFKm3T94yMr7vsnpTnrQRQ7X9GIyKp-c8AWLPOvnNger1x5f1OHk-RuuuYF1HGhK5-K5W0Rz2~TmT3-w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A and B) Profiles of folding energy (average MFE in all windows of 39-nt length) across the bacteriophage genome (blue) vs. an averaged profile corresponding to 100 randomized variants (black) based on dinucleotide preserving randomization; the window index denotes the distance (in nucleotides) from the beginning of the ORF to the beginning of the window. Regions where the folding energy of the wild type genome is significantly higher (red) or lower (green) than in randomized variants are marked at the bottom of the figure. (A) The profiles include the 5′-UTR near the beginning of the ORF (negative window indexes). (B) The profiles include the 3ʹ-UTR near the ending of the ORF (positive window indexes). (C) Histograms of mean local folding energies (folding energies averaged over all the windows of each gene) compared with randomized mean local folding energies obtained from two models: (i) protein + dinucleotides preserving and (ii) protein + codon usage bias preserving (see also Supplementary Fig. S3). (D) Histograms of log[ARS index]. Eight analyses were performed: two types of reference genomes; bacterial and viral, two type of randomizations; dinucleotide and codons, two groups of genes; early and late. In each histogram, the wild type distribution is compared with the mean random distribution (1,000 random genomes). The P-values were calculated according to Wilcoxon signed-rank test." path-from-xml="dsx005f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106354752/dsx005f3.png" path-from-xml="dsx005f3.png" target="_blank">View large</a><a section="106354752" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/dsx005f3.png?Expires=2147483647&amp;Signature=zWtF3yMPYAL82lcm38zHubjb4bx5sF4VbNtxETCEK79R05~gJu69rmPiC5ZPU8a5xRg6SuPHiCwRM~8MCHhHRqqadRIJQRW8GCgP4XnvjCP0DR4XMdjuPdeMSS~wSXXkwzD-iWtFZXzEWAmO7RyKBx8i4QcdKIHeStmyTUdVhoDS7EDMGMdR96WP7CVgjd2m4tN024bY-sQ9upANRlM9nbGmjRyY9YbnsJDneGI2NnHHDbyvw-We0mRyvNUWMhnDwtjCtILeuxq1zUU94QtHeeNUtCejnadTVBXmJ4MaW01GdTADQuo6XVK6PTeIUDkoKwNe018NtKp99BjC10eXkQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106354752&amp;ar=3058514&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx005f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A and B) Profiles of folding energy (average MFE in all windows of 39-nt length) across the bacteriophage genome (blue) vs. an averaged profile corresponding to 100 randomized variants (black) based on dinucleotide preserving randomization; the window index denotes the distance (in nucleotides) from the beginning of the ORF to the beginning of the window. Regions where the folding energy of the wild type genome is significantly higher (red) or lower (green) than in randomized variants are marked at the bottom of the figure. (A) The profiles include the 5′-UTR near the beginning of the ORF (negative window indexes). (B) The profiles include the 3ʹ-UTR near the ending of the ORF (positive window indexes). (C) Histograms of mean local folding energies (folding energies averaged over all the windows of each gene) compared with randomized mean local folding energies obtained from two models: (i) protein + dinucleotides preserving and (ii) protein + codon usage bias preserving (see also <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/supplemental_material_final_dsx005.pdf?Expires=2147483647&amp;Signature=JfV351g5JZJlaQLNJvevlyaEXZLTzH-T39nHddiYwSmsEsgAFanrPc8EVN7JnK2D7oFgOQCwnRY9DWiP-056DkUChg74QKm8urRjrXYZrKtNzVdxIPxRg8t0pqmRljaU18jUVZDV~pYPzgSUZjesvalmlTY0dYZotLEUh3AAaBQ8Iz-2311-Vc5o9CNPIq7GuWPQ-wk4FwA3f-qDwOyYFvrsaeUGkwaumZDALzopR28H1ROQz75jGGJfxATVapbAA5w7rvo22w3GnseL82lIad-gT-ArjTI-zLtIQ0jBvuiQKGwg7FWf01zQSyuBSoXZDiM0B6juYPwtYfqH3sZn7w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). (D) Histograms of log[ARS index]. Eight analyses were performed: two types of reference genomes; bacterial and viral, two type of randomizations; dinucleotide and codons, two groups of genes; early and late. In each histogram, the wild type distribution is compared with the mean random distribution (1,000 random genomes). The <em>P</em>-values were calculated according to Wilcoxon signed-rank test.</p></div></div><p>Our analysis suggests that in general, there is an evidence of evolutionary selection for strong folding in the coding region of the bacteriophage in the case of the late genes but not in the case of the early genes and this is not related to very specific region along the coding region (<a reveal-id="dsx005-F3" data-open="dsx005-F3" class="link link-reveal link-table xref-fig">Fig. 3A–C</a>).This selection is manifested by lower average <em>MFE</em> values than expected in random (<a reveal-id="dsx005-F3" data-open="dsx005-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/supplemental_material_final_dsx005.pdf?Expires=2147483647&Signature=JfV351g5JZJlaQLNJvevlyaEXZLTzH-T39nHddiYwSmsEsgAFanrPc8EVN7JnK2D7oFgOQCwnRY9DWiP-056DkUChg74QKm8urRjrXYZrKtNzVdxIPxRg8t0pqmRljaU18jUVZDV~pYPzgSUZjesvalmlTY0dYZotLEUh3AAaBQ8Iz-2311-Vc5o9CNPIq7GuWPQ-wk4FwA3f-qDwOyYFvrsaeUGkwaumZDALzopR28H1ROQz75jGGJfxATVapbAA5w7rvo22w3GnseL82lIad-gT-ArjTI-zLtIQ0jBvuiQKGwg7FWf01zQSyuBSoXZDiM0B6juYPwtYfqH3sZn7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>) and a higher number of positions selected for strong folding (27 in the early and 123 in the late genes in <a reveal-id="dsx005-F3" data-open="dsx005-F3" class="link link-reveal link-table xref-fig">Fig. 3A;</a> 37 in the early and 144 in the late genes in <a reveal-id="dsx005-F3" data-open="dsx005-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>).</p><p>Local genomic regions (both in UTRs and in coding sequences) selected for strong mRNA folding may be related to various functional mRNA structures in the coding regions that effect the viral life cycle<a reveal-id="dsx005-B16" data-open="dsx005-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsx005-B17" data-open="dsx005-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsx005-B34" data-open="dsx005-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a>; specifically, it is possible that the mRNA structures are more important in the late genes due to less canonical regulatory aspects needed at the later phases of viral life. Interestingly, we found that in the case of the late genes there is an evidence of selection for weak folding at the 5ʹ end of the coding regions, suggesting that there is a significant evolutionary pressure for improved initiation efficiency.<a reveal-id="dsx005-B34" data-open="dsx005-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a></p><p>Secondly, we performed an analysis with a measure that detects long sub-sequences in the viral genome (<em>ARS index</em>) and were able to detect gene expression codes in an unsupervised manner<a reveal-id="dsx005-B35" data-open="dsx005-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> (see technical details in the Methods section). Our analysis (<a reveal-id="dsx005-F3" data-open="dsx005-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>) demonstrates that both in the early and late genes there are significant gene expression codes in the bacteriophage coding regions that cannot be explained by the viral codon frequency or dinucleotide composition. These results suggest that in parallel to single codon adaptations, the bacteriophage coding regions undergoes an evolutionary pressure to include more complex/longer regulatory signals in the coding region. This suggests that additional future studies should be performed on this topic.</p>                            <h2 scrollto-destination=106354756 id="106354756" class="section-title" >4. Conclusions</h2>
<p>The analyses performed in this study emphasize the way evolution shape the coding region of Bacteriophage Lambda to improve its expression levels and fitness. We demonstrate that evolution shapes the codon content of the Bacteriophage genes to fit them to the dynamic intracellular environment during the Bacteriophage development. Specifically, we show that the codon frequencies of the early and late genes were shaped in a different manner: (i) there is high intra group similarity in codon usage frequencies than inter-group similarities; (ii) the codons frequencies of both groups are significantly different than the codon frequencies of the host; (iii) whilst the decoding rates of both group is higher than expected based on the amino acid content and dinucleotide composition, the mean decoding rate of the early genes is relatively higher/lower than the late genes during the initial/late stages of the viral development, respectively.</p><p>The results reported here demonstrate and suggest that it may be possible to better understand the function of viral genes via the analysis of their codon distribution. They can also promote developing novel approaches for vaccine development and viral therapies: For example, based on the codon frequencies of different viral ORF we may be able to predict if they are early of late genes. In addition, engineering the viral genome such that its decoding rate (e.g. as measured by ribosome profiling) is attenuated may enable generating live attenuated based vaccines; on the other hand, improving the viral decoding rate may contribute to developing efficient oncolytic viruses and contribute towards developing efficient procedures for generating inactivated vaccines.</p><p>Our analysis support the conjecture that the that tRNA pool in the host changes during the bactriophage development with a tRNA pool similar to the host pool in natural conditions at the early points but more different than the host natural tRNA pool during the late points (see <a reveal-id="dsx005-F2" data-open="dsx005-F2" class="link link-reveal link-table xref-fig">Fig. 2F</a>). Thus, the accurate measure for estimating the optimization of the codon decoding rates of the bactriophage genes is the <em>TDR</em> and measures based on the genomic tRNA copy number (such as the <em>tAI</em>).</p><p>Nevertheless, it is important to mention that previous studies suggest that relative levels of tRNAs tend to have high correlation among different conditions and tissues (see for example<a reveal-id="dsx005-B36" data-open="dsx005-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a><sup>,</sup><a reveal-id="dsx005-B37" data-open="dsx005-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a>). This may be related, among others, to the fact that a significant change of tRNA levels ranking may have strong effect on co-translation protein folding (see for example<a reveal-id="dsx005-B38" data-open="dsx005-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a>) and thus a strong effect on the functionality of many proteins, effecting both the host and the bacteriophage. Similar phenomena was observed also in this study—there is significant correlation between codon decoding rates at different time points and the relevant genomic tRNA copy number (see <a reveal-id="dsx005-F2" data-open="dsx005-F2" class="link link-reveal link-table xref-fig">Fig. 2F</a>); however, as mention, this correlation is lower at the later time points.</p><p>At each time point the ‘optimal’ solution relate to the optimization of translation elongation speed and the optimal allocation of translation factors include using the codons with the highest codon decoding rates (i.e. the ones related to factors such as tRNA molecules that have relatively higher abundance in the cell). Thus, genes undergo selection to have codons similar to the translation factor concentration and this selection is expected to be higher in highly expressed genes (see<a reveal-id="dsx005-B39" data-open="dsx005-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a><sup>,</sup><a reveal-id="dsx005-B40" data-open="dsx005-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a>) in each condition since non-optimality in these genes should have higher effect (in comparison to lowly expressed genes) on the organism/viral fitness; eventually, at early/late stages the early/late genes are expected to have higher mean decoding rate than late/early genes, respectively (as we see in <a reveal-id="dsx005-F2" data-open="dsx005-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>).</p><p>Whilst the codons of the early/late genes are different both groups use all the codons. However, the fact that the two groups of genes have different codons also means that they use a little different set of tRNA molecules—the late genes use tRNA molecules less used by the early genes.</p><p>An interesting topic for future study is measuring the tRNA levels in different time point and an interesting topic for future study will be to perform such measurements to validate and better understand the different codon frequencies of early/late genes.</p><p>Finally, whilst this study focused on codon frequencies and its relation to translation elongation our analysis suggest that additional regulatory aspects are encoded in via the local folding of the viral RNA and possibly additional viral genomic motives (longer than singe codon; <a reveal-id="dsx005-F3" data-open="dsx005-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>); these results and the previous ones demonstrate that the complexity and information content of the Bacteriophage genome is higher than thought before and encourages further studies in this direction.</p>                            <h2 scrollto-destination=106354765 id="106354765" class="backsection-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/supplemental_material_final_dsx005.pdf?Expires=2147483647&Signature=JfV351g5JZJlaQLNJvevlyaEXZLTzH-T39nHddiYwSmsEsgAFanrPc8EVN7JnK2D7oFgOQCwnRY9DWiP-056DkUChg74QKm8urRjrXYZrKtNzVdxIPxRg8t0pqmRljaU18jUVZDV~pYPzgSUZjesvalmlTY0dYZotLEUh3AAaBQ8Iz-2311-Vc5o9CNPIq7GuWPQ-wk4FwA3f-qDwOyYFvrsaeUGkwaumZDALzopR28H1ROQz75jGGJfxATVapbAA5w7rvo22w3GnseL82lIad-gT-ArjTI-zLtIQ0jBvuiQKGwg7FWf01zQSyuBSoXZDiM0B6juYPwtYfqH3sZn7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> Online.</p>                            <h2 scrollto-destination=106354767 id="106354767" class="backacknowledgements-title" >Acknowledgements</h2>
<p>E.G. and A.D. are supported, in part, by a fellowship from the Edmond J. Safra Center for Bioinformatics at Tel-Aviv Universit. A.D. is grateful to the Azrieli Foundation for the award of an Azrieli Fellowship. T.T. is grateful to the Minerva ARCHES award.</p>                            <h3 scrollto-destination=106354769 id="106354769" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h2 scrollto-destination=106354771 id="106354771" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx005-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Haq</div>
               <div class="given-names">I.U.</div>
            </div>, <div class="name string-name"><div class="surname">Chaudhry</div> <div class="given-names">W.N.</div></div>, <div class="name string-name"><div class="surname">Akhtar</div> <div class="given-names">M.N.</div></div>, <div class="name string-name"><div class="surname">Andleeb</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Qadri</div> <div class="given-names">I.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Bacteriophages and their implications on future biotechnology: a review</div>. <div class="source">Virol. J</div> ., <div class="volume">9</div>.</div></div></div></div><div content-id="dsx005-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Clark</div>
               <div class="given-names">J.R.</div>
            </div>, <div class="name string-name"><div class="surname">March</div> <div class="given-names">J.B.</div></div></div>
          <div class="year">2006</div>, 
<div class="article-title">Bacteriophages and biotechnology: vaccines, gene therapy and antibacterials</div>. <div class="source">Trends Biotechnol</div> ., <div class="volume">24</div>, <div class="fpage">212</div>–<div class="lpage">218</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacteriophages%20and%20biotechnology%3A%20vaccines%2C%20gene%20therapy%20and%20antibacterials&amp;author=J.R.%20Clark&amp;author=J.B.%20March&amp;publication_year=2006&amp;journal=Trends%20Biotechnol&amp;volume=24&amp;pages=212-218" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2006.03.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2006.03.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2006.03.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16567009" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hermoso</div>
               <div class="given-names">J.A.</div>
            </div>, <div class="name string-name"><div class="surname">Garcia</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Garcia</div> <div class="given-names">P.</div></div></div>
          <div class="year">2007</div>, 
<div class="article-title">Taking aim on bacterial pathogens: from phage therapy to enzybiotics</div>. <div class="source">Curr. Opin. Microbiol</div> ., <div class="volume">10</div>, <div class="fpage">461</div>–<div class="lpage">472</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Taking%20aim%20on%20bacterial%20pathogens%3A%20from%20phage%20therapy%20to%20enzybiotics&amp;author=J.A.%20Hermoso&amp;author=J.L.%20Garcia&amp;author=P.%20Garcia&amp;publication_year=2007&amp;journal=Curr.%20Opin.%20Microbiol&amp;volume=10&amp;pages=461-472" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2007.08.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2007.08.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2007.08.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17904412" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bull</div>
               <div class="given-names">J.J.</div>
            </div>, <div class="name string-name"><div class="surname">Cunningham</div> <div class="given-names">C.W.</div></div>, <div class="name string-name"><div class="surname">Molineux</div> <div class="given-names">I.J.</div></div>, <div class="name string-name"><div class="surname">Badgett</div> <div class="given-names">M.R.</div></div>, <div class="name string-name"><div class="surname">Hillis</div> <div class="given-names">D.M.</div></div></div>
          <div class="year">1993</div>, 
<div class="article-title">Experimental molecular evolution of bacteriophage-T7</div>. <div class="source">Evolution</div> , <div class="volume">47</div>, <div class="fpage">993</div>–<div class="lpage">1007</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20molecular%20evolution%20of%20bacteriophage-T7&amp;author=J.J.%20Bull&amp;author=C.W.%20Cunningham&amp;author=I.J.%20Molineux&amp;author=M.R.%20Badgett&amp;author=D.M.%20Hillis&amp;publication_year=1993&amp;journal=Evolution&amp;volume=47&amp;pages=993-1007" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/evo.1993.47.issue-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fevo.1993.47.issue-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fevo.1993.47.issue-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28564289" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Arias</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Weisburd</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Stern-Ginossar</div> <div class="given-names">N.</div></div></div>, et al.  <div class="year">2014</div>, 
<div class="article-title">KSHV 2.0: a comprehensive annotation of the Kaposi's sarcoma-associated herpesvirus genome using nextgeneration sequencing reveals novel genomic and functional features</div>. <div class="source">Plos Pathog</div> ., <div class="volume">10</div>.</div></div></div></div><div content-id="dsx005-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yang</div>
               <div class="given-names">Z.L.</div>
            </div>, <div class="name string-name"><div class="surname">Cao</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Martens</div> <div class="given-names">C.A.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Deciphering poxvirus gene expression by RNA sequencing and ribosome profiling</div>. <div class="source">J. Virol</div> ., <div class="volume">89</div>, <div class="fpage">6874</div>–<div class="lpage">6886</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deciphering%20poxvirus%20gene%20expression%20by%20RNA%20sequencing%20and%20ribosome%20profiling&amp;author=Z.L.%20Yang&amp;author=S.%20Cao&amp;author=C.A.%20Martens&amp;publication_year=2015&amp;journal=J.%20Virol&amp;volume=89&amp;pages=6874-6886" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JVI.00528-15" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJVI.00528-15" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJVI.00528-15"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25903347" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Stern-Ginossar</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Ingolia</div> <div class="given-names">N.T.</div></div></div>
          <div class="year">2015</div>, Ribosome profiling as a tool to decipher viral complexity. In: L.W., Enquist, (ed.), <em>Annual Review of Virology, Vol. 2</em>, pp. <div class="fpage">335</div>–<div class="lpage">349</div>.</div></div></div></div><div content-id="dsx005-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Irigoyen</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Firth</div> <div class="given-names">A.E.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">J.D.</div></div>, <div class="name string-name"><div class="surname">Chung</div> <div class="given-names">B.Y.W.</div></div>, <div class="name string-name"><div class="surname">Siddell</div> <div class="given-names">S.G.</div></div>, <div class="name string-name"><div class="surname">Brierley</div> <div class="given-names">I.</div></div></div>
          <div class="year">2016</div>, 
<div class="article-title">High-resolution analysis of coronavirus gene expression by RNA sequencing and ribosome profiling</div>. <div class="source">Plos Pathog</div> ., <div class="volume">12</div>.</div></div></div></div><div content-id="dsx005-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Liu</div>
               <div class="given-names">X.Q.</div>
            </div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">H.F.</div></div>, <div class="name string-name"><div class="surname">Gu</div> <div class="given-names">Z.L.</div></div>, <div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">J.W.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">High-resolution view of bacteriophage lambda gene expression by ribosome profiling</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">110</div>, <div class="fpage">11928</div>–<div class="lpage">11933</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20view%20of%20bacteriophage%20lambda%20gene%20expression%20by%20ribosome%20profiling&amp;author=X.Q.%20Liu&amp;author=H.F.%20Jiang&amp;author=Z.L.%20Gu&amp;author=J.W.%20Roberts&amp;publication_year=2013&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=110&amp;pages=11928-11933" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1309739110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1309739110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1309739110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23812753" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Carbone</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="year">2008</div>, 
<div class="article-title">Codon bias is a major factor explaining phage evolution in translationally biased hosts</div>. <div class="source">J. Mol. Evol</div> ., <div class="volume">66</div>, <div class="fpage">210</div>–<div class="lpage">223</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20bias%20is%20a%20major%20factor%20explaining%20phage%20evolution%20in%20translationally%20biased%20hosts&amp;author=A.%20Carbone&amp;publication_year=2008&amp;journal=J.%20Mol.%20Evol&amp;volume=66&amp;pages=210-223" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-008-9068-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-008-9068-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-008-9068-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18286220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Coleman</div>
               <div class="given-names">J.R.</div>
            </div>, <div class="name string-name"><div class="surname">Papamichail</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Skiena</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Futcher</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wimmer</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Mueller</div> <div class="given-names">S.</div></div></div>
          <div class="year">2008</div>, 
<div class="article-title">Virus attenuation by genome-scale changes in codon pair bias</div>. <div class="source">Science</div> , <div class="volume">320</div>, <div class="fpage">1784</div>–<div class="lpage">1787</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Virus%20attenuation%20by%20genome-scale%20changes%20in%20codon%20pair%20bias&amp;author=J.R.%20Coleman&amp;author=D.%20Papamichail&amp;author=S.%20Skiena&amp;author=B.%20Futcher&amp;author=E.%20Wimmer&amp;author=S.%20Mueller&amp;publication_year=2008&amp;journal=Science&amp;volume=320&amp;pages=1784-1787" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1155761" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1155761" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1155761"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18583614" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jenkins</div>
               <div class="given-names">G.M.</div>
            </div>, <div class="name string-name"><div class="surname">Pagel</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Gould</div> <div class="given-names">E.A.</div></div>, <div class="name string-name"><div class="surname">Zanotto</div> <div class="given-names">P.M.D.</div></div>, <div class="name string-name"><div class="surname">Holmes</div> <div class="given-names">E.C.</div></div></div>
          <div class="year">2001</div>, 
<div class="article-title">Evolution of base composition and codon usage bias in the genus Flavivirus</div>. <div class="source">J. Mol. Evol</div> ., <div class="volume">52</div>, <div class="fpage">383</div>–<div class="lpage">390</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20base%20composition%20and%20codon%20usage%20bias%20in%20the%20genus%20Flavivirus&amp;author=G.M.%20Jenkins&amp;author=M.%20Pagel&amp;author=E.A.%20Gould&amp;author=P.M.D.%20Zanotto&amp;author=E.C.%20Holmes&amp;publication_year=2001&amp;journal=J.%20Mol.%20Evol&amp;volume=52&amp;pages=383-390" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002390010168" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002390010168" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002390010168"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11343134" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Shackelton</div>
               <div class="given-names">L.A.</div>
            </div>, <div class="name string-name"><div class="surname">Parrish</div> <div class="given-names">C.R.</div></div>, <div class="name string-name"><div class="surname">Holmes</div> <div class="given-names">E.C.</div></div></div>
          <div class="year">2006</div>, 
<div class="article-title">Evolutionary basis of codon usage and nucleotide composition bias in vertebrate DNA viruses</div>. <div class="source">J. Mol. Evol</div> ., <div class="volume">62</div>, <div class="fpage">551</div>–<div class="lpage">563</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20basis%20of%20codon%20usage%20and%20nucleotide%20composition%20bias%20in%20vertebrate%20DNA%20viruses&amp;author=L.A.%20Shackelton&amp;author=C.R.%20Parrish&amp;author=E.C.%20Holmes&amp;publication_year=2006&amp;journal=J.%20Mol.%20Evol&amp;volume=62&amp;pages=551-563" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-005-0221-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-005-0221-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-005-0221-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16557338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Herskowi</div>
               <div class="given-names">I.</div>
            </div>
          </div>
          <div class="year">1973</div>, 
<div class="article-title">Control of gene-expression in bacteriophage-lambda</div>. <div class="source">Ann. Rev. Genet</div> ., <div class="volume">7</div>, <div class="fpage">289</div>–<div class="lpage">324</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Control%20of%20gene-expression%20in%20bacteriophage-lambda&amp;author=I.%20Herskowi&amp;publication_year=1973&amp;journal=Ann.%20Rev.%20Genet&amp;volume=7&amp;pages=289-324" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.ge.07.120173.001445" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.ge.07.120173.001445" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.ge.07.120173.001445"> </span></div> </div></div></div></div></div><div content-id="dsx005-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Casjens</div>
               <div class="given-names">S.R.</div>
            </div>, <div class="name string-name"><div class="surname">Hendrix</div> <div class="given-names">R.W.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Bacteriophage lambda: early pioneer and still relevant</div>. <div class="source">Virology</div> , <div class="volume">479</div>, <div class="fpage">310</div>–<div class="lpage">330</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacteriophage%20lambda%3A%20early%20pioneer%20and%20still%20relevant&amp;author=S.R.%20Casjens&amp;author=R.W.%20Hendrix&amp;publication_year=2015&amp;journal=Virology&amp;volume=479&amp;pages=310-330" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.virol.2015.02.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.virol.2015.02.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.virol.2015.02.010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25742714" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Goz</div>
               <div class="given-names">E.T.T.</div>
            </div>
          </div>
          <div class="year">2016</div>, 
<div class="article-title">Evidence of a direct evolutionary selection for strong folding and mutational robustness within HIV coding regions</div>. <div class="source">J. Comput. Biol</div> ., <div class="volume">23</div>.</div></div></div></div><div content-id="dsx005-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Goz</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Widespread signatures of local mRNA folding structure selection in four Dengue virus serotypes</div>. <div class="source">BMC Genom</div> ., <div class="volume">16</div>.</div></div></div></div><div content-id="dsx005-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">Q.</div>
            </div>, <div class="name string-name"><div class="surname">Barr</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">C.</div></div></div>
          <div class="year">2008</div>, 
<div class="article-title">Evidence of a novel RNA secondary structure in the coding region of HIV-1 pol gene</div>. <div class="source">RNA-Publ. RNA Soc</div> ., <div class="volume">14</div>, <div class="fpage">2478</div>–<div class="lpage">2488</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20of%20a%20novel%20RNA%20secondary%20structure%20in%20the%20coding%20region%20of%20HIV-1%20pol%20gene&amp;author=Q.%20Wang&amp;author=I.%20Barr&amp;author=F.%20Guo&amp;author=C.%20Lee&amp;publication_year=2008&amp;journal=RNA-Publ.%20RNA%20Soc&amp;volume=14&amp;pages=2478-2488" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.1252608" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.1252608" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.1252608"> </span></div> </div></div></div></div></div><div content-id="dsx005-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Alvarez</div>
               <div class="given-names">D.E.</div>
            </div>, <div class="name string-name"><div class="surname">Ezcurra</div> <div class="given-names">A.L.D.</div></div>, <div class="name string-name"><div class="surname">Fucito</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Gamarnik</div> <div class="given-names">A.V.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title">Role of RNA structures present at the 3ʹ UTR of dengue virus on translation, RNA synthesis, and viral replication</div>. <div class="source">Virology</div> , <div class="volume">339</div>, <div class="fpage">200</div>–<div class="lpage">212</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20RNA%20structures%20present%20at%20the%203%CA%B9%20UTR%20of%20dengue%20virus%20on%20translation%2C%20RNA%20synthesis%2C%20and%20viral%20replication&amp;author=D.E.%20Alvarez&amp;author=A.L.D.%20Ezcurra&amp;author=S.%20Fucito&amp;author=A.V.%20Gamarnik&amp;publication_year=2005&amp;journal=Virology&amp;volume=339&amp;pages=200-212" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.virol.2005.06.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.virol.2005.06.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.virol.2005.06.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16002117" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Smith</div>
               <div class="given-names">D.B.</div>
            </div>, <div class="name string-name"><div class="surname">Mellor</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Jarvis</div> <div class="given-names">L.M.</div></div></div>, et al.  <div class="year">1995</div>, 
<div class="article-title">variation of the hepatitis-C virus 5ʹ noncoding region – implications for secondary structure, virus detection and typing</div>. <div class="source">J. Gener. Virol</div> ., <div class="volume">76</div>, <div class="fpage">1749</div>–<div class="lpage">1761</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=variation%20of%20the%20hepatitis-C%20virus%205%CA%B9%20noncoding%20region%20%E2%80%93%20implications%20for%20secondary%20structure%2C%20virus%20detection%20and%20typing&amp;author=D.B.%20Smith&amp;author=J.%20Mellor&amp;author=L.M.%20Jarvis&amp;publication_year=1995&amp;journal=J.%20Gener.%20Virol&amp;volume=76&amp;pages=1749-1761" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/0022-1317-76-7-1749" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2F0022-1317-76-7-1749" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2F0022-1317-76-7-1749"> </span></div> </div></div></div></div></div><div content-id="dsx005-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Jenkins</div>
               <div class="given-names">G.M.</div>
            </div>, <div class="name string-name"><div class="surname">Holmes</div> <div class="given-names">E.C.</div></div></div>
          <div class="year">2003</div>, 
<div class="article-title">The extent of codon usage bias in human RNA viruses and its evolutionary origin</div>. <div class="source">Virus Res</div> ., <div class="volume">92</div>, <div class="fpage">1</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20extent%20of%20codon%20usage%20bias%20in%20human%20RNA%20viruses%20and%20its%20evolutionary%20origin&amp;author=G.M.%20Jenkins&amp;author=E.C.%20Holmes&amp;publication_year=2003&amp;journal=Virus%20Res&amp;volume=92&amp;pages=1-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-1702(02)00309-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-1702(02)00309-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-1702(02)00309-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12606071" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sharp</div>
               <div class="given-names">P.M.</div>
            </div>, <div class="name string-name"><div class="surname">Rogers</div> <div class="given-names">M.S.</div></div>, <div class="name string-name"><div class="surname">McConnell</div> <div class="given-names">D.J.</div></div></div>
          <div class="year">1985</div>, 
<div class="article-title">selection pressures on codon usage in the complete genome of bacteriophage-T7</div>. <div class="source">J. Mol. Evol</div> ., <div class="volume">21</div>, <div class="fpage">150</div>–<div class="lpage">160</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=selection%20pressures%20on%20codon%20usage%20in%20the%20complete%20genome%20of%20bacteriophage-T7&amp;author=P.M.%20Sharp&amp;author=M.S.%20Rogers&amp;author=D.J.%20McConnell&amp;publication_year=1985&amp;journal=J.%20Mol.%20Evol&amp;volume=21&amp;pages=150-160" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02100089" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02100089" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02100089"> </span></div> </div></div></div></div></div><div content-id="dsx005-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sahu</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">S.K.</div></div>, <div class="name string-name"><div class="surname">Sau</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ghosh</div> <div class="given-names">T.C.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title">Comparative analysis of the base composition and codon usages in fourteen mycobacteriophage genomes</div>. <div class="source">J. Biomol. Struct. Dyn</div> ., <div class="volume">23</div>, <div class="fpage">63</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20the%20base%20composition%20and%20codon%20usages%20in%20fourteen%20mycobacteriophage%20genomes&amp;author=K.%20Sahu&amp;author=S.K.%20Gupta&amp;author=S.%20Sau&amp;author=T.C.%20Ghosh&amp;publication_year=2005&amp;journal=J.%20Biomol.%20Struct.%20Dyn&amp;volume=23&amp;pages=63-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/07391102.2005.10507047" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F07391102.2005.10507047" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F07391102.2005.10507047"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15918677" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sau</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Gupta</div> <div class="given-names">S.K.</div></div>, <div class="name string-name"><div class="surname">Sau</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ghosh</div> <div class="given-names">T.C.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title">Synonymous codon usage bias in 16 <em>Staphylococcus aureus</em> phages: implication in phage therapy</div>. <div class="source">Virus Res</div> ., <div class="volume">113</div>, <div class="fpage">123</div>–<div class="lpage">131</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20usage%20bias%20in%2016%20Staphylococcus%20aureus%20phages%3A%20implication%20in%20phage%20therapy&amp;author=K.%20Sau&amp;author=S.K.%20Gupta&amp;author=S.%20Sau&amp;author=T.C.%20Ghosh&amp;publication_year=2005&amp;journal=Virus%20Res&amp;volume=113&amp;pages=123-131" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.virusres.2005.05.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.virusres.2005.05.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.virusres.2005.05.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15970346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kunisawa</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kutter</div> <div class="given-names">E.</div></div></div>
          <div class="year">1998</div>, 
<div class="article-title">Comparison of synonymous codon distribution patterns of bacteriophage and host genomes</div>. <div class="source">DNA Res</div> ., <div class="volume">5</div>, <div class="fpage">319</div>–<div class="lpage">326</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20synonymous%20codon%20distribution%20patterns%20of%20bacteriophage%20and%20host%20genomes&amp;author=T.%20Kunisawa&amp;author=S.%20Kanaya&amp;author=E.%20Kutter&amp;publication_year=1998&amp;journal=DNA%20Res&amp;volume=5&amp;pages=319-326" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/5.6.319" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2F5.6.319" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2F5.6.319"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10048480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">MARTIN</div>
               <div class="given-names">M.</div>
            </div>
          </div>
          <div class="year">2011</div>, 
<div class="article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</div>. <div class="source">EMBnet J</div> ., <div class="volume">17</div>, <div class="fpage">2226</div>–<div class="lpage">6089</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&amp;author=M.%20MARTIN&amp;publication_year=2011&amp;journal=EMBnet%20J&amp;volume=17&amp;pages=2226-6089" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.14806/ej.17.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.14806%2Fej.17.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.14806%2Fej.17.1"> </span></div> </div></div></div></div></div><div content-id="dsx005-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>. <div class="source">Genome Biol</div> ., <div class="volume">10</div>.</div></div></div></div><div content-id="dsx005-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">Y.</div>
            </div>, <div class="name string-name"><div class="surname">Ponty</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Blanchette</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Lecuyer</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Waldispuhl</div> <div class="given-names">J.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">SPARCS: a web server to analyze (un)structured regions in coding RNA sequences</div>. <div class="source">Nucl. Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">W480</div>–<div class="lpage">W485</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SPARCS%3A%20a%20web%20server%20to%20analyze%20(un)structured%20regions%20in%20coding%20RNA%20sequences&amp;author=Y.%20Zhang&amp;author=Y.%20Ponty&amp;author=M.%20Blanchette&amp;author=E.%20Lecuyer&amp;author=J.%20Waldispuhl&amp;publication_year=2013&amp;journal=Nucl.%20Acids%20Res&amp;volume=41&amp;pages=W480-W485" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt461" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt461" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt461"> </span></div> </div></div></div></div></div><div content-id="dsx005-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">dos Reis</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Savva</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Wernisch</div> <div class="given-names">L.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">Solving the riddle of codon usage preferences: a test for translational selection</div>. <div class="source">Nucl. Acids Res</div> ., <div class="volume">32</div>, <div class="fpage">5036</div>–<div class="lpage">5044</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Solving%20the%20riddle%20of%20codon%20usage%20preferences%3A%20a%20test%20for%20translational%20selection&amp;author=M.%20dos%20Reis&amp;author=R.%20Savva&amp;author=L.%20Wernisch&amp;publication_year=2004&amp;journal=Nucl.%20Acids%20Res&amp;volume=32&amp;pages=5036-5044" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh834" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh834" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh834"> </span></div> </div></div></div></div></div><div content-id="dsx005-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dana</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">The effect of tRNA levels on decoding times of mRNA codons</div>. <div class="source">Nucl. Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">9171</div>–<div class="lpage">9181</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20effect%20of%20tRNA%20levels%20on%20decoding%20times%20of%20mRNA%20codons&amp;author=A.%20Dana&amp;author=T.%20Tuller&amp;publication_year=2014&amp;journal=Nucl.%20Acids%20Res&amp;volume=42&amp;pages=9171-9181" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku646" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku646" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku646"> </span></div> </div></div></div></div></div><div content-id="dsx005-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lorenz</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Bernhart</div> <div class="given-names">S.H.</div></div>, <div class="name string-name"><div class="surname">Siederdissen</div> <div class="given-names">C.H.Z.</div></div></div>, et al.  <div class="year">2011</div>, 
<div class="article-title">ViennaRNA package 2.0</div>. <div class="source">Algorithms Mol. Biol</div> ., <div class="volume">6</div>.</div></div></div></div><div content-id="dsx005-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Waldman</div> <div class="given-names">Y.Y.</div></div>, <div class="name string-name"><div class="surname">Kupiec</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ruppin</div> <div class="given-names">E.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">Translation efficiency is determined by both codon bias and folding energy</div>. <div class="source">Proc. Natl. Acad. Sci. U. S. A</div> ., <div class="volume">107</div>, <div class="fpage">3645</div>–<div class="lpage">3650</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20efficiency%20is%20determined%20by%20both%20codon%20bias%20and%20folding%20energy&amp;author=T.%20Tuller&amp;author=Y.Y.%20Waldman&amp;author=M.%20Kupiec&amp;author=E.%20Ruppin&amp;publication_year=2010&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=107&amp;pages=3645-3650" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0909910107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0909910107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0909910107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20133581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zuker</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Stiegler</div> <div class="given-names">P</div></div></div>
          <div class="year">1981</div>, 
<div class="article-title">Optimal computer folding of large RNA sequences using thermodynamics and auxiliary information</div>. <div class="source">Nucl. Acids Res</div> ., <div class="volume">9</div>, <div class="fpage">133</div>–<div class="lpage">148</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Optimal%20computer%20folding%20of%20large%20RNA%20sequences%20using%20thermodynamics%20and%20auxiliary%20information&amp;author=M.%20Zuker&amp;author=P%20Stiegler&amp;publication_year=1981&amp;journal=Nucl.%20Acids%20Res&amp;volume=9&amp;pages=133-148" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/9.1.133" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F9.1.133" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F9.1.133"> </span></div> </div></div></div></div></div><div content-id="dsx005-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Zur</div> <div class="given-names">H.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Multiple roles of the coding sequence 5ʹ end in gene expression regulation</div>. <div class="source">Nucl. Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">13</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20roles%20of%20the%20coding%20sequence%205%CA%B9%20end%20in%20gene%20expression%20regulation&amp;author=T.%20Tuller&amp;author=H.%20Zur&amp;publication_year=2015&amp;journal=Nucl.%20Acids%20Res&amp;volume=43&amp;pages=13-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1313" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1313" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1313"> </span></div> </div></div></div></div></div><div content-id="dsx005-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zur</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div>
          <div class="year">2015</div>, 
<div class="article-title">Exploiting hidden information interleaved in the redundancy of the genetic code without prior knowledge</div>. <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">1161</div>–<div class="lpage">1168</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploiting%20hidden%20information%20interleaved%20in%20the%20redundancy%20of%20the%20genetic%20code%20without%20prior%20knowledge&amp;author=H.%20Zur&amp;author=T.%20Tuller&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=1161-1168" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu797" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu797" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu797"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25433697" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tuller</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Carmi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Vestsigian</div> <div class="given-names">K.</div></div></div>, et al.  <div class="year">2010</div>, 
<div class="article-title">An evolutionarily conserved mechanism for controlling the efficiency of protein translation</div>. <div class="source">Cell</div> , <div class="volume">141</div>, <div class="fpage">344</div>–<div class="lpage">354</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20evolutionarily%20conserved%20mechanism%20for%20controlling%20the%20efficiency%20of%20protein%20translation&amp;author=T.%20Tuller&amp;author=A.%20Carmi&amp;author=K.%20Vestsigian&amp;publication_year=2010&amp;journal=Cell&amp;volume=141&amp;pages=344-354" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2010.03.031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2010.03.031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2010.03.031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20403328" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mahlab</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Linial</div> <div class="given-names">M.</div></div></div>
          <div class="year">2012</div>, 
<div class="article-title">Conservation of the relative tRNA composition in healthy and cancerous tissues</div>. <div class="source">RNA-Publ. RNA Soc</div> ., <div class="volume">18</div>, <div class="fpage">640</div>–<div class="lpage">652</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conservation%20of%20the%20relative%20tRNA%20composition%20in%20healthy%20and%20cancerous%20tissues&amp;author=S.%20Mahlab&amp;author=T.%20Tuller&amp;author=M.%20Linial&amp;publication_year=2012&amp;journal=RNA-Publ.%20RNA%20Soc&amp;volume=18&amp;pages=640-652" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1261/rna.030775.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1261%2Frna.030775.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1261%2Frna.030775.111"> </span></div> </div></div></div></div></div><div content-id="dsx005-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yu</div>
               <div class="given-names">C.H.</div>
            </div>, <div class="name string-name"><div class="surname">Dang</div> <div class="given-names">Y.K.</div></div>, <div class="name string-name"><div class="surname">Zhou</div> <div class="given-names">Z.P.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Codon usage influences the local rate of translation elongation to regulate co-translational protein folding</div>. <div class="source">Mol. Cell</div> , <div class="volume">59</div>, <div class="fpage">744</div>–<div class="lpage">754</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20influences%20the%20local%20rate%20of%20translation%20elongation%20to%20regulate%20co-translational%20protein%20folding&amp;author=C.H.%20Yu&amp;author=Y.K.%20Dang&amp;author=Z.P.%20Zhou&amp;publication_year=2015&amp;journal=Mol.%20Cell&amp;volume=59&amp;pages=744-754" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2015.07.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2015.07.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2015.07.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26321254" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sabi</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Tuller</div> <div class="given-names">T.</div></div></div>
          <div class="year">2014</div>, 
<div class="article-title">Modelling the efficiency of codon–tRNA interactions based on codon usage bias</div>. <div class="source">DNA Res</div> ., <div class="volume">21</div>, <div class="fpage">511</div>–<div class="lpage">525</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modelling%20the%20efficiency%20of%20codon%E2%80%93tRNA%20interactions%20based%20on%20codon%20usage%20bias&amp;author=R.%20Sabi&amp;author=T.%20Tuller&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=511-525" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsu017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsu017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsu017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24906480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx005-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx005-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">dos Reis</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Wernisch</div> <div class="given-names">L.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">Estimating translational selection in eukaryotic genomes</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">26</div>, <div class="fpage">451</div>–<div class="lpage">461</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20translational%20selection%20in%20eukaryotic%20genomes&amp;author=M.%20dos%20Reis&amp;author=L.%20Wernisch&amp;publication_year=2009&amp;journal=Mol.%20Biol.%20Evol&amp;volume=26&amp;pages=451-461" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msn272" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsn272" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsn272"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19033257" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3058514&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/4/10.1093_dnares_dsx005/2/supplemental_material_final_dsx005.pdf?Expires=2147483647&amp;Signature=JfV351g5JZJlaQLNJvevlyaEXZLTzH-T39nHddiYwSmsEsgAFanrPc8EVN7JnK2D7oFgOQCwnRY9DWiP-056DkUChg74QKm8urRjrXYZrKtNzVdxIPxRg8t0pqmRljaU18jUVZDV~pYPzgSUZjesvalmlTY0dYZotLEUh3AAaBQ8Iz-2311-Vc5o9CNPIq7GuWPQ-wk4FwA3f-qDwOyYFvrsaeUGkwaumZDALzopR28H1ROQz75jGGJfxATVapbAA5w7rvo22w3GnseL82lIad-gT-ArjTI-zLtIQ0jBvuiQKGwg7FWf01zQSyuBSoXZDiM0B6juYPwtYfqH3sZn7w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">752</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000407965100001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx005"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3058514"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000407965100001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f4%2f333%2f3058514">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000407965100001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F4%2F333%2F3058514" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3058514'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Evidence of translation efficiency adaptation of the coding regions of the bacteriophage lambda" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx005");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f4%2f333%2f3058514">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

