

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":1735,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/12/6/389/344376","@type":"ScholarlyArticle","name":"Genome Comparison  In Silico  in  Neisseria  Suggests Integration of Filamentous Bacteriophages by their Own Transposase","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/12/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2005-01-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsi021","keywords":["transposase","filamentous bacteriophage","integration","prophage","genome comparison"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/12/6/389/344376","author":[{"name":"Kawai, Mikihiko","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Science, The University of Tokyo  Japan    2  Institute of Medical Science, The University of Tokyo  4-6-1 Shirokanedai, Minato-ku, Tokyo 108-8639, Japan    3  Division of Pathology, Immunology and Microbiology, Graduate School of Medicine, The University of Tokyo  Japan","@type":"Person"},{"name":"Uchiyama, Ikuo","affiliation":"4  Research Center for Computational Science, National Institutes of Natural Sciences  Nishigonaka 38, Myodaiji, Okazaki 444-8585, Japan","@type":"Person"},{"name":"Kobayashi, Ichizo","affiliation":"1  Department of Medical Genome Sciences, Graduate School of Frontier Science, The University of Tokyo  Japan    2  Institute of Medical Science, The University of Tokyo  4-6-1 Shirokanedai, Minato-ku, Tokyo 108-8639, Japan    5  Graduate Program in Biophysics and Biochemistry, Graduate School of Science, The University of Tokyo  Japan","@type":"Person"}],"description":"Abstract.  We have identified filamentous prophages, Nf (Neisserial filamentous phages), during an in silico genome comparison in Neisseria. Comparison of three","pageStart":"389","pageEnd":"401","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/6/0/m_dnaresearch12_6.cover.gif?Expires=2147483647&Signature=vmBj-dEpEdSISGGZZVQQ800UdbaQ61VgN5SO6nMO30mtIcmmHegLeBqq~JmeB3Cl71F-HZSxEtC05awn7JLvbmS~R8ehKSKlG67TzUmDNfl2uJ9iwd8X3XVTg535TrTFouPO8FrSAc3H4exNAvHuAshR8~cghIB3WqxBjwF3q4V-gWs6yVOhRl523VyJV6-aueYhJA8pI1jFRHUmPhSY-RKzDUrwgfwrVrH153kk1IiVH3chlu~YmBF~XczCNeM~pNFI-0DwLreboeiSvqsHnGPMQGgfWos-lNOnEyjyXTWeiJBGXgUTUflQE1fArs2l0eo950LpKbLah37z6bPoSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome Comparison  In Silico  in  Neisseria  Suggests Integration of Filamentous Bacteriophages by their Own Transposase","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/6/0/m_dnaresearch12_6.cover.gif?Expires=2147483647&Signature=vmBj-dEpEdSISGGZZVQQ800UdbaQ61VgN5SO6nMO30mtIcmmHegLeBqq~JmeB3Cl71F-HZSxEtC05awn7JLvbmS~R8ehKSKlG67TzUmDNfl2uJ9iwd8X3XVTg535TrTFouPO8FrSAc3H4exNAvHuAshR8~cghIB3WqxBjwF3q4V-gWs6yVOhRl523VyJV6-aueYhJA8pI1jFRHUmPhSY-RKzDUrwgfwrVrH153kk1IiVH3chlu~YmBF~XczCNeM~pNFI-0DwLreboeiSvqsHnGPMQGgfWos-lNOnEyjyXTWeiJBGXgUTUflQE1fArs2l0eo950LpKbLah37z6bPoSQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome Comparison  In Silico  in  Neisseria  Suggests Integration of Filamentous Bacteriophages by their Own Transposase" />
    <meta property="og:description" content="Abstract.  We have identified filamentous prophages, Nf (Neisserial filamentous phages), during an in silico genome comparison in Neisseria. Comparison of three" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/12/6/389/344376" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2005-01-01" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kawai, Mikihiko" /><meta name="citation_author_institution" content="1Department of Medical Genome Sciences, Graduate School of Frontier Science, The University of TokyoJapan" /><meta name="citation_author_institution" content="2Institute of Medical Science, The University of Tokyo4-6-1 Shirokanedai, Minato-ku, Tokyo 108-8639, Japan" /><meta name="citation_author_institution" content="3Division of Pathology, Immunology and Microbiology, Graduate School of Medicine, The University of TokyoJapan" /><meta name="citation_author" content="Uchiyama, Ikuo" /><meta name="citation_author_institution" content="4Research Center for Computational Science, National Institutes of Natural SciencesNishigonaka 38, Myodaiji, Okazaki 444-8585, Japan" /><meta name="citation_author" content="Kobayashi, Ichizo" /><meta name="citation_author_institution" content="1Department of Medical Genome Sciences, Graduate School of Frontier Science, The University of TokyoJapan" /><meta name="citation_author_institution" content="2Institute of Medical Science, The University of Tokyo4-6-1 Shirokanedai, Minato-ku, Tokyo 108-8639, Japan" /><meta name="citation_author_institution" content="5Graduate Program in Biophysics and Biochemistry, Graduate School of Science, The University of TokyoJapan" /><meta name="citation_title" content="Genome Comparison In Silico in Neisseria Suggests Integration of Filamentous Bacteriophages by their Own Transposase" /><meta name="citation_firstpage" content="389" /><meta name="citation_lastpage" content="401" /><meta name="citation_doi" content="10.1093/dnares/dsi021" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="12" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2005/01/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Insertion with long target duplication: a mechanism for gene mobility suggested from comparison of two related bacterial genomes; Gene;  citation_year=2000;  citation_volume=259;  citation_pages=99-108; " /><meta name="citation_reference" content="citation_title=Comparison between Pyrococcus horikoshii and Pyrococcus abyssi genome sequences reveals linkage of restriction-modification genes with large genome polymorphisms; Gene;  citation_year=2000;  citation_volume=259;  citation_pages=109-121; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of Neisseria meningitidis serogroup B strain MC58; Science;  citation_year=2000;  citation_volume=287;  citation_pages=1809-1815; " /><meta name="citation_reference" content="citation_title=The Bacteriophages; citation_publisher=Plenum Publishing Corporation, New York, NY;  citation_year=1988;  citation_volume=Vol. 2;  citation_pages=375-456; " /><meta name="citation_reference" content="citation_title=Lysogenic conversion by a filamentous phage encoding cholera toxin; Science;  citation_year=1996;  citation_volume=272;  citation_pages=1910-1914; " /><meta name="citation_reference" content="citation_title=Filamentous phage integration requires the host recombinases XerC and XerD; Nature;  citation_year=2002;  citation_volume=417;  citation_pages=656-659; " /><meta name="citation_reference" content="citation_title=Bacteriophage and phenotypic variation in Pseudomonas aeruginosa biofilm development; J. Bacteriol.;  citation_year=2004;  citation_volume=186;  citation_pages=8066-8073; " /><meta name="citation_reference" content="citation_title=Complete DNA sequence of a serogroup A strain of Neisseria meningitidis Z2491; Nature;  citation_year=2000;  citation_volume=404;  citation_pages=502-506; " /><meta name="citation_reference" content="citation_title=A novel IS element, IS621, of the IS110/IS492 family transposes to a specific site in repetitive extragenic palindromic sequences in Escherichia coli; J. Bacteriol.;  citation_year=2003;  citation_volume=185;  citation_pages=4891-4900; " /><meta name="citation_reference" content="citation_title=Analysis of the Piv recombinase-related gene family of Neisseria gonorrhoeae; J. Bacteriol.;  citation_year=2005;  citation_volume=187;  citation_pages=1276-1286; " /><meta name="citation_reference" content="citation_title=Amino acid sequence homology between Piv, an essential protein in site-specific DNA inversion in Moraxella lacunata, and transposases of an unusual family of insertion elements; J. Bacteriol.;  citation_year=1994;  citation_volume=176;  citation_pages=4160-4164; " /><meta name="citation_reference" content="citation_title=A chromosomally integrated bacteriophage in invasive meningococci; J. Exp. Med.;  citation_year=2005;  citation_volume=201;  citation_pages=1905-1913; " /><meta name="citation_reference" content="citation_title=Genetic isolation of meningococci of the electrophoretic type 37 complex; J. Bacteriol.;  citation_year=2001;  citation_volume=183;  citation_pages=2570-2575; " /><meta name="citation_reference" content="citation_title=Structural analysis of a putative hypovirulent plasmid, pJTPS1, found in a spontaneous avirulent mutant of Ralstonia solanacearum; Ann. Phytopathol. Soc. Jpn.;  citation_year=1999;  citation_volume=65;  citation_pages=184-188; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of a plasmid DNA from periodontopathogenic bacterium, Eikenella corrodens 1073, which affects pilus formation and colony morphology; Gene;  citation_year=2005;  citation_volume=351;  citation_pages=143-148; " /><meta name="citation_reference" content="citation_title=Nucleotide sequence of the filamentous bacteriophage M13 DNA genome: comparison with phage fd; Gene;  citation_year=1980;  citation_volume=11;  citation_pages=129-148; " /><meta name="citation_reference" content="citation_title=DNA sequence of the filamentous bacteriophage Pf1; J. Mol. Biol.;  citation_year=1991;  citation_volume=218;  citation_pages=349-364; " /><meta name="citation_reference" content="citation_title=Nucleotide sequence of the genome of Pf3, an IncP-1 plasmid-specific filamentous bacteriophage of Pseudomonas aeruginosa; J. Virol.;  citation_year=1985;  citation_volume=56;  citation_pages=268-276; " /><meta name="citation_reference" content="citation_title=VGJφ, a novel filamentous phage of Vibrio cholerae, integrates into the same chromosomal site as CTXφ; J. Bacteriol.;  citation_year=2003;  citation_volume=185;  citation_pages=5685-5696; " /><meta name="citation_reference" content="citation_title=Filamentous phage active on the gram-positive bacterium Propionibacterium freudenreichii; J. Bacteriol.;  citation_year=2002;  citation_volume=184;  citation_pages=2030-2033; " /><meta name="citation_reference" content="citation_title=GeneMark.hmm: new solutions for gene finding; Nucleic Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=1107-1115; " /><meta name="citation_reference" content="citation_title=Genome Informatics 2000; citation_publisher=Universal Academy Press, Tokyo, Japan;  citation_year=2000;  citation_pages=341-343; " /><meta name="citation_reference" content="citation_title=MBGD: microbial genome database for comparative analysis; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=58-62; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-3402; " /><meta name="citation_reference" content="citation_title=Improved tools for biological sequence comparison; Proc. Natl Acad. Sci. USA;  citation_year=1988;  citation_volume=85;  citation_pages=2444-2448; " /><meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice; Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=4673-4680; " /><meta name="citation_reference" content="citation_title=WWW-query: an on-line retrieval system for biological sequence banks; Biochimie;  citation_year=1996;  citation_volume=78;  citation_pages=364-369; " /><meta name="citation_reference" content="citation_title=Escherichia coli and Salmonella: Cellular and Molecular Biology, Vol. 1; citation_publisher=American Society of Microbiology, Washington, D.C.;  citation_year=1996;  citation_pages=792-821; " /><meta name="citation_reference" content="citation_title=DBTBS: database of transcriptional regulation in Bacillus subtilis and its contribution to comparative genomics; Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=D75-77; " /><meta name="citation_reference" content="citation_title=Long DNA palindromes, cruciform structures, genetic instability and secondary structure repair; Bioessays;  citation_year=1994;  citation_volume=16;  citation_pages=893-900; " /><meta name="citation_reference" content="citation_title=Instability of long inverted repeats within mouse transgenes; EMBO J.;  citation_year=1996;  citation_volume=15;  citation_pages=1163-1171; " /><meta name="citation_reference" content="citation_title=One-ended insertion of IS911; J. Bacteriol.;  citation_year=1994;  citation_volume=176;  citation_pages=1192-1196; " /><meta name="citation_reference" content="citation_title=Transposase-induced excision and circularization of the bacterial insertion sequence IS911; EMBO J.;  citation_year=1992;  citation_volume=11;  citation_pages=5079-5090; " /><meta name="citation_reference" content="citation_title=Translational control in production of transposase and in transposition of insertion sequence IS3; J. Mol. Biol.;  citation_year=1994;  citation_volume=235;  citation_pages=1406-1420; " /><meta name="citation_reference" content="citation_title=Excision of IS492 requires flanking target sequences and results in circle formation in Pseudoalteromonas atlantica; J. Bacteriol.;  citation_year=1999;  citation_volume=181;  citation_pages=4937-4948; " /><meta name="citation_reference" content="citation_title=Conserved amino acid motifs from the novel Piv/MooV family of transposases and site-specific recombinases are required for catalysis of DNA inversion by Piv; Mol. Microbiol.;  citation_year=2001;  citation_volume=39;  citation_pages=641-651; " /><meta name="citation_reference" content="citation_title=Primary structure and sidechain interactions of PFL filamentous bacterial virus coat protein; Nature;  citation_year=1975;  citation_volume=253;  citation_pages=68-71; " /><meta name="citation_reference" content="citation_title=The DNA-binding protein of Pf1 filamentous bacteriophage: amino-acid sequence and structure of the gene; EMBO J.;  citation_year=1982;  citation_volume=1;  citation_pages=255-261; " /><meta name="citation_reference" content="citation_title=Major coat protein and single-stranded DNA-binding protein of filamentous virus Pf3; Proc. Natl Acad. Sci. USA;  citation_year=1984;  citation_volume=81;  citation_pages=699-703; " /><meta name="citation_reference" content="citation_title=The second cholera toxin, Zot, and its plasmid-encoded and phage-encoded homologues constitute a group of putative ATPases with an altered purine NTP-binding motif; FEBS Lett.;  citation_year=1992;  citation_volume=312;  citation_pages=3-6; " /><meta name="citation_reference" content="citation_title=Filamentous phage assembly: variation on a protein export theme; Gene;  citation_year=1997;  citation_volume=192;  citation_pages=23-32; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/12/6/389/17462403/dsi021.pdf" /><meta name="description" content="Abstract.  We have identified filamentous prophages, Nf (Neisserial filamentous phages), during an in silico genome comparison in Neisseria. Comparison of three" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/12/6/389/344376" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/12/6/389/344376" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/12/6/389/344376");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsi021';
    </script>
<title>
	Genome Comparison In Silico in Neisseria Suggests Integration of Filamentous Bacteriophages by their Own Transposase | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./344376" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZRfPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXdhaSwgTWlraWhpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIxRGVwYXJ0bWVudCBvZiBNZWRpY2FsIEdlbm9tZSBTY2llbmNlcywgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2UsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSIySW5zdGl0dXRlIG9mIE1lZGljYWwgU2NpZW5jZSwgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW80LTYtMSBTaGlyb2thbmVkYWksIE1pbmF0by1rdSwgVG9reW8gMTA4LTg2MzksIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iM0RpdmlzaW9uIG9mIFBhdGhvbG9neSwgSW1tdW5vbG9neSBhbmQgTWljcm9iaW9sb2d5LCBHcmFkdWF0ZSBTY2hvb2wgb2YgTWVkaWNpbmUsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJVY2hpeWFtYSwgSWt1byIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjRSZXNlYXJjaCBDZW50ZXIgZm9yIENvbXB1dGF0aW9uYWwgU2NpZW5jZSwgTmF0aW9uYWwgSW5zdGl0dXRlcyBvZiBOYXR1cmFsIFNjaWVuY2VzTmlzaGlnb25ha2EgMzgsIE15b2RhaWppLCBPa2F6YWtpIDQ0NC04NTg1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvYmF5YXNoaSwgSWNoaXpvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMURlcGFydG1lbnQgb2YgTWVkaWNhbCBHZW5vbWUgU2NpZW5jZXMsIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5b0phcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iMkluc3RpdHV0ZSBvZiBNZWRpY2FsIFNjaWVuY2UsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvNC02LTEgU2hpcm9rYW5lZGFpLCBNaW5hdG8ta3UsIFRva3lvIDEwOC04NjM5LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IjVHcmFkdWF0ZSBQcm9ncmFtIGluIEJpb3BoeXNpY3MgYW5kIEJpb2NoZW1pc3RyeSwgR3JhZHVhdGUgU2Nob29sIG9mIFNjaWVuY2UsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ikdlbm9tZSBDb21wYXJpc29uIEluIFNpbGljbyBpbiBOZWlzc2VyaWEgU3VnZ2VzdHMgSW50ZWdyYXRpb24gb2YgRmlsYW1lbnRvdXMgQmFjdGVyaW9waGFnZXMgYnkgdGhlaXIgT3duIFRyYW5zcG9zYXNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzg5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0MDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc2kwMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAwNS8wMS8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc2VydGlvbiB3aXRoIGxvbmcgdGFyZ2V0IGR1cGxpY2F0aW9uOiBhIG1lY2hhbmlzbSBmb3IgZ2VuZSBtb2JpbGl0eSBzdWdnZXN0ZWQgZnJvbSBjb21wYXJpc29uIG9mIHR3byByZWxhdGVkIGJhY3RlcmlhbCBnZW5vbWVzOyBHZW5lOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI1OTsgIGNpdGF0aW9uX3BhZ2VzPTk5LTEwODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBiZXR3ZWVuIFB5cm9jb2NjdXMgaG9yaWtvc2hpaSBhbmQgUHlyb2NvY2N1cyBhYnlzc2kgZ2Vub21lIHNlcXVlbmNlcyByZXZlYWxzIGxpbmthZ2Ugb2YgcmVzdHJpY3Rpb24tbW9kaWZpY2F0aW9uIGdlbmVzIHdpdGggbGFyZ2UgZ2Vub21lIHBvbHltb3JwaGlzbXM7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjU5OyAgY2l0YXRpb25fcGFnZXM9MTA5LTEyMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIE5laXNzZXJpYSBtZW5pbmdpdGlkaXMgc2Vyb2dyb3VwIEIgc3RyYWluIE1DNTg7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjg3OyAgY2l0YXRpb25fcGFnZXM9MTgwOS0xODE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQmFjdGVyaW9waGFnZXM7IGNpdGF0aW9uX3B1Ymxpc2hlcj1QbGVudW0gUHVibGlzaGluZyBDb3Jwb3JhdGlvbiwgTmV3IFlvcmssIE5ZOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPVZvbC4gMjsgIGNpdGF0aW9uX3BhZ2VzPTM3NS00NTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUx5c29nZW5pYyBjb252ZXJzaW9uIGJ5IGEgZmlsYW1lbnRvdXMgcGhhZ2UgZW5jb2RpbmcgY2hvbGVyYSB0b3hpbjsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0yNzI7ICBjaXRhdGlvbl9wYWdlcz0xOTEwLTE5MTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbGFtZW50b3VzIHBoYWdlIGludGVncmF0aW9uIHJlcXVpcmVzIHRoZSBob3N0IHJlY29tYmluYXNlcyBYZXJDIGFuZCBYZXJEOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDE3OyAgY2l0YXRpb25fcGFnZXM9NjU2LTY1OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjdGVyaW9waGFnZSBhbmQgcGhlbm90eXBpYyB2YXJpYXRpb24gaW4gUHNldWRvbW9uYXMgYWVydWdpbm9zYSBiaW9maWxtIGRldmVsb3BtZW50OyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE4NjsgIGNpdGF0aW9uX3BhZ2VzPTgwNjYtODA3MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgRE5BIHNlcXVlbmNlIG9mIGEgc2Vyb2dyb3VwIEEgc3RyYWluIG9mIE5laXNzZXJpYSBtZW5pbmdpdGlkaXMgWjI0OTE7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT00MDQ7ICBjaXRhdGlvbl9wYWdlcz01MDItNTA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIElTIGVsZW1lbnQsIElTNjIxLCBvZiB0aGUgSVMxMTAvSVM0OTIgZmFtaWx5IHRyYW5zcG9zZXMgdG8gYSBzcGVjaWZpYyBzaXRlIGluIHJlcGV0aXRpdmUgZXh0cmFnZW5pYyBwYWxpbmRyb21pYyBzZXF1ZW5jZXMgaW4gRXNjaGVyaWNoaWEgY29saTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xODU7ICBjaXRhdGlvbl9wYWdlcz00ODkxLTQ5MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIHRoZSBQaXYgcmVjb21iaW5hc2UtcmVsYXRlZCBnZW5lIGZhbWlseSBvZiBOZWlzc2VyaWEgZ29ub3JyaG9lYWU7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTg3OyAgY2l0YXRpb25fcGFnZXM9MTI3Ni0xMjg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbWlubyBhY2lkIHNlcXVlbmNlIGhvbW9sb2d5IGJldHdlZW4gUGl2LCBhbiBlc3NlbnRpYWwgcHJvdGVpbiBpbiBzaXRlLXNwZWNpZmljIEROQSBpbnZlcnNpb24gaW4gTW9yYXhlbGxhIGxhY3VuYXRhLCBhbmQgdHJhbnNwb3Nhc2VzIG9mIGFuIHVudXN1YWwgZmFtaWx5IG9mIGluc2VydGlvbiBlbGVtZW50czsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzY7ICBjaXRhdGlvbl9wYWdlcz00MTYwLTQxNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY2hyb21vc29tYWxseSBpbnRlZ3JhdGVkIGJhY3RlcmlvcGhhZ2UgaW4gaW52YXNpdmUgbWVuaW5nb2NvY2NpOyBKLiBFeHAuIE1lZC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjAxOyAgY2l0YXRpb25fcGFnZXM9MTkwNS0xOTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGlzb2xhdGlvbiBvZiBtZW5pbmdvY29jY2kgb2YgdGhlIGVsZWN0cm9waG9yZXRpYyB0eXBlIDM3IGNvbXBsZXg7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTgzOyAgY2l0YXRpb25fcGFnZXM9MjU3MC0yNTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmFsIGFuYWx5c2lzIG9mIGEgcHV0YXRpdmUgaHlwb3ZpcnVsZW50IHBsYXNtaWQsIHBKVFBTMSwgZm91bmQgaW4gYSBzcG9udGFuZW91cyBhdmlydWxlbnQgbXV0YW50IG9mIFJhbHN0b25pYSBzb2xhbmFjZWFydW07IEFubi4gUGh5dG9wYXRob2wuIFNvYy4gSnBuLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT02NTsgIGNpdGF0aW9uX3BhZ2VzPTE4NC0xODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlzb2xhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBhIHBsYXNtaWQgRE5BIGZyb20gcGVyaW9kb250b3BhdGhvZ2VuaWMgYmFjdGVyaXVtLCBFaWtlbmVsbGEgY29ycm9kZW5zIDEwNzMsIHdoaWNoIGFmZmVjdHMgcGlsdXMgZm9ybWF0aW9uIGFuZCBjb2xvbnkgbW9ycGhvbG9neTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zNTE7ICBjaXRhdGlvbl9wYWdlcz0xNDMtMTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OdWNsZW90aWRlIHNlcXVlbmNlIG9mIHRoZSBmaWxhbWVudG91cyBiYWN0ZXJpb3BoYWdlIE0xMyBETkEgZ2Vub21lOiBjb21wYXJpc29uIHdpdGggcGhhZ2UgZmQ7IEdlbmU7ICBjaXRhdGlvbl95ZWFyPTE5ODA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xMjktMTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgc2VxdWVuY2Ugb2YgdGhlIGZpbGFtZW50b3VzIGJhY3RlcmlvcGhhZ2UgUGYxOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTIxODsgIGNpdGF0aW9uX3BhZ2VzPTM0OS0zNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51Y2xlb3RpZGUgc2VxdWVuY2Ugb2YgdGhlIGdlbm9tZSBvZiBQZjMsIGFuIEluY1AtMSBwbGFzbWlkLXNwZWNpZmljIGZpbGFtZW50b3VzIGJhY3RlcmlvcGhhZ2Ugb2YgUHNldWRvbW9uYXMgYWVydWdpbm9zYTsgSi4gVmlyb2wuOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTU2OyAgY2l0YXRpb25fcGFnZXM9MjY4LTI3NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VkdKz4YsIGEgbm92ZWwgZmlsYW1lbnRvdXMgcGhhZ2Ugb2YgVmlicmlvIGNob2xlcmFlLCBpbnRlZ3JhdGVzIGludG8gdGhlIHNhbWUgY2hyb21vc29tYWwgc2l0ZSBhcyBDVFjPhjsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xODU7ICBjaXRhdGlvbl9wYWdlcz01Njg1LTU2OTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbGFtZW50b3VzIHBoYWdlIGFjdGl2ZSBvbiB0aGUgZ3JhbS1wb3NpdGl2ZSBiYWN0ZXJpdW0gUHJvcGlvbmliYWN0ZXJpdW0gZnJldWRlbnJlaWNoaWk7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTg0OyAgY2l0YXRpb25fcGFnZXM9MjAzMC0yMDMzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lTWFyay5obW06IG5ldyBzb2x1dGlvbnMgZm9yIGdlbmUgZmluZGluZzsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9MTEwNy0xMTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgSW5mb3JtYXRpY3MgMjAwMDsgY2l0YXRpb25fcHVibGlzaGVyPVVuaXZlcnNhbCBBY2FkZW15IFByZXNzLCBUb2t5bywgSmFwYW47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl9wYWdlcz0zNDEtMzQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NQkdEOiBtaWNyb2JpYWwgZ2Vub21lIGRhdGFiYXNlIGZvciBjb21wYXJhdGl2ZSBhbmFseXNpczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9NTgtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdhcHBlZCBCTEFTVCBhbmQgUFNJLUJMQVNUOiBhIG5ldyBnZW5lcmF0aW9uIG9mIHByb3RlaW4gZGF0YWJhc2Ugc2VhcmNoIHByb2dyYW1zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zMzg5LTM0MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmVkIHRvb2xzIGZvciBiaW9sb2dpY2FsIHNlcXVlbmNlIGNvbXBhcmlzb247IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5ODg7ICBjaXRhdGlvbl92b2x1bWU9ODU7ICBjaXRhdGlvbl9wYWdlcz0yNDQ0LTI0NDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNMVVNUQUwgVzogaW1wcm92aW5nIHRoZSBzZW5zaXRpdml0eSBvZiBwcm9ncmVzc2l2ZSBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgdGhyb3VnaCBzZXF1ZW5jZSB3ZWlnaHRpbmcsIHBvc2l0aW9uLXNwZWNpZmljIGdhcCBwZW5hbHRpZXMgYW5kIHdlaWdodCBtYXRyaXggY2hvaWNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz00NjczLTQ2ODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdXVy1xdWVyeTogYW4gb24tbGluZSByZXRyaWV2YWwgc3lzdGVtIGZvciBiaW9sb2dpY2FsIHNlcXVlbmNlIGJhbmtzOyBCaW9jaGltaWU7ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9Nzg7ICBjaXRhdGlvbl9wYWdlcz0zNjQtMzY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc2NoZXJpY2hpYSBjb2xpIGFuZCBTYWxtb25lbGxhOiBDZWxsdWxhciBhbmQgTW9sZWN1bGFyIEJpb2xvZ3ksIFZvbC4gMTsgY2l0YXRpb25fcHVibGlzaGVyPUFtZXJpY2FuIFNvY2lldHkgb2YgTWljcm9iaW9sb2d5LCBXYXNoaW5ndG9uLCBELkMuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fcGFnZXM9NzkyLTgyMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9REJUQlM6IGRhdGFiYXNlIG9mIHRyYW5zY3JpcHRpb25hbCByZWd1bGF0aW9uIGluIEJhY2lsbHVzIHN1YnRpbGlzIGFuZCBpdHMgY29udHJpYnV0aW9uIHRvIGNvbXBhcmF0aXZlIGdlbm9taWNzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz1ENzUtNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxvbmcgRE5BIHBhbGluZHJvbWVzLCBjcnVjaWZvcm0gc3RydWN0dXJlcywgZ2VuZXRpYyBpbnN0YWJpbGl0eSBhbmQgc2Vjb25kYXJ5IHN0cnVjdHVyZSByZXBhaXI7IEJpb2Vzc2F5czsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTg5My05MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluc3RhYmlsaXR5IG9mIGxvbmcgaW52ZXJ0ZWQgcmVwZWF0cyB3aXRoaW4gbW91c2UgdHJhbnNnZW5lczsgRU1CTyBKLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTExNjMtMTE3MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T25lLWVuZGVkIGluc2VydGlvbiBvZiBJUzkxMTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0xNzY7ICBjaXRhdGlvbl9wYWdlcz0xMTkyLTExOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zcG9zYXNlLWluZHVjZWQgZXhjaXNpb24gYW5kIGNpcmN1bGFyaXphdGlvbiBvZiB0aGUgYmFjdGVyaWFsIGluc2VydGlvbiBzZXF1ZW5jZSBJUzkxMTsgRU1CTyBKLjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTUwNzktNTA5MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNsYXRpb25hbCBjb250cm9sIGluIHByb2R1Y3Rpb24gb2YgdHJhbnNwb3Nhc2UgYW5kIGluIHRyYW5zcG9zaXRpb24gb2YgaW5zZXJ0aW9uIHNlcXVlbmNlIElTMzsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3ZvbHVtZT0yMzU7ICBjaXRhdGlvbl9wYWdlcz0xNDA2LTE0MjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4Y2lzaW9uIG9mIElTNDkyIHJlcXVpcmVzIGZsYW5raW5nIHRhcmdldCBzZXF1ZW5jZXMgYW5kIHJlc3VsdHMgaW4gY2lyY2xlIGZvcm1hdGlvbiBpbiBQc2V1ZG9hbHRlcm9tb25hcyBhdGxhbnRpY2E7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTgxOyAgY2l0YXRpb25fcGFnZXM9NDkzNy00OTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXJ2ZWQgYW1pbm8gYWNpZCBtb3RpZnMgZnJvbSB0aGUgbm92ZWwgUGl2L01vb1YgZmFtaWx5IG9mIHRyYW5zcG9zYXNlcyBhbmQgc2l0ZS1zcGVjaWZpYyByZWNvbWJpbmFzZXMgYXJlIHJlcXVpcmVkIGZvciBjYXRhbHlzaXMgb2YgRE5BIGludmVyc2lvbiBieSBQaXY7IE1vbC4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPTY0MS02NTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByaW1hcnkgc3RydWN0dXJlIGFuZCBzaWRlY2hhaW4gaW50ZXJhY3Rpb25zIG9mIFBGTCBmaWxhbWVudG91cyBiYWN0ZXJpYWwgdmlydXMgY29hdCBwcm90ZWluOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5NzU7ICBjaXRhdGlvbl92b2x1bWU9MjUzOyAgY2l0YXRpb25fcGFnZXM9NjgtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBETkEtYmluZGluZyBwcm90ZWluIG9mIFBmMSBmaWxhbWVudG91cyBiYWN0ZXJpb3BoYWdlOiBhbWluby1hY2lkIHNlcXVlbmNlIGFuZCBzdHJ1Y3R1cmUgb2YgdGhlIGdlbmU7IEVNQk8gSi47ICBjaXRhdGlvbl95ZWFyPTE5ODI7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTI1NS0yNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1ham9yIGNvYXQgcHJvdGVpbiBhbmQgc2luZ2xlLXN0cmFuZGVkIEROQS1iaW5kaW5nIHByb3RlaW4gb2YgZmlsYW1lbnRvdXMgdmlydXMgUGYzOyBQcm9jLiBOYXRsIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0xOTg0OyAgY2l0YXRpb25fdm9sdW1lPTgxOyAgY2l0YXRpb25fcGFnZXM9Njk5LTcwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNlY29uZCBjaG9sZXJhIHRveGluLCBab3QsIGFuZCBpdHMgcGxhc21pZC1lbmNvZGVkIGFuZCBwaGFnZS1lbmNvZGVkIGhvbW9sb2d1ZXMgY29uc3RpdHV0ZSBhIGdyb3VwIG9mIHB1dGF0aXZlIEFUUGFzZXMgd2l0aCBhbiBhbHRlcmVkIHB1cmluZSBOVFAtYmluZGluZyBtb3RpZjsgRkVCUyBMZXR0LjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0zMTI7ICBjaXRhdGlvbl9wYWdlcz0zLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbGFtZW50b3VzIHBoYWdlIGFzc2VtYmx5OiB2YXJpYXRpb24gb24gYSBwcm90ZWluIGV4cG9ydCB0aGVtZTsgR2VuZTsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTI7ICBjaXRhdGlvbl9wYWdlcz0yMy0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzEyLzYvMzg5LzE3NDYyNDAzL2RzaTAyMS5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgV2UgaGF2ZSBpZGVudGlmaWVkIGZpbGFtZW50b3VzIHByb3BoYWdlcywgTmYgKE5laXNzZXJpYWwgZmlsYW1lbnRvdXMgcGhhZ2VzKSwgZHVyaW5nIGFuIGluIHNpbGljbyBnZW5vbWUgY29tcGFyaXNvbiBpbiBOZWlzc2VyaWEuIENvbXBhcmlzb24gb2YgdGhyZWUiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTM0NDM3NiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSGwduPJMUV+d+IDvSCJadCL4ctGg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f389%2f344376" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f389%2f344376" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=13225" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/12/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/12/6/0/m_dnaresearch12_6.cover.gif?Expires=2147483647&amp;Signature=vmBj-dEpEdSISGGZZVQQ800UdbaQ61VgN5SO6nMO30mtIcmmHegLeBqq~JmeB3Cl71F-HZSxEtC05awn7JLvbmS~R8ehKSKlG67TzUmDNfl2uJ9iwd8X3XVTg535TrTFouPO8FrSAc3H4exNAvHuAshR8~cghIB3WqxBjwF3q4V-gWs6yVOhRl523VyJV6-aueYhJA8pI1jFRHUmPhSY-RKzDUrwgfwrVrH153kk1IiVH3chlu~YmBF~XczCNeM~pNFI-0DwLreboeiSvqsHnGPMQGgfWos-lNOnEyjyXTWeiJBGXgUTUflQE1fArs2l0eo950LpKbLah37z6bPoSQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/12/6">
                        <div class="volume trailing-comma">Volume 12</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
2005        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87193780">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193780">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193782">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193782">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193786">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193786">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193794">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193794">3. Results and Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87193922">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87193922">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/12/6/403/344476">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome Comparison <em>In Silico</em> in <em>Neisseria</em> Suggests Integration of Filamentous Bacteriophages by their Own Transposase
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mikihiko Kawai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Medical Genome Sciences, Graduate School of Frontier Science, The University of Tokyo</div><div class="addr-line">Japan</div></div><div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Institute of Medical Science, The University of Tokyo</div><div class="addr-line">4-6-1 Shirokanedai, Minato-ku, Tokyo 108-8639, Japan</div></div><div class="aff"><div class="label -label">3</div><sup>3</sup><div class="insititution">Division of Pathology, Immunology and Microbiology, Graduate School of Medicine, The University of Tokyo</div><div class="addr-line">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mikihiko+Kawai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawai M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawai M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mikihiko Kawai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ikuo Uchiyama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><sup>4</sup><div class="insititution">Research Center for Computational Science, National Institutes of Natural Sciences</div><div class="addr-line">Nishigonaka 38, Myodaiji, Okazaki 444-8585, Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ikuo+Uchiyama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Uchiyama I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Uchiyama I&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ikuo Uchiyama</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ichizo Kobayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><sup>1</sup><div class="insititution">Department of Medical Genome Sciences, Graduate School of Frontier Science, The University of Tokyo</div><div class="addr-line">Japan</div></div><div class="aff"><div class="label -label">2</div><sup>2</sup><div class="insititution">Institute of Medical Science, The University of Tokyo</div><div class="addr-line">4-6-1 Shirokanedai, Minato-ku, Tokyo 108-8639, Japan</div></div><div class="aff"><div class="label -label">5</div><sup>5</sup><div class="insititution">Graduate Program in Biophysics and Biochemistry, Graduate School of Science, The University of Tokyo</div><div class="addr-line">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <sup>*</sup>To whom correspondence should be addressed. Tel. +81-3-5449-5326, Fax. +81-3-5449-5422, E-mail: <a href="mailto:ikobaya@ims.u-tokyo.ac.jp" target="_blank">ikobaya@ims.u-tokyo.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ichizo+Kobayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kobayashi I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kobayashi I&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ichizo Kobayashi</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Communicated by Kenta Nakai</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 12, Issue 6, 1 January 2005, Pages 389–401, <a href='https://doi.org/10.1093/dnares/dsi021'>https://doi.org/10.1093/dnares/dsi021</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">01 January 2005</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">24 August 2005</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Revision Received:</div>
                                <div class="wi-date">24 November 2005</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="344376" href="/dnaresearch/article-pdf/12/6/389/17462403/dsi021.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Mikihiko Kawai, Ikuo Uchiyama, Ichizo Kobayashi;  Genome Comparison <em>In Silico</em> in <em>Neisseria</em> Suggests Integration of Filamentous Bacteriophages by their Own Transposase, <em>DNA Research</em>, Volume 12, Issue 6, 1 January 2005, Pages 389–401, <a href='https://doi.org/10.1093/dnares/dsi021'>https://doi.org/10.1093/dnares/dsi021</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/344376?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/344376?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/344376?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/344376?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/344376?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/344376?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/344376?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome%20Comparison%20%26lt%3Bem%26gt%3BIn%20Silico%26lt%3B%2Fem%26gt%3B%20in%20%26lt%3Bem%26gt%3BNeisseria%26lt%3B%2Fem%26gt%3B%20Suggests%20Integration%20of%20Filamentous%20Bacteriophages%20by%20their%20Own%20Transposase&amp;publicationDate=2005-01-01&amp;volumeNum=12&amp;issueNum=6&amp;author=Kawai%2C%20Mikihiko%3B%20Uchiyama%2C%20Ikuo&amp;startPage=389&amp;endPage=401&amp;contentId=10.1093%2Fdnares%2Fdsi021&amp;oa=&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f389%2f344376" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87193780 id="87193780" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>We have identified filamentous prophages, Nf (<em>Neisserial</em> filamentous phages), during an <em>in silico</em> genome comparison in <em>Neisseria.</em> Comparison of three genomes of <em>Neisseria meningitidis</em> and one of <em>Neisseria gonorrhoeae</em> revealed four subtypes of Nf. Eleven intact copies are located at different loci in the four genomes. Each intact copy of Nf is flanked by duplication of 5′-<span class="monospace">CT</span> and, at its right end, carries a transposase homologue (<em>pivNM/irg</em>) of RNaseH/Retroviral integrase superfamily. The phylogeny of these putative transposases and that of phage-related proteins on Nfs are congruent. Following circularization of Nfs, a promoter-like sequence forms. The sequence at the junction of these predicted circular forms (5′-<span class="monospace">atCTtatat</span>) was found in a related plasmid (pMU1) at a corresponding locus. Several structural variants of Nfs—partially inverted, internally deleted and truncated—were also identified. The partial inversion seems to be a product of site-specific recombination between two 5′-<span class="monospace">CTtat</span> sequences that are in inverse orientation, one at its end and the other upstream of <em>pivNM/irg</em>. Formation of internally deleted variants probably proceeded through replicative transposition that also involved two 5′-<span class="monospace">CTtat</span> sequences. We concluded that the PivNM/Irg transposase on Nfs integrated their circular forms into the chromosomal 5′-CT-containing sequences and probably mediated the above rearrangements.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="transposase">transposase</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;filamentous bacteriophage&quot;">filamentous bacteriophage</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="integration">integration</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="prophage">prophage</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome comparison&quot;">genome comparison</a></div>                            <h2 scrollto-destination=87193782 id="87193782" class="section-title" >1. Introduction</h2>
<p>Genome comparison involving closely related organisms can be useful for understanding the mechanisms of genome rearrangements and for identifying elements that participate in them.<a reveal-id="b1" data-open="b1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="b2" data-open="b2" class="link link-ref link-reveal xref-bibr">2</a> During examination of a large chromosomal rearrangement that was seen when a comparison was made between two genomes of <em>Neisseria meningitidis</em>,<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a> we encountered filamentous prophages.</p><p>The filamentous bacteriophages have a single-stranded circular DNA genome.<a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a> Some of them, such as M13, propagate in a double-stranded circular form in their host bacteria,<a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a> whereas others, such as CTXφ of <em>Vibrio cholerae</em>, integrate themselves into the host chromosome.<a reveal-id="b5" data-open="b5" class="link link-ref link-reveal xref-bibr">5</a> So far, their integration is known to be mediated by one of two types of tyrosine recombinases: integration into a <em>dif</em>-like site by the host-encoded XerC/D recombinase<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> and integration into a tRNA gene by the phage-encoded recombinase.<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a> Quite different from these tyrosine recombinases in sequence and molecular mechanism are the <em>pivNM/irg</em> genes<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="b8  b10" data-open="b8  b10" class="link link-ref link-reveal xref-bibr">8–10</a> that belong to the Piv subfamily of the IS<em>110</em>/IS<em>492</em> transposase family of RNaseH/Retrovial Integrase superfamily.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> The <em>irg</em> genes in <em>Neisseria gonorrhoeae</em> were regarded as the transposases of small insertion sequence (IS) elements,<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> which were found linked with two homologues of filamentous phage proteins and were thought to be integrated into the host chromosome as part of the bacteriophage genome.<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a></p><p>In the present work, we identified prophages of this family in four <em>Neisseria</em> genomes and named this phage family Nf (<em>Neisserial</em> filamentous phages). We found that each intact copy of Nf carries a <em>pivNM/irg</em> homologue at their right end. Our further analysis strongly indicates that this phage family is integrated by a novel mechanism using its own PivNM/Irg transposase. This transposase is probably responsible for variously rearranged prophage genomes. Recently, Bille et al.<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a> published an experimental work that was in agreement with ours (see Section 3.6).</p>                            <h2 scrollto-destination=87193786 id="87193786" class="section-title" >2. Materials and Methods</h2>
                            <h3 scrollto-destination=87193787 id="87193787" class="section-title" >2.1. Accession numbers and gene names</h3>
<p>Accession numbers of the complete genomes are as follows: <em>N. meningitidis</em> serogroup A strain Z2491 (<em>Nme</em>A) (NC_003116.1),<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a>,<em>N. meningitidis</em> serogroup B strain MC58 (<em>Nme</em>B) (NC_003112.1),<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a>,<em>N. gonorrhoeae</em> strain FA1090 (<em>Ngo</em>) (NC_002946.2) (<a class="link link-uri" href="http://www.genome.ou.edu/gono.html" target="_blank"></a>; <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/genomes/framik.cgi?db=genome&amp;gi=635" target="_blank"></a>). The genome sequence of <em>N. meningitidis</em> serogroup C strain FAM18 (<em>Nme</em>C) was obtained from The Sanger Institute (<a class="link" href="ftp://ftp.sanger.ac.uk/pub/pathogens/nm/" target="_blank"></a>). Accession numbers for the sequences of plasmids, filamentous phages and IS are as follows: pJS-B (NC_004758.1),<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> pJTPS1 (NC_001399.1),<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> pMU1 (NC_007093.1),<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> M13 (NC_003287.2),<a reveal-id="b16" data-open="b16" class="link link-ref link-reveal xref-bibr">16</a> Pf1 (NC_001331.1),<a reveal-id="b17" data-open="b17" class="link link-ref link-reveal xref-bibr">17</a> Pf3 (NC_001418.1),<a reveal-id="b18" data-open="b18" class="link link-ref link-reveal xref-bibr">18</a> VGJφ (NC_004736.1),<a reveal-id="b19" data-open="b19" class="link link-ref link-reveal xref-bibr">19</a> B5 (NC_003460.1),<a reveal-id="b20" data-open="b20" class="link link-ref link-reveal xref-bibr">20</a> IS<em>621</em>(AB097054-AB097056).<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a></p><p>We used the entries of the complete genome sequences of <em>Nme</em>A, <em>Nme</em>B and <em>Ngo</em> in the NCBI RefSeq (<a class="link" href="ftp://ftp.ncbi.nlm.nih.gov./genomes/Bacteria/" target="_blank"></a>) for gene names, amino acid sequences and coordinates. Genes of Nfs in <em>Nme</em>C are predicted by GeneMark.hmm for Prokaryotes<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> (Version 2.4; <a class="link link-uri" href="http://opal.biology.gatech.edu/GeneMark/gmhmm2_prok.cgi" target="_blank"></a>), with the nucleotide sequence of Nfs (from the left position to the right position, as shown in <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>) as input sequences and <em>N. meningitidis</em> as species.</p>                        <a id="87193790" scrollto-destination="87193790"></a>
<div content-id="tbl1" class="table-modal"><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Nfs found in four <em>Neisseria</em> genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Subtype </th><th>Name<a reveal-id="tf1-2" data-open="tf1-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Length (bp) </th><th colspan="2">Position<a reveal-id="tf1-3" data-open="tf1-3" class="link link-ref link-reveal xref-fn">c</a> </th><th>Strand </th><th>ORFs contained </th><th><em>pivNM/irg</em> contained<a reveal-id="tf1-4" data-open="tf1-4" class="link link-ref link-reveal xref-fn">d</a> </th><th>Locus<a reveal-id="tf1-5" data-open="tf1-5" class="link link-ref link-reveal xref-fn">e</a> </th><th>Comments </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th colspan="2"><hr /> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th>Left </th><th>Right </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td><em>Nme</em>A </td><td>1 </td><td>Nf1-A<a reveal-id="tf1-6" data-open="tf1-6" class="link link-ref link-reveal xref-fn">f</a> </td><td>8029 </td><td>17 37 172 </td><td>17 45 200 </td><td>+ </td><td>NMA1792-1800 </td><td>NMA1800 </td><td> </td><td> </td></tr><tr><td><em>Nme</em>A </td><td>2 </td><td>Nf2-A1(tr) </td><td>3555 </td><td>11 18 972 </td><td>11 22 526 </td><td>+ </td><td>NMA1167-1173 </td><td>– </td><td>I </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>A </td><td>2 </td><td>Nf2-A2(tr) </td><td>3552 </td><td>19 37 109 </td><td>19 40 660 </td><td>+ </td><td>NMA1999-2005 </td><td>– </td><td>II </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>A </td><td>3 </td><td>Nf3-A<a reveal-id="tf1-6" data-open="tf1-6" class="link link-ref link-reveal xref-fn">f</a> </td><td>7996 </td><td>7 69 557 </td><td>7 61 562 </td><td>− </td><td>NMA0783-0772 </td><td><em>pivNM3</em>(NMA0772) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>B </td><td>1 </td><td>Nf1-B1 </td><td>8695 </td><td>16 00 454 </td><td>16 09 148 </td><td>+ </td><td>NMB1542-1552 </td><td><em>pivNM1</em>-<em>A</em>(NMB1552) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>B </td><td>1 </td><td>Nf1-B2 </td><td>8191 </td><td>16 98 770 </td><td>16 90 580 </td><td>− </td><td>NMB1635-1625 </td><td><em>pivNM1-B</em>(NMB1625) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>B </td><td>2 </td><td>Nf2-B1(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3551 </td><td>9 88 033 </td><td>9 91 583 </td><td>+ </td><td>NMB0971-0975 </td><td>– </td><td>(I) </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>B </td><td>2 </td><td>Nf2-B2(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3552 </td><td>5 05 188 </td><td>5 01 637 </td><td>− </td><td>NMB0485-0480 </td><td>– </td><td>II </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>B </td><td>2 </td><td>Nf2-B3 </td><td>7649 </td><td>18 25 043 </td><td>18 32 691 </td><td>+ </td><td>NMB1742-1750 </td><td><em>pivNM2</em>(NMB1750) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C1 </td><td>8046 </td><td>33 529 </td><td>25 484 </td><td>− </td><td>Not annotated </td><td>[26 583-25 627] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C2 </td><td>8046 </td><td>2 79 961 </td><td>2 88 006 </td><td>+ </td><td>Not annotated </td><td>[2 86 906-2 87 862] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C3 </td><td>8010 </td><td>17 43 242 </td><td>17 51 251 </td><td>+ </td><td>Not annotated </td><td>[17 50 152-17 51 108] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C4 </td><td>8032 </td><td>18 93 109 </td><td>19 01 140 </td><td>+ </td><td>Not annotated </td><td>[19 00 040-1 90 0996] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>2 </td><td>Nf2-C1(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3556 </td><td>9 52 056 </td><td>9 55 611 </td><td>+ </td><td>Not annotated </td><td>– </td><td>I </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>C </td><td>2 </td><td>Nf2-C2(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3551 </td><td>16 96 828 </td><td>17 00 378 </td><td>+ </td><td>Not annotated </td><td>– </td><td>II </td><td>Right side truncated </td></tr><tr><td><em>Ngo</em> </td><td>1 </td><td>Nf1-G(tr)<a reveal-id="tf1-6" data-open="tf1-6" class="link link-ref link-reveal xref-fn">f</a> </td><td>2956 </td><td>16 03 726 </td><td>16 06 681 </td><td>+ </td><td>NGO1646-1648 </td><td><em>irg7</em>(NGO1648) </td><td> </td><td>Left side truncated; head-to-head with Nf4-G6(tr) separated by 73 bp </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G1(del) </td><td>1100 </td><td>7 67 755 </td><td>7 68 854 </td><td>+ </td><td>NGO0773 </td><td><em>irg1</em>(NGO0773) </td><td> </td><td>An internal region deleted </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G2(inv) </td><td>8176 </td><td>10 88 351 </td><td>10 80 176 </td><td>− </td><td>NGO1147-1137 </td><td><em>irg2</em>(NGO1137) </td><td> </td><td>A region containing irg2 is inverted </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G3 </td><td>7975 </td><td>11 10 860 </td><td>11 02 886 </td><td>− </td><td>NGO1170-1164 </td><td><em>irg3</em>(NGO1164) </td><td> </td><td> </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G4(del) </td><td>1174 </td><td>11 48 077 </td><td>11 46 904 </td><td>− </td><td>NGO1200 </td><td><em>irg4</em>(NGO1200) </td><td> </td><td>An internal region deleted </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G5 </td><td>7964 </td><td>12 23 799 </td><td>12 15 836 </td><td>− </td><td>NGO1270-1262 </td><td><em>irg5</em>(NGO1262) </td><td> </td><td> </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G6(tr) </td><td>4406 </td><td>16 03 652 </td><td>15 99 247 </td><td>− </td><td>NGO1645-1641 </td><td><em>irg6</em>(NGO1641) </td><td> </td><td>Left side truncated; head-to-head with Nf1-G(tr) separated by 73 bp </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G8(del)<a reveal-id="tf1-8" data-open="tf1-8" class="link link-ref link-reveal xref-fn">h</a> </td><td>1141 </td><td>16 59 009 </td><td>16 60 149 </td><td>+ </td><td>NGO1703 </td><td><em>irg8</em>(NGO1703) </td><td> </td><td>An internal region deleted </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Subtype </th><th>Name<a reveal-id="tf1-2" data-open="tf1-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Length (bp) </th><th colspan="2">Position<a reveal-id="tf1-3" data-open="tf1-3" class="link link-ref link-reveal xref-fn">c</a> </th><th>Strand </th><th>ORFs contained </th><th><em>pivNM/irg</em> contained<a reveal-id="tf1-4" data-open="tf1-4" class="link link-ref link-reveal xref-fn">d</a> </th><th>Locus<a reveal-id="tf1-5" data-open="tf1-5" class="link link-ref link-reveal xref-fn">e</a> </th><th>Comments </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th colspan="2"><hr /> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th>Left </th><th>Right </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td><em>Nme</em>A </td><td>1 </td><td>Nf1-A<a reveal-id="tf1-6" data-open="tf1-6" class="link link-ref link-reveal xref-fn">f</a> </td><td>8029 </td><td>17 37 172 </td><td>17 45 200 </td><td>+ </td><td>NMA1792-1800 </td><td>NMA1800 </td><td> </td><td> </td></tr><tr><td><em>Nme</em>A </td><td>2 </td><td>Nf2-A1(tr) </td><td>3555 </td><td>11 18 972 </td><td>11 22 526 </td><td>+ </td><td>NMA1167-1173 </td><td>– </td><td>I </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>A </td><td>2 </td><td>Nf2-A2(tr) </td><td>3552 </td><td>19 37 109 </td><td>19 40 660 </td><td>+ </td><td>NMA1999-2005 </td><td>– </td><td>II </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>A </td><td>3 </td><td>Nf3-A<a reveal-id="tf1-6" data-open="tf1-6" class="link link-ref link-reveal xref-fn">f</a> </td><td>7996 </td><td>7 69 557 </td><td>7 61 562 </td><td>− </td><td>NMA0783-0772 </td><td><em>pivNM3</em>(NMA0772) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>B </td><td>1 </td><td>Nf1-B1 </td><td>8695 </td><td>16 00 454 </td><td>16 09 148 </td><td>+ </td><td>NMB1542-1552 </td><td><em>pivNM1</em>-<em>A</em>(NMB1552) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>B </td><td>1 </td><td>Nf1-B2 </td><td>8191 </td><td>16 98 770 </td><td>16 90 580 </td><td>− </td><td>NMB1635-1625 </td><td><em>pivNM1-B</em>(NMB1625) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>B </td><td>2 </td><td>Nf2-B1(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3551 </td><td>9 88 033 </td><td>9 91 583 </td><td>+ </td><td>NMB0971-0975 </td><td>– </td><td>(I) </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>B </td><td>2 </td><td>Nf2-B2(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3552 </td><td>5 05 188 </td><td>5 01 637 </td><td>− </td><td>NMB0485-0480 </td><td>– </td><td>II </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>B </td><td>2 </td><td>Nf2-B3 </td><td>7649 </td><td>18 25 043 </td><td>18 32 691 </td><td>+ </td><td>NMB1742-1750 </td><td><em>pivNM2</em>(NMB1750) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C1 </td><td>8046 </td><td>33 529 </td><td>25 484 </td><td>− </td><td>Not annotated </td><td>[26 583-25 627] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C2 </td><td>8046 </td><td>2 79 961 </td><td>2 88 006 </td><td>+ </td><td>Not annotated </td><td>[2 86 906-2 87 862] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C3 </td><td>8010 </td><td>17 43 242 </td><td>17 51 251 </td><td>+ </td><td>Not annotated </td><td>[17 50 152-17 51 108] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C4 </td><td>8032 </td><td>18 93 109 </td><td>19 01 140 </td><td>+ </td><td>Not annotated </td><td>[19 00 040-1 90 0996] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>2 </td><td>Nf2-C1(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3556 </td><td>9 52 056 </td><td>9 55 611 </td><td>+ </td><td>Not annotated </td><td>– </td><td>I </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>C </td><td>2 </td><td>Nf2-C2(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3551 </td><td>16 96 828 </td><td>17 00 378 </td><td>+ </td><td>Not annotated </td><td>– </td><td>II </td><td>Right side truncated </td></tr><tr><td><em>Ngo</em> </td><td>1 </td><td>Nf1-G(tr)<a reveal-id="tf1-6" data-open="tf1-6" class="link link-ref link-reveal xref-fn">f</a> </td><td>2956 </td><td>16 03 726 </td><td>16 06 681 </td><td>+ </td><td>NGO1646-1648 </td><td><em>irg7</em>(NGO1648) </td><td> </td><td>Left side truncated; head-to-head with Nf4-G6(tr) separated by 73 bp </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G1(del) </td><td>1100 </td><td>7 67 755 </td><td>7 68 854 </td><td>+ </td><td>NGO0773 </td><td><em>irg1</em>(NGO0773) </td><td> </td><td>An internal region deleted </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G2(inv) </td><td>8176 </td><td>10 88 351 </td><td>10 80 176 </td><td>− </td><td>NGO1147-1137 </td><td><em>irg2</em>(NGO1137) </td><td> </td><td>A region containing irg2 is inverted </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G3 </td><td>7975 </td><td>11 10 860 </td><td>11 02 886 </td><td>− </td><td>NGO1170-1164 </td><td><em>irg3</em>(NGO1164) </td><td> </td><td> </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G4(del) </td><td>1174 </td><td>11 48 077 </td><td>11 46 904 </td><td>− </td><td>NGO1200 </td><td><em>irg4</em>(NGO1200) </td><td> </td><td>An internal region deleted </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G5 </td><td>7964 </td><td>12 23 799 </td><td>12 15 836 </td><td>− </td><td>NGO1270-1262 </td><td><em>irg5</em>(NGO1262) </td><td> </td><td> </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G6(tr) </td><td>4406 </td><td>16 03 652 </td><td>15 99 247 </td><td>− </td><td>NGO1645-1641 </td><td><em>irg6</em>(NGO1641) </td><td> </td><td>Left side truncated; head-to-head with Nf1-G(tr) separated by 73 bp </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G8(del)<a reveal-id="tf1-8" data-open="tf1-8" class="link link-ref link-reveal xref-fn">h</a> </td><td>1141 </td><td>16 59 009 </td><td>16 60 149 </td><td>+ </td><td>NGO1703 </td><td><em>irg8</em>(NGO1703) </td><td> </td><td>An internal region deleted </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Copies are sorted by strain, subtype and name.</p></div></div><div content-id="tf1-1"><div class="fn"><p><sup>a</sup><em>Nme</em>A: <em>N. meningitidis</em> Z2491 (serogroup A). <em>Nme</em>B: <em>N. meningitidis</em> MC58 (serogroup B). <em>Nme</em>C: <em>N. meningitidis</em> FAM18 (serogroup C). <em>Ngo</em>: <em>N. gonorrhoeae</em> FA1090.</p></div></div><div content-id="tf1-2"><div class="fn"><p><sup>b</sup>Nfs not specified as ‘(tr)’, ‘(del)’ nor ‘(inv)’ represent intact copies.</p></div></div><div content-id="tf1-3"><div class="fn"><p><sup>c</sup>The genome coordinates indicate their left end (<span class="monospace">T</span> next to the left direct repeat, <span class="monospace">CT<span class="underline">t</span>atat</span>) and their right end (<span class="monospace">T</span> next to the right direct repeat, <span class="monospace">a<span class="underline">t</span>CT</span>).</p></div></div><div content-id="tf1-4"><div class="fn"><p><sup>d</sup><em>pivNM</em> of <em>Nme</em>C are expressed in genome coordinates annotated by ‘GeneMark.hmm for Prokaryote’.</p></div></div><div content-id="tf1-5"><div class="fn"><p><sup>e</sup>Each number indicates corresponding locus. Each Nf not numbered is located at a unique locus. A rearranged locus is in parentheses.</p></div></div><div content-id="tf1-6"><div class="fn"><p><sup>f</sup>No ordinal number is given to the single copy of the subtype in the genome.</p></div></div><div content-id="tf1-7"><div class="fn"><p><sup>g</sup>The ordinal number of the Nf at the corresponding locus in <em>Nme</em>A (see locus column) is given.</p></div></div><div content-id="tf1-8"><div class="fn"><p><sup>h</sup>‘8’ stands for containing <em>irg8</em>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193790" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="tbl1" class="table-wrap-title" data-id="tbl1"><div class="label -label"><strong>Table 1</strong></div><div class="caption"><p>Nfs found in four <em>Neisseria</em> genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Strain<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Subtype </th><th>Name<a reveal-id="tf1-2" data-open="tf1-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Length (bp) </th><th colspan="2">Position<a reveal-id="tf1-3" data-open="tf1-3" class="link link-ref link-reveal xref-fn">c</a> </th><th>Strand </th><th>ORFs contained </th><th><em>pivNM/irg</em> contained<a reveal-id="tf1-4" data-open="tf1-4" class="link link-ref link-reveal xref-fn">d</a> </th><th>Locus<a reveal-id="tf1-5" data-open="tf1-5" class="link link-ref link-reveal xref-fn">e</a> </th><th>Comments </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th colspan="2"><hr /> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th>Left </th><th>Right </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td><em>Nme</em>A </td><td>1 </td><td>Nf1-A<a reveal-id="tf1-6" data-open="tf1-6" class="link link-ref link-reveal xref-fn">f</a> </td><td>8029 </td><td>17 37 172 </td><td>17 45 200 </td><td>+ </td><td>NMA1792-1800 </td><td>NMA1800 </td><td> </td><td> </td></tr><tr><td><em>Nme</em>A </td><td>2 </td><td>Nf2-A1(tr) </td><td>3555 </td><td>11 18 972 </td><td>11 22 526 </td><td>+ </td><td>NMA1167-1173 </td><td>– </td><td>I </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>A </td><td>2 </td><td>Nf2-A2(tr) </td><td>3552 </td><td>19 37 109 </td><td>19 40 660 </td><td>+ </td><td>NMA1999-2005 </td><td>– </td><td>II </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>A </td><td>3 </td><td>Nf3-A<a reveal-id="tf1-6" data-open="tf1-6" class="link link-ref link-reveal xref-fn">f</a> </td><td>7996 </td><td>7 69 557 </td><td>7 61 562 </td><td>− </td><td>NMA0783-0772 </td><td><em>pivNM3</em>(NMA0772) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>B </td><td>1 </td><td>Nf1-B1 </td><td>8695 </td><td>16 00 454 </td><td>16 09 148 </td><td>+ </td><td>NMB1542-1552 </td><td><em>pivNM1</em>-<em>A</em>(NMB1552) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>B </td><td>1 </td><td>Nf1-B2 </td><td>8191 </td><td>16 98 770 </td><td>16 90 580 </td><td>− </td><td>NMB1635-1625 </td><td><em>pivNM1-B</em>(NMB1625) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>B </td><td>2 </td><td>Nf2-B1(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3551 </td><td>9 88 033 </td><td>9 91 583 </td><td>+ </td><td>NMB0971-0975 </td><td>– </td><td>(I) </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>B </td><td>2 </td><td>Nf2-B2(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3552 </td><td>5 05 188 </td><td>5 01 637 </td><td>− </td><td>NMB0485-0480 </td><td>– </td><td>II </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>B </td><td>2 </td><td>Nf2-B3 </td><td>7649 </td><td>18 25 043 </td><td>18 32 691 </td><td>+ </td><td>NMB1742-1750 </td><td><em>pivNM2</em>(NMB1750) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C1 </td><td>8046 </td><td>33 529 </td><td>25 484 </td><td>− </td><td>Not annotated </td><td>[26 583-25 627] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C2 </td><td>8046 </td><td>2 79 961 </td><td>2 88 006 </td><td>+ </td><td>Not annotated </td><td>[2 86 906-2 87 862] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C3 </td><td>8010 </td><td>17 43 242 </td><td>17 51 251 </td><td>+ </td><td>Not annotated </td><td>[17 50 152-17 51 108] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C4 </td><td>8032 </td><td>18 93 109 </td><td>19 01 140 </td><td>+ </td><td>Not annotated </td><td>[19 00 040-1 90 0996] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>2 </td><td>Nf2-C1(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3556 </td><td>9 52 056 </td><td>9 55 611 </td><td>+ </td><td>Not annotated </td><td>– </td><td>I </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>C </td><td>2 </td><td>Nf2-C2(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3551 </td><td>16 96 828 </td><td>17 00 378 </td><td>+ </td><td>Not annotated </td><td>– </td><td>II </td><td>Right side truncated </td></tr><tr><td><em>Ngo</em> </td><td>1 </td><td>Nf1-G(tr)<a reveal-id="tf1-6" data-open="tf1-6" class="link link-ref link-reveal xref-fn">f</a> </td><td>2956 </td><td>16 03 726 </td><td>16 06 681 </td><td>+ </td><td>NGO1646-1648 </td><td><em>irg7</em>(NGO1648) </td><td> </td><td>Left side truncated; head-to-head with Nf4-G6(tr) separated by 73 bp </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G1(del) </td><td>1100 </td><td>7 67 755 </td><td>7 68 854 </td><td>+ </td><td>NGO0773 </td><td><em>irg1</em>(NGO0773) </td><td> </td><td>An internal region deleted </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G2(inv) </td><td>8176 </td><td>10 88 351 </td><td>10 80 176 </td><td>− </td><td>NGO1147-1137 </td><td><em>irg2</em>(NGO1137) </td><td> </td><td>A region containing irg2 is inverted </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G3 </td><td>7975 </td><td>11 10 860 </td><td>11 02 886 </td><td>− </td><td>NGO1170-1164 </td><td><em>irg3</em>(NGO1164) </td><td> </td><td> </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G4(del) </td><td>1174 </td><td>11 48 077 </td><td>11 46 904 </td><td>− </td><td>NGO1200 </td><td><em>irg4</em>(NGO1200) </td><td> </td><td>An internal region deleted </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G5 </td><td>7964 </td><td>12 23 799 </td><td>12 15 836 </td><td>− </td><td>NGO1270-1262 </td><td><em>irg5</em>(NGO1262) </td><td> </td><td> </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G6(tr) </td><td>4406 </td><td>16 03 652 </td><td>15 99 247 </td><td>− </td><td>NGO1645-1641 </td><td><em>irg6</em>(NGO1641) </td><td> </td><td>Left side truncated; head-to-head with Nf1-G(tr) separated by 73 bp </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G8(del)<a reveal-id="tf1-8" data-open="tf1-8" class="link link-ref link-reveal xref-fn">h</a> </td><td>1141 </td><td>16 59 009 </td><td>16 60 149 </td><td>+ </td><td>NGO1703 </td><td><em>irg8</em>(NGO1703) </td><td> </td><td>An internal region deleted </td></tr></tbody></table></div><div content-id="tbl1" class="table-modal"><table><thead name="thead" align=""><tr><th>Strain<a reveal-id="tf1-1" data-open="tf1-1" class="link link-ref link-reveal xref-fn">a</a> </th><th>Subtype </th><th>Name<a reveal-id="tf1-2" data-open="tf1-2" class="link link-ref link-reveal xref-fn">b</a> </th><th>Length (bp) </th><th colspan="2">Position<a reveal-id="tf1-3" data-open="tf1-3" class="link link-ref link-reveal xref-fn">c</a> </th><th>Strand </th><th>ORFs contained </th><th><em>pivNM/irg</em> contained<a reveal-id="tf1-4" data-open="tf1-4" class="link link-ref link-reveal xref-fn">d</a> </th><th>Locus<a reveal-id="tf1-5" data-open="tf1-5" class="link link-ref link-reveal xref-fn">e</a> </th><th>Comments </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th colspan="2"><hr /> </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr><tr><th> </th><th> </th><th> </th><th> </th><th>Left </th><th>Right </th><th> </th><th> </th><th> </th><th> </th><th> </th></tr></thead><tbody><tr><td><em>Nme</em>A </td><td>1 </td><td>Nf1-A<a reveal-id="tf1-6" data-open="tf1-6" class="link link-ref link-reveal xref-fn">f</a> </td><td>8029 </td><td>17 37 172 </td><td>17 45 200 </td><td>+ </td><td>NMA1792-1800 </td><td>NMA1800 </td><td> </td><td> </td></tr><tr><td><em>Nme</em>A </td><td>2 </td><td>Nf2-A1(tr) </td><td>3555 </td><td>11 18 972 </td><td>11 22 526 </td><td>+ </td><td>NMA1167-1173 </td><td>– </td><td>I </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>A </td><td>2 </td><td>Nf2-A2(tr) </td><td>3552 </td><td>19 37 109 </td><td>19 40 660 </td><td>+ </td><td>NMA1999-2005 </td><td>– </td><td>II </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>A </td><td>3 </td><td>Nf3-A<a reveal-id="tf1-6" data-open="tf1-6" class="link link-ref link-reveal xref-fn">f</a> </td><td>7996 </td><td>7 69 557 </td><td>7 61 562 </td><td>− </td><td>NMA0783-0772 </td><td><em>pivNM3</em>(NMA0772) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>B </td><td>1 </td><td>Nf1-B1 </td><td>8695 </td><td>16 00 454 </td><td>16 09 148 </td><td>+ </td><td>NMB1542-1552 </td><td><em>pivNM1</em>-<em>A</em>(NMB1552) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>B </td><td>1 </td><td>Nf1-B2 </td><td>8191 </td><td>16 98 770 </td><td>16 90 580 </td><td>− </td><td>NMB1635-1625 </td><td><em>pivNM1-B</em>(NMB1625) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>B </td><td>2 </td><td>Nf2-B1(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3551 </td><td>9 88 033 </td><td>9 91 583 </td><td>+ </td><td>NMB0971-0975 </td><td>– </td><td>(I) </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>B </td><td>2 </td><td>Nf2-B2(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3552 </td><td>5 05 188 </td><td>5 01 637 </td><td>− </td><td>NMB0485-0480 </td><td>– </td><td>II </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>B </td><td>2 </td><td>Nf2-B3 </td><td>7649 </td><td>18 25 043 </td><td>18 32 691 </td><td>+ </td><td>NMB1742-1750 </td><td><em>pivNM2</em>(NMB1750) </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C1 </td><td>8046 </td><td>33 529 </td><td>25 484 </td><td>− </td><td>Not annotated </td><td>[26 583-25 627] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C2 </td><td>8046 </td><td>2 79 961 </td><td>2 88 006 </td><td>+ </td><td>Not annotated </td><td>[2 86 906-2 87 862] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C3 </td><td>8010 </td><td>17 43 242 </td><td>17 51 251 </td><td>+ </td><td>Not annotated </td><td>[17 50 152-17 51 108] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>1 </td><td>Nf1-C4 </td><td>8032 </td><td>18 93 109 </td><td>19 01 140 </td><td>+ </td><td>Not annotated </td><td>[19 00 040-1 90 0996] </td><td> </td><td> </td></tr><tr><td><em>Nme</em>C </td><td>2 </td><td>Nf2-C1(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3556 </td><td>9 52 056 </td><td>9 55 611 </td><td>+ </td><td>Not annotated </td><td>– </td><td>I </td><td>Right side truncated </td></tr><tr><td><em>Nme</em>C </td><td>2 </td><td>Nf2-C2(tr)<a reveal-id="tf1-7" data-open="tf1-7" class="link link-ref link-reveal xref-fn">g</a> </td><td>3551 </td><td>16 96 828 </td><td>17 00 378 </td><td>+ </td><td>Not annotated </td><td>– </td><td>II </td><td>Right side truncated </td></tr><tr><td><em>Ngo</em> </td><td>1 </td><td>Nf1-G(tr)<a reveal-id="tf1-6" data-open="tf1-6" class="link link-ref link-reveal xref-fn">f</a> </td><td>2956 </td><td>16 03 726 </td><td>16 06 681 </td><td>+ </td><td>NGO1646-1648 </td><td><em>irg7</em>(NGO1648) </td><td> </td><td>Left side truncated; head-to-head with Nf4-G6(tr) separated by 73 bp </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G1(del) </td><td>1100 </td><td>7 67 755 </td><td>7 68 854 </td><td>+ </td><td>NGO0773 </td><td><em>irg1</em>(NGO0773) </td><td> </td><td>An internal region deleted </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G2(inv) </td><td>8176 </td><td>10 88 351 </td><td>10 80 176 </td><td>− </td><td>NGO1147-1137 </td><td><em>irg2</em>(NGO1137) </td><td> </td><td>A region containing irg2 is inverted </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G3 </td><td>7975 </td><td>11 10 860 </td><td>11 02 886 </td><td>− </td><td>NGO1170-1164 </td><td><em>irg3</em>(NGO1164) </td><td> </td><td> </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G4(del) </td><td>1174 </td><td>11 48 077 </td><td>11 46 904 </td><td>− </td><td>NGO1200 </td><td><em>irg4</em>(NGO1200) </td><td> </td><td>An internal region deleted </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G5 </td><td>7964 </td><td>12 23 799 </td><td>12 15 836 </td><td>− </td><td>NGO1270-1262 </td><td><em>irg5</em>(NGO1262) </td><td> </td><td> </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G6(tr) </td><td>4406 </td><td>16 03 652 </td><td>15 99 247 </td><td>− </td><td>NGO1645-1641 </td><td><em>irg6</em>(NGO1641) </td><td> </td><td>Left side truncated; head-to-head with Nf1-G(tr) separated by 73 bp </td></tr><tr><td><em>Ngo</em> </td><td>4 </td><td>Nf4-G8(del)<a reveal-id="tf1-8" data-open="tf1-8" class="link link-ref link-reveal xref-fn">h</a> </td><td>1141 </td><td>16 59 009 </td><td>16 60 149 </td><td>+ </td><td>NGO1703 </td><td><em>irg8</em>(NGO1703) </td><td> </td><td>An internal region deleted </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Copies are sorted by strain, subtype and name.</p></div></div><div content-id="tf1-1"><div class="fn"><p><sup>a</sup><em>Nme</em>A: <em>N. meningitidis</em> Z2491 (serogroup A). <em>Nme</em>B: <em>N. meningitidis</em> MC58 (serogroup B). <em>Nme</em>C: <em>N. meningitidis</em> FAM18 (serogroup C). <em>Ngo</em>: <em>N. gonorrhoeae</em> FA1090.</p></div></div><div content-id="tf1-2"><div class="fn"><p><sup>b</sup>Nfs not specified as ‘(tr)’, ‘(del)’ nor ‘(inv)’ represent intact copies.</p></div></div><div content-id="tf1-3"><div class="fn"><p><sup>c</sup>The genome coordinates indicate their left end (<span class="monospace">T</span> next to the left direct repeat, <span class="monospace">CT<span class="underline">t</span>atat</span>) and their right end (<span class="monospace">T</span> next to the right direct repeat, <span class="monospace">a<span class="underline">t</span>CT</span>).</p></div></div><div content-id="tf1-4"><div class="fn"><p><sup>d</sup><em>pivNM</em> of <em>Nme</em>C are expressed in genome coordinates annotated by ‘GeneMark.hmm for Prokaryote’.</p></div></div><div content-id="tf1-5"><div class="fn"><p><sup>e</sup>Each number indicates corresponding locus. Each Nf not numbered is located at a unique locus. A rearranged locus is in parentheses.</p></div></div><div content-id="tf1-6"><div class="fn"><p><sup>f</sup>No ordinal number is given to the single copy of the subtype in the genome.</p></div></div><div content-id="tf1-7"><div class="fn"><p><sup>g</sup>The ordinal number of the Nf at the corresponding locus in <em>Nme</em>A (see locus column) is given.</p></div></div><div content-id="tf1-8"><div class="fn"><p><sup>h</sup>‘8’ stands for containing <em>irg8</em>.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87193790" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=87193791 id="87193791" class="section-title" >2.2. Bioinformatic analysis</h3>
<p>The searches for macroscopic genome rearrangements and conserved gene clusters were performed by CGAT<a reveal-id="b22" data-open="b22" class="link link-ref link-reveal xref-bibr">22</a> and MBGD<a reveal-id="b23" data-open="b23" class="link link-ref link-reveal xref-bibr">23</a> (<a class="link link-uri" href="http://mbgd.genome.ad.jp/" target="_blank"></a>), respectively. CGAT was used to detect nucleotide sequence homology of a locus with the other parts of the genome. MBGD was used to detect homologous open reading frames (ORFs) and their neighbouring genes.</p><p>A homology search was performed by BLAST<a reveal-id="b24" data-open="b24" class="link link-ref link-reveal xref-bibr">24</a> and fasta2 package.<a reveal-id="b25" data-open="b25" class="link link-ref link-reveal xref-bibr">25</a> Multiple sequence alignments were constructed by ClustalW.<a reveal-id="b26" data-open="b26" class="link link-ref link-reveal xref-bibr">26</a> Default parameters were used unless otherwise specified. A phylogenetic tree was constructed using ClustalW, NJplot<a reveal-id="b27" data-open="b27" class="link link-ref link-reveal xref-bibr">27</a> and TreeViewPPC (<a class="link link-uri" href="http://taxonomy.zoology.gla.ac.uk/rod/treeview.html" target="_blank"></a>).</p>                            <h2 scrollto-destination=87193794 id="87193794" class="section-title" >3. Results and Discussion</h2>
                            <h3 scrollto-destination=87193795 id="87193795" class="section-title" >3.1. Filamentous prophages in the genomes of Neisseria</h3>
                            <h4 scrollto-destination=87193796 id="87193796" class="section-title" >3.1.1. Nomenclature and terminology of Nf prophages</h4>
<p>In <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>, we have listed all Nf prophages that were found in the four <em>Neisseria</em> genomes.</p><p>First, we describe the nomenclature of Nf prophages. Nf4-G1(del) may be used as an example of our nomenclature of Nf prophages. Here (i) ‘Nf’ stands for <em>Neisseria</em> filamentous phage, (ii) ‘4’ stands for subtype (subtype 1–4, as classified in Section 3.1.3), (iii) ‘G’ stands for strain (A for <em>Nme</em>A, B for <em>Nme</em>B, C for <em>Nme</em>C and G for <em>Ngo</em>), (iv) ‘1’ stands for ordinal number within a subtype that is found within the same genome (a smaller number was given to the copy with smaller genome coordinates; for exceptions, see footnotes f–h of <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>), and (v) ‘(del)’ stands for internal deletion. Partially inverted, truncated and internally deleted copies are designated as ‘(inv)’, ‘(tr)’ and ‘(del)’, respectively. The putative intact copies are not given such a description.</p><p>We followed the following terminology. ‘A copy of Nf’ means a stretch of DNA sequence that encodes at least one gene that is homologous to the genes of ‘consensus, intact’ Nfs. By ‘the left/right of Nfs’, we mean the left/right of Nfs, as shown in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>. ‘Intact’ copies are those that encode a potentially full set of homologues of phage genes that are oriented in the same direction and carry the expected left and right junction sequences (see Section 3.2 for a determination of the junctions). An only ‘partially inverted’ copy [Nf4-G2(inv)] resembles the intact copies, except for partial inversion of its <em>irg</em> gene (<em>irg2</em>) region. ‘Truncated’ copies of Nfs are the copies with truncation of the right end or the left end. Three of the <em>irg</em> copies (<em>irg1, 4</em> and <em>8</em>) are not neighboured by the homologues of the above conserved genes.<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> However, three regions containing these <em>irg</em> genes have the expected left and right junctions (see Section 3.3.2), so we regarded them as ‘internally deleted’ copies of Nfs and named them Nf4-G1(del), Nf4-G4(del) and Nf4-G8(del) (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>).</p>                        <a id="87193800" scrollto-destination="87193800"></a>
<div id="fig1" data-id="fig1" class="fig fig-section"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/m_dsi021f1.jpeg?Expires=2147483647&Signature=it64LtBNRRsEVJ7t0axhccBS366ClHgMK26PLVMGCM8QYwbRjzfLKYcqeyoMFD6ORpcKcy9nFSO1AZ~ifKon-k6JE3HlwRFzbB5j116VJeivQIsqEANUo2FBjK-JkzvnsZ76uU7lgvAjxgzTf77YddVUJBn49LwsjB0Xa2TmZZntqzvBNh69IqcgvThKBBMsJp-OjZWzKZxI1uWKGUwJWqak13hwLnfzoyAVT8KBqYKfWcZMq9fEJwbOXFIYxrqMlqS91UaYxvDkcDlh93os~Y0msmPygKW0pbPNel2NCWapO3mNBB~8nikJJAT5tHkNoMC961JWEzeL4Zg4zPlBog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Nf filamentous prophages on Neisseria chromosome, other filamentous bacteriophages and related plasmids. In the box are shown keys to gene families (the corresponding gene of M13 in parentheses) and DNA features (upper case letters, CT, indicate direct repeats flanking Nfs). Gene names are written above the arrows for colored genes. The alternative names for pivNM/irg3,9,10 are shown in parentheses. Note that members listed in (B) and (C) have been shown to be present in circular DNA form. (A) Nf prophages located at different loci in Neisseria genomes. The left half brackets indicate subtypes. Shown are Nfs located at different loci (Table 1) on four strains—NmeA Z2491, NmeB MC58, NmeC FAM18 and Ngo FA1090. Genes of Nf1-C1, C2, C3 and C4 are annotated by GeneMark.hmm for Prokaryotes.21 Nf2-B1(tr) and Nf2-C1(tr) are not shown because they are located at the same locus and are quite similar to Nf2-A1(tr), as are Nf2-B2(tr) and Nf2-C2(tr) with Nf2-A2(tr) (Table 1). Nomenclature of Nf is described in Section 3.1.1 and the footnote for Table 1. Supplementary data is available at  for details about colored genes and about the cg stem–loop region, of which the consensus sequence is 5′-cccccctnnnctaayaggggggg. (B) Filamentous bacteriophages. M134 is shown with functional modules, along with four other representative filamentous phages.17–20 Grouping of genes by color is based on previous literature18–20,37–41 and/or on our following bioinformatic analysis: Pf1p05 of Pf1 and p06 of VGJφ as gIII and Pf3_4 of Pf3 and p07 of VGJφ as gVI (see Supplementary data are available at ), Pf3_7 as gII (homologous with replication protein of pVT736-1(AAC37125)). Phages listed in (B) are aligned with the first base of the replication initiator gene as the left end. (C) Plasmids that have genes homologous to those of Nf. pJS-B13 from Neisseria meningitidis ET37, pMU115 from Eikenella corrodens and pJTPS114 from Ralstonia solanacearum. pJS-B is aligned with the 1501st bp of the original sequence as the left end. pMU1 is aligned with tatat in the presumed junction atCTtatat as the left end. pJTPS1 is aligned with the first base pair of the replication gene as the left end." path-from-xml="dsi021f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193800/dsi021f1.jpeg" path-from-xml="dsi021f1.jpeg" target="_blank">View large</a><a section="87193800" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/dsi021f1.jpeg?Expires=2147483647&Signature=PP~G7k~cJSzOzDYufQ4918IyMSDGsULcBBTFJeywzOlMss1w~9LYVLwMpp5CoxXB5UoQv68u-0oQJlIKxVLLda5wji33sI8U0Q1f8iztjwK2lDY0FpPQtU9lY8rcVQqux1eEy0-ZN-OD8UB8ZWymGgItb2oACaWE2RoDeiLCrovSwkkQWQd3Vx7LkT~RyNA6mWuNmFY2xPFaNpY-mKRpII5R02pCpNeMWLskUt7bDTDxQ-yosISrat4Kr1BgzYDhoAi93NFjHL7GQxyxcHLdChAbWu59Yo5bH-FcuBQQKj6hKX8yI1Vt-uKhFxmUilqHYfiYZ5fIhG9O~pLUOBBD5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193800&ar=344376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi021f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Nf filamentous prophages on <em>Neisseria</em> chromosome, other filamentous bacteriophages and related plasmids. In the box are shown keys to gene families (the corresponding gene of M13 in parentheses) and DNA features (upper case letters, <span class="monospace">CT</span>, indicate direct repeats flanking Nfs). Gene names are written above the arrows for colored genes. The alternative names for <em>pivNM</em>/<em>irg</em><a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> are shown in parentheses. Note that members listed in (B) and (C) have been shown to be present in circular DNA form. (A) Nf prophages located at different loci in <em>Neisseria</em> genomes. The left half brackets indicate subtypes. Shown are Nfs located at different loci (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>) on four strains—<em>Nme</em>A Z2491, <em>Nme</em>B MC58, <em>Nme</em>C FAM18 and <em>Ngo</em> FA1090. Genes of Nf1-C1, C2, C3 and C4 are annotated by GeneMark.hmm for Prokaryotes.<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> Nf2-B1(tr) and Nf2-C1(tr) are not shown because they are located at the same locus and are quite similar to Nf2-A1(tr), as are Nf2-B2(tr) and Nf2-C2(tr) with Nf2-A2(tr) (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). Nomenclature of Nf is described in Section 3.1.1 and the footnote for <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>. Supplementary data is available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a> for details about colored genes and about the cg stem–loop region, of which the consensus sequence is 5′-<span class="monospace">cccccctnnnctaayaggggggg</span>. (B) Filamentous bacteriophages. M13<sup>4</sup> is shown with functional modules, along with four other representative filamentous phages.<a reveal-id="b17  b20" data-open="b17  b20" class="link link-ref link-reveal xref-bibr">17–20</a> Grouping of genes by color is based on previous literature<a reveal-id="b18  b20" data-open="b18  b20" class="link link-ref link-reveal xref-bibr">18–20</a>,<a reveal-id="b37  b41" data-open="b37  b41" class="link link-ref link-reveal xref-bibr">37–41</a> and/or on our following bioinformatic analysis: Pf1p05 of Pf1 and p06 of VGJφ as <em>gIII</em> and Pf3_4 of Pf3 and p07 of VGJφ as <em>gVI</em> (see Supplementary data are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>), Pf3_7 as <em>gII</em> (homologous with replication protein of pVT736-1(AAC37125)). Phages listed in (B) are aligned with the first base of the replication initiator gene as the left end. (C) Plasmids that have genes homologous to those of Nf. pJS-B<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> from <em>Neisseria meningitidis</em> ET37, pMU1<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> from <em>Eikenella corrodens</em> and pJTPS1<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> from <em>Ralstonia solanacearum</em>. pJS-B is aligned with the 1501st bp of the original sequence as the left end. pMU1 is aligned with <span class="monospace"><span class="underline">t</span>atat</span> in the presumed junction <span class="monospace">atCTtatat</span> as the left end. pJTPS1 is aligned with the first base pair of the replication gene as the left end.</p></div></div><div content-id="fig1" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 1</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/m_dsi021f1.jpeg?Expires=2147483647&Signature=it64LtBNRRsEVJ7t0axhccBS366ClHgMK26PLVMGCM8QYwbRjzfLKYcqeyoMFD6ORpcKcy9nFSO1AZ~ifKon-k6JE3HlwRFzbB5j116VJeivQIsqEANUo2FBjK-JkzvnsZ76uU7lgvAjxgzTf77YddVUJBn49LwsjB0Xa2TmZZntqzvBNh69IqcgvThKBBMsJp-OjZWzKZxI1uWKGUwJWqak13hwLnfzoyAVT8KBqYKfWcZMq9fEJwbOXFIYxrqMlqS91UaYxvDkcDlh93os~Y0msmPygKW0pbPNel2NCWapO3mNBB~8nikJJAT5tHkNoMC961JWEzeL4Zg4zPlBog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Nf filamentous prophages on Neisseria chromosome, other filamentous bacteriophages and related plasmids. In the box are shown keys to gene families (the corresponding gene of M13 in parentheses) and DNA features (upper case letters, CT, indicate direct repeats flanking Nfs). Gene names are written above the arrows for colored genes. The alternative names for pivNM/irg3,9,10 are shown in parentheses. Note that members listed in (B) and (C) have been shown to be present in circular DNA form. (A) Nf prophages located at different loci in Neisseria genomes. The left half brackets indicate subtypes. Shown are Nfs located at different loci (Table 1) on four strains—NmeA Z2491, NmeB MC58, NmeC FAM18 and Ngo FA1090. Genes of Nf1-C1, C2, C3 and C4 are annotated by GeneMark.hmm for Prokaryotes.21 Nf2-B1(tr) and Nf2-C1(tr) are not shown because they are located at the same locus and are quite similar to Nf2-A1(tr), as are Nf2-B2(tr) and Nf2-C2(tr) with Nf2-A2(tr) (Table 1). Nomenclature of Nf is described in Section 3.1.1 and the footnote for Table 1. Supplementary data is available at  for details about colored genes and about the cg stem–loop region, of which the consensus sequence is 5′-cccccctnnnctaayaggggggg. (B) Filamentous bacteriophages. M134 is shown with functional modules, along with four other representative filamentous phages.17–20 Grouping of genes by color is based on previous literature18–20,37–41 and/or on our following bioinformatic analysis: Pf1p05 of Pf1 and p06 of VGJφ as gIII and Pf3_4 of Pf3 and p07 of VGJφ as gVI (see Supplementary data are available at ), Pf3_7 as gII (homologous with replication protein of pVT736-1(AAC37125)). Phages listed in (B) are aligned with the first base of the replication initiator gene as the left end. (C) Plasmids that have genes homologous to those of Nf. pJS-B13 from Neisseria meningitidis ET37, pMU115 from Eikenella corrodens and pJTPS114 from Ralstonia solanacearum. pJS-B is aligned with the 1501st bp of the original sequence as the left end. pMU1 is aligned with tatat in the presumed junction atCTtatat as the left end. pJTPS1 is aligned with the first base pair of the replication gene as the left end." path-from-xml="dsi021f1.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193800/dsi021f1.jpeg" path-from-xml="dsi021f1.jpeg" target="_blank">View large</a><a section="87193800" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/dsi021f1.jpeg?Expires=2147483647&Signature=PP~G7k~cJSzOzDYufQ4918IyMSDGsULcBBTFJeywzOlMss1w~9LYVLwMpp5CoxXB5UoQv68u-0oQJlIKxVLLda5wji33sI8U0Q1f8iztjwK2lDY0FpPQtU9lY8rcVQqux1eEy0-ZN-OD8UB8ZWymGgItb2oACaWE2RoDeiLCrovSwkkQWQd3Vx7LkT~RyNA6mWuNmFY2xPFaNpY-mKRpII5R02pCpNeMWLskUt7bDTDxQ-yosISrat4Kr1BgzYDhoAi93NFjHL7GQxyxcHLdChAbWu59Yo5bH-FcuBQQKj6hKX8yI1Vt-uKhFxmUilqHYfiYZ5fIhG9O~pLUOBBD5g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193800&ar=344376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi021f1.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Nf filamentous prophages on <em>Neisseria</em> chromosome, other filamentous bacteriophages and related plasmids. In the box are shown keys to gene families (the corresponding gene of M13 in parentheses) and DNA features (upper case letters, <span class="monospace">CT</span>, indicate direct repeats flanking Nfs). Gene names are written above the arrows for colored genes. The alternative names for <em>pivNM</em>/<em>irg</em><a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> are shown in parentheses. Note that members listed in (B) and (C) have been shown to be present in circular DNA form. (A) Nf prophages located at different loci in <em>Neisseria</em> genomes. The left half brackets indicate subtypes. Shown are Nfs located at different loci (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>) on four strains—<em>Nme</em>A Z2491, <em>Nme</em>B MC58, <em>Nme</em>C FAM18 and <em>Ngo</em> FA1090. Genes of Nf1-C1, C2, C3 and C4 are annotated by GeneMark.hmm for Prokaryotes.<a reveal-id="b21" data-open="b21" class="link link-ref link-reveal xref-bibr">21</a> Nf2-B1(tr) and Nf2-C1(tr) are not shown because they are located at the same locus and are quite similar to Nf2-A1(tr), as are Nf2-B2(tr) and Nf2-C2(tr) with Nf2-A2(tr) (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). Nomenclature of Nf is described in Section 3.1.1 and the footnote for <a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>. Supplementary data is available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a> for details about colored genes and about the cg stem–loop region, of which the consensus sequence is 5′-<span class="monospace">cccccctnnnctaayaggggggg</span>. (B) Filamentous bacteriophages. M13<sup>4</sup> is shown with functional modules, along with four other representative filamentous phages.<a reveal-id="b17  b20" data-open="b17  b20" class="link link-ref link-reveal xref-bibr">17–20</a> Grouping of genes by color is based on previous literature<a reveal-id="b18  b20" data-open="b18  b20" class="link link-ref link-reveal xref-bibr">18–20</a>,<a reveal-id="b37  b41" data-open="b37  b41" class="link link-ref link-reveal xref-bibr">37–41</a> and/or on our following bioinformatic analysis: Pf1p05 of Pf1 and p06 of VGJφ as <em>gIII</em> and Pf3_4 of Pf3 and p07 of VGJφ as <em>gVI</em> (see Supplementary data are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>), Pf3_7 as <em>gII</em> (homologous with replication protein of pVT736-1(AAC37125)). Phages listed in (B) are aligned with the first base of the replication initiator gene as the left end. (C) Plasmids that have genes homologous to those of Nf. pJS-B<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> from <em>Neisseria meningitidis</em> ET37, pMU1<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> from <em>Eikenella corrodens</em> and pJTPS1<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> from <em>Ralstonia solanacearum</em>. pJS-B is aligned with the 1501st bp of the original sequence as the left end. pMU1 is aligned with <span class="monospace"><span class="underline">t</span>atat</span> in the presumed junction <span class="monospace">atCTtatat</span> as the left end. pJTPS1 is aligned with the first base pair of the replication gene as the left end.</p></div></div><p>We used Courier font for nucleotide and amino acid sequences for clarity.</p>                            <h4 scrollto-destination=87193802 id="87193802" class="section-title" >3.1.2. Nf prophages in four Neisseria genomes</h4>
<p>A large chromosomal rearrangement is seen as a comparison between two genomes of <em>N. meningitidis, Nme</em>A and <em>Nme</em>B.<a reveal-id="b3" data-open="b3" class="link link-ref link-reveal xref-bibr">3</a> During examination of this rearrangement, we encountered homologues of filamentous phages. We also found several homologous prophages on the other loci. We named these prophages Nfs (for <em>Neisserial</em> filamentous phages).</p><p>By investigating four sequenced <em>Neisseria</em> genomes (<em>Nme</em>A, <em>Nme</em>B, <em>Nme</em>C and <em>Ngo</em>), we identified 23 copies of Nf, including 11 intact, 1 partially inverted, 8 truncated and 3 internally deleted copies (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>). Two right-truncated homologues were located at the corresponding loci in three <em>N. meningitidis</em> genomes but not in the <em>N. gonorrhoeae</em> genome—that is, Nf2-A1(tr), Nf2-B1(tr) and Nf2-C1(tr) are at one corresponding locus and Nf2-A2(tr), Nf2-B2(tr) and Nf2-C2(tr) are at the other locus. So, there are 23 copies of Nf at 19 different loci in the four genomes. Notably, all the 11 intact and the 1 partially inverted copies are located at 12 different loci.</p><p>All the 11 intact copies and the 1 partially inverted copy of Nfs are ∼8 kb in length (<a reveal-id="tbl1" data-open="tbl1" class="link link-reveal link-table xref-fig">Table 1</a>), with gene organization being similar to that of filamentous phages (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a> versus B), except for the presence of a <em>pivNM/irg</em> transposase gene homologue at the right end (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>).</p><p>As well as previously noticed homologues of the <em>rstA</em> protein of filamentous phage CTXφ (for phage DNA replication; corresponding to <em>gII</em> of M13) (NMA1792 and other brown arrows in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>) and <em>gI</em> protein (for assembly) (NMA1799 and other light-blue arrows in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>),<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> we found other <em>Nf</em> genes corresponding to filamentous phage genes (<em>gV</em>, <em>gVIII</em>, <em>gIII</em> and <em>gVI</em>) by <em>in silico</em> research (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>; Supplementary Figure S1 is available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>). In <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>, these functionally corresponding, but not necessarily homologous, genes are shown with the same color.</p><p>Genes that are conserved among Nfs are oriented in the same direction (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>), except for NGO1137 (<em>irg2</em>) of Nf4-G2(inv) (as discussed in Section 3.3.1). We assumed that their sense strand represents their single-strand (plus strand) within the presumed virion.</p><p>Plasmids that have genes that are homologous to those of Nf are also shown (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). pJS-B<a reveal-id="b13" data-open="b13" class="link link-ref link-reveal xref-bibr">13</a> from <em>N. meningitidis</em> ET37, and pJTPS1<a reveal-id="b14" data-open="b14" class="link link-ref link-reveal xref-bibr">14</a> from <em>Ralstonia solanacearum</em>, do not carry a <em>pivNM/irg</em> homologue, whereas pMU1<a reveal-id="b15" data-open="b15" class="link link-ref link-reveal xref-bibr">15</a> from <em>Eikenella corrodens</em> does.</p>                            <h4 scrollto-destination=87193812 id="87193812" class="section-title" >3.1.3. Classification of Nf into four subtypes</h4>
<p>We classified the Nfs into four subtypes, Nf1–Nf4, based on similarity of their three long ORFs: phage DNA replication protein homologues (corresponding to pII; homologous with <em>rstA</em> of CTXφ; brown arrows in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>), adsorption protein homologues (corresponding to pIII; previously annotated as TspB) (yellow arrows in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>) and pI homologues (light-blue arrows in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>) [<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2A</a>, (1)–(3)], and by overall nucleotide sequence similarity (data not shown).</p>                        <a id="87193817" scrollto-destination="87193817"></a>
<div id="fig2" data-id="fig2" class="fig fig-section"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/m_dsi021f2.jpeg?Expires=2147483647&Signature=3l~bQI-ctr18KpKZJUJ5TGTnkOF69LY9BgtCSOBd5yVzvp-WLdRoPn1nVxhaUmsG6~5kCmv-lMusCaNTeYmdbN42f04Ht7V8thBCsoV0O0tYrTZ2wL9drFEwIZxCW4iMEOrryx3YTBjxG54X8GVU93oRxEdnBrn3pnGM6ZDjepxjQK8T2ugikhkAhvm8LX6T2AKHS7Dx6Avwmmh6~FdQ-zQpDn~wSoSDydx2ot6asZG79E3yf~OgTwXS3lSi8L9M~puwhydRIYUggdqP9OL~QlBeg2qgtLQ~olmAZEtK1QVHi1H-hTen~qsymKCNDAPVVoQ6RljH3qhuzxHUrbbQ2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of phylogenetic trees of three Nf proteins and transposase on Nf. (A) (1) pII analogues. (2) pIII analogues. (3) pI homologues. (B) PivNM/Irg transposase. Proteins without a gene name are not annotated in the entry in NCBI's RefSeq. A bar indicates substitution of 0.1 amino acid. Bootstrap values based on 1000 computer-generated trees are indicated for the nodes separating the four subtypes. In (A)(1) and (2), regions corresponding to the shortest gene [NMA1167 for (A)(1) and a gene on Nf1-C1 for (A)(2)] are used." path-from-xml="dsi021f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193817/dsi021f2.jpeg" path-from-xml="dsi021f2.jpeg" target="_blank">View large</a><a section="87193817" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/dsi021f2.jpeg?Expires=2147483647&Signature=CRdhEZfhN2~ap8ez2oaKEF6dB8mMh7teJq-Q8egQDjC7McvoK9cY9TFWek7VTgie4PvRmwzd5cBAHcFtGy2M0epA3ZQzhfRVR-opPCJsD1ESfkPk-s6RgbOeuTX2k-q0VE-8qkHx4sAewnjmSOPF8oezpfqQlO7KPRnvmn-ZxGanNYKS1c-wSLvt0ZL~nNYW6z1nl-OOvQohlIkEHTyWf01D6ilr9JGeTMjxUzzLcY1Qik02-Z1G~sRzkk9odV~dN7ELvtNyO2J0cdeUaItvzM9I4EiKfbE4ujfSGqSC~HhViUPfcNJS-Ep6Q7mZVWRptAHUxVS3gXimWooEEQdDKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193817&ar=344376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi021f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of phylogenetic trees of three Nf proteins and transposase on Nf. (A) (1) pII analogues. (2) pIII analogues. (3) pI homologues. (B) PivNM/Irg transposase. Proteins without a gene name are not annotated in the entry in NCBI's RefSeq. A bar indicates substitution of 0.1 amino acid. Bootstrap values based on 1000 computer-generated trees are indicated for the nodes separating the four subtypes. In (A)(1) and (2), regions corresponding to the shortest gene [NMA1167 for (A)(1) and a gene on Nf1-C1 for (A)(2)] are used.</p></div></div><div content-id="fig2" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 2</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/m_dsi021f2.jpeg?Expires=2147483647&Signature=3l~bQI-ctr18KpKZJUJ5TGTnkOF69LY9BgtCSOBd5yVzvp-WLdRoPn1nVxhaUmsG6~5kCmv-lMusCaNTeYmdbN42f04Ht7V8thBCsoV0O0tYrTZ2wL9drFEwIZxCW4iMEOrryx3YTBjxG54X8GVU93oRxEdnBrn3pnGM6ZDjepxjQK8T2ugikhkAhvm8LX6T2AKHS7Dx6Avwmmh6~FdQ-zQpDn~wSoSDydx2ot6asZG79E3yf~OgTwXS3lSi8L9M~puwhydRIYUggdqP9OL~QlBeg2qgtLQ~olmAZEtK1QVHi1H-hTen~qsymKCNDAPVVoQ6RljH3qhuzxHUrbbQ2A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of phylogenetic trees of three Nf proteins and transposase on Nf. (A) (1) pII analogues. (2) pIII analogues. (3) pI homologues. (B) PivNM/Irg transposase. Proteins without a gene name are not annotated in the entry in NCBI's RefSeq. A bar indicates substitution of 0.1 amino acid. Bootstrap values based on 1000 computer-generated trees are indicated for the nodes separating the four subtypes. In (A)(1) and (2), regions corresponding to the shortest gene [NMA1167 for (A)(1) and a gene on Nf1-C1 for (A)(2)] are used." path-from-xml="dsi021f2.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193817/dsi021f2.jpeg" path-from-xml="dsi021f2.jpeg" target="_blank">View large</a><a section="87193817" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/dsi021f2.jpeg?Expires=2147483647&Signature=CRdhEZfhN2~ap8ez2oaKEF6dB8mMh7teJq-Q8egQDjC7McvoK9cY9TFWek7VTgie4PvRmwzd5cBAHcFtGy2M0epA3ZQzhfRVR-opPCJsD1ESfkPk-s6RgbOeuTX2k-q0VE-8qkHx4sAewnjmSOPF8oezpfqQlO7KPRnvmn-ZxGanNYKS1c-wSLvt0ZL~nNYW6z1nl-OOvQohlIkEHTyWf01D6ilr9JGeTMjxUzzLcY1Qik02-Z1G~sRzkk9odV~dN7ELvtNyO2J0cdeUaItvzM9I4EiKfbE4ujfSGqSC~HhViUPfcNJS-Ep6Q7mZVWRptAHUxVS3gXimWooEEQdDKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193817&ar=344376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi021f2.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of phylogenetic trees of three Nf proteins and transposase on Nf. (A) (1) pII analogues. (2) pIII analogues. (3) pI homologues. (B) PivNM/Irg transposase. Proteins without a gene name are not annotated in the entry in NCBI's RefSeq. A bar indicates substitution of 0.1 amino acid. Bootstrap values based on 1000 computer-generated trees are indicated for the nodes separating the four subtypes. In (A)(1) and (2), regions corresponding to the shortest gene [NMA1167 for (A)(1) and a gene on Nf1-C1 for (A)(2)] are used.</p></div></div><p>The only one exception to a clear classification of sequence similarity was seen for the DNA replication protein homologue of Nf4-G2(inv). The branch of Nf4-G2(inv), which was located differently from those of the other Nf4 copies, can be explained by the assumption that Nf4-G2(inv) might have resulted from recombination between an Nf3-like copy and an Nf4 copy. In fact, the right 5942 bp part of Nf4-G2(inv) is similar to the corresponding part, if any, of the other Nf4 copies (data not shown), whereas its left part contains three regions: the left junction (see Section 3.2.2), the phage DNA replication protein (NGO1146) gene [<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2A</a>, (1)] and the cg stem–loop (Supplementary Figure S1E is available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank"></a>), all of which are highly similar to those of Nf3-A but not to those of other Nf4s.</p>                            <h3 scrollto-destination=87193820 id="87193820" class="section-title" >3.2. Junctions of integrated Nf prophages and the host genome</h3>
<p>Except for the right junction of Nf2 and Nf3, all the junctions were determined by alignment of copies of the same subtype, as detailed below (Sections 3.2.1–3.2.5). To determine the right junctions of Nf2 and Nf3, we took advantage of the formation of a promoter-like sequence by joining the right and the left ends of Nfs, as described below (Section 3.2.6).</p>                            <h4 scrollto-destination=87193824 id="87193824" class="section-title" >3.2.1. Sequences flanking Nfs</h4>
<p>The dRS3 repeats constitute one family of repeated sequences in <em>Neisseria</em> and have a consensus sequence of <span class="monospace">attcccnnnnnnnngggaat</span>.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> Alignment of Nf1 copies (as shown in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>) can be interpreted as Nf1s being able to target 5′-<span class="monospace">CT</span> in specific members of the dRS3 repeat family, <span class="monospace">attccc(g/a)cCT(g/a)cgcgg(g/a)aa(t/g)</span>, and were integrated forming flanking direct repeats of 5′-<span class="monospace">CT</span>. The consensus sequence that was targeted by Nf1 (5′-<span class="monospace">attcccgcCTgcgcgggaat</span>) is the most abundant sequence of the dRS3 repeats (291 out of 672 copies in <em>Nme</em>A).</p>                        <a id="87193828" scrollto-destination="87193828"></a>
<div id="fig3" data-id="fig3" class="fig fig-section"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/m_dsi021f3.jpeg?Expires=2147483647&Signature=gQl-uJbDBuT0lxhL3pwVwf~I1HJFrYdDDjYaNe04dLTRWP9ZlvclKtceN7u4NLUhkFA575e-YUvt4FyVfhfQy9AyF-LOdQUXzAZ57FO4d82vxsSu2DpkYrPlkISU-bdIxDUVasFtQVj1rLo7lgfFpbJtuTMCAz~eMOS7w~~syMprEaRs2iI~Nn5XUoeuir-gsRnt-WqFEdfCOblvd324oGu8fPhmezrsxeKH1M8OfofalCDVj2ffAzOj3jlf68rcO1XLJq1RdQL61hrpp4JS621N6~kvPKXDXBgbEs0zVHxAKF3xY9jr3yJmu7iLkR~TZXJyXFwxxPaUOgxEl9WGcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA features of Nf prophages. In (D) and (E), as in Fig. 1, a triangle, a filled circle and an inverted triangle indicate three conserved motifs of Nf, the left junction, the right junction and a 5 bp sequence (ataAG) that is complementary to the left junction. A black arrow indicates a pivNM/irg homologue. (A) Left and right junctions of Nf1. The dRS3 sequences constitute one family of repeated sequences in Neisseria.8 Two base pair direct repeats of ‘CT’ are boxed with solid line. The two dotted boxes along with two boxed 2 bp direct repeats indicate a dRS3 sequence split as a result of insertion of Nf1. (B) Left junction of intact, truncated or partially inverted Nfs. (1) Nf2 and Nf4. (2) Nf3-A and Nf4-G2(inv). (3) Sequence common to all the Nf copies. The common left junction sequence is boxed. (C) Right junction. (1) Nf4. (2) Sequence common to Nf1 and Nf4. The common right junction sequence is boxed. (D) Diagrams of Nf and pMU1. (1) Nf present on the chromosome. (2) The predicted circular form of Nf. (3) pMU1. (E) Promoter-like sequence predicted to form by circularization of Nf1 and Nf4. The numbers indicate the length between the stop codon of pivNM/irg genes and ‘a’ of atCT. Both the junctions of Nf1, Nf4 and IS621, and the homologous locus of pMU1 are shown. The ends of IS621 are shown in the left. Boxes in thick line and boxes in dotted line indicate sequences resembling the −35 and −10 sequences, respectively, of typical eubacterial promoters. The lengths between the −35-like sequence and the −10-like sequence of the predicted circular form of Nfs are indicated below the alignment. In the last line, a typical eubacterial promoter sequence recognized by the primary σ factors28,29 is illustrated. (F) Predicted right junction of Nf2-B3 and Nf3-A. Candidate sequences for flanking atCT of Nf2-B3 and Nf3-A are aligned and compared with the −35 sequence." path-from-xml="dsi021f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193828/dsi021f3.jpeg" path-from-xml="dsi021f3.jpeg" target="_blank">View large</a><a section="87193828" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/dsi021f3.jpeg?Expires=2147483647&Signature=rI8RmaUhAdT4C0Xf-N6~0a~mcOj5Tb-IYBZANCju84CxO1EftkBz6~iUlwMv9gUL-ESJhFtWyoJryX8UWIRmRs6aLPQvS2lvDnA7Sv1OIq1GRajiAg~asGJGD27c9YHiFzQhO3u1Pp1V7-U2gCjxfvtkt7WjbSbhWK1pzM35XGCGUL-kLzKXeoCOHLpJtKMfSdSJSBEtyHHiILCU6VpVybFCiqCuY-6NoACY71GyppfWkIJvbYhtV3x~S1c08JVmVCa3CWYZ-tzJU0oE6zlzHbvXzowxvWMix1IGt5huVIMWQKaj0983bHHVMSkFso7RdE79jCL9QAtXxpyLcN3RPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193828&ar=344376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi021f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA features of Nf prophages. In (D) and (E), as in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>, a triangle, a filled circle and an inverted triangle indicate three conserved motifs of Nf, the left junction, the right junction and a 5 bp sequence (<span class="monospace">ataAG</span>) that is complementary to the left junction. A black arrow indicates a <em>pivNM/irg</em> homologue. (A) Left and right junctions of Nf1. The dRS3 sequences constitute one family of repeated sequences in <em>Neisseria</em>.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> Two base pair direct repeats of ‘<span class="monospace">CT</span>’ are boxed with solid line. The two dotted boxes along with two boxed 2 bp direct repeats indicate a dRS3 sequence split as a result of insertion of Nf1. (B) Left junction of intact, truncated or partially inverted Nfs. (1) Nf2 and Nf4. (2) Nf3-A and Nf4-G2(inv). (3) Sequence common to all the Nf copies. The common left junction sequence is boxed. (C) Right junction. (1) Nf4. (2) Sequence common to Nf1 and Nf4. The common right junction sequence is boxed. (D) Diagrams of Nf and pMU1. (1) Nf present on the chromosome. (2) The predicted circular form of Nf. (3) pMU1. (E) Promoter-like sequence predicted to form by circularization of Nf1 and Nf4. The numbers indicate the length between the stop codon of <em>pivNM/irg</em> genes and ‘<span class="monospace">a</span>’ of <span class="monospace"><span class="underline">a</span>tCT</span>. Both the junctions of Nf1, Nf4 and IS<em>621</em>, and the homologous locus of pMU1 are shown. The ends of IS<em>621</em> are shown in the left. Boxes in thick line and boxes in dotted line indicate sequences resembling the −35 and −10 sequences, respectively, of typical eubacterial promoters. The lengths between the −35-like sequence and the −10-like sequence of the predicted circular form of Nfs are indicated below the alignment. In the last line, a typical eubacterial promoter sequence recognized by the primary σ factors<a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> is illustrated. (F) Predicted right junction of Nf2-B3 and Nf3-A. Candidate sequences for flanking <span class="monospace">atCT</span> of Nf2-B3 and Nf3-A are aligned and compared with the −35 sequence.</p></div></div><div content-id="fig3" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 3</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/m_dsi021f3.jpeg?Expires=2147483647&Signature=gQl-uJbDBuT0lxhL3pwVwf~I1HJFrYdDDjYaNe04dLTRWP9ZlvclKtceN7u4NLUhkFA575e-YUvt4FyVfhfQy9AyF-LOdQUXzAZ57FO4d82vxsSu2DpkYrPlkISU-bdIxDUVasFtQVj1rLo7lgfFpbJtuTMCAz~eMOS7w~~syMprEaRs2iI~Nn5XUoeuir-gsRnt-WqFEdfCOblvd324oGu8fPhmezrsxeKH1M8OfofalCDVj2ffAzOj3jlf68rcO1XLJq1RdQL61hrpp4JS621N6~kvPKXDXBgbEs0zVHxAKF3xY9jr3yJmu7iLkR~TZXJyXFwxxPaUOgxEl9WGcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="DNA features of Nf prophages. In (D) and (E), as in Fig. 1, a triangle, a filled circle and an inverted triangle indicate three conserved motifs of Nf, the left junction, the right junction and a 5 bp sequence (ataAG) that is complementary to the left junction. A black arrow indicates a pivNM/irg homologue. (A) Left and right junctions of Nf1. The dRS3 sequences constitute one family of repeated sequences in Neisseria.8 Two base pair direct repeats of ‘CT’ are boxed with solid line. The two dotted boxes along with two boxed 2 bp direct repeats indicate a dRS3 sequence split as a result of insertion of Nf1. (B) Left junction of intact, truncated or partially inverted Nfs. (1) Nf2 and Nf4. (2) Nf3-A and Nf4-G2(inv). (3) Sequence common to all the Nf copies. The common left junction sequence is boxed. (C) Right junction. (1) Nf4. (2) Sequence common to Nf1 and Nf4. The common right junction sequence is boxed. (D) Diagrams of Nf and pMU1. (1) Nf present on the chromosome. (2) The predicted circular form of Nf. (3) pMU1. (E) Promoter-like sequence predicted to form by circularization of Nf1 and Nf4. The numbers indicate the length between the stop codon of pivNM/irg genes and ‘a’ of atCT. Both the junctions of Nf1, Nf4 and IS621, and the homologous locus of pMU1 are shown. The ends of IS621 are shown in the left. Boxes in thick line and boxes in dotted line indicate sequences resembling the −35 and −10 sequences, respectively, of typical eubacterial promoters. The lengths between the −35-like sequence and the −10-like sequence of the predicted circular form of Nfs are indicated below the alignment. In the last line, a typical eubacterial promoter sequence recognized by the primary σ factors28,29 is illustrated. (F) Predicted right junction of Nf2-B3 and Nf3-A. Candidate sequences for flanking atCT of Nf2-B3 and Nf3-A are aligned and compared with the −35 sequence." path-from-xml="dsi021f3.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193828/dsi021f3.jpeg" path-from-xml="dsi021f3.jpeg" target="_blank">View large</a><a section="87193828" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/dsi021f3.jpeg?Expires=2147483647&Signature=rI8RmaUhAdT4C0Xf-N6~0a~mcOj5Tb-IYBZANCju84CxO1EftkBz6~iUlwMv9gUL-ESJhFtWyoJryX8UWIRmRs6aLPQvS2lvDnA7Sv1OIq1GRajiAg~asGJGD27c9YHiFzQhO3u1Pp1V7-U2gCjxfvtkt7WjbSbhWK1pzM35XGCGUL-kLzKXeoCOHLpJtKMfSdSJSBEtyHHiILCU6VpVybFCiqCuY-6NoACY71GyppfWkIJvbYhtV3x~S1c08JVmVCa3CWYZ-tzJU0oE6zlzHbvXzowxvWMix1IGt5huVIMWQKaj0983bHHVMSkFso7RdE79jCL9QAtXxpyLcN3RPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193828&ar=344376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi021f3.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>DNA features of Nf prophages. In (D) and (E), as in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1</a>, a triangle, a filled circle and an inverted triangle indicate three conserved motifs of Nf, the left junction, the right junction and a 5 bp sequence (<span class="monospace">ataAG</span>) that is complementary to the left junction. A black arrow indicates a <em>pivNM/irg</em> homologue. (A) Left and right junctions of Nf1. The dRS3 sequences constitute one family of repeated sequences in <em>Neisseria</em>.<a reveal-id="b8" data-open="b8" class="link link-ref link-reveal xref-bibr">8</a> Two base pair direct repeats of ‘<span class="monospace">CT</span>’ are boxed with solid line. The two dotted boxes along with two boxed 2 bp direct repeats indicate a dRS3 sequence split as a result of insertion of Nf1. (B) Left junction of intact, truncated or partially inverted Nfs. (1) Nf2 and Nf4. (2) Nf3-A and Nf4-G2(inv). (3) Sequence common to all the Nf copies. The common left junction sequence is boxed. (C) Right junction. (1) Nf4. (2) Sequence common to Nf1 and Nf4. The common right junction sequence is boxed. (D) Diagrams of Nf and pMU1. (1) Nf present on the chromosome. (2) The predicted circular form of Nf. (3) pMU1. (E) Promoter-like sequence predicted to form by circularization of Nf1 and Nf4. The numbers indicate the length between the stop codon of <em>pivNM/irg</em> genes and ‘<span class="monospace">a</span>’ of <span class="monospace"><span class="underline">a</span>tCT</span>. Both the junctions of Nf1, Nf4 and IS<em>621</em>, and the homologous locus of pMU1 are shown. The ends of IS<em>621</em> are shown in the left. Boxes in thick line and boxes in dotted line indicate sequences resembling the −35 and −10 sequences, respectively, of typical eubacterial promoters. The lengths between the −35-like sequence and the −10-like sequence of the predicted circular form of Nfs are indicated below the alignment. In the last line, a typical eubacterial promoter sequence recognized by the primary σ factors<a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> is illustrated. (F) Predicted right junction of Nf2-B3 and Nf3-A. Candidate sequences for flanking <span class="monospace">atCT</span> of Nf2-B3 and Nf3-A are aligned and compared with the −35 sequence.</p></div></div><p>For the other three subtypes, we could not find any obvious similarity or pattern in the flanking sequences, although some sequences were observed several times—that is, <span class="monospace">aaac</span> flanks the left of two Nf2 copies and two Nf4 copies and <span class="monospace">cgtca</span> flanks the right of four Nf4 copies (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B and C</a>).</p>                            <h4 scrollto-destination=87193831 id="87193831" class="section-title" >3.2.2. Left junction</h4>
<p>From a comparison of the left junction of three groups—Nf1 (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>), Nf2 and Nf4 [<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B</a>, (1)], and Nf3-A and Nf4-G2(inv) [<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B</a>, (2)]—we concluded that the sequence 5′-<span class="monospace">CTtatat</span> represents the left junction sequence for all of the Nfs examined [<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B</a>, (3)].</p>                            <h4 scrollto-destination=87193834 id="87193834" class="section-title" >3.2.3. Right junction of Nf1 and Nf4</h4>
<p>From a comparison of the right junction sequences of Nf1 (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3A</a>) and Nf4 [<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3C</a>, (1)], we found that the right junction sequence that is common to Nf1 and Nf4 is 5′-<span class="monospace">atCT</span> [<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3C</a>, (2)]. The ‘<span class="monospace">T</span>’ of the <span class="monospace">atC<span class="underline">T</span></span> of Nf4 is the residue that is reported to be at the 3′ end of aligned downstream regions of <em>irg</em> genes.<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a></p>                            <h4 scrollto-destination=87193837 id="87193837" class="section-title" >3.2.4. Predicted circular form of Nf1 and Nf4</h4>
<p>As filamentous phages have a circular phage genome,<a reveal-id="b4" data-open="b4" class="link link-ref link-reveal xref-bibr">4</a> it is reasonable to assume that Nf1 and Nf4 would have a circular phage genome before integration. From the comparison of the common left and right junctions of Nf1 and Nf4 [<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3D</a>, (1)], the sequence at the junction of the hypothetical circular form is predicted to be 5′-<span class="monospace">atCTtatat</span> [<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3D</a>, (2)].</p><p>Notably, we found that pMU1, which has the homologous gene set with Nf (see Section 3.1.2 and <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1C</a>), has this predicted junction sequence (5′-<span class="monospace">atCTtatat</span>) at the corresponding locus—that is, between the <em>pivNM/irg</em> homologue and the replication protein homologue [at position 3757–3749 bp; <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3D</a>, (3)].</p>                            <h4 scrollto-destination=87193841 id="87193841" class="section-title" >3.2.5. Assembly of a promoter-like sequence by circularization of Nf1 and Nf4</h4>
<p>The sequence that is shared between the right end of Nf1 and Nf4 (5′-<span class="monospace">tg<span class="underline">ttgncn</span></span>) partially matched with 5′-<span class="monospace">ttgaca</span>, a typical −35 sequence of the −35 element of eubacterial promoters that is recognized by primary σ factors<a reveal-id="b28" data-open="b28" class="link link-ref link-reveal xref-bibr">28</a>,<a reveal-id="b29" data-open="b29" class="link link-ref link-reveal xref-bibr">29</a> (boxed in solid, thick line in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3E</a>). Because the predicted junction of the hypothetical circular form of Nf, <span class="monospace">atCTta<span class="underline">tatnnt</span></span>, contains a −10 promoter-like sequence (boxed with dotted line in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3E</a>), we assumed that the junction region forms a promoter in the circular form.</p><p>The related plasmid pMU1 (see <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1C</a>) also carries this promoter-like sequence (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3E</a>). The <em>pivNM/irg</em> homologue of pMU1 is most similar to the <em>pivNM/irg</em> of Nfs so far.</p><p>The transposase gene of IS<em>621</em> is another close homologue of <em>pivNM/irg</em>.<a reveal-id="b9" data-open="b9" class="link link-ref link-reveal xref-bibr">9</a> The left end of IS<em>621</em> is <span class="monospace">CTtgtat</span> and the right end is <span class="monospace">atCT</span> (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3E</a>), the former being similar to the left end of Nfs and the latter being identical to their right end. IS<em>621</em> would also form a promoter-like sequence after assumed circularization (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3E</a>).</p><p>The promoter-like sequence of the predicted circular form of Nfs and pMU1 could regulate the expression of phage replication gene, although some Nfs code another short ORF between the promoter-like sequence and this gene [e.g. Nf1-B1, Nf1-B2, Nf3-A and Nf4-G2(inv); <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>].</p>                            <h4 scrollto-destination=87193847 id="87193847" class="section-title" >3.2.6. Predicted right junction of Nf2 and Nf3</h4>
<p>We cannot determine the right junction of Nf2 and Nf3 by alignment, as with Nf1 and Nf4, because Nf2-B3 and Nf3-A are the single examples of the Nf2 and Nf3 subtypes, respectively, that have an intact right junction (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>).</p><p>Instead, we assumed that both Nf2-B3 and Nf3-A would have an <span class="monospace">atCT</span> sequence at the right junction, as Nf1 and Nf4 copies. Because the right junction sequence <span class="monospace">atCT</span> of Nf1 and Nf4 is located at 142 and 129 bp, respectively, downstream of the stop codon of <em>pivNM/irg</em> (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3E</a>), we searched the range of 250 bp downstream of the stop codon of <em>pivNM</em> in Nf2-B3 and Nf3-A and found <span class="monospace">atCT</span> at only one site in Nf2-B3 and at two sites in Nf3-A (<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3F</a>). Next, we looked for a potential −35 element upstream of these three candidate sequences and found a −35-like <span class="monospace">ttgann</span> sequence in two of them (boxed with solid line in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3F</a>). From these, we concluded that the right junction of Nf2-B3 and Nf3-A is <span class="monospace">atCT</span> that is located 162 and 145 bp, respectively, downstream of the <em>pivNM</em> gene.</p>                            <h4 scrollto-destination=87193850 id="87193850" class="section-title" >3.2.7. Junction sequences common to four Nf subtypes</h4>
<p>Altogether, we concluded that Nfs are flanked with <span class="monospace">CT</span> at both ends. The common sequence at the left end of Nf is <span class="monospace">CTtatat</span>, and that at the right end is <span class="monospace">atCT</span>, as seen in <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3D</a>, (1).</p>                            <h3 scrollto-destination=87193854 id="87193854" class="section-title" >3.3. Rearranged Nfs</h3>
                            <h4 scrollto-destination=87193855 id="87193855" class="section-title" >3.3.1. A partially inverted copy of Nf [Nf4-G2(inv)]</h4>
<p>The <em>irg2</em> gene of Nf4-G2(inv), a transposase homologue, is inverted relative to the other three genes<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a> and the remaining Nf4 homology (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). From a comparison with the intact copies of Nf4, the recombination sites of this partial inversion are identified as the 5 bp sequence (boxed in alignment of <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). One recombination site, <span class="monospace">CTtat</span>, is a part of the right junction <span class="monospace">CTtatat</span> (an open triangle and underlined in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4A</a>), which is composed of the right direct repeat (<span class="monospace">CT</span>) and the flanking 5 bp host sequence (<span class="monospace">tatat</span>; shown in italics). The other recombination site, <span class="monospace">ataAG</span> (complement is <span class="monospace">CTtat</span>), is located between <em>irg2</em> and the other genes of Nf4-G2(inv) (<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). This sequence is complementary to the 5′ bp sequence of the left common junction sequence of Nf, <span class="monospace">CTtat</span>, and is located 10, 8 and 9 bp upstream of the start codon of <em>pivNM/irg</em> in Nf1, Nf2 and Nf4, respectively. Nf3 has a similar sequence (<span class="monospace">gtaAG</span>) at 10 bp upstream. The <span class="monospace">at<span class="underline">aAG</span></span> sequence partially overlaps with the ribosome binding site <span class="monospace"><span class="underline">aag</span>gr</span>, according to Skaar et al.<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a></p>                        <a id="87193859" scrollto-destination="87193859"></a>
<div id="fig4" data-id="fig4" class="fig fig-section"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/m_dsi021f4.jpeg?Expires=2147483647&Signature=xebzZiH3Kg6fR27osKIQuV58xWmOXMd8blQJOEXNQyAYt-J4ynU8dHPT9~SX40mp362rtBvqd8Tx2KbhvBJQMBTzVTmAIRKtS46toTqi2rGa34FEq~4yWHEcf47kydR1FX53UjktuuFwsIlW0OCuulnKcWjPDe6I~3AIlZ62cXfNBkwCCeO6NV25jlEc3jTwHkAPtm44MBZQn07Bd6ouepImkj~~yLEd1Yypem5qLY8lzBYLuWZiQEQL6kNwUIbZFw0qW1GuXrfOFxAtEsR2j0c6bCvYgX0BtVMtxF~5dAov5wkfa1U86tBtnrHgrFO-U7iw8GUtMrNTaFONYbWH7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Rearranged Nfs. Sequence alignments are shown for the relevant regions. A filled triangle, a filled circle and an inverted filled triangle indicate three conserved motifs of Nf, the left junction, the right junction, and a 5 bp sequence that is complementary to the left junction as in Figs 1 and 3. A black arrow indicates a pivNM/irg homologue. (A) Nf with a partial inversion [Nf4-G2(inv)]. A box in thick line indicates the 5 bp junction sequence of the inversion. The right junction of Nf4-G2(inv), CTtatat (underlined and marked with an open triangle) is composed of the right direct repeat (CT) that is shared by all Nfs and the flanking host sequence (tatat; shown in italics). The dotted underline of the hypothetical parental form indicates the sequence that is identical to the predicted junction sequence of hypothetical circular Nf. (B) Nfs with an internal deletion. (1) Nf4-G4(del). Comparison with the left junction and the ataAG region of Nf4-G3 and Nf4-G5. Inverted arrows indicate inverted repeats. Dotted arrows indicate the homologous regions between the left end of intact Nf4 and the complement of Nf4-G4(del). (2) Nf4-G1(del) and Nf4-G8(del). Boxes in dashed line (ccct) and boxes in dotted line (tgcgt) indicate short repeats that are probably responsible for deletions that generated Nf4-G8(del) and Nf4-G1(del), respectively. (3) Possible steps of formation of Nf4-G4(del). An open circle with a dotted line indicates a hypothetical site that the transposase recognized. The thick arrow indicates a sequence that is involved in the replicative transposition event to form long inverted repeats. Boxes in dashed line with dots indicate short repeats that are probably responsible for deletion." path-from-xml="dsi021f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193859/dsi021f4.jpeg" path-from-xml="dsi021f4.jpeg" target="_blank">View large</a><a section="87193859" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/dsi021f4.jpeg?Expires=2147483647&Signature=U0DwjNkg6uiXid-EFi-ZMTF8Bh~IcChxAnp9ddiRY2bkkZ4n-mfuwY81f1kNtSSUPnFObIVGhzcrCTgET3iMFqy2YGLqAMgJOiTnAcljxJ2UxF6TM9NnfQiPWWIU6lQC0-2iztlPZIPWBMMAA03u8kNdTQsIgSfH-jag4o6hxsln2JQSblRXD-R7CFmDnm2tRm5S3~rvxrGIwKzA6eARdkkvgzNHFHNd2L0Fr1I4nfTNTGi1mGbRfmxheDj0zqTKV8u9TwqcOWLNLdDMiNVCkgoUFeQ963NaA9uCxE0zCLo8ZdAv8exK~6HjdK9l6rFvU7D93RixVyCrSCZJ4rAMBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193859&ar=344376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi021f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Rearranged Nfs. Sequence alignments are shown for the relevant regions. A filled triangle, a filled circle and an inverted filled triangle indicate three conserved motifs of Nf, the left junction, the right junction, and a 5 bp sequence that is complementary to the left junction as in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Figs 1</a> and <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">3</a>. A black arrow indicates a <em>pivNM/irg</em> homologue. (A) Nf with a partial inversion [Nf4-G2(inv)]. A box in thick line indicates the 5 bp junction sequence of the inversion. The right junction of Nf4-G2(inv), <span class="monospace">CTtatat</span> (underlined and marked with an open triangle) is composed of the right direct repeat (<span class="monospace">CT</span>) that is shared by all Nfs and the flanking host sequence (<span class="monospace">tatat</span>; shown in italics). The dotted underline of the hypothetical parental form indicates the sequence that is identical to the predicted junction sequence of hypothetical circular Nf. (B) Nfs with an internal deletion. (1) Nf4-G4(del). Comparison with the left junction and the <span class="monospace">ataAG</span> region of Nf4-G3 and Nf4-G5. Inverted arrows indicate inverted repeats. Dotted arrows indicate the homologous regions between the left end of intact Nf4 and the complement of Nf4-G4(del). (2) Nf4-G1(del) and Nf4-G8(del). Boxes in dashed line (<span class="monospace">ccct</span>) and boxes in dotted line (<span class="monospace">tgcgt</span>) indicate short repeats that are probably responsible for deletions that generated Nf4-G8(del) and Nf4-G1(del), respectively. (3) Possible steps of formation of Nf4-G4(del). An open circle with a dotted line indicates a hypothetical site that the transposase recognized. The thick arrow indicates a sequence that is involved in the replicative transposition event to form long inverted repeats. Boxes in dashed line with dots indicate short repeats that are probably responsible for deletion.</p></div></div><div content-id="fig4" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 4</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/m_dsi021f4.jpeg?Expires=2147483647&Signature=xebzZiH3Kg6fR27osKIQuV58xWmOXMd8blQJOEXNQyAYt-J4ynU8dHPT9~SX40mp362rtBvqd8Tx2KbhvBJQMBTzVTmAIRKtS46toTqi2rGa34FEq~4yWHEcf47kydR1FX53UjktuuFwsIlW0OCuulnKcWjPDe6I~3AIlZ62cXfNBkwCCeO6NV25jlEc3jTwHkAPtm44MBZQn07Bd6ouepImkj~~yLEd1Yypem5qLY8lzBYLuWZiQEQL6kNwUIbZFw0qW1GuXrfOFxAtEsR2j0c6bCvYgX0BtVMtxF~5dAov5wkfa1U86tBtnrHgrFO-U7iw8GUtMrNTaFONYbWH7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Rearranged Nfs. Sequence alignments are shown for the relevant regions. A filled triangle, a filled circle and an inverted filled triangle indicate three conserved motifs of Nf, the left junction, the right junction, and a 5 bp sequence that is complementary to the left junction as in Figs 1 and 3. A black arrow indicates a pivNM/irg homologue. (A) Nf with a partial inversion [Nf4-G2(inv)]. A box in thick line indicates the 5 bp junction sequence of the inversion. The right junction of Nf4-G2(inv), CTtatat (underlined and marked with an open triangle) is composed of the right direct repeat (CT) that is shared by all Nfs and the flanking host sequence (tatat; shown in italics). The dotted underline of the hypothetical parental form indicates the sequence that is identical to the predicted junction sequence of hypothetical circular Nf. (B) Nfs with an internal deletion. (1) Nf4-G4(del). Comparison with the left junction and the ataAG region of Nf4-G3 and Nf4-G5. Inverted arrows indicate inverted repeats. Dotted arrows indicate the homologous regions between the left end of intact Nf4 and the complement of Nf4-G4(del). (2) Nf4-G1(del) and Nf4-G8(del). Boxes in dashed line (ccct) and boxes in dotted line (tgcgt) indicate short repeats that are probably responsible for deletions that generated Nf4-G8(del) and Nf4-G1(del), respectively. (3) Possible steps of formation of Nf4-G4(del). An open circle with a dotted line indicates a hypothetical site that the transposase recognized. The thick arrow indicates a sequence that is involved in the replicative transposition event to form long inverted repeats. Boxes in dashed line with dots indicate short repeats that are probably responsible for deletion." path-from-xml="dsi021f4.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193859/dsi021f4.jpeg" path-from-xml="dsi021f4.jpeg" target="_blank">View large</a><a section="87193859" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/dsi021f4.jpeg?Expires=2147483647&Signature=U0DwjNkg6uiXid-EFi-ZMTF8Bh~IcChxAnp9ddiRY2bkkZ4n-mfuwY81f1kNtSSUPnFObIVGhzcrCTgET3iMFqy2YGLqAMgJOiTnAcljxJ2UxF6TM9NnfQiPWWIU6lQC0-2iztlPZIPWBMMAA03u8kNdTQsIgSfH-jag4o6hxsln2JQSblRXD-R7CFmDnm2tRm5S3~rvxrGIwKzA6eARdkkvgzNHFHNd2L0Fr1I4nfTNTGi1mGbRfmxheDj0zqTKV8u9TwqcOWLNLdDMiNVCkgoUFeQ963NaA9uCxE0zCLo8ZdAv8exK~6HjdK9l6rFvU7D93RixVyCrSCZJ4rAMBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193859&ar=344376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi021f4.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Rearranged Nfs. Sequence alignments are shown for the relevant regions. A filled triangle, a filled circle and an inverted filled triangle indicate three conserved motifs of Nf, the left junction, the right junction, and a 5 bp sequence that is complementary to the left junction as in <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Figs 1</a> and <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">3</a>. A black arrow indicates a <em>pivNM/irg</em> homologue. (A) Nf with a partial inversion [Nf4-G2(inv)]. A box in thick line indicates the 5 bp junction sequence of the inversion. The right junction of Nf4-G2(inv), <span class="monospace">CTtatat</span> (underlined and marked with an open triangle) is composed of the right direct repeat (<span class="monospace">CT</span>) that is shared by all Nfs and the flanking host sequence (<span class="monospace">tatat</span>; shown in italics). The dotted underline of the hypothetical parental form indicates the sequence that is identical to the predicted junction sequence of hypothetical circular Nf. (B) Nfs with an internal deletion. (1) Nf4-G4(del). Comparison with the left junction and the <span class="monospace">ataAG</span> region of Nf4-G3 and Nf4-G5. Inverted arrows indicate inverted repeats. Dotted arrows indicate the homologous regions between the left end of intact Nf4 and the complement of Nf4-G4(del). (2) Nf4-G1(del) and Nf4-G8(del). Boxes in dashed line (<span class="monospace">ccct</span>) and boxes in dotted line (<span class="monospace">tgcgt</span>) indicate short repeats that are probably responsible for deletions that generated Nf4-G8(del) and Nf4-G1(del), respectively. (3) Possible steps of formation of Nf4-G4(del). An open circle with a dotted line indicates a hypothetical site that the transposase recognized. The thick arrow indicates a sequence that is involved in the replicative transposition event to form long inverted repeats. Boxes in dashed line with dots indicate short repeats that are probably responsible for deletion.</p></div></div><p>This inversion structure is perfectly consistent with conservative site-specific recombination involving the 5 bp sequences. The inversion was probably mediated by PivNM/Irg itself on an integrated form of a presumed parental Nf4-G2 (the second line of <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4A</a>) because the right junction (<span class="monospace">atCT</span>) of the parental Nf4-G2 sequence, as well as the flanking host sequence (<span class="monospace">tatat</span>) is the same as the predicted junction of the hypothetical circular form of Nfs, <span class="monospace">atCTtatat</span> (underlined with a dotted line in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4A</a>). Alternatively, the inversion could have occurred on the circular form of an Nf4 member. If so, the partially inverted circular form may have been integrated into the chromosome by Irg2-mediated recombination between the junction sequence <span class="monospace">…ccCTtatatat…</span> [<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4A</a> and <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">3B</a>, (2)] and a chromosomal sequence <span class="monospace">…acCTtatatgc…</span> [<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3B</a>, (2) and <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">4A</a>].</p>                            <h4 scrollto-destination=87193865 id="87193865" class="section-title" >3.3.2. Internally deleted copies of Nf</h4>
<p>The <em>irg1, 4</em> and <em>8</em> genes are the only exceptions of the <em>pivNM/irg</em> homologues that are not neighboured by the homologues of phage genes at the left (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). The region downstream of these <em>irg</em> genes to <span class="monospace">atCT</span>, the right junction of Nfs, is highly homologous to that in the other Nf4 copies [<a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3C</a>, (1)]. At a variable distance upstream from the <em>irg</em> genes, we identified the left junction sequence of Nfs, <span class="monospace">CTtatat</span> [<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4B</a>, (1) and (2)]. We named these regions bound by these junction sequences: Nf4-G1(del), Nf4-G4(del) and Nf4-G8(del).</p><p>Indeed, the left 33 bp sequence of Nf4-G4(del) (solid, rightward arrow) aligned well with the left end of the intact Nf4 copies (Nf4-G3 and Nf4-G5) [<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4B</a>, (1)]. Notably, the complementary strand of the sequence just upstream of <em>irg4</em> aligned longer with the left end of intact Nf4 [dotted arrows in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4B</a>, (1)]. In Nf4-G4(del), the left end of the 33 bp sequence (<span class="monospace">CTtatat…</span>) and the 34 bp sequence upstream of <em>irg</em> [<span class="monospace">…atataAG</span>; solid, leftward arrow in Nf4-G4(del) of <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4B</a>, (1)] form inverted repeats (33/34 bp match). Note that the right junction (<span class="monospace">ataAG</span>; boxed and indicated by an inverted triangle in <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4B</a>, (1)] is the same site as the recombination sequence for partial inversion (see Section 3.3.1 and <a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4A</a>).</p><p>Another putative deletion derivative, Nf4-G8(del), aligned very well with Nf4-G4(del) with a deletion [<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4B</a>, (2)]. Nf4-G1(del) aligned well with these two deletion derivatives, [Nf4-G4(del) and Nf4-G8(del)], and with the intact Nf4s (Nf4-G3 and Nf4-G5) with a deletion [<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4B</a>, (2)].</p><p>Formation of Nf4-G8(del) is readily explained by illegitimate recombination involving short direct repeats, <span class="monospace">ccct</span> (boxed in dashed line), in an Nf4-G4(del)-type parent [<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4B</a>, (2)]. Deletions stimulated by palindromic sequences were shown to occur between short direct repeats in the laboratory.<a reveal-id="b30" data-open="b30" class="link link-ref link-reveal xref-bibr">30</a>,<a reveal-id="b31" data-open="b31" class="link link-ref link-reveal xref-bibr">31</a></p><p>Similarly, Nf4-G1(del) would be formed through illegitimate recombination involving short direct repeats, <span class="monospace">tgcgt</span> (boxed in dotted line), from these types of deletion derivatives [Nf4-G4(del) and Nf4-G8(del)] or from the intact Nf4 copies [<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4B</a>, (2)]. As the parent, we prefer the structure of Nf4-G4(del), or similar, with long inverted repeats, as the inverted repeats will stimulate and select this type of deletion.<a reveal-id="b31" data-open="b31" class="link link-ref link-reveal xref-bibr">31</a></p><p>Formation of Nf4-G4(del) from intact Nf4 copies can be explained by the following replicative transposition model. (i) A copy of a region of the left terminus (longer than 33 bp) is inserted into the left of the <span class="monospace">ataAG</span> sequence upstream of the transposase gene. This is a reaction that is similar to replicative transposition. The transposase may have recognized a secondary site within Nf as in one-ended transposition.<a reveal-id="b32" data-open="b32" class="link link-ref link-reveal xref-bibr">32</a> (ii) The resulting long inverted repeats make the entire region unstable and cause or select an illegitimate recombination event involving a short sequence identity (<span class="monospace">aaggg</span>). This mechanism is the same as we proposed for the other two deletion derivatives. This model can provide a uniform explanation to the formation of the three deletion derivatives.</p><p>Although these three copies are probably deletion derivatives of intact Nfs, the possibility remains that they are integrated into these sites after the deletion event. Unfortunately, we do not know how much of the Nf sequence is necessary <em>in cis</em> for integration or whether these three copies can express a functional transposase. The deletion in Nf4-G1(del) apparently removed the first residue of the start codon of the <em>irg</em> gene [<a reveal-id="fig4" data-open="fig4" class="link link-reveal link-table xref-fig">Fig. 4B</a>, (2)] and resulted in the start codon of NGO0772 (<em>irg1</em>) to be 33 bp upstream from the left end of Nf4-G1(del).</p>                            <h3 scrollto-destination=87193882 id="87193882" class="section-title" >3.4. Simple insertion revealed by comparison with another genome without Nf</h3>
<p>Comparison of Nf1-A integrated into <em>Nme</em>A with the corresponding unoccupied locus of <em>Ngo</em> (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5A</a>) made it clear that the target sequence of Nf1-A integration is <span class="monospace">CT</span>. Unfortunately, with the other intact copies of Nf1, such an obvious assignment was not possible because of the variability of the dRS3 repeats.</p>                        <a id="87193884" scrollto-destination="87193884"></a>
<div id="fig5" data-id="fig5" class="fig fig-section"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/m_dsi021f5.jpeg?Expires=2147483647&Signature=baPOGkhgPHBZZNQcXcuzcfj1cRAkGntcpIMmr6t7dUD7khP6jvqG7zbG2G-dkxGselxcJYcOsnMFvCUpxN5lTmo5P5gxSpHuWjNtO4VoS-gQo0izNpiqwi2Z19C7Asby2BEb5DXSutKMrbH0~WEcIZW9u3ZCIzH5IwR3518NFpFZQNpj~Jt2on12izPcvEFXrPX~2rVB0yfmJt~Ya8mfCOc-iurrEu0SltPf9TQptw9NilyqRHTsTphfRjOdmqk7HZA0Ji9fxSOMRvKItnLqJcOcLF~3opq3Zfb-GhM6p5eG~xT43sJwQZvFx-O66cMiB~3Qh8u8QKJ9BZrWN-HCAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignment with an unoccupied integration site. A box indicates a target CT sequence. (A) Nf1-A. Nf1-A is integrated at the loop of a dRS3 repeat. The consensus of dRS3 is shown below with its inverted repeats indicated by arrows. (B) Nf4-G1(del)." path-from-xml="dsi021f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193884/dsi021f5.jpeg" path-from-xml="dsi021f5.jpeg" target="_blank">View large</a><a section="87193884" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/dsi021f5.jpeg?Expires=2147483647&Signature=S4MSWIg~LD9F5KqchlAm65GqYyxWqLcqLBOtQS0cpf0FmOE3~WdGPYoNrRv~3R5NTeJ7lFgLvvZNRfWL33kjZLJCK4XbiKY276qxZpY7UiPaMqS14qHR8EUZmVwr4obXbgfezwlkhmX-BEu4QOrQEbvXmAgT~iSzOYz8SPGK2c6I~kecfp~Q7MIXo7fONJys0KiIPcvdia0q57-VX8jJLr5M7Cwi~82rDBQ2FI2ISBXU28KW4-PWt18eMme3mJd1GP7t-yD1-qiTgG5zWg9HnG~QSOdQjHZdLR8cPpOIR4f0scO5kIzmVdlfx5~GfMwBhYfvrbwebDMEQdqWCvyHaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193884&ar=344376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi021f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment with an unoccupied integration site. A box indicates a target <span class="monospace">CT</span> sequence. (A) Nf1-A. Nf1-A is integrated at the loop of a dRS3 repeat. The consensus of dRS3 is shown below with its inverted repeats indicated by arrows. (B) Nf4-G1(del).</p></div></div><div content-id="fig5" class="fig fig-modal reveal-modal"><div class="label fig-label"><strong>Figure 5</strong></div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/m_dsi021f5.jpeg?Expires=2147483647&Signature=baPOGkhgPHBZZNQcXcuzcfj1cRAkGntcpIMmr6t7dUD7khP6jvqG7zbG2G-dkxGselxcJYcOsnMFvCUpxN5lTmo5P5gxSpHuWjNtO4VoS-gQo0izNpiqwi2Z19C7Asby2BEb5DXSutKMrbH0~WEcIZW9u3ZCIzH5IwR3518NFpFZQNpj~Jt2on12izPcvEFXrPX~2rVB0yfmJt~Ya8mfCOc-iurrEu0SltPf9TQptw9NilyqRHTsTphfRjOdmqk7HZA0Ji9fxSOMRvKItnLqJcOcLF~3opq3Zfb-GhM6p5eG~xT43sJwQZvFx-O66cMiB~3Qh8u8QKJ9BZrWN-HCAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Alignment with an unoccupied integration site. A box indicates a target CT sequence. (A) Nf1-A. Nf1-A is integrated at the loop of a dRS3 repeat. The consensus of dRS3 is shown below with its inverted repeats indicated by arrows. (B) Nf4-G1(del)." path-from-xml="dsi021f5.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87193884/dsi021f5.jpeg" path-from-xml="dsi021f5.jpeg" target="_blank">View large</a><a section="87193884" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/dsi021f5.jpeg?Expires=2147483647&Signature=S4MSWIg~LD9F5KqchlAm65GqYyxWqLcqLBOtQS0cpf0FmOE3~WdGPYoNrRv~3R5NTeJ7lFgLvvZNRfWL33kjZLJCK4XbiKY276qxZpY7UiPaMqS14qHR8EUZmVwr4obXbgfezwlkhmX-BEu4QOrQEbvXmAgT~iSzOYz8SPGK2c6I~kecfp~Q7MIXo7fONJys0KiIPcvdia0q57-VX8jJLr5M7Cwi~82rDBQ2FI2ISBXU28KW4-PWt18eMme3mJd1GP7t-yD1-qiTgG5zWg9HnG~QSOdQjHZdLR8cPpOIR4f0scO5kIzmVdlfx5~GfMwBhYfvrbwebDMEQdqWCvyHaw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87193884&ar=344376&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsi021f5.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Alignment with an unoccupied integration site. A box indicates a target <span class="monospace">CT</span> sequence. (A) Nf1-A. Nf1-A is integrated at the loop of a dRS3 repeat. The consensus of dRS3 is shown below with its inverted repeats indicated by arrows. (B) Nf4-G1(del).</p></div></div><p>Comparison of one of the internally deleted copies, Nf4-G1(del), in <em>Ngo</em> with the corresponding unoccupied locus in <em>Nme</em>A strongly indicated that it also targeted <span class="monospace">CT</span> (<a reveal-id="fig5" data-open="fig5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). According to the comparison between <em>Ngo</em> and <em>Nme</em>A, the target sequence of Nf4-G8(del) also seems to be <span class="monospace">CT</span>, although the two base pairs that are 3′ to <span class="monospace">CT</span> are different from those in the corresponding unoccupied locus. For the other copies, comparison is not straight forward because of the complex structures that flank them.</p>                            <h3 scrollto-destination=87193890 id="87193890" class="section-title" >3.5. Alternative models for Nf integration</h3>
<p>In some of the above arguments (see Section 3.3), we implicitly assumed that PivNM/Irg transposase that is present on the Nf phage genome is likely to have mediated integration of the phage genome into the chromosome. In this section, we will list the alternative models and provide evidence against them.</p>                            <h4 scrollto-destination=87193894 id="87193894" class="section-title" >3.5.1. Four possible models</h4>
<p>The first model we have been pursuing (A) is that the Nf phage was integrated by its own <em>pivNM/irg</em> transposase (transposase-as-integrase model). Three other models are as follows: (B) Nf without <em>pivNM/irg</em> was integrated by some unspecified mechanism and later the <em>pivNM/irg</em> was inserted as IS (phage-then-IS model); (C) <em>pivNM/irg</em> was inserted into the Nf phage genome and later the phage genome with <em>pivNM/irg</em> was integrated by some unspecified mechanism unrelated to <em>pivNM/irg</em> (phage-with-IS model) and (D) the <em>pivNM/irg</em> was inserted into the chromosome as IS and later, by some unspecified mechanism, Nf without <em>pivNM/irg</em> was integrated between the left junction of the IS and the start codon of <em>pivNM/irg</em> (IS-then-phage model). An idea similar to the phage-with-IS model was presented earlier.<a reveal-id="b10" data-open="b10" class="link link-ref link-reveal xref-bibr">10</a></p>                            <h4 scrollto-destination=87193900 id="87193900" class="section-title" >3.5.2. Evidences for or/and against the models</h4>
<p>Below, the transposase-as-integrase model (model A) is discussed in comparison with the other three models (models B–D). </p><ul class="roman-lower"><li><p>Models B–D provide no explanation as to how the phage was integrated into the chromosome. On the other hand, model A is straightforward by explaining that <em>pivNM/irg</em> transposase acts as the integrase for integration of Nf.</p></li><li><p>Tight linkage of phage-related genes of the Nf and <em>pivNM/irg</em> gene is a straightforward consequence of models A and C, but is not a straightforward consequence of models B and D. The apparently solitary <em>irg</em> genes [Nf4-G1(del), Nf4-G4(del) and Nf4-G8(del)] can be explained by deletion from an intact copy of Nf as detailed above (see Section 3.3.2). This supports model A. These deletion events may have happened after the integration of Nf, although the possibility that the deletion derivatives were integrated as simple IS cannot be excluded (see Section 3.3.2). Thus, we cannot exclude model D from this argument, but it is difficult to explain, by model D, why Nfs without <em>pivNM/irg</em> specifically target the fixed point between the left junction of IS<em>pivNM/irg</em> and the start codon of <em>pivNM/irg</em> and why they are not integrated to the other loci of <em>Neisseria</em> genomes.</p></li><li><p>Similarity of the junction sequences of Nf and IS<em>621</em>, the second closest homologue of <em>pivNM/irg</em> of Nf (see Section 3.2.5), supports the idea that both junctions of Nf were derived from the activity of PivNM/Irg and supports models A and D but not the other two models.</p></li><li><p>The closest homologue of <em>pivNM/irg</em> is the one that is coded on pMU1. pMU1 has the <em>pivNM/irg</em> homologue, along with other homologues of Nf phage genes (see Section 3.1.2 above and <a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). pMU1 encodes <em>pivNM/irg</em> and has an <span class="monospace">atCTtatat</span> sequence, which is exactly the predicted junction sequence for the hypothetical circular form of Nf at the corresponding locus (see Section 3.2.4 above and <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3E</a>). This form resembles an intermediate form of transposition of several IS families, such as the IS<em>3</em> family.<a reveal-id="b33" data-open="b33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="b34" data-open="b34" class="link link-ref link-reveal xref-bibr">34</a> Nf and pMU1, along with IS<em>621</em>, have the assembled promoter-like sequence (see Section 3.2.5 and <a reveal-id="fig3" data-open="fig3" class="link link-reveal link-table xref-fig">Fig. 3</a>). IS<em>492</em> of the same IS<em>110</em>/IS<em>492</em> family as <em>pivNM/irg</em> takes a circular form with the junction consisting of a single copy of the direct repeats and with a promoter-like sequence.<a reveal-id="b35" data-open="b35" class="link link-ref link-reveal xref-bibr">35</a> These points argue for the possibility that Nfs are likely to have a circular form together with <em>pivNM/irg</em> and that the predicted junction sequence ‘<span class="monospace">CT</span>’ is similar to that of pMU1 and support models A and D but not the other two models.</p></li><li><p>Each of the subtypes, Nf1–Nf4, carries a distinct type of <em>pivNM/irg</em> gene (<em>pivNM1/irg7</em> for Nf1, <em>pivNM2</em> for Nf2, <em>pivNM3</em> for Nf3 and <em>irg1-6, 8</em> for Nf4), as revealed by phylogenetic analyses (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2A</a> compared with 2B). This means that the copies of one subtype of Nf are linked with one specific subtype of <em>pivNM/irg</em> and vice versa; this is compatible with models A and C. On the other hand, in models B and D, there is no obvious reason to assume that copies of the same subtype of Nf carry <em>pivNM/irg</em> genes of the same type.</p></li><li><p>Amino acid sequences of phage-related genes of Nf2 and Nf4 are similar [<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2A</a>, (1)–(3)], but those of their transposases, <em>pivNM2</em> and <em>irg</em>, are not similar to each other (<a reveal-id="fig2" data-open="fig2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). Nucleotide sequences of the central regions of Nf2 and Nf4 are also similar but those of <em>pivNM2</em> and <em>irg</em> are diverged (data not shown). This might support models B and D, which indicate that there are different origins for phages and <em>pivNM/irg</em>. However, if we assume that the <em>pivNM/irg</em> of each subtype was somehow inserted independently into Nf (Nf without <em>pivNM/irg</em>), models A and C would be supported. pJS-B and pJTPS1 have similar gene organizations to that of Nf, but do not encode the <em>pivNM/irg</em> homologue nor have an <span class="monospace">atCTtatat</span> junction sequence (<a reveal-id="fig1" data-open="fig1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). Independent insertion of <em>pivNM/irg</em> into the circular form of Nf without <em>pivNM/irg</em> (like pJS-B) might have occurred during evolution. The mechanism of this insertion might be different from that of Nf integration into bacterial chromosomes because Nf integration is marked by flanking <span class="monospace">CT</span> repeats, whereas we could not find such duplication for <em>pivNM/irg</em> insertion into hypothetical Nf without <em>pivNM/irg</em>, although we cannot exclude the possibility that loss of the <span class="monospace">CT</span> sequence was selected.</p></li></ul><p>From the above observations and considerations, we concluded that the most likely mechanism is the transposase-as-integrase model (model A). The circular form of Nf before integration into (and after excision, if any, from) the chromosome would have an <span class="monospace">atCTtatat</span> sequence at the junction and recombine with the target <span class="monospace">CT</span> of the chromosome by the activity of the PivNM/Irg transposase. Although this reaction mechanism is similar to that of site-specific recombinase, Piv family proteins do not have amino acid motifs that are conserved among the tyrosine- or serine-recombinase families,<a reveal-id="b11" data-open="b11" class="link link-ref link-reveal xref-bibr">11</a> but are similar to DDE transposases.<a reveal-id="b36" data-open="b36" class="link link-ref link-reveal xref-bibr">36</a></p><p>This mechanism is distinct from the two currently known mechanisms for filamentous phage integration, integration into a <em>dif</em>-like site mediated by XerC/D site-specific tyrosine-recombinase<a reveal-id="b6" data-open="b6" class="link link-ref link-reveal xref-bibr">6</a> and integration into tRNA probably mediated by an integrase of the tyrosine-recombinase family.<a reveal-id="b7" data-open="b7" class="link link-ref link-reveal xref-bibr">7</a></p>                            <h3 scrollto-destination=87193910 id="87193910" class="section-title" >3.6. Comparison with the work by Bille et al.<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a></h3>
<p>After this work was completed, Bille et al.<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a> published an experimental work of which some results are in agreement with those that we described above. Their results related to ours are listed below [(i)–(vi)], whereas the findings that are specific to our work are also listed [(a)–(e)].</p><p>(i) Discovery of filamentous phages (corresponding to seven intact copies of Nf1, according to our naming) in the genomes of <em>Neisseria</em> based on observations of their low G+C content, homology of the first ORF to bacteriophage replication proteins and the similarity to the order and size of ORFs that are encoded by other filamentous phages. (ii) Those phages (Nf1) are inserted into dRS3. (iii) Extrachromosomal circular single-stranded DNA of one prophage (Nf1-A, according to our naming) was detected by polymerase chain reaction (PCR). (iv) Junction of the circular form was sequenced. (v) The region containing the <em>pivNM</em> gene was contained in the circular form, as judged from the length of PCR products (supporting our result that <em>pivNM/irg</em> genes are integral part of Nfs). (vi) The inactivation of four genes—NMA1792, 1797 and 1799 on Nf1-A and <em>pilQ</em> on a locus of the host chromosome—had predicted effects on extracellular and cytoplasmic circular DNA forms.</p><p>Our findings that are not overlapping with theirs are as follows: (a) we found and characterized not only the Nf1 subtype, but also three other subtypes (Nf2, Nf3 and Nf4). (b) The left and right sequences common to all the four Nf subtypes are <span class="monospace">CTtatat</span> and <span class="monospace">atCT</span>, respectively. (c) Our analyses strongly indicate that the integration of Nf is mediated by its own PivNM/Irg transposase (transposase-as-integrase model). (d) We identified and characterized structural variants of Nfs. We were able to explain their origin in terms of aberrant activity of this transposase. (e) We also added further bioinformatic evidence that ORFs of Nfs correspond to proteins of filamentous phages.</p><p>Note that their work<a reveal-id="b12" data-open="b12" class="link link-ref link-reveal xref-bibr">12</a> further supports the transposase-as-integrase model. Our results proved the power of bioinformatic approach through genome comparison for detailed and thorough analysis of mobile elements and genome rearrangements.</p><p>We thank Noriko Takahashi and Takeshi Tsuru for helpful suggestions on the manuscript. This work was supported by grants from MEXT of the Japanese government to I.K. (Genome Biology, Genome Homeostasis, DNA Repair, Kiban-evolution, Kiban-genome, 21COE: genome language).</p>                            <h2 scrollto-destination=87193922 id="87193922" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="b1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nobusato</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Uchiyama</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Ohashi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Insertion with long target duplication: a mechanism for gene mobility suggested from comparison of two related bacterial genomes</div>, <div class="source">Gene</div> , <div class="year">2000</div>, vol. <div class="volume">259</div> (pg. <div class="fpage">99</div>-<div class="lpage">108</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insertion%20with%20long%20target%20duplication%3A%20a%20mechanism%20for%20gene%20mobility%20suggested%20from%20comparison%20of%20two%20related%20bacterial%20genomes&amp;author=A.%20Nobusato&amp;author=I.%20Uchiyama&amp;author=S.%20Ohashi&amp;author=I.%20Kobayashi&amp;publication_year=2000&amp;journal=Gene&amp;volume=259&amp;pages=99-108" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(00)00456-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(00)00456-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(00)00456-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11163967" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chinen</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Uchiyama</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Comparison between <em>Pyrococcus horikoshii</em> and <em>Pyrococcus abyssi</em> genome sequences reveals linkage of restriction-modification genes with large genome polymorphisms</div>, <div class="source">Gene</div> , <div class="year">2000</div>, vol. <div class="volume">259</div> (pg. <div class="fpage">109</div>-<div class="lpage">121</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20between%20Pyrococcus%20horikoshii%20and%20Pyrococcus%20abyssi%20genome%20sequences%20reveals%20linkage%20of%20restriction-modification%20genes%20with%20large%20genome%20polymorphisms&amp;author=A.%20Chinen&amp;author=I.%20Uchiyama&amp;author=I.%20Kobayashi&amp;publication_year=2000&amp;journal=Gene&amp;volume=259&amp;pages=109-121" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(00)00459-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(00)00459-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(00)00459-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11163968" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tettelin</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Saunders</div> <div class="given-names">N. J.</div></div>,  <div class="name"><div class="surname">Heidelberg</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Complete genome sequence of <em>Neisseria meningitidis</em> serogroup B strain MC58</div>, <div class="source">Science</div> , <div class="year">2000</div>, vol. <div class="volume">287</div> (pg. <div class="fpage">1809</div>-<div class="lpage">1815</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20Neisseria%20meningitidis%20serogroup%20B%20strain%20MC58&amp;author=H.%20Tettelin&amp;author=N.%20J.%20Saunders&amp;author=J.%20Heidelberg&amp;publication_year=2000&amp;journal=Science&amp;volume=287&amp;pages=1809-1815" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.287.5459.1809" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.287.5459.1809" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.287.5459.1809"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10710307" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Model</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Russel</div> <div class="given-names">M.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Calendar</div> <div class="given-names">R.</div></div>. </div><div class="article-title">The Bacteriophages</div>, <div class="year">1988</div>, vol. <div class="volume">Vol. 2</div> <div class="publisher-loc">New York, NY</div><div class="publisher-name">Plenum Publishing Corporation</div>(pg. <div class="fpage">375</div>-<div class="lpage">456</div>)</div></div></div></div><div content-id="b5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Waldor</div> <div class="given-names">M. K.</div></div>,  <div class="name"><div class="surname">Mekalanos</div> <div class="given-names">J. J.</div></div>. </div><div class="article-title">Lysogenic conversion by a filamentous phage encoding cholera toxin</div>, <div class="source">Science</div> , <div class="year">1996</div>, vol. <div class="volume">272</div> (pg. <div class="fpage">1910</div>-<div class="lpage">1914</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Lysogenic%20conversion%20by%20a%20filamentous%20phage%20encoding%20cholera%20toxin&amp;author=M.%20K.%20Waldor&amp;author=J.%20J.%20Mekalanos&amp;publication_year=1996&amp;journal=Science&amp;volume=272&amp;pages=1910-1914" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.272.5270.1910" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.272.5270.1910" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.272.5270.1910"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8658163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huber</div> <div class="given-names">K. E.</div></div>,  <div class="name"><div class="surname">Waldor</div> <div class="given-names">M. K.</div></div>. </div><div class="article-title">Filamentous phage integration requires the host recombinases XerC and XerD</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">417</div> (pg. <div class="fpage">656</div>-<div class="lpage">659</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Filamentous%20phage%20integration%20requires%20the%20host%20recombinases%20XerC%20and%20XerD&amp;author=K.%20E.%20Huber&amp;author=M.%20K.%20Waldor&amp;publication_year=2002&amp;journal=Nature&amp;volume=417&amp;pages=656-659" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature00782" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature00782" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature00782"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12050668" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Webb</div> <div class="given-names">J. S.</div></div>,  <div class="name"><div class="surname">Lau</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kjelleberg</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Bacteriophage and phenotypic variation in <em>Pseudomonas aeruginosa</em> biofilm development</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2004</div>, vol. <div class="volume">186</div> (pg. <div class="fpage">8066</div>-<div class="lpage">8073</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacteriophage%20and%20phenotypic%20variation%20in%20Pseudomonas%20aeruginosa%20biofilm%20development&amp;author=J.%20S.%20Webb&amp;author=M.%20Lau&amp;author=S.%20Kjelleberg&amp;publication_year=2004&amp;journal=J.%20Bacteriol.&amp;volume=186&amp;pages=8066-8073" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.23.8066-8073.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.23.8066-8073.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.23.8066-8073.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15547279" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parkhill</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Achtman</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">James</div> <div class="given-names">K. D.</div></div>, et al. </div><div class="article-title">Complete DNA sequence of a serogroup A strain of <em>Neisseria meningitidis</em> Z2491</div>, <div class="source">Nature</div> , <div class="year">2000</div>, vol. <div class="volume">404</div> (pg. <div class="fpage">502</div>-<div class="lpage">506</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20DNA%20sequence%20of%20a%20serogroup%20A%20strain%20of%20Neisseria%20meningitidis%20Z2491&amp;author=J.%20Parkhill&amp;author=M.%20Achtman&amp;author=K.%20D.%20James&amp;publication_year=2000&amp;journal=Nature&amp;volume=404&amp;pages=502-506" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35006655" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35006655" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35006655"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10761919" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choi</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ohta</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ohtsubo</div> <div class="given-names">E.</div></div>. </div><div class="article-title">A novel IS element, IS<em>621</em>, of the IS<em>110</em>/IS<em>492</em> family transposes to a specific site in repetitive extragenic palindromic sequences in <em>Escherichia coli</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2003</div>, vol. <div class="volume">185</div> (pg. <div class="fpage">4891</div>-<div class="lpage">4900</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20IS%20element%2C%20IS621%2C%20of%20the%20IS110%2FIS492%20family%20transposes%20to%20a%20specific%20site%20in%20repetitive%20extragenic%20palindromic%20sequences%20in%20Escherichia%20coli&amp;author=S.%20Choi&amp;author=S.%20Ohta&amp;author=E.%20Ohtsubo&amp;publication_year=2003&amp;journal=J.%20Bacteriol.&amp;volume=185&amp;pages=4891-4900" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.185.16.4891-4900.2003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.185.16.4891-4900.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.185.16.4891-4900.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12897009" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Skaar</div> <div class="given-names">E. P.</div></div>,  <div class="name"><div class="surname">Lecuyer</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Lenich</div> <div class="given-names">A. G.</div></div>, et al. </div><div class="article-title">Analysis of the Piv recombinase-related gene family of <em>Neisseria gonorrhoeae</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2005</div>, vol. <div class="volume">187</div> (pg. <div class="fpage">1276</div>-<div class="lpage">1286</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20the%20Piv%20recombinase-related%20gene%20family%20of%20Neisseria%20gonorrhoeae&amp;author=E.%20P.%20Skaar&amp;author=B.%20Lecuyer&amp;author=A.%20G.%20Lenich&amp;publication_year=2005&amp;journal=J.%20Bacteriol.&amp;volume=187&amp;pages=1276-1286" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.4.1276-1286.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.4.1276-1286.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.4.1276-1286.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15687191" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lenich</div> <div class="given-names">A. G.</div></div>,  <div class="name"><div class="surname">Glasgow</div> <div class="given-names">A. C.</div></div>. </div><div class="article-title">Amino acid sequence homology between Piv, an essential protein in site-specific DNA inversion in <em>Moraxella lacunata</em>, and transposases of an unusual family of insertion elements</div>, <div class="source">J. Bacteriol.</div> , <div class="year">1994</div>, vol. <div class="volume">176</div> (pg. <div class="fpage">4160</div>-<div class="lpage">4164</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amino%20acid%20sequence%20homology%20between%20Piv%2C%20an%20essential%20protein%20in%20site-specific%20DNA%20inversion%20in%20Moraxella%20lacunata%2C%20and%20transposases%20of%20an%20unusual%20family%20of%20insertion%20elements&amp;author=A.%20G.%20Lenich&amp;author=A.%20C.%20Glasgow&amp;publication_year=1994&amp;journal=J.%20Bacteriol.&amp;volume=176&amp;pages=4160-4164" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/jb.176.13.4160-4164.1994" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2Fjb.176.13.4160-4164.1994" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2Fjb.176.13.4160-4164.1994"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8021196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bille</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Zahar</div> <div class="given-names">J. R.</div></div>,  <div class="name"><div class="surname">Perrin</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">A chromosomally integrated bacteriophage in invasive meningococci</div>, <div class="source">J. Exp. Med.</div> , <div class="year">2005</div>, vol. <div class="volume">201</div> (pg. <div class="fpage">1905</div>-<div class="lpage">1913</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20chromosomally%20integrated%20bacteriophage%20in%20invasive%20meningococci&amp;author=E.%20Bille&amp;author=J.%20R.%20Zahar&amp;author=A.%20Perrin&amp;publication_year=2005&amp;journal=J.%20Exp.%20Med.&amp;volume=201&amp;pages=1905-1913" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1084/jem.20050112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1084%2Fjem.20050112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1084%2Fjem.20050112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15967821" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Claus</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Stoevesandt</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Frosch</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Vogel</div> <div class="given-names">U.</div></div>. </div><div class="article-title">Genetic isolation of meningococci of the electrophoretic type 37 complex</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2001</div>, vol. <div class="volume">183</div> (pg. <div class="fpage">2570</div>-<div class="lpage">2575</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20isolation%20of%20meningococci%20of%20the%20electrophoretic%20type%2037%20complex&amp;author=H.%20Claus&amp;author=J.%20Stoevesandt&amp;author=M.%20Frosch&amp;author=U.%20Vogel&amp;publication_year=2001&amp;journal=J.%20Bacteriol.&amp;volume=183&amp;pages=2570-2575" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.183.8.2570-2575.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.183.8.2570-2575.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.183.8.2570-2575.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11274117" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shimizu</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Akaishi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Negishi</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Structural analysis of a putative hypovirulent plasmid, pJTPS1, found in a spontaneous avirulent mutant of <em>Ralstonia solanacearum</em></div>, <div class="source">Ann. Phytopathol. Soc. Jpn.</div> , <div class="year">1999</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">184</div>-<div class="lpage">188</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20analysis%20of%20a%20putative%20hypovirulent%20plasmid%2C%20pJTPS1%2C%20found%20in%20a%20spontaneous%20avirulent%20mutant%20of%20Ralstonia%20solanacearum&amp;author=R.%20Shimizu&amp;author=K.%20Akaishi&amp;author=H.%20Negishi&amp;publication_year=1999&amp;journal=Ann.%20Phytopathol.%20Soc.%20Jpn.&amp;volume=65&amp;pages=184-188" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3186/jjphytopath.65.184" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3186%2Fjjphytopath.65.184" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3186%2Fjjphytopath.65.184"> </span></div> </div></div></div></div></div><div content-id="b15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Azakami</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Akimichi</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Usui</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Yumoto</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ebisu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kato</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Isolation and characterization of a plasmid DNA from periodontopathogenic bacterium, <em>Eikenella corrodens</em> 1073, which affects pilus formation and colony morphology</div>, <div class="source">Gene</div> , <div class="year">2005</div>, vol. <div class="volume">351</div> (pg. <div class="fpage">143</div>-<div class="lpage">148</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20a%20plasmid%20DNA%20from%20periodontopathogenic%20bacterium%2C%20Eikenella%20corrodens%201073%2C%20which%20affects%20pilus%20formation%20and%20colony%20morphology&amp;author=H.%20Azakami&amp;author=H.%20Akimichi&amp;author=M.%20Usui&amp;author=H.%20Yumoto&amp;author=S.%20Ebisu&amp;author=A.%20Kato&amp;publication_year=2005&amp;journal=Gene&amp;volume=351&amp;pages=143-148" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2005.03.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2005.03.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2005.03.015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15869847" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">van Wezenbeek</div> <div class="given-names">P. M.</div></div>,  <div class="name"><div class="surname">Hulsebos</div> <div class="given-names">T. J.</div></div>,  <div class="name"><div class="surname">Schoenmakers</div> <div class="given-names">J. G.</div></div>. </div><div class="article-title">Nucleotide sequence of the filamentous bacteriophage M13 DNA genome: comparison with phage fd</div>, <div class="source">Gene</div> , <div class="year">1980</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">129</div>-<div class="lpage">148</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20sequence%20of%20the%20filamentous%20bacteriophage%20M13%20DNA%20genome%3A%20comparison%20with%20phage%20fd&amp;author=P.%20M.%20van%20Wezenbeek&amp;author=T.%20J.%20Hulsebos&amp;author=J.%20G.%20Schoenmakers&amp;publication_year=1980&amp;journal=Gene&amp;volume=11&amp;pages=129-148" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0378-1119(80)90093-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0378-1119(80)90093-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0378-1119(80)90093-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6254849" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hill</div> <div class="given-names">D. F.</div></div>,  <div class="name"><div class="surname">Short</div> <div class="given-names">N. J.</div></div>,  <div class="name"><div class="surname">Perham</div> <div class="given-names">R. N.</div></div>,  <div class="name"><div class="surname">Petersen</div> <div class="given-names">G. B.</div></div>. </div><div class="article-title">DNA sequence of the filamentous bacteriophage Pf1</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1991</div>, vol. <div class="volume">218</div> (pg. <div class="fpage">349</div>-<div class="lpage">364</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20sequence%20of%20the%20filamentous%20bacteriophage%20Pf1&amp;author=D.%20F.%20Hill&amp;author=N.%20J.%20Short&amp;author=R.%20N.%20Perham&amp;author=G.%20B.%20Petersen&amp;publication_year=1991&amp;journal=J.%20Mol.%20Biol.&amp;volume=218&amp;pages=349-364" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(91)90717-K" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(91)90717-K" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(91)90717-K"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2010913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Luiten</div> <div class="given-names">R. G.</div></div>,  <div class="name"><div class="surname">Putterman</div> <div class="given-names">D. G.</div></div>,  <div class="name"><div class="surname">Schoenmakers</div> <div class="given-names">J. G.</div></div>,  <div class="name"><div class="surname">Konings</div> <div class="given-names">R. N.</div></div>,  <div class="name"><div class="surname">Day</div> <div class="given-names">L. A.</div></div>. </div><div class="article-title">Nucleotide sequence of the genome of Pf3, an IncP-1 plasmid-specific filamentous bacteriophage of <em>Pseudomonas aeruginosa</em></div>, <div class="source">J. Virol.</div> , <div class="year">1985</div>, vol. <div class="volume">56</div> (pg. <div class="fpage">268</div>-<div class="lpage">276</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nucleotide%20sequence%20of%20the%20genome%20of%20Pf3%2C%20an%20IncP-1%20plasmid-specific%20filamentous%20bacteriophage%20of%20Pseudomonas%20aeruginosa&amp;author=R.%20G.%20Luiten&amp;author=D.%20G.%20Putterman&amp;author=J.%20G.%20Schoenmakers&amp;author=R.%20N.%20Konings&amp;author=L.%20A.%20Day&amp;publication_year=1985&amp;journal=J.%20Virol.&amp;volume=56&amp;pages=268-276" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3928901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Campos</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Martínez</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Suzarte</div> <div class="given-names">E.</div></div>, et al. </div><div class="article-title">VGJφ, a novel filamentous phage of <em>Vibrio cholerae</em>, integrates into the same chromosomal site as CTXφ</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2003</div>, vol. <div class="volume">185</div> (pg. <div class="fpage">5685</div>-<div class="lpage">5696</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=VGJ%CF%86%2C%20a%20novel%20filamentous%20phage%20of%20Vibrio%20cholerae%2C%20integrates%20into%20the%20same%20chromosomal%20site%20as%20CTX%CF%86&amp;author=J.%20Campos&amp;author=E.%20Mart%C3%ADnez&amp;author=E.%20Suzarte&amp;publication_year=2003&amp;journal=J.%20Bacteriol.&amp;volume=185&amp;pages=5685-5696" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.185.19.5685-5696.2003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.185.19.5685-5696.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.185.19.5685-5696.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13129939" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chopin</div> <div class="given-names">M. C.</div></div>,  <div class="name"><div class="surname">Rouault</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ehrlich</div> <div class="given-names">S. D.</div></div>,  <div class="name"><div class="surname">Gautier</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Filamentous phage active on the gram-positive bacterium <em>Propionibacterium freudenreichii</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2002</div>, vol. <div class="volume">184</div> (pg. <div class="fpage">2030</div>-<div class="lpage">2033</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Filamentous%20phage%20active%20on%20the%20gram-positive%20bacterium%20Propionibacterium%20freudenreichii&amp;author=M.%20C.%20Chopin&amp;author=A.%20Rouault&amp;author=S.%20D.%20Ehrlich&amp;author=M.%20Gautier&amp;publication_year=2002&amp;journal=J.%20Bacteriol.&amp;volume=184&amp;pages=2030-2033" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.184.7.2030-2033.2002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.184.7.2030-2033.2002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.184.7.2030-2033.2002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11889111" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lukashin</div> <div class="given-names">A. V.</div></div>,  <div class="name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>. </div><div class="article-title">GeneMark.hmm: new solutions for gene finding</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">1107</div>-<div class="lpage">1115</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GeneMark.hmm%3A%20new%20solutions%20for%20gene%20finding&amp;author=A.%20V.%20Lukashin&amp;author=M.%20Borodovsky&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res.&amp;volume=26&amp;pages=1107-1115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.4.1107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.4.1107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.4.1107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9461475" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uchiyama</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Higuchi</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Kobayashi</div> <div class="given-names">I.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Dunker</div> <div class="given-names">A. K.</div></div>,  <div class="name"><div class="surname">Konagaya</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Miyano</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Takagi</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Genome Informatics 2000</div>, <div class="year">2000</div><div class="publisher-loc">Tokyo, Japan</div><div class="publisher-name">Universal Academy Press</div>(pg. <div class="fpage">341</div>-<div class="lpage">343</div>)</div></div></div></div><div content-id="b23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Uchiyama</div> <div class="given-names">I.</div></div>. </div><div class="article-title">MBGD: microbial genome database for comparative analysis</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">58</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MBGD%3A%20microbial%20genome%20database%20for%20comparative%20analysis&amp;author=I.%20Uchiyama&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=58-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12519947" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T. L.</div></div>,  <div class="name"><div class="surname">Schäffer</div> <div class="given-names">A. A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">3402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.%20F.%20Altschul&amp;author=T.%20L.%20Madden&amp;author=A.%20A.%20Sch%C3%A4ffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-3402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pearson</div> <div class="given-names">W. R.</div></div>,  <div class="name"><div class="surname">Lipman</div> <div class="given-names">D. J.</div></div>. </div><div class="article-title">Improved tools for biological sequence comparison</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1988</div>, vol. <div class="volume">85</div> (pg. <div class="fpage">2444</div>-<div class="lpage">2448</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20tools%20for%20biological%20sequence%20comparison&amp;author=W.%20R.%20Pearson&amp;author=D.%20J.%20Lipman&amp;publication_year=1988&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=85&amp;pages=2444-2448" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.85.8.2444" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.85.8.2444" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.85.8.2444"> </span></div> </div></div></div></div></div><div content-id="b26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J. D.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D. G.</div></div>,  <div class="name"><div class="surname">Gibson</div> <div class="given-names">T. J.</div></div>. </div><div class="article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1994</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">4673</div>-<div class="lpage">4680</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%20W%3A%20improving%20the%20sensitivity%20of%20progressive%20multiple%20sequence%20alignment%20through%20sequence%20weighting%2C%20position-specific%20gap%20penalties%20and%20weight%20matrix%20choice&amp;author=J.%20D.%20Thompson&amp;author=D.%20G.%20Higgins&amp;author=T.%20J.%20Gibson&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=22&amp;pages=4673-4680" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.22.4673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.22.4673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7984417" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perrière</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Gouy</div> <div class="given-names">M.</div></div>. </div><div class="article-title">WWW-query: an on-line retrieval system for biological sequence banks</div>, <div class="source">Biochimie</div> , <div class="year">1996</div>, vol. <div class="volume">78</div> (pg. <div class="fpage">364</div>-<div class="lpage">369</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WWW-query%3A%20an%20on-line%20retrieval%20system%20for%20biological%20sequence%20banks&amp;author=G.%20Perri%C3%A8re&amp;author=M.%20Gouy&amp;publication_year=1996&amp;journal=Biochimie&amp;volume=78&amp;pages=364-369" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0300-9084(96)84768-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0300-9084(96)84768-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0300-9084(96)84768-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8905155" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Record</div> <div class="given-names">M. T.</div>Jr</div>,  <div class="name"><div class="surname">Reznikoff</div> <div class="given-names">W. S.</div></div>,  <div class="name"><div class="surname">Craig</div> <div class="given-names">M. L.</div></div>,  <div class="name"><div class="surname">McQuade</div> <div class="given-names">K. L.</div></div>,  <div class="name"><div class="surname">Schlax</div> <div class="given-names">P. J.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Neidhardt</div> <div class="given-names">F. C.</div></div>,  <div class="name"><div class="surname">Curtiss</div> <div class="given-names">R.</div>III</div>,  <div class="name"><div class="surname">Ingraham</div> <div class="given-names">J. L.</div></div>, et al. </div><div class="article-title"><em>Escherichia coli</em> and <em>Salmonella</em>: Cellular and Molecular Biology, Vol. 1</div>, <div class="year">1996</div><div class="publisher-loc">Washington, D.C.</div><div class="publisher-name">American Society of Microbiology</div>(pg. <div class="fpage">792</div>-<div class="lpage">821</div>)</div></div></div></div><div content-id="b29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Makita</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nakao</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div>. </div><div class="article-title">DBTBS: database of transcriptional regulation in <em>Bacillus subtilis</em> and its contribution to comparative genomics</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2004</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">D75</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DBTBS%3A%20database%20of%20transcriptional%20regulation%20in%20Bacillus%20subtilis%20and%20its%20contribution%20to%20comparative%20genomics&amp;author=Y.%20Makita&amp;author=M.%20Nakao&amp;author=N.%20Ogasawara&amp;author=K.%20Nakai&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=D75-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh074" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh074" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh074"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681362" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Leach</div> <div class="given-names">D. R.</div></div>. </div><div class="article-title">Long DNA palindromes, cruciform structures, genetic instability and secondary structure repair</div>, <div class="source">Bioessays</div> , <div class="year">1994</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">893</div>-<div class="lpage">900</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Long%20DNA%20palindromes%2C%20cruciform%20structures%2C%20genetic%20instability%20and%20secondary%20structure%20repair&amp;author=D.%20R.%20Leach&amp;publication_year=1994&amp;journal=Bioessays&amp;volume=16&amp;pages=893-900" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-1878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-1878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-1878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7840768" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Collick</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Drew</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Penberth</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Instability of long inverted repeats within mouse transgenes</div>, <div class="source">EMBO J.</div> , <div class="year">1996</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">1163</div>-<div class="lpage">1171</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Instability%20of%20long%20inverted%20repeats%20within%20mouse%20transgenes&amp;author=A.%20Collick&amp;author=J.%20Drew&amp;author=J.%20Penberth&amp;publication_year=1996&amp;journal=EMBO%20J.&amp;volume=15&amp;pages=1163-1171" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8605887" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Polard</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Seroude</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Fayet</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Prère</div> <div class="given-names">M. F.</div></div>,  <div class="name"><div class="surname">Chandler</div> <div class="given-names">M.</div></div>. </div><div class="article-title">One-ended insertion of IS<em>911</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">1994</div>, vol. <div class="volume">176</div> (pg. <div class="fpage">1192</div>-<div class="lpage">1196</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=One-ended%20insertion%20of%20IS911&amp;author=P.%20Polard&amp;author=L.%20Seroude&amp;author=O.%20Fayet&amp;author=M.%20F.%20Pr%C3%A8re&amp;author=M.%20Chandler&amp;publication_year=1994&amp;journal=J.%20Bacteriol.&amp;volume=176&amp;pages=1192-1196" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/jb.176.4.1192-1196.1994" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2Fjb.176.4.1192-1196.1994" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2Fjb.176.4.1192-1196.1994"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8106332" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Polard</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Prère</div> <div class="given-names">M. F.</div></div>,  <div class="name"><div class="surname">Fayet</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Chandler</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Transposase-induced excision and circularization of the bacterial insertion sequence IS<em>911</em></div>, <div class="source">EMBO J.</div> , <div class="year">1992</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">5079</div>-<div class="lpage">5090</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposase-induced%20excision%20and%20circularization%20of%20the%20bacterial%20insertion%20sequence%20IS911&amp;author=P.%20Polard&amp;author=M.%20F.%20Pr%C3%A8re&amp;author=O.%20Fayet&amp;author=M.%20Chandler&amp;publication_year=1992&amp;journal=EMBO%20J.&amp;volume=11&amp;pages=5079-5090" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1334464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sekine</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Eisaki</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ohtsubo</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Translational control in production of transposase and in transposition of insertion sequence IS<em>3</em></div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1994</div>, vol. <div class="volume">235</div> (pg. <div class="fpage">1406</div>-<div class="lpage">1420</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20control%20in%20production%20of%20transposase%20and%20in%20transposition%20of%20insertion%20sequence%20IS3&amp;author=Y.%20Sekine&amp;author=N.%20Eisaki&amp;author=E.%20Ohtsubo&amp;publication_year=1994&amp;journal=J.%20Mol.%20Biol.&amp;volume=235&amp;pages=1406-1420" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1994.1097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1994.1097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1994.1097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8107082" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Perkins-Balding</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Duval-Valentin</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Glasgow</div> <div class="given-names">A. C.</div></div>. </div><div class="article-title">Excision of IS<em>492</em> requires flanking target sequences and results in circle formation in <em>Pseudoalteromonas atlantica</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">1999</div>, vol. <div class="volume">181</div> (pg. <div class="fpage">4937</div>-<div class="lpage">4948</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Excision%20of%20IS492%20requires%20flanking%20target%20sequences%20and%20results%20in%20circle%20formation%20in%20Pseudoalteromonas%20atlantica&amp;author=D.%20Perkins-Balding&amp;author=G.%20Duval-Valentin&amp;author=A.%20C.%20Glasgow&amp;publication_year=1999&amp;journal=J.%20Bacteriol.&amp;volume=181&amp;pages=4937-4948" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10438765" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tobiason</div> <div class="given-names">D. M.</div></div>,  <div class="name"><div class="surname">Buchner</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Thiel</div> <div class="given-names">W. H.</div></div>,  <div class="name"><div class="surname">Gernert</div> <div class="given-names">K. M.</div></div>,  <div class="name"><div class="surname">Karls</div> <div class="given-names">A. C.</div></div>. </div><div class="article-title">Conserved amino acid motifs from the novel Piv/MooV family of transposases and site-specific recombinases are required for catalysis of DNA inversion by Piv</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2001</div>, vol. <div class="volume">39</div> (pg. <div class="fpage">641</div>-<div class="lpage">651</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conserved%20amino%20acid%20motifs%20from%20the%20novel%20Piv%2FMooV%20family%20of%20transposases%20and%20site-specific%20recombinases%20are%20required%20for%20catalysis%20of%20DNA%20inversion%20by%20Piv&amp;author=D.%20M.%20Tobiason&amp;author=J.%20M.%20Buchner&amp;author=W.%20H.%20Thiel&amp;author=K.%20M.%20Gernert&amp;author=A.%20C.%20Karls&amp;publication_year=2001&amp;journal=Mol.%20Microbiol.&amp;volume=39&amp;pages=641-651" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2958.2001.02276.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2958.2001.02276.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2958.2001.02276.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11169105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakashima</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Wiseman</div> <div class="given-names">R. L.</div></div>,  <div class="name"><div class="surname">Konigsberg</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Marvin</div> <div class="given-names">D. A.</div></div>. </div><div class="article-title">Primary structure and sidechain interactions of PFL filamentous bacterial virus coat protein</div>, <div class="source">Nature</div> , <div class="year">1975</div>, vol. <div class="volume">253</div> (pg. <div class="fpage">68</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primary%20structure%20and%20sidechain%20interactions%20of%20PFL%20filamentous%20bacterial%20virus%20coat%20protein&amp;author=Y.%20Nakashima&amp;author=R.%20L.%20Wiseman&amp;author=W.%20Konigsberg&amp;author=D.%20A.%20Marvin&amp;publication_year=1975&amp;journal=Nature&amp;volume=253&amp;pages=68-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/253068a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F253068a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F253068a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1110754" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maeda</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kneale</div> <div class="given-names">G. G.</div></div>,  <div class="name"><div class="surname">Tsugita</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">The DNA-binding protein of Pf1 filamentous bacteriophage: amino-acid sequence and structure of the gene</div>, <div class="source">EMBO J.</div> , <div class="year">1982</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">255</div>-<div class="lpage">261</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20DNA-binding%20protein%20of%20Pf1%20filamentous%20bacteriophage%3A%20amino-acid%20sequence%20and%20structure%20of%20the%20gene&amp;author=K.%20Maeda&amp;author=G.%20G.%20Kneale&amp;author=A.%20Tsugita&amp;publication_year=1982&amp;journal=EMBO%20J.&amp;volume=1&amp;pages=255-261" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16453414" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Putterman</div> <div class="given-names">D. G.</div></div>,  <div class="name"><div class="surname">Casadevall</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Boyle</div> <div class="given-names">P. D.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">H. L.</div></div>,  <div class="name"><div class="surname">Frangione</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Day</div> <div class="given-names">L. A.</div></div>. </div><div class="article-title">Major coat protein and single-stranded DNA-binding protein of filamentous virus Pf3</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1984</div>, vol. <div class="volume">81</div> (pg. <div class="fpage">699</div>-<div class="lpage">703</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Major%20coat%20protein%20and%20single-stranded%20DNA-binding%20protein%20of%20filamentous%20virus%20Pf3&amp;author=D.%20G.%20Putterman&amp;author=A.%20Casadevall&amp;author=P.%20D.%20Boyle&amp;author=H.%20L.%20Yang&amp;author=B.%20Frangione&amp;author=L.%20A.%20Day&amp;publication_year=1984&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=81&amp;pages=699-703" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.81.3.699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.81.3.699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.81.3.699"> </span></div> </div></div></div></div></div><div content-id="b40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koonin</div> <div class="given-names">E. V.</div></div>. </div><div class="article-title">The second cholera toxin, Zot, and its plasmid-encoded and phage-encoded homologues constitute a group of putative ATPases with an altered purine NTP-binding motif</div>, <div class="source">FEBS Lett.</div> , <div class="year">1992</div>, vol. <div class="volume">312</div> (pg. <div class="fpage">3</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20second%20cholera%20toxin%2C%20Zot%2C%20and%20its%20plasmid-encoded%20and%20phage-encoded%20homologues%20constitute%20a%20group%20of%20putative%20ATPases%20with%20an%20altered%20purine%20NTP-binding%20motif&amp;author=E.%20V.%20Koonin&amp;publication_year=1992&amp;journal=FEBS%20Lett.&amp;volume=312&amp;pages=3-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-5793(92)81398-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-5793(92)81398-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-5793(92)81398-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1426234" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="b41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="b41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Russel</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Linderoth</div> <div class="given-names">N. A.</div></div>,  <div class="name"><div class="surname">Sâli</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Filamentous phage assembly: variation on a protein export theme</div>, <div class="source">Gene</div> , <div class="year">1997</div>, vol. <div class="volume">192</div> (pg. <div class="fpage">23</div>-<div class="lpage">32</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Filamentous%20phage%20assembly%3A%20variation%20on%20a%20protein%20export%20theme&amp;author=M.%20Russel&amp;author=N.%20A.%20Linderoth&amp;author=A.%20S%C3%A2li&amp;publication_year=1997&amp;journal=Gene&amp;volume=192&amp;pages=23-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0378-1119(96)00801-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0378-1119(96)00801-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0378-1119(96)00801-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9224870" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2006. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=344376&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/12/6/10.1093_dnares_dsi021/1/dsi021_Supplementary_Data.zip?Expires=2147483647&amp;Signature=oJlJlpd4SDd3XXjTkouKq2Vzt3vG5ra12YlqCSzXdeWhmP4Ta~LIhtXX1jxAQTQ~3JyVSTA~iNwQB0UV1qtdoWKLFGYhI4oWG4mbSzDFdSMFhAeHr9TeAfdS-j4SYVlNMbJqQV-j3loRQu39kc9ok2bEod1HSbubqhd2QcdfqoomP0Bj5-wArJDxRboecMaMI5HjSb-Bqlq-~CLhgErTn5kviGr4lfIPp8oDPfYWI~p-~901jkd6X6DplDwj6Ob9UOiCz8T-hftUbdAIc6jZAK7pBk1qt8IewCBw6OAc-31mjQyXpGhJeAMGH~BbvEMuKI5PCVszTZt~GOTxyYHgLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">354</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119900001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">37</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsi021"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="344376"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000242119900001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f12%2f6%2f389%2f344376">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000242119900001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (37)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F12%2F6%2F389%2F344376" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/344376'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome Comparison In Silico in Neisseria Suggests Integration of Filamentous Bacteriophages by their Own Transposase" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsi021");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f12%2f6%2f389%2f344376">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

