

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" /><script type="text/javascript">window.NREUM||(NREUM={});NREUM.info = {"beacon":"bam.nr-data.net","errorBeacon":"bam.nr-data.net","licenseKey":"8d76ca6f26","applicationID":"29565620","transactionName":"ZQBRMkVUCEtRVkINXlxKcjVnGgdKRFxVCFQcBEAWTw==","queueTime":0,"applicationTime":3608,"agent":"","atts":""}</script><script type="text/javascript">(window.NREUM||(NREUM={})).loader_config={xpid:"XAYBV15WGwEJUVdXAQMH"};window.NREUM||(NREUM={}),__nr_require=function(t,e,n){function r(n){if(!e[n]){var o=e[n]={exports:{}};t[n][0].call(o.exports,function(e){var o=t[n][1][e];return r(o||e)},o,o.exports)}return e[n].exports}if("function"==typeof __nr_require)return __nr_require;for(var o=0;o<n.length;o++)r(n[o]);return r}({1:[function(t,e,n){function r(t){try{s.console&&console.log(t)}catch(e){}}var o,i=t("ee"),a=t(21),s={};try{o=localStorage.getItem("__nr_flags").split(","),console&&"function"==typeof console.log&&(s.console=!0,o.indexOf("dev")!==-1&&(s.dev=!0),o.indexOf("nr_dev")!==-1&&(s.nrDev=!0))}catch(c){}s.nrDev&&i.on("internal-error",function(t){r(t.stack)}),s.dev&&i.on("fn-err",function(t,e,n){r(n.stack)}),s.dev&&(r("NR AGENT IN DEVELOPMENT MODE"),r("flags: "+a(s,function(t,e){return t}).join(", ")))},{}],2:[function(t,e,n){function r(t,e,n,r,s){try{l?l-=1:o(s||new UncaughtException(t,e,n),!0)}catch(f){try{i("ierr",[f,c.now(),!0])}catch(d){}}return"function"==typeof u&&u.apply(this,a(arguments))}function UncaughtException(t,e,n){this.message=t||"Uncaught error with no additional information",this.sourceURL=e,this.line=n}function o(t,e){var n=e?null:c.now();i("err",[t,n])}var i=t("handle"),a=t(22),s=t("ee"),c=t("loader"),f=t("gos"),u=window.onerror,d=!1,p="nr@seenError",l=0;c.features.err=!0,t(1),window.onerror=r;try{throw new Error}catch(h){"stack"in h&&(t(13),t(12),"addEventListener"in window&&t(6),c.xhrWrappable&&t(14),d=!0)}s.on("fn-start",function(t,e,n){d&&(l+=1)}),s.on("fn-err",function(t,e,n){d&&!n[p]&&(f(n,p,function(){return!0}),this.thrown=!0,o(n))}),s.on("fn-end",function(){d&&!this.thrown&&l>0&&(l-=1)}),s.on("internal-error",function(t){i("ierr",[t,c.now(),!0])})},{}],3:[function(t,e,n){t("loader").features.ins=!0},{}],4:[function(t,e,n){function r(){M++,N=y.hash,this[u]=g.now()}function o(){M--,y.hash!==N&&i(0,!0);var t=g.now();this[h]=~~this[h]+t-this[u],this[d]=t}function i(t,e){E.emit("newURL",[""+y,e])}function a(t,e){t.on(e,function(){this[e]=g.now()})}var s="-start",c="-end",f="-body",u="fn"+s,d="fn"+c,p="cb"+s,l="cb"+c,h="jsTime",m="fetch",v="addEventListener",w=window,y=w.location,g=t("loader");if(w[v]&&g.xhrWrappable){var b=t(10),x=t(11),E=t(8),O=t(6),P=t(13),R=t(7),T=t(14),L=t(9),j=t("ee"),S=j.get("tracer");t(15),g.features.spa=!0;var N,M=0;j.on(u,r),j.on(p,r),j.on(d,o),j.on(l,o),j.buffer([u,d,"xhr-done","xhr-resolved"]),O.buffer([u]),P.buffer(["setTimeout"+c,"clearTimeout"+s,u]),T.buffer([u,"new-xhr","send-xhr"+s]),R.buffer([m+s,m+"-done",m+f+s,m+f+c]),E.buffer(["newURL"]),b.buffer([u]),x.buffer(["propagate",p,l,"executor-err","resolve"+s]),S.buffer([u,"no-"+u]),L.buffer(["new-jsonp","cb-start","jsonp-error","jsonp-end"]),a(T,"send-xhr"+s),a(j,"xhr-resolved"),a(j,"xhr-done"),a(R,m+s),a(R,m+"-done"),a(L,"new-jsonp"),a(L,"jsonp-end"),a(L,"cb-start"),E.on("pushState-end",i),E.on("replaceState-end",i),w[v]("hashchange",i,!0),w[v]("load",i,!0),w[v]("popstate",function(){i(0,M>1)},!0)}},{}],5:[function(t,e,n){function r(t){}if(window.performance&&window.performance.timing&&window.performance.getEntriesByType){var o=t("ee"),i=t("handle"),a=t(13),s=t(12),c="learResourceTimings",f="addEventListener",u="resourcetimingbufferfull",d="bstResource",p="resource",l="-start",h="-end",m="fn"+l,v="fn"+h,w="bstTimer",y="pushState",g=t("loader");g.features.stn=!0,t(8);var b=NREUM.o.EV;o.on(m,function(t,e){var n=t[0];n instanceof b&&(this.bstStart=g.now())}),o.on(v,function(t,e){var n=t[0];n instanceof b&&i("bst",[n,e,this.bstStart,g.now()])}),a.on(m,function(t,e,n){this.bstStart=g.now(),this.bstType=n}),a.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),this.bstType])}),s.on(m,function(){this.bstStart=g.now()}),s.on(v,function(t,e){i(w,[e,this.bstStart,g.now(),"requestAnimationFrame"])}),o.on(y+l,function(t){this.time=g.now(),this.startPath=location.pathname+location.hash}),o.on(y+h,function(t){i("bstHist",[location.pathname+location.hash,this.startPath,this.time])}),f in window.performance&&(window.performance["c"+c]?window.performance[f](u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["c"+c]()},!1):window.performance[f]("webkit"+u,function(t){i(d,[window.performance.getEntriesByType(p)]),window.performance["webkitC"+c]()},!1)),document[f]("scroll",r,{passive:!0}),document[f]("keypress",r,!1),document[f]("click",r,!1)}},{}],6:[function(t,e,n){function r(t){for(var e=t;e&&!e.hasOwnProperty(u);)e=Object.getPrototypeOf(e);e&&o(e)}function o(t){s.inPlace(t,[u,d],"-",i)}function i(t,e){return t[1]}var a=t("ee").get("events"),s=t(24)(a,!0),c=t("gos"),f=XMLHttpRequest,u="addEventListener",d="removeEventListener";e.exports=a,"getPrototypeOf"in Object?(r(document),r(window),r(f.prototype)):f.prototype.hasOwnProperty(u)&&(o(window),o(f.prototype)),a.on(u+"-start",function(t,e){var n=t[1],r=c(n,"nr@wrapped",function(){function t(){if("function"==typeof n.handleEvent)return n.handleEvent.apply(n,arguments)}var e={object:t,"function":n}[typeof n];return e?s(e,"fn-",null,e.name||"anonymous"):n});this.wrapped=t[1]=r}),a.on(d+"-start",function(t){t[1]=this.wrapped||t[1]})},{}],7:[function(t,e,n){function r(t,e,n){var r=t[e];"function"==typeof r&&(t[e]=function(){var t=r.apply(this,arguments);return o.emit(n+"start",arguments,t),t.then(function(e){return o.emit(n+"end",[null,e],t),e},function(e){throw o.emit(n+"end",[e],t),e})})}var o=t("ee").get("fetch"),i=t(21);e.exports=o;var a=window,s="fetch-",c=s+"body-",f=["arrayBuffer","blob","json","text","formData"],u=a.Request,d=a.Response,p=a.fetch,l="prototype";u&&d&&p&&(i(f,function(t,e){r(u[l],e,c),r(d[l],e,c)}),r(a,"fetch",s),o.on(s+"end",function(t,e){var n=this;if(e){var r=e.headers.get("content-length");null!==r&&(n.rxSize=r),o.emit(s+"done",[null,e],n)}else o.emit(s+"done",[t],n)}))},{}],8:[function(t,e,n){var r=t("ee").get("history"),o=t(24)(r);e.exports=r,o.inPlace(window.history,["pushState","replaceState"],"-")},{}],9:[function(t,e,n){function r(t){function e(){c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}function n(){c.emit("jsonp-error",[],p),c.emit("jsonp-end",[],p),t.removeEventListener("load",e,!1),t.removeEventListener("error",n,!1)}var r=t&&"string"==typeof t.nodeName&&"script"===t.nodeName.toLowerCase();if(r){var o="function"==typeof t.addEventListener;if(o){var a=i(t.src);if(a){var u=s(a),d="function"==typeof u.parent[u.key];if(d){var p={};f.inPlace(u.parent,[u.key],"cb-",p),t.addEventListener("load",e,!1),t.addEventListener("error",n,!1),c.emit("new-jsonp",[t.src],p)}}}}}function o(){return"addEventListener"in window}function i(t){var e=t.match(u);return e?e[1]:null}function a(t,e){var n=t.match(p),r=n[1],o=n[3];return o?a(o,e[r]):e[r]}function s(t){var e=t.match(d);return e&&e.length>=3?{key:e[2],parent:a(e[1],window)}:{key:t,parent:window}}var c=t("ee").get("jsonp"),f=t(24)(c);if(e.exports=c,o()){var u=/[?&](?:callback|cb)=([^&#]+)/,d=/(.*)\.([^.]+)/,p=/^(\w+)(\.|$)(.*)$/,l=["appendChild","insertBefore","replaceChild"];f.inPlace(HTMLElement.prototype,l,"dom-"),f.inPlace(HTMLHeadElement.prototype,l,"dom-"),f.inPlace(HTMLBodyElement.prototype,l,"dom-"),c.on("dom-start",function(t){r(t[0])})}},{}],10:[function(t,e,n){var r=t("ee").get("mutation"),o=t(24)(r),i=NREUM.o.MO;e.exports=r,i&&(window.MutationObserver=function(t){return this instanceof i?new i(o(t,"fn-")):i.apply(this,arguments)},MutationObserver.prototype=i.prototype)},{}],11:[function(t,e,n){function r(t){var e=a.context(),n=s(t,"executor-",e),r=new f(n);return a.context(r).getCtx=function(){return e},a.emit("new-promise",[r,e],e),r}function o(t,e){return e}var i=t(24),a=t("ee").get("promise"),s=i(a),c=t(21),f=NREUM.o.PR;e.exports=a,f&&(window.Promise=r,["all","race"].forEach(function(t){var e=f[t];f[t]=function(n){function r(t){return function(){a.emit("propagate",[null,!o],i),o=o||!t}}var o=!1;c(n,function(e,n){Promise.resolve(n).then(r("all"===t),r(!1))});var i=e.apply(f,arguments),s=f.resolve(i);return s}}),["resolve","reject"].forEach(function(t){var e=f[t];f[t]=function(t){var n=e.apply(f,arguments);return t!==n&&a.emit("propagate",[t,!0],n),n}}),f.prototype["catch"]=function(t){return this.then(null,t)},f.prototype=Object.create(f.prototype,{constructor:{value:r}}),c(Object.getOwnPropertyNames(f),function(t,e){try{r[e]=f[e]}catch(n){}}),a.on("executor-start",function(t){t[0]=s(t[0],"resolve-",this),t[1]=s(t[1],"resolve-",this)}),a.on("executor-err",function(t,e,n){t[1](n)}),s.inPlace(f.prototype,["then"],"then-",o),a.on("then-start",function(t,e){this.promise=e,t[0]=s(t[0],"cb-",this),t[1]=s(t[1],"cb-",this)}),a.on("then-end",function(t,e,n){this.nextPromise=n;var r=this.promise;a.emit("propagate",[r,!0],n)}),a.on("cb-end",function(t,e,n){a.emit("propagate",[n,!0],this.nextPromise)}),a.on("propagate",function(t,e,n){this.getCtx&&!e||(this.getCtx=function(){if(t instanceof Promise)var e=a.context(t);return e&&e.getCtx?e.getCtx():this})}),r.toString=function(){return""+f})},{}],12:[function(t,e,n){var r=t("ee").get("raf"),o=t(24)(r),i="equestAnimationFrame";e.exports=r,o.inPlace(window,["r"+i,"mozR"+i,"webkitR"+i,"msR"+i],"raf-"),r.on("raf-start",function(t){t[0]=o(t[0],"fn-")})},{}],13:[function(t,e,n){function r(t,e,n){t[0]=a(t[0],"fn-",null,n)}function o(t,e,n){this.method=n,this.timerDuration=isNaN(t[1])?0:+t[1],t[0]=a(t[0],"fn-",this,n)}var i=t("ee").get("timer"),a=t(24)(i),s="setTimeout",c="setInterval",f="clearTimeout",u="-start",d="-";e.exports=i,a.inPlace(window,[s,"setImmediate"],s+d),a.inPlace(window,[c],c+d),a.inPlace(window,[f,"clearImmediate"],f+d),i.on(c+u,r),i.on(s+u,o)},{}],14:[function(t,e,n){function r(t,e){d.inPlace(e,["onreadystatechange"],"fn-",s)}function o(){var t=this,e=u.context(t);t.readyState>3&&!e.resolved&&(e.resolved=!0,u.emit("xhr-resolved",[],t)),d.inPlace(t,y,"fn-",s)}function i(t){g.push(t),h&&(x?x.then(a):v?v(a):(E=-E,O.data=E))}function a(){for(var t=0;t<g.length;t++)r([],g[t]);g.length&&(g=[])}function s(t,e){return e}function c(t,e){for(var n in t)e[n]=t[n];return e}t(6);var f=t("ee"),u=f.get("xhr"),d=t(24)(u),p=NREUM.o,l=p.XHR,h=p.MO,m=p.PR,v=p.SI,w="readystatechange",y=["onload","onerror","onabort","onloadstart","onloadend","onprogress","ontimeout"],g=[];e.exports=u;var b=window.XMLHttpRequest=function(t){var e=new l(t);try{u.emit("new-xhr",[e],e),e.addEventListener(w,o,!1)}catch(n){try{u.emit("internal-error",[n])}catch(r){}}return e};if(c(l,b),b.prototype=l.prototype,d.inPlace(b.prototype,["open","send"],"-xhr-",s),u.on("send-xhr-start",function(t,e){r(t,e),i(e)}),u.on("open-xhr-start",r),h){var x=m&&m.resolve();if(!v&&!m){var E=1,O=document.createTextNode(E);new h(a).observe(O,{characterData:!0})}}else f.on("fn-end",function(t){t[0]&&t[0].type===w||a()})},{}],15:[function(t,e,n){function r(t){var e=this.params,n=this.metrics;if(!this.ended){this.ended=!0;for(var r=0;r<d;r++)t.removeEventListener(u[r],this.listener,!1);if(!e.aborted){if(n.duration=a.now()-this.startTime,4===t.readyState){e.status=t.status;var i=o(t,this.lastSize);if(i&&(n.rxSize=i),this.sameOrigin){var c=t.getResponseHeader("X-NewRelic-App-Data");c&&(e.cat=c.split(", ").pop())}}else e.status=0;n.cbTime=this.cbTime,f.emit("xhr-done",[t],t),s("xhr",[e,n,this.startTime])}}}function o(t,e){var n=t.responseType;if("json"===n&&null!==e)return e;var r="arraybuffer"===n||"blob"===n||"json"===n?t.response:t.responseText;return h(r)}function i(t,e){var n=c(e),r=t.params;r.host=n.hostname+":"+n.port,r.pathname=n.pathname,t.sameOrigin=n.sameOrigin}var a=t("loader");if(a.xhrWrappable){var s=t("handle"),c=t(16),f=t("ee"),u=["load","error","abort","timeout"],d=u.length,p=t("id"),l=t(19),h=t(18),m=window.XMLHttpRequest;a.features.xhr=!0,t(14),f.on("new-xhr",function(t){var e=this;e.totalCbs=0,e.called=0,e.cbTime=0,e.end=r,e.ended=!1,e.xhrGuids={},e.lastSize=null,l&&(l>34||l<10)||window.opera||t.addEventListener("progress",function(t){e.lastSize=t.loaded},!1)}),f.on("open-xhr-start",function(t){this.params={method:t[0]},i(this,t[1]),this.metrics={}}),f.on("open-xhr-end",function(t,e){"loader_config"in NREUM&&"xpid"in NREUM.loader_config&&this.sameOrigin&&e.setRequestHeader("X-NewRelic-ID",NREUM.loader_config.xpid)}),f.on("send-xhr-start",function(t,e){var n=this.metrics,r=t[0],o=this;if(n&&r){var i=h(r);i&&(n.txSize=i)}this.startTime=a.now(),this.listener=function(t){try{"abort"===t.type&&(o.params.aborted=!0),("load"!==t.type||o.called===o.totalCbs&&(o.onloadCalled||"function"!=typeof e.onload))&&o.end(e)}catch(n){try{f.emit("internal-error",[n])}catch(r){}}};for(var s=0;s<d;s++)e.addEventListener(u[s],this.listener,!1)}),f.on("xhr-cb-time",function(t,e,n){this.cbTime+=t,e?this.onloadCalled=!0:this.called+=1,this.called!==this.totalCbs||!this.onloadCalled&&"function"==typeof n.onload||this.end(n)}),f.on("xhr-load-added",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&!this.xhrGuids[n]&&(this.xhrGuids[n]=!0,this.totalCbs+=1)}),f.on("xhr-load-removed",function(t,e){var n=""+p(t)+!!e;this.xhrGuids&&this.xhrGuids[n]&&(delete this.xhrGuids[n],this.totalCbs-=1)}),f.on("addEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-added",[t[1],t[2]],e)}),f.on("removeEventListener-end",function(t,e){e instanceof m&&"load"===t[0]&&f.emit("xhr-load-removed",[t[1],t[2]],e)}),f.on("fn-start",function(t,e,n){e instanceof m&&("onload"===n&&(this.onload=!0),("load"===(t[0]&&t[0].type)||this.onload)&&(this.xhrCbStart=a.now()))}),f.on("fn-end",function(t,e){this.xhrCbStart&&f.emit("xhr-cb-time",[a.now()-this.xhrCbStart,this.onload,e],e)})}},{}],16:[function(t,e,n){e.exports=function(t){var e=document.createElement("a"),n=window.location,r={};e.href=t,r.port=e.port;var o=e.href.split("://");!r.port&&o[1]&&(r.port=o[1].split("/")[0].split("@").pop().split(":")[1]),r.port&&"0"!==r.port||(r.port="https"===o[0]?"443":"80"),r.hostname=e.hostname||n.hostname,r.pathname=e.pathname,r.protocol=o[0],"/"!==r.pathname.charAt(0)&&(r.pathname="/"+r.pathname);var i=!e.protocol||":"===e.protocol||e.protocol===n.protocol,a=e.hostname===document.domain&&e.port===n.port;return r.sameOrigin=i&&(!e.hostname||a),r}},{}],17:[function(t,e,n){function r(){}function o(t,e,n){return function(){return i(t,[f.now()].concat(s(arguments)),e?null:this,n),e?void 0:this}}var i=t("handle"),a=t(21),s=t(22),c=t("ee").get("tracer"),f=t("loader"),u=NREUM;"undefined"==typeof window.newrelic&&(newrelic=u);var d=["setPageViewName","setCustomAttribute","setErrorHandler","finished","addToTrace","inlineHit","addRelease"],p="api-",l=p+"ixn-";a(d,function(t,e){u[e]=o(p+e,!0,"api")}),u.addPageAction=o(p+"addPageAction",!0),u.setCurrentRouteName=o(p+"routeName",!0),e.exports=newrelic,u.interaction=function(){return(new r).get()};var h=r.prototype={createTracer:function(t,e){var n={},r=this,o="function"==typeof e;return i(l+"tracer",[f.now(),t,n],r),function(){if(c.emit((o?"":"no-")+"fn-start",[f.now(),r,o],n),o)try{return e.apply(this,arguments)}catch(t){throw c.emit("fn-err",[arguments,this,t],n),t}finally{c.emit("fn-end",[f.now()],n)}}}};a("actionText,setName,setAttribute,save,ignore,onEnd,getContext,end,get".split(","),function(t,e){h[e]=o(l+e)}),newrelic.noticeError=function(t){"string"==typeof t&&(t=new Error(t)),i("err",[t,f.now()])}},{}],18:[function(t,e,n){e.exports=function(t){if("string"==typeof t&&t.length)return t.length;if("object"==typeof t){if("undefined"!=typeof ArrayBuffer&&t instanceof ArrayBuffer&&t.byteLength)return t.byteLength;if("undefined"!=typeof Blob&&t instanceof Blob&&t.size)return t.size;if(!("undefined"!=typeof FormData&&t instanceof FormData))try{return JSON.stringify(t).length}catch(e){return}}}},{}],19:[function(t,e,n){var r=0,o=navigator.userAgent.match(/Firefox[\/\s](\d+\.\d+)/);o&&(r=+o[1]),e.exports=r},{}],20:[function(t,e,n){function r(t,e){if(!o)return!1;if(t!==o)return!1;if(!e)return!0;if(!i)return!1;for(var n=i.split("."),r=e.split("."),a=0;a<r.length;a++)if(r[a]!==n[a])return!1;return!0}var o=null,i=null,a=/Version\/(\S+)\s+Safari/;if(navigator.userAgent){var s=navigator.userAgent,c=s.match(a);c&&s.indexOf("Chrome")===-1&&s.indexOf("Chromium")===-1&&(o="Safari",i=c[1])}e.exports={agent:o,version:i,match:r}},{}],21:[function(t,e,n){function r(t,e){var n=[],r="",i=0;for(r in t)o.call(t,r)&&(n[i]=e(r,t[r]),i+=1);return n}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],22:[function(t,e,n){function r(t,e,n){e||(e=0),"undefined"==typeof n&&(n=t?t.length:0);for(var r=-1,o=n-e||0,i=Array(o<0?0:o);++r<o;)i[r]=t[e+r];return i}e.exports=r},{}],23:[function(t,e,n){e.exports={exists:"undefined"!=typeof window.performance&&window.performance.timing&&"undefined"!=typeof window.performance.timing.navigationStart}},{}],24:[function(t,e,n){function r(t){return!(t&&t instanceof Function&&t.apply&&!t[a])}var o=t("ee"),i=t(22),a="nr@original",s=Object.prototype.hasOwnProperty,c=!1;e.exports=function(t,e){function n(t,e,n,o){function nrWrapper(){var r,a,s,c;try{a=this,r=i(arguments),s="function"==typeof n?n(r,a):n||{}}catch(f){p([f,"",[r,a,o],s])}u(e+"start",[r,a,o],s);try{return c=t.apply(a,r)}catch(d){throw u(e+"err",[r,a,d],s),d}finally{u(e+"end",[r,a,c],s)}}return r(t)?t:(e||(e=""),nrWrapper[a]=t,d(t,nrWrapper),nrWrapper)}function f(t,e,o,i){o||(o="");var a,s,c,f="-"===o.charAt(0);for(c=0;c<e.length;c++)s=e[c],a=t[s],r(a)||(t[s]=n(a,f?s+o:o,i,s))}function u(n,r,o){if(!c||e){var i=c;c=!0;try{t.emit(n,r,o,e)}catch(a){p([a,n,r,o])}c=i}}function d(t,e){if(Object.defineProperty&&Object.keys)try{var n=Object.keys(t);return n.forEach(function(n){Object.defineProperty(e,n,{get:function(){return t[n]},set:function(e){return t[n]=e,e}})}),e}catch(r){p([r])}for(var o in t)s.call(t,o)&&(e[o]=t[o]);return e}function p(e){try{t.emit("internal-error",e)}catch(n){}}return t||(t=o),n.inPlace=f,n.flag=a,n}},{}],ee:[function(t,e,n){function r(){}function o(t){function e(t){return t&&t instanceof r?t:t?c(t,s,i):i()}function n(n,r,o,i){if(!p.aborted||i){t&&t(n,r,o);for(var a=e(o),s=m(n),c=s.length,f=0;f<c;f++)s[f].apply(a,r);var d=u[g[n]];return d&&d.push([b,n,r,a]),a}}function l(t,e){y[t]=m(t).concat(e)}function h(t,e){var n=y[t];if(n)for(var r=0;r<n.length;r++)n[r]===e&&n.splice(r,1)}function m(t){return y[t]||[]}function v(t){return d[t]=d[t]||o(n)}function w(t,e){f(t,function(t,n){e=e||"feature",g[n]=e,e in u||(u[e]=[])})}var y={},g={},b={on:l,addEventListener:l,removeEventListener:h,emit:n,get:v,listeners:m,context:e,buffer:w,abort:a,aborted:!1};return b}function i(){return new r}function a(){(u.api||u.feature)&&(p.aborted=!0,u=p.backlog={})}var s="nr@context",c=t("gos"),f=t(21),u={},d={},p=e.exports=o();p.backlog=u},{}],gos:[function(t,e,n){function r(t,e,n){if(o.call(t,e))return t[e];var r=n();if(Object.defineProperty&&Object.keys)try{return Object.defineProperty(t,e,{value:r,writable:!0,enumerable:!1}),r}catch(i){}return t[e]=r,r}var o=Object.prototype.hasOwnProperty;e.exports=r},{}],handle:[function(t,e,n){function r(t,e,n,r){o.buffer([t],r),o.emit(t,e,n)}var o=t("ee").get("handle");e.exports=r,r.ee=o},{}],id:[function(t,e,n){function r(t){var e=typeof t;return!t||"object"!==e&&"function"!==e?-1:t===window?0:a(t,i,function(){return o++})}var o=1,i="nr@id",a=t("gos");e.exports=r},{}],loader:[function(t,e,n){function r(){if(!E++){var t=x.info=NREUM.info,e=l.getElementsByTagName("script")[0];if(setTimeout(u.abort,3e4),!(t&&t.licenseKey&&t.applicationID&&e))return u.abort();f(g,function(e,n){t[e]||(t[e]=n)}),c("mark",["onload",a()+x.offset],null,"api");var n=l.createElement("script");n.src="https://"+t.agent,e.parentNode.insertBefore(n,e)}}function o(){"complete"===l.readyState&&i()}function i(){c("mark",["domContent",a()+x.offset],null,"api")}function a(){return O.exists&&performance.now?Math.round(performance.now()):(s=Math.max((new Date).getTime(),s))-x.offset}var s=(new Date).getTime(),c=t("handle"),f=t(21),u=t("ee"),d=t(20),p=window,l=p.document,h="addEventListener",m="attachEvent",v=p.XMLHttpRequest,w=v&&v.prototype;NREUM.o={ST:setTimeout,SI:p.setImmediate,CT:clearTimeout,XHR:v,REQ:p.Request,EV:p.Event,PR:p.Promise,MO:p.MutationObserver};var y=""+location,g={beacon:"bam.nr-data.net",errorBeacon:"bam.nr-data.net",agent:"js-agent.newrelic.com/nr-spa-1099.min.js"},b=v&&w&&w[h]&&!/CriOS/.test(navigator.userAgent),x=e.exports={offset:s,now:a,origin:y,features:{},xhrWrappable:b,userAgent:d};t(17),l[h]?(l[h]("DOMContentLoaded",i,!1),p[h]("load",r,!1)):(l[m]("onreadystatechange",o),p[m]("onload",r)),c("mark",["firstbyte",s],null,"api");var E=0,O=t(23)},{}]},{},["loader",2,15,5,3,4]);</script>
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/2/159/2960055","@type":"ScholarlyArticle","name":"Large-scale collection of full-length cDNA and transcriptome analysis in  Hevea brasiliensis ","datePublished":"2017-01-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2017-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsw056","keywords":["RNA-Seq","rubber biosynthesis","latex","transcription factor","full-length cDNA"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/2/159/2960055","author":[{"name":"Makita, Yuko","affiliation":"1 Synthetic Genomics Research Group, Biomass Engineering Research Division, RIKEN Center for Sustainable Resource Science (CSRS), Yokohama, Kanagawa 230-0045, Japan","@type":"Person"},{"name":"Ng, Kiaw Kiaw","affiliation":"1 Synthetic Genomics Research Group, Biomass Engineering Research Division, RIKEN Center for Sustainable Resource Science (CSRS), Yokohama, Kanagawa 230-0045, Japan   2 Molecular Ecology and Evolution Research Laboratory, School of Biological Sciences, Universiti Sains Malaysia, 11800 Minden, Pulau Pinang, Malaysia","@type":"Person"},{"name":"Veera Singham, G.","affiliation":"1 Synthetic Genomics Research Group, Biomass Engineering Research Division, RIKEN Center for Sustainable Resource Science (CSRS), Yokohama, Kanagawa 230-0045, Japan   4 Centre for Chemical Biology, Universiti Sains Malaysia, 11900 Bayan Lepas, Pulau Pinang, Malaysia","@type":"Person"},{"name":"Kawashima, Mika","affiliation":"1 Synthetic Genomics Research Group, Biomass Engineering Research Division, RIKEN Center for Sustainable Resource Science (CSRS), Yokohama, Kanagawa 230-0045, Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"3 Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu Chiba 292-0818, Japan","@type":"Person"},{"name":"Sato, Shusei","affiliation":"3 Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu Chiba 292-0818, Japan","@type":"Person"},{"name":"Othman, Ahmad Sofiman","affiliation":"2 Molecular Ecology and Evolution Research Laboratory, School of Biological Sciences, Universiti Sains Malaysia, 11800 Minden, Pulau Pinang, Malaysia   4 Centre for Chemical Biology, Universiti Sains Malaysia, 11900 Bayan Lepas, Pulau Pinang, Malaysia","@type":"Person"},{"name":"Matsui, Minami","affiliation":"1 Synthetic Genomics Research Group, Biomass Engineering Research Division, RIKEN Center for Sustainable Resource Science (CSRS), Yokohama, Kanagawa 230-0045, Japan","@type":"Person"}],"description":"Abstract.  Natural rubber has unique physical properties that cannot be replaced by products from other latex-producing plants or petrochemically produced synth","pageStart":"159","pageEnd":"167","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Large-scale collection of full-length cDNA and transcriptome analysis in  Hevea brasiliensis ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Large-scale collection of full-length cDNA and transcriptome analysis in  Hevea brasiliensis " />
    <meta property="og:description" content="Abstract.  Natural rubber has unique physical properties that cannot be replaced by products from other latex-producing plants or petrochemically produced synth" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/2/159/2960055" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-01-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Makita, Yuko" /><meta name="citation_author_institution" content="Synthetic Genomics Research Group, Biomass Engineering Research Division, RIKEN Center for Sustainable Resource Science (CSRS), Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author" content="Ng, Kiaw Kiaw" /><meta name="citation_author_institution" content="Synthetic Genomics Research Group, Biomass Engineering Research Division, RIKEN Center for Sustainable Resource Science (CSRS), Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author_institution" content="Molecular Ecology and Evolution Research Laboratory, School of Biological Sciences, Universiti Sains Malaysia, 11800 Minden, Pulau Pinang, Malaysia" /><meta name="citation_author" content="Veera Singham, G." /><meta name="citation_author_institution" content="Synthetic Genomics Research Group, Biomass Engineering Research Division, RIKEN Center for Sustainable Resource Science (CSRS), Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author_institution" content="Centre for Chemical Biology, Universiti Sains Malaysia, 11900 Bayan Lepas, Pulau Pinang, Malaysia" /><meta name="citation_author" content="Kawashima, Mika" /><meta name="citation_author_institution" content="Synthetic Genomics Research Group, Biomass Engineering Research Division, RIKEN Center for Sustainable Resource Science (CSRS), Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu Chiba 292-0818, Japan" /><meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu Chiba 292-0818, Japan" /><meta name="citation_author" content="Othman, Ahmad Sofiman" /><meta name="citation_author_institution" content="Molecular Ecology and Evolution Research Laboratory, School of Biological Sciences, Universiti Sains Malaysia, 11800 Minden, Pulau Pinang, Malaysia" /><meta name="citation_author_institution" content="Centre for Chemical Biology, Universiti Sains Malaysia, 11900 Bayan Lepas, Pulau Pinang, Malaysia" /><meta name="citation_author" content="Matsui, Minami" /><meta name="citation_author_institution" content="Synthetic Genomics Research Group, Biomass Engineering Research Division, RIKEN Center for Sustainable Resource Science (CSRS), Yokohama, Kanagawa 230-0045, Japan" /><meta name="citation_title" content="Large-scale collection of full-length cDNA and transcriptome analysis in Hevea brasiliensis" /><meta name="citation_firstpage" content="159" /><meta name="citation_lastpage" content="167" /><meta name="citation_doi" content="10.1093/dnares/dsw056" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2017/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Similarities and differences in rubber biochemistry among plant species;  citation_journal_title=Phytochemistry;  citation_year=2001;  citation_volume=57;  citation_pages=1123-34; " /><meta name="citation_reference" content="citation_title=Establishment of new crops for the production of natural rubber;  citation_journal_title=Trends Biotechnol;  citation_year=2007;  citation_volume=25;  citation_pages=522-9; " /><meta name="citation_reference" content="citation_title=Alternative sources of natural rubber;  citation_journal_title=Appl. Microbiol. Biotechnol.;  citation_year=2000;  citation_volume=53;  citation_pages=355-65; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of the rubber tree Hevea brasiliensis;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=75" /><meta name="citation_reference" content="citation_title=The rubber tree genome shows expansion of gene family associated with rubber biosynthesis;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=28594" /><meta name="citation_reference" content="citation_title=The rubber tree genome reveals new insights into rubber production and species adaptation;  citation_journal_title=Nat. Plants;  citation_year=2016;  citation_volume=2;  citation_pages=16073" /><meta name="citation_reference" content="citation_title=Genes expressed in the latex of Hevea brasiliensis;  citation_journal_title=Tree Physiol.;  citation_year=2000;  citation_volume=20;  citation_pages=503-10; " /><meta name="citation_reference" content="citation_title=Molecular cloning, expression and characterization of cDNA encoding cis-prenyltransferases from Hevea brasiliensis. A key factor participating in natural rubber biosynthesis;  citation_journal_title=Eur. J. Biochem;  citation_year=2003;  citation_volume=270;  citation_pages=4671-80; " /><meta name="citation_reference" content="citation_title=The 14.6 kd rubber elongation factor (Hev b 1) and 24 kd (Hev b 3) rubber particle proteins are recognized by IgE from patients with spina bifida and latex allergy;  citation_journal_title=J. Allergy Clin. Immunol;  citation_year=1996;  citation_volume=98;  citation_pages=628-39; " /><meta name="citation_reference" content="citation_title=Rubber elongation factor (REF), a major allergen component in Hevea brasiliensis latex has amyloid properties;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e48065" /><meta name="citation_reference" content="citation_title=A rubber transferase activator is necessary for natural rubber biosynthesis in dandelion;  citation_journal_title=Nat. Plants;  citation_year=2015;  citation_volume=1;  citation_pages=15048" /><meta name="citation_reference" content="citation_title=Insights into rubber biosynthesis from transcriptome analysis of Hevea brasiliensis latex;  citation_journal_title=J. Exp. Bot;  citation_year=2007;  citation_volume=58;  citation_pages=2429-40; " /><meta name="citation_reference" content="citation_title=Transcriptome sequencing of Hevea brasiliensis for development of microsatellite markers and construction of a genetic linkage map;  citation_journal_title=DNA Res;  citation_year=2011;  citation_volume=18;  citation_pages=471-82; " /><meta name="citation_reference" content="citation_title=De novo transcriptome analysis of Hevea brasiliensis tissues by RNA-seq and screening for molecular markers;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=236" /><meta name="citation_reference" content="citation_publisher=Malaysian Rubber Board (MRB, Malaysia;  citation_title=Rubber Plantation and Processing Technologies;  citation_year=2009;  citation_pages=21-68; " /><meta name="citation_reference" content="citation_title=Extraction of high-quality RNA from rubber tree leaves;  citation_journal_title=Biosci. Biotechnol. Biochem.;  citation_year=2012;  citation_volume=76;  citation_pages=1394-6; " /><meta name="citation_reference" content="citation_title=High-efficiency full-length cDNA cloning by biotinylated CAP trapper;  citation_journal_title=Genomics;  citation_year=1996;  citation_volume=37;  citation_pages=327-36; " /><meta name="citation_reference" content="citation_title=Balanced-size and long-size cloning of full-length, cap-trapped cDNAs into vectors of the novel lambda-FLC family allows enhanced gene discovery rate and functional analysis;  citation_journal_title=Genomics;  citation_year=2001;  citation_volume=77;  citation_pages=79-90; " /><meta name="citation_reference" content="citation_title=CAP3: A DNA sequence assembly program;  citation_journal_title=Genome Res;  citation_year=1999;  citation_volume=9;  citation_pages=868-77; " /><meta name="citation_reference" content="citation_title=Cutadapt removes adapter sequences from high-throughput sequencing reads;  citation_journal_title=EMBnet.journal;  citation_year=2011;  citation_volume=17;  citation_pages=10-2; " /><meta name="citation_reference" content="citation_title=The complete genome sequence of Escherichia coli K-12;  citation_journal_title=Science;  citation_year=1997;  citation_volume=277;  citation_pages=1453-62; " /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome;  citation_journal_title=Genome Biol;  citation_year=2009;  citation_volume=10;  citation_pages=R25" /><meta name="citation_reference" content="citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome;  citation_journal_title=Nat. Biotechnol;  citation_year=2011;  citation_volume=29;  citation_pages=644-52; " /><meta name="citation_reference" content="citation_title=Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies.;  citation_journal_title=Nucleic Acids Res;  citation_year=2003;  citation_volume=31;  citation_pages=5654-66; " /><meta name="citation_reference" content="citation_title=GMAP: a genomic mapping and alignment program for mRNA and EST sequences;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=1859-75; " /><meta name="citation_reference" content="citation_title=BLAT–the BLAST-like alignment tool;  citation_journal_title=Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=656-64; " /><meta name="citation_reference" content="citation_title=TopHat: discovering splice junctions with RNA-Seq;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1105-11; " /><meta name="citation_reference" content="citation_title=STAR: ultrafast universal RNA-seq aligner;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=15-21; " /><meta name="citation_reference" content="citation_title=Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks;  citation_journal_title=Nat. Protoc;  citation_year=2012;  citation_volume=7;  citation_pages=562-78; " /><meta name="citation_reference" content="citation_title=Controlling the false discovery rate: a practical and powerful approach to multiple testing;  citation_journal_title=J. Roy. Stat. Soc. Ser;  citation_year=1995;  citation_volume=B57;  citation_pages=289-300; " /><meta name="citation_reference" content="citation_title=Gene Ontology annotations and resources;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=D530-5; " /><meta name="citation_reference" content="citation_title=MOROKOSHI: transcriptome database in Sorghum bicolor;  citation_journal_title=Plant Cell Physiol;  citation_year=2015;  citation_volume=56;  citation_pages=e6" /><meta name="citation_reference" content="citation_title=Functional and expression analyses of transcripts based on full-length cDNAs of Sorghum bicolor;  citation_journal_title=DNA Res;  citation_year=2015;  citation_volume=22;  citation_pages=485-93; " /><meta name="citation_reference" content="citation_title=The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools;  citation_journal_title=Nucleic Acids Res;  citation_year=2012;  citation_volume=40;  citation_pages=D1202-10; " /><meta name="citation_reference" content="citation_title=Sequence analysis of the genome of an oil-bearing tree, Jatropha curcas L;  citation_journal_title=DNA Res;  citation_year=2011;  citation_volume=18;  citation_pages=65-76; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of the oilseed species Ricinus communis;  citation_journal_title=Nat. Biotechnol;  citation_year=2010;  citation_volume=28;  citation_pages=951-6; " /><meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome;  citation_journal_title=Nature;  citation_year=2001;  citation_volume=409;  citation_pages=860-921; " /><meta name="citation_reference" content="citation_title=Finishing the euchromatic sequence of the human genome;  citation_journal_title=Nature;  citation_year=2004;  citation_volume=431;  citation_pages=931-45; " /><meta name="citation_reference" content="citation_title=Systematic evaluation of spliced alignment programs for RNA-seq data;  citation_journal_title=Nat. Methods;  citation_year=2013;  citation_volume=10;  citation_pages=1185-91; " /><meta name="citation_reference" content="citation_publisher=CRC Press, Boca Raton, FL;  citation_title=Physiology of Rubber Tree Latex;  citation_year=1989;  citation_pages=145-164; " /><meta name="citation_reference" content="citation_title=Metabolic routes affecting rubber biosynthesis in Hevea brasiliensis latex;  citation_journal_title=J. Exp. Bot.;  citation_year=2012;  citation_volume=63;  citation_pages=1863-71; " /><meta name="citation_reference" content="citation_title=Isopentenyl diphosphate isomerase: a checkpoint to isoprenoid biosynthesis;  citation_journal_title=Biochimie;  citation_year=2012;  citation_volume=94;  citation_pages=1621-34; " /><meta name="citation_reference" content="citation_title=The Dof family of plant transcription factors;  citation_journal_title=Trends Plant Sci;  citation_year=2002;  citation_volume=7;  citation_pages=555-60; " /><meta name="citation_reference" content="citation_title=A novel family of Ca2+/calmodulin-binding proteins involved in transcriptional regulation: interaction with fsh/Ring3 class transcription activators;  citation_journal_title=Plant Mol. Biol;  citation_year=2004;  citation_volume=54;  citation_pages=549-69; " /><meta name="citation_reference" content="citation_title=DNA binding properties of the Arabidopsis floral development protein AINTEGUMENTA;  citation_journal_title=Nucleic Acids Res;  citation_year=2000;  citation_volume=28;  citation_pages=4076-82; " /><meta name="citation_reference" content="citation_title=Breeding for disease resistance in Hevea spp. - Status, potential threats, and possible strategies;  citation_journal_title=General Technical Report. PSW-GTR-240. U.S. Department of Agriculture;  citation_year=2012;  citation_pages=240-51; " /><meta name="citation_reference" content="citation_title=Molecular communications between plant heat shock responses and disease resistance;  citation_journal_title=Mol. Cells;  citation_year=2012;  citation_volume=34;  citation_pages=109-16; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/2/159/13688918/dsw056.pdf" /><meta name="description" content="Abstract.  Natural rubber has unique physical properties that cannot be replaced by products from other latex-producing plants or petrochemically produced synth" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/2/159/2960055" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/2/159/2960055" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/2/159/2960055");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsw056';
    </script>
<title>
	Large-scale collection of full-length cDNA and transcriptome analysis in Hevea brasiliensis | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2960055" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bet1PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYWtpdGEsIFl1a28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTeW50aGV0aWMgR2Vub21pY3MgUmVzZWFyY2ggR3JvdXAsIEJpb21hc3MgRW5naW5lZXJpbmcgUmVzZWFyY2ggRGl2aXNpb24sIFJJS0VOIENlbnRlciBmb3IgU3VzdGFpbmFibGUgUmVzb3VyY2UgU2NpZW5jZSAoQ1NSUyksIFlva29oYW1hLCBLYW5hZ2F3YSAyMzAtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOZywgS2lhdyBLaWF3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3ludGhldGljIEdlbm9taWNzIFJlc2VhcmNoIEdyb3VwLCBCaW9tYXNzIEVuZ2luZWVyaW5nIFJlc2VhcmNoIERpdmlzaW9uLCBSSUtFTiBDZW50ZXIgZm9yIFN1c3RhaW5hYmxlIFJlc291cmNlIFNjaWVuY2UgKENTUlMpLCBZb2tvaGFtYSwgS2FuYWdhd2EgMjMwLTAwNDUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTW9sZWN1bGFyIEVjb2xvZ3kgYW5kIEV2b2x1dGlvbiBSZXNlYXJjaCBMYWJvcmF0b3J5LCBTY2hvb2wgb2YgQmlvbG9naWNhbCBTY2llbmNlcywgVW5pdmVyc2l0aSBTYWlucyBNYWxheXNpYSwgMTE4MDAgTWluZGVuLCBQdWxhdSBQaW5hbmcsIE1hbGF5c2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVmVlcmEgU2luZ2hhbSwgRy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTeW50aGV0aWMgR2Vub21pY3MgUmVzZWFyY2ggR3JvdXAsIEJpb21hc3MgRW5naW5lZXJpbmcgUmVzZWFyY2ggRGl2aXNpb24sIFJJS0VOIENlbnRlciBmb3IgU3VzdGFpbmFibGUgUmVzb3VyY2UgU2NpZW5jZSAoQ1NSUyksIFlva29oYW1hLCBLYW5hZ2F3YSAyMzAtMDA0NSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50cmUgZm9yIENoZW1pY2FsIEJpb2xvZ3ksIFVuaXZlcnNpdGkgU2FpbnMgTWFsYXlzaWEsIDExOTAwIEJheWFuIExlcGFzLCBQdWxhdSBQaW5hbmcsIE1hbGF5c2lhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2F3YXNoaW1hLCBNaWthIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU3ludGhldGljIEdlbm9taWNzIFJlc2VhcmNoIEdyb3VwLCBCaW9tYXNzIEVuZ2luZWVyaW5nIFJlc2VhcmNoIERpdmlzaW9uLCBSSUtFTiBDZW50ZXIgZm9yIFN1c3RhaW5hYmxlIFJlc291cmNlIFNjaWVuY2UgKENTUlMpLCBZb2tvaGFtYSwgS2FuYWdhd2EgMjMwLTAwNDUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlyYWthd2EsIEhpZGVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1IENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhdG8sIFNodXNlaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCAyLTYtNyBLYXp1c2EtS2FtYXRhcmksIEtpc2FyYXp1IENoaWJhIDI5Mi0wODE4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik90aG1hbiwgQWhtYWQgU29maW1hbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1vbGVjdWxhciBFY29sb2d5IGFuZCBFdm9sdXRpb24gUmVzZWFyY2ggTGFib3JhdG9yeSwgU2Nob29sIG9mIEJpb2xvZ2ljYWwgU2NpZW5jZXMsIFVuaXZlcnNpdGkgU2FpbnMgTWFsYXlzaWEsIDExODAwIE1pbmRlbiwgUHVsYXUgUGluYW5nLCBNYWxheXNpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNlbnRyZSBmb3IgQ2hlbWljYWwgQmlvbG9neSwgVW5pdmVyc2l0aSBTYWlucyBNYWxheXNpYSwgMTE5MDAgQmF5YW4gTGVwYXMsIFB1bGF1IFBpbmFuZywgTWFsYXlzaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYXRzdWksIE1pbmFtaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN5bnRoZXRpYyBHZW5vbWljcyBSZXNlYXJjaCBHcm91cCwgQmlvbWFzcyBFbmdpbmVlcmluZyBSZXNlYXJjaCBEaXZpc2lvbiwgUklLRU4gQ2VudGVyIGZvciBTdXN0YWluYWJsZSBSZXNvdXJjZSBTY2llbmNlIChDU1JTKSwgWW9rb2hhbWEsIEthbmFnYXdhIDIzMC0wMDQ1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iTGFyZ2Utc2NhbGUgY29sbGVjdGlvbiBvZiBmdWxsLWxlbmd0aCBjRE5BIGFuZCB0cmFuc2NyaXB0b21lIGFuYWx5c2lzIGluIEhldmVhIGJyYXNpbGllbnNpcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE1OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTY3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN3MDU2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTcvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TaW1pbGFyaXRpZXMgYW5kIGRpZmZlcmVuY2VzIGluIHJ1YmJlciBiaW9jaGVtaXN0cnkgYW1vbmcgcGxhbnQgc3BlY2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGh5dG9jaGVtaXN0cnk7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NTc7ICBjaXRhdGlvbl9wYWdlcz0xMTIzLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc3RhYmxpc2htZW50IG9mIG5ldyBjcm9wcyBmb3IgdGhlIHByb2R1Y3Rpb24gb2YgbmF0dXJhbCBydWJiZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9NTIyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVybmF0aXZlIHNvdXJjZXMgb2YgbmF0dXJhbCBydWJiZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFwcGwuIE1pY3JvYmlvbC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz0zNTUtNjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgcnViYmVyIHRyZWUgSGV2ZWEgYnJhc2lsaWVuc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz03NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBydWJiZXIgdHJlZSBnZW5vbWUgc2hvd3MgZXhwYW5zaW9uIG9mIGdlbmUgZmFtaWx5IGFzc29jaWF0ZWQgd2l0aCBydWJiZXIgYmlvc3ludGhlc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9Mjg1OTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcnViYmVyIHRyZWUgZ2Vub21lIHJldmVhbHMgbmV3IGluc2lnaHRzIGludG8gcnViYmVyIHByb2R1Y3Rpb24gYW5kIHNwZWNpZXMgYWRhcHRhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQbGFudHM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTE2MDczIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXMgZXhwcmVzc2VkIGluIHRoZSBsYXRleCBvZiBIZXZlYSBicmFzaWxpZW5zaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZWUgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz01MDMtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjbG9uaW5nLCBleHByZXNzaW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGNETkEgZW5jb2RpbmcgY2lzLXByZW55bHRyYW5zZmVyYXNlcyBmcm9tIEhldmVhIGJyYXNpbGllbnNpcy4gQSBrZXkgZmFjdG9yIHBhcnRpY2lwYXRpbmcgaW4gbmF0dXJhbCBydWJiZXIgYmlvc3ludGhlc2lzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FdXIuIEouIEJpb2NoZW07ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MjcwOyAgY2l0YXRpb25fcGFnZXM9NDY3MS04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIDE0LjYga2QgcnViYmVyIGVsb25nYXRpb24gZmFjdG9yIChIZXYgYiAxKSBhbmQgMjQga2QgKEhldiBiIDMpIHJ1YmJlciBwYXJ0aWNsZSBwcm90ZWlucyBhcmUgcmVjb2duaXplZCBieSBJZ0UgZnJvbSBwYXRpZW50cyB3aXRoIHNwaW5hIGJpZmlkYSBhbmQgbGF0ZXggYWxsZXJneTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQWxsZXJneSBDbGluLiBJbW11bm9sOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTk4OyAgY2l0YXRpb25fcGFnZXM9NjI4LTM5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SdWJiZXIgZWxvbmdhdGlvbiBmYWN0b3IgKFJFRiksIGEgbWFqb3IgYWxsZXJnZW4gY29tcG9uZW50IGluIEhldmVhIGJyYXNpbGllbnNpcyBsYXRleCBoYXMgYW15bG9pZCBwcm9wZXJ0aWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTQ4MDY1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBydWJiZXIgdHJhbnNmZXJhc2UgYWN0aXZhdG9yIGlzIG5lY2Vzc2FyeSBmb3IgbmF0dXJhbCBydWJiZXIgYmlvc3ludGhlc2lzIGluIGRhbmRlbGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQbGFudHM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTE1MDQ4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5zaWdodHMgaW50byBydWJiZXIgYmlvc3ludGhlc2lzIGZyb20gdHJhbnNjcmlwdG9tZSBhbmFseXNpcyBvZiBIZXZlYSBicmFzaWxpZW5zaXMgbGF0ZXg7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEV4cC4gQm90OyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9MjQyOS00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBzZXF1ZW5jaW5nIG9mIEhldmVhIGJyYXNpbGllbnNpcyBmb3IgZGV2ZWxvcG1lbnQgb2YgbWljcm9zYXRlbGxpdGUgbWFya2VycyBhbmQgY29uc3RydWN0aW9uIG9mIGEgZ2VuZXRpYyBsaW5rYWdlIG1hcDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RE5BIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTQ3MS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byB0cmFuc2NyaXB0b21lIGFuYWx5c2lzIG9mIEhldmVhIGJyYXNpbGllbnNpcyB0aXNzdWVzIGJ5IFJOQS1zZXEgYW5kIHNjcmVlbmluZyBmb3IgbW9sZWN1bGFyIG1hcmtlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTIzNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1NYWxheXNpYW4gUnViYmVyIEJvYXJkIChNUkIsIE1hbGF5c2lhOyAgY2l0YXRpb25fdGl0bGU9UnViYmVyIFBsYW50YXRpb24gYW5kIFByb2Nlc3NpbmcgVGVjaG5vbG9naWVzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fcGFnZXM9MjEtNjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dHJhY3Rpb24gb2YgaGlnaC1xdWFsaXR5IFJOQSBmcm9tIHJ1YmJlciB0cmVlIGxlYXZlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qmlvc2NpLiBCaW90ZWNobm9sLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03NjsgIGNpdGF0aW9uX3BhZ2VzPTEzOTQtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1lZmZpY2llbmN5IGZ1bGwtbGVuZ3RoIGNETkEgY2xvbmluZyBieSBiaW90aW55bGF0ZWQgQ0FQIHRyYXBwZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MzI3LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYWxhbmNlZC1zaXplIGFuZCBsb25nLXNpemUgY2xvbmluZyBvZiBmdWxsLWxlbmd0aCwgY2FwLXRyYXBwZWQgY0ROQXMgaW50byB2ZWN0b3JzIG9mIHRoZSBub3ZlbCBsYW1iZGEtRkxDIGZhbWlseSBhbGxvd3MgZW5oYW5jZWQgZ2VuZSBkaXNjb3ZlcnkgcmF0ZSBhbmQgZnVuY3Rpb25hbCBhbmFseXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9Nzc7ICBjaXRhdGlvbl9wYWdlcz03OS05MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0FQMzogQSBETkEgc2VxdWVuY2UgYXNzZW1ibHkgcHJvZ3JhbTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ODY4LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DdXRhZGFwdCByZW1vdmVzIGFkYXB0ZXIgc2VxdWVuY2VzIGZyb20gaGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmcgcmVhZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVNQm5ldC5qb3VybmFsOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTAtMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGNvbXBsZXRlIGdlbm9tZSBzZXF1ZW5jZSBvZiBFc2NoZXJpY2hpYSBjb2xpIEstMTI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9Mjc3OyAgY2l0YXRpb25fcGFnZXM9MTQ1My02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VWx0cmFmYXN0IGFuZCBtZW1vcnktZWZmaWNpZW50IGFsaWdubWVudCBvZiBzaG9ydCBETkEgc2VxdWVuY2VzIHRvIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9UjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVsbC1sZW5ndGggdHJhbnNjcmlwdG9tZSBhc3NlbWJseSBmcm9tIFJOQS1TZXEgZGF0YSB3aXRob3V0IGEgcmVmZXJlbmNlIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBCaW90ZWNobm9sOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9NjQ0LTUyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZpbmcgdGhlIEFyYWJpZG9wc2lzIGdlbm9tZSBhbm5vdGF0aW9uIHVzaW5nIG1heGltYWwgdHJhbnNjcmlwdCBhbGlnbm1lbnQgYXNzZW1ibGllcy47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTMxOyAgY2l0YXRpb25fcGFnZXM9NTY1NC02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R01BUDogYSBnZW5vbWljIG1hcHBpbmcgYW5kIGFsaWdubWVudCBwcm9ncmFtIGZvciBtUk5BIGFuZCBFU1Qgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTE4NTktNzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJMQVTigJN0aGUgQkxBU1QtbGlrZSBhbGlnbm1lbnQgdG9vbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz02NTYtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvcEhhdDogZGlzY292ZXJpbmcgc3BsaWNlIGp1bmN0aW9ucyB3aXRoIFJOQS1TZXE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTEwNS0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U1RBUjogdWx0cmFmYXN0IHVuaXZlcnNhbCBSTkEtc2VxIGFsaWduZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MTUtMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhbCBnZW5lIGFuZCB0cmFuc2NyaXB0IGV4cHJlc3Npb24gYW5hbHlzaXMgb2YgUk5BLXNlcSBleHBlcmltZW50cyB3aXRoIFRvcEhhdCBhbmQgQ3VmZmxpbmtzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFByb3RvYzsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NTYyLTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db250cm9sbGluZyB0aGUgZmFsc2UgZGlzY292ZXJ5IHJhdGU6IGEgcHJhY3RpY2FsIGFuZCBwb3dlcmZ1bCBhcHByb2FjaCB0byBtdWx0aXBsZSB0ZXN0aW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBSb3kuIFN0YXQuIFNvYy4gU2VyOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPUI1NzsgIGNpdGF0aW9uX3BhZ2VzPTI4OS0zMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgT250b2xvZ3kgYW5ub3RhdGlvbnMgYW5kIHJlc291cmNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz1ENTMwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1PUk9LT1NISTogdHJhbnNjcmlwdG9tZSBkYXRhYmFzZSBpbiBTb3JnaHVtIGJpY29sb3I7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGwgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT01NjsgIGNpdGF0aW9uX3BhZ2VzPWU2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuY3Rpb25hbCBhbmQgZXhwcmVzc2lvbiBhbmFseXNlcyBvZiB0cmFuc2NyaXB0cyBiYXNlZCBvbiBmdWxsLWxlbmd0aCBjRE5BcyBvZiBTb3JnaHVtIGJpY29sb3I7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz00ODUtOTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBBcmFiaWRvcHNpcyBJbmZvcm1hdGlvbiBSZXNvdXJjZSAoVEFJUik6IGltcHJvdmVkIGdlbmUgYW5ub3RhdGlvbiBhbmQgbmV3IHRvb2xzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPUQxMjAyLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jZSBhbmFseXNpcyBvZiB0aGUgZ2Vub21lIG9mIGFuIG9pbC1iZWFyaW5nIHRyZWUsIEphdHJvcGhhIGN1cmNhcyBMOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9NjUtNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiB0aGUgb2lsc2VlZCBzcGVjaWVzIFJpY2ludXMgY29tbXVuaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTk1MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbml0aWFsIHNlcXVlbmNpbmcgYW5kIGFuYWx5c2lzIG9mIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00MDk7ICBjaXRhdGlvbl9wYWdlcz04NjAtOTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaW5pc2hpbmcgdGhlIGV1Y2hyb21hdGljIHNlcXVlbmNlIG9mIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT00MzE7ICBjaXRhdGlvbl9wYWdlcz05MzEtNDU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5c3RlbWF0aWMgZXZhbHVhdGlvbiBvZiBzcGxpY2VkIGFsaWdubWVudCBwcm9ncmFtcyBmb3IgUk5BLXNlcSBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xMTg1LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q1JDIFByZXNzLCBCb2NhIFJhdG9uLCBGTDsgIGNpdGF0aW9uX3RpdGxlPVBoeXNpb2xvZ3kgb2YgUnViYmVyIFRyZWUgTGF0ZXg7ICBjaXRhdGlvbl95ZWFyPTE5ODk7ICBjaXRhdGlvbl9wYWdlcz0xNDUtMTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRhYm9saWMgcm91dGVzIGFmZmVjdGluZyBydWJiZXIgYmlvc3ludGhlc2lzIGluIEhldmVhIGJyYXNpbGllbnNpcyBsYXRleDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gRXhwLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTYzOyAgY2l0YXRpb25fcGFnZXM9MTg2My03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvcGVudGVueWwgZGlwaG9zcGhhdGUgaXNvbWVyYXNlOiBhIGNoZWNrcG9pbnQgdG8gaXNvcHJlbm9pZCBiaW9zeW50aGVzaXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoaW1pZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05NDsgIGNpdGF0aW9uX3BhZ2VzPTE2MjEtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBEb2YgZmFtaWx5IG9mIHBsYW50IHRyYW5zY3JpcHRpb24gZmFjdG9yczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIFBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9NTU1LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIGZhbWlseSBvZiBDYTIrL2NhbG1vZHVsaW4tYmluZGluZyBwcm90ZWlucyBpbnZvbHZlZCBpbiB0cmFuc2NyaXB0aW9uYWwgcmVndWxhdGlvbjogaW50ZXJhY3Rpb24gd2l0aCBmc2gvUmluZzMgY2xhc3MgdHJhbnNjcmlwdGlvbiBhY3RpdmF0b3JzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz01NDktNjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBiaW5kaW5nIHByb3BlcnRpZXMgb2YgdGhlIEFyYWJpZG9wc2lzIGZsb3JhbCBkZXZlbG9wbWVudCBwcm90ZWluIEFJTlRFR1VNRU5UQTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9Mjg7ICBjaXRhdGlvbl9wYWdlcz00MDc2LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CcmVlZGluZyBmb3IgZGlzZWFzZSByZXNpc3RhbmNlIGluIEhldmVhIHNwcC4gLSBTdGF0dXMsIHBvdGVudGlhbCB0aHJlYXRzLCBhbmQgcG9zc2libGUgc3RyYXRlZ2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXJhbCBUZWNobmljYWwgUmVwb3J0LiBQU1ctR1RSLTI0MC4gVS5TLiBEZXBhcnRtZW50IG9mIEFncmljdWx0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fcGFnZXM9MjQwLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2xlY3VsYXIgY29tbXVuaWNhdGlvbnMgYmV0d2VlbiBwbGFudCBoZWF0IHNob2NrIHJlc3BvbnNlcyBhbmQgZGlzZWFzZSByZXNpc3RhbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIENlbGxzOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9MTA5LTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjQvMi8xNTkvMTM2ODg5MTgvZHN3MDU2LnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBOYXR1cmFsIHJ1YmJlciBoYXMgdW5pcXVlIHBoeXNpY2FsIHByb3BlcnRpZXMgdGhhdCBjYW5ub3QgYmUgcmVwbGFjZWQgYnkgcHJvZHVjdHMgZnJvbSBvdGhlciBsYXRleC1wcm9kdWNpbmcgcGxhbnRzIG9yIHBldHJvY2hlbWljYWxseSBwcm9kdWNlZCBzeW50aCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9Mjk2MDA1NSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTKKelyJo1DDP5uxTCAcZre8B8oWA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f159%2f2960055" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f159%2f2960055" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=97104" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/2/2/m_dnares_24_2cover.png?Expires=2147483647&amp;Signature=C8sLyIyfepNN-Oy5BR2xLoVJLx2W1jQBt~48a2lQ~mjys3EWTAKP5k2kV~t7b8sG~Qk1B4iVdz8BINFP5bR-Hc0gvwqekGHM8qavh8kDaLAfN~qNxrDj7NNeLHJ-IURMi5v1ATu23u5wB01wP27BHknbA3W9JgGnq6AVGPqeLfhK7AJcTzOYzKpGYwaedh5crkYXYHp~9GCcHuTjoInf3sKaYC2Ew-HnuA-yIL45j8ocXrNQuIxJV8rFXwx1Tt0b7I1ZJg09R4-7m-GZB9vPgdehgAXUMUNQ4gDatBY22xUt9COqnlb7RZKNUXFDo7Qe0LNJgSHwNUOJvsh-SLpXkg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/2">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="78055166">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055166">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055168">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055168">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055174">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055174">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055191">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055191">3. Results and Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055222">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055222">4. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055226">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055226">Data Availability</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055228">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055228">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="78055236">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#78055236">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/2/143/2870506">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/2/169/2870503">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Large-scale collection of full-length cDNA and transcriptome analysis in <em>Hevea brasiliensis</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuko Makita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Synthetic Genomics Research Group, Biomass Engineering Research Division, RIKEN Center for Sustainable Resource Science (CSRS), Yokohama, Kanagawa 230-0045, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuko+Makita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Makita Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Makita Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuko Makita</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kiaw Kiaw Ng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Synthetic Genomics Research Group, Biomass Engineering Research Division, RIKEN Center for Sustainable Resource Science (CSRS), Yokohama, Kanagawa 230-0045, Japan</div><div class="aff"><div class="label -label">2</div>Molecular Ecology and Evolution Research Laboratory, School of Biological Sciences, Universiti Sains Malaysia, 11800 Minden, Pulau Pinang, Malaysia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kiaw+Kiaw+Ng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ng K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ng K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kiaw Kiaw Ng</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">G. Veera Singham</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Synthetic Genomics Research Group, Biomass Engineering Research Division, RIKEN Center for Sustainable Resource Science (CSRS), Yokohama, Kanagawa 230-0045, Japan</div><div class="aff"><div class="label -label">4</div>Centre for Chemical Biology, Universiti Sains Malaysia, 11900 Bayan Lepas, Pulau Pinang, Malaysia</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=G.+Veera+Singham">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Veera Singham G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Veera Singham G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">G. Veera Singham</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mika Kawashima</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Synthetic Genomics Research Group, Biomass Engineering Research Division, RIKEN Center for Sustainable Resource Science (CSRS), Yokohama, Kanagawa 230-0045, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mika+Kawashima">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawashima M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawashima M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mika Kawashima</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu Chiba 292-0818, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div>Kazusa DNA Research Institute, 2-6-7 Kazusa-Kamatari, Kisarazu Chiba 292-0818, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ahmad Sofiman Othman</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Molecular Ecology and Evolution Research Laboratory, School of Biological Sciences, Universiti Sains Malaysia, 11800 Minden, Pulau Pinang, Malaysia</div><div class="aff"><div class="label -label">4</div>Centre for Chemical Biology, Universiti Sains Malaysia, 11900 Bayan Lepas, Pulau Pinang, Malaysia</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw056-cor1" data-open="dsw056-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: +604-653-3181. Fax: +604-656-5125. Email: <a href="mailto:sofiman@usm.my" target="_blank">sofiman@usm.my</a> (A.S.O.); Tel: +81-45-503-9585. Fax: +81-45-503-9586. Email: <a href="mailto:minami@riken.jp" target="_blank">minami@riken.jp</a> (M.M.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ahmad+Sofiman+Othman">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Othman A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Othman A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ahmad Sofiman Othman</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Minami Matsui</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Synthetic Genomics Research Group, Biomass Engineering Research Division, RIKEN Center for Sustainable Resource Science (CSRS), Yokohama, Kanagawa 230-0045, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsw056-cor1" data-open="dsw056-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel: +604-653-3181. Fax: +604-656-5125. Email: <a href="mailto:sofiman@usm.my" target="_blank">sofiman@usm.my</a> (A.S.O.); Tel: +81-45-503-9585. Fax: +81-45-503-9586. Email: <a href="mailto:minami@riken.jp" target="_blank">minami@riken.jp</a> (M.M.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Minami+Matsui">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Matsui M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Matsui M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Minami Matsui</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw056-FM1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw056-FM2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present address: SS; Graduate School of Life Sciences, Tohoku University, 2-1-1 Katahira, Aoba-ku, Sendai 980-8577, Japan.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsw056-FM3"><div class="fn"><div class="footnote-content">Edited by Dr. Kazuo Shinozaki</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 159–167, <a href='https://doi.org/10.1093/dnares/dsw056'>https://doi.org/10.1093/dnares/dsw056</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 January 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">29 June 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">18 November 2016</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2960055" href="/dnaresearch/article-pdf/24/2/159/13688918/dsw056.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yuko Makita, Kiaw Kiaw Ng, G. Veera Singham, Mika Kawashima, Hideki Hirakawa, Shusei Sato, Ahmad Sofiman Othman, Minami Matsui;  Large-scale collection of full-length cDNA and transcriptome analysis in <em>Hevea brasiliensis</em>, <em>DNA Research</em>, Volume 24, Issue 2, 1 April 2017, Pages 159–167, <a href='https://doi.org/10.1093/dnares/dsw056'>https://doi.org/10.1093/dnares/dsw056</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2960055?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960055?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960055?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960055?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960055?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960055?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2960055?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Large-scale%20collection%20of%20full-length%20cDNA%20and%20transcriptome%20analysis%20in%20%26lt%3Bem%26gt%3BHevea%20brasiliensis%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2017-01-27&amp;volumeNum=24&amp;issueNum=2&amp;author=Makita%2C%20Yuko%3B%20Ng%2C%20Kiaw%20Kiaw&amp;startPage=159&amp;endPage=167&amp;contentId=10.1093%2Fdnares%2Fdsw056&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f159%2f2960055" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=78055166 id="78055166" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Natural rubber has unique physical properties that cannot be replaced by products from other latex-producing plants or petrochemically produced synthetic rubbers. Rubber from <em>Hevea brasiliensis</em> is the main commercial source for this natural rubber that has a <em>cis</em>-polyisoprene configuration. For sustainable production of enough rubber to meet demand elucidation of the molecular mechanisms involved in the production of latex is vital. To this end, we firstly constructed rubber full-length cDNA libraries of RRIM 600 cultivar and sequenced around 20,000 clones by the Sanger method and over 15,000 contigs by Illumina sequencer. With these data, we updated around 5,500 gene structures and newly annotated around 9,500 transcription start sites. Second, to elucidate the rubber biosynthetic pathways and their transcriptional regulation, we carried out tissue- and cultivar-specific RNA-Seq analysis. By using our recently published genome sequence, we confirmed the expression patterns of the rubber biosynthetic genes. Our data suggest that the cytoplasmic mevalonate (MVA) pathway is the main route for isoprenoid biosynthesis in latex production. In addition to the well-studied polymerization factors, we suggest that rubber elongation factor 8 (REF8) is a candidate factor in <em>cis</em>-polyisoprene biosynthesis. We have also identified 39 transcription factors that may be key regulators in latex production. Expression profile analysis using two additional cultivars, RRIM 901 and PB 350, via an RNA-Seq approach revealed possible expression differences between a high latex-yielding cultivar and a disease-resistant cultivar.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="RNA-Seq">RNA-Seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;rubber biosynthesis&quot;">rubber biosynthesis</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="latex">latex</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcription factor&quot;">transcription factor</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;full-length cDNA&quot;">full-length cDNA</a></div>                            <h2 scrollto-destination=78055168 id="78055168" class="section-title" data-legacy-id=SEC1>1. Introduction</h2>
<p>Natural rubber is an indispensable biomass material produced from <em>Hevea brasiliensis</em>, or the para rubber tree, and its <em>cis</em>-1,4-polyisoprene polymer has superior characteristics that cannot be replaced by synthetic rubber produced from petroleum oils.<a reveal-id="dsw056-B1" data-open="dsw056-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsw056-B2" data-open="dsw056-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> The demand for it is increasing in accordance with advances in road and air transport, and medical devices.<a reveal-id="dsw056-B3" data-open="dsw056-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a></p><p>In Malaysia rubber has been bred for better productivity and quality. It has been studied for about 90 years since the first rubber tree was imported. Rubber breeders have been crossing a few varieties to achieve high productivity and quality. In 2013, the first draft genome sequence of <em>H. brasiliensis</em> (Willd.) Muell.-Arg. RRIM 600 was reported using 43-fold sequence coverage data.<a reveal-id="dsw056-B4" data-open="dsw056-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> Recently, we reported a much more comprehensive RRIM 600 genome assembly based on ∼155-fold coverage<a reveal-id="dsw056-B5" data-open="dsw056-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and, in addition, an assembly of Reyan7-33-97, another rubber cultivar genome, has been released that has 94-fold coverage.<a reveal-id="dsw056-B6" data-open="dsw056-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> The burgeoning research interest in the para rubber tree reflects its significance as an important commodity crop globally.</p><p>The rubber tree has very specialized cells in the bark around the vascular bundle called laticifer cells and latex is produced from these cells. It is composed of rubber serum and rubber particles. The particles are composed of lipid monolayers and it is suggested that there are several proteins localized on the surface of these. They include the rubber elongation factor (REF), small rubber particle protein (SRPP), and <em>cis</em>-prenyl transferase (CPT), and they are highly induced in latex compared with leaf and bark.<a reveal-id="dsw056-B7" data-open="dsw056-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsw056-B8" data-open="dsw056-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> REF and SRPP are the cause of rubber allergy.<a reveal-id="dsw056-B9" data-open="dsw056-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsw056-B10" data-open="dsw056-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> CPT functions as a polymerizing enzyme for the condensation reaction of isopentenyl diphosphate to form polyisoprene. Recently, the Nogo-B receptor (or CPTL) was reported as a component of the rubber particle<a reveal-id="dsw056-B11" data-open="dsw056-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> but the complete picture of how natural rubber is produced is not clear.</p><p>To further improve knowledge on latex production, we performed transcriptome analyses of full-length cDNA (FL-cDNA) and RNA-Seq. The sequence information derived from FL-cDNA libraries provides exact transcribed regions, such as transcription units, splicing variants and transcription start sites (TSSs). We constructed rubber FL-cDNA libraries and collected 19,487 clones as bioresources. Using the Sanger and high throughput sequences from the rubber FL-cDNA libraries, we confirmed the transcripts of 7,704 loci and updated 5,466 with 22 novel loci. To increase the coverage of expressed genes, we further added public expressed sequence tags (ESTs),<a reveal-id="dsw056-B12 dsw056-B13 dsw056-B14" data-open="dsw056-B12 dsw056-B13 dsw056-B14" class="link link-ref link-reveal xref-bibr"><sup>12–14</sup></a> confirmed the expression of 23,790 loci and updated the annotation of 17,201 loci.</p><p>RNA-Seq reveals the presence and quantity of genome-wide gene expression in biological samples. To predict the candidate genes related to rubber biosynthesis, we performed transcriptome analysis on four tissues (latex, bark, petiole and leaf) and on three rubber cultivars (RRIM 600, PB 350 and RRIM 901). RRIM 600 is an average latex-yielding cultivar with a production of ∼1,350 kg of latex per hectare per year in estates over 20 years old. This cultivar is susceptible to disease caused by the pathogenic fungal <em>Phytophthora</em> spp. On the other hand, PB 350 is a latex-timber (∼1,862 kg/ha/year), disease-resistant cultivar and RRIM 901 produces a higher latex yield (∼1,971 kg/ha/year) when compared with RRIM 600.<a reveal-id="dsw056-B15" data-open="dsw056-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> We also focus on the transcriptional regulatory factors of rubber biosynthesis and list transcription factors (TFs) that are highly expressed only in latex.</p>                            <h2 scrollto-destination=78055174 id="78055174" class="section-title" >2. Materials and Methods</h2>
                            <h3 scrollto-destination=78055175 id="78055175" class="section-title" >2.1. RNA isolation and construction of two FL-cDNA libraries</h3>
<div class="block-child-p">Samples from leaf, petiole, bark and/or latex of RRIM 600 were collected from 2-, 5-, and 15-year-old trees located at Liman Plantation, Kedah, Malaysia (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&amp;Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). They were collected in the morning during the rainy season (November 2012 to January 2013). Total RNA from each tissue was extracted using a protocol optimized for rubber tree.<a reveal-id="dsw056-B16" data-open="dsw056-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> RNA samples were purified with Qiagen RNeasy plant mini kits (Qiagen Inc., Hilden, Germany) following the manufacturer's instructions in order to obtain high quality RNA. We mixed seven samples (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&amp;Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) to make two FL-cDNA libraries. Construction of the libraries from poly(A)<sup>+ </sup>RNA was performed using the biotinylated CAP-trapper method.<a reveal-id="dsw056-B17" data-open="dsw056-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> The double-stranded FL-cDNAs were digested with <em>Bam</em>HI and <em>Xho</em>I and inserted into the <em>Bam</em>HI and <em>Sal</em>I sites of a λ-FLC-III vector.<a reveal-id="dsw056-B18" data-open="dsw056-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> The phage library was amplified on solid plates and phage particles were subsequently eluted with SM buffer. Plasmids were generated from the amplified phage library by <em>in vitro</em> excision. They were transformed into DH10B™ T1 phage-resistant <em>Escherichia</em><em>coli</em> and eluted with SOC medium with 13% glycerol. The titer of the two FL-cDNA plasmid libraries were (no. 1) 1.3 × 10<sup>5</sup> cfu/ml and (no 2) 7.6 × 10<sup>5</sup> cfu/ml (<a reveal-id="dsw056-F1" data-open="dsw056-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). When we checked the lengths of cDNAs in the two libraries with electrophoresis (data not shown), library no. 1 contained various lengths of cDNAs whereas library no. 2 had relatively short cDNAs. Since library no. 1 was contaminated with <em>E. coli</em> genomic DNA, we decided to use library no. 2 for Sanger sequencing of individual clones, and both libraries for Illumina sequencing (<a reveal-id="dsw056-F1" data-open="dsw056-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). For Sanger sequencing, we randomly selected 19,968 clones (52 plates of a 384 format) and cultured them in LB medium with 7% glycerol at 30 °C overnight. <div data-id="dsw056-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/m_dsw056f1.png?Expires=2147483647&amp;Signature=ZgTrUWgsE~rGSSkqECIBVtR03ljfHiW9UpNHEqSnINZfw6IKxs1PWhOaem0OrBpz5asna3CXz93vF8oY-aNZE~7lD3To-YWb-YiDbOxHwNn4vl47QCQ59IqOn0Bqba6R59ahNEBzlQj3yRlwflEKJGOYQAsQ7mr4psNNoUSJPmF-bU3wHzH9IObOj~hyDX3Ea4rNpFnNr17I9NCfsPfdtdid8unFMy887CE-eK2copSyjtpGYU5R0KaNYmmEluhsZ02p6UphU7dMExEMRl6tlVMJsrYueFQ~yhj0OKauu~DgkLzgditp-9Mnu6F65UkAR-4Rn3LFwVT~ZdqGPYmhDw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flowchart of the steps in the construction of the FL-cDNA libraries, sequencing and data analysis." path-from-xml="dsw056f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055176/dsw056f1.tif" path-from-xml="dsw056f1.tif" target="_blank">View large</a><a section="78055176" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056f1.png?Expires=2147483647&amp;Signature=xU~LB9dlrE0S0XHfwQSn~s~FSgrbPp4bP-CKVCpYBxIvZLQfNOW1PnJjKH1WTmtFAsu2zjsIdkBm6qAtxlIXabmP5uN5TQ~no9SopGMZD5KSNfpHyQ5yYAZqz7bw~wGcL3aqgz6FLWM~o0b8jKikJ8GN5mnpEyjpZBs2OHSF7biZewnyQRtce~TTmZvUNTpcsy-d04OnjcAB-vHMY4fJQ8YIBnaNJzkX8968RtF3UQmzQQPNJYhCGDlahrMLxtjWrvndu8BT51Cd6kmMUXI6xf5OU6Dx8acJ8A65ul8V126y6JL13qlZyle-f4xuvg0T6lNIkSwT~fRBODc-M8ul2w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055176&amp;ar=2960055&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw056f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flowchart of the steps in the construction of the FL-cDNA libraries, sequencing and data analysis.</p></div></div><div content-id="dsw056-F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/m_dsw056f1.png?Expires=2147483647&amp;Signature=ZgTrUWgsE~rGSSkqECIBVtR03ljfHiW9UpNHEqSnINZfw6IKxs1PWhOaem0OrBpz5asna3CXz93vF8oY-aNZE~7lD3To-YWb-YiDbOxHwNn4vl47QCQ59IqOn0Bqba6R59ahNEBzlQj3yRlwflEKJGOYQAsQ7mr4psNNoUSJPmF-bU3wHzH9IObOj~hyDX3Ea4rNpFnNr17I9NCfsPfdtdid8unFMy887CE-eK2copSyjtpGYU5R0KaNYmmEluhsZ02p6UphU7dMExEMRl6tlVMJsrYueFQ~yhj0OKauu~DgkLzgditp-9Mnu6F65UkAR-4Rn3LFwVT~ZdqGPYmhDw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flowchart of the steps in the construction of the FL-cDNA libraries, sequencing and data analysis." path-from-xml="dsw056f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055176/dsw056f1.tif" path-from-xml="dsw056f1.tif" target="_blank">View large</a><a section="78055176" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056f1.png?Expires=2147483647&amp;Signature=xU~LB9dlrE0S0XHfwQSn~s~FSgrbPp4bP-CKVCpYBxIvZLQfNOW1PnJjKH1WTmtFAsu2zjsIdkBm6qAtxlIXabmP5uN5TQ~no9SopGMZD5KSNfpHyQ5yYAZqz7bw~wGcL3aqgz6FLWM~o0b8jKikJ8GN5mnpEyjpZBs2OHSF7biZewnyQRtce~TTmZvUNTpcsy-d04OnjcAB-vHMY4fJQ8YIBnaNJzkX8968RtF3UQmzQQPNJYhCGDlahrMLxtjWrvndu8BT51Cd6kmMUXI6xf5OU6Dx8acJ8A65ul8V126y6JL13qlZyle-f4xuvg0T6lNIkSwT~fRBODc-M8ul2w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055176&amp;ar=2960055&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw056f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flowchart of the steps in the construction of the FL-cDNA libraries, sequencing and data analysis.</p></div></div></div>                            <h3 scrollto-destination=78055177 id="78055177" class="section-title" >2.2. FL-cDNA library no. 2: Sanger sequencing and quality control</h3>
<p>End sequencing was performed by the Sanger method using ABI 3730xl capillary sequencers (Applied Biosystems, Foster City, CA, USA). The M13Fw (-20) primer (5′-GTAAAACGACGGCCAG-3′) and the M13Rvdt primer (5′-GCGGATAACAATTTCACACAGG-3′) were used for forward and reverse sequencing, respectively. The cDNA library was sequenced by the Sanger method from the 5′ end and a total of 19,532 sequences were generated that had a Phred quality of ≥20. To reduce the number of clones to sequence from 3′ end, we remove the duplicated sequences that show blast <em>e</em>-value ≤ 1e-100 and identity ≥95%. 5,395 clones were selected and Sanger sequenced from 3′ end. Of which, 4,915 sequences had a Phred quality of  ≥20. We applied 19,532 5′ ESTs and 4,915 3′ ESTs to SeqClean (<a class="link link-uri" href="http://sourceforge.net/projects/seqclean/" target="_blank">http://sourceforge.net/projects/seqclean/</a>), which validates and trims DNA sequences. Then, 19,487 5′ ESTs and 4,480 3′ ESTs were assembled by CAP3<a reveal-id="dsw056-B19" data-open="dsw056-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> with the following parameters (MinDistance = 100 bp and MaxDistance = 15000 bp).</p>                            <h3 scrollto-destination=78055179 id="78055179" class="section-title" >2.3. FL-cDNA library Nos. 1 and 2: Illumina sequencing and assembly</h3>
<p>Total DNA from two FL-cDNA libraries was prepared using the Illumina TruSeq DNA LT Sample Prep Kit (Illumina, USA) following the manufacturer’s instructions. Sequencing was performed on the HiSeq 2000 (Illumina, USA) with 100 bp read length. Adapter sequences and poly-A were trimmed with cutadapt (v1.3),<a reveal-id="dsw056-B20" data-open="dsw056-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> and FLC-III vector<a reveal-id="dsw056-B18" data-open="dsw056-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and <em>E. coli</em> genome<a reveal-id="dsw056-B21" data-open="dsw056-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> sequences were removed using Bowtie 1.0.0<a reveal-id="dsw056-B22" data-open="dsw056-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> mapping software for no. 1 library . After these trimming and filtering processes, we assembled ESTs using Trinity (r2013-02-25) software<a reveal-id="dsw056-B23" data-open="dsw056-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> with default parameters.</p>                            <h3 scrollto-destination=78055181 id="78055181" class="section-title" >2.4. Genome annotation with the FL-cDNA sequences and public ESTs</h3>
<p>We applied both the Sanger sequenced FL-cDNA ESTs and Illumina sequenced data to the PASA (Programme to Assemble Spliced Alignments) pipeline<a reveal-id="dsw056-B24" data-open="dsw056-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> and updated the gene models. In this pipeline, we used two alignment software programmes, GMAP<a reveal-id="dsw056-B25" data-open="dsw056-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> and BLAT,<a reveal-id="dsw056-B26" data-open="dsw056-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> with default parameters and we accepted only transcripts with over 75% coverage (–MIN_PERCENT_ALIGNED = 75) and over 95% identity (–MIN_AVG_PER_ID = 95). To increase the gene coverage, we collected three public EST datasets<a reveal-id="dsw056-B12 dsw056-B13 dsw056-B14" data-open="dsw056-B12 dsw056-B13 dsw056-B14" class="link link-ref link-reveal xref-bibr"><sup>12–14</sup></a>: (i) Chow <em>et al.</em><a reveal-id="dsw056-B12" data-open="dsw056-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> obtained 9,860 ESTs (NCBI accession: EC600050–EC609910) of RRIM 600 latex; (ii) Triwitayakorn <em>et al.</em><a reveal-id="dsw056-B13" data-open="dsw056-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> sequenced 2,311,497 reads from the vegetative shoot apical tissue of RRIM 600 (DDBJ: DRA000170) and (iii) Salgado <em>et al.</em><a reveal-id="dsw056-B14" data-open="dsw056-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> extracted RNAs from 33 organs and seedlings from open pollination of RRIM 600 and obtained 19,708 assembled sequences. We applied these three publicly accessible EST sources to the PASA pipeline with the same flow as our analysis.</p>                            <h3 scrollto-destination=78055183 id="78055183" class="section-title" >2.5. TSS annotation with 5′ FL-cDNA sequences</h3>
<p>We counted TSSs using 17,667 5′ FL-cDNA sequences that were successfully mapped to the genome with over 75% coverage and over 95% identity. As a first step, we eliminated redundancy and determined 9,486 TSSs. These were located at 5,482 loci. More specifically, 8,864 TSSs were located 1000 bp upstream or inside 5,042 genic regions, 54 were located on the reverse strand of 42 genic regions (candidates for antisense transcripts) and the other 618 TSSs were clustered in 398 intergenic regions that were defined using a simple distance-based approach with a maximum allowed distance of 1,000 bp between two neighboring TSSs.</p>                            <h3 scrollto-destination=78055185 id="78055185" class="section-title" >2.6. RNA-Seq preparation and sequence analysis</h3>
<p>Sampling, total RNA extraction and preparation of RNA-Seq libraries were performed using the same methods as for the FL-cDNA libraries (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). These libraries were high-throughput sequenced on the Illumina HiSeq 2000 platform using directed paired-end technology (2 × 100 bp).</p><p>To obtain more accurate results, we applied two mapping software programmes, TopHat v2.0.13<a reveal-id="dsw056-B27" data-open="dsw056-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> and STAR version 020201.<a reveal-id="dsw056-B28" data-open="dsw056-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> Cufflinks v2.2.1<a reveal-id="dsw056-B29" data-open="dsw056-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> assembled the mapped transcripts and calculated FPKM values for each sample.</p><p>For quality control, we drew a clustering dendrogram and a distance matrix to identify outlier replicates using an R module of CummeRbund<a reveal-id="dsw056-B29" data-open="dsw056-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Except for RRIM 901 latex, each two replicates showed the closest Jensen-Shannon (JS) distance in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figure S2</a></span>. Since the JS distance of the two RRIM 901 replicates (JS distance = 0.16) was smaller than the JS distances in other tissue combinations (JS distance &gt; 0.2), we decided to take both data for subsequent analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>b).</p><p>For gene set enrichment tests, we carried out a hypergeometric test to identify significantly enriched TF families or GO categories as follows:</p><p><span class="inline-formula"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><mi>P</mi><mo> </mo><mo>−</mo><mo> </mo><mtext>value</mtext><mo> </mo><mo>=</mo><mo> </mo><mn>1</mn><mo> </mo><mo>−</mo><mo> </mo><msubsup><mo>∑</mo><mrow><mi>i</mi><mo>=</mo><mn>0</mn></mrow><mi>m</mi></msubsup><mrow><mfrac><mrow><msubsup><mo stretchy="false">(</mo><mi>j</mi><mi>M</mi></msubsup><mo stretchy="false">)</mo><msubsup><mo stretchy="false">(</mo><mrow><mi>n</mi><mo>−</mo><mi>i</mi></mrow><mrow><mi>N</mi><mo>−</mo><mi>M</mi></mrow></msubsup><mo stretchy="false">)</mo></mrow><mrow><msubsup><mo stretchy="false">(</mo><mi>n</mi><mi>N</mi></msubsup><mo stretchy="false">)</mo></mrow></mfrac></mrow></mrow></math>⁠</span>, where N is the total number of all genes with TF family (or GO category) annotation, n is the number of differentially expressed genes (DEGs) in N, M is the number of genes in a given TF family (or GO category), and m is the number of DEGs in M. For this statistical calculation, we used SciPy (<a class="link link-uri" href="http://www.scipy.org/" target="_blank">http://www.scipy.org/</a>), which is a scientific library for Python. We then controlled the proportion of false positives by calculating the false discovery rate (FDR) corresponding to each <em>P</em>-value.<a reveal-id="dsw056-B29" data-open="dsw056-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> The TF families (or GO categories) with a <em>q</em>-value of ≤ 0.05 were defined as significantly enriched. The TF family and GO annotation were carried out in our previous work.<a reveal-id="dsw056-B5" data-open="dsw056-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a></p>                            <h2 scrollto-destination=78055191 id="78055191" class="section-title" >3. Results and Discussion</h2>
                            <h3 scrollto-destination=78055192 id="78055192" class="section-title" >3.1. Cloning and data analysis of Sanger sequencing of rubber FL-cDNA library</h3>
<p>FL-cDNA sequences are important for correct annotation and identification of authentic transcripts from plant tissues. We constructed two FL-cDNA libraries from <em>H. brasiliensis</em> RRIM 600 whose draft genome was determined recently.<a reveal-id="dsw056-B5" data-open="dsw056-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> Since library no. 1 was contaminated with <em>E. coli</em> genomic DNA, we used library no. 2 for constructing FL-cDNA clones and applied both libraries to Illumina sequencing (<a reveal-id="dsw056-F1" data-open="dsw056-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>).</p><p>After quality control, we obtained 19,487 FL-cDNA Sanger sequences from the 5′ ends (<a reveal-id="dsw056-T1" data-open="dsw056-T1" class="link link-reveal link-table xref-fig">Table 1</a>). We removed 5′ EST duplicates and selected 5,395 FL-cDNA clones. 4,840 clones were Sanger sequenced from their 3′ ends with Phred quality ≥ 20. A total of 24,327 FL-cDNA clones based on 5′/3′ ESTs were assembled into 4,590 contigs with CAP3, a DNA sequence assembly programme<a reveal-id="dsw056-B31" data-open="dsw056-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> that also output 3,570 singletons. Of the 4,590 contigs, 3,011 (65.6%) were reached from both ends. We applied the PASA pipeline to annotate 8,160 unigenes (the sum of contigs and singletons) and they were mapped to 6,883 clustered positions. They were annotated as 3,877 known genes with 116 new isoforms and 258 structural gene modifications. <div content-id="dsw056-T1" class="table-modal"><div class="table-wrap"><div id="dsw056-T1" data-id="dsw056-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Summary of sequence resources of FL-cDNAs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>No. of sequences </th><th>Min. length (bp) </th><th>Max. length (bp) </th><th>Mean length (bp) </th><th>Mapping ratio (by PASA) </th></tr></thead><tbody><tr><td rowspan="2">Sanger FL-cDNA </td><td>5′ ESTs </td><td char=".">19,487 </td><td char=".">102 </td><td char=".">920 </td><td char=".">667.4 </td><td rowspan="2"> </td></tr><tr><td>3′ ESTs </td><td char=".">4,840 </td><td char=".">111 </td><td char=".">876 </td><td char=".">668.7 </td></tr><tr><td colspan="2">Sanger FL-cDNA contigs<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">4,590 </td><td char=".">111 </td><td char=".">2,026 </td><td char=".">868.4 </td><td char="."> </td></tr><tr><td colspan="2">Sanger FL-cDNA singletons </td><td char=".">3,570 </td><td char=".">103 </td><td char=".">884 </td><td char=".">655.7 </td><td char="."> </td></tr><tr><td colspan="2">Assembled Illumina FL-cDNA contigs </td><td char=".">15,683 </td><td char=".">201 </td><td char=".">11,217 </td><td char=".">966.3 </td><td char="."> </td></tr><tr><td colspan="2">Merged FL-cDNA (contigs) </td><td char=".">23,843 </td><td char=".">103 </td><td char=".">11,217 </td><td char=".">893.9 </td><td char=".">91.4% </td></tr></tbody></table></div><div content-id="dsw056-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>No. of sequences </th><th>Min. length (bp) </th><th>Max. length (bp) </th><th>Mean length (bp) </th><th>Mapping ratio (by PASA) </th></tr></thead><tbody><tr><td rowspan="2">Sanger FL-cDNA </td><td>5′ ESTs </td><td char=".">19,487 </td><td char=".">102 </td><td char=".">920 </td><td char=".">667.4 </td><td rowspan="2"> </td></tr><tr><td>3′ ESTs </td><td char=".">4,840 </td><td char=".">111 </td><td char=".">876 </td><td char=".">668.7 </td></tr><tr><td colspan="2">Sanger FL-cDNA contigs<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">4,590 </td><td char=".">111 </td><td char=".">2,026 </td><td char=".">868.4 </td><td char="."> </td></tr><tr><td colspan="2">Sanger FL-cDNA singletons </td><td char=".">3,570 </td><td char=".">103 </td><td char=".">884 </td><td char=".">655.7 </td><td char="."> </td></tr><tr><td colspan="2">Assembled Illumina FL-cDNA contigs </td><td char=".">15,683 </td><td char=".">201 </td><td char=".">11,217 </td><td char=".">966.3 </td><td char="."> </td></tr><tr><td colspan="2">Merged FL-cDNA (contigs) </td><td char=".">23,843 </td><td char=".">103 </td><td char=".">11,217 </td><td char=".">893.9 </td><td char=".">91.4% </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>The number of reads per contig was 4.5.</p></div></div></div></div></div><div class="table-wrap"><div id="dsw056-T1" data-id="dsw056-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Summary of sequence resources of FL-cDNAs</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>No. of sequences </th><th>Min. length (bp) </th><th>Max. length (bp) </th><th>Mean length (bp) </th><th>Mapping ratio (by PASA) </th></tr></thead><tbody><tr><td rowspan="2">Sanger FL-cDNA </td><td>5′ ESTs </td><td char=".">19,487 </td><td char=".">102 </td><td char=".">920 </td><td char=".">667.4 </td><td rowspan="2"> </td></tr><tr><td>3′ ESTs </td><td char=".">4,840 </td><td char=".">111 </td><td char=".">876 </td><td char=".">668.7 </td></tr><tr><td colspan="2">Sanger FL-cDNA contigs<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">4,590 </td><td char=".">111 </td><td char=".">2,026 </td><td char=".">868.4 </td><td char="."> </td></tr><tr><td colspan="2">Sanger FL-cDNA singletons </td><td char=".">3,570 </td><td char=".">103 </td><td char=".">884 </td><td char=".">655.7 </td><td char="."> </td></tr><tr><td colspan="2">Assembled Illumina FL-cDNA contigs </td><td char=".">15,683 </td><td char=".">201 </td><td char=".">11,217 </td><td char=".">966.3 </td><td char="."> </td></tr><tr><td colspan="2">Merged FL-cDNA (contigs) </td><td char=".">23,843 </td><td char=".">103 </td><td char=".">11,217 </td><td char=".">893.9 </td><td char=".">91.4% </td></tr></tbody></table></div><div content-id="dsw056-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th> </th><th>No. of sequences </th><th>Min. length (bp) </th><th>Max. length (bp) </th><th>Mean length (bp) </th><th>Mapping ratio (by PASA) </th></tr></thead><tbody><tr><td rowspan="2">Sanger FL-cDNA </td><td>5′ ESTs </td><td char=".">19,487 </td><td char=".">102 </td><td char=".">920 </td><td char=".">667.4 </td><td rowspan="2"> </td></tr><tr><td>3′ ESTs </td><td char=".">4,840 </td><td char=".">111 </td><td char=".">876 </td><td char=".">668.7 </td></tr><tr><td colspan="2">Sanger FL-cDNA contigs<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td char=".">4,590 </td><td char=".">111 </td><td char=".">2,026 </td><td char=".">868.4 </td><td char="."> </td></tr><tr><td colspan="2">Sanger FL-cDNA singletons </td><td char=".">3,570 </td><td char=".">103 </td><td char=".">884 </td><td char=".">655.7 </td><td char="."> </td></tr><tr><td colspan="2">Assembled Illumina FL-cDNA contigs </td><td char=".">15,683 </td><td char=".">201 </td><td char=".">11,217 </td><td char=".">966.3 </td><td char="."> </td></tr><tr><td colspan="2">Merged FL-cDNA (contigs) </td><td char=".">23,843 </td><td char=".">103 </td><td char=".">11,217 </td><td char=".">893.9 </td><td char=".">91.4% </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>The number of reads per contig was 4.5.</p></div></div></div></div></p><p>We also confirmed the functional bias of our FL-cDNA clones with GO slim<a reveal-id="dsw056-B31" data-open="dsw056-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). In most of the GO terms, the percentage of the cloned FL-cDNAs was around 10% of all GO annotation. The following 11 terms were enriched with a FDR adjusted <em>q</em>-value ≤ 0.05 in biological function: generation of precursor metabolites and energy (GO:0006091), biological_process (GO:0008150), photosynthesis (GO:0015979), catabolic process (GO:0009056), cellular homeostasis (GO:0019725), response to abiotic stimulus (GO:0009628), response to stress (GO:0006950), cellular component organization (GO:0016043), secondary metabolic process (GO:0019748). We infer that the enriched ‘response to abiotic stimulus’, ‘response to stress’, ‘response to external stimulus’ terms are related to our sampling of rubber tapping and latex. Also, ‘pollen-pistil interactions’ (GO:0009875) showed the lowest percentage and matched our sampling.</p>                            <h3 scrollto-destination=78055196 id="78055196" class="section-title" >3.2. Data analysis of Illumina-sequenced rubber FL-cDNA library</h3>
<p>Additionally to the Sanger-sequenced FL-cDNA clones, we sequenced our two FL-cDNA libraries with Illumina HiSeq 2000 and obtained 169,017,764 raw reads. After quality control, we used 81,624,108 trimmed and filtered read pairs and assembled them with Trinity.<a reveal-id="dsw056-B23" data-open="dsw056-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> As a result, we obtained 15,683 assembled contigs. When we compared the Sanger- and Illumina-sequenced FL-cDNA contigs, it was found that 99% of the Sanger unigenes overlapped with the Illumina contigs.</p><p>We mapped all 23,843 FL-cDNA unigenes to the newly updated rubber genome,<a reveal-id="dsw056-B5" data-open="dsw056-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and 91.4% of them successfully mapped on the new draft genome while 80.4% mapped on the previous draft sequence (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). This high percentage suggests that the NGS sequenced data are efficient at collecting authentic transcripts. The results also suggest that the newest RRIM 600 draft genome has more comprehensive structural gene information than the previous one.</p>                            <h3 scrollto-destination=78055199 id="78055199" class="section-title" >3.3. Structural gene modification with RIKEN FL-cDNA libraries and public EST data</h3>
<p>We applied the PASA pipeline<a reveal-id="dsw056-B24" data-open="dsw056-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> to update the current gene annotation. The PASA improved the 5,466 known genes and the 22 newly defined genes including four isoforms (<a reveal-id="dsw056-T2" data-open="dsw056-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Details of the 5,466 modifications are as follows: 165 genes were fused into 78 genes and 3 genes were divided into 6 genes; 238 isoforms against 218 loci were newly annotated; 2,605 loci were improved in their 5′ and/or 3′ UTRs and a total of 2,901 annotated protein sequences were updated. <div content-id="dsw056-T2" class="table-modal"><div class="table-wrap"><div id="dsw056-T2" data-id="dsw056-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Statistics of updated structural gene annotation in <em>H. brasiliensis.</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Categories </th><th>RIKEN data </th><th>+ public ESTs </th></tr></thead><tbody><tr><td><em>H. brasiliensis</em> gene annotation </td><td colspan="2">84,440 </td></tr><tr><td>No. of loci detected the expression </td><td char=".">7,704 </td><td char=".">23,790 </td></tr><tr><td>Total no. of loci updated with the PASA analysis </td><td char=".">5,466 </td><td char=".">17,201 </td></tr><tr><td>No. of fused genes </td><td char=".">78 </td><td char=".">236 </td></tr><tr><td>FL-cDNAs split single gene into multiple genes </td><td char=".">6 </td><td char=".">6 </td></tr><tr><td>No. of genes modified in UTRs </td><td char=".">2,605 </td><td char=".">7,525 </td></tr><tr><td>No. of genes modified in exon structure </td><td char=".">1,998 </td><td char=".">6,087 </td></tr><tr><td>No. of gene with newly defined isoforms </td><td char=".">218 </td><td char=".">1,967 </td></tr><tr><td>Proteins with CDS modification </td><td char=".">2,901 </td><td char=".">7,645 </td></tr><tr><td>Novel genes </td><td char=".">22 </td><td char=".">22 </td></tr><tr><td>Novel isoforms in the novel genes </td><td char=".">4 </td><td char=".">5 </td></tr></tbody></table></div><div content-id="dsw056-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Categories </th><th>RIKEN data </th><th>+ public ESTs </th></tr></thead><tbody><tr><td><em>H. brasiliensis</em> gene annotation </td><td colspan="2">84,440 </td></tr><tr><td>No. of loci detected the expression </td><td char=".">7,704 </td><td char=".">23,790 </td></tr><tr><td>Total no. of loci updated with the PASA analysis </td><td char=".">5,466 </td><td char=".">17,201 </td></tr><tr><td>No. of fused genes </td><td char=".">78 </td><td char=".">236 </td></tr><tr><td>FL-cDNAs split single gene into multiple genes </td><td char=".">6 </td><td char=".">6 </td></tr><tr><td>No. of genes modified in UTRs </td><td char=".">2,605 </td><td char=".">7,525 </td></tr><tr><td>No. of genes modified in exon structure </td><td char=".">1,998 </td><td char=".">6,087 </td></tr><tr><td>No. of gene with newly defined isoforms </td><td char=".">218 </td><td char=".">1,967 </td></tr><tr><td>Proteins with CDS modification </td><td char=".">2,901 </td><td char=".">7,645 </td></tr><tr><td>Novel genes </td><td char=".">22 </td><td char=".">22 </td></tr><tr><td>Novel isoforms in the novel genes </td><td char=".">4 </td><td char=".">5 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsw056-T2" data-id="dsw056-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Statistics of updated structural gene annotation in <em>H. brasiliensis.</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Categories </th><th>RIKEN data </th><th>+ public ESTs </th></tr></thead><tbody><tr><td><em>H. brasiliensis</em> gene annotation </td><td colspan="2">84,440 </td></tr><tr><td>No. of loci detected the expression </td><td char=".">7,704 </td><td char=".">23,790 </td></tr><tr><td>Total no. of loci updated with the PASA analysis </td><td char=".">5,466 </td><td char=".">17,201 </td></tr><tr><td>No. of fused genes </td><td char=".">78 </td><td char=".">236 </td></tr><tr><td>FL-cDNAs split single gene into multiple genes </td><td char=".">6 </td><td char=".">6 </td></tr><tr><td>No. of genes modified in UTRs </td><td char=".">2,605 </td><td char=".">7,525 </td></tr><tr><td>No. of genes modified in exon structure </td><td char=".">1,998 </td><td char=".">6,087 </td></tr><tr><td>No. of gene with newly defined isoforms </td><td char=".">218 </td><td char=".">1,967 </td></tr><tr><td>Proteins with CDS modification </td><td char=".">2,901 </td><td char=".">7,645 </td></tr><tr><td>Novel genes </td><td char=".">22 </td><td char=".">22 </td></tr><tr><td>Novel isoforms in the novel genes </td><td char=".">4 </td><td char=".">5 </td></tr></tbody></table></div><div content-id="dsw056-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Categories </th><th>RIKEN data </th><th>+ public ESTs </th></tr></thead><tbody><tr><td><em>H. brasiliensis</em> gene annotation </td><td colspan="2">84,440 </td></tr><tr><td>No. of loci detected the expression </td><td char=".">7,704 </td><td char=".">23,790 </td></tr><tr><td>Total no. of loci updated with the PASA analysis </td><td char=".">5,466 </td><td char=".">17,201 </td></tr><tr><td>No. of fused genes </td><td char=".">78 </td><td char=".">236 </td></tr><tr><td>FL-cDNAs split single gene into multiple genes </td><td char=".">6 </td><td char=".">6 </td></tr><tr><td>No. of genes modified in UTRs </td><td char=".">2,605 </td><td char=".">7,525 </td></tr><tr><td>No. of genes modified in exon structure </td><td char=".">1,998 </td><td char=".">6,087 </td></tr><tr><td>No. of gene with newly defined isoforms </td><td char=".">218 </td><td char=".">1,967 </td></tr><tr><td>Proteins with CDS modification </td><td char=".">2,901 </td><td char=".">7,645 </td></tr><tr><td>Novel genes </td><td char=".">22 </td><td char=".">22 </td></tr><tr><td>Novel isoforms in the novel genes </td><td char=".">4 </td><td char=".">5 </td></tr></tbody></table></div></div></p><p>Since our FL-cDNA sequences cover only 9.1% (7,704 genes out of 84,440 predicted rubber genes) of all the structural genes in <em>H. brasiliensis</em>, we used publicly available ESTs from three publications<a reveal-id="dsw056-B7 dsw056-B8 dsw056-B9" data-open="dsw056-B7 dsw056-B8 dsw056-B9" class="link link-ref link-reveal xref-bibr"><sup>7–9</sup></a> and compared these with the new rubber genome. As a result, the coverage was raised to 28.2% (23,790 genes) and the annotation of 17,201 genes was improved (<a reveal-id="dsw056-T2" data-open="dsw056-T2" class="link link-reveal link-table xref-fig">Table 2</a>). We think that the excessive numbers of current gene predictions cause the limited gene coverage (see Section 3.5).</p>                            <h3 scrollto-destination=78055202 id="78055202" class="section-title" >3.4. TSSs from FL-5′ ESTs</h3>
<p>The PASA pipeline can use complete FL-cDNA sequences but not the 5′ TSS information from 5′ ESTs. For this reason, we separately analyzed rubber TSSs using 5′ ESTs. We mapped 19,487 5′ ESTs of FL-cDNA clones to the rubber genome. 17,667 5′ ESTs were successfully mapped and this corresponds to 9,486 unique TSSs. Only 55 of our TSSs exactly matched with the current TSS annotation. This very limited overlap was similar to the result obtained with Sorghum FL-cDNAs.<a reveal-id="dsw056-B32" data-open="dsw056-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a><sup>,</sup><a reveal-id="dsw056-B33" data-open="dsw056-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> The 9,486 TSSs were located at 5,482 loci, which contain 5,042 annotated genes, 42 antisense and 398 intergenic regions. Of these 398, 28 mapped to rubber draft scaffolds that have no annotated genes.</p>                            <h3 scrollto-destination=78055204 id="78055204" class="section-title" >3.5. The gene coverage of multi-transcriptomic data</h3>
<p>It is estimated that there are 84,440 genes in the new RRIM 600 draft genome.<a reveal-id="dsw056-B5" data-open="dsw056-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> This number is much larger compared with the 27,416 genes in <em>Arabidopsis thaliana</em>,<a reveal-id="dsw056-B34" data-open="dsw056-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> 21,573 genes in <em>Jatropha curcas</em><a reveal-id="dsw056-B35" data-open="dsw056-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and 31,344 genes in <em>Ricinus communis</em> (castor bean).<a reveal-id="dsw056-B36" data-open="dsw056-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Since there are more and more transcriptome data, false-positive gene predictions result in fewer problems than false-negative predictions. When we map RNA-Seq data to a genome, we can ignore the false-positives because they show 0 expression. On the other hand, problems still arise relating to quality and prediction of new genes using only RNA-Seq data. Therefore, the current number may be the result of reducing false-negative predictions. It will be greatly reduced after further annotation as was the case in the human genome (currently 26,000 genes).<a reveal-id="dsw056-B37" data-open="dsw056-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a><sup>,</sup><a reveal-id="dsw056-B38" data-open="dsw056-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Transcriptomic data such as FL-cDNAs, ESTs and RNA-Seq are key information required to distinguish functional genes and pseudogenes.</p><p>We used six samples (leaf, petiole, bark and latex from RRIM 600 and latex from RRIM 901 and PB350) for RNA-Seq (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). In total, we obtained 118.8 Gb of RNA-Seq data (an average of 48 million reads) from the six samples with two replicates. These data were analyzed using two mapping software programmes, TopHat<a reveal-id="dsw056-B26" data-open="dsw056-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> and STAR<a reveal-id="dsw056-B27" data-open="dsw056-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a>. With our RNA-Seq data, STAR showed ∼10% better mapping ratio than TopHat (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Engström <em>et al.</em><a reveal-id="dsw056-B39" data-open="dsw056-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> reported that TopHat showed lower tolerance for mismatches than STAR. As we consider that the current rubber draft genome still has many mismatches, we decided to use the mapping results from STAR for further analyses.</p><p>As a result of RNA-Seq, we confirmed that 36,544 genes were expressed in at least one of our samples with the criterion of FPKM &gt; 0. Combining these with the 23,790 expressed genes obtained from the FL-cDNAs and ESTs, a total of 38,986 expressed genes (46.2% of the annotated genes) are validated in our analysis.</p>                            <h3 scrollto-destination=78055208 id="78055208" class="section-title" >3.6. Tissue- and cultivar-specific gene expression based on RNA-Seq</h3>
<div class="block-child-p">To investigate the important genes involved in rubber biosynthesis, we compared RNA expression of leaves, petioles, bark and latex in 15-year-old RRIM 600 plants. We found 24,821 genes were commonly expressed in all tissues and 745 genes were uniquely expressed in latex (<a reveal-id="dsw056-F2" data-open="dsw056-F2" class="link link-reveal link-table xref-fig">Fig. 2a</a>). GO enrichment analysis of the 745 latex-specific genes showed that enzymatic activities (transferase activity (GO:0016740), kinase activity (GO:0016301), enzyme regulatory activity (GO:0030234), hydrolase activity (GO:0016787)) and binding (GO:0005488) were significantly enriched with a FDR adjusted <em>q</em>-value &lt; 0.05. The total number of expressed genes is at least 4,000 fewer in latex than in other tissues (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&amp;Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). <div data-id="dsw056-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/m_dsw056f2.png?Expires=2147483647&amp;Signature=cDQP3rfXQM4lcukurwxO2mQjC45aqKbAqug5dMolntz-tF4soDHAXi7zMCDtBqrvnCDBrdqiAzGmvaQAWI~HFXNXyDePkuxdQaAP09KZnJhNzs691e9MpHEJEkg9VYFv69A1X6z4Zh30s0iZaMHt1e4DW~uo2GPuLcV2njTNjHeYmJ3dCVEfC63DQ2lBtbAybtwR9XtQ11enDlJWyQsiq4u8yb9iG6CMTrt08Eb7m2Js2Ai39bEYV4ogrQbeFguIYdcDIKoYQ~5iF-YpxqmbaTLaWBYcTSluJKjOjhkTE-9Gp~WDJvSI8qs~XW~BMvUoI9-ow~Q-k9lESnd1qAOPtA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(a) Numbers of tissue-specific and co-expressed genes in RRIM 600 latex, bark, petiole and leaf. (b) Numbers of clone-specific and co-expressed genes in PB 350, RRIM 901 and RRIM 600 latexes. We defined expressed genes with FPKM &gt; 0." path-from-xml="dsw056f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055209/dsw056f2.tif" path-from-xml="dsw056f2.tif" target="_blank">View large</a><a section="78055209" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056f2.png?Expires=2147483647&amp;Signature=Krs39awFrSbjR1qYFF-kWJ5daLLKNcMvybEhL-VKzamvogDDiUQG2Bzy~5-n1kfThex3kxNGqY4WVnCtR5-b5YmSYSXbzx4OcJLmRzwBYf9~KlKIXJAsKTn5P-zJAQOcGWP6kzZXYusAno5~fbcVvNqINYi50M0Mfe32yr9OvOQnQpMcQ7GLpuNg0iDO3DGnIlbiVJ~KJUtu~xhOpQ98zcFQQNGR9WrEy91hpVT6tAPJC1n7L10TKIijc85EBWs35gdCyNcFXIftf3lbncFLuXCUvmWTE0WAdTpxhVtcZK3lv1bd5TNrQyEuxtn--OiqQ4xrxFKuWc41hg3Mf3K5yQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055209&amp;ar=2960055&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw056f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) Numbers of tissue-specific and co-expressed genes in RRIM 600 latex, bark, petiole and leaf. <strong>(b)</strong> Numbers of clone-specific and co-expressed genes in PB 350, RRIM 901 and RRIM 600 latexes. We defined expressed genes with FPKM &gt; 0.</p></div></div><div content-id="dsw056-F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/m_dsw056f2.png?Expires=2147483647&amp;Signature=cDQP3rfXQM4lcukurwxO2mQjC45aqKbAqug5dMolntz-tF4soDHAXi7zMCDtBqrvnCDBrdqiAzGmvaQAWI~HFXNXyDePkuxdQaAP09KZnJhNzs691e9MpHEJEkg9VYFv69A1X6z4Zh30s0iZaMHt1e4DW~uo2GPuLcV2njTNjHeYmJ3dCVEfC63DQ2lBtbAybtwR9XtQ11enDlJWyQsiq4u8yb9iG6CMTrt08Eb7m2Js2Ai39bEYV4ogrQbeFguIYdcDIKoYQ~5iF-YpxqmbaTLaWBYcTSluJKjOjhkTE-9Gp~WDJvSI8qs~XW~BMvUoI9-ow~Q-k9lESnd1qAOPtA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(a) Numbers of tissue-specific and co-expressed genes in RRIM 600 latex, bark, petiole and leaf. (b) Numbers of clone-specific and co-expressed genes in PB 350, RRIM 901 and RRIM 600 latexes. We defined expressed genes with FPKM &gt; 0." path-from-xml="dsw056f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055209/dsw056f2.tif" path-from-xml="dsw056f2.tif" target="_blank">View large</a><a section="78055209" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056f2.png?Expires=2147483647&amp;Signature=Krs39awFrSbjR1qYFF-kWJ5daLLKNcMvybEhL-VKzamvogDDiUQG2Bzy~5-n1kfThex3kxNGqY4WVnCtR5-b5YmSYSXbzx4OcJLmRzwBYf9~KlKIXJAsKTn5P-zJAQOcGWP6kzZXYusAno5~fbcVvNqINYi50M0Mfe32yr9OvOQnQpMcQ7GLpuNg0iDO3DGnIlbiVJ~KJUtu~xhOpQ98zcFQQNGR9WrEy91hpVT6tAPJC1n7L10TKIijc85EBWs35gdCyNcFXIftf3lbncFLuXCUvmWTE0WAdTpxhVtcZK3lv1bd5TNrQyEuxtn--OiqQ4xrxFKuWc41hg3Mf3K5yQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=78055209&amp;ar=2960055&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsw056f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(a) Numbers of tissue-specific and co-expressed genes in RRIM 600 latex, bark, petiole and leaf. <strong>(b)</strong> Numbers of clone-specific and co-expressed genes in PB 350, RRIM 901 and RRIM 600 latexes. We defined expressed genes with FPKM &gt; 0.</p></div></div></div><p>In Malaysia rubber has been improved by conventional breeding programmes, which have created cultivars with high and low latex yields. We have analyzed two other cultivars, RRIM 901 that has a high latex yield and PB 350 that yields a moderate amount of latex but has improved disease resistance and high timber production. Our results show that 24,081 genes were commonly expressed and 800-1,100 genes were specifically expressed in these cultivars (<a reveal-id="dsw056-F2" data-open="dsw056-F2" class="link link-reveal link-table xref-fig">Fig. 2b</a>).</p><p>We looked into whether the 745 latex-specific genes in RRIM 600 are also expressed in other cultivars. Although 96.7% of the expressed genes in RRIM 600 were shared with RRIM 901 and/or PB 350 (<a reveal-id="dsw056-F2" data-open="dsw056-F2" class="link link-reveal link-table xref-fig">Fig. 2b</a>), only 50.5% of the latex-specific genes in RRIM 600 were shared with RRIM 901 and/or PB 350. In addition, although we expected that the highly expressed latex-specific genes in RRIM 600 would be similar in all the cultivars, we did not find that this was the case (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). It is unlikely that essential genes for latex synthesis are different between cultivars. Tissue-specificity might be less important to elucidate complete pathway of latex synthesis.</p><p>To search for the key factors that confer higher latex yield in RRIM 901, we examined genes whose expression in latex is over 10 times higher in RRIM 901 than in RRIM 600 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). At this threshold, most of the genes are categorized as having unknown function. When we considered genes whose expression is five times higher in RRIM 901 compared with RRIM 600 we found they belonged to several GO classes: growth (GO:0040007), cellular homeostasis (GO:0019725), lipid binding (GO:0008289), biological process (GO:0008150) and protein metabolic process (GO:0019538). These genes are candidates for the factor or factors that confer high latex production in RRIM 901.</p>                            <h3 scrollto-destination=78055213 id="78055213" class="section-title" >3.7. Expression profile of isoprenoid biosynthesis genes</h3>
<div class="block-child-p">Natural rubber is a biopolymer consisting of <em>cis</em>-1,4-polyisoprene which is formed from isopentenyl diphosphate (IPP). IPP is synthesized by the cytosolic mevalonate (MVA) pathway from acetyl-CoA.<a reveal-id="dsw056-B40" data-open="dsw056-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> As an alternative route, the plastidic 2-C-methyl-D-erythritol-4-phosphate (MEP) pathway from glyceraldehyde 3-phosphate (G3P) and pyruvate has also been demonstrated.<a reveal-id="dsw056-B41" data-open="dsw056-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Using RNA-Seq technology, we have confirmed the expression of 37 genes in the MVA pathway and 21 in the MEP pathway, and they are defined in our previous genome article<a reveal-id="dsw056-B5" data-open="dsw056-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> (<a reveal-id="dsw056-F3" data-open="dsw056-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). As a comparison of their expression in RRIM 600 latex, genes in the MVA pathway (the average FPKM is 80.3) are more highly expressed than the genes in the MEP pathway (the average FPKM is 10.0). Also, the average FPKM value of CMK (4-(cytidine 5-diphospho)-2-C-methyl-D-erythritol kinase) in the MEP pathway was only 0.2 in the latex. This result is the same in both the PB 350 and the RRM 901 latex. Our data suggest that MVA is the main IPP synthetic pathway in latex. Also, the expression levels of the genes in the MEP pathway are higher in non-latex samples (leaf, petiole, and bark) compared with latex. This high expression in non-latex samples may be associated with non-latex terpenoid biosynthesis. IPP isomerase is known as a key rate-limiting step of terpenoid biosynthesis.<a reveal-id="dsw056-B42" data-open="dsw056-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> Our data indicate that the expression level of the IPP isomerase gene in latex is also low and so, in order to enhance latex production, this gene can be an ideal candidate for modification of expression and activity. <div data-id="dsw056-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/m_dsw056f3.png?Expires=2147483647&Signature=B8osS8VPUWcgvDaO719pTkE8t51LQHN7W~6VCKHqOcGWsF2Xb3UvyADySdfNi1XT4WtbbaTLa7ZuKmiS9vpR9urgHlGbWXkHgrqReqS-WliGjldUV6gmOdG9LYbD9eB8thuDt5R0pBH-WT6Yg1lerkOFQnAkf04ESzoiH88-5AMjdk-ShBDc8itV9k4KtPmwAonIAM2mu8cSnr-D5sIPUHY3xKz9sa3tmtuyM6rfgVsQocEoKOGVVHiUpHbYPYqsL2FDwcBd6z6uqV1DlciVZYBN9~Sz6FSUbStkb3a4UTwjuY5ZDcZAcHeBIeHgfnWNWTT8kxib9rYpQU1n2n-YNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Latex biosynthesis pathway and heat maps of scaled expression values (log10(FPKM + 1)) for genes associated with latex biosynthesis. The lines of each heat map represent duplicate genes and the columns represent samples: PB 350 latex, RRIM 901 latex, RRIM 600 latex, RRIM 600 bark, RRIM 600 petiole and RRIM 600 leaf from the left. 37 MVA pathway, 21 MEP pathway and 18 prenyl-PP biosynthetic genes, 7 CPTs, 1 CPTL, 9 REFs and 8 SRPP genes are defined in our draft genome article.5 In the heat maps, genes were sorted by the expression value in RRIM 600 latex. These figures were drawn by an R package of gplots. Abbreviations in the MVA pathway: PD, pyruvate dehydrogenase; PDC, pyruvate dehydrogenase complex; DLD, dihydrolipoamide dehydrogenase; AACT, acetyl-CoA acetyltransferase; HMGS, hydroxymethyglutaryl coenzyme A synthase; HMG-CoA, hydroxymethylglutaryl coenzyme A; HMGR, hydroxymethyglutaryl coenzyme A reductase; MK, mevalonate kinase; mevalonate-OP, mevalonate monophosphate; PMK, phosphomevalonate kinase; mavalonate-OPP, mevalonate diphosphate; MDC, diphosphomevalonate decarboxylase; in the plastidic MEP pathway: DXS, 1-deoxy-D-xylulose 5-phosphate synthase; DXP, 1-deoxy-D-xylulose-5-phosphate; DXR, 1-deoxy-D-xylulose 5-phosphate reductoisomerase; MEP, 2-C-methyl-D-erythritol 4-phosphate; MCT, 2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase; CDP-ME, 2-C-methyl-D-erythritol-4-phosphate; CMK, 4-(cytidine 5′-diphospho)-2-C-methyl-D-erythritol kinase; CDP-ME2P, 4-(cytidine 5′-diphospho)-2-C-methyl-D-erythritol; MDS, 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase; MEcPP, 2-C-methyl-D-erythritol 2,4-cyclodiphosphate; HDS, 4-hydroxy-3-methylbut-2-enyl diphosphate synthase; HMB-PP, 4-hydroxy-3-methyl-but-2-en-1-yl diphosphate; HDR, 4-hydroxy-3-methylbut-2-enyl diphosphate reductase. Corresponding gene IDs are as follow: PD#1:Hb_001638_290, PD#2:Hb_006420_080, PD#3:Hb_132840_110, PD#4:Hb_001500_040, PD#5:Hb_116349_120, PD#6:Hb_001123_160, PD#7:Hb_027506_040, PD#8:Hb_003397_030, PDC#1:Hb_000800_090, PDC#2:Hb_005306_180, PDC#3:Hb_002477_290, PDC#4:Hb_000110_310, PDC#5:Hb_003411_090, PDC#6:Hb_000207_300, PDC#7:Hb_002876_240, DLD#1:Hb_001723_060, DLD#2:Hb_002217_100, DLD#3:Hb_000645_180, DLD#4:Hb_012305_100, AACT#1:Hb_000613_200, AACT#2:Hb_000184_040, AACT#3: Hb_000377_090, AACT#4:Hb_000795_060, HMGS#1:Hb_003207_110, HMGS#2:Hb_000107_440, HMGS#3:Hb_004032_440, HMGS#4:Hb_001153_150, HMGS# 5:Hb_014497_120, HMGS#6:Hb_142382_010, HMGS#7:Hb_000116_530, MK#1:Hb_035273_030, MK#2:Hb_127194_010, MK#3:Hb_002042_080, PMK#1:Hb_002759_250, PMK#2:Hb_116349_100, MDC#1:Hb_00 0539_030, MDC#2:Hb_000721_030, DXS#1:Hb_003185_080, DXS#2:Hb_062135_030, DXS#3:Hb_019 153_040, DXS#4:Hb_ 002473_110, DXS#5:Hb_021596_070, DXS#6:Hb_000046_120, DXS#7:Hb_005628_050, DXS#8:Hb_012796_050, DXS#9:Hb_124247_030, DXR#1:Hb_007520_020, DXR#2:Hb_000922_160, MCT#1:Hb_029142_030, MCT#2:Hb_007576_140, CMK#1:Hb_000453_240, CMK#2:Hb_001975_040, MDS#1:Hb_177215_030, MDS#2:Hb_ 002529_090, HDS#1:Hb_000029_350, HDS#2:Hb_000008_410, HDR#1:Hb_159809_090, HDR#2:Hb_000963_090, IPPI#1:Hb_005686_130, IPPI#2:Hb_000608_190, DMAT#1:Hb_001842_100, DMAT#2:Hb_000428_050, DMAT#3:Hb_000130_260, FPPS#1:Hb_000072_070, FPPS#2:Hb_100414_010, FPPS#3:Hb_011224_160, GGPPS# 1:Hb_1 15919_020, GGPPS#2:Hb_000371_100, GGPPS#3:Hb_134789_010, GGPPS#4:Hb_020367_050, GGPPS#5:Hb_000114_150, GGPPS#6:Hb_002768_080, GGPPS# 7:Hb_000009_180, GGPPS#8:Hb_010672_040, GGPPS#9:Hb_009780_060, GGPPS#10:Hb_001948_040." path-from-xml="dsw056f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055214/dsw056f3.tif" path-from-xml="dsw056f3.tif" target="_blank">View large</a><a section="78055214" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056f3.png?Expires=2147483647&Signature=q8tjFgAe~aaIntBThkZN26~WpHTIzTp83qRoAC37QYy7sopm0czRhUllKHnSc~aMX7uK9o3uqWK7AwyHykzb6g3qwEJ3MiuHEVRqa1hmx4JV~9GJb-B0poeuwcw5gTfus0FToVm2BOw1bPPf8z5MPu6Se2AsvY68wdJXpV7CoWaDYkIiCZ5MTyUChL1lqBh~zjWyr6sooKbRCDMQqpYijH~sXyPkBiDc0H9WBBFPf4yC~IvyAVf-7Vn3gdx3U6V2w7sMgIX7PbFnRSai9mo~tBr8UU0sgzfFk9DPMB4dFRHqCipCMRlQnv0fUI8rdF8ttxWJvqRHrDC6-mfoUdt0Ug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055214&ar=2960055&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw056f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Latex biosynthesis pathway and heat maps of scaled expression values (log10(FPKM + 1)) for genes associated with latex biosynthesis. The lines of each heat map represent duplicate genes and the columns represent samples: PB 350 latex, RRIM 901 latex, RRIM 600 latex, RRIM 600 bark, RRIM 600 petiole and RRIM 600 leaf from the left. 37 MVA pathway, 21 MEP pathway and 18 prenyl-PP biosynthetic genes, 7 CPTs, 1 CPTL, 9 REFs and 8 SRPP genes are defined in our draft genome article.<a reveal-id="dsw056-B5" data-open="dsw056-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> In the heat maps, genes were sorted by the expression value in RRIM 600 latex. These figures were drawn by an R package of gplots. <strong>Abbreviations in the MVA pathway:</strong> PD, pyruvate dehydrogenase; PDC, pyruvate dehydrogenase complex; DLD, dihydrolipoamide dehydrogenase; AACT, acetyl-CoA acetyltransferase; HMGS, hydroxymethyglutaryl coenzyme A synthase; HMG-CoA, hydroxymethylglutaryl coenzyme A; HMGR, hydroxymethyglutaryl coenzyme A reductase; MK, mevalonate kinase; mevalonate-OP, mevalonate monophosphate; PMK, phosphomevalonate kinase; mavalonate-OPP, mevalonate diphosphate; MDC, diphosphomevalonate decarboxylase; in the plastidic MEP pathway: DXS, 1-deoxy-D-xylulose 5-phosphate synthase; DXP, 1-deoxy-D-xylulose-5-phosphate; DXR, 1-deoxy-D-xylulose 5-phosphate reductoisomerase; MEP, 2-C-methyl-D-erythritol 4-phosphate; MCT, 2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase; CDP-ME, 2-C-methyl-D-erythritol-4-phosphate; CMK, 4-(cytidine 5′-diphospho)-2-C-methyl-D-erythritol kinase; CDP-ME2P, 4-(cytidine 5′-diphospho)-2-C-methyl-D-erythritol; MDS, 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase; MEcPP, 2-C-methyl-D-erythritol 2,4-cyclodiphosphate; HDS, 4-hydroxy-3-methylbut-2-enyl diphosphate synthase; HMB-PP, 4-hydroxy-3-methyl-but-2-en-1-yl diphosphate; HDR, 4-hydroxy-3-methylbut-2-enyl diphosphate reductase. Corresponding gene IDs are as follow: PD#1:Hb_001638_290, PD#2:Hb_006420_080, PD#3:Hb_132840_110, PD#4:Hb_001500_040, PD#5:Hb_116349_120, PD#6:Hb_001123_160, PD#7:Hb_027506_040, PD#8:Hb_003397_030, PDC#1:Hb_000800_090, PDC#2:Hb_005306_180, PDC#3:Hb_002477_290, PDC#4:Hb_000110_310, PDC#5:Hb_003411_090, PDC#6:Hb_000207_300, PDC#7:Hb_002876_240, DLD#1:Hb_001723_060, DLD#2:Hb_002217_100, DLD#3:Hb_000645_180, DLD#4:Hb_012305_100, AACT#1:Hb_000613_200, AACT#2:Hb_000184_040, AACT#3: Hb_000377_090, AACT#4:Hb_000795_060, HMGS#1:Hb_003207_110, HMGS#2:Hb_000107_440, HMGS#3:Hb_004032_440, HMGS#4:Hb_001153_150, HMGS# 5:Hb_014497_120, HMGS#6:Hb_142382_010, HMGS#7:Hb_000116_530, MK#1:Hb_035273_030, MK#2:Hb_127194_010, MK#3:Hb_002042_080, PMK#1:Hb_002759_250, PMK#2:Hb_116349_100, MDC#1:Hb_00 0539_030, MDC#2:Hb_000721_030, DXS#1:Hb_003185_080, DXS#2:Hb_062135_030, DXS#3:Hb_019 153_040, DXS#4:Hb_ 002473_110, DXS#5:Hb_021596_070, DXS#6:Hb_000046_120, DXS#7:Hb_005628_050, DXS#8:Hb_012796_050, DXS#9:Hb_124247_030, DXR#1:Hb_007520_020, DXR#2:Hb_000922_160, MCT#1:Hb_029142_030, MCT#2:Hb_007576_140, CMK#1:Hb_000453_240, CMK#2:Hb_001975_040, MDS#1:Hb_177215_030, MDS#2:Hb_ 002529_090, HDS#1:Hb_000029_350, HDS#2:Hb_000008_410, HDR#1:Hb_159809_090, HDR#2:Hb_000963_090, IPPI#1:Hb_005686_130, IPPI#2:Hb_000608_190, DMAT#1:Hb_001842_100, DMAT#2:Hb_000428_050, DMAT#3:Hb_000130_260, FPPS#1:Hb_000072_070, FPPS#2:Hb_100414_010, FPPS#3:Hb_011224_160, GGPPS# 1:Hb_1 15919_020, GGPPS#2:Hb_000371_100, GGPPS#3:Hb_134789_010, GGPPS#4:Hb_020367_050, GGPPS#5:Hb_000114_150, GGPPS#6:Hb_002768_080, GGPPS# 7:Hb_000009_180, GGPPS#8:Hb_010672_040, GGPPS#9:Hb_009780_060, GGPPS#10:Hb_001948_040.</p></div></div><div content-id="dsw056-F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/m_dsw056f3.png?Expires=2147483647&Signature=B8osS8VPUWcgvDaO719pTkE8t51LQHN7W~6VCKHqOcGWsF2Xb3UvyADySdfNi1XT4WtbbaTLa7ZuKmiS9vpR9urgHlGbWXkHgrqReqS-WliGjldUV6gmOdG9LYbD9eB8thuDt5R0pBH-WT6Yg1lerkOFQnAkf04ESzoiH88-5AMjdk-ShBDc8itV9k4KtPmwAonIAM2mu8cSnr-D5sIPUHY3xKz9sa3tmtuyM6rfgVsQocEoKOGVVHiUpHbYPYqsL2FDwcBd6z6uqV1DlciVZYBN9~Sz6FSUbStkb3a4UTwjuY5ZDcZAcHeBIeHgfnWNWTT8kxib9rYpQU1n2n-YNQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Latex biosynthesis pathway and heat maps of scaled expression values (log10(FPKM + 1)) for genes associated with latex biosynthesis. The lines of each heat map represent duplicate genes and the columns represent samples: PB 350 latex, RRIM 901 latex, RRIM 600 latex, RRIM 600 bark, RRIM 600 petiole and RRIM 600 leaf from the left. 37 MVA pathway, 21 MEP pathway and 18 prenyl-PP biosynthetic genes, 7 CPTs, 1 CPTL, 9 REFs and 8 SRPP genes are defined in our draft genome article.5 In the heat maps, genes were sorted by the expression value in RRIM 600 latex. These figures were drawn by an R package of gplots. Abbreviations in the MVA pathway: PD, pyruvate dehydrogenase; PDC, pyruvate dehydrogenase complex; DLD, dihydrolipoamide dehydrogenase; AACT, acetyl-CoA acetyltransferase; HMGS, hydroxymethyglutaryl coenzyme A synthase; HMG-CoA, hydroxymethylglutaryl coenzyme A; HMGR, hydroxymethyglutaryl coenzyme A reductase; MK, mevalonate kinase; mevalonate-OP, mevalonate monophosphate; PMK, phosphomevalonate kinase; mavalonate-OPP, mevalonate diphosphate; MDC, diphosphomevalonate decarboxylase; in the plastidic MEP pathway: DXS, 1-deoxy-D-xylulose 5-phosphate synthase; DXP, 1-deoxy-D-xylulose-5-phosphate; DXR, 1-deoxy-D-xylulose 5-phosphate reductoisomerase; MEP, 2-C-methyl-D-erythritol 4-phosphate; MCT, 2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase; CDP-ME, 2-C-methyl-D-erythritol-4-phosphate; CMK, 4-(cytidine 5′-diphospho)-2-C-methyl-D-erythritol kinase; CDP-ME2P, 4-(cytidine 5′-diphospho)-2-C-methyl-D-erythritol; MDS, 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase; MEcPP, 2-C-methyl-D-erythritol 2,4-cyclodiphosphate; HDS, 4-hydroxy-3-methylbut-2-enyl diphosphate synthase; HMB-PP, 4-hydroxy-3-methyl-but-2-en-1-yl diphosphate; HDR, 4-hydroxy-3-methylbut-2-enyl diphosphate reductase. Corresponding gene IDs are as follow: PD#1:Hb_001638_290, PD#2:Hb_006420_080, PD#3:Hb_132840_110, PD#4:Hb_001500_040, PD#5:Hb_116349_120, PD#6:Hb_001123_160, PD#7:Hb_027506_040, PD#8:Hb_003397_030, PDC#1:Hb_000800_090, PDC#2:Hb_005306_180, PDC#3:Hb_002477_290, PDC#4:Hb_000110_310, PDC#5:Hb_003411_090, PDC#6:Hb_000207_300, PDC#7:Hb_002876_240, DLD#1:Hb_001723_060, DLD#2:Hb_002217_100, DLD#3:Hb_000645_180, DLD#4:Hb_012305_100, AACT#1:Hb_000613_200, AACT#2:Hb_000184_040, AACT#3: Hb_000377_090, AACT#4:Hb_000795_060, HMGS#1:Hb_003207_110, HMGS#2:Hb_000107_440, HMGS#3:Hb_004032_440, HMGS#4:Hb_001153_150, HMGS# 5:Hb_014497_120, HMGS#6:Hb_142382_010, HMGS#7:Hb_000116_530, MK#1:Hb_035273_030, MK#2:Hb_127194_010, MK#3:Hb_002042_080, PMK#1:Hb_002759_250, PMK#2:Hb_116349_100, MDC#1:Hb_00 0539_030, MDC#2:Hb_000721_030, DXS#1:Hb_003185_080, DXS#2:Hb_062135_030, DXS#3:Hb_019 153_040, DXS#4:Hb_ 002473_110, DXS#5:Hb_021596_070, DXS#6:Hb_000046_120, DXS#7:Hb_005628_050, DXS#8:Hb_012796_050, DXS#9:Hb_124247_030, DXR#1:Hb_007520_020, DXR#2:Hb_000922_160, MCT#1:Hb_029142_030, MCT#2:Hb_007576_140, CMK#1:Hb_000453_240, CMK#2:Hb_001975_040, MDS#1:Hb_177215_030, MDS#2:Hb_ 002529_090, HDS#1:Hb_000029_350, HDS#2:Hb_000008_410, HDR#1:Hb_159809_090, HDR#2:Hb_000963_090, IPPI#1:Hb_005686_130, IPPI#2:Hb_000608_190, DMAT#1:Hb_001842_100, DMAT#2:Hb_000428_050, DMAT#3:Hb_000130_260, FPPS#1:Hb_000072_070, FPPS#2:Hb_100414_010, FPPS#3:Hb_011224_160, GGPPS# 1:Hb_1 15919_020, GGPPS#2:Hb_000371_100, GGPPS#3:Hb_134789_010, GGPPS#4:Hb_020367_050, GGPPS#5:Hb_000114_150, GGPPS#6:Hb_002768_080, GGPPS# 7:Hb_000009_180, GGPPS#8:Hb_010672_040, GGPPS#9:Hb_009780_060, GGPPS#10:Hb_001948_040." path-from-xml="dsw056f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/78055214/dsw056f3.tif" path-from-xml="dsw056f3.tif" target="_blank">View large</a><a section="78055214" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056f3.png?Expires=2147483647&Signature=q8tjFgAe~aaIntBThkZN26~WpHTIzTp83qRoAC37QYy7sopm0czRhUllKHnSc~aMX7uK9o3uqWK7AwyHykzb6g3qwEJ3MiuHEVRqa1hmx4JV~9GJb-B0poeuwcw5gTfus0FToVm2BOw1bPPf8z5MPu6Se2AsvY68wdJXpV7CoWaDYkIiCZ5MTyUChL1lqBh~zjWyr6sooKbRCDMQqpYijH~sXyPkBiDc0H9WBBFPf4yC~IvyAVf-7Vn3gdx3U6V2w7sMgIX7PbFnRSai9mo~tBr8UU0sgzfFk9DPMB4dFRHqCipCMRlQnv0fUI8rdF8ttxWJvqRHrDC6-mfoUdt0Ug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=78055214&ar=2960055&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsw056f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Latex biosynthesis pathway and heat maps of scaled expression values (log10(FPKM + 1)) for genes associated with latex biosynthesis. The lines of each heat map represent duplicate genes and the columns represent samples: PB 350 latex, RRIM 901 latex, RRIM 600 latex, RRIM 600 bark, RRIM 600 petiole and RRIM 600 leaf from the left. 37 MVA pathway, 21 MEP pathway and 18 prenyl-PP biosynthetic genes, 7 CPTs, 1 CPTL, 9 REFs and 8 SRPP genes are defined in our draft genome article.<a reveal-id="dsw056-B5" data-open="dsw056-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> In the heat maps, genes were sorted by the expression value in RRIM 600 latex. These figures were drawn by an R package of gplots. <strong>Abbreviations in the MVA pathway:</strong> PD, pyruvate dehydrogenase; PDC, pyruvate dehydrogenase complex; DLD, dihydrolipoamide dehydrogenase; AACT, acetyl-CoA acetyltransferase; HMGS, hydroxymethyglutaryl coenzyme A synthase; HMG-CoA, hydroxymethylglutaryl coenzyme A; HMGR, hydroxymethyglutaryl coenzyme A reductase; MK, mevalonate kinase; mevalonate-OP, mevalonate monophosphate; PMK, phosphomevalonate kinase; mavalonate-OPP, mevalonate diphosphate; MDC, diphosphomevalonate decarboxylase; in the plastidic MEP pathway: DXS, 1-deoxy-D-xylulose 5-phosphate synthase; DXP, 1-deoxy-D-xylulose-5-phosphate; DXR, 1-deoxy-D-xylulose 5-phosphate reductoisomerase; MEP, 2-C-methyl-D-erythritol 4-phosphate; MCT, 2-C-methyl-D-erythritol 4-phosphate cytidylyltransferase; CDP-ME, 2-C-methyl-D-erythritol-4-phosphate; CMK, 4-(cytidine 5′-diphospho)-2-C-methyl-D-erythritol kinase; CDP-ME2P, 4-(cytidine 5′-diphospho)-2-C-methyl-D-erythritol; MDS, 2-C-methyl-D-erythritol 2,4-cyclodiphosphate synthase; MEcPP, 2-C-methyl-D-erythritol 2,4-cyclodiphosphate; HDS, 4-hydroxy-3-methylbut-2-enyl diphosphate synthase; HMB-PP, 4-hydroxy-3-methyl-but-2-en-1-yl diphosphate; HDR, 4-hydroxy-3-methylbut-2-enyl diphosphate reductase. Corresponding gene IDs are as follow: PD#1:Hb_001638_290, PD#2:Hb_006420_080, PD#3:Hb_132840_110, PD#4:Hb_001500_040, PD#5:Hb_116349_120, PD#6:Hb_001123_160, PD#7:Hb_027506_040, PD#8:Hb_003397_030, PDC#1:Hb_000800_090, PDC#2:Hb_005306_180, PDC#3:Hb_002477_290, PDC#4:Hb_000110_310, PDC#5:Hb_003411_090, PDC#6:Hb_000207_300, PDC#7:Hb_002876_240, DLD#1:Hb_001723_060, DLD#2:Hb_002217_100, DLD#3:Hb_000645_180, DLD#4:Hb_012305_100, AACT#1:Hb_000613_200, AACT#2:Hb_000184_040, AACT#3: Hb_000377_090, AACT#4:Hb_000795_060, HMGS#1:Hb_003207_110, HMGS#2:Hb_000107_440, HMGS#3:Hb_004032_440, HMGS#4:Hb_001153_150, HMGS# 5:Hb_014497_120, HMGS#6:Hb_142382_010, HMGS#7:Hb_000116_530, MK#1:Hb_035273_030, MK#2:Hb_127194_010, MK#3:Hb_002042_080, PMK#1:Hb_002759_250, PMK#2:Hb_116349_100, MDC#1:Hb_00 0539_030, MDC#2:Hb_000721_030, DXS#1:Hb_003185_080, DXS#2:Hb_062135_030, DXS#3:Hb_019 153_040, DXS#4:Hb_ 002473_110, DXS#5:Hb_021596_070, DXS#6:Hb_000046_120, DXS#7:Hb_005628_050, DXS#8:Hb_012796_050, DXS#9:Hb_124247_030, DXR#1:Hb_007520_020, DXR#2:Hb_000922_160, MCT#1:Hb_029142_030, MCT#2:Hb_007576_140, CMK#1:Hb_000453_240, CMK#2:Hb_001975_040, MDS#1:Hb_177215_030, MDS#2:Hb_ 002529_090, HDS#1:Hb_000029_350, HDS#2:Hb_000008_410, HDR#1:Hb_159809_090, HDR#2:Hb_000963_090, IPPI#1:Hb_005686_130, IPPI#2:Hb_000608_190, DMAT#1:Hb_001842_100, DMAT#2:Hb_000428_050, DMAT#3:Hb_000130_260, FPPS#1:Hb_000072_070, FPPS#2:Hb_100414_010, FPPS#3:Hb_011224_160, GGPPS# 1:Hb_1 15919_020, GGPPS#2:Hb_000371_100, GGPPS#3:Hb_134789_010, GGPPS#4:Hb_020367_050, GGPPS#5:Hb_000114_150, GGPPS#6:Hb_002768_080, GGPPS# 7:Hb_000009_180, GGPPS#8:Hb_010672_040, GGPPS#9:Hb_009780_060, GGPPS#10:Hb_001948_040.</p></div></div></div>                            <h3 scrollto-destination=78055215 id="78055215" class="section-title" >3.8. Expression profile of rubber biosynthesis genes</h3>
<p>Latex is an emulsion of rubber serum and rubber particles produced from laticifer cells localized in the outer layers of the vascular bundle of the bark. A rubber particle is composed of a lipid monolayer membrane and it is suggested that there are three major rubber-related proteins (CPTs, REFs and SRPPs) localized on the surface of this particle. We confirmed the expression of 7 CPTs, 1 CPTL, 9 REFs and 8 SRPPs (<a reveal-id="dsw056-F3" data-open="dsw056-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). We found that CPT1, CPT2, CPTL, REF1, REF2 and SRPP1 were expressed at least 20 times more in latex than in leaf. The newly identified REF8 from the latest RRIM 600 draft genome sequence also shows a similar expression pattern to REF1 and REF2. This protein sequence has a REF domain and shows 59% identity with 90/117 amino acid alignment to SRPP1 with blastp. Its FPKM value is 11,221 and its level is 250 times higher in latex than in leaves. Therefore, REF8 is a candidate for further experimental validation to allow more characterization of its function.</p><p>When we calculated the correlation coefficients of all FPKM values between each sample, latex is more highly correlated in bark (correlation coefficient is 0.82) than in the petiole and leaf (correlation coefficient is 0.75–0.76) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Unlike the overall trend, many of the well-studied latex polymerization-related factors, such as CPTL, CPT2, REF1, REF2 and SRPP1, showed high expression in latex and petioles, but not in bark (<a reveal-id="dsw056-F3" data-open="dsw056-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Since the petiole can produce a small amount of latex such a correlation may explain this observation.</p>                            <h3 scrollto-destination=78055218 id="78055218" class="section-title" >3.9. Candidate TFs that regulate rubber biosynthesis genes</h3>
<p>Little is known about the transcriptional regulation of rubber biosynthesis genes. We calculated the FPKM value for the 3,126 predicted TFs and selected 39 whose average FPKM value of three latex samples was more than five times higher than the average FPKM value of other tissues (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). We carried out gene set enrichment analysis against TF families and four of them were significant with a FDR adjusted <em>q</em>-value &lt; 0.05 (<a reveal-id="dsw056-T3" data-open="dsw056-T3" class="link link-reveal link-table xref-fig">Table 3</a>). The C2C2-Dof,<a reveal-id="dsw056-B43" data-open="dsw056-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> TAZ<a reveal-id="dsw056-B44" data-open="dsw056-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> and AP2<a reveal-id="dsw056-B45" data-open="dsw056-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> families are related to the stress response. Since latex is exuded after tissue injury, these TFs are important candidate regulators. The <em>cis</em>-element of AP2 is known in <em>A</em><em>.</em><em>thaliana</em> and we confirmed partially similar motifs in the promoters of the CPT genes (<a reveal-id="dsw056-T3" data-open="dsw056-T3" class="link link-reveal link-table xref-fig">Table 3</a>). <div content-id="dsw056-T3" class="table-modal"><div class="table-wrap"><div id="dsw056-T3" data-id="dsw056-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>Gene set enrichment analysis against TF families that are significantly highly expressed in latex.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>TF family </th><th>No. of TFs </th><th>Highly expressed TFs in latex </th><th><em>q</em>-value </th><th> </th><th>Function </th><th><em>cis</em>-element </th></tr></thead><tbody><tr><td>C2C2-Dof </td><td char=".">49 </td><td char=".">7 </td><td char=".">0.009 </td><td>plant specific </td><td>Plant-specific phenomena including light, phytohormone and defense responses, seed development and germination </td><td>— </td></tr><tr><td>TAZ </td><td char=".">7 </td><td char=".">2 </td><td char=".">0.027 </td><td>plant specific </td><td>Responses to different stress stimuli </td><td>— </td></tr><tr><td>AP2 </td><td char=".">28 </td><td char=".">4 </td><td char=".">0.032 </td><td>plant specific </td><td>A variety of biological processes and response to biotic and environmental stress </td><td>5′-gCAC(A/G)N(A/T) TcCC(a/g)ANG(c/t)-3′ </td></tr></tbody></table></div><div content-id="dsw056-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>TF family </th><th>No. of TFs </th><th>Highly expressed TFs in latex </th><th><em>q</em>-value </th><th> </th><th>Function </th><th><em>cis</em>-element </th></tr></thead><tbody><tr><td>C2C2-Dof </td><td char=".">49 </td><td char=".">7 </td><td char=".">0.009 </td><td>plant specific </td><td>Plant-specific phenomena including light, phytohormone and defense responses, seed development and germination </td><td>— </td></tr><tr><td>TAZ </td><td char=".">7 </td><td char=".">2 </td><td char=".">0.027 </td><td>plant specific </td><td>Responses to different stress stimuli </td><td>— </td></tr><tr><td>AP2 </td><td char=".">28 </td><td char=".">4 </td><td char=".">0.032 </td><td>plant specific </td><td>A variety of biological processes and response to biotic and environmental stress </td><td>5′-gCAC(A/G)N(A/T) TcCC(a/g)ANG(c/t)-3′ </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsw056-T3" data-id="dsw056-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p>Gene set enrichment analysis against TF families that are significantly highly expressed in latex.</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>TF family </th><th>No. of TFs </th><th>Highly expressed TFs in latex </th><th><em>q</em>-value </th><th> </th><th>Function </th><th><em>cis</em>-element </th></tr></thead><tbody><tr><td>C2C2-Dof </td><td char=".">49 </td><td char=".">7 </td><td char=".">0.009 </td><td>plant specific </td><td>Plant-specific phenomena including light, phytohormone and defense responses, seed development and germination </td><td>— </td></tr><tr><td>TAZ </td><td char=".">7 </td><td char=".">2 </td><td char=".">0.027 </td><td>plant specific </td><td>Responses to different stress stimuli </td><td>— </td></tr><tr><td>AP2 </td><td char=".">28 </td><td char=".">4 </td><td char=".">0.032 </td><td>plant specific </td><td>A variety of biological processes and response to biotic and environmental stress </td><td>5′-gCAC(A/G)N(A/T) TcCC(a/g)ANG(c/t)-3′ </td></tr></tbody></table></div><div content-id="dsw056-T3" class="table-modal"><table><thead name="thead" align=""><tr><th>TF family </th><th>No. of TFs </th><th>Highly expressed TFs in latex </th><th><em>q</em>-value </th><th> </th><th>Function </th><th><em>cis</em>-element </th></tr></thead><tbody><tr><td>C2C2-Dof </td><td char=".">49 </td><td char=".">7 </td><td char=".">0.009 </td><td>plant specific </td><td>Plant-specific phenomena including light, phytohormone and defense responses, seed development and germination </td><td>— </td></tr><tr><td>TAZ </td><td char=".">7 </td><td char=".">2 </td><td char=".">0.027 </td><td>plant specific </td><td>Responses to different stress stimuli </td><td>— </td></tr><tr><td>AP2 </td><td char=".">28 </td><td char=".">4 </td><td char=".">0.032 </td><td>plant specific </td><td>A variety of biological processes and response to biotic and environmental stress </td><td>5′-gCAC(A/G)N(A/T) TcCC(a/g)ANG(c/t)-3′ </td></tr></tbody></table></div></div></p>                            <h3 scrollto-destination=78055220 id="78055220" class="section-title" >3.10. Expression profile for disease-resistance genes</h3>
<p>Corynespora leaf fall disease caused by infection with <em>Corynespora cassiicola</em> is one of the major fungal diseases in rubber plantations and it can cause a severe reduction in rubber latex production.<a reveal-id="dsw056-B46" data-open="dsw056-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> We performed cultivar-specific RNA-Seq analysis using the <em>C. cassiicola</em> resistant cultivar PB 350 and the susceptible cultivars RRIM 600 and 901. At first we focused on 483 disease-resistance genes (R genes) that play important roles in the response against the pathogen. In normal or uninfected tissue, the expression of the R genes in PB 350 was the same or slightly lower than RRIM 600 and 901 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). We also examined genes whose expression in PB 350 is over 10 times higher than in RRIM 600 and detected 41 genes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>). 40% (17 out of 41) of these are enriched with heat shock proteins (HSPs). Since HSPs are known to be critical for the plant defense response and some function as chaperones of R proteins for effector-triggered immunity,<a reveal-id="dsw056-B47" data-open="dsw056-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> HSPs are likely to be important regulatory factors for the pathogen-stress response in rubber and contribute to the higher disease resistance of PB 350.</p>                            <h2 scrollto-destination=78055222 id="78055222" class="section-title" >4. Conclusion</h2>
<p>This study is the first large-scale collection of FL-cDNAs in <em>H. brasiliensis</em> in which 19,487 FL-cDNA clones harboring genetic information for 7,704 known genes and 22 new genes were obtained. The FL-cDNAs also provide 9,486 TSSs but the current coverage is not enough to carry out comprehensive TSS analysis. For better understanding, Cap Analysis of Gene Expression profiling or other genome-wide expression analyses will be necessary.</p><p>In this study, we used our newly sequenced rubber draft genome as a reference for transcriptome analysis.<a reveal-id="dsw056-B5" data-open="dsw056-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> The genome sequence offers a more robust analysis of the generated RNA-Seq data in turn giving a more accurate expression profile analysis as the gene models will be more precise. Our findings reveal the tissue-specific expression patterns of the rubber biosynthesis genes and list candidate transcriptional regulators.</p><p>Our FL-cDNA sequences and RNA <em>de novo</em> assembly will be useful information for future improvement of the rubber tree and also as a basis for comparative analysis with other economically important Euphorbiaceae, such as cassava and the castor oil plant.</p>                            <h2 scrollto-destination=78055226 id="78055226" class="section-title" >Data Availability</h2>
<p>Our FL-cDNA sequence data and RNA-Seq data are available at DDBJ/EMBL/Genbank BioProject under accession PRJDB4387. We also provide the genome browser at <a class="link link-uri" href="http://matsui-lab.riken.jp/rubber/" target="_blank">http://matsui-lab.riken.jp/rubber/</a>. Rubber FL-cDNA clones are available from the RIKEN BioResource Center (BRC).</p>                            <h2 scrollto-destination=78055228 id="78055228" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Setsuko Shimada (RIKEN Center for Sustainable Resource Science) for her constructive discussions and Shunsuke Imai (Sumitomo Riko Co.) for his advice on <a reveal-id="dsw056-F3" data-open="dsw056-F3" class="link link-reveal link-table xref-fig">Figure 3</a>.</p>                            <h3 scrollto-destination=78055230 id="78055230" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=78055232 id="78055232" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org" target="_blank">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h3 scrollto-destination=78055234 id="78055234" class="section-title" >Funding</h3>
<p>This research was conducted under the research programme of the RIKEN Biomass Engineering Programme.</p>                            <h2 scrollto-destination=78055236 id="78055236" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsw056-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cornish</div> <div class="given-names">K.</div></div></div>, <div class="year">2001</div>, 
<div class="article-title">Similarities and differences in rubber biochemistry among plant species</div>. <div class="source">Phytochemistry</div> , <div class="volume">57</div>, <div class="fpage">1123</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Similarities%20and%20differences%20in%20rubber%20biochemistry%20among%20plant%20species&amp;author=K.%20Cornish&amp;publication_year=2001&amp;journal=Phytochemistry&amp;volume=57&amp;pages=1123-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0031-9422(01)00097-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0031-9422(01)00097-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0031-9422(01)00097-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11430985" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">van Beilen</div> <div class="given-names">J.B.</div></div>, <div class="name string-name"><div class="surname">Poirier</div> <div class="given-names">Y.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Establishment of new crops for the production of natural rubber</div>. <div class="source">Trends Biotechnol</div> ., <div class="volume">25</div>, <div class="fpage">522</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Establishment%20of%20new%20crops%20for%20the%20production%20of%20natural%20rubber&amp;author=J.B.%20van%20Beilen&amp;author=Y.%20Poirier&amp;publication_year=2007&amp;journal=Trends%20Biotechnol&amp;volume=25&amp;pages=522-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibtech.2007.08.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibtech.2007.08.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibtech.2007.08.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17936926" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mooibroek</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Cornish</div> <div class="given-names">K.</div></div></div>, <div class="year">2000</div>, 
<div class="article-title">Alternative sources of natural rubber</div>. <div class="source">Appl. Microbiol. Biotechnol.</div>  <div class="volume">53</div>, <div class="fpage">355</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Alternative%20sources%20of%20natural%20rubber&amp;author=H.%20Mooibroek&amp;author=K.%20Cornish&amp;publication_year=2000&amp;journal=Appl.%20Microbiol.%20Biotechnol.&amp;volume=53&amp;pages=355-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002530051627" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002530051627" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002530051627"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10803889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Rahman</div> <div class="given-names">A.Y.</div></div>, <div class="name string-name"><div class="surname">Usharraj</div> <div class="given-names">A.O.</div></div>, <div class="name string-name"><div class="surname">Misra</div> <div class="given-names">B.B.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Draft genome sequence of the rubber tree <em>Hevea brasiliensis</em></div>. <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20the%20rubber%20tree%20Hevea%20brasiliensis&amp;author=A.Y.%20Rahman&amp;author=A.O.%20Usharraj&amp;author=B.B.%20Misra&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-75" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-75" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-75"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23375136" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lau</div> <div class="given-names">N.S.</div></div>, <div class="name string-name"><div class="surname">Makita</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kawashima</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">The rubber tree genome shows expansion of gene family associated with rubber biosynthesis</div>. <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">28594</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rubber%20tree%20genome%20shows%20expansion%20of%20gene%20family%20associated%20with%20rubber%20biosynthesis&amp;author=N.S.%20Lau&amp;author=Y.%20Makita&amp;author=M.%20Kawashima&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=28594" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep28594" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep28594" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep28594"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27339202" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Fang</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2016</div>, 
<div class="article-title">The rubber tree genome reveals new insights into rubber production and species adaptation</div>. <div class="source">Nat. Plants</div> , <div class="volume">2</div>, <div class="fpage">16073</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rubber%20tree%20genome%20reveals%20new%20insights%20into%20rubber%20production%20and%20species%20adaptation&amp;author=C.%20Tang&amp;author=M.%20Yang&amp;author=Y.%20Fang&amp;publication_year=2016&amp;journal=Nat.%20Plants&amp;volume=2&amp;pages=16073" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nplants.2016.73" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnplants.2016.73" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnplants.2016.73"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27255837" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Han</div> <div class="given-names">K.H.</div></div>, <div class="name string-name"><div class="surname">Shin</div> <div class="given-names">D.H.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">I.J.</div></div>, <div class="name string-name"><div class="surname">Oh</div> <div class="given-names">S.K.</div></div>, <div class="name string-name"><div class="surname">Chow</div> <div class="given-names">K.S.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Genes expressed in the latex of <em>Hevea brasiliensis</em></div>. <div class="source">Tree Physiol.</div>  <div class="volume">20</div>, <div class="fpage">503</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20expressed%20in%20the%20latex%20of%20Hevea%20brasiliensis&amp;author=K.H.%20Han&amp;author=D.H.%20Shin&amp;author=J.%20Yang&amp;author=I.J.%20Kim&amp;author=S.K.%20Oh&amp;author=K.S.%20Chow&amp;publication_year=2000&amp;journal=Tree%20Physiol.&amp;volume=20&amp;pages=503-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/treephys/20.8.503" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Ftreephys%2F20.8.503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Ftreephys%2F20.8.503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12651430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Asawatreratanakul</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Y.W.</div></div>, <div class="name string-name"><div class="surname">Wititsuwannakul</div> <div class="given-names">D.</div></div></div>, et al.  . <div class="year">2003</div>, 
<div class="article-title">Molecular cloning, expression and characterization of cDNA encoding <em>cis</em>-prenyltransferases from <em>Hevea brasiliensis</em>. A key factor participating in natural rubber biosynthesis</div>. <div class="source">Eur. J. Biochem</div> ., <div class="volume">270</div>, <div class="fpage">4671</div>–<div class="lpage">80</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%2C%20expression%20and%20characterization%20of%20cDNA%20encoding%20cis-prenyltransferases%20from%20Hevea%20brasiliensis.%20A%20key%20factor%20participating%20in%20natural%20rubber%20biosynthesis&amp;author=K.%20Asawatreratanakul&amp;author=Y.W.%20Zhang&amp;author=D.%20Wititsuwannakul&amp;publication_year=2003&amp;journal=Eur.%20J.%20Biochem&amp;volume=270&amp;pages=4671-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1432-1033.2003.03863.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1432-1033.2003.03863.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1432-1033.2003.03863.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14622254" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yeang</div> <div class="given-names">H.Y.</div></div>, <div class="name string-name"><div class="surname">Cheong</div> <div class="given-names">K.F.</div></div>, <div class="name string-name"><div class="surname">Sunderasan</div> <div class="given-names">E.</div></div></div>, et al.  . <div class="year">1996</div>, 
<div class="article-title">The 14.6 kd rubber elongation factor (Hev b 1) and 24 kd (Hev b 3) rubber particle proteins are recognized by IgE from patients with spina bifida and latex allergy</div>. <div class="source">J. Allergy Clin. Immunol</div> ., <div class="volume">98</div>, <div class="fpage">628</div>–<div class="lpage">39</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%2014.6%20kd%20rubber%20elongation%20factor%20(Hev%20b%201)%20and%2024%20kd%20(Hev%20b%203)%20rubber%20particle%20proteins%20are%20recognized%20by%20IgE%20from%20patients%20with%20spina%20bifida%20and%20latex%20allergy&amp;author=H.Y.%20Yeang&amp;author=K.F.%20Cheong&amp;author=E.%20Sunderasan&amp;publication_year=1996&amp;journal=J.%20Allergy%20Clin.%20Immunol&amp;volume=98&amp;pages=628-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0091-6749(96)70097-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0091-6749(96)70097-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0091-6749(96)70097-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8828541" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Berthelot</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Lecomte</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Estevez</div> <div class="given-names">Y.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Rubber elongation factor (REF), a major allergen component in <em>Hevea brasiliensis</em> latex has amyloid properties</div>. <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e48065</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rubber%20elongation%20factor%20(REF)%2C%20a%20major%20allergen%20component%20in%20Hevea%20brasiliensis%20latex%20has%20amyloid%20properties&amp;author=K.%20Berthelot&amp;author=S.%20Lecomte&amp;author=Y.%20Estevez&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e48065" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0048065" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0048065" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0048065"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23133547" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Epping</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">van Deenen</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Niephaus</div> <div class="given-names">E.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">A rubber transferase activator is necessary for natural rubber biosynthesis in dandelion</div>. <div class="source">Nat. Plants</div> , <div class="volume">1</div>, <div class="fpage">15048</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20rubber%20transferase%20activator%20is%20necessary%20for%20natural%20rubber%20biosynthesis%20in%20dandelion&amp;author=J.%20Epping&amp;author=N.%20van%20Deenen&amp;author=E.%20Niephaus&amp;publication_year=2015&amp;journal=Nat.%20Plants&amp;volume=1&amp;pages=15048" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nplants.2015.48" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnplants.2015.48" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnplants.2015.48"> </span></div> </div></div></div></div></div><div content-id="dsw056-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chow</div> <div class="given-names">K.S.</div></div>, <div class="name string-name"><div class="surname">Wan</div> <div class="given-names">K.L.</div></div>, <div class="name string-name"><div class="surname">Isa</div> <div class="given-names">M.N.</div></div></div>, et al.  . <div class="year">2007</div>, 
<div class="article-title">Insights into rubber biosynthesis from transcriptome analysis of <em>Hevea brasiliensis</em> latex</div>. <div class="source">J. Exp. Bot</div> ., <div class="volume">58</div>, <div class="fpage">2429</div>–<div class="lpage">40</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Insights%20into%20rubber%20biosynthesis%20from%20transcriptome%20analysis%20of%20Hevea%20brasiliensis%20latex&amp;author=K.S.%20Chow&amp;author=K.L.%20Wan&amp;author=M.N.%20Isa&amp;publication_year=2007&amp;journal=J.%20Exp.%20Bot&amp;volume=58&amp;pages=2429-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erm093" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferm093" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferm093"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17545224" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Triwitayakorn</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Chatkulkawin</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kanjanawattanawong</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Transcriptome sequencing of <em>Hevea brasiliensis</em> for development of microsatellite markers and construction of a genetic linkage map</div>. <div class="source">DNA Res</div> ., <div class="volume">18</div>, <div class="fpage">471</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20of%20Hevea%20brasiliensis%20for%20development%20of%20microsatellite%20markers%20and%20construction%20of%20a%20genetic%20linkage%20map&amp;author=K.%20Triwitayakorn&amp;author=P.%20Chatkulkawin&amp;author=S.%20Kanjanawattanawong&amp;publication_year=2011&amp;journal=DNA%20Res&amp;volume=18&amp;pages=471-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsr034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsr034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsr034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22086998" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Salgado</div> <div class="given-names">L.R.</div></div>, <div class="name string-name"><div class="surname">Koop</div> <div class="given-names">D.M.</div></div>, <div class="name string-name"><div class="surname">Pinheiro</div> <div class="given-names">D.G.</div></div></div>, et al.  . <div class="year">2014</div>, 
<div class="article-title">De novo transcriptome analysis of <em>Hevea brasiliensis</em> tissues by RNA-seq and screening for molecular markers</div>. <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">236</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20transcriptome%20analysis%20of%20Hevea%20brasiliensis%20tissues%20by%20RNA-seq%20and%20screening%20for%20molecular%20markers&amp;author=L.R.%20Salgado&amp;author=D.M.%20Koop&amp;author=D.G.%20Pinheiro&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=236" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24670056" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lembaga Getah</div> <div class="given-names">Malaysia</div></div></div>, <div class="year">2009</div>, Chapter 2: development of rubber clones. <div class="source">Rubber Plantation and Processing Technologies</div> , pp. <div class="fpage">21</div>-<div class="lpage">68</div>. 
<div class="publisher-name">Malaysian Rubber Board (MRB</div>), 
<div class="publisher-loc">Malaysia</div>.</div></div></div></div><div content-id="dsw056-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Deng</div> <div class="given-names">L.H.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">M.W.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">C.F.</div></div>, <div class="name string-name"><div class="surname">Zeng</div> <div class="given-names">H.C.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Extraction of high-quality RNA from rubber tree leaves</div>. <div class="source">Biosci. Biotechnol. Biochem.</div>  <div class="volume">76</div>, <div class="fpage">1394</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extraction%20of%20high-quality%20RNA%20from%20rubber%20tree%20leaves&amp;author=L.H.%20Deng&amp;author=M.W.%20Luo&amp;author=C.F.%20Zhang&amp;author=H.C.%20Zeng&amp;publication_year=2012&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=76&amp;pages=1394-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.120014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.120014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.120014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22785464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carninci</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kvam</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kitamura</div></div></div>, et al.  . <div class="year">1996</div>, 
<div class="article-title">High-efficiency full-length cDNA cloning by biotinylated CAP trapper</div>. <div class="source">Genomics</div> , <div class="volume">37</div>, <div class="fpage">327</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-efficiency%20full-length%20cDNA%20cloning%20by%20biotinylated%20CAP%20trapper&amp;author=P.%20Carninci&amp;author=C.%20Kvam&amp;author=%20Kitamura&amp;publication_year=1996&amp;journal=Genomics&amp;volume=37&amp;pages=327-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/geno.1996.0567" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fgeno.1996.0567" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fgeno.1996.0567"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8938445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carninci</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Shibata</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Hayatsu</div> <div class="given-names">N.</div></div></div>, et al.  . <div class="year">2001</div>, 
<div class="article-title">Balanced-size and long-size cloning of full-length, cap-trapped cDNAs into vectors of the novel lambda-FLC family allows enhanced gene discovery rate and functional analysis</div>. <div class="source">Genomics</div> , <div class="volume">77</div>, <div class="fpage">79</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Balanced-size%20and%20long-size%20cloning%20of%20full-length%2C%20cap-trapped%20cDNAs%20into%20vectors%20of%20the%20novel%20lambda-FLC%20family%20allows%20enhanced%20gene%20discovery%20rate%20and%20functional%20analysis&amp;author=P.%20Carninci&amp;author=Y.%20Shibata&amp;author=N.%20Hayatsu&amp;publication_year=2001&amp;journal=Genomics&amp;volume=77&amp;pages=79-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/geno.2001.6601" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fgeno.2001.6601" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fgeno.2001.6601"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11543636" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huang</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Madan</div> <div class="given-names">A.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">CAP3: A DNA sequence assembly program</div>. <div class="source">Genome Res</div> ., <div class="volume">9</div>, <div class="fpage">868</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CAP3%3A%20A%20DNA%20sequence%20assembly%20program&amp;author=X.%20Huang&amp;author=A.%20Madan&amp;publication_year=1999&amp;journal=Genome%20Res&amp;volume=9&amp;pages=868-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.9.9.868" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.9.9.868" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.9.9.868"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10508846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Martin</div> <div class="given-names">M.</div></div></div>, <div class="year">2011</div>, 
<div class="article-title">Cutadapt removes adapter sequences from high-throughput sequencing reads</div>. <div class="source">EMBnet.journal</div> , <div class="volume">17</div>, <div class="fpage">10</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cutadapt%20removes%20adapter%20sequences%20from%20high-throughput%20sequencing%20reads&amp;author=M.%20Martin&amp;publication_year=2011&amp;journal=EMBnet.journal&amp;volume=17&amp;pages=10-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.14806/ej.17.1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.14806%2Fej.17.1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.14806%2Fej.17.1"> </span></div> </div></div></div></div></div><div content-id="dsw056-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blattner</div> <div class="given-names">F.R.</div></div>, <div class="name string-name"><div class="surname">Plunkett</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Bloch</div> <div class="given-names">C.A.</div></div></div>, et al.  . <div class="year">1997</div>, 
<div class="article-title">The complete genome sequence of <em>Escherichia coli</em> K-12</div>. <div class="source">Science</div> , <div class="volume">277</div>, <div class="fpage">1453</div>–<div class="lpage">62</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complete%20genome%20sequence%20of%20Escherichia%20coli%20K-12&amp;author=F.R.%20Blattner&amp;author=G.%20Plunkett&amp;author=C.A.%20Bloch&amp;publication_year=1997&amp;journal=Science&amp;volume=277&amp;pages=1453-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.277.5331.1453" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.277.5331.1453" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.277.5331.1453"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9278503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>. <div class="source">Genome Biol</div> ., <div class="volume">10</div>, <div class="fpage">R25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Genome%20Biol&amp;volume=10&amp;pages=R25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-3-r25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-3-r25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grabherr</div> <div class="given-names">M.G.</div></div>, <div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Yassour</div> <div class="given-names">M.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</div>. <div class="source">Nat. Biotechnol</div> ., <div class="volume">29</div>, <div class="fpage">644</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;author=M.G.%20Grabherr&amp;author=B.J.%20Haas&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol&amp;volume=29&amp;pages=644-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1883" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div>, <div class="name string-name"><div class="surname">Mount</div> <div class="given-names">S.M.</div></div></div>, et al.  . <div class="year">2003</div>, 
<div class="article-title">Improving the Arabidopsis genome annotation using maximal transcript alignment assemblies<em>.</em></div><div class="source">Nucleic Acids Res</div> ., <div class="volume">31</div>, <div class="fpage">5654</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20the%20Arabidopsis%20genome%20annotation%20using%20maximal%20transcript%20alignment%20assemblies.&amp;author=B.J.%20Haas&amp;author=A.L.%20Delcher&amp;author=S.M.%20Mount&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res&amp;volume=31&amp;pages=5654-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg770" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg770" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg770"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14500829" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">T.D.</div></div>, <div class="name string-name"><div class="surname">Watanabe</div> <div class="given-names">C.K.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">GMAP: a genomic mapping and alignment program for mRNA and EST sequences</div>. <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">1859</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GMAP%3A%20a%20genomic%20mapping%20and%20alignment%20program%20for%20mRNA%20and%20EST%20sequences&amp;author=T.D.%20Wu&amp;author=C.K.%20Watanabe&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=1859-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti310" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti310" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti310"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">BLAT–the BLAST-like alignment tool</div>. <div class="source">Genome Res.</div>  <div class="volume">12</div>, <div class="fpage">656</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%93the%20BLAST-like%20alignment%20tool&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=656-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pachter</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">TopHat: discovering splice junctions with RNA-Seq</div>. <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1105</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat%3A%20discovering%20splice%20junctions%20with%20RNA-Seq&amp;author=C.%20Trapnell&amp;author=L.%20Pachter&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19289445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dobin</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Davis</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">Schlesinger</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Drenkow</div> <div class="given-names">J.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">STAR: ultrafast universal RNA-seq aligner</div>. <div class="source">Bioinformatics</div> , <div class="volume">29</div>, <div class="fpage">15</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=STAR%3A%20ultrafast%20universal%20RNA-seq%20aligner&amp;author=A.%20Dobin&amp;author=C.A.%20Davis&amp;author=F.%20Schlesinger&amp;author=J.%20Drenkow&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=15-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts635" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts635" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts635"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23104886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Goff</div> <div class="given-names">L.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks</div>. <div class="source">Nat. Protoc</div> ., <div class="volume">7</div>, <div class="fpage">562</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20gene%20and%20transcript%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20TopHat%20and%20Cufflinks&amp;author=C.%20Trapnell&amp;author=A.%20Roberts&amp;author=L.%20Goff&amp;publication_year=2012&amp;journal=Nat.%20Protoc&amp;volume=7&amp;pages=562-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2012.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2012.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2012.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22383036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Benjamini</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Hochberg</div> <div class="given-names">Y.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Controlling the false discovery rate: a practical and powerful approach to multiple testing</div>. <div class="source">J. Roy. Stat. Soc. Ser</div> ., <div class="volume">B57</div>:<div class="fpage">289</div>–<div class="lpage">300</div>.</div></div></div></div><div content-id="dsw056-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blake</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Dolan</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Drabkin</div> <div class="given-names">H.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Gene Ontology annotations and resources</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">D530</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20Ontology%20annotations%20and%20resources&amp;author=J.A.%20Blake&amp;author=M.%20Dolan&amp;author=H.%20Drabkin&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=D530-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1050" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1050" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1050"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23161678" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Makita</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Shimada</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kawashima</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kondou-Kuriyama</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Toyoda</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Matsui</div> <div class="given-names">M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">MOROKOSHI: transcriptome database in <em>Sorghum bicolor</em></div>. <div class="source">Plant Cell Physiol</div> ., <div class="volume">56</div>, <div class="fpage">e6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MOROKOSHI%3A%20transcriptome%20database%20in%20Sorghum%20bicolor&amp;author=Y.%20Makita&amp;author=S.%20Shimada&amp;author=M.%20Kawashima&amp;author=T.%20Kondou-Kuriyama&amp;author=T.%20Toyoda&amp;author=M.%20Matsui&amp;publication_year=2015&amp;journal=Plant%20Cell%20Physiol&amp;volume=56&amp;pages=e6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcu187" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcu187" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcu187"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25505007" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shimada</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Makita</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kuriyama-Kondou</div> <div class="given-names">T.</div></div></div>, et al.  . <div class="year">2015</div>, 
<div class="article-title">Functional and expression analyses of transcripts based on full-length cDNAs of <em>Sorghum bicolor</em></div>. <div class="source">DNA Res</div> ., <div class="volume">22</div>, <div class="fpage">485</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20and%20expression%20analyses%20of%20transcripts%20based%20on%20full-length%20cDNAs%20of%20Sorghum%20bicolor&amp;author=S.%20Shimada&amp;author=Y.%20Makita&amp;author=T.%20Kuriyama-Kondou&amp;publication_year=2015&amp;journal=DNA%20Res&amp;volume=22&amp;pages=485-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsv030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsv030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsv030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26546227" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lamesch</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Berardini</div> <div class="given-names">T.Z.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">D.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">The Arabidopsis Information Resource (TAIR): improved gene annotation and new tools</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">40</div>, <div class="fpage">D1202</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Arabidopsis%20Information%20Resource%20(TAIR)%3A%20improved%20gene%20annotation%20and%20new%20tools&amp;author=P.%20Lamesch&amp;author=T.Z.%20Berardini&amp;author=D.%20Li&amp;publication_year=2012&amp;journal=Nucleic%20Acids%20Res&amp;volume=40&amp;pages=D1202-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr1090" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr1090" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr1090"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22140109" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div></div>, et al.  . <div class="year">2011</div>, 
<div class="article-title">Sequence analysis of the genome of an oil-bearing tree, <em>Jatropha curcas</em> L</div>. <div class="source">DNA Res</div> ., <div class="volume">18</div>, <div class="fpage">65</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20analysis%20of%20the%20genome%20of%20an%20oil-bearing%20tree%2C%20Jatropha%20curcas%20L&amp;author=S.%20Sato&amp;author=H.%20Hirakawa&amp;author=S.%20Isobe&amp;publication_year=2011&amp;journal=DNA%20Res&amp;volume=18&amp;pages=65-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21149391" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chan</div> <div class="given-names">A.P.</div></div>, <div class="name string-name"><div class="surname">Crabtree</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">Q.</div></div></div>, et al.  . <div class="year">2010</div>, 
<div class="article-title">Draft genome sequence of the oilseed species <em>Ricinus communis</em></div>. <div class="source">Nat. Biotechnol</div> ., <div class="volume">28</div>, <div class="fpage">951</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20the%20oilseed%20species%20Ricinus%20communis&amp;author=A.P.%20Chan&amp;author=J.%20Crabtree&amp;author=Q.%20Zhao&amp;publication_year=2010&amp;journal=Nat.%20Biotechnol&amp;volume=28&amp;pages=951-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1674" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1674" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1674"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20729833" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lander</div> <div class="given-names">E.S.</div></div>, <div class="name string-name"><div class="surname">Linton</div> <div class="given-names">L.M.</div></div>, <div class="name string-name"><div class="surname">Birren</div> <div class="given-names">B.</div></div></div>, et al.  . <div class="year">2001</div>, 
<div class="article-title">Initial sequencing and analysis of the human genome</div>. <div class="source">Nature</div> , <div class="volume">409</div>, <div class="fpage">860</div>–<div class="lpage">921</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initial%20sequencing%20and%20analysis%20of%20the%20human%20genome&amp;author=E.S.%20Lander&amp;author=L.M.%20Linton&amp;author=B.%20Birren&amp;publication_year=2001&amp;journal=Nature&amp;volume=409&amp;pages=860-921" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35057062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35057062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35057062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11237011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Consortium</div> <div class="given-names">I.H.G.S.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">Finishing the euchromatic sequence of the human genome</div>. <div class="source">Nature</div> , <div class="volume">431</div>, <div class="fpage">931</div>–<div class="lpage">45</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Finishing%20the%20euchromatic%20sequence%20of%20the%20human%20genome&amp;author=I.H.G.S.%20Consortium&amp;publication_year=2004&amp;journal=Nature&amp;volume=431&amp;pages=931-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature03001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature03001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature03001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15496913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Engström</div> <div class="given-names">P.G.</div></div>, <div class="name string-name"><div class="surname">Steijger</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Sipos</div> <div class="given-names">B.</div></div></div>, et al.  . <div class="year">2013</div>, 
<div class="article-title">Systematic evaluation of spliced alignment programs for RNA-seq data</div>. <div class="source">Nat. Methods</div> , <div class="volume">10</div>, <div class="fpage">1185</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20evaluation%20of%20spliced%20alignment%20programs%20for%20RNA-seq%20data&amp;author=P.G.%20Engstr%C3%B6m&amp;author=T.%20Steijger&amp;author=B.%20Sipos&amp;publication_year=2013&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=1185-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2722" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2722" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2722"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24185836" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kekwick</div> <div class="given-names">R.G.O.</div></div></div> <div class="year">1989</div>, The formation of isoprenoids in <em>Hevea</em> latex. In: <div class="person-group"><div class="name string-name"><div class="surname">d’Auzac</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Jacob</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Chrestin</div> <div class="given-names">L.</div></div></div>, editors. <div class="source">Physiology of Rubber Tree Latex</div> , pp. <div class="fpage">145</div>–<div class="lpage">164</div>. 
<div class="publisher-loc">Boca Raton, FL</div>: 
<div class="publisher-name">CRC Press</div>.</div></div></div></div><div content-id="dsw056-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chow</div> <div class="given-names">K.S.</div></div>, <div class="name string-name"><div class="surname">Mat-Isa</div> <div class="given-names">M.N.</div></div>, <div class="name string-name"><div class="surname">Bahari</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Metabolic routes affecting rubber biosynthesis in <em>Hevea brasiliensis</em> latex</div>. <div class="source">J. Exp. Bot.</div>  <div class="volume">63</div>, <div class="fpage">1863</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Metabolic%20routes%20affecting%20rubber%20biosynthesis%20in%20Hevea%20brasiliensis%20latex&amp;author=K.S.%20Chow&amp;author=M.N.%20Mat-Isa&amp;author=A.%20Bahari&amp;publication_year=2012&amp;journal=J.%20Exp.%20Bot.&amp;volume=63&amp;pages=1863-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/err363" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferr363" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferr363"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22162870" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Berthelot</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Estevez</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Deffieux</div> <div class="given-names">A.</div></div></div>, et al.  . <div class="year">2012</div>, 
<div class="article-title">Isopentenyl diphosphate isomerase: a checkpoint to isoprenoid biosynthesis</div>. <div class="source">Biochimie</div> , <div class="volume">94</div>, <div class="fpage">1621</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isopentenyl%20diphosphate%20isomerase%3A%20a%20checkpoint%20to%20isoprenoid%20biosynthesis&amp;author=K.%20Berthelot&amp;author=Y.%20Estevez&amp;author=A.%20Deffieux&amp;publication_year=2012&amp;journal=Biochimie&amp;volume=94&amp;pages=1621-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biochi.2012.03.021" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biochi.2012.03.021" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biochi.2012.03.021"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22503704" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yanagisawa</div> <div class="given-names">S.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">The Dof family of plant transcription factors</div>. <div class="source">Trends Plant Sci</div> ., <div class="volume">7</div>, <div class="fpage">555</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Dof%20family%20of%20plant%20transcription%20factors&amp;author=S.%20Yanagisawa&amp;publication_year=2002&amp;journal=Trends%20Plant%20Sci&amp;volume=7&amp;pages=555-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1360-1385(02)02362-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1360-1385(02)02362-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1360-1385(02)02362-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12475498" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Du</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Poovaiah</div> <div class="given-names">B.W.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">A novel family of Ca2+/calmodulin-binding proteins involved in transcriptional regulation: interaction with fsh/Ring3 class transcription activators</div>. <div class="source">Plant Mol. Biol</div> ., <div class="volume">54</div>, <div class="fpage">549</div>–<div class="lpage">69</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20family%20of%20Ca2%2B%2Fcalmodulin-binding%20proteins%20involved%20in%20transcriptional%20regulation%3A%20interaction%20with%20fsh%2FRing3%20class%20transcription%20activators&amp;author=L.%20Du&amp;author=B.W.%20Poovaiah&amp;publication_year=2004&amp;journal=Plant%20Mol.%20Biol&amp;volume=54&amp;pages=549-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/B:PLAN.0000038269.98972.bb" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FB:PLAN.0000038269.98972.bb" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FB:PLAN.0000038269.98972.bb"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15316289" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nole-Wilson</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Krizek</div> <div class="given-names">B.A.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">DNA binding properties of the Arabidopsis floral development protein AINTEGUMENTA</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">28</div>, <div class="fpage">4076</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20binding%20properties%20of%20the%20Arabidopsis%20floral%20development%20protein%20AINTEGUMENTA&amp;author=S.%20Nole-Wilson&amp;author=B.A.%20Krizek&amp;publication_year=2000&amp;journal=Nucleic%20Acids%20Res&amp;volume=28&amp;pages=4076-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/28.21.4076" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F28.21.4076" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F28.21.4076"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11058102" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsw056-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Narayanan</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Mydin</div> <div class="given-names">K.K.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Breeding for disease resistance in <em>Hevea</em> spp. - Status, potential threats, and possible strategies</div>. <div class="source">General Technical Report. PSW-GTR-240. U.S. Department of Agriculture</div> . pp. <div class="fpage">240</div>–<div class="lpage">51</div>.</div></div></div></div><div content-id="dsw056-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsw056-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">J.H.</div></div>, <div class="name string-name"><div class="surname">Yun</div> <div class="given-names">H.S.</div></div>, <div class="name string-name"><div class="surname">Kwon</div> <div class="given-names">C.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Molecular communications between plant heat shock responses and disease resistance</div>. <div class="source">Mol. Cells</div> , <div class="volume">34</div>, <div class="fpage">109</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20communications%20between%20plant%20heat%20shock%20responses%20and%20disease%20resistance&amp;author=J.H.%20Lee&amp;author=H.S.%20Yun&amp;author=C.%20Kwon&amp;publication_year=2012&amp;journal=Mol.%20Cells&amp;volume=34&amp;pages=109-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10059-012-0121-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10059-012-0121-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10059-012-0121-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22710621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2960055&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/2/10.1093_dnares_dsw056/2/dsw056_Supp.zip?Expires=2147483647&amp;Signature=wwo~5vwH8uX850Sa~ZlPQlODe7NN7raDUqEtVtXBCLmDUe-12BQRulYvZjFAUmuB6~bztYacnpo7qAI5QIrt6yDczE2YAq2A6TYgJ6VNYiddKhqaPaZ8qzgWNVuvX3VS1iaurlEyjOW7A9J0gYqL384kO5zEGyMnwcwbWChcJD6IHR7NqHj4Rf7fJv~TLthYdvyDE4okkx-nW2da1R4UpkaypK6UdiUq1L5ACE2y3nCPYIhxdenJ0jAlx6QhtwuppqcMZi0Z965syKKFi-jtmi8QH3iy-uxP0TtUQTIzboafgF0rQcCDjfJ0HvzlnO7BUFZO8zhfYAur6FLi8Vz9Rw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">900</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000400913000005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">3</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsw056"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2960055"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000400913000005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f2%2f159%2f2960055">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000400913000005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (3)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F2%2F159%2F2960055" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2960055'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Large-scale collection of full-length cDNA and transcriptome analysis in Hevea brasiliensis" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsw056");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f2%2f159%2f2960055">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

