

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/5/353/449021","@type":"ScholarlyArticle","name":"Profile-Statistical Periodicity of DNA Coding Regions","datePublished":"2011-07-25","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2011-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr023","keywords":["latent periodicity","latent profility","spectral–statistical approach","gene structure"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/5/353/449021","author":[{"name":"Chaley, Maria","affiliation":"1  Institute of Mathematical Problems of Biology, Russian Academy of Sciences ,  Institutskaya st., 4, 142290 Pushchino ,  Russia","@type":"Person"},{"name":"Kutyrkin, Vladimir","affiliation":"2  Department of Computational Mathematics and Mathematical Physics ,  Moscow State Technical University n.a. N.E. Bauman ,  the 2nd Baumanskaya st., 5, 105005 Moscow ,  Russia","@type":"Person"}],"description":"Abstract.  Novel methods for identifying a new type of DNA latent periodicity, called latent profile periodicity or latent profility, are used to search for per","pageStart":"353","pageEnd":"362","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/5/1/m_cover.gif?Expires=2147483647&Signature=fZbX~i0F4YmMgemzi~KYwJoeRvQcK3vESx0rzJw5EUkahhhho4dt2jCm2s4eb4lcFKiJ3IkaVuW~mFnQ0N-xsBlJ-atzuGchQMdBMlEaTp9QoaKOLQdSjpbwxtx3IKeAKX9MAFcEHgiyPIddtsEKqmKyu2yRzRpJfM6fHjheCB8GOlWvmsNX-mG7yzjfRRAgK2-xl55jEo9vwfkLAfBWwh9slsk8CoXZ~kVyHDiYgXlvdg5m4LybAGcWIMBm9UYCeJTV~chzGeImI3yLbjJOa6lXjiaaTcisKEGSS9lHedM-W~GtV9CboT5Ke43Hf0WuKYesrRRMqe~T4KpwdHK-gg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Profile-Statistical Periodicity of DNA Coding Regions","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/5/1/m_cover.gif?Expires=2147483647&Signature=fZbX~i0F4YmMgemzi~KYwJoeRvQcK3vESx0rzJw5EUkahhhho4dt2jCm2s4eb4lcFKiJ3IkaVuW~mFnQ0N-xsBlJ-atzuGchQMdBMlEaTp9QoaKOLQdSjpbwxtx3IKeAKX9MAFcEHgiyPIddtsEKqmKyu2yRzRpJfM6fHjheCB8GOlWvmsNX-mG7yzjfRRAgK2-xl55jEo9vwfkLAfBWwh9slsk8CoXZ~kVyHDiYgXlvdg5m4LybAGcWIMBm9UYCeJTV~chzGeImI3yLbjJOa6lXjiaaTcisKEGSS9lHedM-W~GtV9CboT5Ke43Hf0WuKYesrRRMqe~T4KpwdHK-gg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Profile-Statistical Periodicity of DNA Coding Regions" />
    <meta property="og:description" content="Abstract.  Novel methods for identifying a new type of DNA latent periodicity, called latent profile periodicity or latent profility, are used to search for per" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/5/353/449021" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-07-25" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Chaley, Maria" /><meta name="citation_author_institution" content="Institute of Mathematical Problems of Biology, Russian Academy of Sciences, Institutskaya st., 4, 142290 Pushchino, Russia" /><meta name="citation_author" content="Kutyrkin, Vladimir" /><meta name="citation_author_institution" content="Department of Computational Mathematics and Mathematical Physics, Moscow State Technical University n.a. N.E. Bauman, the 2nd Baumanskaya st., 5, 105005 Moscow, Russia" /><meta name="citation_title" content="Profile-Statistical Periodicity of DNA Coding Regions" /><meta name="citation_firstpage" content="353" /><meta name="citation_lastpage" content="362" /><meta name="citation_doi" content="10.1093/dnares/dsr023" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2011/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Order and correlations in genomic DNA sequences. The spectral approach; Physics–Uspekhi.;  citation_year=2000;  citation_volume=43;  citation_pages=55-78; " /><meta name="citation_reference" content="citation_title=The study of correlation structures of DNA sequences: a critical review; Comput. Chem.;  citation_year=1997;  citation_volume=21;  citation_pages=257-71; " /><meta name="citation_reference" content="citation_title=Method to determine the reading frame of a protein from the purine/pyrimidine genome sequence and its possible evolutionary justification; Proc. Natl Acad. Sci. USA;  citation_year=1981;  citation_volume=78;  citation_pages=1596-600; " /><meta name="citation_reference" content="citation_title=Tandem repeats finder: a program to analyze DNA sequences; Nucleic Acids Res.;  citation_year=1999;  citation_volume=27;  citation_pages=573-80; " /><meta name="citation_reference" content="citation_title=Selection against frameshift mutations limits microsatellite expansion in coding DNA; Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=72-80; " /><meta name="citation_reference" content="citation_title=Tandem repeats in protein coding regions of primate genes; Genome Res.;  citation_year=2002;  citation_volume=12;  citation_pages=909-15; " /><meta name="citation_reference" content="citation_title=Translation framing code and frame-monitoring mechanism as suggested by the analysis of mRNA and 16S RNA nucleotide sequences; J. Mol. Biol.;  citation_year=1987;  citation_volume=194;  citation_pages=643-52; " /><meta name="citation_reference" content="citation_title=Periodicity in DNA coding sequences: implications in gene evolutions; J. Theor. Biol.;  citation_year=1991;  citation_volume=151;  citation_pages=323-31; " /><meta name="citation_reference" content="citation_title=On the origin of the periodicity of three in protein coding DNA sequences; J. Theor. Biol.;  citation_year=1994;  citation_volume=167;  citation_pages=413-14; " /><meta name="citation_reference" content="citation_title=Assessment of protein coding measures; Nucleic Acid Res.;  citation_year=1992;  citation_volume=20;  citation_pages=6441-50; " /><meta name="citation_reference" content="citation_title=A measure of DNA periodicity; J. Theor. Biol.;  citation_year=1986;  citation_volume=118;  citation_pages=295-300; " /><meta name="citation_reference" content="citation_title=Prediction of probable genes by Fourier analysis of genomic sequences; Comput. Appl. Biosci.;  citation_year=1997;  citation_volume=13;  citation_pages=263-70; " /><meta name="citation_reference" content="citation_title=Locating probable genes using Fourier transform approach; Bioinformatics;  citation_year=2002;  citation_volume=18;  citation_pages=196-97; " /><meta name="citation_reference" content="citation_title=Model of perfect tandem repeat with random pattern and empirical homogeneity testing poly-criteria for latent periodicity revelation in biological sequences; Math. Biosci.;  citation_year=2008;  citation_volume=211;  citation_pages=186-204; " /><meta name="citation_reference" content="citation_title=Information decomposition method to analyze symbolical sequences; Phys. Lett. A;  citation_year=2003;  citation_volume=312;  citation_pages=198-210; " /><meta name="citation_reference" content="citation_title=Analysis of sequence periodicity in E. coli proteins: empirical investigation of the “duplication and divergence” theory of protein evolution; J. Mol. Evol.;  citation_year=2003;  citation_volume=57;  citation_pages=149-58; " /><meta name="citation_reference" content="citation_title=Genomes are covered with ubiquitous 11 bp periodic patterns, the class A flexible patterns; BMC Bioinformatics;  citation_year=2005;  citation_volume=6;  citation_pages=206" /><meta name="citation_reference" content="citation_title=Structure of proteins and latent periodicity in their genes; Moscow Univ. Biol. Sci. Bull.;  citation_year=2010;  citation_volume=65;  citation_pages=133-35; " /><meta name="citation_reference" content="citation_title=mreps: efficient and flexible detection of tandem repeats in DNA; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=3672-78; " /><meta name="citation_reference" content="citation_title=STRING: finding tandem repeats in DNA sequences; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=1733-38; " /><meta name="citation_reference" content="citation_title=IMEx: imperfect microsatellite extractor; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1181-87; " /><meta name="citation_reference" content="citation_title=Spectral Repeat Finder (SRF): identification of repetitive sequences using Fourier transformation; Bioinformatics;  citation_year=2004;  citation_volume=20;  citation_pages=1404-12; " /><meta name="citation_reference" content="citation_title=Evolution of apolipoprotein E: mouse sequence and evidence for an 11-nucleotide ancestral unit; Proc. Natl Acad. Sci. USA;  citation_year=1985;  citation_volume=82;  citation_pages=8085-89; " /><meta name="citation_reference" content="citation_title=Structural design and molecular evolution of a cytokine receptor superfamily; Proc. Natl Acad. Sci. USA;  citation_year=1990;  citation_volume=87;  citation_pages=6934-38; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/5/353/1227311/dsr023.pdf" /><meta name="description" content="Abstract.  Novel methods for identifying a new type of DNA latent periodicity, called latent profile periodicity or latent profility, are used to search for per" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/5/353/449021" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/5/353/449021" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Behind_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/5/353/449021");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr023';
    </script>
<title>
	Profile-Statistical Periodicity of DNA Coding Regions | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./449021" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYQ2PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaGFsZXksIE1hcmlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW5zdGl0dXRlIG9mIE1hdGhlbWF0aWNhbCBQcm9ibGVtcyBvZiBCaW9sb2d5LCBSdXNzaWFuIEFjYWRlbXkgb2YgU2NpZW5jZXMsIEluc3RpdHV0c2theWEgc3QuLCA0LCAxNDIyOTAgUHVzaGNoaW5vLCBSdXNzaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLdXR5cmtpbiwgVmxhZGltaXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENvbXB1dGF0aW9uYWwgTWF0aGVtYXRpY3MgYW5kIE1hdGhlbWF0aWNhbCBQaHlzaWNzLCBNb3Njb3cgU3RhdGUgVGVjaG5pY2FsIFVuaXZlcnNpdHkgbi5hLiBOLkUuIEJhdW1hbiwgdGhlIDJuZCBCYXVtYW5za2F5YSBzdC4sIDUsIDEwNTAwNSBNb3Njb3csIFJ1c3NpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iUHJvZmlsZS1TdGF0aXN0aWNhbCBQZXJpb2RpY2l0eSBvZiBETkEgQ29kaW5nIFJlZ2lvbnMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzNTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjM2MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcjAyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDExLzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JkZXIgYW5kIGNvcnJlbGF0aW9ucyBpbiBnZW5vbWljIEROQSBzZXF1ZW5jZXMuIFRoZSBzcGVjdHJhbCBhcHByb2FjaDsgUGh5c2ljc+KAk1VzcGVraGkuOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NTUtNzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzdHVkeSBvZiBjb3JyZWxhdGlvbiBzdHJ1Y3R1cmVzIG9mIEROQSBzZXF1ZW5jZXM6IGEgY3JpdGljYWwgcmV2aWV3OyBDb21wdXQuIENoZW0uOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MjU3LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NZXRob2QgdG8gZGV0ZXJtaW5lIHRoZSByZWFkaW5nIGZyYW1lIG9mIGEgcHJvdGVpbiBmcm9tIHRoZSBwdXJpbmUvcHlyaW1pZGluZSBnZW5vbWUgc2VxdWVuY2UgYW5kIGl0cyBwb3NzaWJsZSBldm9sdXRpb25hcnkganVzdGlmaWNhdGlvbjsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT03ODsgIGNpdGF0aW9uX3BhZ2VzPTE1OTYtNjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYW5kZW0gcmVwZWF0cyBmaW5kZXI6IGEgcHJvZ3JhbSB0byBhbmFseXplIEROQSBzZXF1ZW5jZXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTU3My04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VsZWN0aW9uIGFnYWluc3QgZnJhbWVzaGlmdCBtdXRhdGlvbnMgbGltaXRzIG1pY3Jvc2F0ZWxsaXRlIGV4cGFuc2lvbiBpbiBjb2RpbmcgRE5BOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTcyLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYW5kZW0gcmVwZWF0cyBpbiBwcm90ZWluIGNvZGluZyByZWdpb25zIG9mIHByaW1hdGUgZ2VuZXM7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9OTA5LTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbiBmcmFtaW5nIGNvZGUgYW5kIGZyYW1lLW1vbml0b3JpbmcgbWVjaGFuaXNtIGFzIHN1Z2dlc3RlZCBieSB0aGUgYW5hbHlzaXMgb2YgbVJOQSBhbmQgMTZTIFJOQSBudWNsZW90aWRlIHNlcXVlbmNlczsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTQ7ICBjaXRhdGlvbl9wYWdlcz02NDMtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlcmlvZGljaXR5IGluIEROQSBjb2Rpbmcgc2VxdWVuY2VzOiBpbXBsaWNhdGlvbnMgaW4gZ2VuZSBldm9sdXRpb25zOyBKLiBUaGVvci4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MTUxOyAgY2l0YXRpb25fcGFnZXM9MzIzLTMxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbiB0aGUgb3JpZ2luIG9mIHRoZSBwZXJpb2RpY2l0eSBvZiB0aHJlZSBpbiBwcm90ZWluIGNvZGluZyBETkEgc2VxdWVuY2VzOyBKLiBUaGVvci4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTY3OyAgY2l0YXRpb25fcGFnZXM9NDEzLTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Bc3Nlc3NtZW50IG9mIHByb3RlaW4gY29kaW5nIG1lYXN1cmVzOyBOdWNsZWljIEFjaWQgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5MjsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTY0NDEtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWVhc3VyZSBvZiBETkEgcGVyaW9kaWNpdHk7IEouIFRoZW9yLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTg7ICBjaXRhdGlvbl9wYWdlcz0yOTUtMzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVkaWN0aW9uIG9mIHByb2JhYmxlIGdlbmVzIGJ5IEZvdXJpZXIgYW5hbHlzaXMgb2YgZ2Vub21pYyBzZXF1ZW5jZXM7IENvbXB1dC4gQXBwbC4gQmlvc2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTI2My03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9jYXRpbmcgcHJvYmFibGUgZ2VuZXMgdXNpbmcgRm91cmllciB0cmFuc2Zvcm0gYXBwcm9hY2g7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTk2LTk3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2RlbCBvZiBwZXJmZWN0IHRhbmRlbSByZXBlYXQgd2l0aCByYW5kb20gcGF0dGVybiBhbmQgZW1waXJpY2FsIGhvbW9nZW5laXR5IHRlc3RpbmcgcG9seS1jcml0ZXJpYSBmb3IgbGF0ZW50IHBlcmlvZGljaXR5IHJldmVsYXRpb24gaW4gYmlvbG9naWNhbCBzZXF1ZW5jZXM7IE1hdGguIEJpb3NjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjExOyAgY2l0YXRpb25fcGFnZXM9MTg2LTIwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5mb3JtYXRpb24gZGVjb21wb3NpdGlvbiBtZXRob2QgdG8gYW5hbHl6ZSBzeW1ib2xpY2FsIHNlcXVlbmNlczsgUGh5cy4gTGV0dC4gQTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTI7ICBjaXRhdGlvbl9wYWdlcz0xOTgtMjEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFseXNpcyBvZiBzZXF1ZW5jZSBwZXJpb2RpY2l0eSBpbiBFLiBjb2xpIHByb3RlaW5zOiBlbXBpcmljYWwgaW52ZXN0aWdhdGlvbiBvZiB0aGUg4oCcZHVwbGljYXRpb24gYW5kIGRpdmVyZ2VuY2XigJ0gdGhlb3J5IG9mIHByb3RlaW4gZXZvbHV0aW9uOyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9MTQ5LTU4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWVzIGFyZSBjb3ZlcmVkIHdpdGggdWJpcXVpdG91cyAxMSBicCBwZXJpb2RpYyBwYXR0ZXJucywgdGhlIGNsYXNzIEEgZmxleGlibGUgcGF0dGVybnM7IEJNQyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MjA2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlIG9mIHByb3RlaW5zIGFuZCBsYXRlbnQgcGVyaW9kaWNpdHkgaW4gdGhlaXIgZ2VuZXM7IE1vc2NvdyBVbml2LiBCaW9sLiBTY2kuIEJ1bGwuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTY1OyAgY2l0YXRpb25fcGFnZXM9MTMzLTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1tcmVwczogZWZmaWNpZW50IGFuZCBmbGV4aWJsZSBkZXRlY3Rpb24gb2YgdGFuZGVtIHJlcGVhdHMgaW4gRE5BOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zNjcyLTc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TVFJJTkc6IGZpbmRpbmcgdGFuZGVtIHJlcGVhdHMgaW4gRE5BIHNlcXVlbmNlczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xNzMzLTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JTUV4OiBpbXBlcmZlY3QgbWljcm9zYXRlbGxpdGUgZXh0cmFjdG9yOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTExODEtODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwZWN0cmFsIFJlcGVhdCBGaW5kZXIgKFNSRik6IGlkZW50aWZpY2F0aW9uIG9mIHJlcGV0aXRpdmUgc2VxdWVuY2VzIHVzaW5nIEZvdXJpZXIgdHJhbnNmb3JtYXRpb247IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MTQwNC0xMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIGFwb2xpcG9wcm90ZWluIEU6IG1vdXNlIHNlcXVlbmNlIGFuZCBldmlkZW5jZSBmb3IgYW4gMTEtbnVjbGVvdGlkZSBhbmNlc3RyYWwgdW5pdDsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MTk4NTsgIGNpdGF0aW9uX3ZvbHVtZT04MjsgIGNpdGF0aW9uX3BhZ2VzPTgwODUtODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyYWwgZGVzaWduIGFuZCBtb2xlY3VsYXIgZXZvbHV0aW9uIG9mIGEgY3l0b2tpbmUgcmVjZXB0b3Igc3VwZXJmYW1pbHk7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9ODc7ICBjaXRhdGlvbl9wYWdlcz02OTM0LTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTgvNS8zNTMvMTIyNzMxMS9kc3IwMjMucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIE5vdmVsIG1ldGhvZHMgZm9yIGlkZW50aWZ5aW5nIGEgbmV3IHR5cGUgb2YgRE5BIGxhdGVudCBwZXJpb2RpY2l0eSwgY2FsbGVkIGxhdGVudCBwcm9maWxlIHBlcmlvZGljaXR5IG9yIGxhdGVudCBwcm9maWxpdHksIGFyZSB1c2VkIHRvIHNlYXJjaCBmb3IgcGVyIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00NDkwMjEmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkX4cToc0+1IndTLy1tm76iOmO6Ik=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f5%2f353%2f449021" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f5%2f353%2f449021" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17545" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/5/1/m_cover.gif?Expires=2147483647&amp;Signature=fZbX~i0F4YmMgemzi~KYwJoeRvQcK3vESx0rzJw5EUkahhhho4dt2jCm2s4eb4lcFKiJ3IkaVuW~mFnQ0N-xsBlJ-atzuGchQMdBMlEaTp9QoaKOLQdSjpbwxtx3IKeAKX9MAFcEHgiyPIddtsEKqmKyu2yRzRpJfM6fHjheCB8GOlWvmsNX-mG7yzjfRRAgK2-xl55jEo9vwfkLAfBWwh9slsk8CoXZ~kVyHDiYgXlvdg5m4LybAGcWIMBm9UYCeJTV~chzGeImI3yLbjJOa6lXjiaaTcisKEGSS9lHedM-W~GtV9CboT5Ke43Hf0WuKYesrRRMqe~T4KpwdHK-gg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/5">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6726557">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6726557">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6726559">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6726559">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6726574">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6726574">2. Methods of identifying latent profile periodicity in DNA</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6726627">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6726627">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6726690">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6726690">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/5/343/448814">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/18/5/363/449181">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Profile-Statistical Periodicity of DNA Coding Regions
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Maria Chaley</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Institute of Mathematical Problems of Biology, Russian Academy of Sciences</div>, <div class="addr-line">Institutskaya st., 4, 142290 Pushchino</div>, <div class="country">Russia</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +7 4967-318537. Fax. +7 4967-318500. E-mail: <a href="mailto:maramaria@yandex.ru" target="_blank">maramaria@yandex.ru</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Maria+Chaley">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chaley M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chaley M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Maria Chaley</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Vladimir Kutyrkin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Computational Mathematics and Mathematical Physics</div>, <div class="insititution">Moscow State Technical University n.a. N.E. Bauman</div>, <div class="addr-line">the 2nd Baumanskaya st., 5, 105005 Moscow</div>, <div class="country">Russia</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Vladimir+Kutyrkin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kutyrkin V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kutyrkin V&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Vladimir Kutyrkin</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Hiroyuki Toh</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 5, 1 October 2011, Pages 353–362, <a href='https://doi.org/10.1093/dnares/dsr023'>https://doi.org/10.1093/dnares/dsr023</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">25 July 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">07 March 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">13 June 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="449021" href="/dnaresearch/article-pdf/18/5/353/1227311/dsr023.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Maria Chaley, Vladimir Kutyrkin;  Profile-Statistical Periodicity of DNA Coding Regions, <em>DNA Research</em>, Volume 18, Issue 5, 1 October 2011, Pages 353–362, <a href='https://doi.org/10.1093/dnares/dsr023'>https://doi.org/10.1093/dnares/dsr023</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/449021?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/449021?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/449021?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/449021?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/449021?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/449021?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/449021?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Profile-Statistical%20Periodicity%20of%20DNA%20Coding%20Regions&amp;publicationDate=2011-07-25&amp;volumeNum=18&amp;issueNum=5&amp;author=Chaley%2C%20Maria%3B%20Kutyrkin%2C%20Vladimir&amp;startPage=353&amp;endPage=362&amp;contentId=10.1093%2Fdnares%2Fdsr023&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f5%2f353%2f449021" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6726557 id="6726557" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Novel methods for identifying a new type of DNA latent periodicity, called latent profile periodicity or latent profility, are used to search for periodic structures in genes. These methods reveal two distinct levels of organization of genetic information encoding. It is shown that latent profility in genes may correlate with specific structural features of their encoded proteins.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;latent periodicity&quot;">latent periodicity</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;latent profility&quot;">latent profility</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;spectral–statistical approach&quot;">spectral–statistical approach</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene structure&quot;">gene structure</a></div>                            <h2 scrollto-destination=6726559 id="6726559" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The notion of latent periodicity in nucleotide DNA sequences has arisen from the discovery of various regularity levels in the structural organization of a DNA molecule. For example, a DNA double-helix pitch has been found to equal ∼10–11 bp, a length of ∼200 bp has been found for DNA fragments in nucleosomes, and a loop length of ∼2 × 10<sup>4</sup>–10<sup>5</sup> bp has been determined at the higher level of quasi-regular DNA compactization.<sup><a reveal-id="DSR023C1" data-open="DSR023C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Such particularities are probably due to non-random alternation of bases in the original DNA sequence. Thus, research on both short- and long-range base correlations<sup><a reveal-id="DSR023C2" data-open="DSR023C2" class="link link-ref link-reveal xref-bibr">2</a></sup> is of great importance for understanding known structural particularities in DNA sequences and for revealing new ones.</p><p>Images of various functions demonstrating nucleotide correlations in DNA coding regions show regular peaks with the steps of three bases corresponding to the triplet nature of the genetic code. This has led to the notion of triplet periodicity in the coding regions. It is likely that use of the term ‘latent triplet periodicity’ has also been influenced by the hypothesis on the universal RNY triplet pattern (R, purine; Y, pyrimidine; N, purine or pyrimidine) of codons in ancient genes, because correlations with this pattern can be traced in base distribution over recent coding regions.<sup><a reveal-id="DSR023C3" data-open="DSR023C3" class="link link-ref link-reveal xref-bibr">3</a></sup></p><p>In light of the current understanding of latent periodicity as approximate tandem repeats,<sup><a reveal-id="DSR023C4" data-open="DSR023C4" class="link link-ref link-reveal xref-bibr">4</a></sup> the occurrence of periodicity has been corroborated by textual ‘consensus pattern’, which is an estimate of a pattern in the original repeat. If alterations in the copies of the pattern account for more than 30% of the pattern, the validity of the revealed consensus pattern is in doubt. Although tandem repeats of tri- and hexa-nucleotides occur in coding regions,<sup><a reveal-id="DSR023C5" data-open="DSR023C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSR023C6" data-open="DSR023C6" class="link link-ref link-reveal xref-bibr">6</a></sup> as a rule it is impossible to deduce a reliable consensus pattern of an approximate tandem repeat over the whole length of a coding region.</p><p>Weak three-base periodicity in <em>Escherichia coli</em> mRNA with pattern G-non-G-N, complemented by pattern NNC in 16S RNA, has been considered as a mechanism for monitoring translation frames in ribosomes.<sup><a reveal-id="DSR023C7" data-open="DSR023C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Despite the discovery of a few short tracks of corresponding patterns, this case would be more accurate to consider only as the domination of G and C bases in corresponding triplet positions. The weak preference of a certain type of base in a fixed position of triplets in the coding region promotes the appearance of a dominant peak in the Fourier spectrum at a frequency of 0.33 corresponding to the three-base period.<sup><a reveal-id="DSR023C8" data-open="DSR023C8" class="link link-ref link-reveal xref-bibr">8</a></sup> Nevertheless, this preference is not the key cause behind this observation. It appears that the greater the variance of certain base distributions over period positions, the more impact such a base has on the amplitude of spectral density at the three-period frequency, even when the base is not dominant at triplet positions.<sup><a reveal-id="DSR023C9" data-open="DSR023C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Thus, appearance of the peak at the frequency of 0.33 in the Fourier spectrum is due to non-uniform base distribution over the triplet positions.</p><p>Use of the Fourier methods for revealing imperfect periodicity<sup><a reveal-id="DSR023C8" data-open="DSR023C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSR023C10  DSR023C13" data-open="DSR023C10  DSR023C13" class="link link-ref link-reveal xref-bibr">10–13</a></sup> has become common. Other statistical methods have also arisen for determining latent periodicity in nucleotide sequences.<sup><a reveal-id="DSR023C14  DSR023C16" data-open="DSR023C14  DSR023C16" class="link link-ref link-reveal xref-bibr">14–16</a></sup> These methods are based on measuring heterogeneity in base distributions over period positions.<sup><a reveal-id="DSR023C14" data-open="DSR023C14" class="link link-ref link-reveal xref-bibr">14</a></sup> In practice, in the absence of weak periodicity in a sequence that is not an approximate tandem repeat, a high index of heterogeneity and a Fourier spectrum with a dominant peak may be observed. It is incorrect in this case to use the term ‘latent periodicity’, until the discovery of a pattern indicating some new type of periodicity, for example, such as the flexible patterns<sup><a reveal-id="DSR023C17" data-open="DSR023C17" class="link link-ref link-reveal xref-bibr">17</a></sup> of 11th nucleotide periodicity in the genomes of prokaryotes and low eukaryotes.</p><p>In the present work, a spectral–statistical approach (2S approach)<sup><a reveal-id="DSR023C14" data-open="DSR023C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSR023C18" data-open="DSR023C18" class="link link-ref link-reveal xref-bibr">18</a></sup> to identifying a new type of latent periodicity, called profile periodicity or profility, is proposed. The notion of latent profility in DNA sequences has been introduced earlier.<sup><a reveal-id="DSR023C14" data-open="DSR023C14" class="link link-ref link-reveal xref-bibr">14</a></sup> It expands on the idea of approximate tandem repeat<sup><a reveal-id="DSR023C4" data-open="DSR023C4" class="link link-ref link-reveal xref-bibr">4</a></sup> in which textual string (DNA sequence) is presented as a chain of eroded copies (with ∼80% identity) of the textual pattern. Latent profile periodicity occurs in DNA regions where nucleotide correlations can be described by hypothesizing on the generation of successive uni-length DNA fragments according to a fixed probability distribution of bases appearance at each fragment position. A pattern of latent profility can be described with the aid of a finite random string consisting of independent random characters with corresponding probability distribution for the textual characters from the DNA alphabet. Excepting certain cases, the DNA sequences considered in this work could not be identified as approximate tandem repeats. The usual methods<sup><a reveal-id="DSR023C4" data-open="DSR023C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSR023C19  DSR023C22" data-open="DSR023C19  DSR023C22" class="link link-ref link-reveal xref-bibr">19–22</a></sup> applied for identification of approximate tandem repeats therefore cannot be used to reveal new types of latent periodicity.</p><p>Via a procedure that identifies patterns of latent profile periodicity in DNA, it is possible to reveal the two levels of organization in encoded genetic information: regular heterogeneity of nucleotide distribution over positions in the codons and latent profility. It is shown that the Fourier analysis does not enable the second level (latent profility) to be distinguished in the genetic encoding. The Fourier spectra for the DNA coding regions have been built using FFT programs,<sup><a reveal-id="DSR023C12" data-open="DSR023C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSR023C13" data-open="DSR023C13" class="link link-ref link-reveal xref-bibr">13</a></sup> available from <a class="link link-uri" href="http://www.imtech.res.in/raghava/ftg/">http://www.imtech.res.in/raghava/ftg/</a>.</p><p>A number of examples are given when the latent profility revealed in the DNA coding regions is translated into features of protein structure. The direct detection of such features is challenging because the goal of the search is not <em>a priori</em> known. The present work shows that it is possible to determine signs of local features in the structure of genes and corresponding proteins through latent profile periodicity (latent profility).</p>                            <h2 scrollto-destination=6726574 id="6726574" class="section-title" data-legacy-id=s2>2. Methods of identifying latent profile periodicity in DNA</h2>
<p>In the proposed model of latent profile periodicity, a DNA sequence (textual string) is considered as a realization of a special random periodic string called a profile string. This string, consisting of independent random characters, represents a perfect tandem repeat of a random string, so called because of its random periodicity pattern. Therefore, in order to determine the latent profile periodicity (profility) in the DNA sequence (textual string), it is necessary to specify the criteria for considering an analysed textual string as a realization of some profile string. The random pattern of such a profile string is not dependent on the quality of the periodicity pattern for the analysed textual string.</p>                            <h3 scrollto-destination=6726576 id="6726576" class="section-title" data-legacy-id=s2a>2.1. Statistical structure of random strings consisting of independent random characters</h3>
<p>DNA sequences are considered as textual strings in the four-character (<em>K</em> = 4) simply ordered alphabet <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq1.gif?Expires=2147483647&Signature=bED3R54fh8Q96U~ccZ-Brsvj~pQxPu4WnV3gLRp89TA7TdWGq3msVS~F3kmP0IK60q5p6OeN2Q48kTi8zXr~y-6JlrMvoANMVTbViFTdvGzmZFRitW1CYeBbWuN5SvSFJdXnOmO5Bq~hJHLr5DgDDDb68jsI1KXxVSePhjAX7oS2~omqONGOSCzVFko55zHZxohB90m~VX~7gKBDy63oqUYDrkGz~cqOpjscAi5IxsVx0-LhIRctgtQMLM5RPHWaprk8LtftjlxHAoMMaR2Si3~Op2dNb7cCknz6OAAWuOoj4qaFIhkJs5DbkCXJM-jQlj94j7CteZ8ePs3VZ6zzfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq1.gif" />⁠</span>, where <em>a</em> is the adenine, <em>g</em> the guanine, <em>t</em> the thymine and <em>c</em> the cytosine.</p><p>Let <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq2.gif?Expires=2147483647&amp;Signature=Px1fNjLqR6xPrDMVGsxlfkrLxSdY4xknqQ9F9lu0YYbsDfKIBGXanX9tsBpGE0~B19axn5VH407ZTLCK7RrxDjWjcEzkn1TBR3GLbbCIm5JjITxBZg15ddJvhoRnt2NEEEMUGgAp9ZvCSNmqV8v-PmAXBZGiVKVJLxYLzdwc-tcfbpEugEjkNXA1z7QHY3hdCwZ5rbx-OFo92Z-OeFU0UDtDygIwE5B37EP2F6nNAkjKO0jWPWHPyBgw-NDV3GS50~Bc1L0ilkBVn4Px6N5igliRPBD~0RKgfpNtnC8Zc6mLkqgyrD-2pt1uHAmjs8f5ETkr70wmdMoT1DVoIi690w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq2.gif" /></span> be a random character with the frequency column <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq3.gif?Expires=2147483647&amp;Signature=qqlh6M3gsTYtfJvW~yTwK8xGV56eriDMAMw-Zz5vTiEPTxU3Y47fhouk50wJWi0d4~L0NaEwX4MxfcsLNUa56hHgPRVWChMkEEEBZoiKcrzPafe09AdPWq0cFziL6ofEYWNkqsIXY9PKSlF5yKC~cgZHn0xF7qrdCL4SO85LM1x6OpTkHbd-c8eQd3eDBWYVNjHH6fh-SGcEMArVEn2KtvTjKfV5atD89TTV9ZKVU8M7jCDiSeq2GSxpqKvVMdZVeYo-QOI~wQWMG9hxqFD2TUvRL67yWOYuSH7c0hFGPnBGPhcB3VmNf~uP3e10m4u~f2s-ljfOljJ7lxkbWS0QFg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq3.gif" />⁠</span>. Such a character is a random variable that takes the value of the <em>i</em>th character of the alphabet <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq4.gif?Expires=2147483647&amp;Signature=e9Z5~sXK~BwMQ9t9gdKoS-LPEFhPIOIW15eijfCH7i4sn~Zoi-JNPYhOk76Jj3lct~QkMchmJYjVHr-bH9L9hnZO9aYLJhgp6SeK4irmGy7gU2pRmUALpzMCVQD~hOU3ohiUO5avRidLg4sLJf79BEAhWbkWguxMElqk7cAMwas3xwNIaoEl~ieWFcOM1lvK1FjvcSbwKIl3lbSuJgbgC3lLrFOx5uhIFemv0S~R8E03KXOBdnIhfN5EyURWdsTzoeYWU3uoYRTt6uVixCQe-JaYBOLGGQQ2DSaQP0~tPWjcch7U8c6s6s4HaX0DnNV6nYWvLW~MhvxHboVfcTrTiA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq4.gif" /></span> with a probability of <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq5.gif?Expires=2147483647&amp;Signature=YAYkjj1TN0G7ZDrx-p7CAV-ILkatdOuU7fF2VhdssAzMVQBwq7W~o4DvYgkzTbgX0uEKdn-37diVmE5r2-yWLPbI1afdNUJ9cQE6XtACULxf~xoaB92lVQBbUjuJpIJrWg-RswGlZO6S~sMpwJA1SjqG5yLZoBM1uj~Oq9GKC25uTxkEqCyLtdDp5yAVaslw2KUBtNnobpehBLlOSwPI79DiRyA-r4DZksE3xxPa7dZHiBYo41zXRURBXJya3KS5rfbMpJB1nz4XJJ~ybgDmdMLjHqJWvT3JJkk0CgP3A~d-i6v3IDI8~CaKjpCvggcEQ0H5jfTlKaetpUps2XCwfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq5.gif" />⁠</span>.</p><p>A special random string <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq6.gif?Expires=2147483647&amp;Signature=AvSpj6TkCmJANzBAe6JyCLLx1BJq0TCBo-b9jrD2w91MDvWV95EECxJhP3PUz0OqsOSQ-r2FKK6D2DRCzS8GYlH9Zm3csHs75dlGyZG~7ISDvBXZvdH2gIQVbSA5RHPY25yxWkok0Nm9zmBZk-K38tv~sfVnXwg8FOI5n~ujwNwMyrcsFLnMltEstY50CKY1z-t-k8y6JvcJjztj91Hsoj2M3HQfJklxhpreVNp6z-bBveEuM9fRWAI8VJIumvhhSH-OzBpEHuL7QRG7lStI5bzNPYqMzYXYZ7DhDRuNAXkZ9Zsnn0GIrxtO9eCBMl5eaAQ73aAjtZOmA8P8XA9jCg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq6.gif" /></span> of <em>n</em> independent random characters is induced by the matrix <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq7.gif?Expires=2147483647&amp;Signature=qFEKUS1uZtPTeyW-lxyQfSJ5y7yM05793QOT~2B~jcDn29qGv7Fupar0m5k7DM6shx3zk8Nx3Rd3q1~kCvfj7OxalW1S3vKGAXov0HuwLfy4S-95X71iyPsxdlNSW4AzfDZNReUtolvlxCI8qsv6WFMgaIvGeMaoY4kAFctYpxcJJvjs0vrouEIBBT7iqG5qbPirKcIQuE847lPKpyyRxNddQn0SYlWLUeKzOOPJ5f9C8deJ-aEVBI6ly~68h31IrcfcrcopwUNQO~QSMN84HEd0o5xEerGBHHmJU5iRnVHubyEOBppFk~AwlZizFLvhfFQslw4vSn6Hd1~38SgplA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq7.gif" />⁠</span>, called an <em>n</em>-profile matrix. Let <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq8.gif?Expires=2147483647&amp;Signature=ar8fKAJuQHk-0Z9RHbhSrfHzVahXbsEIsrAOIo27xr05uRb1-ks9ZvL2ebdNyQc8JYL4ZS9MzRathaloMZzC0SuBxOuAZBECeFnIFkzse0NdkAmRa3RfLEWeTcClqzfUBJiTvIbsWjNjkDma5j~~iJasZA0Q6vtN0sxtUUhaknW08oynYDZFAieqnD1nTmxX5Z3ohiF7HtZ1hnuJJqe5aC92-OE4jcpNyrn6NSmzaYiLDHK5EO5Ud2So1VYPYphN-c2ENUQxT7h-dTFK1da5nbHiOkF83gKZoCAqy~bcxZQnLZ5p008kywsFZpN8EL3QQjC4UUB8U5MdXhgLCjOBPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq8.gif" /></span> be a textual string, where <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq9.gif?Expires=2147483647&amp;Signature=k8Ye~NOP6H-e8~YrZuxF5D~BYaJthCJDO2SKTxHOhoO8rIF4VxSNaWIp8rNcdaqhYxbJQJ4YvaeAPhS-uR8Q2z6vAw~YpLFq0DgyDwoPKsW5UDW8pGmg1C3s3170F3wIV6nLk2YnJmj2RTIGJXYi82gZvRG3sqeNCfj4E~nK~2uyKdM8OmKXQouYX1EsoqIKkmP7Xtnv5HKsAWzYOCzgpbEAxE8ncv35GGHBNNksdES3m8a9EJZLG7ofI0BiT-EzrMwF3I7xZDRQn3w8YmIJRJK-96XbfOJ5yufkqL0p4yiZnP-Zlhvgqz18eqQrujP3Md5M8ZrHmm3m71Q8M1b9Sg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq9.gif" /></span> is the number of a character <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq10.gif?Expires=2147483647&amp;Signature=Cqb3kuld~CCnQvyAo7I1TnhWOtFYzwooVjN4rhP5fGh5w4l2thiuqzCxUfgUCRChoDWiyBRcWbMOpcGA8yDpTVPjLAZyEMWP7Weit79kTqvSE4IFdbzzrbLOqP2xnlVE4JuU2ZfoZVWEd~Ao-xvfQrg8z4xXwWEUkDINeuo16mNAn0MIm7jDYqgghbdKytlDyvpDTwOcsI7o62P96fra2NlKk1cJ3fuwv5gmHQ2k4bKe0ea2pj-fCDgatt1ckpsVq~6DW1QBkK-05ldAuuezuzcHWJBgAX5UoiaCdXMvhC~tQCF8Al-cjWqOAQEYjQGwlFKaKBE56c43WtmOc6POTA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq10.gif" /></span> in alphabet <em>A</em>. If the str is a realization of the random string <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq11.gif?Expires=2147483647&amp;Signature=YAA1wZXeMYBiyKFflj0QXRxY2saDdOdJogdSbIZRKsNElJjAT1w4yaJnQCXrP~bB2osJaXHINpMK8GAPXXFOWAU2TFZzkf1Yd7Em1fISaWvT6-OL6-vqMKrHv924GF7b-3bi0~IYVIkllBn2Os3nHX4Gxx2totf8otnMEy~TnY4KjoJyapEytwNxQnPqW6pGtFw3xglgc8iM-H-Pqd5o~bDjdVLs3dQJ-tWZbPq~6KpTik7IOtriq~P5KCxYR55c5Uy6ZFktJxWCc-sN05gLMhhqAZzDzwYWTnKsA1VosJ4u~2Z9YxhCjd5gKlWfWbLXhehbBv5~Bi5woko1pfj9bQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq11.gif" />⁠</span>, then the product <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq12.gif?Expires=2147483647&amp;Signature=aWt1~xOptPELAZdosOpu~CfVH79t7tyUqaok7CC1JraHkmrwPI5EMwaIHOd93ptPyv1ASCrZ3Re-CJzPg1xN1AAtnmgc8Mp6bwRDpDAIN1zKatRW6dc2mp1QdBU8LTnjNyjb~wssvmhxuXXY3uYEX-wdGW03vTjlx-FqpmG9wttwZmo1fLRkjdIN~clx5cSyCfjgQxxcsN0mN89lbPZRM-QmZ6kTY3P8cSqFEG9SWlXzSHdsW~0VkpgD4tuTXzjohewviMM0Q2MJJsgeucCxokvB7rUzNu2ai7bQCZRu5W2Xo253D1B22JfvuhBzUKbcRepiq-f9H4w2nKNfUjfPdg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq12.gif" /></span> determines the probability of such a realization.</p><p>The character <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq13.gif?Expires=2147483647&Signature=1BUK8rHokuWecEO~GTrpq~lh~jGy~tvtDc~o4amF~7SjMNqHWkQ-sb9wT6LuhvQOQoZOcHL~YZ0YyOcDNvihnpYSq37xjsGRAVLuMDyK7qshkcYMQUWIHTIg5Dv4VWlSLI~2l1iU62kYMAwyyPqeZMU4OwHT-spwr7H2VK~J8AdS-LSdF7dganHnAHL37HkIBc03-dsqtkiKGvt2tE2ZBqHu2sizYdgtemWc9btLjUmIMrGVJPwrCc5ZUjlOs0TOHmrd8t~iGXnPD1G8uFpQCmx-OpokT1Vuqi6YAkNKvqoea4T9RxXWivmXHqJOYyHPu5nRN8QyLoIUHAcoaIyuVg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq13.gif" /></span> can be identified with a random character for which all components of the frequency column are null, excepting for the <em>i</em>th, which is a unity component. Therefore, any textual string in alphabet <em>A</em> can be identified with the corresponding special random string of the same length.</p><p>An integer number <em>L</em> from the diapason <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq14.gif?Expires=2147483647&amp;Signature=4mNTyzRrk9KD10CGQUa-0svGOKQ~dnBROtb25faoP0qSYo7vfyjgaW5NogbK9m4t1nIrKN~7PC5aPClceSumazB5INsYHrz0hCN-18E3UKo7N3l3SJcZsk2o4JX2XlnU3wL9A3yc0UKdwcCPNbcZsY16hyrvph1Ab33ZQaNkPk8XrggNGgXivRJ89id9gT0-6xNrI~a15g4dT3cUd0unzxqiXH6gW2vKInodwzP6n-IYehrfIe6cpHkWVrnbgmiHAmvhM752zT449Y7hLa8V1B5B4L1nYh5Xg28E-rtyf7KKPg0AVNFa4evulH2gmC401ko7CATdLdWEeMUqr21oFA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq14.gif" />⁠</span>, where <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq15.gif?Expires=2147483647&amp;Signature=4spL28Jtwe0UxkjbtGe1x-12cmHHWein0flyIvyyImSccKIcPDYJ-IMA9qTC0mjoVgakVQdre2dm-z20aZjYUyIvT0RqqEJLMP2AB~LTpuzNR-yHnOM4hethyMVT9M0ooid69cc6NMXBBUvqpNj8VEqVZFQzZNUbNzE-Ypm0sPktUsbdlGE8XrXdcA~JydEePl9-oEEaRmeK6TvQ9kWYHztWA9sgN2SJuzhC~FbR6A5rLbXdiPM0ZVoOdIqr23zLpkz1IpgR0OiFSbT376b-alP8dkoc-8DTXuvJM6kZluXU-hQhalVbEbiCpmfIDloJ~Z4y4WuO66v6jnL3ceH1wg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq15.gif" />⁠</span>, is called the test-period of the string <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq16.gif?Expires=2147483647&amp;Signature=r~ZdlUddKSjJs45D8EO0vIt9P~Up5NQQTQE3ReiiA5xG3CIZYNjcE6QRh1u8J67S7otOTcZtyZrai~C5aGBV9yNJ5z2xzKkXQrIADfFCpu-XXygLNoBJlpNejl-AqbAYevh4feUfsyTMaAOOjYUK8bT7Msi~8ZUlsA0wI0A6jegHBufN2gyI6yzqR5stGh1mGeWPsA9q~vL0h0byIZvpN0BbFnh9AH52VSMuqtoXyLIvHKRivgEFOWOnJCYfNIf4p-aiBpnfUp3ZFHELwRYtHSo1Ozc4bmgsmHOXyM-Z-6J2GgV~SvlLGRmsxnVyPty0zxtM6G0-0edhFaZW1jDUTw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq16.gif" />⁠</span>.</p><p>Let <em>L</em> be the test-period of random string <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq17.gif?Expires=2147483647&amp;Signature=zq5UaxIPbrz028iDhqdeHCcHLEFa6sdHKPrXM2F3AtL6nmWTeNaE3lWoy59PF96IDkaXVMTPeB6Zq4n5Y-zuKhgcGN32MuaB4U5nqZiCWIxdRbOtjSTMG5Nnp8D6U40q84SzPpEC3XaXjbfamk7j64gZwT-VHEZW1eaRzwOcgpUgCO7ErcseAQNlyxKD4JfMNJ1ZNehnD3mlkzEKPFq-bOQFTq0ZnwlmnvnE4BQiLjqOzHOAgykBLT-62rBeRIKSY3trKiZBB9EuqxLytMz6yIW~-~0RIiD-ZXHmOQLmrPwYQGnThxGe6F~znO8SAiKHUetGmq6y57vGO5lnT7z6ng__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq17.gif" />⁠</span>, and <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq18.gif?Expires=2147483647&amp;Signature=GD3hL90SLhFcT~HDqEXrZ851x-vGxRQF7mDPN~5TyaCB4EY9J~dv6Q4TMasDfErY9Ts~m1MmD7w-aYDq605zdQsOSoOft-tStb0LMO~480azMbmLtDspOxnry0nxpbRmCGQ-kPCoDpKecLpheCAGs4rQauoP3ve8~RZmZKHoyv3MIyx~~ketTqZpoIBPtbUFMLyLGC80nQwbNdwlKhCQwNPgUcqOgrVN8guE9SwmqG7tj3x275KRkH95PbSnzf66GcMuvX4BjAVe~-TTKOazZ5g86Pf7ADmKdHn4TgKxwqe0Yzwxkhy2Di56KY9evFLfWLLqeoCVvfgIQ9fTF1u1Iw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq18.gif" /></span> be a decomposition of string Str into substrings of length <em>L</em>, where <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq19.gif?Expires=2147483647&amp;Signature=feA-Kcf7HHTuIuTePv86gugEw6xTMtXzNxrhC-0VWN9MpbmMclXdVjRpMiLSFvgGe6cX6-vFF15wVXxsCHoarUgG3Fu41OAVHBDP-PHxzZtSbvcfHA5cSLzR7eiHKATAS86RTGcXzeyTavaX2anFcoucypnxPo~U354nhk4KkgOXIR7G-BZffhMkMAG5VF4pRLm~OFflXCROB7j3SChHXz6MhyBS6De7VDBnwsZ-PiYpen2CCRnMIqEcCh3Pdqpt4rJ2TgZO6BxFUX9mar0Wqvn6XAtwQfw4pdk7EVZtIn3ime~TDG1WzwNOYSnw-KsAW3hguEM2JTiM51E--E-bVw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq19.gif" /></span> (if <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq20.gif?Expires=2147483647&amp;Signature=mhqCUaAIi2dbiQSh0flUs~nmUyEki0sU9erM02nnaTi7UzMj6bSWDfw~f6q9f4wC9jJB2FVIMhV09m-SADt1HA21nK8yS-Mj-tn3VCdQg0BoNIEllvn1Ru6mizyH1uyRphQ7OFyhxkfUsaUe0SYBg220~V9xNkfBwQS~Xiw-87QmI38kdSAxO1qhW-oSpMAMbkVG8OltBLj~nhYdSQge-bqCPybVRnpBMrDuT4COG6QhqDXQ4T5t2-RwYljsXGDl~HAHKS0YfcBA5gyVeXoNtjUT~JopXppFvm0nTPEd2MPwVjtFv~Zrw1VO2rPEhQiYf80-a8AEETtc3WLz0Z7YaA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq20.gif" />⁠</span>, substring <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq21.gif?Expires=2147483647&amp;Signature=Z8CyNsMEJ8w1KZXTOG6gSVlNcVHlu-drA-rscgTPXVJ7ht0~kABZu8VnuQhzkk4RzAh9x9xuy3Xa6-tl8TAlQF7iEEsZVBz8AV~F16F7LmITwDcWuvDoveC5mXVS62HkIQ0Ut8nL4AxmHYGfPmDCNKDNeSHE5NMq9iCQHEnD8mqNt7gIVSL1tVMl~FaAQB4lIUxa1yu9tAuxYjbGrJ70ZbkZuYdo5u2424hrsN~F48RynGC5-x66r-F~eae2JK8VYzEzXHnO~h0QOwUtR7ugy8t4pd~K5VoqicXjS8HGN9itmLNbUN1gljP2JPrjAWL2vGdjx7DtuKY2L63L47MLEQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq21.gif" /></span> is not complete; if <em>M</em>= 0, substring <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq22.gif?Expires=2147483647&amp;Signature=HDc2S7ooQhrvhefNJW7HmnoPPK0kGONpQaA~Xt4n21VhJv1tCPcOc8lDQVB4~aVYKvYUi411iafUmJWquOeIOT0cBxyejAK8PN-NxvwHjRMt-8kuY35bQRsJYKvKRjopPicDhiq9oyYOH06aBP8KzqmdiA4xGeAT3JDSTGk9U6Er5r-~X7VofduBgnwV2f7IZhQVHMu1pyE0JFkhut6FyEo5wuCBCC7jNQbllGpf~YLeGXP4Klxewi4StGSRDIBGfAq0aAHNKNywQgZ5~CYbQplCBe~OtDVpI~hr-DN5pzkN~TisdIDMZYVc49gw9ORcJvsxCt5TaRX4BpT-zG4UGw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq22.gif" /></span> is empty). Then, if <em>M</em>= 0, the matrix <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq23.gif?Expires=2147483647&amp;Signature=ERegxa~1CFzFg-JNP~qT11xpeNPTI4H3H~qDrTlhBWGZLaWi-yf-qiizb6HvMT6fQhJpASLhEWJrznQq9G1aLEZT63q~7Ee~pkByRa3BnpR0ng79Mo3bnVVjFBa-5APwfKdPeSXmR0q8MacwOk3p6WSypLX7~mWyQIcpB9LsF6q3UydlBJffVvVHMpHeCE0CMYoJ0Kmy3dpCQXBzBD9Uv-X1jC1bppdZypUWGQ7yJTgDDL4V3EMiCacT5PBnaCdc~MKsGYqlQL35~KlHwxfapGncXjv7xwZBTCf~SnAlBhAlwibGVvFOsdMHKdQ6-B1QspCHcefuXszwCuEcHXYs2Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq23.gif" /></span> is called the <em>L</em>-profile matrix of string Str. If <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq24.gif?Expires=2147483647&amp;Signature=QbIBQixehHF-X01vJ55hBzSZTWjMGdFHhFcpUbpTeo-hx4iQMWs2YpsN-LEb8e8U7I1WV1nPeiZ3ApDU5V6Q0ZMmAItL7-qlN12MUxmBM0VB6GCMqvUMCzDUTyfgX3dlD7ou4h3S0H6K-ijmd5O9Jiy7YWkP9XFX8BknxOLOqNOJb2Ek4JTvpMd1nQld9lwBDtWO85dPrERtzC599g5Rn9N2qUuzL4ytBU0CTDIxd6gaGzLRquWm0ro~mZi59evW~EwPnoPDuQHjmKWhQwPK1iReXxazy2bD~Y7p9wBREJIXfrt7AmElOkL6X1MYGhBtaNGCFpj4OLb1PQmsFv8qpQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq24.gif" />⁠</span>, then corrections are made in the <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq25.gif?Expires=2147483647&amp;Signature=hjtbjgaFOe1AIJTpoYA9AiZP1lPE6F2WZfCRy4-TaY~y-9EVxZPaYvIfXAAp5Z8hyQhQXcKxr1qJn7Zs1AZXe~nEn9VcLA9thsxl8sH8uWq~Gp4fbKpXV2pp3cBkUk~FWRyByKaK-j1Vy888UuRaDFIiqQpe9k5xNlH9oX9naBQwpEf6ikoGkv4vRkWx5cPg0mv0GFco64pineNrnirigVW0Wm5FKhlPgNGWCH~k4Q6W5BXYgOdTnBwheuqTPL7YSOMY2Kh8S1KS~~YEPQyimcNi3nUvVNXcv82O~aSW-or55Myz-8t2ZGUdgh5uK7omkn69pAC57vt6efftM~973g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq25.gif" /></span> matrix. Thus, the profile-matrix spectrum <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq26.gif?Expires=2147483647&amp;Signature=pvGcxLdLEMyI-1H6~Jes03QQGxZqgZkl3shRngQEjZBdO3H8gV6Emp0AprwU~EvPdNJpTu~szDXX6r6dxPoho2MdN9NEl1XbPa22q2xI0Yb7wiDHd2cHNSzJsQw8asRB45zBbTi4mj5oHfp9s-QF2caJ2k-B8fN1UIQatGm26tnR45t1J4Jlb1sEHFD7Lq1uDQnUKMD~UaEQZo6UNm4JMzKs7BCCxWw1oo4l2wny8AG5ZN28FP02Z6iuGvwAwk7n0U64d~v79GPTXogT2a1NgDj4NXZP1OrcjLajpxfti7Wjv-cSKtrHD29sddWPhA~ulSng5BU85N3~EoCzZFfZqg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq26.gif" />⁠</span>, defined at each test-period, is introduced for string Str. The profile-matrix spectrum characterizes the statistical structure of the realizations of random string Str. If the statistical structures of string Str and of analysed textual string str are indistinguishable (at a corresponding level of significance), then it can be considered that the string str is a realization of random string Str. Further methods for verifying this will be proposed on the basis of the latent profile periodicity model.</p>                            <h3 scrollto-destination=6726583 id="6726583" class="section-title" data-legacy-id=s2b>2.2. A stochastic model of the latent profile periodicity</h3>
<p>Occurrence of the latent profile periodicity in the analysed textual string manifests in the statistical string structure observed in the sample profile-matrix spectrum. In fact, if the analysed string is sufficiently long, this sample spectrum takes the form of the profile-matrix spectrum of periodic random string Str consisting of independent random characters. In this case, random string Str is given by <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq27.gif?Expires=2147483647&amp;Signature=YOjLtHuLIw9xgCsaUBlRoYg-X~tdWfb1~JGfcuV5BOVKLGvgqU9cntMq1YPqqFaB2yElHLDOsXd-7VD81VkoHDINwMBLQmuHgl81UZODfHoe1VPxJW7Cx8NPRZFZIMCRf20V-BedgjiV5w7IpcL9leRFM-4q3MRMDXeIh~M3eLdaofXj6GSKLWdZP7hlS7s25qU3VjjFrJR1zmBziGENSG2R9mPEnzmF7z0rID9ZD7YOMKbqZbBDce~kWc2Pa38odogcJUScQDXd8lEKTR1fcvlH-uf-KlNi1mYs5LPYJBJKDMaTXkHOmiJOc75rqXqDsKCDE4BIkOa0-FknrHeChg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq27.gif" />⁠</span>, where <em>L</em> is the period of string Str, <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq28.gif?Expires=2147483647&amp;Signature=0ZW-FA0Vh95ZmkYQWZzCIfL024dfV9oXua7ndJrX-Qpa1eMhp0O7PunBmtOmyO7bdveKR4OF1u9Bw-ZKvcQvrhhduhSCMJFfEnL4TNDnGqNR4EH5JTDvDgoGIHQm-7arrXFJJKRFlRggeNUPn05jNUZRchn9M8RfHAbn4Q18cINimo3vAJOf0J3ELHjeqbL2CiAayz4tc-dvDlO7TmP7y7UPeL3wpU1Gdp7iowWJ33D5UdgomqHdLa5mYH5wXEfby8FTwBMsfxYXXqIQcgzjH~jb~tdsp3osK7mhf1Qk5qbn-ppdZ6Qe-av~NmlD3y0A9pPtR4YeVK-757Mkip3Y2g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq28.gif" />⁠</span>, <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq29.gif?Expires=2147483647&amp;Signature=CtyCB1UN-NO6jefIFo6zNZgYTDRdwcWukuQ1V8vEjYKu8-6YmyKdBd3N4-2hnUOiBIviDtC8wgGb3rMxvHn7xDsJ52DdPdUmXAJ5m5hPfe7AOnaTaSsGQ5EKYyTVwVJSnZYnE~ERwxYIai7vDN095jzwTh2UGSSL6SZwgv-qF~Ks9F4e8fi1aBLF575zoNvmgt~kJ7y1ELLhfN8UPNClO9FVnASggiOdtSixO2XMWlzEDYRtwhdsrFeaDL-q-9PT3ZZWwc584XyqOEExuZgKTEGqrIel1CZFacVdVHaktMyTOCZuoDwKjgwdLl~iKokQdKlp7gzc0QLy2dNmRppOUg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq29.gif" /></span> and <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq30.gif?Expires=2147483647&amp;Signature=ihA2cUpd9Rd41VItFqPtSWS2m9f8BvKtcJV-P8moMZNFrKxD-h4injaiqGs1Bb6hvTisRlq-QHwfxmkzUvFXVaJNJVGpucGgduSGjivmHMnRpywHE6B-SJ2oIV6AfV1SZzpoLLGbKLCpZS6XJbHDgALwdEyCx65bY18YYnRsE~v2iHqek1xdS33a3v8Zp1JkT-TLZRBXHxXhs7Uv68Z22MxAURnO~DRj35FXtiC63wd3O7-ZL71gJxX0--PvkBPiIVYlpdaF3XrfKZXegTvD7H7LogG1wUZ5aOhnfXpQ4IYGNbvtLx0bNaCe~WGBtd-7ro9rvXtSu90rqiA3rTJyBg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq30.gif" />⁠</span>. Such a string Str is called the <em>L</em>-profile string with a random periodicity pattern <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq31.gif?Expires=2147483647&amp;Signature=U54E7nJInDt24xf7SDJTdK0fh54MB83CQ0OD2AfNptaFZl9pL07YKD0ufYbh5A6xg~BeT8-CrGpaZsGduh46RM4CQ7cYPRrKxIvVV3hiYsZ4niwqCucTB6TxAKkZpST~0CUFFTqvZWD08HIFRDBz0Zg93rcDpviiY5f6F817Ck1q7lLAadEbpePUOTHvS6a8btsS3WvOd2mhkNpsn4TAA4O3P8GP0ONgiKf22ZqE7llDslkyCBt82eUkI2WF1P4hZZqRbu44IhkQQXAP3izXpswXNchi42nwXU83uTa8D4bI55i14Lfq9aqryM~OwpLENPqu5ky0hiOf8c~uOUZgUw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq31.gif" />⁠</span>. Moreover, in this case, the designation <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq32.gif?Expires=2147483647&amp;Signature=kiQtx3uiBOW3Mqe3kgKf01KX4vtGCsi-jFdhIMsXJHNe7wBxlnx3z8z0RdlSIxF4Cxi4A1O6fGpdXAuOkcWDH7rlU5N9H2zC6vAOsTzIKNMjSH3b5cwEFG4LfcxNpX-8ndlzCNg4yWH6XKF7o~vvdBfm-EgM9WljEDHWfpybskQ7CGTrcfLYekYWkng2Ku6Tk~avhSO1vb396FectF6ULhxHunWDR3NlTYco982Zut5mW2qSciyCMUdY18-7z4m1T050v~~EE-~W7vWVyERkDhLRBUp4IRDXPMs0yf9SnKrNaILlHHgOPjNgeXelgPowYupiPIuzS0xhxkCS8umMiQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq32.gif" /></span> is used for the string Str.</p><p>Matrix <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq33.gif?Expires=2147483647&amp;Signature=BEVxJV6BtdwPkHPFsjt~NIEghanjc5zfDuFtwEzObFBJEXtYhkHRvHIjjXX~T3mHFXaWBFqqzYLq-SKju7biQyEdLwc1nKzQ7BJgFSMTRqcDao9zu5HtiKqYu3kjI-xbOyKyo1CBw39qm7tTH62vfwb~TCJW~mS2h~cQ1Pia3wgvA3drgMQJNVM1cTDt6GMmJ6SJUenb1j~JhXlp59UCy7DrW57E~ihqn~JZF4E00bX5Pvsreby6AzmoKt7uZhC0fxqsPuBp2RucH0fW8SLQOwwgdl8emvqyXPC0fLQcjq~bGvqgsGmP8XNTZz8SGY~YQO4jh1jDBk3iFjFpB90BNw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq33.gif" /></span> is called the main profile matrix of the string <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq34.gif?Expires=2147483647&amp;Signature=rFrR7l1Q0LxFqLq0WKd7QVcZyqzyz51J4J2GW50BGVXQ0PNYU51i8MCo4FY9ISz1-7qbpj9-VEGDHZbsNDVtS7rcC9r~btx1PaI67y3fokmyZQzorWrZKXzZE7leQUmlWSM9V5tpbmNgYSPDy3Gg-9zmx1A4aVG3IjCejN4mLLBcoLUjXxPZOPWj0HsF9Igcg6jnIwJTVVlYwq7ga7cafXifjoNihJYFrobnq~4v6Uxu0NHuC4QFbKv6wb2WqEXJEYKrR89jsc1eJBWYSvwRWxV5qz~R06KMCIPp4BAfgFeO0LZRxYUX0bvK3WUihDy0i09mq4YrMjziVRq6rzJD4w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq34.gif" /></span> because matrix <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq35.gif?Expires=2147483647&amp;Signature=qjkHuJ~3uSwy~Ikl4U4VzJw0eZ8V9zz~CI~cWGkH7c~Zjyj2wEOSkVv~ELUd0NmxS2Nk6YVn4OwSLEX2q-l3NkgQgProuxAlE2-cQISG~HdjyFPlVxjLk29EaBTpVxxNLxSdWfBhH36dQHWzNHhW0zuQvXPs9zfm8Cw1YTwbDv9CGtPJv6F~uh9TtsgHU1BFbKPX8z0j~Bzg77bUt-iYsn1U~k59Vk-gS5M5NvkSx-kEmeszbTfq2vmRxPovbvw9vlO9Lm-OnxxDnJ5XuKYcv23IhN5yNQQxn1xiEwNZvGaCKOIXZCdCv6dYnAczNdt7Sf4A0OYspi2iPC6JcF9Tcg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq35.gif" /></span> initiates the entire profile-matrix spectrum of this string. Profile string <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq36.gif?Expires=2147483647&amp;Signature=dvZnW2~ycVgAeJYRTWkJe0Z1o0X~xf0F3JJvwrnQwTuDkiRZNbhDw1mu9GaIpNhrpWGec5yInVEsMIuXT7AEKWMKZjn2vFds8D-5GhLyMyOTh9ETHTQj8rV4exm2FjxzCKwa2BUa3e3KYOUGDNpnRgLEU-nlXUxxcYa5FjAwMTk8C5bHiWmlroDC2JLDTL0T1rsC1k-5elMbHpePE~jIYdY9wMCFtoG7UI7WEHBD-DMyyoYfTmxWq5KFU3KNUn-i817N-GEZ920NICm5s8O5j8YJbLpUWonxwx~jC5ZBlNsbYv4u0OkQ7Rt7AB4MzPRj21j8Bnp2Kq0t5d3462bp3w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq36.gif" /></span> is a perfect tandem repeat with a random periodicity pattern <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq37.gif?Expires=2147483647&amp;Signature=sDQcPtwrXh7PGdo98RnWsXRoRacBoj~jvn6n7vLdwXLRlyGuCcJvCCkm787Z6VpzlZVzxXcHcKZuc5m-wKBjLN9~7DK7fwbCeEd8x-xdHAQPz7iY7tW0kU-AY48LRCsYWQHKQfV4FlgfnIS9sCoVfLZaWimKZa30R9h2Bzyi87YkcV~xeye8Fi9hM14wqmBX~dvH1VlrnQ~bVzmvD7PrrpoLVX7yFyY7KTZe5Ps-f5aL1S1E1swZbNwVvvefXxEngpxeyqUDIQOu98D2XXBFqfdWI0NVDp9wihns7rGSwovmbzGdRTos3rryo6LVwzhktor5EuFuexwx3or-RAfpVA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq37.gif" />⁠</span>. The profile-matrix spectrum of the string <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq38.gif?Expires=2147483647&amp;Signature=sXTx7ltLXT9HiltebdEnyBoanR1bCUbDELg1rGMOH~u0J2Ri3MSFC5XpG0d5yC80qgNJwSfk9H7xjXaQPW~~sOB-Z7HYX7Xe3prZqRWJeKrtthg2Y-HW~zB2EcWMmf7VUX1j1FP9hzhKZG1EVba7My8N-7HO6fiVULpzSmKOGyS2-le~oqF9IUR7vxbUHgkZXo2AMgYS19xfkIrefwnQtv1KHHLRvH105A1K5hMfSgXVxPlWFx6-VumZX2gst2aSY6ZKioTtu6AvhqUou8FXTZQrps0sQ7ukfMv-cKG5AQ4GOEOtgEBRSWTLG~8WGocAxeocATjvUBb~fTMQmEJDqQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq38.gif" /></span> can be considered as a stochastic model of heterogeneity manifestation in textual strings that are realizations of the string <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq39.gif?Expires=2147483647&amp;Signature=mrBsiDaAAirnXboaCheEvnoytDBg~Wv3NLAgTR2LyJ59CjPyJT2xcXj9ivJujtuFWZ3N5aC9MqtJ445N1X--eiY4KVBKsqYB1h5LW8Wja8Z8GL3l6SSqOvi5wGQZZX-HUy2J3O-pircaLcIZxmKdw4NhJrZPdolD5mKKziM-Uxv9d~mvqN0b014AuJUHrv-oL~4PFnn9xRbZr3u5k~xHT4tXCt2drY3AeQEI4YiZj-Mul4YEqtLlupd3JBGPoUnVAVwtTUJBbsP5NPPV9KZA481AbPiFDh-A~-rif1Rdjhnu8YGlvfWDlDoxd51dbpE7WxFElGGp99YM47nAeHWvIQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq39.gif" />⁠</span>.</p>                            <h3 scrollto-destination=6726586 id="6726586" class="section-title" data-legacy-id=s2c>2.3. Size estimation for pattern of latent profile periodicity in a textual string</h3>
<p>To estimate the pattern size of the latent profile periodicity, a characteristic spectrum is established for a textual string. Characteristic spectra of the three approximate tandem repeats from the database TRDB (<a class="link link-uri" href="http://tandem.bu.edu/cgi-bin/trdb/">http://tandem.bu.edu/cgi-bin/trdb/</a>) are shown in Fig. <a reveal-id="DSR023F1" data-open="DSR023F1" class="link link-reveal link-table xref-fig">1</a>a–c. In each of these spectra, the first clear maximum arises at the test-period that is a period of approximate tandem repeat. A similar observation is made for the characteristic spectra of textual strings with latent profile periodicity (Fig. <a reveal-id="DSR023F2" data-open="DSR023F2" class="link link-reveal link-table xref-fig">2</a>a–c). Therefore, the first test-period at which the maximum value of the characteristic spectrum for the analysed textual string is clear is used to estimate the pattern size for the latent profile periodicity, or profility. </p>                        <a id="6726588" scrollto-destination="6726588"></a>
<div id="DSR023F1" data-id="DSR023F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02301.jpeg?Expires=2147483647&Signature=nbf3pf5y1jVGUfFQac20304rYcErhH~Kd-cqJH6wWRt8KgDGLcl5yQAgCS2K0mz2~yCgRj25q0WiOBVEZFgHwLwnOQH~vY9gp1e2wSgi4-2SC9VwJzqq3WsRlYlrxIDy27EwAjCADZxoQ7zF~7Qp00Ou3ZSAh4RVMqpqs8fphaAcTHCt7oUDPW6wUZekk2WYjXN95fHhptjWnSfbpBbBRKGd0xiRCm662ebfgEVJvGY59clGhxRRWfl9LaGvt8ZV26ZAAjU5SD7EfPDmLbNpiTmMLhL~GNUXZcOpr0JgM7hLfUPk0VUOgYh887RYAJoI2Qj4DIy4uSdhS61uMFCq9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characteristic spectra of approximate tandem repeats from the TRDB database for (a) C. elegance chromosome IV (688 744–698 236 bp, period = 102 bp, %mismatches = 3, %indels = 0, copies = 93.1), (b) C. elegance chromosome V (1 809 784–1 810 492 bp, period = 17 bp, %mismatches = 12, %indels = 0, copies = 41.8), (c) M. musculus chromosome I (26 399 024– 26 399 410 bp, period = 12 bp, %mismatches = 17, %indels = 0, copies = 32.5). (d) Fourier spectrum for the tandem repeat (c) of M. musculus. The maximal peak is reached at frequency 0.25 and corresponds to the period of 4 bp." path-from-xml="dsr02301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726588/dsr02301.jpeg" path-from-xml="dsr02301.jpeg" target="_blank">View large</a><a section="6726588" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02301.jpeg?Expires=2147483647&Signature=2UUEIKw4TOndhSCQjJt29Wf247BKa4MjxNYQoE3YJhRA2ThyXi1b7OnWo3mwdEogfZG2EutVvnkmwKik4zKrY2Uip498pT8~IFk2mo7a4YzXSNVdjbRVVkgu657QVX628v4XFaz9w37MeuanJKXq5LGLeXOdfLK1W2X43ycu0~od7Uzjljoy0JY43SNVF93ORn9vEb918CVfCqGw4AS1C1N9~UkZza~mnIk3cCmLJ0Ni4EwZP5jkB~hXPzdF~UDrTZqV5DvwHMF2UidqqGafmNGrlbj8gV2~9~Sk5~m9nNiWdHMKjc5HSptueujSH-CTTAJmM8NU-IJD9~OZ-jgBww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726588&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characteristic spectra of approximate tandem repeats from the TRDB database for (a) <em>C. elegance</em> chromosome IV (688 744–698 236 bp, period = 102 bp, %mismatches = 3, %indels = 0, copies = 93.1), (b) <em>C. elegance</em> chromosome V (1 809 784–1 810 492 bp, period = 17 bp, %mismatches = 12, %indels = 0, copies = 41.8), (c) <em>M. musculus</em> chromosome I (26 399 024– 26 399 410 bp, period = 12 bp, %mismatches = 17, %indels = 0, copies = 32.5). (d) Fourier spectrum for the tandem repeat (c) of <em>M. musculus</em>. The maximal peak is reached at frequency 0.25 and corresponds to the period of 4 bp.</p></div></div><div content-id="DSR023F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02301.jpeg?Expires=2147483647&Signature=nbf3pf5y1jVGUfFQac20304rYcErhH~Kd-cqJH6wWRt8KgDGLcl5yQAgCS2K0mz2~yCgRj25q0WiOBVEZFgHwLwnOQH~vY9gp1e2wSgi4-2SC9VwJzqq3WsRlYlrxIDy27EwAjCADZxoQ7zF~7Qp00Ou3ZSAh4RVMqpqs8fphaAcTHCt7oUDPW6wUZekk2WYjXN95fHhptjWnSfbpBbBRKGd0xiRCm662ebfgEVJvGY59clGhxRRWfl9LaGvt8ZV26ZAAjU5SD7EfPDmLbNpiTmMLhL~GNUXZcOpr0JgM7hLfUPk0VUOgYh887RYAJoI2Qj4DIy4uSdhS61uMFCq9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characteristic spectra of approximate tandem repeats from the TRDB database for (a) C. elegance chromosome IV (688 744–698 236 bp, period = 102 bp, %mismatches = 3, %indels = 0, copies = 93.1), (b) C. elegance chromosome V (1 809 784–1 810 492 bp, period = 17 bp, %mismatches = 12, %indels = 0, copies = 41.8), (c) M. musculus chromosome I (26 399 024– 26 399 410 bp, period = 12 bp, %mismatches = 17, %indels = 0, copies = 32.5). (d) Fourier spectrum for the tandem repeat (c) of M. musculus. The maximal peak is reached at frequency 0.25 and corresponds to the period of 4 bp." path-from-xml="dsr02301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726588/dsr02301.jpeg" path-from-xml="dsr02301.jpeg" target="_blank">View large</a><a section="6726588" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02301.jpeg?Expires=2147483647&Signature=2UUEIKw4TOndhSCQjJt29Wf247BKa4MjxNYQoE3YJhRA2ThyXi1b7OnWo3mwdEogfZG2EutVvnkmwKik4zKrY2Uip498pT8~IFk2mo7a4YzXSNVdjbRVVkgu657QVX628v4XFaz9w37MeuanJKXq5LGLeXOdfLK1W2X43ycu0~od7Uzjljoy0JY43SNVF93ORn9vEb918CVfCqGw4AS1C1N9~UkZza~mnIk3cCmLJ0Ni4EwZP5jkB~hXPzdF~UDrTZqV5DvwHMF2UidqqGafmNGrlbj8gV2~9~Sk5~m9nNiWdHMKjc5HSptueujSH-CTTAJmM8NU-IJD9~OZ-jgBww__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726588&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characteristic spectra of approximate tandem repeats from the TRDB database for (a) <em>C. elegance</em> chromosome IV (688 744–698 236 bp, period = 102 bp, %mismatches = 3, %indels = 0, copies = 93.1), (b) <em>C. elegance</em> chromosome V (1 809 784–1 810 492 bp, period = 17 bp, %mismatches = 12, %indels = 0, copies = 41.8), (c) <em>M. musculus</em> chromosome I (26 399 024– 26 399 410 bp, period = 12 bp, %mismatches = 17, %indels = 0, copies = 32.5). (d) Fourier spectrum for the tandem repeat (c) of <em>M. musculus</em>. The maximal peak is reached at frequency 0.25 and corresponds to the period of 4 bp.</p></div></div>                        <a id="6726589" scrollto-destination="6726589"></a>
<div id="DSR023F2" data-id="DSR023F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02302.jpeg?Expires=2147483647&Signature=WJjUxj8AEu36-1eonssr7VyX~A1LiIox4a7re6O92kGNzITkxKjnkclaDQooB3Erpd2lweEHwaidcsA1kcdhpKsfnN-fgJvaJ6Wgc8IE6M~hXszh54Y~BBxYgdoR5r0Y4-ZihrxUvhvDPtZ~w3zhx7t0n47RLzLSdCsKUNRcn1oMSZzaFw7BqjbTJTK7v0I0M4FgFTeVLA8lcjJBOx1HGjC140PVAIxYg-FXyxahmRJe~IMu6zKnkDjao~VUvoh5NJARMTfVPSdgJFwmYFlaB2FLQSh5nhoYdzKRor-ZSwHJlYSeS4DFzq2veC-5ogo~HqFb4V3BcIX9p3kvILry8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characteristic spectra of mRNA coding regions of the PF01442 apolipoprotein family from the Pfam database for (a) M. musculus Apo E (GenBank M12414, region: 1–936 bp), (b) S. aurata Apo A-I (GenBank AF013120, region: 34–816 bp), (c) G. gallus Apo A-IV (GenBank Y16534, region: 37–1137 bp). (d) Fourier spectrum for the coding region of the apolipoprotein mRNA (c) of G. gallus. The maximal peak is reached at frequency 0.33 and corresponds to 3-regularity." path-from-xml="dsr02302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726589/dsr02302.jpeg" path-from-xml="dsr02302.jpeg" target="_blank">View large</a><a section="6726589" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02302.jpeg?Expires=2147483647&Signature=SQUDwWzAOtOOAdih~jxpduNOdBTHIbCp~t~2scJmHec1WUEHFnHq0YNkjRu2YbmVp-K-hIma3Z-ygZKPmAsycgtuu~99qGH0tSNqwVgzhlTWYzNmURcwmSytG3LKpki84~ahN3TCYsSv57iEI4kuVOD5EofC5iPRr1OtUao086sTxfxm7uPVoaAlRymGExHAAES1kJri5Ft8PmStf2IF0QkLDNvGkOvyupmxzAQXgpY16PhmhUyn6rywELSMpaYw5XK9XtblF0hlAHLZMef-WPKXntnSMDUfNIi94h9xpFFZeCibU08Qr69A8MTK8AgiWANpfoPuRDYYi68ylc9-9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726589&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characteristic spectra of mRNA coding regions of the PF01442 apolipoprotein family from the Pfam database for (a) <em>M. musculus</em> Apo E (GenBank M12414, region: 1–936 bp), (b) <em>S. aurata</em> Apo A-I (GenBank AF013120, region: 34–816 bp), (c) <em>G. gallus</em> Apo A-IV (GenBank Y16534, region: 37–1137 bp). (d) Fourier spectrum for the coding region of the apolipoprotein mRNA (c) of <em>G. gallus</em>. The maximal peak is reached at frequency 0.33 and corresponds to 3-regularity.</p></div></div><div content-id="DSR023F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02302.jpeg?Expires=2147483647&Signature=WJjUxj8AEu36-1eonssr7VyX~A1LiIox4a7re6O92kGNzITkxKjnkclaDQooB3Erpd2lweEHwaidcsA1kcdhpKsfnN-fgJvaJ6Wgc8IE6M~hXszh54Y~BBxYgdoR5r0Y4-ZihrxUvhvDPtZ~w3zhx7t0n47RLzLSdCsKUNRcn1oMSZzaFw7BqjbTJTK7v0I0M4FgFTeVLA8lcjJBOx1HGjC140PVAIxYg-FXyxahmRJe~IMu6zKnkDjao~VUvoh5NJARMTfVPSdgJFwmYFlaB2FLQSh5nhoYdzKRor-ZSwHJlYSeS4DFzq2veC-5ogo~HqFb4V3BcIX9p3kvILry8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characteristic spectra of mRNA coding regions of the PF01442 apolipoprotein family from the Pfam database for (a) M. musculus Apo E (GenBank M12414, region: 1–936 bp), (b) S. aurata Apo A-I (GenBank AF013120, region: 34–816 bp), (c) G. gallus Apo A-IV (GenBank Y16534, region: 37–1137 bp). (d) Fourier spectrum for the coding region of the apolipoprotein mRNA (c) of G. gallus. The maximal peak is reached at frequency 0.33 and corresponds to 3-regularity." path-from-xml="dsr02302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726589/dsr02302.jpeg" path-from-xml="dsr02302.jpeg" target="_blank">View large</a><a section="6726589" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02302.jpeg?Expires=2147483647&Signature=SQUDwWzAOtOOAdih~jxpduNOdBTHIbCp~t~2scJmHec1WUEHFnHq0YNkjRu2YbmVp-K-hIma3Z-ygZKPmAsycgtuu~99qGH0tSNqwVgzhlTWYzNmURcwmSytG3LKpki84~ahN3TCYsSv57iEI4kuVOD5EofC5iPRr1OtUao086sTxfxm7uPVoaAlRymGExHAAES1kJri5Ft8PmStf2IF0QkLDNvGkOvyupmxzAQXgpY16PhmhUyn6rywELSMpaYw5XK9XtblF0hlAHLZMef-WPKXntnSMDUfNIi94h9xpFFZeCibU08Qr69A8MTK8AgiWANpfoPuRDYYi68ylc9-9A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726589&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characteristic spectra of mRNA coding regions of the PF01442 apolipoprotein family from the Pfam database for (a) <em>M. musculus</em> Apo E (GenBank M12414, region: 1–936 bp), (b) <em>S. aurata</em> Apo A-I (GenBank AF013120, region: 34–816 bp), (c) <em>G. gallus</em> Apo A-IV (GenBank Y16534, region: 37–1137 bp). (d) Fourier spectrum for the coding region of the apolipoprotein mRNA (c) of <em>G. gallus</em>. The maximal peak is reached at frequency 0.33 and corresponds to 3-regularity.</p></div></div><div class="block-child-p">The characteristic spectrum for the analysed textual string str of length <em>n</em> in alphabet <em>A</em> is determined as follows. For every test-period <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq40.gif?Expires=2147483647&amp;Signature=GOrqVtsF7SYvj8FInbO~c~7NsPc8zGDrXqDf8i-EYodAbxPU3qGa88oCFEJ8D~eoxfB60Gw-sQifunokxrhzhbniDwHLPZnWYWYlLk1uYEpSwN2g6lspGPXXcmGM2PZI2jp1CIk~ikP7xBD1sizHosD~wQl4cv26PuHFSer8OzbuTlbsAn2~Me~uwVfIc86V51nuvGa3~6Isa~mLrNGnODXMQDSaF6rwITIDvB5TJOKA6rmeyPhcpzcE01sObHdP~LwdGtwODPS9KSUf1Ae1lvmpDU-cvUnutoma0qzy9hjN8l7KeIVLKQOm8aTWlU7TN8kU0G~lgQ4ELDaqhTap-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq40.gif" /></span> of this string, the profile string <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq41.gif?Expires=2147483647&amp;Signature=EmLDcCksNZMnqFxy9J4myTy4EosQXYRxU2nYChszSCGPNxpCrMWpm39rvQ-hFW5vAIi-hsBlAGrIydna~YsxdHVhT4olGM5wYm4Z4V9S3DMtYGkv8wbpu~J1mFdoTlAbnahyLM0oPAGwIYZbUdXWrt6hL~r5FLtcjMvrCynqoDRopxFAJXkdl2haceCmPKDdie4rmnPJBQyCUm2i9vLnldp9S3LkDzvsQbTRTm~AZAfcNlj7SrFc68DuLA~77O7GK5uACths7gGLHISPHq8X2ki5ZOxdUqhtFJiKXSJqGbVjlMbTHx~uVh30RZnCbq5xeqRog9587Gtt3BtyMY62Cw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq41.gif" /></span> is created, and the Pearson statistics<sup><a reveal-id="DSR023C14" data-open="DSR023C14" class="link link-ref link-reveal xref-bibr">14</a></sup> is introduced: <a id="DSR023M1" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSR023M1"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr023eq1.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023eq1.gif?Expires=2147483647&amp;Signature=an3akYk6mAgQI47beVBqcU1HWCqEOfxo-yiyCcwxqcV4wl8mo1-2bAGBs1o7xvHTUTucXCunadyTqQor86SCW25NTDwAvzG75a9g8yvF6eILD8eDHPpoE0IYQ273u5QF8RWKJgLUqh6UhKcYt~BZjJdu~rCzAnL4AQDgCkjiKzXMaJI08yBZzXnyy4rc-aGmEJ-2VtasHYxCEfSO8txUmloOr~p8V3ea65un4dxu89uRPRwqwwEfZsu33ZmAA5np8dhEJu4YMJ3wJpPUkUFIApnslIwSfpIxf4Eguyn-ajJ~GVIxTNzeSoYApcMKDqmfNs4LqyrhzKJq52chaDPvsA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr023eq1.gif" /></a></div></div><div class="label">(1)</div></div> where <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq42.gif?Expires=2147483647&amp;Signature=2In6-Mp8XgkS4F3q~-2EnIz64ScRmUHb~fYPyM7NWE8TdQyYBFvX5cO9IyOvN58c60xZkJOsB98MXwGWuK4jnFkv0yBy3KcXjW4ZOIBYrgHb5HUOsoDdSJz~RRXSaapltfbXnWp-ozTxtzmuHPiIhyKLwyaQ8IhBzBRuP0-p-49TMHOC~~FEy49rbUvZREnQaG1VZMotYdc9~ke~Xzjkq~NJgHDPCFLzsqix5OapsM7GapibpDS1Mwo-FnsB3mjn1OzvysY7Z5CarlWKdiIbP6izCY5o3xS9LmKMYCl7wih0U-VkJ1LzLqJ4qV0J1tIhLnUeTZcOVVMuFDyTW8gXVw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq42.gif" /></span> and <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq43.gif?Expires=2147483647&amp;Signature=G856n0WuknrazbpeFmK6bEl6q4PJCqDBdvJnoJk9U7UoP-PLl7SW~YHuGsrtpcMM5U8p72BeFKl3nzRp-nGcGnh~0pROFM9tCzJhgP2Q8jNPKLv0fp6wj-LV19zZ-FkyfzRNf6sG2Y9p-Qx8PiXRQSpebMGigXzG2SIJ1Q~Uv6F9fmR8mE5Ybplp4ZFXuxhwxE36sExY~XRTV-5yBHq0XRID2rflwJ7L~P~rGRBYn5LULBVzW3~Hlo~f9c9gySCkpL-t3J6Tzd6vmJQYrPqaNEaFEF7Jlkws1DbBEHDEa2W9ImCdFlDSKuHoN7S5liOTofwgNTd~MYLq6CODvnYcNA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq43.gif" />⁠</span>, and <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq44.gif?Expires=2147483647&amp;Signature=BpSKv6g~mVm4wLaQV7CSzFCUoJXwX1Rir00tsSuzg0Mi19qKQDzCiolxg7jaCVVDq3ovjdtW8gl6KHayZ5Qs8sZCx9-BIkWqy2Es46Jxocmt-fJ38akSEDCjA3gSU-NBy8JoZ26IdYZmuEKFwQQhnw-F8kyuUwEVboJpd2rqEqoUJeFOjLlbWhmuOZQr~-r18EsEQHK9foJR~ByVadgvrxE~JGt3yV-7lka3rSfwi0yhEXmjZ4Drqz4WEFwoOhy2cFrZ5kiW~JYLv77SlfP2SyTyyjp6ZQggmwBcVBOPupElZ7x8ulRxKVk17PIEjS53DRguB8PcoJINVcsF7cS0wA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq44.gif" /></span> is a χ2 distribution with N degrees of freedom. When Λ = 1, the value of the characteristic spectrum <em>H</em>(<em>λ</em>) at the test-period <em>λ</em> is calculated by <a id="DSR023M2" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSR023M2"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr023eq2.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023eq2.gif?Expires=2147483647&amp;Signature=ivJJD9QP9Hm08cAAm2aEamKYI1qaBB8mctnjJ4x4LbPxkeVnx8s4NFKkqGWdNqrV4U0iUJBCnKOSGt26JNsATwoj4PQ2qhye4BRMVOESHPLibwp6hcIyHBftJVmEh-ZlRJUEpquAZ2ASbtSGFinK9XIKcir98t2VGYoJBOWrRWMP0JfLTjHhqWharr41iSw~P6QKtZCQTy9Hwj8wIBDWkCyJNQRI7MUuBL7dagPekRLsgOK3YO0YSniMewfi56RTzUNYxrWSmOjWmePK9YxZcb~NNTw8N48g5PH7Z3bcHT2CxNw6fkcc5WcFhQywX05prOVpb14gnszdKtgzTBZEkA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr023eq2.gif" /></a></div></div><div class="label">(2)</div></div> where <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq45.gif?Expires=2147483647&amp;Signature=hz8b88GDUHdMX4HKH1UTRRS5XvMYTI3zvsm0-CbJONI72V-ppYFfLwQPzzB4N8dkF5y17ggDEFMtwATSp4lcOmolMpEL4V4I6QcICPBZ98thR-2AkaPApxT4eCOUzkrJ2uryxffEPDr0-S1B-64tAfmDiEzamsGtHYRwDrcfkqEdOUz8Pf7mNg3gyEyJaZe3v1JiQwctzoCZSt2kNXewqfT9w9d4MyRDUZyEwNxMrruGDXQJ5to0B0k8MyFKVq1uDCd4fUH4jcNXv2lW1wQ3PBfmmxJ3DbdfgW9XdRABnl2rERiDvFWgcM~~xsWJwKFF3VjMMu~jtWK8v55LAEEpfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq45.gif" /></span> is the mathematical expectation of <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq46.gif?Expires=2147483647&amp;Signature=KR8tId3vwwUU44cMY1tSdnu0t2CMsZaAenpVWWs5s9aAQ7nG2jXjr15tW~SmH-wWT0W08G57wT1i1IbSKPdnoJD0ZmH75y3FQomcTGmQrvAQ-CJvS6FJ70VH0VyM8YWL8s9NFiC7RKgg-Y4gysYNFndMJJRO6El94A8NE73LS9SuEp6VSq5cvRABTxATFoXoEBneWyugTvPWxGBfzddN~JNzuxSGXHCkXkP8LY9oJMgy6vsw06tYLo5TnEEJkfe39rEqIJ~iyWoVujzdDt0v5tNPX1NqAhZ8bXVUWvs9fC7his7524Ez~MCvGHNuWtQe7Kw1BwJ-v6IRYyQR4C~Z8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq46.gif" />⁠</span>.</div><p>As noted above, the first test-period <em>L</em> with a clear maximum value of spectrum <em>H</em> provides an estimate of the latent period of profility in string str (Fig. <a reveal-id="DSR023F2" data-open="DSR023F2" class="link link-reveal link-table xref-fig">2</a>a–c).</p>                            <h3 scrollto-destination=6726593 id="6726593" class="section-title" data-legacy-id=s2d>2.4. Estimation of pattern of latent profile periodicity</h3>
<p>Let <em>L</em> be the proposed estimation of pattern size for the latent profile periodicity of analysed textual string str of length <em>n</em> in alphabet <em>A</em> of <em>K</em> characters. Then, to estimate the pattern of the latent periodicity in this string, the periodicity pattern of profile string <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq47.gif?Expires=2147483647&amp;Signature=Uhh8yFE~5o7hE3q8QzpaGHYfYnaX-jGUO5hV2hIMuZWwFe9o7nwSCrlJ5HDWBd91gJW9xfEf65cHxjOdyN1Zjl80H3JZt3Em5vSgARoTrGOq9prDB8f48SvoWrKAu628iXXN5b4MJLwmiQmiGJ9-b-LV1-vLj6AtpnzltsLGKEisgdttz8ekashlqwBM1jeyJdYwyCKuVtEuLqE3gWQkDP8CT~iQ7bsBrww~qtUtUYj9IDqzl8gcklHNfoqNIxfNDkjkg~y7RTQTAgo44CVPaxwSBWU14-nFvMIgyvdGu17Q0I2X44skNAFTHJkJBzSeRioEkSMdiiMB5Fuu0STJlA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq47.gif" /></span> is proposed. Hence, <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq48.gif?Expires=2147483647&amp;Signature=OxaDFPg5yH21DCev-I-DHtLUoYBArepXzj-I01byvco62XwT3Q2d-ZdIUoG~Jt3FLe1cvnP992n9ntPxJLLDL1JjqI2AtFkKp7xlRc-nQkMi~8nGp76M0sV4A3K02U2QsGTUV7Fm7fwkFAWk8~B1osGMaSzSFNMBxxM-gMpNVkr8Qxji1XdpHefWqluJ9PAksCqYkGDrpX-Q99gMsh7YuNNcWz0ydy0MSzKPZetMJbivJqe7Q2iwZK8lnaIdsDjgLzuAV9y-iD5oC2klQoYo~gksb0X0j9hyqrw-Vd9QNiG10ViVoCp5kGQJkN~1Veq7wKSJPYP0PtFREwv1-AGw-Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq48.gif" /></span> is an estimation of the pattern of latent profile periodicity in an analysed string str. If such an estimation is valid then string str is statistically indistinguishable from the profile string Tdm<em><sub>L</sub></em>. In this case, string str can be considered a realization of the string Tdm<em><sub>L</sub></em>.</p><div class="block-child-p">To check the statistical indistinguishability of strings str and Tdm<em><sub>L</sub></em>, the <em>D<sub>L</sub></em> spectrum, called a spectrum of string str deviation from <em>L-</em>profility, is used. At test-period <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq49.gif?Expires=2147483647&amp;Signature=P2cH0BnDVXNKaHLKtiUulRYyu84g7DO833K5K5AWWMVMSvqTBzjDa1-wcsrcR8U59zawAMQw-PuCrNZDX5RrJ2YgFClfRvrLlRvHMw13lJF38Mb271k0TJi03-6vuJRB9PbYDY1Zvlgpuj9yI18qce~DPww9GknpjUlt~TJ~zYdv6I24lyFvUg5JVCPa7IkQH5SdEwn0mUIThQOmRWKdA0D8rzra7eFnqaJf09qyYrml8rIPhUBoVRXNrHiU4pJtWuCI-AOuauHlqqMC6M~-R2Avab3qNblyS62ggj07bP6Q2fwM1wbCIa07haZOgwB8ZZVoHXp0nTRZOIkke3iQPA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq49.gif" /></span> of string str the <em>D<sub>L</sub></em> spectrum has the value <a id="DSR023M3" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSR023M3"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr023eq3.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023eq3.gif?Expires=2147483647&amp;Signature=mvsVLv9m6qH9EEbPqCFzSbRuyuM~k8y8qe-n3e9KZqR3ijDNyBjEd-oQpIXm3byp58pTV0v8vwzuxOrej2-4r7EYK1skoXwfjnaOVg6dwbgAa6eDEu5~wt6HhAC2ARryHtWlMEfkf7ePO~U1bbnFpT18HVSIBkFp1AxifLNNwc5Gm7agD0fWEexqxAdH5IWAAySyz4pDhG9zSErhpQeKKke6coaNFLrtcciTEyGuDt64vwdbOl0rvvntLXzY2DW7muc09GNXIfGXdog60KzviT9xOnGWZIlElQpLXcjqyNAD9oMmx062SSJyt8dcDLLXK6r2LdkB1NavpOjlsptzoQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr023eq3.gif" /></a></div></div><div class="label">(3)</div></div> where <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq50.gif?Expires=2147483647&amp;Signature=Fx2-PxYmDf9TrtQ9EW2p-E9YbgO6cCKZrt35mKBSJrrjgIstY4Ny-b20KxEgAIImPJlJiCwKKatZaVp5gkJlTH8VjPBKhoDnaIcRp-TQBnPjg6wvJom2bECInThsqoQ1SqyuYqFur9CoF~H2LD58J~mrM6WUtQRgtOp5hlufB66CVd5ITNHGEnRs~tBR3tkXRGHE1tEIf079luUFMIDErTQUqcKld~-mTqqplmSO5yvp~KpBXG9UWUZSY~CNnSwncGGAF--MJh8j49BlR-IlPpSB6bPVJndlyVNBdAFdEMP3FemmDLEGFDMrTlY0XnKCAhTYypgXF1I3EljChYGFzQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq50.gif" /></span> statistics has been introduced in Equation (1), and <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq51.gif?Expires=2147483647&amp;Signature=rvho9Jjpct-pOeLccNPhWBaUr~UbPethFvtWjJJbQf6FDhAGGYU9SpGTnaddEa103BGQQ7RAcL60vXgt6DowshRCJkAciy0Yf-eSFP0XtFGVuMu7g1RDUvajlhMl2NT-~ad1LnazXXh4CZXtPhEjD9DfCIneKjdLyD~fyebTxjDXwIvqI905fLVKFgTHoB5bMskHwlY8ir4o6~Heny9P0WnBpd-wjzQn9PCAK2~Uhaf-t4cIB~Vmq8oYlMbffLvtK3uQbxFIOFTlh8wUmNtTdi~laWVAlPbvIP2Aaw5fGoQU-xjnvi0G0q57w6gWI-G9j6IqP6zSlDrjox~ZfJ3VDQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq51.gif" /></span> is the left-hand-side critical value of the <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq52.gif?Expires=2147483647&amp;Signature=oePfMURMR4hvaA0BJvn1-yT-3FKkJfavIVxPjt4mPHX3vw8unre39CCsXQW2ft01l5jdefEOodZaSWkpuTsniMarGXvobBHUjEAbxd3p6UZ59IPERZxw0osHVHnUvsbhhLFjTRFV7AlwYh2AHhjTNLiVIdFsqzFPNift~FurFn~bokB2iG~VeV-UmAPmE3O1L5MS6C6cWFKQdrCuYg6MG9pWMAQsleke-3ztTgozQ7lCnF29Gb5mYpb-5C-VBoTwUf5fsMyNfz1pN4Ha9Zdi6hbNGWdQcycrGVQ-Lm4~u3RRn9rRd0izOCyntcX1yAeRB5kEbnIboO8QE17hlM7m~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq52.gif" /></span> distribution at a significance value <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq53.gif?Expires=2147483647&amp;Signature=h4uF-J5ESiKEXV7~P~JDvUOJMy6tuPW4WCszMT9k0-0OM7EfghaNiniud-4Zls-Zh~UaXVJw8SAGcayWG~uIrtoBq-r7XHmdasXwBglrGV-Nf8dcowb0Xt~upFxYfmdig0EVOzemTY2ccMP6WFYXfDnYFYNdapCRQKMBTc-xFcNc0cO2qOFDQXFZYEusitOpFA2JF~CRFoPrc~Vf8wH0xjP-cERqo6yRDgKCdlKF1luSLOd8FY7yxt6jcJd3s-FCB~By-Z-0gTKY~ct7jmwQnzYnidwHQmUMD4umYD4T0hrTzcFrMX-yCBj17bZnhUyu28M9g~ihktHxWYW5BVFmXA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq53.gif" />⁠</span>. When <em>L</em>= 1, the <em>D</em><sub>1</sub> spectrum is called a spectrum of string str deviation from homogeneity. At test-period <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq54.gif?Expires=2147483647&amp;Signature=tPeml0GkoUeEzeCWTaRjac3mSif0oNMI8Az4ofdr4pRPDQre9D1H8ueyRmebLgUGRPFsBvC6qIUqB-4W2szLEXBalCHab9y5bSgOu8Jfr4GqSEWHqgpwBk83PTNBNwYKy5KoQ9b~Zu3a4wLOsqx4xg2s9H-1zWoB1zvhd1x9Y0dRHvehiRDK42QA26h8PwP8ON3EqZ46IGYu3MJooBHVKFyQkNpqSjJViycZak0hTViC60s~fqo~vqV3Z5JAfqPGJkhitqYC5N9YVnbW1t3Y5iVvtBqozRyEx~CtezAY-BbILFtqUBVzpoVCB2GG4Fey1qOTK6BFqbvZ~dq4v2~ysQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq54.gif" /></span> of string str, the <em>D</em><sub>1</sub> spectrum takes on the value <a id="DSR023M4" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSR023M4"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr023eq4.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023eq4.gif?Expires=2147483647&amp;Signature=BhGnuuY2I5M4RP1pvreoXuFecDabM24CYVbuJ7BIbXHq7TizaXDCwao-9Q3kqqB9mlxFsFWCNd0wonYNHzwArRiNhRZMHHWzH4Y-CL377KHOzX445pY4PRz~CMlBfV~uSoY2QdmiCpec3FlBDxAO0QovKYLuH83BK3djzN3Gz3xDcpV4d0mBmcfkNMwVMiGiToyoMmuGZxdHCsFhBK1~qElDUdBSXj-xN9GKgM4O8vi~7jGefdREp6ee4LgCUx~u~6Jb1dRGj89qjIsi-6UelN8aVrv2rLXnq4khvSwb89mNwx9077xF9cOZ2jXvfWsAsoI2EiooA45fPO1PYGCN0Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr023eq4.gif" /></a></div></div><div class="label">(4)</div></div> where Tdm<sub>1</sub> = Tdm<sub>1</sub>(П<sub>str</sub>(1), <em>n</em>).</div><p>The hypothesis on the statistical indistinguishability of the strings str and Tdm<em><sub>L</sub></em> = Tdm<em><sub>L</sub></em>(П<sub>str</sub>(<em>L</em>), <em>n</em>) is accepted if the condition <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq55.gif?Expires=2147483647&amp;Signature=SwVIqLXUuHAe5KhkWTzaqrRcBOOH4ybw~effp21H9CusyrUfnMGpNYbcDw1W0gLYb4TNDQg-Df26OV2CfdEMn2U~5K4nsMv7kVdsKKBzURAoZPRJsHsLkw01M95HDVZBqvKvm9WDU~94PIeHSun8-9btHTztP~KC~W46neAeTh7DQbdOqfc1qB1oCWVn3SmQXfGgdEmPDV7dVaoxWPKpLwiP0FzYuE-9zvZrZ03SoDP7WaBpX7FqK0zF-8UwbYISSbwYg5NNlYInkCOJhb7hPm38JJHezsvjcd5~soLo~pH~rhdlh93DCgOKXr1qReOawH--4I7kEYSJC5tsDTGxhQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq55.gif" /></span> is met, where <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq56.gif?Expires=2147483647&amp;Signature=JUEyDLD~IPlK2o9a78KRL7l4TQXEhd3o8e3gJq5qc5m6h9hYvp1WjboS5dfOWd-nC5N54ArEePesSeDNEyt9X4onUQ3hGEZ05vJiEsUFBtUIQX4l1XpJdCRHiCciebQncIvOcN0tcwK~n0m6BjVD1NKW89N~ca7A9XnG52YhLFtzbiDLdBoKGr4htQSTM8amJua9rk7M-TUqC0IGFYgchzM3aOwcdVTyBITemwHtYuozbTAeoJpjIA5X1~XoixUmxrNSCAqm-Ckn9pxxv94VdmTOksLDWcHojLmd9K~alMToogCoLaYgJuj-lmc9IKA-kk0Wvq~GJp4ScwAnCYnMaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq56.gif" /></span> and <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq57.gif?Expires=2147483647&amp;Signature=bo1FVQelijY6ZU~8qVXvEctohVAXRK-73bNFBhBAHW9hPSXbb5R-00Idxm-cBuDya8cAZk8UaLgLR6xw7hJz0PN7nyaZXXUOuuESeUxDiRSRJJDEYZ6cqvz5Y92VEjyHFrXSYyAGGunlCQ~FcQNHp-L~9nAl8xm4gpCcrDFoFd6dSkDDLcjfOE27aFiHpu07NZu-2Dv~h9qWf2KzKODU5QNGIwyaxmo-crUFaOFZKoZ-AGtoBdtrsRgm8M7NX1EFLXMjv30r~aWv67zE20STJS-8YQbOKCIc7l4b2tVpHh9t1DDKAf486Sewyx3PEAdQMBgIjRYa~E5VjSFEj1cWaQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq57.gif" /></span> is the number of test-periods at which the values of the spectrum <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq58.gif?Expires=2147483647&amp;Signature=sYI4SyptFGN4XHDNMP-YfVS3c7sao8~xTkO7B4SJaVMIrYxbwOisOlfOSVsljadkisF9WfdQ7LgeEA5wbWjYmJ272po-wrBFhnPqlEnLXOXIA4RwF5FsjwwR~FG6mFwSfZo~HjokZ5oe857934ohhMOgLZrmRMVN33~IdsHiIEwEjI~G735l7z8qRGSf3MryL3SK9uHjynUGiIOIyqP-rTXZTcIsfqsRpt54GrEU8emLnvXrtPpn1qeKm43-S8aV4dLeWLjPJHnRr1eU98EKM9tdUh4BhObY8aZyvX~Qedl8sKgItGbbDR0gqa2RiQPvBr7pqrNzrqcY-DqJn8odfg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq58.gif" />⁠</span>. For example, as can be seen in Fig. <a reveal-id="DSR023F3" data-open="DSR023F3" class="link link-reveal link-table xref-fig">3</a>, for the coding region of chicken <em>Gallus gallus</em> Apo A-IV mRNA, the hypothesis is accepted if <em>L</em>= 33, and it is rejected if <em>L</em>= 3. </p>                        <a id="6726606" scrollto-destination="6726606"></a>
<div id="DSR023F3" data-id="DSR023F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02303.jpeg?Expires=2147483647&Signature=kIqMkD9SifAOMJ6PWz8KrgAVuFd~gtRL-FhJs3m26W2DkxQeABYVeQIZBx1AiF7JKol2d6lm43~pNEGKNpNpcmOq-pazuHwtPYCOV9XXrdSCflfWjJfl6-wIk5lr-2OJ71y0B9othG-mMMCsE43carzOOxSvws7ewHS2yaBZUhKYm~6xq5W-x2xpn54hhK6vGj2-J--tYyw09YLrWMp-0eRF29hnNi32B4oWw6l9kAxAXiW89Yx6VxTz8nYBWwMiB4K2TqAXHxq1cN-zU3w9RQGQumfLhI-IFljkEommoWmTLH13r9ArxHg5jK6GGUHs1QFeeHvRtnNd8Q--vBndgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="For the coding region of G. gallus Apo A-IV mRNA (GenBank Y16534, region: 37–1137 bp), the spectra of deviation from the proposed latent profility [Equation (3)]. (a) Deviation from the proposed 33-profility. (b) Deviation from the proposed 3-profility." path-from-xml="dsr02303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726606/dsr02303.jpeg" path-from-xml="dsr02303.jpeg" target="_blank">View large</a><a section="6726606" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02303.jpeg?Expires=2147483647&Signature=gbTS1~5mZpJ~IramrDpaa5H7WOH-cpR4ZAh~R5eX3OpZloQaLutXAnTarHzCyInhU5b0FhsY6PPFvlywudtY1IUTLKo1bmLugOudvYqDs3bh2JAj6E4xlP5q-fBzHsoqKRmexqdOQu8PRxEfrRpWExfAUR-RXmlyHwgquz1c~4k8bqtahT5hbDjiWPq3wny8znajczHMtC1JaZexPTxNwResdLlIhh3rW~J14SM-9qfPO9KZOPIOHe5Ypse-uBQCcEXCHLf~W0gGbdqUh9eHQvFnD9HmHZZ8M4XaqWDIMatqs8fymP5xpCa7uv4XG6wmV2sNONrcaTYNPgAyv9WJjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726606&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>For the coding region of <em>G. gallus</em> Apo A-IV mRNA (GenBank Y16534, region: 37–1137 bp), the spectra of deviation from the proposed latent profility [Equation (3)]. (a) Deviation from the proposed 33-profility. (b) Deviation from the proposed 3-profility.</p></div></div><div content-id="DSR023F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02303.jpeg?Expires=2147483647&Signature=kIqMkD9SifAOMJ6PWz8KrgAVuFd~gtRL-FhJs3m26W2DkxQeABYVeQIZBx1AiF7JKol2d6lm43~pNEGKNpNpcmOq-pazuHwtPYCOV9XXrdSCflfWjJfl6-wIk5lr-2OJ71y0B9othG-mMMCsE43carzOOxSvws7ewHS2yaBZUhKYm~6xq5W-x2xpn54hhK6vGj2-J--tYyw09YLrWMp-0eRF29hnNi32B4oWw6l9kAxAXiW89Yx6VxTz8nYBWwMiB4K2TqAXHxq1cN-zU3w9RQGQumfLhI-IFljkEommoWmTLH13r9ArxHg5jK6GGUHs1QFeeHvRtnNd8Q--vBndgQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="For the coding region of G. gallus Apo A-IV mRNA (GenBank Y16534, region: 37–1137 bp), the spectra of deviation from the proposed latent profility [Equation (3)]. (a) Deviation from the proposed 33-profility. (b) Deviation from the proposed 3-profility." path-from-xml="dsr02303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726606/dsr02303.jpeg" path-from-xml="dsr02303.jpeg" target="_blank">View large</a><a section="6726606" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02303.jpeg?Expires=2147483647&Signature=gbTS1~5mZpJ~IramrDpaa5H7WOH-cpR4ZAh~R5eX3OpZloQaLutXAnTarHzCyInhU5b0FhsY6PPFvlywudtY1IUTLKo1bmLugOudvYqDs3bh2JAj6E4xlP5q-fBzHsoqKRmexqdOQu8PRxEfrRpWExfAUR-RXmlyHwgquz1c~4k8bqtahT5hbDjiWPq3wny8znajczHMtC1JaZexPTxNwResdLlIhh3rW~J14SM-9qfPO9KZOPIOHe5Ypse-uBQCcEXCHLf~W0gGbdqUh9eHQvFnD9HmHZZ8M4XaqWDIMatqs8fymP5xpCa7uv4XG6wmV2sNONrcaTYNPgAyv9WJjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726606&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>For the coding region of <em>G. gallus</em> Apo A-IV mRNA (GenBank Y16534, region: 37–1137 bp), the spectra of deviation from the proposed latent profility [Equation (3)]. (a) Deviation from the proposed 33-profility. (b) Deviation from the proposed 3-profility.</p></div></div>                            <h3 scrollto-destination=6726609 id="6726609" class="section-title" data-legacy-id=s2e>2.5. Verification of periodicity pattern estimation</h3>
<p>To confirm the validity of the Str<em><sub>L</sub></em>(П<sub>str</sub>(<em>L</em>)) estimation for the latent profility pattern in a textual string str, a reconstruction is built of the <em>D</em><sub>1</sub> spectrum [Equation (4)] of string str deviation from homogeneity. The <em>D</em><sub>1</sub> spectrum has been chosen as the most informative from the spectra pool of str deviation from the profility [Equation (3)].</p><div class="block-child-p">The reconstruction is realized on the basis of the Str<em><sub>L</sub></em>(П<sub>str</sub>(<em>L</em>)) pattern inducing the periodic profile string Tdm<em><sub>L</sub></em> = Tdm<em><sub>L</sub></em>(П<sub>str</sub>(<em>L</em>), <em>n</em>). Thus, by analogy with Equation (4), for theoretical reconstruction of the <em>D</em><sub>1</sub> spectrum, the Th<em><sub>L</sub></em> spectrum is chosen that at test-period <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq59.gif?Expires=2147483647&amp;Signature=lQHT9o-bZ2N9zvxVHH5ovYoruiwPFOJplNon9jz5k3q2xTQEYIgqNO1EV0wrLv1TdfuwJynhC5nGkC36DiiUItCNDPrxV204TsZ62lCHQSOZJRQK5IC2LDCeKLxK46HgYkX29~SFaPrxeBpa9SyqAp43hw3gI~0WGnmAC7T75gVrWbjCh0UfqKpJzbSh6fmH1KdynX3hKVByybjWveirEuFKYvATwq5cw6EElnu6XbNMd9~BHKdScO-ph~Rfl1~O7VIiMf6HxOeBIK~nclpp-iahrV7rKtd~yRzy2IasTr0-YmklmRAgw8YBGDheqVot9ChEJDUu~yLP5PpclLKUbg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq59.gif" /></span> of string str takes on the value <a id="DSR023M5" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSR023M5"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr023eq5.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023eq5.gif?Expires=2147483647&amp;Signature=ubXgFXIanySq6tXcOUpeIKOJJD57pcXXQgvgVD5y2FdzcuxwVwRCLT0joAvMwhZhxKRmm8OTviFU1GXrZWWcZ~MlrshuzhT-3~0ImfKOEK7CL5r~75fkwHRETq2OQJIQbSz1LqVbhpY~fYiTHo7fsYfLHO0eOWjo0DKCeIqS0MSj~G2VpGwGF6mbFhh9VL4Zs-JgrdGQOkbfnf6AylhAFCVVmcBtPCBRuYZ6MDzf3ARdZmIymd8w3e3ov6OjxWquDquxaRdpkmIut3SpVMRF-X6nlFKNSUewzPI71ut-eegr2Kq6OLtW3S8w54xeXz04vveFOY4cvkxbBMy7qXjgLQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr023eq5.gif" /></a></div></div><div class="label">(5)</div></div> If the Th<em><sub>L</sub></em> spectrum follows the <em>D</em><sub>1</sub> spectrum (Fig. <a reveal-id="DSR023F4" data-open="DSR023F4" class="link link-reveal link-table xref-fig">4</a>a and b), then the Str<em><sub>L</sub></em>(П<sub>str</sub>(<em>L</em>)) estimation of the pattern of latent <em>L</em>-profile periodicity (<em>L</em>-profility) in analysed textual string str is correct. Therefore, latent <em>L</em>-profile periodicity (<em>L</em>-profility) in string str is confirmed. </div>                        <a id="6726619" scrollto-destination="6726619"></a>
<div id="DSR023F4" data-id="DSR023F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02304.jpeg?Expires=2147483647&Signature=RsTMzoL9Xa5IZU4b1EQCpH895HVZbCPfx6-zrHetTANwEaxngW1~fqsqF63omXcCM1XVA8o~TFaANLGeCPJBG-k8Houm2u11Tto0A8Txzb46mcGPw3g7luytXe2633PuMTcRTLYO6ZgahYVrjZ~E6Yb~Mm6MyxICELYaWE3PIQ2hgaf8HUCnLQ3hTPLbUuZQ9pBPCqvV1~ENqaxkffbngU2yK-YknkmMMsQ-VrCZIMqyIQD31UPSaAFdui37-md3AL0puUKTw6XROpQKKuq8~~gE7ZSy~3oEqEaDPcBmYV6cAOfVfy2Hr9nLjgjlronjqjWBlnVCR6VC6F8ooROLQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Verification of pattern estimation for latent profile periodicity of 33 bp (33-profility) in the coding region of G. gallus Apo A-IV mRNA (GenBank Y16534, region: 37–1137 bp). (a) Spectrum of deviation from homogeneity [Equation (4)]. (b) Theoretical [Equation (5)] and (c) statistical [Equation (6)] reconstructions of the spectrum in (a), under the assumption of 33-profility in the region. (d) Theoretical reconstruction of the spectrum in (a), under the assumption of 3-profility in the region." path-from-xml="dsr02304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726619/dsr02304.jpeg" path-from-xml="dsr02304.jpeg" target="_blank">View large</a><a section="6726619" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02304.jpeg?Expires=2147483647&Signature=12I-JmG8H3UHglprvR4AWlmqJ2JEvlny53w423au3a00~qxl-ftZBSb629i4iLcvpcAB~0HESp8Ma6oU~edF1HvjbIh~7gaBBmOmMOFNAGHWnHRamCulJNu49H1yvicGhB-LMO70nkE98H8w5NHU~g9D5aDvuJwupBaL-JIh40Rb9aLr1pK0gdOO75HMsUJPNkW0nMU4OYOmp~zzAHy9SJO15UCrM1MQkQpiN-pLnFQRMf3sibk17iSGKjnv4H4NgIN52yT488EKhWN1xDxedtR2KdKDEEWdSRjwROWQNRFD4atKk~VYvXWBFU5HkAsQ7e7ylfHunQaxoDLCq7WeFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726619&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Verification of pattern estimation for latent profile periodicity of 33 bp (33-profility) in the coding region of <em>G. gallus</em> Apo A-IV mRNA (GenBank Y16534, region: 37–1137 bp). (a) Spectrum of deviation from homogeneity [Equation (4)]. (b) Theoretical [Equation (5)] and (c) statistical [Equation (6)] reconstructions of the spectrum in (a), under the assumption of 33-profility in the region. (d) Theoretical reconstruction of the spectrum in (a), under the assumption of 3-profility in the region.</p></div></div><div content-id="DSR023F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02304.jpeg?Expires=2147483647&Signature=RsTMzoL9Xa5IZU4b1EQCpH895HVZbCPfx6-zrHetTANwEaxngW1~fqsqF63omXcCM1XVA8o~TFaANLGeCPJBG-k8Houm2u11Tto0A8Txzb46mcGPw3g7luytXe2633PuMTcRTLYO6ZgahYVrjZ~E6Yb~Mm6MyxICELYaWE3PIQ2hgaf8HUCnLQ3hTPLbUuZQ9pBPCqvV1~ENqaxkffbngU2yK-YknkmMMsQ-VrCZIMqyIQD31UPSaAFdui37-md3AL0puUKTw6XROpQKKuq8~~gE7ZSy~3oEqEaDPcBmYV6cAOfVfy2Hr9nLjgjlronjqjWBlnVCR6VC6F8ooROLQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Verification of pattern estimation for latent profile periodicity of 33 bp (33-profility) in the coding region of G. gallus Apo A-IV mRNA (GenBank Y16534, region: 37–1137 bp). (a) Spectrum of deviation from homogeneity [Equation (4)]. (b) Theoretical [Equation (5)] and (c) statistical [Equation (6)] reconstructions of the spectrum in (a), under the assumption of 33-profility in the region. (d) Theoretical reconstruction of the spectrum in (a), under the assumption of 3-profility in the region." path-from-xml="dsr02304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726619/dsr02304.jpeg" path-from-xml="dsr02304.jpeg" target="_blank">View large</a><a section="6726619" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02304.jpeg?Expires=2147483647&Signature=12I-JmG8H3UHglprvR4AWlmqJ2JEvlny53w423au3a00~qxl-ftZBSb629i4iLcvpcAB~0HESp8Ma6oU~edF1HvjbIh~7gaBBmOmMOFNAGHWnHRamCulJNu49H1yvicGhB-LMO70nkE98H8w5NHU~g9D5aDvuJwupBaL-JIh40Rb9aLr1pK0gdOO75HMsUJPNkW0nMU4OYOmp~zzAHy9SJO15UCrM1MQkQpiN-pLnFQRMf3sibk17iSGKjnv4H4NgIN52yT488EKhWN1xDxedtR2KdKDEEWdSRjwROWQNRFD4atKk~VYvXWBFU5HkAsQ7e7ylfHunQaxoDLCq7WeFQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726619&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Verification of pattern estimation for latent profile periodicity of 33 bp (33-profility) in the coding region of <em>G. gallus</em> Apo A-IV mRNA (GenBank Y16534, region: 37–1137 bp). (a) Spectrum of deviation from homogeneity [Equation (4)]. (b) Theoretical [Equation (5)] and (c) statistical [Equation (6)] reconstructions of the spectrum in (a), under the assumption of 33-profility in the region. (d) Theoretical reconstruction of the spectrum in (a), under the assumption of 3-profility in the region.</p></div></div><div class="block-child-p">Instead of the theoretical reconstruction spectrum [Equation (5)], we can use the statistical (St<em><sub>L</sub></em>) reconstruction of the <em>D</em><sub>1</sub> spectrum (Fig. <a reveal-id="DSR023F4" data-open="DSR023F4" class="link link-reveal link-table xref-fig">4</a>a and c). In this case, by using a random number generator and the main <em>L</em>-profile matrix П<sub>str</sub>(<em>L</em>) of the string Tdm<em><sub>L</sub></em>, string str* is created as a realization of the string Tdm<sub>L</sub>. Then, by analogy with Equation (4), the value of the St<em><sub>L</sub></em> spectrum at the test-period <span class="inline-formula no-formula-id"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023ileq60.gif?Expires=2147483647&amp;Signature=aJVZZhXA~RNwRxXQ4WyhCP1WVx4w-Kz44RTbJxuCdjbXYN6-Q7OTGQ1KbC9xiKtnLn04PS-GHfOM85izhxAdW4kJ8m3~4LsjW188Uuok0Je2F~XxCU8pugjeb2LEP7WUKCM1Gt7BpK3Xj~utNbwHEj8Ln-pU3q9fQL20l5nIHDp8ckYlilL6QWyum-tLUciC6Q2tAA8cO1RdQoAoZX4UGY2A851Cn70xG56JzcCJGXh5uRPJ3cv699cG887pIDxAES7ghd9n6yWKCamTktckbm2aG9JXiAy1M3nLTpsdeLZm-NFiaRsTKrtWlcbvvY8CjbdYYm92y9yKK503-Sy1UQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="forumla" path-from-xml="dsr023ileq60.gif" /></span> is calculated as follows: <a id="DSR023M6" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSR023M6"><div class="fig-graphic"><a class="jumplink-placeholder"  id="dsr023eq6.gif"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr023eq6.gif?Expires=2147483647&amp;Signature=TnQnen0W8m~n6xSXu0ov2S-gZAtgU9w4nP8FCdafaiBbC8x611f999hM8Y6m0JJsAvrL6zybaRPAlavqAWnQ~-pH3yE1zNB4KihNxy5a~a~eYkoEaqkxpUkB-ojeS2PF68OAF2js9Cr9OCqwQ5R3xqnFvPn2tAFNtnCPllYpfVVsZ6h1U9UTYaWnA~Jnd-p7U2NSoTPygAr3o7WmLomVZ1owGWLItxJs4MgDio8zhxU21gVzOiG3J4-PJl1CrMZSy7~O9ABbFcxwXyRUnFUQr-2J4dRFVE4aKgJUN~hSa1CCVMFLLH-Y-FrbCB0PQHRYi1mrhavYYf0-JoPvVHs1tQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="formula" path-from-xml="dsr023eq6.gif" /></a></div></div><div class="label">(6)</div></div> where Tdm*<sub>1</sub> = Tdm<sub>1</sub>(П<sub>str*</sub>(1), <em>n</em>). Statistical reconstruction should be used when regular minima in the <em>D</em><sub>1</sub> spectrum clear deviate from null.</div>                            <h2 scrollto-destination=6726627 id="6726627" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
<p>Methods of identifying a new type of latent periodicity in DNA called latent profile periodicity, or profility, have been proposed in the present work. A characteristic of this profile periodicity is the random nature of its pattern. The profile matrix of the pattern determines statistical periodicity in the appearance of the characters in textual strings. As a result, latent profile periodicity manifests in the analysed string.</p>                            <h3 scrollto-destination=6726634 id="6726634" class="section-title" data-legacy-id=s3a>3.1. Profile-statistical basis of structural domains in protein families</h3>
<p>Application of the methods proposed in this work enabled us to discover the occurrence of latent profility for the 33 nucleotides (33-profility) in the coding gene regions of the PF01442 apolipoprotein family from the Pfam (<a class="link link-uri" href="http://pfam.sanger.ac.uk/">http://pfam.sanger.ac.uk/</a>) database of protein families. This family contains the apolipoproteins Apo A, Apo C and Apo E, which are members of a multigene family that probably evolved from a common ancestral gene. Apolipoproteins function in lipid transport as structural components of lipoprotein particles, cofactors for enzymes and ligands for cell-surface receptors. The family contains more than 800 protein sequences from ∼100 species. In each position of the family, multiple alignment shows an average identity of amino acids of ∼30%. By taking this apolipoprotein family as a case study, we can demonstrate a procedure for identifying latent profility.</p><p>The characteristic spectra of coding regions for the apolipoproteins Apo E of house mouse <em>Mus musculus</em>, Apo A-I of gilt-head sea bream <em>Sparus aurata</em> and Apo A-IV of chicken <em>G. gallus</em> are shown in Fig. <a reveal-id="DSR023F2" data-open="DSR023F2" class="link link-reveal link-table xref-fig">2</a>a–c. In these spectra, the first clear maximum is found at the test-period of the 33 base pairs (bp). Thus, the estimation of a pattern size equal to the 33 bp is proposed. The maximum values in the spectra of deviation from the 33-profility [Equation (3)] do not exceed a figure of one (<em>D</em><sub>33</sub> &lt; 1) that illustrated in Fig. <a reveal-id="DSR023F3" data-open="DSR023F3" class="link link-reveal link-table xref-fig">3</a>a for chicken Apo A-IV. Using the result for the considered coding regions, estimates of patterns for the 33-profile periodicity may be proposed. These estimates are determined by a sample 33-profile matrix of the corresponding analysed region. The reasonableness of each pattern estimate is confirmed by similarity between the spectrum of deviation from homogeneity and its theoretical, or statistical, reconstruction for the analysed coding region. An example of verification of pattern estimation for latent 33-profility in the chicken Apo A-IV coding region is shown in Fig. <a reveal-id="DSR023F4" data-open="DSR023F4" class="link link-reveal link-table xref-fig">4</a>. Comparison of Fig. <a reveal-id="DSR023F4" data-open="DSR023F4" class="link link-reveal link-table xref-fig">4</a>a with d disproves the presence of latent 3-profility in the region, though a peak at frequency 0.33 corresponding to the test-period of 3 bp dominates in the Fourier spectrum (Fig. <a reveal-id="DSR023F2" data-open="DSR023F2" class="link link-reveal link-table xref-fig">2</a>d).</p><p>To check a robustness of the latent 33-profility pattern estimation found for the coding regions of apolipoproteins the damages in different consecutive segments of 33 bp have been simulated. Every <em>k</em>th (<em>k</em> = 5,4,3,2) segment of the coding region has been substituted by a fragment (of the same length) from homogeneous sequence with equally probable distribution of the nucleotides. The example of analysis of gene sequence with such a noise is shown in Fig. <a reveal-id="DSR023F5" data-open="DSR023F5" class="link link-reveal link-table xref-fig">5</a> for Apo A-IV mRNA of chicken <em>G. gallus</em>. Spectral–statistical analysis reveals a cutoff in pattern recognition when the sequence damages became equal to 25% (<em>k</em> = 4). In this case, there is a dilemma that what pattern size should be chosen—33 or 66 bp. According to the theoretical reconstructions (Fig. <a reveal-id="DSR023F5" data-open="DSR023F5" class="link link-reveal link-table xref-fig">5</a>c and d) of the spectrum of deviation from homogeneity (Fig. <a reveal-id="DSR023F5" data-open="DSR023F5" class="link link-reveal link-table xref-fig">5</a>b), an estimate of 66 bp appears to be more preferable. Such a preference becomes obvious with 50% of damages when the latent profile periodicity of 66 bp arises naturally. In the analysis done, no essential regions in the apolipoprotein gene sequences were revealed which determinate the occurrence of latent profility in the genes. The latent 33-profility of the apolipoprotein genes seems to be a consequence of consistent statistical low of their structural organization. </p>                        <a id="6726646" scrollto-destination="6726646"></a>
<div id="DSR023F5" data-id="DSR023F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02305.jpeg?Expires=2147483647&Signature=tzKM6o-NCn5FVg2ng55NM1bdxssjNKa3vTf-DtWRXFjExu5~CFR32rTZLw4OBicaTimzk8Z92q2VDTWIWTf7FWgEYXtRFKdVu0kXR80WJow25e8WAtrxUBkVnEdyAVyJcBBuatHekfmy4Io-udloRRTJJ3OogocUZ~rzzMGhll-~9WR0IUUIRs19z3-BQQnKotMEBFopDs-jMTrLjCuUkm-Ia2gTUb2YtYgZu~ct5UFMO41pDK1fvC9zrWQingKDuw5WAeEu9U4lVJHLVec-wPY3XKX8OG4GbbbcaJs7vUCcdvHLehTyuH-HGz3cV45VmlDCqAPKDDUmqrtalIY5fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Robustness analysis of pattern estimation for the 33-profility in the coding region of G. gallus Apo A-IV mRNA (GenBank Y16534, region: 37–1137 bp). See Fig. 2c for the original characteristic spectrum of the region. Upper series: spectral-statistical analysis of the region sequence containing 25% of the destroyed 33-segments. Lower series: analysis of the region sequence containing 50% of the destroyed 33-segments. For the corresponding analysed sequences: (a and e) characteristic spectrum [Equation (2)], (b and f) the D1 spectrum [Equation (4)] of sequence deviation from homogeneity, (c and g) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 66-profility in the analysed sequence, (d and h) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 33-profility in the analysed sequence." path-from-xml="dsr02305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726646/dsr02305.jpeg" path-from-xml="dsr02305.jpeg" target="_blank">View large</a><a section="6726646" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02305.jpeg?Expires=2147483647&Signature=rqv6Ps-iORuCw0nTCdpk7rRbpWuFENXELNur0be6YQmBb0FOKvaQvYJ3fln5BHKjM9m~0-Na16Ueue1JIDwD6bspbc8DuCgVlAPdo6c~v5Y6IpmrOcVj7K2dxMstwcaLZakuBu~9T4vMXZ0sG6R9k~u9jA79fOn-jBmJ4hFJUbAq-gT5pPwWcB4SAbF5pyUmnPAl-MjVxkViXyo1DZQpRn9lupRT24VS6WS3P12~03jwluFxTa9b5qLtLwp2MjDeQxuFBovVOqHU6flhY40fRIhgFXKr0TtISnCEBP7zpBOlErF4TlId5ZmxML-43JotngkbMcmO6RN~4oLYo6BpqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726646&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Robustness analysis of pattern estimation for the 33-profility in the coding region of <em>G. gallus</em> Apo A-IV mRNA (GenBank Y16534, region: 37–1137 bp). See Fig. <a reveal-id="DSR023F2" data-open="DSR023F2" class="link link-reveal link-table xref-fig">2</a>c for the original characteristic spectrum of the region. Upper series: spectral-statistical analysis of the region sequence containing 25% of the destroyed 33-segments. Lower series: analysis of the region sequence containing 50% of the destroyed 33-segments. For the corresponding analysed sequences: (a and e) characteristic spectrum [Equation (2)], (b and f) the D1 spectrum [Equation (4)] of sequence deviation from homogeneity, (c and g) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 66-profility in the analysed sequence, (d and h) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 33-profility in the analysed sequence.</p></div></div><div content-id="DSR023F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02305.jpeg?Expires=2147483647&Signature=tzKM6o-NCn5FVg2ng55NM1bdxssjNKa3vTf-DtWRXFjExu5~CFR32rTZLw4OBicaTimzk8Z92q2VDTWIWTf7FWgEYXtRFKdVu0kXR80WJow25e8WAtrxUBkVnEdyAVyJcBBuatHekfmy4Io-udloRRTJJ3OogocUZ~rzzMGhll-~9WR0IUUIRs19z3-BQQnKotMEBFopDs-jMTrLjCuUkm-Ia2gTUb2YtYgZu~ct5UFMO41pDK1fvC9zrWQingKDuw5WAeEu9U4lVJHLVec-wPY3XKX8OG4GbbbcaJs7vUCcdvHLehTyuH-HGz3cV45VmlDCqAPKDDUmqrtalIY5fg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Robustness analysis of pattern estimation for the 33-profility in the coding region of G. gallus Apo A-IV mRNA (GenBank Y16534, region: 37–1137 bp). See Fig. 2c for the original characteristic spectrum of the region. Upper series: spectral-statistical analysis of the region sequence containing 25% of the destroyed 33-segments. Lower series: analysis of the region sequence containing 50% of the destroyed 33-segments. For the corresponding analysed sequences: (a and e) characteristic spectrum [Equation (2)], (b and f) the D1 spectrum [Equation (4)] of sequence deviation from homogeneity, (c and g) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 66-profility in the analysed sequence, (d and h) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 33-profility in the analysed sequence." path-from-xml="dsr02305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726646/dsr02305.jpeg" path-from-xml="dsr02305.jpeg" target="_blank">View large</a><a section="6726646" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02305.jpeg?Expires=2147483647&Signature=rqv6Ps-iORuCw0nTCdpk7rRbpWuFENXELNur0be6YQmBb0FOKvaQvYJ3fln5BHKjM9m~0-Na16Ueue1JIDwD6bspbc8DuCgVlAPdo6c~v5Y6IpmrOcVj7K2dxMstwcaLZakuBu~9T4vMXZ0sG6R9k~u9jA79fOn-jBmJ4hFJUbAq-gT5pPwWcB4SAbF5pyUmnPAl-MjVxkViXyo1DZQpRn9lupRT24VS6WS3P12~03jwluFxTa9b5qLtLwp2MjDeQxuFBovVOqHU6flhY40fRIhgFXKr0TtISnCEBP7zpBOlErF4TlId5ZmxML-43JotngkbMcmO6RN~4oLYo6BpqQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726646&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Robustness analysis of pattern estimation for the 33-profility in the coding region of <em>G. gallus</em> Apo A-IV mRNA (GenBank Y16534, region: 37–1137 bp). See Fig. <a reveal-id="DSR023F2" data-open="DSR023F2" class="link link-reveal link-table xref-fig">2</a>c for the original characteristic spectrum of the region. Upper series: spectral-statistical analysis of the region sequence containing 25% of the destroyed 33-segments. Lower series: analysis of the region sequence containing 50% of the destroyed 33-segments. For the corresponding analysed sequences: (a and e) characteristic spectrum [Equation (2)], (b and f) the D1 spectrum [Equation (4)] of sequence deviation from homogeneity, (c and g) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 66-profility in the analysed sequence, (d and h) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 33-profility in the analysed sequence.</p></div></div><p>In earlier work,<sup><a reveal-id="DSR023C23" data-open="DSR023C23" class="link link-ref link-reveal xref-bibr">23</a></sup> after diagonal dot matrix analysis of internal homology within <em>M. musculus</em> Apo E mRNA, it was concluded that gene evolution took place by duplication of an 11-bp ancestral sequence. The supposition was also made that, before the genes of Apo E, Apo A-I and Apo A-IV were formed by duplication of the general 33-bp unit, copies of the ancient 11-‘pattern’ in the tandem 33-repeat underwent essential mutational alterations. In mouse Apo E mRNA fragment (GenBank M12414, 275–604 bp), 3-, 11- and 33-profility were examined in the present work. It is this fragment for which an ancestral 11-bp sequence was derived previously.<sup><a reveal-id="DSR023C23" data-open="DSR023C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Using the methods proposed here, only 33-profility has been revealed and confirmed for the fragment. The same conclusion is made for the entire mouse Apo E mRNA sequence. Fig. <a reveal-id="DSR023F6" data-open="DSR023F6" class="link link-reveal link-table xref-fig">6</a> illustrates the performed analysis. Theoretical reconstructions of the <em>D</em><sub>1</sub> spectra of deviation from homogeneity (Fig. <a reveal-id="DSR023F6" data-open="DSR023F6" class="link link-reveal link-table xref-fig">6</a>a and e) were undertaken with the assumption of the occurrence of latent 33-profility in both the particular fragment and the entire Apo E mRNA (Fig. <a reveal-id="DSR023F6" data-open="DSR023F6" class="link link-reveal link-table xref-fig">6</a>b and f, respectively), follow the corresponding <em>D</em><sub>1</sub> spectra. Theoretical reconstructions (Fig. <a reveal-id="DSR023F6" data-open="DSR023F6" class="link link-reveal link-table xref-fig">6</a>c and g) of the same <em>D</em><sub>1</sub> spectra, undertaken with the assumption of the presence of latent 11-profility in the analysed sequences, are not similar to the corresponding <em>D</em><sub>1</sub> spectra. Moreover, the <em>D</em><sub>11</sub> spectra of deviation from 11-profility (Fig. <a reveal-id="DSR023F6" data-open="DSR023F6" class="link link-reveal link-table xref-fig">6</a>d and h) at numerous test-periods exceed the threshold (<em>D</em><sub>11</sub> &gt; 1), indicating the absence of 11-profility in the analysed sequences. Domination of some nucleotides (more than 50%), revealed earlier<sup><a reveal-id="DSR023C23" data-open="DSR023C23" class="link link-ref link-reveal xref-bibr">23</a></sup> in four positions (the 2nd, 3rd, 7th and 9th) of the quasi-pattern of the 11 bp, is probably due to structural particularity of textual 33-repeat from the central part (275–604 bp) of mouse Apo E mRNA. In contrast, 33-profility undoubtedly settles a fixed periodicity of appearance for the nucleotides both in the central part and over the whole analysed Apo E mRNA. </p>                        <a id="6726651" scrollto-destination="6726651"></a>
<div id="DSR023F6" data-id="DSR023F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02306.jpeg?Expires=2147483647&Signature=feMvWcG1KVS5dXTo1Nyh4e~FJDrZXfdnI40OkFgAr5f~Wp--27SsC8lbqeIYVF8lVBUpyiIH05oH0HKnTf8HuGO~4BPeHc71dfq5wgrLZcbq4JAvttswZafPl3YHzmMJK3bA2EmoDpIAlCmNxeUJQusZcjlxVoPNJXBgzPnQxl57FIJyTP7UOKNQVG8pQH8AjREk903NZuCkT2J0VVqwIxNaD57Z8Oi~upf1jRQXMf2mzmRROflin~82ZxHXLlh~0fA~mFH6KLXxFjuPWmLPHOYeGfiUn0YSNz6qIkrTvrUOJb1ouzSF-O65iieNTyCSoQcsQi9do9sH4Iqy8-NSkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Upper series: verification of pattern estimation for latent profile periodicity of 33 bp (33-profility) in the central part of M. musculus Apo E mRNA (GenBank M12414, region: 275–604 bp). Lower series: verification of pattern estimation for latent 33-profility in the whole Apo E mRNA (GenBank M12414, region: 1–936 bp). For the corresponding analysed sequences: (a and e) the D1 spectrum [Equation (4)] of sequence deviation from homogeneity, (b and f) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 33-profility in an analysed sequence, (c and g) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 11-profility in an analysed sequence, (d and h) the D11 spectrum [Equation (4)] of sequence deviation from the 11-profility." path-from-xml="dsr02306.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726651/dsr02306.jpeg" path-from-xml="dsr02306.jpeg" target="_blank">View large</a><a section="6726651" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02306.jpeg?Expires=2147483647&Signature=C~N69XPb0WFwTNzqSHAbOVphFBJbr1BzUBwD92wlgdBnCVuQ9GMCt-MZabnM5LWPD~6ltaUFa59LD9c9Pmhl6A2TF0v2wFHh3cJZq09GZbo5KQGn-GHYV1otVUNRGs-2O4UECeBXJ5VG9BMvblK40Aa4wPThi2PyZ6H-sU6GgI55x9H31FB3KFbpFRNaiRD~4KZlgW-zHDce~HAc9HzOXIaDW2WfOV6ARiMkYaNn-vpv3CSDEHQ4np-Wb3pZWxawu6XxbeHML8tFwdewTIbzMfBxdv1nNKE2VlQhguAtf2gfRRokYWAQHK5UwFXXQEE~1yAIqI7J-kFcakqtpfRJWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726651&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02306.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Upper series: verification of pattern estimation for latent profile periodicity of 33 bp (33-profility) in the central part of <em>M. musculus</em> Apo E mRNA (GenBank M12414, region: 275–604 bp). Lower series: verification of pattern estimation for latent 33-profility in the whole Apo E mRNA (GenBank M12414, region: 1–936 bp). For the corresponding analysed sequences: (a and e) the D1 spectrum [Equation (4)] of sequence deviation from homogeneity, (b and f) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 33-profility in an analysed sequence, (c and g) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 11-profility in an analysed sequence, (d and h) the D11 spectrum [Equation (4)] of sequence deviation from the 11-profility.</p></div></div><div content-id="DSR023F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02306.jpeg?Expires=2147483647&Signature=feMvWcG1KVS5dXTo1Nyh4e~FJDrZXfdnI40OkFgAr5f~Wp--27SsC8lbqeIYVF8lVBUpyiIH05oH0HKnTf8HuGO~4BPeHc71dfq5wgrLZcbq4JAvttswZafPl3YHzmMJK3bA2EmoDpIAlCmNxeUJQusZcjlxVoPNJXBgzPnQxl57FIJyTP7UOKNQVG8pQH8AjREk903NZuCkT2J0VVqwIxNaD57Z8Oi~upf1jRQXMf2mzmRROflin~82ZxHXLlh~0fA~mFH6KLXxFjuPWmLPHOYeGfiUn0YSNz6qIkrTvrUOJb1ouzSF-O65iieNTyCSoQcsQi9do9sH4Iqy8-NSkQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Upper series: verification of pattern estimation for latent profile periodicity of 33 bp (33-profility) in the central part of M. musculus Apo E mRNA (GenBank M12414, region: 275–604 bp). Lower series: verification of pattern estimation for latent 33-profility in the whole Apo E mRNA (GenBank M12414, region: 1–936 bp). For the corresponding analysed sequences: (a and e) the D1 spectrum [Equation (4)] of sequence deviation from homogeneity, (b and f) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 33-profility in an analysed sequence, (c and g) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 11-profility in an analysed sequence, (d and h) the D11 spectrum [Equation (4)] of sequence deviation from the 11-profility." path-from-xml="dsr02306.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726651/dsr02306.jpeg" path-from-xml="dsr02306.jpeg" target="_blank">View large</a><a section="6726651" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02306.jpeg?Expires=2147483647&Signature=C~N69XPb0WFwTNzqSHAbOVphFBJbr1BzUBwD92wlgdBnCVuQ9GMCt-MZabnM5LWPD~6ltaUFa59LD9c9Pmhl6A2TF0v2wFHh3cJZq09GZbo5KQGn-GHYV1otVUNRGs-2O4UECeBXJ5VG9BMvblK40Aa4wPThi2PyZ6H-sU6GgI55x9H31FB3KFbpFRNaiRD~4KZlgW-zHDce~HAc9HzOXIaDW2WfOV6ARiMkYaNn-vpv3CSDEHQ4np-Wb3pZWxawu6XxbeHML8tFwdewTIbzMfBxdv1nNKE2VlQhguAtf2gfRRokYWAQHK5UwFXXQEE~1yAIqI7J-kFcakqtpfRJWQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726651&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02306.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Upper series: verification of pattern estimation for latent profile periodicity of 33 bp (33-profility) in the central part of <em>M. musculus</em> Apo E mRNA (GenBank M12414, region: 275–604 bp). Lower series: verification of pattern estimation for latent 33-profility in the whole Apo E mRNA (GenBank M12414, region: 1–936 bp). For the corresponding analysed sequences: (a and e) the D1 spectrum [Equation (4)] of sequence deviation from homogeneity, (b and f) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 33-profility in an analysed sequence, (c and g) theoretical reconstruction [Equation (5)] of the D1 spectrum under supposition of the presence of 11-profility in an analysed sequence, (d and h) the D11 spectrum [Equation (4)] of sequence deviation from the 11-profility.</p></div></div><p>The known secondary structure of the apolipoprotein family PF01442 contains several pairs of α helices of the 11 and 22 amino acid residues. Such a spatial organization correlates with the 33-bp profile periodicity of the apolipoprotein genes. The generic size of the pattern of latent profile periodicity in the PF01442 family genes possibly influences the formation of the typical secondary structure for the protein family and agrees well with the hypothesis on family origin from a common ancestral gene.</p><p>A generic size of ∼290 bp for the latent profile periodicity pattern is observed in the genes of fibronectin type III domain-containing protein, which is a protein of an intercellular matrix. It is a glycoprotein that many cells synthesize and secrete into intercellular space. The fibronectin consists of two identical polypeptide chains joined by disulfide bridges near the C-terminuses. Each polypeptide chain contains ∼10 domains, each of which holds the specific sites binding the various substances. The proposed spatial structure of the domain contains seven antiparallel β-strands.<sup><a reveal-id="DSR023C24" data-open="DSR023C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p><p>Orthologous genes of the fibronectin type III domain family that have been analysed in the present work are listed in Table <a reveal-id="DSR023TB1" data-open="DSR023TB1" class="link link-reveal link-table xref-fig">1</a>. Table <a reveal-id="DSR023TB2" data-open="DSR023TB2" class="link link-reveal link-table xref-fig">2</a>, using data from the KEGG database (<a class="link link-uri" href="http://www.genome.jp/kegg/">http://www.genome.jp/kegg/</a>), shows an identity percentage between pairs of the protein family. In characteristic spectra (e.g. see Fig. <a reveal-id="DSR023F7" data-open="DSR023F7" class="link link-reveal link-table xref-fig">7</a>a) of genes from this family, the generic pattern size of latent profile periodicity is found to be 291 bp. Statistical reconstruction (Fig. <a reveal-id="DSR023F7" data-open="DSR023F7" class="link link-reveal link-table xref-fig">7</a>с) corresponding to the pattern size of 291 bp reconstitutes the spectrum of deviation from homogeneity (Fig. <a reveal-id="DSR023F7" data-open="DSR023F7" class="link link-reveal link-table xref-fig">7</a>b), which verifies latent profile periodicity with this period. The pattern size of 291 bp is in good agreement with the size of repeated domains (∼90–100 amino acids) in the proteins of the family. </p>                        <a id="6726660" scrollto-destination="6726660"></a>
<div content-id="DSR023TB1" class="table-modal"><div class="table-wrap"><div id="DSR023TB1" class="table-wrap-title" data-id="DSR023TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Analysed orthologous genes of the fibronectin type III domain family</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Organism </th><th>KEGG entry </th><th>CDS (bp) </th><th>Protein </th><th>Number of domains </th></tr></thead><tbody><tr><td><em>Homo sapience</em> (human) </td><td>hsa:22862 </td><td>3597 </td><td>Fibronectin type III domain containing protein 3A, 1198 amino acids </td><td>9 </td></tr><tr><td><em>Mus musculus</em> (mouse) </td><td>mmu:319448 </td><td>3597 </td><td>Fibronectin type III domain containing protein 3A, 1198 amino acids </td><td>9 </td></tr><tr><td><em>Gallus gallus</em> (chicken) </td><td>gga:418863 </td><td>3600 </td><td>Fibronectin type III domain containing protein 3A, 1199 amino acids </td><td>9 </td></tr><tr><td><em>Xenopus laevis</em> (frog) </td><td>xla:446899 </td><td>3600 </td><td>Fibronectin type III domain containing protein 3A, 1199 amino acids </td><td>9 </td></tr></tbody></table></div><div content-id="DSR023TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Organism </th><th>KEGG entry </th><th>CDS (bp) </th><th>Protein </th><th>Number of domains </th></tr></thead><tbody><tr><td><em>Homo sapience</em> (human) </td><td>hsa:22862 </td><td>3597 </td><td>Fibronectin type III domain containing protein 3A, 1198 amino acids </td><td>9 </td></tr><tr><td><em>Mus musculus</em> (mouse) </td><td>mmu:319448 </td><td>3597 </td><td>Fibronectin type III domain containing protein 3A, 1198 amino acids </td><td>9 </td></tr><tr><td><em>Gallus gallus</em> (chicken) </td><td>gga:418863 </td><td>3600 </td><td>Fibronectin type III domain containing protein 3A, 1199 amino acids </td><td>9 </td></tr><tr><td><em>Xenopus laevis</em> (frog) </td><td>xla:446899 </td><td>3600 </td><td>Fibronectin type III domain containing protein 3A, 1199 amino acids </td><td>9 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6726660" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR023TB1" class="table-wrap-title" data-id="DSR023TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Analysed orthologous genes of the fibronectin type III domain family</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Organism </th><th>KEGG entry </th><th>CDS (bp) </th><th>Protein </th><th>Number of domains </th></tr></thead><tbody><tr><td><em>Homo sapience</em> (human) </td><td>hsa:22862 </td><td>3597 </td><td>Fibronectin type III domain containing protein 3A, 1198 amino acids </td><td>9 </td></tr><tr><td><em>Mus musculus</em> (mouse) </td><td>mmu:319448 </td><td>3597 </td><td>Fibronectin type III domain containing protein 3A, 1198 amino acids </td><td>9 </td></tr><tr><td><em>Gallus gallus</em> (chicken) </td><td>gga:418863 </td><td>3600 </td><td>Fibronectin type III domain containing protein 3A, 1199 amino acids </td><td>9 </td></tr><tr><td><em>Xenopus laevis</em> (frog) </td><td>xla:446899 </td><td>3600 </td><td>Fibronectin type III domain containing protein 3A, 1199 amino acids </td><td>9 </td></tr></tbody></table></div><div content-id="DSR023TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Organism </th><th>KEGG entry </th><th>CDS (bp) </th><th>Protein </th><th>Number of domains </th></tr></thead><tbody><tr><td><em>Homo sapience</em> (human) </td><td>hsa:22862 </td><td>3597 </td><td>Fibronectin type III domain containing protein 3A, 1198 amino acids </td><td>9 </td></tr><tr><td><em>Mus musculus</em> (mouse) </td><td>mmu:319448 </td><td>3597 </td><td>Fibronectin type III domain containing protein 3A, 1198 amino acids </td><td>9 </td></tr><tr><td><em>Gallus gallus</em> (chicken) </td><td>gga:418863 </td><td>3600 </td><td>Fibronectin type III domain containing protein 3A, 1199 amino acids </td><td>9 </td></tr><tr><td><em>Xenopus laevis</em> (frog) </td><td>xla:446899 </td><td>3600 </td><td>Fibronectin type III domain containing protein 3A, 1199 amino acids </td><td>9 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6726660" target="_blank">View Large</a></div></div>                        <a id="6726662" scrollto-destination="6726662"></a>
<div content-id="DSR023TB2" class="table-modal"><div class="table-wrap"><div id="DSR023TB2" class="table-wrap-title" data-id="DSR023TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Identity percentage between the protein pairs from Table <a reveal-id="DSR023TB1" data-open="DSR023TB1" class="link link-reveal link-table xref-fig">1</a> according to the KEGG database</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>hsa:22862 </th><th>mmu:319448 </th><th>gga:418863 </th><th>xla:446899 </th></tr></thead><tbody><tr><td>hsa:22862 </td><td>100.0 </td><td>91.1 </td><td>80.2 </td><td>53.3 </td></tr><tr><td>mmu:319448 </td><td>91.1 </td><td>100.0 </td><td>68.5 </td><td>52.3 </td></tr><tr><td>gga:418863 </td><td>80.2 </td><td>68.5 </td><td>100.0 </td><td>53.1 </td></tr><tr><td>xla:446899 </td><td>53.3 </td><td>52.3 </td><td>53.1 </td><td>100.0 </td></tr></tbody></table></div><div content-id="DSR023TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>hsa:22862 </th><th>mmu:319448 </th><th>gga:418863 </th><th>xla:446899 </th></tr></thead><tbody><tr><td>hsa:22862 </td><td>100.0 </td><td>91.1 </td><td>80.2 </td><td>53.3 </td></tr><tr><td>mmu:319448 </td><td>91.1 </td><td>100.0 </td><td>68.5 </td><td>52.3 </td></tr><tr><td>gga:418863 </td><td>80.2 </td><td>68.5 </td><td>100.0 </td><td>53.1 </td></tr><tr><td>xla:446899 </td><td>53.3 </td><td>52.3 </td><td>53.1 </td><td>100.0 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6726662" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR023TB2" class="table-wrap-title" data-id="DSR023TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Identity percentage between the protein pairs from Table <a reveal-id="DSR023TB1" data-open="DSR023TB1" class="link link-reveal link-table xref-fig">1</a> according to the KEGG database</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>hsa:22862 </th><th>mmu:319448 </th><th>gga:418863 </th><th>xla:446899 </th></tr></thead><tbody><tr><td>hsa:22862 </td><td>100.0 </td><td>91.1 </td><td>80.2 </td><td>53.3 </td></tr><tr><td>mmu:319448 </td><td>91.1 </td><td>100.0 </td><td>68.5 </td><td>52.3 </td></tr><tr><td>gga:418863 </td><td>80.2 </td><td>68.5 </td><td>100.0 </td><td>53.1 </td></tr><tr><td>xla:446899 </td><td>53.3 </td><td>52.3 </td><td>53.1 </td><td>100.0 </td></tr></tbody></table></div><div content-id="DSR023TB2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>hsa:22862 </th><th>mmu:319448 </th><th>gga:418863 </th><th>xla:446899 </th></tr></thead><tbody><tr><td>hsa:22862 </td><td>100.0 </td><td>91.1 </td><td>80.2 </td><td>53.3 </td></tr><tr><td>mmu:319448 </td><td>91.1 </td><td>100.0 </td><td>68.5 </td><td>52.3 </td></tr><tr><td>gga:418863 </td><td>80.2 </td><td>68.5 </td><td>100.0 </td><td>53.1 </td></tr><tr><td>xla:446899 </td><td>53.3 </td><td>52.3 </td><td>53.1 </td><td>100.0 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6726662" target="_blank">View Large</a></div></div>                        <a id="6726664" scrollto-destination="6726664"></a>
<div id="DSR023F7" data-id="DSR023F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02307.jpeg?Expires=2147483647&Signature=oB-or5V~dAOIc0z6yrq4qlFMqHlXc1U7uWyuxorxptm2WjFYC36e~dL-OifqaoUqvmrAkRbfjY5-ZHSotN4HXMdb5ArYDpVMTmAJS305CkcLn5r615iEswy4vLiE3dJc3SczeinjhNnW7PgC1M5h-3g7g8asPLsVyUCOQNgvL-6ByC5206UyACQPch3DIRDlqYy4dPIb73CdI5gx2HAgoj7kFeXASvylVa9yRdEJX1SdksSg~BRNy0Jce1niQO03S-n1otHrAuHg3qjHuOqAvM3UEmgU8ro2lH4OInem0GDWwXmM9lMnYEMxjQxXjesRa3O3FqfCy8JWtX7jI4EwiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of latent profility of 291 bp in the gene coding region of fibronectin type III domain-containing protein (KEGG GENES hsa:22862). (a) The characteristic spectrum. (b) The D1 spectrum of deviation from homogeneity (1-profility). (c) The statistical reconstruction of the D1 spectrum carried out assuming the presence of 291-profility in the sequence." path-from-xml="dsr02307.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726664/dsr02307.jpeg" path-from-xml="dsr02307.jpeg" target="_blank">View large</a><a section="6726664" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02307.jpeg?Expires=2147483647&Signature=uI7-PsvspED-XiaPl7vKk9b3TFDZpkK6yHCA3svLXbbdPhJwTjYNHssaXyezZU9w0wSrIfTXj7-ZqnVM2wRwU7UAs6L7qIdojPVZ2UUxIqj4QdC5n-BOIeAUFaIC~rs5WKWm8bkcVZmZaRaqCFE1oUUZU5rs83-L3ftMJ6GIjT82z~zk3utsc8FO43TXTsJ~Z2HL5uDfbQDMaq-oMrJdmn3~Duf-OItWfriCJJAYzrTGBr-CAZpHqsXORxCa08bLK7ZPeRdCFOwW1TKYdNFS1AgMR2K9irISGHtI9og9eLegmcx6bNLZWj85iUacXqn29s8REMx~TeFCoPyurL3yKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726664&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02307.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification of latent profility of 291 bp in the gene coding region of fibronectin type III domain-containing protein (KEGG GENES hsa:22862). (a) The characteristic spectrum. (b) The D1 spectrum of deviation from homogeneity (1-profility). (c) The statistical reconstruction of the D1 spectrum carried out assuming the presence of 291-profility in the sequence.</p></div></div><div content-id="DSR023F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02307.jpeg?Expires=2147483647&Signature=oB-or5V~dAOIc0z6yrq4qlFMqHlXc1U7uWyuxorxptm2WjFYC36e~dL-OifqaoUqvmrAkRbfjY5-ZHSotN4HXMdb5ArYDpVMTmAJS305CkcLn5r615iEswy4vLiE3dJc3SczeinjhNnW7PgC1M5h-3g7g8asPLsVyUCOQNgvL-6ByC5206UyACQPch3DIRDlqYy4dPIb73CdI5gx2HAgoj7kFeXASvylVa9yRdEJX1SdksSg~BRNy0Jce1niQO03S-n1otHrAuHg3qjHuOqAvM3UEmgU8ro2lH4OInem0GDWwXmM9lMnYEMxjQxXjesRa3O3FqfCy8JWtX7jI4EwiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of latent profility of 291 bp in the gene coding region of fibronectin type III domain-containing protein (KEGG GENES hsa:22862). (a) The characteristic spectrum. (b) The D1 spectrum of deviation from homogeneity (1-profility). (c) The statistical reconstruction of the D1 spectrum carried out assuming the presence of 291-profility in the sequence." path-from-xml="dsr02307.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726664/dsr02307.jpeg" path-from-xml="dsr02307.jpeg" target="_blank">View large</a><a section="6726664" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02307.jpeg?Expires=2147483647&Signature=uI7-PsvspED-XiaPl7vKk9b3TFDZpkK6yHCA3svLXbbdPhJwTjYNHssaXyezZU9w0wSrIfTXj7-ZqnVM2wRwU7UAs6L7qIdojPVZ2UUxIqj4QdC5n-BOIeAUFaIC~rs5WKWm8bkcVZmZaRaqCFE1oUUZU5rs83-L3ftMJ6GIjT82z~zk3utsc8FO43TXTsJ~Z2HL5uDfbQDMaq-oMrJdmn3~Duf-OItWfriCJJAYzrTGBr-CAZpHqsXORxCa08bLK7ZPeRdCFOwW1TKYdNFS1AgMR2K9irISGHtI9og9eLegmcx6bNLZWj85iUacXqn29s8REMx~TeFCoPyurL3yKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726664&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02307.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification of latent profility of 291 bp in the gene coding region of fibronectin type III domain-containing protein (KEGG GENES hsa:22862). (a) The characteristic spectrum. (b) The D1 spectrum of deviation from homogeneity (1-profility). (c) The statistical reconstruction of the D1 spectrum carried out assuming the presence of 291-profility in the sequence.</p></div></div>                            <h3 scrollto-destination=6726667 id="6726667" class="section-title" data-legacy-id=s3b>3.2. Manifestation of levels of organization of genetic information encoding</h3>
<p>Regularity of the peaks at 3 bp is observed in the characteristic spectra of the coding regions (Figs <a reveal-id="DSR023F2" data-open="DSR023F2" class="link link-reveal link-table xref-fig">2</a>a–c, <a reveal-id="DSR023F7" data-open="DSR023F7" class="link link-reveal link-table xref-fig">7</a>a and <a reveal-id="DSR023F8" data-open="DSR023F8" class="link link-reveal link-table xref-fig">8</a>). Thus, an encoding organization level caused by the genetic triplet code is manifested. This regularity of a characteristic spectrum is called further as 3-regular heterogeneity, or 3-regularity. As in the Fourier spectra, 3-regular heterogeneity of a characteristic spectrum can be observed in the absence of latent periodicity of 3 bp (see Fig. <a reveal-id="DSR023F2" data-open="DSR023F2" class="link link-reveal link-table xref-fig">2</a>, for example). If 3-regularity exists in the characteristic spectrum of a coding region, then revealing latent profility (different from 3-profility) in the spectrum determines the second level of the encoding organization. For example, Fig. <a reveal-id="DSR023F2" data-open="DSR023F2" class="link link-reveal link-table xref-fig">2</a>a–c shows the characteristic spectra in which, as discussed above (Figs <a reveal-id="DSR023F3" data-open="DSR023F3" class="link link-reveal link-table xref-fig">3</a> and <a reveal-id="DSR023F4" data-open="DSR023F4" class="link link-reveal link-table xref-fig">4</a>), latent 33-profility is revealed against the background of 3-regularity. </p>                        <a id="6726672" scrollto-destination="6726672"></a>
<div id="DSR023F8" data-id="DSR023F8" class="fig fig-section"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02308.jpeg?Expires=2147483647&Signature=1OwQ9d3F9HHefUZjtAO0KG8cztpG2ce0oZRzDDVqGq5NCCqtaDe2CTHqbyxAXUu8QWKA~a88KJeI789WtZXYjd2cii2tCS8~uAA6DF7svrqGZ6vD0wsGF56r-v-w5oyz9sUrW4wppKKsbpZe8G0vAScCBsJujCC5H689yuWA-Nu3O5cDYd5soCsKKv2YeAkrovWA5x~GKjZrJWHfVmJeHvX5WqfUImkQlcawjzS9w-1q0vEu60hKy~ZyYrxVuBK7-Mj9d~MScIbPxC0AYWBAOztpLqxzpNzPq8MKBCWEf44ztaP3GnuCXMNsE1oGrxBVtUUaw303QWeeses3UgC2Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The characteristic spectrum for the coding region of cya gene from bacterium B. pertussis (GenBank Y00545, region: 981–6101 bp)." path-from-xml="dsr02308.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726672/dsr02308.jpeg" path-from-xml="dsr02308.jpeg" target="_blank">View large</a><a section="6726672" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02308.jpeg?Expires=2147483647&Signature=QV11Bd5EMltlLqnME545MEAr-camDzcbcT5ifPApxxrAoVYU1irCYcFxHJN-5tNf-zuID7nTKj6NbzFE6qHb53BYoMFHqWUmIpmKlHK~nH4YUW9sv~7EYed4df5spfjN~R78Du~KoP7ekzMMl3v-BZLWF17lfDw-kJguDGeQlqKSv-4j829Ud9RKxZZndPLhqf0~hORQYaRN52nMOnTPgAiAM1kMn-yNFy-dyPrkg1UzMcwbAn7NnzuiwxYvMQeAhJKveAwV9OqHODSggSgn8MgnH6jI5iJnQY-mtu6-UA8iUgosRO9ih3ljt3xNNcLtur4o1~7XWGLdxjkueLBfiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726672&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02308.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The characteristic spectrum for the coding region of <em>cya</em> gene from bacterium <em>B. pertussis</em> (GenBank Y00545, region: 981–6101 bp).</p></div></div><div content-id="DSR023F8" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 8.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02308.jpeg?Expires=2147483647&Signature=1OwQ9d3F9HHefUZjtAO0KG8cztpG2ce0oZRzDDVqGq5NCCqtaDe2CTHqbyxAXUu8QWKA~a88KJeI789WtZXYjd2cii2tCS8~uAA6DF7svrqGZ6vD0wsGF56r-v-w5oyz9sUrW4wppKKsbpZe8G0vAScCBsJujCC5H689yuWA-Nu3O5cDYd5soCsKKv2YeAkrovWA5x~GKjZrJWHfVmJeHvX5WqfUImkQlcawjzS9w-1q0vEu60hKy~ZyYrxVuBK7-Mj9d~MScIbPxC0AYWBAOztpLqxzpNzPq8MKBCWEf44ztaP3GnuCXMNsE1oGrxBVtUUaw303QWeeses3UgC2Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The characteristic spectrum for the coding region of cya gene from bacterium B. pertussis (GenBank Y00545, region: 981–6101 bp)." path-from-xml="dsr02308.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726672/dsr02308.jpeg" path-from-xml="dsr02308.jpeg" target="_blank">View large</a><a section="6726672" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02308.jpeg?Expires=2147483647&Signature=QV11Bd5EMltlLqnME545MEAr-camDzcbcT5ifPApxxrAoVYU1irCYcFxHJN-5tNf-zuID7nTKj6NbzFE6qHb53BYoMFHqWUmIpmKlHK~nH4YUW9sv~7EYed4df5spfjN~R78Du~KoP7ekzMMl3v-BZLWF17lfDw-kJguDGeQlqKSv-4j829Ud9RKxZZndPLhqf0~hORQYaRN52nMOnTPgAiAM1kMn-yNFy-dyPrkg1UzMcwbAn7NnzuiwxYvMQeAhJKveAwV9OqHODSggSgn8MgnH6jI5iJnQY-mtu6-UA8iUgosRO9ih3ljt3xNNcLtur4o1~7XWGLdxjkueLBfiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726672&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02308.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The characteristic spectrum for the coding region of <em>cya</em> gene from bacterium <em>B. pertussis</em> (GenBank Y00545, region: 981–6101 bp).</p></div></div><p>An investigation of different levels in the organization of genetic information encoding has been carried out on a sample of 18 140 human coding regions (CDS) from the KEGG GENES-54.1 database (<a class="link link-uri" href="http://www.genome.jp/kegg/genes.html">http://www.genome.jp/kegg/genes.html</a>). Only those coding regions were chosen for which there is experimental evidence of protein translation. Open reading frames, hypothetical proteins, tRNA and rRNA, and genes assumed by their sequences to show similarity to other known genes were excluded from the sample. It appears that 3-regularity in the characteristic spectra is fixed for 93% of the sample (16 786 CDS). Against the background of 3-regular heterogeneity, latent 3-profility is revealed for 62% (11 200 CDS) of the original sample. For the 11% of the sample (1953 CDS), two levels of organization of the encoding are manifested (3-regular heterogeneity and the latent profility different from 3-profility).</p><p>Taking into account the inaccuracy of the statistical methods, the following conclusions can be made from the results obtained. Owing to amino acids triplet encoding, the 3-regular heterogeneity of the characteristic spectra is generic for human genes. However, such regularity is not due to latent periodicity of 3 bp. Thus, it is essential to differentiate between the phenomena of regular heterogeneity and latent periodicity in the genetic sequences. In order to verify the existence of latent periodicity of some type, it is necessary to observe a pattern inducing the periodicity.</p>                            <h3 scrollto-destination=6726677 id="6726677" class="section-title" data-legacy-id=s3c>3.3. Local profile periodicity</h3>
<p>In the coding regions, the manifestation of tha local two-level organization of genetic information encoding is possible. Thus, for the whole coding region, 3-regular heterogeneity only is observed (see Fig. <a reveal-id="DSR023F8" data-open="DSR023F8" class="link link-reveal link-table xref-fig">8</a>, for example). Regions with local profile periodicity (local profility) can be revealed by scanning the sequence with a small window. For example, regarding local profility in the coding region of the <em>cya</em> gene from the bacterium <em>Bordetella pertussis</em> (GenBank Y00545, 981–6101 bp), in the entire coding region, 3-regular heterogeneity only is revealed (Fig. <a reveal-id="DSR023F8" data-open="DSR023F8" class="link link-reveal link-table xref-fig">8</a>), and there is no latent profility. Latent profile periodicity is observed solely in local areas of the coding regions. Three local areas of latent profile periodicity with a period of 27 bp (Figs <a reveal-id="DSR023F9" data-open="DSR023F9" class="link link-reveal link-table xref-fig">9</a>a–c and <a reveal-id="DSR023F10" data-open="DSR023F10" class="link link-reveal link-table xref-fig">10</a>) can be distinguished in the coding region of the <em>cya</em> gene of bacterium <em>B. pertussis</em>. </p>                        <a id="6726679" scrollto-destination="6726679"></a>
<div id="DSR023F9" data-id="DSR023F9" class="fig fig-section"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02309.jpeg?Expires=2147483647&Signature=kgfCWSyVBplV-sIqxqzlV7Pdb2L-JySYqH825fJduoHTmezKCCHNwjhjL6yrObeguyR~dtQw5WBvDD1GCAIfEqC6r-uBloThlwDLVs88v~kJ6t74PoK1GazOfpb-271fzhiGoSoczjJKsPVwpc03HlBJ9WWZkseQQT80ijw46o5lsfB26Q2honGJgNgf0gghkcCbo26Hg6mEd2zn~0EFP153-Cl3DS-V73o8dRU-dKtRIHEc2a8IC8fgCHj4zdNtcODM0fUbSEcUuNTQxCOVHXIrLsjJecOtMQxBYFDQatQYZryikBKTPq2ijes~gVERV9gPW27fN2jSuCORdGCEaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characteristic spectra for the three local areas of the coding region of cya gene from bacterium B. pertussis (GenBank Y00545, region: 981–6101 bp). (a) Local area 4020–4181 bp. (b) Local area 4443–5036 bp. (c) Local area 5211–5840 bp. (d) Fourier spectrum for the local area (c)." path-from-xml="dsr02309.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726679/dsr02309.jpeg" path-from-xml="dsr02309.jpeg" target="_blank">View large</a><a section="6726679" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02309.jpeg?Expires=2147483647&Signature=Vi~ZyUYLDbwJlYH3fTYyEwWfzeIFPasxx5zQvzxS2ekKGuA7T2iGPiahKWJ1CohG~zHOlk5RMi6I89-HvlAPc0sY1Vp2eiBJ31PjuRDuIakMdVy3PGThWRVYzKuqAKHJaePywSEDrth7hOZvmsa~azhzWO5rM-M4-7AQ0BfFsSOwlDXUuQ0EhFjSYepV7x~w85bgSs7NJLZFynBHLSCRlt4682gI4apWhaRuZVgEDRijqHbRIDeugD-gbYmXfhsL3uVaCyDLnadZNqB-h9G00PxZQMSXhsk4hXTKX5kRHZK1PHW~ugpt7UhhcT4wHwg8pOOBXgpZT74WOszzONwWIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726679&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02309.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characteristic spectra for the three local areas of the coding region of <em>cya</em> gene from bacterium <em>B. pertussis</em> (GenBank Y00545, region: 981–6101 bp). (a) Local area 4020–4181 bp. (b) Local area 4443–5036 bp. (c) Local area 5211–5840 bp. (d) Fourier spectrum for the local area (c).</p></div></div><div content-id="DSR023F9" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 9.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02309.jpeg?Expires=2147483647&Signature=kgfCWSyVBplV-sIqxqzlV7Pdb2L-JySYqH825fJduoHTmezKCCHNwjhjL6yrObeguyR~dtQw5WBvDD1GCAIfEqC6r-uBloThlwDLVs88v~kJ6t74PoK1GazOfpb-271fzhiGoSoczjJKsPVwpc03HlBJ9WWZkseQQT80ijw46o5lsfB26Q2honGJgNgf0gghkcCbo26Hg6mEd2zn~0EFP153-Cl3DS-V73o8dRU-dKtRIHEc2a8IC8fgCHj4zdNtcODM0fUbSEcUuNTQxCOVHXIrLsjJecOtMQxBYFDQatQYZryikBKTPq2ijes~gVERV9gPW27fN2jSuCORdGCEaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Characteristic spectra for the three local areas of the coding region of cya gene from bacterium B. pertussis (GenBank Y00545, region: 981–6101 bp). (a) Local area 4020–4181 bp. (b) Local area 4443–5036 bp. (c) Local area 5211–5840 bp. (d) Fourier spectrum for the local area (c)." path-from-xml="dsr02309.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726679/dsr02309.jpeg" path-from-xml="dsr02309.jpeg" target="_blank">View large</a><a section="6726679" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02309.jpeg?Expires=2147483647&Signature=Vi~ZyUYLDbwJlYH3fTYyEwWfzeIFPasxx5zQvzxS2ekKGuA7T2iGPiahKWJ1CohG~zHOlk5RMi6I89-HvlAPc0sY1Vp2eiBJ31PjuRDuIakMdVy3PGThWRVYzKuqAKHJaePywSEDrth7hOZvmsa~azhzWO5rM-M4-7AQ0BfFsSOwlDXUuQ0EhFjSYepV7x~w85bgSs7NJLZFynBHLSCRlt4682gI4apWhaRuZVgEDRijqHbRIDeugD-gbYmXfhsL3uVaCyDLnadZNqB-h9G00PxZQMSXhsk4hXTKX5kRHZK1PHW~ugpt7UhhcT4wHwg8pOOBXgpZT74WOszzONwWIA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726679&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02309.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Characteristic spectra for the three local areas of the coding region of <em>cya</em> gene from bacterium <em>B. pertussis</em> (GenBank Y00545, region: 981–6101 bp). (a) Local area 4020–4181 bp. (b) Local area 4443–5036 bp. (c) Local area 5211–5840 bp. (d) Fourier spectrum for the local area (c).</p></div></div>                        <a id="6726680" scrollto-destination="6726680"></a>
<div id="DSR023F10" data-id="DSR023F10" class="fig fig-section"><div class="label fig-label">Figure 10.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02310.jpeg?Expires=2147483647&Signature=JV1rVf6xdy4FpACrIp-v-QcBgk9redhQK0r7x0ibEQM7WtOU2PSG9enHM3WvHstwBx1jEbbTS9YqqsTorVRETC6JzgaGUEbLU0Tlu-a74-TltI03wJZm8umasIVRok5jhmqiRvRXl3Yv37goofUUgUNg0y5Fx5h2VXCtZLPbzW83z-verncUBKAY97pRImdwgwH3V1hecjIzEogee7uPA~UGz31UjSTlJwEMy4VWr2M-lqfvz9YYimZOK0npCoSbZLfByEaJqlqJg07SkfiGtNT8ChpqcN4XAYdKDxwnD2xgQue3Gla~Ucluse3vLb3cmjV1lWzWHsdBLvZybs5hbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of latent 27-profility in a local area of the coding region of the cya gene from bacterium B. pertussis (GenBank Y00545, 5211–5840 bp). (a, c and d) Spectra of deviation from the 1-, 27- and 3-profility (respectively). (b) A spectrum of theoretical reconstruction of the D1 spectrum in (a) assuming the presence of 27-profility in the local area." path-from-xml="dsr02310.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726680/dsr02310.jpeg" path-from-xml="dsr02310.jpeg" target="_blank">View large</a><a section="6726680" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02310.jpeg?Expires=2147483647&Signature=oZOO4hXgwoGfzXHDmFSANs51Hg4fNAILV~xqeNn0JiSou9JFD~gHPe5WpMnzourLpF6m4dSTn59Jon7qHaFBEdRp1aAbel6DYnQmOK3n8tDpc78V1evrB4dEZmdbupp7v~3YIf3X8lC4UGynTSSd62FPqw94w7h9Dz476n-zyLeEygM0knDqEbq8Sta4W6d7earE~gdWdq-Os6AudDwhCWY3F9rYJjXPmr29Zj9p4BX5VaCPdepSrXIwtT-0pYde4kt-sibxR1pCNi63Ejn89VXB05BPqmZYrYJ4oWP1SmAAvSeud8zPFrz8vjbpcQEAgxocXVng9K45aL0pTfVm8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726680&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02310.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification of latent 27-profility in a local area of the coding region of the <em>cya</em> gene from bacterium <em>B. pertussis</em> (GenBank Y00545, 5211–5840 bp). (a, c and d) Spectra of deviation from the 1-, 27- and 3-profility (respectively). (b) A spectrum of theoretical reconstruction of the D1 spectrum in (a) assuming the presence of 27-profility in the local area.</p></div></div><div content-id="DSR023F10" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 10.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02310.jpeg?Expires=2147483647&Signature=JV1rVf6xdy4FpACrIp-v-QcBgk9redhQK0r7x0ibEQM7WtOU2PSG9enHM3WvHstwBx1jEbbTS9YqqsTorVRETC6JzgaGUEbLU0Tlu-a74-TltI03wJZm8umasIVRok5jhmqiRvRXl3Yv37goofUUgUNg0y5Fx5h2VXCtZLPbzW83z-verncUBKAY97pRImdwgwH3V1hecjIzEogee7uPA~UGz31UjSTlJwEMy4VWr2M-lqfvz9YYimZOK0npCoSbZLfByEaJqlqJg07SkfiGtNT8ChpqcN4XAYdKDxwnD2xgQue3Gla~Ucluse3vLb3cmjV1lWzWHsdBLvZybs5hbA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Identification of latent 27-profility in a local area of the coding region of the cya gene from bacterium B. pertussis (GenBank Y00545, 5211–5840 bp). (a, c and d) Spectra of deviation from the 1-, 27- and 3-profility (respectively). (b) A spectrum of theoretical reconstruction of the D1 spectrum in (a) assuming the presence of 27-profility in the local area." path-from-xml="dsr02310.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726680/dsr02310.jpeg" path-from-xml="dsr02310.jpeg" target="_blank">View large</a><a section="6726680" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02310.jpeg?Expires=2147483647&Signature=oZOO4hXgwoGfzXHDmFSANs51Hg4fNAILV~xqeNn0JiSou9JFD~gHPe5WpMnzourLpF6m4dSTn59Jon7qHaFBEdRp1aAbel6DYnQmOK3n8tDpc78V1evrB4dEZmdbupp7v~3YIf3X8lC4UGynTSSd62FPqw94w7h9Dz476n-zyLeEygM0knDqEbq8Sta4W6d7earE~gdWdq-Os6AudDwhCWY3F9rYJjXPmr29Zj9p4BX5VaCPdepSrXIwtT-0pYde4kt-sibxR1pCNi63Ejn89VXB05BPqmZYrYJ4oWP1SmAAvSeud8zPFrz8vjbpcQEAgxocXVng9K45aL0pTfVm8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726680&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02310.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Identification of latent 27-profility in a local area of the coding region of the <em>cya</em> gene from bacterium <em>B. pertussis</em> (GenBank Y00545, 5211–5840 bp). (a, c and d) Spectra of deviation from the 1-, 27- and 3-profility (respectively). (b) A spectrum of theoretical reconstruction of the D1 spectrum in (a) assuming the presence of 27-profility in the local area.</p></div></div><p>Let us note again that the first level of encoding is manifested in the 3-regularity of characteristic spectrum (Fig. <a reveal-id="DSR023F9" data-open="DSR023F9" class="link link-reveal link-table xref-fig">9</a>a–c) and in existence of dominant peak at frequency equal to 0.33 in the Fourier spectrum (see, for example, Fig. <a reveal-id="DSR023F9" data-open="DSR023F9" class="link link-reveal link-table xref-fig">9</a>d). The second level of encoding organization—the 27-profile periodicity—in the local areas of the <em>cya</em> gene is pointed at by the dominant peaks of characteristic spectra. Such a profile periodicity is proved by reconstruction of the spectrum of deviation from homogeneity in every local area (see, for example, Fig. <a reveal-id="DSR023F10" data-open="DSR023F10" class="link link-reveal link-table xref-fig">10</a>a and b). In contrast to the characteristic spectra, the second level of encoding organization is not manifested in the Fourier spectra (see, for example, Fig. <a reveal-id="DSR023F9" data-open="DSR023F9" class="link link-reveal link-table xref-fig">9</a>d).</p><p>The <em>cya</em> gene encodes bifunctional hemolysin/adenylate cyclase (UniProtKB P15318) in which the areas corresponding to the gene local 27-profility hold the hemolysin-type calcium-binding sites. These sites have a periodic structure of 18 amino acid residues (Fig. <a reveal-id="DSR023F11" data-open="DSR023F11" class="link link-reveal link-table xref-fig">11</a>) corresponding to 54 bp (2 × 27 bp) in the gene. </p>                        <a id="6726683" scrollto-destination="6726683"></a>
<div id="DSR023F11" data-id="DSR023F11" class="fig fig-section"><div class="label fig-label">Figure 11.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02311.jpeg?Expires=2147483647&Signature=UHmncLlvvaqwfdW6~faVBkaBm5yPnqNZSqe090y6F5imEU3cInhN3vQyIGIC8ucUX7N6PY5NV9606Hl3l9MZaINVI7ZVWtXiZlmbEqDHk7jobuzG0gA-Gr3oJSLtPMYOs5nzgyDWed6CY05EyMFm7KHA7SvHUyhh3EE8zTvB~T-NXJeBrGX89U9zHwuNyLFN9Mr1k81VEN1MUfZTTBZXVZS--jXL6Hz3RttC1eV8mN6XXXGgWEQyjTYb8Fw7KdwLcBBgWPjfyB6WSPVSXk5KEC16SzIFdX6I6dkGS2K2wdupvzzZ2-DjdVCaScKCyGIFqx2O2i7FXU9faEOaQmvJCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of bifunctional hemolysin/adenylate cyclase (UniProtKB P15318, 1–1706 amino acids) encoded by the cya gene of bacterium B. pertussis (GenBank Y00545, region: 981–6101 bp). Each grey vertical bar denotes a hemolysin-type calcium-binding site of 18 аmino аcids. The regions A (1014–1067 amino acids), B (1155–1352 amino acids) and C (1411–1620 amino acids) correspond to the three areas of the local 27-profility (4020–4181, 4443–5036 and 5211–5840 bp, respectively) revealed in the cya gene (Fig. 9)." path-from-xml="dsr02311.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726683/dsr02311.jpeg" path-from-xml="dsr02311.jpeg" target="_blank">View large</a><a section="6726683" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02311.jpeg?Expires=2147483647&Signature=DtMQh0YgRKa2pkEoiuGMt1H7B2f5feJeTLMYabXimCvAMd-~mPu-eM9udhlz6G6BaqTcO05DsC9Sy8Sl0y0fQAUsOr1z41EvIShwCjR0~Zlv0kJxo9VW7CfGOqL6cN5TxG8HsOme85D8Wn1pSOwTug1kGsA4fTLNo2szg-4WcAmszE9xK4vSxYdinXtbzlNCBr7fC~MlD9WyU9~59cLl00~KF1Y5gOLYdMLDfbCs7yDLnVspGu5q6XIJ3ccLQM5y-LTHjSetS9oST97ZgupvH~hgM2oUoFM163XsV3dBFp~LQiX3IOZt6oqrsHlT-JQTs8h7Gp6LmizZFgLqa3-yUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726683&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02311.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of bifunctional hemolysin/adenylate cyclase (UniProtKB P15318, 1–1706 amino acids) encoded by the cya gene of bacterium <em>B. pertussis</em> (GenBank Y00545, region: 981–6101 bp). Each grey vertical bar denotes a hemolysin-type calcium-binding site of 18 аmino аcids. The regions A (1014–1067 amino acids), B (1155–1352 amino acids) and C (1411–1620 amino acids) correspond to the three areas of the local 27-profility (4020–4181, 4443–5036 and 5211–5840 bp, respectively) revealed in the <em>cya</em> gene (Fig. <a reveal-id="DSR023F9" data-open="DSR023F9" class="link link-reveal link-table xref-fig">9</a>).</p></div></div><div content-id="DSR023F11" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 11.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/m_dsr02311.jpeg?Expires=2147483647&Signature=UHmncLlvvaqwfdW6~faVBkaBm5yPnqNZSqe090y6F5imEU3cInhN3vQyIGIC8ucUX7N6PY5NV9606Hl3l9MZaINVI7ZVWtXiZlmbEqDHk7jobuzG0gA-Gr3oJSLtPMYOs5nzgyDWed6CY05EyMFm7KHA7SvHUyhh3EE8zTvB~T-NXJeBrGX89U9zHwuNyLFN9Mr1k81VEN1MUfZTTBZXVZS--jXL6Hz3RttC1eV8mN6XXXGgWEQyjTYb8Fw7KdwLcBBgWPjfyB6WSPVSXk5KEC16SzIFdX6I6dkGS2K2wdupvzzZ2-DjdVCaScKCyGIFqx2O2i7FXU9faEOaQmvJCg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of bifunctional hemolysin/adenylate cyclase (UniProtKB P15318, 1–1706 amino acids) encoded by the cya gene of bacterium B. pertussis (GenBank Y00545, region: 981–6101 bp). Each grey vertical bar denotes a hemolysin-type calcium-binding site of 18 аmino аcids. The regions A (1014–1067 amino acids), B (1155–1352 amino acids) and C (1411–1620 amino acids) correspond to the three areas of the local 27-profility (4020–4181, 4443–5036 and 5211–5840 bp, respectively) revealed in the cya gene (Fig. 9)." path-from-xml="dsr02311.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6726683/dsr02311.jpeg" path-from-xml="dsr02311.jpeg" target="_blank">View large</a><a section="6726683" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr023/2/dsr02311.jpeg?Expires=2147483647&Signature=DtMQh0YgRKa2pkEoiuGMt1H7B2f5feJeTLMYabXimCvAMd-~mPu-eM9udhlz6G6BaqTcO05DsC9Sy8Sl0y0fQAUsOr1z41EvIShwCjR0~Zlv0kJxo9VW7CfGOqL6cN5TxG8HsOme85D8Wn1pSOwTug1kGsA4fTLNo2szg-4WcAmszE9xK4vSxYdinXtbzlNCBr7fC~MlD9WyU9~59cLl00~KF1Y5gOLYdMLDfbCs7yDLnVspGu5q6XIJ3ccLQM5y-LTHjSetS9oST97ZgupvH~hgM2oUoFM163XsV3dBFp~LQiX3IOZt6oqrsHlT-JQTs8h7Gp6LmizZFgLqa3-yUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6726683&ar=449021&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02311.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of bifunctional hemolysin/adenylate cyclase (UniProtKB P15318, 1–1706 amino acids) encoded by the cya gene of bacterium <em>B. pertussis</em> (GenBank Y00545, region: 981–6101 bp). Each grey vertical bar denotes a hemolysin-type calcium-binding site of 18 аmino аcids. The regions A (1014–1067 amino acids), B (1155–1352 amino acids) and C (1411–1620 amino acids) correspond to the three areas of the local 27-profility (4020–4181, 4443–5036 and 5211–5840 bp, respectively) revealed in the <em>cya</em> gene (Fig. <a reveal-id="DSR023F9" data-open="DSR023F9" class="link link-reveal link-table xref-fig">9</a>).</p></div></div>                            <h3 scrollto-destination=6726685 id="6726685" class="section-title" data-legacy-id=s3d>3.4. Conclusions</h3>
<p>Methods for identifying a new type of latent periodicity—latent profility in DNA—have been proposed. For DNA coding regions, latent profility enables us to distinguish two levels of organization of genetic information encoding. The first level (the triplet level of encoding), revealed via the Fourier analysis techniques, indicates the phenomenon of regular heterogeneity in the DNA coding regions. The second level of organization in the encoding is due to latent profile periodicity of the DNA sequence. It has been shown that latent profile periodicity in genes of the same family may correlate with the structural features of encoded proteins. Such an effect may manifest in the local areas of coding regions where latent profile periodicity is observed.</p>                            <h2 scrollto-destination=6726690 id="6726690" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR023C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lobzin</div> <div class="given-names">V.V.</div></div>,  <div class="name"><div class="surname">Chechetkin</div> <div class="given-names">V.R.</div></div>. </div><div class="article-title">Order and correlations in genomic DNA sequences. The spectral approach</div>, <div class="source">Physics–Uspekhi.</div> , <div class="year">2000</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">55</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Order%20and%20correlations%20in%20genomic%20DNA%20sequences.%20The%20spectral%20approach&amp;author=V.V.%20Lobzin&amp;author=V.R.%20Chechetkin&amp;publication_year=2000&amp;journal=Physics%E2%80%93Uspekhi.&amp;volume=43&amp;pages=55-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1070/PU2000v043n01ABEH000611" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1070%2FPU2000v043n01ABEH000611" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1070%2FPU2000v043n01ABEH000611"> </span></div> </div></div></div></div></div><div content-id="DSR023C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">W.</div></div>. </div><div class="article-title">The study of correlation structures of DNA sequences: a critical review</div>, <div class="source">Comput. Chem.</div> , <div class="year">1997</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">257</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20study%20of%20correlation%20structures%20of%20DNA%20sequences%3A%20a%20critical%20review&amp;author=W.%20Li&amp;publication_year=1997&amp;journal=Comput.%20Chem.&amp;volume=21&amp;pages=257-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0097-8485(97)00022-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0097-8485(97)00022-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0097-8485(97)00022-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9415988" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shepherd</div> <div class="given-names">J.C.W.</div></div>. </div><div class="article-title">Method to determine the reading frame of a protein from the purine/pyrimidine genome sequence and its possible evolutionary justification</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1981</div>, vol. <div class="volume">78</div> (pg. <div class="fpage">1596</div>-<div class="lpage">600</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Method%20to%20determine%20the%20reading%20frame%20of%20a%20protein%20from%20the%20purine%2Fpyrimidine%20genome%20sequence%20and%20its%20possible%20evolutionary%20justification&amp;author=J.C.W.%20Shepherd&amp;publication_year=1981&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=78&amp;pages=1596-600" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.78.3.1596" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.78.3.1596" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.78.3.1596"> </span></div> </div></div></div></div></div><div content-id="DSR023C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benson</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Tandem repeats finder: a program to analyze DNA sequences</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1999</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">573</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20finder%3A%20a%20program%20to%20analyze%20DNA%20sequences&amp;author=G.%20Benson&amp;publication_year=1999&amp;journal=Nucleic%20Acids%20Res.&amp;volume=27&amp;pages=573-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/27.2.573" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F27.2.573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F27.2.573"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9862982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Metzgar</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Bytof</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wills</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Selection against frameshift mutations limits microsatellite expansion in coding DNA</div>, <div class="source">Genome Res.</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">72</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20against%20frameshift%20mutations%20limits%20microsatellite%20expansion%20in%20coding%20DNA&amp;author=D.%20Metzgar&amp;author=J.%20Bytof&amp;author=C.%20Wills&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=72-80" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10645952" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Borstnik</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Pumpernik</div> <div class="given-names">D.</div></div>. </div><div class="article-title">Tandem repeats in protein coding regions of primate genes</div>, <div class="source">Genome Res.</div> , <div class="year">2002</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">909</div>-<div class="lpage">15</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tandem%20repeats%20in%20protein%20coding%20regions%20of%20primate%20genes&amp;author=B.%20Borstnik&amp;author=D.%20Pumpernik&amp;publication_year=2002&amp;journal=Genome%20Res.&amp;volume=12&amp;pages=909-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.138802" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.138802" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.138802"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12045144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Trifonov</div> <div class="given-names">E.V.</div></div>. </div><div class="article-title">Translation framing code and frame-monitoring mechanism as suggested by the analysis of mRNA and 16S RNA nucleotide sequences</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1987</div>, vol. <div class="volume">194</div> (pg. <div class="fpage">643</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translation%20framing%20code%20and%20frame-monitoring%20mechanism%20as%20suggested%20by%20the%20analysis%20of%20mRNA%20and%2016S%20RNA%20nucleotide%20sequences&amp;author=E.V.%20Trifonov&amp;publication_year=1987&amp;journal=J.%20Mol.%20Biol.&amp;volume=194&amp;pages=643-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(87)90241-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(87)90241-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(87)90241-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2443708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tsonis</div> <div class="given-names">A.A.</div></div>,  <div class="name"><div class="surname">Elsner</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Tsonis</div> <div class="given-names">P.A.</div></div>. </div><div class="article-title">Periodicity in DNA coding sequences: implications in gene evolutions</div>, <div class="source">J. Theor. Biol.</div> , <div class="year">1991</div>, vol. <div class="volume">151</div> (pg. <div class="fpage">323</div>-<div class="lpage">31</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Periodicity%20in%20DNA%20coding%20sequences%3A%20implications%20in%20gene%20evolutions&amp;author=A.A.%20Tsonis&amp;author=J.B.%20Elsner&amp;author=P.A.%20Tsonis&amp;publication_year=1991&amp;journal=J.%20Theor.%20Biol.&amp;volume=151&amp;pages=323-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-5193(05)80381-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-5193(05)80381-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-5193(05)80381-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1943144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gutierrez</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Oliver</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Marin</div> <div class="given-names">A.</div></div>. </div><div class="article-title">On the origin of the periodicity of three in protein coding DNA sequences</div>, <div class="source">J. Theor. Biol.</div> , <div class="year">1994</div>, vol. <div class="volume">167</div> (pg. <div class="fpage">413</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=On%20the%20origin%20of%20the%20periodicity%20of%20three%20in%20protein%20coding%20DNA%20sequences&amp;author=G.%20Gutierrez&amp;author=J.%20Oliver&amp;author=A.%20Marin&amp;publication_year=1994&amp;journal=J.%20Theor.%20Biol.&amp;volume=167&amp;pages=413-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jtbi.1994.1080" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjtbi.1994.1080" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjtbi.1994.1080"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8207954" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fickett</div> <div class="given-names">J.W.</div></div>,  <div class="name"><div class="surname">Tung</div> <div class="given-names">C.-S.</div></div>. </div><div class="article-title">Assessment of protein coding measures</div>, <div class="source">Nucleic Acid Res.</div> , <div class="year">1992</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">6441</div>-<div class="lpage">50</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20protein%20coding%20measures&amp;author=J.W.%20Fickett&amp;author=C.-S.%20Tung&amp;publication_year=1992&amp;journal=Nucleic%20Acid%20Res.&amp;volume=20&amp;pages=6441-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/20.24.6441" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F20.24.6441" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F20.24.6441"> </span></div> </div></div></div></div></div><div content-id="DSR023C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Silverman</div> <div class="given-names">B.D.</div></div>,  <div class="name"><div class="surname">Linsker</div> <div class="given-names">R.</div></div>. </div><div class="article-title">A measure of DNA periodicity</div>, <div class="source">J. Theor. Biol.</div> , <div class="year">1986</div>, vol. <div class="volume">118</div> (pg. <div class="fpage">295</div>-<div class="lpage">300</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20measure%20of%20DNA%20periodicity&amp;author=B.D.%20Silverman&amp;author=R.%20Linsker&amp;publication_year=1986&amp;journal=J.%20Theor.%20Biol.&amp;volume=118&amp;pages=295-300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-5193(86)80060-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-5193(86)80060-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-5193(86)80060-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3713213" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tiwari</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ramachandran</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Bhattacharya</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bhattacharya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ramaswamy</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Prediction of probable genes by Fourier analysis of genomic sequences</div>, <div class="source">Comput. Appl. Biosci.</div> , <div class="year">1997</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">263</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20probable%20genes%20by%20Fourier%20analysis%20of%20genomic%20sequences&amp;author=S.%20Tiwari&amp;author=S.%20Ramachandran&amp;author=A.%20Bhattacharya&amp;author=S.%20Bhattacharya&amp;author=R.%20Ramaswamy&amp;publication_year=1997&amp;journal=Comput.%20Appl.%20Biosci.&amp;volume=13&amp;pages=263-70" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9183531" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Issac</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Kaur</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Raghava</div> <div class="given-names">G.P.S.</div></div>. </div><div class="article-title">Locating probable genes using Fourier transform approach</div>, <div class="source">Bioinformatics</div> , <div class="year">2002</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">196</div>-<div class="lpage">97</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Locating%20probable%20genes%20using%20Fourier%20transform%20approach&amp;author=B.%20Issac&amp;author=H.%20Singh&amp;author=H.%20Kaur&amp;author=G.P.S.%20Raghava&amp;publication_year=2002&amp;journal=Bioinformatics&amp;volume=18&amp;pages=196-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/18.1.196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F18.1.196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F18.1.196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11836230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chaley</div> <div class="given-names">M.B.</div></div>,  <div class="name"><div class="surname">Kutyrkin</div> <div class="given-names">V.A.</div></div>. </div><div class="article-title">Model of perfect tandem repeat with random pattern and empirical homogeneity testing poly-criteria for latent periodicity revelation in biological sequences</div>, <div class="source">Math. Biosci.</div> , <div class="year">2008</div>, vol. <div class="volume">211</div> (pg. <div class="fpage">186</div>-<div class="lpage">204</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Model%20of%20perfect%20tandem%20repeat%20with%20random%20pattern%20and%20empirical%20homogeneity%20testing%20poly-criteria%20for%20latent%20periodicity%20revelation%20in%20biological%20sequences&amp;author=M.B.%20Chaley&amp;author=V.A.%20Kutyrkin&amp;publication_year=2008&amp;journal=Math.%20Biosci.&amp;volume=211&amp;pages=186-204" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mbs.2007.10.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mbs.2007.10.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mbs.2007.10.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18062999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Korotkov</div> <div class="given-names">E.V.</div></div>,  <div class="name"><div class="surname">Korotkova</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Kudryashov</div> <div class="given-names">N.A.</div></div>. </div><div class="article-title">Information decomposition method to analyze symbolical sequences</div>, <div class="source">Phys. Lett. A</div> , <div class="year">2003</div>, vol. <div class="volume">312</div> (pg. <div class="fpage">198</div>-<div class="lpage">210</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Information%20decomposition%20method%20to%20analyze%20symbolical%20sequences&amp;author=E.V.%20Korotkov&amp;author=M.A.%20Korotkova&amp;author=N.A.%20Kudryashov&amp;publication_year=2003&amp;journal=Phys.%20Lett.%20A&amp;volume=312&amp;pages=198-210" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0375-9601(03)00641-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0375-9601(03)00641-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0375-9601(03)00641-8"> </span></div> </div></div></div></div></div><div content-id="DSR023C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gatherer</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">McEwan</div> <div class="given-names">N.R.</div></div>. </div><div class="article-title">Analysis of sequence periodicity in <em>E. coli</em> proteins: empirical investigation of the “duplication and divergence” theory of protein evolution</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2003</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">149</div>-<div class="lpage">58</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20sequence%20periodicity%20in%20E.%20coli%20proteins%3A%20empirical%20investigation%20of%20the%20%E2%80%9Cduplication%20and%20divergence%E2%80%9D%20theory%20of%20protein%20evolution&amp;author=D.%20Gatherer&amp;author=N.R.%20McEwan&amp;publication_year=2003&amp;journal=J.%20Mol.%20Evol.&amp;volume=57&amp;pages=149-58" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00239-002-2462-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00239-002-2462-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00239-002-2462-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14562959" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Larsabal</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Danchin</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Genomes are covered with ubiquitous 11 bp periodic patterns, the class A flexible patterns</div>, <div class="source">BMC Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">206</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomes%20are%20covered%20with%20ubiquitous%2011%20bp%20periodic%20patterns%2C%20the%20class%20A%20flexible%20patterns&amp;author=E.%20Larsabal&amp;author=A.%20Danchin&amp;publication_year=2005&amp;journal=BMC%20Bioinformatics&amp;volume=6&amp;pages=206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-6-206" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-6-206" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-6-206"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16120222" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chaley</div> <div class="given-names">M.B.</div></div>,  <div class="name"><div class="surname">Kutyrkin</div> <div class="given-names">V.A.</div></div>. </div><div class="article-title">Structure of proteins and latent periodicity in their genes</div>, <div class="source">Moscow Univ. Biol. Sci. Bull.</div> , <div class="year">2010</div>, vol. <div class="volume">65</div> (pg. <div class="fpage">133</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20of%20proteins%20and%20latent%20periodicity%20in%20their%20genes&amp;author=M.B.%20Chaley&amp;author=V.A.%20Kutyrkin&amp;publication_year=2010&amp;journal=Moscow%20Univ.%20Biol.%20Sci.%20Bull.&amp;volume=65&amp;pages=133-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3103/S0096392510040012" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3103%2FS0096392510040012" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3103%2FS0096392510040012"> </span></div> </div></div></div></div></div><div content-id="DSR023C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kolpakov</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Kucherov</div> <div class="given-names">G.</div></div>. </div><div class="article-title">mreps: efficient and flexible detection of tandem repeats in DNA</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2003</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">3672</div>-<div class="lpage">78</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=mreps%3A%20efficient%20and%20flexible%20detection%20of%20tandem%20repeats%20in%20DNA&amp;author=R.%20Kolpakov&amp;author=G.%20Kucherov&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=3672-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg617" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg617" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg617"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12824391" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parisi</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Fonzo</div> <div class="given-names">V.D.</div></div>,  <div class="name"><div class="surname">Aluffi-Pentini</div> <div class="given-names">F.</div></div>. </div><div class="article-title">STRING: finding tandem repeats in DNA sequences</div>, <div class="source">Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1733</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=STRING%3A%20finding%20tandem%20repeats%20in%20DNA%20sequences&amp;author=V.%20Parisi&amp;author=V.D.%20Fonzo&amp;author=F.%20Aluffi-Pentini&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=1733-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg268" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg268" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg268"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14512342" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mudunuri</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Nagarajaram</div> <div class="given-names">H.A.</div></div>. </div><div class="article-title">IMEx: imperfect microsatellite extractor</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">1181</div>-<div class="lpage">87</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IMEx%3A%20imperfect%20microsatellite%20extractor&amp;author=S.B.%20Mudunuri&amp;author=H.A.%20Nagarajaram&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1181-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17379689" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR023C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharma</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Issac</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Raghava</div> <div class="given-names">G.P.S.</div></div>,  <div class="name"><div class="surname">Ramaswamy</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Spectral Repeat Finder (SRF): identification of repetitive sequences using Fourier transformation</div>, <div class="source">Bioinformatics</div> , <div class="year">2004</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">1404</div>-<div class="lpage">12</div>)</div></div></div></div><div content-id="DSR023C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rajavashisth</div> <div class="given-names">T.B.</div></div>,  <div class="name"><div class="surname">Kaprein</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Reue</div> <div class="given-names">K.L.</div></div>,  <div class="name"><div class="surname">Lusis</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">Evolution of apolipoprotein E: mouse sequence and evidence for an 11-nucleotide ancestral unit</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1985</div>, vol. <div class="volume">82</div> (pg. <div class="fpage">8085</div>-<div class="lpage">89</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20apolipoprotein%20E%3A%20mouse%20sequence%20and%20evidence%20for%20an%2011-nucleotide%20ancestral%20unit&amp;author=T.B.%20Rajavashisth&amp;author=J.S.%20Kaprein&amp;author=K.L.%20Reue&amp;author=A.J.%20Lusis&amp;publication_year=1985&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=82&amp;pages=8085-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.82.23.8085" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.82.23.8085" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.82.23.8085"> </span></div> </div></div></div></div></div><div content-id="DSR023C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR023C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bazan</div> <div class="given-names">J.F.</div></div>. </div><div class="article-title">Structural design and molecular evolution of a cytokine receptor superfamily</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">1990</div>, vol. <div class="volume">87</div> (pg. <div class="fpage">6934</div>-<div class="lpage">38</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20design%20and%20molecular%20evolution%20of%20a%20cytokine%20receptor%20superfamily&amp;author=J.F.%20Bazan&amp;publication_year=1990&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=87&amp;pages=6934-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.87.18.6934" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.87.18.6934" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.87.18.6934"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=449021&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">107</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000296708600005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">5</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr023"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="449021"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000296708600005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f5%2f353%2f449021">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000296708600005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (5)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F5%2F353%2F449021" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/449021'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Profile-Statistical Periodicity of DNA Coding Regions" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr023");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f5%2f353%2f449021">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

