

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/1/49/4107864","@type":"ScholarlyArticle","name":"Genome-wide mapping of large deletions and their population-genetic properties in dairy cattle","datePublished":"2017-09-08","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2018-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx037","keywords":["dairy cattle","structural variants","whole genome sequence","population genetics","loss-of-function"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/1/49/4107864","author":[{"name":"Mesbah-Uddin, Md","affiliation":"Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark  Animal Genetics and Integrative Biology, UMR 1313 GABI, INRA, AgroParisTech, Université Paris-Saclay, 78350 Jouy-en-Josas, France","@type":"Person"},{"name":"Guldbrandtsen, Bernt","affiliation":"Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark","@type":"Person"},{"name":"Iso-Touru, Terhi","affiliation":"Green Technology, Natural Resources Institute Finland, FI-31600 Jokioinen, Finland","@type":"Person"},{"name":"Vilkki, Johanna","affiliation":"Green Technology, Natural Resources Institute Finland, FI-31600 Jokioinen, Finland","@type":"Person"},{"name":"De Koning, Dirk-Jan","affiliation":"Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden","@type":"Person"},{"name":"Boichard, Didier","affiliation":"Animal Genetics and Integrative Biology, UMR 1313 GABI, INRA, AgroParisTech, Université Paris-Saclay, 78350 Jouy-en-Josas, France","@type":"Person"},{"name":"Lund, Mogens Sandø","affiliation":"Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark","@type":"Person"},{"name":"Sahana, Goutam","affiliation":"Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark","@type":"Person"}],"description":"Abstract.  Large genomic deletions are potential candidate for loss-of-function, which could be lethal as homozygote. Analysing whole genome data of 175 cattle,","pageStart":"49","pageEnd":"59","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-wide mapping of large deletions and their population-genetic properties in dairy cattle","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-wide mapping of large deletions and their population-genetic properties in dairy cattle" />
    <meta property="og:description" content="Abstract.  Large genomic deletions are potential candidate for loss-of-function, which could be lethal as homozygote. Analysing whole genome data of 175 cattle," />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/1/49/4107864" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-09-08" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Mesbah-Uddin, Md" /><meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark" /><meta name="citation_author_institution" content="Animal Genetics and Integrative Biology, UMR 1313 GABI, INRA, AgroParisTech, Université Paris-Saclay, 78350 Jouy-en-Josas, France" /><meta name="citation_author" content="Guldbrandtsen, Bernt" /><meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark" /><meta name="citation_author" content="Iso-Touru, Terhi" /><meta name="citation_author_institution" content="Green Technology, Natural Resources Institute Finland, FI-31600 Jokioinen, Finland" /><meta name="citation_author" content="Vilkki, Johanna" /><meta name="citation_author_institution" content="Green Technology, Natural Resources Institute Finland, FI-31600 Jokioinen, Finland" /><meta name="citation_author" content="De Koning, Dirk-Jan" /><meta name="citation_author_institution" content="Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden" /><meta name="citation_author" content="Boichard, Didier" /><meta name="citation_author_institution" content="Animal Genetics and Integrative Biology, UMR 1313 GABI, INRA, AgroParisTech, Université Paris-Saclay, 78350 Jouy-en-Josas, France" /><meta name="citation_author" content="Lund, Mogens Sandø" /><meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark" /><meta name="citation_author" content="Sahana, Goutam" /><meta name="citation_author_institution" content="Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark" /><meta name="citation_title" content="Genome-wide mapping of large deletions and their population-genetic properties in dairy cattle" /><meta name="citation_firstpage" content="49" /><meta name="citation_lastpage" content="59" /><meta name="citation_doi" content="10.1093/dnares/dsx037" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2018/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=NGS-based reverse genetic screen for common embryonic lethal mutations compromising fertility in livestock;  citation_journal_title=Genome Res;  citation_year=2016;  citation_volume=26;  citation_pages=1333-1341; " /><meta name="citation_reference" content="citation_title=Phenotypic and genetic effects of recessive haplotypes on yield, longevity, and fertility;  citation_journal_title=J Dairy Sci;  citation_year=2016;  citation_volume=99;  citation_pages=7274-88; " /><meta name="citation_reference" content="citation_title=Phenotypic impact of genomic structural variation: insights from and for human disease;  citation_journal_title=Nat. Rev. Genet;  citation_year=2013;  citation_volume=14;  citation_pages=125-138; " /><meta name="citation_reference" content="citation_title=A copy number variation map of the human genome;  citation_journal_title=Nat Rev Genet;  citation_year=2015;  citation_volume=16;  citation_pages=172-183; " /><meta name="citation_reference" content="citation_title=The challenges and importance of structural variation detection in livestock;  citation_journal_title=Front. Genet;  citation_year=2014;  citation_volume=5;  citation_pages=37" /><meta name="citation_reference" content="citation_title=Genome wide CNV analysis reveals additional variants associated with milk production traits in Holsteins;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=683" /><meta name="citation_reference" content="citation_title=A deletion in the bovine FANCI gene compromises fertility by causing fetal death and brachyspina;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e43085" /><meta name="citation_reference" content="citation_title=The holstein friesian lethal haplotype 5 (HH5) results from a complete deletion of TBF1M and cholesterol deficiency (CDH) from an ERV-(LTR) insertion into the coding region of APOB;  citation_journal_title=PLoS One;  citation_year=2016;  citation_volume=11;  citation_pages=e0154602" /><meta name="citation_reference" content="citation_title=A 660-Kb deletion with antagonistic effects on fertility and milk production segregates at high frequency in Nordic Red cattle: additional evidence for the common occurrence of balancing selection in livestock;  citation_journal_title=PLoS Genet;  citation_year=2014;  citation_volume=10;  citation_pages=e1004049" /><meta name="citation_reference" content="citation_title=A 0.5-Mbp deletion on bovine chromosome 23 is a strong candidate for stillbirth in Nordic Red cattle;  citation_journal_title=Genet. Sel. Evol;  citation_year=2016;  citation_volume=48;  citation_pages=35" /><meta name="citation_reference" content="citation_title=Analysis of copy number variations among diverse cattle breeds;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=693-703; " /><meta name="citation_reference" content="citation_title=Genomic characteristics of cattle copy number variations;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=127" /><meta name="citation_reference" content="citation_title=Population-genetic properties of differentiated copy number variations in cattle;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=23161" /><meta name="citation_reference" content="citation_title=Genome structural variation discovery and genotyping;  citation_journal_title=Nat. Rev. Genet;  citation_year=2011;  citation_volume=12;  citation_pages=363-376; " /><meta name="citation_reference" content="citation_title=An integrated map of structural variation in 2,504 human genomes;  citation_journal_title=Nature;  citation_year=2015;  citation_volume=526;  citation_pages=75-81; " /><meta name="citation_reference" content="citation_title=Mapping copy number variation by population-scale genome sequencing;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=470;  citation_pages=59-65; " /><meta name="citation_reference" content="citation_title=Sequence-based characterization of structural variation in the mouse genome;  citation_journal_title=Nature;  citation_year=2011;  citation_volume=477;  citation_pages=326-329; " /><meta name="citation_reference" content="citation_title=TIGRA: a targeted iterative graph routing assembler for breakpoint assembly;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=310-317; " /><meta name="citation_reference" content="citation_title=Mechanisms underlying structural variant formation in genomic disorders;  citation_journal_title=Nat Rev Genet;  citation_year=2016;  citation_volume=17;  citation_pages=224-238; " /><meta name="citation_reference" content="citation_title=Whole-genome sequencing of 234 bulls facilitates mapping of monogenic and complex traits in cattle;  citation_journal_title=Nat. Genet;  citation_year=2014;  citation_volume=46;  citation_pages=858-865; " /><meta name="citation_reference" content="citation_title=Strategies for imputation to whole genome sequence using a single or multi-breed reference population in cattle;  citation_journal_title=BMC Genomics;  citation_year=2014;  citation_volume=15;  citation_pages=728" /><meta name="citation_reference" content="citation_title=Assessment of the genomic variation in a cattle population by re-sequencing of key animals at low to medium coverage;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=446" /><meta name="citation_reference" content="citation_title=Genome-wide study of structural variants in bovine holstein, montbeliarde and normande dairy breeds;  citation_journal_title=PLoS One;  citation_year=2015;  citation_volume=10;  citation_pages=e0135931" /><meta name="citation_reference" content="citation_title=Detection and validation of structural variations in bovine whole-genome sequence data;  citation_journal_title=Genet. Select. Evol;  citation_year=2017;  citation_volume=49;  citation_pages=13" /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-1760; " /><meta name="citation_reference" content="citation_title=The Sequence Alignment/Map format and SAMtools;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-2079; " /><meta name="citation_reference" content="citation_title=The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=1297-1303; " /><meta name="citation_reference" content="citation_title=Discovery and genotyping of genome structural polymorphism by sequencing on a population scale;  citation_journal_title=Nat. Genet;  citation_year=2011;  citation_volume=43;  citation_pages=269-276; " /><meta name="citation_reference" content="citation_title=The UCSC Table Browser data retrieval tool;  citation_journal_title=Nucleic Acids Res.;  citation_year=2004;  citation_volume=32;  citation_pages=D493-496; " /><meta name="citation_reference" content="citation_title=BLAT–the BLAST-like alignment tool;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=656-664; " /><meta name="citation_reference" content="citation_title=A simple salting out procedure for extracting DNA from human nucleated cells;  citation_journal_title=Nucleic Acids Res;  citation_year=1988;  citation_volume=16;  citation_pages=1215" /><meta name="citation_reference" content="citation_title=PLINK: a tool set for whole-genome association and population-based linkage analyses;  citation_journal_title=Am. J. Hum. Genet;  citation_year=2007;  citation_volume=81;  citation_pages=559-575; " /><meta name="citation_reference" content="citation_title=Global variation in copy number in the human genome;  citation_journal_title=Nature;  citation_year=2006;  citation_volume=444;  citation_pages=444-454; " /><meta name="citation_reference" content="citation_title=Evolution in Mendelian Populations;  citation_journal_title=Genetics;  citation_year=1931;  citation_volume=16;  citation_pages=97-159; " /><meta name="citation_reference" content="citation_title=The Ensembl Variant Effect Predictor;  citation_journal_title=Genome Biol;  citation_year=2016;  citation_volume=17;  citation_pages=122" /><meta name="citation_reference" content="citation_title=InterPro in 2017-beyond protein family and domain annotations;  citation_journal_title=Nucleic Acids Res;  citation_year=2017;  citation_volume=45;  citation_pages=D190-D199; " /><meta name="citation_reference" content="citation_title=The Pfam protein families database: towards a more sustainable future;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=D279-285; " /><meta name="citation_reference" content="citation_title=KEGG: new perspectives on genomes, pathways, diseases and drugs;  citation_journal_title=Nucleic Acids Res;  citation_year=2017;  citation_volume=45;  citation_pages=D353-61; " /><meta name="citation_reference" content="citation_title=STRING v10: protein-protein interaction networks, integrated over the tree of life;  citation_journal_title=Nucleic Acids Res;  citation_year=2015;  citation_volume=43;  citation_pages=D447-52; " /><meta name="citation_reference" content="citation_title=Ensembl 2016;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=D710-6; " /><meta name="citation_reference" content="citation_title=Ensembl BioMarts: a hub for data retrieval across taxonomic space;  citation_journal_title=Database (Oxford);  citation_year=2011;  citation_volume=2011;  citation_pages=bar030" /><meta name="citation_reference" content="citation_title=High-throughput discovery of novel developmental phenotypes;  citation_journal_title=Nature;  citation_year=2016;  citation_volume=537;  citation_pages=508-14; " /><meta name="citation_reference" content="citation_title=Developmental progress and current status of the Animal QTLdb;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=D827-33; " /><meta name="citation_reference" content="citation_author=RStudio Team; citation_publisher=RStudio, Inc, Boston, MA;  citation_title=RStudio: integrated development environment for R;  citation_year=2016; " /><meta name="citation_reference" content="citation_author=R Core Team; citation_publisher=R Foundation for Statistical Computing, Vienna, Austria;  citation_title=R: A language and environment for statistical computing;  citation_year=2016; " /><meta name="citation_reference" content="citation_title=BEDTools: a flexible suite of utilities for comparing genomic features;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=841-842; " /><meta name="citation_reference" content="citation_title=DbVar and DGVa: public archives for genomic structural variation;  citation_journal_title=Nucleic Acids Res;  citation_year=2013;  citation_volume=41;  citation_pages=D936-941; " /><meta name="citation_reference" content="citation_title=Integrated detection and population-genetic analysis of SNPs and copy number variation;  citation_journal_title=Nat. Genet;  citation_year=2008;  citation_volume=40;  citation_pages=1166-74; " /><meta name="citation_reference" content="citation_title=Origins and functional impact of copy number variation in the human genome;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=464;  citation_pages=704-12; " /><meta name="citation_reference" content="citation_title=Large multiallelic copy number variations in humans;  citation_journal_title=Nat. Genet;  citation_year=2015;  citation_volume=47;  citation_pages=296-303; " /><meta name="citation_reference" content="citation_title=Runs of homozygosity and distribution of functional variants in the cattle genome;  citation_journal_title=BMC Genomics;  citation_year=2015;  citation_volume=16;  citation_pages=542" /><meta name="citation_reference" content="citation_title=Genome-wide survey of SNP variation uncovers the genetic structure of cattle breeds;  citation_journal_title=Science;  citation_year=2009;  citation_volume=324;  citation_pages=528-32; " /><meta name="citation_reference" content="citation_title=Genome-wide association studies of growth traits in three dairy cattle breeds using whole-genome sequence data;  citation_journal_title=J. Anim. Sci;  citation_year=2016;  citation_volume=94;  citation_pages=1426-37; " /><meta name="citation_reference" content="citation_title=Reliabilities of genomic prediction using combined reference data of the Nordic Red dairy cattle populations;  citation_journal_title=J. Dairy Sci;  citation_year=2011;  citation_volume=94;  citation_pages=4700-4707; " /><meta name="citation_reference" content="citation_title=Genomic signatures reveal new evidences for selection of important traits in domestic cattle;  citation_journal_title=Mol. Biol. Evol;  citation_year=2015;  citation_volume=32;  citation_pages=711-25; " /><meta name="citation_reference" content="citation_title=A genome-wide association study of calf birth weight in Holstein cattle using single nucleotide polymorphisms and phenotypes predicted from auxiliary traits;  citation_journal_title=J Dairy Sci;  citation_year=2014;  citation_volume=97;  citation_pages=3156-3172; " /><meta name="citation_reference" content="citation_title=Beef cattle body temperature during climatic stress: a genome-wide association study;  citation_journal_title=Int. J. Biometeorol;  citation_year=2014;  citation_volume=58;  citation_pages=1665-1672; " /><meta name="citation_reference" content="citation_title=Estimation of genetic parameters and detection of chromosomal regions affecting the major milk proteins and their post translational modifications in Danish Holstein and Danish Jersey cattle;  citation_journal_title=BMC Genet;  citation_year=2016;  citation_volume=17;  citation_pages=114" /><meta name="citation_reference" content="citation_title=A genome scan for quantitative trait loci influencing carcass, post-natal growth and reproductive traits in commercial Angus cattle;  citation_journal_title=Anim. Genet;  citation_year=2010;  citation_volume=41;  citation_pages=597-607; " /><meta name="citation_reference" content="citation_title=Genome-wide association study for calving traits in Danish and Swedish Holstein cattle;  citation_journal_title=J. Dairy Sci;  citation_year=2011;  citation_volume=94;  citation_pages=479-486; " /><meta name="citation_reference" content="citation_title=Genome-wide association analysis for quantitative trait loci influencing Warner-Bratzler shear force in five taurine cattle breeds;  citation_journal_title=Anim. Genet;  citation_year=2012;  citation_volume=43;  citation_pages=662-673; " /><meta name="citation_reference" content="citation_title=Genome-wide association study of growth in crossbred beef cattle;  citation_journal_title=J. Anim. Sci;  citation_year=2010;  citation_volume=88;  citation_pages=837-848; " /><meta name="citation_reference" content="citation_title=Genome-wide association analysis of thirty one production, health, reproduction and body conformation traits in contemporary U.S. Holstein cows;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=408" /><meta name="citation_reference" content="citation_title=Clinical features and hormonal profiles of cloprostenol-induced early abortions in heifers monitored by ultrasonography;  citation_journal_title=Acta Vet. Scand;  citation_year=2006;  citation_volume=48;  citation_pages=23" /><meta name="citation_reference" content="citation_title=Transcriptome profiles of whole blood in Italian Holstein and Italian Simmental lactating cows diverging for genetic merit for milk protein;  citation_journal_title=J. Dairy Sci;  citation_year=2015;  citation_volume=98;  citation_pages=6119-6127; " /><meta name="citation_reference" content="citation_title=Use of a candidate gene array to delineate gene expression patterns in cattle selected for resistance or susceptibility to intestinal nematodes;  citation_journal_title=Vet. Parasitol;  citation_year=2009;  citation_volume=162;  citation_pages=106-115; " /><meta name="citation_reference" content="citation_title=Integrating sequence-based GWAS and RNA-seq provides novel insights into the genetic basis of mastitis and milk production in dairy cattle;  citation_journal_title=Sci. Rep;  citation_year=2017;  citation_volume=7;  citation_pages=45560" /><meta name="citation_reference" content="citation_title=Do essential genes evolve slowly?;  citation_journal_title=Curr. Biol;  citation_year=1999;  citation_volume=9;  citation_pages=747-50; " /><meta name="citation_reference" content="citation_title=Gene essentiality and synthetic lethality in haploid human cells;  citation_journal_title=Science;  citation_year=2015;  citation_volume=350;  citation_pages=1092-6; " /><meta name="citation_reference" content="citation_title=Extensive gains and losses of olfactory receptor genes in mammalian evolution;  citation_journal_title=PLoS One;  citation_year=2007;  citation_volume=2;  citation_pages=e708" /><meta name="citation_reference" content="citation_title=Homozygous deletion of six olfactory receptor genes in a subset of individuals with Beta-thalassemia;  citation_journal_title=PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e17327" /><meta name="citation_reference" content="citation_title=Analysis of cattle olfactory subgenome: the first detail study on the characteristics of the complete olfactory receptor repertoire of a ruminant;  citation_journal_title=BMC Genomics;  citation_year=2013;  citation_volume=14;  citation_pages=596" /><meta name="citation_reference" content="citation_title=Mechanisms of change in gene copy number;  citation_journal_title=Nat. Rev. Genet;  citation_year=2009;  citation_volume=10;  citation_pages=551-64; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/1/49/24151521/dsx037.pdf" /><meta name="description" content="Abstract.  Large genomic deletions are potential candidate for loss-of-function, which could be lethal as homozygote. Analysing whole genome data of 175 cattle," />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/1/49/4107864" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/1/49/4107864" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/1/49/4107864");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx037';
    </script>
<title>
	Genome-wide mapping of large deletions and their population-genetic properties in dairy cattle | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4107864" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BaanATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWVzYmFoLVVkZGluLCBNZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTW9sZWN1bGFyIEJpb2xvZ3kgYW5kIEdlbmV0aWNzLCBDZW50ZXIgZm9yIFF1YW50aXRhdGl2ZSBHZW5ldGljcyBhbmQgR2Vub21pY3MsIEFhcmh1cyBVbml2ZXJzaXR5LCA4ODMwIFRqZWxlLCBEZW5tYXJrIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQW5pbWFsIEdlbmV0aWNzIGFuZCBJbnRlZ3JhdGl2ZSBCaW9sb2d5LCBVTVIgMTMxMyBHQUJJLCBJTlJBLCBBZ3JvUGFyaXNUZWNoLCBVbml2ZXJzaXTDqSBQYXJpcy1TYWNsYXksIDc4MzUwIEpvdXktZW4tSm9zYXMsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikd1bGRicmFuZHRzZW4sIEJlcm50IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgQmlvbG9neSBhbmQgR2VuZXRpY3MsIENlbnRlciBmb3IgUXVhbnRpdGF0aXZlIEdlbmV0aWNzIGFuZCBHZW5vbWljcywgQWFyaHVzIFVuaXZlcnNpdHksIDg4MzAgVGplbGUsIERlbm1hcmsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJc28tVG91cnUsIFRlcmhpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JlZW4gVGVjaG5vbG9neSwgTmF0dXJhbCBSZXNvdXJjZXMgSW5zdGl0dXRlIEZpbmxhbmQsIEZJLTMxNjAwIEpva2lvaW5lbiwgRmlubGFuZCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlZpbGtraSwgSm9oYW5uYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyZWVuIFRlY2hub2xvZ3ksIE5hdHVyYWwgUmVzb3VyY2VzIEluc3RpdHV0ZSBGaW5sYW5kLCBGSS0zMTYwMCBKb2tpb2luZW4sIEZpbmxhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJEZSBLb25pbmcsIERpcmstSmFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBbmltYWwgQnJlZWRpbmcgYW5kIEdlbmV0aWNzLCBTd2VkaXNoIFVuaXZlcnNpdHkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzLCBTRS03NTDigIkwNyBVcHBzYWxhLCBTd2VkZW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJCb2ljaGFyZCwgRGlkaWVyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQW5pbWFsIEdlbmV0aWNzIGFuZCBJbnRlZ3JhdGl2ZSBCaW9sb2d5LCBVTVIgMTMxMyBHQUJJLCBJTlJBLCBBZ3JvUGFyaXNUZWNoLCBVbml2ZXJzaXTDqSBQYXJpcy1TYWNsYXksIDc4MzUwIEpvdXktZW4tSm9zYXMsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikx1bmQsIE1vZ2VucyBTYW5kw7giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1vbGVjdWxhciBCaW9sb2d5IGFuZCBHZW5ldGljcywgQ2VudGVyIGZvciBRdWFudGl0YXRpdmUgR2VuZXRpY3MgYW5kIEdlbm9taWNzLCBBYXJodXMgVW5pdmVyc2l0eSwgODgzMCBUamVsZSwgRGVubWFyayIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhaGFuYSwgR291dGFtIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgQmlvbG9neSBhbmQgR2VuZXRpY3MsIENlbnRlciBmb3IgUXVhbnRpdGF0aXZlIEdlbmV0aWNzIGFuZCBHZW5vbWljcywgQWFyaHVzIFVuaXZlcnNpdHksIDg4MzAgVGplbGUsIERlbm1hcmsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9Ikdlbm9tZS13aWRlIG1hcHBpbmcgb2YgbGFyZ2UgZGVsZXRpb25zIGFuZCB0aGVpciBwb3B1bGF0aW9uLWdlbmV0aWMgcHJvcGVydGllcyBpbiBkYWlyeSBjYXR0bGUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI0OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3gwMzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxOC8wMi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5HUy1iYXNlZCByZXZlcnNlIGdlbmV0aWMgc2NyZWVuIGZvciBjb21tb24gZW1icnlvbmljIGxldGhhbCBtdXRhdGlvbnMgY29tcHJvbWlzaW5nIGZlcnRpbGl0eSBpbiBsaXZlc3RvY2s7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0xMzMzLTEzNDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoZW5vdHlwaWMgYW5kIGdlbmV0aWMgZWZmZWN0cyBvZiByZWNlc3NpdmUgaGFwbG90eXBlcyBvbiB5aWVsZCwgbG9uZ2V2aXR5LCBhbmQgZmVydGlsaXR5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KIERhaXJ5IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT05OTsgIGNpdGF0aW9uX3BhZ2VzPTcyNzQtODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoZW5vdHlwaWMgaW1wYWN0IG9mIGdlbm9taWMgc3RydWN0dXJhbCB2YXJpYXRpb246IGluc2lnaHRzIGZyb20gYW5kIGZvciBodW1hbiBkaXNlYXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz0xMjUtMTM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIGNvcHkgbnVtYmVyIHZhcmlhdGlvbiBtYXAgb2YgdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0IFJldiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTE3Mi0xODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjaGFsbGVuZ2VzIGFuZCBpbXBvcnRhbmNlIG9mIHN0cnVjdHVyYWwgdmFyaWF0aW9uIGRldGVjdGlvbiBpbiBsaXZlc3RvY2s7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MzciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgd2lkZSBDTlYgYW5hbHlzaXMgcmV2ZWFscyBhZGRpdGlvbmFsIHZhcmlhbnRzIGFzc29jaWF0ZWQgd2l0aCBtaWxrIHByb2R1Y3Rpb24gdHJhaXRzIGluIEhvbHN0ZWluczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NjgzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBkZWxldGlvbiBpbiB0aGUgYm92aW5lIEZBTkNJIGdlbmUgY29tcHJvbWlzZXMgZmVydGlsaXR5IGJ5IGNhdXNpbmcgZmV0YWwgZGVhdGggYW5kIGJyYWNoeXNwaW5hOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ZTQzMDg1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGhvbHN0ZWluIGZyaWVzaWFuIGxldGhhbCBoYXBsb3R5cGUgNSAoSEg1KSByZXN1bHRzIGZyb20gYSBjb21wbGV0ZSBkZWxldGlvbiBvZiBUQkYxTSBhbmQgY2hvbGVzdGVyb2wgZGVmaWNpZW5jeSAoQ0RIKSBmcm9tIGFuIEVSVi0oTFRSKSBpbnNlcnRpb24gaW50byB0aGUgY29kaW5nIHJlZ2lvbiBvZiBBUE9COyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPWUwMTU0NjAyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSA2NjAtS2IgZGVsZXRpb24gd2l0aCBhbnRhZ29uaXN0aWMgZWZmZWN0cyBvbiBmZXJ0aWxpdHkgYW5kIG1pbGsgcHJvZHVjdGlvbiBzZWdyZWdhdGVzIGF0IGhpZ2ggZnJlcXVlbmN5IGluIE5vcmRpYyBSZWQgY2F0dGxlOiBhZGRpdGlvbmFsIGV2aWRlbmNlIGZvciB0aGUgY29tbW9uIG9jY3VycmVuY2Ugb2YgYmFsYW5jaW5nIHNlbGVjdGlvbiBpbiBsaXZlc3RvY2s7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMTAwNDA0OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgMC41LU1icCBkZWxldGlvbiBvbiBib3ZpbmUgY2hyb21vc29tZSAyMyBpcyBhIHN0cm9uZyBjYW5kaWRhdGUgZm9yIHN0aWxsYmlydGggaW4gTm9yZGljIFJlZCBjYXR0bGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0LiBTZWwuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz0zNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuYWx5c2lzIG9mIGNvcHkgbnVtYmVyIHZhcmlhdGlvbnMgYW1vbmcgZGl2ZXJzZSBjYXR0bGUgYnJlZWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9NjkzLTcwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBjaGFyYWN0ZXJpc3RpY3Mgb2YgY2F0dGxlIGNvcHkgbnVtYmVyIHZhcmlhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTEyNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcHVsYXRpb24tZ2VuZXRpYyBwcm9wZXJ0aWVzIG9mIGRpZmZlcmVudGlhdGVkIGNvcHkgbnVtYmVyIHZhcmlhdGlvbnMgaW4gY2F0dGxlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MjMxNjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc3RydWN0dXJhbCB2YXJpYXRpb24gZGlzY292ZXJ5IGFuZCBnZW5vdHlwaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0zNjMtMzc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlZ3JhdGVkIG1hcCBvZiBzdHJ1Y3R1cmFsIHZhcmlhdGlvbiBpbiAyLDUwNCBodW1hbiBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9NTI2OyAgY2l0YXRpb25fcGFnZXM9NzUtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgY29weSBudW1iZXIgdmFyaWF0aW9uIGJ5IHBvcHVsYXRpb24tc2NhbGUgZ2Vub21lIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00NzA7ICBjaXRhdGlvbl9wYWdlcz01OS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VxdWVuY2UtYmFzZWQgY2hhcmFjdGVyaXphdGlvbiBvZiBzdHJ1Y3R1cmFsIHZhcmlhdGlvbiBpbiB0aGUgbW91c2UgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDc3OyAgY2l0YXRpb25fcGFnZXM9MzI2LTMyOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VElHUkE6IGEgdGFyZ2V0ZWQgaXRlcmF0aXZlIGdyYXBoIHJvdXRpbmcgYXNzZW1ibGVyIGZvciBicmVha3BvaW50IGFzc2VtYmx5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MzEwLTMxNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVjaGFuaXNtcyB1bmRlcmx5aW5nIHN0cnVjdHVyYWwgdmFyaWFudCBmb3JtYXRpb24gaW4gZ2Vub21pYyBkaXNvcmRlcnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdCBSZXYgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz0yMjQtMjM4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaG9sZS1nZW5vbWUgc2VxdWVuY2luZyBvZiAyMzQgYnVsbHMgZmFjaWxpdGF0ZXMgbWFwcGluZyBvZiBtb25vZ2VuaWMgYW5kIGNvbXBsZXggdHJhaXRzIGluIGNhdHRsZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT00NjsgIGNpdGF0aW9uX3BhZ2VzPTg1OC04NjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cmF0ZWdpZXMgZm9yIGltcHV0YXRpb24gdG8gd2hvbGUgZ2Vub21lIHNlcXVlbmNlIHVzaW5nIGEgc2luZ2xlIG9yIG11bHRpLWJyZWVkIHJlZmVyZW5jZSBwb3B1bGF0aW9uIGluIGNhdHRsZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTE1OyAgY2l0YXRpb25fcGFnZXM9NzI4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZXNzbWVudCBvZiB0aGUgZ2Vub21pYyB2YXJpYXRpb24gaW4gYSBjYXR0bGUgcG9wdWxhdGlvbiBieSByZS1zZXF1ZW5jaW5nIG9mIGtleSBhbmltYWxzIGF0IGxvdyB0byBtZWRpdW0gY292ZXJhZ2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTQ0NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHN0dWR5IG9mIHN0cnVjdHVyYWwgdmFyaWFudHMgaW4gYm92aW5lIGhvbHN0ZWluLCBtb250YmVsaWFyZGUgYW5kIG5vcm1hbmRlIGRhaXJ5IGJyZWVkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1lMDEzNTkzMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldGVjdGlvbiBhbmQgdmFsaWRhdGlvbiBvZiBzdHJ1Y3R1cmFsIHZhcmlhdGlvbnMgaW4gYm92aW5lIHdob2xlLWdlbm9tZSBzZXF1ZW5jZSBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldC4gU2VsZWN0LiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9MTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGFuZCBhY2N1cmF0ZSBzaG9ydCByZWFkIGFsaWdubWVudCB3aXRoIEJ1cnJvd3MtV2hlZWxlciB0cmFuc2Zvcm07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTc1NC0xNzYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgU2VxdWVuY2UgQWxpZ25tZW50L01hcCBmb3JtYXQgYW5kIFNBTXRvb2xzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTIwNzgtMjA3OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEdlbm9tZSBBbmFseXNpcyBUb29sa2l0OiBhIE1hcFJlZHVjZSBmcmFtZXdvcmsgZm9yIGFuYWx5emluZyBuZXh0LWdlbmVyYXRpb24gRE5BIHNlcXVlbmNpbmcgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTEyOTctMTMwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzY292ZXJ5IGFuZCBnZW5vdHlwaW5nIG9mIGdlbm9tZSBzdHJ1Y3R1cmFsIHBvbHltb3JwaGlzbSBieSBzZXF1ZW5jaW5nIG9uIGEgcG9wdWxhdGlvbiBzY2FsZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPTI2OS0yNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBVQ1NDIFRhYmxlIEJyb3dzZXIgZGF0YSByZXRyaWV2YWwgdG9vbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9RDQ5My00OTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJMQVTigJN0aGUgQkxBU1QtbGlrZSBhbGlnbm1lbnQgdG9vbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTY1Ni02NjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc2ltcGxlIHNhbHRpbmcgb3V0IHByb2NlZHVyZSBmb3IgZXh0cmFjdGluZyBETkEgZnJvbSBodW1hbiBudWNsZWF0ZWQgY2VsbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTg4OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTIxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBMSU5LOiBhIHRvb2wgc2V0IGZvciB3aG9sZS1nZW5vbWUgYXNzb2NpYXRpb24gYW5kIHBvcHVsYXRpb24tYmFzZWQgbGlua2FnZSBhbmFseXNlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QW0uIEouIEh1bS4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODE7ICBjaXRhdGlvbl9wYWdlcz01NTktNTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgdmFyaWF0aW9uIGluIGNvcHkgbnVtYmVyIGluIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDQ7ICBjaXRhdGlvbl9wYWdlcz00NDQtNDU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb24gaW4gTWVuZGVsaWFuIFBvcHVsYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTkzMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTk3LTE1OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEVuc2VtYmwgVmFyaWFudCBFZmZlY3QgUHJlZGljdG9yOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTEyMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVyUHJvIGluIDIwMTctYmV5b25kIHByb3RlaW4gZmFtaWx5IGFuZCBkb21haW4gYW5ub3RhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9RDE5MC1EMTk5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgUGZhbSBwcm90ZWluIGZhbWlsaWVzIGRhdGFiYXNlOiB0b3dhcmRzIGEgbW9yZSBzdXN0YWluYWJsZSBmdXR1cmU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9RDI3OS0yODU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUtFR0c6IG5ldyBwZXJzcGVjdGl2ZXMgb24gZ2Vub21lcywgcGF0aHdheXMsIGRpc2Vhc2VzIGFuZCBkcnVnczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz1EMzUzLTYxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TVFJJTkcgdjEwOiBwcm90ZWluLXByb3RlaW4gaW50ZXJhY3Rpb24gbmV0d29ya3MsIGludGVncmF0ZWQgb3ZlciB0aGUgdHJlZSBvZiBsaWZlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzsgIGNpdGF0aW9uX3BhZ2VzPUQ0NDctNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVuc2VtYmwgMjAxNjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz1ENzEwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVuc2VtYmwgQmlvTWFydHM6IGEgaHViIGZvciBkYXRhIHJldHJpZXZhbCBhY3Jvc3MgdGF4b25vbWljIHNwYWNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1EYXRhYmFzZSAoT3hmb3JkKTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDExOyAgY2l0YXRpb25fcGFnZXM9YmFyMDMwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IGRpc2NvdmVyeSBvZiBub3ZlbCBkZXZlbG9wbWVudGFsIHBoZW5vdHlwZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT01Mzc7ICBjaXRhdGlvbl9wYWdlcz01MDgtMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50YWwgcHJvZ3Jlc3MgYW5kIGN1cnJlbnQgc3RhdHVzIG9mIHRoZSBBbmltYWwgUVRMZGI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTQ0OyAgY2l0YXRpb25fcGFnZXM9RDgyNy0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fYXV0aG9yPVJTdHVkaW8gVGVhbTsgY2l0YXRpb25fcHVibGlzaGVyPVJTdHVkaW8sIEluYywgQm9zdG9uLCBNQTsgIGNpdGF0aW9uX3RpdGxlPVJTdHVkaW86IGludGVncmF0ZWQgZGV2ZWxvcG1lbnQgZW52aXJvbm1lbnQgZm9yIFI7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX2F1dGhvcj1SIENvcmUgVGVhbTsgY2l0YXRpb25fcHVibGlzaGVyPVIgRm91bmRhdGlvbiBmb3IgU3RhdGlzdGljYWwgQ29tcHV0aW5nLCBWaWVubmEsIEF1c3RyaWE7ICBjaXRhdGlvbl90aXRsZT1SOiBBIGxhbmd1YWdlIGFuZCBlbnZpcm9ubWVudCBmb3Igc3RhdGlzdGljYWwgY29tcHV0aW5nOyAgY2l0YXRpb25feWVhcj0yMDE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CRURUb29sczogYSBmbGV4aWJsZSBzdWl0ZSBvZiB1dGlsaXRpZXMgZm9yIGNvbXBhcmluZyBnZW5vbWljIGZlYXR1cmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTg0MS04NDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURiVmFyIGFuZCBER1ZhOiBwdWJsaWMgYXJjaGl2ZXMgZm9yIGdlbm9taWMgc3RydWN0dXJhbCB2YXJpYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9RDkzNi05NDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0ZWQgZGV0ZWN0aW9uIGFuZCBwb3B1bGF0aW9uLWdlbmV0aWMgYW5hbHlzaXMgb2YgU05QcyBhbmQgY29weSBudW1iZXIgdmFyaWF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQwOyAgY2l0YXRpb25fcGFnZXM9MTE2Ni03NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9T3JpZ2lucyBhbmQgZnVuY3Rpb25hbCBpbXBhY3Qgb2YgY29weSBudW1iZXIgdmFyaWF0aW9uIGluIHRoZSBodW1hbiBnZW5vbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjQ7ICBjaXRhdGlvbl9wYWdlcz03MDQtMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxhcmdlIG11bHRpYWxsZWxpYyBjb3B5IG51bWJlciB2YXJpYXRpb25zIGluIGh1bWFuczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT00NzsgIGNpdGF0aW9uX3BhZ2VzPTI5Ni0zMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJ1bnMgb2YgaG9tb3p5Z29zaXR5IGFuZCBkaXN0cmlidXRpb24gb2YgZnVuY3Rpb25hbCB2YXJpYW50cyBpbiB0aGUgY2F0dGxlIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9NTQyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgc3VydmV5IG9mIFNOUCB2YXJpYXRpb24gdW5jb3ZlcnMgdGhlIGdlbmV0aWMgc3RydWN0dXJlIG9mIGNhdHRsZSBicmVlZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MzI0OyAgY2l0YXRpb25fcGFnZXM9NTI4LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkaWVzIG9mIGdyb3d0aCB0cmFpdHMgaW4gdGhyZWUgZGFpcnkgY2F0dGxlIGJyZWVkcyB1c2luZyB3aG9sZS1nZW5vbWUgc2VxdWVuY2UgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQW5pbS4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTk0OyAgY2l0YXRpb25fcGFnZXM9MTQyNi0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVsaWFiaWxpdGllcyBvZiBnZW5vbWljIHByZWRpY3Rpb24gdXNpbmcgY29tYmluZWQgcmVmZXJlbmNlIGRhdGEgb2YgdGhlIE5vcmRpYyBSZWQgZGFpcnkgY2F0dGxlIHBvcHVsYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBEYWlyeSBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9OTQ7ICBjaXRhdGlvbl9wYWdlcz00NzAwLTQ3MDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgc2lnbmF0dXJlcyByZXZlYWwgbmV3IGV2aWRlbmNlcyBmb3Igc2VsZWN0aW9uIG9mIGltcG9ydGFudCB0cmFpdHMgaW4gZG9tZXN0aWMgY2F0dGxlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIEJpb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz03MTEtMjU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZHkgb2YgY2FsZiBiaXJ0aCB3ZWlnaHQgaW4gSG9sc3RlaW4gY2F0dGxlIHVzaW5nIHNpbmdsZSBudWNsZW90aWRlIHBvbHltb3JwaGlzbXMgYW5kIHBoZW5vdHlwZXMgcHJlZGljdGVkIGZyb20gYXV4aWxpYXJ5IHRyYWl0czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SiBEYWlyeSBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9OTc7ICBjaXRhdGlvbl9wYWdlcz0zMTU2LTMxNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJlZWYgY2F0dGxlIGJvZHkgdGVtcGVyYXR1cmUgZHVyaW5nIGNsaW1hdGljIHN0cmVzczogYSBnZW5vbWUtd2lkZSBhc3NvY2lhdGlvbiBzdHVkeTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBKLiBCaW9tZXRlb3JvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01ODsgIGNpdGF0aW9uX3BhZ2VzPTE2NjUtMTY3MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0aW1hdGlvbiBvZiBnZW5ldGljIHBhcmFtZXRlcnMgYW5kIGRldGVjdGlvbiBvZiBjaHJvbW9zb21hbCByZWdpb25zIGFmZmVjdGluZyB0aGUgbWFqb3IgbWlsayBwcm90ZWlucyBhbmQgdGhlaXIgcG9zdCB0cmFuc2xhdGlvbmFsIG1vZGlmaWNhdGlvbnMgaW4gRGFuaXNoIEhvbHN0ZWluIGFuZCBEYW5pc2ggSmVyc2V5IGNhdHRsZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5vbWUgc2NhbiBmb3IgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgaW5mbHVlbmNpbmcgY2FyY2FzcywgcG9zdC1uYXRhbCBncm93dGggYW5kIHJlcHJvZHVjdGl2ZSB0cmFpdHMgaW4gY29tbWVyY2lhbCBBbmd1cyBjYXR0bGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFuaW0uIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9NTk3LTYwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZHkgZm9yIGNhbHZpbmcgdHJhaXRzIGluIERhbmlzaCBhbmQgU3dlZGlzaCBIb2xzdGVpbiBjYXR0bGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIERhaXJ5IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT05NDsgIGNpdGF0aW9uX3BhZ2VzPTQ3OS00ODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFzc29jaWF0aW9uIGFuYWx5c2lzIGZvciBxdWFudGl0YXRpdmUgdHJhaXQgbG9jaSBpbmZsdWVuY2luZyBXYXJuZXItQnJhdHpsZXIgc2hlYXIgZm9yY2UgaW4gZml2ZSB0YXVyaW5lIGNhdHRsZSBicmVlZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFuaW0uIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9NjYyLTY3MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZHkgb2YgZ3Jvd3RoIGluIGNyb3NzYnJlZCBiZWVmIGNhdHRsZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQW5pbS4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTg4OyAgY2l0YXRpb25fcGFnZXM9ODM3LTg0ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYXNzb2NpYXRpb24gYW5hbHlzaXMgb2YgdGhpcnR5IG9uZSBwcm9kdWN0aW9uLCBoZWFsdGgsIHJlcHJvZHVjdGlvbiBhbmQgYm9keSBjb25mb3JtYXRpb24gdHJhaXRzIGluIGNvbnRlbXBvcmFyeSBVLlMuIEhvbHN0ZWluIGNvd3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTQwOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsaW5pY2FsIGZlYXR1cmVzIGFuZCBob3Jtb25hbCBwcm9maWxlcyBvZiBjbG9wcm9zdGVub2wtaW5kdWNlZCBlYXJseSBhYm9ydGlvbnMgaW4gaGVpZmVycyBtb25pdG9yZWQgYnkgdWx0cmFzb25vZ3JhcGh5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1BY3RhIFZldC4gU2NhbmQ7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDg7ICBjaXRhdGlvbl9wYWdlcz0yMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgcHJvZmlsZXMgb2Ygd2hvbGUgYmxvb2QgaW4gSXRhbGlhbiBIb2xzdGVpbiBhbmQgSXRhbGlhbiBTaW1tZW50YWwgbGFjdGF0aW5nIGNvd3MgZGl2ZXJnaW5nIGZvciBnZW5ldGljIG1lcml0IGZvciBtaWxrIHByb3RlaW47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIERhaXJ5IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTYxMTktNjEyNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VXNlIG9mIGEgY2FuZGlkYXRlIGdlbmUgYXJyYXkgdG8gZGVsaW5lYXRlIGdlbmUgZXhwcmVzc2lvbiBwYXR0ZXJucyBpbiBjYXR0bGUgc2VsZWN0ZWQgZm9yIHJlc2lzdGFuY2Ugb3Igc3VzY2VwdGliaWxpdHkgdG8gaW50ZXN0aW5hbCBuZW1hdG9kZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVZldC4gUGFyYXNpdG9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE2MjsgIGNpdGF0aW9uX3BhZ2VzPTEwNi0xMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludGVncmF0aW5nIHNlcXVlbmNlLWJhc2VkIEdXQVMgYW5kIFJOQS1zZXEgcHJvdmlkZXMgbm92ZWwgaW5zaWdodHMgaW50byB0aGUgZ2VuZXRpYyBiYXNpcyBvZiBtYXN0aXRpcyBhbmQgbWlsayBwcm9kdWN0aW9uIGluIGRhaXJ5IGNhdHRsZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpLiBSZXA7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTQ1NTYwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RG8gZXNzZW50aWFsIGdlbmVzIGV2b2x2ZSBzbG93bHk/OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz03NDctNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgZXNzZW50aWFsaXR5IGFuZCBzeW50aGV0aWMgbGV0aGFsaXR5IGluIGhhcGxvaWQgaHVtYW4gY2VsbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzUwOyAgY2l0YXRpb25fcGFnZXM9MTA5Mi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHRlbnNpdmUgZ2FpbnMgYW5kIGxvc3NlcyBvZiBvbGZhY3RvcnkgcmVjZXB0b3IgZ2VuZXMgaW4gbWFtbWFsaWFuIGV2b2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPWU3MDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib21venlnb3VzIGRlbGV0aW9uIG9mIHNpeCBvbGZhY3RvcnkgcmVjZXB0b3IgZ2VuZXMgaW4gYSBzdWJzZXQgb2YgaW5kaXZpZHVhbHMgd2l0aCBCZXRhLXRoYWxhc3NlbWlhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9ZTE3MzI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2YgY2F0dGxlIG9sZmFjdG9yeSBzdWJnZW5vbWU6IHRoZSBmaXJzdCBkZXRhaWwgc3R1ZHkgb24gdGhlIGNoYXJhY3RlcmlzdGljcyBvZiB0aGUgY29tcGxldGUgb2xmYWN0b3J5IHJlY2VwdG9yIHJlcGVydG9pcmUgb2YgYSBydW1pbmFudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTE0OyAgY2l0YXRpb25fcGFnZXM9NTk2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWVjaGFuaXNtcyBvZiBjaGFuZ2UgaW4gZ2VuZSBjb3B5IG51bWJlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NTUxLTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjUvMS80OS8yNDE1MTUyMS9kc3gwMzcucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIExhcmdlIGdlbm9taWMgZGVsZXRpb25zIGFyZSBwb3RlbnRpYWwgY2FuZGlkYXRlIGZvciBsb3NzLW9mLWZ1bmN0aW9uLCB3aGljaCBjb3VsZCBiZSBsZXRoYWwgYXMgaG9tb3p5Z290ZS4gQW5hbHlzaW5nIHdob2xlIGdlbm9tZSBkYXRhIG9mIDE3NSBjYXR0bGUsIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00MTA3ODY0JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZCX6HSiWj85f7N0SeUOjlAiRAqxC" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f49%2f4107864" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f49%2f4107864" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=107546" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/1/3/m_dnares_25_1cover.png?Expires=2147483647&amp;Signature=SO6YCeydUh5yJs5n-IY9wmy~LoBoAx4P6Fvm-AjuGlXgQjk9YpdSZprn-w-uvSGcgqbsut967tdIfiY3dSqjk-kYgFErDTIMFh~0aK8LiNrwVszpCD29Va86wemvVduCLQhn8b79YN7Bcvi7qFB5E0wUNp-cxt7rNByaE14ePnoDwCgR5G58Pc5wx03IVhK9jft2n0g0lvAsEdQaWr3LG7PN4prc1KFhItNPOxm0fdg7HmXK23iCiP1EUMnFfqTR655CMN~bUB0aDDAZUT18HRZXXdjkjcMUCT~fjmwOlkGzU~71oDZYn-Ji-8-K8iuA-savjer0tQoSi784gQ8L6w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/1">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="112482662">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482662">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482664">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482664">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482670">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482670">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482699">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482699">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482742">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482742">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="112482751">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#112482751">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/1/39/4103403">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/1/61/4191345">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-wide mapping of large deletions and their population-genetic properties in dairy cattle
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Md Mesbah-Uddin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div><div class="aff">Animal Genetics and Integrative Biology, UMR 1313 GABI, INRA, AgroParisTech, Université Paris-Saclay, 78350 Jouy-en-Josas, France</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx037-cor1" data-open="dsx037-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +45 871 57821. Email: <a href="mailto:mdmesbah@gmail.com" target="_blank">mdmesbah@gmail.com</a> (M.M.-U.); Tel. +45 871 57501. Email: <a href="mailto:goutam.sahana@mbg.au.dk" target="_blank">goutam.sahana@mbg.au.dk</a> (G.S.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Md+Mesbah-Uddin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mesbah-Uddin M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mesbah-Uddin M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Md Mesbah-Uddin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bernt Guldbrandtsen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bernt+Guldbrandtsen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Guldbrandtsen B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Guldbrandtsen B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Bernt Guldbrandtsen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Terhi Iso-Touru</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Green Technology, Natural Resources Institute Finland, FI-31600 Jokioinen, Finland</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Terhi+Iso-Touru">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Iso-Touru T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Iso-Touru T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Terhi Iso-Touru</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Johanna Vilkki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Green Technology, Natural Resources Institute Finland, FI-31600 Jokioinen, Finland</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Johanna+Vilkki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Vilkki J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Vilkki J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Johanna Vilkki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Dirk-Jan De Koning</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Animal Breeding and Genetics, Swedish University of Agricultural Sciences, SE-750 07 Uppsala, Sweden</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Dirk-Jan+De+Koning">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=De Koning D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;De Koning D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Dirk-Jan De Koning</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Didier Boichard</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Animal Genetics and Integrative Biology, UMR 1313 GABI, INRA, AgroParisTech, Université Paris-Saclay, 78350 Jouy-en-Josas, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Didier+Boichard">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Boichard D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Boichard D&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Didier Boichard</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mogens Sand&#248; Lund</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mogens+Sand%c3%b8+Lund">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lund M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lund M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mogens Sandø Lund</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Goutam Sahana</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Molecular Biology and Genetics, Center for Quantitative Genetics and Genomics, Aarhus University, 8830 Tjele, Denmark</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx037-cor1" data-open="dsx037-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +45 871 57821. Email: <a href="mailto:mdmesbah@gmail.com" target="_blank">mdmesbah@gmail.com</a> (M.M.-U.); Tel. +45 871 57501. Email: <a href="mailto:goutam.sahana@mbg.au.dk" target="_blank">goutam.sahana@mbg.au.dk</a> (G.S.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Goutam+Sahana">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sahana G">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sahana G&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Goutam Sahana</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx037-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Minoru Yoshida</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 49–59, <a href='https://doi.org/10.1093/dnares/dsx037'>https://doi.org/10.1093/dnares/dsx037</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">08 September 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">08 June 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">18 August 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4107864" href="/dnaresearch/article-pdf/25/1/49/24151521/dsx037.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Md Mesbah-Uddin, Bernt Guldbrandtsen, Terhi Iso-Touru, Johanna Vilkki, Dirk-Jan De Koning, Didier Boichard, Mogens Sandø Lund, Goutam Sahana;  Genome-wide mapping of large deletions and their population-genetic properties in dairy cattle, <em>DNA Research</em>, Volume 25, Issue 1, 1 February 2018, Pages 49–59, <a href='https://doi.org/10.1093/dnares/dsx037'>https://doi.org/10.1093/dnares/dsx037</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4107864?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4107864?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4107864?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4107864?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4107864?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4107864?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4107864?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-wide%20mapping%20of%20large%20deletions%20and%20their%20population-genetic%20properties%20in%20dairy%20cattle&amp;publicationDate=2017-09-08&amp;volumeNum=25&amp;issueNum=1&amp;author=Mesbah-Uddin%2C%20Md%3B%20Guldbrandtsen%2C%20Bernt&amp;startPage=49&amp;endPage=59&amp;contentId=10.1093%2Fdnares%2Fdsx037&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f49%2f4107864" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=112482662 id="112482662" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Large genomic deletions are potential candidate for loss-of-function, which could be lethal as homozygote. Analysing whole genome data of 175 cattle, we report 8,480 large deletions (199 bp–773 KB) with an overall false discovery rate of 8.8%; 82% of which are novel compared with deletions in the dbVar database. Breakpoint sequence analyses revealed that majority (24 of 29 tested) of the deletions contain microhomology/homology at breakpoint, and therefore, most likely generated by microhomology-mediated end joining. We observed higher differentiation among breeds for deletions in some genic-regions, such as ABCA12, TTC1, VWA3B, TSHR, DST/BPAG1, and CD1D. The genes overlapping deletions are on average evolutionarily less conserved compared with known mouse lethal genes (<em>P</em>-value = 2.3 × 10<sup>−6</sup>). We report 167 natural gene knockouts in cattle that are apparently nonessential as live homozygote individuals are observed. These genes are functionally enriched for immunoglobulin domains, olfactory receptors, and MHC classes (FDR = 2.06 × 10<sup>−22</sup>, 2.06 × 10<sup>−22</sup>, 7.01 × 10<sup>−6</sup>, respectively). We also demonstrate that deletions are enriched for health and fertility related quantitative trait loci (2-and 1.5-fold enrichment, Fisher’s <em>P</em>-value = 8.91 × 10<sup>−10</sup> and 7.4 × 10<sup>−11</sup>, respectively). Finally, we identified and confirmed the breakpoint of a ∼525 KB deletion on Chr23:12,291,761-12,817,087 (overlapping BTBD9, GLO1 and DNAH8), causing stillbirth in Nordic Red Cattle.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;dairy cattle&quot;">dairy cattle</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;structural variants&quot;">structural variants</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole genome sequence&quot;">whole genome sequence</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;population genetics&quot;">population genetics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="loss-of-function">loss-of-function</a></div>                            <h2 scrollto-destination=112482664 id="112482664" class="section-title" >1. Introduction</h2>
<p>Embryonic lethality has become a challenge to cattle breeders, especially for dairy cattle where a limited number of bulls were extensively used in breeding for fast genetic progress in economic traits like milk and protein yield.<a reveal-id="dsx037-B1" data-open="dsx037-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> An estimated yearly loss of ∼$10.74 million is attributed to known recessive lethals in four dairy cattle breeds from USA only, where Holstein accounts for ∼70% of the total losses, followed by Jersey, Brown Swiss, and Ayrshire.<a reveal-id="dsx037-B2" data-open="dsx037-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Hence, understanding the genomic architecture of cattle populations is important, now more than ever, for optimizing genetic gain while constraining negative impact of deleterious mutations responsible for genetic defects and inbreeding depression.</p><p>Unlike single nucleotide polymorphism (SNP) and small insertion or deletion (indel), structural variants (SVs), i.e. DNA alterations larger than 50 base pairs (bp) that include insertions, deletions, duplications, inversions, and translocations,<a reveal-id="dsx037-B3" data-open="dsx037-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> are the least explored polymorphisms in cattle. SVs contribute substantially to phenotypic variations and have a wide-spectrum of impact ranging from beneficial to lethal in both humans<a reveal-id="dsx037-B3" data-open="dsx037-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><a reveal-id="dsx037-B4" data-open="dsx037-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> and animals.<a reveal-id="dsx037-B5" data-open="dsx037-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> The phenotypic impact of SVs in cattle is well evident from numerous studies. For example, Xu et al.<a reveal-id="dsx037-B6" data-open="dsx037-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> showed that a combination of SNPs with SVs could explain additional genetic variance underlying milk production traits, while Charlier et al.,<a reveal-id="dsx037-B7" data-open="dsx037-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Schutz et al.,<a reveal-id="dsx037-B8" data-open="dsx037-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> and Kadri et al.,<a reveal-id="dsx037-B9" data-open="dsx037-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> showed the lethal effect of large deletions in dairy cattle. Furthermore, a ∼525 KB deletion on chromosome 23 is reported to be associated with stillbirth in Nordic Red Cattle.<a reveal-id="dsx037-B10" data-open="dsx037-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a></p><p>Earlier SV studies on cattle were mostly SNP-array based, such as array-comparative genomic hybridization,<a reveal-id="dsx037-B11" data-open="dsx037-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> 50 K BovineSNP50 BeadChip<a reveal-id="dsx037-B12" data-open="dsx037-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> or 777 K BovineHD BeadChip (BovineHD chip)<a reveal-id="dsx037-B13" data-open="dsx037-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> based. But, using these approaches a substantial portion of the genome could not be explored and breakpoint resolution is still an issue.<a reveal-id="dsx037-B14" data-open="dsx037-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> However, whole-genome sequence (WGS) based techniques could improve resolution as well as power to capture SVs in a wide size and frequency spectrum.<a reveal-id="dsx037-B14" data-open="dsx037-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> For example, majority of the novel SVs in humans<a reveal-id="dsx037-B15" data-open="dsx037-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsx037-B16" data-open="dsx037-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> and mouse<a reveal-id="dsx037-B17" data-open="dsx037-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> were detected using WGS approaches. Besides, breakpoint sequences could also be assembled with high accuracy from sequencing reads,<a reveal-id="dsx037-B18" data-open="dsx037-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> which are necessary for elucidating the mechanisms underlying SV formation.<a reveal-id="dsx037-B19" data-open="dsx037-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a></p><p>In the advent of next-generation sequencing (NGS) techniques, hundreds of cattle (bull or cow) genomes were re-sequenced in collaborative initiative such as 1000 Bull Genomes Project (1KBGP)<a reveal-id="dsx037-B20" data-open="dsx037-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> (and other independent projects, e.g.<a reveal-id="dsx037-B21" data-open="dsx037-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a><sup>,</sup><a reveal-id="dsx037-B22" data-open="dsx037-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a>) to build a comprehensive database of sequence variants, mainly SNPs and indels. This NGS data provides a unique opportunity to study SVs in cattle. However, few studies<a reveal-id="dsx037-B23" data-open="dsx037-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsx037-B24" data-open="dsx037-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> utilized these (and/or other) NGS resources so far for studying SVs in cattle.</p><p>Therefore, in this study we scanned the WGSs of 175 cattle from three dairy breeds, namely Holstein, Jersey, and Nordic Red Cattle, to discover large deletions segregating in the population, and analyse their population-genetic properties. In particular, we focused on understating the population diversity, stratification, and plausible functional effects. We also explored the probable mechanisms of SV formation for a set of breakpoint-resolved deletions.</p>                            <h2 scrollto-destination=112482670 id="112482670" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=112482671 id="112482671" class="section-title" >2.1. Animal samples and ethics</h3>
<p>This study was performed on WGS of 175 dairy cattle from three breeds, e.g. 67 Holstein, 27 Jersey, and 81 Nordic Red Cattle. The sample included 7 Holstein cows and 168 bulls from these three breeds−144 animals from Run 5 of 1KBGP<a reveal-id="dsx037-B20" data-open="dsx037-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> and 31 animals from Nordic sequence data.<a reveal-id="dsx037-B21" data-open="dsx037-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> Genome sequences were generated using Illumina paired-end sequencing to an average coverage of 10-fold.</p><p>Here, we did not include any experimentation on animals and only dealt with analysis-ready WGS data; hence, no ethical approval was required.</p>                            <h3 scrollto-destination=112482674 id="112482674" class="section-title" >2.2. Sequence alignment to reference genome and SNPs/indels calling</h3>
<p>Raw sequencing reads were filtered and ‘FASTQ’ files were aligned to bovine reference genome assembly ‘UMD3.1’ using ‘BWA’ software<a reveal-id="dsx037-B25" data-open="dsx037-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> to produce BAM files for subsequent variant calling. In 1KBGP, SNPs and indels were called using ‘SAMtools 0.1.18 mpileup’ software,<a reveal-id="dsx037-B26" data-open="dsx037-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> while ‘GATK v1.6’ software<a reveal-id="dsx037-B27" data-open="dsx037-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> was used for Nordic WGS data (detailed method in<a reveal-id="dsx037-B20" data-open="dsx037-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> and,<a reveal-id="dsx037-B21" data-open="dsx037-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> respectively). For all the analysis, bovine genome assembly ‘UMD3.1’ was used as the reference genome. </p>                            <h3 scrollto-destination=112482676 id="112482676" class="section-title" >2.3. Discovery and genotyping of deletions</h3>
<p>SVs can be detected from NGS data based on sequence signatures such as (discordant) read-pair (RP), split-read (SP), and read-depth (RD), as well as <em>de novo</em> assembly of reads.<a reveal-id="dsx037-B14" data-open="dsx037-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> However, approaches based on only one sequence signature could be constrained by high false discovery rate (FDR),<a reveal-id="dsx037-B28" data-open="dsx037-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> hence we employed a population scale SV detection method called ‘Genome STRucture in Populations (Genome STRiP)’<a reveal-id="dsx037-B28" data-open="dsx037-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a>—which leverages technical (e.g. RP and RD signals) and population-level sequence features (e.g. coherence around shared alleles, and heterogeneity of evidentiary sequences in different genomes) for accurate discovery of deletions, and determines genotype (allelic state) of each locus from RD using a Gaussian mixture model.</p>                            <h4 scrollto-destination=112482678 id="112482678" class="section-title" >2.3.1. Genome STRiP</h4>
<p>For deletion discovery and genotyping ‘Genome STRiP’ software version 2.00.1678<a reveal-id="dsx037-B28" data-open="dsx037-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> was used. Following the documentation, we built a custom reference metadata bundle for cattle samples that includes alignability mask, copy-number mask (CN2 mask), ploidy map, gender map. Alignability mask represents sites on the reference genome that are uniquely alignable by sequence read of a certain length (readLength). Our WGS data was a mixture of different ‘Illumina’ paired-end reads ranging from 90 to 101 bp (Q1 = 90, median = 100, and Q3 = 100), hence genome alignability mask was prepared with readLength value of 90 using ‘ComputeGenomeMask’ utility from <em>Genome STRiP</em>. Copy-number mask (CN2 mask), i.e. regions on the reference genome unlikely to be copy-number variable in most individuals, was produced for the bovine assembly <em>UMD3.1</em> excluding sex chromosome X, unplaced contigs, and repeat sequences (retrieved from <em>RepeatMasker</em> track of <em>UCSC Table Browser</em>,<a reveal-id="dsx037-B29" data-open="dsx037-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> accessed on 4 July 2016).</p>                            <h4 scrollto-destination=112482680 id="112482680" class="section-title" >2.3.2. Pre-processing, deletion discovery and genotyping</h4>
<p>We ran the preprocessing Queue script (dry run) to emit all the commands, prepared bash scripts to run in <em>Portable Batch System</em> job scheduler, and executed these commands proving 175 BAM files (one for each sample) as input.</p><p>Large deletions (100 bp ≤ size ≤ 1 MB) were discovered and filtered using <em>SVDiscovery Queue</em> script. Discovered sites were filtered (default filters) if (i) the site contained too high or too low read pileup, (ii) RPs spacing was inconsistent with a single segregating deletion, (iii) RD and RP evidences were inconsistent across samples, (iv) RD differences were not significant, and (v) RP evidence was thinly distributed across samples (Genome STRiP Tutorial—GATK Workshop 2013, <a class="link link-uri" href="http://software.broadinstitute.org/software/genomestrip/workshop-presentations" target="_blank">http://software.broadinstitute.org/software/genomestrip/workshop-presentations</a>, accessed on 26 August 2016).</p><p>All passed sites were genotyped by <em>SVGenotyper</em> with default parameters. Genotyped deletion calls were then filtered based on following criteria, e.g. (i) sites with excess number of heterozygote calls (inbreeding coefficient ≤ −0.15), (ii) non-variant site based on genotype likelihood (parameter: non-variance score ≥ 13.0), (iii) sites with too low or too high RD (parameter: 0.5 ≥ GSM1 ≥ 2.0), (iv) sites with less than 30% uniquely alignable bases, (v) potential duplicate of another site (parameter: duplicateOverlapThreshold 0.5 and duplicateScoreThreshold ≥ 0.0), (vi) start/end position of a deletion call within 150 bp of assembly gap, (vii) all samples homozygous for reference allele (95% CI), and (viii) sites with ≥ 10% missing genotype.</p>                            <h3 scrollto-destination=112482684 id="112482684" class="section-title" >2.4. Validation of deletions</h3>
                            <h4 scrollto-destination=112482685 id="112482685" class="section-title" >2.4.1. Validation using 777K BovineHD BeadChip intensity data</h4>
<p>We validated deletion calls using 777K BovineHD BeadChip (Illumina, San Diego, CA, USA) intensity data on 26 Holstein samples that were both WGS and 777K chip typed. We calculated FDR for the deletion call-set using <em>IntensityRankSum (IRS)</em> test implemented in <em>Genome STRiP</em>. Intensity file was prepared from raw chip intensity data following the guideline for IRS test. Overall FDR for the call-set was calculated as two times the fraction of sites with IRS <em>P-</em>value ≥ 0.5 (i.e. sites with IRS <em>P-</em>value ≥ 0.5 to the sites with valid <em>P</em>-value). Details of IRS test could be found in.<a reveal-id="dsx037-B15" data-open="dsx037-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsx037-B28" data-open="dsx037-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a></p>                            <h4 scrollto-destination=112482687 id="112482687" class="section-title" >2.4.2. Validation by targeted assembly of breakpoint</h4>
<p>Targeted iterative graph routing assembler (<em>TIGRA-0.4.3</em>) software<a reveal-id="dsx037-B18" data-open="dsx037-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> was used, with default parameters, for assembling deletion breakpoint sequences from a set of randomly selected deletions along with three previously known deletions segregating in the study populations. <em>TIGRA</em> extracted all reads mapped to 500 bp upstream and 50 bp downstream of start coordinate, and 50 bp upstream and 500 bp downstream of end coordinate of a given deletion; and reads were then assembled iteratively using <em>de Bruijn</em> graph assembler with multiple k-mers (e.g. 15 bp followed by 25 bp). We aligned the assembled contigs to <em>UMD3.1</em> using <em>Cow BLAT Search</em><a reveal-id="dsx037-B30" data-open="dsx037-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> from <em>UCSC Genome Browser</em> (<a class="link link-uri" href="https://genome.ucsc.edu/cgi-bin/hgBlat" target="_blank">https://genome.ucsc.edu/cgi-bin/hgBlat</a>) to visualize and infer breakpoints from the alignments.</p>                            <h4 scrollto-destination=112482689 id="112482689" class="section-title" >2.4.3. Validation by PCR and amplicon sequencing</h4>
<p>We validated a previously reported ∼525 KB deletion segregating in Nordic Red Cattle<a reveal-id="dsx037-B10" data-open="dsx037-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> using PCR and amplicon sequencing. Genomic DNA was extracted as described previously by Miller et al.<a reveal-id="dsx037-B31" data-open="dsx037-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> from semen sample of two bulls carrying the deletion and two non-carriers. The PCR reaction was done with the <em>DyNAzyme II DNA Polymerase</em> (Thermo Fisher, MA, US) in a 30 µl volume of 1× PCR buffer, 0.2 mM dNTPs, 10 pmol primer mix (forward primer: 5′- AAGCCACCACAATGAGAAGC -3′ and reverse primer: 5′- TTTGGGGTAGGAGAAGTAGGG -3′) and 50 ng of genomic DNA. The cycling conditions were the following: (i) an initial denaturation at 95 °C for 3 min, (ii) 35 cycles of 30 s denaturation (94 °C), 30 s hybridization (65.2 °C), 30 s elongation (72 °C), and a final 3 min elongation (72 °C). PCR products were separated on a 2% agarose gel, purified and directly sequenced using the <em>BigDye Terminator Cycle Sequencing Kit</em> (Applied Biosystems, CA, US). Electrophoresis of sequencing reactions was performed on ‘3500xL Genetic Analyzers’ (Applied Biosystems, CA, US), and sequences were visualized with <em>Sequencher 5.4.6</em> (Gene Codes Corporation, MI, USA). A 977 bp control amplification, with a primer pair within the deletion (forward primer: 5′- CCCAATGCAAAATCACAAAA -3′ and reverse primer: 5′- CCAGAAAAGCTACACTTGAACTGA -3′), was performed using the same reaction conditions as above except hybridization was performed at 59.8 °C.</p>                            <h3 scrollto-destination=112482691 id="112482691" class="section-title" >2.5. Analysis of population genetic properties</h3>
<p>The population genetic properties of deletions, among the three breeds, were studied in terms of population diversity, population structure, and population differentiation. Population diversity was calculated using ‘VariantsPerSampleAnnotator’ from ‘Genome STRiP’ software, which provides distribution of variants across samples and populations. We performed principal component analysis (PCA) using <em>PLINK</em> (v1.90p) software<a reveal-id="dsx037-B32" data-open="dsx037-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> to distinguish three cattle breeds (details in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/supplementary_figures_and_tables_dsx037.pdf?Expires=2147483647&Signature=N8ekU-AB4-5zVt266s0FuDtj2aJWL~xy4OoOLYvUCqG5eWwdQ1T~hx3fJgruz9mUs~-gic4Ikx0v9SegHmwGCPkze9Kv68VZQdX55etEoJn~THklkmmMjgXPdT4Z6hbCYjDSZoUjw6h46V~h2D~93JPIAonauVegy07VuTGu1ULLUzb3SmNpuoCGtu9mnZYLC2~xIB~SmGTnvLejq8R~UU8F1OuEAfeFu0JRRyNhR50w4KNeh5IEYiHJO29zFin1eQ9YPexYjNL0gSk8XDfCde2m6Ydc8KivHx3eS2U3B~lKJkhpoI56Uel-YQr9K12aYJzJm90N~uuKou0aE8X90w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Material</a></span>). We calculated <em>V</em><sub>ST</sub><a reveal-id="dsx037-B33" data-open="dsx037-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a>—a population stratification measure of SVs (highly correlated with Wright’s fixation index, <em>F</em><sub>ST</sub><a reveal-id="dsx037-B34" data-open="dsx037-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a>), for each deletion locus using variant allele frequency (VAF) and genotypes from pairwise comparison of one breed with the rest, such as Holstein vs Jersey + Nordic Red Cattle, and vice versa.</p>                            <h3 scrollto-destination=112482693 id="112482693" class="section-title" >2.6. Functional annotation and enrichment analysis</h3>
<p>Functional annotation of deletions were performed using ‘Variant Effect Predictor (VEP-87<em>)’</em> software,<a reveal-id="dsx037-B35" data-open="dsx037-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and enrichment of protein domains (<em>InterPro</em><a reveal-id="dsx037-B36" data-open="dsx037-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> and <em>Pfam</em><a reveal-id="dsx037-B37" data-open="dsx037-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a>) and pathways (KEGG<a reveal-id="dsx037-B38" data-open="dsx037-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a>) were analysed using ‘STRING-v10 database’.<a reveal-id="dsx037-B39" data-open="dsx037-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a></p><p>Selective constraints on genes were measured from the ratio of non-synonymous (<em>dN</em>) to synonymous (<em>dS</em>) substitution rate, i.e. <em>dN/dS</em> ratio, between cow-mouse 1-to-1 orthologues downloaded from <em>Ensembl</em> database<a reveal-id="dsx037-B40" data-open="dsx037-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> (release 87, last accessed on 21 February 2017) using <em>BioMart</em>.<a reveal-id="dsx037-B41" data-open="dsx037-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Here we analysed whether <em>dN/dS</em> of genes overlapping deletions are higher (i.e. less constrained) than that of mouse lethal genes (from Dickinson et al.<a reveal-id="dsx037-B42" data-open="dsx037-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a>) using Wilcoxson test. Reported causal genes for cattle were also retrieved from OMIA database (<a class="link link-uri" href="http://omia.angis.org.au/" target="_blank">http://omia.angis.org.au/</a>, last accessed on 10 May 2016) for <em>dN/dS</em> comparison.</p><p>We retrieved cattle quantitative trait loci (QTL) from <em>QTLdb</em> database<a reveal-id="dsx037-B43" data-open="dsx037-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> (release 31; accessed on 6 January 2017); autosomal QTL from Holstein, Jersey, Nordic Red Cattle and Ayrshire, associated to any of the six trait classes, e.g. ‘Reproduction’, ‘Milk’, ‘Production’, ‘Exterior’, ‘Meat and Carcass’, and ‘Health’, were considered for QTL enrichment analysis. We calculated fold enrichment for a trait, such as for Health related QTL: (No. of Health QTL on Deletions ∕ Total QTL on deletions) / (Total Health QTL ∕ Total QTL in the dataset), and statistical significance using ‘Fisher’s exact test’ (two sided).</p>                            <h3 scrollto-destination=112482697 id="112482697" class="section-title" >2.7. Data manipulation, visualization, and statistical analysis</h3>
<p>All statistical analyses and plots were generated in <em>RStudio</em> software<a reveal-id="dsx037-B44" data-open="dsx037-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> running <em>R</em> software version 3.3.2,<a reveal-id="dsx037-B45" data-open="dsx037-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> unless mentioned otherwise. <em>BEDTools</em> (v2.26.0) software<a reveal-id="dsx037-B46" data-open="dsx037-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> is used for identifying the overlap between deletion calls and other genomic features, such as, ‘UMD3.1<em>’</em> assembly gaps (from ‘UCSC Table Browser<em>’</em>), CNVs from <em>dbVar</em> database<a reveal-id="dsx037-B47" data-open="dsx037-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a>, three known deletions from<a reveal-id="dsx037-B7" data-open="dsx037-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx037-B9" data-open="dsx037-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsx037-B10" data-open="dsx037-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a>, QTL from <em>QTLdb</em>. <em>VCFtools</em> (v0.1.15) software and <em>PLINK</em> (v1.90p) software were used for analysing the VCF file.</p>                            <h2 scrollto-destination=112482699 id="112482699" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=112482700 id="112482700" class="section-title" >3.1. Discovery and genotyping of deletions</h3>
<p>Deletion discovery and genotyping were carried out using <em>Genome STRiP</em>. After filtering, we report 8,480 large deletions with genotypes in 67 Holsten, 27 Jersey, and 81 Nordic Red Cattle. The deletion size ranged from 199 bp to 773 KB with a mean of 4.5 KB (median = 1 KB), which is approximately 10 times smaller compared with 184 deletion-CNVs (mean = 44.5 KB, median = 7.7 KB) reported in a recent 777K BovineHD BeadChip (BovineHD chip) based study,<a reveal-id="dsx037-B13" data-open="dsx037-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> reflecting the resolution of our sequence-based calls. Only 18% of the deletion calls have overlap (≥1 bp) with previously reported bovine deletions (or CNV-loss) in the <em>dbVar</em> database (accessed on 27 January 2017), while remaining 82% are novel. However, ∼72% of our deletion regions remained unique when compared with all CNVs (gain or loss) and copy number variable regions in the database. Interestingly, majority (∼80%) of these overlapping regions are from an earlier WGS-based study, where genome sequences of 27 Holstein, 17 Montbéliarde, and 18 Normande bulls were analysed.<a reveal-id="dsx037-B23" data-open="dsx037-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Nonetheless, we were able to broaden the accessible deletion size-range, more importantly towards smaller one unascertainable by usual SNP-array based approaches. We also report high quality genotypes for all the 8,480 deletions. Apparently, there are more low frequency variants than that of high frequency one, and the frequency distribution is very similar to humans<a reveal-id="dsx037-B16" data-open="dsx037-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> (<a reveal-id="dsx037-F1" data-open="dsx037-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). </p>                        <a id="112482703" scrollto-destination="112482703"></a>
<div id="dsx037-F1" data-id="dsx037-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/m_dsx037f1.png?Expires=2147483647&Signature=Ao8vRoyOfJNNWd4QoOHdZl-uZftN0MPbFd~n6hC-cHlAAgrF0CpSgp09hn3r~FkdAUmcx5TN5v9PVE7NUTT-cxLCIx5ugynHXZXWsK0RrIo-UOYWP9KlY~CTtpdi6f-oKL3v-aexlMJFGTty1RQDenL-K3tIwU5LVAcBxqHu41cuIklduwgYMYfjX-2LI0Wn2rra1XG~UOThO2DMRoo2rcEr4FFEf1ayNiqPvQINvNYULTMxgISYt3K~z03vnEVgYvQgTNLRZd5tRxs3RbDHc~YVAvb~nKksBFqNXaX0dlQW4ry7sjwhA6IfJ7C3CTdHpWpJ1yOQBbBc26AJmJt5qg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of ascertained deletions relative to variant allele count. Here, VAF is expressed in terms of variant allele count. Deletions down to an allele count of 1 (VAF = 0.0026 and 0.0032, in cattle and humans, respectively) are also represented here. Human deletion calls by Mills et al.16 were downloaded from 1K Genomes Project FTP server (ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/pilot_data/paper_data_sets/companion_papers/mapping_structural_variation)." path-from-xml="dsx037f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482703/dsx037f1.tif" path-from-xml="dsx037f1.tif" target="_blank">View large</a><a section="112482703" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/dsx037f1.png?Expires=2147483647&Signature=KORYgLpaee6~RxZFOd7mtuM1j7B~RRx4z5gk4ZK050FbQ0spfSifWWxxswWbutnez-0LR5M6AB00ziyv~XfHdsjCnPWFNfkhHil3ARgOOdK8yRXo0TIz1FtxKSt5fqjbHKQhj-VeDog7JrBlvbkT60SEgUpw8BBsqWGACEcTqSSxIzeconNlOwfkXTpo7bBRO9LroEAIc-Z4jcAdP9tgOe3pwYrN3YE5pT8eSviMNX6ydhS~2zNZtNIAwmfuQUmReOI~GaYo0jP7jnPZe4ZubUGQuYQg5HCB8BZtVN48rz6FK7YbU1Nd9gbtGfpAvY2vy9YlH9vARHr4zV1tYLeSHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482703&ar=4107864&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx037f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of ascertained deletions relative to variant allele count. Here, VAF is expressed in terms of variant allele count. Deletions down to an allele count of 1 (VAF = 0.0026 and 0.0032, in cattle and humans, respectively) are also represented here. Human deletion calls by Mills et al.<a reveal-id="dsx037-B16" data-open="dsx037-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> were downloaded from 1K Genomes Project FTP server (<a class="link link-uri" href="http://ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/pilot_data/paper_data_sets/companion_papers/mapping_structural_variation" target="_blank">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/pilot_data/paper_data_sets/companion_papers/mapping_structural_variation</a>).</p></div></div><div content-id="dsx037-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/m_dsx037f1.png?Expires=2147483647&Signature=Ao8vRoyOfJNNWd4QoOHdZl-uZftN0MPbFd~n6hC-cHlAAgrF0CpSgp09hn3r~FkdAUmcx5TN5v9PVE7NUTT-cxLCIx5ugynHXZXWsK0RrIo-UOYWP9KlY~CTtpdi6f-oKL3v-aexlMJFGTty1RQDenL-K3tIwU5LVAcBxqHu41cuIklduwgYMYfjX-2LI0Wn2rra1XG~UOThO2DMRoo2rcEr4FFEf1ayNiqPvQINvNYULTMxgISYt3K~z03vnEVgYvQgTNLRZd5tRxs3RbDHc~YVAvb~nKksBFqNXaX0dlQW4ry7sjwhA6IfJ7C3CTdHpWpJ1yOQBbBc26AJmJt5qg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of ascertained deletions relative to variant allele count. Here, VAF is expressed in terms of variant allele count. Deletions down to an allele count of 1 (VAF = 0.0026 and 0.0032, in cattle and humans, respectively) are also represented here. Human deletion calls by Mills et al.16 were downloaded from 1K Genomes Project FTP server (ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/pilot_data/paper_data_sets/companion_papers/mapping_structural_variation)." path-from-xml="dsx037f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482703/dsx037f1.tif" path-from-xml="dsx037f1.tif" target="_blank">View large</a><a section="112482703" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/dsx037f1.png?Expires=2147483647&Signature=KORYgLpaee6~RxZFOd7mtuM1j7B~RRx4z5gk4ZK050FbQ0spfSifWWxxswWbutnez-0LR5M6AB00ziyv~XfHdsjCnPWFNfkhHil3ARgOOdK8yRXo0TIz1FtxKSt5fqjbHKQhj-VeDog7JrBlvbkT60SEgUpw8BBsqWGACEcTqSSxIzeconNlOwfkXTpo7bBRO9LroEAIc-Z4jcAdP9tgOe3pwYrN3YE5pT8eSviMNX6ydhS~2zNZtNIAwmfuQUmReOI~GaYo0jP7jnPZe4ZubUGQuYQg5HCB8BZtVN48rz6FK7YbU1Nd9gbtGfpAvY2vy9YlH9vARHr4zV1tYLeSHw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482703&ar=4107864&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx037f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of ascertained deletions relative to variant allele count. Here, VAF is expressed in terms of variant allele count. Deletions down to an allele count of 1 (VAF = 0.0026 and 0.0032, in cattle and humans, respectively) are also represented here. Human deletion calls by Mills et al.<a reveal-id="dsx037-B16" data-open="dsx037-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> were downloaded from 1K Genomes Project FTP server (<a class="link link-uri" href="http://ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/pilot_data/paper_data_sets/companion_papers/mapping_structural_variation" target="_blank">ftp://ftp.1000genomes.ebi.ac.uk/vol1/ftp/pilot_data/paper_data_sets/companion_papers/mapping_structural_variation</a>).</p></div></div><p>Previous NGS-based studies on cattle were mostly limited to SV discovery, while copy-number states (genotypes) were inferred using BovineHD chip,<a reveal-id="dsx037-B24" data-open="dsx037-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> or custom SNPs array.<a reveal-id="dsx037-B23" data-open="dsx037-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> However, in this study we estimated the copy-number at each deletion locus (per sample) from RD within the region using a constrained Gaussian mixture model with three classes, e.g. copy-number zero (i.e. homozygous deletion), one (i.e. heterozygous deletion) and two (i.e. homozygous reference). It is known from human studies that majority of the (bi-allelic) common SVs segregate on specific SNP haplotypes,<a reveal-id="dsx037-B48" data-open="dsx037-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a><sup>,</sup><a reveal-id="dsx037-B49" data-open="dsx037-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> which could be imputed with high accuracy.<a reveal-id="dsx037-B28" data-open="dsx037-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a><sup>,</sup><a reveal-id="dsx037-B50" data-open="dsx037-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> Thus, this approach has the potential, albeit with large reference, for accurate haplotype phasing and imputation of SVs to large cohorts of low-density chip-typed animals with no additional cost.</p>                            <h3 scrollto-destination=112482705 id="112482705" class="section-title" >3.2. Validation of deletions</h3>
<p>We validated the results using three approaches: (i) using BovineHD chip intensity data, (ii) breakpoint assembly and alignment, and (iii) PCR + sequencing of amplicons.</p>                            <h4 scrollto-destination=112482707 id="112482707" class="section-title" >3.2.1. BovineHD chip intensity</h4>
<p>We used 777K BovineHD chip intensity data of 26 Holstein animals, both chip-typed and sequenced, to validate the deletion calls using <em>Genome STRiP</em>’ <em>IRS</em> test. We had partial power to investigate all deletions due to the sparsity of the array-probes (one probe per ∼3.5 KB), and were underpowered to accurately verify small deletions (e.g. overlapping one-probe). Furthermore, we could only test a deletion for which at least one of the 26 samples had non-reference allele. Therefore, an estimate of FDR for the deletion call-set is provided here from the overall <em>P</em>-value distribution. In this approach, we were able to interrogate ∼8.3% of the total call, majority of which contain a single array-probe within the region (<a reveal-id="dsx037-T1" data-open="dsx037-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/s1_and_s2_dsx037.xls?Expires=2147483647&Signature=vhdyIqhZPLuViQrCa8hpkyySgekxCBcznuz0oqoUHlTNOr1XKPwZSVPRHMacIv8nDZign9ABA23sFAQO-npvrF48ltv8UdCVFGMod664flAKuArLtJ4UzZ8Q-z9glSbhkJOHUEIXbmXgJBGz31pNp4C-qGH~VxVFQGogjtVnvJggQjRqqZBCZECFO7eNPHkMkQITtkc2IuOOt4n80IUS2Se-6rlPhsTeVqIFX2XtlFVBxgyge20XNsdt~~lNk8vtX56dIqwc8Cpu9uKTv3rSVIw93AFbBjB8uynY6TwdianwvTuICtBKwAmBUWIPaRrrpG-kpawxEsVlbO1bkaob1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). We found that deletions overlapping only one array-probe had higher FDR (11.3%) compared with two or more probes. And finally, we showed that our deletion call-set had an overall FDR of 8.8%, which is within our chosen threshold of FDR ≤ 10%. <div content-id="dsx037-T1" class="table-modal"><div class="table-wrap"><div id="dsx037-T1" data-id="dsx037-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>FDR estimates of Genome STRiP’ deletion calls using 777K BovineHD BeadChip intensity data</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Array-probe Overlap </th><th><strong>A</strong><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><strong>B</strong><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong>FDR</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td><strong>One-probe</strong> </td><td>497 </td><td>28 </td><td>11.3% </td></tr><tr><td><strong>&gt;1 array-probe</strong> </td><td>206 </td><td>3 </td><td>2.9% </td></tr><tr><td><strong>&gt;2 array-probe</strong> </td><td>113 </td><td>1 </td><td>1.8% </td></tr><tr><td><strong>Overall</strong> </td><td><strong>703</strong> </td><td><strong>31</strong> </td><td><strong>8.8%</strong> </td></tr></tbody></table></div><div content-id="dsx037-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Array-probe Overlap </th><th><strong>A</strong><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><strong>B</strong><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong>FDR</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td><strong>One-probe</strong> </td><td>497 </td><td>28 </td><td>11.3% </td></tr><tr><td><strong>&gt;1 array-probe</strong> </td><td>206 </td><td>3 </td><td>2.9% </td></tr><tr><td><strong>&gt;2 array-probe</strong> </td><td>113 </td><td>1 </td><td>1.8% </td></tr><tr><td><strong>Overall</strong> </td><td><strong>703</strong> </td><td><strong>31</strong> </td><td><strong>8.8%</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup><strong>A,</strong> No. of sites with <em>P</em>-value.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup><strong>B,</strong> No. of sites with <em>P</em>-value ≥ 0.5</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>c</sup>FDR estimates were based on ‘Wilcoxson rank sum test’ using BovineHD chip intensity data of 26 Holstein animals. FDR was calculated as (<strong>B/A × 2 × 100</strong>).</p></div></div></div></div></div><div class="table-wrap"><div id="dsx037-T1" data-id="dsx037-T1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>FDR estimates of Genome STRiP’ deletion calls using 777K BovineHD BeadChip intensity data</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Array-probe Overlap </th><th><strong>A</strong><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><strong>B</strong><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong>FDR</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td><strong>One-probe</strong> </td><td>497 </td><td>28 </td><td>11.3% </td></tr><tr><td><strong>&gt;1 array-probe</strong> </td><td>206 </td><td>3 </td><td>2.9% </td></tr><tr><td><strong>&gt;2 array-probe</strong> </td><td>113 </td><td>1 </td><td>1.8% </td></tr><tr><td><strong>Overall</strong> </td><td><strong>703</strong> </td><td><strong>31</strong> </td><td><strong>8.8%</strong> </td></tr></tbody></table></div><div content-id="dsx037-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Array-probe Overlap </th><th><strong>A</strong><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><strong>B</strong><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong>FDR</strong><a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>c</sup></a> </th></tr></thead><tbody><tr><td><strong>One-probe</strong> </td><td>497 </td><td>28 </td><td>11.3% </td></tr><tr><td><strong>&gt;1 array-probe</strong> </td><td>206 </td><td>3 </td><td>2.9% </td></tr><tr><td><strong>&gt;2 array-probe</strong> </td><td>113 </td><td>1 </td><td>1.8% </td></tr><tr><td><strong>Overall</strong> </td><td><strong>703</strong> </td><td><strong>31</strong> </td><td><strong>8.8%</strong> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup><strong>A,</strong> No. of sites with <em>P</em>-value.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup><strong>B,</strong> No. of sites with <em>P</em>-value ≥ 0.5</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>c</sup>FDR estimates were based on ‘Wilcoxson rank sum test’ using BovineHD chip intensity data of 26 Holstein animals. FDR was calculated as (<strong>B/A × 2 × 100</strong>).</p></div></div></div></div></p>                            <h4 scrollto-destination=112482709 id="112482709" class="section-title" >3.2.2. Targeted breakpoint assembly</h4>
<p>We next validated three randomly chosen set of ten-deletions each by assembling breakpoint sequences using TIGRA<a reveal-id="dsx037-B18" data-open="dsx037-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a>: 10 deletions ≤ 500 bp, 10 deletions &gt; 500 bp but ≤ 1 KB, and 10 deletions with VAF ≤ 0.10. Out of the thirty, we successfully resolved breakpoints of 26 deletions (∼87% success rate) using a combination of TIGRA and BLAT search<a reveal-id="dsx037-B30" data-open="dsx037-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/s1_and_s2_dsx037.xls?Expires=2147483647&amp;Signature=vhdyIqhZPLuViQrCa8hpkyySgekxCBcznuz0oqoUHlTNOr1XKPwZSVPRHMacIv8nDZign9ABA23sFAQO-npvrF48ltv8UdCVFGMod664flAKuArLtJ4UzZ8Q-z9glSbhkJOHUEIXbmXgJBGz31pNp4C-qGH~VxVFQGogjtVnvJggQjRqqZBCZECFO7eNPHkMkQITtkc2IuOOt4n80IUS2Se-6rlPhsTeVqIFX2XtlFVBxgyge20XNsdt~~lNk8vtX56dIqwc8Cpu9uKTv3rSVIw93AFbBjB8uynY6TwdianwvTuICtBKwAmBUWIPaRrrpG-kpawxEsVlbO1bkaob1w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Additionally, we assembled breakpoints of three previously reported deletions that were also segregating in our study populations, such as a ∼662 KB deletion on chromosome 12 encompassing <em>RNASEH2B</em>, <em>GUCY1B2</em>, and <em>FAM124A</em>, a ∼3 KB deletion on chromosome 21 encompassing <em>FANCI</em>, and a ∼525 KB deletion on chromosome 23 encompassing <em>BTBD9, GLO1</em>, and <em>DNAH8</em>. Breakpoints of the former two deletions were previously reported in,<a reveal-id="dsx037-B7" data-open="dsx037-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a><sup>,</sup><a reveal-id="dsx037-B9" data-open="dsx037-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> which exactly matched with our predicted breakpoint sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/supplementary_figures_and_tables_dsx037.pdf?Expires=2147483647&amp;Signature=N8ekU-AB4-5zVt266s0FuDtj2aJWL~xy4OoOLYvUCqG5eWwdQ1T~hx3fJgruz9mUs~-gic4Ikx0v9SegHmwGCPkze9Kv68VZQdX55etEoJn~THklkmmMjgXPdT4Z6hbCYjDSZoUjw6h46V~h2D~93JPIAonauVegy07VuTGu1ULLUzb3SmNpuoCGtu9mnZYLC2~xIB~SmGTnvLejq8R~UU8F1OuEAfeFu0JRRyNhR50w4KNeh5IEYiHJO29zFin1eQ9YPexYjNL0gSk8XDfCde2m6Ydc8KivHx3eS2U3B~lKJkhpoI56Uel-YQr9K12aYJzJm90N~uuKou0aE8X90w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S1a–c and S2a–c</a></span>). Although for the later, we resolved breakpoint sequences in this study (<a reveal-id="dsx037-F2" data-open="dsx037-F2" class="link link-reveal link-table xref-fig">Fig. 2a and b</a>). Overall, the success rate of our deletion-breakpoint assembly was better than the reported success rate of TIGRA on similar sized read-length.<a reveal-id="dsx037-B18" data-open="dsx037-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> And <em>Genome STRiP</em>’s breakpoint predictions were on average within 20 bp of the validated breakpoint, which is within the tool’s reported estimate of 1–20 bp.<a reveal-id="dsx037-B28" data-open="dsx037-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a></p>                        <a id="112482711" scrollto-destination="112482711"></a>
<div id="dsx037-F2" data-id="dsx037-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/m_dsx037f2.png?Expires=2147483647&Signature=AY2Z8DgiRuPa2W6b4TkwTjf2vsNVGZ0Jl~vsLJ-LcCbfw7edqBsciA2T8DltrvH9IZkLyDV9b7~H~p2fe5KlmgS1g6YvGDUls7~sSuMwHNOr5~UZ392gipskZmnH6UbsESw4Z~Mev27jqdNvdLjKwSgNDMp0U9NaHnNKZqLDyEvm2nWV-fH5Q7GQYUVw4Z-PkuwEeiLohAn0conLrVxATXL3Wz1UHvDJv7q7kCsuTsO4Zd2~Uwip6YeRyu87L~WhU4hkQ9fnVIrdXv7PXOKThTlE0oH8qb1Fw64f7PZdBEflBmHKYoAU1vBMthVaoyGMN2ofE6E6~z1qn9pFGX77Og__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A ∼525-KB deletion on chromosome 23 discovered using Genome STRiP (a), and resolved breakpoint sequences from TIGRA and BLAT search (b). Shaded bases are a 5-bp microhomology at breakpoint junction. (This figure was drawn and modified using Inkscape version 0.91.)" path-from-xml="dsx037f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482711/dsx037f2.tif" path-from-xml="dsx037f2.tif" target="_blank">View large</a><a section="112482711" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/dsx037f2.png?Expires=2147483647&Signature=WgGQ7sb~22fPxnynkQIKgqj0zxztR5ifbe4ZIyN0hdR9q1wIngW5HgBfEbj3txqQBE5xvr-2UV7EoauLxRHrvvOtAkwQD25P8J4O2wxPrt6Lp~aZLgUvUvxclgTI4cA-NAJ4~m2YVmiRiNgh54RtNvlRrbhzwcH9a8vArfxv9hS8WZPMVfx2DdMcDN6GNF6vBvBk480h7856QWmRsUwhzFD~6bNGe-RUMqGFzaBH843OgOrhgoKqsczKLvRWL1aHpSj5acaXw3lS8bI5zYtRyWH3ernBkMDatPhd-C-LCwKRRRXoc~dKE~~9ayKOG9J5tlIXUVYPa1WM1bVYIkraiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482711&ar=4107864&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx037f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A ∼525-KB deletion on chromosome 23 discovered using <em>Genome STRiP</em> (a), and resolved breakpoint sequences from <em>TIGRA</em> and <em>BLAT</em> search (b). Shaded bases are a 5-bp microhomology at breakpoint junction. (This figure was drawn and modified using <em>Inkscape</em> version 0.91.)</p></div></div><div content-id="dsx037-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/m_dsx037f2.png?Expires=2147483647&Signature=AY2Z8DgiRuPa2W6b4TkwTjf2vsNVGZ0Jl~vsLJ-LcCbfw7edqBsciA2T8DltrvH9IZkLyDV9b7~H~p2fe5KlmgS1g6YvGDUls7~sSuMwHNOr5~UZ392gipskZmnH6UbsESw4Z~Mev27jqdNvdLjKwSgNDMp0U9NaHnNKZqLDyEvm2nWV-fH5Q7GQYUVw4Z-PkuwEeiLohAn0conLrVxATXL3Wz1UHvDJv7q7kCsuTsO4Zd2~Uwip6YeRyu87L~WhU4hkQ9fnVIrdXv7PXOKThTlE0oH8qb1Fw64f7PZdBEflBmHKYoAU1vBMthVaoyGMN2ofE6E6~z1qn9pFGX77Og__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A ∼525-KB deletion on chromosome 23 discovered using Genome STRiP (a), and resolved breakpoint sequences from TIGRA and BLAT search (b). Shaded bases are a 5-bp microhomology at breakpoint junction. (This figure was drawn and modified using Inkscape version 0.91.)" path-from-xml="dsx037f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482711/dsx037f2.tif" path-from-xml="dsx037f2.tif" target="_blank">View large</a><a section="112482711" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/dsx037f2.png?Expires=2147483647&Signature=WgGQ7sb~22fPxnynkQIKgqj0zxztR5ifbe4ZIyN0hdR9q1wIngW5HgBfEbj3txqQBE5xvr-2UV7EoauLxRHrvvOtAkwQD25P8J4O2wxPrt6Lp~aZLgUvUvxclgTI4cA-NAJ4~m2YVmiRiNgh54RtNvlRrbhzwcH9a8vArfxv9hS8WZPMVfx2DdMcDN6GNF6vBvBk480h7856QWmRsUwhzFD~6bNGe-RUMqGFzaBH843OgOrhgoKqsczKLvRWL1aHpSj5acaXw3lS8bI5zYtRyWH3ernBkMDatPhd-C-LCwKRRRXoc~dKE~~9ayKOG9J5tlIXUVYPa1WM1bVYIkraiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482711&ar=4107864&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx037f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A ∼525-KB deletion on chromosome 23 discovered using <em>Genome STRiP</em> (a), and resolved breakpoint sequences from <em>TIGRA</em> and <em>BLAT</em> search (b). Shaded bases are a 5-bp microhomology at breakpoint junction. (This figure was drawn and modified using <em>Inkscape</em> version 0.91.)</p></div></div>                            <h4 scrollto-destination=112482712 id="112482712" class="section-title" >3.2.3. PCR and amplicon sequencing</h4>
<p>We then experimentally validated breakpoints for ‘Chr23:12,291,761-12,817,087’ deletion, previously reported to be associated with stillbirth in Nordic Red Cattle.<a reveal-id="dsx037-B10" data-open="dsx037-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> Four animals were used for PCR validation: two carriers and two non-carriers. The breakpoint spanning PCR products of 359 bp were only observed in the carrier animals, while no amplicon was seen for non-carriers (<a reveal-id="dsx037-F3" data-open="dsx037-F3" class="link link-reveal link-table xref-fig">Fig. 3a</a>). The 359 bp amplicon was then sequenced, and exact breakpoint sequences were observed (<a reveal-id="dsx037-F3" data-open="dsx037-F3" class="link link-reveal link-table xref-fig">Fig. 3b</a>), thus confirming the breakpoint for this deletion. </p>                        <a id="112482714" scrollto-destination="112482714"></a>
<div id="dsx037-F3" data-id="dsx037-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/m_dsx037f3.png?Expires=2147483647&Signature=Jb0TaTjf~UPKe9W-~U3H1V6kS316kIqzIxm-S1UOGrqdNvZcUxuh-h-wlESmskez6TUN-WKdg3kWx37-85VztlOTVQsTYdNO5dGd2A~ZWC9f91Biu7WgOTLaEN9MNpEi3BFLtBm-sYoPim3WQHd~QfZ0CGX3Xwk2JUSxCFZwyrZ3FhGW3WamKABF5yvFmWcjIGz4YOElUdm62Tt4qMTD6yrR8bjH8OXR6VN0eXxbJmrJLkeAyLeYj~TWxlK0srzktJCqY9qi7YqhQ75lDF-MBhuy-8BgzmlAcsWsEBmr5Fm1mrA6s3nA-UcQq-hEach-iTo7~U~w0f30a7rtTlNMUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Experimental validation of the ∼525 KB deletion on chromosome 23. (a) PCR amplification across (left) and within the deletion (right) for two carrier (Δ/+) and two homozygous wild-type (+/+) animals. Water, negative control; M, molecular weight marker (GeneRuler 100 bp DNA ladder, Fermentas). (b) Sequence trace of the 359 bp amplicon bridging the breakpoint. Shaded bases are a 5-bp microhomology at breakpoint junction. (This figure was drawn and modified using Inkscape version 0.91.)" path-from-xml="dsx037f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482714/dsx037f3.tif" path-from-xml="dsx037f3.tif" target="_blank">View large</a><a section="112482714" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/dsx037f3.png?Expires=2147483647&Signature=2pmlodyzm4HOxx6dIgyB-nUIUCZ6-cs6RRamfxCnz-Zmq6fGhvB854lX~RhhOjaeS~kFxM42caA1QKsttjXZJDu-lLZaKAEJpy4fmLDkETfxVwFYDGff3S-mV9F4jg3XKFIEdCOImCJSAgToQjXAoD1C6vZM~7ttXVOgjhgzw5YFtil9ZUO7VqfFBFiq74m-NJevJKdEsMiK8Bt-t~f7mpRNUuxseW1qmRs5tTCMHX1P5cCXY64AdCULwsGGXaHXgWMNFAPFHyBixvvpUjOJ8t2kaYm3MdIjwwPscVvJbvObYJFLBJiSq3J8EPkO~4kCluN2QZcvUGXg6lP~lakxxg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482714&ar=4107864&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx037f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Experimental validation of the ∼525 KB deletion on chromosome 23. (a) PCR amplification across (left) and within the deletion (right) for two carrier (Δ/+) and two homozygous wild-type (+/+) animals. Water, negative control; M, molecular weight marker (GeneRuler 100 bp DNA ladder, Fermentas). (b) Sequence trace of the 359 bp amplicon bridging the breakpoint. Shaded bases are a 5-bp microhomology at breakpoint junction. (This figure was drawn and modified using <em>Inkscape</em> version 0.91.)</p></div></div><div content-id="dsx037-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/m_dsx037f3.png?Expires=2147483647&Signature=Jb0TaTjf~UPKe9W-~U3H1V6kS316kIqzIxm-S1UOGrqdNvZcUxuh-h-wlESmskez6TUN-WKdg3kWx37-85VztlOTVQsTYdNO5dGd2A~ZWC9f91Biu7WgOTLaEN9MNpEi3BFLtBm-sYoPim3WQHd~QfZ0CGX3Xwk2JUSxCFZwyrZ3FhGW3WamKABF5yvFmWcjIGz4YOElUdm62Tt4qMTD6yrR8bjH8OXR6VN0eXxbJmrJLkeAyLeYj~TWxlK0srzktJCqY9qi7YqhQ75lDF-MBhuy-8BgzmlAcsWsEBmr5Fm1mrA6s3nA-UcQq-hEach-iTo7~U~w0f30a7rtTlNMUA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Experimental validation of the ∼525 KB deletion on chromosome 23. (a) PCR amplification across (left) and within the deletion (right) for two carrier (Δ/+) and two homozygous wild-type (+/+) animals. Water, negative control; M, molecular weight marker (GeneRuler 100 bp DNA ladder, Fermentas). (b) Sequence trace of the 359 bp amplicon bridging the breakpoint. Shaded bases are a 5-bp microhomology at breakpoint junction. (This figure was drawn and modified using Inkscape version 0.91.)" path-from-xml="dsx037f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482714/dsx037f3.tif" path-from-xml="dsx037f3.tif" target="_blank">View large</a><a section="112482714" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/dsx037f3.png?Expires=2147483647&Signature=2pmlodyzm4HOxx6dIgyB-nUIUCZ6-cs6RRamfxCnz-Zmq6fGhvB854lX~RhhOjaeS~kFxM42caA1QKsttjXZJDu-lLZaKAEJpy4fmLDkETfxVwFYDGff3S-mV9F4jg3XKFIEdCOImCJSAgToQjXAoD1C6vZM~7ttXVOgjhgzw5YFtil9ZUO7VqfFBFiq74m-NJevJKdEsMiK8Bt-t~f7mpRNUuxseW1qmRs5tTCMHX1P5cCXY64AdCULwsGGXaHXgWMNFAPFHyBixvvpUjOJ8t2kaYm3MdIjwwPscVvJbvObYJFLBJiSq3J8EPkO~4kCluN2QZcvUGXg6lP~lakxxg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482714&ar=4107864&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx037f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Experimental validation of the ∼525 KB deletion on chromosome 23. (a) PCR amplification across (left) and within the deletion (right) for two carrier (Δ/+) and two homozygous wild-type (+/+) animals. Water, negative control; M, molecular weight marker (GeneRuler 100 bp DNA ladder, Fermentas). (b) Sequence trace of the 359 bp amplicon bridging the breakpoint. Shaded bases are a 5-bp microhomology at breakpoint junction. (This figure was drawn and modified using <em>Inkscape</em> version 0.91.)</p></div></div>                            <h3 scrollto-destination=112482715 id="112482715" class="section-title" >3.3. Population genetic properties of deletions</h3>
                            <h4 scrollto-destination=112482716 id="112482716" class="section-title" >3.3.1. Population diversity</h4>
<p>We explored population diversity among the three dairy cattle breeds from per-individual deletion-heterozygosity and homozygosity. We found that individuals from Nordic Red Cattle exhibits 3.5 and 6.4% higher deletion-heterozygosity than in Holstein and Jersey, respectively. Median numbers of heterozygote-deletion were 1,272, 1,229 and 1,196, in Nordic Red Cattle, Holstein, and Jersey, respectively (<a reveal-id="dsx037-F4" data-open="dsx037-F4" class="link link-reveal link-table xref-fig">Fig. 4a</a>). In contrast to heterozygosity, Jersey animals showed highest levels of deletion-homozygosity, followed by Holstein (<a reveal-id="dsx037-F4" data-open="dsx037-F4" class="link link-reveal link-table xref-fig">Fig. 4b</a>). Similar estimates of genetic diversity were also reported for these breeds in a SNP heterozygosity and runs-of-homozygosity analysis—where Jersey exhibited lowest (genome-wide) average nucleotide diversity (and higher number/size of runs-of-homozygosity) followed by Holstein and Nordic Red Cattle.<a reveal-id="dsx037-B51" data-open="dsx037-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> These differences could be understood from the current effective population size (<em>N<sub>e</sub></em>) of these breeds, e.g. <em>N<sub>e</sub></em> of Jersey, Holstein, and Nordic Red Cattle are 73, 99, and 106, respectively<a reveal-id="dsx037-B52" data-open="dsx037-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a>; this entailed higher diversity in Nordic Red Cattle, and Holstein, than in Jersey. From singletons estimate it is also evident that Nordic Red Cattle has more rare deletions compared with Holstein and Jersey (<a reveal-id="dsx037-F4" data-open="dsx037-F4" class="link link-reveal link-table xref-fig">Fig. 4c</a>); partly could be due to incorrect ascertainment of rare ones. </p>                        <a id="112482718" scrollto-destination="112482718"></a>
<div id="dsx037-F4" data-id="dsx037-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/m_dsx037f4.png?Expires=2147483647&Signature=WpvBU9CgmoTakSCvTKzm9viXqpz7ghq1AlGUd8qeQcRnUYM7Q~ZJarYdFbgFIpsvuX3M5IebSGfH7KEDSMJBMeI0lWuj~si6al3ljQc-oIFQOBIrPtMKFlpZR~SO8d~0X8sCVWnsHwYqi0ycVIH9DZ-FsISP~t4astfVBmmCnzWxBncI-DKHm17cI1orMc5kJvs~b3N~N6l8plMOlpMmPLmwVVjMGIVYu2nl~3Il7cqu8r8FRBcpRjEUaiVqpNLH4ul3fFLXKyicqtzXEIpfYVqiRfAna2rWCQc4EK97lpOiXez3R-Ggk1e2HXJZm8lPRbWX-Lg441~jPR~6wc8nRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Population diversity. (a) Heterozygous deletions per genome. (b) Homozygous deletions per genome. (c) Singletons per genome. Only high-confidence genotype calls are included. The y-axis in (a–c) represents the number of heterozygous, homozygous, and singleton deletions per genome, respectively. HOL, Holstein; JER, Jersey; RDC, Nordic Red Cattle." path-from-xml="dsx037f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482718/dsx037f4.tif" path-from-xml="dsx037f4.tif" target="_blank">View large</a><a section="112482718" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/dsx037f4.png?Expires=2147483647&Signature=s5BQIXIPp-qe6Cjv0T-xqNes7cQpvv9yhIk~hxp1u4APsl75pHIl4iuEx6wpe7u9TCCIwUS0SFte2Q2tzbO9jtd29N~6Gp0p7AGgtn6Ot6nmqkgg4PBvCDZwoW3~8OS8c-zzgQ~WuT53WFUrJ4pefsm7RALPMzICD34DMfKtc4SebtU34CX6ymrpKNqPDGtS5IwX21kmCRT5HDU36kylBIBnWJIGM7271EO0qhwr1Os7SQyak4y0hMjPrGI9jjPOBIRYARqwm7gIL2yoTWPF5l8ItTrAqA6iuyRpZjFWwzs9ZOJDGGQP~oFzDbmKHmRBnGdZ~7wQIIa0158q8KtPuQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482718&ar=4107864&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx037f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Population diversity. (a) Heterozygous deletions per genome. (b) Homozygous deletions per genome. (c) Singletons per genome. Only high-confidence genotype calls are included. The <em>y-axis</em> in (a–c) represents the number of heterozygous, homozygous, and singleton deletions per genome, respectively. HOL, Holstein; JER, Jersey; RDC, Nordic Red Cattle.</p></div></div><div content-id="dsx037-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/m_dsx037f4.png?Expires=2147483647&Signature=WpvBU9CgmoTakSCvTKzm9viXqpz7ghq1AlGUd8qeQcRnUYM7Q~ZJarYdFbgFIpsvuX3M5IebSGfH7KEDSMJBMeI0lWuj~si6al3ljQc-oIFQOBIrPtMKFlpZR~SO8d~0X8sCVWnsHwYqi0ycVIH9DZ-FsISP~t4astfVBmmCnzWxBncI-DKHm17cI1orMc5kJvs~b3N~N6l8plMOlpMmPLmwVVjMGIVYu2nl~3Il7cqu8r8FRBcpRjEUaiVqpNLH4ul3fFLXKyicqtzXEIpfYVqiRfAna2rWCQc4EK97lpOiXez3R-Ggk1e2HXJZm8lPRbWX-Lg441~jPR~6wc8nRA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Population diversity. (a) Heterozygous deletions per genome. (b) Homozygous deletions per genome. (c) Singletons per genome. Only high-confidence genotype calls are included. The y-axis in (a–c) represents the number of heterozygous, homozygous, and singleton deletions per genome, respectively. HOL, Holstein; JER, Jersey; RDC, Nordic Red Cattle." path-from-xml="dsx037f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482718/dsx037f4.tif" path-from-xml="dsx037f4.tif" target="_blank">View large</a><a section="112482718" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/dsx037f4.png?Expires=2147483647&Signature=s5BQIXIPp-qe6Cjv0T-xqNes7cQpvv9yhIk~hxp1u4APsl75pHIl4iuEx6wpe7u9TCCIwUS0SFte2Q2tzbO9jtd29N~6Gp0p7AGgtn6Ot6nmqkgg4PBvCDZwoW3~8OS8c-zzgQ~WuT53WFUrJ4pefsm7RALPMzICD34DMfKtc4SebtU34CX6ymrpKNqPDGtS5IwX21kmCRT5HDU36kylBIBnWJIGM7271EO0qhwr1Os7SQyak4y0hMjPrGI9jjPOBIRYARqwm7gIL2yoTWPF5l8ItTrAqA6iuyRpZjFWwzs9ZOJDGGQP~oFzDbmKHmRBnGdZ~7wQIIa0158q8KtPuQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482718&ar=4107864&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx037f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Population diversity. (a) Heterozygous deletions per genome. (b) Homozygous deletions per genome. (c) Singletons per genome. Only high-confidence genotype calls are included. The <em>y-axis</em> in (a–c) represents the number of heterozygous, homozygous, and singleton deletions per genome, respectively. HOL, Holstein; JER, Jersey; RDC, Nordic Red Cattle.</p></div></div>                            <h4 scrollto-destination=112482719 id="112482719" class="section-title" >3.3.2. Principal component analysis</h4>
<p>We performed PCA using the deletion genotypes of the samples. Around 6 K deletions with VAF between 0.02 and 0.90 were used in the analysis. For comparison, we also performed PCA on ∼168 K bi-allelic SNPs randomly selected from 29 autosomes of the same individuals. The first two principal components (PCs) from both deletion and SNP-based PCA clearly distinguished the three breeds, and jointly explained 20 and 16.2% of the variance, respectively (<a reveal-id="dsx037-F5" data-open="dsx037-F5" class="link link-reveal link-table xref-fig">Fig. 5a and b</a>). In addition, PC3 and PC4 recapitulated substructures within Nordic Red cattle (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/supplementary_figures_and_tables_dsx037.pdf?Expires=2147483647&Signature=N8ekU-AB4-5zVt266s0FuDtj2aJWL~xy4OoOLYvUCqG5eWwdQ1T~hx3fJgruz9mUs~-gic4Ikx0v9SegHmwGCPkze9Kv68VZQdX55etEoJn~THklkmmMjgXPdT4Z6hbCYjDSZoUjw6h46V~h2D~93JPIAonauVegy07VuTGu1ULLUzb3SmNpuoCGtu9mnZYLC2~xIB~SmGTnvLejq8R~UU8F1OuEAfeFu0JRRyNhR50w4KNeh5IEYiHJO29zFin1eQ9YPexYjNL0gSk8XDfCde2m6Ydc8KivHx3eS2U3B~lKJkhpoI56Uel-YQr9K12aYJzJm90N~uuKou0aE8X90w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>), and first five PCs cumulatively explained 33.6% (with the deletions) and 28.4% (with the SNPs) of the variance (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/supplementary_figures_and_tables_dsx037.pdf?Expires=2147483647&Signature=N8ekU-AB4-5zVt266s0FuDtj2aJWL~xy4OoOLYvUCqG5eWwdQ1T~hx3fJgruz9mUs~-gic4Ikx0v9SegHmwGCPkze9Kv68VZQdX55etEoJn~THklkmmMjgXPdT4Z6hbCYjDSZoUjw6h46V~h2D~93JPIAonauVegy07VuTGu1ULLUzb3SmNpuoCGtu9mnZYLC2~xIB~SmGTnvLejq8R~UU8F1OuEAfeFu0JRRyNhR50w4KNeh5IEYiHJO29zFin1eQ9YPexYjNL0gSk8XDfCde2m6Ydc8KivHx3eS2U3B~lKJkhpoI56Uel-YQr9K12aYJzJm90N~uuKou0aE8X90w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). Our deletion results agree with the known population structure of the three breeds. Similar population structure (and substructure within Nordic Red Cattle) has been reported using genome-wide SNPs.<a reveal-id="dsx037-B53" data-open="dsx037-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> Nordic Red Cattle from Denmark showed closer relationship with the Holstein in our WGS samples (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/supplementary_figures_and_tables_dsx037.pdf?Expires=2147483647&Signature=N8ekU-AB4-5zVt266s0FuDtj2aJWL~xy4OoOLYvUCqG5eWwdQ1T~hx3fJgruz9mUs~-gic4Ikx0v9SegHmwGCPkze9Kv68VZQdX55etEoJn~THklkmmMjgXPdT4Z6hbCYjDSZoUjw6h46V~h2D~93JPIAonauVegy07VuTGu1ULLUzb3SmNpuoCGtu9mnZYLC2~xIB~SmGTnvLejq8R~UU8F1OuEAfeFu0JRRyNhR50w4KNeh5IEYiHJO29zFin1eQ9YPexYjNL0gSk8XDfCde2m6Ydc8KivHx3eS2U3B~lKJkhpoI56Uel-YQr9K12aYJzJm90N~uuKou0aE8X90w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). This is consistent with the known history of Holstein interrogation in Danish Red cattle, as previously reported based on imputed WGS SNP analysis on a larger sample.<a reveal-id="dsx037-B53" data-open="dsx037-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> It is also known from admixture analysis that genomes of Nordic Red Cattle are a mosaic of multiple ancestral populations, i.e. more ancestral components in Nordic Red Cattle than in Holstein and Jersey<a reveal-id="dsx037-B52" data-open="dsx037-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a><sup>,</sup><a reveal-id="dsx037-B54" data-open="dsx037-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a>; our deletion-based PCA largely corroborate that. </p>                        <a id="112482721" scrollto-destination="112482721"></a>
<div id="dsx037-F5" data-id="dsx037-F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/m_dsx037f5.png?Expires=2147483647&Signature=nDUt-vrBRZ4K5qSiOJmUBBV77HRUOfAmMOy1gfia6wygE9Hm-eHSK3gfQ9EgiHZg-RpnT9HZB9xnETio5HKLMBQ19JioGMWfC3FHRyVeal7tuSa9gFXZ4OqGY6eRp08ssTnHNygG14wU-qf~zkzeVfN7Ap7rU8-ju7R1i2GOGQZPr69BYvxZx-XEu56s0PXPzSCdnRxMljWs2llIPB2fYfClsR7i8h~J8sWnWqQKqhHjCghRrxHjGxyxDZ6CADSiKznccPbOTZpqT77GFl8kw6myI97Wh9P4-uhg4ffQ2cpRQZGhG4a7s~2pAW~xvhRkBbik~EqIDk0GFW0f0UjAlQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PCA depicting three dairy cattle breeds. The analysis is based on (a) ∼6 K deletions (0.02 &lt; VAF &lt; 0.9), and (b) ∼168 K bi-allelic SNPs randomly selected from 29 bovine autosomes. First two PCs from deletions and SNPs are plotted here; jointly explained 20 and 16.2% of the variance, respectively. HOL, Holstein; JER, Jersey; RDC, Nordic Red Cattle." path-from-xml="dsx037f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482721/dsx037f5.tif" path-from-xml="dsx037f5.tif" target="_blank">View large</a><a section="112482721" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/dsx037f5.png?Expires=2147483647&Signature=kDlRfGO62ZJNkTuyH7JIUczimaZDjG7sFDK5c3IuoN3uUdzhtObh5P2rtFR-OkjnHKQnP1NSqtJwJ5cE99HlTrNGw0yRR4MyH0QpD-A~RYVo-vJkx0wzHYchiMojemeGRhPzj4YmxEG2~965IS-psgPo8IgJAxm3Z8rPZlDHUktTBQXy5IxdZnqCUpXQiDl9i-JPgxXN6WOu~trc2epvT2~iJkfUCyXzelN36sS22~P-tYorRnLtVw5-jZq6uv0OpX045NfzB67~gG5oJLYnQ-DUpz-dITvT58vBLU6JNOvmZRXjBnPbMX2eaPTpbd~eud4KSG7LBjzwuR4ZYSUxEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482721&ar=4107864&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx037f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>PCA depicting three dairy cattle breeds. The analysis is based on (a) ∼6 K deletions (0.02 &lt; VAF &lt; 0.9), and (b) ∼168 K bi-allelic SNPs randomly selected from 29 bovine autosomes. First two PCs from deletions and SNPs are plotted here; jointly explained 20 and 16.2% of the variance, respectively. HOL, Holstein; JER, Jersey; RDC, Nordic Red Cattle.</p></div></div><div content-id="dsx037-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/m_dsx037f5.png?Expires=2147483647&Signature=nDUt-vrBRZ4K5qSiOJmUBBV77HRUOfAmMOy1gfia6wygE9Hm-eHSK3gfQ9EgiHZg-RpnT9HZB9xnETio5HKLMBQ19JioGMWfC3FHRyVeal7tuSa9gFXZ4OqGY6eRp08ssTnHNygG14wU-qf~zkzeVfN7Ap7rU8-ju7R1i2GOGQZPr69BYvxZx-XEu56s0PXPzSCdnRxMljWs2llIPB2fYfClsR7i8h~J8sWnWqQKqhHjCghRrxHjGxyxDZ6CADSiKznccPbOTZpqT77GFl8kw6myI97Wh9P4-uhg4ffQ2cpRQZGhG4a7s~2pAW~xvhRkBbik~EqIDk0GFW0f0UjAlQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="PCA depicting three dairy cattle breeds. The analysis is based on (a) ∼6 K deletions (0.02 &lt; VAF &lt; 0.9), and (b) ∼168 K bi-allelic SNPs randomly selected from 29 bovine autosomes. First two PCs from deletions and SNPs are plotted here; jointly explained 20 and 16.2% of the variance, respectively. HOL, Holstein; JER, Jersey; RDC, Nordic Red Cattle." path-from-xml="dsx037f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482721/dsx037f5.tif" path-from-xml="dsx037f5.tif" target="_blank">View large</a><a section="112482721" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/dsx037f5.png?Expires=2147483647&Signature=kDlRfGO62ZJNkTuyH7JIUczimaZDjG7sFDK5c3IuoN3uUdzhtObh5P2rtFR-OkjnHKQnP1NSqtJwJ5cE99HlTrNGw0yRR4MyH0QpD-A~RYVo-vJkx0wzHYchiMojemeGRhPzj4YmxEG2~965IS-psgPo8IgJAxm3Z8rPZlDHUktTBQXy5IxdZnqCUpXQiDl9i-JPgxXN6WOu~trc2epvT2~iJkfUCyXzelN36sS22~P-tYorRnLtVw5-jZq6uv0OpX045NfzB67~gG5oJLYnQ-DUpz-dITvT58vBLU6JNOvmZRXjBnPbMX2eaPTpbd~eud4KSG7LBjzwuR4ZYSUxEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482721&ar=4107864&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx037f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>PCA depicting three dairy cattle breeds. The analysis is based on (a) ∼6 K deletions (0.02 &lt; VAF &lt; 0.9), and (b) ∼168 K bi-allelic SNPs randomly selected from 29 bovine autosomes. First two PCs from deletions and SNPs are plotted here; jointly explained 20 and 16.2% of the variance, respectively. HOL, Holstein; JER, Jersey; RDC, Nordic Red Cattle.</p></div></div>                            <h4 scrollto-destination=112482722 id="112482722" class="section-title" >3.3.3. <em>V</em><sub>ST</sub> analysis</h4>
<p>We analysed population stratification in terms of <em>V</em><sub>ST</sub>,<a reveal-id="dsx037-B33" data-open="dsx037-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> a measure highly correlated with Wright’s fixation index (<em>F</em><sub>ST</sub>),<a reveal-id="dsx037-B34" data-open="dsx037-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> to identify population differentiation. We calculated <em>V</em><sub>ST</sub> for each deletion pairwise amongst the breeds (e.g. Holstein vs Jersey + Nordic Red Cattle) from VAFs. We identified 158 highly stratified deletions (pairwise <em>V</em><sub>ST</sub> ≥ mean + 4 <span class="small-caps">s</span>.<span class="small-caps">d</span>.) among the breeds (<a reveal-id="dsx037-F6" data-open="dsx037-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). Around 27% of these deletions overlap genic elements, i.e. exons, introns, or (upstream/downstream) untranslated regions (UTRs), and remaining 73% are intergenic variants (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/supplementary_figures_and_tables_dsx037.pdf?Expires=2147483647&Signature=N8ekU-AB4-5zVt266s0FuDtj2aJWL~xy4OoOLYvUCqG5eWwdQ1T~hx3fJgruz9mUs~-gic4Ikx0v9SegHmwGCPkze9Kv68VZQdX55etEoJn~THklkmmMjgXPdT4Z6hbCYjDSZoUjw6h46V~h2D~93JPIAonauVegy07VuTGu1ULLUzb3SmNpuoCGtu9mnZYLC2~xIB~SmGTnvLejq8R~UU8F1OuEAfeFu0JRRyNhR50w4KNeh5IEYiHJO29zFin1eQ9YPexYjNL0gSk8XDfCde2m6Ydc8KivHx3eS2U3B~lKJkhpoI56Uel-YQr9K12aYJzJm90N~uuKou0aE8X90w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S3–S5</a></span>). There were eleven sites shared between Holstein and Nordic Red Cattle, two sites between Holstein and Jersey, and one site between Nordic Red Cattle and Jersey. Among these sites were gene variants, such as <em>ABCA12</em> (Chr2:103,682,772-103,684,297 in Holstein &amp; Jersey) associated with growth and development,<a reveal-id="dsx037-B55" data-open="dsx037-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a><sup>,</sup><a reveal-id="dsx037-B56" data-open="dsx037-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a><sup>,</sup><em>TTC1</em> (Chr7:73,725,513-73,725,918 in Holstein &amp; Nordic Red Cattle) with cold tolerance,<a reveal-id="dsx037-B57" data-open="dsx037-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a><sup>,</sup><em>VWA3B</em> (Chr11:3,521,329-3,522,551 in Holstein &amp; Nordic Red Cattle) with milk glycosylated kappa-casein percentage,<a reveal-id="dsx037-B58" data-open="dsx037-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a> and were intergenic variants, such as Chr15:41,393,393-41,393,780 (in Jersey and Nordic Red Cattle) and Chr20:26,812,159-26,812,834 (in Holstein and Jersey) overlap QTL associated with calving traits,<a reveal-id="dsx037-B59" data-open="dsx037-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a><sup>,</sup><a reveal-id="dsx037-B60" data-open="dsx037-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a> Chr20:45,816,245-45,820,519 (in Holstein &amp; Nordic Red Cattle) with meat and carcass trait,<a reveal-id="dsx037-B61" data-open="dsx037-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a> and Chr23:49,778,653-49,782,567 (in Holstein and Nordic Red Cattle) with body weight.<a reveal-id="dsx037-B62" data-open="dsx037-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a> We also identified a highly differentiated fertility associated gene <em>DST/BPAG1</em><a reveal-id="dsx037-B63" data-open="dsx037-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a><sup>,</sup><a reveal-id="dsx037-B64" data-open="dsx037-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> (Chr23:3,486,232-3,486,603) in Nordic Red Cattle. One differentially selected deletion (<em>V</em><sub>ST</sub> = 0.28) of chromosome 3 (Chr3:12,141,822-12,170,916) overlapping <em>ENSBTAG00000047776</em> and <em>ENSBTAG00000024960</em> genes (human orthologue <em>CD1D</em>), drawn our attention (though it marginally failed our selection threshold); Holsteins exhibited VAF of 24.63% (have both homozygous and heterozygous deletion), while it is mostly homozygous for the reference allele in Nordic Red cattle (VAF = 0.62%) and Jersey (VAF = 0.0%). The <em>CD1D</em> gene has known function in host immune response and parasite resistance,<a reveal-id="dsx037-B65" data-open="dsx037-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a><sup>,</sup><a reveal-id="dsx037-B66" data-open="dsx037-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> and also reported differentially expressed post intra-mammary infection.<a reveal-id="dsx037-B67" data-open="dsx037-B67" class="link link-ref link-reveal xref-bibr"><sup>67</sup></a> Majority of these stratified deletion regions are novel compared with previous CNV studies in cattle, and therefore are interesting targets to investigate large deletions undergoing genetic drift or artificial selection. </p>                        <a id="112482724" scrollto-destination="112482724"></a>
<div id="dsx037-F6" data-id="dsx037-F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/m_dsx037f6.png?Expires=2147483647&Signature=h7nr505llSEiO8Ek00wjVGi~tMsVRFz9xiQXD8A~OzoZntX8aCJzwNDLYolwQmdS87bknt2rOSfX15z-mHha1hYQo2eb37shgfsTv8t4wFZGdvx7m~MLPvBj2n0jv7ShZT-37lqtidQT07fE1JPmb9NWFpjQf2k7B0nDbsbBds083kXzoioeyFVANwG6EmcD3W~gvm3xXr6L8enr6Kljs9KmdnBMvGKeHwWzKKTbTIhs6dHQqdxBSqK5kQ71zLg72KReXckRVaNZ8Ls-bZuoDIvnTGm2biBMNrbzJLqvx~l0OK5hIFwBcJBSKkb1SkdyuPyi7fDROJfTQaXCIbfGhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Population stratification based on VST (a measure of differentiation for SV, highly correlated with Wright’s fixation index, FST). Horizontal dash line indicates highly stratified deletion regions (VST ≥ Mean + 4 s.d.). Highly stratified genic-deletions, e.g. overlapping exons, introns, or UTRs, are highlighted with HGNC gene symbol." path-from-xml="dsx037f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482724/dsx037f6.tif" path-from-xml="dsx037f6.tif" target="_blank">View large</a><a section="112482724" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/dsx037f6.png?Expires=2147483647&Signature=TlF6V69xfBTq5DnFHaaZ-~dzOdtNLWU9d3VzDweeD31jXQy9yiQR6tp7kDyVIIEyI7dIj~JqvchZV794wp4OVJVt9H~esFj~kLUfHbO7~AYGjJnfwg-6P7xuEvoPML54cqPdPu-KpuAvgBR01ZEphMtgbu2yNihrhPMv1U4ErckR1o1JtlDFGxKJf04hFm8FSeqHRlgpURH8PkLTIfHZQbKYu5dZLzTIbjY~AHfst4tmhTBk94zvycNy2YXCJCGZOI419Z-MapyTPGogatCm6ZPxJxray9zWaBNX65vn5sh2OH~1QgQoulv8khWcW9WrEkDKqThUABcgoP2OG3G7bg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482724&ar=4107864&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx037f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Population stratification based on <em>V</em><sub>ST</sub> (a measure of differentiation for SV, highly correlated with Wright’s fixation index, <em>F</em><sub>ST</sub>). Horizontal dash line indicates highly stratified deletion regions (<em>V</em><sub>ST</sub> ≥ Mean + 4 <span class="small-caps">s</span>.<span class="small-caps">d</span>.). Highly stratified genic-deletions, e.g. overlapping exons, introns, or UTRs, are highlighted with HGNC gene symbol.</p></div></div><div content-id="dsx037-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/m_dsx037f6.png?Expires=2147483647&Signature=h7nr505llSEiO8Ek00wjVGi~tMsVRFz9xiQXD8A~OzoZntX8aCJzwNDLYolwQmdS87bknt2rOSfX15z-mHha1hYQo2eb37shgfsTv8t4wFZGdvx7m~MLPvBj2n0jv7ShZT-37lqtidQT07fE1JPmb9NWFpjQf2k7B0nDbsbBds083kXzoioeyFVANwG6EmcD3W~gvm3xXr6L8enr6Kljs9KmdnBMvGKeHwWzKKTbTIhs6dHQqdxBSqK5kQ71zLg72KReXckRVaNZ8Ls-bZuoDIvnTGm2biBMNrbzJLqvx~l0OK5hIFwBcJBSKkb1SkdyuPyi7fDROJfTQaXCIbfGhw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Population stratification based on VST (a measure of differentiation for SV, highly correlated with Wright’s fixation index, FST). Horizontal dash line indicates highly stratified deletion regions (VST ≥ Mean + 4 s.d.). Highly stratified genic-deletions, e.g. overlapping exons, introns, or UTRs, are highlighted with HGNC gene symbol." path-from-xml="dsx037f6.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482724/dsx037f6.tif" path-from-xml="dsx037f6.tif" target="_blank">View large</a><a section="112482724" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/dsx037f6.png?Expires=2147483647&Signature=TlF6V69xfBTq5DnFHaaZ-~dzOdtNLWU9d3VzDweeD31jXQy9yiQR6tp7kDyVIIEyI7dIj~JqvchZV794wp4OVJVt9H~esFj~kLUfHbO7~AYGjJnfwg-6P7xuEvoPML54cqPdPu-KpuAvgBR01ZEphMtgbu2yNihrhPMv1U4ErckR1o1JtlDFGxKJf04hFm8FSeqHRlgpURH8PkLTIfHZQbKYu5dZLzTIbjY~AHfst4tmhTBk94zvycNy2YXCJCGZOI419Z-MapyTPGogatCm6ZPxJxray9zWaBNX65vn5sh2OH~1QgQoulv8khWcW9WrEkDKqThUABcgoP2OG3G7bg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482724&ar=4107864&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx037f6.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Population stratification based on <em>V</em><sub>ST</sub> (a measure of differentiation for SV, highly correlated with Wright’s fixation index, <em>F</em><sub>ST</sub>). Horizontal dash line indicates highly stratified deletion regions (<em>V</em><sub>ST</sub> ≥ Mean + 4 <span class="small-caps">s</span>.<span class="small-caps">d</span>.). Highly stratified genic-deletions, e.g. overlapping exons, introns, or UTRs, are highlighted with HGNC gene symbol.</p></div></div>                            <h3 scrollto-destination=112482725 id="112482725" class="section-title" >3.4. Functional impact of deletions</h3>
<p>We annotated all the deletions using <em>Variant Effect Predictor (Ensembl 87)</em>. Around 71% (6,019 SVs) variants were intergenic and remaining 29% (2,461 SVs) overlapped genic elements, such as exons, introns, and UTR. On average, high frequency gene disrupting deletions were somewhat depleted compared with intergenic variants (VAF<sub>intergenic</sub> &gt; VAF<sub>genic</sub>, <em>P-</em>value <em>=</em> 0.04; one-sided Wilcoxon test). Furthermore, we observed many common genic deletions. These genes are relatively less conserved, and majority has multiple paralogs (discussed later). However, deletions on known essential genes were only observed as heterozygote with relatively low VAF (&lt;3%), and generally were private to a specific breed. For example, <em>FANCI</em> deletions (cause brachyspina<a reveal-id="dsx037-B7" data-open="dsx037-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a>) were only observed in Holstein, and <em>RNASEH2B</em> deletions (cause embryonic lethality<a reveal-id="dsx037-B9" data-open="dsx037-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a>) in Nordic Red Cattle.</p>                            <h4 scrollto-destination=112482727 id="112482727" class="section-title" >3.4.1. Selective constraints on genes overlapping deletions</h4>
<p>The relative abundance of high-frequency genic and intergenic variants indicate that majority of these intersected genes are non-essential, and thus did not affect the viability or fecundity of the carriers. To test this hypothesis, we analysed the selective constraints between deleted genes (overlap of any genic element) and known mouse lethal genes (from Dickinson et al.<a reveal-id="dsx037-B42" data-open="dsx037-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a>) in terms of <em>dN/dS</em> ratio of cow-mouse 1-to-1 orthologues (<a reveal-id="dsx037-F7" data-open="dsx037-F7" class="link link-reveal link-table xref-fig">Fig. 7</a>). Here, high <em>dN/dS</em> values indicate low selective constraints on genes, and low value indicates high constraints. We found that genes in deletions have significantly higher <em>dN/dS</em> ratios than lethal genes (<em>P-</em>value 2.3 × 10<sup>−6</sup>; one-sided Wilcoxon test), and thus are evolutionarily less conserved. This is consistent with the rate of evolution seen in essential and non-essential genes—where mutations in essential genes were under strong purifying selection and thus evolved slowly (low <em>dN/dS</em> ratio), while non-essential genes were under relaxed selection, and hence, evolved faster (high <em>dN/dS</em> ratio).<a reveal-id="dsx037-B68" data-open="dsx037-B68" class="link link-ref link-reveal xref-bibr"><sup>68</sup></a> Nonetheless, robustness of these processes is also evident in the evolution of human essential genes. Interestingly, ∼77% human essential genes could even be traced back to pre-metazoans.<a reveal-id="dsx037-B69" data-open="dsx037-B69" class="link link-ref link-reveal xref-bibr"><sup>69</sup></a></p>                        <a id="112482729" scrollto-destination="112482729"></a>
<div id="dsx037-F7" data-id="dsx037-F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/m_dsx037f7.png?Expires=2147483647&Signature=h4FZmraHTpzhX8HxmuNn7y4zDtEizmtGuy3CvqXYetSI-RuXN6sgoorglcw8cs6xej8WILGw5qMYozb6W6pGVvFAByz1lxLwZBQcfFTaYWyY8yA3D-j3IqwAV0QZ~VhAX-c20zkL9BOenfDOFoP4Gi25BsVSTg0Old2DVOlcEbZTZ4AnK5OnuDliGZje1rv51TZ6Fd6aWddkehXsB3mtVzR4nYUmrMJL3ha3WUciqaVLBjGMuiLzpVE1VPsFZKNXz1QrH8x0bhsXgF13qio58HQy13I2cYeX4CUpkoQGpBA94Whor8rFnLz95~xbs~4nAdf81wHrinTxpRfi9MeO7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Difference between dN/dS ratios of mouse-lethal and deletion-overlapped genes in cattle. Cow genes for which one-to-one mouse orthologues available were considered for a one-sided Wilcoxon rank-sum test. Mouse lethal genes are from Dickinson et al.42" path-from-xml="dsx037f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482729/dsx037f7.tif" path-from-xml="dsx037f7.tif" target="_blank">View large</a><a section="112482729" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/dsx037f7.png?Expires=2147483647&Signature=jcRcCf2s6Xf5oLleh2zVS~4SW7IpQVVa8uCcMj9pugmASG0UbWgfjuGus7Zm5ElGCNdmKhdQdralX53JoiPYpV5xvU5hlIErpaYD-2UHa-97L6b7fMFMfD9Qq0Uzlj580YZ6fcUCsRxNTMl4LVuVwQ5j6p5~o~y4MaOlIaS8KQP02Y2Fy4gEeFpKUH2rN4C18RKaMhk7rhIuOpYQu2i~Z0xoNGKnfQ6FTKOktBZ5iYNefqKHera77d-VbYRzVz96Oo5d-4E6DKjVPScDcH~9sRGyeD4PGYTYRYgiSHpGMe56IxGtJZLp3579Zv8bCjTa8uoQ05z3Rn1kXMvEHM2w-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482729&ar=4107864&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx037f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Difference between <em>dN/dS</em> ratios of mouse-lethal and deletion-overlapped genes in cattle. Cow genes for which one-to-one mouse orthologues available were considered for a one-sided Wilcoxon rank-sum test. Mouse lethal genes are from Dickinson et al.<a reveal-id="dsx037-B42" data-open="dsx037-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p></div></div><div content-id="dsx037-F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/m_dsx037f7.png?Expires=2147483647&Signature=h4FZmraHTpzhX8HxmuNn7y4zDtEizmtGuy3CvqXYetSI-RuXN6sgoorglcw8cs6xej8WILGw5qMYozb6W6pGVvFAByz1lxLwZBQcfFTaYWyY8yA3D-j3IqwAV0QZ~VhAX-c20zkL9BOenfDOFoP4Gi25BsVSTg0Old2DVOlcEbZTZ4AnK5OnuDliGZje1rv51TZ6Fd6aWddkehXsB3mtVzR4nYUmrMJL3ha3WUciqaVLBjGMuiLzpVE1VPsFZKNXz1QrH8x0bhsXgF13qio58HQy13I2cYeX4CUpkoQGpBA94Whor8rFnLz95~xbs~4nAdf81wHrinTxpRfi9MeO7g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Difference between dN/dS ratios of mouse-lethal and deletion-overlapped genes in cattle. Cow genes for which one-to-one mouse orthologues available were considered for a one-sided Wilcoxon rank-sum test. Mouse lethal genes are from Dickinson et al.42" path-from-xml="dsx037f7.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/112482729/dsx037f7.tif" path-from-xml="dsx037f7.tif" target="_blank">View large</a><a section="112482729" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/dsx037f7.png?Expires=2147483647&Signature=jcRcCf2s6Xf5oLleh2zVS~4SW7IpQVVa8uCcMj9pugmASG0UbWgfjuGus7Zm5ElGCNdmKhdQdralX53JoiPYpV5xvU5hlIErpaYD-2UHa-97L6b7fMFMfD9Qq0Uzlj580YZ6fcUCsRxNTMl4LVuVwQ5j6p5~o~y4MaOlIaS8KQP02Y2Fy4gEeFpKUH2rN4C18RKaMhk7rhIuOpYQu2i~Z0xoNGKnfQ6FTKOktBZ5iYNefqKHera77d-VbYRzVz96Oo5d-4E6DKjVPScDcH~9sRGyeD4PGYTYRYgiSHpGMe56IxGtJZLp3579Zv8bCjTa8uoQ05z3Rn1kXMvEHM2w-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=112482729&ar=4107864&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx037f7.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Difference between <em>dN/dS</em> ratios of mouse-lethal and deletion-overlapped genes in cattle. Cow genes for which one-to-one mouse orthologues available were considered for a one-sided Wilcoxon rank-sum test. Mouse lethal genes are from Dickinson et al.<a reveal-id="dsx037-B42" data-open="dsx037-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p></div></div>                            <h4 scrollto-destination=112482730 id="112482730" class="section-title" >3.4.2. Nonessential genes in cattle</h4>
<p>In total, we found 5,000 deletions for which at least one individual was homozygous. In the set, we analysed homozygous deletions in genes to find natural gene knockouts. We found 167 deleted genes (transcript-ablation or complete deletion) corresponding to 115 independent deletions that are apparently nonessential based on the occurrences of live homozygote individuals. This is ∼45% more than the previous report.<a reveal-id="dsx037-B23" data-open="dsx037-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Nonetheless, we found ∼44% fewer genes compared with in humans (240 nonessential genes),<a reveal-id="dsx037-B15" data-open="dsx037-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> which could be due to the differences in sample size (175 vs 2,504 individuals) and study populations (3 vs 26 populations in human).</p><p>Among these genes, ∼83% (139 genes) are protein-coding, 12% pseudogenes, and the rest are different types of small RNAs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/supplementary_figures_and_tables_dsx037.pdf?Expires=2147483647&Signature=N8ekU-AB4-5zVt266s0FuDtj2aJWL~xy4OoOLYvUCqG5eWwdQ1T~hx3fJgruz9mUs~-gic4Ikx0v9SegHmwGCPkze9Kv68VZQdX55etEoJn~THklkmmMjgXPdT4Z6hbCYjDSZoUjw6h46V~h2D~93JPIAonauVegy07VuTGu1ULLUzb3SmNpuoCGtu9mnZYLC2~xIB~SmGTnvLejq8R~UU8F1OuEAfeFu0JRRyNhR50w4KNeh5IEYiHJO29zFin1eQ9YPexYjNL0gSk8XDfCde2m6Ydc8KivHx3eS2U3B~lKJkhpoI56Uel-YQr9K12aYJzJm90N~uuKou0aE8X90w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>). Most of these genes belong to multigenic families and are not highly conserved (median cow-mouse <em>dN/dS</em> of 0.17 vs OMIA genes <em>dN/dS</em> of 0.11; <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/supplementary_figures_and_tables_dsx037.pdf?Expires=2147483647&Signature=N8ekU-AB4-5zVt266s0FuDtj2aJWL~xy4OoOLYvUCqG5eWwdQ1T~hx3fJgruz9mUs~-gic4Ikx0v9SegHmwGCPkze9Kv68VZQdX55etEoJn~THklkmmMjgXPdT4Z6hbCYjDSZoUjw6h46V~h2D~93JPIAonauVegy07VuTGu1ULLUzb3SmNpuoCGtu9mnZYLC2~xIB~SmGTnvLejq8R~UU8F1OuEAfeFu0JRRyNhR50w4KNeh5IEYiHJO29zFin1eQ9YPexYjNL0gSk8XDfCde2m6Ydc8KivHx3eS2U3B~lKJkhpoI56Uel-YQr9K12aYJzJm90N~uuKou0aE8X90w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>), as expected for homozygous deletion.<a reveal-id="dsx037-B15" data-open="dsx037-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Moreover, this set of genes are functionally enriched in immunoglobulin domains, olfactory receptors, and MHC classes (FDR = 2.06 × 10<sup>−22</sup>, 2.06 × 10<sup>−22</sup>, 7.01 × 10<sup>−6</sup>, respectively), along with other related domains (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/supplementary_figures_and_tables_dsx037.pdf?Expires=2147483647&Signature=N8ekU-AB4-5zVt266s0FuDtj2aJWL~xy4OoOLYvUCqG5eWwdQ1T~hx3fJgruz9mUs~-gic4Ikx0v9SegHmwGCPkze9Kv68VZQdX55etEoJn~THklkmmMjgXPdT4Z6hbCYjDSZoUjw6h46V~h2D~93JPIAonauVegy07VuTGu1ULLUzb3SmNpuoCGtu9mnZYLC2~xIB~SmGTnvLejq8R~UU8F1OuEAfeFu0JRRyNhR50w4KNeh5IEYiHJO29zFin1eQ9YPexYjNL0gSk8XDfCde2m6Ydc8KivHx3eS2U3B~lKJkhpoI56Uel-YQr9K12aYJzJm90N~uuKou0aE8X90w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S7–S9</a></span>). Similar functional enrichment of nonessential genes was also seen in humans.<a reveal-id="dsx037-B15" data-open="dsx037-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Olfactory receptor related genes are well known for extensive gains and losses in mammalian evolution.<a reveal-id="dsx037-B70" data-open="dsx037-B70" class="link link-ref link-reveal xref-bibr"><sup>70</sup></a> And population specific copy-number variations of olfactory receptor genes were also reported in human (deletions)<a reveal-id="dsx037-B71" data-open="dsx037-B71" class="link link-ref link-reveal xref-bibr"><sup>71</sup></a> and cattle (gains).<a reveal-id="dsx037-B72" data-open="dsx037-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> Nevertheless, this is the first report, to our knowledge, of homozygous deletion of olfactory receptor genes in cattle.</p>                            <h4 scrollto-destination=112482733 id="112482733" class="section-title" >3.4.3. QTL enrichment</h4>
<p>We next explored the enrichment (or depletion) of QTL on deleted regions (at least 1 bp overlap with deletion). We retrieved ∼24 K autosomal QTL from <em>QTLdb</em> reported to be associated with any of the six trait classes, e.g. ‘Health’, ‘Reproduction’, ‘Milk’, ‘Exterior’, ‘Production’, and ‘Meat and Carcass’. The association of deletions with diseases, fitness or fertility related traits is well evident.<a reveal-id="dsx037-B3" data-open="dsx037-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Hence, we suspected enrichment of fitness and fertility related traits for our deletions. As expected, health (2-fold) and reproduction (1.5-fold) related QTL were significantly enriched, while other trait classes were highly depleted (<a reveal-id="dsx037-T2" data-open="dsx037-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Higher enrichment of health related QTL could be driven by immune-system genes, which were also highly enriched in our dataset (discussed earlier). <div content-id="dsx037-T2" class="table-modal"><div class="table-wrap"><div id="dsx037-T2" data-id="dsx037-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Enrichment of QTL on deletions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><strong>Trait classes</strong><a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Fold enrichment </th><th><em>P value</em> (Fisher’s test) </th></tr></thead><tbody><tr><td>Health </td><td><strong>2</strong> </td><td>8.91 × 10<sup>−10</sup> </td></tr><tr><td>Reproduction </td><td><strong>1.5</strong> </td><td>7.4 × 10<sup>−11</sup> </td></tr><tr><td>Milk </td><td>0.8 </td><td>2.45 × 10<sup>−7</sup> </td></tr><tr><td>Exterior </td><td>0.5 </td><td>1.85 × 10<sup>−4</sup> </td></tr><tr><td>Production </td><td>0.5 </td><td>0.002 </td></tr><tr><td>Meat and Carcass </td><td>0.5 </td><td>0.058 </td></tr></tbody></table></div><div content-id="dsx037-T2" class="table-modal"><table><thead name="thead" align=""><tr><th><strong>Trait classes</strong><a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Fold enrichment </th><th><em>P value</em> (Fisher’s test) </th></tr></thead><tbody><tr><td>Health </td><td><strong>2</strong> </td><td>8.91 × 10<sup>−10</sup> </td></tr><tr><td>Reproduction </td><td><strong>1.5</strong> </td><td>7.4 × 10<sup>−11</sup> </td></tr><tr><td>Milk </td><td>0.8 </td><td>2.45 × 10<sup>−7</sup> </td></tr><tr><td>Exterior </td><td>0.5 </td><td>1.85 × 10<sup>−4</sup> </td></tr><tr><td>Production </td><td>0.5 </td><td>0.002 </td></tr><tr><td>Meat and Carcass </td><td>0.5 </td><td>0.058 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p><sup>a</sup>Trait classes are from cattleQTLdb.<a reveal-id="dsx037-B42" data-open="dsx037-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> QTL from autosomes of Holsteins, Jersey, Nordic Red Cattle, and Ayrshires were considered for Fisher’s exact test (two-sided).</p></div></div></div></div></div><div class="table-wrap"><div id="dsx037-T2" data-id="dsx037-T2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Enrichment of QTL on deletions</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><strong>Trait classes</strong><a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Fold enrichment </th><th><em>P value</em> (Fisher’s test) </th></tr></thead><tbody><tr><td>Health </td><td><strong>2</strong> </td><td>8.91 × 10<sup>−10</sup> </td></tr><tr><td>Reproduction </td><td><strong>1.5</strong> </td><td>7.4 × 10<sup>−11</sup> </td></tr><tr><td>Milk </td><td>0.8 </td><td>2.45 × 10<sup>−7</sup> </td></tr><tr><td>Exterior </td><td>0.5 </td><td>1.85 × 10<sup>−4</sup> </td></tr><tr><td>Production </td><td>0.5 </td><td>0.002 </td></tr><tr><td>Meat and Carcass </td><td>0.5 </td><td>0.058 </td></tr></tbody></table></div><div content-id="dsx037-T2" class="table-modal"><table><thead name="thead" align=""><tr><th><strong>Trait classes</strong><a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th>Fold enrichment </th><th><em>P value</em> (Fisher’s test) </th></tr></thead><tbody><tr><td>Health </td><td><strong>2</strong> </td><td>8.91 × 10<sup>−10</sup> </td></tr><tr><td>Reproduction </td><td><strong>1.5</strong> </td><td>7.4 × 10<sup>−11</sup> </td></tr><tr><td>Milk </td><td>0.8 </td><td>2.45 × 10<sup>−7</sup> </td></tr><tr><td>Exterior </td><td>0.5 </td><td>1.85 × 10<sup>−4</sup> </td></tr><tr><td>Production </td><td>0.5 </td><td>0.002 </td></tr><tr><td>Meat and Carcass </td><td>0.5 </td><td>0.058 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p><sup>a</sup>Trait classes are from cattleQTLdb.<a reveal-id="dsx037-B42" data-open="dsx037-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a> QTL from autosomes of Holsteins, Jersey, Nordic Red Cattle, and Ayrshires were considered for Fisher’s exact test (two-sided).</p></div></div></div></div></p>                            <h3 scrollto-destination=112482735 id="112482735" class="section-title" >3.5. Deletion formation mechanisms</h3>
<p>Finally we explored the probable mechanisms of deletion formation. There are two key mechanisms of SVs formation (for detail see review<a reveal-id="dsx037-B19" data-open="dsx037-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsx037-B73" data-open="dsx037-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a>); for example, recurrent SVs often result from ‘non-allelic homologous recombination’ between large low-copy repeats (LCRs), and thus, contain extensive sequence homology provided by LCRs, such as segmental duplicates, at the flanking regions.<a reveal-id="dsx037-B19" data-open="dsx037-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> In contrast, non-recurrent SVs often form either by ‘microhomology-mediated end joining’ or ‘non-homologus end joining’, which requires limited to no sequence homology, and thus could be characterized by microhomologies or simple blunt ends at the breakpoint junction.<a reveal-id="dsx037-B73" data-open="dsx037-B73" class="link link-ref link-reveal xref-bibr"><sup>73</sup></a></p><p>Breakpoint information is crucial for understanding the mechanism, and therefore, we analysed 29 breakpoint resolved deletions from our validation set. We found that 24 of 29 deletions contain microhomology ranging from 2 to 31 bp at the breakpoint, and two of which also contain insertions (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/s1_and_s2_dsx037.xls?Expires=2147483647&Signature=vhdyIqhZPLuViQrCa8hpkyySgekxCBcznuz0oqoUHlTNOr1XKPwZSVPRHMacIv8nDZign9ABA23sFAQO-npvrF48ltv8UdCVFGMod664flAKuArLtJ4UzZ8Q-z9glSbhkJOHUEIXbmXgJBGz31pNp4C-qGH~VxVFQGogjtVnvJggQjRqqZBCZECFO7eNPHkMkQITtkc2IuOOt4n80IUS2Se-6rlPhsTeVqIFX2XtlFVBxgyge20XNsdt~~lNk8vtX56dIqwc8Cpu9uKTv3rSVIw93AFbBjB8uynY6TwdianwvTuICtBKwAmBUWIPaRrrpG-kpawxEsVlbO1bkaob1w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). In addition, four deletions exhibited non-reference insertion at breakpoint junctions, and one deletion with no apparent homology. However, the number of breakpoint sequences analysed here were not a robust representation of our deletion call-set (&lt;0.5% deletions), though selected randomly (for validation), we were able to demonstrate that majority of deletions contain microhomoloy at breakpoint, followed by few insertions, and rarely with no homology. Our results largely agree with the trend reported for large deletions in humans, e.g. 70.8% deletions exhibited microhomology/homology and 16.1% insertions at the breakpoint.<a reveal-id="dsx037-B16" data-open="dsx037-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a></p>                            <h3 scrollto-destination=112482738 id="112482738" class="section-title" >3.6. Limitations</h3>
<p>This study only focused on identifying deletions in cattle because of their potential relevance to loss-of-function and embryonic lethality. However, we had limited success to identify small deletions, such as &lt;200 bp due to reduced sensitivity of the SV caller. It is also not a comprehensive list of deletions for these samples, since we could have missed many true deletions due to sensitivity, coverage, or stringent filtering (among other reasons). Furthermore, the short read length (∼100 bp) in our WGS dataset also made it difficult to resolve breakpoints from regions of long repeats.</p>                            <h3 scrollto-destination=112482740 id="112482740" class="section-title" >3.7. Conclusions</h3>
<p>Loss-of-function variants are responsible for a substantial yearly-economic loss in dairy industry, where a limited number of elite sires are in extensive use for rapid genetic gains. Mapping of such variants is essential for effective breeding planning and genomic selection. Here we showed an NGS-based analytical framework suitable for population-scale mapping of large deletions in cattle, leveraging the available WGSs. Here we described population-genetic, functional, and evolutionary properties of discovered deletions. We identified and confirmed a ∼525 KB deletion on chromosome 23, causing stillbirth in Nordic Red Cattle. We demonstrated that Nordic Red Cattle had higher population diversity than Holstein and Jersey, and deletion-genotype could recapitulate genetic structure of these breeds. Natural gene knockouts are enriched for immune-related and olfactory receptor genes. We also showed that deletions are significantly enriched for health and fertility related QTL, while depleted for production related QTL. Our population genetic and functional analysis showed promise for inclusion of SVs in genomic studies in dairy cattle. This deletion catalog will facilitate discovery, genotyping, and imputation of deletions in large cohorts of animals, and subsequent studies for gene mapping and genomic prediction of breeding values.</p>                            <h2 scrollto-destination=112482742 id="112482742" class="backacknowledgements-title" >Acknowledgements</h2>
<p>Md Mesbah-Uddin benefited from a joint grant from the European Commission within the framework of the Erasmus-Mundus joint doctorate ‘EGS-ABG’.</p>                            <h3 scrollto-destination=112482744 id="112482744" class="section-title" >Funding</h3>
<p>This research was supported by the Center for Genomic Selection in Animals and Plants (GenSAP) funded by Innovation Fund Denmark (grant 0603-00519B). </p>                            <h3 scrollto-destination=112482746 id="112482746" class="section-title" >Data availability</h3>
<p>All relevant results are within the paper and its <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/supplementary_figures_and_tables_dsx037.pdf?Expires=2147483647&Signature=N8ekU-AB4-5zVt266s0FuDtj2aJWL~xy4OoOLYvUCqG5eWwdQ1T~hx3fJgruz9mUs~-gic4Ikx0v9SegHmwGCPkze9Kv68VZQdX55etEoJn~THklkmmMjgXPdT4Z6hbCYjDSZoUjw6h46V~h2D~93JPIAonauVegy07VuTGu1ULLUzb3SmNpuoCGtu9mnZYLC2~xIB~SmGTnvLejq8R~UU8F1OuEAfeFu0JRRyNhR50w4KNeh5IEYiHJO29zFin1eQ9YPexYjNL0gSk8XDfCde2m6Ydc8KivHx3eS2U3B~lKJkhpoI56Uel-YQr9K12aYJzJm90N~uuKou0aE8X90w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> files. VCF file with deletion calls could be found at <a class="link link-uri" href="https://github.com/MMesbahU/Deletions_in_cattle" target="_blank">https://github.com/MMesbahU/Deletions_in_cattle</a>. WGSs of 44 samples (out of 175) are available from the NCBI Sequence Read Archive (project accession numbers SRP039339 and SRP065105). Among the 175 samples, 144 are from Run 6 of 1KBGP. Rest of data are available only upon agreement with the commercial breeding organization and should be requested directly from the senior author (G.S.: <a href="mailto:goutam.sahana@mbg.au.dk" target="_blank">goutam.sahana@mbg.au.dk</a>) or the Center Director (M.S.L.: <a href="mailto:mogens.lund@mbg.au.dk" target="_blank">mogens.lund@mbg.au.dk</a>).</p><p><strong>Conflict of interest:</strong> None declared.</p>                            <h3 scrollto-destination=112482749 id="112482749" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/supplementary_figures_and_tables_dsx037.pdf?Expires=2147483647&Signature=N8ekU-AB4-5zVt266s0FuDtj2aJWL~xy4OoOLYvUCqG5eWwdQ1T~hx3fJgruz9mUs~-gic4Ikx0v9SegHmwGCPkze9Kv68VZQdX55etEoJn~THklkmmMjgXPdT4Z6hbCYjDSZoUjw6h46V~h2D~93JPIAonauVegy07VuTGu1ULLUzb3SmNpuoCGtu9mnZYLC2~xIB~SmGTnvLejq8R~UU8F1OuEAfeFu0JRRyNhR50w4KNeh5IEYiHJO29zFin1eQ9YPexYjNL0gSk8XDfCde2m6Ydc8KivHx3eS2U3B~lKJkhpoI56Uel-YQr9K12aYJzJm90N~uuKou0aE8X90w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=112482751 id="112482751" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx037-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Charlier</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Harland</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">NGS-based reverse genetic screen for common embryonic lethal mutations compromising fertility in livestock</div>. <div class="source">Genome Res</div> ., <div class="volume">26</div>, <div class="fpage">1333</div>–<div class="lpage">1341</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NGS-based%20reverse%20genetic%20screen%20for%20common%20embryonic%20lethal%20mutations%20compromising%20fertility%20in%20livestock&amp;author=C.%20Charlier&amp;author=W.%20Li&amp;author=C.%20Harland&amp;publication_year=2016&amp;journal=Genome%20Res&amp;volume=26&amp;pages=1333-1341" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.207076.116" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.207076.116" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.207076.116"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27646536" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cole</div> <div class="given-names">J.B.</div></div>, <div class="name string-name"><div class="surname">Null</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">VanRaden</div> <div class="given-names">P.M.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Phenotypic and genetic effects of recessive haplotypes on yield, longevity, and fertility</div>. <div class="source">J Dairy Sci</div> ., <div class="volume">99</div>, <div class="fpage">7274</div>–<div class="lpage">88</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phenotypic%20and%20genetic%20effects%20of%20recessive%20haplotypes%20on%20yield%2C%20longevity%2C%20and%20fertility&amp;author=J.B.%20Cole&amp;author=D.J.%20Null&amp;author=P.M.%20VanRaden&amp;publication_year=2016&amp;journal=J%20Dairy%20Sci&amp;volume=99&amp;pages=7274-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3168/jds.2015-10777" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3168%2Fjds.2015-10777" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3168%2Fjds.2015-10777"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27394947" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Weischenfeldt</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Symmons</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Spitz</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Korbel</div> <div class="given-names">J. O.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Phenotypic impact of genomic structural variation: insights from and for human disease</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">14</div>, <div class="fpage">125</div>–<div class="lpage">138</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phenotypic%20impact%20of%20genomic%20structural%20variation%3A%20insights%20from%20and%20for%20human%20disease&amp;author=J.%20Weischenfeldt&amp;author=O.%20Symmons&amp;author=F.%20Spitz&amp;author=J.%20O.%20Korbel&amp;publication_year=2013&amp;journal=Nat.%20Rev.%20Genet&amp;volume=14&amp;pages=125-138" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3373" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3373" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3373"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23329113" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zarrei</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">MacDonald</div> <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">Merico</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Scherer</div> <div class="given-names">S.W.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">A copy number variation map of the human genome</div>. <div class="source">Nat Rev Genet</div> , <div class="volume">16</div>, <div class="fpage">172</div>–<div class="lpage">183</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20copy%20number%20variation%20map%20of%20the%20human%20genome&amp;author=M.%20Zarrei&amp;author=J.R.%20MacDonald&amp;author=D.%20Merico&amp;author=S.W.%20Scherer&amp;publication_year=2015&amp;journal=Nat%20Rev%20Genet&amp;volume=16&amp;pages=172-183" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg3871" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg3871" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg3871"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25645873" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bickhart</div> <div class="given-names">D.M.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">G.E.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">The challenges and importance of structural variation detection in livestock</div>. <div class="source">Front. Genet</div> ., <div class="volume">5</div>, <div class="fpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20challenges%20and%20importance%20of%20structural%20variation%20detection%20in%20livestock&amp;author=D.M.%20Bickhart&amp;author=G.E.%20Liu&amp;publication_year=2014&amp;journal=Front.%20Genet&amp;volume=5&amp;pages=37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fgene.2014.00037" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffgene.2014.00037" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffgene.2014.00037"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24600474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Cole</div> <div class="given-names">J.B.</div></div>, <div class="name string-name"><div class="surname">Bickhart</div> <div class="given-names">D.M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Genome wide CNV analysis reveals additional variants associated with milk production traits in Holsteins</div>. <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">683</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20wide%20CNV%20analysis%20reveals%20additional%20variants%20associated%20with%20milk%20production%20traits%20in%20Holsteins&amp;author=L.%20Xu&amp;author=J.B.%20Cole&amp;author=D.M.%20Bickhart&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=683" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-683" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-683" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-683"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25128478" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Charlier</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Agerholm</div> <div class="given-names">J. S.</div></div>, <div class="name string-name"><div class="surname">Coppieters</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">A deletion in the bovine FANCI gene compromises fertility by causing fetal death and brachyspina</div>. <div class="source">PLoS One</div> , <div class="volume">7</div>, <div class="fpage">e43085</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20deletion%20in%20the%20bovine%20FANCI%20gene%20compromises%20fertility%20by%20causing%20fetal%20death%20and%20brachyspina&amp;author=C.%20Charlier&amp;author=J.%20S.%20Agerholm&amp;author=W.%20Coppieters&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e43085" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0043085" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0043085" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0043085"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22952632" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schutz</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Wehrhahn</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Wanjek</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The holstein friesian lethal haplotype 5 (HH5) results from a complete deletion of TBF1M and cholesterol deficiency (CDH) from an ERV-(LTR) insertion into the coding region of APOB</div>. <div class="source">PLoS One</div> , <div class="volume">11</div>, <div class="fpage">e0154602</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20holstein%20friesian%20lethal%20haplotype%205%20(HH5)%20results%20from%20a%20complete%20deletion%20of%20TBF1M%20and%20cholesterol%20deficiency%20(CDH)%20from%20an%20ERV-(LTR)%20insertion%20into%20the%20coding%20region%20of%20APOB&amp;author=E.%20Schutz&amp;author=C.%20Wehrhahn&amp;author=M.%20Wanjek&amp;publication_year=2016&amp;journal=PLoS%20One&amp;volume=11&amp;pages=e0154602" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0154602" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0154602" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0154602"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27128314" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kadri</div> <div class="given-names">N.K.</div></div>, <div class="name string-name"><div class="surname">Sahana</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Charlier</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">A 660-Kb deletion with antagonistic effects on fertility and milk production segregates at high frequency in Nordic Red cattle: additional evidence for the common occurrence of balancing selection in livestock</div>. <div class="source">PLoS Genet</div> ., <div class="volume">10</div>, <div class="fpage">e1004049</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20660-Kb%20deletion%20with%20antagonistic%20effects%20on%20fertility%20and%20milk%20production%20segregates%20at%20high%20frequency%20in%20Nordic%20Red%20cattle%3A%20additional%20evidence%20for%20the%20common%20occurrence%20of%20balancing%20selection%20in%20livestock&amp;author=N.K.%20Kadri&amp;author=G.%20Sahana&amp;author=C.%20Charlier&amp;publication_year=2014&amp;journal=PLoS%20Genet&amp;volume=10&amp;pages=e1004049" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1004049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1004049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1004049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24391517" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sahana</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Iso-Touru</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Wu</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">A 0.5-Mbp deletion on bovine chromosome 23 is a strong candidate for stillbirth in Nordic Red cattle</div>. <div class="source">Genet. Sel. Evol</div> ., <div class="volume">48</div>, <div class="fpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%200.5-Mbp%20deletion%20on%20bovine%20chromosome%2023%20is%20a%20strong%20candidate%20for%20stillbirth%20in%20Nordic%20Red%20cattle&amp;author=G.%20Sahana&amp;author=T.%20Iso-Touru&amp;author=X.%20Wu&amp;publication_year=2016&amp;journal=Genet.%20Sel.%20Evol&amp;volume=48&amp;pages=35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12711-016-0215-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12711-016-0215-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12711-016-0215-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27091210" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Liu</div> <div class="given-names">G. E.</div></div>, <div class="name string-name"><div class="surname">Hou</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">B.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Analysis of copy number variations among diverse cattle breeds</div>. <div class="source">Genome Res</div> , <div class="volume">20</div>, <div class="fpage">693</div>–<div class="lpage">703</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20copy%20number%20variations%20among%20diverse%20cattle%20breeds&amp;author=G.%20E.%20Liu&amp;author=Y.%20Hou&amp;author=B.%20Zhu&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=693-703" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.105403.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.105403.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.105403.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20212021" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hou</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">G. E.</div></div>, <div class="name string-name"><div class="surname">Bickhart</div> <div class="given-names">D. M.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Genomic characteristics of cattle copy number variations</div>. <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">127</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20characteristics%20of%20cattle%20copy%20number%20variations&amp;author=Y.%20Hou&amp;author=G.%20E.%20Liu&amp;author=D.%20M.%20Bickhart&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=127" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-127" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-127" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-127"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21345189" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Hou</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Bickhart</div> <div class="given-names">D. M.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Population-genetic properties of differentiated copy number variations in cattle</div>. <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">23161</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Population-genetic%20properties%20of%20differentiated%20copy%20number%20variations%20in%20cattle&amp;author=L.%20Xu&amp;author=Y.%20Hou&amp;author=D.%20M.%20Bickhart&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=23161" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep23161" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep23161" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep23161"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27005566" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Alkan</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Coe</div> <div class="given-names">B. P.</div></div>, <div class="name string-name"><div class="surname">Eichler</div> <div class="given-names">E. E.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Genome structural variation discovery and genotyping</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">12</div>, <div class="fpage">363</div>–<div class="lpage">376</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20structural%20variation%20discovery%20and%20genotyping&amp;author=C.%20Alkan&amp;author=B.%20P.%20Coe&amp;author=E.%20E.%20Eichler&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet&amp;volume=12&amp;pages=363-376" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2958" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2958" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2958"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21358748" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sudmant</div> <div class="given-names">P. H.</div></div>, <div class="name string-name"><div class="surname">Rausch</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Gardner</div> <div class="given-names">E. J.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">An integrated map of structural variation in 2,504 human genomes</div>. <div class="source">Nature</div> , <div class="volume">526</div>, <div class="fpage">75</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20map%20of%20structural%20variation%20in%202%2C504%20human%20genomes&amp;author=P.%20H.%20Sudmant&amp;author=T.%20Rausch&amp;author=E.%20J.%20Gardner&amp;publication_year=2015&amp;journal=Nature&amp;volume=526&amp;pages=75-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature15394" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature15394" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature15394"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26432246" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mills</div> <div class="given-names">R. E.</div></div>, <div class="name string-name"><div class="surname">Walter</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Stewart</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Mapping copy number variation by population-scale genome sequencing</div>. <div class="source">Nature</div> , <div class="volume">470</div>, <div class="fpage">59</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20copy%20number%20variation%20by%20population-scale%20genome%20sequencing&amp;author=R.%20E.%20Mills&amp;author=K.%20Walter&amp;author=C.%20Stewart&amp;publication_year=2011&amp;journal=Nature&amp;volume=470&amp;pages=59-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature09708" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09708" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature09708"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21293372" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yalcin</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Agam</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Sequence-based characterization of structural variation in the mouse genome</div>. <div class="source">Nature</div> , <div class="volume">477</div>, <div class="fpage">326</div>–<div class="lpage">329</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence-based%20characterization%20of%20structural%20variation%20in%20the%20mouse%20genome&amp;author=B.%20Yalcin&amp;author=K.%20Wong&amp;author=A.%20Agam&amp;publication_year=2011&amp;journal=Nature&amp;volume=477&amp;pages=326-329" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10432" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10432" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10432"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21921916" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wallis</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ding</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Weinstock</div> <div class="given-names">G.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">TIGRA: a targeted iterative graph routing assembler for breakpoint assembly</div>. <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">310</div>–<div class="lpage">317</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TIGRA%3A%20a%20targeted%20iterative%20graph%20routing%20assembler%20for%20breakpoint%20assembly&amp;author=K.%20Chen&amp;author=L.%20Chen&amp;author=X.%20Fan&amp;author=J.%20Wallis&amp;author=L.%20Ding&amp;author=G.%20Weinstock&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=310-317" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.162883.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.162883.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.162883.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24307552" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carvalho</div> <div class="given-names">C. M.</div></div>, <div class="name string-name"><div class="surname">Lupski</div> <div class="given-names">J. R.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Mechanisms underlying structural variant formation in genomic disorders</div>. <div class="source">Nat Rev Genet</div> , <div class="volume">17</div>, <div class="fpage">224</div>–<div class="lpage">238</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20underlying%20structural%20variant%20formation%20in%20genomic%20disorders&amp;author=C.%20M.%20Carvalho&amp;author=J.%20R.%20Lupski&amp;publication_year=2016&amp;journal=Nat%20Rev%20Genet&amp;volume=17&amp;pages=224-238" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg.2015.25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg.2015.25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg.2015.25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26924765" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Daetwyler</div> <div class="given-names">H. D.</div></div>, <div class="name string-name"><div class="surname">Capitan</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Pausch</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Whole-genome sequencing of 234 bulls facilitates mapping of monogenic and complex traits in cattle</div>. <div class="source">Nat. Genet</div> ., <div class="volume">46</div>, <div class="fpage">858</div>–<div class="lpage">865</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Whole-genome%20sequencing%20of%20234%20bulls%20facilitates%20mapping%20of%20monogenic%20and%20complex%20traits%20in%20cattle&amp;author=H.%20D.%20Daetwyler&amp;author=A.%20Capitan&amp;author=H.%20Pausch&amp;publication_year=2014&amp;journal=Nat.%20Genet&amp;volume=46&amp;pages=858-865" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25017103" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brondum</div> <div class="given-names">R. F.</div></div>, <div class="name string-name"><div class="surname">Guldbrandtsen</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Sahana</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Lund</div> <div class="given-names">M. S.</div></div>, <div class="name string-name"><div class="surname">Su</div> <div class="given-names">G.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Strategies for imputation to whole genome sequence using a single or multi-breed reference population in cattle</div>. <div class="source">BMC Genomics</div> , <div class="volume">15</div>, <div class="fpage">728</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Strategies%20for%20imputation%20to%20whole%20genome%20sequence%20using%20a%20single%20or%20multi-breed%20reference%20population%20in%20cattle&amp;author=R.%20F.%20Brondum&amp;author=B.%20Guldbrandtsen&amp;author=G.%20Sahana&amp;author=M.%20S.%20Lund&amp;author=G.%20Su&amp;publication_year=2014&amp;journal=BMC%20Genomics&amp;volume=15&amp;pages=728" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-15-728" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-15-728" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-15-728"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25164068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jansen</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Aigner</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Pausch</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Assessment of the genomic variation in a cattle population by re-sequencing of key animals at low to medium coverage</div>. <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">446</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Assessment%20of%20the%20genomic%20variation%20in%20a%20cattle%20population%20by%20re-sequencing%20of%20key%20animals%20at%20low%20to%20medium%20coverage&amp;author=S.%20Jansen&amp;author=B.%20Aigner&amp;author=H.%20Pausch&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=446" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-446" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-446" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-446"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23826801" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Boussaha</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Esquerre</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Barbieri</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Genome-wide study of structural variants in bovine holstein, montbeliarde and normande dairy breeds</div>. <div class="source">PLoS One</div> , <div class="volume">10</div>, <div class="fpage">e0135931</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20study%20of%20structural%20variants%20in%20bovine%20holstein%2C%20montbeliarde%20and%20normande%20dairy%20breeds&amp;author=M.%20Boussaha&amp;author=D.%20Esquerre&amp;author=J.%20Barbieri&amp;publication_year=2015&amp;journal=PLoS%20One&amp;volume=10&amp;pages=e0135931" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0135931" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0135931" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0135931"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26317361" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Chamberlain</div> <div class="given-names">A. J.</div></div>, <div class="name string-name"><div class="surname">Reich</div> <div class="given-names">C. M.</div></div>, <div class="name string-name"><div class="surname">Daetwyler</div> <div class="given-names">H. D.</div></div>, <div class="name string-name"><div class="surname">Hayes</div> <div class="given-names">B. J.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">Detection and validation of structural variations in bovine whole-genome sequence data</div>. <div class="source">Genet. Select. Evol</div> ., <div class="volume">49</div>, <div class="fpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20and%20validation%20of%20structural%20variations%20in%20bovine%20whole-genome%20sequence%20data&amp;author=L.%20Chen&amp;author=A.%20J.%20Chamberlain&amp;author=C.%20M.%20Reich&amp;author=H.%20D.%20Daetwyler&amp;author=B.%20J.%20Hayes&amp;publication_year=2017&amp;journal=Genet.%20Select.%20Evol&amp;volume=49&amp;pages=13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12711-017-0286-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12711-017-0286-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12711-017-0286-5"> </span></div> </div></div></div></div></div><div content-id="dsx037-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>. <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">1760</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-1760" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">The Sequence Alignment/Map format and SAMtools</div>. <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">2079</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Sequence%20Alignment%2FMap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-2079" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McKenna</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hanna</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Banks</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">The Genome Analysis Toolkit: a MapReduce framework for analyzing next-generation DNA sequencing data</div>. <div class="source">Genome Res</div> ., <div class="volume">20</div>, <div class="fpage">1297</div>–<div class="lpage">1303</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Genome%20Analysis%20Toolkit%3A%20a%20MapReduce%20framework%20for%20analyzing%20next-generation%20DNA%20sequencing%20data&amp;author=A.%20McKenna&amp;author=M.%20Hanna&amp;author=E.%20Banks&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=1297-1303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.107524.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.107524.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.107524.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20644199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">R. E.</div></div>, <div class="name string-name"><div class="surname">Korn</div> <div class="given-names">J. M.</div></div>, <div class="name string-name"><div class="surname">Nemesh</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">McCarroll</div> <div class="given-names">S. A.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Discovery and genotyping of genome structural polymorphism by sequencing on a population scale</div>. <div class="source">Nat. Genet</div> ., <div class="volume">43</div>, <div class="fpage">269</div>–<div class="lpage">276</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20and%20genotyping%20of%20genome%20structural%20polymorphism%20by%20sequencing%20on%20a%20population%20scale&amp;author=R.%20E.%20Handsaker&amp;author=J.%20M.%20Korn&amp;author=J.%20Nemesh&amp;author=S.%20A.%20McCarroll&amp;publication_year=2011&amp;journal=Nat.%20Genet&amp;volume=43&amp;pages=269-276" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.768" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.768" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.768"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21317889" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Karolchik</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Hinrichs</div> <div class="given-names">A. S.</div></div>, <div class="name string-name"><div class="surname">Furey</div> <div class="given-names">T. S.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">The UCSC Table Browser data retrieval tool</div>. <div class="source">Nucleic Acids Res.</div>  <div class="volume">32</div>, <div class="fpage">D493</div>–<div class="lpage">496</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20UCSC%20Table%20Browser%20data%20retrieval%20tool&amp;author=D.%20Karolchik&amp;author=A.%20S.%20Hinrichs&amp;author=T.%20S.%20Furey&amp;publication_year=2004&amp;journal=Nucleic%20Acids%20Res.&amp;volume=32&amp;pages=D493-496" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkh103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkh103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkh103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14681465" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kent</div> <div class="given-names">W. J.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">BLAT–the BLAST-like alignment tool</div>. <div class="source">Genome Res</div> ., <div class="volume">12</div>, <div class="fpage">656</div>–<div class="lpage">664</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%93the%20BLAST-like%20alignment%20tool&amp;author=W.%20J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=656-664" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Miller</div> <div class="given-names">S. A.</div></div>, <div class="name string-name"><div class="surname">Dykes</div> <div class="given-names">D. D.</div></div>, <div class="name string-name"><div class="surname">Polesky</div> <div class="given-names">H. F.</div></div></div> <div class="year">1988</div>, 
<div class="article-title">A simple salting out procedure for extracting DNA from human nucleated cells</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">16</div>, <div class="fpage">1215</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20simple%20salting%20out%20procedure%20for%20extracting%20DNA%20from%20human%20nucleated%20cells&amp;author=S.%20A.%20Miller&amp;author=D.%20D.%20Dykes&amp;author=H.%20F.%20Polesky&amp;publication_year=1988&amp;journal=Nucleic%20Acids%20Res&amp;volume=16&amp;pages=1215" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/16.3.1215" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F16.3.1215" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F16.3.1215"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3344216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Purcell</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Neale</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Todd-Brown</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">PLINK: a tool set for whole-genome association and population-based linkage analyses</div>. <div class="source">Am. J. Hum. Genet</div> ., <div class="volume">81</div>, <div class="fpage">559</div>–<div class="lpage">575</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PLINK%3A%20a%20tool%20set%20for%20whole-genome%20association%20and%20population-based%20linkage%20analyses&amp;author=S.%20Purcell&amp;author=B.%20Neale&amp;author=K.%20Todd-Brown&amp;publication_year=2007&amp;journal=Am.%20J.%20Hum.%20Genet&amp;volume=81&amp;pages=559-575" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/519795" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F519795" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F519795"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17701901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Redon</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ishikawa</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Fitch</div> <div class="given-names">K. R.</div></div></div>, et al.   <div class="year">2006</div>, 
<div class="article-title">Global variation in copy number in the human genome</div>. <div class="source">Nature</div> , <div class="volume">444</div>, <div class="fpage">444</div>–<div class="lpage">454</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20variation%20in%20copy%20number%20in%20the%20human%20genome&amp;author=R.%20Redon&amp;author=S.%20Ishikawa&amp;author=K.%20R.%20Fitch&amp;publication_year=2006&amp;journal=Nature&amp;volume=444&amp;pages=444-454" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05329" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05329" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05329"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17122850" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wright</div> <div class="given-names">S.</div></div></div> <div class="year">1931</div>, 
<div class="article-title">Evolution in Mendelian Populations</div>. <div class="source">Genetics</div> , <div class="volume">16</div>, <div class="fpage">97</div>–<div class="lpage">159</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20in%20Mendelian%20Populations&amp;author=S.%20Wright&amp;publication_year=1931&amp;journal=Genetics&amp;volume=16&amp;pages=97-159" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17246615" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McLaren</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Gil</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Hunt</div> <div class="given-names">S. E.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The Ensembl Variant Effect Predictor</div>. <div class="source">Genome Biol</div> ., <div class="volume">17</div>, <div class="fpage">122</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Ensembl%20Variant%20Effect%20Predictor&amp;author=W.%20McLaren&amp;author=L.%20Gil&amp;author=S.%20E.%20Hunt&amp;publication_year=2016&amp;journal=Genome%20Biol&amp;volume=17&amp;pages=122" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-016-0974-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-016-0974-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-016-0974-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27268795" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R. D.</div></div>, <div class="name string-name"><div class="surname">Attwood</div> <div class="given-names">T. K.</div></div>, <div class="name string-name"><div class="surname">Babbitt</div> <div class="given-names">P. C.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">InterPro in 2017-beyond protein family and domain annotations</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">45</div>, <div class="fpage">D190</div>–<div class="lpage">D199</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterPro%20in%202017-beyond%20protein%20family%20and%20domain%20annotations&amp;author=R.%20D.%20Finn&amp;author=T.%20K.%20Attwood&amp;author=P.%20C.%20Babbitt&amp;publication_year=2017&amp;journal=Nucleic%20Acids%20Res&amp;volume=45&amp;pages=D190-D199" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkw1107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkw1107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkw1107"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27899635" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Finn</div> <div class="given-names">R. D.</div></div>, <div class="name string-name"><div class="surname">Coggill</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Eberhardt</div> <div class="given-names">R. Y.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The Pfam protein families database: towards a more sustainable future</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">D279</div>–<div class="lpage">285</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Pfam%20protein%20families%20database%3A%20towards%20a%20more%20sustainable%20future&amp;author=R.%20D.%20Finn&amp;author=P.%20Coggill&amp;author=R.%20Y.%20Eberhardt&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=D279-285" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv1344" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv1344" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv1344"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26673716" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kanehisa</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Furumichi</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Tanabe</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Morishima</div> <div class="given-names">K.</div></div></div> <div class="year">2017</div>, 
<div class="article-title">KEGG: new perspectives on genomes, pathways, diseases and drugs</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">45</div>, <div class="fpage">D353</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KEGG%3A%20new%20perspectives%20on%20genomes%2C%20pathways%2C%20diseases%20and%20drugs&amp;author=M.%20Kanehisa&amp;author=M.%20Furumichi&amp;author=M.%20Tanabe&amp;author=Y.%20Sato&amp;author=K.%20Morishima&amp;publication_year=2017&amp;journal=Nucleic%20Acids%20Res&amp;volume=45&amp;pages=D353-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkw1092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkw1092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkw1092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27899662" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Szklarczyk</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Franceschini</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Wyder</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">STRING v10: protein-protein interaction networks, integrated over the tree of life</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">43</div>, <div class="fpage">D447</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=STRING%20v10%3A%20protein-protein%20interaction%20networks%2C%20integrated%20over%20the%20tree%20of%20life&amp;author=D.%20Szklarczyk&amp;author=A.%20Franceschini&amp;author=S.%20Wyder&amp;publication_year=2015&amp;journal=Nucleic%20Acids%20Res&amp;volume=43&amp;pages=D447-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku1003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku1003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku1003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25352553" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yates</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Akanni</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Amode</div> <div class="given-names">M. R.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Ensembl 2016</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">D710</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ensembl%202016&amp;author=A.%20Yates&amp;author=W.%20Akanni&amp;author=M.%20R.%20Amode&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=D710-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv1157" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv1157" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv1157"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26687719" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kinsella</div> <div class="given-names">R. J.</div></div>, <div class="name string-name"><div class="surname">Kahari</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Haider</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Ensembl BioMarts: a hub for data retrieval across taxonomic space</div>. <div class="source">Database (Oxford)</div> , <div class="volume">2011</div>, <div class="fpage">bar030</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ensembl%20BioMarts%3A%20a%20hub%20for%20data%20retrieval%20across%20taxonomic%20space&amp;author=R.%20J.%20Kinsella&amp;author=A.%20Kahari&amp;author=S.%20Haider&amp;publication_year=2011&amp;journal=Database%20(Oxford)&amp;volume=2011&amp;pages=bar030" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/database/bar030" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdatabase%2Fbar030" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdatabase%2Fbar030"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21785142" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dickinson</div> <div class="given-names">M. E.</div></div>, <div class="name string-name"><div class="surname">Flenniken</div> <div class="given-names">A. M.</div></div>, <div class="name string-name"><div class="surname">Ji</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">High-throughput discovery of novel developmental phenotypes</div>. <div class="source">Nature</div> , <div class="volume">537</div>, <div class="fpage">508</div>–<div class="lpage">14</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20discovery%20of%20novel%20developmental%20phenotypes&amp;author=M.%20E.%20Dickinson&amp;author=A.%20M.%20Flenniken&amp;author=X.%20Ji&amp;publication_year=2016&amp;journal=Nature&amp;volume=537&amp;pages=508-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature19356" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature19356" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature19356"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27626380" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hu</div> <div class="given-names">Z. L.</div></div>, <div class="name string-name"><div class="surname">Park</div> <div class="given-names">C. A.</div></div>, <div class="name string-name"><div class="surname">Reecy</div> <div class="given-names">J. M.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Developmental progress and current status of the Animal QTLdb</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">44</div>, <div class="fpage">D827</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Developmental%20progress%20and%20current%20status%20of%20the%20Animal%20QTLdb&amp;author=Z.%20L.%20Hu&amp;author=C.%20A.%20Park&amp;author=J.%20M.%20Reecy&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=D827-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv1233" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv1233" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv1233"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26602686" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="collab">RStudio Team</div> <div class="year">2016</div>, <div class="source">RStudio: integrated development environment for R</div> . 
<div class="publisher-name">RStudio, Inc</div>.: 
<div class="publisher-loc">Boston, MA</div>.</div></div></div></div><div content-id="dsx037-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="collab">R Core Team</div> <div class="year">2016</div>, <div class="source">R: A language and environment for statistical computing</div> . 
<div class="publisher-name">R Foundation for Statistical Computing</div>: 
<div class="publisher-loc">Vienna, Austria</div>.</div></div></div></div><div content-id="dsx037-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quinlan</div> <div class="given-names">A. R.</div></div>, <div class="name string-name"><div class="surname">Hall</div> <div class="given-names">I. M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">BEDTools: a flexible suite of utilities for comparing genomic features</div>. <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">841</div>–<div class="lpage">842</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BEDTools%3A%20a%20flexible%20suite%20of%20utilities%20for%20comparing%20genomic%20features&amp;author=A.%20R.%20Quinlan&amp;author=I.%20M.%20Hall&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=841-842" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq033" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq033" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq033"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20110278" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lappalainen</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Lopez</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Skipper</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">DbVar and DGVa: public archives for genomic structural variation</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">41</div>, <div class="fpage">D936</div>–<div class="lpage">941</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DbVar%20and%20DGVa%3A%20public%20archives%20for%20genomic%20structural%20variation&amp;author=I.%20Lappalainen&amp;author=J.%20Lopez&amp;author=L.%20Skipper&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res&amp;volume=41&amp;pages=D936-941" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1213" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1213" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1213"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23193291" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McCarroll</div> <div class="given-names">S. A.</div></div>, <div class="name string-name"><div class="surname">Kuruvilla</div> <div class="given-names">F. G.</div></div>, <div class="name string-name"><div class="surname">Korn</div> <div class="given-names">J. M.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Integrated detection and population-genetic analysis of SNPs and copy number variation</div>. <div class="source">Nat. Genet</div> ., <div class="volume">40</div>, <div class="fpage">1166</div>–<div class="lpage">74</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrated%20detection%20and%20population-genetic%20analysis%20of%20SNPs%20and%20copy%20number%20variation&amp;author=S.%20A.%20McCarroll&amp;author=F.%20G.%20Kuruvilla&amp;author=J.%20M.%20Korn&amp;publication_year=2008&amp;journal=Nat.%20Genet&amp;volume=40&amp;pages=1166-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.238" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.238" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.238"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18776908" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Conrad</div> <div class="given-names">D. F.</div></div>, <div class="name string-name"><div class="surname">Pinto</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Redon</div> <div class="given-names">R.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Origins and functional impact of copy number variation in the human genome</div>. <div class="source">Nature</div> , <div class="volume">464</div>, <div class="fpage">704</div>–<div class="lpage">12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origins%20and%20functional%20impact%20of%20copy%20number%20variation%20in%20the%20human%20genome&amp;author=D.%20F.%20Conrad&amp;author=D.%20Pinto&amp;author=R.%20Redon&amp;publication_year=2010&amp;journal=Nature&amp;volume=464&amp;pages=704-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08516" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08516" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08516"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19812545" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">R. E.</div></div>, <div class="name string-name"><div class="surname">Van Doren</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Berman</div> <div class="given-names">J. R.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Large multiallelic copy number variations in humans</div>. <div class="source">Nat. Genet</div> ., <div class="volume">47</div>, <div class="fpage">296</div>–<div class="lpage">303</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large%20multiallelic%20copy%20number%20variations%20in%20humans&amp;author=R.%20E.%20Handsaker&amp;author=V.%20Van%20Doren&amp;author=J.%20R.%20Berman&amp;publication_year=2015&amp;journal=Nat.%20Genet&amp;volume=47&amp;pages=296-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.3200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.3200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.3200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25621458" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Guldbrandtsen</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Bosse</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Lund</div> <div class="given-names">M. S.</div></div>, <div class="name string-name"><div class="surname">Sahana</div> <div class="given-names">G.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Runs of homozygosity and distribution of functional variants in the cattle genome</div>. <div class="source">BMC Genomics</div> , <div class="volume">16</div>, <div class="fpage">542</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Runs%20of%20homozygosity%20and%20distribution%20of%20functional%20variants%20in%20the%20cattle%20genome&amp;author=Q.%20Zhang&amp;author=B.%20Guldbrandtsen&amp;author=M.%20Bosse&amp;author=M.%20S.%20Lund&amp;author=G.%20Sahana&amp;publication_year=2015&amp;journal=BMC%20Genomics&amp;volume=16&amp;pages=542" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12864-015-1715-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12864-015-1715-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12864-015-1715-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26198692" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bovine HapMap</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Gibbs</div> <div class="given-names">R. A.</div></div>, <div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">J. F.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Genome-wide survey of SNP variation uncovers the genetic structure of cattle breeds</div>. <div class="source">Science</div> , <div class="volume">324</div>, <div class="fpage">528</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20survey%20of%20SNP%20variation%20uncovers%20the%20genetic%20structure%20of%20cattle%20breeds&amp;author=C.%20Bovine%20HapMap&amp;author=R.%20A.%20Gibbs&amp;author=J.%20F.%20Taylor&amp;publication_year=2009&amp;journal=Science&amp;volume=324&amp;pages=528-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1167936" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1167936" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1167936"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19390050" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Sahana</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">De Koning</div> <div class="given-names">D. J.</div></div>, <div class="name string-name"><div class="surname">Guldbrandtsen</div> <div class="given-names">B.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Genome-wide association studies of growth traits in three dairy cattle breeds using whole-genome sequence data</div>. <div class="source">J. Anim. Sci</div> ., <div class="volume">94</div>, <div class="fpage">1426</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20studies%20of%20growth%20traits%20in%20three%20dairy%20cattle%20breeds%20using%20whole-genome%20sequence%20data&amp;author=X.%20Mao&amp;author=G.%20Sahana&amp;author=D.%20J.%20De%20Koning&amp;author=B.%20Guldbrandtsen&amp;publication_year=2016&amp;journal=J.%20Anim.%20Sci&amp;volume=94&amp;pages=1426-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2527/jas.2015-9838" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2527%2Fjas.2015-9838" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2527%2Fjas.2015-9838"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27136002" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Brondum</div> <div class="given-names">R. F.</div></div>, <div class="name string-name"><div class="surname">Rius-Vilarrasa</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Stranden</div> <div class="given-names">I.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Reliabilities of genomic prediction using combined reference data of the Nordic Red dairy cattle populations</div>. <div class="source">J. Dairy Sci</div> ., <div class="volume">94</div>, <div class="fpage">4700</div>–<div class="lpage">4707</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reliabilities%20of%20genomic%20prediction%20using%20combined%20reference%20data%20of%20the%20Nordic%20Red%20dairy%20cattle%20populations&amp;author=R.%20F.%20Brondum&amp;author=E.%20Rius-Vilarrasa&amp;author=I.%20Stranden&amp;publication_year=2011&amp;journal=J.%20Dairy%20Sci&amp;volume=94&amp;pages=4700-4707" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3168/jds.2010-3765" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3168%2Fjds.2010-3765" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3168%2Fjds.2010-3765"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21854944" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xu</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Bickhart</div> <div class="given-names">D. M.</div></div>, <div class="name string-name"><div class="surname">Cole</div> <div class="given-names">J. B.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Genomic signatures reveal new evidences for selection of important traits in domestic cattle</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">32</div>, <div class="fpage">711</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20signatures%20reveal%20new%20evidences%20for%20selection%20of%20important%20traits%20in%20domestic%20cattle&amp;author=L.%20Xu&amp;author=D.%20M.%20Bickhart&amp;author=J.%20B.%20Cole&amp;publication_year=2015&amp;journal=Mol.%20Biol.%20Evol&amp;volume=32&amp;pages=711-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msu333" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsu333" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsu333"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25431480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cole</div> <div class="given-names">J. B.</div></div>, <div class="name string-name"><div class="surname">Waurich</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wensch-Dorendorf</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Bickhart</div> <div class="given-names">D. M.</div></div>, <div class="name string-name"><div class="surname">Swalve</div> <div class="given-names">H. H.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">A genome-wide association study of calf birth weight in Holstein cattle using single nucleotide polymorphisms and phenotypes predicted from auxiliary traits</div>. <div class="source">J Dairy Sci</div> , <div class="volume">97</div>, <div class="fpage">3156</div>–<div class="lpage">3172</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome-wide%20association%20study%20of%20calf%20birth%20weight%20in%20Holstein%20cattle%20using%20single%20nucleotide%20polymorphisms%20and%20phenotypes%20predicted%20from%20auxiliary%20traits&amp;author=J.%20B.%20Cole&amp;author=B.%20Waurich&amp;author=M.%20Wensch-Dorendorf&amp;author=D.%20M.%20Bickhart&amp;author=H.%20H.%20Swalve&amp;publication_year=2014&amp;journal=J%20Dairy%20Sci&amp;volume=97&amp;pages=3156-3172" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3168/jds.2013-7409" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3168%2Fjds.2013-7409" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3168%2Fjds.2013-7409"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24612794" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Howard</div> <div class="given-names">J. T.</div></div>, <div class="name string-name"><div class="surname">Kachman</div> <div class="given-names">S. D.</div></div>, <div class="name string-name"><div class="surname">Snelling</div> <div class="given-names">W. M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Beef cattle body temperature during climatic stress: a genome-wide association study</div>. <div class="source">Int. J. Biometeorol</div> ., <div class="volume">58</div>, <div class="fpage">1665</div>–<div class="lpage">1672</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Beef%20cattle%20body%20temperature%20during%20climatic%20stress%3A%20a%20genome-wide%20association%20study&amp;author=J.%20T.%20Howard&amp;author=S.%20D.%20Kachman&amp;author=W.%20M.%20Snelling&amp;publication_year=2014&amp;journal=Int.%20J.%20Biometeorol&amp;volume=58&amp;pages=1665-1672" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00484-013-0773-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00484-013-0773-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00484-013-0773-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24362770" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Buitenhuis</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Poulsen</div> <div class="given-names">N. A.</div></div>, <div class="name string-name"><div class="surname">Gebreyesus</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Larsen</div> <div class="given-names">L. B.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Estimation of genetic parameters and detection of chromosomal regions affecting the major milk proteins and their post translational modifications in Danish Holstein and Danish Jersey cattle</div>. <div class="source">BMC Genet</div> ., <div class="volume">17</div>, <div class="fpage">114</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimation%20of%20genetic%20parameters%20and%20detection%20of%20chromosomal%20regions%20affecting%20the%20major%20milk%20proteins%20and%20their%20post%20translational%20modifications%20in%20Danish%20Holstein%20and%20Danish%20Jersey%20cattle&amp;author=B.%20Buitenhuis&amp;author=N.%20A.%20Poulsen&amp;author=G.%20Gebreyesus&amp;author=L.%20B.%20Larsen&amp;publication_year=2016&amp;journal=BMC%20Genet&amp;volume=17&amp;pages=114" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12863-016-0421-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12863-016-0421-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12863-016-0421-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27485317" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McClure</div> <div class="given-names">M. C.</div></div>, <div class="name string-name"><div class="surname">Morsci</div> <div class="given-names">N. S.</div></div>, <div class="name string-name"><div class="surname">Schnabel</div> <div class="given-names">R. D.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">A genome scan for quantitative trait loci influencing carcass, post-natal growth and reproductive traits in commercial Angus cattle</div>. <div class="source">Anim. Genet</div> ., <div class="volume">41</div>, <div class="fpage">597</div>–<div class="lpage">607</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genome%20scan%20for%20quantitative%20trait%20loci%20influencing%20carcass%2C%20post-natal%20growth%20and%20reproductive%20traits%20in%20commercial%20Angus%20cattle&amp;author=M.%20C.%20McClure&amp;author=N.%20S.%20Morsci&amp;author=R.%20D.%20Schnabel&amp;publication_year=2010&amp;journal=Anim.%20Genet&amp;volume=41&amp;pages=597-607" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/age.2010.41.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fage.2010.41.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fage.2010.41.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20477797" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sahana</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Guldbrandtsen</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Lund</div> <div class="given-names">M. S.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Genome-wide association study for calving traits in Danish and Swedish Holstein cattle</div>. <div class="source">J. Dairy Sci</div> ., <div class="volume">94</div>, <div class="fpage">479</div>–<div class="lpage">486</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20for%20calving%20traits%20in%20Danish%20and%20Swedish%20Holstein%20cattle&amp;author=G.%20Sahana&amp;author=B.%20Guldbrandtsen&amp;author=M.%20S.%20Lund&amp;publication_year=2011&amp;journal=J.%20Dairy%20Sci&amp;volume=94&amp;pages=479-486" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3168/jds.2010-3381" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3168%2Fjds.2010-3381" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3168%2Fjds.2010-3381"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21183059" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">McClure</div> <div class="given-names">M. C.</div></div>, <div class="name string-name"><div class="surname">Ramey</div> <div class="given-names">H. R.</div></div>, <div class="name string-name"><div class="surname">Rolf</div> <div class="given-names">M. M.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Genome-wide association analysis for quantitative trait loci influencing Warner-Bratzler shear force in five taurine cattle breeds</div>. <div class="source">Anim. Genet</div> ., <div class="volume">43</div>, <div class="fpage">662</div>–<div class="lpage">673</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20analysis%20for%20quantitative%20trait%20loci%20influencing%20Warner-Bratzler%20shear%20force%20in%20five%20taurine%20cattle%20breeds&amp;author=M.%20C.%20McClure&amp;author=H.%20R.%20Ramey&amp;author=M.%20M.%20Rolf&amp;publication_year=2012&amp;journal=Anim.%20Genet&amp;volume=43&amp;pages=662-673" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/age.2012.43.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fage.2012.43.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fage.2012.43.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22497286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Snelling</div> <div class="given-names">W. M.</div></div>, <div class="name string-name"><div class="surname">Allan</div> <div class="given-names">M. F.</div></div>, <div class="name string-name"><div class="surname">Keele</div> <div class="given-names">J. W.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Genome-wide association study of growth in crossbred beef cattle</div>. <div class="source">J. Anim. Sci</div> ., <div class="volume">88</div>, <div class="fpage">837</div>–<div class="lpage">848</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20of%20growth%20in%20crossbred%20beef%20cattle&amp;author=W.%20M.%20Snelling&amp;author=M.%20F.%20Allan&amp;author=J.%20W.%20Keele&amp;publication_year=2010&amp;journal=J.%20Anim.%20Sci&amp;volume=88&amp;pages=837-848" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2527/jas.2009-2257" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2527%2Fjas.2009-2257" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2527%2Fjas.2009-2257"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19966163" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cole</div> <div class="given-names">J. B.</div></div>, <div class="name string-name"><div class="surname">Wiggans</div> <div class="given-names">G. R.</div></div>, <div class="name string-name"><div class="surname">Ma</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Genome-wide association analysis of thirty one production, health, reproduction and body conformation traits in contemporary U.S. Holstein cows</div>. <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">408</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20analysis%20of%20thirty%20one%20production%2C%20health%2C%20reproduction%20and%20body%20conformation%20traits%20in%20contemporary%20U.S.%20Holstein%20cows&amp;author=J.%20B.%20Cole&amp;author=G.%20R.%20Wiggans&amp;author=L.%20Ma&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=408" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-408" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-408" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-408"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21831322" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lobago</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Gustafsson</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Bekana</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Beckers</div> <div class="given-names">J. F.</div></div>, <div class="name string-name"><div class="surname">Kindahl</div> <div class="given-names">H.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Clinical features and hormonal profiles of cloprostenol-induced early abortions in heifers monitored by ultrasonography</div>. <div class="source">Acta Vet. Scand</div> ., <div class="volume">48</div>, <div class="fpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clinical%20features%20and%20hormonal%20profiles%20of%20cloprostenol-induced%20early%20abortions%20in%20heifers%20monitored%20by%20ultrasonography&amp;author=F.%20Lobago&amp;author=H.%20Gustafsson&amp;author=M.%20Bekana&amp;author=J.%20F.%20Beckers&amp;author=H.%20Kindahl&amp;publication_year=2006&amp;journal=Acta%20Vet.%20Scand&amp;volume=48&amp;pages=23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1751-0147-48-23" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1751-0147-48-23" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1751-0147-48-23"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17121683" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sandri</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Stefanon</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Loor</div> <div class="given-names">J. J.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Transcriptome profiles of whole blood in Italian Holstein and Italian Simmental lactating cows diverging for genetic merit for milk protein</div>. <div class="source">J. Dairy Sci</div> ., <div class="volume">98</div>, <div class="fpage">6119</div>–<div class="lpage">6127</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20profiles%20of%20whole%20blood%20in%20Italian%20Holstein%20and%20Italian%20Simmental%20lactating%20cows%20diverging%20for%20genetic%20merit%20for%20milk%20protein&amp;author=M.%20Sandri&amp;author=B.%20Stefanon&amp;author=J.%20J.%20Loor&amp;publication_year=2015&amp;journal=J.%20Dairy%20Sci&amp;volume=98&amp;pages=6119-6127" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3168/jds.2014-9049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3168%2Fjds.2014-9049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3168%2Fjds.2014-9049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26142863" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Araujo</div> <div class="given-names">R. N.</div></div>, <div class="name string-name"><div class="surname">Padilha</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Zarlenga</div> <div class="given-names">D.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Use of a candidate gene array to delineate gene expression patterns in cattle selected for resistance or susceptibility to intestinal nematodes</div>. <div class="source">Vet. Parasitol</div> ., <div class="volume">162</div>, <div class="fpage">106</div>–<div class="lpage">115</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Use%20of%20a%20candidate%20gene%20array%20to%20delineate%20gene%20expression%20patterns%20in%20cattle%20selected%20for%20resistance%20or%20susceptibility%20to%20intestinal%20nematodes&amp;author=R.%20N.%20Araujo&amp;author=T.%20Padilha&amp;author=D.%20Zarlenga&amp;publication_year=2009&amp;journal=Vet.%20Parasitol&amp;volume=162&amp;pages=106-115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.vetpar.2008.12.017" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.vetpar.2008.12.017" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.vetpar.2008.12.017"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19375862" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Sahana</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Su</div> <div class="given-names">G.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Integrating sequence-based GWAS and RNA-seq provides novel insights into the genetic basis of mastitis and milk production in dairy cattle</div>. <div class="source">Sci. Rep</div> ., <div class="volume">7</div>, <div class="fpage">45560</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrating%20sequence-based%20GWAS%20and%20RNA-seq%20provides%20novel%20insights%20into%20the%20genetic%20basis%20of%20mastitis%20and%20milk%20production%20in%20dairy%20cattle&amp;author=L.%20Fang&amp;author=G.%20Sahana&amp;author=G.%20Su&amp;publication_year=2017&amp;journal=Sci.%20Rep&amp;volume=7&amp;pages=45560" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep45560" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep45560" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep45560"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28358110" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hurst</div> <div class="given-names">L. D.</div></div>, <div class="name string-name"><div class="surname">Smith</div> <div class="given-names">N. G.</div></div></div> <div class="year">1999</div>, 
<div class="article-title">Do essential genes evolve slowly?</div> <div class="source">Curr. Biol</div> ., <div class="volume">9</div>, <div class="fpage">747</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Do%20essential%20genes%20evolve%20slowly%3F&amp;author=L.%20D.%20Hurst&amp;author=N.%20G.%20Smith&amp;publication_year=1999&amp;journal=Curr.%20Biol&amp;volume=9&amp;pages=747-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0960-9822(99)80334-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0960-9822(99)80334-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0960-9822(99)80334-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10421576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Blomen</div> <div class="given-names">V. A.</div></div>, <div class="name string-name"><div class="surname">Majek</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Jae</div> <div class="given-names">L. T.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Gene essentiality and synthetic lethality in haploid human cells</div>. <div class="source">Science</div> , <div class="volume">350</div>, <div class="fpage">1092</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20essentiality%20and%20synthetic%20lethality%20in%20haploid%20human%20cells&amp;author=V.%20A.%20Blomen&amp;author=P.%20Majek&amp;author=L.%20T.%20Jae&amp;publication_year=2015&amp;journal=Science&amp;volume=350&amp;pages=1092-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.aac7557" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aac7557" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.aac7557"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26472760" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Niimura</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Nei</div> <div class="given-names">M.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Extensive gains and losses of olfactory receptor genes in mammalian evolution</div>. <div class="source">PLoS One</div> , <div class="volume">2</div>, <div class="fpage">e708</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20gains%20and%20losses%20of%20olfactory%20receptor%20genes%20in%20mammalian%20evolution&amp;author=Y.%20Niimura&amp;author=M.%20Nei&amp;publication_year=2007&amp;journal=PLoS%20One&amp;volume=2&amp;pages=e708" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0000708" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0000708" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0000708"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17684554" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Van Ziffle</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Chehab</div> <div class="given-names">F. F.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Homozygous deletion of six olfactory receptor genes in a subset of individuals with Beta-thalassemia</div>. <div class="source">PLoS One</div> , <div class="volume">6</div>, <div class="fpage">e17327</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Homozygous%20deletion%20of%20six%20olfactory%20receptor%20genes%20in%20a%20subset%20of%20individuals%20with%20Beta-thalassemia&amp;author=J.%20Van%20Ziffle&amp;author=W.%20Yang&amp;author=F.%20F.%20Chehab&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e17327" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0017327" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0017327" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0017327"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21390308" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lee</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Nguyen</div> <div class="given-names">D. T.</div></div>, <div class="name string-name"><div class="surname">Choi</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Analysis of cattle olfactory subgenome: the first detail study on the characteristics of the complete olfactory receptor repertoire of a ruminant</div>. <div class="source">BMC Genomics</div> , <div class="volume">14</div>, <div class="fpage">596</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20cattle%20olfactory%20subgenome%3A%20the%20first%20detail%20study%20on%20the%20characteristics%20of%20the%20complete%20olfactory%20receptor%20repertoire%20of%20a%20ruminant&amp;author=K.%20Lee&amp;author=D.%20T.%20Nguyen&amp;author=M.%20Choi&amp;publication_year=2013&amp;journal=BMC%20Genomics&amp;volume=14&amp;pages=596" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-596" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-596" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-596"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24004971" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx037-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx037-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hastings</div> <div class="given-names">P. J.</div></div>, <div class="name string-name"><div class="surname">Lupski</div> <div class="given-names">J. R.</div></div>, <div class="name string-name"><div class="surname">Rosenberg</div> <div class="given-names">S. M.</div></div>, <div class="name string-name"><div class="surname">Ira</div> <div class="given-names">G.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Mechanisms of change in gene copy number</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">10</div>, <div class="fpage">551</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mechanisms%20of%20change%20in%20gene%20copy%20number&amp;author=P.%20J.%20Hastings&amp;author=J.%20R.%20Lupski&amp;author=S.%20M.%20Rosenberg&amp;author=G.%20Ira&amp;publication_year=2009&amp;journal=Nat.%20Rev.%20Genet&amp;volume=10&amp;pages=551-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2593" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2593" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2593"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19597530" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4107864&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/supplementary_figures_and_tables_dsx037.pdf?Expires=2147483647&amp;Signature=N8ekU-AB4-5zVt266s0FuDtj2aJWL~xy4OoOLYvUCqG5eWwdQ1T~hx3fJgruz9mUs~-gic4Ikx0v9SegHmwGCPkze9Kv68VZQdX55etEoJn~THklkmmMjgXPdT4Z6hbCYjDSZoUjw6h46V~h2D~93JPIAonauVegy07VuTGu1ULLUzb3SmNpuoCGtu9mnZYLC2~xIB~SmGTnvLejq8R~UU8F1OuEAfeFu0JRRyNhR50w4KNeh5IEYiHJO29zFin1eQ9YPexYjNL0gSk8XDfCde2m6Ydc8KivHx3eS2U3B~lKJkhpoI56Uel-YQr9K12aYJzJm90N~uuKou0aE8X90w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figures and Tables</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/1/10.1093_dnares_dsx037/3/s1_and_s2_dsx037.xls?Expires=2147483647&amp;Signature=vhdyIqhZPLuViQrCa8hpkyySgekxCBcznuz0oqoUHlTNOr1XKPwZSVPRHMacIv8nDZign9ABA23sFAQO-npvrF48ltv8UdCVFGMod664flAKuArLtJ4UzZ8Q-z9glSbhkJOHUEIXbmXgJBGz31pNp4C-qGH~VxVFQGogjtVnvJggQjRqqZBCZECFO7eNPHkMkQITtkc2IuOOt4n80IUS2Se-6rlPhsTeVqIFX2XtlFVBxgyge20XNsdt~~lNk8vtX56dIqwc8Cpu9uKTv3rSVIw93AFbBjB8uynY6TwdianwvTuICtBKwAmBUWIPaRrrpG-kpawxEsVlbO1bkaob1w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S1-S2</div></a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,519</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000425954100005&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx037"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4107864"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000425954100005&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f1%2f49%2f4107864">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000425954100005&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F1%2F49%2F4107864" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4107864'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-wide mapping of large deletions and their population-genetic properties in dairy cattle" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx037");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f1%2f49%2f4107864">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

