

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/22/1/101/442161","@type":"ScholarlyArticle","name":"A complete view of the genetic diversity of the  Escherichia coli  O-antigen biosynthesis gene cluster","datePublished":"2014-11-26","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/22/1","@type":"PublicationIssue","issueNumber":"1","datePublished":"2015-02-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu043","keywords":["<italic>E. coli</italic>","O-antigen biosynthesis gene cluster","horizontal gene transfer","O serogroup","genomic diversity"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/22/1/101/442161","author":[{"name":"Iguchi, Atsushi","affiliation":"1  Department of Animal and Grassland Sciences, Faculty of Agriculture ,  University of Miyazaki ,  Miyazaki 889-2192 ,  Japan","@type":"Person"},{"name":"Iyoda, Sunao","affiliation":"2  Department of Bacteriology I ,  National Institute of Infectious Diseases ,  Tokyo 162-8640 ,  Japan","@type":"Person"},{"name":"Kikuchi, Taisei","affiliation":"3  Division of Parasitology, Department of Infectious Diseases, Faculty of Medicine ,  University of Miyazaki ,  Miyazaki 889-1692 ,  Japan","@type":"Person"},{"name":"Ogura, Yoshitoshi","affiliation":"4  Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine ,  University of Miyazaki ,  Miyazaki 889-1692 ,  Japan    5  Division of Bioenvironmental Science, Frontier Science Research Center ,  University of Miyazaki ,  Miyazaki 889-1692 ,  Japan","@type":"Person"},{"name":"Katsura, Keisuke","affiliation":"5  Division of Bioenvironmental Science, Frontier Science Research Center ,  University of Miyazaki ,  Miyazaki 889-1692 ,  Japan","@type":"Person"},{"name":"Ohnishi, Makoto","affiliation":"2  Department of Bacteriology I ,  National Institute of Infectious Diseases ,  Tokyo 162-8640 ,  Japan","@type":"Person"},{"name":"Hayashi, Tetsuya","affiliation":"4  Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine ,  University of Miyazaki ,  Miyazaki 889-1692 ,  Japan    5  Division of Bioenvironmental Science, Frontier Science Research Center ,  University of Miyazaki ,  Miyazaki 889-1692 ,  Japan","@type":"Person"},{"name":"Thomson, Nicholas R.","affiliation":"6  Pathogen Genomics ,  The Wellcome Trust Sanger Institute ,  Cambridge CB10 1SA ,  UK    7  Department of Infectious and Tropical Diseases ,  London School of Hygiene and Tropical Medicine ,  London WC1E 7HT ,  UK","@type":"Person"}],"description":"Abstract.  The O antigen constitutes the outermost part of the lipopolysaccharide layer in Gram-negative bacteria. The chemical composition and structure of the","pageStart":"101","pageEnd":"107","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"A complete view of the genetic diversity of the  Escherichia coli  O-antigen biosynthesis gene cluster","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="complete view of the genetic diversity of the  Escherichia coli  O-antigen biosynthesis gene cluster" />
    <meta property="og:description" content="Abstract.  The O antigen constitutes the outermost part of the lipopolysaccharide layer in Gram-negative bacteria. The chemical composition and structure of the" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/22/1/101/442161" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-11-26" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Iguchi, Atsushi" /><meta name="citation_author_institution" content="Department of Animal and Grassland Sciences, Faculty of Agriculture, University of Miyazaki, Miyazaki 889-2192, Japan" /><meta name="citation_author" content="Iyoda, Sunao" /><meta name="citation_author_institution" content="Department of Bacteriology I, National Institute of Infectious Diseases, Tokyo 162-8640, Japan" /><meta name="citation_author" content="Kikuchi, Taisei" /><meta name="citation_author_institution" content="Division of Parasitology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki, Miyazaki 889-1692, Japan" /><meta name="citation_author" content="Ogura, Yoshitoshi" /><meta name="citation_author_institution" content="Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki, Miyazaki 889-1692, Japan" /><meta name="citation_author_institution" content="Division of Bioenvironmental Science, Frontier Science Research Center, University of Miyazaki, Miyazaki 889-1692, Japan" /><meta name="citation_author" content="Katsura, Keisuke" /><meta name="citation_author_institution" content="Division of Bioenvironmental Science, Frontier Science Research Center, University of Miyazaki, Miyazaki 889-1692, Japan" /><meta name="citation_author" content="Ohnishi, Makoto" /><meta name="citation_author_institution" content="Department of Bacteriology I, National Institute of Infectious Diseases, Tokyo 162-8640, Japan" /><meta name="citation_author" content="Hayashi, Tetsuya" /><meta name="citation_author_institution" content="Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine, University of Miyazaki, Miyazaki 889-1692, Japan" /><meta name="citation_author_institution" content="Division of Bioenvironmental Science, Frontier Science Research Center, University of Miyazaki, Miyazaki 889-1692, Japan" /><meta name="citation_author" content="Thomson, Nicholas R." /><meta name="citation_author_institution" content="Pathogen Genomics, The Wellcome Trust Sanger Institute, Cambridge CB10 1SA, UK" /><meta name="citation_author_institution" content="Department of Infectious and Tropical Diseases, London School of Hygiene and Tropical Medicine, London WC1E 7HT, UK" /><meta name="citation_title" content="A complete view of the genetic diversity of the Escherichia coli O-antigen biosynthesis gene cluster" /><meta name="citation_firstpage" content="101" /><meta name="citation_lastpage" content="107" /><meta name="citation_doi" content="10.1093/dnares/dsu043" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="22" /><meta name="citation_issue" content="1" /><meta name="citation_publication_date" content="2015/02/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Bacterial extracellular polysaccharides; Adv. Exp. Med. Biol.;  citation_year=2011;  citation_volume=715;  citation_pages=213-26; " /><meta name="citation_reference" content="citation_title=Structural diversity in Salmonella O antigens and its genetic basis; FEMS Microbiol. Rev.;  citation_year=2013;  citation_volume=38;  citation_pages=56-89; " /><meta name="citation_reference" content="citation_title=The structures of Escherichia coli O-polysaccharide antigens; FEMS Microbiol. Rev.;  citation_year=2006;  citation_volume=30;  citation_pages=382-403; " /><meta name="citation_reference" content="citation_title=Genetic and functional diversity of Pseudomonas aeruginosa lipopolysaccharide; Front Microbiol.;  citation_year=2011;  citation_volume=2;  citation_pages=118" /><meta name="citation_reference" content="citation_title=Diversity of lipopolysaccharide structures in Campylobacter jejuni; J. Infect. Dis.;  citation_year=1997;  citation_volume=176;  citation_issue=Suppl. 2;  citation_pages=S135-138; " /><meta name="citation_reference" content="citation_title=Emerging foodborne pathogens: Escherichia coli O157:H7 as a model of entry of a new pathogen into the food supply of the developed world; Epidemiol Rev.;  citation_year=1996;  citation_volume=18;  citation_pages=29-51; " /><meta name="citation_reference" content="citation_title=Shiga-toxin-producing Escherichia coli and haemolytic uraemic syndrome; Lancet;  citation_year=2005;  citation_volume=365;  citation_pages=1073-86; " /><meta name="citation_reference" content="citation_title=The emerging clinical importance of non-O157 Shiga toxin-producing Escherichia coli; Clin. Infect. Dis.;  citation_year=2006;  citation_volume=43;  citation_pages=1587-95; " /><meta name="citation_reference" content="citation_title=German outbreak of Escherichia coli O104:H4 associated with sprouts; N. Engl. J. Med.;  citation_year=2011;  citation_volume=365;  citation_pages=1763-70; " /><meta name="citation_reference" content="citation_title=Molecular epidemiology of Escherichia coli producing CTX-M beta-lactamases: the worldwide emergence of clone ST131 O25:H4; Int. J. Antimicrob. Agents;  citation_year=2010;  citation_volume=35;  citation_pages=316-21; " /><meta name="citation_reference" content="citation_title=Biosynthesis of O-antigens: genes and pathways involved in nucleotide sugar precursor synthesis and O-antigen assembly; Carbohydr. Res.;  citation_year=2003;  citation_volume=338;  citation_pages=2503-19; " /><meta name="citation_reference" content="citation_title=Detection of O antigens in Escherichia coli; Anim. Health Res. Rev.;  citation_year=2011;  citation_volume=12;  citation_pages=169-85; " /><meta name="citation_reference" content="citation_title=A precise reconstruction of the emergence and constrained radiations of Escherichia coli O157 portrayed by backbone concatenomic analysis; Proc. Natl Acad. Sci. USA;  citation_year=2009;  citation_volume=106;  citation_pages=8713-8; " /><meta name="citation_reference" content="citation_title=Wide distribution of O157-antigen biosynthesis gene clusters in Escherichia coli; PLoS ONE;  citation_year=2011;  citation_volume=6;  citation_pages=e23250" /><meta name="citation_reference" content="citation_title=Molecular characterization reveals three distinct clonal groups among clinical Shiga toxin-producing Escherichia coli strains of serogroup O103; J. Clin. Microbiol.;  citation_year=2012;  citation_volume=50;  citation_pages=2894-900; " /><meta name="citation_reference" content="citation_title=Identification of the two glycosyltransferase genes responsible for the difference between Escherichia coli O107 and O117 O-antigens; Glycobiology;  citation_year=2012;  citation_volume=22;  citation_pages=281-7; " /><meta name="citation_reference" content="citation_title=A group of Escherichia coli and Salmonella enterica O antigens sharing a common backbone structure; Microbiology;  citation_year=2007;  citation_volume=153;  citation_pages=2159-67; " /><meta name="citation_reference" content="citation_title=Novel microarray design for molecular serotyping of Shiga toxin-producing Escherichia coli isolated from fresh produce; Appl. Environ. Microbiol.;  citation_year=2014;  citation_volume=80;  citation_pages=4677-82; " /><meta name="citation_reference" content="citation_title=A rapid procedure for the detection and isolation of enterohaemorrhagic Escherichia coli (EHEC) serogroup O26, O103, O111, O118, O121, O145 and O157 strains and the aggregative EHEC O104:H4 strain from ready-to-eat vegetables; Int. J. Food Microbiol.;  citation_year=2012;  citation_volume=152;  citation_pages=19-30; " /><meta name="citation_reference" content="citation_title=Development of a serogroup-specific multiplex PCR assay to detect a set of Escherichia coli serogroups based on the identification of their O-antigen gene clusters; Mol. Cell Probes;  citation_year=2010;  citation_volume=24;  citation_pages=286-90; " /><meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice; Nucleic Acids Res.;  citation_year=1994;  citation_volume=22;  citation_pages=4673-80; " /><meta name="citation_reference" content="citation_title=MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0; Mol. Biol. Evol.;  citation_year=2007;  citation_volume=24;  citation_pages=1596-9; " /><meta name="citation_reference" content="citation_title=Deletion of the Escherichia coli O14:K7 O antigen gene cluster; Can. J. Microbiol.;  citation_year=2004;  citation_volume=50;  citation_pages=299-302; " /><meta name="citation_reference" content="citation_title=Multiple independent origins of Shigella clones of Escherichia coli and convergent evolution of many of their characteristics; Proc. Natl Acad. Sci. USA;  citation_year=2000;  citation_volume=97;  citation_pages=10567-72; " /><meta name="citation_reference" content="citation_title=Structure and genetics of Shigella O antigens; FEMS Microbiol. Rev.;  citation_year=2008;  citation_volume=32;  citation_pages=627-53; " /><meta name="citation_reference" content="citation_title=Emergence of a novel Shiga toxin-producing Escherichia coli O serogroup cross-reacting with Shigella boydii type 10; J. Clin. Microbiol.;  citation_year=2011;  citation_volume=49;  citation_pages=3678-80; " /><meta name="citation_reference" content="citation_title=Enteroaggregative Escherichia coli O78:H10, the cause of an outbreak of urinary tract infection; J. Clin. Microbiol.;  citation_year=2012;  citation_volume=50;  citation_pages=3703-11; " /><meta name="citation_reference" content="citation_title=Multi-locus sequence typing of enteroaggregative Escherichia coli isolates from Nigerian children uncovers multiple lineages; PLoS ONE;  citation_year=2010;  citation_volume=5;  citation_pages=e14093" /><meta name="citation_reference" content="citation_title=Phylogenetic comparisons reveal multiple acquisitions of the toxin genes by enterotoxigenic Escherichia coli strains of different evolutionary lineages; J. Clin. Microbiol.;  citation_year=2006;  citation_volume=44;  citation_pages=4528-36; " /><meta name="citation_reference" content="citation_title=Discovery and phylogenetic analysis of novel members of class b enterotoxigenic Escherichia coli adhesive fimbriae; J. Clin. Microbiol.;  citation_year=2011;  citation_volume=49;  citation_pages=1403-10; " /><meta name="citation_reference" content="citation_title=Characterization of farm, food, and clinical Shiga toxin-producing Escherichia coli (STEC) O113; Foodborne Pathog. Dis.;  citation_year=2012;  citation_volume=9;  citation_pages=1088-96; " /><meta name="citation_reference" content="citation_title=Phylogenetic and molecular analysis of food-borne shiga toxin-producing Escherichia coli; Appl. Environ. Microbiol.;  citation_year=2013;  citation_volume=79;  citation_pages=2731-40; " /><meta name="citation_reference" content="citation_title=Serotype-converting bacteriophages and O-antigen modification in Shigella flexneri; Trends Microbiol.;  citation_year=2000;  citation_volume=8;  citation_pages=17-23; " /><meta name="citation_reference" content="citation_title=A novel plasmid-encoded serotype conversion mechanism through addition of phosphoethanolamine to the O-antigen of Shigella flexneri; PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e46095" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/22/1/101/1211974/dsu043.pdf" /><meta name="description" content="Abstract.  The O antigen constitutes the outermost part of the lipopolysaccharide layer in Gram-negative bacteria. The chemical composition and structure of the" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/22/1/101/442161" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/22/1/101/442161" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/22/1/101/442161");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu043';
    </script>
<title>
	complete view of the genetic diversity of the Escherichia coli O-antigen biosynthesis gene cluster | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./442161" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BZJdPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJZ3VjaGksIEF0c3VzaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFuaW1hbCBhbmQgR3Jhc3NsYW5kIFNjaWVuY2VzLCBGYWN1bHR5IG9mIEFncmljdWx0dXJlLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCBNaXlhemFraSA4ODktMjE5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJJeW9kYSwgU3VuYW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJhY3RlcmlvbG9neSBJLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgSW5mZWN0aW91cyBEaXNlYXNlcywgVG9reW8gMTYyLTg2NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS2lrdWNoaSwgVGFpc2VpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgUGFyYXNpdG9sb2d5LCBEZXBhcnRtZW50IG9mIEluZmVjdGlvdXMgRGlzZWFzZXMsIEZhY3VsdHkgb2YgTWVkaWNpbmUsIFVuaXZlcnNpdHkgb2YgTWl5YXpha2ksIE1peWF6YWtpIDg4OS0xNjkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9ndXJhLCBZb3NoaXRvc2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgTWljcm9iaW9sb2d5LCBEZXBhcnRtZW50IG9mIEluZmVjdGlvdXMgRGlzZWFzZXMsIEZhY3VsdHkgb2YgTWVkaWNpbmUsIFVuaXZlcnNpdHkgb2YgTWl5YXpha2ksIE1peWF6YWtpIDg4OS0xNjkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIEJpb2Vudmlyb25tZW50YWwgU2NpZW5jZSwgRnJvbnRpZXIgU2NpZW5jZSBSZXNlYXJjaCBDZW50ZXIsIFVuaXZlcnNpdHkgb2YgTWl5YXpha2ksIE1peWF6YWtpIDg4OS0xNjkyLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkthdHN1cmEsIEtlaXN1a2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBCaW9lbnZpcm9ubWVudGFsIFNjaWVuY2UsIEZyb250aWVyIFNjaWVuY2UgUmVzZWFyY2ggQ2VudGVyLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCBNaXlhemFraSA4ODktMTY5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPaG5pc2hpLCBNYWtvdG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJhY3RlcmlvbG9neSBJLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgSW5mZWN0aW91cyBEaXNlYXNlcywgVG9reW8gMTYyLTg2NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGF5YXNoaSwgVGV0c3V5YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIE1pY3JvYmlvbG9neSwgRGVwYXJ0bWVudCBvZiBJbmZlY3Rpb3VzIERpc2Vhc2VzLCBGYWN1bHR5IG9mIE1lZGljaW5lLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCBNaXlhemFraSA4ODktMTY5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBCaW9lbnZpcm9ubWVudGFsIFNjaWVuY2UsIEZyb250aWVyIFNjaWVuY2UgUmVzZWFyY2ggQ2VudGVyLCBVbml2ZXJzaXR5IG9mIE1peWF6YWtpLCBNaXlhemFraSA4ODktMTY5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUaG9tc29uLCBOaWNob2xhcyBSLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlBhdGhvZ2VuIEdlbm9taWNzLCBUaGUgV2VsbGNvbWUgVHJ1c3QgU2FuZ2VyIEluc3RpdHV0ZSwgQ2FtYnJpZGdlIENCMTAgMVNBLCBVSyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgSW5mZWN0aW91cyBhbmQgVHJvcGljYWwgRGlzZWFzZXMsIExvbmRvbiBTY2hvb2wgb2YgSHlnaWVuZSBhbmQgVHJvcGljYWwgTWVkaWNpbmUsIExvbmRvbiBXQzFFIDdIVCwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkEgY29tcGxldGUgdmlldyBvZiB0aGUgZ2VuZXRpYyBkaXZlcnNpdHkgb2YgdGhlIEVzY2hlcmljaGlhIGNvbGkgTy1hbnRpZ2VuIGJpb3N5bnRoZXNpcyBnZW5lIGNsdXN0ZXIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEwNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdTA0MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE1LzAyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QmFjdGVyaWFsIGV4dHJhY2VsbHVsYXIgcG9seXNhY2NoYXJpZGVzOyBBZHYuIEV4cC4gTWVkLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT03MTU7ICBjaXRhdGlvbl9wYWdlcz0yMTMtMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN0cnVjdHVyYWwgZGl2ZXJzaXR5IGluIFNhbG1vbmVsbGEgTyBhbnRpZ2VucyBhbmQgaXRzIGdlbmV0aWMgYmFzaXM7IEZFTVMgTWljcm9iaW9sLiBSZXYuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9NTYtODk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBzdHJ1Y3R1cmVzIG9mIEVzY2hlcmljaGlhIGNvbGkgTy1wb2x5c2FjY2hhcmlkZSBhbnRpZ2VuczsgRkVNUyBNaWNyb2Jpb2wuIFJldi47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0zODItNDAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGFuZCBmdW5jdGlvbmFsIGRpdmVyc2l0eSBvZiBQc2V1ZG9tb25hcyBhZXJ1Z2lub3NhIGxpcG9wb2x5c2FjY2hhcmlkZTsgRnJvbnQgTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOyAgY2l0YXRpb25fcGFnZXM9MTE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGl2ZXJzaXR5IG9mIGxpcG9wb2x5c2FjY2hhcmlkZSBzdHJ1Y3R1cmVzIGluIENhbXB5bG9iYWN0ZXIgamVqdW5pOyBKLiBJbmZlY3QuIERpcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MTc2OyAgY2l0YXRpb25faXNzdWU9U3VwcGwuIDI7ICBjaXRhdGlvbl9wYWdlcz1TMTM1LTEzODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW1lcmdpbmcgZm9vZGJvcm5lIHBhdGhvZ2VuczogRXNjaGVyaWNoaWEgY29saSBPMTU3Okg3IGFzIGEgbW9kZWwgb2YgZW50cnkgb2YgYSBuZXcgcGF0aG9nZW4gaW50byB0aGUgZm9vZCBzdXBwbHkgb2YgdGhlIGRldmVsb3BlZCB3b3JsZDsgRXBpZGVtaW9sIFJldi47ICBjaXRhdGlvbl95ZWFyPTE5OTY7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0yOS01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2hpZ2EtdG94aW4tcHJvZHVjaW5nIEVzY2hlcmljaGlhIGNvbGkgYW5kIGhhZW1vbHl0aWMgdXJhZW1pYyBzeW5kcm9tZTsgTGFuY2V0OyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM2NTsgIGNpdGF0aW9uX3BhZ2VzPTEwNzMtODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBlbWVyZ2luZyBjbGluaWNhbCBpbXBvcnRhbmNlIG9mIG5vbi1PMTU3IFNoaWdhIHRveGluLXByb2R1Y2luZyBFc2NoZXJpY2hpYSBjb2xpOyBDbGluLiBJbmZlY3QuIERpcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz0xNTg3LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZXJtYW4gb3V0YnJlYWsgb2YgRXNjaGVyaWNoaWEgY29saSBPMTA0Okg0IGFzc29jaWF0ZWQgd2l0aCBzcHJvdXRzOyBOLiBFbmdsLiBKLiBNZWQuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTM2NTsgIGNpdGF0aW9uX3BhZ2VzPTE3NjMtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBlcGlkZW1pb2xvZ3kgb2YgRXNjaGVyaWNoaWEgY29saSBwcm9kdWNpbmcgQ1RYLU0gYmV0YS1sYWN0YW1hc2VzOiB0aGUgd29ybGR3aWRlIGVtZXJnZW5jZSBvZiBjbG9uZSBTVDEzMSBPMjU6SDQ7IEludC4gSi4gQW50aW1pY3JvYi4gQWdlbnRzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM1OyAgY2l0YXRpb25fcGFnZXM9MzE2LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9zeW50aGVzaXMgb2YgTy1hbnRpZ2VuczogZ2VuZXMgYW5kIHBhdGh3YXlzIGludm9sdmVkIGluIG51Y2xlb3RpZGUgc3VnYXIgcHJlY3Vyc29yIHN5bnRoZXNpcyBhbmQgTy1hbnRpZ2VuIGFzc2VtYmx5OyBDYXJib2h5ZHIuIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzM4OyAgY2l0YXRpb25fcGFnZXM9MjUwMy0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW9uIG9mIE8gYW50aWdlbnMgaW4gRXNjaGVyaWNoaWEgY29saTsgQW5pbS4gSGVhbHRoIFJlcy4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTE2OS04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwcmVjaXNlIHJlY29uc3RydWN0aW9uIG9mIHRoZSBlbWVyZ2VuY2UgYW5kIGNvbnN0cmFpbmVkIHJhZGlhdGlvbnMgb2YgRXNjaGVyaWNoaWEgY29saSBPMTU3IHBvcnRyYXllZCBieSBiYWNrYm9uZSBjb25jYXRlbm9taWMgYW5hbHlzaXM7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA2OyAgY2l0YXRpb25fcGFnZXM9ODcxMy04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1XaWRlIGRpc3RyaWJ1dGlvbiBvZiBPMTU3LWFudGlnZW4gYmlvc3ludGhlc2lzIGdlbmUgY2x1c3RlcnMgaW4gRXNjaGVyaWNoaWEgY29saTsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUyMzI1MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjaGFyYWN0ZXJpemF0aW9uIHJldmVhbHMgdGhyZWUgZGlzdGluY3QgY2xvbmFsIGdyb3VwcyBhbW9uZyBjbGluaWNhbCBTaGlnYSB0b3hpbi1wcm9kdWNpbmcgRXNjaGVyaWNoaWEgY29saSBzdHJhaW5zIG9mIHNlcm9ncm91cCBPMTAzOyBKLiBDbGluLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9Mjg5NC05MDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIHRoZSB0d28gZ2x5Y29zeWx0cmFuc2ZlcmFzZSBnZW5lcyByZXNwb25zaWJsZSBmb3IgdGhlIGRpZmZlcmVuY2UgYmV0d2VlbiBFc2NoZXJpY2hpYSBjb2xpIE8xMDcgYW5kIE8xMTcgTy1hbnRpZ2VuczsgR2x5Y29iaW9sb2d5OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MjgxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZ3JvdXAgb2YgRXNjaGVyaWNoaWEgY29saSBhbmQgU2FsbW9uZWxsYSBlbnRlcmljYSBPIGFudGlnZW5zIHNoYXJpbmcgYSBjb21tb24gYmFja2JvbmUgc3RydWN0dXJlOyBNaWNyb2Jpb2xvZ3k7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTUzOyAgY2l0YXRpb25fcGFnZXM9MjE1OS02NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm92ZWwgbWljcm9hcnJheSBkZXNpZ24gZm9yIG1vbGVjdWxhciBzZXJvdHlwaW5nIG9mIFNoaWdhIHRveGluLXByb2R1Y2luZyBFc2NoZXJpY2hpYSBjb2xpIGlzb2xhdGVkIGZyb20gZnJlc2ggcHJvZHVjZTsgQXBwbC4gRW52aXJvbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT04MDsgIGNpdGF0aW9uX3BhZ2VzPTQ2NzctODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgcmFwaWQgcHJvY2VkdXJlIGZvciB0aGUgZGV0ZWN0aW9uIGFuZCBpc29sYXRpb24gb2YgZW50ZXJvaGFlbW9ycmhhZ2ljIEVzY2hlcmljaGlhIGNvbGkgKEVIRUMpIHNlcm9ncm91cCBPMjYsIE8xMDMsIE8xMTEsIE8xMTgsIE8xMjEsIE8xNDUgYW5kIE8xNTcgc3RyYWlucyBhbmQgdGhlIGFnZ3JlZ2F0aXZlIEVIRUMgTzEwNDpINCBzdHJhaW4gZnJvbSByZWFkeS10by1lYXQgdmVnZXRhYmxlczsgSW50LiBKLiBGb29kIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTUyOyAgY2l0YXRpb25fcGFnZXM9MTktMzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGEgc2Vyb2dyb3VwLXNwZWNpZmljIG11bHRpcGxleCBQQ1IgYXNzYXkgdG8gZGV0ZWN0IGEgc2V0IG9mIEVzY2hlcmljaGlhIGNvbGkgc2Vyb2dyb3VwcyBiYXNlZCBvbiB0aGUgaWRlbnRpZmljYXRpb24gb2YgdGhlaXIgTy1hbnRpZ2VuIGdlbmUgY2x1c3RlcnM7IE1vbC4gQ2VsbCBQcm9iZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0yODYtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNMVVNUQUwgVzogaW1wcm92aW5nIHRoZSBzZW5zaXRpdml0eSBvZiBwcm9ncmVzc2l2ZSBtdWx0aXBsZSBzZXF1ZW5jZSBhbGlnbm1lbnQgdGhyb3VnaCBzZXF1ZW5jZSB3ZWlnaHRpbmcsIHBvc2l0aW9uLXNwZWNpZmljIGdhcCBwZW5hbHRpZXMgYW5kIHdlaWdodCBtYXRyaXggY2hvaWNlOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz00NjczLTgwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NRUdBNDogTW9sZWN1bGFyIEV2b2x1dGlvbmFyeSBHZW5ldGljcyBBbmFseXNpcyAoTUVHQSkgc29mdHdhcmUgdmVyc2lvbiA0LjA7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNTk2LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlbGV0aW9uIG9mIHRoZSBFc2NoZXJpY2hpYSBjb2xpIE8xNDpLNyBPIGFudGlnZW4gZ2VuZSBjbHVzdGVyOyBDYW4uIEouIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9wYWdlcz0yOTktMzAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZSBpbmRlcGVuZGVudCBvcmlnaW5zIG9mIFNoaWdlbGxhIGNsb25lcyBvZiBFc2NoZXJpY2hpYSBjb2xpIGFuZCBjb252ZXJnZW50IGV2b2x1dGlvbiBvZiBtYW55IG9mIHRoZWlyIGNoYXJhY3RlcmlzdGljczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT05NzsgIGNpdGF0aW9uX3BhZ2VzPTEwNTY3LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TdHJ1Y3R1cmUgYW5kIGdlbmV0aWNzIG9mIFNoaWdlbGxhIE8gYW50aWdlbnM7IEZFTVMgTWljcm9iaW9sLiBSZXYuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9NjI3LTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbWVyZ2VuY2Ugb2YgYSBub3ZlbCBTaGlnYSB0b3hpbi1wcm9kdWNpbmcgRXNjaGVyaWNoaWEgY29saSBPIHNlcm9ncm91cCBjcm9zcy1yZWFjdGluZyB3aXRoIFNoaWdlbGxhIGJveWRpaSB0eXBlIDEwOyBKLiBDbGluLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9MzY3OC04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW50ZXJvYWdncmVnYXRpdmUgRXNjaGVyaWNoaWEgY29saSBPNzg6SDEwLCB0aGUgY2F1c2Ugb2YgYW4gb3V0YnJlYWsgb2YgdXJpbmFyeSB0cmFjdCBpbmZlY3Rpb247IEouIENsaW4uIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9wYWdlcz0zNzAzLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aS1sb2N1cyBzZXF1ZW5jZSB0eXBpbmcgb2YgZW50ZXJvYWdncmVnYXRpdmUgRXNjaGVyaWNoaWEgY29saSBpc29sYXRlcyBmcm9tIE5pZ2VyaWFuIGNoaWxkcmVuIHVuY292ZXJzIG11bHRpcGxlIGxpbmVhZ2VzOyBQTG9TIE9ORTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTE0MDkzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5bG9nZW5ldGljIGNvbXBhcmlzb25zIHJldmVhbCBtdWx0aXBsZSBhY3F1aXNpdGlvbnMgb2YgdGhlIHRveGluIGdlbmVzIGJ5IGVudGVyb3RveGlnZW5pYyBFc2NoZXJpY2hpYSBjb2xpIHN0cmFpbnMgb2YgZGlmZmVyZW50IGV2b2x1dGlvbmFyeSBsaW5lYWdlczsgSi4gQ2xpbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT00NDsgIGNpdGF0aW9uX3BhZ2VzPTQ1MjgtMzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc2NvdmVyeSBhbmQgcGh5bG9nZW5ldGljIGFuYWx5c2lzIG9mIG5vdmVsIG1lbWJlcnMgb2YgY2xhc3MgYiBlbnRlcm90b3hpZ2VuaWMgRXNjaGVyaWNoaWEgY29saSBhZGhlc2l2ZSBmaW1icmlhZTsgSi4gQ2xpbi4gTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT00OTsgIGNpdGF0aW9uX3BhZ2VzPTE0MDMtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgZmFybSwgZm9vZCwgYW5kIGNsaW5pY2FsIFNoaWdhIHRveGluLXByb2R1Y2luZyBFc2NoZXJpY2hpYSBjb2xpIChTVEVDKSBPMTEzOyBGb29kYm9ybmUgUGF0aG9nLiBEaXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0xMDg4LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaHlsb2dlbmV0aWMgYW5kIG1vbGVjdWxhciBhbmFseXNpcyBvZiBmb29kLWJvcm5lIHNoaWdhIHRveGluLXByb2R1Y2luZyBFc2NoZXJpY2hpYSBjb2xpOyBBcHBsLiBFbnZpcm9uLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTc5OyAgY2l0YXRpb25fcGFnZXM9MjczMS00MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2Vyb3R5cGUtY29udmVydGluZyBiYWN0ZXJpb3BoYWdlcyBhbmQgTy1hbnRpZ2VuIG1vZGlmaWNhdGlvbiBpbiBTaGlnZWxsYSBmbGV4bmVyaTsgVHJlbmRzIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE3LTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5vdmVsIHBsYXNtaWQtZW5jb2RlZCBzZXJvdHlwZSBjb252ZXJzaW9uIG1lY2hhbmlzbSB0aHJvdWdoIGFkZGl0aW9uIG9mIHBob3NwaG9ldGhhbm9sYW1pbmUgdG8gdGhlIE8tYW50aWdlbiBvZiBTaGlnZWxsYSBmbGV4bmVyaTsgUExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPWU0NjA5NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMi8xLzEwMS8xMjExOTc0L2RzdTA0My5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgVGhlIE8gYW50aWdlbiBjb25zdGl0dXRlcyB0aGUgb3V0ZXJtb3N0IHBhcnQgb2YgdGhlIGxpcG9wb2x5c2FjY2hhcmlkZSBsYXllciBpbiBHcmFtLW5lZ2F0aXZlIGJhY3RlcmlhLiBUaGUgY2hlbWljYWwgY29tcG9zaXRpb24gYW5kIHN0cnVjdHVyZSBvZiB0aGUiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ0MjE2MSZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGTkTcy0UWJdIZpUOQchUrN3Y0MDiw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f101%2f442161" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f101%2f442161" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17206" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/22/1"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/22/1/1/m_cover.gif?Expires=2147483647&amp;Signature=Vhf6NToQrkVldazSweShdrlrfCnUbBpTokrfu8JXGVRMaoyIjioben46GoxM3BSEoRr~te73o-dfY7QEM91wVumB~VX8hAxN9ywAZ1zRAbXXmtU23LiP8gcw~y1-D8ACKe3CZXmetX85Oj4q~Op8xej8CbOs0MGBcAElTMf~GIynbEl8oPRbKlaZUEBETFvDqCw1PlymDq~5degwzATpcv5kHJhRgAnT8PjHfQEqeZxMFeCt-B-phqwpjq5AMA0hAnSz5a5U5SMfCC3Naxv-~~yWr1GxeMNVTp3iBMlBqhdGNwFRsnBBquWLbTiF47z2gcE0wubs8CNqU4CX2qcpAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/22/1">
                        <div class="volume trailing-comma">Volume 22</div>
                        <div class="issue">Issue 1</div>

            </a>

        </div>
        <div class="ii-pub-date">
February 2015        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6612229">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6612229">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6612234">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6612234">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6612243">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6612243">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6612257">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6612257">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6612281">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6612281">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6612287">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6612287">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6612289">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6612289">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6612291">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6612291">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6612293">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6612293">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/22/1/91/441732">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/22/1/109/442245">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">A complete view of the genetic diversity of the <em>Escherichia coli</em> O-antigen biosynthesis gene cluster
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi Iguchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal and Grassland Sciences, Faculty of Agriculture</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">Miyazaki 889-2192</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel/Fax. +81 985-58-7507. E-mail: <a href="mailto:iguchi@med.miyazaki-u.ac.jp" target="_blank">iguchi@med.miyazaki-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsushi+Iguchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Iguchi A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Iguchi A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Atsushi Iguchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sunao Iyoda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bacteriology I</div>, <div class="insititution">National Institute of Infectious Diseases</div>, <div class="addr-line">Tokyo 162-8640</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sunao+Iyoda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Iyoda S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Iyoda S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sunao Iyoda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Taisei Kikuchi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Division of Parasitology, Department of Infectious Diseases, Faculty of Medicine</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">Miyazaki 889-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Taisei+Kikuchi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kikuchi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kikuchi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Taisei Kikuchi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshitoshi Ogura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">Miyazaki 889-1692</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">5</div><div class="addr-line">Division of Bioenvironmental Science, Frontier Science Research Center</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">Miyazaki 889-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshitoshi+Ogura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogura Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshitoshi Ogura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Keisuke Katsura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Division of Bioenvironmental Science, Frontier Science Research Center</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">Miyazaki 889-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Keisuke+Katsura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Katsura K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Katsura K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Keisuke Katsura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Makoto Ohnishi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Bacteriology I</div>, <div class="insititution">National Institute of Infectious Diseases</div>, <div class="addr-line">Tokyo 162-8640</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Makoto+Ohnishi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohnishi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohnishi M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Makoto Ohnishi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuya Hayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Division of Microbiology, Department of Infectious Diseases, Faculty of Medicine</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">Miyazaki 889-1692</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">5</div><div class="addr-line">Division of Bioenvironmental Science, Frontier Science Research Center</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">Miyazaki 889-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuya+Hayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hayashi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hayashi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tetsuya Hayashi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Nicholas R. Thomson</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="addr-line">Pathogen Genomics</div>, <div class="insititution">The Wellcome Trust Sanger Institute</div>, <div class="addr-line">Cambridge CB10 1SA</div>, <div class="country">UK</div></div><div class="aff"><div class="label -label">7</div><div class="addr-line">Department of Infectious and Tropical Diseases</div>, <div class="insititution">London School of Hygiene and Tropical Medicine</div>, <div class="addr-line">London WC1E 7HT</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Nicholas+R.+Thomson">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Thomson N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Thomson N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Nicholas R. Thomson</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 101107, <a href='https://doi.org/10.1093/dnares/dsu043'>https://doi.org/10.1093/dnares/dsu043</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">26 November 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">16 September 2014</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">03 November 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="442161" href="/dnaresearch/article-pdf/22/1/101/1211974/dsu043.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Atsushi Iguchi, Sunao Iyoda, Taisei Kikuchi, Yoshitoshi Ogura, Keisuke Katsura, Makoto Ohnishi, Tetsuya Hayashi, Nicholas R. Thomson;  A complete view of the genetic diversity of the <em>Escherichia coli</em> O-antigen biosynthesis gene cluster, <em>DNA Research</em>, Volume 22, Issue 1, 1 February 2015, Pages 101107, <a href='https://doi.org/10.1093/dnares/dsu043'>https://doi.org/10.1093/dnares/dsu043</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/442161?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/442161?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/442161?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/442161?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/442161?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/442161?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/442161?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=A%20complete%20view%20of%20the%20genetic%20diversity%20of%20the%20%26lt%3Bem%26gt%3BEscherichia%20coli%26lt%3B%2Fem%26gt%3B%20O-antigen%20biosynthesis%20gene%20cluster&amp;publicationDate=2014-11-26&amp;volumeNum=22&amp;issueNum=1&amp;author=Iguchi%2C%20Atsushi%3B%20Iyoda%2C%20Sunao&amp;startPage=101&amp;endPage=107&amp;contentId=10.1093%2Fdnares%2Fdsu043&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f101%2f442161" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6612229 id="6612229" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The O antigen constitutes the outermost part of the lipopolysaccharide layer in Gram-negative bacteria. The chemical composition and structure of the O antigen show high levels of variation even within a single species revealing itself as serological diversity. Here, we present a complete sequence set for the O-antigen biosynthesis gene clusters (O-AGCs) from all 184 recognized <em>Escherichia coli</em> O serogroups. By comparing these sequences, we identified 161 well-defined O-AGCs. Based on the <em>wzx</em>/<em>wzy</em> or <em>wzm</em>/<em>wzt</em> gene sequences, in addition to 145 singletons, 37 serogroups were placed into 16 groups. Furthermore, phylogenetic analysis of all the <em>E. coli</em> O-serogroup reference strains revealed that the nearly one-quarter of the 184 serogroups were found in the ST10 lineage, which may have a unique genetic background allowing a more successful exchange of O-AGCs. Our data provide a complete view of the genetic diversity of O-AGCs in <em>E. coli</em> showing a stronger association between host phylogenetic lineage and O-serogroup diversification than previously recognized. These data will be a valuable basis for developing a systematic molecular O-typing scheme that will allow traditional typing approaches to be linked to genomic exploration of <em>E. coli</em> diversity.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;E. coli&quot;"><em>E. coli</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;O-antigen biosynthesis gene cluster&quot;">O-antigen biosynthesis gene cluster</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;horizontal gene transfer&quot;">horizontal gene transfer</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;O serogroup&quot;">O serogroup</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genomic diversity&quot;">genomic diversity</a></div>                            <h2 scrollto-destination=6612234 id="6612234" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Cell-surface polysaccharides play an essential role in the ability of bacteria to survive and persist in the environment and in host organisms.<sup><a reveal-id="DSU043C1" data-open="DSU043C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The O-antigen polysaccharide constitutes the outermost part of the lipopolysaccharide (LPS) present in the outer membrane of Gram-negative bacteria. The chemical composition and structure of the O-antigen exhibit high levels of variation even within a single species.<sup><a reveal-id="DSU043C2  DSU043C5" data-open="DSU043C2  DSU043C5" class="link link-ref link-reveal xref-bibr">25</a></sup> This observation is corroborated by the huge serological variation of somatic O antigens. Currently, the O serogrouping, sometimes combined with H (flagellar) antigens and K (capsular polysaccharide) antigens, is a standard method for subtyping of <em>Escherichia coli</em> strains in taxonomical and epidemiological studies. In particular, identification of strains of the same O serogroup is a prerequisite to start any actions for outbreak investigations and surveillance.</p><p>Thus far, the World Health Organization Collaborating Centre for Reference and Research on <em>Escherichia</em> and <em>Klebsiella</em> based at the Statens Serum Institut (SSI) in Denmark (<a class="link link-uri" href="http://www.ssi.dk/English.aspx">http://www.ssi.dk/English.aspx</a>) has recognized 184 <em>E. coli</em> O serogroups. It is generally believed that the O serogrouping of <em>E. coli</em> strains provides valuable information for identifying pathogenic clonal groups, especially for public health surveillance. For example, O157 is a leading O serogroup associated with enterohemorrhagic <em>E. coli</em> (EHEC) and is a significant food-borne pathogen worldwide.<sup><a reveal-id="DSU043C6" data-open="DSU043C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSU043C7" data-open="DSU043C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Other important EHEC O serogroups include O26, O103, and O111.<sup><a reveal-id="DSU043C8" data-open="DSU043C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The Shiga toxin-producing <em>E. coli</em> O104:H4 was found responsible for a large human food-borne disease outbreak in Europe, 2011.<sup><a reveal-id="DSU043C9" data-open="DSU043C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Another notable example is strains of serogroup O25; extended-spectrum beta lactamase (ESBL)-producing, multidrug-resistant <em>E. coli</em> O25:H4 has emerged worldwide to cause a wide variety of community and nosocomial infections.<sup><a reveal-id="DSU043C10" data-open="DSU043C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p>In <em>E. coli</em>, the genes required for O-antigen biosynthesis are clustered at a chromosomal locus flanked by the colanic acid biosynthesis gene cluster (<em>wca</em> genes) and the histidine biosynthesis (<em>his</em>) operon. Generally, the O-antigen biosynthesis genes fall into three classes: (i) the nucleotide sugar biosynthesis genes, (ii) the sugar transferase genes, and (iii) those for O-unit translocation and chain synthesis (<em>wzx</em>/<em>wzy</em> in the Wzx/Wzy-dependent pathway and <em>wzm</em>/<em>wzt</em> in the Wzm/Wzt-dependent ABC transporter pathway).<sup><a reveal-id="DSU043C11" data-open="DSU043C11" class="link link-ref link-reveal xref-bibr">11</a></sup> To date, &gt;90 types of O-antigen biosynthesis gene cluster (O-AGC) sequences have been determined, with the majority derived from major human and animal pathogens.<sup><a reveal-id="DSU043C12" data-open="DSU043C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Sequence comparisons of these O-AGCs indicate a great variety of genetic structures. Several studies have provided evidence to show that horizontal transfer and replacement of a part or all of the O-AGC have caused shifts in O serogroups.<sup><a reveal-id="DSU043C13  DSU043C15" data-open="DSU043C13  DSU043C15" class="link link-ref link-reveal xref-bibr">1315</a></sup> Alternatively, point mutations in the glycosyltransferase genes in the O-AGC or acquisition of alternative O-antigen modification genes, which are located outside of the O-AGC, have also been shown to result in structural alterations of O antigen and concomitant change in the serotype of the isolate.<sup><a reveal-id="DSU043C16" data-open="DSU043C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSU043C17" data-open="DSU043C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p><p>Genes or DNA sequences specific for each O serogroup can be used as targets for the identification of O serogroups via molecular approaches, such as PCR-based and hybridization-based methods. Such systems have already been developed by several researchers to target specific O-antigen types.<sup><a reveal-id="DSU043C12" data-open="DSU043C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSU043C18  DSU043C20" data-open="DSU043C18  DSU043C20" class="link link-ref link-reveal xref-bibr">1820</a></sup> In particular, molecular assays targeting major O serogroups are routinely used in EHEC surveillance for clinical or food sample screening. Considering the range of diseases caused by <em>E. coli</em> strains belonging to many different serogroups, a more comprehensive and detailed O-AGC information for the complete set of <em>E. coli</em> O serogroups is of significant clinical importance for generating a rational molecular typing scheme. This molecular typing scheme, which could be performed <em>in silico</em> directly on sequence data, also offers a mechanism with which to link the ever-expanding genomic data to our extensive epidemiological and biological knowledge of this pathogen, based on O-antigen typing. Moreover, these data will also provide a much better understanding of the complex mechanisms by which a huge diversity in O serogroups have arisen. Here, we present a complete sequence set for the O-AGCs from all 184 <em>E. coli</em> O serogroups, which include recently added serogroups (O182O187), providing a complete picture of the O-AGC diversity in <em>E. coli</em>.</p>                            <h2 scrollto-destination=6612243 id="6612243" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6612245 id="6612245" class="section-title" data-legacy-id=s2a>2.1. Bacterial strains, culture condition, and DNA preparation</h3>
<p>Reference strains of all 184 recognized <em>E. coli</em> O serogroups were obtained from SSI (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Cells were grown to the stationary phase at 37C in LuriaBertani medium. Genomic DNA was purified using the Wizard Genomic DNA purification kit (Promega) according to the manufacturer's instructions.</p>                            <h3 scrollto-destination=6612250 id="6612250" class="section-title" data-legacy-id=s2b>2.2. O-AGC sequences and comparative analyses</h3>
<p>One hundred and eight <em>E. coli</em> O-AGC sequences were determined by Sanger-based capillary sequencing and/or Illumina MiSeq sequencing from PCR products covering O-AGCs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The O-AGC regions of the reference strains were amplified by PCR using 10 ng of genomic DNA as template with the Tks Gflex DNA polymerase (Takara Bio Inc.) by 25 amplification cycles for 10 s at 98C and for 16 m at 69C, and with a combination of three forward primers (TATGCCAGCGGCACCAAACG, ATACCGGCGATGAAAGCC, and GCGGGTGGGATTAAGTCTCT) designed on the <em>hisFI</em> genes and two reverse primers (GTGATGCAGGAATCCTCTGT and CCACGCTAATTACGCCATCTT) designed on the <em>wcaM</em> genes, or strain-specific primers designed based on the draft genome sequences determined using the MiSeq system from reference strains. Identification and functional annotation of the CDSs were performed based on the results of homology searches against the public, non-redundant protein database using BLASTP. The sequences reported in this article have been deposited in the GenBank database (accession no. AB811596-AB811624, AB812020-AB812085, and AB972413-AB972425). The other 76 <em>E. coli</em> O-AGC sequences were obtained from public databases. For a list of accession numbers, see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>.</p>                            <h3 scrollto-destination=6612253 id="6612253" class="section-title" data-legacy-id=s2c>2.3. Phylogenetic analysis</h3>
<p>Multilocus sequence typing (MLST) was carried out according to the protocol described on the <em>E. coli</em> MLST website (<a class="link link-uri" href="http://mlst.warwick.ac.uk/mlst/dbs/Ecoli">http://mlst.warwick.ac.uk/mlst/dbs/Ecoli</a>), and the phylogenetic relationships of reference strains were analysed based on the concatenated sequences (3,423 bp) of seven housekeeping genes (<em>adk</em>, <em>fumC</em>, <em>gyrB</em>, <em>icd</em>, <em>mdh</em>, <em>purA</em>, and <em>recA</em>) used for MLST. Multiple alignments of DNA and amino acid sequences were constructed by using the CLUSTAL W program.<sup><a reveal-id="DSU043C21" data-open="DSU043C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Phylogenetic trees were constructed by using the neighbour-joining algorithm using the MEGA4 softrware.<sup><a reveal-id="DSU043C22" data-open="DSU043C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p>                            <h2 scrollto-destination=6612257 id="6612257" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6612259 id="6612259" class="section-title" data-legacy-id=s3a>3.1. Genetic structures of the O-AGCs from all <em>E. coli</em> O serogroups</h3>
<p>Of the 184 known O serogroups, 76 complete O-AGC sequences were obtained from public databases. The sequence of the other 108 O-AGC was determined in this study from <em>E. coli</em> O-serogroup reference strains (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Our analysis of these sequences confirmed several previously observed characteristics of O-AGCs in <em>E. coli</em> (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). In brief, O-AGCs are located between the <em>wca</em> and <em>his</em> operons. This region contains three housekeeping genes: <em>galF</em> (encoding UTP-glucose-1-phosphate uridylyltransferase), <em>gnd</em> (6-phosphogluconate dehydrogenase), and <em>ugd</em> (UDP-glucose 6-dehydrogenase), and most genes for O-antigen biosynthesis in each cluster are directly flanked by <em>galF</em> and <em>gnd</em>/<em>ugd</em>, while <em>gne</em> (UDP-GalNAc-4-epimerase) and <em>wzz</em> (O-antigen chain length determination protein) located immediately outside of the region between <em>galF</em> and <em>gnd</em>/<em>ugd</em> (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>)<em>.</em> The exceptions for this are the O-AGCs for O serogroups O14 and O57, which contain no O-antigen genes at the typical locus. However, it is known that the <em>E. coli</em> O14 reference strain Su4411-41 shows an O rough phenotype and lacks the O-AGC.<sup><a reveal-id="DSU043C23" data-open="DSU043C23" class="link link-ref link-reveal xref-bibr">23</a></sup> For O57, a further analysis is also required to investigate the presence of O-antigen structure in the LPS of the reference strain. Our data revealed that the O-AGCs located between <em>galF</em> and <em>gnd</em> ranged in size from 4.5 kbp (O155, including four genes) to 19.5 kbp (O108, including 18 genes).</p>                            <h4 scrollto-destination=6612263 id="6612263" class="section-title" data-legacy-id=s3a1>3.1.1. Nucleotide sugar biosynthesis genes</h4>
<p>Genes required for the deoxythymidine diphosphate (dTDP)-sugar biosynthesis pathway (<em>rmlBDAC)</em> to synthesize dTDP-<span class="small-caps">l</span>-rhamnose (dTDP-<span class="small-caps">l</span>-Rha), the precursor of <span class="small-caps">l</span>-Rha, were widely distributed in the O-AGCs (conserved in 56 O-serogroup O-AGCs; see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The <em>vioAB</em> operon, for the biosynthesis of dTDP-<em>N</em>-acetylviosamine (dTDP-VioNAc), the precursor of VioNAc, was present in three O-serogroup O-AGCs; the <em>fnlABC</em> operon for the synthesis of uridine diphosphate (UDP)-<em>N</em>-acetyl-<span class="small-caps">l</span>-fucosamine (UDP-<span class="small-caps">l</span>-FucNAc), the precursor of <span class="small-caps">l</span>-FucNAc, was in 11 O-serogroup O-AGCs; the <em>fnlA-qnlBC</em> genes for the synthesis of UDP-<em>N</em>-acetyl-<span class="small-caps">l</span>-quinovosamine (UDP-<span class="small-caps">l</span>-QuiNAc), the precursor of <span class="small-caps">l</span>-QuiNAc, were in four O-serogroup O-AGCs; the <em>nnaDBCA</em> genes for synthesis of cytidine monophosphate (CMP)-<em>N</em>-acetylneuraminate (CMPNeuNAc), the precursor of <em>N</em>-acetylneuraminic acid (Neu5Ac or sialic acid), were found in six O-serogroup O-AGCs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). In addition, a gene set comprising seven genes putatively involved in the synthesis of di-<em>N</em>-acetyl-8-epilegionaminic acid (8eLeg5Ac7Ac) were found in three O-serogroup O-AGCs. For at least 49 O serogroups, gene sets for nucleotide sugar biosynthesis were not found in their O-AGCs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>), suggesting that, in these serogroups, nucleotide sugars required for O-antigen biosynthesis were synthesized by pathways encoded by the genes located outside of the O-AGCs.</p>                            <h4 scrollto-destination=6612265 id="6612265" class="section-title" data-legacy-id=s3a2>3.1.2. Glycosyltransferase</h4>
<p>Each O-AGC contained two to six genes encoding putative glycosyltransferases for synthesizing O-antigen subunits and a total of 611 glycosyltransferase genes identified in all O-AGCs. Pfam analysis revealed that at least 25 types of glycosyltransferase-related domains were found in the 611 glycosyltransferase genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Glycosyl transferases group 1 (PF00534) and Glycosyl transferase family 2 (PF00535) were the most widely distributed domains, which were found in 216 and 253 genes, respectively. Except for the five genes belonging to Glycosyltransferase family 52 (PF07922), which were found in five of the six <em>nnaDBCA</em>-containing O-AGCs (O24, O56, O104, O131, and O171), there were no relationships between the type of glycosyltransferase-related domain and the gene set for sugar synthesis in each O-AGC.</p>                            <h4 scrollto-destination=6612267 id="6612267" class="section-title" data-legacy-id=s3a3>3.1.3. O-antigen subunit translocation and chain synthesis</h4>
<p>All O-AGCs carried either <em>wzx</em>/<em>wzy</em> or <em>wzm</em>/<em>wzt</em> gene pairs. Of the 182 O-AGCs (the above-mentioned O14 and O57 were excluded from the 184 clusters analysed in this study), 171 carried the <em>wzx</em>/<em>wzy</em> genes, and the other 11 carried the <em>wzm</em>/<em>wzt</em> genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1 and Table S1</a>). Detailed sequence comparisons of the <em>wzx</em>/<em>wzy</em> and <em>wzm</em>/<em>wzt</em> genes are described below.</p>                            <h3 scrollto-destination=6612269 id="6612269" class="section-title" data-legacy-id=s3b>3.2. Grouping the O-AGCs by sequence</h3>
<p>On the basis of sequences and genetic structures of the entire O-AGC regions, in addition to 145 unique O-AGCs from different <em>E. coli</em> O serogroups, the O-AGCs from 37 O serogroups could be placed into 16 groups (named Gp1Gp16) with the members of each group having identical or very similar O-AGC genes (mostly sharing 95% DNA sequence identity) (Fig.<a reveal-id="DSU043F1" data-open="DSU043F1" class="link link-reveal link-table xref-fig">1</a>). This included nine groups with members of different serogroups but which carried identical O-AGC gene sets (Gp1Gp9) and one group, Gp10, where two strains (O13 and O129) of the three-member group carried an identical O-AGC gene set (sharing 98.399.9% DNA sequence identity) (Fig.<a reveal-id="DSU043F1" data-open="DSU043F1" class="link link-reveal link-table xref-fig">1</a>). The reason(s) why they belong to different O serogroups even though they have identical O-AGCs are discussed in the Discussion section. Indels or exchange of one or more genes was also shown to explain the differences between O135 and other members of Gp10 and members Gp11Gp16, which otherwise carried highly conserved orthologous genes (summarized in Fig.<a reveal-id="DSU043F1" data-open="DSU043F1" class="link link-reveal link-table xref-fig">1</a>). Simple insertions of insertion sequence (IS) elements containing one or two transposase genes were found in three groups without any gene disruption: an IS<em>629</em> insertion in O18ab of Gp12, IS<em>Ec11</em> in O164 of Gp13, and IS<em>1</em> in O62 of Gp14. IS element-associated replacement of the right-end portion of the O-AGC had occurred in three groups, Gp14, Gp15, and Gp16, resulting in the replacement (or deletion) of glycosyltransferase gene(s). Exchange of the <em>wzx</em> gene had also occurred in Gp16. These data suggest that IS elements are important drivers for generating O-antigen biosynthesis gene replacement and therefore diversity. </p>                        <a id="6612271" scrollto-destination="6612271"></a>
<div id="DSU043F1" data-id="DSU043F1" class="fig fig-section"><div class="label fig-label">Figure1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu043/2/m_dsu04301.png?Expires=2147483647&Signature=fj4RLc09Wr5iSt0-rFGtYi0DqdXo2fDTvbtWPMq4UpFKoQ9o6DcLSEvplTUrhyg74cC54~hpp6G3OpcWyQ0c0ZzI15X6T2Ucld9yvZgGItQmlnYxl73b1S2KFbQhugsJxOW5FkoqoXlNYaGuIsz3WdTV8wpgCXuoIk2rjznNfPpBQWRLqgZNwyxht~ITGRwQQceknoRumkU41SyyXy-1pHdRMt5QweILeoJpCwonmALbBP3QP0qN~QHo-VVGYq9WFlbA~u19n1oz5AOVVswJbAV5KfuhnAjhAMSCnKCFPCtcs6rbevZ6sUV34FwzozWlYdCG9rvPvGtRdTKZZ410QA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sixteen Escherichia coli O-AGC groups identified in this study. Group members have different O serogroups in each group, but these share nearly identical or highly similar genetic organizations. Group names (Gp) are indicated at the left side. DNA sequence identities (%) between group members are indicated in each group." path-from-xml="dsu04301.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6612271/dsu04301.tif" path-from-xml="dsu04301.tif" target="_blank">View large</a><a section="6612271" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu043/2/dsu04301.png?Expires=2147483647&Signature=UqNGSGL-ZhC7hCBoqJZALee5ufvE87o18zTsJZ8ccjGHA0o7qAgG7qgeQ6R8G-ec5XKgGTOthxYVriS5TWmYCbtCrsnWSGDCS2ygvpcMRLR~dLf~j5MUe1AhlW7W7zZGdIw5D6waMYt9ffFNAjtG7nShxo8~4zCNLReeqP6a~l2UjKaY-uKTb7y~koDllEatRgFmp62CgrR1~uQiRwrYp93eEoqubcjoZcqPMLpxt24qVM6md-Fbus~cZZ~jLXXDNo-MhLUMe4RHUTliYBSwdlMGLE23wfDthSMkUibelcnillXlXW9~~j6Wp1Z~AxVQV1TLW3of3AVsOkbZ50EMpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6612271&ar=442161&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04301.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sixteen <em>Escherichia coli</em> O-AGC groups identified in this study. Group members have different O serogroups in each group, but these share nearly identical or highly similar genetic organizations. Group names (Gp) are indicated at the left side. DNA sequence identities (%) between group members are indicated in each group.</p></div></div><div content-id="DSU043F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu043/2/m_dsu04301.png?Expires=2147483647&Signature=fj4RLc09Wr5iSt0-rFGtYi0DqdXo2fDTvbtWPMq4UpFKoQ9o6DcLSEvplTUrhyg74cC54~hpp6G3OpcWyQ0c0ZzI15X6T2Ucld9yvZgGItQmlnYxl73b1S2KFbQhugsJxOW5FkoqoXlNYaGuIsz3WdTV8wpgCXuoIk2rjznNfPpBQWRLqgZNwyxht~ITGRwQQceknoRumkU41SyyXy-1pHdRMt5QweILeoJpCwonmALbBP3QP0qN~QHo-VVGYq9WFlbA~u19n1oz5AOVVswJbAV5KfuhnAjhAMSCnKCFPCtcs6rbevZ6sUV34FwzozWlYdCG9rvPvGtRdTKZZ410QA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sixteen Escherichia coli O-AGC groups identified in this study. Group members have different O serogroups in each group, but these share nearly identical or highly similar genetic organizations. Group names (Gp) are indicated at the left side. DNA sequence identities (%) between group members are indicated in each group." path-from-xml="dsu04301.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6612271/dsu04301.tif" path-from-xml="dsu04301.tif" target="_blank">View large</a><a section="6612271" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu043/2/dsu04301.png?Expires=2147483647&Signature=UqNGSGL-ZhC7hCBoqJZALee5ufvE87o18zTsJZ8ccjGHA0o7qAgG7qgeQ6R8G-ec5XKgGTOthxYVriS5TWmYCbtCrsnWSGDCS2ygvpcMRLR~dLf~j5MUe1AhlW7W7zZGdIw5D6waMYt9ffFNAjtG7nShxo8~4zCNLReeqP6a~l2UjKaY-uKTb7y~koDllEatRgFmp62CgrR1~uQiRwrYp93eEoqubcjoZcqPMLpxt24qVM6md-Fbus~cZZ~jLXXDNo-MhLUMe4RHUTliYBSwdlMGLE23wfDthSMkUibelcnillXlXW9~~j6Wp1Z~AxVQV1TLW3of3AVsOkbZ50EMpg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6612271&ar=442161&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04301.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sixteen <em>Escherichia coli</em> O-AGC groups identified in this study. Group members have different O serogroups in each group, but these share nearly identical or highly similar genetic organizations. Group names (Gp) are indicated at the left side. DNA sequence identities (%) between group members are indicated in each group.</p></div></div>                            <h3 scrollto-destination=6612272 id="6612272" class="section-title" data-legacy-id=s3c>3.3. Diversity and specificity of the <em>wzx</em>/<em>wzy</em> or <em>wzm</em>/<em>wzt</em> genes among the <em>E. coli</em> O-AGCs</h3>
<p>As previously proposed,<sup><a reveal-id="DSU043C12" data-open="DSU043C12" class="link link-ref link-reveal xref-bibr">12</a></sup> most <em>wzx</em>/<em>wzy</em> or <em>wzm</em>/<em>wzt</em> orthologues showed high levels of sequence diversity and their sequences were unique to each O-AGC or O-AGC group described above (Fig.<a reveal-id="DSU043F2" data-open="DSU043F2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). DNA sequence identities of the closest pairs were &lt;70%, except for the O96/O170 pair, the <em>wzx</em> genes of which showed 86% DNA sequence identity. Within the 16 O-AGC groups, the orthologous <em>wzx</em>/<em>wzy</em> or <em>wzm</em>/<em>wzt</em> genes also showed high sequence conservation (95% DNA sequence identity, but mostly 97% identity), except for Gp16 that shared only the <em>wzy</em> gene (Fig.<a reveal-id="DSU043F2" data-open="DSU043F2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="6612274" scrollto-destination="6612274"></a>
<div id="DSU043F2" data-id="DSU043F2" class="fig fig-section"><div class="label fig-label">Figure2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu043/2/m_dsu04302.png?Expires=2147483647&Signature=mGAzKKTbGL1InLTDo28COfAa~8OWc6wv-YGFhY1LrFtPmRWUMBScPdbFThKkWV4AkzOTmXTRuhsWMrDVgH-okQXNWGpdR9Ws~B6lIvStDbALZuTOsCHtFv-FFV4ODgnkI5NwMxX7Qpg2Ejo9pub8N2GAO5EBIrsXsDdqTSfy0XiK2yzz3pKLsbcEt1H2M~0Kaan7m4yCKnM7v6RYwrfSrWcY3ttLB215C-nLwWGit8jzvA~k0jjN6WsByxKYbCQXkTEinTeheKN5djVfowTs5AAz5YCAVGmHcchEt5INTm-dKX2A5yW1gvAbq1RQo-DZmP2ZFq7LbNPHH8V1-qMAZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic analysis of homologues of (A) Wzx and Wzy and (B) Wzm and Wzt from Escherichia coli O-serogroup reference strains based on the amino acid sequences. The group names are indicated outside of trees. The pair or groups of homologues with high DNA sequence identity (95%, mostly 97%) are indicated in red. The Wzx homologues of O96 and O170, which are indicated in blue and by an asterisk, showed 86% DNA sequence identity, but in all other proteins showed low-sequence homologies to each other (&lt;70% identity). Note that while the DNA sequence identity between the wzx_O46 and wzx_O134 in Gp6 is 99.7%, the wzx_O46 has a 2-bp deletion at the 3-region, causing a frame shift." path-from-xml="dsu04302.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6612274/dsu04302.tif" path-from-xml="dsu04302.tif" target="_blank">View large</a><a section="6612274" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu043/2/dsu04302.png?Expires=2147483647&Signature=XiqtSiH6s6rz7~2CC0E9oTU2A3vsjdM094pp2Cn~YjSwb47b1IJt3SctT7L9gvFIpreQvzMx3Sx0RxifsWfuaSECTUjyroP~y9bbi39Xs0DLr92T13cjo~oUpaC8MVEdgoG3jrOQ-Zj7h2xZunXDQlpCXutmV7a7soJsGT6QcN7I-WCiXbdXWQPShUx3qTuy7y0zxRuV4fCcYzBQnfaCp1ibyHak3WCO2UmG3eDtsQ4fD-eSw1iX9DCZkWBYcjDoP5C2CfqE~5Rw1wDj3XRH1UlR9x2v0V5kXIifS3Cxxs1n9zjZliuPogEH4RJe~gIStXfdOIahoeMf52HbrATNqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6612274&ar=442161&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04302.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic analysis of homologues of (A) Wzx and Wzy and (B) Wzm and Wzt from <em>Escherichia coli</em> O-serogroup reference strains based on the amino acid sequences. The group names are indicated outside of trees. The pair or groups of homologues with high DNA sequence identity (95%, mostly 97%) are indicated in red. The Wzx homologues of O96 and O170, which are indicated in blue and by an asterisk, showed 86% DNA sequence identity, but in all other proteins showed low-sequence homologies to each other (&lt;70% identity). Note that while the DNA sequence identity between the <em>wzx</em>_O46 and <em>wzx</em>_O134 in Gp6 is 99.7%, the <em>wzx</em>_O46 has a 2-bp deletion at the 3-region, causing a frame shift.</p></div></div><div content-id="DSU043F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu043/2/m_dsu04302.png?Expires=2147483647&Signature=mGAzKKTbGL1InLTDo28COfAa~8OWc6wv-YGFhY1LrFtPmRWUMBScPdbFThKkWV4AkzOTmXTRuhsWMrDVgH-okQXNWGpdR9Ws~B6lIvStDbALZuTOsCHtFv-FFV4ODgnkI5NwMxX7Qpg2Ejo9pub8N2GAO5EBIrsXsDdqTSfy0XiK2yzz3pKLsbcEt1H2M~0Kaan7m4yCKnM7v6RYwrfSrWcY3ttLB215C-nLwWGit8jzvA~k0jjN6WsByxKYbCQXkTEinTeheKN5djVfowTs5AAz5YCAVGmHcchEt5INTm-dKX2A5yW1gvAbq1RQo-DZmP2ZFq7LbNPHH8V1-qMAZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic analysis of homologues of (A) Wzx and Wzy and (B) Wzm and Wzt from Escherichia coli O-serogroup reference strains based on the amino acid sequences. The group names are indicated outside of trees. The pair or groups of homologues with high DNA sequence identity (95%, mostly 97%) are indicated in red. The Wzx homologues of O96 and O170, which are indicated in blue and by an asterisk, showed 86% DNA sequence identity, but in all other proteins showed low-sequence homologies to each other (&lt;70% identity). Note that while the DNA sequence identity between the wzx_O46 and wzx_O134 in Gp6 is 99.7%, the wzx_O46 has a 2-bp deletion at the 3-region, causing a frame shift." path-from-xml="dsu04302.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6612274/dsu04302.tif" path-from-xml="dsu04302.tif" target="_blank">View large</a><a section="6612274" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu043/2/dsu04302.png?Expires=2147483647&Signature=XiqtSiH6s6rz7~2CC0E9oTU2A3vsjdM094pp2Cn~YjSwb47b1IJt3SctT7L9gvFIpreQvzMx3Sx0RxifsWfuaSECTUjyroP~y9bbi39Xs0DLr92T13cjo~oUpaC8MVEdgoG3jrOQ-Zj7h2xZunXDQlpCXutmV7a7soJsGT6QcN7I-WCiXbdXWQPShUx3qTuy7y0zxRuV4fCcYzBQnfaCp1ibyHak3WCO2UmG3eDtsQ4fD-eSw1iX9DCZkWBYcjDoP5C2CfqE~5Rw1wDj3XRH1UlR9x2v0V5kXIifS3Cxxs1n9zjZliuPogEH4RJe~gIStXfdOIahoeMf52HbrATNqw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6612274&ar=442161&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04302.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic analysis of homologues of (A) Wzx and Wzy and (B) Wzm and Wzt from <em>Escherichia coli</em> O-serogroup reference strains based on the amino acid sequences. The group names are indicated outside of trees. The pair or groups of homologues with high DNA sequence identity (95%, mostly 97%) are indicated in red. The Wzx homologues of O96 and O170, which are indicated in blue and by an asterisk, showed 86% DNA sequence identity, but in all other proteins showed low-sequence homologies to each other (&lt;70% identity). Note that while the DNA sequence identity between the <em>wzx</em>_O46 and <em>wzx</em>_O134 in Gp6 is 99.7%, the <em>wzx</em>_O46 has a 2-bp deletion at the 3-region, causing a frame shift.</p></div></div>                            <h3 scrollto-destination=6612275 id="6612275" class="section-title" data-legacy-id=s3d>3.4. Phylogenetic relationships of <em>E. coli</em> O-serogroup reference strains</h3>
<p>Based on the concatenated nucleotide sequences of seven housekeeping genes used for MLST, we determined the evolutionary relationships of all <em>E. coli</em> O-serogroup reference strains (Fig.<a reveal-id="DSU043F3" data-open="DSU043F3" class="link link-reveal link-table xref-fig">3</a>). This analysis revealed that the members of five groups sharing the common O-AGCs (Gp8, Gp10, Gp11, Gp14, and Gp15) and two members (O17 and O77) of Gp9 were found in closely related lineages. However, the members of other groups (and three members of Gp9) were found in distinct evolutionary lineages. For example, O20 and O137, both carrying the Gp1 O-AGC, were found in two distinct lineages, each belonging to phylogroups A and E/D, respectively, and five serogroups (O17/O77, O44, O73, and O106) belonging to Gp9 were found in multiple lineages (A, E/D, and B1). </p>                        <a id="6612277" scrollto-destination="6612277"></a>
<div id="DSU043F3" data-id="DSU043F3" class="fig fig-section"><div class="label fig-label">Figure3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu043/2/m_dsu04303.png?Expires=2147483647&Signature=Vc1gZYhn7BX0LYMsemyCxDfn28jSitYUiQ2OZTpz~t9~mAHSqcPB4Novt1UiG4F1wWMHqS-yyG-orq1d6MW3bdGVn5za8GyfZUhjbhOsqZ7crbjBGmqFXiQhLVlh-n8ZLlyA6x-gY9xM9m01qv0bc8Z4EN46FE1AakHbXsSgYd5f1vmPjpom7kJzWgNW8Cj1GkVWk1gf9UJZJsgd4RexaZRLreX-HoMDuP1WJhv9EnZWQRhc3C-6qCQ4Bk~b9k7w8lq~oKKHCNEUhTZre6OrfY0nEmRnQsFKAMu0Orfh0Pm-qWX8Rt38TtesuAaZgG4bsVSPIAoHrByYAITai0K3Uw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the Escherichia coli evolutionary lineages and the distribution of O-AGCs. The phylogenetic tree was constructed based on the concatenated sequences of seven housekeeping genes from all 184 E. coli O-serogroup reference strains. The group names of O-AGCs (Gp1Gp16) are indicated in the outermost region. Members in groups indicated in green were found to belong to the same or very closely related lineage, whereas members of the groups indicated in blue were found in distinct lineages. The outer circle next to the O serogroup names indicates the distribution of sugar synthesis gene sets identified in each O-AGC. The inner circle indicates the type of O-antigen processing system (wzx/wzy or wzm/wzt). Phylogenetic groups (A, B1, B2, D, and E) were determined by comparing the sequences of the strains tested with the known sequences from the ECOR collection (http://mlst.warwick.ac.uk/mlst/dbs/Ecoli)." path-from-xml="dsu04303.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6612277/dsu04303.tif" path-from-xml="dsu04303.tif" target="_blank">View large</a><a section="6612277" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu043/2/dsu04303.png?Expires=2147483647&Signature=ojxOHcFhQ4CDKJpvNY53lBKnVnlb0Y2nPfIU4uX6-GkiF3shnbBqXj02MwP3kfKHEgpuS6W~qOtbAMhcahyGJm2huNhMcCPa2E3MIvsgVmusUX7O0AFAMAehGxmdKqhR7PACsgW96AD7g2ezozMs4h7G0ZfA8diXP6Mc5Sm0fwy7LyVZ2soBce8XdRCeJ71UjGlhOCWWtDSdzD0TrZ5izsGixRf~g~LUCYczCBE086o2yiVJeOFybnTKekaIAlCCYYKmQYM0CgnRZP1pB02zm03q8Dh-0RsskRyKjS7eLXwn1~rkcivMkZxeOmeqtQgvhY-MJo9Sd~CEs1EkysZCZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6612277&ar=442161&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04303.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the <em>Escherichia coli</em> evolutionary lineages and the distribution of O-AGCs. The phylogenetic tree was constructed based on the concatenated sequences of seven housekeeping genes from all 184 <em>E. coli</em> O-serogroup reference strains. The group names of O-AGCs (Gp1Gp16) are indicated in the outermost region. Members in groups indicated in green were found to belong to the same or very closely related lineage, whereas members of the groups indicated in blue were found in distinct lineages. The outer circle next to the O serogroup names indicates the distribution of sugar synthesis gene sets identified in each O-AGC. The inner circle indicates the type of O-antigen processing system (<em>wzx</em>/<em>wzy</em> or <em>wzm</em>/<em>wzt</em>). Phylogenetic groups (A, B1, B2, D, and E) were determined by comparing the sequences of the strains tested with the known sequences from the ECOR collection (<a class="link link-uri" href="http://mlst.warwick.ac.uk/mlst/dbs/Ecoli">http://mlst.warwick.ac.uk/mlst/dbs/Ecoli</a>).</p></div></div><div content-id="DSU043F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu043/2/m_dsu04303.png?Expires=2147483647&Signature=Vc1gZYhn7BX0LYMsemyCxDfn28jSitYUiQ2OZTpz~t9~mAHSqcPB4Novt1UiG4F1wWMHqS-yyG-orq1d6MW3bdGVn5za8GyfZUhjbhOsqZ7crbjBGmqFXiQhLVlh-n8ZLlyA6x-gY9xM9m01qv0bc8Z4EN46FE1AakHbXsSgYd5f1vmPjpom7kJzWgNW8Cj1GkVWk1gf9UJZJsgd4RexaZRLreX-HoMDuP1WJhv9EnZWQRhc3C-6qCQ4Bk~b9k7w8lq~oKKHCNEUhTZre6OrfY0nEmRnQsFKAMu0Orfh0Pm-qWX8Rt38TtesuAaZgG4bsVSPIAoHrByYAITai0K3Uw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Correlation between the Escherichia coli evolutionary lineages and the distribution of O-AGCs. The phylogenetic tree was constructed based on the concatenated sequences of seven housekeeping genes from all 184 E. coli O-serogroup reference strains. The group names of O-AGCs (Gp1Gp16) are indicated in the outermost region. Members in groups indicated in green were found to belong to the same or very closely related lineage, whereas members of the groups indicated in blue were found in distinct lineages. The outer circle next to the O serogroup names indicates the distribution of sugar synthesis gene sets identified in each O-AGC. The inner circle indicates the type of O-antigen processing system (wzx/wzy or wzm/wzt). Phylogenetic groups (A, B1, B2, D, and E) were determined by comparing the sequences of the strains tested with the known sequences from the ECOR collection (http://mlst.warwick.ac.uk/mlst/dbs/Ecoli)." path-from-xml="dsu04303.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6612277/dsu04303.tif" path-from-xml="dsu04303.tif" target="_blank">View large</a><a section="6612277" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu043/2/dsu04303.png?Expires=2147483647&Signature=ojxOHcFhQ4CDKJpvNY53lBKnVnlb0Y2nPfIU4uX6-GkiF3shnbBqXj02MwP3kfKHEgpuS6W~qOtbAMhcahyGJm2huNhMcCPa2E3MIvsgVmusUX7O0AFAMAehGxmdKqhR7PACsgW96AD7g2ezozMs4h7G0ZfA8diXP6Mc5Sm0fwy7LyVZ2soBce8XdRCeJ71UjGlhOCWWtDSdzD0TrZ5izsGixRf~g~LUCYczCBE086o2yiVJeOFybnTKekaIAlCCYYKmQYM0CgnRZP1pB02zm03q8Dh-0RsskRyKjS7eLXwn1~rkcivMkZxeOmeqtQgvhY-MJo9Sd~CEs1EkysZCZQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6612277&ar=442161&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu04303.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Correlation between the <em>Escherichia coli</em> evolutionary lineages and the distribution of O-AGCs. The phylogenetic tree was constructed based on the concatenated sequences of seven housekeeping genes from all 184 <em>E. coli</em> O-serogroup reference strains. The group names of O-AGCs (Gp1Gp16) are indicated in the outermost region. Members in groups indicated in green were found to belong to the same or very closely related lineage, whereas members of the groups indicated in blue were found in distinct lineages. The outer circle next to the O serogroup names indicates the distribution of sugar synthesis gene sets identified in each O-AGC. The inner circle indicates the type of O-antigen processing system (<em>wzx</em>/<em>wzy</em> or <em>wzm</em>/<em>wzt</em>). Phylogenetic groups (A, B1, B2, D, and E) were determined by comparing the sequences of the strains tested with the known sequences from the ECOR collection (<a class="link link-uri" href="http://mlst.warwick.ac.uk/mlst/dbs/Ecoli">http://mlst.warwick.ac.uk/mlst/dbs/Ecoli</a>).</p></div></div><p>The systematic phylogenetic analysis of all <em>E. coli</em> O-serogroup reference strains further revealed that one-quarter of the reference strains (46/184) belonged to a single clonal group (99.9% sequence identity), which was represented by sequence type (ST) 10 and its very close relatives in phylogroup A (Fig.<a reveal-id="DSU043F3" data-open="DSU043F3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Additionally, three clonal groups containing five or more reference strains were also identified in phylogroups A (ST34 and ST57) and B1 (ST300) (Fig.<a reveal-id="DSU043F3" data-open="DSU043F3" class="link link-reveal link-table xref-fig">3</a>). The phylogenetic analysis also showed that the types of sugar synthesis gene sets and processing gene sets (<em>wzx</em>/<em>wzy</em> and <em>wzm</em>/<em>wzt</em>) were not limited to a specific lineage (Fig.<a reveal-id="DSU043F3" data-open="DSU043F3" class="link link-reveal link-table xref-fig">3</a>).</p>                            <h3 scrollto-destination=6612279 id="6612279" class="section-title" data-legacy-id=s3e>3.5. Relationships of the <em>E. coli</em> and <em>Shigella</em> O-AGCs</h3>
<p><em>Shigella</em> and <em>E. coli</em> belong to the same species complex<sup><a reveal-id="DSU043C24" data-open="DSU043C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and many <em>Shigella</em> O antigens are known to be serologically and genetically identical or very similar to some <em>E. coli</em> O antigens, as summarized by Liu <em>et al.</em><sup><a reveal-id="DSU043C25" data-open="DSU043C25" class="link link-ref link-reveal xref-bibr">25</a></sup> In addition to the 21 previously shown relationships, we found two additional O-AGC groups shared by <em>E. coli</em> and <em>Shigella</em>; O38 and <em>Shigella dysenteria</em> type 8 (SD8), and O169/O183 and <em>Shigella boydii</em> type 6/10 (SB6/SB10) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). The O183-AGC was highly similar to the <em>S. boydii</em> types 10 cluster (sharing 98.2% DNA sequence identity). In our previous study,<sup><a reveal-id="DSU043C26" data-open="DSU043C26" class="link link-ref link-reveal xref-bibr">26</a></sup> we provisionally named a novel O serogroup for a group of Shiga toxin-producing <em>E. coli</em> strains as OSB10, which cross-reacted with <em>S. boydii</em> type 10. Sequence comparisons in this study revealed that OSB10 is not only serologically but also genetically identical to the new serogroup O183 of Gp16.</p>                            <h2 scrollto-destination=6612281 id="6612281" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Much of what we know about <em>E. coli</em> is defined at some level by O serogroups. To link genomic information to the wealth of data held in public databases, in our collective knowledge, outbreak, and disease reports and elsewhere, we endeavoured to determine whether molecular O-serogroup identification, targeting O-serogroup-specific genes (or unique sequences), was a valuable method to capture this information and maintain this important link. Not only do we show evidence supporting the effectiveness of molecular O-typing, but also we open up the possibility of generating a molecular O-typing scheme and relate O serogroups to the underlying phylogeny of this bacterium.</p><p>By determining and comparing the sequences of O-AGCs from all known <em>E. coli</em> O serogroups, we newly defined the sequence and gene content of 145 unique O-AGCs and showed that O-AGCs from 37 O serogroups could be placed into 16 groups based on members in each group sharing nearly identical or highly similar O-AGCs. It is clear from these data that many of the grouped O-AGCs (Gp1-16) were found in distinct phylogenetic lineages indicating that these O-AGCs have been spread across this species by horizontal gene transfer. Moreover, several lineages that contained multiple O serogroups, ST10, ST34, ST57, and ST300, show that frequent exchange occurs between and within lineages. ST10 and its close relatives are particularly interesting as one-quarter of <em>E. coli</em> O-serogroup reference strains fell within this clonal group. ST10 and its clonal complex are clinically very important being recently found to include ESBL-producing <em>E. coli</em> from human and animals in Spain,<sup><a reveal-id="DSU043C25" data-open="DSU043C25" class="link link-ref link-reveal xref-bibr">25</a></sup> Italy and Denmark,<sup><a reveal-id="DSU043C26" data-open="DSU043C26" class="link link-ref link-reveal xref-bibr">26</a></sup> China,<sup><a reveal-id="DSU043C27" data-open="DSU043C27" class="link link-ref link-reveal xref-bibr">27</a></sup> and the Netherlands,<sup><a reveal-id="DSU043C28" data-open="DSU043C28" class="link link-ref link-reveal xref-bibr">28</a></sup> and in various intra-intestinal pathotypes of <em>E. coli</em>, such as enteroaggregative <em>E. coli</em>,<sup><a reveal-id="DSU043C27" data-open="DSU043C27" class="link link-ref link-reveal xref-bibr">27</a>,<a reveal-id="DSU043C28" data-open="DSU043C28" class="link link-ref link-reveal xref-bibr">28</a></sup> enterotoxigenic <em>E. coli</em>,<sup><a reveal-id="DSU043C29" data-open="DSU043C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSU043C30" data-open="DSU043C30" class="link link-ref link-reveal xref-bibr">30</a></sup> and EHEC.<sup><a reveal-id="DSU043C31" data-open="DSU043C31" class="link link-ref link-reveal xref-bibr">31</a>,<a reveal-id="DSU043C32" data-open="DSU043C32" class="link link-ref link-reveal xref-bibr">32</a></sup> In most cases, the O serogroups of these ST10 or ST10-related strains are unusual compared with the typical O serogroups that represent that pathotype.</p><p>Acquisition of O-antigen modification genes located on the genomes of serotype-converting bacteriophages or plasmids is also an important strategy for diversifying O-antigen structures. This mechanism has been well investigated in <em>Shigella flexneri</em>.<sup><a reveal-id="DSU043C33" data-open="DSU043C33" class="link link-ref link-reveal xref-bibr">33</a>,<a reveal-id="DSU043C34" data-open="DSU043C34" class="link link-ref link-reveal xref-bibr">34</a></sup> In <em>E. coli</em>, the O-serogroup conversion by a prophage-like element has been reported for O17 and O44,<sup><a reveal-id="DSU043C17" data-open="DSU043C17" class="link link-ref link-reveal xref-bibr">17</a></sup> which belong to Gp9 defined in this study. Another possible mechanism to generate the variation of O antigens is the mutations in the genes of the O-AGC as observed for O107 and O117,<sup><a reveal-id="DSU043C16" data-open="DSU043C16" class="link link-ref link-reveal xref-bibr">16</a></sup> which belong to Gp8. In this case, point mutations in a glycosyltransferase gene are responsible for the alteration of O-antigen structure (and thus that of O serogroup).<sup><a reveal-id="DSU043C16" data-open="DSU043C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Five O-AGC groups including Gp2, Gp5, Gp7, Gp12, and Gp13 also contained differences in the amino acid sequence of their glycosyltransferases. O serogroup differences in these groups may be generated by the point mutations in glycosyltransferase genes. On the other hand, all glycosyltransferase genes in Gp1, Gp3, Gp4, Gp6, and Gp11; four strains from Gp9 (O17, O44, O73, and O77) and two from Gp10 (O13 and O129) showed 100% amino acid sequence identity. These results suggest that the serological differences between the members of these seven groups have been generated by acquisition of modification genes outside of the O-AGC as shown for O17 and O44 of Gp9.<sup><a reveal-id="DSU043C17" data-open="DSU043C17" class="link link-ref link-reveal xref-bibr">17</a></sup></p><p>We believe that the remarkable sequence diversity observed in the <em>wzx</em>/<em>wzy</em> and <em>wzm</em>/<em>wzt</em> O-AGC genes of all known <em>E. coli</em> O serogroups appears to be sufficiently discriminative from one another to make identification of each of the known O serogroups possible. Therefore, our sequence data will serve as a valuable resource for the development of rationally designed molecular methods for O-typing as well as for detecting novel O serogroups.</p><p>In conclusion, our study provides a complete sequence set of O-AGCs of all known <em>E. coli</em> O serogroups and thus offers a full view on the genetic diversity of O-AGCs of this bacterium. In addition, the results presented suggest that horizontal gene transfer has been involved in the O serogroup diversification in <em>E. coli</em> more frequently and in a more biased or lineage-dependent fashion than previously thought.</p>                            <h2 scrollto-destination=6612287 id="6612287" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6612289 id="6612289" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by Health Labor Sciences Research Grants from the Ministry of Health, Labor, and Welfare, Japan to A.I. (H25-Syokuhin-Wakate-018) and M.O. (H24-Shinkou-Ippan-012); Adaptable and Seamless Technology Transfer Program through Target-driven R&amp;D (AS242Z00217P) from Japan Science and Technology Agency to A.I.; and a Scientific Research Grant on Priority Areas from the University of Miyazaki and the Program to Disseminate Tenure Tracking System from the Japanese Ministry of Education, Culture, Sports, Science, and Technology to A.I. (<a class="link link-uri" href="http://www.miyazaki-u.ac.jp/ir/english/index.html">http://www.miyazaki-u.ac.jp/ir/english/index.html</a>). This work was also supported by Wellcome Trust grant (098051). Funding to pay the Open Access publication charges for this article was provided by the University of Miyazaki, Japan.</p>                            <h2 scrollto-destination=6612291 id="6612291" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank A. Akiyoshi, Y. Kato, and A. Yoshida for technical assistance.</p>                            <h2 scrollto-destination=6612293 id="6612293" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU043C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bazaka</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Crawford</div> <div class="given-names">R.J.</div></div>, <div class="name"><div class="surname">Nazarenko</div> <div class="given-names">E.L.</div></div>, <div class="name"><div class="surname">Ivanova</div> <div class="given-names">E.P.</div></div>.</div><div class="article-title">Bacterial extracellular polysaccharides</div>,<div class="source">Adv. Exp. Med. Biol.</div>,<div class="year">2011</div>,vol.<div class="volume">715</div>(pg.<div class="fpage">213</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bacterial%20extracellular%20polysaccharides&amp;author=K.%20Bazaka&amp;author=R.J.%20Crawford&amp;author=E.L.%20Nazarenko&amp;author=E.P.%20Ivanova&amp;publication_year=2011&amp;journal=Adv.%20Exp.%20Med.%20Biol.&amp;volume=715&amp;pages=213-26" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21557066" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>, <div class="name"><div class="surname">Knirel</div> <div class="given-names">Y.A.</div></div>, <div class="name"><div class="surname">Feng</div> <div class="given-names">L.</div></div>,et al.</div><div class="article-title">Structural diversity in <em>Salmonella</em> O antigens and its genetic basis</div>,<div class="source">FEMS Microbiol. Rev.</div>,<div class="year">2013</div>,vol.<div class="volume">38</div>(pg.<div class="fpage">56</div>-<div class="lpage">89</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20diversity%20in%20Salmonella%20O%20antigens%20and%20its%20genetic%20basis&amp;author=B.%20Liu&amp;author=Y.A.%20Knirel&amp;author=L.%20Feng&amp;publication_year=2013&amp;journal=FEMS%20Microbiol.%20Rev.&amp;volume=38&amp;pages=56-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/1574-6976.12034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2F1574-6976.12034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2F1574-6976.12034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23848592" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stenutz</div> <div class="given-names">R.</div></div>, <div class="name"><div class="surname">Weintraub</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Widmalm</div> <div class="given-names">G.</div></div>.</div><div class="article-title">The structures of <em>Escherichia coli</em> O-polysaccharide antigens</div>,<div class="source">FEMS Microbiol. Rev.</div>,<div class="year">2006</div>,vol.<div class="volume">30</div>(pg.<div class="fpage">382</div>-<div class="lpage">403</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20structures%20of%20Escherichia%20coli%20O-polysaccharide%20antigens&amp;author=R.%20Stenutz&amp;author=A.%20Weintraub&amp;author=G.%20Widmalm&amp;publication_year=2006&amp;journal=FEMS%20Microbiol.%20Rev.&amp;volume=30&amp;pages=382-403" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2006.00016.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2006.00016.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2006.00016.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16594963" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lam</div> <div class="given-names">J.S.</div></div>, <div class="name"><div class="surname">Taylor</div> <div class="given-names">V.L.</div></div>, <div class="name"><div class="surname">Islam</div> <div class="given-names">S.T.</div></div>, <div class="name"><div class="surname">Hao</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Kocincova</div> <div class="given-names">D.</div></div>.</div><div class="article-title">Genetic and functional diversity of <em>Pseudomonas aeruginosa</em> lipopolysaccharide</div>,<div class="source">Front Microbiol.</div>,<div class="year">2011</div>,vol.<div class="volume">2</div>pg.<div class="fpage">118</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20and%20functional%20diversity%20of%20Pseudomonas%20aeruginosa%20lipopolysaccharide&amp;author=J.S.%20Lam&amp;author=V.L.%20Taylor&amp;author=S.T.%20Islam&amp;author=Y.%20Hao&amp;author=D.%20Kocincova&amp;publication_year=2011&amp;journal=Front%20Microbiol.&amp;volume=2&amp;pages=118" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fmicb.2011.00118" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffmicb.2011.00118" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffmicb.2011.00118"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21687428" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Penner</div> <div class="given-names">J.L.</div></div>, <div class="name"><div class="surname">Aspinall</div> <div class="given-names">G.O.</div></div>.</div><div class="article-title">Diversity of lipopolysaccharide structures in <em>Campylobacter jejuni</em></div>,<div class="source">J. Infect. Dis.</div>,<div class="year">1997</div>,vol.<div class="volume">176</div><div class="issue">Suppl. 2</div>(pg.<div class="fpage">S135</div>-<div class="lpage">138</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Diversity%20of%20lipopolysaccharide%20structures%20in%20Campylobacter%20jejuni&amp;author=J.L.%20Penner&amp;author=G.O.%20Aspinall&amp;publication_year=1997&amp;journal=J.%20Infect.%20Dis.&amp;volume=176&amp;pages=S135-138" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/jid.1997.176.issue-s2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2Fjid.1997.176.issue-s2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2Fjid.1997.176.issue-s2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9396697" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Armstrong</div> <div class="given-names">G.L.</div></div>, <div class="name"><div class="surname">Hollingsworth</div> <div class="given-names">J.</div></div>, <div class="name"><div class="surname">Morris</div> <div class="given-names">J.G.</div>Jr.</div>.</div><div class="article-title">Emerging foodborne pathogens: <em>Escherichia coli</em> O157:H7 as a model of entry of a new pathogen into the food supply of the developed world</div>,<div class="source">Epidemiol Rev.</div>,<div class="year">1996</div>,vol.<div class="volume">18</div>(pg.<div class="fpage">29</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Emerging%20foodborne%20pathogens%3A%20Escherichia%20coli%20O157%3AH7%20as%20a%20model%20of%20entry%20of%20a%20new%20pathogen%20into%20the%20food%20supply%20of%20the%20developed%20world&amp;author=G.L.%20Armstrong&amp;author=J.%20Hollingsworth&amp;author=J.G.%20Morris&amp;publication_year=1996&amp;journal=Epidemiol%20Rev.&amp;volume=18&amp;pages=29-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.epirev.a017914" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.epirev.a017914" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.epirev.a017914"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8877329" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tarr</div> <div class="given-names">P.I.</div></div>, <div class="name"><div class="surname">Gordon</div> <div class="given-names">C.A.</div></div>, <div class="name"><div class="surname">Chandler</div> <div class="given-names">W.L.</div></div>.</div><div class="article-title">Shiga-toxin-producing <em>Escherichia coli</em> and haemolytic uraemic syndrome</div>,<div class="source">Lancet</div>,<div class="year">2005</div>,vol.<div class="volume">365</div>(pg.<div class="fpage">1073</div>-<div class="lpage">86</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Shiga-toxin-producing%20Escherichia%20coli%20and%20haemolytic%20uraemic%20syndrome&amp;author=P.I.%20Tarr&amp;author=C.A.%20Gordon&amp;author=W.L.%20Chandler&amp;publication_year=2005&amp;journal=Lancet&amp;volume=365&amp;pages=1073-86" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15781103" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Johnson</div> <div class="given-names">K.E.</div></div>, <div class="name"><div class="surname">Thorpe</div> <div class="given-names">C.M.</div></div>, <div class="name"><div class="surname">Sears</div> <div class="given-names">C.L.</div></div>.</div><div class="article-title">The emerging clinical importance of non-O157 Shiga toxin-producing <em>Escherichia coli</em></div>,<div class="source">Clin. Infect. Dis.</div>,<div class="year">2006</div>,vol.<div class="volume">43</div>(pg.<div class="fpage">1587</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20emerging%20clinical%20importance%20of%20non-O157%20Shiga%20toxin-producing%20Escherichia%20coli&amp;author=K.E.%20Johnson&amp;author=C.M.%20Thorpe&amp;author=C.L.%20Sears&amp;publication_year=2006&amp;journal=Clin.%20Infect.%20Dis.&amp;volume=43&amp;pages=1587-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1086/509573" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1086%2F509573" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1086%2F509573"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17109294" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Buchholz</div> <div class="given-names">U.</div></div>, <div class="name"><div class="surname">Bernard</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Werber</div> <div class="given-names">D.</div></div>,et al.</div><div class="article-title">German outbreak of <em>Escherichia coli</em> O104:H4 associated with sprouts</div>,<div class="source">N. Engl. J. Med.</div>,<div class="year">2011</div>,vol.<div class="volume">365</div>(pg.<div class="fpage">1763</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=German%20outbreak%20of%20Escherichia%20coli%20O104%3AH4%20associated%20with%20sprouts&amp;author=U.%20Buchholz&amp;author=H.%20Bernard&amp;author=D.%20Werber&amp;publication_year=2011&amp;journal=N.%20Engl.%20J.%20Med.&amp;volume=365&amp;pages=1763-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1056/NEJMoa1106482" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1056%2FNEJMoa1106482" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1056%2FNEJMoa1106482"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22029753" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Peirano</div> <div class="given-names">G.</div></div>, <div class="name"><div class="surname">Pitout</div> <div class="given-names">J.D.</div></div>.</div><div class="article-title">Molecular epidemiology of <em>Escherichia coli</em> producing CTX-M beta-lactamases: the worldwide emergence of clone ST131 O25:H4</div>,<div class="source">Int. J. Antimicrob. Agents</div>,<div class="year">2010</div>,vol.<div class="volume">35</div>(pg.<div class="fpage">316</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20epidemiology%20of%20Escherichia%20coli%20producing%20CTX-M%20beta-lactamases%3A%20the%20worldwide%20emergence%20of%20clone%20ST131%20O25%3AH4&amp;author=G.%20Peirano&amp;author=J.D.%20Pitout&amp;publication_year=2010&amp;journal=Int.%20J.%20Antimicrob.%20Agents&amp;volume=35&amp;pages=316-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ijantimicag.2009.11.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijantimicag.2009.11.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ijantimicag.2009.11.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20060273" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Samuel</div> <div class="given-names">G.</div></div>, <div class="name"><div class="surname">Reeves</div> <div class="given-names">P.</div></div>.</div><div class="article-title">Biosynthesis of O-antigens: genes and pathways involved in nucleotide sugar precursor synthesis and O-antigen assembly</div>,<div class="source">Carbohydr. Res.</div>,<div class="year">2003</div>,vol.<div class="volume">338</div>(pg.<div class="fpage">2503</div>-<div class="lpage">19</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biosynthesis%20of%20O-antigens%3A%20genes%20and%20pathways%20involved%20in%20nucleotide%20sugar%20precursor%20synthesis%20and%20O-antigen%20assembly&amp;author=G.%20Samuel&amp;author=P.%20Reeves&amp;publication_year=2003&amp;journal=Carbohydr.%20Res.&amp;volume=338&amp;pages=2503-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.carres.2003.07.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.carres.2003.07.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.carres.2003.07.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14670712" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">DebRoy</div> <div class="given-names">C.</div></div>, <div class="name"><div class="surname">Roberts</div> <div class="given-names">E.</div></div>, <div class="name"><div class="surname">Fratamico</div> <div class="given-names">P.M.</div></div>.</div><div class="article-title">Detection of O antigens in <em>Escherichia coli</em></div>,<div class="source">Anim. Health Res. Rev.</div>,<div class="year">2011</div>,vol.<div class="volume">12</div>(pg.<div class="fpage">169</div>-<div class="lpage">85</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20of%20O%20antigens%20in%20Escherichia%20coli&amp;author=C.%20DebRoy&amp;author=E.%20Roberts&amp;author=P.M.%20Fratamico&amp;publication_year=2011&amp;journal=Anim.%20Health%20Res.%20Rev.&amp;volume=12&amp;pages=169-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1017/S1466252311000193" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1017%2FS1466252311000193" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1017%2FS1466252311000193"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22152292" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Leopold</div> <div class="given-names">S.R.</div></div>, <div class="name"><div class="surname">Magrini</div> <div class="given-names">V.</div></div>, <div class="name"><div class="surname">Holt</div> <div class="given-names">N.J.</div></div>,et al.</div><div class="article-title">A precise reconstruction of the emergence and constrained radiations of <em>Escherichia coli</em> O157 portrayed by backbone concatenomic analysis</div>,<div class="source">Proc. Natl Acad. Sci. USA</div>,<div class="year">2009</div>,vol.<div class="volume">106</div>(pg.<div class="fpage">8713</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20precise%20reconstruction%20of%20the%20emergence%20and%20constrained%20radiations%20of%20Escherichia%20coli%20O157%20portrayed%20by%20backbone%20concatenomic%20analysis&amp;author=S.R.%20Leopold&amp;author=V.%20Magrini&amp;author=N.J.%20Holt&amp;publication_year=2009&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=106&amp;pages=8713-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0812949106" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0812949106" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0812949106"> </span></div></div></div></div></div></div><div content-id="DSU043C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iguchi</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Shirai</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Seto</div> <div class="given-names">K.</div></div>,et al.</div><div class="article-title">Wide distribution of O157-antigen biosynthesis gene clusters in <em>Escherichia coli</em></div>,<div class="source">PLoS ONE</div>,<div class="year">2011</div>,vol.<div class="volume">6</div>pg.<div class="fpage">e23250</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Wide%20distribution%20of%20O157-antigen%20biosynthesis%20gene%20clusters%20in%20Escherichia%20coli&amp;author=A.%20Iguchi&amp;author=H.%20Shirai&amp;author=K.%20Seto&amp;publication_year=2011&amp;journal=PLoS%20ONE&amp;volume=6&amp;pages=e23250" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0023250" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0023250" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0023250"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21876740" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iguchi</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Iyoda</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Ohnishi</div> <div class="given-names">M.</div></div>.</div><div class="article-title">Molecular characterization reveals three distinct clonal groups among clinical Shiga toxin-producing <em>Escherichia coli</em> strains of serogroup O103</div>,<div class="source">J. Clin. Microbiol.</div>,<div class="year">2012</div>,vol.<div class="volume">50</div>(pg.<div class="fpage">2894</div>-<div class="lpage">900</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characterization%20reveals%20three%20distinct%20clonal%20groups%20among%20clinical%20Shiga%20toxin-producing%20Escherichia%20coli%20strains%20of%20serogroup%20O103&amp;author=A.%20Iguchi&amp;author=S.%20Iyoda&amp;author=M.%20Ohnishi&amp;publication_year=2012&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=50&amp;pages=2894-900" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JCM.00789-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJCM.00789-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJCM.00789-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22718945" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Q.</div></div>, <div class="name"><div class="surname">Perepelov</div> <div class="given-names">A.V.</div></div>, <div class="name"><div class="surname">Wen</div> <div class="given-names">L.</div></div>,et al.</div><div class="article-title">Identification of the two glycosyltransferase genes responsible for the difference between <em>Escherichia coli</em> O107 and O117 O-antigens</div>,<div class="source">Glycobiology</div>,<div class="year">2012</div>,vol.<div class="volume">22</div>(pg.<div class="fpage">281</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20the%20two%20glycosyltransferase%20genes%20responsible%20for%20the%20difference%20between%20Escherichia%20coli%20O107%20and%20O117%20O-antigens&amp;author=Q.%20Wang&amp;author=A.V.%20Perepelov&amp;author=L.%20Wen&amp;publication_year=2012&amp;journal=Glycobiology&amp;volume=22&amp;pages=281-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/glycob/cwr137" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fglycob%2Fcwr137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fglycob%2Fcwr137"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21968437" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">W.</div></div>, <div class="name"><div class="surname">Perepelov</div> <div class="given-names">A.V.</div></div>, <div class="name"><div class="surname">Feng</div> <div class="given-names">L.</div></div>,et al.</div><div class="article-title">A group of <em>Escherichia coli</em> and <em>Salmonella</em> enterica O antigens sharing a common backbone structure</div>,<div class="source">Microbiology</div>,<div class="year">2007</div>,vol.<div class="volume">153</div>(pg.<div class="fpage">2159</div>-<div class="lpage">67</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20group%20of%20Escherichia%20coli%20and%20Salmonella%20enterica%20O%20antigens%20sharing%20a%20common%20backbone%20structure&amp;author=W.%20Wang&amp;author=A.V.%20Perepelov&amp;author=L.%20Feng&amp;publication_year=2007&amp;journal=Microbiology&amp;volume=153&amp;pages=2159-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.2007/004192-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.2007%2F004192-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.2007%2F004192-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17600060" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lacher</div> <div class="given-names">D.W.</div></div>, <div class="name"><div class="surname">Gangiredla</div> <div class="given-names">J.</div></div>, <div class="name"><div class="surname">Jackson</div> <div class="given-names">S.A.</div></div>, <div class="name"><div class="surname">Elkins</div> <div class="given-names">C.A.</div></div>, <div class="name"><div class="surname">Feng</div> <div class="given-names">P.C.</div></div>.</div><div class="article-title">Novel microarray design for molecular serotyping of Shiga toxin-producing <em>Escherichia coli</em> isolated from fresh produce</div>,<div class="source">Appl. Environ. Microbiol.</div>,<div class="year">2014</div>,vol.<div class="volume">80</div>(pg.<div class="fpage">4677</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20microarray%20design%20for%20molecular%20serotyping%20of%20Shiga%20toxin-producing%20Escherichia%20coli%20isolated%20from%20fresh%20produce&amp;author=D.W.%20Lacher&amp;author=J.%20Gangiredla&amp;author=S.A.%20Jackson&amp;author=C.A.%20Elkins&amp;author=P.C.%20Feng&amp;publication_year=2014&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=80&amp;pages=4677-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.01049-14" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.01049-14" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.01049-14"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24837388" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tzschoppe</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Martin</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Beutin</div> <div class="given-names">L.</div></div>.</div><div class="article-title">A rapid procedure for the detection and isolation of enterohaemorrhagic <em>Escherichia coli</em> (EHEC) serogroup O26, O103, O111, O118, O121, O145 and O157 strains and the aggregative EHEC O104:H4 strain from ready-to-eat vegetables</div>,<div class="source">Int. J. Food Microbiol.</div>,<div class="year">2012</div>,vol.<div class="volume">152</div>(pg.<div class="fpage">19</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20rapid%20procedure%20for%20the%20detection%20and%20isolation%20of%20enterohaemorrhagic%20Escherichia%20coli%20(EHEC)%20serogroup%20O26%2C%20O103%2C%20O111%2C%20O118%2C%20O121%2C%20O145%20and%20O157%20strains%20and%20the%20aggregative%20EHEC%20O104%3AH4%20strain%20from%20ready-to-eat%20vegetables&amp;author=M.%20Tzschoppe&amp;author=A.%20Martin&amp;author=L.%20Beutin&amp;publication_year=2012&amp;journal=Int.%20J.%20Food%20Microbiol.&amp;volume=152&amp;pages=19-30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ijfoodmicro.2011.10.009" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ijfoodmicro.2011.10.009" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ijfoodmicro.2011.10.009"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22071287" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Q.</div></div>, <div class="name"><div class="surname">Ruan</div> <div class="given-names">X.</div></div>, <div class="name"><div class="surname">Wei</div> <div class="given-names">D.</div></div>,et al.</div><div class="article-title">Development of a serogroup-specific multiplex PCR assay to detect a set of <em>Escherichia coli</em> serogroups based on the identification of their O-antigen gene clusters</div>,<div class="source">Mol. Cell Probes</div>,<div class="year">2010</div>,vol.<div class="volume">24</div>(pg.<div class="fpage">286</div>-<div class="lpage">90</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20serogroup-specific%20multiplex%20PCR%20assay%20to%20detect%20a%20set%20of%20Escherichia%20coli%20serogroups%20based%20on%20the%20identification%20of%20their%20O-antigen%20gene%20clusters&amp;author=Q.%20Wang&amp;author=X.%20Ruan&amp;author=D.%20Wei&amp;publication_year=2010&amp;journal=Mol.%20Cell%20Probes&amp;volume=24&amp;pages=286-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mcp.2010.06.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mcp.2010.06.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mcp.2010.06.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20561581" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J.D.</div></div>, <div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.</div></div>, <div class="name"><div class="surname">Gibson</div> <div class="given-names">T.J.</div></div>.</div><div class="article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</div>,<div class="source">Nucleic Acids Res.</div>,<div class="year">1994</div>,vol.<div class="volume">22</div>(pg.<div class="fpage">4673</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%20W%3A%20improving%20the%20sensitivity%20of%20progressive%20multiple%20sequence%20alignment%20through%20sequence%20weighting%2C%20position-specific%20gap%20penalties%20and%20weight%20matrix%20choice&amp;author=J.D.%20Thompson&amp;author=D.G.%20Higgins&amp;author=T.J.%20Gibson&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res.&amp;volume=22&amp;pages=4673-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.22.4673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.22.4673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7984417" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Dudley</div> <div class="given-names">J.</div></div>, <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div>.</div><div class="article-title">MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0</div>,<div class="source">Mol. Biol. Evol.</div>,<div class="year">2007</div>,vol.<div class="volume">24</div>(pg.<div class="fpage">1596</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA4%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20(MEGA)%20software%20version%204.0&amp;author=K.%20Tamura&amp;author=J.%20Dudley&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=24&amp;pages=1596-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17488738" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jensen</div> <div class="given-names">S.O.</div></div>, <div class="name"><div class="surname">Reeves</div> <div class="given-names">P.R.</div></div>.</div><div class="article-title">Deletion of the <em>Escherichia coli</em> O14:K7 O antigen gene cluster</div>,<div class="source">Can. J. Microbiol.</div>,<div class="year">2004</div>,vol.<div class="volume">50</div>(pg.<div class="fpage">299</div>-<div class="lpage">302</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deletion%20of%20the%20Escherichia%20coli%20O14%3AK7%20O%20antigen%20gene%20cluster&amp;author=S.O.%20Jensen&amp;author=P.R.%20Reeves&amp;publication_year=2004&amp;journal=Can.%20J.%20Microbiol.&amp;volume=50&amp;pages=299-302" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/w04-010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fw04-010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fw04-010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15213754" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pupo</div> <div class="given-names">G.M.</div></div>, <div class="name"><div class="surname">Lan</div> <div class="given-names">R.</div></div>, <div class="name"><div class="surname">Reeves</div> <div class="given-names">P.R.</div></div>.</div><div class="article-title">Multiple independent origins of <em>Shigella</em> clones of <em>Escherichia coli</em> and convergent evolution of many of their characteristics</div>,<div class="source">Proc. Natl Acad. Sci. USA</div>,<div class="year">2000</div>,vol.<div class="volume">97</div>(pg.<div class="fpage">10567</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20independent%20origins%20of%20Shigella%20clones%20of%20Escherichia%20coli%20and%20convergent%20evolution%20of%20many%20of%20their%20characteristics&amp;author=G.M.%20Pupo&amp;author=R.%20Lan&amp;author=P.R.%20Reeves&amp;publication_year=2000&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=97&amp;pages=10567-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.180094797" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.180094797" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.180094797"> </span></div></div></div></div></div></div><div content-id="DSU043C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">B.</div></div>, <div class="name"><div class="surname">Knirel</div> <div class="given-names">Y.A.</div></div>, <div class="name"><div class="surname">Feng</div> <div class="given-names">L.</div></div>,et al.</div><div class="article-title">Structure and genetics of <em>Shigella</em> O antigens</div>,<div class="source">FEMS Microbiol. Rev.</div>,<div class="year">2008</div>,vol.<div class="volume">32</div>(pg.<div class="fpage">627</div>-<div class="lpage">53</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure%20and%20genetics%20of%20Shigella%20O%20antigens&amp;author=B.%20Liu&amp;author=Y.A.%20Knirel&amp;author=L.%20Feng&amp;publication_year=2008&amp;journal=FEMS%20Microbiol.%20Rev.&amp;volume=32&amp;pages=627-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1574-6976.2008.00114.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1574-6976.2008.00114.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1574-6976.2008.00114.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18422615" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iguchi</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Iyoda</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Seto</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Ohnishi</div> <div class="given-names">M.</div></div>.</div><div class="article-title">Emergence of a novel Shiga toxin-producing <em>Escherichia coli</em> O serogroup cross-reacting with <em>Shigella boydii</em> type 10</div>,<div class="source">J. Clin. Microbiol.</div>,<div class="year">2011</div>,vol.<div class="volume">49</div>(pg.<div class="fpage">3678</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Emergence%20of%20a%20novel%20Shiga%20toxin-producing%20Escherichia%20coli%20O%20serogroup%20cross-reacting%20with%20Shigella%20boydii%20type%2010&amp;author=A.%20Iguchi&amp;author=S.%20Iyoda&amp;author=K.%20Seto&amp;author=M.%20Ohnishi&amp;publication_year=2011&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=49&amp;pages=3678-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JCM.01197-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJCM.01197-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJCM.01197-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21865424" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Olesen</div> <div class="given-names">B.</div></div>, <div class="name"><div class="surname">Scheutz</div> <div class="given-names">F.</div></div>, <div class="name"><div class="surname">Andersen</div> <div class="given-names">R.L.</div></div>,et al.</div><div class="article-title">Enteroaggregative <em>Escherichia coli</em> O78:H10, the cause of an outbreak of urinary tract infection</div>,<div class="source">J. Clin. Microbiol.</div>,<div class="year">2012</div>,vol.<div class="volume">50</div>(pg.<div class="fpage">3703</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enteroaggregative%20Escherichia%20coli%20O78%3AH10%2C%20the%20cause%20of%20an%20outbreak%20of%20urinary%20tract%20infection&amp;author=B.%20Olesen&amp;author=F.%20Scheutz&amp;author=R.L.%20Andersen&amp;publication_year=2012&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=50&amp;pages=3703-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JCM.01909-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJCM.01909-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJCM.01909-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22972830" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okeke</div> <div class="given-names">I.N.</div></div>, <div class="name"><div class="surname">Wallace-Gadsden</div> <div class="given-names">F.</div></div>, <div class="name"><div class="surname">Simons</div> <div class="given-names">H.R.</div></div>,et al.</div><div class="article-title">Multi-locus sequence typing of enteroaggregative <em>Escherichia coli</em> isolates from Nigerian children uncovers multiple lineages</div>,<div class="source">PLoS ONE</div>,<div class="year">2010</div>,vol.<div class="volume">5</div>pg.<div class="fpage">e14093</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multi-locus%20sequence%20typing%20of%20enteroaggregative%20Escherichia%20coli%20isolates%20from%20Nigerian%20children%20uncovers%20multiple%20lineages&amp;author=I.N.%20Okeke&amp;author=F.%20Wallace-Gadsden&amp;author=H.R.%20Simons&amp;publication_year=2010&amp;journal=PLoS%20ONE&amp;volume=5&amp;pages=e14093" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0014093" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0014093" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0014093"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21124856" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Turner</div> <div class="given-names">S.M.</div></div>, <div class="name"><div class="surname">Chaudhuri</div> <div class="given-names">R.R.</div></div>, <div class="name"><div class="surname">Jiang</div> <div class="given-names">Z.D.</div></div>,et al.</div><div class="article-title">Phylogenetic comparisons reveal multiple acquisitions of the toxin genes by enterotoxigenic <em>Escherichia coli</em> strains of different evolutionary lineages</div>,<div class="source">J. Clin. Microbiol.</div>,<div class="year">2006</div>,vol.<div class="volume">44</div>(pg.<div class="fpage">4528</div>-<div class="lpage">36</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20comparisons%20reveal%20multiple%20acquisitions%20of%20the%20toxin%20genes%20by%20enterotoxigenic%20Escherichia%20coli%20strains%20of%20different%20evolutionary%20lineages&amp;author=S.M.%20Turner&amp;author=R.R.%20Chaudhuri&amp;author=Z.D.%20Jiang&amp;publication_year=2006&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=44&amp;pages=4528-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JCM.01474-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJCM.01474-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJCM.01474-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17050815" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nada</div> <div class="given-names">R.A.</div></div>, <div class="name"><div class="surname">Shaheen</div> <div class="given-names">H.I.</div></div>, <div class="name"><div class="surname">Khalil</div> <div class="given-names">S.B.</div></div>,et al.</div><div class="article-title">Discovery and phylogenetic analysis of novel members of class b enterotoxigenic <em>Escherichia coli</em> adhesive fimbriae</div>,<div class="source">J. Clin. Microbiol.</div>,<div class="year">2011</div>,vol.<div class="volume">49</div>(pg.<div class="fpage">1403</div>-<div class="lpage">10</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20and%20phylogenetic%20analysis%20of%20novel%20members%20of%20class%20b%20enterotoxigenic%20Escherichia%20coli%20adhesive%20fimbriae&amp;author=R.A.%20Nada&amp;author=H.I.%20Shaheen&amp;author=S.B.%20Khalil&amp;publication_year=2011&amp;journal=J.%20Clin.%20Microbiol.&amp;volume=49&amp;pages=1403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JCM.02006-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJCM.02006-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJCM.02006-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21289147" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Monaghan</div> <div class="given-names">A.M.</div></div>, <div class="name"><div class="surname">Byrne</div> <div class="given-names">B.</div></div>, <div class="name"><div class="surname">McDowell</div> <div class="given-names">D.</div></div>, <div class="name"><div class="surname">Carroll</div> <div class="given-names">A.M.</div></div>, <div class="name"><div class="surname">McNamara</div> <div class="given-names">E.B.</div></div>, <div class="name"><div class="surname">Bolton</div> <div class="given-names">D.J.</div></div>.</div><div class="article-title">Characterization of farm, food, and clinical Shiga toxin-producing <em>Escherichia coli</em> (STEC) O113</div>,<div class="source">Foodborne Pathog. Dis.</div>,<div class="year">2012</div>,vol.<div class="volume">9</div>(pg.<div class="fpage">1088</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20farm%2C%20food%2C%20and%20clinical%20Shiga%20toxin-producing%20Escherichia%20coli%20(STEC)%20O113&amp;author=A.M.%20Monaghan&amp;author=B.%20Byrne&amp;author=D.%20McDowell&amp;author=A.M.%20Carroll&amp;author=E.B.%20McNamara&amp;author=D.J.%20Bolton&amp;publication_year=2012&amp;journal=Foodborne%20Pathog.%20Dis.&amp;volume=9&amp;pages=1088-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/fpd.2012.1257" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Ffpd.2012.1257" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Ffpd.2012.1257"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23237408" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hauser</div> <div class="given-names">E.</div></div>, <div class="name"><div class="surname">Mellmann</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Semmler</div> <div class="given-names">T.</div></div>,et al.</div><div class="article-title">Phylogenetic and molecular analysis of food-borne shiga toxin-producing <em>Escherichia coli</em></div>,<div class="source">Appl. Environ. Microbiol.</div>,<div class="year">2013</div>,vol.<div class="volume">79</div>(pg.<div class="fpage">2731</div>-<div class="lpage">40</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phylogenetic%20and%20molecular%20analysis%20of%20food-borne%20shiga%20toxin-producing%20Escherichia%20coli&amp;author=E.%20Hauser&amp;author=A.%20Mellmann&amp;author=T.%20Semmler&amp;publication_year=2013&amp;journal=Appl.%20Environ.%20Microbiol.&amp;volume=79&amp;pages=2731-40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/AEM.03552-12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FAEM.03552-12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FAEM.03552-12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23417002" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Allison</div> <div class="given-names">G.E.</div></div>, <div class="name"><div class="surname">Verma</div> <div class="given-names">N.K.</div></div>.</div><div class="article-title">Serotype-converting bacteriophages and O-antigen modification in <em>Shigella flexneri</em></div>,<div class="source">Trends Microbiol.</div>,<div class="year">2000</div>,vol.<div class="volume">8</div>(pg.<div class="fpage">17</div>-<div class="lpage">23</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Serotype-converting%20bacteriophages%20and%20O-antigen%20modification%20in%20Shigella%20flexneri&amp;author=G.E.%20Allison&amp;author=N.K.%20Verma&amp;publication_year=2000&amp;journal=Trends%20Microbiol.&amp;volume=8&amp;pages=17-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0966-842X(99)01646-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0966-842X(99)01646-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0966-842X(99)01646-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10637639" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSU043C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU043C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">Q.</div></div>, <div class="name"><div class="surname">Knirel</div> <div class="given-names">Y.A.</div></div>, <div class="name"><div class="surname">Lan</div> <div class="given-names">R.</div></div>,et al.</div><div class="article-title">A novel plasmid-encoded serotype conversion mechanism through addition of phosphoethanolamine to the O-antigen of <em>Shigella flexneri</em></div>,<div class="source">PLoS ONE</div>,<div class="year">2012</div>,vol.<div class="volume">7</div>pg.<div class="fpage">e46095</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20plasmid-encoded%20serotype%20conversion%20mechanism%20through%20addition%20of%20phosphoethanolamine%20to%20the%20O-antigen%20of%20Shigella%20flexneri&amp;author=Q.%20Sun&amp;author=Y.A.%20Knirel&amp;author=R.%20Lan&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e46095" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0046095" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0046095" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0046095"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23049947" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement"> The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/4.0/), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=442161&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/22/1/10.1093/dnares/dsu043/2/dsu043-dsu043supp_data.pdf?Expires=2147483647&amp;Signature=uad3l2XexgVjh0SO0rgq4NwbIcN~dgOdkpd~9wLI0FrqTO7ed~3QG1A6~9YP9woczU3lxziBGOxuSzd0XSVS4yA9KHoBGAqB2mucPC1yACQOhyCbJR3zRKpXY7B-GOKmC0dKnBO~m9EGaEtjiLbRJQYoMsaXdMm6yayAubUE6wSEDUhXsAcEKCRvPWihnT3hsSNs1zMU720XWhbuoyERL-H~3DQjjGnhgSUJv2BkMDKaL4ljUoEYVKWqAPmamJKd6yaiXK8rqRYkuf74zzK6WnWUHPtmYW6BwQ7AEbMXdtCw82WM3ZdMTgSbtWdy~RKxvaU8bPgXAuMR3WT9HLVRLg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,081</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100010&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">64</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu043"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="442161"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000350442100010&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f22%2f1%2f101%2f442161">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000350442100010&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (64)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F22%2F1%2F101%2F442161" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/442161'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionduplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="complete view of the genetic diversity of the Escherichia coli O-antigen biosynthesis gene cluster" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu043");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f22%2f1%2f101%2f442161">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

