

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/17/6/353/484440","@type":"ScholarlyArticle","name":"Genome-Wide Survey and Expression Analysis Suggest Diverse Roles of Glutaredoxin Gene Family Members During Development and Response to Various Stimuli in Rice","datePublished":"2010-11-02","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/17/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2010-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsq023","keywords":["glutaredoxin","rice","development","stress","gene family"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/17/6/353/484440","author":[{"name":"Garg, Rohini","affiliation":"National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Jhanwar, Shalu","affiliation":"National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Tyagi, Akhilesh K.","affiliation":"National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"},{"name":"Jain, Mukesh","affiliation":"National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067 ,  India","@type":"Person"}],"description":"Abstract.  Glutaredoxins (GRXs) are glutathione-dependent oxidoreductase enzymes involved in a variety of cellular processes. In this study, our analysis reveal","pageStart":"353","pageEnd":"367","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/6/1/m_cover.gif?Expires=2147483647&Signature=GE4QDx7vPXS7wjUaoLIKdeQMVYvKpNXob0I6QlcCM7oe~eKCfP8E380kotmiw9z~axHxmBJESzx9vV88~C9g-CA5dZPWzHX3y0HSadJ2Jrf2k3i9guvDwVaIIvgF6cZI1iLndCWPhjSM1C0sC8Omes66kjDc9sn4S8VlQHryn-wE9mPjK2sZ-quW-rVkH966kcOElJ2hx~w7cpINeghmflNy5cmWJN-jp6Sp3iFBqCubFpEA3485f5KqRLMCbhc5nMYcpZWmmznkyRdy0CL0lxP-Xr1JDHnrIseUrn4fTYBDpMQY0Dv8f9-vnTHQwjT0Mz0eJIlyt6vd81Yiu4619w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome-Wide Survey and Expression Analysis Suggest Diverse Roles of Glutaredoxin Gene Family Members During Development and Response to Various Stimuli in Rice","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/6/1/m_cover.gif?Expires=2147483647&Signature=GE4QDx7vPXS7wjUaoLIKdeQMVYvKpNXob0I6QlcCM7oe~eKCfP8E380kotmiw9z~axHxmBJESzx9vV88~C9g-CA5dZPWzHX3y0HSadJ2Jrf2k3i9guvDwVaIIvgF6cZI1iLndCWPhjSM1C0sC8Omes66kjDc9sn4S8VlQHryn-wE9mPjK2sZ-quW-rVkH966kcOElJ2hx~w7cpINeghmflNy5cmWJN-jp6Sp3iFBqCubFpEA3485f5KqRLMCbhc5nMYcpZWmmznkyRdy0CL0lxP-Xr1JDHnrIseUrn4fTYBDpMQY0Dv8f9-vnTHQwjT0Mz0eJIlyt6vd81Yiu4619w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome-Wide Survey and Expression Analysis Suggest Diverse Roles of Glutaredoxin Gene Family Members During Development and Response to Various Stimuli in Rice" />
    <meta property="og:description" content="Abstract.  Glutaredoxins (GRXs) are glutathione-dependent oxidoreductase enzymes involved in a variety of cellular processes. In this study, our analysis reveal" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/17/6/353/484440" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2010-11-02" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Garg, Rohini" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Jhanwar, Shalu" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Tyagi, Akhilesh K." /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_author" content="Jain, Mukesh" /><meta name="citation_author_institution" content="National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067, India" /><meta name="citation_title" content="Genome-Wide Survey and Expression Analysis Suggest Diverse Roles of Glutaredoxin Gene Family Members During Development and Response to Various Stimuli in Rice" /><meta name="citation_firstpage" content="353" /><meta name="citation_lastpage" content="367" /><meta name="citation_doi" content="10.1093/dnares/dsq023" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="17" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2010/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Glutaredoxin systems; Biochim. Biophys. Acta;  citation_year=2008;  citation_volume=1780;  citation_pages=1304-17; " /><meta name="citation_reference" content="citation_title=The role of glutathione in photosynthetic organisms: emerging functions for glutaredoxins and glutathionylation; Annu. Rev. Plant Biol.;  citation_year=2008;  citation_volume=59;  citation_pages=143-66; " /><meta name="citation_reference" content="citation_title=Plant glutaredoxins: still mysterious reducing systems; Cell Mol. Life Sci.;  citation_year=2004;  citation_volume=61;  citation_pages=1266-77; " /><meta name="citation_reference" content="citation_title=The glutaredoxin family in oxygenic photosynthetic organisms; Photosynth. Res.;  citation_year=2004;  citation_volume=79;  citation_pages=305-18; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of plant glutaredoxin systems; J. Exp. Bot.;  citation_year=2006;  citation_volume=57;  citation_pages=1685-96; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of a new peroxiredoxin from poplar sieve tubes that uses either glutaredoxin or thioredoxin as a proton donor; Plant Physiol.;  citation_year=2001;  citation_volume=127;  citation_pages=1299-309; " /><meta name="citation_reference" content="citation_title=Exploring the active site of plant glutaredoxin by site-directed mutagenesis; FEBS Lett.;  citation_year=2002;  citation_volume=511;  citation_pages=145-9; " /><meta name="citation_reference" content="citation_title=Functional, structural, and spectroscopic characterization of a glutathione-ligated [2Fe-2S] cluster in poplar glutaredoxin C1; Proc. Natl Acad. Sci. USA;  citation_year=2007;  citation_volume=104;  citation_pages=7379-84; " /><meta name="citation_reference" content="citation_title=AtGRXcp, an Arabidopsis chloroplastic glutaredoxin, is critical for protection against protein oxidative damage; J. Biol. Chem.;  citation_year=2006;  citation_volume=281;  citation_pages=26280-8; " /><meta name="citation_reference" content="citation_title=AtGRX4, an Arabidopsis chloroplastic monothiol glutaredoxin, is able to suppress yeast grx5 mutant phenotypes and respond to oxidative stress; FEBS Lett.;  citation_year=2008;  citation_volume=582;  citation_pages=848-54; " /><meta name="citation_reference" content="citation_title=ROXY1, a member of the plant glutaredoxin family, is required for petal development in Arabidopsis thaliana; Development;  citation_year=2005;  citation_volume=132;  citation_pages=1555-65; " /><meta name="citation_reference" content="citation_title=ROXY1 and ROXY2, two Arabidopsis glutaredoxin genes, are required for anther development; Plant J.;  citation_year=2008;  citation_volume=53;  citation_pages=790-801; " /><meta name="citation_reference" content="citation_title=Conserved functions of Arabidopsis and rice CC-type glutaredoxins in flower development and pathogen response; Mol. Plant;  citation_year=2009;  citation_volume=2;  citation_pages=323-35; " /><meta name="citation_reference" content="citation_title=SA-inducible Arabidopsis glutaredoxin interacts with TGA factors and suppresses JA-responsive PDF1.2 transcription; Plant J.;  citation_year=2007;  citation_volume=50;  citation_pages=128-39; " /><meta name="citation_reference" content="citation_title=Cloning of the cDNA for glutaredoxin, an abundant sieve-tube exudate protein from Ricinus communis L. and characterisation of the glutathione-dependent thiol-reduction system in sieve tubes; Planta;  citation_year=1997;  citation_volume=202;  citation_pages=349-56; " /><meta name="citation_reference" content="citation_title=Cloning and sequence analysis of a cDNA encoding rice glutaredoxin; FEBS Lett.;  citation_year=1994;  citation_volume=337;  citation_pages=157-60; " /><meta name="citation_reference" content="citation_title=Purification and characterization of glutaredoxin (thioltransferase) from rice (Oryza sativa L.); J. Biochem.;  citation_year=1997;  citation_volume=121;  citation_pages=842-8; " /><meta name="citation_reference" content="citation_title=Locating proteins in the cell using TargetP, SignalP and related tools; Nat. Protoc.;  citation_year=2007;  citation_volume=2;  citation_pages=953-71; " /><meta name="citation_reference" content="citation_title=Fitting a mixture model by expectation maximization to discover motifs in biopolymers; citation_publisher=AAAI Press, Menlo Park, CA;  citation_year=1994;  citation_pages=28-36; " /><meta name="citation_reference" content="citation_title=An expression atlas of rice mRNAs and small RNAs; Nat. Biotechnol.;  citation_year=2007;  citation_volume=25;  citation_pages=473-7; " /><meta name="citation_reference" content="citation_title=F-box proteins in rice. Genome-wide analysis, classification, temporal and spatial gene expression during panicle and seed development, and regulation by light and abiotic stress; Plant Physiol.;  citation_year=2007;  citation_volume=143;  citation_pages=1467-83; " /><meta name="citation_reference" content="citation_title=Rice-arsenate interactions in hydroponics: whole genome transcriptional analysis; J. Exp. Bot.;  citation_year=2008;  citation_volume=59;  citation_pages=2267-76; " /><meta name="citation_reference" content="citation_title=Transcript profiling reveals diverse roles of auxin-responsive genes during reproductive development and abiotic stress in rice; FEBS J.;  citation_year=2009;  citation_volume=276;  citation_pages=3148-62; " /><meta name="citation_reference" content="citation_title=Susceptibility of rice to the blast fungus, Magnaporthe grisea; J. Plant Physiol.;  citation_year=2008;  citation_volume=165;  citation_pages=114-24; " /><meta name="citation_reference" content="citation_title=Global patterns of gene expression in rice cultivars undergoing a susceptible or resistant interaction with the parasitic plant Striga hermonthica; New Phytol.;  citation_year=2008;  citation_volume=179;  citation_pages=515-29; " /><meta name="citation_reference" content="citation_title=Validation of housekeeping genes as internal control for studying gene expression in rice by quantitative real-time PCR; Biochem. Biophys. Res. Commun.;  citation_year=2006;  citation_volume=345;  citation_pages=646-51; " /><meta name="citation_reference" content="citation_title=Comparative protein modelling by satisfaction of spatial restraints; J. Mol. Biol.;  citation_year=1993;  citation_volume=234;  citation_pages=779-815; " /><meta name="citation_reference" content="citation_title=PROCHECK: a program to check the stereochemical quality of protein structures; J. Appl. Crystallogr.;  citation_year=1993;  citation_volume=26;  citation_pages=283-91; " /><meta name="citation_reference" content="citation_title=Redox regulation and flower development: a novel function for glutaredoxins; Plant Biol.;  citation_year=2006;  citation_volume=8;  citation_pages=547-55; " /><meta name="citation_reference" content="citation_title=Identification of a new family of plant proteins loosely related to glutaredoxins with four CxxC motives; Photosynth. Res.;  citation_year=2006;  citation_volume=89;  citation_pages=71-9; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of intronless genes in rice and Arabidopsis; Funct. Integr. Genomics;  citation_year=2008;  citation_volume=8;  citation_pages=69-78; " /><meta name="citation_reference" content="citation_title=Introns in, introns out in plant gene families: a genomic approach of the dynamics of gene structure; J. Struct. Funct. Genomics;  citation_year=2003;  citation_volume=3;  citation_pages=111-6; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis of Arabidopsis pentatricopeptide repeat proteins reveals their essential role in organelle biogenesis; Plant Cell;  citation_year=2004;  citation_volume=16;  citation_pages=2089-103; " /><meta name="citation_reference" content="citation_title=Genome-wide analysis, evolutionary expansion, and expression of early auxin-responsive SAUR gene family in rice (Oryza sativa); Genomics;  citation_year=2006;  citation_volume=88;  citation_pages=360-71; " /><meta name="citation_reference" content="citation_title=Origin and diversification of land plant CC-type glutaredoxins; Genome Biol. Evol.;  citation_year=2009;  citation_volume=1;  citation_pages=265-77; " /><meta name="citation_reference" content="citation_title=Nuclear activity of ROXY1, a glutaredoxin interacting with TGA factors, is required for petal development in Arabidopsis thaliana; Plant Cell;  citation_year=2009;  citation_volume=21;  citation_pages=429-41; " /><meta name="citation_reference" content="citation_title=Structure-function relationship of the chloroplastic glutaredoxin S12 with an atypical WCSYS active site; J. Biol. Chem.;  citation_year=2009;  citation_volume=284;  citation_pages=9299-310; " /><meta name="citation_reference" content="citation_title=Identification and localization of the first glutaredoxin in leaves of a higher plant; FEBS Lett.;  citation_year=1995;  citation_volume=369;  citation_pages=149-52; " /><meta name="citation_reference" content="citation_title=Cloning and characterization of CXIP1, a novel PICOT domain-containing Arabidopsis protein that associates with CAX1; J. Biol. Chem.;  citation_year=2003;  citation_volume=278;  citation_pages=6503-9; " /><meta name="citation_reference" content="citation_title=Expression of a Pteris vittata glutaredoxin PvGRX5 in transgenic Arabidopsis thaliana increases plant arsenic tolerance and decreases arsenic accumulation in the leaves; Plant Cell Environ.;  citation_year=2009;  citation_volume=32;  citation_pages=851-8; " /><meta name="citation_reference" content="citation_title=Rice gene network inferred from expression profiling of plants overexpressing OsWRKY13, a positive regulator of disease resistance; Mol. Plant;  citation_year=2008;  citation_volume=1;  citation_pages=538-51; " /><meta name="citation_reference" content="citation_title=Identification of plant glutaredoxin targets; Antioxid. Redox Signal.;  citation_year=2005;  citation_volume=7;  citation_pages=919-29; " /><meta name="citation_reference" content="citation_title=The evolutionary fate and consequences of duplicate genes; Science;  citation_year=2000;  citation_volume=290;  citation_pages=1151-5; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/17/6/353/1309562/dsq023.pdf" /><meta name="description" content="Abstract.  Glutaredoxins (GRXs) are glutathione-dependent oxidoreductase enzymes involved in a variety of cellular processes. In this study, our analysis reveal" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/17/6/353/484440" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/17/6/353/484440" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/17/6/353/484440");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsq023';
    </script>
<title>
	Genome-Wide Survey and Expression Analysis Suggest Diverse Roles of Glutaredoxin Gene Family Members During Development and Response to Various Stimuli in Rice | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./484440" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeBgPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHYXJnLCBSb2hpbmkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAgMDY3LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkpoYW53YXIsIFNoYWx1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iTmF0aW9uYWwgSW5zdGl0dXRlIG9mIFBsYW50IEdlbm9tZSBSZXNlYXJjaCAoTklQR1IpLCBBcnVuYSBBc2FmIEFsaSBNYXJnLCBOZXcgRGVsaGkgMTEwIDA2NywgSW5kaWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUeWFnaSwgQWtoaWxlc2ggSy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJOYXRpb25hbCBJbnN0aXR1dGUgb2YgUGxhbnQgR2Vub21lIFJlc2VhcmNoIChOSVBHUiksIEFydW5hIEFzYWYgQWxpIE1hcmcsIE5ldyBEZWxoaSAxMTAgMDY3LCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkphaW4sIE11a2VzaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik5hdGlvbmFsIEluc3RpdHV0ZSBvZiBQbGFudCBHZW5vbWUgUmVzZWFyY2ggKE5JUEdSKSwgQXJ1bmEgQXNhZiBBbGkgTWFyZywgTmV3IERlbGhpIDExMCAwNjcsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5vbWUtV2lkZSBTdXJ2ZXkgYW5kIEV4cHJlc3Npb24gQW5hbHlzaXMgU3VnZ2VzdCBEaXZlcnNlIFJvbGVzIG9mIEdsdXRhcmVkb3hpbiBHZW5lIEZhbWlseSBNZW1iZXJzIER1cmluZyBEZXZlbG9wbWVudCBhbmQgUmVzcG9uc2UgdG8gVmFyaW91cyBTdGltdWxpIGluIFJpY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIzNTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjM2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzcTAyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEwLzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2x1dGFyZWRveGluIHN5c3RlbXM7IEJpb2NoaW0uIEJpb3BoeXMuIEFjdGE7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTc4MDsgIGNpdGF0aW9uX3BhZ2VzPTEzMDQtMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSByb2xlIG9mIGdsdXRhdGhpb25lIGluIHBob3Rvc3ludGhldGljIG9yZ2FuaXNtczogZW1lcmdpbmcgZnVuY3Rpb25zIGZvciBnbHV0YXJlZG94aW5zIGFuZCBnbHV0YXRoaW9ueWxhdGlvbjsgQW5udS4gUmV2LiBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01OTsgIGNpdGF0aW9uX3BhZ2VzPTE0My02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnQgZ2x1dGFyZWRveGluczogc3RpbGwgbXlzdGVyaW91cyByZWR1Y2luZyBzeXN0ZW1zOyBDZWxsIE1vbC4gTGlmZSBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9MTI2Ni03NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdsdXRhcmVkb3hpbiBmYW1pbHkgaW4gb3h5Z2VuaWMgcGhvdG9zeW50aGV0aWMgb3JnYW5pc21zOyBQaG90b3N5bnRoLiBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTc5OyAgY2l0YXRpb25fcGFnZXM9MzA1LTE4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBwbGFudCBnbHV0YXJlZG94aW4gc3lzdGVtczsgSi4gRXhwLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTU3OyAgY2l0YXRpb25fcGFnZXM9MTY4NS05NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgbmV3IHBlcm94aXJlZG94aW4gZnJvbSBwb3BsYXIgc2lldmUgdHViZXMgdGhhdCB1c2VzIGVpdGhlciBnbHV0YXJlZG94aW4gb3IgdGhpb3JlZG94aW4gYXMgYSBwcm90b24gZG9ub3I7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEyNzsgIGNpdGF0aW9uX3BhZ2VzPTEyOTktMzA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeHBsb3JpbmcgdGhlIGFjdGl2ZSBzaXRlIG9mIHBsYW50IGdsdXRhcmVkb3hpbiBieSBzaXRlLWRpcmVjdGVkIG11dGFnZW5lc2lzOyBGRUJTIExldHQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTUxMTsgIGNpdGF0aW9uX3BhZ2VzPTE0NS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdW5jdGlvbmFsLCBzdHJ1Y3R1cmFsLCBhbmQgc3BlY3Ryb3Njb3BpYyBjaGFyYWN0ZXJpemF0aW9uIG9mIGEgZ2x1dGF0aGlvbmUtbGlnYXRlZCBbMkZlLTJTXSBjbHVzdGVyIGluIHBvcGxhciBnbHV0YXJlZG94aW4gQzE7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9NzM3OS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXRHUlhjcCwgYW4gQXJhYmlkb3BzaXMgY2hsb3JvcGxhc3RpYyBnbHV0YXJlZG94aW4sIGlzIGNyaXRpY2FsIGZvciBwcm90ZWN0aW9uIGFnYWluc3QgcHJvdGVpbiBveGlkYXRpdmUgZGFtYWdlOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yODE7ICBjaXRhdGlvbl9wYWdlcz0yNjI4MC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BdEdSWDQsIGFuIEFyYWJpZG9wc2lzIGNobG9yb3BsYXN0aWMgbW9ub3RoaW9sIGdsdXRhcmVkb3hpbiwgaXMgYWJsZSB0byBzdXBwcmVzcyB5ZWFzdCBncng1IG11dGFudCBwaGVub3R5cGVzIGFuZCByZXNwb25kIHRvIG94aWRhdGl2ZSBzdHJlc3M7IEZFQlMgTGV0dC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTgyOyAgY2l0YXRpb25fcGFnZXM9ODQ4LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ST1hZMSwgYSBtZW1iZXIgb2YgdGhlIHBsYW50IGdsdXRhcmVkb3hpbiBmYW1pbHksIGlzIHJlcXVpcmVkIGZvciBwZXRhbCBkZXZlbG9wbWVudCBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgRGV2ZWxvcG1lbnQ7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTMyOyAgY2l0YXRpb25fcGFnZXM9MTU1NS02NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk9YWTEgYW5kIFJPWFkyLCB0d28gQXJhYmlkb3BzaXMgZ2x1dGFyZWRveGluIGdlbmVzLCBhcmUgcmVxdWlyZWQgZm9yIGFudGhlciBkZXZlbG9wbWVudDsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTM7ICBjaXRhdGlvbl9wYWdlcz03OTAtODAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zZXJ2ZWQgZnVuY3Rpb25zIG9mIEFyYWJpZG9wc2lzIGFuZCByaWNlIENDLXR5cGUgZ2x1dGFyZWRveGlucyBpbiBmbG93ZXIgZGV2ZWxvcG1lbnQgYW5kIHBhdGhvZ2VuIHJlc3BvbnNlOyBNb2wuIFBsYW50OyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0zMjMtMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNBLWluZHVjaWJsZSBBcmFiaWRvcHNpcyBnbHV0YXJlZG94aW4gaW50ZXJhY3RzIHdpdGggVEdBIGZhY3RvcnMgYW5kIHN1cHByZXNzZXMgSkEtcmVzcG9uc2l2ZSBQREYxLjIgdHJhbnNjcmlwdGlvbjsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NTA7ICBjaXRhdGlvbl9wYWdlcz0xMjgtMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNsb25pbmcgb2YgdGhlIGNETkEgZm9yIGdsdXRhcmVkb3hpbiwgYW4gYWJ1bmRhbnQgc2lldmUtdHViZSBleHVkYXRlIHByb3RlaW4gZnJvbSBSaWNpbnVzIGNvbW11bmlzIEwuIGFuZCBjaGFyYWN0ZXJpc2F0aW9uIG9mIHRoZSBnbHV0YXRoaW9uZS1kZXBlbmRlbnQgdGhpb2wtcmVkdWN0aW9uIHN5c3RlbSBpbiBzaWV2ZSB0dWJlczsgUGxhbnRhOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTIwMjsgIGNpdGF0aW9uX3BhZ2VzPTM0OS01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xvbmluZyBhbmQgc2VxdWVuY2UgYW5hbHlzaXMgb2YgYSBjRE5BIGVuY29kaW5nIHJpY2UgZ2x1dGFyZWRveGluOyBGRUJTIExldHQuOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTMzNzsgIGNpdGF0aW9uX3BhZ2VzPTE1Ny02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHVyaWZpY2F0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGdsdXRhcmVkb3hpbiAodGhpb2x0cmFuc2ZlcmFzZSkgZnJvbSByaWNlIChPcnl6YSBzYXRpdmEgTC4pOyBKLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0xMjE7ICBjaXRhdGlvbl9wYWdlcz04NDItODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TG9jYXRpbmcgcHJvdGVpbnMgaW4gdGhlIGNlbGwgdXNpbmcgVGFyZ2V0UCwgU2lnbmFsUCBhbmQgcmVsYXRlZCB0b29sczsgTmF0LiBQcm90b2MuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz05NTMtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpdHRpbmcgYSBtaXh0dXJlIG1vZGVsIGJ5IGV4cGVjdGF0aW9uIG1heGltaXphdGlvbiB0byBkaXNjb3ZlciBtb3RpZnMgaW4gYmlvcG9seW1lcnM7IGNpdGF0aW9uX3B1Ymxpc2hlcj1BQUFJIFByZXNzLCBNZW5sbyBQYXJrLCBDQTsgIGNpdGF0aW9uX3llYXI9MTk5NDsgIGNpdGF0aW9uX3BhZ2VzPTI4LTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBleHByZXNzaW9uIGF0bGFzIG9mIHJpY2UgbVJOQXMgYW5kIHNtYWxsIFJOQXM7IE5hdC4gQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz00NzMtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Ri1ib3ggcHJvdGVpbnMgaW4gcmljZS4gR2Vub21lLXdpZGUgYW5hbHlzaXMsIGNsYXNzaWZpY2F0aW9uLCB0ZW1wb3JhbCBhbmQgc3BhdGlhbCBnZW5lIGV4cHJlc3Npb24gZHVyaW5nIHBhbmljbGUgYW5kIHNlZWQgZGV2ZWxvcG1lbnQsIGFuZCByZWd1bGF0aW9uIGJ5IGxpZ2h0IGFuZCBhYmlvdGljIHN0cmVzczsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTQzOyAgY2l0YXRpb25fcGFnZXM9MTQ2Ny04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmljZS1hcnNlbmF0ZSBpbnRlcmFjdGlvbnMgaW4gaHlkcm9wb25pY3M6IHdob2xlIGdlbm9tZSB0cmFuc2NyaXB0aW9uYWwgYW5hbHlzaXM7IEouIEV4cC4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01OTsgIGNpdGF0aW9uX3BhZ2VzPTIyNjctNzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHQgcHJvZmlsaW5nIHJldmVhbHMgZGl2ZXJzZSByb2xlcyBvZiBhdXhpbi1yZXNwb25zaXZlIGdlbmVzIGR1cmluZyByZXByb2R1Y3RpdmUgZGV2ZWxvcG1lbnQgYW5kIGFiaW90aWMgc3RyZXNzIGluIHJpY2U7IEZFQlMgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mjc2OyAgY2l0YXRpb25fcGFnZXM9MzE0OC02MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3VzY2VwdGliaWxpdHkgb2YgcmljZSB0byB0aGUgYmxhc3QgZnVuZ3VzLCBNYWduYXBvcnRoZSBncmlzZWE7IEouIFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE2NTsgIGNpdGF0aW9uX3BhZ2VzPTExNC0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2xvYmFsIHBhdHRlcm5zIG9mIGdlbmUgZXhwcmVzc2lvbiBpbiByaWNlIGN1bHRpdmFycyB1bmRlcmdvaW5nIGEgc3VzY2VwdGlibGUgb3IgcmVzaXN0YW50IGludGVyYWN0aW9uIHdpdGggdGhlIHBhcmFzaXRpYyBwbGFudCBTdHJpZ2EgaGVybW9udGhpY2E7IE5ldyBQaHl0b2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE3OTsgIGNpdGF0aW9uX3BhZ2VzPTUxNS0yOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmFsaWRhdGlvbiBvZiBob3VzZWtlZXBpbmcgZ2VuZXMgYXMgaW50ZXJuYWwgY29udHJvbCBmb3Igc3R1ZHlpbmcgZ2VuZSBleHByZXNzaW9uIGluIHJpY2UgYnkgcXVhbnRpdGF0aXZlIHJlYWwtdGltZSBQQ1I7IEJpb2NoZW0uIEJpb3BoeXMuIFJlcy4gQ29tbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNDU7ICBjaXRhdGlvbl9wYWdlcz02NDYtNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIHByb3RlaW4gbW9kZWxsaW5nIGJ5IHNhdGlzZmFjdGlvbiBvZiBzcGF0aWFsIHJlc3RyYWludHM7IEouIE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTM7ICBjaXRhdGlvbl92b2x1bWU9MjM0OyAgY2l0YXRpb25fcGFnZXM9Nzc5LTgxNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UFJPQ0hFQ0s6IGEgcHJvZ3JhbSB0byBjaGVjayB0aGUgc3RlcmVvY2hlbWljYWwgcXVhbGl0eSBvZiBwcm90ZWluIHN0cnVjdHVyZXM7IEouIEFwcGwuIENyeXN0YWxsb2dyLjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTI4My05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVkb3ggcmVndWxhdGlvbiBhbmQgZmxvd2VyIGRldmVsb3BtZW50OiBhIG5vdmVsIGZ1bmN0aW9uIGZvciBnbHV0YXJlZG94aW5zOyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9NTQ3LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBhIG5ldyBmYW1pbHkgb2YgcGxhbnQgcHJvdGVpbnMgbG9vc2VseSByZWxhdGVkIHRvIGdsdXRhcmVkb3hpbnMgd2l0aCBmb3VyIEN4eEMgbW90aXZlczsgUGhvdG9zeW50aC4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT04OTsgIGNpdGF0aW9uX3BhZ2VzPTcxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGFuYWx5c2lzIG9mIGludHJvbmxlc3MgZ2VuZXMgaW4gcmljZSBhbmQgQXJhYmlkb3BzaXM7IEZ1bmN0LiBJbnRlZ3IuIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz02OS03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50cm9ucyBpbiwgaW50cm9ucyBvdXQgaW4gcGxhbnQgZ2VuZSBmYW1pbGllczogYSBnZW5vbWljIGFwcHJvYWNoIG9mIHRoZSBkeW5hbWljcyBvZiBnZW5lIHN0cnVjdHVyZTsgSi4gU3RydWN0LiBGdW5jdC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTExMS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcyBvZiBBcmFiaWRvcHNpcyBwZW50YXRyaWNvcGVwdGlkZSByZXBlYXQgcHJvdGVpbnMgcmV2ZWFscyB0aGVpciBlc3NlbnRpYWwgcm9sZSBpbiBvcmdhbmVsbGUgYmlvZ2VuZXNpczsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTIwODktMTAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBhbmFseXNpcywgZXZvbHV0aW9uYXJ5IGV4cGFuc2lvbiwgYW5kIGV4cHJlc3Npb24gb2YgZWFybHkgYXV4aW4tcmVzcG9uc2l2ZSBTQVVSIGdlbmUgZmFtaWx5IGluIHJpY2UgKE9yeXphIHNhdGl2YSk7IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTg4OyAgY2l0YXRpb25fcGFnZXM9MzYwLTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcmlnaW4gYW5kIGRpdmVyc2lmaWNhdGlvbiBvZiBsYW5kIHBsYW50IENDLXR5cGUgZ2x1dGFyZWRveGluczsgR2Vub21lIEJpb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0yNjUtNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51Y2xlYXIgYWN0aXZpdHkgb2YgUk9YWTEsIGEgZ2x1dGFyZWRveGluIGludGVyYWN0aW5nIHdpdGggVEdBIGZhY3RvcnMsIGlzIHJlcXVpcmVkIGZvciBwZXRhbCBkZXZlbG9wbWVudCBpbiBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgUGxhbnQgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTQyOS00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJlLWZ1bmN0aW9uIHJlbGF0aW9uc2hpcCBvZiB0aGUgY2hsb3JvcGxhc3RpYyBnbHV0YXJlZG94aW4gUzEyIHdpdGggYW4gYXR5cGljYWwgV0NTWVMgYWN0aXZlIHNpdGU7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTI4NDsgIGNpdGF0aW9uX3BhZ2VzPTkyOTktMzEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgbG9jYWxpemF0aW9uIG9mIHRoZSBmaXJzdCBnbHV0YXJlZG94aW4gaW4gbGVhdmVzIG9mIGEgaGlnaGVyIHBsYW50OyBGRUJTIExldHQuOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTM2OTsgIGNpdGF0aW9uX3BhZ2VzPTE0OS01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xvbmluZyBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiBDWElQMSwgYSBub3ZlbCBQSUNPVCBkb21haW4tY29udGFpbmluZyBBcmFiaWRvcHNpcyBwcm90ZWluIHRoYXQgYXNzb2NpYXRlcyB3aXRoIENBWDE7IEouIEJpb2wuIENoZW0uOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTI3ODsgIGNpdGF0aW9uX3BhZ2VzPTY1MDMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXhwcmVzc2lvbiBvZiBhIFB0ZXJpcyB2aXR0YXRhIGdsdXRhcmVkb3hpbiBQdkdSWDUgaW4gdHJhbnNnZW5pYyBBcmFiaWRvcHNpcyB0aGFsaWFuYSBpbmNyZWFzZXMgcGxhbnQgYXJzZW5pYyB0b2xlcmFuY2UgYW5kIGRlY3JlYXNlcyBhcnNlbmljIGFjY3VtdWxhdGlvbiBpbiB0aGUgbGVhdmVzOyBQbGFudCBDZWxsIEVudmlyb24uOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9ODUxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJpY2UgZ2VuZSBuZXR3b3JrIGluZmVycmVkIGZyb20gZXhwcmVzc2lvbiBwcm9maWxpbmcgb2YgcGxhbnRzIG92ZXJleHByZXNzaW5nIE9zV1JLWTEzLCBhIHBvc2l0aXZlIHJlZ3VsYXRvciBvZiBkaXNlYXNlIHJlc2lzdGFuY2U7IE1vbC4gUGxhbnQ7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTsgIGNpdGF0aW9uX3BhZ2VzPTUzOC01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgcGxhbnQgZ2x1dGFyZWRveGluIHRhcmdldHM7IEFudGlveGlkLiBSZWRveCBTaWduYWwuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz05MTktMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBldm9sdXRpb25hcnkgZmF0ZSBhbmQgY29uc2VxdWVuY2VzIG9mIGR1cGxpY2F0ZSBnZW5lczsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0yOTA7ICBjaXRhdGlvbl9wYWdlcz0xMTUxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8xNy82LzM1My8xMzA5NTYyL2RzcTAyMy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgR2x1dGFyZWRveGlucyAoR1JYcykgYXJlIGdsdXRhdGhpb25lLWRlcGVuZGVudCBveGlkb3JlZHVjdGFzZSBlbnp5bWVzIGludm9sdmVkIGluIGEgdmFyaWV0eSBvZiBjZWxsdWxhciBwcm9jZXNzZXMuIEluIHRoaXMgc3R1ZHksIG91ciBhbmFseXNpcyByZXZlYWwiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ4NDQ0MCZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGRaJUNQ/jsI+2v5L8j62+bE36eBOg==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f353%2f484440" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f353%2f484440" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=19064" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/17/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/17/6/1/m_cover.gif?Expires=2147483647&amp;Signature=GE4QDx7vPXS7wjUaoLIKdeQMVYvKpNXob0I6QlcCM7oe~eKCfP8E380kotmiw9z~axHxmBJESzx9vV88~C9g-CA5dZPWzHX3y0HSadJ2Jrf2k3i9guvDwVaIIvgF6cZI1iLndCWPhjSM1C0sC8Omes66kjDc9sn4S8VlQHryn-wE9mPjK2sZ-quW-rVkH966kcOElJ2hx~w7cpINeghmflNy5cmWJN-jp6Sp3iFBqCubFpEA3485f5KqRLMCbhc5nMYcpZWmmznkyRdy0CL0lxP-Xr1JDHnrIseUrn4fTYBDpMQY0Dv8f9-vnTHQwjT0Mz0eJIlyt6vd81Yiu4619w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/17/6">
                        <div class="volume trailing-comma">Volume 17</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2010        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7358986">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7358986">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7358989">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7358989">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7358993">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7358993">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7359016">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7359016">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7359056">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7359056">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7359058">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7359058">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7359060">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7359060">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/17/6/343/484317">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/17/6/369/484698">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome-Wide Survey and Expression Analysis Suggest Diverse Roles of Glutaredoxin Gene Family Members During Development and Response to Various Stimuli in Rice
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rohini Garg</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rohini+Garg">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Garg R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Garg R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rohini Garg</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shalu Jhanwar</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shalu+Jhanwar">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jhanwar S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jhanwar S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shalu Jhanwar</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Akhilesh K. Tyagi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Akhilesh+K.+Tyagi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tyagi A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tyagi A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Akhilesh K. Tyagi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mukesh Jain</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="addr-line">National Institute of Plant Genome Research (NIPGR), Aruna Asaf Ali Marg, New Delhi 110 067</div>, <div class="country">India</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +91 11-26735182. Fax. <div class="fax">+91 11-26741658</div>. E-mail: <a href="mailto:mjain@nipgr.res.in" target="_blank">mjain@nipgr.res.in</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mukesh+Jain">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jain M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jain M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mukesh Jain</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Kazuo Shinozaki</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 17, Issue 6, 1 December 2010, Pages 353–367, <a href='https://doi.org/10.1093/dnares/dsq023'>https://doi.org/10.1093/dnares/dsq023</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">02 November 2010</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">28 June 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">14 September 2010</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="484440" href="/dnaresearch/article-pdf/17/6/353/1309562/dsq023.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Rohini Garg, Shalu Jhanwar, Akhilesh K. Tyagi, Mukesh Jain;  Genome-Wide Survey and Expression Analysis Suggest Diverse Roles of Glutaredoxin Gene Family Members During Development and Response to Various Stimuli in Rice, <em>DNA Research</em>, Volume 17, Issue 6, 1 December 2010, Pages 353–367, <a href='https://doi.org/10.1093/dnares/dsq023'>https://doi.org/10.1093/dnares/dsq023</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/484440?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/484440?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/484440?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/484440?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/484440?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/484440?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/484440?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome-Wide%20Survey%20and%20Expression%20Analysis%20Suggest%20Diverse%20Roles%20of%20Glutaredoxin%20Gene%20Family%20Members%20During%20Development%20and%20Response%20to%20Various%20Stimuli%20in%20Rice&amp;publicationDate=2010-11-02&amp;volumeNum=17&amp;issueNum=6&amp;author=Garg%2C%20Rohini%3B%20Jhanwar%2C%20Shalu&amp;startPage=353&amp;endPage=367&amp;contentId=10.1093%2Fdnares%2Fdsq023&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f353%2f484440" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7358986 id="7358986" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Glutaredoxins (GRXs) are glutathione-dependent oxidoreductase enzymes involved in a variety of cellular processes. In this study, our analysis revealed the presence of 48 genes encoding GRX proteins in the rice genome. GRX proteins could be classified into four classes, namely CC-, CGFS-, CPYC- and GRL-type, based on phylogenetic analysis. The classification was supported with organization of predicted conserved putative motifs in GRX proteins. We found that expansion of this gene family has occurred largely via whole genome duplication events in a species-specific manner. We explored rice oligonucleotide array data to gain insights into the function of GRX gene family members during various stages of development and in response to environmental stimuli. The comprehensive expression analysis suggested diverse roles of GRX genes during growth and development in rice. Some of the GRX genes were expressed in specific organs/developmental stages only. The expression of many of rice GRX genes was influenced by various phytohormones, abiotic and biotic stress conditions, suggesting an important role of GRX proteins in response to these stimuli. The identification of GRX genes showing differential expression in specific tissues or in response to environmental stimuli provide a new avenue for in-depth characterization of selected genes of importance.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="glutaredoxin">glutaredoxin</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="rice">rice</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="development">development</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="stress">stress</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene family&quot;">gene family</a></div>                            <h2 scrollto-destination=7358989 id="7358989" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Glutaredoxins (GRXs) are ubiquitous oxidoreductase enzymes that mediate reversible reduction of disulphide bonds of their substrate proteins in the presence of glutathione (GSH) via a dithiol or monothiol mechanism. GRXs generally contain a conserved cys-x-x-cys (dithiol GRXs) or cys-x-x-ser (monothiol GRXs) active-site motif, which is involved in reduction reaction. GRXs maintain and regulate cellular redox state and redox-dependent signaling pathways.<sup><a reveal-id="DSQ023C1" data-open="DSQ023C1" class="link link-ref link-reveal xref-bibr">1</a></sup> These enzymes are involved in diverse cellular processes and play an important role in defense against oxidative stress.<sup><a reveal-id="DSQ023C2" data-open="DSQ023C2" class="link link-ref link-reveal xref-bibr">2</a></sup> GRXs are present in a number of isoforms in prokaryotes and eukaryotes.</p><p>Several cDNAs encoding GRX proteins have been isolated from various plant species.<sup><a reveal-id="DSQ023C3" data-open="DSQ023C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSQ023C4" data-open="DSQ023C4" class="link link-ref link-reveal xref-bibr">4</a></sup> In <em>Arabidopsis</em>, at least 31 GRX genes have been predicted and divided into three major classes, namely CPYC-, CGFS- and CC-type, based on their predicted amino acid sequences and the composition of active-site motifs.<sup><a reveal-id="DSQ023C3  DSQ023C5" data-open="DSQ023C3  DSQ023C5" class="link link-ref link-reveal xref-bibr">3–5</a></sup> Among these, CPYC- and CGFS-type classes exist in all organisms from prokaryotes to eukaryotes including plants, whereas the CC-type class has so far been identified only in land plants. The information on biochemical activity, structure and function of plant GRXs is very limited. Plant GRX isoforms possess the ability to reduce dehydroascorbate and type II peroxiredoxin.<sup><a reveal-id="DSQ023C6" data-open="DSQ023C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSQ023C7" data-open="DSQ023C7" class="link link-ref link-reveal xref-bibr">7</a></sup> Recently, structure–function analyses indicated that poplar GrxC1 contains a {2Fe–2S} cluster in the holo form, suggesting a potential role of this Grx in sensing oxidative stress and iron–sulfur biosynthesis.<sup><a reveal-id="DSQ023C8" data-open="DSQ023C8" class="link link-ref link-reveal xref-bibr">8</a></sup> The chloroplast-targeted AtGRXcp and AtGRX4 have been shown to play a critical role in oxidative stress response and redox regulation.<sup><a reveal-id="DSQ023C9" data-open="DSQ023C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSQ023C10" data-open="DSQ023C10" class="link link-ref link-reveal xref-bibr">10</a></sup> Molecular genetic evidence suggests that <em>ROXY1</em>, encoding a CC-type GRX, is required for petal development in <em>Arabidopsis</em>.<sup><a reveal-id="DSQ023C11" data-open="DSQ023C11" class="link link-ref link-reveal xref-bibr">11</a></sup> Further, it has been shown that ROXY1 together with its closest homolog, ROXY2, play a major role in control of male gametogenesis.<sup><a reveal-id="DSQ023C12" data-open="DSQ023C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Recently, the function of their closest homologs in rice have also been found to be conserved.<sup><a reveal-id="DSQ023C13" data-open="DSQ023C13" class="link link-ref link-reveal xref-bibr">13</a></sup> It has been demonstrated that GRX interacts with TGA class of bZIP transcription factors and is involved in salicylic acid/jasmonic acid crosstalk.<sup><a reveal-id="DSQ023C14" data-open="DSQ023C14" class="link link-ref link-reveal xref-bibr">14</a></sup> The probable role of plant GRXs in long-distance signaling has also been proposed due to their detection in phloem sieve tubes.<sup><a reveal-id="DSQ023C3" data-open="DSQ023C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSQ023C15" data-open="DSQ023C15" class="link link-ref link-reveal xref-bibr">15</a></sup></p><p>Although some of the members of GRX gene family have been identified, cloned and characterized in rice,<sup><a reveal-id="DSQ023C13" data-open="DSQ023C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSQ023C16" data-open="DSQ023C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSQ023C17" data-open="DSQ023C17" class="link link-ref link-reveal xref-bibr">17</a></sup> the model monocot plant, a systematic genomic analysis at a genome-wide level is lacking. Previously, the presence of a total of 27 GRX genes has been reported in rice.<sup><a reveal-id="DSQ023C5" data-open="DSQ023C5" class="link link-ref link-reveal xref-bibr">5</a></sup> In this study, we have identified 48 members of GRX gene family in rice and divided them into four classes based on the phylogenetic analysis and sequence conservation. The gene structure and putative conserved motifs have been analyzed. The expression studies indicated that GRXs exhibit a variety of gene expression patterns during development and in response to various environmental stimuli. Our analysis provides the framework for future studies to dissect functions of this important gene family.</p>                            <h2 scrollto-destination=7358993 id="7358993" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7358994 id="7358994" class="section-title" data-legacy-id=s2a>2.1. Identification of GRX proteins</h3>
<p>A preliminary search for GRX proteins was performed using keyword ‘GRX’ in the Rice Genome Annotation Project (RGAP) database for rice and TAIR database for <em>Arabidopsis</em>. Another approach used was to search for the proteins containing PFam domain PF00462, which encodes GRX domain. Further, the rice and <em>Arabidopsis</em> proteomes downloaded from RGAP (version 6.1) and TAIR (version 9), respectively, were searched using the hidden Markov model (HMM) profile (build 2.3.2) for GRX domain (PF00462) downloaded from the PFam database. The results obtained from all the above approaches were pooled together and redundancy was removed. Further, all the proteins were searched in the SMART and PFam databases to confirm the presence of GRX domain. The GRX proteins in <em>Populus trichocarpa</em>, <em>Glycine max</em> and <em>Zea mays</em> were identified by BLASTP searches of all the identified rice and <em>Arabidopsis</em> GRX proteins against their annotated proteomes followed by confirmation of the presence of GRX domain in the SMART and PFam databases.</p>                            <h3 scrollto-destination=7358997 id="7358997" class="section-title" data-legacy-id=s2b>2.2. Protein sequence alignment and phylogenetic analysis</h3>
<p>Multiple sequence alignments of amino acid sequences were performed using ClustalX (version 1.83) program and were corrected manually. The unrooted phylogenetic trees were generated by the neighbor-joining (NJ) method using ClustalX and Phylogeny Inference Package (PHYLIP v3.69) programs with default parameters and displayed using NJPlot program. Bootstrap analysis was performed with 1000 replicates to obtain a support value for each branch.</p>                            <h3 scrollto-destination=7358999 id="7358999" class="section-title" data-legacy-id=s2c>2.3. Sequence analysis</h3>
<p>The amino acid sequences of all the predicted GRX proteins were analyzed for subcellular localization by TargetP1.1.<sup><a reveal-id="DSQ023C18" data-open="DSQ023C18" class="link link-ref link-reveal xref-bibr">18</a></sup> The putative conserved motifs were identified using MEME (version 4.1.0) program with default parameters except the minimum number of sites for a motif to find was defined as five.<sup><a reveal-id="DSQ023C19" data-open="DSQ023C19" class="link link-ref link-reveal xref-bibr">19</a></sup></p>                            <h3 scrollto-destination=7359002 id="7359002" class="section-title" data-legacy-id=s2d>2.4. Chromosomal localization and gene duplication</h3>
<p>All the GRX genes were positioned on the rice and <em>Arabidopsis</em> chromosomes using BLASTN. The GRX genes present on duplicated chromosomal segments were identified by segmental genome duplication of rice available at RGAP with the maximum length distance permitted between collinear gene pairs of 500 kb. The presence of <em>Arabidopsis</em> GRX genes on duplicated chromosomal segments were investigated using ‘Paralogons in Arabidopsis’ (<a class="link link-uri" href="http://wolfe.gen.tcd.ie/athal/dup">http://wolfe.gen.tcd.ie/athal/dup</a>). The GRX genes separated by a maximum of five genes were identified as tandemly duplicated genes.</p>                            <h3 scrollto-destination=7359004 id="7359004" class="section-title" data-legacy-id=s2e>2.5. Expression analysis using cDNA/EST and massively parallel signature sequencing data</h3>
<p>To find cDNA and EST sequences corresponding to <em>OsGRX</em> genes, BLASTN search was performed in non-redundant and EST databases at NCBI. The gene expression evidence search page available at RGAP database was also used to find the availability of corresponding FL-cDNA and/or EST evidences. The MPSS (massively parallel signature sequencing) data were searched at Rice MPSS project for 20-base signatures from 22 mRNA libraries representing 18 different tissues/organs in rice for expression evidence analysis.<sup><a reveal-id="DSQ023C20" data-open="DSQ023C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Only the signatures, which uniquely identify an individual gene and show perfect match (100% identity over 100% of the length of the tag), were used for the analysis. The normalized abundance (tags per million, tpm) of these signatures for a given gene in a given library represents the quantitative estimate of the expression of that gene.</p>                            <h3 scrollto-destination=7359007 id="7359007" class="section-title" data-legacy-id=s2f>2.6. Expression analysis using microarray data</h3>
<p>For rice, the microarray data available at Gene Expression Omnibus (GEO) database under the series accession numbers GSE6893 (various stages of development),<sup><a reveal-id="DSQ023C21" data-open="DSQ023C21" class="link link-ref link-reveal xref-bibr">21</a></sup> GSE6901 (abiotic stress),<sup><a reveal-id="DSQ023C21" data-open="DSQ023C21" class="link link-ref link-reveal xref-bibr">21</a></sup> GSE4471 (arsenate stress),<sup><a reveal-id="DSQ023C22" data-open="DSQ023C22" class="link link-ref link-reveal xref-bibr">22</a></sup> GSE5167 (auxin and cytokinin treatment),<sup><a reveal-id="DSQ023C23" data-open="DSQ023C23" class="link link-ref link-reveal xref-bibr">23</a></sup> GSE7256 (virulent infection by <em>Magnaporthe grisea</em>)<sup><a reveal-id="DSQ023C24" data-open="DSQ023C24" class="link link-ref link-reveal xref-bibr">24</a></sup> and GSE10373 (interaction with the parasitic plant <em>Striga hermonthica</em>)<sup><a reveal-id="DSQ023C25" data-open="DSQ023C25" class="link link-ref link-reveal xref-bibr">25</a></sup> were used for expression analysis of <em>OsGRX</em> genes. In addition, microarray analysis was performed for oxidative stress treatments with methyl viologen and hydrogen peroxide, and hormone treatments including ABA, ACC, SA and JA using Affymetrix whole genome rice arrays as described earlier.<sup><a reveal-id="DSQ023C21" data-open="DSQ023C21" class="link link-ref link-reveal xref-bibr">21</a></sup> For <em>Arabidopsis</em>, the microarray data for various stages of developments, hormone treatments and abiotic stress treatments corresponding to those analyzed for rice were downloaded from GEO (series accession numbers GSE5620, GSE5621, GSE5623, GSE5624 and GSE5629–GSE5634) and AtGenExpress (<a class="link link-uri" href="http://www.arabidopsis.org/portals/expression/microarray/ATGenExpress.jsp">http://www.arabidopsis.org/portals/expression/microarray/ATGenExpress.jsp</a>) databases. The Affymetrix CEL files were imported into Genespring GX (version 11) software (Agilent Technologies). The normalization and probe summarization was performed by the Gene Chip Robust Multi Array (GCRMA) method. A stringent statistical analysis consisting of one-way ANOVA over all the samples in a series was performed and the Benjamini–Hoschberg multiple testing correction was applied to the data (<em>P</em>≤ 0.05).</p><p>The IDs of probe sets present on the Affymetrix rice genome array representing the <em>OsGRX</em> and <em>AtGRX</em> genes were identified. The data for only one probe set for each GRX gene were used for expression analysis. This resulted in identification of probe sets for 40 <em>OsGRX</em> and 38 <em>AtGRX</em> genes that were represented on the rice and <em>Arabidopsis</em> Affymetrix arrays, respectively. A fold change of at least 2-fold with a <em>P</em>-value of at least 0.05 was considered to be significant to identify differentially expressed genes. The tab-delimited files for the average log signal values for development data and fold-change values for abiotic stress, biotic stress and hormone treatments were imported into TIGR MultiExperiment Viewer (MeV) to generate heatmaps based on hierarchical clustering.</p>                            <h3 scrollto-destination=7359010 id="7359010" class="section-title" data-legacy-id=s2g>2.7. Real-time PCR analysis</h3>
<p>The real-time PCR analysis was performed using gene-specific primers as described earlier.<sup><a reveal-id="DSQ023C26" data-open="DSQ023C26" class="link link-ref link-reveal xref-bibr">26</a></sup> All the primer sequences used in this study are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. At least two biological replicates of each sample and three technical replicates of each biological replicate were analyzed for real-time PCR analysis. The expression of each gene in different RNA samples was normalized with the expression of the suitable internal control gene, <em>UBQ5</em>,<sup><a reveal-id="DSQ023C26" data-open="DSQ023C26" class="link link-ref link-reveal xref-bibr">26</a></sup> to ensure the equal amount of cDNA used for individual reactions. The relative mRNA levels for each candidate gene in different tissue samples were calculated using the ▵▵<em>C</em><sub>T</sub> method.</p>                            <h3 scrollto-destination=7359012 id="7359012" class="section-title" data-legacy-id=s2h>2.8. Structure model</h3>
<p>To identify the best template structures, all the OsGRX proteins were searched by BLASTP in all the GRX proteins for which a three-dimensional structure is known in Protein Data Bank (PDB). The crystal structures of <em>Populus</em> GRX S12 (PDB ID 3fza) and GRXC1 (PDB ID 2e7p) were used as templates for constructing the structure models of rice CPYC- and CC-type class proteins. Similarly, crystal structures of human GRX5 (PDB ID 2wul) and <em>Escherichia coli</em> GRX4 (PDB ID 2wci) were used as templates for constructing the structure models of rice CGFS-type class proteins. The sequences of template and representative OsGRX proteins were aligned by align2D_Mult program in the MODELLER package.<sup><a reveal-id="DSQ023C27" data-open="DSQ023C27" class="link link-ref link-reveal xref-bibr">27</a></sup> On the basis of the alignment, structures were built using Model_Mult of MODELLER package. The stereochemical properties of all the optimized generated models were assessed using PROCHECK to select the best model for a given protein.<sup><a reveal-id="DSQ023C28" data-open="DSQ023C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Dock (6.2) program was used for docking of GSH to the GRX proteins.</p>                            <h3 scrollto-destination=7359014 id="7359014" class="section-title" data-legacy-id=s2i>2.9. Plant growth</h3>
<p>For salt, desiccation, cold, arsenate, methyl viologen and hydrogen peroxide stress treatments, 7-day-old light-grown rice (<em>Oryza sativa</em> L. ssp. <em>indica</em> var. IR64) seedlings were transferred to a beaker containing 200 mM NaCl solution, dried between folds of tissue paper at 28 ± 1°C, kept at 4 ± 1°C, transferred to a beaker containing 50 µM sodium arsenate solution, transferred to a beaker containing 100 µM methyl viologen solution and transferred to a beaker containing 10 mM hydrogen peroxide solution, respectively, each for 3 h. Likewise, 7-day-old light-grown rice seedlings were transferred to a beaker containing 50 µM solutions of indole-3-acetic acid and benzyl aminopurine and 100 µM solutions of abscisic acid, 1-aminocyclopropane-1-carboxylic acid (ACC), salicylic acid and jasmonic acid for different hormone treatments. The control seedlings were kept in water for 3 h, at 28 ± 1°C.</p>                            <h2 scrollto-destination=7359016 id="7359016" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=7359017 id="7359017" class="section-title" data-legacy-id=s3a>3.1. Identification of GRXs in rice and <em>Arabidopsis</em></h3>
<p>A total of 31, 36 and 27 GRX proteins have been identified in <em>Arabidopsis</em>, <em>Populus</em> and rice, respectively, and divided into three classes.<sup><a reveal-id="DSQ023C3" data-open="DSQ023C3" class="link link-ref link-reveal xref-bibr">3</a>,<a reveal-id="DSQ023C4" data-open="DSQ023C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Further, the presence of 21, 22 and 17 plant-specific CC-type GRXs has been reported in <em>Arabidopsis</em>, <em>Populus</em> and rice, respectively.<sup><a reveal-id="DSQ023C5" data-open="DSQ023C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSQ023C13" data-open="DSQ023C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSQ023C29" data-open="DSQ023C29" class="link link-ref link-reveal xref-bibr">29</a></sup> In addition, 15 non-canonical GRX-like (GRL) proteins have been identified in <em>Arabidopsis</em>.<sup><a reveal-id="DSQ023C30" data-open="DSQ023C30" class="link link-ref link-reveal xref-bibr">30</a></sup> In this study, multiple searches followed by confirming the presence of GRX domain using SMART and PFam, resulted in the identification of 48 loci in the rice genome that encode for putative GRX proteins. We identified an identical number (48) of GRX genes in the <em>Arabidopsis</em> genome as well. Despite the difference in genome sizes between <em>Arabidopsis</em> and rice, the two plant species have identical number of genes encoding GRX proteins. Phylogenetic tree among the rice and <em>Arabidopsis</em> GRX proteins was constructed by the NJ method using ClustalX and PHYLIP programs. The 96 GRXs were grouped into four distinct groups with high bootstrap support using ClustalX program (Fig. <a reveal-id="DSQ023F1" data-open="DSQ023F1" class="link link-reveal link-table xref-fig">1</a>A). An identical grouping of GRX proteins was observed in the phylogenetic tree constructed using PHYLIP program (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). The classes include CC-type (17 in rice and 21 in <em>Arabidopsis</em>), CGFS-type (5 in rice and 5 in <em>Arabidopsis</em>), CPYC- type (7 each in rice and <em>Arabidopsis</em>) and GRL-type (19 in rice and 16 in <em>Arabidopsis</em>) (Fig. <a reveal-id="DSQ023F1" data-open="DSQ023F1" class="link link-reveal link-table xref-fig">1</a>A; Table <a reveal-id="DSQ023TB1" data-open="DSQ023TB1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S2 and S3</a>). The number of CC-type GRXs identified in this study is identical as that reported earlier.<sup><a reveal-id="DSQ023C13" data-open="DSQ023C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The members of CC-type, CPYC-type and CGFS-type classes contain a conserved active-site motif and were designated as <em>OsGRX</em> in rice and <em>AtGRX</em> in <em>Arabidopsis</em>, followed by a number according to their position on rice or <em>Arabidopsis</em> chromosomes from top to bottom (Table <a reveal-id="DSQ023TB1" data-open="DSQ023TB1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S2 and S3</a>). The members included in GRL-type class showed low homology to classical GRXs and do not harbor conserved active-site motif as reported earlier.<sup><a reveal-id="DSQ023C30" data-open="DSQ023C30" class="link link-ref link-reveal xref-bibr">30</a></sup> To distinguish these proteins from classical GRXs, these proteins were designated as <em>OsGRL</em> in rice and <em>AtGRL</em> in <em>Arabidopsis</em> followed by a number according to their position on rice or <em>Arabidopsis</em> chromosomes from top to bottom (Table <a reveal-id="DSQ023TB1" data-open="DSQ023TB1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S2 and S3</a>). The detailed genomic information about all the predicted GRX genes in rice and <em>Arabidopsis</em> is given in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S2 and S3</a>. </p>                        <a id="7359019" scrollto-destination="7359019"></a>
<div content-id="DSQ023TB1" class="table-modal"><div class="table-wrap"><div id="DSQ023TB1" class="table-wrap-title" data-id="DSQ023TB1"><div class="label -label">Table 1.</div><div class="caption"><p>GRX genes predicted in rice</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Class </th><th>Gene name<sup>a</sup> </th><th>Locus<sup>b</sup> </th><th>Active site<sup>c</sup> </th></tr></thead><tbody><tr><td>CC type </td><td><em>OsGRX2</em> </td><td>Os01g09830 </td><td>CYMA </td></tr><tr><td> </td><td><em>OsGRX3</em> </td><td>Os01g13950 </td><td>CCMA </td></tr><tr><td> </td><td><em>OsGRX4</em> </td><td>Os01g27140 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX6</em> </td><td>Os01g47760 </td><td>CCLI </td></tr><tr><td> </td><td><em>OsGRX7</em> </td><td>Os01g70990 </td><td>CFMC </td></tr><tr><td> </td><td><em>OsGRX8</em> </td><td>Os02g30850 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX13</em> </td><td>Os04g32300 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX15</em> </td><td>Os05g05730 </td><td>CGMS </td></tr><tr><td> </td><td><em>OsGRX16</em> </td><td>Os05g10930 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX17</em> </td><td>Os05g48930 </td><td>CCLS </td></tr><tr><td> </td><td><em>OsGRX19</em> </td><td>Os07g05630 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX23</em> </td><td>Os11g43520 </td><td>CGMC </td></tr><tr><td> </td><td><em>OsGRX24</em> </td><td>Os11g43530 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX25</em> </td><td>Os11g43550 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX26</em> </td><td>Os11g43580 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX28</em> </td><td>Os12g35330 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX29</em> </td><td>Os12g35340 </td><td>CPMC </td></tr><tr><td>CGFS type </td><td><em>OsGRX1</em> </td><td>Os01g07950 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX5</em> </td><td>Os01g34620 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX11</em> </td><td>Os03g63420 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX22</em> </td><td>Os10g35720 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX27</em> </td><td>Os12g07650 </td><td>CGFS </td></tr><tr><td>CPYC type </td><td><em>OsGRX9</em> </td><td>Os02g40500 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX10</em> </td><td>Os02g43180 </td><td>CPYS </td></tr><tr><td> </td><td><em>OsGRX12</em> </td><td>Os04g17050 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX14</em> </td><td>Os04g42930 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX18</em> </td><td>Os06g44910 </td><td>CPYC </td></tr><tr><td> </td><td><em>OsGRX20</em> </td><td>Os08g44400 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX21</em> </td><td>Os08g45140 </td><td>CSYC </td></tr><tr><td>GRL type </td><td><em>OsGRL1</em> </td><td>Os01g13480 </td><td> </td></tr><tr><td> </td><td><em>OsGRL2</em> </td><td>Os01g61350 </td><td> </td></tr><tr><td> </td><td><em>OsGRL3</em> </td><td>Os02g01200 </td><td> </td></tr><tr><td> </td><td><em>OsGRL4</em> </td><td>Os02g51370 </td><td> </td></tr><tr><td> </td><td><em>OsGRL5</em> </td><td>Os03g07470 </td><td> </td></tr><tr><td> </td><td><em>OsGRL6</em> </td><td>Os03g24030 </td><td> </td></tr><tr><td> </td><td><em>OsGRL7</em> </td><td>Os03g44650 </td><td> </td></tr><tr><td> </td><td><em>OsGRL8</em> </td><td>Os04g33680 </td><td> </td></tr><tr><td> </td><td><em>OsGRL9</em> </td><td>Os04g54860 </td><td> </td></tr><tr><td> </td><td><em>OsGRL10</em> </td><td>Os05g28530 </td><td> </td></tr><tr><td> </td><td><em>OsGRL11</em> </td><td>Os05g39450 </td><td> </td></tr><tr><td> </td><td><em>OsGRL12</em> </td><td>Os06g12030 </td><td> </td></tr><tr><td> </td><td><em>OsGRL13</em> </td><td>Os06g12190 </td><td> </td></tr><tr><td> </td><td><em>OsGRL14</em> </td><td>Os07g06600 </td><td> </td></tr><tr><td> </td><td><em>OsGRL15</em> </td><td>Os07g46410 </td><td> </td></tr><tr><td> </td><td><em>OsGRL16</em> </td><td>Os07g46570 </td><td> </td></tr><tr><td> </td><td><em>OsGRL17</em> </td><td>Os08g07450 </td><td> </td></tr><tr><td> </td><td><em>OsGRL18</em> </td><td>Os08g44070 </td><td> </td></tr><tr><td> </td><td><em>OsGRL19</em> </td><td>Os10g34170 </td><td> </td></tr></tbody></table></div><div content-id="DSQ023TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Class </th><th>Gene name<sup>a</sup> </th><th>Locus<sup>b</sup> </th><th>Active site<sup>c</sup> </th></tr></thead><tbody><tr><td>CC type </td><td><em>OsGRX2</em> </td><td>Os01g09830 </td><td>CYMA </td></tr><tr><td> </td><td><em>OsGRX3</em> </td><td>Os01g13950 </td><td>CCMA </td></tr><tr><td> </td><td><em>OsGRX4</em> </td><td>Os01g27140 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX6</em> </td><td>Os01g47760 </td><td>CCLI </td></tr><tr><td> </td><td><em>OsGRX7</em> </td><td>Os01g70990 </td><td>CFMC </td></tr><tr><td> </td><td><em>OsGRX8</em> </td><td>Os02g30850 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX13</em> </td><td>Os04g32300 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX15</em> </td><td>Os05g05730 </td><td>CGMS </td></tr><tr><td> </td><td><em>OsGRX16</em> </td><td>Os05g10930 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX17</em> </td><td>Os05g48930 </td><td>CCLS </td></tr><tr><td> </td><td><em>OsGRX19</em> </td><td>Os07g05630 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX23</em> </td><td>Os11g43520 </td><td>CGMC </td></tr><tr><td> </td><td><em>OsGRX24</em> </td><td>Os11g43530 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX25</em> </td><td>Os11g43550 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX26</em> </td><td>Os11g43580 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX28</em> </td><td>Os12g35330 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX29</em> </td><td>Os12g35340 </td><td>CPMC </td></tr><tr><td>CGFS type </td><td><em>OsGRX1</em> </td><td>Os01g07950 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX5</em> </td><td>Os01g34620 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX11</em> </td><td>Os03g63420 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX22</em> </td><td>Os10g35720 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX27</em> </td><td>Os12g07650 </td><td>CGFS </td></tr><tr><td>CPYC type </td><td><em>OsGRX9</em> </td><td>Os02g40500 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX10</em> </td><td>Os02g43180 </td><td>CPYS </td></tr><tr><td> </td><td><em>OsGRX12</em> </td><td>Os04g17050 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX14</em> </td><td>Os04g42930 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX18</em> </td><td>Os06g44910 </td><td>CPYC </td></tr><tr><td> </td><td><em>OsGRX20</em> </td><td>Os08g44400 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX21</em> </td><td>Os08g45140 </td><td>CSYC </td></tr><tr><td>GRL type </td><td><em>OsGRL1</em> </td><td>Os01g13480 </td><td> </td></tr><tr><td> </td><td><em>OsGRL2</em> </td><td>Os01g61350 </td><td> </td></tr><tr><td> </td><td><em>OsGRL3</em> </td><td>Os02g01200 </td><td> </td></tr><tr><td> </td><td><em>OsGRL4</em> </td><td>Os02g51370 </td><td> </td></tr><tr><td> </td><td><em>OsGRL5</em> </td><td>Os03g07470 </td><td> </td></tr><tr><td> </td><td><em>OsGRL6</em> </td><td>Os03g24030 </td><td> </td></tr><tr><td> </td><td><em>OsGRL7</em> </td><td>Os03g44650 </td><td> </td></tr><tr><td> </td><td><em>OsGRL8</em> </td><td>Os04g33680 </td><td> </td></tr><tr><td> </td><td><em>OsGRL9</em> </td><td>Os04g54860 </td><td> </td></tr><tr><td> </td><td><em>OsGRL10</em> </td><td>Os05g28530 </td><td> </td></tr><tr><td> </td><td><em>OsGRL11</em> </td><td>Os05g39450 </td><td> </td></tr><tr><td> </td><td><em>OsGRL12</em> </td><td>Os06g12030 </td><td> </td></tr><tr><td> </td><td><em>OsGRL13</em> </td><td>Os06g12190 </td><td> </td></tr><tr><td> </td><td><em>OsGRL14</em> </td><td>Os07g06600 </td><td> </td></tr><tr><td> </td><td><em>OsGRL15</em> </td><td>Os07g46410 </td><td> </td></tr><tr><td> </td><td><em>OsGRL16</em> </td><td>Os07g46570 </td><td> </td></tr><tr><td> </td><td><em>OsGRL17</em> </td><td>Os08g07450 </td><td> </td></tr><tr><td> </td><td><em>OsGRL18</em> </td><td>Os08g44070 </td><td> </td></tr><tr><td> </td><td><em>OsGRL19</em> </td><td>Os10g34170 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Systematic name given to GRX genes identified in rice.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Locus identified from RGAP.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Conserved active-site motif in the predicted GRX protein sequences.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7359019" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ023TB1" class="table-wrap-title" data-id="DSQ023TB1"><div class="label -label">Table 1.</div><div class="caption"><p>GRX genes predicted in rice</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Class </th><th>Gene name<sup>a</sup> </th><th>Locus<sup>b</sup> </th><th>Active site<sup>c</sup> </th></tr></thead><tbody><tr><td>CC type </td><td><em>OsGRX2</em> </td><td>Os01g09830 </td><td>CYMA </td></tr><tr><td> </td><td><em>OsGRX3</em> </td><td>Os01g13950 </td><td>CCMA </td></tr><tr><td> </td><td><em>OsGRX4</em> </td><td>Os01g27140 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX6</em> </td><td>Os01g47760 </td><td>CCLI </td></tr><tr><td> </td><td><em>OsGRX7</em> </td><td>Os01g70990 </td><td>CFMC </td></tr><tr><td> </td><td><em>OsGRX8</em> </td><td>Os02g30850 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX13</em> </td><td>Os04g32300 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX15</em> </td><td>Os05g05730 </td><td>CGMS </td></tr><tr><td> </td><td><em>OsGRX16</em> </td><td>Os05g10930 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX17</em> </td><td>Os05g48930 </td><td>CCLS </td></tr><tr><td> </td><td><em>OsGRX19</em> </td><td>Os07g05630 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX23</em> </td><td>Os11g43520 </td><td>CGMC </td></tr><tr><td> </td><td><em>OsGRX24</em> </td><td>Os11g43530 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX25</em> </td><td>Os11g43550 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX26</em> </td><td>Os11g43580 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX28</em> </td><td>Os12g35330 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX29</em> </td><td>Os12g35340 </td><td>CPMC </td></tr><tr><td>CGFS type </td><td><em>OsGRX1</em> </td><td>Os01g07950 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX5</em> </td><td>Os01g34620 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX11</em> </td><td>Os03g63420 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX22</em> </td><td>Os10g35720 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX27</em> </td><td>Os12g07650 </td><td>CGFS </td></tr><tr><td>CPYC type </td><td><em>OsGRX9</em> </td><td>Os02g40500 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX10</em> </td><td>Os02g43180 </td><td>CPYS </td></tr><tr><td> </td><td><em>OsGRX12</em> </td><td>Os04g17050 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX14</em> </td><td>Os04g42930 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX18</em> </td><td>Os06g44910 </td><td>CPYC </td></tr><tr><td> </td><td><em>OsGRX20</em> </td><td>Os08g44400 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX21</em> </td><td>Os08g45140 </td><td>CSYC </td></tr><tr><td>GRL type </td><td><em>OsGRL1</em> </td><td>Os01g13480 </td><td> </td></tr><tr><td> </td><td><em>OsGRL2</em> </td><td>Os01g61350 </td><td> </td></tr><tr><td> </td><td><em>OsGRL3</em> </td><td>Os02g01200 </td><td> </td></tr><tr><td> </td><td><em>OsGRL4</em> </td><td>Os02g51370 </td><td> </td></tr><tr><td> </td><td><em>OsGRL5</em> </td><td>Os03g07470 </td><td> </td></tr><tr><td> </td><td><em>OsGRL6</em> </td><td>Os03g24030 </td><td> </td></tr><tr><td> </td><td><em>OsGRL7</em> </td><td>Os03g44650 </td><td> </td></tr><tr><td> </td><td><em>OsGRL8</em> </td><td>Os04g33680 </td><td> </td></tr><tr><td> </td><td><em>OsGRL9</em> </td><td>Os04g54860 </td><td> </td></tr><tr><td> </td><td><em>OsGRL10</em> </td><td>Os05g28530 </td><td> </td></tr><tr><td> </td><td><em>OsGRL11</em> </td><td>Os05g39450 </td><td> </td></tr><tr><td> </td><td><em>OsGRL12</em> </td><td>Os06g12030 </td><td> </td></tr><tr><td> </td><td><em>OsGRL13</em> </td><td>Os06g12190 </td><td> </td></tr><tr><td> </td><td><em>OsGRL14</em> </td><td>Os07g06600 </td><td> </td></tr><tr><td> </td><td><em>OsGRL15</em> </td><td>Os07g46410 </td><td> </td></tr><tr><td> </td><td><em>OsGRL16</em> </td><td>Os07g46570 </td><td> </td></tr><tr><td> </td><td><em>OsGRL17</em> </td><td>Os08g07450 </td><td> </td></tr><tr><td> </td><td><em>OsGRL18</em> </td><td>Os08g44070 </td><td> </td></tr><tr><td> </td><td><em>OsGRL19</em> </td><td>Os10g34170 </td><td> </td></tr></tbody></table></div><div content-id="DSQ023TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Class </th><th>Gene name<sup>a</sup> </th><th>Locus<sup>b</sup> </th><th>Active site<sup>c</sup> </th></tr></thead><tbody><tr><td>CC type </td><td><em>OsGRX2</em> </td><td>Os01g09830 </td><td>CYMA </td></tr><tr><td> </td><td><em>OsGRX3</em> </td><td>Os01g13950 </td><td>CCMA </td></tr><tr><td> </td><td><em>OsGRX4</em> </td><td>Os01g27140 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX6</em> </td><td>Os01g47760 </td><td>CCLI </td></tr><tr><td> </td><td><em>OsGRX7</em> </td><td>Os01g70990 </td><td>CFMC </td></tr><tr><td> </td><td><em>OsGRX8</em> </td><td>Os02g30850 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX13</em> </td><td>Os04g32300 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX15</em> </td><td>Os05g05730 </td><td>CGMS </td></tr><tr><td> </td><td><em>OsGRX16</em> </td><td>Os05g10930 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX17</em> </td><td>Os05g48930 </td><td>CCLS </td></tr><tr><td> </td><td><em>OsGRX19</em> </td><td>Os07g05630 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX23</em> </td><td>Os11g43520 </td><td>CGMC </td></tr><tr><td> </td><td><em>OsGRX24</em> </td><td>Os11g43530 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX25</em> </td><td>Os11g43550 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX26</em> </td><td>Os11g43580 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX28</em> </td><td>Os12g35330 </td><td>CCMC </td></tr><tr><td> </td><td><em>OsGRX29</em> </td><td>Os12g35340 </td><td>CPMC </td></tr><tr><td>CGFS type </td><td><em>OsGRX1</em> </td><td>Os01g07950 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX5</em> </td><td>Os01g34620 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX11</em> </td><td>Os03g63420 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX22</em> </td><td>Os10g35720 </td><td>CGFS </td></tr><tr><td> </td><td><em>OsGRX27</em> </td><td>Os12g07650 </td><td>CGFS </td></tr><tr><td>CPYC type </td><td><em>OsGRX9</em> </td><td>Os02g40500 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX10</em> </td><td>Os02g43180 </td><td>CPYS </td></tr><tr><td> </td><td><em>OsGRX12</em> </td><td>Os04g17050 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX14</em> </td><td>Os04g42930 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX18</em> </td><td>Os06g44910 </td><td>CPYC </td></tr><tr><td> </td><td><em>OsGRX20</em> </td><td>Os08g44400 </td><td>CPFC </td></tr><tr><td> </td><td><em>OsGRX21</em> </td><td>Os08g45140 </td><td>CSYC </td></tr><tr><td>GRL type </td><td><em>OsGRL1</em> </td><td>Os01g13480 </td><td> </td></tr><tr><td> </td><td><em>OsGRL2</em> </td><td>Os01g61350 </td><td> </td></tr><tr><td> </td><td><em>OsGRL3</em> </td><td>Os02g01200 </td><td> </td></tr><tr><td> </td><td><em>OsGRL4</em> </td><td>Os02g51370 </td><td> </td></tr><tr><td> </td><td><em>OsGRL5</em> </td><td>Os03g07470 </td><td> </td></tr><tr><td> </td><td><em>OsGRL6</em> </td><td>Os03g24030 </td><td> </td></tr><tr><td> </td><td><em>OsGRL7</em> </td><td>Os03g44650 </td><td> </td></tr><tr><td> </td><td><em>OsGRL8</em> </td><td>Os04g33680 </td><td> </td></tr><tr><td> </td><td><em>OsGRL9</em> </td><td>Os04g54860 </td><td> </td></tr><tr><td> </td><td><em>OsGRL10</em> </td><td>Os05g28530 </td><td> </td></tr><tr><td> </td><td><em>OsGRL11</em> </td><td>Os05g39450 </td><td> </td></tr><tr><td> </td><td><em>OsGRL12</em> </td><td>Os06g12030 </td><td> </td></tr><tr><td> </td><td><em>OsGRL13</em> </td><td>Os06g12190 </td><td> </td></tr><tr><td> </td><td><em>OsGRL14</em> </td><td>Os07g06600 </td><td> </td></tr><tr><td> </td><td><em>OsGRL15</em> </td><td>Os07g46410 </td><td> </td></tr><tr><td> </td><td><em>OsGRL16</em> </td><td>Os07g46570 </td><td> </td></tr><tr><td> </td><td><em>OsGRL17</em> </td><td>Os08g07450 </td><td> </td></tr><tr><td> </td><td><em>OsGRL18</em> </td><td>Os08g44070 </td><td> </td></tr><tr><td> </td><td><em>OsGRL19</em> </td><td>Os10g34170 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Systematic name given to GRX genes identified in rice.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>Locus identified from RGAP.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>Conserved active-site motif in the predicted GRX protein sequences.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7359019" target="_blank">View Large</a></div></div>                        <a id="7359020" scrollto-destination="7359020"></a>
<div id="DSQ023F1" data-id="DSQ023F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/m_dsq02301.jpeg?Expires=2147483647&Signature=3jJ213TD0Sl4~4LtL9fsoy-95pTdefmpSQZQLD-Bnsb0FKMzFBuI712SJSS-qHoBq918~C9cBIcc3fQJFfTh6knoE9-4nB--1No~HNTm2b8dX5R4dmJDNyCDC2RfXWUdp9tIkG~UBMToCNGQ-K0SVEl9EA1-U2gRnTnG2-Gl5V4YAOBBfOIM6eROmPDh1oJL0YnexH5AxTaP42K~Yz7l1K1h8hC-ga-rwRtgiXkZyNfbp87axQ~YDvyusieWHaeqUD2Y5GO83ivRlU-swDRAJWwPUYyD5T9V~cML0pT6GwLnjQpzTymfs7t9Zd-QdmPKy-nq9f-WvTcEvrg87wGpgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic relationship among rice and Arabidopsis GRXs, their classification and gene structure. (A) The unrooted tree was constructed using ClustalX program based on the multiple sequence alignment of the GRX domain sequences by the NJ method. The number at the nodes represent the bootstrap values (&gt;50%) from 1000 replicates. Different types of GRXs have been indicated. Rice- and Arabidopsis-specific clades have been indicated in different grey-shaded boxes. (B) The gene structure (exon–intron organization) of the coding sequence of GRXs is shown. Exons are represented by black boxes and introns as lines." path-from-xml="dsq02301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7359020/dsq02301.jpeg" path-from-xml="dsq02301.jpeg" target="_blank">View large</a><a section="7359020" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq02301.jpeg?Expires=2147483647&Signature=vhStDlgV7rakh8Prp5vuYn5phLQP0UbmcTd8~xqjVzN-eJJ3o4olNJ~LE1jcG5c3lyB8p9vZEiN7GTax7FXG31FCMJKNeS7vgHmq97XnjGsOr5CJ1I4SgBrTKVlsP2s6Y-phQQeqMB79i7oL6-DZURS61hqvTT-F-XtQiDwQr-cKZag9U-mRPGnC~85odVO36EU1OKg9c3Ztz7u-9ZkfTG2YV588PVs8hbBB1OHUgdcG4koPiqhYDiTR~LLVR8090ypYOFrF3w-LgMRdyKKRSLZr7Vi1WjpYICV-jVXUhsaITk1o1iq8rg2Y4udGuIxrI6XuUWbTN5fS439RycdIqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7359020&ar=484440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationship among rice and <em>Arabidopsis</em> GRXs, their classification and gene structure. (A) The unrooted tree was constructed using ClustalX program based on the multiple sequence alignment of the GRX domain sequences by the NJ method. The number at the nodes represent the bootstrap values (&gt;50%) from 1000 replicates. Different types of GRXs have been indicated. Rice- and <em>Arabidopsis</em>-specific clades have been indicated in different grey-shaded boxes. (B) The gene structure (exon–intron organization) of the coding sequence of GRXs is shown. Exons are represented by black boxes and introns as lines.</p></div></div><div content-id="DSQ023F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/m_dsq02301.jpeg?Expires=2147483647&Signature=3jJ213TD0Sl4~4LtL9fsoy-95pTdefmpSQZQLD-Bnsb0FKMzFBuI712SJSS-qHoBq918~C9cBIcc3fQJFfTh6knoE9-4nB--1No~HNTm2b8dX5R4dmJDNyCDC2RfXWUdp9tIkG~UBMToCNGQ-K0SVEl9EA1-U2gRnTnG2-Gl5V4YAOBBfOIM6eROmPDh1oJL0YnexH5AxTaP42K~Yz7l1K1h8hC-ga-rwRtgiXkZyNfbp87axQ~YDvyusieWHaeqUD2Y5GO83ivRlU-swDRAJWwPUYyD5T9V~cML0pT6GwLnjQpzTymfs7t9Zd-QdmPKy-nq9f-WvTcEvrg87wGpgw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phylogenetic relationship among rice and Arabidopsis GRXs, their classification and gene structure. (A) The unrooted tree was constructed using ClustalX program based on the multiple sequence alignment of the GRX domain sequences by the NJ method. The number at the nodes represent the bootstrap values (&gt;50%) from 1000 replicates. Different types of GRXs have been indicated. Rice- and Arabidopsis-specific clades have been indicated in different grey-shaded boxes. (B) The gene structure (exon–intron organization) of the coding sequence of GRXs is shown. Exons are represented by black boxes and introns as lines." path-from-xml="dsq02301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7359020/dsq02301.jpeg" path-from-xml="dsq02301.jpeg" target="_blank">View large</a><a section="7359020" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq02301.jpeg?Expires=2147483647&Signature=vhStDlgV7rakh8Prp5vuYn5phLQP0UbmcTd8~xqjVzN-eJJ3o4olNJ~LE1jcG5c3lyB8p9vZEiN7GTax7FXG31FCMJKNeS7vgHmq97XnjGsOr5CJ1I4SgBrTKVlsP2s6Y-phQQeqMB79i7oL6-DZURS61hqvTT-F-XtQiDwQr-cKZag9U-mRPGnC~85odVO36EU1OKg9c3Ztz7u-9ZkfTG2YV588PVs8hbBB1OHUgdcG4koPiqhYDiTR~LLVR8090ypYOFrF3w-LgMRdyKKRSLZr7Vi1WjpYICV-jVXUhsaITk1o1iq8rg2Y4udGuIxrI6XuUWbTN5fS439RycdIqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7359020&ar=484440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phylogenetic relationship among rice and <em>Arabidopsis</em> GRXs, their classification and gene structure. (A) The unrooted tree was constructed using ClustalX program based on the multiple sequence alignment of the GRX domain sequences by the NJ method. The number at the nodes represent the bootstrap values (&gt;50%) from 1000 replicates. Different types of GRXs have been indicated. Rice- and <em>Arabidopsis</em>-specific clades have been indicated in different grey-shaded boxes. (B) The gene structure (exon–intron organization) of the coding sequence of GRXs is shown. Exons are represented by black boxes and introns as lines.</p></div></div><p>A comparison of the coding sequence with the corresponding genomic DNA sequences showed that quite a large number of GRX genes (69% in rice and 71% in <em>Arabidopsis</em>) are intronless (Fig. <a reveal-id="DSQ023F1" data-open="DSQ023F1" class="link link-reveal link-table xref-fig">1</a>B), which is very large number as compared at the whole genome level (20%).<sup><a reveal-id="DSQ023C31" data-open="DSQ023C31" class="link link-ref link-reveal xref-bibr">31</a></sup> Striking gene structure conservation was found within the members of each class. Notably, all but one member of CC-type class and most of members of GRL-type class GRX genes are intronless. Such intronless class/family of genes can evolve rapidly by gene duplication or reverse transcription followed by integration in the genome.<sup><a reveal-id="DSQ023C31  DSQ023C34" data-open="DSQ023C31  DSQ023C34" class="link link-ref link-reveal xref-bibr">31–34</a></sup> This might explain the larger number of members of these classes of proteins, which may perform diverse plant-specific functions. There are only few GRX genes found in other organisms. The large number of GRX genes present in rice and <em>Arabidopsis</em> genome highlights the importance of these genes in plants. Although CGFS-type and CPYC-type class GRX proteins represent ancestral genes from prokaryotes to eukaryotes, the plant-specific CC-type and GRL-type class GRX proteins are more in number both in rice and <em>Arabidopsis</em>, further indicating their importance. Earlier, a comparative analysis in various plant species, including <em>Physcomitrella patens</em>, <em>Pinus taeda</em>, <em>O. sativa</em>, <em>P. trichocarpa</em> and <em>Arabidopsis thaliana</em> revealed that the number of CC-type class GRXs has expanded during plant evolution.<sup><a reveal-id="DSQ023C29" data-open="DSQ023C29" class="link link-ref link-reveal xref-bibr">29</a></sup> On the contrary, the number of CPYC- and CGFS-type class members remained constant. For example, only two CC-type GRXs have been found in <em>Physcomitrella</em>,<sup><a reveal-id="DSQ023C29" data-open="DSQ023C29" class="link link-ref link-reveal xref-bibr">29</a></sup> whereas 17 in rice and 21 in <em>Arabidopsis</em>. On the basis of this observation, a possible function of CC-type class GRXs in contributing to the evolution of land plants that forms organs of higher complexity has been proposed.<sup><a reveal-id="DSQ023C29" data-open="DSQ023C29" class="link link-ref link-reveal xref-bibr">29</a></sup></p>                            <h3 scrollto-destination=7359022 id="7359022" class="section-title" data-legacy-id=s3b>3.2. Genomic organization and expansion of GRX gene family</h3>
<p>The physical locations of all the 48 <em>OsGRX</em>s were assigned to the 12 rice chromosomes (Fig. <a reveal-id="DSQ023F2" data-open="DSQ023F2" class="link link-reveal link-table xref-fig">2</a>). The distribution of GRX genes among the chromosomes appears to be uneven; chromosome 9 harbors no GRX genes, whereas chromosome 1 harbors nine GRX genes. Other chromosomes have 2–5 GRX genes localized on them. Only two clusters of tandemly arranged <em>GRX</em> genes were found in rice; one of four GRXs on chromosome 11 and other of two GRXs on chromosome 12. On the other hand, we found 10 pairs (20 members) of GRX genes were located on duplicated chromosomal segments in rice. The distribution of GRX genes relative to duplicated chromosomal segments in rice is illustrated in Fig. <a reveal-id="DSQ023F2" data-open="DSQ023F2" class="link link-reveal link-table xref-fig">2</a>. In <em>Arabidopsis</em>, the 48 GRXs were localized on all the five chromosomes ranging from 7 to 13 in number (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Three clusters were observed, one of two GRXs on chromosome 2; second of three GRXs on chromosome 3 and third of five GRXs on chromosome 4. Seven pairs (14 members) of GRX genes were located on duplicated chromosomal segments in <em>Arabidopsis</em>. Further analysis revealed that a similar set of genes were found to be present in the proximity of GRX gene pairs predicted to be located on duplicated chromosomal segments in rice and <em>Arabidopsis</em>. The details of the genes present in the proximity of the GRX gene pairs located on chromosomal segments in rice and <em>Arabidopsis</em>, including their gene identifiers, description, chromosomal location and <em>E</em>-value are given in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S4 and S5</a>, respectively. </p>                        <a id="7359024" scrollto-destination="7359024"></a>
<div id="DSQ023F2" data-id="DSQ023F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/m_dsq02302.jpeg?Expires=2147483647&Signature=gbW9XMTiHJws3TQ4mGpYyj-05AO8hpkzRri2dPH3torJZvLmH8t1uE5Q6prhS0SUcch8EoKWJmiLEcS9t5GB5FdaqdvFZDLb8TAZOk8zG0sCRrB9UPZZpAVrgBCg4VoxX6etFa498TrL3F5ctiX1aFytS~CjbJ9uG5blejnkF7UVYaWxlVgeOC-gOwahJi9bHC1QquGuA1MYPDxyfRzOYYITLIsX5wDXLSvg6bATFIZ0w9xGFKT21-SN6S5RSjhk8HPURPpM87~9067XDOP41N5144eXuEnBJojjptXsEmhT2RRUL52Fs2Iemgw9leYeoWi0-ILaOfVAD1jNAH5NBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chromosomal distribution and segmental duplication events of OsGRX genes. The chromosome number is indicated on the top of each bar. The GRX genes localized on duplicated chromosomal segments are connected by lines. The clusters of tandemly arranged GRX genes are indicated in boxes." path-from-xml="dsq02302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7359024/dsq02302.jpeg" path-from-xml="dsq02302.jpeg" target="_blank">View large</a><a section="7359024" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq02302.jpeg?Expires=2147483647&Signature=Wyu9HpbaLVnCAhTwMRYYsOHNHC5yip4-EFA6FFz-0XrtWYVZnABkpcfgXri8ZDZeCkhSjs1cDmCMoJP3wxuKqFor0y3griMmko4uYf7skRy48eJB-sWVIadS36pxWPqw38T8ZqFZOLLXpTo1Qq5WTuocx8cq-g3Dgz8KkT2-i5Fbc9i7wzUkXebtkBh5JaW1bHpbjs5ZYl21Pl8d10zKA~N3wbLBV99LmWKFzAn6SImKGONtEbAQVvlCRF~xzIKeIObZdCUt1yXB6TcWXCCAqlGBrEH3~Uih-VpibWrpR~QONJD2jvk9hSasVIGvKIrHXcO6lzE85JRmZgxLrCdQXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7359024&ar=484440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chromosomal distribution and segmental duplication events of <em>OsGRX</em> genes. The chromosome number is indicated on the top of each bar. The GRX genes localized on duplicated chromosomal segments are connected by lines. The clusters of tandemly arranged GRX genes are indicated in boxes.</p></div></div><div content-id="DSQ023F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/m_dsq02302.jpeg?Expires=2147483647&Signature=gbW9XMTiHJws3TQ4mGpYyj-05AO8hpkzRri2dPH3torJZvLmH8t1uE5Q6prhS0SUcch8EoKWJmiLEcS9t5GB5FdaqdvFZDLb8TAZOk8zG0sCRrB9UPZZpAVrgBCg4VoxX6etFa498TrL3F5ctiX1aFytS~CjbJ9uG5blejnkF7UVYaWxlVgeOC-gOwahJi9bHC1QquGuA1MYPDxyfRzOYYITLIsX5wDXLSvg6bATFIZ0w9xGFKT21-SN6S5RSjhk8HPURPpM87~9067XDOP41N5144eXuEnBJojjptXsEmhT2RRUL52Fs2Iemgw9leYeoWi0-ILaOfVAD1jNAH5NBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Chromosomal distribution and segmental duplication events of OsGRX genes. The chromosome number is indicated on the top of each bar. The GRX genes localized on duplicated chromosomal segments are connected by lines. The clusters of tandemly arranged GRX genes are indicated in boxes." path-from-xml="dsq02302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7359024/dsq02302.jpeg" path-from-xml="dsq02302.jpeg" target="_blank">View large</a><a section="7359024" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq02302.jpeg?Expires=2147483647&Signature=Wyu9HpbaLVnCAhTwMRYYsOHNHC5yip4-EFA6FFz-0XrtWYVZnABkpcfgXri8ZDZeCkhSjs1cDmCMoJP3wxuKqFor0y3griMmko4uYf7skRy48eJB-sWVIadS36pxWPqw38T8ZqFZOLLXpTo1Qq5WTuocx8cq-g3Dgz8KkT2-i5Fbc9i7wzUkXebtkBh5JaW1bHpbjs5ZYl21Pl8d10zKA~N3wbLBV99LmWKFzAn6SImKGONtEbAQVvlCRF~xzIKeIObZdCUt1yXB6TcWXCCAqlGBrEH3~Uih-VpibWrpR~QONJD2jvk9hSasVIGvKIrHXcO6lzE85JRmZgxLrCdQXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7359024&ar=484440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Chromosomal distribution and segmental duplication events of <em>OsGRX</em> genes. The chromosome number is indicated on the top of each bar. The GRX genes localized on duplicated chromosomal segments are connected by lines. The clusters of tandemly arranged GRX genes are indicated in boxes.</p></div></div><p>Taken together, it appears that the chromosomal segment duplications have mainly contributed to the expansion giving rise to 42 and 29% of GRX genes in rice and <em>Arabidopsis</em>, respectively. However, tandem or local gene duplication occurred less frequently in <em>Arabidopsis</em> and rice. Further, it is also clear that chromosomal segment duplication has contributed more for expansion of GRX genes in rice when compared with <em>Arabidopsis</em>. In addition, in the phylogenetic tree constructed from rice and <em>Arabidopsis</em> GRX domain sequences, we observed some species-specific clades within a class (Fig. <a reveal-id="DSQ023F1" data-open="DSQ023F1" class="link link-reveal link-table xref-fig">1</a>A, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). This indicates that the expansion of different classes has occurred independently in a species-specific manner in rice and <em>Arabidopsis</em>. This observation was further validated by constructing a phylogenetic tree among the GRX domain sequences from a total of 259 GRX proteins identified from various plant species, including rice (48), <em>Arabidopsis</em> (48), <em>Populus</em> (45), <em>G. max</em> (82) and <em>Z. mays</em> (36) using ClustalX (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>) and PHYLIP (data not shown) programs. We found both lineage-specific (monocot- and dicot-specific) and species-specific clades in these phylogenetic trees as well, further confirming the expansion of GRX gene family in the monocots and dicots independently. Recently, the expansion of CC-type class GRX proteins in angiosperms has also been proposed mainly through paleopolyploidy duplication events after the monocot–dicot split.<sup><a reveal-id="DSQ023C35" data-open="DSQ023C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p>                            <h3 scrollto-destination=7359026 id="7359026" class="section-title" data-legacy-id=s3c>3.3. Sequence analysis</h3>
<p>The length of GRX proteins varied greatly ranging from minimum of 103 (OsGRX4 and OsGRX28) to 711 amino acids (OsGRL12) in rice. The pairwise comparison of GRX protein sequences showed considerable sequence diversity. However, relatively more identity was found among the members of the same class. The multiple sequence alignments of full-length GRX protein sequences showed that the core region, which represents the GRX domain, is most conserved. All the members of CGFS-type, CPYC-type and CC-type class GRX proteins harbor the conserved active site (Table <a reveal-id="DSQ023TB1" data-open="DSQ023TB1" class="link link-reveal link-table xref-fig">1</a>). The cysteine at first position of the active site was invariably conserved in all the proteins; however, variations were found at other positions. All the five members of CGFS-type class contain perfectly conserved CGFS as an active site. However, only one member harbors characteristic CPYC motif in CPYC-type class, other members have CPFC (four), CPYS (one) and CSYC (one) as active site. In CC-type class GRX proteins, nine members harbor CCMC as active site. The characteristic two cysteine residues were conserved in all CC-type GRXs except for five. In fact, the second cysteine residue has not been found crucial for the activity of CC-type GRXs in <em>Arabidopsis</em>.<sup><a reveal-id="DSQ023C11" data-open="DSQ023C11" class="link link-ref link-reveal xref-bibr">11</a></sup> In addition, most of the members of CC-type class rice GRX proteins harbor four C-terminal residues, A(L/I)W(L/V), which were found to be essential for their <em>in planta</em> function and physical interaction with target proteins.<sup><a reveal-id="DSQ023C36" data-open="DSQ023C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Most of the members of GRL-type class proteins harbor eight conserved cysteines, arranged in two CxxCx7CxxC clusters in rice at the C-terminal end. Similar pattern of conserved cysteines has been reported in <em>Arabidopsis</em> GRL proteins as well.<sup><a reveal-id="DSQ023C30" data-open="DSQ023C30" class="link link-ref link-reveal xref-bibr">30</a></sup></p><p>To identify additional putative conserved motifs in GRX proteins, all the 96 GRX proteins from rice and <em>Arabidopsis</em> were searched using MEME program. Ten highly significant (&lt;e−50) putative motifs of at least six amino acids in length conserved in more than 10 GRX proteins were identified (Table <a reveal-id="DSQ023TB2" data-open="DSQ023TB2" class="link link-reveal link-table xref-fig">2</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). Notably, all the putative motifs identified are conserved in rice and <em>Arabidopsis</em>. Some of the motifs were present in the members of more than one class. However, others were specific to a particular class. For example, motif 1, which represents a part of GRX domain, was present in most of the members of all classes. Motif 2 was present in CPYC-type and CC-type GRX proteins. Motifs 3, 5 and 7 were present only in GRL type of GRX proteins. Motif 4 was present in CC-type and CPYC-type proteins only. Motifs 9 and 10 were present in CC-type proteins only, whereas motifs 6 and 8 were present in CGFS type only. Overall, the organization of putative conserved motifs coincided with the grouping of GRXs in various classes and may be responsible for functional specificity among GRX proteins of various class types. </p>                        <a id="7359033" scrollto-destination="7359033"></a>
<div content-id="DSQ023TB2" class="table-modal"><div class="table-wrap"><div id="DSQ023TB2" class="table-wrap-title" data-id="DSQ023TB2"><div class="label -label">Table 2.</div><div class="caption"><p>List of putative conserved motifs in rice and <em>Arabidopsis</em> GRX proteins predicted by MEME</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Motif no. </th><th>Motif sequence </th><th>Number of sites in rice </th><th>Number of sites in <em>Arabidopsis</em> </th><th>Length </th><th><em>E</em>-value </th><th>Class of GRX proteins </th></tr></thead><tbody><tr><td>1 </td><td>PQVFIGGKHIGGCDQVMSMHENGELVPMLK </td><td>45 </td><td>48 </td><td>30 </td><td>3.3e-1153 </td><td>CC-type, CGFS-type, CPYC-typeand GRL-type </td></tr><tr><td>2 </td><td>VIFSKSYCCMCHTVKTLFQDL </td><td>22 </td><td>27 </td><td>21 </td><td>2.2e-474 </td><td>CC-typeand CPYC-type </td></tr><tr><td>3 </td><td>RVVLYFTSLRGIRKTYEDCCAVRAILRGHRVWVDERDVSMH </td><td>16 </td><td>13 </td><td>41 </td><td>4.9e-636 </td><td>GRL-type </td></tr><tr><td>4 </td><td>GVNPTIHELDQDPDGWEIQRALAQWGCQP </td><td>15 </td><td>21 </td><td>29 </td><td>1.8e-372 </td><td>CC-typeand CPYC-type </td></tr><tr><td>5 </td><td>GWRRCPGCNENRFVPCPNC </td><td>28 </td><td>25 </td><td>19 </td><td>4.4e-362 </td><td>GRL-type </td></tr><tr><td>6 </td><td>RLEQLVNSHPVMLFMKGTPDEPQCGFSQKVVQILKQYNVPFGSFDILTDE </td><td>7 </td><td>6 </td><td>50 </td><td>1.6e-222 </td><td>CGFS-type </td></tr><tr><td>7 </td><td>TPPNEPEVINTWELMAGLDD </td><td>6 </td><td>7 </td><td>20 </td><td>3.8e-082 </td><td>GRL-type </td></tr><tr><td>8 </td><td>ELRQGLKNYSNWPTF </td><td>7 </td><td>6 </td><td>15 </td><td>2.7e-073 </td><td>CGFS-type </td></tr><tr><td>9 </td><td>AGALWL </td><td>15 </td><td>15 </td><td>6 </td><td>3.5e-065 </td><td>CC-type </td></tr><tr><td>10 </td><td>MDMVQRMASEK </td><td>10 </td><td>13 </td><td>11 </td><td>2.0e-058 </td><td>CC-type </td></tr></tbody></table></div><div content-id="DSQ023TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Motif no. </th><th>Motif sequence </th><th>Number of sites in rice </th><th>Number of sites in <em>Arabidopsis</em> </th><th>Length </th><th><em>E</em>-value </th><th>Class of GRX proteins </th></tr></thead><tbody><tr><td>1 </td><td>PQVFIGGKHIGGCDQVMSMHENGELVPMLK </td><td>45 </td><td>48 </td><td>30 </td><td>3.3e-1153 </td><td>CC-type, CGFS-type, CPYC-typeand GRL-type </td></tr><tr><td>2 </td><td>VIFSKSYCCMCHTVKTLFQDL </td><td>22 </td><td>27 </td><td>21 </td><td>2.2e-474 </td><td>CC-typeand CPYC-type </td></tr><tr><td>3 </td><td>RVVLYFTSLRGIRKTYEDCCAVRAILRGHRVWVDERDVSMH </td><td>16 </td><td>13 </td><td>41 </td><td>4.9e-636 </td><td>GRL-type </td></tr><tr><td>4 </td><td>GVNPTIHELDQDPDGWEIQRALAQWGCQP </td><td>15 </td><td>21 </td><td>29 </td><td>1.8e-372 </td><td>CC-typeand CPYC-type </td></tr><tr><td>5 </td><td>GWRRCPGCNENRFVPCPNC </td><td>28 </td><td>25 </td><td>19 </td><td>4.4e-362 </td><td>GRL-type </td></tr><tr><td>6 </td><td>RLEQLVNSHPVMLFMKGTPDEPQCGFSQKVVQILKQYNVPFGSFDILTDE </td><td>7 </td><td>6 </td><td>50 </td><td>1.6e-222 </td><td>CGFS-type </td></tr><tr><td>7 </td><td>TPPNEPEVINTWELMAGLDD </td><td>6 </td><td>7 </td><td>20 </td><td>3.8e-082 </td><td>GRL-type </td></tr><tr><td>8 </td><td>ELRQGLKNYSNWPTF </td><td>7 </td><td>6 </td><td>15 </td><td>2.7e-073 </td><td>CGFS-type </td></tr><tr><td>9 </td><td>AGALWL </td><td>15 </td><td>15 </td><td>6 </td><td>3.5e-065 </td><td>CC-type </td></tr><tr><td>10 </td><td>MDMVQRMASEK </td><td>10 </td><td>13 </td><td>11 </td><td>2.0e-058 </td><td>CC-type </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7359033" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSQ023TB2" class="table-wrap-title" data-id="DSQ023TB2"><div class="label -label">Table 2.</div><div class="caption"><p>List of putative conserved motifs in rice and <em>Arabidopsis</em> GRX proteins predicted by MEME</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Motif no. </th><th>Motif sequence </th><th>Number of sites in rice </th><th>Number of sites in <em>Arabidopsis</em> </th><th>Length </th><th><em>E</em>-value </th><th>Class of GRX proteins </th></tr></thead><tbody><tr><td>1 </td><td>PQVFIGGKHIGGCDQVMSMHENGELVPMLK </td><td>45 </td><td>48 </td><td>30 </td><td>3.3e-1153 </td><td>CC-type, CGFS-type, CPYC-typeand GRL-type </td></tr><tr><td>2 </td><td>VIFSKSYCCMCHTVKTLFQDL </td><td>22 </td><td>27 </td><td>21 </td><td>2.2e-474 </td><td>CC-typeand CPYC-type </td></tr><tr><td>3 </td><td>RVVLYFTSLRGIRKTYEDCCAVRAILRGHRVWVDERDVSMH </td><td>16 </td><td>13 </td><td>41 </td><td>4.9e-636 </td><td>GRL-type </td></tr><tr><td>4 </td><td>GVNPTIHELDQDPDGWEIQRALAQWGCQP </td><td>15 </td><td>21 </td><td>29 </td><td>1.8e-372 </td><td>CC-typeand CPYC-type </td></tr><tr><td>5 </td><td>GWRRCPGCNENRFVPCPNC </td><td>28 </td><td>25 </td><td>19 </td><td>4.4e-362 </td><td>GRL-type </td></tr><tr><td>6 </td><td>RLEQLVNSHPVMLFMKGTPDEPQCGFSQKVVQILKQYNVPFGSFDILTDE </td><td>7 </td><td>6 </td><td>50 </td><td>1.6e-222 </td><td>CGFS-type </td></tr><tr><td>7 </td><td>TPPNEPEVINTWELMAGLDD </td><td>6 </td><td>7 </td><td>20 </td><td>3.8e-082 </td><td>GRL-type </td></tr><tr><td>8 </td><td>ELRQGLKNYSNWPTF </td><td>7 </td><td>6 </td><td>15 </td><td>2.7e-073 </td><td>CGFS-type </td></tr><tr><td>9 </td><td>AGALWL </td><td>15 </td><td>15 </td><td>6 </td><td>3.5e-065 </td><td>CC-type </td></tr><tr><td>10 </td><td>MDMVQRMASEK </td><td>10 </td><td>13 </td><td>11 </td><td>2.0e-058 </td><td>CC-type </td></tr></tbody></table></div><div content-id="DSQ023TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Motif no. </th><th>Motif sequence </th><th>Number of sites in rice </th><th>Number of sites in <em>Arabidopsis</em> </th><th>Length </th><th><em>E</em>-value </th><th>Class of GRX proteins </th></tr></thead><tbody><tr><td>1 </td><td>PQVFIGGKHIGGCDQVMSMHENGELVPMLK </td><td>45 </td><td>48 </td><td>30 </td><td>3.3e-1153 </td><td>CC-type, CGFS-type, CPYC-typeand GRL-type </td></tr><tr><td>2 </td><td>VIFSKSYCCMCHTVKTLFQDL </td><td>22 </td><td>27 </td><td>21 </td><td>2.2e-474 </td><td>CC-typeand CPYC-type </td></tr><tr><td>3 </td><td>RVVLYFTSLRGIRKTYEDCCAVRAILRGHRVWVDERDVSMH </td><td>16 </td><td>13 </td><td>41 </td><td>4.9e-636 </td><td>GRL-type </td></tr><tr><td>4 </td><td>GVNPTIHELDQDPDGWEIQRALAQWGCQP </td><td>15 </td><td>21 </td><td>29 </td><td>1.8e-372 </td><td>CC-typeand CPYC-type </td></tr><tr><td>5 </td><td>GWRRCPGCNENRFVPCPNC </td><td>28 </td><td>25 </td><td>19 </td><td>4.4e-362 </td><td>GRL-type </td></tr><tr><td>6 </td><td>RLEQLVNSHPVMLFMKGTPDEPQCGFSQKVVQILKQYNVPFGSFDILTDE </td><td>7 </td><td>6 </td><td>50 </td><td>1.6e-222 </td><td>CGFS-type </td></tr><tr><td>7 </td><td>TPPNEPEVINTWELMAGLDD </td><td>6 </td><td>7 </td><td>20 </td><td>3.8e-082 </td><td>GRL-type </td></tr><tr><td>8 </td><td>ELRQGLKNYSNWPTF </td><td>7 </td><td>6 </td><td>15 </td><td>2.7e-073 </td><td>CGFS-type </td></tr><tr><td>9 </td><td>AGALWL </td><td>15 </td><td>15 </td><td>6 </td><td>3.5e-065 </td><td>CC-type </td></tr><tr><td>10 </td><td>MDMVQRMASEK </td><td>10 </td><td>13 </td><td>11 </td><td>2.0e-058 </td><td>CC-type </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7359033" target="_blank">View Large</a></div></div><p>GRXs have been predicted to localize in all cellular components except nucleus using prediction software programs.<sup><a reveal-id="DSQ023C3" data-open="DSQ023C3" class="link link-ref link-reveal xref-bibr">3</a></sup> However, recently, it has been shown that nuclear activity of one of CC-type GRX, ROXY1, is required for its role in petal development.<sup><a reveal-id="DSQ023C36" data-open="DSQ023C36" class="link link-ref link-reveal xref-bibr">36</a></sup> We also analyzed the putative subcellular localization of OsGRX proteins using TargetP program.<sup><a reveal-id="DSQ023C18" data-open="DSQ023C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Using TargetP, 11, 15 and 10 of GRXs were predicted to localize to chloroplast, mitochondria and secretory pathway, respectively, other 12 were predicted to localize to any other location. Further, TMHMM predicted the presence of transmembrane α-helix in only two of GRX proteins (one predicted in secretory pathway and second in any other location). Although these results provide a clue toward putative subcellular localization of GRX proteins, experimental evidences are required to validate the results.</p>                            <h3 scrollto-destination=7359035 id="7359035" class="section-title" data-legacy-id=s3d>3.4. Structure modeling</h3>
<p>Although the structure of a number of GRXs have been determined by X-ray and NMR from various organisms, the structure of only two plant GRXs (from <em>Populus</em>) have been determined so far.<sup><a reveal-id="DSQ023C8" data-open="DSQ023C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSQ023C35" data-open="DSQ023C35" class="link link-ref link-reveal xref-bibr">35</a></sup> We made an attempt to study the structure of representative members of different classes of GRXs based on homology modeling. One representative member from each of CGFS-type, CPYC-type and CC-type classes showing best homology with known proteins in PDB database was selected and analyzed. Rice GRL-type class proteins did not show any significant homology to known GRX proteins in PDB and thus were not analyzed. The three-dimensional structure modeling of representative GRX proteins from CGFS-type, CPYC-type and CC-type classes showed that, in general, the structure of OsGRXs are conserved (Fig. <a reveal-id="DSQ023F3" data-open="DSQ023F3" class="link link-reveal link-table xref-fig">3</a>A). The structure examination showed that GRX protein shared the same conformation of structural elements of α-helices and β-sheets as that of templates (six α-helices and four β-sheets for CC- and CPYC-type, and six α-helices and three β-sheets for CGFS-type). However, structural differences are present in the variable loop regions, where sequence conservation is relatively weak. In the modeled structures, the positions of three conserved regions (constituted by WCSY, TVP and GST residues in <em>Populus</em>),<sup><a reveal-id="DSQ023C37" data-open="DSQ023C37" class="link link-ref link-reveal xref-bibr">37</a></sup> representing GSH-binding sites, were marked and found to be conserved structurally in the members of CPYC-type, whereas the modeled structures of CC-type and CGFS-type exhibited slight variability. However, these variations were not significant as depicted by root mean-square deviation calculations of modeled structures and templates using Chimera tool (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Further, the Ramachandran plot analysis revealed that 98–100% of residues lie within allowed regions, only 1–2% being in disallowed regions in the case of CGFS type (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Overall, the structures of CPYC-type and CC-type class members were more similar than that of CGFS-type. It has been proposed that the CC-type GRXs probably arose by diversification from the CPYC-type.<sup><a reveal-id="DSQ023C35" data-open="DSQ023C35" class="link link-ref link-reveal xref-bibr">35</a></sup> The molecular docking of GSH was done with representative OsGRX proteins based on the known structure of <em>Populus</em> GRX S12 complexed with GSH (PDB ID 3fza). The analysis showed that in the structure of OsGRX proteins GSH could fit well in the groove constituted by conserved active site residues (Fig. <a reveal-id="DSQ023F3" data-open="DSQ023F3" class="link link-reveal link-table xref-fig">3</a>B). The thermodynamically stable binding of ligand GSH was revealed by calculation of grid score (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). The negative energy values indicated thermodynamic stability of the binding. The stability of GSH complexed OsGRX proteins was further substantiated using consensus scoring function X-score (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Taken together, the overall structure including GSH-binding site of OsGRX proteins seems to be conserved as that of known GRX proteins, suggesting their similar biochemical properties. </p>                        <a id="7359037" scrollto-destination="7359037"></a>
<div id="DSQ023F3" data-id="DSQ023F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/m_dsq02303.jpeg?Expires=2147483647&Signature=JZ4dqY6wOTqj2XjJgHLP4HEEeKAjz5qTr-q21o5jc7T6l5I8gGI4sC5CW1MC4EKehxZ7T5jIMOtZAxIQlJC0~2NDohBKpdJUwd3EEyakMEt7xmn9Zd2curvOvuST-7rgiN-7otmLaE~dNXj8AChqSshy6KLS491nwmBscaN4y05YmEI6XGeZ0dHY6bvS0u1ekGANV-0tAyOCMWArJH5B7m1iI-VOnj4x0Nj-cDW2~xDhXDq6rvWeIvh~GYtOm7WUTKCxKm1s~k21IPEVw9CGBnRa23PQGyyAXndpnNjXWqtnHW9kozWDfIj-FFv0sXa-QwzHdmTRHY7JONio5sUPPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The structure of a representative CPYC-type, CGFS-type and CC-type GRX protein from rice. (A) Predicted structure of OsGRX21, OsGRX11 and OsGRX4. The residues of GSH-binding sites are marked in different colors. (B) Surf structure of OsGRX21, OsGRX11 and OsGRX4 showing surface topology complexed with GSH. The location of residues of the GSH-binding site is indicated in purple color." path-from-xml="dsq02303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7359037/dsq02303.jpeg" path-from-xml="dsq02303.jpeg" target="_blank">View large</a><a section="7359037" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq02303.jpeg?Expires=2147483647&Signature=Dr--yKDSbe5uugVed-P2PJrg2u~IlU3rVYf28MYovrEQa~9bniDmmDHvSOBQg9BRSM3aL6hqNtDfYwPkKXNhzVUhhfOn4q5aAGPoy0kn9Wsebt8pJT6nSFJzJMQFwGsK8YeTjTOAtnIv3xeAxAtMk~li0S0F2hF2itRpifeuuSEtgh6eode3LTyBShsUmLtG05iYdDr7PZ5R0WOfJXuqRSXJiyISCvZZ~~2Ihx4WGHhbtlbsjTlzsUZx~XpC14A7KLfjygR-E5bpC378uGukA14Onj~RseoeweEdQnRsHrME-YN73HXgXkkIfFWVgnw2bACP4D-vIq9If4n9XY~sGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7359037&ar=484440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The structure of a representative CPYC-type, CGFS-type and CC-type GRX protein from rice. (A) Predicted structure of OsGRX21, OsGRX11 and OsGRX4. The residues of GSH-binding sites are marked in different colors. (B) Surf structure of OsGRX21, OsGRX11 and OsGRX4 showing surface topology complexed with GSH. The location of residues of the GSH-binding site is indicated in purple color.</p></div></div><div content-id="DSQ023F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/m_dsq02303.jpeg?Expires=2147483647&Signature=JZ4dqY6wOTqj2XjJgHLP4HEEeKAjz5qTr-q21o5jc7T6l5I8gGI4sC5CW1MC4EKehxZ7T5jIMOtZAxIQlJC0~2NDohBKpdJUwd3EEyakMEt7xmn9Zd2curvOvuST-7rgiN-7otmLaE~dNXj8AChqSshy6KLS491nwmBscaN4y05YmEI6XGeZ0dHY6bvS0u1ekGANV-0tAyOCMWArJH5B7m1iI-VOnj4x0Nj-cDW2~xDhXDq6rvWeIvh~GYtOm7WUTKCxKm1s~k21IPEVw9CGBnRa23PQGyyAXndpnNjXWqtnHW9kozWDfIj-FFv0sXa-QwzHdmTRHY7JONio5sUPPQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The structure of a representative CPYC-type, CGFS-type and CC-type GRX protein from rice. (A) Predicted structure of OsGRX21, OsGRX11 and OsGRX4. The residues of GSH-binding sites are marked in different colors. (B) Surf structure of OsGRX21, OsGRX11 and OsGRX4 showing surface topology complexed with GSH. The location of residues of the GSH-binding site is indicated in purple color." path-from-xml="dsq02303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7359037/dsq02303.jpeg" path-from-xml="dsq02303.jpeg" target="_blank">View large</a><a section="7359037" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq02303.jpeg?Expires=2147483647&Signature=Dr--yKDSbe5uugVed-P2PJrg2u~IlU3rVYf28MYovrEQa~9bniDmmDHvSOBQg9BRSM3aL6hqNtDfYwPkKXNhzVUhhfOn4q5aAGPoy0kn9Wsebt8pJT6nSFJzJMQFwGsK8YeTjTOAtnIv3xeAxAtMk~li0S0F2hF2itRpifeuuSEtgh6eode3LTyBShsUmLtG05iYdDr7PZ5R0WOfJXuqRSXJiyISCvZZ~~2Ihx4WGHhbtlbsjTlzsUZx~XpC14A7KLfjygR-E5bpC378uGukA14Onj~RseoeweEdQnRsHrME-YN73HXgXkkIfFWVgnw2bACP4D-vIq9If4n9XY~sGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7359037&ar=484440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The structure of a representative CPYC-type, CGFS-type and CC-type GRX protein from rice. (A) Predicted structure of OsGRX21, OsGRX11 and OsGRX4. The residues of GSH-binding sites are marked in different colors. (B) Surf structure of OsGRX21, OsGRX11 and OsGRX4 showing surface topology complexed with GSH. The location of residues of the GSH-binding site is indicated in purple color.</p></div></div>                            <h3 scrollto-destination=7359038 id="7359038" class="section-title" data-legacy-id=s3e>3.5. Differential expression of OsGRX genes in various organs/developmental stages</h3>
<p>The gene expression patterns can provide important clues for the gene function. We looked for the expression evidence for all the <em>OsGRX</em> genes in cDNA, EST and MPSS databases. This analysis revealed that one or more cDNA and/or EST sequences were available for 39 (81%) of <em>OsGRX</em>s, providing strong indication for expression of most <em>OsGRX</em>s (Fig. <a reveal-id="DSQ023F4" data-open="DSQ023F4" class="link link-reveal link-table xref-fig">4</a>A). However, the frequency of ESTs for GRX genes varied and also the ESTs were derived from various tissues/organs, indicating their differential expression. Further, we surveyed Rice MPSS Project database for each of <em>OsGRX</em> gene to collect the expression evidence in terms of MPSS tags. We found significant unique 20 b signatures corresponding to at least 31 <em>OsGRX</em> genes (Fig. <a reveal-id="DSQ023F4" data-open="DSQ023F4" class="link link-reveal link-table xref-fig">4</a>A). This included MPSS signatures for three GRX genes, for which no cDNA and/or EST evidence was available. The number of tags (in tpm) also varied significantly in various tissues/organs indicating differential expression of <em>OsGRX</em> genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). Taken together, we found expression evidence for at least 42 (88%) <em>OsGRX</em> genes in terms of availability of cDNA, EST and/or MPSS tags. </p>                        <a id="7359040" scrollto-destination="7359040"></a>
<div id="DSQ023F4" data-id="DSQ023F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/m_dsq02304.jpeg?Expires=2147483647&Signature=c8El9IYvIQNLSP-2qxMqzggxohRYaP8LbGjxTnhev1J0vxD9jpWJbuQCAz-kv5ZsYKoGy07vCHJsXI-Tv~pyaYT0Hv8KKNRJ1N5QyF2whjGL11P4K8g57f5tHby6dNzkA8Go7mDrH8PMIi56utKDOctqfdXmQnahnK7u-cGMH~Xl7AnCOPHjvVRSMMIzsqO41flEKN5HvCJoKQssw7y1CgBgKlcamvAUp22AZQ7v9fLaGJYMqJosfd3jYunytpp~igK187-5yRNMOlPAYs8B1Nf-ZiPXDcCDoKtnVDe3vNj91t1Sry34c8hU7XVWjnaNOzLzZJrLVfkEGbH2ghlC7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of OsGRX genes in various tissues. (A) The green box represents detection of cDNA (column 1), EST (column 2), MPSS signature (column 3) and/or microarray data (column 4) gene expression evidence. (B) Heatmap showing hierarchical clustering of 40 OsGRX genes based on their average log signal values in various tissues/developmental stages analyzed. ML, mature leaf; YL, Y leaf; SAM, shoot apical meristem; P1-I to P1-III and P1 to P6, stages of panicle development; S1–S5, stages of seed development. The color scale is given at the bottom." path-from-xml="dsq02304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7359040/dsq02304.jpeg" path-from-xml="dsq02304.jpeg" target="_blank">View large</a><a section="7359040" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq02304.jpeg?Expires=2147483647&Signature=mWVBmorqGzKbp2bqdLNcXFQ-qOjMFiNwSStALzbSldeSrYSzdyQhHzJ2agXtX1Ul72G1usWjuFfBertHHcwN9-EnXvk7znyth1~gWkmSa-BHYXJ-2UbZYAAMav5fRVGQPkYhEaUWsgk7HFHZjWFgM0re3Uv7QWT~CZwyLjN2qiwXIUzxR2yfTLMXzYPNhV3UF9M7W2a8EzK~ZdIkRQe44bfIsBVh5KPPsB2rw14Py~YUZSBDCuU~Kt2njk3ZSoSnEHsK7wUtQv0rCDBjn4V5U2uGC-AMeMiCLhN5L46EYF5x9r7iMQxQwd~1BFwCHhfICyBX~OMGkCauPL-PEFIHGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7359040&ar=484440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of <em>OsGRX</em> genes in various tissues. (A) The green box represents detection of cDNA (column 1), EST (column 2), MPSS signature (column 3) and/or microarray data (column 4) gene expression evidence. (B) Heatmap showing hierarchical clustering of 40 <em>OsGRX</em> genes based on their average log signal values in various tissues/developmental stages analyzed. ML, mature leaf; YL, Y leaf; SAM, shoot apical meristem; P1-I to P1-III and P1 to P6, stages of panicle development; S1–S5, stages of seed development. The color scale is given at the bottom.</p></div></div><div content-id="DSQ023F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/m_dsq02304.jpeg?Expires=2147483647&Signature=c8El9IYvIQNLSP-2qxMqzggxohRYaP8LbGjxTnhev1J0vxD9jpWJbuQCAz-kv5ZsYKoGy07vCHJsXI-Tv~pyaYT0Hv8KKNRJ1N5QyF2whjGL11P4K8g57f5tHby6dNzkA8Go7mDrH8PMIi56utKDOctqfdXmQnahnK7u-cGMH~Xl7AnCOPHjvVRSMMIzsqO41flEKN5HvCJoKQssw7y1CgBgKlcamvAUp22AZQ7v9fLaGJYMqJosfd3jYunytpp~igK187-5yRNMOlPAYs8B1Nf-ZiPXDcCDoKtnVDe3vNj91t1Sry34c8hU7XVWjnaNOzLzZJrLVfkEGbH2ghlC7A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of OsGRX genes in various tissues. (A) The green box represents detection of cDNA (column 1), EST (column 2), MPSS signature (column 3) and/or microarray data (column 4) gene expression evidence. (B) Heatmap showing hierarchical clustering of 40 OsGRX genes based on their average log signal values in various tissues/developmental stages analyzed. ML, mature leaf; YL, Y leaf; SAM, shoot apical meristem; P1-I to P1-III and P1 to P6, stages of panicle development; S1–S5, stages of seed development. The color scale is given at the bottom." path-from-xml="dsq02304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7359040/dsq02304.jpeg" path-from-xml="dsq02304.jpeg" target="_blank">View large</a><a section="7359040" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq02304.jpeg?Expires=2147483647&Signature=mWVBmorqGzKbp2bqdLNcXFQ-qOjMFiNwSStALzbSldeSrYSzdyQhHzJ2agXtX1Ul72G1usWjuFfBertHHcwN9-EnXvk7znyth1~gWkmSa-BHYXJ-2UbZYAAMav5fRVGQPkYhEaUWsgk7HFHZjWFgM0re3Uv7QWT~CZwyLjN2qiwXIUzxR2yfTLMXzYPNhV3UF9M7W2a8EzK~ZdIkRQe44bfIsBVh5KPPsB2rw14Py~YUZSBDCuU~Kt2njk3ZSoSnEHsK7wUtQv0rCDBjn4V5U2uGC-AMeMiCLhN5L46EYF5x9r7iMQxQwd~1BFwCHhfICyBX~OMGkCauPL-PEFIHGg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7359040&ar=484440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of <em>OsGRX</em> genes in various tissues. (A) The green box represents detection of cDNA (column 1), EST (column 2), MPSS signature (column 3) and/or microarray data (column 4) gene expression evidence. (B) Heatmap showing hierarchical clustering of 40 <em>OsGRX</em> genes based on their average log signal values in various tissues/developmental stages analyzed. ML, mature leaf; YL, Y leaf; SAM, shoot apical meristem; P1-I to P1-III and P1 to P6, stages of panicle development; S1–S5, stages of seed development. The color scale is given at the bottom.</p></div></div><p>In addition to sequence-based expression analysis methods, we previewed the expression profiles of <em>OsGRX</em> genes during various stages of development using microarray data. The microarray data from 54 arrays representing 18 stages of development throughout the life cycle of rice, including root, mature leaf, Y leaf, shoot apical meristem (SAM), stages of panicle development (P1-I to P1-III, P1 to P6) and stages of seed development (S1 to S5) was analyzed. The average log signal values for all the 40 <em>OsGRX</em> genes represented on Affymetrix Rice Genome array were analyzed. Following whole chip data processing, the average log signal values were extracted (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). The hierarchical cluster analysis based on average log signal values indicated that <em>OsGRX</em> genes display diverse expression patterns (Fig. <a reveal-id="DSQ023F4" data-open="DSQ023F4" class="link link-reveal link-table xref-fig">4</a>B). Most of the 40 GRX genes are expressed in at least one of the 18 rice tissues investigated in this study. Most diverse expression patterns were found in the members of CC-type GRX genes, indicating that this class of proteins are most evolved with specialized function. <em>OsGRX8</em> has a significantly higher expression in SAM and stages of panicle development than in other tissues, whereas <em>OsGRX3</em> seems to be predominantly expressed in late stage of panicle development (P6) and stages of seed development. <em>OsGRX28</em> exhibited specific expression during S1 stage of seed development only. <em>OsGRX13</em> and <em>OsGRX19</em> were also expressed preferentially in SAM and early stages of panicle development. Two members of CGFS-type class, <em>OsGRX11</em> and <em>OsGRX27</em> were expressed at lower level in SAM and stages of panicle development as compared to other tissues/developmental stages. <em>OsGRX20</em>, a CPYC-class member, was expressed at higher level in mature and Y leaf. At least six members of GRL type showed very low level of expression in all the tissues examined. <em>OsGRL1</em>, a GRL-type GRX, showed maximum expression levels in mature leaf and stages of seed development. <em>OsGRL19</em> also exhibited similar expression profile with some expression in late stages of panicle development. However, <em>OsGRL16</em> showed distinct very low level of expression in SAM as compared to all other tissues/developmental stages analyzed. Similar to rice, the expression patterns of CC-type GRX genes were most diverse in <em>Arabidopsis</em> as well, showing differential and/or specific expression in various tissues/developmental stages analyzed (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5A</a>). The members of CGFS- and CPYC-type classes were expressed at high level in all the tissue samples analyzed, whereas few genes of GRL-type class also exhibited tissue-/developmental stage-specific expression patterns (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5A</a>).</p><p>Although the expression profiles of GRXs have been reported in various plant species,<sup><a reveal-id="DSQ023C15" data-open="DSQ023C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSQ023C16" data-open="DSQ023C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSQ023C38" data-open="DSQ023C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSQ023C39" data-open="DSQ023C39" class="link link-ref link-reveal xref-bibr">39</a></sup> the evidence for their role in plant growth and development is lacking except for floral organ development. The molecular genetic analysis revealed the role of two <em>Arabidopsis</em> CC-type GRXs named as ROXY1 and ROXY2, in petal and anther development.<sup><a reveal-id="DSQ023C11" data-open="DSQ023C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSQ023C12" data-open="DSQ023C12" class="link link-ref link-reveal xref-bibr">12</a></sup> It has been demonstrated that nuclear localization of ROXY1 protein is crucial for its role in petal development.<sup><a reveal-id="DSQ023C36" data-open="DSQ023C36" class="link link-ref link-reveal xref-bibr">36</a></sup> In a yeast two-hybrid screen, ROXY1 was found to interact with TGA family of transcription factors.<sup><a reveal-id="DSQ023C36" data-open="DSQ023C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Recently, complementation analysis revealed the conserved function in floral organ development of orthologs of ROXY1 and ROXY2 in rice as well.<sup><a reveal-id="DSQ023C13" data-open="DSQ023C13" class="link link-ref link-reveal xref-bibr">13</a></sup> In our study also, we found the preferential expression of <em>OsROXY1</em> (<em>OsGRX8</em>) during stages of panicle development, which further validates its role in floral organ development. Similarly, other <em>OsGRX</em> genes expressed preferentially in specific tissue/developmental stage might perform specific function crucial for the development.</p>                            <h3 scrollto-destination=7359043 id="7359043" class="section-title" data-legacy-id=s3f>3.6. Differential expression of OsGRX genes in response to hormone treatments</h3>
<p>We also studied the expression profiles of <em>OsGRX</em> genes in rice seedlings subjected to various hormone treatments auxin (IAA), cytokinin (BAP), abscisic acid (ABA), ethylene derivative (ACC), salicylic acid (SA) and jasmonic acid (JA)). A total of 11 GRX genes showed differential expression more than 2-fold in at least one hormone treatment as compared to control (Fig. <a reveal-id="DSQ023F5" data-open="DSQ023F5" class="link link-reveal link-table xref-fig">5</a>A). Six (<em>OsGRX3</em>, <em>GRX6</em>, <em>GRX8</em>, <em>GRX9</em>, <em>GRX14</em> and <em>GRX28</em>) genes showed up-regulation, while three (<em>OsGRX2</em>, <em>GRX27</em> and <em>GRX29</em>) genes showed down-regulation in different hormone treatments as compared to control. <em>OsGRX23</em> showed down-regulation in response to IAA, ABA, SA and JA, however, up-regulation in response to BAP. <em>OsGRX16</em> was up-regulated in response to ABA, but down-regulated in response to BAP. Five of <em>OsGRX</em> genes showed response to specific hormone treatments only, while others were responsive to two or more hormone treatments. The fold-change values of <em>OsGRX</em> genes showing differential expression is given in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>. The real-time PCR results of selected differentially expressed genes under various hormone treatments were consistent with the microarray data (Fig. <a reveal-id="DSQ023F5" data-open="DSQ023F5" class="link link-reveal link-table xref-fig">5</a>B). In <em>Arabidopsis</em> also, at least eight GRX genes (five CC type and three GRL type) showed significant differential expression in response to various plant hormones (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5B</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>). A very little information is available regarding the direct link between GRX and hormonal response so far. One of the TGA-interacting GRX protein has been proposed as a potential regulatory component of SA/JA antagonism.<sup><a reveal-id="DSQ023C14" data-open="DSQ023C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Our results clearly show that GRXs might play important role in response to various plant hormones. </p>                        <a id="7359045" scrollto-destination="7359045"></a>
<div id="DSQ023F5" data-id="DSQ023F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/m_dsq02305.jpeg?Expires=2147483647&Signature=ACt3Qvh8exinLccFPtPnj0kx1N-5fl3F6IooI2xwIdwFc76oU4ON6SqW1hO6vfhT4npKQ5fg~FVzGcwHffrQgva2j7b0NluQf8cNrJqc9swe6dhwhNShRYYQSmGKuC-OKYmOoTI7N3-twozs~wLAgNeXKEwJyWCC3VN9pLZz7-d1dReik0EJsyiURR1RqCAtNDye1x8IodxRY71hi5dOweOotGkkN-WeOXIL1RPekppSjgXuoCIwJ0lZlvtspHd5-5ND98faQe8WAfdy27z65niefOv3g~vOrbkvPcJ2Nxr8fu6hvE6lt2deIxv6aCo4CfUNfdVtmgJrgwSny8XGXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of OsGRX genes showing differential expression in response to various hormone treatments. (A) Heatmap showing the differential expression of OsGRX genes in response to various hormone treatments. The heatmap has been generated based on the fold-change values in the treated sample when compared with its mock-treated control sample. The color scale for fold-change values is shown at the bottom. (B) Validation of differential expression of representative GRX genes in response to various hormone treatments. Relative mRNA levels of GRX genes in response to various hormone treatments when compared with its mock-treated control sample are shown. IAA, auxin; BAP, cytokinin; ABA, abscisic acid; ACC, ethylene derivative; SA, salicylic acid; JA, jasmonic acid." path-from-xml="dsq02305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7359045/dsq02305.jpeg" path-from-xml="dsq02305.jpeg" target="_blank">View large</a><a section="7359045" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq02305.jpeg?Expires=2147483647&Signature=pT902FDhsOegxSRFNojHSd4p3nENQ-jgtaSIJs6MpK~6jqcySjfp3xwdUhNqljzl4LDis772zsSdiCx5ut-AvI7NIsJgro2BNAweP2eM93~H5jtEhoYlXX756YyeHXpTHVVAxXQYvuUVixMdBPTjimiz9KKfke-mZgawqMITxCSzOgOjhgcTEBdCO50Vif3O5aYCh8nHKhR0oH~6LTwRbLUuBOL7tP61zCTMY8xXUTwmCCY3MMGN6vAW5REMzTpn6llk13wRVfRsPB4wxAJ~oDs7FCdN-M5nf3Ki0270hOBwfuoYT6o~ebdFbiCx4Kq8hfGd7rGM80XXJh8k28txMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7359045&ar=484440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of <em>OsGRX</em> genes showing differential expression in response to various hormone treatments. (A) Heatmap showing the differential expression of <em>OsGRX</em> genes in response to various hormone treatments. The heatmap has been generated based on the fold-change values in the treated sample when compared with its mock-treated control sample. The color scale for fold-change values is shown at the bottom. (B) Validation of differential expression of representative GRX genes in response to various hormone treatments. Relative mRNA levels of GRX genes in response to various hormone treatments when compared with its mock-treated control sample are shown. IAA, auxin; BAP, cytokinin; ABA, abscisic acid; ACC, ethylene derivative; SA, salicylic acid; JA, jasmonic acid.</p></div></div><div content-id="DSQ023F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/m_dsq02305.jpeg?Expires=2147483647&Signature=ACt3Qvh8exinLccFPtPnj0kx1N-5fl3F6IooI2xwIdwFc76oU4ON6SqW1hO6vfhT4npKQ5fg~FVzGcwHffrQgva2j7b0NluQf8cNrJqc9swe6dhwhNShRYYQSmGKuC-OKYmOoTI7N3-twozs~wLAgNeXKEwJyWCC3VN9pLZz7-d1dReik0EJsyiURR1RqCAtNDye1x8IodxRY71hi5dOweOotGkkN-WeOXIL1RPekppSjgXuoCIwJ0lZlvtspHd5-5ND98faQe8WAfdy27z65niefOv3g~vOrbkvPcJ2Nxr8fu6hvE6lt2deIxv6aCo4CfUNfdVtmgJrgwSny8XGXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of OsGRX genes showing differential expression in response to various hormone treatments. (A) Heatmap showing the differential expression of OsGRX genes in response to various hormone treatments. The heatmap has been generated based on the fold-change values in the treated sample when compared with its mock-treated control sample. The color scale for fold-change values is shown at the bottom. (B) Validation of differential expression of representative GRX genes in response to various hormone treatments. Relative mRNA levels of GRX genes in response to various hormone treatments when compared with its mock-treated control sample are shown. IAA, auxin; BAP, cytokinin; ABA, abscisic acid; ACC, ethylene derivative; SA, salicylic acid; JA, jasmonic acid." path-from-xml="dsq02305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7359045/dsq02305.jpeg" path-from-xml="dsq02305.jpeg" target="_blank">View large</a><a section="7359045" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq02305.jpeg?Expires=2147483647&Signature=pT902FDhsOegxSRFNojHSd4p3nENQ-jgtaSIJs6MpK~6jqcySjfp3xwdUhNqljzl4LDis772zsSdiCx5ut-AvI7NIsJgro2BNAweP2eM93~H5jtEhoYlXX756YyeHXpTHVVAxXQYvuUVixMdBPTjimiz9KKfke-mZgawqMITxCSzOgOjhgcTEBdCO50Vif3O5aYCh8nHKhR0oH~6LTwRbLUuBOL7tP61zCTMY8xXUTwmCCY3MMGN6vAW5REMzTpn6llk13wRVfRsPB4wxAJ~oDs7FCdN-M5nf3Ki0270hOBwfuoYT6o~ebdFbiCx4Kq8hfGd7rGM80XXJh8k28txMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7359045&ar=484440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of <em>OsGRX</em> genes showing differential expression in response to various hormone treatments. (A) Heatmap showing the differential expression of <em>OsGRX</em> genes in response to various hormone treatments. The heatmap has been generated based on the fold-change values in the treated sample when compared with its mock-treated control sample. The color scale for fold-change values is shown at the bottom. (B) Validation of differential expression of representative GRX genes in response to various hormone treatments. Relative mRNA levels of GRX genes in response to various hormone treatments when compared with its mock-treated control sample are shown. IAA, auxin; BAP, cytokinin; ABA, abscisic acid; ACC, ethylene derivative; SA, salicylic acid; JA, jasmonic acid.</p></div></div>                            <h3 scrollto-destination=7359046 id="7359046" class="section-title" data-legacy-id=s3g>3.7. Differential expression of OsGRX genes in response to different stresses</h3>
<p>To gain insight into the role of <em>OsGRX</em> genes during various abiotic stress conditions, their expression patterns were investigated in rice seedlings subjected to desiccation, salt, cold, methyl viologen, hydrogen peroxide and arsenate stress. A total of 19 (40%) GRX genes showed differential expression under at least one of the abiotic stress conditions analyzed (Fig. <a reveal-id="DSQ023F6" data-open="DSQ023F6" class="link link-reveal link-table xref-fig">6</a>A). Notably, 11 of these genes belong to CC-type and five from GRL-type class. Some of them were regulated by a specific stress condition only; however, others were regulated by more than one stress condition. The fold-change values of <em>OsGRX</em> genes showing differential expression is given in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>. The validation of expression profiles of some of the selected genes by real-time PCR analysis confirmed their differential expression (Fig. <a reveal-id="DSQ023F6" data-open="DSQ023F6" class="link link-reveal link-table xref-fig">6</a>B). Likewise, a total of 23 (48%) of <em>AtGRX</em> genes, including 14 CC type, 1 CGFS type, 3 CPYC type and 5 GRL type, were differentially expressed in root and shoot tissues subjected to various abiotic stress conditions (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5C</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S10</a>). </p>                        <a id="7359048" scrollto-destination="7359048"></a>
<div id="DSQ023F6" data-id="DSQ023F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/m_dsq02306.jpeg?Expires=2147483647&Signature=Cy8Eox~S3bYt25lcf2KuBlfv-F4EkIaHV3U6T74~JwbgvzS31wrPdL7sDXA7LODTFP5-8QY0IYRN253OGviuur8EnpDCGvid36EuSMTwiINpmONKQVlvz6JCdeOiVgTu7PzWHXDniwusiEcu2Iicqw0DU-Sp75tzPJpYlWVvZtsfYjH6A7y1qQ5dmlR5RZ6FiYIw5Nn1GtmhadT8OlTEL2py-xgVLgzF4ODrO8JrygSOh3T1wyG5Oqs61fbMisWRRX-LcEapIbeFaLXcOovtyTc3KYy7d5~WZbRC-WGQFSn5RPQZBGRySD6HWG8F3z2aI~jbioB~xCABY1t1Eee5Hg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of OsGRX genes showing differential expression in response to various abiotic stress treatments. (A) Heatmap showing the differential expression of OsGRX genes in response to various abiotic stress treatments. The heatmap has been generated based on the fold-change values in the treated sample when compared with its mock-treated control sample. The color scale for fold-change values in shown at the bottom. (B) Validation of differential expression of representative GRX genes in response to various abiotic stress treatments. Relative mRNA levels of GRX genes in response to various abiotic stress treatments when compared with its mock-treated control sample are shown. DS, desiccation; SS, salt; CS, cold; MV, methyl viologen; H2O2, hydrogen peroxide, Ars, arsenate stress. Azucena and Bala represent arsenate-sensitive and arsenate-resistant rice varieties, respectively." path-from-xml="dsq02306.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7359048/dsq02306.jpeg" path-from-xml="dsq02306.jpeg" target="_blank">View large</a><a section="7359048" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq02306.jpeg?Expires=2147483647&Signature=dqSuM3vypFn05L3k2dOctkIQZ8kNGUNwDi8w07ak6pwsq~j7g3UV~K0x89MraL15sWwVk35k6tAZMrlRxnYaBkTTeahgNQzEPkl8E~GwZyM0cB7zIroLIUn03wvel8X5GF8dDCCXw7rDAzvItvrF1qxyVfrcZTtyYg3l1705QeSvSANEbFmhdvcYfncl-0z~7FurZgEL1U0CCCwDHHwTPesTJEe5w8CXb6qttqS0tqBni9nMedQHRebZUxEQnzNdkivDdrypwP3XIGOr2~kPgVHHmMastOAiUaFxvF8UBYoHKAQWc4VFhtlvxjy3ajgY18HTvFHh2V5eiL2~eRYezQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7359048&ar=484440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02306.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of <em>OsGRX</em> genes showing differential expression in response to various abiotic stress treatments. (A) Heatmap showing the differential expression of <em>OsGRX</em> genes in response to various abiotic stress treatments. The heatmap has been generated based on the fold-change values in the treated sample when compared with its mock-treated control sample. The color scale for fold-change values in shown at the bottom. (B) Validation of differential expression of representative GRX genes in response to various abiotic stress treatments. Relative mRNA levels of GRX genes in response to various abiotic stress treatments when compared with its mock-treated control sample are shown. DS, desiccation; SS, salt; CS, cold; MV, methyl viologen; H<sub>2</sub>O<sub>2</sub>, hydrogen peroxide, Ars, arsenate stress. Azucena and Bala represent arsenate-sensitive and arsenate-resistant rice varieties, respectively.</p></div></div><div content-id="DSQ023F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/m_dsq02306.jpeg?Expires=2147483647&Signature=Cy8Eox~S3bYt25lcf2KuBlfv-F4EkIaHV3U6T74~JwbgvzS31wrPdL7sDXA7LODTFP5-8QY0IYRN253OGviuur8EnpDCGvid36EuSMTwiINpmONKQVlvz6JCdeOiVgTu7PzWHXDniwusiEcu2Iicqw0DU-Sp75tzPJpYlWVvZtsfYjH6A7y1qQ5dmlR5RZ6FiYIw5Nn1GtmhadT8OlTEL2py-xgVLgzF4ODrO8JrygSOh3T1wyG5Oqs61fbMisWRRX-LcEapIbeFaLXcOovtyTc3KYy7d5~WZbRC-WGQFSn5RPQZBGRySD6HWG8F3z2aI~jbioB~xCABY1t1Eee5Hg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of OsGRX genes showing differential expression in response to various abiotic stress treatments. (A) Heatmap showing the differential expression of OsGRX genes in response to various abiotic stress treatments. The heatmap has been generated based on the fold-change values in the treated sample when compared with its mock-treated control sample. The color scale for fold-change values in shown at the bottom. (B) Validation of differential expression of representative GRX genes in response to various abiotic stress treatments. Relative mRNA levels of GRX genes in response to various abiotic stress treatments when compared with its mock-treated control sample are shown. DS, desiccation; SS, salt; CS, cold; MV, methyl viologen; H2O2, hydrogen peroxide, Ars, arsenate stress. Azucena and Bala represent arsenate-sensitive and arsenate-resistant rice varieties, respectively." path-from-xml="dsq02306.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7359048/dsq02306.jpeg" path-from-xml="dsq02306.jpeg" target="_blank">View large</a><a section="7359048" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq02306.jpeg?Expires=2147483647&Signature=dqSuM3vypFn05L3k2dOctkIQZ8kNGUNwDi8w07ak6pwsq~j7g3UV~K0x89MraL15sWwVk35k6tAZMrlRxnYaBkTTeahgNQzEPkl8E~GwZyM0cB7zIroLIUn03wvel8X5GF8dDCCXw7rDAzvItvrF1qxyVfrcZTtyYg3l1705QeSvSANEbFmhdvcYfncl-0z~7FurZgEL1U0CCCwDHHwTPesTJEe5w8CXb6qttqS0tqBni9nMedQHRebZUxEQnzNdkivDdrypwP3XIGOr2~kPgVHHmMastOAiUaFxvF8UBYoHKAQWc4VFhtlvxjy3ajgY18HTvFHh2V5eiL2~eRYezQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7359048&ar=484440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02306.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of <em>OsGRX</em> genes showing differential expression in response to various abiotic stress treatments. (A) Heatmap showing the differential expression of <em>OsGRX</em> genes in response to various abiotic stress treatments. The heatmap has been generated based on the fold-change values in the treated sample when compared with its mock-treated control sample. The color scale for fold-change values in shown at the bottom. (B) Validation of differential expression of representative GRX genes in response to various abiotic stress treatments. Relative mRNA levels of GRX genes in response to various abiotic stress treatments when compared with its mock-treated control sample are shown. DS, desiccation; SS, salt; CS, cold; MV, methyl viologen; H<sub>2</sub>O<sub>2</sub>, hydrogen peroxide, Ars, arsenate stress. Azucena and Bala represent arsenate-sensitive and arsenate-resistant rice varieties, respectively.</p></div></div><p>We also analyzed the response of <em>OsGRX</em> genes to infection of fungus <em>M. grisea</em> and obligate root hemiparasite <em>S. hermonthica</em>, which cause severe loss to rice yield. For this analysis, we used microarray data from two earlier studies on transcriptome analysis of rice (Nipponbare) after infection with <em>M. grisea</em> and gene expression profiling in roots of susceptible (IAC165) and highly resistant (Nipponbare) cultivars after infection with <em>S. hermonthica</em>.<sup><a reveal-id="DSQ023C24" data-open="DSQ023C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSQ023C25" data-open="DSQ023C25" class="link link-ref link-reveal xref-bibr">25</a></sup> The data analysis revealed that a total of 18 <em>OsGRX</em> genes showed response to biotic stress conditions (Fig. <a reveal-id="DSQ023F7" data-open="DSQ023F7" class="link link-reveal link-table xref-fig">7</a>). Among these, nine genes belong to CC-type class. The fold-change values of <em>OsGRX</em> genes showing differential expression is given in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S9</a>. These results further confirm that members of CC-type class GRX genes are more functionally evolved as compared to other classes. </p>                        <a id="7359050" scrollto-destination="7359050"></a>
<div id="DSQ023F7" data-id="DSQ023F7" class="fig fig-section"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/m_dsq02307.jpeg?Expires=2147483647&Signature=QvEp2dC~ZOFkrEjHLTiS-iWwvrHsViNrPorbK4U6Djo3taF~KCBJBzLB0eZlQ53ryH6qVqAwXvJ8uBfdb9TcFyEd5xqmLCoM6M6e4g6Amdjs-GRGRKId1vlZD0uOhl3cf2TrAGimHn~HwVkxv4Ty781WglrE1K3dAM8O-0hHczcFD57ivB-vcGAmOP0a0tKRh7EbfrXeveHN1u9wIloezp4SH8U~EIFrdST33uQaaG6QtAv8VGdi9kU5pCEUuX~8wwQajRrJwElUFI5NcshdnCUGWtIGFlTMLzZLVsmR2V-~Y0~~YzjDN8nOBpyAXbysWbZDg7o9WhNXkFqBzZmyUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of OsGRX genes showing differential expression in response to various biotic stress treatments. Heatmap showing the differential expression of OsGRX genes in response to various biotic stress treatments. The heatmap has been generated based on the fold-change values in the treated sample when compared with its mock-treated control sample. The color scale for fold-change values in shown at the bottom. dpi, days post inoculation." path-from-xml="dsq02307.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7359050/dsq02307.jpeg" path-from-xml="dsq02307.jpeg" target="_blank">View large</a><a section="7359050" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq02307.jpeg?Expires=2147483647&Signature=dk6UWUkV3jtQrhw6CeXkITyAgjwk8AdGG2vcgTF~bGB~yE-MW6m5hy9OZTtoXmDwe3zs7c5baU0XcYI~5OhBFebzAScTw8l5g~r~W28Ke39kc3Q-cRDZc8itd1CIW1cT1wjMvu4byrWtepJDS6zr~Pf9ElVuFzrx6kqc9YSMriGYVw-2Hy8PljPY3KWQTwXWqHNzwmXCUrRck9jRGkfjplpq2~~MXAXJGrFQvRG9L5UdiraO2DeNq-UQHa69MWKvvHqdosMPkFD70oGTGL6E2CaeZPms06IvTicPrT-~TmenbrnIBfN8BZV7~OTij1Xl4XGt9xntazkl-od3PXDrnQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7359050&ar=484440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02307.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of <em>OsGRX</em> genes showing differential expression in response to various biotic stress treatments. Heatmap showing the differential expression of <em>OsGRX</em> genes in response to various biotic stress treatments. The heatmap has been generated based on the fold-change values in the treated sample when compared with its mock-treated control sample. The color scale for fold-change values in shown at the bottom. dpi, days post inoculation.</p></div></div><div content-id="DSQ023F7" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 7.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/m_dsq02307.jpeg?Expires=2147483647&Signature=QvEp2dC~ZOFkrEjHLTiS-iWwvrHsViNrPorbK4U6Djo3taF~KCBJBzLB0eZlQ53ryH6qVqAwXvJ8uBfdb9TcFyEd5xqmLCoM6M6e4g6Amdjs-GRGRKId1vlZD0uOhl3cf2TrAGimHn~HwVkxv4Ty781WglrE1K3dAM8O-0hHczcFD57ivB-vcGAmOP0a0tKRh7EbfrXeveHN1u9wIloezp4SH8U~EIFrdST33uQaaG6QtAv8VGdi9kU5pCEUuX~8wwQajRrJwElUFI5NcshdnCUGWtIGFlTMLzZLVsmR2V-~Y0~~YzjDN8nOBpyAXbysWbZDg7o9WhNXkFqBzZmyUw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Expression profiles of OsGRX genes showing differential expression in response to various biotic stress treatments. Heatmap showing the differential expression of OsGRX genes in response to various biotic stress treatments. The heatmap has been generated based on the fold-change values in the treated sample when compared with its mock-treated control sample. The color scale for fold-change values in shown at the bottom. dpi, days post inoculation." path-from-xml="dsq02307.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7359050/dsq02307.jpeg" path-from-xml="dsq02307.jpeg" target="_blank">View large</a><a section="7359050" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq02307.jpeg?Expires=2147483647&Signature=dk6UWUkV3jtQrhw6CeXkITyAgjwk8AdGG2vcgTF~bGB~yE-MW6m5hy9OZTtoXmDwe3zs7c5baU0XcYI~5OhBFebzAScTw8l5g~r~W28Ke39kc3Q-cRDZc8itd1CIW1cT1wjMvu4byrWtepJDS6zr~Pf9ElVuFzrx6kqc9YSMriGYVw-2Hy8PljPY3KWQTwXWqHNzwmXCUrRck9jRGkfjplpq2~~MXAXJGrFQvRG9L5UdiraO2DeNq-UQHa69MWKvvHqdosMPkFD70oGTGL6E2CaeZPms06IvTicPrT-~TmenbrnIBfN8BZV7~OTij1Xl4XGt9xntazkl-od3PXDrnQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7359050&ar=484440&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsq02307.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Expression profiles of <em>OsGRX</em> genes showing differential expression in response to various biotic stress treatments. Heatmap showing the differential expression of <em>OsGRX</em> genes in response to various biotic stress treatments. The heatmap has been generated based on the fold-change values in the treated sample when compared with its mock-treated control sample. The color scale for fold-change values in shown at the bottom. dpi, days post inoculation.</p></div></div><p>The role of GRXs in stress responses has been proposed in many studies. The <em>Arabidopsis</em> chloroplastic GRXs, <em>AtGRXcp</em> and <em>AtGRX4</em>, were induced by various stresses and suppressed the yeast <em>grx5</em> mutant sensitivity to oxidants.<sup><a reveal-id="DSQ023C9" data-open="DSQ023C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSQ023C10" data-open="DSQ023C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The <em>atgrxcp</em> mutant plants showed defects in early seedling growth under oxidative stress suggesting the biological role in protecting cells against oxidative damage.<sup><a reveal-id="DSQ023C9" data-open="DSQ023C9" class="link link-ref link-reveal xref-bibr">9</a></sup> The overexpression of <em>Pteris vittata</em> GRX, <em>PvGRX5</em>, improved arsenic tolerance in transgenic <em>Arabidopsis</em> plants via decreasing arsenic accumulation.<sup><a reveal-id="DSQ023C40" data-open="DSQ023C40" class="link link-ref link-reveal xref-bibr">40</a></sup> The analysis of transgenic plants overexpressing <em>OsWRKY13</em> revealed that OsWRKY-associated disease resistance pathway positively interact with GSH/GRX system to monitor redox homeostatsis.<sup><a reveal-id="DSQ023C41" data-open="DSQ023C41" class="link link-ref link-reveal xref-bibr">41</a></sup> Recently, it has been reported that the overexpression of rice CC-type GRXs in <em>Arabidopsis</em> led to an increased accumulation of H<sub>2</sub>O<sub>2</sub> levels and hyper-susceptibility to infection from necrotrophic pathogen <em>Botrytis cinerea</em>.<sup><a reveal-id="DSQ023C13" data-open="DSQ023C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Several plant GRX targets have been identified which are involved in various cellular processes and located in different subcellular compartments. The study suggested that GRXs have a broad capacity to reduce oxidized disulfides and to provide electrons to many enzymatic reduction reactions maintaining redox homeostasis.<sup><a reveal-id="DSQ023C42" data-open="DSQ023C42" class="link link-ref link-reveal xref-bibr">42</a></sup> Taken together, the differential expression of several <em>OsGRX</em> genes indicates their important role in various abiotic and biotic stress responses and they might be involved in maintaining redox homeostasis during stress conditions and overexpression of few of these might impart stress tolerance in transgenic plants.</p>                            <h3 scrollto-destination=7359052 id="7359052" class="section-title" data-legacy-id=s3h>3.8. Duplicated GRX genes exhibit redundant and divergent expression patterns</h3>
<p>Gene duplication is an important event, which gives rise to functional redundancy and diversification. Gene duplication may lead to sub- or neo-functionalization in addition to non- or hypo-functionalization.<sup><a reveal-id="DSQ023C43" data-open="DSQ023C43" class="link link-ref link-reveal xref-bibr">43</a></sup> We found many segmentally duplicated and a few tandemly duplicated <em>OsGRX</em> genes. We investigated differences in the expression patterns of duplicated <em>OsGRX</em> genes to reveal the role of gene duplication in functional diversification. The expression data were available for six gene pairs present on duplicated chromosomal segments and one gene pair localized in tandem. Among the six gene pairs, three, <em>OsGRX2</em>/<em>16</em>, <em>OsGRX6</em>/<em>17</em> and <em>OsGRX8</em>/<em>13</em>, showed similar expression patterns with high (&gt;0.75) Pearson correlation coefficient (PCC). Both the genes in gene pair <em>OsGRL9</em>/<em>17</em> showed very low expression levels in all the tissues examined. Two gene pairs <em>OsGRX9</em>/<em>14</em> and <em>OsGRL7</em>/<em>14</em> showed highly divergent expression patterns with PCC value of 0.19 and −0.45, respectively. One gene pair (<em>OsGRX28</em>/<em>29</em>), which is tandemly localized, also showed divergent expression patterns with PCC value of 0.56. The gene pairs with divergent expression patterns might represent the events of sub- or neo-functionalization, suggesting their functional diversification during evolution.</p>                            <h3 scrollto-destination=7359054 id="7359054" class="section-title" data-legacy-id=s3i>3.9. Conclusions</h3>
<p>In this study, we have identified 48 GRX genes in rice and compared with those of <em>Arabidopsis</em>. The GRX proteins formed four classes as supported by phylogeny and motif organization. Gene duplication analysis revealed that expansion of this gene family has occurred independently in rice and <em>Arabidopsis</em> via whole genome duplication events. Further, the phylogenetic analysis showed that GRX gene family has expanded in species-specific manner after monocot–dicot split. The expression analysis revealed the differential temporal and spatial expression patterns of <em>OsGRX</em> genes, which suggested their role in regulating plant growth and development throughout the lifecycle of plant. Further, we found the expression of <em>OsGRX</em> genes is influenced by several environmental stimuli, including hormones, abiotic stress and biotic stress conditions indicating their role in hormonal and stress responses. Although from our expression analysis results in rice and <em>Arabidopsis</em> (Figs <a reveal-id="DSQ023F4  DSQ023F7" data-open="DSQ023F4  DSQ023F7" class="link link-reveal link-table xref-fig">4–7</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>), the GRX proteins from different classes could not be linked to a specific function and/or response pattern, it is clear that CC-type class GRX proteins are most functionally evolved and are involved in diverse biological processes, including development and response to various environmental stimuli (hormone, abiotic and biotic stress). However, the GRL-type class GRX proteins appear to be more specifically involved in response to various environmental stimuli. Previously, the members of CGFS and CPYC type have been linked only to response to oxidative stress. Our results also support their role in various stress responses. However, the functional validation and biochemical characterization of various members will only provide definitive clues about the specific roles of different classes of GRX proteins. The data provided in this study will serve as useful information for further in-depth functional analysis of <em>GRX</em> genes in rice and <em>Arabidopsis</em>.</p>                            <h2 scrollto-destination=7359056 id="7359056" class="section-title" data-legacy-id=s4>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7359058 id="7359058" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was supported by the Department of Biotechnology, Government of India, New Delhi, under the Innovative Young Biotechnologists Award scheme and core grant from NIPGR to M.J.</p>                            <h2 scrollto-destination=7359060 id="7359060" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSQ023C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lillig</div> <div class="given-names">C.H.</div></div>,  <div class="name"><div class="surname">Berndt</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Holmgren</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Glutaredoxin systems</div>, <div class="source">Biochim. Biophys. Acta</div> , <div class="year">2008</div>, vol. <div class="volume">1780</div> (pg. <div class="fpage">1304</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Glutaredoxin%20systems&amp;author=C.H.%20Lillig&amp;author=C.%20Berndt&amp;author=A.%20Holmgren&amp;publication_year=2008&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=1780&amp;pages=1304-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbagen.2008.06.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbagen.2008.06.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbagen.2008.06.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18621099" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rouhier</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Lemaire</div> <div class="given-names">S.D.</div></div>,  <div class="name"><div class="surname">Jacquot</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">The role of glutathione in photosynthetic organisms: emerging functions for glutaredoxins and glutathionylation</div>, <div class="source">Annu. Rev. Plant Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">143</div>-<div class="lpage">66</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev.arplant.59.032607.092811" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1146/annurev.arplant.59.032607.092811</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20role%20of%20glutathione%20in%20photosynthetic%20organisms%3A%20emerging%20functions%20for%20glutaredoxins%20and%20glutathionylation&amp;author=N.%20Rouhier&amp;author=S.D.%20Lemaire&amp;author=J.P.%20Jacquot&amp;publication_year=2008&amp;journal=Annu.%20Rev.%20Plant%20Biol.&amp;volume=59&amp;pages=143-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.59.032607.092811" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.59.032607.092811" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.59.032607.092811"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18444899" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rouhier</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Gelhaye</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Jacquot</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Plant glutaredoxins: still mysterious reducing systems</div>, <div class="source">Cell Mol. Life Sci.</div> , <div class="year">2004</div>, vol. <div class="volume">61</div> (pg. <div class="fpage">1266</div>-<div class="lpage">77</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00018-004-3410-y" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00018-004-3410-y</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20glutaredoxins%3A%20still%20mysterious%20reducing%20systems&amp;author=N.%20Rouhier&amp;author=E.%20Gelhaye&amp;author=J.P.%20Jacquot&amp;publication_year=2004&amp;journal=Cell%20Mol.%20Life%20Sci.&amp;volume=61&amp;pages=1266-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00018-004-3410-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00018-004-3410-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00018-004-3410-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15170506" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lemaire</div> <div class="given-names">S.D.</div></div>. </div><div class="article-title">The glutaredoxin family in oxygenic photosynthetic organisms</div>, <div class="source">Photosynth. Res.</div> , <div class="year">2004</div>, vol. <div class="volume">79</div> (pg. <div class="fpage">305</div>-<div class="lpage">18</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1023/B:PRES.0000017174.60951.74" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1023/B:PRES.0000017174.60951.74</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20glutaredoxin%20family%20in%20oxygenic%20photosynthetic%20organisms&amp;author=S.D.%20Lemaire&amp;publication_year=2004&amp;journal=Photosynth.%20Res.&amp;volume=79&amp;pages=305-18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/B:PRES.0000017174.60951.74" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FB:PRES.0000017174.60951.74" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FB:PRES.0000017174.60951.74"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16328797" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rouhier</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Couturier</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Jacquot</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Genome-wide analysis of plant glutaredoxin systems</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2006</div>, vol. <div class="volume">57</div> (pg. <div class="fpage">1685</div>-<div class="lpage">96</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/jxb/erl001" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/jxb/erl001</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20plant%20glutaredoxin%20systems&amp;author=N.%20Rouhier&amp;author=J.%20Couturier&amp;author=J.P.%20Jacquot&amp;publication_year=2006&amp;journal=J.%20Exp.%20Bot.&amp;volume=57&amp;pages=1685-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erl001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferl001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferl001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16720602" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rouhier</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Gelhaye</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Sautiere</div> <div class="given-names">P.E.</div></div>, et al. </div><div class="article-title">Isolation and characterization of a new peroxiredoxin from poplar sieve tubes that uses either glutaredoxin or thioredoxin as a proton donor</div>, <div class="source">Plant Physiol.</div> , <div class="year">2001</div>, vol. <div class="volume">127</div> (pg. <div class="fpage">1299</div>-<div class="lpage">309</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.010586" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.010586</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20a%20new%20peroxiredoxin%20from%20poplar%20sieve%20tubes%20that%20uses%20either%20glutaredoxin%20or%20thioredoxin%20as%20a%20proton%20donor&amp;author=N.%20Rouhier&amp;author=E.%20Gelhaye&amp;author=P.E.%20Sautiere&amp;publication_year=2001&amp;journal=Plant%20Physiol.&amp;volume=127&amp;pages=1299-309" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.010586" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.010586" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.010586"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11706208" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rouhier</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Gelhaye</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Jacquot</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Exploring the active site of plant glutaredoxin by site-directed mutagenesis</div>, <div class="source">FEBS Lett.</div> , <div class="year">2002</div>, vol. <div class="volume">511</div> (pg. <div class="fpage">145</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0014-5793(01)03302-6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0014-5793(01)03302-6</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exploring%20the%20active%20site%20of%20plant%20glutaredoxin%20by%20site-directed%20mutagenesis&amp;author=N.%20Rouhier&amp;author=E.%20Gelhaye&amp;author=J.P.%20Jacquot&amp;publication_year=2002&amp;journal=FEBS%20Lett.&amp;volume=511&amp;pages=145-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0014-5793(01)03302-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0014-5793(01)03302-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0014-5793(01)03302-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11821065" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rouhier</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Unno</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Bandyopadhyay</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Functional, structural, and spectroscopic characterization of a glutathione-ligated [2Fe-2S] cluster in poplar glutaredoxin C1</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2007</div>, vol. <div class="volume">104</div> (pg. <div class="fpage">7379</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0702268104" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0702268104</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%2C%20structural%2C%20and%20spectroscopic%20characterization%20of%20a%20glutathione-ligated%20%5B2Fe-2S%5D%20cluster%20in%20poplar%20glutaredoxin%20C1&amp;author=N.%20Rouhier&amp;author=H.%20Unno&amp;author=S.%20Bandyopadhyay&amp;publication_year=2007&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=104&amp;pages=7379-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0702268104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0702268104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0702268104"> </span></div> </div></div></div></div></div><div content-id="DSQ023C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cheng</div> <div class="given-names">N.H.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">J.Z.</div></div>,  <div class="name"><div class="surname">Brock</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Nelson</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Hirschi</div> <div class="given-names">K.D.</div></div>. </div><div class="article-title">AtGRXcp, an <em>Arabidopsis</em> chloroplastic glutaredoxin, is critical for protection against protein oxidative damage</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2006</div>, vol. <div class="volume">281</div> (pg. <div class="fpage">26280</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1074/jbc.M601354200" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1074/jbc.M601354200</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AtGRXcp%2C%20an%20Arabidopsis%20chloroplastic%20glutaredoxin%2C%20is%20critical%20for%20protection%20against%20protein%20oxidative%20damage&amp;author=N.H.%20Cheng&amp;author=J.Z.%20Liu&amp;author=A.%20Brock&amp;author=R.S.%20Nelson&amp;author=K.D.%20Hirschi&amp;publication_year=2006&amp;journal=J.%20Biol.%20Chem.&amp;volume=281&amp;pages=26280-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M601354200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M601354200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M601354200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16829529" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cheng</div> <div class="given-names">N.H.</div></div>. </div><div class="article-title">AtGRX4, an <em>Arabidopsis</em> chloroplastic monothiol glutaredoxin, is able to suppress yeast <em>grx5</em> mutant phenotypes and respond to oxidative stress</div>, <div class="source">FEBS Lett.</div> , <div class="year">2008</div>, vol. <div class="volume">582</div> (pg. <div class="fpage">848</div>-<div class="lpage">54</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.febslet.2008.02.006" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.febslet.2008.02.006</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=AtGRX4%2C%20an%20Arabidopsis%20chloroplastic%20monothiol%20glutaredoxin%2C%20is%20able%20to%20suppress%20yeast%20grx5%20mutant%20phenotypes%20and%20respond%20to%20oxidative%20stress&amp;author=N.H.%20Cheng&amp;publication_year=2008&amp;journal=FEBS%20Lett.&amp;volume=582&amp;pages=848-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.febslet.2008.02.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.febslet.2008.02.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.febslet.2008.02.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18275854" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xing</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Rosso</div> <div class="given-names">M.G.</div></div>,  <div class="name"><div class="surname">Zachgo</div> <div class="given-names">S.</div></div>. </div><div class="article-title">ROXY1, a member of the plant glutaredoxin family, is required for petal development in <em>Arabidopsis thaliana</em></div>, <div class="source">Development</div> , <div class="year">2005</div>, vol. <div class="volume">132</div> (pg. <div class="fpage">1555</div>-<div class="lpage">65</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1242/dev.01725" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1242/dev.01725</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ROXY1%2C%20a%20member%20of%20the%20plant%20glutaredoxin%20family%2C%20is%20required%20for%20petal%20development%20in%20Arabidopsis%20thaliana&amp;author=S.%20Xing&amp;author=M.G.%20Rosso&amp;author=S.%20Zachgo&amp;publication_year=2005&amp;journal=Development&amp;volume=132&amp;pages=1555-65" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/dev.01725" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fdev.01725" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fdev.01725"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15728668" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xing</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zachgo</div> <div class="given-names">S.</div></div>. </div><div class="article-title"><em>ROXY1</em> and <em>ROXY2</em>, two <em>Arabidopsis</em> glutaredoxin genes, are required for anther development</div>, <div class="source">Plant J.</div> , <div class="year">2008</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">790</div>-<div class="lpage">801</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-313X.2007.03375.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1365-313X.2007.03375.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ROXY1%20and%20ROXY2%2C%20two%20Arabidopsis%20glutaredoxin%20genes%2C%20are%20required%20for%20anther%20development&amp;author=S.%20Xing&amp;author=S.%20Zachgo&amp;publication_year=2008&amp;journal=Plant%20J.&amp;volume=53&amp;pages=790-801" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2008.53.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2008.53.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2008.53.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18036205" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Xing</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Birkenbihl</div> <div class="given-names">R.P.</div></div>,  <div class="name"><div class="surname">Zachgo</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Conserved functions of <em>Arabidopsis</em> and rice CC-type glutaredoxins in flower development and pathogen response</div>, <div class="source">Mol. Plant</div> , <div class="year">2009</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">323</div>-<div class="lpage">35</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/mp/ssn078" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/mp/ssn078</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conserved%20functions%20of%20Arabidopsis%20and%20rice%20CC-type%20glutaredoxins%20in%20flower%20development%20and%20pathogen%20response&amp;author=Z.%20Wang&amp;author=S.%20Xing&amp;author=R.P.%20Birkenbihl&amp;author=S.%20Zachgo&amp;publication_year=2009&amp;journal=Mol.%20Plant&amp;volume=2&amp;pages=323-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/mp/ssn078" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmp%2Fssn078" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmp%2Fssn078"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19825617" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ndamukong</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Abdallat</div> <div class="given-names">A.A.</div></div>,  <div class="name"><div class="surname">Thurow</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">SA-inducible <em>Arabidopsis</em> glutaredoxin interacts with TGA factors and suppresses JA-responsive PDF1.2 transcription</div>, <div class="source">Plant J.</div> , <div class="year">2007</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">128</div>-<div class="lpage">39</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-313X.2007.03039.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1365-313X.2007.03039.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SA-inducible%20Arabidopsis%20glutaredoxin%20interacts%20with%20TGA%20factors%20and%20suppresses%20JA-responsive%20PDF1.2%20transcription&amp;author=I.%20Ndamukong&amp;author=A.A.%20Abdallat&amp;author=C.%20Thurow&amp;publication_year=2007&amp;journal=Plant%20J.&amp;volume=50&amp;pages=128-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2007.03039.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2007.03039.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2007.03039.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17397508" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Szederkenyi</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Komor</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Schobert</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Cloning of the cDNA for glutaredoxin, an abundant sieve-tube exudate protein from <em>Ricinus communis</em> L. and characterisation of the glutathione-dependent thiol-reduction system in sieve tubes</div>, <div class="source">Planta</div> , <div class="year">1997</div>, vol. <div class="volume">202</div> (pg. <div class="fpage">349</div>-<div class="lpage">56</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s004250050137" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s004250050137</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20of%20the%20cDNA%20for%20glutaredoxin%2C%20an%20abundant%20sieve-tube%20exudate%20protein%20from%20Ricinus%20communis%20L.%20and%20characterisation%20of%20the%20glutathione-dependent%20thiol-reduction%20system%20in%20sieve%20tubes&amp;author=J.%20Szederkenyi&amp;author=E.%20Komor&amp;author=C.%20Schobert&amp;publication_year=1997&amp;journal=Planta&amp;volume=202&amp;pages=349-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s004250050137" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs004250050137" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs004250050137"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9232906" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Minakuchi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yabushita</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Masumura</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ichihara</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Tanaka</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Cloning and sequence analysis of a cDNA encoding rice glutaredoxin</div>, <div class="source">FEBS Lett.</div> , <div class="year">1994</div>, vol. <div class="volume">337</div> (pg. <div class="fpage">157</div>-<div class="lpage">60</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0014-5793(94)80264-5" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0014-5793(94)80264-5</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20sequence%20analysis%20of%20a%20cDNA%20encoding%20rice%20glutaredoxin&amp;author=K.%20Minakuchi&amp;author=T.%20Yabushita&amp;author=T.%20Masumura&amp;author=K.%20Ichihara&amp;author=K.%20Tanaka&amp;publication_year=1994&amp;journal=FEBS%20Lett.&amp;volume=337&amp;pages=157-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-5793(94)80264-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-5793(94)80264-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-5793(94)80264-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8287970" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sha</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Minakuchi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Higaki</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Purification and characterization of glutaredoxin (thioltransferase) from rice (<em>Oryza sativa</em> L.)</div>, <div class="source">J. Biochem.</div> , <div class="year">1997</div>, vol. <div class="volume">121</div> (pg. <div class="fpage">842</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Purification%20and%20characterization%20of%20glutaredoxin%20(thioltransferase)%20from%20rice%20(Oryza%20sativa%20L.)&amp;author=S.%20Sha&amp;author=K.%20Minakuchi&amp;author=N.%20Higaki&amp;publication_year=1997&amp;journal=J.%20Biochem.&amp;volume=121&amp;pages=842-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.jbchem.a021663" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.jbchem.a021663" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.jbchem.a021663"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9192723" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Emanuelsson</div> <div class="given-names">O.</div></div>,  <div class="name"><div class="surname">Brunak</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">von Heijne</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Nielsen</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Locating proteins in the cell using TargetP, SignalP and related tools</div>, <div class="source">Nat. Protoc.</div> , <div class="year">2007</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">953</div>-<div class="lpage">71</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nprot.2007.131" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nprot.2007.131</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Locating%20proteins%20in%20the%20cell%20using%20TargetP%2C%20SignalP%20and%20related%20tools&amp;author=O.%20Emanuelsson&amp;author=S.%20Brunak&amp;author=G.%20von%20Heijne&amp;author=H.%20Nielsen&amp;publication_year=2007&amp;journal=Nat.%20Protoc.&amp;volume=2&amp;pages=953-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2007.131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2007.131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2007.131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17446895" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bailey</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Elkan</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Fitting a mixture model by expectation maximization to discover motifs in biopolymers</div>, <div class="year">1994</div><div class="conf-name">Proceedings of the Second International Conference on Intelligent Systems for Molecular Biology</div><div class="publisher-loc">Menlo Park, CA</div><div class="publisher-name">AAAI Press</div>(pg. <div class="fpage">28</div>-<div class="lpage">36</div>)</div></div></div></div><div content-id="DSQ023C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nobuta</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Venu</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">An expression atlas of rice mRNAs and small RNAs</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2007</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">473</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt1291" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt1291</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20expression%20atlas%20of%20rice%20mRNAs%20and%20small%20RNAs&amp;author=K.%20Nobuta&amp;author=R.C.%20Venu&amp;author=C.%20Lu&amp;publication_year=2007&amp;journal=Nat.%20Biotechnol.&amp;volume=25&amp;pages=473-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1291" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1291" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1291"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17351617" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nijhawan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Arora</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">F-box proteins in rice. Genome-wide analysis, classification, temporal and spatial gene expression during panicle and seed development, and regulation by light and abiotic stress</div>, <div class="source">Plant Physiol.</div> , <div class="year">2007</div>, vol. <div class="volume">143</div> (pg. <div class="fpage">1467</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.106.091900" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.106.091900</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=F-box%20proteins%20in%20rice.%20Genome-wide%20analysis%2C%20classification%2C%20temporal%20and%20spatial%20gene%20expression%20during%20panicle%20and%20seed%20development%2C%20and%20regulation%20by%20light%20and%20abiotic%20stress&amp;author=M.%20Jain&amp;author=A.%20Nijhawan&amp;author=R.%20Arora&amp;publication_year=2007&amp;journal=Plant%20Physiol.&amp;volume=143&amp;pages=1467-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.106.091900" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.106.091900" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.106.091900"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17293439" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Norton</div> <div class="given-names">G.J.</div></div>,  <div class="name"><div class="surname">Lou-Hing</div> <div class="given-names">D.E.</div></div>,  <div class="name"><div class="surname">Meharg</div> <div class="given-names">A.A.</div></div>,  <div class="name"><div class="surname">Price</div> <div class="given-names">A.H.</div></div>. </div><div class="article-title">Rice-arsenate interactions in hydroponics: whole genome transcriptional analysis</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2008</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">2267</div>-<div class="lpage">76</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/jxb/ern097" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/jxb/ern097</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rice-arsenate%20interactions%20in%20hydroponics%3A%20whole%20genome%20transcriptional%20analysis&amp;author=G.J.%20Norton&amp;author=D.E.%20Lou-Hing&amp;author=A.A.%20Meharg&amp;author=A.H.%20Price&amp;publication_year=2008&amp;journal=J.%20Exp.%20Bot.&amp;volume=59&amp;pages=2267-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/ern097" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Fern097" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Fern097"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18453530" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Khurana</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Transcript profiling reveals diverse roles of auxin-responsive genes during reproductive development and abiotic stress in rice</div>, <div class="source">FEBS J.</div> , <div class="year">2009</div>, vol. <div class="volume">276</div> (pg. <div class="fpage">3148</div>-<div class="lpage">62</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1742-4658.2009.07033.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1742-4658.2009.07033.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20profiling%20reveals%20diverse%20roles%20of%20auxin-responsive%20genes%20during%20reproductive%20development%20and%20abiotic%20stress%20in%20rice&amp;author=M.%20Jain&amp;author=J.P.%20Khurana&amp;publication_year=2009&amp;journal=FEBS%20J.&amp;volume=276&amp;pages=3148-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/ejb.2009.276.issue-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fejb.2009.276.issue-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fejb.2009.276.issue-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19490115" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ribot</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Hirsch</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Balzergue</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Susceptibility of rice to the blast fungus, <em>Magnaporthe grisea</em></div>, <div class="source">J. Plant Physiol.</div> , <div class="year">2008</div>, vol. <div class="volume">165</div> (pg. <div class="fpage">114</div>-<div class="lpage">24</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.jplph.2007.06.013" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.jplph.2007.06.013</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Susceptibility%20of%20rice%20to%20the%20blast%20fungus%2C%20Magnaporthe%20grisea&amp;author=C.%20Ribot&amp;author=J.%20Hirsch&amp;author=S.%20Balzergue&amp;publication_year=2008&amp;journal=J.%20Plant%20Physiol.&amp;volume=165&amp;pages=114-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jplph.2007.06.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jplph.2007.06.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jplph.2007.06.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17905473" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Swarbrick</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Liu</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Slate</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Press</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Scholes</div> <div class="given-names">J.D.</div></div>. </div><div class="article-title">Global patterns of gene expression in rice cultivars undergoing a susceptible or resistant interaction with the parasitic plant <em>Striga hermonthica</em></div>, <div class="source">New Phytol.</div> , <div class="year">2008</div>, vol. <div class="volume">179</div> (pg. <div class="fpage">515</div>-<div class="lpage">29</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1469-8137.2008.02484.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1469-8137.2008.02484.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20patterns%20of%20gene%20expression%20in%20rice%20cultivars%20undergoing%20a%20susceptible%20or%20resistant%20interaction%20with%20the%20parasitic%20plant%20Striga%20hermonthica&amp;author=P.J.%20Swarbrick&amp;author=K.%20Huang&amp;author=G.%20Liu&amp;author=J.%20Slate&amp;author=M.C.%20Press&amp;author=J.D.%20Scholes&amp;publication_year=2008&amp;journal=New%20Phytol.&amp;volume=179&amp;pages=515-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2008.179.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2008.179.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2008.179.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19086183" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Nijhawan</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Khurana</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Validation of housekeeping genes as internal control for studying gene expression in rice by quantitative real-time PCR</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">2006</div>, vol. <div class="volume">345</div> (pg. <div class="fpage">646</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.bbrc.2006.04.140" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.bbrc.2006.04.140</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Validation%20of%20housekeeping%20genes%20as%20internal%20control%20for%20studying%20gene%20expression%20in%20rice%20by%20quantitative%20real-time%20PCR&amp;author=M.%20Jain&amp;author=A.%20Nijhawan&amp;author=A.K.%20Tyagi&amp;author=J.P.%20Khurana&amp;publication_year=2006&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=345&amp;pages=646-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbrc.2006.04.140" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbrc.2006.04.140" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbrc.2006.04.140"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16690022" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sali</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Blundell</div> <div class="given-names">T.L.</div></div>. </div><div class="article-title">Comparative protein modelling by satisfaction of spatial restraints</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1993</div>, vol. <div class="volume">234</div> (pg. <div class="fpage">779</div>-<div class="lpage">815</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1006/jmbi.1993.1626" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1006/jmbi.1993.1626</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20protein%20modelling%20by%20satisfaction%20of%20spatial%20restraints&amp;author=A.%20Sali&amp;author=T.L.%20Blundell&amp;publication_year=1993&amp;journal=J.%20Mol.%20Biol.&amp;volume=234&amp;pages=779-815" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/jmbi.1993.1626" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fjmbi.1993.1626" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fjmbi.1993.1626"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8254673" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Laskowski</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">MacArthur</div> <div class="given-names">M.W.</div></div>,  <div class="name"><div class="surname">Moss</div> <div class="given-names">D.S.</div></div>,  <div class="name"><div class="surname">Thornton</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">PROCHECK: a program to check the stereochemical quality of protein structures</div>, <div class="source">J. Appl. Crystallogr.</div> , <div class="year">1993</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">283</div>-<div class="lpage">91</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1107/S0021889892009944" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1107/S0021889892009944</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PROCHECK%3A%20a%20program%20to%20check%20the%20stereochemical%20quality%20of%20protein%20structures&amp;author=R.A.%20Laskowski&amp;author=M.W.%20MacArthur&amp;author=D.S.%20Moss&amp;author=J.M.%20Thornton&amp;publication_year=1993&amp;journal=J.%20Appl.%20Crystallogr.&amp;volume=26&amp;pages=283-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1107/S0021889892009944" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1107%2FS0021889892009944" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1107%2FS0021889892009944"> </span></div> </div></div></div></div></div><div content-id="DSQ023C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xing</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lauri</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Zachgo</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Redox regulation and flower development: a novel function for glutaredoxins</div>, <div class="source">Plant Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">547</div>-<div class="lpage">55</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1055/s-2006-924278" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1055/s-2006-924278</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Redox%20regulation%20and%20flower%20development%3A%20a%20novel%20function%20for%20glutaredoxins&amp;author=S.%20Xing&amp;author=A.%20Lauri&amp;author=S.%20Zachgo&amp;publication_year=2006&amp;journal=Plant%20Biol.&amp;volume=8&amp;pages=547-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1055/s-2006-924278" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1055%2Fs-2006-924278" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1055%2Fs-2006-924278"> </span></div> </div></div></div></div></div><div content-id="DSQ023C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Navrot</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Gelhaye</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Jacquot</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Rouhier</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Identification of a new family of plant proteins loosely related to glutaredoxins with four CxxC motives</div>, <div class="source">Photosynth. Res.</div> , <div class="year">2006</div>, vol. <div class="volume">89</div> (pg. <div class="fpage">71</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11120-006-9083-7" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11120-006-9083-7</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20new%20family%20of%20plant%20proteins%20loosely%20related%20to%20glutaredoxins%20with%20four%20CxxC%20motives&amp;author=N.%20Navrot&amp;author=E.%20Gelhaye&amp;author=J.P.%20Jacquot&amp;author=N.%20Rouhier&amp;publication_year=2006&amp;journal=Photosynth.%20Res.&amp;volume=89&amp;pages=71-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11120-006-9083-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11120-006-9083-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11120-006-9083-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16915354" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Khurana</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Khurana</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Genome-wide analysis of intronless genes in rice and <em>Arabidopsis</em></div>, <div class="source">Funct. Integr. Genomics</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">69</div>-<div class="lpage">78</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s10142-007-0052-9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s10142-007-0052-9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20intronless%20genes%20in%20rice%20and%20Arabidopsis&amp;author=M.%20Jain&amp;author=P.%20Khurana&amp;author=A.K.%20Tyagi&amp;author=J.P.%20Khurana&amp;publication_year=2008&amp;journal=Funct.%20Integr.%20Genomics&amp;volume=8&amp;pages=69-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10142-007-0052-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10142-007-0052-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10142-007-0052-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17578610" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lecharny</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Boudet</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Gy</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Aubourg</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kreis</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Introns in, introns out in plant gene families: a genomic approach of the dynamics of gene structure</div>, <div class="source">J. Struct. Funct. Genomics</div> , <div class="year">2003</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">111</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1023/A:1022614001371" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1023/A:1022614001371</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Introns%20in%2C%20introns%20out%20in%20plant%20gene%20families%3A%20a%20genomic%20approach%20of%20the%20dynamics%20of%20gene%20structure&amp;author=A.%20Lecharny&amp;author=N.%20Boudet&amp;author=I.%20Gy&amp;author=S.%20Aubourg&amp;author=M.%20Kreis&amp;publication_year=2003&amp;journal=J.%20Struct.%20Funct.%20Genomics&amp;volume=3&amp;pages=111-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1022614001371" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1022614001371" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1022614001371"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12836690" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lurin</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Andres</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Aubourg</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Genome-wide analysis of <em>Arabidopsis</em> pentatricopeptide repeat proteins reveals their essential role in organelle biogenesis</div>, <div class="source">Plant Cell</div> , <div class="year">2004</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">2089</div>-<div class="lpage">103</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1105/tpc.104.022236" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1105/tpc.104.022236</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%20of%20Arabidopsis%20pentatricopeptide%20repeat%20proteins%20reveals%20their%20essential%20role%20in%20organelle%20biogenesis&amp;author=C.%20Lurin&amp;author=C.%20Andres&amp;author=S.%20Aubourg&amp;publication_year=2004&amp;journal=Plant%20Cell&amp;volume=16&amp;pages=2089-103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.104.022236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.104.022236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.104.022236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15269332" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Khurana</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Genome-wide analysis, evolutionary expansion, and expression of early auxin-responsive <em>SAUR</em> gene family in rice (<em>Oryza sativa</em>)</div>, <div class="source">Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">360</div>-<div class="lpage">71</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.ygeno.2006.04.008" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.ygeno.2006.04.008</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20analysis%2C%20evolutionary%20expansion%2C%20and%20expression%20of%20early%20auxin-responsive%20SAUR%20gene%20family%20in%20rice%20(Oryza%20sativa)&amp;author=M.%20Jain&amp;author=A.K.%20Tyagi&amp;author=J.P.%20Khurana&amp;publication_year=2006&amp;journal=Genomics&amp;volume=88&amp;pages=360-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2006.04.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2006.04.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2006.04.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16707243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ziemann</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Bhave</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Zachgo</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Origin and diversification of land plant CC-type glutaredoxins</div>, <div class="source">Genome Biol. Evol.</div> , <div class="year">2009</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">265</div>-<div class="lpage">77</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/gbe/evp025" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/gbe/evp025</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Origin%20and%20diversification%20of%20land%20plant%20CC-type%20glutaredoxins&amp;author=M.%20Ziemann&amp;author=M.%20Bhave&amp;author=S.%20Zachgo&amp;publication_year=2009&amp;journal=Genome%20Biol.%20Evol.&amp;volume=1&amp;pages=265-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evp025" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevp025" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevp025"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20333196" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lauri</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ziemann</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Busch</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bhave</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Zachgo</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Nuclear activity of ROXY1, a glutaredoxin interacting with TGA factors, is required for petal development in <em>Arabidopsis thaliana</em></div>, <div class="source">Plant Cell</div> , <div class="year">2009</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">429</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1105/tpc.108.064477" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1105/tpc.108.064477</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20activity%20of%20ROXY1%2C%20a%20glutaredoxin%20interacting%20with%20TGA%20factors%2C%20is%20required%20for%20petal%20development%20in%20Arabidopsis%20thaliana&amp;author=S.%20Li&amp;author=A.%20Lauri&amp;author=M.%20Ziemann&amp;author=A.%20Busch&amp;author=M.%20Bhave&amp;author=S.%20Zachgo&amp;publication_year=2009&amp;journal=Plant%20Cell&amp;volume=21&amp;pages=429-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1105/tpc.108.064477" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1105%2Ftpc.108.064477" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1105%2Ftpc.108.064477"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19218396" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Couturier</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Koh</div> <div class="given-names">C.S.</div></div>,  <div class="name"><div class="surname">Zaffagnini</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Structure-function relationship of the chloroplastic glutaredoxin S12 with an atypical WCSYS active site</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2009</div>, vol. <div class="volume">284</div> (pg. <div class="fpage">9299</div>-<div class="lpage">310</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1074/jbc.M807998200" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1074/jbc.M807998200</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structure-function%20relationship%20of%20the%20chloroplastic%20glutaredoxin%20S12%20with%20an%20atypical%20WCSYS%20active%20site&amp;author=J.%20Couturier&amp;author=C.S.%20Koh&amp;author=M.%20Zaffagnini&amp;publication_year=2009&amp;journal=J.%20Biol.%20Chem.&amp;volume=284&amp;pages=9299-310" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M807998200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M807998200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M807998200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19158074" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morell</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Follmann</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Haberlein</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Identification and localization of the first glutaredoxin in leaves of a higher plant</div>, <div class="source">FEBS Lett.</div> , <div class="year">1995</div>, vol. <div class="volume">369</div> (pg. <div class="fpage">149</div>-<div class="lpage">52</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0014-5793(95)00690-B" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0014-5793(95)00690-B</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20localization%20of%20the%20first%20glutaredoxin%20in%20leaves%20of%20a%20higher%20plant&amp;author=S.%20Morell&amp;author=H.%20Follmann&amp;author=I.%20Haberlein&amp;publication_year=1995&amp;journal=FEBS%20Lett.&amp;volume=369&amp;pages=149-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-5793(95)00690-B" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-5793(95)00690-B" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-5793(95)00690-B"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7649248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cheng</div> <div class="given-names">N.H.</div></div>,  <div class="name"><div class="surname">Hirschi</div> <div class="given-names">K.D.</div></div>. </div><div class="article-title">Cloning and characterization of CXIP1, a novel PICOT domain-containing <em>Arabidopsis</em> protein that associates with CAX1</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">2003</div>, vol. <div class="volume">278</div> (pg. <div class="fpage">6503</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1074/jbc.M210883200" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1074/jbc.M210883200</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20characterization%20of%20CXIP1%2C%20a%20novel%20PICOT%20domain-containing%20Arabidopsis%20protein%20that%20associates%20with%20CAX1&amp;author=N.H.%20Cheng&amp;author=K.D.%20Hirschi&amp;publication_year=2003&amp;journal=J.%20Biol.%20Chem.&amp;volume=278&amp;pages=6503-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.M210883200" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.M210883200" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.M210883200"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12480930" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sundaram</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Wu</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">L.Q.</div></div>,  <div class="name"><div class="surname">Rathinasabapathi</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Expression of a <em>Pteris vittata</em> glutaredoxin <em>PvGRX5</em> in transgenic <em>Arabidopsis thaliana</em> increases plant arsenic tolerance and decreases arsenic accumulation in the leaves</div>, <div class="source">Plant Cell Environ.</div> , <div class="year">2009</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">851</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-3040.2009.01963.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1365-3040.2009.01963.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Expression%20of%20a%20Pteris%20vittata%20glutaredoxin%20PvGRX5%20in%20transgenic%20Arabidopsis%20thaliana%20increases%20plant%20arsenic%20tolerance%20and%20decreases%20arsenic%20accumulation%20in%20the%20leaves&amp;author=S.%20Sundaram&amp;author=S.%20Wu&amp;author=L.Q.%20Ma&amp;author=B.%20Rathinasabapathi&amp;publication_year=2009&amp;journal=Plant%20Cell%20Environ.&amp;volume=32&amp;pages=851-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pce.2009.32.issue-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpce.2009.32.issue-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpce.2009.32.issue-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19236608" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qiu</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Xiao</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Xie</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Rice gene network inferred from expression profiling of plants overexpressing OsWRKY13, a positive regulator of disease resistance</div>, <div class="source">Mol. Plant</div> , <div class="year">2008</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">538</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/mp/ssn012" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/mp/ssn012</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rice%20gene%20network%20inferred%20from%20expression%20profiling%20of%20plants%20overexpressing%20OsWRKY13%2C%20a%20positive%20regulator%20of%20disease%20resistance&amp;author=D.%20Qiu&amp;author=J.%20Xiao&amp;author=W.%20Xie&amp;publication_year=2008&amp;journal=Mol.%20Plant&amp;volume=1&amp;pages=538-51" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19825559" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rouhier</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Villarejo</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Srivastava</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Identification of plant glutaredoxin targets</div>, <div class="source">Antioxid. Redox Signal.</div> , <div class="year">2005</div>, vol. <div class="volume">7</div> (pg. <div class="fpage">919</div>-<div class="lpage">29</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1089/ars.2005.7.919" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1089/ars.2005.7.919</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20plant%20glutaredoxin%20targets&amp;author=N.%20Rouhier&amp;author=A.%20Villarejo&amp;author=M.%20Srivastava&amp;publication_year=2005&amp;journal=Antioxid.%20Redox%20Signal.&amp;volume=7&amp;pages=919-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/ars.2005.7.919" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fars.2005.7.919" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fars.2005.7.919"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15998247" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSQ023C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSQ023C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lynch</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Conery</div> <div class="given-names">J.S.</div></div>. </div><div class="article-title">The evolutionary fate and consequences of duplicate genes</div>, <div class="source">Science</div> , <div class="year">2000</div>, vol. <div class="volume">290</div> (pg. <div class="fpage">1151</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.290.5494.1151" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.290.5494.1151</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20evolutionary%20fate%20and%20consequences%20of%20duplicate%20genes&amp;author=M.%20Lynch&amp;author=J.S.%20Conery&amp;publication_year=2000&amp;journal=Science&amp;volume=290&amp;pages=1151-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.290.5494.1151" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.290.5494.1151" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.290.5494.1151"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11073452" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2010. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=484440&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq023-dsq023supp_table9.xls?Expires=2147483647&amp;Signature=J5k5ZUzxzrogHVjMA8ESKjMznys~jGdyLTJ2kzC2~0XTAysR~nuS5xoLpYfY9QIjQqrTeFxqvPiF4SV0h-ChgoUCgHqhjj1lIW7ksq~HQaFB7Zv1OBP7~Bd4OG4O1~et3uOmqBHfPEmiu9O6QXlSp15tstpVQgpxbaerK~~9x8L7dr1wEbQhTw6u4BcQ68EcayiwAFSH~NwsxtsF9vcSb5N7F7qJjn9Xu9bHeHCbp2fQQlCs6b2W3ofXbNPEjh-wgiYpVQ9zvadbUgzVApJwAhEz5d60uqKERPm-RV1sibcrzXMZUmK97ys-7XjfMoPEW135hqi-KIHaVFzHLxdI1g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-15</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq023-dsq023supp_table8.xls?Expires=2147483647&amp;Signature=kragTDKl~387mekjNebiwZp54TdkXr5z0DohrZs0K3CWLnUZTrj3d13pQBV3FZtWGvTUvCUS4SgkieJa0K4D0Pmk4xNWbOKZlOlVKmOBYlHkApspIFcZ1~7sbarlgdgWU2zYPDFrsTHPFWcTNLf5BVqsyYMT10Jjhysye-ImLK6vMWHZENbSCH~lI3tPRXaDEfGOPfJVE~dX0H~geBhBWAyXZxONMb2pUhM1b0B3inUsmGsVoe4jDc7TRPjMQoDWy4M6mrwNimjymujH~dQ5EClXhS623Cwv-V9QfC3uI-5aXqaCLac0FGz4Sy6zELQCr0uV4e17QHuInqasOvgMHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-14</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq023-dsq023supp_table7.xls?Expires=2147483647&amp;Signature=StpYBaqJOVOzPPK2iXi4GfKjY5GK5gqOSe7x1vd-ICaAJVmngsXnWMVOTcRmgC8rVinixeCYjbVtznIxzpe9CkTWIocqWXIFezNerS7LZMpIwT-36OC8Xmr8NOPLniF4h60oDOhiMhjK6III1aP1cpTc0rAGQx3wuPNxEcwyM0MnEy5iX208O-XwU8E6CVB3UW5k~10IJMgwOlhLF9qoy5h6qWOiIxGH~fHkW2KcnPPbR5kZj6-dKtBv9fWEq3GrxwWc2zjTMNR0PkoHLBPddCFd1wCZHbg~XWxpvGezs8X1AL~RgAwTUb0-9jK7WH1I0CsEG6ROskAWn1Lo-VfG-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-13</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq023-dsq023supp_table6.doc?Expires=2147483647&amp;Signature=snjJ3Bc9VDM5lseqBFwyY7ToTC51PfslXdhKA-FW7lH-VyyMT0movu9RZqTiQmmXHrm9pwM8bb3fexuy848skCpyqAT4Ovxyz6Ug-IFnkPK3PReSU1i6GWtnhXbekHf7eXD9ZNdKPtKBcFOWFcw9K~UHU1Nxu9Vqy5JNZKjRNfiUMDK4H~0tfS2-9KsOmYuFokh35vHqQm7hSFRChmJYhGUYwR7UJUnnpe2alL6sK9NXXFEGFML7n18frIbwmQAbwKqvq5i1PEpmu7UzxP2CG6E9472EIAg7SIg2LACfKIY6VnLiHmgorG2y8TyMFTMNFu4CxR741rT-Y7C0towc7Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-12</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq023-dsq023supp_table5.xls?Expires=2147483647&amp;Signature=AFZl9AGPnFljRp1KD5t7IBmRUCk8tranJA9l13tjJvph7CYSgb6ZrASCEMa6S~NSTgRsALLr0-1XLi70wQCjrLdYIOZLTf9lmZUr2pHg3M~5r25cbwxXL2GUFoM1rR-Bli3V-BErQn29-h9~ylbo0Xh6JXxx0B6frxLTPjA2R9LA~jU3yRY7R15wUMCKzBFDeIcqdF0MWVrT5s-XGrNkt7X~HGo~YgxCAwWaaOvJS4VmzOmRM3tlfEgt6OcFzPqfToaFu2FmRKoQe0a0tZtm9J7bgwxnrZxKhmL1G~15bsZCx8RX-FX6rPv1FLNfQNmQJbLRN0tFDVHRbHslv9ttGw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-11</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq023-dsq023supp_table4.xls?Expires=2147483647&amp;Signature=WgL3b3x7jsPTvjktWJtMRdlrojPHWXMvzyyK43085cWaSvxyxyn1ZAbWZqH9PkamTwyo0q5S-3NxRkeSHeOdJkNfwlCWusPJcTY91vf4ACxt~~1TlsDN84nTP80r2FKjUqwzUdjV9PBsL3Gy5jBTtudbEX8IH7UrhqS54EMk17pI3WnIi1g2rFQiqLIKxuQE~Y3P3K5crFGmPnX0bpKmb8bUUpflzPLu-~~oLsFZHiphVcpOrCyQfoXIQ01Uj7tvsVQQw3pMzwjQnVcwNikMhdwnyNOGJJrd9~MWh3aVi8bvs3H8j0c6G~L~TXAoWs3uwE0U4z3l0AAh-uWVOgTmLA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-10</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq023-dsq023supp_table3.doc?Expires=2147483647&amp;Signature=u~iNkpJq335NK1dcpyc8AC2muMgHqY88PH-BtPZC49FnYclG69iVc6H14hgJXy2VXD6F1khnO-bwfYwVYt-CAglHbNNdFfy8~gU2FnNOOGsFsbR9U4F6gPyEYn0mXCq7V0DEjwfzpr5ELstkOYUAUGn3IoYQdcYrchaxUHYPiPlfVP7lFPoxn6dSRbHwQJ7q4vzz2sCzO5x3wLmuNMjWpi7-FODDtXQdACfxsYIMsG2Yxr-tkrpGJGIwvzbiZ4TVllYMxzlQckc2dlNLu57OLCBTAzsR7jDHSu1Z4BqkU~fVm64gl00Ob8OBmqccf-PEgq~YyhcoLDllKg014Vxybw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq023-dsq023supp_table2.doc?Expires=2147483647&amp;Signature=FGmrgttfV55PNGgJfMnsl9zqqgi1L9m6X56Bculpq9aV8G5NrfgO2otz0UgzpnRDc9oY3TXfsgK3y7mqyHWJfSHplDzTT1x9H5R2OUUm3gLElF-zWsic-2bttz3TKsjlBrwuxQ3ViId-sKhnrRNUubWh7pluZwHwmrruFWaSYtrqCtTMqEa9QxeZIhNZUjXGksX2gVEipju-bg4fiIUj1M5-jZROW2owoJpE3Ddl~J-kOu6ioNjLHrDzT5ZNSTGXqQYs0iEBpC1LibO~u0cBwHL2NKlw9NKynTsg0SSqb7mOeiGRuc~trNuCqDFS~~s8iPJnJgCn9XuAqmFRrZYwBA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq023-dsq023supp_table10.xls?Expires=2147483647&amp;Signature=0AfNFTAlxKl3bihNaUXfVj0M61qEJbK6Vh5m1BdNTfb64eA4g5MaT4WkTiY~gZYunu2rM7bnTScHnjtWrCUJJAsdz3CzvrtbyJr9pSEHQZ~tFvCcV2YhHaLQHMCcHvH21SnpuG7O3JgTLWd64uXxn10DvvXQdnd4uq6Tr3qpRNFi6ETn8~PNtj-ig2fd0D3ovIgdC6gQjEEtMVfIKd9~Ihct5kY5~hMgy7x3nzhxROabiOVzZkdUfMga2rqi9BXzj77vz1GrCOEgWq1rs8AZ70Wxjj2HRVzsyhp0jXPYsDbOGIY6XxXf6shKgRzZ4MTy2t~MIs4LT2ztTwkTnd4KrA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq023-dsq023supp_table1.doc?Expires=2147483647&amp;Signature=WFU4YL0HQfTzN27H4n8F9tn-493It9mhrYH2IAAbWypsrYtJCJIH37efm31er3LYi21mErCvqo5W8Ucw~8i5ndQ3phGP56QjW-BKRmhV0sBk~MfA6wYLAUzu2pTsq83HHk~7IhWcJB3PRccrIESjHS9RG8I8RGbF6i6akJqhToM~S6dvLyT9rVm~hny0cpgeGIDtHyD~YCBR1H047nTBKAz~~JesrqaNasP~zT5Lzr~RP498MKNzxol4ARX-CAXs8NPpvrQOCdeLDwtnxlmT9wolaXwXrjdlM5xstDap9TvBX8cldtBNJYE9kxtqvhf5yiq3Pufnxzc9zeO8NyczTA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq023-dsq023supp_fig5.ppt?Expires=2147483647&amp;Signature=UjfARVI6b~RGpatZNkSSgEC-t1SAQyliT2sJ-zO8KA1Va1SwxXragXWfvvMPhOTarnrWV0LN9GXWczIWpTAz1-rNPXxYa2iwr8vfkSW8bIP42dWLAd0DczTwmsjMG6wwfEs6pyKNhQPL8vCLbbdHkkkCpBdesNztNXwXBcOczMALt-FgqGnV7q8fEC1Eqf~RzkEh~hMiboGBuZa~h8faPCQtNigrjKGHj2vIpYFsrbBklUVVQKb-uALjmNjEwrO47MzGAM7~Vfq-cPjRC90ONQdh5y~neQYxXW1vCbfuyIOYVcIfNEdfApaQuEfH9jRG6r-U1nv6qax~~l5Vm9qGjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq023-dsq023supp_fig4.doc?Expires=2147483647&amp;Signature=rFHeieyRG7UiBVSZMBFx3sylF8YCMZw-copGAuquVAyDX81CGv6zDfNMt5We6gaRnTLajKnYOO7cVVU5QCLmjXxPy2r-Y5pQftXjrV3GJ9XihRKMnMDKYMpdLhnvjaB~jECRiXklBig4RRlfDnX8kLb87kyrMutGyLAumMaY7oEtl8wpsKk0nr74OushfOwN5urJF6FQn4okcLifZb2QDSK8XCLjNL8USUypY4MNUXdrfF2qi18girpqZwkodztPz-Emb~o-bSPQJJsfyth~Y6xb-56b-YZMmJPmENT2cD0NOEr2ZEa~HhghjOwtxmdIYV9vUKaG4uSAyqcKzaJZBw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq023-dsq023supp_fig3.ppt?Expires=2147483647&amp;Signature=HNBG12~c1I3~oqI7hrd2oj~G9tsZKbOdrHne37jmS~6BTPZgicw0e6i3HOKGut2NOjWCw-6z~vuuH8T~Crw5m5HVEf2LJ-0E9plPnvs~n-jJfPD8fHDxf4COABxhoh4O8MXC79sjT1dlp-2J-0pM6iIX9mT6SPn0jJj32jTOzOScq-bEE0G6wt0QWiwGKPMK5gAGy6D8NS8doIQZOrhEbKZSvIahoAsv9efMwmiitx4ih0iG6Ma2voFiepJinchwDTv3ffRAz8u2D9S9Dssqqipb5W0MmeXn1VrJmnEDFmeXIxIORvoC4SCp4Ud2T8iO5iV4lTxiGx8NWk4BZTi6bQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq023-dsq023supp_fig2.ppt?Expires=2147483647&amp;Signature=ivwXIqnvgmOY5NGOWM-j0yogI08wncgwjkWwhQOkkJXkUYAu9tYV5VPYbH24uC-FD9uKJtywjHhzxNr0H2obfIU7mfvziIH--EnjpnCCyrAD4MO3OqYNj1Ve8jLXqNS~zqCxMUl-Avj7IpW1BI3rVeIAtm0oWfHU75gAhYhkRWX2j-UXWaaBHeXCwZkvpdYRfDS0S~Nui6k~dyLYq-yQPTKIz8IhTfmbk1XEPzm3kKUbutESv55VeJjgl-Y5Qw38bYeX8kzrhj5k~Lq19dUHcm3ZuCn0vEHFlFoZ5kju~0KDm6Dt123x5BaefbTWDWFcwd8WMrsyLkki8ERx5bp90A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/17/6/10.1093/dnares/dsq023/2/dsq023-dsq023supp_fig1.ppt?Expires=2147483647&amp;Signature=mO~66c6OCds1DxeaaY0bjv47UMyB8S7KVYjE2ZYFpJ1uKoe7vpNVWD4zgkfrMw9qnGEp2hP1o-vH~anHA-B-DqY1zqiQsSvPACA4zeuVWsDZABjx-1WyQqHPs5d3i9li1UkzyExEtWI3Xts2CGc324qaSirRsgut5~kWXX9W14e95BdtvtJnJ1qFl1z5PTwZ6TOYrbMFau~MdlM-UgAr5IFk5QDwgApWXD~tCUT-9z~IZ2MGigOIBzV1DG8t1ZARHbvuD7eHJMTm8MDZNCXTSRIJRbYOWTLqV2IPBupFwkRm2xLqd23quIvn4WJR4N0w6i-lt5ZtMsX-tOTVsVBPGw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - ppt file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">268</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000285800200003&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">30</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsq023"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="484440"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000285800200003&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f17%2f6%2f353%2f484440">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000285800200003&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (30)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F17%2F6%2F353%2F484440" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/484440'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome-Wide Survey and Expression Analysis Suggest Diverse Roles of Glutaredoxin Gene Family Members During Development and Response to Various Stimuli in Rice" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsq023");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f17%2f6%2f353%2f484440">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

