

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/4/353/4922629","@type":"ScholarlyArticle","name":"Homologue-specific chromosome sequencing characterizes translocation junctions and permits allelic assignment","datePublished":"2018-03-06","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2018-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsy007","keywords":["chromosome sorting","allelic profile","fusion gene","breakpoint junctions","variant frequency"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/4/353/4922629","author":[{"name":"Kasai, Fumio","affiliation":"Japanese Collection of Research Bioresources (JCRB) Cell Bank, Laboratory of Cell Cultures, National Institutes of Biomedical Innovation, Health and Nutrition, Ibaraki, Osaka, Japan  Cambridge Resource Centre for Comparative Genomics, Department of Veterinary Medicine, University of Cambridge, Cambridge, UK","@type":"Person"},{"name":"Pereira, Jorge C","affiliation":"Cambridge Resource Centre for Comparative Genomics, Department of Veterinary Medicine, University of Cambridge, Cambridge, UK","@type":"Person"},{"name":"Kohara, Arihiro","affiliation":"Japanese Collection of Research Bioresources (JCRB) Cell Bank, Laboratory of Cell Cultures, National Institutes of Biomedical Innovation, Health and Nutrition, Ibaraki, Osaka, Japan","@type":"Person"},{"name":"Ferguson-Smith, Malcolm A","affiliation":"Cambridge Resource Centre for Comparative Genomics, Department of Veterinary Medicine, University of Cambridge, Cambridge, UK","@type":"Person"}],"description":"Abstract.  Chromosome translocations can be detected by cytogenetic analysis, but it is hard to characterize the breakpoints at the sequence level. Chromosome s","pageStart":"353","pageEnd":"360","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Homologue-specific chromosome sequencing characterizes translocation junctions and permits allelic assignment","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Homologue-specific chromosome sequencing characterizes translocation junctions and permits allelic assignment" />
    <meta property="og:description" content="Abstract.  Chromosome translocations can be detected by cytogenetic analysis, but it is hard to characterize the breakpoints at the sequence level. Chromosome s" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/4/353/4922629" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2018-03-06" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Kasai, Fumio" /><meta name="citation_author_institution" content="Japanese Collection of Research Bioresources (JCRB) Cell Bank, Laboratory of Cell Cultures, National Institutes of Biomedical Innovation, Health and Nutrition, Ibaraki, Osaka, Japan" /><meta name="citation_author_institution" content="Cambridge Resource Centre for Comparative Genomics, Department of Veterinary Medicine, University of Cambridge, Cambridge, UK" /><meta name="citation_author" content="Pereira, Jorge C" /><meta name="citation_author_institution" content="Cambridge Resource Centre for Comparative Genomics, Department of Veterinary Medicine, University of Cambridge, Cambridge, UK" /><meta name="citation_author" content="Kohara, Arihiro" /><meta name="citation_author_institution" content="Japanese Collection of Research Bioresources (JCRB) Cell Bank, Laboratory of Cell Cultures, National Institutes of Biomedical Innovation, Health and Nutrition, Ibaraki, Osaka, Japan" /><meta name="citation_author" content="Ferguson-Smith, Malcolm A" /><meta name="citation_author_institution" content="Cambridge Resource Centre for Comparative Genomics, Department of Veterinary Medicine, University of Cambridge, Cambridge, UK" /><meta name="citation_title" content="Homologue-specific chromosome sequencing characterizes translocation junctions and permits allelic assignment" /><meta name="citation_firstpage" content="353" /><meta name="citation_lastpage" content="360" /><meta name="citation_doi" content="10.1093/dnares/dsy007" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2018/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Next-generation sequencing and large genome assemblies;  citation_journal_title=Pharmacogenomics;  citation_year=2012;  citation_volume=13;  citation_pages=901-15; " /><meta name="citation_reference" content="citation_title=A survey of tools for variant analysis of next-generation genome sequencing data,;  citation_journal_title=Brief. Bioinformatics;  citation_year=2014;  citation_volume=15;  citation_pages=256-78; " /><meta name="citation_reference" content="citation_title=The cancer cell line encyclopedia enables predictive modelling of anticancer drug sensitivity;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=483;  citation_pages=603-7; " /><meta name="citation_reference" content="citation_title=COSMIC: somatic cancer genetics at high-resolution;  citation_journal_title=Nucleic Acids Res;  citation_year=2017;  citation_volume=45;  citation_pages=D777-83; " /><meta name="citation_reference" content="citation_title=Tumor heterog eneity: causes and consequences;  citation_journal_title=Biochim. Biophys. Acta;  citation_year=2010;  citation_volume=1805;  citation_pages=105-17; " /><meta name="citation_reference" content="citation_title=Clonal evolution in cancer;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=481;  citation_pages=306-13; " /><meta name="citation_reference" content="citation_title=Changes of heterogeneous cell population in the Ishikawa cell line during long-term culture; proposal of an in vitro clonal evolution model of tumor cells;  citation_journal_title=Genomics;  citation_year=2016;  citation_volume=107;  citation_pages=259-66; " /><meta name="citation_reference" content="citation_title=Single-cell mutational profiling and clonal phylogeny in cancer;  citation_journal_title=Genome Res;  citation_year=2013;  citation_volume=23;  citation_pages=2115-25; " /><meta name="citation_reference" content="citation_title=Study of X chromosome abnormality in XX males using bivariate flow karyotype analysis and flow sorted dot blots;  citation_journal_title=Cytometry;  citation_year=1990;  citation_volume=11;  citation_pages=202-7; " /><meta name="citation_reference" content="citation_title=Reassessment of genome size in turtle and crocodile based on chromosome measurement by flow karyotyping: close similarity to chicken;  citation_journal_title=Biol. Lett;  citation_year=2012;  citation_volume=8;  citation_pages=631-5; " /><meta name="citation_reference" content="citation_title=Afrotheria genome; overestimation of genome size and distinct chromosome GC content revealed by flow karyotyping;  citation_journal_title=Genomics;  citation_year=2013;  citation_volume=102;  citation_pages=468-71; " /><meta name="citation_reference" content="citation_title=Genetic analysis by chromosome sorting and painting: phylogenetic and diagnostic applications;  citation_journal_title=Eur. J. Hum. Genet;  citation_year=1997;  citation_volume=5;  citation_pages=253-65; " /><meta name="citation_reference" content="citation_title=Emergence of complex rearrangements at translocation breakpoints in a transgenic mouse; implications for mechanisms involved in the formation of chromosome rearrangements;  citation_journal_title=Cytogenet. Genome Res;  citation_year=2007;  citation_volume=119;  citation_pages=83-90; " /><meta name="citation_reference" content="citation_title=Reverse chromosome painting: a method for the rapid analysis of aberrant chromosomes in clinical cytogenetics;  citation_journal_title=J. Med. Genet;  citation_year=1992;  citation_volume=29;  citation_pages=299-307; " /><meta name="citation_reference" content="citation_title=Comparative mapping using chromosome sorting and painting;  citation_journal_title=ILAR J;  citation_year=1998;  citation_volume=39;  citation_pages=68-76; " /><meta name="citation_reference" content="citation_title=Gestational experience alters sex allocation in the subsequent generation;  citation_journal_title=R. Soc. Open Sci;  citation_year=2016;  citation_volume=3;  citation_pages=160210" /><meta name="citation_reference" content="citation_title=Chromosomal phylogeny of Vampyressine bats (Chiroptera, Phyllostomidae) with description of two new sex chromosome systems;  citation_journal_title=BMC Evol. Biol;  citation_year=2016;  citation_volume=16;  citation_pages=119" /><meta name="citation_reference" content="citation_title=Establishment of a new human endometrial adenocarcinoma cell line, Ishikawa cells, containing estrogen and progesterone receptors;  citation_journal_title=Nihon Sanka Fujinka Gakkai Zasshi;  citation_year=1985;  citation_volume=37;  citation_pages=1103-11; " /><meta name="citation_reference" content="citation_title=Causes of oncogenic chromosomal translocation;  citation_journal_title=Trends Genet;  citation_year=2006;  citation_volume=22;  citation_pages=46-55; " /><meta name="citation_reference" content="citation_title=The biogenesis of chromosome translocations;  citation_journal_title=Nat. Cell Biol;  citation_year=2014;  citation_volume=16;  citation_pages=293-300; " /><meta name="citation_reference" content="citation_title=Precise detection of chromosomal translocation or inversion breakpoints by whole-genome sequencing;  citation_journal_title=J. Hum. Genet;  citation_year=2014;  citation_volume=59;  citation_pages=649-54; " /><meta name="citation_reference" content="citation_title=FusionMap: detecting fusion genes from next-generation sequencing data at base-pair resolution;  citation_journal_title=Bioinformatics,;  citation_year=2011;  citation_volume=27;  citation_pages=1922-8; " /><meta name="citation_reference" content="citation_title=Protracted dormancy of pre-leukemic stem cells;  citation_journal_title=Leukemia;  citation_year=2015;  citation_volume=29;  citation_pages=2202-7; " /><meta name="citation_reference" content="citation_title=Identification of SPAG9 as a novel JAK2 fusion partner gene in pediatric acute lymphoblastic leukemia with t(9; 17)(p24; q21);  citation_journal_title=Genes Chromosomes Cancer;  citation_year=2015;  citation_volume=54;  citation_pages=401-8; " /><meta name="citation_reference" content="citation_title=bHLH-PAS proteins in cancer;  citation_journal_title=Nat. Rev. Cancer;  citation_year=2013;  citation_volume=13;  citation_pages=827-41; " /><meta name="citation_reference" content="citation_title=NPAS3 demonstrates features of a tumor suppressive role in driving the progression of Astrocytomas;  citation_journal_title=Am. J. Pathol;  citation_year=2011;  citation_volume=179;  citation_pages=462-76; " /><meta name="citation_reference" content="citation_title=The impact of chromosome sorting and painting on the comparative analysis of primate genomes;  citation_journal_title=Cytogenet. Genome Res;  citation_year=2005;  citation_volume=108;  citation_pages=112-21; " /><meta name="citation_reference" content="citation_title=Extensive homology of chicken macrochromosomes in the karyotypes of Trachemys scripta elegans and Crocodylus niloticus revealed by chromosome painting despite long divergence times;  citation_journal_title=Cytogenet. Genome Res;  citation_year=2012;  citation_volume=136;  citation_pages=303-7; " /><meta name="citation_reference" content="citation_title=A complete comparative chromosome map for the dog, red fox, and human and its integration with canine genetic maps;  citation_journal_title=Genomics;  citation_year=1999;  citation_volume=62;  citation_pages=189-202; " /><meta name="citation_reference" content="citation_title=Dog chromosome-specific paints reveal evolutionary inter- and intrachromosomal rearrangements in the American mink and human;  citation_journal_title=Cytogenet. Cell Genet;  citation_year=2000;  citation_volume=90;  citation_pages=275-8; " /><meta name="citation_reference" content="citation_title=Molecular cytogenetic definition of the chicken genome: the first complete avian karyotype;  citation_journal_title=Genetics;  citation_year=2004;  citation_volume=166;  citation_pages=1367-73; " /><meta name="citation_reference" content="citation_title=Completely phased genome sequencing through chromosome sorting;  citation_journal_title=Proc. Natl Acad. Sci. U S A;  citation_year=2011;  citation_volume=108;  citation_pages=12-7; " /><meta name="citation_reference" content="citation_title=Array painting: a protocol for the rapid analysis of aberrant chromosomes using DNA microarrays;  citation_journal_title=Nat. Protoc;  citation_year=2009;  citation_volume=4;  citation_pages=1722-36; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/4/353/25501615/dsy007.pdf" /><meta name="description" content="Abstract.  Chromosome translocations can be detected by cytogenetic analysis, but it is hard to characterize the breakpoints at the sequence level. Chromosome s" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/4/353/4922629" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/4/353/4922629" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/4/353/4922629");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsy007';
    </script>
<title>
	Homologue-specific chromosome sequencing characterizes translocation junctions and permits allelic assignment | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4922629" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYFVPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXNhaSwgRnVtaW8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJKYXBhbmVzZSBDb2xsZWN0aW9uIG9mIFJlc2VhcmNoIEJpb3Jlc291cmNlcyAoSkNSQikgQ2VsbCBCYW5rLCBMYWJvcmF0b3J5IG9mIENlbGwgQ3VsdHVyZXMsIE5hdGlvbmFsIEluc3RpdHV0ZXMgb2YgQmlvbWVkaWNhbCBJbm5vdmF0aW9uLCBIZWFsdGggYW5kIE51dHJpdGlvbiwgSWJhcmFraSwgT3Nha2EsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2FtYnJpZGdlIFJlc291cmNlIENlbnRyZSBmb3IgQ29tcGFyYXRpdmUgR2Vub21pY3MsIERlcGFydG1lbnQgb2YgVmV0ZXJpbmFyeSBNZWRpY2luZSwgVW5pdmVyc2l0eSBvZiBDYW1icmlkZ2UsIENhbWJyaWRnZSwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJQZXJlaXJhLCBKb3JnZSBDIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2FtYnJpZGdlIFJlc291cmNlIENlbnRyZSBmb3IgQ29tcGFyYXRpdmUgR2Vub21pY3MsIERlcGFydG1lbnQgb2YgVmV0ZXJpbmFyeSBNZWRpY2luZSwgVW5pdmVyc2l0eSBvZiBDYW1icmlkZ2UsIENhbWJyaWRnZSwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb2hhcmEsIEFyaWhpcm8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJKYXBhbmVzZSBDb2xsZWN0aW9uIG9mIFJlc2VhcmNoIEJpb3Jlc291cmNlcyAoSkNSQikgQ2VsbCBCYW5rLCBMYWJvcmF0b3J5IG9mIENlbGwgQ3VsdHVyZXMsIE5hdGlvbmFsIEluc3RpdHV0ZXMgb2YgQmlvbWVkaWNhbCBJbm5vdmF0aW9uLCBIZWFsdGggYW5kIE51dHJpdGlvbiwgSWJhcmFraSwgT3Nha2EsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iRmVyZ3Vzb24tU21pdGgsIE1hbGNvbG0gQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNhbWJyaWRnZSBSZXNvdXJjZSBDZW50cmUgZm9yIENvbXBhcmF0aXZlIEdlbm9taWNzLCBEZXBhcnRtZW50IG9mIFZldGVyaW5hcnkgTWVkaWNpbmUsIFVuaXZlcnNpdHkgb2YgQ2FtYnJpZGdlLCBDYW1icmlkZ2UsIFVLIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJIb21vbG9ndWUtc3BlY2lmaWMgY2hyb21vc29tZSBzZXF1ZW5jaW5nIGNoYXJhY3Rlcml6ZXMgdHJhbnNsb2NhdGlvbiBqdW5jdGlvbnMgYW5kIHBlcm1pdHMgYWxsZWxpYyBhc3NpZ25tZW50IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzUzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzNjAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3kwMDciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxOC8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nIGFuZCBsYXJnZSBnZW5vbWUgYXNzZW1ibGllczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGhhcm1hY29nZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTkwMS0xNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdXJ2ZXkgb2YgdG9vbHMgZm9yIHZhcmlhbnQgYW5hbHlzaXMgb2YgbmV4dC1nZW5lcmF0aW9uIGdlbm9tZSBzZXF1ZW5jaW5nIGRhdGEsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CcmllZi4gQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz0yNTYtNzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjYW5jZXIgY2VsbCBsaW5lIGVuY3ljbG9wZWRpYSBlbmFibGVzIHByZWRpY3RpdmUgbW9kZWxsaW5nIG9mIGFudGljYW5jZXIgZHJ1ZyBzZW5zaXRpdml0eTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTQ4MzsgIGNpdGF0aW9uX3BhZ2VzPTYwMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DT1NNSUM6IHNvbWF0aWMgY2FuY2VyIGdlbmV0aWNzIGF0IGhpZ2gtcmVzb2x1dGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz1ENzc3LTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UdW1vciBoZXRlcm9nIGVuZWl0eTogY2F1c2VzIGFuZCBjb25zZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2NoaW0uIEJpb3BoeXMuIEFjdGE7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTgwNTsgIGNpdGF0aW9uX3BhZ2VzPTEwNS0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2xvbmFsIGV2b2x1dGlvbiBpbiBjYW5jZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00ODE7ICBjaXRhdGlvbl9wYWdlcz0zMDYtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYW5nZXMgb2YgaGV0ZXJvZ2VuZW91cyBjZWxsIHBvcHVsYXRpb24gaW4gdGhlIElzaGlrYXdhIGNlbGwgbGluZSBkdXJpbmcgbG9uZy10ZXJtIGN1bHR1cmU7IHByb3Bvc2FsIG9mIGFuIGluIHZpdHJvIGNsb25hbCBldm9sdXRpb24gbW9kZWwgb2YgdHVtb3IgY2VsbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTEwNzsgIGNpdGF0aW9uX3BhZ2VzPTI1OS02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2luZ2xlLWNlbGwgbXV0YXRpb25hbCBwcm9maWxpbmcgYW5kIGNsb25hbCBwaHlsb2dlbnkgaW4gY2FuY2VyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTIzOyAgY2l0YXRpb25fcGFnZXM9MjExNS0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3R1ZHkgb2YgWCBjaHJvbW9zb21lIGFibm9ybWFsaXR5IGluIFhYIG1hbGVzIHVzaW5nIGJpdmFyaWF0ZSBmbG93IGthcnlvdHlwZSBhbmFseXNpcyBhbmQgZmxvdyBzb3J0ZWQgZG90IGJsb3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DeXRvbWV0cnk7ICBjaXRhdGlvbl95ZWFyPTE5OTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0yMDItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVhc3Nlc3NtZW50IG9mIGdlbm9tZSBzaXplIGluIHR1cnRsZSBhbmQgY3JvY29kaWxlIGJhc2VkIG9uIGNocm9tb3NvbWUgbWVhc3VyZW1lbnQgYnkgZmxvdyBrYXJ5b3R5cGluZzogY2xvc2Ugc2ltaWxhcml0eSB0byBjaGlja2VuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9sLiBMZXR0OyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz02MzEtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWZyb3RoZXJpYSBnZW5vbWU7IG92ZXJlc3RpbWF0aW9uIG9mIGdlbm9tZSBzaXplIGFuZCBkaXN0aW5jdCBjaHJvbW9zb21lIEdDIGNvbnRlbnQgcmV2ZWFsZWQgYnkgZmxvdyBrYXJ5b3R5cGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTAyOyAgY2l0YXRpb25fcGFnZXM9NDY4LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGFuYWx5c2lzIGJ5IGNocm9tb3NvbWUgc29ydGluZyBhbmQgcGFpbnRpbmc6IHBoeWxvZ2VuZXRpYyBhbmQgZGlhZ25vc3RpYyBhcHBsaWNhdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1ci4gSi4gSHVtLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MjUzLTY1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbWVyZ2VuY2Ugb2YgY29tcGxleCByZWFycmFuZ2VtZW50cyBhdCB0cmFuc2xvY2F0aW9uIGJyZWFrcG9pbnRzIGluIGEgdHJhbnNnZW5pYyBtb3VzZTsgaW1wbGljYXRpb25zIGZvciBtZWNoYW5pc21zIGludm9sdmVkIGluIHRoZSBmb3JtYXRpb24gb2YgY2hyb21vc29tZSByZWFycmFuZ2VtZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3l0b2dlbmV0LiBHZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTExOTsgIGNpdGF0aW9uX3BhZ2VzPTgzLTkwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZXZlcnNlIGNocm9tb3NvbWUgcGFpbnRpbmc6IGEgbWV0aG9kIGZvciB0aGUgcmFwaWQgYW5hbHlzaXMgb2YgYWJlcnJhbnQgY2hyb21vc29tZXMgaW4gY2xpbmljYWwgY3l0b2dlbmV0aWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBNZWQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0xOTkyOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9Mjk5LTMwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgbWFwcGluZyB1c2luZyBjaHJvbW9zb21lIHNvcnRpbmcgYW5kIHBhaW50aW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1JTEFSIEo7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz02OC03NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VzdGF0aW9uYWwgZXhwZXJpZW5jZSBhbHRlcnMgc2V4IGFsbG9jYXRpb24gaW4gdGhlIHN1YnNlcXVlbnQgZ2VuZXJhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Ui4gU29jLiBPcGVuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTYwMjEwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q2hyb21vc29tYWwgcGh5bG9nZW55IG9mIFZhbXB5cmVzc2luZSBiYXRzIChDaGlyb3B0ZXJhLCBQaHlsbG9zdG9taWRhZSkgd2l0aCBkZXNjcmlwdGlvbiBvZiB0d28gbmV3IHNleCBjaHJvbW9zb21lIHN5c3RlbXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBFdm9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXN0YWJsaXNobWVudCBvZiBhIG5ldyBodW1hbiBlbmRvbWV0cmlhbCBhZGVub2NhcmNpbm9tYSBjZWxsIGxpbmUsIElzaGlrYXdhIGNlbGxzLCBjb250YWluaW5nIGVzdHJvZ2VuIGFuZCBwcm9nZXN0ZXJvbmUgcmVjZXB0b3JzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OaWhvbiBTYW5rYSBGdWppbmthIEdha2thaSBaYXNzaGk7ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0xMTAzLTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYXVzZXMgb2Ygb25jb2dlbmljIGNocm9tb3NvbWFsIHRyYW5zbG9jYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZW5kcyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTQ2LTU1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYmlvZ2VuZXNpcyBvZiBjaHJvbW9zb21lIHRyYW5zbG9jYXRpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENlbGwgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTI5My0zMDA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWNpc2UgZGV0ZWN0aW9uIG9mIGNocm9tb3NvbWFsIHRyYW5zbG9jYXRpb24gb3IgaW52ZXJzaW9uIGJyZWFrcG9pbnRzIGJ5IHdob2xlLWdlbm9tZSBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBIdW0uIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTU5OyAgY2l0YXRpb25fcGFnZXM9NjQ5LTU0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdXNpb25NYXA6IGRldGVjdGluZyBmdXNpb24gZ2VuZXMgZnJvbSBuZXh0LWdlbmVyYXRpb24gc2VxdWVuY2luZyBkYXRhIGF0IGJhc2UtcGFpciByZXNvbHV0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljcyw7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0xOTIyLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb3RyYWN0ZWQgZG9ybWFuY3kgb2YgcHJlLWxldWtlbWljIHN0ZW0gY2VsbHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUxldWtlbWlhOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MjIwMi03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBvZiBTUEFHOSBhcyBhIG5vdmVsIEpBSzIgZnVzaW9uIHBhcnRuZXIgZ2VuZSBpbiBwZWRpYXRyaWMgYWN1dGUgbHltcGhvYmxhc3RpYyBsZXVrZW1pYSB3aXRoIHQoOTsgMTcpKHAyNDsgcTIxKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXMgQ2hyb21vc29tZXMgQ2FuY2VyOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTU0OyAgY2l0YXRpb25fcGFnZXM9NDAxLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPWJITEgtUEFTIHByb3RlaW5zIGluIGNhbmNlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIENhbmNlcjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTgyNy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TlBBUzMgZGVtb25zdHJhdGVzIGZlYXR1cmVzIG9mIGEgdHVtb3Igc3VwcHJlc3NpdmUgcm9sZSBpbiBkcml2aW5nIHRoZSBwcm9ncmVzc2lvbiBvZiBBc3Ryb2N5dG9tYXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtLiBKLiBQYXRob2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTc5OyAgY2l0YXRpb25fcGFnZXM9NDYyLTc2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaW1wYWN0IG9mIGNocm9tb3NvbWUgc29ydGluZyBhbmQgcGFpbnRpbmcgb24gdGhlIGNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIHByaW1hdGUgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3l0b2dlbmV0LiBHZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTExMi0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXh0ZW5zaXZlIGhvbW9sb2d5IG9mIGNoaWNrZW4gbWFjcm9jaHJvbW9zb21lcyBpbiB0aGUga2FyeW90eXBlcyBvZiBUcmFjaGVteXMgc2NyaXB0YSBlbGVnYW5zIGFuZCBDcm9jb2R5bHVzIG5pbG90aWN1cyByZXZlYWxlZCBieSBjaHJvbW9zb21lIHBhaW50aW5nIGRlc3BpdGUgbG9uZyBkaXZlcmdlbmNlIHRpbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DeXRvZ2VuZXQuIEdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM2OyAgY2l0YXRpb25fcGFnZXM9MzAzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgY29tcGxldGUgY29tcGFyYXRpdmUgY2hyb21vc29tZSBtYXAgZm9yIHRoZSBkb2csIHJlZCBmb3gsIGFuZCBodW1hbiBhbmQgaXRzIGludGVncmF0aW9uIHdpdGggY2FuaW5lIGdlbmV0aWMgbWFwczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9NjI7ICBjaXRhdGlvbl9wYWdlcz0xODktMjAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Eb2cgY2hyb21vc29tZS1zcGVjaWZpYyBwYWludHMgcmV2ZWFsIGV2b2x1dGlvbmFyeSBpbnRlci0gYW5kIGludHJhY2hyb21vc29tYWwgcmVhcnJhbmdlbWVudHMgaW4gdGhlIEFtZXJpY2FuIG1pbmsgYW5kIGh1bWFuOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DeXRvZ2VuZXQuIENlbGwgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9OTA7ICBjaXRhdGlvbl9wYWdlcz0yNzUtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGN5dG9nZW5ldGljIGRlZmluaXRpb24gb2YgdGhlIGNoaWNrZW4gZ2Vub21lOiB0aGUgZmlyc3QgY29tcGxldGUgYXZpYW4ga2FyeW90eXBlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xNjY7ICBjaXRhdGlvbl9wYWdlcz0xMzY3LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wbGV0ZWx5IHBoYXNlZCBnZW5vbWUgc2VxdWVuY2luZyB0aHJvdWdoIGNocm9tb3NvbWUgc29ydGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UHJvYy4gTmF0bCBBY2FkLiBTY2kuIFUgUyBBOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEwODsgIGNpdGF0aW9uX3BhZ2VzPTEyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFycmF5IHBhaW50aW5nOiBhIHByb3RvY29sIGZvciB0aGUgcmFwaWQgYW5hbHlzaXMgb2YgYWJlcnJhbnQgY2hyb21vc29tZXMgdXNpbmcgRE5BIG1pY3JvYXJyYXlzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFByb3RvYzsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MTcyMi0zNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzI1LzQvMzUzLzI1NTAxNjE1L2RzeTAwNy5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgQ2hyb21vc29tZSB0cmFuc2xvY2F0aW9ucyBjYW4gYmUgZGV0ZWN0ZWQgYnkgY3l0b2dlbmV0aWMgYW5hbHlzaXMsIGJ1dCBpdCBpcyBoYXJkIHRvIGNoYXJhY3Rlcml6ZSB0aGUgYnJlYWtwb2ludHMgYXQgdGhlIHNlcXVlbmNlIGxldmVsLiBDaHJvbW9zb21lIHMiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQ5MjI2MjkmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkCv09g6NzlFvvjsP6LwUgiJb85Ms=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f353%2f4922629" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f353%2f4922629" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=109685" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/4/1/m_dnares_25_4cover.png?Expires=2147483647&amp;Signature=lD6dhAB99HpTltGbT4-zO-h9Z6msKpBpjdlZuCIFAdO0RvvEM~IgMElNp5bE6Rdf~OSfdWzB2y8SIAkqvb6Fc-0GRLo0HVHIHIXLjJF8ZJ94ZrTCEDv2Gn-92BvjK~Ebx3LV-ePsMx8j4josbKy1FscEJ3-qj3zB2TEHnK~iAQqommn5aOw82hhC1dOH5f1P15MUh9o1QIOhJ9WAnJwCFYDxRE0ZmdX~qCpKaFm8XPRL-O71Rxc1eeV8iTb9yQzgpbw2OwX1g4mQ5iALY~7Bk3L2RgUkQZIzgNqb8ANqfHjAsIlrzFHcKAl7tTiKtoGbsHvDqkXNPcKjgjrdKdIXQw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/4">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="120121753">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121753">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120121755">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121755">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120121759">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121759">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120121772">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121772">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120121787">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121787">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120121793">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121793">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="120121797">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#120121797">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/4/343/4850982">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/4/361/4939498">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Homologue-specific chromosome sequencing characterizes translocation junctions and permits allelic assignment
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fumio Kasai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Japanese Collection of Research Bioresources (JCRB) Cell Bank, Laboratory of Cell Cultures, National Institutes of Biomedical Innovation, Health and Nutrition, Ibaraki, Osaka, Japan</div><div class="aff">Cambridge Resource Centre for Comparative Genomics, Department of Veterinary Medicine, University of Cambridge, Cambridge, UK</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsy007-cor1" data-open="dsy007-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81 72 641 9851. Fax. +81 72 641 9859. Email: <a href="mailto:k-230@umin.ac.jp" target="_blank">k-230@umin.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fumio+Kasai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kasai F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kasai F&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Fumio Kasai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jorge C Pereira</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Cambridge Resource Centre for Comparative Genomics, Department of Veterinary Medicine, University of Cambridge, Cambridge, UK</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jorge+C+Pereira">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pereira J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pereira J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jorge C Pereira</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Arihiro Kohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Japanese Collection of Research Bioresources (JCRB) Cell Bank, Laboratory of Cell Cultures, National Institutes of Biomedical Innovation, Health and Nutrition, Ibaraki, Osaka, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Arihiro+Kohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohara A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohara A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Arihiro Kohara</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Malcolm A Ferguson-Smith</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Cambridge Resource Centre for Comparative Genomics, Department of Veterinary Medicine, University of Cambridge, Cambridge, UK</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Malcolm+A+Ferguson-Smith">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ferguson-Smith M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ferguson-Smith M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Malcolm A Ferguson-Smith</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsy007-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Yuji Kohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 4, 1 August 2018, Pages 353–360, <a href='https://doi.org/10.1093/dnares/dsy007'>https://doi.org/10.1093/dnares/dsy007</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">06 March 2018</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 August 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">12 February 2018</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4922629" href="/dnaresearch/article-pdf/25/4/353/25501615/dsy007.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Fumio Kasai, Jorge C Pereira, Arihiro Kohara, Malcolm A Ferguson-Smith;  Homologue-specific chromosome sequencing characterizes translocation junctions and permits allelic assignment, <em>DNA Research</em>, Volume 25, Issue 4, 1 August 2018, Pages 353–360, <a href='https://doi.org/10.1093/dnares/dsy007'>https://doi.org/10.1093/dnares/dsy007</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4922629?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4922629?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4922629?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4922629?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4922629?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4922629?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4922629?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Homologue-specific%20chromosome%20sequencing%20characterizes%20translocation%20junctions%20and%20permits%20allelic%20assignment&amp;publicationDate=2018-03-06&amp;volumeNum=25&amp;issueNum=4&amp;author=Kasai%2C%20Fumio%3B%20Pereira%2C%20Jorge%20C&amp;startPage=353&amp;endPage=360&amp;contentId=10.1093%2Fdnares%2Fdsy007&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f353%2f4922629" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=120121753 id="120121753" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Chromosome translocations can be detected by cytogenetic analysis, but it is hard to characterize the breakpoints at the sequence level. Chromosome sorting by flow cytometry produces flow karyotypes that enable the isolation of abnormal chromosomes and the generation of chromosome-specific DNA. In this study, a derivative chromosome t(9; 14) and its homologous normal chromosomes 9 and 14 from the Ishikawa 3-H-12 cell line were sorted to collect homologue-specific samples. Chromosome sequencing identified the breakpoint junction in the der(9) at 9p24.3 and 14q13.1 and uncovered the formation of a fusion gene, WASH1–NPAS3. Amplicon sequencing targeted for neighbouring genes at the fusion breakpoint revealed that the variant frequencies correlate with the allelic copy number. Sequencing of sorted chromosomes permits the assignment of allelic variants and can lead to the characterization of abnormal chromosomes. We show that allele-specific chromosome sequencing of homologues is a robust technique for distinguishing alleles and this provides an efficient approach for the comprehensive analysis of genomic changes.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;chromosome sorting&quot;">chromosome sorting</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;allelic profile&quot;">allelic profile</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;fusion gene&quot;">fusion gene</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;breakpoint junctions&quot;">breakpoint junctions</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;variant frequency&quot;">variant frequency</a></div>                            <h2 scrollto-destination=120121755 id="120121755" class="section-title" >1. Introduction</h2>
<p>Next-generation sequencing provides whole genome analysis of structural variants as well as sequence variants.<a reveal-id="dsy007-B1" data-open="dsy007-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsy007-B2" data-open="dsy007-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> The technique has made a significant contribution to cancer genomics and to the analysis of human cancer cell lines that have been accumulated in the Cancer Cell Line Encyclopedia<a reveal-id="dsy007-B3" data-open="dsy007-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> and the Catalogue of Somatic Mutations in Cancer.<a reveal-id="dsy007-B4" data-open="dsy007-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> It is noted that clinical tumour samples are often contaminated with normal cells and show heterogeneity due to clonal evolution<a reveal-id="dsy007-B5" data-open="dsy007-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a><sup>,</sup><a reveal-id="dsy007-B6" data-open="dsy007-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> which is also observed in tumour cell lines.<a reveal-id="dsy007-B7" data-open="dsy007-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> These two features present problems in sequencing tumour samples. Although recent advances in single cell analysis can distinguish differences between cells and help to uncover genetic heterogeneity in tumour cells,<a reveal-id="dsy007-B8" data-open="dsy007-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> the investigation of allelic differences remains challenging.</p><p>Chromosomes can be distinguished in fluid suspensions using a bivariate fluorescence-activated cell sorter which separates them by size and base pair ratio and displays the results in the form of a flow karyotype,<a reveal-id="dsy007-B9" data-open="dsy007-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> allowing accurate estimation of chromosome size and GC content.<a reveal-id="dsy007-B10" data-open="dsy007-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsy007-B11" data-open="dsy007-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> The procedure enables the isolation and collection not only of aberrant chromosomes but sometimes also of normal homologues that contain measurable differences in repetitive DNA.<a reveal-id="dsy007-B12" data-open="dsy007-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Analysis of sorted chromosomes can characterize cryptic allelic variants occurring in a reciprocal chromosome translocation.<a reveal-id="dsy007-B13" data-open="dsy007-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> Sorted chromosome samples have been extensively exploited in diagnostic cytogenetics and in the production of chromosome-specific DNA for chromosome painting<a reveal-id="dsy007-B14" data-open="dsy007-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a><sup>,</sup><a reveal-id="dsy007-B15" data-open="dsy007-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> and chromosome sequencing.<a reveal-id="dsy007-B16" data-open="dsy007-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><a reveal-id="dsy007-B17" data-open="dsy007-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> The latter serves to focus on regions of interest and has application in refining genome analysis and assisting genome assembly.</p><p>In this study, chromosome-specific sequencing reveals the breakpoint junction and identifies a novel fusion gene in a translocation between chromosomes 9 and 14 observed in a human cancer cell line. The sequence analysis of homologous regions on chromosomes separated by flow sorting has enabled alleles at several loci to be distinguished by their nucleotide differences.</p>                            <h2 scrollto-destination=120121759 id="120121759" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=120121760 id="120121760" class="section-title" >2.1. Cell lines and DNA preparation</h3>
<p>The Ishikawa 3-H-12 cell line registered as JCRB1505 was established from an endometrial cancer.<a reveal-id="dsy007-B18" data-open="dsy007-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> Cells were cultured using MEM with 15% non-heat-inactivated foetal bovine serum. Genomic DNA was extracted using the AllPrep DNA/RNA Mini Kit (Qiagen). The karyotypes and SNP array profiles are available from our previous study (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1A</a></span>).<a reveal-id="dsy007-B7" data-open="dsy007-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Because t(9; 14) was detected in all the metaphases examined and therefore expected to be a reference marker for this cell line, this study was focused on der(9).</p>                            <h3 scrollto-destination=120121762 id="120121762" class="section-title" >2.2. Chromosome preparation and staining</h3>
<p>Mitotic cells were collected by shake-off from three 175-cm<sup>2</sup> flasks at passage 25* after colcemid treatment (KaryoMAX Colcemid, Gibco) at a final concentration of 0.1 μg/ml for 5 h. Cells were centrifuged at 250 <em>g</em> for 8 min in 50 ml tubes. The supernatant was discarded and replaced by 10 ml of hypotonic solution (75 mM KCl, 0.2 mM spermine, 0.5 mM spermidine, 10 mM MgSO<sub>4</sub>·7H<sub>2</sub>O, pH 8.0) and incubated at RT for 15 min. After centrifugation, the cell pellet was resuspended in 1 ml of ice-cold polyamine buffer (2 mM EDTA, 0.5 mM EGTA, 80 mM KCl, 0.25% Triton X-100, 0.2 mM spermine, 0.5 mM spermidine, 20 mM NaCl, pH 7.2) and incubated on ice for 10 min. The suspension was then vortexed vigorously for 12 s and the chromosomes were stained by 5 μg/ml of Hoechst 33258 (Sigma-Aldrich, B2883), 40 μg/ml of Chromomycin A3 (Sigma-Aldrich, C2659), and 10 mM MgSO<sub>4</sub>. 10 mM sodium citrate and 25 mM sodium sulphite were added 1 h before flow sorting.</p>                            <h3 scrollto-destination=120121764 id="120121764" class="section-title" >2.3. Chromosome sorting</h3>
<p>Flow cytometric analysis was carried out at the Cambridge Resource Centre for Comparative Genomics. Stained chromosome suspensions were analysed on a flow cytometer (MoFlo, Beckman Coulter, Fullerton, CA, USA) equipped with two water-cooled lasers (Spectra-Physics Model 2020) spatially separated at the flow chamber. The first laser was tuned to emit multiline UV (330–360 nm) which excites Hoechst 33258. The second laser was tuned to emit light at 457 nm to excite Chromomycin A3. The power of both lasers was set to 300 mW and kept constant using light control feedback. Fluorescence emitted from Hoechst 33258 was collected using a 400 nm long pass filter combined with a 480 nm short pass filter. Chromomycin A3 fluorescence was collected using a 490 nm long pass filter. The instrument was configured for two-way sorting with high speed sorting at a data rate of 8,000–12,000 events per second with the optimal setting of the sheath pressure to ∼ 60 psi and the drop drive frequency to ∼ 95 kHz using a 70 μm cytonozzle tip. The flow karyotype was displayed by Hoechst 33258 versus Chromomycin A3 fluorescence and acquired by a total of 50,000–100,000 events at a rate of 1,000 events per second. Data collected from the experiments were analysed using Summit Software Version 4.1 (Beckman Coulter). Approximately 2,000 chromosomes were flow sorted into a 0.5-ml UV-treated PCR tube from each chromosome peak.</p>                            <h3 scrollto-destination=120121766 id="120121766" class="section-title" >2.4. Chromosome sequencing</h3>
<p>For chromosome-specific sequencing, chromosomes were amplified by the REPLI-g single cell kit (Qiagen) and a genomic DNA fragment library was prepared using the Ion Xpress Plus Fragment Library kit (Life Technologies). Emulsion PCR of the library was performed by the Ion One Touch 2 system using the Ion PGM Template OT2 400 kit (Life Technologies). Sequencing was carried out by the Ion PGM sequencer using the Ion PGM Sequencing 400 kit and the Ion 318 Chip (Life Technologies). All protocols followed the manufacturer’s instructions. Alignment of sequencing reads with the human genome reference (GRCh37/hg19) was performed using the Torrent Suite v4.1.2 software. The bam file was mapped to the hg19 reference using the TMAP software v5.0.13 with standard options (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). This was visualized in the Integrative Genomics Viewer (IGV) using the mapping quality value as 1. BLAT search in the UCSC Genome tools was used to find similarity of sequence reads in the genome database.</p>                            <h3 scrollto-destination=120121768 id="120121768" class="section-title" >2.5. Fusion point analysis</h3>
<p>The breakpoint junction was amplified by PCR at an annealing temperature of 65°C using the following primer pairs designed using the Primer3 software; der9-BPf: GGAAGGCTCAGACAGAGAGG and der9-BPr: CTCTCCCTTGCACTCAGCTC. Total RNA was prepared using the AllPrep DNA/RNA Mini Kit (Qiagen) and treated with DNaseI. Reverse transcription of RNA into cDNA was performed using the PrimeScript RT-PCR Kit (Takara) with random hexamers. Amplification of the WASH1–NPAS3 fusion transcript was performed with annealing temperature at 63°C using the PCR primer pair of WASH1f: GGAAGGCTCAGACAGAGAGG and NPAS3r: AGCATCTCGGGATTTCTCCT.</p>                            <h3 scrollto-destination=120121770 id="120121770" class="section-title" >2.6. Variant analysis</h3>
<p>Multiplex primer pools for eight genes at 9p24.3, 10 genes at 14q13 and the breakpoint boundary region at 14q13.1 were obtained using the Ion AmpliSeq Designer and are listed in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_table_s2r.xlsx?Expires=2147483647&amp;Signature=I6WvEWc8z2oHpUJxY-3VJX4pXx1131d7jHlB9mm9QxBvCFX8CF-FLtDwPXaCP4lQbtGK6etF-Y8cuBg6YFgfI5hDqy39ptmjSutiXDq7wRqrMwffaR~3qTLKb~zKFKj5pmA-P2T6K6iB6jYLGtPhPxYG624ob7H4Q6bLbIsA1QbyzRccqclFrG3qyaLLBFszcEP5siAeG2KbHWHqByMOithxJ00msEEQsM2XHE2cVB7rvjaEI5NbT85CWA5x0KK379woINCw5Jrlh2gxEzOc6xrSKb1NoQMO6F7ksAkD8I~eigRejs7sIz4bNkm8j6K3p~tYv~PqreakcgQjakCsxg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. The Ampliseq custom panels for 9p24.3 and 14q13 consist of 118 and 120 amplicons, covering 33,607 and 33,275 bp in total size, respectively. Sequence library and template preparations were performed using the Ion AmpliSeq Kit for Chef DL8 and Ion PGM Hi-Q Chef Kit (Life Technologies), respectively. Sequencing was run on the Ion PGM using the Ion 316 chip. The analysis was carried out using the Variant Caller plug-in in the Torrent Suit and the parameters are given in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&amp;Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>.</p>                            <h2 scrollto-destination=120121772 id="120121772" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=120121773 id="120121773" class="section-title" >3.1. Flow karyotype</h3>
<p>The flow karyotype of the Ishikawa 3-H-12 cell line at passage 25* is shown in <a reveal-id="dsy007-F1" data-open="dsy007-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>, indicating the peaks resulting from chromosome 9, one of two chromosomes 14 (14S) and der(9). Based on array data, the chromosome size of der(9) is estimated to be ∼ 215 Mb. It is contained in an abnormal peak, close to the normal chromosome 3 peak, and weaker than the other normal peaks because it contains only half the number of chromosomes, i.e. only the derivative chromosome rather than the pairs of homologues present in other peaks. Chromosome 14 is heteromorphic and this allows the separation of one of the two chromosome 14 homologues by sorting. In this study, the sorted chromosome 14 is referred to as 14S, and the other unsorted homologue is referred to as 14U. </p>                        <a id="120121775" scrollto-destination="120121775"></a>
<div id="dsy007-F1" data-id="dsy007-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/m_dsy007f1.png?Expires=2147483647&Signature=vl1qWV0E0RM~zKVBzdJXY94IhwnxXE1hOuozuMl~SZHpQGiVgSUq3ESWDmUrCOYANzpgViy8DVTrVTE4cQuYQ5fdQ4KP5yP4keSN8XVG25004rXd0Qy3bqhtUfSHSmoA9LN-MCC8fCGmBaKrTh5MdlI-EMMltqKmMHnWW~VVwMFDUybgrcjLI3LSse5o~0e5XQrt07xkjYqwBxMgpQGFww4bpy2tvl57inE1IkN2uklb7mrmPZEIG-6Z4~GD7E7ccqur4AeDX1~Qy6V5TOK~21ufHhh8qkw156xtxbQszV8oDoUyK7HCOc3CQGIP8TNWLOHWeCSDZ128gSI~VUgYAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flow karyotype of the Ishikawa 3-H-12 cell line at P25* without removing the background manually by image processing was obtained after sorting a larger number of chromosomes than usual to show clearly the der(9) peak close to the normal chromosome 3 peak. The der(9) peak is weaker than other peaks that have pairs of normal chromosomes as it exists in only one copy in each cell. Normal chromosome 9 and one of the chromosome 14 homologues, 14S, are also selected for sorting." path-from-xml="dsy007f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121775/dsy007f1.tif" path-from-xml="dsy007f1.tif" target="_blank">View large</a><a section="120121775" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007f1.png?Expires=2147483647&Signature=x4oMGGx1U6wMoDGnI1CZixKjd6qT~Q0mpDXm3fsisSCfRBflEsNa59xRzO2YJT-tb5erA7sxuTNGlyn~unp59Cb6DVevUQRIkis2uXSTaK1fx4ituP7MD9G8BLcMkRQofv1l9~oIIn73kXAFS4lSyRYfwFj8p~8qOsiQGtIXdrC7NpqopNvwfSH5C2mqd2ANv~RrZUN3~NTkMGzZ0DvviZY8CZc9QGx0l27gCwjn2zpVhgo3Ay64q2Hp-GxOvXRjO2PEFz4zJn2eOLMVO4zClF01~0zw8rU2Wv2jp5WAJaf7p7h6bJnidBFL0-iCtQM11oomfijf-OC0JzpWFUqtfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121775&ar=4922629&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy007f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow karyotype of the Ishikawa 3-H-12 cell line at P25* without removing the background manually by image processing was obtained after sorting a larger number of chromosomes than usual to show clearly the der(9) peak close to the normal chromosome 3 peak. The der(9) peak is weaker than other peaks that have pairs of normal chromosomes as it exists in only one copy in each cell. Normal chromosome 9 and one of the chromosome 14 homologues, 14S, are also selected for sorting.</p></div></div><div content-id="dsy007-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/m_dsy007f1.png?Expires=2147483647&Signature=vl1qWV0E0RM~zKVBzdJXY94IhwnxXE1hOuozuMl~SZHpQGiVgSUq3ESWDmUrCOYANzpgViy8DVTrVTE4cQuYQ5fdQ4KP5yP4keSN8XVG25004rXd0Qy3bqhtUfSHSmoA9LN-MCC8fCGmBaKrTh5MdlI-EMMltqKmMHnWW~VVwMFDUybgrcjLI3LSse5o~0e5XQrt07xkjYqwBxMgpQGFww4bpy2tvl57inE1IkN2uklb7mrmPZEIG-6Z4~GD7E7ccqur4AeDX1~Qy6V5TOK~21ufHhh8qkw156xtxbQszV8oDoUyK7HCOc3CQGIP8TNWLOHWeCSDZ128gSI~VUgYAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flow karyotype of the Ishikawa 3-H-12 cell line at P25* without removing the background manually by image processing was obtained after sorting a larger number of chromosomes than usual to show clearly the der(9) peak close to the normal chromosome 3 peak. The der(9) peak is weaker than other peaks that have pairs of normal chromosomes as it exists in only one copy in each cell. Normal chromosome 9 and one of the chromosome 14 homologues, 14S, are also selected for sorting." path-from-xml="dsy007f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121775/dsy007f1.tif" path-from-xml="dsy007f1.tif" target="_blank">View large</a><a section="120121775" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007f1.png?Expires=2147483647&Signature=x4oMGGx1U6wMoDGnI1CZixKjd6qT~Q0mpDXm3fsisSCfRBflEsNa59xRzO2YJT-tb5erA7sxuTNGlyn~unp59Cb6DVevUQRIkis2uXSTaK1fx4ituP7MD9G8BLcMkRQofv1l9~oIIn73kXAFS4lSyRYfwFj8p~8qOsiQGtIXdrC7NpqopNvwfSH5C2mqd2ANv~RrZUN3~NTkMGzZ0DvviZY8CZc9QGx0l27gCwjn2zpVhgo3Ay64q2Hp-GxOvXRjO2PEFz4zJn2eOLMVO4zClF01~0zw8rU2Wv2jp5WAJaf7p7h6bJnidBFL0-iCtQM11oomfijf-OC0JzpWFUqtfQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121775&ar=4922629&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy007f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flow karyotype of the Ishikawa 3-H-12 cell line at P25* without removing the background manually by image processing was obtained after sorting a larger number of chromosomes than usual to show clearly the der(9) peak close to the normal chromosome 3 peak. The der(9) peak is weaker than other peaks that have pairs of normal chromosomes as it exists in only one copy in each cell. Normal chromosome 9 and one of the chromosome 14 homologues, 14S, are also selected for sorting.</p></div></div>                            <h3 scrollto-destination=120121776 id="120121776" class="section-title" >3.2. Sequence analysis of der(9) and characterization of the fusion breakpoint</h3>
<p>In order to identify the genomic breakpoint junction in der(9), two sequence libraries were prepared from amplification of the sorted der(9) chromosomes. The sequence data from these two runs show 1.7 and 1.0Gb consisting of 6.4 and 4.0 M reads with the read length modes of 372 and 416 bp, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). Coverage analysis based on the human reference genome revealed that most sequence reads map to chromosomes 9 and 14, with some on chromosome 3 (<a reveal-id="dsy007-F2" data-open="dsy007-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>) corresponding to a small amount of contamination during chromosome sorting due to the close proximity of the der(9) and chromosome 3 peaks. Although a small number of reads were not aligned to the reference, more than 99% of the reads were aligned with 95 or 96% identities of base pairs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). This indicates that some reads are partially aligned with soft-clipped sequences, corresponding to breakpoint regions. Mapping data were displayed in the IGV software focusing on the distal end of the 9p region (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). Three reads measuring 242, 356 and 400 bp including unmapped sequences prove to be fragments that cover the fusion point. These three sequences were subjected to a BLAT search, which show homologies to both chromosomes 9 and 14 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). The fragments localize to the second introns of both the WASH1 (9p24.3) and NPAS3 (14q13.1) genes (<a reveal-id="dsy007-F3" data-open="dsy007-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a>). Absence of sequence homology between chromosomes 9 and 14 at the breakpoints indicates non-homologous end joining. The junction was amplified by PCR on genomic DNA using primers from chromosomes 9 and 14. A fusion transcript is formed as expected based on the orientation of two genes. RT-PCR followed by sequencing confirms the in-frame fusion of WASH1 exon 2 and NPAS3 exon 3 (<a reveal-id="dsy007-F3" data-open="dsy007-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). WASH1 is located at the distal end of the short arm of chromosome 9, ∼ 22 kb from the telomere according to the genome database. As the SNP microarray reported in our previous study<a reveal-id="dsy007-B7" data-open="dsy007-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> does not have markers located at the distal end, this rearrangement could not be characterized. The chromosome sequence data and the breakpoint sequence are deposited in DRA/DDBJ (accession number: DRA006485 and LC331302). </p>                        <a id="120121778" scrollto-destination="120121778"></a>
<div id="dsy007-F2" data-id="dsy007-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/m_dsy007f2.png?Expires=2147483647&Signature=AwYvuin2dTtk2MNKRGVUMz8335ZZZ0vRPJnjUn67n-~5Gqt80ApMauwmxhnEXLFNtzQSLcW0eBnhXPVy5ciLWUFjk7TJxLyrhnLRnqiCoN-eYNpfPy96p8qjCwayQUEqFEx6pGX5iWSrEpt4e8Xv5o8k35WI6uZd24~qFrZFnlYtzt2EhI3ZfmGnK1THgRrVKZKKcgjUZ68QylmZgrDEzXCFETet~zwpfXPtVtZfex6bXu0Fr~XdoLm~mylZrr9kjk-dw6EIgk1cnITK~y6Lcez4iXHM0JuHqaxyj5yyAjqHQyA2mL7bgms7neRqFevTV8yW49QXU8W9ON1qejbuoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Reference coverage charts from chromosome sequencing of der(9) are shown based on average base read depth. (A) Coverage on each chromosome indicates its contents aligned to chromosomes 9 and 14, with some sequences mapped to chromosome 3 because of contamination from the closely located peak in the flow karyotype. (B) Sequence reads are detected along chromosome 9, with a large gap at the region of the centromere and pericentromeric heterochromatin. (C) The majority of reads appear in the 14q13-qter region, which indicates the breakpoint at 14q13.1 shown by an arrowhead." path-from-xml="dsy007f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121778/dsy007f2.tif" path-from-xml="dsy007f2.tif" target="_blank">View large</a><a section="120121778" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007f2.png?Expires=2147483647&Signature=wHJvOlD-W-XWnh8wNR4eRbdKwfhmopp-~QV8b0MHV8NF28Txs8IxQKoUpYgHVjY0P-H7yFI4PHj8tWSSlgnDeQtCg55Uiw-8J3jL~w3Nq8AMe6eBwYoU0Wvmf70R2shFKx-Ho52idlKEJpouoCwBpWBuDijSXLW8NNvZf93Ijvc1XxJIoilh0OKAlApRkC05wUaqwZclHPDYxOhEIQqb57FaxHDrC8FgES8hht6XqUyuivzkcOniqcx2YoJJB-e3peZPvwJ9Z9S2dWQyUAjamS2chi3uxFiqYwQDvLMGbdjykIbSsaMdiN11BNAS00uwhKKMxcybMrlsyWWijU0wAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121778&ar=4922629&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy007f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Reference coverage charts from chromosome sequencing of der(9) are shown based on average base read depth. (A) Coverage on each chromosome indicates its contents aligned to chromosomes 9 and 14, with some sequences mapped to chromosome 3 because of contamination from the closely located peak in the flow karyotype. (B) Sequence reads are detected along chromosome 9, with a large gap at the region of the centromere and pericentromeric heterochromatin. (C) The majority of reads appear in the 14q13-qter region, which indicates the breakpoint at 14q13.1 shown by an arrowhead.</p></div></div><div content-id="dsy007-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/m_dsy007f2.png?Expires=2147483647&Signature=AwYvuin2dTtk2MNKRGVUMz8335ZZZ0vRPJnjUn67n-~5Gqt80ApMauwmxhnEXLFNtzQSLcW0eBnhXPVy5ciLWUFjk7TJxLyrhnLRnqiCoN-eYNpfPy96p8qjCwayQUEqFEx6pGX5iWSrEpt4e8Xv5o8k35WI6uZd24~qFrZFnlYtzt2EhI3ZfmGnK1THgRrVKZKKcgjUZ68QylmZgrDEzXCFETet~zwpfXPtVtZfex6bXu0Fr~XdoLm~mylZrr9kjk-dw6EIgk1cnITK~y6Lcez4iXHM0JuHqaxyj5yyAjqHQyA2mL7bgms7neRqFevTV8yW49QXU8W9ON1qejbuoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Reference coverage charts from chromosome sequencing of der(9) are shown based on average base read depth. (A) Coverage on each chromosome indicates its contents aligned to chromosomes 9 and 14, with some sequences mapped to chromosome 3 because of contamination from the closely located peak in the flow karyotype. (B) Sequence reads are detected along chromosome 9, with a large gap at the region of the centromere and pericentromeric heterochromatin. (C) The majority of reads appear in the 14q13-qter region, which indicates the breakpoint at 14q13.1 shown by an arrowhead." path-from-xml="dsy007f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121778/dsy007f2.tif" path-from-xml="dsy007f2.tif" target="_blank">View large</a><a section="120121778" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007f2.png?Expires=2147483647&Signature=wHJvOlD-W-XWnh8wNR4eRbdKwfhmopp-~QV8b0MHV8NF28Txs8IxQKoUpYgHVjY0P-H7yFI4PHj8tWSSlgnDeQtCg55Uiw-8J3jL~w3Nq8AMe6eBwYoU0Wvmf70R2shFKx-Ho52idlKEJpouoCwBpWBuDijSXLW8NNvZf93Ijvc1XxJIoilh0OKAlApRkC05wUaqwZclHPDYxOhEIQqb57FaxHDrC8FgES8hht6XqUyuivzkcOniqcx2YoJJB-e3peZPvwJ9Z9S2dWQyUAjamS2chi3uxFiqYwQDvLMGbdjykIbSsaMdiN11BNAS00uwhKKMxcybMrlsyWWijU0wAg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121778&ar=4922629&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy007f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Reference coverage charts from chromosome sequencing of der(9) are shown based on average base read depth. (A) Coverage on each chromosome indicates its contents aligned to chromosomes 9 and 14, with some sequences mapped to chromosome 3 because of contamination from the closely located peak in the flow karyotype. (B) Sequence reads are detected along chromosome 9, with a large gap at the region of the centromere and pericentromeric heterochromatin. (C) The majority of reads appear in the 14q13-qter region, which indicates the breakpoint at 14q13.1 shown by an arrowhead.</p></div></div>                        <a id="120121779" scrollto-destination="120121779"></a>
<div id="dsy007-F3" data-id="dsy007-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/m_dsy007f3.png?Expires=2147483647&Signature=LC0e-Ia4PdWnaR0WZ6~5o1jllt34xdOE15bvVI2QerWWU1QjkXqUubRgd1jRKwiztQ-RDziUQDlaPfflqeS1rZ-wzSAojVjd8kPDtpHDLZWiqDLGGA2cscyXvIKbvjJSIOTkU8JrOwIJodSuZTJk~Ca3yJ4jmdl0cqUaybfmSC9-BFg83tju2J4tz0msXwHeAb70913UaeMd~lio0KPxBn5rcEA9ooc6SgksrwDiJwbqvUCTZcKHrHd0z5IJOj~9hSFqUV7-rKVcSi37z2VtejsInkOQ4o6KhUP8-4ljZNk6EKGMEbaehnHRBfqedrjn2yznYg8Y6oeIFghZaqPecw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Breakpoint junction sequence of the der(9) chromosome. Upper, middle and lower sequences indicate reference sequence of chromosome 14, derivative chromosome and reference sequence of chromosome 9, respectively. Breakpoint positions are marked with arrowheads. Numbers are based on the nucleotide position in the UCSC genome browser (GRCh37/hg19). Cen, centromere; tel, telomere. (B) The nucleotide and amino acid sequences of the fusion gene. The chimeric WASH1–NPAS3 gene shows in-frame fusion of exon 2 of WASH1 to exon 3 of NPAS3." path-from-xml="dsy007f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121779/dsy007f3.tif" path-from-xml="dsy007f3.tif" target="_blank">View large</a><a section="120121779" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007f3.png?Expires=2147483647&Signature=tpdf72Hp5HFsRjkUBzxS9HMrz2LsZMDy4dYCEoN5~e9bf2kYynacN7ydGGeGvEzoBjWjlGhRPtm~HCqAHfog3QSNOHWXZ32QuOQnsEqjip7QqEsEN-bDfOhOD~Pybysrb3Y9nmxfzglscR8Bq3-ZeEO3CdvO1RJf-5ZFlT517JQSa0pMOj4Nf7zgpJ-tfknDmfrvo9fh79RlOJ~yJRBGL9yrUGd5x-wco95xjaDRvJQbGNFnHtxWV5BfHlAfSMv2ZjKkOeFwP9A6DVCZnXGu7viy8ySWu16H1W6VGLyNG3s-UuO7V9y1einffbIE7B9ZA5bmuk3JxNp4HtxSf5FBog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121779&ar=4922629&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy007f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Breakpoint junction sequence of the der(9) chromosome. Upper, middle and lower sequences indicate reference sequence of chromosome 14, derivative chromosome and reference sequence of chromosome 9, respectively. Breakpoint positions are marked with arrowheads. Numbers are based on the nucleotide position in the UCSC genome browser (GRCh37/hg19). Cen, centromere; tel, telomere. (B) The nucleotide and amino acid sequences of the fusion gene. The chimeric WASH1–NPAS3 gene shows in-frame fusion of exon 2 of WASH1 to exon 3 of NPAS3.</p></div></div><div content-id="dsy007-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/m_dsy007f3.png?Expires=2147483647&Signature=LC0e-Ia4PdWnaR0WZ6~5o1jllt34xdOE15bvVI2QerWWU1QjkXqUubRgd1jRKwiztQ-RDziUQDlaPfflqeS1rZ-wzSAojVjd8kPDtpHDLZWiqDLGGA2cscyXvIKbvjJSIOTkU8JrOwIJodSuZTJk~Ca3yJ4jmdl0cqUaybfmSC9-BFg83tju2J4tz0msXwHeAb70913UaeMd~lio0KPxBn5rcEA9ooc6SgksrwDiJwbqvUCTZcKHrHd0z5IJOj~9hSFqUV7-rKVcSi37z2VtejsInkOQ4o6KhUP8-4ljZNk6EKGMEbaehnHRBfqedrjn2yznYg8Y6oeIFghZaqPecw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="(A) Breakpoint junction sequence of the der(9) chromosome. Upper, middle and lower sequences indicate reference sequence of chromosome 14, derivative chromosome and reference sequence of chromosome 9, respectively. Breakpoint positions are marked with arrowheads. Numbers are based on the nucleotide position in the UCSC genome browser (GRCh37/hg19). Cen, centromere; tel, telomere. (B) The nucleotide and amino acid sequences of the fusion gene. The chimeric WASH1–NPAS3 gene shows in-frame fusion of exon 2 of WASH1 to exon 3 of NPAS3." path-from-xml="dsy007f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121779/dsy007f3.tif" path-from-xml="dsy007f3.tif" target="_blank">View large</a><a section="120121779" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007f3.png?Expires=2147483647&Signature=tpdf72Hp5HFsRjkUBzxS9HMrz2LsZMDy4dYCEoN5~e9bf2kYynacN7ydGGeGvEzoBjWjlGhRPtm~HCqAHfog3QSNOHWXZ32QuOQnsEqjip7QqEsEN-bDfOhOD~Pybysrb3Y9nmxfzglscR8Bq3-ZeEO3CdvO1RJf-5ZFlT517JQSa0pMOj4Nf7zgpJ-tfknDmfrvo9fh79RlOJ~yJRBGL9yrUGd5x-wco95xjaDRvJQbGNFnHtxWV5BfHlAfSMv2ZjKkOeFwP9A6DVCZnXGu7viy8ySWu16H1W6VGLyNG3s-UuO7V9y1einffbIE7B9ZA5bmuk3JxNp4HtxSf5FBog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121779&ar=4922629&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy007f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>(A) Breakpoint junction sequence of the der(9) chromosome. Upper, middle and lower sequences indicate reference sequence of chromosome 14, derivative chromosome and reference sequence of chromosome 9, respectively. Breakpoint positions are marked with arrowheads. Numbers are based on the nucleotide position in the UCSC genome browser (GRCh37/hg19). Cen, centromere; tel, telomere. (B) The nucleotide and amino acid sequences of the fusion gene. The chimeric WASH1–NPAS3 gene shows in-frame fusion of exon 2 of WASH1 to exon 3 of NPAS3.</p></div></div>                            <h3 scrollto-destination=120121780 id="120121780" class="section-title" >3.3. Variant profiles in the neighbourhood genes of the breakpoint</h3>
<p>Amplicon sequencing on total genomic DNA reveals that 117 and 118 target regions in 9p24 and 14q13, respectively, close to the breakpoint are amplified with at least 200 reads. As three amplicons in the 14q13 region have less than 30 reads, these targets are excluded from the data analysis. The average number of reads per amplicon is calculated to be 2,198 and 1,982 for 9p24 and 14q13 regions, respectively.</p><p>Nucleotide variants in the total genomic DNA sample are detected at 37 positions in the 9p24.3 region, but five positions were of low quality. The remaining 32 positions which did not include missense mutations serve as informative markers for distinguishing between alleles at variant loci. According to their frequency in the sequence reads, each variant allele can be classified into two types, around 50 or nearly 100% of reads. This suggests that two alleles are present in this region, supported by our previous data based on the SNP microarray and M-FISH which showed two chromosome copies (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3A</a></span>).<a reveal-id="dsy007-B7" data-open="dsy007-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Of the 32 sequence variants, 16 were detected at 100% frequency, reflecting their presence in both chromosomes (<a reveal-id="dsy007-T1" data-open="dsy007-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The other 16 variants with a frequency of around 50% indicate their presence in one of the two chromosomes, but these could not be localized to a chromosome in the genomic DNA sample. Chromosome-specific sequencing on der(9) and normal chromosome 9 reveal that allelic variants were detected at 23 and 25 positions, respectively, all of which showed nearly 100% frequency. This demonstrates that seven and nine variants are specific for der(9) and the normal chromosome 9, respectively, i.e., one allele at each locus. Because the other nine positions are not amplified in sorted chromosome samples, the alleles at these positions could not be distinguished. <div content-id="dsy007-T1" class="table-modal"><div class="table-wrap"><div id="dsy007-T1" data-id="dsy007-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Variant frequency for the 9p24.3 region</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene ID </th><th>Position </th><th>Ref </th><th>Variant </th><th>Total </th><th>der(9) </th><th>chr 9 </th></tr></thead><tbody><tr><td>WASH1 </td><td>14665 </td><td>G </td><td>A </td><td>61.2 </td><td> </td><td>100 </td></tr><tr><td>FOXD4 </td><td>116800 </td><td>C </td><td>G </td><td>48.0 </td><td>100 </td><td> </td></tr><tr><td>FOXD4 </td><td>116832 </td><td>T </td><td>C </td><td>47.5 </td><td>100 </td><td> </td></tr><tr><td>FOXD4 </td><td>118173 </td><td>C </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>CBWD1 </td><td>154795 </td><td>T </td><td>C </td><td>49.9 </td><td>96.1 </td><td> </td></tr><tr><td>DOCK8 </td><td>286593 </td><td>C </td><td>A </td><td>49.9 </td><td> </td><td>99.3 </td></tr><tr><td>DOCK8 </td><td>327938 </td><td>G </td><td>C </td><td>53.6 </td><td> </td><td>100 </td></tr><tr><td>DOCK8 </td><td>327957 </td><td>T </td><td>G </td><td>53.4 </td><td> </td><td>100 </td></tr><tr><td>DOCK8 </td><td>328006 </td><td>T </td><td>C </td><td>53.3 </td><td> </td><td>99.1 </td></tr><tr><td>DOCK8 </td><td>334524 </td><td>G </td><td>A </td><td>42.6 </td><td>100 </td><td> </td></tr><tr><td>DOCK8 </td><td>417990 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>98.9 </td></tr><tr><td>DOCK8 </td><td>418003 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>422157 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>432330 </td><td>C </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>433978 </td><td>A </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>439453 </td><td>G </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>439467 </td><td>A </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>441827 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>441952 </td><td>G </td><td>A </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>442030 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>446352 </td><td>G </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>449700 </td><td>G </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>KANK1 </td><td>730054 </td><td>A </td><td>G </td><td>47.7 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>738434 </td><td>C </td><td>T </td><td>55.1 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>740901 </td><td>C </td><td>T </td><td>51.3 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>741084 </td><td>C </td><td>T </td><td>52.2 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>742342 </td><td>A </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DMRT1 </td><td>841825 </td><td>C </td><td>T </td><td>53.2 </td><td>89.0 </td><td> </td></tr><tr><td>DMRT1 </td><td>841971 </td><td>T </td><td>A </td><td>52.6 </td><td>89.1 </td><td> </td></tr><tr><td>DMRT1 </td><td>967981 </td><td>G </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DMRT3 </td><td>990840 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DMRT2 </td><td>1057318 </td><td>A </td><td>G </td><td>46.5 </td><td>100 </td><td> </td></tr></tbody></table></div><div content-id="dsy007-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene ID </th><th>Position </th><th>Ref </th><th>Variant </th><th>Total </th><th>der(9) </th><th>chr 9 </th></tr></thead><tbody><tr><td>WASH1 </td><td>14665 </td><td>G </td><td>A </td><td>61.2 </td><td> </td><td>100 </td></tr><tr><td>FOXD4 </td><td>116800 </td><td>C </td><td>G </td><td>48.0 </td><td>100 </td><td> </td></tr><tr><td>FOXD4 </td><td>116832 </td><td>T </td><td>C </td><td>47.5 </td><td>100 </td><td> </td></tr><tr><td>FOXD4 </td><td>118173 </td><td>C </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>CBWD1 </td><td>154795 </td><td>T </td><td>C </td><td>49.9 </td><td>96.1 </td><td> </td></tr><tr><td>DOCK8 </td><td>286593 </td><td>C </td><td>A </td><td>49.9 </td><td> </td><td>99.3 </td></tr><tr><td>DOCK8 </td><td>327938 </td><td>G </td><td>C </td><td>53.6 </td><td> </td><td>100 </td></tr><tr><td>DOCK8 </td><td>327957 </td><td>T </td><td>G </td><td>53.4 </td><td> </td><td>100 </td></tr><tr><td>DOCK8 </td><td>328006 </td><td>T </td><td>C </td><td>53.3 </td><td> </td><td>99.1 </td></tr><tr><td>DOCK8 </td><td>334524 </td><td>G </td><td>A </td><td>42.6 </td><td>100 </td><td> </td></tr><tr><td>DOCK8 </td><td>417990 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>98.9 </td></tr><tr><td>DOCK8 </td><td>418003 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>422157 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>432330 </td><td>C </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>433978 </td><td>A </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>439453 </td><td>G </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>439467 </td><td>A </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>441827 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>441952 </td><td>G </td><td>A </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>442030 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>446352 </td><td>G </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>449700 </td><td>G </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>KANK1 </td><td>730054 </td><td>A </td><td>G </td><td>47.7 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>738434 </td><td>C </td><td>T </td><td>55.1 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>740901 </td><td>C </td><td>T </td><td>51.3 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>741084 </td><td>C </td><td>T </td><td>52.2 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>742342 </td><td>A </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DMRT1 </td><td>841825 </td><td>C </td><td>T </td><td>53.2 </td><td>89.0 </td><td> </td></tr><tr><td>DMRT1 </td><td>841971 </td><td>T </td><td>A </td><td>52.6 </td><td>89.1 </td><td> </td></tr><tr><td>DMRT1 </td><td>967981 </td><td>G </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DMRT3 </td><td>990840 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DMRT2 </td><td>1057318 </td><td>A </td><td>G </td><td>46.5 </td><td>100 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Color table is available in online version.</p></div></div></div></div></div><div class="table-wrap"><div id="dsy007-T1" data-id="dsy007-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Variant frequency for the 9p24.3 region</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene ID </th><th>Position </th><th>Ref </th><th>Variant </th><th>Total </th><th>der(9) </th><th>chr 9 </th></tr></thead><tbody><tr><td>WASH1 </td><td>14665 </td><td>G </td><td>A </td><td>61.2 </td><td> </td><td>100 </td></tr><tr><td>FOXD4 </td><td>116800 </td><td>C </td><td>G </td><td>48.0 </td><td>100 </td><td> </td></tr><tr><td>FOXD4 </td><td>116832 </td><td>T </td><td>C </td><td>47.5 </td><td>100 </td><td> </td></tr><tr><td>FOXD4 </td><td>118173 </td><td>C </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>CBWD1 </td><td>154795 </td><td>T </td><td>C </td><td>49.9 </td><td>96.1 </td><td> </td></tr><tr><td>DOCK8 </td><td>286593 </td><td>C </td><td>A </td><td>49.9 </td><td> </td><td>99.3 </td></tr><tr><td>DOCK8 </td><td>327938 </td><td>G </td><td>C </td><td>53.6 </td><td> </td><td>100 </td></tr><tr><td>DOCK8 </td><td>327957 </td><td>T </td><td>G </td><td>53.4 </td><td> </td><td>100 </td></tr><tr><td>DOCK8 </td><td>328006 </td><td>T </td><td>C </td><td>53.3 </td><td> </td><td>99.1 </td></tr><tr><td>DOCK8 </td><td>334524 </td><td>G </td><td>A </td><td>42.6 </td><td>100 </td><td> </td></tr><tr><td>DOCK8 </td><td>417990 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>98.9 </td></tr><tr><td>DOCK8 </td><td>418003 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>422157 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>432330 </td><td>C </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>433978 </td><td>A </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>439453 </td><td>G </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>439467 </td><td>A </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>441827 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>441952 </td><td>G </td><td>A </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>442030 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>446352 </td><td>G </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>449700 </td><td>G </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>KANK1 </td><td>730054 </td><td>A </td><td>G </td><td>47.7 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>738434 </td><td>C </td><td>T </td><td>55.1 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>740901 </td><td>C </td><td>T </td><td>51.3 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>741084 </td><td>C </td><td>T </td><td>52.2 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>742342 </td><td>A </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DMRT1 </td><td>841825 </td><td>C </td><td>T </td><td>53.2 </td><td>89.0 </td><td> </td></tr><tr><td>DMRT1 </td><td>841971 </td><td>T </td><td>A </td><td>52.6 </td><td>89.1 </td><td> </td></tr><tr><td>DMRT1 </td><td>967981 </td><td>G </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DMRT3 </td><td>990840 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DMRT2 </td><td>1057318 </td><td>A </td><td>G </td><td>46.5 </td><td>100 </td><td> </td></tr></tbody></table></div><div content-id="dsy007-T1" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene ID </th><th>Position </th><th>Ref </th><th>Variant </th><th>Total </th><th>der(9) </th><th>chr 9 </th></tr></thead><tbody><tr><td>WASH1 </td><td>14665 </td><td>G </td><td>A </td><td>61.2 </td><td> </td><td>100 </td></tr><tr><td>FOXD4 </td><td>116800 </td><td>C </td><td>G </td><td>48.0 </td><td>100 </td><td> </td></tr><tr><td>FOXD4 </td><td>116832 </td><td>T </td><td>C </td><td>47.5 </td><td>100 </td><td> </td></tr><tr><td>FOXD4 </td><td>118173 </td><td>C </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>CBWD1 </td><td>154795 </td><td>T </td><td>C </td><td>49.9 </td><td>96.1 </td><td> </td></tr><tr><td>DOCK8 </td><td>286593 </td><td>C </td><td>A </td><td>49.9 </td><td> </td><td>99.3 </td></tr><tr><td>DOCK8 </td><td>327938 </td><td>G </td><td>C </td><td>53.6 </td><td> </td><td>100 </td></tr><tr><td>DOCK8 </td><td>327957 </td><td>T </td><td>G </td><td>53.4 </td><td> </td><td>100 </td></tr><tr><td>DOCK8 </td><td>328006 </td><td>T </td><td>C </td><td>53.3 </td><td> </td><td>99.1 </td></tr><tr><td>DOCK8 </td><td>334524 </td><td>G </td><td>A </td><td>42.6 </td><td>100 </td><td> </td></tr><tr><td>DOCK8 </td><td>417990 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>98.9 </td></tr><tr><td>DOCK8 </td><td>418003 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>422157 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>432330 </td><td>C </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>433978 </td><td>A </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>439453 </td><td>G </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>439467 </td><td>A </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>441827 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>441952 </td><td>G </td><td>A </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>442030 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>446352 </td><td>G </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DOCK8 </td><td>449700 </td><td>G </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>KANK1 </td><td>730054 </td><td>A </td><td>G </td><td>47.7 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>738434 </td><td>C </td><td>T </td><td>55.1 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>740901 </td><td>C </td><td>T </td><td>51.3 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>741084 </td><td>C </td><td>T </td><td>52.2 </td><td> </td><td>100 </td></tr><tr><td>KANK1 </td><td>742342 </td><td>A </td><td>G </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DMRT1 </td><td>841825 </td><td>C </td><td>T </td><td>53.2 </td><td>89.0 </td><td> </td></tr><tr><td>DMRT1 </td><td>841971 </td><td>T </td><td>A </td><td>52.6 </td><td>89.1 </td><td> </td></tr><tr><td>DMRT1 </td><td>967981 </td><td>G </td><td>C </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DMRT3 </td><td>990840 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>100 </td></tr><tr><td>DMRT2 </td><td>1057318 </td><td>A </td><td>G </td><td>46.5 </td><td>100 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>Color table is available in online version.</p></div></div></div></div></p><p>Variant analysis of 10 gene loci at 14q13 in the total genomic DNA reveals 36 informative positions (<a reveal-id="dsy007-T2" data-open="dsy007-T2" class="link link-reveal link-table xref-fig">Table 2</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). The allele frequencies at these positions show three classes; either around 33 or 66, or nearly 100% of reads, corresponding to the presence of variants in one of three, two of three or all three alleles at these positions, respectively. This suggests that this region in 14q13 is present in three copies, which agrees with the SNP microarray profile and the karyotype analysis that reveal three copies of chromosome 14 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>).<a reveal-id="dsy007-B7" data-open="dsy007-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> The common variants present in all three alleles with 100% frequency are detected at six positions. The remaining 23 and 7 variants could be detected by frequencies of 33 and 66%, corresponding to their presence in one and two chromosomes, respectively. Chromosome sequencing was run twice in each sample and showed that variant frequencies were all nearly 100% (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). By sorting 14S separately from the unsorted chromosome 14 (14U), variant frequencies in 14S are either 0 or 100% (<a reveal-id="dsy007-T2" data-open="dsy007-T2" class="link link-reveal link-table xref-fig">Table 2</a>), indicating successful chromosome sorting without contamination between homologues. This demonstrates that der(9) and 14S samples have 13 and 29 variants, respectively (<a reveal-id="dsy007-T2" data-open="dsy007-T2" class="link link-reveal link-table xref-fig">Table 2</a>). Of these, 23 variants are specific for 14S. The total DNA sample indicates the presence of variants in 7 loci in two of three chromosomes, but they are only detected in der(9) and absent in 14S (<a reveal-id="dsy007-T2" data-open="dsy007-T2" class="link link-reveal link-table xref-fig">Table 2</a>). This implies that these variants must be shared between the chromosome 14 homologous region on der(9) and 14U, indicating that the chromosome 14 region in der(9) represents a duplication of 14U. Among the variants in the 10 genes, five missense mutations are identified (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). One of them is known to be a polymorphism, localized to der(9) and 14U. The other four mutations involving three different gene loci are included in 14S (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). <div content-id="dsy007-T2" class="table-modal"><div class="table-wrap"><div id="dsy007-T2" data-id="dsy007-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Variant frequency for the 14q13 region</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene ID </th><th>Position </th><th>Ref </th><th>Variant </th><th>Total </th><th>der(9) </th><th>14S </th><th>14U </th></tr></thead><tbody><tr><td>EGLN3 </td><td>34395149 </td><td>A </td><td>– </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>EAPP </td><td>34985645 </td><td>G </td><td>A </td><td>34.1 </td><td> </td><td>100 </td><td> </td></tr><tr><td>EAPP </td><td>34998705 </td><td>T </td><td>C </td><td>33.1 </td><td> </td><td>100 </td><td> </td></tr><tr><td>SNX6 </td><td>35062166 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>SNX6 </td><td>35062312 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>99.4 </td><td> </td></tr><tr><td>SNX6 </td><td>35075022 </td><td>G </td><td>T </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>BAZ1A </td><td>35222627 </td><td>– </td><td>TTGT </td><td>32.3 </td><td> </td><td>90.0 </td><td> </td></tr><tr><td>BAZ1A </td><td>35223956 </td><td>A </td><td>G </td><td>35.0 </td><td> </td><td>91.6 </td><td> </td></tr><tr><td>BAZ1A </td><td>35223971 </td><td>A </td><td>G </td><td>34.9 </td><td> </td><td>92.5 </td><td> </td></tr><tr><td>BAZ1A </td><td>35227880 </td><td>G </td><td>T </td><td>33.1 </td><td> </td><td>89.9 </td><td> </td></tr><tr><td>BAZ1A </td><td>35228090 </td><td>G </td><td>A </td><td>30.2 </td><td> </td><td>89.2 </td><td> </td></tr><tr><td>BAZ1A </td><td>35230906 </td><td>G </td><td>A </td><td>32.7 </td><td> </td><td>91.6 </td><td> </td></tr><tr><td>BAZ1A </td><td>35233740 </td><td>T </td><td>C </td><td>34.5 </td><td> </td><td>93.1 </td><td> </td></tr><tr><td>BAZ1A </td><td>35245748 </td><td>A </td><td>G </td><td>29.5 </td><td> </td><td>87.3 </td><td> </td></tr><tr><td>SRP54 </td><td>35483882 </td><td>A </td><td>C </td><td>36.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>SRP54 </td><td>35488309 </td><td>A </td><td>G </td><td>96.9 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>SRP54 </td><td>35492128 </td><td>C </td><td>T </td><td>60.9 </td><td>100 </td><td> </td><td> </td></tr><tr><td>SRP54 </td><td>35498443 </td><td>G </td><td>A </td><td>35.3 </td><td> </td><td>98.9 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35565856 </td><td>A </td><td>T </td><td>32.3 </td><td> </td><td>95.8 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35568756 </td><td>– </td><td>AA </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35576458 </td><td>A </td><td>C </td><td>32.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35578797 </td><td>C </td><td>T </td><td>71.0 </td><td>98.1 </td><td> </td><td> </td></tr><tr><td>PPP2R3C </td><td>35579927 </td><td>G </td><td>A </td><td>30.7 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35591174 </td><td>T </td><td>C </td><td>61.2 </td><td>98.6 </td><td> </td><td> </td></tr><tr><td>PPP2R3C </td><td>35591316 </td><td>G </td><td>A </td><td>62.3 </td><td>98.2 </td><td> </td><td> </td></tr><tr><td>KIAA0391 </td><td>35592491 </td><td>C </td><td>A </td><td>34.2 </td><td> </td><td>100 </td><td> </td></tr><tr><td>KIAA0391 </td><td>35592538 </td><td>C </td><td>G </td><td>66.3 </td><td>100 </td><td> </td><td> </td></tr><tr><td>KIAA0391 </td><td>35593599 </td><td>C </td><td>G </td><td>67.3 </td><td>98.2 </td><td> </td><td> </td></tr><tr><td>KIAA0391 </td><td>35596092 </td><td>C </td><td>A </td><td>32.9 </td><td> </td><td>100 </td><td> </td></tr><tr><td>KIAA0391 </td><td>35736075 </td><td>A </td><td>G </td><td>33.2 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35761573 </td><td>C </td><td>T </td><td>31.9 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35761675 </td><td>C </td><td>G </td><td>68.5 </td><td>96.8 </td><td> </td><td> </td></tr><tr><td>PSMA6 </td><td>35777232 </td><td>C </td><td>T </td><td>33.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35777243 </td><td>A </td><td>G </td><td>33.8 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35777270 </td><td>A </td><td>C </td><td>33.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35777278 </td><td>C </td><td>G </td><td>33.5 </td><td> </td><td>100 </td><td> </td></tr></tbody></table></div><div content-id="dsy007-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene ID </th><th>Position </th><th>Ref </th><th>Variant </th><th>Total </th><th>der(9) </th><th>14S </th><th>14U </th></tr></thead><tbody><tr><td>EGLN3 </td><td>34395149 </td><td>A </td><td>– </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>EAPP </td><td>34985645 </td><td>G </td><td>A </td><td>34.1 </td><td> </td><td>100 </td><td> </td></tr><tr><td>EAPP </td><td>34998705 </td><td>T </td><td>C </td><td>33.1 </td><td> </td><td>100 </td><td> </td></tr><tr><td>SNX6 </td><td>35062166 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>SNX6 </td><td>35062312 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>99.4 </td><td> </td></tr><tr><td>SNX6 </td><td>35075022 </td><td>G </td><td>T </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>BAZ1A </td><td>35222627 </td><td>– </td><td>TTGT </td><td>32.3 </td><td> </td><td>90.0 </td><td> </td></tr><tr><td>BAZ1A </td><td>35223956 </td><td>A </td><td>G </td><td>35.0 </td><td> </td><td>91.6 </td><td> </td></tr><tr><td>BAZ1A </td><td>35223971 </td><td>A </td><td>G </td><td>34.9 </td><td> </td><td>92.5 </td><td> </td></tr><tr><td>BAZ1A </td><td>35227880 </td><td>G </td><td>T </td><td>33.1 </td><td> </td><td>89.9 </td><td> </td></tr><tr><td>BAZ1A </td><td>35228090 </td><td>G </td><td>A </td><td>30.2 </td><td> </td><td>89.2 </td><td> </td></tr><tr><td>BAZ1A </td><td>35230906 </td><td>G </td><td>A </td><td>32.7 </td><td> </td><td>91.6 </td><td> </td></tr><tr><td>BAZ1A </td><td>35233740 </td><td>T </td><td>C </td><td>34.5 </td><td> </td><td>93.1 </td><td> </td></tr><tr><td>BAZ1A </td><td>35245748 </td><td>A </td><td>G </td><td>29.5 </td><td> </td><td>87.3 </td><td> </td></tr><tr><td>SRP54 </td><td>35483882 </td><td>A </td><td>C </td><td>36.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>SRP54 </td><td>35488309 </td><td>A </td><td>G </td><td>96.9 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>SRP54 </td><td>35492128 </td><td>C </td><td>T </td><td>60.9 </td><td>100 </td><td> </td><td> </td></tr><tr><td>SRP54 </td><td>35498443 </td><td>G </td><td>A </td><td>35.3 </td><td> </td><td>98.9 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35565856 </td><td>A </td><td>T </td><td>32.3 </td><td> </td><td>95.8 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35568756 </td><td>– </td><td>AA </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35576458 </td><td>A </td><td>C </td><td>32.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35578797 </td><td>C </td><td>T </td><td>71.0 </td><td>98.1 </td><td> </td><td> </td></tr><tr><td>PPP2R3C </td><td>35579927 </td><td>G </td><td>A </td><td>30.7 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35591174 </td><td>T </td><td>C </td><td>61.2 </td><td>98.6 </td><td> </td><td> </td></tr><tr><td>PPP2R3C </td><td>35591316 </td><td>G </td><td>A </td><td>62.3 </td><td>98.2 </td><td> </td><td> </td></tr><tr><td>KIAA0391 </td><td>35592491 </td><td>C </td><td>A </td><td>34.2 </td><td> </td><td>100 </td><td> </td></tr><tr><td>KIAA0391 </td><td>35592538 </td><td>C </td><td>G </td><td>66.3 </td><td>100 </td><td> </td><td> </td></tr><tr><td>KIAA0391 </td><td>35593599 </td><td>C </td><td>G </td><td>67.3 </td><td>98.2 </td><td> </td><td> </td></tr><tr><td>KIAA0391 </td><td>35596092 </td><td>C </td><td>A </td><td>32.9 </td><td> </td><td>100 </td><td> </td></tr><tr><td>KIAA0391 </td><td>35736075 </td><td>A </td><td>G </td><td>33.2 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35761573 </td><td>C </td><td>T </td><td>31.9 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35761675 </td><td>C </td><td>G </td><td>68.5 </td><td>96.8 </td><td> </td><td> </td></tr><tr><td>PSMA6 </td><td>35777232 </td><td>C </td><td>T </td><td>33.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35777243 </td><td>A </td><td>G </td><td>33.8 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35777270 </td><td>A </td><td>C </td><td>33.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35777278 </td><td>C </td><td>G </td><td>33.5 </td><td> </td><td>100 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>Color table is available in online version.</p></div></div></div></div></div><div class="table-wrap"><div id="dsy007-T2" data-id="dsy007-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Variant frequency for the 14q13 region</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Gene ID </th><th>Position </th><th>Ref </th><th>Variant </th><th>Total </th><th>der(9) </th><th>14S </th><th>14U </th></tr></thead><tbody><tr><td>EGLN3 </td><td>34395149 </td><td>A </td><td>– </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>EAPP </td><td>34985645 </td><td>G </td><td>A </td><td>34.1 </td><td> </td><td>100 </td><td> </td></tr><tr><td>EAPP </td><td>34998705 </td><td>T </td><td>C </td><td>33.1 </td><td> </td><td>100 </td><td> </td></tr><tr><td>SNX6 </td><td>35062166 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>SNX6 </td><td>35062312 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>99.4 </td><td> </td></tr><tr><td>SNX6 </td><td>35075022 </td><td>G </td><td>T </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>BAZ1A </td><td>35222627 </td><td>– </td><td>TTGT </td><td>32.3 </td><td> </td><td>90.0 </td><td> </td></tr><tr><td>BAZ1A </td><td>35223956 </td><td>A </td><td>G </td><td>35.0 </td><td> </td><td>91.6 </td><td> </td></tr><tr><td>BAZ1A </td><td>35223971 </td><td>A </td><td>G </td><td>34.9 </td><td> </td><td>92.5 </td><td> </td></tr><tr><td>BAZ1A </td><td>35227880 </td><td>G </td><td>T </td><td>33.1 </td><td> </td><td>89.9 </td><td> </td></tr><tr><td>BAZ1A </td><td>35228090 </td><td>G </td><td>A </td><td>30.2 </td><td> </td><td>89.2 </td><td> </td></tr><tr><td>BAZ1A </td><td>35230906 </td><td>G </td><td>A </td><td>32.7 </td><td> </td><td>91.6 </td><td> </td></tr><tr><td>BAZ1A </td><td>35233740 </td><td>T </td><td>C </td><td>34.5 </td><td> </td><td>93.1 </td><td> </td></tr><tr><td>BAZ1A </td><td>35245748 </td><td>A </td><td>G </td><td>29.5 </td><td> </td><td>87.3 </td><td> </td></tr><tr><td>SRP54 </td><td>35483882 </td><td>A </td><td>C </td><td>36.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>SRP54 </td><td>35488309 </td><td>A </td><td>G </td><td>96.9 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>SRP54 </td><td>35492128 </td><td>C </td><td>T </td><td>60.9 </td><td>100 </td><td> </td><td> </td></tr><tr><td>SRP54 </td><td>35498443 </td><td>G </td><td>A </td><td>35.3 </td><td> </td><td>98.9 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35565856 </td><td>A </td><td>T </td><td>32.3 </td><td> </td><td>95.8 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35568756 </td><td>– </td><td>AA </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35576458 </td><td>A </td><td>C </td><td>32.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35578797 </td><td>C </td><td>T </td><td>71.0 </td><td>98.1 </td><td> </td><td> </td></tr><tr><td>PPP2R3C </td><td>35579927 </td><td>G </td><td>A </td><td>30.7 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35591174 </td><td>T </td><td>C </td><td>61.2 </td><td>98.6 </td><td> </td><td> </td></tr><tr><td>PPP2R3C </td><td>35591316 </td><td>G </td><td>A </td><td>62.3 </td><td>98.2 </td><td> </td><td> </td></tr><tr><td>KIAA0391 </td><td>35592491 </td><td>C </td><td>A </td><td>34.2 </td><td> </td><td>100 </td><td> </td></tr><tr><td>KIAA0391 </td><td>35592538 </td><td>C </td><td>G </td><td>66.3 </td><td>100 </td><td> </td><td> </td></tr><tr><td>KIAA0391 </td><td>35593599 </td><td>C </td><td>G </td><td>67.3 </td><td>98.2 </td><td> </td><td> </td></tr><tr><td>KIAA0391 </td><td>35596092 </td><td>C </td><td>A </td><td>32.9 </td><td> </td><td>100 </td><td> </td></tr><tr><td>KIAA0391 </td><td>35736075 </td><td>A </td><td>G </td><td>33.2 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35761573 </td><td>C </td><td>T </td><td>31.9 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35761675 </td><td>C </td><td>G </td><td>68.5 </td><td>96.8 </td><td> </td><td> </td></tr><tr><td>PSMA6 </td><td>35777232 </td><td>C </td><td>T </td><td>33.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35777243 </td><td>A </td><td>G </td><td>33.8 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35777270 </td><td>A </td><td>C </td><td>33.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35777278 </td><td>C </td><td>G </td><td>33.5 </td><td> </td><td>100 </td><td> </td></tr></tbody></table></div><div content-id="dsy007-T2" class="table-modal"><table><thead name="thead" align=""><tr><th>Gene ID </th><th>Position </th><th>Ref </th><th>Variant </th><th>Total </th><th>der(9) </th><th>14S </th><th>14U </th></tr></thead><tbody><tr><td>EGLN3 </td><td>34395149 </td><td>A </td><td>– </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>EAPP </td><td>34985645 </td><td>G </td><td>A </td><td>34.1 </td><td> </td><td>100 </td><td> </td></tr><tr><td>EAPP </td><td>34998705 </td><td>T </td><td>C </td><td>33.1 </td><td> </td><td>100 </td><td> </td></tr><tr><td>SNX6 </td><td>35062166 </td><td>T </td><td>C </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>SNX6 </td><td>35062312 </td><td>C </td><td>T </td><td>100 </td><td>100 </td><td>99.4 </td><td> </td></tr><tr><td>SNX6 </td><td>35075022 </td><td>G </td><td>T </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>BAZ1A </td><td>35222627 </td><td>– </td><td>TTGT </td><td>32.3 </td><td> </td><td>90.0 </td><td> </td></tr><tr><td>BAZ1A </td><td>35223956 </td><td>A </td><td>G </td><td>35.0 </td><td> </td><td>91.6 </td><td> </td></tr><tr><td>BAZ1A </td><td>35223971 </td><td>A </td><td>G </td><td>34.9 </td><td> </td><td>92.5 </td><td> </td></tr><tr><td>BAZ1A </td><td>35227880 </td><td>G </td><td>T </td><td>33.1 </td><td> </td><td>89.9 </td><td> </td></tr><tr><td>BAZ1A </td><td>35228090 </td><td>G </td><td>A </td><td>30.2 </td><td> </td><td>89.2 </td><td> </td></tr><tr><td>BAZ1A </td><td>35230906 </td><td>G </td><td>A </td><td>32.7 </td><td> </td><td>91.6 </td><td> </td></tr><tr><td>BAZ1A </td><td>35233740 </td><td>T </td><td>C </td><td>34.5 </td><td> </td><td>93.1 </td><td> </td></tr><tr><td>BAZ1A </td><td>35245748 </td><td>A </td><td>G </td><td>29.5 </td><td> </td><td>87.3 </td><td> </td></tr><tr><td>SRP54 </td><td>35483882 </td><td>A </td><td>C </td><td>36.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>SRP54 </td><td>35488309 </td><td>A </td><td>G </td><td>96.9 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>SRP54 </td><td>35492128 </td><td>C </td><td>T </td><td>60.9 </td><td>100 </td><td> </td><td> </td></tr><tr><td>SRP54 </td><td>35498443 </td><td>G </td><td>A </td><td>35.3 </td><td> </td><td>98.9 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35565856 </td><td>A </td><td>T </td><td>32.3 </td><td> </td><td>95.8 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35568756 </td><td>– </td><td>AA </td><td>100 </td><td>100 </td><td>100 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35576458 </td><td>A </td><td>C </td><td>32.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35578797 </td><td>C </td><td>T </td><td>71.0 </td><td>98.1 </td><td> </td><td> </td></tr><tr><td>PPP2R3C </td><td>35579927 </td><td>G </td><td>A </td><td>30.7 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PPP2R3C </td><td>35591174 </td><td>T </td><td>C </td><td>61.2 </td><td>98.6 </td><td> </td><td> </td></tr><tr><td>PPP2R3C </td><td>35591316 </td><td>G </td><td>A </td><td>62.3 </td><td>98.2 </td><td> </td><td> </td></tr><tr><td>KIAA0391 </td><td>35592491 </td><td>C </td><td>A </td><td>34.2 </td><td> </td><td>100 </td><td> </td></tr><tr><td>KIAA0391 </td><td>35592538 </td><td>C </td><td>G </td><td>66.3 </td><td>100 </td><td> </td><td> </td></tr><tr><td>KIAA0391 </td><td>35593599 </td><td>C </td><td>G </td><td>67.3 </td><td>98.2 </td><td> </td><td> </td></tr><tr><td>KIAA0391 </td><td>35596092 </td><td>C </td><td>A </td><td>32.9 </td><td> </td><td>100 </td><td> </td></tr><tr><td>KIAA0391 </td><td>35736075 </td><td>A </td><td>G </td><td>33.2 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35761573 </td><td>C </td><td>T </td><td>31.9 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35761675 </td><td>C </td><td>G </td><td>68.5 </td><td>96.8 </td><td> </td><td> </td></tr><tr><td>PSMA6 </td><td>35777232 </td><td>C </td><td>T </td><td>33.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35777243 </td><td>A </td><td>G </td><td>33.8 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35777270 </td><td>A </td><td>C </td><td>33.4 </td><td> </td><td>100 </td><td> </td></tr><tr><td>PSMA6 </td><td>35777278 </td><td>C </td><td>G </td><td>33.5 </td><td> </td><td>100 </td><td> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>Color table is available in online version.</p></div></div></div></div></p>                            <h3 scrollto-destination=120121784 id="120121784" class="section-title" >3.4. Copy number changes at the 14q13 region during passage</h3>
<p>The SNP microarray performed in our previous study shows differences of the Ishikawa 3-H-12 genome profiles between low and high passages.<a reveal-id="dsy007-B7" data-open="dsy007-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> Although the der(9) has already been found in all cells at low passage, the enlarged profiles focused on the breakpoint region at 14q13.1 indicate copy number changes occurring in one locus at P3 and three loci at P20 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3B</a></span>). The change observed at both P3 and P20 corresponds to the breakpoint that forms der(9). The additional changes at P20 between the other two loci involve breakpoints that cause a decrease in copy number. Because this is not detected at P3, the cryptic deletion must have arisen in the cell population after long-term culture. Amplicon sequencing targeted for the 10 kb region around the breakpoint revealed that the whole region is amplified in 14S (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). As the region upstream of the der(9) breakpoint is absent in der(9), only the downstream region is amplified. Similar profiles of the downstream region shared by 14S and der(9) suggest that no additional chromosome rearrangement has occurred in the region. The patterns of variant frequencies across this region in the total genomic DNA sample demonstrates the change from 50 to 33/66/100% at P3 and from 100 to 50/100% at P20 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>), corresponding to the chromosome copy number changes from 2 to 3 and from 1 to 2 that have occurred in this region (<a reveal-id="dsy007-F4" data-open="dsy007-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). Variants detected in the der(9) would be expected to be present at P3, indicating that three of them, 66% at P3, are shared with 14U. However, these variants are not detected in total genomic DNA at P20. This implies that two additional breaks have occurred in 14U and led to a cryptic deletion. Because the other variants detected in one of three loci at P3 are specific for der(9), these changes can be related to the breakpoint event that occurred following duplication of chromosome 14. </p>                        <a id="120121786" scrollto-destination="120121786"></a>
<div id="dsy007-F4" data-id="dsy007-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/m_dsy007f4.png?Expires=2147483647&Signature=l7LGoTqQOnSviG9lyOSRMKvuGJvdXhEYbRmHVMgBVwmT5yQ5rc0El~C7fc~tm2H0GRRpwC4rPZAGIPQYZt0-~KXTXpywcCZBNnmHDRBx8uJ374KWGE6deh8FVq9RyxTusJesZhUrSDo7mlh-C-p3k6kUBRf~RDbsunIrjdvvoNtNQWtXm5cGhTFFYTOmcGiNbNLpap0u2ULfMxSM8on4p70jKe8jpGW7K8QtedDN~unQn3Rlc1TXCVdKW6VmUpCaWB3LRJ4xMCtJeBcjlnq1TyiJainRAt2A34KdhVmGMVuazO9Wz2REJihNJVapuyRJDcEJISz~yi35LYPT-K962w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome structure at the 14q13.1 region is illustrated from the array profiles and combined with variant frequencies obtained from amplicon sequencing, with comparisons between passages, P3 (A) and P20 (B). The breakpoint in the der(9) is located at the position 33,583,729 bp. Variants are assigned to der(9) and 14S by amplicon sequencing in the chromosome-specific DNA. The frequencies are distinguished by four different symbols. Variant assignments on chromosome homologues at P3 are estimated from the P20 profile. A series of 100, 50 and 33/66% in total genomic DNA correspond to 1, 2 and 3 copies, respectively. Frequencies at three positions with 66% at low passage change to 50% at high passage, suggesting the presence of these variants in the 14U deletion indicated by a dashed line. gDNA: total genomic DNA." path-from-xml="dsy007f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121786/dsy007f4.tif" path-from-xml="dsy007f4.tif" target="_blank">View large</a><a section="120121786" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007f4.png?Expires=2147483647&Signature=r8lYpxl4z9z1VqLxgEsVarll4NPA2wPL1XU4xhsdpaNbUXXD6nSYrdlWi95V2mMftRN~9uQDGIrD0jgPrqqepYNg9UkY1D06E5Cas3bWDzccoJx8giXuRRlp0s8w4z6plBLSQosME07cW1UI7ygI0hK59nL6sDtBm2Dl1Qw54GzUH24XevY7dS1bQvQfhzTpolzHZxf1GaYPHvGvyOPobVpGR9uJIzleEwWQEpE6LfyZwQH40nEGqz-4Bo1LIP8a0kjgACaFkD4CwJlAby12c3rDst5lQORneMqpqN2iUFXjXuYa1PGLmECLOZ3k~aqW~evKL1awGrYvdZ0Py-fHGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121786&ar=4922629&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy007f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome structure at the 14q13.1 region is illustrated from the array profiles and combined with variant frequencies obtained from amplicon sequencing, with comparisons between passages, P3 (A) and P20 (B). The breakpoint in the der(9) is located at the position 33,583,729 bp. Variants are assigned to der(9) and 14S by amplicon sequencing in the chromosome-specific DNA. The frequencies are distinguished by four different symbols. Variant assignments on chromosome homologues at P3 are estimated from the P20 profile. A series of 100, 50 and 33/66% in total genomic DNA correspond to 1, 2 and 3 copies, respectively. Frequencies at three positions with 66% at low passage change to 50% at high passage, suggesting the presence of these variants in the 14U deletion indicated by a dashed line. gDNA: total genomic DNA.</p></div></div><div content-id="dsy007-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/m_dsy007f4.png?Expires=2147483647&Signature=l7LGoTqQOnSviG9lyOSRMKvuGJvdXhEYbRmHVMgBVwmT5yQ5rc0El~C7fc~tm2H0GRRpwC4rPZAGIPQYZt0-~KXTXpywcCZBNnmHDRBx8uJ374KWGE6deh8FVq9RyxTusJesZhUrSDo7mlh-C-p3k6kUBRf~RDbsunIrjdvvoNtNQWtXm5cGhTFFYTOmcGiNbNLpap0u2ULfMxSM8on4p70jKe8jpGW7K8QtedDN~unQn3Rlc1TXCVdKW6VmUpCaWB3LRJ4xMCtJeBcjlnq1TyiJainRAt2A34KdhVmGMVuazO9Wz2REJihNJVapuyRJDcEJISz~yi35LYPT-K962w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome structure at the 14q13.1 region is illustrated from the array profiles and combined with variant frequencies obtained from amplicon sequencing, with comparisons between passages, P3 (A) and P20 (B). The breakpoint in the der(9) is located at the position 33,583,729 bp. Variants are assigned to der(9) and 14S by amplicon sequencing in the chromosome-specific DNA. The frequencies are distinguished by four different symbols. Variant assignments on chromosome homologues at P3 are estimated from the P20 profile. A series of 100, 50 and 33/66% in total genomic DNA correspond to 1, 2 and 3 copies, respectively. Frequencies at three positions with 66% at low passage change to 50% at high passage, suggesting the presence of these variants in the 14U deletion indicated by a dashed line. gDNA: total genomic DNA." path-from-xml="dsy007f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121786/dsy007f4.tif" path-from-xml="dsy007f4.tif" target="_blank">View large</a><a section="120121786" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007f4.png?Expires=2147483647&Signature=r8lYpxl4z9z1VqLxgEsVarll4NPA2wPL1XU4xhsdpaNbUXXD6nSYrdlWi95V2mMftRN~9uQDGIrD0jgPrqqepYNg9UkY1D06E5Cas3bWDzccoJx8giXuRRlp0s8w4z6plBLSQosME07cW1UI7ygI0hK59nL6sDtBm2Dl1Qw54GzUH24XevY7dS1bQvQfhzTpolzHZxf1GaYPHvGvyOPobVpGR9uJIzleEwWQEpE6LfyZwQH40nEGqz-4Bo1LIP8a0kjgACaFkD4CwJlAby12c3rDst5lQORneMqpqN2iUFXjXuYa1PGLmECLOZ3k~aqW~evKL1awGrYvdZ0Py-fHGQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121786&ar=4922629&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy007f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome structure at the 14q13.1 region is illustrated from the array profiles and combined with variant frequencies obtained from amplicon sequencing, with comparisons between passages, P3 (A) and P20 (B). The breakpoint in the der(9) is located at the position 33,583,729 bp. Variants are assigned to der(9) and 14S by amplicon sequencing in the chromosome-specific DNA. The frequencies are distinguished by four different symbols. Variant assignments on chromosome homologues at P3 are estimated from the P20 profile. A series of 100, 50 and 33/66% in total genomic DNA correspond to 1, 2 and 3 copies, respectively. Frequencies at three positions with 66% at low passage change to 50% at high passage, suggesting the presence of these variants in the 14U deletion indicated by a dashed line. gDNA: total genomic DNA.</p></div></div>                            <h2 scrollto-destination=120121787 id="120121787" class="section-title" >4. Discussion</h2>
<p>Chromosome translocation is a common feature of chromosome rearrangements in tumour cells and can lead to the formation of a fusion gene.<a reveal-id="dsy007-B19" data-open="dsy007-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsy007-B20" data-open="dsy007-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Although fusion genes are found by RNA-seq, it is hard to identify the genomic breakpoint because normal and abnormal alleles usually coexist in total genomic DNA samples and cannot be distinguished by conventional analysis.<a reveal-id="dsy007-B21" data-open="dsy007-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> The human genome size is 3,099 Mbp according to the golden path length of the GRCh37.p13 assembly. As the majority of the Ishikawa karyotypes are 45, X, -X and have some gains,<a reveal-id="dsy007-B7" data-open="dsy007-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> the size of each Ishikawa nuclear DNA is estimated to be ∼ 6,200 Mbp, including one copy of the der(9). When the breakpoint junction is identified by conventional whole genome sequencing using total genomic DNA, theoretically it can appear in 1 per 15,500,000 fragments based on a 400 bp sequencing sample. However, it can be increased to 1 per 500,000 fragments when the sample is from the sorted der(9) chromosome, that is more efficient than whole genome sequencing. It is reported that alignment of single-end long reads is straightforward to identify fusion junctions.<a reveal-id="dsy007-B22" data-open="dsy007-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> We demonstrate that identification of genomic breakpoints detected by conventional karyotypes can be achieved by mapping analysis of single end reads on 400 bp scale without <em>de novo</em> assembly. Although considerable efforts involving conventional cloning techniques have been devoted to uncover breakpoint junctions in tumour genomes,<a reveal-id="dsy007-B23" data-open="dsy007-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a>, chromosome sequencing helps the detection and analysis of the genomic breakpoint junction.</p><p>Fusion genes serve as genetic markers and can be associated with disease phenotypes.<a reveal-id="dsy007-B24" data-open="dsy007-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> Sequence analysis of the der(9) chromosome-specific DNA isolated by flow sorting in our study reveals a WASH1–NPAS3 fusion gene at the translocation breakpoint. This chimeric gene is formed mostly by the NPAS3 component, suggesting that altered NPAS3 expression may be oncogenic. Although no report has shown fusion genes between these two genes, NPAS3 is a transcription factor of a basic helix–loop–helix Per-ARNT-Sim domain family associated with tumour development in multiple forms of cancer.<a reveal-id="dsy007-B25" data-open="dsy007-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Also, it is reported that NPAS3 acts as a tumour suppressor in human malignant astrocytomas.<a reveal-id="dsy007-B26" data-open="dsy007-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> The fusion detected by the genomic breakpoint junction and confirmed by the transcript can be a definitive marker for the Ishikawa cell line and used, possibly, for future studies on endometrial carcinoma.</p><p>Amplicon sequencing of the genes neighbouring the breakpoint demonstrates that variant frequencies in the total genomic DNA are linked to chromosome copy number. Variant frequencies of 50% of reads detected in chromosome 9 indicate the presence of the variant in one of two homologues. The variant frequencies of 33 or 66% of reads in chromosome 14 reflect one or two of three homologues, respectively (<a reveal-id="dsy007-F4" data-open="dsy007-F4" class="link link-reveal link-table xref-fig">Fig. 4</a>). This suggests that a series of patterns consisting of frequencies of approximate 50/100 or 33/66/100% are strongly correlated to the presence of copy numbers of two or three chromosomes, respectively. While the fusion can be clearly distinguished at the chromosome level, sequence variants cannot be regarded as allelic unless the chromosomes are sequenced separately. When amplicon sequencing is performed on chromosome-specific DNA, the variant frequencies of 0 or 100% of reads indicate that the variant is either absent or present in an amplicon. This permits chromosome assignment of each variant to a specific chromosome and enables distinction between homologues. Two chromosomes sharing the same allelic profile suggest that they have a common origin and are formed by duplication. Allele frequencies of ∼ 66% of reads reflects the presence of the same variant in two of the three alleles in three chromosomes corresponding to one in der(9) and one in 14U (<a reveal-id="dsy007-T2" data-open="dsy007-T2" class="link link-reveal link-table xref-fig">Table 2</a>, <a reveal-id="dsy007-F5" data-open="dsy007-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). Further analysis of the breakpoint by amplicon sequencing of the 10 kb region that encompasses the junction in the second intron of NPAS3 identified a cryptic deletion occurring in the chromosome 14 allele shared with der(9). These detailed variant analyses elucidate the process of chromosome rearrangements involving chromosomes 9 and 14. SNP allele peaks show two different allelic profiles of chromosome 14 in Ishikawa cells,<a reveal-id="dsy007-B7" data-open="dsy007-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> indicating that the duplication of 14q13–ter has occurred prior to the formation of der(9). Although the variant frequency at 14q13 reveals that the cryptic deletion and the breakpoint in the der(9) occur in different chromosome 14 homologues, these two chromosomes must have been formed through duplication because of their otherwise identical allelic profiles. As the deletion cannot be identified in the array profile at low passage, it must have occurred after the formation of der(9) and during cell culture. These two rearrangements have occurred independently, possibly suggesting a tendency to breakage in the region. </p>                        <a id="120121791" scrollto-destination="120121791"></a>
<div id="dsy007-F5" data-id="dsy007-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/m_dsy007f5.png?Expires=2147483647&Signature=5HOS4-u1YgceAt8FsUAYaISn5iZqyQyNSfZV9c1Bmv0oD2CLaQ4jgCSV~lfZQFSN52GPaecbgUzRyIxio~I2NDhP~o4QO4n0GEGlyvGa7mDNyGaihqJbj3Kz5wMk~CvcxwHR0pQDDD-Ctb2DzPx4ZIYEYq8puJj35EqFrJ926I~P8VoXOzGk6rX3DOYI2iwP5~24BUAcXcirj93Go-9zRfHuHQOfS3Tsktfo6qkIR~YfATA1CXMZuGE-HLpgK3gqP8j6hFZiV9YyZlu4guwgV9nogTwEWsKCIBW9AjZiAO1609QpIJmOMS2Y9QzdhT~oICyZceEKvU3XcFTM7qtHkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The process of chromosome rearrangement involved in chromosome 14. Chromosomes 9 and 14 regions are marked by green and blue, respectively, and the two homologues of chromosome 14 are distinguished by vertical and horizontal patterns. Because chromosome 14 consists of two different allele contents in the SNP array, it must have originated from a normal pair with different allele profiles. Variant analysis in the sorted chromosome samples show that 14S and der(9) have different chromosome 14 allelic profiles, but der(9) and 14U have the same chromosome 14 allelic profiles. This indicates that the origin of der(9) was associated with duplication of 14U. One of the homologues, 14S, does not have detectable changes at chromosomal level and der(9) remains unchanged during cell culture. However, duplication of the long arm of 14U is fused with 9p to form der(9). The 14U chromosome at passage P20 has a cryptic deletion in the same region as the breakpoint in der(9). Black circles and squares show telomere and centromere, respectively. Color figure is available in online version." path-from-xml="dsy007f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121791/dsy007f5.tif" path-from-xml="dsy007f5.tif" target="_blank">View large</a><a section="120121791" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007f5.png?Expires=2147483647&Signature=ESYZZZ24He5yXjbBSaeNtRndRhytFG6v9y9OoFIsqdWSUojKPMwHy3Ylrm1~dLphT5aNMqaVcP-SS20LAGCxteSDsf2nfeefDtSj69u-Rfh2EoIJwZhm54WMksVOENTeXCYboyy~KJHOJngTWTl6ZES3EMEsXlr4GObYNcLF0UDlVrGJzWZKpA7qhtgfkI8D3oRx9ApVb4XADoqE2wb33PKLHtLw5MKNiBoM4ZpBXDDtTHsVfsHpL4Dzgf53OpU7FApKIKPVmA-B5i4zrDY8ewJNYHvj0T2Kdv~zKnuixPeWVokw43~P1McM2Na7OKouoF1yVYvfqriF1fPjJ-mCcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121791&ar=4922629&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy007f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The process of chromosome rearrangement involved in chromosome 14. Chromosomes 9 and 14 regions are marked by green and blue, respectively, and the two homologues of chromosome 14 are distinguished by vertical and horizontal patterns. Because chromosome 14 consists of two different allele contents in the SNP array, it must have originated from a normal pair with different allele profiles. Variant analysis in the sorted chromosome samples show that 14S and der(9) have different chromosome 14 allelic profiles, but der(9) and 14U have the same chromosome 14 allelic profiles. This indicates that the origin of der(9) was associated with duplication of 14U. One of the homologues, 14S, does not have detectable changes at chromosomal level and der(9) remains unchanged during cell culture. However, duplication of the long arm of 14U is fused with 9p to form der(9). The 14U chromosome at passage P20 has a cryptic deletion in the same region as the breakpoint in der(9). Black circles and squares show telomere and centromere, respectively. Color figure is available in online version.</p></div></div><div content-id="dsy007-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/m_dsy007f5.png?Expires=2147483647&Signature=5HOS4-u1YgceAt8FsUAYaISn5iZqyQyNSfZV9c1Bmv0oD2CLaQ4jgCSV~lfZQFSN52GPaecbgUzRyIxio~I2NDhP~o4QO4n0GEGlyvGa7mDNyGaihqJbj3Kz5wMk~CvcxwHR0pQDDD-Ctb2DzPx4ZIYEYq8puJj35EqFrJ926I~P8VoXOzGk6rX3DOYI2iwP5~24BUAcXcirj93Go-9zRfHuHQOfS3Tsktfo6qkIR~YfATA1CXMZuGE-HLpgK3gqP8j6hFZiV9YyZlu4guwgV9nogTwEWsKCIBW9AjZiAO1609QpIJmOMS2Y9QzdhT~oICyZceEKvU3XcFTM7qtHkw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The process of chromosome rearrangement involved in chromosome 14. Chromosomes 9 and 14 regions are marked by green and blue, respectively, and the two homologues of chromosome 14 are distinguished by vertical and horizontal patterns. Because chromosome 14 consists of two different allele contents in the SNP array, it must have originated from a normal pair with different allele profiles. Variant analysis in the sorted chromosome samples show that 14S and der(9) have different chromosome 14 allelic profiles, but der(9) and 14U have the same chromosome 14 allelic profiles. This indicates that the origin of der(9) was associated with duplication of 14U. One of the homologues, 14S, does not have detectable changes at chromosomal level and der(9) remains unchanged during cell culture. However, duplication of the long arm of 14U is fused with 9p to form der(9). The 14U chromosome at passage P20 has a cryptic deletion in the same region as the breakpoint in der(9). Black circles and squares show telomere and centromere, respectively. Color figure is available in online version." path-from-xml="dsy007f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/120121791/dsy007f5.tif" path-from-xml="dsy007f5.tif" target="_blank">View large</a><a section="120121791" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007f5.png?Expires=2147483647&Signature=ESYZZZ24He5yXjbBSaeNtRndRhytFG6v9y9OoFIsqdWSUojKPMwHy3Ylrm1~dLphT5aNMqaVcP-SS20LAGCxteSDsf2nfeefDtSj69u-Rfh2EoIJwZhm54WMksVOENTeXCYboyy~KJHOJngTWTl6ZES3EMEsXlr4GObYNcLF0UDlVrGJzWZKpA7qhtgfkI8D3oRx9ApVb4XADoqE2wb33PKLHtLw5MKNiBoM4ZpBXDDtTHsVfsHpL4Dzgf53OpU7FApKIKPVmA-B5i4zrDY8ewJNYHvj0T2Kdv~zKnuixPeWVokw43~P1McM2Na7OKouoF1yVYvfqriF1fPjJ-mCcg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=120121791&ar=4922629&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsy007f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The process of chromosome rearrangement involved in chromosome 14. Chromosomes 9 and 14 regions are marked by green and blue, respectively, and the two homologues of chromosome 14 are distinguished by vertical and horizontal patterns. Because chromosome 14 consists of two different allele contents in the SNP array, it must have originated from a normal pair with different allele profiles. Variant analysis in the sorted chromosome samples show that 14S and der(9) have different chromosome 14 allelic profiles, but der(9) and 14U have the same chromosome 14 allelic profiles. This indicates that the origin of der(9) was associated with duplication of 14U. One of the homologues, 14S, does not have detectable changes at chromosomal level and der(9) remains unchanged during cell culture. However, duplication of the long arm of 14U is fused with 9p to form der(9). The 14U chromosome at passage P20 has a cryptic deletion in the same region as the breakpoint in der(9). Black circles and squares show telomere and centromere, respectively. Color figure is available in online version.</p></div></div><p>Our chromosome sorting using a dual laser flow cytometer was first introduced in 1992 to generate chromosome-specific DNA for diagnostic cytogenetics.<a reveal-id="dsy007-B14" data-open="dsy007-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> This led to the development of chromosome painting probes for phylogenetic studies in a wide variety of species including human that has revealed homologous syntenic blocks conserved between species not only in mammals but also among birds and reptiles.<a reveal-id="dsy007-B15" data-open="dsy007-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a><sup>,</sup><a reveal-id="dsy007-B27" data-open="dsy007-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><a reveal-id="dsy007-B28" data-open="dsy007-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> These studies provide many examples of chromosome-specific mapping and sequencing. In one study in dogs (2<em>n</em> = 78), although there are many acrocentric chromosomes of similar size and seven peaks in the flow karyotype that contain 2 chromosome pairs,<a reveal-id="dsy007-B29" data-open="dsy007-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> each dog chromosome was successfully isolated, some by single chromosome sorting; a complete set of painting probes was produced with each probe specific for a single type of dog chromosome.<a reveal-id="dsy007-B30" data-open="dsy007-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> Another study demonstrated that chicken microchromosomes can be characterized by single chromosome sorting.<a reveal-id="dsy007-B31" data-open="dsy007-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Single chromosome sorting has been applied more recently to haplotyping normal human chromosome 19, by demonstrating allele heterozygosity.<a reveal-id="dsy007-B32" data-open="dsy007-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> When derivative chromosomes are different in size from the normal homologues, they form a unique peak in the flow karyotype and can be resolved easily and collected by flow sorting.<a reveal-id="dsy007-B33" data-open="dsy007-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> We show here the isolation of chromosome homologues in a tumour cell line by chromosome sorting that allows homologue-specific sequencing, chromosome assignment of allelic variants and improved characterization of abnormal chromosomes.</p>                            <h2 scrollto-destination=120121793 id="120121793" class="backacknowledgements-title" >Acknowledgements</h2>
<p>Authors would like to thank Dr. Ryoka Miki (Thermo Fisher Scientific, Japan) and Mr. Takashi Ishikura (Thermo Fisher Scientific, Japan) for their technical comments on data analysis.</p>                            <h3 scrollto-destination=120121795 id="120121795" class="section-title" >Conflict of interest</h3>
<p>None declared. </p>                            <h2 scrollto-destination=120121797 id="120121797" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsy007-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Henson</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Tischler</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Ning</div> <div class="given-names">Z.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Next-generation sequencing and large genome assemblies</div>, <div class="source">Pharmacogenomics</div> , <div class="volume">13</div>, <div class="fpage">901</div>–<div class="lpage">15</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Next-generation%20sequencing%20and%20large%20genome%20assemblies&amp;author=J.%20Henson&amp;author=G.%20Tischler&amp;author=Z.%20Ning&amp;publication_year=2012&amp;journal=Pharmacogenomics&amp;volume=13&amp;pages=901-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2217/pgs.12.72" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2217%2Fpgs.12.72" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2217%2Fpgs.12.72"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22676195" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pabinger</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Dander</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Fischer</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">A survey of tools for variant analysis of next-generation genome sequencing data,</div> <div class="source">Brief. Bioinformatics</div> , <div class="volume">15</div>, <div class="fpage">256</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20survey%20of%20tools%20for%20variant%20analysis%20of%20next-generation%20genome%20sequencing%20data%2C&amp;author=S.%20Pabinger&amp;author=A.%20Dander&amp;author=M.%20Fischer&amp;publication_year=2014&amp;journal=Brief.%20Bioinformatics&amp;volume=15&amp;pages=256-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbs086" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbs086" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbs086"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23341494" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Barretina</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Caponigro</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Stransky</div> <div class="given-names">N.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">The cancer cell line encyclopedia enables predictive modelling of anticancer drug sensitivity</div>, <div class="source">Nature</div> , <div class="volume">483</div>, <div class="fpage">603</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20cancer%20cell%20line%20encyclopedia%20enables%20predictive%20modelling%20of%20anticancer%20drug%20sensitivity&amp;author=J.%20Barretina&amp;author=G.%20Caponigro&amp;author=N.%20Stransky&amp;publication_year=2012&amp;journal=Nature&amp;volume=483&amp;pages=603-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22460905" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Forbes</div> <div class="given-names">S. A.</div></div>, <div class="name string-name"><div class="surname">Beare</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Boutselakis</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">COSMIC: somatic cancer genetics at high-resolution</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">45</div>, <div class="fpage">D777</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=COSMIC%3A%20somatic%20cancer%20genetics%20at%20high-resolution&amp;author=S.%20A.%20Forbes&amp;author=D.%20Beare&amp;author=H.%20Boutselakis&amp;publication_year=2017&amp;journal=Nucleic%20Acids%20Res&amp;volume=45&amp;pages=D777-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkw1121" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkw1121" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkw1121"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27899578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marusyk</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Polyak</div> <div class="given-names">K.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Tumor heterog eneity: causes and consequences</div>, <div class="source">Biochim. Biophys. Acta</div> , <div class="volume">1805</div>, <div class="fpage">105</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tumor%20heterog%20eneity%3A%20causes%20and%20consequences&amp;author=A.%20Marusyk&amp;author=K.%20Polyak&amp;publication_year=2010&amp;journal=Biochim.%20Biophys.%20Acta&amp;volume=1805&amp;pages=105-17" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19931353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Greaves</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Maley</div> <div class="given-names">C.C.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Clonal evolution in cancer</div>, <div class="source">Nature</div> , <div class="volume">481</div>, <div class="fpage">306</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Clonal%20evolution%20in%20cancer&amp;author=M.%20Greaves&amp;author=C.C.%20Maley&amp;publication_year=2012&amp;journal=Nature&amp;volume=481&amp;pages=306-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature10762" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature10762" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature10762"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22258609" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kasai</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Hirayama</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Ozawa</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Iemura</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kohara</div> <div class="given-names">A.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Changes of heterogeneous cell population in the Ishikawa cell line during long-term culture; proposal of an <em>in vitro</em> clonal evolution model of tumor cells</div>, <div class="source">Genomics</div> , <div class="volume">107</div>, <div class="fpage">259</div>–<div class="lpage">66</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Changes%20of%20heterogeneous%20cell%20population%20in%20the%20Ishikawa%20cell%20line%20during%20long-term%20culture%3B%20proposal%20of%20an%20in%20vitro%20clonal%20evolution%20model%20of%20tumor%20cells&amp;author=F.%20Kasai&amp;author=N.%20Hirayama&amp;author=M.%20Ozawa&amp;author=M.%20Iemura&amp;author=A.%20Kohara&amp;publication_year=2016&amp;journal=Genomics&amp;volume=107&amp;pages=259-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2016.04.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2016.04.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2016.04.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27107655" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Potter</div> <div class="given-names">N.E.</div></div>, <div class="name string-name"><div class="surname">Ermini</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Papaemmanuil</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Single-cell mutational profiling and clonal phylogeny in cancer</div>, <div class="source">Genome Res</div> ., <div class="volume">23</div>, <div class="fpage">2115</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Single-cell%20mutational%20profiling%20and%20clonal%20phylogeny%20in%20cancer&amp;author=N.E.%20Potter&amp;author=L.%20Ermini&amp;author=E.%20Papaemmanuil&amp;publication_year=2013&amp;journal=Genome%20Res&amp;volume=23&amp;pages=2115-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.159913.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.159913.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.159913.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24056532" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carter</div> <div class="given-names">N.P.</div></div>, <div class="name string-name"><div class="surname">Ferguson-Smith</div> <div class="given-names">M.E.</div></div>, <div class="name string-name"><div class="surname">Affara</div> <div class="given-names">N.A.</div></div>, <div class="name string-name"><div class="surname">Briggs</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Ferguson-Smith</div> <div class="given-names">M.A.</div></div></div> <div class="year">1990</div>, 
<div class="article-title">Study of X chromosome abnormality in XX males using bivariate flow karyotype analysis and flow sorted dot blots</div>, <div class="source">Cytometry</div> , <div class="volume">11</div>, <div class="fpage">202</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Study%20of%20X%20chromosome%20abnormality%20in%20XX%20males%20using%20bivariate%20flow%20karyotype%20analysis%20and%20flow%20sorted%20dot%20blots&amp;author=N.P.%20Carter&amp;author=M.E.%20Ferguson-Smith&amp;author=N.A.%20Affara&amp;author=H.%20Briggs&amp;author=M.A.%20Ferguson-Smith&amp;publication_year=1990&amp;journal=Cytometry&amp;volume=11&amp;pages=202-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0320" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0320" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0320"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2106420" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kasai</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">O'Brien</div> <div class="given-names">P.C.M.</div></div>, <div class="name string-name"><div class="surname">Ferguson-Smith</div> <div class="given-names">M.A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Reassessment of genome size in turtle and crocodile based on chromosome measurement by flow karyotyping: close similarity to chicken</div>, <div class="source">Biol. Lett</div> ., <div class="volume">8</div>, <div class="fpage">631</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reassessment%20of%20genome%20size%20in%20turtle%20and%20crocodile%20based%20on%20chromosome%20measurement%20by%20flow%20karyotyping%3A%20close%20similarity%20to%20chicken&amp;author=F.%20Kasai&amp;author=P.C.M.%20O'Brien&amp;author=M.A.%20Ferguson-Smith&amp;publication_year=2012&amp;journal=Biol.%20Lett&amp;volume=8&amp;pages=631-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rsbl.2012.0141" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frsbl.2012.0141" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frsbl.2012.0141"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22491763" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kasai</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">O'Brien</div> <div class="given-names">P.C.M.</div></div>, <div class="name string-name"><div class="surname">Ferguson-Smith</div> <div class="given-names">M.A.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Afrotheria genome; overestimation of genome size and distinct chromosome GC content revealed by flow karyotyping</div>, <div class="source">Genomics</div> , <div class="volume">102</div>, <div class="fpage">468</div>–<div class="lpage">71</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Afrotheria%20genome%3B%20overestimation%20of%20genome%20size%20and%20distinct%20chromosome%20GC%20content%20revealed%20by%20flow%20karyotyping&amp;author=F.%20Kasai&amp;author=P.C.M.%20O'Brien&amp;author=M.A.%20Ferguson-Smith&amp;publication_year=2013&amp;journal=Genomics&amp;volume=102&amp;pages=468-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2013.09.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2013.09.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2013.09.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24055950" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ferguson-Smith</div> <div class="given-names">M.A.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">Genetic analysis by chromosome sorting and painting: phylogenetic and diagnostic applications</div>, <div class="source">Eur. J. Hum. Genet</div> ., <div class="volume">5</div>, <div class="fpage">253</div>–<div class="lpage">65</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analysis%20by%20chromosome%20sorting%20and%20painting%3A%20phylogenetic%20and%20diagnostic%20applications&amp;author=M.A.%20Ferguson-Smith&amp;publication_year=1997&amp;journal=Eur.%20J.%20Hum.%20Genet&amp;volume=5&amp;pages=253-65" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9412781" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kasai</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Yoshihara</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Matsukuma</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">O'Brien</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Ferguson-Smith</div> <div class="given-names">M.A.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Emergence of complex rearrangements at translocation breakpoints in a transgenic mouse; implications for mechanisms involved in the formation of chromosome rearrangements</div>, <div class="source">Cytogenet. Genome Res</div> ., <div class="volume">119</div>, <div class="fpage">83</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Emergence%20of%20complex%20rearrangements%20at%20translocation%20breakpoints%20in%20a%20transgenic%20mouse%3B%20implications%20for%20mechanisms%20involved%20in%20the%20formation%20of%20chromosome%20rearrangements&amp;author=F.%20Kasai&amp;author=M.%20Yoshihara&amp;author=S.%20Matsukuma&amp;author=P.%20O'Brien&amp;author=M.A.%20Ferguson-Smith&amp;publication_year=2007&amp;journal=Cytogenet.%20Genome%20Res&amp;volume=119&amp;pages=83-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000109623" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000109623" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000109623"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18160786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Carter</div> <div class="given-names">N.P.</div></div>, <div class="name string-name"><div class="surname">Ferguson-Smith</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Perryman</div> <div class="given-names">M.T.</div></div></div>, et al.   <div class="year">1992</div>, 
<div class="article-title">Reverse chromosome painting: a method for the rapid analysis of aberrant chromosomes in clinical cytogenetics</div>, <div class="source">J. Med. Genet</div> ., <div class="volume">29</div>, <div class="fpage">299</div>–<div class="lpage">307</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reverse%20chromosome%20painting%3A%20a%20method%20for%20the%20rapid%20analysis%20of%20aberrant%20chromosomes%20in%20clinical%20cytogenetics&amp;author=N.P.%20Carter&amp;author=M.A.%20Ferguson-Smith&amp;author=M.T.%20Perryman&amp;publication_year=1992&amp;journal=J.%20Med.%20Genet&amp;volume=29&amp;pages=299-307" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1136/jmg.29.5.299" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1136%2Fjmg.29.5.299" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1136%2Fjmg.29.5.299"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1583656" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ferguson-Smith</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">O'Brien</div> <div class="given-names">P.C.</div></div></div> <div class="year">1998</div>, 
<div class="article-title">Comparative mapping using chromosome sorting and painting</div>, <div class="source">ILAR J</div> ., <div class="volume">39</div>, <div class="fpage">68</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20mapping%20using%20chromosome%20sorting%20and%20painting&amp;author=M.A.%20Ferguson-Smith&amp;author=F.%20Yang&amp;author=P.C.%20O'Brien&amp;publication_year=1998&amp;journal=ILAR%20J&amp;volume=39&amp;pages=68-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/ilar.39.2-3.68" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Filar.39.2-3.68" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Filar.39.2-3.68"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11528066" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Edwards</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Cameron</div> <div class="given-names">E.Z.</div></div>, <div class="name string-name"><div class="surname">Pereira</div> <div class="given-names">J.C.</div></div>, <div class="name string-name"><div class="surname">Wapstra</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Ferguson-Smith</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Horton</div> <div class="given-names">S.R.</div></div>, <div class="name string-name"><div class="surname">Thomasson</div> <div class="given-names">K.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Gestational experience alters sex allocation in the subsequent generation</div>, <div class="source">R. Soc. Open Sci</div> ., <div class="volume">3</div>, <div class="fpage">160210</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gestational%20experience%20alters%20sex%20allocation%20in%20the%20subsequent%20generation&amp;author=A.M.%20Edwards&amp;author=E.Z.%20Cameron&amp;author=J.C.%20Pereira&amp;author=E.%20Wapstra&amp;author=M.A.%20Ferguson-Smith&amp;author=S.R.%20Horton&amp;author=K.%20Thomasson&amp;publication_year=2016&amp;journal=R.%20Soc.%20Open%20Sci&amp;volume=3&amp;pages=160210" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rsos.160210" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frsos.160210" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frsos.160210"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27493776" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gomes</div> <div class="given-names">A.J.</div></div>, <div class="name string-name"><div class="surname">Nagamachi</div> <div class="given-names">C.Y.</div></div>, <div class="name string-name"><div class="surname">Rodrigues</div> <div class="given-names">L.R.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Chromosomal phylogeny of Vampyressine bats (Chiroptera, Phyllostomidae) with description of two new sex chromosome systems</div>, <div class="source">BMC Evol. Biol</div> ., <div class="volume">16</div>, <div class="fpage">119</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20phylogeny%20of%20Vampyressine%20bats%20(Chiroptera%2C%20Phyllostomidae)%20with%20description%20of%20two%20new%20sex%20chromosome%20systems&amp;author=A.J.%20Gomes&amp;author=C.Y.%20Nagamachi&amp;author=L.R.%20Rodrigues&amp;publication_year=2016&amp;journal=BMC%20Evol.%20Biol&amp;volume=16&amp;pages=119" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s12862-016-0689-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs12862-016-0689-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs12862-016-0689-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27260645" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Nishida</div> <div class="given-names">M.</div></div></div> <div class="year">1985</div>, 
<div class="article-title">Establishment of a new human endometrial adenocarcinoma cell line, Ishikawa cells, containing estrogen and progesterone receptors</div>, <div class="source">Nihon Sanka Fujinka Gakkai Zasshi</div> , <div class="volume">37</div>, <div class="fpage">1103</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Establishment%20of%20a%20new%20human%20endometrial%20adenocarcinoma%20cell%20line%2C%20Ishikawa%20cells%2C%20containing%20estrogen%20and%20progesterone%20receptors&amp;author=M.%20Nishida&amp;publication_year=1985&amp;journal=Nihon%20Sanka%20Fujinka%20Gakkai%20Zasshi&amp;volume=37&amp;pages=1103-11" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/4031568" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Aplan</div> <div class="given-names">P. D.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Causes of oncogenic chromosomal translocation</div>, <div class="source">Trends Genet</div> ., <div class="volume">22</div>, <div class="fpage">46</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Causes%20of%20oncogenic%20chromosomal%20translocation&amp;author=P.%20D.%20Aplan&amp;publication_year=2006&amp;journal=Trends%20Genet&amp;volume=22&amp;pages=46-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2005.10.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2005.10.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2005.10.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16257470" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Roukos</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Misteli</div> <div class="given-names">T.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">The biogenesis of chromosome translocations</div>, <div class="source">Nat. Cell Biol</div> ., <div class="volume">16</div>, <div class="fpage">293</div>–<div class="lpage">300</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20biogenesis%20of%20chromosome%20translocations&amp;author=V.%20Roukos&amp;author=T.%20Misteli&amp;publication_year=2014&amp;journal=Nat.%20Cell%20Biol&amp;volume=16&amp;pages=293-300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncb2941" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncb2941" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncb2941"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24691255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Tsurusaki</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Nakashima</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Miyake</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Saitsu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Takeda</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">N.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Precise detection of chromosomal translocation or inversion breakpoints by whole-genome sequencing</div>, <div class="source">J. Hum. Genet</div> ., <div class="volume">59</div>, <div class="fpage">649</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Precise%20detection%20of%20chromosomal%20translocation%20or%20inversion%20breakpoints%20by%20whole-genome%20sequencing&amp;author=T.%20Suzuki&amp;author=Y.%20Tsurusaki&amp;author=M.%20Nakashima&amp;author=N.%20Miyake&amp;author=H.%20Saitsu&amp;author=S.%20Takeda&amp;author=N.%20Matsumoto&amp;publication_year=2014&amp;journal=J.%20Hum.%20Genet&amp;volume=59&amp;pages=649-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/jhg.2014.88" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fjhg.2014.88" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fjhg.2014.88"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25296578" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ge</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Liu</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Juan</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Fang</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Newman</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hoeck</div> <div class="given-names">W.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">FusionMap: detecting fusion genes from next-generation sequencing data at base-pair resolution</div>, <div class="source">Bioinformatics,</div>  <div class="volume">27</div>, <div class="fpage">1922</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FusionMap%3A%20detecting%20fusion%20genes%20from%20next-generation%20sequencing%20data%20at%20base-pair%20resolution&amp;author=H.%20Ge&amp;author=K.%20Liu&amp;author=T.%20Juan&amp;author=F.%20Fang&amp;author=M.%20Newman&amp;author=W.%20Hoeck&amp;publication_year=2011&amp;journal=Bioinformatics%2C&amp;volume=27&amp;pages=1922-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr310" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr310" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr310"> </span></div> </div></div></div></div></div><div content-id="dsy007-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ford</div> <div class="given-names">A.M.</div></div>, <div class="name string-name"><div class="surname">Mansur</div> <div class="given-names">M.B.</div></div>, <div class="name string-name"><div class="surname">Furness</div> <div class="given-names">C.L.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Protracted dormancy of pre-leukemic stem cells</div>, <div class="source">Leukemia</div> , <div class="volume">29</div>, <div class="fpage">2202</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Protracted%20dormancy%20of%20pre-leukemic%20stem%20cells&amp;author=A.M.%20Ford&amp;author=M.B.%20Mansur&amp;author=C.L.%20Furness&amp;publication_year=2015&amp;journal=Leukemia&amp;volume=29&amp;pages=2202-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/leu.2015.132" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fleu.2015.132" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fleu.2015.132"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26017033" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kawamura</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Taki</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Kaku</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Ohki</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Hayashi</div> <div class="given-names">Y.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Identification of SPAG9 as a novel JAK2 fusion partner gene in pediatric acute lymphoblastic leukemia with t(9; 17)(p24; q21)</div>, <div class="source">Genes Chromosomes Cancer</div> , <div class="volume">54</div>, <div class="fpage">401</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20SPAG9%20as%20a%20novel%20JAK2%20fusion%20partner%20gene%20in%20pediatric%20acute%20lymphoblastic%20leukemia%20with%20t(9%3B%2017)(p24%3B%20q21)&amp;author=M.%20Kawamura&amp;author=T.%20Taki&amp;author=H.%20Kaku&amp;author=K.%20Ohki&amp;author=Y.%20Hayashi&amp;publication_year=2015&amp;journal=Genes%20Chromosomes%20Cancer&amp;volume=54&amp;pages=401-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/gcc.22251" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fgcc.22251" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fgcc.22251"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25951811" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bersten</div> <div class="given-names">D.C.</div></div>, <div class="name string-name"><div class="surname">Sullivan</div> <div class="given-names">A.E.</div></div>, <div class="name string-name"><div class="surname">Peet</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Whitelaw</div> <div class="given-names">M.L.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">bHLH-PAS proteins in cancer</div>, <div class="source">Nat. Rev. Cancer</div> , <div class="volume">13</div>, <div class="fpage">827</div>–<div class="lpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=bHLH-PAS%20proteins%20in%20cancer&amp;author=D.C.%20Bersten&amp;author=A.E.%20Sullivan&amp;author=D.J.%20Peet&amp;author=M.L.%20Whitelaw&amp;publication_year=2013&amp;journal=Nat.%20Rev.%20Cancer&amp;volume=13&amp;pages=827-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrc3621" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrc3621" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrc3621"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24263188" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Moreira</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Kiehl</div> <div class="given-names">T.R.</div></div>, <div class="name string-name"><div class="surname">So</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">NPAS3 demonstrates features of a tumor suppressive role in driving the progression of Astrocytomas</div>, <div class="source">Am. J. Pathol</div> ., <div class="volume">179</div>, <div class="fpage">462</div>–<div class="lpage">76</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=NPAS3%20demonstrates%20features%20of%20a%20tumor%20suppressive%20role%20in%20driving%20the%20progression%20of%20Astrocytomas&amp;author=F.%20Moreira&amp;author=T.R.%20Kiehl&amp;author=K.%20So&amp;publication_year=2011&amp;journal=Am.%20J.%20Pathol&amp;volume=179&amp;pages=462-76" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ajpath.2011.03.044" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ajpath.2011.03.044" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ajpath.2011.03.044"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21703424" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ferguson-Smith</div> <div class="given-names">M.A.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Rens</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">O'Brien</div> <div class="given-names">P.C.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">The impact of chromosome sorting and painting on the comparative analysis of primate genomes</div>, <div class="source">Cytogenet. Genome Res</div> ., <div class="volume">108</div>, <div class="fpage">112</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20impact%20of%20chromosome%20sorting%20and%20painting%20on%20the%20comparative%20analysis%20of%20primate%20genomes&amp;author=M.A.%20Ferguson-Smith&amp;author=F.%20Yang&amp;author=W.%20Rens&amp;author=P.C.%20O'Brien&amp;publication_year=2005&amp;journal=Cytogenet.%20Genome%20Res&amp;volume=108&amp;pages=112-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000080809" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000080809" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000080809"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15545723" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kasai</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">O'Brien</div> <div class="given-names">P.C.</div></div>, <div class="name string-name"><div class="surname">Martin</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Ferguson-Smith</div> <div class="given-names">M.A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Extensive homology of chicken macrochromosomes in the karyotypes of <em>Trachemys scripta elegans</em> and <em>Crocodylus niloticus</em> revealed by chromosome painting despite long divergence times</div>, <div class="source">Cytogenet. Genome Res</div> ., <div class="volume">136</div>, <div class="fpage">303</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20homology%20of%20chicken%20macrochromosomes%20in%20the%20karyotypes%20of%20Trachemys%20scripta%20elegans%20and%20Crocodylus%20niloticus%20revealed%20by%20chromosome%20painting%20despite%20long%20divergence%20times&amp;author=F.%20Kasai&amp;author=P.C.%20O'Brien&amp;author=S.%20Martin&amp;author=M.A.%20Ferguson-Smith&amp;publication_year=2012&amp;journal=Cytogenet.%20Genome%20Res&amp;volume=136&amp;pages=303-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000338111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000338111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000338111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22572532" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">O'Brien</div> <div class="given-names">P.C.</div></div>, <div class="name string-name"><div class="surname">Milne</div> <div class="given-names">B.S.</div></div></div>, et al.   <div class="year">1999</div>, 
<div class="article-title">A complete comparative chromosome map for the dog, red fox, and human and its integration with canine genetic maps</div>, <div class="source">Genomics</div> , <div class="volume">62</div>, <div class="fpage">189</div>–<div class="lpage">202</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20complete%20comparative%20chromosome%20map%20for%20the%20dog%2C%20red%20fox%2C%20and%20human%20and%20its%20integration%20with%20canine%20genetic%20maps&amp;author=F.%20Yang&amp;author=P.C.%20O'Brien&amp;author=B.S.%20Milne&amp;publication_year=1999&amp;journal=Genomics&amp;volume=62&amp;pages=189-202" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/geno.1999.5989" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fgeno.1999.5989" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fgeno.1999.5989"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10610712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Graphodatsky</div> <div class="given-names">A.S.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Serdukova</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Perelman</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Zhdanova</div> <div class="given-names">N.S.</div></div>, <div class="name string-name"><div class="surname">Ferguson-Smith</div> <div class="given-names">M.A.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Dog chromosome-specific paints reveal evolutionary inter- and intrachromosomal rearrangements in the American mink and human</div>, <div class="source">Cytogenet. Cell Genet</div> ., <div class="volume">90</div>, <div class="fpage">275</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dog%20chromosome-specific%20paints%20reveal%20evolutionary%20inter-%20and%20intrachromosomal%20rearrangements%20in%20the%20American%20mink%20and%20human&amp;author=A.S.%20Graphodatsky&amp;author=F.%20Yang&amp;author=N.%20Serdukova&amp;author=P.%20Perelman&amp;author=N.S.%20Zhdanova&amp;author=M.A.%20Ferguson-Smith&amp;publication_year=2000&amp;journal=Cytogenet.%20Cell%20Genet&amp;volume=90&amp;pages=275-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1159/000056788" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1159%2F000056788" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1159%2F000056788"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11124533" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Masabanda</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Burt</div> <div class="given-names">D.W.</div></div>, <div class="name string-name"><div class="surname">O'Brien</div> <div class="given-names">P.C.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">Molecular cytogenetic definition of the chicken genome: the first complete avian karyotype</div>, <div class="source">Genetics</div> , <div class="volume">166</div>, <div class="fpage">1367</div>–<div class="lpage">73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cytogenetic%20definition%20of%20the%20chicken%20genome%3A%20the%20first%20complete%20avian%20karyotype&amp;author=J.S.%20Masabanda&amp;author=D.W.%20Burt&amp;author=P.C.%20O'Brien&amp;publication_year=2004&amp;journal=Genetics&amp;volume=166&amp;pages=1367-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.166.3.1367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.166.3.1367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.166.3.1367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15082555" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wong</div> <div class="given-names">W.H.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Completely phased genome sequencing through chromosome sorting</div>, <div class="source">Proc. Natl Acad. Sci. U S A</div> ., <div class="volume">108</div>, <div class="fpage">12</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Completely%20phased%20genome%20sequencing%20through%20chromosome%20sorting&amp;author=H.%20Yang&amp;author=X.%20Chen&amp;author=W.H.%20Wong&amp;publication_year=2011&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U%20S%20A&amp;volume=108&amp;pages=12-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1016725108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1016725108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1016725108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21169219" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsy007-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsy007-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gribble</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Ng</div> <div class="given-names">B.L.</div></div>, <div class="name string-name"><div class="surname">Prigmore</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Fitzgerald</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Carter</div> <div class="given-names">N.P.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Array painting: a protocol for the rapid analysis of aberrant chromosomes using DNA microarrays</div>, <div class="source">Nat. Protoc</div> ., <div class="volume">4</div>, <div class="fpage">1722</div>–<div class="lpage">36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Array%20painting%3A%20a%20protocol%20for%20the%20rapid%20analysis%20of%20aberrant%20chromosomes%20using%20DNA%20microarrays&amp;author=S.M.%20Gribble&amp;author=B.L.%20Ng&amp;author=E.%20Prigmore&amp;author=T.%20Fitzgerald&amp;author=N.P.%20Carter&amp;publication_year=2009&amp;journal=Nat.%20Protoc&amp;volume=4&amp;pages=1722-36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2009.183" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2009.183" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2009.183"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19893508" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author(s) 2018. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4922629&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_figures_s1-s4_tables_s1-s6.pdf?Expires=2147483647&amp;Signature=KvFRPDEdww1SmMJml3HPph9H2jnIujvIWNPN5C5y5a8MtjuuoP6xJS10HCudiRtXXKG5Hbzo8gaG4QBl-g9uqus6DTKB0CWKste1qDMpphh96CbAwGOml8dIq8hAT7qFI23ncN3oFDLJpumD1H5pqEhkUKhSNf7--i6WOPi2Kf6kRRNQ0CVrcn0spXLiXjXGdIosQfo1TASEkk6GCVkXsHoLY3ljxvkcn4VAW6Li7ND6~b9Djm32Roe3gQAFKpni5CgzTx9hBatbBjJ-VqDcR3qRbYNpHQ923GByAeFP3T4FDPLd694voxTV1SMPvNo1FwfspH8XQjjyHD-X8Pf0Vg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figures S1-S4 Tables S1-S6</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/4/10.1093_dnares_dsy007/1/dsy007_supplementary_table_s2r.xlsx?Expires=2147483647&amp;Signature=I6WvEWc8z2oHpUJxY-3VJX4pXx1131d7jHlB9mm9QxBvCFX8CF-FLtDwPXaCP4lQbtGK6etF-Y8cuBg6YFgfI5hDqy39ptmjSutiXDq7wRqrMwffaR~3qTLKb~zKFKj5pmA-P2T6K6iB6jYLGtPhPxYG624ob7H4Q6bLbIsA1QbyzRccqclFrG3qyaLLBFszcEP5siAeG2KbHWHqByMOithxJ00msEEQsM2XHE2cVB7rvjaEI5NbT85CWA5x0KK379woINCw5Jrlh2gxEzOc6xrSKb1NoQMO6F7ksAkD8I~eigRejs7sIz4bNkm8j6K3p~tYv~PqreakcgQjakCsxg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S2</div></a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">780</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000442930900002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">2</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsy007"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4922629"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000442930900002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f4%2f353%2f4922629">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000442930900002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (2)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F4%2F353%2F4922629" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4922629'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Homologue-specific chromosome sequencing characterizes translocation junctions and permits allelic assignment" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsy007");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f4%2f353%2f4922629">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

