

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/2/173/333781","@type":"ScholarlyArticle","name":"Integrated Consensus Map of Cultivated Peanut and Wild Relatives Reveals Structures of the A and B Genomes of  Arachis  and Divergence of the Legume Genomes","datePublished":"2013-01-12","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2013-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss042","keywords":["<italic>Arachis</italic> spp.","comparative genomics","genetic linkage map","integrated consensus map","legume genome"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/2/173/333781","author":[{"name":"Shirasawa, Kenta","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba ,  Japan","@type":"Person"},{"name":"Bertioli, David J.","affiliation":"2  University of Brasília ,  Brasília, DF ,  Brazil","@type":"Person"},{"name":"Varshney, Rajeev K.","affiliation":"3  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru, AP ,  India","@type":"Person"},{"name":"Moretzsohn, Marcio C.","affiliation":"4  Embrapa Genetic Resources and Biotechnology ,  Brasília, DF ,  Brazil","@type":"Person"},{"name":"Leal-Bertioli, Soraya C. M.","affiliation":"4  Embrapa Genetic Resources and Biotechnology ,  Brasília, DF ,  Brazil","@type":"Person"},{"name":"Thudi, Mahendar","affiliation":"3  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru, AP ,  India","@type":"Person"},{"name":"Pandey, Manish K.","affiliation":"3  International Crops Research Institute for the Semi-Arid Tropics (ICRISAT) ,  Patancheru, AP ,  India","@type":"Person"},{"name":"Rami, Jean-Francois","affiliation":"5  Centre de Coopération Internationale en Recherche Agronomique pour le Développement (CIRAD), UMR AGAP ,  F-34398 Montpellier ,  France","@type":"Person"},{"name":"Foncéka, Daniel","affiliation":"5  Centre de Coopération Internationale en Recherche Agronomique pour le Développement (CIRAD), UMR AGAP ,  F-34398 Montpellier ,  France","@type":"Person"},{"name":"Gowda, Makanahally V. C.","affiliation":"6  University of Agricultural Sciences (UAS-D) ,  Dharwad, KA ,  India","@type":"Person"},{"name":"Qin, Hongde","affiliation":"7  University of Georgia ,  Tifton, GA ,  USA    8  Hubei Academy of Agricultural Sciences (HAAS) ,  Wuhan, Hubei ,  China","@type":"Person"},{"name":"Guo, Baozhu","affiliation":"9  USDA-Agricultural Research Service ,  Tifton, GA ,  USA","@type":"Person"},{"name":"Hong, Yanbin","affiliation":"10  Guangdong Academy of Agricultural Sciences (GAAS) ,  Guangzhou, Guangdong ,  China","@type":"Person"},{"name":"Liang, Xuanqiang","affiliation":"10  Guangdong Academy of Agricultural Sciences (GAAS) ,  Guangzhou, Guangdong ,  China","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba ,  Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba ,  Japan","@type":"Person"},{"name":"Isobe, Sachiko","affiliation":"1  Kazusa DNA Research Institute ,  Kisarazu, Chiba ,  Japan","@type":"Person"}],"description":"Abstract.  The complex, tetraploid genome structure of peanut (Arachis hypogaea) has obstructed advances in genetics and genomics in the species. The aim of thi","pageStart":"173","pageEnd":"184","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/2/1/m_cover.gif?Expires=2147483647&Signature=Yq7dbhq6hb-Iyo5ewEGt-Y1T99brR4Xh8FvVnCI2Bg1SIlPGztlGCk-Bm0A9A72SaqDKcetVlEQehNtxyAGT~T55ZFFD19A3hKk~A1XbnQ-JfXYKj2-gYHz-VirOlvLe-8XkC6qvT4h2PniinfIJH6d85JsIf3rhRRtN086VVGvi2qNVR0THPJOeMpQvElVQEEN-4bpo3wEYmq80uwX58y5ScJ0pcJt7sV40xw1Fd6mp92v-su9vXuGT~IIVjyHRDyuiGt~ZiUzcXtor6WX4MT1zcCCf5ySC3Wm7DfaZruIvVsbtJxHUYCwePrgmcTq1vhdeOWHam2svM-jBZyoDoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Integrated Consensus Map of Cultivated Peanut and Wild Relatives Reveals Structures of the A and B Genomes of  Arachis  and Divergence of the Legume Genomes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/2/1/m_cover.gif?Expires=2147483647&Signature=Yq7dbhq6hb-Iyo5ewEGt-Y1T99brR4Xh8FvVnCI2Bg1SIlPGztlGCk-Bm0A9A72SaqDKcetVlEQehNtxyAGT~T55ZFFD19A3hKk~A1XbnQ-JfXYKj2-gYHz-VirOlvLe-8XkC6qvT4h2PniinfIJH6d85JsIf3rhRRtN086VVGvi2qNVR0THPJOeMpQvElVQEEN-4bpo3wEYmq80uwX58y5ScJ0pcJt7sV40xw1Fd6mp92v-su9vXuGT~IIVjyHRDyuiGt~ZiUzcXtor6WX4MT1zcCCf5ySC3Wm7DfaZruIvVsbtJxHUYCwePrgmcTq1vhdeOWHam2svM-jBZyoDoA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Integrated Consensus Map of Cultivated Peanut and Wild Relatives Reveals Structures of the A and B Genomes of  Arachis  and Divergence of the Legume Genomes" />
    <meta property="og:description" content="Abstract.  The complex, tetraploid genome structure of peanut (Arachis hypogaea) has obstructed advances in genetics and genomics in the species. The aim of thi" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/2/173/333781" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-01-12" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Shirasawa, Kenta" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba, Japan" /><meta name="citation_author" content="Bertioli, David J." /><meta name="citation_author_institution" content="University of Brasília, Brasília, DF, Brazil" /><meta name="citation_author" content="Varshney, Rajeev K." /><meta name="citation_author_institution" content="International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru, AP, India" /><meta name="citation_author" content="Moretzsohn, Marcio C." /><meta name="citation_author_institution" content="Embrapa Genetic Resources and Biotechnology, Brasília, DF, Brazil" /><meta name="citation_author" content="Leal-Bertioli, Soraya C. M." /><meta name="citation_author_institution" content="Embrapa Genetic Resources and Biotechnology, Brasília, DF, Brazil" /><meta name="citation_author" content="Thudi, Mahendar" /><meta name="citation_author_institution" content="International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru, AP, India" /><meta name="citation_author" content="Pandey, Manish K." /><meta name="citation_author_institution" content="International Crops Research Institute for the Semi-Arid Tropics (ICRISAT), Patancheru, AP, India" /><meta name="citation_author" content="Rami, Jean-Francois" /><meta name="citation_author_institution" content="Centre de Coopération Internationale en Recherche Agronomique pour le Développement (CIRAD), UMR AGAP, F-34398 Montpellier, France" /><meta name="citation_author" content="Foncéka, Daniel" /><meta name="citation_author_institution" content="Centre de Coopération Internationale en Recherche Agronomique pour le Développement (CIRAD), UMR AGAP, F-34398 Montpellier, France" /><meta name="citation_author" content="Gowda, Makanahally V. C." /><meta name="citation_author_institution" content="University of Agricultural Sciences (UAS-D), Dharwad, KA, India" /><meta name="citation_author" content="Qin, Hongde" /><meta name="citation_author_institution" content="University of Georgia, Tifton, GA, USA" /><meta name="citation_author_institution" content="Hubei Academy of Agricultural Sciences (HAAS), Wuhan, Hubei, China" /><meta name="citation_author" content="Guo, Baozhu" /><meta name="citation_author_institution" content="USDA-Agricultural Research Service, Tifton, GA, USA" /><meta name="citation_author" content="Hong, Yanbin" /><meta name="citation_author_institution" content="Guangdong Academy of Agricultural Sciences (GAAS), Guangzhou, Guangdong, China" /><meta name="citation_author" content="Liang, Xuanqiang" /><meta name="citation_author_institution" content="Guangdong Academy of Agricultural Sciences (GAAS), Guangzhou, Guangdong, China" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba, Japan" /><meta name="citation_author" content="Isobe, Sachiko" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba, Japan" /><meta name="citation_title" content="Integrated Consensus Map of Cultivated Peanut and Wild Relatives Reveals Structures of the A and B Genomes of Arachis and Divergence of the Legume Genomes" /><meta name="citation_firstpage" content="173" /><meta name="citation_lastpage" content="184" /><meta name="citation_doi" content="10.1093/dnares/dss042" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2013/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The rest of the iceberg. Legume diversity and evolution in a phylogenetic context; Plant Physiol.;  citation_year=2003;  citation_volume=131;  citation_pages=900-10; " /><meta name="citation_reference" content="citation_title=A phylogeny of legumes (Leguminosae) based on analysis of the plastid matK gene resolves many well-supported subclades within the family; Am. J. Bot.;  citation_year=2004;  citation_volume=91;  citation_pages=1846-62; " /><meta name="citation_reference" content="citation_title=Estimating genome conservation between crop and model legume species; Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=15289-94; " /><meta name="citation_reference" content="citation_title=Comparative genetic mapping and discovery of linkage disequilibrium across linkage groups in white clover (Trifolium repens L.); G3;  citation_year=2012;  citation_volume=2;  citation_pages=607-17; " /><meta name="citation_reference" content="citation_title=Genome structure of the legume, Lotus japonicus; DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=227-39; " /><meta name="citation_reference" content="citation_title=Genome sequence of the palaeopolyploid soybean; Nature;  citation_year=2010;  citation_volume=463;  citation_pages=178-83; " /><meta name="citation_reference" content="citation_title=The Medicago genome provides insight into the evolution of rhizobial symbioses; Nature;  citation_year=2011;  citation_volume=480;  citation_pages=520-4; " /><meta name="citation_reference" content="citation_title=Draft genome sequence of pigeonpea (Cajanus cajan), an orphan legume crop of resource-poor farmers; Nat. Biotechnol.;  citation_year=2012;  citation_volume=30;  citation_pages=83-9; " /><meta name="citation_reference" content="citation_title=Bridging model and crop legumes through comparative genomics; Plant Physiol.;  citation_year=2005;  citation_volume=137;  citation_pages=1189-96; " /><meta name="citation_reference" content="citation_title=Taxonomia del genero Arachis (Leguminosae); Bonplandia;  citation_year=1994;  citation_volume=8;  citation_pages=1-186; " /><meta name="citation_reference" content="citation_title=A new species in section Arachis of peanuts with a D genome; Am. J. Bot.;  citation_year=1991;  citation_volume=78;  citation_pages=630-7; " /><meta name="citation_reference" content="citation_title=Species relationships among the wild B genome of Arachis species (section Arachis) based on FISH mapping of rDNA loci and heterochromatin detection: a new proposal for genome arrangement; Theor. Appl. Genet.;  citation_year=2010;  citation_volume=121;  citation_pages=1033-46; " /><meta name="citation_reference" content="citation_title=Species relations among wild Arachis species with the A genome as revealed by FISH mapping of rDNA loci and heterochromatin detection; Theor. Appl. Genet.;  citation_year=2009;  citation_volume=118;  citation_pages=1295-307; " /><meta name="citation_reference" content="citation_title=Genomic relationships between the cultivated peanut (Arachis hypogaea, Leguminosae) and its close relatives revealed by double GISH; Am. J. Bot.;  citation_year=2007;  citation_volume=94;  citation_pages=1963-71; " /><meta name="citation_reference" content="citation_title=Physical mapping of the 5S and 18S-25S rRNA genes by FISH as evidence that Arachis duranensis and A. ipaënsis are the wild diploid progenitors of A. hypogaea (Leguminosae); Am. J. Bot.;  citation_year=2004;  citation_volume=91;  citation_pages=1294-303; " /><meta name="citation_reference" content="citation_title=Genome size variation in Arachis hypogaea and A. monticola re-evaluated; Genome;  citation_year=2000;  citation_volume=43;  citation_pages=449-51; " /><meta name="citation_reference" content="citation_title=Development of an RFLP linkaeg map in diploid peanut species; Theor. Appl. Genet.;  citation_year=1993;  citation_volume=87;  citation_pages=379-84; " /><meta name="citation_reference" content="citation_title=Identification of candidate genome regions controlling disease resistance in Arachis; BMC Plant Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=112" /><meta name="citation_reference" content="citation_title=A microsatellite-based, gene-rich linkage map for the AA genome of Arachis (Fabaceae); Theor. Appl. Genet.;  citation_year=2005;  citation_volume=111;  citation_pages=1060-71; " /><meta name="citation_reference" content="citation_title=A linkage map for the B-genome of Arachis (Fabaceae) and its synteny to the A-genome; BMC Plant Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=40" /><meta name="citation_reference" content="citation_title=A high-density genetic map of Arachis duranensis, a diploid ancestor of cultivated peanut; BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=469" /><meta name="citation_reference" content="citation_title=Comparative mapping in intraspecific populations uncovers a high degree of macrosynteny between A- and B-genome diploid species of peanut; BMC Genomics;  citation_year=2012;  citation_volume=13;  citation_pages=608" /><meta name="citation_reference" content="citation_title=Transmission genetics of chromatin from a synthetic amphidiploid to cultivated peanut (Arachis hypogaea L.): broadening the gene pool of a monophyletic polyploid species; Genetics;  citation_year=2001;  citation_volume=159;  citation_pages=823-37; " /><meta name="citation_reference" content="citation_title=Genetic mapping of wild introgressions into cultivated peanut: a way toward enlarging the genetic basis of a recent allotetraploid; BMC Plant Biol.;  citation_year=2009;  citation_volume=9;  citation_pages=103" /><meta name="citation_reference" content="citation_title=Quantitative trait locus analysis and construction of consensus genetic map for drought tolerance traits based on three recombinant inbred line populations in cultivated groundnut (Arachis hypogaea L.); Mol. Breed.;  citation_year=2012;  citation_volume=30;  citation_pages=773-88; " /><meta name="citation_reference" content="citation_title=A SSR-based composite genetic linkage map for the cultivated peanut (Arachis hypogaea L.) genome; BMC Plant Biol.;  citation_year=2010;  citation_volume=10;  citation_pages=17" /><meta name="citation_reference" content="citation_title=A QTL study on late leaf spot and rust revealed one major QTL for molecular breeding for rust resistance in groundnut (Arachis hypogaea L.); Theor. Appl. Genet.;  citation_year=2010;  citation_volume=121;  citation_pages=971-84; " /><meta name="citation_reference" content="citation_title=An integrated genetic linkage map of cultivated peanut (Arachis hypogaea L.) constructed from two RIL populations; Theor. Appl. Genet.;  citation_year=2012;  citation_volume=124;  citation_pages=653-64; " /><meta name="citation_reference" content="citation_title=Identification of several small main-effect QTLs and a large number of epistatic QTLs for drought tolerance related traits in groundnut (Arachis hypogaea L.); Theor. Appl. Genet.;  citation_year=2011;  citation_volume=122;  citation_pages=1119-32; " /><meta name="citation_reference" content="citation_title=Identification of quantitative trait loci for protein content, oil content and oil quality for groundnut (Arachis hypogaea L.); Field Crops Res.;  citation_year=2011;  citation_volume=122;  citation_pages=49-59; " /><meta name="citation_reference" content="citation_title=Quantitative trait locus analysis and construction of consensus genetic map for foliar disease resistance based on two recombinant inbred line populations in cultivated groundnut (Arachis hypogaea L.); Mol. Breed.;  citation_year=2012;  citation_volume=30;  citation_pages=773-88; " /><meta name="citation_reference" content="citation_title=The first SSR-based genetic linkage map for cultivated groundnut (Arachis hypogaea L.); Theor. Appl. Genet.;  citation_year=2009;  citation_volume=118;  citation_pages=729-39; " /><meta name="citation_reference" content="citation_title=Development and characterization of BAC-end sequence derived SSRs, and their incorporation into a new higher density genetic map for cultivated peanut (Arachis hypogaea L.); BMC Plant Biol.;  citation_year=2012;  citation_volume=12;  citation_pages=10" /><meta name="citation_reference" content="citation_title=Advances in Arachis genomics for peanut improvement; Biotechnol. Adv.;  citation_year=2012;  citation_volume=30;  citation_pages=639-51; " /><meta name="citation_reference" content="citation_title=An international reference consensus genetic map with 897 marker loci based on 11 mapping populations for tetraploid groundnut (Arachis hypogaea L.); PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e41213" /><meta name="citation_reference" content="citation_title=In silico polymorphism analysis for the development of simple sequence repeat and transposon markers and construction of linkage map in cultivated peanut; BMC Plant Biol.;  citation_year=2012;  citation_volume=12;  citation_pages=80" /><meta name="citation_reference" content="citation_title=Large-scale development of expressed sequence tag-derived simple sequence repeat markers and diversity analysis in Arachis spp.; Mol. Breed.;  citation_year=2012;  citation_volume=30;  citation_pages=125-38; " /><meta name="citation_reference" content="citation_title=Characterization of active miniature inverted-repeat transposable elements in the peanut genome; Theor. Appl. Genet.;  citation_year=2012;  citation_volume=124;  citation_pages=1429-38; " /><meta name="citation_reference" content="citation_title=An analysis of synteny of Arachis with Lotus and Medicago sheds new light on the structure, stability and evolution of legume genomes; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=45" /><meta name="citation_reference" content="citation_title=Genetic linkage maps of Eucalyptus grandis and Eucalyptus urophylla using a pseudo-testcross: mapping strategy and RAPD markers; Genetics;  citation_year=1994;  citation_volume=137;  citation_pages=1121-37; " /><meta name="citation_reference" content="citation_publisher=Kyazma. BV, Wageningen, The Netherlands; JoinMap®4, Software for the calculation of genetic linkage maps in experimental populations;  citation_year=2006; " /><meta name="citation_reference" content="citation_title=MapChart: software for the graphical presentation of linkage maps and QTLs; J. Hered.;  citation_year=2002;  citation_volume=93;  citation_pages=77-8; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Circos: an information aesthetic for comparative genomics; Genome Res.;  citation_year=2009;  citation_volume=19;  citation_pages=1639-45; " /><meta name="citation_reference" content="citation_title=FIDEL-a retrovirus-like retrotransposon and its distinct evolutionary histories in the A- and B-genome components of cultivated peanut; Chromosome Res.;  citation_year=2010;  citation_volume=18;  citation_pages=227-46; " /><meta name="citation_reference" content="citation_title=Matita, a new retroelement from peanut: characterization and evolutionary context in the light of the Arachis A-B genome divergence; Mol. Genet. Genomics;  citation_year=2012;  citation_volume=287;  citation_pages=21-38; " /><meta name="citation_reference" content="citation_title=MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations; Genomics;  citation_year=1987;  citation_volume=1;  citation_pages=174-81; " /><meta name="citation_reference" content="citation_title=Development of a large SNP genotyping array and generation of high-density genetic maps in tomato; PLoS ONE;  citation_year=2012;  citation_volume=7;  citation_pages=e40563" /><meta name="citation_reference" content="citation_title=An interspecific linkage map of SSR and intronic polymorphism markers in tomato; Theor. Appl. Genet.;  citation_year=2010;  citation_volume=121;  citation_pages=731-9; " /><meta name="citation_reference" content="citation_title=SNP discovery and linkage map construction in cultivated tomato; DNA Res.;  citation_year=2010;  citation_volume=17;  citation_pages=381-91; " /><meta name="citation_reference" content="citation_title=Chromosomal and genic barriers to introgression in Helianthus; Genetics;  citation_year=1995;  citation_volume=141;  citation_pages=1163-71; " /><meta name="citation_reference" content="citation_title=Evolutionary rates analysis of Leguminosae implicates a rapid diversification of lineages during the tertiary; Syst. Biol.;  citation_year=2005;  citation_volume=54;  citation_pages=575" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/2/173/954974/dss042.pdf" /><meta name="description" content="Abstract.  The complex, tetraploid genome structure of peanut (Arachis hypogaea) has obstructed advances in genetics and genomics in the species. The aim of thi" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/2/173/333781" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/2/173/333781" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/2/173/333781");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss042';
    </script>
<title>
	Integrated Consensus Map of Cultivated Peanut and Wild Relatives Reveals Structures of the A and B Genomes of Arachis and Divergence of the Legume Genomes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./333781" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfCHATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2hpcmFzYXdhLCBLZW50YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLaXNhcmF6dSwgQ2hpYmEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmVydGlvbGksIERhdmlkIEouIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVW5pdmVyc2l0eSBvZiBCcmFzw61saWEsIEJyYXPDrWxpYSwgREYsIEJyYXppbCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlZhcnNobmV5LCBSYWplZXYgSy4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnRlcm5hdGlvbmFsIENyb3BzIFJlc2VhcmNoIEluc3RpdHV0ZSBmb3IgdGhlIFNlbWktQXJpZCBUcm9waWNzIChJQ1JJU0FUKSwgUGF0YW5jaGVydSwgQVAsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTW9yZXR6c29obiwgTWFyY2lvIEMuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRW1icmFwYSBHZW5ldGljIFJlc291cmNlcyBhbmQgQmlvdGVjaG5vbG9neSwgQnJhc8OtbGlhLCBERiwgQnJhemlsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGVhbC1CZXJ0aW9saSwgU29yYXlhIEMuIE0uIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRW1icmFwYSBHZW5ldGljIFJlc291cmNlcyBhbmQgQmlvdGVjaG5vbG9neSwgQnJhc8OtbGlhLCBERiwgQnJhemlsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGh1ZGksIE1haGVuZGFyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW50ZXJuYXRpb25hbCBDcm9wcyBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIHRoZSBTZW1pLUFyaWQgVHJvcGljcyAoSUNSSVNBVCksIFBhdGFuY2hlcnUsIEFQLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBhbmRleSwgTWFuaXNoIEsuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSW50ZXJuYXRpb25hbCBDcm9wcyBSZXNlYXJjaCBJbnN0aXR1dGUgZm9yIHRoZSBTZW1pLUFyaWQgVHJvcGljcyAoSUNSSVNBVCksIFBhdGFuY2hlcnUsIEFQLCBJbmRpYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlJhbWksIEplYW4tRnJhbmNvaXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDZW50cmUgZGUgQ29vcMOpcmF0aW9uIEludGVybmF0aW9uYWxlIGVuIFJlY2hlcmNoZSBBZ3Jvbm9taXF1ZSBwb3VyIGxlIETDqXZlbG9wcGVtZW50IChDSVJBRCksIFVNUiBBR0FQLCBGLTM0Mzk4IE1vbnRwZWxsaWVyLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGb25jw6lrYSwgRGFuaWVsIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ2VudHJlIGRlIENvb3DDqXJhdGlvbiBJbnRlcm5hdGlvbmFsZSBlbiBSZWNoZXJjaGUgQWdyb25vbWlxdWUgcG91ciBsZSBEw6l2ZWxvcHBlbWVudCAoQ0lSQUQpLCBVTVIgQUdBUCwgRi0zNDM5OCBNb250cGVsbGllciwgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iR293ZGEsIE1ha2FuYWhhbGx5IFYuIEMuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVW5pdmVyc2l0eSBvZiBBZ3JpY3VsdHVyYWwgU2NpZW5jZXMgKFVBUy1EKSwgRGhhcndhZCwgS0EsIEluZGlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUWluLCBIb25nZGUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJVbml2ZXJzaXR5IG9mIEdlb3JnaWEsIFRpZnRvbiwgR0EsIFVTQSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikh1YmVpIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzIChIQUFTKSwgV3VoYW4sIEh1YmVpLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikd1bywgQmFvemh1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iVVNEQS1BZ3JpY3VsdHVyYWwgUmVzZWFyY2ggU2VydmljZSwgVGlmdG9uLCBHQSwgVVNBIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSG9uZywgWWFuYmluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3Vhbmdkb25nIEFjYWRlbXkgb2YgQWdyaWN1bHR1cmFsIFNjaWVuY2VzIChHQUFTKSwgR3Vhbmd6aG91LCBHdWFuZ2RvbmcsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTGlhbmcsIFh1YW5xaWFuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikd1YW5nZG9uZyBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcyAoR0FBUyksIEd1YW5nemhvdSwgR3Vhbmdkb25nLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhpcmFrYXdhLCBIaWRla2kiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2lzYXJhenUsIENoaWJhLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhYmF0YSwgU2F0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLaXNhcmF6dSwgQ2hpYmEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSXNvYmUsIFNhY2hpa28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLYXp1c2EgRE5BIFJlc2VhcmNoIEluc3RpdHV0ZSwgS2lzYXJhenUsIENoaWJhLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iSW50ZWdyYXRlZCBDb25zZW5zdXMgTWFwIG9mIEN1bHRpdmF0ZWQgUGVhbnV0IGFuZCBXaWxkIFJlbGF0aXZlcyBSZXZlYWxzIFN0cnVjdHVyZXMgb2YgdGhlIEEgYW5kIEIgR2Vub21lcyBvZiBBcmFjaGlzIGFuZCBEaXZlcmdlbmNlIG9mIHRoZSBMZWd1bWUgR2Vub21lcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE3MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTg0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNzMDQyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTMvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmVzdCBvZiB0aGUgaWNlYmVyZy4gTGVndW1lIGRpdmVyc2l0eSBhbmQgZXZvbHV0aW9uIGluIGEgcGh5bG9nZW5ldGljIGNvbnRleHQ7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzMTsgIGNpdGF0aW9uX3BhZ2VzPTkwMC0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBwaHlsb2dlbnkgb2YgbGVndW1lcyAoTGVndW1pbm9zYWUpIGJhc2VkIG9uIGFuYWx5c2lzIG9mIHRoZSBwbGFzdGlkIG1hdEsgZ2VuZSByZXNvbHZlcyBtYW55IHdlbGwtc3VwcG9ydGVkIHN1YmNsYWRlcyB3aXRoaW4gdGhlIGZhbWlseTsgQW0uIEouIEJvdC47ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9OTE7ICBjaXRhdGlvbl9wYWdlcz0xODQ2LTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fc3RpbWF0aW5nIGdlbm9tZSBjb25zZXJ2YXRpb24gYmV0d2VlbiBjcm9wIGFuZCBtb2RlbCBsZWd1bWUgc3BlY2llczsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDQ7ICBjaXRhdGlvbl92b2x1bWU9MTAxOyAgY2l0YXRpb25fcGFnZXM9MTUyODktOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIGdlbmV0aWMgbWFwcGluZyBhbmQgZGlzY292ZXJ5IG9mIGxpbmthZ2UgZGlzZXF1aWxpYnJpdW0gYWNyb3NzIGxpbmthZ2UgZ3JvdXBzIGluIHdoaXRlIGNsb3ZlciAoVHJpZm9saXVtIHJlcGVucyBMLik7IEczOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz02MDctMTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzdHJ1Y3R1cmUgb2YgdGhlIGxlZ3VtZSwgTG90dXMgamFwb25pY3VzOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTIyNy0zOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNlIG9mIHRoZSBwYWxhZW9wb2x5cGxvaWQgc295YmVhbjsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2MzsgIGNpdGF0aW9uX3BhZ2VzPTE3OC04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE1lZGljYWdvIGdlbm9tZSBwcm92aWRlcyBpbnNpZ2h0IGludG8gdGhlIGV2b2x1dGlvbiBvZiByaGl6b2JpYWwgc3ltYmlvc2VzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NDgwOyAgY2l0YXRpb25fcGFnZXM9NTIwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBzZXF1ZW5jZSBvZiBwaWdlb25wZWEgKENhamFudXMgY2FqYW4pLCBhbiBvcnBoYW4gbGVndW1lIGNyb3Agb2YgcmVzb3VyY2UtcG9vciBmYXJtZXJzOyBOYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9ODMtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QnJpZGdpbmcgbW9kZWwgYW5kIGNyb3AgbGVndW1lcyB0aHJvdWdoIGNvbXBhcmF0aXZlIGdlbm9taWNzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzc7ICBjaXRhdGlvbl9wYWdlcz0xMTg5LTk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UYXhvbm9taWEgZGVsIGdlbmVybyBBcmFjaGlzIChMZWd1bWlub3NhZSk7IEJvbnBsYW5kaWE7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTEtMTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBzcGVjaWVzIGluIHNlY3Rpb24gQXJhY2hpcyBvZiBwZWFudXRzIHdpdGggYSBEIGdlbm9tZTsgQW0uIEouIEJvdC47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9Nzg7ICBjaXRhdGlvbl9wYWdlcz02MzAtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BlY2llcyByZWxhdGlvbnNoaXBzIGFtb25nIHRoZSB3aWxkIEIgZ2Vub21lIG9mIEFyYWNoaXMgc3BlY2llcyAoc2VjdGlvbiBBcmFjaGlzKSBiYXNlZCBvbiBGSVNIIG1hcHBpbmcgb2YgckROQSBsb2NpIGFuZCBoZXRlcm9jaHJvbWF0aW4gZGV0ZWN0aW9uOiBhIG5ldyBwcm9wb3NhbCBmb3IgZ2Vub21lIGFycmFuZ2VtZW50OyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEyMTsgIGNpdGF0aW9uX3BhZ2VzPTEwMzMtNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwZWNpZXMgcmVsYXRpb25zIGFtb25nIHdpbGQgQXJhY2hpcyBzcGVjaWVzIHdpdGggdGhlIEEgZ2Vub21lIGFzIHJldmVhbGVkIGJ5IEZJU0ggbWFwcGluZyBvZiByRE5BIGxvY2kgYW5kIGhldGVyb2Nocm9tYXRpbiBkZXRlY3Rpb247IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTE4OyAgY2l0YXRpb25fcGFnZXM9MTI5NS0zMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9taWMgcmVsYXRpb25zaGlwcyBiZXR3ZWVuIHRoZSBjdWx0aXZhdGVkIHBlYW51dCAoQXJhY2hpcyBoeXBvZ2FlYSwgTGVndW1pbm9zYWUpIGFuZCBpdHMgY2xvc2UgcmVsYXRpdmVzIHJldmVhbGVkIGJ5IGRvdWJsZSBHSVNIOyBBbS4gSi4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT05NDsgIGNpdGF0aW9uX3BhZ2VzPTE5NjMtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeXNpY2FsIG1hcHBpbmcgb2YgdGhlIDVTIGFuZCAxOFMtMjVTIHJSTkEgZ2VuZXMgYnkgRklTSCBhcyBldmlkZW5jZSB0aGF0IEFyYWNoaXMgZHVyYW5lbnNpcyBhbmQgQS4gaXBhw6tuc2lzIGFyZSB0aGUgd2lsZCBkaXBsb2lkIHByb2dlbml0b3JzIG9mIEEuIGh5cG9nYWVhIChMZWd1bWlub3NhZSk7IEFtLiBKLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTkxOyAgY2l0YXRpb25fcGFnZXM9MTI5NC0zMDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzaXplIHZhcmlhdGlvbiBpbiBBcmFjaGlzIGh5cG9nYWVhIGFuZCBBLiBtb250aWNvbGEgcmUtZXZhbHVhdGVkOyBHZW5vbWU7ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NDM7ICBjaXRhdGlvbl9wYWdlcz00NDktNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGFuIFJGTFAgbGlua2FlZyBtYXAgaW4gZGlwbG9pZCBwZWFudXQgc3BlY2llczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT04NzsgIGNpdGF0aW9uX3BhZ2VzPTM3OS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgY2FuZGlkYXRlIGdlbm9tZSByZWdpb25zIGNvbnRyb2xsaW5nIGRpc2Vhc2UgcmVzaXN0YW5jZSBpbiBBcmFjaGlzOyBCTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTExMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbWljcm9zYXRlbGxpdGUtYmFzZWQsIGdlbmUtcmljaCBsaW5rYWdlIG1hcCBmb3IgdGhlIEFBIGdlbm9tZSBvZiBBcmFjaGlzIChGYWJhY2VhZSk7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTExOyAgY2l0YXRpb25fcGFnZXM9MTA2MC03MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBsaW5rYWdlIG1hcCBmb3IgdGhlIEItZ2Vub21lIG9mIEFyYWNoaXMgKEZhYmFjZWFlKSBhbmQgaXRzIHN5bnRlbnkgdG8gdGhlIEEtZ2Vub21lOyBCTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTQwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBoaWdoLWRlbnNpdHkgZ2VuZXRpYyBtYXAgb2YgQXJhY2hpcyBkdXJhbmVuc2lzLCBhIGRpcGxvaWQgYW5jZXN0b3Igb2YgY3VsdGl2YXRlZCBwZWFudXQ7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMzsgIGNpdGF0aW9uX3BhZ2VzPTQ2OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIG1hcHBpbmcgaW4gaW50cmFzcGVjaWZpYyBwb3B1bGF0aW9ucyB1bmNvdmVycyBhIGhpZ2ggZGVncmVlIG9mIG1hY3Jvc3ludGVueSBiZXR3ZWVuIEEtIGFuZCBCLWdlbm9tZSBkaXBsb2lkIHNwZWNpZXMgb2YgcGVhbnV0OyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz02MDgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc21pc3Npb24gZ2VuZXRpY3Mgb2YgY2hyb21hdGluIGZyb20gYSBzeW50aGV0aWMgYW1waGlkaXBsb2lkIHRvIGN1bHRpdmF0ZWQgcGVhbnV0IChBcmFjaGlzIGh5cG9nYWVhIEwuKTogYnJvYWRlbmluZyB0aGUgZ2VuZSBwb29sIG9mIGEgbW9ub3BoeWxldGljIHBvbHlwbG9pZCBzcGVjaWVzOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTk7ICBjaXRhdGlvbl9wYWdlcz04MjMtMzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbWFwcGluZyBvZiB3aWxkIGludHJvZ3Jlc3Npb25zIGludG8gY3VsdGl2YXRlZCBwZWFudXQ6IGEgd2F5IHRvd2FyZCBlbmxhcmdpbmcgdGhlIGdlbmV0aWMgYmFzaXMgb2YgYSByZWNlbnQgYWxsb3RldHJhcGxvaWQ7IEJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTAzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIHRyYWl0IGxvY3VzIGFuYWx5c2lzIGFuZCBjb25zdHJ1Y3Rpb24gb2YgY29uc2Vuc3VzIGdlbmV0aWMgbWFwIGZvciBkcm91Z2h0IHRvbGVyYW5jZSB0cmFpdHMgYmFzZWQgb24gdGhyZWUgcmVjb21iaW5hbnQgaW5icmVkIGxpbmUgcG9wdWxhdGlvbnMgaW4gY3VsdGl2YXRlZCBncm91bmRudXQgKEFyYWNoaXMgaHlwb2dhZWEgTC4pOyBNb2wuIEJyZWVkLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTc3My04ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBTU1ItYmFzZWQgY29tcG9zaXRlIGdlbmV0aWMgbGlua2FnZSBtYXAgZm9yIHRoZSBjdWx0aXZhdGVkIHBlYW51dCAoQXJhY2hpcyBoeXBvZ2FlYSBMLikgZ2Vub21lOyBCTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz0xNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgUVRMIHN0dWR5IG9uIGxhdGUgbGVhZiBzcG90IGFuZCBydXN0IHJldmVhbGVkIG9uZSBtYWpvciBRVEwgZm9yIG1vbGVjdWxhciBicmVlZGluZyBmb3IgcnVzdCByZXNpc3RhbmNlIGluIGdyb3VuZG51dCAoQXJhY2hpcyBoeXBvZ2FlYSBMLik7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTIxOyAgY2l0YXRpb25fcGFnZXM9OTcxLTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlZ3JhdGVkIGdlbmV0aWMgbGlua2FnZSBtYXAgb2YgY3VsdGl2YXRlZCBwZWFudXQgKEFyYWNoaXMgaHlwb2dhZWEgTC4pIGNvbnN0cnVjdGVkIGZyb20gdHdvIFJJTCBwb3B1bGF0aW9uczsgVGhlb3IuIEFwcGwuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjQ7ICBjaXRhdGlvbl9wYWdlcz02NTMtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIHNldmVyYWwgc21hbGwgbWFpbi1lZmZlY3QgUVRMcyBhbmQgYSBsYXJnZSBudW1iZXIgb2YgZXBpc3RhdGljIFFUTHMgZm9yIGRyb3VnaHQgdG9sZXJhbmNlIHJlbGF0ZWQgdHJhaXRzIGluIGdyb3VuZG51dCAoQXJhY2hpcyBoeXBvZ2FlYSBMLik7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTIyOyAgY2l0YXRpb25fcGFnZXM9MTExOS0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgcXVhbnRpdGF0aXZlIHRyYWl0IGxvY2kgZm9yIHByb3RlaW4gY29udGVudCwgb2lsIGNvbnRlbnQgYW5kIG9pbCBxdWFsaXR5IGZvciBncm91bmRudXQgKEFyYWNoaXMgaHlwb2dhZWEgTC4pOyBGaWVsZCBDcm9wcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyMjsgIGNpdGF0aW9uX3BhZ2VzPTQ5LTU5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFudGl0YXRpdmUgdHJhaXQgbG9jdXMgYW5hbHlzaXMgYW5kIGNvbnN0cnVjdGlvbiBvZiBjb25zZW5zdXMgZ2VuZXRpYyBtYXAgZm9yIGZvbGlhciBkaXNlYXNlIHJlc2lzdGFuY2UgYmFzZWQgb24gdHdvIHJlY29tYmluYW50IGluYnJlZCBsaW5lIHBvcHVsYXRpb25zIGluIGN1bHRpdmF0ZWQgZ3JvdW5kbnV0IChBcmFjaGlzIGh5cG9nYWVhIEwuKTsgTW9sLiBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz03NzMtODg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBmaXJzdCBTU1ItYmFzZWQgZ2VuZXRpYyBsaW5rYWdlIG1hcCBmb3IgY3VsdGl2YXRlZCBncm91bmRudXQgKEFyYWNoaXMgaHlwb2dhZWEgTC4pOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTExODsgIGNpdGF0aW9uX3BhZ2VzPTcyOS0zOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV2ZWxvcG1lbnQgYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgQkFDLWVuZCBzZXF1ZW5jZSBkZXJpdmVkIFNTUnMsIGFuZCB0aGVpciBpbmNvcnBvcmF0aW9uIGludG8gYSBuZXcgaGlnaGVyIGRlbnNpdHkgZ2VuZXRpYyBtYXAgZm9yIGN1bHRpdmF0ZWQgcGVhbnV0IChBcmFjaGlzIGh5cG9nYWVhIEwuKTsgQk1DIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZHZhbmNlcyBpbiBBcmFjaGlzIGdlbm9taWNzIGZvciBwZWFudXQgaW1wcm92ZW1lbnQ7IEJpb3RlY2hub2wuIEFkdi47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz02MzktNTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGludGVybmF0aW9uYWwgcmVmZXJlbmNlIGNvbnNlbnN1cyBnZW5ldGljIG1hcCB3aXRoIDg5NyBtYXJrZXIgbG9jaSBiYXNlZCBvbiAxMSBtYXBwaW5nIHBvcHVsYXRpb25zIGZvciB0ZXRyYXBsb2lkIGdyb3VuZG51dCAoQXJhY2hpcyBoeXBvZ2FlYSBMLik7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lNDEyMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbiBzaWxpY28gcG9seW1vcnBoaXNtIGFuYWx5c2lzIGZvciB0aGUgZGV2ZWxvcG1lbnQgb2Ygc2ltcGxlIHNlcXVlbmNlIHJlcGVhdCBhbmQgdHJhbnNwb3NvbiBtYXJrZXJzIGFuZCBjb25zdHJ1Y3Rpb24gb2YgbGlua2FnZSBtYXAgaW4gY3VsdGl2YXRlZCBwZWFudXQ7IEJNQyBQbGFudCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTgwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFyZ2Utc2NhbGUgZGV2ZWxvcG1lbnQgb2YgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZy1kZXJpdmVkIHNpbXBsZSBzZXF1ZW5jZSByZXBlYXQgbWFya2VycyBhbmQgZGl2ZXJzaXR5IGFuYWx5c2lzIGluIEFyYWNoaXMgc3BwLjsgTW9sLiBCcmVlZC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz0xMjUtMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgYWN0aXZlIG1pbmlhdHVyZSBpbnZlcnRlZC1yZXBlYXQgdHJhbnNwb3NhYmxlIGVsZW1lbnRzIGluIHRoZSBwZWFudXQgZ2Vub21lOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTEyNDsgIGNpdGF0aW9uX3BhZ2VzPTE0MjktMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGFuYWx5c2lzIG9mIHN5bnRlbnkgb2YgQXJhY2hpcyB3aXRoIExvdHVzIGFuZCBNZWRpY2FnbyBzaGVkcyBuZXcgbGlnaHQgb24gdGhlIHN0cnVjdHVyZSwgc3RhYmlsaXR5IGFuZCBldm9sdXRpb24gb2YgbGVndW1lIGdlbm9tZXM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTQ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXRpYyBsaW5rYWdlIG1hcHMgb2YgRXVjYWx5cHR1cyBncmFuZGlzIGFuZCBFdWNhbHlwdHVzIHVyb3BoeWxsYSB1c2luZyBhIHBzZXVkby10ZXN0Y3Jvc3M6IG1hcHBpbmcgc3RyYXRlZ3kgYW5kIFJBUEQgbWFya2VyczsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTM3OyAgY2l0YXRpb25fcGFnZXM9MTEyMS0zNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fcHVibGlzaGVyPUt5YXptYS4gQlYsIFdhZ2VuaW5nZW4sIFRoZSBOZXRoZXJsYW5kczsgSm9pbk1hcMKuNCwgU29mdHdhcmUgZm9yIHRoZSBjYWxjdWxhdGlvbiBvZiBnZW5ldGljIGxpbmthZ2UgbWFwcyBpbiBleHBlcmltZW50YWwgcG9wdWxhdGlvbnM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcENoYXJ0OiBzb2Z0d2FyZSBmb3IgdGhlIGdyYXBoaWNhbCBwcmVzZW50YXRpb24gb2YgbGlua2FnZSBtYXBzIGFuZCBRVExzOyBKLiBIZXJlZC47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz03Ny04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM4OS00MDI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNpcmNvczogYW4gaW5mb3JtYXRpb24gYWVzdGhldGljIGZvciBjb21wYXJhdGl2ZSBnZW5vbWljczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTk7ICBjaXRhdGlvbl9wYWdlcz0xNjM5LTQ1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GSURFTC1hIHJldHJvdmlydXMtbGlrZSByZXRyb3RyYW5zcG9zb24gYW5kIGl0cyBkaXN0aW5jdCBldm9sdXRpb25hcnkgaGlzdG9yaWVzIGluIHRoZSBBLSBhbmQgQi1nZW5vbWUgY29tcG9uZW50cyBvZiBjdWx0aXZhdGVkIHBlYW51dDsgQ2hyb21vc29tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MjI3LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXRpdGEsIGEgbmV3IHJldHJvZWxlbWVudCBmcm9tIHBlYW51dDogY2hhcmFjdGVyaXphdGlvbiBhbmQgZXZvbHV0aW9uYXJ5IGNvbnRleHQgaW4gdGhlIGxpZ2h0IG9mIHRoZSBBcmFjaGlzIEEtQiBnZW5vbWUgZGl2ZXJnZW5jZTsgTW9sLiBHZW5ldC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9Mjg3OyAgY2l0YXRpb25fcGFnZXM9MjEtMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1BUE1BS0VSOiBhbiBpbnRlcmFjdGl2ZSBjb21wdXRlciBwYWNrYWdlIGZvciBjb25zdHJ1Y3RpbmcgcHJpbWFyeSBnZW5ldGljIGxpbmthZ2UgbWFwcyBvZiBleHBlcmltZW50YWwgYW5kIG5hdHVyYWwgcG9wdWxhdGlvbnM7IEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xNzQtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGEgbGFyZ2UgU05QIGdlbm90eXBpbmcgYXJyYXkgYW5kIGdlbmVyYXRpb24gb2YgaGlnaC1kZW5zaXR5IGdlbmV0aWMgbWFwcyBpbiB0b21hdG87IFBMb1MgT05FOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lNDA1NjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlcnNwZWNpZmljIGxpbmthZ2UgbWFwIG9mIFNTUiBhbmQgaW50cm9uaWMgcG9seW1vcnBoaXNtIG1hcmtlcnMgaW4gdG9tYXRvOyBUaGVvci4gQXBwbC4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTEyMTsgIGNpdGF0aW9uX3BhZ2VzPTczMS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TTlAgZGlzY292ZXJ5IGFuZCBsaW5rYWdlIG1hcCBjb25zdHJ1Y3Rpb24gaW4gY3VsdGl2YXRlZCB0b21hdG87IEROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MzgxLTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbW9zb21hbCBhbmQgZ2VuaWMgYmFycmllcnMgdG8gaW50cm9ncmVzc2lvbiBpbiBIZWxpYW50aHVzOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDE7ICBjaXRhdGlvbl9wYWdlcz0xMTYzLTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Fdm9sdXRpb25hcnkgcmF0ZXMgYW5hbHlzaXMgb2YgTGVndW1pbm9zYWUgaW1wbGljYXRlcyBhIHJhcGlkIGRpdmVyc2lmaWNhdGlvbiBvZiBsaW5lYWdlcyBkdXJpbmcgdGhlIHRlcnRpYXJ5OyBTeXN0LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT01NDsgIGNpdGF0aW9uX3BhZ2VzPTU3NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMC8yLzE3My85NTQ5NzQvZHNzMDQyLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgY29tcGxleCwgdGV0cmFwbG9pZCBnZW5vbWUgc3RydWN0dXJlIG9mIHBlYW51dCAoQXJhY2hpcyBoeXBvZ2FlYSkgaGFzIG9ic3RydWN0ZWQgYWR2YW5jZXMgaW4gZ2VuZXRpY3MgYW5kIGdlbm9taWNzIGluIHRoZSBzcGVjaWVzLiBUaGUgYWltIG9mIHRoaSIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9MzMzNzgxJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZPt2a0RzqANEdhiKxT2L49HsXcBe" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f173%2f333781" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f173%2f333781" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=12698" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/2/1/m_cover.gif?Expires=2147483647&amp;Signature=Yq7dbhq6hb-Iyo5ewEGt-Y1T99brR4Xh8FvVnCI2Bg1SIlPGztlGCk-Bm0A9A72SaqDKcetVlEQehNtxyAGT~T55ZFFD19A3hKk~A1XbnQ-JfXYKj2-gYHz-VirOlvLe-8XkC6qvT4h2PniinfIJH6d85JsIf3rhRRtN086VVGvi2qNVR0THPJOeMpQvElVQEEN-4bpo3wEYmq80uwX58y5ScJ0pcJt7sV40xw1Fd6mp92v-su9vXuGT~IIVjyHRDyuiGt~ZiUzcXtor6WX4MT1zcCCf5ySC3Wm7DfaZruIvVsbtJxHUYCwePrgmcTq1vhdeOWHam2svM-jBZyoDoA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/2">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="4624705">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4624705">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4624707">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4624707">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4624715">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4624715">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4624724">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4624724">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4624748">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4624748">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4624763">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4624763">5. Conclusion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4624767">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4624767">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4624769">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4624769">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4624771">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4624771">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="4624773">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#4624773">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/2/163/333645">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/2/185/333880">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Integrated Consensus Map of Cultivated Peanut and Wild Relatives Reveals Structures of the A and B Genomes of <em>Arachis</em> and Divergence of the Legume Genomes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kenta Shirasawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 438-52-3935. Fax. +81 438-52-3934. E-mail: <a href="mailto:shirasaw@kazusa.or.jp" target="_blank">shirasaw@kazusa.or.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kenta+Shirasawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shirasawa K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shirasawa K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Kenta Shirasawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">David J. Bertioli</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">University of Brasília</div>, <div class="addr-line">Brasília, DF</div>, <div class="country">Brazil</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=David+J.+Bertioli">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Bertioli D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Bertioli D&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">David J. Bertioli</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rajeev K. Varshney</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru, AP</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rajeev+K.+Varshney">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Varshney R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Varshney R&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Rajeev K. Varshney</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Marcio C. Moretzsohn</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Embrapa Genetic Resources and Biotechnology</div>, <div class="addr-line">Brasília, DF</div>, <div class="country">Brazil</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Marcio+C.+Moretzsohn">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Moretzsohn M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Moretzsohn M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Marcio C. Moretzsohn</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Soraya C. M. Leal-Bertioli</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="insititution">Embrapa Genetic Resources and Biotechnology</div>, <div class="addr-line">Brasília, DF</div>, <div class="country">Brazil</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Soraya+C.+M.+Leal-Bertioli">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Leal-Bertioli S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Leal-Bertioli S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Soraya C. M. Leal-Bertioli</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mahendar Thudi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru, AP</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mahendar+Thudi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Thudi M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Thudi M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Mahendar Thudi</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Manish K. Pandey</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">International Crops Research Institute for the Semi-Arid Tropics (ICRISAT)</div>, <div class="addr-line">Patancheru, AP</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Manish+K.+Pandey">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pandey M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pandey M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Manish K. Pandey</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jean-Francois Rami</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Centre de Coopération Internationale en Recherche Agronomique pour le Développement (CIRAD), UMR AGAP</div>, <div class="addr-line">F-34398 Montpellier</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jean-Francois+Rami">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Rami J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Rami J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Jean-Francois Rami</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Daniel Fonc&#233;ka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="insititution">Centre de Coopération Internationale en Recherche Agronomique pour le Développement (CIRAD), UMR AGAP</div>, <div class="addr-line">F-34398 Montpellier</div>, <div class="country">France</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Daniel+Fonc%c3%a9ka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fonc&#233;ka D">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fonc&#233;ka D&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Daniel Foncéka</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Makanahally V. C. Gowda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">University of Agricultural Sciences (UAS-D)</div>, <div class="addr-line">Dharwad, KA</div>, <div class="country">India</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Makanahally+V.+C.+Gowda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gowda M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gowda M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Makanahally V. C. Gowda</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hongde Qin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">7</div><div class="insititution">University of Georgia</div>, <div class="addr-line">Tifton, GA</div>, <div class="country">USA</div></div><div class="aff"><div class="label -label">8</div><div class="insititution">Hubei Academy of Agricultural Sciences (HAAS)</div>, <div class="addr-line">Wuhan, Hubei</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hongde+Qin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Qin H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Qin H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hongde Qin</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Baozhu Guo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">9</div><div class="insititution">USDA-Agricultural Research Service</div>, <div class="addr-line">Tifton, GA</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Baozhu+Guo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Guo B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Guo B&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Baozhu Guo</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yanbin Hong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">10</div><div class="insititution">Guangdong Academy of Agricultural Sciences (GAAS)</div>, <div class="addr-line">Guangzhou, Guangdong</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yanbin+Hong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hong Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hong Y&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Yanbin Hong</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xuanqiang Liang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">10</div><div class="insititution">Guangdong Academy of Agricultural Sciences (GAAS)</div>, <div class="addr-line">Guangzhou, Guangdong</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xuanqiang+Liang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Liang X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Liang X&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Xuanqiang Liang</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sachiko Isobe</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="insititution">Kazusa DNA Research Institute</div>, <div class="addr-line">Kisarazu, Chiba</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sachiko+Isobe">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Isobe S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Isobe S&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Sachiko Isobe</a>
                                        </span>
                                </div>
                            </div>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 2, 1 April 2013, Pages 173–184, <a href='https://doi.org/10.1093/dnares/dss042'>https://doi.org/10.1093/dnares/dss042</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">12 January 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 October 2012</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">21 December 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="333781" href="/dnaresearch/article-pdf/20/2/173/954974/dss042.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Kenta Shirasawa, David J. Bertioli, Rajeev K. Varshney, Marcio C. Moretzsohn, Soraya C. M. Leal-Bertioli, Mahendar Thudi, Manish K. Pandey, Jean-Francois Rami, Daniel Foncéka, Makanahally V. C. Gowda, Hongde Qin, Baozhu Guo, Yanbin Hong, Xuanqiang Liang, Hideki Hirakawa, Satoshi Tabata, Sachiko Isobe;  Integrated Consensus Map of Cultivated Peanut and Wild Relatives Reveals Structures of the A and B Genomes of <em>Arachis</em> and Divergence of the Legume Genomes, <em>DNA Research</em>, Volume 20, Issue 2, 1 April 2013, Pages 173–184, <a href='https://doi.org/10.1093/dnares/dss042'>https://doi.org/10.1093/dnares/dss042</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/333781?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/333781?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/333781?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/333781?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/333781?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/333781?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/333781?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Integrated%20Consensus%20Map%20of%20Cultivated%20Peanut%20and%20Wild%20Relatives%20Reveals%20Structures%20of%20the%20A%20and%20B%20Genomes%20of%20%26lt%3Bem%26gt%3BArachis%26lt%3B%2Fem%26gt%3B%20and%20Divergence%20of%20the%20Legume%20Genomes&amp;publicationDate=2013-01-12&amp;volumeNum=20&amp;issueNum=2&amp;author=Shirasawa%2C%20Kenta%3B%20Bertioli%2C%20David%20J.&amp;startPage=173&amp;endPage=184&amp;contentId=10.1093%2Fdnares%2Fdss042&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f173%2f333781" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=4624705 id="4624705" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The complex, tetraploid genome structure of peanut (<em>Arachis hypogaea</em>) has obstructed advances in genetics and genomics in the species. The aim of this study is to understand the genome structure of <em>Arachis</em> by developing a high-density integrated consensus map. Three recombinant inbred line populations derived from crosses between the A genome diploid species, <em>Arachis duranensis</em> and <em>Arachis stenosperma</em>; the B genome diploid species, <em>Arachis ipaënsis</em> and <em>Arachis magna</em>; and between the AB genome tetraploids, <em>A. hypogaea</em> and an artificial amphidiploid (<em>A. ipaënsis</em> × <em>A. duranensis</em>)<sup>4×</sup>, were used to construct genetic linkage maps: 10 linkage groups (LGs) of 544 cM with 597 loci for the A genome; 10 LGs of 461 cM with 798 loci for the B genome; and 20 LGs of 1442 cM with 1469 loci for the AB genome. The resultant maps plus 13 published maps were integrated into a consensus map covering 2651 cM with 3693 marker loci which was anchored to 20 consensus LGs corresponding to the A and B genomes. The comparative genomics with genome sequences of <em>Cajanus cajan</em>, <em>Glycine max</em>, <em>Lotus japonicus</em>, and <em>Medicago truncatula</em> revealed that the <em>Arachis</em> genome has segmented synteny relationship to the other legumes. The comparative maps in legumes, integrated tetraploid consensus maps, and genome-specific diploid maps will increase the genetic and genomic understanding of <em>Arachis</em> and should facilitate molecular breeding.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Arachis spp.&quot;"><em>Arachis</em> spp.</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;comparative genomics&quot;">comparative genomics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genetic linkage map&quot;">genetic linkage map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;integrated consensus map&quot;">integrated consensus map</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;legume genome&quot;">legume genome</a></div>                            <h2 scrollto-destination=4624707 id="4624707" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Fabaceae, or Leguminosae, is composed of ∼700 genera and 20 000 species.<sup><a reveal-id="DSS042C1" data-open="DSS042C1" class="link link-ref link-reveal xref-bibr">1</a></sup> The family is classified into three major subfamilies Mimosoideae, Caesalpinioideae, and Papilionoideae.<sup><a reveal-id="DSS042C2" data-open="DSS042C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Papilionoideae is divided into four clades Genistoids, Dalbergioids, Phaseoloids, and Galegoids. Most tropical and cool season legumes used as crops and studied as model plants for symbiosis of rhizobium are members of the Phaseoloids, e.g. genera <em>Cajanus</em>, <em>Glycine</em>, <em>Phaseolus</em>, and <em>Vigna</em>, or the Galegoids, e.g. genera <em>Lotus</em>, <em>Medicago</em>, <em>Pisum</em>, <em>Trifolium</em>, and <em>Vicia</em>.<sup><a reveal-id="DSS042C2" data-open="DSS042C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Therefore, legume genetics and genomics, including genome sequencing and comparative genomics, have greatly advanced in the Phaseoloids and Galegoids.<sup><a reveal-id="DSS042C3  DSS042C9" data-open="DSS042C3  DSS042C9" class="link link-ref link-reveal xref-bibr">3–9</a></sup></p><p>The Dalbergioids, which include the genus, <em>Arachis</em>, are more basal in their divergence within the Papilionoideae than the Phaseoloids and Galegoids. The genus <em>Arachis</em> comprises 80 species (2<em>n</em> = 2<em>x</em> = 18, 2<em>n</em> = 2<em>x</em> = 20, and 2<em>n</em> = 4<em>x</em> = 40) possessing A, B, C, D, E, F, K, R, and T genomes.<sup><a reveal-id="DSS042C10  DSS042C12" data-open="DSS042C10  DSS042C12" class="link link-ref link-reveal xref-bibr">10–12</a></sup> As regarding the A genome diploids consisting of 15 wild species, they are morphologically similar, cross compatible, produce fertile hybrids with near normal chromosome pairing, and have a common karyotype structure.<sup><a reveal-id="DSS042C13" data-open="DSS042C13" class="link link-ref link-reveal xref-bibr">13</a></sup> The B-genome diploids also have similar morphological and chromosome feature, cross-compatibility, bivalent formation at meiosis in interspecific hybrids, and the same karyotype structure.<sup><a reveal-id="DSS042C12" data-open="DSS042C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Peanut (<em>Arachis hypogaea</em>), or groundnut, is an autogamous allotetraploid legume (2<em>n</em> = 4<em>x</em> = 40) harbouring homoeologous A and B genomes that are derived from two diploids, most likely <em>Arachis duranensis</em> (A genome) and <em>Arachis ipaënsis</em> (B genome).<sup><a reveal-id="DSS042C14" data-open="DSS042C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSS042C15" data-open="DSS042C15" class="link link-ref link-reveal xref-bibr">15</a></sup> While peanut is an important source of food and oil, the genetics and genomics of this legume lag far behind those of the Phaseoloids and Galegoids due to a complex genome structure with challenging features, such as tetraploidy, low genetic diversity, and a relatively large genome size of 2.8 Gb.<sup><a reveal-id="DSS042C16" data-open="DSS042C16" class="link link-ref link-reveal xref-bibr">16</a></sup></p><p>Molecular genetic studies of <em>Arachis</em> initially progressed by using diploid rather than tetraploid peanut due to the greater simplicity of diploids as genetic models.<sup><a reveal-id="DSS042C17" data-open="DSS042C17" class="link link-ref link-reveal xref-bibr">17</a></sup> The first genetic linkage map in genus <em>Arachis</em> was developed based on an interspecific crossing between diploids <em>Arachis stenosperma</em> (A genome) and <em>Arachis cardenasii</em> (also A genome),<sup><a reveal-id="DSS042C17" data-open="DSS042C17" class="link link-ref link-reveal xref-bibr">17</a></sup> followed by a cross between <em>A. duranensis</em> and <em>A. stenosperma</em> (both A genomes)<sup><a reveal-id="DSS042C18" data-open="DSS042C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSS042C19" data-open="DSS042C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and between <em>A. ipaënsis</em> and <em>Arachis magna</em> (both B genomes).<sup><a reveal-id="DSS042C20" data-open="DSS042C20" class="link link-ref link-reveal xref-bibr">20</a></sup> However, because of the small number of available DNA markers at the time, the number of mapped marker loci was limited to between 117 and 369. Recently, intraspecific maps for A-genome diploid (<em>A. duranensis</em>) and B-genome diploid (<em>Arachi batizocoi</em>) have been developed with 1724 and 449 marker loci, respectively, including single nucleotide polymorphisms and expressed sequence tag-simple sequence repeats (EST-SSRs) markers.<sup><a reveal-id="DSS042C21" data-open="DSS042C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSS042C22" data-open="DSS042C22" class="link link-ref link-reveal xref-bibr">22</a></sup></p><p>Meanwhile, an artificial amphidiploid derived from a three-way cross between <em>A. batizocoi</em> (B genome) and a hybrid between <em>A. cardenasii</em> (A genome) and <em>A. diogoi</em> (A genome) has been crossed with <em>A. hypogaea</em> to introduce genetic diversity from wild diploid species into the tetraploid-cultivated peanut. The BC<sub>1</sub> progenies of the cross have contributed to the generation of a tetraploid genetic linkage map with 370 restriction-fragment length polymorphisms.<sup><a reveal-id="DSS042C23" data-open="DSS042C23" class="link link-ref link-reveal xref-bibr">23</a></sup> In addition, another artificial amphidiploid line, derived from a hybrid between <em>A. duranensis</em> and <em>A. ipaënsis</em>, was used to establish a genetic linkage map made up of 298 SSR marker loci.<sup><a reveal-id="DSS042C24" data-open="DSS042C24" class="link link-ref link-reveal xref-bibr">24</a></sup> In the latter map, the linkage groups (LGs) were firstly anchored to the A and B genomes by identifying the genome origins of the mapped loci in the LGs based on the corresponding sizes of the DNA amplified from the original diploids. In the tetraploid-cultivated peanut (<em>A. hypogaea</em>), the population-specific linkage maps comprise 318 marker loci. Due to low genetic diversity,<sup><a reveal-id="DSS042C25  DSS042C33" data-open="DSS042C25  DSS042C33" class="link link-ref link-reveal xref-bibr">25–33</a></sup> this is the maximum number of marker loci for population-specific linkage maps, although &gt;6000 SSR markers have been developed for mapping.<sup><a reveal-id="DSS042C34" data-open="DSS042C34" class="link link-ref link-reveal xref-bibr">34</a></sup> By integrating the 11 linkage maps of the artificial amphidiploids and cultivated tetraploid peanuts, a reference consensus map consisting of 897 marker loci was constructed and annotated with consensus nomenclature for the LGs (a01–a10 and b01–b10).<sup><a reveal-id="DSS042C35" data-open="DSS042C35" class="link link-ref link-reveal xref-bibr">35</a></sup></p><p>Concurrent with the progress of the above genetic studies in <em>Arachis</em> spp., a high-density linkage map of cultivated peanut covering 2166 cM with 1114 marker loci has also been reported.<sup><a reveal-id="DSS042C36" data-open="DSS042C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Because most of the markers on this map were published after the development of the other <em>Arachis</em> maps,<sup><a reveal-id="DSS042C36  DSS042C38" data-open="DSS042C36  DSS042C38" class="link link-ref link-reveal xref-bibr">36–38</a></sup> the markers do not appear on them.<sup><a reveal-id="DSS042C17  DSS042C20" data-open="DSS042C17  DSS042C20" class="link link-ref link-reveal xref-bibr">17–20</a>,<a reveal-id="DSS042C23  DSS042C32" data-open="DSS042C23  DSS042C32" class="link link-ref link-reveal xref-bibr">23–32</a></sup> In addition, due to an insufficient number of commonly mapped loci in the published diploid maps, the LGs have not been assigned to either A or B genome types, even though homoeologous LGs have been identified.<sup><a reveal-id="DSS042C36" data-open="DSS042C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Anchoring the genetic LGs to the A and B genomes will enable enrichment of marker density on the consensus and diploid maps.<sup><a reveal-id="DSS042C19" data-open="DSS042C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSS042C20" data-open="DSS042C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSS042C35" data-open="DSS042C35" class="link link-ref link-reveal xref-bibr">35</a></sup> This would contribute to a better understanding of genome evolution within <em>Arachis</em> and among the Fabaceae. In addition, it would serve to increase the efficiency of introgression of desirable alleles from diploid wild species into the tetraploid-cultivated peanut through marker-assisted selection.</p><p>Comparative analysis of the A genome diploid <em>Arachis</em> species, e.g. <em>A. duranensis</em> and <em>A. stenosperma</em>, and model legumes has previously been performed. This comparative analysis revealed candidate synteny blocks between the genomes of <em>Lotus japonicus</em>, <em>Medicago truncatula</em>, and <em>Glycine max</em>.<sup><a reveal-id="DSS042C21" data-open="DSS042C21" class="link link-ref link-reveal xref-bibr">21</a>,<a reveal-id="DSS042C39" data-open="DSS042C39" class="link link-ref link-reveal xref-bibr">39</a></sup> To date, no comparative analysis of cultivated peanut and other legume species has been reported. Comparison of the tetraploid <em>Arachis</em> genome, represented by the high-density consensus map of the genomes of legumes <em>Cajanus cajan</em>, <em>G. max</em>, <em>L. japonicus</em>, and <em>M. truncatula</em>, would provide further insights into the legume genomes.</p><p>To address the above issues, it was considered that high-density linkage maps of the tetraploid and diploid species, on which common makers were mapped, would be required. Therefore, we tried to map the large number of DNA markers reported for the two cultivated peanut maps published by Shirasawa <em>et al.</em><sup><a reveal-id="DSS042C36" data-open="DSS042C36" class="link link-ref link-reveal xref-bibr">36</a></sup> onto the previously reported linkage maps of the A and B genomes,<sup><a reveal-id="DSS042C19" data-open="DSS042C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSS042C20" data-open="DSS042C20" class="link link-ref link-reveal xref-bibr">20</a></sup> along with that of an artificial amphidiploid derived from a hybrid of diploid species. The established linkage maps were integrated with 13 reported tetraploid <em>Arachis</em> maps to increase the number of mapped loci on the previously published consensus map.<sup><a reveal-id="DSS042C35" data-open="DSS042C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Subsequently, the maps were subjected to comparative analysis with four legume genomes, <em>C. cajan</em>, <em>G. max</em>, <em>L. japonicus</em>, and <em>M. truncatula</em>, to clarify features of the genome structure of the genus <em>Arachis</em>.</p>                            <h2 scrollto-destination=4624715 id="4624715" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=4624716 id="4624716" class="section-title" data-legacy-id=s2a>2.1. Plant materials</h3>
<p>In previous studies, two F<sub>2</sub> mapping populations, derived from interspecific crosses between the two A genome diploid species, <em>A. duranensis</em> ‘K7988’ and <em>A. stenosperma</em> ‘V10309’, and between the two B genome diploid species, <em>A. ipaënsis</em> ‘K30076’ and <em>A. magna</em> ‘K30097’, were used to construct genetic linkage maps.<sup><a reveal-id="DSS042C19" data-open="DSS042C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSS042C20" data-open="DSS042C20" class="link link-ref link-reveal xref-bibr">20</a></sup> In the present study, the F<sub>5</sub> and F<sub>6</sub> progenies of the diploid A and B genome mapping populations, respectively, were generated by single seed descent, and used as recombinant inbred mapping populations. The A and B genome mapping populations were named as AF5 (<em>n</em> = 89) and BF6 (<em>n</em> = 94), respectively. In addition, other recombinant inbred lines were used to construct AB genome tetraploid (<em>n</em> = 91: population TF6) maps. This population consisted of F<sub>6</sub> lines derived from a cross between <em>A. hypogaea</em> ‘Runner IAC 886’ and an artificial amphidiploid (<em>A. ipaënsis</em> × <em>A. duranensis</em>)<sup>4×</sup>, which was developed by hybridizing <em>A. ipaënsis</em> ‘K30076’ and <em>A. duranensis</em> ‘V14167’. Genomic DNA from each line was extracted from young leaflets essentially as described by Grattapaglia and Sederoff.<sup><a reveal-id="DSS042C40" data-open="DSS042C40" class="link link-ref link-reveal xref-bibr">40</a></sup></p>                            <h3 scrollto-destination=4624718 id="4624718" class="section-title" data-legacy-id=s2b>2.2. Polymorphism analysis with DNA markers</h3>
<p>A total of 3902 DNA markers comprising 1894 genomic SSRs <em>A. hypogaea</em> genomic SSR (AHGS),<sup><a reveal-id="DSS042C36" data-open="DSS042C36" class="link link-ref link-reveal xref-bibr">36</a></sup> 1571 EST-SSRs <em>A. hypogaea</em> EST-SSR (AHS),<sup><a reveal-id="DSS042C37" data-open="DSS042C37" class="link link-ref link-reveal xref-bibr">37</a></sup> and 437 transposon-based markers <em>A. hypogaea</em> transposable element (AhTE)<sup><a reveal-id="DSS042C36" data-open="DSS042C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSS042C38" data-open="DSS042C38" class="link link-ref link-reveal xref-bibr">38</a></sup> were screened against the parental lines of the populations AF5, BF6, and TF6 using an ABI-3730<em>xl</em> fluorescent fragment analyser (Applied Biosystems). Polymorphic markers were analysed on the three mapping populations using 10% polyacrylamide gels for markers exhibiting ≥10-bp allele size differences, or the fluorescent fragment analyser for markers exhibiting &lt;10-bp allele size differences between the parental lines. PCR, electrophoresis, and data scoring were performed as described previously.<sup><a reveal-id="DSS042C36  DSS042C38" data-open="DSS042C36  DSS042C38" class="link link-ref link-reveal xref-bibr">36–38</a></sup></p>                            <h3 scrollto-destination=4624720 id="4624720" class="section-title" data-legacy-id=s2c>2.3. Construction of linkage maps and the integrated map</h3>
<p>Linkage analysis of segregated genotypic data obtained in this study was performed together with linkage analysis of the previously investigated genotypes with mapped markers on the diploid maps<sup><a reveal-id="DSS042C19" data-open="DSS042C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSS042C20" data-open="DSS042C20" class="link link-ref link-reveal xref-bibr">20</a></sup> using JoinMap<sup>®</sup> version 4.<sup><a reveal-id="DSS042C41" data-open="DSS042C41" class="link link-ref link-reveal xref-bibr">41</a></sup> The marker loci were roughly classified using the JoinMap<sup>®</sup> Grouping Module with logarithm of odds (LOD) scores of 4.0–10.0. The JoinMap<sup>®</sup> Combine Groups for Map Integration Module was used to integrate the linkage maps developed in this study and the 13 previously published maps into a tetraploid map, i.e. the cultivated peanut map<sup><a reveal-id="DSS042C25  DSS042C32" data-open="DSS042C25  DSS042C32" class="link link-ref link-reveal xref-bibr">25–32</a>,<a reveal-id="DSS042C36" data-open="DSS042C36" class="link link-ref link-reveal xref-bibr">36</a></sup> and an artificial amphidiploid map.<sup><a reveal-id="DSS042C24" data-open="DSS042C24" class="link link-ref link-reveal xref-bibr">24</a></sup> Marker order and genetic distance were calculated using a regression mapping algorithm with the following parameters: Haldane's mapping function, recombination frequency ≤0.30, and LOD score ≥2.0. The graphical linkage maps were drawn using the MapChart program.<sup><a reveal-id="DSS042C42" data-open="DSS042C42" class="link link-ref link-reveal xref-bibr">42</a></sup></p>                            <h3 scrollto-destination=4624722 id="4624722" class="section-title" data-legacy-id=s2d>2.4. Comparative analysis of the <em>Arachis</em> maps with other legume genomes</h3>
<p>BLASTN<sup><a reveal-id="DSS042C43" data-open="DSS042C43" class="link link-ref link-reveal xref-bibr">43</a></sup> was used to conduct similarity searches (using the nucleotide sequences from which the mapped DNA markers were designed) against each pseudomolecule of the genome sequences of <em>C. cajan</em>,<sup><a reveal-id="DSS042C8" data-open="DSS042C8" class="link link-ref link-reveal xref-bibr">8</a></sup><em>G. max</em> (Glyma1),<sup><a reveal-id="DSS042C6" data-open="DSS042C6" class="link link-ref link-reveal xref-bibr">6</a></sup><em>L. japonicus</em> (build 2.5),<sup><a reveal-id="DSS042C5" data-open="DSS042C5" class="link link-ref link-reveal xref-bibr">5</a></sup> and <em>M. truncatula</em> (Mt3.5v4),<sup><a reveal-id="DSS042C7" data-open="DSS042C7" class="link link-ref link-reveal xref-bibr">7</a></sup> with a threshold <em>E</em>-value of 1e-20. The graphical comparative maps were drawn using the Circos program.<sup><a reveal-id="DSS042C44" data-open="DSS042C44" class="link link-ref link-reveal xref-bibr">44</a></sup></p>                            <h2 scrollto-destination=4624724 id="4624724" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=4624725 id="4624725" class="section-title" data-legacy-id=s3a>3.1. Polymorphism screening of the parental lines of the mapping populations and construction of the linkage maps</h3>
<p>Prior to polymorphism analysis between the parental lines, a total of 3902 markers, i.e. 1894 AHGS, 1571 AHS, and 437 AhTE markers, were pre-selected from the 6680 AHGS, 3187 AHS, and 1039 AhTE markers that have been published.<sup><a reveal-id="DSS042C36  DSS042C38" data-open="DSS042C36  DSS042C38" class="link link-ref link-reveal xref-bibr">36–38</a></sup> The markers were pre-selected according to the following criteria: (i) AHGS markers were selected if they were expected to detect polymorphism among the cultivated peanut lines according to the results of <em>in silico</em> polymorphism analysis;<sup><a reveal-id="DSS042C36" data-open="DSS042C36" class="link link-ref link-reveal xref-bibr">36</a></sup> (ii) AHS markers were selected if they showed polymorphisms between cultivated lines and wild species;<sup><a reveal-id="DSS042C37" data-open="DSS042C37" class="link link-ref link-reveal xref-bibr">37</a></sup> (iii) AhTE markers were selected if they amplified single or double bands.<sup><a reveal-id="DSS042C36" data-open="DSS042C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSS042C38" data-open="DSS042C38" class="link link-ref link-reveal xref-bibr">38</a></sup></p>                            <h4 scrollto-destination=4624727 id="4624727" class="section-title" data-legacy-id=s3a1>3.1.1. The AF5 population</h4>
<p>A total of 582 markers (14.9% = 582/3902), including 450 AHGS (23.8% = 450/1894), 109 AHS (6.9% = 109/1571), and 23 AhTE (5.3% = 23/437) markers, were selected as polymorphism candidates between the parental lines of the AF5 population, <em>A. duranensis</em> ‘K7988’ and <em>A. stenosperma</em> ‘V10309’. Out of the 582 polymorphic markers, 395 generated a total of 437 segregation loci, since several markers detected more than one locus. Specifically, 359, 31, 4, and 1 marker(s) detected 1, 2, 3, and 4 polymorphic loci, respectively, which were suffixed with ‘_a1’ and ‘_a2’, e.g. AHGS1647_a1 and AHGS1647_a2, to distinguish the loci each other. The 437 polymorphic loci, 284 codominant, and 153 dominant loci, were successfully mapped onto 10 LGs covering 544 cM, together with 160 previously mapped polymorphic loci from 158 markers (Fig. <a reveal-id="DSS042F1" data-open="DSS042F1" class="link link-reveal link-table xref-fig">1</a>, Table <a reveal-id="DSS042TB1" data-open="DSS042TB1" class="link link-reveal link-table xref-fig">1</a>, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).<sup><a reveal-id="DSS042C19" data-open="DSS042C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The average marker density of the map was 0.9 cM, and segregation distortions were observed in 65% (390 loci) of the mapped loci (Table <a reveal-id="DSS042TB1" data-open="DSS042TB1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="4624729" scrollto-destination="4624729"></a>
<div content-id="DSS042TB1" class="table-modal"><div class="table-wrap"><div id="DSS042TB1" class="table-wrap-title" data-id="DSS042TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Descriptions of the genetic linkage maps AF5, BF6, and TF6</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">LGs </th><th colspan="4">AF5 map<hr /> </th><th rowspan="2">LGs </th><th colspan="4">BF6 map<hr /> </th><th rowspan="2">LGs </th><th colspan="4">TF6 map<hr /> </th></tr><tr><th>Locus no. </th><th>cM </th><th>cM/
locus </th><th>Percentages of loci with segregation distortion<sup>a</sup> </th><th>Locus no. </th><th>cM </th><th>cM/
locus </th><th>Percentages of loci with segregation distortion<sup>a</sup> </th><th>Locus no. </th><th>cM </th><th>cM/locus </th><th>Percentages of loci with segregation distortion<sup>a</sup> </th></tr></thead><tbody><tr><td>AA01 </td><td>84 </td><td>71.8 </td><td>0.9 </td><td>69 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA01 </td><td>108 </td><td>102.9 </td><td>1.0 </td><td>36 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB01 </td><td>70 </td><td>31.4 </td><td>0.5 </td><td>46 </td><td>TB01 </td><td>47 </td><td>52.1 </td><td>1.1 </td><td>17 </td></tr><tr><td>AA02 </td><td>55 </td><td>51.3 </td><td>1.0 </td><td>20 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA02 </td><td>73 </td><td>30.2 </td><td>0.4 </td><td>33 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB02 </td><td>62 </td><td>34.9 </td><td>0.6 </td><td>92 </td><td>TB02 </td><td>47 </td><td>35.3 </td><td>0.8 </td><td>36 </td></tr><tr><td>AA03 </td><td>63 </td><td>33.3 </td><td>0.5 </td><td>78 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA03 </td><td>154 </td><td>112.1 </td><td>0.7 </td><td>4 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB03 </td><td>113 </td><td>34.7 </td><td>0.3 </td><td>8 </td><td>TB03 </td><td>50 </td><td>96.5 </td><td>2.0 </td><td>26 </td></tr><tr><td>AA04 </td><td>75 </td><td>84.2 </td><td>1.1 </td><td>97 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA04 </td><td>105 </td><td>103.5 </td><td>1.0 </td><td>7 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB04 </td><td>85 </td><td>68.1 </td><td>0.8 </td><td>48 </td><td>TB04 </td><td>62 </td><td>66.2 </td><td>1.1 </td><td>23 </td></tr><tr><td>AA05 </td><td>56 </td><td>56.9 </td><td>1.0 </td><td>82 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA05 </td><td>81 </td><td>46.7 </td><td>0.6 </td><td>23 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB05 </td><td>75 </td><td>73.5 </td><td>1.0 </td><td>13 </td><td>TB05 </td><td>60 </td><td>67.7 </td><td>1.1 </td><td>8 </td></tr><tr><td>AA06 </td><td>63 </td><td>48.7 </td><td>0.8 </td><td>65 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA06 </td><td>67 </td><td>90.2 </td><td>1.4 </td><td>7 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB06 </td><td>84 </td><td>44.4 </td><td>0.5 </td><td>35 </td><td>TB06 </td><td>49 </td><td>81.3 </td><td>1.7 </td><td>55 </td></tr><tr><td>AA07 </td><td>31 </td><td>30.3 </td><td>1.0 </td><td>16 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA07 </td><td>70 </td><td>54.6 </td><td>0.8 </td><td>20 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB07 </td><td>78 </td><td>56.3 </td><td>0.7 </td><td>31 </td><td>TB07 </td><td>40 </td><td>60.4 </td><td>1.5 </td><td>20 </td></tr><tr><td>AA08 </td><td>54 </td><td>74.4 </td><td>1.4 </td><td>28 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA08 </td><td>98 </td><td>110.4 </td><td>1.1 </td><td>16 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB08 </td><td>61 </td><td>29.4 </td><td>0.5 </td><td>11 </td><td>TB08 </td><td>50 </td><td>33.1 </td><td>0.7 </td><td>14 </td></tr><tr><td>AA09 </td><td>48 </td><td>24.5 </td><td>0.5 </td><td>100 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA09 </td><td>96 </td><td>82.2 </td><td>0.9 </td><td>3 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB09 </td><td>91 </td><td>44.8 </td><td>0.5 </td><td>35 </td><td>TB09 </td><td>56 </td><td>60.7 </td><td>1.1 </td><td>18 </td></tr><tr><td>AA10 </td><td>68 </td><td>68.4 </td><td>1.0 </td><td>65 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA10 </td><td>109 </td><td>108.4 </td><td>1.0 </td><td>39 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB10 </td><td>79 </td><td>43.9 </td><td>0.6 </td><td>11 </td><td>TB10 </td><td>47 </td><td>47.5 </td><td>1.0 </td><td>2 </td></tr><tr><td>Total </td><td>597 </td><td>543.8 </td><td>0.9 </td><td>65 </td><td>Total </td><td>798 </td><td>461.4 </td><td>0.6 </td><td>31 </td><td>Total </td><td>1469 </td><td>1441.8 </td><td>1.0 </td><td>19 </td></tr></tbody></table></div><div content-id="DSS042TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">LGs </th><th colspan="4">AF5 map<hr /> </th><th rowspan="2">LGs </th><th colspan="4">BF6 map<hr /> </th><th rowspan="2">LGs </th><th colspan="4">TF6 map<hr /> </th></tr><tr><th>Locus no. </th><th>cM </th><th>cM/
locus </th><th>Percentages of loci with segregation distortion<sup>a</sup> </th><th>Locus no. </th><th>cM </th><th>cM/
locus </th><th>Percentages of loci with segregation distortion<sup>a</sup> </th><th>Locus no. </th><th>cM </th><th>cM/locus </th><th>Percentages of loci with segregation distortion<sup>a</sup> </th></tr></thead><tbody><tr><td>AA01 </td><td>84 </td><td>71.8 </td><td>0.9 </td><td>69 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA01 </td><td>108 </td><td>102.9 </td><td>1.0 </td><td>36 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB01 </td><td>70 </td><td>31.4 </td><td>0.5 </td><td>46 </td><td>TB01 </td><td>47 </td><td>52.1 </td><td>1.1 </td><td>17 </td></tr><tr><td>AA02 </td><td>55 </td><td>51.3 </td><td>1.0 </td><td>20 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA02 </td><td>73 </td><td>30.2 </td><td>0.4 </td><td>33 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB02 </td><td>62 </td><td>34.9 </td><td>0.6 </td><td>92 </td><td>TB02 </td><td>47 </td><td>35.3 </td><td>0.8 </td><td>36 </td></tr><tr><td>AA03 </td><td>63 </td><td>33.3 </td><td>0.5 </td><td>78 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA03 </td><td>154 </td><td>112.1 </td><td>0.7 </td><td>4 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB03 </td><td>113 </td><td>34.7 </td><td>0.3 </td><td>8 </td><td>TB03 </td><td>50 </td><td>96.5 </td><td>2.0 </td><td>26 </td></tr><tr><td>AA04 </td><td>75 </td><td>84.2 </td><td>1.1 </td><td>97 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA04 </td><td>105 </td><td>103.5 </td><td>1.0 </td><td>7 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB04 </td><td>85 </td><td>68.1 </td><td>0.8 </td><td>48 </td><td>TB04 </td><td>62 </td><td>66.2 </td><td>1.1 </td><td>23 </td></tr><tr><td>AA05 </td><td>56 </td><td>56.9 </td><td>1.0 </td><td>82 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA05 </td><td>81 </td><td>46.7 </td><td>0.6 </td><td>23 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB05 </td><td>75 </td><td>73.5 </td><td>1.0 </td><td>13 </td><td>TB05 </td><td>60 </td><td>67.7 </td><td>1.1 </td><td>8 </td></tr><tr><td>AA06 </td><td>63 </td><td>48.7 </td><td>0.8 </td><td>65 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA06 </td><td>67 </td><td>90.2 </td><td>1.4 </td><td>7 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB06 </td><td>84 </td><td>44.4 </td><td>0.5 </td><td>35 </td><td>TB06 </td><td>49 </td><td>81.3 </td><td>1.7 </td><td>55 </td></tr><tr><td>AA07 </td><td>31 </td><td>30.3 </td><td>1.0 </td><td>16 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA07 </td><td>70 </td><td>54.6 </td><td>0.8 </td><td>20 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB07 </td><td>78 </td><td>56.3 </td><td>0.7 </td><td>31 </td><td>TB07 </td><td>40 </td><td>60.4 </td><td>1.5 </td><td>20 </td></tr><tr><td>AA08 </td><td>54 </td><td>74.4 </td><td>1.4 </td><td>28 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA08 </td><td>98 </td><td>110.4 </td><td>1.1 </td><td>16 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB08 </td><td>61 </td><td>29.4 </td><td>0.5 </td><td>11 </td><td>TB08 </td><td>50 </td><td>33.1 </td><td>0.7 </td><td>14 </td></tr><tr><td>AA09 </td><td>48 </td><td>24.5 </td><td>0.5 </td><td>100 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA09 </td><td>96 </td><td>82.2 </td><td>0.9 </td><td>3 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB09 </td><td>91 </td><td>44.8 </td><td>0.5 </td><td>35 </td><td>TB09 </td><td>56 </td><td>60.7 </td><td>1.1 </td><td>18 </td></tr><tr><td>AA10 </td><td>68 </td><td>68.4 </td><td>1.0 </td><td>65 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA10 </td><td>109 </td><td>108.4 </td><td>1.0 </td><td>39 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB10 </td><td>79 </td><td>43.9 </td><td>0.6 </td><td>11 </td><td>TB10 </td><td>47 </td><td>47.5 </td><td>1.0 </td><td>2 </td></tr><tr><td>Total </td><td>597 </td><td>543.8 </td><td>0.9 </td><td>65 </td><td>Total </td><td>798 </td><td>461.4 </td><td>0.6 </td><td>31 </td><td>Total </td><td>1469 </td><td>1441.8 </td><td>1.0 </td><td>19 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Percentages of loci that showed segregation distortion (<em>P</em> &lt; 0.01).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4624729" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS042TB1" class="table-wrap-title" data-id="DSS042TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Descriptions of the genetic linkage maps AF5, BF6, and TF6</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">LGs </th><th colspan="4">AF5 map<hr /> </th><th rowspan="2">LGs </th><th colspan="4">BF6 map<hr /> </th><th rowspan="2">LGs </th><th colspan="4">TF6 map<hr /> </th></tr><tr><th>Locus no. </th><th>cM </th><th>cM/
locus </th><th>Percentages of loci with segregation distortion<sup>a</sup> </th><th>Locus no. </th><th>cM </th><th>cM/
locus </th><th>Percentages of loci with segregation distortion<sup>a</sup> </th><th>Locus no. </th><th>cM </th><th>cM/locus </th><th>Percentages of loci with segregation distortion<sup>a</sup> </th></tr></thead><tbody><tr><td>AA01 </td><td>84 </td><td>71.8 </td><td>0.9 </td><td>69 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA01 </td><td>108 </td><td>102.9 </td><td>1.0 </td><td>36 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB01 </td><td>70 </td><td>31.4 </td><td>0.5 </td><td>46 </td><td>TB01 </td><td>47 </td><td>52.1 </td><td>1.1 </td><td>17 </td></tr><tr><td>AA02 </td><td>55 </td><td>51.3 </td><td>1.0 </td><td>20 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA02 </td><td>73 </td><td>30.2 </td><td>0.4 </td><td>33 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB02 </td><td>62 </td><td>34.9 </td><td>0.6 </td><td>92 </td><td>TB02 </td><td>47 </td><td>35.3 </td><td>0.8 </td><td>36 </td></tr><tr><td>AA03 </td><td>63 </td><td>33.3 </td><td>0.5 </td><td>78 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA03 </td><td>154 </td><td>112.1 </td><td>0.7 </td><td>4 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB03 </td><td>113 </td><td>34.7 </td><td>0.3 </td><td>8 </td><td>TB03 </td><td>50 </td><td>96.5 </td><td>2.0 </td><td>26 </td></tr><tr><td>AA04 </td><td>75 </td><td>84.2 </td><td>1.1 </td><td>97 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA04 </td><td>105 </td><td>103.5 </td><td>1.0 </td><td>7 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB04 </td><td>85 </td><td>68.1 </td><td>0.8 </td><td>48 </td><td>TB04 </td><td>62 </td><td>66.2 </td><td>1.1 </td><td>23 </td></tr><tr><td>AA05 </td><td>56 </td><td>56.9 </td><td>1.0 </td><td>82 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA05 </td><td>81 </td><td>46.7 </td><td>0.6 </td><td>23 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB05 </td><td>75 </td><td>73.5 </td><td>1.0 </td><td>13 </td><td>TB05 </td><td>60 </td><td>67.7 </td><td>1.1 </td><td>8 </td></tr><tr><td>AA06 </td><td>63 </td><td>48.7 </td><td>0.8 </td><td>65 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA06 </td><td>67 </td><td>90.2 </td><td>1.4 </td><td>7 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB06 </td><td>84 </td><td>44.4 </td><td>0.5 </td><td>35 </td><td>TB06 </td><td>49 </td><td>81.3 </td><td>1.7 </td><td>55 </td></tr><tr><td>AA07 </td><td>31 </td><td>30.3 </td><td>1.0 </td><td>16 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA07 </td><td>70 </td><td>54.6 </td><td>0.8 </td><td>20 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB07 </td><td>78 </td><td>56.3 </td><td>0.7 </td><td>31 </td><td>TB07 </td><td>40 </td><td>60.4 </td><td>1.5 </td><td>20 </td></tr><tr><td>AA08 </td><td>54 </td><td>74.4 </td><td>1.4 </td><td>28 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA08 </td><td>98 </td><td>110.4 </td><td>1.1 </td><td>16 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB08 </td><td>61 </td><td>29.4 </td><td>0.5 </td><td>11 </td><td>TB08 </td><td>50 </td><td>33.1 </td><td>0.7 </td><td>14 </td></tr><tr><td>AA09 </td><td>48 </td><td>24.5 </td><td>0.5 </td><td>100 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA09 </td><td>96 </td><td>82.2 </td><td>0.9 </td><td>3 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB09 </td><td>91 </td><td>44.8 </td><td>0.5 </td><td>35 </td><td>TB09 </td><td>56 </td><td>60.7 </td><td>1.1 </td><td>18 </td></tr><tr><td>AA10 </td><td>68 </td><td>68.4 </td><td>1.0 </td><td>65 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA10 </td><td>109 </td><td>108.4 </td><td>1.0 </td><td>39 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB10 </td><td>79 </td><td>43.9 </td><td>0.6 </td><td>11 </td><td>TB10 </td><td>47 </td><td>47.5 </td><td>1.0 </td><td>2 </td></tr><tr><td>Total </td><td>597 </td><td>543.8 </td><td>0.9 </td><td>65 </td><td>Total </td><td>798 </td><td>461.4 </td><td>0.6 </td><td>31 </td><td>Total </td><td>1469 </td><td>1441.8 </td><td>1.0 </td><td>19 </td></tr></tbody></table></div><div content-id="DSS042TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">LGs </th><th colspan="4">AF5 map<hr /> </th><th rowspan="2">LGs </th><th colspan="4">BF6 map<hr /> </th><th rowspan="2">LGs </th><th colspan="4">TF6 map<hr /> </th></tr><tr><th>Locus no. </th><th>cM </th><th>cM/
locus </th><th>Percentages of loci with segregation distortion<sup>a</sup> </th><th>Locus no. </th><th>cM </th><th>cM/
locus </th><th>Percentages of loci with segregation distortion<sup>a</sup> </th><th>Locus no. </th><th>cM </th><th>cM/locus </th><th>Percentages of loci with segregation distortion<sup>a</sup> </th></tr></thead><tbody><tr><td>AA01 </td><td>84 </td><td>71.8 </td><td>0.9 </td><td>69 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA01 </td><td>108 </td><td>102.9 </td><td>1.0 </td><td>36 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB01 </td><td>70 </td><td>31.4 </td><td>0.5 </td><td>46 </td><td>TB01 </td><td>47 </td><td>52.1 </td><td>1.1 </td><td>17 </td></tr><tr><td>AA02 </td><td>55 </td><td>51.3 </td><td>1.0 </td><td>20 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA02 </td><td>73 </td><td>30.2 </td><td>0.4 </td><td>33 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB02 </td><td>62 </td><td>34.9 </td><td>0.6 </td><td>92 </td><td>TB02 </td><td>47 </td><td>35.3 </td><td>0.8 </td><td>36 </td></tr><tr><td>AA03 </td><td>63 </td><td>33.3 </td><td>0.5 </td><td>78 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA03 </td><td>154 </td><td>112.1 </td><td>0.7 </td><td>4 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB03 </td><td>113 </td><td>34.7 </td><td>0.3 </td><td>8 </td><td>TB03 </td><td>50 </td><td>96.5 </td><td>2.0 </td><td>26 </td></tr><tr><td>AA04 </td><td>75 </td><td>84.2 </td><td>1.1 </td><td>97 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA04 </td><td>105 </td><td>103.5 </td><td>1.0 </td><td>7 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB04 </td><td>85 </td><td>68.1 </td><td>0.8 </td><td>48 </td><td>TB04 </td><td>62 </td><td>66.2 </td><td>1.1 </td><td>23 </td></tr><tr><td>AA05 </td><td>56 </td><td>56.9 </td><td>1.0 </td><td>82 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA05 </td><td>81 </td><td>46.7 </td><td>0.6 </td><td>23 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB05 </td><td>75 </td><td>73.5 </td><td>1.0 </td><td>13 </td><td>TB05 </td><td>60 </td><td>67.7 </td><td>1.1 </td><td>8 </td></tr><tr><td>AA06 </td><td>63 </td><td>48.7 </td><td>0.8 </td><td>65 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA06 </td><td>67 </td><td>90.2 </td><td>1.4 </td><td>7 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB06 </td><td>84 </td><td>44.4 </td><td>0.5 </td><td>35 </td><td>TB06 </td><td>49 </td><td>81.3 </td><td>1.7 </td><td>55 </td></tr><tr><td>AA07 </td><td>31 </td><td>30.3 </td><td>1.0 </td><td>16 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA07 </td><td>70 </td><td>54.6 </td><td>0.8 </td><td>20 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB07 </td><td>78 </td><td>56.3 </td><td>0.7 </td><td>31 </td><td>TB07 </td><td>40 </td><td>60.4 </td><td>1.5 </td><td>20 </td></tr><tr><td>AA08 </td><td>54 </td><td>74.4 </td><td>1.4 </td><td>28 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA08 </td><td>98 </td><td>110.4 </td><td>1.1 </td><td>16 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB08 </td><td>61 </td><td>29.4 </td><td>0.5 </td><td>11 </td><td>TB08 </td><td>50 </td><td>33.1 </td><td>0.7 </td><td>14 </td></tr><tr><td>AA09 </td><td>48 </td><td>24.5 </td><td>0.5 </td><td>100 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA09 </td><td>96 </td><td>82.2 </td><td>0.9 </td><td>3 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB09 </td><td>91 </td><td>44.8 </td><td>0.5 </td><td>35 </td><td>TB09 </td><td>56 </td><td>60.7 </td><td>1.1 </td><td>18 </td></tr><tr><td>AA10 </td><td>68 </td><td>68.4 </td><td>1.0 </td><td>65 </td><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>TA10 </td><td>109 </td><td>108.4 </td><td>1.0 </td><td>39 </td></tr><tr><td> </td><td>— </td><td>— </td><td>— </td><td>— </td><td>BB10 </td><td>79 </td><td>43.9 </td><td>0.6 </td><td>11 </td><td>TB10 </td><td>47 </td><td>47.5 </td><td>1.0 </td><td>2 </td></tr><tr><td>Total </td><td>597 </td><td>543.8 </td><td>0.9 </td><td>65 </td><td>Total </td><td>798 </td><td>461.4 </td><td>0.6 </td><td>31 </td><td>Total </td><td>1469 </td><td>1441.8 </td><td>1.0 </td><td>19 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Percentages of loci that showed segregation distortion (<em>P</em> &lt; 0.01).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4624729" target="_blank">View Large</a></div></div>                        <a id="4624730" scrollto-destination="4624730"></a>
<div id="DSS042F1" data-id="DSS042F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss042/2/m_dss04201.jpeg?Expires=2147483647&Signature=rVy8oVDPYMnVkMiqKcEP9ea9XHisOBzxDyMtPY9BbyHCAywRZ3276m1Tj5bytP324UWQqsLZ-r4C3okxz80uuQMKh4OvjCUWv2poKx5G5UOC4t27k9H0NE6o7vQLs25aOjBi~p6CwfhV9sgfcwZfBEHSdAsTIWIm9wfQD~X7pZSMh3JCBhON9N-rqdu6uAPdfm2FV-m5Fu1YTngGm5Puyk5xF94127XTb2hgy0jr6-e5CfnQz~YICyQQHH2gAF8L~de8B8ahKiJQfqmRnkwJCnSt6Y~uRzTd528T0cMXTixlQib5QBZps0rsrIE6EJn5CatluLaRRkowCSJNC4o2RA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage maps obtained for AF5, BF6, and TF6 populations. The linkage groups in the AF5, BF6, and TF6 maps are indicated by AA, BB, and TA or TB, respectively, and the homoeologous groups are prefixed with HGs. Homologous or homoeologous marker loci are connected by lines." path-from-xml="dss04201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4624730/dss04201.jpeg" path-from-xml="dss04201.jpeg" target="_blank">View large</a><a section="4624730" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss042/2/dss04201.jpeg?Expires=2147483647&Signature=C6bWcRjqiIWUzoS~HeJKmsHAkmnn47wGyawXFj0TYgU6tkiMNiH8jLD4NHGxm9WVCwdzxhqRm6~Z4HRtLeVg~jh5q0-AqeLz1x5Is6Budc~BjnjmteF~naLvPBqtscztGxmD6wMyL-eTg8XIMnIp2mLJzL4QoZrmaCjSMJijuOZkDXpLZMRpKy3cFw9bfFiud-lmgCWKmmkaH6qwHaEBYTU7or5yFQEfJk09TGun39ZmDSqwT8BRR6DFdAjCPJnOaOwCjqEe3NP8~x~HU2rlMXF5tbY0KEvGr2QUuLYsrh8iWvgVwFz9HivHFQox2TnrqSL~xPdz94cmcsRk3sQLfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4624730&ar=333781&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss04201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic linkage maps obtained for AF5, BF6, and TF6 populations. The linkage groups in the AF5, BF6, and TF6 maps are indicated by AA, BB, and TA or TB, respectively, and the homoeologous groups are prefixed with HGs. Homologous or homoeologous marker loci are connected by lines.</p></div></div><div content-id="DSS042F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss042/2/m_dss04201.jpeg?Expires=2147483647&Signature=rVy8oVDPYMnVkMiqKcEP9ea9XHisOBzxDyMtPY9BbyHCAywRZ3276m1Tj5bytP324UWQqsLZ-r4C3okxz80uuQMKh4OvjCUWv2poKx5G5UOC4t27k9H0NE6o7vQLs25aOjBi~p6CwfhV9sgfcwZfBEHSdAsTIWIm9wfQD~X7pZSMh3JCBhON9N-rqdu6uAPdfm2FV-m5Fu1YTngGm5Puyk5xF94127XTb2hgy0jr6-e5CfnQz~YICyQQHH2gAF8L~de8B8ahKiJQfqmRnkwJCnSt6Y~uRzTd528T0cMXTixlQib5QBZps0rsrIE6EJn5CatluLaRRkowCSJNC4o2RA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genetic linkage maps obtained for AF5, BF6, and TF6 populations. The linkage groups in the AF5, BF6, and TF6 maps are indicated by AA, BB, and TA or TB, respectively, and the homoeologous groups are prefixed with HGs. Homologous or homoeologous marker loci are connected by lines." path-from-xml="dss04201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4624730/dss04201.jpeg" path-from-xml="dss04201.jpeg" target="_blank">View large</a><a section="4624730" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss042/2/dss04201.jpeg?Expires=2147483647&Signature=C6bWcRjqiIWUzoS~HeJKmsHAkmnn47wGyawXFj0TYgU6tkiMNiH8jLD4NHGxm9WVCwdzxhqRm6~Z4HRtLeVg~jh5q0-AqeLz1x5Is6Budc~BjnjmteF~naLvPBqtscztGxmD6wMyL-eTg8XIMnIp2mLJzL4QoZrmaCjSMJijuOZkDXpLZMRpKy3cFw9bfFiud-lmgCWKmmkaH6qwHaEBYTU7or5yFQEfJk09TGun39ZmDSqwT8BRR6DFdAjCPJnOaOwCjqEe3NP8~x~HU2rlMXF5tbY0KEvGr2QUuLYsrh8iWvgVwFz9HivHFQox2TnrqSL~xPdz94cmcsRk3sQLfg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4624730&ar=333781&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss04201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genetic linkage maps obtained for AF5, BF6, and TF6 populations. The linkage groups in the AF5, BF6, and TF6 maps are indicated by AA, BB, and TA or TB, respectively, and the homoeologous groups are prefixed with HGs. Homologous or homoeologous marker loci are connected by lines.</p></div></div>                            <h4 scrollto-destination=4624731 id="4624731" class="section-title" data-legacy-id=s3a2>3.1.2. The BF6 population</h4>
<p>A total of 862 markers (22.1% = 862/3902), including 513 AHGS (27.1% = 513/1894), 279 AHS (17.8% = 279/1571), and 70 AhTE (16.0% = 70/437) markers, were selected as polymorphism candidates between the parental lines of the BF6 population, <em>A. ipaënsis</em> ‘K30076’ and <em>A. magna</em> ‘K30097’. Out of the 862 polymorphic markers, 624 generated a total of 680 segregation loci, since, as with the AF5 population, several of these markers detected more than one locus. Specifically, 579, 35, 9, and 1 marker(s) detected 1, 2, 3, and 4 polymorphic loci, respectively, which were suffixed with ‘_b1’ and ‘_b2’, e.g. AHGS1478_b1 and AHGS1478_b2, to distinguish the loci each other. The 680 polymorphic loci, 549 codominant, and 131 dominant, were successfully mapped onto 10 LGs covering 461 cM together with 118 previously mapped loci from 116 markers (Fig. <a reveal-id="DSS042F1" data-open="DSS042F1" class="link link-reveal link-table xref-fig">1</a>, Table <a reveal-id="DSS042TB1" data-open="DSS042TB1" class="link link-reveal link-table xref-fig">1</a>, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).<sup><a reveal-id="DSS042C20" data-open="DSS042C20" class="link link-ref link-reveal xref-bibr">20</a></sup> The average marker density of the map was 0.6 cM, and segregation distortions were observed in 31% (250 loci) of the mapped loci (Table <a reveal-id="DSS042TB1" data-open="DSS042TB1" class="link link-reveal link-table xref-fig">1</a>).</p>                            <h4 scrollto-destination=4624733 id="4624733" class="section-title" data-legacy-id=s3a3>3.1.3. The TF6 population</h4>
<p>Together with the parental lines of TF6, the two original diploids of the artificial amphidiploid, <em>A. ipaënsis</em> ‘K30076’ and <em>A. duranensis</em> ‘V14167’, were also genotyped to identify the genome origin (A or B genome) of the polymorphic DNA fragments. A total of 1144 markers (29.3% = 1144/3902), including 664 AHGS (35.1% = 664/1894), 257 AHS (16.4% = 257/1571), and 223 AhTE (51.0% = 223/437) were selected as polymorphism candidates between the parental lines of the TF6 population, <em>A. hypogaea</em> ‘Runner IAC 886’ and an artificial amphidiploid (<em>A. ipaënsis</em> ‘K30076’ × <em>A. duranensis</em> ‘V14167’)<sup>4×</sup>. Out of the 1144 polymorphic markers, 1055 generated a total of 1261 segregation loci, since, as with the AF5 and BF6 populations, several of these markers detected more than one locus. Specifically, 879, 150, 22, and 4 markers detected 1, 2, 3, and 4 polymorphic loci, respectively, which were suffixed with ‘_t1’ and ‘_t2’, e.g. AHGS1991_t1 and AHGS1991_t2, to distinguish the loci each other. The 1261 polymorphic loci, 970 codominant, and 291 dominant, were successfully mapped onto 20 LGs covering 1442 cM together with 208 loci from 180 markers (Moretzsohn <em>et al</em>. unpublished data). The 20 LGs were classified into 10 pairs of A and B genomes, according to the DNA fragment size of mapped loci corresponding to ‘K30076’ or ‘V14167’ (Fig. <a reveal-id="DSS042F1" data-open="DSS042F1" class="link link-reveal link-table xref-fig">1</a>, Table <a reveal-id="DSS042TB1" data-open="DSS042TB1" class="link link-reveal link-table xref-fig">1</a>, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The average marker density of the map was 1.0 cM, and segregation distortions were observed in 19% (285 loci) of the mapped loci (Table <a reveal-id="DSS042TB1" data-open="DSS042TB1" class="link link-reveal link-table xref-fig">1</a>).</p><p>In total, the number of non-redundant AHGS, AHS, and AhTE markers mapped on the three maps were 890, 446, and 211, respectively. Out of the total 1547 markers, 73 (67 AHGSs and six AHSs) and 381 (298 AHGSs, 59 AHSs, and 24 AhTEs) were mapped on three and two maps, respectively, while 1093 (525 AHGSs, 381 AHSs, and 187 AhTEs) were mapped on just one map; the AF5, BF6, or TF6 map (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Comparing the three maps, 203 of the 597 mapped loci on the AF5 map, and 157 out of 798 loci on the BF6 map, were also located on the TF6 map (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>).</p><p>The LGs were designated AA, BB, and TA or TB in the AF5, BF6 and TF6 maps, respectively. The nomenclature of the LGs of the diploid maps was tentatively determined according to the commonly mapped marker loci between the present and the previously constructed maps.<sup><a reveal-id="DSS042C19" data-open="DSS042C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSS042C20" data-open="DSS042C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Eight LGs of the AF5 map (AA01 and AA03–AA09) and all 10 of the LGs of the BF6 map (BB01–BB10) were assigned to the corresponding LGs on the previous maps (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The two LGs of the AF5 map were assigned to ‘Group 02’ in the previous map,<sup><a reveal-id="DSS042C19" data-open="DSS042C19" class="link link-ref link-reveal xref-bibr">19</a></sup> and they were tentatively named AA02 and AA10 (Table <a reveal-id="DSS042TB1" data-open="DSS042TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). ‘Group 10’ in the previous map<sup><a reveal-id="DSS042C19" data-open="DSS042C19" class="link link-ref link-reveal xref-bibr">19</a></sup> was disassembled on the AF5 map. On the other hand, in the TF6 map, TA01, TA03–TA09, and TB01–TB10 were assigned to the corresponding LGs of the AF5 and BF6 maps, and TA02 and TA10 were assigned to the AA02 and AA10 LGs, respectively (Table <a reveal-id="DSS042TB1" data-open="DSS042TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). After integrating all of these data, 10 homoeologous groups (HGs) were identified on the TF6 map.</p><p>The marker order was almost completely conserved between the HGs except for HG04, HG07, and HG08 (Fig. <a reveal-id="DSS042F1" data-open="DSS042F1" class="link link-reveal link-table xref-fig">1</a>). In HG04, the lower part of TA04 corresponded to the upper part of TB04 (Fig. <a reveal-id="DSS042F1" data-open="DSS042F1" class="link link-reveal link-table xref-fig">1</a>). In HG07, the loci mapped onto TB07 corresponded to those of TA07 and the upper part of TA08. Moreover, corresponding loci on TB08 were observed on the lower parts of TA08 and TA07 (Fig. <a reveal-id="DSS042F1" data-open="DSS042F1" class="link link-reveal link-table xref-fig">1</a>), indicating a translocation between the A and B genomes as previously suggested.<sup><a reveal-id="DSS042C20" data-open="DSS042C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSS042C22" data-open="DSS042C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSS042C24" data-open="DSS042C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p>                            <h3 scrollto-destination=4624738 id="4624738" class="section-title" data-legacy-id=s3b>3.2. Integration of the genetic linkage maps into a consensus map</h3>
<p>An integrated consensus map was constructed based on the segregation genotypes of 16 populations: three from this study and 13 from previous studies (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>).<sup><a reveal-id="DSS042C19" data-open="DSS042C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSS042C20" data-open="DSS042C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSS042C24" data-open="DSS042C24" class="link link-ref link-reveal xref-bibr">24</a>–<a reveal-id="DSS042C32" data-open="DSS042C32" class="link link-ref link-reveal xref-bibr">32</a>,<a reveal-id="DSS042C36" data-open="DSS042C36" class="link link-ref link-reveal xref-bibr">36</a></sup> The integrated consensus map was 2651 cM in total length and comprised 20 LGs, on which 3693 loci, including 1564 AHGS, 569 AHS, 450 AhTE, and 1110 other loci, were mapped (Table <a reveal-id="DSS042TB2" data-open="DSS042TB2" class="link link-reveal link-table xref-fig">2</a>, Fig. <a reveal-id="DSS042F2" data-open="DSS042F2" class="link link-reveal link-table xref-fig">2</a>, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>), out of which markers mapped on mote than two loci were suffixed with ‘_c1’ and ‘_c2’, e.g. AHGS1403_c1 and AHGS1403_c2, to distinguish each other. The average marker density of this map was 0.7 cM (Table <a reveal-id="DSS042TB2" data-open="DSS042TB2" class="link link-reveal link-table xref-fig">2</a>). On the consensus map, 391 marker loci were commonly mapped onto pairs of LGs in each HG, and the order of the markers was roughly conserved (Fig. <a reveal-id="DSS042F2" data-open="DSS042F2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="4624740" scrollto-destination="4624740"></a>
<div content-id="DSS042TB2" class="table-modal"><div class="table-wrap"><div id="DSS042TB2" class="table-wrap-title" data-id="DSS042TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Descriptions of the integrated consensus map and integrated LGs of the population-specific maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LGs </th><th>Locus no. </th><th>cM </th><th>cM/locus </th><th>Length of the locus cluster<sup>a</sup> </th><th>Locus no. in the marker cluster </th></tr></thead><tbody><tr><td>A01 </td><td>238 </td><td>183.8 </td><td>0.8 </td><td>25 </td><td>158 </td></tr><tr><td>B01 </td><td>168 </td><td>84.7 </td><td>0.5 </td><td>25 </td><td>129 </td></tr><tr><td>A02 </td><td>126 </td><td>78.6 </td><td>0.6 </td><td>20 </td><td>86 </td></tr><tr><td>B02 </td><td>154 </td><td>112.0 </td><td>0.7 </td><td>25 </td><td>104 </td></tr><tr><td>A03 </td><td>272 </td><td>151.6 </td><td>0.6 </td><td>40 </td><td>175 </td></tr><tr><td>B03 </td><td>234 </td><td>144.8 </td><td>0.6 </td><td>40 </td><td>167 </td></tr><tr><td>A04 </td><td>213 </td><td>106.8 </td><td>0.5 </td><td>20 </td><td>106 </td></tr><tr><td>B04 </td><td>208 </td><td>132.1 </td><td>0.6 </td><td>30 </td><td>122 </td></tr><tr><td>A05 </td><td>189 </td><td>126.0 </td><td>0.7 </td><td>30 </td><td>135 </td></tr><tr><td>B05 </td><td>169 </td><td>112.7 </td><td>0.7 </td><td>15 </td><td>77 </td></tr><tr><td>A06 </td><td>187 </td><td>136.4 </td><td>0.7 </td><td>15 </td><td>85 </td></tr><tr><td>B06 </td><td>176 </td><td>152.5 </td><td>0.9 </td><td>20 </td><td>109 </td></tr><tr><td>A07 </td><td>145 </td><td>167.0 </td><td>1.2 </td><td>20 </td><td>76 </td></tr><tr><td>B07 </td><td>162 </td><td>179.1 </td><td>1.1 </td><td>25 </td><td>105 </td></tr><tr><td>A08 </td><td>167 </td><td>126.3 </td><td>0.8 </td><td>20 </td><td>53 </td></tr><tr><td>B08 </td><td>176 </td><td>107.3 </td><td>0.6 </td><td>30 </td><td>138 </td></tr><tr><td>A09 </td><td>194 </td><td>126.5 </td><td>0.7 </td><td>30 </td><td>141 </td></tr><tr><td>B09 </td><td>179 </td><td>170.5 </td><td>1.0 </td><td>30 </td><td>121 </td></tr><tr><td>A10 </td><td>172 </td><td>131.1 </td><td>0.8 </td><td>20 </td><td>117 </td></tr><tr><td>B10 </td><td>164 </td><td>121.3 </td><td>0.7 </td><td>25 </td><td>115 </td></tr><tr><td>Total </td><td>3693 </td><td>2651.1 </td><td>0.7 </td><td>505 </td><td>2319 </td></tr></tbody></table></div><div content-id="DSS042TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>LGs </th><th>Locus no. </th><th>cM </th><th>cM/locus </th><th>Length of the locus cluster<sup>a</sup> </th><th>Locus no. in the marker cluster </th></tr></thead><tbody><tr><td>A01 </td><td>238 </td><td>183.8 </td><td>0.8 </td><td>25 </td><td>158 </td></tr><tr><td>B01 </td><td>168 </td><td>84.7 </td><td>0.5 </td><td>25 </td><td>129 </td></tr><tr><td>A02 </td><td>126 </td><td>78.6 </td><td>0.6 </td><td>20 </td><td>86 </td></tr><tr><td>B02 </td><td>154 </td><td>112.0 </td><td>0.7 </td><td>25 </td><td>104 </td></tr><tr><td>A03 </td><td>272 </td><td>151.6 </td><td>0.6 </td><td>40 </td><td>175 </td></tr><tr><td>B03 </td><td>234 </td><td>144.8 </td><td>0.6 </td><td>40 </td><td>167 </td></tr><tr><td>A04 </td><td>213 </td><td>106.8 </td><td>0.5 </td><td>20 </td><td>106 </td></tr><tr><td>B04 </td><td>208 </td><td>132.1 </td><td>0.6 </td><td>30 </td><td>122 </td></tr><tr><td>A05 </td><td>189 </td><td>126.0 </td><td>0.7 </td><td>30 </td><td>135 </td></tr><tr><td>B05 </td><td>169 </td><td>112.7 </td><td>0.7 </td><td>15 </td><td>77 </td></tr><tr><td>A06 </td><td>187 </td><td>136.4 </td><td>0.7 </td><td>15 </td><td>85 </td></tr><tr><td>B06 </td><td>176 </td><td>152.5 </td><td>0.9 </td><td>20 </td><td>109 </td></tr><tr><td>A07 </td><td>145 </td><td>167.0 </td><td>1.2 </td><td>20 </td><td>76 </td></tr><tr><td>B07 </td><td>162 </td><td>179.1 </td><td>1.1 </td><td>25 </td><td>105 </td></tr><tr><td>A08 </td><td>167 </td><td>126.3 </td><td>0.8 </td><td>20 </td><td>53 </td></tr><tr><td>B08 </td><td>176 </td><td>107.3 </td><td>0.6 </td><td>30 </td><td>138 </td></tr><tr><td>A09 </td><td>194 </td><td>126.5 </td><td>0.7 </td><td>30 </td><td>141 </td></tr><tr><td>B09 </td><td>179 </td><td>170.5 </td><td>1.0 </td><td>30 </td><td>121 </td></tr><tr><td>A10 </td><td>172 </td><td>131.1 </td><td>0.8 </td><td>20 </td><td>117 </td></tr><tr><td>B10 </td><td>164 </td><td>121.3 </td><td>0.7 </td><td>25 </td><td>115 </td></tr><tr><td>Total </td><td>3693 </td><td>2651.1 </td><td>0.7 </td><td>505 </td><td>2319 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Regions in which ≥10 loci were mapped in 5 cM windows.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4624740" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS042TB2" class="table-wrap-title" data-id="DSS042TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Descriptions of the integrated consensus map and integrated LGs of the population-specific maps</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>LGs </th><th>Locus no. </th><th>cM </th><th>cM/locus </th><th>Length of the locus cluster<sup>a</sup> </th><th>Locus no. in the marker cluster </th></tr></thead><tbody><tr><td>A01 </td><td>238 </td><td>183.8 </td><td>0.8 </td><td>25 </td><td>158 </td></tr><tr><td>B01 </td><td>168 </td><td>84.7 </td><td>0.5 </td><td>25 </td><td>129 </td></tr><tr><td>A02 </td><td>126 </td><td>78.6 </td><td>0.6 </td><td>20 </td><td>86 </td></tr><tr><td>B02 </td><td>154 </td><td>112.0 </td><td>0.7 </td><td>25 </td><td>104 </td></tr><tr><td>A03 </td><td>272 </td><td>151.6 </td><td>0.6 </td><td>40 </td><td>175 </td></tr><tr><td>B03 </td><td>234 </td><td>144.8 </td><td>0.6 </td><td>40 </td><td>167 </td></tr><tr><td>A04 </td><td>213 </td><td>106.8 </td><td>0.5 </td><td>20 </td><td>106 </td></tr><tr><td>B04 </td><td>208 </td><td>132.1 </td><td>0.6 </td><td>30 </td><td>122 </td></tr><tr><td>A05 </td><td>189 </td><td>126.0 </td><td>0.7 </td><td>30 </td><td>135 </td></tr><tr><td>B05 </td><td>169 </td><td>112.7 </td><td>0.7 </td><td>15 </td><td>77 </td></tr><tr><td>A06 </td><td>187 </td><td>136.4 </td><td>0.7 </td><td>15 </td><td>85 </td></tr><tr><td>B06 </td><td>176 </td><td>152.5 </td><td>0.9 </td><td>20 </td><td>109 </td></tr><tr><td>A07 </td><td>145 </td><td>167.0 </td><td>1.2 </td><td>20 </td><td>76 </td></tr><tr><td>B07 </td><td>162 </td><td>179.1 </td><td>1.1 </td><td>25 </td><td>105 </td></tr><tr><td>A08 </td><td>167 </td><td>126.3 </td><td>0.8 </td><td>20 </td><td>53 </td></tr><tr><td>B08 </td><td>176 </td><td>107.3 </td><td>0.6 </td><td>30 </td><td>138 </td></tr><tr><td>A09 </td><td>194 </td><td>126.5 </td><td>0.7 </td><td>30 </td><td>141 </td></tr><tr><td>B09 </td><td>179 </td><td>170.5 </td><td>1.0 </td><td>30 </td><td>121 </td></tr><tr><td>A10 </td><td>172 </td><td>131.1 </td><td>0.8 </td><td>20 </td><td>117 </td></tr><tr><td>B10 </td><td>164 </td><td>121.3 </td><td>0.7 </td><td>25 </td><td>115 </td></tr><tr><td>Total </td><td>3693 </td><td>2651.1 </td><td>0.7 </td><td>505 </td><td>2319 </td></tr></tbody></table></div><div content-id="DSS042TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>LGs </th><th>Locus no. </th><th>cM </th><th>cM/locus </th><th>Length of the locus cluster<sup>a</sup> </th><th>Locus no. in the marker cluster </th></tr></thead><tbody><tr><td>A01 </td><td>238 </td><td>183.8 </td><td>0.8 </td><td>25 </td><td>158 </td></tr><tr><td>B01 </td><td>168 </td><td>84.7 </td><td>0.5 </td><td>25 </td><td>129 </td></tr><tr><td>A02 </td><td>126 </td><td>78.6 </td><td>0.6 </td><td>20 </td><td>86 </td></tr><tr><td>B02 </td><td>154 </td><td>112.0 </td><td>0.7 </td><td>25 </td><td>104 </td></tr><tr><td>A03 </td><td>272 </td><td>151.6 </td><td>0.6 </td><td>40 </td><td>175 </td></tr><tr><td>B03 </td><td>234 </td><td>144.8 </td><td>0.6 </td><td>40 </td><td>167 </td></tr><tr><td>A04 </td><td>213 </td><td>106.8 </td><td>0.5 </td><td>20 </td><td>106 </td></tr><tr><td>B04 </td><td>208 </td><td>132.1 </td><td>0.6 </td><td>30 </td><td>122 </td></tr><tr><td>A05 </td><td>189 </td><td>126.0 </td><td>0.7 </td><td>30 </td><td>135 </td></tr><tr><td>B05 </td><td>169 </td><td>112.7 </td><td>0.7 </td><td>15 </td><td>77 </td></tr><tr><td>A06 </td><td>187 </td><td>136.4 </td><td>0.7 </td><td>15 </td><td>85 </td></tr><tr><td>B06 </td><td>176 </td><td>152.5 </td><td>0.9 </td><td>20 </td><td>109 </td></tr><tr><td>A07 </td><td>145 </td><td>167.0 </td><td>1.2 </td><td>20 </td><td>76 </td></tr><tr><td>B07 </td><td>162 </td><td>179.1 </td><td>1.1 </td><td>25 </td><td>105 </td></tr><tr><td>A08 </td><td>167 </td><td>126.3 </td><td>0.8 </td><td>20 </td><td>53 </td></tr><tr><td>B08 </td><td>176 </td><td>107.3 </td><td>0.6 </td><td>30 </td><td>138 </td></tr><tr><td>A09 </td><td>194 </td><td>126.5 </td><td>0.7 </td><td>30 </td><td>141 </td></tr><tr><td>B09 </td><td>179 </td><td>170.5 </td><td>1.0 </td><td>30 </td><td>121 </td></tr><tr><td>A10 </td><td>172 </td><td>131.1 </td><td>0.8 </td><td>20 </td><td>117 </td></tr><tr><td>B10 </td><td>164 </td><td>121.3 </td><td>0.7 </td><td>25 </td><td>115 </td></tr><tr><td>Total </td><td>3693 </td><td>2651.1 </td><td>0.7 </td><td>505 </td><td>2319 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Regions in which ≥10 loci were mapped in 5 cM windows.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4624740" target="_blank">View Large</a></div></div>                        <a id="4624741" scrollto-destination="4624741"></a>
<div id="DSS042F2" data-id="DSS042F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss042/2/m_dss04202.jpeg?Expires=2147483647&Signature=XseExNgdjlkFJ-Iy13DO7ZsWJC7Ilvpwrrl41iAdYvO7MVcLcSmFf0uPPoC5xqhOCpc3Ump8OflMDtAYGPG8p08tyYXFyMZ0MpPGQW8rWNmC-4u4IieWFwuTpTpMbgYjJ6J5GBf-q3WSGDbNrFzSATODDJZiNwZLRb0NgRVcmWMGx~PfDGYGVTl-t40OPctR1PcC19CwW0hf14UgPgXsHH37wTtSmjn0Nnmj56KewIBYXEvV4dr9VjHnF9zA4bX4BzoIXvrcvjrA1I3XrcRbouQHqOR6yrRdeSEV3d4CrfT4V7K0H5tcXXvJ0sy3r-Pg~tdxAfLPktR8JldssUIP4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the integrated consensus tetraploid Arachis map with diploid maps. The abbreviations AA and BB indicate LGs on the AF5 and BF6 maps, respectively, and the prefixes A and B show LGs on the integrated consensus map. Horizontal lines on the LGs indicate the positions of the mapped loci; the loci derived from one, two, three, four, five, and more than five genetic linkage maps are shown in black, blue, light blue, green, yellow, and red. Vertical bars on the left side of the LGs indicate the locus clusters. Homologous and homoeologous loci are connected by black lines." path-from-xml="dss04202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4624741/dss04202.jpeg" path-from-xml="dss04202.jpeg" target="_blank">View large</a><a section="4624741" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss042/2/dss04202.jpeg?Expires=2147483647&Signature=PORTdqpOuEjmXCzSnQliFxittL2RMBpHqHZqZ75UU4xeoWC-TsSSJdQLS1lVWCDI6fIMkXe24GspQshsik0~3ox-CmVXeiU60LBySmAXM0IeZat79n8zIF9fjeKPpZryjTYfpHbmNllexfs4Z8yWo7ICuypgyuJex~lu2byAoht3WrCiT4YPya8eCIEP96E5NCt3v4yno0olAdomA1RVBQ7M3Z533n7kTETLoE8Afck1lbWMivHOkpvlbi8ekB4y7bF-KtR3ndugQMWkSEv4lUyuv~w44d4GiR6rnblCBm4~aGo4vA88F8pKjKJeZjU3h~u5DnXRKxjv-7HHu6LGYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4624741&ar=333781&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss04202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the integrated consensus tetraploid <em>Arachis</em> map with diploid maps. The abbreviations AA and BB indicate LGs on the AF5 and BF6 maps, respectively, and the prefixes A and B show LGs on the integrated consensus map. Horizontal lines on the LGs indicate the positions of the mapped loci; the loci derived from one, two, three, four, five, and more than five genetic linkage maps are shown in black, blue, light blue, green, yellow, and red. Vertical bars on the left side of the LGs indicate the locus clusters. Homologous and homoeologous loci are connected by black lines.</p></div></div><div content-id="DSS042F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss042/2/m_dss04202.jpeg?Expires=2147483647&Signature=XseExNgdjlkFJ-Iy13DO7ZsWJC7Ilvpwrrl41iAdYvO7MVcLcSmFf0uPPoC5xqhOCpc3Ump8OflMDtAYGPG8p08tyYXFyMZ0MpPGQW8rWNmC-4u4IieWFwuTpTpMbgYjJ6J5GBf-q3WSGDbNrFzSATODDJZiNwZLRb0NgRVcmWMGx~PfDGYGVTl-t40OPctR1PcC19CwW0hf14UgPgXsHH37wTtSmjn0Nnmj56KewIBYXEvV4dr9VjHnF9zA4bX4BzoIXvrcvjrA1I3XrcRbouQHqOR6yrRdeSEV3d4CrfT4V7K0H5tcXXvJ0sy3r-Pg~tdxAfLPktR8JldssUIP4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of the integrated consensus tetraploid Arachis map with diploid maps. The abbreviations AA and BB indicate LGs on the AF5 and BF6 maps, respectively, and the prefixes A and B show LGs on the integrated consensus map. Horizontal lines on the LGs indicate the positions of the mapped loci; the loci derived from one, two, three, four, five, and more than five genetic linkage maps are shown in black, blue, light blue, green, yellow, and red. Vertical bars on the left side of the LGs indicate the locus clusters. Homologous and homoeologous loci are connected by black lines." path-from-xml="dss04202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4624741/dss04202.jpeg" path-from-xml="dss04202.jpeg" target="_blank">View large</a><a section="4624741" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss042/2/dss04202.jpeg?Expires=2147483647&Signature=PORTdqpOuEjmXCzSnQliFxittL2RMBpHqHZqZ75UU4xeoWC-TsSSJdQLS1lVWCDI6fIMkXe24GspQshsik0~3ox-CmVXeiU60LBySmAXM0IeZat79n8zIF9fjeKPpZryjTYfpHbmNllexfs4Z8yWo7ICuypgyuJex~lu2byAoht3WrCiT4YPya8eCIEP96E5NCt3v4yno0olAdomA1RVBQ7M3Z533n7kTETLoE8Afck1lbWMivHOkpvlbi8ekB4y7bF-KtR3ndugQMWkSEv4lUyuv~w44d4GiR6rnblCBm4~aGo4vA88F8pKjKJeZjU3h~u5DnXRKxjv-7HHu6LGYQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4624741&ar=333781&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss04202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of the integrated consensus tetraploid <em>Arachis</em> map with diploid maps. The abbreviations AA and BB indicate LGs on the AF5 and BF6 maps, respectively, and the prefixes A and B show LGs on the integrated consensus map. Horizontal lines on the LGs indicate the positions of the mapped loci; the loci derived from one, two, three, four, five, and more than five genetic linkage maps are shown in black, blue, light blue, green, yellow, and red. Vertical bars on the left side of the LGs indicate the locus clusters. Homologous and homoeologous loci are connected by black lines.</p></div></div><p>In the present study, locus clusters were defined as regions in which ≥10 loci were mapped in 5 cM windows. They were observed in all of the LGs (Fig. <a reveal-id="DSS042F2" data-open="DSS042F2" class="link link-reveal link-table xref-fig">2</a>, Table <a reveal-id="DSS042TB2" data-open="DSS042TB2" class="link link-reveal link-table xref-fig">2</a>, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). The total length of the locus clusters was 505 cM (101 clusters), varying from 15 cM (three clusters) in A06 and B05 to 40 cM (eight clusters) in A03 and B03. A total of 2319 loci, representing 62.8% of the mapped loci, were located in 101 clusters, which varied in loci content from 53 (A08) to 175 (A03) loci.</p>                            <h3 scrollto-destination=4624743 id="4624743" class="section-title" data-legacy-id=s3c>3.3. Comparative mapping between the <em>Arachis</em> spp. and the four comparison legumes, <em>C. cajan, G. max, L. japonicus</em>, and <em>M. truncatula</em></h3>
<p>Corresponding sequences were available for 3473 of the 3693 mapped loci (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Of these 3473 sequences, 869 showed significant similarity to one of the four legume genome sequences (<em>C. cajan</em>, <em>G. max</em>, <em>L. japonicus</em>, and <em>M. truncatula</em>). For each legume species, 515, 781, 436, and 419 sequences showed similarities to sequences in the <em>C. cajan</em>, <em>G. max</em>, <em>L. japonicus</em>, and <em>M. truncatula</em> genomes, respectively, and 170 sequences showed similarity to sequences in all four legume genomes.</p><p>Segmental syntenic regions were observed between the <em>Arachis</em> HGs and the genomes of the four comparison legumes (Fig. <a reveal-id="DSS042F3" data-open="DSS042F3" class="link link-reveal link-table xref-fig">3</a>, Table <a reveal-id="DSS042TB3" data-open="DSS042TB3" class="link link-reveal link-table xref-fig">3</a>, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Most of the LG pairs in each HG showed similar synteny relationships to those in the four comparison legumes. Between the four comparison legume species, the genome of <em>G. max</em> showed the densest synteny of LGs, corresponding to those of <em>Arachis</em>. In addition, most of the LGs in <em>Arachis</em> were identified as shared syntenies with LGs on each of two chromosomes of <em>G. max</em>. This might reflect the palaeopolyploid nature of the genomes of these species. Comparing <em>Arachis</em> with the other three legume genomes, clearly syntenic segments were observed as follows: <em>Arachis</em> HG01 shared synteny with <em>C. cajan</em> chromosome 11 (<em>Cc</em>11) and <em>M. truncatula</em> chromosome 4 (<em>Mt</em>04); <em>Arachis</em> HG04 shared synteny with <em>L. japonicus</em> chromosome 4 (<em>Lj</em>04); <em>Arachis</em> HG05 shared synteny with <em>Cc</em>06, <em>Lj</em>02, and <em>Mt</em>05; <em>Arachis</em> HG06 shared synteny with <em>Lj</em>01 and <em>Mt</em>07; <em>Arachis</em> HG07 shared synteny with <em>Cc</em>02, <em>Lj</em>02, and <em>Mt</em>05; <em>Arachis</em> HG08 shared synteny with <em>Cc</em>03; <em>Arachis</em> HG09 shared synteny with Cc02, <em>Lj</em>05, and <em>Mt</em>01; and <em>Arachis</em> HG10 shared synteny with <em>Lj</em>01. On <em>Arachis</em> HG03, segmental blocks were identified that showed shared synteny between the comparison legume genomes, while no obvious synteny was observed between <em>Arachis</em> HG02 and the genomes of <em>C. cajan</em>, <em>L. japonicus</em>, and <em>M. truncatula</em>. </p>                        <a id="4624746" scrollto-destination="4624746"></a>
<div content-id="DSS042TB3" class="table-modal"><div class="table-wrap"><div id="DSS042TB3" class="table-wrap-title" data-id="DSS042TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Similarity of other legume chromosomes with that of <em>Arachis</em> homoeologous groups</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><em>Arachis</em> </th><th><em>C. cajan</em> </th><th><em>G. max</em> </th><th><em>L. japonicus</em> </th><th><em>M. truncatula</em> </th></tr></thead><tbody><tr><td>HG01<sup>a</sup> </td><td>11 </td><td>5, 7, 13, 17 </td><td>4, 6 </td><td>4 </td></tr><tr><td>HG02 </td><td>— </td><td>3, 7 </td><td>— </td><td>— </td></tr><tr><td>HG03 </td><td>4, 5, 8, 9, 10 </td><td>4, 5, 6, 8, 9, 11, 12, 18 </td><td>3, 4 </td><td>4, 8 </td></tr><tr><td>HG04 </td><td>7, 8 </td><td>3, 7, 8, 9, 12, 13, 15, 18, 20 </td><td>6 </td><td>2, 7 </td></tr><tr><td>HG05 </td><td>6 </td><td>1, 9, 11, 16 </td><td>2 </td><td>5 </td></tr><tr><td>HG06 </td><td>2, 3, 7 </td><td>2, 3, 16, 19 </td><td>1 </td><td>7 </td></tr><tr><td>HG07 </td><td>2 </td><td>2, 6, 13 </td><td>2 </td><td>5 </td></tr><tr><td>HG08 </td><td>3 </td><td>4, 6, 8, 12, 13 </td><td>1, 3 </td><td>2, 3 </td></tr><tr><td>HG09 </td><td>2 </td><td>7, 10, 20 </td><td>5 </td><td>1 </td></tr><tr><td>HG10 </td><td>1, 8 </td><td>10, 15, 19 </td><td>1 </td><td>2, 3 </td></tr></tbody></table></div><div content-id="DSS042TB3" class="table-modal"><table><thead name="thead" align=""><tr><th><em>Arachis</em> </th><th><em>C. cajan</em> </th><th><em>G. max</em> </th><th><em>L. japonicus</em> </th><th><em>M. truncatula</em> </th></tr></thead><tbody><tr><td>HG01<sup>a</sup> </td><td>11 </td><td>5, 7, 13, 17 </td><td>4, 6 </td><td>4 </td></tr><tr><td>HG02 </td><td>— </td><td>3, 7 </td><td>— </td><td>— </td></tr><tr><td>HG03 </td><td>4, 5, 8, 9, 10 </td><td>4, 5, 6, 8, 9, 11, 12, 18 </td><td>3, 4 </td><td>4, 8 </td></tr><tr><td>HG04 </td><td>7, 8 </td><td>3, 7, 8, 9, 12, 13, 15, 18, 20 </td><td>6 </td><td>2, 7 </td></tr><tr><td>HG05 </td><td>6 </td><td>1, 9, 11, 16 </td><td>2 </td><td>5 </td></tr><tr><td>HG06 </td><td>2, 3, 7 </td><td>2, 3, 16, 19 </td><td>1 </td><td>7 </td></tr><tr><td>HG07 </td><td>2 </td><td>2, 6, 13 </td><td>2 </td><td>5 </td></tr><tr><td>HG08 </td><td>3 </td><td>4, 6, 8, 12, 13 </td><td>1, 3 </td><td>2, 3 </td></tr><tr><td>HG09 </td><td>2 </td><td>7, 10, 20 </td><td>5 </td><td>1 </td></tr><tr><td>HG10 </td><td>1, 8 </td><td>10, 15, 19 </td><td>1 </td><td>2, 3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>HG indicates homologous group.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4624746" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS042TB3" class="table-wrap-title" data-id="DSS042TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Similarity of other legume chromosomes with that of <em>Arachis</em> homoeologous groups</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th><em>Arachis</em> </th><th><em>C. cajan</em> </th><th><em>G. max</em> </th><th><em>L. japonicus</em> </th><th><em>M. truncatula</em> </th></tr></thead><tbody><tr><td>HG01<sup>a</sup> </td><td>11 </td><td>5, 7, 13, 17 </td><td>4, 6 </td><td>4 </td></tr><tr><td>HG02 </td><td>— </td><td>3, 7 </td><td>— </td><td>— </td></tr><tr><td>HG03 </td><td>4, 5, 8, 9, 10 </td><td>4, 5, 6, 8, 9, 11, 12, 18 </td><td>3, 4 </td><td>4, 8 </td></tr><tr><td>HG04 </td><td>7, 8 </td><td>3, 7, 8, 9, 12, 13, 15, 18, 20 </td><td>6 </td><td>2, 7 </td></tr><tr><td>HG05 </td><td>6 </td><td>1, 9, 11, 16 </td><td>2 </td><td>5 </td></tr><tr><td>HG06 </td><td>2, 3, 7 </td><td>2, 3, 16, 19 </td><td>1 </td><td>7 </td></tr><tr><td>HG07 </td><td>2 </td><td>2, 6, 13 </td><td>2 </td><td>5 </td></tr><tr><td>HG08 </td><td>3 </td><td>4, 6, 8, 12, 13 </td><td>1, 3 </td><td>2, 3 </td></tr><tr><td>HG09 </td><td>2 </td><td>7, 10, 20 </td><td>5 </td><td>1 </td></tr><tr><td>HG10 </td><td>1, 8 </td><td>10, 15, 19 </td><td>1 </td><td>2, 3 </td></tr></tbody></table></div><div content-id="DSS042TB3" class="table-modal"><table><thead name="thead" align=""><tr><th><em>Arachis</em> </th><th><em>C. cajan</em> </th><th><em>G. max</em> </th><th><em>L. japonicus</em> </th><th><em>M. truncatula</em> </th></tr></thead><tbody><tr><td>HG01<sup>a</sup> </td><td>11 </td><td>5, 7, 13, 17 </td><td>4, 6 </td><td>4 </td></tr><tr><td>HG02 </td><td>— </td><td>3, 7 </td><td>— </td><td>— </td></tr><tr><td>HG03 </td><td>4, 5, 8, 9, 10 </td><td>4, 5, 6, 8, 9, 11, 12, 18 </td><td>3, 4 </td><td>4, 8 </td></tr><tr><td>HG04 </td><td>7, 8 </td><td>3, 7, 8, 9, 12, 13, 15, 18, 20 </td><td>6 </td><td>2, 7 </td></tr><tr><td>HG05 </td><td>6 </td><td>1, 9, 11, 16 </td><td>2 </td><td>5 </td></tr><tr><td>HG06 </td><td>2, 3, 7 </td><td>2, 3, 16, 19 </td><td>1 </td><td>7 </td></tr><tr><td>HG07 </td><td>2 </td><td>2, 6, 13 </td><td>2 </td><td>5 </td></tr><tr><td>HG08 </td><td>3 </td><td>4, 6, 8, 12, 13 </td><td>1, 3 </td><td>2, 3 </td></tr><tr><td>HG09 </td><td>2 </td><td>7, 10, 20 </td><td>5 </td><td>1 </td></tr><tr><td>HG10 </td><td>1, 8 </td><td>10, 15, 19 </td><td>1 </td><td>2, 3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>HG indicates homologous group.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/4624746" target="_blank">View Large</a></div></div>                        <a id="4624747" scrollto-destination="4624747"></a>
<div id="DSS042F3" data-id="DSS042F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss042/2/m_dss04203.jpeg?Expires=2147483647&Signature=fPcRUMIO5qSdKOjnKdavdKWnah2JlbS3QwiIfUzYSn3PsXzroA3waWLhn5amJZodevw-xuv2AC8VkMAGsdP-KfMhW4moJWtbdMee7WwsNfUDMC4RuqKR8oBjrACB2ovngH4EvJgIH-eoTdlVuT0YW4DL2cNJa~MfNusx~Q3lDLu2dc6ZglDHrD16xqgPPNzakmmbOK5Wq84nM5BPBMThY9wZPk~9tU81hNrKJ~8gCg-CsS3NgFi8G-aIij-d9oJRq175-A8BkgHm1z9YhiHYWYSs85pgOShQOfDu-hiRC7YPdye-CoToZSy~EzE9D4vsx-xQqGTpgLOz2dLLv-VzXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative maps of Arachis and four other legumes, C. cajan, G. max, L. japonicus, and M. truncatula. Circled bars in brown, magenta, green, yellow, and blue indicate the genomes of Arachis, C. cajan (A), G. max (B), L. japonicus (C), and M. truncatula (D), respectively. Homologous loci are connected by lines." path-from-xml="dss04203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4624747/dss04203.jpeg" path-from-xml="dss04203.jpeg" target="_blank">View large</a><a section="4624747" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss042/2/dss04203.jpeg?Expires=2147483647&Signature=Xjp~pswnExwO7OmzqSbIXClz9xKwsoMSySBSR76rFrL5gvAoBTvDXAlFXOEEfOCAukVG51OT5-ElpPciR1LuJhP1B2dy01YzLa7yyHxS8Gh8XfCRDo~Ms7aEJSN9Ou9NUpKycOAs7O87gxJFBKbJ0Kf6IoAswluoKqGfE0altPHI1i0FLu4aWsly8cP0xzsweZNhrnfkE74i2kKp41Sh6BEgp~nfLe~LubuaWRKm32xI0ofZgzfDmlLM47jXYWHfA58mbgf3usIi8VyITzazmGi6jw6kmzcvAaCX0p6~hsrOchsDIyeeZwMyx~VosbSJcSObUpJ~w4OkSxp7Ufr-4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4624747&ar=333781&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss04203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative maps of <em>Arachis</em> and four other legumes, <em>C. cajan</em>, <em>G. max</em>, <em>L. japonicus</em>, and <em>M. truncatula.</em> Circled bars in brown, magenta, green, yellow, and blue indicate the genomes of <em>Arachis</em>, <em>C. cajan</em> (A), <em>G. max</em> (B), <em>L. japonicus</em> (C), and <em>M. truncatula</em> (D), respectively. Homologous loci are connected by lines.</p></div></div><div content-id="DSS042F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss042/2/m_dss04203.jpeg?Expires=2147483647&Signature=fPcRUMIO5qSdKOjnKdavdKWnah2JlbS3QwiIfUzYSn3PsXzroA3waWLhn5amJZodevw-xuv2AC8VkMAGsdP-KfMhW4moJWtbdMee7WwsNfUDMC4RuqKR8oBjrACB2ovngH4EvJgIH-eoTdlVuT0YW4DL2cNJa~MfNusx~Q3lDLu2dc6ZglDHrD16xqgPPNzakmmbOK5Wq84nM5BPBMThY9wZPk~9tU81hNrKJ~8gCg-CsS3NgFi8G-aIij-d9oJRq175-A8BkgHm1z9YhiHYWYSs85pgOShQOfDu-hiRC7YPdye-CoToZSy~EzE9D4vsx-xQqGTpgLOz2dLLv-VzXg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparative maps of Arachis and four other legumes, C. cajan, G. max, L. japonicus, and M. truncatula. Circled bars in brown, magenta, green, yellow, and blue indicate the genomes of Arachis, C. cajan (A), G. max (B), L. japonicus (C), and M. truncatula (D), respectively. Homologous loci are connected by lines." path-from-xml="dss04203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/4624747/dss04203.jpeg" path-from-xml="dss04203.jpeg" target="_blank">View large</a><a section="4624747" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss042/2/dss04203.jpeg?Expires=2147483647&Signature=Xjp~pswnExwO7OmzqSbIXClz9xKwsoMSySBSR76rFrL5gvAoBTvDXAlFXOEEfOCAukVG51OT5-ElpPciR1LuJhP1B2dy01YzLa7yyHxS8Gh8XfCRDo~Ms7aEJSN9Ou9NUpKycOAs7O87gxJFBKbJ0Kf6IoAswluoKqGfE0altPHI1i0FLu4aWsly8cP0xzsweZNhrnfkE74i2kKp41Sh6BEgp~nfLe~LubuaWRKm32xI0ofZgzfDmlLM47jXYWHfA58mbgf3usIi8VyITzazmGi6jw6kmzcvAaCX0p6~hsrOchsDIyeeZwMyx~VosbSJcSObUpJ~w4OkSxp7Ufr-4g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=4624747&ar=333781&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss04203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparative maps of <em>Arachis</em> and four other legumes, <em>C. cajan</em>, <em>G. max</em>, <em>L. japonicus</em>, and <em>M. truncatula.</em> Circled bars in brown, magenta, green, yellow, and blue indicate the genomes of <em>Arachis</em>, <em>C. cajan</em> (A), <em>G. max</em> (B), <em>L. japonicus</em> (C), and <em>M. truncatula</em> (D), respectively. Homologous loci are connected by lines.</p></div></div>                            <h2 scrollto-destination=4624748 id="4624748" class="section-title" data-legacy-id=s4>4. Discussion</h2>
                            <h3 scrollto-destination=4624749 id="4624749" class="section-title" data-legacy-id=s4a>4.1. Polymorphism potential of the DNA markers mapped on the genetic linkage map</h3>
<p>The polymorphism potential of DNA markers in <em>Arachis</em> is usually evaluated on the basis of the polymorphism information content (PIC) value.<sup><a reveal-id="DSS042C34" data-open="DSS042C34" class="link link-ref link-reveal xref-bibr">34</a></sup> While the PIC values of the AHS were estimated from genotyping data from 16 <em>Arachis</em> lines,<sup><a reveal-id="DSS042C37" data-open="DSS042C37" class="link link-ref link-reveal xref-bibr">37</a></sup> those of the AHGS and AhTE have not been investigated thoroughly.<sup><a reveal-id="DSS042C36" data-open="DSS042C36" class="link link-ref link-reveal xref-bibr">36</a>,<a reveal-id="DSS042C38" data-open="DSS042C38" class="link link-ref link-reveal xref-bibr">38</a></sup> In the present study and in previous studies,<sup><a reveal-id="DSS042C36" data-open="DSS042C36" class="link link-ref link-reveal xref-bibr">36</a></sup> 968 (416 AHGS, 395 AHS, and 157 AhTE), 666 (392 AHGS, 65 AHS, and 209 AhTE), 245 (197 AHGS, 10 AHS, and 38 AhTE), 73 (72 AHGS and 1 AhTE), and 3 (AHGS only) markers were mapped on single, double, triple, quadruple, and quintuple maps, respectively. Of these, the PIC values of the 395, 65, and 10 AHS markers were estimated to be, on average, 0.23, 0.28, and 0.36, respectively, from the results of Koilkonda <em>et al.</em><sup><a reveal-id="DSS042C37" data-open="DSS042C37" class="link link-ref link-reveal xref-bibr">37</a></sup> A positive correlation was observed between the PIC values and the number of maps on which the AHS was mapped. Therefore, if this estimation can be generalized, the AHGS and AhTE markers mapped on multiple maps can be considered as potential sources for obtaining polymorphic markers in other mapping populations.</p><p>In the screening of polymorphic markers between the parental lines of the mapping populations, 582, 862, and 1144 markers were selected as polymorphism candidates, but 187, 238, and 89 were excluded from the mapping analysis. The excluded markers might amplify DNAs from not the target locus but non-allelic repetitive and putative paralogous sequences in the <em>Arachis</em> genome,<sup><a reveal-id="DSS042C38" data-open="DSS042C38" class="link link-ref link-reveal xref-bibr">38</a>,<a reveal-id="DSS042C45" data-open="DSS042C45" class="link link-ref link-reveal xref-bibr">45</a>,<a reveal-id="DSS042C46" data-open="DSS042C46" class="link link-ref link-reveal xref-bibr">46</a></sup> which are not suitable for segregation and linkage analysis.</p>                            <h3 scrollto-destination=4624752 id="4624752" class="section-title" data-legacy-id=s4b>4.2. Genetic linkage maps for three mapping populations</h3>
<p>Comparison of the AF5, BF6, and TF6 maps revealed that the marker order was almost perfectly conserved between the HGs, with the exception of HG04, HG07, and HG08 (Fig. <a reveal-id="DSS042F1" data-open="DSS042F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). This result indicated that the A and B genomes of <em>Arachis</em> evolved from a common ancestor species. Exceptions to this shared marker order were found within HG04, and between HG07 and HG08. The former was observed for the first time in this study, while the latter confirms previous observations.<sup><a reveal-id="DSS042C20" data-open="DSS042C20" class="link link-ref link-reveal xref-bibr">20</a>,<a reveal-id="DSS042C24" data-open="DSS042C24" class="link link-ref link-reveal xref-bibr">24</a></sup> These findings suggest that the chromosome translocations might have occurred at, or after, the divergence of the A and B genomes in <em>Arachis</em>.</p><p>The AF5 and BF6 linkage maps have been improved by subsequent mapping of the newly developed markers after they were originally published.<sup><a reveal-id="DSS042C19" data-open="DSS042C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSS042C20" data-open="DSS042C20" class="link link-ref link-reveal xref-bibr">20</a></sup> However, despite the increase in the number of mapped loci from 170 to 597 in the AF5 map and from 149 to 798 in the BF6 map, the total map length decreased from 1231 to 544 cM in the AF5 map and from 1294 to 461 cM in the BF6 map. In the maps generated in the present study, the marker loci that mapped to the ends of LGs in previous studies<sup><a reveal-id="DSS042C19" data-open="DSS042C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSS042C20" data-open="DSS042C20" class="link link-ref link-reveal xref-bibr">20</a></sup> were excluded, and the intervals between the markers were shorter than those in previous maps. Mapping telomere-based makers will clarify whether the maps from this study have been saturated or not.</p><p>In the linkage map for the TF6 population, the total length of the map was 1442 cM, which was shorter than that of the maps reported for <em>Arachis</em> tetraploids (2210 cM with 370 loci;<sup><a reveal-id="DSS042C23" data-open="DSS042C23" class="link link-ref link-reveal xref-bibr">23</a></sup> 2166 cM with 1114 loci;<sup><a reveal-id="DSS042C36" data-open="DSS042C36" class="link link-ref link-reveal xref-bibr">36</a></sup> 1844 cM with 298 loci;<sup><a reveal-id="DSS042C24" data-open="DSS042C24" class="link link-ref link-reveal xref-bibr">24</a></sup> 1785 cM with 191 loci<sup><a reveal-id="DSS042C29" data-open="DSS042C29" class="link link-ref link-reveal xref-bibr">29</a></sup>). Nagy <em>et al.</em><sup><a reveal-id="DSS042C21" data-open="DSS042C21" class="link link-ref link-reveal xref-bibr">21</a></sup> reported that the length of the genetic linkage map in <em>A. duranensis</em> ranged from 1081.1 to 2056.5 cM (depending on the mapping program used) when 1054 segregation loci were used for the linkage analysis. Therefore, the different programs, algorithms, and functions used for mapping may explain the differences in map length reported by the present and previous studies. Our preliminary analysis, in which MapMaker program<sup><a reveal-id="DSS042C47" data-open="DSS042C47" class="link link-ref link-reveal xref-bibr">47</a></sup> gave longer map lengths than the JoinMap, also supported these explanations. Alternatively, as Sim <em>et al.</em><sup><a reveal-id="DSS042C48" data-open="DSS042C48" class="link link-ref link-reveal xref-bibr">48</a></sup> suggested (based on linkage analysis in tomato), different ratios between the number of markers and population size in each population might have resulted in the length differences.</p><p>Segregation distortions were observed in the three maps, the AF5, BF6, and TF6, and the ratios were different among the populations and LGs (Table <a reveal-id="DSS042TB1" data-open="DSS042TB1" class="link link-reveal link-table xref-fig">1</a>). Similar observations were found in our previous studies on tomato on which high and low ratios of segregation distortion were observed in inter- and intraspecific maps, respectively.<sup><a reveal-id="DSS042C49" data-open="DSS042C49" class="link link-ref link-reveal xref-bibr">49</a>,<a reveal-id="DSS042C50" data-open="DSS042C50" class="link link-ref link-reveal xref-bibr">50</a></sup> Segregation distortion is generally caused by the chromosomal structural differences or the presence of transmission ration distorter factors on some chromosome.<sup><a reveal-id="DSS042C51" data-open="DSS042C51" class="link link-ref link-reveal xref-bibr">51</a></sup> Our findings will contribute to identify factors for the segregation distortion.</p>                            <h3 scrollto-destination=4624757 id="4624757" class="section-title" data-legacy-id=s4c>4.3. Characterization of the cultivated peanut genome using the <em>Arachis</em> consensus map</h3>
<p>The consensus map was constructed using segregation data from 16 populations, in which five species, <em>A. duranensis</em> (A), <em>A. stenosperma</em> (A), <em>A. ipaënsis</em> (B), <em>A. magna</em> (B), and <em>A. hypogaea</em> (AB), were involved (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). A total of 889 loci derived from 421 markers were commonly mapped onto all of the paired LGs as HGs; however, the locus orders were not always conserved. The regions in which mapped loci showed non-corresponding orders between the paired LGs in each HG were considered as candidate regions having A or B genome-specific structures. Most of the locus orders were conserved between AA (diploid) and A (tetraploid) LGs, or between BB (diploid) and B (tetraploid) LGs (Fig. <a reveal-id="DSS042F2" data-open="DSS042F2" class="link link-reveal link-table xref-fig">2</a>). However, several regions showed evidence of rearrangement, such as between BB03 and B03, BB09 and B09, and AA10 and A10. <em>Arachis duranensis</em> and <em>A. ipaënsis</em> are considered to be the most probable ancestors of <em>A. hypogaea</em>, because of similar karyotypes of <em>A. duranensis</em> and <em>A. ipaënsis</em> to that of <em>A. hypogaea</em>.<sup><a reveal-id="DSS042C12  DSS042C15" data-open="DSS042C12  DSS042C15" class="link link-ref link-reveal xref-bibr">12–15</a></sup> By contrast, even though the structures of A genome of different species are well conserved, the <em>A. stenosperma</em> genome is distinguishable from that of <em>A. duranensis</em> on the basis of the variability observed in the heterochromatin and 18S–26S rRNA loci.<sup><a reveal-id="DSS042C13" data-open="DSS042C13" class="link link-ref link-reveal xref-bibr">13</a></sup> It is considered the possibility that such differences in the genome structure of the parental lines of the mapping populations might disrupt the marker order between diploid and tetraploid consensus maps.</p><p>The average marker density of the consensus map was 0.7 cM/locus (Table <a reveal-id="DSS042TB2" data-open="DSS042TB2" class="link link-reveal link-table xref-fig">2</a>). Because the genome size of <em>A. hypogaea</em> is estimated to be ∼2.8 Gb,<sup><a reveal-id="DSS042C16" data-open="DSS042C16" class="link link-ref link-reveal xref-bibr">16</a></sup> the DNA markers were located at &lt;760 kb intervals on average. In other words, a map-based cloning strategy has become a realistic approach for molecular genetics in <em>Arachis</em> spp., because a candidate region for the location of a target gene can, on average, be identified to within 760 kb. Furthermore, this high-density map should aid in genome sequencing analysis in <em>Arachis</em> spp., because the locus order can work as a reference for the orientation of sequence contigs and scaffolds. Moreover, the marker clusters were observed in all of the LGs (Fig. <a reveal-id="DSS042F2" data-open="DSS042F2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Because chromosomal crossover, and the resulting genetic recombination, are severely suppressed in heterochromatin,<sup><a reveal-id="DSS042C6" data-open="DSS042C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS042C49" data-open="DSS042C49" class="link link-ref link-reveal xref-bibr">49</a></sup> the clustered regions were speculated to be located in heterochromatin, which makes up 10–20% of the <em>Arachis</em> chromosomes in somatic metaphase.<sup><a reveal-id="DSS042C13" data-open="DSS042C13" class="link link-ref link-reveal xref-bibr">13</a>,<a reveal-id="DSS042C15" data-open="DSS042C15" class="link link-ref link-reveal xref-bibr">15</a></sup> In the case of soybean, broad pericentromeric regions show markedly diminished recombination (4.2 Mb/cM), while the euchromatic regions at the chromosome ends maintain consistent levels of recombination (193 kb/cM).<sup><a reveal-id="DSS042C6" data-open="DSS042C6" class="link link-ref link-reveal xref-bibr">6</a></sup> This pattern is also observed in tomato, with 6042 and 172 kb/cM in heterochromatic and euchromatic regions, respectively.<sup><a reveal-id="DSS042C49" data-open="DSS042C49" class="link link-ref link-reveal xref-bibr">49</a></sup> As with soybean and tomato, it was predicted that the physical marker-to-marker distance intervals in the peanut genome were different between heterochromatic and euchromatic regions.</p>                            <h3 scrollto-destination=4624760 id="4624760" class="section-title" data-legacy-id=s4d>4.4. Comparative genomics of <em>Arachis</em> spp. with <em>C. cajan, G. max, L. japonicus, and M. truncatula</em></h3>
<p>All of the <em>Arachis</em> HGs showed similarities to the genomes of the four legumes (Fig. <a reveal-id="DSS042F3" data-open="DSS042F3" class="link link-reveal link-table xref-fig">3</a>, Table <a reveal-id="DSS042TB3" data-open="DSS042TB3" class="link link-reveal link-table xref-fig">3</a>, and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The genetic composition of <em>Arachis</em> corresponded to that of the <em>C. cajan, L. japonicus, and M. truncatula</em> genomes on a one-on-one basis, but was doubled when compared with the <em>G. max</em> genome. These observations support the estimation that a whole-genome duplication occurred in legumes ∼58 million years ago (mya), at the time of the Papilionoid origin,<sup><a reveal-id="DSS042C52" data-open="DSS042C52" class="link link-ref link-reveal xref-bibr">52</a></sup> whereas the soybean-specific duplication is estimated to have occurred &lt;13 mya.<sup><a reveal-id="DSS042C6" data-open="DSS042C6" class="link link-ref link-reveal xref-bibr">6</a></sup> These insights are based on a comparison of the A genome species between <em>Arachis</em> and <em>L. japonicus</em> and <em>M. truncatula</em>.<sup><a reveal-id="DSS042C39" data-open="DSS042C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p><p>At the chromosome level, each chromosome of <em>Arachis,</em> which belongs to the Dalbergioids, might have evolved differently and show different rearrangements from those in the ancestral chromosomes of the legumes after isolation from the other subfamilies, the Phaseoloids and Galegoids, ∼55 mya.<sup><a reveal-id="DSS042C52" data-open="DSS042C52" class="link link-ref link-reveal xref-bibr">52</a></sup> Of the 10 <em>Arachis</em> HGs, HG01, 05, 07 and 09 showed microsyntenies with the chromosomes of <em>C. cajan</em>, <em>L. japonicus</em>, and <em>M. truncatula</em>. This suggested the possibility that the major chromosome rearrangements did not occur after the time of divergence of subfamilies in the Papilionoideae. By contrast, HG02 showed the most disparate syntenies with the genomes of <em>C. cajan</em>, <em>G. max</em>, <em>L. japonicus</em>, and <em>M. truncatula</em> (Table <a reveal-id="DSS042TB3" data-open="DSS042TB3" class="link link-reveal link-table xref-fig">3</a>), which suggested frequent chromosome rearrangement.</p>                            <h2 scrollto-destination=4624763 id="4624763" class="section-title" data-legacy-id=s5>5. Conclusion</h2>
<p>In this study, we developed three linkage maps for <em>Arachis</em> spp., and anchored the LGs to the A and B genomes. Integration of the tetraploid maps developed in this study with the 13 previously published studies generated a high-density consensus map of tetraploid <em>Arachis</em>. The developed maps identified structural features within the <em>Arachis</em> genome using comparative genomic analysis<em>,</em> and also identified differences between the <em>Arachis</em> genome and that of other legumes, <em>C. cajan</em>, <em>G. max</em>, <em>L. japonicus</em>, and <em>M. truncatula</em>. The results obtained in this study will bridge the gaps in our knowledge regarding the genomes of <em>Arachis</em> and other legumes, and will further the genetic/genomic study and molecular breeding of <em>Arachis</em>.</p>                            <h3 scrollto-destination=4624765 id="4624765" class="section-title" data-legacy-id=s5a>5.1. Availability</h3>
<p>Information for the genetic linkage maps and the integrated consensus map is available at <a class="link link-uri" href="http://marker.kazusa.or.jp">http://marker.kazusa.or.jp</a>.</p>                            <h2 scrollto-destination=4624767 id="4624767" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the Kazusa DNA Research Institute Foundation.</p>                            <h2 scrollto-destination=4624769 id="4624769" class="section-title" data-legacy-id=s7>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data</a> are available at <a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=4624771 id="4624771" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We are grateful to S. Sasamoto, S. Nakayama, A. Watanabe, T. Fujishiro, M. Kato, Y. Kishida, M. Kohara, C. Minami, K. Nanri, C. Takahashi, and H. Tsuruoka in the Kazusa DNA Research Institute for their technical assistance.</p>                            <h2 scrollto-destination=4624773 id="4624773" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS042C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Doyle</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Luckow</div> <div class="given-names">M.A.</div></div>. </div><div class="article-title">The rest of the iceberg. Legume diversity and evolution in a phylogenetic context</div>, <div class="source">Plant Physiol.</div> , <div class="year">2003</div>, vol. <div class="volume">131</div> (pg. <div class="fpage">900</div>-<div class="lpage">10</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.102.018150" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.102.018150</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rest%20of%20the%20iceberg.%20Legume%20diversity%20and%20evolution%20in%20a%20phylogenetic%20context&amp;author=J.J.%20Doyle&amp;author=M.A.%20Luckow&amp;publication_year=2003&amp;journal=Plant%20Physiol.&amp;volume=131&amp;pages=900-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.102.018150" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.102.018150" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.102.018150"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12644643" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wojciechowski</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Lavin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sanderson</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">A phylogeny of legumes (Leguminosae) based on analysis of the plastid <em>matK</em> gene resolves many well-supported subclades within the family</div>, <div class="source">Am. J. Bot.</div> , <div class="year">2004</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">1846</div>-<div class="lpage">62</div>)<div class="pub-id-doi"><a href="https://doi.org/10.3732/ajb.91.11.1846" class="link link-pub-id-doi" target="_blank">https://doi.org/10.3732/ajb.91.11.1846</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20phylogeny%20of%20legumes%20(Leguminosae)%20based%20on%20analysis%20of%20the%20plastid%20matK%20gene%20resolves%20many%20well-supported%20subclades%20within%20the%20family&amp;author=M.F.%20Wojciechowski&amp;author=M.%20Lavin&amp;author=M.J.%20Sanderson&amp;publication_year=2004&amp;journal=Am.%20J.%20Bot.&amp;volume=91&amp;pages=1846-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.91.11.1846" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.91.11.1846" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.91.11.1846"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21652332" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Choi</div> <div class="given-names">H.K.</div></div>,  <div class="name"><div class="surname">Mun</div> <div class="given-names">J.H.</div></div>,  <div class="name"><div class="surname">Kim</div> <div class="given-names">D.J.</div></div>, et al. </div><div class="article-title">Estimating genome conservation between crop and model legume species</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">15289</div>-<div class="lpage">94</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0402251101" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0402251101</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Estimating%20genome%20conservation%20between%20crop%20and%20model%20legume%20species&amp;author=H.K.%20Choi&amp;author=J.H.%20Mun&amp;author=D.J.%20Kim&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=15289-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0402251101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0402251101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0402251101"> </span></div> </div></div></div></div></div><div content-id="DSS042C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Isobe</div> <div class="given-names">S.N.</div></div>,  <div class="name"><div class="surname">Hisano</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Comparative genetic mapping and discovery of linkage disequilibrium across linkage groups in white clover (<em>Trifolium repens</em> L.)</div>, <div class="source">G3</div> , <div class="year">2012</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">607</div>-<div class="lpage">17</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1534/g3.112.002600" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1534/g3.112.002600</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genetic%20mapping%20and%20discovery%20of%20linkage%20disequilibrium%20across%20linkage%20groups%20in%20white%20clover%20(Trifolium%20repens%20L.)&amp;author=S.N.%20Isobe&amp;author=H.%20Hisano&amp;author=S.%20Sato&amp;publication_year=2012&amp;journal=G3&amp;volume=2&amp;pages=607-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/g3.112.002600" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fg3.112.002600" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fg3.112.002600"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22670230" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kaneko</div> <div class="given-names">T.</div></div>, et al. </div><div class="article-title">Genome structure of the legume, <em>Lotus japonicus</em></div>, <div class="source">DNA Res.</div> , <div class="year">2008</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">227</div>-<div class="lpage">39</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsn008" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsn008</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20structure%20of%20the%20legume%2C%20Lotus%20japonicus&amp;author=S.%20Sato&amp;author=Y.%20Nakamura&amp;author=T.%20Kaneko&amp;publication_year=2008&amp;journal=DNA%20Res.&amp;volume=15&amp;pages=227-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18511435" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schmutz</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Cannon</div> <div class="given-names">S.B.</div></div>,  <div class="name"><div class="surname">Schlueter</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Genome sequence of the palaeopolyploid soybean</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">463</div> (pg. <div class="fpage">178</div>-<div class="lpage">83</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature08670" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature08670</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequence%20of%20the%20palaeopolyploid%20soybean&amp;author=J.%20Schmutz&amp;author=S.B.%20Cannon&amp;author=J.%20Schlueter&amp;publication_year=2010&amp;journal=Nature&amp;volume=463&amp;pages=178-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20075913" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Young</div> <div class="given-names">N.D.</div></div>,  <div class="name"><div class="surname">Debellé</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Oldroyd</div> <div class="given-names">G.E.</div></div>, et al. </div><div class="article-title">The <em>Medicago</em> genome provides insight into the evolution of rhizobial symbioses</div>, <div class="source">Nature</div> , <div class="year">2011</div>, vol. <div class="volume">480</div> (pg. <div class="fpage">520</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Medicago%20genome%20provides%20insight%20into%20the%20evolution%20of%20rhizobial%20symbioses&amp;author=N.D.%20Young&amp;author=F.%20Debell%C3%A9&amp;author=G.E.%20Oldroyd&amp;publication_year=2011&amp;journal=Nature&amp;volume=480&amp;pages=520-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/480162a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F480162a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F480162a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22089132" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Draft genome sequence of pigeonpea (<em>Cajanus cajan</em>), an orphan legume crop of resource-poor farmers</div>, <div class="source">Nat. Biotechnol.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">83</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nbt.2022" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nbt.2022</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20sequence%20of%20pigeonpea%20(Cajanus%20cajan)%2C%20an%20orphan%20legume%20crop%20of%20resource-poor%20farmers&amp;author=R.K.%20Varshney&amp;author=W.%20Chen&amp;author=Y.%20Li&amp;publication_year=2012&amp;journal=Nat.%20Biotechnol.&amp;volume=30&amp;pages=83-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.2022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.2022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.2022"> </span></div> </div></div></div></div></div><div content-id="DSS042C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Choi</div> <div class="given-names">H.K.</div></div>,  <div class="name"><div class="surname">Cook</div> <div class="given-names">D.R.</div></div>,  <div class="name"><div class="surname">Shoemaker</div> <div class="given-names">R.C.</div></div>. </div><div class="article-title">Bridging model and crop legumes through comparative genomics</div>, <div class="source">Plant Physiol.</div> , <div class="year">2005</div>, vol. <div class="volume">137</div> (pg. <div class="fpage">1189</div>-<div class="lpage">96</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.104.058891" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.104.058891</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bridging%20model%20and%20crop%20legumes%20through%20comparative%20genomics&amp;author=H.%20Zhu&amp;author=H.K.%20Choi&amp;author=D.R.%20Cook&amp;author=R.C.%20Shoemaker&amp;publication_year=2005&amp;journal=Plant%20Physiol.&amp;volume=137&amp;pages=1189-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.058891" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.058891" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.058891"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15824281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krapovickas</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gregory</div> <div class="given-names">W.</div></div>. </div><div class="article-title">Taxonomia del genero <em>Arachis</em> (Leguminosae)</div>, <div class="source">Bonplandia</div> , <div class="year">1994</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">1</div>-<div class="lpage">186</div>)</div></div></div></div><div content-id="DSS042C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Stalker</div> <div class="given-names">H.T.</div></div>. </div><div class="article-title">A new species in section <em>Arachis</em> of peanuts with a D genome</div>, <div class="source">Am. J. Bot.</div> , <div class="year">1991</div>, vol. <div class="volume">78</div> (pg. <div class="fpage">630</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2307/2445084" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2307/2445084</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20species%20in%20section%20Arachis%20of%20peanuts%20with%20a%20D%20genome&amp;author=H.T.%20Stalker&amp;publication_year=1991&amp;journal=Am.%20J.%20Bot.&amp;volume=78&amp;pages=630-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2445084" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2445084" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2445084"> </span></div> </div></div></div></div></div><div content-id="DSS042C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Robledo</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Seijo</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Species relationships among the wild B genome of <em>Arachis</em> species (section <em>Arachis</em>) based on FISH mapping of rDNA loci and heterochromatin detection: a new proposal for genome arrangement</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">121</div> (pg. <div class="fpage">1033</div>-<div class="lpage">46</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-010-1369-7" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-010-1369-7</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Species%20relationships%20among%20the%20wild%20B%20genome%20of%20Arachis%20species%20(section%20Arachis)%20based%20on%20FISH%20mapping%20of%20rDNA%20loci%20and%20heterochromatin%20detection%3A%20a%20new%20proposal%20for%20genome%20arrangement&amp;author=G.%20Robledo&amp;author=G.%20Seijo&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=121&amp;pages=1033-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1369-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1369-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1369-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20552326" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Robledo</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lavia</div> <div class="given-names">G.I.</div></div>,  <div class="name"><div class="surname">Seijo</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Species relations among wild <em>Arachis</em> species with the A genome as revealed by FISH mapping of rDNA loci and heterochromatin detection</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">118</div> (pg. <div class="fpage">1295</div>-<div class="lpage">307</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-009-0981-x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-009-0981-x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Species%20relations%20among%20wild%20Arachis%20species%20with%20the%20A%20genome%20as%20revealed%20by%20FISH%20mapping%20of%20rDNA%20loci%20and%20heterochromatin%20detection&amp;author=G.%20Robledo&amp;author=G.I.%20Lavia&amp;author=G.%20Seijo&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=118&amp;pages=1295-307" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-009-0981-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-009-0981-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-009-0981-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19234686" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Seijo</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lavia</div> <div class="given-names">G.I.</div></div>,  <div class="name"><div class="surname">Fernandez</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Genomic relationships between the cultivated peanut (<em>Arachis hypogaea</em>, Leguminosae) and its close relatives revealed by double GISH</div>, <div class="source">Am. J. Bot.</div> , <div class="year">2007</div>, vol. <div class="volume">94</div> (pg. <div class="fpage">1963</div>-<div class="lpage">71</div>)<div class="pub-id-doi"><a href="https://doi.org/10.3732/ajb.94.12.1963" class="link link-pub-id-doi" target="_blank">https://doi.org/10.3732/ajb.94.12.1963</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20relationships%20between%20the%20cultivated%20peanut%20(Arachis%20hypogaea%2C%20Leguminosae)%20and%20its%20close%20relatives%20revealed%20by%20double%20GISH&amp;author=G.%20Seijo&amp;author=G.I.%20Lavia&amp;author=A.%20Fernandez&amp;publication_year=2007&amp;journal=Am.%20J.%20Bot.&amp;volume=94&amp;pages=1963-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.94.12.1963" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.94.12.1963" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.94.12.1963"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21636391" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Seijo</div> <div class="given-names">J.G.</div></div>,  <div class="name"><div class="surname">Lavia</div> <div class="given-names">G.I.</div></div>,  <div class="name"><div class="surname">Fernandez</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Krapovickas</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Ducasse</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Moscone</div> <div class="given-names">E.A.</div></div>. </div><div class="article-title">Physical mapping of the 5S and 18S-25S rRNA genes by FISH as evidence that <em>Arachis duranensis</em> and <em>A. ipaënsis</em> are the wild diploid progenitors of <em>A. hypogaea</em> (Leguminosae)</div>, <div class="source">Am. J. Bot.</div> , <div class="year">2004</div>, vol. <div class="volume">91</div> (pg. <div class="fpage">1294</div>-<div class="lpage">303</div>)<div class="pub-id-doi"><a href="https://doi.org/10.3732/ajb.91.9.1294" class="link link-pub-id-doi" target="_blank">https://doi.org/10.3732/ajb.91.9.1294</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Physical%20mapping%20of%20the%205S%20and%2018S-25S%20rRNA%20genes%20by%20FISH%20as%20evidence%20that%20Arachis%20duranensis%20and%20A.%20ipa%C3%ABnsis%20are%20the%20wild%20diploid%20progenitors%20of%20A.%20hypogaea%20(Leguminosae)&amp;author=J.G.%20Seijo&amp;author=G.I.%20Lavia&amp;author=A.%20Fernandez&amp;author=A.%20Krapovickas&amp;author=D.%20Ducasse&amp;author=E.A.%20Moscone&amp;publication_year=2004&amp;journal=Am.%20J.%20Bot.&amp;volume=91&amp;pages=1294-303" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3732/ajb.91.9.1294" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3732%2Fajb.91.9.1294" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3732%2Fajb.91.9.1294"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21652361" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Temsch</div> <div class="given-names">E.M.</div></div>,  <div class="name"><div class="surname">Greilhuber</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Genome size variation in <em>Arachis hypogaea</em> and <em>A. monticola</em> re-evaluated</div>, <div class="source">Genome</div> , <div class="year">2000</div>, vol. <div class="volume">43</div> (pg. <div class="fpage">449</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20size%20variation%20in%20Arachis%20hypogaea%20and%20A.%20monticola%20re-evaluated&amp;author=E.M.%20Temsch&amp;author=J.%20Greilhuber&amp;publication_year=2000&amp;journal=Genome&amp;volume=43&amp;pages=449-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1139/gen-43-3-449" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1139%2Fgen-43-3-449" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1139%2Fgen-43-3-449"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10902707" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Halward</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Stalker</div> <div class="given-names">H.T.</div></div>,  <div class="name"><div class="surname">Kochert</div> <div class="given-names">G.</div></div>. </div><div class="article-title">Development of an RFLP linkaeg map in diploid peanut species</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">1993</div>, vol. <div class="volume">87</div> (pg. <div class="fpage">379</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF01184927" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF01184927</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20an%20RFLP%20linkaeg%20map%20in%20diploid%20peanut%20species&amp;author=T.%20Halward&amp;author=H.T.%20Stalker&amp;author=G.%20Kochert&amp;publication_year=1993&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=87&amp;pages=379-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01184927" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01184927" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01184927"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24190266" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Leal-Bertioli</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Jose</div> <div class="given-names">A.C.</div></div>,  <div class="name"><div class="surname">Alves-Freitas</div> <div class="given-names">D.M.</div></div>, et al. </div><div class="article-title">Identification of candidate genome regions controlling disease resistance in <em>Arachis</em></div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> pg. <div class="fpage">112</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-9-112" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-9-112</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20candidate%20genome%20regions%20controlling%20disease%20resistance%20in%20Arachis&amp;author=S.C.%20Leal-Bertioli&amp;author=A.C.%20Jose&amp;author=D.M.%20Alves-Freitas&amp;publication_year=2009&amp;journal=BMC%20Plant%20Biol.&amp;volume=9&amp;pages=112" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-9-112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-9-112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-9-112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19698131" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moretzsohn</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Leoi</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Proite</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">A microsatellite-based, gene-rich linkage map for the AA genome of <em>Arachis</em> (Fabaceae)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">111</div> (pg. <div class="fpage">1060</div>-<div class="lpage">71</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-005-0028-x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-005-0028-x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20microsatellite-based%2C%20gene-rich%20linkage%20map%20for%20the%20AA%20genome%20of%20Arachis%20(Fabaceae)&amp;author=M.C.%20Moretzsohn&amp;author=L.%20Leoi&amp;author=K.%20Proite&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=111&amp;pages=1060-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-005-0028-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-005-0028-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-005-0028-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16088397" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moretzsohn</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Barbosa</div> <div class="given-names">A.V.</div></div>,  <div class="name"><div class="surname">Alves-Freitas</div> <div class="given-names">D.M.</div></div>, et al. </div><div class="article-title">A linkage map for the B-genome of <em>Arachis</em> (Fabaceae) and its synteny to the A-genome</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> pg. <div class="fpage">40</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-9-40" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-9-40</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20linkage%20map%20for%20the%20B-genome%20of%20Arachis%20(Fabaceae)%20and%20its%20synteny%20to%20the%20A-genome&amp;author=M.C.%20Moretzsohn&amp;author=A.V.%20Barbosa&amp;author=D.M.%20Alves-Freitas&amp;publication_year=2009&amp;journal=BMC%20Plant%20Biol.&amp;volume=9&amp;pages=40" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-9-40" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-9-40" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-9-40"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19351409" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nagy</div> <div class="given-names">E.D.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Tang</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">A high-density genetic map of <em>Arachis duranensis</em>, a diploid ancestor of cultivated peanut</div>, <div class="source">BMC Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">469</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-13-469" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-13-469</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-density%20genetic%20map%20of%20Arachis%20duranensis%2C%20a%20diploid%20ancestor%20of%20cultivated%20peanut&amp;author=E.D.%20Nagy&amp;author=Y.%20Guo&amp;author=S.%20Tang&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=469" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-469" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-469" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-469"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22967170" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Guo</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Khanal</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tang</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Comparative mapping in intraspecific populations uncovers a high degree of macrosynteny between A- and B-genome diploid species of peanut</div>, <div class="source">BMC Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">13</div> pg. <div class="fpage">608</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-13-608" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-13-608</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20mapping%20in%20intraspecific%20populations%20uncovers%20a%20high%20degree%20of%20macrosynteny%20between%20A-%20and%20B-genome%20diploid%20species%20of%20peanut&amp;author=Y.%20Guo&amp;author=S.%20Khanal&amp;author=S.%20Tang&amp;publication_year=2012&amp;journal=BMC%20Genomics&amp;volume=13&amp;pages=608" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-13-608" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-13-608" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-13-608"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23140574" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Burow</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Simpson</div> <div class="given-names">C.E.</div></div>,  <div class="name"><div class="surname">Starr</div> <div class="given-names">J.L.</div></div>,  <div class="name"><div class="surname">Paterson</div> <div class="given-names">A.H.</div></div>. </div><div class="article-title">Transmission genetics of chromatin from a synthetic amphidiploid to cultivated peanut (<em>Arachis hypogaea</em> L.): broadening the gene pool of a monophyletic polyploid species</div>, <div class="source">Genetics</div> , <div class="year">2001</div>, vol. <div class="volume">159</div> (pg. <div class="fpage">823</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transmission%20genetics%20of%20chromatin%20from%20a%20synthetic%20amphidiploid%20to%20cultivated%20peanut%20(Arachis%20hypogaea%20L.)%3A%20broadening%20the%20gene%20pool%20of%20a%20monophyletic%20polyploid%20species&amp;author=M.D.%20Burow&amp;author=C.E.%20Simpson&amp;author=J.L.%20Starr&amp;author=A.H.%20Paterson&amp;publication_year=2001&amp;journal=Genetics&amp;volume=159&amp;pages=823-37" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11606556" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Foncéka</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Hodo-Abalo</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Rivallan</div> <div class="given-names">R.</div></div>, et al. </div><div class="article-title">Genetic mapping of wild introgressions into cultivated peanut: a way toward enlarging the genetic basis of a recent allotetraploid</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> pg. <div class="fpage">103</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-9-103" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-9-103</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20mapping%20of%20wild%20introgressions%20into%20cultivated%20peanut%3A%20a%20way%20toward%20enlarging%20the%20genetic%20basis%20of%20a%20recent%20allotetraploid&amp;author=D.%20Fonc%C3%A9ka&amp;author=T.%20Hodo-Abalo&amp;author=R.%20Rivallan&amp;publication_year=2009&amp;journal=BMC%20Plant%20Biol.&amp;volume=9&amp;pages=103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-9-103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-9-103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-9-103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19650911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gautami</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Pandey</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Vadez</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">Quantitative trait locus analysis and construction of consensus genetic map for drought tolerance traits based on three recombinant inbred line populations in cultivated groundnut (<em>Arachis hypogaea</em> L.)</div>, <div class="source">Mol. Breed.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">773</div>-<div class="lpage">88</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11032-011-9661-z" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11032-011-9661-z</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20locus%20analysis%20and%20construction%20of%20consensus%20genetic%20map%20for%20drought%20tolerance%20traits%20based%20on%20three%20recombinant%20inbred%20line%20populations%20in%20cultivated%20groundnut%20(Arachis%20hypogaea%20L.)&amp;author=B.%20Gautami&amp;author=M.K.%20Pandey&amp;author=V.%20Vadez&amp;publication_year=2012&amp;journal=Mol.%20Breed.&amp;volume=30&amp;pages=773-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-011-9660-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-011-9660-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-011-9660-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22924018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hong</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Liang</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">A SSR-based composite genetic linkage map for the cultivated peanut (<em>Arachis hypogaea</em> L.) genome</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> pg. <div class="fpage">17</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-10-17" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-10-17</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20SSR-based%20composite%20genetic%20linkage%20map%20for%20the%20cultivated%20peanut%20(Arachis%20hypogaea%20L.)%20genome&amp;author=Y.%20Hong&amp;author=X.%20Chen&amp;author=X.%20Liang&amp;publication_year=2010&amp;journal=BMC%20Plant%20Biol.&amp;volume=10&amp;pages=17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-10-17" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-10-17" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-10-17"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20105299" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Khedikar</div> <div class="given-names">Y.P.</div></div>,  <div class="name"><div class="surname">Gowda</div> <div class="given-names">M.V.</div></div>,  <div class="name"><div class="surname">Sarvamangala</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Patgar</div> <div class="given-names">K.V.</div></div>,  <div class="name"><div class="surname">Upadhyaya</div> <div class="given-names">H.D.</div></div>,  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>. </div><div class="article-title">A QTL study on late leaf spot and rust revealed one major QTL for molecular breeding for rust resistance in groundnut (<em>Arachis hypogaea</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">121</div> (pg. <div class="fpage">971</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-010-1366-x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-010-1366-x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20QTL%20study%20on%20late%20leaf%20spot%20and%20rust%20revealed%20one%20major%20QTL%20for%20molecular%20breeding%20for%20rust%20resistance%20in%20groundnut%20(Arachis%20hypogaea%20L.)&amp;author=Y.P.%20Khedikar&amp;author=M.V.%20Gowda&amp;author=C.%20Sarvamangala&amp;author=K.V.%20Patgar&amp;author=H.D.%20Upadhyaya&amp;author=R.K.%20Varshney&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=121&amp;pages=971-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1366-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1366-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1366-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20526757" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qin</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">An integrated genetic linkage map of cultivated peanut (<em>Arachis hypogaea</em> L.) constructed from two RIL populations</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">124</div> (pg. <div class="fpage">653</div>-<div class="lpage">64</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-011-1737-y" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-011-1737-y</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20integrated%20genetic%20linkage%20map%20of%20cultivated%20peanut%20(Arachis%20hypogaea%20L.)%20constructed%20from%20two%20RIL%20populations&amp;author=H.%20Qin&amp;author=S.%20Feng&amp;author=C.%20Chen&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=124&amp;pages=653-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-011-1737-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-011-1737-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-011-1737-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22072100" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ravi</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Vadez</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Identification of several small main-effect QTLs and a large number of epistatic QTLs for drought tolerance related traits in groundnut (<em>Arachis hypogaea</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">122</div> (pg. <div class="fpage">1119</div>-<div class="lpage">32</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-010-1517-0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-010-1517-0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20several%20small%20main-effect%20QTLs%20and%20a%20large%20number%20of%20epistatic%20QTLs%20for%20drought%20tolerance%20related%20traits%20in%20groundnut%20(Arachis%20hypogaea%20L.)&amp;author=K.%20Ravi&amp;author=V.%20Vadez&amp;author=S.%20Isobe&amp;publication_year=2011&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=122&amp;pages=1119-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1517-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1517-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1517-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21191568" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sarvamangala</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Gowda</div> <div class="given-names">M.V.C.</div></div>,  <div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>. </div><div class="article-title">Identification of quantitative trait loci for protein content, oil content and oil quality for groundnut (<em>Arachis hypogaea</em> L.)</div>, <div class="source">Field Crops Res.</div> , <div class="year">2011</div>, vol. <div class="volume">122</div> (pg. <div class="fpage">49</div>-<div class="lpage">59</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.fcr.2011.02.010" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.fcr.2011.02.010</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20quantitative%20trait%20loci%20for%20protein%20content%2C%20oil%20content%20and%20oil%20quality%20for%20groundnut%20(Arachis%20hypogaea%20L.)&amp;author=C.%20Sarvamangala&amp;author=M.V.C.%20Gowda&amp;author=R.K.%20Varshney&amp;publication_year=2011&amp;journal=Field%20Crops%20Res.&amp;volume=122&amp;pages=49-59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fcr.2011.02.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fcr.2011.02.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fcr.2011.02.010"> </span></div> </div></div></div></div></div><div content-id="DSS042C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sujay</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">Gowda</div> <div class="given-names">M.V.</div></div>,  <div class="name"><div class="surname">Pandey</div> <div class="given-names">M.K.</div></div>, et al. </div><div class="article-title">Quantitative trait locus analysis and construction of consensus genetic map for foliar disease resistance based on two recombinant inbred line populations in cultivated groundnut (<em>Arachis hypogaea</em> L.)</div>, <div class="source">Mol. Breed.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">773</div>-<div class="lpage">88</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11032-011-9661-z" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11032-011-9661-z</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20trait%20locus%20analysis%20and%20construction%20of%20consensus%20genetic%20map%20for%20foliar%20disease%20resistance%20based%20on%20two%20recombinant%20inbred%20line%20populations%20in%20cultivated%20groundnut%20(Arachis%20hypogaea%20L.)&amp;author=V.%20Sujay&amp;author=M.V.%20Gowda&amp;author=M.K.%20Pandey&amp;publication_year=2012&amp;journal=Mol.%20Breed.&amp;volume=30&amp;pages=773-88" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-011-9661-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-011-9661-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-011-9661-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22924018" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Varshney</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Bertioli</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Moretzsohn</div> <div class="given-names">M.C.</div></div>, et al. </div><div class="article-title">The first SSR-based genetic linkage map for cultivated groundnut (<em>Arachis hypogaea</em> L.)</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2009</div>, vol. <div class="volume">118</div> (pg. <div class="fpage">729</div>-<div class="lpage">39</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-008-0933-x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-008-0933-x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20first%20SSR-based%20genetic%20linkage%20map%20for%20cultivated%20groundnut%20(Arachis%20hypogaea%20L.)&amp;author=R.K.%20Varshney&amp;author=D.J.%20Bertioli&amp;author=M.C.%20Moretzsohn&amp;publication_year=2009&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=118&amp;pages=729-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-008-0933-x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-008-0933-x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-008-0933-x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19048225" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Penmetsa</div> <div class="given-names">R.V.</div></div>,  <div class="name"><div class="surname">Yuan</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Development and characterization of BAC-end sequence derived SSRs, and their incorporation into a new higher density genetic map for cultivated peanut (<em>Arachis hypogaea</em> L.)</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2012</div>, vol. <div class="volume">12</div> pg. <div class="fpage">10</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-12-10" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-12-10</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20and%20characterization%20of%20BAC-end%20sequence%20derived%20SSRs%2C%20and%20their%20incorporation%20into%20a%20new%20higher%20density%20genetic%20map%20for%20cultivated%20peanut%20(Arachis%20hypogaea%20L.)&amp;author=H.%20Wang&amp;author=R.V.%20Penmetsa&amp;author=M.%20Yuan&amp;publication_year=2012&amp;journal=BMC%20Plant%20Biol.&amp;volume=12&amp;pages=10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-12-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-12-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-12-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22260238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pandey</div> <div class="given-names">M.K.</div></div>,  <div class="name"><div class="surname">Monyo</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Ozias-Akins</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Advances in <em>Arachis</em> genomics for peanut improvement</div>, <div class="source">Biotechnol. Adv.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">639</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.biotechadv.2011.11.001" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.biotechadv.2011.11.001</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20Arachis%20genomics%20for%20peanut%20improvement&amp;author=M.K.%20Pandey&amp;author=E.%20Monyo&amp;author=P.%20Ozias-Akins&amp;publication_year=2012&amp;journal=Biotechnol.%20Adv.&amp;volume=30&amp;pages=639-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biotechadv.2011.11.001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biotechadv.2011.11.001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biotechadv.2011.11.001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22094114" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gautami</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Fonceka</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Pandey</div> <div class="given-names">M.K.</div></div>, et al. </div><div class="article-title">An international reference consensus genetic map with 897 marker loci based on 11 mapping populations for tetraploid groundnut (<em>Arachis hypogaea</em> L.)</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e41213</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0041213" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0041213</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20international%20reference%20consensus%20genetic%20map%20with%20897%20marker%20loci%20based%20on%2011%20mapping%20populations%20for%20tetraploid%20groundnut%20(Arachis%20hypogaea%20L.)&amp;author=B.%20Gautami&amp;author=D.%20Fonceka&amp;author=M.K.%20Pandey&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e41213" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0041213" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0041213" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0041213"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22815973" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Koilkonda</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Aoki</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title"><em>In silico</em> polymorphism analysis for the development of simple sequence repeat and transposon markers and construction of linkage map in cultivated peanut</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2012</div>, vol. <div class="volume">12</div> pg. <div class="fpage">80</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-12-80" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-12-80</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=In%20silico%20polymorphism%20analysis%20for%20the%20development%20of%20simple%20sequence%20repeat%20and%20transposon%20markers%20and%20construction%20of%20linkage%20map%20in%20cultivated%20peanut&amp;author=K.%20Shirasawa&amp;author=P.%20Koilkonda&amp;author=K.%20Aoki&amp;publication_year=2012&amp;journal=BMC%20Plant%20Biol.&amp;volume=12&amp;pages=80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-12-80" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-12-80" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-12-80"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22672714" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Koilkonda</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Sato</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Large-scale development of expressed sequence tag-derived simple sequence repeat markers and diversity analysis in <em>Arachis</em> spp.</div>, <div class="source">Mol. Breed.</div> , <div class="year">2012</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">125</div>-<div class="lpage">38</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11032-011-9604-8" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11032-011-9604-8</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20development%20of%20expressed%20sequence%20tag-derived%20simple%20sequence%20repeat%20markers%20and%20diversity%20analysis%20in%20Arachis%20spp.&amp;author=P.%20Koilkonda&amp;author=S.%20Sato&amp;author=S.%20Tabata&amp;publication_year=2012&amp;journal=Mol.%20Breed.&amp;volume=30&amp;pages=125-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11032-011-9604-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11032-011-9604-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11032-011-9604-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22707912" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Tabata</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Characterization of active miniature inverted-repeat transposable elements in the peanut genome</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2012</div>, vol. <div class="volume">124</div> (pg. <div class="fpage">1429</div>-<div class="lpage">38</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-012-1798-6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-012-1798-6</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20active%20miniature%20inverted-repeat%20transposable%20elements%20in%20the%20peanut%20genome&amp;author=K.%20Shirasawa&amp;author=H.%20Hirakawa&amp;author=S.%20Tabata&amp;publication_year=2012&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=124&amp;pages=1429-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-012-1798-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-012-1798-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-012-1798-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22294450" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bertioli</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Moretzsohn</div> <div class="given-names">M.C.</div></div>,  <div class="name"><div class="surname">Madsen</div> <div class="given-names">L.H.</div></div>, et al. </div><div class="article-title">An analysis of synteny of <em>Arachis</em> with <em>Lotus</em> and <em>Medicago</em> sheds new light on the structure, stability and evolution of legume genomes</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">45</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-10-45" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-10-45</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20analysis%20of%20synteny%20of%20Arachis%20with%20Lotus%20and%20Medicago%20sheds%20new%20light%20on%20the%20structure%2C%20stability%20and%20evolution%20of%20legume%20genomes&amp;author=D.J.%20Bertioli&amp;author=M.C.%20Moretzsohn&amp;author=L.H.%20Madsen&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-45" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-45" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-45"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19166586" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grattapaglia</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Sederoff</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Genetic linkage maps of <em>Eucalyptus grandis</em> and <em>Eucalyptus urophylla</em> using a pseudo-testcross: mapping strategy and RAPD markers</div>, <div class="source">Genetics</div> , <div class="year">1994</div>, vol. <div class="volume">137</div> (pg. <div class="fpage">1121</div>-<div class="lpage">37</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20maps%20of%20Eucalyptus%20grandis%20and%20Eucalyptus%20urophylla%20using%20a%20pseudo-testcross%3A%20mapping%20strategy%20and%20RAPD%20markers&amp;author=D.%20Grattapaglia&amp;author=R.%20Sederoff&amp;publication_year=1994&amp;journal=Genetics&amp;volume=137&amp;pages=1121-37" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7982566" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Ooijen</div> <div class="given-names">J.W.</div></div>. </div>, <div class="source">JoinMap<sup>®</sup>4, Software for the calculation of genetic linkage maps in experimental populations</div> , <div class="year">2006</div><div class="publisher-loc">Wageningen, The Netherlands</div><div class="publisher-name">Kyazma. BV</div></div></div></div></div><div content-id="DSS042C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Voorrips</div> <div class="given-names">R.E.</div></div>. </div><div class="article-title">MapChart: software for the graphical presentation of linkage maps and QTLs</div>, <div class="source">J. Hered.</div> , <div class="year">2002</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">77</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/jhered/93.1.77" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/jhered/93.1.77</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapChart%3A%20software%20for%20the%20graphical%20presentation%20of%20linkage%20maps%20and%20QTLs&amp;author=R.E.%20Voorrips&amp;publication_year=2002&amp;journal=J.%20Hered.&amp;volume=93&amp;pages=77-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jhered/93.1.77" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjhered%2F93.1.77" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjhered%2F93.1.77"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12011185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/25.17.3389" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/25.17.3389</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Krzywinski</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Schein</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Birol</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">Circos: an information aesthetic for comparative genomics</div>, <div class="source">Genome Res.</div> , <div class="year">2009</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">1639</div>-<div class="lpage">45</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.092759.109" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.092759.109</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Circos%3A%20an%20information%20aesthetic%20for%20comparative%20genomics&amp;author=M.%20Krzywinski&amp;author=J.%20Schein&amp;author=I.%20Birol&amp;publication_year=2009&amp;journal=Genome%20Res.&amp;volume=19&amp;pages=1639-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.092759.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.092759.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.092759.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19541911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nielen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Campos-Fonseca</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Leal-Bertioli</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">FIDEL-a retrovirus-like retrotransposon and its distinct evolutionary histories in the A- and B-genome components of cultivated peanut</div>, <div class="source">Chromosome Res.</div> , <div class="year">2010</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">227</div>-<div class="lpage">46</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s10577-009-9109-z" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s10577-009-9109-z</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=FIDEL-a%20retrovirus-like%20retrotransposon%20and%20its%20distinct%20evolutionary%20histories%20in%20the%20A-%20and%20B-genome%20components%20of%20cultivated%20peanut&amp;author=S.%20Nielen&amp;author=F.%20Campos-Fonseca&amp;author=S.%20Leal-Bertioli&amp;publication_year=2010&amp;journal=Chromosome%20Res.&amp;volume=18&amp;pages=227-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10577-009-9109-z" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10577-009-9109-z" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10577-009-9109-z"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20127167" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nielen</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Vidigal</div> <div class="given-names">B.S.</div></div>,  <div class="name"><div class="surname">Leal-Bertioli</div> <div class="given-names">S.C.</div></div>, et al. </div><div class="article-title"><em>Matita</em>, a new retroelement from peanut: characterization and evolutionary context in the light of the <em>Arachis</em> A-B genome divergence</div>, <div class="source">Mol. Genet. Genomics</div> , <div class="year">2012</div>, vol. <div class="volume">287</div> (pg. <div class="fpage">21</div>-<div class="lpage">38</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00438-011-0656-6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00438-011-0656-6</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Matita%2C%20a%20new%20retroelement%20from%20peanut%3A%20characterization%20and%20evolutionary%20context%20in%20the%20light%20of%20the%20Arachis%20A-B%20genome%20divergence&amp;author=S.%20Nielen&amp;author=B.S.%20Vidigal&amp;author=S.C.%20Leal-Bertioli&amp;publication_year=2012&amp;journal=Mol.%20Genet.%20Genomics&amp;volume=287&amp;pages=21-38" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00438-011-0656-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00438-011-0656-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00438-011-0656-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22120641" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lander</div> <div class="given-names">E.S.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Abrahamson</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Barlow</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Daly</div> <div class="given-names">M.J.</div></div>,  <div class="name"><div class="surname">Lincoln</div> <div class="given-names">S.E.</div></div>,  <div class="name"><div class="surname">Newberg</div> <div class="given-names">L.A.</div></div>. </div><div class="article-title">MAPMAKER: an interactive computer package for constructing primary genetic linkage maps of experimental and natural populations</div>, <div class="source">Genomics</div> , <div class="year">1987</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">174</div>-<div class="lpage">81</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0888-7543(87)90010-3" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0888-7543(87)90010-3</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MAPMAKER%3A%20an%20interactive%20computer%20package%20for%20constructing%20primary%20genetic%20linkage%20maps%20of%20experimental%20and%20natural%20populations&amp;author=E.S.%20Lander&amp;author=P.%20Green&amp;author=J.%20Abrahamson&amp;author=A.%20Barlow&amp;author=M.J.%20Daly&amp;author=S.E.%20Lincoln&amp;author=L.A.%20Newberg&amp;publication_year=1987&amp;journal=Genomics&amp;volume=1&amp;pages=174-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0888-7543(87)90010-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0888-7543(87)90010-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0888-7543(87)90010-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3692487" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sim</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Durstewitz</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Plieske</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Development of a large SNP genotyping array and generation of high-density genetic maps in tomato</div>, <div class="source">PLoS ONE</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e40563</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0040563" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0040563</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Development%20of%20a%20large%20SNP%20genotyping%20array%20and%20generation%20of%20high-density%20genetic%20maps%20in%20tomato&amp;author=S.C.%20Sim&amp;author=G.%20Durstewitz&amp;author=J.%20Plieske&amp;publication_year=2012&amp;journal=PLoS%20ONE&amp;volume=7&amp;pages=e40563" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0040563" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0040563" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0040563"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22802968" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Asamizu</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Fukuoka</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">An interspecific linkage map of SSR and intronic polymorphism markers in tomato</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2010</div>, vol. <div class="volume">121</div> (pg. <div class="fpage">731</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-010-1344-3" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-010-1344-3</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20interspecific%20linkage%20map%20of%20SSR%20and%20intronic%20polymorphism%20markers%20in%20tomato&amp;author=K.%20Shirasawa&amp;author=E.%20Asamizu&amp;author=H.%20Fukuoka&amp;publication_year=2010&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=121&amp;pages=731-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-010-1344-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-010-1344-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-010-1344-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20431859" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Isobe</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">SNP discovery and linkage map construction in cultivated tomato</div>, <div class="source">DNA Res.</div> , <div class="year">2010</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">381</div>-<div class="lpage">91</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsq024" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsq024</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SNP%20discovery%20and%20linkage%20map%20construction%20in%20cultivated%20tomato&amp;author=K.%20Shirasawa&amp;author=S.%20Isobe&amp;author=H.%20Hirakawa&amp;publication_year=2010&amp;journal=DNA%20Res.&amp;volume=17&amp;pages=381-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq024" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq024" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq024"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21044984" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rieseberg</div> <div class="given-names">L.H.</div></div>,  <div class="name"><div class="surname">Linder</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Seiler</div> <div class="given-names">G.J.</div></div>. </div><div class="article-title">Chromosomal and genic barriers to introgression in Helianthus</div>, <div class="source">Genetics</div> , <div class="year">1995</div>, vol. <div class="volume">141</div> (pg. <div class="fpage">1163</div>-<div class="lpage">71</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosomal%20and%20genic%20barriers%20to%20introgression%20in%20Helianthus&amp;author=L.H.%20Rieseberg&amp;author=C.L.%20Linder&amp;author=G.J.%20Seiler&amp;publication_year=1995&amp;journal=Genetics&amp;volume=141&amp;pages=1163-71" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8582621" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS042C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS042C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lavin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Herendeen</div> <div class="given-names">P.S.</div></div>,  <div class="name"><div class="surname">Wojciechowski</div> <div class="given-names">M.F.</div></div>. </div><div class="article-title">Evolutionary rates analysis of Leguminosae implicates a rapid diversification of lineages during the tertiary</div>, <div class="source">Syst. Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">54</div> pg. <div class="fpage">575</div> <div class="pub-id-doi"><a href="https://doi.org/10.1080/10635150590947131" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1080/10635150590947131</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20rates%20analysis%20of%20Leguminosae%20implicates%20a%20rapid%20diversification%20of%20lineages%20during%20the%20tertiary&amp;author=M.%20Lavin&amp;author=P.S.%20Herendeen&amp;author=M.F.%20Wojciechowski&amp;publication_year=2005&amp;journal=Syst.%20Biol.&amp;volume=54&amp;pages=575" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/10635150590947131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F10635150590947131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F10635150590947131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16085576" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Kazuhiro Sato</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by-nc/3.0/), which permits non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=333781&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss042/2/dss042-dss042supp_table3.xls?Expires=2147483647&amp;Signature=pa9tO9GPlZ8aKw2~zA06APFK38UUNUr1kvy0GTz4hIgfx1ZZX57I7-sin0CUX-Pvv1F9SqOoqaxrIAkKmT0olW7FTWYMq0MBseBrQUurxjggjzcX5r859aMFTQtzHeueDFvH0cidD~WC-kL-v8FoJmQAgAHqQHjc9bb0SwQN-XkrovKEwOqdnuZNdhf~oDkKmAjdoYAoeLGZql5WWrlSoKm5DOfgp5uM4jort9HAA300JeIEpJ3fNFO9j3RvVEd7Ok1Rot8M5vaMzDRY7GlUcOjvIDH7UK0vo4LADoWEenl64Q0brLMEw4jUoQ76OLdbrVr697v0Qr-KyGQ1wpfu3A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss042/2/dss042-dss042supp_table2.xls?Expires=2147483647&amp;Signature=gPGB3~XOsID20-gF8I00XRgSB7xpHAYwv-5Kq-Na9NQxHuyCiAB411kzZ6CEckfFHz5-e5QPXz4qCoemHCd4523Gm6sTgQnn9SN8z3mVRzZ5d9iXau-q66d2A-aXr6A8biaZfyGRGbtDKhFRUrvf56je7mH-tnSS3t~LWyCr9JZubFyLoJ2ZnX-qZxtJbkjdNYdp8-w37Na961LEh2qVv9tNPIEjVpalTY6kOcCTlT-8ETGkjsAJExxE~mb-Vt9I5mwghiXLda8U8VWD99UGSG4ieXPUpQDKtt6C9eDpaE-jIs-D7VCBxrtjGK4oU1gwr0WSSqTGe8gnYsAYXCiUlQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/2/10.1093/dnares/dss042/2/dss042-dss042supp_table1.xls?Expires=2147483647&amp;Signature=seMtw21azP0UHMUIsU~i1EaA8MjlS8S5E~PrXRrHO7579Ppm-ZAIKQEE~klWjGuxvIKBEvWCaEyUO6bxm6oB37PcIqKdhRJR5pw2XdmLqqfwqpDyVKwK0lMpsYn2I~oifF3xmq~rxDzitJI5ndwSpFyODXphtZx9sZzbLe14CKRQRmQOOIvQPZ25QXxS44GD9IUtg72oEnjRN~-PpdrzK025zy59wbNNwGCwALBR3d9m0R6GYrfprqd0Dezt-kbBzhdI5b6Cr5QHypn~otaAar1vex6PTfcCgGX1qGUQ0kk2UNohcBmsix70E3XQRanLHomFHg5kJxVkCfG~rz9ijw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - xls file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">520</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000318584600006&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">54</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss042"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="333781"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000318584600006&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f2%2f173%2f333781">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000318584600006&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (54)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F2%2F173%2F333781" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/333781'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Integrated Consensus Map of Cultivated Peanut and Wild Relatives Reveals Structures of the A and B Genomes of Arachis and Divergence of the Legume Genomes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss042");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f2%2f173%2f333781">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

