

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/20/6/605/420907","@type":"ScholarlyArticle","name":"Epigenetic Basis of Regeneration: Analysis of Genomic DNA Methylation Profiles in the MRL/MpJ Mouse","datePublished":"2013-08-08","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/20/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2013-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dst034","keywords":["MRL/MpJ mouse","regeneration","DNA methylation","epigenetics","genome-wide DNA methylation profiling"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/20/6/605/420907","author":[{"name":"Górnikiewicz, Bartosz","affiliation":"1  Department of Microbiology ,  Gdańsk University of Technology ,  Gdańsk ,  Poland","@type":"Person"},{"name":"Ronowicz, Anna","affiliation":"2  Department of Biology and Pharmaceutical Botany ,  Medical University of Gdańsk ,  Gdańsk ,  Poland","@type":"Person"},{"name":"Podolak, Justyna","affiliation":"1  Department of Microbiology ,  Gdańsk University of Technology ,  Gdańsk ,  Poland","@type":"Person"},{"name":"Madanecki, Piotr","affiliation":"2  Department of Biology and Pharmaceutical Botany ,  Medical University of Gdańsk ,  Gdańsk ,  Poland","@type":"Person"},{"name":"Stanisławska-Sachadyn,  Anna","affiliation":"3  Department of Biology and Genetics ,  Medical University of Gdańsk ,  Gdańsk ,  Poland","@type":"Person"},{"name":"Sachadyn, PaweŁ","affiliation":"1  Department of Microbiology ,  Gdańsk University of Technology ,  Gdańsk ,  Poland","@type":"Person"}],"description":"Abstract.  Epigenetic regulation plays essential role in cell differentiation and dedifferentiation, which are the intrinsic processes involved in regeneration.","pageStart":"605","pageEnd":"621","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/6/1/m_cover.gif?Expires=2147483647&Signature=VPhijSxXdAPLpw4YhUouOC1tx3X8x9GMJFVJIau9z4hN5cdWK-jPPF1wEurkS1gh6aQR9TQYLHOUG1kzMtF4HcnoT9E-wJJg-rleKg0FU2PY4tefYkghKgj~tiS7jfzXHN~ky5UBYwxoXbHB1rhYpbzfxlAWCYEab07wd~Go~sCK2Z9WKtR5wEAnnMAXr1I8E45jivf904J4XT3ytlfCnSQKIVeT43OyeE29BJRkSUT0SEXeMt6-ucAdFaiaEOPvq~y1xtuGK9d7YWhoevE9wdrzD7fhQ5jWGawbllCtUUH0ojTdHlsKW2bYFxcwuER72KVIwWBylO57igmYb52w8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Epigenetic Basis of Regeneration: Analysis of Genomic DNA Methylation Profiles in the MRL/MpJ Mouse","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/6/1/m_cover.gif?Expires=2147483647&Signature=VPhijSxXdAPLpw4YhUouOC1tx3X8x9GMJFVJIau9z4hN5cdWK-jPPF1wEurkS1gh6aQR9TQYLHOUG1kzMtF4HcnoT9E-wJJg-rleKg0FU2PY4tefYkghKgj~tiS7jfzXHN~ky5UBYwxoXbHB1rhYpbzfxlAWCYEab07wd~Go~sCK2Z9WKtR5wEAnnMAXr1I8E45jivf904J4XT3ytlfCnSQKIVeT43OyeE29BJRkSUT0SEXeMt6-ucAdFaiaEOPvq~y1xtuGK9d7YWhoevE9wdrzD7fhQ5jWGawbllCtUUH0ojTdHlsKW2bYFxcwuER72KVIwWBylO57igmYb52w8A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Epigenetic Basis of Regeneration: Analysis of Genomic DNA Methylation Profiles in the MRL/MpJ Mouse" />
    <meta property="og:description" content="Abstract.  Epigenetic regulation plays essential role in cell differentiation and dedifferentiation, which are the intrinsic processes involved in regeneration." />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/20/6/605/420907" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2013-08-08" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Górnikiewicz, Bartosz" /><meta name="citation_author_institution" content="Department of Microbiology, Gdańsk University of Technology, Gdańsk, Poland" /><meta name="citation_author" content="Ronowicz, Anna" /><meta name="citation_author_institution" content="Department of Biology and Pharmaceutical Botany, Medical University of Gdańsk, Gdańsk, Poland" /><meta name="citation_author" content="Podolak, Justyna" /><meta name="citation_author_institution" content="Department of Microbiology, Gdańsk University of Technology, Gdańsk, Poland" /><meta name="citation_author" content="Madanecki, Piotr" /><meta name="citation_author_institution" content="Department of Biology and Pharmaceutical Botany, Medical University of Gdańsk, Gdańsk, Poland" /><meta name="citation_author" content="Stanisławska-Sachadyn,  Anna" /><meta name="citation_author_institution" content="Department of Biology and Genetics, Medical University of Gdańsk, Gdańsk, Poland" /><meta name="citation_author" content="Sachadyn, PaweŁ" /><meta name="citation_author_institution" content="Department of Microbiology, Gdańsk University of Technology, Gdańsk, Poland" /><meta name="citation_title" content="Epigenetic Basis of Regeneration: Analysis of Genomic DNA Methylation Profiles in the MRL/MpJ Mouse" /><meta name="citation_firstpage" content="605" /><meta name="citation_lastpage" content="621" /><meta name="citation_doi" content="10.1093/dnares/dst034" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="20" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2013/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Correlation between Shh expression and DNA methylation status of the limb-specific Shh enhancer region during limb regeneration in amphibians; Dev. Biol.;  citation_year=2007;  citation_volume=312;  citation_pages=171-82; " /><meta name="citation_reference" content="citation_title=Folate regulation of axonal regeneration in the rodent central nervous system through DNA methylation; J. Clin. Invest.;  citation_year=2010;  citation_volume=120;  citation_pages=1603-16; " /><meta name="citation_reference" content="citation_title=A new murine model for mammalian wound repair and regeneration; Clin. Immunol. Immunopathol.;  citation_year=1998;  citation_volume=88;  citation_pages=35-45; " /><meta name="citation_reference" content="citation_title=Accelerated wound healing of alkali-burned corneas in MRL mice is associated with a reduced inflammatory signature; Invest. Ophthalmol. Vis. Sci.;  citation_year=2005;  citation_volume=46;  citation_pages=4097-106; " /><meta name="citation_reference" content="citation_title=Elevated MMP expression in the MRL mouse retina creates a permissive environment for retinal regeneration; Invest. Ophthalmol. Vis. Sci.;  citation_year=2008;  citation_volume=49;  citation_pages=1686-95; " /><meta name="citation_reference" content="citation_title=Enhanced retinal pigment epithelium regeneration after injury in MRL/MpJ mice; Exp. Eye Res.;  citation_year=2011;  citation_volume=93;  citation_pages=862-72; " /><meta name="citation_reference" content="citation_title=Digit tip regrowth and differential gene expression in MRL/Mpj, DBA/2, and C57BL/6 mice; Wound Repair Regen.;  citation_year=2007;  citation_volume=15;  citation_pages=275-84; " /><meta name="citation_reference" content="citation_title=Dynamic changes after murine digit amputation: the MRL mouse digit shows waves of tissue remodeling, growth, and apoptosis; Wound Repair Regen.;  citation_year=2009;  citation_volume=17;  citation_pages=447-55; " /><meta name="citation_reference" content="citation_title=Heart regeneration in adult MRL mice; Proc. Natl. Acad. Sci. USA;  citation_year=2001;  citation_volume=98;  citation_pages=9830-5; " /><meta name="citation_reference" content="citation_title=Evidence for articular cartilage regeneration in MRL/MpJ mice; Osteoarthritis Cartilage;  citation_year=2008;  citation_volume=16;  citation_pages=1319-26; " /><meta name="citation_reference" content="citation_title=Genetic control of the rate of wound healing in mice; Heredity;  citation_year=2001;  citation_volume=86;  citation_pages=668-74; " /><meta name="citation_reference" content="citation_title=Matrix metalloproteinase activity correlates with blasteoma formation in the regenerating MRL mouse ear hole model; Dev. Dyn.;  citation_year=2003;  citation_volume=226;  citation_pages=377-87; " /><meta name="citation_reference" content="citation_title=Amphibian limb regeneration: rebuilding a complex structure; Science;  citation_year=1997;  citation_volume=276;  citation_pages=81-7; " /><meta name="citation_reference" content="citation_title=Urodele spinal cord regeneration and related processes; Dev. Dyn.;  citation_year=2003;  citation_volume=226;  citation_pages=295-307; " /><meta name="citation_reference" content="citation_title=Heart development and regeneration in urodeles; Int. J. Dev. Biol.;  citation_year=1996;  citation_volume=40;  citation_pages=719-25; " /><meta name="citation_reference" content="citation_title=Reparative myocardial mechanisms in adult C57BL/6 and MRL mice following injury; Physiol. Genomics;  citation_year=2007;  citation_volume=30;  citation_pages=44-52; " /><meta name="citation_reference" content="citation_title=Early postmyocardial infarction survival in Murphy Roths large mice is mediated by attenuated apoptosis and inflammation but depends on genetic background; Exp. Physiol.;  citation_year=2012;  citation_volume=97;  citation_pages=102-14; " /><meta name="citation_reference" content="citation_title=The MRL mouse heart does not recover ventricular function after a myocardial infarction; Cardiovasc. Pathol.;  citation_year=2008;  citation_volume=17;  citation_pages=32-9; " /><meta name="citation_reference" content="citation_title=MRL mice fail to heal the heart in response to ischemia-reperfusion injury; Wound Repair Regen.;  citation_year=2005;  citation_volume=13;  citation_pages=205-8; " /><meta name="citation_reference" content="citation_title=Absence of regeneration in the MRL/MpJ mouse heart following infarction or cryoinjury; Cardiovasc. Pathol.;  citation_year=2008;  citation_volume=17;  citation_pages=6-13; " /><meta name="citation_reference" content="citation_title=Limitations of the MRL mouse as a model for cardiac regeneration; J. Pharm. Pharmacol.;  citation_year=2011;  citation_volume=63;  citation_pages=648-56; " /><meta name="citation_reference" content="citation_title=The MRL mouse repairs both cryogenic and ischemic myocardial infarcts with scar; Cardiovasc. Pathol.;  citation_year=2008;  citation_volume=17;  citation_pages=14-22; " /><meta name="citation_reference" content="citation_title=Sexually dimorphic genes regulate healing and regeneration in MRL mice; Mamm. Genome;  citation_year=2003;  citation_volume=14;  citation_pages=250-60; " /><meta name="citation_reference" content="citation_title=Retained features of embryonic metabolism in the adult MRL mouse; Mol. Genet. Metab.;  citation_year=2009;  citation_volume=96;  citation_pages=133-44; " /><meta name="citation_reference" content="citation_title=Enhanced functional recovery in MRL/MpJ mice after spinal cord dorsal hemisection; PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e30904" /><meta name="citation_reference" content="citation_title=Chromosome-wide and promoter-specific analyses identify sites of differential DNA methylation in normal and transformed human cells; Nat. Genet.;  citation_year=2005;  citation_volume=37;  citation_pages=853-62; " /><meta name="citation_reference" content="citation_title=Regeneration of comparative genomic hybridization oligonucleotide microarrays with dimethylurea; Anal. Biochem.;  citation_year=2012;  citation_volume=426;  citation_pages=91-3; " /><meta name="citation_reference" content="citation_title=Idiographica: a general-purpose web application to build idiograms on-demand for human, mouse and rat; Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=2945-6; " /><meta name="citation_reference" content="citation_title=Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources; Nat. Protoc.;  citation_year=2009;  citation_volume=4;  citation_pages=44-57; " /><meta name="citation_reference" content="citation_title=Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists; Nucleic Acids Res.;  citation_year=2009;  citation_volume=37;  citation_pages=1-13; " /><meta name="citation_reference" content="citation_title=The relationship between inflammation and regeneration in the MRL mouse: potential relevance for putative human regenerative (scarless wound healing) capacities?; Ann. N Y Acad. Sci.;  citation_year=2009;  citation_volume=1172;  citation_pages=110-4; " /><meta name="citation_reference" content="citation_title=GREB 1 is a critical regulator of hormone dependent breast cancer growth; Breast Cancer Res. Treat.;  citation_year=2005;  citation_volume=92;  citation_pages=141-9; " /><meta name="citation_reference" content="citation_title=PDCD2 controls hematopoietic stem cell differentiation during development; Stem Cells Dev.;  citation_year=2013;  citation_volume=22;  citation_pages=58-72; " /><meta name="citation_reference" content="citation_title=Molecular signature of cell cycle exit induced in human T lymphoblasts by IL-2 withdrawal; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=261" /><meta name="citation_reference" content="citation_title=Combined analysis of three genome-wide association studies on vWF and FVIII plasma levels; BMC Med. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=102" /><meta name="citation_reference" content="citation_title=Genetic variant on PDGFRL associated with Behcet disease in Chinese Han populations; Hum. Mutat.;  citation_year=2012;  citation_volume=34;  citation_pages=74-8; " /><meta name="citation_reference" content="citation_title=Transcriptomic analysis reveals early signs of liver toxicity in female MRL +/+ mice exposed to the acylating chemicals dichloroacetyl chloride and dichloroacetic anhydride; Chem. Res. Toxicol.;  citation_year=2008;  citation_volume=21;  citation_pages=572-82; " /><meta name="citation_reference" content="citation_title=Integration of QTL and bioinformatic tools to identify candidate genes for triglycerides in mice; J. Lipid Res.;  citation_year=2011;  citation_volume=52;  citation_pages=1672-82; " /><meta name="citation_reference" content="citation_title=Epigenetic regulation of Nanog gene in embryonic stem and trophoblast stem cells; Genes Cells;  citation_year=2007;  citation_volume=12;  citation_pages=387-96; " /><meta name="citation_reference" content="citation_title=Conserved role of intragenic DNA methylation in regulating alternative promoters; Nature;  citation_year=2010;  citation_volume=466;  citation_pages=253-7; " /><meta name="citation_reference" content="citation_title=Recurrent variations in DNA methylation in human pluripotent stem cells and their differentiated derivatives; Cell Stem Cell;  citation_year=2012;  citation_volume=10;  citation_pages=620-34; " /><meta name="citation_reference" content="citation_title=Altered olfactory function in the MRL model of CNS lupus; Behav. Brain Res.;  citation_year=2012;  citation_volume=234;  citation_pages=303-11; " /><meta name="citation_reference" content="citation_title=Foxc2 transcription factor: a newly described regulator of angiogenesis; Trends Cardiovasc. Med.;  citation_year=2008;  citation_volume=18;  citation_pages=224-8; " /><meta name="citation_reference" content="citation_title=Genome-wide survey reveals dynamic widespread tissue-specific changes in DNA methylation during development; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=231" /><meta name="citation_reference" content="citation_title=DNA methylation analysis at distal and proximal promoter regions of the oestrogen receptor gene in breast cancers; Br. J. Cancer;  citation_year=1999;  citation_volume=80;  citation_pages=1982-6; " /><meta name="citation_reference" content="citation_title=Increasing alternative promoter repertories is positively associated with differential expression and disease susceptibility; PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e9482" /><meta name="citation_reference" content="citation_title=The functional consequences of alternative promoter use in mammalian genomes; Trends Genet.;  citation_year=2008;  citation_volume=24;  citation_pages=167-77; " /><meta name="citation_reference" content="citation_title=LIM-domain proteins in transforming growth factor beta-induced epithelial-to-mesenchymal transition and myofibroblast differentiation; Cell Signal.;  citation_year=2012;  citation_volume=24;  citation_pages=819-25; " /><meta name="citation_reference" content="citation_title=Role of platelet-derived growth factors in physiology and medicine; Genes Dev.;  citation_year=2008;  citation_volume=22;  citation_pages=1276-312; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/20/6/605/1166213/dst034.pdf" /><meta name="description" content="Abstract.  Epigenetic regulation plays essential role in cell differentiation and dedifferentiation, which are the intrinsic processes involved in regeneration." />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/20/6/605/420907" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/20/6/605/420907" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/20/6/605/420907");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dst034';
    </script>
<title>
	Epigenetic Basis of Regeneration: Analysis of Genomic DNA Methylation Profiles in the MRL/MpJ Mouse | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./420907" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BeJrPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJHw7NybmlraWV3aWN6LCBCYXJ0b3N6IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBNaWNyb2Jpb2xvZ3ksIEdkYcWEc2sgVW5pdmVyc2l0eSBvZiBUZWNobm9sb2d5LCBHZGHFhHNrLCBQb2xhbmQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJSb25vd2ljeiwgQW5uYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9neSBhbmQgUGhhcm1hY2V1dGljYWwgQm90YW55LCBNZWRpY2FsIFVuaXZlcnNpdHkgb2YgR2RhxYRzaywgR2RhxYRzaywgUG9sYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUG9kb2xhaywgSnVzdHluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWljcm9iaW9sb2d5LCBHZGHFhHNrIFVuaXZlcnNpdHkgb2YgVGVjaG5vbG9neSwgR2RhxYRzaywgUG9sYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWFkYW5lY2tpLCBQaW90ciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmlvbG9neSBhbmQgUGhhcm1hY2V1dGljYWwgQm90YW55LCBNZWRpY2FsIFVuaXZlcnNpdHkgb2YgR2RhxYRzaywgR2RhxYRzaywgUG9sYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3RhbmlzxYJhd3NrYS1TYWNoYWR5biwgIEFubmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJpb2xvZ3kgYW5kIEdlbmV0aWNzLCBNZWRpY2FsIFVuaXZlcnNpdHkgb2YgR2RhxYRzaywgR2RhxYRzaywgUG9sYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FjaGFkeW4sIFBhd2XFgSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgTWljcm9iaW9sb2d5LCBHZGHFhHNrIFVuaXZlcnNpdHkgb2YgVGVjaG5vbG9neSwgR2RhxYRzaywgUG9sYW5kIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJFcGlnZW5ldGljIEJhc2lzIG9mIFJlZ2VuZXJhdGlvbjogQW5hbHlzaXMgb2YgR2Vub21pYyBETkEgTWV0aHlsYXRpb24gUHJvZmlsZXMgaW4gdGhlIE1STC9NcEogTW91c2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI2MDUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjYyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzdDAzNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDEzLzEyLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29ycmVsYXRpb24gYmV0d2VlbiBTaGggZXhwcmVzc2lvbiBhbmQgRE5BIG1ldGh5bGF0aW9uIHN0YXR1cyBvZiB0aGUgbGltYi1zcGVjaWZpYyBTaGggZW5oYW5jZXIgcmVnaW9uIGR1cmluZyBsaW1iIHJlZ2VuZXJhdGlvbiBpbiBhbXBoaWJpYW5zOyBEZXYuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTMxMjsgIGNpdGF0aW9uX3BhZ2VzPTE3MS04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Rm9sYXRlIHJlZ3VsYXRpb24gb2YgYXhvbmFsIHJlZ2VuZXJhdGlvbiBpbiB0aGUgcm9kZW50IGNlbnRyYWwgbmVydm91cyBzeXN0ZW0gdGhyb3VnaCBETkEgbWV0aHlsYXRpb247IEouIENsaW4uIEludmVzdC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTIwOyAgY2l0YXRpb25fcGFnZXM9MTYwMy0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBuZXcgbXVyaW5lIG1vZGVsIGZvciBtYW1tYWxpYW4gd291bmQgcmVwYWlyIGFuZCByZWdlbmVyYXRpb247IENsaW4uIEltbXVub2wuIEltbXVub3BhdGhvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9ODg7ICBjaXRhdGlvbl9wYWdlcz0zNS00NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjZWxlcmF0ZWQgd291bmQgaGVhbGluZyBvZiBhbGthbGktYnVybmVkIGNvcm5lYXMgaW4gTVJMIG1pY2UgaXMgYXNzb2NpYXRlZCB3aXRoIGEgcmVkdWNlZCBpbmZsYW1tYXRvcnkgc2lnbmF0dXJlOyBJbnZlc3QuIE9waHRoYWxtb2wuIFZpcy4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00NjsgIGNpdGF0aW9uX3BhZ2VzPTQwOTctMTA2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbGV2YXRlZCBNTVAgZXhwcmVzc2lvbiBpbiB0aGUgTVJMIG1vdXNlIHJldGluYSBjcmVhdGVzIGEgcGVybWlzc2l2ZSBlbnZpcm9ubWVudCBmb3IgcmV0aW5hbCByZWdlbmVyYXRpb247IEludmVzdC4gT3BodGhhbG1vbC4gVmlzLiBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTQ5OyAgY2l0YXRpb25fcGFnZXM9MTY4Ni05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5oYW5jZWQgcmV0aW5hbCBwaWdtZW50IGVwaXRoZWxpdW0gcmVnZW5lcmF0aW9uIGFmdGVyIGluanVyeSBpbiBNUkwvTXBKIG1pY2U7IEV4cC4gRXllIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9OTM7ICBjaXRhdGlvbl9wYWdlcz04NjItNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZ2l0IHRpcCByZWdyb3d0aCBhbmQgZGlmZmVyZW50aWFsIGdlbmUgZXhwcmVzc2lvbiBpbiBNUkwvTXBqLCBEQkEvMiwgYW5kIEM1N0JMLzYgbWljZTsgV291bmQgUmVwYWlyIFJlZ2VuLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTI3NS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RHluYW1pYyBjaGFuZ2VzIGFmdGVyIG11cmluZSBkaWdpdCBhbXB1dGF0aW9uOiB0aGUgTVJMIG1vdXNlIGRpZ2l0IHNob3dzIHdhdmVzIG9mIHRpc3N1ZSByZW1vZGVsaW5nLCBncm93dGgsIGFuZCBhcG9wdG9zaXM7IFdvdW5kIFJlcGFpciBSZWdlbi47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTc7ICBjaXRhdGlvbl9wYWdlcz00NDctNTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhlYXJ0IHJlZ2VuZXJhdGlvbiBpbiBhZHVsdCBNUkwgbWljZTsgUHJvYy4gTmF0bC4gQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9OTg7ICBjaXRhdGlvbl9wYWdlcz05ODMwLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciBhcnRpY3VsYXIgY2FydGlsYWdlIHJlZ2VuZXJhdGlvbiBpbiBNUkwvTXBKIG1pY2U7IE9zdGVvYXJ0aHJpdGlzIENhcnRpbGFnZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTEzMTktMjY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgY29udHJvbCBvZiB0aGUgcmF0ZSBvZiB3b3VuZCBoZWFsaW5nIGluIG1pY2U7IEhlcmVkaXR5OyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTg2OyAgY2l0YXRpb25fcGFnZXM9NjY4LTc0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXRyaXggbWV0YWxsb3Byb3RlaW5hc2UgYWN0aXZpdHkgY29ycmVsYXRlcyB3aXRoIGJsYXN0ZW9tYSBmb3JtYXRpb24gaW4gdGhlIHJlZ2VuZXJhdGluZyBNUkwgbW91c2UgZWFyIGhvbGUgbW9kZWw7IERldi4gRHluLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMjY7ICBjaXRhdGlvbl9wYWdlcz0zNzctODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFtcGhpYmlhbiBsaW1iIHJlZ2VuZXJhdGlvbjogcmVidWlsZGluZyBhIGNvbXBsZXggc3RydWN0dXJlOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI3NjsgIGNpdGF0aW9uX3BhZ2VzPTgxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVyb2RlbGUgc3BpbmFsIGNvcmQgcmVnZW5lcmF0aW9uIGFuZCByZWxhdGVkIHByb2Nlc3NlczsgRGV2LiBEeW4uOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTIyNjsgIGNpdGF0aW9uX3BhZ2VzPTI5NS0zMDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhlYXJ0IGRldmVsb3BtZW50IGFuZCByZWdlbmVyYXRpb24gaW4gdXJvZGVsZXM7IEludC4gSi4gRGV2LiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT00MDsgIGNpdGF0aW9uX3BhZ2VzPTcxOS0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVwYXJhdGl2ZSBteW9jYXJkaWFsIG1lY2hhbmlzbXMgaW4gYWR1bHQgQzU3QkwvNiBhbmQgTVJMIG1pY2UgZm9sbG93aW5nIGluanVyeTsgUGh5c2lvbC4gR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzA7ICBjaXRhdGlvbl9wYWdlcz00NC01MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RWFybHkgcG9zdG15b2NhcmRpYWwgaW5mYXJjdGlvbiBzdXJ2aXZhbCBpbiBNdXJwaHkgUm90aHMgbGFyZ2UgbWljZSBpcyBtZWRpYXRlZCBieSBhdHRlbnVhdGVkIGFwb3B0b3NpcyBhbmQgaW5mbGFtbWF0aW9uIGJ1dCBkZXBlbmRzIG9uIGdlbmV0aWMgYmFja2dyb3VuZDsgRXhwLiBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05NzsgIGNpdGF0aW9uX3BhZ2VzPTEwMi0xNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIE1STCBtb3VzZSBoZWFydCBkb2VzIG5vdCByZWNvdmVyIHZlbnRyaWN1bGFyIGZ1bmN0aW9uIGFmdGVyIGEgbXlvY2FyZGlhbCBpbmZhcmN0aW9uOyBDYXJkaW92YXNjLiBQYXRob2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MzItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TVJMIG1pY2UgZmFpbCB0byBoZWFsIHRoZSBoZWFydCBpbiByZXNwb25zZSB0byBpc2NoZW1pYS1yZXBlcmZ1c2lvbiBpbmp1cnk7IFdvdW5kIFJlcGFpciBSZWdlbi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yMDUtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWJzZW5jZSBvZiByZWdlbmVyYXRpb24gaW4gdGhlIE1STC9NcEogbW91c2UgaGVhcnQgZm9sbG93aW5nIGluZmFyY3Rpb24gb3IgY3J5b2luanVyeTsgQ2FyZGlvdmFzYy4gUGF0aG9sLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTYtMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpbWl0YXRpb25zIG9mIHRoZSBNUkwgbW91c2UgYXMgYSBtb2RlbCBmb3IgY2FyZGlhYyByZWdlbmVyYXRpb247IEouIFBoYXJtLiBQaGFybWFjb2wuOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTYzOyAgY2l0YXRpb25fcGFnZXM9NjQ4LTU2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgTVJMIG1vdXNlIHJlcGFpcnMgYm90aCBjcnlvZ2VuaWMgYW5kIGlzY2hlbWljIG15b2NhcmRpYWwgaW5mYXJjdHMgd2l0aCBzY2FyOyBDYXJkaW92YXNjLiBQYXRob2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MTQtMjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNleHVhbGx5IGRpbW9ycGhpYyBnZW5lcyByZWd1bGF0ZSBoZWFsaW5nIGFuZCByZWdlbmVyYXRpb24gaW4gTVJMIG1pY2U7IE1hbW0uIEdlbm9tZTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTI1MC02MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmV0YWluZWQgZmVhdHVyZXMgb2YgZW1icnlvbmljIG1ldGFib2xpc20gaW4gdGhlIGFkdWx0IE1STCBtb3VzZTsgTW9sLiBHZW5ldC4gTWV0YWIuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk2OyAgY2l0YXRpb25fcGFnZXM9MTMzLTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmhhbmNlZCBmdW5jdGlvbmFsIHJlY292ZXJ5IGluIE1STC9NcEogbWljZSBhZnRlciBzcGluYWwgY29yZCBkb3JzYWwgaGVtaXNlY3Rpb247IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lMzA5MDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaHJvbW9zb21lLXdpZGUgYW5kIHByb21vdGVyLXNwZWNpZmljIGFuYWx5c2VzIGlkZW50aWZ5IHNpdGVzIG9mIGRpZmZlcmVudGlhbCBETkEgbWV0aHlsYXRpb24gaW4gbm9ybWFsIGFuZCB0cmFuc2Zvcm1lZCBodW1hbiBjZWxsczsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz04NTMtNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ2VuZXJhdGlvbiBvZiBjb21wYXJhdGl2ZSBnZW5vbWljIGh5YnJpZGl6YXRpb24gb2xpZ29udWNsZW90aWRlIG1pY3JvYXJyYXlzIHdpdGggZGltZXRoeWx1cmVhOyBBbmFsLiBCaW9jaGVtLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00MjY7ICBjaXRhdGlvbl9wYWdlcz05MS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGlvZ3JhcGhpY2E6IGEgZ2VuZXJhbC1wdXJwb3NlIHdlYiBhcHBsaWNhdGlvbiB0byBidWlsZCBpZGlvZ3JhbXMgb24tZGVtYW5kIGZvciBodW1hbiwgbW91c2UgYW5kIHJhdDsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0yOTQ1LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5c3RlbWF0aWMgYW5kIGludGVncmF0aXZlIGFuYWx5c2lzIG9mIGxhcmdlIGdlbmUgbGlzdHMgdXNpbmcgREFWSUQgYmlvaW5mb3JtYXRpY3MgcmVzb3VyY2VzOyBOYXQuIFByb3RvYy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTQ0LTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CaW9pbmZvcm1hdGljcyBlbnJpY2htZW50IHRvb2xzOiBwYXRocyB0b3dhcmQgdGhlIGNvbXByZWhlbnNpdmUgZnVuY3Rpb25hbCBhbmFseXNpcyBvZiBsYXJnZSBnZW5lIGxpc3RzOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0xLTEzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmVsYXRpb25zaGlwIGJldHdlZW4gaW5mbGFtbWF0aW9uIGFuZCByZWdlbmVyYXRpb24gaW4gdGhlIE1STCBtb3VzZTogcG90ZW50aWFsIHJlbGV2YW5jZSBmb3IgcHV0YXRpdmUgaHVtYW4gcmVnZW5lcmF0aXZlIChzY2FybGVzcyB3b3VuZCBoZWFsaW5nKSBjYXBhY2l0aWVzPzsgQW5uLiBOIFkgQWNhZC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTcyOyAgY2l0YXRpb25fcGFnZXM9MTEwLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdSRUIgMSBpcyBhIGNyaXRpY2FsIHJlZ3VsYXRvciBvZiBob3Jtb25lIGRlcGVuZGVudCBicmVhc3QgY2FuY2VyIGdyb3d0aDsgQnJlYXN0IENhbmNlciBSZXMuIFRyZWF0LjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT05MjsgIGNpdGF0aW9uX3BhZ2VzPTE0MS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QRENEMiBjb250cm9scyBoZW1hdG9wb2lldGljIHN0ZW0gY2VsbCBkaWZmZXJlbnRpYXRpb24gZHVyaW5nIGRldmVsb3BtZW50OyBTdGVtIENlbGxzIERldi47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjI7ICBjaXRhdGlvbl9wYWdlcz01OC03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIHNpZ25hdHVyZSBvZiBjZWxsIGN5Y2xlIGV4aXQgaW5kdWNlZCBpbiBodW1hbiBUIGx5bXBob2JsYXN0cyBieSBJTC0yIHdpdGhkcmF3YWw7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTI2MSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbWJpbmVkIGFuYWx5c2lzIG9mIHRocmVlIGdlbm9tZS13aWRlIGFzc29jaWF0aW9uIHN0dWRpZXMgb24gdldGIGFuZCBGVklJSSBwbGFzbWEgbGV2ZWxzOyBCTUMgTWVkLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0xMDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIHZhcmlhbnQgb24gUERHRlJMIGFzc29jaWF0ZWQgd2l0aCBCZWhjZXQgZGlzZWFzZSBpbiBDaGluZXNlIEhhbiBwb3B1bGF0aW9uczsgSHVtLiBNdXRhdC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz03NC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21pYyBhbmFseXNpcyByZXZlYWxzIGVhcmx5IHNpZ25zIG9mIGxpdmVyIHRveGljaXR5IGluIGZlbWFsZSBNUkwgKy8rIG1pY2UgZXhwb3NlZCB0byB0aGUgYWN5bGF0aW5nIGNoZW1pY2FscyBkaWNobG9yb2FjZXR5bCBjaGxvcmlkZSBhbmQgZGljaGxvcm9hY2V0aWMgYW5oeWRyaWRlOyBDaGVtLiBSZXMuIFRveGljb2wuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9NTcyLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlZ3JhdGlvbiBvZiBRVEwgYW5kIGJpb2luZm9ybWF0aWMgdG9vbHMgdG8gaWRlbnRpZnkgY2FuZGlkYXRlIGdlbmVzIGZvciB0cmlnbHljZXJpZGVzIGluIG1pY2U7IEouIExpcGlkIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NTI7ICBjaXRhdGlvbl9wYWdlcz0xNjcyLTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcGlnZW5ldGljIHJlZ3VsYXRpb24gb2YgTmFub2cgZ2VuZSBpbiBlbWJyeW9uaWMgc3RlbSBhbmQgdHJvcGhvYmxhc3Qgc3RlbSBjZWxsczsgR2VuZXMgQ2VsbHM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0zODctOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbnNlcnZlZCByb2xlIG9mIGludHJhZ2VuaWMgRE5BIG1ldGh5bGF0aW9uIGluIHJlZ3VsYXRpbmcgYWx0ZXJuYXRpdmUgcHJvbW90ZXJzOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDY2OyAgY2l0YXRpb25fcGFnZXM9MjUzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY3VycmVudCB2YXJpYXRpb25zIGluIEROQSBtZXRoeWxhdGlvbiBpbiBodW1hbiBwbHVyaXBvdGVudCBzdGVtIGNlbGxzIGFuZCB0aGVpciBkaWZmZXJlbnRpYXRlZCBkZXJpdmF0aXZlczsgQ2VsbCBTdGVtIENlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz02MjAtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFsdGVyZWQgb2xmYWN0b3J5IGZ1bmN0aW9uIGluIHRoZSBNUkwgbW9kZWwgb2YgQ05TIGx1cHVzOyBCZWhhdi4gQnJhaW4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMzQ7ICBjaXRhdGlvbl9wYWdlcz0zMDMtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZveGMyIHRyYW5zY3JpcHRpb24gZmFjdG9yOiBhIG5ld2x5IGRlc2NyaWJlZCByZWd1bGF0b3Igb2YgYW5naW9nZW5lc2lzOyBUcmVuZHMgQ2FyZGlvdmFzYy4gTWVkLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTIyNC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBzdXJ2ZXkgcmV2ZWFscyBkeW5hbWljIHdpZGVzcHJlYWQgdGlzc3VlLXNwZWNpZmljIGNoYW5nZXMgaW4gRE5BIG1ldGh5bGF0aW9uIGR1cmluZyBkZXZlbG9wbWVudDsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MjMxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RE5BIG1ldGh5bGF0aW9uIGFuYWx5c2lzIGF0IGRpc3RhbCBhbmQgcHJveGltYWwgcHJvbW90ZXIgcmVnaW9ucyBvZiB0aGUgb2VzdHJvZ2VuIHJlY2VwdG9yIGdlbmUgaW4gYnJlYXN0IGNhbmNlcnM7IEJyLiBKLiBDYW5jZXI7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9ODA7ICBjaXRhdGlvbl9wYWdlcz0xOTgyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluY3JlYXNpbmcgYWx0ZXJuYXRpdmUgcHJvbW90ZXIgcmVwZXJ0b3JpZXMgaXMgcG9zaXRpdmVseSBhc3NvY2lhdGVkIHdpdGggZGlmZmVyZW50aWFsIGV4cHJlc3Npb24gYW5kIGRpc2Vhc2Ugc3VzY2VwdGliaWxpdHk7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lOTQ4MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBmdW5jdGlvbmFsIGNvbnNlcXVlbmNlcyBvZiBhbHRlcm5hdGl2ZSBwcm9tb3RlciB1c2UgaW4gbWFtbWFsaWFuIGdlbm9tZXM7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNjctNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxJTS1kb21haW4gcHJvdGVpbnMgaW4gdHJhbnNmb3JtaW5nIGdyb3d0aCBmYWN0b3IgYmV0YS1pbmR1Y2VkIGVwaXRoZWxpYWwtdG8tbWVzZW5jaHltYWwgdHJhbnNpdGlvbiBhbmQgbXlvZmlicm9ibGFzdCBkaWZmZXJlbnRpYXRpb247IENlbGwgU2lnbmFsLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTgxOS0yNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9sZSBvZiBwbGF0ZWxldC1kZXJpdmVkIGdyb3d0aCBmYWN0b3JzIGluIHBoeXNpb2xvZ3kgYW5kIG1lZGljaW5lOyBHZW5lcyBEZXYuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9MTI3Ni0zMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yMC82LzYwNS8xMTY2MjEzL2RzdDAzNC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkFic3RyYWN0LiAgRXBpZ2VuZXRpYyByZWd1bGF0aW9uIHBsYXlzIGVzc2VudGlhbCByb2xlIGluIGNlbGwgZGlmZmVyZW50aWF0aW9uIGFuZCBkZWRpZmZlcmVudGlhdGlvbiwgd2hpY2ggYXJlIHRoZSBpbnRyaW5zaWMgcHJvY2Vzc2VzIGludm9sdmVkIGluIHJlZ2VuZXJhdGlvbi4iIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVyLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTQyMDkwNyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGQ1uUSjXCTSWJ6dsL52trefPXfgrA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f605%2f420907" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f605%2f420907" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=16367" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/20/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/20/6/1/m_cover.gif?Expires=2147483647&amp;Signature=VPhijSxXdAPLpw4YhUouOC1tx3X8x9GMJFVJIau9z4hN5cdWK-jPPF1wEurkS1gh6aQR9TQYLHOUG1kzMtF4HcnoT9E-wJJg-rleKg0FU2PY4tefYkghKgj~tiS7jfzXHN~ky5UBYwxoXbHB1rhYpbzfxlAWCYEab07wd~Go~sCK2Z9WKtR5wEAnnMAXr1I8E45jivf904J4XT3ytlfCnSQKIVeT43OyeE29BJRkSUT0SEXeMt6-ucAdFaiaEOPvq~y1xtuGK9d7YWhoevE9wdrzD7fhQ5jWGawbllCtUUH0ojTdHlsKW2bYFxcwuER72KVIwWBylO57igmYb52w8A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/20/6">
                        <div class="volume trailing-comma">Volume 20</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2013        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6237313">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6237313">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6237315">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6237315">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6237320">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6237320">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6237353">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6237353">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6237404">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6237404">4. Concluding remarks</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6237408">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6237408">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6237410">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6237410">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6237412">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6237412">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6237414">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6237414">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/20/6/593/420691">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/20/6/623/421469">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Epigenetic Basis of Regeneration: Analysis of Genomic DNA Methylation Profiles in the MRL/MpJ Mouse
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Bartosz G&#243;rnikiewicz</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">Gdańsk University of Technology</div>, <div class="addr-line">Gdańsk</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Bartosz+G%c3%b3rnikiewicz">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=G&#243;rnikiewicz B">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;G&#243;rnikiewicz B&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Bartosz Górnikiewicz</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Anna Ronowicz</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biology and Pharmaceutical Botany</div>, <div class="insititution">Medical University of Gdańsk</div>, <div class="addr-line">Gdańsk</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Anna+Ronowicz">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ronowicz A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ronowicz A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Anna Ronowicz</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Justyna Podolak</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">Gdańsk University of Technology</div>, <div class="addr-line">Gdańsk</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Justyna+Podolak">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Podolak J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Podolak J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Justyna Podolak</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Piotr Madanecki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Department of Biology and Pharmaceutical Botany</div>, <div class="insititution">Medical University of Gdańsk</div>, <div class="addr-line">Gdańsk</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Piotr+Madanecki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Madanecki P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Madanecki P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Piotr Madanecki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Anna Stanisławska-Sachadyn</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Biology and Genetics</div>, <div class="insititution">Medical University of Gdańsk</div>, <div class="addr-line">Gdańsk</div>, <div class="country">Poland</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Anna+Stanis%c5%82awska-Sachadyn">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Stanisławska-Sachadyn  ">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Stanisławska-Sachadyn  &quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Anna Stanisławska-Sachadyn</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">PaweŁ Sachadyn</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">Gdańsk University of Technology</div>, <div class="addr-line">Gdańsk</div>, <div class="country">Poland</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +48 58-3472406. Fax. +48 58-3471822. E-mail: <a href="mailto:psach@altis.chem.pg.gda.pl" target="_blank">psach@altis.chem.pg.gda.pl</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Pawe%c5%81+Sachadyn">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sachadyn P">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sachadyn P&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">PaweŁ Sachadyn</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 20, Issue 6, 1 December 2013, Pages 605–621, <a href='https://doi.org/10.1093/dnares/dst034'>https://doi.org/10.1093/dnares/dst034</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">08 August 2013</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">11 February 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">06 July 2013</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="420907" href="/dnaresearch/article-pdf/20/6/605/1166213/dst034.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Bartosz Górnikiewicz, Anna Ronowicz, Justyna Podolak, Piotr Madanecki, Anna Stanisławska-Sachadyn, PaweŁ Sachadyn;  Epigenetic Basis of Regeneration: Analysis of Genomic DNA Methylation Profiles in the MRL/MpJ Mouse, <em>DNA Research</em>, Volume 20, Issue 6, 1 December 2013, Pages 605–621, <a href='https://doi.org/10.1093/dnares/dst034'>https://doi.org/10.1093/dnares/dst034</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/420907?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/420907?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/420907?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/420907?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/420907?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/420907?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/420907?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Epigenetic%20Basis%20of%20Regeneration%3A%20Analysis%20of%20Genomic%20DNA%20Methylation%20Profiles%20in%20the%20MRL%2FMpJ%20Mouse&amp;publicationDate=2013-08-08&amp;volumeNum=20&amp;issueNum=6&amp;author=G%C3%B3rnikiewicz%2C%20Bartosz%3B%20Ronowicz%2C%20Anna&amp;startPage=605&amp;endPage=621&amp;contentId=10.1093%2Fdnares%2Fdst034&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f605%2f420907" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6237313 id="6237313" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Epigenetic regulation plays essential role in cell differentiation and dedifferentiation, which are the intrinsic processes involved in regeneration. To investigate the epigenetic basis of regeneration capacity, we choose DNA methylation as one of the most important epigenetic mechanisms and the MRL/MpJ mouse as a model of mammalian regeneration known to exhibit enhanced regeneration response in different organs. We report the comparative analysis of genomic DNA methylation profiles of the MRL/MpJ and the control C57BL/6J mouse. Methylated DNA immunoprecipitation followed by microarray analysis using the Nimblegen ‘3 × 720 K CpG Island Plus RefSeq Promoter’ platform was applied in order to carry out genome-wide DNA methylation profiling covering 20 404 promoter regions. We identified hundreds of hypo- and hypermethylated genes and CpG islands in the heart, liver, and spleen, and 37 of them in the three tissues. Decreased inter-tissue diversification and the shift of DNA methylation balance upstream the genes distinguish the genomic methylation patterns of the MRL/MpJ mouse from the C57BL/6J. Homeobox genes and a number of other genes involved in embryonic morphogenesis are significantly overrepresented among the genes hypomethylated in the MRL/MpJ mouse. These findings indicate that epigenetic patterning might be a likely molecular basis of regeneration capability in the MRL/MpJ mouse.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;MRL/MpJ mouse&quot;">MRL/MpJ mouse</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="regeneration">regeneration</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;DNA methylation&quot;">DNA methylation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="epigenetics">epigenetics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome-wide DNA methylation profiling&quot;">genome-wide DNA methylation profiling</a></div>                            <h2 scrollto-destination=6237315 id="6237315" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The phenomenon of regeneration has become the object of extensive research in recent years. The investigations on the molecular basis of regeneration in different models were rarely focused on epigenetic aspects, while it seems that these are the epigenetic mechanisms which could play a key role in both maintaining the capacity for regeneration and remodelling involved in the regenerative response. All organs in an organism originate from a single cell (zygote), which clearly indicates that a single genome contains the information needed to grow organs and tissues, and possibly to restore lost and injured structures. This information is stored through the whole life. However, the expression of this information is gradually repressed during development, with epigenetic mechanisms including DNA methylation playing an essential part.</p><p>Although the direct evidence on the involvement of DNA methylation in regeneration is scarce, a few remarkable findings are worth mentioning including the role of the <em>Shh</em> gene methylation in amphibian limb regeneration<sup><a reveal-id="DST034C1" data-open="DST034C1" class="link link-ref link-reveal xref-bibr">1</a></sup> and the changes of DNA methylation status in the <em>Gadd45a</em> promoter found during folate-induced regeneration response in the rodent spinal cord.<sup><a reveal-id="DST034C2" data-open="DST034C2" class="link link-ref link-reveal xref-bibr">2</a></sup> To our knowledge, no analysis of genome-wide DNA methylation profiles related to regeneration has been reported so far.</p><p>The MRL/MpJ mouse, an inbred laboratory strain of <em>Mus musculus</em>, emerged as the model of mammalian regeneration in the 1990s when the unusual phenomenon of regenerative ear-hole closure was discovered<sup><a reveal-id="DST034C3" data-open="DST034C3" class="link link-ref link-reveal xref-bibr">3</a></sup> to be followed by the observations of enhanced regenerative response in other tissues including cornea,<sup><a reveal-id="DST034C4" data-open="DST034C4" class="link link-ref link-reveal xref-bibr">4</a></sup> retina,<sup><a reveal-id="DST034C5" data-open="DST034C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DST034C6" data-open="DST034C6" class="link link-ref link-reveal xref-bibr">6</a></sup> digit tips,<sup><a reveal-id="DST034C7" data-open="DST034C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DST034C8" data-open="DST034C8" class="link link-ref link-reveal xref-bibr">8</a></sup> heart,<sup><a reveal-id="DST034C9" data-open="DST034C9" class="link link-ref link-reveal xref-bibr">9</a></sup> articular cartilage,<sup><a reveal-id="DST034C10" data-open="DST034C10" class="link link-ref link-reveal xref-bibr">10</a></sup> and spinal cord. The ability to close 2 mm through-and-through ear holes in the ear pinnae, which are used for a lifelong animal marking, clearly distinguishes the MRL/MpJ from most other mouse strains, although it is shared with the LG/J mouse, the MRL/MpJ's major ancestral strain.<sup><a reveal-id="DST034C11" data-open="DST034C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The response to injury in the MRL/MpJ mouse does not result in scar formation, but in complete restoration of excised fragment with normal tissue architecture, including blood vessels, hair follicles, sebaceous glands, and cartilage, that occurs within 30 days.<sup><a reveal-id="DST034C3" data-open="DST034C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Since this process requires reinstatement of different types of tissues, it resembles epimorphic regeneration observed in amphibians. One important feature of injury response in the MRL/MpJ mouse is increased activity of matrix metalloproteinases that decompose the extracellular matrix breaking the basal lamina, a layer that initially covers the wound surface. This process allows the formation of the blastema-like structure of undifferentiated cells that drive the process of regeneration.<sup><a reveal-id="DST034C12" data-open="DST034C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Another important feature of the MRL/MpJ mouse is its abnormal cell cycle profile with accumulation of the cells in the G2/M phase. This hallmark has been connected with the deficiency of p21, an inhibitor of cell cycle progression, that is not detected in the tissues of the MRL/MpJ mouse even after γ-irradiation. Although the regeneration abilities of the MRL/MpJ expand themselves on other tissues, they do not reach as spectacular effects as those observed in urodele amphibians which restore lost limbs<sup><a reveal-id="DST034C13" data-open="DST034C13" class="link link-ref link-reveal xref-bibr">13</a></sup> and regenerate as serious injuries as the complete transection of the spinal cord<sup><a reveal-id="DST034C14" data-open="DST034C14" class="link link-ref link-reveal xref-bibr">14</a></sup> and partial heart resection.<sup><a reveal-id="DST034C15" data-open="DST034C15" class="link link-ref link-reveal xref-bibr">15</a></sup> In the MRL/MpJ mouse, lost digits do not re-grow,<sup><a reveal-id="DST034C8" data-open="DST034C8" class="link link-ref link-reveal xref-bibr">8</a></sup> heart regeneration is rather limited,<sup><a reveal-id="DST034C9" data-open="DST034C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DST034C16" data-open="DST034C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DST034C17" data-open="DST034C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and the heart does not heal in some types of injuries.<sup><a reveal-id="DST034C18  DST034C22" data-open="DST034C18  DST034C22" class="link link-ref link-reveal xref-bibr">18–22</a></sup> The regenerative phenotype of the MRL/MpJ mouse has been found to be multigenic.<sup><a reveal-id="DST034C23" data-open="DST034C23" class="link link-ref link-reveal xref-bibr">23</a></sup> It is also remarkable that increased regenerative response in the MRL/MpJ mouse is found in different tissues, and the strain maintains selected features of embryonic metabolism in adults including the enhanced expression of key stem cell markers, <em>Sox2</em> and <em>Nanog</em>.<sup><a reveal-id="DST034C24" data-open="DST034C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DST034C25" data-open="DST034C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p><p>The object of the presented study is the comparative analysis of the genome-wide DNA methylation profiles in the MRL/MpJ “healer” mouse, using the C57BL/6J mouse as the reference. In our attempt to understand the epigenetic basis of regeneration, we would like to identify the differentially methylated genes and to evaluate their possible association with the regenerative capacity of the MRL/MpJ mouse. The study was designed to analyse global DNA methylation profiles in heart tissue, as it is known to have poor, if any, regenerative capacity, in order to refer to the liver, which has a strong regenerative potential, in addition to spleen, selected for its involvement in cell-mediated pathways of the immune system. As the regenerating tissues are difficult to obtain in sufficient amounts and the DNA methylation patterns in cultured cell lines are prone to change, we would like to focus the study on the normal tissues, thus examining the basis of regeneration rather than the regenerative response.</p>                            <h2 scrollto-destination=6237320 id="6237320" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6237321 id="6237321" class="section-title" data-legacy-id=s2a>2.1. Tissue samples and nucleic acid extraction</h3>
<p>Tissue samples (hearts, livers, and spleens) from 5-week-old males and 8-week-old females of MRL/MpJ (stock #000486) and C57BL/6J (stock #000664) murine strains were purchased from the Jackson Laboratory. The samples from 8 week-old females were not used for the microarray analyses, but for the validation experiments only. Genomic DNA was isolated using the DNeasy Blood &amp;Tissue Kit (QIAGEN, cat. no. 69504) and RNA with the RNeasy Mini Kit (QIAGEN, cat. no. 74104) according to the manufacturer's protocols. Tissues were delivered in RNAlater stablilization reagent (QIAGEN, cat. no. 76104) and were disrupted in liquid nitrogen before nucleic acids isolation.</p>                            <h3 scrollto-destination=6237323 id="6237323" class="section-title" data-legacy-id=s2b>2.2. Methylated DNA immunoprecipitation</h3>
<p>Methylated DNA immunoprecipitation (MeDIP) was performed according to the modified Weber's protocol.<sup><a reveal-id="DST034C26" data-open="DST034C26" class="link link-ref link-reveal xref-bibr">26</a></sup> Equal amounts of genomic DNA from three individuals were pooled for each strain and tissue. Each pooled sample (7 µg) was diluted in 150 µl of water and randomly sheared by sonication to generate fragments between 300 and 1000 bp. A 4-µg portion of sonicated DNA was diluted in 450 µL of 10 mM Tris–HCl, pH 8.0, heat denatured at 98°C for 10 min and immediately cooled on ice for 10 min. A portion of the untreated, sonicated DNA was left to serve as the input control. The CpG-methylated DNA fragments were precipitated with anti-5-methyl cytidine antibody (Eurogentec cat. no. BI-MECY-1000), and the antibody–DNA complexes were captured with Dynabeads (Invitrogen, cat. no. 11202). The collected beads were washed in order to remove non-specifically-bound DNA, following the treatment with proteinase K for 20–24 h at 50°C in order to remove proteins. The CpG-methylated DNA was extracted with the phenol:chloroform method, precipitated with ethanol and glycogen, and resuspended in 60 μl of 10 mM Tris–Cl, pH 8.0.</p>                            <h3 scrollto-destination=6237325 id="6237325" class="section-title" data-legacy-id=s2c>2.3. DNA Labelling and Hybridization</h3>
<p>DNA labelling and hybridization was performed according to NimbleGen's protocol with our modifications.<sup><a reveal-id="DST034C27" data-open="DST034C27" class="link link-ref link-reveal xref-bibr">27</a></sup> The immunoprecipitated CpG-methylated DNA (test) and the untreated, sonicated DNA (input control) were labelled by using the random priming with the Nimblegen Dual-Color DNA Labelling kit (Roche-Nimblegen, cat. no. 06370250001) with fluorescent dyes Cy3 and Cy5, respectively. The combined test and input DNA samples were suspended in hybridization buffer (Roche, cat. no. 05583683001) co-hybridized onto Mouse DNA Methylation 3 × 720 K CpG Island Plus RefSeq Promoter Arrays for 20 h at 42°C, following washing with the Nimblegen Wash Kit (Roche cat. no. 05584507001).</p>                            <h3 scrollto-destination=6237327 id="6237327" class="section-title" data-legacy-id=s2d>2.4. Microarray data acquisition and processing</h3>
<p>We performed image acquisition with an MS200 Scanner (Roche, NimbleGen) at 2 µm resolution by using high-sensitivity and autogain settings. The data from scanned images were extracted and processed with DEVA v. 1.0.2 (Roche, Nimblegen) using default parameters. Data processing included obtaining log<sub>2</sub> ratios, <em>P</em>-scores, and peak identification. Log<sub>2</sub> ratios represent the ratios of the immunoprecipitated DNA signal to the input DNA signal. The mean log<sub>2</sub> ratios were calculated using a Tukey biweight mean. <em>P</em>-scores were calculated from log<sub>2</sub> ratios by performing the sliding-window (750 bp) Kolmogorov–Smirnov (KS) test around each probe applied to determine a −log10 <em>P</em>-value from the windowed KS test around that probe. The methylation peaks were obtained by merging consecutive probes with <em>P</em>-scores over the assumed cut-off (default 2.0), with maximum 500 bp spacing and minimum of two probes per peak. The processed data (<em>P</em>-scores) and raw data were deposited in the Gene Expression Omnibus (GEO) database under the accession number GSE49221.</p>                            <h3 scrollto-destination=6237329 id="6237329" class="section-title" data-legacy-id=s2e>2.5. Identification of differentially methylated genes</h3>
<p>The methylation peaks were mapped to features (transcription start sites, primary transcripts, CpG islands, and other tiled regions) using DEVA v. 1.0.2, assuming the default distances 5000 bp upstream and 1000 bp downstream the feature The annotated data for the methylation peaks obtained at cut-off of 1.0 and 2.0 were merged using GenixNet, a script created by Dr P. Madanecki and adjusted to the analysed datasets, so that a single table containing all genes, tiled regions, and CpG islands with the methylation peak values for each strain and tissue at two different cut-off points was created. The data merged using GenixNet were sorted to identify the differentially methylated genes using the Excel spreadsheet. If a peak was mapped to more than one feature (e.g. transcription start site and primary transcript), the data listed in the tables of differentially methylated peaks (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) were restricted to a single feature selected according to the following order: transcription start site, primary transcript, CpG island, and tiled region.</p>                            <h3 scrollto-destination=6237331 id="6237331" class="section-title" data-legacy-id=s2f>2.6. Validation of the MeDIP/microarray results and gene expression level quantification</h3>
<p>The methylation status of selected CpG nucleotides was evaluated using bisulphite sequencing and/or genomic DNA digestion with a CpG methylation-sensitive <em>Hpa</em>II restriction endonuclease. Gene expression levels were quantified using real-time PCR.</p><p>The bisulphite conversion of genomic DNA was carried out with the EZ DNA Methylation™ Kit (Zymo Research, cat. no. D5002) using 500 ng of genomic DNA. Approximately 100 ng of obtained converted DNA was used for the subsequent PCR reaction with the Maxima Hot Start Taq DNA polymerase system (ThermoScientificBio, cat. no. EP0601). The PCR reactions were performed on Veriti<sup>®</sup> 96-well Thermal Cycler (Applied Biosystems). The PCR amplified DNA fragments were extracted from the agarose gel and purified using the ISOLATE Gel and PCR kit (Bioline, cat. no. BIO-52029). The PCR products were sequenced using the Applied Biosystems ABI 3730XL/ABI3700 DNA sequencer (Genomed, Poland), and the chromatograms were analysed with ContigExpress (VectorNTI 11.0, Invitrogen).</p><p>The CpG methylation-sensitive digestion of genomic DNA was carried out with the EpiJET DNA Methylation Analysis Kit based on <em>Msp</em>I/<em>Hpa</em>II digestion (ThermoScientificBio, cat. no. K1441) using 500 ng of genomic DNA and 50 ng of digested DNA for quantitative real-time PCR analysis. The methylation level was quantified using the 2-ΔCt method and expressed as the ratio between <em>HpaII-</em>digested DNA (target) and input/non-digested DNA (reference) used for enzymatic reaction. cDNA synthesis was performed with Maxima Reverse Transcriptase (ThermoScientificBio, cat. no. EP0742) using 200 ng of total RNA and oligo dT<sub>20</sub>. Approximately 10 ng of cDNA was used for subsequent real-time PCR reactions. The gene expression levels were calculated using the 2-ΔCt method with <em>Actb</em> as the reference gene. The results are presented as means ± SD.</p><p>Real-time PCR reactions were carried out with FastStart Essential DNA Green Master (Roche, cat. no. 06402712001) on LightCycler<sup>®</sup> Nano (Roche). The primer sequences are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>.</p>                            <h3 scrollto-destination=6237336 id="6237336" class="section-title" data-legacy-id=s2g>2.7. Linear regression and scatter plots</h3>
<p>Linear regression coefficients were determined, and scatter plots were made using the XLSTAT package (Addinsoft).</p>                            <h3 scrollto-destination=6237338 id="6237338" class="section-title" data-legacy-id=s2h>2.8. Chromosomal ideograms with the locations of differentially methylated genes</h3>
<p>The chromosomal maps with the locations of differentially methylated genes were plotted using Idiographica v. 2.1 (<a class="link link-uri" href="http://www.ncrna.org/idiographica">http://www.ncrna.org/idiographica</a>).<sup><a reveal-id="DST034C28" data-open="DST034C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p>                            <h3 scrollto-destination=6237340 id="6237340" class="section-title" data-legacy-id=s2i>2.9. Counting the DNA methylation peaks at different value ranges</h3>
<p>The lists of unique peaks were obtained from the lists of mapped peaks generated by DEVA v. 1.0.2. The numbers of peaks at different methylation values were calculated in Excel spreadsheet.</p>                            <h3 scrollto-destination=6237342 id="6237342" class="section-title" data-legacy-id=s2j>2.10. Gene set enrichment analysis</h3>
<p>The gene set enrichment analyses were performed and functional annotations were assigned by using DAVID Bioinformatics Resource 6.7 [National Institute of Allergy and Infectious Diseases (NIAID), National Institute of Health<sup><a reveal-id="DST034C29" data-open="DST034C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DST034C30" data-open="DST034C30" class="link link-ref link-reveal xref-bibr">30</a></sup>]. An EASE Score, a modified Fisher's Exact test, was applied to calculate statistical significance of gene set enrichments.</p>                            <h3 scrollto-destination=6237344 id="6237344" class="section-title" data-legacy-id=s2k>2.11. Processing and analysis of genomic DNA methylation microarray data downloaded from the Gene Omnibus Expression Database</h3>
                            <h4 scrollto-destination=6237345 id="6237345" class="section-title" data-legacy-id=s2k1>2.11.1. Gene expression data</h4>
<p>The microarray data on gene expression in the heart and liver of the MRL/MpJ mouse were retrieved from GSE19322, GSE4710, GSE25322, and GSE5241 datasets deposited in the GEO database. The gene expression values were merged with the annotations from the corresponding platform files in Excel spreadsheet. The logarithmed signal values were exponentiated. Average values were calculated from replicates. To calculate median and quartile expression values for selected gene subsets (e.g. the genes hypomethylated in the MRL/MpJ mouse), the expression values were extracted in Excel spreadsheet using gene identifiers. Gene expression values between the subsets of genes selected according to their methylation levels were compared by Wilcoxon rank-sum two-tailed tests using the SAS statistical package version 9.3 (SAS Institute, Inc., Cary, NC, USA).</p>                            <h4 scrollto-destination=6237347 id="6237347" class="section-title" data-legacy-id=s2k2>2.11.2. Gene methylation data</h4>
<p>Processed DNA methylation data (<em>P</em>-scores) were downloaded from the GSE21415 dataset deposited in the GEO database. The genome coordinates were converted from the NCBI MM8 build to the NCBI MM9 build using ‘Lift Genome Annotations’ service from the University of California Santa Cruz Genome browser (<a class="link link-uri" href="http://genome.ucsc.edu/cgi-bin/hgLiftOver">http://genome.ucsc.edu/cgi-bin/hgLiftOver</a>). The methylation peaks were obtained with NimbleScan 2.6 (Roche, Nimblegen) at cut-off of 1.0 using default parameters (at least two probes per peak with maximum spacing of 500 bp). Average values were calculated from replicates. The methylation peaks were mapped to gene features using DEVA v. 1.0.2, assuming the standard distance 5000 bp upstream and 1000 bp downstream the feature. Average values were used for the features with more than one methylation peak.</p>                            <h3 scrollto-destination=6237349 id="6237349" class="section-title" data-legacy-id=s2l>2.12. Analysis of correlations between DNA methylation and gene expression data</h3>
<p>Gene expression and methylation data were merged using GenixNet. Pearson's correlation coefficients were calculated using the SAS statistical package version 9.3 (SAS Institute, Inc.).</p>                            <h3 scrollto-destination=6237351 id="6237351" class="section-title" data-legacy-id=s2m>2.13. Analysis of correlations between genome-wide DNA methylation profiles</h3>
<p>As the formats of microarray platforms applied in this study and that by Liang <em>et al</em>. are different, we restricted the analysis to the peaks mapped to the transcription start sites shared by both platforms (18 199 unique genes included in Roche MM8 385 K RefSeq Promoter Array). If more than a single peak was mapped to a gene, average values were calculated by using Excel ‘pivot tables’ function. The methylation peaks for this analysis were obtained at cut-off of 1.0 in order to reduce the number of ‘0’ under the cut-off, which, in fact, represent a spectrum of values. The peaks under the cut-off of 1.0 were given the value ‘0’. The arrays were merged using Entrez identifiers. Pearson's correlation coefficients were calculated using the SAS statistical package version 9.3 (SAS Institute, Inc.).</p>                            <h2 scrollto-destination=6237353 id="6237353" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
<p>We investigated the epigenetic basis of the regeneration phenomenon in the MRL/MpJ mouse using genome-wide DNA methylation profiling. As the MRL/MpJ strain does not have an immediate relative, which does not exhibit enhanced regeneration abilities, we chose the C57BL/6J mouse as the reference strain. We carried out genome-wide DNA methylation profiling in hearts, livers, and spleens of the MRL/MpJ and C57BL/6J strains using the MeDIP approach followed by microarray analysis. The Nimblegen microarray system ‘3 × 720 K CpG Island Plus RefSeq Promoter’ we applied in the study includes 20 404 promoter regions; 22 881 transcripts and 15 980 CpG Islands. The DNA methylation profiles we obtained for the MRL/MpJ and C57BL/6J were compared in order to identify the differentially methylated genes and regions. The overall comparisons of inter-strain similarities and differences between the examined genomic DNA methylation profiles are visualized using scatter plots (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p><p>This study was designed as an initial analysis of genome-wide DNA methylation patterns and we restricted the selection of tissues to heart, liver, and spleen. Heart and liver were selected to contrast the non-regenerating organ with the regenerating one. Spleen was chosen for its role in the immune system, as immune response to injury seems to be one of the most critical events in the regeneration process.<sup><a reveal-id="DST034C31" data-open="DST034C31" class="link link-ref link-reveal xref-bibr">31</a></sup></p>                            <h3 scrollto-destination=6237356 id="6237356" class="section-title" data-legacy-id=s3a>3.1. Genes differentially methylated in the MRL/MpJ mouse</h3>
<p>As the DNA methylation patterns are prone to individual and conditional variation, we focused on the analysis of the genes revealing the most distinguishing differences in the methylation status. We assumed a gene/region to be differentially methylated if a methylation peak mapped to this gene/region meets at least one of two conditions: either the peak value is at least 3.0 or higher and it is at least twice higher than in the reference (at cut-off of 1.0) or it is at least 2.0 and it is lower than 1.0 for the reference (see Section 2 for the definition of peak value). A peak is mapped to a feature (transcription start site, primary transcript, CpG island, or tiled region) if it is located within 5000 bp upstream and 1000 bp downstream the region (a transcription start site, primary transcript, or CpG island).</p><p>As the result several hundreds of differentially methylated genes, we will refer to as ‘hypomethylated’ and ‘hypermethylated’, were identified in liver, spleen, and heart tissue of the MRL/MpJ mouse (Fig. <a reveal-id="DST034F1" data-open="DST034F1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The majority of differentially methylated genes and regions were not shared by the three examined tissues: heart, liver, and spleen. Twenty-three genes and CpG islands were found to be hypomethylated and 14 to be hypermethylated in the MRL/MpJ mouse in the three examined tissues. These genes and CpG islands were located on all chromosomes except the 18, 19, X, and Y (Fig. <a reveal-id="DST034F2" data-open="DST034F2" class="link link-reveal link-table xref-fig">2</a>), and they are associated with a number of different functions. The complete list and the descriptions of these genes are presented in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>, while a selection of remarkable representatives of this group (Table <a reveal-id="DST034TB1" data-open="DST034TB1" class="link link-reveal link-table xref-fig">1</a>) are discussed later in this article. </p>                        <a id="6237359" scrollto-destination="6237359"></a>
<div content-id="DST034TB1" class="table-modal"><div class="table-wrap"><div id="DST034TB1" class="table-wrap-title" data-id="DST034TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The remarkable representatives of the genes differentially methylated in the MRL/MpJ mouse</p></div></div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03407a.gif?Expires=2147483647&amp;Signature=kHzs4smn50isVHH~axK-V6u1kmdFv6lOBDDRbw5FaeGPzKebQt3BChlA2Pihrz2fXyeSpbtCvcVC~5VFVqcNCMNGNzP0I~hGtnzEg8-ckYIiYhVdNW3HBeRofxCbxoIk4LmqnaFf3-EyyiD1uo~CwdtxZOyAWoERSozrDRtQbc~zX0s3vXswD~-hn0NwjwY3zvfy8JB92q~kvd8ml44orHiVdbUQmPT~5DCaAphvW1GVCCIOPRbA7x9QnQ9yucvFMOLp-~IZAKtUPXK6vkZ7stKfv5k2E4WDb9yHBfXjY3Lk94a0ZdRp3pmfvCwPNPN0xEVg8BJ5vyKBtqRCgnu97g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dst03407a.gif" /> </td></tr><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03407b.gif?Expires=2147483647&amp;Signature=1M8tNViVYvkkn1fRuos2xmMclWrh2-Bljv0GXqZmNf9yNRLhbAOHXRs9pkdHKx8UD4NKm-1fk391mZA6AtsLR3LBGUeLimAtyE~TGI~BQIx3oxLxORS3yGnK7ES7bYW-En4MXYK6t-5DsPqE8SNo9InUPszhiLLuC61f-EVMVPVbA8aGiEW2O4kqPCuHaJ0n-pJz0gvqE0JOY-dZGapH5AXkqhEmIeTLC85B~sXAGGLAoThkhrI0eIx8DBIGbPrthEnMKkCe5egAw9V7QpgA11elqJf3JKL74Conpbgh9AFRIw7QV~1oeLAL4097BR4xFuJvGV~lr9eTVJDZrQ8nJQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dst03407b.gif" /> </td></tr></tbody></table></div><div content-id="DST034TB1" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03407a.gif?Expires=2147483647&amp;Signature=kHzs4smn50isVHH~axK-V6u1kmdFv6lOBDDRbw5FaeGPzKebQt3BChlA2Pihrz2fXyeSpbtCvcVC~5VFVqcNCMNGNzP0I~hGtnzEg8-ckYIiYhVdNW3HBeRofxCbxoIk4LmqnaFf3-EyyiD1uo~CwdtxZOyAWoERSozrDRtQbc~zX0s3vXswD~-hn0NwjwY3zvfy8JB92q~kvd8ml44orHiVdbUQmPT~5DCaAphvW1GVCCIOPRbA7x9QnQ9yucvFMOLp-~IZAKtUPXK6vkZ7stKfv5k2E4WDb9yHBfXjY3Lk94a0ZdRp3pmfvCwPNPN0xEVg8BJ5vyKBtqRCgnu97g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dst03407a.gif" /> </td></tr><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03407b.gif?Expires=2147483647&amp;Signature=1M8tNViVYvkkn1fRuos2xmMclWrh2-Bljv0GXqZmNf9yNRLhbAOHXRs9pkdHKx8UD4NKm-1fk391mZA6AtsLR3LBGUeLimAtyE~TGI~BQIx3oxLxORS3yGnK7ES7bYW-En4MXYK6t-5DsPqE8SNo9InUPszhiLLuC61f-EVMVPVbA8aGiEW2O4kqPCuHaJ0n-pJz0gvqE0JOY-dZGapH5AXkqhEmIeTLC85B~sXAGGLAoThkhrI0eIx8DBIGbPrthEnMKkCe5egAw9V7QpgA11elqJf3JKL74Conpbgh9AFRIw7QV~1oeLAL4097BR4xFuJvGV~lr9eTVJDZrQ8nJQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dst03407b.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>If not indicated otherwise, the information on gene functions is taken from DAVID and Gene Entrez (E, <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/gene/">http://www.ncbi.nlm.nih.gov/gene/</a>).</p></div></div><div content-id=""><div class="fn"><p>H: heart; L: liver; S: spleen; TSS: transcription start site; PT: primary transcript.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6237359" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST034TB1" class="table-wrap-title" data-id="DST034TB1"><div class="label -label">Table 1.</div><div class="caption"><p>The remarkable representatives of the genes differentially methylated in the MRL/MpJ mouse</p></div></div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03407a.gif?Expires=2147483647&amp;Signature=kHzs4smn50isVHH~axK-V6u1kmdFv6lOBDDRbw5FaeGPzKebQt3BChlA2Pihrz2fXyeSpbtCvcVC~5VFVqcNCMNGNzP0I~hGtnzEg8-ckYIiYhVdNW3HBeRofxCbxoIk4LmqnaFf3-EyyiD1uo~CwdtxZOyAWoERSozrDRtQbc~zX0s3vXswD~-hn0NwjwY3zvfy8JB92q~kvd8ml44orHiVdbUQmPT~5DCaAphvW1GVCCIOPRbA7x9QnQ9yucvFMOLp-~IZAKtUPXK6vkZ7stKfv5k2E4WDb9yHBfXjY3Lk94a0ZdRp3pmfvCwPNPN0xEVg8BJ5vyKBtqRCgnu97g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dst03407a.gif" /> </td></tr><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03407b.gif?Expires=2147483647&amp;Signature=1M8tNViVYvkkn1fRuos2xmMclWrh2-Bljv0GXqZmNf9yNRLhbAOHXRs9pkdHKx8UD4NKm-1fk391mZA6AtsLR3LBGUeLimAtyE~TGI~BQIx3oxLxORS3yGnK7ES7bYW-En4MXYK6t-5DsPqE8SNo9InUPszhiLLuC61f-EVMVPVbA8aGiEW2O4kqPCuHaJ0n-pJz0gvqE0JOY-dZGapH5AXkqhEmIeTLC85B~sXAGGLAoThkhrI0eIx8DBIGbPrthEnMKkCe5egAw9V7QpgA11elqJf3JKL74Conpbgh9AFRIw7QV~1oeLAL4097BR4xFuJvGV~lr9eTVJDZrQ8nJQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dst03407b.gif" /> </td></tr></tbody></table></div><div content-id="DST034TB1" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03407a.gif?Expires=2147483647&amp;Signature=kHzs4smn50isVHH~axK-V6u1kmdFv6lOBDDRbw5FaeGPzKebQt3BChlA2Pihrz2fXyeSpbtCvcVC~5VFVqcNCMNGNzP0I~hGtnzEg8-ckYIiYhVdNW3HBeRofxCbxoIk4LmqnaFf3-EyyiD1uo~CwdtxZOyAWoERSozrDRtQbc~zX0s3vXswD~-hn0NwjwY3zvfy8JB92q~kvd8ml44orHiVdbUQmPT~5DCaAphvW1GVCCIOPRbA7x9QnQ9yucvFMOLp-~IZAKtUPXK6vkZ7stKfv5k2E4WDb9yHBfXjY3Lk94a0ZdRp3pmfvCwPNPN0xEVg8BJ5vyKBtqRCgnu97g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dst03407a.gif" /> </td></tr><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03407b.gif?Expires=2147483647&amp;Signature=1M8tNViVYvkkn1fRuos2xmMclWrh2-Bljv0GXqZmNf9yNRLhbAOHXRs9pkdHKx8UD4NKm-1fk391mZA6AtsLR3LBGUeLimAtyE~TGI~BQIx3oxLxORS3yGnK7ES7bYW-En4MXYK6t-5DsPqE8SNo9InUPszhiLLuC61f-EVMVPVbA8aGiEW2O4kqPCuHaJ0n-pJz0gvqE0JOY-dZGapH5AXkqhEmIeTLC85B~sXAGGLAoThkhrI0eIx8DBIGbPrthEnMKkCe5egAw9V7QpgA11elqJf3JKL74Conpbgh9AFRIw7QV~1oeLAL4097BR4xFuJvGV~lr9eTVJDZrQ8nJQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dst03407b.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>If not indicated otherwise, the information on gene functions is taken from DAVID and Gene Entrez (E, <a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/gene/">http://www.ncbi.nlm.nih.gov/gene/</a>).</p></div></div><div content-id=""><div class="fn"><p>H: heart; L: liver; S: spleen; TSS: transcription start site; PT: primary transcript.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6237359" target="_blank">View Large</a></div></div>                        <a id="6237360" scrollto-destination="6237360"></a>
<div id="DST034F1" data-id="DST034F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03401.png?Expires=2147483647&Signature=ViKzmv3ZjQwnJmbGczQW6Y7cRAFO2YxuIemAO0tJe5GHiu8CPn75mbz8mO8viXoYm8pVZ5-WXRSl7EmTNkdjhzu~k4zAU0nUeg9tDoehZ6mescBLMwgkNSboKKE~piYu5c8DC0nAijtF3ozWj1ueJDJGkopjJkZVrk3G9wZAcipRY3WbYCuhiUO-vDLFAhuXwJhbh06Cyv4-Bgak~sVn2KBDbpKeW8xPFkNY5i9OvshIytJtjKVvIQzgoPvfp9LVl8pSSip6qEqnte2h58nBghvWtD9VVK6V6FQPkz-ngTrVq-PrNhTIgzTDJ0xOwPimbZYL60IRa~lE7iR09sRsmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The numbers of differentially methylated genes and CpG islands in the MRL/MpJ mouse in the heart, liver, and spleen tissues. The numbers of CpGs distant from the genes (out of −5000 bp and + 1000 bp range) in brackets. It is assumed here that a gene/region is differentially methylated if it meets at least one of two conditions: either (i) the peak value is at least 3.0 or higher and it is at least twice higher than in the reference or (ii) it is at least 2.0 and it is lower than 1.0 for the reference." path-from-xml="dst03401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237360/dst03401.tiff" path-from-xml="dst03401.tiff" target="_blank">View large</a><a section="6237360" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst03401.png?Expires=2147483647&Signature=WYbWpR4h0erQEM1Ir6UdC3L4gyGEVAJs-YFN3Kt8Y~I4oJKlujuXzLd1EfgkcZX7lDi~zNyr4g~hkCrudo5s2yYq2lSrbmUhED-MR6cZ0TfEmZSwgygZtnlS8HoUFyBB6E0yWLZJhGK7-D5TBXNxwnQNMQSpQ3TUEqrTHNfE~AegUpeqQrNITzEzEuc~uLwGjJIMA5EuDOcse6J4EsIbrTjl20klvxmhoVEaiFRYPrbMLN5cDK8nwvQ0F3fJ3n2pWvlMrxGNIfGmQ70O36BOw-umdkq2PUNMPVOZLTwIBa98yI4uz2l9fFNhAsLpBURAmeqD732M83Z5AksIZEdncA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237360&ar=420907&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The numbers of differentially methylated genes and CpG islands in the MRL/MpJ mouse in the heart, liver, and spleen tissues. The numbers of CpGs distant from the genes (out of −5000 bp and + 1000 bp range) in brackets. It is assumed here that a gene/region is differentially methylated if it meets at least one of two conditions: either (i) the peak value is at least 3.0 or higher and it is at least twice higher than in the reference or (ii) it is at least 2.0 and it is lower than 1.0 for the reference.</p></div></div><div content-id="DST034F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03401.png?Expires=2147483647&Signature=ViKzmv3ZjQwnJmbGczQW6Y7cRAFO2YxuIemAO0tJe5GHiu8CPn75mbz8mO8viXoYm8pVZ5-WXRSl7EmTNkdjhzu~k4zAU0nUeg9tDoehZ6mescBLMwgkNSboKKE~piYu5c8DC0nAijtF3ozWj1ueJDJGkopjJkZVrk3G9wZAcipRY3WbYCuhiUO-vDLFAhuXwJhbh06Cyv4-Bgak~sVn2KBDbpKeW8xPFkNY5i9OvshIytJtjKVvIQzgoPvfp9LVl8pSSip6qEqnte2h58nBghvWtD9VVK6V6FQPkz-ngTrVq-PrNhTIgzTDJ0xOwPimbZYL60IRa~lE7iR09sRsmQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The numbers of differentially methylated genes and CpG islands in the MRL/MpJ mouse in the heart, liver, and spleen tissues. The numbers of CpGs distant from the genes (out of −5000 bp and + 1000 bp range) in brackets. It is assumed here that a gene/region is differentially methylated if it meets at least one of two conditions: either (i) the peak value is at least 3.0 or higher and it is at least twice higher than in the reference or (ii) it is at least 2.0 and it is lower than 1.0 for the reference." path-from-xml="dst03401.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237360/dst03401.tiff" path-from-xml="dst03401.tiff" target="_blank">View large</a><a section="6237360" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst03401.png?Expires=2147483647&Signature=WYbWpR4h0erQEM1Ir6UdC3L4gyGEVAJs-YFN3Kt8Y~I4oJKlujuXzLd1EfgkcZX7lDi~zNyr4g~hkCrudo5s2yYq2lSrbmUhED-MR6cZ0TfEmZSwgygZtnlS8HoUFyBB6E0yWLZJhGK7-D5TBXNxwnQNMQSpQ3TUEqrTHNfE~AegUpeqQrNITzEzEuc~uLwGjJIMA5EuDOcse6J4EsIbrTjl20klvxmhoVEaiFRYPrbMLN5cDK8nwvQ0F3fJ3n2pWvlMrxGNIfGmQ70O36BOw-umdkq2PUNMPVOZLTwIBa98yI4uz2l9fFNhAsLpBURAmeqD732M83Z5AksIZEdncA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237360&ar=420907&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03401.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The numbers of differentially methylated genes and CpG islands in the MRL/MpJ mouse in the heart, liver, and spleen tissues. The numbers of CpGs distant from the genes (out of −5000 bp and + 1000 bp range) in brackets. It is assumed here that a gene/region is differentially methylated if it meets at least one of two conditions: either (i) the peak value is at least 3.0 or higher and it is at least twice higher than in the reference or (ii) it is at least 2.0 and it is lower than 1.0 for the reference.</p></div></div>                        <a id="6237361" scrollto-destination="6237361"></a>
<div id="DST034F2" data-id="DST034F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03402.png?Expires=2147483647&Signature=GhmhI~6gKmT1t8mt9g7Jw8XmisqNuqwNQ2ea85~6NIkPSwc2vT23OTVHHTqfqc9CXVup1DQ~DyxxeEDbfiWV0yGwG2DxuhEFq~jLL-cTWZ6p5u5e-fF79hJtTTTyIa87pbwJIBMiZiEI-9EE5R7hrWWI9SjUcM9V40294DH1jxXhdglN-v15ajoxqL98q16b1LQo1Ah-FRFMaVXUuZgOMxohHodo9-qtOb3uj19vAfsiZE3KS0xzphlFQ-PUVCeM~BGFxSLrNSzdaBN4cNiodTrWvGf-ewWH41klaKu744ROA~uuV5Unm75XKvbbe31S-i91PYRHNMsJc94k-qum0A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The chromosomal locations of genes and CpG islands differentially methylated in the MRL/MpJ mouse. (A) The genes and CpG islands that are differentially methylated in the three examined tissues: heart, liver, and spleen. PT - DNA methylation peak mapped to a primary transcript, kb—distance in kb is given if the methylation peak is located out of the range 5000 bp upstream and 1000 bp downstream the feature. (B) The genes and CpG islands differentially methylated in the heart, liver, and spleen. The genes displaying substantially higher and lower methylation degree in the MRL/MpJ mouse are marked red (hypermethylated, grey in the print) and blue (hypomethylated, black in the print), respectively. The values and genomic coordinates of the DNA methylation peaks as well as the descriptions of mapped of genes are listed in Supplementary Table S1. Heart vs. liver analysis singled out the genes and CpG islands, which have different methylation status in the heart than in the liver of the MRL/MpJ, but not the C57BL/6J, mouse. (see the chapter: ‘Genes with similar DNA methylation status in the heart and liver of the MRL/MpJ but not the C57BL/6J mouse’)." path-from-xml="dst03402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237361/dst03402.tiff" path-from-xml="dst03402.tiff" target="_blank">View large</a><a section="6237361" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst03402.png?Expires=2147483647&Signature=rwLgDUwe1fWKUjodP4ImvdhYKGMVQp5SkFWjJHjEcqCLiF4Sq~s5z3qxQVCsH3i~T7vV8xx0RNx-yYJ-bxMUb5I~qwchE-D9QCSzcXNXPMPBOicf2iDvAbt7Do0RQPbXpW0ud15qKByFlvXjIxrNXp1y3mDjjCumh2UlXz~uC0sc-lIf526lhWl68nDYX-2-60GtmVqrYoUPZ2P8n2fR2-kcGLv80wOZQGTxUwtH5ApQqJSft-46UEWfaniz0lev5l-yNO93wZhs80t5CBTz-i67~sGYk-Lbn2bbrylvAAUItxWDtM~23GEjTyLVCOzc3viVTxZCFeOzpqvpNbt~-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237361&ar=420907&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The chromosomal locations of genes and CpG islands differentially methylated in the MRL/MpJ mouse. (A) The genes and CpG islands that are differentially methylated in the three examined tissues: heart, liver, and spleen. PT - DNA methylation peak mapped to a primary transcript, kb—distance in kb is given if the methylation peak is located out of the range 5000 bp upstream and 1000 bp downstream the feature. (B) The genes and CpG islands differentially methylated in the heart, liver, and spleen. The genes displaying substantially higher and lower methylation degree in the MRL/MpJ mouse are marked red (hypermethylated, grey in the print) and blue (hypomethylated, black in the print), respectively. The values and genomic coordinates of the DNA methylation peaks as well as the descriptions of mapped of genes are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. Heart vs. liver analysis singled out the genes and CpG islands, which have different methylation status in the heart than in the liver of the MRL/MpJ, but not the C57BL/6J, mouse. (see the chapter: ‘Genes with similar DNA methylation status in the heart and liver of the MRL/MpJ but not the C57BL/6J mouse’).</p></div></div><div content-id="DST034F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03402.png?Expires=2147483647&Signature=GhmhI~6gKmT1t8mt9g7Jw8XmisqNuqwNQ2ea85~6NIkPSwc2vT23OTVHHTqfqc9CXVup1DQ~DyxxeEDbfiWV0yGwG2DxuhEFq~jLL-cTWZ6p5u5e-fF79hJtTTTyIa87pbwJIBMiZiEI-9EE5R7hrWWI9SjUcM9V40294DH1jxXhdglN-v15ajoxqL98q16b1LQo1Ah-FRFMaVXUuZgOMxohHodo9-qtOb3uj19vAfsiZE3KS0xzphlFQ-PUVCeM~BGFxSLrNSzdaBN4cNiodTrWvGf-ewWH41klaKu744ROA~uuV5Unm75XKvbbe31S-i91PYRHNMsJc94k-qum0A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The chromosomal locations of genes and CpG islands differentially methylated in the MRL/MpJ mouse. (A) The genes and CpG islands that are differentially methylated in the three examined tissues: heart, liver, and spleen. PT - DNA methylation peak mapped to a primary transcript, kb—distance in kb is given if the methylation peak is located out of the range 5000 bp upstream and 1000 bp downstream the feature. (B) The genes and CpG islands differentially methylated in the heart, liver, and spleen. The genes displaying substantially higher and lower methylation degree in the MRL/MpJ mouse are marked red (hypermethylated, grey in the print) and blue (hypomethylated, black in the print), respectively. The values and genomic coordinates of the DNA methylation peaks as well as the descriptions of mapped of genes are listed in Supplementary Table S1. Heart vs. liver analysis singled out the genes and CpG islands, which have different methylation status in the heart than in the liver of the MRL/MpJ, but not the C57BL/6J, mouse. (see the chapter: ‘Genes with similar DNA methylation status in the heart and liver of the MRL/MpJ but not the C57BL/6J mouse’)." path-from-xml="dst03402.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237361/dst03402.tiff" path-from-xml="dst03402.tiff" target="_blank">View large</a><a section="6237361" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst03402.png?Expires=2147483647&Signature=rwLgDUwe1fWKUjodP4ImvdhYKGMVQp5SkFWjJHjEcqCLiF4Sq~s5z3qxQVCsH3i~T7vV8xx0RNx-yYJ-bxMUb5I~qwchE-D9QCSzcXNXPMPBOicf2iDvAbt7Do0RQPbXpW0ud15qKByFlvXjIxrNXp1y3mDjjCumh2UlXz~uC0sc-lIf526lhWl68nDYX-2-60GtmVqrYoUPZ2P8n2fR2-kcGLv80wOZQGTxUwtH5ApQqJSft-46UEWfaniz0lev5l-yNO93wZhs80t5CBTz-i67~sGYk-Lbn2bbrylvAAUItxWDtM~23GEjTyLVCOzc3viVTxZCFeOzpqvpNbt~-g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237361&ar=420907&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03402.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The chromosomal locations of genes and CpG islands differentially methylated in the MRL/MpJ mouse. (A) The genes and CpG islands that are differentially methylated in the three examined tissues: heart, liver, and spleen. PT - DNA methylation peak mapped to a primary transcript, kb—distance in kb is given if the methylation peak is located out of the range 5000 bp upstream and 1000 bp downstream the feature. (B) The genes and CpG islands differentially methylated in the heart, liver, and spleen. The genes displaying substantially higher and lower methylation degree in the MRL/MpJ mouse are marked red (hypermethylated, grey in the print) and blue (hypomethylated, black in the print), respectively. The values and genomic coordinates of the DNA methylation peaks as well as the descriptions of mapped of genes are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. Heart vs. liver analysis singled out the genes and CpG islands, which have different methylation status in the heart than in the liver of the MRL/MpJ, but not the C57BL/6J, mouse. (see the chapter: ‘Genes with similar DNA methylation status in the heart and liver of the MRL/MpJ but not the C57BL/6J mouse’).</p></div></div>                            <h3 scrollto-destination=6237362 id="6237362" class="section-title" data-legacy-id=s3b>3.2. Validation of the MeDIP/microarray results</h3>
<p>A selection of DNA methylation peaks were validated in the MRL/MpJ and the C57BL/6J reference using bisulphite sequencing and/or digestion with a CpG methylation-sensitive restriction endonuclease <em>Hpa</em>II, followed by quantitative real-time PCR analysis [<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a> (summarized results), <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary File S1</a> (experiment details), and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary File S2</a> (representative DNA sequencing chromatograms assembled with the reference sequences)]. Gene expression levels of selected genes were examined by quantitative real-time PCR analysis [<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a> (summarized results) and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary File S1</a> (experiment details)].</p>                            <h3 scrollto-destination=6237364 id="6237364" class="section-title" data-legacy-id=s3c>3.3. The chromosomal locations of differentially methylated regions</h3>
<p>The hyper- and hypomethylated genes and regions were found on all chromosomes. Several chromosomal loci display significant positional enrichments of the genes differentially methylated in the MRL/MpJ mouse, but no remarkable regions of peak accumulation could be distinguished (Fig. <a reveal-id="DST034F2" data-open="DST034F2" class="link link-reveal link-table xref-fig">2</a>A and B).</p>                            <h3 scrollto-destination=6237366 id="6237366" class="section-title" data-legacy-id=s3d>3.4. Gene methylation status and gene expression</h3>
<p>DNA methylation of gene promoters is known to repress gene expression. With regard to evaluating the impact of DNA methylation on gene expression in the examined tissues, we analysed the available expression microarray data for the MRL/MpJ heart and liver tissues reported in four different studies.<sup><a reveal-id="DST034C16" data-open="DST034C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DST034C17" data-open="DST034C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DST034C37" data-open="DST034C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DST034C38" data-open="DST034C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Our evaluation was based on the correlations between DNA methylation status and expression signals, as well as mean values of expression signals calculated for subsets of genes at different peak value ranges. We found that the expression signal values were increasing with the decrease of DNA methylation status (Fig. <a reveal-id="DST034F3" data-open="DST034F3" class="link link-reveal link-table xref-fig">3</a>A), and the values of methylation peaks and gene expression signals were reversely correlated (Fig. <a reveal-id="DST034F3" data-open="DST034F3" class="link link-reveal link-table xref-fig">3</a>B). The Pearson's correlation coefficients for gene methylation and expression values in the heart though significant for both the C57BL/6J and MRL/MpJ mouse were lower for the latter one. This analysis confirmed that the obtained genomic DNA methylation patterns are adequate to the transcriptomic profiles. However, the gene expression signals in hearts for the groups of genes differentially methylated in the MRL/MpJ and C57BL/6J did not show statistically significant differences between these two strains (Fig. <a reveal-id="DST034F3" data-open="DST034F3" class="link link-reveal link-table xref-fig">3</a>A). This inconsistence could be explained by age differences, as we used DNA methylation data obtained for 5-week-old animals and the gene expression data had been collected for much older mice (8- to 20-week old). It should also be noted that the majority of DNA methylation peaks mapped to the differentially methylated genes in the MRL/MpJ are located at higher distance than those in the C57BL/6J mouse (Fig. <a reveal-id="DST034F6" data-open="DST034F6" class="link link-reveal link-table xref-fig">6</a>B). After all, in spite of the age differences, the overall correlations between DNA methylation and gene expression were confirmed. </p>                        <a id="6237368" scrollto-destination="6237368"></a>
<div id="DST034F3" data-id="DST034F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03403.png?Expires=2147483647&Signature=P9oKrwB0L3wCDhrdgKAzeQf-318Wr-dtDGl5yyrdJVYgAbxRi8GDX7fr6qbJeCTDu9H2Et~yXpKVynmGbA11q7i2Ph880s9AUcbIfuH~iQ0vF9A2jm9q~ovlNEN9MrHpXRobPYWHEVoxus2qlaqmiuHzgpOdK0InnoqpkwMekeVFrAmayT~S9aLCMQd63tHzkexbBB0~boZdvclIVYcFPI9A6RpmlMb~LJAtvVevJsb9WxOmggbBfNNEm22h9G8NElJsRFy37ixZ8Is-vyDngIGTD0ehqtA3nRJtbCet6RFdQHKYRkFCDmd3-jL7NLc38ROTP7x-55~gEFPm90yHhA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene methylation and gene expression profiles. (A) Gene expression levels at different DNA methylation peak value ranges in heart and livers of the MRL/MpJ mouse. The microarray gene expression data representing the tissues collected from uninjured organs were retrieved from the following datasets GSE4710 and GSE19222 (hearts), GSE5241 and GSE25322 (livers) that had been obtained in four independent studies.16,17,37,38 Wilcoxon rank-sum two-tailed tests were performed in order to compare gene expression values between the groups of genes selected according to the DNA methylation level ranges (determined in this study). The median and quartile values were normalized to the highest value from each dataset. (B) The correlations between the DNA methylation and gene expression levels. The Pearson's correlation coefficients were calculated for the gene methylation values determined in this study for the heart and liver of the MRL/MpJ and C57BL/6J mouse, and corresponding gene expression values that were obtained from four other independent studies,16,17,37,38 as described above." path-from-xml="dst03403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237368/dst03403.tiff" path-from-xml="dst03403.tiff" target="_blank">View large</a><a section="6237368" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst03403.png?Expires=2147483647&Signature=My8HcdLG5Kk1Wsww-BroT-I7dc0gAKzP1rCdAe1kzdW047bWiXbdxNqAEUtjmWZs1nkOi8txLAUup-esF03EKHKzDrdG1ZwIvnQAPwML4X3rqNkWX361EGyBRndydpcyjN6WJwkdXra~RdFN0VoSfoVBnOERFY0rgSBSXUJ1N-OmvytEeO67ndED1BR~jmMkF59a~ZHwiQoXqWss3MaOUcDEaL9k~C1eTU7GHWIdfxbKeaGhaXLpLem0b4AdbgmCGclBx9K~uAPIlQj~2Euud-HoFnNYP17wemiu-jUco9Jqd3hRYRxO5CCbZzpd5TB3Q9Fa5KkDScO7~lTsJe3Gug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237368&ar=420907&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene methylation and gene expression profiles. (A) Gene expression levels at different DNA methylation peak value ranges in heart and livers of the MRL/MpJ mouse. The microarray gene expression data representing the tissues collected from uninjured organs were retrieved from the following datasets GSE4710 and GSE19222 (hearts), GSE5241 and GSE25322 (livers) that had been obtained in four independent studies.<sup><a reveal-id="DST034C16" data-open="DST034C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DST034C17" data-open="DST034C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DST034C37" data-open="DST034C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DST034C38" data-open="DST034C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Wilcoxon rank-sum two-tailed tests were performed in order to compare gene expression values between the groups of genes selected according to the DNA methylation level ranges (determined in this study). The median and quartile values were normalized to the highest value from each dataset. (B) The correlations between the DNA methylation and gene expression levels. The Pearson's correlation coefficients were calculated for the gene methylation values determined in this study for the heart and liver of the MRL/MpJ and C57BL/6J mouse, and corresponding gene expression values that were obtained from four other independent studies,<sup><a reveal-id="DST034C16" data-open="DST034C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DST034C17" data-open="DST034C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DST034C37" data-open="DST034C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DST034C38" data-open="DST034C38" class="link link-ref link-reveal xref-bibr">38</a></sup> as described above.</p></div></div><div content-id="DST034F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03403.png?Expires=2147483647&Signature=P9oKrwB0L3wCDhrdgKAzeQf-318Wr-dtDGl5yyrdJVYgAbxRi8GDX7fr6qbJeCTDu9H2Et~yXpKVynmGbA11q7i2Ph880s9AUcbIfuH~iQ0vF9A2jm9q~ovlNEN9MrHpXRobPYWHEVoxus2qlaqmiuHzgpOdK0InnoqpkwMekeVFrAmayT~S9aLCMQd63tHzkexbBB0~boZdvclIVYcFPI9A6RpmlMb~LJAtvVevJsb9WxOmggbBfNNEm22h9G8NElJsRFy37ixZ8Is-vyDngIGTD0ehqtA3nRJtbCet6RFdQHKYRkFCDmd3-jL7NLc38ROTP7x-55~gEFPm90yHhA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Gene methylation and gene expression profiles. (A) Gene expression levels at different DNA methylation peak value ranges in heart and livers of the MRL/MpJ mouse. The microarray gene expression data representing the tissues collected from uninjured organs were retrieved from the following datasets GSE4710 and GSE19222 (hearts), GSE5241 and GSE25322 (livers) that had been obtained in four independent studies.16,17,37,38 Wilcoxon rank-sum two-tailed tests were performed in order to compare gene expression values between the groups of genes selected according to the DNA methylation level ranges (determined in this study). The median and quartile values were normalized to the highest value from each dataset. (B) The correlations between the DNA methylation and gene expression levels. The Pearson's correlation coefficients were calculated for the gene methylation values determined in this study for the heart and liver of the MRL/MpJ and C57BL/6J mouse, and corresponding gene expression values that were obtained from four other independent studies,16,17,37,38 as described above." path-from-xml="dst03403.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237368/dst03403.tiff" path-from-xml="dst03403.tiff" target="_blank">View large</a><a section="6237368" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst03403.png?Expires=2147483647&Signature=My8HcdLG5Kk1Wsww-BroT-I7dc0gAKzP1rCdAe1kzdW047bWiXbdxNqAEUtjmWZs1nkOi8txLAUup-esF03EKHKzDrdG1ZwIvnQAPwML4X3rqNkWX361EGyBRndydpcyjN6WJwkdXra~RdFN0VoSfoVBnOERFY0rgSBSXUJ1N-OmvytEeO67ndED1BR~jmMkF59a~ZHwiQoXqWss3MaOUcDEaL9k~C1eTU7GHWIdfxbKeaGhaXLpLem0b4AdbgmCGclBx9K~uAPIlQj~2Euud-HoFnNYP17wemiu-jUco9Jqd3hRYRxO5CCbZzpd5TB3Q9Fa5KkDScO7~lTsJe3Gug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237368&ar=420907&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03403.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Gene methylation and gene expression profiles. (A) Gene expression levels at different DNA methylation peak value ranges in heart and livers of the MRL/MpJ mouse. The microarray gene expression data representing the tissues collected from uninjured organs were retrieved from the following datasets GSE4710 and GSE19222 (hearts), GSE5241 and GSE25322 (livers) that had been obtained in four independent studies.<sup><a reveal-id="DST034C16" data-open="DST034C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DST034C17" data-open="DST034C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DST034C37" data-open="DST034C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DST034C38" data-open="DST034C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Wilcoxon rank-sum two-tailed tests were performed in order to compare gene expression values between the groups of genes selected according to the DNA methylation level ranges (determined in this study). The median and quartile values were normalized to the highest value from each dataset. (B) The correlations between the DNA methylation and gene expression levels. The Pearson's correlation coefficients were calculated for the gene methylation values determined in this study for the heart and liver of the MRL/MpJ and C57BL/6J mouse, and corresponding gene expression values that were obtained from four other independent studies,<sup><a reveal-id="DST034C16" data-open="DST034C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DST034C17" data-open="DST034C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DST034C37" data-open="DST034C37" class="link link-ref link-reveal xref-bibr">37</a>,<a reveal-id="DST034C38" data-open="DST034C38" class="link link-ref link-reveal xref-bibr">38</a></sup> as described above.</p></div></div><p>We also would like to punctuate three possible reasons why gene methylation and expression levels could be inconsistent. First, it should be noted that tissues unlike cell lines are more likely to be mosaics of cells that show different DNA methylation and gene expression patterns, thus a methylation peak may represent gene methylation status in a fraction of tissue cells, while another fraction of cells is responsible for high gene expression. Secondly, an individual gene does not have to show higher expression when unmethylated, as other repression mechanisms could be involved. Thirdly, some genes are induced due to demethylation of selected CpGs within a CpG island.<sup><a reveal-id="DST034C39" data-open="DST034C39" class="link link-ref link-reveal xref-bibr">39</a></sup></p>                            <h3 scrollto-destination=6237370 id="6237370" class="section-title" data-legacy-id=s3e>3.5. Functional profiling of genes differentially methylated in the MRL/MpJ mouse: gene set enrichment analysis</h3>
<p>Once the genes differentially methylated in the MRL/MpJ mouse were singled out, we could examine which functional and structural categories were overrepresented among these genes, to be followed by looking for associations with the phenotype. As the first step, we carried out gene set enrichment analysis for the groups of hypo- and hypermethylated genes using the DAVID algorithm and database.<sup><a reveal-id="DST034C29" data-open="DST034C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DST034C30" data-open="DST034C30" class="link link-ref link-reveal xref-bibr">30</a></sup> We examined the sets of genes hypo- and hypermethylated in hearts, livers, and spleens (six gene sets, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The analysis revealed a number of statistically significant functional terms (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). Although terms which are not statistically significant, as well as those found for one of tissues, could provide useful clues on the association between DNA methylation and regeneration potential, we decided to focus on the statistically significant terms representative of the three examined tissues. The statistically significant functional terms representing the hypomethylated genes in the three examined tissues (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>) could be associated with three major categories: phosphoproteins, alternative splicing, and negative regulation of metabolism (repressors), thus suggesting that the MRL/MpJ mouse is likely to produce more phosphoproteins, repressors, and alternatively spliced transcripts. Interestingly, intragenic DNA methylation has been associated with the use of alternative promoters and the synthesis of alternative transcripts,<sup><a reveal-id="DST034C40" data-open="DST034C40" class="link link-ref link-reveal xref-bibr">40</a></sup> while positive regulation of phosphate metabolic processes in human foetal heart tissues has been concluded from the analysis of DNA methylation microarray data.<sup><a reveal-id="DST034C41" data-open="DST034C41" class="link link-ref link-reveal xref-bibr">41</a></sup></p><p>An additional term shared by the term sets obtained for the genes hypomethylated in liver, heart, and spleen of the MRL/MpJ mouse was associated with embryonic morphogenesis. This suggests that the DNA methylation pattern of MRL/MpJ mouse might have retained some embryonic features and the possibility will be discussed in the next chapter.</p><p>The remarkable features of the functional terms representing the hypermethylated genes identified in the heart and spleen of the MRL/MpJ mouse were associated with three groups: G-protein-coupled receptors, including mostly olfactory receptors, Ig-like V-type, and natural killer receptor Ly49 genes. Over 1500 olfactory receptor genes are one of the most abundant gene family in mouse, and this may explain their overrepresentation. Nevertheless, it is worth noting that altered olfactory function has been reported in the MRL mouse model of the central nervous system lupus.<sup><a reveal-id="DST034C42" data-open="DST034C42" class="link link-ref link-reveal xref-bibr">42</a></sup> The majority of selected Ig-like V-type genes belongs to the cluster of selection and upkeep of intraepithelial T cell genes. The region of chromosome 4, where the selection and upkeep of intraepithelial T cells genes are located, is polymorphic, and we found that the MRL/MpJ mouse lacks the nucleotide sequences corresponding to the DNA methylation peaks in the promoter regions of <em>Skint3</em> and <em>Skint4</em> (not shown), so it is not the epigenetic but the genetic impact rather which should be considered for this group of genes. The hypermethylated genes of natural killer Ly49 receptors included mostly killer cell lectin-like receptor subfamily A genes: <em>Klra12</em>, <em>Klra13-ps</em>, <em>Klra4</em>, <em>Klra15</em>, <em>Klra23</em>, <em>Klra22</em>, <em>Klra18</em>, and <em>Klra33</em>, located on the chromosome 6.</p><p>None of the functional terms produced for the genes hypermethylated in the liver was shared by the term sets produced for the genes hypermethylated in the heart and spleen of the MRL/MpJ mouse.</p>                            <h3 scrollto-destination=6237375 id="6237375" class="section-title" data-legacy-id=s3f>3.6. Do the genomic DNA methylation patterns of the MRL/MpJ mouse retain embryonic features?</h3>
<p>Enhanced healing typical of foetal and neonatal period suggests that the regeneration abilities of the MRL/MpJ mouse could be embryonic relics preserved in the adult organism. As indicated by Naviaux <em>et al.</em>,<sup><a reveal-id="DST034C24" data-open="DST034C24" class="link link-ref link-reveal xref-bibr">24</a></sup> the adult <em>MRL</em>/<em>MpJ</em> mouse retains a selection of embryonic metabolic features. With regard to finding out as to whether the genomic DNA methylation profiles of the MRL/MpJ mouse exhibit embryonic traits, we investigated the representation of the genes associated with embryo development among the genes differentially methylated in the MRL/MpJ mouse, as well as we evaluated the overall similarity of the MRL/MpJ and embryonic methylation patterns.</p>                            <h3 scrollto-destination=6237377 id="6237377" class="section-title" data-legacy-id=s3g>3.7. Genes associated with embryonic development are overrepresented among those hypomethylated in the MRL/MpJ mouse</h3>
<p>The gene set enrichment analysis for the genes hypomethylated in the MRL/MpJ showed excessive numbers of genes associated with embryonic development and homeobox genes (Table <a reveal-id="DST034TB2" data-open="DST034TB2" class="link link-reveal link-table xref-fig">2</a>). ‘Embryonic organ morphogenesis,’ one of the terms related to embryonic development, was shared by the functional term sets produced for the three examined tissues. Three different groups of genes associated with embryonic organ morphogenesis which were identified in the liver, heart, and spleen shared two elements: the insulin induced gene 2 (<em>Insig2</em>) and forkhead box C2 (<em>Foxc2</em>) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). The latter could be of particular interest of regenerative medicine as a potent regulator of angiogenesis.<sup><a reveal-id="DST034C43" data-open="DST034C43" class="link link-ref link-reveal xref-bibr">43</a></sup></p>                        <a id="6237379" scrollto-destination="6237379"></a>
<div content-id="DST034TB2" class="table-modal"><div class="table-wrap"><div id="DST034TB2" class="table-wrap-title" data-id="DST034TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Excessive numbers of homeobox and embryonic morphogenesis genes are hypomethylated in the tissues of adult MRL/MpJ mouse</p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03408b.gif?Expires=2147483647&Signature=b9~50--a8Cm0H468rrGg4q8Yysu8kQyES39Dz1pPlaXKNGD9iIeCwIlFKuAGIYw81bMQLAv3mH3nSL9amNWA1AGYL9siD3rmiyHV2v2yuOLmYQ9Unzjy46gdsDjTqI758FFC47um7INcoEHaVcWEGy4zNKRluXC3noM39aRHTBc4Bx-6B0BsdsFN7KaSk9nzAqT1wuFl50i0u03nXgScrHE4Hx3WkfYX6AebQWpmaRmFZFwF5F6Sw0VCeAyKhALrCn7l7Y4RV4zmkfC3I2qEjebyNgZXRI4wJXXeD0Lp7VzRDcBe9OcW~9S3B3ZEK9z-zqQXiXn-i20YrIxpXvsFcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dst03408b.gif" /> </td></tr></tbody></table></div><div content-id="DST034TB2" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03408b.gif?Expires=2147483647&Signature=b9~50--a8Cm0H468rrGg4q8Yysu8kQyES39Dz1pPlaXKNGD9iIeCwIlFKuAGIYw81bMQLAv3mH3nSL9amNWA1AGYL9siD3rmiyHV2v2yuOLmYQ9Unzjy46gdsDjTqI758FFC47um7INcoEHaVcWEGy4zNKRluXC3noM39aRHTBc4Bx-6B0BsdsFN7KaSk9nzAqT1wuFl50i0u03nXgScrHE4Hx3WkfYX6AebQWpmaRmFZFwF5F6Sw0VCeAyKhALrCn7l7Y4RV4zmkfC3I2qEjebyNgZXRI4wJXXeD0Lp7VzRDcBe9OcW~9S3B3ZEK9z-zqQXiXn-i20YrIxpXvsFcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dst03408b.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The gene set enrichment analysis carried out using DAVID showed that homeobox genes and a number of other genes involved in multiple developmental processes were significantly overrepresented in the group of the MRL/MpJ hypomethylated genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The enrichment terms which are statistically significant with modified Fisher's exact test (<em>p</em>-values &lt;0.05) are indicated with orange fields (darker shade of grey in the print) and those with <em>p</em>-values &lt;0.05 after Benjamini correction are indicated with red fields (darker shade of grey in the print).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6237379" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DST034TB2" class="table-wrap-title" data-id="DST034TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Excessive numbers of homeobox and embryonic morphogenesis genes are hypomethylated in the tissues of adult MRL/MpJ mouse</p></div> </div><div class="table-overflow"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03408b.gif?Expires=2147483647&Signature=b9~50--a8Cm0H468rrGg4q8Yysu8kQyES39Dz1pPlaXKNGD9iIeCwIlFKuAGIYw81bMQLAv3mH3nSL9amNWA1AGYL9siD3rmiyHV2v2yuOLmYQ9Unzjy46gdsDjTqI758FFC47um7INcoEHaVcWEGy4zNKRluXC3noM39aRHTBc4Bx-6B0BsdsFN7KaSk9nzAqT1wuFl50i0u03nXgScrHE4Hx3WkfYX6AebQWpmaRmFZFwF5F6Sw0VCeAyKhALrCn7l7Y4RV4zmkfC3I2qEjebyNgZXRI4wJXXeD0Lp7VzRDcBe9OcW~9S3B3ZEK9z-zqQXiXn-i20YrIxpXvsFcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dst03408b.gif" /> </td></tr></tbody></table></div><div content-id="DST034TB2" class="table-modal"><table><tbody><tr><td><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03408b.gif?Expires=2147483647&Signature=b9~50--a8Cm0H468rrGg4q8Yysu8kQyES39Dz1pPlaXKNGD9iIeCwIlFKuAGIYw81bMQLAv3mH3nSL9amNWA1AGYL9siD3rmiyHV2v2yuOLmYQ9Unzjy46gdsDjTqI758FFC47um7INcoEHaVcWEGy4zNKRluXC3noM39aRHTBc4Bx-6B0BsdsFN7KaSk9nzAqT1wuFl50i0u03nXgScrHE4Hx3WkfYX6AebQWpmaRmFZFwF5F6Sw0VCeAyKhALrCn7l7Y4RV4zmkfC3I2qEjebyNgZXRI4wJXXeD0Lp7VzRDcBe9OcW~9S3B3ZEK9z-zqQXiXn-i20YrIxpXvsFcA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="graphic" path-from-xml="dst03408b.gif" /> </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The gene set enrichment analysis carried out using DAVID showed that homeobox genes and a number of other genes involved in multiple developmental processes were significantly overrepresented in the group of the MRL/MpJ hypomethylated genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). The enrichment terms which are statistically significant with modified Fisher's exact test (<em>p</em>-values &lt;0.05) are indicated with orange fields (darker shade of grey in the print) and those with <em>p</em>-values &lt;0.05 after Benjamini correction are indicated with red fields (darker shade of grey in the print).</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6237379" target="_blank">View Large</a></div></div><p>Homeobox genes have been found to be hypomethylated in the embryos of the C57BL/6J mouse in hearts and livers (spleens were not examined in that study).<sup><a reveal-id="DST034C44" data-open="DST034C44" class="link link-ref link-reveal xref-bibr">44</a></sup> Homeobox genes were overrepresented among the genes differentially methylated in the spleens and livers, but not in the hearts of the MRL/MpJ mouse. Thirty-nine and 20 homeobox genes were found to be hypomethylated in the liver and spleen in the MRL/MpJ mouse, respectively (Table <a reveal-id="DST034TB2" data-open="DST034TB2" class="link link-reveal link-table xref-fig">2</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). Seven of these homeobox genes were hypomethylated in both tissues.</p>                            <h3 scrollto-destination=6237381 id="6237381" class="section-title" data-legacy-id=s3h>3.8. Genomic DNA methylation patterns of the MRL/MpJ mouse reveal lower inter-tissue diversification</h3>
<p>To evaluate the overall similarity between the genomic DNA methylation profiles of embryos and the adult MRL/MpJ mouse, we applied Pearson's correlation test. We analysed the DNA methylation data obtained in our study and that reported by Liang <em>et al.</em><sup><a reveal-id="DST034C44" data-open="DST034C44" class="link link-ref link-reveal xref-bibr">44</a></sup> As we were comparing the data produced by different generations of Nimblegen microarrays, we restricted the analysis to the transcription start sites shared by both platforms. The analysis of correlation data showed that the inter-tissue diversification of the genomic DNA methylation profiles was lower in the MRL/MpJ than in the C57BL/6J mouse (Fig. <a reveal-id="DST034F4" data-open="DST034F4" class="link link-reveal link-table xref-fig">4</a>). Assuming that the DNA methylation patterns reflect tissue differentiation, which advances with development, the lower diversification of inter-tissue DNA methylation patterns seems likely to occur at earlier developmental stages. With regard to this hypothesis, we analysed the changes in correlation coefficients between the genomic DNA methylation profiles in embryo, newborn, and adult tissues using the data reported by Liang <em>et al.</em><sup><a reveal-id="DST034C44" data-open="DST034C44" class="link link-ref link-reveal xref-bibr">44</a></sup> to find out that in most cases the inter-tissue correlations of DNA methylation profiles decrease with age (Fig. <a reveal-id="DST034F4" data-open="DST034F4" class="link link-reveal link-table xref-fig">4</a>, inset). To our knowledge, low inter-tissue diversification of the embryonic DNA methylation profiles has never been reported to date. </p>                        <a id="6237383" scrollto-destination="6237383"></a>
<div id="DST034F4" data-id="DST034F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03404.png?Expires=2147483647&Signature=njDxjGNxkYt0sDP32l2QYGlkFkZSJ5g7oWc3jMo4Ad2l0hYV7QWrxOj5TOhnUpjMSatzHEGPmgBcpoutYr2ZwhXLumjpvDyXmxcXKwesMiyDQz29-xYSLlGePPXkuG1d0p8nqlKUDHGCiKFbNSnSlRNScnHLpYoWJFD04W7oO6XY0mCX5Ar9ZYv-IQLlTVI7MzfT9sI5W~Q7GwWeaOYA5Gxt5w0Dn2wB0Lz1kg-B8G7TbmB5BdIpkzkeEKb7af9ks87cnyZ~GrfS8G9Vi2R-VBnIWr0yZdbgnEA9ijy0Vuydpn-uJZlt-6c9QVu6D08sR7jOEfZXSqIbJlV692kuaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome-wide DNA methylation profiles display lower inter-tissue diversification in the MRL/MpJ than in the C57BL/6J mouse. Inset: Similarly, we observed lower inter-tissue diversification of genome-wide DNA methylation profiles reported by Liang et al.44 in embryos and newborns than in adults (with the exception for the correlations between the heart and brain). The inter-tissue correlations between pairs of genomic DNA methylation profiles were determined using Pearson's correlations coefficients. All Pearson's correlation coefficients were statistically significant with P-values of &lt;0.0001. Pearson's correlation coefficients were calculated for the pairs of genomic DNA methylation arrays obtained in this study: heart–spleen, spleen–liver, and heart–liver for both the MRL/MpJ and C57BL/6J mouse. Analogically, Pearson's correlation coefficients were determined for the genomic DNA methylation profiles in different tissues of murine embryos, newborns, and adults reported by Liang et al.44 The microarray data were extracted from the dataset deposited in the GEO database under the accession number GSE21415. The Pearson's correlation coefficients shown in the inset were determined for the average values obtained from two biological replicates, but similar trends were found in the correlations determined for individual animals (not shown)." path-from-xml="dst03404.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237383/dst03404.tiff" path-from-xml="dst03404.tiff" target="_blank">View large</a><a section="6237383" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst03404.png?Expires=2147483647&Signature=Q5ZsJ2qw236Hj86j9mXd7EDwfF4qGc8fcu4VTyfMbchaN7Yfw2V~eKSw1BlnJ9pEXF8FPR3rf7g38yNsrD8J74CNnp~ACqvVPgZ0GulTjIbfxFz74eSZCJsJs7r6XSQFdNC-7gqQQM-n9c2iCh6BAh3b0Q8cNmjhH9uH7XjXRyeYHM~j7sTY~yJhBYLkFgxMiVTdpVe9rbbz8fhmxLBOWEnE7c5mqKgUG10GfOiN-woCkhZcPgbxiDuc6L9Cp~zkuDaqVg5v9Yk0kXNEyaChiCFa8C5u7M4xiCicZX-7dQNtRNaFZUSsRmR93SgIODnCdKl-GcD-b7qCn3CdIA0KSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237383&ar=420907&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03404.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide DNA methylation profiles display lower inter-tissue diversification in the MRL/MpJ than in the C57BL/6J mouse. Inset: Similarly, we observed lower inter-tissue diversification of genome-wide DNA methylation profiles reported by Liang <em>et al.</em><sup><a reveal-id="DST034C44" data-open="DST034C44" class="link link-ref link-reveal xref-bibr">44</a></sup> in embryos and newborns than in adults (with the exception for the correlations between the heart and brain). The inter-tissue correlations between pairs of genomic DNA methylation profiles were determined using Pearson's correlations coefficients. All Pearson's correlation coefficients were statistically significant with <em>P</em>-values of &lt;0.0001. Pearson's correlation coefficients were calculated for the pairs of genomic DNA methylation arrays obtained in this study: heart–spleen, spleen–liver, and heart–liver for both the MRL/MpJ and C57BL/6J mouse. Analogically, Pearson's correlation coefficients were determined for the genomic DNA methylation profiles in different tissues of murine embryos, newborns, and adults reported by Liang <em>et al.</em><sup><a reveal-id="DST034C44" data-open="DST034C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The microarray data were extracted from the dataset deposited in the GEO database under the accession number GSE21415. The Pearson's correlation coefficients shown in the inset were determined for the average values obtained from two biological replicates, but similar trends were found in the correlations determined for individual animals (not shown).</p></div></div><div content-id="DST034F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03404.png?Expires=2147483647&Signature=njDxjGNxkYt0sDP32l2QYGlkFkZSJ5g7oWc3jMo4Ad2l0hYV7QWrxOj5TOhnUpjMSatzHEGPmgBcpoutYr2ZwhXLumjpvDyXmxcXKwesMiyDQz29-xYSLlGePPXkuG1d0p8nqlKUDHGCiKFbNSnSlRNScnHLpYoWJFD04W7oO6XY0mCX5Ar9ZYv-IQLlTVI7MzfT9sI5W~Q7GwWeaOYA5Gxt5w0Dn2wB0Lz1kg-B8G7TbmB5BdIpkzkeEKb7af9ks87cnyZ~GrfS8G9Vi2R-VBnIWr0yZdbgnEA9ijy0Vuydpn-uJZlt-6c9QVu6D08sR7jOEfZXSqIbJlV692kuaQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Genome-wide DNA methylation profiles display lower inter-tissue diversification in the MRL/MpJ than in the C57BL/6J mouse. Inset: Similarly, we observed lower inter-tissue diversification of genome-wide DNA methylation profiles reported by Liang et al.44 in embryos and newborns than in adults (with the exception for the correlations between the heart and brain). The inter-tissue correlations between pairs of genomic DNA methylation profiles were determined using Pearson's correlations coefficients. All Pearson's correlation coefficients were statistically significant with P-values of &lt;0.0001. Pearson's correlation coefficients were calculated for the pairs of genomic DNA methylation arrays obtained in this study: heart–spleen, spleen–liver, and heart–liver for both the MRL/MpJ and C57BL/6J mouse. Analogically, Pearson's correlation coefficients were determined for the genomic DNA methylation profiles in different tissues of murine embryos, newborns, and adults reported by Liang et al.44 The microarray data were extracted from the dataset deposited in the GEO database under the accession number GSE21415. The Pearson's correlation coefficients shown in the inset were determined for the average values obtained from two biological replicates, but similar trends were found in the correlations determined for individual animals (not shown)." path-from-xml="dst03404.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237383/dst03404.tiff" path-from-xml="dst03404.tiff" target="_blank">View large</a><a section="6237383" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst03404.png?Expires=2147483647&Signature=Q5ZsJ2qw236Hj86j9mXd7EDwfF4qGc8fcu4VTyfMbchaN7Yfw2V~eKSw1BlnJ9pEXF8FPR3rf7g38yNsrD8J74CNnp~ACqvVPgZ0GulTjIbfxFz74eSZCJsJs7r6XSQFdNC-7gqQQM-n9c2iCh6BAh3b0Q8cNmjhH9uH7XjXRyeYHM~j7sTY~yJhBYLkFgxMiVTdpVe9rbbz8fhmxLBOWEnE7c5mqKgUG10GfOiN-woCkhZcPgbxiDuc6L9Cp~zkuDaqVg5v9Yk0kXNEyaChiCFa8C5u7M4xiCicZX-7dQNtRNaFZUSsRmR93SgIODnCdKl-GcD-b7qCn3CdIA0KSw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237383&ar=420907&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03404.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Genome-wide DNA methylation profiles display lower inter-tissue diversification in the MRL/MpJ than in the C57BL/6J mouse. Inset: Similarly, we observed lower inter-tissue diversification of genome-wide DNA methylation profiles reported by Liang <em>et al.</em><sup><a reveal-id="DST034C44" data-open="DST034C44" class="link link-ref link-reveal xref-bibr">44</a></sup> in embryos and newborns than in adults (with the exception for the correlations between the heart and brain). The inter-tissue correlations between pairs of genomic DNA methylation profiles were determined using Pearson's correlations coefficients. All Pearson's correlation coefficients were statistically significant with <em>P</em>-values of &lt;0.0001. Pearson's correlation coefficients were calculated for the pairs of genomic DNA methylation arrays obtained in this study: heart–spleen, spleen–liver, and heart–liver for both the MRL/MpJ and C57BL/6J mouse. Analogically, Pearson's correlation coefficients were determined for the genomic DNA methylation profiles in different tissues of murine embryos, newborns, and adults reported by Liang <em>et al.</em><sup><a reveal-id="DST034C44" data-open="DST034C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The microarray data were extracted from the dataset deposited in the GEO database under the accession number GSE21415. The Pearson's correlation coefficients shown in the inset were determined for the average values obtained from two biological replicates, but similar trends were found in the correlations determined for individual animals (not shown).</p></div></div><p>The genomic DNA methylation profiles of the adult MRL/MpJ mouse are not embryonic, but they display several features resembling those of embryos. The analysis of genomic DNA methylation patterns in the embryonic and newborn tissues of the MRL/MpJ mouse is necessary to provide further evidence.</p>                            <h3 scrollto-destination=6237385 id="6237385" class="section-title" data-legacy-id=s3i>3.9. Enhanced proportion of high methylation peaks in the MRL/MpJ compared with C57BL/6J mouse</h3>
<p>In addition to the above-discussed low inter-tissue diversification, another characteristic feature of the genomic DNA methylation profiles of the MRL/MpJ mouse, we found, is an enhanced proportion of high methylation peaks (with the values of 4.0 and over) in comparison with the C57BL/6J (Fig. <a reveal-id="DST034F5" data-open="DST034F5" class="link link-reveal link-table xref-fig">5</a>). </p>                        <a id="6237387" scrollto-destination="6237387"></a>
<div id="DST034F5" data-id="DST034F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03405.png?Expires=2147483647&Signature=bNR2jbnAqjV6oybw0DpIJJyFI~HLy-2QAzMbNFnl8-YWhujle4ecdB~S2lOJG0WSCBsnT1MIkp2BBVXbSg3tiBqYbyI9gA4WmqH3A18sSJw8u1n~a1mUzarMxZWJ1G0fIx8O0Ze1Ol551v-X2ksTFEJu1~AR65RzFsrPpgE8H4fdu9dNfViKBfEFOpUCS2DsxBca4Cd5Aa8AymI8mduPT7eT08-ynZMLexGPDtGtS7hFDepXvInNqkITNgbIdU0XMWb3jVL0tZxumx7YaFDb7si08Qnbg86dO5bnrQP~eCQcT6cGHlsFyyVWXud1DNGUqLEIERDEsA-r8vek9Y~A-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Excessive numbers of high DNA methylation peaks in the genome-wide DNA methylation profiles in the tissues of MRL/MpJ mouse. It is assumed that high methylation peaks have the values of 4.0 and over." path-from-xml="dst03405.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237387/dst03405.tiff" path-from-xml="dst03405.tiff" target="_blank">View large</a><a section="6237387" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst03405.png?Expires=2147483647&Signature=xoPJFAWu~Ajk2devMWajZi~csBz42CqtW0QuTCd0wlElXgKo4BFnJxSiLRQ-V-GbKnHrCBzA-ez72C2Q~-Q4l5zgjfDJbNZsE0AfNhK1tLhUgy20iKtjh3OedHGshlg18zW9b8tuylJRmry~RO-0nLE7uN3DGDqM2F7ETPnUykLFYUS1doi106HuoO9O1821NkO4lxffrwIN~pokjVxVo2MsX0qQ1wenQ1ASEL7pdmVhCrdJ9WoS8VEXbxIuLTFYuB3LncPnJQFFUg9so37F9fRLnUxSLNFh8FhVVkDO81w3RUVxavSVWdmeg5b-GckRk7ltKR9d0wW2HvmSFV8Udw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237387&ar=420907&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03405.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Excessive numbers of high DNA methylation peaks in the genome-wide DNA methylation profiles in the tissues of MRL/MpJ mouse. It is assumed that high methylation peaks have the values of 4.0 and over.</p></div></div><div content-id="DST034F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03405.png?Expires=2147483647&Signature=bNR2jbnAqjV6oybw0DpIJJyFI~HLy-2QAzMbNFnl8-YWhujle4ecdB~S2lOJG0WSCBsnT1MIkp2BBVXbSg3tiBqYbyI9gA4WmqH3A18sSJw8u1n~a1mUzarMxZWJ1G0fIx8O0Ze1Ol551v-X2ksTFEJu1~AR65RzFsrPpgE8H4fdu9dNfViKBfEFOpUCS2DsxBca4Cd5Aa8AymI8mduPT7eT08-ynZMLexGPDtGtS7hFDepXvInNqkITNgbIdU0XMWb3jVL0tZxumx7YaFDb7si08Qnbg86dO5bnrQP~eCQcT6cGHlsFyyVWXud1DNGUqLEIERDEsA-r8vek9Y~A-Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Excessive numbers of high DNA methylation peaks in the genome-wide DNA methylation profiles in the tissues of MRL/MpJ mouse. It is assumed that high methylation peaks have the values of 4.0 and over." path-from-xml="dst03405.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237387/dst03405.tiff" path-from-xml="dst03405.tiff" target="_blank">View large</a><a section="6237387" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst03405.png?Expires=2147483647&Signature=xoPJFAWu~Ajk2devMWajZi~csBz42CqtW0QuTCd0wlElXgKo4BFnJxSiLRQ-V-GbKnHrCBzA-ez72C2Q~-Q4l5zgjfDJbNZsE0AfNhK1tLhUgy20iKtjh3OedHGshlg18zW9b8tuylJRmry~RO-0nLE7uN3DGDqM2F7ETPnUykLFYUS1doi106HuoO9O1821NkO4lxffrwIN~pokjVxVo2MsX0qQ1wenQ1ASEL7pdmVhCrdJ9WoS8VEXbxIuLTFYuB3LncPnJQFFUg9so37F9fRLnUxSLNFh8FhVVkDO81w3RUVxavSVWdmeg5b-GckRk7ltKR9d0wW2HvmSFV8Udw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237387&ar=420907&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03405.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Excessive numbers of high DNA methylation peaks in the genome-wide DNA methylation profiles in the tissues of MRL/MpJ mouse. It is assumed that high methylation peaks have the values of 4.0 and over.</p></div></div>                            <h3 scrollto-destination=6237388 id="6237388" class="section-title" data-legacy-id=s3j>3.10. The balance of DNA methylation is shifted upstream the genes in the MRL/MpJ mouse</h3>
<p>It is not only the degree of CpG methylation in the regulatory region of a gene, but also the distance of methylated CpGs from the transcription start sites that contributes to the repression of gene expression.<sup><a reveal-id="DST034C39" data-open="DST034C39" class="link link-ref link-reveal xref-bibr">39</a>,<a reveal-id="DST034C45" data-open="DST034C45" class="link link-ref link-reveal xref-bibr">45</a></sup> In other words, a DNA methylation peak though mapped to a gene may have different impacts on gene expression dependent on its location.</p><p>To examine the distribution of DNA methylation peaks in relation to the distance from their promoters, we calculated peak numbers at different distance ranges from transcription start sites and primary transcripts in hearts, livers, and spleens of the MRL/MpJ and C57BL/6J mouse (Fig. <a reveal-id="DST034F6" data-open="DST034F6" class="link link-reveal link-table xref-fig">6</a>). The microarray design of the Nimblegen platform we used includes mainly the transcription start sites associated with the basal promoters. </p>                        <a id="6237391" scrollto-destination="6237391"></a>
<div id="DST034F6" data-id="DST034F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03406.png?Expires=2147483647&Signature=3KF8GC86xFA0yI5msz11rYlsaqConErdHS9SitADQTyPvKXtNjD55~fSxSFmqpjeZPkh26OwlClgnOH83Kdw5qqeFOLT~bOJgHfqXTj5Ifu1bd87BF9g1zMtvdOgS8hkVs3kx-S8dcTdsmGsRYzc8OIhJ1Rbb0RfwcLDEtReYOxq2Kg2NorbEGu-b5rAZ2P89JUa4Edf~eS7QNzEyFW6vf1SKSHXfDDwua6pIWFxHqzx4ogRFFVnAIcZFpsaVqprNmxM0-0i0OL9IjSkY9pV5IjCYbxqIImlwVblbQLRDjNblb4w0s2kUEgjQq2p-ZArAYuQl06TPRANH3527TnkEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The numbers of DNA methylation peaks at different distance range from transcription start sites and primary transcripts in the MRL/MpJ and C57BL/6J mouse. DNA methylation peaks were divided into four categories, according to their location from the genes they were mapped to: (i) distal, mapped from 5000 bp upstream up to 500 bp upstream from transcription start sites (TSSs), (ii) proximal, mapped from 500 bp upstream up to 100 bp downstream from transcription start sites, (iii) mapped within primary transcripts (PT) and 1000 bp downstream from PT 3′ ends, except those up to 100 bp from TSSs, (iv) the peaks within remote CpG islands that were not mapped to any PT, i.e. those located over 5000 bp upstream from TSSs and 1000 bp downstream from PT 3′ ends. (A) Peak counts including all DNA methylation peaks of ≥2.0. (B) Peak counts including differentially methylated peaks only (the peaks selected according to the criteria specified in the chapter: ‘Genes differentially methylated in the MRL/MpJ mouse’)." path-from-xml="dst03406.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237391/dst03406.tiff" path-from-xml="dst03406.tiff" target="_blank">View large</a><a section="6237391" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst03406.png?Expires=2147483647&Signature=hcizzyfWBxGcxU4ax1tRzrdXBptJ6N5IvQWCarKSAp1-HWiY90dOU5VvW4UEWFNlYIWtYR5qb4DYQ4e19YDaa0-m8GJpeixEt67lQ8C6T4~lUOpVHLhvxN65JLNA~aJifDdW0B8-NrNmPHEUHsqQriWNSaIrx1UcHT1bkJ3EMtefx5eGQ0UWLkA--R8Pij023zH4Ys3676NmqhpGhc~l5561prowh6cp1sAcD-36RLQlwlWwHIJFJdtk0GuVSRYT8d~r1km~bXYcOMgGdFjUav~sTOxDjZBC70QnIjBdaOlDGO8j6NGLGeE0s-xAWvdcBD2WBcSscl-P5bsp9EHEiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237391&ar=420907&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03406.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The numbers of DNA methylation peaks at different distance range from transcription start sites and primary transcripts in the MRL/MpJ and C57BL/6J mouse. DNA methylation peaks were divided into four categories, according to their location from the genes they were mapped to: (i) distal, mapped from 5000 bp upstream up to 500 bp upstream from transcription start sites (TSSs), (ii) proximal, mapped from 500 bp upstream up to 100 bp downstream from transcription start sites, (iii) mapped within primary transcripts (PT) and 1000 bp downstream from PT 3′ ends, except those up to 100 bp from TSSs, (iv) the peaks within remote CpG islands that were not mapped to any PT, i.e. those located over 5000 bp upstream from TSSs and 1000 bp downstream from PT 3′ ends. (A) Peak counts including all DNA methylation peaks of ≥2.0. (B) Peak counts including differentially methylated peaks only (the peaks selected according to the criteria specified in the chapter: ‘Genes differentially methylated in the MRL/MpJ mouse’).</p></div></div><div content-id="DST034F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/m_dst03406.png?Expires=2147483647&Signature=3KF8GC86xFA0yI5msz11rYlsaqConErdHS9SitADQTyPvKXtNjD55~fSxSFmqpjeZPkh26OwlClgnOH83Kdw5qqeFOLT~bOJgHfqXTj5Ifu1bd87BF9g1zMtvdOgS8hkVs3kx-S8dcTdsmGsRYzc8OIhJ1Rbb0RfwcLDEtReYOxq2Kg2NorbEGu-b5rAZ2P89JUa4Edf~eS7QNzEyFW6vf1SKSHXfDDwua6pIWFxHqzx4ogRFFVnAIcZFpsaVqprNmxM0-0i0OL9IjSkY9pV5IjCYbxqIImlwVblbQLRDjNblb4w0s2kUEgjQq2p-ZArAYuQl06TPRANH3527TnkEQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The numbers of DNA methylation peaks at different distance range from transcription start sites and primary transcripts in the MRL/MpJ and C57BL/6J mouse. DNA methylation peaks were divided into four categories, according to their location from the genes they were mapped to: (i) distal, mapped from 5000 bp upstream up to 500 bp upstream from transcription start sites (TSSs), (ii) proximal, mapped from 500 bp upstream up to 100 bp downstream from transcription start sites, (iii) mapped within primary transcripts (PT) and 1000 bp downstream from PT 3′ ends, except those up to 100 bp from TSSs, (iv) the peaks within remote CpG islands that were not mapped to any PT, i.e. those located over 5000 bp upstream from TSSs and 1000 bp downstream from PT 3′ ends. (A) Peak counts including all DNA methylation peaks of ≥2.0. (B) Peak counts including differentially methylated peaks only (the peaks selected according to the criteria specified in the chapter: ‘Genes differentially methylated in the MRL/MpJ mouse’)." path-from-xml="dst03406.tiff" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6237391/dst03406.tiff" path-from-xml="dst03406.tiff" target="_blank">View large</a><a section="6237391" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst03406.png?Expires=2147483647&Signature=hcizzyfWBxGcxU4ax1tRzrdXBptJ6N5IvQWCarKSAp1-HWiY90dOU5VvW4UEWFNlYIWtYR5qb4DYQ4e19YDaa0-m8GJpeixEt67lQ8C6T4~lUOpVHLhvxN65JLNA~aJifDdW0B8-NrNmPHEUHsqQriWNSaIrx1UcHT1bkJ3EMtefx5eGQ0UWLkA--R8Pij023zH4Ys3676NmqhpGhc~l5561prowh6cp1sAcD-36RLQlwlWwHIJFJdtk0GuVSRYT8d~r1km~bXYcOMgGdFjUav~sTOxDjZBC70QnIjBdaOlDGO8j6NGLGeE0s-xAWvdcBD2WBcSscl-P5bsp9EHEiA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6237391&ar=420907&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dst03406.tiff" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The numbers of DNA methylation peaks at different distance range from transcription start sites and primary transcripts in the MRL/MpJ and C57BL/6J mouse. DNA methylation peaks were divided into four categories, according to their location from the genes they were mapped to: (i) distal, mapped from 5000 bp upstream up to 500 bp upstream from transcription start sites (TSSs), (ii) proximal, mapped from 500 bp upstream up to 100 bp downstream from transcription start sites, (iii) mapped within primary transcripts (PT) and 1000 bp downstream from PT 3′ ends, except those up to 100 bp from TSSs, (iv) the peaks within remote CpG islands that were not mapped to any PT, i.e. those located over 5000 bp upstream from TSSs and 1000 bp downstream from PT 3′ ends. (A) Peak counts including all DNA methylation peaks of ≥2.0. (B) Peak counts including differentially methylated peaks only (the peaks selected according to the criteria specified in the chapter: ‘Genes differentially methylated in the MRL/MpJ mouse’).</p></div></div><p>We found that the number of DNA methylation peaks situated over 500 bp upstream from transcription start sites in the MRL/MpJ substantially exceeds than that in the C57BL/6J mouse (Fig. <a reveal-id="DST034F6" data-open="DST034F6" class="link link-reveal link-table xref-fig">6</a>A). The number of DNA methylation peaks mapped within close promoter neighbourhood, in the range from 500 bp upstream up to 100 bp downstream from transcription start sites, is in turn higher in the C57BL/6J than that in the MRL/MpJ mouse (Fig. <a reveal-id="DST034F6" data-open="DST034F6" class="link link-reveal link-table xref-fig">6</a>A). Such differences were revealed in all examined tissues and they do not reflect the differences in total peak numbers. The numbers of DNA methylation peaks located within the primary transcripts and up to 1000 bp downstream, which are mostly the intragenic ones, were remarkably higher in the C57BL/6J than that in the MRL/MpJ mouse in the spleens and livers, while in the hearts they were similar (Fig. <a reveal-id="DST034F6" data-open="DST034F6" class="link link-reveal link-table xref-fig">6</a>A).</p><p>We also found that the numbers of DNA methylation peaks mapped over 5000 bp upstream and over 1000 bp downstream from primary transcripts, we may consider intergenic, are lower in the MRL/MpJ compared with the C57BL/6J mouse (Fig. <a reveal-id="DST034F6" data-open="DST034F6" class="link link-reveal link-table xref-fig">6</a>A). However, the microarray platform, we used, is a promoter one, and few probes are derived from the intergenic regions. Consequently, this estimate of intergenic DNA methylation may not be representative.</p><p>Interestingly, the differences described above are dramatically more pronounced for the differentially methylated genes (Fig. <a reveal-id="DST034F6" data-open="DST034F6" class="link link-reveal link-table xref-fig">6</a>B).</p><p>The results indicate that the balance of DNA methylation in the MRL/MpJ mouse is shifted towards the distal alternative promoters and other distal regulatory elements as, for example, enhancers. This shift is particularly remarkable for the differentially methylated genes. However, it should be stressed that the observed methylation of distal regulatory regions is not prevailing, but rather more accentuated in the MRL/MpJ than in the C57BL/6J mouse. Alternative promoters are known to be associated with development and tissue and cell differentiation,<sup><a reveal-id="DST034C46" data-open="DST034C46" class="link link-ref link-reveal xref-bibr">46</a>,<a reveal-id="DST034C47" data-open="DST034C47" class="link link-ref link-reveal xref-bibr">47</a></sup> which implicates the question as to whether this trait of DNA methylation profiles could impact the cellular differentiation potential in the MRL/MpJ mouse.</p>                            <h3 scrollto-destination=6237396 id="6237396" class="section-title" data-legacy-id=s3k>3.11. Genes with similar DNA methylation status in the heart and liver of the MRL/MpJ, but not the C57BL/6J, mouse</h3>
<p>As regeneration abilities of mammalian liver remarkably exceed those of the heart, the genes that display strong differences in the DNA methylation status between heart and liver could be of particular interest with regard to regeneration. Therefore, we singled out the genes that had ‘liver-like’ methylation status in the MRL/MpJ, but not in the C57BL/6J heart. We identified two, relatively small, groups of such genes. One group consisting of 18 genes and CpG islands were the genes that were hypermethylated in the heart and hypomethylated in the C57BL/6J liver, while hypomethylated both in the heart and liver of the MRL/MpJ mouse. Three genes involved in embryonic morphogenesis: <em>Epha2</em>, <em>Pax2</em>, and <em>Gatad2a</em> (Table <a reveal-id="DST034TB1" data-open="DST034TB1" class="link link-reveal link-table xref-fig">1</a>) deserve particular attention in this group. The other set, including 47 genes and CpG islands, were hypermethylated in the C57BL/6J heart and hypomethylated in the liver, while hypermethylated both in the heart and liver of the MRL/MpJ mouse. Two groups of genes, 10 genes associated with acetylation, and 3 genes encoding LIM domains were overrepresented in this group (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). The result was not statistically significant for the first group, but it includes as an important gene as histone deacetylase 1 (<em>Hdac1</em>), which is one of key factors responsible for chromatin remodelling. The repression of this factor could prevent chromatin condensation, thus promoting the transcription of a number of target genes. The overrepresentation of three LIM zinc-binding domain genes (<em>Csrp1</em>, <em>Ablim1</em>, and <em>Fhl1</em>) was statistically significant, and LIM domains are known to play important roles in the regulation of developmental pathways.<sup><a reveal-id="DST034C48" data-open="DST034C48" class="link link-ref link-reveal xref-bibr">48</a></sup></p>                            <h3 scrollto-destination=6237398 id="6237398" class="section-title" data-legacy-id=s3l>3.12. Remarkable representatives of the differentially methylated genes</h3>
<p>As we identified as many as hundreds of differentially methylated genes in three tissues of the MRL/MpJ mouse (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>), we would like to give a short description of a selection of these genes (Table <a reveal-id="DST034TB1" data-open="DST034TB1" class="link link-reveal link-table xref-fig">1</a>). The selection includes the genes differentially methylated in all examined tissues and several others that could be of exceptional interest due to their potential importance to regenerative abilities. A few genes from this list which arrested our distinct attention and were not discussed before are discussed below.</p><p>Since the MRL/MpJ mouse exhibits enhanced healing abilities, wound repair-related genes could be expected to be found in the group of the genes differentially methylated in this strain. As the genes related to wound healing are not a precisely defined category, it is difficult to evaluate if such genes are overrepresented among those differentially methylated in the MRL/MpJ. We found two genes associated with wound healing among the genes hypomethylated in the MRL/MpJ mouse that deserve particular attention: platelet-derived growth factor alpha (<em>Pdgfa</em>) and platelet-derived growth factor receptor alpha (<em>Pdgfra</em>). The roles of platelet-derived growth factors (platelet-derived growth factor beta in particular) and their receptors in wound healing have already been established.<sup><a reveal-id="DST034C49" data-open="DST034C49" class="link link-ref link-reveal xref-bibr">49</a></sup></p><p>The promoter regions of the gene encoding <em>de novo</em> methylase Dnmt3b are hypomethylated in the heart and spleen of the MRL/MpJ mouse. Dnmt3b could be of particular importance as it is expressed in embryonic stem cells and embryo inner cell mass and when expressed in adults could contribute to epigenetic plasticity.</p><p>The promoter regions of the growth arrest and DNA damage-inducible 45 alpha (<em>Gadd45a</em>) gene were hypermethylated in the MRL/MpJ mouse in the spleen and heart (but not in the liver). Interestingly, enhanced methylation of this region has been observed in response to folate intervention during spinal cord regeneration.<sup><a reveal-id="DST034C2" data-open="DST034C2" class="link link-ref link-reveal xref-bibr">2</a></sup></p><p>Regeneration abilities are often associated with the pluripotency of stem cells. Nevertheless, the list of genes differentially methylated in the MRL/MpJ mouse does not include the key stem cell marker genes, such as <em>Oct4</em> (<em>Pouf51</em>), <em>Klf4</em>, <em>Sox2</em>, <em>Nanog</em>, and <em>Islet1</em> (which are covered by the array), where the three latter have been reported to display enhanced expression in the MRL/MpJ mouse.<sup><a reveal-id="DST034C24" data-open="DST034C24" class="link link-ref link-reveal xref-bibr">24</a></sup></p>                            <h2 scrollto-destination=6237404 id="6237404" class="section-title" data-legacy-id=s4>4. Concluding remarks</h2>
<p>The analysis of genomic DNA methylation profiles revealed hundreds of genes hypo- and hypermethylated in the heart, spleen, and liver of the MRL/MpJ mouse strain in comparison with the C57BL/6J control, and 37 of these genes were differentially methylated in the three examined tissues. The distinguishing features of the genomic methylation patterns in the tissues of adult MRL/MpJ mouse, we found, were the shift of DNA methylation balance upstream the genes and relatively low inter-tissue diversity, resembling that we observed in previously reported genomic DNA methylation profiles of embryos and newborns.<sup><a reveal-id="DST034C44" data-open="DST034C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The possibility that the DNA methylation patterns of the MRL/MpJ mouse retain embryonic relics is supported by another observation, which is the overrepresentation of the genes involved in embryogenesis among the genes hypomethylated in the MRL/MpJ mouse. To our knowledge, this is the first study to report embryonic features of the genomic DNA methylation profile in adult organism.</p><p>This study presents a group of genes differentially methylated in the MRL/MpJ mouse, which could be analysed as candidate genes of regenerative phenotype. The examination of additional reference strains is necessary to narrow the group by eliminating the genes related to inter-strain differences that are not linked with regeneration. Our results show that the embryo-like features of the DNA methylation patterns are likely to contribute to enhanced regenerative capacity observed in different tissues of the MRL/MpJ mouse. The analysis of the DNA methylation changes in the wound area in the MRL/MpJ and non-healing reference strains is necessary to examine the epigenetic differences which occur in response to injury.</p><p>The <em>Fas</em><sup>lpr</sup> mutant of the MRL/MpJ mouse has been widely investigated as a model of systemic lupus erythematosus and other autoimmune diseases. The MRL/MpJ mouse has been also examined in the context of cholesterol metabolism.<sup><a reveal-id="DST034C38" data-open="DST034C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Although we focused on wound healing and regeneration aspects rather, we think that our data could also be useful for autoimmune and lipid metabolism studies.</p>                            <h2 scrollto-destination=6237408 id="6237408" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org.</a></p>                            <h2 scrollto-destination=6237410 id="6237410" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This study was supported by the research grant of National Science Centre of Poland 2011/01/B/NZ2/05352 and in part by the Foundation for Polish Science (FNP) support grant for the Kolumb programme fellows ‘The molecular basis of regeneration in the MRL mouse’ awarded to Dr P. Sachadyn.</p>                            <h2 scrollto-destination=6237412 id="6237412" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Prof. J. Renata Ochocka and Dr Arkadiusz Piotrowski for their help and the access to the Microarray Laboratory of the Department of Biology and Pharmaceutical Botany<sub>,</sub> Medical University of Gdańsk, funded by the Foundation for Polish Science (FOCUS 4/2008 and FOCUS 4/08/2009 grants).</p>                            <h2 scrollto-destination=6237414 id="6237414" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DST034C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yakushiji</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Satoh</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Correlation between Shh expression and DNA methylation status of the limb-specific Shh enhancer region during limb regeneration in amphibians</div>, <div class="source">Dev. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">312</div> (pg. <div class="fpage">171</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20Shh%20expression%20and%20DNA%20methylation%20status%20of%20the%20limb-specific%20Shh%20enhancer%20region%20during%20limb%20regeneration%20in%20amphibians&amp;author=N.%20Yakushiji&amp;author=M.%20Suzuki&amp;author=A.%20Satoh&amp;publication_year=2007&amp;journal=Dev.%20Biol.&amp;volume=312&amp;pages=171-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ydbio.2007.09.022" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ydbio.2007.09.022" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ydbio.2007.09.022"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17961537" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iskandar</div> <div class="given-names">B.J.</div></div>,  <div class="name"><div class="surname">Rizk</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Meier</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Folate regulation of axonal regeneration in the rodent central nervous system through DNA methylation</div>, <div class="source">J. Clin. Invest.</div> , <div class="year">2010</div>, vol. <div class="volume">120</div> (pg. <div class="fpage">1603</div>-<div class="lpage">16</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Folate%20regulation%20of%20axonal%20regeneration%20in%20the%20rodent%20central%20nervous%20system%20through%20DNA%20methylation&amp;author=B.J.%20Iskandar&amp;author=E.%20Rizk&amp;author=B.%20Meier&amp;publication_year=2010&amp;journal=J.%20Clin.%20Invest.&amp;volume=120&amp;pages=1603-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1172/JCI40000" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1172%2FJCI40000" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1172%2FJCI40000"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20424322" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Clark</div> <div class="given-names">L.D.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Heber-Katz</div> <div class="given-names">E.</div></div>. </div><div class="article-title">A new murine model for mammalian wound repair and regeneration</div>, <div class="source">Clin. Immunol. Immunopathol.</div> , <div class="year">1998</div>, vol. <div class="volume">88</div> (pg. <div class="fpage">35</div>-<div class="lpage">45</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20murine%20model%20for%20mammalian%20wound%20repair%20and%20regeneration&amp;author=L.D.%20Clark&amp;author=R.K.%20Clark&amp;author=E.%20Heber-Katz&amp;publication_year=1998&amp;journal=Clin.%20Immunol.%20Immunopathol.&amp;volume=88&amp;pages=35-45" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/clin.1998.4519" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fclin.1998.4519" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fclin.1998.4519"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9683548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ueno</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lyons</div> <div class="given-names">B.L.</div></div>,  <div class="name"><div class="surname">Burzenski</div> <div class="given-names">L.M.</div></div>, et al. </div><div class="article-title">Accelerated wound healing of alkali-burned corneas in MRL mice is associated with a reduced inflammatory signature</div>, <div class="source">Invest. Ophthalmol. Vis. Sci.</div> , <div class="year">2005</div>, vol. <div class="volume">46</div> (pg. <div class="fpage">4097</div>-<div class="lpage">106</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accelerated%20wound%20healing%20of%20alkali-burned%20corneas%20in%20MRL%20mice%20is%20associated%20with%20a%20reduced%20inflammatory%20signature&amp;author=M.%20Ueno&amp;author=B.L.%20Lyons&amp;author=L.M.%20Burzenski&amp;publication_year=2005&amp;journal=Invest.%20Ophthalmol.%20Vis.%20Sci.&amp;volume=46&amp;pages=4097-106" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1167/iovs.05-0548" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1167%2Fiovs.05-0548" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1167%2Fiovs.05-0548"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16249486" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tucker</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Klassen</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">D.F.</div></div>,  <div class="name"><div class="surname">Young</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">Elevated MMP expression in the MRL mouse retina creates a permissive environment for retinal regeneration</div>, <div class="source">Invest. Ophthalmol. Vis. Sci.</div> , <div class="year">2008</div>, vol. <div class="volume">49</div> (pg. <div class="fpage">1686</div>-<div class="lpage">95</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Elevated%20MMP%20expression%20in%20the%20MRL%20mouse%20retina%20creates%20a%20permissive%20environment%20for%20retinal%20regeneration&amp;author=B.%20Tucker&amp;author=H.%20Klassen&amp;author=L.%20Yang&amp;author=D.F.%20Chen&amp;author=M.J.%20Young&amp;publication_year=2008&amp;journal=Invest.%20Ophthalmol.%20Vis.%20Sci.&amp;volume=49&amp;pages=1686-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1167/iovs.07-1058" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1167%2Fiovs.07-1058" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1167%2Fiovs.07-1058"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18385092" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Xia</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Krebs</div> <div class="given-names">M.P.</div></div>,  <div class="name"><div class="surname">Kaushal</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Scott</div> <div class="given-names">E.W.</div></div>. </div><div class="article-title">Enhanced retinal pigment epithelium regeneration after injury in MRL/MpJ mice</div>, <div class="source">Exp. Eye Res.</div> , <div class="year">2011</div>, vol. <div class="volume">93</div> (pg. <div class="fpage">862</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enhanced%20retinal%20pigment%20epithelium%20regeneration%20after%20injury%20in%20MRL%2FMpJ%20mice&amp;author=H.%20Xia&amp;author=M.P.%20Krebs&amp;author=S.%20Kaushal&amp;author=E.W.%20Scott&amp;publication_year=2011&amp;journal=Exp.%20Eye%20Res.&amp;volume=93&amp;pages=862-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.exer.2011.09.020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.exer.2011.09.020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.exer.2011.09.020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21989111" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chadwick</div> <div class="given-names">R.B.</div></div>,  <div class="name"><div class="surname">Bu</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Digit tip regrowth and differential gene expression in MRL/Mpj, DBA/2, and C57BL/6 mice</div>, <div class="source">Wound Repair Regen.</div> , <div class="year">2007</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">275</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Digit%20tip%20regrowth%20and%20differential%20gene%20expression%20in%20MRL%2FMpj%2C%20DBA%2F2%2C%20and%20C57BL%2F6%20mice&amp;author=R.B.%20Chadwick&amp;author=L.%20Bu&amp;author=H.%20Yu&amp;publication_year=2007&amp;journal=Wound%20Repair%20Regen.&amp;volume=15&amp;pages=275-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/wrr.2007.15.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fwrr.2007.15.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fwrr.2007.15.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17352761" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gourevitch</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Bedelbaeva</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Leferovich</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Heber-Katz</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Dynamic changes after murine digit amputation: the MRL mouse digit shows waves of tissue remodeling, growth, and apoptosis</div>, <div class="source">Wound Repair Regen.</div> , <div class="year">2009</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">447</div>-<div class="lpage">55</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20changes%20after%20murine%20digit%20amputation%3A%20the%20MRL%20mouse%20digit%20shows%20waves%20of%20tissue%20remodeling%2C%20growth%2C%20and%20apoptosis&amp;author=D.L.%20Gourevitch&amp;author=L.%20Clark&amp;author=K.%20Bedelbaeva&amp;author=J.%20Leferovich&amp;author=E.%20Heber-Katz&amp;publication_year=2009&amp;journal=Wound%20Repair%20Regen.&amp;volume=17&amp;pages=447-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/wrr.2009.17.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fwrr.2009.17.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fwrr.2009.17.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19660054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Leferovich</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Bedelbaeva</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Samulewicz</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Heart regeneration in adult MRL mice</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2001</div>, vol. <div class="volume">98</div> (pg. <div class="fpage">9830</div>-<div class="lpage">5</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heart%20regeneration%20in%20adult%20MRL%20mice&amp;author=J.M.%20Leferovich&amp;author=K.%20Bedelbaeva&amp;author=S.%20Samulewicz&amp;publication_year=2001&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=98&amp;pages=9830-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.181329398" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.181329398" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.181329398"> </span></div> </div></div></div></div></div><div content-id="DST034C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fitzgerald</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Rich</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Burkhardt</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Allen</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Herzka</div> <div class="given-names">A.S.</div></div>,  <div class="name"><div class="surname">Little</div> <div class="given-names">C.B.</div></div>. </div><div class="article-title">Evidence for articular cartilage regeneration in MRL/MpJ mice</div>, <div class="source">Osteoarthritis Cartilage</div> , <div class="year">2008</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">1319</div>-<div class="lpage">26</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20articular%20cartilage%20regeneration%20in%20MRL%2FMpJ%20mice&amp;author=J.%20Fitzgerald&amp;author=C.%20Rich&amp;author=D.%20Burkhardt&amp;author=J.%20Allen&amp;author=A.S.%20Herzka&amp;author=C.B.%20Little&amp;publication_year=2008&amp;journal=Osteoarthritis%20Cartilage&amp;volume=16&amp;pages=1319-26" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.joca.2008.03.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.joca.2008.03.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.joca.2008.03.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18455447" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Gu</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Masinde</div> <div class="given-names">G.</div></div>, et al. </div><div class="article-title">Genetic control of the rate of wound healing in mice</div>, <div class="source">Heredity</div> , <div class="year">2001</div>, vol. <div class="volume">86</div> (pg. <div class="fpage">668</div>-<div class="lpage">74</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20control%20of%20the%20rate%20of%20wound%20healing%20in%20mice&amp;author=X.%20Li&amp;author=W.%20Gu&amp;author=G.%20Masinde&amp;publication_year=2001&amp;journal=Heredity&amp;volume=86&amp;pages=668-74" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-2540.2001.00879.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-2540.2001.00879.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-2540.2001.00879.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11595047" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gourevitch</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Seitz</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Samulewicz</div> <div class="given-names">S.J.</div></div>,  <div class="name"><div class="surname">Heber-Katz</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Matrix metalloproteinase activity correlates with blasteoma formation in the regenerating MRL mouse ear hole model</div>, <div class="source">Dev. Dyn.</div> , <div class="year">2003</div>, vol. <div class="volume">226</div> (pg. <div class="fpage">377</div>-<div class="lpage">87</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Matrix%20metalloproteinase%20activity%20correlates%20with%20blasteoma%20formation%20in%20the%20regenerating%20MRL%20mouse%20ear%20hole%20model&amp;author=D.%20Gourevitch&amp;author=L.%20Clark&amp;author=P.%20Chen&amp;author=A.%20Seitz&amp;author=S.J.%20Samulewicz&amp;author=E.%20Heber-Katz&amp;publication_year=2003&amp;journal=Dev.%20Dyn.&amp;volume=226&amp;pages=377-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0177" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0177" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0177"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12557216" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brockes</div> <div class="given-names">J.P.</div></div>. </div><div class="article-title">Amphibian limb regeneration: rebuilding a complex structure</div>, <div class="source">Science</div> , <div class="year">1997</div>, vol. <div class="volume">276</div> (pg. <div class="fpage">81</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Amphibian%20limb%20regeneration%3A%20rebuilding%20a%20complex%20structure&amp;author=J.P.%20Brockes&amp;publication_year=1997&amp;journal=Science&amp;volume=276&amp;pages=81-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.276.5309.81" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.276.5309.81" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.276.5309.81"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9082990" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chernoff</div> <div class="given-names">E.A.</div></div>,  <div class="name"><div class="surname">Stocum</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Nye</div> <div class="given-names">H.L.</div></div>,  <div class="name"><div class="surname">Cameron</div> <div class="given-names">J.A.</div></div>. </div><div class="article-title">Urodele spinal cord regeneration and related processes</div>, <div class="source">Dev. Dyn.</div> , <div class="year">2003</div>, vol. <div class="volume">226</div> (pg. <div class="fpage">295</div>-<div class="lpage">307</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Urodele%20spinal%20cord%20regeneration%20and%20related%20processes&amp;author=E.A.%20Chernoff&amp;author=D.L.%20Stocum&amp;author=H.L.%20Nye&amp;author=J.A.%20Cameron&amp;publication_year=2003&amp;journal=Dev.%20Dyn.&amp;volume=226&amp;pages=295-307" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1097-0177" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1097-0177" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1097-0177"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12557207" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Neff</div> <div class="given-names">A.W.</div></div>,  <div class="name"><div class="surname">Dent</div> <div class="given-names">A.E.</div></div>,  <div class="name"><div class="surname">Armstrong</div> <div class="given-names">J.B.</div></div>. </div><div class="article-title">Heart development and regeneration in urodeles</div>, <div class="source">Int. J. Dev. Biol.</div> , <div class="year">1996</div>, vol. <div class="volume">40</div> (pg. <div class="fpage">719</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Heart%20development%20and%20regeneration%20in%20urodeles&amp;author=A.W.%20Neff&amp;author=A.E.%20Dent&amp;author=J.B.%20Armstrong&amp;publication_year=1996&amp;journal=Int.%20J.%20Dev.%20Biol.&amp;volume=40&amp;pages=719-25" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8877445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Naseem</div> <div class="given-names">R.H.</div></div>,  <div class="name"><div class="surname">Meeson</div> <div class="given-names">A.P.</div></div>,  <div class="name"><div class="surname">Michael Dimaio</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Reparative myocardial mechanisms in adult C57BL/6 and MRL mice following injury</div>, <div class="source">Physiol. Genomics</div> , <div class="year">2007</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">44</div>-<div class="lpage">52</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Reparative%20myocardial%20mechanisms%20in%20adult%20C57BL%2F6%20and%20MRL%20mice%20following%20injury&amp;author=R.H.%20Naseem&amp;author=A.P.%20Meeson&amp;author=J.%20Michael%20Dimaio&amp;publication_year=2007&amp;journal=Physiol.%20Genomics&amp;volume=30&amp;pages=44-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1152/physiolgenomics.00070.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1152%2Fphysiolgenomics.00070.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1152%2Fphysiolgenomics.00070.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17327495" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hunt</div> <div class="given-names">D.L.</div></div>,  <div class="name"><div class="surname">Campbell</div> <div class="given-names">P.H.</div></div>,  <div class="name"><div class="surname">Zambon</div> <div class="given-names">A.C.</div></div>, et al. </div><div class="article-title">Early postmyocardial infarction survival in Murphy Roths large mice is mediated by attenuated apoptosis and inflammation but depends on genetic background</div>, <div class="source">Exp. Physiol.</div> , <div class="year">2012</div>, vol. <div class="volume">97</div> (pg. <div class="fpage">102</div>-<div class="lpage">14</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Early%20postmyocardial%20infarction%20survival%20in%20Murphy%20Roths%20large%20mice%20is%20mediated%20by%20attenuated%20apoptosis%20and%20inflammation%20but%20depends%20on%20genetic%20background&amp;author=D.L.%20Hunt&amp;author=P.H.%20Campbell&amp;author=A.C.%20Zambon&amp;publication_year=2012&amp;journal=Exp.%20Physiol.&amp;volume=97&amp;pages=102-14" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1113/expphysiol.2011.060269" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1113%2Fexpphysiol.2011.060269" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1113%2Fexpphysiol.2011.060269"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21967898" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cimini</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Fazel</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Fujii</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">The MRL mouse heart does not recover ventricular function after a myocardial infarction</div>, <div class="source">Cardiovasc. Pathol.</div> , <div class="year">2008</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">32</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20MRL%20mouse%20heart%20does%20not%20recover%20ventricular%20function%20after%20a%20myocardial%20infarction&amp;author=M.%20Cimini&amp;author=S.%20Fazel&amp;author=H.%20Fujii&amp;publication_year=2008&amp;journal=Cardiovasc.%20Pathol.&amp;volume=17&amp;pages=32-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.carpath.2007.06.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.carpath.2007.06.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.carpath.2007.06.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18160058" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Abdullah</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Lepore</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Epstein</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Parmacek</div> <div class="given-names">M.S.</div></div>,  <div class="name"><div class="surname">Gruber</div> <div class="given-names">P.J.</div></div>. </div><div class="article-title">MRL mice fail to heal the heart in response to ischemia-reperfusion injury</div>, <div class="source">Wound Repair Regen.</div> , <div class="year">2005</div>, vol. <div class="volume">13</div> (pg. <div class="fpage">205</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MRL%20mice%20fail%20to%20heal%20the%20heart%20in%20response%20to%20ischemia-reperfusion%20injury&amp;author=I.%20Abdullah&amp;author=J.J.%20Lepore&amp;author=J.A.%20Epstein&amp;author=M.S.%20Parmacek&amp;author=P.J.%20Gruber&amp;publication_year=2005&amp;journal=Wound%20Repair%20Regen.&amp;volume=13&amp;pages=205-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/wrr.2005.13.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fwrr.2005.13.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fwrr.2005.13.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15828946" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Robey</div> <div class="given-names">T.E.</div></div>,  <div class="name"><div class="surname">Murry</div> <div class="given-names">C.E.</div></div>. </div><div class="article-title">Absence of regeneration in the MRL/MpJ mouse heart following infarction or cryoinjury</div>, <div class="source">Cardiovasc. Pathol.</div> , <div class="year">2008</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">6</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Absence%20of%20regeneration%20in%20the%20MRL%2FMpJ%20mouse%20heart%20following%20infarction%20or%20cryoinjury&amp;author=T.E.%20Robey&amp;author=C.E.%20Murry&amp;publication_year=2008&amp;journal=Cardiovasc.%20Pathol.&amp;volume=17&amp;pages=6-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.carpath.2007.01.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.carpath.2007.01.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.carpath.2007.01.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18160055" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Moseley</div> <div class="given-names">F.L.</div></div>,  <div class="name"><div class="surname">Faircloth</div> <div class="given-names">M.E.</div></div>,  <div class="name"><div class="surname">Lockwood</div> <div class="given-names">W.</div></div>, et al. </div><div class="article-title">Limitations of the MRL mouse as a model for cardiac regeneration</div>, <div class="source">J. Pharm. Pharmacol.</div> , <div class="year">2011</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">648</div>-<div class="lpage">56</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Limitations%20of%20the%20MRL%20mouse%20as%20a%20model%20for%20cardiac%20regeneration&amp;author=F.L.%20Moseley&amp;author=M.E.%20Faircloth&amp;author=W.%20Lockwood&amp;publication_year=2011&amp;journal=J.%20Pharm.%20Pharmacol.&amp;volume=63&amp;pages=648-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.2042-7158.2011.01261.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.2042-7158.2011.01261.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.2042-7158.2011.01261.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21492166" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Grisel</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Meinhardt</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Lehr</div> <div class="given-names">H.A.</div></div>,  <div class="name"><div class="surname">Kappenberger</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Barrandon</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Vassalli</div> <div class="given-names">G.</div></div>. </div><div class="article-title">The MRL mouse repairs both cryogenic and ischemic myocardial infarcts with scar</div>, <div class="source">Cardiovasc. Pathol.</div> , <div class="year">2008</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">14</div>-<div class="lpage">22</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20MRL%20mouse%20repairs%20both%20cryogenic%20and%20ischemic%20myocardial%20infarcts%20with%20scar&amp;author=P.%20Grisel&amp;author=A.%20Meinhardt&amp;author=H.A.%20Lehr&amp;author=L.%20Kappenberger&amp;author=Y.%20Barrandon&amp;author=G.%20Vassalli&amp;publication_year=2008&amp;journal=Cardiovasc.%20Pathol.&amp;volume=17&amp;pages=14-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.carpath.2007.01.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.carpath.2007.01.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.carpath.2007.01.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18160056" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blankenhorn</div> <div class="given-names">E.P.</div></div>,  <div class="name"><div class="surname">Troutman</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Clark</div> <div class="given-names">L.D.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.M.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Heber-Katz</div> <div class="given-names">E.</div></div>. </div><div class="article-title">Sexually dimorphic genes regulate healing and regeneration in MRL mice</div>, <div class="source">Mamm. Genome</div> , <div class="year">2003</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">250</div>-<div class="lpage">60</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sexually%20dimorphic%20genes%20regulate%20healing%20and%20regeneration%20in%20MRL%20mice&amp;author=E.P.%20Blankenhorn&amp;author=S.%20Troutman&amp;author=L.D.%20Clark&amp;author=X.M.%20Zhang&amp;author=P.%20Chen&amp;author=E.%20Heber-Katz&amp;publication_year=2003&amp;journal=Mamm.%20Genome&amp;volume=14&amp;pages=250-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00335-002-2222-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00335-002-2222-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00335-002-2222-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12682777" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Naviaux</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Le</div> <div class="given-names">T.P.</div></div>,  <div class="name"><div class="surname">Bedelbaeva</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Retained features of embryonic metabolism in the adult MRL mouse</div>, <div class="source">Mol. Genet. Metab.</div> , <div class="year">2009</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">133</div>-<div class="lpage">44</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Retained%20features%20of%20embryonic%20metabolism%20in%20the%20adult%20MRL%20mouse&amp;author=R.K.%20Naviaux&amp;author=T.P.%20Le&amp;author=K.%20Bedelbaeva&amp;publication_year=2009&amp;journal=Mol.%20Genet.%20Metab.&amp;volume=96&amp;pages=133-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ymgme.2008.11.164" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ymgme.2008.11.164" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ymgme.2008.11.164"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19131261" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thuret</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Thallmair</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Horky</div> <div class="given-names">L.L.</div></div>,  <div class="name"><div class="surname">Gage</div> <div class="given-names">F.H.</div></div>. </div><div class="article-title">Enhanced functional recovery in MRL/MpJ mice after spinal cord dorsal hemisection</div>, <div class="source">PLoS One</div> , <div class="year">2012</div>, vol. <div class="volume">7</div> pg. <div class="fpage">e30904</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enhanced%20functional%20recovery%20in%20MRL%2FMpJ%20mice%20after%20spinal%20cord%20dorsal%20hemisection&amp;author=S.%20Thuret&amp;author=M.%20Thallmair&amp;author=L.L.%20Horky&amp;author=F.H.%20Gage&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e30904" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0030904" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0030904" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0030904"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22348029" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Weber</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Davies</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Wittig</div> <div class="given-names">D.</div></div>, et al. </div><div class="article-title">Chromosome-wide and promoter-specific analyses identify sites of differential DNA methylation in normal and transformed human cells</div>, <div class="source">Nat. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">853</div>-<div class="lpage">62</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Chromosome-wide%20and%20promoter-specific%20analyses%20identify%20sites%20of%20differential%20DNA%20methylation%20in%20normal%20and%20transformed%20human%20cells&amp;author=M.%20Weber&amp;author=J.J.%20Davies&amp;author=D.%20Wittig&amp;publication_year=2005&amp;journal=Nat.%20Genet.&amp;volume=37&amp;pages=853-62" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng1598" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng1598" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng1598"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16007088" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ronowicz</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Brzeskwiniewicz</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Madanecki</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Regeneration of comparative genomic hybridization oligonucleotide microarrays with dimethylurea</div>, <div class="source">Anal. Biochem.</div> , <div class="year">2012</div>, vol. <div class="volume">426</div> (pg. <div class="fpage">91</div>-<div class="lpage">3</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regeneration%20of%20comparative%20genomic%20hybridization%20oligonucleotide%20microarrays%20with%20dimethylurea&amp;author=A.%20Ronowicz&amp;author=M.%20Brzeskwiniewicz&amp;author=P.%20Madanecki&amp;publication_year=2012&amp;journal=Anal.%20Biochem.&amp;volume=426&amp;pages=91-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ab.2012.04.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ab.2012.04.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ab.2012.04.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22504026" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kin</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ono</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Idiographica: a general-purpose web application to build idiograms on-demand for human, mouse and rat</div>, <div class="source">Bioinformatics</div> , <div class="year">2007</div>, vol. <div class="volume">23</div> (pg. <div class="fpage">2945</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Idiographica%3A%20a%20general-purpose%20web%20application%20to%20build%20idiograms%20on-demand%20for%20human%2C%20mouse%20and%20rat&amp;author=T.%20Kin&amp;author=Y.%20Ono&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=2945-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm455" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm455" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm455"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17893084" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang da</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Sherman</div> <div class="given-names">B.T.</div></div>,  <div class="name"><div class="surname">Lempicki</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Systematic and integrative analysis of large gene lists using DAVID bioinformatics resources</div>, <div class="source">Nat. Protoc.</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">44</div>-<div class="lpage">57</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20and%20integrative%20analysis%20of%20large%20gene%20lists%20using%20DAVID%20bioinformatics%20resources&amp;author=W.%20Huang%20da&amp;author=B.T.%20Sherman&amp;author=R.A.%20Lempicki&amp;publication_year=2009&amp;journal=Nat.%20Protoc.&amp;volume=4&amp;pages=44-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2008.211" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2008.211" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2008.211"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19131956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Huang da</div> <div class="given-names">W.</div></div>,  <div class="name"><div class="surname">Sherman</div> <div class="given-names">B.T.</div></div>,  <div class="name"><div class="surname">Lempicki</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Bioinformatics enrichment tools: paths toward the comprehensive functional analysis of large gene lists</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2009</div>, vol. <div class="volume">37</div> (pg. <div class="fpage">1</div>-<div class="lpage">13</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Bioinformatics%20enrichment%20tools%3A%20paths%20toward%20the%20comprehensive%20functional%20analysis%20of%20large%20gene%20lists&amp;author=W.%20Huang%20da&amp;author=B.T.%20Sherman&amp;author=R.A.%20Lempicki&amp;publication_year=2009&amp;journal=Nucleic%20Acids%20Res.&amp;volume=37&amp;pages=1-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19033363" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Heber-Katz</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Gourevitch</div> <div class="given-names">D.</div></div>. </div><div class="article-title">The relationship between inflammation and regeneration in the MRL mouse: potential relevance for putative human regenerative (scarless wound healing) capacities?</div>, <div class="source">Ann. N Y Acad. Sci.</div> , <div class="year">2009</div>, vol. <div class="volume">1172</div> (pg. <div class="fpage">110</div>-<div class="lpage">4</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20relationship%20between%20inflammation%20and%20regeneration%20in%20the%20MRL%20mouse%3A%20potential%20relevance%20for%20putative%20human%20regenerative%20(scarless%20wound%20healing)%20capacities%3F&amp;author=E.%20Heber-Katz&amp;author=D.%20Gourevitch&amp;publication_year=2009&amp;journal=Ann.%20N%20Y%20Acad.%20Sci.&amp;volume=1172&amp;pages=110-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1749-6632.2009.04499.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1749-6632.2009.04499.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1749-6632.2009.04499.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19735244" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rae</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Johnson</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Scheys</div> <div class="given-names">J.O.</div></div>,  <div class="name"><div class="surname">Cordero</div> <div class="given-names">K.E.</div></div>,  <div class="name"><div class="surname">Larios</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Lippman</div> <div class="given-names">M.E.</div></div>. </div><div class="article-title">GREB 1 is a critical regulator of hormone dependent breast cancer growth</div>, <div class="source">Breast Cancer Res. Treat.</div> , <div class="year">2005</div>, vol. <div class="volume">92</div> (pg. <div class="fpage">141</div>-<div class="lpage">9</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GREB%201%20is%20a%20critical%20regulator%20of%20hormone%20dependent%20breast%20cancer%20growth&amp;author=J.M.%20Rae&amp;author=M.D.%20Johnson&amp;author=J.O.%20Scheys&amp;author=K.E.%20Cordero&amp;author=J.M.%20Larios&amp;author=M.E.%20Lippman&amp;publication_year=2005&amp;journal=Breast%20Cancer%20Res.%20Treat.&amp;volume=92&amp;pages=141-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10549-005-1483-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10549-005-1483-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10549-005-1483-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15986123" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kramer</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Granier</div> <div class="given-names">C.J.</div></div>,  <div class="name"><div class="surname">Davis</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">PDCD2 controls hematopoietic stem cell differentiation during development</div>, <div class="source">Stem Cells Dev.</div> , <div class="year">2013</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">58</div>-<div class="lpage">72</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PDCD2%20controls%20hematopoietic%20stem%20cell%20differentiation%20during%20development&amp;author=J.%20Kramer&amp;author=C.J.%20Granier&amp;author=S.%20Davis&amp;publication_year=2013&amp;journal=Stem%20Cells%20Dev.&amp;volume=22&amp;pages=58-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/scd.2012.0074" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fscd.2012.0074" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fscd.2012.0074"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22800338" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chechlinska</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Siwicki</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Gos</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Molecular signature of cell cycle exit induced in human T lymphoblasts by IL-2 withdrawal</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">261</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20signature%20of%20cell%20cycle%20exit%20induced%20in%20human%20T%20lymphoblasts%20by%20IL-2%20withdrawal&amp;author=M.%20Chechlinska&amp;author=J.K.%20Siwicki&amp;author=M.%20Gos&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=261" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-261" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-261" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-261"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505301" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Antoni</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Oudot-Mellakh</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Dimitromanolakis</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Combined analysis of three genome-wide association studies on vWF and FVIII plasma levels</div>, <div class="source">BMC Med. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">102</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Combined%20analysis%20of%20three%20genome-wide%20association%20studies%20on%20vWF%20and%20FVIII%20plasma%20levels&amp;author=G.%20Antoni&amp;author=T.%20Oudot-Mellakh&amp;author=A.%20Dimitromanolakis&amp;publication_year=2011&amp;journal=BMC%20Med.%20Genet.&amp;volume=12&amp;pages=102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2350-12-102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2350-12-102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2350-12-102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21810271" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hou</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Xiao</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">Genetic variant on PDGFRL associated with Behcet disease in Chinese Han populations</div>, <div class="source">Hum. Mutat.</div> , <div class="year">2012</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">74</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20variant%20on%20PDGFRL%20associated%20with%20Behcet%20disease%20in%20Chinese%20Han%20populations&amp;author=S.%20Hou&amp;author=X.%20Xiao&amp;author=Y.%20Zhou&amp;publication_year=2012&amp;journal=Hum.%20Mutat.&amp;volume=34&amp;pages=74-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/humu.22208" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fhumu.22208" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fhumu.22208"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22926996" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Konig</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Cai</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Guo</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Ansari</div> <div class="given-names">G.A.</div></div>. </div><div class="article-title">Transcriptomic analysis reveals early signs of liver toxicity in female MRL +/+ mice exposed to the acylating chemicals dichloroacetyl chloride and dichloroacetic anhydride</div>, <div class="source">Chem. Res. Toxicol.</div> , <div class="year">2008</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">572</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptomic%20analysis%20reveals%20early%20signs%20of%20liver%20toxicity%20in%20female%20MRL%20%2B%2F%2B%20mice%20exposed%20to%20the%20acylating%20chemicals%20dichloroacetyl%20chloride%20and%20dichloroacetic%20anhydride&amp;author=R.%20Konig&amp;author=P.%20Cai&amp;author=X.%20Guo&amp;author=G.A.%20Ansari&amp;publication_year=2008&amp;journal=Chem.%20Res.%20Toxicol.&amp;volume=21&amp;pages=572-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/tx7002728" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Ftx7002728" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Ftx7002728"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18293905" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Leduc</div> <div class="given-names">M.S.</div></div>,  <div class="name"><div class="surname">Hageman</div> <div class="given-names">R.S.</div></div>,  <div class="name"><div class="surname">Verdugo</div> <div class="given-names">R.A.</div></div>, et al. </div><div class="article-title">Integration of QTL and bioinformatic tools to identify candidate genes for triglycerides in mice</div>, <div class="source">J. Lipid Res.</div> , <div class="year">2011</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">1672</div>-<div class="lpage">82</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integration%20of%20QTL%20and%20bioinformatic%20tools%20to%20identify%20candidate%20genes%20for%20triglycerides%20in%20mice&amp;author=M.S.%20Leduc&amp;author=R.S.%20Hageman&amp;author=R.A.%20Verdugo&amp;publication_year=2011&amp;journal=J.%20Lipid%20Res.&amp;volume=52&amp;pages=1672-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1194/jlr.M011130" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1194%2Fjlr.M011130" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1194%2Fjlr.M011130"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21622629" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hattori</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Imao</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nishino</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Epigenetic regulation of Nanog gene in embryonic stem and trophoblast stem cells</div>, <div class="source">Genes Cells</div> , <div class="year">2007</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">387</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epigenetic%20regulation%20of%20Nanog%20gene%20in%20embryonic%20stem%20and%20trophoblast%20stem%20cells&amp;author=N.%20Hattori&amp;author=Y.%20Imao&amp;author=K.%20Nishino&amp;publication_year=2007&amp;journal=Genes%20Cells&amp;volume=12&amp;pages=387-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/gtc.2007.12.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fgtc.2007.12.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fgtc.2007.12.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17352742" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Maunakea</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Nagarajan</div> <div class="given-names">R.P.</div></div>,  <div class="name"><div class="surname">Bilenky</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Conserved role of intragenic DNA methylation in regulating alternative promoters</div>, <div class="source">Nature</div> , <div class="year">2010</div>, vol. <div class="volume">466</div> (pg. <div class="fpage">253</div>-<div class="lpage">7</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Conserved%20role%20of%20intragenic%20DNA%20methylation%20in%20regulating%20alternative%20promoters&amp;author=A.K.%20Maunakea&amp;author=R.P.%20Nagarajan&amp;author=M.%20Bilenky&amp;publication_year=2010&amp;journal=Nature&amp;volume=466&amp;pages=253-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature09165" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature09165" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature09165"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20613842" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nazor</div> <div class="given-names">K.L.</div></div>,  <div class="name"><div class="surname">Altun</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Lynch</div> <div class="given-names">C.</div></div>, et al. </div><div class="article-title">Recurrent variations in DNA methylation in human pluripotent stem cells and their differentiated derivatives</div>, <div class="source">Cell Stem Cell</div> , <div class="year">2012</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">620</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recurrent%20variations%20in%20DNA%20methylation%20in%20human%20pluripotent%20stem%20cells%20and%20their%20differentiated%20derivatives&amp;author=K.L.%20Nazor&amp;author=G.%20Altun&amp;author=C.%20Lynch&amp;publication_year=2012&amp;journal=Cell%20Stem%20Cell&amp;volume=10&amp;pages=620-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.stem.2012.02.013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.stem.2012.02.013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.stem.2012.02.013"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22560082" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kapadia</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Stanojcic</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Earls</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Pulapaka</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Lee</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Sakic</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Altered olfactory function in the MRL model of CNS lupus</div>, <div class="source">Behav. Brain Res.</div> , <div class="year">2012</div>, vol. <div class="volume">234</div> (pg. <div class="fpage">303</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Altered%20olfactory%20function%20in%20the%20MRL%20model%20of%20CNS%20lupus&amp;author=M.%20Kapadia&amp;author=M.%20Stanojcic&amp;author=A.M.%20Earls&amp;author=S.%20Pulapaka&amp;author=J.%20Lee&amp;author=B.%20Sakic&amp;publication_year=2012&amp;journal=Behav.%20Brain%20Res.&amp;volume=234&amp;pages=303-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.bbr.2012.07.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.bbr.2012.07.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.bbr.2012.07.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22796602" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kume</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Foxc2 transcription factor: a newly described regulator of angiogenesis</div>, <div class="source">Trends Cardiovasc. Med.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">224</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Foxc2%20transcription%20factor%3A%20a%20newly%20described%20regulator%20of%20angiogenesis&amp;author=T.%20Kume&amp;publication_year=2008&amp;journal=Trends%20Cardiovasc.%20Med.&amp;volume=18&amp;pages=224-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tcm.2008.11.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tcm.2008.11.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tcm.2008.11.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19185813" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liang</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Song</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Ghosh</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Genome-wide survey reveals dynamic widespread tissue-specific changes in DNA methylation during development</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">231</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20survey%20reveals%20dynamic%20widespread%20tissue-specific%20changes%20in%20DNA%20methylation%20during%20development&amp;author=P.%20Liang&amp;author=F.%20Song&amp;author=S.%20Ghosh&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=231" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-231" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-231" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-231"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21569359" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iwase</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Omoto</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Iwata</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">DNA methylation analysis at distal and proximal promoter regions of the oestrogen receptor gene in breast cancers</div>, <div class="source">Br. J. Cancer</div> , <div class="year">1999</div>, vol. <div class="volume">80</div> (pg. <div class="fpage">1982</div>-<div class="lpage">6</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20methylation%20analysis%20at%20distal%20and%20proximal%20promoter%20regions%20of%20the%20oestrogen%20receptor%20gene%20in%20breast%20cancers&amp;author=H.%20Iwase&amp;author=Y.%20Omoto&amp;author=H.%20Iwata&amp;publication_year=1999&amp;journal=Br.%20J.%20Cancer&amp;volume=80&amp;pages=1982-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/sj.bjc.6690631" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsj.bjc.6690631" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsj.bjc.6690631"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10471050" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liu</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Increasing alternative promoter repertories is positively associated with differential expression and disease susceptibility</div>, <div class="source">PLoS One</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e9482</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Increasing%20alternative%20promoter%20repertories%20is%20positively%20associated%20with%20differential%20expression%20and%20disease%20susceptibility&amp;author=S.%20Liu&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e9482" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0009482" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0009482" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0009482"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20208995" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Davuluri</div> <div class="given-names">R.V.</div></div>,  <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Plass</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">T.H.</div></div>. </div><div class="article-title">The functional consequences of alternative promoter use in mammalian genomes</div>, <div class="source">Trends Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">167</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20functional%20consequences%20of%20alternative%20promoter%20use%20in%20mammalian%20genomes&amp;author=R.V.%20Davuluri&amp;author=Y.%20Suzuki&amp;author=S.%20Sugano&amp;author=C.%20Plass&amp;author=T.H.%20Huang&amp;publication_year=2008&amp;journal=Trends%20Genet.&amp;volume=24&amp;pages=167-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2008.01.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2008.01.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2008.01.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18329129" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jarvinen</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Laiho</div> <div class="given-names">M.</div></div>. </div><div class="article-title">LIM-domain proteins in transforming growth factor beta-induced epithelial-to-mesenchymal transition and myofibroblast differentiation</div>, <div class="source">Cell Signal.</div> , <div class="year">2012</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">819</div>-<div class="lpage">25</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=LIM-domain%20proteins%20in%20transforming%20growth%20factor%20beta-induced%20epithelial-to-mesenchymal%20transition%20and%20myofibroblast%20differentiation&amp;author=P.M.%20Jarvinen&amp;author=M.%20Laiho&amp;publication_year=2012&amp;journal=Cell%20Signal.&amp;volume=24&amp;pages=819-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cellsig.2011.12.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cellsig.2011.12.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cellsig.2011.12.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22182513" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DST034C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DST034C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Andrae</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Gallini</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Betsholtz</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Role of platelet-derived growth factors in physiology and medicine</div>, <div class="source">Genes Dev.</div> , <div class="year">2008</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">1276</div>-<div class="lpage">312</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20platelet-derived%20growth%20factors%20in%20physiology%20and%20medicine&amp;author=J.%20Andrae&amp;author=R.%20Gallini&amp;author=C.%20Betsholtz&amp;publication_year=2008&amp;journal=Genes%20Dev.&amp;volume=22&amp;pages=1276-312" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1653708" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1653708" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1653708"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18483217" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Dr Mitsuo Oshimura</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2013. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (http://creativecommons.org/licenses/by/3.0/), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=420907&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst034-dst034supp_table6.doc?Expires=2147483647&amp;Signature=PCwReGSrapEmjcYOp6i9rsVCyXhuS2gPVSw9CdRPHllQhRA5hi1gGlpXunEMmo24FKfjAkCCbH8PgQdxnnT~WKxiSSLBoBryOuuTUZqmaQZggMPfJ8UE-~NyNapavZSuNazXqG3~bjMfHCW7QUKQ4MiubcSU-j5LHT4PyV3AWNNEkTjRpPTUsDWiq8lmoFaAcGCT5aKDFgyD9sUiPKISSoFLe1iqwNj5TEiwTXlx6y8XqpJFW5BrgRyRj2FLabuFe~1371N723BqOKPO57EBKOgDWIMxwpz61S8dCSXCrIwdKvDr6jLckPDLdpig1rFrfvzdOi1pdqUWcqI-5RmrEw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst034-dst034supp_table5.doc?Expires=2147483647&amp;Signature=VZrRcv7uFtiKrUW-osMqDKqWDRcvo4g5l-pma1uN9xgXe-e8TXsDhHNFLwlrTWieuwOVyi8q5nWQ-Fahh0Ev7Q8r8~vEDZP03jm3r-8Al7vt6oAKbK5YZPSV28xDdNtmNC323MQtgZeWJXjN2IDmQgt-eazcZrVFp3o1S69kAwCiQi3sUwvz43zcB0cktfuSNfQPhIt-1PTeqGiYXvREKvnbqyU7dHqSUTPSU0KGGA-g87QTuLV3A51YTra1slSYapAoSqfd6EFkxOlA-T1L6mvLZ3Bm-7kVxnNh37n1~4cwwkr4YdxOPv9N0H0WuHA-ayeaycTLVjurV2SRlxKR~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst034-dst034supp_table4.xlsx?Expires=2147483647&amp;Signature=L0TLSKlS75lH67d4jqFkq8gTpMKxtisn7P~pwVN6wLjNKRH9ur588RLTCc6U4fIoO~ZDmKm9UkDlg9f50WBgS3hCRcySrqIN98ZC1hlThaCiEmtg~FsNkXt7DF7sHyRlhmQtS9wY-MQSFP98rR2nM9Ws5Lf0WsmgLqz4IuH2RKFB0r9efFj9eFJRUk5tYLfiJ0XZrihS6RiFt8o7wmbAId6GKJSjhk97169LZB8hs9eYw4eRlMayxlj6sYFl7MLacurgso7G8stOJeJ3SLd9ZQdhfgOOdOsVIId~NSerKGg-BNR3dhC411VHjcNZsesRipJ-A4WA-hF5zoStHyl~qw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst034-dst034supp_table3.doc?Expires=2147483647&amp;Signature=GO-URho94NPjg23TW4~RFjQWmZ-aWdnM9xlu2plpD8RUmDNsIl4mu1gtqa0mcBc9hbb-c9JlGbD1N6DhU-L6vb8LSdaP-9s7sC-8iIoluKKetv6PLf4E2bx7SZa9YyLteqslJdp2jDDyDssFnD-Wi-efv~gn-SgpgrO283Sj2finhkvKrwSkvfCdHjHZeT2GLjs-0uD5pqsh6j3wxmPO07ra778aVx6xPTzSgR9y4t3YQEuYU05bIevgxUXxWwbnXPTMo~stX6PaMJWx-1Utwq8iAtjW8uWJYHOiSV3ZX7htMrz5SV0arJ0lZQ~5J9s4nLdZw8sb~BMNrP8lwPVhTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst034-dst034supp_table2.doc?Expires=2147483647&amp;Signature=KBLkjxgsF5I5oy3R1FpACjC1HIyJr7dpN49zKLlfoV2unkqRVNdxzvUB~C47T1m45WDm8A8gHLIn7OndPgq6gNylyge6NvqKNMj11~0w0LpWqQzGFtKYkiT11GneKh-msXr5cyQ2GP2OAPDZOz4Lxmj3NQJa3-CbESGiGwmXmiA9T6qyQ-RiZ1NqJC2U8Wsj3Hy133vaUb-IKnM9sfrVedTjYeF1l0shEmYPo7VtJquWQk88b2HQHEI94m0Zl4Odn328tsGqdSuE~nczrXvfoSI17-Esnxp7PBea4AbraCAGoDVgFqGFjrQe7DDTOsr26ze9veDoKwquK6sq5qi8LQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst034-dst034supp_table1.xlsx?Expires=2147483647&amp;Signature=UGEuWNmhd31vv2yUTlT90G3lhk25DdciERLEGX6sEDFrKusXV5DzJAKeODN6saFX5xAObHDNKmESFFieEbQM6-nsjviK-i7H-CuuzgsTgP3qZpml7s91~DewwUBPyDqxzcLJ-ONWpdsbcsAXpgkaGX7VsKqMh7m~fhWX-XAr8rEukRCJ2eaiQ-flr26RExO8LLnOmaStJkx11tnpIG6jrkPDX99KTmecLtP0HUUG9YmRIZBr33IaSBzsLqfdSb1bPw7TlCVuawizSAnMtTms5mc81~CLDGaRG8ChJWWfziY8JHlK9DzgGqpGuEgiKZke~nuONbzsNVbRA2f7FLttQA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst034-dst034supp_file.xlsx?Expires=2147483647&amp;Signature=faUllQEwOb~TIW2H7AFtWz8ufB7jQ9~bWMKeKT6S07Rq9ncxEDrdi7b4L1mFO~WjSy1LQOefz4WG4hJsa0vxF1GCi6kHsJbOjbXHWZqxeCKJq1PRE1u~rZrP-FZPzsiAm~ZEfns3oXmZibvKTebZ762HdQPH79ikYwKYy~E-mSruUxi5QeJGW4Dhue4Bn9il6U7tmxJefae4cs7qd8vNt~fAXf5LU~MJwE7KVRqzVfILkvYGBmtXYqFMIJwB-22z6Oeep9nEub49f1SstMxu7uw1bM1AufOdg0hciWRAum4gXDEbPhGaqN1x7Wql3BpAcjLp4DQSsVcTusgR7Vu6Eg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/20/6/10.1093/dnares/dst034/2/dst034-dst034supp_data.zip?Expires=2147483647&amp;Signature=YLtFfd~akfIEHYnoB7j5tfP5~UbpE9k4teBK3NzBSwSImzOeK91x5wWVo-i4ICjKadV2HHoBrUrbxj4W8KIa8o664y7KDf2x2YefDtjm6016UkRZ0q~7b1M3DrZew2OIIfNnb25HrMW4vuLAN3LrbHUdTnzYDLU-kYPunyrJEZWPAs63kBETzpHzWpBaiSWJJf~SspQxSsq~xw~nSDjTMg2Rd9jY6oyhuXxW8e4YRmUbDp8SB5Yd60iwJxrDVohdWgOS9Q6mrpPcHD1ASHfLLDAGEvh6ocmQ6B0zIX7JNIdjDSjVaHurTW7hfTv1HX7s8OA7B2muUyj2htSiGXXefg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">539</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330168100007&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">11</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dst034"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="420907"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000330168100007&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f20%2f6%2f605%2f420907">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000330168100007&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (11)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F20%2F6%2F605%2F420907" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/420907'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Epigenetic Basis of Regeneration: Analysis of Genomic DNA Methylation Profiles in the MRL/MpJ Mouse" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dst034");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f20%2f6%2f605%2f420907">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

