

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/3/177/444849","@type":"ScholarlyArticle","name":"Exploration for Functional Nucleotide Sequence Candidates within Coding Regions of Mammalian Genes","datePublished":"2011-05-17","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2011-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr010","keywords":["mammal","protein coding","nucleotide conservation"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/3/177/444849","author":[{"name":"Suzuki, Rumiko","affiliation":"1  Department of Genetics, School of Life Science ,  Graduate University for Advanced Studies ,  Mishima 411-8540 ,  Japan    2  Division of Population Genetics ,  National Institute of Genetics ,  Mishima 411-8540 ,  Japan","@type":"Person"},{"name":"Saitou, Naruya","affiliation":"1  Department of Genetics, School of Life Science ,  Graduate University for Advanced Studies ,  Mishima 411-8540 ,  Japan    2  Division of Population Genetics ,  National Institute of Genetics ,  Mishima 411-8540 ,  Japan","@type":"Person"}],"description":"Abstract.  The primary role of a protein coding gene is to encode amino acids. Therefore, synonymous sites of codons, which do not change the encoded amino acid","pageStart":"177","pageEnd":"187","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/3/1/m_cover.gif?Expires=2147483647&Signature=wj9vBfD3w2y4NVjjTT0hEq0DGILHhVJxl~hGkTYdv3f3RyfzKeaufbdZYgMTMKQvtZKo6x4TIuq0dMkBwN~dx6uKMb2t7UtWPZaFFD84b7m4O-CF7vPeyaegVwS-GVQTx9~LaL1HF9CGtMgbMthYT6eCwup0WynCZrD9syn~ryIjdMCj0Vfd6c~OnMKAAEkNIbY-5LT2dVafsguUGzIjP~lnGiuCdOABMm56~fJ~khgnHeAXFCc89K-eA167vdlnePhOZRt6~sb9AVRfnOlCiFtENgEc2O8AEAnKUMBucCj6xmHIeTCmvFwDpxMdx1ubEwugqbONa2xb-e6HSc9wlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Exploration for Functional Nucleotide Sequence Candidates within Coding Regions of Mammalian Genes","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/3/1/m_cover.gif?Expires=2147483647&Signature=wj9vBfD3w2y4NVjjTT0hEq0DGILHhVJxl~hGkTYdv3f3RyfzKeaufbdZYgMTMKQvtZKo6x4TIuq0dMkBwN~dx6uKMb2t7UtWPZaFFD84b7m4O-CF7vPeyaegVwS-GVQTx9~LaL1HF9CGtMgbMthYT6eCwup0WynCZrD9syn~ryIjdMCj0Vfd6c~OnMKAAEkNIbY-5LT2dVafsguUGzIjP~lnGiuCdOABMm56~fJ~khgnHeAXFCc89K-eA167vdlnePhOZRt6~sb9AVRfnOlCiFtENgEc2O8AEAnKUMBucCj6xmHIeTCmvFwDpxMdx1ubEwugqbONa2xb-e6HSc9wlg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Exploration for Functional Nucleotide Sequence Candidates within Coding Regions of Mammalian Genes" />
    <meta property="og:description" content="Abstract.  The primary role of a protein coding gene is to encode amino acids. Therefore, synonymous sites of codons, which do not change the encoded amino acid" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/3/177/444849" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-05-17" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Suzuki, Rumiko" /><meta name="citation_author_institution" content="Department of Genetics, School of Life Science, Graduate University for Advanced Studies, Mishima 411-8540, Japan" /><meta name="citation_author_institution" content="Division of Population Genetics, National Institute of Genetics, Mishima 411-8540, Japan" /><meta name="citation_author" content="Saitou, Naruya" /><meta name="citation_author_institution" content="Department of Genetics, School of Life Science, Graduate University for Advanced Studies, Mishima 411-8540, Japan" /><meta name="citation_author_institution" content="Division of Population Genetics, National Institute of Genetics, Mishima 411-8540, Japan" /><meta name="citation_title" content="Exploration for Functional Nucleotide Sequence Candidates within Coding Regions of Mammalian Genes" /><meta name="citation_firstpage" content="177" /><meta name="citation_lastpage" content="187" /><meta name="citation_doi" content="10.1093/dnares/dsr010" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2011/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_publisher=Cambridge University Press, Cambridge; The Neutral Theory of Molecular Evolution;  citation_year=1983; " /><meta name="citation_reference" content="citation_publisher=Columbia Univ. Press, New York; Molecular Evolutionary Genetics;  citation_year=1987; " /><meta name="citation_reference" content="citation_title=Correlation between the abundance of yeast transfer RNAs and the occurrence of the respective codons in protein genes. Differences in synonymous codon choice patterns of yeast and Escherichia coli with reference to the abundance of isoaccepting transfer RNAs; J. Mol. Biol.;  citation_year=1982;  citation_volume=158;  citation_pages=573-97; " /><meta name="citation_reference" content="citation_title=Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the E. coli translational system; J. Mol. Biol.;  citation_year=1981;  citation_volume=151;  citation_pages=389-409; " /><meta name="citation_reference" content="citation_title=Codon usage and tRNA genes in eukaryotes: correlation of codon usage diversity with translation efficiency and with CG-dinucleotide usage as assessed by multivariate analysis; J. Mol. Evol.;  citation_year=2001;  citation_volume=53;  citation_pages=290-8; " /><meta name="citation_reference" content="citation_title=Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy; Genetics;  citation_year=1994;  citation_volume=136;  citation_pages=927-35; " /><meta name="citation_reference" content="citation_title=Codon bias and gene expression; FEBS Lett.;  citation_year=1991;  citation_volume=285;  citation_pages=165-9; " /><meta name="citation_reference" content="citation_title=Selection on codon bias; Annu. Rev. Genet.;  citation_year=2008;  citation_volume=42;  citation_pages=287-99; " /><meta name="citation_reference" content="citation_title=Codon usage and tRNA content in unicellular and multicellular organisms; Mol. Biol. Evol.;  citation_year=1985;  citation_volume=2;  citation_pages=13-34; " /><meta name="citation_reference" content="citation_title=The rate of synonymous substitution in enterobacterial genes is inversely related to codon usage bias; Mol. Biol. Evol.;  citation_year=1987;  citation_volume=4;  citation_pages=222-30; " /><meta name="citation_reference" content="citation_title=Translational selection and yeast proteome evolution; Genetics;  citation_year=2003;  citation_volume=164;  citation_pages=1291-303; " /><meta name="citation_reference" content="citation_title=An analysis of codon usage in mammals: selection or mutation bias?; J. Mol. Evol.;  citation_year=1991;  citation_volume=33;  citation_pages=442-9; " /><meta name="citation_reference" content="citation_title=DNA sequence evolution: the sounds of silence; Philos. Trans. R Soc. Lond. B Biol. Sci.;  citation_year=1995;  citation_volume=349;  citation_pages=241-7; " /><meta name="citation_reference" content="citation_title=Initial splice-site recognition and pairing during pre-mRNA splicing; Curr. Opin. Genet. Dev.;  citation_year=1996;  citation_volume=6;  citation_pages=215-20; " /><meta name="citation_reference" content="citation_title=Exonic splicing enhancers: mechanism of action, diversity and role in human genetic diseases; Trends Biochem. Sci;  citation_year=2000;  citation_volume=25;  citation_pages=106-10; " /><meta name="citation_reference" content="citation_title=Effect of exonic splicing regulation on synonymous codon usage in alternatively spliced exons of Dscam; BMC Evol. Biol;  citation_year=2009;  citation_volume=9;  citation_pages=214" /><meta name="citation_reference" content="citation_title=Exonic splicing regulatory elements skew synonymous codon usage near intron-exon boundaries in mammals; Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=1600-3; " /><meta name="citation_reference" content="citation_title=Ultraconserved elements in the human genome; Science;  citation_year=2004;  citation_volume=304;  citation_pages=1321-5; " /><meta name="citation_reference" content="citation_title=Regions of extreme synonymous codon selection in mammalian genes; Nucleic Acids Res;  citation_year=2006;  citation_volume=34;  citation_pages=1700-10; " /><meta name="citation_reference" content="citation_title=Unproductive splicing of SR genes associated with highly conserved and ultraconserved DNA elements; Nature;  citation_year=2007;  citation_volume=446;  citation_pages=926-9; " /><meta name="citation_reference" content="citation_title=Ultraconserved coding regions outside the homeobox of mammalian Hox genes; BMC Evol. Biol;  citation_year=2008;  citation_volume=8;  citation_pages=260" /><meta name="citation_reference" content="citation_title=An ultraconserved Hox-Pbx responsive element resides in the coding sequence of Hoxa2 and is active in rhombomere 4; Nucleic Acids Res;  citation_year=2008;  citation_volume=36;  citation_pages=3214-25; " /><meta name="citation_reference" content="citation_title=RNA processing and its regulation: global insights into biological networks; Nat. Rev. Genet;  citation_year=2010;  citation_volume=11;  citation_pages=75-87; " /><meta name="citation_reference" content="citation_title=Control of human potassium channel inactivation by editing of a small mRNA hairpin; Nat. Struct. Mol. Biol;  citation_year=2004;  citation_volume=11;  citation_pages=950-6; " /><meta name="citation_reference" content="citation_title=Spi-1/PU.1 proto-oncogene induces opposite effects on monocytic and erythroid differentiation of K562 cells; Biochem. Biophys. Res. Commun;  citation_year=1998;  citation_volume=252;  citation_pages=383-91; " /><meta name="citation_reference" content="citation_title=Identification of a conserved sequence in the non-coding regions of many human genes; Nucleic. Acids Res;  citation_year=1989;  citation_volume=17;  citation_pages=699-710; " /><meta name="citation_reference" content="citation_title=Making (anti)sense of non-coding sequence conservation; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3580-3; " /><meta name="citation_reference" content="citation_title=Enrichment of regulatory signals in conserved non-coding genomic sequence; Bioinformatics;  citation_year=2001;  citation_volume=17;  citation_pages=871-7; " /><meta name="citation_reference" content="citation_title=Arrays of ultraconserved non-coding regions span the loci of key developmental genes in vertebrate genomes; BMC Genomics;  citation_year=2004;  citation_volume=5;  citation_pages=99" /><meta name="citation_reference" content="citation_title=Small fitness effect of mutations in highly conserved non-coding regions; Hum. Mol. Genet;  citation_year=2005;  citation_volume=14;  citation_pages=2221-9; " /><meta name="citation_reference" content="citation_title=Ensembl 2007; Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=D610-7; " /><meta name="citation_reference" content="citation_title=CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice; Nucleic Acids Res;  citation_year=1994;  citation_volume=22;  citation_pages=4673-80; " /><meta name="citation_reference" content="citation_title=Controlling the false discovery rate in behavior genetics research; Behav. Brain Res;  citation_year=2001;  citation_volume=125;  citation_pages=279-84; " /><meta name="citation_reference" content="citation_title=Predictive identification of exonic splicing enhancers in human genes; Science;  citation_year=2002;  citation_volume=297;  citation_pages=1007-13; " /><meta name="citation_reference" content="citation_title=Tissue-specific codon usage and the expression of human genes; Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=12588-91; " /><meta name="citation_reference" content="citation_title=Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution; Cell;  citation_year=2008;  citation_volume=134;  citation_pages=341-52; " /><meta name="citation_reference" content="citation_title=Synonymous and nonsynonymous substitution distances are correlated in mouse and rat genes; J. Mol. Evol;  citation_year=1998;  citation_volume=47;  citation_pages=119-21; " /><meta name="citation_reference" content="citation_title=R: a language for data analysis and graphics; J. Comp. Graph. Stat;  citation_year=1996;  citation_volume=5;  citation_pages=299-314; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/3/177/1217240/dsr010.pdf" /><meta name="description" content="Abstract.  The primary role of a protein coding gene is to encode amino acids. Therefore, synonymous sites of codons, which do not change the encoded amino acid" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/3/177/444849" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/3/177/444849" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/3/177/444849");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr010';
    </script>
<title>
	Exploration for Functional Nucleotide Sequence Candidates within Coding Regions of Mammalian Genes | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./444849" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfRRPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdXp1a2ksIFJ1bWlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgR2VuZXRpY3MsIFNjaG9vbCBvZiBMaWZlIFNjaWVuY2UsIEdyYWR1YXRlIFVuaXZlcnNpdHkgZm9yIEFkdmFuY2VkIFN0dWRpZXMsIE1pc2hpbWEgNDExLTg1NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgUG9wdWxhdGlvbiBHZW5ldGljcywgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEdlbmV0aWNzLCBNaXNoaW1hIDQxMS04NTQwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNhaXRvdSwgTmFydXlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBHZW5ldGljcywgU2Nob29sIG9mIExpZmUgU2NpZW5jZSwgR3JhZHVhdGUgVW5pdmVyc2l0eSBmb3IgQWR2YW5jZWQgU3R1ZGllcywgTWlzaGltYSA0MTEtODU0MCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEaXZpc2lvbiBvZiBQb3B1bGF0aW9uIEdlbmV0aWNzLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgR2VuZXRpY3MsIE1pc2hpbWEgNDExLTg1NDAsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJFeHBsb3JhdGlvbiBmb3IgRnVuY3Rpb25hbCBOdWNsZW90aWRlIFNlcXVlbmNlIENhbmRpZGF0ZXMgd2l0aGluIENvZGluZyBSZWdpb25zIG9mIE1hbW1hbGlhbiBHZW5lcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE3NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTg3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNyMDEwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTEvMDYvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q2FtYnJpZGdlIFVuaXZlcnNpdHkgUHJlc3MsIENhbWJyaWRnZTsgVGhlIE5ldXRyYWwgVGhlb3J5IG9mIE1vbGVjdWxhciBFdm9sdXRpb247ICBjaXRhdGlvbl95ZWFyPTE5ODM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1Db2x1bWJpYSBVbml2LiBQcmVzcywgTmV3IFlvcms7IE1vbGVjdWxhciBFdm9sdXRpb25hcnkgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5ODc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvcnJlbGF0aW9uIGJldHdlZW4gdGhlIGFidW5kYW5jZSBvZiB5ZWFzdCB0cmFuc2ZlciBSTkFzIGFuZCB0aGUgb2NjdXJyZW5jZSBvZiB0aGUgcmVzcGVjdGl2ZSBjb2RvbnMgaW4gcHJvdGVpbiBnZW5lcy4gRGlmZmVyZW5jZXMgaW4gc3lub255bW91cyBjb2RvbiBjaG9pY2UgcGF0dGVybnMgb2YgeWVhc3QgYW5kIEVzY2hlcmljaGlhIGNvbGkgd2l0aCByZWZlcmVuY2UgdG8gdGhlIGFidW5kYW5jZSBvZiBpc29hY2NlcHRpbmcgdHJhbnNmZXIgUk5BczsgSi4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk4MjsgIGNpdGF0aW9uX3ZvbHVtZT0xNTg7ICBjaXRhdGlvbl9wYWdlcz01NzMtOTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvcnJlbGF0aW9uIGJldHdlZW4gdGhlIGFidW5kYW5jZSBvZiBFc2NoZXJpY2hpYSBjb2xpIHRyYW5zZmVyIFJOQXMgYW5kIHRoZSBvY2N1cnJlbmNlIG9mIHRoZSByZXNwZWN0aXZlIGNvZG9ucyBpbiBpdHMgcHJvdGVpbiBnZW5lczogYSBwcm9wb3NhbCBmb3IgYSBzeW5vbnltb3VzIGNvZG9uIGNob2ljZSB0aGF0IGlzIG9wdGltYWwgZm9yIHRoZSBFLiBjb2xpIHRyYW5zbGF0aW9uYWwgc3lzdGVtOyBKLiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTE1MTsgIGNpdGF0aW9uX3BhZ2VzPTM4OS00MDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvZG9uIHVzYWdlIGFuZCB0Uk5BIGdlbmVzIGluIGV1a2FyeW90ZXM6IGNvcnJlbGF0aW9uIG9mIGNvZG9uIHVzYWdlIGRpdmVyc2l0eSB3aXRoIHRyYW5zbGF0aW9uIGVmZmljaWVuY3kgYW5kIHdpdGggQ0ctZGludWNsZW90aWRlIHVzYWdlIGFzIGFzc2Vzc2VkIGJ5IG11bHRpdmFyaWF0ZSBhbmFseXNpczsgSi4gTW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTI5MC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeW5vbnltb3VzIGNvZG9uIHVzYWdlIGluIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyOiBuYXR1cmFsIHNlbGVjdGlvbiBhbmQgdHJhbnNsYXRpb25hbCBhY2N1cmFjeTsgR2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTM2OyAgY2l0YXRpb25fcGFnZXM9OTI3LTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db2RvbiBiaWFzIGFuZCBnZW5lIGV4cHJlc3Npb247IEZFQlMgTGV0dC47ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9Mjg1OyAgY2l0YXRpb25fcGFnZXM9MTY1LTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlbGVjdGlvbiBvbiBjb2RvbiBiaWFzOyBBbm51LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00MjsgIGNpdGF0aW9uX3BhZ2VzPTI4Ny05OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29kb24gdXNhZ2UgYW5kIHRSTkEgY29udGVudCBpbiB1bmljZWxsdWxhciBhbmQgbXVsdGljZWxsdWxhciBvcmdhbmlzbXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5ODU7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTEzLTM0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgcmF0ZSBvZiBzeW5vbnltb3VzIHN1YnN0aXR1dGlvbiBpbiBlbnRlcm9iYWN0ZXJpYWwgZ2VuZXMgaXMgaW52ZXJzZWx5IHJlbGF0ZWQgdG8gY29kb24gdXNhZ2UgYmlhczsgTW9sLiBCaW9sLiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk4NzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9MjIyLTMwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2xhdGlvbmFsIHNlbGVjdGlvbiBhbmQgeWVhc3QgcHJvdGVvbWUgZXZvbHV0aW9uOyBHZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xNjQ7ICBjaXRhdGlvbl9wYWdlcz0xMjkxLTMwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYW5hbHlzaXMgb2YgY29kb24gdXNhZ2UgaW4gbWFtbWFsczogc2VsZWN0aW9uIG9yIG11dGF0aW9uIGJpYXM/OyBKLiBNb2wuIEV2b2wuOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9NDQyLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUROQSBzZXF1ZW5jZSBldm9sdXRpb246IHRoZSBzb3VuZHMgb2Ygc2lsZW5jZTsgUGhpbG9zLiBUcmFucy4gUiBTb2MuIExvbmQuIEIgQmlvbC4gU2NpLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0zNDk7ICBjaXRhdGlvbl9wYWdlcz0yNDEtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW5pdGlhbCBzcGxpY2Utc2l0ZSByZWNvZ25pdGlvbiBhbmQgcGFpcmluZyBkdXJpbmcgcHJlLW1STkEgc3BsaWNpbmc7IEN1cnIuIE9waW4uIEdlbmV0LiBEZXYuOyAgY2l0YXRpb25feWVhcj0xOTk2OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz0yMTUtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4b25pYyBzcGxpY2luZyBlbmhhbmNlcnM6IG1lY2hhbmlzbSBvZiBhY3Rpb24sIGRpdmVyc2l0eSBhbmQgcm9sZSBpbiBodW1hbiBnZW5ldGljIGRpc2Vhc2VzOyBUcmVuZHMgQmlvY2hlbS4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MTA2LTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZlY3Qgb2YgZXhvbmljIHNwbGljaW5nIHJlZ3VsYXRpb24gb24gc3lub255bW91cyBjb2RvbiB1c2FnZSBpbiBhbHRlcm5hdGl2ZWx5IHNwbGljZWQgZXhvbnMgb2YgRHNjYW07IEJNQyBFdm9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0yMTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FeG9uaWMgc3BsaWNpbmcgcmVndWxhdG9yeSBlbGVtZW50cyBza2V3IHN5bm9ueW1vdXMgY29kb24gdXNhZ2UgbmVhciBpbnRyb24tZXhvbiBib3VuZGFyaWVzIGluIG1hbW1hbHM7IE1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTE2MDAtMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VWx0cmFjb25zZXJ2ZWQgZWxlbWVudHMgaW4gdGhlIGh1bWFuIGdlbm9tZTsgU2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDQ7ICBjaXRhdGlvbl9wYWdlcz0xMzIxLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ2lvbnMgb2YgZXh0cmVtZSBzeW5vbnltb3VzIGNvZG9uIHNlbGVjdGlvbiBpbiBtYW1tYWxpYW4gZ2VuZXM7IE51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9MTcwMC0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VW5wcm9kdWN0aXZlIHNwbGljaW5nIG9mIFNSIGdlbmVzIGFzc29jaWF0ZWQgd2l0aCBoaWdobHkgY29uc2VydmVkIGFuZCB1bHRyYWNvbnNlcnZlZCBETkEgZWxlbWVudHM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00NDY7ICBjaXRhdGlvbl9wYWdlcz05MjYtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VWx0cmFjb25zZXJ2ZWQgY29kaW5nIHJlZ2lvbnMgb3V0c2lkZSB0aGUgaG9tZW9ib3ggb2YgbWFtbWFsaWFuIEhveCBnZW5lczsgQk1DIEV2b2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTI2MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIHVsdHJhY29uc2VydmVkIEhveC1QYnggcmVzcG9uc2l2ZSBlbGVtZW50IHJlc2lkZXMgaW4gdGhlIGNvZGluZyBzZXF1ZW5jZSBvZiBIb3hhMiBhbmQgaXMgYWN0aXZlIGluIHJob21ib21lcmUgNDsgTnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz0zMjE0LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1STkEgcHJvY2Vzc2luZyBhbmQgaXRzIHJlZ3VsYXRpb246IGdsb2JhbCBpbnNpZ2h0cyBpbnRvIGJpb2xvZ2ljYWwgbmV0d29ya3M7IE5hdC4gUmV2LiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTc1LTg3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db250cm9sIG9mIGh1bWFuIHBvdGFzc2l1bSBjaGFubmVsIGluYWN0aXZhdGlvbiBieSBlZGl0aW5nIG9mIGEgc21hbGwgbVJOQSBoYWlycGluOyBOYXQuIFN0cnVjdC4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9OTUwLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNwaS0xL1BVLjEgcHJvdG8tb25jb2dlbmUgaW5kdWNlcyBvcHBvc2l0ZSBlZmZlY3RzIG9uIG1vbm9jeXRpYyBhbmQgZXJ5dGhyb2lkIGRpZmZlcmVudGlhdGlvbiBvZiBLNTYyIGNlbGxzOyBCaW9jaGVtLiBCaW9waHlzLiBSZXMuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNTI7ICBjaXRhdGlvbl9wYWdlcz0zODMtOTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGEgY29uc2VydmVkIHNlcXVlbmNlIGluIHRoZSBub24tY29kaW5nIHJlZ2lvbnMgb2YgbWFueSBodW1hbiBnZW5lczsgTnVjbGVpYy4gQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9Njk5LTcxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFraW5nIChhbnRpKXNlbnNlIG9mIG5vbi1jb2Rpbmcgc2VxdWVuY2UgY29uc2VydmF0aW9uOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0zNTgwLTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVucmljaG1lbnQgb2YgcmVndWxhdG9yeSBzaWduYWxzIGluIGNvbnNlcnZlZCBub24tY29kaW5nIGdlbm9taWMgc2VxdWVuY2U7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9ODcxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFycmF5cyBvZiB1bHRyYWNvbnNlcnZlZCBub24tY29kaW5nIHJlZ2lvbnMgc3BhbiB0aGUgbG9jaSBvZiBrZXkgZGV2ZWxvcG1lbnRhbCBnZW5lcyBpbiB2ZXJ0ZWJyYXRlIGdlbm9tZXM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9OTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TbWFsbCBmaXRuZXNzIGVmZmVjdCBvZiBtdXRhdGlvbnMgaW4gaGlnaGx5IGNvbnNlcnZlZCBub24tY29kaW5nIHJlZ2lvbnM7IEh1bS4gTW9sLiBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTIyMjEtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RW5zZW1ibCAyMDA3OyBOdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zNTsgIGNpdGF0aW9uX3BhZ2VzPUQ2MTAtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0xVU1RBTCBXOiBpbXByb3ZpbmcgdGhlIHNlbnNpdGl2aXR5IG9mIHByb2dyZXNzaXZlIG11bHRpcGxlIHNlcXVlbmNlIGFsaWdubWVudCB0aHJvdWdoIHNlcXVlbmNlIHdlaWdodGluZywgcG9zaXRpb24tc3BlY2lmaWMgZ2FwIHBlbmFsdGllcyBhbmQgd2VpZ2h0IG1hdHJpeCBjaG9pY2U7IE51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0xOTk0OyAgY2l0YXRpb25fdm9sdW1lPTIyOyAgY2l0YXRpb25fcGFnZXM9NDY3My04MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29udHJvbGxpbmcgdGhlIGZhbHNlIGRpc2NvdmVyeSByYXRlIGluIGJlaGF2aW9yIGdlbmV0aWNzIHJlc2VhcmNoOyBCZWhhdi4gQnJhaW4gUmVzOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTEyNTsgIGNpdGF0aW9uX3BhZ2VzPTI3OS04NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJlZGljdGl2ZSBpZGVudGlmaWNhdGlvbiBvZiBleG9uaWMgc3BsaWNpbmcgZW5oYW5jZXJzIGluIGh1bWFuIGdlbmVzOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTI5NzsgIGNpdGF0aW9uX3BhZ2VzPTEwMDctMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRpc3N1ZS1zcGVjaWZpYyBjb2RvbiB1c2FnZSBhbmQgdGhlIGV4cHJlc3Npb24gb2YgaHVtYW4gZ2VuZXM7IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTEyNTg4LTkxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaXN0cmFuc2xhdGlvbi1pbmR1Y2VkIHByb3RlaW4gbWlzZm9sZGluZyBhcyBhIGRvbWluYW50IGNvbnN0cmFpbnQgb24gY29kaW5nLXNlcXVlbmNlIGV2b2x1dGlvbjsgQ2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMzQ7ICBjaXRhdGlvbl9wYWdlcz0zNDEtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVN5bm9ueW1vdXMgYW5kIG5vbnN5bm9ueW1vdXMgc3Vic3RpdHV0aW9uIGRpc3RhbmNlcyBhcmUgY29ycmVsYXRlZCBpbiBtb3VzZSBhbmQgcmF0IGdlbmVzOyBKLiBNb2wuIEV2b2w7ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz0xMTktMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVI6IGEgbGFuZ3VhZ2UgZm9yIGRhdGEgYW5hbHlzaXMgYW5kIGdyYXBoaWNzOyBKLiBDb21wLiBHcmFwaC4gU3RhdDsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9Mjk5LTMxNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE4LzMvMTc3LzEyMTcyNDAvZHNyMDEwLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgcHJpbWFyeSByb2xlIG9mIGEgcHJvdGVpbiBjb2RpbmcgZ2VuZSBpcyB0byBlbmNvZGUgYW1pbm8gYWNpZHMuIFRoZXJlZm9yZSwgc3lub255bW91cyBzaXRlcyBvZiBjb2RvbnMsIHdoaWNoIGRvIG5vdCBjaGFuZ2UgdGhlIGVuY29kZWQgYW1pbm8gYWNpZCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDQ0ODQ5JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZM9dO0ng3FPrPPpicxbnlHtC9F6m" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f177%2f444849" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f177%2f444849" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17382" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/3/1/m_cover.gif?Expires=2147483647&amp;Signature=wj9vBfD3w2y4NVjjTT0hEq0DGILHhVJxl~hGkTYdv3f3RyfzKeaufbdZYgMTMKQvtZKo6x4TIuq0dMkBwN~dx6uKMb2t7UtWPZaFFD84b7m4O-CF7vPeyaegVwS-GVQTx9~LaL1HF9CGtMgbMthYT6eCwup0WynCZrD9syn~ryIjdMCj0Vfd6c~OnMKAAEkNIbY-5LT2dVafsguUGzIjP~lnGiuCdOABMm56~fJ~khgnHeAXFCc89K-eA167vdlnePhOZRt6~sb9AVRfnOlCiFtENgEc2O8AEAnKUMBucCj6xmHIeTCmvFwDpxMdx1ubEwugqbONa2xb-e6HSc9wlg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/3">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6650479">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6650479">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6650481">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6650481">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6650487">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6650487">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6650502">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6650502">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6650543">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6650543">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6650549">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6650549">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6650551">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6650551">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6650553">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6650553">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6650555">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6650555">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/3/165/444512">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/18/3/189/445126">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Exploration for Functional Nucleotide Sequence Candidates within Coding Regions of Mammalian Genes
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rumiko Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Genetics, School of Life Science</div>, <div class="insititution">Graduate University for Advanced Studies</div>, <div class="addr-line">Mishima 411-8540</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Division of Population Genetics</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Mishima 411-8540</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rumiko+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rumiko Suzuki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naruya Saitou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Genetics, School of Life Science</div>, <div class="insititution">Graduate University for Advanced Studies</div>, <div class="addr-line">Mishima 411-8540</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Division of Population Genetics</div>, <div class="insititution">National Institute of Genetics</div>, <div class="addr-line">Mishima 411-8540</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 559-81-6790. Fax. +81 559-81-6789. E-mail: <a href="mailto:saitounr@lab.nig.ac.jp" target="_blank">saitounr@lab.nig.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naruya+Saitou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Saitou N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Saitou N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Naruya Saitou</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: Department of Environmental and Preventive Medicine, Oita University, Yufu 879-5593, Japan.</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 3, 1 June 2011, Pages 177–187, <a href='https://doi.org/10.1093/dnares/dsr010'>https://doi.org/10.1093/dnares/dsr010</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">17 May 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">12 September 2010</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">14 April 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="444849" href="/dnaresearch/article-pdf/18/3/177/1217240/dsr010.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Rumiko Suzuki, Naruya Saitou;  Exploration for Functional Nucleotide Sequence Candidates within Coding Regions of Mammalian Genes, <em>DNA Research</em>, Volume 18, Issue 3, 1 June 2011, Pages 177–187, <a href='https://doi.org/10.1093/dnares/dsr010'>https://doi.org/10.1093/dnares/dsr010</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/444849?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/444849?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/444849?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/444849?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/444849?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/444849?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/444849?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Exploration%20for%20Functional%20Nucleotide%20Sequence%20Candidates%20within%20Coding%20Regions%20of%20Mammalian%20Genes&amp;publicationDate=2011-05-17&amp;volumeNum=18&amp;issueNum=3&amp;author=Suzuki%2C%20Rumiko%3B%20Saitou%2C%20Naruya&amp;startPage=177&amp;endPage=187&amp;contentId=10.1093%2Fdnares%2Fdsr010&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f177%2f444849" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6650479 id="6650479" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The primary role of a protein coding gene is to encode amino acids. Therefore, synonymous sites of codons, which do not change the encoded amino acid, are regarded as evolving neutrally. However, if a certain region of a protein coding gene contains a functional nucleotide element (e.g. splicing signals), synonymous sites in the region may have selective pressure. The existence of such elements would be detected by searching regions of low nucleotide substitution. We explored invariant nucleotide sequences in 10 790 orthologous genes of six mammalian species (<em>Homo sapiens</em>, <em>Macaca mulatta</em>, <em>Mus musculus</em>, <em>Rattus norvegicus</em>, <em>Bos taurus</em>, and <em>Canis familiaris</em>), and extracted 4150 sequences whose conservation is significantly stronger than other regions of the gene and named them significantly conserved coding sequences (SCCSs). SCCSs are observed in 2273 genes. The genes are mainly involved with development, transcriptional regulation, and the neurons, and are expressed in the nervous system and the head and neck organs. No strong influence of conventional factors that affect synonymous substitution was observed in SCCSs. These results imply that SCCSs may have double function as nucleotide element and protein coding sequence and retained in the course of mammalian evolution.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="mammal">mammal</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;protein coding&quot;">protein coding</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;nucleotide conservation&quot;">nucleotide conservation</a></div>                            <h2 scrollto-destination=6650481 id="6650481" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The neutral theory of molecular evolution<sup><a reveal-id="DSR010C1" data-open="DSR010C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSR010C2" data-open="DSR010C2" class="link link-ref link-reveal xref-bibr">2</a></sup> predicts that synonymous sites of codons are evolving faster than non-synonymous sites because of the smaller selective pressure. This is true in general; however, several factors are known to influence on a certain region of a coding sequence and suppress synonymous substitution.</p><p>One of the well-known factors is the codon bias towards optimum codons. Optimum codons reflect the composition of genomic tRNA pool.<sup><a reveal-id="DSR010C3  DSR010C5" data-open="DSR010C3  DSR010C5" class="link link-ref link-reveal xref-bibr">3–5</a></sup> Because optimum codons are advantageous for fast and accurate translation, highly expressed or biologically important genes would have more optimum codons than others.<sup><a reveal-id="DSR010C6  DSR010C8" data-open="DSR010C6  DSR010C8" class="link link-ref link-reveal xref-bibr">6–8</a></sup> Changes from an optimum codon to a non-optimum codon will be suppressed in such genes. Because optimum codons are similar in closely related species, highly expressed genes tend to show similar codon usage; therefore synonymous substitution is lowered. In fact, the requirement for translational efficiency or accuracy enhances the optimum codon usage and suppresses nucleotide changes through purifying selection.<sup><a reveal-id="DSR010C5" data-open="DSR010C5" class="link link-ref link-reveal xref-bibr">5</a>,<a reveal-id="DSR010C6" data-open="DSR010C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSR010C9  DSR010C11" data-open="DSR010C9  DSR010C11" class="link link-ref link-reveal xref-bibr">9–11</a></sup> Codon bias towards optimum codons is strong in fast-growing organisms such as <em>Escherichia coli</em> or <em>Saccharomyces cerevisiae</em>, but generally weak in species with slow growth rate small population size.<sup><a reveal-id="DSR010C12" data-open="DSR010C12" class="link link-ref link-reveal xref-bibr">12</a>,<a reveal-id="DSR010C13" data-open="DSR010C13" class="link link-ref link-reveal xref-bibr">13</a></sup></p><p>Another factor is exonic splicing enhancer or silencer, which are splicing signals embedded in exons.<sup><a reveal-id="DSR010C14" data-open="DSR010C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSR010C15" data-open="DSR010C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Existence of such elements lowers the synonymous substitution.<sup><a reveal-id="DSR010C16" data-open="DSR010C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSR010C17" data-open="DSR010C17" class="link link-ref link-reveal xref-bibr">17</a></sup> In addition, ultraconserved elements (UCEs), which majorly reside in non-protein coding regions, sometimes extend to coding regions.<sup><a reveal-id="DSR010C18" data-open="DSR010C18" class="link link-ref link-reveal xref-bibr">18</a></sup> In mammals, UCEs are reported to exist near to or overlap with genes associated with nucleotide binding, transcriptional regulation, RNA recognition motif, zinc finger domain, and homeobox domain.<sup><a reveal-id="DSR010C18  DSR010C20" data-open="DSR010C18  DSR010C20" class="link link-ref link-reveal xref-bibr">18–20</a></sup> Hox genes also contain long conserved nucleotide regions other than UCEs outside the homeobox domain.<sup><a reveal-id="DSR010C21" data-open="DSR010C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p><p>Although the primary role of protein coding region is to encode amino acids, there may be also functional nucleotide elements embedded within coding regions. For example, transcription-factor-binding sites are found in coding regions,<sup><a reveal-id="DSR010C22" data-open="DSR010C22" class="link link-ref link-reveal xref-bibr">22</a></sup> messenger RNAs are targeted by various post-transcriptional regulations,<sup><a reveal-id="DSR010C23" data-open="DSR010C23" class="link link-ref link-reveal xref-bibr">23</a></sup> and the requirement for a specific secondary structure for RNA editing decreases synonymous substitution.<sup><a reveal-id="DSR010C24" data-open="DSR010C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSR010C25" data-open="DSR010C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p><p>Functional nucleotide elements are extensively explored in the non-coding regions,<sup><a reveal-id="DSR010C26  DSR010C30" data-open="DSR010C26  DSR010C30" class="link link-ref link-reveal xref-bibr">26–30</a></sup> but less studies have been done to explore probable functional elements within the coding regions. We extracted significantly conserved coding sequences (SCCSs) from orthologous genes of six mammalian species (human, rhesus macaque, mouse, rat, cow, and dog), and compared genes containing SCCSs and genes without SCCSs. Analyses on gene ontology (GO), InterPro codes, and KEGG pathways enlighten difference between the two gene groups. We also investigated RNA secondary structures, codon preference, GC content, exonic splicing signals, and gene expression of SCCSs to survey the influence of these factors.</p>                            <h2 scrollto-destination=6650487 id="6650487" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6650488 id="6650488" class="section-title" data-legacy-id=s2a>2.1. Genome data</h3>
<p>We obtained peptide and nucleotide sequences of orthologous genes of six mammalian species (<em>Homo sapiens</em>, <em>Macaca mulatta</em>, <em>Mus musculus</em>, <em>Rattus norvegicus</em>, <em>Bos taurus</em>, and <em>Canis familiaris</em>) from Ensembl database<sup><a reveal-id="DSR010C31" data-open="DSR010C31" class="link link-ref link-reveal xref-bibr">31</a></sup> version 54 (<a class="link link-uri" href="http://May2009.archive.ensembl.org/index.html">http://May2009.archive.ensembl.org/index.html</a>). These species are selected considering genome data quality and evolutionary diversity. We selected one-to-one type single copy orthologs and compiled 10 790 orthologous gene sets (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary file S1</a>). We then constructed multiple alignments of peptide sequences using ClustalW<sup><a reveal-id="DSR010C32" data-open="DSR010C32" class="link link-ref link-reveal xref-bibr">32</a></sup> and constructed nucleotide alignments based on the peptide alignments. From the nucleotide alignments, we extracted sequences that are invariant among all the species.</p>                            <h3 scrollto-destination=6650490 id="6650490" class="section-title" data-legacy-id=s2b>2.2. Identification of SCCSs</h3>
<p>We performed permutation simulation to identify SCCSs, or abbreviated as SCCSs, which are invariant longer than 10 codons. This length is set to confine the permutation run time within a feasible range. For an <em>N</em>-codon long alignment, we generated a non-redundant series of random numbers from 1 to <em>N</em>, and permuted codon columns (rows of codons in the same site of the alignment) according to the generated random numbers. In this process, gap sites are fixed and the rest of the sites are permuted. Then the length and numbers of invariant sequences in the permuted alignment are counted. We repeated this process 500 000 times per ortholog set and took averages of the frequency of invariant sequences. We used the length and averaged frequency of invariant sequences obtained from the permutation results as random expectation, and evaluated the probability of invariant sequences in the original alignment based on the expectation. This approach helps identify sequences whose conservation is rare to occur in the substitution background of each alignment. Multiple testing correction of p-values is done by FDR (false discovery rate).<sup><a reveal-id="DSR010C33" data-open="DSR010C33" class="link link-ref link-reveal xref-bibr">33</a></sup> Then we identified invariant sequences longer than 10 codons and <em>P</em> &lt; 0.01 as SCCSs.</p>                            <h3 scrollto-destination=6650492 id="6650492" class="section-title" data-legacy-id=s2c>2.3. Analysis on GO, InterPro, and KEGG pathways</h3>
<p>Protein coding genes that contain at least one SCCS is named SCCS genes and those that do not contain an SCCS is named non-SCCS genes. We used Fatigo web service (<a class="link link-uri" href="http://babelomics.bioinfo.cipf.es/functional.html">http://babelomics.bioinfo.cipf.es/functional.html</a>) to identify GO terms, InterPro codes, and KEGG pathways that are significantly enriched with the SCCS gene group or with the non-SCCS gene group. Fatigo accepts a list of Ensembl gene IDs as input and provides <em>P</em>-values for enrichment of the above terms. <em>P</em>-values are calculated by Fisher's exact test and corrected by FDR. We used Ensembl gene IDs of <em>H. sapiens</em> as input and performed two-tailed comparison between SCCS genes and non-SCCS genes.</p>                            <h3 scrollto-destination=6650494 id="6650494" class="section-title" data-legacy-id=s2d>2.4. Analysis on preferred codons and average codon degeneracy of SCCSs</h3>
<p>We defined preferred codons as the most frequently used codons for a given amino acid referring to Codon Usage Database (<a class="link link-uri" href="http://www.kazusa.or.jp/codon/index.html">http://www.kazusa.or.jp/codon/index.html</a>) provided by Kazusa DNA Research Institute. Because the codon usage pattern was similar among the six species we used, the codon table of human was used as the representative. We counted the number of preferred codons in a SCCS and divided it by the codon length of the SCCS, and then used the quotient as the ratio of preferred codons. The average codon degeneracy is calculated by summing up the degeneracy of each codon and dividing it by the codon length of the SCCS.</p>                            <h3 scrollto-destination=6650496 id="6650496" class="section-title" data-legacy-id=s2e>2.5. Prediction of RNA secondary structures</h3>
<p>We computationally predicted secondary structures and free folding energy of SCCSs using Vienna RNA software package<sup><a reveal-id="DSR010C34" data-open="DSR010C34" class="link link-ref link-reveal xref-bibr">34</a></sup> (<a class="link link-uri" href="http://www.tbi.univie.ac.at/~ivo/RNA/">http://www.tbi.univie.ac.at/~ivo/RNA/</a>). Because folding free energy varies depending on the sequence length, we constructed free energy distribution by 1000 randomly chosen sequences for each length (33–246 nucleotides). The <em>P</em>-value for a given free energy was evaluated based on these distributions. Multi testing correction is done by FDR.</p>                            <h3 scrollto-destination=6650498 id="6650498" class="section-title" data-legacy-id=s2f>2.6. Evaluation of exonic splicing enhancer density</h3>
<p>We obtained 238 hexamers from RESCUE-ESE Web Server<sup><a reveal-id="DSR010C34" data-open="DSR010C34" class="link link-ref link-reveal xref-bibr">34</a></sup> as candidates of exonic splicing enhancers. We counted the number of the hexamers in the SCCSs and the rest of the coding regions of the 10 790 genes (human sequences). We also measured the total nucleotide length of the SCCS and the other regions and applied the chi-square test for the ratio of hexamers.</p>                            <h3 scrollto-destination=6650500 id="6650500" class="section-title" data-legacy-id=s2g>2.7. Analysis on gene expression</h3>
<p>We used EGenetics (<a class="link link-uri" href="http://www.nhmrc.gov.au/your_health/egenetics/index.htm">http://www.nhmrc.gov.au/your_health/egenetics/index.htm</a>) to investigate gene expression of SCCS genes and non-SCCS genes. Human anatomical system data, which give information about in which organs a gene is expressed, were obtained from EGenetics database by Ensemble Biomart. We counted how many of SCCS genes and non-SCCS genes are expressed in each organ and divided the numbers by the total number of SCCS genes and non-SCCS genes, respectively. Then we performed the Fisher's exact test to evaluate whether the difference between SCCS and non-SCCS genes is significant. All <em>P</em>-values were corrected by FDR.</p>                            <h2 scrollto-destination=6650502 id="6650502" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6650503 id="6650503" class="section-title" data-legacy-id=s3a>3.1. Identification of SCCSs</h3>
<p>If an alignment has a high ratio of conservation, long invariant sequences may occur easily, and vice versa. Therefore, the rareness of invariant sequences differs depending on the conservation background in each alignment. To compensate this, we performed permutation simulation. The idea of permutation is to count the length and frequency of invariant sequences after random change of loci and use the result as the random expectation.</p><p>We used the frequency distribution of invariant sequences in the permuted alignments as the random expectation and evaluated probability of invariant sequences in the original alignments, and defined invariant sequences longer than 10 codons and whose probability is below 0.01 (corrected by FDR) as SCCSs.</p><p>In total, 4150 SCCSs (192 306 bp) were obtained from 2273 alignments of 10 790 orthologous gene sets (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary file S2</a>). This occupies 0.94% of the coding region of the 10 790 genes. Table <a reveal-id="DSR010TB1" data-open="DSR010TB1" class="link link-reveal link-table xref-fig">1</a> shows the number of SCCSs per gene and the number of genes that contain that number of SCCSs. Figure <a reveal-id="DSR010F1" data-open="DSR010F1" class="link link-reveal link-table xref-fig">1</a> is a graph of lengths and numbers of SCCSs (grey bars). Black dots indicate the random expectation obtained from the permuted alignments. </p>                        <a id="6650507" scrollto-destination="6650507"></a>
<div content-id="DSR010TB1" class="table-modal"><div class="table-wrap"><div id="DSR010TB1" class="table-wrap-title" data-id="DSR010TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Number of SCCSs in coding genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Number of SCCSs (per gene) </th><th>Number of genes </th></tr></thead><tbody><tr><td rowspan="6">SCCS genes </td><td>1 </td><td>1366 </td></tr><tr><td>2 </td><td>475 </td></tr><tr><td>3 </td><td>219 </td></tr><tr><td>4 </td><td>105 </td></tr><tr><td>5 </td><td>42 </td></tr><tr><td>&gt;5 </td><td>66 </td></tr><tr><td>Non-SCCS genes </td><td> </td><td>8517 </td></tr></tbody></table></div><div content-id="DSR010TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Number of SCCSs (per gene) </th><th>Number of genes </th></tr></thead><tbody><tr><td rowspan="6">SCCS genes </td><td>1 </td><td>1366 </td></tr><tr><td>2 </td><td>475 </td></tr><tr><td>3 </td><td>219 </td></tr><tr><td>4 </td><td>105 </td></tr><tr><td>5 </td><td>42 </td></tr><tr><td>&gt;5 </td><td>66 </td></tr><tr><td>Non-SCCS genes </td><td> </td><td>8517 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6650507" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR010TB1" class="table-wrap-title" data-id="DSR010TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Number of SCCSs in coding genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Number of SCCSs (per gene) </th><th>Number of genes </th></tr></thead><tbody><tr><td rowspan="6">SCCS genes </td><td>1 </td><td>1366 </td></tr><tr><td>2 </td><td>475 </td></tr><tr><td>3 </td><td>219 </td></tr><tr><td>4 </td><td>105 </td></tr><tr><td>5 </td><td>42 </td></tr><tr><td>&gt;5 </td><td>66 </td></tr><tr><td>Non-SCCS genes </td><td> </td><td>8517 </td></tr></tbody></table></div><div content-id="DSR010TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Number of SCCSs (per gene) </th><th>Number of genes </th></tr></thead><tbody><tr><td rowspan="6">SCCS genes </td><td>1 </td><td>1366 </td></tr><tr><td>2 </td><td>475 </td></tr><tr><td>3 </td><td>219 </td></tr><tr><td>4 </td><td>105 </td></tr><tr><td>5 </td><td>42 </td></tr><tr><td>&gt;5 </td><td>66 </td></tr><tr><td>Non-SCCS genes </td><td> </td><td>8517 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6650507" target="_blank">View Large</a></div></div>                        <a id="6650508" scrollto-destination="6650508"></a>
<div id="DSR010F1" data-id="DSR010F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/m_dsr01001.jpeg?Expires=2147483647&Signature=GzmrxRfMusoxlEzXNMHlaxL5sH9rT65e4dS8jhnGbdBNce3FmkoIbrDufKTuEuP9wSUY4WwUInQqQp5URxP840tOCbMveyPFVe1eW29j-eIT8kiH~MQZov9v2i-iBn7Qt8QX9GCYFzG4nsIOoINgej2aJ8p0cEwkxX9LLjhrgSQ627B6cNlEGTLPcIcggboAz-YkL1LckS9da0Vy081pO7DNywkgNxCJqClQzqJAt-Jb9YMMaFMTx9VYcU2m7GapkoEyZ-4DjGM1dMJASRFbINi7PhC71tHrAz6jJeh7krk2Xg0DR4EPuvilzAqK7xGm-lSeU37VMGzjgiNIovt7AQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The length and number of SCCSs. X and Y-axes represent the length and frequency of SCCSs, respectively. Grey bars show the length and the number of SCCSs. Black dots indicate the number of sequences with probability below 0.01 in the permuted alignments." path-from-xml="dsr01001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6650508/dsr01001.jpeg" path-from-xml="dsr01001.jpeg" target="_blank">View large</a><a section="6650508" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/dsr01001.jpeg?Expires=2147483647&Signature=cOgHhEN01DKKnF5X-xXEz5xFL0s5H7XeGZaLfaesoaJICZFv9QGYjir7ZRxbU7pquzXBoFNdzPdfsv0URijs9-7DAS694pfZ~nK9RdhGC2gREU43Bg9b0WFZx1k96hH~IUuMkfqaU2ldmcQ4CGxpn6rqi4ygqvpY-NPLvs3-ECFTuljiPGWc8cRxWFWSWth2VfwO0ueDhGo28g~Na6rIMIS5cElNCLtHfxihJGLvPczWCtzShdOt2zXWxPMAnW35nbd4sfFeZiIKBXs2IKK3Yt6iAYmK7XCcndviUmD3bnes1lwhJCVPcGfS7ZK14OgguCmygfRUKFUAOKvLLI1x9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6650508&ar=444849&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The length and number of SCCSs. X and Y-axes represent the length and frequency of SCCSs, respectively. Grey bars show the length and the number of SCCSs. Black dots indicate the number of sequences with probability below 0.01 in the permuted alignments.</p></div></div><div content-id="DSR010F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/m_dsr01001.jpeg?Expires=2147483647&Signature=GzmrxRfMusoxlEzXNMHlaxL5sH9rT65e4dS8jhnGbdBNce3FmkoIbrDufKTuEuP9wSUY4WwUInQqQp5URxP840tOCbMveyPFVe1eW29j-eIT8kiH~MQZov9v2i-iBn7Qt8QX9GCYFzG4nsIOoINgej2aJ8p0cEwkxX9LLjhrgSQ627B6cNlEGTLPcIcggboAz-YkL1LckS9da0Vy081pO7DNywkgNxCJqClQzqJAt-Jb9YMMaFMTx9VYcU2m7GapkoEyZ-4DjGM1dMJASRFbINi7PhC71tHrAz6jJeh7krk2Xg0DR4EPuvilzAqK7xGm-lSeU37VMGzjgiNIovt7AQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The length and number of SCCSs. X and Y-axes represent the length and frequency of SCCSs, respectively. Grey bars show the length and the number of SCCSs. Black dots indicate the number of sequences with probability below 0.01 in the permuted alignments." path-from-xml="dsr01001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6650508/dsr01001.jpeg" path-from-xml="dsr01001.jpeg" target="_blank">View large</a><a section="6650508" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/dsr01001.jpeg?Expires=2147483647&Signature=cOgHhEN01DKKnF5X-xXEz5xFL0s5H7XeGZaLfaesoaJICZFv9QGYjir7ZRxbU7pquzXBoFNdzPdfsv0URijs9-7DAS694pfZ~nK9RdhGC2gREU43Bg9b0WFZx1k96hH~IUuMkfqaU2ldmcQ4CGxpn6rqi4ygqvpY-NPLvs3-ECFTuljiPGWc8cRxWFWSWth2VfwO0ueDhGo28g~Na6rIMIS5cElNCLtHfxihJGLvPczWCtzShdOt2zXWxPMAnW35nbd4sfFeZiIKBXs2IKK3Yt6iAYmK7XCcndviUmD3bnes1lwhJCVPcGfS7ZK14OgguCmygfRUKFUAOKvLLI1x9Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6650508&ar=444849&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The length and number of SCCSs. X and Y-axes represent the length and frequency of SCCSs, respectively. Grey bars show the length and the number of SCCSs. Black dots indicate the number of sequences with probability below 0.01 in the permuted alignments.</p></div></div><p>In the permuted alignments, there are 141 sequences (their total length is 5550 bp) whose probability is below 0.01, which means the region size of SCCS is 35-fold larger than this expectation (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary file S3</a>). The χ<sup>2</sup> test between SCCSs and the permutation result showed a significant difference (<em>P</em> &lt; 2.2E−16).</p><p>If invariant and variant sites distributed randomly in the original alignment, the frequency of invariant sequences in the permuted alignments would show similar frequency to the original alignment because of the randomness at the start point. The difference before and after the permutation suggests that the distribution of invariant sites in the original alignments is rather clustered than being random.</p>                            <h3 scrollto-destination=6650511 id="6650511" class="section-title" data-legacy-id=s3b>3.2. GO, InterPro, and KEGG pathways enriched in SCCS containing genes</h3>
<p>We used Fatigo web service to investigate difference in GO, InterPro codes, and KEGG pathways between the SCCS genes and non-SCCS genes. The difference was evaluated by the two-tailed Fisher's test, and <em>P</em>-values were corrected by FDR. Tables <a reveal-id="DSR010TB2" data-open="DSR010TB2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSR010TB3" data-open="DSR010TB3" class="link link-reveal link-table xref-fig">3</a> show GO terms, InterPro codes, and KEGG pathways significantly enriched (<em>P</em> &lt; 0.01) in SCCS genes. </p>                        <a id="6650513" scrollto-destination="6650513"></a>
<div content-id="DSR010TB2" class="table-modal"><div class="table-wrap"><div id="DSR010TB2" class="table-wrap-title" data-id="DSR010TB2"><div class="label -label">Table 2.</div><div class="caption"><p>GO terms significantly (<em>P</em> &lt; 0.01) enriched with SCCS genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Terms </th><th><em>P</em>* </th><th>In SCCS containing genes (%)<sup>a</sup> </th><th>In non-SCCS containing genes (%)<sup>b</sup> </th><th>Fold<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="5">Biological process </td></tr><tr><td> GO:0035136: Forelimb morphogenesis </td><td>7.75E−05 </td><td>0.53 </td><td>0.04 </td><td>13.25 </td></tr><tr><td> GO:0035115: Embryonic forelimb morphogenesis </td><td>2.50E−04 </td><td>0.48 </td><td>0.04 </td><td>12.00 </td></tr><tr><td> GO:0060070: Canonical Wnt receptor signalling pathway </td><td>2.11E−03 </td><td>0.4 </td><td>0.04 </td><td>10.00 </td></tr><tr><td> GO:0035137: Hindlimb morphogenesis </td><td>4.88E−04 </td><td>0.53 </td><td>0.06 </td><td>8.83 </td></tr><tr><td> GO:0001702: Gastrulation with mouth forming second </td><td>1.78E−03 </td><td>0.44 </td><td>0.05 </td><td>8.80 </td></tr><tr><td> GO:0009954: Proximal/distal pattern formation </td><td>3.80E−04 </td><td>0.57 </td><td>0.07 </td><td>8.14 </td></tr><tr><td> GO:0031128: Developmental induction </td><td>1.03E−03 </td><td>0.53 </td><td>0.07 </td><td>7.57 </td></tr><tr><td> GO:0048593: Camera-type eye morphogenesis </td><td>1.22E−05 </td><td>0.88 </td><td>0.13 </td><td>6.77 </td></tr><tr><td> GO:0021510: Spinal cord development </td><td>2.37E−04 </td><td>0.75 </td><td>0.13 </td><td>5.77 </td></tr><tr><td> GO:0031016: Pancreas development </td><td>1.85E−03 </td><td>0.62 </td><td>0.12 </td><td>5.17 </td></tr><tr><td colspan="5">Cellular components </td></tr><tr><td> GO:0014704: Intercalated disc </td><td>8.43E−03 </td><td>0.35 </td><td>0.04 </td><td>8.75 </td></tr><tr><td> GO:0043198: Dendritic shaft </td><td>2.38E−03 </td><td>0.48 </td><td>0.06 </td><td>8.00 </td></tr><tr><td> GO:0030425: Dendrite </td><td>2.38E−03 </td><td>2.29 </td><td>1.1 </td><td>2.08 </td></tr><tr><td> GO:0043025: Neuronal cell body </td><td>2.38E−03 </td><td>2.24 </td><td>1.11 </td><td>2.02 </td></tr><tr><td> GO:0015629: Actin cytoskeleton </td><td>1.15E−03 </td><td>3.39 </td><td>1.8 </td><td>1.88 </td></tr><tr><td> GO:0043005: Neuron projection </td><td>1.15E−03 </td><td>4.09 </td><td>2.32 </td><td>1.76 </td></tr><tr><td colspan="5">Molecular function </td></tr><tr><td> GO:0035254: Glutamate receptor binding </td><td>3.41E−03 </td><td>0.35 </td><td>0.02 </td><td>17.50 </td></tr><tr><td> GO:0005072: Transforming growth factor beta receptor, cytoplasmic mediator activity </td><td>8.92E−03 </td><td>0.31 </td><td>0.02 </td><td>15.50 </td></tr><tr><td> GO:0004843: Ubiquitin-specific protease activity </td><td>3.71E−03 </td><td>0.48 </td><td>0.07 </td><td>6.86 </td></tr><tr><td> GO:0031625: Ubiquitin protein ligase binding </td><td>6.44E−03 </td><td>0.62 </td><td>0.14 </td><td>4.43 </td></tr><tr><td> GO:0003725: Double-stranded RNA binding </td><td>6.44E−03 </td><td>0.62 </td><td>0.14 </td><td>4.43 </td></tr><tr><td> GO:0042054: Histone methyltransferase activity </td><td>8.64E−03 </td><td>0.57 </td><td>0.13 </td><td>4.38 </td></tr><tr><td> GO:0050825: Ice binding </td><td>6.50E−11 </td><td>2.86 </td><td>0.76 </td><td>3.76 </td></tr><tr><td> GO:0004221: Ubiquitin thiolesterase activity </td><td>4.18E−04 </td><td>1.01 </td><td>0.27 </td><td>3.74 </td></tr><tr><td> GO:0005199: Structural constituent of cell wall </td><td>1.34E−03 </td><td>0.92 </td><td>0.25 </td><td>3.68 </td></tr><tr><td> GO:0003682: Chromatin binding </td><td>4.08E−08 </td><td>2.15 </td><td>0.59 </td><td>3.64 </td></tr></tbody></table></div><div content-id="DSR010TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Terms </th><th><em>P</em>* </th><th>In SCCS containing genes (%)<sup>a</sup> </th><th>In non-SCCS containing genes (%)<sup>b</sup> </th><th>Fold<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="5">Biological process </td></tr><tr><td> GO:0035136: Forelimb morphogenesis </td><td>7.75E−05 </td><td>0.53 </td><td>0.04 </td><td>13.25 </td></tr><tr><td> GO:0035115: Embryonic forelimb morphogenesis </td><td>2.50E−04 </td><td>0.48 </td><td>0.04 </td><td>12.00 </td></tr><tr><td> GO:0060070: Canonical Wnt receptor signalling pathway </td><td>2.11E−03 </td><td>0.4 </td><td>0.04 </td><td>10.00 </td></tr><tr><td> GO:0035137: Hindlimb morphogenesis </td><td>4.88E−04 </td><td>0.53 </td><td>0.06 </td><td>8.83 </td></tr><tr><td> GO:0001702: Gastrulation with mouth forming second </td><td>1.78E−03 </td><td>0.44 </td><td>0.05 </td><td>8.80 </td></tr><tr><td> GO:0009954: Proximal/distal pattern formation </td><td>3.80E−04 </td><td>0.57 </td><td>0.07 </td><td>8.14 </td></tr><tr><td> GO:0031128: Developmental induction </td><td>1.03E−03 </td><td>0.53 </td><td>0.07 </td><td>7.57 </td></tr><tr><td> GO:0048593: Camera-type eye morphogenesis </td><td>1.22E−05 </td><td>0.88 </td><td>0.13 </td><td>6.77 </td></tr><tr><td> GO:0021510: Spinal cord development </td><td>2.37E−04 </td><td>0.75 </td><td>0.13 </td><td>5.77 </td></tr><tr><td> GO:0031016: Pancreas development </td><td>1.85E−03 </td><td>0.62 </td><td>0.12 </td><td>5.17 </td></tr><tr><td colspan="5">Cellular components </td></tr><tr><td> GO:0014704: Intercalated disc </td><td>8.43E−03 </td><td>0.35 </td><td>0.04 </td><td>8.75 </td></tr><tr><td> GO:0043198: Dendritic shaft </td><td>2.38E−03 </td><td>0.48 </td><td>0.06 </td><td>8.00 </td></tr><tr><td> GO:0030425: Dendrite </td><td>2.38E−03 </td><td>2.29 </td><td>1.1 </td><td>2.08 </td></tr><tr><td> GO:0043025: Neuronal cell body </td><td>2.38E−03 </td><td>2.24 </td><td>1.11 </td><td>2.02 </td></tr><tr><td> GO:0015629: Actin cytoskeleton </td><td>1.15E−03 </td><td>3.39 </td><td>1.8 </td><td>1.88 </td></tr><tr><td> GO:0043005: Neuron projection </td><td>1.15E−03 </td><td>4.09 </td><td>2.32 </td><td>1.76 </td></tr><tr><td colspan="5">Molecular function </td></tr><tr><td> GO:0035254: Glutamate receptor binding </td><td>3.41E−03 </td><td>0.35 </td><td>0.02 </td><td>17.50 </td></tr><tr><td> GO:0005072: Transforming growth factor beta receptor, cytoplasmic mediator activity </td><td>8.92E−03 </td><td>0.31 </td><td>0.02 </td><td>15.50 </td></tr><tr><td> GO:0004843: Ubiquitin-specific protease activity </td><td>3.71E−03 </td><td>0.48 </td><td>0.07 </td><td>6.86 </td></tr><tr><td> GO:0031625: Ubiquitin protein ligase binding </td><td>6.44E−03 </td><td>0.62 </td><td>0.14 </td><td>4.43 </td></tr><tr><td> GO:0003725: Double-stranded RNA binding </td><td>6.44E−03 </td><td>0.62 </td><td>0.14 </td><td>4.43 </td></tr><tr><td> GO:0042054: Histone methyltransferase activity </td><td>8.64E−03 </td><td>0.57 </td><td>0.13 </td><td>4.38 </td></tr><tr><td> GO:0050825: Ice binding </td><td>6.50E−11 </td><td>2.86 </td><td>0.76 </td><td>3.76 </td></tr><tr><td> GO:0004221: Ubiquitin thiolesterase activity </td><td>4.18E−04 </td><td>1.01 </td><td>0.27 </td><td>3.74 </td></tr><tr><td> GO:0005199: Structural constituent of cell wall </td><td>1.34E−03 </td><td>0.92 </td><td>0.25 </td><td>3.68 </td></tr><tr><td> GO:0003682: Chromatin binding </td><td>4.08E−08 </td><td>2.15 </td><td>0.59 </td><td>3.64 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The percentages of SCCS genes that have the GO term.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The percentages of non-SCCS genes that have the GO term.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The fold of ‘a’ to ‘b’. Terms are listed in the descending order of the fold difference. Terms with the highest 10-folds are shown for Biological Process and Molecular function.</p></div></div><div content-id=""><div class="fn"><p>*Probability for enrichment of the GO term in the SCCS group.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6650513" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR010TB2" class="table-wrap-title" data-id="DSR010TB2"><div class="label -label">Table 2.</div><div class="caption"><p>GO terms significantly (<em>P</em> &lt; 0.01) enriched with SCCS genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Terms </th><th><em>P</em>* </th><th>In SCCS containing genes (%)<sup>a</sup> </th><th>In non-SCCS containing genes (%)<sup>b</sup> </th><th>Fold<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="5">Biological process </td></tr><tr><td> GO:0035136: Forelimb morphogenesis </td><td>7.75E−05 </td><td>0.53 </td><td>0.04 </td><td>13.25 </td></tr><tr><td> GO:0035115: Embryonic forelimb morphogenesis </td><td>2.50E−04 </td><td>0.48 </td><td>0.04 </td><td>12.00 </td></tr><tr><td> GO:0060070: Canonical Wnt receptor signalling pathway </td><td>2.11E−03 </td><td>0.4 </td><td>0.04 </td><td>10.00 </td></tr><tr><td> GO:0035137: Hindlimb morphogenesis </td><td>4.88E−04 </td><td>0.53 </td><td>0.06 </td><td>8.83 </td></tr><tr><td> GO:0001702: Gastrulation with mouth forming second </td><td>1.78E−03 </td><td>0.44 </td><td>0.05 </td><td>8.80 </td></tr><tr><td> GO:0009954: Proximal/distal pattern formation </td><td>3.80E−04 </td><td>0.57 </td><td>0.07 </td><td>8.14 </td></tr><tr><td> GO:0031128: Developmental induction </td><td>1.03E−03 </td><td>0.53 </td><td>0.07 </td><td>7.57 </td></tr><tr><td> GO:0048593: Camera-type eye morphogenesis </td><td>1.22E−05 </td><td>0.88 </td><td>0.13 </td><td>6.77 </td></tr><tr><td> GO:0021510: Spinal cord development </td><td>2.37E−04 </td><td>0.75 </td><td>0.13 </td><td>5.77 </td></tr><tr><td> GO:0031016: Pancreas development </td><td>1.85E−03 </td><td>0.62 </td><td>0.12 </td><td>5.17 </td></tr><tr><td colspan="5">Cellular components </td></tr><tr><td> GO:0014704: Intercalated disc </td><td>8.43E−03 </td><td>0.35 </td><td>0.04 </td><td>8.75 </td></tr><tr><td> GO:0043198: Dendritic shaft </td><td>2.38E−03 </td><td>0.48 </td><td>0.06 </td><td>8.00 </td></tr><tr><td> GO:0030425: Dendrite </td><td>2.38E−03 </td><td>2.29 </td><td>1.1 </td><td>2.08 </td></tr><tr><td> GO:0043025: Neuronal cell body </td><td>2.38E−03 </td><td>2.24 </td><td>1.11 </td><td>2.02 </td></tr><tr><td> GO:0015629: Actin cytoskeleton </td><td>1.15E−03 </td><td>3.39 </td><td>1.8 </td><td>1.88 </td></tr><tr><td> GO:0043005: Neuron projection </td><td>1.15E−03 </td><td>4.09 </td><td>2.32 </td><td>1.76 </td></tr><tr><td colspan="5">Molecular function </td></tr><tr><td> GO:0035254: Glutamate receptor binding </td><td>3.41E−03 </td><td>0.35 </td><td>0.02 </td><td>17.50 </td></tr><tr><td> GO:0005072: Transforming growth factor beta receptor, cytoplasmic mediator activity </td><td>8.92E−03 </td><td>0.31 </td><td>0.02 </td><td>15.50 </td></tr><tr><td> GO:0004843: Ubiquitin-specific protease activity </td><td>3.71E−03 </td><td>0.48 </td><td>0.07 </td><td>6.86 </td></tr><tr><td> GO:0031625: Ubiquitin protein ligase binding </td><td>6.44E−03 </td><td>0.62 </td><td>0.14 </td><td>4.43 </td></tr><tr><td> GO:0003725: Double-stranded RNA binding </td><td>6.44E−03 </td><td>0.62 </td><td>0.14 </td><td>4.43 </td></tr><tr><td> GO:0042054: Histone methyltransferase activity </td><td>8.64E−03 </td><td>0.57 </td><td>0.13 </td><td>4.38 </td></tr><tr><td> GO:0050825: Ice binding </td><td>6.50E−11 </td><td>2.86 </td><td>0.76 </td><td>3.76 </td></tr><tr><td> GO:0004221: Ubiquitin thiolesterase activity </td><td>4.18E−04 </td><td>1.01 </td><td>0.27 </td><td>3.74 </td></tr><tr><td> GO:0005199: Structural constituent of cell wall </td><td>1.34E−03 </td><td>0.92 </td><td>0.25 </td><td>3.68 </td></tr><tr><td> GO:0003682: Chromatin binding </td><td>4.08E−08 </td><td>2.15 </td><td>0.59 </td><td>3.64 </td></tr></tbody></table></div><div content-id="DSR010TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Terms </th><th><em>P</em>* </th><th>In SCCS containing genes (%)<sup>a</sup> </th><th>In non-SCCS containing genes (%)<sup>b</sup> </th><th>Fold<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="5">Biological process </td></tr><tr><td> GO:0035136: Forelimb morphogenesis </td><td>7.75E−05 </td><td>0.53 </td><td>0.04 </td><td>13.25 </td></tr><tr><td> GO:0035115: Embryonic forelimb morphogenesis </td><td>2.50E−04 </td><td>0.48 </td><td>0.04 </td><td>12.00 </td></tr><tr><td> GO:0060070: Canonical Wnt receptor signalling pathway </td><td>2.11E−03 </td><td>0.4 </td><td>0.04 </td><td>10.00 </td></tr><tr><td> GO:0035137: Hindlimb morphogenesis </td><td>4.88E−04 </td><td>0.53 </td><td>0.06 </td><td>8.83 </td></tr><tr><td> GO:0001702: Gastrulation with mouth forming second </td><td>1.78E−03 </td><td>0.44 </td><td>0.05 </td><td>8.80 </td></tr><tr><td> GO:0009954: Proximal/distal pattern formation </td><td>3.80E−04 </td><td>0.57 </td><td>0.07 </td><td>8.14 </td></tr><tr><td> GO:0031128: Developmental induction </td><td>1.03E−03 </td><td>0.53 </td><td>0.07 </td><td>7.57 </td></tr><tr><td> GO:0048593: Camera-type eye morphogenesis </td><td>1.22E−05 </td><td>0.88 </td><td>0.13 </td><td>6.77 </td></tr><tr><td> GO:0021510: Spinal cord development </td><td>2.37E−04 </td><td>0.75 </td><td>0.13 </td><td>5.77 </td></tr><tr><td> GO:0031016: Pancreas development </td><td>1.85E−03 </td><td>0.62 </td><td>0.12 </td><td>5.17 </td></tr><tr><td colspan="5">Cellular components </td></tr><tr><td> GO:0014704: Intercalated disc </td><td>8.43E−03 </td><td>0.35 </td><td>0.04 </td><td>8.75 </td></tr><tr><td> GO:0043198: Dendritic shaft </td><td>2.38E−03 </td><td>0.48 </td><td>0.06 </td><td>8.00 </td></tr><tr><td> GO:0030425: Dendrite </td><td>2.38E−03 </td><td>2.29 </td><td>1.1 </td><td>2.08 </td></tr><tr><td> GO:0043025: Neuronal cell body </td><td>2.38E−03 </td><td>2.24 </td><td>1.11 </td><td>2.02 </td></tr><tr><td> GO:0015629: Actin cytoskeleton </td><td>1.15E−03 </td><td>3.39 </td><td>1.8 </td><td>1.88 </td></tr><tr><td> GO:0043005: Neuron projection </td><td>1.15E−03 </td><td>4.09 </td><td>2.32 </td><td>1.76 </td></tr><tr><td colspan="5">Molecular function </td></tr><tr><td> GO:0035254: Glutamate receptor binding </td><td>3.41E−03 </td><td>0.35 </td><td>0.02 </td><td>17.50 </td></tr><tr><td> GO:0005072: Transforming growth factor beta receptor, cytoplasmic mediator activity </td><td>8.92E−03 </td><td>0.31 </td><td>0.02 </td><td>15.50 </td></tr><tr><td> GO:0004843: Ubiquitin-specific protease activity </td><td>3.71E−03 </td><td>0.48 </td><td>0.07 </td><td>6.86 </td></tr><tr><td> GO:0031625: Ubiquitin protein ligase binding </td><td>6.44E−03 </td><td>0.62 </td><td>0.14 </td><td>4.43 </td></tr><tr><td> GO:0003725: Double-stranded RNA binding </td><td>6.44E−03 </td><td>0.62 </td><td>0.14 </td><td>4.43 </td></tr><tr><td> GO:0042054: Histone methyltransferase activity </td><td>8.64E−03 </td><td>0.57 </td><td>0.13 </td><td>4.38 </td></tr><tr><td> GO:0050825: Ice binding </td><td>6.50E−11 </td><td>2.86 </td><td>0.76 </td><td>3.76 </td></tr><tr><td> GO:0004221: Ubiquitin thiolesterase activity </td><td>4.18E−04 </td><td>1.01 </td><td>0.27 </td><td>3.74 </td></tr><tr><td> GO:0005199: Structural constituent of cell wall </td><td>1.34E−03 </td><td>0.92 </td><td>0.25 </td><td>3.68 </td></tr><tr><td> GO:0003682: Chromatin binding </td><td>4.08E−08 </td><td>2.15 </td><td>0.59 </td><td>3.64 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The percentages of SCCS genes that have the GO term.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The percentages of non-SCCS genes that have the GO term.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The fold of ‘a’ to ‘b’. Terms are listed in the descending order of the fold difference. Terms with the highest 10-folds are shown for Biological Process and Molecular function.</p></div></div><div content-id=""><div class="fn"><p>*Probability for enrichment of the GO term in the SCCS group.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6650513" target="_blank">View Large</a></div></div>                        <a id="6650514" scrollto-destination="6650514"></a>
<div content-id="DSR010TB3" class="table-modal"><div class="table-wrap"><div id="DSR010TB3" class="table-wrap-title" data-id="DSR010TB3"><div class="label -label">Table 3.</div><div class="caption"><p>InterPro and KEGG terms significantly (<em>P</em> &lt; 0.01) enriched with SCCS genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Terms </th><th><em>P</em>* </th><th>In SCCS containing genes (%)<sup>a</sup> </th><th>In non-SCCS containing genes (%)<sup>b</sup> </th><th>Fold<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="5">InterPro </td></tr><tr><td>IPR003619: MAD homology 1, Dwarfin-type </td><td>1.55E−04 </td><td>0.44 </td><td>0.01 </td><td>44.00 </td></tr><tr><td> IPR001827: Homeobox protein, antennapedia type, conserved site </td><td>1.77E−03 </td><td>0.44 </td><td>0.04 </td><td>11.00 </td></tr><tr><td> IPR000569: HECT </td><td>1.26E−05 </td><td>0.75 </td><td>0.07 </td><td>10.71 </td></tr><tr><td> IPR002077: Voltage-dependent calcium channel, alpha-1 subunit </td><td>5.65E−04 </td><td>0.53 </td><td>0.05 </td><td>10.60 </td></tr><tr><td> IPR010982: Lambda repressor-like, DNA-binding </td><td>1.49E−03 </td><td>0.48 </td><td>0.05 </td><td>9.60 </td></tr><tr><td> IPR002343: Paraneoplastic encephalomyelitis antigen </td><td>9.91E−05 </td><td>0.7 </td><td>0.08 </td><td>8.75 </td></tr><tr><td> IPR018359: Bromodomain, conserved site </td><td>9.75E−04 </td><td>0.57 </td><td>0.07 </td><td>8.14 </td></tr><tr><td> IPR001487: Bromodomain </td><td>1.94E−06 </td><td>0.97 </td><td>0.12 </td><td>8.08 </td></tr><tr><td> IPR017995: Homeobox protein, antennapedia type </td><td>7.36E−04 </td><td>0.62 </td><td>0.08 </td><td>7.75 </td></tr><tr><td> IPR004088: K Homology, type 1 </td><td>7.36E−04 </td><td>0.62 </td><td>0.08 </td><td>7.75 </td></tr><tr><td colspan="5">KEGG </td></tr><tr><td> hsa03018: RNA degradation </td><td>1.05E−03 </td><td>0.92 </td><td>0.24 </td><td>3.83 </td></tr><tr><td> hsa04340: Hedgehog signalling pathway </td><td>6.75E−03 </td><td>0.88 </td><td>0.29 </td><td>3.03 </td></tr><tr><td> hsa04520: Adherens junction </td><td>4.55E−03 </td><td>1.06 </td><td>0.37 </td><td>2.86 </td></tr><tr><td> hsa05211: Renal cell carcinoma </td><td>9.88E−03 </td><td>0.92 </td><td>0.33 </td><td>2.79 </td></tr><tr><td> hsa04120: Ubiquitin-mediated proteolysis </td><td>1.05E−03 </td><td>1.5 </td><td>0.57 </td><td>2.63 </td></tr><tr><td> hsa04310: Wnt signalling pathway </td><td>2.22E−04 </td><td>2.07 </td><td>0.79 </td><td>2.62 </td></tr><tr><td> hsa04360: Axon guidance </td><td>3.47E−04 </td><td>1.8 </td><td>0.69 </td><td>2.61 </td></tr><tr><td> hsa04810: Regulation of actin cytoskeleton </td><td>1.07E−03 </td><td>2.07 </td><td>0.94 </td><td>2.20 </td></tr><tr><td> hsa04010: MAPK signaling pathway </td><td>3.23E−04 </td><td>3.08 </td><td>1.5 </td><td>2.05 </td></tr></tbody></table></div><div content-id="DSR010TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Terms </th><th><em>P</em>* </th><th>In SCCS containing genes (%)<sup>a</sup> </th><th>In non-SCCS containing genes (%)<sup>b</sup> </th><th>Fold<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="5">InterPro </td></tr><tr><td>IPR003619: MAD homology 1, Dwarfin-type </td><td>1.55E−04 </td><td>0.44 </td><td>0.01 </td><td>44.00 </td></tr><tr><td> IPR001827: Homeobox protein, antennapedia type, conserved site </td><td>1.77E−03 </td><td>0.44 </td><td>0.04 </td><td>11.00 </td></tr><tr><td> IPR000569: HECT </td><td>1.26E−05 </td><td>0.75 </td><td>0.07 </td><td>10.71 </td></tr><tr><td> IPR002077: Voltage-dependent calcium channel, alpha-1 subunit </td><td>5.65E−04 </td><td>0.53 </td><td>0.05 </td><td>10.60 </td></tr><tr><td> IPR010982: Lambda repressor-like, DNA-binding </td><td>1.49E−03 </td><td>0.48 </td><td>0.05 </td><td>9.60 </td></tr><tr><td> IPR002343: Paraneoplastic encephalomyelitis antigen </td><td>9.91E−05 </td><td>0.7 </td><td>0.08 </td><td>8.75 </td></tr><tr><td> IPR018359: Bromodomain, conserved site </td><td>9.75E−04 </td><td>0.57 </td><td>0.07 </td><td>8.14 </td></tr><tr><td> IPR001487: Bromodomain </td><td>1.94E−06 </td><td>0.97 </td><td>0.12 </td><td>8.08 </td></tr><tr><td> IPR017995: Homeobox protein, antennapedia type </td><td>7.36E−04 </td><td>0.62 </td><td>0.08 </td><td>7.75 </td></tr><tr><td> IPR004088: K Homology, type 1 </td><td>7.36E−04 </td><td>0.62 </td><td>0.08 </td><td>7.75 </td></tr><tr><td colspan="5">KEGG </td></tr><tr><td> hsa03018: RNA degradation </td><td>1.05E−03 </td><td>0.92 </td><td>0.24 </td><td>3.83 </td></tr><tr><td> hsa04340: Hedgehog signalling pathway </td><td>6.75E−03 </td><td>0.88 </td><td>0.29 </td><td>3.03 </td></tr><tr><td> hsa04520: Adherens junction </td><td>4.55E−03 </td><td>1.06 </td><td>0.37 </td><td>2.86 </td></tr><tr><td> hsa05211: Renal cell carcinoma </td><td>9.88E−03 </td><td>0.92 </td><td>0.33 </td><td>2.79 </td></tr><tr><td> hsa04120: Ubiquitin-mediated proteolysis </td><td>1.05E−03 </td><td>1.5 </td><td>0.57 </td><td>2.63 </td></tr><tr><td> hsa04310: Wnt signalling pathway </td><td>2.22E−04 </td><td>2.07 </td><td>0.79 </td><td>2.62 </td></tr><tr><td> hsa04360: Axon guidance </td><td>3.47E−04 </td><td>1.8 </td><td>0.69 </td><td>2.61 </td></tr><tr><td> hsa04810: Regulation of actin cytoskeleton </td><td>1.07E−03 </td><td>2.07 </td><td>0.94 </td><td>2.20 </td></tr><tr><td> hsa04010: MAPK signaling pathway </td><td>3.23E−04 </td><td>3.08 </td><td>1.5 </td><td>2.05 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The percentages of SCCS genes that have the InterPro or KEGG term</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The percentages of non-SCCS genes that have the InterPro or KEGG term.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The fold of ‘a’ to ‘b’. Terms are listed in the descending order of the fold difference.</p></div></div><div content-id=""><div class="fn"><p>*Probability for enrichment of the InterPro or KEGG terms in the SCCS group.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6650514" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR010TB3" class="table-wrap-title" data-id="DSR010TB3"><div class="label -label">Table 3.</div><div class="caption"><p>InterPro and KEGG terms significantly (<em>P</em> &lt; 0.01) enriched with SCCS genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Terms </th><th><em>P</em>* </th><th>In SCCS containing genes (%)<sup>a</sup> </th><th>In non-SCCS containing genes (%)<sup>b</sup> </th><th>Fold<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="5">InterPro </td></tr><tr><td>IPR003619: MAD homology 1, Dwarfin-type </td><td>1.55E−04 </td><td>0.44 </td><td>0.01 </td><td>44.00 </td></tr><tr><td> IPR001827: Homeobox protein, antennapedia type, conserved site </td><td>1.77E−03 </td><td>0.44 </td><td>0.04 </td><td>11.00 </td></tr><tr><td> IPR000569: HECT </td><td>1.26E−05 </td><td>0.75 </td><td>0.07 </td><td>10.71 </td></tr><tr><td> IPR002077: Voltage-dependent calcium channel, alpha-1 subunit </td><td>5.65E−04 </td><td>0.53 </td><td>0.05 </td><td>10.60 </td></tr><tr><td> IPR010982: Lambda repressor-like, DNA-binding </td><td>1.49E−03 </td><td>0.48 </td><td>0.05 </td><td>9.60 </td></tr><tr><td> IPR002343: Paraneoplastic encephalomyelitis antigen </td><td>9.91E−05 </td><td>0.7 </td><td>0.08 </td><td>8.75 </td></tr><tr><td> IPR018359: Bromodomain, conserved site </td><td>9.75E−04 </td><td>0.57 </td><td>0.07 </td><td>8.14 </td></tr><tr><td> IPR001487: Bromodomain </td><td>1.94E−06 </td><td>0.97 </td><td>0.12 </td><td>8.08 </td></tr><tr><td> IPR017995: Homeobox protein, antennapedia type </td><td>7.36E−04 </td><td>0.62 </td><td>0.08 </td><td>7.75 </td></tr><tr><td> IPR004088: K Homology, type 1 </td><td>7.36E−04 </td><td>0.62 </td><td>0.08 </td><td>7.75 </td></tr><tr><td colspan="5">KEGG </td></tr><tr><td> hsa03018: RNA degradation </td><td>1.05E−03 </td><td>0.92 </td><td>0.24 </td><td>3.83 </td></tr><tr><td> hsa04340: Hedgehog signalling pathway </td><td>6.75E−03 </td><td>0.88 </td><td>0.29 </td><td>3.03 </td></tr><tr><td> hsa04520: Adherens junction </td><td>4.55E−03 </td><td>1.06 </td><td>0.37 </td><td>2.86 </td></tr><tr><td> hsa05211: Renal cell carcinoma </td><td>9.88E−03 </td><td>0.92 </td><td>0.33 </td><td>2.79 </td></tr><tr><td> hsa04120: Ubiquitin-mediated proteolysis </td><td>1.05E−03 </td><td>1.5 </td><td>0.57 </td><td>2.63 </td></tr><tr><td> hsa04310: Wnt signalling pathway </td><td>2.22E−04 </td><td>2.07 </td><td>0.79 </td><td>2.62 </td></tr><tr><td> hsa04360: Axon guidance </td><td>3.47E−04 </td><td>1.8 </td><td>0.69 </td><td>2.61 </td></tr><tr><td> hsa04810: Regulation of actin cytoskeleton </td><td>1.07E−03 </td><td>2.07 </td><td>0.94 </td><td>2.20 </td></tr><tr><td> hsa04010: MAPK signaling pathway </td><td>3.23E−04 </td><td>3.08 </td><td>1.5 </td><td>2.05 </td></tr></tbody></table></div><div content-id="DSR010TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Terms </th><th><em>P</em>* </th><th>In SCCS containing genes (%)<sup>a</sup> </th><th>In non-SCCS containing genes (%)<sup>b</sup> </th><th>Fold<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="5">InterPro </td></tr><tr><td>IPR003619: MAD homology 1, Dwarfin-type </td><td>1.55E−04 </td><td>0.44 </td><td>0.01 </td><td>44.00 </td></tr><tr><td> IPR001827: Homeobox protein, antennapedia type, conserved site </td><td>1.77E−03 </td><td>0.44 </td><td>0.04 </td><td>11.00 </td></tr><tr><td> IPR000569: HECT </td><td>1.26E−05 </td><td>0.75 </td><td>0.07 </td><td>10.71 </td></tr><tr><td> IPR002077: Voltage-dependent calcium channel, alpha-1 subunit </td><td>5.65E−04 </td><td>0.53 </td><td>0.05 </td><td>10.60 </td></tr><tr><td> IPR010982: Lambda repressor-like, DNA-binding </td><td>1.49E−03 </td><td>0.48 </td><td>0.05 </td><td>9.60 </td></tr><tr><td> IPR002343: Paraneoplastic encephalomyelitis antigen </td><td>9.91E−05 </td><td>0.7 </td><td>0.08 </td><td>8.75 </td></tr><tr><td> IPR018359: Bromodomain, conserved site </td><td>9.75E−04 </td><td>0.57 </td><td>0.07 </td><td>8.14 </td></tr><tr><td> IPR001487: Bromodomain </td><td>1.94E−06 </td><td>0.97 </td><td>0.12 </td><td>8.08 </td></tr><tr><td> IPR017995: Homeobox protein, antennapedia type </td><td>7.36E−04 </td><td>0.62 </td><td>0.08 </td><td>7.75 </td></tr><tr><td> IPR004088: K Homology, type 1 </td><td>7.36E−04 </td><td>0.62 </td><td>0.08 </td><td>7.75 </td></tr><tr><td colspan="5">KEGG </td></tr><tr><td> hsa03018: RNA degradation </td><td>1.05E−03 </td><td>0.92 </td><td>0.24 </td><td>3.83 </td></tr><tr><td> hsa04340: Hedgehog signalling pathway </td><td>6.75E−03 </td><td>0.88 </td><td>0.29 </td><td>3.03 </td></tr><tr><td> hsa04520: Adherens junction </td><td>4.55E−03 </td><td>1.06 </td><td>0.37 </td><td>2.86 </td></tr><tr><td> hsa05211: Renal cell carcinoma </td><td>9.88E−03 </td><td>0.92 </td><td>0.33 </td><td>2.79 </td></tr><tr><td> hsa04120: Ubiquitin-mediated proteolysis </td><td>1.05E−03 </td><td>1.5 </td><td>0.57 </td><td>2.63 </td></tr><tr><td> hsa04310: Wnt signalling pathway </td><td>2.22E−04 </td><td>2.07 </td><td>0.79 </td><td>2.62 </td></tr><tr><td> hsa04360: Axon guidance </td><td>3.47E−04 </td><td>1.8 </td><td>0.69 </td><td>2.61 </td></tr><tr><td> hsa04810: Regulation of actin cytoskeleton </td><td>1.07E−03 </td><td>2.07 </td><td>0.94 </td><td>2.20 </td></tr><tr><td> hsa04010: MAPK signaling pathway </td><td>3.23E−04 </td><td>3.08 </td><td>1.5 </td><td>2.05 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The percentages of SCCS genes that have the InterPro or KEGG term</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The percentages of non-SCCS genes that have the InterPro or KEGG term.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The fold of ‘a’ to ‘b’. Terms are listed in the descending order of the fold difference.</p></div></div><div content-id=""><div class="fn"><p>*Probability for enrichment of the InterPro or KEGG terms in the SCCS group.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6650514" target="_blank">View Large</a></div></div><p>The all terms in GO Biological process section is related to developmental process. One term of Cellular components (GO:0014704) mediate mechanical and electrochemical integration between cardiomyocytes and the rest of the five (GO:0043198, GO:0030425, GO:0043025, GO:0015629, and GO:0043005) have an association with the neuron. In the molecular function category, three terms (GO:0004843, GO:0031625, and GO:0004221) are related to the ubiquitin system, two (GO:00042054 and GO:00003682) are associated with chromatin. Ubiquitins are known to be involved not only with protein degeneration but also with signal transduction, chromatin modification, and cell cycle.</p><p>Of the ten Interpro codes listed in Table <a reveal-id="DSR010TB3" data-open="DSR010TB3" class="link link-reveal link-table xref-fig">3</a>, IPR001827 is related to ubiquitin and IPR002077 represents calcium channel and other eight are all associated with DNA or RNA-binding functions that mediate transcriptional regulation or chromatin modification.</p><p>Tow KEGG pathways (hsa04340 and hsa04310) in Table <a reveal-id="DSR010TB3" data-open="DSR010TB3" class="link link-reveal link-table xref-fig">3</a> are developmental signalling pathways, hsa04120 is related to the ubiquitin system, and hsa04360 is involved in axon guidance, which well corresponds with the GO and Interpro terms.</p><p>Table <a reveal-id="DSR010TB4" data-open="DSR010TB4" class="link link-reveal link-table xref-fig">4</a> shows the terms that are significantly scarce in SCCS genes. In contrast to Tables <a reveal-id="DSR010TB2" data-open="DSR010TB2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSR010TB3" data-open="DSR010TB3" class="link link-reveal link-table xref-fig">3</a>, majority of the terms are involved with metabolic processes. Fatigo can also explore enrichment of micro RNA target and transcription-factor-binding sites; no significant item was found. </p>                        <a id="6650519" scrollto-destination="6650519"></a>
<div content-id="DSR010TB4" class="table-modal"><div class="table-wrap"><div id="DSR010TB4" class="table-wrap-title" data-id="DSR010TB4"><div class="label -label">Table 4.</div><div class="caption"><p>GO, InterPro and KEGG terms significantly (<em>P</em> &lt; 0.01) scarce in SCCS genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Terms </th><th><em>P</em>* </th><th>In SCCS containing genes (%)<sup>a</sup> </th><th>In non-SCCS containing genes (%)<sup>b</sup> </th><th>Fold<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="5">Biological process </td></tr><tr><td>GO:0043039: tRNA aminoacylation </td><td>6.15E−03 </td><td>0.09 </td><td>0.63 </td><td>0.14 </td></tr><tr><td> GO:0006725: Cellular aromatic compound metabolic process </td><td>7.31E−03 </td><td>0.48 </td><td>1.3 </td><td>0.37 </td></tr><tr><td> GO:0051186: Cofactor metabolic process </td><td>6.05E−03 </td><td>0.66 </td><td>1.59 </td><td>0.42 </td></tr><tr><td> GO:0055114: Oxidation-reduction process </td><td>5.86E−04 </td><td>2.11 </td><td>3.87 </td><td>0.55 </td></tr><tr><td> GO:0019752: Carboxylic acid metabolic process </td><td>3.54E−04 </td><td>2.42 </td><td>4.34 </td><td>0.56 </td></tr><tr><td> GO:0006082: Organic acid metabolic process </td><td>3.54E−04 </td><td>2.42 </td><td>4.35 </td><td>0.56 </td></tr><tr><td> GO:0044255: Cellular lipid metabolic process </td><td>8.60E−03 </td><td>3.3 </td><td>4.98 </td><td>0.66 </td></tr><tr><td colspan="5">Cellular component </td></tr><tr><td> GO:0019866: Organelle inner membrane </td><td>2.38E−03 </td><td>0.66 </td><td>1.72 </td><td>0.38 </td></tr><tr><td> GO:0044429: Mitochondrial part </td><td>1.27E−03 </td><td>1.63 </td><td>3.24 </td><td>0.50 </td></tr><tr><td> GO:0005615: Extracellular space </td><td>2.87E−03 </td><td>2.29 </td><td>3.93 </td><td>0.58 </td></tr><tr><td colspan="5">Molecular function </td></tr><tr><td> GO:0001595: Angiotensin receptor activity </td><td>8.64E−03 </td><td>0 </td><td>0.4 </td><td>0.00 </td></tr><tr><td> GO:0016616: Oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor </td><td>6.42E−03 </td><td>0.18 </td><td>0.82 </td><td>0.22 </td></tr><tr><td> GO:0016614: Oxidoreductase activity, acting on CH-OH group of donors </td><td>8.92E−03 </td><td>0.26 </td><td>0.97 </td><td>0.27 </td></tr><tr><td colspan="5">Interpro </td></tr><tr><td> IPR002198: Short-chain dehydrogenase/reductase SDR </td><td>8.96E−03 </td><td>0 </td><td>0.46 </td><td>0.00 </td></tr><tr><td colspan="5">Kegg </td></tr><tr><td> hsa04060: Cytokine–cytokine receptor interaction </td><td>2.86E−03 </td><td>0.57 </td><td>1.56 </td><td>0.37 </td></tr></tbody></table></div><div content-id="DSR010TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Terms </th><th><em>P</em>* </th><th>In SCCS containing genes (%)<sup>a</sup> </th><th>In non-SCCS containing genes (%)<sup>b</sup> </th><th>Fold<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="5">Biological process </td></tr><tr><td>GO:0043039: tRNA aminoacylation </td><td>6.15E−03 </td><td>0.09 </td><td>0.63 </td><td>0.14 </td></tr><tr><td> GO:0006725: Cellular aromatic compound metabolic process </td><td>7.31E−03 </td><td>0.48 </td><td>1.3 </td><td>0.37 </td></tr><tr><td> GO:0051186: Cofactor metabolic process </td><td>6.05E−03 </td><td>0.66 </td><td>1.59 </td><td>0.42 </td></tr><tr><td> GO:0055114: Oxidation-reduction process </td><td>5.86E−04 </td><td>2.11 </td><td>3.87 </td><td>0.55 </td></tr><tr><td> GO:0019752: Carboxylic acid metabolic process </td><td>3.54E−04 </td><td>2.42 </td><td>4.34 </td><td>0.56 </td></tr><tr><td> GO:0006082: Organic acid metabolic process </td><td>3.54E−04 </td><td>2.42 </td><td>4.35 </td><td>0.56 </td></tr><tr><td> GO:0044255: Cellular lipid metabolic process </td><td>8.60E−03 </td><td>3.3 </td><td>4.98 </td><td>0.66 </td></tr><tr><td colspan="5">Cellular component </td></tr><tr><td> GO:0019866: Organelle inner membrane </td><td>2.38E−03 </td><td>0.66 </td><td>1.72 </td><td>0.38 </td></tr><tr><td> GO:0044429: Mitochondrial part </td><td>1.27E−03 </td><td>1.63 </td><td>3.24 </td><td>0.50 </td></tr><tr><td> GO:0005615: Extracellular space </td><td>2.87E−03 </td><td>2.29 </td><td>3.93 </td><td>0.58 </td></tr><tr><td colspan="5">Molecular function </td></tr><tr><td> GO:0001595: Angiotensin receptor activity </td><td>8.64E−03 </td><td>0 </td><td>0.4 </td><td>0.00 </td></tr><tr><td> GO:0016616: Oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor </td><td>6.42E−03 </td><td>0.18 </td><td>0.82 </td><td>0.22 </td></tr><tr><td> GO:0016614: Oxidoreductase activity, acting on CH-OH group of donors </td><td>8.92E−03 </td><td>0.26 </td><td>0.97 </td><td>0.27 </td></tr><tr><td colspan="5">Interpro </td></tr><tr><td> IPR002198: Short-chain dehydrogenase/reductase SDR </td><td>8.96E−03 </td><td>0 </td><td>0.46 </td><td>0.00 </td></tr><tr><td colspan="5">Kegg </td></tr><tr><td> hsa04060: Cytokine–cytokine receptor interaction </td><td>2.86E−03 </td><td>0.57 </td><td>1.56 </td><td>0.37 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The percentages of SCCS genes that have the GO, InterPro or KEGG term.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The percentages of non-SCCS genes that have the GO, InterPro or KEGG term.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The fold of ‘a’ to ‘b’. Terms are listed in the ascending order of the fold difference.</p></div></div><div content-id=""><div class="fn"><p>*Probability for enrichment of the GO, InterPro or KEGG terms in the SCCS group.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6650519" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR010TB4" class="table-wrap-title" data-id="DSR010TB4"><div class="label -label">Table 4.</div><div class="caption"><p>GO, InterPro and KEGG terms significantly (<em>P</em> &lt; 0.01) scarce in SCCS genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Terms </th><th><em>P</em>* </th><th>In SCCS containing genes (%)<sup>a</sup> </th><th>In non-SCCS containing genes (%)<sup>b</sup> </th><th>Fold<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="5">Biological process </td></tr><tr><td>GO:0043039: tRNA aminoacylation </td><td>6.15E−03 </td><td>0.09 </td><td>0.63 </td><td>0.14 </td></tr><tr><td> GO:0006725: Cellular aromatic compound metabolic process </td><td>7.31E−03 </td><td>0.48 </td><td>1.3 </td><td>0.37 </td></tr><tr><td> GO:0051186: Cofactor metabolic process </td><td>6.05E−03 </td><td>0.66 </td><td>1.59 </td><td>0.42 </td></tr><tr><td> GO:0055114: Oxidation-reduction process </td><td>5.86E−04 </td><td>2.11 </td><td>3.87 </td><td>0.55 </td></tr><tr><td> GO:0019752: Carboxylic acid metabolic process </td><td>3.54E−04 </td><td>2.42 </td><td>4.34 </td><td>0.56 </td></tr><tr><td> GO:0006082: Organic acid metabolic process </td><td>3.54E−04 </td><td>2.42 </td><td>4.35 </td><td>0.56 </td></tr><tr><td> GO:0044255: Cellular lipid metabolic process </td><td>8.60E−03 </td><td>3.3 </td><td>4.98 </td><td>0.66 </td></tr><tr><td colspan="5">Cellular component </td></tr><tr><td> GO:0019866: Organelle inner membrane </td><td>2.38E−03 </td><td>0.66 </td><td>1.72 </td><td>0.38 </td></tr><tr><td> GO:0044429: Mitochondrial part </td><td>1.27E−03 </td><td>1.63 </td><td>3.24 </td><td>0.50 </td></tr><tr><td> GO:0005615: Extracellular space </td><td>2.87E−03 </td><td>2.29 </td><td>3.93 </td><td>0.58 </td></tr><tr><td colspan="5">Molecular function </td></tr><tr><td> GO:0001595: Angiotensin receptor activity </td><td>8.64E−03 </td><td>0 </td><td>0.4 </td><td>0.00 </td></tr><tr><td> GO:0016616: Oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor </td><td>6.42E−03 </td><td>0.18 </td><td>0.82 </td><td>0.22 </td></tr><tr><td> GO:0016614: Oxidoreductase activity, acting on CH-OH group of donors </td><td>8.92E−03 </td><td>0.26 </td><td>0.97 </td><td>0.27 </td></tr><tr><td colspan="5">Interpro </td></tr><tr><td> IPR002198: Short-chain dehydrogenase/reductase SDR </td><td>8.96E−03 </td><td>0 </td><td>0.46 </td><td>0.00 </td></tr><tr><td colspan="5">Kegg </td></tr><tr><td> hsa04060: Cytokine–cytokine receptor interaction </td><td>2.86E−03 </td><td>0.57 </td><td>1.56 </td><td>0.37 </td></tr></tbody></table></div><div content-id="DSR010TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>Terms </th><th><em>P</em>* </th><th>In SCCS containing genes (%)<sup>a</sup> </th><th>In non-SCCS containing genes (%)<sup>b</sup> </th><th>Fold<sup>c</sup> </th></tr></thead><tbody><tr><td colspan="5">Biological process </td></tr><tr><td>GO:0043039: tRNA aminoacylation </td><td>6.15E−03 </td><td>0.09 </td><td>0.63 </td><td>0.14 </td></tr><tr><td> GO:0006725: Cellular aromatic compound metabolic process </td><td>7.31E−03 </td><td>0.48 </td><td>1.3 </td><td>0.37 </td></tr><tr><td> GO:0051186: Cofactor metabolic process </td><td>6.05E−03 </td><td>0.66 </td><td>1.59 </td><td>0.42 </td></tr><tr><td> GO:0055114: Oxidation-reduction process </td><td>5.86E−04 </td><td>2.11 </td><td>3.87 </td><td>0.55 </td></tr><tr><td> GO:0019752: Carboxylic acid metabolic process </td><td>3.54E−04 </td><td>2.42 </td><td>4.34 </td><td>0.56 </td></tr><tr><td> GO:0006082: Organic acid metabolic process </td><td>3.54E−04 </td><td>2.42 </td><td>4.35 </td><td>0.56 </td></tr><tr><td> GO:0044255: Cellular lipid metabolic process </td><td>8.60E−03 </td><td>3.3 </td><td>4.98 </td><td>0.66 </td></tr><tr><td colspan="5">Cellular component </td></tr><tr><td> GO:0019866: Organelle inner membrane </td><td>2.38E−03 </td><td>0.66 </td><td>1.72 </td><td>0.38 </td></tr><tr><td> GO:0044429: Mitochondrial part </td><td>1.27E−03 </td><td>1.63 </td><td>3.24 </td><td>0.50 </td></tr><tr><td> GO:0005615: Extracellular space </td><td>2.87E−03 </td><td>2.29 </td><td>3.93 </td><td>0.58 </td></tr><tr><td colspan="5">Molecular function </td></tr><tr><td> GO:0001595: Angiotensin receptor activity </td><td>8.64E−03 </td><td>0 </td><td>0.4 </td><td>0.00 </td></tr><tr><td> GO:0016616: Oxidoreductase activity, acting on the CH-OH group of donors, NAD or NADP as acceptor </td><td>6.42E−03 </td><td>0.18 </td><td>0.82 </td><td>0.22 </td></tr><tr><td> GO:0016614: Oxidoreductase activity, acting on CH-OH group of donors </td><td>8.92E−03 </td><td>0.26 </td><td>0.97 </td><td>0.27 </td></tr><tr><td colspan="5">Interpro </td></tr><tr><td> IPR002198: Short-chain dehydrogenase/reductase SDR </td><td>8.96E−03 </td><td>0 </td><td>0.46 </td><td>0.00 </td></tr><tr><td colspan="5">Kegg </td></tr><tr><td> hsa04060: Cytokine–cytokine receptor interaction </td><td>2.86E−03 </td><td>0.57 </td><td>1.56 </td><td>0.37 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The percentages of SCCS genes that have the GO, InterPro or KEGG term.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The percentages of non-SCCS genes that have the GO, InterPro or KEGG term.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The fold of ‘a’ to ‘b’. Terms are listed in the ascending order of the fold difference.</p></div></div><div content-id=""><div class="fn"><p>*Probability for enrichment of the GO, InterPro or KEGG terms in the SCCS group.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6650519" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6650520 id="6650520" class="section-title" data-legacy-id=s3c>3.3. Overlap with UCEs</h3>
<p>UCEs are defined as nucleotide sequences that are absolutely conserved longer than 200 bp between orthologous regions of the human, rat and mouse.<sup><a reveal-id="DSR010C18" data-open="DSR010C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSR010C19" data-open="DSR010C19" class="link link-ref link-reveal xref-bibr">19</a></sup> UCEs are found in both coding and non-coding regions. Precedent studies report that genes with low synonymous substitution or genes overlapped with UCEs are associated with DNA binding, RNA binding, transcription activity, and Homeobox.<sup><a reveal-id="DSR010C18" data-open="DSR010C18" class="link link-ref link-reveal xref-bibr">18</a>,<a reveal-id="DSR010C19" data-open="DSR010C19" class="link link-ref link-reveal xref-bibr">19</a></sup> In our 10 790 genes, 4009 bp in 29 genes are found to overlap with UCEs. In the 4009 bp region, 2835 bp in 22 genes overlap with SCCSs (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary file S4</a>). Because SCCSs are conserved in six mammalian species, including the three species referred for UCEs, the 2835 bp reflect conservation in other three species (macaque, cow, and dog). We surveyed nucleotide sites in the 10 790 genes that are conserved among human, rat, and mouse, and evaluated how many of them are also conserved in macaque, cow, and dog. The resulted ratio is 0.751; therefore the expected conservation is 3035 bp. This matches well with the observation. The regions overlapped with UCEs make only 1.47% of the entire SCCS regions. Other SCCSs convey shorter but deeper conservation than UCEs.</p>                            <h3 scrollto-destination=6650522 id="6650522" class="section-title" data-legacy-id=s3d>3.4. SCCSs that form stable RNA secondary structures</h3>
<p>There are cases that a secondary structure of mRNA conveys functions.<sup><a reveal-id="DSR010C24" data-open="DSR010C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSR010C25" data-open="DSR010C25" class="link link-ref link-reveal xref-bibr">25</a></sup> We examined secondary structures and free energy of the SCCSs using Vienna RNA package. We found three SCCSs whose folding energy were significantly low (Table <a reveal-id="DSR010TB5" data-open="DSR010TB5" class="link link-reveal link-table xref-fig">5</a>). </p>                        <a id="6650524" scrollto-destination="6650524"></a>
<div content-id="DSR010TB5" class="table-modal"><div class="table-wrap"><div id="DSR010TB5" class="table-wrap-title" data-id="DSR010TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Genes containing SCCS with significantly (<em>P</em> &lt; 0.001) low free folding energy</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Gene </th><th>Length </th><th>Free energy </th></tr></thead><tbody><tr><td><em>polg</em> </td><td>DNA polymerase subunit gamma-1 </td><td>36 </td><td>−19.9 </td></tr><tr><td><em>gal3st3</em> </td><td>Galactose-3-O-sulfotransferase 3 </td><td>36 </td><td>−22.6 </td></tr><tr><td><em>smarcd3</em> </td><td>SWI/SNF-related matrix-associated actin-dependent regulator of chromatin subfamily D member </td><td>39 </td><td>−23.9 </td></tr></tbody></table></div><div content-id="DSR010TB5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Gene </th><th>Length </th><th>Free energy </th></tr></thead><tbody><tr><td><em>polg</em> </td><td>DNA polymerase subunit gamma-1 </td><td>36 </td><td>−19.9 </td></tr><tr><td><em>gal3st3</em> </td><td>Galactose-3-O-sulfotransferase 3 </td><td>36 </td><td>−22.6 </td></tr><tr><td><em>smarcd3</em> </td><td>SWI/SNF-related matrix-associated actin-dependent regulator of chromatin subfamily D member </td><td>39 </td><td>−23.9 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The gene names are represented by those of human.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6650524" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR010TB5" class="table-wrap-title" data-id="DSR010TB5"><div class="label -label">Table 5.</div><div class="caption"><p>Genes containing SCCS with significantly (<em>P</em> &lt; 0.001) low free folding energy</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Gene </th><th>Length </th><th>Free energy </th></tr></thead><tbody><tr><td><em>polg</em> </td><td>DNA polymerase subunit gamma-1 </td><td>36 </td><td>−19.9 </td></tr><tr><td><em>gal3st3</em> </td><td>Galactose-3-O-sulfotransferase 3 </td><td>36 </td><td>−22.6 </td></tr><tr><td><em>smarcd3</em> </td><td>SWI/SNF-related matrix-associated actin-dependent regulator of chromatin subfamily D member </td><td>39 </td><td>−23.9 </td></tr></tbody></table></div><div content-id="DSR010TB5" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Gene </th><th>Length </th><th>Free energy </th></tr></thead><tbody><tr><td><em>polg</em> </td><td>DNA polymerase subunit gamma-1 </td><td>36 </td><td>−19.9 </td></tr><tr><td><em>gal3st3</em> </td><td>Galactose-3-O-sulfotransferase 3 </td><td>36 </td><td>−22.6 </td></tr><tr><td><em>smarcd3</em> </td><td>SWI/SNF-related matrix-associated actin-dependent regulator of chromatin subfamily D member </td><td>39 </td><td>−23.9 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>The gene names are represented by those of human.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6650524" target="_blank">View Large</a></div></div><p><em>Polg</em> encodes a catalytic subunit of mitochondrial DNA polymerase POLG. The POLG protein is the only polymerase known to be involved in replication of mtDNA. <em>Gal3st3</em> encodes a member of the galactose-3-O-sulfotransferase protein family. This protein exists on the membrane of Golgi apparatus. <em>Smarcd3</em> encodes a protein of SWI/SNF family, whose members display helicase and ATPase activities. This protein is thought to regulate transcription of target genes by altering the chromatin structure around those genes.</p><p>Figure <a reveal-id="DSR010F2" data-open="DSR010F2" class="link link-reveal link-table xref-fig">2</a> shows the probability density of the folding free energy constructed by randomly extracted sequences. Each line shows the free energy of the sequences of the same length as the above three SCCSs. Gene names on the lines represent free energy of the SCCSs. This figure suggests that these three SCCSs have extremely low free energy and that these regions will form stable secondary structures. These SCCSs may be conserved because of the requirement for the secondary structures. However, substitution restriction of this type might be modest because a secondary structure can be retained by another combination of nucleotides as far as the complementarity is maintained. </p>                        <a id="6650527" scrollto-destination="6650527"></a>
<div id="DSR010F2" data-id="DSR010F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/m_dsr01002.jpeg?Expires=2147483647&Signature=GRwgut9aTKClsQ6KFADCVpsVDp2LH3erX518SOGWYAsfDkMkLJkW1bnkHLGwf--hX5KjK9D7N0~xC059ccU~eRuJ00b32cUvsc03zHn~Qt8QKTPBdez6qTK622m2Y3eVGi7sQeQGuNkwwSxtv2fnc90EwUk0oAPdCkCE2685SoyU0rHlE15LfcL4jyp5IEmrXpOlIErtrObR5oktPLj1B249560aaa0LQvG2rDHk9jsdUhFoaQLPf6~wIBKtUPUYc2hML4IxmddPkPfb8QSuo4JI1QOHUCEjXV0YBlWtFeOEs5buYBR92bzAYKyjMw0VGxON4Wm0cCQKG6nPa2Z4gA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The probability density of the folding free energy. This graph shows the probability density of the folding free energy for 36 and 39 nucleotide-long sequences. The three SCCSs with significantly low free energy are indicated on the graph. Probability density was created using software package R.38" path-from-xml="dsr01002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6650527/dsr01002.jpeg" path-from-xml="dsr01002.jpeg" target="_blank">View large</a><a section="6650527" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/dsr01002.jpeg?Expires=2147483647&Signature=JkV7G3Ev6TQOfNiQCou9XO4qtOfCv1I6z1GM31txSzMmxWyd~SFHmV12Lh4TpoQKOcsBCDBYq-7yQBGZparmZBQXXQOTM2qhsHJMvg42JO0SdG-K3M5cjWuoodahkmE80QMpLLTriM6mNVjsHnr1krzNOkQQhmKLDe0wkhQHAP2jfUugti3ViCKSR2U2retQxLtBQA3fJb-g-ettZQEabXD0Ax1kZ-xd0d6OHheu5in61RlC52CiYJiL195-EU6ydrPAtlb7qfFa-j-GnLM8WP2CLnGSn02OQ3evu8B6VCUMKeHWY2NBWN5WRmIXKJjD2eE82QYXYGw8NdPq8VbvqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6650527&ar=444849&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The probability density of the folding free energy. This graph shows the probability density of the folding free energy for 36 and 39 nucleotide-long sequences. The three SCCSs with significantly low free energy are indicated on the graph. Probability density was created using software package R.<sup><a reveal-id="DSR010C38" data-open="DSR010C38" class="link link-ref link-reveal xref-bibr">38</a></sup></p></div></div><div content-id="DSR010F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/m_dsr01002.jpeg?Expires=2147483647&Signature=GRwgut9aTKClsQ6KFADCVpsVDp2LH3erX518SOGWYAsfDkMkLJkW1bnkHLGwf--hX5KjK9D7N0~xC059ccU~eRuJ00b32cUvsc03zHn~Qt8QKTPBdez6qTK622m2Y3eVGi7sQeQGuNkwwSxtv2fnc90EwUk0oAPdCkCE2685SoyU0rHlE15LfcL4jyp5IEmrXpOlIErtrObR5oktPLj1B249560aaa0LQvG2rDHk9jsdUhFoaQLPf6~wIBKtUPUYc2hML4IxmddPkPfb8QSuo4JI1QOHUCEjXV0YBlWtFeOEs5buYBR92bzAYKyjMw0VGxON4Wm0cCQKG6nPa2Z4gA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The probability density of the folding free energy. This graph shows the probability density of the folding free energy for 36 and 39 nucleotide-long sequences. The three SCCSs with significantly low free energy are indicated on the graph. Probability density was created using software package R.38" path-from-xml="dsr01002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6650527/dsr01002.jpeg" path-from-xml="dsr01002.jpeg" target="_blank">View large</a><a section="6650527" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/dsr01002.jpeg?Expires=2147483647&Signature=JkV7G3Ev6TQOfNiQCou9XO4qtOfCv1I6z1GM31txSzMmxWyd~SFHmV12Lh4TpoQKOcsBCDBYq-7yQBGZparmZBQXXQOTM2qhsHJMvg42JO0SdG-K3M5cjWuoodahkmE80QMpLLTriM6mNVjsHnr1krzNOkQQhmKLDe0wkhQHAP2jfUugti3ViCKSR2U2retQxLtBQA3fJb-g-ettZQEabXD0Ax1kZ-xd0d6OHheu5in61RlC52CiYJiL195-EU6ydrPAtlb7qfFa-j-GnLM8WP2CLnGSn02OQ3evu8B6VCUMKeHWY2NBWN5WRmIXKJjD2eE82QYXYGw8NdPq8VbvqA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6650527&ar=444849&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The probability density of the folding free energy. This graph shows the probability density of the folding free energy for 36 and 39 nucleotide-long sequences. The three SCCSs with significantly low free energy are indicated on the graph. Probability density was created using software package R.<sup><a reveal-id="DSR010C38" data-open="DSR010C38" class="link link-ref link-reveal xref-bibr">38</a></sup></p></div></div>                            <h3 scrollto-destination=6650528 id="6650528" class="section-title" data-legacy-id=s3e>3.5. The density of exonic splicing enhancers in SCCSs and in the other coding regions</h3>
<p>One of the well-known functional nucleotide elements in the coding region is splicing signals. We obtained 238 hexamers from RESCUE-ESE Web server as candidates of exonic splicing enhancers, and counted the number of hexamers in SCCSs and non-SCCS regions of the 10 790 genes (Table <a reveal-id="DSR010TB6" data-open="DSR010TB6" class="link link-reveal link-table xref-fig">6</a>). Splicing signals in non-SCCS regions are counted on human sequences. We observed 20 420 hits of signals in 192 306 bp of SCCS regions and 2 183 544 hits in 205 666 520 of non-SCCS regions. The number of the signals per nucleotide is both 0.106 for SCCS and non-SCCS, and there was no significant difference (<em>P</em> = 0.99, χ<sup>2</sup> test). </p>                        <a id="6650530" scrollto-destination="6650530"></a>
<div content-id="DSR010TB6" class="table-modal"><div class="table-wrap"><div id="DSR010TB6" class="table-wrap-title" data-id="DSR010TB6"><div class="label -label">Table 6.</div><div class="caption"><p>Splicing signals in SCCS and non-SCCS regions of 10 790 genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Region size (bp) </th><th>#Splicing signals </th><th>Per nucleotide </th></tr></thead><tbody><tr><td>SCCS </td><td>192 306 </td><td>20 420 </td><td>0.106 </td></tr><tr><td>Non-SCCS </td><td>20 566 520 </td><td>2 183 544 </td><td>0.106 </td></tr></tbody></table></div><div content-id="DSR010TB6" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Region size (bp) </th><th>#Splicing signals </th><th>Per nucleotide </th></tr></thead><tbody><tr><td>SCCS </td><td>192 306 </td><td>20 420 </td><td>0.106 </td></tr><tr><td>Non-SCCS </td><td>20 566 520 </td><td>2 183 544 </td><td>0.106 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Splicing signals in non-SCCS regions are counted on human sequences.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6650530" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR010TB6" class="table-wrap-title" data-id="DSR010TB6"><div class="label -label">Table 6.</div><div class="caption"><p>Splicing signals in SCCS and non-SCCS regions of 10 790 genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Region size (bp) </th><th>#Splicing signals </th><th>Per nucleotide </th></tr></thead><tbody><tr><td>SCCS </td><td>192 306 </td><td>20 420 </td><td>0.106 </td></tr><tr><td>Non-SCCS </td><td>20 566 520 </td><td>2 183 544 </td><td>0.106 </td></tr></tbody></table></div><div content-id="DSR010TB6" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Region size (bp) </th><th>#Splicing signals </th><th>Per nucleotide </th></tr></thead><tbody><tr><td>SCCS </td><td>192 306 </td><td>20 420 </td><td>0.106 </td></tr><tr><td>Non-SCCS </td><td>20 566 520 </td><td>2 183 544 </td><td>0.106 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>Splicing signals in non-SCCS regions are counted on human sequences.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6650530" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6650531 id="6650531" class="section-title" data-legacy-id=s3f>3.6. Gene expression</h3>
<p>We investigated the difference of gene expression between SCCS genes and non-SCCS genes referring to anatomical system data of EGenetics, which give qualitative information about in what organs a gene is expressed. We counted the number of SCCS genes and non-SCCS genes expressed in the organs and performed the Fisher's exact test as described in Materials and method section.</p><p>We compared the percentages of genes that are expressed in each organ. Table <a reveal-id="DSR010TB7" data-open="DSR010TB7" class="link link-reveal link-table xref-fig">7</a> shows organs in which significantly higher percentage of SCCS genes are expressed compared with non-SCCS genes. In general, SCCS genes are expressed in a wider variety of organs. This observation agrees with a previous study.<sup><a reveal-id="DSR010C35" data-open="DSR010C35" class="link link-ref link-reveal xref-bibr">35</a></sup> Only in medulla oblongata and trophoblast, non-SCCS genes showed significantly higher percentage than SCCS genes (data not shown). </p>                        <a id="6650534" scrollto-destination="6650534"></a>
<div content-id="DSR010TB7" class="table-modal"><div class="table-wrap"><div id="DSR010TB7" class="table-wrap-title" data-id="DSR010TB7"><div class="label -label">Table 7.</div><div class="caption"><p>Organs in which significantly (<em>P</em> &lt; 0.001) higher percentage of SCCS genes are expressed compared with non-SCCS genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Organ </th><th rowspan="2"><em>P</em>* </th><th colspan="2">SCCS<hr /> </th><th colspan="2">Non-SCCS<hr /> </th><th rowspan="2">Fold<sup>c</sup> </th></tr><tr><th>#Expressed </th><th>Percentage<sup>a</sup> </th><th>#Expressed </th><th>Percentage<sup>b</sup> </th></tr></thead><tbody><tr><td>Amygdala </td><td>1.32E−11 </td><td>201 </td><td>9.86 </td><td>318 </td><td>5.37 </td><td>1.84 </td></tr><tr><td>Cochlea </td><td>1.24E−22 </td><td>436 </td><td>21.39 </td><td>723 </td><td>12.21 </td><td>1.75 </td></tr><tr><td>Small intestine </td><td>5.28E−03 </td><td>49 </td><td>2.40 </td><td>86 </td><td>1.45 </td><td>1.66 </td></tr><tr><td>Amnion </td><td>1.03E−04 </td><td>102 </td><td>5.00 </td><td>183 </td><td>3.09 </td><td>1.62 </td></tr><tr><td>Amniotic fluid </td><td>9.12E−07 </td><td>175 </td><td>8.59 </td><td>321 </td><td>5.42 </td><td>1.58 </td></tr><tr><td>Spinal cord </td><td>7.21E−05 </td><td>129 </td><td>6.33 </td><td>243 </td><td>4.10 </td><td>1.54 </td></tr><tr><td>Artery </td><td>9.80E−05 </td><td>133 </td><td>6.53 </td><td>254 </td><td>4.29 </td><td>1.52 </td></tr><tr><td>Cerebellum cortex </td><td>2.74E−03 </td><td>83 </td><td>4.07 </td><td>160 </td><td>2.70 </td><td>1.51 </td></tr><tr><td>Cerebellum </td><td>3.62E−08 </td><td>277 </td><td>13.59 </td><td>543 </td><td>9.17 </td><td>1.48 </td></tr><tr><td>Trabecular meshwork </td><td>1.38E−05 </td><td>199 </td><td>9.76 </td><td>399 </td><td>6.74 </td><td>1.45 </td></tr><tr><td>Frontal lobe </td><td>1.63E−28 </td><td>899 </td><td>44.11 </td><td>1803 </td><td>30.45 </td><td>1.45 </td></tr><tr><td>Hypopharynx </td><td>1.75E−06 </td><td>246 </td><td>12.07 </td><td>497 </td><td>8.39 </td><td>1.44 </td></tr><tr><td>Pituitary gland </td><td>1.67E−09 </td><td>421 </td><td>20.66 </td><td>877 </td><td>14.81 </td><td>1.39 </td></tr><tr><td>Sympathetic chain </td><td>9.65E−06 </td><td>271 </td><td>13.30 </td><td>575 </td><td>9.71 </td><td>1.37 </td></tr><tr><td>Breast </td><td>5.95E−30 </td><td>1133 </td><td>55.59 </td><td>2430 </td><td>41.04 </td><td>1.35 </td></tr><tr><td>Larynx </td><td>9.05E−13 </td><td>642 </td><td>31.50 </td><td>1385 </td><td>23.39 </td><td>1.35 </td></tr><tr><td>Tongue </td><td>4.55E−07 </td><td>377 </td><td>18.50 </td><td>816 </td><td>13.78 </td><td>1.34 </td></tr><tr><td>Smooth muscle </td><td>1.36E−05 </td><td>307 </td><td>15.06 </td><td>670 </td><td>11.32 </td><td>1.33 </td></tr><tr><td>Thyroid </td><td>3.36E−23 </td><td>1076 </td><td>52.80 </td><td>2375 </td><td>40.11 </td><td>1.32 </td></tr><tr><td>Adrenal gland </td><td>4.51E−06 </td><td>362 </td><td>17.76 </td><td>801 </td><td>13.53 </td><td>1.31 </td></tr></tbody></table></div><div content-id="DSR010TB7" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Organ </th><th rowspan="2"><em>P</em>* </th><th colspan="2">SCCS<hr /> </th><th colspan="2">Non-SCCS<hr /> </th><th rowspan="2">Fold<sup>c</sup> </th></tr><tr><th>#Expressed </th><th>Percentage<sup>a</sup> </th><th>#Expressed </th><th>Percentage<sup>b</sup> </th></tr></thead><tbody><tr><td>Amygdala </td><td>1.32E−11 </td><td>201 </td><td>9.86 </td><td>318 </td><td>5.37 </td><td>1.84 </td></tr><tr><td>Cochlea </td><td>1.24E−22 </td><td>436 </td><td>21.39 </td><td>723 </td><td>12.21 </td><td>1.75 </td></tr><tr><td>Small intestine </td><td>5.28E−03 </td><td>49 </td><td>2.40 </td><td>86 </td><td>1.45 </td><td>1.66 </td></tr><tr><td>Amnion </td><td>1.03E−04 </td><td>102 </td><td>5.00 </td><td>183 </td><td>3.09 </td><td>1.62 </td></tr><tr><td>Amniotic fluid </td><td>9.12E−07 </td><td>175 </td><td>8.59 </td><td>321 </td><td>5.42 </td><td>1.58 </td></tr><tr><td>Spinal cord </td><td>7.21E−05 </td><td>129 </td><td>6.33 </td><td>243 </td><td>4.10 </td><td>1.54 </td></tr><tr><td>Artery </td><td>9.80E−05 </td><td>133 </td><td>6.53 </td><td>254 </td><td>4.29 </td><td>1.52 </td></tr><tr><td>Cerebellum cortex </td><td>2.74E−03 </td><td>83 </td><td>4.07 </td><td>160 </td><td>2.70 </td><td>1.51 </td></tr><tr><td>Cerebellum </td><td>3.62E−08 </td><td>277 </td><td>13.59 </td><td>543 </td><td>9.17 </td><td>1.48 </td></tr><tr><td>Trabecular meshwork </td><td>1.38E−05 </td><td>199 </td><td>9.76 </td><td>399 </td><td>6.74 </td><td>1.45 </td></tr><tr><td>Frontal lobe </td><td>1.63E−28 </td><td>899 </td><td>44.11 </td><td>1803 </td><td>30.45 </td><td>1.45 </td></tr><tr><td>Hypopharynx </td><td>1.75E−06 </td><td>246 </td><td>12.07 </td><td>497 </td><td>8.39 </td><td>1.44 </td></tr><tr><td>Pituitary gland </td><td>1.67E−09 </td><td>421 </td><td>20.66 </td><td>877 </td><td>14.81 </td><td>1.39 </td></tr><tr><td>Sympathetic chain </td><td>9.65E−06 </td><td>271 </td><td>13.30 </td><td>575 </td><td>9.71 </td><td>1.37 </td></tr><tr><td>Breast </td><td>5.95E−30 </td><td>1133 </td><td>55.59 </td><td>2430 </td><td>41.04 </td><td>1.35 </td></tr><tr><td>Larynx </td><td>9.05E−13 </td><td>642 </td><td>31.50 </td><td>1385 </td><td>23.39 </td><td>1.35 </td></tr><tr><td>Tongue </td><td>4.55E−07 </td><td>377 </td><td>18.50 </td><td>816 </td><td>13.78 </td><td>1.34 </td></tr><tr><td>Smooth muscle </td><td>1.36E−05 </td><td>307 </td><td>15.06 </td><td>670 </td><td>11.32 </td><td>1.33 </td></tr><tr><td>Thyroid </td><td>3.36E−23 </td><td>1076 </td><td>52.80 </td><td>2375 </td><td>40.11 </td><td>1.32 </td></tr><tr><td>Adrenal gland </td><td>4.51E−06 </td><td>362 </td><td>17.76 </td><td>801 </td><td>13.53 </td><td>1.31 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The percentages of SCCS genes that are expressed in the organ.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The percentages of non-SCCS genes that are expressed in the organ.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The fold of ‘a’ to ‘b’ Terms are listed in the descending order of the fold difference.</p></div></div><div content-id=""><div class="fn"><p>*Probability for enrichment of the expressed genes in the SCCS group.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6650534" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR010TB7" class="table-wrap-title" data-id="DSR010TB7"><div class="label -label">Table 7.</div><div class="caption"><p>Organs in which significantly (<em>P</em> &lt; 0.001) higher percentage of SCCS genes are expressed compared with non-SCCS genes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Organ </th><th rowspan="2"><em>P</em>* </th><th colspan="2">SCCS<hr /> </th><th colspan="2">Non-SCCS<hr /> </th><th rowspan="2">Fold<sup>c</sup> </th></tr><tr><th>#Expressed </th><th>Percentage<sup>a</sup> </th><th>#Expressed </th><th>Percentage<sup>b</sup> </th></tr></thead><tbody><tr><td>Amygdala </td><td>1.32E−11 </td><td>201 </td><td>9.86 </td><td>318 </td><td>5.37 </td><td>1.84 </td></tr><tr><td>Cochlea </td><td>1.24E−22 </td><td>436 </td><td>21.39 </td><td>723 </td><td>12.21 </td><td>1.75 </td></tr><tr><td>Small intestine </td><td>5.28E−03 </td><td>49 </td><td>2.40 </td><td>86 </td><td>1.45 </td><td>1.66 </td></tr><tr><td>Amnion </td><td>1.03E−04 </td><td>102 </td><td>5.00 </td><td>183 </td><td>3.09 </td><td>1.62 </td></tr><tr><td>Amniotic fluid </td><td>9.12E−07 </td><td>175 </td><td>8.59 </td><td>321 </td><td>5.42 </td><td>1.58 </td></tr><tr><td>Spinal cord </td><td>7.21E−05 </td><td>129 </td><td>6.33 </td><td>243 </td><td>4.10 </td><td>1.54 </td></tr><tr><td>Artery </td><td>9.80E−05 </td><td>133 </td><td>6.53 </td><td>254 </td><td>4.29 </td><td>1.52 </td></tr><tr><td>Cerebellum cortex </td><td>2.74E−03 </td><td>83 </td><td>4.07 </td><td>160 </td><td>2.70 </td><td>1.51 </td></tr><tr><td>Cerebellum </td><td>3.62E−08 </td><td>277 </td><td>13.59 </td><td>543 </td><td>9.17 </td><td>1.48 </td></tr><tr><td>Trabecular meshwork </td><td>1.38E−05 </td><td>199 </td><td>9.76 </td><td>399 </td><td>6.74 </td><td>1.45 </td></tr><tr><td>Frontal lobe </td><td>1.63E−28 </td><td>899 </td><td>44.11 </td><td>1803 </td><td>30.45 </td><td>1.45 </td></tr><tr><td>Hypopharynx </td><td>1.75E−06 </td><td>246 </td><td>12.07 </td><td>497 </td><td>8.39 </td><td>1.44 </td></tr><tr><td>Pituitary gland </td><td>1.67E−09 </td><td>421 </td><td>20.66 </td><td>877 </td><td>14.81 </td><td>1.39 </td></tr><tr><td>Sympathetic chain </td><td>9.65E−06 </td><td>271 </td><td>13.30 </td><td>575 </td><td>9.71 </td><td>1.37 </td></tr><tr><td>Breast </td><td>5.95E−30 </td><td>1133 </td><td>55.59 </td><td>2430 </td><td>41.04 </td><td>1.35 </td></tr><tr><td>Larynx </td><td>9.05E−13 </td><td>642 </td><td>31.50 </td><td>1385 </td><td>23.39 </td><td>1.35 </td></tr><tr><td>Tongue </td><td>4.55E−07 </td><td>377 </td><td>18.50 </td><td>816 </td><td>13.78 </td><td>1.34 </td></tr><tr><td>Smooth muscle </td><td>1.36E−05 </td><td>307 </td><td>15.06 </td><td>670 </td><td>11.32 </td><td>1.33 </td></tr><tr><td>Thyroid </td><td>3.36E−23 </td><td>1076 </td><td>52.80 </td><td>2375 </td><td>40.11 </td><td>1.32 </td></tr><tr><td>Adrenal gland </td><td>4.51E−06 </td><td>362 </td><td>17.76 </td><td>801 </td><td>13.53 </td><td>1.31 </td></tr></tbody></table></div><div content-id="DSR010TB7" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Organ </th><th rowspan="2"><em>P</em>* </th><th colspan="2">SCCS<hr /> </th><th colspan="2">Non-SCCS<hr /> </th><th rowspan="2">Fold<sup>c</sup> </th></tr><tr><th>#Expressed </th><th>Percentage<sup>a</sup> </th><th>#Expressed </th><th>Percentage<sup>b</sup> </th></tr></thead><tbody><tr><td>Amygdala </td><td>1.32E−11 </td><td>201 </td><td>9.86 </td><td>318 </td><td>5.37 </td><td>1.84 </td></tr><tr><td>Cochlea </td><td>1.24E−22 </td><td>436 </td><td>21.39 </td><td>723 </td><td>12.21 </td><td>1.75 </td></tr><tr><td>Small intestine </td><td>5.28E−03 </td><td>49 </td><td>2.40 </td><td>86 </td><td>1.45 </td><td>1.66 </td></tr><tr><td>Amnion </td><td>1.03E−04 </td><td>102 </td><td>5.00 </td><td>183 </td><td>3.09 </td><td>1.62 </td></tr><tr><td>Amniotic fluid </td><td>9.12E−07 </td><td>175 </td><td>8.59 </td><td>321 </td><td>5.42 </td><td>1.58 </td></tr><tr><td>Spinal cord </td><td>7.21E−05 </td><td>129 </td><td>6.33 </td><td>243 </td><td>4.10 </td><td>1.54 </td></tr><tr><td>Artery </td><td>9.80E−05 </td><td>133 </td><td>6.53 </td><td>254 </td><td>4.29 </td><td>1.52 </td></tr><tr><td>Cerebellum cortex </td><td>2.74E−03 </td><td>83 </td><td>4.07 </td><td>160 </td><td>2.70 </td><td>1.51 </td></tr><tr><td>Cerebellum </td><td>3.62E−08 </td><td>277 </td><td>13.59 </td><td>543 </td><td>9.17 </td><td>1.48 </td></tr><tr><td>Trabecular meshwork </td><td>1.38E−05 </td><td>199 </td><td>9.76 </td><td>399 </td><td>6.74 </td><td>1.45 </td></tr><tr><td>Frontal lobe </td><td>1.63E−28 </td><td>899 </td><td>44.11 </td><td>1803 </td><td>30.45 </td><td>1.45 </td></tr><tr><td>Hypopharynx </td><td>1.75E−06 </td><td>246 </td><td>12.07 </td><td>497 </td><td>8.39 </td><td>1.44 </td></tr><tr><td>Pituitary gland </td><td>1.67E−09 </td><td>421 </td><td>20.66 </td><td>877 </td><td>14.81 </td><td>1.39 </td></tr><tr><td>Sympathetic chain </td><td>9.65E−06 </td><td>271 </td><td>13.30 </td><td>575 </td><td>9.71 </td><td>1.37 </td></tr><tr><td>Breast </td><td>5.95E−30 </td><td>1133 </td><td>55.59 </td><td>2430 </td><td>41.04 </td><td>1.35 </td></tr><tr><td>Larynx </td><td>9.05E−13 </td><td>642 </td><td>31.50 </td><td>1385 </td><td>23.39 </td><td>1.35 </td></tr><tr><td>Tongue </td><td>4.55E−07 </td><td>377 </td><td>18.50 </td><td>816 </td><td>13.78 </td><td>1.34 </td></tr><tr><td>Smooth muscle </td><td>1.36E−05 </td><td>307 </td><td>15.06 </td><td>670 </td><td>11.32 </td><td>1.33 </td></tr><tr><td>Thyroid </td><td>3.36E−23 </td><td>1076 </td><td>52.80 </td><td>2375 </td><td>40.11 </td><td>1.32 </td></tr><tr><td>Adrenal gland </td><td>4.51E−06 </td><td>362 </td><td>17.76 </td><td>801 </td><td>13.53 </td><td>1.31 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>The percentages of SCCS genes that are expressed in the organ.</p></div></div><div content-id=""><div class="fn"><p><sup>b</sup>The percentages of non-SCCS genes that are expressed in the organ.</p></div></div><div content-id=""><div class="fn"><p><sup>c</sup>The fold of ‘a’ to ‘b’ Terms are listed in the descending order of the fold difference.</p></div></div><div content-id=""><div class="fn"><p>*Probability for enrichment of the expressed genes in the SCCS group.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6650534" target="_blank">View Large</a></div></div><p>Seven organs (amygdala, spinal cord, cerebellum cortex, cerebellum, frontal lobe, pituitary grand, and sympathetic chain) in Table <a reveal-id="DSR010TB7" data-open="DSR010TB7" class="link link-reveal link-table xref-fig">7</a> are related with the nervous system and six organs (cochlea, trabecular meshwork, hypopharynx, larynx, tongue, and thyroid) are associated with head and neck.</p>                            <h3 scrollto-destination=6650536 id="6650536" class="section-title" data-legacy-id=s3g>3.7. Preferred codons, GC content, and codon degeneracy of SCCSs</h3>
<p>Codon usage biases towards optimum codons are known to suppress synonymous substitution. Optimum codons reflect the composition of the genomic tRNA pool and are advantageous for translation efficiency or accuracy. As the approximate index of optimum codons, we used preferred codons, or most frequently used codon for an amino acid, and evaluated the preferred codon fraction in SCCSs. In SCCS regions, 28 188 of 64 102 codons are preferred codons and in non-SCCS regions, 2 961 482 of 6 855 505 codons are preferred codons. The ratio of preferred codons in SCCS and non-SCCS regions are 0.440 and 0.432, respectively. The difference is significant at the 0.05 significance level (<em>P</em> = 0.013) but the difference of the ratios is merely 0.008. We also observed that the ratio of preferred codon decreases as the length of SCCS increases (Fig. <a reveal-id="DSR010F3" data-open="DSR010F3" class="link link-reveal link-table xref-fig">3</a>). Judging from these results, SCCSs are unlikely to be retained solely by codon preference. </p>                        <a id="6650538" scrollto-destination="6650538"></a>
<div id="DSR010F3" data-id="DSR010F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/m_dsr01003.jpeg?Expires=2147483647&Signature=WDtAlHdeiVlclvZAFoC61w~8KprsAs5PTLDEewus~hbovJEEsBfyzd5vNL7ljWsskpvVp18~hGpQiyaX5X46ZoFxLq9EW5As11oIYfe2FZI4QH9W09y~LbNQN8cgBp8qSBDLQA0rwdcz7JhsNSIrrVxty-Tu4ARxWhnRZqM~39zga~jDeEaNX8pdh1ttlZcCP2eNjb0UnxYUXE-S5HNwh1avFDlt6L7xSg9rW8b5TlWSbd2XNAELzdNiLOZ-h~2pemOTbev20r~imVHphUJYtYB1TvwVENLdMKHOQsPGFUiZOjDl647qspuWkWe0AqgZiaJx4iGDIZrTnBY4LJPZzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The ratio of preferred codons in SCCSs. X-axis represents the length of SCCSs, and Y-axis represents the ratio of preferred codon of the sequences. Classes whose sample size &lt; 20 were combined. Error bars represent 1 SE." path-from-xml="dsr01003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6650538/dsr01003.jpeg" path-from-xml="dsr01003.jpeg" target="_blank">View large</a><a section="6650538" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/dsr01003.jpeg?Expires=2147483647&Signature=e6sVtXsSPV~dLX-mJo3QsOqHT3wA8JVHGctf6ImksJGoAT51XbAMnaiNiJKpF48Y52AyB0LjtsPSpxmplaFkMl~GPuUUXkVppa~6~2U2SFDUoh2ueqBMy7gW2t8pn8Axo-1T9kriift016awqqjGZRX24WeN3GcUZatndZffV9cQ7ABgJzg~Gw7gPKmTZVBicKhjZwWSTGuU3UZA9xaqMV8OLimgebAiJL9nkeB~G4J9ATfmGeaUrh~9Bs6xtMtozSrE5aRLbKhbMjIu4l8FHydcUQ6LzizF2lgthphl-u5b8EsFiYI4Dvv9b6Flb51tR5csPhhdOktOHXXG~KD~GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6650538&ar=444849&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The ratio of preferred codons in SCCSs. <em>X</em>-axis represents the length of SCCSs, and <em>Y</em>-axis represents the ratio of preferred codon of the sequences. Classes whose sample size &lt; 20 were combined. Error bars represent 1 SE.</p></div></div><div content-id="DSR010F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/m_dsr01003.jpeg?Expires=2147483647&Signature=WDtAlHdeiVlclvZAFoC61w~8KprsAs5PTLDEewus~hbovJEEsBfyzd5vNL7ljWsskpvVp18~hGpQiyaX5X46ZoFxLq9EW5As11oIYfe2FZI4QH9W09y~LbNQN8cgBp8qSBDLQA0rwdcz7JhsNSIrrVxty-Tu4ARxWhnRZqM~39zga~jDeEaNX8pdh1ttlZcCP2eNjb0UnxYUXE-S5HNwh1avFDlt6L7xSg9rW8b5TlWSbd2XNAELzdNiLOZ-h~2pemOTbev20r~imVHphUJYtYB1TvwVENLdMKHOQsPGFUiZOjDl647qspuWkWe0AqgZiaJx4iGDIZrTnBY4LJPZzA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The ratio of preferred codons in SCCSs. X-axis represents the length of SCCSs, and Y-axis represents the ratio of preferred codon of the sequences. Classes whose sample size &lt; 20 were combined. Error bars represent 1 SE." path-from-xml="dsr01003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6650538/dsr01003.jpeg" path-from-xml="dsr01003.jpeg" target="_blank">View large</a><a section="6650538" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/dsr01003.jpeg?Expires=2147483647&Signature=e6sVtXsSPV~dLX-mJo3QsOqHT3wA8JVHGctf6ImksJGoAT51XbAMnaiNiJKpF48Y52AyB0LjtsPSpxmplaFkMl~GPuUUXkVppa~6~2U2SFDUoh2ueqBMy7gW2t8pn8Axo-1T9kriift016awqqjGZRX24WeN3GcUZatndZffV9cQ7ABgJzg~Gw7gPKmTZVBicKhjZwWSTGuU3UZA9xaqMV8OLimgebAiJL9nkeB~G4J9ATfmGeaUrh~9Bs6xtMtozSrE5aRLbKhbMjIu4l8FHydcUQ6LzizF2lgthphl-u5b8EsFiYI4Dvv9b6Flb51tR5csPhhdOktOHXXG~KD~GA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6650538&ar=444849&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The ratio of preferred codons in SCCSs. <em>X</em>-axis represents the length of SCCSs, and <em>Y</em>-axis represents the ratio of preferred codon of the sequences. Classes whose sample size &lt; 20 were combined. Error bars represent 1 SE.</p></div></div><p>In mammals, of the influence GC content on nucleotide change as a result of CpG hyper mutability. We investigated GC content of SCCSs. GC content in the first (GC1), second (GC2), and third position (GC3) of codons show different patterns along the sequence length (Fig. <a reveal-id="DSR010F4" data-open="DSR010F4" class="link link-reveal link-table xref-fig">4</a>). GC1 is mostly constant but GC2 increases while GC3 decreases as the length of SCCS increases. Because mammalian genomes prefer GC-ending codons, the decrease of GC3 corresponds to the decrease of preferred codons. The decrease of GC3 seems to be complementary with the increase of GC2 because GC content as a whole is constant (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). </p>                        <a id="6650540" scrollto-destination="6650540"></a>
<div id="DSR010F4" data-id="DSR010F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/m_dsr01004.jpeg?Expires=2147483647&Signature=pB3z5Jx4-kkfcibL49KyZwbuzSMI8TR7h5uMXv7WqE9gh6uqc4F9Mcq~4413bAWijmBXn0eVVEpCZUM2uDwQUE85GbufMYr4a5GEDv~uZ5XuVT94wGQ7DSNggXYeOwAjdoBFccv07CQ4CX-h8~DLiOdB7OOb9suWRv5i-drB-ChUkLSh34RnSlPoG--HhqXx9DEyDaGaO~zqyhwpcPryqJVYZ3~uHBxOrUXUMHtMTttLqS7RAw5wHyRZ7KYLwrS44XC74Rg8IUWsDdZjuw-lh5UIHxyGAWA-3sQ0Vv5Z7q4liWLtRcjC4p73nMFHd7o2gpa~WwZgwFe0GQ9V~X23tQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GC content of the first (GC1), the second (GC2), and the third (GC3) position of codons in SCCSs: (A) GC1, (B) GC2, (C) GC3. X-axis represents the length of SCCSs, and Y-axis represents GC content of the sequences. Classes whose sample size &lt; 20 were combined. Error bars represent 1 SE." path-from-xml="dsr01004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6650540/dsr01004.jpeg" path-from-xml="dsr01004.jpeg" target="_blank">View large</a><a section="6650540" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/dsr01004.jpeg?Expires=2147483647&Signature=natxPEq17p4PLbQ-yvPmSqdJzvGcWa1GncHst42xKpYYtufDyAaWJioHZ8CzYGGWnreKLn5mMh-v75-lLXs0GAg9ilRq5Da3-WWBTVwZeBIIuTVb4Nyv-WPaPziUwHTEoz0-7zAvAYotFCWZ7Jn0f54fTFOv9iUKFUziSGD8j00kNcwjuDR9SkUdvPkpZ2zkc6ixHgjdTKiz9L0Yhvo0o8ZkpE~XhMxBivbrzGf-o~ifo7aKu3n5tDFM1VoKrPuu-t40abXPB6EFieTWPnq3gBMznIRI7DRkDfphZHvp9inEXycrHo881sidEQqmH9t6UZwL7fShtX6O-05AM~RzZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6650540&ar=444849&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GC content of the first (GC1), the second (GC2), and the third (GC3) position of codons in SCCSs: (<strong>A</strong>) GC1, (<strong>B</strong>) GC2, (<strong>C</strong>) GC3. <em>X</em>-axis represents the length of SCCSs, and <em>Y</em>-axis represents GC content of the sequences. Classes whose sample size &lt; 20 were combined. Error bars represent 1 SE.</p></div></div><div content-id="DSR010F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/m_dsr01004.jpeg?Expires=2147483647&Signature=pB3z5Jx4-kkfcibL49KyZwbuzSMI8TR7h5uMXv7WqE9gh6uqc4F9Mcq~4413bAWijmBXn0eVVEpCZUM2uDwQUE85GbufMYr4a5GEDv~uZ5XuVT94wGQ7DSNggXYeOwAjdoBFccv07CQ4CX-h8~DLiOdB7OOb9suWRv5i-drB-ChUkLSh34RnSlPoG--HhqXx9DEyDaGaO~zqyhwpcPryqJVYZ3~uHBxOrUXUMHtMTttLqS7RAw5wHyRZ7KYLwrS44XC74Rg8IUWsDdZjuw-lh5UIHxyGAWA-3sQ0Vv5Z7q4liWLtRcjC4p73nMFHd7o2gpa~WwZgwFe0GQ9V~X23tQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GC content of the first (GC1), the second (GC2), and the third (GC3) position of codons in SCCSs: (A) GC1, (B) GC2, (C) GC3. X-axis represents the length of SCCSs, and Y-axis represents GC content of the sequences. Classes whose sample size &lt; 20 were combined. Error bars represent 1 SE." path-from-xml="dsr01004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6650540/dsr01004.jpeg" path-from-xml="dsr01004.jpeg" target="_blank">View large</a><a section="6650540" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/dsr01004.jpeg?Expires=2147483647&Signature=natxPEq17p4PLbQ-yvPmSqdJzvGcWa1GncHst42xKpYYtufDyAaWJioHZ8CzYGGWnreKLn5mMh-v75-lLXs0GAg9ilRq5Da3-WWBTVwZeBIIuTVb4Nyv-WPaPziUwHTEoz0-7zAvAYotFCWZ7Jn0f54fTFOv9iUKFUziSGD8j00kNcwjuDR9SkUdvPkpZ2zkc6ixHgjdTKiz9L0Yhvo0o8ZkpE~XhMxBivbrzGf-o~ifo7aKu3n5tDFM1VoKrPuu-t40abXPB6EFieTWPnq3gBMznIRI7DRkDfphZHvp9inEXycrHo881sidEQqmH9t6UZwL7fShtX6O-05AM~RzZg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6650540&ar=444849&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GC content of the first (GC1), the second (GC2), and the third (GC3) position of codons in SCCSs: (<strong>A</strong>) GC1, (<strong>B</strong>) GC2, (<strong>C</strong>) GC3. <em>X</em>-axis represents the length of SCCSs, and <em>Y</em>-axis represents GC content of the sequences. Classes whose sample size &lt; 20 were combined. Error bars represent 1 SE.</p></div></div><p>Conservation of SCCSs may occur by chance in a region where amino acid constraint is strong and codon degeneracy is low. We investigated codon degeneracy of SCCSs to examine this possibility. The average degeneracy is between three and four and increases as the sequence length increases (Fig. <a reveal-id="DSR010F5" data-open="DSR010F5" class="link link-reveal link-table xref-fig">5</a>). This result suggests that even if the first and second positions of codons are restricted, the third position has enough freedom to change. Therefore, it is unlikely that SCCSs are conserved because of the amino acid constraint combined with the low degeneracy of codons. </p>                        <a id="6650542" scrollto-destination="6650542"></a>
<div id="DSR010F5" data-id="DSR010F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/m_dsr01005.jpeg?Expires=2147483647&Signature=xXWGARuPqnoJQnJjxS4syH0m-gT5lF~bNfpyaLA-2IVy~S3CExROKxshHLPbOFJLLwoBHrQPO-mOphtOMQY0LpeNuCCh~sfPTtmXTuv7~xoshgFP~LaAWk1ahEBWDpwqDzLrLRvPSV-kzg-xJctY7zfemSKVNHeRn3XGHHF~kDT3mpTRVVDka~TsS9kweGMt3Hrp~GebMPg-ZFlG1lrZ3g6ZI9zFjnk3pnGBLEKLFe4QQ0bZtBaV9MkIe6r0OZDXwt-kQaaMWjLPUj1oha75R9XE7QeGcS6dt93LJOGP3pUEQsMLwiT7ViKC1H6ruUKt1gUh3mu-uzGxdVHm~Jf0tQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Codon degeneracy of SCCSs. X-axis represents the length of SCCSs and Y-axis represents the averaged codon degeneracy of the sequences. Classes whose sample size &lt; 20 were combined. Error bars represent 1 SE." path-from-xml="dsr01005.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6650542/dsr01005.jpeg" path-from-xml="dsr01005.jpeg" target="_blank">View large</a><a section="6650542" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/dsr01005.jpeg?Expires=2147483647&Signature=sF~0bC8ycQzXAva0xxSt0k-T-bPYKdWZc8FPV7e-3Z~rR4TS0cyURVuDaqPRnewHP7dRGr7J2L3R7gTKlsp8G2Pl8JFRIGU8c11DYEdpVw6sbquEaNhIiYSyZZDQOxZyncFzVLswk18aDd0H~nOz~JHCQPjlXRvgU2PnZ~ErArjHpn3d-0SO-a0CzpyNPnbjlk8iY4JCjtxdWI4lTCv2YLkntQshxP-TYwi-BbpdLnduVKpl7rxLxW875NCEldQXf-toNPDt5CyJZSiWm7gYWLzc2kXBkPbTXhYR97hjlahnY8DbbBeYnJViN3HZhpu0SKwRCOhgTNYvrFduBJUooQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6650542&ar=444849&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01005.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Codon degeneracy of SCCSs. <em>X</em>-axis represents the length of SCCSs and <em>Y</em>-axis represents the averaged codon degeneracy of the sequences. Classes whose sample size &lt; 20 were combined. Error bars represent 1 SE.</p></div></div><div content-id="DSR010F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/m_dsr01005.jpeg?Expires=2147483647&Signature=xXWGARuPqnoJQnJjxS4syH0m-gT5lF~bNfpyaLA-2IVy~S3CExROKxshHLPbOFJLLwoBHrQPO-mOphtOMQY0LpeNuCCh~sfPTtmXTuv7~xoshgFP~LaAWk1ahEBWDpwqDzLrLRvPSV-kzg-xJctY7zfemSKVNHeRn3XGHHF~kDT3mpTRVVDka~TsS9kweGMt3Hrp~GebMPg-ZFlG1lrZ3g6ZI9zFjnk3pnGBLEKLFe4QQ0bZtBaV9MkIe6r0OZDXwt-kQaaMWjLPUj1oha75R9XE7QeGcS6dt93LJOGP3pUEQsMLwiT7ViKC1H6ruUKt1gUh3mu-uzGxdVHm~Jf0tQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Codon degeneracy of SCCSs. X-axis represents the length of SCCSs and Y-axis represents the averaged codon degeneracy of the sequences. Classes whose sample size &lt; 20 were combined. Error bars represent 1 SE." path-from-xml="dsr01005.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6650542/dsr01005.jpeg" path-from-xml="dsr01005.jpeg" target="_blank">View large</a><a section="6650542" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/dsr01005.jpeg?Expires=2147483647&Signature=sF~0bC8ycQzXAva0xxSt0k-T-bPYKdWZc8FPV7e-3Z~rR4TS0cyURVuDaqPRnewHP7dRGr7J2L3R7gTKlsp8G2Pl8JFRIGU8c11DYEdpVw6sbquEaNhIiYSyZZDQOxZyncFzVLswk18aDd0H~nOz~JHCQPjlXRvgU2PnZ~ErArjHpn3d-0SO-a0CzpyNPnbjlk8iY4JCjtxdWI4lTCv2YLkntQshxP-TYwi-BbpdLnduVKpl7rxLxW875NCEldQXf-toNPDt5CyJZSiWm7gYWLzc2kXBkPbTXhYR97hjlahnY8DbbBeYnJViN3HZhpu0SKwRCOhgTNYvrFduBJUooQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6650542&ar=444849&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr01005.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Codon degeneracy of SCCSs. <em>X</em>-axis represents the length of SCCSs and <em>Y</em>-axis represents the averaged codon degeneracy of the sequences. Classes whose sample size &lt; 20 were combined. Error bars represent 1 SE.</p></div></div>                            <h2 scrollto-destination=6650543 id="6650543" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>GO terms, InterPro codes, and KEGG pathways enriched with SCCS genes show a strong commitment to the developmental process, transcriptional regulation, and the neurons. Genes associated with transcriptional regulation or the neurons are known to have a low synonymous substitution ratio. This phenomenon is discussed in relation with codon biases to improve translational efficiency or accuracy. However, our analysis on the preferred codons in SCCSs suggests that codon preference is not likely the major factor influencing on the conservation of SCCSs.</p><p>Analyses on the ratio of preferred codons, GC content, and codon degeneracy enlighten the characteristics of SCCSs. The ratio of preferred codons decreases as the SCCS length increases. Drummond and Wilke<sup><a reveal-id="DSR010C36" data-open="DSR010C36" class="link link-ref link-reveal xref-bibr">36</a></sup> investigated the correlation between synonymous substitution rate (dS) and fraction of optimum (Fop) codons, and detected negative correlations between dS and Fop in rodents (mouse and rat) and positive correlation in human–dog comparison. If the SCCSs have the same trend as the rodents of the previous study, the ratio of preferred codons in SCCSs should be high; however, our result is not. There was no factor that would lower nucleotide substitution in GC content and codon degeneracy.</p><p>Methodological difference is that our research focused on local and complete conservation of nucleotides instead of the dS in the entire region of a gene and that we investigated conservation among the six mammalian species instead of a pair-wise comparison. The difference of results may suggest that factors underlying local and strong conservation such as SCCS differ from the factors working on the gene-wide conservation.</p><p>Makalowski <em>et al</em>.<sup><a reveal-id="DSR010C37" data-open="DSR010C37" class="link link-ref link-reveal xref-bibr">37</a></sup> showed a correlation between synonymous substitution rate (dS) and non-synonymous substitution rate (dN). Such correlations may occur when the constraint on a certain nucleotide sequence is so strong that dN is also lowered.</p><p>The usage of relatively rare codons and strong local conservation of SCCSs may be preferable as regulatory signals. The fraction of SCCSs in the coding region of the 10 790 genes is 0.94%. This fraction is so small that it would not have an influence on conventional evolutionary analysis. Although the fraction is small, or because the fraction is small, SCCSs may have potential as regulatory elements.</p>                            <h2 scrollto-destination=6650549 id="6650549" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6650551 id="6650551" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This study was supported partly by Grant-in-Aid for scientific research from the Ministry of Education, Culture, Sports, Science, and Technology of Japan to N.S.</p>                            <h2 scrollto-destination=6650553 id="6650553" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Drs Kenta Sumiyama, Kiyoshi Ezawa, and Hiroshi Akashi for their insightful suggestion and discussion.</p>                            <h2 scrollto-destination=6650555 id="6650555" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR010C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kimura</div> <div class="given-names">M.</div></div>. </div>, <div class="source">The Neutral Theory of Molecular Evolution</div> , <div class="year">1983</div><div class="publisher-loc">Cambridge</div><div class="publisher-name">Cambridge University Press</div></div></div></div></div><div content-id="DSR010C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div>, <div class="source">Molecular Evolutionary Genetics</div> , <div class="year">1987</div><div class="publisher-loc">New York</div><div class="publisher-name">Columbia Univ. Press</div></div></div></div></div><div content-id="DSR010C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Correlation between the abundance of yeast transfer RNAs and the occurrence of the respective codons in protein genes. Differences in synonymous codon choice patterns of yeast and Escherichia coli with reference to the abundance of isoaccepting transfer RNAs</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1982</div>, vol. <div class="volume">158</div> (pg. <div class="fpage">573</div>-<div class="lpage">97</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0022-2836(82)90250-9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0022-2836(82)90250-9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20the%20abundance%20of%20yeast%20transfer%20RNAs%20and%20the%20occurrence%20of%20the%20respective%20codons%20in%20protein%20genes.%20Differences%20in%20synonymous%20codon%20choice%20patterns%20of%20yeast%20and%20Escherichia%20coli%20with%20reference%20to%20the%20abundance%20of%20isoaccepting%20transfer%20RNAs&amp;author=T.%20Ikemura&amp;publication_year=1982&amp;journal=J.%20Mol.%20Biol.&amp;volume=158&amp;pages=573-97" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(82)90250-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(82)90250-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(82)90250-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6750137" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Correlation between the abundance of Escherichia coli transfer RNAs and the occurrence of the respective codons in its protein genes: a proposal for a synonymous codon choice that is optimal for the <em>E. coli</em> translational system</div>, <div class="source">J. Mol. Biol.</div> , <div class="year">1981</div>, vol. <div class="volume">151</div> (pg. <div class="fpage">389</div>-<div class="lpage">409</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0022-2836(81)90003-6" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0022-2836(81)90003-6</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Correlation%20between%20the%20abundance%20of%20Escherichia%20coli%20transfer%20RNAs%20and%20the%20occurrence%20of%20the%20respective%20codons%20in%20its%20protein%20genes%3A%20a%20proposal%20for%20a%20synonymous%20codon%20choice%20that%20is%20optimal%20for%20the%20E.%20coli%20translational%20system&amp;author=T.%20Ikemura&amp;publication_year=1981&amp;journal=J.%20Mol.%20Biol.&amp;volume=151&amp;pages=389-409" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(81)90003-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(81)90003-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(81)90003-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/6175758" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanaya</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Yamada</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Kinouchi</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Kudo</div> <div class="given-names">Y.,</div></div>,  <div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Codon usage and tRNA genes in eukaryotes: correlation of codon usage diversity with translation efficiency and with CG-dinucleotide usage as assessed by multivariate analysis</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">2001</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">290</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s002390010219" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s002390010219</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20tRNA%20genes%20in%20eukaryotes%3A%20correlation%20of%20codon%20usage%20diversity%20with%20translation%20efficiency%20and%20with%20CG-dinucleotide%20usage%20as%20assessed%20by%20multivariate%20analysis&amp;author=S.%20Kanaya&amp;author=Y.%20Yamada&amp;author=M.%20Kinouchi&amp;author=Y.%2C%20Kudo&amp;author=T.%20Ikemura&amp;publication_year=2001&amp;journal=J.%20Mol.%20Evol.&amp;volume=53&amp;pages=290-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s002390010219" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs002390010219" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs002390010219"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11675589" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akashi</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Synonymous codon usage in Drosophila melanogaster: natural selection and translational accuracy</div>, <div class="source">Genetics</div> , <div class="year">1994</div>, vol. <div class="volume">136</div> (pg. <div class="fpage">927</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20codon%20usage%20in%20Drosophila%20melanogaster%3A%20natural%20selection%20and%20translational%20accuracy&amp;author=H.%20Akashi&amp;publication_year=1994&amp;journal=Genetics&amp;volume=136&amp;pages=927-35" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8005445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kurland</div> <div class="given-names">C.G.</div></div>. </div><div class="article-title">Codon bias and gene expression</div>, <div class="source">FEBS Lett.</div> , <div class="year">1991</div>, vol. <div class="volume">285</div> (pg. <div class="fpage">165</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/0014-5793(91)80797-7" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/0014-5793(91)80797-7</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20bias%20and%20gene%20expression&amp;author=C.G.%20Kurland&amp;publication_year=1991&amp;journal=FEBS%20Lett.&amp;volume=285&amp;pages=165-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0014-5793(91)80797-7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0014-5793(91)80797-7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0014-5793(91)80797-7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1855585" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hershberg</div> <div class="given-names">R.,</div></div>,  <div class="name"><div class="surname">Petrov</div> <div class="given-names">D.A.</div></div>. </div><div class="article-title">Selection on codon bias</div>, <div class="source">Annu. Rev. Genet.</div> , <div class="year">2008</div>, vol. <div class="volume">42</div> (pg. <div class="fpage">287</div>-<div class="lpage">99</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev.genet.42.110807.091442" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1146/annurev.genet.42.110807.091442</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Selection%20on%20codon%20bias&amp;author=R.%2C%20Hershberg&amp;author=D.A.%20Petrov&amp;publication_year=2008&amp;journal=Annu.%20Rev.%20Genet.&amp;volume=42&amp;pages=287-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.genet.42.110807.091442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.genet.42.110807.091442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.genet.42.110807.091442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18983258" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ikemura</div> <div class="given-names">T.</div></div>. </div><div class="article-title">Codon usage and tRNA content in unicellular and multicellular organisms</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1985</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">13</div>-<div class="lpage">34</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Codon%20usage%20and%20tRNA%20content%20in%20unicellular%20and%20multicellular%20organisms&amp;author=T.%20Ikemura&amp;publication_year=1985&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=2&amp;pages=13-34" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3916708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.,</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">W.H.</div></div>. </div><div class="article-title">The rate of synonymous substitution in enterobacterial genes is inversely related to codon usage bias</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1987</div>, vol. <div class="volume">4</div> (pg. <div class="fpage">222</div>-<div class="lpage">30</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20rate%20of%20synonymous%20substitution%20in%20enterobacterial%20genes%20is%20inversely%20related%20to%20codon%20usage%20bias&amp;author=P.M.%2C%20Sharp&amp;author=W.H.%20Li&amp;publication_year=1987&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=4&amp;pages=222-30" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3328816" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Akashi</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Translational selection and yeast proteome evolution</div>, <div class="source">Genetics</div> , <div class="year">2003</div>, vol. <div class="volume">164</div> (pg. <div class="fpage">1291</div>-<div class="lpage">303</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Translational%20selection%20and%20yeast%20proteome%20evolution&amp;author=H.%20Akashi&amp;publication_year=2003&amp;journal=Genetics&amp;volume=164&amp;pages=1291-303" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12930740" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Eyre-Walker</div> <div class="given-names">A.C.</div></div>. </div><div class="article-title">An analysis of codon usage in mammals: selection or mutation bias?</div>, <div class="source">J. Mol. Evol.</div> , <div class="year">1991</div>, vol. <div class="volume">33</div> (pg. <div class="fpage">442</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/BF02103136" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/BF02103136</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20analysis%20of%20codon%20usage%20in%20mammals%3A%20selection%20or%20mutation%20bias%3F&amp;author=A.C.%20Eyre-Walker&amp;publication_year=1991&amp;journal=J.%20Mol.%20Evol.&amp;volume=33&amp;pages=442-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF02103136" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF02103136" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF02103136"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1960741" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharp</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Averof</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Lloyd</div> <div class="given-names">A.T.</div></div>,  <div class="name"><div class="surname">Matassi</div> <div class="given-names">G.,</div></div>,  <div class="name"><div class="surname">Peden</div> <div class="given-names">J.F.</div></div>. </div><div class="article-title">DNA sequence evolution: the sounds of silence</div>, <div class="source">Philos. Trans. R Soc. Lond. B Biol. Sci.</div> , <div class="year">1995</div>, vol. <div class="volume">349</div> (pg. <div class="fpage">241</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1098/rstb.1995.0108" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1098/rstb.1995.0108</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20sequence%20evolution%3A%20the%20sounds%20of%20silence&amp;author=P.M.%20Sharp&amp;author=M.%20Averof&amp;author=A.T.%20Lloyd&amp;author=G.%2C%20Matassi&amp;author=J.F.%20Peden&amp;publication_year=1995&amp;journal=Philos.%20Trans.%20R%20Soc.%20Lond.%20B%20Biol.%20Sci.&amp;volume=349&amp;pages=241-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.1995.0108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.1995.0108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.1995.0108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8577834" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Reed</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Initial splice-site recognition and pairing during pre-mRNA splicing</div>, <div class="source">Curr. Opin. Genet. Dev.</div> , <div class="year">1996</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">215</div>-<div class="lpage">20</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0959-437X(96)80053-0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0959-437X(96)80053-0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Initial%20splice-site%20recognition%20and%20pairing%20during%20pre-mRNA%20splicing&amp;author=R.%20Reed&amp;publication_year=1996&amp;journal=Curr.%20Opin.%20Genet.%20Dev.&amp;volume=6&amp;pages=215-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0959-437X(96)80053-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0959-437X(96)80053-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0959-437X(96)80053-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8722179" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Blencowe</div> <div class="given-names">B.J.</div></div>. </div><div class="article-title">Exonic splicing enhancers: mechanism of action, diversity and role in human genetic diseases</div>, <div class="source">Trends Biochem. Sci</div> , <div class="year">2000</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">106</div>-<div class="lpage">10</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0968-0004(00)01549-8" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0968-0004(00)01549-8</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exonic%20splicing%20enhancers%3A%20mechanism%20of%20action%2C%20diversity%20and%20role%20in%20human%20genetic%20diseases&amp;author=B.J.%20Blencowe&amp;publication_year=2000&amp;journal=Trends%20Biochem.%20Sci&amp;volume=25&amp;pages=106-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0968-0004(00)01549-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0968-0004(00)01549-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0968-0004(00)01549-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10694877" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Takahashi</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Effect of exonic splicing regulation on synonymous codon usage in alternatively spliced exons of Dscam</div>, <div class="source">BMC Evol. Biol</div> , <div class="year">2009</div>, vol. <div class="volume">9</div> pg. <div class="fpage">214</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2148-9-214" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2148-9-214</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Effect%20of%20exonic%20splicing%20regulation%20on%20synonymous%20codon%20usage%20in%20alternatively%20spliced%20exons%20of%20Dscam&amp;author=A.%20Takahashi&amp;publication_year=2009&amp;journal=BMC%20Evol.%20Biol&amp;volume=9&amp;pages=214" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-9-214" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-9-214" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-9-214"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19709440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parmley</div> <div class="given-names">J.L.,</div></div>,  <div class="name"><div class="surname">Hurst</div> <div class="given-names">L.D.</div></div>. </div><div class="article-title">Exonic splicing regulatory elements skew synonymous codon usage near intron-exon boundaries in mammals</div>, <div class="source">Mol. Biol. Evol</div> , <div class="year">2007</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">1600</div>-<div class="lpage">3</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/molbev/msm104" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/molbev/msm104</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Exonic%20splicing%20regulatory%20elements%20skew%20synonymous%20codon%20usage%20near%20intron-exon%20boundaries%20in%20mammals&amp;author=J.L.%2C%20Parmley&amp;author=L.D.%20Hurst&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=1600-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17525472" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bejerano</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Pheasant</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Makunin</div> <div class="given-names">I.</div></div>, et al. </div><div class="article-title">Ultraconserved elements in the human genome</div>, <div class="source">Science</div> , <div class="year">2004</div>, vol. <div class="volume">304</div> (pg. <div class="fpage">1321</div>-<div class="lpage">5</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1098119" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1098119</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultraconserved%20elements%20in%20the%20human%20genome&amp;author=G.%20Bejerano&amp;author=M.%20Pheasant&amp;author=I.%20Makunin&amp;publication_year=2004&amp;journal=Science&amp;volume=304&amp;pages=1321-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1098119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1098119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1098119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15131266" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schattner</div> <div class="given-names">P.,</div></div>,  <div class="name"><div class="surname">Diekhans</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Regions of extreme synonymous codon selection in mammalian genes</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">1700</div>-<div class="lpage">10</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkl095" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkl095</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regions%20of%20extreme%20synonymous%20codon%20selection%20in%20mammalian%20genes&amp;author=P.%2C%20Schattner&amp;author=M.%20Diekhans&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res&amp;volume=34&amp;pages=1700-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl095" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl095" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl095"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16556911" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lareau</div> <div class="given-names">L.F.</div></div>,  <div class="name"><div class="surname">Inada</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">R.E.</div></div>,  <div class="name"><div class="surname">Wengrod</div> <div class="given-names">J.C.,</div></div>,  <div class="name"><div class="surname">Brenner</div> <div class="given-names">S.E.</div></div>. </div><div class="article-title">Unproductive splicing of SR genes associated with highly conserved and ultraconserved DNA elements</div>, <div class="source">Nature</div> , <div class="year">2007</div>, vol. <div class="volume">446</div> (pg. <div class="fpage">926</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature05676" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature05676</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Unproductive%20splicing%20of%20SR%20genes%20associated%20with%20highly%20conserved%20and%20ultraconserved%20DNA%20elements&amp;author=L.F.%20Lareau&amp;author=M.%20Inada&amp;author=R.E.%20Green&amp;author=J.C.%2C%20Wengrod&amp;author=S.E.%20Brenner&amp;publication_year=2007&amp;journal=Nature&amp;volume=446&amp;pages=926-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05676" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05676" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05676"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17361132" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lin</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">H.,</div></div>,  <div class="name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Ultraconserved coding regions outside the homeobox of mammalian Hox genes</div>, <div class="source">BMC Evol. Biol</div> , <div class="year">2008</div>, vol. <div class="volume">8</div> pg. <div class="fpage">260</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2148-8-260" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2148-8-260</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultraconserved%20coding%20regions%20outside%20the%20homeobox%20of%20mammalian%20Hox%20genes&amp;author=Z.%20Lin&amp;author=H.%2C%20Ma&amp;author=M.%20Nei&amp;publication_year=2008&amp;journal=BMC%20Evol.%20Biol&amp;volume=8&amp;pages=260" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2148-8-260" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2148-8-260" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2148-8-260"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18816392" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lampe</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Samad</div> <div class="given-names">O.A.</div></div>,  <div class="name"><div class="surname">Guiguen</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">An ultraconserved Hox-Pbx responsive element resides in the coding sequence of Hoxa2 and is active in rhombomere 4</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">2008</div>, vol. <div class="volume">36</div> (pg. <div class="fpage">3214</div>-<div class="lpage">25</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkn148" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkn148</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20ultraconserved%20Hox-Pbx%20responsive%20element%20resides%20in%20the%20coding%20sequence%20of%20Hoxa2%20and%20is%20active%20in%20rhombomere%204&amp;author=X.%20Lampe&amp;author=O.A.%20Samad&amp;author=A.%20Guiguen&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res&amp;volume=36&amp;pages=3214-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn148" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn148" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn148"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18417536" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Licatalosi</div> <div class="given-names">D.D.,</div></div>,  <div class="name"><div class="surname">Darnell</div> <div class="given-names">R.B.</div></div>. </div><div class="article-title">RNA processing and its regulation: global insights into biological networks</div>, <div class="source">Nat. Rev. Genet</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">75</div>-<div class="lpage">87</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrg2673" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nrg2673</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20processing%20and%20its%20regulation%3A%20global%20insights%20into%20biological%20networks&amp;author=D.D.%2C%20Licatalosi&amp;author=R.B.%20Darnell&amp;publication_year=2010&amp;journal=Nat.%20Rev.%20Genet&amp;volume=11&amp;pages=75-87" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2673" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20019688" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bhalla</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Rosenthal</div> <div class="given-names">J.J.</div></div>,  <div class="name"><div class="surname">Holmgren</div> <div class="given-names">M.,</div></div>,  <div class="name"><div class="surname">Reenan</div> <div class="given-names">R.</div></div>. </div><div class="article-title">Control of human potassium channel inactivation by editing of a small mRNA hairpin</div>, <div class="source">Nat. Struct. Mol. Biol</div> , <div class="year">2004</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">950</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nsmb825" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nsmb825</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Control%20of%20human%20potassium%20channel%20inactivation%20by%20editing%20of%20a%20small%20mRNA%20hairpin&amp;author=T.%20Bhalla&amp;author=J.J.%20Rosenthal&amp;author=M.%2C%20Holmgren&amp;author=R.%20Reenan&amp;publication_year=2004&amp;journal=Nat.%20Struct.%20Mol.%20Biol&amp;volume=11&amp;pages=950-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nsmb825" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnsmb825" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnsmb825"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15361858" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Delgado</div> <div class="given-names">M.D.</div></div>,  <div class="name"><div class="surname">Gutierrez</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Richard</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Cuadrado</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Moreau-Gachelin</div> <div class="given-names">F.,</div></div>,  <div class="name"><div class="surname">Leon</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Spi-1/PU.1 proto-oncogene induces opposite effects on monocytic and erythroid differentiation of K562 cells</div>, <div class="source">Biochem. Biophys. Res. Commun</div> , <div class="year">1998</div>, vol. <div class="volume">252</div> (pg. <div class="fpage">383</div>-<div class="lpage">91</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1006/bbrc.1998.9587" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1006/bbrc.1998.9587</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Spi-1%2FPU.1%20proto-oncogene%20induces%20opposite%20effects%20on%20monocytic%20and%20erythroid%20differentiation%20of%20K562%20cells&amp;author=M.D.%20Delgado&amp;author=P.%20Gutierrez&amp;author=C.%20Richard&amp;author=M.A.%20Cuadrado&amp;author=F.%2C%20Moreau-Gachelin&amp;author=J.%20Leon&amp;publication_year=1998&amp;journal=Biochem.%20Biophys.%20Res.%20Commun&amp;volume=252&amp;pages=383-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/bbrc.1998.9587" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fbbrc.1998.9587" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fbbrc.1998.9587"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9826539" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Donehower</div> <div class="given-names">L.A.</div></div>,  <div class="name"><div class="surname">Slagle</div> <div class="given-names">B.L.</div></div>,  <div class="name"><div class="surname">Wilde</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Darlington</div> <div class="given-names">G.,</div></div>,  <div class="name"><div class="surname">Butel</div> <div class="given-names">J.S.</div></div>. </div><div class="article-title">Identification of a conserved sequence in the non-coding regions of many human genes</div>, <div class="source">Nucleic. Acids Res</div> , <div class="year">1989</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">699</div>-<div class="lpage">710</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/17.2.699" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/17.2.699</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20conserved%20sequence%20in%20the%20non-coding%20regions%20of%20many%20human%20genes&amp;author=L.A.%20Donehower&amp;author=B.L.%20Slagle&amp;author=M.%20Wilde&amp;author=G.%2C%20Darlington&amp;author=J.S.%20Butel&amp;publication_year=1989&amp;journal=Nucleic.%20Acids%20Res&amp;volume=17&amp;pages=699-710" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/17.2.699" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F17.2.699" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F17.2.699"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2536922" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div>. </div><div class="article-title">Making (anti)sense of non-coding sequence conservation</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3580</div>-<div class="lpage">3</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/25.18.3580" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/25.18.3580</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Making%20(anti)sense%20of%20non-coding%20sequence%20conservation&amp;author=D.J.%20Lipman&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3580-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.18.3580" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.18.3580" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.18.3580"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9278476" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Levy</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Hannenhalli</div> <div class="given-names">S.,</div></div>,  <div class="name"><div class="surname">Workman</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Enrichment of regulatory signals in conserved non-coding genomic sequence</div>, <div class="source">Bioinformatics</div> , <div class="year">2001</div>, vol. <div class="volume">17</div> (pg. <div class="fpage">871</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/17.10.871" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/17.10.871</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Enrichment%20of%20regulatory%20signals%20in%20conserved%20non-coding%20genomic%20sequence&amp;author=S.%20Levy&amp;author=S.%2C%20Hannenhalli&amp;author=C.%20Workman&amp;publication_year=2001&amp;journal=Bioinformatics&amp;volume=17&amp;pages=871-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/17.10.871" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F17.10.871" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F17.10.871"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11673231" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sandelin</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Bailey</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Bruce</div> <div class="given-names">S.</div></div>, et al. </div><div class="article-title">Arrays of ultraconserved non-coding regions span the loci of key developmental genes in vertebrate genomes</div>, <div class="source">BMC Genomics</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> pg. <div class="fpage">99</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-5-99" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-5-99</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Arrays%20of%20ultraconserved%20non-coding%20regions%20span%20the%20loci%20of%20key%20developmental%20genes%20in%20vertebrate%20genomes&amp;author=A.%20Sandelin&amp;author=P.%20Bailey&amp;author=S.%20Bruce&amp;publication_year=2004&amp;journal=BMC%20Genomics&amp;volume=5&amp;pages=99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-5-99" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-5-99" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-5-99"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15613238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kryukov</div> <div class="given-names">G.V.</div></div>,  <div class="name"><div class="surname">Schmidt</div> <div class="given-names">S.,</div></div>,  <div class="name"><div class="surname">Sunyaev</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Small fitness effect of mutations in highly conserved non-coding regions</div>, <div class="source">Hum. Mol. Genet</div> , <div class="year">2005</div>, vol. <div class="volume">14</div> (pg. <div class="fpage">2221</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/hmg/ddi226" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/hmg/ddi226</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Small%20fitness%20effect%20of%20mutations%20in%20highly%20conserved%20non-coding%20regions&amp;author=G.V.%20Kryukov&amp;author=S.%2C%20Schmidt&amp;author=S.%20Sunyaev&amp;publication_year=2005&amp;journal=Hum.%20Mol.%20Genet&amp;volume=14&amp;pages=2221-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/hmg/ddi226" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fhmg%2Fddi226" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fhmg%2Fddi226"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15994173" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hubbard</div> <div class="given-names">T.J.</div></div>,  <div class="name"><div class="surname">Aken</div> <div class="given-names">B.L.</div></div>,  <div class="name"><div class="surname">Beal</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Ensembl 2007</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">2007</div>, vol. <div class="volume">35</div> (pg. <div class="fpage">D610</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkl996" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkl996</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ensembl%202007&amp;author=T.J.%20Hubbard&amp;author=B.L.%20Aken&amp;author=K.%20Beal&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=D610-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl996" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl996" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl996"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17148474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Thompson</div> <div class="given-names">J.D.</div></div>,  <div class="name"><div class="surname">Higgins</div> <div class="given-names">D.G.,</div></div>,  <div class="name"><div class="surname">Gibson</div> <div class="given-names">T.J.</div></div>. </div><div class="article-title">CLUSTAL W: improving the sensitivity of progressive multiple sequence alignment through sequence weighting, position-specific gap penalties and weight matrix choice</div>, <div class="source">Nucleic Acids Res</div> , <div class="year">1994</div>, vol. <div class="volume">22</div> (pg. <div class="fpage">4673</div>-<div class="lpage">80</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/22.22.4673" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/22.22.4673</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSTAL%20W%3A%20improving%20the%20sensitivity%20of%20progressive%20multiple%20sequence%20alignment%20through%20sequence%20weighting%2C%20position-specific%20gap%20penalties%20and%20weight%20matrix%20choice&amp;author=J.D.%20Thompson&amp;author=D.G.%2C%20Higgins&amp;author=T.J.%20Gibson&amp;publication_year=1994&amp;journal=Nucleic%20Acids%20Res&amp;volume=22&amp;pages=4673-80" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/22.22.4673" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F22.22.4673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F22.22.4673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7984417" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Benjamini</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Drai</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Elmer</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Kafkafi</div> <div class="given-names">N.,</div></div>,  <div class="name"><div class="surname">Golani</div> <div class="given-names">I.</div></div>. </div><div class="article-title">Controlling the false discovery rate in behavior genetics research</div>, <div class="source">Behav. Brain Res</div> , <div class="year">2001</div>, vol. <div class="volume">125</div> (pg. <div class="fpage">279</div>-<div class="lpage">84</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S0166-4328(01)00297-2" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S0166-4328(01)00297-2</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Controlling%20the%20false%20discovery%20rate%20in%20behavior%20genetics%20research&amp;author=Y.%20Benjamini&amp;author=D.%20Drai&amp;author=G.%20Elmer&amp;author=N.%2C%20Kafkafi&amp;author=I.%20Golani&amp;publication_year=2001&amp;journal=Behav.%20Brain%20Res&amp;volume=125&amp;pages=279-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0166-4328(01)00297-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0166-4328(01)00297-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0166-4328(01)00297-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11682119" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fairbrother</div> <div class="given-names">W.G.</div></div>,  <div class="name"><div class="surname">Yeh</div> <div class="given-names">R.F.</div></div>,  <div class="name"><div class="surname">Sharp</div> <div class="given-names">P.A.,</div></div>,  <div class="name"><div class="surname">Burge</div> <div class="given-names">C.B.</div></div>. </div><div class="article-title">Predictive identification of exonic splicing enhancers in human genes</div>, <div class="source">Science</div> , <div class="year">2002</div>, vol. <div class="volume">297</div> (pg. <div class="fpage">1007</div>-<div class="lpage">13</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1073774" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1073774</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Predictive%20identification%20of%20exonic%20splicing%20enhancers%20in%20human%20genes&amp;author=W.G.%20Fairbrother&amp;author=R.F.%20Yeh&amp;author=P.A.%2C%20Sharp&amp;author=C.B.%20Burge&amp;publication_year=2002&amp;journal=Science&amp;volume=297&amp;pages=1007-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1073774" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1073774" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1073774"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12114529" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Plotkin</div> <div class="given-names">J.B.</div></div>,  <div class="name"><div class="surname">Robins</div> <div class="given-names">H.,</div></div>,  <div class="name"><div class="surname">Levine</div> <div class="given-names">A.J.</div></div>. </div><div class="article-title">Tissue-specific codon usage and the expression of human genes</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">12588</div>-<div class="lpage">91</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0404957101" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0404957101</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tissue-specific%20codon%20usage%20and%20the%20expression%20of%20human%20genes&amp;author=J.B.%20Plotkin&amp;author=H.%2C%20Robins&amp;author=A.J.%20Levine&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=12588-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0404957101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0404957101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0404957101"> </span></div> </div></div></div></div></div><div content-id="DSR010C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Drummond</div> <div class="given-names">D.A.,</div></div>,  <div class="name"><div class="surname">Wilke</div> <div class="given-names">C.O.</div></div>. </div><div class="article-title">Mistranslation-induced protein misfolding as a dominant constraint on coding-sequence evolution</div>, <div class="source">Cell</div> , <div class="year">2008</div>, vol. <div class="volume">134</div> (pg. <div class="fpage">341</div>-<div class="lpage">52</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.cell.2008.05.042" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.cell.2008.05.042</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mistranslation-induced%20protein%20misfolding%20as%20a%20dominant%20constraint%20on%20coding-sequence%20evolution&amp;author=D.A.%2C%20Drummond&amp;author=C.O.%20Wilke&amp;publication_year=2008&amp;journal=Cell&amp;volume=134&amp;pages=341-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2008.05.042" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2008.05.042" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2008.05.042"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18662548" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Makalowski</div> <div class="given-names">W.,</div></div>,  <div class="name"><div class="surname">Boguski</div> <div class="given-names">M.S.</div></div>. </div><div class="article-title">Synonymous and nonsynonymous substitution distances are correlated in mouse and rat genes</div>, <div class="source">J. Mol. Evol</div> , <div class="year">1998</div>, vol. <div class="volume">47</div> (pg. <div class="fpage">119</div>-<div class="lpage">21</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/PL00006367" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/PL00006367</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Synonymous%20and%20nonsynonymous%20substitution%20distances%20are%20correlated%20in%20mouse%20and%20rat%20genes&amp;author=W.%2C%20Makalowski&amp;author=M.S.%20Boguski&amp;publication_year=1998&amp;journal=J.%20Mol.%20Evol&amp;volume=47&amp;pages=119-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/PL00006367" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FPL00006367" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FPL00006367"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9694659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR010C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR010C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ihaka</div> <div class="given-names">R.,</div></div>,  <div class="name"><div class="surname">Gentleman</div> <div class="given-names">R.</div></div>. </div><div class="article-title">R: a language for data analysis and graphics</div>, <div class="source">J. Comp. Graph. Stat</div> , <div class="year">1996</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">299</div>-<div class="lpage">314</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2307/1390807" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2307/1390807</a></div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/2.5), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=444849&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/dsr010-dsr010supp_table4.xls?Expires=2147483647&amp;Signature=IMGvHk4pUuvn1527QGynEvSQCB35U3b-nMzieFmZkkCnHNVoN4p5vjOp6rtYM4hNfDR5h5x86L~sWdCEx0W~Ouks79pI1H~JzRGVgGP0kcoXiESKAJgOOga1Q-NZ8ruOrvd0sKNB~vw931MKuSd4aNG~xpVcGR5CnJlmW5U3LvStmArGp5a0hfOXr8PrLChiPRdo0Hr-5u7oc-2PYcjhjVeL74kQozvPqeZzPDcRjWk9dL~G-ztpqjDW~5yLZRWPKPpVKtXsEBng4nxLdCA1reh4JKpmc5PHzrwTCde55HgpRoP2OnuWHQACA2MqozMzAJR24U4yDd0YTo62hSw80g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/dsr010-dsr010supp_table3.xls?Expires=2147483647&amp;Signature=IXt8G4NHsprUmNCTa9-JuX5P6hEXJZpbxsih7wBtxR6zRPASjSt7tSpZpC~QJ~BeZGERif8EPPnYjzKwUUoG4OdV7aCVysjlsrWJQTh4at3VeT7KQf3s1Je1TL4K~0Z6St8Zauahr8-a9eS-7IEZqgc2rqMzp7JjZP~LPOiuNUNGVHQFblvj9MYTjrT3rqE3qRQyUM-yiS-JBieeLMESyb3rNPsn1QF~TkB5jxysdt3AVK0Eg3Qux33-c~TcDqjdmS4BjF3hSf~m~QXqzxky0zLXxR7UMLLtojotfzOjVqzLbIArqWCYEoRuwdd2WZgXvCvJLPjIv6VqTl9FUspDmw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/dsr010-dsr010supp_table2.xls?Expires=2147483647&amp;Signature=ppbHLHli0w3cj-PDW6LPD-oIHgdfd9AcPEN84OBbFjncASeT8kB1n-oubo60X1dgnyl7XCR8G-9H1wy-6oyD3kimgbhwFVcuLXURjSoz55mK-KRPrRomolOAw5ro6bCP5jl7HGMBiKqdK9zUb8eNuvTYL060~zcFG-c2srksw~b1OfMzPS8ihA84LUqC1T5ASwlM-6TWk~stTm5FC4Q7o2TWHu6PoAQ~dQ5rnIDQ4fbsLcTPQt4Z4PjCp~hSG9yVkQTZk5TDx3lya7mmSfMeLXTlJY45dwA5GLAw5WrosUZBAUk3E~PnVJLFT13M4Ua7H4hPalPfCuEb9YmF4AOZ2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/dsr010-dsr010supp_table1.xls?Expires=2147483647&amp;Signature=Ywh53xgTTq279fS8Lmd8ae7NZ2qSwUkFV9EpPPRyfVydyRRWfav9SueULnmWkQZBhEBhjHg-NMWjsfHswRV--Kuq9m~VpYgDgvnFjVkK6UG~teYe2QZvnSLo9Afoos5qzn6M2Fi3NEtXw2-Ph98NZHxt2HgGrnwDvdaFEYJcVysUZJDR7aXuD6JsAeGujD3fth2AWYsWyS14AKxwPubAFwcyM088mOqMJJIhPI75zvebgGyHxznmvNfqqwrKto3rkFM4mh0Mmh4CElua9p0Enc70LuVTCHRLR0tMoLNgip1P~x790EfPVJQ~TlB5GrssKDr6M1-oo-y501A6R98zCw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/3/10.1093/dnares/dsr010/2/dsr010-dsr010supp.doc?Expires=2147483647&amp;Signature=crDrTvsOMj9qIFwXuVh44C6X4i-xRQAH7GTqfB4ScYXH6-q3jFG6tgoaFv0a6Em7WuIH3ug4alIDncVuTnXNOHRzVzRBWHg3vTkF0H0ul-jnO0uyfIzObM1QejYSy9LzL4CUg-4C1VAYj52Aip0KiuKx7yJbU2OoO9n7df6pDmKFYG~GUvhDE9AxZp7c5vvzKNraYhympyasxwNX5N77eVPQLtG86bPKoQWSJLgC6TDtnsTLotYpC8oPjAfQ1dr20BNSUzSDrNgoy3T50tjT8t25cDaeFydD9G35hqWA3dvn9ngIU7D9AYpvylauaIAniMVbq~zus~gcynN13t~XTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">83</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000292070400004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">5</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr010"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="444849"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000292070400004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f3%2f177%2f444849">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000292070400004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (5)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F3%2F177%2F444849" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/444849'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Exploration for Functional Nucleotide Sequence Candidates within Coding Regions of Mammalian Genes" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr010");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f3%2f177%2f444849">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

