

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/2/113/4554776","@type":"ScholarlyArticle","name":"Genome structure of  Rosa multiflora , a wild ancestor of cultivated roses","datePublished":"2017-10-16","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2018-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx042","keywords":["<italic>Rosa multiflora</italic>","rose","genome sequencing","gene prediction","flower"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/2/113/4554776","author":[{"name":"Nakamura, Noriko","affiliation":"Suntory Global Innovation Center Ltd, Seika-cho, Soraku-gun, Kyoto 619-0284, Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Sato, Shusei","affiliation":"Graduate School of Life Sciences, Tohoku University, Aoba-ku, Sendai, Miyagi 980-8577, Japan","@type":"Person"},{"name":"Otagaki, Shungo","affiliation":"Graduate School of Bioagricultural Sciences, Nagoya University, Furo-cho, Chikusa-ku, Nagoya, Aichi 464-8601, Japan","@type":"Person"},{"name":"Matsumoto, Shogo","affiliation":"Graduate School of Bioagricultural Sciences, Nagoya University, Furo-cho, Chikusa-ku, Nagoya, Aichi 464-8601, Japan","@type":"Person"},{"name":"Tabata, Satoshi","affiliation":"Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan","@type":"Person"},{"name":"Tanaka, Yoshikazu","affiliation":"Suntory Global Innovation Center Ltd, Seika-cho, Soraku-gun, Kyoto 619-0284, Japan","@type":"Person"}],"description":"Abstract.  The draft genome sequence of a wild rose (Rosa multiflora Thunb.) was determined using Illumina MiSeq and HiSeq platforms. The total length of the sc","pageStart":"113","pageEnd":"121","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Genome structure of  Rosa multiflora , a wild ancestor of cultivated roses","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Genome structure of  Rosa multiflora , a wild ancestor of cultivated roses" />
    <meta property="og:description" content="Abstract.  The draft genome sequence of a wild rose (Rosa multiflora Thunb.) was determined using Illumina MiSeq and HiSeq platforms. The total length of the sc" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/2/113/4554776" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-10-16" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Nakamura, Noriko" /><meta name="citation_author_institution" content="Suntory Global Innovation Center Ltd, Seika-cho, Soraku-gun, Kyoto 619-0284, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Sato, Shusei" /><meta name="citation_author_institution" content="Graduate School of Life Sciences, Tohoku University, Aoba-ku, Sendai, Miyagi 980-8577, Japan" /><meta name="citation_author" content="Otagaki, Shungo" /><meta name="citation_author_institution" content="Graduate School of Bioagricultural Sciences, Nagoya University, Furo-cho, Chikusa-ku, Nagoya, Aichi 464-8601, Japan" /><meta name="citation_author" content="Matsumoto, Shogo" /><meta name="citation_author_institution" content="Graduate School of Bioagricultural Sciences, Nagoya University, Furo-cho, Chikusa-ku, Nagoya, Aichi 464-8601, Japan" /><meta name="citation_author" content="Tabata, Satoshi" /><meta name="citation_author_institution" content="Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan" /><meta name="citation_author" content="Tanaka, Yoshikazu" /><meta name="citation_author_institution" content="Suntory Global Innovation Center Ltd, Seika-cho, Soraku-gun, Kyoto 619-0284, Japan" /><meta name="citation_title" content="Genome structure of Rosa multiflora, a wild ancestor of cultivated roses" /><meta name="citation_firstpage" content="113" /><meta name="citation_lastpage" content="121" /><meta name="citation_doi" content="10.1093/dnares/dsx042" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2018/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The history of garden roses. Part I;  citation_journal_title=J. Roy. Hort. Soc;  citation_year=1954;  citation_volume=79;  citation_pages=555-71; " /><meta name="citation_reference" content="citation_title=The history of garden roses. Part II;  citation_journal_title=J. Roy. Hort. Soc;  citation_year=1954;  citation_volume=80;  citation_pages=8-24; " /><meta name="citation_reference" content="citation_title=The history of garden roses. Part III;  citation_journal_title=J. Roy. Hort. Soc;  citation_year=1954;  citation_volume=80;  citation_pages=77-87; " /><meta name="citation_reference" content="citation_title=Plant biochemistry: anthocyanin biosynthesis in roses;  citation_journal_title=Nature;  citation_year=2005;  citation_volume=435;  citation_pages=757-8; " /><meta name="citation_reference" content="citation_title=Biosynthesis of monoterpene scent compounds in roses; citation_author=PLANT VOLATILES;  citation_journal_title=Science;  citation_year=2015;  citation_volume=349;  citation_pages=81-3; " /><meta name="citation_reference" content="citation_title=Seasonal induction of alternative principal pathway for rose flower scent;  citation_journal_title=Sci. Rep;  citation_year=2016;  citation_volume=6;  citation_pages=20234" /><meta name="citation_reference" content="citation_title=The genome of the domesticated apple (Malus × domestica Borkh.);  citation_journal_title=Nat. Genet;  citation_year=2010;  citation_volume=42;  citation_pages=833-9; " /><meta name="citation_reference" content="citation_title=The genome of woodland strawberry (Fragaria vesca);  citation_journal_title=Nat. Genet;  citation_year=2011;  citation_volume=43;  citation_pages=109-16; " /><meta name="citation_reference" content="citation_title=The genome of Prunus mume;  citation_journal_title=Nat. Commun;  citation_year=2012;  citation_volume=3;  citation_pages=1318" /><meta name="citation_reference" content="citation_title=The high-quality draft genome of peach (Prunus persica) identifies unique patterns of genetic diversity, domestication and genome evolution;  citation_journal_title=Nat. Genet;  citation_year=2013;  citation_volume=45;  citation_pages=487-94; " /><meta name="citation_reference" content="citation_title=The genome of the pear (Pyrus bretschneideri Rehd.);  citation_journal_title=Genome Res;  citation_year=2013;  citation_volume=23;  citation_pages=396-408; " /><meta name="citation_reference" content="citation_title=The draft genome sequence of European pear (Pyrus communis L. ′Bartlett′);  citation_journal_title=PLoS One;  citation_year=2014;  citation_volume=9;  citation_pages=e92644" /><meta name="citation_reference" content="citation_title=Genome survey sequencing for the characterization of the genetic background of Rosa roxburghii tratt and leaf ascorbate metabolism genes;  citation_journal_title=PLoS One;  citation_year=2016;  citation_volume=11;  citation_pages=e0147530" /><meta name="citation_reference" content="citation_title=Towards the rose genome sequence and its use in research and breeding;  citation_journal_title=Acta Hortic;  citation_year=2015;  citation_volume=1064;  citation_pages=167-75; " /><meta name="citation_reference" content="citation_title=Notes on the origin and evolution of our garden roses;  citation_journal_title=J. Roy. Hort. Soc;  citation_year=1941;  citation_volume=66;  citation_pages=73-82; " /><meta name="citation_reference" content="citation_title=Mining disease-resistance genes in roses: functional and molecular characterization of the Rdr1 locus;  citation_journal_title=Front. Plant Sci;  citation_year=2011;  citation_volume=2;  citation_pages=35" /><meta name="citation_reference" content="citation_title=Nuclear DNA content variation within the Rosaceae;  citation_journal_title=Am. J. Bot;  citation_year=1992;  citation_volume=79;  citation_pages=1081-6; " /><meta name="citation_reference" content="citation_title=High-density SNP-based genetic maps for the parents of an outcrossed and a selfed tetraploid garden rose cross, inferred from admixed progeny using the 68k rose SNP array;  citation_journal_title=Hortic. Res;  citation_year=2016;  citation_volume=3;  citation_pages=16052" /><meta name="citation_reference" content="citation_title=Engineering of the rose flavonoid biosynthetic pathway successfully generated blue-hued flowers accumulating delphinidin;  citation_journal_title=Plant Cell Physiol;  citation_year=2007;  citation_volume=48;  citation_pages=1589-600; " /><meta name="citation_reference" content="citation_title=Sequencing quality assessment tools to enable data-driven informatics for high throughput genomics;  citation_journal_title=Front. Genet;  citation_year=2013;  citation_volume=4;  citation_pages=288" /><meta name="citation_reference" content="citation_title=Quality control and preprocessing of metagenomic datasets;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=863-4; " /><meta name="citation_reference" content="citation_title=A fast, lock-free approach for efficient parallel counting of occurrences of k-mers;  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=764-70; " /><meta name="citation_reference" content="citation_title=Dissection of the octoploid strawberry genome by deep sequencing of the genomes of Fragaria species;  citation_journal_title=DNA Res;  citation_year=2014;  citation_volume=21;  citation_pages=169-81; " /><meta name="citation_reference" content="citation_title=De novo assembly of human genomes with massively parallel short read sequencing;  citation_journal_title=Genome Res;  citation_year=2010;  citation_volume=20;  citation_pages=265-72; " /><meta name="citation_reference" content="citation_title=Full-length transcriptome assembly from RNA-Seq data without a reference genome;  citation_journal_title=Nat. Biotechnol;  citation_year=2011;  citation_volume=29;  citation_pages=644-52; " /><meta name="citation_reference" content="citation_title=RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome;  citation_journal_title=BMC Bioinform;  citation_year=2011;  citation_volume=12;  citation_pages=323" /><meta name="citation_reference" content="citation_title=BLAT–the BLAST-like alignment tool;  citation_journal_title=Genome Res;  citation_year=2002;  citation_volume=12;  citation_pages=656-64; " /><meta name="citation_reference" content="citation_title=L_RNA_scaffolder: scaffolding genomes with transcripts;  citation_journal_title=BMC Genom;  citation_year=2013;  citation_volume=14;  citation_pages=604" /><meta name="citation_reference" content="citation_title=De novo identification of repeat families in large genomes;  citation_journal_title=Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=i351-8; " /><meta name="citation_reference" content="citation_title=TopHat: discovering splice junctions with RNA-Seq;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1105-11; " /><meta name="citation_reference" content="citation_title=Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks;  citation_journal_title=Nat. Protoc;  citation_year=2012;  citation_volume=7;  citation_pages=562-78; " /><meta name="citation_reference" content="citation_title=The sequence alignment/map format and SAMtools; citation_author=1000 Genome Project Data Processing Subgroup;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=2078-9; " /><meta name="citation_reference" content="citation_title=CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes;  citation_journal_title=Bioinformatics;  citation_year=2007;  citation_volume=23;  citation_pages=1061-7; " /><meta name="citation_reference" content="citation_title=BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs;  citation_journal_title=Bioinformatics;  citation_year=2015;  citation_volume=31;  citation_pages=3210-2; " /><meta name="citation_reference" content="citation_title=BRAKER1: unsupervised RNA-seq-based genome annotation with GeneMark-ET and AUGUSTUS;  citation_journal_title=Bioinformatics;  citation_year=2016;  citation_volume=32;  citation_pages=767-9; " /><meta name="citation_reference" content="citation_title=Integration of mapped RNA-Seq reads into automatic training of eukaryotic gene finding algorithm;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=e119" /><meta name="citation_reference" content="citation_title=Gene prediction with a hidden Markov model and a new intron submodel;  citation_journal_title=Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=ii215-25; " /><meta name="citation_reference" content="citation_title=InterProScan: protein domains identifier;  citation_journal_title=Nucleic Acids Res;  citation_year=2005;  citation_volume=33;  citation_pages=W116-20; " /><meta name="citation_reference" content="citation_title=The Gypsy Database (GyDB) of mobile genetic elements: release 2.0;  citation_journal_title=Nucleic Acids Res;  citation_year=2011;  citation_volume=39;  citation_pages=D70-4; " /><meta name="citation_reference" content="citation_title=A new generation of homology search tools based on probabilistic inference;  citation_journal_title=Genome Inform;  citation_year=2009;  citation_volume=23;  citation_pages=205-11; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence;  citation_journal_title=Nucl. Acids Res;  citation_year=1997;  citation_volume=25;  citation_pages=955-64; " /><meta name="citation_reference" content="citation_title=Dendroscope 3: an interactive tool for rooted phylogenetic trees and networks;  citation_journal_title=Syst. Biol;  citation_year=2012;  citation_volume=61;  citation_pages=1061-7; " /><meta name="citation_reference" content="citation_title=The COG database: an updated version includes eukaryotes;  citation_journal_title=BMC Bioinform;  citation_year=2003;  citation_volume=4;  citation_pages=41" /><meta name="citation_reference" content="citation_title=OrthoMCL: identification of ortholog groups for eukaryotic genomes;  citation_journal_title=Genome Res;  citation_year=2003;  citation_volume=13;  citation_pages=2178-89; " /><meta name="citation_reference" content="citation_title=Versatile and open software for comparing large genomes;  citation_journal_title=Genome Biol;  citation_year=2004;  citation_volume=5;  citation_pages=R12" /><meta name="citation_reference" content="citation_title=A genomic analysis of disease-resistance genes encoding nucleotide binding sites in Sorghum bicolor;  citation_journal_title=Genet. Mol. Biol;  citation_year=2010;  citation_volume=33;  citation_pages=292-7; " /><meta name="citation_reference" content="citation_title=Using native and syntenically mapped cDNA alignments to improve de novo gene finding;  citation_journal_title=Bioinformatics,;  citation_year=2008;  citation_volume=24;  citation_pages=637-44; " /><meta name="citation_reference" content="citation_title=Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads;  citation_journal_title=Genome Res;  citation_year=2014;  citation_volume=24;  citation_pages=1384-95; " /><meta name="citation_reference" content="citation_title=Plant pigments for coloration: anthocyanins, betalains and carotenoids;  citation_journal_title=Plant J;  citation_year=2008;  citation_volume=54;  citation_pages=733-49; " /><meta name="citation_reference" content="citation_title=Draft genome assembly and annotation of Glycyrrhiza uralensis, a medicinal legume;  citation_journal_title=Plant J;  citation_year=2017;  citation_volume=89;  citation_pages=181-94; " /><meta name="citation_reference" content="citation_title=Production of 2-phenylethanol in roses as the dominant floral scent compound from L-phenylalanine by two key enzymes, a PLP-dependent decarboxylase and a phenylacetaldehyde reductase;  citation_journal_title=Biosci. Biotechnol. Biochem;  citation_year=2007;  citation_volume=71;  citation_pages=2408-19; " /><meta name="citation_reference" content="citation_title=Functional characterization of rose phenylacetaldehyde reductase (PAR), an enzyme involved in the biosynthesis of the scent compound 2-phenylethanol;  citation_journal_title=J. Plant Physiol;  citation_year=2011;  citation_volume=168;  citation_pages=88-95; " /><meta name="citation_reference" content="citation_title=Volatile components of tea-scented modern roses and ancient Chinese roses;  citation_journal_title=Flavour Fragr. J;  citation_year=2005;  citation_volume=20;  citation_pages=152-7; " /><meta name="citation_reference" content="citation_title=Scent evolution in Chinese roses;  citation_journal_title=Proc. Natl. Acad. Sci. USA;  citation_year=2008;  citation_volume=105;  citation_pages=5927-32; " /><meta name="citation_reference" content="citation_title=Floral volatiles: from biosynthesis to function;  citation_journal_title=Plant. Cell Environ;  citation_year=2014;  citation_volume=37;  citation_pages=1936-49; " /><meta name="citation_reference" content="citation_title=Plant carotenoid cleavage oxygenases and their apocarotenoid products;  citation_journal_title=Curr. Opin. Plant Biol;  citation_year=2006;  citation_volume=9;  citation_pages=315-21; " /><meta name="citation_reference" content="citation_title=Development of floral organ identity: stories from MADS house;  citation_journal_title=Plant Biol;  citation_year=2001;  citation_volume=4;  citation_pages=75-85; " /><meta name="citation_reference" content="citation_title=Cloning and characterization of three APETALA1/FRUITFULL like genes in different flower types of Rosa × hybrida L;  citation_journal_title=J. Plant Growth Regul;  citation_year=2011;  citation_volume=30;  citation_pages=272-85; " /><meta name="citation_reference" content="citation_title=Rose MADS-box genes ′MASAKO BP and B3′homologous to class B floral identity genes;  citation_journal_title=Plant Sci;  citation_year=2001;  citation_volume=161;  citation_pages=549-57; " /><meta name="citation_reference" content="citation_title=Structural and functional analysis of rose class B MADS-box genes ′MASAKO BP, euB3, and B3′: Paleo-type AP3 homologue ′MASAKO B3′ association with petal development;  citation_journal_title=Plant Sci;  citation_year=2006;  citation_volume=170;  citation_pages=778-85; " /><meta name="citation_reference" content="citation_title=Rose MADS-box genes ′MASAKO C1 and D1′ homologous to class C floral identity genes;  citation_journal_title=Plant Sci;  citation_year=2000;  citation_volume=151;  citation_pages=121-34; " /><meta name="citation_reference" content="citation_title=Genetic engineering of floricultural crops: Modification of flower colour, flowering and shape;  citation_journal_title=Acta Hortic;  citation_year=2012;  citation_volume=953;  citation_pages=209-16; " /><meta name="citation_reference" content="citation_title=MADS-box genes in rose: expression analyses of AGAMOUS, PISTILLATA, APETALA3 and SEPALLATA homologue genes in the green rose;  citation_journal_title=Acta Hortic;  citation_year=2005;  citation_volume=690;  citation_pages=203-10; " /><meta name="citation_reference" content="citation_title=Tinkering with the C-Function: a molecular frame for the selection of double flowers in cultivated roses;  citation_journal_title=PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e9288" /><meta name="citation_reference" content="citation_title=The TFL1 homologue KSN is a regulator of continuous flowering in rose and strawberry;  citation_journal_title=Plant J;  citation_year=2012;  citation_volume=69;  citation_pages=116-25; " /><meta name="citation_reference" content="citation_title=Flower opening and closure: an update;  citation_journal_title=J. Exp. Bot;  citation_year=2014;  citation_volume=65;  citation_pages=5749-57; " /><meta name="citation_reference" content="citation_title=Rh-PIP2; 1, a rose aquaporin gene, is involved in ethylene-regulated petal expansion;  citation_journal_title=Plant Physiol;  citation_year=2008;  citation_volume=148;  citation_pages=894-907; " /><meta name="citation_reference" content="citation_title=Cell wall extensibility and effect of cell-wall-loosening proteins during rose flower opening;  citation_journal_title=J. Japan. Soc. Hort. Sci;  citation_year=2009;  citation_volume=78;  citation_pages=242-51; " /><meta name="citation_reference" content="citation_title=Isolation of Rh-TIP1; 1, an aquaporin gene and its expression in rose flowers in response to ethylene and water deficit;  citation_journal_title=Postharvest Biol. Technol;  citation_year=2009;  citation_volume=51;  citation_pages=407-13; " /><meta name="citation_reference" content="citation_title=RhNAC2 and RhEXPA4 are involved in the regulation of dehydration tolerance during the expansion of rose petals;  citation_journal_title=Plant Physiol;  citation_year=2012;  citation_volume=160;  citation_pages=2064-82; " /><meta name="citation_reference" content="citation_title=Involvement of rose aquaporin RhPIP1; 1 in ethylene-regulated petal expansion through interaction with RhPIP2; 1;  citation_journal_title=Plant Mol. Biol;  citation_year=2013;  citation_volume=83;  citation_pages=219-33; " /><meta name="citation_reference" content="citation_title=Evolution of Rosaceae fruit types based on nuclear phylogeny in the context of geological times and genome duplication;  citation_journal_title=Mol. Biol. Evol;  citation_year=2017;  citation_volume=34;  citation_pages=262-81; " /><meta name="citation_reference" content="citation_title=Filling gaps with construction of a genetic linkage map in tetraploid roses;  citation_journal_title=Front. Plant Sci;  citation_year=2014;  citation_volume=5;  citation_pages=796" /><meta name="citation_reference" content="citation_title=Genetic linkage maps of rose constructed with new microsatellite markers and locating QTL controlling flowering traits;  citation_journal_title=Tree Genet. Genomes;  citation_year=2008;  citation_volume=4;  citation_pages=11-23; " /><meta name="citation_reference" content="citation_title=Microsatellite marker development in rose and its application in tetraploid mapping;  citation_journal_title=J. Am. Soc. Hort. Sci;  citation_year=2006;  citation_volume=131;  citation_pages=380-7; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/2/113/24662136/dsx042.pdf" /><meta name="description" content="Abstract.  The draft genome sequence of a wild rose (Rosa multiflora Thunb.) was determined using Illumina MiSeq and HiSeq platforms. The total length of the sc" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/2/113/4554776" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/2/113/4554776" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/2/113/4554776");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsx042';
    </script>
<title>
	Genome structure of Rosa multiflora, a wild ancestor of cultivated roses | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4554776" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfikATxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTmFrYW11cmEsIE5vcmlrbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlN1bnRvcnkgR2xvYmFsIElubm92YXRpb24gQ2VudGVyIEx0ZCwgU2Vpa2EtY2hvLCBTb3Jha3UtZ3VuLCBLeW90byA2MTktMDI4NCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIaXJha2F3YSwgSGlkZWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2F6dXNhIEROQSBSZXNlYXJjaCBJbnN0aXR1dGUsIEtpc2FyYXp1LCBDaGliYSAyOTItMDgxOCwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTYXRvLCBTaHVzZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHcmFkdWF0ZSBTY2hvb2wgb2YgTGlmZSBTY2llbmNlcywgVG9ob2t1IFVuaXZlcnNpdHksIEFvYmEta3UsIFNlbmRhaSwgTWl5YWdpIDk4MC04NTc3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik90YWdha2ksIFNodW5nbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBCaW9hZ3JpY3VsdHVyYWwgU2NpZW5jZXMsIE5hZ295YSBVbml2ZXJzaXR5LCBGdXJvLWNobywgQ2hpa3VzYS1rdSwgTmFnb3lhLCBBaWNoaSA0NjQtODYwMSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYXRzdW1vdG8sIFNob2dvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEJpb2FncmljdWx0dXJhbCBTY2llbmNlcywgTmFnb3lhIFVuaXZlcnNpdHksIEZ1cm8tY2hvLCBDaGlrdXNhLWt1LCBOYWdveWEsIEFpY2hpIDQ2NC04NjAxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlRhYmF0YSwgU2F0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkthenVzYSBETkEgUmVzZWFyY2ggSW5zdGl0dXRlLCBLaXNhcmF6dSwgQ2hpYmEgMjkyLTA4MTgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iVGFuYWthLCBZb3NoaWthenUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTdW50b3J5IEdsb2JhbCBJbm5vdmF0aW9uIENlbnRlciBMdGQsIFNlaWthLWNobywgU29yYWt1LWd1biwgS3lvdG8gNjE5LTAyODQsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJHZW5vbWUgc3RydWN0dXJlIG9mIFJvc2EgbXVsdGlmbG9yYSwgYSB3aWxkIGFuY2VzdG9yIG9mIGN1bHRpdmF0ZWQgcm9zZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIxMTMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjEyMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeDA0MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE4LzA0LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGhpc3Rvcnkgb2YgZ2FyZGVuIHJvc2VzLiBQYXJ0IEk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFJveS4gSG9ydC4gU29jOyAgY2l0YXRpb25feWVhcj0xOTU0OyAgY2l0YXRpb25fdm9sdW1lPTc5OyAgY2l0YXRpb25fcGFnZXM9NTU1LTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgaGlzdG9yeSBvZiBnYXJkZW4gcm9zZXMuIFBhcnQgSUk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFJveS4gSG9ydC4gU29jOyAgY2l0YXRpb25feWVhcj0xOTU0OyAgY2l0YXRpb25fdm9sdW1lPTgwOyAgY2l0YXRpb25fcGFnZXM9OC0yNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGhpc3Rvcnkgb2YgZ2FyZGVuIHJvc2VzLiBQYXJ0IElJSTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gUm95LiBIb3J0LiBTb2M7ICBjaXRhdGlvbl95ZWFyPTE5NTQ7ICBjaXRhdGlvbl92b2x1bWU9ODA7ICBjaXRhdGlvbl9wYWdlcz03Ny04NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnQgYmlvY2hlbWlzdHJ5OiBhbnRob2N5YW5pbiBiaW9zeW50aGVzaXMgaW4gcm9zZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00MzU7ICBjaXRhdGlvbl9wYWdlcz03NTctODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Qmlvc3ludGhlc2lzIG9mIG1vbm90ZXJwZW5lIHNjZW50IGNvbXBvdW5kcyBpbiByb3NlczsgY2l0YXRpb25fYXV0aG9yPVBMQU5UIFZPTEFUSUxFUzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9U2NpZW5jZTsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0zNDk7ICBjaXRhdGlvbl9wYWdlcz04MS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZWFzb25hbCBpbmR1Y3Rpb24gb2YgYWx0ZXJuYXRpdmUgcHJpbmNpcGFsIHBhdGh3YXkgZm9yIHJvc2UgZmxvd2VyIHNjZW50OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2kuIFJlcDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9MjAyMzQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIHRoZSBkb21lc3RpY2F0ZWQgYXBwbGUgKE1hbHVzIMOXIGRvbWVzdGljYSBCb3JraC4pOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9ODMzLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2Ygd29vZGxhbmQgc3RyYXdiZXJyeSAoRnJhZ2FyaWEgdmVzY2EpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTQzOyAgY2l0YXRpb25fcGFnZXM9MTA5LTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIG9mIFBydW51cyBtdW1lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIENvbW11bjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTMxOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBoaWdoLXF1YWxpdHkgZHJhZnQgZ2Vub21lIG9mIHBlYWNoIChQcnVudXMgcGVyc2ljYSkgaWRlbnRpZmllcyB1bmlxdWUgcGF0dGVybnMgb2YgZ2VuZXRpYyBkaXZlcnNpdHksIGRvbWVzdGljYXRpb24gYW5kIGdlbm9tZSBldm9sdXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDU7ICBjaXRhdGlvbl9wYWdlcz00ODctOTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBnZW5vbWUgb2YgdGhlIHBlYXIgKFB5cnVzIGJyZXRzY2huZWlkZXJpIFJlaGQuKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTM5Ni00MDg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBkcmFmdCBnZW5vbWUgc2VxdWVuY2Ugb2YgRXVyb3BlYW4gcGVhciAoUHlydXMgY29tbXVuaXMgTC4g4oCyQmFydGxldHTigLIpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9ZTkyNjQ0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHN1cnZleSBzZXF1ZW5jaW5nIGZvciB0aGUgY2hhcmFjdGVyaXphdGlvbiBvZiB0aGUgZ2VuZXRpYyBiYWNrZ3JvdW5kIG9mIFJvc2Egcm94YnVyZ2hpaSB0cmF0dCBhbmQgbGVhZiBhc2NvcmJhdGUgbWV0YWJvbGlzbSBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz1lMDE0NzUzMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRvd2FyZHMgdGhlIHJvc2UgZ2Vub21lIHNlcXVlbmNlIGFuZCBpdHMgdXNlIGluIHJlc2VhcmNoIGFuZCBicmVlZGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWN0YSBIb3J0aWM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTA2NDsgIGNpdGF0aW9uX3BhZ2VzPTE2Ny03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm90ZXMgb24gdGhlIG9yaWdpbiBhbmQgZXZvbHV0aW9uIG9mIG91ciBnYXJkZW4gcm9zZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFJveS4gSG9ydC4gU29jOyAgY2l0YXRpb25feWVhcj0xOTQxOyAgY2l0YXRpb25fdm9sdW1lPTY2OyAgY2l0YXRpb25fcGFnZXM9NzMtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pbmluZyBkaXNlYXNlLXJlc2lzdGFuY2UgZ2VuZXMgaW4gcm9zZXM6IGZ1bmN0aW9uYWwgYW5kIG1vbGVjdWxhciBjaGFyYWN0ZXJpemF0aW9uIG9mIHRoZSBSZHIxIGxvY3VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Gcm9udC4gUGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0zNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU51Y2xlYXIgRE5BIGNvbnRlbnQgdmFyaWF0aW9uIHdpdGhpbiB0aGUgUm9zYWNlYWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFtLiBKLiBCb3Q7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9Nzk7ICBjaXRhdGlvbl9wYWdlcz0xMDgxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUhpZ2gtZGVuc2l0eSBTTlAtYmFzZWQgZ2VuZXRpYyBtYXBzIGZvciB0aGUgcGFyZW50cyBvZiBhbiBvdXRjcm9zc2VkIGFuZCBhIHNlbGZlZCB0ZXRyYXBsb2lkIGdhcmRlbiByb3NlIGNyb3NzLCBpbmZlcnJlZCBmcm9tIGFkbWl4ZWQgcHJvZ2VueSB1c2luZyB0aGUgNjhrIHJvc2UgU05QIGFycmF5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Ib3J0aWMuIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTYwNTIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbmdpbmVlcmluZyBvZiB0aGUgcm9zZSBmbGF2b25vaWQgYmlvc3ludGhldGljIHBhdGh3YXkgc3VjY2Vzc2Z1bGx5IGdlbmVyYXRlZCBibHVlLWh1ZWQgZmxvd2VycyBhY2N1bXVsYXRpbmcgZGVscGhpbmlkaW47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IENlbGwgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00ODsgIGNpdGF0aW9uX3BhZ2VzPTE1ODktNjAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nIHF1YWxpdHkgYXNzZXNzbWVudCB0b29scyB0byBlbmFibGUgZGF0YS1kcml2ZW4gaW5mb3JtYXRpY3MgZm9yIGhpZ2ggdGhyb3VnaHB1dCBnZW5vbWljczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnJvbnQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0yODgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1RdWFsaXR5IGNvbnRyb2wgYW5kIHByZXByb2Nlc3Npbmcgb2YgbWV0YWdlbm9taWMgZGF0YXNldHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTI3OyAgY2l0YXRpb25fcGFnZXM9ODYzLTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgZmFzdCwgbG9jay1mcmVlIGFwcHJvYWNoIGZvciBlZmZpY2llbnQgcGFyYWxsZWwgY291bnRpbmcgb2Ygb2NjdXJyZW5jZXMgb2Ygay1tZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTc2NC03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzc2VjdGlvbiBvZiB0aGUgb2N0b3Bsb2lkIHN0cmF3YmVycnkgZ2Vub21lIGJ5IGRlZXAgc2VxdWVuY2luZyBvZiB0aGUgZ2Vub21lcyBvZiBGcmFnYXJpYSBzcGVjaWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTIxOyAgY2l0YXRpb25fcGFnZXM9MTY5LTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGFzc2VtYmx5IG9mIGh1bWFuIGdlbm9tZXMgd2l0aCBtYXNzaXZlbHkgcGFyYWxsZWwgc2hvcnQgcmVhZCBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTIwOyAgY2l0YXRpb25fcGFnZXM9MjY1LTcyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GdWxsLWxlbmd0aCB0cmFuc2NyaXB0b21lIGFzc2VtYmx5IGZyb20gUk5BLVNlcSBkYXRhIHdpdGhvdXQgYSByZWZlcmVuY2UgZ2Vub21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz02NDQtNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJTRU06IGFjY3VyYXRlIHRyYW5zY3JpcHQgcXVhbnRpZmljYXRpb24gZnJvbSBSTkEtU2VxIGRhdGEgd2l0aCBvciB3aXRob3V0IGEgcmVmZXJlbmNlIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybTsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTMyMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJMQVTigJN0aGUgQkxBU1QtbGlrZSBhbGlnbm1lbnQgdG9vbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTY1Ni02NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TF9STkFfc2NhZmZvbGRlcjogc2NhZmZvbGRpbmcgZ2Vub21lcyB3aXRoIHRyYW5zY3JpcHRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub207ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz02MDQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGlkZW50aWZpY2F0aW9uIG9mIHJlcGVhdCBmYW1pbGllcyBpbiBsYXJnZSBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPWkzNTEtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG9wSGF0OiBkaXNjb3ZlcmluZyBzcGxpY2UganVuY3Rpb25zIHdpdGggUk5BLVNlcTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xMTA1LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnRpYWwgZ2VuZSBhbmQgdHJhbnNjcmlwdCBleHByZXNzaW9uIGFuYWx5c2lzIG9mIFJOQS1zZXEgZXhwZXJpbWVudHMgd2l0aCBUb3BIYXQgYW5kIEN1ZmZsaW5rczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQcm90b2M7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTU2Mi03ODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIHNlcXVlbmNlIGFsaWdubWVudC9tYXAgZm9ybWF0IGFuZCBTQU10b29sczsgY2l0YXRpb25fYXV0aG9yPTEwMDAgR2Vub21lIFByb2plY3QgRGF0YSBQcm9jZXNzaW5nIFN1Ymdyb3VwOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTIwNzgtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q0VHTUE6IGEgcGlwZWxpbmUgdG8gYWNjdXJhdGVseSBhbm5vdGF0ZSBjb3JlIGdlbmVzIGluIGV1a2FyeW90aWMgZ2Vub21lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjM7ICBjaXRhdGlvbl9wYWdlcz0xMDYxLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJVU0NPOiBhc3Nlc3NpbmcgZ2Vub21lIGFzc2VtYmx5IGFuZCBhbm5vdGF0aW9uIGNvbXBsZXRlbmVzcyB3aXRoIHNpbmdsZS1jb3B5IG9ydGhvbG9nczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0zMjEwLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJSQUtFUjE6IHVuc3VwZXJ2aXNlZCBSTkEtc2VxLWJhc2VkIGdlbm9tZSBhbm5vdGF0aW9uIHdpdGggR2VuZU1hcmstRVQgYW5kIEFVR1VTVFVTOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0zMjsgIGNpdGF0aW9uX3BhZ2VzPTc2Ny05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlZ3JhdGlvbiBvZiBtYXBwZWQgUk5BLVNlcSByZWFkcyBpbnRvIGF1dG9tYXRpYyB0cmFpbmluZyBvZiBldWthcnlvdGljIGdlbmUgZmluZGluZyBhbGdvcml0aG07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTQyOyAgY2l0YXRpb25fcGFnZXM9ZTExOSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmUgcHJlZGljdGlvbiB3aXRoIGEgaGlkZGVuIE1hcmtvdiBtb2RlbCBhbmQgYSBuZXcgaW50cm9uIHN1Ym1vZGVsOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xOTsgIGNpdGF0aW9uX3BhZ2VzPWlpMjE1LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlclByb1NjYW46IHByb3RlaW4gZG9tYWlucyBpZGVudGlmaWVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zMzsgIGNpdGF0aW9uX3BhZ2VzPVcxMTYtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBHeXBzeSBEYXRhYmFzZSAoR3lEQikgb2YgbW9iaWxlIGdlbmV0aWMgZWxlbWVudHM6IHJlbGVhc2UgMi4wOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zOTsgIGNpdGF0aW9uX3BhZ2VzPUQ3MC00OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBnZW5lcmF0aW9uIG9mIGhvbW9sb2d5IHNlYXJjaCB0b29scyBiYXNlZCBvbiBwcm9iYWJpbGlzdGljIGluZmVyZW5jZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEluZm9ybTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yMzsgIGNpdGF0aW9uX3BhZ2VzPTIwNS0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9dFJOQXNjYW4tU0U6IGEgcHJvZ3JhbSBmb3IgaW1wcm92ZWQgZGV0ZWN0aW9uIG9mIHRyYW5zZmVyIFJOQSBnZW5lcyBpbiBnZW5vbWljIHNlcXVlbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsLiBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTE5OTc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz05NTUtNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlbmRyb3Njb3BlIDM6IGFuIGludGVyYWN0aXZlIHRvb2wgZm9yIHJvb3RlZCBwaHlsb2dlbmV0aWMgdHJlZXMgYW5kIG5ldHdvcmtzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TeXN0LiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTYxOyAgY2l0YXRpb25fcGFnZXM9MTA2MS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQ09HIGRhdGFiYXNlOiBhbiB1cGRhdGVkIHZlcnNpb24gaW5jbHVkZXMgZXVrYXJ5b3RlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Qk1DIEJpb2luZm9ybTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT00OyAgY2l0YXRpb25fcGFnZXM9NDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PcnRob01DTDogaWRlbnRpZmljYXRpb24gb2Ygb3J0aG9sb2cgZ3JvdXBzIGZvciBldWthcnlvdGljIGdlbm9tZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTM7ICBjaXRhdGlvbl9wYWdlcz0yMTc4LTg5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1WZXJzYXRpbGUgYW5kIG9wZW4gc29mdHdhcmUgZm9yIGNvbXBhcmluZyBsYXJnZSBnZW5vbWVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9UjEyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBnZW5vbWljIGFuYWx5c2lzIG9mIGRpc2Vhc2UtcmVzaXN0YW5jZSBnZW5lcyBlbmNvZGluZyBudWNsZW90aWRlIGJpbmRpbmcgc2l0ZXMgaW4gU29yZ2h1bSBiaWNvbG9yOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldC4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9MjkyLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIG5hdGl2ZSBhbmQgc3ludGVuaWNhbGx5IG1hcHBlZCBjRE5BIGFsaWdubWVudHMgdG8gaW1wcm92ZSBkZSBub3ZvIGdlbmUgZmluZGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3MsOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9NjM3LTQ0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZpY2llbnQgZGUgbm92byBhc3NlbWJseSBvZiBoaWdobHkgaGV0ZXJvenlnb3VzIGdlbm9tZXMgZnJvbSB3aG9sZS1nZW5vbWUgc2hvdGd1biBzaG9ydCByZWFkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTEzODQtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IHBpZ21lbnRzIGZvciBjb2xvcmF0aW9uOiBhbnRob2N5YW5pbnMsIGJldGFsYWlucyBhbmQgY2Fyb3Rlbm9pZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEo7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTQ7ICBjaXRhdGlvbl9wYWdlcz03MzMtNDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURyYWZ0IGdlbm9tZSBhc3NlbWJseSBhbmQgYW5ub3RhdGlvbiBvZiBHbHljeXJyaGl6YSB1cmFsZW5zaXMsIGEgbWVkaWNpbmFsIGxlZ3VtZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgSjsgIGNpdGF0aW9uX3llYXI9MjAxNzsgIGNpdGF0aW9uX3ZvbHVtZT04OTsgIGNpdGF0aW9uX3BhZ2VzPTE4MS05NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvZHVjdGlvbiBvZiAyLXBoZW55bGV0aGFub2wgaW4gcm9zZXMgYXMgdGhlIGRvbWluYW50IGZsb3JhbCBzY2VudCBjb21wb3VuZCBmcm9tIEwtcGhlbnlsYWxhbmluZSBieSB0d28ga2V5IGVuenltZXMsIGEgUExQLWRlcGVuZGVudCBkZWNhcmJveHlsYXNlIGFuZCBhIHBoZW55bGFjZXRhbGRlaHlkZSByZWR1Y3Rhc2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb3NjaS4gQmlvdGVjaG5vbC4gQmlvY2hlbTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT03MTsgIGNpdGF0aW9uX3BhZ2VzPTI0MDgtMTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZ1bmN0aW9uYWwgY2hhcmFjdGVyaXphdGlvbiBvZiByb3NlIHBoZW55bGFjZXRhbGRlaHlkZSByZWR1Y3Rhc2UgKFBBUiksIGFuIGVuenltZSBpbnZvbHZlZCBpbiB0aGUgYmlvc3ludGhlc2lzIG9mIHRoZSBzY2VudCBjb21wb3VuZCAyLXBoZW55bGV0aGFub2w7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFBsYW50IFBoeXNpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTY4OyAgY2l0YXRpb25fcGFnZXM9ODgtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZvbGF0aWxlIGNvbXBvbmVudHMgb2YgdGVhLXNjZW50ZWQgbW9kZXJuIHJvc2VzIGFuZCBhbmNpZW50IENoaW5lc2Ugcm9zZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZsYXZvdXIgRnJhZ3IuIEo7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz0xNTItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2NlbnQgZXZvbHV0aW9uIGluIENoaW5lc2Ugcm9zZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTEwNTsgIGNpdGF0aW9uX3BhZ2VzPTU5MjctMzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZsb3JhbCB2b2xhdGlsZXM6IGZyb20gYmlvc3ludGhlc2lzIHRvIGZ1bmN0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudC4gQ2VsbCBFbnZpcm9uOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTM3OyAgY2l0YXRpb25fcGFnZXM9MTkzNi00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGxhbnQgY2Fyb3Rlbm9pZCBjbGVhdmFnZSBveHlnZW5hc2VzIGFuZCB0aGVpciBhcG9jYXJvdGVub2lkIHByb2R1Y3RzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBQbGFudCBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz0zMTUtMjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURldmVsb3BtZW50IG9mIGZsb3JhbCBvcmdhbiBpZGVudGl0eTogc3RvcmllcyBmcm9tIE1BRFMgaG91c2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTc1LTg1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbG9uaW5nIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHRocmVlIEFQRVRBTEExL0ZSVUlURlVMTCBsaWtlIGdlbmVzIGluIGRpZmZlcmVudCBmbG93ZXIgdHlwZXMgb2YgUm9zYSDDlyBoeWJyaWRhIEw7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIFBsYW50IEdyb3d0aCBSZWd1bDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTI3Mi04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9zZSBNQURTLWJveCBnZW5lcyDigLJNQVNBS08gQlAgYW5kIEIz4oCyaG9tb2xvZ291cyB0byBjbGFzcyBCIGZsb3JhbCBpZGVudGl0eSBnZW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTE2MTsgIGNpdGF0aW9uX3BhZ2VzPTU0OS01NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3RydWN0dXJhbCBhbmQgZnVuY3Rpb25hbCBhbmFseXNpcyBvZiByb3NlIGNsYXNzIEIgTUFEUy1ib3ggZ2VuZXMg4oCyTUFTQUtPIEJQLCBldUIzLCBhbmQgQjPigLI6IFBhbGVvLXR5cGUgQVAzIGhvbW9sb2d1ZSDigLJNQVNBS08gQjPigLIgYXNzb2NpYXRpb24gd2l0aCBwZXRhbCBkZXZlbG9wbWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgU2NpOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE3MDsgIGNpdGF0aW9uX3BhZ2VzPTc3OC04NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Um9zZSBNQURTLWJveCBnZW5lcyDigLJNQVNBS08gQzEgYW5kIEQx4oCyIGhvbW9sb2dvdXMgdG8gY2xhc3MgQyBmbG9yYWwgaWRlbnRpdHkgZ2VuZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwMDsgIGNpdGF0aW9uX3ZvbHVtZT0xNTE7ICBjaXRhdGlvbl9wYWdlcz0xMjEtMzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZW5naW5lZXJpbmcgb2YgZmxvcmljdWx0dXJhbCBjcm9wczogTW9kaWZpY2F0aW9uIG9mIGZsb3dlciBjb2xvdXIsIGZsb3dlcmluZyBhbmQgc2hhcGU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUFjdGEgSG9ydGljOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTk1MzsgIGNpdGF0aW9uX3BhZ2VzPTIwOS0xNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TUFEUy1ib3ggZ2VuZXMgaW4gcm9zZTogZXhwcmVzc2lvbiBhbmFseXNlcyBvZiBBR0FNT1VTLCBQSVNUSUxMQVRBLCBBUEVUQUxBMyBhbmQgU0VQQUxMQVRBIGhvbW9sb2d1ZSBnZW5lcyBpbiB0aGUgZ3JlZW4gcm9zZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QWN0YSBIb3J0aWM7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjkwOyAgY2l0YXRpb25fcGFnZXM9MjAzLTEwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaW5rZXJpbmcgd2l0aCB0aGUgQy1GdW5jdGlvbjogYSBtb2xlY3VsYXIgZnJhbWUgZm9yIHRoZSBzZWxlY3Rpb24gb2YgZG91YmxlIGZsb3dlcnMgaW4gY3VsdGl2YXRlZCByb3NlczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPWU5Mjg4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIFRGTDEgaG9tb2xvZ3VlIEtTTiBpcyBhIHJlZ3VsYXRvciBvZiBjb250aW51b3VzIGZsb3dlcmluZyBpbiByb3NlIGFuZCBzdHJhd2JlcnJ5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBKOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTY5OyAgY2l0YXRpb25fcGFnZXM9MTE2LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GbG93ZXIgb3BlbmluZyBhbmQgY2xvc3VyZTogYW4gdXBkYXRlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBFeHAuIEJvdDsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT02NTsgIGNpdGF0aW9uX3BhZ2VzPTU3NDktNTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJoLVBJUDI7IDEsIGEgcm9zZSBhcXVhcG9yaW4gZ2VuZSwgaXMgaW52b2x2ZWQgaW4gZXRoeWxlbmUtcmVndWxhdGVkIHBldGFsIGV4cGFuc2lvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UGxhbnQgUGh5c2lvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDg7ICBjaXRhdGlvbl9wYWdlcz04OTQtOTA3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DZWxsIHdhbGwgZXh0ZW5zaWJpbGl0eSBhbmQgZWZmZWN0IG9mIGNlbGwtd2FsbC1sb29zZW5pbmcgcHJvdGVpbnMgZHVyaW5nIHJvc2UgZmxvd2VyIG9wZW5pbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEphcGFuLiBTb2MuIEhvcnQuIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT03ODsgIGNpdGF0aW9uX3BhZ2VzPTI0Mi01MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIG9mIFJoLVRJUDE7IDEsIGFuIGFxdWFwb3JpbiBnZW5lIGFuZCBpdHMgZXhwcmVzc2lvbiBpbiByb3NlIGZsb3dlcnMgaW4gcmVzcG9uc2UgdG8gZXRoeWxlbmUgYW5kIHdhdGVyIGRlZmljaXQ7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBvc3RoYXJ2ZXN0IEJpb2wuIFRlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz00MDctMTM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJoTkFDMiBhbmQgUmhFWFBBNCBhcmUgaW52b2x2ZWQgaW4gdGhlIHJlZ3VsYXRpb24gb2YgZGVoeWRyYXRpb24gdG9sZXJhbmNlIGR1cmluZyB0aGUgZXhwYW5zaW9uIG9mIHJvc2UgcGV0YWxzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1QbGFudCBQaHlzaW9sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE2MDsgIGNpdGF0aW9uX3BhZ2VzPTIwNjQtODI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludm9sdmVtZW50IG9mIHJvc2UgYXF1YXBvcmluIFJoUElQMTsgMSBpbiBldGh5bGVuZS1yZWd1bGF0ZWQgcGV0YWwgZXhwYW5zaW9uIHRocm91Z2ggaW50ZXJhY3Rpb24gd2l0aCBSaFBJUDI7IDE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBsYW50IE1vbC4gQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT04MzsgIGNpdGF0aW9uX3BhZ2VzPTIxOS0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXZvbHV0aW9uIG9mIFJvc2FjZWFlIGZydWl0IHR5cGVzIGJhc2VkIG9uIG51Y2xlYXIgcGh5bG9nZW55IGluIHRoZSBjb250ZXh0IG9mIGdlb2xvZ2ljYWwgdGltZXMgYW5kIGdlbm9tZSBkdXBsaWNhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDE3OyAgY2l0YXRpb25fdm9sdW1lPTM0OyAgY2l0YXRpb25fcGFnZXM9MjYyLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GaWxsaW5nIGdhcHMgd2l0aCBjb25zdHJ1Y3Rpb24gb2YgYSBnZW5ldGljIGxpbmthZ2UgbWFwIGluIHRldHJhcGxvaWQgcm9zZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZyb250LiBQbGFudCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTc5NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgbGlua2FnZSBtYXBzIG9mIHJvc2UgY29uc3RydWN0ZWQgd2l0aCBuZXcgbWljcm9zYXRlbGxpdGUgbWFya2VycyBhbmQgbG9jYXRpbmcgUVRMIGNvbnRyb2xsaW5nIGZsb3dlcmluZyB0cmFpdHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVRyZWUgR2VuZXQuIEdlbm9tZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTExLTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NaWNyb3NhdGVsbGl0ZSBtYXJrZXIgZGV2ZWxvcG1lbnQgaW4gcm9zZSBhbmQgaXRzIGFwcGxpY2F0aW9uIGluIHRldHJhcGxvaWQgbWFwcGluZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQW0uIFNvYy4gSG9ydC4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTEzMTsgIGNpdGF0aW9uX3BhZ2VzPTM4MC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjUvMi8xMTMvMjQ2NjIxMzYvZHN4MDQyLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgZHJhZnQgZ2Vub21lIHNlcXVlbmNlIG9mIGEgd2lsZCByb3NlIChSb3NhIG11bHRpZmxvcmEgVGh1bmIuKSB3YXMgZGV0ZXJtaW5lZCB1c2luZyBJbGx1bWluYSBNaVNlcSBhbmQgSGlTZXEgcGxhdGZvcm1zLiBUaGUgdG90YWwgbGVuZ3RoIG9mIHRoZSBzYyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDU1NDc3NiZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGSnOsLcKv5lSNO8uFX5iU/pukstoA==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f113%2f4554776" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f113%2f4554776" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=108668" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&amp;Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/2">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="115733142">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733142">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733144">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733144">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733149">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733149">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733171">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733171">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733208">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733208">Availability of data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733210">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733210">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733218">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733218">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/25/2/123/4657153">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Noriko Nakamura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Suntory Global Innovation Center Ltd, Seika-cho, Soraku-gun, Kyoto 619-0284, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Noriko+Nakamura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakamura N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakamura N&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Noriko Nakamura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shusei Sato</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Life Sciences, Tohoku University, Aoba-ku, Sendai, Miyagi 980-8577, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shusei+Sato">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sato S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sato S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shusei Sato</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shungo Otagaki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Bioagricultural Sciences, Nagoya University, Furo-cho, Chikusa-ku, Nagoya, Aichi 464-8601, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shungo+Otagaki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Otagaki S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Otagaki S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shungo Otagaki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shogo Matsumoto</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Graduate School of Bioagricultural Sciences, Nagoya University, Furo-cho, Chikusa-ku, Nagoya, Aichi 464-8601, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shogo+Matsumoto">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Matsumoto S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Matsumoto S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shogo Matsumoto</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoshi Tabata</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Kazusa DNA Research Institute, Kisarazu, Chiba 292-0818, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoshi+Tabata">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tabata S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tabata S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Satoshi Tabata</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshikazu Tanaka</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Suntory Global Innovation Center Ltd, Seika-cho, Soraku-gun, Kyoto 619-0284, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx042-cor1" data-open="dsx042-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81 50 3182 0451. Fax. +81 774 98 6262. Email: <a href="mailto:Yoshikazu_Tanaka@suntory.co.jp" target="_blank">Yoshikazu_Tanaka@suntory.co.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshikazu+Tanaka">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tanaka Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tanaka Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshikazu Tanaka</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx042-FM1"><div class="fn"><div class="footnote-content">Noriko Nakamura and Hideki Hirakawa contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx042-FM2"><div class="fn"><div class="footnote-content">Edited by Dr. Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 2, 1 April 2018, Pages 113–121, <a href='https://doi.org/10.1093/dnares/dsx042'>https://doi.org/10.1093/dnares/dsx042</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">16 October 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">12 June 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">19 September 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4554776" href="/dnaresearch/article-pdf/25/2/113/24662136/dsx042.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Noriko Nakamura, Hideki Hirakawa, Shusei Sato, Shungo Otagaki, Shogo Matsumoto, Satoshi Tabata, Yoshikazu Tanaka;  Genome structure of <em>Rosa multiflora</em>, a wild ancestor of cultivated roses, <em>DNA Research</em>, Volume 25, Issue 2, 1 April 2018, Pages 113–121, <a href='https://doi.org/10.1093/dnares/dsx042'>https://doi.org/10.1093/dnares/dsx042</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4554776?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4554776?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4554776?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4554776?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4554776?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4554776?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4554776?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Genome%20structure%20of%20%26lt%3Bem%26gt%3BRosa%20multiflora%26lt%3B%2Fem%26gt%3B%2C%20a%20wild%20ancestor%20of%20cultivated%20roses&amp;publicationDate=2017-10-16&amp;volumeNum=25&amp;issueNum=2&amp;author=Nakamura%2C%20Noriko%3B%20Hirakawa%2C%20Hideki&amp;startPage=113&amp;endPage=121&amp;contentId=10.1093%2Fdnares%2Fdsx042&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f113%2f4554776" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=115733142 id="115733142" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The draft genome sequence of a wild rose (<em>Rosa multiflora</em> Thunb.) was determined using Illumina MiSeq and HiSeq platforms. The total length of the scaffolds was 739,637,845 bp, consisting of 83,189 scaffolds, which was close to the 711 Mbp length estimated by <em>k</em>-mer analysis. N50 length of the scaffolds was 90,830 bp, and extent of the longest was 1,133,259 bp. The average GC content of the scaffolds was 38.9%. After gene prediction, 67,380 candidates exhibiting sequence homology to known genes and domains were extracted, which included complete and partial gene structures. This large number of genes for a diploid plant may reflect heterogeneity of the genome originating from self-incompatibility in <em>R. multiflora</em>. According to CEGMA analysis, 91.9% and 98.0% of the core eukaryotic genes were completely and partially conserved in the scaffolds, respectively. Genes presumably involved in flower color, scent and flowering are assigned. The results of this study will serve as a valuable resource for fundamental and applied research in the rose, including breeding and phylogenetic study of cultivated roses.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Rosa multiflora&quot;"><em>Rosa multiflora</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="rose">rose</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome sequencing&quot;">genome sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;gene prediction&quot;">gene prediction</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="flower">flower</a></div>                            <h2 scrollto-destination=115733144 id="115733144" class="section-title" >1. Introduction</h2>
<p>Cultivated roses (<em>Rosa </em>×<em> hybrida</em>) are by far the most beloved flowers and the most important floricultural crop. The genus <em>Rosa</em> contains 120–200 species distributed in the Northern hemisphere. <em>R. hybrida</em> constitutes approximately eight <em>Rosa</em> species (<em>R. multiflora</em>, <em>R. luciae</em>, <em>R. moschata</em>, <em>R. damascene</em>, <em>R. gallica</em>, <em>R. chinensis</em>, <em>R. gigantea</em>, and <em>R. foetida</em>)<a reveal-id="dsx042-B1 dsx042-B2 dsx042-B3" data-open="dsx042-B1 dsx042-B2 dsx042-B3" class="link link-ref link-reveal xref-bibr"><sup>1–3</sup></a> by repeated natural and artificial hybridizations. <em>R. hybrida</em> is a tetraploid (2<em>n</em> = 4<em>x</em> = 28), and the ancestral roses are diploid (2<em>n</em> = 2<em>x</em> = 14). As a consequence of natural and artificial hybridization, cultivated roses have various floral characteristics, including intense red, orange, and yellow colors, ever-flowering and larger inflorescences with a large number of petals. Roses have been partly characterized focusing on ornamentally important characters such as flower color, scent, flowering, floral morphogenesis, and development. Roses generally share common mechanisms of these characters with other flowering plants. Roses also contain unique enzymes such as anthocyanin 5, 3-glucosyltransferase,<a reveal-id="dsx042-B4" data-open="dsx042-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> nucleoside diphosphate linked some moiety X hydrolase 1 (Nudix 1) leading to monoterpenes,<a reveal-id="dsx042-B5" data-open="dsx042-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> and phenylpyruvate decarboxylase (RyPPDC) leading to 2-phenylethanol (2PE).<a reveal-id="dsx042-B6" data-open="dsx042-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a></p><p>The <em>Rosaceae</em> family contains many important fruit plants and some genome structures have been studied, such as apple (<em>Malus × domestica</em>),<a reveal-id="dsx042-B7" data-open="dsx042-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> woodland strawberry (<em>Fragaria vesca</em>),<a reveal-id="dsx042-B8" data-open="dsx042-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> Japanese apricot (<em>Prunus mume</em>),<a reveal-id="dsx042-B9" data-open="dsx042-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> peach (<em>Prunus persica</em>),<a reveal-id="dsx042-B10" data-open="dsx042-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> pear (<em>Pyrus bretschneideri</em>),<a reveal-id="dsx042-B11" data-open="dsx042-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> and European pear (<em>Pyrus communis</em>).<a reveal-id="dsx042-B12" data-open="dsx042-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Recently, the <em>Rosa roxburghii</em> genome, a Chinese medicinal rose, was surveyed.<a reveal-id="dsx042-B13" data-open="dsx042-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> The Rose Genome Sequence Initiative is presently obtaining a high-quality genome sequence of <em>R. chinensis</em> cultivar Old Blush and provides RNA-Seq data.<a reveal-id="dsx042-B14" data-open="dsx042-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> The scaffold sequences of <em>Rosa × damascena</em> (accession number PRJNA322107) and Illumina raw reads from <em>R. dumalis</em>, <em>R. inodora</em>, and <em>R. canina</em> (PRJEB15546) are available from the NCBI SRA database. Currently, the whole genome sequence of an ancestral species of <em>R. hybrida</em> has not been revealed. </p><p><em>R. multiflora</em> belonging to the section <em>Synstylae</em> is native to eastern Asia, including Japan. It is a thorny perennial shrub and exhibits clusters of white or pale fragrant flowers of five petals. <em>R. multiflora</em>, derived from Japan, was utilized to breed modern cultivated roses<a reveal-id="dsx042-B15" data-open="dsx042-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> to confer clustering fluorescence to <em>R. hybrida</em>. Its resistance locus (<em>Rdr1</em>) to black spot caused by <em>Diplocarpon rosae</em> Wolf has been introgressed into <em>R. hybrida</em>.<a reveal-id="dsx042-B16" data-open="dsx042-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> A genomic region of 265,477 bp containing <em>Rdr1</em> with a cluster of nine highly related TIR-NBS-LRR candidate genes has been reported.<a reveal-id="dsx042-B16" data-open="dsx042-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> The nuclear (2 C) DNA amounts of <em>R. multiflora</em> has been estimated to be 1.65 pg,<a reveal-id="dsx042-B17" data-open="dsx042-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> indicating its haploid genome size is approximately 750 Mb.</p><p>To deepen fundamental understanding of <em>R. multiflora</em> and related species, structural analysis of the whole genome of <em>R. multiflora</em> was performed. This genomic study will also be a valuable resource for rose breeding, in combination with the genetic map<a reveal-id="dsx042-B18" data-open="dsx042-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and pave the way to clarify complex pedigree of the cultivated roses in terms of genome level.</p>                            <h2 scrollto-destination=115733149 id="115733149" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=115733150 id="115733150" class="section-title" >2.1. Materials</h3>
<p><em>R. multiflora</em> studied here was obtained from Keisei Rose Nurseries (Chiba, Japan) (<a reveal-id="dsx042-F1" data-open="dsx042-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). This plant cultivar originated from Sawara, Chiba prefecture, Japan. The flavonoids of the petals were analysed previously as described.<a reveal-id="dsx042-B19" data-open="dsx042-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Genomic DNA was prepared from the young leaves using DNeasy Plant Maxi Kit (QIAGEN, Valencia, USA). Total RNA was prepared from the petal of bud, young leaf, and young root of <em>R. multiflora</em> (<a reveal-id="dsx042-F1" data-open="dsx042-F1" class="link link-reveal link-table xref-fig">Fig. 1B, C, and D</a>) and from the young petal and young leaf of the <em>R. hybrida</em> cultivar ‘Rote Rose’ using RNeasy Plant Mini Kit (QIAGEN, Valencia, USA). </p>                        <a id="115733152" scrollto-destination="115733152"></a>
<div id="dsx042-F1" data-id="dsx042-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/m_dsx042f1.png?Expires=2147483647&Signature=buJHS9p7d-3xgiSUD~199ibssjV8qpD2jUj4dwXqq087pNUUbq57taZg6fAGjfD9y5OcIpSEKQdUo8P-uArV-rJR3Ua1CUbxrWN1tkPFWaUAHwlj4L-MmZkRTMWE8zvrWFZZ9Bg~UYj0OqalhprXZmqK82cKkT0b69Ekaa5MU6L7Bw~VEAB9jg5h22wL7FY5bjkkGLSg8i3-Q6ZIJtAwKJBqMW3WT8PzZLwAYSTNml2nuM6Xl2zgHZHayk~Vmqwr7LqsdSVplDD-5JCDRsdue1sIQ-P-HsbGvHQABU2Ujs1O80QrMXXIsYvKrlerFJFCn-8Iamzcuom1iVxOA~I-dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="R. multiflora used in this study (A). Total RNA was prepared from the petals of buds (B), leaves (C) and roots (D) for RNA-Seq analysis. " path-from-xml="dsx042f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733152/dsx042f1.tif" path-from-xml="dsx042f1.tif" target="_blank">View large</a><a section="115733152" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042f1.png?Expires=2147483647&Signature=sID8qUXCxdwmw3-EzC3TaOVt1usnIZSVPjF2gefoWZGmk6FAuWSriP7ts8pVdZiiBxtbYEUAfRmagBstQGCTWx6beIHx3U9QQ3AeG9Os30rlt5GEZQ-kkYHJxNJXN9ZPNx5HOP0wVYCfCXILXDZx7FppNdukYNEWiWZunTQS4fdmY8GoyRfMrNk65wx~ie9-2DqsAv~Q2vDcxindOHbhYBz1807NnJiKlg44n5HPxX83wRrMfCWUz0vznkjHlpjhd1Cyr7TcRgNgIFjufsXapOpRye108hHCs72ChmPG4eEDtD0hVC4OnkKvEPUmFVQxRXHaEvABSgFDit5qVE1Fjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733152&ar=4554776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx042f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>R. multiflora</em> used in this study (A). Total RNA was prepared from the petals of buds (B), leaves (C) and roots (D) for RNA-Seq analysis. </p></div></div><div content-id="dsx042-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/m_dsx042f1.png?Expires=2147483647&Signature=buJHS9p7d-3xgiSUD~199ibssjV8qpD2jUj4dwXqq087pNUUbq57taZg6fAGjfD9y5OcIpSEKQdUo8P-uArV-rJR3Ua1CUbxrWN1tkPFWaUAHwlj4L-MmZkRTMWE8zvrWFZZ9Bg~UYj0OqalhprXZmqK82cKkT0b69Ekaa5MU6L7Bw~VEAB9jg5h22wL7FY5bjkkGLSg8i3-Q6ZIJtAwKJBqMW3WT8PzZLwAYSTNml2nuM6Xl2zgHZHayk~Vmqwr7LqsdSVplDD-5JCDRsdue1sIQ-P-HsbGvHQABU2Ujs1O80QrMXXIsYvKrlerFJFCn-8Iamzcuom1iVxOA~I-dw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="R. multiflora used in this study (A). Total RNA was prepared from the petals of buds (B), leaves (C) and roots (D) for RNA-Seq analysis. " path-from-xml="dsx042f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733152/dsx042f1.tif" path-from-xml="dsx042f1.tif" target="_blank">View large</a><a section="115733152" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042f1.png?Expires=2147483647&Signature=sID8qUXCxdwmw3-EzC3TaOVt1usnIZSVPjF2gefoWZGmk6FAuWSriP7ts8pVdZiiBxtbYEUAfRmagBstQGCTWx6beIHx3U9QQ3AeG9Os30rlt5GEZQ-kkYHJxNJXN9ZPNx5HOP0wVYCfCXILXDZx7FppNdukYNEWiWZunTQS4fdmY8GoyRfMrNk65wx~ie9-2DqsAv~Q2vDcxindOHbhYBz1807NnJiKlg44n5HPxX83wRrMfCWUz0vznkjHlpjhd1Cyr7TcRgNgIFjufsXapOpRye108hHCs72ChmPG4eEDtD0hVC4OnkKvEPUmFVQxRXHaEvABSgFDit5qVE1Fjg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733152&ar=4554776&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx042f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p><em>R. multiflora</em> used in this study (A). Total RNA was prepared from the petals of buds (B), leaves (C) and roots (D) for RNA-Seq analysis. </p></div></div>                            <h3 scrollto-destination=115733153 id="115733153" class="section-title" >2.2. Shotgun sequencing</h3>
                            <h4 scrollto-destination=115733154 id="115733154" class="section-title" >2.2.1. Next-generation sequencing</h4>
<p>Shotgun sequencing was carried out using HiSeq 2000 and MiSeq platforms (Illumina Inc., CA, USA). The paired-end (PE) library with insert size of approximately 500 bp was prepared by TruSeq Nano DNA LT Sample Prep Kit. Mate-pair (MP) libraries with insert sizes 2, 5, 10, and 20 kb were constructed by Nextera Mate Pair Sample Prep Kit and GS FLX Titanium Paired End Adaptor Set. These samples were run on HiSeq 2000 and MiSeq with 101 and 301 cycles sequencing kits, respectively. Sequencing of transcripts (RNA-Seq) was also performed for <em>R. multiflora</em> and <em>R. hybrida</em> cultivar ‘Rote Rose’, the most common rose cultivar in Japan. The PE libraries of RNA-Seq sampled from bud, leaf, and root were sequenced by MiSeq sequencer.</p>                            <h4 scrollto-destination=115733156 id="115733156" class="section-title" >2.2.2. Quality control</h4>
<p>The read quality was checked by FastQC 0.11.2.<a reveal-id="dsx042-B20" data-open="dsx042-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Nucleotides with quality value &lt;10 and adaptor sequences at 3′ termini of reads were trimmed by PRINSEQ 0.20.4<a reveal-id="dsx042-B21" data-open="dsx042-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> and FASTX-toolkit 0.0.14 (<a class="link link-uri" href="http://hannonlab.cshl.edu/fastx_toolkit" target="_blank">http://hannonlab.cshl.edu/fastx_toolkit</a>), respectively. After trimming, PE and MP reads with &gt;100 bases and PE reads with size of 250–300 bases were selected for sequencing by HiSeq 2000 and MiSeq, respectively. Reads shorter than 99 bases and including unknown nucleotides were excluded.</p>                            <h4 scrollto-destination=115733158 id="115733158" class="section-title" >2.2.3. Estimation of genome size</h4>
<p>The genome size of <em>R. multiflora</em> was estimated using HiSeq 2000 and MiSeq PE reads with <em>k</em>-mer size = 17. The <em>k</em>-mer distribution was investigated using Jellyfish 2.1.3.<a reveal-id="dsx042-B22" data-open="dsx042-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> Genome size was estimated using the peak in the <em>k</em>-mer frequency distribution curve according to the method used in a previous study.<a reveal-id="dsx042-B23" data-open="dsx042-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a></p>                            <h3 scrollto-destination=115733160 id="115733160" class="section-title" >2.3. Assembly, expression level analysis, and SNP detection</h3>
<p>The trimmed PE and MP reads were used for genome assembly by SOAPdenovo rev240 (-M 1)<a reveal-id="dsx042-B24" data-open="dsx042-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> with <em>k</em>-mer sizes, 71, 81, and 91. After assembly, gaps on scaffolds were closed by GapCloser 1.10 (<a class="link link-uri" href="http://soap.genomics.org.cn/soapdenovo.html" target="_blank">http://soap.genomics.org.cn/soapdenovo.html</a>) (<em>p </em>= 31). The scaffolds were subjected to BLASTN searches with <em>E</em>-value cutoff of 1<em>E</em>-10 and length coverage ≥10% against bacteria, fungi, and human genome sequences (hg19) from NCBI, vector sequences from UniVec (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/" target="_blank">http://www.ncbi.nlm.nih.gov/tools/vecscreen/univec/</a>), genome sequences of chloroplast of <em>Fragaria vesca</em> (NC_015206.1) and mitochondria of <em>Arabidopsis thaliana</em> (NC_001284.2), and PhiX sequence used in Illumina sequencing. Scaffolds exhibiting homology to these were excluded as contamination.</p><p>RNA-Seq reads sampled from bud, leaf, and root of <em>R. multiflora</em> were assembled by Trinity r20140717.<a reveal-id="dsx042-B25" data-open="dsx042-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> At same gene locus, several contigs derived from splicing variants were predicted; therefore, the contig with the highest IsoPct value calculated by RSEM 1.2.15<a reveal-id="dsx042-B26" data-open="dsx042-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> was selected as the transcript. The contigs thus obtained were mapped onto the scaffolds by BLAT v34<a reveal-id="dsx042-B27" data-open="dsx042-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> with ≥90% similarity and score ≥200 (–maxIntron = 10,000). According to the mapping results, scaffolds were connected by L_RNA_scaffolder.<a reveal-id="dsx042-B28" data-open="dsx042-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> As a result, scaffolds longer than 300 bases were selected and designated RMU_r2.0. Repetitive sequences were detected by RepeatScout 1.0.5<a reveal-id="dsx042-B29" data-open="dsx042-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> and RepeatMasker 4.0.3 (<a class="link link-uri" href="http://www.repeatmasker.org" target="_blank">http://www.repeatmasker.org</a>) according to the method used in a previous study.<a reveal-id="dsx042-B23" data-open="dsx042-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a></p><p>Expression levels of the genes in bud, leaf, and root were investigated for <em>R. multiflora</em>. The RNA-Seq reads were mapped onto the scaffolds of RMU_r2.0 by TopHat v2.0.14.<a reveal-id="dsx042-B30" data-open="dsx042-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> The FPKM (fragments per kilobase of exon per million fragments mapped reads) value of the genes were calculated by Cufflinks v2.2.1.<a reveal-id="dsx042-B31" data-open="dsx042-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> RNA-Seq reads of <em>R. hybrida</em> cultivar ‘Rote Rose’ were used for detection of SNPs distinguishing <em>R. multiflora</em>. The RNA-Seq reads were mapped onto the scaffolds of RMU_r2.0 by TopHat v2.0.14.<a reveal-id="dsx042-B30" data-open="dsx042-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> The BAM files obtained were used for SNP detection by SAMtools v0.1.19.<a reveal-id="dsx042-B32" data-open="dsx042-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a></p>                            <h3 scrollto-destination=115733164 id="115733164" class="section-title" >2.4. Authenticity</h3>
<p>The full BAC sequence of the black spot resistance <em>muRdr1</em> gene locus of <em>R. multiflora</em> line 88/124-46<a reveal-id="dsx042-B16" data-open="dsx042-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> was obtained from NCBI nuccore database (HQ455834.1; 265,477 bp). The nine TIR-NBS-LRR resistance proteins, muRdr1A-muRdr1I, were encoded in the BAC sequence. The genes were mapped onto the scaffolds of RMU_r2.0 by BLAT with ≥95% similarity and score ≥200 (–minIdentity = 95). To compare the genic regions among the related species, the EST sequences of <em>R. hybrida</em> (12,649 sequences; <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>), <em>R. luciae</em> (1,936 sequences; <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>), and <em>R. virginiana</em> (5,978 sequences; <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) obtained from NCBI’s dbEST were mapped onto the genome sequence of RMU_r2.0 by BLAT with ≥95% similarity and score ≥100. Conservation of the core eukaryotic genes and single-copy orthologous genes were investigated using CEGMA v2.5<a reveal-id="dsx042-B33" data-open="dsx042-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> and BUSCO ver. 1.1b,<a reveal-id="dsx042-B34" data-open="dsx042-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> respectively. In CEGMA, genome completeness was estimated by using 248 CEGs (Core Eukaryotic Genes) to classify them into complete and partial genes. In BUSCO, genome completeness was estimated by using single-copy orthologous genes selected from OrthoDB to classify them into complete genes (single-copy and duplicated), fragmented genes, and missing genes.</p>                            <h3 scrollto-destination=115733166 id="115733166" class="section-title" >2.5. Gene assignment and annotation</h3>
<p>The RNA-Seq reads were mapped onto the scaffolds of RMU_r2.0 with TopHat 2.0.14<a reveal-id="dsx042-B30" data-open="dsx042-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> to generate a BAM file. This file was used for making a training set by BRAKER1 v1.3.<a reveal-id="dsx042-B35" data-open="dsx042-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> GeneMark-ET 4.21,<a reveal-id="dsx042-B36" data-open="dsx042-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> and Augustus 3.0.3<a reveal-id="dsx042-B37" data-open="dsx042-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> were initially applied to build the training set, and Augustus 3.0.3 was applied to predict genes using the training set. The predicted genes were subjected to homology searches against NCBI NR database (<a class="link link-uri" href="http://ftp://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/nr.gz" target="_blank">ftp://ftp.ncbi.nlm.nih.gov/blast/db/FASTA/nr.gz</a>) and translated genes of <em>A. thaliana</em> in TAIR10 (<a class="link link-uri" href="https://www.arabidopsis.org" target="_blank">https://www.arabidopsis.org</a>) using BLASTP with an <em>E</em>-value cutoff of 1<em>E</em>-10. Domain searches against InterPro (<a class="link link-uri" href="http://www.ebi.ac.uk/interpro/" target="_blank">http://www.ebi.ac.uk/interpro/</a>) were conducted using InterProScan<a reveal-id="dsx042-B38" data-open="dsx042-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> with an <em>E</em>-value cutoff of 1.0. Genes related to transposable elements (TEs) were inferred according to the results of BLAST searches against the NCBI NR database, and domain searches against InterPro and GyDB 2.0<a reveal-id="dsx042-B39" data-open="dsx042-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> using hmmsearch in HMMER 3.0<a reveal-id="dsx042-B40" data-open="dsx042-B40" class="link link-ref link-reveal xref-bibr"><sup>40</sup></a> with an <em>E</em>-value cutoff of 1.0. Transfer RNA genes (tRNAs) were predicted by tRNAscan-SE 1.23.<a reveal-id="dsx042-B41" data-open="dsx042-B41" class="link link-ref link-reveal xref-bibr"><sup>41</sup></a> Ribosomal RNA genes (rRNAs) were predicted by HMMER 3.0 searches against the Rfam 11.0 database.</p><p>The predicted peptide sequences of <em>R. multiflora</em> were searched to assign enzymes/proteins to various biosynthetic pathways by BLASTP using counterpart protein sequences as queries, and one or a few best-matched sequences with low <em>E</em>-values were selected. In the case of phylogenetic analysis of MADS-box genes, expansins, xyloglucan endotransglycosylase/hydrolases (XTHs) or aquaporins, we used BLASTP and keyword search (the word ‘MADS’, ‘expansin’, ‘xyloglucan endotransglucosylase/hydrolase’, or ‘aquaporin’ was used) in the <em>Rosa multiflora</em> Genome DataBase (<a class="link link-uri" href="http://rosa.kazusa.or.jp" target="_blank">http://rosa.kazusa.or.jp</a>). For BLASTP search, we used amino acid sequences of each genes characterized in model plants as queries with <em>E</em>-value cutoff of 1E-20. Multiple sequence alignments were constructed by DDBJ CLUSTALW version 2.1 (<a class="link link-uri" href="http://clustalw.ddbj.nig.ac.jp/" target="_blank">http://clustalw.ddbj.nig.ac.jp/</a>) with default parameters. Phylogenetic trees were generated using the neighbor-joining method included in the DDBJ CLUSTALW version 2.1 with default parameters and circular cladograms were constructed using Dendroscope 3.<a reveal-id="dsx042-B42" data-open="dsx042-B42" class="link link-ref link-reveal xref-bibr"><sup>42</sup></a></p>                            <h3 scrollto-destination=115733169 id="115733169" class="section-title" >2.6. Functional analyses of the predicted genes</h3>
<p>The predicted genes were classified into the functional categories defined in NCBI’s ‘euKaryotic clusters of Orthologous Groups (KOG)’<a reveal-id="dsx042-B43" data-open="dsx042-B43" class="link link-ref link-reveal xref-bibr"><sup>43</sup></a> by BLASTP searches with an <em>E</em>-value cutoff of 1<em>E</em>-4. The genes were mapped onto the KEGG reference pathways by BLAST searches against the KEGG GENES database (<a class="link link-uri" href="http://www.genome.jp/kegg/genes.html" target="_blank">http://www.genome.jp/kegg/genes.html</a>) and UniProt (TrEMBL + Swiss-Prot) database (<a class="link link-uri" href="http://www.uniprot.org" target="_blank">http://www.uniprot.org</a>) with an <em>E</em>-value cutoff of 1<em>E</em>-80, length coverage of 25%, and identity of 50%. The genes were also mapped onto the KEGG reference pathways of <em>F. vesca</em> (v2.0a1), <em>P. persica</em> (peach; v2.0a1), and <em>Malus </em>×<em> domestica</em> (apple; v1.0p). The genes of the four plant species were compared by OrthoMCL v2.0.9.<a reveal-id="dsx042-B44" data-open="dsx042-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> The genome sequence, CDS and pep sequences, and annotation of <em>F. vesca</em> (v2.0a1), peach (v2.0a1), and apple (v1.0p) were obtained from the Genome Database for Rosaceae (GDR; <a class="link link-uri" href="https://www.rosaceae.org" target="_blank">https://www.rosaceae.org</a>). The Gene Ontology (GO) categories were assigned to the genes based on the raw files obtained by InterProScan. The distribution of the genes in GO categories was investigated according to GOslim (<a class="link link-uri" href="http://www.geneontology.org/page/go-slim-and-subset-guide" target="_blank">http://www.geneontology.org/page/go-slim-and-subset-guide</a>).</p>                            <h2 scrollto-destination=115733171 id="115733171" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=115733172 id="115733172" class="section-title" >3.1. Sequencing of the rose genome</h3>
                            <h4 scrollto-destination=115733173 id="115733173" class="section-title" >3.1.1. Shotgun sequencing of the rose genome</h4>
<p>The 1.1 G PE reads were obtained by HiSeq 2000 with length 100 bp and 178 M PE reads for MiSeq with length 301 bp. The MP reads of insert sizes, 2, 5, 10, and 20 kb, were obtained by HiSeq 2000 for 501, 610, 425, and 394 M reads with length 101 bp. The obtained reads are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. The quality of reads was checked using FastQC, and quality trimming and adaptor trimming were performed by PRINSEQ and FastX-toolkit, respectively. The one nucleotide on 3′ termini was trimmed because of low quality. Finally, reads with lengths 100 and 250–300 bp were selected for HiSeq 2000 and MiSeq reads, respectively, and divided into paired and single reads. After quality control, 95.3% of MiSeq PE reads and 88.7% of HiSeq 2000 PE reads were used for assembly. For MP reads with insert sizes, 2, 5, 10, and 20 kb obtained by HiSeq 2000, 26.3%, 4.4%, 9.3%, and 4.9% were used for assembly, respectively.</p>                            <h4 scrollto-destination=115733175 id="115733175" class="section-title" >3.1.2. Assembly authenticity and sequence features</h4>
<p>The <em>k</em>-mer frequency distribution curve (<em>k</em>-mer =17) derived from PE reads is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&amp;Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>. As a result, the genome size was estimated at 1,087,968,027 and 711,129,940 bases using the two peaks at multiplicity = 117 (coverage = 133.7) and 179 (coverage = 204.5), respectively. According to the peak (multiplicity = 179), the haploid genome size was estimated to 711 Mb, which was close to the estimated size,<a reveal-id="dsx042-B17" data-open="dsx042-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a><sup>,</sup><a reveal-id="dsx042-B23" data-open="dsx042-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> and thus the peak at multiplicity = 117 may reflect the presence of heterozygosity in <em>R. multiflora</em>. The numbers of raw and trimmed reads are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&amp;Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. The trimmed reads were applied to the assembly using SOAPdenovo2 with <em>k</em>-mer sizes = 71, 81, and 91. From the results of the assemblies, the N50 lengths using <em>k</em>-mer sizes 71, 81, and 91 were 30,732, 35,285, and 28,059 bases, respectively. Judging from the N50 lengths, the scaffolds assembled with <em>k</em>-mer size = 81 were used for further analysis (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&amp;Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). The gaps on the scaffolds were closed by GapCloser 1.10. The RNA-Seq from bud, leaf, and root was assembled by Trinity, and splicing variants were excluded by RSEM (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&amp;Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). The contigs were used for scaffolding using L_RNA_scaffolder. As a result, 158,733 scaffolds with total length 767,886,425 and N50 length 86,097 bp were obtained (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&amp;Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). Finally, 75,439 scaffolds shorter than 500 bp and 105 scaffolds with probable contamination hit against chloroplast, mitochondrial, fungal and bacterial genomes were excluded, and the resultant 83,189 scaffolds were determined as representing the draft genome sequence, RMU_r2.0 (<a reveal-id="dsx042-T1" data-open="dsx042-T1" class="link link-reveal link-table xref-fig">Table 1</a>). The total length of RMU_r2.0 was 739,637,845 bp, and N50 length was 90,830 bp. The scaffolds were prefixed with ‘Rmu’ followed by a seven-digit identifier and sequence version (e.g. Rmu_ssc0000001.1). Considering the genome size estimated by distribution of <em>k</em>-mer frequency, the total length of the assembled genome sequence was somewhat longer, probably due to heterozygosity. <div content-id="dsx042-T1" class="table-modal"><div class="table-wrap"><div id="dsx042-T1" data-id="dsx042-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Genomic feature of RMU_r2.0 and RMU_r2.0_cds</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>RMU_r2.0 (genome) </th><th>RMU_r2.0_cds (CDS) </th></tr></thead><tbody><tr><td>Number of sequences </td><td>83,189 </td><td>67,380 </td></tr><tr><td>Total length (bases) </td><td>739,637,845 </td><td>66,058,172 </td></tr><tr><td>Average length (bases) </td><td>8,891 </td><td>980 </td></tr><tr><td>Max length (bases) </td><td>1,133,259 </td><td>20,538 </td></tr><tr><td>Min length (bases) </td><td>501 </td><td>149 </td></tr><tr><td>N50 length (bases) </td><td>90,830 </td><td>1,272 </td></tr><tr><td>G+C% </td><td>38.9 </td><td>45.9 </td></tr><tr><td>Repeat% </td><td>56.4 </td><td>– </td></tr><tr><td>Complete genes </td><td>– </td><td>54,893 </td></tr><tr><td>Partial genes </td><td>– </td><td>12,487 </td></tr></tbody></table></div><div content-id="dsx042-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>RMU_r2.0 (genome) </th><th>RMU_r2.0_cds (CDS) </th></tr></thead><tbody><tr><td>Number of sequences </td><td>83,189 </td><td>67,380 </td></tr><tr><td>Total length (bases) </td><td>739,637,845 </td><td>66,058,172 </td></tr><tr><td>Average length (bases) </td><td>8,891 </td><td>980 </td></tr><tr><td>Max length (bases) </td><td>1,133,259 </td><td>20,538 </td></tr><tr><td>Min length (bases) </td><td>501 </td><td>149 </td></tr><tr><td>N50 length (bases) </td><td>90,830 </td><td>1,272 </td></tr><tr><td>G+C% </td><td>38.9 </td><td>45.9 </td></tr><tr><td>Repeat% </td><td>56.4 </td><td>– </td></tr><tr><td>Complete genes </td><td>– </td><td>54,893 </td></tr><tr><td>Partial genes </td><td>– </td><td>12,487 </td></tr></tbody></table></div></div></div><div class="table-wrap"><div id="dsx042-T1" data-id="dsx042-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Genomic feature of RMU_r2.0 and RMU_r2.0_cds</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>RMU_r2.0 (genome) </th><th>RMU_r2.0_cds (CDS) </th></tr></thead><tbody><tr><td>Number of sequences </td><td>83,189 </td><td>67,380 </td></tr><tr><td>Total length (bases) </td><td>739,637,845 </td><td>66,058,172 </td></tr><tr><td>Average length (bases) </td><td>8,891 </td><td>980 </td></tr><tr><td>Max length (bases) </td><td>1,133,259 </td><td>20,538 </td></tr><tr><td>Min length (bases) </td><td>501 </td><td>149 </td></tr><tr><td>N50 length (bases) </td><td>90,830 </td><td>1,272 </td></tr><tr><td>G+C% </td><td>38.9 </td><td>45.9 </td></tr><tr><td>Repeat% </td><td>56.4 </td><td>– </td></tr><tr><td>Complete genes </td><td>– </td><td>54,893 </td></tr><tr><td>Partial genes </td><td>– </td><td>12,487 </td></tr></tbody></table></div><div content-id="dsx042-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>RMU_r2.0 (genome) </th><th>RMU_r2.0_cds (CDS) </th></tr></thead><tbody><tr><td>Number of sequences </td><td>83,189 </td><td>67,380 </td></tr><tr><td>Total length (bases) </td><td>739,637,845 </td><td>66,058,172 </td></tr><tr><td>Average length (bases) </td><td>8,891 </td><td>980 </td></tr><tr><td>Max length (bases) </td><td>1,133,259 </td><td>20,538 </td></tr><tr><td>Min length (bases) </td><td>501 </td><td>149 </td></tr><tr><td>N50 length (bases) </td><td>90,830 </td><td>1,272 </td></tr><tr><td>G+C% </td><td>38.9 </td><td>45.9 </td></tr><tr><td>Repeat% </td><td>56.4 </td><td>– </td></tr><tr><td>Complete genes </td><td>– </td><td>54,893 </td></tr><tr><td>Partial genes </td><td>– </td><td>12,487 </td></tr></tbody></table></div></div></p><p>Authenticity of the assembled genome sequence was performed by similarity searches for the full BAC sequence of <em>R. multiflora</em> and for EST sequences of <em>R. hybrida</em>, <em>R. lucieae</em>, and <em>R. virginiana</em>. In the BAC sequence, only nine TIR-NBS-LRR resistance genes were annotated. Therefore, gene prediction was conducted to the BAC sequence by the same method applied to the scaffolds. As a result, 53 genes were predicted from the BAC sequence, and 10 genes of them were homologous to TIR-NBS-LRR resistance genes. Synteny between Rmu_sc0000110.1 and the BAC sequence was investigated using nucmer,<a reveal-id="dsx042-B45" data-open="dsx042-B45" class="link link-ref link-reveal xref-bibr"><sup>45</sup></a> as shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>. There are two regions without similarity between them. The red and blue bars indicate the genic regions on plus and minus strands, respectively. The gray bar indicate the unknown nucleotides (Ns). The region from 160 kb to 200 kb on the BAC sequence corresponded to the region with unknown nucleotides on Rmu_sc0000110.1. On the other hand, the region from 20 kb to 120 kb was not similar, which encoded TIR-NBR-LRR resistance genes. The BAC clone encodes TIR-NBS-LRR resistance genes, and the genome region of TIR-NBS-LRR resistance genes tend to be rearranged.<a reveal-id="dsx042-B46" data-open="dsx042-B46" class="link link-ref link-reveal xref-bibr"><sup>46</sup></a> As the strain of the BAC clone (breeding line 88/124-46) is different from our target, the non-conserved region of the BAC clone could be caused by genome rearrangement. However, the half region on 3’ terminal without TIR-NBS-LRR genes on the BAC sequence was conserved well. Moreover, 12,003 of 12,649 (94.8%) of EST sequences of <em>R. hybrida</em>, 1,455 of 1,936 (75.1%) of EST sequences from <em>R. lucieae</em>, and 5,040 of 5,978 (84.3%) of EST sequences in <em>R. virginiana</em> were mapped onto RMU_r2.0, respectively.</p><p>Authenticity of the assembled genome sequence was also verified by use of CEGMA<a reveal-id="dsx042-B33" data-open="dsx042-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> and BUSCO<a reveal-id="dsx042-B34" data-open="dsx042-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> programs. The results of CEGMA and BUSCO were shown in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>. Of the 248 core genes, 228 (91.9%) were completely conserved, while 15 (6.1%) were partially conserved, according to the CEGMA output. In contrast, 845 of the 956 genes defined in BUSCO program (88.4%) were classified as ‘complete single-copy.’ 348 genes (36.4%) were categorized as ‘complete duplicated,’ 40 (4.2%) were classified as ‘fragmented,’ and 71 genes (7.4%) were classified as ‘missing’ when analysed by BUSCO. The 91.9% and 88.4% of the genes defined by CEGMA and BUSCO were classified into complete structure, which indicate that the genes were conserved with high rates judged from the status of the genome assembly. In BUSCO analysis, 348 (36.4%) complete duplicated genes were detected. This high rate might be due to the high heterozygosity in RMU_r2.0. These results indicated that the core genes and single-copy orthologous genes might be conserved in RMU_r2.0. The core genes duplicated in RMU_r2.0 might be derived from the contigs separated by heterozygosity.</p>                            <h3 scrollto-destination=115733179 id="115733179" class="section-title" >3.2. Characteristic features of the rose genome</h3>
                            <h4 scrollto-destination=115733180 id="115733180" class="section-title" >3.2.1. Repetitive sequences</h4>
<p>The known and unique repetitive sequences identified in RMU_r2.0 are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S6</a></span>. Total length of the known repeats was 417,242,576 bp (56.4% of the total) and in which Class I LTR elements were frequently present. In contrast, the unique repeats that have not been sequenced were newly identified in our analysis; the total length of these was 290,260,400 bp (39.2% of the total).</p>                            <h4 scrollto-destination=115733182 id="115733182" class="section-title" >3.2.2. Prediction of protein-encoding genes and annotation</h4>
<p>According to the mapping results for RNA-Seq reads, 3,622,550 of 5,123,157 read pairs (70.7%) were mapped onto the draft genome sequence (RMU_r2.0). The training set for <em>R. multiflora</em> was constructed by BRAKER1,<a reveal-id="dsx042-B35" data-open="dsx042-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and used for gene prediction by Augustus 3.0.3.<a reveal-id="dsx042-B47" data-open="dsx042-B47" class="link link-ref link-reveal xref-bibr"><sup>47</sup></a> As a result, we predicted 178,512 genes on the genome sequence. According to the BLAST search against the NCBI NR database and by domain searches against InterPro, 67, 380 genes were selected with support by sequence alignment evidence. The first gene set was called after RMU_r2.0.cds, and second one was called after RMU_r2.0.braker1.cds. The statistics of the predicted genes are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&amp;Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>. The total length of the CDSs was 66,058,172 bp with 45.9% GC content. This number of CDSs was higher compared with other <em>Rosaceae</em> plants, which may be caused by the presence of genes derived from duplicated contigs due to heterozygous genome regions that self incompatibility of <em>R. multiflora</em> results in. Kajitani et al. simulated the genome assembly of <em>Caenorhabditis elegans</em> with various levels of heterozygosity in Illumina reads ranged from 0.1 to 2.0%, and they indicated that the lengths of the contigs and scaffolds were shorter as higher heterozygosity.<a reveal-id="dsx042-B48" data-open="dsx042-B48" class="link link-ref link-reveal xref-bibr"><sup>48</sup></a> This means that the genes would be partial in the case of high heterozygosity. The gene name was prefixed with a seven-digit identifier followed by scaffold or contig number, for example, Rmu_ssc0000001.1_g000001.1. According to the presence of start or stop codons, genes were tagged as partial (with start or stop codons or without start or stop codons) or pseudogenes (presence of stop codons in CDS). Genes less than 50 amino acids were tagged as short. Genes having similarity to NR database entries by BLAST with <em>E</em>-values ≤1<em>E</em>-20 and identity ≥70% were tagged with ‘f,’ and those with <em>E</em>-values ≤1<em>E</em>-20 and identity &lt;70% were tagged with ‘p.’ Genes having hits against InterPro with <em>E</em>-values ≤1.0 were labeled with ‘d.’ In RMU_r2.0.braker1.cds, genes having similarity to transposable elements were tagged with ‘TE.’ The number of genes with TE tag was 46,505. In RMU_r2.0.cds, the numbers of the tags ‘/fd (tags f and d),’ ‘/pd (tags p and d),’ ‘/f-,’ ‘/p-,’ and ‘/-d’ were 24,121, 16,461, 2,416, 5, 259, and 6,636. The numbers of the tags for partial genes ‘/partial/fd,’ ‘/partial/pd,’ ‘/partial/f,’ ‘/partial/p,’ ‘/partial/d’ were 6,842, 2,067, 1,240, 900, 1,438, respectively. The numbers of 5.8 S, 18 S, and 25 S rRNA genes were 4, 22, and 31, respectively. The tRNA and rRNA genes predicted in <em>R. multiflora</em>, <em>F. vesca</em>, and <em>P. persica</em> are compared in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&amp;Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S8 and S9</a></span>, respectively. The distributions of KOG functional categories of <em>R. multiflora</em>, <em>F. vesca</em>, and <em>P. persica</em> were similar (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&amp;Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). The numbers of the genes mapped onto KEGG metabolic pathways classified into ‘1. Metabolism’ are shown in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&amp;Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>. The 17,677 genes (26.2%) of <em>R. multiflora</em> were mapped onto 346 of the 476 metabolic pathways in the KEGG database, whereas the 8,262 (24.6%), 11,710 (39.3%), 12,753 (48.2%), and 12,934 (46.7%) genes of <em>F. vesca, M.</em> × <em>domestica</em>, <em>P. persica</em>, and <em>A. thaliana</em> were mapped onto 344, 342, 344, and 345 pathways, respectively. The pathways were categorized as to which genes in the <em>R. multiflora</em> genome were uniquely mapped and were as follows: ‘Ascorbate and aldarate metabolism’ in ‘1.1. Carbohydrate metabolism,’ ‘Methane metabolism’ in ‘1.2 Energy metabolism,’ ‘Riboflavin metabolism’ in ‘1.8 Metabolism of cofactors and vitamins,’ ‘Monoterpenoid biosynthesis’ in ‘1.9 Metabolism of terpenoids and polyketides,’ ‘Isoquinoline alkaloid biosynthesis’ in ‘1.10 Biosynthesis of other secondary metabolites.’</p>                            <h4 scrollto-destination=115733184 id="115733184" class="section-title" >3.2.3. Flavonoid biosynthetic genes</h4>
<p>Pink- to red-flower colors of roses are derived from cyanidin or pelargonidin glucosides belonging to anthocyanins, a class of colored flavonoids (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>). Rose flowers do not contain the delphinidin or flavone that is common in blue or violet flowers. Anthocyanin biosynthesis leading to anthocyanidin 3-glucoside is well conserved in higher plants.<a reveal-id="dsx042-B49" data-open="dsx042-B49" class="link link-ref link-reveal xref-bibr"><sup>49</sup></a> The <em>R. multiflora</em> genome contains amino acid sequences exhibiting high identity to reported biosynthetic enzymes (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>).</p><p><em>R. multiflora</em> petals contain large amounts of kaempferol (3.975 ± 0.183 mg/g fresh petals), small quantities of quercetin (0.109 ± 0.014), and cyanidin (0.001 ± 0.001). Flavones and 3′, 5′-hydroxylated flavonoids such as delphinidin and myricetin were not detected. The flavonoid profiles indicate the presence of flavonol synthase (FLS) and flavonoid 3′-hydroxylase (F3′H) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&amp;Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>) and the absence of flavonoid 3’,5’-hydroxylase (F3′5′H) and flavone synthase (FNS) in <em>R. multiflora</em>. Genomic genes corresponding to FLS and F3′H were identified in the genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&amp;Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>), and those corresponding to F3′5′H and FNS were not found. The genes involved in regulation of flavonoid biosynthesis and vacuolar transport will be reported separately. The flavonoid biosynthetic pathway includes plural cytochromes P450 (P450) and UDP-sugar dependent glucosyltrasnferases/glycosyltransferases (GT). The <em>R. multiflora</em> genome contains 677 P450 and 507 GT ORFs in the scaffold sequences of RMU_r2.0. These numbers exceed those in other plant genomes, confirming that the <em>R. multiflora</em> genome is heterogeneous. Many GT and P450 genes are found to form clusters in the same scaffolds. Rmu_sc0005080.1 contains 11 GT genes and Rmu_sc0000698.1 contains 10 P450 genes (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&amp;Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S11 and S12</a></span>). Such clusters of P450 and GT genes are revealed in <em>Glycyrrhiza uralensis</em>.<a reveal-id="dsx042-B50" data-open="dsx042-B50" class="link link-ref link-reveal xref-bibr"><sup>50</sup></a> P450 and GT co-localize in eight scaffolds in <em>G. uralensis</em> and in four scaffolds of the <em>R. multiflora</em> genome (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&amp;Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S13</a></span>).</p>                            <h4 scrollto-destination=115733187 id="115733187" class="section-title" >3.2.4. Floral scent and carotenoid-related genes</h4>
<p>Rose floral scent compounds are mainly benzenoids such as 2-phenylethanol (2PE) and terpenoids, including geraniol. Scent compounds of modern roses are derived from their ancestral wild roses. 2PE is synthesized by two pathways: one is via aromatic amino acid decarboxylase (AADC)<a reveal-id="dsx042-B51" data-open="dsx042-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> and phenylacetaldehyde reductase (PAR) in winter<a reveal-id="dsx042-B52" data-open="dsx042-B52" class="link link-ref link-reveal xref-bibr"><sup>52</sup></a> and the other is via phenylpyruvate decarboxylase (PPDC) in summer.<a reveal-id="dsx042-B6" data-open="dsx042-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> The biosynthetic pathway of floral scent compounds, the relevant enzymes, and corresponding <em>R. multiflora</em> genes are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>. <em>R. multiflora</em> predominantly produces 2PE but not 3, 5-dimethoxytoluene (DMT) or 1, 3, 5-trimethoxybenzene (TMB).<a reveal-id="dsx042-B53" data-open="dsx042-B53" class="link link-ref link-reveal xref-bibr"><sup>53</sup></a> 2PE has a rose-like floral note and is one of the key scent compounds in roses.<a reveal-id="dsx042-B51" data-open="dsx042-B51" class="link link-ref link-reveal xref-bibr"><sup>51</sup></a> Genes corresponding to AADC, PAR, and PPDC are found in the genome (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). Eugenol synthase and eugenol methyltransferase genes were also found.</p><p>The ‘tea scent’ compound of tea roses has been shown to be DMT, which is derived from <em>R. chinensis</em> and synthesized by catalysis of two closely related orcinol <em>O</em>-methyltransferases.<a reveal-id="dsx042-B54" data-open="dsx042-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> Phenolic methyl ethers such as DMT or TMB, the characteristic ‘tea scent’ compounds of tea roses, have an earthy and spicy note.<a reveal-id="dsx042-B54" data-open="dsx042-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> Only one orcinol methyltransferase gene (Rmu_sc0002707.1_g000004.1) was found in the <em>R. multiflora</em> genome, while Chinese roses producing DMT and TMB contain two closely related orcinol methyltransferase (<em>OOMT1</em> and <em>OOMT2</em>) genes.<a reveal-id="dsx042-B54" data-open="dsx042-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> This is consistent with the absence of DMT or TMB in <em>R. multiflora</em>. It has been suggested that <em>R. chinensis</em> OOMT1 contains a tyrosine residue at amino acid 127, whereas OOMT2 has a phenylalanine residue at this position.<a reveal-id="dsx042-B54" data-open="dsx042-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> It has been suggested that OOMT1 does not catalyze 3-methoxy-5-hydroxytoluene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>) due to structural hindrance by the hydroxyl group of the tyrosine residue but OOMT2 does.<a reveal-id="dsx042-B54" data-open="dsx042-B54" class="link link-ref link-reveal xref-bibr"><sup>54</sup></a> The gene Rmu_sc0002707.1_g000004.1 encoding phenylalanine at this position may be indicative of the presence of <em>OOMT2</em> catalyzing methylation of 3-methoxy-5-hydroxytoluene. Terpenoids are the largest floral scent group and are synthesized from prenyl diphosphate precursors by terpene synthases. In particular, monoterpenes (C<sub>10</sub>) such as geraniol or linalool, which are synthesized from geranyl diphosphate (GPP) and have flower-like notes, are major scent components in some modern roses.<a reveal-id="dsx042-B55" data-open="dsx042-B55" class="link link-ref link-reveal xref-bibr"><sup>55</sup></a> Although several genes encoding linalool, nerolidol, or α-pinene synthase homologues were found here, these compounds are not known to produce in <em>R. multiflora</em> flowers. It is interesting that <em>R. multiflora</em> contains two genomic genes encoding nucleoside diphosphate linked some moiety X hydrolase 1 (NUDX1) which is suggested to be involved in an alternative pathway<a reveal-id="dsx042-B5" data-open="dsx042-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> to synthesize geraniol in spite of the absence of geraniol and its derivatives in <em>R. multiflora.</em> Carotenoid cleavage dioxygenase 1 (CCD1) cleaves β-carotene at the 9–10 and the 9′–10′ positions and generates two β-ionones (C<sub>13</sub> product), which has violet-like notes.<a reveal-id="dsx042-B56" data-open="dsx042-B56" class="link link-ref link-reveal xref-bibr"><sup>56</sup></a> The <em>CCD1</em> gene leading to β-ionone was also assigned (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>).</p>                            <h4 scrollto-destination=115733190 id="115733190" class="section-title" >3.2.5. MADS-box genes related to ABCDE model for floral development</h4>
<p>The ABCDE model has been developed for identification of different floral organs, namely sepals, petals, stamens, and carpels, and these organs are categorized as so-called A-class for sepal and petal specification, B-class for petal and stamen specification, C-class for stamen and carpel specification, D-class for carpel and ovule specification, and E-class for sepal, petal, stamen, and carpel specification by homeotic genes (mostly MADS-box-genes).<a reveal-id="dsx042-B57" data-open="dsx042-B57" class="link link-ref link-reveal xref-bibr"><sup>57</sup></a> Until now, 11 MADS-box-genes have been identified from wild and cultivated roses: class A genes <em>RhAP1-1, RhAP1-2</em>, and <em>RhFUL</em><a reveal-id="dsx042-B58" data-open="dsx042-B58" class="link link-ref link-reveal xref-bibr"><sup>58</sup></a>; class B genes <em>MASAKO BP</em>, <em>MASAKO B3</em>, and <em>MASAKO euB3</em><a reveal-id="dsx042-B59" data-open="dsx042-B59" class="link link-ref link-reveal xref-bibr"><sup>59</sup></a><sup>,</sup><a reveal-id="dsx042-B60" data-open="dsx042-B60" class="link link-ref link-reveal xref-bibr"><sup>60</sup></a>; class C/D genes <em>MASAKO C1</em> and <em>MASAKO D1</em><a reveal-id="dsx042-B61" data-open="dsx042-B61" class="link link-ref link-reveal xref-bibr"><sup>61</sup></a>; and class E genes <em>RhSEP3</em>, <em>MASAKO S1</em>, and <em>MASAKO S3.</em><a reveal-id="dsx042-B62" data-open="dsx042-B62" class="link link-ref link-reveal xref-bibr"><sup>62</sup></a><sup>,</sup><a reveal-id="dsx042-B63" data-open="dsx042-B63" class="link link-ref link-reveal xref-bibr"><sup>63</sup></a> Attenuated <em>MASAKO C1</em> (<em>RhAG</em>) expression under low temperature condition causes an additional petal and petaloid stamens formation in cultivated rose,<a reveal-id="dsx042-B64" data-open="dsx042-B64" class="link link-ref link-reveal xref-bibr"><sup>64</sup></a> and severe reduction of <em>MASAKO C1</em> expression was observed sterilized (anemone type) <em>R. luciae</em> flowers by our analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). However, the <em>R</em>. <em>multiflora MASAKO C1</em> (Rmu_sc0003469.1_g000007.1) have an intact open reading frame without a frame shift or transposon insertion. This is consistent with its normal flower phenotype.</p><p><em>R. multiflora</em> contains 94 MADS-box genes, including three class A, four class B, two class C/D, one class D, five class E, and three <em>SHORT VEGETATIVE PHASE</em> (<em>SVP</em>) genes (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&amp;Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S14</a></span>). The 15 class ABCDE genes belonged to two clades consisting of class B, and class A, C/D and E except for one class D (Rmu_sc0000512.1_g000020.1) and one class E (<em>SEP4</em>) (Rmu_sc0003558.1_g000005.1) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&amp;Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>).</p>                            <h4 scrollto-destination=115733193 id="115733193" class="section-title" >3.2.6. Everblooming character</h4>
<p>The everblooming characteristic is one of the most important in modern roses and originated from everblooming sport of <em>R. chinensis</em>. A retrotransposon insertion in the <em>KSN</em> (terminal flower 1 homologue) gene resulted in this characteristic.<a reveal-id="dsx042-B65" data-open="dsx042-B65" class="link link-ref link-reveal xref-bibr"><sup>65</sup></a> Although Rmu_sc0010986.1_g000002.1 protein exhibited two amino acid residue differences from <em>R. chinensis</em> KSN protein and the <em>R. multiflora KSN</em> gene has a shorter first intron and a longer third intron than the <em>R. chinensis KSN</em> gene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>), no retrotransposon insertion is found in the Rmu_sc0010986.1_g000002.1 sequence. This is consistent with once flowering phenotype of <em>R. multiflora</em>.</p>                            <h4 scrollto-destination=115733195 id="115733195" class="section-title" >3.2.7. Genes related to flower opening of rose petals</h4>
<p>Fully extended petals and a long vase life are prerequisites for increasing the ornamental value of rose flowers. For extending petals, expansion of the petal cells plays a pivotal role. Expansins, XTHs and aquaporins participate in this process by loosening the cell wall or mediating influx of water into cells.<a reveal-id="dsx042-B66" data-open="dsx042-B66" class="link link-ref link-reveal xref-bibr"><sup>66</sup></a> Studies in Arabidopsis and other model plants disclose that these three proteins comprise a multigene superfamily. In rose, three aquaporin, four expansin and four XTHs genes have been identified as relevant to the expansion of petal cells.<a reveal-id="dsx042-B67 dsx042-B68 dsx042-B69 dsx042-B70 dsx042-B71" data-open="dsx042-B67 dsx042-B68 dsx042-B69 dsx042-B70 dsx042-B71" class="link link-ref link-reveal xref-bibr"><sup>67–71</sup></a></p><p>BLAST and keyword search identified 65 aquaporins, 47 expansins and 60 XTHs (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&amp;Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S15–17</a></span>). In aquaporins, 55 out of 65 genes were predicted to have at least two transmembrane helix, which is conserved in aquaporin homologues (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&amp;Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S15</a></span>). Through the phylogenetic analysis aquaporins excluding seven genes annotated with partial by Augustus 3.0.3 classified into 17 plasma membrane intrinsic proteins, 11 tonoplast intrinsic proteins, 10 nodulin 26-like intrinsic proteins, 6 small basic intrinsic proteins (SIPs) and 4 X intrinsic proteins (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&amp;Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12 and S15</a></span>). Similarly, expansins annotated with InterProScan accession PR01225 or PR01226 were classified into 3 subfamilies as expected (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&amp;Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&amp;Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S16</a></span>). RNA-Seq analysis predicted that 21 aquaporins, 13 expansins and 22 XTHs were expressed in the flower bud, among which 10 genes show particularly high expression level (RPKM &gt; 50) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&amp;Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs S9–S12</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&amp;Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S15–S17</a></span>). Because seven highly expressed genes were not clustered with known rose genes, it may be interesting to investigate whether these genes actually act as a novel regulator for the expansion of petal cells in rose in the future analysis.</p>                            <h3 scrollto-destination=115733198 id="115733198" class="section-title" >3.3. Comparative genomics</h3>
                            <h4 scrollto-destination=115733199 id="115733199" class="section-title" >3.3.1. Comparison with other rosaceae genomes at the gene level</h4>
<p>The genes predicted in <em>R. multiflora</em>, <em>F. vesca</em>, <em>P. persica</em>, <em>M.</em> × <em>domestica</em> were compared by a clustering method using OrthoMCL.<a reveal-id="dsx042-B44" data-open="dsx042-B44" class="link link-ref link-reveal xref-bibr"><sup>44</sup></a> In <em>R. multiflora</em>, the 67,380 genes were classified into intrinsic and partial genes and were used for this comparison. The number of clusters is shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>. In the common region, 7,665 clusters were included. In the clusters, 18,956, 10,877, 13,187, and 14,069 genes were included for <em>R. multiflora</em>, <em>F. vesca</em>, <em>P. persica</em>, and <em>M.</em> × <em>domestica</em>, respectively. The number of clusters shared in common in <em>R. multiflora</em> and <em>F. vesca</em>, <em>R. multiflora</em> and <em>P. persica</em>, and <em>R. multiflora</em> and <em>M.</em> × <em>domestica</em> were 1,287, 904, and 241, respectively. This indicates that <em>R. multiflora</em> is closely related to <em>F. vesca</em>, <em>P. persica</em>, and <em>M.</em> × <em>domestica</em> in a stepwise manner. The distances corresponded well to the phylogenetic relationship in Rosaceae reported by Xiang et al.<a reveal-id="dsx042-B72" data-open="dsx042-B72" class="link link-ref link-reveal xref-bibr"><sup>72</sup></a> The number of clusters uniquely found in <em>R. multiflora</em> were 2.5 times (3,482 in <em>R. multiflora</em>/1,397 in <em>F. vesca</em>) higher than that in <em>F. vesca</em>. This was due to the heterozygosity in <em>R. multiflora</em>. In addition, the number of genes in the clusters uniquely found in <em>R. multiflora</em> was 3.3 times (14,663 in <em>R. multiflora</em>/4,482 in <em>F. vesca</em>) higher than that in <em>F. vesca</em>, which means that the duplicated or partial genes were included in <em>R. multiflora</em> more than in <em>F. vesca</em>.</p>                            <h4 scrollto-destination=115733201 id="115733201" class="section-title" >3.3.2. Comparison with the apple genome at the pseudomolecule level</h4>
<p>To investigate possible syntenic relationships among <em>R. multiflora</em> and other rosaceous taxa genomes, the status of conservation of relative gene positions was surveyed using the scaffolds of rose genomic sequences. Among the 3,932 scaffolds with five or more predicted genes, conservation of the relative positions of three or more genes was observed in 1,968 scaffolds (50%) and 2,312 scaffolds (59%) against genes predicted in the <em>F. vesca</em> and <em>P. persica</em> genomic sequences, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S18</a></span>). It appears that a significant degree of micro-synteny can be expected within the family Rosaceae.</p><p>By anchoring scaffolds of rose genome with SSR marker information in the recent integrated genetic map of roses,<a reveal-id="dsx042-B73 dsx042-B74 dsx042-B75" data-open="dsx042-B73 dsx042-B74 dsx042-B75" class="link link-ref link-reveal xref-bibr"><sup>73–75</sup></a> macro-syntenic relationships between <em>R. multiflora</em> and other <em>Rosaceae</em> genomes were investigated. A total of 160 scaffolds, with 17.9 Mb total length, were anchored on the seven linkage groups of <em>R. multiflora</em>. Among the 127 markers with corresponding scaffolds, 33 markers (26%) correspond to two scaffolds, which presumably indicated the presence of redundant scaffolds corresponding to heterozygous alleles (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S19</a></span>). The micro-syntenic relations identified on the anchored scaffolds revealed macro-level synteny between <em>R. multiflora</em> and <em>F. vesca</em>, with entire chromosome level synteny between <em>R. multiflora</em> linkage group (RG) 1 and <em>F. vesca</em> chromosome (FC)7, RG4 and FC4, RG5 and FC3, RG6 and FC2, and RG7 and FC5. A single large translocation was evident, as scaffolds anchored on RG2 showed syntenic relation to either FC1 or the top portion of FC6 and those on RG3 showed syntenic relation to bottom portion of FC6. Macro-syntenic relationships identified between <em>R. multiflora</em> and <em>P. persica</em> corresponded well to the syntenic relationship between <em>F. vesca</em> and <em>P. persica</em>,<a reveal-id="dsx042-B8" data-open="dsx042-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> that is, RG1 and the <em>P. persica</em> linkage group (PG)2, RG2-PG3/PG7, RG3-PG6, RG4-PG1, PG5-PG4/PG6, PG6-PG1/PG8, and PG7PG1/PG5.</p>                            <h4 scrollto-destination=115733204 id="115733204" class="section-title" >3.3.3. SNP analysis of wild and cultivated roses</h4>
<p>To examine molecular similarities among wild rose, <em>R. multiflora</em>, and cultivated rose, <em>R. hybrida</em>, transcriptome reads of <em>R. hybrida</em> cultivar ‘Rote Rose’ were mapped to our <em>R. multiflora</em> genome sequence. A total of 55,086 <em>R. multiflora</em> genes were mapped by 20 or more <em>R. hybrida</em> transcriptome reads, and 198,807 SNPs were identified on these mapped regions, corresponding to 201 Mb (∼27%) of the genome. Although the average SNP density could be calculated as one in every 1.01 kb, a higher SNP density might be expected across the entire genome if nontranscribed regions were also included in this analysis.</p>                            <h3 scrollto-destination=115733206 id="115733206" class="section-title" >3.4. Databases</h3>
<p>The genomic and gene sequences and their annotation are available at ‘<em>Rosa multiflora</em> Genome DataBase (<a class="link link-uri" href="http://rosa.kazusa.or.jp" target="_blank">http://rosa.kazusa.or.jp</a>).’ Users can input query sequences to perform BLAST searches against the genomic and gene sequences (transcripts, CDSs, and proteins) on the ‘BLAST’ page. Keyword search of the BLAST search result against NR and TAIR10 pep for each gene is available on the ‘KEYWORD’ page. At the ‘DOWNLOAD’ page, data for the genomic and gene (cds, pep, and transcripts) sequences, annotation file (gff3 format), and the InterProScan search results (raw format) can be downloaded.</p>                            <h2 scrollto-destination=115733208 id="115733208" class="section-title" >Availability of data</h2>
<p>The Bioproject identifier for <em>R. multiflora</em> is PRJDB4738. The accession numbers of the assembled genome sequences of RMU_r2.0 are BDJD01000001-BDJD01083189 (83,189 entries). The SRA accession numbers for the Illumina reads (HiSeq and MiSeq) used in this study are summarized in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>.</p>                            <h2 scrollto-destination=115733210 id="115733210" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors are grateful to Professor Ueda of Gifu International Academy of Horticulture for his valuable suggestions about the lineage of <em>R. hybrida</em> and to Mr. Takeuchi of Keisei Rose Nurseries for providing the <em>R. multiflora</em> plant. The authors are indebted to Dr. Katsumoto and Ms. Nakajima for their kind encouragement during this study. Mr. Ito and Mses. Nakata, Yamauchi, Morimoto, and Akahoshi are acknowledged for their technical assistance.</p>                            <h3 scrollto-destination=115733212 id="115733212" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=115733214 id="115733214" class="section-title" >Funding</h3>
<p>This work was supported by the Kazusa DNA Research Institute Foundation.</p>                            <h3 scrollto-destination=115733216 id="115733216" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=115733218 id="115733218" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx042-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wylie</div> <div class="given-names">A.P.</div></div></div> <div class="year">1954</div>, 
<div class="article-title">The history of garden roses. Part I</div>, <div class="source">J. Roy. Hort. Soc</div> ., <div class="volume">79</div>, <div class="fpage">555</div>–<div class="lpage">71</div>.</div></div></div></div><div content-id="dsx042-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wylie</div> <div class="given-names">A.P.</div></div></div> <div class="year">1954</div>, 
<div class="article-title">The history of garden roses. Part II</div>, <div class="source">J. Roy. Hort. Soc</div> ., <div class="volume">80</div>, <div class="fpage">8</div>–<div class="lpage">24</div>.</div></div></div></div><div content-id="dsx042-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wylie</div> <div class="given-names">A.P.</div></div></div> <div class="year">1954</div>, 
<div class="article-title">The history of garden roses. Part III</div>, <div class="source">J. Roy. Hort. Soc</div> <em>.</em>, <div class="volume">80</div>, <div class="fpage">77</div>–<div class="lpage">87</div>.</div></div></div></div><div content-id="dsx042-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ogata</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Kanno</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Itoh</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Tsugawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">M.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Plant biochemistry: anthocyanin biosynthesis in roses</div>, <div class="source">Nature</div> , <div class="volume">435</div>, <div class="fpage">757</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20biochemistry%3A%20anthocyanin%20biosynthesis%20in%20roses&amp;author=J.%20Ogata&amp;author=Y.%20Kanno&amp;author=Y.%20Itoh&amp;author=H.%20Tsugawa&amp;author=M.%20Suzuki&amp;publication_year=2005&amp;journal=Nature&amp;volume=435&amp;pages=757-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature435757a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature435757a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature435757a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15944692" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Magnard</div> <div class="given-names">J.L.</div></div>, <div class="name string-name"><div class="surname">Roccia</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Caissard</div> <div class="given-names">J.C.</div></div></div>, <div class="collab">PLANT VOLATILES</div>., et al.   <div class="year">2015</div>, 
<div class="article-title">Biosynthesis of monoterpene scent compounds in roses</div>, <div class="source">Science</div> , <div class="volume">349</div>, <div class="fpage">81</div>–<div class="lpage">3</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biosynthesis%20of%20monoterpene%20scent%20compounds%20in%20roses&amp;author=J.L.%20Magnard&amp;author=A.%20Roccia&amp;author=J.C.%20Caissard&amp;publication_year=2015&amp;journal=Science&amp;volume=349&amp;pages=81-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.aab0696" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.aab0696" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.aab0696"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26138978" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hirata</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Ohnishi</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Tomida</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Seasonal induction of alternative principal pathway for rose flower scent</div>, <div class="source">Sci. Rep</div> ., <div class="volume">6</div>, <div class="fpage">20234</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Seasonal%20induction%20of%20alternative%20principal%20pathway%20for%20rose%20flower%20scent&amp;author=H.%20Hirata&amp;author=T.%20Ohnishi&amp;author=K.%20Tomida&amp;publication_year=2016&amp;journal=Sci.%20Rep&amp;volume=6&amp;pages=20234" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/srep20234" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fsrep20234" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fsrep20234"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26831950" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Velasco</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Zharkikh</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Affourtit</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">The genome of the domesticated apple (<em>Malus</em> × <em>domestica</em> Borkh.)</div>, <div class="source">Nat. Genet</div> ., <div class="volume">42</div>, <div class="fpage">833</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20domesticated%20apple%20(Malus%20%C3%97%20domestica%20Borkh.)&amp;author=R.%20Velasco&amp;author=A.%20Zharkikh&amp;author=J.%20Affourtit&amp;publication_year=2010&amp;journal=Nat.%20Genet&amp;volume=42&amp;pages=833-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.654" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.654" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.654"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20802477" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shulaev</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Sargent</div> <div class="given-names">D.J.</div></div>, <div class="name string-name"><div class="surname">Crowhurst</div> <div class="given-names">R.N.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">The genome of woodland strawberry (<em>Fragaria vesca</em>)</div>, <div class="source">Nat. Genet</div> ., <div class="volume">43</div>, <div class="fpage">109</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20woodland%20strawberry%20(Fragaria%20vesca)&amp;author=V.%20Shulaev&amp;author=D.J.%20Sargent&amp;author=R.N.%20Crowhurst&amp;publication_year=2011&amp;journal=Nat.%20Genet&amp;volume=43&amp;pages=109-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.740" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.740" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.740"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21186353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Sun</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The genome of <em>Prunus mume</em></div>, <div class="source">Nat. Commun</div> ., <div class="volume">3</div>, <div class="fpage">1318</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20Prunus%20mume&amp;author=Q.%20Zhang&amp;author=W.%20Chen&amp;author=L.%20Sun&amp;publication_year=2012&amp;journal=Nat.%20Commun&amp;volume=3&amp;pages=1318" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ncomms2290" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fncomms2290" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fncomms2290"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23271652" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Verde</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Abbott</div> <div class="given-names">A.G.</div></div>, <div class="name string-name"><div class="surname">Scalabrin</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The high-quality draft genome of peach (<em>Prunus persica</em>) identifies unique patterns of genetic diversity, domestication and genome evolution</div>, <div class="source">Nat. Genet</div> ., <div class="volume">45</div>, <div class="fpage">487</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20high-quality%20draft%20genome%20of%20peach%20(Prunus%20persica)%20identifies%20unique%20patterns%20of%20genetic%20diversity%2C%20domestication%20and%20genome%20evolution&amp;author=I.%20Verde&amp;author=A.G.%20Abbott&amp;author=S.%20Scalabrin&amp;publication_year=2013&amp;journal=Nat.%20Genet&amp;volume=45&amp;pages=487-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2586" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2586" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2586"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23525075" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Shi</div> <div class="given-names">Z.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The genome of the pear (<em>Pyrus bretschneideri</em> Rehd.)</div>, <div class="source">Genome Res</div> ., <div class="volume">23</div>, <div class="fpage">396</div>–<div class="lpage">408</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20of%20the%20pear%20(Pyrus%20bretschneideri%20Rehd.)&amp;author=J.%20Wu&amp;author=Z.%20Wang&amp;author=Z.%20Shi&amp;publication_year=2013&amp;journal=Genome%20Res&amp;volume=23&amp;pages=396-408" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.144311.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.144311.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.144311.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23149293" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chagne</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Crowhurst</div> <div class="given-names">R.N.</div></div>, <div class="name string-name"><div class="surname">Pindo</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">The draft genome sequence of European pear (<em>Pyrus communis</em> L. ′Bartlett′)</div>, <div class="source">PLoS One</div> ., <div class="volume">9</div>, <div class="fpage">e92644</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20draft%20genome%20sequence%20of%20European%20pear%20(Pyrus%20communis%20L.%20%E2%80%B2Bartlett%E2%80%B2)&amp;author=D.%20Chagne&amp;author=R.N.%20Crowhurst&amp;author=M.%20Pindo&amp;publication_year=2014&amp;journal=PLoS%20One&amp;volume=9&amp;pages=e92644" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0092644" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0092644" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0092644"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24699266" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lu</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">An</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Jain</div> <div class="given-names">M.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Genome survey sequencing for the characterization of the genetic background of <em>Rosa roxburghii</em> tratt and leaf ascorbate metabolism genes</div>, <div class="source">PLoS One</div> , <div class="volume">11</div>, <div class="fpage">e0147530</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20survey%20sequencing%20for%20the%20characterization%20of%20the%20genetic%20background%20of%20Rosa%20roxburghii%20tratt%20and%20leaf%20ascorbate%20metabolism%20genes&amp;author=M.%20Lu&amp;author=H.%20An&amp;author=L.%20Li&amp;author=M.%20Jain&amp;publication_year=2016&amp;journal=PLoS%20One&amp;volume=11&amp;pages=e0147530" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0147530" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0147530" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0147530"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26849133" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Foucher</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Hibrand-Saint Oyant</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Hamama</div> <div class="given-names">L.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Towards the rose genome sequence and its use in research and breeding</div>, <div class="source">Acta Hortic</div> , <div class="volume">1064</div>, <div class="fpage">167</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Towards%20the%20rose%20genome%20sequence%20and%20its%20use%20in%20research%20and%20breeding&amp;author=F.%20Foucher&amp;author=L.%20Hibrand-Saint%20Oyant&amp;author=L.%20Hamama&amp;publication_year=2015&amp;journal=Acta%20Hortic&amp;volume=1064&amp;pages=167-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.17660/ActaHortic.2015.1064.19" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.17660%2FActaHortic.2015.1064.19" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.17660%2FActaHortic.2015.1064.19"> </span></div> </div></div></div></div></div><div content-id="dsx042-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hurst</div> <div class="given-names">C.C.</div></div></div> <div class="year">1941</div>, 
<div class="article-title">Notes on the origin and evolution of our garden roses</div>, <div class="source">J. Roy. Hort. Soc</div> ., <div class="volume">66</div>, <div class="fpage">73</div>–<div class="lpage">82</div>.</div></div></div></div><div content-id="dsx042-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Terefe-Ayana</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Yasmin</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Le</div> <div class="given-names">T. L.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Mining disease-resistance genes in roses: functional and molecular characterization of the <em>Rdr1</em> locus</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">2</div>, <div class="fpage">35</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mining%20disease-resistance%20genes%20in%20roses%3A%20functional%20and%20molecular%20characterization%20of%20the%20Rdr1%20locus&amp;author=D.%20Terefe-Ayana&amp;author=A.%20Yasmin&amp;author=T.%20L.%20Le&amp;publication_year=2011&amp;journal=Front.%20Plant%20Sci&amp;volume=2&amp;pages=35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fpls.2011.00035" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffpls.2011.00035" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffpls.2011.00035"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22639591" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dickson</div> <div class="given-names">E.E.</div></div>, <div class="name string-name"><div class="surname">Arumuganathan</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kresovich</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Doyle</div> <div class="given-names">J.J.</div></div></div> <div class="year">1992</div>, 
<div class="article-title">Nuclear DNA content variation within the Rosaceae</div>, <div class="source">Am. J. Bot</div> ., <div class="volume">79</div>, <div class="fpage">1081</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nuclear%20DNA%20content%20variation%20within%20the%20Rosaceae&amp;author=E.E.%20Dickson&amp;author=K.%20Arumuganathan&amp;author=S.%20Kresovich&amp;author=J.J.%20Doyle&amp;publication_year=1992&amp;journal=Am.%20J.%20Bot&amp;volume=79&amp;pages=1081-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/ajb2.1992.79.issue-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fajb2.1992.79.issue-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fajb2.1992.79.issue-9"> </span></div> </div></div></div></div></div><div content-id="dsx042-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Vukosavljev</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Arens</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Voorrips</div> <div class="given-names">R.E.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">High-density SNP-based genetic maps for the parents of an outcrossed and a selfed tetraploid garden rose cross, inferred from admixed progeny using the 68k rose SNP array</div>, <div class="source">Hortic. Res</div> ., <div class="volume">3</div>, <div class="fpage">16052</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-density%20SNP-based%20genetic%20maps%20for%20the%20parents%20of%20an%20outcrossed%20and%20a%20selfed%20tetraploid%20garden%20rose%20cross%2C%20inferred%20from%20admixed%20progeny%20using%20the%2068k%20rose%20SNP%20array&amp;author=M.%20Vukosavljev&amp;author=P.%20Arens&amp;author=R.E.%20Voorrips&amp;publication_year=2016&amp;journal=Hortic.%20Res&amp;volume=3&amp;pages=16052" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/hortres.2016.52" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fhortres.2016.52" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fhortres.2016.52"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27818777" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Katsumoto</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Fukuchi-Mizutani</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Fukui</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Engineering of the rose flavonoid biosynthetic pathway successfully generated blue-hued flowers accumulating delphinidin</div>, <div class="source">Plant Cell Physiol</div> ., <div class="volume">48</div>, <div class="fpage">1589</div>–<div class="lpage">600</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Engineering%20of%20the%20rose%20flavonoid%20biosynthetic%20pathway%20successfully%20generated%20blue-hued%20flowers%20accumulating%20delphinidin&amp;author=Y.%20Katsumoto&amp;author=M.%20Fukuchi-Mizutani&amp;author=Y.%20Fukui&amp;publication_year=2007&amp;journal=Plant%20Cell%20Physiol&amp;volume=48&amp;pages=1589-600" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcm131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcm131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcm131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17925311" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Leggett</div> <div class="given-names">R.M.</div></div>, <div class="name string-name"><div class="surname">Ramirez-Gonzalez</div> <div class="given-names">R.H.</div></div>, <div class="name string-name"><div class="surname">Clavijo</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Waite</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Davey</div> <div class="given-names">R.P.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Sequencing quality assessment tools to enable data-driven informatics for high throughput genomics</div>, <div class="source">Front. Genet</div> ., <div class="volume">4</div>, <div class="fpage">288</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20quality%20assessment%20tools%20to%20enable%20data-driven%20informatics%20for%20high%20throughput%20genomics&amp;author=R.M.%20Leggett&amp;author=R.H.%20Ramirez-Gonzalez&amp;author=B.J.%20Clavijo&amp;author=D.%20Waite&amp;author=R.P.%20Davey&amp;publication_year=2013&amp;journal=Front.%20Genet&amp;volume=4&amp;pages=288" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.3389/fgene.2013.00288" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.3389%2Ffgene.2013.00288" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.3389%2Ffgene.2013.00288"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24381581" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Schmieder</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Edwards</div> <div class="given-names">R.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Quality control and preprocessing of metagenomic datasets</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">863</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quality%20control%20and%20preprocessing%20of%20metagenomic%20datasets&amp;author=R.%20Schmieder&amp;author=R.%20Edwards&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=863-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr026"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21278185" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Marçais</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Kingsford</div> <div class="given-names">C.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">A fast, lock-free approach for efficient parallel counting of occurrences of k-mers</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">764</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20fast%2C%20lock-free%20approach%20for%20efficient%20parallel%20counting%20of%20occurrences%20of%20k-mers&amp;author=G.%20Mar%C3%A7ais&amp;author=C.%20Kingsford&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=764-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217122" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Shirasawa</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kosugi</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Dissection of the octoploid strawberry genome by deep sequencing of the genomes of <em>Fragaria</em> species</div>, <div class="source">DNA Res</div> ., <div class="volume">21</div>, <div class="fpage">169</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dissection%20of%20the%20octoploid%20strawberry%20genome%20by%20deep%20sequencing%20of%20the%20genomes%20of%20Fragaria%20species&amp;author=H.%20Hirakawa&amp;author=K.%20Shirasawa&amp;author=S.%20Kosugi&amp;publication_year=2014&amp;journal=DNA%20Res&amp;volume=21&amp;pages=169-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dst049" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdst049" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdst049"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24282021" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Ruan</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title"><em>De novo</em> assembly of human genomes with massively parallel short read sequencing</div>, <div class="source">Genome Res</div> ., <div class="volume">20</div>, <div class="fpage">265</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20human%20genomes%20with%20massively%20parallel%20short%20read%20sequencing&amp;author=R.%20Li&amp;author=H.%20Zhu&amp;author=J.%20Ruan&amp;publication_year=2010&amp;journal=Genome%20Res&amp;volume=20&amp;pages=265-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.097261.109" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.097261.109" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.097261.109"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20019144" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Grabherr</div> <div class="given-names">M.G.</div></div>, <div class="name string-name"><div class="surname">Haas</div> <div class="given-names">B.J.</div></div>, <div class="name string-name"><div class="surname">Yassour</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Full-length transcriptome assembly from RNA-Seq data without a reference genome</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">29</div>, <div class="fpage">644</div>–<div class="lpage">52</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Full-length%20transcriptome%20assembly%20from%20RNA-Seq%20data%20without%20a%20reference%20genome&amp;author=M.G.%20Grabherr&amp;author=B.J.%20Haas&amp;author=M.%20Yassour&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol&amp;volume=29&amp;pages=644-52" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1883" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1883" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1883"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21572440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Dewey</div> <div class="given-names">C.N.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">RSEM: accurate transcript quantification from RNA-Seq data with or without a reference genome</div>, <div class="source">BMC Bioinform</div> ., <div class="volume">12</div>, <div class="fpage">323</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RSEM%3A%20accurate%20transcript%20quantification%20from%20RNA-Seq%20data%20with%20or%20without%20a%20reference%20genome&amp;author=B.%20Li&amp;author=C.N.%20Dewey&amp;publication_year=2011&amp;journal=BMC%20Bioinform&amp;volume=12&amp;pages=323" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-12-323" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-12-323" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-12-323"> </span></div> </div></div></div></div></div><div content-id="dsx042-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kent</div> <div class="given-names">W.J.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">BLAT–the BLAST-like alignment tool</div>, <div class="source">Genome Res</div> ., <div class="volume">12</div>, <div class="fpage">656</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BLAT%E2%80%93the%20BLAST-like%20alignment%20tool&amp;author=W.J.%20Kent&amp;publication_year=2002&amp;journal=Genome%20Res&amp;volume=12&amp;pages=656-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.229202" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.229202" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.229202"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11932250" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xue</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">J.T.</div></div>, <div class="name string-name"><div class="surname">Zhu</div> <div class="given-names">Y.P.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">L_RNA_scaffolder: scaffolding genomes with transcripts</div>, <div class="source">BMC Genom</div> ., <div class="volume">14</div>, <div class="fpage">604</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=L_RNA_scaffolder%3A%20scaffolding%20genomes%20with%20transcripts&amp;author=W.%20Xue&amp;author=J.T.%20Li&amp;author=Y.P.%20Zhu&amp;publication_year=2013&amp;journal=BMC%20Genom&amp;volume=14&amp;pages=604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-14-604" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-14-604" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-14-604"> </span></div> </div></div></div></div></div><div content-id="dsx042-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Price</div> <div class="given-names">A.L.</div></div>, <div class="name string-name"><div class="surname">Jones</div> <div class="given-names">N.C.</div></div>, <div class="name string-name"><div class="surname">Pevzner</div> <div class="given-names">P.A.</div></div></div> <div class="year">2005</div>, 
<div class="article-title"><em>De novo</em> identification of repeat families in large genomes</div>, <div class="source">Bioinformatics</div> , <div class="volume">21</div>, <div class="fpage">i351</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20identification%20of%20repeat%20families%20in%20large%20genomes&amp;author=A.L.%20Price&amp;author=N.C.%20Jones&amp;author=P.A.%20Pevzner&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=i351-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti1018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti1018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti1018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15961478" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pachter</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">TopHat: discovering splice junctions with RNA-Seq</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1105</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat%3A%20discovering%20splice%20junctions%20with%20RNA-Seq&amp;author=C.%20Trapnell&amp;author=L.%20Pachter&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19289445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Goff</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks</div>, <div class="source">Nat. Protoc</div> ., <div class="volume">7</div>, <div class="fpage">562</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20gene%20and%20transcript%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20TopHat%20and%20Cufflinks&amp;author=C.%20Trapnell&amp;author=A.%20Roberts&amp;author=L.%20Goff&amp;publication_year=2012&amp;journal=Nat.%20Protoc&amp;volume=7&amp;pages=562-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2012.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2012.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2012.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22383036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Handsaker</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Wysoker</div> <div class="given-names">A.</div></div></div>, <div class="collab">1000 Genome Project Data Processing Subgroup</div>., et al.   <div class="year">2009</div>, 
<div class="article-title">The sequence alignment/map format and SAMtools</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">2078</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20sequence%20alignment%2Fmap%20format%20and%20SAMtools&amp;author=H.%20Li&amp;author=B.%20Handsaker&amp;author=A.%20Wysoker&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=2078-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp352" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp352" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp352"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19505943" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Parra</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Bradnam</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Korf</div> <div class="given-names">I.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">CEGMA: a pipeline to accurately annotate core genes in eukaryotic genomes</div>, <div class="source">Bioinformatics</div> , <div class="volume">23</div>, <div class="fpage">1061</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CEGMA%3A%20a%20pipeline%20to%20accurately%20annotate%20core%20genes%20in%20eukaryotic%20genomes&amp;author=G.%20Parra&amp;author=K.%20Bradnam&amp;author=I.%20Korf&amp;publication_year=2007&amp;journal=Bioinformatics&amp;volume=23&amp;pages=1061-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btm071" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtm071" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtm071"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17332020" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Simao</div> <div class="given-names">F.A.</div></div>, <div class="name string-name"><div class="surname">Waterhouse</div> <div class="given-names">R.M.</div></div>, <div class="name string-name"><div class="surname">Ioannidis</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Kriventseva</div> <div class="given-names">E.V.</div></div>, <div class="name string-name"><div class="surname">Zdobnov</div> <div class="given-names">E.M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">BUSCO: assessing genome assembly and annotation completeness with single-copy orthologs</div>, <div class="source">Bioinformatics</div> , <div class="volume">31</div>, <div class="fpage">3210</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BUSCO%3A%20assessing%20genome%20assembly%20and%20annotation%20completeness%20with%20single-copy%20orthologs&amp;author=F.A.%20Simao&amp;author=R.M.%20Waterhouse&amp;author=P.%20Ioannidis&amp;author=E.V.%20Kriventseva&amp;author=E.M.%20Zdobnov&amp;publication_year=2015&amp;journal=Bioinformatics&amp;volume=31&amp;pages=3210-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv351" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv351" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv351"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26059717" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hoff</div> <div class="given-names">K.J.</div></div>, <div class="name string-name"><div class="surname">Lange</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Lomsadze</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">BRAKER1: unsupervised RNA-seq-based genome annotation with GeneMark-ET and AUGUSTUS</div>, <div class="source">Bioinformatics</div> , <div class="volume">32</div>, <div class="fpage">767</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BRAKER1%3A%20unsupervised%20RNA-seq-based%20genome%20annotation%20with%20GeneMark-ET%20and%20AUGUSTUS&amp;author=K.J.%20Hoff&amp;author=S.%20Lange&amp;author=A.%20Lomsadze&amp;author=M.%20Borodovsky&amp;author=M.%20Stanke&amp;publication_year=2016&amp;journal=Bioinformatics&amp;volume=32&amp;pages=767-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btv661" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtv661" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtv661"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26559507" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lomsadze</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Burns</div> <div class="given-names">P.D.</div></div>, <div class="name string-name"><div class="surname">Borodovsky</div> <div class="given-names">M.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Integration of mapped RNA-Seq reads into automatic training of eukaryotic gene finding algorithm</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">e119</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integration%20of%20mapped%20RNA-Seq%20reads%20into%20automatic%20training%20of%20eukaryotic%20gene%20finding%20algorithm&amp;author=A.%20Lomsadze&amp;author=P.D.%20Burns&amp;author=M.%20Borodovsky&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=e119" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku557" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku557" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku557"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24990371" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Waack</div> <div class="given-names">S.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Gene prediction with a hidden Markov model and a new intron submodel</div>, <div class="source">Bioinformatics</div> , <div class="volume">19</div>, <div class="fpage">ii215</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20prediction%20with%20a%20hidden%20Markov%20model%20and%20a%20new%20intron%20submodel&amp;author=M.%20Stanke&amp;author=S.%20Waack&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=ii215-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg1080" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg1080" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg1080"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14534192" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quevillon</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Silventoinen</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Pillai</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">InterProScan: protein domains identifier</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">33</div>, <div class="fpage">W116</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=InterProScan%3A%20protein%20domains%20identifier&amp;author=E.%20Quevillon&amp;author=V.%20Silventoinen&amp;author=S.%20Pillai&amp;publication_year=2005&amp;journal=Nucleic%20Acids%20Res&amp;volume=33&amp;pages=W116-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gki442" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgki442" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgki442"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15980438" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Llorens</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Futami</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Covelli</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">The Gypsy Database (GyDB) of mobile genetic elements: release 2.0</div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">39</div>, <div class="fpage">D70</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Gypsy%20Database%20(GyDB)%20of%20mobile%20genetic%20elements%3A%20release%202.0&amp;author=C.%20Llorens&amp;author=R.%20Futami&amp;author=L.%20Covelli&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res&amp;volume=39&amp;pages=D70-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq1061" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq1061" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq1061"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21036865" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B40" class="ref-content "><div class="label -label">40</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">A new generation of homology search tools based on probabilistic inference</div>, <div class="source">Genome Inform</div> ., <div class="volume">23</div>, <div class="fpage">205</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20new%20generation%20of%20homology%20search%20tools%20based%20on%20probabilistic%20inference&amp;author=S.R.%20Eddy&amp;publication_year=2009&amp;journal=Genome%20Inform&amp;volume=23&amp;pages=205-11" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20180275" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B41" class="ref-content "><div class="label -label">41</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Lowe</div> <div class="given-names">T.M.</div></div>, <div class="name string-name"><div class="surname">Eddy</div> <div class="given-names">S.R.</div></div></div> <div class="year">1997</div>, 
<div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucl. Acids Res</div> ., <div class="volume">25</div>, <div class="fpage">955</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.M.%20Lowe&amp;author=S.R.%20Eddy&amp;publication_year=1997&amp;journal=Nucl.%20Acids%20Res&amp;volume=25&amp;pages=955-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div> </div></div></div></div></div><div content-id="dsx042-B42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B42" class="ref-content "><div class="label -label">42</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Huson</div> <div class="given-names">D.H.</div></div>, <div class="name string-name"><div class="surname">Scornavacca</div> <div class="given-names">C.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Dendroscope 3: an interactive tool for rooted phylogenetic trees and networks</div>, <div class="source">Syst. Biol</div> ., <div class="volume">61</div>, <div class="fpage">1061</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dendroscope%203%3A%20an%20interactive%20tool%20for%20rooted%20phylogenetic%20trees%20and%20networks&amp;author=D.H.%20Huson&amp;author=C.%20Scornavacca&amp;publication_year=2012&amp;journal=Syst.%20Biol&amp;volume=61&amp;pages=1061-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/sysbio/sys062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fsysbio%2Fsys062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fsysbio%2Fsys062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22780991" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B43" class="ref-content "><div class="label -label">43</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tatusov</div> <div class="given-names">R.L.</div></div>, <div class="name string-name"><div class="surname">Fedorova</div> <div class="given-names">N.D.</div></div>, <div class="name string-name"><div class="surname">Jackson</div> <div class="given-names">J.D.</div></div></div>, et al.   <div class="year">2003</div>, 
<div class="article-title">The COG database: an updated version includes eukaryotes</div>, <div class="source">BMC Bioinform</div> ., <div class="volume">4</div>, <div class="fpage">41</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20COG%20database%3A%20an%20updated%20version%20includes%20eukaryotes&amp;author=R.L.%20Tatusov&amp;author=N.D.%20Fedorova&amp;author=J.D.%20Jackson&amp;publication_year=2003&amp;journal=BMC%20Bioinform&amp;volume=4&amp;pages=41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2105-4-41" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2105-4-41" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2105-4-41"> </span></div> </div></div></div></div></div><div content-id="dsx042-B44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B44" class="ref-content "><div class="label -label">44</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Stoeckert</div> <div class="given-names">C.J.J.</div></div>, <div class="name string-name"><div class="surname">Roos</div> <div class="given-names">D.S.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">OrthoMCL: identification of ortholog groups for eukaryotic genomes</div>, <div class="source">Genome Res</div> ., <div class="volume">13</div>, <div class="fpage">2178</div>–<div class="lpage">89</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=OrthoMCL%3A%20identification%20of%20ortholog%20groups%20for%20eukaryotic%20genomes&amp;author=L.%20Li&amp;author=C.J.J.%20Stoeckert&amp;author=D.S.%20Roos&amp;publication_year=2003&amp;journal=Genome%20Res&amp;volume=13&amp;pages=2178-89" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.1224503" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.1224503" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.1224503"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12952885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B45" class="ref-content "><div class="label -label">45</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kurtz</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Phillippy</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Delcher</div> <div class="given-names">A.L.</div></div></div>, et al.   <div class="year">2004</div>, 
<div class="article-title">Versatile and open software for comparing large genomes</div>, <div class="source">Genome Biol</div> ., <div class="volume">5</div>, <div class="fpage">R12</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Versatile%20and%20open%20software%20for%20comparing%20large%20genomes&amp;author=S.%20Kurtz&amp;author=A.%20Phillippy&amp;author=A.L.%20Delcher&amp;publication_year=2004&amp;journal=Genome%20Biol&amp;volume=5&amp;pages=R12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2004-5-2-r12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2004-5-2-r12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2004-5-2-r12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14759262" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B46" class="ref-content "><div class="label -label">46</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Cheng</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Zhao</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">A genomic analysis of disease-resistance genes encoding nucleotide binding sites in <em>Sorghum bicolor</em></div>, <div class="source">Genet. Mol. Biol</div> ., <div class="volume">33</div>, <div class="fpage">292</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20genomic%20analysis%20of%20disease-resistance%20genes%20encoding%20nucleotide%20binding%20sites%20in%20Sorghum%20bicolor&amp;author=X.%20Cheng&amp;author=H.%20Jiang&amp;author=Y.%20Zhao&amp;publication_year=2010&amp;journal=Genet.%20Mol.%20Biol&amp;volume=33&amp;pages=292-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1590/S1415-47572010005000036" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1590%2FS1415-47572010005000036" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1590%2FS1415-47572010005000036"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21637485" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B47" class="ref-content "><div class="label -label">47</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Stanke</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Diekhans</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Baertsch</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Haussler</div> <div class="given-names">D.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Using native and syntenically mapped cDNA alignments to improve de novo gene finding</div>, <div class="source">Bioinformatics,</div>  <div class="volume">24</div>, <div class="fpage">637</div>–<div class="lpage">44</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20native%20and%20syntenically%20mapped%20cDNA%20alignments%20to%20improve%20de%20novo%20gene%20finding&amp;author=M.%20Stanke&amp;author=M.%20Diekhans&amp;author=R.%20Baertsch&amp;author=D.%20Haussler&amp;publication_year=2008&amp;journal=Bioinformatics%2C&amp;volume=24&amp;pages=637-44" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn013" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn013" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn013"> </span></div> </div></div></div></div></div><div content-id="dsx042-B48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B48" class="ref-content "><div class="label -label">48</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kajitani</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Toshimoto</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Noguchi</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Efficient de novo assembly of highly heterozygous genomes from whole-genome shotgun short reads</div>, <div class="source">Genome Res</div> ., <div class="volume">24</div>, <div class="fpage">1384</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efficient%20de%20novo%20assembly%20of%20highly%20heterozygous%20genomes%20from%20whole-genome%20shotgun%20short%20reads&amp;author=R.%20Kajitani&amp;author=K.%20Toshimoto&amp;author=H.%20Noguchi&amp;publication_year=2014&amp;journal=Genome%20Res&amp;volume=24&amp;pages=1384-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.170720.113" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.170720.113" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.170720.113"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24755901" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B49" class="ref-content "><div class="label -label">49</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tanaka</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Sasaki</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Ohmiya</div> <div class="given-names">A.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Plant pigments for coloration: anthocyanins, betalains and carotenoids</div>, <div class="source">Plant J</div> ., <div class="volume">54</div>, <div class="fpage">733</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20pigments%20for%20coloration%3A%20anthocyanins%2C%20betalains%20and%20carotenoids&amp;author=Y.%20Tanaka&amp;author=N.%20Sasaki&amp;author=A.%20Ohmiya&amp;publication_year=2008&amp;journal=Plant%20J&amp;volume=54&amp;pages=733-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1365-313X.2008.03447.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1365-313X.2008.03447.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1365-313X.2008.03447.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18476875" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B50" class="ref-content "><div class="label -label">50</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mochida</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Sakurai</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Seki</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Draft genome assembly and annotation of <em>Glycyrrhiza uralensis</em>, a medicinal legume</div>, <div class="source">Plant J</div> ., <div class="volume">89</div>, <div class="fpage">181</div>–<div class="lpage">94</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Draft%20genome%20assembly%20and%20annotation%20of%20Glycyrrhiza%20uralensis%2C%20a%20medicinal%20legume&amp;author=K.%20Mochida&amp;author=T.%20Sakurai&amp;author=H.%20Seki&amp;publication_year=2017&amp;journal=Plant%20J&amp;volume=89&amp;pages=181-94" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2017.89.issue-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2017.89.issue-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2017.89.issue-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27775193" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B51" class="ref-content "><div class="label -label">51</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sakai</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hirata</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Sayama</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Production of 2-phenylethanol in roses as the dominant floral scent compound from L-phenylalanine by two key enzymes, a PLP-dependent decarboxylase and a phenylacetaldehyde reductase</div>, <div class="source">Biosci. Biotechnol. Biochem</div> ., <div class="volume">71</div>, <div class="fpage">2408</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Production%20of%202-phenylethanol%20in%20roses%20as%20the%20dominant%20floral%20scent%20compound%20from%20L-phenylalanine%20by%20two%20key%20enzymes%2C%20a%20PLP-dependent%20decarboxylase%20and%20a%20phenylacetaldehyde%20reductase&amp;author=M.%20Sakai&amp;author=H.%20Hirata&amp;author=H.%20Sayama&amp;publication_year=2007&amp;journal=Biosci.%20Biotechnol.%20Biochem&amp;volume=71&amp;pages=2408-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.70090" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.70090" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.70090"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17928708" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B52" class="ref-content "><div class="label -label">52</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">X.M.</div></div>, <div class="name string-name"><div class="surname">Kobayashi</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Sakai</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Functional characterization of rose phenylacetaldehyde reductase (PAR), an enzyme involved in the biosynthesis of the scent compound 2-phenylethanol</div>, <div class="source">J. Plant Physiol</div> ., <div class="volume">168</div>, <div class="fpage">88</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Functional%20characterization%20of%20rose%20phenylacetaldehyde%20reductase%20(PAR)%2C%20an%20enzyme%20involved%20in%20the%20biosynthesis%20of%20the%20scent%20compound%202-phenylethanol&amp;author=X.M.%20Chen&amp;author=H.%20Kobayashi&amp;author=M.%20Sakai&amp;publication_year=2011&amp;journal=J.%20Plant%20Physiol&amp;volume=168&amp;pages=88-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jplph.2010.06.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jplph.2010.06.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jplph.2010.06.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20650544" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B53" class="ref-content "><div class="label -label">53</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Joichi</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Yomogida</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Awano</div> <div class="given-names">K-I.</div></div>, <div class="name string-name"><div class="surname">Ueda</div> <div class="given-names">Y.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Volatile components of tea-scented modern roses and ancient Chinese roses</div>, <div class="source">Flavour Fragr. J</div> ., <div class="volume">20</div>, <div class="fpage">152</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Volatile%20components%20of%20tea-scented%20modern%20roses%20and%20ancient%20Chinese%20roses&amp;author=A.%20Joichi&amp;author=K.%20Yomogida&amp;author=K-I.%20Awano&amp;author=Y.%20Ueda&amp;publication_year=2005&amp;journal=Flavour%20Fragr.%20J&amp;volume=20&amp;pages=152-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1099-1026" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1099-1026" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1099-1026"> </span></div> </div></div></div></div></div><div content-id="dsx042-B54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B54" class="ref-content "><div class="label -label">54</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Scalliet</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Piola</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Douady</div> <div class="given-names">C.J.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Scent evolution in Chinese roses</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="volume">105</div>, <div class="fpage">5927</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Scent%20evolution%20in%20Chinese%20roses&amp;author=G.%20Scalliet&amp;author=F.%20Piola&amp;author=C.J.%20Douady&amp;publication_year=2008&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=5927-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0711551105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0711551105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0711551105"> </span></div> </div></div></div></div></div><div content-id="dsx042-B55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B55" class="ref-content "><div class="label -label">55</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Muhlemann</div> <div class="given-names">J.K.</div></div>, <div class="name string-name"><div class="surname">Klempien</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Dudareva</div> <div class="given-names">N.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Floral volatiles: from biosynthesis to function</div>, <div class="source">Plant. Cell Environ</div> ., <div class="volume">37</div>, <div class="fpage">1936</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Floral%20volatiles%3A%20from%20biosynthesis%20to%20function&amp;author=J.K.%20Muhlemann&amp;author=A.%20Klempien&amp;author=N.%20Dudareva&amp;publication_year=2014&amp;journal=Plant.%20Cell%20Environ&amp;volume=37&amp;pages=1936-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/pce.12314" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fpce.12314" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fpce.12314"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24588567" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B56" class="ref-content "><div class="label -label">56</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Auldridge</div> <div class="given-names">M.E.</div></div>, <div class="name string-name"><div class="surname">McCarty</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Klee</div> <div class="given-names">H.J.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Plant carotenoid cleavage oxygenases and their apocarotenoid products</div>, <div class="source">Curr. Opin. Plant Biol</div> ., <div class="volume">9</div>, <div class="fpage">315</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20carotenoid%20cleavage%20oxygenases%20and%20their%20apocarotenoid%20products&amp;author=M.E.%20Auldridge&amp;author=D.R.%20McCarty&amp;author=H.J.%20Klee&amp;publication_year=2006&amp;journal=Curr.%20Opin.%20Plant%20Biol&amp;volume=9&amp;pages=315-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.pbi.2006.03.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.pbi.2006.03.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.pbi.2006.03.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16616608" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B57" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B57" class="ref-content "><div class="label -label">57</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Theßen</div> <div class="given-names">G.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Development of floral organ identity: stories from MADS house</div>, <div class="source">Plant Biol</div> ., <div class="volume">4</div>, <div class="fpage">75</div>–<div class="lpage">85</div>.</div></div></div></div><div content-id="dsx042-B58" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B58" class="ref-content "><div class="label -label">58</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Mibus</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Heckl</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Serek</div> <div class="given-names">M.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Cloning and characterization of three <em>APETALA1/FRUITFULL</em> like genes in different flower types of <em>Rosa</em> × <em>hybrida</em> L</div>, <div class="source">J. Plant Growth Regul</div> ., <div class="volume">30</div>, <div class="fpage">272</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cloning%20and%20characterization%20of%20three%20APETALA1%2FFRUITFULL%20like%20genes%20in%20different%20flower%20types%20of%20Rosa%20%C3%97%20hybrida%20L&amp;author=H.%20Mibus&amp;author=D.%20Heckl&amp;author=M.%20Serek&amp;publication_year=2011&amp;journal=J.%20Plant%20Growth%20Regul&amp;volume=30&amp;pages=272-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00344-010-9190-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00344-010-9190-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00344-010-9190-8"> </span></div> </div></div></div></div></div><div content-id="dsx042-B59" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B59" class="ref-content "><div class="label -label">59</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kitahara</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Hirai</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Fukui</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">S.</div></div></div> <div class="year">2001</div>, 
<div class="article-title">Rose MADS-box genes ′MASAKO BP and B3′homologous to class B floral identity genes</div>, <div class="source">Plant Sci</div> ., <div class="volume">161</div>, <div class="fpage">549</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rose%20MADS-box%20genes%20%E2%80%B2MASAKO%20BP%20and%20B3%E2%80%B2homologous%20to%20class%20B%20floral%20identity%20genes&amp;author=K.%20Kitahara&amp;author=S.%20Hirai&amp;author=H.%20Fukui&amp;author=S.%20Matsumoto&amp;publication_year=2001&amp;journal=Plant%20Sci&amp;volume=161&amp;pages=549-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9452(01)00442-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9452(01)00442-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9452(01)00442-3"> </span></div> </div></div></div></div></div><div content-id="dsx042-B60" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B60" class="ref-content "><div class="label -label">60</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hibino</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kitahara</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Hirai</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">S.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Structural and functional analysis of rose class B MADS-box genes ′MASAKO BP, euB3, and B3′: Paleo-type AP3 homologue ′MASAKO B3′ association with petal development</div>, <div class="source">Plant Sci</div> ., <div class="volume">170</div>, <div class="fpage">778</div>–<div class="lpage">85</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Structural%20and%20functional%20analysis%20of%20rose%20class%20B%20MADS-box%20genes%20%E2%80%B2MASAKO%20BP%2C%20euB3%2C%20and%20B3%E2%80%B2%3A%20Paleo-type%20AP3%20homologue%20%E2%80%B2MASAKO%20B3%E2%80%B2%20association%20with%20petal%20development&amp;author=Y.%20Hibino&amp;author=K.%20Kitahara&amp;author=S.%20Hirai&amp;author=S.%20Matsumoto&amp;publication_year=2006&amp;journal=Plant%20Sci&amp;volume=170&amp;pages=778-85" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plantsci.2005.11.010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2005.11.010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plantsci.2005.11.010"> </span></div> </div></div></div></div></div><div content-id="dsx042-B61" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B61" class="ref-content "><div class="label -label">61</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kitahara</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">S.</div></div></div> <div class="year">2000</div>, 
<div class="article-title">Rose MADS-box genes ′MASAKO C1 and D1′ homologous to class C floral identity genes</div>, <div class="source">Plant Sci</div> ., <div class="volume">151</div>, <div class="fpage">121</div>–<div class="lpage">34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rose%20MADS-box%20genes%20%E2%80%B2MASAKO%20C1%20and%20D1%E2%80%B2%20homologous%20to%20class%20C%20floral%20identity%20genes&amp;author=K.%20Kitahara&amp;author=S.%20Matsumoto&amp;publication_year=2000&amp;journal=Plant%20Sci&amp;volume=151&amp;pages=121-34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0168-9452(99)00206-X" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0168-9452(99)00206-X" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0168-9452(99)00206-X"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10808068" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B62" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B62" class="ref-content "><div class="label -label">62</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gion</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Suzuri</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ishiguro</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Genetic engineering of floricultural crops: Modification of flower colour, flowering and shape</div>, <div class="source">Acta Hortic</div> ., <div class="volume">953</div>, <div class="fpage">209</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20engineering%20of%20floricultural%20crops%3A%20Modification%20of%20flower%20colour%2C%20flowering%20and%20shape&amp;author=K.%20Gion&amp;author=R.%20Suzuri&amp;author=K.%20Ishiguro&amp;publication_year=2012&amp;journal=Acta%20Hortic&amp;volume=953&amp;pages=209-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.17660/ActaHortic.2012.953.29" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.17660%2FActaHortic.2012.953.29" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.17660%2FActaHortic.2012.953.29"> </span></div> </div></div></div></div></div><div content-id="dsx042-B63" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B63" class="ref-content "><div class="label -label">63</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Kitahara</div> <div class="given-names">K.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">MADS-box genes in rose: expression analyses of AGAMOUS, PISTILLATA, APETALA3 and SEPALLATA homologue genes in the green rose</div>, <div class="source">Acta Hortic</div> ., <div class="volume">690</div>, <div class="fpage">203</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MADS-box%20genes%20in%20rose%3A%20expression%20analyses%20of%20AGAMOUS%2C%20PISTILLATA%2C%20APETALA3%20and%20SEPALLATA%20homologue%20genes%20in%20the%20green%20rose&amp;author=S.%20Matsumoto&amp;author=K.%20Kitahara&amp;publication_year=2005&amp;journal=Acta%20Hortic&amp;volume=690&amp;pages=203-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.17660/ActaHortic.2005.690.31" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.17660%2FActaHortic.2005.690.31" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.17660%2FActaHortic.2005.690.31"> </span></div> </div></div></div></div></div><div content-id="dsx042-B64" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B64" class="ref-content "><div class="label -label">64</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dubois</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Raymond</div> <div class="given-names">O.</div></div>, <div class="name string-name"><div class="surname">Maene</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Tinkering with the C-Function: a molecular frame for the selection of double flowers in cultivated roses</div>, <div class="source">PLoS One</div> , <div class="volume">5</div>, <div class="fpage">e9288</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Tinkering%20with%20the%20C-Function%3A%20a%20molecular%20frame%20for%20the%20selection%20of%20double%20flowers%20in%20cultivated%20roses&amp;author=A.%20Dubois&amp;author=O.%20Raymond&amp;author=M.%20Maene&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e9288" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0009288" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0009288" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0009288"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20174587" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B65" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B65" class="ref-content "><div class="label -label">65</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Iwata</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Gaston</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Remay</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The TFL1 homologue KSN is a regulator of continuous flowering in rose and strawberry</div>, <div class="source">Plant J</div> ., <div class="volume">69</div>, <div class="fpage">116</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20TFL1%20homologue%20KSN%20is%20a%20regulator%20of%20continuous%20flowering%20in%20rose%20and%20strawberry&amp;author=H.%20Iwata&amp;author=A.%20Gaston&amp;author=A.%20Remay&amp;publication_year=2012&amp;journal=Plant%20J&amp;volume=69&amp;pages=116-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2011.69.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2011.69.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2011.69.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21895811" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B66" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B66" class="ref-content "><div class="label -label">66</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">van Doorn</div> <div class="given-names">W.G.</div></div>, <div class="name string-name"><div class="surname">Kamdee</div> <div class="given-names">C.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Flower opening and closure: an update</div>, <div class="source">J. Exp. Bot</div> ., <div class="volume">65</div>, <div class="fpage">5749</div>–<div class="lpage">57</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Flower%20opening%20and%20closure%3A%20an%20update&amp;author=W.G.%20van%20Doorn&amp;author=C.%20Kamdee&amp;publication_year=2014&amp;journal=J.%20Exp.%20Bot&amp;volume=65&amp;pages=5749-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/eru327" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Feru327" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Feru327"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25135521" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B67" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B67" class="ref-content "><div class="label -label">67</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ma</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Xue</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Rh-PIP2; 1, a rose aquaporin gene, is involved in ethylene-regulated petal expansion</div>, <div class="source">Plant Physiol</div> ., <div class="volume">148</div>, <div class="fpage">894</div>–<div class="lpage">907</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rh-PIP2%3B%201%2C%20a%20rose%20aquaporin%20gene%2C%20is%20involved%20in%20ethylene-regulated%20petal%20expansion&amp;author=N.%20Ma&amp;author=J.%20Xue&amp;author=Y.%20Li&amp;publication_year=2008&amp;journal=Plant%20Physiol&amp;volume=148&amp;pages=894-907" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.108.120154" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.108.120154" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.108.120154"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18715962" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B68" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B68" class="ref-content "><div class="label -label">68</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yamada</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Takahashi</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Fujitani</div> <div class="given-names">C.</div></div></div>, et al.   <div class="year">2009</div>, 
<div class="article-title">Cell wall extensibility and effect of cell-wall-loosening proteins during rose flower opening</div>, <div class="source">J. Japan. Soc. Hort. Sci</div> ., <div class="volume">78</div>, <div class="fpage">242</div>–<div class="lpage">51</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cell%20wall%20extensibility%20and%20effect%20of%20cell-wall-loosening%20proteins%20during%20rose%20flower%20opening&amp;author=K.%20Yamada&amp;author=R.%20Takahashi&amp;author=C.%20Fujitani&amp;publication_year=2009&amp;journal=J.%20Japan.%20Soc.%20Hort.%20Sci&amp;volume=78&amp;pages=242-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2503/jjshs1.78.242" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2503%2Fjjshs1.78.242" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2503%2Fjjshs1.78.242"> </span></div> </div></div></div></div></div><div content-id="dsx042-B69" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B69" class="ref-content "><div class="label -label">69</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xue</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Yang</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Gao</div> <div class="given-names">J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Isolation of Rh-TIP1; 1, an aquaporin gene and its expression in rose flowers in response to ethylene and water deficit</div>, <div class="source">Postharvest Biol. Technol</div> ., <div class="volume">51</div>, <div class="fpage">407</div>–<div class="lpage">13</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20of%20Rh-TIP1%3B%201%2C%20an%20aquaporin%20gene%20and%20its%20expression%20in%20rose%20flowers%20in%20response%20to%20ethylene%20and%20water%20deficit&amp;author=J.%20Xue&amp;author=F.%20Yang&amp;author=J.%20Gao&amp;publication_year=2009&amp;journal=Postharvest%20Biol.%20Technol&amp;volume=51&amp;pages=407-13" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.postharvbio.2008.08.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.postharvbio.2008.08.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.postharvbio.2008.08.011"> </span></div> </div></div></div></div></div><div content-id="dsx042-B70" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B70" class="ref-content "><div class="label -label">70</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dai</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Jiang</div> <div class="given-names">X.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">RhNAC2 and RhEXPA4 are involved in the regulation of dehydration tolerance during the expansion of rose petals</div>, <div class="source">Plant Physiol</div> ., <div class="volume">160</div>, <div class="fpage">2064</div>–<div class="lpage">82</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RhNAC2%20and%20RhEXPA4%20are%20involved%20in%20the%20regulation%20of%20dehydration%20tolerance%20during%20the%20expansion%20of%20rose%20petals&amp;author=F.%20Dai&amp;author=C.%20Zhang&amp;author=X.%20Jiang&amp;publication_year=2012&amp;journal=Plant%20Physiol&amp;volume=160&amp;pages=2064-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.112.207720" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.112.207720" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.112.207720"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23093360" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B71" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B71" class="ref-content "><div class="label -label">71</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Chen</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Yin</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wang</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Involvement of rose aquaporin RhPIP1; 1 in ethylene-regulated petal expansion through interaction with RhPIP2; 1</div>, <div class="source">Plant Mol. Biol</div> ., <div class="volume">83</div>, <div class="fpage">219</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Involvement%20of%20rose%20aquaporin%20RhPIP1%3B%201%20in%20ethylene-regulated%20petal%20expansion%20through%20interaction%20with%20RhPIP2%3B%201&amp;author=W.%20Chen&amp;author=X.%20Yin&amp;author=L.%20Wang&amp;publication_year=2013&amp;journal=Plant%20Mol.%20Biol&amp;volume=83&amp;pages=219-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-013-0084-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-013-0084-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-013-0084-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23748738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B72" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B72" class="ref-content "><div class="label -label">72</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Xiang</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">C. H.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Evolution of Rosaceae fruit types based on nuclear phylogeny in the context of geological times and genome duplication</div>, <div class="source">Mol. Biol. Evol</div> <em>.</em>, <div class="volume">34</div>, <div class="fpage">262</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20Rosaceae%20fruit%20types%20based%20on%20nuclear%20phylogeny%20in%20the%20context%20of%20geological%20times%20and%20genome%20duplication&amp;author=Y.%20Xiang&amp;author=C.%20H.%20Huang&amp;author=Y.%20Hu&amp;publication_year=2017&amp;journal=Mol.%20Biol.%20Evol&amp;volume=34&amp;pages=262-81" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27856652" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B73" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B73" class="ref-content "><div class="label -label">73</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Luo</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Pan</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Guo</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Wan</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Filling gaps with construction of a genetic linkage map in tetraploid roses</div>, <div class="source">Front. Plant Sci</div> ., <div class="volume">5</div>, <div class="fpage">796</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Filling%20gaps%20with%20construction%20of%20a%20genetic%20linkage%20map%20in%20tetraploid%20roses&amp;author=C.%20Yu&amp;author=L.%20Luo&amp;author=H.%20Pan&amp;author=X.%20Guo&amp;author=H.%20Wan&amp;author=Q.%20Zhang&amp;publication_year=2014&amp;journal=Front.%20Plant%20Sci&amp;volume=5&amp;pages=796" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25628638" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx042-B74" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B74" class="ref-content "><div class="label -label">74</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hibrand-Saint Oyant</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Crespel</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Rajapakse</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Foucher</div> <div class="given-names">F.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Genetic linkage maps of rose constructed with new microsatellite markers and locating QTL controlling flowering traits</div>, <div class="source">Tree Genet. Genomes</div> , <div class="volume">4</div>, <div class="fpage">11</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20linkage%20maps%20of%20rose%20constructed%20with%20new%20microsatellite%20markers%20and%20locating%20QTL%20controlling%20flowering%20traits&amp;author=L.%20Hibrand-Saint%20Oyant&amp;author=L.%20Crespel&amp;author=S.%20Rajapakse&amp;author=L.%20Zhang&amp;author=F.%20Foucher&amp;publication_year=2008&amp;journal=Tree%20Genet.%20Genomes&amp;volume=4&amp;pages=11-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-007-0084-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-007-0084-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-007-0084-2"> </span></div> </div></div></div></div></div><div content-id="dsx042-B75" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx042-B75" class="ref-content "><div class="label -label">75</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zhang</div> <div class="given-names">L.H.</div></div>, <div class="name string-name"><div class="surname">Byrne</div> <div class="given-names">D.H.</div></div>, <div class="name string-name"><div class="surname">Ballard</div> <div class="given-names">R.E.</div></div>, <div class="name string-name"><div class="surname">Rajapakse</div> <div class="given-names">S.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Microsatellite marker development in rose and its application in tetraploid mapping</div>, <div class="source">J. Am. Soc. Hort. Sci</div> ., <div class="volume">131</div>, <div class="fpage">380</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microsatellite%20marker%20development%20in%20rose%20and%20its%20application%20in%20tetraploid%20mapping&amp;author=L.H.%20Zhang&amp;author=D.H.%20Byrne&amp;author=R.E.%20Ballard&amp;author=S.%20Rajapakse&amp;publication_year=2006&amp;journal=J.%20Am.%20Soc.%20Hort.%20Sci&amp;volume=131&amp;pages=380-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.scienta.2005.09.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.scienta.2005.09.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.scienta.2005.09.004"> </span></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4554776&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_figures_v3.ppt?Expires=2147483647&amp;Signature=K739ZxcHPNLMts~GAa~0tR2AoGvld4AANa64MAtsB0A1wpU4ndvkyusbxQK4dfDk2NKvYELFtwlRT6zmlRvz9NgbDhtOkfW1DVLj~XZHVdgreXWRUeS-azVXPCW8xgZU2E37m3NWZefB-cwexgw3NQ8NujG2jbP8gfxqZpU73kvdUEI-pNuWvpWWDt9gw4ga21~Azqbcg-Fhcw17a7q8jTNsepKVh-uN8lvzes128FJ3TvMzdYKqCLRgBCC-rUaxD4m~HcbfiGMKc6V2AUGVgb7-QBy0in6dF0RjAease4fb3c5hBTlXjvd6I1D0plWUTQaWtTKBw97ljKR8WNTfrQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figures</div></a> - ppt file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx042/1/dsx042_supplementary_tables_v2.xlsx?Expires=2147483647&amp;Signature=huOblyEtef08KImxfv2WbGrXjBcR98XNka3i58EkiObUgWSQ5GipUC0AA2ct63sDL1OBlB6d0BY8FwbFmVK4XMVUuC0fsV4GMhcLoC5TQWMyRhnZTr4aqrAZMmpGFJTOdbfjfMA2NydY2yOplV4OityXhQqBa503JnX8HfjghcyXSYe~xOMUrYrTgD~SHxIYpco5ECw~VItrc5v4Q-Z1iDGjITVvCJnXCmwKdH2zmo1Z2t4oMkmJcsj2Fw1TWrSFAXxT8zAw4DL1A4AANZKTp3n-reqQbvSubl6WfHXlnzKI8BJd9CwwS9jxSlUCdF16VYEn2JmFvUF6MCVCVHYrag__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Tables</div></a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">2,415</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000430686200001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">5</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx042"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4554776"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000430686200001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f2%2f113%2f4554776">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000430686200001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (5)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F2%2F113%2F4554776" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4554776'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Genome structure of Rosa multiflora, a wild ancestor of cultivated roses" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx042");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f113%2f4554776">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

