

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/18/5/343/448814","@type":"ScholarlyArticle","name":"Analysis of Muscle and Ovary Transcriptome of  Sus scrofa : Assembly, Annotation and Marker Discovery","datePublished":"2011-07-05","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/18/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2011-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsr021","keywords":["transcriptome","<italic>Sus scrofa</italic>","454 sequencing","SNPs","SSRs"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/18/5/343/448814","author":[{"name":"Nie, Qinghua","affiliation":"1  Department of Animal Genetics, Breeding and Reproduction ,  College of Animal Science, South China Agricultural University ,  Guangzhou ,  Guangdong 510642 ,  People's Republic of China    2  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding ,  Guangzhou ,  Guangdong 510642 ,  People's Republic of China","@type":"Person"},{"name":"Fang, Meixia","affiliation":"3  Department of Laboratory Animal Science ,  Medical College of Jinan University ,  Guangzhou, Guangdong 510632 ,  People's Republic of China","@type":"Person"},{"name":"Jia, Xinzheng","affiliation":"1  Department of Animal Genetics, Breeding and Reproduction ,  College of Animal Science, South China Agricultural University ,  Guangzhou ,  Guangdong 510642 ,  People's Republic of China    2  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding ,  Guangzhou ,  Guangdong 510642 ,  People's Republic of China","@type":"Person"},{"name":"Zhang, Wei","affiliation":"1  Department of Animal Genetics, Breeding and Reproduction ,  College of Animal Science, South China Agricultural University ,  Guangzhou ,  Guangdong 510642 ,  People's Republic of China    2  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding ,  Guangzhou ,  Guangdong 510642 ,  People's Republic of China","@type":"Person"},{"name":"Zhou, Xiaoning","affiliation":"1  Department of Animal Genetics, Breeding and Reproduction ,  College of Animal Science, South China Agricultural University ,  Guangzhou ,  Guangdong 510642 ,  People's Republic of China    2  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding ,  Guangzhou ,  Guangdong 510642 ,  People's Republic of China","@type":"Person"},{"name":"He, Xiaomei","affiliation":"1  Department of Animal Genetics, Breeding and Reproduction ,  College of Animal Science, South China Agricultural University ,  Guangzhou ,  Guangdong 510642 ,  People's Republic of China    2  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding ,  Guangzhou ,  Guangdong 510642 ,  People's Republic of China","@type":"Person"},{"name":"Zhang, Xiquan","affiliation":"1  Department of Animal Genetics, Breeding and Reproduction ,  College of Animal Science, South China Agricultural University ,  Guangzhou ,  Guangdong 510642 ,  People's Republic of China    2  Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding ,  Guangzhou ,  Guangdong 510642 ,  People's Republic of China","@type":"Person"}],"description":"Abstract.  Pig (Sus scrofa) is an important organism for both agricultural and medical purpose. This study aims to investigate the S. scrofa transcriptome by th","pageStart":"343","pageEnd":"351","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/5/1/m_cover.gif?Expires=2147483647&Signature=fZbX~i0F4YmMgemzi~KYwJoeRvQcK3vESx0rzJw5EUkahhhho4dt2jCm2s4eb4lcFKiJ3IkaVuW~mFnQ0N-xsBlJ-atzuGchQMdBMlEaTp9QoaKOLQdSjpbwxtx3IKeAKX9MAFcEHgiyPIddtsEKqmKyu2yRzRpJfM6fHjheCB8GOlWvmsNX-mG7yzjfRRAgK2-xl55jEo9vwfkLAfBWwh9slsk8CoXZ~kVyHDiYgXlvdg5m4LybAGcWIMBm9UYCeJTV~chzGeImI3yLbjJOa6lXjiaaTcisKEGSS9lHedM-W~GtV9CboT5Ke43Hf0WuKYesrRRMqe~T4KpwdHK-gg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Analysis of Muscle and Ovary Transcriptome of  Sus scrofa : Assembly, Annotation and Marker Discovery","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/5/1/m_cover.gif?Expires=2147483647&Signature=fZbX~i0F4YmMgemzi~KYwJoeRvQcK3vESx0rzJw5EUkahhhho4dt2jCm2s4eb4lcFKiJ3IkaVuW~mFnQ0N-xsBlJ-atzuGchQMdBMlEaTp9QoaKOLQdSjpbwxtx3IKeAKX9MAFcEHgiyPIddtsEKqmKyu2yRzRpJfM6fHjheCB8GOlWvmsNX-mG7yzjfRRAgK2-xl55jEo9vwfkLAfBWwh9slsk8CoXZ~kVyHDiYgXlvdg5m4LybAGcWIMBm9UYCeJTV~chzGeImI3yLbjJOa6lXjiaaTcisKEGSS9lHedM-W~GtV9CboT5Ke43Hf0WuKYesrRRMqe~T4KpwdHK-gg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Analysis of Muscle and Ovary Transcriptome of  Sus scrofa : Assembly, Annotation and Marker Discovery" />
    <meta property="og:description" content="Abstract.  Pig (Sus scrofa) is an important organism for both agricultural and medical purpose. This study aims to investigate the S. scrofa transcriptome by th" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/18/5/343/448814" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2011-07-05" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Nie, Qinghua" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou, Guangdong 510642, People's Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, Guangzhou, Guangdong 510642, People's Republic of China" /><meta name="citation_author" content="Fang, Meixia" /><meta name="citation_author_institution" content="Department of Laboratory Animal Science, Medical College of Jinan University, Guangzhou, Guangdong 510632, People's Republic of China" /><meta name="citation_author" content="Jia, Xinzheng" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou, Guangdong 510642, People's Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, Guangzhou, Guangdong 510642, People's Republic of China" /><meta name="citation_author" content="Zhang, Wei" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou, Guangdong 510642, People's Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, Guangzhou, Guangdong 510642, People's Republic of China" /><meta name="citation_author" content="Zhou, Xiaoning" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou, Guangdong 510642, People's Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, Guangzhou, Guangdong 510642, People's Republic of China" /><meta name="citation_author" content="He, Xiaomei" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou, Guangdong 510642, People's Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, Guangzhou, Guangdong 510642, People's Republic of China" /><meta name="citation_author" content="Zhang, Xiquan" /><meta name="citation_author_institution" content="Department of Animal Genetics, Breeding and Reproduction, College of Animal Science, South China Agricultural University, Guangzhou, Guangdong 510642, People's Republic of China" /><meta name="citation_author_institution" content="Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding, Guangzhou, Guangdong 510642, People's Republic of China" /><meta name="citation_title" content="Analysis of Muscle and Ovary Transcriptome of Sus scrofa: Assembly, Annotation and Marker Discovery" /><meta name="citation_firstpage" content="343" /><meta name="citation_lastpage" content="351" /><meta name="citation_doi" content="10.1093/dnares/dsr021" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="18" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2011/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Domestic-animal genomics: deciphering the genetics of complex traits; Nat. Rev. Genet.;  citation_year=2004;  citation_volume=5;  citation_pages=202-12; " /><meta name="citation_reference" content="citation_title=Identification of a mutation in porcine ryanodine receptor associated with malignant hyperthermia; Science;  citation_year=1991;  citation_volume=253;  citation_pages=448-51; " /><meta name="citation_reference" content="citation_title=A regulatory mutation in IGF2 causes a major QTL effect on muscle growth in the pig; Nature;  citation_year=2003;  citation_volume=425;  citation_pages=832-6; " /><meta name="citation_reference" content="citation_title=Pigs in sequence space: a 0.66X coverage pig genome survey based on shotgun sequencing; BMC Genomics;  citation_year=2005;  citation_volume=6;  citation_pages=70" /><meta name="citation_reference" content="citation_title=Mining for single nucleotide polymorphisms in pig genome sequence data; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=4" /><meta name="citation_reference" content="citation_title=Design of a high density SNP genotyping assay in the pig using SNPs identified and characterized by next generation sequencing technology; PLoS One;  citation_year=2009;  citation_volume=4;  citation_pages=e6524" /><meta name="citation_reference" content="citation_title=Genome-wide association study identifies loci for body composition and structural soundness traits in pigs; PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e14726" /><meta name="citation_reference" content="citation_title=Microarray profiling for differential gene expression in PMSG-hCG stimulated preovulatory ovarian follicles of Chinese Taihu and Large White sows; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=111" /><meta name="citation_reference" content="citation_title=Genome-wide linkage analysis of global gene expression in loin muscle tissue identifies candidate genes in pigs; PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e16766" /><meta name="citation_reference" content="citation_title=RNA sequencing: advances, challenges and opportunities; Nat. Rev. Genet.;  citation_year=2011;  citation_volume=12;  citation_pages=87-98; " /><meta name="citation_reference" content="citation_title=Transcriptome sequencing of malignant pleural mesothelioma tumors; Proc. Natl Acad. Sci. USA;  citation_year=2008;  citation_volume=105;  citation_pages=3521-6; " /><meta name="citation_reference" content="citation_title=Dynamic transcriptomes during neural differentiation of human embryonic stem cells revealed by short, long, and paired-end sequencing; Proc. Natl Acad. Sci. USA;  citation_year=2010;  citation_volume=107;  citation_pages=5254-9; " /><meta name="citation_reference" content="citation_title=Transcriptomic and proteomic profiling of two porcine tissues using high-throughput technologies; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=30" /><meta name="citation_reference" content="citation_title=Transcriptomics of the bed bug (Cimex lectularius); PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e16336" /><meta name="citation_reference" content="citation_title=Characterization of the rainbow trout transcriptome using Sanger and 454-pyrosequencing approaches; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=564" /><meta name="citation_reference" content="citation_title=Transcriptomic signatures of ash (Fraxinus spp.) phloem; PLoS One;  citation_year=2011;  citation_volume=6;  citation_pages=e16368" /><meta name="citation_reference" content="citation_title=Transcriptome sequencing in an ecologically important tree species: assembly, annotation, and marker discovery; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=180" /><meta name="citation_reference" content="citation_title=Gut microbial gene expression in mother-fed and formula-fed piglets; PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e12459" /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Gene ontology: tool for the unification of biology; Nat. Genet.;  citation_year=2000;  citation_volume=25;  citation_pages=25-9; " /><meta name="citation_reference" content="citation_title=DEGseq: an R package for identifying differentially expressed genes from RNA-seq data; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=136-8; " /><meta name="citation_reference" content="citation_title=Initial sequencing and analysis of the human genome; citation_author=International Human Genome Sequencing Consortium; Nature;  citation_year=2001;  citation_volume=409;  citation_pages=860-921; " /><meta name="citation_reference" content="citation_title=Initial sequencing and comparative analysis of the mouse genome; citation_author=Mouse Genome Sequencing Consortium; Nature;  citation_year=2002;  citation_volume=420;  citation_pages=520-62; " /><meta name="citation_reference" content="citation_title=Sequencing, de novo annotation and analysis of the first Anguilla anguilla transcriptome: EeelBase opens new perspectives for the study of the critically endangered European eel; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=635" /><meta name="citation_reference" content="citation_title=De novo assembly of chickpea transcriptome using short reads for gene discovery and marker identification; DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=53-63; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/18/5/343/1227029/dsr021.pdf" /><meta name="description" content="Abstract.  Pig (Sus scrofa) is an important organism for both agricultural and medical purpose. This study aims to investigate the S. scrofa transcriptome by th" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/18/5/343/448814" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/18/5/343/448814" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/18/5/343/448814");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsr021';
    </script>
<title>
	Analysis of Muscle and Ovary Transcriptome of Sus scrofa: Assembly, Annotation and Marker Discovery | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./448814" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Be1NPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOaWUsIFFpbmdodWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFuaW1hbCBHZW5ldGljcywgQnJlZWRpbmcgYW5kIFJlcHJvZHVjdGlvbiwgQ29sbGVnZSBvZiBBbmltYWwgU2NpZW5jZSwgU291dGggQ2hpbmEgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIEd1YW5nemhvdSwgR3Vhbmdkb25nIDUxMDY0MiwgUGVvcGxlJ3MgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHdWFuZ2RvbmcgUHJvdmluY2lhbCBLZXkgTGFiIG9mIEFncm8tQW5pbWFsIEdlbm9taWNzIGFuZCBNb2xlY3VsYXIgQnJlZWRpbmcsIEd1YW5nemhvdSwgR3Vhbmdkb25nIDUxMDY0MiwgUGVvcGxlJ3MgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGYW5nLCBNZWl4aWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIExhYm9yYXRvcnkgQW5pbWFsIFNjaWVuY2UsIE1lZGljYWwgQ29sbGVnZSBvZiBKaW5hbiBVbml2ZXJzaXR5LCBHdWFuZ3pob3UsIEd1YW5nZG9uZyA1MTA2MzIsIFBlb3BsZSdzIFJlcHVibGljIG9mIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSmlhLCBYaW56aGVuZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQW5pbWFsIEdlbmV0aWNzLCBCcmVlZGluZyBhbmQgUmVwcm9kdWN0aW9uLCBDb2xsZWdlIG9mIEFuaW1hbCBTY2llbmNlLCBTb3V0aCBDaGluYSBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgR3Vhbmd6aG91LCBHdWFuZ2RvbmcgNTEwNjQyLCBQZW9wbGUncyBSZXB1YmxpYyBvZiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikd1YW5nZG9uZyBQcm92aW5jaWFsIEtleSBMYWIgb2YgQWdyby1BbmltYWwgR2Vub21pY3MgYW5kIE1vbGVjdWxhciBCcmVlZGluZywgR3Vhbmd6aG91LCBHdWFuZ2RvbmcgNTEwNjQyLCBQZW9wbGUncyBSZXB1YmxpYyBvZiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlpoYW5nLCBXZWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEFuaW1hbCBHZW5ldGljcywgQnJlZWRpbmcgYW5kIFJlcHJvZHVjdGlvbiwgQ29sbGVnZSBvZiBBbmltYWwgU2NpZW5jZSwgU291dGggQ2hpbmEgQWdyaWN1bHR1cmFsIFVuaXZlcnNpdHksIEd1YW5nemhvdSwgR3Vhbmdkb25nIDUxMDY0MiwgUGVvcGxlJ3MgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJHdWFuZ2RvbmcgUHJvdmluY2lhbCBLZXkgTGFiIG9mIEFncm8tQW5pbWFsIEdlbm9taWNzIGFuZCBNb2xlY3VsYXIgQnJlZWRpbmcsIEd1YW5nemhvdSwgR3Vhbmdkb25nIDUxMDY0MiwgUGVvcGxlJ3MgUmVwdWJsaWMgb2YgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaG91LCBYaWFvbmluZyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQW5pbWFsIEdlbmV0aWNzLCBCcmVlZGluZyBhbmQgUmVwcm9kdWN0aW9uLCBDb2xsZWdlIG9mIEFuaW1hbCBTY2llbmNlLCBTb3V0aCBDaGluYSBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgR3Vhbmd6aG91LCBHdWFuZ2RvbmcgNTEwNjQyLCBQZW9wbGUncyBSZXB1YmxpYyBvZiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikd1YW5nZG9uZyBQcm92aW5jaWFsIEtleSBMYWIgb2YgQWdyby1BbmltYWwgR2Vub21pY3MgYW5kIE1vbGVjdWxhciBCcmVlZGluZywgR3Vhbmd6aG91LCBHdWFuZ2RvbmcgNTEwNjQyLCBQZW9wbGUncyBSZXB1YmxpYyBvZiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhlLCBYaWFvbWVpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBBbmltYWwgR2VuZXRpY3MsIEJyZWVkaW5nIGFuZCBSZXByb2R1Y3Rpb24sIENvbGxlZ2Ugb2YgQW5pbWFsIFNjaWVuY2UsIFNvdXRoIENoaW5hIEFncmljdWx0dXJhbCBVbml2ZXJzaXR5LCBHdWFuZ3pob3UsIEd1YW5nZG9uZyA1MTA2NDIsIFBlb3BsZSdzIFJlcHVibGljIG9mIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3Vhbmdkb25nIFByb3ZpbmNpYWwgS2V5IExhYiBvZiBBZ3JvLUFuaW1hbCBHZW5vbWljcyBhbmQgTW9sZWN1bGFyIEJyZWVkaW5nLCBHdWFuZ3pob3UsIEd1YW5nZG9uZyA1MTA2NDIsIFBlb3BsZSdzIFJlcHVibGljIG9mIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhhbmcsIFhpcXVhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQW5pbWFsIEdlbmV0aWNzLCBCcmVlZGluZyBhbmQgUmVwcm9kdWN0aW9uLCBDb2xsZWdlIG9mIEFuaW1hbCBTY2llbmNlLCBTb3V0aCBDaGluYSBBZ3JpY3VsdHVyYWwgVW5pdmVyc2l0eSwgR3Vhbmd6aG91LCBHdWFuZ2RvbmcgNTEwNjQyLCBQZW9wbGUncyBSZXB1YmxpYyBvZiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikd1YW5nZG9uZyBQcm92aW5jaWFsIEtleSBMYWIgb2YgQWdyby1BbmltYWwgR2Vub21pY3MgYW5kIE1vbGVjdWxhciBCcmVlZGluZywgR3Vhbmd6aG91LCBHdWFuZ2RvbmcgNTEwNjQyLCBQZW9wbGUncyBSZXB1YmxpYyBvZiBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iQW5hbHlzaXMgb2YgTXVzY2xlIGFuZCBPdmFyeSBUcmFuc2NyaXB0b21lIG9mIFN1cyBzY3JvZmE6IEFzc2VtYmx5LCBBbm5vdGF0aW9uIGFuZCBNYXJrZXIgRGlzY292ZXJ5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMzQzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIzNTEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3IwMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMS8xMC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURvbWVzdGljLWFuaW1hbCBnZW5vbWljczogZGVjaXBoZXJpbmcgdGhlIGdlbmV0aWNzIG9mIGNvbXBsZXggdHJhaXRzOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz0yMDItMTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGEgbXV0YXRpb24gaW4gcG9yY2luZSByeWFub2RpbmUgcmVjZXB0b3IgYXNzb2NpYXRlZCB3aXRoIG1hbGlnbmFudCBoeXBlcnRoZXJtaWE7IFNjaWVuY2U7ICBjaXRhdGlvbl95ZWFyPTE5OTE7ICBjaXRhdGlvbl92b2x1bWU9MjUzOyAgY2l0YXRpb25fcGFnZXM9NDQ4LTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHJlZ3VsYXRvcnkgbXV0YXRpb24gaW4gSUdGMiBjYXVzZXMgYSBtYWpvciBRVEwgZWZmZWN0IG9uIG11c2NsZSBncm93dGggaW4gdGhlIHBpZzsgTmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTQyNTsgIGNpdGF0aW9uX3BhZ2VzPTgzMi02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaWdzIGluIHNlcXVlbmNlIHNwYWNlOiBhIDAuNjZYIGNvdmVyYWdlIHBpZyBnZW5vbWUgc3VydmV5IGJhc2VkIG9uIHNob3RndW4gc2VxdWVuY2luZzsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz03MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pbmluZyBmb3Igc2luZ2xlIG51Y2xlb3RpZGUgcG9seW1vcnBoaXNtcyBpbiBwaWcgZ2Vub21lIHNlcXVlbmNlIGRhdGE7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXNpZ24gb2YgYSBoaWdoIGRlbnNpdHkgU05QIGdlbm90eXBpbmcgYXNzYXkgaW4gdGhlIHBpZyB1c2luZyBTTlBzIGlkZW50aWZpZWQgYW5kIGNoYXJhY3Rlcml6ZWQgYnkgbmV4dCBnZW5lcmF0aW9uIHNlcXVlbmNpbmcgdGVjaG5vbG9neTsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWU2NTI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lLXdpZGUgYXNzb2NpYXRpb24gc3R1ZHkgaWRlbnRpZmllcyBsb2NpIGZvciBib2R5IGNvbXBvc2l0aW9uIGFuZCBzdHJ1Y3R1cmFsIHNvdW5kbmVzcyB0cmFpdHMgaW4gcGlnczsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxNDcyNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvYXJyYXkgcHJvZmlsaW5nIGZvciBkaWZmZXJlbnRpYWwgZ2VuZSBleHByZXNzaW9uIGluIFBNU0ctaENHIHN0aW11bGF0ZWQgcHJlb3Z1bGF0b3J5IG92YXJpYW4gZm9sbGljbGVzIG9mIENoaW5lc2UgVGFpaHUgYW5kIExhcmdlIFdoaXRlIHNvd3M7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTExMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIGxpbmthZ2UgYW5hbHlzaXMgb2YgZ2xvYmFsIGdlbmUgZXhwcmVzc2lvbiBpbiBsb2luIG11c2NsZSB0aXNzdWUgaWRlbnRpZmllcyBjYW5kaWRhdGUgZ2VuZXMgaW4gcGlnczsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxNjc2NiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQSBzZXF1ZW5jaW5nOiBhZHZhbmNlcywgY2hhbGxlbmdlcyBhbmQgb3Bwb3J0dW5pdGllczsgTmF0LiBSZXYuIEdlbmV0LjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMjsgIGNpdGF0aW9uX3BhZ2VzPTg3LTk4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21lIHNlcXVlbmNpbmcgb2YgbWFsaWduYW50IHBsZXVyYWwgbWVzb3RoZWxpb21hIHR1bW9yczsgUHJvYy4gTmF0bCBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xMDU7ICBjaXRhdGlvbl9wYWdlcz0zNTIxLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUR5bmFtaWMgdHJhbnNjcmlwdG9tZXMgZHVyaW5nIG5ldXJhbCBkaWZmZXJlbnRpYXRpb24gb2YgaHVtYW4gZW1icnlvbmljIHN0ZW0gY2VsbHMgcmV2ZWFsZWQgYnkgc2hvcnQsIGxvbmcsIGFuZCBwYWlyZWQtZW5kIHNlcXVlbmNpbmc7IFByb2MuIE5hdGwgQWNhZC4gU2NpLiBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTA3OyAgY2l0YXRpb25fcGFnZXM9NTI1NC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0b21pYyBhbmQgcHJvdGVvbWljIHByb2ZpbGluZyBvZiB0d28gcG9yY2luZSB0aXNzdWVzIHVzaW5nIGhpZ2gtdGhyb3VnaHB1dCB0ZWNobm9sb2dpZXM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTMwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9taWNzIG9mIHRoZSBiZWQgYnVnIChDaW1leCBsZWN0dWxhcml1cyk7IFBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz1lMTYzMzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DaGFyYWN0ZXJpemF0aW9uIG9mIHRoZSByYWluYm93IHRyb3V0IHRyYW5zY3JpcHRvbWUgdXNpbmcgU2FuZ2VyIGFuZCA0NTQtcHlyb3NlcXVlbmNpbmcgYXBwcm9hY2hlczsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NTY0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9taWMgc2lnbmF0dXJlcyBvZiBhc2ggKEZyYXhpbnVzIHNwcC4pIHBobG9lbTsgUExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPWUxNjM2OCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHRvbWUgc2VxdWVuY2luZyBpbiBhbiBlY29sb2dpY2FsbHkgaW1wb3J0YW50IHRyZWUgc3BlY2llczogYXNzZW1ibHksIGFubm90YXRpb24sIGFuZCBtYXJrZXIgZGlzY292ZXJ5OyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz0xODAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HdXQgbWljcm9iaWFsIGdlbmUgZXhwcmVzc2lvbiBpbiBtb3RoZXItZmVkIGFuZCBmb3JtdWxhLWZlZCBwaWdsZXRzOyBQTG9TIE9uZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEyNDU5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FwcGVkIEJMQVNUIGFuZCBQU0ktQkxBU1Q6IGEgbmV3IGdlbmVyYXRpb24gb2YgcHJvdGVpbiBkYXRhYmFzZSBzZWFyY2ggcHJvZ3JhbXM7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMzODktNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5lIG9udG9sb2d5OiB0b29sIGZvciB0aGUgdW5pZmljYXRpb24gb2YgYmlvbG9neTsgTmF0LiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yNS05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ERUdzZXE6IGFuIFIgcGFja2FnZSBmb3IgaWRlbnRpZnlpbmcgZGlmZmVyZW50aWFsbHkgZXhwcmVzc2VkIGdlbmVzIGZyb20gUk5BLXNlcSBkYXRhOyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTEzNi04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbml0aWFsIHNlcXVlbmNpbmcgYW5kIGFuYWx5c2lzIG9mIHRoZSBodW1hbiBnZW5vbWU7IGNpdGF0aW9uX2F1dGhvcj1JbnRlcm5hdGlvbmFsIEh1bWFuIEdlbm9tZSBTZXF1ZW5jaW5nIENvbnNvcnRpdW07IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00MDk7ICBjaXRhdGlvbl9wYWdlcz04NjAtOTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Jbml0aWFsIHNlcXVlbmNpbmcgYW5kIGNvbXBhcmF0aXZlIGFuYWx5c2lzIG9mIHRoZSBtb3VzZSBnZW5vbWU7IGNpdGF0aW9uX2F1dGhvcj1Nb3VzZSBHZW5vbWUgU2VxdWVuY2luZyBDb25zb3J0aXVtOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9NDIwOyAgY2l0YXRpb25fcGFnZXM9NTIwLTYyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nLCBkZSBub3ZvIGFubm90YXRpb24gYW5kIGFuYWx5c2lzIG9mIHRoZSBmaXJzdCBBbmd1aWxsYSBhbmd1aWxsYSB0cmFuc2NyaXB0b21lOiBFZWVsQmFzZSBvcGVucyBuZXcgcGVyc3BlY3RpdmVzIGZvciB0aGUgc3R1ZHkgb2YgdGhlIGNyaXRpY2FsbHkgZW5kYW5nZXJlZCBFdXJvcGVhbiBlZWw7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTYzNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlIG5vdm8gYXNzZW1ibHkgb2YgY2hpY2twZWEgdHJhbnNjcmlwdG9tZSB1c2luZyBzaG9ydCByZWFkcyBmb3IgZ2VuZSBkaXNjb3ZlcnkgYW5kIG1hcmtlciBpZGVudGlmaWNhdGlvbjsgRE5BIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz01My02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE4LzUvMzQzLzEyMjcwMjkvZHNyMDIxLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBQaWcgKFN1cyBzY3JvZmEpIGlzIGFuIGltcG9ydGFudCBvcmdhbmlzbSBmb3IgYm90aCBhZ3JpY3VsdHVyYWwgYW5kIG1lZGljYWwgcHVycG9zZS4gVGhpcyBzdHVkeSBhaW1zIHRvIGludmVzdGlnYXRlIHRoZSBTLiBzY3JvZmEgdHJhbnNjcmlwdG9tZSBieSB0aCIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDQ4ODE0JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZDGCvRPt9iV7d6KQKbLMK4BlXTyZ" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f5%2f343%2f448814" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f5%2f343%2f448814" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17545" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/18/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/18/5/1/m_cover.gif?Expires=2147483647&amp;Signature=fZbX~i0F4YmMgemzi~KYwJoeRvQcK3vESx0rzJw5EUkahhhho4dt2jCm2s4eb4lcFKiJ3IkaVuW~mFnQ0N-xsBlJ-atzuGchQMdBMlEaTp9QoaKOLQdSjpbwxtx3IKeAKX9MAFcEHgiyPIddtsEKqmKyu2yRzRpJfM6fHjheCB8GOlWvmsNX-mG7yzjfRRAgK2-xl55jEo9vwfkLAfBWwh9slsk8CoXZ~kVyHDiYgXlvdg5m4LybAGcWIMBm9UYCeJTV~chzGeImI3yLbjJOa6lXjiaaTcisKEGSS9lHedM-W~GtV9CboT5Ke43Hf0WuKYesrRRMqe~T4KpwdHK-gg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/18/5">
                        <div class="volume trailing-comma">Volume 18</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2011        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6724209">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6724209">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6724213">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6724213">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6724222">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6724222">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6724240">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6724240">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6724284">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6724284">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6724300">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6724300">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6724304">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6724304">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6724309">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6724309">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/18/5/333/448714">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/18/5/353/449021">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Analysis of Muscle and Ovary Transcriptome of <em>Sus scrofa</em>: Assembly, Annotation and Marker Discovery
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Qinghua Nie</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Genetics, Breeding and Reproduction</div>, <div class="insititution">College of Animal Science, South China Agricultural University</div>, <div class="addr-line">Guangzhou</div>, <div class="addr-line">Guangdong 510642</div>, <div class="country">People's Republic of China</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding</div>, <div class="addr-line">Guangzhou</div>, <div class="addr-line">Guangdong 510642</div>, <div class="country">People's Republic of China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Qinghua+Nie">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nie Q">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nie Q&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Qinghua Nie</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Meixia Fang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Department of Laboratory Animal Science</div>, <div class="insititution">Medical College of Jinan University</div>, <div class="addr-line">Guangzhou, Guangdong 510632</div>, <div class="country">People's Republic of China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Meixia+Fang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Fang M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Fang M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Meixia Fang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xinzheng Jia</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Genetics, Breeding and Reproduction</div>, <div class="insititution">College of Animal Science, South China Agricultural University</div>, <div class="addr-line">Guangzhou</div>, <div class="addr-line">Guangdong 510642</div>, <div class="country">People's Republic of China</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding</div>, <div class="addr-line">Guangzhou</div>, <div class="addr-line">Guangdong 510642</div>, <div class="country">People's Republic of China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xinzheng+Jia">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jia X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jia X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xinzheng Jia</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wei Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Genetics, Breeding and Reproduction</div>, <div class="insititution">College of Animal Science, South China Agricultural University</div>, <div class="addr-line">Guangzhou</div>, <div class="addr-line">Guangdong 510642</div>, <div class="country">People's Republic of China</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding</div>, <div class="addr-line">Guangzhou</div>, <div class="addr-line">Guangdong 510642</div>, <div class="country">People's Republic of China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wei+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wei Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaoning Zhou</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Genetics, Breeding and Reproduction</div>, <div class="insititution">College of Animal Science, South China Agricultural University</div>, <div class="addr-line">Guangzhou</div>, <div class="addr-line">Guangdong 510642</div>, <div class="country">People's Republic of China</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding</div>, <div class="addr-line">Guangzhou</div>, <div class="addr-line">Guangdong 510642</div>, <div class="country">People's Republic of China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaoning+Zhou">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhou X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhou X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xiaoning Zhou</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiaomei He</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Genetics, Breeding and Reproduction</div>, <div class="insititution">College of Animal Science, South China Agricultural University</div>, <div class="addr-line">Guangzhou</div>, <div class="addr-line">Guangdong 510642</div>, <div class="country">People's Republic of China</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding</div>, <div class="addr-line">Guangzhou</div>, <div class="addr-line">Guangdong 510642</div>, <div class="country">People's Republic of China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiaomei+He">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=He X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;He X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xiaomei He</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiquan Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Animal Genetics, Breeding and Reproduction</div>, <div class="insititution">College of Animal Science, South China Agricultural University</div>, <div class="addr-line">Guangzhou</div>, <div class="addr-line">Guangdong 510642</div>, <div class="country">People's Republic of China</div></div><div class="aff"><div class="label -label">2</div><div class="insititution">Guangdong Provincial Key Lab of Agro-Animal Genomics and Molecular Breeding</div>, <div class="addr-line">Guangzhou</div>, <div class="addr-line">Guangdong 510642</div>, <div class="country">People's Republic of China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 20-85285703. Fax. +86 20-85280740. Email: <a href="mailto:xqzhang@scau.edu.cn" target="_blank">xqzhang@scau.edu.cn</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiquan+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xiquan Zhang</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Minoru Ko</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 18, Issue 5, 1 October 2011, Pages 343–351, <a href='https://doi.org/10.1093/dnares/dsr021'>https://doi.org/10.1093/dnares/dsr021</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">05 July 2011</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 April 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">08 June 2011</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="448814" href="/dnaresearch/article-pdf/18/5/343/1227029/dsr021.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Qinghua Nie, Meixia Fang, Xinzheng Jia, Wei Zhang, Xiaoning Zhou, Xiaomei He, Xiquan Zhang;  Analysis of Muscle and Ovary Transcriptome of <em>Sus scrofa</em>: Assembly, Annotation and Marker Discovery, <em>DNA Research</em>, Volume 18, Issue 5, 1 October 2011, Pages 343–351, <a href='https://doi.org/10.1093/dnares/dsr021'>https://doi.org/10.1093/dnares/dsr021</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/448814?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/448814?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/448814?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/448814?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/448814?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/448814?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/448814?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Analysis%20of%20Muscle%20and%20Ovary%20Transcriptome%20of%20%26lt%3Bem%26gt%3BSus%20scrofa%26lt%3B%2Fem%26gt%3B%3A%20Assembly%2C%20Annotation%20and%20Marker%20Discovery&amp;publicationDate=2011-07-05&amp;volumeNum=18&amp;issueNum=5&amp;author=Nie%2C%20Qinghua%3B%20Fang%2C%20Meixia&amp;startPage=343&amp;endPage=351&amp;contentId=10.1093%2Fdnares%2Fdsr021&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f18%2f5%2f343%2f448814" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6724209 id="6724209" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Pig (<em>Sus scrofa</em>) is an important organism for both agricultural and medical purpose. This study aims to investigate the <em>S. scrofa</em> transcriptome by the use of Roche 454 pyrosequencing. We obtained a total of 558 743 and 528 260 reads for the back-leg muscle and ovary tissue each. The overall 1 087 003 reads give rise to 421 767 341 bp total residues averaging 388 bp per read. The <em>de novo</em> assemblies yielded 11 057 contigs and 60 270 singletons for the back-leg muscle, 12 204 contigs and 70 192 singletons for the ovary and 18 938 contigs and 102 361 singletons for combined tissues. The overall GC content of <em>S. scrofa</em> transcriptome is 42.3% for assembled contigs. Alternative splicing was found within 4394 contigs, giving rise to 1267 isogroups or genes. A total of 56 589 transcripts are involved in molecular function (40 916), biological process (38 563), cellular component (35 787) by further gene ontology analyses. Comparison analyses showed that 336 and 553 genes had significant higher expression in the back-leg muscle and ovary each. In addition, we obtained a total of 24 214 single-nucleotide polymorphisms and 11 928 simple sequence repeats. These results contribute to the understanding of the genetic makeup of <em>S. scrofa</em> transcriptome and provide useful information for functional genomic research in future.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcriptome">transcriptome</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Sus scrofa&quot;"><em>Sus scrofa</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;454 sequencing&quot;">454 sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SNPs">SNPs</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="SSRs">SSRs</a></div>                            <h2 scrollto-destination=6724213 id="6724213" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>As a predominant domestic animal, pig (<em>Sus scrofa</em>) not only provides us plenty meat, but also is an important model organism for medical research. Understanding the genetic principle of growth and reproduction traits is helpful for <em>S. scrofa</em> production and also has scientific significance for basic biology and human medicine.<sup><a reveal-id="DSR021C1" data-open="DSR021C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Notable progress has been achieved to identify causative genes or single-nucleotide polymorphisms (SNPs) underlie complex traits in the past decades. A recessive missense mutation of <em>S. scrofa</em> ryanodine receptor (<em>RYR1</em>) gene was proved to induce malignant hyperthermia and halothane sensitivity in <em>S. scrofa</em>, which was effectively applied in selection program by many breeding farms.<sup><a reveal-id="DSR021C2" data-open="DSR021C2" class="link link-ref link-reveal xref-bibr">2</a></sup> In 2003, an SNP in intron 3 of insulin-like growth factor 2 was identified as a causative quantitative trait nucleotide for porcine muscle growth.<sup><a reveal-id="DSR021C3" data-open="DSR021C3" class="link link-ref link-reveal xref-bibr">3</a></sup></p><p>With the accomplishment of <em>S. scrofa</em> genome sequencing, it is realistic and efficient to identify genes or genetic markers underlying complex traits at whole-genome or transcriptome level. In 2005, a 0.66× coverage <em>S. scrofa</em> genome including 3.84 million shotgun sequences was generated from Hampshire, Yorkshire, Landrace, Duroc and ErHuaLian pigs, giving rise to 2.08 billion nucleotide bases, indicating that the <em>S. scrofa</em> genome is much closer to human than mouse does.<sup><a reveal-id="DSR021C4" data-open="DSR021C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Later, the complete <em>S. scrofa</em> genome was available online in 2009 (November 2009, SGSC Sscrofa9.2/susScr2; <a class="link link-uri" href="http://genome.ucsc.edu/cgi-bin/hgBlat">http://genome.ucsc.edu/cgi-bin/hgBlat</a>). From 4.8 million whole-genome shotgun sequences, 98 151 SNPs were predicted with one sequence representing the polymorphism, and most SNPs were confirmed by testing in three purebred boar lines and wild boar.<sup><a reveal-id="DSR021C5" data-open="DSR021C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Recently, a high-density SNP chip (Illumina Porcine 60K + SNP iSelect Beadchip) was designed and supplied for commercial use, which included 64 232 SNPs after reliable validation.<sup><a reveal-id="DSR021C6" data-open="DSR021C6" class="link link-ref link-reveal xref-bibr">6</a></sup> The developed SNP chip was subsequently used in whole-genome association analysis to identify genes for body composition and structural soundness traits in <em>S. scrofa</em>.<sup><a reveal-id="DSR021C7" data-open="DSR021C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>Based on genomic background, investigation of <em>S. scrofa</em> transcriptome is realistic and extremely useful for identification of candidate genes account for quantitative traits at the global level. The cDNA microarray or gene chip is a common tool for transcriptomic analysis. Confirmed by real-time RT–PCR and association analyses, some candidate genes and SNP markers for <em>S. scrofa</em> reproduction traits have been identified with microarray profiling by Affymetrix Porcine Genechip™.<sup><a reveal-id="DSR021C8" data-open="DSR021C8" class="link link-ref link-reveal xref-bibr">8</a></sup> With the use of <em>S. scrofa</em> whole-genome 70-mer oligonucleotide microarray, 62 expression quantitative traits loci (eQTLs) were successfully identified from loin muscle tissue through global genome-wide linkage analysis.<sup><a reveal-id="DSR021C9" data-open="DSR021C9" class="link link-ref link-reveal xref-bibr">9</a></sup> Nevertheless, cDNA microarray has some limitations as it fails to recognize new genes (transcripts) and sequence variations.</p><p>RNA sequencing (RNA-seq) is a new but efficient technology for the thorough investigation on transcriptome. With the rapid development of second-generation sequencing, RNA-seq becomes more efficient and less costive by some latterly developed platforms, i.e. Roche 454, Illuminate Solexa GA IIx, Life Technology SOLID, Helicos Biosciences tSMS and others.<sup><a reveal-id="DSR021C10" data-open="DSR021C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The Roche 454 can generate long reads and is generally used in transcriptome analysis in human,<sup><a reveal-id="DSR021C11" data-open="DSR021C11" class="link link-ref link-reveal xref-bibr">11</a>,<a reveal-id="DSR021C12" data-open="DSR021C12" class="link link-ref link-reveal xref-bibr">12</a></sup> mammals,<sup><a reveal-id="DSR021C13" data-open="DSR021C13" class="link link-ref link-reveal xref-bibr">13</a></sup> insects,<sup><a reveal-id="DSR021C14" data-open="DSR021C14" class="link link-ref link-reveal xref-bibr">14</a></sup> fish,<sup><a reveal-id="DSR021C15" data-open="DSR021C15" class="link link-ref link-reveal xref-bibr">15</a></sup> plants<sup><a reveal-id="DSR021C16" data-open="DSR021C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSR021C17" data-open="DSR021C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and microorganisms.<sup><a reveal-id="DSR021C18" data-open="DSR021C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p><p>Until now, reports on <em>S. scrofa</em> transcriptome by RNA-seq technology are very limited. In this study, we performed 454 pyrosequencing of muscle and ovary tissues to characterize <em>S. scrofa</em> transcriptome and to identify potential markers for growth and reproduction traits.</p>                            <h2 scrollto-destination=6724222 id="6724222" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6724223 id="6724223" class="section-title" data-legacy-id=s2a>2.1. Animal and RNA preparation</h3>
<p>One landrace female at 6-month age was subject to transcriptomic analysis of <em>S. scrofa</em>. Landrace is a typical commercial <em>S. scrofa</em> strain and widely used in domestic livestock production. The animal was slaughtered quickly to collect two tissues of the ovary and back-leg muscle. The fresh tissues were steeped in liquid nitrogen immediately after collection, and then kept at −80°C refrigerator (Thermo Forma, USA) for preservation before use. Trizol (Invitrogen, CA, USA) was used to isolate total RNA following the manufacturer's protocols.</p>                            <h3 scrollto-destination=6724225 id="6724225" class="section-title" data-legacy-id=s2b>2.2. Construction of cDNA library and 454 sequencing</h3>
<p>Approximately 10-µg total RNA was delivered to Shanghai Majorbio Bio-pharm Biotechnology Co., Ltd. (Shanghai, China) for the construction of a cDNA library. RNA quality was assessed by 260/280 and 260/230 ratios with the Agilent 2100 Bioanalyzer. The SMART cDNA library construction kit (Clontech, Mountain View, CA, USA) was used to construct the cDNA library of the muscle and ovary tissues following the manufacturer's protocol step by step. cDNA was sheared by nebulization and DNA bands (500–800 bp) were extracted from gel after agarose gel electrophoresis. The obtained DNA was purified, blunt ended, ligated to adapters and finally small fragments were removed. The quality control of a double DNA library was performed using High Sensitivity Chip (Agilent Technologies). The concentration was examined by TBS 380 Fluorometer. One-plate whole run sequencing was performed on the GS FLX Titanium chemistry (Roche Diagnostics, Indianapolis, IN, USA) by Shanghai Majorbio Bio-pharm Biotechnology Co., Ltd. following the manufacturer's protocol.</p>                            <h3 scrollto-destination=6724227 id="6724227" class="section-title" data-legacy-id=s2c>2.3. Bioinformatic analysis</h3>
                            <h4 scrollto-destination=6724228 id="6724228" class="section-title" data-legacy-id=s2c1>2.3.1. Reads trimming and assemble</h4>
<p>For each of the sequencing reads, low-quality bases and the sequencing adapter were trimmed using LUCY and SeqClean. The remained 454 reads of the ovary and back-leg muscle were first assembled using the Newbler software using default parameters. The combined reads of the ovary and back-leg muscle were also assembled using the Newbler software.</p>                            <h4 scrollto-destination=6724230 id="6724230" class="section-title" data-legacy-id=s2c2>2.3.2. Assemble of EST</h4>
<p>To improve the assembler quality, we collected the ESTs of <em>S. scrofa</em> from PigEST database (<a class="link link-uri" href="http://pigest.ku.dk/download/index.html">http://pigest.ku.dk/download/index.html</a>), which included 398 837 Pub EST sequences and 823 871 Sino-Danish <em>S. scrofa</em> EST sequences. All the ESTs from Roche 454 and PigEST databases were used to run the final assemble of <em>S. scrofa</em>. We chosen the trans-Abyss software for the final assemble, which assembled the ESTs through combining the results of different Kmer parameters. The unigenes with &gt;100 bp in length were used for the subsequent analysis. Moreover, all ESTs were mapped to the <em>S. scrofa</em> draft genome version 9 (download from Ensembl ftp server <a class="link link-uri" href="http://asia.ensembl.org/info/data/ftp/index.html">http://asia.ensembl.org/info/data/ftp/index.html</a>). ESTs were considered to be mapped successfully if it had over 95% identities to the corresponding genome sequences.</p>                            <h4 scrollto-destination=6724232 id="6724232" class="section-title" data-legacy-id=s2c3>2.3.3. Transcriptome annotation</h4>
<p>The unigenes were compared with the protein non-redundant database using BlastX<sup><a reveal-id="DSR021C19" data-open="DSR021C19" class="link link-ref link-reveal xref-bibr">19</a></sup> with <em>E</em>-value &lt; 1.0 × 10<sup>−5</sup> (<em>E</em>-values &lt;1.0 × 10<sup>−5</sup> were considered as a significant level). Gene ontology (GO) terms<sup><a reveal-id="DSR021C20" data-open="DSR021C20" class="link link-ref link-reveal xref-bibr">20</a></sup> were extracted from the best hits obtained from the BlastX against the nr database (<em>E</em>-value ≤ 1.0 × 10<sup>−6</sup>) using blast2go and then were sorted for the GO categories using in-house perl scripts. The metabolic pathway was performed using Kyoto Encyclopaedia of genes and genomes.</p>                            <h4 scrollto-destination=6724234 id="6724234" class="section-title" data-legacy-id=s2c4>2.3.4. Expression analysis</h4>
<p>EST reads from the ovary and back-leg muscle were mapped to a unigene sequence using the SSAHA software, respectively. The expression of each unigene was calculated using the numbers of reads with a specific match. Genes with different expression were identified using R package DGEseq.<sup><a reveal-id="DSR021C21" data-open="DSR021C21" class="link link-ref link-reveal xref-bibr">21</a></sup></p>                            <h4 scrollto-destination=6724236 id="6724236" class="section-title" data-legacy-id=s2c5>2.3.5. Bioinformatic mining of microsatellites and SNP makers</h4>
<p>The unigene sequences were screened for microsatellites using software MISA (MicroSAtellite, <a class="link link-uri" href="http://pgrc.ipk-gatersleben.de/misa/">http://pgrc.ipk-gatersleben.de/misa/</a>). All the 454 reads were mapped to the unigenes using SSAHA. The SNPs were extracted using VarScan with the default parameter only when both alleles were detected from 454 reads. The released <em>S. scrofa</em> genome (download from Ensembl ftp server <a class="link link-uri" href="http://asia.ensembl.org/info/data/ftp/index.html">http://asia.ensembl.org/info/data/ftp/index.html</a>) was used to confirm and locate the SNPs. Only those SNPs that both specifically match a certain genomic region and have a minor allelic frequency no less than 20% are included in analysis.</p>                            <h3 scrollto-destination=6724238 id="6724238" class="section-title" data-legacy-id=s2d>2.4. Data deposition</h3>
<p>The Roche 454 reads of <em>S. scrofa</em> by this study are now available from Animal Genomics databases by National Animal Genome Research Program (NAGRP, USA) with URL of <a class="link link-uri" href="http://www.animalgenome.org/repository/pub/SCAU2011.0502/">http://www.animalgenome.org/repository/pub/SCAU2011.0502/</a>.</p>                            <h2 scrollto-destination=6724240 id="6724240" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6724241 id="6724241" class="section-title" data-legacy-id=s3a>3.1. 454 sequencing and assembly</h3>
<p>In this study, we constructed two cDNA libraries and subsequently obtained two sets of transcriptomic reads for the back-leg muscle and ovary each. The schematic of 454 EST analyses is showed by Fig. <a reveal-id="DSR021F1" data-open="DSR021F1" class="link link-reveal link-table xref-fig">1</a>. For the back-leg muscle, Roche 454 sequencing yielded a total of 558 743 reads with a total nucleotide size of 219 021 745 bp, giving rise to average 392 bp per read (Table <a reveal-id="DSR021TB1" data-open="DSR021TB1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). For the ovary, meanwhile, we obtained 528 260 reads for 202 745 596 bp in total, giving rise to average 384 bp per read (Table <a reveal-id="DSR021TB1" data-open="DSR021TB1" class="link link-reveal link-table xref-fig">1</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>). </p>                        <a id="6724243" scrollto-destination="6724243"></a>
<div content-id="DSR021TB1" class="table-modal"><div class="table-wrap"><div id="DSR021TB1" class="table-wrap-title" data-id="DSR021TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Draft sequence data by 454 sequencing</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Types </th><th>Muscle (RL4) </th><th>Ovary (RL10) </th><th>Combined tissues </th></tr></thead><tbody><tr><td>Number of reads </td><td>558 743 </td><td>528 260 </td><td>1 087 003 </td></tr><tr><td>Total residues (bp) </td><td>219 021 745 </td><td>202 745 596 </td><td>421 767 341 </td></tr><tr><td>Smallest (bp) </td><td>27 </td><td>23 </td><td>23 </td></tr><tr><td>Largest (bp) </td><td>772 </td><td>813 </td><td>813 </td></tr><tr><td>Average length (bp) </td><td>392 </td><td>384 </td><td>388 </td></tr></tbody></table></div><div content-id="DSR021TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Types </th><th>Muscle (RL4) </th><th>Ovary (RL10) </th><th>Combined tissues </th></tr></thead><tbody><tr><td>Number of reads </td><td>558 743 </td><td>528 260 </td><td>1 087 003 </td></tr><tr><td>Total residues (bp) </td><td>219 021 745 </td><td>202 745 596 </td><td>421 767 341 </td></tr><tr><td>Smallest (bp) </td><td>27 </td><td>23 </td><td>23 </td></tr><tr><td>Largest (bp) </td><td>772 </td><td>813 </td><td>813 </td></tr><tr><td>Average length (bp) </td><td>392 </td><td>384 </td><td>388 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6724243" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR021TB1" class="table-wrap-title" data-id="DSR021TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Draft sequence data by 454 sequencing</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Types </th><th>Muscle (RL4) </th><th>Ovary (RL10) </th><th>Combined tissues </th></tr></thead><tbody><tr><td>Number of reads </td><td>558 743 </td><td>528 260 </td><td>1 087 003 </td></tr><tr><td>Total residues (bp) </td><td>219 021 745 </td><td>202 745 596 </td><td>421 767 341 </td></tr><tr><td>Smallest (bp) </td><td>27 </td><td>23 </td><td>23 </td></tr><tr><td>Largest (bp) </td><td>772 </td><td>813 </td><td>813 </td></tr><tr><td>Average length (bp) </td><td>392 </td><td>384 </td><td>388 </td></tr></tbody></table></div><div content-id="DSR021TB1" class="table-modal"><table><thead name="thead" align=""><tr><th>Types </th><th>Muscle (RL4) </th><th>Ovary (RL10) </th><th>Combined tissues </th></tr></thead><tbody><tr><td>Number of reads </td><td>558 743 </td><td>528 260 </td><td>1 087 003 </td></tr><tr><td>Total residues (bp) </td><td>219 021 745 </td><td>202 745 596 </td><td>421 767 341 </td></tr><tr><td>Smallest (bp) </td><td>27 </td><td>23 </td><td>23 </td></tr><tr><td>Largest (bp) </td><td>772 </td><td>813 </td><td>813 </td></tr><tr><td>Average length (bp) </td><td>392 </td><td>384 </td><td>388 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6724243" target="_blank">View Large</a></div></div>                        <a id="6724244" scrollto-destination="6724244"></a>
<div id="DSR021F1" data-id="DSR021F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/m_dsr02101.jpeg?Expires=2147483647&Signature=irA4ZsfgleAgzmj-yWNzHdAQpEkVY4nIbIXAWj~CpvR4opb7BuU3ktzMOlK4DVALfexloirUzTK9syT1aDB9g1wLcEGY-ggZb2hvhUPEce1yz2rwb5Gq1WyCmeJQbote-LrfgwfnpsSsB3JgwSav0qq0dFAfP-JCdJQ0cJANY2nbEPXO1HgSpDpGQp8i1rbNFU5zYpzWhFtdW~F-vc5Id4V19UvMoPX506zMtbcRiAbU~n9nSJ0bGJRFNFRMV4baWHZPMl1M3WapZgMjv~YPdlQnaAaBc-YGOb3kIBjIupsO4QMcbD2w6XxHJKtoFQp62WYjlC-UMf8oufcs~afUyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic of 454 EST analyses. The steps include 454 sequencing, assembly of reads into contigs and isogroups, GO annotation, KEGG analysis and discovery of SNPs and SSRs." path-from-xml="dsr02101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6724244/dsr02101.jpeg" path-from-xml="dsr02101.jpeg" target="_blank">View large</a><a section="6724244" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/dsr02101.jpeg?Expires=2147483647&Signature=mz2Mc4EVHgkxZcv1zfyuMLgLl0SGFIzLNqhuzDneYRG8qu9~Q4A~n-ZfMsdIa2CM-dBoKrhYoBZNfw02B8~Y7F7MmH9FFaZFKXoW6TKmUkBi~baNEwQSyhr94gV3PeCk4-xN0-SkTXOzFIgmGflROiDbOSHQFBWd6rTXi56aqsFfap3ahYYJyAssEP9kCUqq6itmrUCWh1abzWHzf2xhRKrihRnp5Z8QzuPyl6D9u-uEp9u-mBe1PMJUt4Q~nyd1qJwZlkMpv--0JkdOYpOGguF8RxuZ2fyyBreP2MU93rRxdCRNC3qTnNPVQkbB6u1zA1A4tcIEUuP3svWQb7bs2Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6724244&ar=448814&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic of 454 EST analyses. The steps include 454 sequencing, assembly of reads into contigs and isogroups, GO annotation, KEGG analysis and discovery of SNPs and SSRs.</p></div></div><div content-id="DSR021F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/m_dsr02101.jpeg?Expires=2147483647&Signature=irA4ZsfgleAgzmj-yWNzHdAQpEkVY4nIbIXAWj~CpvR4opb7BuU3ktzMOlK4DVALfexloirUzTK9syT1aDB9g1wLcEGY-ggZb2hvhUPEce1yz2rwb5Gq1WyCmeJQbote-LrfgwfnpsSsB3JgwSav0qq0dFAfP-JCdJQ0cJANY2nbEPXO1HgSpDpGQp8i1rbNFU5zYpzWhFtdW~F-vc5Id4V19UvMoPX506zMtbcRiAbU~n9nSJ0bGJRFNFRMV4baWHZPMl1M3WapZgMjv~YPdlQnaAaBc-YGOb3kIBjIupsO4QMcbD2w6XxHJKtoFQp62WYjlC-UMf8oufcs~afUyg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic of 454 EST analyses. The steps include 454 sequencing, assembly of reads into contigs and isogroups, GO annotation, KEGG analysis and discovery of SNPs and SSRs." path-from-xml="dsr02101.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6724244/dsr02101.jpeg" path-from-xml="dsr02101.jpeg" target="_blank">View large</a><a section="6724244" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/dsr02101.jpeg?Expires=2147483647&Signature=mz2Mc4EVHgkxZcv1zfyuMLgLl0SGFIzLNqhuzDneYRG8qu9~Q4A~n-ZfMsdIa2CM-dBoKrhYoBZNfw02B8~Y7F7MmH9FFaZFKXoW6TKmUkBi~baNEwQSyhr94gV3PeCk4-xN0-SkTXOzFIgmGflROiDbOSHQFBWd6rTXi56aqsFfap3ahYYJyAssEP9kCUqq6itmrUCWh1abzWHzf2xhRKrihRnp5Z8QzuPyl6D9u-uEp9u-mBe1PMJUt4Q~nyd1qJwZlkMpv--0JkdOYpOGguF8RxuZ2fyyBreP2MU93rRxdCRNC3qTnNPVQkbB6u1zA1A4tcIEUuP3svWQb7bs2Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6724244&ar=448814&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02101.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic of 454 EST analyses. The steps include 454 sequencing, assembly of reads into contigs and isogroups, GO annotation, KEGG analysis and discovery of SNPs and SSRs.</p></div></div><p>By assemble analysis, we obtained 71 327 ESTs (11 057 contigs averaging 787 bp and 60 270 singletons) for the back-leg muscle and 82 396 ESTs (12 204 contigs averaging 780 bp and 70 192 singletons) for the ovary, as well as 121 299 ESTs (18 938 contigs averaging 810 bp and 102 361 singletons) for combined tissues (Table <a reveal-id="DSR021TB2" data-open="DSR021TB2" class="link link-reveal link-table xref-fig">2</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Most of these contigs distributed in the 401–1500-bp region, and about half of them distributed in 401–700 bp for each of the back-leg muscle (51.35%), ovary (51.79%) and combined tissue (49.62%) (Table <a reveal-id="DSR021TB3" data-open="DSR021TB3" class="link link-reveal link-table xref-fig">3</a>; <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). </p>                        <a id="6724246" scrollto-destination="6724246"></a>
<div content-id="DSR021TB2" class="table-modal"><div class="table-wrap"><div id="DSR021TB2" class="table-wrap-title" data-id="DSR021TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary on assemble analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Types </th><th>Muscle (RL4) </th><th>Ovary (RL10) </th><th>Combined tissues </th></tr></thead><tbody><tr><td>Num of contigs </td><td>11 057 </td><td>12 204 </td><td>18 938 </td></tr><tr><td>Smallest (bp) </td><td>42 </td><td>44 </td><td>42 </td></tr><tr><td>Largest (bp) </td><td>3540 </td><td>3462 </td><td>4218 </td></tr><tr><td>Total length (bp) </td><td>8 703 645 </td><td>9 517 255 </td><td>15 332 944 </td></tr><tr><td>Average length (bp) </td><td>787 </td><td>780 </td><td>810 </td></tr><tr><td>Num of isogroups </td><td>9496 </td><td>10 440 </td><td>15 825 </td></tr><tr><td>Num of isogroups (contigs ≥ 2) </td><td>662 </td><td>719 </td><td>1267 </td></tr><tr><td>Singleton </td><td>60 270 </td><td>70 192 </td><td>102 361 </td></tr><tr><td>Total </td><td>71 327 </td><td>82 396 </td><td>121 299 </td></tr></tbody></table></div><div content-id="DSR021TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Types </th><th>Muscle (RL4) </th><th>Ovary (RL10) </th><th>Combined tissues </th></tr></thead><tbody><tr><td>Num of contigs </td><td>11 057 </td><td>12 204 </td><td>18 938 </td></tr><tr><td>Smallest (bp) </td><td>42 </td><td>44 </td><td>42 </td></tr><tr><td>Largest (bp) </td><td>3540 </td><td>3462 </td><td>4218 </td></tr><tr><td>Total length (bp) </td><td>8 703 645 </td><td>9 517 255 </td><td>15 332 944 </td></tr><tr><td>Average length (bp) </td><td>787 </td><td>780 </td><td>810 </td></tr><tr><td>Num of isogroups </td><td>9496 </td><td>10 440 </td><td>15 825 </td></tr><tr><td>Num of isogroups (contigs ≥ 2) </td><td>662 </td><td>719 </td><td>1267 </td></tr><tr><td>Singleton </td><td>60 270 </td><td>70 192 </td><td>102 361 </td></tr><tr><td>Total </td><td>71 327 </td><td>82 396 </td><td>121 299 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6724246" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR021TB2" class="table-wrap-title" data-id="DSR021TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Summary on assemble analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Types </th><th>Muscle (RL4) </th><th>Ovary (RL10) </th><th>Combined tissues </th></tr></thead><tbody><tr><td>Num of contigs </td><td>11 057 </td><td>12 204 </td><td>18 938 </td></tr><tr><td>Smallest (bp) </td><td>42 </td><td>44 </td><td>42 </td></tr><tr><td>Largest (bp) </td><td>3540 </td><td>3462 </td><td>4218 </td></tr><tr><td>Total length (bp) </td><td>8 703 645 </td><td>9 517 255 </td><td>15 332 944 </td></tr><tr><td>Average length (bp) </td><td>787 </td><td>780 </td><td>810 </td></tr><tr><td>Num of isogroups </td><td>9496 </td><td>10 440 </td><td>15 825 </td></tr><tr><td>Num of isogroups (contigs ≥ 2) </td><td>662 </td><td>719 </td><td>1267 </td></tr><tr><td>Singleton </td><td>60 270 </td><td>70 192 </td><td>102 361 </td></tr><tr><td>Total </td><td>71 327 </td><td>82 396 </td><td>121 299 </td></tr></tbody></table></div><div content-id="DSR021TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Types </th><th>Muscle (RL4) </th><th>Ovary (RL10) </th><th>Combined tissues </th></tr></thead><tbody><tr><td>Num of contigs </td><td>11 057 </td><td>12 204 </td><td>18 938 </td></tr><tr><td>Smallest (bp) </td><td>42 </td><td>44 </td><td>42 </td></tr><tr><td>Largest (bp) </td><td>3540 </td><td>3462 </td><td>4218 </td></tr><tr><td>Total length (bp) </td><td>8 703 645 </td><td>9 517 255 </td><td>15 332 944 </td></tr><tr><td>Average length (bp) </td><td>787 </td><td>780 </td><td>810 </td></tr><tr><td>Num of isogroups </td><td>9496 </td><td>10 440 </td><td>15 825 </td></tr><tr><td>Num of isogroups (contigs ≥ 2) </td><td>662 </td><td>719 </td><td>1267 </td></tr><tr><td>Singleton </td><td>60 270 </td><td>70 192 </td><td>102 361 </td></tr><tr><td>Total </td><td>71 327 </td><td>82 396 </td><td>121 299 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6724246" target="_blank">View Large</a></div></div>                        <a id="6724247" scrollto-destination="6724247"></a>
<div content-id="DSR021TB3" class="table-modal"><div class="table-wrap"><div id="DSR021TB3" class="table-wrap-title" data-id="DSR021TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Statistics of contigs by 454 sequencing</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Length </th><th colspan="2">Muscle<hr /> </th><th colspan="2">Ovary<hr /> </th><th colspan="2">Combined tissues<hr /> </th></tr><tr><th> </th><th>Numbers </th><th>Per cent (%) </th><th>Numbers </th><th>Per cent (%) </th><th>Numbers </th><th>Per cent (%) </th></tr></thead><tbody><tr><td>1–100 </td><td>12 </td><td>0.11 </td><td>11 </td><td>0.09 </td><td>12 </td><td>0.06 </td></tr><tr><td>101–400 </td><td>225 </td><td>2.03 </td><td>309 </td><td>2.53 </td><td>411 </td><td>2.17 </td></tr><tr><td>401–700 </td><td>5678 </td><td>51.35 </td><td>6320 </td><td>51.79 </td><td>9397 </td><td>49.62 </td></tr><tr><td>701–1000 </td><td>2813 </td><td>25.44 </td><td>3050 </td><td>24.99 </td><td>4760 </td><td>25.13 </td></tr><tr><td>1001–1500 </td><td>1756 </td><td>15.88 </td><td>1928 </td><td>15.80 </td><td>3187 </td><td>16.83 </td></tr><tr><td>1501–2000 </td><td>449 </td><td>4.06 </td><td>474 </td><td>3.88 </td><td>864 </td><td>4.56 </td></tr><tr><td>&gt;2000 </td><td>124 </td><td>1.12 </td><td>112 </td><td>0.92 </td><td>307 </td><td>1.62 </td></tr><tr><td>Total </td><td>11 057 </td><td>100 </td><td>12 204 </td><td>100 </td><td>18 938 </td><td>100 </td></tr></tbody></table></div><div content-id="DSR021TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Length </th><th colspan="2">Muscle<hr /> </th><th colspan="2">Ovary<hr /> </th><th colspan="2">Combined tissues<hr /> </th></tr><tr><th> </th><th>Numbers </th><th>Per cent (%) </th><th>Numbers </th><th>Per cent (%) </th><th>Numbers </th><th>Per cent (%) </th></tr></thead><tbody><tr><td>1–100 </td><td>12 </td><td>0.11 </td><td>11 </td><td>0.09 </td><td>12 </td><td>0.06 </td></tr><tr><td>101–400 </td><td>225 </td><td>2.03 </td><td>309 </td><td>2.53 </td><td>411 </td><td>2.17 </td></tr><tr><td>401–700 </td><td>5678 </td><td>51.35 </td><td>6320 </td><td>51.79 </td><td>9397 </td><td>49.62 </td></tr><tr><td>701–1000 </td><td>2813 </td><td>25.44 </td><td>3050 </td><td>24.99 </td><td>4760 </td><td>25.13 </td></tr><tr><td>1001–1500 </td><td>1756 </td><td>15.88 </td><td>1928 </td><td>15.80 </td><td>3187 </td><td>16.83 </td></tr><tr><td>1501–2000 </td><td>449 </td><td>4.06 </td><td>474 </td><td>3.88 </td><td>864 </td><td>4.56 </td></tr><tr><td>&gt;2000 </td><td>124 </td><td>1.12 </td><td>112 </td><td>0.92 </td><td>307 </td><td>1.62 </td></tr><tr><td>Total </td><td>11 057 </td><td>100 </td><td>12 204 </td><td>100 </td><td>18 938 </td><td>100 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6724247" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR021TB3" class="table-wrap-title" data-id="DSR021TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Statistics of contigs by 454 sequencing</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Length </th><th colspan="2">Muscle<hr /> </th><th colspan="2">Ovary<hr /> </th><th colspan="2">Combined tissues<hr /> </th></tr><tr><th> </th><th>Numbers </th><th>Per cent (%) </th><th>Numbers </th><th>Per cent (%) </th><th>Numbers </th><th>Per cent (%) </th></tr></thead><tbody><tr><td>1–100 </td><td>12 </td><td>0.11 </td><td>11 </td><td>0.09 </td><td>12 </td><td>0.06 </td></tr><tr><td>101–400 </td><td>225 </td><td>2.03 </td><td>309 </td><td>2.53 </td><td>411 </td><td>2.17 </td></tr><tr><td>401–700 </td><td>5678 </td><td>51.35 </td><td>6320 </td><td>51.79 </td><td>9397 </td><td>49.62 </td></tr><tr><td>701–1000 </td><td>2813 </td><td>25.44 </td><td>3050 </td><td>24.99 </td><td>4760 </td><td>25.13 </td></tr><tr><td>1001–1500 </td><td>1756 </td><td>15.88 </td><td>1928 </td><td>15.80 </td><td>3187 </td><td>16.83 </td></tr><tr><td>1501–2000 </td><td>449 </td><td>4.06 </td><td>474 </td><td>3.88 </td><td>864 </td><td>4.56 </td></tr><tr><td>&gt;2000 </td><td>124 </td><td>1.12 </td><td>112 </td><td>0.92 </td><td>307 </td><td>1.62 </td></tr><tr><td>Total </td><td>11 057 </td><td>100 </td><td>12 204 </td><td>100 </td><td>18 938 </td><td>100 </td></tr></tbody></table></div><div content-id="DSR021TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Length </th><th colspan="2">Muscle<hr /> </th><th colspan="2">Ovary<hr /> </th><th colspan="2">Combined tissues<hr /> </th></tr><tr><th> </th><th>Numbers </th><th>Per cent (%) </th><th>Numbers </th><th>Per cent (%) </th><th>Numbers </th><th>Per cent (%) </th></tr></thead><tbody><tr><td>1–100 </td><td>12 </td><td>0.11 </td><td>11 </td><td>0.09 </td><td>12 </td><td>0.06 </td></tr><tr><td>101–400 </td><td>225 </td><td>2.03 </td><td>309 </td><td>2.53 </td><td>411 </td><td>2.17 </td></tr><tr><td>401–700 </td><td>5678 </td><td>51.35 </td><td>6320 </td><td>51.79 </td><td>9397 </td><td>49.62 </td></tr><tr><td>701–1000 </td><td>2813 </td><td>25.44 </td><td>3050 </td><td>24.99 </td><td>4760 </td><td>25.13 </td></tr><tr><td>1001–1500 </td><td>1756 </td><td>15.88 </td><td>1928 </td><td>15.80 </td><td>3187 </td><td>16.83 </td></tr><tr><td>1501–2000 </td><td>449 </td><td>4.06 </td><td>474 </td><td>3.88 </td><td>864 </td><td>4.56 </td></tr><tr><td>&gt;2000 </td><td>124 </td><td>1.12 </td><td>112 </td><td>0.92 </td><td>307 </td><td>1.62 </td></tr><tr><td>Total </td><td>11 057 </td><td>100 </td><td>12 204 </td><td>100 </td><td>18 938 </td><td>100 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6724247" target="_blank">View Large</a></div></div><p>Contigs were further assembled to 9496 isogroups averaging 662 bp for the back-leg muscle, 10 440 isogroups averaging 719 bp for the ovary and 15 825 isogroups averaging 1267 bp for combined tissues (Table <a reveal-id="DSR021TB2" data-open="DSR021TB2" class="link link-reveal link-table xref-fig">2</a>). In average, each isogroup has 1.16, 1.17 and 1.20 contigs for the back-leg muscle (11 057 contigs for 9496 isogroups), ovary (12 204 contigs for 10 440 isogroups) and combined tissues (18 938 contigs for 15 825 isogroups), respectively. As far as all contigs were concerned, the concentration of A, T, C and G was 29.0, 28.7, 21.1 and 21.2%, respectively, giving rise to the overall GC content of 42.3% for <em>S. scrofa</em> transcriptome.</p><p>By comparison, our original 454 sequence with the <em>S. scrofa</em> genome, 246 628 ESTs could match the genome, and among them, 216 739 specifically locate on one region and 29 889 locate on two or more regions. As far as different chromosomes are compared, chromosome 1 (23 237 or 10.72%), 4 (17 087 or 7.88%), 2 (16 957 or 7.82%), 14 (16 724 or 7.72%) and 13 (15 738 or 7.26%) contained more transcripts than others, whereas X (4495 or 2.07%) and mitochondrial DNA (mtDNA; 1665 or 0.77%) had very fewer EST (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>).</p>                            <h3 scrollto-destination=6724250 id="6724250" class="section-title" data-legacy-id=s3b>3.2. Alternative splicing</h3>
<p>More contigs than isogroups are found because some contigs or called isocontigs are attributed to the same isogroups due to alternative splicing (Tables <a reveal-id="DSR021TB2" data-open="DSR021TB2" class="link link-reveal link-table xref-fig">2</a> and <a reveal-id="DSR021TB4" data-open="DSR021TB4" class="link link-reveal link-table xref-fig">4</a>). There are 7.0% (662 of 9496), 6.9% (719 of 10 440) and 8.0% (1267 of 15 825) isogroups have no less than two contigs in the back-leg muscle, ovary and combined tissues, respectively. The alternative-spliced isogroups in the back-leg muscle, ovary and combined tissues averagely have 3.4, 3.5, and 3.5 isocontigs, respectively (Table <a reveal-id="DSR021TB4" data-open="DSR021TB4" class="link link-reveal link-table xref-fig">4</a>). </p>                        <a id="6724252" scrollto-destination="6724252"></a>
<div content-id="DSR021TB4" class="table-modal"><div class="table-wrap"><div id="DSR021TB4" class="table-wrap-title" data-id="DSR021TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Variant transcripts by assemble analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>No.<sup>a</sup> </th><th colspan="3">Numbers of isogroups (contigs)<hr /> </th></tr><tr><th> </th><th>Muscle </th><th>Ovary </th><th>Combined tissues </th></tr></thead><tbody><tr><td>2 </td><td>452 (904) </td><td>498 (996) </td><td>857 (1714) </td></tr><tr><td>3 </td><td>71 (142) </td><td>76 (152) </td><td>143 (286) </td></tr><tr><td>4 </td><td>64 (128) </td><td>58 (116) </td><td>115 (230) </td></tr><tr><td>5 </td><td>20 (40) </td><td>12 (24) </td><td>29 (58) </td></tr><tr><td>6 </td><td>18 (36) </td><td>20 (40) </td><td>29 (58) </td></tr><tr><td>7 </td><td>6 (12) </td><td>8 (16) </td><td>17 (34) </td></tr><tr><td>8 </td><td>5 (10) </td><td>10 (20) </td><td>15 (30) </td></tr><tr><td>9 </td><td>4 (8) </td><td>8 (72) </td><td>14 (28) </td></tr><tr><td>≥10 </td><td>22 (529) </td><td>29 (644) </td><td>48 (1106) </td></tr><tr><td>In total (≥2) </td><td>662 (2228) </td><td>719 (2488) </td><td>1267 (4393) </td></tr></tbody></table></div><div content-id="DSR021TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>No.<sup>a</sup> </th><th colspan="3">Numbers of isogroups (contigs)<hr /> </th></tr><tr><th> </th><th>Muscle </th><th>Ovary </th><th>Combined tissues </th></tr></thead><tbody><tr><td>2 </td><td>452 (904) </td><td>498 (996) </td><td>857 (1714) </td></tr><tr><td>3 </td><td>71 (142) </td><td>76 (152) </td><td>143 (286) </td></tr><tr><td>4 </td><td>64 (128) </td><td>58 (116) </td><td>115 (230) </td></tr><tr><td>5 </td><td>20 (40) </td><td>12 (24) </td><td>29 (58) </td></tr><tr><td>6 </td><td>18 (36) </td><td>20 (40) </td><td>29 (58) </td></tr><tr><td>7 </td><td>6 (12) </td><td>8 (16) </td><td>17 (34) </td></tr><tr><td>8 </td><td>5 (10) </td><td>10 (20) </td><td>15 (30) </td></tr><tr><td>9 </td><td>4 (8) </td><td>8 (72) </td><td>14 (28) </td></tr><tr><td>≥10 </td><td>22 (529) </td><td>29 (644) </td><td>48 (1106) </td></tr><tr><td>In total (≥2) </td><td>662 (2228) </td><td>719 (2488) </td><td>1267 (4393) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Numbers of contigs per isogroup.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6724252" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR021TB4" class="table-wrap-title" data-id="DSR021TB4"><div class="label -label">Table 4.</div><div class="caption"><p>Variant transcripts by assemble analysis</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>No.<sup>a</sup> </th><th colspan="3">Numbers of isogroups (contigs)<hr /> </th></tr><tr><th> </th><th>Muscle </th><th>Ovary </th><th>Combined tissues </th></tr></thead><tbody><tr><td>2 </td><td>452 (904) </td><td>498 (996) </td><td>857 (1714) </td></tr><tr><td>3 </td><td>71 (142) </td><td>76 (152) </td><td>143 (286) </td></tr><tr><td>4 </td><td>64 (128) </td><td>58 (116) </td><td>115 (230) </td></tr><tr><td>5 </td><td>20 (40) </td><td>12 (24) </td><td>29 (58) </td></tr><tr><td>6 </td><td>18 (36) </td><td>20 (40) </td><td>29 (58) </td></tr><tr><td>7 </td><td>6 (12) </td><td>8 (16) </td><td>17 (34) </td></tr><tr><td>8 </td><td>5 (10) </td><td>10 (20) </td><td>15 (30) </td></tr><tr><td>9 </td><td>4 (8) </td><td>8 (72) </td><td>14 (28) </td></tr><tr><td>≥10 </td><td>22 (529) </td><td>29 (644) </td><td>48 (1106) </td></tr><tr><td>In total (≥2) </td><td>662 (2228) </td><td>719 (2488) </td><td>1267 (4393) </td></tr></tbody></table></div><div content-id="DSR021TB4" class="table-modal"><table><thead name="thead" align=""><tr><th>No.<sup>a</sup> </th><th colspan="3">Numbers of isogroups (contigs)<hr /> </th></tr><tr><th> </th><th>Muscle </th><th>Ovary </th><th>Combined tissues </th></tr></thead><tbody><tr><td>2 </td><td>452 (904) </td><td>498 (996) </td><td>857 (1714) </td></tr><tr><td>3 </td><td>71 (142) </td><td>76 (152) </td><td>143 (286) </td></tr><tr><td>4 </td><td>64 (128) </td><td>58 (116) </td><td>115 (230) </td></tr><tr><td>5 </td><td>20 (40) </td><td>12 (24) </td><td>29 (58) </td></tr><tr><td>6 </td><td>18 (36) </td><td>20 (40) </td><td>29 (58) </td></tr><tr><td>7 </td><td>6 (12) </td><td>8 (16) </td><td>17 (34) </td></tr><tr><td>8 </td><td>5 (10) </td><td>10 (20) </td><td>15 (30) </td></tr><tr><td>9 </td><td>4 (8) </td><td>8 (72) </td><td>14 (28) </td></tr><tr><td>≥10 </td><td>22 (529) </td><td>29 (644) </td><td>48 (1106) </td></tr><tr><td>In total (≥2) </td><td>662 (2228) </td><td>719 (2488) </td><td>1267 (4393) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Numbers of contigs per isogroup.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6724252" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6724253 id="6724253" class="section-title" data-legacy-id=s3c>3.3. GO assignments</h3>
<p>A total of 56 589 transcripts of <em>S. scrofa</em> were assigned for GO analysis based on matches with sequences whose functions were known previously. Among these transcripts, 45 846 transcripts were successfully annotated with confident matches. As many as 38 563 transcripts are involved in biological process, including cellular process (34 217 transcripts with percentages of 16.98%), metabolic process (29 471; 14.62%), biological regulation (16 944; 8.41%), regulation of biological process (16 002; 7.94%), multicellular organismal process (10 148; 5.04%), response to stimulus (10 023; 4.97%), localization (9680; 4.80%), cellular component organization or biogenesis (9276; 4.60%), developmental process (8610; 4.27%), establishment of localization (8393; 4.16%), signalling (7083; 3.51%), positive regulation of biological process (6576; 3.26%), negative regulation of biological process (6487; 3.22%) and signalling process (5517; 2.74%), as well as other activities (23 100; 11.46%) (Fig. <a reveal-id="DSR021F2" data-open="DSR021F2" class="link link-reveal link-table xref-fig">2</a>A). </p>                        <a id="6724255" scrollto-destination="6724255"></a>
<div id="DSR021F2" data-id="DSR021F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/m_dsr02102.jpeg?Expires=2147483647&Signature=mYZ1~uUlFy7wCnvjZQooZBb2qVjBj8S4oRzvY0Q3moBxcy-YVWZ2Rk9En6InSGVyxG4ta0oNSkwNSzcX3-hNJsfFXiltI4Tk59W4KluIZcHYdd7E9z3O5NoT88XP4ZJRmFK5quDrsJl-c3sa4iwC8k0EJ8yWrz7GVF8F4csQ6-NqlJPxwEbVZ0eJGs63ZXlVcaYplpHNsDcoynB83u1i4XHXPlH0E-39LOhM4h7zdzkQ9~T0ydGoKgbmhjYDqpX5wzPFhl5zG8doMTV3-REMf~l3KpvfanHl9MIibfp3fZlCgBBCQMrvXAmV~5P~vvXYK5GGKxysfJjF2y~tZoCvxg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional classification of S. scrofa transcriptome. (A) GO: Biological process. (B) Cellular component. (C) GO: Molecular function. In some cases, one transcript or gene has multiple functions." path-from-xml="dsr02102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6724255/dsr02102.jpeg" path-from-xml="dsr02102.jpeg" target="_blank">View large</a><a section="6724255" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/dsr02102.jpeg?Expires=2147483647&Signature=MzNQV~sqzNROxHcymXmdRnuDrFWZ-TZNne6OQaQETua3uMDoTmZGttva~3wgrz9kFqImHHnnOD07RX3faQY4iMAgduNIQA7UsqmuObybspaU~Ixo4N0RrL99h~RkwrZBkIGfDHuLp1jJNg7TTzN3uC0Z4MG99MEa9r8yQ1mkrWeLtvx5556UatS9NH1rxt5XaBNZwGoRs3adEEoOA2jPwFYhFnkcY09l4i98-61yeeEsD0TVqrj8BO~wZZBU5OnE-vxFqEoa2LF0NRiTzkphnhqH522fC7hXGbiainBEd7VPZlVGouFRnHHepf1hsxA6Eh22qnjwy4MdkBMTWs1zyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6724255&ar=448814&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional classification of <em>S. scrofa</em> transcriptome. (A) GO: Biological process. (B) Cellular component. (C) GO: Molecular function. In some cases, one transcript or gene has multiple functions.</p></div></div><div content-id="DSR021F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/m_dsr02102.jpeg?Expires=2147483647&Signature=mYZ1~uUlFy7wCnvjZQooZBb2qVjBj8S4oRzvY0Q3moBxcy-YVWZ2Rk9En6InSGVyxG4ta0oNSkwNSzcX3-hNJsfFXiltI4Tk59W4KluIZcHYdd7E9z3O5NoT88XP4ZJRmFK5quDrsJl-c3sa4iwC8k0EJ8yWrz7GVF8F4csQ6-NqlJPxwEbVZ0eJGs63ZXlVcaYplpHNsDcoynB83u1i4XHXPlH0E-39LOhM4h7zdzkQ9~T0ydGoKgbmhjYDqpX5wzPFhl5zG8doMTV3-REMf~l3KpvfanHl9MIibfp3fZlCgBBCQMrvXAmV~5P~vvXYK5GGKxysfJjF2y~tZoCvxg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Functional classification of S. scrofa transcriptome. (A) GO: Biological process. (B) Cellular component. (C) GO: Molecular function. In some cases, one transcript or gene has multiple functions." path-from-xml="dsr02102.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6724255/dsr02102.jpeg" path-from-xml="dsr02102.jpeg" target="_blank">View large</a><a section="6724255" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/dsr02102.jpeg?Expires=2147483647&Signature=MzNQV~sqzNROxHcymXmdRnuDrFWZ-TZNne6OQaQETua3uMDoTmZGttva~3wgrz9kFqImHHnnOD07RX3faQY4iMAgduNIQA7UsqmuObybspaU~Ixo4N0RrL99h~RkwrZBkIGfDHuLp1jJNg7TTzN3uC0Z4MG99MEa9r8yQ1mkrWeLtvx5556UatS9NH1rxt5XaBNZwGoRs3adEEoOA2jPwFYhFnkcY09l4i98-61yeeEsD0TVqrj8BO~wZZBU5OnE-vxFqEoa2LF0NRiTzkphnhqH522fC7hXGbiainBEd7VPZlVGouFRnHHepf1hsxA6Eh22qnjwy4MdkBMTWs1zyQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6724255&ar=448814&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsr02102.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Functional classification of <em>S. scrofa</em> transcriptome. (A) GO: Biological process. (B) Cellular component. (C) GO: Molecular function. In some cases, one transcript or gene has multiple functions.</p></div></div><p>Moreover, 35 787 transcripts are subject to a cellular component and could be divided into cell (33 486; 24.06%), cell part (33 483; 24.06%), organelle (25 767; 18.52%), organelle part (16 894; 12.14%), macromolecular complex (13 151; 9.45%), membrane-enclosed lumen (6469; 4.65%), extracellular region (5378; 3.86%), extracellular region part (3457; 2.48%) and others (1082; 0.78%) (Fig. <a reveal-id="DSR021F2" data-open="DSR021F2" class="link link-reveal link-table xref-fig">2</a>B). GO analysis also showed that 40 916 transcripts had potential molecular function, such as binding (36 550; 48.15%), catalytic activity (21 865; 28.8%), structural molecule activity (4385; 5.78%), transporter activity (3188; 4.2%), molecular transducer activity (2379; 3.13%), enzyme regulator activity (2337; 3.08%), transcription regulator activity (2224; 2.93%), nucleic acid binding transcription factor activity (1317; 1.74%), electron carrier activity (948; 1.25%) and others (714; 0.94%) (Fig. <a reveal-id="DSR021F2" data-open="DSR021F2" class="link link-reveal link-table xref-fig">2</a>C).</p>                            <h3 scrollto-destination=6724259 id="6724259" class="section-title" data-legacy-id=s3d>3.4. Metabolic pathways by KEGG analysis</h3>
<p>A total of 4268 transcripts are involved in 132 predicted KEGG metabolic pathways, and the numbers of transcripts in different pathways ranged from 1 to 1352. The top 20 pathways with EST numbers are shown in Table <a reveal-id="DSR021TB5" data-open="DSR021TB5" class="link link-reveal link-table xref-fig">5</a>, and the highest of the number of transcripts is involved in the biosynthesis of secondary metabolites. Ten biosynthesis pathways included biosynthesis of alkaloids derived from histidine and purine (417), biosynthesis of alkaloids derived from ornithine, lysine and nicotinic acid (368), biosynthesis of alkaloids derived from the shikimate pathway (380), biosynthesis of alkaloids derived from terpenoid and polyketide (399), biosynthesis of secondary metabolites (1352), biosynthesis of plant hormones (625), biosynthesis of phenylpropanoids (466), biosynthesis of terpenoids and steroids (401) and biosynthesis of unsaturated fatty acids (104), as well as biosynthesis of ansamycins (7). </p>                        <a id="6724263" scrollto-destination="6724263"></a>
<div content-id="DSR021TB5" class="table-modal"><div class="table-wrap"><div id="DSR021TB5" class="table-wrap-title" data-id="DSR021TB5"><div class="label -label">Table 5.</div><div class="caption"><p>The top 20 pathways with the highest EST numbers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>No. </th><th>Pathways </th><th>Number of ESTs </th></tr></thead><tbody><tr><td>1 </td><td>Biosynthesis of secondary metabolites </td><td>1352 </td></tr><tr><td>2 </td><td>Oxidative phosphorylation </td><td>1111 </td></tr><tr><td>3 </td><td>Microbial metabolism in diverse environments </td><td>964 </td></tr><tr><td>4 </td><td>Purine metabolism </td><td>690 </td></tr><tr><td>5 </td><td>Biosynthesis of plant hormones </td><td>625 </td></tr><tr><td>6 </td><td>Biosynthesis of phenylpropanoids </td><td>466 </td></tr><tr><td>7 </td><td>Biosynthesis of alkaloids derived from histidine and purine </td><td>417 </td></tr><tr><td>8 </td><td>Pyrimidine metabolism </td><td>408 </td></tr><tr><td>9 </td><td>Biosynthesis of terpenoids and steroids </td><td>401 </td></tr><tr><td>10 </td><td>Biosynthesis of alkaloids derived from terpenoid and polyketide </td><td>399 </td></tr><tr><td>11 </td><td>Methane metabolism </td><td>399 </td></tr><tr><td>12 </td><td>Biosynthesis of alkaloids derived from shikimate pathway </td><td>380 </td></tr><tr><td>13 </td><td>Glycolysis/gluconeogenesis </td><td>379 </td></tr><tr><td>14 </td><td>Biosynthesis of alkaloids derived from ornithine, lysine and nicotinic acid </td><td>368 </td></tr><tr><td>15 </td><td>Glutathione metabolism </td><td>318 </td></tr><tr><td>16 </td><td>Pyruvate metabolism </td><td>268 </td></tr><tr><td>17 </td><td>Arginine and proline metabolism </td><td>261 </td></tr><tr><td>18 </td><td>Glycerophospholipid metabolism </td><td>222 </td></tr><tr><td>19 </td><td>Fatty acid metabolism </td><td>221 </td></tr><tr><td>20 </td><td>Valine, leucine and isoleucine degradation </td><td>218 </td></tr></tbody></table></div><div content-id="DSR021TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>No. </th><th>Pathways </th><th>Number of ESTs </th></tr></thead><tbody><tr><td>1 </td><td>Biosynthesis of secondary metabolites </td><td>1352 </td></tr><tr><td>2 </td><td>Oxidative phosphorylation </td><td>1111 </td></tr><tr><td>3 </td><td>Microbial metabolism in diverse environments </td><td>964 </td></tr><tr><td>4 </td><td>Purine metabolism </td><td>690 </td></tr><tr><td>5 </td><td>Biosynthesis of plant hormones </td><td>625 </td></tr><tr><td>6 </td><td>Biosynthesis of phenylpropanoids </td><td>466 </td></tr><tr><td>7 </td><td>Biosynthesis of alkaloids derived from histidine and purine </td><td>417 </td></tr><tr><td>8 </td><td>Pyrimidine metabolism </td><td>408 </td></tr><tr><td>9 </td><td>Biosynthesis of terpenoids and steroids </td><td>401 </td></tr><tr><td>10 </td><td>Biosynthesis of alkaloids derived from terpenoid and polyketide </td><td>399 </td></tr><tr><td>11 </td><td>Methane metabolism </td><td>399 </td></tr><tr><td>12 </td><td>Biosynthesis of alkaloids derived from shikimate pathway </td><td>380 </td></tr><tr><td>13 </td><td>Glycolysis/gluconeogenesis </td><td>379 </td></tr><tr><td>14 </td><td>Biosynthesis of alkaloids derived from ornithine, lysine and nicotinic acid </td><td>368 </td></tr><tr><td>15 </td><td>Glutathione metabolism </td><td>318 </td></tr><tr><td>16 </td><td>Pyruvate metabolism </td><td>268 </td></tr><tr><td>17 </td><td>Arginine and proline metabolism </td><td>261 </td></tr><tr><td>18 </td><td>Glycerophospholipid metabolism </td><td>222 </td></tr><tr><td>19 </td><td>Fatty acid metabolism </td><td>221 </td></tr><tr><td>20 </td><td>Valine, leucine and isoleucine degradation </td><td>218 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6724263" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR021TB5" class="table-wrap-title" data-id="DSR021TB5"><div class="label -label">Table 5.</div><div class="caption"><p>The top 20 pathways with the highest EST numbers</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>No. </th><th>Pathways </th><th>Number of ESTs </th></tr></thead><tbody><tr><td>1 </td><td>Biosynthesis of secondary metabolites </td><td>1352 </td></tr><tr><td>2 </td><td>Oxidative phosphorylation </td><td>1111 </td></tr><tr><td>3 </td><td>Microbial metabolism in diverse environments </td><td>964 </td></tr><tr><td>4 </td><td>Purine metabolism </td><td>690 </td></tr><tr><td>5 </td><td>Biosynthesis of plant hormones </td><td>625 </td></tr><tr><td>6 </td><td>Biosynthesis of phenylpropanoids </td><td>466 </td></tr><tr><td>7 </td><td>Biosynthesis of alkaloids derived from histidine and purine </td><td>417 </td></tr><tr><td>8 </td><td>Pyrimidine metabolism </td><td>408 </td></tr><tr><td>9 </td><td>Biosynthesis of terpenoids and steroids </td><td>401 </td></tr><tr><td>10 </td><td>Biosynthesis of alkaloids derived from terpenoid and polyketide </td><td>399 </td></tr><tr><td>11 </td><td>Methane metabolism </td><td>399 </td></tr><tr><td>12 </td><td>Biosynthesis of alkaloids derived from shikimate pathway </td><td>380 </td></tr><tr><td>13 </td><td>Glycolysis/gluconeogenesis </td><td>379 </td></tr><tr><td>14 </td><td>Biosynthesis of alkaloids derived from ornithine, lysine and nicotinic acid </td><td>368 </td></tr><tr><td>15 </td><td>Glutathione metabolism </td><td>318 </td></tr><tr><td>16 </td><td>Pyruvate metabolism </td><td>268 </td></tr><tr><td>17 </td><td>Arginine and proline metabolism </td><td>261 </td></tr><tr><td>18 </td><td>Glycerophospholipid metabolism </td><td>222 </td></tr><tr><td>19 </td><td>Fatty acid metabolism </td><td>221 </td></tr><tr><td>20 </td><td>Valine, leucine and isoleucine degradation </td><td>218 </td></tr></tbody></table></div><div content-id="DSR021TB5" class="table-modal"><table><thead name="thead" align=""><tr><th>No. </th><th>Pathways </th><th>Number of ESTs </th></tr></thead><tbody><tr><td>1 </td><td>Biosynthesis of secondary metabolites </td><td>1352 </td></tr><tr><td>2 </td><td>Oxidative phosphorylation </td><td>1111 </td></tr><tr><td>3 </td><td>Microbial metabolism in diverse environments </td><td>964 </td></tr><tr><td>4 </td><td>Purine metabolism </td><td>690 </td></tr><tr><td>5 </td><td>Biosynthesis of plant hormones </td><td>625 </td></tr><tr><td>6 </td><td>Biosynthesis of phenylpropanoids </td><td>466 </td></tr><tr><td>7 </td><td>Biosynthesis of alkaloids derived from histidine and purine </td><td>417 </td></tr><tr><td>8 </td><td>Pyrimidine metabolism </td><td>408 </td></tr><tr><td>9 </td><td>Biosynthesis of terpenoids and steroids </td><td>401 </td></tr><tr><td>10 </td><td>Biosynthesis of alkaloids derived from terpenoid and polyketide </td><td>399 </td></tr><tr><td>11 </td><td>Methane metabolism </td><td>399 </td></tr><tr><td>12 </td><td>Biosynthesis of alkaloids derived from shikimate pathway </td><td>380 </td></tr><tr><td>13 </td><td>Glycolysis/gluconeogenesis </td><td>379 </td></tr><tr><td>14 </td><td>Biosynthesis of alkaloids derived from ornithine, lysine and nicotinic acid </td><td>368 </td></tr><tr><td>15 </td><td>Glutathione metabolism </td><td>318 </td></tr><tr><td>16 </td><td>Pyruvate metabolism </td><td>268 </td></tr><tr><td>17 </td><td>Arginine and proline metabolism </td><td>261 </td></tr><tr><td>18 </td><td>Glycerophospholipid metabolism </td><td>222 </td></tr><tr><td>19 </td><td>Fatty acid metabolism </td><td>221 </td></tr><tr><td>20 </td><td>Valine, leucine and isoleucine degradation </td><td>218 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6724263" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6724265 id="6724265" class="section-title" data-legacy-id=s3e>3.5. Tissue-specific analysis for differentially expressed genes</h3>
<p>Comparison of gene expression by DEGseq showed that a total of 336 genes expressed in the back-leg muscle with a significantly higher level than that of the ovary. These genes are involved in biological process (160 genes), cellular components (96) and molecular function (80) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). In addition, another 553 genes significantly expressed in the ovary rather than the back-leg muscle. These genes are involved in biological process (295 genes), cellular components (130) and molecular function (128) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>).</p>                            <h3 scrollto-destination=6724269 id="6724269" class="section-title" data-legacy-id=s3f>3.6. Single-nucleotide polymorphisms</h3>
<p>By excluding those that either could not specifically match the <em>S. scrofa</em> genome or had minor allele frequencies lower than 20%, we totally obtained 24 214 SNPs which comprised various substitutions of A-G (8484), C-T (8697), A-C (1635), A-T (1991), C-G (1325) and G-T (2082). The ratio of transitions (17 181) to transversions (7033) is ∼2.44. Except for 125 SNPs in mtDNA, all others (24 089) are in nucleic DNA including 1–18 autosomes (23 481) and X chromosome (608). The distribution of SNPs in each chromosome is described in Table <a reveal-id="DSR021TB6" data-open="DSR021TB6" class="link link-reveal link-table xref-fig">6</a>. </p>                        <a id="6724275" scrollto-destination="6724275"></a>
<div content-id="DSR021TB6" class="table-modal"><div class="table-wrap"><div id="DSR021TB6" class="table-wrap-title" data-id="DSR021TB6"><div class="label -label">Table 6.</div><div class="caption"><p>Distribution of SNPs in the <em>S. scrofa</em> genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosomes<sup>a</sup> </th><th>Counts </th></tr></thead><tbody><tr><td>1 </td><td>2358 </td></tr><tr><td>2 </td><td>1901 </td></tr><tr><td>3 </td><td>1467 </td></tr><tr><td>4 </td><td>1693 </td></tr><tr><td>5 </td><td>1133 </td></tr><tr><td>6 </td><td>1492 </td></tr><tr><td>7 </td><td>1922 </td></tr><tr><td>8 </td><td>1260 </td></tr><tr><td>9 </td><td>1415 </td></tr><tr><td>10 </td><td>968 </td></tr><tr><td>11 </td><td>533 </td></tr><tr><td>12 </td><td>1073 </td></tr><tr><td>13 </td><td>1418 </td></tr><tr><td>14 </td><td>2008 </td></tr><tr><td>15 </td><td>1086 </td></tr><tr><td>16 </td><td>580 </td></tr><tr><td>17 </td><td>680 </td></tr><tr><td>18 </td><td>494 </td></tr><tr><td>X </td><td>608 </td></tr><tr><td>MT </td><td>125 </td></tr><tr><td>Total </td><td>24 214 </td></tr></tbody></table></div><div content-id="DSR021TB6" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosomes<sup>a</sup> </th><th>Counts </th></tr></thead><tbody><tr><td>1 </td><td>2358 </td></tr><tr><td>2 </td><td>1901 </td></tr><tr><td>3 </td><td>1467 </td></tr><tr><td>4 </td><td>1693 </td></tr><tr><td>5 </td><td>1133 </td></tr><tr><td>6 </td><td>1492 </td></tr><tr><td>7 </td><td>1922 </td></tr><tr><td>8 </td><td>1260 </td></tr><tr><td>9 </td><td>1415 </td></tr><tr><td>10 </td><td>968 </td></tr><tr><td>11 </td><td>533 </td></tr><tr><td>12 </td><td>1073 </td></tr><tr><td>13 </td><td>1418 </td></tr><tr><td>14 </td><td>2008 </td></tr><tr><td>15 </td><td>1086 </td></tr><tr><td>16 </td><td>580 </td></tr><tr><td>17 </td><td>680 </td></tr><tr><td>18 </td><td>494 </td></tr><tr><td>X </td><td>608 </td></tr><tr><td>MT </td><td>125 </td></tr><tr><td>Total </td><td>24 214 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Chromosomes 1–18 and X indicate 18 autosomes and sex chromosome each, whereas MT indicates mtDNA.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6724275" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR021TB6" class="table-wrap-title" data-id="DSR021TB6"><div class="label -label">Table 6.</div><div class="caption"><p>Distribution of SNPs in the <em>S. scrofa</em> genome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Chromosomes<sup>a</sup> </th><th>Counts </th></tr></thead><tbody><tr><td>1 </td><td>2358 </td></tr><tr><td>2 </td><td>1901 </td></tr><tr><td>3 </td><td>1467 </td></tr><tr><td>4 </td><td>1693 </td></tr><tr><td>5 </td><td>1133 </td></tr><tr><td>6 </td><td>1492 </td></tr><tr><td>7 </td><td>1922 </td></tr><tr><td>8 </td><td>1260 </td></tr><tr><td>9 </td><td>1415 </td></tr><tr><td>10 </td><td>968 </td></tr><tr><td>11 </td><td>533 </td></tr><tr><td>12 </td><td>1073 </td></tr><tr><td>13 </td><td>1418 </td></tr><tr><td>14 </td><td>2008 </td></tr><tr><td>15 </td><td>1086 </td></tr><tr><td>16 </td><td>580 </td></tr><tr><td>17 </td><td>680 </td></tr><tr><td>18 </td><td>494 </td></tr><tr><td>X </td><td>608 </td></tr><tr><td>MT </td><td>125 </td></tr><tr><td>Total </td><td>24 214 </td></tr></tbody></table></div><div content-id="DSR021TB6" class="table-modal"><table><thead name="thead" align=""><tr><th>Chromosomes<sup>a</sup> </th><th>Counts </th></tr></thead><tbody><tr><td>1 </td><td>2358 </td></tr><tr><td>2 </td><td>1901 </td></tr><tr><td>3 </td><td>1467 </td></tr><tr><td>4 </td><td>1693 </td></tr><tr><td>5 </td><td>1133 </td></tr><tr><td>6 </td><td>1492 </td></tr><tr><td>7 </td><td>1922 </td></tr><tr><td>8 </td><td>1260 </td></tr><tr><td>9 </td><td>1415 </td></tr><tr><td>10 </td><td>968 </td></tr><tr><td>11 </td><td>533 </td></tr><tr><td>12 </td><td>1073 </td></tr><tr><td>13 </td><td>1418 </td></tr><tr><td>14 </td><td>2008 </td></tr><tr><td>15 </td><td>1086 </td></tr><tr><td>16 </td><td>580 </td></tr><tr><td>17 </td><td>680 </td></tr><tr><td>18 </td><td>494 </td></tr><tr><td>X </td><td>608 </td></tr><tr><td>MT </td><td>125 </td></tr><tr><td>Total </td><td>24 214 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Chromosomes 1–18 and X indicate 18 autosomes and sex chromosome each, whereas MT indicates mtDNA.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6724275" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=6724277 id="6724277" class="section-title" data-legacy-id=s3g>3.7. Simple sequence repeats or microsatellites</h3>
<p>We obtained 11 928 simple sequence repeats (SSRs), of which 45.72% were di-nucleotide repeats (5453), followed by 36.06% tri-nucleotide repeats (4301) and 14.85% tetra-nucleotide repeats (1771), as well as 3.38% penta-nucleotide repeats (403) (Table <a reveal-id="DSR021TB7" data-open="DSR021TB7" class="link link-reveal link-table xref-fig">7</a>). There are six types of di-nucleotide repeats, and among them (GT)<sub><em>n</em></sub>, (AC)<sub><em>n</em></sub> and (AT)<sub><em>n</em></sub> are three predominant types with frequencies of 28.5, 27.6 and 23.12%, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). Among tri-nucleotide repeats, the frequencies of 20 SSR types seem to vary moderately from 0.19 to 17.11%, and the most common repeats are (GTT)<sub><em>n</em></sub> (17.11%) and (ACC)<sub><em>n</em></sub> (10.14%) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). As many as 45 SSRs present for tetra-nucleotide repeats, and five of them [15.70% for (ATTT)<sub><em>n</em></sub>, 14.85% for (AAAT)<sub><em>n</em></sub>, 14.00% for (GTTT)<sub><em>n</em></sub>, 12.70% for (CTTT)<sub><em>n</em></sub> and 12.37% for (AAAC)<sub><em>n</em></sub>] are major ones with frequencies over 10% (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>). Among 21 penta-nucleotide repeats types, (GTTTT)<sub><em>n</em></sub> (34.99%) and (AAAAC)<sub><em>n</em></sub> (20.60%) are two predominant types, followed by 10.42% for (ATTTT)<sub><em>n</em></sub>, and the rest less than 10% (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). </p>                        <a id="6724281" scrollto-destination="6724281"></a>
<div content-id="DSR021TB7" class="table-modal"><div class="table-wrap"><div id="DSR021TB7" class="table-wrap-title" data-id="DSR021TB7"><div class="label -label">Table 7.</div><div class="caption"><p>Summary on microsatellite loci in <em>S. scrofa</em> transcriptome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Number of repeats </th><th>Di-nucleotide repeats </th><th>Tri-nucleotide repeats </th><th>Tetra-nucleotide repeats </th><th>Penta-nucleotide repeats </th></tr></thead><tbody><tr><td>4 </td><td>— </td><td>2511 </td><td>1134 </td><td>303 </td></tr><tr><td>5 </td><td>— </td><td>1004 </td><td>386 </td><td>74 </td></tr><tr><td>6 </td><td>1858 </td><td>434 </td><td>118 </td><td>18 </td></tr><tr><td>7 </td><td>986 </td><td>189 </td><td>31 </td><td>6 </td></tr><tr><td>8 </td><td>592 </td><td>80 </td><td>20 </td><td>— </td></tr><tr><td>9 </td><td>384 </td><td>23 </td><td>26 </td><td>1 </td></tr><tr><td>10 </td><td>257 </td><td>22 </td><td>17 </td><td>— </td></tr><tr><td>11 </td><td>202 </td><td>16 </td><td>9 </td><td>1 </td></tr><tr><td>12 </td><td>187 </td><td>11 </td><td>6 </td><td>— </td></tr><tr><td>13 </td><td>184 </td><td>6 </td><td>13 </td><td>— </td></tr><tr><td>14 </td><td>131 </td><td>2 </td><td>11 </td><td>— </td></tr><tr><td>15 </td><td>115 </td><td>1 </td><td>— </td><td>— </td></tr><tr><td>16 </td><td>100 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>17 </td><td>93 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>18 </td><td>49 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>19 </td><td>63 </td><td>2 </td><td>— </td><td>— </td></tr><tr><td>20 </td><td>54 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>21 </td><td>48 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>22 </td><td>41 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>23 </td><td>29 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>24 </td><td>18 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>25 </td><td>18 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>26 </td><td>9 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>27 </td><td>10 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>28 </td><td>14 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>29 </td><td>11 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>Total </td><td>5453 </td><td>4301 </td><td>1771 </td><td>403 </td></tr></tbody></table></div><div content-id="DSR021TB7" class="table-modal"><table><thead name="thead" align=""><tr><th>Number of repeats </th><th>Di-nucleotide repeats </th><th>Tri-nucleotide repeats </th><th>Tetra-nucleotide repeats </th><th>Penta-nucleotide repeats </th></tr></thead><tbody><tr><td>4 </td><td>— </td><td>2511 </td><td>1134 </td><td>303 </td></tr><tr><td>5 </td><td>— </td><td>1004 </td><td>386 </td><td>74 </td></tr><tr><td>6 </td><td>1858 </td><td>434 </td><td>118 </td><td>18 </td></tr><tr><td>7 </td><td>986 </td><td>189 </td><td>31 </td><td>6 </td></tr><tr><td>8 </td><td>592 </td><td>80 </td><td>20 </td><td>— </td></tr><tr><td>9 </td><td>384 </td><td>23 </td><td>26 </td><td>1 </td></tr><tr><td>10 </td><td>257 </td><td>22 </td><td>17 </td><td>— </td></tr><tr><td>11 </td><td>202 </td><td>16 </td><td>9 </td><td>1 </td></tr><tr><td>12 </td><td>187 </td><td>11 </td><td>6 </td><td>— </td></tr><tr><td>13 </td><td>184 </td><td>6 </td><td>13 </td><td>— </td></tr><tr><td>14 </td><td>131 </td><td>2 </td><td>11 </td><td>— </td></tr><tr><td>15 </td><td>115 </td><td>1 </td><td>— </td><td>— </td></tr><tr><td>16 </td><td>100 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>17 </td><td>93 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>18 </td><td>49 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>19 </td><td>63 </td><td>2 </td><td>— </td><td>— </td></tr><tr><td>20 </td><td>54 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>21 </td><td>48 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>22 </td><td>41 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>23 </td><td>29 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>24 </td><td>18 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>25 </td><td>18 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>26 </td><td>9 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>27 </td><td>10 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>28 </td><td>14 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>29 </td><td>11 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>Total </td><td>5453 </td><td>4301 </td><td>1771 </td><td>403 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6724281" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSR021TB7" class="table-wrap-title" data-id="DSR021TB7"><div class="label -label">Table 7.</div><div class="caption"><p>Summary on microsatellite loci in <em>S. scrofa</em> transcriptome</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Number of repeats </th><th>Di-nucleotide repeats </th><th>Tri-nucleotide repeats </th><th>Tetra-nucleotide repeats </th><th>Penta-nucleotide repeats </th></tr></thead><tbody><tr><td>4 </td><td>— </td><td>2511 </td><td>1134 </td><td>303 </td></tr><tr><td>5 </td><td>— </td><td>1004 </td><td>386 </td><td>74 </td></tr><tr><td>6 </td><td>1858 </td><td>434 </td><td>118 </td><td>18 </td></tr><tr><td>7 </td><td>986 </td><td>189 </td><td>31 </td><td>6 </td></tr><tr><td>8 </td><td>592 </td><td>80 </td><td>20 </td><td>— </td></tr><tr><td>9 </td><td>384 </td><td>23 </td><td>26 </td><td>1 </td></tr><tr><td>10 </td><td>257 </td><td>22 </td><td>17 </td><td>— </td></tr><tr><td>11 </td><td>202 </td><td>16 </td><td>9 </td><td>1 </td></tr><tr><td>12 </td><td>187 </td><td>11 </td><td>6 </td><td>— </td></tr><tr><td>13 </td><td>184 </td><td>6 </td><td>13 </td><td>— </td></tr><tr><td>14 </td><td>131 </td><td>2 </td><td>11 </td><td>— </td></tr><tr><td>15 </td><td>115 </td><td>1 </td><td>— </td><td>— </td></tr><tr><td>16 </td><td>100 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>17 </td><td>93 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>18 </td><td>49 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>19 </td><td>63 </td><td>2 </td><td>— </td><td>— </td></tr><tr><td>20 </td><td>54 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>21 </td><td>48 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>22 </td><td>41 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>23 </td><td>29 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>24 </td><td>18 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>25 </td><td>18 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>26 </td><td>9 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>27 </td><td>10 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>28 </td><td>14 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>29 </td><td>11 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>Total </td><td>5453 </td><td>4301 </td><td>1771 </td><td>403 </td></tr></tbody></table></div><div content-id="DSR021TB7" class="table-modal"><table><thead name="thead" align=""><tr><th>Number of repeats </th><th>Di-nucleotide repeats </th><th>Tri-nucleotide repeats </th><th>Tetra-nucleotide repeats </th><th>Penta-nucleotide repeats </th></tr></thead><tbody><tr><td>4 </td><td>— </td><td>2511 </td><td>1134 </td><td>303 </td></tr><tr><td>5 </td><td>— </td><td>1004 </td><td>386 </td><td>74 </td></tr><tr><td>6 </td><td>1858 </td><td>434 </td><td>118 </td><td>18 </td></tr><tr><td>7 </td><td>986 </td><td>189 </td><td>31 </td><td>6 </td></tr><tr><td>8 </td><td>592 </td><td>80 </td><td>20 </td><td>— </td></tr><tr><td>9 </td><td>384 </td><td>23 </td><td>26 </td><td>1 </td></tr><tr><td>10 </td><td>257 </td><td>22 </td><td>17 </td><td>— </td></tr><tr><td>11 </td><td>202 </td><td>16 </td><td>9 </td><td>1 </td></tr><tr><td>12 </td><td>187 </td><td>11 </td><td>6 </td><td>— </td></tr><tr><td>13 </td><td>184 </td><td>6 </td><td>13 </td><td>— </td></tr><tr><td>14 </td><td>131 </td><td>2 </td><td>11 </td><td>— </td></tr><tr><td>15 </td><td>115 </td><td>1 </td><td>— </td><td>— </td></tr><tr><td>16 </td><td>100 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>17 </td><td>93 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>18 </td><td>49 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>19 </td><td>63 </td><td>2 </td><td>— </td><td>— </td></tr><tr><td>20 </td><td>54 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>21 </td><td>48 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>22 </td><td>41 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>23 </td><td>29 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>24 </td><td>18 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>25 </td><td>18 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>26 </td><td>9 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>27 </td><td>10 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>28 </td><td>14 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>29 </td><td>11 </td><td>— </td><td>— </td><td>— </td></tr><tr><td>Total </td><td>5453 </td><td>4301 </td><td>1771 </td><td>403 </td></tr></tbody></table></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6724281" target="_blank">View Large</a></div></div>                            <h2 scrollto-destination=6724284 id="6724284" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>The obtained transcriptomic sequences by this study are useful for us to understand the genetic makeup of <em>S. scrofa</em> whole transcriptome, and to our knowledge, it is very limited until now. Even though the draft <em>S. scrofa</em> genome was released on November 2009, very fewer articles could be found regarding the <em>S. scrofa</em> genome or trancriptome based on large-scale sequence data. The 454 pyrosequencing yield 1 087 003 reads in total, including 558 743 for the back-leg muscle and 528 260 for the ovary and 421 Mb nucleotide residues for <em>S. scrofa</em> transcriptome by this study. A recent research obtained 1 253 361 454 sequences for the skeletal muscle (701 695) and heart (551 666) and showed the reproducibility within 454 sequencing and cDNA microarray; however, further analyses of <em>S. scrofa</em> transcriptome with 454 sequences were not reported thereby.<sup><a reveal-id="DSR021C13" data-open="DSR021C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Most transcripts (78.7%) identified by this study could match the released <em>S. scrofa</em> genome, and they scattered on chromosome 1–18, X and mtDNA. The bigger chromosomes seem to contain more transcripts compared with others. The obtained overall GC content of <em>S. scrofa</em> transcriptome was 42.3%, which was lower than the reported GC content in 5′UTR (59.2%), coding (49.6%), but a little bit higher than 3′UTR (41.8%) in the <em>S. scrofa</em> genome.<sup><a reveal-id="DSR021C4" data-open="DSR021C4" class="link link-ref link-reveal xref-bibr">4</a></sup> It seemed that more 3′UTR than other regions (coding and 5′UTR) were included in analyses, as far as only assembled contigs rather than singletons were concerned. The genome-wide average GC content of the human genome is 41%, varying from different chromosomes and regions.<sup><a reveal-id="DSR021C22" data-open="DSR021C22" class="link link-ref link-reveal xref-bibr">22</a></sup> It is known that the mouse has a slightly higher overall GC content (42%), but the distribution is tighter.<sup><a reveal-id="DSR021C23" data-open="DSR021C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Over 1 million 454 reads and 121 299 ESTs by this study are useful resource for further research on <em>S. scrofa</em> functional genomics.</p><p>Both gene annotation and pathway analyses are helpful for us to predict potential genes and their functions at a whole-transcriptome level. In <em>S. scrofa</em> transcriptome, as discovered by this study, the predominant gene clusters are involved in the cellular process and metabolic process of biological process, the binding and catalytic activity of molecular function, as well as the cell, cell part and organelle of a cellular component. Similar results are found in European eel,<sup><a reveal-id="DSR021C24" data-open="DSR021C24" class="link link-ref link-reveal xref-bibr">24</a></sup> rainbow trout<sup><a reveal-id="DSR021C15" data-open="DSR021C15" class="link link-ref link-reveal xref-bibr">15</a></sup> and Red bugs.<sup><a reveal-id="DSR021C14" data-open="DSR021C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Whereas, in Chickpea transcriptome, genes are predominantly involved in the protein metabolism of biological process and the transferase activity of molecular function, as well as the chloroplast of cellular component, which indicated notable differences between animals and plants.<sup><a reveal-id="DSR021C25" data-open="DSR021C25" class="link link-ref link-reveal xref-bibr">25</a></sup> In addition, we also predicted overall 4268 ESTs (or transcripts) that are involved in 132 predicted KEGG metabolic pathways, and two major pathways (biosynthesis of secondary metabolites and oxidative phosphorylation) comprised over 1000 ESTs. The predicted pathways altogether with gene annotation are useful for further investigation on gene function in future.</p><p>The differentially expressed genes in the back-leg muscle and ovary tissues are probably related to their metabolism and functions. In this study, we found 336 and 553 genes that were significantly expressed in the back-leg muscle and ovary, and they were involved in biological process, cellular components and molecular function by further gene annotation. A recent study identified a total of 306 differently expressed genes between muscle and heart tissue based on 1 253 361 Roche 454 reads and confirmed most genes by a microarray approach.<sup><a reveal-id="DSR021C13" data-open="DSR021C13" class="link link-ref link-reveal xref-bibr">13</a></sup> In this study, more differently expressed genes were found between back-leg muscle and ovary based on 1 087 003 reads. As far as different tissues are used by both studies, these differently expressed genes still require for further confirmation. As the muscle and ovary are crucial tissues for growth and reproduction, the identified genes with different expression are the potential candidate for growth and reproduction traits of <em>S. scrofa</em>.</p><p>Plenty expression SNPs (eSNPs) as identified by this study are valuable molecular markers for further research on <em>S. scrofa</em>. A total of 24 214 SNPs (minor allele frequencies ≥0.2) were found to specifically match the <em>S. scrofa</em> genome, in which 608 and 125 are in X chromosome and mtDNA, and the rest (23 481) in 18 autosomes. In general, these SNPs should be reliable eSNPs and act as candidate markers for identification of eQTL. In the genome level, a total of 98 151 SNPs were predicted based on 4.8 million whole-genome shotgun sequences.<sup><a reveal-id="DSR021C5" data-open="DSR021C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Another study, moreover, discovered as many as 372 886 SNPs by sequencing with Illumina's Genome Analyzer (GA), and 62 621 loci of them were used to design the Illumina Porcine 60K + SNP iSelect Beadchip.<sup><a reveal-id="DSR021C6" data-open="DSR021C6" class="link link-ref link-reveal xref-bibr">6</a></sup> This SNP chip is useful for identification of candidate genes or QTLs underling quantitative traits such as body composition.<sup><a reveal-id="DSR021C7" data-open="DSR021C7" class="link link-ref link-reveal xref-bibr">7</a></sup></p><p>SSRs, or microsatellite, are neutral molecular markers that wildly distribute in a genome. It was formerly proved that SSRs comprise 3% of the human genome, with the greatest contribution from di-nucleotide repeats (0.5%).<sup><a reveal-id="DSR021C22" data-open="DSR021C22" class="link link-ref link-reveal xref-bibr">22</a></sup> In this study, 45.72% of 11 928 SSRs are di-nucleotide repeats, follows by tri-nucleotide repeats (36.06%) and tetra-nucleotide repeats (14.85%), as well as penta-nucleotide repeats (3.38%). In addition to (AC)<sub><em>n</em></sub> and (AT)<sub><em>n</em></sub> of di-nucleotide repeats, and (AAT)<sub><em>n</em></sub> and (AAC)<sub><em>n</em></sub> of tri-nucleotide, (GT)<sub><em>n</em></sub> and (GTT)<sub><em>n</em></sub> also have high frequencies as indicated by this study, which is different from that of the human genome.<sup><a reveal-id="DSR021C22" data-open="DSR021C22" class="link link-ref link-reveal xref-bibr">22</a></sup> It is probably because (GT)<sub><em>n</em></sub> and (GTT)<sub><em>n</em></sub> are equal to (AC)<sub><em>n</em></sub> and (ACC)<sub><em>n</em></sub> each, since the reverse strands for some ESTs are used in <em>de novo</em> analysis. In fact, slight differences are found for SSRs among human, mouse and dog, as well as <em>S. scrofa</em>.<sup><a reveal-id="DSR021C23" data-open="DSR021C23" class="link link-ref link-reveal xref-bibr">23</a></sup></p><p>In conclusion, we have demonstrated the muscle and ovary transcriptome of <em>S. scrofa</em> by the use of high-throughout 454 pyrosequencing. Our study obtained a set of 121 299 transcripts or ESTs and demonstrated some important features of <em>S. scrofa</em> transcriptome, such as GC content, gene annotation and pathways across whole transcriptome. In addition, we identified reliable markers of 24 214 SNPs and 11 928 SSRs. This study is helpful for understanding the genetic architecture of <em>S. scrofa</em> transcriptome and provides useful resource and markers for functional genomic research in future.</p>                            <h2 scrollto-destination=6724300 id="6724300" class="section-title" data-legacy-id=s5>Supplementary data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=6724304 id="6724304" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This research was supported by the National Major Special Projects on New Varieties Cultivation for Transgenic Organisms (2008ZX08006-005 and 2009ZX08009-145B) and the Important Projects in Key Fields in Guangdong and Hongkong, 2008 (2008A02).</p>                            <h2 scrollto-destination=6724309 id="6724309" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSR021C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Andersson</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Georges</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Domestic-animal genomics: deciphering the genetics of complex traits</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2004</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">202</div>-<div class="lpage">12</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrg1294" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nrg1294</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Domestic-animal%20genomics%3A%20deciphering%20the%20genetics%20of%20complex%20traits&amp;author=L.%20Andersson&amp;author=M.%20Georges&amp;publication_year=2004&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=5&amp;pages=202-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg1294" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg1294" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg1294"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14970822" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujii</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Otsu</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Zorzato</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">Identification of a mutation in porcine ryanodine receptor associated with malignant hyperthermia</div>, <div class="source">Science</div> , <div class="year">1991</div>, vol. <div class="volume">253</div> (pg. <div class="fpage">448</div>-<div class="lpage">51</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1126/science.1862346" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1126/science.1862346</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20mutation%20in%20porcine%20ryanodine%20receptor%20associated%20with%20malignant%20hyperthermia&amp;author=J.%20Fujii&amp;author=K.%20Otsu&amp;author=F.%20Zorzato&amp;publication_year=1991&amp;journal=Science&amp;volume=253&amp;pages=448-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1862346" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1862346" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1862346"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1862346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Van Laere</div> <div class="given-names">A.S.</div></div>,  <div class="name"><div class="surname">Nguyen</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Braunschweig</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">A regulatory mutation in IGF2 causes a major QTL effect on muscle growth in the pig</div>, <div class="source">Nature</div> , <div class="year">2003</div>, vol. <div class="volume">425</div> (pg. <div class="fpage">832</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature02064" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature02064</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20regulatory%20mutation%20in%20IGF2%20causes%20a%20major%20QTL%20effect%20on%20muscle%20growth%20in%20the%20pig&amp;author=A.S.%20Van%20Laere&amp;author=M.%20Nguyen&amp;author=M.%20Braunschweig&amp;publication_year=2003&amp;journal=Nature&amp;volume=425&amp;pages=832-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature02064" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature02064" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature02064"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14574411" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wernersson</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Schierup</div> <div class="given-names">M.H.</div></div>,  <div class="name"><div class="surname">Jørgensen</div> <div class="given-names">F.G.</div></div>, et al. </div><div class="article-title">Pigs in sequence space: a 0.66X coverage pig genome survey based on shotgun sequencing</div>, <div class="source">BMC Genomics</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> pg. <div class="fpage">70</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-6-70" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-6-70</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pigs%20in%20sequence%20space%3A%20a%200.66X%20coverage%20pig%20genome%20survey%20based%20on%20shotgun%20sequencing&amp;author=R.%20Wernersson&amp;author=M.H.%20Schierup&amp;author=F.G.%20J%C3%B8rgensen&amp;publication_year=2005&amp;journal=BMC%20Genomics&amp;volume=6&amp;pages=70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-6-70" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-6-70" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-6-70"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15885146" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kerstens</div> <div class="given-names">H.H.</div></div>,  <div class="name"><div class="surname">Kollers</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kommadath</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Mining for single nucleotide polymorphisms in pig genome sequence data</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">4</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-10-4" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-10-4</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mining%20for%20single%20nucleotide%20polymorphisms%20in%20pig%20genome%20sequence%20data&amp;author=H.H.%20Kerstens&amp;author=S.%20Kollers&amp;author=A.%20Kommadath&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-4" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-4" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-4"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19126189" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ramos</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Crooijmans</div> <div class="given-names">R.P.</div></div>,  <div class="name"><div class="surname">Affara</div> <div class="given-names">N.A.</div></div>, et al. </div><div class="article-title">Design of a high density SNP genotyping assay in the pig using SNPs identified and characterized by next generation sequencing technology</div>, <div class="source">PLoS One</div> , <div class="year">2009</div>, vol. <div class="volume">4</div> pg. <div class="fpage">e6524</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0006524" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0006524</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Design%20of%20a%20high%20density%20SNP%20genotyping%20assay%20in%20the%20pig%20using%20SNPs%20identified%20and%20characterized%20by%20next%20generation%20sequencing%20technology&amp;author=A.M.%20Ramos&amp;author=R.P.%20Crooijmans&amp;author=N.A.%20Affara&amp;publication_year=2009&amp;journal=PLoS%20One&amp;volume=4&amp;pages=e6524" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0006524" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0006524" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0006524"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19654876" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fan</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Onteru</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Du</div> <div class="given-names">Z.Q.</div></div>,  <div class="name"><div class="surname">Garrick</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Stalder</div> <div class="given-names">K.J.</div></div>,  <div class="name"><div class="surname">Rothschild</div> <div class="given-names">M.F.</div></div>. </div><div class="article-title">Genome-wide association study identifies loci for body composition and structural soundness traits in pigs</div>, <div class="source">PLoS One</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e14726</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0014726" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0014726</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20association%20study%20identifies%20loci%20for%20body%20composition%20and%20structural%20soundness%20traits%20in%20pigs&amp;author=B.%20Fan&amp;author=S.K.%20Onteru&amp;author=Z.Q.%20Du&amp;author=D.J.%20Garrick&amp;author=K.J.%20Stalder&amp;author=M.F.%20Rothschild&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e14726" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0014726" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0014726" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0014726"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21383979" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sun</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Mei</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Tao</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Microarray profiling for differential gene expression in PMSG-hCG stimulated preovulatory ovarian follicles of Chinese Taihu and Large White sows</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">111</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-12-111" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-12-111</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microarray%20profiling%20for%20differential%20gene%20expression%20in%20PMSG-hCG%20stimulated%20preovulatory%20ovarian%20follicles%20of%20Chinese%20Taihu%20and%20Large%20White%20sows&amp;author=X.%20Sun&amp;author=S.%20Mei&amp;author=H.%20Tao&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=111" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21324170" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Steibel</div> <div class="given-names">J.P.</div></div>,  <div class="name"><div class="surname">Bates</div> <div class="given-names">R.O.</div></div>,  <div class="name"><div class="surname">Rosa</div> <div class="given-names">G.J.</div></div>, et al. </div><div class="article-title">Genome-wide linkage analysis of global gene expression in loin muscle tissue identifies candidate genes in pigs</div>, <div class="source">PLoS One</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e16766</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0016766" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0016766</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20linkage%20analysis%20of%20global%20gene%20expression%20in%20loin%20muscle%20tissue%20identifies%20candidate%20genes%20in%20pigs&amp;author=J.P.%20Steibel&amp;author=R.O.%20Bates&amp;author=G.J.%20Rosa&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e16766" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0016766" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0016766" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0016766"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21346809" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ozsolak</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Milos</div> <div class="given-names">P.M.</div></div>. </div><div class="article-title">RNA sequencing: advances, challenges and opportunities</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">87</div>-<div class="lpage">98</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nrg2934" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nrg2934</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA%20sequencing%3A%20advances%2C%20challenges%20and%20opportunities&amp;author=F.%20Ozsolak&amp;author=P.M.%20Milos&amp;publication_year=2011&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=12&amp;pages=87-98" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2934" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2934" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2934"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21191423" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sugarbaker</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Richards</div> <div class="given-names">W.G.</div></div>,  <div class="name"><div class="surname">Gordon</div> <div class="given-names">G.J.</div></div>, et al. </div><div class="article-title">Transcriptome sequencing of malignant pleural mesothelioma tumors</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2008</div>, vol. <div class="volume">105</div> (pg. <div class="fpage">3521</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0712399105" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0712399105</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20of%20malignant%20pleural%20mesothelioma%20tumors&amp;author=D.J.%20Sugarbaker&amp;author=W.G.%20Richards&amp;author=G.J.%20Gordon&amp;publication_year=2008&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=105&amp;pages=3521-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0712399105" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0712399105" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0712399105"> </span></div> </div></div></div></div></div><div content-id="DSR021C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">J.Q.</div></div>,  <div class="name"><div class="surname">Habegger</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Noisa</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Dynamic transcriptomes during neural differentiation of human embryonic stem cells revealed by short, long, and paired-end sequencing</div>, <div class="source">Proc. Natl Acad. Sci. USA</div> , <div class="year">2010</div>, vol. <div class="volume">107</div> (pg. <div class="fpage">5254</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1073/pnas.0914114107" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1073/pnas.0914114107</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dynamic%20transcriptomes%20during%20neural%20differentiation%20of%20human%20embryonic%20stem%20cells%20revealed%20by%20short%2C%20long%2C%20and%20paired-end%20sequencing&amp;author=J.Q.%20Wu&amp;author=L.%20Habegger&amp;author=P.%20Noisa&amp;publication_year=2010&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20USA&amp;volume=107&amp;pages=5254-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0914114107" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0914114107" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0914114107"> </span></div> </div></div></div></div></div><div content-id="DSR021C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hornshøj</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Bendixen</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Conley</div> <div class="given-names">L.N.</div></div>, et al. </div><div class="article-title">Transcriptomic and proteomic profiling of two porcine tissues using high-throughput technologies</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">30</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-10-30" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-10-30</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptomic%20and%20proteomic%20profiling%20of%20two%20porcine%20tissues%20using%20high-throughput%20technologies&amp;author=H.%20Hornsh%C3%B8j&amp;author=E.%20Bendixen&amp;author=L.N.%20Conley&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=30" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-30" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-30" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-30"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19152685" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bai</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Mamidala</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Rajarapu</div> <div class="given-names">S.P.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">S.C.</div></div>,  <div class="name"><div class="surname">Mittapalli</div> <div class="given-names">O.</div></div>. </div><div class="article-title">Transcriptomics of the bed bug (<em>Cimex lectularius</em>)</div>, <div class="source">PLoS One</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e16336</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0016336" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0016336</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptomics%20of%20the%20bed%20bug%20(Cimex%20lectularius)&amp;author=X.%20Bai&amp;author=P.%20Mamidala&amp;author=S.P.%20Rajarapu&amp;author=S.C.%20Jones&amp;author=O.%20Mittapalli&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e16336" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0016336" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0016336" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0016336"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21283830" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Salem</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Rexroad</div> <div class="given-names">C.E.</div>3rd.</div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Thorgaard</div> <div class="given-names">G.H.</div></div>,  <div class="name"><div class="surname">Yao</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Characterization of the rainbow trout transcriptome using Sanger and 454-pyrosequencing approaches</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">564</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-11-564" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-11-564</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20the%20rainbow%20trout%20transcriptome%20using%20Sanger%20and%20454-pyrosequencing%20approaches&amp;author=M.%20Salem&amp;author=C.E.%20Rexroad&amp;author=J.%20Wang&amp;author=G.H.%20Thorgaard&amp;author=J.%20Yao&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=564" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-564" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-564" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-564"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20942956" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bai</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Rivera-Vega</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Mamidala</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Transcriptomic signatures of ash (<em>Fraxinus</em> spp.) phloem</div>, <div class="source">PLoS One</div> , <div class="year">2011</div>, vol. <div class="volume">6</div> pg. <div class="fpage">e16368</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0016368" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0016368</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptomic%20signatures%20of%20ash%20(Fraxinus%20spp.)%20phloem&amp;author=X.%20Bai&amp;author=L.%20Rivera-Vega&amp;author=P.%20Mamidala&amp;publication_year=2011&amp;journal=PLoS%20One&amp;volume=6&amp;pages=e16368" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0016368" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0016368" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0016368"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21283712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parchman</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Geist</div> <div class="given-names">K.S.</div></div>,  <div class="name"><div class="surname">Grahnen</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Benkman</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Buerkle</div> <div class="given-names">C.A.</div></div>. </div><div class="article-title">Transcriptome sequencing in an ecologically important tree species: assembly, annotation, and marker discovery</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">180</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-11-180" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-11-180</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20in%20an%20ecologically%20important%20tree%20species%3A%20assembly%2C%20annotation%2C%20and%20marker%20discovery&amp;author=T.L.%20Parchman&amp;author=K.S.%20Geist&amp;author=J.A.%20Grahnen&amp;author=C.W.%20Benkman&amp;author=C.A.%20Buerkle&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=180" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-180" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-180" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-180"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20233449" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Poroyko</div> <div class="given-names">V.</div></div>,  <div class="name"><div class="surname">White</div> <div class="given-names">J.R.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Gut microbial gene expression in mother-fed and formula-fed piglets</div>, <div class="source">PLoS One</div> , <div class="year">2010</div>, vol. <div class="volume">5</div> pg. <div class="fpage">e12459</div> <div class="pub-id-doi"><a href="https://doi.org/10.1371/journal.pone.0012459" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1371/journal.pone.0012459</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gut%20microbial%20gene%20expression%20in%20mother-fed%20and%20formula-fed%20piglets&amp;author=V.%20Poroyko&amp;author=J.R.%20White&amp;author=M.%20Wang&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e12459" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0012459" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0012459" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0012459"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20805981" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S.F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A.A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">402</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/25.17.3389" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/25.17.3389</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ashburner</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ball</div> <div class="given-names">C.A.</div></div>,  <div class="name"><div class="surname">Blake</div> <div class="given-names">J.A.</div></div>, et al. </div><div class="article-title">Gene ontology: tool for the unification of biology</div>, <div class="source">Nat. Genet.</div> , <div class="year">2000</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">25</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/75556" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/75556</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gene%20ontology%3A%20tool%20for%20the%20unification%20of%20biology&amp;author=M.%20Ashburner&amp;author=C.A.%20Ball&amp;author=J.A.%20Blake&amp;publication_year=2000&amp;journal=Nat.%20Genet.&amp;volume=25&amp;pages=25-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/75556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F75556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F75556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10802651" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>. </div><div class="article-title">DEGseq: an R package for identifying differentially expressed genes from RNA-seq data</div>, <div class="source">Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">136</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btp612" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btp612</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DEGseq%3A%20an%20R%20package%20for%20identifying%20differentially%20expressed%20genes%20from%20RNA-seq%20data&amp;author=L.%20Wang&amp;author=Z.%20Feng&amp;author=X.%20Wang&amp;author=X.%20Wang&amp;author=X.%20Zhang&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=136-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp612" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp612" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp612"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19855105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="collab">International Human Genome Sequencing Consortium</div><div class="article-title">Initial sequencing and analysis of the human genome</div>, <div class="source">Nature</div> , <div class="year">2001</div>, vol. <div class="volume">409</div> (pg. <div class="fpage">860</div>-<div class="lpage">921</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/35057062" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/35057062</a></div><div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/35057062" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F35057062" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F35057062"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11237011" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="collab">Mouse Genome Sequencing Consortium</div><div class="article-title">Initial sequencing and comparative analysis of the mouse genome</div>, <div class="source">Nature</div> , <div class="year">2002</div>, vol. <div class="volume">420</div> (pg. <div class="fpage">520</div>-<div class="lpage">62</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nature01262" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nature01262</a></div><div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature01262" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature01262" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature01262"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12466850" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Coppe</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Pujola</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Mase</div> <div class="given-names">G.E.</div></div>, et al. </div><div class="article-title">Sequencing, <em>de novo</em> annotation and analysis of the first <em>Anguilla anguilla</em> transcriptome: EeelBase opens new perspectives for the study of the critically endangered European eel</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">635</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-11-635" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-11-635</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%2C%20de%20novo%20annotation%20and%20analysis%20of%20the%20first%20Anguilla%20anguilla%20transcriptome%3A%20EeelBase%20opens%20new%20perspectives%20for%20the%20study%20of%20the%20critically%20endangered%20European%20eel&amp;author=A.%20Coppe&amp;author=J.M.%20Pujola&amp;author=G.E.%20Mase&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=635" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-635" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-635" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-635"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21080939" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSR021C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSR021C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Patel</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>. </div><div class="article-title"><em>De novo</em> assembly of chickpea transcriptome using short reads for gene discovery and marker identification</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">53</div>-<div class="lpage">63</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsq028" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsq028</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20chickpea%20transcriptome%20using%20short%20reads%20for%20gene%20discovery%20and%20marker%20identification&amp;author=R.%20Garg&amp;author=R.K.%20Patel&amp;author=A.K.%20Tyagi&amp;author=M.%20Jain&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=53-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217129" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2011. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=448814&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/dsr021-dsr021supp_table4.doc?Expires=2147483647&amp;Signature=CZorqrGwrjVGQPLMQDQ3LaZ6TKCoSBp96~WgOVwRibdJa2halaMqAvpw1hrFLE-mSKncIcgK2upzODPw8vRr32h5S1qF3fejaFwERNfcFet5XWBuu9uWgfV-zmpHDYN2bjd1OK48OWeA~htdzL5s6sW1Dr7YduGESo90w1fpOpo5xbEQMpuN7pvmKRQ44almLlPdoJLIVFEq8zgHTc~RDj9Mu3SodnAIWAwEIaerhDBmbHGWeQNJTeq9GLhy6blqbihRP1CMB0bJL3UfG5uQ0SYM8RQdo0RYEsvGQ95l1RHTFJvCfiW-eii2JyJUUKv5SsQqLLlGWWwZg1NAdQPtwg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-10</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/dsr021-dsr021supp_table3.doc?Expires=2147483647&amp;Signature=wpTX9IremZypJmokQUVvwnXqRNZlP7Yzy8ZobyeMllf0KbYDRd6SO0AtRTUo~5sh7QUjQoI5KGkurTdffViU9tMhUau-vwi5TBrS6PgqYQyiIio~YNLguKMovIG8g~XHumb84YOK8bZo~V0vJV5ofJUYIQ5mSMbECjB1LA-fOu9X50slFecUcTm9qX0uV02BwffhPFOUdrkn4BFCowV1IZPPbBYkY1mk-E8RT5~cz9uVfdyJoeTmomVN-25kZ7srK8D82EpM9QdnN9m2nSIRoIY2BcrbQ5qfynlmir0GgS2nz1xGxsVR5Jah8BYVFYejEvGE6nB3mwiOo93h1HLwJA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-9</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/dsr021-dsr021supp_table2.doc?Expires=2147483647&amp;Signature=IKANKGD3T1gvorZekCGs2FfA6kURAyMQw-Be~4cZPNIAgDG8PKb~WTcwfWcu2NsD~3VxlheXoFnM9zCx3f~A73SVRvjJVvO0tpw7drgpOWGbcXQIzr-BLaDcHmuUcSMphw0lgr5AUTy4g5tRBttEl~6atUOV8IZsE2S~d6Wok3w1araNcKKQ8DeiLaRRwBaKke1LudSkaKF5UzbnMb7Z4onaSj8deCumefQN32PyWaUkNY15c5Z3PmWlJJfe1NmBIFJJAyB~cBETaa0VZCun6byiX6aDttqkMlZPnQZAbRx0iGyFSdK~-iuQa5tIZOL8hdpSX7VprQkhma4wECuM-g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-8</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/dsr021-dsr021supp_table1.doc?Expires=2147483647&amp;Signature=P7H-RbI530-c58m7PF~8F9dAz-RMBF4DVXMr9Ssfs06MLMH5cKX4oCZsWUTYNXSSWFrSqJwqq2AFUeBr0LtsqznevX5UBvIo4N8F03HjK0sjVVaFSpWyGq3WOWM9FUS03DFZ1trd0nRGUcS~gJeJOEY~jS2Z4Cpq4fG6b6J3uLp-6WsE1zVw-crd11mNEevxiCMBlGvetZGv3XyptMo-DuDTKTZnbf9SqAFZZWtmkFPaeTlhG316zuJTzKP3FLjgD-HlG5IB78TzbgBffCETxoWfkehC8ZZN7anVr~fa7~BYEysEf9F3MTBnzQj7TnC7ioQAiBC8SlaA1fqKQq2LIQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-7</a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/dsr021-dsr021supp_fig5.png?Expires=2147483647&amp;Signature=cwPydusCciMeTLtimH6UBQf0ewPszYDs3KRB2G5cHRV6kvS0hVtNrN1FvD5Qj37VsvKXzL3Gk9lPjnKNyKFFV9JrRsEsFAy1fu~5OVDqBq3gdA0OL32KPRy-FHfRBMVsjlXpWikuqpRV~v-KoGSduDQmmkvV9~CVAysTZWrlBKbOGMi9D903bCZHBU-uE2HdskTDkulPvDXl-WWWQ9e7hDx2adnVj4sdzgVdtyfrWf7edaoJLe1YqZSer9cgPzFhOhLArRKGQeQgMMZSmnGN7ctsF34E2YlPJixo65ITTxQ3qerAFfFENnCNK~Rt-p37zrHlk~~pF3EXd5FzmoUHTA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-6</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/dsr021-dsr021supp_fig4.png?Expires=2147483647&amp;Signature=L07otXQTWW~5ryhTmyr5~ttCFeRPqIbx3iCD~3Hch3EShMKwGhT-uNvLuRW2M~OoDH3ed~gWo5-oDMFDmOLqyTdTlkHLVeEoLBelkbW0HqlKN~T7FPeP1FVFgFFRDJbs8I1WdAmTOLwb3RpWhmA4vFKjGJOCn~taPn7uAZdctlgQtkrZ7E5uvMOTb-Aq~Y8L9pKETLENent7Ahdg2CLHiYkf4ojw4l-7yJ2ZpDfVgOFNUf8Y53~J-uj4Mo0hLdwX--dzom0zYWbu3zUcBo2WRx81jec1urr6QFCVk0JL2E80WVjmAidjxmAllEF-cNerS4YXhtp2AZFrJF0~N4t9TA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-5</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/dsr021-dsr021supp_fig3.png?Expires=2147483647&amp;Signature=eJK3J8jG88~SIvDUN8EdzJ~0-Mtzr33lBD0PToRs1bwfXFbg-UbXDw76JElYUQ1uUClncH1D7xnreAHRveJDtiK3un6QtS0XA-EtcOpmiy~PDsHNK8O8BCvIHvFTpkWttRHwuj~ORHJBUVeH~1Q~j18jSeHfAFJ~GBj4CLNTOqwJNwdoz70gAbcazhjadF6f~8I8fC4giaSfCg1z9db0cFYWdjcSXZLlKq5N0WOHBFA~aRjQgEz~ZaOZJF-l0ogRSW1p0aJYEgOuCh2HXcNxIO65zKeC4JLnNtRDetBnNYwCjTnEbP~Oz~S0xZORVw~-KJLVyke9sIF2f5UWemKIbw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-4</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/dsr021-dsr021supp_fig2.png?Expires=2147483647&amp;Signature=0xHnJssxtCpR43odkO38uJw-ZYxuJxg7QWqVmmHEj9KoJT06XW041kV3H4WsYxTv9cTU6epz3VxdXh4qASPpDMHLyfqoOSW1q6pnOTgVoaZbP2LhrovYQpYLHNhxwt7Alhv-WkhrRzu-bfrMRk5JzoPdV9fhJ7OzyVE~j7I05Fnm4E~BTWQyUBkk-bQYR4SccGQjjtBLVnhbzfuTSgiGNFGWIwjxTmwRin1GEntMk~BVter0kaqolKUxg-cTQQPUoc2LyatR-DL-yhL31JYwnxAYykRlrZ~HFW7C9jTyxo6b~jjYgy3lKLaqTpJXW-sZcGUg6cR7y66R-wYeXD00~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/dsr021-dsr021supp_fig1.png?Expires=2147483647&amp;Signature=KQOs8R3-7mwDjoORMnLAIhsWOLOJjnG2XYmDpb6zXtPYTntDk5otb9Kua-OLByt4AbOepJrzleGH2DekropQplMMn1ZR37e-15EqgAwkPt2BJqm0XcEUx-Dh3MnYxaHsQGwLOhX8LoJACzZ~KF8atu6LNpaaG9UFQNMuto5s2YZRcEDek6O9QRWN0Wj07Iak2aUTV8loZNeWuwLZkvGgrgxpOdpKqnLzp3teMuz76p3p9g983WMXcCbh4wmbhht0qRGtKAatUYAMFsie~SCUXN7DQ~EYNzPTFPMc-OiRJFXCVNrGg~5ftnvASQAYGLja9gNYVn8gGYEN58Qu1J4aAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - png file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/18/5/10.1093/dnares/dsr021/2/dsr021-dsr021supp.doc?Expires=2147483647&amp;Signature=tePpIj9kcChks7s~viZwCUTK8emGqKY5d0ycCaisLS6btPDAGib5qmpnCUxiOTUYpgCSd1-DqhOwx3XykTU1fpnTiQvMUN08cOJAKuCBCc1agbpwowvdWrWTa01BE4Zw7CFLTAwLnlor0EA1e5cwgOWQ5JYHwa1uzG71zFLg4zH0MCX7PACHlqQSOyP46SR7AbGxkoDeIr24u-Jwk3oikT4OLjbx5fwsPZpDNOwPF2cMQJDQEUSj0kbprIdQGbZp4FwEIYtx~8M41xJvOZQsbsiEj4wX3FEGOs~FyB7x0cM5SPyJctiqH0Res~XTYLCKuv6vC9D11pMzZ5vYYx6b~w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">192</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000296708600004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">16</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsr021"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="448814"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000296708600004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f18%2f5%2f343%2f448814">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000296708600004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (16)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F18%2F5%2F343%2F448814" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/448814'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Analysis of Muscle and Ovary Transcriptome of Sus scrofa: Assembly, Annotation and Marker Discovery" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsr021");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f18%2f5%2f343%2f448814">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

