

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/15/4/215/376506","@type":"ScholarlyArticle","name":"Determination of the Genome Sequence of  Porphyromonas gingivalis  Strain ATCC 33277 and Genomic Comparison with Strain W83 Revealed Extensive Genome Rearrangements in  P. gingivalis ","datePublished":"2008-06-03","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/15/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2008-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsn013","keywords":["<italic>Porphyromonas gingivalis</italic>","whole genome sequence","genome rearrangement","conjugative transposon","MITE"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/15/4/215/376506","author":[{"name":"Naito, Mariko","affiliation":"1  Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunology ,  Nagasaki University Graduate School of Biomedical Sciences ,  Sakamoto 1-7-1, Nagasaki 852-8588 ,  Japan","@type":"Person"},{"name":"Hirakawa, Hideki","affiliation":"2  Graduate School of Systems Life Sciences, Kyushu University ,  Hakozaki 6-10-1, Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"},{"name":"Yamashita, Atsushi","affiliation":"3  Kitasato Institute for Life Sciences, Kitasato University ,  Kitasato 1-15-1, Sagamihara, Kanagawa 228-8555 ,  Japan","@type":"Person"},{"name":"Ohara, Naoya","affiliation":"1  Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunology ,  Nagasaki University Graduate School of Biomedical Sciences ,  Sakamoto 1-7-1, Nagasaki 852-8588 ,  Japan","@type":"Person"},{"name":"Shoji, Mikio","affiliation":"1  Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunology ,  Nagasaki University Graduate School of Biomedical Sciences ,  Sakamoto 1-7-1, Nagasaki 852-8588 ,  Japan","@type":"Person"},{"name":"Yukitake, Hideharu","affiliation":"1  Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunology ,  Nagasaki University Graduate School of Biomedical Sciences ,  Sakamoto 1-7-1, Nagasaki 852-8588 ,  Japan","@type":"Person"},{"name":"Nakayama, Keisuke","affiliation":"4  Division of Bioenvironmental Science, Frontier Science Research Center ,  University of Miyazaki ,  Kihara 5200, Kiyotake, Miyazaki 889-1692 ,  Japan","@type":"Person"},{"name":"Toh, Hidehiro","affiliation":"3  Kitasato Institute for Life Sciences, Kitasato University ,  Kitasato 1-15-1, Sagamihara, Kanagawa 228-8555 ,  Japan","@type":"Person"},{"name":"Yoshimura, Fuminobu","affiliation":"5  Department of Microbiology ,  School of Dentistry, Aichi-Gakuin University ,  1-100 Kusumoto-cho, Chikusa-ku, Nagoya 464-8650 ,  Japan","@type":"Person"},{"name":"Kuhara, Satoru","affiliation":"6  Graduate School of Genetic Resources Technology, Kyushu University ,  Hakozaki 6-10-1, Higashi-ku, Fukuoka 812-8581 ,  Japan","@type":"Person"},{"name":"Hattori, Masahira","affiliation":"3  Kitasato Institute for Life Sciences, Kitasato University ,  Kitasato 1-15-1, Sagamihara, Kanagawa 228-8555 ,  Japan    7  Department of Computational Biology ,  Graduate School of Frontier Sciences, The University of Tokyo ,  Kashiwanoha 5-1-5, Chiba 277-8561 ,  Japan","@type":"Person"},{"name":"Hayashi, Tetsuya","affiliation":"4  Division of Bioenvironmental Science, Frontier Science Research Center ,  University of Miyazaki ,  Kihara 5200, Kiyotake, Miyazaki 889-1692 ,  Japan","@type":"Person"},{"name":"Nakayama, Koji","affiliation":"1  Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunology ,  Nagasaki University Graduate School of Biomedical Sciences ,  Sakamoto 1-7-1, Nagasaki 852-8588 ,  Japan","@type":"Person"}],"description":"Abstract.  The gram-negative anaerobic bacterium Porphyromonas gingivalis is a major causative agent of chronic periodontitis. Porphyromonas gingivalis strains ","pageStart":"215","pageEnd":"225","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/4/1/m_cover.gif?Expires=2147483647&Signature=RLDuHMUA6eE0ByVqiq8aUGyMxZAqCYh6CUPq-ezrEq~vX1AX9OwM0EdcZgw7bZO5GF08rcIkuMn~SdMlIqZ-OMoLaS-efZLrQFFAEDhygPUYmAsVLKwz1N0-SZE8tkL655rpVOyhNsmCq0HeMyZ1CRYm6AEF3IsWa0HPWbYG3ZsNpccsrKjShJ5nSPlVjprfmUgaJjeTeE3~oZCdhOYmMPqtkUKaLjbpdnpy8KGSYF3y2vs7JTB1QHc1IXmul6fvj2x9yPzU-FZHZaBu8~fDI-1ze4-bV3EV~DhG3VaQYIzUv05cvil7Rnpu6ewBIQhqRHLzHcljRy6MHn~J052jxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Determination of the Genome Sequence of  Porphyromonas gingivalis  Strain ATCC 33277 and Genomic Comparison with Strain W83 Revealed Extensive Genome Rearrangements in  P. gingivalis ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/4/1/m_cover.gif?Expires=2147483647&Signature=RLDuHMUA6eE0ByVqiq8aUGyMxZAqCYh6CUPq-ezrEq~vX1AX9OwM0EdcZgw7bZO5GF08rcIkuMn~SdMlIqZ-OMoLaS-efZLrQFFAEDhygPUYmAsVLKwz1N0-SZE8tkL655rpVOyhNsmCq0HeMyZ1CRYm6AEF3IsWa0HPWbYG3ZsNpccsrKjShJ5nSPlVjprfmUgaJjeTeE3~oZCdhOYmMPqtkUKaLjbpdnpy8KGSYF3y2vs7JTB1QHc1IXmul6fvj2x9yPzU-FZHZaBu8~fDI-1ze4-bV3EV~DhG3VaQYIzUv05cvil7Rnpu6ewBIQhqRHLzHcljRy6MHn~J052jxA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Determination of the Genome Sequence of  Porphyromonas gingivalis  Strain ATCC 33277 and Genomic Comparison with Strain W83 Revealed Extensive Genome Rearrangements in  P. gingivalis " />
    <meta property="og:description" content="Abstract.  The gram-negative anaerobic bacterium Porphyromonas gingivalis is a major causative agent of chronic periodontitis. Porphyromonas gingivalis strains " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/15/4/215/376506" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2008-06-03" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Naito, Mariko" /><meta name="citation_author_institution" content="Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunology, Nagasaki University Graduate School of Biomedical Sciences, Sakamoto 1-7-1, Nagasaki 852-8588, Japan" /><meta name="citation_author" content="Hirakawa, Hideki" /><meta name="citation_author_institution" content="Graduate School of Systems Life Sciences, Kyushu University, Hakozaki 6-10-1, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_author" content="Yamashita, Atsushi" /><meta name="citation_author_institution" content="Kitasato Institute for Life Sciences, Kitasato University, Kitasato 1-15-1, Sagamihara, Kanagawa 228-8555, Japan" /><meta name="citation_author" content="Ohara, Naoya" /><meta name="citation_author_institution" content="Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunology, Nagasaki University Graduate School of Biomedical Sciences, Sakamoto 1-7-1, Nagasaki 852-8588, Japan" /><meta name="citation_author" content="Shoji, Mikio" /><meta name="citation_author_institution" content="Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunology, Nagasaki University Graduate School of Biomedical Sciences, Sakamoto 1-7-1, Nagasaki 852-8588, Japan" /><meta name="citation_author" content="Yukitake, Hideharu" /><meta name="citation_author_institution" content="Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunology, Nagasaki University Graduate School of Biomedical Sciences, Sakamoto 1-7-1, Nagasaki 852-8588, Japan" /><meta name="citation_author" content="Nakayama, Keisuke" /><meta name="citation_author_institution" content="Division of Bioenvironmental Science, Frontier Science Research Center, University of Miyazaki, Kihara 5200, Kiyotake, Miyazaki 889-1692, Japan" /><meta name="citation_author" content="Toh, Hidehiro" /><meta name="citation_author_institution" content="Kitasato Institute for Life Sciences, Kitasato University, Kitasato 1-15-1, Sagamihara, Kanagawa 228-8555, Japan" /><meta name="citation_author" content="Yoshimura, Fuminobu" /><meta name="citation_author_institution" content="Department of Microbiology, School of Dentistry, Aichi-Gakuin University, 1-100 Kusumoto-cho, Chikusa-ku, Nagoya 464-8650, Japan" /><meta name="citation_author" content="Kuhara, Satoru" /><meta name="citation_author_institution" content="Graduate School of Genetic Resources Technology, Kyushu University, Hakozaki 6-10-1, Higashi-ku, Fukuoka 812-8581, Japan" /><meta name="citation_author" content="Hattori, Masahira" /><meta name="citation_author_institution" content="Kitasato Institute for Life Sciences, Kitasato University, Kitasato 1-15-1, Sagamihara, Kanagawa 228-8555, Japan" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwanoha 5-1-5, Chiba 277-8561, Japan" /><meta name="citation_author" content="Hayashi, Tetsuya" /><meta name="citation_author_institution" content="Division of Bioenvironmental Science, Frontier Science Research Center, University of Miyazaki, Kihara 5200, Kiyotake, Miyazaki 889-1692, Japan" /><meta name="citation_author" content="Nakayama, Koji" /><meta name="citation_author_institution" content="Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunology, Nagasaki University Graduate School of Biomedical Sciences, Sakamoto 1-7-1, Nagasaki 852-8588, Japan" /><meta name="citation_title" content="Determination of the Genome Sequence of Porphyromonas gingivalis Strain ATCC 33277 and Genomic Comparison with Strain W83 Revealed Extensive Genome Rearrangements in P. gingivalis" /><meta name="citation_firstpage" content="215" /><meta name="citation_lastpage" content="225" /><meta name="citation_doi" content="10.1093/dnares/dsn013" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="15" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2008/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Epidemiology of periodontal diseases: an update; J. Int. Acad. Periodontol.;  citation_year=1999;  citation_volume=1;  citation_pages=110-116; " /><meta name="citation_reference" content="citation_title=Epidemiology of periodontal disease: a review and clinical perspectives; J. Int. Acad. Periodontol.;  citation_year=2001;  citation_volume=3;  citation_pages=14-21; " /><meta name="citation_reference" content="citation_title=Periodontal diseases: diagnosis; Ann. Periodontol.;  citation_year=1996;  citation_volume=1;  citation_pages=37-215; " /><meta name="citation_reference" content="citation_title=Periodontal diseases in the United States population; J. Periodontol.;  citation_year=1998;  citation_volume=69;  citation_pages=269-278; " /><meta name="citation_reference" content="citation_title=Dental infection and the risk of new coronary events: prospective study of patients with documented coronary artery disease; Clin. Infect. Dis.;  citation_year=1995;  citation_volume=20;  citation_pages=588-592; " /><meta name="citation_reference" content="citation_title=Periodontal disease and cardiovascular disease; J. Periodontol.;  citation_year=1996;  citation_volume=67;  citation_pages=1123-1137; " /><meta name="citation_reference" content="citation_title=Poor oral health and coronary heart disease; J. Dent. Res.;  citation_year=1996;  citation_volume=75;  citation_pages=1631-1636; " /><meta name="citation_reference" content="citation_title=Periodontal disease and risk of fatal coronary heart and cerebrovascular diseases; J. Cardiovasc. Risk;  citation_year=1999;  citation_volume=6;  citation_pages=7-11; " /><meta name="citation_reference" content="citation_title=Multiple infections in carotid atherosclerotic plaques; Am. Heart J.;  citation_year=1999;  citation_volume=138;  citation_pages=S534-S536; " /><meta name="citation_reference" content="citation_title=Human atherosclerotic plaque contains viable Actinbacillus actinomycetemcomitans and Porphyromonas gingivalis; Arterioscler. Thromb. Vasc. Biol.;  citation_year=2005;  citation_volume=25;  citation_pages=e17-e18; " /><meta name="citation_reference" content=" citation_edition=2nd Ed; citation_publisher=New York, Springer-Verlag; Bergey's Manual of Systematic Bacteriology;  citation_year=2002;  citation_volume=Vol. 1; " /><meta name="citation_reference" content="citation_title=Biology of asaccharolytic black-pigmented Bacteroides species; Microbiol. Rev.;  citation_year=1988;  citation_volume=52;  citation_pages=134-152; " /><meta name="citation_reference" content="citation_title=Life below the gum line: pathogenic mechanisms of Porphyromonas gingivalis; Microbiol. Mol. Biol. Rev.;  citation_year=1998;  citation_volume=62;  citation_pages=1244-1263; " /><meta name="citation_reference" content="citation_title=Virulence factors of Porphyromonas gingivalis; Periodontology;  citation_year=1999;  citation_volume=20;  citation_pages=168-238; " /><meta name="citation_reference" content="citation_title=Disruption of epithelial barrier and impairment of cellular function by Porphyromonas gingivalis; Front. Biosci.;  citation_year=2007;  citation_volume=12;  citation_pages=3965-3974; " /><meta name="citation_reference" content="citation_title=Characterization of biologically active cell surface components of a periodontal pathogen: the roles of major and minor fimbriae of Porphyromonas gingivalis; J. Periodontol.;  citation_year=2003;  citation_volume=74;  citation_pages=119-122; " /><meta name="citation_reference" content="citation_title=Porphyromonas gingivalis proteinases as virulence determinants in progression of periodontal diseases; J. Biochem.;  citation_year=2000;  citation_volume=128;  citation_pages=153-159; " /><meta name="citation_reference" content="citation_title=Pathogenicity; citation_publisher=CRC Press Inc, Boca Raton, FL, pp; Biology of the Species Porphyromonas gingivalis;  citation_year=1993;  citation_pages=127-131; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of the oral pathogenic Bacterium Porphyromonas gingivalis strain W83; J. Bacteriol.;  citation_year=2003;  citation_volume=185;  citation_pages=5591-5601; " /><meta name="citation_reference" content="citation_title=Rapid viability loss on exposure to air in a superoxide dismutase-deficient mutant of Porphyromonas gingivalis; J. Bacteriol.;  citation_year=1994;  citation_volume=176;  citation_pages=1939-1943; " /><meta name="citation_reference" content="citation_title=Construction and characterization of arginine-specific cysteine proteinase (Arg-gingipain)-deficient mutants of Porphyromonas gingivalis: evidence for significant contribution of Arg-gingipain to virulence; J. Biol. Chem.;  citation_year=1995;  citation_volume=270;  citation_pages=23619-23626; " /><meta name="citation_reference" content="citation_title=Involvement of a lysine-specific cysteine proteinase in hemoglobin adsorption and heme accumulation by Porphyromonas gingivalis; J. Biol. Chem.;  citation_year=1998;  citation_volume=273;  citation_pages=21225-21231; " /><meta name="citation_reference" content="citation_title=Genetic analyses of proteolysis, hemoglobin binding, and hemagglutination of Porphyromonas gingivalis: construction of mutants with a combination of rgpA, rgpB, kgp, and hagA; J. Biol. Chem.;  citation_year=1999;  citation_volume=274;  citation_pages=17955-17960; " /><meta name="citation_reference" content="citation_title=Porphyromonas gingivalis-induced platelet aggregation in plasma depends on Hgp44 adhesin but not Rgp proteinase; Mol. Microbiol.;  citation_year=2006;  citation_volume=59;  citation_pages=152-167; " /><meta name="citation_reference" content="citation_title=Isolation and characterization of transposon-induced mutants of Porphyromonas gingivalis deficient in fimbriation; Microb. Pathog.;  citation_year=1998;  citation_volume=24;  citation_pages=25-35; " /><meta name="citation_reference" content="citation_title=Base-calling of automated sequencer traces using Phred. II. Error probabilities; Genome Res.;  citation_year=1998;  citation_volume=8;  citation_pages=186-194; " /><meta name="citation_reference" content="citation_title=Automated finishing with autofinish; Genome Res.;  citation_year=2001;  citation_volume=11;  citation_pages=614-625; " /><meta name="citation_reference" content="citation_title=Determination of the genome size of the oral anaerobic bacterium Porphyromonas gingivalis by pulsed field gel electrophoresis; Dent. Jpn.;  citation_year=1995;  citation_volume=32;  citation_pages=25-28; " /><meta name="citation_reference" content="citation_title=An automated system for genome analysis to support microbial whole-genome shotgun sequencing; Biosci. Biotechnol. Biochem.;  citation_year=2000;  citation_volume=64;  citation_pages=670-673; " /><meta name="citation_reference" content="citation_title=CRITICA: coding region identification tool invoking comparative analysis; Mol. Biol. Evol.;  citation_year=1999;  citation_volume=16;  citation_pages=512-524; " /><meta name="citation_reference" content="citation_title=Modeling and predicting transcriptional units of Escherichia coli genes using hidden Markov models; Bioinformatics;  citation_year=1999;  citation_volume=15;  citation_pages=987-993; " /><meta name="citation_reference" content="citation_title=Combining diverse evidence for gene recognition in completely sequenced bacterial genomes; Nucleic Acids Res.;  citation_year=1998;  citation_volume=26;  citation_pages=2941-2947; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-3402; " /><meta name="citation_reference" content="citation_title=tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence; Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=955-964; " /><meta name="citation_reference" content="citation_title=Fast algorithms for large-scale genome alignment and comparison; Nucleic Acids Res.;  citation_year=2002;  citation_volume=30;  citation_pages=2478-2483; " /><meta name="citation_reference" content="citation_title=PipMaker—a web server for aligning two genomic DNA sequences; Genome Res.;  citation_year=2000;  citation_volume=10;  citation_pages=577-586; " /><meta name="citation_reference" content="citation_title=A new mathematical model for relative quantification in real-time RT–PCR; Nucleic Acids Res.;  citation_year=2001;  citation_volume=29; " /><meta name="citation_reference" content="citation_title=Genetic and structural analysis of the Bacteroides conjugative transposon CTn341; J. Bacteriol.;  citation_year=2005;  citation_volume=187;  citation_pages=2858-2869; " /><meta name="citation_reference" content="citation_title=Transfer region of a bacteroides conjugative transposon, CTnDOT; Plasmid;  citation_year=2001;  citation_volume=45;  citation_pages=41-51; " /><meta name="citation_reference" content="citation_title=Genomic analysis of Bacteroides fragilis reveals extensive DNA inversions regulating cell surface adaptation; Proc. Natl. Acad. Sci. USA;  citation_year=2004;  citation_volume=101;  citation_pages=14919-14924; " /><meta name="citation_reference" content="citation_title=Plant transposable elements: where genetics meets genomics; Nat. Rev. Genet.;  citation_year=2002;  citation_volume=3;  citation_pages=329-341; " /><meta name="citation_reference" content="citation_title=Genomic loci of the Porphyromonas gingivalis insertion element IS1126; Infect. Immun.;  citation_year=1999;  citation_volume=67;  citation_pages=3416-3423; " /><meta name="citation_reference" content="citation_title=Identification and characterization of the capsular (K-antigen) locus of Porphyromonas gingivalis; Infect. Immun.;  citation_year=2006;  citation_volume=74;  citation_pages=449-460; " /><meta name="citation_reference" content="citation_title=Molecular cloning and sequencing of the fimbrilin gene of Porphyromonas gingivalis strains and characterization of recombinant proteins; Biochem. Biophys. Res. Commun.;  citation_year=1993;  citation_volume=197;  citation_pages=241-247; " /><meta name="citation_reference" content="citation_title=Sequence diversity and antigenic variation at the rag locus of Porphyromonas gingivalis; Infect. Immun.;  citation_year=2005;  citation_volume=73;  citation_pages=4253-4262; " /><meta name="citation_reference" content="citation_title=Extensive DNA inversions in the B. fragilis genome control variable gene expression; Science;  citation_year=2005;  citation_volume=307;  citation_pages=1463-1465; " /><meta name="citation_reference" content="citation_title=Complete genome sequence of Shigella flexneri 5b and comparison with Shigella flexneri 2a; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=173" /><meta name="citation_reference" content="citation_title=Complete genome sequence and comparative genomics of Shigella flexneri serotype 2a strain 2457T; Infect. Immun.;  citation_year=2003;  citation_volume=71;  citation_pages=2775-2786; " /><meta name="citation_reference" content="citation_title=Role of oxyR in the oral anaerobe Porphyromonas gingivalis; J. Bacteriol.;  citation_year=2006;  citation_volume=188;  citation_pages=2454-2462; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/15/4/215/1070107/dsn013.pdf" /><meta name="description" content="Abstract.  The gram-negative anaerobic bacterium Porphyromonas gingivalis is a major causative agent of chronic periodontitis. Porphyromonas gingivalis strains " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/15/4/215/376506" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/15/4/215/376506" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/15/4/215/376506");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsn013';
    </script>
<title>
	Determination of the Genome Sequence of Porphyromonas gingivalis Strain ATCC 33277 and Genomic Comparison with Strain W83 Revealed Extensive Genome Rearrangements in P. gingivalis | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./376506" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bc1+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWl0bywgTWFyaWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgTWljcm9iaW9sb2d5IGFuZCBPcmFsIEluZmVjdGlvbiwgRGVwYXJ0bWVudCBvZiBNb2xlY3VsYXIgTWljcm9iaW9sb2d5IGFuZCBJbW11bm9sb2d5LCBOYWdhc2FraSBVbml2ZXJzaXR5IEdyYWR1YXRlIFNjaG9vbCBvZiBCaW9tZWRpY2FsIFNjaWVuY2VzLCBTYWthbW90byAxLTctMSwgTmFnYXNha2kgODUyLTg1ODgsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGlyYWthd2EsIEhpZGVraSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkdyYWR1YXRlIFNjaG9vbCBvZiBTeXN0ZW1zIExpZmUgU2NpZW5jZXMsIEt5dXNodSBVbml2ZXJzaXR5LCBIYWtvemFraSA2LTEwLTEsIEhpZ2FzaGkta3UsIEZ1a3Vva2EgODEyLTg1ODEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWWFtYXNoaXRhLCBBdHN1c2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2l0YXNhdG8gSW5zdGl0dXRlIGZvciBMaWZlIFNjaWVuY2VzLCBLaXRhc2F0byBVbml2ZXJzaXR5LCBLaXRhc2F0byAxLTE1LTEsIFNhZ2FtaWhhcmEsIEthbmFnYXdhIDIyOC04NTU1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik9oYXJhLCBOYW95YSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIE1pY3JvYmlvbG9neSBhbmQgT3JhbCBJbmZlY3Rpb24sIERlcGFydG1lbnQgb2YgTW9sZWN1bGFyIE1pY3JvYmlvbG9neSBhbmQgSW1tdW5vbG9neSwgTmFnYXNha2kgVW5pdmVyc2l0eSBHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvbWVkaWNhbCBTY2llbmNlcywgU2FrYW1vdG8gMS03LTEsIE5hZ2FzYWtpIDg1Mi04NTg4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlNob2ppLCBNaWtpbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIE1pY3JvYmlvbG9neSBhbmQgT3JhbCBJbmZlY3Rpb24sIERlcGFydG1lbnQgb2YgTW9sZWN1bGFyIE1pY3JvYmlvbG9neSBhbmQgSW1tdW5vbG9neSwgTmFnYXNha2kgVW5pdmVyc2l0eSBHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvbWVkaWNhbCBTY2llbmNlcywgU2FrYW1vdG8gMS03LTEsIE5hZ2FzYWtpIDg1Mi04NTg4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ill1a2l0YWtlLCBIaWRlaGFydSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIE1pY3JvYmlvbG9neSBhbmQgT3JhbCBJbmZlY3Rpb24sIERlcGFydG1lbnQgb2YgTW9sZWN1bGFyIE1pY3JvYmlvbG9neSBhbmQgSW1tdW5vbG9neSwgTmFnYXNha2kgVW5pdmVyc2l0eSBHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvbWVkaWNhbCBTY2llbmNlcywgU2FrYW1vdG8gMS03LTEsIE5hZ2FzYWtpIDg1Mi04NTg4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik5ha2F5YW1hLCBLZWlzdWtlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgQmlvZW52aXJvbm1lbnRhbCBTY2llbmNlLCBGcm9udGllciBTY2llbmNlIFJlc2VhcmNoIENlbnRlciwgVW5pdmVyc2l0eSBvZiBNaXlhemFraSwgS2loYXJhIDUyMDAsIEtpeW90YWtlLCBNaXlhemFraSA4ODktMTY5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUb2gsIEhpZGVoaXJvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iS2l0YXNhdG8gSW5zdGl0dXRlIGZvciBMaWZlIFNjaWVuY2VzLCBLaXRhc2F0byBVbml2ZXJzaXR5LCBLaXRhc2F0byAxLTE1LTEsIFNhZ2FtaWhhcmEsIEthbmFnYXdhIDIyOC04NTU1LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Illvc2hpbXVyYSwgRnVtaW5vYnUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE1pY3JvYmlvbG9neSwgU2Nob29sIG9mIERlbnRpc3RyeSwgQWljaGktR2FrdWluIFVuaXZlcnNpdHksIDEtMTAwIEt1c3Vtb3RvLWNobywgQ2hpa3VzYS1rdSwgTmFnb3lhIDQ2NC04NjUwLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikt1aGFyYSwgU2F0b3J1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3JhZHVhdGUgU2Nob29sIG9mIEdlbmV0aWMgUmVzb3VyY2VzIFRlY2hub2xvZ3ksIEt5dXNodSBVbml2ZXJzaXR5LCBIYWtvemFraSA2LTEwLTEsIEhpZ2FzaGkta3UsIEZ1a3Vva2EgODEyLTg1ODEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iSGF0dG9yaSwgTWFzYWhpcmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJLaXRhc2F0byBJbnN0aXR1dGUgZm9yIExpZmUgU2NpZW5jZXMsIEtpdGFzYXRvIFVuaXZlcnNpdHksIEtpdGFzYXRvIDEtMTUtMSwgU2FnYW1paGFyYSwgS2FuYWdhd2EgMjI4LTg1NTUsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBDb21wdXRhdGlvbmFsIEJpb2xvZ3ksIEdyYWR1YXRlIFNjaG9vbCBvZiBGcm9udGllciBTY2llbmNlcywgVGhlIFVuaXZlcnNpdHkgb2YgVG9reW8sIEthc2hpd2Fub2hhIDUtMS01LCBDaGliYSAyNzctODU2MSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIYXlhc2hpLCBUZXRzdXlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgQmlvZW52aXJvbm1lbnRhbCBTY2llbmNlLCBGcm9udGllciBTY2llbmNlIFJlc2VhcmNoIENlbnRlciwgVW5pdmVyc2l0eSBvZiBNaXlhemFraSwgS2loYXJhIDUyMDAsIEtpeW90YWtlLCBNaXlhemFraSA4ODktMTY5MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJOYWtheWFtYSwgS29qaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIE1pY3JvYmlvbG9neSBhbmQgT3JhbCBJbmZlY3Rpb24sIERlcGFydG1lbnQgb2YgTW9sZWN1bGFyIE1pY3JvYmlvbG9neSBhbmQgSW1tdW5vbG9neSwgTmFnYXNha2kgVW5pdmVyc2l0eSBHcmFkdWF0ZSBTY2hvb2wgb2YgQmlvbWVkaWNhbCBTY2llbmNlcywgU2FrYW1vdG8gMS03LTEsIE5hZ2FzYWtpIDg1Mi04NTg4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iRGV0ZXJtaW5hdGlvbiBvZiB0aGUgR2Vub21lIFNlcXVlbmNlIG9mIFBvcnBoeXJvbW9uYXMgZ2luZ2l2YWxpcyBTdHJhaW4gQVRDQyAzMzI3NyBhbmQgR2Vub21pYyBDb21wYXJpc29uIHdpdGggU3RyYWluIFc4MyBSZXZlYWxlZCBFeHRlbnNpdmUgR2Vub21lIFJlYXJyYW5nZW1lbnRzIGluIFAuIGdpbmdpdmFsaXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSIyMTUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjIyNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzbjAxMyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIxNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDA4LzA4LzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXBpZGVtaW9sb2d5IG9mIHBlcmlvZG9udGFsIGRpc2Vhc2VzOiBhbiB1cGRhdGU7IEouIEludC4gQWNhZC4gUGVyaW9kb250b2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTE7ICBjaXRhdGlvbl9wYWdlcz0xMTAtMTE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcGlkZW1pb2xvZ3kgb2YgcGVyaW9kb250YWwgZGlzZWFzZTogYSByZXZpZXcgYW5kIGNsaW5pY2FsIHBlcnNwZWN0aXZlczsgSi4gSW50LiBBY2FkLiBQZXJpb2RvbnRvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTE0LTIxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZXJpb2RvbnRhbCBkaXNlYXNlczogZGlhZ25vc2lzOyBBbm4uIFBlcmlvZG9udG9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9MzctMjE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QZXJpb2RvbnRhbCBkaXNlYXNlcyBpbiB0aGUgVW5pdGVkIFN0YXRlcyBwb3B1bGF0aW9uOyBKLiBQZXJpb2RvbnRvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9Njk7ICBjaXRhdGlvbl9wYWdlcz0yNjktMjc4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZW50YWwgaW5mZWN0aW9uIGFuZCB0aGUgcmlzayBvZiBuZXcgY29yb25hcnkgZXZlbnRzOiBwcm9zcGVjdGl2ZSBzdHVkeSBvZiBwYXRpZW50cyB3aXRoIGRvY3VtZW50ZWQgY29yb25hcnkgYXJ0ZXJ5IGRpc2Vhc2U7IENsaW4uIEluZmVjdC4gRGlzLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTU4OC01OTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlcmlvZG9udGFsIGRpc2Vhc2UgYW5kIGNhcmRpb3Zhc2N1bGFyIGRpc2Vhc2U7IEouIFBlcmlvZG9udG9sLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT02NzsgIGNpdGF0aW9uX3BhZ2VzPTExMjMtMTEzNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9vciBvcmFsIGhlYWx0aCBhbmQgY29yb25hcnkgaGVhcnQgZGlzZWFzZTsgSi4gRGVudC4gUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NjsgIGNpdGF0aW9uX3ZvbHVtZT03NTsgIGNpdGF0aW9uX3BhZ2VzPTE2MzEtMTYzNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGVyaW9kb250YWwgZGlzZWFzZSBhbmQgcmlzayBvZiBmYXRhbCBjb3JvbmFyeSBoZWFydCBhbmQgY2VyZWJyb3Zhc2N1bGFyIGRpc2Vhc2VzOyBKLiBDYXJkaW92YXNjLiBSaXNrOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTY7ICBjaXRhdGlvbl9wYWdlcz03LTExOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NdWx0aXBsZSBpbmZlY3Rpb25zIGluIGNhcm90aWQgYXRoZXJvc2NsZXJvdGljIHBsYXF1ZXM7IEFtLiBIZWFydCBKLjsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzg7ICBjaXRhdGlvbl9wYWdlcz1TNTM0LVM1MzY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUh1bWFuIGF0aGVyb3NjbGVyb3RpYyBwbGFxdWUgY29udGFpbnMgdmlhYmxlIEFjdGluYmFjaWxsdXMgYWN0aW5vbXljZXRlbWNvbWl0YW5zIGFuZCBQb3JwaHlyb21vbmFzIGdpbmdpdmFsaXM7IEFydGVyaW9zY2xlci4gVGhyb21iLiBWYXNjLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPWUxNy1lMTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9IiBjaXRhdGlvbl9lZGl0aW9uPTJuZCBFZDsgY2l0YXRpb25fcHVibGlzaGVyPU5ldyBZb3JrLCBTcHJpbmdlci1WZXJsYWc7IEJlcmdleSdzIE1hbnVhbCBvZiBTeXN0ZW1hdGljIEJhY3RlcmlvbG9neTsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT1Wb2wuIDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJpb2xvZ3kgb2YgYXNhY2NoYXJvbHl0aWMgYmxhY2stcGlnbWVudGVkIEJhY3Rlcm9pZGVzIHNwZWNpZXM7IE1pY3JvYmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTEzNC0xNTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpZmUgYmVsb3cgdGhlIGd1bSBsaW5lOiBwYXRob2dlbmljIG1lY2hhbmlzbXMgb2YgUG9ycGh5cm9tb25hcyBnaW5naXZhbGlzOyBNaWNyb2Jpb2wuIE1vbC4gQmlvbC4gUmV2LjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT02MjsgIGNpdGF0aW9uX3BhZ2VzPTEyNDQtMTI2MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VmlydWxlbmNlIGZhY3RvcnMgb2YgUG9ycGh5cm9tb25hcyBnaW5naXZhbGlzOyBQZXJpb2RvbnRvbG9neTsgIGNpdGF0aW9uX3llYXI9MTk5OTsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTE2OC0yMzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpc3J1cHRpb24gb2YgZXBpdGhlbGlhbCBiYXJyaWVyIGFuZCBpbXBhaXJtZW50IG9mIGNlbGx1bGFyIGZ1bmN0aW9uIGJ5IFBvcnBoeXJvbW9uYXMgZ2luZ2l2YWxpczsgRnJvbnQuIEJpb3NjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0zOTY1LTM5NzQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgYmlvbG9naWNhbGx5IGFjdGl2ZSBjZWxsIHN1cmZhY2UgY29tcG9uZW50cyBvZiBhIHBlcmlvZG9udGFsIHBhdGhvZ2VuOiB0aGUgcm9sZXMgb2YgbWFqb3IgYW5kIG1pbm9yIGZpbWJyaWFlIG9mIFBvcnBoeXJvbW9uYXMgZ2luZ2l2YWxpczsgSi4gUGVyaW9kb250b2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTc0OyAgY2l0YXRpb25fcGFnZXM9MTE5LTEyMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9ycGh5cm9tb25hcyBnaW5naXZhbGlzIHByb3RlaW5hc2VzIGFzIHZpcnVsZW5jZSBkZXRlcm1pbmFudHMgaW4gcHJvZ3Jlc3Npb24gb2YgcGVyaW9kb250YWwgZGlzZWFzZXM7IEouIEJpb2NoZW0uOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTEyODsgIGNpdGF0aW9uX3BhZ2VzPTE1My0xNTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhdGhvZ2VuaWNpdHk7IGNpdGF0aW9uX3B1Ymxpc2hlcj1DUkMgUHJlc3MgSW5jLCBCb2NhIFJhdG9uLCBGTCwgcHA7IEJpb2xvZ3kgb2YgdGhlIFNwZWNpZXMgUG9ycGh5cm9tb25hcyBnaW5naXZhbGlzOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fcGFnZXM9MTI3LTEzMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIHRoZSBvcmFsIHBhdGhvZ2VuaWMgQmFjdGVyaXVtIFBvcnBoeXJvbW9uYXMgZ2luZ2l2YWxpcyBzdHJhaW4gVzgzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE4NTsgIGNpdGF0aW9uX3BhZ2VzPTU1OTEtNTYwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmFwaWQgdmlhYmlsaXR5IGxvc3Mgb24gZXhwb3N1cmUgdG8gYWlyIGluIGEgc3VwZXJveGlkZSBkaXNtdXRhc2UtZGVmaWNpZW50IG11dGFudCBvZiBQb3JwaHlyb21vbmFzIGdpbmdpdmFsaXM7IEouIEJhY3RlcmlvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTQ7ICBjaXRhdGlvbl92b2x1bWU9MTc2OyAgY2l0YXRpb25fcGFnZXM9MTkzOS0xOTQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db25zdHJ1Y3Rpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgYXJnaW5pbmUtc3BlY2lmaWMgY3lzdGVpbmUgcHJvdGVpbmFzZSAoQXJnLWdpbmdpcGFpbiktZGVmaWNpZW50IG11dGFudHMgb2YgUG9ycGh5cm9tb25hcyBnaW5naXZhbGlzOiBldmlkZW5jZSBmb3Igc2lnbmlmaWNhbnQgY29udHJpYnV0aW9uIG9mIEFyZy1naW5naXBhaW4gdG8gdmlydWxlbmNlOyBKLiBCaW9sLiBDaGVtLjsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3ZvbHVtZT0yNzA7ICBjaXRhdGlvbl9wYWdlcz0yMzYxOS0yMzYyNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW52b2x2ZW1lbnQgb2YgYSBseXNpbmUtc3BlY2lmaWMgY3lzdGVpbmUgcHJvdGVpbmFzZSBpbiBoZW1vZ2xvYmluIGFkc29ycHRpb24gYW5kIGhlbWUgYWNjdW11bGF0aW9uIGJ5IFBvcnBoeXJvbW9uYXMgZ2luZ2l2YWxpczsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9MjczOyAgY2l0YXRpb25fcGFnZXM9MjEyMjUtMjEyMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgYW5hbHlzZXMgb2YgcHJvdGVvbHlzaXMsIGhlbW9nbG9iaW4gYmluZGluZywgYW5kIGhlbWFnZ2x1dGluYXRpb24gb2YgUG9ycGh5cm9tb25hcyBnaW5naXZhbGlzOiBjb25zdHJ1Y3Rpb24gb2YgbXV0YW50cyB3aXRoIGEgY29tYmluYXRpb24gb2YgcmdwQSwgcmdwQiwga2dwLCBhbmQgaGFnQTsgSi4gQmlvbC4gQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9Mjc0OyAgY2l0YXRpb25fcGFnZXM9MTc5NTUtMTc5NjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBvcnBoeXJvbW9uYXMgZ2luZ2l2YWxpcy1pbmR1Y2VkIHBsYXRlbGV0IGFnZ3JlZ2F0aW9uIGluIHBsYXNtYSBkZXBlbmRzIG9uIEhncDQ0IGFkaGVzaW4gYnV0IG5vdCBSZ3AgcHJvdGVpbmFzZTsgTW9sLiBNaWNyb2Jpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTU5OyAgY2l0YXRpb25fcGFnZXM9MTUyLTE2NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SXNvbGF0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHRyYW5zcG9zb24taW5kdWNlZCBtdXRhbnRzIG9mIFBvcnBoeXJvbW9uYXMgZ2luZ2l2YWxpcyBkZWZpY2llbnQgaW4gZmltYnJpYXRpb247IE1pY3JvYi4gUGF0aG9nLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTI1LTM1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNlLWNhbGxpbmcgb2YgYXV0b21hdGVkIHNlcXVlbmNlciB0cmFjZXMgdXNpbmcgUGhyZWQuIElJLiBFcnJvciBwcm9iYWJpbGl0aWVzOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT04OyAgY2l0YXRpb25fcGFnZXM9MTg2LTE5NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXV0b21hdGVkIGZpbmlzaGluZyB3aXRoIGF1dG9maW5pc2g7IEdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NjE0LTYyNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZXJtaW5hdGlvbiBvZiB0aGUgZ2Vub21lIHNpemUgb2YgdGhlIG9yYWwgYW5hZXJvYmljIGJhY3Rlcml1bSBQb3JwaHlyb21vbmFzIGdpbmdpdmFsaXMgYnkgcHVsc2VkIGZpZWxkIGdlbCBlbGVjdHJvcGhvcmVzaXM7IERlbnQuIEpwbi47ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MzI7ICBjaXRhdGlvbl9wYWdlcz0yNS0yODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gYXV0b21hdGVkIHN5c3RlbSBmb3IgZ2Vub21lIGFuYWx5c2lzIHRvIHN1cHBvcnQgbWljcm9iaWFsIHdob2xlLWdlbm9tZSBzaG90Z3VuIHNlcXVlbmNpbmc7IEJpb3NjaS4gQmlvdGVjaG5vbC4gQmlvY2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NjQ7ICBjaXRhdGlvbl9wYWdlcz02NzAtNjczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DUklUSUNBOiBjb2RpbmcgcmVnaW9uIGlkZW50aWZpY2F0aW9uIHRvb2wgaW52b2tpbmcgY29tcGFyYXRpdmUgYW5hbHlzaXM7IE1vbC4gQmlvbC4gRXZvbC47ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTY7ICBjaXRhdGlvbl9wYWdlcz01MTItNTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb2RlbGluZyBhbmQgcHJlZGljdGluZyB0cmFuc2NyaXB0aW9uYWwgdW5pdHMgb2YgRXNjaGVyaWNoaWEgY29saSBnZW5lcyB1c2luZyBoaWRkZW4gTWFya292IG1vZGVsczsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTU7ICBjaXRhdGlvbl9wYWdlcz05ODctOTkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21iaW5pbmcgZGl2ZXJzZSBldmlkZW5jZSBmb3IgZ2VuZSByZWNvZ25pdGlvbiBpbiBjb21wbGV0ZWx5IHNlcXVlbmNlZCBiYWN0ZXJpYWwgZ2Vub21lczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk4OyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9Mjk0MS0yOTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HYXBwZWQgQkxBU1QgYW5kIFBTSS1CTEFTVDogYSBuZXcgZ2VuZXJhdGlvbiBvZiBwcm90ZWluIGRhdGFiYXNlIHNlYXJjaCBwcm9ncmFtczsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0xOTk3OyAgY2l0YXRpb25fdm9sdW1lPTI1OyAgY2l0YXRpb25fcGFnZXM9MzM4OS0zNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT10Uk5Bc2Nhbi1TRTogYSBwcm9ncmFtIGZvciBpbXByb3ZlZCBkZXRlY3Rpb24gb2YgdHJhbnNmZXIgUk5BIGdlbmVzIGluIGdlbm9taWMgc2VxdWVuY2U7IE51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTk1NS05NjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgYWxnb3JpdGhtcyBmb3IgbGFyZ2Utc2NhbGUgZ2Vub21lIGFsaWdubWVudCBhbmQgY29tcGFyaXNvbjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTMwOyAgY2l0YXRpb25fcGFnZXM9MjQ3OC0yNDgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QaXBNYWtlcuKAlGEgd2ViIHNlcnZlciBmb3IgYWxpZ25pbmcgdHdvIGdlbm9taWMgRE5BIHNlcXVlbmNlczsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz01NzctNTg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIG5ldyBtYXRoZW1hdGljYWwgbW9kZWwgZm9yIHJlbGF0aXZlIHF1YW50aWZpY2F0aW9uIGluIHJlYWwtdGltZSBSVOKAk1BDUjsgTnVjbGVpYyBBY2lkcyBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDAxOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljIGFuZCBzdHJ1Y3R1cmFsIGFuYWx5c2lzIG9mIHRoZSBCYWN0ZXJvaWRlcyBjb25qdWdhdGl2ZSB0cmFuc3Bvc29uIENUbjM0MTsgSi4gQmFjdGVyaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xODc7ICBjaXRhdGlvbl9wYWdlcz0yODU4LTI4Njk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zZmVyIHJlZ2lvbiBvZiBhIGJhY3Rlcm9pZGVzIGNvbmp1Z2F0aXZlIHRyYW5zcG9zb24sIENUbkRPVDsgUGxhc21pZDsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT00NTsgIGNpdGF0aW9uX3BhZ2VzPTQxLTUxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIGFuYWx5c2lzIG9mIEJhY3Rlcm9pZGVzIGZyYWdpbGlzIHJldmVhbHMgZXh0ZW5zaXZlIEROQSBpbnZlcnNpb25zIHJlZ3VsYXRpbmcgY2VsbCBzdXJmYWNlIGFkYXB0YXRpb247IFByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVVNBOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEwMTsgIGNpdGF0aW9uX3BhZ2VzPTE0OTE5LTE0OTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCB0cmFuc3Bvc2FibGUgZWxlbWVudHM6IHdoZXJlIGdlbmV0aWNzIG1lZXRzIGdlbm9taWNzOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0zMjktMzQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljIGxvY2kgb2YgdGhlIFBvcnBoeXJvbW9uYXMgZ2luZ2l2YWxpcyBpbnNlcnRpb24gZWxlbWVudCBJUzExMjY7IEluZmVjdC4gSW1tdW4uOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fdm9sdW1lPTY3OyAgY2l0YXRpb25fcGFnZXM9MzQxNi0zNDIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JZGVudGlmaWNhdGlvbiBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiB0aGUgY2Fwc3VsYXIgKEstYW50aWdlbikgbG9jdXMgb2YgUG9ycGh5cm9tb25hcyBnaW5naXZhbGlzOyBJbmZlY3QuIEltbXVuLjsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03NDsgIGNpdGF0aW9uX3BhZ2VzPTQ0OS00NjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBjbG9uaW5nIGFuZCBzZXF1ZW5jaW5nIG9mIHRoZSBmaW1icmlsaW4gZ2VuZSBvZiBQb3JwaHlyb21vbmFzIGdpbmdpdmFsaXMgc3RyYWlucyBhbmQgY2hhcmFjdGVyaXphdGlvbiBvZiByZWNvbWJpbmFudCBwcm90ZWluczsgQmlvY2hlbS4gQmlvcGh5cy4gUmVzLiBDb21tdW4uOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTE5NzsgIGNpdGF0aW9uX3BhZ2VzPTI0MS0yNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNlcXVlbmNlIGRpdmVyc2l0eSBhbmQgYW50aWdlbmljIHZhcmlhdGlvbiBhdCB0aGUgcmFnIGxvY3VzIG9mIFBvcnBoeXJvbW9uYXMgZ2luZ2l2YWxpczsgSW5mZWN0LiBJbW11bi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NzM7ICBjaXRhdGlvbl9wYWdlcz00MjUzLTQyNjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4dGVuc2l2ZSBETkEgaW52ZXJzaW9ucyBpbiB0aGUgQi4gZnJhZ2lsaXMgZ2Vub21lIGNvbnRyb2wgdmFyaWFibGUgZ2VuZSBleHByZXNzaW9uOyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTMwNzsgIGNpdGF0aW9uX3BhZ2VzPTE0NjMtMTQ2NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIG9mIFNoaWdlbGxhIGZsZXhuZXJpIDViIGFuZCBjb21wYXJpc29uIHdpdGggU2hpZ2VsbGEgZmxleG5lcmkgMmE7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTczIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGxldGUgZ2Vub21lIHNlcXVlbmNlIGFuZCBjb21wYXJhdGl2ZSBnZW5vbWljcyBvZiBTaGlnZWxsYSBmbGV4bmVyaSBzZXJvdHlwZSAyYSBzdHJhaW4gMjQ1N1Q7IEluZmVjdC4gSW1tdW4uOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTcxOyAgY2l0YXRpb25fcGFnZXM9Mjc3NS0yNzg2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Sb2xlIG9mIG94eVIgaW4gdGhlIG9yYWwgYW5hZXJvYmUgUG9ycGh5cm9tb25hcyBnaW5naXZhbGlzOyBKLiBCYWN0ZXJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE4ODsgIGNpdGF0aW9uX3BhZ2VzPTI0NTQtMjQ2MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzE1LzQvMjE1LzEwNzAxMDcvZHNuMDEzLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBUaGUgZ3JhbS1uZWdhdGl2ZSBhbmFlcm9iaWMgYmFjdGVyaXVtIFBvcnBoeXJvbW9uYXMgZ2luZ2l2YWxpcyBpcyBhIG1ham9yIGNhdXNhdGl2ZSBhZ2VudCBvZiBjaHJvbmljIHBlcmlvZG9udGl0aXMuIFBvcnBoeXJvbW9uYXMgZ2luZ2l2YWxpcyBzdHJhaW5zICIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXIvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9Mzc2NTA2JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZKp+leE29AcXwrofoICvC5TZdQgF" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f215%2f376506" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f215%2f376506" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=14644" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/15/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/15/4/1/m_cover.gif?Expires=2147483647&amp;Signature=RLDuHMUA6eE0ByVqiq8aUGyMxZAqCYh6CUPq-ezrEq~vX1AX9OwM0EdcZgw7bZO5GF08rcIkuMn~SdMlIqZ-OMoLaS-efZLrQFFAEDhygPUYmAsVLKwz1N0-SZE8tkL655rpVOyhNsmCq0HeMyZ1CRYm6AEF3IsWa0HPWbYG3ZsNpccsrKjShJ5nSPlVjprfmUgaJjeTeE3~oZCdhOYmMPqtkUKaLjbpdnpy8KGSYF3y2vs7JTB1QHc1IXmul6fvj2x9yPzU-FZHZaBu8~fDI-1ze4-bV3EV~DhG3VaQYIzUv05cvil7Rnpu6ewBIQhqRHLzHcljRy6MHn~J052jxA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/15/4">
                        <div class="volume trailing-comma">Volume 15</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2008        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="5484489">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484489">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5484491">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484491">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5484495">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484495">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5484507">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484507">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5484541">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484541">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5484543">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484543">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5484545">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484545">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="5484547">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#5484547">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/15/4/201/376260">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/15/4/227/375737">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Determination of the Genome Sequence of <em>Porphyromonas gingivalis</em> Strain ATCC 33277 and Genomic Comparison with Strain W83 Revealed Extensive Genome Rearrangements in <em>P. gingivalis</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mariko Naito</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunology</div>, <div class="insititution">Nagasaki University Graduate School of Biomedical Sciences</div>, <div class="addr-line">Sakamoto 1-7-1, Nagasaki 852-8588</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mariko+Naito">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Naito M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Naito M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Mariko Naito</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideki Hirakawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="insititution">Graduate School of Systems Life Sciences, Kyushu University</div>, <div class="addr-line">Hakozaki 6-10-1, Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideki+Hirakawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hirakawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hirakawa H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hideki Hirakawa</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Atsushi Yamashita</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Kitasato Institute for Life Sciences, Kitasato University</div>, <div class="addr-line">Kitasato 1-15-1, Sagamihara, Kanagawa 228-8555</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Atsushi+Yamashita">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yamashita A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yamashita A&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Atsushi Yamashita</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Naoya Ohara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunology</div>, <div class="insititution">Nagasaki University Graduate School of Biomedical Sciences</div>, <div class="addr-line">Sakamoto 1-7-1, Nagasaki 852-8588</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Naoya+Ohara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ohara N">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ohara N&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Naoya Ohara</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mikio Shoji</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunology</div>, <div class="insititution">Nagasaki University Graduate School of Biomedical Sciences</div>, <div class="addr-line">Sakamoto 1-7-1, Nagasaki 852-8588</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mikio+Shoji">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Shoji M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Shoji M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Mikio Shoji</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideharu Yukitake</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunology</div>, <div class="insititution">Nagasaki University Graduate School of Biomedical Sciences</div>, <div class="addr-line">Sakamoto 1-7-1, Nagasaki 852-8588</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideharu+Yukitake">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yukitake H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yukitake H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hideharu Yukitake</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Keisuke Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Division of Bioenvironmental Science, Frontier Science Research Center</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">Kihara 5200, Kiyotake, Miyazaki 889-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Keisuke+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama K&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Keisuke Nakayama</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hidehiro Toh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Kitasato Institute for Life Sciences, Kitasato University</div>, <div class="addr-line">Kitasato 1-15-1, Sagamihara, Kanagawa 228-8555</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hidehiro+Toh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Toh H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Toh H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Hidehiro Toh</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Fuminobu Yoshimura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Department of Microbiology</div>, <div class="insititution">School of Dentistry, Aichi-Gakuin University</div>, <div class="addr-line">1-100 Kusumoto-cho, Chikusa-ku, Nagoya 464-8650</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Fuminobu+Yoshimura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yoshimura F">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yoshimura F&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Fuminobu Yoshimura</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Satoru Kuhara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">6</div><div class="insititution">Graduate School of Genetic Resources Technology, Kyushu University</div>, <div class="addr-line">Hakozaki 6-10-1, Higashi-ku, Fukuoka 812-8581</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Satoru+Kuhara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kuhara S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kuhara S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">Satoru Kuhara</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masahira Hattori</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="insititution">Kitasato Institute for Life Sciences, Kitasato University</div>, <div class="addr-line">Kitasato 1-15-1, Sagamihara, Kanagawa 228-8555</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">7</div><div class="addr-line">Department of Computational Biology</div>, <div class="insititution">Graduate School of Frontier Sciences, The University of Tokyo</div>, <div class="addr-line">Kashiwanoha 5-1-5, Chiba 277-8561</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masahira+Hattori">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hattori M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hattori M&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Masahira Hattori</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuya Hayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Division of Bioenvironmental Science, Frontier Science Research Center</div>, <div class="insititution">University of Miyazaki</div>, <div class="addr-line">Kihara 5200, Kiyotake, Miyazaki 889-1692</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuya+Hayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hayashi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hayashi T&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Tetsuya Hayashi</a>
                                        </span>
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Koji Nakayama</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Division of Microbiology and Oral Infection, Department of Molecular Microbiology and Immunology</div>, <div class="insititution">Nagasaki University Graduate School of Biomedical Sciences</div>, <div class="addr-line">Sakamoto 1-7-1, Nagasaki 852-8588</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81 95-819-7649. Fax. <div class="fax">+81 95-819-7650</div>. E-mail: <a href="mailto:knak@nagasaki-u.ac.jp" target="_blank">knak@nagasaki-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Koji+Nakayama">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Nakayama K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Nakayama K&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">Koji Nakayama</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Katsumi Isono</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">Present address: Department of Immunology, National Institute of Infectious Diseases, 1-23-1 Toyama, Shinjuku-ku, Tokyo 162-8640, Japan</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="FN2"><div class="fn"><div class="label -label">‡</div><div class="footnote-content">Present address: RIKEN Genomic Sciences Center, 1-7-22 Suehiro-cho, Tsurumi-ku, Yokohama 230-0045, Japan</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 15, Issue 4, 1 August 2008, Pages 215–225, <a href='https://doi.org/10.1093/dnares/dsn013'>https://doi.org/10.1093/dnares/dsn013</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">03 June 2008</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">10 March 2008</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">13 May 2008</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="376506" href="/dnaresearch/article-pdf/15/4/215/1070107/dsn013.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Mariko Naito, Hideki Hirakawa, Atsushi Yamashita, Naoya Ohara, Mikio Shoji, Hideharu Yukitake, Keisuke Nakayama, Hidehiro Toh, Fuminobu Yoshimura, Satoru Kuhara, Masahira Hattori, Tetsuya Hayashi, Koji Nakayama;  Determination of the Genome Sequence of <em>Porphyromonas gingivalis</em> Strain ATCC 33277 and Genomic Comparison with Strain W83 Revealed Extensive Genome Rearrangements in <em>P. gingivalis</em>, <em>DNA Research</em>, Volume 15, Issue 4, 1 August 2008, Pages 215–225, <a href='https://doi.org/10.1093/dnares/dsn013'>https://doi.org/10.1093/dnares/dsn013</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/376506?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/376506?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/376506?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/376506?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/376506?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/376506?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/376506?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Determination%20of%20the%20Genome%20Sequence%20of%20%26lt%3Bem%26gt%3BPorphyromonas%20gingivalis%26lt%3B%2Fem%26gt%3B%20Strain%20ATCC%2033277%20and%20Genomic%20Comparison%20with%20Strain%20W83%20Revealed%20Extensive%20Genome%20Rearrangements%20in%20%26lt%3Bem%26gt%3BP.%20gingivalis%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2008-06-03&amp;volumeNum=15&amp;issueNum=4&amp;author=Naito%2C%20Mariko%3B%20Hirakawa%2C%20Hideki&amp;startPage=215&amp;endPage=225&amp;contentId=10.1093%2Fdnares%2Fdsn013&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f215%2f376506" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=5484489 id="5484489" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>The gram-negative anaerobic bacterium <em>Porphyromonas gingivalis</em> is a major causative agent of chronic periodontitis. <em>Porphyromonas gingivalis</em> strains have been classified into virulent and less-virulent strains by mouse subcutaneous soft tissue abscess model analysis. Here, we present the whole genome sequence of <em>P. gingivalis</em> ATCC 33277, which is classified as a less-virulent strain. We identified 2090 protein-coding sequences (CDSs), 4 RNA operons, and 53 tRNA genes in the ATCC 33277 genome. By genomic comparison with the virulent strain W83, we identified 461 ATCC 33277-specific and 415 W83-specific CDSs. Extensive genomic rearrangements were observed between the two strains: 175 regions in which genomic rearrangements have occurred were identified. Thirty-five of those genomic rearrangements were inversion or translocation and 140 were simple insertion, deletion, or replacement. Both strains contained large numbers of mobile elements, such as insertion sequences, miniature inverted-repeat transposable elements (MITEs), and conjugative transposons, which are frequently associated with genomic rearrangements. These findings indicate that the mobile genetic elements have been deeply involved in the extensive genome rearrangement of <em>P. gingivalis</em> and the occurrence of many of the strain-specific CDSs. We also describe here a very unique feature of MITE400, which we renamed MITE<em>PgRS</em> (MITE of <em><span class="underline">P</span>. <span class="underline">g</span>ingivalis</em> with <span class="underline">R</span>epeating <span class="underline">S</span>equences).</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Porphyromonas gingivalis&quot;"><em>Porphyromonas gingivalis</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;whole genome sequence&quot;">whole genome sequence</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;genome rearrangement&quot;">genome rearrangement</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;conjugative transposon&quot;">conjugative transposon</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="MITE">MITE</a></div>                            <h2 scrollto-destination=5484491 id="5484491" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Periodontal disease, the major cause of tooth loss in the general populations of industrial nations,<sup><a reveal-id="DSN013C1" data-open="DSN013C1" class="link link-ref link-reveal xref-bibr">1</a>,<a reveal-id="DSN013C2" data-open="DSN013C2" class="link link-ref link-reveal xref-bibr">2</a></sup> is a chronic inflammatory disease of the periodontium that leads to erosion of the attachment apparatus and supporting bone for teeth<sup><a reveal-id="DSN013C3" data-open="DSN013C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and is one of the most frequently occurring infectious diseases in humans.<sup><a reveal-id="DSN013C4" data-open="DSN013C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Recently, a number of epidemiological studies have shown significant relationships between periodontal diseases and cardiovascular diseases.<sup><a reveal-id="DSN013C5  DSN013C8" data-open="DSN013C5  DSN013C8" class="link link-ref link-reveal xref-bibr">5–8</a></sup> Several periodontal pathogens, including <em>Porphyromonas gingivalis</em>, have been found in atherosclerotic plaques.<sup><a reveal-id="DSN013C9" data-open="DSN013C9" class="link link-ref link-reveal xref-bibr">9</a>,<a reveal-id="DSN013C10" data-open="DSN013C10" class="link link-ref link-reveal xref-bibr">10</a></sup></p><p><em>Porphyromonas gingivalis</em> is a gram-negative anaerobic bacterium that is classified in the genus <em>Porphyromonas</em>, family <em>Porphyromonadaceae</em>, order <em>Bacteroidales</em>, class <em>Bacteroides</em>, phylum <em>Bacteroidetes</em>.<sup><a reveal-id="DSN013C11" data-open="DSN013C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The bacterium, which is often found in deep periodontal pockets of humans, is asaccharolytic and highly proteolytic. <em>Porphyromonas gingivalis</em> produces a broad array of potential virulence factors involved in tissue colonization and destruction as well as host defense perturbation. Potential virulence factors of <em>P. gingivalis</em> have been extensively described in several reviews.<sup><a reveal-id="DSN013C12  DSN013C14" data-open="DSN013C12  DSN013C14" class="link link-ref link-reveal xref-bibr">12–14</a></sup> Among these, fimbriae (FimA fimbriae and Mfa1 fimbriae), which are responsible for attachment of bacterial cells to host cell surfaces, and proteolytic enzymes such as Arg-gingipain (Rgp) and Lys-gingipain (Kgp), which degrade various host proteins, have been studied in detail.<sup><a reveal-id="DSN013C15  DSN013C17" data-open="DSN013C15  DSN013C17" class="link link-ref link-reveal xref-bibr">15–17</a></sup> However, no systematic analysis of <em>P. gingivalis</em> virulence factors has yet been carried out.</p><p>Using the mouse subcutaneous soft tissue abscess model, <em>P. gingivalis</em> strains are divided into virulent and less-virulent strains. Virulent strains such as W83 typically induce a necrotic lesion at the site of injection within 24 h. An extending gangrene-like necrosis spreads, secondarily along fascial planes into the abdominal and thoracic areas between 24 and 48 h, producing a foul-smelling and bloody exudate under the animal's skin. Sloughing and scab formation from necrosis of the epidermis, weight and hair loss, and animal death from microbial sepsis also are common clinical features. Less-virulent strains such as ATCC 33277 produce only a localized abscess 3 days after subcutaneous inoculation.<sup><a reveal-id="DSN013C18" data-open="DSN013C18" class="link link-ref link-reveal xref-bibr">18</a></sup> W83 has already been genome-sequenced.<sup><a reveal-id="DSN013C19" data-open="DSN013C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Strain ATCC 33277 is the type strain of <em>P. gingivalis</em> and has been widely used for characterization of pathophysiological features of the microorganism. We have used molecular genetic techniques to study <em>P. gingivalis</em> and have constructed a number of mutants from strain ATCC 33277 to investigate the roles of various genes in the pathogenicity of <em>P. gingivalis</em>.<sup><a reveal-id="DSN013C20  DSN013C25" data-open="DSN013C20  DSN013C25" class="link link-ref link-reveal xref-bibr">20–25</a></sup> In the present study, we determined the whole genome sequence of strain ATCC 33277 and performed a genomic comparison of ATCC 33277 and W83. Our findings showed that extensive genome rearrangements have taken place between the two strains. Transposable elements appear to have played central roles in the generation of these genome rearrangements, which have in turn created many strain-specific genes, including several potentially virulence-related genes.</p>                            <h2 scrollto-destination=5484495 id="5484495" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=5484496 id="5484496" class="section-title" data-legacy-id=s2a>2.1. Genome sequencing</h3>
<p><em>Porphyromonas gingivalis</em> ATCC 33277 was obtained from the American Type Culture Collection (ATCC) and has been kept for more than 20 years. <em>Porphyromonas gingivalis</em> W83 was obtained from Dr M. J. Duncan (Department of Molecular Genetics, The Forsyth Institute). <em>Porphyromonas gingivalis</em> GAI7802 was obtained from Dr E. Hoshino (Niigata University School of Dentistry), TDC60, TDC117, and TDC275 were obtained from Dr K. Ishihara (Tokyo Dental College), and SU63 was obtained from Dr M. Yoneda (Fukuoka Dental College). For preparing the genomic DNA, a single colony of each <em>P. gingivalis</em> strain was grown at 37°C anaerobically (10% CO<sub>2</sub>, 10% H<sub>2</sub>, 80% N<sub>2</sub>) in brain heart infusion broth (BD Bioscience, San Jose, CA, USA) supplemented with 5 µg of hemin and 0.5 µg of menadione per ml.</p><p>The genomic DNA was randomly sheared by Hydroshear (GeneMachines) and used for genomic library construction. We prepared two pTS1-based random genomic libraries with insert sizes of 1–2 kb or ∼10 kb. Sequencing was carried out using BigDye v3.1 chemistry on ABI 3700 or ABI 3730 sequencers (Applied Biosystems) or ET chemistry on MegaBACE 4500 sequencers (GE Healthcare). The whole genome sequence was obtained by assembling 36 394 reads (9.5-fold coverage) from both shotgun libraries. The Phred/Phrap software package<sup><a reveal-id="DSN013C26" data-open="DSN013C26" class="link link-ref link-reveal xref-bibr">26</a></sup> was used for base-calling, quality assessment, and sequence assembly. Assemblies were visualized for counting-based variations and detecting misassembly using Consed software.<sup><a reveal-id="DSN013C27" data-open="DSN013C27" class="link link-ref link-reveal xref-bibr">27</a></sup> Numbers and lengths of the <em>Not</em>I fragments of the ATCC 33277 chromosome predicted from the determined nucleotide sequence agreed well with those observed in pulsed field gel electrophoresis analysis of <em>Not</em>I-digested ATCC 33277 genomic DNA.<sup><a reveal-id="DSN013C28" data-open="DSN013C28" class="link link-ref link-reveal xref-bibr">28</a></sup></p>                            <h3 scrollto-destination=5484499 id="5484499" class="section-title" data-legacy-id=s2b>2.2. Sequence analysis</h3>
<p>Protein-coding sequences (CDSs) were identified by using the combination of GENOME GAMBLER v1.51,<sup><a reveal-id="DSN013C29" data-open="DSN013C29" class="link link-ref link-reveal xref-bibr">29</a></sup> CRITICA,<sup><a reveal-id="DSN013C30" data-open="DSN013C30" class="link link-ref link-reveal xref-bibr">30</a></sup> GENEHACKER,<sup><a reveal-id="DSN013C31" data-open="DSN013C31" class="link link-ref link-reveal xref-bibr">31</a></sup> and GLIMMER v2.0 programs.<sup><a reveal-id="DSN013C32" data-open="DSN013C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The sequences of 3′ terminal regions of all 16S ribosomal RNA genes in W83, and ATCC 33277 were identical to that of <em>Bacteroides fragilis</em> (AGAAAGGAGG, the accession number M61006). Each CDS was thus reviewed manually for the presence of a start codon (ATG, TTG, or GTG) and a potential ribosome-binding sequence that should be related to a part of the AGAAAGGAGG sequence. Functional annotation of the CDSs was made on the basis of results of homology searches against public protein (nr) database from NCBI (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/">http://www.ncbi.nlm.nih.gov/</a>) by the BLASTP program.<sup><a reveal-id="DSN013C33" data-open="DSN013C33" class="link link-ref link-reveal xref-bibr">33</a></sup> tRNA genes were identified by the tRNAscan-SE program.<sup><a reveal-id="DSN013C34" data-open="DSN013C34" class="link link-ref link-reveal xref-bibr">34</a></sup> All-to-all BLASTP analysis of CDSs was performed between W83 and ATCC 33277 to identify conserved and strain-specific CDSs. Since each genome contained a number of multi-copied CDSs such as transposase genes, we first grouped these multi-copied CDSs in each genome using BLASTCLUST<sup><a reveal-id="DSN013C33" data-open="DSN013C33" class="link link-ref link-reveal xref-bibr">33</a></sup> (The threshold used was ≥90% amino-acid sequence identity.). The largest CDS in each multi-copied CDS group was used as the representative of each group for the identification of conserved and strain-specific CDSs. In the present study, we defined conserved CDSs as ones that had 60–140% of the length of a query sequence and showed ≥90% sequence identity in bidirectional best-hit analysis. The MUMmer program<sup><a reveal-id="DSN013C35" data-open="DSN013C35" class="link link-ref link-reveal xref-bibr">35</a></sup> was used to define conserved genomic regions, inversions, and translocations between the two genomes. The Pip Maker program<sup><a reveal-id="DSN013C36" data-open="DSN013C36" class="link link-ref link-reveal xref-bibr">36</a></sup> was also used for DNA sequence alignment.</p>                            <h3 scrollto-destination=5484501 id="5484501" class="section-title" data-legacy-id=s2c>2.3. PCR analysis</h3>
<p>The total genomic DNA of ATCC 33277 and two pairs of PCR primers (CTnPg1-left and CTnPg1-right, CTnPg1-up and CTnPg1-down, <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>) were used to detect the excision of a conjugative transposon (CTn), CTnPg1. These primer pairs were designed to amplify the <em>attP</em> and <em>attB</em> regions for CTnPg1, respectively. PCR amplification was performed by using 100 ng of the genomic DNA and <em>LA Taq</em> (Takara Shuzo, Tokyo, Japan) in the following setting: preheating (94°C for 1 min) and 30 cycles of DNA denaturation (94°C for 20 s), primer annealing (55°C for 30 s) and DNA extension (68°C for 2 min). The amplified fragments were subjected to direct sequencing to determine the nucleotide sequences of the <em>attP</em> and <em>attB</em> sites.</p>                            <h3 scrollto-destination=5484503 id="5484503" class="section-title" data-legacy-id=s2d>2.4. RNA isolation and real-time PCR</h3>
<p><em>Porphyromonas gingivalis</em> cells were grown to the mid-exponential phase. Total RNA was isolated from the harvested cells using an RNeasy Mini Kit (Qiagen Sciences, Valencia, CA, USA) and reverse-transcribed in a reaction mixture containing a random primer (Promega Co., Madison, WI, USA), dNTP mixture, RNase inhibitor (Wako Pure Chemical Industries, Ltd., Osaka, Japan), dithiothreitol, Superscript II (Invitrogen, Carlsbad, CA, USA), and DEPC-treated water. Real-time quantitative PCR was performed using Full Velocity SYBR Green QPCR Master Mix (Stratagene, La Jolla, CA, USA) according to the manufacturer's instructions. Primer sequences for the real-time PCR are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>. PCR amplification was performed in the following setting: preheating (95°C for 5 min) and 30 cycles of DNA denaturation (95°C for 10 s), and primer annealing/DNA extension (60°C for 30 s). The expression level of each targeted gene was normalized to that of the 16S rRNA gene. The comparative cycle threshold method<sup><a reveal-id="DSN013C37" data-open="DSN013C37" class="link link-ref link-reveal xref-bibr">37</a></sup> was used for relative quantification.</p>                            <h3 scrollto-destination=5484505 id="5484505" class="section-title" data-legacy-id=s2e>2.5. Nucleotide sequence accession numbers</h3>
<p>The fully annotated genome sequence of <em>P. gingivalis</em> strain ATCC 33277 has been deposited in GenBank/EMBL/DDBJ databases under the accession number AP009380. Nucleotide sequences of the glucose kinase-encoding genes (<em>glk</em>) of five <em>P. gingivalis</em> strains have been deposited under the accession numbers AB293447 (strain TDC60), AB293448 (TDC117), AB293449 (TDC275), AB293450 (SU63), and AB293451 (GAI7802).</p>                            <h2 scrollto-destination=5484507 id="5484507" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=5484508 id="5484508" class="section-title" data-legacy-id=s3a>3.1. General features of the ATCC 33277 genome</h3>
<p>The genome of ATCC 33277 comprised a single circular chromosome of 2 354 886 bp with an average G + C content of 48.4% (Fig. <a reveal-id="DSN013F1" data-open="DSN013F1" class="link link-reveal link-table xref-fig">1</a>). The size was almost the same as that of W83 (2 343 476 bp). The ATCC 33277 genome contained 2090 CDSs (PGN No.) with an average size of 970 bp, covering 86.1% of the whole chromosome sequence. It contained 4 RNA operons (<em>rrn</em>, 5S rRNA-23S rRNA-tRNA<sup>Ala</sup>-tRNA<sup>Ile</sup>-16S rRNA) and 53 tRNA genes that provide specificity for all kinds of amino acids. The numbers of <em>rrn</em> operons and tRNA genes were identical to those of W83. By the <em>χ</em><sup><a reveal-id="DSN013C2" data-open="DSN013C2" class="link link-ref link-reveal xref-bibr">2</a></sup> analysis, we identified 13 regions with atypical nucleotide composition on the ATCC 33277 chromosome (Fig. <a reveal-id="DSN013F1" data-open="DSN013F1" class="link link-reveal link-table xref-fig">1</a>, 8th circle). Many genes in the regions exhibited higher similarity to the genes in other bacterial species such as <em>B. fragilis</em> than those in strain W83, suggesting that they have been introduced to ATCC 33277 by horizontal gene transfer.</p>                        <a id="5484510" scrollto-destination="5484510"></a>
<div id="DSN013F1" data-id="DSN013F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/m_dsn01301.jpeg?Expires=2147483647&Signature=AwR~cBzcAmVeS0vhOiDG0cXdd8k8TxMbmataiBLZVuN99kxccJdk~jagMidGIrNpT2zuEr20HViv9uLRppIWth3JSOUd6WnZJC-GIiMTB8Kxq6ru4RyOwl72tFbPPqaEN9NC3w-sq70jKYJXqWD6684AuLIOJn4CGw0GjoETfzdWKRuEoISRi3~MKq9KBJrrkKru7wPBESy-PSnq0jYjVEj94C5rEvgu8-sf0~Y1HS22fB6Qk4PklWbqKj89GXbtae39WT7d3ezjI0YbVzTkeESbku6fvKJbBmROvH8pbpxrxAfeYfY5qY2c7Di8Ax8aMNhnh7C~-kdUeuOk9bfY7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circular map of the chromosome of P. gingivalis strain ATCC 33277. From the outside, the first and second circles show CDSs on the plus and minus strands, respectively. CDSs conserved in strains ATCC 33277 and W83 are indicated in red and ATCC 33277-specific CDSs in blue. The 3rd to 5th circles show IS elements (orange, ISPg1; light green, ISPg2; magenta, ISPg3; cyan, ISPg4; brown, ISPg5; blue, ISPg6), MITEs (magenta, MITE239; black, MITEPgRS; cyan, MITE700), CTns, and Tns (blue, CTnPg1-a, CTnPg1-b, CTnPg2, and CTnPg3; red, TnPg17), respectively. The 6th and 7th circles show rrn operons and tRNA genes, respectively. The 8th circle shows the result of χ2 analysis of nucleotide composition. Regions exhibiting values of &gt;600 are indicated in red and those of &lt;600 are indicated in blue. The G + C skew and G + C content are shown in the 9th and 10th circles, respectively." path-from-xml="dsn01301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484510/dsn01301.jpeg" path-from-xml="dsn01301.jpeg" target="_blank">View large</a><a section="5484510" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/dsn01301.jpeg?Expires=2147483647&Signature=Np~41Ogqu~tnJbdvegYLrsMl2bUDMTzve37dPjV5LZytSMJYVcRYdymtP5XiybhSO66xjLL1-k6OqKELY-6639bI9A2cy6u6Jy2fEMJGhn6aUhb9FsSY5mbX~4102ddFZetzSAr9IOgRsjWSFdCS9ILZaHqsPPpF2bRG9bq5Xk040sgLssIw2TF5DEvVp26gMesYmRfQ87VkIFtpqvJlcp2kaVHXnJ3-BrJVyXwphQQ5VG0GQORHyJE8kfy9trjbSuLUHILowDlU80WFCeoFe9O8LMgQUjp9hGC43C2UQObixpgSxlUzvKeS8OVHFhn9ctJM1dL2Nf9dC--oGvYfAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484510&ar=376506&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circular map of the chromosome of <em>P. gingivalis</em> strain ATCC 33277. From the outside, the first and second circles show CDSs on the plus and minus strands, respectively. CDSs conserved in strains ATCC 33277 and W83 are indicated in red and ATCC 33277-specific CDSs in blue. The 3rd to 5th circles show IS elements (orange, IS<em>Pg1</em>; light green, IS<em>Pg2</em>; magenta, IS<em>Pg3</em>; cyan, IS<em>Pg4</em>; brown, IS<em>Pg5</em>; blue, IS<em>Pg6</em>), MITEs (magenta, MITE239; black, MITE<em>PgRS</em>; cyan, MITE700), CTns, and Tns (blue, CTnPg1-a, CTnPg1-b, CTnPg2, and CTnPg3; red, Tn<em>Pg17</em>), respectively. The 6th and 7th circles show <em>rrn</em> operons and tRNA genes, respectively. The 8th circle shows the result of <em>χ</em><sup>2</sup> analysis of nucleotide composition. Regions exhibiting values of &gt;600 are indicated in red and those of &lt;600 are indicated in blue. The G + C skew and G + C content are shown in the 9th and 10th circles, respectively.</p></div></div><div content-id="DSN013F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/m_dsn01301.jpeg?Expires=2147483647&Signature=AwR~cBzcAmVeS0vhOiDG0cXdd8k8TxMbmataiBLZVuN99kxccJdk~jagMidGIrNpT2zuEr20HViv9uLRppIWth3JSOUd6WnZJC-GIiMTB8Kxq6ru4RyOwl72tFbPPqaEN9NC3w-sq70jKYJXqWD6684AuLIOJn4CGw0GjoETfzdWKRuEoISRi3~MKq9KBJrrkKru7wPBESy-PSnq0jYjVEj94C5rEvgu8-sf0~Y1HS22fB6Qk4PklWbqKj89GXbtae39WT7d3ezjI0YbVzTkeESbku6fvKJbBmROvH8pbpxrxAfeYfY5qY2c7Di8Ax8aMNhnh7C~-kdUeuOk9bfY7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Circular map of the chromosome of P. gingivalis strain ATCC 33277. From the outside, the first and second circles show CDSs on the plus and minus strands, respectively. CDSs conserved in strains ATCC 33277 and W83 are indicated in red and ATCC 33277-specific CDSs in blue. The 3rd to 5th circles show IS elements (orange, ISPg1; light green, ISPg2; magenta, ISPg3; cyan, ISPg4; brown, ISPg5; blue, ISPg6), MITEs (magenta, MITE239; black, MITEPgRS; cyan, MITE700), CTns, and Tns (blue, CTnPg1-a, CTnPg1-b, CTnPg2, and CTnPg3; red, TnPg17), respectively. The 6th and 7th circles show rrn operons and tRNA genes, respectively. The 8th circle shows the result of χ2 analysis of nucleotide composition. Regions exhibiting values of &gt;600 are indicated in red and those of &lt;600 are indicated in blue. The G + C skew and G + C content are shown in the 9th and 10th circles, respectively." path-from-xml="dsn01301.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484510/dsn01301.jpeg" path-from-xml="dsn01301.jpeg" target="_blank">View large</a><a section="5484510" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/dsn01301.jpeg?Expires=2147483647&Signature=Np~41Ogqu~tnJbdvegYLrsMl2bUDMTzve37dPjV5LZytSMJYVcRYdymtP5XiybhSO66xjLL1-k6OqKELY-6639bI9A2cy6u6Jy2fEMJGhn6aUhb9FsSY5mbX~4102ddFZetzSAr9IOgRsjWSFdCS9ILZaHqsPPpF2bRG9bq5Xk040sgLssIw2TF5DEvVp26gMesYmRfQ87VkIFtpqvJlcp2kaVHXnJ3-BrJVyXwphQQ5VG0GQORHyJE8kfy9trjbSuLUHILowDlU80WFCeoFe9O8LMgQUjp9hGC43C2UQObixpgSxlUzvKeS8OVHFhn9ctJM1dL2Nf9dC--oGvYfAw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484510&ar=376506&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01301.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Circular map of the chromosome of <em>P. gingivalis</em> strain ATCC 33277. From the outside, the first and second circles show CDSs on the plus and minus strands, respectively. CDSs conserved in strains ATCC 33277 and W83 are indicated in red and ATCC 33277-specific CDSs in blue. The 3rd to 5th circles show IS elements (orange, IS<em>Pg1</em>; light green, IS<em>Pg2</em>; magenta, IS<em>Pg3</em>; cyan, IS<em>Pg4</em>; brown, IS<em>Pg5</em>; blue, IS<em>Pg6</em>), MITEs (magenta, MITE239; black, MITE<em>PgRS</em>; cyan, MITE700), CTns, and Tns (blue, CTnPg1-a, CTnPg1-b, CTnPg2, and CTnPg3; red, Tn<em>Pg17</em>), respectively. The 6th and 7th circles show <em>rrn</em> operons and tRNA genes, respectively. The 8th circle shows the result of <em>χ</em><sup>2</sup> analysis of nucleotide composition. Regions exhibiting values of &gt;600 are indicated in red and those of &lt;600 are indicated in blue. The G + C skew and G + C content are shown in the 9th and 10th circles, respectively.</p></div></div>                            <h3 scrollto-destination=5484511 id="5484511" class="section-title" data-legacy-id=s3b>3.2. Strain-specific CDSs</h3>
<p>To more precisely compare the CDS sets encoded on the ATCC 33277 and W83 genomes, we reannotated CDSs on the W83 genome by the same criteria as those used for ATCC 33277. We detected 114 CDSs (PGa No.) in the W83 genome that had not been annotated by Nelson et al.<sup><a reveal-id="DSN013C19" data-open="DSN013C19" class="link link-ref link-reveal xref-bibr">19</a></sup> They included many fragments of transposases of insertion sequences (ISs) but at least 27 function-assignable genes, such as those for translocase SecE subunit, pseudouridine synthases, and lysine-specific cysteine proteinase (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). In total, we identified 2023 CDSs in W83.</p><p>ATCC 33277 and W83 genomes had a number of multi-copied CDSs such as those for IS transposases. ATCC 33277 and W83 contained 53 and 32 multi-copied CDS groups, respectively; the gene products of each group exhibited ≥90% amino-acid sequence identity (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S3 and S4</a>). In bidirectional best-hit analysis to identify conserved CDSs between the two strains, we used the largest CDS in each multi-copied CDS group as the representative of each group. By the analysis, we identified 1490 conserved CDSs, 461 ATCC 33277-specific CDSs, and 415 W83-specific CDSs. The strain-specific CDSs are listed in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Tables S5 and S6</a>, respectively.</p><p>Most of the strain-specific CDSs encoded hypothetical proteins of unknown functions, but function-predictable W83-specific CDSs included several genes that may be related to the higher virulence of the strain, such as those for glycosyltransferase (PG0110), a protein required for capsular polysaccharide biosynthesis (PG0111), sensor histidine kinase (PG0719), surface antigen PgaA (PG0742), and thiol protease (PG1055). The two strains encoded different sets of DNA restriction-modification system proteins.</p>                            <h3 scrollto-destination=5484515 id="5484515" class="section-title" data-legacy-id=s3c>3.3. CTn and transposon</h3>
<p>ATCC 33277 contained a variety of mobile genetic elements. We found four copies of CTns that were absent in W83 (Fig. <a reveal-id="DSN013F2" data-open="DSN013F2" class="link link-reveal link-table xref-fig">2</a>). CTnPg1-a is 44.3 kb in size and encodes 50 CDSs (PGN_0046 to PGN_0095), including a set of genes for conjugative transfer and integration as well as those for an Na<sup>+</sup>-driven multi-drug efflux pump. Several genes showed moderate sequence homologies to the genes of CTns of <em>Bacteroides</em> species.<sup><a reveal-id="DSN013C38  DSN013C40" data-open="DSN013C38  DSN013C40" class="link link-ref link-reveal xref-bibr">38–40</a></sup> CTnPg1-b, which is 9.7 kb in size and encodes 15 CDSs (PGN_1281 to PGN_1295), is identical to a part of CTnPg1-a (PGN_0046 to PGN_0060). One end of CTnPg1-b has been disrupted by multiple IS insertions. We identified two additional CTns, CTnPg2 and CTnPg3, but both were also truncated and highly degraded by multiple IS insertions.</p>                        <a id="5484517" scrollto-destination="5484517"></a>
<div id="DSN013F2" data-id="DSN013F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/m_dsn01302.jpeg?Expires=2147483647&Signature=if2L-wcfzHszGhcwra5KQdmCva5lb4kQ8-tmgmL0oOMAnZjAj02D7-jB6F4rhImqJKpU43N68jSIK9uN561zAgOCVF-7KotOrHb746D-xutBbfKgfoyfQ8cj~sD7Bb0WaxLNKMlKzuilvLC3xn1zK2ETuCxLokib-HVL6EmZYl0rml-d6BdDGqh51eGNxnTbfa8umuf7c87RvELjatsVB03yTUZVKxRac5jUsJZdUHLTQ~j24MvajfV7UFLFBrka~C2UnDLWrdE~R1dhr0c65mDV3gRzMElxiO6h~kP~6J5f8r9weIANoiUTVJxAFcoemnbMtfDT6nFKE6eLUclk7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Novel CTns and Tn identified in the ATCC 33277 genome. (A) Structures of CTnPg1-a, CTnPg1-b, CTnPg2, and CTnPg3. (B) Structure of TnPg17. CDSs are depicted by arrows and IS elements by open boxes (vertically striped arrow, tra or mob genes; thin arrow in box, IS transposase; black arrow, partial transposase; hatched arrow, other functionally annotated CDS; white arrow, hypothetical protein). Black triangles in CTnPg1-a and CTnPg1-b indicate direct repeat sequences, and black boxes in TnPg17 MITEs. The regions of CTnPg1-a and CTnPg1-b indicated by gray shading have an identical sequence." path-from-xml="dsn01302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484517/dsn01302.jpeg" path-from-xml="dsn01302.jpeg" target="_blank">View large</a><a section="5484517" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/dsn01302.jpeg?Expires=2147483647&Signature=oDpPg2zOf7-61Ql8zPwP8xT8ZsZVgVDg0mmMZ7yl-4SUahdkHZUSFMW8BOua8BM0Igg5ErnOgOm92bgdCzUNjWBZXtS5DpHYIC3iVIMOlswfnRidfPQmW42ALz5Cnzol5~4mWqkeFzk3U3m0qbrxMk5QOEPp~Jit~SQc52rimN6oxhytq4PnYx3kcGFnTfzFPAv5tqG9pkP6MSEsfMn8ICfEvot6M4fr0J~NkM~qtYc0edWHpXKHHf5dG87pXdsU5Ws~95ky3nosMDQNGbBmpflA~GBovXdniI2dBsH~EdHmJZwvlqQOnpL6OGCnmNFJc-E2T-62IYZWscuw~xzWDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484517&ar=376506&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Novel CTns and Tn identified in the ATCC 33277 genome. (<strong>A</strong>) Structures of CTnPg1-a, CTnPg1-b, CTnPg2, and CTnPg3. (<strong>B</strong>) Structure of Tn<em>Pg17</em>. CDSs are depicted by arrows and IS elements by open boxes (vertically striped arrow, <em>tra</em> or <em>mob</em> genes; thin arrow in box, IS transposase; black arrow, partial transposase; hatched arrow, other functionally annotated CDS; white arrow, hypothetical protein). Black triangles in CTnPg1-a and CTnPg1-b indicate direct repeat sequences, and black boxes in Tn<em>Pg17</em> MITEs. The regions of CTnPg1-a and CTnPg1-b indicated by gray shading have an identical sequence.</p></div></div><div content-id="DSN013F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/m_dsn01302.jpeg?Expires=2147483647&Signature=if2L-wcfzHszGhcwra5KQdmCva5lb4kQ8-tmgmL0oOMAnZjAj02D7-jB6F4rhImqJKpU43N68jSIK9uN561zAgOCVF-7KotOrHb746D-xutBbfKgfoyfQ8cj~sD7Bb0WaxLNKMlKzuilvLC3xn1zK2ETuCxLokib-HVL6EmZYl0rml-d6BdDGqh51eGNxnTbfa8umuf7c87RvELjatsVB03yTUZVKxRac5jUsJZdUHLTQ~j24MvajfV7UFLFBrka~C2UnDLWrdE~R1dhr0c65mDV3gRzMElxiO6h~kP~6J5f8r9weIANoiUTVJxAFcoemnbMtfDT6nFKE6eLUclk7Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Novel CTns and Tn identified in the ATCC 33277 genome. (A) Structures of CTnPg1-a, CTnPg1-b, CTnPg2, and CTnPg3. (B) Structure of TnPg17. CDSs are depicted by arrows and IS elements by open boxes (vertically striped arrow, tra or mob genes; thin arrow in box, IS transposase; black arrow, partial transposase; hatched arrow, other functionally annotated CDS; white arrow, hypothetical protein). Black triangles in CTnPg1-a and CTnPg1-b indicate direct repeat sequences, and black boxes in TnPg17 MITEs. The regions of CTnPg1-a and CTnPg1-b indicated by gray shading have an identical sequence." path-from-xml="dsn01302.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484517/dsn01302.jpeg" path-from-xml="dsn01302.jpeg" target="_blank">View large</a><a section="5484517" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/dsn01302.jpeg?Expires=2147483647&Signature=oDpPg2zOf7-61Ql8zPwP8xT8ZsZVgVDg0mmMZ7yl-4SUahdkHZUSFMW8BOua8BM0Igg5ErnOgOm92bgdCzUNjWBZXtS5DpHYIC3iVIMOlswfnRidfPQmW42ALz5Cnzol5~4mWqkeFzk3U3m0qbrxMk5QOEPp~Jit~SQc52rimN6oxhytq4PnYx3kcGFnTfzFPAv5tqG9pkP6MSEsfMn8ICfEvot6M4fr0J~NkM~qtYc0edWHpXKHHf5dG87pXdsU5Ws~95ky3nosMDQNGbBmpflA~GBovXdniI2dBsH~EdHmJZwvlqQOnpL6OGCnmNFJc-E2T-62IYZWscuw~xzWDQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484517&ar=376506&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01302.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Novel CTns and Tn identified in the ATCC 33277 genome. (<strong>A</strong>) Structures of CTnPg1-a, CTnPg1-b, CTnPg2, and CTnPg3. (<strong>B</strong>) Structure of Tn<em>Pg17</em>. CDSs are depicted by arrows and IS elements by open boxes (vertically striped arrow, <em>tra</em> or <em>mob</em> genes; thin arrow in box, IS transposase; black arrow, partial transposase; hatched arrow, other functionally annotated CDS; white arrow, hypothetical protein). Black triangles in CTnPg1-a and CTnPg1-b indicate direct repeat sequences, and black boxes in Tn<em>Pg17</em> MITEs. The regions of CTnPg1-a and CTnPg1-b indicated by gray shading have an identical sequence.</p></div></div><p>We identified two identical copies of a novel composite transposon (Tn) named Tn<em>Pg17</em>-a and Tn<em>Pg17</em>-b in the ATCC 33277 genome. Tn<em>Pg17</em> is 16.8 kb in size and has IS<em>Pg3</em> at both ends. Target site duplications of 4 and 7 bp were found for Tn<em>Pg17</em>-a and Tn<em>Pg17</em>-b, respectively. Tn<em>Pg17</em> carries genes for a <em>tetR</em> family transcriptional regulator, ABC transporter ATP-binding proteins, and a carboxyl-terminal processing protease.</p>                            <h3 scrollto-destination=5484519 id="5484519" class="section-title" data-legacy-id=s3d>3.4. IS and miniature inverted-repeat transposable element</h3>
<p>A total of 93 IS elements (including 38 partial copies) and 48 miniature inverted-repeat transposable elements (MITEs) (including 18 partial copies) were found in ATCC 33277 (Table <a reveal-id="DSN013TB1" data-open="DSN013TB1" class="link link-reveal link-table xref-fig">1</a>). The IS elements identified were classified into six types, IS<em>Pg1</em>–IS<em>Pg6</em>, all of which are also present in W83.<sup><a reveal-id="DSN013C19" data-open="DSN013C19" class="link link-ref link-reveal xref-bibr">19</a></sup> MITEs comprise a group of small mobile genetic elements and are massively amplified often in plants.<sup><a reveal-id="DSN013C41" data-open="DSN013C41" class="link link-ref link-reveal xref-bibr">41</a></sup> They do not encode transposases by themselves but have terminal inverted repeats (TIRs) that are the same as or very similar to those of some IS elements, and they are thus transposable by the action of transposase provided <em>in trans</em> by the cognate IS element. In ATCC 33277, we identified three types of MITEs, MITE239, MITE700, and MITE464, all of which have also been identified on the W83 genome.<sup><a reveal-id="DSN013C19" data-open="DSN013C19" class="link link-ref link-reveal xref-bibr">19</a></sup> The structure of MITE239 was well conserved between the copies, 239 bp in length and with the same TIR as that of IS<em>Pg3</em>. However, MITE700 and MITE464 exhibited highly variable structures, and their structural features have not been described in detail in the previous report.<sup><a reveal-id="DSN013C19" data-open="DSN013C19" class="link link-ref link-reveal xref-bibr">19</a></sup> Therefore, we first determined the structural features of these two MITEs by comparing the sequences of all of the MITE700 and MITE464 copies identified in ATCC 33277.</p>                        <a id="5484521" scrollto-destination="5484521"></a>
<div content-id="DSN013TB1" class="table-modal"><div class="table-wrap"><div id="DSN013TB1" class="table-wrap-title" data-id="DSN013TB1"><div class="label -label">Table 1</div><div class="caption"><p>IS elements and MITEs on the ATCC 33277 and W83 genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Mobile genetic element </th><th colspan="2">Strain<hr /> </th></tr><tr><th>ATCC 33277<br />Number of intact copies<br />(number of fragments) </th><th>W83<sup>a</sup><br />Number of intact copies<br />(number of fragments) </th></tr></thead><tbody><tr><td>IS<em>Pg1</em> </td><td>31 (18) </td><td>11 (40) </td></tr><tr><td>IS<em>Pg2</em> </td><td>2 (4) </td><td>5 (3) </td></tr><tr><td>IS<em>Pg3</em> </td><td>22 (7) </td><td>4 (5) </td></tr><tr><td>IS<em>Pg4</em> </td><td>0 (2) </td><td>10 (0) </td></tr><tr><td>IS<em>Pg5</em> </td><td>0 (4) </td><td>10 (1) </td></tr><tr><td>IS<em>Pg6</em> </td><td>0 (3) </td><td>1 (2) </td></tr><tr><td>IS<em>Pg7</em> </td><td>0 (0) </td><td>1 (0) </td></tr><tr><td>MITE239 </td><td>12 (2) </td><td>5 (0) </td></tr><tr><td>MITE<em>PgRS</em> </td><td>11 (9) </td><td>14 (7) </td></tr><tr><td>MITE700 </td><td>7 (7) </td><td>7 (2) </td></tr></tbody></table></div><div content-id="DSN013TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Mobile genetic element </th><th colspan="2">Strain<hr /> </th></tr><tr><th>ATCC 33277<br />Number of intact copies<br />(number of fragments) </th><th>W83<sup>a</sup><br />Number of intact copies<br />(number of fragments) </th></tr></thead><tbody><tr><td>IS<em>Pg1</em> </td><td>31 (18) </td><td>11 (40) </td></tr><tr><td>IS<em>Pg2</em> </td><td>2 (4) </td><td>5 (3) </td></tr><tr><td>IS<em>Pg3</em> </td><td>22 (7) </td><td>4 (5) </td></tr><tr><td>IS<em>Pg4</em> </td><td>0 (2) </td><td>10 (0) </td></tr><tr><td>IS<em>Pg5</em> </td><td>0 (4) </td><td>10 (1) </td></tr><tr><td>IS<em>Pg6</em> </td><td>0 (3) </td><td>1 (2) </td></tr><tr><td>IS<em>Pg7</em> </td><td>0 (0) </td><td>1 (0) </td></tr><tr><td>MITE239 </td><td>12 (2) </td><td>5 (0) </td></tr><tr><td>MITE<em>PgRS</em> </td><td>11 (9) </td><td>14 (7) </td></tr><tr><td>MITE700 </td><td>7 (7) </td><td>7 (2) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Determined based on the reannotated W83 data.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5484521" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSN013TB1" class="table-wrap-title" data-id="DSN013TB1"><div class="label -label">Table 1</div><div class="caption"><p>IS elements and MITEs on the ATCC 33277 and W83 genomes</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Mobile genetic element </th><th colspan="2">Strain<hr /> </th></tr><tr><th>ATCC 33277<br />Number of intact copies<br />(number of fragments) </th><th>W83<sup>a</sup><br />Number of intact copies<br />(number of fragments) </th></tr></thead><tbody><tr><td>IS<em>Pg1</em> </td><td>31 (18) </td><td>11 (40) </td></tr><tr><td>IS<em>Pg2</em> </td><td>2 (4) </td><td>5 (3) </td></tr><tr><td>IS<em>Pg3</em> </td><td>22 (7) </td><td>4 (5) </td></tr><tr><td>IS<em>Pg4</em> </td><td>0 (2) </td><td>10 (0) </td></tr><tr><td>IS<em>Pg5</em> </td><td>0 (4) </td><td>10 (1) </td></tr><tr><td>IS<em>Pg6</em> </td><td>0 (3) </td><td>1 (2) </td></tr><tr><td>IS<em>Pg7</em> </td><td>0 (0) </td><td>1 (0) </td></tr><tr><td>MITE239 </td><td>12 (2) </td><td>5 (0) </td></tr><tr><td>MITE<em>PgRS</em> </td><td>11 (9) </td><td>14 (7) </td></tr><tr><td>MITE700 </td><td>7 (7) </td><td>7 (2) </td></tr></tbody></table></div><div content-id="DSN013TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Mobile genetic element </th><th colspan="2">Strain<hr /> </th></tr><tr><th>ATCC 33277<br />Number of intact copies<br />(number of fragments) </th><th>W83<sup>a</sup><br />Number of intact copies<br />(number of fragments) </th></tr></thead><tbody><tr><td>IS<em>Pg1</em> </td><td>31 (18) </td><td>11 (40) </td></tr><tr><td>IS<em>Pg2</em> </td><td>2 (4) </td><td>5 (3) </td></tr><tr><td>IS<em>Pg3</em> </td><td>22 (7) </td><td>4 (5) </td></tr><tr><td>IS<em>Pg4</em> </td><td>0 (2) </td><td>10 (0) </td></tr><tr><td>IS<em>Pg5</em> </td><td>0 (4) </td><td>10 (1) </td></tr><tr><td>IS<em>Pg6</em> </td><td>0 (3) </td><td>1 (2) </td></tr><tr><td>IS<em>Pg7</em> </td><td>0 (0) </td><td>1 (0) </td></tr><tr><td>MITE239 </td><td>12 (2) </td><td>5 (0) </td></tr><tr><td>MITE<em>PgRS</em> </td><td>11 (9) </td><td>14 (7) </td></tr><tr><td>MITE700 </td><td>7 (7) </td><td>7 (2) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><sup>a</sup>Determined based on the reannotated W83 data.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/5484521" target="_blank">View Large</a></div></div><p>MITE700 also contains the same TIRs as those of IS<em>Pg3</em>, but its internal sequence is not related to that of MITE239. We identified 14 copies of MITE700 (including seven partial copies) in ATCC 33277. Multiple sequence alignment analysis revealed that MITE700 is ∼720 bp in length but exhibits a high sequence variation due to internal deletions/insertions. The sequence of a region located just downstream of the left TIR is highly variable between the copies. We identified three subtypes of MITE700 based on the sequence variation in this region (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>).</p><p>MITE464 has the same TIRs as those of IS<em>Pg1.</em> We found that its internal sequence consists of three types of repeat sequences (referred to as Repeats A, B, and C) (Fig. <a reveal-id="DSN013F3" data-open="DSN013F3" class="link link-reveal link-table xref-fig">3</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>). Repeats A and B exist just inside both TIRs, constituting a direct-repeat-like structure. Between the two Repeat A/B regions, multiple Repeat C regions exist in tandem, but the number of Repeat C regions varies between MITE464 copies, up to 14 repeats. Since no such repeating structure has been found in any MITEs described so far, we propose a new name for this MITE, MITE<em>PgRS</em> (MITE of <em><span class="underline">P</span>. <span class="underline">g</span>ingivalis</em> with <span class="underline">R</span>epeating <span class="underline">S</span>equences).</p>                        <a id="5484524" scrollto-destination="5484524"></a>
<div id="DSN013F3" data-id="DSN013F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/m_dsn01303.jpeg?Expires=2147483647&Signature=IC4g7y6NgW0lAbEU2h1er595LxJ0LiYDZfxtOJSH8cNoUsNX~LtUwcLvtXoRT6c1lSmuomcvcXuvQ4JpHJYxS7j5NVq-Y3gimN3WR4bkNm3RpN9TMqtz-VsTukXCNu3r0-2OZTS4rjZxgcZpuyEOnAnrZpi97WxKkvJa92F8qLMV-vLvF8yzI9Ul9q3nyzKR8ZV1-h1UUjJ3vCE143aEu2vOjSM~s9-2WE0c-MsT~lV9RwrEcHVjrP6k3IRCWsUlYbviDqlzQZryEIlm09~vlSZESc6NCU0eYprka91LVRl9ofHd7i6aOu~DwMRzDCd7FObeW6wA1YoUHjxVaJuyJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MITE in P. gingivalis with Repeating Structure (MITEPgRS). (A) Schematic presentation of the consensus structure of MITEPgRS and the structures of 20 copies of MITEPgRS identified in the ATCC 33277 genome are shown. Three kind of repeat sequences, Repeats A, B, and C, are depicted by colored boxes. Red triangles indicate IR sequences and a black thick line in MITEPgRS_08 a unique nucleotide sequence. (B) Consensus sequences of Repeats A, B, and C are shown." path-from-xml="dsn01303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484524/dsn01303.jpeg" path-from-xml="dsn01303.jpeg" target="_blank">View large</a><a section="5484524" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/dsn01303.jpeg?Expires=2147483647&Signature=J3PPypk1XV~J611XV8c4Enw1qs9RPV0Zus91yE-z4Me93VhWkvYM~yNaWLA1477i2yd4nZ2of5g0yP-9viqqnqK06-wcq6nF5e-0i2tN2xw-aK5s3XCRLrH5ypvCOxTjb7TTe-XLOG~N3xbZYai5jZoQJ-t7aktVBmYEn9GTnPXzXUbm6ilqJhc27fYmXsV~zjCf7xSqNTNKkbvM3WXnfEl-EbDE3Uyo~uzOgNG5E1JM0fxeJ~b6pBVepkwdavf92mLNmJwsvp1jzpHKBbVXU34inA-r2T-tee~AwL381pbQmMIIWC3RYO9o51xgEWQ6c0wdfesyp2eYEHQYo-7xQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484524&ar=376506&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>MITE in <em>P. gingivalis</em> with Repeating Structure (MITE<em>PgRS</em>). (<strong>A</strong>) Schematic presentation of the consensus structure of MITE<em>PgRS</em> and the structures of 20 copies of MITE<em>PgRS</em> identified in the ATCC 33277 genome are shown. Three kind of repeat sequences, Repeats A, B, and C, are depicted by colored boxes. Red triangles indicate IR sequences and a black thick line in MITE<em>PgRS</em>_08 a unique nucleotide sequence. (<strong>B</strong>) Consensus sequences of Repeats A, B, and C are shown.</p></div></div><div content-id="DSN013F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/m_dsn01303.jpeg?Expires=2147483647&Signature=IC4g7y6NgW0lAbEU2h1er595LxJ0LiYDZfxtOJSH8cNoUsNX~LtUwcLvtXoRT6c1lSmuomcvcXuvQ4JpHJYxS7j5NVq-Y3gimN3WR4bkNm3RpN9TMqtz-VsTukXCNu3r0-2OZTS4rjZxgcZpuyEOnAnrZpi97WxKkvJa92F8qLMV-vLvF8yzI9Ul9q3nyzKR8ZV1-h1UUjJ3vCE143aEu2vOjSM~s9-2WE0c-MsT~lV9RwrEcHVjrP6k3IRCWsUlYbviDqlzQZryEIlm09~vlSZESc6NCU0eYprka91LVRl9ofHd7i6aOu~DwMRzDCd7FObeW6wA1YoUHjxVaJuyJA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="MITE in P. gingivalis with Repeating Structure (MITEPgRS). (A) Schematic presentation of the consensus structure of MITEPgRS and the structures of 20 copies of MITEPgRS identified in the ATCC 33277 genome are shown. Three kind of repeat sequences, Repeats A, B, and C, are depicted by colored boxes. Red triangles indicate IR sequences and a black thick line in MITEPgRS_08 a unique nucleotide sequence. (B) Consensus sequences of Repeats A, B, and C are shown." path-from-xml="dsn01303.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484524/dsn01303.jpeg" path-from-xml="dsn01303.jpeg" target="_blank">View large</a><a section="5484524" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/dsn01303.jpeg?Expires=2147483647&Signature=J3PPypk1XV~J611XV8c4Enw1qs9RPV0Zus91yE-z4Me93VhWkvYM~yNaWLA1477i2yd4nZ2of5g0yP-9viqqnqK06-wcq6nF5e-0i2tN2xw-aK5s3XCRLrH5ypvCOxTjb7TTe-XLOG~N3xbZYai5jZoQJ-t7aktVBmYEn9GTnPXzXUbm6ilqJhc27fYmXsV~zjCf7xSqNTNKkbvM3WXnfEl-EbDE3Uyo~uzOgNG5E1JM0fxeJ~b6pBVepkwdavf92mLNmJwsvp1jzpHKBbVXU34inA-r2T-tee~AwL381pbQmMIIWC3RYO9o51xgEWQ6c0wdfesyp2eYEHQYo-7xQA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484524&ar=376506&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01303.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>MITE in <em>P. gingivalis</em> with Repeating Structure (MITE<em>PgRS</em>). (<strong>A</strong>) Schematic presentation of the consensus structure of MITE<em>PgRS</em> and the structures of 20 copies of MITE<em>PgRS</em> identified in the ATCC 33277 genome are shown. Three kind of repeat sequences, Repeats A, B, and C, are depicted by colored boxes. Red triangles indicate IR sequences and a black thick line in MITE<em>PgRS</em>_08 a unique nucleotide sequence. (<strong>B</strong>) Consensus sequences of Repeats A, B, and C are shown.</p></div></div><p>On the basis of the structural features of each MITE, we rescreened IS elements and MITEs on the W83 genome, and we identified 93 IS elements (including 51 partial ones) and 35 MITEs (including nine partial ones) (Table <a reveal-id="DSN013TB1" data-open="DSN013TB1" class="link link-reveal link-table xref-fig">1</a> and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). The total numbers of IS elements and MITEs of the two strains were very similar, but the compositions showed a significant difference. In ATCC 33277, IS<em>Pg1</em>, IS<em>Pg3</em>, and MITE239 have been significantly expanded, whereas no intact copies of IS<em>Pg4</em> and IS<em>Pg5</em> are present. A previous study also showed that the numbers of copies and insertion sites of IS<em>Pg1</em> (formerly IS<em>1126</em>) markedly varied among <em>P. gingivalis</em> strains.<sup><a reveal-id="DSN013C42" data-open="DSN013C42" class="link link-ref link-reveal xref-bibr">42</a></sup> The composition of IS elements and MITEs may be an important feature to distinguish strains of <em>P. gingivalis</em>.</p>                            <h3 scrollto-destination=5484526 id="5484526" class="section-title" data-legacy-id=s3e>3.5. Genome rearrangement</h3>
<p>Whole genome sequence alignment analysis by using the MUMmer program revealed that extensive genomic rearrangements have taken place between ATCC 33277 and W83 (Fig. <a reveal-id="DSN013F4" data-open="DSN013F4" class="link link-reveal link-table xref-fig">4</a>). A number of X-shaped structures were observed, indicating that symmetrical inversions repeatedly occurred around the replication axis. We identified a total of 175 genomic regions in which genomic rearrangements took place in either strain. Inversions or translocations were observed in 35 regions, and simple insertions, deletions, or replacements were observed in 140 regions. Remarkably, about two-thirds of these genomic rearrangements were associated with the presence of mobile genetic elements (IS, MITE, Tn, CTn, and a not well-defined large mobile element of W83). Large inversions or translocations have occurred between two <em>rrn</em> operons, duplicated DNA regions coding for a histone-like family DNA-binding protein (PGN_0614 and PGN_1407), a hypothetical protein (PGN_0615 and PGN_1406), elongation factor P (PGN_0616 and PGN_1405), two identical 11 bp sequences (TAATCATAATA), and two similar 12/13 bp sequences (TTTTC(GCC/AATG)AAAA). DNA sequences similar to the 12/13 bp sequences were also present in the <em>att</em> sites for CTn<em>Pg1</em> described in the following section. These rearrangements appear to be deeply involved in the generation of strain-specific CDSs: 60% of ATCC 33277-specific CDSs and 68% of W83-specific CDSs were created by these genomic rearrangements.</p>                        <a id="5484528" scrollto-destination="5484528"></a>
<div id="DSN013F4" data-id="DSN013F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/m_dsn01304.jpeg?Expires=2147483647&Signature=XkD8tsdlZ4sYzz38LoNd2nUBJJy4LPTM~8Ul0hx9AE3ryH9kSGJ5bw7rp2IvJ3BTwYGcZJ9qS1Mkk~dnyDSYYItObX4EySk1OA21lLt24IOuxlOmBrX3fzjGxt6oMk2jHWf-RzStbOcOmE~krSzfju5J7sHKNDtTEwdt3JM6NBk7ZBWWUnKboWmSIrMOj7jYZL~1WOr6Kh7i2oZQZ~zEvEyNaFqWxdt7-u-881Yd4Q0G1uL8JysBFhJ4zHjo5U-QjR89TlDGhGTTr5OwOcgrBHoLznb70q-FI2tW-yVQygH78MVsfERWIZd6llMPIcA3ApPxA9j~pCxPKz8eyOV0OQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The DNA sequence identity plot of P. gingivalis ATCC 33277 and W83 chromosomes. The dnaA gene is located at the left and bottom corner. Black circles indicate mobile genetic elements (CTn Tn, IS, MITE, or a not-well-defined large mobile element of W83). The chromosomal locations of other genetic elements that mediated inversions or translocations are shown in the right: rrn operons (black squares), duplicated regions coding for a histone-like DNA binding protein, a hypothetical protein and elongation factor P (open squares), 12/13 bp repeat sequences (black triangle), and 11 bp repeat sequences (open triangle)." path-from-xml="dsn01304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484528/dsn01304.jpeg" path-from-xml="dsn01304.jpeg" target="_blank">View large</a><a section="5484528" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/dsn01304.jpeg?Expires=2147483647&Signature=s3zukdTBrfWYtFmAZmhV~8Fm1OilLRgrUXqSXzcE5JftJiVKKFJ~1hxcOsUtaU9W6pst48j-aFXtGtPLZFzc3cOeyTk50YWafm7tROaaAPb~5gCZwKziimrIXQbiwsMGsoQLhWmpH5oRLQ5WMWuw--IHvHIDu-dwIc1MmVpXhxDo-N7otDNMX-b5gGMgNJN6dzmD~F3-X3ypYn24uxOXSLf3ZP24mEcIHqhwYjbDBF8XDKRqe-HqnMNJr13hsfa1jd7UNw1KFE8lApharRB2Px2DienbsWvndvQjF79t8KRPt8KQaOqQB6tKTAAaEDgff19GfzL-ii-mCS~s--2z7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484528&ar=376506&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The DNA sequence identity plot of <em>P. gingivalis</em> ATCC 33277 and W83 chromosomes. The <em>dnaA</em> gene is located at the left and bottom corner. Black circles indicate mobile genetic elements (CTn Tn, IS, MITE, or a not-well-defined large mobile element of W83). The chromosomal locations of other genetic elements that mediated inversions or translocations are shown in the right: <em>rrn</em> operons (black squares), duplicated regions coding for a histone-like DNA binding protein, a hypothetical protein and elongation factor P (open squares), 12/13 bp repeat sequences (black triangle), and 11 bp repeat sequences (open triangle).</p></div></div><div content-id="DSN013F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/m_dsn01304.jpeg?Expires=2147483647&Signature=XkD8tsdlZ4sYzz38LoNd2nUBJJy4LPTM~8Ul0hx9AE3ryH9kSGJ5bw7rp2IvJ3BTwYGcZJ9qS1Mkk~dnyDSYYItObX4EySk1OA21lLt24IOuxlOmBrX3fzjGxt6oMk2jHWf-RzStbOcOmE~krSzfju5J7sHKNDtTEwdt3JM6NBk7ZBWWUnKboWmSIrMOj7jYZL~1WOr6Kh7i2oZQZ~zEvEyNaFqWxdt7-u-881Yd4Q0G1uL8JysBFhJ4zHjo5U-QjR89TlDGhGTTr5OwOcgrBHoLznb70q-FI2tW-yVQygH78MVsfERWIZd6llMPIcA3ApPxA9j~pCxPKz8eyOV0OQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="The DNA sequence identity plot of P. gingivalis ATCC 33277 and W83 chromosomes. The dnaA gene is located at the left and bottom corner. Black circles indicate mobile genetic elements (CTn Tn, IS, MITE, or a not-well-defined large mobile element of W83). The chromosomal locations of other genetic elements that mediated inversions or translocations are shown in the right: rrn operons (black squares), duplicated regions coding for a histone-like DNA binding protein, a hypothetical protein and elongation factor P (open squares), 12/13 bp repeat sequences (black triangle), and 11 bp repeat sequences (open triangle)." path-from-xml="dsn01304.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484528/dsn01304.jpeg" path-from-xml="dsn01304.jpeg" target="_blank">View large</a><a section="5484528" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/dsn01304.jpeg?Expires=2147483647&Signature=s3zukdTBrfWYtFmAZmhV~8Fm1OilLRgrUXqSXzcE5JftJiVKKFJ~1hxcOsUtaU9W6pst48j-aFXtGtPLZFzc3cOeyTk50YWafm7tROaaAPb~5gCZwKziimrIXQbiwsMGsoQLhWmpH5oRLQ5WMWuw--IHvHIDu-dwIc1MmVpXhxDo-N7otDNMX-b5gGMgNJN6dzmD~F3-X3ypYn24uxOXSLf3ZP24mEcIHqhwYjbDBF8XDKRqe-HqnMNJr13hsfa1jd7UNw1KFE8lApharRB2Px2DienbsWvndvQjF79t8KRPt8KQaOqQB6tKTAAaEDgff19GfzL-ii-mCS~s--2z7w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484528&ar=376506&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01304.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>The DNA sequence identity plot of <em>P. gingivalis</em> ATCC 33277 and W83 chromosomes. The <em>dnaA</em> gene is located at the left and bottom corner. Black circles indicate mobile genetic elements (CTn Tn, IS, MITE, or a not-well-defined large mobile element of W83). The chromosomal locations of other genetic elements that mediated inversions or translocations are shown in the right: <em>rrn</em> operons (black squares), duplicated regions coding for a histone-like DNA binding protein, a hypothetical protein and elongation factor P (open squares), 12/13 bp repeat sequences (black triangle), and 11 bp repeat sequences (open triangle).</p></div></div><p>ATCC 33277 and W83 both contain four <em>rrn</em> operons of identical nucleotide sequences, but chromosomal locations of the four <em>rrn</em> operons differ markedly between the strains. By comparing the <em>rrn</em> operon-flanking regions in the two strains, we found that an inversion had taken place between <em>rrn1</em> and <em>rrn4</em> (see <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). Additional genomic rearrangements that have occurred in the genomic loci other than <em>rrn</em> operons further altered the relative locations of the four <em>rrn</em> operons on the two genomes. We analyzed the structures of <em>rrn</em> operon-flanking regions in five other strains of <em>P. gingivalis</em> (TDC60, TDC117, TDC275, SU63, and GAI7802) using a set of orientation-specific primer pairs, and we found that all of the <em>rrn</em> operon-flanking regions of these five strains have the same structures as those of ATCC 33277 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). This result suggests that inversions between <em>rrn1</em> and <em>rrn4</em> have taken place specifically in the W83 strain lineage among the strains tested.</p><p>The genomic regions for the biosynthesis of cell surface molecules have also significantly diverged between the two strains. They included the regions for FimA fimbrilin, Mfa1 fimbrilin, capsular polysaccharides, RagA and RagB antigens, and glycosyl transferase.<sup><a reveal-id="DSN013C42  DSN013C45" data-open="DSN013C42  DSN013C45" class="link link-ref link-reveal xref-bibr">42–45</a></sup> Among these, the difference in the locus for capsular polysaccharide biosynthesis (GP1 locus) is particularly important because capsular polysaccharide is known to be one of the major virulence factors of <em>P. gingivalis</em>. The GP1 locus of ATCC 33277 (PGN_0223-PGN_0236) is identical to that reported for strain 381<sup><a reveal-id="DSN013C43" data-open="DSN013C43" class="link link-ref link-reveal xref-bibr">43</a></sup> except that one nonsense mutation was found in the PGN_0223-homolog of strain 381 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>).</p><p>The clustered regularly interspaced short palindromic repeats (CRISPR) locus also exhibits notable structural difference between the two strains. The numbers of repeats in the repeat/spacer region of the CRISPR-30-36 locus are 120 in ATCC 33277 and 23 in W83 (Fig. <a reveal-id="DSN013F5" data-open="DSN013F5" class="link link-reveal link-table xref-fig">5</a>). The nucleotide sequences of the repeats are identical, but there is no homology in the spacer regions. A part of the CRISPR-associated gene (<em>cas</em>) encoding region has been replaced by very different sequences. Of interest is that the gene organization of the <em>cas</em>-encoding region of ATCC 33277 is nearly identical to that of <em>B. fragilis</em> YCH46 but very different from that of W83. The gene products also exhibit a high level of similarity of amino-acid sequences to those of <em>B. fragilis</em> YCH46. These results suggest that these genes may have been horizontally transferred between <em>P. gingivalis</em> and <em>B. fragilis</em>.</p>                        <a id="5484532" scrollto-destination="5484532"></a>
<div id="DSN013F5" data-id="DSN013F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/m_dsn01305.jpeg?Expires=2147483647&Signature=RRBAsgVp1wAZChSOJ4HqV~fVTaNF5vpMpfBE3gFyQcqo-bvQ9~aFPdnEZgxJTAjQhz1r-JO1aW6qk2hTI4W7OYoNSS1pgjKHYSVzZoqdqM9SmxARYYjSIUKAJciJ80dQVmQ3T~XMxzdLYnYtugNRHCWnLhi2~CX9FFQ6EZla6I1zB4494Q4y8zqF-mW4jDoMGvH7slph0qUpiwJLTPH7bDANNtk43xjylWGX23Y6LrhEoQ-mEhttCFb9H7m2xHA~E15ux~phx2Tt5qIXlGRJ9I9wtmqZMj6Bd9tiUxm-CEw1FzvsYnJog78BDM3vVLPT1Z215sbw92i3vROJNLMyWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of CRISPR-30-36 regions of P. gingivalis and B. fragilis. Locations and directions of CDSs (arrows) and repeat regions (black rectangles) are drawn to scale. Homologous CDSs are indicated by gray shading, and their amino-acid sequence identities are also shown. CDSs for IS transposases are indicated by black arrows, cas genes by vertically striped arrows, other functionally annotated CDSs by hatched arrows, and CDSs for hypothetical proteins by white arrows. The identity between PGN_1964 and PG2016 is 15.7%." path-from-xml="dsn01305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484532/dsn01305.jpeg" path-from-xml="dsn01305.jpeg" target="_blank">View large</a><a section="5484532" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/dsn01305.jpeg?Expires=2147483647&Signature=LFw3p~chbSqjD-4vXwdkVqPAZ3r9aiZdmYe7nh1TnwMc8ufWHs4cVrU-qOJpB1Sl4-zDQitg7MR54FBH9ilJNi74nLNelsvkK8GcB06Z4TZUMvMFhW0lSw9M0Zs6nRor5hgjAdkdxyPfV6IMaHjR5cPGQyK-5dLErbhfOa5ePsx94b~4kZ-MYNcKks0GCsElM2OpuPFq1QqHewP3-Zg0GT6tznyP04nYQpfQrqazJV6wPbQYtUcM0JhtMHkOsLBqmv5Vu2q8n8mUaZvxgL0jzc1gCYUAWapMb904Fw5U7t7EN8jWcuzM9olc7hoq9rJGc-yES0-t0QdLad0RrdOItA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484532&ar=376506&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of CRISPR-30-36 regions of <em>P. gingivalis</em> and <em>B. fragilis</em>. Locations and directions of CDSs (arrows) and repeat regions (black rectangles) are drawn to scale. Homologous CDSs are indicated by gray shading, and their amino-acid sequence identities are also shown. CDSs for IS transposases are indicated by black arrows, <em>cas</em> genes by vertically striped arrows, other functionally annotated CDSs by hatched arrows, and CDSs for hypothetical proteins by white arrows. The identity between PGN_1964 and PG2016 is 15.7%.</p></div></div><div content-id="DSN013F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/m_dsn01305.jpeg?Expires=2147483647&Signature=RRBAsgVp1wAZChSOJ4HqV~fVTaNF5vpMpfBE3gFyQcqo-bvQ9~aFPdnEZgxJTAjQhz1r-JO1aW6qk2hTI4W7OYoNSS1pgjKHYSVzZoqdqM9SmxARYYjSIUKAJciJ80dQVmQ3T~XMxzdLYnYtugNRHCWnLhi2~CX9FFQ6EZla6I1zB4494Q4y8zqF-mW4jDoMGvH7slph0qUpiwJLTPH7bDANNtk43xjylWGX23Y6LrhEoQ-mEhttCFb9H7m2xHA~E15ux~phx2Tt5qIXlGRJ9I9wtmqZMj6Bd9tiUxm-CEw1FzvsYnJog78BDM3vVLPT1Z215sbw92i3vROJNLMyWA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Comparison of CRISPR-30-36 regions of P. gingivalis and B. fragilis. Locations and directions of CDSs (arrows) and repeat regions (black rectangles) are drawn to scale. Homologous CDSs are indicated by gray shading, and their amino-acid sequence identities are also shown. CDSs for IS transposases are indicated by black arrows, cas genes by vertically striped arrows, other functionally annotated CDSs by hatched arrows, and CDSs for hypothetical proteins by white arrows. The identity between PGN_1964 and PG2016 is 15.7%." path-from-xml="dsn01305.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484532/dsn01305.jpeg" path-from-xml="dsn01305.jpeg" target="_blank">View large</a><a section="5484532" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/dsn01305.jpeg?Expires=2147483647&Signature=LFw3p~chbSqjD-4vXwdkVqPAZ3r9aiZdmYe7nh1TnwMc8ufWHs4cVrU-qOJpB1Sl4-zDQitg7MR54FBH9ilJNi74nLNelsvkK8GcB06Z4TZUMvMFhW0lSw9M0Zs6nRor5hgjAdkdxyPfV6IMaHjR5cPGQyK-5dLErbhfOa5ePsx94b~4kZ-MYNcKks0GCsElM2OpuPFq1QqHewP3-Zg0GT6tznyP04nYQpfQrqazJV6wPbQYtUcM0JhtMHkOsLBqmv5Vu2q8n8mUaZvxgL0jzc1gCYUAWapMb904Fw5U7t7EN8jWcuzM9olc7hoq9rJGc-yES0-t0QdLad0RrdOItA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484532&ar=376506&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01305.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Comparison of CRISPR-30-36 regions of <em>P. gingivalis</em> and <em>B. fragilis</em>. Locations and directions of CDSs (arrows) and repeat regions (black rectangles) are drawn to scale. Homologous CDSs are indicated by gray shading, and their amino-acid sequence identities are also shown. CDSs for IS transposases are indicated by black arrows, <em>cas</em> genes by vertically striped arrows, other functionally annotated CDSs by hatched arrows, and CDSs for hypothetical proteins by white arrows. The identity between PGN_1964 and PG2016 is 15.7%.</p></div></div><p>Among the bacterial species so far sequenced, <em>Bacteroides</em> species are phylogenically most closely related to <em>P. gingivalis</em>. However, two sequenced <em>B. fragilis</em> strains show no such extensive genomic rearrangement as seen in <em>P. gingivalis</em>.<sup><a reveal-id="DSN013C40" data-open="DSN013C40" class="link link-ref link-reveal xref-bibr">40</a>,<a reveal-id="DSN013C46" data-open="DSN013C46" class="link link-ref link-reveal xref-bibr">46</a></sup><em>Shigella flexneri</em>, a pathogen for dysentery, contains a large number of IS elements and the bacterium has induced extensive genomic rearrangements among strains, which may create differences in virulence and epidemicity.<sup><a reveal-id="DSN013C47" data-open="DSN013C47" class="link link-ref link-reveal xref-bibr">47</a>,<a reveal-id="DSN013C48" data-open="DSN013C48" class="link link-ref link-reveal xref-bibr">48</a></sup> In <em>P. gingivalis</em>, the genomic rearrangements induced by IS and other mobile genetic elements may also have been involved in the generation of strain-to-strain difference in virulence. In this context, a recent finding that treatment of <em>P. gingivalis</em> cells with H<sub>2</sub>O<sub>2</sub> induces expression of the IS<em>Pg1</em> transposase gene is noteworthy.<sup><a reveal-id="DSN013C49" data-open="DSN013C49" class="link link-ref link-reveal xref-bibr">49</a></sup> The fact that the number of copies of IS<em>Pg1</em> varies among <em>P. gingivalis</em> strains and the fact that IS<em>Pg1</em> is frequently associated with genome rearrangements suggest that oxidative stress-induced expression of the IS<em>Pg1</em> transposase gene results in transposition of IS<em>Pg1</em> that may mediate genomic rearrangements in <em>P. gingivalis</em>, and such rearrangements may contribute to the adaptation of <em>P. gingivalis</em> strains to an oxygen concentration-changeable environment in the gingival crevice.</p>                            <h3 scrollto-destination=5484534 id="5484534" class="section-title" data-legacy-id=s3f>3.6. Excision of CTnPg1</h3>
<p>We could not exactly determine the attachment (<em>att</em>) site of CTnPg1 by comparing the genome sequences of ATCC 33277 and W83 because integration of CTnPg1-a has induced a genomic rearrangement and half of CTnPg1-b has been deleted. We therefore examined whether CTnPg1 can be excised from the chromosome and form a circular intermediate. By PCR analysis of the chromosomal DNA of ATCC 33277 using two primers targeting the left and right ends of CTnPg1 (Fig. <a reveal-id="DSN013F6" data-open="DSN013F6" class="link link-reveal link-table xref-fig">6</a>A), we obtained a PCR product of 1500 bp in size (Fig. <a reveal-id="DSN013F6" data-open="DSN013F6" class="link link-reveal link-table xref-fig">6</a>B). We further investigated whether a cast-off genome can be generated by the excision of CTnPg1 using two PCR primers targeting the CTnPg1-a-flanking regions, and we obtained a PCR product of 1000 bp in size (Fig. <a reveal-id="DSN013F6" data-open="DSN013F6" class="link link-reveal link-table xref-fig">6</a>A and B). By comparing the sequences of the two PCR products with the genome sequence of ATCC 33277, we identified the core sequence of the <em>att</em> site for CTnPg1, ATTTTCA(CA/TT)GAAAA (Fig. <a reveal-id="DSN013F6" data-open="DSN013F6" class="link link-reveal link-table xref-fig">6</a>C). The same sequence was also found at one end of CTnPg1-b.</p>                        <a id="5484536" scrollto-destination="5484536"></a>
<div id="DSN013F6" data-id="DSN013F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/m_dsn01306.jpeg?Expires=2147483647&Signature=Gm1JM3jUq-zMg8-i~mBwnUIw9kqhpXkJD4Yimxa2UNQf6UiU~KJNXHAiwLOSmcZlcQ7vxqqwxwo72BcXQZCyTMB4LTNN4MTQnm~cGi8sfCyC1tA8uGQEg7wevoq01rewXYA0gdtv~IHSJAV5iKWr4SiRwclvg0EDXfaHyLiB2y0YWk2M5g1rAgzKyy6SMm0d5tQzb48dUNbG5-9W9HC4isNFcQrVNEu7NcYxSfM2YGQMMX4EvRD1GdvGEEhItnOH291UhIFI6kgGNezbHm53-lqOOa7zaDeDgD8s2IURreH6ENIha0kL-oK50o0~L1jAeMJHRUXeFBRsLaLzSzqJ4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Excision of CTnPg1-a. (A) Schematic presentation of the structure of CTnPg1-a and the strategy to detect the excised circular intermediate and cast-off chromosome. Locations of PCR primers are indicated by black arrow heads. CDSs on CTnPg1 are depicted by black arrows and other CDSs by open arrows. Open triangles indicate att regions of CTnPg1. (B) Agarose gel electrophoresis of PCR products obtained by the primer pairs CTnPg1-right/CTnPg1-left (lane 1) and CTnPg1-up/CTnPg1-down (lane 2). (C) Sequence alignment of attP, attB, attL, and attR regions of CTnPg1. The 14 bp core sequence is indicated by a box." path-from-xml="dsn01306.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484536/dsn01306.jpeg" path-from-xml="dsn01306.jpeg" target="_blank">View large</a><a section="5484536" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/dsn01306.jpeg?Expires=2147483647&Signature=TAoxfZw6s~s7kkWz2Lw97kjhEFoM6pJVfOfRoZj05wUobe81uE-r0cgsjVfmtL3fRR98STFm9DGU1vKVZz4p9kjcaQXbwSzyJWrrd7iEoyiQXW-lad57sypB22l67K9ab57sw1oZ2yVXUG9pY5c3yozFAhAF83BJWjcqLOV38u4xbAt9uBg4Jvtke~Yf6UaU0jeJm15DQ-V2RibQ~Toq629OW~FMnxSBkVS8P6S2M76MmPq4IWhRyu0PcH4Pj3Yr8e6VawB2UM~4t8U954jBtc1uGXyGJwtDKbtU8AIFxXCztWlp6LO1SLkkWcwu02j767c2CVMczMZVlDxpcBHRhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484536&ar=376506&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01306.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Excision of CTnPg1-a. (<strong>A</strong>) Schematic presentation of the structure of CTnPg1-a and the strategy to detect the excised circular intermediate and cast-off chromosome. Locations of PCR primers are indicated by black arrow heads. CDSs on CTnPg1 are depicted by black arrows and other CDSs by open arrows. Open triangles indicate <em>att</em> regions of CTnPg1. (<strong>B</strong>) Agarose gel electrophoresis of PCR products obtained by the primer pairs CTnPg1-right/CTnPg1-left (lane 1) and CTnPg1-up/CTnPg1-down (lane 2). (<strong>C</strong>) Sequence alignment of <em>att</em>P, <em>attB</em>, <em>attL</em>, and <em>attR</em> regions of CTnPg1. The 14 bp core sequence is indicated by a box.</p></div></div><div content-id="DSN013F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/m_dsn01306.jpeg?Expires=2147483647&Signature=Gm1JM3jUq-zMg8-i~mBwnUIw9kqhpXkJD4Yimxa2UNQf6UiU~KJNXHAiwLOSmcZlcQ7vxqqwxwo72BcXQZCyTMB4LTNN4MTQnm~cGi8sfCyC1tA8uGQEg7wevoq01rewXYA0gdtv~IHSJAV5iKWr4SiRwclvg0EDXfaHyLiB2y0YWk2M5g1rAgzKyy6SMm0d5tQzb48dUNbG5-9W9HC4isNFcQrVNEu7NcYxSfM2YGQMMX4EvRD1GdvGEEhItnOH291UhIFI6kgGNezbHm53-lqOOa7zaDeDgD8s2IURreH6ENIha0kL-oK50o0~L1jAeMJHRUXeFBRsLaLzSzqJ4Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Excision of CTnPg1-a. (A) Schematic presentation of the structure of CTnPg1-a and the strategy to detect the excised circular intermediate and cast-off chromosome. Locations of PCR primers are indicated by black arrow heads. CDSs on CTnPg1 are depicted by black arrows and other CDSs by open arrows. Open triangles indicate att regions of CTnPg1. (B) Agarose gel electrophoresis of PCR products obtained by the primer pairs CTnPg1-right/CTnPg1-left (lane 1) and CTnPg1-up/CTnPg1-down (lane 2). (C) Sequence alignment of attP, attB, attL, and attR regions of CTnPg1. The 14 bp core sequence is indicated by a box." path-from-xml="dsn01306.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/5484536/dsn01306.jpeg" path-from-xml="dsn01306.jpeg" target="_blank">View large</a><a section="5484536" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/dsn01306.jpeg?Expires=2147483647&Signature=TAoxfZw6s~s7kkWz2Lw97kjhEFoM6pJVfOfRoZj05wUobe81uE-r0cgsjVfmtL3fRR98STFm9DGU1vKVZz4p9kjcaQXbwSzyJWrrd7iEoyiQXW-lad57sypB22l67K9ab57sw1oZ2yVXUG9pY5c3yozFAhAF83BJWjcqLOV38u4xbAt9uBg4Jvtke~Yf6UaU0jeJm15DQ-V2RibQ~Toq629OW~FMnxSBkVS8P6S2M76MmPq4IWhRyu0PcH4Pj3Yr8e6VawB2UM~4t8U954jBtc1uGXyGJwtDKbtU8AIFxXCztWlp6LO1SLkkWcwu02j767c2CVMczMZVlDxpcBHRhQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=5484536&ar=376506&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsn01306.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Excision of CTnPg1-a. (<strong>A</strong>) Schematic presentation of the structure of CTnPg1-a and the strategy to detect the excised circular intermediate and cast-off chromosome. Locations of PCR primers are indicated by black arrow heads. CDSs on CTnPg1 are depicted by black arrows and other CDSs by open arrows. Open triangles indicate <em>att</em> regions of CTnPg1. (<strong>B</strong>) Agarose gel electrophoresis of PCR products obtained by the primer pairs CTnPg1-right/CTnPg1-left (lane 1) and CTnPg1-up/CTnPg1-down (lane 2). (<strong>C</strong>) Sequence alignment of <em>att</em>P, <em>attB</em>, <em>attL</em>, and <em>attR</em> regions of CTnPg1. The 14 bp core sequence is indicated by a box.</p></div></div>                            <h3 scrollto-destination=5484537 id="5484537" class="section-title" data-legacy-id=s3g>3.7. Glucose kinase-encoding gene</h3>
<p>Disability of saccharolysis is one of the major characteristics of <em>P. gingivalis</em>. Consistent with this, the glucose kinase-encoding gene (<em>glk</em>) (PG1737) has a nonsense mutation in strain W83 (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5</a>). Nelson et al.<sup><a reveal-id="DSN013C19" data-open="DSN013C19" class="link link-ref link-reveal xref-bibr">19</a></sup> suggested that the defect of <em>glk</em> accounts for asaccharolysis of <em>P. gingivalis</em>. In ATCC 33277, the <em>glk</em> gene (PGN_0380) has also been disrupted by an insertion of MITE239 but contains no nonsense mutation. Therefore, we analyzed the <em>glk</em> genes of other <em>P. gingivalis</em> strains to know which type of genetic defect is generally observed in <em>P. gingivalis</em>. Unexpectedly, however, the <em>glk</em> genes of five strains examined (strains TDC60, TDC117, TDC275, SU63, and GAI7802) were all intact. We detected no nonsense mutation or MITE239 insertion although a few amino-acid substitutions were observed in these <em>glk</em> genes (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S5B</a>). To determine whether the <em>glk</em> gene is expressed in the five strains, we quantified the mRNA of the <em>glk</em> gene using the Kgp-encoding gene (<em>kgp</em>) as a control (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). In this analysis, a large amount of the <em>glk</em> gene transcript was detected in all of the five strains. On the other hand, all the strains used in this study (ATCC 33277, W83, and five other strains) were confirmed to be asaccharolytic (data not shown). Thus, these data suggest that defects of <em>glk</em>, which were detected in W83 and ATCC 33277, cannot always account for asaccharolysis of <em>P. gingivalis</em>. Further work is needed to clarify physiological roles of glucose kinase in <em>P. gingivalis</em> and what is responsible for asaccharolysis of <em>P. gingivalis.</em></p>                            <h3 scrollto-destination=5484539 id="5484539" class="section-title" data-legacy-id=s3h>3.8. Conclusion</h3>
<p>In this study, we determined the whole genome sequence of ATCC 33277, a less-virulent <em>P. gingivalis</em> strain, and carried out a genomic comparison with a virulent strain, W83. Although the genome size and GC content are almost the same, we detected extensive rearrangements between the two strains, many of which have been induced by various mobile genetic elements (IS, MITE, Tn, and CTn). Such structural alterations of the <em>P. gingivalis</em> genomes generated many strain-specific CDSs and may be closely associated with difference in virulence of the two strains.</p>                            <h2 scrollto-destination=5484541 id="5484541" class="section-title" data-legacy-id=s4>Funding</h2>
<p>This work was supported by Grants-in-Aid for Scientific Research on Priority Areas “Comprehensive Genomics” (No. 17020007) to M.H.; “Applied Genomics” (No. 18018032) to K.N. from the Ministry of Education, Culture, Sports, Science and Technology of Japan.</p>                            <h2 scrollto-destination=5484543 id="5484543" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary data are available online at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=5484545 id="5484545" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank K. Oshima, K. Furuya, C. Yoshino, H. Inaba, K. Motomura, and Y. Hattori (University of Tokyo), A. Tamura and N. Itoh (Kitasato University), and Y. Kikuchi (Matsumoto Dental College) for their technical support.</p>                            <h2 scrollto-destination=5484547 id="5484547" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSN013C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Papapanou</div> <div class="given-names">P. N.</div></div>. </div><div class="article-title">Epidemiology of periodontal diseases: an update</div>, <div class="source">J. Int. Acad. Periodontol.</div> , <div class="year">1999</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">110</div>-<div class="lpage">116</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epidemiology%20of%20periodontal%20diseases%3A%20an%20update&amp;author=P.%20N.%20Papapanou&amp;publication_year=1999&amp;journal=J.%20Int.%20Acad.%20Periodontol.&amp;volume=1&amp;pages=110-116" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12666955" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Irfan</div> <div class="given-names">U. M.</div></div>,  <div class="name"><div class="surname">Dawson</div> <div class="given-names">D. V.</div></div>,  <div class="name"><div class="surname">Bissada</div> <div class="given-names">N. F.</div></div>. </div><div class="article-title">Epidemiology of periodontal disease: a review and clinical perspectives</div>, <div class="source">J. Int. Acad. Periodontol.</div> , <div class="year">2001</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">14</div>-<div class="lpage">21</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epidemiology%20of%20periodontal%20disease%3A%20a%20review%20and%20clinical%20perspectives&amp;author=U.%20M.%20Irfan&amp;author=D.%20V.%20Dawson&amp;author=N.%20F.%20Bissada&amp;publication_year=2001&amp;journal=J.%20Int.%20Acad.%20Periodontol.&amp;volume=3&amp;pages=14-21" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12666973" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Armitage</div> <div class="given-names">G. C.</div></div>. </div><div class="article-title">Periodontal diseases: diagnosis</div>, <div class="source">Ann. Periodontol.</div> , <div class="year">1996</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">37</div>-<div class="lpage">215</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Periodontal%20diseases%3A%20diagnosis&amp;author=G.%20C.%20Armitage&amp;publication_year=1996&amp;journal=Ann.%20Periodontol.&amp;volume=1&amp;pages=37-215" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1902/annals.1996.1.1.37" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1902%2Fannals.1996.1.1.37" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1902%2Fannals.1996.1.1.37"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9118264" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Oliver</div> <div class="given-names">R. C.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">L. J.</div></div>,  <div class="name"><div class="surname">Loe</div> <div class="given-names">H.</div></div>. </div><div class="article-title">Periodontal diseases in the United States population</div>, <div class="source">J. Periodontol.</div> , <div class="year">1998</div>, vol. <div class="volume">69</div> (pg. <div class="fpage">269</div>-<div class="lpage">278</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Periodontal%20diseases%20in%20the%20United%20States%20population&amp;author=R.%20C.%20Oliver&amp;author=L.%20J.%20Brown&amp;author=H.%20Loe&amp;publication_year=1998&amp;journal=J.%20Periodontol.&amp;volume=69&amp;pages=269-278" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1902/jop.1998.69.2.269" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1902%2Fjop.1998.69.2.269" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1902%2Fjop.1998.69.2.269"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9526927" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mattila</div> <div class="given-names">K. J.</div></div>,  <div class="name"><div class="surname">Valtonen</div> <div class="given-names">V. V.</div></div>,  <div class="name"><div class="surname">Nieminen</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Huttunen</div> <div class="given-names">J. K.</div></div>. </div><div class="article-title">Dental infection and the risk of new coronary events: prospective study of patients with documented coronary artery disease</div>, <div class="source">Clin. Infect. Dis.</div> , <div class="year">1995</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">588</div>-<div class="lpage">592</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Dental%20infection%20and%20the%20risk%20of%20new%20coronary%20events%3A%20prospective%20study%20of%20patients%20with%20documented%20coronary%20artery%20disease&amp;author=K.%20J.%20Mattila&amp;author=V.%20V.%20Valtonen&amp;author=M.%20Nieminen&amp;author=J.%20K.%20Huttunen&amp;publication_year=1995&amp;journal=Clin.%20Infect.%20Dis.&amp;volume=20&amp;pages=588-592" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/clinids/20.3.588" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fclinids%2F20.3.588" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fclinids%2F20.3.588"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7756480" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Beck</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Garcia</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Heiss</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Vokonas</div> <div class="given-names">P. S.</div></div>,  <div class="name"><div class="surname">Offenbacher</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Periodontal disease and cardiovascular disease</div>, <div class="source">J. Periodontol.</div> , <div class="year">1996</div>, vol. <div class="volume">67</div> (pg. <div class="fpage">1123</div>-<div class="lpage">1137</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Periodontal%20disease%20and%20cardiovascular%20disease&amp;author=J.%20Beck&amp;author=R.%20Garcia&amp;author=G.%20Heiss&amp;author=P.%20S.%20Vokonas&amp;author=S.%20Offenbacher&amp;publication_year=1996&amp;journal=J.%20Periodontol.&amp;volume=67&amp;pages=1123-1137" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1902/jop.1996.67.10s.1123" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1902%2Fjop.1996.67.10s.1123" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1902%2Fjop.1996.67.10s.1123"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8910831" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Joshipura</div> <div class="given-names">K. J.</div></div>,  <div class="name"><div class="surname">Rimm</div> <div class="given-names">E. B.</div></div>,  <div class="name"><div class="surname">Douglass</div> <div class="given-names">C. W.</div></div>,  <div class="name"><div class="surname">Trichopoulos</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Ascherio</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Willett</div> <div class="given-names">W. C.</div></div>. </div><div class="article-title">Poor oral health and coronary heart disease</div>, <div class="source">J. Dent. Res.</div> , <div class="year">1996</div>, vol. <div class="volume">75</div> (pg. <div class="fpage">1631</div>-<div class="lpage">1636</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Poor%20oral%20health%20and%20coronary%20heart%20disease&amp;author=K.%20J.%20Joshipura&amp;author=E.%20B.%20Rimm&amp;author=C.%20W.%20Douglass&amp;author=D.%20Trichopoulos&amp;author=A.%20Ascherio&amp;author=W.%20C.%20Willett&amp;publication_year=1996&amp;journal=J.%20Dent.%20Res.&amp;volume=75&amp;pages=1631-1636" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1177/00220345960750090301" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1177%2F00220345960750090301" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1177%2F00220345960750090301"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8952614" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Morrison</div> <div class="given-names">H. I.</div></div>,  <div class="name"><div class="surname">Ellison</div> <div class="given-names">L. F.</div></div>,  <div class="name"><div class="surname">Taylor</div> <div class="given-names">G. W.</div></div>. </div><div class="article-title">Periodontal disease and risk of fatal coronary heart and cerebrovascular diseases</div>, <div class="source">J. Cardiovasc. Risk</div> , <div class="year">1999</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">7</div>-<div class="lpage">11</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Periodontal%20disease%20and%20risk%20of%20fatal%20coronary%20heart%20and%20cerebrovascular%20diseases&amp;author=H.%20I.%20Morrison&amp;author=L.%20F.%20Ellison&amp;author=G.%20W.%20Taylor&amp;publication_year=1999&amp;journal=J.%20Cardiovasc.%20Risk&amp;volume=6&amp;pages=7-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1177/204748739900600102" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1177%2F204748739900600102" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1177%2F204748739900600102"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10197286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chiu</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Multiple infections in carotid atherosclerotic plaques</div>, <div class="source">Am. Heart J.</div> , <div class="year">1999</div>, vol. <div class="volume">138</div> (pg. <div class="fpage">S534</div>-<div class="lpage">S536</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Multiple%20infections%20in%20carotid%20atherosclerotic%20plaques&amp;author=B.%20Chiu&amp;publication_year=1999&amp;journal=Am.%20Heart%20J.&amp;volume=138&amp;pages=S534-S536" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0002-8703(99)70294-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0002-8703(99)70294-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0002-8703(99)70294-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10539867" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kozarov</div> <div class="given-names">E. V.</div></div>,  <div class="name"><div class="surname">Dorn</div> <div class="given-names">B. R.</div></div>,  <div class="name"><div class="surname">Shelburne</div> <div class="given-names">C. E.</div></div>,  <div class="name"><div class="surname">Dunn</div> <div class="given-names">W. A.</div>Jr</div>,  <div class="name"><div class="surname">Progulske-Fox</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Human atherosclerotic plaque contains viable <em>Actinbacillus actinomycetemcomitans</em> and <em>Porphyromonas gingivalis</em></div>, <div class="source">Arterioscler. Thromb. Vasc. Biol.</div> , <div class="year">2005</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">e17</div>-<div class="lpage">e18</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Human%20atherosclerotic%20plaque%20contains%20viable%20Actinbacillus%20actinomycetemcomitans%20and%20Porphyromonas%20gingivalis&amp;author=E.%20V.%20Kozarov&amp;author=B.%20R.%20Dorn&amp;author=C.%20E.%20Shelburne&amp;author=W.%20A.%20Dunn&amp;author=A.%20Progulske-Fox&amp;publication_year=2005&amp;journal=Arterioscler.%20Thromb.%20Vasc.%20Biol.&amp;volume=25&amp;pages=e17-e18" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1161/01.ATV.0000155018.67835.1a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1161%2F01.ATV.0000155018.67835.1a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1161%2F01.ATV.0000155018.67835.1a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15662025" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Boone</div> <div class="given-names">D. R.</div></div>,  <div class="name"><div class="surname">Castenholtz</div> <div class="given-names">R. W.</div></div>. </div>, <div class="source">Bergey's Manual of Systematic Bacteriology</div> , <div class="year">2002</div>, vol. <div class="volume">Vol. 1</div> <div class="edition">2nd Ed</div><div class="publisher-loc">Springer-Verlag</div><div class="publisher-name">New York</div></div></div></div></div><div content-id="DSN013C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mayrand</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Holt</div> <div class="given-names">S. C.</div></div>. </div><div class="article-title">Biology of asaccharolytic black-pigmented Bacteroides species</div>, <div class="source">Microbiol. Rev.</div> , <div class="year">1988</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">134</div>-<div class="lpage">152</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Biology%20of%20asaccharolytic%20black-pigmented%20Bacteroides%20species&amp;author=D.%20Mayrand&amp;author=S.%20C.%20Holt&amp;publication_year=1988&amp;journal=Microbiol.%20Rev.&amp;volume=52&amp;pages=134-152" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3280965" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lamont</div> <div class="given-names">R. J.</div></div>,  <div class="name"><div class="surname">Jenkinson</div> <div class="given-names">H. F.</div></div>. </div><div class="article-title">Life below the gum line: pathogenic mechanisms of <em>Porphyromonas gingivalis</em></div>, <div class="source">Microbiol. Mol. Biol. Rev.</div> , <div class="year">1998</div>, vol. <div class="volume">62</div> (pg. <div class="fpage">1244</div>-<div class="lpage">1263</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Life%20below%20the%20gum%20line%3A%20pathogenic%20mechanisms%20of%20Porphyromonas%20gingivalis&amp;author=R.%20J.%20Lamont&amp;author=H.%20F.%20Jenkinson&amp;publication_year=1998&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev.&amp;volume=62&amp;pages=1244-1263" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9841671" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Holt</div> <div class="given-names">S. C.</div></div>,  <div class="name"><div class="surname">Kesavalu</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Walker</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Genco</div> <div class="given-names">C. A.</div></div>. </div><div class="article-title">Virulence factors of <em>Porphyromonas gingivalis</em></div>, <div class="source">Periodontology</div> , <div class="year">1999</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">168</div>-<div class="lpage">238</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Virulence%20factors%20of%20Porphyromonas%20gingivalis&amp;author=S.%20C.%20Holt&amp;author=L.%20Kesavalu&amp;author=S.%20Walker&amp;author=C.%20A.%20Genco&amp;publication_year=1999&amp;journal=Periodontology&amp;volume=20&amp;pages=168-238" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/prd.1999.20.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fprd.1999.20.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fprd.1999.20.issue-1"> </span></div> </div></div></div></div></div><div content-id="DSN013C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Amano</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Disruption of epithelial barrier and impairment of cellular function by <em>Porphyromonas gingivalis</em></div>, <div class="source">Front. Biosci.</div> , <div class="year">2007</div>, vol. <div class="volume">12</div> (pg. <div class="fpage">3965</div>-<div class="lpage">3974</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Disruption%20of%20epithelial%20barrier%20and%20impairment%20of%20cellular%20function%20by%20Porphyromonas%20gingivalis&amp;author=A.%20Amano&amp;publication_year=2007&amp;journal=Front.%20Biosci.&amp;volume=12&amp;pages=3965-3974" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2741/2363" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2741%2F2363" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2741%2F2363"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17485350" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Umemoto</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Hamada</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Characterization of biologically active cell surface components of a periodontal pathogen: the roles of major and minor fimbriae of <em>Porphyromonas gingivalis</em></div>, <div class="source">J. Periodontol.</div> , <div class="year">2003</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">119</div>-<div class="lpage">122</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20biologically%20active%20cell%20surface%20components%20of%20a%20periodontal%20pathogen%3A%20the%20roles%20of%20major%20and%20minor%20fimbriae%20of%20Porphyromonas%20gingivalis&amp;author=T.%20Umemoto&amp;author=N.%20Hamada&amp;publication_year=2003&amp;journal=J.%20Periodontol.&amp;volume=74&amp;pages=119-122" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1902/jop.2003.74.1.119" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1902%2Fjop.2003.74.1.119" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1902%2Fjop.2003.74.1.119"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12593606" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kadowaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakayama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Okamoto</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title"><em>Porphyromonas gingivalis</em> proteinases as virulence determinants in progression of periodontal diseases</div>, <div class="source">J. Biochem.</div> , <div class="year">2000</div>, vol. <div class="volume">128</div> (pg. <div class="fpage">153</div>-<div class="lpage">159</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Porphyromonas%20gingivalis%20proteinases%20as%20virulence%20determinants%20in%20progression%20of%20periodontal%20diseases&amp;author=T.%20Kadowaki&amp;author=K.%20Nakayama&amp;author=K.%20Okamoto&amp;publication_year=2000&amp;journal=J.%20Biochem.&amp;volume=128&amp;pages=153-159" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.jbchem.a022735" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.jbchem.a022735" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.jbchem.a022735"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10920248" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Slots</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Rams</div> <div class="given-names">E. T.</div></div>. </div><div class="person-group"><div class="name"><div class="surname">Shah</div> <div class="given-names">H. N.</div></div>,  <div class="name"><div class="surname">Mayrand</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Genco</div> <div class="given-names">R. J.</div></div>. </div><div class="article-title">Pathogenicity</div>, <div class="source">Biology of the Species Porphyromonas gingivalis</div> , <div class="year">1993</div><div class="publisher-loc">Boca Raton, FL, pp</div><div class="publisher-name">CRC Press Inc</div>(pg. <div class="fpage">127</div>-<div class="lpage">131</div>)</div></div></div></div><div content-id="DSN013C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nelson</div> <div class="given-names">K. E.</div></div>,  <div class="name"><div class="surname">Fleischmann</div> <div class="given-names">R. D.</div></div>,  <div class="name"><div class="surname">DeBoy</div> <div class="given-names">R. T.</div></div>, et al. </div><div class="article-title">Complete genome sequence of the oral pathogenic Bacterium <em>Porphyromonas gingivalis</em> strain W83</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2003</div>, vol. <div class="volume">185</div> (pg. <div class="fpage">5591</div>-<div class="lpage">5601</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20the%20oral%20pathogenic%20Bacterium%20Porphyromonas%20gingivalis%20strain%20W83&amp;author=K.%20E.%20Nelson&amp;author=R.%20D.%20Fleischmann&amp;author=R.%20T.%20DeBoy&amp;publication_year=2003&amp;journal=J.%20Bacteriol.&amp;volume=185&amp;pages=5591-5601" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.185.18.5591-5601.2003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.185.18.5591-5601.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.185.18.5591-5601.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12949112" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakayama</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Rapid viability loss on exposure to air in a superoxide dismutase-deficient mutant of <em>Porphyromonas gingivalis</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">1994</div>, vol. <div class="volume">176</div> (pg. <div class="fpage">1939</div>-<div class="lpage">1943</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20viability%20loss%20on%20exposure%20to%20air%20in%20a%20superoxide%20dismutase-deficient%20mutant%20of%20Porphyromonas%20gingivalis&amp;author=K.%20Nakayama&amp;publication_year=1994&amp;journal=J.%20Bacteriol.&amp;volume=176&amp;pages=1939-1943" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8144460" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakayama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kadowaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Okamoto</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Construction and characterization of arginine-specific cysteine proteinase (Arg-gingipain)-deficient mutants of <em>Porphyromonas gingivalis</em>: evidence for significant contribution of Arg-gingipain to virulence</div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1995</div>, vol. <div class="volume">270</div> (pg. <div class="fpage">23619</div>-<div class="lpage">23626</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Construction%20and%20characterization%20of%20arginine-specific%20cysteine%20proteinase%20(Arg-gingipain)-deficient%20mutants%20of%20Porphyromonas%20gingivalis%3A%20evidence%20for%20significant%20contribution%20of%20Arg-gingipain%20to%20virulence&amp;author=K.%20Nakayama&amp;author=T.%20Kadowaki&amp;author=K.%20Okamoto&amp;author=K.%20Yamamoto&amp;publication_year=1995&amp;journal=J.%20Biol.%20Chem.&amp;volume=270&amp;pages=23619-23626" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.270.40.23619" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.270.40.23619" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.270.40.23619"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7559528" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Okamoto</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nakayama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Kadowaki</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Ratnayake</div> <div class="given-names">D. B.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Involvement of a lysine-specific cysteine proteinase in hemoglobin adsorption and heme accumulation by <em>Porphyromonas gingivalis</em></div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1998</div>, vol. <div class="volume">273</div> (pg. <div class="fpage">21225</div>-<div class="lpage">21231</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Involvement%20of%20a%20lysine-specific%20cysteine%20proteinase%20in%20hemoglobin%20adsorption%20and%20heme%20accumulation%20by%20Porphyromonas%20gingivalis&amp;author=K.%20Okamoto&amp;author=K.%20Nakayama&amp;author=T.%20Kadowaki&amp;author=N.%20Abe&amp;author=D.%20B.%20Ratnayake&amp;author=K.%20Yamamoto&amp;publication_year=1998&amp;journal=J.%20Biol.%20Chem.&amp;volume=273&amp;pages=21225-21231" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.273.33.21225" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.273.33.21225" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.273.33.21225"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9694880" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shi</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Ratnayake</div> <div class="given-names">D. B.</div></div>,  <div class="name"><div class="surname">Okamoto</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Yamamoto</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Nakayama</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Genetic analyses of proteolysis, hemoglobin binding, and hemagglutination of <em>Porphyromonas gingivalis</em>: construction of mutants with a combination of <em>rgpA</em>, <em>rgpB</em>, <em>kgp</em>, and <em>hagA</em></div>, <div class="source">J. Biol. Chem.</div> , <div class="year">1999</div>, vol. <div class="volume">274</div> (pg. <div class="fpage">17955</div>-<div class="lpage">17960</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20analyses%20of%20proteolysis%2C%20hemoglobin%20binding%2C%20and%20hemagglutination%20of%20Porphyromonas%20gingivalis%3A%20construction%20of%20mutants%20with%20a%20combination%20of%20rgpA%2C%20rgpB%2C%20kgp%2C%20and%20hagA&amp;author=Y.%20Shi&amp;author=D.%20B.%20Ratnayake&amp;author=K.%20Okamoto&amp;author=N.%20Abe&amp;author=K.%20Yamamoto&amp;author=K.%20Nakayama&amp;publication_year=1999&amp;journal=J.%20Biol.%20Chem.&amp;volume=274&amp;pages=17955-17960" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1074/jbc.274.25.17955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1074%2Fjbc.274.25.17955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1074%2Fjbc.274.25.17955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10364243" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Naito</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Sakai</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Shi</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title"><em>Porphyromonas gingivalis</em>-induced platelet aggregation in plasma depends on Hgp44 adhesin but not Rgp proteinase</div>, <div class="source">Mol. Microbiol.</div> , <div class="year">2006</div>, vol. <div class="volume">59</div> (pg. <div class="fpage">152</div>-<div class="lpage">167</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Porphyromonas%20gingivalis-induced%20platelet%20aggregation%20in%20plasma%20depends%20on%20Hgp44%20adhesin%20but%20not%20Rgp%20proteinase&amp;author=M.%20Naito&amp;author=E.%20Sakai&amp;author=Y.%20Shi&amp;publication_year=2006&amp;journal=Mol.%20Microbiol.&amp;volume=59&amp;pages=152-167" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/MMI.2006.59.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2FMMI.2006.59.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2FMMI.2006.59.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16359325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Watanabe-Kato</div> <div class="given-names">T</div></div>,  <div class="name"><div class="surname">Hayashi</div> <div class="given-names">JI</div></div>,  <div class="name"><div class="surname">Terazawa</div> <div class="given-names">Y</div></div>, et al. </div><div class="article-title">Isolation and characterization of transposon-induced mutants of <em>Porphyromonas gingivalis</em> deficient in fimbriation</div>, <div class="source">Microb. Pathog.</div> , <div class="year">1998</div>, vol. <div class="volume">24</div> (pg. <div class="fpage">25</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Isolation%20and%20characterization%20of%20transposon-induced%20mutants%20of%20Porphyromonas%20gingivalis%20deficient%20in%20fimbriation&amp;author=T%20Watanabe-Kato&amp;author=JI%20Hayashi&amp;author=Y%20Terazawa&amp;publication_year=1998&amp;journal=Microb.%20Pathog.&amp;volume=24&amp;pages=25-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/mpat.1997.0170" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fmpat.1997.0170" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fmpat.1997.0170"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9466944" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ewing</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Base-calling of automated sequencer traces using Phred. II. Error probabilities</div>, <div class="source">Genome Res.</div> , <div class="year">1998</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">186</div>-<div class="lpage">194</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Base-calling%20of%20automated%20sequencer%20traces%20using%20Phred.%20II.%20Error%20probabilities&amp;author=B.%20Ewing&amp;author=P.%20Green&amp;publication_year=1998&amp;journal=Genome%20Res.&amp;volume=8&amp;pages=186-194" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.8.3.186" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.8.3.186" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.8.3.186"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9521922" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gordon</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Desmarais</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Green</div> <div class="given-names">P.</div></div>. </div><div class="article-title">Automated finishing with autofinish</div>, <div class="source">Genome Res.</div> , <div class="year">2001</div>, vol. <div class="volume">11</div> (pg. <div class="fpage">614</div>-<div class="lpage">625</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Automated%20finishing%20with%20autofinish&amp;author=D.%20Gordon&amp;author=C.%20Desmarais&amp;author=P.%20Green&amp;publication_year=2001&amp;journal=Genome%20Res.&amp;volume=11&amp;pages=614-625" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.171401" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.171401" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.171401"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11282977" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nakayama</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Determination of the genome size of the oral anaerobic bacterium <em>Porphyromonas gingivalis</em> by pulsed field gel electrophoresis</div>, <div class="source">Dent. Jpn.</div> , <div class="year">1995</div>, vol. <div class="volume">32</div> (pg. <div class="fpage">25</div>-<div class="lpage">28</div>)</div></div></div></div><div content-id="DSN013C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sakiyama</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Takami</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Ogasawara</div> <div class="given-names">N.</div></div>, et al. </div><div class="article-title">An automated system for genome analysis to support microbial whole-genome shotgun sequencing</div>, <div class="source">Biosci. Biotechnol. Biochem.</div> , <div class="year">2000</div>, vol. <div class="volume">64</div> (pg. <div class="fpage">670</div>-<div class="lpage">673</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20automated%20system%20for%20genome%20analysis%20to%20support%20microbial%20whole-genome%20shotgun%20sequencing&amp;author=T.%20Sakiyama&amp;author=H.%20Takami&amp;author=N.%20Ogasawara&amp;publication_year=2000&amp;journal=Biosci.%20Biotechnol.%20Biochem.&amp;volume=64&amp;pages=670-673" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1271/bbb.64.670" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1271%2Fbbb.64.670" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1271%2Fbbb.64.670"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10803979" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Badger</div> <div class="given-names">J. H.</div></div>,  <div class="name"><div class="surname">Olsen</div> <div class="given-names">G. J.</div></div>. </div><div class="article-title">CRITICA: coding region identification tool invoking comparative analysis</div>, <div class="source">Mol. Biol. Evol.</div> , <div class="year">1999</div>, vol. <div class="volume">16</div> (pg. <div class="fpage">512</div>-<div class="lpage">524</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CRITICA%3A%20coding%20region%20identification%20tool%20invoking%20comparative%20analysis&amp;author=J.%20H.%20Badger&amp;author=G.%20J.%20Olsen&amp;publication_year=1999&amp;journal=Mol.%20Biol.%20Evol.&amp;volume=16&amp;pages=512-524" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/oxfordjournals.molbev.a026133" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Foxfordjournals.molbev.a026133" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Foxfordjournals.molbev.a026133"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10331277" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yada</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Nakao</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Totoki</div> <div class="given-names">Y.</div></div>,  <div class="name"><div class="surname">Nakai</div> <div class="given-names">K.</div></div>. </div><div class="article-title">Modeling and predicting transcriptional units of <em>Escherichia coli</em> genes using hidden Markov models</div>, <div class="source">Bioinformatics</div> , <div class="year">1999</div>, vol. <div class="volume">15</div> (pg. <div class="fpage">987</div>-<div class="lpage">993</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Modeling%20and%20predicting%20transcriptional%20units%20of%20Escherichia%20coli%20genes%20using%20hidden%20Markov%20models&amp;author=T.%20Yada&amp;author=M.%20Nakao&amp;author=Y.%20Totoki&amp;author=K.%20Nakai&amp;publication_year=1999&amp;journal=Bioinformatics&amp;volume=15&amp;pages=987-993" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/15.12.987" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2F15.12.987" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2F15.12.987"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10745988" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Frishman</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Mironov</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Mewes</div> <div class="given-names">H. W.</div></div>,  <div class="name"><div class="surname">Gelfand</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Combining diverse evidence for gene recognition in completely sequenced bacterial genomes</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1998</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">2941</div>-<div class="lpage">2947</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Combining%20diverse%20evidence%20for%20gene%20recognition%20in%20completely%20sequenced%20bacterial%20genomes&amp;author=D.%20Frishman&amp;author=A.%20Mironov&amp;author=H.%20W.%20Mewes&amp;author=M.%20Gelfand&amp;publication_year=1998&amp;journal=Nucleic%20Acids%20Res.&amp;volume=26&amp;pages=2941-2947" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/26.12.2941" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F26.12.2941" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F26.12.2941"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9611239" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Altschul</div> <div class="given-names">S. F.</div></div>,  <div class="name"><div class="surname">Madden</div> <div class="given-names">T. L.</div></div>,  <div class="name"><div class="surname">Schaffer</div> <div class="given-names">A. A.</div></div>, et al. </div><div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">3389</div>-<div class="lpage">3402</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.%20F.%20Altschul&amp;author=T.%20L.%20Madden&amp;author=A.%20A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-3402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lowe</div> <div class="given-names">T. M.</div></div>,  <div class="name"><div class="surname">Eddy</div> <div class="given-names">S. R.</div></div>. </div><div class="article-title">tRNAscan-SE: a program for improved detection of transfer RNA genes in genomic sequence</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">1997</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">955</div>-<div class="lpage">964</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=tRNAscan-SE%3A%20a%20program%20for%20improved%20detection%20of%20transfer%20RNA%20genes%20in%20genomic%20sequence&amp;author=T.%20M.%20Lowe&amp;author=S.%20R.%20Eddy&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=955-964" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.5.0955" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.5.0955" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.5.0955"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9023104" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Delcher</div> <div class="given-names">A. L.</div></div>,  <div class="name"><div class="surname">Phillippy</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Carlton</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Salzberg</div> <div class="given-names">S. L.</div></div>. </div><div class="article-title">Fast algorithms for large-scale genome alignment and comparison</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2002</div>, vol. <div class="volume">30</div> (pg. <div class="fpage">2478</div>-<div class="lpage">2483</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20algorithms%20for%20large-scale%20genome%20alignment%20and%20comparison&amp;author=A.%20L.%20Delcher&amp;author=A.%20Phillippy&amp;author=J.%20Carlton&amp;author=S.%20L.%20Salzberg&amp;publication_year=2002&amp;journal=Nucleic%20Acids%20Res.&amp;volume=30&amp;pages=2478-2483" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/30.11.2478" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F30.11.2478" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F30.11.2478"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12034836" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schwartz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Frazer</div> <div class="given-names">K. A.</div></div>, et al. </div><div class="article-title">PipMaker—a web server for aligning two genomic DNA sequences</div>, <div class="source">Genome Res.</div> , <div class="year">2000</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">577</div>-<div class="lpage">586</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PipMaker%E2%80%94a%20web%20server%20for%20aligning%20two%20genomic%20DNA%20sequences&amp;author=S.%20Schwartz&amp;author=Z.%20Zhang&amp;author=K.%20A.%20Frazer&amp;publication_year=2000&amp;journal=Genome%20Res.&amp;volume=10&amp;pages=577-586" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.10.4.577" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.10.4.577" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.10.4.577"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10779500" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pfaffl</div> <div class="given-names">M. W.</div></div>. </div><div class="article-title">A new mathematical model for relative quantification in real-time RT–PCR</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2001</div>, vol. <div class="volume">29</div> </div></div></div></div><div content-id="DSN013C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bacic</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Parker</div> <div class="given-names">A. C.</div></div>,  <div class="name"><div class="surname">Stagg</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Genetic and structural analysis of the <em>Bacteroides</em> conjugative transposon CTn341</div>, <div class="source">J. Bacteriol.</div> , <div class="year">2005</div>, vol. <div class="volume">187</div> (pg. <div class="fpage">2858</div>-<div class="lpage">2869</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20and%20structural%20analysis%20of%20the%20Bacteroides%20conjugative%20transposon%20CTn341&amp;author=M.%20Bacic&amp;author=A.%20C.%20Parker&amp;author=J.%20Stagg&amp;publication_year=2005&amp;journal=J.%20Bacteriol.&amp;volume=187&amp;pages=2858-2869" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.187.8.2858-2869.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.187.8.2858-2869.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.187.8.2858-2869.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15805532" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bonheyo</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Graham</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Shoemaker</div> <div class="given-names">N. B.</div></div>,  <div class="name"><div class="surname">Salyers</div> <div class="given-names">A. A.</div></div>. </div><div class="article-title">Transfer region of a bacteroides conjugative transposon, CTnDOT</div>, <div class="source">Plasmid</div> , <div class="year">2001</div>, vol. <div class="volume">45</div> (pg. <div class="fpage">41</div>-<div class="lpage">51</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transfer%20region%20of%20a%20bacteroides%20conjugative%20transposon%2C%20CTnDOT&amp;author=G.%20Bonheyo&amp;author=D.%20Graham&amp;author=N.%20B.%20Shoemaker&amp;author=A.%20A.%20Salyers&amp;publication_year=2001&amp;journal=Plasmid&amp;volume=45&amp;pages=41-51" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/plas.2000.1495" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fplas.2000.1495" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fplas.2000.1495"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11319931" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kuwahara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Yamashita</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Hirakawa</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Genomic analysis of <em>Bacteroides fragilis</em> reveals extensive DNA inversions regulating cell surface adaptation</div>, <div class="source">Proc. Natl. Acad. Sci. USA</div> , <div class="year">2004</div>, vol. <div class="volume">101</div> (pg. <div class="fpage">14919</div>-<div class="lpage">14924</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20analysis%20of%20Bacteroides%20fragilis%20reveals%20extensive%20DNA%20inversions%20regulating%20cell%20surface%20adaptation&amp;author=T.%20Kuwahara&amp;author=A.%20Yamashita&amp;author=H.%20Hirakawa&amp;publication_year=2004&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=101&amp;pages=14919-14924" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0404172101" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0404172101" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0404172101"> </span></div> </div></div></div></div></div><div content-id="DSN013C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Feschotte</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Jiang</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Wessler</div> <div class="given-names">S. R.</div></div>. </div><div class="article-title">Plant transposable elements: where genetics meets genomics</div>, <div class="source">Nat. Rev. Genet.</div> , <div class="year">2002</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">329</div>-<div class="lpage">341</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20transposable%20elements%3A%20where%20genetics%20meets%20genomics&amp;author=C.%20Feschotte&amp;author=N.%20Jiang&amp;author=S.%20R.%20Wessler&amp;publication_year=2002&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=3&amp;pages=329-341" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg793" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg793" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg793"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11988759" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dong</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Dewhirst</div> <div class="given-names">F. E.</div></div>,  <div class="name"><div class="surname">Fleischmann</div> <div class="given-names">R. D.</div></div>,  <div class="name"><div class="surname">Fraser</div> <div class="given-names">C. M.</div></div>,  <div class="name"><div class="surname">Duncan</div> <div class="given-names">M. J.</div></div>. </div><div class="article-title">Genomic loci of the <em>Porphyromonas gingivalis</em> insertion element IS1126</div>, <div class="source">Infect. Immun.</div> , <div class="year">1999</div>, vol. <div class="volume">67</div> (pg. <div class="fpage">3416</div>-<div class="lpage">3423</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20loci%20of%20the%20Porphyromonas%20gingivalis%20insertion%20element%20IS1126&amp;author=H.%20Dong&amp;author=T.%20Chen&amp;author=F.%20E.%20Dewhirst&amp;author=R.%20D.%20Fleischmann&amp;author=C.%20M.%20Fraser&amp;author=M.%20J.%20Duncan&amp;publication_year=1999&amp;journal=Infect.%20Immun.&amp;volume=67&amp;pages=3416-3423" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10377121" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Aduse-Opoku</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Slaney</div> <div class="given-names">J. M.</div></div>,  <div class="name"><div class="surname">Hashim</div> <div class="given-names">A.</div></div>, et al. </div><div class="article-title">Identification and characterization of the capsular (K-antigen) locus of <em>Porphyromonas gingivalis</em></div>, <div class="source">Infect. Immun.</div> , <div class="year">2006</div>, vol. <div class="volume">74</div> (pg. <div class="fpage">449</div>-<div class="lpage">460</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20the%20capsular%20(K-antigen)%20locus%20of%20Porphyromonas%20gingivalis&amp;author=J.%20Aduse-Opoku&amp;author=J.%20M.%20Slaney&amp;author=A.%20Hashim&amp;publication_year=2006&amp;journal=Infect.%20Immun.&amp;volume=74&amp;pages=449-460" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.74.1.449-460.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.74.1.449-460.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.74.1.449-460.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16369001" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Fujiwara</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Morishima</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Takahashi</div> <div class="given-names">I.</div></div>,  <div class="name"><div class="surname">Hamada</div> <div class="given-names">S.</div></div>. </div><div class="article-title">Molecular cloning and sequencing of the fimbrilin gene of <em>Porphyromonas gingivalis</em> strains and characterization of recombinant proteins</div>, <div class="source">Biochem. Biophys. Res. Commun.</div> , <div class="year">1993</div>, vol. <div class="volume">197</div> (pg. <div class="fpage">241</div>-<div class="lpage">247</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%20and%20sequencing%20of%20the%20fimbrilin%20gene%20of%20Porphyromonas%20gingivalis%20strains%20and%20characterization%20of%20recombinant%20proteins&amp;author=T.%20Fujiwara&amp;author=S.%20Morishima&amp;author=I.%20Takahashi&amp;author=S.%20Hamada&amp;publication_year=1993&amp;journal=Biochem.%20Biophys.%20Res.%20Commun.&amp;volume=197&amp;pages=241-247" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1006/bbrc.1993.2467" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1006%2Fbbrc.1993.2467" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1006%2Fbbrc.1993.2467"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7902712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hall</div> <div class="given-names">L. M.</div></div>,  <div class="name"><div class="surname">Fawell</div> <div class="given-names">S. C.</div></div>,  <div class="name"><div class="surname">Shi</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Sequence diversity and antigenic variation at the rag locus of <em>Porphyromonas gingivalis</em></div>, <div class="source">Infect. Immun.</div> , <div class="year">2005</div>, vol. <div class="volume">73</div> (pg. <div class="fpage">4253</div>-<div class="lpage">4262</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequence%20diversity%20and%20antigenic%20variation%20at%20the%20rag%20locus%20of%20Porphyromonas%20gingivalis&amp;author=L.%20M.%20Hall&amp;author=S.%20C.%20Fawell&amp;author=X.%20Shi&amp;publication_year=2005&amp;journal=Infect.%20Immun.&amp;volume=73&amp;pages=4253-4262" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.73.7.4253-4262.2005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.73.7.4253-4262.2005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.73.7.4253-4262.2005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15972517" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cerdeno-Tarraga</div> <div class="given-names">A. M.</div></div>,  <div class="name"><div class="surname">Patrick</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Crossman</div> <div class="given-names">L. C.</div></div>, et al. </div><div class="article-title">Extensive DNA inversions in the <em>B. fragilis</em> genome control variable gene expression</div>, <div class="source">Science</div> , <div class="year">2005</div>, vol. <div class="volume">307</div> (pg. <div class="fpage">1463</div>-<div class="lpage">1465</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Extensive%20DNA%20inversions%20in%20the%20B.%20fragilis%20genome%20control%20variable%20gene%20expression&amp;author=A.%20M.%20Cerdeno-Tarraga&amp;author=S.%20Patrick&amp;author=L.%20C.%20Crossman&amp;publication_year=2005&amp;journal=Science&amp;volume=307&amp;pages=1463-1465" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1107008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1107008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1107008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15746427" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nie</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">F.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>, et al. </div><div class="article-title">Complete genome sequence of <em>Shigella flexneri</em> 5b and comparison with <em>Shigella flexneri</em> 2a</div>, <div class="source">BMC Genomics</div> , <div class="year">2006</div>, vol. <div class="volume">7</div> pg. <div class="fpage">173</div> <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20of%20Shigella%20flexneri%205b%20and%20comparison%20with%20Shigella%20flexneri%202a&amp;author=H.%20Nie&amp;author=F.%20Yang&amp;author=X.%20Zhang&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=173" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-173" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-173" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-173"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16822325" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wei</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Goldberg</div> <div class="given-names">M. B.</div></div>,  <div class="name"><div class="surname">Burland</div> <div class="given-names">V.</div></div>, et al. </div><div class="article-title">Complete genome sequence and comparative genomics of <em>Shigella flexneri</em> serotype 2a strain 2457T</div>, <div class="source">Infect. Immun.</div> , <div class="year">2003</div>, vol. <div class="volume">71</div> (pg. <div class="fpage">2775</div>-<div class="lpage">2786</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Complete%20genome%20sequence%20and%20comparative%20genomics%20of%20Shigella%20flexneri%20serotype%202a%20strain%202457T&amp;author=J.%20Wei&amp;author=M.%20B.%20Goldberg&amp;author=V.%20Burland&amp;publication_year=2003&amp;journal=Infect.%20Immun.&amp;volume=71&amp;pages=2775-2786" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.71.5.2775-2786.2003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.71.5.2775-2786.2003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.71.5.2775-2786.2003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12704152" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSN013C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSN013C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Diaz</div> <div class="given-names">P. I.</div></div>,  <div class="name"><div class="surname">Slakeski</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Reynolds</div> <div class="given-names">E. C.</div></div>,  <div class="name"><div class="surname">Morona</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Rogers</div> <div class="given-names">A. H.</div></div>,  <div class="name"><div class="surname">Kolenbrander</div> <div class="given-names">P. E.</div></div>. </div><div class="article-title">Role of <em>oxyR</em> in the oral anaerobe <em>Porphyromonas gingivalis</em></div>, <div class="source">J. Bacteriol.</div> , <div class="year">2006</div>, vol. <div class="volume">188</div> (pg. <div class="fpage">2454</div>-<div class="lpage">2462</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Role%20of%20oxyR%20in%20the%20oral%20anaerobe%20Porphyromonas%20gingivalis&amp;author=P.%20I.%20Diaz&amp;author=N.%20Slakeski&amp;author=E.%20C.%20Reynolds&amp;author=R.%20Morona&amp;author=A.%20H.%20Rogers&amp;author=P.%20E.%20Kolenbrander&amp;publication_year=2006&amp;journal=J.%20Bacteriol.&amp;volume=188&amp;pages=2454-2462" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.188.7.2454-2462.2006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.188.7.2454-2462.2006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.188.7.2454-2462.2006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16547032" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2008. Kazusa DNA Research Institute</div><div class="license"><div class="license-p">The online version of this article has been published under an open access model. Users are entitled to use, reproduce, disseminate, or display the open access version of this article for non-commercial purposes provided that: the original authorship is properly and fully attributed; the Journal and Oxford University Press are attributed as the original place of publication with the correct citation details given; if an article is subsequently reproduced or disseminated not in its entirety but only in part or as a derivative work this must be clearly indicated. For commercial re-use, please contact journals.permissions@oxfordjournals.org</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=376506&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/15/4/10.1093/dnares/dsn013/2/dsn013-dsn013supp.pdf?Expires=2147483647&amp;Signature=u720ndOfh29i2MLnPZCvIkiYZeyNO91MO-7g59cfRfaNjf05Q5YZonIxGb5vt1WloD404ZKmkTuATEj8HF0UdgkGKp3AMKqn~Q9VXLdkFrRADNWWFu84vzZrwbRt99L198BYH-CAURmfYPBrNunSdBWX9Y-49WYHpKqHRRdI9jrPEe1~UWBz5sVX8UXcr9LcgRK100HMt8ovgAZ9T8X4cKy3y25TzGmPPgg-bku8YnQbgSUEYb7xg~OkuGZhiLGE~ztLlmnTBwvotnEIUwiuAlKUlcbt9ygApJT8QG0Qc4GjXVERK6upy82PQmPzvH6g48UXxGxS6fvEqv-NELO09A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">726</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260564600004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">143</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsn013"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="376506"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000260564600004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f15%2f4%2f215%2f376506">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000260564600004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (143)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F15%2F4%2F215%2F376506" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/376506'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Determination of the Genome Sequence of Porphyromonas gingivalis Strain ATCC 33277 and Genomic Comparison with Strain W83 Revealed Extensive Genome Rearrangements in P. gingivalis" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsn013");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f15%2f4%2f215%2f376506">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

