

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/5/459/3778237","@type":"ScholarlyArticle","name":"Short and long-term evolutionary dynamics of subtelomeric piRNA clusters in  Drosophila ","datePublished":"2017-04-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/5","@type":"PublicationIssue","issueNumber":"5","datePublished":"2017-10-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx017","keywords":["subtelomeres","TAS","<italic>Drosophila</italic>","piRNA","transposable element"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/5/459/3778237","author":[{"name":"Asif-Laidin, Amna","affiliation":"1 Sorbonne Universités, UPMC University of Paris 06, CNRS, Biologie du Développement Paris-Seine, Institut de Biologie Paris-Seine (LBD-IBPS), 75005 Paris, France","@type":"Person"},{"name":"Delmarre, Valérie","affiliation":"1 Sorbonne Universités, UPMC University of Paris 06, CNRS, Biologie du Développement Paris-Seine, Institut de Biologie Paris-Seine (LBD-IBPS), 75005 Paris, France","@type":"Person"},{"name":"Laurentie, Jeanne","affiliation":"1 Sorbonne Universités, UPMC University of Paris 06, CNRS, Biologie du Développement Paris-Seine, Institut de Biologie Paris-Seine (LBD-IBPS), 75005 Paris, France","@type":"Person"},{"name":"Miller, Wolfgang J.","affiliation":"2 Lab Genome Dynamics, Department for Cell &amp; Developmental Biology, Center of Anatomy and Cell Biology, Medical University of Vienna, A-1090 Vienna, Austria","@type":"Person"},{"name":"Ronsseray, Stéphane","affiliation":"1 Sorbonne Universités, UPMC University of Paris 06, CNRS, Biologie du Développement Paris-Seine, Institut de Biologie Paris-Seine (LBD-IBPS), 75005 Paris, France","@type":"Person"},{"name":"Teysset, Laure","affiliation":"1 Sorbonne Universités, UPMC University of Paris 06, CNRS, Biologie du Développement Paris-Seine, Institut de Biologie Paris-Seine (LBD-IBPS), 75005 Paris, France","@type":"Person"}],"description":"Abstract.  Two Telomeric Associated Sequences, TAS-R and TAS-L, form the principal subtelomeric repeat families identified in Drosophila melanogaster. They are ","pageStart":"459","pageEnd":"472","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=2147483647&Signature=jul3oxjgkP675GHMbQEy~zZ1my2m2g0xI0mr9LkJEjA-PhoC4BnM1ARo9FdXFQKdaeXjXdDG8Th7l01-PuT824GoN40EPQF0GwZOV5tGDYZCguawjvcFtz41hVQe~O6T29XL-3ITGszkj0QvmRZLajQvyRnp825zpQTjNCOtAlB0rKMc7Dcdixzjc2uv2jilEDMqD9YZCuMkOXRLrwDQftwYgb~q8EavwLngcuRRSkILOW9lFSQBCUNAqNd4mDnFW5RKty7z4w3HYGKz5wJomnNLGyCwwAyELyFWHWV4xtKHzE5xEUd8do1ZdHc6pZe4TftEycCVzz1M4dnVR5tpug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Short and long-term evolutionary dynamics of subtelomeric piRNA clusters in  Drosophila ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=2147483647&Signature=jul3oxjgkP675GHMbQEy~zZ1my2m2g0xI0mr9LkJEjA-PhoC4BnM1ARo9FdXFQKdaeXjXdDG8Th7l01-PuT824GoN40EPQF0GwZOV5tGDYZCguawjvcFtz41hVQe~O6T29XL-3ITGszkj0QvmRZLajQvyRnp825zpQTjNCOtAlB0rKMc7Dcdixzjc2uv2jilEDMqD9YZCuMkOXRLrwDQftwYgb~q8EavwLngcuRRSkILOW9lFSQBCUNAqNd4mDnFW5RKty7z4w3HYGKz5wJomnNLGyCwwAyELyFWHWV4xtKHzE5xEUd8do1ZdHc6pZe4TftEycCVzz1M4dnVR5tpug__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Short and long-term evolutionary dynamics of subtelomeric piRNA clusters in  Drosophila " />
    <meta property="og:description" content="Abstract.  Two Telomeric Associated Sequences, TAS-R and TAS-L, form the principal subtelomeric repeat families identified in Drosophila melanogaster. They are " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/5/459/3778237" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-04-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Asif-Laidin, Amna" /><meta name="citation_author_institution" content="Sorbonne Universités, UPMC University of Paris 06, CNRS, Biologie du Développement Paris-Seine, Institut de Biologie Paris-Seine (LBD-IBPS), 75005 Paris, France" /><meta name="citation_author" content="Delmarre, Valérie" /><meta name="citation_author_institution" content="Sorbonne Universités, UPMC University of Paris 06, CNRS, Biologie du Développement Paris-Seine, Institut de Biologie Paris-Seine (LBD-IBPS), 75005 Paris, France" /><meta name="citation_author" content="Laurentie, Jeanne" /><meta name="citation_author_institution" content="Sorbonne Universités, UPMC University of Paris 06, CNRS, Biologie du Développement Paris-Seine, Institut de Biologie Paris-Seine (LBD-IBPS), 75005 Paris, France" /><meta name="citation_author" content="Miller, Wolfgang J." /><meta name="citation_author_institution" content="Lab Genome Dynamics, Department for Cell &amp; Developmental Biology, Center of Anatomy and Cell Biology, Medical University of Vienna, A-1090 Vienna, Austria" /><meta name="citation_author" content="Ronsseray, Stéphane" /><meta name="citation_author_institution" content="Sorbonne Universités, UPMC University of Paris 06, CNRS, Biologie du Développement Paris-Seine, Institut de Biologie Paris-Seine (LBD-IBPS), 75005 Paris, France" /><meta name="citation_author" content="Teysset, Laure" /><meta name="citation_author_institution" content="Sorbonne Universités, UPMC University of Paris 06, CNRS, Biologie du Développement Paris-Seine, Institut de Biologie Paris-Seine (LBD-IBPS), 75005 Paris, France" /><meta name="citation_title" content="Short and long-term evolutionary dynamics of subtelomeric piRNA clusters in Drosophila" /><meta name="citation_firstpage" content="459" /><meta name="citation_lastpage" content="472" /><meta name="citation_doi" content="10.1093/dnares/dsx017" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="5" /><meta name="citation_publication_date" content="2017/10/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Analysis of subtelomeric heterochromatin in the Drosophila minichromosome Dp1187 by single P element insertional mutagenesis;  citation_journal_title=Genetics;  citation_year=1992;  citation_volume=132;  citation_pages=737-753; " /><meta name="citation_reference" content="citation_title=Transposons in place of telomeric repeats at a Drosophila telomere;  citation_journal_title=Cell;  citation_year=1993;  citation_volume=75;  citation_pages=1083-1093; " /><meta name="citation_reference" content="citation_title=DNA organization and polymorphism of a wild-type Drosophila telomere region;  citation_journal_title=Chromosoma;  citation_year=1995;  citation_volume=104;  citation_pages=229-241; " /><meta name="citation_reference" content="citation_title=Genomic analysis of Drosophila melanogaster telomeres: full-length copies of HeT-A and TART elements at telomeres;  citation_journal_title=Mol. Biol. Evol;  citation_year=2004;  citation_volume=21;  citation_pages=1613-1619; " /><meta name="citation_reference" content="  Bergman C.M., Quesneville H., Anxolabéhère D., Ashburner M.  2006 , Recurrent insertion and duplication generate networks of transposable element sequences in the Drosophila melanogaster genome. Genome Biol., 7 , R112. " /><meta name="citation_reference" content="citation_title=Viable deletions of a telomere from a Drosophila chromosome;  citation_journal_title=Cell;  citation_year=1989;  citation_volume=58;  citation_pages=791-801; " /><meta name="citation_reference" content="citation_title=An epigenetic activation role of Piwi and a Piwi-associated piRNA in Drosophila melanogaster;  citation_journal_title=Nature;  citation_year=2007;  citation_volume=450;  citation_pages=304-308; " /><meta name="citation_reference" content="citation_title=The JIL-1 kinase affects telomere expression in the different telomere domains of Drosophila;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e81543" /><meta name="citation_reference" content="citation_publisher=Springer, Berlin;  citation_title=Subtelomeres;  citation_year=2014;  citation_pages=211-225; " /><meta name="citation_reference" content="citation_title=Targeting of P-element reporters to heterochromatic domains by transposable element 1360 in Drosophila melanogaster;  citation_journal_title=Genetics;  citation_year=2016;  citation_volume=202;  citation_pages=565-582; " /><meta name="citation_reference" content="citation_title=Molecular characteristics of diverse populations are consistent with the hypothesis of a recent invasion of Drosophila melanogaster by mobile P elements;  citation_journal_title=Mol. Biol. Evol;  citation_year=1988;  citation_volume=5;  citation_pages=252-269; " /><meta name="citation_reference" content="citation_title=Copy number and distribution of P and I mobile elements in Drosophila melanogaster populations;  citation_journal_title=Chromosoma;  citation_year=1989;  citation_volume=98;  citation_pages=207-214; " /><meta name="citation_reference" content="citation_title=The maternally inherited regulation of P elements in Drosophila melanogaster can be elicited by two P copies at cytological site 1A on the X chromosome;  citation_journal_title=Genetics;  citation_year=1991;  citation_volume=129;  citation_pages=501-512; " /><meta name="citation_reference" content="citation_title=Telomeric trans-silencing: an epigenetic repression combining RNA silencing and heterochromatin formation;  citation_journal_title=PLoS Genet;  citation_year=2007;  citation_volume=3;  citation_pages=1633-1643; " /><meta name="citation_reference" content="citation_title=Trans-silencing by P elements inserted in subtelomeric heterochromatin involves the Drosophila Polycomb group gene, Enhancer of zeste;  citation_journal_title=Genetics;  citation_year=1998;  citation_volume=149;  citation_pages=1839-1855; " /><meta name="citation_reference" content="citation_title=Telomeric transgenes and trans-silencing in Drosophila;  citation_journal_title=Genetica;  citation_year=2003;  citation_volume=117;  citation_pages=327-335; " /><meta name="citation_reference" content="citation_title=Discrete small RNA-generating loci as master regulators of transposon activity in Drosophila;  citation_journal_title=Cell;  citation_year=2007;  citation_volume=128;  citation_pages=1089-1103; " /><meta name="citation_reference" content="citation_title=One loop to rule them all: the ping-pong cycle and piRNA-guided silencing;  citation_journal_title=Trends Biochem. Sci;  citation_year=2016;  citation_volume=41;  citation_pages=324-337; " /><meta name="citation_reference" content="citation_title=Telomeric trans-silencing in Drosophila melanogaster: tissue specificity, development and functional interactions between non-homologous telomeres;  citation_journal_title=PLoS One;  citation_year=2008;  citation_volume=3;  citation_pages=e3249" /><meta name="citation_reference" content="citation_title=An epigenetic role for maternally inherited piRNAs in transposon silencing;  citation_journal_title=Science;  citation_year=2008;  citation_volume=322;  citation_pages=1387-1392; " /><meta name="citation_reference" content="citation_title=The epigenetic trans-silencing effect in Drosophila involves maternally-transmitted small RNAs whose production depends on the piRNA pathway and HP1;  citation_journal_title=PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e11032" /><meta name="citation_reference" content="citation_title=Paramutation in Drosophila linked to emergence of a piRNA-producing locus;  citation_journal_title=Nature;  citation_year=2012;  citation_volume=490;  citation_pages=112-115; " /><meta name="citation_reference" content="  Muerdter F., Olovnikov I., Molaro A. , 2012 , Production of artificial piRNAs in flies and mice. RNA , 18 , 42 – 52. " /><meta name="citation_reference" content="citation_title=Detection in situ of genomic regulatory elements in Drosophila;  citation_journal_title=Proc. Natl Acad. Sci. U. S. A;  citation_year=1987;  citation_volume=84;  citation_pages=9123-9127; " /><meta name="citation_reference" content="citation_title=P-Element repression in Drosophila melanogaster by a naturally occurring defective telomeric P copy;  citation_journal_title=Genetics;  citation_year=2000;  citation_volume=155;  citation_pages=1841-1854; " /><meta name="citation_reference" content="citation_title=From first base: the sequence of the tip of the X chromosome of Drosophila melanogaster, a comparison of two sequencing strategies;  citation_journal_title=Genome Res;  citation_year=2001;  citation_volume=11;  citation_pages=710-730; " /><meta name="citation_reference" content="citation_title=Wanderings of hobo: a transposon in Drosophila melanogaster and its close relatives;  citation_journal_title=Genetica;  citation_year=2004;  citation_volume=120;  citation_pages=125-136; " /><meta name="citation_reference" content="citation_title=The genome sequence of Drosophila melanogaster;  citation_journal_title=Science;  citation_year=2000;  citation_volume=287;  citation_pages=2185-2195; " /><meta name="citation_reference" content="citation_title=SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building;  citation_journal_title=Mol. Biol. Evol;  citation_year=2010;  citation_volume=27;  citation_pages=221-224; " /><meta name="citation_reference" content="citation_title=Paramutation in Drosophila requires both nuclear and cytoplasmic actors of the piRNA pathway and induces Cis-spreading of piRNA production;  citation_journal_title=Genetics;  citation_year=2015;  citation_volume=201;  citation_pages=1381-1396; " /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome;  citation_journal_title=Genome Biol;  citation_year=2009;  citation_volume=10;  citation_pages=R25" /><meta name="citation_reference" content="citation_title=Three distinct chromatin domains in telomere ends of polytene chromosomes in Drosophila melanogaster Tel mutants;  citation_journal_title=J. Cell Sci;  citation_year=2005;  citation_volume=118;  citation_pages=5465-5477; " /><meta name="citation_reference" content="citation_title=Molecular population genetics of Drosophila subtelomeric DNA;  citation_journal_title=Genetics;  citation_year=2008;  citation_volume=178;  citation_pages=477-487; " /><meta name="citation_reference" content="citation_title=Non-random disjunction in Drosophila;  citation_journal_title=Genetics;  citation_year=1951;  citation_volume=36;  citation_pages=267-280; " /><meta name="citation_reference" content="citation_title=Recurrent deletion and gene presence/absence polymorphism: telomere dynamics dominate evolution at the tip of 3L in Drosophila melanogaster and D. simulans;  citation_journal_title=Genetics;  citation_year=2008;  citation_volume=179;  citation_pages=1021-1027; " /><meta name="citation_reference" content="citation_title=The complex structure and dynamic evolution of human subtelomeres;  citation_journal_title=Nat. Rev. Genet;  citation_year=2002;  citation_volume=3;  citation_pages=91-102; " /><meta name="citation_reference" content="citation_title=The genetic analysis of meiosis in female Drosophila melanogaster;  citation_journal_title=Philos. Trans. R. Soc. Lond. B Biol. Sci;  citation_year=1977;  citation_volume=277;  citation_pages=295-312; " /><meta name="citation_reference" content="citation_title=Molecular cloning of lethal(2)giant larvae, a recessive oncogene of Drosophila melanogaster;  citation_journal_title=EMBO J;  citation_year=1985;  citation_volume=4;  citation_pages=1551-1557; " /><meta name="citation_reference" content="citation_title=Turning junk into gold: domestication of transposable elements and the creation of new genes in eukaryotes;  citation_journal_title=Bioessays;  citation_year=2006;  citation_volume=28;  citation_pages=913-922; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/5/459/22892915/dsx017.pdf" /><meta name="description" content="Abstract.  Two Telomeric Associated Sequences, TAS-R and TAS-L, form the principal subtelomeric repeat families identified in Drosophila melanogaster. They are " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/5/459/3778237" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/5/459/3778237" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/5/459/3778237");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx017';
    </script>
<title>
	Short and long-term evolutionary dynamics of subtelomeric piRNA clusters in Drosophila | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3778237" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bb1jPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJBc2lmLUxhaWRpbiwgQW1uYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNvcmJvbm5lIFVuaXZlcnNpdMOpcywgVVBNQyBVbml2ZXJzaXR5IG9mIFBhcmlzIDA2LCBDTlJTLCBCaW9sb2dpZSBkdSBEw6l2ZWxvcHBlbWVudCBQYXJpcy1TZWluZSwgSW5zdGl0dXQgZGUgQmlvbG9naWUgUGFyaXMtU2VpbmUgKExCRC1JQlBTKSwgNzUwMDUgUGFyaXMsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkRlbG1hcnJlLCBWYWzDqXJpZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNvcmJvbm5lIFVuaXZlcnNpdMOpcywgVVBNQyBVbml2ZXJzaXR5IG9mIFBhcmlzIDA2LCBDTlJTLCBCaW9sb2dpZSBkdSBEw6l2ZWxvcHBlbWVudCBQYXJpcy1TZWluZSwgSW5zdGl0dXQgZGUgQmlvbG9naWUgUGFyaXMtU2VpbmUgKExCRC1JQlBTKSwgNzUwMDUgUGFyaXMsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkxhdXJlbnRpZSwgSmVhbm5lIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iU29yYm9ubmUgVW5pdmVyc2l0w6lzLCBVUE1DIFVuaXZlcnNpdHkgb2YgUGFyaXMgMDYsIENOUlMsIEJpb2xvZ2llIGR1IETDqXZlbG9wcGVtZW50IFBhcmlzLVNlaW5lLCBJbnN0aXR1dCBkZSBCaW9sb2dpZSBQYXJpcy1TZWluZSAoTEJELUlCUFMpLCA3NTAwNSBQYXJpcywgRnJhbmNlIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWlsbGVyLCBXb2xmZ2FuZyBKLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkxhYiBHZW5vbWUgRHluYW1pY3MsIERlcGFydG1lbnQgZm9yIENlbGwgJmFtcDsgRGV2ZWxvcG1lbnRhbCBCaW9sb2d5LCBDZW50ZXIgb2YgQW5hdG9teSBhbmQgQ2VsbCBCaW9sb2d5LCBNZWRpY2FsIFVuaXZlcnNpdHkgb2YgVmllbm5hLCBBLTEwOTAgVmllbm5hLCBBdXN0cmlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUm9uc3NlcmF5LCBTdMOpcGhhbmUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJTb3Jib25uZSBVbml2ZXJzaXTDqXMsIFVQTUMgVW5pdmVyc2l0eSBvZiBQYXJpcyAwNiwgQ05SUywgQmlvbG9naWUgZHUgRMOpdmVsb3BwZW1lbnQgUGFyaXMtU2VpbmUsIEluc3RpdHV0IGRlIEJpb2xvZ2llIFBhcmlzLVNlaW5lIChMQkQtSUJQUyksIDc1MDA1IFBhcmlzLCBGcmFuY2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUZXlzc2V0LCBMYXVyZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IlNvcmJvbm5lIFVuaXZlcnNpdMOpcywgVVBNQyBVbml2ZXJzaXR5IG9mIFBhcmlzIDA2LCBDTlJTLCBCaW9sb2dpZSBkdSBEw6l2ZWxvcHBlbWVudCBQYXJpcy1TZWluZSwgSW5zdGl0dXQgZGUgQmlvbG9naWUgUGFyaXMtU2VpbmUgKExCRC1JQlBTKSwgNzUwMDUgUGFyaXMsIEZyYW5jZSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl90aXRsZSIgY29udGVudD0iU2hvcnQgYW5kIGxvbmctdGVybSBldm9sdXRpb25hcnkgZHluYW1pY3Mgb2Ygc3VidGVsb21lcmljIHBpUk5BIGNsdXN0ZXJzIGluIERyb3NvcGhpbGEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI0NTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fbGFzdHBhZ2UiIGNvbnRlbnQ9IjQ3MiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9kb2kiIGNvbnRlbnQ9IjEwLjEwOTMvZG5hcmVzL2RzeDAxNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX3RpdGxlIiBjb250ZW50PSJETkEgUmVzZWFyY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF9hYmJyZXYiIGNvbnRlbnQ9IkROQSBSZXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdm9sdW1lIiBjb250ZW50PSIyNCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3N1ZSIgY29udGVudD0iNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaWNhdGlvbl9kYXRlIiBjb250ZW50PSIyMDE3LzEwLzAxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc24iIGNvbnRlbnQ9IjEzNDAtMjgzOCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wdWJsaXNoZXIiIGNvbnRlbnQ9Ik94Zm9yZCBVbml2ZXJzaXR5IFByZXNzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW5hbHlzaXMgb2Ygc3VidGVsb21lcmljIGhldGVyb2Nocm9tYXRpbiBpbiB0aGUgRHJvc29waGlsYSBtaW5pY2hyb21vc29tZSBEcDExODcgYnkgc2luZ2xlIFAgZWxlbWVudCBpbnNlcnRpb25hbCBtdXRhZ2VuZXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MTMyOyAgY2l0YXRpb25fcGFnZXM9NzM3LTc1MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNwb3NvbnMgaW4gcGxhY2Ugb2YgdGVsb21lcmljIHJlcGVhdHMgYXQgYSBEcm9zb3BoaWxhIHRlbG9tZXJlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0xOTkzOyAgY2l0YXRpb25fdm9sdW1lPTc1OyAgY2l0YXRpb25fcGFnZXM9MTA4My0xMDkzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1ETkEgb3JnYW5pemF0aW9uIGFuZCBwb2x5bW9ycGhpc20gb2YgYSB3aWxkLXR5cGUgRHJvc29waGlsYSB0ZWxvbWVyZSByZWdpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNocm9tb3NvbWE7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9MjI5LTI0MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21pYyBhbmFseXNpcyBvZiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlciB0ZWxvbWVyZXM6IGZ1bGwtbGVuZ3RoIGNvcGllcyBvZiBIZVQtQSBhbmQgVEFSVCBlbGVtZW50cyBhdCB0ZWxvbWVyZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTE2MTMtMTYxOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iICBCZXJnbWFuIEMuTS4sIFF1ZXNuZXZpbGxlIEguLCBBbnhvbGFiw6low6hyZSBELiwgQXNoYnVybmVyIE0uICAyMDA2ICwgUmVjdXJyZW50IGluc2VydGlvbiBhbmQgZHVwbGljYXRpb24gZ2VuZXJhdGUgbmV0d29ya3Mgb2YgdHJhbnNwb3NhYmxlIGVsZW1lbnQgc2VxdWVuY2VzIGluIHRoZSBEcm9zb3BoaWxhIG1lbGFub2dhc3RlciBnZW5vbWUuIEdlbm9tZSBCaW9sLiwgNyAsIFIxMTIuICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVZpYWJsZSBkZWxldGlvbnMgb2YgYSB0ZWxvbWVyZSBmcm9tIGEgRHJvc29waGlsYSBjaHJvbW9zb21lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0xOTg5OyAgY2l0YXRpb25fdm9sdW1lPTU4OyAgY2l0YXRpb25fcGFnZXM9NzkxLTgwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QW4gZXBpZ2VuZXRpYyBhY3RpdmF0aW9uIHJvbGUgb2YgUGl3aSBhbmQgYSBQaXdpLWFzc29jaWF0ZWQgcGlSTkEgaW4gRHJvc29waGlsYSBtZWxhbm9nYXN0ZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00NTA7ICBjaXRhdGlvbl9wYWdlcz0zMDQtMzA4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgSklMLTEga2luYXNlIGFmZmVjdHMgdGVsb21lcmUgZXhwcmVzc2lvbiBpbiB0aGUgZGlmZmVyZW50IHRlbG9tZXJlIGRvbWFpbnMgb2YgRHJvc29waGlsYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU4MTU0MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3B1Ymxpc2hlcj1TcHJpbmdlciwgQmVybGluOyAgY2l0YXRpb25fdGl0bGU9U3VidGVsb21lcmVzOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fcGFnZXM9MjExLTIyNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGFyZ2V0aW5nIG9mIFAtZWxlbWVudCByZXBvcnRlcnMgdG8gaGV0ZXJvY2hyb21hdGljIGRvbWFpbnMgYnkgdHJhbnNwb3NhYmxlIGVsZW1lbnQgMTM2MCBpbiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9MjAyOyAgY2l0YXRpb25fcGFnZXM9NTY1LTU4MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNoYXJhY3RlcmlzdGljcyBvZiBkaXZlcnNlIHBvcHVsYXRpb25zIGFyZSBjb25zaXN0ZW50IHdpdGggdGhlIGh5cG90aGVzaXMgb2YgYSByZWNlbnQgaW52YXNpb24gb2YgRHJvc29waGlsYSBtZWxhbm9nYXN0ZXIgYnkgbW9iaWxlIFAgZWxlbWVudHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MTk4ODsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9MjUyLTI2OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29weSBudW1iZXIgYW5kIGRpc3RyaWJ1dGlvbiBvZiBQIGFuZCBJIG1vYmlsZSBlbGVtZW50cyBpbiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlciBwb3B1bGF0aW9uczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2hyb21vc29tYTsgIGNpdGF0aW9uX3llYXI9MTk4OTsgIGNpdGF0aW9uX3ZvbHVtZT05ODsgIGNpdGF0aW9uX3BhZ2VzPTIwNy0yMTQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBtYXRlcm5hbGx5IGluaGVyaXRlZCByZWd1bGF0aW9uIG9mIFAgZWxlbWVudHMgaW4gRHJvc29waGlsYSBtZWxhbm9nYXN0ZXIgY2FuIGJlIGVsaWNpdGVkIGJ5IHR3byBQIGNvcGllcyBhdCBjeXRvbG9naWNhbCBzaXRlIDFBIG9uIHRoZSBYIGNocm9tb3NvbWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0xOTkxOyAgY2l0YXRpb25fdm9sdW1lPTEyOTsgIGNpdGF0aW9uX3BhZ2VzPTUwMS01MTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRlbG9tZXJpYyB0cmFucy1zaWxlbmNpbmc6IGFuIGVwaWdlbmV0aWMgcmVwcmVzc2lvbiBjb21iaW5pbmcgUk5BIHNpbGVuY2luZyBhbmQgaGV0ZXJvY2hyb21hdGluIGZvcm1hdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBHZW5ldDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MTYzMy0xNjQzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFucy1zaWxlbmNpbmcgYnkgUCBlbGVtZW50cyBpbnNlcnRlZCBpbiBzdWJ0ZWxvbWVyaWMgaGV0ZXJvY2hyb21hdGluIGludm9sdmVzIHRoZSBEcm9zb3BoaWxhIFBvbHljb21iIGdyb3VwIGdlbmUsIEVuaGFuY2VyIG9mIHplc3RlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk5ODsgIGNpdGF0aW9uX3ZvbHVtZT0xNDk7ICBjaXRhdGlvbl9wYWdlcz0xODM5LTE4NTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRlbG9tZXJpYyB0cmFuc2dlbmVzIGFuZCB0cmFucy1zaWxlbmNpbmcgaW4gRHJvc29waGlsYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY2E7ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MTE3OyAgY2l0YXRpb25fcGFnZXM9MzI3LTMzNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzY3JldGUgc21hbGwgUk5BLWdlbmVyYXRpbmcgbG9jaSBhcyBtYXN0ZXIgcmVndWxhdG9ycyBvZiB0cmFuc3Bvc29uIGFjdGl2aXR5IGluIERyb3NvcGhpbGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNlbGw7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTI4OyAgY2l0YXRpb25fcGFnZXM9MTA4OS0xMTAzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PbmUgbG9vcCB0byBydWxlIHRoZW0gYWxsOiB0aGUgcGluZy1wb25nIGN5Y2xlIGFuZCBwaVJOQS1ndWlkZWQgc2lsZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1UcmVuZHMgQmlvY2hlbS4gU2NpOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTQxOyAgY2l0YXRpb25fcGFnZXM9MzI0LTMzNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGVsb21lcmljIHRyYW5zLXNpbGVuY2luZyBpbiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlcjogdGlzc3VlIHNwZWNpZmljaXR5LCBkZXZlbG9wbWVudCBhbmQgZnVuY3Rpb25hbCBpbnRlcmFjdGlvbnMgYmV0d2VlbiBub24taG9tb2xvZ291cyB0ZWxvbWVyZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz1lMzI0OSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGVwaWdlbmV0aWMgcm9sZSBmb3IgbWF0ZXJuYWxseSBpbmhlcml0ZWQgcGlSTkFzIGluIHRyYW5zcG9zb24gc2lsZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTMyMjsgIGNpdGF0aW9uX3BhZ2VzPTEzODctMTM5MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGVwaWdlbmV0aWMgdHJhbnMtc2lsZW5jaW5nIGVmZmVjdCBpbiBEcm9zb3BoaWxhIGludm9sdmVzIG1hdGVybmFsbHktdHJhbnNtaXR0ZWQgc21hbGwgUk5BcyB3aG9zZSBwcm9kdWN0aW9uIGRlcGVuZHMgb24gdGhlIHBpUk5BIHBhdGh3YXkgYW5kIEhQMTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPWUxMTAzMiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhcmFtdXRhdGlvbiBpbiBEcm9zb3BoaWxhIGxpbmtlZCB0byBlbWVyZ2VuY2Ugb2YgYSBwaVJOQS1wcm9kdWNpbmcgbG9jdXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT00OTA7ICBjaXRhdGlvbl9wYWdlcz0xMTItMTE1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSIgIE11ZXJkdGVyIEYuLCBPbG92bmlrb3YgSS4sIE1vbGFybyBBLiAsIDIwMTIgLCBQcm9kdWN0aW9uIG9mIGFydGlmaWNpYWwgcGlSTkFzIGluIGZsaWVzIGFuZCBtaWNlLiBSTkEgLCAxOCAsIDQyIOKAkyA1Mi4gIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGV0ZWN0aW9uIGluIHNpdHUgb2YgZ2Vub21pYyByZWd1bGF0b3J5IGVsZW1lbnRzIGluIERyb3NvcGhpbGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwgQWNhZC4gU2NpLiBVLiBTLiBBOyAgY2l0YXRpb25feWVhcj0xOTg3OyAgY2l0YXRpb25fdm9sdW1lPTg0OyAgY2l0YXRpb25fcGFnZXM9OTEyMy05MTI3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QLUVsZW1lbnQgcmVwcmVzc2lvbiBpbiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlciBieSBhIG5hdHVyYWxseSBvY2N1cnJpbmcgZGVmZWN0aXZlIHRlbG9tZXJpYyBQIGNvcHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmV0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTE1NTsgIGNpdGF0aW9uX3BhZ2VzPTE4NDEtMTg1NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnJvbSBmaXJzdCBiYXNlOiB0aGUgc2VxdWVuY2Ugb2YgdGhlIHRpcCBvZiB0aGUgWCBjaHJvbW9zb21lIG9mIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyLCBhIGNvbXBhcmlzb24gb2YgdHdvIHNlcXVlbmNpbmcgc3RyYXRlZ2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTcxMC03MzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdhbmRlcmluZ3Mgb2YgaG9ibzogYSB0cmFuc3Bvc29uIGluIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyIGFuZCBpdHMgY2xvc2UgcmVsYXRpdmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljYTsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMjA7ICBjaXRhdGlvbl9wYWdlcz0xMjUtMTM2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgZ2Vub21lIHNlcXVlbmNlIG9mIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1TY2llbmNlOyAgY2l0YXRpb25feWVhcj0yMDAwOyAgY2l0YXRpb25fdm9sdW1lPTI4NzsgIGNpdGF0aW9uX3BhZ2VzPTIxODUtMjE5NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U2VhVmlldyB2ZXJzaW9uIDQ6IGEgbXVsdGlwbGF0Zm9ybSBncmFwaGljYWwgdXNlciBpbnRlcmZhY2UgZm9yIHNlcXVlbmNlIGFsaWdubWVudCBhbmQgcGh5bG9nZW5ldGljIHRyZWUgYnVpbGRpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1vbC4gQmlvbC4gRXZvbDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTIyMS0yMjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBhcmFtdXRhdGlvbiBpbiBEcm9zb3BoaWxhIHJlcXVpcmVzIGJvdGggbnVjbGVhciBhbmQgY3l0b3BsYXNtaWMgYWN0b3JzIG9mIHRoZSBwaVJOQSBwYXRod2F5IGFuZCBpbmR1Y2VzIENpcy1zcHJlYWRpbmcgb2YgcGlSTkEgcHJvZHVjdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MjAxOyAgY2l0YXRpb25fcGFnZXM9MTM4MS0xMzk2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbHRyYWZhc3QgYW5kIG1lbW9yeS1lZmZpY2llbnQgYWxpZ25tZW50IG9mIHNob3J0IEROQSBzZXF1ZW5jZXMgdG8gdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTA7ICBjaXRhdGlvbl9wYWdlcz1SMjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaHJlZSBkaXN0aW5jdCBjaHJvbWF0aW4gZG9tYWlucyBpbiB0ZWxvbWVyZSBlbmRzIG9mIHBvbHl0ZW5lIGNocm9tb3NvbWVzIGluIERyb3NvcGhpbGEgbWVsYW5vZ2FzdGVyIFRlbCBtdXRhbnRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBDZWxsIFNjaTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xMTg7ICBjaXRhdGlvbl9wYWdlcz01NDY1LTU0Nzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1vbGVjdWxhciBwb3B1bGF0aW9uIGdlbmV0aWNzIG9mIERyb3NvcGhpbGEgc3VidGVsb21lcmljIEROQTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTc4OyAgY2l0YXRpb25fcGFnZXM9NDc3LTQ4NzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Tm9uLXJhbmRvbSBkaXNqdW5jdGlvbiBpbiBEcm9zb3BoaWxhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MTk1MTsgIGNpdGF0aW9uX3ZvbHVtZT0zNjsgIGNpdGF0aW9uX3BhZ2VzPTI2Ny0yODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlY3VycmVudCBkZWxldGlvbiBhbmQgZ2VuZSBwcmVzZW5jZS9hYnNlbmNlIHBvbHltb3JwaGlzbTogdGVsb21lcmUgZHluYW1pY3MgZG9taW5hdGUgZXZvbHV0aW9uIGF0IHRoZSB0aXAgb2YgM0wgaW4gRHJvc29waGlsYSBtZWxhbm9nYXN0ZXIgYW5kIEQuIHNpbXVsYW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5ldGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNzk7ICBjaXRhdGlvbl9wYWdlcz0xMDIxLTEwMjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBjb21wbGV4IHN0cnVjdHVyZSBhbmQgZHluYW1pYyBldm9sdXRpb24gb2YgaHVtYW4gc3VidGVsb21lcmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIFJldi4gR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTkxLTEwMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGdlbmV0aWMgYW5hbHlzaXMgb2YgbWVpb3NpcyBpbiBmZW1hbGUgRHJvc29waGlsYSBtZWxhbm9nYXN0ZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBoaWxvcy4gVHJhbnMuIFIuIFNvYy4gTG9uZC4gQiBCaW9sLiBTY2k7ICBjaXRhdGlvbl95ZWFyPTE5Nzc7ICBjaXRhdGlvbl92b2x1bWU9Mjc3OyAgY2l0YXRpb25fcGFnZXM9Mjk1LTMxMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIGNsb25pbmcgb2YgbGV0aGFsKDIpZ2lhbnQgbGFydmFlLCBhIHJlY2Vzc2l2ZSBvbmNvZ2VuZSBvZiBEcm9zb3BoaWxhIG1lbGFub2dhc3RlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RU1CTyBKOyAgY2l0YXRpb25feWVhcj0xOTg1OyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0xNTUxLTE1NTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVR1cm5pbmcganVuayBpbnRvIGdvbGQ6IGRvbWVzdGljYXRpb24gb2YgdHJhbnNwb3NhYmxlIGVsZW1lbnRzIGFuZCB0aGUgY3JlYXRpb24gb2YgbmV3IGdlbmVzIGluIGV1a2FyeW90ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2Vzc2F5czsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0yODsgIGNpdGF0aW9uX3BhZ2VzPTkxMy05MjI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC81LzQ1OS8yMjg5MjkxNS9kc3gwMTcucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFR3byBUZWxvbWVyaWMgQXNzb2NpYXRlZCBTZXF1ZW5jZXMsIFRBUy1SIGFuZCBUQVMtTCwgZm9ybSB0aGUgcHJpbmNpcGFsIHN1YnRlbG9tZXJpYyByZXBlYXQgZmFtaWxpZXMgaWRlbnRpZmllZCBpbiBEcm9zb3BoaWxhIG1lbGFub2dhc3Rlci4gVGhleSBhcmUgIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zNzc4MjM3JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZORzhqYC4OANnNHXiiEhYM1hX4MM" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f459%2f3778237" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f459%2f3778237" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=99990" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/5"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/5/1/m_dnares_24_5cover.png?Expires=2147483647&amp;Signature=jul3oxjgkP675GHMbQEy~zZ1my2m2g0xI0mr9LkJEjA-PhoC4BnM1ARo9FdXFQKdaeXjXdDG8Th7l01-PuT824GoN40EPQF0GwZOV5tGDYZCguawjvcFtz41hVQe~O6T29XL-3ITGszkj0QvmRZLajQvyRnp825zpQTjNCOtAlB0rKMc7Dcdixzjc2uv2jilEDMqD9YZCuMkOXRLrwDQftwYgb~q8EavwLngcuRRSkILOW9lFSQBCUNAqNd4mDnFW5RKty7z4w3HYGKz5wJomnNLGyCwwAyELyFWHWV4xtKHzE5xEUd8do1ZdHc6pZe4TftEycCVzz1M4dnVR5tpug__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/5">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 5</div>

            </a>

        </div>
        <div class="ii-pub-date">
October 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="106355402">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355402">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355404">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355404">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355411">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355411">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355428">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355428">3. Results and conclusions</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355454">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355454">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355464">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355464">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="106355472">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#106355472">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/5/445/3760268">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/5/473/3769337">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Short and long-term evolutionary dynamics of subtelomeric piRNA clusters in <em>Drosophila</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Amna Asif-Laidin</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Sorbonne Universités, UPMC University of Paris 06, CNRS, Biologie du Développement Paris-Seine, Institut de Biologie Paris-Seine (LBD-IBPS), 75005 Paris, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Amna+Asif-Laidin">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Asif-Laidin A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Asif-Laidin A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Amna Asif-Laidin</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Val&#233;rie Delmarre</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Sorbonne Universités, UPMC University of Paris 06, CNRS, Biologie du Développement Paris-Seine, Institut de Biologie Paris-Seine (LBD-IBPS), 75005 Paris, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Val%c3%a9rie+Delmarre">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Delmarre V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Delmarre V&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Valérie Delmarre</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jeanne Laurentie</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Sorbonne Universités, UPMC University of Paris 06, CNRS, Biologie du Développement Paris-Seine, Institut de Biologie Paris-Seine (LBD-IBPS), 75005 Paris, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jeanne+Laurentie">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Laurentie J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Laurentie J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jeanne Laurentie</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wolfgang J. Miller</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div>Lab Genome Dynamics, Department for Cell &amp; Developmental Biology, Center of Anatomy and Cell Biology, Medical University of Vienna, A-1090 Vienna, Austria</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wolfgang+J.+Miller">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Miller W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Miller W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wolfgang J. Miller</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">St&#233;phane Ronsseray</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Sorbonne Universités, UPMC University of Paris 06, CNRS, Biologie du Développement Paris-Seine, Institut de Biologie Paris-Seine (LBD-IBPS), 75005 Paris, France</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=St%c3%a9phane+Ronsseray">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ronsseray S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ronsseray S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Stéphane Ronsseray</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Laure Teysset</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div>Sorbonne Universités, UPMC University of Paris 06, CNRS, Biologie du Développement Paris-Seine, Institut de Biologie Paris-Seine (LBD-IBPS), 75005 Paris, France</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx017-cor1" data-open="dsx017-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. (+33) 1 44 27 22 88. Fax. (+33) 1 44 27 34 45. Email: <a href="mailto:laure.teysset@upmc.fr" target="_blank">laure.teysset@upmc.fr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Laure+Teysset">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Teysset L">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Teysset L&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Laure Teysset</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx017-FM1"><div class="fn"><div class="label -label">*</div><div class="footnote-content">Edited by Dr. Yuji Kohara</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 5, 1 October 2017, Pages 459–472, <a href='https://doi.org/10.1093/dnares/dsx017'>https://doi.org/10.1093/dnares/dsx017</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 April 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">17 December 2016</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">29 March 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3778237" href="/dnaresearch/article-pdf/24/5/459/22892915/dsx017.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Amna Asif-Laidin, Valérie Delmarre, Jeanne Laurentie, Wolfgang J. Miller, Stéphane Ronsseray, Laure Teysset;  Short and long-term evolutionary dynamics of subtelomeric piRNA clusters in <em>Drosophila</em>, <em>DNA Research</em>, Volume 24, Issue 5, 1 October 2017, Pages 459–472, <a href='https://doi.org/10.1093/dnares/dsx017'>https://doi.org/10.1093/dnares/dsx017</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3778237?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3778237?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3778237?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3778237?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3778237?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3778237?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3778237?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Short%20and%20long-term%20evolutionary%20dynamics%20of%20subtelomeric%20piRNA%20clusters%20in%20%26lt%3Bem%26gt%3BDrosophila%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2017-04-27&amp;volumeNum=24&amp;issueNum=5&amp;author=Asif-Laidin%2C%20Amna%3B%20Delmarre%2C%20Val%C3%A9rie&amp;startPage=459&amp;endPage=472&amp;contentId=10.1093%2Fdnares%2Fdsx017&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f459%2f3778237" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=106355402 id="106355402" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Two Telomeric Associated Sequences, TAS-R and TAS-L, form the principal subtelomeric repeat families identified in <em>Drosophila melanogaster</em>. They are PIWI-interacting RNA (piRNA) clusters involved in repression of Transposable Elements. In this study, we revisited TAS structural and functional dynamics in <em>D. melanogaster</em> and in related species. <em>In silico</em> analysis revealed that TAS-R family members are composed of previously uncharacterized domains. This analysis also showed that TAS-L repeats are composed of arrays of a region we have named “<em>TAS-L like</em>” (<em>TLL</em>) identified specifically in one TAS-R family member, <em>X-TAS</em>. <em>TLL</em> were also present in other species of the <em>melanogaster</em> subgroup. Therefore, it is possible that <em>TLL</em> represents an ancestral subtelomeric piRNA core-cluster. Furthermore, all <em>D. melanogaster</em> genomes tested possessed at least one TAS-R locus, whereas TAS-L can be absent. A screen of 110 <em>D. melanogaster</em> lines showed that <em>X-TAS</em> is always present in flies living in the wild, but often absent in long-term laboratory stocks and that natural populations frequently lost their <em>X-TAS</em> within 2 years upon lab conditioning. Therefore, the unexpected structural and temporal dynamics of subtelomeric piRNA clusters demonstrated here suggests that genome organization is subjected to distinct selective pressures in the wild and upon domestication in the laboratory. </p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="subtelomeres">subtelomeres</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="TAS">TAS</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="Drosophila"><em>Drosophila</em></a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="piRNA">piRNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transposable element&quot;">transposable element</a></div>                            <h2 scrollto-destination=106355404 id="106355404" class="section-title" >1. Introduction</h2>
<p>In the <em>Drosophila melanogaster</em> genome, Telomeric Associated Sequences (TAS) are moderately repeated sequences that correspond to non-coding heterochromatin. They are positioned between euchromatin and the distal telomeric transposon arrays, consisting of non-LTR retrotransposons: <em>HeT-A</em>, <em>TAHRE</em> and <em>TART</em> (<em>HTT</em>), not present elsewhere in the genome.<a reveal-id="dsx017-B1" data-open="dsx017-B1" class="link link-ref link-reveal xref-bibr"><sup>1–4</sup></a> Due to their position, TAS constitute a transition zone between telomeres and genes.</p><p>The initial molecular analysis of TAS repeats comes from the study of a 10-kb fragment isolated from the <em>Dp1187 X</em> minichromosome.<a reveal-id="dsx017-B1" data-open="dsx017-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> The telomeric region of this chromosome contains four tandem TAS repeats of 1.8 kb for the longest (<em>X-TAS</em>) and two repeats of 0.9 kb<a reveal-id="dsx017-B1" data-open="dsx017-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> (<a reveal-id="dsx017-F1" data-open="dsx017-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). Each TAS repeat contains three tandem subrepeats of 172 bp that were later identified as solo LTRs of the <em>Invader4</em> retrotransposon (<em>INV-4</em>).<a reveal-id="dsx017-B5" data-open="dsx017-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a></p>                        <a id="106355407" scrollto-destination="106355407"></a>
<div id="dsx017-F1" data-id="dsx017-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/m_dsx017f1.png?Expires=2147483647&Signature=G1A379tuMHZJN35L50mSJSOrjD-tPOD-QwVdVpye8514sx6zFpPv9IP4mlxow9NKJEP7Uc35HJ8KxvWXmnOkiTucL40LQbi1k4pb-6jh46UKaMrPHwxFsU2KiwMaXs~vscLxfNMeOBChexsDT~sJJXohhWlk~23A7tMb9qVb9Gsbfn73tT1mSMytxb50wMQuLmNID7l0UlCE6cvsc9VXi8lZBKoqiUxwM7~ZDAFG6Z-JqGbOdxiWwyQ~kGpKPQsld6AsIsqB1VQ~z56HSBSOJl9ldArfgzlEEFFiyWzn10bCRNFt9RjlcKxtayo4YqIdDrDJkZKWLUwXVR3eWkFDJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Subtelomeric regions in Drosophila melanogaster. (A) X subtelomeric region from the Dp1187 minichromosome contains four TAS repeats [TAS-A (205 bp), TAS-B (1,872 bp), TAS-C (1,858 bp) and TAS-D (1,575 bp)] located between telomeric elements (HeT-A) and two 0.9 kb repeats. (B–D) Schematic representation of the TAS-R family repeats represented by TAS-B for X-TAS (B), 2R-TAS (C) and 3R-TAS (D) located on the X, second and third chromosomes, respectively. They are composed of several domains [INV-4 solo LTR, T1, T2, T3, T4 and T5 domain]. In order to simplify the comparison between members of TAS-R family, the starting point for each TAS repeat was arbitrary positioned at the beginning of the INV-4 sequence. (E) TAS-L repeats on 2L and 3L chromosomes are composed of six fragments of a sequence found in the T3 domain of X-TAS repeats, that we named TLL (hatched boxes). (F) The TLL domain is also found in the 0.9-kb repeats adjacent to X-TAS. The T3-TLL sequence (B) is taken as the reference here for determining percentage of identity between sequences (E, F). Identity between the TLL of 164 bp in the 0.9 kb repeat and the TLL of 147 bp in TAS-L is 85%. (G) The X chromosome of the Canton-S strain is lacking X-TAS repeats. (H) The X chromosome of the y1; cn1bw1sp1 line lacks both X-TAS repeats and 0.9 kb repeats. In all schematic representations of telomeric sequences the telomeres are to the left. TAS-R probes used in in situ hybridization were PCR amplified using primers indicated by black arrows (A). Primers used to PCR amplify TAS-L repeats are indicated by arrows (E)." path-from-xml="dsx017f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355407/dsx017f1.png" path-from-xml="dsx017f1.png" target="_blank">View large</a><a section="106355407" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017f1.png?Expires=2147483647&Signature=1LqABuigXRnld7~W-G6--Q8nB9pjdVhT-zfnY-R5m388ldWq0HpqB2yCwzbdR~4TAU6tXpsugPBPolJqQunrgvBkFmTPah0Q~KpkaYjZPFq2R4Flgvtnql~r--HnEYHBvl81cFTohajhgzJQe1dJON3SjyzsnkILyz6VMDmtPJ2xZa~zyyXrqQ1zRirhOHkgMnPd5iRM0scsP3GD7qxBTOfGkF-YJV~ACEARnneypDgCMxitMP--UHAchNOVuXFokSpG1Zltck3gqtU~9euZvnYguvvVU1fe2FgRy1O-TahcdD7DC51hsAx-UaU8A~vOsONiaCukKTsnOv1IbjJPeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355407&ar=3778237&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx017f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Subtelomeric regions in Drosophila melanogaster. (A) <em>X</em> subtelomeric region from the <em>Dp1187</em> minichromosome contains four TAS repeats [<em>TAS-A</em> (205 bp), <em>TAS-B</em> (1,872 bp), <em>TAS-C</em> (1,858 bp) and <em>TAS-D</em> (1,575 bp)] located between telomeric elements (<em>HeT-A</em>) and two 0.9 kb repeats. (B–D) Schematic representation of the <em>TAS-R</em> family repeats represented by <em>TAS-B</em> for <em>X-TAS</em> (B), <em>2R-TAS</em> (C) and <em>3R-TAS</em> (D) located on the <em>X</em>, second and third chromosomes, respectively. They are composed of several domains [<em>INV-4</em> solo LTR, T1, T2, T3, T4 and T5 domain]. In order to simplify the comparison between members of TAS-R family, the starting point for each TAS repeat was arbitrary positioned at the beginning of the <em>INV-4</em> sequence. (E) TAS-L repeats on <em>2L</em> and <em>3L</em> chromosomes are composed of six fragments of a sequence found in the T3 domain of <em>X-TAS</em> repeats, that we named <em>TLL</em> (hatched boxes). (F) The <em>TLL</em> domain is also found in the 0.9-kb repeats adjacent to <em>X-TAS</em>. The <em>T3-TLL</em> sequence (B) is taken as the reference here for determining percentage of identity between sequences (E, F). Identity between the <em>TLL</em> of 164 bp in the 0.9 kb repeat and the <em>TLL</em> of 147 bp in TAS-L is 85%. (G) The <em>X</em> chromosome of the <em>Canton</em>-<em>S</em> strain is lacking <em>X-TAS</em> repeats. (H) The <em>X</em> chromosome of the <em>y</em><sup>1</sup><em>; cn</em><sup>1</sup><em>bw</em><sup>1</sup><em>sp<sup>1</sup></em> line lacks both <em>X-TAS</em> repeats and 0.9 kb repeats. In all schematic representations of telomeric sequences the telomeres are to the left. TAS-R probes used in <em>in situ</em> hybridization were PCR amplified using primers indicated by black arrows (A). Primers used to PCR amplify TAS-L repeats are indicated by arrows (E).</p></div></div><div content-id="dsx017-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/m_dsx017f1.png?Expires=2147483647&Signature=G1A379tuMHZJN35L50mSJSOrjD-tPOD-QwVdVpye8514sx6zFpPv9IP4mlxow9NKJEP7Uc35HJ8KxvWXmnOkiTucL40LQbi1k4pb-6jh46UKaMrPHwxFsU2KiwMaXs~vscLxfNMeOBChexsDT~sJJXohhWlk~23A7tMb9qVb9Gsbfn73tT1mSMytxb50wMQuLmNID7l0UlCE6cvsc9VXi8lZBKoqiUxwM7~ZDAFG6Z-JqGbOdxiWwyQ~kGpKPQsld6AsIsqB1VQ~z56HSBSOJl9ldArfgzlEEFFiyWzn10bCRNFt9RjlcKxtayo4YqIdDrDJkZKWLUwXVR3eWkFDJg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Subtelomeric regions in Drosophila melanogaster. (A) X subtelomeric region from the Dp1187 minichromosome contains four TAS repeats [TAS-A (205 bp), TAS-B (1,872 bp), TAS-C (1,858 bp) and TAS-D (1,575 bp)] located between telomeric elements (HeT-A) and two 0.9 kb repeats. (B–D) Schematic representation of the TAS-R family repeats represented by TAS-B for X-TAS (B), 2R-TAS (C) and 3R-TAS (D) located on the X, second and third chromosomes, respectively. They are composed of several domains [INV-4 solo LTR, T1, T2, T3, T4 and T5 domain]. In order to simplify the comparison between members of TAS-R family, the starting point for each TAS repeat was arbitrary positioned at the beginning of the INV-4 sequence. (E) TAS-L repeats on 2L and 3L chromosomes are composed of six fragments of a sequence found in the T3 domain of X-TAS repeats, that we named TLL (hatched boxes). (F) The TLL domain is also found in the 0.9-kb repeats adjacent to X-TAS. The T3-TLL sequence (B) is taken as the reference here for determining percentage of identity between sequences (E, F). Identity between the TLL of 164 bp in the 0.9 kb repeat and the TLL of 147 bp in TAS-L is 85%. (G) The X chromosome of the Canton-S strain is lacking X-TAS repeats. (H) The X chromosome of the y1; cn1bw1sp1 line lacks both X-TAS repeats and 0.9 kb repeats. In all schematic representations of telomeric sequences the telomeres are to the left. TAS-R probes used in in situ hybridization were PCR amplified using primers indicated by black arrows (A). Primers used to PCR amplify TAS-L repeats are indicated by arrows (E)." path-from-xml="dsx017f1.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355407/dsx017f1.png" path-from-xml="dsx017f1.png" target="_blank">View large</a><a section="106355407" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017f1.png?Expires=2147483647&Signature=1LqABuigXRnld7~W-G6--Q8nB9pjdVhT-zfnY-R5m388ldWq0HpqB2yCwzbdR~4TAU6tXpsugPBPolJqQunrgvBkFmTPah0Q~KpkaYjZPFq2R4Flgvtnql~r--HnEYHBvl81cFTohajhgzJQe1dJON3SjyzsnkILyz6VMDmtPJ2xZa~zyyXrqQ1zRirhOHkgMnPd5iRM0scsP3GD7qxBTOfGkF-YJV~ACEARnneypDgCMxitMP--UHAchNOVuXFokSpG1Zltck3gqtU~9euZvnYguvvVU1fe2FgRy1O-TahcdD7DC51hsAx-UaU8A~vOsONiaCukKTsnOv1IbjJPeQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355407&ar=3778237&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx017f1.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Subtelomeric regions in Drosophila melanogaster. (A) <em>X</em> subtelomeric region from the <em>Dp1187</em> minichromosome contains four TAS repeats [<em>TAS-A</em> (205 bp), <em>TAS-B</em> (1,872 bp), <em>TAS-C</em> (1,858 bp) and <em>TAS-D</em> (1,575 bp)] located between telomeric elements (<em>HeT-A</em>) and two 0.9 kb repeats. (B–D) Schematic representation of the <em>TAS-R</em> family repeats represented by <em>TAS-B</em> for <em>X-TAS</em> (B), <em>2R-TAS</em> (C) and <em>3R-TAS</em> (D) located on the <em>X</em>, second and third chromosomes, respectively. They are composed of several domains [<em>INV-4</em> solo LTR, T1, T2, T3, T4 and T5 domain]. In order to simplify the comparison between members of TAS-R family, the starting point for each TAS repeat was arbitrary positioned at the beginning of the <em>INV-4</em> sequence. (E) TAS-L repeats on <em>2L</em> and <em>3L</em> chromosomes are composed of six fragments of a sequence found in the T3 domain of <em>X-TAS</em> repeats, that we named <em>TLL</em> (hatched boxes). (F) The <em>TLL</em> domain is also found in the 0.9-kb repeats adjacent to <em>X-TAS</em>. The <em>T3-TLL</em> sequence (B) is taken as the reference here for determining percentage of identity between sequences (E, F). Identity between the <em>TLL</em> of 164 bp in the 0.9 kb repeat and the <em>TLL</em> of 147 bp in TAS-L is 85%. (G) The <em>X</em> chromosome of the <em>Canton</em>-<em>S</em> strain is lacking <em>X-TAS</em> repeats. (H) The <em>X</em> chromosome of the <em>y</em><sup>1</sup><em>; cn</em><sup>1</sup><em>bw</em><sup>1</sup><em>sp<sup>1</sup></em> line lacks both <em>X-TAS</em> repeats and 0.9 kb repeats. In all schematic representations of telomeric sequences the telomeres are to the left. TAS-R probes used in <em>in situ</em> hybridization were PCR amplified using primers indicated by black arrows (A). Primers used to PCR amplify TAS-L repeats are indicated by arrows (E).</p></div></div><p>Early <em>in situ</em> hybridization studies on polytene chromosomes and more recent sequence alignments identified common subtelomeric regions, corresponding to the <em>INV-4</em> LTRs, for the <em>X</em>, <em>2R</em> and <em>3R</em> chromosomes (<em>X-TAS</em>, <em>2R-TAS</em> and <em>3R-TAS</em>).<a reveal-id="dsx017-B1" data-open="dsx017-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx017-B6" data-open="dsx017-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx017-B7" data-open="dsx017-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> These three subtelomeric regions have been grouped together as the TAS-R family to differentiate them from the TAS-L family, which is devoid of <em>INV-4</em> sequences and localized at <em>2L</em> and <em>3L</em> telomeres.<a reveal-id="dsx017-B8" data-open="dsx017-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> TAS-L are made-up of non-coding repeats.<a reveal-id="dsx017-B3" data-open="dsx017-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> Recent publications mentioned sequences identities between the TAS-R and TAS-L families.<a reveal-id="dsx017-B9" data-open="dsx017-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsx017-B10" data-open="dsx017-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a></p><p><em>X-TAS</em>, a TAS-R family member, has been shown to play a major role in the establishment of <em>P</em> element repression following invasion of the <em>D. melanogaster</em> genome between the 1950s and the 1970s.<a reveal-id="dsx017-B11" data-open="dsx017-B11" class="link link-ref link-reveal xref-bibr"><sup>11–13</sup></a> The study of this repression led to the discovery of an ovarian silencing mechanism involving PIWI interacting RNA (piRNA) biogenesis called the <em>Trans</em>-Silencing Effect (TSE).<a reveal-id="dsx017-B14" data-open="dsx017-B14" class="link link-ref link-reveal xref-bibr"><sup>14–16</sup></a> piRNAs are a class of non coding small RNAs of 23–29 nt produced from specific loci, called piRNA clusters.<a reveal-id="dsx017-B17" data-open="dsx017-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> piRNA clusters are heterochromatic, mainly consisting of fragments of Transposable Elements (TEs), and involved in repression of TE transposition at transcriptional and/or posttranscriptional levels in animal gonads.<a reveal-id="dsx017-B18" data-open="dsx017-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> TAS-R, especially <em>X-TAS</em>, appear to be piRNA clusters, into which <em>P</em> elements (as well as <em>P</em>-derived transgenes) can insert.<a reveal-id="dsx017-B1" data-open="dsx017-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx017-B10" data-open="dsx017-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsx017-B19" data-open="dsx017-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> Once inserted, <em>P</em> sequences are included in the piRNA produced and these piRNA will participate to repression of euchromatic <em>P</em> copies in the germline leading to the so-called <em>TSE</em>.<a reveal-id="dsx017-B20" data-open="dsx017-B20" class="link link-ref link-reveal xref-bibr"><sup>20–23</sup></a></p><p>Current literature studying repression of TEs is dealing with the fact that each TE might be potentially regulated by multiple piRNA clusters at the same time. Here, we focus on a specific class of piRNA clusters, the subtelomeric clusters. We extended the analysis of their structure, function and dynamics in long-term laboratory stocks and flies recently collected from the wild. We further addressed their evolutionary origin by comparative analysis of Drosophilidae genomes. Using <em>in situ</em> hybridization on polytenes chromosomes, PCR amplification, ovarian small RNA sequencing and <em>in silico</em> sequence comparison, we identified a core sequence, that we called <em>TLL</em>, for <em>TAS-L like</em>, in <em>D. melanogaster</em> subtelomeric piRNA clusters. In addition, the <em>TLL</em> sequence was shown to be a conserved part of piRNA clusters in subtelomeric regions of related <em>melanogaster</em> subgroup species. Finally, a striking difference in the presence and absence of <em>X-TAS</em> between long-term laboratory stocks and wild <em>D. melanogaster</em> populations was observed. An extensive analysis of lines established from flies from the wild showed that they all possessed <em>X-TAS</em>, while this was not the case for many laboratory stocks. In addition, a survey over a 2-year period of the wild-type lines allowed us to witness the loss of <em>X-TAS</em> during <em>Drosophila</em> domestication under laboratory conditions. This study reveals an unexpected feature for piRNA clusters that, although fulfilling an essential role as guardians of the genome by repressing TEs, can be lost under artificial laboratory conditions without significantly impairing genome integrity of their hosts.</p>                            <h2 scrollto-destination=106355411 id="106355411" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=106355412 id="106355412" class="section-title" >2.1. <em>Drosophila</em> stocks</h3>
<p>The following Fly stocks from the Bloomington stock center were used: <em>y</em><sup>1</sup><em>; cn</em><sup>1</sup><em>bw</em><sup>1</sup><em>sp<sup>1</sup></em> (FBst0002057), <em>Canton-S</em> (FBst0000001), <em>Oregon-R-C</em> (FBst0000005), <em>VAG_2</em> (FBst0003876), <em>Wild_11C</em> (FBst0003894), <em>BER_2</em> (FBst0003840) and <em>Wild_5A</em> (FBst0003885). The <em>w<sup>1118</sup></em>, <em>P-1152, NA-P</em>(1A) and <em>Lk-P</em>(1A) belong to our laboratory stock collection. <em>P-1152</em> carries two <em>P-lacZ</em> fusion enhancer trap transgenes (FBti0005700), inserted in <em>X-TAS</em>, that contain an in-frame translational fusion of the <em>E. coli lacZ</em> gene to the second exon of the <em>P</em> transposase gene, as well as the <em>rosy<sup>+</sup></em> gene as a transformation marker.<a reveal-id="dsx017-B19" data-open="dsx017-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsx017-B24" data-open="dsx017-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> The <em>Lk-P</em>(1A) line carries two full-length <em>P</em> elements inserted in <em>X-TAS</em>. <em>NA-P</em>(1A) carries a <em>P</em> element inserted in <em>X-TAS</em> that has undergone a telomeric deletion leaving a 5′ deleted <em>P</em> element. These telomeric insertions have been genetically isolated from inbred lines derived from wild populations collected in Lerik, Azerbaijan, for <em>Lk-P</em>(1A)<a reveal-id="dsx017-B13" data-open="dsx017-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> and from Nasr’Allah, Tunisia, for <em>NA-P</em>(1A)<a reveal-id="dsx017-B25" data-open="dsx017-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> in a genome devoid of other <em>P</em> elements. The <em>Drosophila sechellia</em> strain was kindly provided by Aurélie Hua-Van (EGCE, CNRS-IRD-University Paris-Sud-University Paris Saclay). Between 2013 and 2015, we collected <em>Drosophila</em> populations from the wild in Europe (France and Austria), Africa (Ethiopia and Senegal) and South America (Brazil and French Guiana). In French Guiana, flies were collected in Saul, an isolated town in the middle of the rainforest. The strains collected in 2013 in Marsais (France) and in 2012 in Gotheron (Valence, France) were kindly provided by Jean-Michel Gibert (LBD, IBPS, UPMC) and Cristina Vieira (LBBE, CNRS, University Lyon 1), respectively.</p>                            <h3 scrollto-destination=106355414 id="106355414" class="section-title" >2.2. Sequence accession numbers</h3>
<p>The following sequences were used in this study: <em>X-TAS</em> (L03284),<a reveal-id="dsx017-B1" data-open="dsx017-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> 2L-TAS (U35404),<a reveal-id="dsx017-B3" data-open="dsx017-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a><sup>,</sup><em>Canton-S</em> cosmid (AL031884),<a reveal-id="dsx017-B26" data-open="dsx017-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> clone AF520587,<a reveal-id="dsx017-B27" data-open="dsx017-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a><sup>,</sup><em>Invader 4</em> (FBgn0063427), <em>2R-TAS</em> and <em>3R-TAS</em> recovered from Lin and Yin, 2007<a reveal-id="dsx017-B7" data-open="dsx017-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> and <em>y</em><sup>1</sup><em>; cn</em><sup>1</sup><em>bw</em><sup>1</sup><em>sp<sup>1</sup></em> from <a class="link link-uri" href="http://FlyBase.org" target="_blank">FlyBase.org</a>.<a reveal-id="dsx017-B28" data-open="dsx017-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a></p>                            <h3 scrollto-destination=106355416 id="106355416" class="section-title" >2.3. Preparation of polytene chromosomes and fluorescence <em>in situ</em> hybridization</h3>
<p>Salivary glands from four to eight third instar larvae grown at 18°C were dissected and fixed in 45% acetic acid. After squashing, slides were quick-frozen in liquid nitrogen and dehydrated in ethanol (90%). Slides were then washed 30 min in 2× SSC at 65°C and dehydrated in ethanol baths (90%). Before hybridization, chromosomal DNA was denatured in NaOH 0.07N. Probes were prepared from PCR amplification products using primers allowing amplification of full length <em>TAS-B</em> repeat of 1.8 kb from the <em>P-1152</em> genome (<a reveal-id="dsx017-F1" data-open="dsx017-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>), a full length 2.9–kb <em>P</em> element from the <em>Lk-P</em>(1A) genome and the 855 bp repeats from the <em>D. sechellia</em> genome for <em>in situ</em> hybridization experiments in <a reveal-id="dsx017-F2" data-open="dsx017-F2" class="link link-reveal link-table xref-fig">Fig. 2</a>. Except for the 855 bp repeats, these PCR products were then cloned into the pCR2.1 plasmid (Thermofisher Scientific) for preparing either biotinylated DNA probes using the BioNick Labelling System (Invitrogen) and digoxigenin probes using the DIG-Nick Translation Mix (Roche). Slides were incubated with 1 μg of probe at 37°C overnight. Slides were washed two times in 2× SSC at 37°C. For detection, we used FITC-extravidine at 50 μg/ml (Sigma) and anti-DIG-rhodamine at 4 μg/ml (Roche) final concentrations. Slides were stained in DAPI-VectaShield mounting medium and observed using a Leica TCS SP5 confocal microscope system. </p>                        <a id="106355418" scrollto-destination="106355418"></a>
<div id="dsx017-F2" data-id="dsx017-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/m_dsx017f2.png?Expires=2147483647&amp;Signature=SfV8TmRryMIuJuFUXuuNrmDw3J6R-Y55Rg0VkcSQPJmDp72j5koKCi0ZkI2-GCDJ1dPvm0k~lSaZCjR72g5xOWinT-Am5T2FCdSIfddsKQLHUY1nmVxtgjqMvR8XVmy5RfjyjV4nxysVPaFeJTYgfQpl9k-S-S2i1OGTvhLcCTHFz1Er~kmejAVHvOAulzFVnofDVFMdy~b3AuKBniwJkLjlx-JcdhBP0uhLdSTaoNrnBoSND2ahlzpyXNBKSZ9PrCePWpp1o9pX~IWNWoSoTf27raWx-CU7p6OsZApJTP6KSsAbw-W1ZbVgT9xb15ckynXh7BQyCxvGMs9~rZbuZQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Subtelomeric piRNA clusters in the melanogaster subgroup. (A) D. sechellia Scaffold 31 contains five complete 855 bp repeats. Each repeat is composed of two subrepeats (191 and 138 bp) with sequence identities to D. melanogaster TLLs (hatched boxes), one 129 bp sequence derived from TLL (hatched boxes) (Supplementary Fig. S7) and two unrelated sequences of 274 bp and 123 bp. (B) D. simulans Scaffold 3R contains a 403-bp region composed of one TLL and two sequences of 105 and 129 bp displaying sequence identity with the 129 bp of D. sechellia followed by a T2 and a T5 domains. (C) In situ hybridizations on D. sechellia, D. simulans (Chantemesle) and D. melanogaster (Oregon-R-C) polytene chromosomes were performed using the D. sechellia 855 bp repeats as a probe. The probe was PCR amplified using primers shown by arrowheads in (A). Probe hybridizations at telomeres indicated by arrowheads are consistent with the in silico analysis. Labelling at the D. melanogaster centromeres can also be detected corresponding to smaller domains of identity (see text). (D–E) TLL-containing domains identified in D. sechellia and D. simulans are part of subtelomeric piRNA clusters. Normalized germline small RNAs were mapped to each reference sequences: the 855 bp D. sechellia repeat, the 403 bp D. simulans sequence and the 158 bp-TLL D. melanogaster sequences. (D) Size distribution histograms indicate that the majority of small RNAs produced from telomeric regions in ovaries of the three Drosophila species are between 23 and 29 nt in length. The number in each panel is the proportion of 23–29 nt beginning with a 5′ uridine (1 U bias). A bias exists toward sense (plus strand) vs. antisense (minus strand) strand production. (E) Readmaps show the abundance and the distribution of the piRNAs mapping to each of the reference sequences." path-from-xml="dsx017f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355418/dsx017f2.png" path-from-xml="dsx017f2.png" target="_blank">View large</a><a section="106355418" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017f2.png?Expires=2147483647&amp;Signature=QMnQj5ZBS29yDwdKfK3gnZji9MbmEGfDVxtsixNKXSzKigDwneA6oeqMeXgBhgf39coW~kQknTxgQp8qakosBJN46R6-mwBUtFEjEEhOa6X2Ei6C68TEp-JgxLMoI4d33M9xCwff4tbtIRz8bzStMPy0wGZoAq5QEsA5dbijbI7PKc6Gl0jq6oZvhktGOLdqosIEVxmLwa-POaBI5ZeXZsH-I-NqZWmq0Bf3jjjYcUZ0~5RAAMcxEcFpC5LyWrMdDODPXZLD3WO0sIGpptbz7qefefX3y~dFilk~7q9-pYrrE3WJVKMyxUJ-ekkOJHkl7Q9rqX124vJ5xjHo9yddZg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106355418&amp;ar=3778237&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx017f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Subtelomeric piRNA clusters in the <em>melanogaster</em> subgroup. (A) <em>D. sechellia</em> Scaffold 31 contains five complete 855 bp repeats. Each repeat is composed of two subrepeats (191 and 138 bp) with sequence identities to <em>D. melanogaster TLLs</em> (hatched boxes), one 129 bp sequence derived from <em>TLL</em> (hatched boxes) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&amp;Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>) and two unrelated sequences of 274 bp and 123 bp. (B) <em>D. simulans</em> Scaffold 3R contains a 403-bp region composed of one <em>TLL</em> and two sequences of 105 and 129 bp displaying sequence identity with the 129 bp of <em>D. sechellia</em> followed by a T2 and a T5 domains. (C) <em>In situ</em> hybridizations on <em>D. sechellia</em>, <em>D. simulans</em> (Chantemesle) and <em>D. melanogaster</em> (<em>Oregon-R-C</em>) polytene chromosomes were performed using the <em>D. sechellia</em> 855 bp repeats as a probe. The probe was PCR amplified using primers shown by arrowheads in (A). Probe hybridizations at telomeres indicated by arrowheads are consistent with the <em>in silico</em> analysis. Labelling at the <em>D. melanogaster</em> centromeres can also be detected corresponding to smaller domains of identity (see text). (D–E) <em>TLL</em>-containing domains identified in <em>D. sechellia</em> and <em>D. simulans</em> are part of subtelomeric piRNA clusters. Normalized germline small RNAs were mapped to each reference sequences: the 855 bp <em>D. sechellia</em> repeat, the 403 bp <em>D. simulans</em> sequence and the 158 bp<em>-TLL D. melanogaster</em> sequences. (D) Size distribution histograms indicate that the majority of small RNAs produced from telomeric regions in ovaries of the three <em>Drosophila</em> species are between 23 and 29 nt in length. The number in each panel is the proportion of 23–29 nt beginning with a 5′ uridine (1 U bias). A bias exists toward sense (plus strand) <em>vs</em>. antisense (minus strand) strand production. (E) Readmaps show the abundance and the distribution of the piRNAs mapping to each of the reference sequences.</p></div></div><div content-id="dsx017-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/m_dsx017f2.png?Expires=2147483647&amp;Signature=SfV8TmRryMIuJuFUXuuNrmDw3J6R-Y55Rg0VkcSQPJmDp72j5koKCi0ZkI2-GCDJ1dPvm0k~lSaZCjR72g5xOWinT-Am5T2FCdSIfddsKQLHUY1nmVxtgjqMvR8XVmy5RfjyjV4nxysVPaFeJTYgfQpl9k-S-S2i1OGTvhLcCTHFz1Er~kmejAVHvOAulzFVnofDVFMdy~b3AuKBniwJkLjlx-JcdhBP0uhLdSTaoNrnBoSND2ahlzpyXNBKSZ9PrCePWpp1o9pX~IWNWoSoTf27raWx-CU7p6OsZApJTP6KSsAbw-W1ZbVgT9xb15ckynXh7BQyCxvGMs9~rZbuZQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Subtelomeric piRNA clusters in the melanogaster subgroup. (A) D. sechellia Scaffold 31 contains five complete 855 bp repeats. Each repeat is composed of two subrepeats (191 and 138 bp) with sequence identities to D. melanogaster TLLs (hatched boxes), one 129 bp sequence derived from TLL (hatched boxes) (Supplementary Fig. S7) and two unrelated sequences of 274 bp and 123 bp. (B) D. simulans Scaffold 3R contains a 403-bp region composed of one TLL and two sequences of 105 and 129 bp displaying sequence identity with the 129 bp of D. sechellia followed by a T2 and a T5 domains. (C) In situ hybridizations on D. sechellia, D. simulans (Chantemesle) and D. melanogaster (Oregon-R-C) polytene chromosomes were performed using the D. sechellia 855 bp repeats as a probe. The probe was PCR amplified using primers shown by arrowheads in (A). Probe hybridizations at telomeres indicated by arrowheads are consistent with the in silico analysis. Labelling at the D. melanogaster centromeres can also be detected corresponding to smaller domains of identity (see text). (D–E) TLL-containing domains identified in D. sechellia and D. simulans are part of subtelomeric piRNA clusters. Normalized germline small RNAs were mapped to each reference sequences: the 855 bp D. sechellia repeat, the 403 bp D. simulans sequence and the 158 bp-TLL D. melanogaster sequences. (D) Size distribution histograms indicate that the majority of small RNAs produced from telomeric regions in ovaries of the three Drosophila species are between 23 and 29 nt in length. The number in each panel is the proportion of 23–29 nt beginning with a 5′ uridine (1 U bias). A bias exists toward sense (plus strand) vs. antisense (minus strand) strand production. (E) Readmaps show the abundance and the distribution of the piRNAs mapping to each of the reference sequences." path-from-xml="dsx017f2.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355418/dsx017f2.png" path-from-xml="dsx017f2.png" target="_blank">View large</a><a section="106355418" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017f2.png?Expires=2147483647&amp;Signature=QMnQj5ZBS29yDwdKfK3gnZji9MbmEGfDVxtsixNKXSzKigDwneA6oeqMeXgBhgf39coW~kQknTxgQp8qakosBJN46R6-mwBUtFEjEEhOa6X2Ei6C68TEp-JgxLMoI4d33M9xCwff4tbtIRz8bzStMPy0wGZoAq5QEsA5dbijbI7PKc6Gl0jq6oZvhktGOLdqosIEVxmLwa-POaBI5ZeXZsH-I-NqZWmq0Bf3jjjYcUZ0~5RAAMcxEcFpC5LyWrMdDODPXZLD3WO0sIGpptbz7qefefX3y~dFilk~7q9-pYrrE3WJVKMyxUJ-ekkOJHkl7Q9rqX124vJ5xjHo9yddZg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106355418&amp;ar=3778237&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx017f2.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Subtelomeric piRNA clusters in the <em>melanogaster</em> subgroup. (A) <em>D. sechellia</em> Scaffold 31 contains five complete 855 bp repeats. Each repeat is composed of two subrepeats (191 and 138 bp) with sequence identities to <em>D. melanogaster TLLs</em> (hatched boxes), one 129 bp sequence derived from <em>TLL</em> (hatched boxes) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&amp;Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>) and two unrelated sequences of 274 bp and 123 bp. (B) <em>D. simulans</em> Scaffold 3R contains a 403-bp region composed of one <em>TLL</em> and two sequences of 105 and 129 bp displaying sequence identity with the 129 bp of <em>D. sechellia</em> followed by a T2 and a T5 domains. (C) <em>In situ</em> hybridizations on <em>D. sechellia</em>, <em>D. simulans</em> (Chantemesle) and <em>D. melanogaster</em> (<em>Oregon-R-C</em>) polytene chromosomes were performed using the <em>D. sechellia</em> 855 bp repeats as a probe. The probe was PCR amplified using primers shown by arrowheads in (A). Probe hybridizations at telomeres indicated by arrowheads are consistent with the <em>in silico</em> analysis. Labelling at the <em>D. melanogaster</em> centromeres can also be detected corresponding to smaller domains of identity (see text). (D–E) <em>TLL</em>-containing domains identified in <em>D. sechellia</em> and <em>D. simulans</em> are part of subtelomeric piRNA clusters. Normalized germline small RNAs were mapped to each reference sequences: the 855 bp <em>D. sechellia</em> repeat, the 403 bp <em>D. simulans</em> sequence and the 158 bp<em>-TLL D. melanogaster</em> sequences. (D) Size distribution histograms indicate that the majority of small RNAs produced from telomeric regions in ovaries of the three <em>Drosophila</em> species are between 23 and 29 nt in length. The number in each panel is the proportion of 23–29 nt beginning with a 5′ uridine (1 U bias). A bias exists toward sense (plus strand) <em>vs</em>. antisense (minus strand) strand production. (E) Readmaps show the abundance and the distribution of the piRNAs mapping to each of the reference sequences.</p></div></div>                            <h3 scrollto-destination=106355419 id="106355419" class="section-title" >2.4. Sequence analysis</h3>
<p>Sequence comparisons were performed using BLAST (NCBI) and SeaView using MUSCLE for multiple sequence alignments.<a reveal-id="dsx017-B29" data-open="dsx017-B29" class="link link-ref link-reveal xref-bibr"><sup>29</sup></a> Minor manual gap adjustments were introduced when needed.</p>                            <h3 scrollto-destination=106355421 id="106355421" class="section-title" >2.5. DNA extraction and PCR amplification</h3>
<p>Flies were manually homogenized in a buffer containing 0.1-M Tris pH9, 0.1-M EDTA pH8 and 1% SDS and then incubated at 70°C for 30 min. Potassium Acetate 8 M and isopropanol were added prior centrifugation. Pellets of DNA were washed and resuspended in water and PCR was performed using primers (available upon request). This protocol was used for DNA extraction from single flies (<a reveal-id="dsx017-T2" data-open="dsx017-T2" class="link link-reveal link-table xref-fig">Tables 2</a> and <a reveal-id="dsx017-T3" data-open="dsx017-T3" class="link link-reveal link-table xref-fig">3</a>, <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_tables.pdf?Expires=2147483647&Signature=FwvzbfDMugjK1KDcWcYS3hV8j4C9hBqZi4cFJAIEqzTEGwmQGIl3Efr4o0BNod5Da5pJKh15VBV~IUtuNvZktizgJg~9FCXHAiZydTFr-jfgdPKL51SQUgT8JGwEb7DBgeG0oeICN8HVkOb~a9T5CtRb1zV3rvqAwWCdxz23ue56MdKE13LpJ~V~WFpdzkdYifuwuqjpHHoCTNdqoN3I1AdJ~SqLbv4fxbPx2tFTv22Hgc2yDzuX-~4pKDk2TDlukWMjQUCm7Q08N5UMs3uzjx8vHY9-Ms7rr9ZO-Osei8Rli~Y8BLfuMzrDvtRgCEkphuoLu99zb3263UmEnQ6Low__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S2 and S3</a></span>) as well as from groups of twenty to fifty flies (<a reveal-id="dsx017-F3" data-open="dsx017-F3" class="link link-reveal link-table xref-fig">Fig. 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_tables.pdf?Expires=2147483647&Signature=FwvzbfDMugjK1KDcWcYS3hV8j4C9hBqZi4cFJAIEqzTEGwmQGIl3Efr4o0BNod5Da5pJKh15VBV~IUtuNvZktizgJg~9FCXHAiZydTFr-jfgdPKL51SQUgT8JGwEb7DBgeG0oeICN8HVkOb~a9T5CtRb1zV3rvqAwWCdxz23ue56MdKE13LpJ~V~WFpdzkdYifuwuqjpHHoCTNdqoN3I1AdJ~SqLbv4fxbPx2tFTv22Hgc2yDzuX-~4pKDk2TDlukWMjQUCm7Q08N5UMs3uzjx8vHY9-Ms7rr9ZO-Osei8Rli~Y8BLfuMzrDvtRgCEkphuoLu99zb3263UmEnQ6Low__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Since the absence of a PCR amplification product could be attributed to poor DNA quality, we confirmed the amplificability of DNA samples using independent sets of primers (data not shown). </p>                        <a id="106355423" scrollto-destination="106355423"></a>
<div id="dsx017-F3" data-id="dsx017-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/m_dsx017f3.png?Expires=2147483647&Signature=ohqbut4ViA74Dg1MYpTOdul4oDNp9e6G-b9QDr6wvbebKv~ficQxRcJ2Z9qaZCl-O2fKD3V8Leg0zY49Luo9tzxekd8CkizzOeX5Vav2E3w5Mcg8feRumvCQEFwgpFBa98zsCb1zOnWDUGo7NckQ3Qb9AScrlPRZWpzpoOVJpMsl9d3-T4gZf444XLsbbrF3hjRXYurWK8jsfeZSvwPBoKDIAIiJq0976FYy0xer7LyibMl31IHBW4rHlWYIsjBXLC2hie9HRr9YPMs4hwhzIGRSR3RCVo7vGqhUFyQbnzlA7MSvTtIqL8xMfZkOUGtuVX66W4R5FvxApEMoVg7c3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of X subtelomeric regions after X-TAS loss. Four types of X subtelomeres were identified: (A) with one each of the TAS-B, -C, -D and the 0.9 kb repeats like the Dp1187 minichromosome1; (B) with only theTAS-D repeat and the 0.9 kb repeats like NA-P(1A)25; (C) with only the 0.9 kb repeats like Canton-S; (D) with neither TAS repeats nor the 0.9 kb repeats like y1; cn1bw1sp1. Stocks found to be devoid of X-TAS repeats by in situ hybridization and/or PCR amplification were tested to identify their X subtelomeric structure. This was assayed by PCR amplifications using specific primers for TAS-B (see arrows under), TAS-D (see arrows) and the 0.9 kb repeats (see arrows) (A). Noticeably, the 0.9 kb repeats were systematically amplified in stocks that possess TAS-B and/or TAS-D. The 0.9 kb repeats can also be stably maintained on their own at the tip of the X chromosome as observed for the Canton-S strain. Strains shown to have recently lost X-TAS (see Table 3) are marked by an asterisk (*). Loss of X-TAS might occur by telomeric terminal breakage rather than by internal deletion that might be repaired by transposition of new telomeric HTT (HeT-A, TAHRE)." path-from-xml="dsx017f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355423/dsx017f3.png" path-from-xml="dsx017f3.png" target="_blank">View large</a><a section="106355423" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017f3.png?Expires=2147483647&Signature=RqnXDBjCqsb6tBZBYWXtKVECDE3W2xiNNrxrTkhyAvhgIeLYoK6s0ceZ0ZTzG3tcRlUJdoWv1CQwpw7C2Hg~C-884ffphHyRZiE6mrIQzQmpM-aRM0mMeL9xv04cNlJwsN8SKZXS1fXiCi8xbRfLb8Yt8kArVhlkCEaL61vIcH00l0N9sSJl~ZHgnXsLpR4OIBQ-SYWiWhvXEProYhxg5y6BeKv0pYthauBW-xs2TuAXUgBLYqy5UaOMR2ubaq1hzWK2zauO7kRZaGW-G49WnpRPWbvcJWi6ZBi2TfnDKEU68N7yQ-nMD30wwXL5SDaYxIWvw3YEn8lGSENJiaqF1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355423&ar=3778237&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx017f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of <em>X</em> subtelomeric regions after <em>X-TAS</em> loss. Four types of <em>X</em> subtelomeres were identified: (A) with one each of the <em>TAS-B, -C, -D</em> and the 0.9 kb repeats like the <em>Dp1187</em> minichromosome<a reveal-id="dsx017-B1" data-open="dsx017-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a>; (B) with only the<em>TAS-D</em> repeat and the 0.9 kb repeats like <em>NA-P</em>(1A)<a reveal-id="dsx017-B25" data-open="dsx017-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a>; (C) with only the 0.9 kb repeats like <em>Canton-S</em>; (D) with neither TAS repeats nor the 0.9 kb repeats like <em>y</em><sup><em>1</em></sup><em>; cn</em><sup><em>1</em></sup><em>bw</em><sup><em>1</em></sup><em>sp<sup>1</sup></em>. Stocks found to be devoid of <em>X-TAS</em> repeats by <em>in situ</em> hybridization and/or PCR amplification were tested to identify their <em>X</em> subtelomeric structure. This was assayed by PCR amplifications using specific primers for <em>TAS-B</em> (see arrows under), <em>TAS-D</em> (see arrows) and the 0.9 kb repeats (see arrows) (A). Noticeably, the 0.9 kb repeats were systematically amplified in stocks that possess <em>TAS-B</em> and/or <em>TAS-D</em>. The 0.9 kb repeats can also be stably maintained on their own at the tip of the <em>X</em> chromosome as observed for the <em>Canton-S</em> strain. Strains shown to have recently lost <em>X-TAS</em> (see <a reveal-id="dsx017-T3" data-open="dsx017-T3" class="link link-reveal link-table xref-fig">Table 3</a>) are marked by an asterisk (*). Loss of <em>X-TAS</em> might occur by telomeric terminal breakage rather than by internal deletion that might be repaired by transposition of new telomeric <em>HTT</em> (<em>HeT-A, TAHRE</em>).</p></div></div><div content-id="dsx017-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/m_dsx017f3.png?Expires=2147483647&Signature=ohqbut4ViA74Dg1MYpTOdul4oDNp9e6G-b9QDr6wvbebKv~ficQxRcJ2Z9qaZCl-O2fKD3V8Leg0zY49Luo9tzxekd8CkizzOeX5Vav2E3w5Mcg8feRumvCQEFwgpFBa98zsCb1zOnWDUGo7NckQ3Qb9AScrlPRZWpzpoOVJpMsl9d3-T4gZf444XLsbbrF3hjRXYurWK8jsfeZSvwPBoKDIAIiJq0976FYy0xer7LyibMl31IHBW4rHlWYIsjBXLC2hie9HRr9YPMs4hwhzIGRSR3RCVo7vGqhUFyQbnzlA7MSvTtIqL8xMfZkOUGtuVX66W4R5FvxApEMoVg7c3Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic representation of X subtelomeric regions after X-TAS loss. Four types of X subtelomeres were identified: (A) with one each of the TAS-B, -C, -D and the 0.9 kb repeats like the Dp1187 minichromosome1; (B) with only theTAS-D repeat and the 0.9 kb repeats like NA-P(1A)25; (C) with only the 0.9 kb repeats like Canton-S; (D) with neither TAS repeats nor the 0.9 kb repeats like y1; cn1bw1sp1. Stocks found to be devoid of X-TAS repeats by in situ hybridization and/or PCR amplification were tested to identify their X subtelomeric structure. This was assayed by PCR amplifications using specific primers for TAS-B (see arrows under), TAS-D (see arrows) and the 0.9 kb repeats (see arrows) (A). Noticeably, the 0.9 kb repeats were systematically amplified in stocks that possess TAS-B and/or TAS-D. The 0.9 kb repeats can also be stably maintained on their own at the tip of the X chromosome as observed for the Canton-S strain. Strains shown to have recently lost X-TAS (see Table 3) are marked by an asterisk (*). Loss of X-TAS might occur by telomeric terminal breakage rather than by internal deletion that might be repaired by transposition of new telomeric HTT (HeT-A, TAHRE)." path-from-xml="dsx017f3.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355423/dsx017f3.png" path-from-xml="dsx017f3.png" target="_blank">View large</a><a section="106355423" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017f3.png?Expires=2147483647&Signature=RqnXDBjCqsb6tBZBYWXtKVECDE3W2xiNNrxrTkhyAvhgIeLYoK6s0ceZ0ZTzG3tcRlUJdoWv1CQwpw7C2Hg~C-884ffphHyRZiE6mrIQzQmpM-aRM0mMeL9xv04cNlJwsN8SKZXS1fXiCi8xbRfLb8Yt8kArVhlkCEaL61vIcH00l0N9sSJl~ZHgnXsLpR4OIBQ-SYWiWhvXEProYhxg5y6BeKv0pYthauBW-xs2TuAXUgBLYqy5UaOMR2ubaq1hzWK2zauO7kRZaGW-G49WnpRPWbvcJWi6ZBi2TfnDKEU68N7yQ-nMD30wwXL5SDaYxIWvw3YEn8lGSENJiaqF1Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355423&ar=3778237&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx017f3.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic representation of <em>X</em> subtelomeric regions after <em>X-TAS</em> loss. Four types of <em>X</em> subtelomeres were identified: (A) with one each of the <em>TAS-B, -C, -D</em> and the 0.9 kb repeats like the <em>Dp1187</em> minichromosome<a reveal-id="dsx017-B1" data-open="dsx017-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a>; (B) with only the<em>TAS-D</em> repeat and the 0.9 kb repeats like <em>NA-P</em>(1A)<a reveal-id="dsx017-B25" data-open="dsx017-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a>; (C) with only the 0.9 kb repeats like <em>Canton-S</em>; (D) with neither TAS repeats nor the 0.9 kb repeats like <em>y</em><sup><em>1</em></sup><em>; cn</em><sup><em>1</em></sup><em>bw</em><sup><em>1</em></sup><em>sp<sup>1</sup></em>. Stocks found to be devoid of <em>X-TAS</em> repeats by <em>in situ</em> hybridization and/or PCR amplification were tested to identify their <em>X</em> subtelomeric structure. This was assayed by PCR amplifications using specific primers for <em>TAS-B</em> (see arrows under), <em>TAS-D</em> (see arrows) and the 0.9 kb repeats (see arrows) (A). Noticeably, the 0.9 kb repeats were systematically amplified in stocks that possess <em>TAS-B</em> and/or <em>TAS-D</em>. The 0.9 kb repeats can also be stably maintained on their own at the tip of the <em>X</em> chromosome as observed for the <em>Canton-S</em> strain. Strains shown to have recently lost <em>X-TAS</em> (see <a reveal-id="dsx017-T3" data-open="dsx017-T3" class="link link-reveal link-table xref-fig">Table 3</a>) are marked by an asterisk (*). Loss of <em>X-TAS</em> might occur by telomeric terminal breakage rather than by internal deletion that might be repaired by transposition of new telomeric <em>HTT</em> (<em>HeT-A, TAHRE</em>).</p></div></div>                            <h3 scrollto-destination=106355424 id="106355424" class="section-title" >2.6. Small RNA sequencing</h3>
<p>Total RNA extraction, small RNA library preparation and sequencing were performed as described in Hermant et al.<a reveal-id="dsx017-B30" data-open="dsx017-B30" class="link link-ref link-reveal xref-bibr"><sup>30</sup></a> Sequence reads in FASTQ format were trimmed from the adapter sequence 5′-TGGAATTCTCGGGTGCCAAG-3′ and matched to reference sequences (TAS<em>, P</em> element, or <em>D. sechellia</em> repeats) using Bowtie.<a reveal-id="dsx017-B31" data-open="dsx017-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Multiple mappers from 23 to 29 nt matching the reference sequences with zero mismatches were retained for subsequent analysis. Annotation of small RNA libraries and normalization factors are described in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_tables.pdf?Expires=2147483647&Signature=FwvzbfDMugjK1KDcWcYS3hV8j4C9hBqZi4cFJAIEqzTEGwmQGIl3Efr4o0BNod5Da5pJKh15VBV~IUtuNvZktizgJg~9FCXHAiZydTFr-jfgdPKL51SQUgT8JGwEb7DBgeG0oeICN8HVkOb~a9T5CtRb1zV3rvqAwWCdxz23ue56MdKE13LpJ~V~WFpdzkdYifuwuqjpHHoCTNdqoN3I1AdJ~SqLbv4fxbPx2tFTv22Hgc2yDzuX-~4pKDk2TDlukWMjQUCm7Q08N5UMs3uzjx8vHY9-Ms7rr9ZO-Osei8Rli~Y8BLfuMzrDvtRgCEkphuoLu99zb3263UmEnQ6Low__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>. Sequence length distributions and small RNA mapping were generated from Bowtie alignments using Python and R (<a class="link link-uri" href="http://www.r-project.org/" target="_blank">www.r-project.org/</a>) scripts, which were wrapped and run in a Galaxy instance publicly available at <a class="link link-uri" href="http://plastissipi.fr" target="_blank">http://plastissipi.fr</a>. Tools and workflows used in this study can be downloaded from this Galaxy instance.</p>                            <h3 scrollto-destination=106355426 id="106355426" class="section-title" >2.7. Data availability</h3>
<p>Small RNA sequences have been deposited at the European Nucleotide Archive (ENA) under accession number PRJEB19350.</p>                            <h2 scrollto-destination=106355428 id="106355428" class="section-title" >3. Results and conclusions</h2>
                            <h3 scrollto-destination=106355429 id="106355429" class="section-title" >3.1. Structural analysis of TAS-R family members reveals that they are more highly related than previously suspected</h3>
<p><em>X</em>, <em>2R</em> and <em>3R</em> chromosomal arms in <em>D. melanogaster</em> contain subtelomeric TAS-R sequences called <em>X-TAS</em>, <em>2R-TAS</em> and <em>3R-TAS</em>, respectively, though not all strains present all three. Despite their fundamental role as piRNA clusters involved in repression of TEs (such as <em>P</em> element), little is known about their dynamics. To address this question, we first revisited the precise structure of these TAS.</p><p>The <em>X</em> subtelomeric region was studied by revisiting the <em>Dp1187 X</em> minichromosome sequence encoding the only complete <em>X-TAS</em> locus sequenced to date.<a reveal-id="dsx017-B1" data-open="dsx017-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> According to this initial study, it contains a tandem of four TAS repeats, named <em>TAS-A</em> to <em>-D</em>, upstream of two 0.9 kb repeats (<a reveal-id="dsx017-F1" data-open="dsx017-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>). <em>TAS-B</em> (1,872 bp), the longest, was considered to be the reference for <em>X-TAS</em> repeats.<a reveal-id="dsx017-B1" data-open="dsx017-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><em>TAS-A</em> (205 bp) appears to originate from a telomeric 5′ terminal deletion within a longer TAS repeat repaired by telomeric <em>HTT</em> transposition events. <em>TAS-C</em> (1,858 bp) differs by a few indels from <em>TAS-B</em>. <em>TAS-D</em> (1,575 bp, 95% identical with <em>TAS-B</em>) contains a 313-bp deletion within the 3′ region of the unit compared to <em>TAS-B</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). To revisit the portion of TAS-R originating from <em>INV-4</em> LTRs, we performed sequence comparisons between the LTR of the canonical <em>INV-4</em> 346 bp sequence and TAS-R sequences. We found that the portion of the TAS-R that matches with the <em>INV-4</em> LTR were more extensive than the three 172 bp repeats originally described.<a reveal-id="dsx017-B1" data-open="dsx017-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx017-B5" data-open="dsx017-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> For instance, the three partial <em>INV-4</em> LTR sequences identified in <em>X-TAS</em> were 248, 172 and 235 bp in length with more than 83% similarity with the canonical LTR. <em>2R-TAS</em> was shown to contain four tandem <em>INV-4</em> LTR partial sequences and <em>3R-TAS</em>, three <em>INV-4</em> LTR partial sequences, two of which were longer than 172 bp (for details see <a reveal-id="dsx017-F1" data-open="dsx017-F1" class="link link-reveal link-table xref-fig">Fig. 1B–D</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>).</p><p>We next asked whether the shared origin between the <em>X-TAS</em>, <em>2R-TAS</em> and <em>3R-TAS</em> was limited to the <em>INV-4</em> LTR region. By performing multiple alignments between the remnants of the three loci, five other domains were identified that we named T1, T2, T3, T4 and T5 (<a reveal-id="dsx017-F1" data-open="dsx017-F1" class="link link-reveal link-table xref-fig">Fig. 1B–D</a>). The T1 and T4 domains were present in all three TAS-R loci, the T2 domain in <em>X-TAS</em> and <em>3R-TAS</em>, and the T5 domain in 2<em>R-TAS</em> and <em>3R-TAS</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). Only the T3 domain, 909 bp in length, was found to be restricted to one TAS-R family member, <em>X-TAS</em>, except for an internal domain (see below).</p><p>Taken together, the characterization of TAS-R sequences indicates that members of this family are more closely related to each other than originally suspected. Indeed, they are a mosaic of six domains (T1–T5 and <em>INV-4</em>) organized in a specific order at subtelomeres.</p>                            <h3 scrollto-destination=106355434 id="106355434" class="section-title" >3.2. TAS-R and TAS-L families are related</h3>
<p>We next investigated the origin of each of the “T”domains (T1–T5) identified above. Using several databases, the only sequence alignments recovered corresponded to those of a 158-bp region located in the T3 domain with sequences in subtelomeric TAS-L family repeats located on the <em>2L</em> and <em>3L</em> chromosomes (<a reveal-id="dsx017-F1" data-open="dsx017-F1" class="link link-reveal link-table xref-fig">Fig. 1B and E</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>) as well with sequences in the two domains of the 0.9 kb repeats next to <em>X-TAS</em> (<a reveal-id="dsx017-F1" data-open="dsx017-F1" class="link link-reveal link-table xref-fig">Fig. 1B and F</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5</a></span>). The <em>2L-TAS</em> and <em>3L-TAS</em> are identical subtelomeric minisatellite repeats.<a reveal-id="dsx017-B3" data-open="dsx017-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> In the <em>y</em><sup><em>1</em></sup><em>; cn</em><sup><em>1</em></sup><em>bw</em><sup><em>1</em></sup><em>sp<sup>1</sup></em> reference genome (<a class="link link-uri" href="http://FlyBase.org" target="_blank">FlyBase.org</a>), we found 11 repeats and 42 repeats of 458 bp on the <em>2L</em> and <em>3L</em> subtelomeres, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>). Such similarities between TAS-R and TAS-L families have been suggested recently.<a reveal-id="dsx017-B9" data-open="dsx017-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a><sup>,</sup><a reveal-id="dsx017-B10" data-open="dsx017-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> In this study, we specify these similarities showing that TAS-L repeats consists of six tandemly-repeated sequences of 147, 53, 29, 44, 45 and 20 bp in length that share over 80% sequence similarity with the 158 bp region identified in the T3 domain of <em>X-TAS</em> (<a reveal-id="dsx017-F1" data-open="dsx017-F1" class="link link-reveal link-table xref-fig">Fig. 1E</a>). For this reason, we named this 158 bp domain, the <em>TLL</em> domain. BLAST searches using the whole T3 domain also unveiled similarities between <em>TLL</em> and an unmapped clone composed of fragments of TEs isolated from a <em>D. melanogaster</em> strain collected in Samarkand, Uzbekistan (73% identity)<a reveal-id="dsx017-B27" data-open="dsx017-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> as well as with two domains of 146 bp in pericentromeric regions of <em>y</em><sup><em>1</em></sup><em>; cn</em><sup><em>1</em></sup><em>bw</em><sup><em>1</em></sup><em>sp<sup>1</sup></em> (∼80% identity, data not shown).</p><p>With the identification of the <em>TLL</em> domain within chromosomic regions carrying remnants of TEs, it is tempting to speculate that <em>TLL</em> could be of TE origin. Since the <em>TLL</em> domain was found to be part of a TAS-R member, <em>X-TAS</em>, of the adjacent 0.9 kb repeats and of the TAS-L family repeats, it can be proposed to represent the common ancestor sequence of the main subtelomeres in the <em>D. melanogaster</em> genome.</p>                            <h3 scrollto-destination=106355437 id="106355437" class="section-title" >3.3. TAS sequences in the melanogaster subgroup are likely piRNA clusters</h3>
<p>After the identification of the modular structure of TAS loci and the <em>TLL</em> core domain common between <em>D. melanogaster</em> TAS-R and TAS-L, we addressed the evolutionary origin of these sequences by studying other Drosophilidae. We performed BLAST searches using the T3 sequence against other published <em>Drosophila</em> genomes (<a class="link link-uri" href="http://FlyBase.org" target="_blank">FlyBase.org</a>). In <em>D. sechellia</em>, we found nine scaffolds containing sequences ∼80% identical to the <em>TLL</em> of <em>D. melanogaster</em>. Scaffold 31 consists of an array of five 855-bp repeats that were identified earlier.<a reveal-id="dsx017-B9" data-open="dsx017-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> Here, we found that each repeat contains two <em>TLLs</em> of 191 and 138 bp, and two unrelated sequences of 403 and 123 bp (<a reveal-id="dsx017-F2" data-open="dsx017-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7A</a></span>). This array is flanked on one side by a <em>TART</em> telomeric element and on the other side by a region containing a T5 domain, a T2 domain (&gt;90 and 85% identity, respectively) and, 8.8 kb from T2, the <em>GM16406</em> gene, the ortholog of <em>D. melanogaster Map205</em> the most distal gene on <em>3R</em> chromosome. Scaffold 14 contains a similar 855 bp domain in proximity to the ortholog of the <em>D. melanogaster l(2)gl</em> gene, the most distal gene on <em>2L</em> (data not shown). Using the <em>D. sechellia</em> 855 bp repeat, BLAST searches identified eight scaffolds within the <em>D. simulans</em> genome. Among them, Scaffold 3R contains the <em>GD16184</em> gene, the ortholog of the <em>Map205</em> gene 2.3 kb away from a region containing a T2 domain (86% identity), a T5 domain (90% identity), a <em>TLL</em> (&gt;80% identity) and two related domains of 105 and 129 bp (<a reveal-id="dsx017-F2" data-open="dsx017-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). Moreover, syntenic regions consisting of one <em>TLL</em> and one 129 bp-like sequence were found in the 855 bp repeats of <em>D. sechellia</em>, in <em>D. simulans</em> and in the <em>D. melanogaster</em> TAS-L and 0.9-kb repeat (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7B and C</a></span>). Results of the alignment between the <em>D. sechellia</em> 129-bp sequence and the <em>D. melanogaster</em> TAS-L strongly suggest that the 129-bp sequence and <em>TLL</em> are related (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7B and C</a></span>). In <em>D. erecta</em>, seven scaffolds were identified containing smaller <em>TLL</em> repeats (60–65 bp) 85% identical to the <em>D. sechellia TLL</em>. Among them, Scaffold 4820 contains four tandemly repeated domains of 998 bp which include an 128-bp sequence aligning with the <em>TLL</em> and which are at a distance of 13 kb from the ortholog of <em>l(2)gl</em> (<em>GG24690</em>) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7A</a></span>). Finally, in <em>D. yakuba</em>, Scaffold 2L contains two almost identical <em>TLLs</em> of 64 bp, spaced by a region of 102 bp, 2 kb from the <em>GE16527</em> gene, the ortholog of <em>l(2)gl</em> in this species (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7A</a></span>). Six other scaffolds in <em>D. yakuba</em> contain ∼170 bp 88% identical to the 274 bp internal sequence of the <em>D. sechellia</em> 855 bp repeats. The conserved synteny within the <em>melanogaster</em> subgroup suggested a telomeric location that was confirmed by <em>in situ</em> hybridization performed on polytene chromosomes from <em>D. sechellia</em> and <em>D. simulans</em> using a probe prepared with the 855 bp repeat of <em>D. sechellia</em> (<a reveal-id="dsx017-F2" data-open="dsx017-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). When hybridization was performed on <em>D. melanogaster</em> chromosomes (<em>Oregon-R-C</em>), the 855 bp probe labelled <em>X-TAS</em>, <em>2L-TAS</em> and <em>3L-TAS</em>, as well as <em>3R-TAS</em> and on the chromocenter (<a reveal-id="dsx017-F2" data-open="dsx017-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). Finally, analysis of small RNAs extracted from <em>D. sechellia, D. simulans</em> and <em>D. melanogaster</em> ovaries revealed that the repeats identified in these species are able to produce small RNAs of a restricted size distribution of 23–29 nt with a clear uridine bias at their 5′ end (1U bias) (<a reveal-id="dsx017-F2" data-open="dsx017-F2" class="link link-reveal link-table xref-fig">Fig. 2D and E</a>) that are two key features of piRNAs.<a reveal-id="dsx017-B20" data-open="dsx017-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> Therefore, these results strongly suggested that the conserved repeats identified in these species represent subtelomeric piRNA clusters.</p>                            <h3 scrollto-destination=106355439 id="106355439" class="section-title" >3.4. Distribution of TAS-R family in <em>D. melanogaster</em> laboratory stocks</h3>
<p>In order to address the question as to whether variability in the presence of TAS sequences exists between strains, we conducted an extensive analysis of various laboratory stocks focusing on the TAS-R family. The <em>y</em><sup><em>1</em></sup><em>; cn</em><sup><em>1</em></sup><em>bw</em><sup><em>1</em></sup><em>sp<sup>1</sup></em> genome contains at least six <em>3R-TAS</em>, one <em>2R-TAS</em> but no <em>X-TAS</em> repeats (<a class="link link-uri" href="http://flybase.org" target="_blank">flybase.org</a>), while molecular analysis identified <em>X-TAS</em> in <em>Lk-P</em>(1A), <em>NA-P</em>(1A) and <em>P-1152</em>.<a reveal-id="dsx017-B13" data-open="dsx017-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsx017-B19" data-open="dsx017-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a><sup>,</sup><a reveal-id="dsx017-B25" data-open="dsx017-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a></p><p>We first assayed the distribution of TAS-R within the five unrelated standard laboratory stocks, i.e. <em>Canton-S</em>, <em>Oregon-R-C, w<sup>1118</sup>, yw</em><sup>c</sup> and <em>y</em><sup><em>1</em></sup><em>cn</em><sup><em>1</em></sup><em>bw</em><sup><em>1</em></sup><em>sp<sup>1</sup></em>, by performing <em>in situ</em> hybridization on polytene chromosomes of salivary glands of third instar larvae with a <em>D. melanogaster X-TAS</em> probe (<a reveal-id="dsx017-F1" data-open="dsx017-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). This probe is expected to label specifically the three TAS-R loci due to the number of shared domains between the TAS-R family members. Control experiments using <em>Lk-P</em>(1A) and <em>P-1152</em> lines showed strong labellings of the <em>X</em> and <em>2R</em> telomeres for <em>Lk-P</em>(1A), and of the <em>X</em> and <em>3R</em> telomeres for <em>P-1152</em> (<a reveal-id="dsx017-F4" data-open="dsx017-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). As summarized in <a reveal-id="dsx017-T1" data-open="dsx017-T1" class="link link-reveal link-table xref-fig">Table 1</a>, the five standard laboratory strains contain TAS on <em>2R</em> and/or <em>3R</em> autosomal loci and none on the <em>X</em> chromosome, except for <em>Oregon-R-C</em> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). It is important to note that depending on individual stocks tested, <em>Oregon-R-C</em> possesses (this study) or lacks<a reveal-id="dsx017-B32" data-open="dsx017-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> the <em>X-TAS</em> subtelomeric locus. <em>In situ</em> hybridization results for the presence/absence of <em>X-TAS</em> were consistent with results of PCR amplifications using specific primers for the T3 domain present in <em>TAS-B</em> (<a reveal-id="dsx017-T1" data-open="dsx017-T1" class="link link-reveal link-table xref-fig">Table 1</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>), providing us with an easy assay to systematically screen for the presence/absence of <em>X-TAS</em>. Using sequences available in databases,<a reveal-id="dsx017-B1" data-open="dsx017-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx017-B26" data-open="dsx017-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a><sup>,</sup><a reveal-id="dsx017-B28" data-open="dsx017-B28" class="link link-ref link-reveal xref-bibr"><sup>28</sup></a> absence of <em>X-TAS</em> repeats in <em>Canton-S</em> and <em>y</em><sup><em>1</em></sup><em>; cn</em><sup><em>1</em></sup><em>bw</em><sup><em>1</em></sup><em>sp<sup>1</sup></em> was confirmed and presence (<em>Canton-S</em>) or absence (<em>y</em><sup><em>1</em></sup><em>; cn</em><sup><em>1</em></sup><em>bw</em><sup><em>1</em></sup><em>sp<sup>1</sup></em>) of the 0.9 kb repeats was demonstrated (<a reveal-id="dsx017-F1" data-open="dsx017-F1" class="link link-reveal link-table xref-fig">Fig. 1G and H</a>). <div content-id="dsx017-T1" class="table-modal"><div class="table-wrap"><div id="dsx017-T1" data-id="dsx017-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Distribution of TAS family in various laboratory strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Year of capture </th><th><strong>T3 domain</strong><a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th><th><strong><em>X-TAS</em></strong><a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th><th><strong><em>2R-TAS</em></strong><a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th><th><strong><em>3R-TAS</em></strong><a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th><th><strong>TAS-L</strong><a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th></tr></thead><tbody><tr><td><em>Lk-P</em>(1A)<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><sup>,c</sup> </td><td>NA </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td><em>NA-P</em>(1A)<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><sup>,c</sup> </td><td>NA </td><td>− </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td><em>P-1152</em><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><sup>,c</sup> </td><td>NA </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td><em>w<sup>1118</sup></em><sup>a</sup> </td><td>NA </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>yw</em><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>NA </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>y<sup>1</sup>; cn<sup>1</sup> bw<sup>1</sup> sp<sup>1</sup></em><sup>b</sup> </td><td>NA </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>Canton-S</em><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, OH </td><td>1935 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>Oregon-R-C</em><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, OR </td><td>1925 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Gruta<sup>a</sup>, France </td><td>1950 </td><td>− </td><td>− </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>BER_2<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, Bermuda </td><td>1954 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>VAG_2<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, Greece </td><td>1965 </td><td>+ </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td></tr><tr><td>Wild_11C<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, NC </td><td>1966 </td><td>+ </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td></tr><tr><td>Wild_5A<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, GA </td><td>1966 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>Tautavel<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, France </td><td>1967 </td><td>− </td><td>− </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>Harwich<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, MA </td><td>1967 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Chatelet<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, France </td><td>1970 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Para Wirra<sup>a</sup>, Australia </td><td>1972 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Tachkent<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, Uzbekistan </td><td>1981 </td><td>− </td><td>− </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>Chimkent<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, Kazakhstan </td><td>1983 </td><td>− </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td></tr></tbody></table></div><div content-id="dsx017-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Year of capture </th><th><strong>T3 domain</strong><a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th><th><strong><em>X-TAS</em></strong><a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th><th><strong><em>2R-TAS</em></strong><a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th><th><strong><em>3R-TAS</em></strong><a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th><th><strong>TAS-L</strong><a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th></tr></thead><tbody><tr><td><em>Lk-P</em>(1A)<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><sup>,c</sup> </td><td>NA </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td><em>NA-P</em>(1A)<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><sup>,c</sup> </td><td>NA </td><td>− </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td><em>P-1152</em><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><sup>,c</sup> </td><td>NA </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td><em>w<sup>1118</sup></em><sup>a</sup> </td><td>NA </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>yw</em><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>NA </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>y<sup>1</sup>; cn<sup>1</sup> bw<sup>1</sup> sp<sup>1</sup></em><sup>b</sup> </td><td>NA </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>Canton-S</em><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, OH </td><td>1935 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>Oregon-R-C</em><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, OR </td><td>1925 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Gruta<sup>a</sup>, France </td><td>1950 </td><td>− </td><td>− </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>BER_2<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, Bermuda </td><td>1954 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>VAG_2<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, Greece </td><td>1965 </td><td>+ </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td></tr><tr><td>Wild_11C<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, NC </td><td>1966 </td><td>+ </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td></tr><tr><td>Wild_5A<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, GA </td><td>1966 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>Tautavel<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, France </td><td>1967 </td><td>− </td><td>− </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>Harwich<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, MA </td><td>1967 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Chatelet<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, France </td><td>1970 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Para Wirra<sup>a</sup>, Australia </td><td>1972 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Tachkent<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, Uzbekistan </td><td>1981 </td><td>− </td><td>− </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>Chimkent<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, Kazakhstan </td><td>1983 </td><td>− </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Our laboratory stocks.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>Bloomington stock center.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>c</sup>These stocks were used as control for TAS.</p></div></div><div content-id="tblfn4"><div class="fn"><p><sup>d</sup>The data were obtained by PCR amplification using specific primers testing the presence of T3 domain or TAS-L family.</p></div></div><div content-id="tblfn5"><div class="fn"><p><sup>e</sup>The data were obtained by <em>in situ</em> hybridization on polytene chromosome of at least four larvae using a TAS-R probe.</p></div></div><div content-id="tblfn6"><div class="fn"><p>NA: Not Applicable because these lines have undergone chromosomal substitutions.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx017-T1" data-id="dsx017-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Distribution of TAS family in various laboratory strains</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Year of capture </th><th><strong>T3 domain</strong><a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th><th><strong><em>X-TAS</em></strong><a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th><th><strong><em>2R-TAS</em></strong><a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th><th><strong><em>3R-TAS</em></strong><a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th><th><strong>TAS-L</strong><a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th></tr></thead><tbody><tr><td><em>Lk-P</em>(1A)<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><sup>,c</sup> </td><td>NA </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td><em>NA-P</em>(1A)<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><sup>,c</sup> </td><td>NA </td><td>− </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td><em>P-1152</em><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><sup>,c</sup> </td><td>NA </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td><em>w<sup>1118</sup></em><sup>a</sup> </td><td>NA </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>yw</em><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>NA </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>y<sup>1</sup>; cn<sup>1</sup> bw<sup>1</sup> sp<sup>1</sup></em><sup>b</sup> </td><td>NA </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>Canton-S</em><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, OH </td><td>1935 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>Oregon-R-C</em><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, OR </td><td>1925 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Gruta<sup>a</sup>, France </td><td>1950 </td><td>− </td><td>− </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>BER_2<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, Bermuda </td><td>1954 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>VAG_2<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, Greece </td><td>1965 </td><td>+ </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td></tr><tr><td>Wild_11C<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, NC </td><td>1966 </td><td>+ </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td></tr><tr><td>Wild_5A<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, GA </td><td>1966 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>Tautavel<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, France </td><td>1967 </td><td>− </td><td>− </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>Harwich<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, MA </td><td>1967 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Chatelet<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, France </td><td>1970 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Para Wirra<sup>a</sup>, Australia </td><td>1972 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Tachkent<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, Uzbekistan </td><td>1981 </td><td>− </td><td>− </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>Chimkent<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, Kazakhstan </td><td>1983 </td><td>− </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td></tr></tbody></table></div><div content-id="dsx017-T1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Year of capture </th><th><strong>T3 domain</strong><a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th><th><strong><em>X-TAS</em></strong><a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th><th><strong><em>2R-TAS</em></strong><a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th><th><strong><em>3R-TAS</em></strong><a reveal-id="tblfn5" data-open="tblfn5" class="link link-ref link-reveal xref-fn"><sup>e</sup></a> </th><th><strong>TAS-L</strong><a reveal-id="tblfn4" data-open="tblfn4" class="link link-ref link-reveal xref-fn"><sup>d</sup></a> </th></tr></thead><tbody><tr><td><em>Lk-P</em>(1A)<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><sup>,c</sup> </td><td>NA </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>− </td></tr><tr><td><em>NA-P</em>(1A)<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><sup>,c</sup> </td><td>NA </td><td>− </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td><em>P-1152</em><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a><sup>,c</sup> </td><td>NA </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td><em>w<sup>1118</sup></em><sup>a</sup> </td><td>NA </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>yw</em><a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>NA </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>y<sup>1</sup>; cn<sup>1</sup> bw<sup>1</sup> sp<sup>1</sup></em><sup>b</sup> </td><td>NA </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>Canton-S</em><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, OH </td><td>1935 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td><em>Oregon-R-C</em><a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, OR </td><td>1925 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Gruta<sup>a</sup>, France </td><td>1950 </td><td>− </td><td>− </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>BER_2<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, Bermuda </td><td>1954 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>VAG_2<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, Greece </td><td>1965 </td><td>+ </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td></tr><tr><td>Wild_11C<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, NC </td><td>1966 </td><td>+ </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td></tr><tr><td>Wild_5A<a reveal-id="tblfn2" data-open="tblfn2" class="link link-ref link-reveal xref-fn"><sup>b</sup></a>, GA </td><td>1966 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>Tautavel<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, France </td><td>1967 </td><td>− </td><td>− </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>Harwich<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, MA </td><td>1967 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Chatelet<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, France </td><td>1970 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Para Wirra<sup>a</sup>, Australia </td><td>1972 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>Tachkent<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, Uzbekistan </td><td>1981 </td><td>− </td><td>− </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>Chimkent<a reveal-id="tblfn1" data-open="tblfn1" class="link link-ref link-reveal xref-fn"><sup>a</sup></a>, Kazakhstan </td><td>1983 </td><td>− </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p><sup>a</sup>Our laboratory stocks.</p></div></div><div content-id="tblfn2"><div class="fn"><p><sup>b</sup>Bloomington stock center.</p></div></div><div content-id="tblfn3"><div class="fn"><p><sup>c</sup>These stocks were used as control for TAS.</p></div></div><div content-id="tblfn4"><div class="fn"><p><sup>d</sup>The data were obtained by PCR amplification using specific primers testing the presence of T3 domain or TAS-L family.</p></div></div><div content-id="tblfn5"><div class="fn"><p><sup>e</sup>The data were obtained by <em>in situ</em> hybridization on polytene chromosome of at least four larvae using a TAS-R probe.</p></div></div><div content-id="tblfn6"><div class="fn"><p>NA: Not Applicable because these lines have undergone chromosomal substitutions.</p></div></div></div></div></p>                        <a id="106355442" scrollto-destination="106355442"></a>
<div id="dsx017-F4" data-id="dsx017-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/m_dsx017f4.png?Expires=2147483647&amp;Signature=4dvOwa6U4ax14bncGxnnxdA7W9ARt57BhcknNxx~-I1bKl-0dCmC2DttmEs~cCZy29qbwTq343OP4tvoL5z3w5aWjiVEW3~3H-AbFuDTrVnkdYkNotsgawBTCkoE-YY0zBtZsEQG5hcEGVOC3h5ZT6pxLdG058ZFyt7Ol-hWG3~RSoLMg63aHehfQYy11SJKwQxn6IWSq7haHWzFMdmLslsve1U-ZAep53CXlTrJg5KpTpFcCNBU0Wab60s9EDF4SRiTPzIBafgmvycrS5B2By76-Mncbb35YsP6LFTrVTCAdDLcecyLMLCwnBWp8rhjyJKPJDI-UGdV6AaSRLdVrw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of piRNA subtelomeric clusters in recently collected D. melanogaster. Results of analysis of the Lk-P(1A) laboratory line and three recently collected lines [Chantemesle and Reth (Europe) and Cacao H1C (South America)] are shown on first, second, third and fourth columns, respectively. (A) Polytene chromosomes of salivary gland of third instar larvae (DAPI staining) were hybridized with specific probes for TAS-R repeats. TAS-R sequences are present on the X and 2R telomeres in the Lk-P(1A) line and on X, 2R and 3R telomeres in Chantemesle, Reth and Cacao H1C. Other examples of TAS-R localization in various stocks can be found in Supplementary Figs. S8 and S9 and summarized on Tables 1 and 2. (B-G) Graphs show normalized profiles of ovarian small RNAs mapping with 0 mismatches to subtelomeric loci [T3 domain, representative of X-TAS belonging to the TAS-R family (B, C), the 0.9 kb repeats (D, E) and the TAS-L family (F, G)]. Size distribution histograms (B, D and F) and read maps (C, E and G) show the length of small RNA (20–29 nt) and the abundance of normalized piRNAs (23–29 nt) matching to each reference sequence, respectively. Numbers at the bottom right of the size distribution histograms indicate the ratio of 23–29 nt small RNAs starting with an uridine (1 U bias) (ND: Not Determined). Although the piRNA profiles between lines appear similar, differences in the amount of piRNA can be noticed. Absence of TAS-L piRNA in Lk-P(1A) suggests that the 2L and 3L-TAS loci were lost. PCR amplification screen using TAS-L specific primers identified a total of two stocks out of 43 missing the TAS-L family sequences (Tables 1 and 2). This suggested that, contrary to the TAS-R family, complete deletion of TAS-L loci does not affect viability. Other analysis of small RNA profiles can be found in Supplementary Fig. S10." path-from-xml="dsx017f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355442/dsx017f4.png" path-from-xml="dsx017f4.png" target="_blank">View large</a><a section="106355442" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017f4.png?Expires=2147483647&amp;Signature=qDyC3wVaA4c6DunFobKydy~~1aRohHY2jKehiOvQTA3cit9T2lc1XG5qD7oKFpQzg5Lqpcunq7m1fKsSqSly82V5kRNjNjG3Fpr~jcwZ8IukPV0s~jYgqA41FQ0zqQf3Gx1TucO8mQ3yHxzqJ6X4n4IakptWbam~md9B--WzT6GMhehT8q6J5ynfSgeEgxX36a2zAfKe1XMw8gVP~0GvLSttqtpu55kyzNCi3CxfNzuEJpWRqUvOw1iofCcuv9bgiikgQs3tsOb1VeKbbG8~Pcx7kOeLj8m2EvMExExG509-vYulX~q04LbFqaMtzvH7asNoal3V8k40Df25PHZ56Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106355442&amp;ar=3778237&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx017f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of piRNA subtelomeric clusters in recently collected <em>D. melanogaster</em>. Results of analysis of the <em>Lk-P</em>(1A) laboratory line and three recently collected lines [Chantemesle and Reth (Europe) and Cacao H1C (South America)] are shown on first, second, third and fourth columns, respectively. (A) Polytene chromosomes of salivary gland of third instar larvae (DAPI staining) were hybridized with specific probes for TAS-R repeats. TAS-R sequences are present on the <em>X</em> and <em>2R</em> telomeres in the <em>Lk-P</em>(1A) line and on <em>X</em>, <em>2R</em> and <em>3R</em> telomeres in Chantemesle, Reth and Cacao H1C. Other examples of TAS-R localization in various stocks can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&amp;Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S8 and S9</a></span> and summarized on <a reveal-id="dsx017-T1" data-open="dsx017-T1" class="link link-reveal link-table xref-fig">Tables 1</a> and <a reveal-id="dsx017-T2" data-open="dsx017-T2" class="link link-reveal link-table xref-fig">2</a>. (B-G) Graphs show normalized profiles of ovarian small RNAs mapping with 0 mismatches to subtelomeric loci [T3 domain, representative of <em>X-TAS</em> belonging to the TAS-R family (B, C), the 0.9 kb repeats (D, E) and the TAS-L family (F, G)]. Size distribution histograms (B, D and F) and read maps (C, E and G) show the length of small RNA (20–29 nt) and the abundance of normalized piRNAs (23–29 nt) matching to each reference sequence, respectively. Numbers at the bottom right of the size distribution histograms indicate the ratio of 23–29 nt small RNAs starting with an uridine (1 U bias) (ND: Not Determined). Although the piRNA profiles between lines appear similar, differences in the amount of piRNA can be noticed. Absence of TAS-L piRNA in <em>Lk-P</em>(1A) suggests that the <em>2L</em> and <em>3L-TAS</em> loci were lost. PCR amplification screen using TAS-L specific primers identified a total of two stocks out of 43 missing the TAS-L family sequences (<a reveal-id="dsx017-T1" data-open="dsx017-T1" class="link link-reveal link-table xref-fig">Tables 1</a> and <a reveal-id="dsx017-T2" data-open="dsx017-T2" class="link link-reveal link-table xref-fig">2</a>). This suggested that, contrary to the TAS-R family, complete deletion of TAS-L loci does not affect viability. Other analysis of small RNA profiles can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&amp;Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>.</p></div></div><div content-id="dsx017-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/m_dsx017f4.png?Expires=2147483647&amp;Signature=4dvOwa6U4ax14bncGxnnxdA7W9ARt57BhcknNxx~-I1bKl-0dCmC2DttmEs~cCZy29qbwTq343OP4tvoL5z3w5aWjiVEW3~3H-AbFuDTrVnkdYkNotsgawBTCkoE-YY0zBtZsEQG5hcEGVOC3h5ZT6pxLdG058ZFyt7Ol-hWG3~RSoLMg63aHehfQYy11SJKwQxn6IWSq7haHWzFMdmLslsve1U-ZAep53CXlTrJg5KpTpFcCNBU0Wab60s9EDF4SRiTPzIBafgmvycrS5B2By76-Mncbb35YsP6LFTrVTCAdDLcecyLMLCwnBWp8rhjyJKPJDI-UGdV6AaSRLdVrw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of piRNA subtelomeric clusters in recently collected D. melanogaster. Results of analysis of the Lk-P(1A) laboratory line and three recently collected lines [Chantemesle and Reth (Europe) and Cacao H1C (South America)] are shown on first, second, third and fourth columns, respectively. (A) Polytene chromosomes of salivary gland of third instar larvae (DAPI staining) were hybridized with specific probes for TAS-R repeats. TAS-R sequences are present on the X and 2R telomeres in the Lk-P(1A) line and on X, 2R and 3R telomeres in Chantemesle, Reth and Cacao H1C. Other examples of TAS-R localization in various stocks can be found in Supplementary Figs. S8 and S9 and summarized on Tables 1 and 2. (B-G) Graphs show normalized profiles of ovarian small RNAs mapping with 0 mismatches to subtelomeric loci [T3 domain, representative of X-TAS belonging to the TAS-R family (B, C), the 0.9 kb repeats (D, E) and the TAS-L family (F, G)]. Size distribution histograms (B, D and F) and read maps (C, E and G) show the length of small RNA (20–29 nt) and the abundance of normalized piRNAs (23–29 nt) matching to each reference sequence, respectively. Numbers at the bottom right of the size distribution histograms indicate the ratio of 23–29 nt small RNAs starting with an uridine (1 U bias) (ND: Not Determined). Although the piRNA profiles between lines appear similar, differences in the amount of piRNA can be noticed. Absence of TAS-L piRNA in Lk-P(1A) suggests that the 2L and 3L-TAS loci were lost. PCR amplification screen using TAS-L specific primers identified a total of two stocks out of 43 missing the TAS-L family sequences (Tables 1 and 2). This suggested that, contrary to the TAS-R family, complete deletion of TAS-L loci does not affect viability. Other analysis of small RNA profiles can be found in Supplementary Fig. S10." path-from-xml="dsx017f4.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355442/dsx017f4.png" path-from-xml="dsx017f4.png" target="_blank">View large</a><a section="106355442" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017f4.png?Expires=2147483647&amp;Signature=qDyC3wVaA4c6DunFobKydy~~1aRohHY2jKehiOvQTA3cit9T2lc1XG5qD7oKFpQzg5Lqpcunq7m1fKsSqSly82V5kRNjNjG3Fpr~jcwZ8IukPV0s~jYgqA41FQ0zqQf3Gx1TucO8mQ3yHxzqJ6X4n4IakptWbam~md9B--WzT6GMhehT8q6J5ynfSgeEgxX36a2zAfKe1XMw8gVP~0GvLSttqtpu55kyzNCi3CxfNzuEJpWRqUvOw1iofCcuv9bgiikgQs3tsOb1VeKbbG8~Pcx7kOeLj8m2EvMExExG509-vYulX~q04LbFqaMtzvH7asNoal3V8k40Df25PHZ56Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106355442&amp;ar=3778237&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx017f4.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of piRNA subtelomeric clusters in recently collected <em>D. melanogaster</em>. Results of analysis of the <em>Lk-P</em>(1A) laboratory line and three recently collected lines [Chantemesle and Reth (Europe) and Cacao H1C (South America)] are shown on first, second, third and fourth columns, respectively. (A) Polytene chromosomes of salivary gland of third instar larvae (DAPI staining) were hybridized with specific probes for TAS-R repeats. TAS-R sequences are present on the <em>X</em> and <em>2R</em> telomeres in the <em>Lk-P</em>(1A) line and on <em>X</em>, <em>2R</em> and <em>3R</em> telomeres in Chantemesle, Reth and Cacao H1C. Other examples of TAS-R localization in various stocks can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&amp;Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S8 and S9</a></span> and summarized on <a reveal-id="dsx017-T1" data-open="dsx017-T1" class="link link-reveal link-table xref-fig">Tables 1</a> and <a reveal-id="dsx017-T2" data-open="dsx017-T2" class="link link-reveal link-table xref-fig">2</a>. (B-G) Graphs show normalized profiles of ovarian small RNAs mapping with 0 mismatches to subtelomeric loci [T3 domain, representative of <em>X-TAS</em> belonging to the TAS-R family (B, C), the 0.9 kb repeats (D, E) and the TAS-L family (F, G)]. Size distribution histograms (B, D and F) and read maps (C, E and G) show the length of small RNA (20–29 nt) and the abundance of normalized piRNAs (23–29 nt) matching to each reference sequence, respectively. Numbers at the bottom right of the size distribution histograms indicate the ratio of 23–29 nt small RNAs starting with an uridine (1 U bias) (ND: Not Determined). Although the piRNA profiles between lines appear similar, differences in the amount of piRNA can be noticed. Absence of TAS-L piRNA in <em>Lk-P</em>(1A) suggests that the <em>2L</em> and <em>3L-TAS</em> loci were lost. PCR amplification screen using TAS-L specific primers identified a total of two stocks out of 43 missing the TAS-L family sequences (<a reveal-id="dsx017-T1" data-open="dsx017-T1" class="link link-reveal link-table xref-fig">Tables 1</a> and <a reveal-id="dsx017-T2" data-open="dsx017-T2" class="link link-reveal link-table xref-fig">2</a>). This suggested that, contrary to the TAS-R family, complete deletion of TAS-L loci does not affect viability. Other analysis of small RNA profiles can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&amp;Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>.</p></div></div><p>In order to extend this analysis, we performed <em>in situ</em> hybridization screen on polytene chromosomes and/or PCR amplifications on 11 unrelated fly stocks from diverse locations in the world and introduced into our laboratory or Stock Centers several decades ago. <em>In situ</em> hybridization results summarized in <a reveal-id="dsx017-T1" data-open="dsx017-T1" class="link link-reveal link-table xref-fig">Table 1</a> showed that none of the strains are totally devoid of TAS-R loci, but that the majority are devoid of <em>X-TAS</em> (8/11) (some examples are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). Interestingly, BER_2, the first strain found to carry <em>P</em> elements in the 1950s,<a reveal-id="dsx017-B11" data-open="dsx017-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> still possesses <em>X-TAS</em> containing <em>P</em> sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). Moreover, the Para Wirra strain, initially described to contain <em>P</em> element sequences inserted at the 1A cytological site, suggesting the presence of <em>X-TAS</em> at the time,<a reveal-id="dsx017-B12" data-open="dsx017-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> no longer presents <em>X-TAS</em> today (<a reveal-id="dsx017-T1" data-open="dsx017-T1" class="link link-reveal link-table xref-fig">Table 1</a>).</p><p>Therefore, none of the stocks tested are completely devoid of TAS-R family repeats. Nevertheless, <em>X-TAS</em> appear to have a particular status, since the majority of laboratory stocks tested lacks this locus (12/16), a bias that is not seen for the <em>2R</em>- (1/13) or <em>3R-TAS</em> (2/13) loci. This finding implicates that it may be more critical to maintain <em>2R-</em> and <em>3R-TAS</em> than <em>X-TAS</em>, suggesting a differential functional role between these TAS in fly biology.</p>                            <h3 scrollto-destination=106355445 id="106355445" class="section-title" >3.5. Subtelomeric regions in <em>D. melanogaster</em> wild populations</h3>
<p>To investigate the frequency of <em>X-TAS</em> in flies living in the wild, we generated a total of 95 <em>D. melanogaster</em> stocks from flies sampled in three continents (Africa, America and Europe). We performed <em>in situ</em> hybridizations and/or PCR amplifications immediately upon generation of all 95 stocks (i.e. on the founders of the lines) and a few months after their capture (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&amp;Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_tables.pdf?Expires=2147483647&amp;Signature=FwvzbfDMugjK1KDcWcYS3hV8j4C9hBqZi4cFJAIEqzTEGwmQGIl3Efr4o0BNod5Da5pJKh15VBV~IUtuNvZktizgJg~9FCXHAiZydTFr-jfgdPKL51SQUgT8JGwEb7DBgeG0oeICN8HVkOb~a9T5CtRb1zV3rvqAwWCdxz23ue56MdKE13LpJ~V~WFpdzkdYifuwuqjpHHoCTNdqoN3I1AdJ~SqLbv4fxbPx2tFTv22Hgc2yDzuX-~4pKDk2TDlukWMjQUCm7Q08N5UMs3uzjx8vHY9-Ms7rr9ZO-Osei8Rli~Y8BLfuMzrDvtRgCEkphuoLu99zb3263UmEnQ6Low__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). As shown in <a reveal-id="dsx017-T2" data-open="dsx017-T2" class="link link-reveal link-table xref-fig">Table 2</a>, the 23 lines tested by <em>in situ</em> hybridization possess at least two TAS-R loci. Lines isolated from the same location displayed polymorphism as regarding the presence of autosomic TAS within a population (see Marsais, Niort and Saul strains). In contrast to the long-term laboratory stocks, all 95 stocks established from flies recently collected from the wild possessed the <em>X-TAS</em> locus except for one, i.e. the line from Ilha Grande (Brazil) (<a reveal-id="dsx017-T2" data-open="dsx017-T2" class="link link-reveal link-table xref-fig">Table 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_tables.pdf?Expires=2147483647&amp;Signature=FwvzbfDMugjK1KDcWcYS3hV8j4C9hBqZi4cFJAIEqzTEGwmQGIl3Efr4o0BNod5Da5pJKh15VBV~IUtuNvZktizgJg~9FCXHAiZydTFr-jfgdPKL51SQUgT8JGwEb7DBgeG0oeICN8HVkOb~a9T5CtRb1zV3rvqAwWCdxz23ue56MdKE13LpJ~V~WFpdzkdYifuwuqjpHHoCTNdqoN3I1AdJ~SqLbv4fxbPx2tFTv22Hgc2yDzuX-~4pKDk2TDlukWMjQUCm7Q08N5UMs3uzjx8vHY9-Ms7rr9ZO-Osei8Rli~Y8BLfuMzrDvtRgCEkphuoLu99zb3263UmEnQ6Low__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). In the latter case, the stock was established from one captured female. Two sublines were then generated (Ilha Grande-1 and Ilha Grande-2) that were tested 3 months post-sampling for the presence/absence of <em>X-TAS</em>. Since at this point one of the two sublines, Ilha Grande-1, carried no <em>X-TAS</em> (<a reveal-id="dsx017-T2" data-open="dsx017-T2" class="link link-reveal link-table xref-fig">Table 2</a> and <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_tables.pdf?Expires=2147483647&amp;Signature=FwvzbfDMugjK1KDcWcYS3hV8j4C9hBqZi4cFJAIEqzTEGwmQGIl3Efr4o0BNod5Da5pJKh15VBV~IUtuNvZktizgJg~9FCXHAiZydTFr-jfgdPKL51SQUgT8JGwEb7DBgeG0oeICN8HVkOb~a9T5CtRb1zV3rvqAwWCdxz23ue56MdKE13LpJ~V~WFpdzkdYifuwuqjpHHoCTNdqoN3I1AdJ~SqLbv4fxbPx2tFTv22Hgc2yDzuX-~4pKDk2TDlukWMjQUCm7Q08N5UMs3uzjx8vHY9-Ms7rr9ZO-Osei8Rli~Y8BLfuMzrDvtRgCEkphuoLu99zb3263UmEnQ6Low__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), it is tempting to speculate that loss of the <em>X-TAS</em> occurred soon after this subline was established. <div content-id="dsx017-T2" class="table-modal"><div class="table-wrap"><div id="dsx017-T2" data-id="dsx017-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Distribution of TAS family in recently collected strains</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Year of capture </th><th><strong>T3 domain</strong><a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><strong><em>X-TAS</em></strong><a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong><em>2R-TAS</em></strong><a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong><em>3R-TAS</em></strong><a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong>TAS-L</strong><a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th></tr></thead><tbody><tr><td colspan="7">Europe </td></tr><tr><td>    Paris </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Combs-la-Ville </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Viry-Châtillon-3 </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Chantemesle-iso </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Marsais-53Ja </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Marsais-80J </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Marsais-86C </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Marsais-87C </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Niort-A1 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-G3 </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Niort-P1.3 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-P2.1 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Reth </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Mauzé Thouarsais-2 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Mauzé Thouarsais-3 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Sanguinet-1.5 </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Rouans1 </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Gotheron, Valence-iso3 </td><td>2012 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td colspan="7">Africa </td></tr><tr><td>    Addis Ababa 2 </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Kafountine </td><td>2015 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td colspan="7">South America </td></tr><tr><td>    Saul, Cacao H2A2 </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Saul, Cacao H1C </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Ilha Grande 1, Rio Janeiro </td><td>2014 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr></tbody></table></div><div content-id="dsx017-T2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Year of capture </th><th><strong>T3 domain</strong><a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><strong><em>X-TAS</em></strong><a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong><em>2R-TAS</em></strong><a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong><em>3R-TAS</em></strong><a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong>TAS-L</strong><a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th></tr></thead><tbody><tr><td colspan="7">Europe </td></tr><tr><td>    Paris </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Combs-la-Ville </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Viry-Châtillon-3 </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Chantemesle-iso </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Marsais-53Ja </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Marsais-80J </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Marsais-86C </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Marsais-87C </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Niort-A1 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-G3 </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Niort-P1.3 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-P2.1 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Reth </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Mauzé Thouarsais-2 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Mauzé Thouarsais-3 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Sanguinet-1.5 </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Rouans1 </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Gotheron, Valence-iso3 </td><td>2012 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td colspan="7">Africa </td></tr><tr><td>    Addis Ababa 2 </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Kafountine </td><td>2015 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td colspan="7">South America </td></tr><tr><td>    Saul, Cacao H2A2 </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Saul, Cacao H1C </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Ilha Grande 1, Rio Janeiro </td><td>2014 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn7"><div class="fn"><p><sup>a</sup>The data were obtained by PCR amplification using specific primers testing the presence of T3 domain or TAS-L family.</p></div></div><div content-id="tblfn8"><div class="fn"><p><sup>b</sup>The data were obtained by <em>in situ</em> hybridization on polytene chromosome of at least four larvae using a TAS-R probe.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx017-T2" data-id="dsx017-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Distribution of TAS family in recently collected strains</p></div></div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th>Year of capture </th><th><strong>T3 domain</strong><a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><strong><em>X-TAS</em></strong><a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong><em>2R-TAS</em></strong><a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong><em>3R-TAS</em></strong><a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong>TAS-L</strong><a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th></tr></thead><tbody><tr><td colspan="7">Europe </td></tr><tr><td>    Paris </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Combs-la-Ville </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Viry-Châtillon-3 </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Chantemesle-iso </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Marsais-53Ja </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Marsais-80J </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Marsais-86C </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Marsais-87C </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Niort-A1 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-G3 </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Niort-P1.3 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-P2.1 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Reth </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Mauzé Thouarsais-2 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Mauzé Thouarsais-3 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Sanguinet-1.5 </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Rouans1 </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Gotheron, Valence-iso3 </td><td>2012 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td colspan="7">Africa </td></tr><tr><td>    Addis Ababa 2 </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Kafountine </td><td>2015 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td colspan="7">South America </td></tr><tr><td>    Saul, Cacao H2A2 </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Saul, Cacao H1C </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Ilha Grande 1, Rio Janeiro </td><td>2014 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr></tbody></table></div><div content-id="dsx017-T2" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th>Year of capture </th><th><strong>T3 domain</strong><a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th><th><strong><em>X-TAS</em></strong><a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong><em>2R-TAS</em></strong><a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong><em>3R-TAS</em></strong><a reveal-id="tblfn8" data-open="tblfn8" class="link link-ref link-reveal xref-fn"><sup>b</sup></a> </th><th><strong>TAS-L</strong><a reveal-id="tblfn7" data-open="tblfn7" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </th></tr></thead><tbody><tr><td colspan="7">Europe </td></tr><tr><td>    Paris </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Combs-la-Ville </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Viry-Châtillon-3 </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Chantemesle-iso </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Marsais-53Ja </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Marsais-80J </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Marsais-86C </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Marsais-87C </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Niort-A1 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-G3 </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Niort-P1.3 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-P2.1 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Reth </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Mauzé Thouarsais-2 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Mauzé Thouarsais-3 </td><td>2013 </td><td>+ </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Sanguinet-1.5 </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Rouans1 </td><td>2013 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Gotheron, Valence-iso3 </td><td>2012 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td colspan="7">Africa </td></tr><tr><td>    Addis Ababa 2 </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Kafountine </td><td>2015 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td colspan="7">South America </td></tr><tr><td>    Saul, Cacao H2A2 </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>− </td><td>+ </td></tr><tr><td>    Saul, Cacao H1C </td><td>2014 </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td><td>+ </td></tr><tr><td>    Ilha Grande 1, Rio Janeiro </td><td>2014 </td><td>− </td><td>− </td><td>+ </td><td>+ </td><td>+ </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn7"><div class="fn"><p><sup>a</sup>The data were obtained by PCR amplification using specific primers testing the presence of T3 domain or TAS-L family.</p></div></div><div content-id="tblfn8"><div class="fn"><p><sup>b</sup>The data were obtained by <em>in situ</em> hybridization on polytene chromosome of at least four larvae using a TAS-R probe.</p></div></div></div></div></p><p>Finally, ovarian small RNA sequencing using three lines established from recently collected flies (Chantemesle, Reth and Cacao H1C) and three laboratory stocks (<em>Lk-P</em>(1A), <em>Oregon-R-C</em> and <em>P-1152</em>), all carrying <em>X-TAS</em>, demonstrated the production of small RNAs from the T3 domain in all cases. These small RNAs have a defined length of 23–29 nt and a 5′ uridine bias consistent with piRNA molecular properties (<a reveal-id="dsx017-F4" data-open="dsx017-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&amp;Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10A</a></span>). Moreover, a bias toward production of sense piRNAs was noted for all stocks tested, suggesting that <em>X</em>-<em>TAS</em> repeats are asymmetric piRNA clusters. The presence of <em>P</em> elements (<em>Lk-P</em>(1A), Reth, Cacao H1C) or <em>P</em>-transgenes (<em>P-1152</em>) in <em>X-TAS</em> did not affect piRNA production when compared with strains devoid of <em>P</em> elements in <em>X-TAS</em> (<em>Oregon-R-C</em>, Chantemesle) (<a reveal-id="dsx017-F4" data-open="dsx017-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&amp;Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Figs. S9, S10</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplfiglegends_suppresults.pdf?Expires=2147483647&amp;Signature=uRjogxkb2t0eFeSq7MSWiJLL4~hOKaxenYuuHDLixapfODiWx6WVsCKBzWUVK1IfpLKAgw9d6nDMSRGZKy8L3N9-XZsr--AyitmD~JaSD5veRSd3TCSus8bXM198eeGqMyKQ3tMW~iJWbFwEc7YRO5iEkFzP4Xx7xDTYxf1yvCDFNWqMOGxIhnE2kNO5lbOuRX4u-SCZwsYaSl~qIRlfk7V5mohqH1lfFqwxPFe2QJCac15bLFthzJfiXuanR75SQIhdMTaIOahB5QITAQGyqAUncqs4nhF3XeveaQuYXecrLYVk9iBx0DsVkfLgEbwL0xTSpC6BueCPKn98xvDh8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Results</a></span>). Interestingly, the 0.9-kb repeats were also able to produce piRNAs independent of the presence or absence of <em>X-TAS</em> arrays (<a reveal-id="dsx017-F4" data-open="dsx017-F4" class="link link-reveal link-table xref-fig">Fig. 4D</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&amp;Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10C</a></span>). The 0.9-kb repeats and the <em>X-TAS</em> appear to be two independent piRNA clusters.</p><p>Therefore, the prevalence of functional dual-strand subtelomeric piRNA clusters in recently collected <em>D. melanogaster</em> stocks (22/23 stocks) is significantly higher than that in stocks kept for a long period of time in standardized laboratory conditions (4/16) (Chi square test, <em>P</em> &lt; 0.001).</p>                            <h3 scrollto-destination=106355449 id="106355449" class="section-title" >3.6. The <em>X-TAS</em> locus can be lost in stocks within 2 years of their introduction into the laboratory</h3>
<p>The results above suggest that loss of <em>X-TAS</em> may occur early during establishment of stocks in the laboratory from flies collected in the wild. In order to determine the frequency of <em>X-TAS</em> loss upon establishment and maintenance of such fly stocks in laboratories, and estimate the kinetics of this process, two independent stocks, called A and B were established upon recent sampling of flies in the wild (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_tables.pdf?Expires=2147483647&Signature=FwvzbfDMugjK1KDcWcYS3hV8j4C9hBqZi4cFJAIEqzTEGwmQGIl3Efr4o0BNod5Da5pJKh15VBV~IUtuNvZktizgJg~9FCXHAiZydTFr-jfgdPKL51SQUgT8JGwEb7DBgeG0oeICN8HVkOb~a9T5CtRb1zV3rvqAwWCdxz23ue56MdKE13LpJ~V~WFpdzkdYifuwuqjpHHoCTNdqoN3I1AdJ~SqLbv4fxbPx2tFTv22Hgc2yDzuX-~4pKDk2TDlukWMjQUCm7Q08N5UMs3uzjx8vHY9-Ms7rr9ZO-Osei8Rli~Y8BLfuMzrDvtRgCEkphuoLu99zb3263UmEnQ6Low__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). Presence of <em>X-TAS</em> repeats was assayed by PCR amplifications at different time points after the sampling. After 1 year of maintenance, the stocks Saul A3-A and Saul 13-B from French Guiana have lost <em>X-TAS</em> repeats (<a reveal-id="dsx017-T3" data-open="dsx017-T3" class="link link-reveal link-table xref-fig">Table 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_tables.pdf?Expires=2147483647&Signature=FwvzbfDMugjK1KDcWcYS3hV8j4C9hBqZi4cFJAIEqzTEGwmQGIl3Efr4o0BNod5Da5pJKh15VBV~IUtuNvZktizgJg~9FCXHAiZydTFr-jfgdPKL51SQUgT8JGwEb7DBgeG0oeICN8HVkOb~a9T5CtRb1zV3rvqAwWCdxz23ue56MdKE13LpJ~V~WFpdzkdYifuwuqjpHHoCTNdqoN3I1AdJ~SqLbv4fxbPx2tFTv22Hgc2yDzuX-~4pKDk2TDlukWMjQUCm7Q08N5UMs3uzjx8vHY9-Ms7rr9ZO-Osei8Rli~Y8BLfuMzrDvtRgCEkphuoLu99zb3263UmEnQ6Low__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). We extended the analysis to strains kept for 2 years and identified seven additional stocks that no longer possess <em>X-TAS</em> (Niort G2-A and Niort P2.4-A; Marsais 53Ja-B, Marsais 90C-B, Niort P2.6-B, Niort L3-B and Saul FG24-B) (<a reveal-id="dsx017-T3" data-open="dsx017-T3" class="link link-reveal link-table xref-fig">Table 3</a> and <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_tables.pdf?Expires=2147483647&Signature=FwvzbfDMugjK1KDcWcYS3hV8j4C9hBqZi4cFJAIEqzTEGwmQGIl3Efr4o0BNod5Da5pJKh15VBV~IUtuNvZktizgJg~9FCXHAiZydTFr-jfgdPKL51SQUgT8JGwEb7DBgeG0oeICN8HVkOb~a9T5CtRb1zV3rvqAwWCdxz23ue56MdKE13LpJ~V~WFpdzkdYifuwuqjpHHoCTNdqoN3I1AdJ~SqLbv4fxbPx2tFTv22Hgc2yDzuX-~4pKDk2TDlukWMjQUCm7Q08N5UMs3uzjx8vHY9-Ms7rr9ZO-Osei8Rli~Y8BLfuMzrDvtRgCEkphuoLu99zb3263UmEnQ6Low__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>). To confirm <em>X-TAS</em> loss, we further analyzed Marsais 53Ja stocks A and B 2 years after collection from the wild by <em>in situ</em> hybridization on polytene chromosome from salivary glands of ten male larvae (i.e. hemizygote for the <em>X</em> chromosome). While stock A had conserved the three TAS-R loci for all nuclei tested (<em>n</em> = 285) a few months post-sampling, <em>X-TAS</em> loci were consistently absent from all nuclei tested of stock B (<em>n</em> = 297) (<a reveal-id="dsx017-F5" data-open="dsx017-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). In order to increase the number of tested flies, we also individually PCR amplified 20 males from the stock B, and found that 18 out of 20 males have lost the <em>X-TAS</em> locus (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_tables.pdf?Expires=2147483647&Signature=FwvzbfDMugjK1KDcWcYS3hV8j4C9hBqZi4cFJAIEqzTEGwmQGIl3Efr4o0BNod5Da5pJKh15VBV~IUtuNvZktizgJg~9FCXHAiZydTFr-jfgdPKL51SQUgT8JGwEb7DBgeG0oeICN8HVkOb~a9T5CtRb1zV3rvqAwWCdxz23ue56MdKE13LpJ~V~WFpdzkdYifuwuqjpHHoCTNdqoN3I1AdJ~SqLbv4fxbPx2tFTv22Hgc2yDzuX-~4pKDk2TDlukWMjQUCm7Q08N5UMs3uzjx8vHY9-Ms7rr9ZO-Osei8Rli~Y8BLfuMzrDvtRgCEkphuoLu99zb3263UmEnQ6Low__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>). <div content-id="dsx017-T3" class="table-modal"><div class="table-wrap"><div id="dsx017-T3" data-id="dsx017-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p><em>X-TAS</em> loss in <em>Drosophila</em> stocks within their first two years in laboratory</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Origin </th><th rowspan="2">Year </th><th rowspan="2">Few generations after trapping from wild </th><th colspan="2">Stock A<hr /> </th><th colspan="2">Stock B<hr /> </th></tr><tr><th>Tested 1 year after collection </th><th>Tested 2 years after collection </th><th>Tested 1 year after collection </th><th>Tested 2 years after collection </th></tr></thead><tbody><tr><td colspan="7">France </td></tr><tr><td>    Marsais-53Ja </td><td>2013 </td><td>± </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>    Marsais-90C </td><td>2013 </td><td>NT </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>    Niort-G2 </td><td>2013 </td><td>NT </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-P2.4 </td><td>2013 </td><td>NT </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-P2.6 </td><td>2013 </td><td>NT </td><td>+ </td><td>NT </td><td>+ </td><td>− </td></tr><tr><td>    Niort-L3 </td><td>2013 </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td><td>− </td></tr><tr><td colspan="7">French Guiana </td></tr><tr><td>    Saul FG24 </td><td>2013 </td><td>+ </td><td>+ </td><td>NT </td><td>+ </td><td>− </td></tr><tr><td>    Saul A3 </td><td>2014 </td><td>+ </td><td>− </td><td>NT </td><td>+ </td><td>NT </td></tr><tr><td>    Saul 13 </td><td>2014 </td><td>+ </td><td>+ </td><td>NT </td><td>− </td><td>NT </td></tr></tbody></table></div><div content-id="dsx017-T3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Origin </th><th rowspan="2">Year </th><th rowspan="2">Few generations after trapping from wild </th><th colspan="2">Stock A<hr /> </th><th colspan="2">Stock B<hr /> </th></tr><tr><th>Tested 1 year after collection </th><th>Tested 2 years after collection </th><th>Tested 1 year after collection </th><th>Tested 2 years after collection </th></tr></thead><tbody><tr><td colspan="7">France </td></tr><tr><td>    Marsais-53Ja </td><td>2013 </td><td>± </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>    Marsais-90C </td><td>2013 </td><td>NT </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>    Niort-G2 </td><td>2013 </td><td>NT </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-P2.4 </td><td>2013 </td><td>NT </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-P2.6 </td><td>2013 </td><td>NT </td><td>+ </td><td>NT </td><td>+ </td><td>− </td></tr><tr><td>    Niort-L3 </td><td>2013 </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td><td>− </td></tr><tr><td colspan="7">French Guiana </td></tr><tr><td>    Saul FG24 </td><td>2013 </td><td>+ </td><td>+ </td><td>NT </td><td>+ </td><td>− </td></tr><tr><td>    Saul A3 </td><td>2014 </td><td>+ </td><td>− </td><td>NT </td><td>+ </td><td>NT </td></tr><tr><td>    Saul 13 </td><td>2014 </td><td>+ </td><td>+ </td><td>NT </td><td>− </td><td>NT </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn9"><div class="fn"><p>“+” when PCR amplifications were obtained, "-" when no PCR amplifications were obtained.</p></div></div><div content-id="tblfn10"><div class="fn"><p>Please refer to <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_tables.pdf?Expires=2147483647&Signature=FwvzbfDMugjK1KDcWcYS3hV8j4C9hBqZi4cFJAIEqzTEGwmQGIl3Efr4o0BNod5Da5pJKh15VBV~IUtuNvZktizgJg~9FCXHAiZydTFr-jfgdPKL51SQUgT8JGwEb7DBgeG0oeICN8HVkOb~a9T5CtRb1zV3rvqAwWCdxz23ue56MdKE13LpJ~V~WFpdzkdYifuwuqjpHHoCTNdqoN3I1AdJ~SqLbv4fxbPx2tFTv22Hgc2yDzuX-~4pKDk2TDlukWMjQUCm7Q08N5UMs3uzjx8vHY9-Ms7rr9ZO-Osei8Rli~Y8BLfuMzrDvtRgCEkphuoLu99zb3263UmEnQ6Low__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> for a complete view of results for all the stocks tested.</p></div></div><div content-id="tblfn11"><div class="fn"><p>NT: Not Tested.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx017-T3" data-id="dsx017-T3" class="table-wrap-title"><div class="label -label">Table 3</div><div class="caption"><p><em>X-TAS</em> loss in <em>Drosophila</em> stocks within their first two years in laboratory</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Origin </th><th rowspan="2">Year </th><th rowspan="2">Few generations after trapping from wild </th><th colspan="2">Stock A<hr /> </th><th colspan="2">Stock B<hr /> </th></tr><tr><th>Tested 1 year after collection </th><th>Tested 2 years after collection </th><th>Tested 1 year after collection </th><th>Tested 2 years after collection </th></tr></thead><tbody><tr><td colspan="7">France </td></tr><tr><td>    Marsais-53Ja </td><td>2013 </td><td>± </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>    Marsais-90C </td><td>2013 </td><td>NT </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>    Niort-G2 </td><td>2013 </td><td>NT </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-P2.4 </td><td>2013 </td><td>NT </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-P2.6 </td><td>2013 </td><td>NT </td><td>+ </td><td>NT </td><td>+ </td><td>− </td></tr><tr><td>    Niort-L3 </td><td>2013 </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td><td>− </td></tr><tr><td colspan="7">French Guiana </td></tr><tr><td>    Saul FG24 </td><td>2013 </td><td>+ </td><td>+ </td><td>NT </td><td>+ </td><td>− </td></tr><tr><td>    Saul A3 </td><td>2014 </td><td>+ </td><td>− </td><td>NT </td><td>+ </td><td>NT </td></tr><tr><td>    Saul 13 </td><td>2014 </td><td>+ </td><td>+ </td><td>NT </td><td>− </td><td>NT </td></tr></tbody></table></div><div content-id="dsx017-T3" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Origin </th><th rowspan="2">Year </th><th rowspan="2">Few generations after trapping from wild </th><th colspan="2">Stock A<hr /> </th><th colspan="2">Stock B<hr /> </th></tr><tr><th>Tested 1 year after collection </th><th>Tested 2 years after collection </th><th>Tested 1 year after collection </th><th>Tested 2 years after collection </th></tr></thead><tbody><tr><td colspan="7">France </td></tr><tr><td>    Marsais-53Ja </td><td>2013 </td><td>± </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>    Marsais-90C </td><td>2013 </td><td>NT </td><td>+ </td><td>+ </td><td>+ </td><td>− </td></tr><tr><td>    Niort-G2 </td><td>2013 </td><td>NT </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-P2.4 </td><td>2013 </td><td>NT </td><td>+ </td><td>− </td><td>+ </td><td>+ </td></tr><tr><td>    Niort-P2.6 </td><td>2013 </td><td>NT </td><td>+ </td><td>NT </td><td>+ </td><td>− </td></tr><tr><td>    Niort-L3 </td><td>2013 </td><td>NT </td><td>NT </td><td>NT </td><td>+ </td><td>− </td></tr><tr><td colspan="7">French Guiana </td></tr><tr><td>    Saul FG24 </td><td>2013 </td><td>+ </td><td>+ </td><td>NT </td><td>+ </td><td>− </td></tr><tr><td>    Saul A3 </td><td>2014 </td><td>+ </td><td>− </td><td>NT </td><td>+ </td><td>NT </td></tr><tr><td>    Saul 13 </td><td>2014 </td><td>+ </td><td>+ </td><td>NT </td><td>− </td><td>NT </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn9"><div class="fn"><p>“+” when PCR amplifications were obtained, "-" when no PCR amplifications were obtained.</p></div></div><div content-id="tblfn10"><div class="fn"><p>Please refer to <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_tables.pdf?Expires=2147483647&Signature=FwvzbfDMugjK1KDcWcYS3hV8j4C9hBqZi4cFJAIEqzTEGwmQGIl3Efr4o0BNod5Da5pJKh15VBV~IUtuNvZktizgJg~9FCXHAiZydTFr-jfgdPKL51SQUgT8JGwEb7DBgeG0oeICN8HVkOb~a9T5CtRb1zV3rvqAwWCdxz23ue56MdKE13LpJ~V~WFpdzkdYifuwuqjpHHoCTNdqoN3I1AdJ~SqLbv4fxbPx2tFTv22Hgc2yDzuX-~4pKDk2TDlukWMjQUCm7Q08N5UMs3uzjx8vHY9-Ms7rr9ZO-Osei8Rli~Y8BLfuMzrDvtRgCEkphuoLu99zb3263UmEnQ6Low__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> for a complete view of results for all the stocks tested.</p></div></div><div content-id="tblfn11"><div class="fn"><p>NT: Not Tested.</p></div></div></div></div></p>                        <a id="106355451" scrollto-destination="106355451"></a>
<div id="dsx017-F5" data-id="dsx017-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/m_dsx017f5.png?Expires=2147483647&amp;Signature=5Ilat9EeLGNGA7QbooXg~whH~vbwCSi9fQh00k2D0Ede41YqRB8OA0hWVz10oM2Lr4PTkUSy8VDh5t4eck~DG1LOS~D-gN3sUVmygTu8vhOhaKqI0eUM4JDht3cT3iLlQBIiHSAWgGfEuk0Q8FBBIzlDG0rQ5tMugS2Hgw~08smclvPI1SEKWt36myX~XS825psPGtmYDAlP4qvmmXQ~h6AchGzHGtHDe35XXXp9LECVlXWVdxqtOXO6ZSZSFnZhsyjgcaI01Xzgw8QK6M3YIv2jZq9LFB~SM-86tABLuy7NQ0ze3hrE0Q9rkzrxq~q0GatCO25MQy3AaMvc9wbuLg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Complete loss of the X-TAS locus in a two-year period. In situ hybridizations on polytene chromosomes from salivary glands from third instar larvae were performed on stocks at different time points after collection from the wild. Results are presented for two Marsais 53Ja sublines maintained separately (sublines A and B) after 2 years under laboratory conditions. (A) In stock A, the three TAS-R loci are labeled thus presenting the same pattern as when tested a few months after the capture (data not shown). Chromosomal ends of the two 3R arm homologs have been separated during the squash, showing the labeling on each of the homologous chromosomes. (B) In stock B, only two TAS (at 2R and 3R) are labeled, whereas a short time after capture all three TAS-R loci were present (data not shown) indicating the loss of X-TAS in this subline. Therefore, it is possible to witness, in the space of a two-year period, the loss of an X-TAS from a line introduced into the laboratory from the wild. Numbers in parenthesis indicate the number of nuclei studied isolated from ten male larvae. Additional in situ hybridizations are discussed in the Supplementary results." path-from-xml="dsx017f5.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355451/dsx017f5.png" path-from-xml="dsx017f5.png" target="_blank">View large</a><a section="106355451" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017f5.png?Expires=2147483647&amp;Signature=bU~I84NUPxrppE~hniKYRMzJfT-JqZyGt0y7jgpOV1OUcPYxSVaZyIJT-3uc4ZUKCPbYdMqbOpkLuUameibsm3tFJcKzoU3KXW4OODO8ceudnxWaNaomIXmlvfXHCFuM~r1iIVWkzHki7NYqArST-wEmKwA3GjTcq9xXOdzFCH3Al8NOR1oZVE5bqbNdYTp0ftesUoj5~b2ckb~7XE-AZX~34zUa89y34wyek4Fi4Z9TFMF-aKPL7qRjTHJ3AMlYFeWwVqU7gOsoiD1P-Cezet~BUsJMsEFZyTGOgVgQdC6W2SzVyjRnU8iVgi8YQJER94BEGd2Oq-soUf8hDL2k5Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106355451&amp;ar=3778237&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx017f5.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Complete loss of the <em>X-TAS</em> locus in a two-year period<em>. In situ</em> hybridizations on polytene chromosomes from salivary glands from third instar larvae were performed on stocks at different time points after collection from the wild. Results are presented for two Marsais 53Ja sublines maintained separately (sublines A and B) after 2 years under laboratory conditions. (A) In stock A, the three TAS-R loci are labeled thus presenting the same pattern as when tested a few months after the capture (data not shown). Chromosomal ends of the two <em>3R</em> arm homologs have been separated during the squash, showing the labeling on each of the homologous chromosomes. (B) In stock B, only two TAS (at <em>2R</em> and <em>3R</em>) are labeled, whereas a short time after capture all three TAS-R loci were present (data not shown) indicating the loss of <em>X-TAS</em> in this subline. Therefore, it is possible to witness, in the space of a two-year period, the loss of an <em>X-TAS</em> from a line introduced into the laboratory from the wild. Numbers in parenthesis indicate the number of nuclei studied isolated from ten male larvae. Additional <em>in situ</em> hybridizations are discussed in the <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplfiglegends_suppresults.pdf?Expires=2147483647&amp;Signature=uRjogxkb2t0eFeSq7MSWiJLL4~hOKaxenYuuHDLixapfODiWx6WVsCKBzWUVK1IfpLKAgw9d6nDMSRGZKy8L3N9-XZsr--AyitmD~JaSD5veRSd3TCSus8bXM198eeGqMyKQ3tMW~iJWbFwEc7YRO5iEkFzP4Xx7xDTYxf1yvCDFNWqMOGxIhnE2kNO5lbOuRX4u-SCZwsYaSl~qIRlfk7V5mohqH1lfFqwxPFe2QJCac15bLFthzJfiXuanR75SQIhdMTaIOahB5QITAQGyqAUncqs4nhF3XeveaQuYXecrLYVk9iBx0DsVkfLgEbwL0xTSpC6BueCPKn98xvDh8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary results</a></span>.</p></div></div><div content-id="dsx017-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/m_dsx017f5.png?Expires=2147483647&amp;Signature=5Ilat9EeLGNGA7QbooXg~whH~vbwCSi9fQh00k2D0Ede41YqRB8OA0hWVz10oM2Lr4PTkUSy8VDh5t4eck~DG1LOS~D-gN3sUVmygTu8vhOhaKqI0eUM4JDht3cT3iLlQBIiHSAWgGfEuk0Q8FBBIzlDG0rQ5tMugS2Hgw~08smclvPI1SEKWt36myX~XS825psPGtmYDAlP4qvmmXQ~h6AchGzHGtHDe35XXXp9LECVlXWVdxqtOXO6ZSZSFnZhsyjgcaI01Xzgw8QK6M3YIv2jZq9LFB~SM-86tABLuy7NQ0ze3hrE0Q9rkzrxq~q0GatCO25MQy3AaMvc9wbuLg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Complete loss of the X-TAS locus in a two-year period. In situ hybridizations on polytene chromosomes from salivary glands from third instar larvae were performed on stocks at different time points after collection from the wild. Results are presented for two Marsais 53Ja sublines maintained separately (sublines A and B) after 2 years under laboratory conditions. (A) In stock A, the three TAS-R loci are labeled thus presenting the same pattern as when tested a few months after the capture (data not shown). Chromosomal ends of the two 3R arm homologs have been separated during the squash, showing the labeling on each of the homologous chromosomes. (B) In stock B, only two TAS (at 2R and 3R) are labeled, whereas a short time after capture all three TAS-R loci were present (data not shown) indicating the loss of X-TAS in this subline. Therefore, it is possible to witness, in the space of a two-year period, the loss of an X-TAS from a line introduced into the laboratory from the wild. Numbers in parenthesis indicate the number of nuclei studied isolated from ten male larvae. Additional in situ hybridizations are discussed in the Supplementary results." path-from-xml="dsx017f5.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355451/dsx017f5.png" path-from-xml="dsx017f5.png" target="_blank">View large</a><a section="106355451" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017f5.png?Expires=2147483647&amp;Signature=bU~I84NUPxrppE~hniKYRMzJfT-JqZyGt0y7jgpOV1OUcPYxSVaZyIJT-3uc4ZUKCPbYdMqbOpkLuUameibsm3tFJcKzoU3KXW4OODO8ceudnxWaNaomIXmlvfXHCFuM~r1iIVWkzHki7NYqArST-wEmKwA3GjTcq9xXOdzFCH3Al8NOR1oZVE5bqbNdYTp0ftesUoj5~b2ckb~7XE-AZX~34zUa89y34wyek4Fi4Z9TFMF-aKPL7qRjTHJ3AMlYFeWwVqU7gOsoiD1P-Cezet~BUsJMsEFZyTGOgVgQdC6W2SzVyjRnU8iVgi8YQJER94BEGd2Oq-soUf8hDL2k5Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=106355451&amp;ar=3778237&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx017f5.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Complete loss of the <em>X-TAS</em> locus in a two-year period<em>. In situ</em> hybridizations on polytene chromosomes from salivary glands from third instar larvae were performed on stocks at different time points after collection from the wild. Results are presented for two Marsais 53Ja sublines maintained separately (sublines A and B) after 2 years under laboratory conditions. (A) In stock A, the three TAS-R loci are labeled thus presenting the same pattern as when tested a few months after the capture (data not shown). Chromosomal ends of the two <em>3R</em> arm homologs have been separated during the squash, showing the labeling on each of the homologous chromosomes. (B) In stock B, only two TAS (at <em>2R</em> and <em>3R</em>) are labeled, whereas a short time after capture all three TAS-R loci were present (data not shown) indicating the loss of <em>X-TAS</em> in this subline. Therefore, it is possible to witness, in the space of a two-year period, the loss of an <em>X-TAS</em> from a line introduced into the laboratory from the wild. Numbers in parenthesis indicate the number of nuclei studied isolated from ten male larvae. Additional <em>in situ</em> hybridizations are discussed in the <span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplfiglegends_suppresults.pdf?Expires=2147483647&amp;Signature=uRjogxkb2t0eFeSq7MSWiJLL4~hOKaxenYuuHDLixapfODiWx6WVsCKBzWUVK1IfpLKAgw9d6nDMSRGZKy8L3N9-XZsr--AyitmD~JaSD5veRSd3TCSus8bXM198eeGqMyKQ3tMW~iJWbFwEc7YRO5iEkFzP4Xx7xDTYxf1yvCDFNWqMOGxIhnE2kNO5lbOuRX4u-SCZwsYaSl~qIRlfk7V5mohqH1lfFqwxPFe2QJCac15bLFthzJfiXuanR75SQIhdMTaIOahB5QITAQGyqAUncqs4nhF3XeveaQuYXecrLYVk9iBx0DsVkfLgEbwL0xTSpC6BueCPKn98xvDh8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary results</a></span>.</p></div></div><p>To elucidate the structure of <em>X</em> chromosome extremities after <em>X-TAS</em> loss, PCR amplifications of <em>X-TAS</em> depleted stocks using specific primers for T3 (<em>TAS-B</em>) and the 0.9-kb repeats were performed (see above). However, although <em>in situ</em> hybridization with the TAS probe on the <em>NA-P</em>(1A) line showed a faint band on the <em>X</em> telomere (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>), no PCR amplification was obtained using T3 specific primers. By designing new primer sets, we found that only the <em>TAS-D</em> repeat, the most divergent repeat from <em>TAS-B</em>, was present (<a reveal-id="dsx017-F5" data-open="dsx017-F5" class="link link-reveal link-table xref-fig">Fig. 5</a>). By using specific PCR primer sets for <em>TAS-B</em>, <em>TAS-D</em> and the 0.9-kb repeats, we found that <em>TAS-B</em> amplification was always associated with <em>TAS-D</em> and 0.9-kb repeat PCR amplifications, and that <em>TAS-D</em> amplification was always associated with 0.9-kb repeats amplification. Therefore, the organization of <em>X</em> subtelomeres, when present, is similar to the one described from the minichromosome more than 20 years ago. <a reveal-id="dsx017-F3" data-open="dsx017-F3" class="link link-reveal link-table xref-fig">Figure 3</a> displays a schematic representation of subtelomeric structures as deduced from the PCR amplifications. It shows that three of the recently collected lines that lost <em>X-TAS</em> still possess the <em>TAS-D</em> repeat like <em>NA-P</em>(1A) (<a reveal-id="dsx017-F3" data-open="dsx017-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>), another one of these lines that has retained the 0.9-kb repeats like <em>Canton-S</em> (<a reveal-id="dsx017-F3" data-open="dsx017-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>) and five other lines that have lost the three domains tested (<em>TAS-B</em>, <em>TAS-D</em> and 0.9 kb) like <em>y</em><sup><em>1</em></sup><em>; cn</em><sup><em>1</em></sup><em>bw</em><sup><em>1</em></sup><em>sp<sup>1</sup></em> (<a reveal-id="dsx017-F3" data-open="dsx017-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>). Finally, we also tested strains, which have been maintained in laboratory conditions for decades and are devoid of <em>X-TAS</em> locus. Seven out of the 14 lines tested have a deletion of the whole <em>X</em> subtelomere (<a reveal-id="dsx017-F3" data-open="dsx017-F3" class="link link-reveal link-table xref-fig">Fig. 3D</a>) and three have only retained the 0.9-kb repeats (<a reveal-id="dsx017-F3" data-open="dsx017-F3" class="link link-reveal link-table xref-fig">Fig. 3C</a>).</p><p>What is the molecular mechanism of TAS loss? Anderson et al.<a reveal-id="dsx017-B33" data-open="dsx017-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> suggested the existence of a recombination hot spot in the most distal portion of the <em>X</em> chromosome consistent with a role in evolution of subtelomeric DNA. Moreover, Novitski<a reveal-id="dsx017-B34" data-open="dsx017-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> proposed the existence of a meiotic drive in the <em>Drosophila</em> oocyte that favors transmission of shorter telomeres. Our results suggest that loss of the three types of repeats (i.e. <em>TAS-B</em>, <em>TAS-D</em> and the 0.9 kb repeats) might happen after terminal chromosomal breakage rather than by random internal deletion in the locus. These events could then have been selected for meiotic drive that would confer a segregation advantage for flies raised under laboratory conditions. Since males are hemizygotes for the <em>X</em> chromosome, this loss could then be rapidly fixed in laboratory populations. In any case, we assume that these terminal deletions were subsequently healed by retrotransposition of telomeric <em>HTT</em> elements as it has been previously described.<a reveal-id="dsx017-B25" data-open="dsx017-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a><sup>,</sup><a reveal-id="dsx017-B35" data-open="dsx017-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a></p>                            <h2 scrollto-destination=106355454 id="106355454" class="section-title" >4. Discussion</h2>
                            <h3 scrollto-destination=106355455 id="106355455" class="section-title" >4.1. Subtelomeres in <em>Drosophila</em> are ancestral piRNA clusters</h3>
<p>Our investigations reveal that not only members of the TAS-R family are more related to each other than previously thought, but that there is also a relation between TAS-R and TAS-L. Besides the three tandemly repeated <em>INV-4</em> solo LTRs already known to define the TAS-R family, we identified five other shared domains (named T1, T2, T3, T4 and T5). The T3 domain was found to be restricted to <em>X-TAS</em>, along with another sequence, newly characterized here, of a 158-bp that we called <em>TLL</em>. Indeed, <em>in silico</em> analyses demonstrated that <em>TLLs</em> are the tandemly repeated unit substructure of TAS-L. Interestingly, specific polymorphisms exist among the <em>X-</em>, <em>2R-</em>, <em>3R-, 2L-</em> and <em>3L-TAS</em> loci, suggesting the existence of a mechanism(s) that prevents exchange in <em>trans</em> between subtelomeric loci in <em>D. melanogaster</em>, contrary to what has been shown for humans and <em>S. cerevisiae</em>.<a reveal-id="dsx017-B36" data-open="dsx017-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> This is consistent with previous results showing suppression of crossing-over in regions proximal to telomeres in <em>Drosophila</em>.<a reveal-id="dsx017-B37" data-open="dsx017-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a></p><p><em>In silico</em> comparisons show that <em>TLL</em> sequences are found in related members of the <em>melanogaster</em> subgroup, but not in more distantly related Drosophilidae. Our results indicate that a <em>TLL</em> duplication event, giving rise to a 129-bp sequence, occurred after the divergence between <em>D. melanogaster/D. sechellia</em>/<em>D. simulans</em> and <em>D. erecta</em>/<em>D. yakuba</em> (see model in <a reveal-id="dsx017-F6" data-open="dsx017-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). In agreement with the results presented here, we have hypothesized that <em>TLL</em> and the 129-bp sequence are thus paralogs and along with T2 and T5 were already existed in a common ancestor (<a reveal-id="dsx017-F6" data-open="dsx017-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). According to this hypothesis, 0.9-kb repeats and the TAS-R and TAS-L families could have emerged then from successive genomic events (deletion, insertion and duplication) in <em>D. melanogaster</em> (<a reveal-id="dsx017-F6" data-open="dsx017-F6" class="link link-reveal link-table xref-fig">Fig. 6</a>). Finally, results of <em>in situ</em> hybridization and ovarian small RNA sequencing experiments strongly suggest that the <em>TLL</em> sequence could be at the origin of subtelomeric dual-strand piRNA clusters in the <em>melanogaster</em> subgroup that might be involved in TE repression. </p>                        <a id="106355458" scrollto-destination="106355458"></a>
<div id="dsx017-F6" data-id="dsx017-F6" class="fig fig-section"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/m_dsx017f6.png?Expires=2147483647&Signature=H-ryLcDQZuQSSiFk7I0ycrDkGX-fzFt1pCD~MVJfDXKtDv6kMn2wspvNkjSthxJY~FkpJz2u4vkgTuh4jvCrZEk7VmQxX3Ou3t9eAVegxPf3vdj2rjgWhpgqnZt~hr6NPk~8WvJv8NzBnyRDZNsldDwNHf3oTl-2AORIkJJcgK8GNg5aZdcf5RNzyHajb-809v9NGbuL9ErP7jemiGUhYBGXR8OIaHiRLBP~V4KjNE7R~x91I0BYDE2~cSV5kCBVYJu9eTQtw5fL3smSmY~mft5vmMuxe8Agkai7oWAfiFW4yQQm1kO55Z-wYHfDBq~UPsJ~-oMdbEYLiBPMYGHkzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Model of succession of genomic events during the course of TAS family repeat evolution in the melanogaster subgroup. Shown on the melanogaster subgroup phylogenetic tree, an ancestral TLL gave rise to subtelomeric repeats in D. erecta and D. yakuba on the one hand, while on the other hand duplication of TLL and insertion of sequences leading to the T2 and T5 domains occurred in the common ancestor of D. sechellia, D. simulans and D. melanogaster. During speciation of D. melanogaster, deletion of the T2 and T5 domains followed by amplification events led to the establishment of TAS-L and the 0.9 kb repeats. The TAS-R family arose by insertion of sequences leading to the T1, T4, T3′ and T3″ domains. The T3′, T3″ and TLL correspond to the full-length of the T3 domain. More recently, an INV-4 TE inserted into the locus, recombined so as to leave a solo LTR and then was amplified. A scenario of successive events (insertion, deletion and recombination) giving rise to the three TAS-R family members known today is proposed here. Note that domains and repeat units are not drawn to scale to facilitate the graphic representation. The origin of the ‘T’ domains is unknown and discussed in the text. Telomeres are to the left of the structural scheme for each chromosomal region depicted." path-from-xml="dsx017f6.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355458/dsx017f6.png" path-from-xml="dsx017f6.png" target="_blank">View large</a><a section="106355458" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017f6.png?Expires=2147483647&Signature=O-kQma3JwsNZF2Pv2n-Z-7g8ltVopJ3uwuyGV6j~WzoU0ZQXLIIwwceop2LDj2nFu~wfmkeaehcIGjGMImVJ8h3NQSwdwhjdvCv~mh9BdCUNZuYPGQIIU8yKu2ApQ~AiPo4LNHxG1O9WkQUAx5IyTjo9ZGg8y44jH6BhXNOo8v~SidSgBEvY3hlxMbP5IuN2RkIaMKhZi-Dx2nic2oiq2LLRno3rhRV~V90JJpKEvP7txsUKmWUA5WxG4gifVKlBUzNiF9~QuY74Y8XKMaP1I5yQ1xzXbgNb2KtkuvDOyYZ-9gFvSIBXt0oUtjec9kpgbFVSP69k5vn9btstsQ-2WQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355458&ar=3778237&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx017f6.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Model of succession of genomic events during the course of TAS family repeat evolution in the <em>melanogaster</em> subgroup. Shown on the <em>melanogaster</em> subgroup phylogenetic tree, an ancestral <em>TLL</em> gave rise to subtelomeric repeats in <em>D. erecta</em> and <em>D. yakuba</em> on the one hand, while on the other hand duplication of <em>TLL</em> and insertion of sequences leading to the T2 and T5 domains occurred in the common ancestor of <em>D. sechellia</em>, <em>D. simulans</em> and <em>D. melanogaster</em>. During speciation of <em>D. melanogaster</em>, deletion of the T2 and T5 domains followed by amplification events led to the establishment of TAS-L and the 0.9 kb repeats. The TAS-R family arose by insertion of sequences leading to the T1, T4, T3′ and T3″ domains. The T3′, T3″ and <em>TLL</em> correspond to the full-length of the T3 domain. More recently, an <em>INV-4</em> TE inserted into the locus, recombined so as to leave a solo LTR and then was amplified. A scenario of successive events (insertion, deletion and recombination) giving rise to the three TAS-R family members known today is proposed here. Note that domains and repeat units are not drawn to scale to facilitate the graphic representation. The origin of the ‘T’ domains is unknown and discussed in the text. Telomeres are to the left of the structural scheme for each chromosomal region depicted.</p></div></div><div content-id="dsx017-F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/m_dsx017f6.png?Expires=2147483647&Signature=H-ryLcDQZuQSSiFk7I0ycrDkGX-fzFt1pCD~MVJfDXKtDv6kMn2wspvNkjSthxJY~FkpJz2u4vkgTuh4jvCrZEk7VmQxX3Ou3t9eAVegxPf3vdj2rjgWhpgqnZt~hr6NPk~8WvJv8NzBnyRDZNsldDwNHf3oTl-2AORIkJJcgK8GNg5aZdcf5RNzyHajb-809v9NGbuL9ErP7jemiGUhYBGXR8OIaHiRLBP~V4KjNE7R~x91I0BYDE2~cSV5kCBVYJu9eTQtw5fL3smSmY~mft5vmMuxe8Agkai7oWAfiFW4yQQm1kO55Z-wYHfDBq~UPsJ~-oMdbEYLiBPMYGHkzw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Model of succession of genomic events during the course of TAS family repeat evolution in the melanogaster subgroup. Shown on the melanogaster subgroup phylogenetic tree, an ancestral TLL gave rise to subtelomeric repeats in D. erecta and D. yakuba on the one hand, while on the other hand duplication of TLL and insertion of sequences leading to the T2 and T5 domains occurred in the common ancestor of D. sechellia, D. simulans and D. melanogaster. During speciation of D. melanogaster, deletion of the T2 and T5 domains followed by amplification events led to the establishment of TAS-L and the 0.9 kb repeats. The TAS-R family arose by insertion of sequences leading to the T1, T4, T3′ and T3″ domains. The T3′, T3″ and TLL correspond to the full-length of the T3 domain. More recently, an INV-4 TE inserted into the locus, recombined so as to leave a solo LTR and then was amplified. A scenario of successive events (insertion, deletion and recombination) giving rise to the three TAS-R family members known today is proposed here. Note that domains and repeat units are not drawn to scale to facilitate the graphic representation. The origin of the ‘T’ domains is unknown and discussed in the text. Telomeres are to the left of the structural scheme for each chromosomal region depicted." path-from-xml="dsx017f6.png" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/106355458/dsx017f6.png" path-from-xml="dsx017f6.png" target="_blank">View large</a><a section="106355458" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017f6.png?Expires=2147483647&Signature=O-kQma3JwsNZF2Pv2n-Z-7g8ltVopJ3uwuyGV6j~WzoU0ZQXLIIwwceop2LDj2nFu~wfmkeaehcIGjGMImVJ8h3NQSwdwhjdvCv~mh9BdCUNZuYPGQIIU8yKu2ApQ~AiPo4LNHxG1O9WkQUAx5IyTjo9ZGg8y44jH6BhXNOo8v~SidSgBEvY3hlxMbP5IuN2RkIaMKhZi-Dx2nic2oiq2LLRno3rhRV~V90JJpKEvP7txsUKmWUA5WxG4gifVKlBUzNiF9~QuY74Y8XKMaP1I5yQ1xzXbgNb2KtkuvDOyYZ-9gFvSIBXt0oUtjec9kpgbFVSP69k5vn9btstsQ-2WQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=106355458&ar=3778237&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx017f6.png" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Model of succession of genomic events during the course of TAS family repeat evolution in the <em>melanogaster</em> subgroup. Shown on the <em>melanogaster</em> subgroup phylogenetic tree, an ancestral <em>TLL</em> gave rise to subtelomeric repeats in <em>D. erecta</em> and <em>D. yakuba</em> on the one hand, while on the other hand duplication of <em>TLL</em> and insertion of sequences leading to the T2 and T5 domains occurred in the common ancestor of <em>D. sechellia</em>, <em>D. simulans</em> and <em>D. melanogaster</em>. During speciation of <em>D. melanogaster</em>, deletion of the T2 and T5 domains followed by amplification events led to the establishment of TAS-L and the 0.9 kb repeats. The TAS-R family arose by insertion of sequences leading to the T1, T4, T3′ and T3″ domains. The T3′, T3″ and <em>TLL</em> correspond to the full-length of the T3 domain. More recently, an <em>INV-4</em> TE inserted into the locus, recombined so as to leave a solo LTR and then was amplified. A scenario of successive events (insertion, deletion and recombination) giving rise to the three TAS-R family members known today is proposed here. Note that domains and repeat units are not drawn to scale to facilitate the graphic representation. The origin of the ‘T’ domains is unknown and discussed in the text. Telomeres are to the left of the structural scheme for each chromosomal region depicted.</p></div></div><p>This <em>X</em> subtelomeric region was particularly informative in this study since this region appears to be at the crossroad between the two main TAS families of <em>D. melanogaster</em> autosomes: <em>X-TAS</em> possess large regions of homology with the TAS-R family, <em>via</em> the T1, T2, T3, T4, T5 and <em>INV-4</em> sequences, as well as with the TAS-L family, <em>via TLL</em> and the 129-bp sequences present in the adjacent 0.9-kb repeats (<a reveal-id="dsx017-F1" data-open="dsx017-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7</a></span>C). Interestingly, it was not possible to find an origin for the different “T” domains. Therefore, it is tempting to speculate that TAS derive from successive ancestral insertions of diverse TEs that got trapped, similar to the recently trapped <em>INV-4</em> and <em>P</em> elements. According to this scenario, the majority of TAS would consist of remnants of extinct TEs.</p>                            <h3 scrollto-destination=106355460 id="106355460" class="section-title" >4.2. Short-term dynamics of subtelomeric piRNA clusters</h3>
<p>Subtelomeric variability, i.e. variability in the absence/presence of specific sequence elements and/or in the number of repeats, has been described previously,<a reveal-id="dsx017-B6" data-open="dsx017-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a><sup>,</sup><a reveal-id="dsx017-B33" data-open="dsx017-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsx017-B35" data-open="dsx017-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a><sup>,</sup><a reveal-id="dsx017-B38" data-open="dsx017-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> however, the dynamics of subtelomeric sequence element gain or loss has not been addressed to a great extent, nor has the potential impact of subtelomeric dynamics on strain fitness.</p><p>Here, we report an original survey of the genetic variation at <em>X</em> subtelomeres between <em>D. melanogaster</em> laboratory stocks and recently collected stocks. From this study, no laboratory stock or wild-type stock completely devoid of TAS-R was identified, whereas TAS-L were altogether absent in some stocks. This suggests that TAS-L, but not TAS-R, are dispensable in some cases. In addition, the majority of laboratory stocks were found to be devoid of an <em>X-TAS</em> locus. In contrast, <em>X-TAS</em> repeats were present in flies directly sampled from the wild and in recently established stocks from flies collected in Europe, South America and Africa. However, once these recently established stocks were maintained under standardized laboratory conditions for several months up to 2 years and retested <em>X-TAS</em> were no longer present for some of the sublines. Therefore, we were able to follow <em>X-TAS</em> loss in real time, most likely due to frequent chromosomal breakage rather than internal deletions inside the locus (<a reveal-id="dsx017-F3" data-open="dsx017-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Our data suggest that loss of <em>X</em>-<em>TAS</em> is not influenced by the absence or presence of <em>P</em> element insertions in this locus (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplfiglegends_suppresults.pdf?Expires=2147483647&amp;Signature=uRjogxkb2t0eFeSq7MSWiJLL4~hOKaxenYuuHDLixapfODiWx6WVsCKBzWUVK1IfpLKAgw9d6nDMSRGZKy8L3N9-XZsr--AyitmD~JaSD5veRSd3TCSus8bXM198eeGqMyKQ3tMW~iJWbFwEc7YRO5iEkFzP4Xx7xDTYxf1yvCDFNWqMOGxIhnE2kNO5lbOuRX4u-SCZwsYaSl~qIRlfk7V5mohqH1lfFqwxPFe2QJCac15bLFthzJfiXuanR75SQIhdMTaIOahB5QITAQGyqAUncqs4nhF3XeveaQuYXecrLYVk9iBx0DsVkfLgEbwL0xTSpC6BueCPKn98xvDh8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary results</a></span>), nor by temperature (see stocks A and B maintained at 18 and 25°C, respectively) (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_tables.pdf?Expires=2147483647&amp;Signature=FwvzbfDMugjK1KDcWcYS3hV8j4C9hBqZi4cFJAIEqzTEGwmQGIl3Efr4o0BNod5Da5pJKh15VBV~IUtuNvZktizgJg~9FCXHAiZydTFr-jfgdPKL51SQUgT8JGwEb7DBgeG0oeICN8HVkOb~a9T5CtRb1zV3rvqAwWCdxz23ue56MdKE13LpJ~V~WFpdzkdYifuwuqjpHHoCTNdqoN3I1AdJ~SqLbv4fxbPx2tFTv22Hgc2yDzuX-~4pKDk2TDlukWMjQUCm7Q08N5UMs3uzjx8vHY9-Ms7rr9ZO-Osei8Rli~Y8BLfuMzrDvtRgCEkphuoLu99zb3263UmEnQ6Low__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), nor by the geographical origin of the flies<em>.</em> Therefore, we hypothesize that <em>X-TAS</em> fulfill a fundamental function for life in the wild that is not required under laboratory conditions. Alternatively, <em>X-TAS</em> loss under laboratory conditions could be simply the consequence of inbreeding effects due to isogenisation.</p><p>The nature of the selective forces acting on subtelomeric polymorphism identified here remains unclear. Nevertheless, subtelomeric piRNA clusters, especially <em>X-TAS</em>, appear to have an impact on fitness in the natural environment while not affecting the viability of flies raised under laboratory conditions. To our knowledge, this is the first systematic study monitoring natural loss of a piRNA cluster under specific conditions. The apparent paradox in the fact that loci involved in genome defense are subject to an absence/presence polymorphism suggests possible functional redundancy between the 140 piRNA clusters scattered throughout the <em>D. melanogaster</em> genome. This may thus provide an efficient genomic mechanism to control a tremendous load of TEs in gonads. Therefore, piRNA clusters appear to be capable of rapid evolution may be because they are, on the one hand, able to attract several kb of endogenous and/or exogenous TE copies and, on the other hand, able to lose sequences that are several kb in length without affecting viability under some, still undetermined environmental conditions. This capacity offers a time window for TEs to be derepressed (as it was observed for <em>P</em> elements in the Marsais 53Ja B, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span>). This opportunity for transposition appears to be essential to shape the genome by allowing new insertions that generate genomic diversity, as well as novelty. Among these new insertions, rare cases will be maintained in particular environments, as in the case of telomeres in <em>Drosophila</em>, CENP-B of centromeres and the RAG system in vertebrates.<a reveal-id="dsx017-B39" data-open="dsx017-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a></p>                            <h2 scrollto-destination=106355464 id="106355464" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank Clément Carré, Emile Civel, Aurélie Hua-Van, Claude Lespinet, Gilbert and Micheline Lespinet, Pauline Marie, Valerie Ribeiro, Claude et Michèle Teysset, and Anne Laure Todeschini for their help in collecting flies, the Bloomington Stock Center for providing stocks, <a class="link link-uri" href="http://FlyBase.org" target="_blank">FlyBase.org</a> for providing valuable databases and ARTbio/Bioinformatics analyses platform at the IBPS for the development of valuable tools. We thank Jean-François Gilles and Susanne Bolte for the help with confocal microscopy carried out at the Institute of Biology Paris-Seine Imaging Facility, supported by the “Conseil Regional Ile-de France”, the French national research council (CNRS) and Sorbonne University, UPMC Univ Paris 06. We thank Ritha Zamy and Shang Nong HU for technical assistance, Robert W. Levis for helpful discussions and Jean-Michel Gibert, Elena Casacuberta, Antoine Boivin, Sandra Duharcourt and Anne-Marie Pret for critical reading of the manuscript. </p>                            <h3 scrollto-destination=106355466 id="106355466" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=106355468 id="106355468" class="section-title" >Supplementary data</h3>
<p><span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplfiglegends_suppresults.pdf?Expires=2147483647&Signature=uRjogxkb2t0eFeSq7MSWiJLL4~hOKaxenYuuHDLixapfODiWx6WVsCKBzWUVK1IfpLKAgw9d6nDMSRGZKy8L3N9-XZsr--AyitmD~JaSD5veRSd3TCSus8bXM198eeGqMyKQ3tMW~iJWbFwEc7YRO5iEkFzP4Xx7xDTYxf1yvCDFNWqMOGxIhnE2kNO5lbOuRX4u-SCZwsYaSl~qIRlfk7V5mohqH1lfFqwxPFe2QJCac15bLFthzJfiXuanR75SQIhdMTaIOahB5QITAQGyqAUncqs4nhF3XeveaQuYXecrLYVk9iBx0DsVkfLgEbwL0xTSpC6BueCPKn98xvDh8g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> Online.</p>                            <h3 scrollto-destination=106355470 id="106355470" class="section-title" >Funding</h3>
<p>This study was supported by fellowships from the Ministère de l’Enseignement Supérieur et de la Recherche to A.A.-L. and by grants from the Association de la Recherche contre le Cancer (Fondation ARC, SFI20121205921, SFI20131200470), the Fondation pour la Recherche Médicale, (FRM DEP20131128532), the Agence Nationale de la Recherche (ANR, project “plastisipi”) to S.R., and the University Pierre et Marie Curie (Emergence EME1223) to L.T. Sampling of <em>D. melanogaster</em> in French Guiana was funded by a CEBA grant (MONISPEC ANR-10-LABX-25-01) and a CNRS Nouragues Travel Grant 2015 (ACCISPEC NTG 2015) to W.J.M. The funders had no role in study design, data collection, analysis, decision to publish, or preparation of the manuscript.</p>                            <h2 scrollto-destination=106355472 id="106355472" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx017-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Karpen</div>
               <div class="given-names">G.H.</div>
            </div>, <div class="name string-name"><div class="surname">Spradling</div> <div class="given-names">A.C.</div></div></div>
          <div class="year">1992</div>, 
<div class="article-title">Analysis of subtelomeric heterochromatin in the Drosophila minichromosome Dp1187 by single P element insertional mutagenesis</div>. <div class="source">Genetics</div> , <div class="volume">132</div>, <div class="fpage">737</div>–<div class="lpage">753</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Analysis%20of%20subtelomeric%20heterochromatin%20in%20the%20Drosophila%20minichromosome%20Dp1187%20by%20single%20P%20element%20insertional%20mutagenesis&amp;author=G.H.%20Karpen&amp;author=A.C.%20Spradling&amp;publication_year=1992&amp;journal=Genetics&amp;volume=132&amp;pages=737-753" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1334894" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Levis</div>
               <div class="given-names">R.W.</div>
            </div>, <div class="name string-name"><div class="surname">Ganesan</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Houtchens</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Tolar</div> <div class="given-names">L.A.</div></div>, <div class="name string-name"><div class="surname">Sheen</div> <div class="given-names">F.M.</div></div></div>
          <div class="year">1993</div>, 
<div class="article-title">Transposons in place of telomeric repeats at a <em>Drosophila telomere</em></div>. <div class="source">Cell</div> , <div class="volume">75</div>, <div class="fpage">1083</div>–<div class="lpage">1093</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transposons%20in%20place%20of%20telomeric%20repeats%20at%20a%20Drosophila%20telomere&amp;author=R.W.%20Levis&amp;author=R.%20Ganesan&amp;author=K.%20Houtchens&amp;author=L.A.%20Tolar&amp;author=F.M.%20Sheen&amp;publication_year=1993&amp;journal=Cell&amp;volume=75&amp;pages=1083-1093" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(93)90318-K" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(93)90318-K" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(93)90318-K"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8261510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Walter</div>
               <div class="given-names">M.F.</div>
            </div>, <div class="name string-name"><div class="surname">Jang</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Kasravi</div> <div class="given-names">B.</div></div></div>, et al.  <div class="year">1995</div>, 
<div class="article-title">DNA organization and polymorphism of a wild-type Drosophila telomere region</div>. <div class="source">Chromosoma</div> , <div class="volume">104</div>, <div class="fpage">229</div>–<div class="lpage">241</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DNA%20organization%20and%20polymorphism%20of%20a%20wild-type%20Drosophila%20telomere%20region&amp;author=M.F.%20Walter&amp;author=C.%20Jang&amp;author=B.%20Kasravi&amp;publication_year=1995&amp;journal=Chromosoma&amp;volume=104&amp;pages=229-241" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00352254" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00352254" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00352254"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/8565699" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Abad</div>
               <div class="given-names">J.P.</div>
            </div>, <div class="name string-name"><div class="surname">De Pablos</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Osoegawa</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">De Jong</div> <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Martin-Gallardo</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Villasante</div> <div class="given-names">A.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">Genomic analysis of <em>Drosophila melanogaster</em> telomeres: full-length copies of HeT-A and TART elements at telomeres</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">21</div>, <div class="fpage">1613</div>–<div class="lpage">1619</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomic%20analysis%20of%20Drosophila%20melanogaster%20telomeres%3A%20full-length%20copies%20of%20HeT-A%20and%20TART%20elements%20at%20telomeres&amp;author=J.P.%20Abad&amp;author=B.%20De%20Pablos&amp;author=K.%20Osoegawa&amp;author=P.J.%20De%20Jong&amp;author=A.%20Martin-Gallardo&amp;author=A.%20Villasante&amp;publication_year=2004&amp;journal=Mol.%20Biol.%20Evol&amp;volume=21&amp;pages=1613-1619" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msh174" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsh174" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsh174"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15163766" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bergman</div>
               <div class="given-names">C.M.</div>
            </div>, <div class="name string-name"><div class="surname">Quesneville</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Anxolabéhère</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Ashburner</div> <div class="given-names">M.</div></div></div>
          <div class="year">2006</div>, 
<div class="article-title">Recurrent insertion and duplication generate networks of transposable element sequences in the Drosophila melanogaster genome</div>. Genome Biol., <strong>7</strong>, R112.</div></div></div></div><div content-id="dsx017-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Levis</div>
               <div class="given-names">R.W.</div>
            </div>
          </div>
          <div class="year">1989</div>, 
<div class="article-title">Viable deletions of a telomere from a <em>Drosophila chromosome</em></div>. <div class="source">Cell</div> , <div class="volume">58</div>, <div class="fpage">791</div>–<div class="lpage">801</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Viable%20deletions%20of%20a%20telomere%20from%20a%20Drosophila%20chromosome&amp;author=R.W.%20Levis&amp;publication_year=1989&amp;journal=Cell&amp;volume=58&amp;pages=791-801" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0092-8674(89)90112-8" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0092-8674(89)90112-8" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0092-8674(89)90112-8"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2548737" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Yin</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Lin</div> <div class="given-names">H.</div></div></div>
          <div class="year">2007</div>, 
<div class="article-title">An epigenetic activation role of Piwi and a Piwi-associated piRNA in <em>Drosophila melanogaster</em></div>. <div class="source">Nature</div> , <div class="volume">450</div>, <div class="fpage">304</div>–<div class="lpage">308</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20epigenetic%20activation%20role%20of%20Piwi%20and%20a%20Piwi-associated%20piRNA%20in%20Drosophila%20melanogaster&amp;author=H.%20Yin&amp;author=H.%20Lin&amp;publication_year=2007&amp;journal=Nature&amp;volume=450&amp;pages=304-308" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature06263" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature06263" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature06263"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17952056" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Silva-Sousa</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Casacuberta</div> <div class="given-names">E.</div></div></div>
          <div class="year">2013</div>, 
<div class="article-title">The JIL-1 kinase affects telomere expression in the different telomere domains of <em>Drosophila</em></div>. <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e81543</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20JIL-1%20kinase%20affects%20telomere%20expression%20in%20the%20different%20telomere%20domains%20of%20Drosophila&amp;author=R.%20Silva-Sousa&amp;author=E.%20Casacuberta&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e81543" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0081543" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0081543" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0081543"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24244743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mason</div>
               <div class="given-names">J.M.</div>
            </div>, <div class="name string-name"><div class="surname">Villasante</div> <div class="given-names">A.</div></div></div>
          <div class="year">2014</div>, Subtelomeres in Drosophila and other Diptera.<em>In</em><div class="source">Subtelomeres</div> , <div class="person-group"><div class="name string-name"><div class="surname">Louis</div> <div class="given-names">E. J.</div></div>, <div class="name string-name"><div class="surname">Becker</div> <div class="given-names">M .M.</div></div></div> (eds.), 
<div class="publisher-name">Springer</div>: 
<div class="publisher-loc">Berlin</div>, <div class="fpage">211</div>–<div class="lpage">225</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Subtelomeres&amp;author=J.M.%20Mason&amp;author=A.%20Villasante&amp;author=E.%20J.%20Louis&amp;author=M%20.M.%20Becker&amp;publication_year=2014&amp;book=Subtelomeres" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/978-3-642-41566-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2F978-3-642-41566-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2F978-3-642-41566-1"> </span></div> </div></div></div></div></div><div content-id="dsx017-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Huisinga</div>
               <div class="given-names">K.L.</div>
            </div>, <div class="name string-name"><div class="surname">Riddle</div> <div class="given-names">N.C.</div></div>, <div class="name string-name"><div class="surname">Leung</div> <div class="given-names">W.</div></div></div>, et al.  <div class="year">2016</div>, 
<div class="article-title">Targeting of P-element reporters to heterochromatic domains by transposable element 1360 in <em>Drosophila melanogaster</em></div>. <div class="source">Genetics</div> , <div class="volume">202</div>, <div class="fpage">565</div>–<div class="lpage">582</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Targeting%20of%20P-element%20reporters%20to%20heterochromatic%20domains%20by%20transposable%20element%201360%20in%20Drosophila%20melanogaster&amp;author=K.L.%20Huisinga&amp;author=N.C.%20Riddle&amp;author=W.%20Leung&amp;publication_year=2016&amp;journal=Genetics&amp;volume=202&amp;pages=565-582" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.115.183228" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.115.183228" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.115.183228"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26680659" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Anxolabehere</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Kidwell</div> <div class="given-names">M.G.</div></div>, <div class="name string-name"><div class="surname">Periquet</div> <div class="given-names">G.</div></div></div>
          <div class="year">1988</div>, 
<div class="article-title">Molecular characteristics of diverse populations are consistent with the hypothesis of a recent invasion of <em>Drosophila melanogaster</em> by mobile P elements</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">5</div>, <div class="fpage">252</div>–<div class="lpage">269</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20characteristics%20of%20diverse%20populations%20are%20consistent%20with%20the%20hypothesis%20of%20a%20recent%20invasion%20of%20Drosophila%20melanogaster%20by%20mobile%20P%20elements&amp;author=D.%20Anxolabehere&amp;author=M.G.%20Kidwell&amp;author=G.%20Periquet&amp;publication_year=1988&amp;journal=Mol.%20Biol.%20Evol&amp;volume=5&amp;pages=252-269" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2838720" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ronsseray</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Lehmann</div> <div class="given-names">M.,</div></div>, <div class="name string-name"><div class="surname">Anxolabehere</div> <div class="given-names">D.</div></div></div>
          <div class="year">1989</div>, 
<div class="article-title">Copy number and distribution of P and I mobile elements in <em>Drosophila melanogaster</em> populations</div>. <div class="source">Chromosoma</div> , <div class="volume">98</div>, <div class="fpage">207</div>–<div class="lpage">214</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Copy%20number%20and%20distribution%20of%20P%20and%20I%20mobile%20elements%20in%20Drosophila%20melanogaster%20populations&amp;author=S.%20Ronsseray&amp;author=M.%2C%20Lehmann&amp;author=D.%20Anxolabehere&amp;publication_year=1989&amp;journal=Chromosoma&amp;volume=98&amp;pages=207-214" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF00329685" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF00329685" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF00329685"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2555115" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ronsseray</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Lehmann</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Anxolabehere</div> <div class="given-names">D.</div></div></div>
          <div class="year">1991</div>, 
<div class="article-title">The maternally inherited regulation of P elements in <em>Drosophila melanogaster</em> can be elicited by two P copies at cytological site 1A on the X chromosome</div>. <div class="source">Genetics</div> , <div class="volume">129</div>, <div class="fpage">501</div>–<div class="lpage">512</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20maternally%20inherited%20regulation%20of%20P%20elements%20in%20Drosophila%20melanogaster%20can%20be%20elicited%20by%20two%20P%20copies%20at%20cytological%20site%201A%20on%20the%20X%20chromosome&amp;author=S.%20Ronsseray&amp;author=M.%20Lehmann&amp;author=D.%20Anxolabehere&amp;publication_year=1991&amp;journal=Genetics&amp;volume=129&amp;pages=501-512" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1660427" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Josse</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Teysset</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Todeschini</div> <div class="given-names">A. L.</div></div>, <div class="name string-name"><div class="surname">Sidor</div> <div class="given-names">C. M.</div></div>, <div class="name string-name"><div class="surname">Anxolabehere</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Ronsseray</div> <div class="given-names">S.</div></div></div>
          <div class="year">2007</div>, 
<div class="article-title">Telomeric trans-silencing: an epigenetic repression combining RNA silencing and heterochromatin formation</div>. <div class="source">PLoS Genet</div> ., <div class="volume">3</div>, <div class="fpage">1633</div>–<div class="lpage">1643</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Telomeric%20trans-silencing%3A%20an%20epigenetic%20repression%20combining%20RNA%20silencing%20and%20heterochromatin%20formation&amp;author=T.%20Josse&amp;author=L.%20Teysset&amp;author=A.%20L.%20Todeschini&amp;author=C.%20M.%20Sidor&amp;author=D.%20Anxolabehere&amp;author=S.%20Ronsseray&amp;publication_year=2007&amp;journal=PLoS%20Genet&amp;volume=3&amp;pages=1633-1643" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.0030158" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.0030158" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.0030158"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17941712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Roche</div>
               <div class="given-names">S.E.</div>
            </div>, <div class="name string-name"><div class="surname">Rio</div> <div class="given-names">D.C.</div></div></div>
          <div class="year">1998</div>, 
<div class="article-title">Trans-silencing by P elements inserted in subtelomeric heterochromatin involves the Drosophila Polycomb group gene, Enhancer of zeste</div>. <div class="source">Genetics</div> , <div class="volume">149</div>, <div class="fpage">1839</div>–<div class="lpage">1855</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Trans-silencing%20by%20P%20elements%20inserted%20in%20subtelomeric%20heterochromatin%20involves%20the%20Drosophila%20Polycomb%20group%20gene%2C%20Enhancer%20of%20zeste&amp;author=S.E.%20Roche&amp;author=D.C.%20Rio&amp;publication_year=1998&amp;journal=Genetics&amp;volume=149&amp;pages=1839-1855" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9691041" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ronsseray</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Josse</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Boivin</div> <div class="given-names">A.,</div></div>, <div class="name string-name"><div class="surname">Anxolabehere</div> <div class="given-names">D.</div></div></div>
          <div class="year">2003</div>, 
<div class="article-title">Telomeric transgenes and trans-silencing in <em>Drosophila</em></div>. <div class="source">Genetica</div> , <div class="volume">117</div>, <div class="fpage">327</div>–<div class="lpage">335</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Telomeric%20transgenes%20and%20trans-silencing%20in%20Drosophila&amp;author=S.%20Ronsseray&amp;author=T.%20Josse&amp;author=A.%2C%20Boivin&amp;author=D.%20Anxolabehere&amp;publication_year=2003&amp;journal=Genetica&amp;volume=117&amp;pages=327-335" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1022929121828" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1022929121828" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1022929121828"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12723712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Brennecke</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Aravin</div> <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Stark</div> <div class="given-names">A.</div></div></div>, et al.  <div class="year">2007</div>, 
<div class="article-title">Discrete small RNA-generating loci as master regulators of transposon activity in <em>Drosophila</em></div>. <div class="source">Cell</div> , <div class="volume">128</div>, <div class="fpage">1089</div>–<div class="lpage">1103</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discrete%20small%20RNA-generating%20loci%20as%20master%20regulators%20of%20transposon%20activity%20in%20Drosophila&amp;author=J.%20Brennecke&amp;author=A.A.%20Aravin&amp;author=A.%20Stark&amp;publication_year=2007&amp;journal=Cell&amp;volume=128&amp;pages=1089-1103" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2007.01.043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2007.01.043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2007.01.043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17346786" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Czech</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Hannon</div> <div class="given-names">G.J.</div></div></div>
          <div class="year">2016</div>, 
<div class="article-title">One loop to rule them all: the ping-pong cycle and piRNA-guided silencing</div>. <div class="source">Trends Biochem. Sci</div> , <div class="volume">41</div>, <div class="fpage">324</div>–<div class="lpage">337</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=One%20loop%20to%20rule%20them%20all%3A%20the%20ping-pong%20cycle%20and%20piRNA-guided%20silencing&amp;author=B.%20Czech&amp;author=G.J.%20Hannon&amp;publication_year=2016&amp;journal=Trends%20Biochem.%20Sci&amp;volume=41&amp;pages=324-337" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tibs.2015.12.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tibs.2015.12.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tibs.2015.12.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26810602" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Josse</div>
               <div class="given-names">T.</div>
            </div>, <div class="name string-name"><div class="surname">Maurel-Zaffran</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">de Vanssay</div> <div class="given-names">A.</div></div></div>, et al.  <div class="year">2008</div>, 
<div class="article-title">Telomeric trans-silencing in <em>Drosophila melanogaster</em>: tissue specificity, development and functional interactions between non-homologous telomeres</div>. <div class="source">PLoS One</div> , <div class="volume">3</div>, <div class="fpage">e3249</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Telomeric%20trans-silencing%20in%20Drosophila%20melanogaster%3A%20tissue%20specificity%2C%20development%20and%20functional%20interactions%20between%20non-homologous%20telomeres&amp;author=T.%20Josse&amp;author=C.%20Maurel-Zaffran&amp;author=A.%20de%20Vanssay&amp;publication_year=2008&amp;journal=PLoS%20One&amp;volume=3&amp;pages=e3249" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0003249" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0003249" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0003249"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18813361" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Brennecke</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Malone</div> <div class="given-names">C.D.</div></div>, <div class="name string-name"><div class="surname">Aravin</div> <div class="given-names">A.A.</div></div>, <div class="name string-name"><div class="surname">Sachidanandam</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Stark</div> <div class="given-names">A.,</div></div>, <div class="name string-name"><div class="surname">Hannon</div> <div class="given-names">G.J.</div></div></div>
          <div class="year">2008</div>, 
<div class="article-title">An epigenetic role for maternally inherited piRNAs in transposon silencing</div>. <div class="source">Science</div> , <div class="volume">322</div>, <div class="fpage">1387</div>–<div class="lpage">1392</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20epigenetic%20role%20for%20maternally%20inherited%20piRNAs%20in%20transposon%20silencing&amp;author=J.%20Brennecke&amp;author=C.D.%20Malone&amp;author=A.A.%20Aravin&amp;author=R.%20Sachidanandam&amp;author=A.%2C%20Stark&amp;author=G.J.%20Hannon&amp;publication_year=2008&amp;journal=Science&amp;volume=322&amp;pages=1387-1392" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.1165171" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.1165171" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.1165171"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19039138" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Todeschini</div>
               <div class="given-names">A.L.</div>
            </div>, <div class="name string-name"><div class="surname">Teysset</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Delmarre</div> <div class="given-names">V.,</div></div>, <div class="name string-name"><div class="surname">Ronsseray</div> <div class="given-names">S.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">The epigenetic trans-silencing effect in Drosophila involves maternally-transmitted small RNAs whose production depends on the piRNA pathway and HP1</div>. <div class="source">PLoS One</div> , <div class="volume">5</div>, <div class="fpage">e11032</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20epigenetic%20trans-silencing%20effect%20in%20Drosophila%20involves%20maternally-transmitted%20small%20RNAs%20whose%20production%20depends%20on%20the%20piRNA%20pathway%20and%20HP1&amp;author=A.L.%20Todeschini&amp;author=L.%20Teysset&amp;author=V.%2C%20Delmarre&amp;author=S.%20Ronsseray&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e11032" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0011032" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0011032" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0011032"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20559422" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">de Vanssay</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Bouge</div> <div class="given-names">A.L.</div></div>, <div class="name string-name"><div class="surname">Boivin</div> <div class="given-names">A.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Paramutation in Drosophila linked to emergence of a piRNA-producing locus</div>. <div class="source">Nature</div> , <div class="volume">490</div>, <div class="fpage">112</div>–<div class="lpage">115</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Paramutation%20in%20Drosophila%20linked%20to%20emergence%20of%20a%20piRNA-producing%20locus&amp;author=A.%20de%20Vanssay&amp;author=A.L.%20Bouge&amp;author=A.%20Boivin&amp;publication_year=2012&amp;journal=Nature&amp;volume=490&amp;pages=112-115" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature11416" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature11416" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature11416"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22922650" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Muerdter</div>
               <div class="given-names">F.</div>
            </div>, <div class="name string-name"><div class="surname">Olovnikov</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Molaro</div> <div class="given-names">A.</div></div></div>, et al.  <div class="year">2012</div>, 
<div class="article-title">Production of artificial piRNAs in flies and mice. <em>RNA</em>,</div><div class="volume">18</div>, <div class="fpage">42</div>–<div class="lpage">52</div>.</div></div></div></div><div content-id="dsx017-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">O’Kane</div>
               <div class="given-names">C.J.</div>
            </div>, <div class="name string-name"><div class="surname">Gehring</div> <div class="given-names">W.J.</div></div></div>
          <div class="year">1987</div>, 
<div class="article-title">Detection in situ of genomic regulatory elements in Drosophila</div>. <div class="source">Proc. Natl Acad. Sci. U. S. A</div> ., <div class="volume">84</div>, <div class="fpage">9123</div>–<div class="lpage">9127</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20in%20situ%20of%20genomic%20regulatory%20elements%20in%20Drosophila&amp;author=C.J.%20O%E2%80%99Kane&amp;author=W.J.%20Gehring&amp;publication_year=1987&amp;journal=Proc.%20Natl%20Acad.%20Sci.%20U.%20S.%20A&amp;volume=84&amp;pages=9123-9127" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.84.24.9123" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.84.24.9123" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.84.24.9123"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2827169" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Marin</div>
               <div class="given-names">L.</div>
            </div>, <div class="name string-name"><div class="surname">Lehmann</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Nouaud</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Izaabel</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Anxolabehere</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Ronsseray</div> <div class="given-names">S.</div></div></div>
          <div class="year">2000</div>, 
<div class="article-title">P-Element repression in <em>Drosophila melanogaster</em> by a naturally occurring defective telomeric P copy</div>. <div class="source">Genetics</div> , <div class="volume">155</div>, <div class="fpage">1841</div>–<div class="lpage">1854</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=P-Element%20repression%20in%20Drosophila%20melanogaster%20by%20a%20naturally%20occurring%20defective%20telomeric%20P%20copy&amp;author=L.%20Marin&amp;author=M.%20Lehmann&amp;author=D.%20Nouaud&amp;author=H.%20Izaabel&amp;author=D.%20Anxolabehere&amp;author=S.%20Ronsseray&amp;publication_year=2000&amp;journal=Genetics&amp;volume=155&amp;pages=1841-1854" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10924479" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Benos</div>
               <div class="given-names">P.V.</div>
            </div>, <div class="name string-name"><div class="surname">Gatt</div> <div class="given-names">M.K.</div></div>, <div class="name string-name"><div class="surname">Murphy</div> <div class="given-names">L.</div></div></div>, et al.  <div class="year">2001</div>, 
<div class="article-title">From first base: the sequence of the tip of the X chromosome of <em>Drosophila melanogaster</em>, a comparison of two sequencing strategies</div>. <div class="source">Genome Res</div> ., <div class="volume">11</div>, <div class="fpage">710</div>–<div class="lpage">730</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=From%20first%20base%3A%20the%20sequence%20of%20the%20tip%20of%20the%20X%20chromosome%20of%20Drosophila%20melanogaster%2C%20a%20comparison%20of%20two%20sequencing%20strategies&amp;author=P.V.%20Benos&amp;author=M.K.%20Gatt&amp;author=L.%20Murphy&amp;publication_year=2001&amp;journal=Genome%20Res&amp;volume=11&amp;pages=710-730" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.173801" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.173801" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.173801"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11337470" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Boussy</div>
               <div class="given-names">I.A.</div>
            </div>, <div class="name string-name"><div class="surname">Itoh</div> <div class="given-names">M.</div></div></div>
          <div class="year">2004</div>, 
<div class="article-title">Wanderings of hobo: a transposon in <em>Drosophila melanogaster</em> and its close relatives</div>. <div class="source">Genetica</div> , <div class="volume">120</div>, <div class="fpage">125</div>–<div class="lpage">136</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Wanderings%20of%20hobo%3A%20a%20transposon%20in%20Drosophila%20melanogaster%20and%20its%20close%20relatives&amp;author=I.A.%20Boussy&amp;author=M.%20Itoh&amp;publication_year=2004&amp;journal=Genetica&amp;volume=120&amp;pages=125-136" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/B:GENE.0000017636.08925.55" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FB:GENE.0000017636.08925.55" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FB:GENE.0000017636.08925.55"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15088653" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Adams</div>
               <div class="given-names">M.D.</div>
            </div>, <div class="name string-name"><div class="surname">Celniker</div> <div class="given-names">S.E.</div></div>, <div class="name string-name"><div class="surname">Holt</div> <div class="given-names">R.A.</div></div></div>, et al.  <div class="year">2000</div>, 
<div class="article-title">The genome sequence of <em>Drosophila melanogaster</em></div>. <div class="source">Science</div> , <div class="volume">287</div>, <div class="fpage">2185</div>–<div class="lpage">2195</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genome%20sequence%20of%20Drosophila%20melanogaster&amp;author=M.D.%20Adams&amp;author=S.E.%20Celniker&amp;author=R.A.%20Holt&amp;publication_year=2000&amp;journal=Science&amp;volume=287&amp;pages=2185-2195" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.287.5461.2185" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.287.5461.2185" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.287.5461.2185"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/10731132" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gouy</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Guindon</div> <div class="given-names">S.,</div></div>, <div class="name string-name"><div class="surname">Gascuel</div> <div class="given-names">O.</div></div></div>
          <div class="year">2010</div>, 
<div class="article-title">SeaView version 4: a multiplatform graphical user interface for sequence alignment and phylogenetic tree building</div>. <div class="source">Mol. Biol. Evol</div> ., <div class="volume">27</div>, <div class="fpage">221</div>–<div class="lpage">224</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SeaView%20version%204%3A%20a%20multiplatform%20graphical%20user%20interface%20for%20sequence%20alignment%20and%20phylogenetic%20tree%20building&amp;author=M.%20Gouy&amp;author=S.%2C%20Guindon&amp;author=O.%20Gascuel&amp;publication_year=2010&amp;journal=Mol.%20Biol.%20Evol&amp;volume=27&amp;pages=221-224" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msp259" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsp259" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsp259"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19854763" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hermant</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Boivin</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Teysset</div> <div class="given-names">L.</div></div></div>, et al.  <div class="year">2015</div>, 
<div class="article-title">Paramutation in Drosophila requires both nuclear and cytoplasmic actors of the piRNA pathway and induces Cis-spreading of piRNA production</div>. <div class="source">Genetics</div> , <div class="volume">201</div>, <div class="fpage">1381</div>–<div class="lpage">1396</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Paramutation%20in%20Drosophila%20requires%20both%20nuclear%20and%20cytoplasmic%20actors%20of%20the%20piRNA%20pathway%20and%20induces%20Cis-spreading%20of%20piRNA%20production&amp;author=C.%20Hermant&amp;author=A.%20Boivin&amp;author=L.%20Teysset&amp;publication_year=2015&amp;journal=Genetics&amp;volume=201&amp;pages=1381-1396" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.115.180307" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.115.180307" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.115.180307"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26482790" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pop</div> <div class="given-names">M.,</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2009</div>, 
<div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>. <div class="source">Genome Biol</div> ., <div class="volume">10</div>, <div class="fpage">R25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%2C%20Pop&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Genome%20Biol&amp;volume=10&amp;pages=R25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-3-r25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-3-r25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Andreyeva</div>
               <div class="given-names">E.N.</div>
            </div>, <div class="name string-name"><div class="surname">Belyaeva</div> <div class="given-names">E.S.</div></div>, <div class="name string-name"><div class="surname">Semeshin</div> <div class="given-names">V.F.</div></div>, <div class="name string-name"><div class="surname">Pokholkova</div> <div class="given-names">G.V.,</div></div>, <div class="name string-name"><div class="surname">Zhimulev</div> <div class="given-names">I.F.</div></div></div>
          <div class="year">2005</div>, 
<div class="article-title">Three distinct chromatin domains in telomere ends of polytene chromosomes in <em>Drosophila melanogaster</em> Tel mutants</div>. <div class="source">J. Cell Sci</div> ., <div class="volume">118</div>, <div class="fpage">5465</div>–<div class="lpage">5477</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Three%20distinct%20chromatin%20domains%20in%20telomere%20ends%20of%20polytene%20chromosomes%20in%20Drosophila%20melanogaster%20Tel%20mutants&amp;author=E.N.%20Andreyeva&amp;author=E.S.%20Belyaeva&amp;author=V.F.%20Semeshin&amp;author=G.V.%2C%20Pokholkova&amp;author=I.F.%20Zhimulev&amp;publication_year=2005&amp;journal=J.%20Cell%20Sci&amp;volume=118&amp;pages=5465-5477" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1242/jcs.02654" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1242%2Fjcs.02654" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1242%2Fjcs.02654"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16278293" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Anderson</div>
               <div class="given-names">J.A.</div>
            </div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">Y.S.,</div></div>, <div class="name string-name"><div class="surname">Langley</div> <div class="given-names">C.H.</div></div></div>
          <div class="year">2008</div>, 
<div class="article-title">Molecular population genetics of Drosophila subtelomeric DNA</div>. <div class="source">Genetics</div> , <div class="volume">178</div>, <div class="fpage">477</div>–<div class="lpage">487</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20population%20genetics%20of%20Drosophila%20subtelomeric%20DNA&amp;author=J.A.%20Anderson&amp;author=Y.S.%2C%20Song&amp;author=C.H.%20Langley&amp;publication_year=2008&amp;journal=Genetics&amp;volume=178&amp;pages=477-487" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.083196" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.083196" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.083196"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18202389" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Novitski</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="year">1951</div>, 
<div class="article-title">Non-random disjunction in Drosophila</div>. <div class="source">Genetics</div> , <div class="volume">36</div>, <div class="fpage">267</div>–<div class="lpage">280</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Non-random%20disjunction%20in%20Drosophila&amp;author=E.%20Novitski&amp;publication_year=1951&amp;journal=Genetics&amp;volume=36&amp;pages=267-280" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14840648" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kern</div>
               <div class="given-names">A.D.,</div>
            </div>, <div class="name string-name"><div class="surname">Begun</div> <div class="given-names">D.J.</div></div></div>
          <div class="year">2008</div>, 
<div class="article-title">Recurrent deletion and gene presence/absence polymorphism: telomere dynamics dominate evolution at the tip of 3L in <em>Drosophila melanogaster</em> and <em>D. simulans</em></div>. <div class="source">Genetics</div> , <div class="volume">179</div>, <div class="fpage">1021</div>–<div class="lpage">1027</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Recurrent%20deletion%20and%20gene%20presence%2Fabsence%20polymorphism%3A%20telomere%20dynamics%20dominate%20evolution%20at%20the%20tip%20of%203L%20in%20Drosophila%20melanogaster%20and%20D.%20simulans&amp;author=A.D.%2C%20Kern&amp;author=D.J.%20Begun&amp;publication_year=2008&amp;journal=Genetics&amp;volume=179&amp;pages=1021-1027" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1534/genetics.107.078345" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1534%2Fgenetics.107.078345" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1534%2Fgenetics.107.078345"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18505885" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mefford</div>
               <div class="given-names">H.C.</div>
            </div>, <div class="name string-name"><div class="surname">Trask</div> <div class="given-names">B.J.</div></div></div>
          <div class="year">2002</div>, 
<div class="article-title">The complex structure and dynamic evolution of human subtelomeres</div>. <div class="source">Nat. Rev. Genet</div> ., <div class="volume">3</div>, <div class="fpage">91</div>–<div class="lpage">102</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20complex%20structure%20and%20dynamic%20evolution%20of%20human%20subtelomeres&amp;author=H.C.%20Mefford&amp;author=B.J.%20Trask&amp;publication_year=2002&amp;journal=Nat.%20Rev.%20Genet&amp;volume=3&amp;pages=91-102" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg727" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg727" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg727"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11836503" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Lindsley</div>
               <div class="given-names">D.L.</div>
            </div>, <div class="name string-name"><div class="surname">Sandler</div> <div class="given-names">L.</div></div></div>
          <div class="year">1977</div>, 
<div class="article-title">The genetic analysis of meiosis in female <em>Drosophila melanogaster</em></div>. <div class="source">Philos. Trans. R. Soc. Lond. B Biol. Sci</div> ., <div class="volume">277</div>, <div class="fpage">295</div>–<div class="lpage">312</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20genetic%20analysis%20of%20meiosis%20in%20female%20Drosophila%20melanogaster&amp;author=D.L.%20Lindsley&amp;author=L.%20Sandler&amp;publication_year=1977&amp;journal=Philos.%20Trans.%20R.%20Soc.%20Lond.%20B%20Biol.%20Sci&amp;volume=277&amp;pages=295-312" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1098/rstb.1977.0019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1098%2Frstb.1977.0019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1098%2Frstb.1977.0019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16292" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Mechler</div>
               <div class="given-names">B.M.</div>
            </div>, <div class="name string-name"><div class="surname">McGinnis</div> <div class="given-names">W.,</div></div>, <div class="name string-name"><div class="surname">Gehring</div> <div class="given-names">W.J.</div></div></div>
          <div class="year">1985</div>, 
<div class="article-title">Molecular cloning of lethal(2)giant larvae, a recessive oncogene of <em>Drosophila melanogaster</em></div>. <div class="source">EMBO J</div> ., <div class="volume">4</div>, <div class="fpage">1551</div>–<div class="lpage">1557</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20cloning%20of%20lethal(2)giant%20larvae%2C%20a%20recessive%20oncogene%20of%20Drosophila%20melanogaster&amp;author=B.M.%20Mechler&amp;author=W.%2C%20McGinnis&amp;author=W.J.%20Gehring&amp;publication_year=1985&amp;journal=EMBO%20J&amp;volume=4&amp;pages=1551-1557" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/3928370" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx017-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx017-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Volff</div>
               <div class="given-names">J.N.</div>
            </div>
          </div>
          <div class="year">2006</div>, 
<div class="article-title">Turning junk into gold: domestication of transposable elements and the creation of new genes in eukaryotes</div>. <div class="source">Bioessays</div> , <div class="volume">28</div>, <div class="fpage">913</div>–<div class="lpage">922</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Turning%20junk%20into%20gold%3A%20domestication%20of%20transposable%20elements%20and%20the%20creation%20of%20new%20genes%20in%20eukaryotes&amp;author=J.N.%20Volff&amp;publication_year=2006&amp;journal=Bioessays&amp;volume=28&amp;pages=913-922" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/(ISSN)1521-1878" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2F(ISSN)1521-1878" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2F(ISSN)1521-1878"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16937363" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3778237&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_figures.pdf?Expires=2147483647&amp;Signature=OGH2n4bnR9NiFX81UTTye-jmElT9p~FHM9nNHzykfAHSmtGWcUpRox9gJ1hbTIJ6rnFefFBlgJc-5spET-PRQPzLiIAvIl31YchAGK3PmDj0cbNDEWM1LpvEcl4uy0g5mpeXSUVhRxw7DAL67AHqBgou1WWQU84U5swJUwnAtFUnMzshY0EbZ7Rzs1kJaiWFyMMzFCeLnPLuC-Wo~~kfEfeNFGgvf0U9AthuESLJMF5ldKZ4lUZk6Di2ELJjZjObQKbGXChcUgSLiD1BFwUL-f4Zl-6YYX73bqUR6R1aGdqUYnSKMvmvIPoQDoVxhsXJkFfY5YrLQcUvZXZVyws~xA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplementary_tables.pdf?Expires=2147483647&amp;Signature=FwvzbfDMugjK1KDcWcYS3hV8j4C9hBqZi4cFJAIEqzTEGwmQGIl3Efr4o0BNod5Da5pJKh15VBV~IUtuNvZktizgJg~9FCXHAiZydTFr-jfgdPKL51SQUgT8JGwEb7DBgeG0oeICN8HVkOb~a9T5CtRb1zV3rvqAwWCdxz23ue56MdKE13LpJ~V~WFpdzkdYifuwuqjpHHoCTNdqoN3I1AdJ~SqLbv4fxbPx2tFTv22Hgc2yDzuX-~4pKDk2TDlukWMjQUCm7Q08N5UMs3uzjx8vHY9-Ms7rr9ZO-Osei8Rli~Y8BLfuMzrDvtRgCEkphuoLu99zb3263UmEnQ6Low__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - pdf file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/5/10.1093_dnares_dsx017/2/dsx017_supp_asiflaidin_supplfiglegends_suppresults.pdf?Expires=2147483647&amp;Signature=uRjogxkb2t0eFeSq7MSWiJLL4~hOKaxenYuuHDLixapfODiWx6WVsCKBzWUVK1IfpLKAgw9d6nDMSRGZKy8L3N9-XZsr--AyitmD~JaSD5veRSd3TCSus8bXM198eeGqMyKQ3tMW~iJWbFwEc7YRO5iEkFzP4Xx7xDTYxf1yvCDFNWqMOGxIhnE2kNO5lbOuRX4u-SCZwsYaSl~qIRlfk7V5mohqH1lfFqwxPFe2QJCac15bLFthzJfiXuanR75SQIhdMTaIOahB5QITAQGyqAUncqs4nhF3XeveaQuYXecrLYVk9iBx0DsVkfLgEbwL0xTSpC6BueCPKn98xvDh8g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">588</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000413018400002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx017"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3778237"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000413018400002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f5%2f459%2f3778237">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000413018400002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F5%2F459%2F3778237" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3778237'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Short and long-term evolutionary dynamics of subtelomeric piRNA clusters in Drosophila" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx017");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f5%2f459%2f3778237">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

