

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/21/4/447/2754535","@type":"ScholarlyArticle","name":"Motif-Independent Prediction of a Secondary Metabolism Gene Cluster Using Comparative Genomics: Application to Sequenced Genomes of  Aspergillus  and Ten Other Filamentous Fungal Species","datePublished":"2014-04-11","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/21/4","@type":"PublicationIssue","issueNumber":"4","datePublished":"2014-08-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsu010","keywords":["secondary metabolism","bioinformatics","filamentous fungi"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/21/4/447/2754535","author":[{"name":"Takeda, Itaru","affiliation":"1  Department of Biotechnology and Life Science ,  Tokyo University of Agriculture and Technology ,  2-24-16 Naka-cho, Koganei, Tokyo 184-8588 ,  Japan    2  Bioproduction Research Institute ,  National Institute of Advanced Industrial Science and Technology (AIST) ,  Higashi 1-1-1, Tsukuba, Ibaraki 305-8566 ,  Japan","@type":"Person"},{"name":"Umemura, Myco","affiliation":"3  Bioproduction Research Institute ,  National Institute of Advanced Industrial Science and Technology (AIST) ,  Higashi-Nijo 17-2-1, Tsukisamu, Sapporo, Hokkaido 062-8517 ,  Japan","@type":"Person"},{"name":"Koike, Hideaki","affiliation":"2  Bioproduction Research Institute ,  National Institute of Advanced Industrial Science and Technology (AIST) ,  Higashi 1-1-1, Tsukuba, Ibaraki 305-8566 ,  Japan","@type":"Person"},{"name":"Asai, Kiyoshi","affiliation":"4  Department of Computational Biology, Graduate School of Frontier Sciences ,  The University of Tokyo ,  5-1-5 Kashiwanoha, Kashiwa-shi Chiba 277-8561 ,  Japan    5  Computational Biology Research Center (CBRC) ,  National Institute of Advanced Industrial Science and Technology (AIST) ,  Tokyo Waterfront Bio-IT Research Building, 2-4-7 Aomi, Koto-ku, Tokyo 135-0064 ,  Japan","@type":"Person"},{"name":"Machida, Masayuki","affiliation":"1  Department of Biotechnology and Life Science ,  Tokyo University of Agriculture and Technology ,  2-24-16 Naka-cho, Koganei, Tokyo 184-8588 ,  Japan    2  Bioproduction Research Institute ,  National Institute of Advanced Industrial Science and Technology (AIST) ,  Higashi 1-1-1, Tsukuba, Ibaraki 305-8566 ,  Japan    3  Bioproduction Research Institute ,  National Institute of Advanced Industrial Science and Technology (AIST) ,  Higashi-Nijo 17-2-1, Tsukisamu, Sapporo, Hokkaido 062-8517 ,  Japan","@type":"Person"}],"description":"Despite their biological importance, a significant number of genes for secondary metabolite biosynthesis (SMB) remain undetected due largely to the fact that th","pageStart":"447","pageEnd":"457","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Motif-Independent Prediction of a Secondary Metabolism Gene Cluster Using Comparative Genomics: Application to Sequenced Genomes of  Aspergillus  and Ten Other Filamentous Fungal Species","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Motif-Independent Prediction of a Secondary Metabolism Gene Cluster Using Comparative Genomics: Application to Sequenced Genomes of  Aspergillus  and Ten Other Filamentous Fungal Species" />
    <meta property="og:description" content="Despite their biological importance, a significant number of genes for secondary metabolite biosynthesis (SMB) remain undetected due largely to the fact that th" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/21/4/447/2754535" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2014-04-11" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Takeda, Itaru" /><meta name="citation_author_institution" content="Department of Biotechnology and Life Science, Tokyo University of Agriculture and Technology, 2-24-16 Naka-cho, Koganei, Tokyo 184-8588, Japan" /><meta name="citation_author_institution" content="Bioproduction Research Institute, National Institute of Advanced Industrial Science and Technology (AIST), Higashi 1-1-1, Tsukuba, Ibaraki 305-8566, Japan" /><meta name="citation_author" content="Umemura, Myco" /><meta name="citation_author_institution" content="Bioproduction Research Institute, National Institute of Advanced Industrial Science and Technology (AIST), Higashi-Nijo 17-2-1, Tsukisamu, Sapporo, Hokkaido 062-8517, Japan" /><meta name="citation_author" content="Koike, Hideaki" /><meta name="citation_author_institution" content="Bioproduction Research Institute, National Institute of Advanced Industrial Science and Technology (AIST), Higashi 1-1-1, Tsukuba, Ibaraki 305-8566, Japan" /><meta name="citation_author" content="Asai, Kiyoshi" /><meta name="citation_author_institution" content="Department of Computational Biology, Graduate School of Frontier Sciences, The University of Tokyo, 5-1-5 Kashiwanoha, Kashiwa-shi Chiba 277-8561, Japan" /><meta name="citation_author_institution" content="Computational Biology Research Center (CBRC), National Institute of Advanced Industrial Science and Technology (AIST), Tokyo Waterfront Bio-IT Research Building, 2-4-7 Aomi, Koto-ku, Tokyo 135-0064, Japan" /><meta name="citation_author" content="Machida, Masayuki" /><meta name="citation_author_institution" content="Department of Biotechnology and Life Science, Tokyo University of Agriculture and Technology, 2-24-16 Naka-cho, Koganei, Tokyo 184-8588, Japan" /><meta name="citation_author_institution" content="Bioproduction Research Institute, National Institute of Advanced Industrial Science and Technology (AIST), Higashi 1-1-1, Tsukuba, Ibaraki 305-8566, Japan" /><meta name="citation_author_institution" content="Bioproduction Research Institute, National Institute of Advanced Industrial Science and Technology (AIST), Higashi-Nijo 17-2-1, Tsukisamu, Sapporo, Hokkaido 062-8517, Japan" /><meta name="citation_title" content="Motif-Independent Prediction of a Secondary Metabolism Gene Cluster Using Comparative Genomics: Application to Sequenced Genomes of Aspergillus and Ten Other Filamentous Fungal Species" /><meta name="citation_firstpage" content="447" /><meta name="citation_lastpage" content="457" /><meta name="citation_doi" content="10.1093/dnares/dsu010" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="21" /><meta name="citation_issue" content="4" /><meta name="citation_publication_date" content="2014/08/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The fumitremorgin gene cluster of Aspergillus fumigatus: identification of a gene encoding brevianamide F synthetase;  citation_journal_title=Chembiochem;  citation_year=2006;  citation_volume=7;  citation_pages=1062-9; " /><meta name="citation_reference" content="citation_title=Genomics-driven discovery of PKS-NRPS hybrid metabolites from Aspergillus nidulans;  citation_journal_title=Nat Chem Biol.;  citation_year=2007;  citation_volume=3;  citation_pages=213-7; " /><meta name="citation_reference" content="citation_title=Genome sequencing and analysis of the versatile cell factory Aspergillus niger CBS 513.88;  citation_journal_title=Nat Biotechnol.;  citation_year=2007;  citation_volume=25;  citation_pages=221-31; " /><meta name="citation_reference" content="citation_title=CLUSEAN: a computer-based framework for the automated analysis of bacterial secondary metabolite biosynthetic gene clusters;  citation_journal_title=J Biotechnol.;  citation_year=2009;  citation_volume=140;  citation_pages=13-7; " /><meta name="citation_reference" content="citation_title=ClustScan: an integrated program package for the semi-automatic annotation of modular biosynthetic gene clusters and in silico prediction of novel chemical structures;  citation_journal_title=Nucleic Acids Res.;  citation_year=2008;  citation_volume=36;  citation_pages=6882-92; " /><meta name="citation_reference" content="citation_title=SMURF: genomic mapping of fungal secondary metabolite clusters;  citation_journal_title=Fungal Genet Biol.;  citation_year=2010;  citation_volume=47;  citation_pages=736-41; " /><meta name="citation_reference" content="citation_title=antiSMASH 2.0—a versatile platform for genome mining of secondary metabolite producers;  citation_journal_title=Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=W204-12; " /><meta name="citation_reference" content="citation_title=Oxylipins in fungi;  citation_journal_title=FEBS J.;  citation_year=2011;  citation_volume=278;  citation_pages=1047-63; " /><meta name="citation_reference" content="citation_title=Identification and characterization of genes responsible for biosynthesis of kojic acid, an industrially important compound from Aspergillus oryzae;  citation_journal_title=Fungal Genet Biol.;  citation_year=2010;  citation_volume=47;  citation_pages=953-61; " /><meta name="citation_reference" content="citation_title=Fine De Novo Sequencing of a Fungal Genome Using only SOLiD Short Read Data: Verification on Aspergillus oryzae RIB40;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e63673" /><meta name="citation_reference" content="citation_title=Advances in Aspergillus secondary metabolite research in the post-genomic era;  citation_journal_title=Nat Prod Rep.;  citation_year=2012;  citation_volume=29;  citation_pages=351-71; " /><meta name="citation_reference" content="citation_title=Genome sequencing and analysis of Aspergillus oryzae;  citation_journal_title=Nature;  citation_year=2005;  citation_volume=438;  citation_pages=1157-61; " /><meta name="citation_reference" content="citation_title=The function and evolution of the Aspergillus genome;  citation_journal_title=Trends Microbiol.;  citation_year=2013;  citation_volume=21;  citation_pages=14-22; " /><meta name="citation_reference" content="citation_title=Gapped BLAST and PSI-BLAST: a new generation of protein database search programs;  citation_journal_title=Nucleic Acids Res.;  citation_year=1997;  citation_volume=25;  citation_pages=3389-402; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool;  citation_journal_title=J Mol Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=Identification of common molecular subsequences;  citation_journal_title=J Mol Biol.;  citation_year=1981;  citation_volume=147;  citation_pages=195-7; " /><meta name="citation_reference" content="citation_title=Comparative biosequence metrics;  citation_journal_title=J Mol Evol.;  citation_year=1981;  citation_volume=18;  citation_pages=38-46; " /><meta name="citation_reference" content="citation_title=antiSMASH: rapid identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genome sequences;  citation_journal_title=Nucleic Acids Res.;  citation_year=2011;  citation_volume=39;  citation_pages=W339-346; " /><meta name="citation_reference" content="citation_title=Horizontal transfer of a large and highly toxic secondary metabolic gene cluster between fungi;  citation_journal_title=Curr Biol.;  citation_year=2011;  citation_volume=21;  citation_pages=134-9; " /><meta name="citation_reference" content="citation_title=Genetics and biosynthesis of aflatoxins and sterigmatocystin; citation_publisher=Springer, Berlin, Germany;  citation_title=Agricultural Applications;  citation_year=2002;  citation_pages=55-69; " /><meta name="citation_reference" content="citation_title=Horizontal transfer and death of a fungal secondary metabolic gene cluster;  citation_journal_title=Genome Biol Evol.;  citation_year=2012;  citation_volume=4;  citation_pages=289-93; " /><meta name="citation_reference" content="citation_title=Fungal secondary metabolism - from biochemistry to genomics;  citation_journal_title=Nat Rev Microbiol.;  citation_year=2005;  citation_volume=3;  citation_pages=937-47; " /><meta name="citation_reference" content="citation_title=Efflux in fungi: la piece de resistance;  citation_journal_title=PLoS Pathog.;  citation_year=2009;  citation_volume=5;  citation_pages=e1000486" /><meta name="citation_reference" content="citation_title=Comparative genome analysis between Aspergillus oryzae strains reveals close relationship between sites of mutation localization and regions of highly divergent genes among Aspergillus species;  citation_journal_title=DNA Res.;  citation_year=2012;  citation_volume=19;  citation_pages=375-82; " /><meta name="citation_reference" content="citation_title=Genomics of Aspergillus oryzae: learning from the history of Koji mold and exploration of its future;  citation_journal_title=DNA Res.;  citation_year=2008;  citation_volume=15;  citation_pages=173-83; " /><meta name="citation_reference" content="citation_title=Two different secondary metabolism gene clusters occupied the same ancestral locus in fungal dermatophytes of the arthrodermataceae;  citation_journal_title=PLoS One;  citation_year=2012;  citation_volume=7;  citation_pages=e41903" /><meta name="citation_reference" content="citation_title=Accurate prediction of secondary metabolite gene clusters in filamentous fungi;  citation_journal_title=Proc Natl Acad Sci USA;  citation_year=2013;  citation_volume=110;  citation_pages=E99-107; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/21/4/447/17462724/dsu010.pdf" /><meta name="description" content="Despite their biological importance, a significant number of genes for secondary metabolite biosynthesis (SMB) remain undetected due largely to the fact that th" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/21/4/447/2754535" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/21/4/447/2754535" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/21/4/447/2754535");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsu010';
    </script>
<title>
	Motif-Independent Prediction of a Secondary Metabolism Gene Cluster Using Comparative Genomics: Application to Sequenced Genomes of Aspergillus and Ten Other Filamentous Fungal Species | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./2754535" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYtQPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUYWtlZGEsIEl0YXJ1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW90ZWNobm9sb2d5IGFuZCBMaWZlIFNjaWVuY2UsIFRva3lvIFVuaXZlcnNpdHkgb2YgQWdyaWN1bHR1cmUgYW5kIFRlY2hub2xvZ3ksIDItMjQtMTYgTmFrYS1jaG8sIEtvZ2FuZWksIFRva3lvIDE4NC04NTg4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb3Byb2R1Y3Rpb24gUmVzZWFyY2ggSW5zdGl0dXRlLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWR2YW5jZWQgSW5kdXN0cmlhbCBTY2llbmNlIGFuZCBUZWNobm9sb2d5IChBSVNUKSwgSGlnYXNoaSAxLTEtMSwgVHN1a3ViYSwgSWJhcmFraSAzMDUtODU2NiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJVbWVtdXJhLCBNeWNvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvcHJvZHVjdGlvbiBSZXNlYXJjaCBJbnN0aXR1dGUsIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZHZhbmNlZCBJbmR1c3RyaWFsIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgKEFJU1QpLCBIaWdhc2hpLU5pam8gMTctMi0xLCBUc3VraXNhbXUsIFNhcHBvcm8sIEhva2thaWRvIDA2Mi04NTE3LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IktvaWtlLCBIaWRlYWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQmlvcHJvZHVjdGlvbiBSZXNlYXJjaCBJbnN0aXR1dGUsIE5hdGlvbmFsIEluc3RpdHV0ZSBvZiBBZHZhbmNlZCBJbmR1c3RyaWFsIFNjaWVuY2UgYW5kIFRlY2hub2xvZ3kgKEFJU1QpLCBIaWdhc2hpIDEtMS0xLCBUc3VrdWJhLCBJYmFyYWtpIDMwNS04NTY2LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkFzYWksIEtpeW9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENvbXB1dGF0aW9uYWwgQmlvbG9neSwgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgNS0xLTUgS2FzaGl3YW5vaGEsIEthc2hpd2Etc2hpIENoaWJhIDI3Ny04NTYxLCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkNvbXB1dGF0aW9uYWwgQmlvbG9neSBSZXNlYXJjaCBDZW50ZXIgKENCUkMpLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWR2YW5jZWQgSW5kdXN0cmlhbCBTY2llbmNlIGFuZCBUZWNobm9sb2d5IChBSVNUKSwgVG9reW8gV2F0ZXJmcm9udCBCaW8tSVQgUmVzZWFyY2ggQnVpbGRpbmcsIDItNC03IEFvbWksIEtvdG8ta3UsIFRva3lvIDEzNS0wMDY0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ik1hY2hpZGEsIE1hc2F5dWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBCaW90ZWNobm9sb2d5IGFuZCBMaWZlIFNjaWVuY2UsIFRva3lvIFVuaXZlcnNpdHkgb2YgQWdyaWN1bHR1cmUgYW5kIFRlY2hub2xvZ3ksIDItMjQtMTYgTmFrYS1jaG8sIEtvZ2FuZWksIFRva3lvIDE4NC04NTg4LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkJpb3Byb2R1Y3Rpb24gUmVzZWFyY2ggSW5zdGl0dXRlLCBOYXRpb25hbCBJbnN0aXR1dGUgb2YgQWR2YW5jZWQgSW5kdXN0cmlhbCBTY2llbmNlIGFuZCBUZWNobm9sb2d5IChBSVNUKSwgSGlnYXNoaSAxLTEtMSwgVHN1a3ViYSwgSWJhcmFraSAzMDUtODU2NiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW9wcm9kdWN0aW9uIFJlc2VhcmNoIEluc3RpdHV0ZSwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEFkdmFuY2VkIEluZHVzdHJpYWwgU2NpZW5jZSBhbmQgVGVjaG5vbG9neSAoQUlTVCksIEhpZ2FzaGktTmlqbyAxNy0yLTEsIFRzdWtpc2FtdSwgU2FwcG9ybywgSG9ra2FpZG8gMDYyLTg1MTcsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJNb3RpZi1JbmRlcGVuZGVudCBQcmVkaWN0aW9uIG9mIGEgU2Vjb25kYXJ5IE1ldGFib2xpc20gR2VuZSBDbHVzdGVyIFVzaW5nIENvbXBhcmF0aXZlIEdlbm9taWNzOiBBcHBsaWNhdGlvbiB0byBTZXF1ZW5jZWQgR2Vub21lcyBvZiBBc3BlcmdpbGx1cyBhbmQgVGVuIE90aGVyIEZpbGFtZW50b3VzIEZ1bmdhbCBTcGVjaWVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iNDQ3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSI0NTciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3UwMTAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMjEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxNC8wOC8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBmdW1pdHJlbW9yZ2luIGdlbmUgY2x1c3RlciBvZiBBc3BlcmdpbGx1cyBmdW1pZ2F0dXM6IGlkZW50aWZpY2F0aW9uIG9mIGEgZ2VuZSBlbmNvZGluZyBicmV2aWFuYW1pZGUgRiBzeW50aGV0YXNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DaGVtYmlvY2hlbTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9MTA2Mi05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljcy1kcml2ZW4gZGlzY292ZXJ5IG9mIFBLUy1OUlBTIGh5YnJpZCBtZXRhYm9saXRlcyBmcm9tIEFzcGVyZ2lsbHVzIG5pZHVsYW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQgQ2hlbSBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9MjEzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZSBzZXF1ZW5jaW5nIGFuZCBhbmFseXNpcyBvZiB0aGUgdmVyc2F0aWxlIGNlbGwgZmFjdG9yeSBBc3BlcmdpbGx1cyBuaWdlciBDQlMgNTEzLjg4OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQgQmlvdGVjaG5vbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0yMjEtMzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNMVVNFQU46IGEgY29tcHV0ZXItYmFzZWQgZnJhbWV3b3JrIGZvciB0aGUgYXV0b21hdGVkIGFuYWx5c2lzIG9mIGJhY3RlcmlhbCBzZWNvbmRhcnkgbWV0YWJvbGl0ZSBiaW9zeW50aGV0aWMgZ2VuZSBjbHVzdGVyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SiBCaW90ZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xNDA7ICBjaXRhdGlvbl9wYWdlcz0xMy03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DbHVzdFNjYW46IGFuIGludGVncmF0ZWQgcHJvZ3JhbSBwYWNrYWdlIGZvciB0aGUgc2VtaS1hdXRvbWF0aWMgYW5ub3RhdGlvbiBvZiBtb2R1bGFyIGJpb3N5bnRoZXRpYyBnZW5lIGNsdXN0ZXJzIGFuZCBpbiBzaWxpY28gcHJlZGljdGlvbiBvZiBub3ZlbCBjaGVtaWNhbCBzdHJ1Y3R1cmVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzY7ICBjaXRhdGlvbl9wYWdlcz02ODgyLTkyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TTVVSRjogZ2Vub21pYyBtYXBwaW5nIG9mIGZ1bmdhbCBzZWNvbmRhcnkgbWV0YWJvbGl0ZSBjbHVzdGVyczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RnVuZ2FsIEdlbmV0IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ3OyAgY2l0YXRpb25fcGFnZXM9NzM2LTQxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1hbnRpU01BU0ggMi4w4oCUYSB2ZXJzYXRpbGUgcGxhdGZvcm0gZm9yIGdlbm9tZSBtaW5pbmcgb2Ygc2Vjb25kYXJ5IG1ldGFib2xpdGUgcHJvZHVjZXJzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9NDE7ICBjaXRhdGlvbl9wYWdlcz1XMjA0LTEyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PeHlsaXBpbnMgaW4gZnVuZ2k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUZFQlMgSi47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc4OyAgY2l0YXRpb25fcGFnZXM9MTA0Ny02MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIGNoYXJhY3Rlcml6YXRpb24gb2YgZ2VuZXMgcmVzcG9uc2libGUgZm9yIGJpb3N5bnRoZXNpcyBvZiBrb2ppYyBhY2lkLCBhbiBpbmR1c3RyaWFsbHkgaW1wb3J0YW50IGNvbXBvdW5kIGZyb20gQXNwZXJnaWxsdXMgb3J5emFlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1GdW5nYWwgR2VuZXQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NDc7ICBjaXRhdGlvbl9wYWdlcz05NTMtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZpbmUgRGUgTm92byBTZXF1ZW5jaW5nIG9mIGEgRnVuZ2FsIEdlbm9tZSBVc2luZyBvbmx5IFNPTGlEIFNob3J0IFJlYWQgRGF0YTogVmVyaWZpY2F0aW9uIG9uIEFzcGVyZ2lsbHVzIG9yeXphZSBSSUI0MDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPbmU7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPWU2MzY3MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFkdmFuY2VzIGluIEFzcGVyZ2lsbHVzIHNlY29uZGFyeSBtZXRhYm9saXRlIHJlc2VhcmNoIGluIHRoZSBwb3N0LWdlbm9taWMgZXJhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQgUHJvZCBSZXAuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTI5OyAgY2l0YXRpb25fcGFnZXM9MzUxLTcxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUgc2VxdWVuY2luZyBhbmQgYW5hbHlzaXMgb2YgQXNwZXJnaWxsdXMgb3J5emFlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDM4OyAgY2l0YXRpb25fcGFnZXM9MTE1Ny02MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIGZ1bmN0aW9uIGFuZCBldm9sdXRpb24gb2YgdGhlIEFzcGVyZ2lsbHVzIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9VHJlbmRzIE1pY3JvYmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0xNC0yMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FwcGVkIEJMQVNUIGFuZCBQU0ktQkxBU1Q6IGEgbmV3IGdlbmVyYXRpb24gb2YgcHJvdGVpbiBkYXRhYmFzZSBzZWFyY2ggcHJvZ3JhbXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MTk5NzsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTMzODktNDAyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CYXNpYyBsb2NhbCBhbGlnbm1lbnQgc2VhcmNoIHRvb2w7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUogTW9sIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTIxNTsgIGNpdGF0aW9uX3BhZ2VzPTQwMy0xMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgY29tbW9uIG1vbGVjdWxhciBzdWJzZXF1ZW5jZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUogTW9sIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTgxOyAgY2l0YXRpb25fdm9sdW1lPTE0NzsgIGNpdGF0aW9uX3BhZ2VzPTE5NS03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBiaW9zZXF1ZW5jZSBtZXRyaWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KIE1vbCBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MTk4MTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTM4LTQ2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1hbnRpU01BU0g6IHJhcGlkIGlkZW50aWZpY2F0aW9uLCBhbm5vdGF0aW9uIGFuZCBhbmFseXNpcyBvZiBzZWNvbmRhcnkgbWV0YWJvbGl0ZSBiaW9zeW50aGVzaXMgZ2VuZSBjbHVzdGVycyBpbiBiYWN0ZXJpYWwgYW5kIGZ1bmdhbCBnZW5vbWUgc2VxdWVuY2VzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mzk7ICBjaXRhdGlvbl9wYWdlcz1XMzM5LTM0NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SG9yaXpvbnRhbCB0cmFuc2ZlciBvZiBhIGxhcmdlIGFuZCBoaWdobHkgdG94aWMgc2Vjb25kYXJ5IG1ldGFib2xpYyBnZW5lIGNsdXN0ZXIgYmV0d2VlbiBmdW5naTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q3VyciBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTEzNC05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5ldGljcyBhbmQgYmlvc3ludGhlc2lzIG9mIGFmbGF0b3hpbnMgYW5kIHN0ZXJpZ21hdG9jeXN0aW47IGNpdGF0aW9uX3B1Ymxpc2hlcj1TcHJpbmdlciwgQmVybGluLCBHZXJtYW55OyAgY2l0YXRpb25fdGl0bGU9QWdyaWN1bHR1cmFsIEFwcGxpY2F0aW9uczsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3BhZ2VzPTU1LTY5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ib3Jpem9udGFsIHRyYW5zZmVyIGFuZCBkZWF0aCBvZiBhIGZ1bmdhbCBzZWNvbmRhcnkgbWV0YWJvbGljIGdlbmUgY2x1c3RlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wgRXZvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPTI4OS05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RnVuZ2FsIHNlY29uZGFyeSBtZXRhYm9saXNtIC0gZnJvbSBiaW9jaGVtaXN0cnkgdG8gZ2Vub21pY3M7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdCBSZXYgTWljcm9iaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zOyAgY2l0YXRpb25fcGFnZXM9OTM3LTQ3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FZmZsdXggaW4gZnVuZ2k6IGxhIHBpZWNlIGRlIHJlc2lzdGFuY2U7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgUGF0aG9nLjsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT01OyAgY2l0YXRpb25fcGFnZXM9ZTEwMDA0ODYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5vbWUgYW5hbHlzaXMgYmV0d2VlbiBBc3BlcmdpbGx1cyBvcnl6YWUgc3RyYWlucyByZXZlYWxzIGNsb3NlIHJlbGF0aW9uc2hpcCBiZXR3ZWVuIHNpdGVzIG9mIG11dGF0aW9uIGxvY2FsaXphdGlvbiBhbmQgcmVnaW9ucyBvZiBoaWdobHkgZGl2ZXJnZW50IGdlbmVzIGFtb25nIEFzcGVyZ2lsbHVzIHNwZWNpZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUROQSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9Mzc1LTgyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWljcyBvZiBBc3BlcmdpbGx1cyBvcnl6YWU6IGxlYXJuaW5nIGZyb20gdGhlIGhpc3Rvcnkgb2YgS29qaSBtb2xkIGFuZCBleHBsb3JhdGlvbiBvZiBpdHMgZnV0dXJlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1ETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xNTsgIGNpdGF0aW9uX3BhZ2VzPTE3My04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHdvIGRpZmZlcmVudCBzZWNvbmRhcnkgbWV0YWJvbGlzbSBnZW5lIGNsdXN0ZXJzIG9jY3VwaWVkIHRoZSBzYW1lIGFuY2VzdHJhbCBsb2N1cyBpbiBmdW5nYWwgZGVybWF0b3BoeXRlcyBvZiB0aGUgYXJ0aHJvZGVybWF0YWNlYWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz1lNDE5MDMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2N1cmF0ZSBwcmVkaWN0aW9uIG9mIHNlY29uZGFyeSBtZXRhYm9saXRlIGdlbmUgY2x1c3RlcnMgaW4gZmlsYW1lbnRvdXMgZnVuZ2k7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MgTmF0bCBBY2FkIFNjaSBVU0E7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9RTk5LTEwNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2Z1bGx0ZXh0X3dvcmxkX3JlYWRhYmxlIiBjb250ZW50PSIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcGRmX3VybCIgY29udGVudD0iaHR0cHM6Ly9hY2FkZW1pYy5vdXAuY29tL2RuYXJlc2VhcmNoL2FydGljbGUtcGRmLzIxLzQvNDQ3LzE3NDYyNzI0L2RzdTAxMC5wZGYiIC8+PG1ldGEgbmFtZT0iZGVzY3JpcHRpb24iIGNvbnRlbnQ9IkRlc3BpdGUgdGhlaXIgYmlvbG9naWNhbCBpbXBvcnRhbmNlLCBhIHNpZ25pZmljYW50IG51bWJlciBvZiBnZW5lcyBmb3Igc2Vjb25kYXJ5IG1ldGFib2xpdGUgYmlvc3ludGhlc2lzIChTTUIpIHJlbWFpbiB1bmRldGVjdGVkIGR1ZSBsYXJnZWx5IHRvIHRoZSBmYWN0IHRoYXQgdGgiIC8+ZAIED2QWAmYPZBYCZg8WBB4EaHJlZgVzLy9vdXAuc2lsdmVyY2hhaXItY2RuLmNvbS9Eb3dubG9hZEZpbGUvRG93bmxvYWRJbWFnZS5hc3B4P2ltYWdlPSZQUFR0eXBlPVNsaWRlU2V0JmFyPTI3NTQ1MzUmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkxBrb9iFYidz3yWq6dAd7omP5Nzk=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f447%2f2754535" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f447%2f2754535" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=17419" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/21/4"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/21/4/1/m_cover.gif?Expires=2147483647&amp;Signature=u1K0F1kMa-64NqKa4J2VQTk-ZgDNBNdy7xP9K68v94JN5WFnh6Of~dA8F~vuNuY-DWKv8sgKc6Tx8dCh~u-HxATm4iXS709rS-0HkL4VrV-yHYOvTEJMgBGMYLmkCZLHLnqJ4CE8~N9vWk3MlqGsd6CiX6Nq4~C6vim6BhKuTZhFWadSb~4gUcfc5RIhx0e6adpzN-8ic5w6tmizeNhHnsbot9fC0b5HPTtgY4T0sWivhNwSa56rOcmWTxrX6Ct6aeNXnMXGfCSv4nXY9sUZJB1uoWiHPNj1cxffLQO70T71y1MESr63u0ogFEZ8T3x1cUhck9QSbq-jL~iCjW~OcQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/21/4">
                        <div class="volume trailing-comma">Volume 21</div>
                        <div class="issue">Issue 4</div>

            </a>

        </div>
        <div class="ii-pub-date">
August 2014        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87196087">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196087">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196092">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196092">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196114">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196114">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196132">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196132">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196134">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196134">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87196136">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87196136">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/21/4/439/2754534">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Motif-Independent Prediction of a Secondary Metabolism Gene Cluster Using Comparative Genomics: Application to Sequenced Genomes of <em>Aspergillus</em> and Ten Other Filamentous Fungal Species
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Itaru Takeda</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biotechnology and Life Science</div>, <div class="insititution">Tokyo University of Agriculture and Technology</div>, <div class="addr-line">2-24-16 Naka-cho, Koganei, Tokyo 184-8588</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Bioproduction Research Institute</div>, <div class="insititution">National Institute of Advanced Industrial Science and Technology (AIST)</div>, <div class="addr-line">Higashi 1-1-1, Tsukuba, Ibaraki 305-8566</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Itaru+Takeda">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Takeda I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Takeda I&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Itaru Takeda</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Myco Umemura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Bioproduction Research Institute</div>, <div class="insititution">National Institute of Advanced Industrial Science and Technology (AIST)</div>, <div class="addr-line">Higashi-Nijo 17-2-1, Tsukisamu, Sapporo, Hokkaido 062-8517</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Myco+Umemura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Umemura M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Umemura M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Myco Umemura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hideaki Koike</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Bioproduction Research Institute</div>, <div class="insititution">National Institute of Advanced Industrial Science and Technology (AIST)</div>, <div class="addr-line">Higashi 1-1-1, Tsukuba, Ibaraki 305-8566</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hideaki+Koike">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Koike H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Koike H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hideaki Koike</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Kiyoshi Asai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">Department of Computational Biology, Graduate School of Frontier Sciences</div>, <div class="insititution">The University of Tokyo</div>, <div class="addr-line">5-1-5 Kashiwanoha, Kashiwa-shi Chiba 277-8561</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">5</div><div class="addr-line">Computational Biology Research Center (CBRC)</div>, <div class="insititution">National Institute of Advanced Industrial Science and Technology (AIST)</div>, <div class="addr-line">Tokyo Waterfront Bio-IT Research Building, 2-4-7 Aomi, Koto-ku, Tokyo 135-0064</div>, <div class="country">Japan</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Kiyoshi+Asai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Asai K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Asai K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Kiyoshi Asai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Masayuki Machida</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Department of Biotechnology and Life Science</div>, <div class="insititution">Tokyo University of Agriculture and Technology</div>, <div class="addr-line">2-24-16 Naka-cho, Koganei, Tokyo 184-8588</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">2</div><div class="addr-line">Bioproduction Research Institute</div>, <div class="insititution">National Institute of Advanced Industrial Science and Technology (AIST)</div>, <div class="addr-line">Higashi 1-1-1, Tsukuba, Ibaraki 305-8566</div>, <div class="country">Japan</div></div><div class="aff"><div class="label -label">3</div><div class="addr-line">Bioproduction Research Institute</div>, <div class="insititution">National Institute of Advanced Industrial Science and Technology (AIST)</div>, <div class="addr-line">Higashi-Nijo 17-2-1, Tsukisamu, Sapporo, Hokkaido 062-8517</div>, <div class="country">Japan</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +81-29-861-9447. Fax. +81-029-861-6174. Email <a href="mailto:m.machida@aist.go.jp" target="_blank">m.machida@aist.go.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Masayuki+Machida">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Machida M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Machida M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Masayuki Machida</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="fn1"><div class="fn"><div class="footnote-content">Edited by Dr Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 21, Issue 4, 1 August 2014, Pages 447–457, <a href='https://doi.org/10.1093/dnares/dsu010'>https://doi.org/10.1093/dnares/dsu010</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">11 April 2014</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">22 September 2013</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">24 February 2014</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="2754535" href="/dnaresearch/article-pdf/21/4/447/17462724/dsu010.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Itaru Takeda, Myco Umemura, Hideaki Koike, Kiyoshi Asai, Masayuki Machida;  Motif-Independent Prediction of a Secondary Metabolism Gene Cluster Using Comparative Genomics: Application to Sequenced Genomes of <em>Aspergillus</em> and Ten Other Filamentous Fungal Species, <em>DNA Research</em>, Volume 21, Issue 4, 1 August 2014, Pages 447–457, <a href='https://doi.org/10.1093/dnares/dsu010'>https://doi.org/10.1093/dnares/dsu010</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/2754535?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754535?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754535?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754535?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754535?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754535?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/2754535?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Motif-Independent%20Prediction%20of%20a%20Secondary%20Metabolism%20Gene%20Cluster%20Using%20Comparative%20Genomics%3A%20Application%20to%20Sequenced%20Genomes%20of%20%26lt%3Bem%26gt%3BAspergillus%26lt%3B%2Fem%26gt%3B%20and%20Ten%20Other%20Filamentous%20Fungal%20Species&amp;publicationDate=2014-04-11&amp;volumeNum=21&amp;issueNum=4&amp;author=Takeda%2C%20Itaru%3B%20Umemura%2C%20Myco&amp;startPage=447&amp;endPage=457&amp;contentId=10.1093%2Fdnares%2Fdsu010&amp;oa=CC%20BY&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f447%2f2754535" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





<section class="abstract"><p>Despite their biological importance, a significant number of genes for secondary metabolite biosynthesis (SMB) remain undetected due largely to the fact that they are highly diverse and are not expressed under a variety of cultivation conditions. Several software tools including SMURF and antiSMASH have been developed to predict fungal SMB gene clusters by finding core genes encoding polyketide synthase, nonribosomal peptide synthetase and dimethylallyltryptophan synthase as well as several others typically present in the cluster. In this work, we have devised a novel comparative genomics method to identify SMB gene clusters that is independent of motif information of the known SMB genes. The method detects SMB gene clusters by searching for a similar order of genes and their presence in nonsyntenic blocks. With this method, we were able to identify many known SMB gene clusters with the core genes in the genomic sequences of 10 filamentous fungi. Furthermore, we have also detected SMB gene clusters without core genes, including the kojic acid biosynthesis gene cluster of <em>Aspergillus oryzae</em>. By varying the detection parameters of the method, a significant difference in the sequence characteristics was detected between the genes residing inside the clusters and those outside the clusters.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;secondary metabolism&quot;">secondary metabolism</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="bioinformatics">bioinformatics</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;filamentous fungi&quot;">filamentous fungi</a></div>                            <h2 scrollto-destination=87196087 id="87196087" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Secondary metabolites are an important resource for bioactive compounds, including lead compounds for new drugs, effective components of functional foods and chemical raw materials. Although a variety of secondary metabolites have been discovered primarily from actinomycetes, fungi and plants, a significantly larger number of secondary metabolites are thought to remain undetected due to the silencing of corresponding biosynthesis genes under the conditions used for screening.<sup><a reveal-id="DSU010C1" data-open="DSU010C1" class="link link-ref link-reveal xref-bibr">1</a>–<a reveal-id="DSU010C3" data-open="DSU010C3" class="link link-ref link-reveal xref-bibr">3</a></sup></p><p>The genes responsible for the biosynthesis of each secondary metabolite are often clustered in the genome.<sup><a reveal-id="DSU010C4" data-open="DSU010C4" class="link link-ref link-reveal xref-bibr">4</a></sup> Furthermore, the basic structures of the known secondary metabolites are often synthesized by the so-called core genes, polyketide synthase (PKS), nonribosomal peptide synthetase (NRPS) and dimethylallyltryptophan synthase (DMAT). Thus, BLAST and Pfam searches for domains in polypeptides encoded by these genes have served as powerful means in identifying essential genes in secondary metabolism biosynthesis (SMB) gene clusters. Clust Scan and CLUSEAN identify core genes for SMB by searching the functional domains and motifs of PKS and NRPS.<sup><a reveal-id="DSU010C4" data-open="DSU010C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSU010C5" data-open="DSU010C5" class="link link-ref link-reveal xref-bibr">5</a></sup> Other software tools, such as SMURF and antiSMASH, first identify the core genes using their motifs and then extend the flanking genes with homology to genes frequently found in the known SMB gene clusters, including hydroxylases, oxidases, methylases, transcription factors (typically Zn(II)Cys6 binuclear cluster types) and transporter genes.<sup><a reveal-id="DSU010C6" data-open="DSU010C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSU010C7" data-open="DSU010C7" class="link link-ref link-reveal xref-bibr">7</a></sup> However, some SMB gene clusters, such as the oxylipin<sup><a reveal-id="DSU010C8" data-open="DSU010C8" class="link link-ref link-reveal xref-bibr">8</a></sup> and kojic acid<sup><a reveal-id="DSU010C9" data-open="DSU010C9" class="link link-ref link-reveal xref-bibr">9</a></sup> biosynthesis gene clusters, lack core genes in their clusters. These examples indicate the importance of devising a method for the prediction of SMB gene clusters without using the known motifs of the core genes.</p><p>Recently, the development of next-generation sequencing technology has dramatically accelerated the sequencing of the genomes of diverse organisms. Even the genomes of filamentous fungi, which have relatively large genome sizes among microbes, can be accurately sequenced without reference genomes.<sup><a reveal-id="DSU010C10" data-open="DSU010C10" class="link link-ref link-reveal xref-bibr">10</a></sup> The extremely high throughput and low cost of sequencing have increased the motivation to sequence the genomes of closely related species and even strains of the same species<sup><a reveal-id="DSU010C11" data-open="DSU010C11" class="link link-ref link-reveal xref-bibr">11</a></sup> for detailed and comprehensive genome comparisons.</p><p>In this study, we developed a novel method that applies a comparative genomics approach to predict SMB gene clusters, including those without core genes. This method depends on the characteristics of secondary metabolism genes, namely that they are highly enriched in non-syntenic blocks<sup><a reveal-id="DSU010C12" data-open="DSU010C12" class="link link-ref link-reveal xref-bibr">12</a></sup> and are rarely orthologous even between clusters producing similar compounds due to generally high sequence diversity.<sup><a reveal-id="DSU010C13" data-open="DSU010C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Our method successfully predicted SMB gene clusters without using motif information from known genes in the SMB gene clusters. Through the optimization of the prediction parameters, we have also depicted the structural characteristics of the SMB gene clusters.</p>                            <h2 scrollto-destination=87196092 id="87196092" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=87196093 id="87196093" class="section-title" data-legacy-id=s2a>2.1. Genome data</h3>
<p>The nucleotide and amino acid sequences of the genomes and deduced coding sequences, respectively, were retrieved from the following databases: <em>Aspergillus flavus</em> (accession no. EQ963472∼EQ963493) and <em>A. oryzae</em> (accession no. AP007150∼AP007177) from DDBJ/EMBL/GenBank DNA database; <em>A. fumigatus</em>, <em>A. nidulans</em> and <em>A. terreus</em> from the <em>Aspergillus</em> comparative database (<a class="link link-uri" href="http://www.broadinstitute.org/annotation/genome/aspergillus_group/MultiHome.html">http://www.broadinstitute.org/annotation/genome/aspergillus_group/MultiHome.html</a>); <em>Magnaporthe grisea</em> from the <em>Magnaporthe</em> comparative database (<a class="link link-uri" href="http://www.broadinstitute.org/annotation/genome/magnaporthe_comparative/MultiHome.html)">http://www.broadinstitute.org/annotation/genome/magnaporthe_comparative/MultiHome.html)</a>; <em>Chaetomium globosum</em> from the <em>Chaetomium globosum</em> database (<a class="link link-uri" href="http://www.broadinstitute.org/annotation/genome/chaetomium_globosum">http://www.broadinstitute.org/annotation/genome/chaetomium_globosum</a>); and <em>Fusarium graminearum</em>, <em>F. oxysporum</em> and <em>F. verticillioides</em> from the <em>Fusarium</em> comparative database (<a class="link link-uri" href="http://www.broadinstitute.org/annotation/genome/fusarium_group/MultiHome.html">http://www.broadinstitute.org/annotation/genome/fusarium_group/MultiHome.html</a>) at The Broad Institute<strong>.</strong> The gene IDs of GenBank was assigned to the genes annotated by Broad Institute by using BLASTP search.</p>                            <h3 scrollto-destination=87196095 id="87196095" class="section-title" data-legacy-id=s2b>2.2. Algorithm overview</h3>
<div class="block-child-p">The method for prediction of gene clusters devised in this study consists of three steps. The first step is to search pairwise similarity between the genes in the two genomes and to perform successive alignment detections of homologous genes (Fig. <a reveal-id="DSU010F1" data-open="DSU010F1" class="link link-reveal link-table xref-fig">1</a>a and b). This step is based on the assumption that SMB gene clusters that produce compounds that are not identical but that have common basic structures have similar member genes. Low gap and mismatch penalties allow the detection of a gene cluster pair containing inversions and/or deletions in their gene members. The second step is to correct the boundary of the predicted gene cluster. This step is achieved by scoring homologous genes, considering genes outside but proximal to the predicted gene cluster (Fig. <a reveal-id="DSU010F1" data-open="DSU010F1" class="link link-reveal link-table xref-fig">1</a>c and d). The third step is to enrich gene clusters with higher probability to function as SMB gene clusters via synteny analysis (Fig. <a reveal-id="DSU010F1" data-open="DSU010F1" class="link link-reveal link-table xref-fig">1</a>e). Secondary metabolism genes are highly enriched on nonsyntenic blocks when the <em>A. oryzae</em> genome is compared with the genome of <em>A. fumigatus</em> or <em>A. nidulans</em>.<sup><a reveal-id="DSU010C12" data-open="DSU010C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Thus, of the gene clusters predicted in the prior step, those forming syntenic blocks can be eliminated (Fig. <a reveal-id="DSU010F1" data-open="DSU010F1" class="link link-reveal link-table xref-fig">1</a>e). <div data-id="DSU010F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/m_dsu01001.jpeg?Expires=2147483647&Signature=dZJZbVOHd-6d5OYIfmtGWM~vpB6hRXyXtj51VxO6~jRuxJ~qhu6Yn6CALRBM12LZG7Y8MxV46E7FOj~N280plpERfZFv0Rk9FSlswIr~xOdtGZ5KgqIzkX3iuJWawm6JB2OkU7DLMyfhoionttnoONxX-1EBj7DAnvWdKOhXJALcMtI02h2JEk~TT4qTREpQuPgyqllEAtQDTi0hslHh9JNh63oimw~PJfl2N9hsdw91hKfA1FyiZWcnkNc5z3rUYkiwWemUuiAQF0EsxcGBnpu1R7kaxNAmy~Hhuy~3PaEvhH0UOg9S70KJn7WlOoQl35jyQEyTgoqWDWplBu46ng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overview of the prediction method for SMB gene clusters. (a) Broken lines represent homologous gene pairs between two genomes. Each pair of ‘xi1’–‘yj1’, ‘xi2’–‘yj2’, ‘xi4’–‘yj4’, ‘xi5’–‘yj8’, ‘xi6’–‘yj5’ and ‘xi8’–‘xj7’ represents a homolog. The xi and yj represent genes in the first and the second genomes, respectively. (b) The genes were aligned in the genome using the Smith–Waterman algorithm (Param2 = −1). Pairs of contiguous genes from ‘xi1’ to ‘xi8’ in genome 1 and from ‘yj1’ to ‘yj7’ represent an example identified as a seed for predicting a gene cluster (R0 or other seed regions). (c) The seed was extended until the prescribed length (Param3 = 35). The symbols lx and ly represent the numbers of genes added to the seed region of the first and the second genomes, respectively. X and Y represent extended clusters in the first and the second genomes, respectively. (d) The boundaries were corrected (Param4 = −1), and a pair of candidate gene clusters, ‘xi1’ through ‘xi8’ and ‘yj1’ through ‘yj8’, was identified. The symbols ibegin and iend represent the locations of the genes at the beginning and end, respectively, of the cluster in the first genome. The symbols jbegin and jend represent the corresponding gene locations in the second genome. The CB value is the sum of the maximum scores for the upstream and the downstream boundaries of a predicted cluster. The integers are indicated as an example for the particular alignment of clusters represented in this figure. (e) Synteny analysis was performed to distinguish the SMB gene cluster from the syntenic block (SB). The SB, a subset of X and Y, represents a set of genes aligned to create a contiguous block of orthologous gene pairs located within the defined distance between neighboring genes (Param5 = 10 kb). The above parameters are examples and not necessarily those used for the actual analyses." path-from-xml="dsu01001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196096/dsu01001.jpeg" path-from-xml="dsu01001.jpeg" target="_blank">View large</a><a section="87196096" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/dsu01001.jpeg?Expires=2147483647&Signature=pxog5KhaCaVDgYyRjJP~HmXRS7-IxINNsMipHQAUYbQm2gATC~IiPgnxMHxLb2bktJn~UplM-Je6KxMeZVh9C4S35oLluhrbFXfv0gulFVyqXzlf~OISR4KnGErTTf-eOffkP2ohRSpzGEckLix9DeGirnUT9KOxkgP2SUeA7dM~hh2F3Ze7uoRFD4qb5LdhVJ4yjko3cvU7HtCDlK20iqanomJzscBa5vdIB7LYCvQopncXBgNi3Qd6wcLzzkPNyv6qktLiAXlbV0mzFBDLPSnFRIM~LM-LqtLkYno6jvyyMSBSaDmUquZ37OBRVCXK1xFMZaG9BFwtYa~vRkvxtQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196096&ar=2754535&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overview of the prediction method for SMB gene clusters. (a) Broken lines represent homologous gene pairs between two genomes. Each pair of ‘<em>x<sub>i</sub></em><sub>1</sub>’–‘<em>y<sub>j</sub></em><sub>1</sub>’, ‘<em>x<sub>i</sub></em><sub>2</sub>’–‘y<sub>j2</sub>’, ‘<em>x<sub>i</sub></em><sub>4</sub>’–‘<em>y<sub>j</sub></em><sub>4</sub>’, ‘<em>x<sub>i</sub></em><sub>5</sub>’–‘<em>y<sub>j</sub></em><sub>8</sub>’, ‘<em>x<sub>i</sub></em><sub>6</sub>’–‘<em>y<sub>j</sub></em><sub>5</sub>’ and ‘<em>x<sub>i</sub></em><sub>8</sub>’–‘<em>x<sub>j</sub></em><sub>7</sub>’ represents a homolog. The <em>x<sub>i</sub></em> and <em>y<sub>j</sub></em> represent genes in the first and the second genomes, respectively. (b) The genes were aligned in the genome using the Smith–Waterman algorithm (Param2 = −1). Pairs of contiguous genes from ‘<em>x<sub>i</sub></em><sub>1</sub>’ to ‘<em>x<sub>i</sub></em><sub>8</sub>’ in genome 1 and from ‘<em>y<sub>j</sub></em><sub>1</sub>’ to ‘<em>y<sub>j</sub></em><sub>7</sub>’ represent an example identified as a seed for predicting a gene cluster (<em>R</em><sub>0</sub> or other seed regions). (c) The seed was extended until the prescribed length (Param3 = 35). The symbols <em>l<sub>x</sub></em> and <em>l<sub>y</sub></em> represent the numbers of genes added to the seed region of the first and the second genomes, respectively. <em>X</em> and <em>Y</em> represent extended clusters in the first and the second genomes, respectively. (d) The boundaries were corrected (Param4 = −1), and a pair of candidate gene clusters, ‘<em>x<sub>i</sub></em><sub>1</sub>’ through ‘<em>x<sub>i</sub></em><sub>8</sub>’ and ‘<em>y<sub>j</sub></em><sub>1</sub>’ through ‘<em>y<sub>j</sub></em><sub>8</sub>’, was identified. The symbols <em>i</em><sub>begin</sub> and <em>i</em><sub>end</sub> represent the locations of the genes at the beginning and end, respectively, of the cluster in the first genome. The symbols <em>j</em><sub>begin</sub> and <em>j</em><sub>end</sub> represent the corresponding gene locations in the second genome. The CB value is the sum of the maximum scores for the upstream and the downstream boundaries of a predicted cluster. The integers are indicated as an example for the particular alignment of clusters represented in this figure. (e) Synteny analysis was performed to distinguish the SMB gene cluster from the syntenic block (SB). The SB, a subset of <em>X</em> and <em>Y</em>, represents a set of genes aligned to create a contiguous block of orthologous gene pairs located within the defined distance between neighboring genes (Param5 = 10 kb). The above parameters are examples and not necessarily those used for the actual analyses.</p></div></div><div content-id="DSU010F1" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/m_dsu01001.jpeg?Expires=2147483647&Signature=dZJZbVOHd-6d5OYIfmtGWM~vpB6hRXyXtj51VxO6~jRuxJ~qhu6Yn6CALRBM12LZG7Y8MxV46E7FOj~N280plpERfZFv0Rk9FSlswIr~xOdtGZ5KgqIzkX3iuJWawm6JB2OkU7DLMyfhoionttnoONxX-1EBj7DAnvWdKOhXJALcMtI02h2JEk~TT4qTREpQuPgyqllEAtQDTi0hslHh9JNh63oimw~PJfl2N9hsdw91hKfA1FyiZWcnkNc5z3rUYkiwWemUuiAQF0EsxcGBnpu1R7kaxNAmy~Hhuy~3PaEvhH0UOg9S70KJn7WlOoQl35jyQEyTgoqWDWplBu46ng__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Overview of the prediction method for SMB gene clusters. (a) Broken lines represent homologous gene pairs between two genomes. Each pair of ‘xi1’–‘yj1’, ‘xi2’–‘yj2’, ‘xi4’–‘yj4’, ‘xi5’–‘yj8’, ‘xi6’–‘yj5’ and ‘xi8’–‘xj7’ represents a homolog. The xi and yj represent genes in the first and the second genomes, respectively. (b) The genes were aligned in the genome using the Smith–Waterman algorithm (Param2 = −1). Pairs of contiguous genes from ‘xi1’ to ‘xi8’ in genome 1 and from ‘yj1’ to ‘yj7’ represent an example identified as a seed for predicting a gene cluster (R0 or other seed regions). (c) The seed was extended until the prescribed length (Param3 = 35). The symbols lx and ly represent the numbers of genes added to the seed region of the first and the second genomes, respectively. X and Y represent extended clusters in the first and the second genomes, respectively. (d) The boundaries were corrected (Param4 = −1), and a pair of candidate gene clusters, ‘xi1’ through ‘xi8’ and ‘yj1’ through ‘yj8’, was identified. The symbols ibegin and iend represent the locations of the genes at the beginning and end, respectively, of the cluster in the first genome. The symbols jbegin and jend represent the corresponding gene locations in the second genome. The CB value is the sum of the maximum scores for the upstream and the downstream boundaries of a predicted cluster. The integers are indicated as an example for the particular alignment of clusters represented in this figure. (e) Synteny analysis was performed to distinguish the SMB gene cluster from the syntenic block (SB). The SB, a subset of X and Y, represents a set of genes aligned to create a contiguous block of orthologous gene pairs located within the defined distance between neighboring genes (Param5 = 10 kb). The above parameters are examples and not necessarily those used for the actual analyses." path-from-xml="dsu01001.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196096/dsu01001.jpeg" path-from-xml="dsu01001.jpeg" target="_blank">View large</a><a section="87196096" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/dsu01001.jpeg?Expires=2147483647&Signature=pxog5KhaCaVDgYyRjJP~HmXRS7-IxINNsMipHQAUYbQm2gATC~IiPgnxMHxLb2bktJn~UplM-Je6KxMeZVh9C4S35oLluhrbFXfv0gulFVyqXzlf~OISR4KnGErTTf-eOffkP2ohRSpzGEckLix9DeGirnUT9KOxkgP2SUeA7dM~hh2F3Ze7uoRFD4qb5LdhVJ4yjko3cvU7HtCDlK20iqanomJzscBa5vdIB7LYCvQopncXBgNi3Qd6wcLzzkPNyv6qktLiAXlbV0mzFBDLPSnFRIM~LM-LqtLkYno6jvyyMSBSaDmUquZ37OBRVCXK1xFMZaG9BFwtYa~vRkvxtQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196096&ar=2754535&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01001.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Overview of the prediction method for SMB gene clusters. (a) Broken lines represent homologous gene pairs between two genomes. Each pair of ‘<em>x<sub>i</sub></em><sub>1</sub>’–‘<em>y<sub>j</sub></em><sub>1</sub>’, ‘<em>x<sub>i</sub></em><sub>2</sub>’–‘y<sub>j2</sub>’, ‘<em>x<sub>i</sub></em><sub>4</sub>’–‘<em>y<sub>j</sub></em><sub>4</sub>’, ‘<em>x<sub>i</sub></em><sub>5</sub>’–‘<em>y<sub>j</sub></em><sub>8</sub>’, ‘<em>x<sub>i</sub></em><sub>6</sub>’–‘<em>y<sub>j</sub></em><sub>5</sub>’ and ‘<em>x<sub>i</sub></em><sub>8</sub>’–‘<em>x<sub>j</sub></em><sub>7</sub>’ represents a homolog. The <em>x<sub>i</sub></em> and <em>y<sub>j</sub></em> represent genes in the first and the second genomes, respectively. (b) The genes were aligned in the genome using the Smith–Waterman algorithm (Param2 = −1). Pairs of contiguous genes from ‘<em>x<sub>i</sub></em><sub>1</sub>’ to ‘<em>x<sub>i</sub></em><sub>8</sub>’ in genome 1 and from ‘<em>y<sub>j</sub></em><sub>1</sub>’ to ‘<em>y<sub>j</sub></em><sub>7</sub>’ represent an example identified as a seed for predicting a gene cluster (<em>R</em><sub>0</sub> or other seed regions). (c) The seed was extended until the prescribed length (Param3 = 35). The symbols <em>l<sub>x</sub></em> and <em>l<sub>y</sub></em> represent the numbers of genes added to the seed region of the first and the second genomes, respectively. <em>X</em> and <em>Y</em> represent extended clusters in the first and the second genomes, respectively. (d) The boundaries were corrected (Param4 = −1), and a pair of candidate gene clusters, ‘<em>x<sub>i</sub></em><sub>1</sub>’ through ‘<em>x<sub>i</sub></em><sub>8</sub>’ and ‘<em>y<sub>j</sub></em><sub>1</sub>’ through ‘<em>y<sub>j</sub></em><sub>8</sub>’, was identified. The symbols <em>i</em><sub>begin</sub> and <em>i</em><sub>end</sub> represent the locations of the genes at the beginning and end, respectively, of the cluster in the first genome. The symbols <em>j</em><sub>begin</sub> and <em>j</em><sub>end</sub> represent the corresponding gene locations in the second genome. The CB value is the sum of the maximum scores for the upstream and the downstream boundaries of a predicted cluster. The integers are indicated as an example for the particular alignment of clusters represented in this figure. (e) Synteny analysis was performed to distinguish the SMB gene cluster from the syntenic block (SB). The SB, a subset of <em>X</em> and <em>Y</em>, represents a set of genes aligned to create a contiguous block of orthologous gene pairs located within the defined distance between neighboring genes (Param5 = 10 kb). The above parameters are examples and not necessarily those used for the actual analyses.</p></div></div></div>                            <h3 scrollto-destination=87196097 id="87196097" class="section-title" data-legacy-id=s2c>2.3. Identification of homologous and orthologous gene pairs</h3>
<p>Prior to comparing the order of genes between a pair of genomes, homologous gene pairs were identified in an amino acid homology search (Fig. <a reveal-id="DSU010F1" data-open="DSU010F1" class="link link-reveal link-table xref-fig">1</a>a) using BLASTP<sup><a reveal-id="DSU010C14" data-open="DSU010C14" class="link link-ref link-reveal xref-bibr">14</a>,<a reveal-id="DSU010C15" data-open="DSU010C15" class="link link-ref link-reveal xref-bibr">15</a></sup> with e-values (Param1) of 1.0e−5, 1.0e−10, 1.0e−15, 1.0e−30 or 1.0e−50 as thresholds. Orthologs were determined using the bidirectional best BLASTP hit method.</p>                            <h3 scrollto-destination=87196099 id="87196099" class="section-title" data-legacy-id=s2d>2.4. Identification of the seed region pair for a gene cluster</h3>
<div class="block-child-p">The regions for which the order of genes was conserved between the genome pair were searched by local alignment of the genes with the Smith–Waterman algorithm<sup><a reveal-id="DSU010C16" data-open="DSU010C16" class="link link-ref link-reveal xref-bibr">16</a>,<a reveal-id="DSU010C17" data-open="DSU010C17" class="link link-ref link-reveal xref-bibr">17</a></sup> (Fig. <a reveal-id="DSU010F1" data-open="DSU010F1" class="link link-reveal link-table xref-fig">1</a>b). The genes in the first and second genomes were defined as <em>x<sub>i</sub></em> (<em>i</em> = 1, 2, …, <em>I</em>) and <em>y<sub>j</sub></em> (<em>j</em> = 1, 2, … , <em>J</em>), respectively. A matrix (SW) of (<em>J</em> + 1) × (<em>I</em> + 1) was prepared by calculating each cell score according to the following formulas:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtext xmlns="">SW</mtext><mo stretchy="false" xmlns="">(</mo><mspace width="thinmathspace" xmlns=""></mspace><mi xmlns="">j</mi><mo xmlns="">,</mo><mi xmlns="">i</mi><mo stretchy="false" xmlns="">)</mo><mo xmlns="">=</mo><mi xmlns="">m</mi><mi xmlns="">a</mi><mi xmlns="">x</mi><mfenced open="{" close="" xmlns=""><mrow><mtable rowspacing="4pt" columnspacing="1em"><mtr><mtd><mrow><mtext>SW</mtext><mo stretchy="false">(</mo><mspace width="thinmathspace"></mspace><mi>j</mi><mo>−</mo><mn>1</mn><mo>,</mo><mi>i</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo>+</mo><mn>1</mn></mrow></mtd></mtr><mtr><mtd><mrow><mtext>SW</mtext><mo stretchy="false">(</mo><mspace width="thinmathspace"></mspace><mi>j</mi><mo>,</mo><mi>i</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo>+</mo><mrow><msub><mi>P</mi><mrow><mrow><mi mathvariant="normal">gap</mi></mrow></mrow></msub></mrow></mrow></mtd></mtr><mtr><mtd><mrow><mtext>SW</mtext><mo stretchy="false">(</mo><mspace width="thinmathspace"></mspace><mi>j</mi><mo>−</mo><mn>1</mn><mo>,</mo><mi>i</mi><mo stretchy="false">)</mo><mo>+</mo><mrow><msub><mi>P</mi><mrow><mrow><mi mathvariant="normal">gap</mi></mrow></mrow></msub></mrow></mrow></mtd></mtr><mtr><mtd><mn>0</mn></mtd></mtr></mtable></mrow></mfenced></math></div></div></div> when a pair of genes, <em>x<sub>i</sub></em> and <em>y<sub>j</sub></em>, are homologous.<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtext xmlns="">SW</mtext><mo stretchy="false" xmlns="">(</mo><mspace width="thinmathspace" xmlns=""></mspace><mi xmlns="">j</mi><mo xmlns="">,</mo><mi xmlns="">i</mi><mo stretchy="false" xmlns="">)</mo><mo xmlns="">=</mo><mi xmlns="">m</mi><mi xmlns="">a</mi><mi xmlns="">x</mi><mfenced open="{" close="" xmlns=""><mrow><mtable rowspacing="4pt" columnspacing="1em"><mtr><mtd><mrow><mtext>SW</mtext><mo stretchy="false">(</mo><mspace width="thinmathspace"></mspace><mi>j</mi><mo>−</mo><mn>1</mn><mo>,</mo><mi>i</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo>+</mo><mrow><msub><mi>P</mi><mrow><mrow><mi mathvariant="normal">mismatch</mi></mrow></mrow></msub></mrow></mrow></mtd></mtr><mtr><mtd><mrow><mtext>SW</mtext><mo stretchy="false">(</mo><mspace width="thinmathspace"></mspace><mi>j</mi><mo>,</mo><mi>i</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo>+</mo><mrow><msub><mi>P</mi><mrow><mrow><mi mathvariant="normal">gap</mi></mrow></mrow></msub></mrow></mrow></mtd></mtr><mtr><mtd><mrow><mtext>SW</mtext><mo stretchy="false">(</mo><mspace width="thinmathspace"></mspace><mi>j</mi><mo>−</mo><mn>1</mn><mo>,</mo><mi>i</mi><mo stretchy="false">)</mo><mo>+</mo><mrow><msub><mi>P</mi><mrow><mrow><mi mathvariant="normal">gap</mi></mrow></mrow></msub></mrow></mrow></mtd></mtr><mtr><mtd><mn>0</mn></mtd></mtr></mtable></mrow></mfenced></math></div></div></div> when <em>x<sub>i</sub></em> and <em>y<sub>j</sub></em> are not homologous.</div><div class="block-child-p">Values of −0.1, −0.2, −0.4, −0.5 or −1 were used for <em>P</em><sub>gap</sub> and <em>P</em><sub>mismatch</sub>, a gap and a mismatch penalty, respectively (Param2). After the scores were calculated for all of the cells in the matrix based on the similarity between any gene pair, the gene cluster coordinates were obtained by tracing the cells from the pair with the maximum score to that with a score of 0 (Fig. <a reveal-id="DSU010F1" data-open="DSU010F1" class="link link-reveal link-table xref-fig">1</a>b). The pair of gene cluster coordinates was defined as <em>R</em><sub>0</sub>, which was used as one of the seeds for the predicted gene clusters.<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>R</mi><mn>0</mn></msub></mrow><mo xmlns="">=</mo><mo fence="false" stretchy="false" xmlns="">{</mo><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><mspace width="thinmathspace"></mspace><msub><mi>j</mi><mn>1</mn></msub></mrow><mo xmlns="">,</mo><mrow xmlns=""><msub><mi>i</mi><mn>1</mn></msub></mrow><mo stretchy="false" xmlns="">)</mo><mo xmlns="">,</mo><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><mspace width="thinmathspace"></mspace><msub><mi>j</mi><mn>2</mn></msub></mrow><mo xmlns="">,</mo><mrow xmlns=""><msub><mi>i</mi><mn>2</mn></msub></mrow><mo stretchy="false" xmlns="">)</mo><mo xmlns="">,</mo><mo xmlns="">…</mo><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><mspace width="thinmathspace"></mspace><msub><mi>j</mi><mi>m</mi></msub></mrow><mo xmlns="">,</mo><mrow xmlns=""><msub><mi>i</mi><mi>n</mi></msub></mrow><mo stretchy="false" xmlns="">)</mo><mo fence="false" stretchy="false" xmlns="">}</mo><mo xmlns="">,</mo><mspace width="thinmathspace" xmlns=""></mspace><mi xmlns="">w</mi><mi xmlns="">h</mi><mi xmlns="">e</mi><mi xmlns="">r</mi><mi xmlns="">e</mi><mspace width="thinmathspace" xmlns=""></mspace><mrow xmlns=""><mspace width="thinmathspace"></mspace><msub><mi>j</mi><mn>1</mn></msub></mrow><mo xmlns="">≤</mo><mrow xmlns=""><mspace width="thinmathspace"></mspace><msub><mi>j</mi><mn>2</mn></msub></mrow><mo xmlns="">≤</mo><mo xmlns="">⋯</mo><mo xmlns="">≤</mo><mrow xmlns=""><mspace width="thinmathspace"></mspace><msub><mi>j</mi><mi>m</mi></msub></mrow><mo xmlns="">,</mo><mrow xmlns=""><msub><mi>i</mi><mn>1</mn></msub></mrow><mo xmlns="">≤</mo><mrow xmlns=""><msub><mi>i</mi><mrow><mn>2</mn></mrow></msub></mrow><mo xmlns="">≤</mo><mo xmlns="">⋯</mo><mo xmlns="">≤</mo><mrow xmlns=""><msub><mi>i</mi><mi>n</mi></msub></mrow></math></div></div></div></div><p>Other seeds were detected by a traceback of the same score matrix (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/dsu010_Supplementary_Data.zip?Expires=2147483647&Signature=MPiG6gc9hUdygTRnGANiyQgA7d8IpwlJMFM~v9-F7bK2pcJoH7ZgE0EsTi3yY56pjA-paZpYbdY18FI7rR9CHSSMngi-qHrGa4vd7pQr0Mc0u5QSIf-DMyDUR0xMFMI1a2Zm~koOiqyHEthZYvNNyleD9NYlf2zn7yFaWqTCk5Mrt9TUB6W84S79etUoGJlJThjWyuKO~XtL3Eiv1ahciIwfegnGKncFUbO0nF4EhNDB1dvl6wHEdlSzllFLh0b5YSPw1U3q0h7roGA9E4JS0wHc~wDa1hOhJZGBzn-eAOeMXIXx2-rAFhqatyWRWNTGau4jy1n5U3VSdyyni1gybQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>). These seeds were subjected to the correction of boundaries in the next step. Gene cluster coordinates were also searched using the reverse orientation for one of the two genomes.</p>                            <h3 scrollto-destination=87196103 id="87196103" class="section-title" data-legacy-id=s2e>2.5. Correcting gene cluster boundaries</h3>
<p><em>R</em><sub>0</sub> and other seed regions, consisting of the genes <em>x<sub>i</sub></em> (<em>i</em><sub>1</sub> ≤ <em>i</em> ≤ <em>i<sub>n</sub></em>) and <em>y<sub>j</sub></em> (<em>j</em><sub>1</sub> ≤ <em>j</em> ≤ <em>j<sub>m</sub></em>), may not have the correct boundaries as a gene cluster for various reasons, particularly when part of the cluster has a reversed orientation in the gene order. This reversal could be caused by a small inversion.</p><p>In this study, by taking the actual, experimentally confirmed sizes of the clusters into consideration, the minimum number of homologous gene pairs was set to 3, and the values 15, 25, 35, 45, 55 and 65 were used for the maximum number of genes contained in a gene cluster (Param3).</p><div class="block-child-p">After the detection of seeds with cluster sizes under the threshold, the same number of genes located in the vicinity of the seeds was added to both ends of the seeds to extend the cluster size to a predefined number of genes for successive boundary corrections (Fig. <a reveal-id="DSU010F1" data-open="DSU010F1" class="link link-reveal link-table xref-fig">1</a>c). If the number of genes to be added was odd, an additional gene was added to either of the two ends of the cluster. In this study, the same values derived with Param3 were used as the cluster sizes after the addition. When 35 genes were applied to Param3, each set of genes that extended the seed in the first and the second genomes was defined as <em>X</em> and <em>Y</em>, respectively, and each number of genes added to the seed was defined as <em>l<sub>x</sub></em> and <em>l<sub>y</sub></em>, respectively:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">X</mi><mo xmlns="">=</mo><mo fence="false" stretchy="false" xmlns="">{</mo><mrow xmlns=""><msub><mi>x</mi><mi>i</mi></msub></mrow><mrow xmlns=""><mo stretchy="false">|</mo></mrow><mi xmlns="">i</mi><mo xmlns="">:</mo><mi xmlns="">i</mi><mi xmlns="">n</mi><mi xmlns="">t</mi><mi xmlns="">e</mi><mi xmlns="">g</mi><mi xmlns="">e</mi><mi xmlns="">r</mi><mspace width="thickmathspace" xmlns=""></mspace><mi xmlns="">a</mi><mi xmlns="">n</mi><mi xmlns="">d</mi><mspace width="thickmathspace" xmlns=""></mspace><mi xmlns="">s</mi><mi xmlns="">a</mi><mi xmlns="">t</mi><mi xmlns="">i</mi><mi xmlns="">s</mi><mi xmlns="">f</mi><mi xmlns="">y</mi><mi xmlns="">i</mi><mi xmlns="">n</mi><mi xmlns="">g</mi><mspace width="thickmathspace" xmlns=""></mspace><mrow xmlns=""><msub><mi>i</mi><mn>1</mn></msub></mrow><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>l</mi><mi>x</mi></msub></mrow><mo xmlns="">≤</mo><mi xmlns="">i</mi><mo xmlns="">≤</mo><mrow xmlns=""><msub><mi>i</mi><mi>n</mi></msub></mrow><mo xmlns="">+</mo><mrow xmlns=""><msub><mi>l</mi><mi>x</mi></msub></mrow><mo xmlns="">,</mo><mspace width="thickmathspace" xmlns=""></mspace><mi xmlns="">w</mi><mi xmlns="">h</mi><mi xmlns="">e</mi><mi xmlns="">r</mi><mi xmlns="">e</mi><mspace width="thickmathspace" xmlns=""></mspace><mrow xmlns=""><msub><mi>i</mi><mi>n</mi></msub></mrow><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>i</mi><mn>1</mn></msub></mrow><mo xmlns="">+</mo><mn xmlns="">2</mn><mrow xmlns=""><msub><mi>l</mi><mi>x</mi></msub></mrow><mo xmlns="">+</mo><mn xmlns="">1</mn><mo xmlns="">=</mo><mn xmlns="">35</mn><mo fence="false" stretchy="false" xmlns="">}</mo></math></div></div></div><a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mi xmlns="">Y</mi><mo xmlns="">=</mo><mo fence="false" stretchy="false" xmlns="">{</mo><mrow xmlns=""><msub><mi>y</mi><mi>j</mi></msub></mrow><mrow xmlns=""><mo stretchy="false">|</mo></mrow><mi xmlns="">j</mi><mo xmlns="">:</mo><mi xmlns="">i</mi><mi xmlns="">n</mi><mi xmlns="">t</mi><mi xmlns="">e</mi><mi xmlns="">g</mi><mi xmlns="">e</mi><mi xmlns="">r</mi><mspace width="thickmathspace" xmlns=""></mspace><mi xmlns="">a</mi><mi xmlns="">n</mi><mi xmlns="">d</mi><mspace width="thickmathspace" xmlns=""></mspace><mi xmlns="">s</mi><mi xmlns="">a</mi><mi xmlns="">t</mi><mi xmlns="">i</mi><mi xmlns="">s</mi><mi xmlns="">f</mi><mi xmlns="">y</mi><mi xmlns="">i</mi><mi xmlns="">n</mi><mi xmlns="">g</mi><mspace width="thickmathspace" xmlns=""></mspace><mrow xmlns=""><mspace width="thinmathspace"></mspace><msub><mi>j</mi><mn>1</mn></msub></mrow><mo xmlns="">−</mo><mrow xmlns=""><msub><mi>l</mi><mi>y</mi></msub></mrow><mo xmlns="">≤</mo><mi xmlns="">j</mi><mo xmlns="">≤</mo><mrow xmlns=""><mspace width="thinmathspace"></mspace><msub><mi>j</mi><mi>m</mi></msub></mrow><mo xmlns="">+</mo><mrow xmlns=""><msub><mi>l</mi><mi>y</mi></msub></mrow><mo xmlns="">,</mo><mspace width="thickmathspace" xmlns=""></mspace><mi xmlns="">w</mi><mi xmlns="">h</mi><mi xmlns="">e</mi><mi xmlns="">r</mi><mi xmlns="">e</mi><mspace width="thickmathspace" xmlns=""></mspace><mrow xmlns=""><mspace width="thinmathspace"></mspace><msub><mi>j</mi><mi>m</mi></msub></mrow><mo xmlns="">−</mo><mrow xmlns=""><mspace width="thinmathspace"></mspace><msub><mi>j</mi><mn>1</mn></msub></mrow><mo xmlns="">+</mo><mn xmlns="">2</mn><mrow xmlns=""><msub><mi>l</mi><mi>y</mi></msub></mrow><mo xmlns="">+</mo><mn xmlns="">1</mn><mo xmlns="">=</mo><mn xmlns="">35</mn><mo fence="false" stretchy="false" xmlns="">}</mo></math></div></div></div></div><div class="block-child-p">To correct the boundaries of a seed of clustered genes, homologous genes were scored from the gene located at the center of the cluster to both ends of the cluster. A score, SC, was calculated for each gene member in <em>X</em> according to the following formulas (Fig. <a reveal-id="DSU010F1" data-open="DSU010F1" class="link link-reveal link-table xref-fig">1</a>d): <a id="DSU010M5" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU010M5"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtable columnalign="right left" rowspacing=".5em" columnspacing="thickmathspace" xmlns=""><mtr><mtd><mtext>SC</mtext><mo stretchy="false">(</mo><mi>i</mi><mo stretchy="false">)</mo><mo>=</mo><mfenced open="{" close=""><mrow><mtable rowspacing="4pt" columnspacing="1em"><mtr><mtd><mrow><mn>1</mn><mo>,</mo></mrow></mtd><mtd><mrow><mi>i</mi><mo>=</mo><mo stretchy="false">(</mo><mrow><msub><mi>i</mi><mn>1</mn></msub></mrow><mo>+</mo><mrow><msub><mi>i</mi><mi>n</mi></msub></mrow><mo stretchy="false">)</mo><mtext>/</mtext><mn>2</mn></mrow></mtd></mtr><mtr><mtd><mrow><mtext>SC</mtext><mo stretchy="false">(</mo><mi>i</mi><mo>+</mo><mn>1</mn><mo stretchy="false">)</mo><mo>+</mo><mn>1</mn><mo>,</mo></mrow></mtd><mtd><mspace width="thinmathspace"></mspace><mspace width="thinmathspace"></mspace><mrow><mi>i</mi><mo>&lt;</mo><mo stretchy="false">(</mo><mrow><msub><mi>i</mi><mn>1</mn></msub></mrow><mo>+</mo><mrow><msub><mi>i</mi><mi>n</mi></msub></mrow><mo stretchy="false">)</mo><mtext>/</mtext><mn>2</mn></mrow><mspace width="1em"></mspace></mtd></mtr><mtr><mtd><mrow><mtext>SC</mtext><mo stretchy="false">(</mo><mi>i</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo>+</mo><mn>1</mn><mo>,</mo></mrow></mtd><mtd><mrow><mi>i</mi><mo>&gt;</mo><mo stretchy="false">(</mo><mrow><msub><mi>i</mi><mn>1</mn></msub></mrow><mo>+</mo><mrow><msub><mi>i</mi><mi>n</mi></msub></mrow><mo stretchy="false">)</mo><mtext>/</mtext><mn>2</mn></mrow></mtd></mtr></mtable></mrow></mfenced><mrow></mrow></mtd></mtr></mtable></math></div></div><div class="label">(1)</div></div> when <em>x<sub>i</sub></em> has at least one homolog among the members of <em>Y</em> and <a id="DSU010M6" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id="DSU010M6"><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mtext xmlns="">SC</mtext><mo stretchy="false" xmlns="">(</mo><mi xmlns="">i</mi><mo stretchy="false" xmlns="">)</mo><mo xmlns="">=</mo><mfenced open="{" close="" xmlns=""><mtable rowspacing="4pt" columnspacing="1em"><mtr><mtd><mrow><mrow><msub><mi>P</mi><mrow><mrow><mi mathvariant="normal">negative</mi></mrow></mrow></msub></mrow><mo>,</mo></mrow></mtd><mtd><mrow><mi>i</mi><mo>=</mo><mo stretchy="false">(</mo><mrow><msub><mi>i</mi><mn>1</mn></msub></mrow><mo>+</mo><mrow><msub><mi>i</mi><mi>n</mi></msub></mrow><mo stretchy="false">)</mo><mrow><mo>/</mo></mrow><mn>2</mn></mrow></mtd></mtr><mtr><mtd><mrow><mtext>SC</mtext><mo stretchy="false">(</mo><mi>i</mi><mo>+</mo><mn>1</mn><mo stretchy="false">)</mo><mo>+</mo><mrow><msub><mi>P</mi><mrow><mrow><mi mathvariant="normal">negative</mi></mrow></mrow></msub></mrow><mo>,</mo></mrow></mtd><mtd><mrow><mi>i</mi><mo>&lt;</mo><mo stretchy="false">(</mo><mrow><msub><mi>i</mi><mn>1</mn></msub></mrow><mo>+</mo><mrow><msub><mi>i</mi><mi>n</mi></msub></mrow><mo stretchy="false">)</mo><mrow><mo>/</mo></mrow><mn>2</mn></mrow></mtd></mtr><mtr><mtd><mrow><mtext>SC</mtext><mo stretchy="false">(</mo><mi>i</mi><mo>−</mo><mn>1</mn><mo stretchy="false">)</mo><mo>+</mo><mrow><msub><mi>P</mi><mrow><mrow><mi mathvariant="normal">negative</mi></mrow></mrow></msub></mrow><mo>,</mo></mrow></mtd><mtd><mrow><mi>i</mi><mo>&gt;</mo><mo stretchy="false">(</mo><mrow><msub><mi>i</mi><mn>1</mn></msub></mrow><mo>+</mo><mrow><msub><mi>i</mi><mi>n</mi></msub></mrow><mo stretchy="false">)</mo><mrow><mo>/</mo></mrow><mn>2</mn></mrow></mtd></mtr></mtable></mfenced></math></div></div><div class="label">(2)</div></div> when <em>x<sub>i</sub></em> has no homologs among the members of <em>Y</em>.</div><div class="block-child-p"><em>P</em><sub>negative</sub> represents a penalty score for the gene that has no homologs in the paired extended seed. Based on the scores for all of the member genes, a gene cluster candidate was defined between the genes (<em>i</em><sub>begin</sub> and <em>i</em><sub>end</sub>) with the maximum scores in the regions indicated by (1–4), respectively. To evaluate similarity between a pair of detected clusters, a CB value was defined as the sum of the maximum scores at both ends. The boundary correction <em>Y</em> was determined in the same manner. Consequently, a pair of gene cluster candidates, <em>x<sub>i</sub></em> and <em>y<sub>j</sub></em>, was defined as follows:<a id="" class="jumplink-placeholder"> </a><div class="formula-wrap"><div class="disp-formula" content-id=""><div class="math"><math xmlns="http://www.w3.org/1998/Math/MathML"><mrow xmlns=""><msub><mi>x</mi><mi>i</mi></msub></mrow><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><msub><mi>i</mi><mrow><mrow><mi mathvariant="normal">begin</mi></mrow></mrow></msub></mrow><mo xmlns="">≤</mo><mi xmlns="">i</mi><mo xmlns="">≤</mo><mrow xmlns=""><msub><mi>i</mi><mrow><mrow><mi mathvariant="normal">end</mi></mrow></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo><mo xmlns="">,</mo><mspace width="1em" xmlns=""></mspace><mrow xmlns=""><msub><mi>y</mi><mi>j</mi></msub></mrow><mspace width="thickmathspace" xmlns=""></mspace><mo stretchy="false" xmlns="">(</mo><mrow xmlns=""><mspace width="thinmathspace"></mspace><msub><mi>j</mi><mrow><mrow><mi mathvariant="normal">begin</mi></mrow></mrow></msub></mrow><mo xmlns="">≤</mo><mi xmlns="">j</mi><mo xmlns="">≤</mo><mrow xmlns=""><mspace width="thinmathspace"></mspace><msub><mi>j</mi><mrow><mrow><mi mathvariant="normal">end</mi></mrow></mrow></msub></mrow><mo stretchy="false" xmlns="">)</mo></math></div></div></div></div><p>In this study, the values −0.1, −0.2, −0.3, −0.4, −0.5 and −1 were used as the negative penalty (Param4).</p>                            <h3 scrollto-destination=87196110 id="87196110" class="section-title" data-legacy-id=s2f>2.6. Synteny analysis</h3>
<p>Secondary metabolism genes are highly enriched in nonsyntenic blocks.<sup><a reveal-id="DSU010C12" data-open="DSU010C12" class="link link-ref link-reveal xref-bibr">12</a></sup> Secondary metabolism genes, which have high sequence diversity in general,<sup><a reveal-id="DSU010C13" data-open="DSU010C13" class="link link-ref link-reveal xref-bibr">13</a></sup> are rarely orthologous in the comparison of genomes between two species. In contrast, syntenic blocks, in which genes existing across species commonly accumulate, have a high proportion of orthologs. Thus, candidate gene clusters that have a high probability of secondary metabolism biosynthesis genes can be selected by referring to their localization in nonsyntenic blocks (Fig. <a reveal-id="DSU010F1" data-open="DSU010F1" class="link link-reveal link-table xref-fig">1</a>e).</p><p>Orthologous gene pairs between <em>X</em> and <em>Y</em> were aligned to create contiguous blocks until no more orthologs were identified within the threshold range of the intergenic distances in both genomes (Fig. <a reveal-id="DSU010F1" data-open="DSU010F1" class="link link-reveal link-table xref-fig">1</a>e). Contiguous blocks composed of at least two orthologs were defined as syntenic blocks (SBs). Non-orthologous genes inserted between orthologs were allowed within the threshold of an intergenic distance of 5, 10, 20, 30, 40 or 50 kb (Param5).</p><p>SBs were subsets of extended seeds, <em>X</em> and <em>Y</em>. If the percentage of the member genes in the subset segment for the number of genes in the entire extended cluster was less than the threshold, the corresponding candidate gene cluster was selected as a predicted secondary metabolism gene cluster. In this study, 10, 15, 20, 25, 30 and 35% were used as the thresholds for the SB percentage (Param6). Multiple predicted clusters overlapping each other were merged into a single cluster similarly to methods used in other SMB gene cluster prediction software.<sup><a reveal-id="DSU010C6" data-open="DSU010C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSU010C18" data-open="DSU010C18" class="link link-ref link-reveal xref-bibr">18</a></sup></p>                            <h2 scrollto-destination=87196114 id="87196114" class="section-title" data-legacy-id=s3>3. Results and discussion</h2>
                            <h3 scrollto-destination=87196115 id="87196115" class="section-title" data-legacy-id=s3a>3.1. Effect of each parameter on the prediction</h3>
<div class="block-child-p">To detect SMB gene cluster candidates, the genome sequences of 10 species of filamentous fungi, including <em>A. oryzae</em> (see Materials and methods), were subjected to a comprehensive pairwise comparison, with the exception of between identical genomes. We first detected known SMB gene clusters from the genomes of <em>A. flavus</em> and <em>A. fumigatus</em> to optimize the parameters of our method. The clusters that the method predicted for the biosynthesis of aflatoxin and gliotoxin from <em>A. flavus</em> and for the biosynthesis of ergot, epipolythiodioxopiperazine-type toxin (ETP), fumitremorgin, gliotoxin, melanin, Pes1, pseurotin and siderophore from <em>A. fumigatus</em>, are listed in Table <a reveal-id="DSU010TB1" data-open="DSU010TB1" class="link link-reveal link-table xref-fig">1</a> and were subjected to the analysis of differences in boundary positions when compared with those from the experimentally confirmed clusters. Absolute values of the differences in gene numbers at the upstream and the downstream boundaries were summed to generate a value defined as the prediction error. The minimum error was obtained from all of the clusters predicted for each gene cluster, and the average of the minimum errors for the 10 gene clusters from <em>A. flavus</em> and <em>A. fumigatus</em> described above was then calculated at each value for the parameters. As shown in Fig. <a reveal-id="DSU010F2" data-open="DSU010F2" class="link link-reveal link-table xref-fig">2</a>, a combination including Param1 = e−10, Param2 = −0.2 and Param4 = −0.3 gave the smallest errors for the prediction of the cluster boundaries. Param3 (extension length), Param5 (intergenic distance) and Param6 (permissible ratio of syntenic blocks) had little influence on the prediction of gene clusters within the range used in this study. Consequently, Param3 = 35 genes, Param5 = 10 kb and Param6 = 25% were used. <div content-id="DSU010TB1" class="table-modal"><div class="table-wrap"><div id="DSU010TB1" data-id="DSU010TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Known gene clusters predicted using this method</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2">Predicted gene cluster<hr /> </th><th rowspan="2">Product </th><th rowspan="2">Verified cluster size (genes) </th><th rowspan="2">Species </th><th rowspan="2">vs. Species (top hit)<sup>a</sup> </th><th rowspan="2">Number of hits<sup>b</sup> </th><th colspan="2">Min difference<sup>c</sup><hr /> </th><th rowspan="2">Min error<sup>d</sup> </th><th rowspan="2">Error<sup>d</sup> at the max CB valve </th><th rowspan="2">Max error<sup>d</sup> </th></tr><tr><th>Begin </th><th>End </th><th>Up </th><th>Down </th></tr></thead><tbody><tr><td>AFLA_139060 </td><td>AFLA_139460 </td><td>Aflatoxin </td><td>29 </td><td><em>A. flavus</em> </td><td><em>M. grisea</em> </td><td>9 </td><td>9 </td><td>2 </td><td>11 </td><td>16 </td><td>25 </td></tr><tr><td>AFLA_064360 </td><td>AFLA_064590 </td><td>Gliotoxin </td><td>33 </td><td><em>A. flavus</em> </td><td><em>A. fumigatus</em> </td><td>8 </td><td>−3 </td><td>−6 </td><td>9 </td><td>9 </td><td>25 </td></tr><tr><td>AO090113000131 </td><td>AO090113000147 </td><td>Kojic acid </td><td>3 </td><td><em>A. oryzae</em> </td><td><em>A. flavus</em> </td><td>1 </td><td>4 </td><td>9 </td><td>13 </td><td>13 </td><td>13 </td></tr><tr><td>ANID_01036 </td><td>ANID_01029 </td><td>Asperfuranone </td><td>8 </td><td><em>A. nidulans</em> </td><td><em>A. terreus</em> </td><td>8 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>2 </td></tr><tr><td>– </td><td>– </td><td>Asperthecin </td><td>3 </td><td><em>A. nidulans</em> </td><td>– </td><td>0 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>ANID_02625 </td><td>ANID_02624 </td><td>Penicillin </td><td>6 </td><td><em>A. nidulans</em> </td><td><em>A. terreus</em> </td><td>1 </td><td>0 </td><td>−3 </td><td>3 </td><td>3 </td><td>3 </td></tr><tr><td>ANID_07805 </td><td>ANID_07825 </td><td>Sterigmatocystin </td><td>25 </td><td><em>A. nidulans</em> </td><td><em>A. terreus</em> </td><td>6 </td><td>−1 </td><td>0 </td><td>1 </td><td>1 </td><td>34 </td></tr><tr><td>ANID_08517 </td><td>ANID_08524 </td><td>Terrequinone </td><td>7 </td><td><em>A. nidulans</em> </td><td><em>F. graminearum</em> </td><td>4 </td><td>−4 </td><td>5 </td><td>9 </td><td>14 </td><td>15 </td></tr><tr><td>Afu2g17960 </td><td>Afu2g18040 </td><td>Ergot </td><td>11 </td><td><em>A. fumigatus</em> </td><td><em>A. terreus</em> </td><td>1 </td><td>0 </td><td>−2 </td><td>2 </td><td>2 </td><td>2 </td></tr><tr><td>Afu3g12890 </td><td>Afu3g12960 </td><td>ETP<sup>e</sup> </td><td>8 </td><td><em>A. fumigatus</em> </td><td><em>A. nidulans</em> </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>4 </td></tr><tr><td>Afu8g00170 </td><td>Afu8g00260 </td><td>Fumitremorgin </td><td>10 </td><td><em>A. fumigatus</em> </td><td><em>A. oryzae</em> </td><td>6 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>5 </td></tr><tr><td>Afu6g09610 </td><td>Afu6g09740 </td><td>Gliotoxin </td><td>12 </td><td><em>A. fumigatus</em> </td><td><em>F. oxysporum</em> </td><td>8 </td><td>2 </td><td>0 </td><td>2 </td><td>4 </td><td>12 </td></tr><tr><td>Afu2g17490 </td><td>Afu2g17610 </td><td>Melanin </td><td>8 </td><td><em>A. fumigatus</em> </td><td><em>F. graminearum</em> </td><td>4 </td><td>4 </td><td>1 </td><td>5 </td><td>5 </td><td>11 </td></tr><tr><td>– </td><td>– </td><td>Pes1 </td><td>2 </td><td><em>A. fumigatus</em> </td><td>– </td><td>0 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>Afu8g00450 </td><td>Afu8g00580 </td><td>Pseurotin </td><td>6 </td><td><em>A. fumigatus</em> </td><td><em>F. verticillioides</em> </td><td>6 </td><td>8 </td><td>0 </td><td>8 </td><td>19 </td><td>19 </td></tr><tr><td>Afu3g03350 </td><td>Afu3g03480 </td><td>Siderophore </td><td>13 </td><td><em>A. fumigatus</em> </td><td><em>F. graminearum</em> </td><td>9 </td><td>0 </td><td>1 </td><td>1 </td><td>2 </td><td>13 </td></tr><tr><td>ATEG_09957 </td><td>ATEG_09977 </td><td>Lovastatin </td><td>17 </td><td><em>A. terreus</em> </td><td><em>A. oryzae</em> </td><td>9 </td><td>1 </td><td>3 </td><td>4 </td><td>8 </td><td>9 </td></tr><tr><td>FGSG_02322 </td><td>FGSG_02330 </td><td>Aurofusarin </td><td>11 </td><td><em>F. graminearum</em> </td><td><em>A. terreus</em> </td><td>7 </td><td>−2 </td><td>0 </td><td>2 </td><td>5 </td><td>7 </td></tr><tr><td>FGSG_02392 </td><td>FGSG_02400 </td><td>Zearalenone </td><td>5 </td><td><em>F. graminearum</em> </td><td><em>A. terreus</em> </td><td>2 </td><td>3 </td><td>−3 </td><td>6 </td><td>7 </td><td>7 </td></tr><tr><td>FVEG_03384 </td><td>FVEG_03379 </td><td>Bikaverin </td><td>6 </td><td><em>F. verticillioides</em> </td><td><em>C. globosum</em> </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>5 </td><td>9 </td></tr><tr><td>FVEG_00329 </td><td>FVEG_00316 </td><td>Fumonisin </td><td>16 </td><td><em>F. verticillioides</em> </td><td><em>A. fumigatus</em> </td><td>4 </td><td>0 </td><td>−2 </td><td>2 </td><td>2 </td><td>9 </td></tr><tr><td>– </td><td>– </td><td>Fusaric acid </td><td>5 </td><td><em>F. verticillioides</em> </td><td>– </td><td>0 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>FVEG_11079 </td><td>FVEG_11086 </td><td>Fusarin </td><td>9 </td><td><em>F. verticillioides</em> </td><td><em>M. grisea</em> </td><td>9 </td><td>−1 </td><td>0 </td><td>1 </td><td>3 </td><td>4 </td></tr><tr><td>FVEG_03698 </td><td>FVEG_03695 </td><td>Perithecium pigment </td><td>6 </td><td><em>F. verticillioides</em> </td><td><em>A. flavus</em> </td><td>4 </td><td>−2 </td><td>0 </td><td>2 </td><td>2 </td><td>3 </td></tr></tbody></table></div><div content-id="DSU010TB1" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2">Predicted gene cluster<hr /> </th><th rowspan="2">Product </th><th rowspan="2">Verified cluster size (genes) </th><th rowspan="2">Species </th><th rowspan="2">vs. Species (top hit)<sup>a</sup> </th><th rowspan="2">Number of hits<sup>b</sup> </th><th colspan="2">Min difference<sup>c</sup><hr /> </th><th rowspan="2">Min error<sup>d</sup> </th><th rowspan="2">Error<sup>d</sup> at the max CB valve </th><th rowspan="2">Max error<sup>d</sup> </th></tr><tr><th>Begin </th><th>End </th><th>Up </th><th>Down </th></tr></thead><tbody><tr><td>AFLA_139060 </td><td>AFLA_139460 </td><td>Aflatoxin </td><td>29 </td><td><em>A. flavus</em> </td><td><em>M. grisea</em> </td><td>9 </td><td>9 </td><td>2 </td><td>11 </td><td>16 </td><td>25 </td></tr><tr><td>AFLA_064360 </td><td>AFLA_064590 </td><td>Gliotoxin </td><td>33 </td><td><em>A. flavus</em> </td><td><em>A. fumigatus</em> </td><td>8 </td><td>−3 </td><td>−6 </td><td>9 </td><td>9 </td><td>25 </td></tr><tr><td>AO090113000131 </td><td>AO090113000147 </td><td>Kojic acid </td><td>3 </td><td><em>A. oryzae</em> </td><td><em>A. flavus</em> </td><td>1 </td><td>4 </td><td>9 </td><td>13 </td><td>13 </td><td>13 </td></tr><tr><td>ANID_01036 </td><td>ANID_01029 </td><td>Asperfuranone </td><td>8 </td><td><em>A. nidulans</em> </td><td><em>A. terreus</em> </td><td>8 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>2 </td></tr><tr><td>– </td><td>– </td><td>Asperthecin </td><td>3 </td><td><em>A. nidulans</em> </td><td>– </td><td>0 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>ANID_02625 </td><td>ANID_02624 </td><td>Penicillin </td><td>6 </td><td><em>A. nidulans</em> </td><td><em>A. terreus</em> </td><td>1 </td><td>0 </td><td>−3 </td><td>3 </td><td>3 </td><td>3 </td></tr><tr><td>ANID_07805 </td><td>ANID_07825 </td><td>Sterigmatocystin </td><td>25 </td><td><em>A. nidulans</em> </td><td><em>A. terreus</em> </td><td>6 </td><td>−1 </td><td>0 </td><td>1 </td><td>1 </td><td>34 </td></tr><tr><td>ANID_08517 </td><td>ANID_08524 </td><td>Terrequinone </td><td>7 </td><td><em>A. nidulans</em> </td><td><em>F. graminearum</em> </td><td>4 </td><td>−4 </td><td>5 </td><td>9 </td><td>14 </td><td>15 </td></tr><tr><td>Afu2g17960 </td><td>Afu2g18040 </td><td>Ergot </td><td>11 </td><td><em>A. fumigatus</em> </td><td><em>A. terreus</em> </td><td>1 </td><td>0 </td><td>−2 </td><td>2 </td><td>2 </td><td>2 </td></tr><tr><td>Afu3g12890 </td><td>Afu3g12960 </td><td>ETP<sup>e</sup> </td><td>8 </td><td><em>A. fumigatus</em> </td><td><em>A. nidulans</em> </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>4 </td></tr><tr><td>Afu8g00170 </td><td>Afu8g00260 </td><td>Fumitremorgin </td><td>10 </td><td><em>A. fumigatus</em> </td><td><em>A. oryzae</em> </td><td>6 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>5 </td></tr><tr><td>Afu6g09610 </td><td>Afu6g09740 </td><td>Gliotoxin </td><td>12 </td><td><em>A. fumigatus</em> </td><td><em>F. oxysporum</em> </td><td>8 </td><td>2 </td><td>0 </td><td>2 </td><td>4 </td><td>12 </td></tr><tr><td>Afu2g17490 </td><td>Afu2g17610 </td><td>Melanin </td><td>8 </td><td><em>A. fumigatus</em> </td><td><em>F. graminearum</em> </td><td>4 </td><td>4 </td><td>1 </td><td>5 </td><td>5 </td><td>11 </td></tr><tr><td>– </td><td>– </td><td>Pes1 </td><td>2 </td><td><em>A. fumigatus</em> </td><td>– </td><td>0 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>Afu8g00450 </td><td>Afu8g00580 </td><td>Pseurotin </td><td>6 </td><td><em>A. fumigatus</em> </td><td><em>F. verticillioides</em> </td><td>6 </td><td>8 </td><td>0 </td><td>8 </td><td>19 </td><td>19 </td></tr><tr><td>Afu3g03350 </td><td>Afu3g03480 </td><td>Siderophore </td><td>13 </td><td><em>A. fumigatus</em> </td><td><em>F. graminearum</em> </td><td>9 </td><td>0 </td><td>1 </td><td>1 </td><td>2 </td><td>13 </td></tr><tr><td>ATEG_09957 </td><td>ATEG_09977 </td><td>Lovastatin </td><td>17 </td><td><em>A. terreus</em> </td><td><em>A. oryzae</em> </td><td>9 </td><td>1 </td><td>3 </td><td>4 </td><td>8 </td><td>9 </td></tr><tr><td>FGSG_02322 </td><td>FGSG_02330 </td><td>Aurofusarin </td><td>11 </td><td><em>F. graminearum</em> </td><td><em>A. terreus</em> </td><td>7 </td><td>−2 </td><td>0 </td><td>2 </td><td>5 </td><td>7 </td></tr><tr><td>FGSG_02392 </td><td>FGSG_02400 </td><td>Zearalenone </td><td>5 </td><td><em>F. graminearum</em> </td><td><em>A. terreus</em> </td><td>2 </td><td>3 </td><td>−3 </td><td>6 </td><td>7 </td><td>7 </td></tr><tr><td>FVEG_03384 </td><td>FVEG_03379 </td><td>Bikaverin </td><td>6 </td><td><em>F. verticillioides</em> </td><td><em>C. globosum</em> </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>5 </td><td>9 </td></tr><tr><td>FVEG_00329 </td><td>FVEG_00316 </td><td>Fumonisin </td><td>16 </td><td><em>F. verticillioides</em> </td><td><em>A. fumigatus</em> </td><td>4 </td><td>0 </td><td>−2 </td><td>2 </td><td>2 </td><td>9 </td></tr><tr><td>– </td><td>– </td><td>Fusaric acid </td><td>5 </td><td><em>F. verticillioides</em> </td><td>– </td><td>0 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>FVEG_11079 </td><td>FVEG_11086 </td><td>Fusarin </td><td>9 </td><td><em>F. verticillioides</em> </td><td><em>M. grisea</em> </td><td>9 </td><td>−1 </td><td>0 </td><td>1 </td><td>3 </td><td>4 </td></tr><tr><td>FVEG_03698 </td><td>FVEG_03695 </td><td>Perithecium pigment </td><td>6 </td><td><em>F. verticillioides</em> </td><td><em>A. flavus</em> </td><td>4 </td><td>−2 </td><td>0 </td><td>2 </td><td>2 </td><td>3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p><sup>a</sup>Species used for comparison when the gene cluster was detected with minimum error.</p></div></div><div content-id="tbnfn2"><div class="fn"><p><sup>b</sup>Number of predicted clusters in a comprehensive pairwise comparison of the 10 genomes.</p></div></div><div content-id="tbnfn3"><div class="fn"><p><sup>c</sup>Difference in the numbers of genes upstream and downstream of the predicted gene cluster compared with the experimentally characterized cluster. The minus and plus quantities indicate under- and over-predictions, respectively.</p></div></div><div content-id="tbnfn4"><div class="fn"><p><sup>d</sup>Error is defined as the sum of absolute values of the differences in gene number at both ends of a predicted gene cluster.</p></div></div><div content-id="tbnfn5"><div class="fn"><p><sup>e</sup>Epipolythiodioxopiperazine-type toxin.</p></div></div></div></div></div><div class="table-wrap"><div id="DSU010TB1" data-id="DSU010TB1" class="table-wrap-title"><div class="label -label">Table 1.</div><div class="caption"><p>Known gene clusters predicted using this method</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th colspan="2">Predicted gene cluster<hr /> </th><th rowspan="2">Product </th><th rowspan="2">Verified cluster size (genes) </th><th rowspan="2">Species </th><th rowspan="2">vs. Species (top hit)<sup>a</sup> </th><th rowspan="2">Number of hits<sup>b</sup> </th><th colspan="2">Min difference<sup>c</sup><hr /> </th><th rowspan="2">Min error<sup>d</sup> </th><th rowspan="2">Error<sup>d</sup> at the max CB valve </th><th rowspan="2">Max error<sup>d</sup> </th></tr><tr><th>Begin </th><th>End </th><th>Up </th><th>Down </th></tr></thead><tbody><tr><td>AFLA_139060 </td><td>AFLA_139460 </td><td>Aflatoxin </td><td>29 </td><td><em>A. flavus</em> </td><td><em>M. grisea</em> </td><td>9 </td><td>9 </td><td>2 </td><td>11 </td><td>16 </td><td>25 </td></tr><tr><td>AFLA_064360 </td><td>AFLA_064590 </td><td>Gliotoxin </td><td>33 </td><td><em>A. flavus</em> </td><td><em>A. fumigatus</em> </td><td>8 </td><td>−3 </td><td>−6 </td><td>9 </td><td>9 </td><td>25 </td></tr><tr><td>AO090113000131 </td><td>AO090113000147 </td><td>Kojic acid </td><td>3 </td><td><em>A. oryzae</em> </td><td><em>A. flavus</em> </td><td>1 </td><td>4 </td><td>9 </td><td>13 </td><td>13 </td><td>13 </td></tr><tr><td>ANID_01036 </td><td>ANID_01029 </td><td>Asperfuranone </td><td>8 </td><td><em>A. nidulans</em> </td><td><em>A. terreus</em> </td><td>8 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>2 </td></tr><tr><td>– </td><td>– </td><td>Asperthecin </td><td>3 </td><td><em>A. nidulans</em> </td><td>– </td><td>0 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>ANID_02625 </td><td>ANID_02624 </td><td>Penicillin </td><td>6 </td><td><em>A. nidulans</em> </td><td><em>A. terreus</em> </td><td>1 </td><td>0 </td><td>−3 </td><td>3 </td><td>3 </td><td>3 </td></tr><tr><td>ANID_07805 </td><td>ANID_07825 </td><td>Sterigmatocystin </td><td>25 </td><td><em>A. nidulans</em> </td><td><em>A. terreus</em> </td><td>6 </td><td>−1 </td><td>0 </td><td>1 </td><td>1 </td><td>34 </td></tr><tr><td>ANID_08517 </td><td>ANID_08524 </td><td>Terrequinone </td><td>7 </td><td><em>A. nidulans</em> </td><td><em>F. graminearum</em> </td><td>4 </td><td>−4 </td><td>5 </td><td>9 </td><td>14 </td><td>15 </td></tr><tr><td>Afu2g17960 </td><td>Afu2g18040 </td><td>Ergot </td><td>11 </td><td><em>A. fumigatus</em> </td><td><em>A. terreus</em> </td><td>1 </td><td>0 </td><td>−2 </td><td>2 </td><td>2 </td><td>2 </td></tr><tr><td>Afu3g12890 </td><td>Afu3g12960 </td><td>ETP<sup>e</sup> </td><td>8 </td><td><em>A. fumigatus</em> </td><td><em>A. nidulans</em> </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>4 </td></tr><tr><td>Afu8g00170 </td><td>Afu8g00260 </td><td>Fumitremorgin </td><td>10 </td><td><em>A. fumigatus</em> </td><td><em>A. oryzae</em> </td><td>6 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>5 </td></tr><tr><td>Afu6g09610 </td><td>Afu6g09740 </td><td>Gliotoxin </td><td>12 </td><td><em>A. fumigatus</em> </td><td><em>F. oxysporum</em> </td><td>8 </td><td>2 </td><td>0 </td><td>2 </td><td>4 </td><td>12 </td></tr><tr><td>Afu2g17490 </td><td>Afu2g17610 </td><td>Melanin </td><td>8 </td><td><em>A. fumigatus</em> </td><td><em>F. graminearum</em> </td><td>4 </td><td>4 </td><td>1 </td><td>5 </td><td>5 </td><td>11 </td></tr><tr><td>– </td><td>– </td><td>Pes1 </td><td>2 </td><td><em>A. fumigatus</em> </td><td>– </td><td>0 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>Afu8g00450 </td><td>Afu8g00580 </td><td>Pseurotin </td><td>6 </td><td><em>A. fumigatus</em> </td><td><em>F. verticillioides</em> </td><td>6 </td><td>8 </td><td>0 </td><td>8 </td><td>19 </td><td>19 </td></tr><tr><td>Afu3g03350 </td><td>Afu3g03480 </td><td>Siderophore </td><td>13 </td><td><em>A. fumigatus</em> </td><td><em>F. graminearum</em> </td><td>9 </td><td>0 </td><td>1 </td><td>1 </td><td>2 </td><td>13 </td></tr><tr><td>ATEG_09957 </td><td>ATEG_09977 </td><td>Lovastatin </td><td>17 </td><td><em>A. terreus</em> </td><td><em>A. oryzae</em> </td><td>9 </td><td>1 </td><td>3 </td><td>4 </td><td>8 </td><td>9 </td></tr><tr><td>FGSG_02322 </td><td>FGSG_02330 </td><td>Aurofusarin </td><td>11 </td><td><em>F. graminearum</em> </td><td><em>A. terreus</em> </td><td>7 </td><td>−2 </td><td>0 </td><td>2 </td><td>5 </td><td>7 </td></tr><tr><td>FGSG_02392 </td><td>FGSG_02400 </td><td>Zearalenone </td><td>5 </td><td><em>F. graminearum</em> </td><td><em>A. terreus</em> </td><td>2 </td><td>3 </td><td>−3 </td><td>6 </td><td>7 </td><td>7 </td></tr><tr><td>FVEG_03384 </td><td>FVEG_03379 </td><td>Bikaverin </td><td>6 </td><td><em>F. verticillioides</em> </td><td><em>C. globosum</em> </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>5 </td><td>9 </td></tr><tr><td>FVEG_00329 </td><td>FVEG_00316 </td><td>Fumonisin </td><td>16 </td><td><em>F. verticillioides</em> </td><td><em>A. fumigatus</em> </td><td>4 </td><td>0 </td><td>−2 </td><td>2 </td><td>2 </td><td>9 </td></tr><tr><td>– </td><td>– </td><td>Fusaric acid </td><td>5 </td><td><em>F. verticillioides</em> </td><td>– </td><td>0 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>FVEG_11079 </td><td>FVEG_11086 </td><td>Fusarin </td><td>9 </td><td><em>F. verticillioides</em> </td><td><em>M. grisea</em> </td><td>9 </td><td>−1 </td><td>0 </td><td>1 </td><td>3 </td><td>4 </td></tr><tr><td>FVEG_03698 </td><td>FVEG_03695 </td><td>Perithecium pigment </td><td>6 </td><td><em>F. verticillioides</em> </td><td><em>A. flavus</em> </td><td>4 </td><td>−2 </td><td>0 </td><td>2 </td><td>2 </td><td>3 </td></tr></tbody></table></div><div content-id="DSU010TB1" class="table-modal"><table><thead name="thead" align=""><tr><th colspan="2">Predicted gene cluster<hr /> </th><th rowspan="2">Product </th><th rowspan="2">Verified cluster size (genes) </th><th rowspan="2">Species </th><th rowspan="2">vs. Species (top hit)<sup>a</sup> </th><th rowspan="2">Number of hits<sup>b</sup> </th><th colspan="2">Min difference<sup>c</sup><hr /> </th><th rowspan="2">Min error<sup>d</sup> </th><th rowspan="2">Error<sup>d</sup> at the max CB valve </th><th rowspan="2">Max error<sup>d</sup> </th></tr><tr><th>Begin </th><th>End </th><th>Up </th><th>Down </th></tr></thead><tbody><tr><td>AFLA_139060 </td><td>AFLA_139460 </td><td>Aflatoxin </td><td>29 </td><td><em>A. flavus</em> </td><td><em>M. grisea</em> </td><td>9 </td><td>9 </td><td>2 </td><td>11 </td><td>16 </td><td>25 </td></tr><tr><td>AFLA_064360 </td><td>AFLA_064590 </td><td>Gliotoxin </td><td>33 </td><td><em>A. flavus</em> </td><td><em>A. fumigatus</em> </td><td>8 </td><td>−3 </td><td>−6 </td><td>9 </td><td>9 </td><td>25 </td></tr><tr><td>AO090113000131 </td><td>AO090113000147 </td><td>Kojic acid </td><td>3 </td><td><em>A. oryzae</em> </td><td><em>A. flavus</em> </td><td>1 </td><td>4 </td><td>9 </td><td>13 </td><td>13 </td><td>13 </td></tr><tr><td>ANID_01036 </td><td>ANID_01029 </td><td>Asperfuranone </td><td>8 </td><td><em>A. nidulans</em> </td><td><em>A. terreus</em> </td><td>8 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>2 </td></tr><tr><td>– </td><td>– </td><td>Asperthecin </td><td>3 </td><td><em>A. nidulans</em> </td><td>– </td><td>0 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>ANID_02625 </td><td>ANID_02624 </td><td>Penicillin </td><td>6 </td><td><em>A. nidulans</em> </td><td><em>A. terreus</em> </td><td>1 </td><td>0 </td><td>−3 </td><td>3 </td><td>3 </td><td>3 </td></tr><tr><td>ANID_07805 </td><td>ANID_07825 </td><td>Sterigmatocystin </td><td>25 </td><td><em>A. nidulans</em> </td><td><em>A. terreus</em> </td><td>6 </td><td>−1 </td><td>0 </td><td>1 </td><td>1 </td><td>34 </td></tr><tr><td>ANID_08517 </td><td>ANID_08524 </td><td>Terrequinone </td><td>7 </td><td><em>A. nidulans</em> </td><td><em>F. graminearum</em> </td><td>4 </td><td>−4 </td><td>5 </td><td>9 </td><td>14 </td><td>15 </td></tr><tr><td>Afu2g17960 </td><td>Afu2g18040 </td><td>Ergot </td><td>11 </td><td><em>A. fumigatus</em> </td><td><em>A. terreus</em> </td><td>1 </td><td>0 </td><td>−2 </td><td>2 </td><td>2 </td><td>2 </td></tr><tr><td>Afu3g12890 </td><td>Afu3g12960 </td><td>ETP<sup>e</sup> </td><td>8 </td><td><em>A. fumigatus</em> </td><td><em>A. nidulans</em> </td><td>4 </td><td>0 </td><td>0 </td><td>0 </td><td>0 </td><td>4 </td></tr><tr><td>Afu8g00170 </td><td>Afu8g00260 </td><td>Fumitremorgin </td><td>10 </td><td><em>A. fumigatus</em> </td><td><em>A. oryzae</em> </td><td>6 </td><td>0 </td><td>0 </td><td>0 </td><td>1 </td><td>5 </td></tr><tr><td>Afu6g09610 </td><td>Afu6g09740 </td><td>Gliotoxin </td><td>12 </td><td><em>A. fumigatus</em> </td><td><em>F. oxysporum</em> </td><td>8 </td><td>2 </td><td>0 </td><td>2 </td><td>4 </td><td>12 </td></tr><tr><td>Afu2g17490 </td><td>Afu2g17610 </td><td>Melanin </td><td>8 </td><td><em>A. fumigatus</em> </td><td><em>F. graminearum</em> </td><td>4 </td><td>4 </td><td>1 </td><td>5 </td><td>5 </td><td>11 </td></tr><tr><td>– </td><td>– </td><td>Pes1 </td><td>2 </td><td><em>A. fumigatus</em> </td><td>– </td><td>0 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>Afu8g00450 </td><td>Afu8g00580 </td><td>Pseurotin </td><td>6 </td><td><em>A. fumigatus</em> </td><td><em>F. verticillioides</em> </td><td>6 </td><td>8 </td><td>0 </td><td>8 </td><td>19 </td><td>19 </td></tr><tr><td>Afu3g03350 </td><td>Afu3g03480 </td><td>Siderophore </td><td>13 </td><td><em>A. fumigatus</em> </td><td><em>F. graminearum</em> </td><td>9 </td><td>0 </td><td>1 </td><td>1 </td><td>2 </td><td>13 </td></tr><tr><td>ATEG_09957 </td><td>ATEG_09977 </td><td>Lovastatin </td><td>17 </td><td><em>A. terreus</em> </td><td><em>A. oryzae</em> </td><td>9 </td><td>1 </td><td>3 </td><td>4 </td><td>8 </td><td>9 </td></tr><tr><td>FGSG_02322 </td><td>FGSG_02330 </td><td>Aurofusarin </td><td>11 </td><td><em>F. graminearum</em> </td><td><em>A. terreus</em> </td><td>7 </td><td>−2 </td><td>0 </td><td>2 </td><td>5 </td><td>7 </td></tr><tr><td>FGSG_02392 </td><td>FGSG_02400 </td><td>Zearalenone </td><td>5 </td><td><em>F. graminearum</em> </td><td><em>A. terreus</em> </td><td>2 </td><td>3 </td><td>−3 </td><td>6 </td><td>7 </td><td>7 </td></tr><tr><td>FVEG_03384 </td><td>FVEG_03379 </td><td>Bikaverin </td><td>6 </td><td><em>F. verticillioides</em> </td><td><em>C. globosum</em> </td><td>3 </td><td>0 </td><td>0 </td><td>0 </td><td>5 </td><td>9 </td></tr><tr><td>FVEG_00329 </td><td>FVEG_00316 </td><td>Fumonisin </td><td>16 </td><td><em>F. verticillioides</em> </td><td><em>A. fumigatus</em> </td><td>4 </td><td>0 </td><td>−2 </td><td>2 </td><td>2 </td><td>9 </td></tr><tr><td>– </td><td>– </td><td>Fusaric acid </td><td>5 </td><td><em>F. verticillioides</em> </td><td>– </td><td>0 </td><td>– </td><td>– </td><td>– </td><td>– </td><td>– </td></tr><tr><td>FVEG_11079 </td><td>FVEG_11086 </td><td>Fusarin </td><td>9 </td><td><em>F. verticillioides</em> </td><td><em>M. grisea</em> </td><td>9 </td><td>−1 </td><td>0 </td><td>1 </td><td>3 </td><td>4 </td></tr><tr><td>FVEG_03698 </td><td>FVEG_03695 </td><td>Perithecium pigment </td><td>6 </td><td><em>F. verticillioides</em> </td><td><em>A. flavus</em> </td><td>4 </td><td>−2 </td><td>0 </td><td>2 </td><td>2 </td><td>3 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn1"><div class="fn"><p><sup>a</sup>Species used for comparison when the gene cluster was detected with minimum error.</p></div></div><div content-id="tbnfn2"><div class="fn"><p><sup>b</sup>Number of predicted clusters in a comprehensive pairwise comparison of the 10 genomes.</p></div></div><div content-id="tbnfn3"><div class="fn"><p><sup>c</sup>Difference in the numbers of genes upstream and downstream of the predicted gene cluster compared with the experimentally characterized cluster. The minus and plus quantities indicate under- and over-predictions, respectively.</p></div></div><div content-id="tbnfn4"><div class="fn"><p><sup>d</sup>Error is defined as the sum of absolute values of the differences in gene number at both ends of a predicted gene cluster.</p></div></div><div content-id="tbnfn5"><div class="fn"><p><sup>e</sup>Epipolythiodioxopiperazine-type toxin.</p></div></div></div></div><div data-id="DSU010F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/m_dsu01002.jpeg?Expires=2147483647&Signature=W6HPGP2vXWoec23actJT3GJaKuehvGeXU1j8qFH-rirb6ymEPyC2S6CI1Uh9iqFaSJCdVcUcC5V0iLQ8HZ6O02jNmd9ph~yfhMjsLL9O5WF9BEHSMy~wnFtGPBd0HgO6BzmYzkvyLuXDXPLS6HeDpYOS~7dIooN-PIr4Rcr85nNMqM3NyseguC5ZfgyYHcwM8NhYnQU-KyrCCLmC4BMnORGmwvlpIHnnsyNYAxOcQeHAITaZhlAXHB~sH4qeBGi2bp7khJ68uyWM6jMzGHSiIAYOKyRDWjhfrupvp4E27arh0R1EODp87Id1ngHh-BdmIRsa8dtuaqP0WUO-YVuzIw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of the average prediction errors. Averages of the minimum error for predicting the known gene clusters (solid line) of A. flavus and A. fumigatus are shown together with the total numbers of genes in the predicted gene clusters of A. flavus (dotted line) and A. fumigatus (broken line). The default values, except for the parameter indicated in each panel, were the same as those used in Fig. 3." path-from-xml="dsu01002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196116/dsu01002.jpeg" path-from-xml="dsu01002.jpeg" target="_blank">View large</a><a section="87196116" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/dsu01002.jpeg?Expires=2147483647&Signature=NJ~GNDTIhi03NCA3Q6A5zBIbcd5eoqRfEBPfakT0s-Lbf41cJT4djzYu~l~NLpr9HxchcxUlAcd563fsKCrNa2ntG67yoW9FngKfeV4BxvvAQPt5x16pvgpP4Bz2546DzcmGKVZTPU89PfRThUsE4w8eNYFEcZ-dKGrEymQnbdughvIVgbXcisTMJyICfvkfMPlfbu0eDocC92VCboo4DgozR75AaT-93BY78GNmtbSWjWWsP4szdx9FFr3GRsIc2m4D34ciGJ7R41c0DVQ3iRdz398EaQgP6cQfbfNolY~Icl9~AJY60ISTso36tpnNlISewfRjGamHOJtaaCktSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196116&ar=2754535&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of the average prediction errors. Averages of the minimum error for predicting the known gene clusters (solid line) of <em>A. flavus</em> and <em>A. fumigatus</em> are shown together with the total numbers of genes in the predicted gene clusters of <em>A. flavus</em> (dotted line) and <em>A. fumigatus</em> (broken line). The default values, except for the parameter indicated in each panel, were the same as those used in Fig. <a reveal-id="DSU010F3" data-open="DSU010F3" class="link link-reveal link-table xref-fig">3</a>.</p></div></div><div content-id="DSU010F2" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/m_dsu01002.jpeg?Expires=2147483647&Signature=W6HPGP2vXWoec23actJT3GJaKuehvGeXU1j8qFH-rirb6ymEPyC2S6CI1Uh9iqFaSJCdVcUcC5V0iLQ8HZ6O02jNmd9ph~yfhMjsLL9O5WF9BEHSMy~wnFtGPBd0HgO6BzmYzkvyLuXDXPLS6HeDpYOS~7dIooN-PIr4Rcr85nNMqM3NyseguC5ZfgyYHcwM8NhYnQU-KyrCCLmC4BMnORGmwvlpIHnnsyNYAxOcQeHAITaZhlAXHB~sH4qeBGi2bp7khJ68uyWM6jMzGHSiIAYOKyRDWjhfrupvp4E27arh0R1EODp87Id1ngHh-BdmIRsa8dtuaqP0WUO-YVuzIw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Analysis of the average prediction errors. Averages of the minimum error for predicting the known gene clusters (solid line) of A. flavus and A. fumigatus are shown together with the total numbers of genes in the predicted gene clusters of A. flavus (dotted line) and A. fumigatus (broken line). The default values, except for the parameter indicated in each panel, were the same as those used in Fig. 3." path-from-xml="dsu01002.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196116/dsu01002.jpeg" path-from-xml="dsu01002.jpeg" target="_blank">View large</a><a section="87196116" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/dsu01002.jpeg?Expires=2147483647&Signature=NJ~GNDTIhi03NCA3Q6A5zBIbcd5eoqRfEBPfakT0s-Lbf41cJT4djzYu~l~NLpr9HxchcxUlAcd563fsKCrNa2ntG67yoW9FngKfeV4BxvvAQPt5x16pvgpP4Bz2546DzcmGKVZTPU89PfRThUsE4w8eNYFEcZ-dKGrEymQnbdughvIVgbXcisTMJyICfvkfMPlfbu0eDocC92VCboo4DgozR75AaT-93BY78GNmtbSWjWWsP4szdx9FFr3GRsIc2m4D34ciGJ7R41c0DVQ3iRdz398EaQgP6cQfbfNolY~Icl9~AJY60ISTso36tpnNlISewfRjGamHOJtaaCktSg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196116&ar=2754535&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01002.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Analysis of the average prediction errors. Averages of the minimum error for predicting the known gene clusters (solid line) of <em>A. flavus</em> and <em>A. fumigatus</em> are shown together with the total numbers of genes in the predicted gene clusters of <em>A. flavus</em> (dotted line) and <em>A. fumigatus</em> (broken line). The default values, except for the parameter indicated in each panel, were the same as those used in Fig. <a reveal-id="DSU010F3" data-open="DSU010F3" class="link link-reveal link-table xref-fig">3</a>.</p></div></div></div><div class="block-child-p">To evaluate the performance of our method, we detected known SMB gene clusters using the genomes of 10 filamentous fungal species. Of the 24 gene clusters that have been identified to date, together with their corresponding products (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/dsu010_Supplementary_Data.zip?Expires=2147483647&amp;Signature=MPiG6gc9hUdygTRnGANiyQgA7d8IpwlJMFM~v9-F7bK2pcJoH7ZgE0EsTi3yY56pjA-paZpYbdY18FI7rR9CHSSMngi-qHrGa4vd7pQr0Mc0u5QSIf-DMyDUR0xMFMI1a2Zm~koOiqyHEthZYvNNyleD9NYlf2zn7yFaWqTCk5Mrt9TUB6W84S79etUoGJlJThjWyuKO~XtL3Eiv1ahciIwfegnGKncFUbO0nF4EhNDB1dvl6wHEdlSzllFLh0b5YSPw1U3q0h7roGA9E4JS0wHc~wDa1hOhJZGBzn-eAOeMXIXx2-rAFhqatyWRWNTGau4jy1n5U3VSdyyni1gybQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>), 21 gene clusters were successfully detected with the optimized parameters described above (Table <a reveal-id="DSU010TB1" data-open="DSU010TB1" class="link link-reveal link-table xref-fig">1</a>). The minimum and the maximum errors among all of the predicted gene clusters and the error for the cluster with the maximum CB value are also indicated. Figure <a reveal-id="DSU010F3" data-open="DSU010F3" class="link link-reveal link-table xref-fig">3</a> shows the effects of Param1, Param2 and Param4 on the number of known SMB gene clusters that were detected within the minimum error of 10 genes. The number of clusters increased by decreasing the stringency of Param1 and Param2 simply because of the increased sensitivity for seed detection. A similar increase in the detected clusters was observed in the detection performed with the <em>A. fumigatus</em> genome regardless of whether the clusters were previously known (Fig. <a reveal-id="DSU010F4" data-open="DSU010F4" class="link link-reveal link-table xref-fig">4</a>a and b). In contrast, decreasing the stringency of Param4 resulted in a decrease in the number of detected clusters (Fig. <a reveal-id="DSU010F3" data-open="DSU010F3" class="link link-reveal link-table xref-fig">3</a>c). This result was also observed for the detection of clusters with fewer member genes, i.e. &lt;10 (Fig. <a reveal-id="DSU010F4" data-open="DSU010F4" class="link link-reveal link-table xref-fig">4</a>c). Some gene clusters located within a short distance of each other in the genome were predicted as a merged single cluster of genes when a low stringency for Param4 was given. Accordingly, this low stringency led to an increase in the number of clusters with large cluster sizes (Fig. <a reveal-id="DSU010F4" data-open="DSU010F4" class="link link-reveal link-table xref-fig">4</a>c). <div data-id="DSU010F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/m_dsu01003.jpeg?Expires=2147483647&amp;Signature=FoTEEdPma68gRYEYOzG61G9zyMJGoe1wDAGN4~KRCcgwo9AxU8fHfm5i80Vpp4oKI9NxPHAU~4~c1~RDNXisLKtRl9Uapn8XsNqTQnZRevjSs0Dq6ERBO5lFEYUMlg2oLr2OH7LJDZliYeRwdiGi00VMSIIi8f~9VrOqCopgeX3zVUjYXvgLOoj0EYSxUz93GnF7myXjO4LISP4HmWYnC8lPb8Kwh3ulHjQnPEPTr1YRsNmDN8UhwrYMbI2ALYfup~l6DBpMw~2KfVLRJugO3~ePhxWiQV4VK-Fe2S6KEdMr1IRuMHAPKqemAY5j8i6~7DL05YjD~hlqKFdcqu7Ajg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Prediction of known gene clusters. The numbers of known gene clusters that were predicted within the minimum error of 10 genes were analyzed by varying three parameters, Param1, Param2 and Param4, one by one in a, b and c, respectively. The tentative default values, except for the parameter indicated in each panel, were Param1 = e−10, Param2 = −0.2, Param3 = 35 genes, Param4 = −0.3, Param5 = 10 kb and Param6 = 25%." path-from-xml="dsu01003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196117/dsu01003.jpeg" path-from-xml="dsu01003.jpeg" target="_blank">View large</a><a section="87196117" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/dsu01003.jpeg?Expires=2147483647&amp;Signature=0D4YaY5FG7U~hV28f-d0b6ho6Wxf0Hj8Va6tDrT-bHGmtXhvE3KuK-d77tSracjwsNJiMjV15ke-f9c0Mmtgt5pJrTOp684bUPvUPzN12FRYOtKOkk~jv6kf24TmcVeYz~5Q4rjqq0nv2WSOAp6BzlCiVRSWfAJrLfd6Rs4gZxKg7zzM3p7v4lghWR25PMWNsgf9OjB9E~dcpY3oIEnRjBnYw08d-jcYDi16PCUWl4vpJ8sc7TT~SmCCvmzCc-aqt9FosbDLEFRYecTaJo~iZ1e-nNEqAzuBRb7DOpUa9FmzbJHPzIiMLbLKit6pIzr~8zQyv120OceaW8Y71yUOPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196117&amp;ar=2754535&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Prediction of known gene clusters. The numbers of known gene clusters that were predicted within the minimum error of 10 genes were analyzed by varying three parameters, Param1, Param2 and Param4, one by one in a, b and c, respectively. The tentative default values, except for the parameter indicated in each panel, were Param1 = e−10, Param2 = −0.2, Param3 = 35 genes, Param4 = −0.3, Param5 = 10 kb and Param6 = 25%.</p></div></div><div content-id="DSU010F3" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/m_dsu01003.jpeg?Expires=2147483647&amp;Signature=FoTEEdPma68gRYEYOzG61G9zyMJGoe1wDAGN4~KRCcgwo9AxU8fHfm5i80Vpp4oKI9NxPHAU~4~c1~RDNXisLKtRl9Uapn8XsNqTQnZRevjSs0Dq6ERBO5lFEYUMlg2oLr2OH7LJDZliYeRwdiGi00VMSIIi8f~9VrOqCopgeX3zVUjYXvgLOoj0EYSxUz93GnF7myXjO4LISP4HmWYnC8lPb8Kwh3ulHjQnPEPTr1YRsNmDN8UhwrYMbI2ALYfup~l6DBpMw~2KfVLRJugO3~ePhxWiQV4VK-Fe2S6KEdMr1IRuMHAPKqemAY5j8i6~7DL05YjD~hlqKFdcqu7Ajg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Prediction of known gene clusters. The numbers of known gene clusters that were predicted within the minimum error of 10 genes were analyzed by varying three parameters, Param1, Param2 and Param4, one by one in a, b and c, respectively. The tentative default values, except for the parameter indicated in each panel, were Param1 = e−10, Param2 = −0.2, Param3 = 35 genes, Param4 = −0.3, Param5 = 10 kb and Param6 = 25%." path-from-xml="dsu01003.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196117/dsu01003.jpeg" path-from-xml="dsu01003.jpeg" target="_blank">View large</a><a section="87196117" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/dsu01003.jpeg?Expires=2147483647&amp;Signature=0D4YaY5FG7U~hV28f-d0b6ho6Wxf0Hj8Va6tDrT-bHGmtXhvE3KuK-d77tSracjwsNJiMjV15ke-f9c0Mmtgt5pJrTOp684bUPvUPzN12FRYOtKOkk~jv6kf24TmcVeYz~5Q4rjqq0nv2WSOAp6BzlCiVRSWfAJrLfd6Rs4gZxKg7zzM3p7v4lghWR25PMWNsgf9OjB9E~dcpY3oIEnRjBnYw08d-jcYDi16PCUWl4vpJ8sc7TT~SmCCvmzCc-aqt9FosbDLEFRYecTaJo~iZ1e-nNEqAzuBRb7DOpUa9FmzbJHPzIiMLbLKit6pIzr~8zQyv120OceaW8Y71yUOPQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196117&amp;ar=2754535&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01003.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Prediction of known gene clusters. The numbers of known gene clusters that were predicted within the minimum error of 10 genes were analyzed by varying three parameters, Param1, Param2 and Param4, one by one in a, b and c, respectively. The tentative default values, except for the parameter indicated in each panel, were Param1 = e−10, Param2 = −0.2, Param3 = 35 genes, Param4 = −0.3, Param5 = 10 kb and Param6 = 25%.</p></div></div><div data-id="DSU010F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/m_dsu01004.jpeg?Expires=2147483647&amp;Signature=0K~seuoiseaxjqNHXGau0hYe1upmNW6cjfg7AaphxXNZZO~LDKW9UDfohqPYkUfZUlA3nrp0JMQbr5aBBk5A-WNMnplyviMcrjEaA2t0zX9t9YHNaqB0taz6F1JJNDLfn8OcKvQcc9yP2NkbXf~riuo~q57Cbh1SfacGWQ6cn3Q18Z05P2FUFP83jACrxLRmvswvaavXAgvaDDB1HjQPDjaO6ogONwQQnsn9joVQetaNSOE1Vq8N6EWAdOU8TOyZiZq8YiyFTky5bVf3frVtbbHXnWjl~fh7-~1JA8-BkG4XAGvjfax4xSKaLQ0SsvkUHZHhzfKwaBqsVoRZCT0lUA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Prediction of Aspergillus fumigatus gene clusters. The number of predicted gene clusters (Y axis) in the indicated size range (X axis) was analyzed by varying Param1, Param2 and Param4 using the results of A. fumigatus as an example. The default values, except for the parameter indicated in each panel, were the same as those used in Fig. 3." path-from-xml="dsu01004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196117/dsu01004.jpeg" path-from-xml="dsu01004.jpeg" target="_blank">View large</a><a section="87196117" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/dsu01004.jpeg?Expires=2147483647&amp;Signature=ff2miYKTHaBZ1J67Kc3NH9KYOWI9hClaHNiLNQ4xcnUKug3rZiUCd9MBU~6w7pPS4tGZCwY15vfPX-wtz97e9NbvI4WlIWWmQ~AWq5ZHtE5asOvV5DrqyrgW1ZRCK0vu2wI4FQ65Rz8nFvKZKMO9AzRREim68XvSV0dw0SlFyrMPOQX3Jh2d5kamA82hkOdFkU6weR0~Gdg2AefLmnGnCPpsWqzl3BlaymmQ~HTxPgfauDpeK4MCnx08EATwQOSoXR7l4sxJWZzjOVEje9K6O-NMpJzlf9qijkvKNTw-Tu8iERSFsVFguZxfq9LRFi87n5sMx1QEMSq6-n9d4pLIsw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196117&amp;ar=2754535&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Prediction of <em>Aspergillus fumigatus</em> gene clusters. The number of predicted gene clusters (<em>Y</em> axis) in the indicated size range (<em>X</em> axis) was analyzed by varying Param1, Param2 and Param4 using the results of <em>A. fumigatus</em> as an example. The default values, except for the parameter indicated in each panel, were the same as those used in Fig. <a reveal-id="DSU010F3" data-open="DSU010F3" class="link link-reveal link-table xref-fig">3</a>.</p></div></div><div content-id="DSU010F4" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/m_dsu01004.jpeg?Expires=2147483647&amp;Signature=0K~seuoiseaxjqNHXGau0hYe1upmNW6cjfg7AaphxXNZZO~LDKW9UDfohqPYkUfZUlA3nrp0JMQbr5aBBk5A-WNMnplyviMcrjEaA2t0zX9t9YHNaqB0taz6F1JJNDLfn8OcKvQcc9yP2NkbXf~riuo~q57Cbh1SfacGWQ6cn3Q18Z05P2FUFP83jACrxLRmvswvaavXAgvaDDB1HjQPDjaO6ogONwQQnsn9joVQetaNSOE1Vq8N6EWAdOU8TOyZiZq8YiyFTky5bVf3frVtbbHXnWjl~fh7-~1JA8-BkG4XAGvjfax4xSKaLQ0SsvkUHZHhzfKwaBqsVoRZCT0lUA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Prediction of Aspergillus fumigatus gene clusters. The number of predicted gene clusters (Y axis) in the indicated size range (X axis) was analyzed by varying Param1, Param2 and Param4 using the results of A. fumigatus as an example. The default values, except for the parameter indicated in each panel, were the same as those used in Fig. 3." path-from-xml="dsu01004.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196117/dsu01004.jpeg" path-from-xml="dsu01004.jpeg" target="_blank">View large</a><a section="87196117" href="/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/dsu01004.jpeg?Expires=2147483647&amp;Signature=ff2miYKTHaBZ1J67Kc3NH9KYOWI9hClaHNiLNQ4xcnUKug3rZiUCd9MBU~6w7pPS4tGZCwY15vfPX-wtz97e9NbvI4WlIWWmQ~AWq5ZHtE5asOvV5DrqyrgW1ZRCK0vu2wI4FQ65Rz8nFvKZKMO9AzRREim68XvSV0dw0SlFyrMPOQX3Jh2d5kamA82hkOdFkU6weR0~Gdg2AefLmnGnCPpsWqzl3BlaymmQ~HTxPgfauDpeK4MCnx08EATwQOSoXR7l4sxJWZzjOVEje9K6O-NMpJzlf9qijkvKNTw-Tu8iERSFsVFguZxfq9LRFi87n5sMx1QEMSq6-n9d4pLIsw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87196117&amp;ar=2754535&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsu01004.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Prediction of <em>Aspergillus fumigatus</em> gene clusters. The number of predicted gene clusters (<em>Y</em> axis) in the indicated size range (<em>X</em> axis) was analyzed by varying Param1, Param2 and Param4 using the results of <em>A. fumigatus</em> as an example. The default values, except for the parameter indicated in each panel, were the same as those used in Fig. <a reveal-id="DSU010F3" data-open="DSU010F3" class="link link-reveal link-table xref-fig">3</a>.</p></div></div></div><p>Although Param2 and Param4 are penalties for the alignment of homologous genes, the former takes the order of the genes into consideration, but the latter does not. A decrease in the number of predicted clusters for a stringent Param2 and little change on the number by Param4, as shown above, indicated that a pair of gene clusters has similar gene contents in terms of sequence similarity, although the order of the genes might be partially rearranged, such as by inversion.</p>                            <h3 scrollto-destination=87196119 id="87196119" class="section-title" data-legacy-id=s3b>3.2. Detailed analysis of successful and failed predictions of known gene clusters</h3>
<p>Of the 21 known SMB gene clusters predicted using our method (Table <a reveal-id="DSU010TB1" data-open="DSU010TB1" class="link link-reveal link-table xref-fig">1</a>), some of the clusters were predicted by comparing two genomes belonging to different genera. For example, the gene clusters for the biosynthesis of aflatoxin in <em>A. flavus</em> and fumonisin in <em>F. verticillioides</em> were predicted by comparison with the <em>M. grisea</em> and <em>A. fumigatus</em> genomes, respectively. The SMB gene clusters appear to be composed of genes with common sequence characteristics, even between genomes from different species with phylogenetically extensive distances.</p><p>Despite the high probability of detecting the known SMB gene clusters described above, the detection of clusters for Pes1, fusaric acid and asperthecin failed. The Pes1 and asperthecin biosynthesis gene clusters consisted of only two and three genes, respectively, and had little or no chance of having conserved homologous pairs longer than three genes in the same order in the genome. The fusaric acid biosynthesis gene cluster, which contains a total of five genes, included three genes that had unique sequences. Given the abovementioned reasons, ∼12.5% of the known SMB gene clusters are thought to remain unpredicted. The kojic acid biosynthesis gene cluster, which consisted of three genes with only weak similarity to the genes sequenced to date, was successfully detected, although its cluster size was overestimated. It is thought that the existence of genes adjacent to a cluster with a high similarity to genes of a distantly related gene cluster led to the successful detection of this short gene cluster (Table <a reveal-id="DSU010TB1" data-open="DSU010TB1" class="link link-reveal link-table xref-fig">1</a>).</p><p>Considering the accuracy of detecting the known SMB gene clusters described above, the predicted unknown gene clusters without the core genes are highly likely to also be involved in SMB (see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/dsu010_Supplementary_Data.zip?Expires=2147483647&Signature=MPiG6gc9hUdygTRnGANiyQgA7d8IpwlJMFM~v9-F7bK2pcJoH7ZgE0EsTi3yY56pjA-paZpYbdY18FI7rR9CHSSMngi-qHrGa4vd7pQr0Mc0u5QSIf-DMyDUR0xMFMI1a2Zm~koOiqyHEthZYvNNyleD9NYlf2zn7yFaWqTCk5Mrt9TUB6W84S79etUoGJlJThjWyuKO~XtL3Eiv1ahciIwfegnGKncFUbO0nF4EhNDB1dvl6wHEdlSzllFLh0b5YSPw1U3q0h7roGA9E4JS0wHc~wDa1hOhJZGBzn-eAOeMXIXx2-rAFhqatyWRWNTGau4jy1n5U3VSdyyni1gybQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Data</a></span>, complete lists of predicted clusters from <em>A. nidulans</em>, <em>A. fumigatus, A. flavus</em> and <em>A. oryzae</em>). To further evaluate the probability of a relationship with SMB, the content of Q (secondary metabolism) genes in the euKaryotic Orthologous Groups (KOG) functional category was analyzed. The ratios of the Q genes in the predicted clusters and the remainder of the genes on nonsyntenic blocks from the <em>A. fumigatus</em> genome were 119/1,038 and 100/2,297, respectively. The successive statistical analysis of this result indicated enrichment for Q genes in the predicted clusters with a <em>P</em>-value of 10<sup>−13</sup>, which strongly suggested that the predicted unknown gene clusters were related to SMB regardless of the existence of core genes in the cluster.</p><div class="block-child-p">Interestingly, some known gene clusters were detected by comparison with the gene cluster that appeared to have little relationship except for the core structure of the products, such as polyketide, a nonribosomal peptide. For example, the <em>A. nidulans</em> asperfuranone biosynthesis and <em>A. terreus</em> lovastatin biosynthesis gene clusters (Fig. <a reveal-id="DSU010F5" data-open="DSU010F5" class="link link-reveal link-table xref-fig">5</a>, Table <a reveal-id="DSU010TB2" data-open="DSU010TB2" class="link link-reveal link-table xref-fig">2</a>) consisted of genes annotated as PKS, oxidoreductase and a transporter (Table <a reveal-id="DSU010TB2" data-open="DSU010TB2" class="link link-reveal link-table xref-fig">2</a>). These gene clusters were aligned in the forward and reverse directions to create a seed (Fig. <a reveal-id="DSU010F5" data-open="DSU010F5" class="link link-reveal link-table xref-fig">5</a>). <div content-id="DSU010TB2" class="table-modal"><div class="table-wrap"><div id="DSU010TB2" data-id="DSU010TB2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Examples of member genes in a predicted gene cluster</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>GID </th><th>Protein </th><th>Predicted function </th><th>GID </th><th>Protein<sup>a</sup> </th><th>Predicted function </th><th>E-value<sup>b</sup> </th></tr></thead><tbody><tr><td>ANID_01030 </td><td>406 aa </td><td>Zinc-binding oxidoreductase </td><td>ATEG_09963 </td><td>364 aa </td><td>hypothetical protein similar to enoyl reductase </td><td>2.00E−18 </td></tr><tr><td>ANID_01031 </td><td>564 aa </td><td>MFS transporter </td><td>ATEG_09967 </td><td>543 aa </td><td>hypothetical protein similar to efflux pump </td><td>7.00E−97 </td></tr><tr><td>ANID_01032 </td><td>298 aa </td><td>Conserved hypothetical protein </td><td>ATEG_09962 </td><td>257 aa </td><td>hypothetical protein similar to oxidoreductase </td><td>7.00E−12 </td></tr><tr><td>ANID_01034 </td><td>2723 aa </td><td>Polyketide synthase </td><td>ATEG_09961 </td><td>3005 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>6.00E−57 </td></tr><tr><td>ANID_01034 </td><td>2723 aa </td><td>Polyketide synthase </td><td>ATEG_09968 </td><td>2453 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>5.00E−34 </td></tr><tr><td>ANID_01036 </td><td>2528 aa </td><td>Polyketide synthase </td><td>ATEG_09968 </td><td>2543 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>0.00E+00 </td></tr><tr><td>ANID_01036 </td><td>2528 aa </td><td>Polyketide synthase </td><td>ATEG_09961 </td><td>3005 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>0.00E+00 </td></tr></tbody></table></div><div content-id="DSU010TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>GID </th><th>Protein </th><th>Predicted function </th><th>GID </th><th>Protein<sup>a</sup> </th><th>Predicted function </th><th>E-value<sup>b</sup> </th></tr></thead><tbody><tr><td>ANID_01030 </td><td>406 aa </td><td>Zinc-binding oxidoreductase </td><td>ATEG_09963 </td><td>364 aa </td><td>hypothetical protein similar to enoyl reductase </td><td>2.00E−18 </td></tr><tr><td>ANID_01031 </td><td>564 aa </td><td>MFS transporter </td><td>ATEG_09967 </td><td>543 aa </td><td>hypothetical protein similar to efflux pump </td><td>7.00E−97 </td></tr><tr><td>ANID_01032 </td><td>298 aa </td><td>Conserved hypothetical protein </td><td>ATEG_09962 </td><td>257 aa </td><td>hypothetical protein similar to oxidoreductase </td><td>7.00E−12 </td></tr><tr><td>ANID_01034 </td><td>2723 aa </td><td>Polyketide synthase </td><td>ATEG_09961 </td><td>3005 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>6.00E−57 </td></tr><tr><td>ANID_01034 </td><td>2723 aa </td><td>Polyketide synthase </td><td>ATEG_09968 </td><td>2453 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>5.00E−34 </td></tr><tr><td>ANID_01036 </td><td>2528 aa </td><td>Polyketide synthase </td><td>ATEG_09968 </td><td>2543 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>0.00E+00 </td></tr><tr><td>ANID_01036 </td><td>2528 aa </td><td>Polyketide synthase </td><td>ATEG_09961 </td><td>3005 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>0.00E+00 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn6"><div class="fn"><p><sup>a</sup>Length of the polypeptide in amino acids.</p></div></div><div content-id="tbnfn7"><div class="fn"><p><sup>b</sup>E-value of the similarity between the proteins of the detected gene clusters.</p></div></div></div></div></div><div class="table-wrap"><div id="DSU010TB2" data-id="DSU010TB2" class="table-wrap-title"><div class="label -label">Table 2.</div><div class="caption"><p>Examples of member genes in a predicted gene cluster</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>GID </th><th>Protein </th><th>Predicted function </th><th>GID </th><th>Protein<sup>a</sup> </th><th>Predicted function </th><th>E-value<sup>b</sup> </th></tr></thead><tbody><tr><td>ANID_01030 </td><td>406 aa </td><td>Zinc-binding oxidoreductase </td><td>ATEG_09963 </td><td>364 aa </td><td>hypothetical protein similar to enoyl reductase </td><td>2.00E−18 </td></tr><tr><td>ANID_01031 </td><td>564 aa </td><td>MFS transporter </td><td>ATEG_09967 </td><td>543 aa </td><td>hypothetical protein similar to efflux pump </td><td>7.00E−97 </td></tr><tr><td>ANID_01032 </td><td>298 aa </td><td>Conserved hypothetical protein </td><td>ATEG_09962 </td><td>257 aa </td><td>hypothetical protein similar to oxidoreductase </td><td>7.00E−12 </td></tr><tr><td>ANID_01034 </td><td>2723 aa </td><td>Polyketide synthase </td><td>ATEG_09961 </td><td>3005 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>6.00E−57 </td></tr><tr><td>ANID_01034 </td><td>2723 aa </td><td>Polyketide synthase </td><td>ATEG_09968 </td><td>2453 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>5.00E−34 </td></tr><tr><td>ANID_01036 </td><td>2528 aa </td><td>Polyketide synthase </td><td>ATEG_09968 </td><td>2543 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>0.00E+00 </td></tr><tr><td>ANID_01036 </td><td>2528 aa </td><td>Polyketide synthase </td><td>ATEG_09961 </td><td>3005 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>0.00E+00 </td></tr></tbody></table></div><div content-id="DSU010TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>GID </th><th>Protein </th><th>Predicted function </th><th>GID </th><th>Protein<sup>a</sup> </th><th>Predicted function </th><th>E-value<sup>b</sup> </th></tr></thead><tbody><tr><td>ANID_01030 </td><td>406 aa </td><td>Zinc-binding oxidoreductase </td><td>ATEG_09963 </td><td>364 aa </td><td>hypothetical protein similar to enoyl reductase </td><td>2.00E−18 </td></tr><tr><td>ANID_01031 </td><td>564 aa </td><td>MFS transporter </td><td>ATEG_09967 </td><td>543 aa </td><td>hypothetical protein similar to efflux pump </td><td>7.00E−97 </td></tr><tr><td>ANID_01032 </td><td>298 aa </td><td>Conserved hypothetical protein </td><td>ATEG_09962 </td><td>257 aa </td><td>hypothetical protein similar to oxidoreductase </td><td>7.00E−12 </td></tr><tr><td>ANID_01034 </td><td>2723 aa </td><td>Polyketide synthase </td><td>ATEG_09961 </td><td>3005 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>6.00E−57 </td></tr><tr><td>ANID_01034 </td><td>2723 aa </td><td>Polyketide synthase </td><td>ATEG_09968 </td><td>2453 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>5.00E−34 </td></tr><tr><td>ANID_01036 </td><td>2528 aa </td><td>Polyketide synthase </td><td>ATEG_09968 </td><td>2543 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>0.00E+00 </td></tr><tr><td>ANID_01036 </td><td>2528 aa </td><td>Polyketide synthase </td><td>ATEG_09961 </td><td>3005 aa </td><td>hypothetical protein similar to polyketide synthase </td><td>0.00E+00 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tbnfn6"><div class="fn"><p><sup>a</sup>Length of the polypeptide in amino acids.</p></div></div><div content-id="tbnfn7"><div class="fn"><p><sup>b</sup>E-value of the similarity between the proteins of the detected gene clusters.</p></div></div></div></div><div data-id="DSU010F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/m_dsu01005.jpeg?Expires=2147483647&Signature=t09T1JNt7L7rIqAN6sgEpcu6JCHIiuM90E4r9jW8ZvKd1Jr2J9Uq7og87koCDe8Q2tIyLSfkTSilHKPaFkAIs6ZEbCs0ux1se~QHTDEHEytwRzZzGZm6ytnGvDxFc98nY18wqG-YojlY6Zl3aqzy8viZaOm5jBMN2~VuAj9nDLAG8~36R3bZiQeHR1FslYsVSpvfqXwaHSBsZHYt6wMHjkIily98vE3neaPnsSgTYijdqXvo9COVNlBptZ~KhryPqK646tS7bYGuaZ3~viSrmrvF1ksi6W5O0vRFAN6F4v4nkruPOWhXXG9TQ0sRnQdA5et9OadlUu0szoNeeyUqMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic drawing of an example of a predicted known SMB gene cluster. The top figures represent seeds used in the detection of a pair of SMB gene clusters for A. nidulans asperfuranone and A. terreus lovastatin. The left and the right panels show the alignments in the forward and reverse directions, respectively. The bottom figure shows all of the homologous gene pairs included between the two clusters. No orthologs were identified in this pair of gene clusters." path-from-xml="dsu01005.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196123/dsu01005.jpeg" path-from-xml="dsu01005.jpeg" target="_blank">View large</a><a section="87196123" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/dsu01005.jpeg?Expires=2147483647&Signature=oPyDh1XVK6dZZboH~zEnHjs0Q-5zOv284wAlWoZX0pyr6lQ2oXfXkDpwWkqn0ZhnNXdGR90UFhhiLpw7~Mp-W7SvQJ1ChfsHMksDcEqUnek~oO6IWcSTYSQ0k5jWoIgiDdJzFI5ejBAqfyfGQ4fe09SDDa1XOfp5s6UyfzeRqpKVkW-GtAdCqhxVnHLTpC78cyHiQnpGfOW9kdaDGcxgnOuVzLPInBhhdm4Y4o5Sdgjtk4JbWQ7ETHomDZtsZ-XvY4nLxl83CxzyyRrQQfqFNnoLmyYBdbH3OIeVJ6jommoht~mc-JvUMjen1wZ87BMKSjocVonip~4XQlFsom5fiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196123&ar=2754535&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01005.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic drawing of an example of a predicted known SMB gene cluster. The top figures represent seeds used in the detection of a pair of SMB gene clusters for <em>A. nidulans</em> asperfuranone and <em>A. terreus</em> lovastatin. The left and the right panels show the alignments in the forward and reverse directions, respectively. The bottom figure shows all of the homologous gene pairs included between the two clusters. No orthologs were identified in this pair of gene clusters.</p></div></div><div content-id="DSU010F5" class="fig fig-modal reveal-modal" data-reveal="data-reveal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/m_dsu01005.jpeg?Expires=2147483647&Signature=t09T1JNt7L7rIqAN6sgEpcu6JCHIiuM90E4r9jW8ZvKd1Jr2J9Uq7og87koCDe8Q2tIyLSfkTSilHKPaFkAIs6ZEbCs0ux1se~QHTDEHEytwRzZzGZm6ytnGvDxFc98nY18wqG-YojlY6Zl3aqzy8viZaOm5jBMN2~VuAj9nDLAG8~36R3bZiQeHR1FslYsVSpvfqXwaHSBsZHYt6wMHjkIily98vE3neaPnsSgTYijdqXvo9COVNlBptZ~KhryPqK646tS7bYGuaZ3~viSrmrvF1ksi6W5O0vRFAN6F4v4nkruPOWhXXG9TQ0sRnQdA5et9OadlUu0szoNeeyUqMw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schematic drawing of an example of a predicted known SMB gene cluster. The top figures represent seeds used in the detection of a pair of SMB gene clusters for A. nidulans asperfuranone and A. terreus lovastatin. The left and the right panels show the alignments in the forward and reverse directions, respectively. The bottom figure shows all of the homologous gene pairs included between the two clusters. No orthologs were identified in this pair of gene clusters." path-from-xml="dsu01005.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87196123/dsu01005.jpeg" path-from-xml="dsu01005.jpeg" target="_blank">View large</a><a section="87196123" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/dsu01005.jpeg?Expires=2147483647&Signature=oPyDh1XVK6dZZboH~zEnHjs0Q-5zOv284wAlWoZX0pyr6lQ2oXfXkDpwWkqn0ZhnNXdGR90UFhhiLpw7~Mp-W7SvQJ1ChfsHMksDcEqUnek~oO6IWcSTYSQ0k5jWoIgiDdJzFI5ejBAqfyfGQ4fe09SDDa1XOfp5s6UyfzeRqpKVkW-GtAdCqhxVnHLTpC78cyHiQnpGfOW9kdaDGcxgnOuVzLPInBhhdm4Y4o5Sdgjtk4JbWQ7ETHomDZtsZ-XvY4nLxl83CxzyyRrQQfqFNnoLmyYBdbH3OIeVJ6jommoht~mc-JvUMjen1wZ87BMKSjocVonip~4XQlFsom5fiQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87196123&ar=2754535&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsu01005.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schematic drawing of an example of a predicted known SMB gene cluster. The top figures represent seeds used in the detection of a pair of SMB gene clusters for <em>A. nidulans</em> asperfuranone and <em>A. terreus</em> lovastatin. The left and the right panels show the alignments in the forward and reverse directions, respectively. The bottom figure shows all of the homologous gene pairs included between the two clusters. No orthologs were identified in this pair of gene clusters.</p></div></div></div>                            <h3 scrollto-destination=87196124 id="87196124" class="section-title" data-legacy-id=s3c>3.3. Properties of secondary metabolism genes</h3>
<p>We devised a comparative genomics method for predicting SMB gene clusters by effectively utilizing the rapidly growing accumulation of genome sequences. In this study, we have successfully identified the known SMB gene clusters with a high probability (21 out of 24 clusters). The results indicate overall similarities in the amino acid sequences and/or the order of member genes between the pairs of gene clusters, including those involved in the biosynthesis of <em>A. nidulans</em> asperfuranone and <em>A. terreus</em> lovastatin, <em>A. fumigatus</em> fumitremorgin and <em>F. verticillioides</em> fumonisin and <em>A. fumigatus</em> melanin and <em>F. graminearum</em> aurofusarin.</p><p>Secondary metabolism genes are highly enriched in the nonsyntenic blocks in a comparison of the genomes of three <em>Aspergillus</em> species.<sup><a reveal-id="DSU010C12" data-open="DSU010C12" class="link link-ref link-reveal xref-bibr">12</a></sup> We have applied this observation to our method and have successfully identified various known SMB gene clusters from the genomes of the 10 fungal species, including those outside the genus <em>Aspergillus</em>. This observation indicates that the high enrichment of secondary metabolism genes in nonsyntenic blocks was conserved among various species for at least the 10 fungal species used in this study. However, SMB gene clusters producing common products in phylogenetically close species may often be syntenic, as previously shown in various reports, which has resulted in the failure of detection by comparisons of the corresponding clusters in the respective genomes. Typical examples of unsuccessful detections involved the combinations of SMB gene clusters for <em>A. flavus</em> aflatoxin and <em>A. nidulans</em> sterigmatocystin<sup><a reveal-id="DSU010C19" data-open="DSU010C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSU010C20" data-open="DSU010C20" class="link link-ref link-reveal xref-bibr">20</a></sup> and <em>F. verticillioides</em> and <em>F. oxysporum</em> bikaverin cluster homologs.<sup><a reveal-id="DSU010C21" data-open="DSU010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> Similarly, horizontal transfer of a gene cluster may also result in unsuccessful detection of the cluster, even between species with large phylogenetic distances.<sup><a reveal-id="DSU010C19" data-open="DSU010C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSU010C21" data-open="DSU010C21" class="link link-ref link-reveal xref-bibr">21</a></sup> An SMB gene cluster is known to consist of genes encoding proteins of particular characteristic functions, such as PKSs, NRPSs, Zn(II)<sub>2</sub>-Cys<sub>6</sub> transcription factors,<sup><a reveal-id="DSU010C22" data-open="DSU010C22" class="link link-ref link-reveal xref-bibr">22</a></sup> and major facilitator superfamily (MFS) transporters.<sup><a reveal-id="DSU010C23" data-open="DSU010C23" class="link link-ref link-reveal xref-bibr">23</a></sup> Significant enrichment of these genes allowed identification of SMB gene clusters, owing to the overall similarity among various clusters producing different compounds and between species with large phylogenetic distances. Our method, which first detected seeds by local gene alignments and successively corrected their boundaries using simple similarity searches independent of synteny, identified SMB gene clusters more efficiently than expected prior to this study, even though nonsyntenic blocks are known to have high diversity.<sup><a reveal-id="DSU010C24" data-open="DSU010C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSU010C25" data-open="DSU010C25" class="link link-ref link-reveal xref-bibr">25</a></sup></p><p>The previously reported methods predicted SMB gene clusters based on the sequence similarity of the core genes in the cluster, such as NRPS, PKS, a hybrid NRPS-PKS enzyme and DMAT.<sup><a reveal-id="DSU010C6" data-open="DSU010C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSU010C7" data-open="DSU010C7" class="link link-ref link-reveal xref-bibr">7</a>,<a reveal-id="DSU010C18" data-open="DSU010C18" class="link link-ref link-reveal xref-bibr">18</a></sup> In contrast to these methods, our method does not depend on the presence of core genes. Due to this remarkable feature, the <em>A. oryzae</em> kojic acid biosynthesis gene cluster, which does not include core genes, was successfully predicted using this method. In contrast, there are also examples of missing predictions of known short SMB gene clusters, such as those responsible for the biosynthesis of asperthecin in <em>A. nidulans</em> and fusaric acid in <em>F. verticillioides</em> (Table <a reveal-id="DSU010TB1" data-open="DSU010TB1" class="link link-reveal link-table xref-fig">1</a>). The inability to identify the gene clusters named above was due to the existence of an inversion in the former cluster and unique genes in the latter one, resulting in the failure of the local alignment of homologous gene pairs. In both cases, the short sizes of the clusters (three to five genes) prevented the remaining portions of the clusters from being identified. Similarly, intervention of a cluster by a horizontal gene transfer of another cluster (more than five genes), dividing the cluster into small segments,<sup><a reveal-id="DSU010C26" data-open="DSU010C26" class="link link-ref link-reveal xref-bibr">26</a></sup> may also cause detection failure.</p><p>In this study, many known SMB gene clusters were identified within 19 genes as errors for the maximum CB score, when error is defined as the sum of absolute differences of cluster margins at both ends. Because our method does not depend on gene order within the length of the cluster size for the correction of cluster boundaries, this observation strongly suggests that the genes inside and outside of the clusters have different sequence characteristics. Accordingly, the probability of homology between the genes inside the clusters from the two genomes is significantly higher than (i) the probability of homology between the genes outside the clusters or (ii) the probability of homology between the genes inside the clusters and the genes outside the clusters (<em>P</em> = 6.2 × 10–121, χ<sup>2</sup> test). In contrast, the cluster sizes of some gene clusters, e.g. the kojic acid biosynthesis gene cluster, were overestimated. This overestimation suggests two possibilities: (i) the two clusters were located side by side with few or no non-SMB genes in between or (ii) the gene cluster may be a part of the ancestral SMB gene cluster, with the remainder of the genes being presently inactive. In the cases of clusters with errors larger than 10 at the maximum CB value, such as the clusters for aflatoxin, terrequinone and pseurotin biosynthesis as well as kojic acid biosynthesis in Table <a reveal-id="DSU010TB1" data-open="DSU010TB1" class="link link-reveal link-table xref-fig">1</a>, genes with characteristics of SMB genes were identified beyond the experimentally determined cluster margins.</p><p>As described above, our method is a useful means to predict SMB gene clusters, particularly novel clusters without core genes; thus, this method has the potential to discover novel mechanisms of unknown SMBs. Two major problems of our method are that short gene clusters might not be detected in some cases and that the prediction of a cluster boundary might not always be accurate. Recently, a method for predicting accurate margins of SMB gene clusters by analyzing the co-expression of neighboring genes has been reported,<sup><a reveal-id="DSU010C27" data-open="DSU010C27" class="link link-ref link-reveal xref-bibr">27</a></sup> with the condition that the gene indispensable for the SMB gene cluster is identified using the known sequence of the core gene. Therefore, the combination of our method and the expression analysis method described above could effectively compensate for the problems that currently exist in both methods. However, our method is essentially not applicable to SMB gene clusters that are unique to particular genomes. Nevertheless, of the 24 known SMB gene clusters on the 10 genomes used in this study, 21 clusters were identified via comprehensive pairwise comparisons. The problem of detecting ‘rare SMB gene clusters’ could be solved by increasing the number of genomes used for predictions. The acceleration of sequence accumulation due to the rapid development of sequencing technologies is expected to significantly increase our method's performance of in a short period of time. A comprehensive analysis of the distribution of secondary metabolism genes and motifs in translated polypeptides across diverse species, together with the structural analyses of corresponding compounds, will open a new era in the study of secondary metabolism.</p>                            <h3 scrollto-destination=87196130 id="87196130" class="section-title" data-legacy-id=s3d>3.4. Availability</h3>
<p>We intend to provide the present method as a web service (<a class="link link-uri" href="http://www.fung-metb.net/">http://www.fung-metb.net/</a>).</p>                            <h2 scrollto-destination=87196132 id="87196132" class="section-title" data-legacy-id=s5>Funding</h2>
<p>This work was partly supported by the commission for the Development of Artificial Gene Synthesis Technology for Creating Innovative Biomaterial from the Ministry of Economy, Trade and Industry (METI), Japan.</p>                            <h2 scrollto-destination=87196134 id="87196134" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We thank AMERICAN JOURNAL EXPERTS for proofing the manuscript.</p>                            <h2 scrollto-destination=87196136 id="87196136" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSU010C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Maiya</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Grundmann</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">S.M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Turner</div>
               <div class="given-names">G.</div>
            </div>
          </div>
          <div class="article-title">The fumitremorgin gene cluster of <em>Aspergillus fumigatus</em>: identification of a gene encoding brevianamide F synthetase</div>
          <div class="source">Chembiochem</div> 
          <div class="year">2006</div>
          <div class="volume">7</div>
          <div class="fpage">1062</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20fumitremorgin%20gene%20cluster%20of%20Aspergillus%20fumigatus%3A%20identification%20of%20a%20gene%20encoding%20brevianamide%20F%20synthetase&amp;author=S.%20Maiya&amp;author=A.%20Grundmann&amp;author=S.M.%20Li&amp;author=G.%20Turner&amp;publication_year=2006&amp;journal=Chembiochem&amp;volume=7&amp;pages=1062-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/cbic.v7:7" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fcbic.v7:7" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fcbic.v7:7"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16755625" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Bergmann</div>
               <div class="given-names">S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Schumann</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Scherlach</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lange</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Brakhage</div>
               <div class="given-names">A.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hertweck</div>
               <div class="given-names">C.</div>
            </div>
          </div>
          <div class="article-title">Genomics-driven discovery of PKS-NRPS hybrid metabolites from <em>Aspergillus nidulans</em></div>
          <div class="source">Nat Chem Biol.</div> 
          <div class="year">2007</div>
          <div class="volume">3</div>
          <div class="fpage">213</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics-driven%20discovery%20of%20PKS-NRPS%20hybrid%20metabolites%20from%20Aspergillus%20nidulans&amp;author=S.%20Bergmann&amp;author=J.%20Schumann&amp;author=K.%20Scherlach&amp;author=C.%20Lange&amp;author=A.A.%20Brakhage&amp;author=C.%20Hertweck&amp;publication_year=2007&amp;journal=Nat%20Chem%20Biol.&amp;volume=3&amp;pages=213-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nchembio869" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnchembio869" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnchembio869"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17369821" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Pel</div>
               <div class="given-names">H.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">de Winde</div>
               <div class="given-names">J.H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Archer</div>
               <div class="given-names">D.B.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Genome sequencing and analysis of the versatile cell factory <em>Aspergillus niger</em> CBS 513.88</div>
          <div class="source">Nat Biotechnol.</div> 
          <div class="year">2007</div>
          <div class="volume">25</div>
          <div class="fpage">221</div>
          <div class="lpage">31</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20and%20analysis%20of%20the%20versatile%20cell%20factory%20Aspergillus%20niger%20CBS%20513.88&amp;author=H.J.%20Pel&amp;author=J.H.%20de%20Winde&amp;author=D.B.%20Archer&amp;publication_year=2007&amp;journal=Nat%20Biotechnol.&amp;volume=25&amp;pages=221-31" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt1282" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt1282" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt1282"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17259976" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Weber</div>
               <div class="given-names">T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Rausch</div>
               <div class="given-names">C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lopez</div>
               <div class="given-names">P.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">CLUSEAN: a computer-based framework for the automated analysis of bacterial secondary metabolite biosynthetic gene clusters</div>
          <div class="source">J Biotechnol.</div> 
          <div class="year">2009</div>
          <div class="volume">140</div>
          <div class="fpage">13</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=CLUSEAN%3A%20a%20computer-based%20framework%20for%20the%20automated%20analysis%20of%20bacterial%20secondary%20metabolite%20biosynthetic%20gene%20clusters&amp;author=T.%20Weber&amp;author=C.%20Rausch&amp;author=P.%20Lopez&amp;publication_year=2009&amp;journal=J%20Biotechnol.&amp;volume=140&amp;pages=13-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jbiotec.2009.01.007" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jbiotec.2009.01.007" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jbiotec.2009.01.007"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19297688" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Starcevic</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Zucko</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Simunkovic</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Long</div>
               <div class="given-names">P.F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Cullum</div>
               <div class="given-names">J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Hranueli</div>
               <div class="given-names">D.</div>
            </div>
          </div>
          <div class="article-title">ClustScan: an integrated program package for the semi-automatic annotation of modular biosynthetic gene clusters and in silico prediction of novel chemical structures</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2008</div>
          <div class="volume">36</div>
          <div class="fpage">6882</div>
          <div class="lpage">92</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ClustScan%3A%20an%20integrated%20program%20package%20for%20the%20semi-automatic%20annotation%20of%20modular%20biosynthetic%20gene%20clusters%20and%20in%20silico%20prediction%20of%20novel%20chemical%20structures&amp;author=A.%20Starcevic&amp;author=J.%20Zucko&amp;author=J.%20Simunkovic&amp;author=P.F.%20Long&amp;author=J.%20Cullum&amp;author=D.%20Hranueli&amp;publication_year=2008&amp;journal=Nucleic%20Acids%20Res.&amp;volume=36&amp;pages=6882-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkn685" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkn685" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkn685"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18978015" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Khaldi</div>
               <div class="given-names">N.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Seifuddin</div>
               <div class="given-names">F.T.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Turner</div>
               <div class="given-names">G.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">SMURF: genomic mapping of fungal secondary metabolite clusters</div>
          <div class="source">Fungal Genet Biol.</div> 
          <div class="year">2010</div>
          <div class="volume">47</div>
          <div class="fpage">736</div>
          <div class="lpage">41</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SMURF%3A%20genomic%20mapping%20of%20fungal%20secondary%20metabolite%20clusters&amp;author=N.%20Khaldi&amp;author=F.T.%20Seifuddin&amp;author=G.%20Turner&amp;publication_year=2010&amp;journal=Fungal%20Genet%20Biol.&amp;volume=47&amp;pages=736-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2010.06.003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2010.06.003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2010.06.003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20554054" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Blin</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Medema</div>
               <div class="given-names">M.H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Kazempour</div>
               <div class="given-names">D.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">antiSMASH 2.0—a versatile platform for genome mining of secondary metabolite producers</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2013</div>
          <div class="volume">41</div>
          <div class="fpage">W204</div>
          <div class="lpage">12</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=antiSMASH%202.0%E2%80%94a%20versatile%20platform%20for%20genome%20mining%20of%20secondary%20metabolite%20producers&amp;author=K.%20Blin&amp;author=M.H.%20Medema&amp;author=D.%20Kazempour&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=W204-12" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkt449" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkt449" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkt449"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23737449" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Brodhun</div>
               <div class="given-names">F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Feussner</div>
               <div class="given-names">I.</div>
            </div>
          </div>
          <div class="article-title">Oxylipins in fungi</div>
          <div class="source">FEBS J.</div> 
          <div class="year">2011</div>
          <div class="volume">278</div>
          <div class="fpage">1047</div>
          <div class="lpage">63</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Oxylipins%20in%20fungi&amp;author=F.%20Brodhun&amp;author=I.%20Feussner&amp;publication_year=2011&amp;journal=FEBS%20J.&amp;volume=278&amp;pages=1047-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1742-4658.2011.08027.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1742-4658.2011.08027.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1742-4658.2011.08027.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21281447" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Terabayashi</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sano</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Yamane</div>
               <div class="given-names">N.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Identification and characterization of genes responsible for biosynthesis of kojic acid, an industrially important compound from <em>Aspergillus oryzae</em></div>
          <div class="source">Fungal Genet Biol.</div> 
          <div class="year">2010</div>
          <div class="volume">47</div>
          <div class="fpage">953</div>
          <div class="lpage">61</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20genes%20responsible%20for%20biosynthesis%20of%20kojic%20acid%2C%20an%20industrially%20important%20compound%20from%20Aspergillus%20oryzae&amp;author=Y.%20Terabayashi&amp;author=M.%20Sano&amp;author=N.%20Yamane&amp;publication_year=2010&amp;journal=Fungal%20Genet%20Biol.&amp;volume=47&amp;pages=953-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.fgb.2010.08.014" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.fgb.2010.08.014" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.fgb.2010.08.014"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20849972" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Umemura</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Koyama</div>
               <div class="given-names">Y.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Takeda</div>
               <div class="given-names">I.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Fine De Novo Sequencing of a Fungal Genome Using only SOLiD Short Read Data: Verification on <em>Aspergillus oryzae</em> RIB40</div>
          <div class="source">PLoS One</div> 
          <div class="year">2013</div>
          <div class="volume">8</div>
          <div class="fpage">e63673</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fine%20De%20Novo%20Sequencing%20of%20a%20Fungal%20Genome%20Using%20only%20SOLiD%20Short%20Read%20Data%3A%20Verification%20on%20Aspergillus%20oryzae%20RIB40&amp;author=M.%20Umemura&amp;author=Y.%20Koyama&amp;author=I.%20Takeda&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e63673" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0063673" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0063673" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0063673"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23667655" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Sanchez</div>
               <div class="given-names">J.F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Somoza</div>
               <div class="given-names">A.D.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Keller</div>
               <div class="given-names">N.P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">C.C.</div>
            </div>
          </div>
          <div class="article-title">Advances in <em>Aspergillus</em> secondary metabolite research in the post-genomic era</div>
          <div class="source">Nat Prod Rep.</div> 
          <div class="year">2012</div>
          <div class="volume">29</div>
          <div class="fpage">351</div>
          <div class="lpage">71</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Advances%20in%20Aspergillus%20secondary%20metabolite%20research%20in%20the%20post-genomic%20era&amp;author=J.F.%20Sanchez&amp;author=A.D.%20Somoza&amp;author=N.P.%20Keller&amp;author=C.C.%20Wang&amp;publication_year=2012&amp;journal=Nat%20Prod%20Rep.&amp;volume=29&amp;pages=351-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1039/c2np00084a" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1039%2Fc2np00084a" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1039%2Fc2np00084a"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22228366" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Machida</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Asai</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Sano</div>
               <div class="given-names">M.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Genome sequencing and analysis of <em>Aspergillus oryzae</em></div>
          <div class="source">Nature</div> 
          <div class="year">2005</div>
          <div class="volume">438</div>
          <div class="fpage">1157</div>
          <div class="lpage">61</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20and%20analysis%20of%20Aspergillus%20oryzae&amp;author=M.%20Machida&amp;author=K.%20Asai&amp;author=M.%20Sano&amp;publication_year=2005&amp;journal=Nature&amp;volume=438&amp;pages=1157-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature04300" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature04300" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature04300"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16372010" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Gibbons</div>
               <div class="given-names">J.G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Rokas</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="article-title">The function and evolution of the <em>Aspergillus</em> genome</div>
          <div class="source">Trends Microbiol.</div> 
          <div class="year">2013</div>
          <div class="volume">21</div>
          <div class="fpage">14</div>
          <div class="lpage">22</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20function%20and%20evolution%20of%20the%20Aspergillus%20genome&amp;author=J.G.%20Gibbons&amp;author=A.%20Rokas&amp;publication_year=2013&amp;journal=Trends%20Microbiol.&amp;volume=21&amp;pages=14-22" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tim.2012.09.005" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tim.2012.09.005" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tim.2012.09.005"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23084572" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Altschul</div>
               <div class="given-names">S.F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Madden</div>
               <div class="given-names">T.L.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Schaffer</div>
               <div class="given-names">A.A.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Gapped BLAST and PSI-BLAST: a new generation of protein database search programs</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">1997</div>
          <div class="volume">25</div>
          <div class="fpage">3389</div>
          <div class="lpage">402</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Gapped%20BLAST%20and%20PSI-BLAST%3A%20a%20new%20generation%20of%20protein%20database%20search%20programs&amp;author=S.F.%20Altschul&amp;author=T.L.%20Madden&amp;author=A.A.%20Schaffer&amp;publication_year=1997&amp;journal=Nucleic%20Acids%20Res.&amp;volume=25&amp;pages=3389-402" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/25.17.3389" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2F25.17.3389" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2F25.17.3389"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/9254694" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Altschul</div>
               <div class="given-names">S.F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gish</div>
               <div class="given-names">W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Miller</div>
               <div class="given-names">W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Myers</div>
               <div class="given-names">E.W.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Lipman</div>
               <div class="given-names">D.J.</div>
            </div>
          </div>
          <div class="article-title">Basic local alignment search tool</div>
          <div class="source">J Mol Biol.</div> 
          <div class="year">1990</div>
          <div class="volume">215</div>
          <div class="fpage">403</div>
          <div class="lpage">10</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J%20Mol%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Smith</div>
               <div class="given-names">T.F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Waterman</div>
               <div class="given-names">M.S.</div>
            </div>
          </div>
          <div class="article-title">Identification of common molecular subsequences</div>
          <div class="source">J Mol Biol.</div> 
          <div class="year">1981</div>
          <div class="volume">147</div>
          <div class="fpage">195</div>
          <div class="lpage">7</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20common%20molecular%20subsequences&amp;author=T.F.%20Smith&amp;author=M.S.%20Waterman&amp;publication_year=1981&amp;journal=J%20Mol%20Biol.&amp;volume=147&amp;pages=195-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/0022-2836(81)90087-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2F0022-2836(81)90087-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2F0022-2836(81)90087-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7265238" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Smith</div>
               <div class="given-names">T.F.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Waterman</div>
               <div class="given-names">M.S.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Fitch</div>
               <div class="given-names">W.M.</div>
            </div>
          </div>
          <div class="article-title">Comparative biosequence metrics</div>
          <div class="source">J Mol Evol.</div> 
          <div class="year">1981</div>
          <div class="volume">18</div>
          <div class="fpage">38</div>
          <div class="lpage">46</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20biosequence%20metrics&amp;author=T.F.%20Smith&amp;author=M.S.%20Waterman&amp;author=W.M.%20Fitch&amp;publication_year=1981&amp;journal=J%20Mol%20Evol.&amp;volume=18&amp;pages=38-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/BF01733210" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2FBF01733210" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2FBF01733210"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7334527" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Medema</div>
               <div class="given-names">M.H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Blin</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Cimermancic</div>
               <div class="given-names">P.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">antiSMASH: rapid identification, annotation and analysis of secondary metabolite biosynthesis gene clusters in bacterial and fungal genome sequences</div>
          <div class="source">Nucleic Acids Res.</div> 
          <div class="year">2011</div>
          <div class="volume">39</div>
          <div class="fpage">W339</div>
          <div class="lpage">346</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=antiSMASH%3A%20rapid%20identification%2C%20annotation%20and%20analysis%20of%20secondary%20metabolite%20biosynthesis%20gene%20clusters%20in%20bacterial%20and%20fungal%20genome%20sequences&amp;author=M.H.%20Medema&amp;author=K.%20Blin&amp;author=P.%20Cimermancic&amp;publication_year=2011&amp;journal=Nucleic%20Acids%20Res.&amp;volume=39&amp;pages=W339-346" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkr466" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkr466" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkr466"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21672958" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Slot</div>
               <div class="given-names">J.C.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Rokas</div>
               <div class="given-names">A.</div>
            </div>
          </div>
          <div class="article-title">Horizontal transfer of a large and highly toxic secondary metabolic gene cluster between fungi</div>
          <div class="source">Curr Biol.</div> 
          <div class="year">2011</div>
          <div class="volume">21</div>
          <div class="fpage">134</div>
          <div class="lpage">9</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Horizontal%20transfer%20of%20a%20large%20and%20highly%20toxic%20secondary%20metabolic%20gene%20cluster%20between%20fungi&amp;author=J.C.%20Slot&amp;author=A.%20Rokas&amp;publication_year=2011&amp;journal=Curr%20Biol.&amp;volume=21&amp;pages=134-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cub.2010.12.020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cub.2010.12.020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cub.2010.12.020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21194949" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Hicks</div>
               <div class="given-names">J.K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Shimizu</div>
               <div class="given-names">K.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Keller</div>
               <div class="given-names">N.P.</div>
            </div>
          </div>
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kempken</div>
               <div class="given-names">F.</div>
            </div>
          </div>
          <div class="article-title">Genetics and biosynthesis of aflatoxins and sterigmatocystin</div>
          <div class="source">Agricultural Applications</div> 
          <div class="year">2002</div>
          <div class="publisher-loc">Berlin, Germany</div>
          <div class="publisher-name">Springer</div>
          <div class="fpage">55</div>
          <div class="lpage">69</div>
        </div></div></div></div><div content-id="DSU010C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Campbell</div>
               <div class="given-names">M.A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Rokas</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Slot</div>
               <div class="given-names">J.C.</div>
            </div>
          </div>
          <div class="article-title">Horizontal transfer and death of a fungal secondary metabolic gene cluster</div>
          <div class="source">Genome Biol Evol.</div> 
          <div class="year">2012</div>
          <div class="volume">4</div>
          <div class="fpage">289</div>
          <div class="lpage">93</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Horizontal%20transfer%20and%20death%20of%20a%20fungal%20secondary%20metabolic%20gene%20cluster&amp;author=M.A.%20Campbell&amp;author=A.%20Rokas&amp;author=J.C.%20Slot&amp;publication_year=2012&amp;journal=Genome%20Biol%20Evol.&amp;volume=4&amp;pages=289-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/gbe/evs011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fgbe%2Fevs011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fgbe%2Fevs011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22294497" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Keller</div>
               <div class="given-names">N.P.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Turner</div>
               <div class="given-names">G.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Bennett</div>
               <div class="given-names">J.W.</div>
            </div>
          </div>
          <div class="article-title">Fungal secondary metabolism - from biochemistry to genomics</div>
          <div class="source">Nat Rev Microbiol.</div> 
          <div class="year">2005</div>
          <div class="volume">3</div>
          <div class="fpage">937</div>
          <div class="lpage">47</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fungal%20secondary%20metabolism%20-%20from%20biochemistry%20to%20genomics&amp;author=N.P.%20Keller&amp;author=G.%20Turner&amp;author=J.W.%20Bennett&amp;publication_year=2005&amp;journal=Nat%20Rev%20Microbiol.&amp;volume=3&amp;pages=937-47" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrmicro1286" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrmicro1286" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrmicro1286"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16322742" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Coleman</div>
               <div class="given-names">J.J.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Mylonakis</div>
               <div class="given-names">E.</div>
            </div>
          </div>
          <div class="article-title">Efflux in fungi: la piece de resistance</div>
          <div class="source">PLoS Pathog.</div> 
          <div class="year">2009</div>
          <div class="volume">5</div>
          <div class="fpage">e1000486</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Efflux%20in%20fungi%3A%20la%20piece%20de%20resistance&amp;author=J.J.%20Coleman&amp;author=E.%20Mylonakis&amp;publication_year=2009&amp;journal=PLoS%20Pathog.&amp;volume=5&amp;pages=e1000486" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.ppat.1000486" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.ppat.1000486" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.ppat.1000486"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19557154" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Umemura</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Koike</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Yamane</div>
               <div class="given-names">N.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Comparative genome analysis between <em>Aspergillus oryzae</em> strains reveals close relationship between sites of mutation localization and regions of highly divergent genes among <em>Aspergillus</em> species</div>
          <div class="source">DNA Res.</div> 
          <div class="year">2012</div>
          <div class="volume">19</div>
          <div class="fpage">375</div>
          <div class="lpage">82</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genome%20analysis%20between%20Aspergillus%20oryzae%20strains%20reveals%20close%20relationship%20between%20sites%20of%20mutation%20localization%20and%20regions%20of%20highly%20divergent%20genes%20among%20Aspergillus%20species&amp;author=M.%20Umemura&amp;author=H.%20Koike&amp;author=N.%20Yamane&amp;publication_year=2012&amp;journal=DNA%20Res.&amp;volume=19&amp;pages=375-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dss019" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdss019" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdss019"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22912434" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Machida</div>
               <div class="given-names">M.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Yamada</div>
               <div class="given-names">O.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Gomi</div>
               <div class="given-names">K.</div>
            </div>
          </div>
          <div class="article-title">Genomics of <em>Aspergillus oryzae</em>: learning from the history of Koji mold and exploration of its future</div>
          <div class="source">DNA Res.</div> 
          <div class="year">2008</div>
          <div class="volume">15</div>
          <div class="fpage">173</div>
          <div class="lpage">83</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genomics%20of%20Aspergillus%20oryzae%3A%20learning%20from%20the%20history%20of%20Koji%20mold%20and%20exploration%20of%20its%20future&amp;author=M.%20Machida&amp;author=O.%20Yamada&amp;author=K.%20Gomi&amp;publication_year=2008&amp;journal=DNA%20Res.&amp;volume=15&amp;pages=173-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsn020" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsn020" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsn020"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18820080" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Zhang</div>
               <div class="given-names">H.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Rokas</div>
               <div class="given-names">A.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Slot</div>
               <div class="given-names">J.C.</div>
            </div>
          </div>
          <div class="article-title">Two different secondary metabolism gene clusters occupied the same ancestral locus in fungal dermatophytes of the arthrodermataceae</div>
          <div class="source">PLoS One</div> 
          <div class="year">2012</div>
          <div class="volume">7</div>
          <div class="fpage">e41903</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20different%20secondary%20metabolism%20gene%20clusters%20occupied%20the%20same%20ancestral%20locus%20in%20fungal%20dermatophytes%20of%20the%20arthrodermataceae&amp;author=H.%20Zhang&amp;author=A.%20Rokas&amp;author=J.C.%20Slot&amp;publication_year=2012&amp;journal=PLoS%20One&amp;volume=7&amp;pages=e41903" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0041903" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0041903" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0041903"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22860027" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSU010C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSU010C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Andersen</div>
               <div class="given-names">M.R.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Nielsen</div>
               <div class="given-names">J.B.</div>
            </div>
            <div class="name string-name">
              <div class="surname">Klitgaard</div>
               <div class="given-names">A.</div>
            </div>
            et al.  
          </div>
          <div class="article-title">Accurate prediction of secondary metabolite gene clusters in filamentous fungi</div>
          <div class="source">Proc Natl Acad Sci USA</div> 
          <div class="year">2013</div>
          <div class="volume">110</div>
          <div class="fpage">E99</div>
          <div class="lpage">107</div>
        <div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20prediction%20of%20secondary%20metabolite%20gene%20clusters%20in%20filamentous%20fungi&amp;author=M.R.%20Andersen&amp;author=J.B.%20Nielsen&amp;author=A.%20Klitgaard&amp;publication_year=2013&amp;journal=Proc%20Natl%20Acad%20Sci%20USA&amp;volume=110&amp;pages=E99-107" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1205532110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1205532110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1205532110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23248299" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2014. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution License (<a class="link link-uri" href="http://creativecommons.org/licenses/by/4.0/" target="_blank">http://creativecommons.org/licenses/by/4.0/</a>), which permits unrestricted reuse, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=2754535&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/21/4/10.1093_dnares_dsu010/2/dsu010_Supplementary_Data.zip?Expires=2147483647&amp;Signature=MPiG6gc9hUdygTRnGANiyQgA7d8IpwlJMFM~v9-F7bK2pcJoH7ZgE0EsTi3yY56pjA-paZpYbdY18FI7rR9CHSSMngi-qHrGa4vd7pQr0Mc0u5QSIf-DMyDUR0xMFMI1a2Zm~koOiqyHEthZYvNNyleD9NYlf2zn7yFaWqTCk5Mrt9TUB6W84S79etUoGJlJThjWyuKO~XtL3Eiv1ahciIwfegnGKncFUbO0nF4EhNDB1dvl6wHEdlSzllFLh0b5YSPw1U3q0h7roGA9E4JS0wHc~wDa1hOhJZGBzn-eAOeMXIXx2-rAFhqatyWRWNTGau4jy1n5U3VSdyyni1gybQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">449</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344243900010&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">14</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsu010"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="2754535"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000344243900010&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f21%2f4%2f447%2f2754535">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000344243900010&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (14)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F21%2F4%2F447%2F2754535" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/2754535'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Motif-Independent Prediction of a Secondary Metabolism Gene Cluster Using Comparative Genomics: Application to Sequenced Genomes of Aspergillus and Ten Other Filamentous Fungal Species" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsu010");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f21%2f4%2f447%2f2754535">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

