

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/3/209/474733","@type":"ScholarlyArticle","name":"Accurate Measurement of the Relative Abundance of Different DNA Species in Complex DNA Mixtures","datePublished":"2012-02-14","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/3","@type":"PublicationIssue","issueNumber":"3","datePublished":"2012-06-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss002","keywords":["competitive PCR","relative quantitation","transcription"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/3/209/474733","author":[{"name":"Jeong, Sangkyun","affiliation":"1  Medical Research Division ,  Korea Institute of Oriental Medicine ,  483 Exporo, Yusong-gu, Daejon 305-811 ,  Republic of  Korea","@type":"Person"},{"name":"Yu, Hyunjoo","affiliation":"1  Medical Research Division ,  Korea Institute of Oriental Medicine ,  483 Exporo, Yusong-gu, Daejon 305-811 ,  Republic of  Korea","@type":"Person"},{"name":"Pfeifer, Karl","affiliation":"2  Laboratory of Mammalian Genes and Development ,  National Institutes of Child Health and Development, National Institute of Health ,  Bethesda, MD 20892 ,  USA","@type":"Person"}],"description":"Abstract.  A molecular tool that can compare the abundances of different DNA sequences is necessary for comparing intergenic or interspecific gene expression. W","pageStart":"209","pageEnd":"218","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/3/1/m_cover.gif?Expires=2147483647&Signature=Fdyy0J9TWIOvyTsobWsaSe0-4dpdJpPal81dGnPphMlVPQ-~doKkHPlJp1UwhE6iBMO5EiCDjakz8lsMXANxWPYyveBKgVvoq77bYMGovqWIYBOCCELrdx0j6HVYZwX9Y74Ddg90XP77LfXNnNk59wdvjztMTMkN-3mdBevmXsNmZKVV5PmBqkKimC9vUcC6~0jrzJOVoAsfn74p43Pf6SY8Ru3oEUvpmOEuP9brDQjSX9QkiRbtOgqGZ2ApS-2m29LaARntNx2YmeI-Qi7~alsILkg5AbRepwVxxLytg4b29~~Z7IOxt-m0JR0e1LCuqAZKEhE4gn5lRwgzeExSjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Accurate Measurement of the Relative Abundance of Different DNA Species in Complex DNA Mixtures","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/3/1/m_cover.gif?Expires=2147483647&Signature=Fdyy0J9TWIOvyTsobWsaSe0-4dpdJpPal81dGnPphMlVPQ-~doKkHPlJp1UwhE6iBMO5EiCDjakz8lsMXANxWPYyveBKgVvoq77bYMGovqWIYBOCCELrdx0j6HVYZwX9Y74Ddg90XP77LfXNnNk59wdvjztMTMkN-3mdBevmXsNmZKVV5PmBqkKimC9vUcC6~0jrzJOVoAsfn74p43Pf6SY8Ru3oEUvpmOEuP9brDQjSX9QkiRbtOgqGZ2ApS-2m29LaARntNx2YmeI-Qi7~alsILkg5AbRepwVxxLytg4b29~~Z7IOxt-m0JR0e1LCuqAZKEhE4gn5lRwgzeExSjA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Accurate Measurement of the Relative Abundance of Different DNA Species in Complex DNA Mixtures" />
    <meta property="og:description" content="Abstract.  A molecular tool that can compare the abundances of different DNA sequences is necessary for comparing intergenic or interspecific gene expression. W" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/3/209/474733" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-02-14" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Jeong, Sangkyun" /><meta name="citation_author_institution" content="Medical Research Division, Korea Institute of Oriental Medicine, 483 Exporo, Yusong-gu, Daejon 305-811, Republic ofKorea" /><meta name="citation_author" content="Yu, Hyunjoo" /><meta name="citation_author_institution" content="Medical Research Division, Korea Institute of Oriental Medicine, 483 Exporo, Yusong-gu, Daejon 305-811, Republic ofKorea" /><meta name="citation_author" content="Pfeifer, Karl" /><meta name="citation_author_institution" content="Laboratory of Mammalian Genes and Development, National Institutes of Child Health and Development, National Institute of Health, Bethesda, MD 20892, USA" /><meta name="citation_title" content="Accurate Measurement of the Relative Abundance of Different DNA Species in Complex DNA Mixtures" /><meta name="citation_firstpage" content="209" /><meta name="citation_lastpage" content="218" /><meta name="citation_doi" content="10.1093/dnares/dss002" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="3" /><meta name="citation_publication_date" content="2012/06/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Genetics of human gene expression: mapping DNA variants that influence gene expression; Nat. Rev. Genet.;  citation_year=2009;  citation_volume=10;  citation_pages=595-604; " /><meta name="citation_reference" content="citation_title=Quantitative monitoring of gene expression patterns with a complementary DNA microarray; Science;  citation_year=1995;  citation_volume=270;  citation_pages=467-70; " /><meta name="citation_reference" content="citation_title=RNA-seq: an assessment of technical reproducibility and comparison with gene expression arrays; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1509-17; " /><meta name="citation_reference" content="citation_title=Evaluation of gene expression measurements from commercial microarray platforms; Nucleic Acids Res.;  citation_year=2003;  citation_volume=31;  citation_pages=5676-84; " /><meta name="citation_reference" content="citation_title=Large scale real-time PCR validation on gene expression measurements from two commercial long-oligonucleotide microarrays; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=59" /><meta name="citation_reference" content="citation_title=Competitive PCR for precise nucleic acid quantification; Nat. Protoc.;  citation_year=2007;  citation_volume=2;  citation_pages=2092-104; " /><meta name="citation_reference" content="citation_title=Competitive PCR; Nature;  citation_year=1992;  citation_volume=359;  citation_pages=557-8; " /><meta name="citation_reference" content="citation_title=A high-throughput gene expression analysis technique using competitive PCR and matrix-assisted laser desorption ionization time-of-flight MS; Proc. Natl. Acad. Sci. USA;  citation_year=2003;  citation_volume=100;  citation_pages=3059-64; " /><meta name="citation_reference" content="citation_title=Digital single-nucleotide polymorphism analysis for allelic imbalance; Methods Mol. Med.;  citation_year=2005;  citation_volume=103;  citation_pages=137-41; " /><meta name="citation_reference" content="citation_title=Accurate quantitation of allele-specific expression patterns by analysis of DNA melting; Genome Res.;  citation_year=2007;  citation_volume=17;  citation_pages=1093-100; " /><meta name="citation_reference" content="citation_title=Relative quantitation of restriction fragment length polymorphic DNAs via DNA melting analysis provides an effective way to determine allele frequencies; Genomics;  citation_year=2009;  citation_volume=94;  citation_pages=355-61; " /><meta name="citation_reference" content="citation_title=The functional consequences of alternative promoter use in mammalian genomes; Trends Genet.;  citation_year=2008;  citation_volume=24;  citation_pages=167-77; " /><meta name="citation_reference" content="citation_title=Regulatory mechanisms at the mouse Igf2/H19 locus; Mol. Cell Biol.;  citation_year=2001;  citation_volume=21;  citation_pages=8189-96; " /><meta name="citation_reference" content="citation_title=Evolution of insulin-like growth factor II: characterization of the mouse IGF-II gene and identification of two pseudo-exons; DNA Cell Biol.;  citation_year=1990;  citation_volume=9;  citation_pages=725-35; " /><meta name="citation_reference" content="citation_title=Per cell normalization method for mRNA measurement by quantitative PCR and microarrays; BMC Genomics;  citation_year=2006;  citation_volume=7;  citation_pages=64" /><meta name="citation_reference" content="citation_title=Absolute quantification of the budding yeast transcriptome by means of competitive PCR between genomic and complementary DNAs; BMC Genomics;  citation_year=2008;  citation_volume=9;  citation_pages=574" /><meta name="citation_reference" content="citation_title=Large-scale copy number variants (CNVs): distribution in normal subjects and FISH/real-time qPCR analysis; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=167" /><meta name="citation_reference" content="citation_title=A novel technique for measuring variations in DNA copy-number: competitive genomic polymerase chain reaction; BMC Genomics;  citation_year=2007;  citation_volume=8;  citation_pages=206" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/3/209/1283467/dss002.pdf" /><meta name="description" content="Abstract.  A molecular tool that can compare the abundances of different DNA sequences is necessary for comparing intergenic or interspecific gene expression. W" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/3/209/474733" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/3/209/474733" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/3/209/474733");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss002';
    </script>
<title>
	Accurate Measurement of the Relative Abundance of Different DNA Species in Complex DNA Mixtures | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./474733" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BdwuPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJKZW9uZywgU2FuZ2t5dW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJNZWRpY2FsIFJlc2VhcmNoIERpdmlzaW9uLCBLb3JlYSBJbnN0aXR1dGUgb2YgT3JpZW50YWwgTWVkaWNpbmUsIDQ4MyBFeHBvcm8sIFl1c29uZy1ndSwgRGFlam9uIDMwNS04MTEsIFJlcHVibGljIG9mS29yZWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJZdSwgSHl1bmpvbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ik1lZGljYWwgUmVzZWFyY2ggRGl2aXNpb24sIEtvcmVhIEluc3RpdHV0ZSBvZiBPcmllbnRhbCBNZWRpY2luZSwgNDgzIEV4cG9ybywgWXVzb25nLWd1LCBEYWVqb24gMzA1LTgxMSwgUmVwdWJsaWMgb2ZLb3JlYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlBmZWlmZXIsIEthcmwiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJMYWJvcmF0b3J5IG9mIE1hbW1hbGlhbiBHZW5lcyBhbmQgRGV2ZWxvcG1lbnQsIE5hdGlvbmFsIEluc3RpdHV0ZXMgb2YgQ2hpbGQgSGVhbHRoIGFuZCBEZXZlbG9wbWVudCwgTmF0aW9uYWwgSW5zdGl0dXRlIG9mIEhlYWx0aCwgQmV0aGVzZGEsIE1EIDIwODkyLCBVU0EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkFjY3VyYXRlIE1lYXN1cmVtZW50IG9mIHRoZSBSZWxhdGl2ZSBBYnVuZGFuY2Ugb2YgRGlmZmVyZW50IEROQSBTcGVjaWVzIGluIENvbXBsZXggRE5BIE1peHR1cmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2ZpcnN0cGFnZSIgY29udGVudD0iMjA5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2xhc3RwYWdlIiBjb250ZW50PSIyMTgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZG9pIiBjb250ZW50PSIxMC4xMDkzL2RuYXJlcy9kc3MwMDIiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fam91cm5hbF90aXRsZSIgY29udGVudD0iRE5BIFJlc2VhcmNoIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfYWJicmV2IiBjb250ZW50PSJETkEgUmVzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3ZvbHVtZSIgY29udGVudD0iMTkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzdWUiIGNvbnRlbnQ9IjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGljYXRpb25fZGF0ZSIgY29udGVudD0iMjAxMi8wNi8wMSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9pc3NuIiBjb250ZW50PSIxMzQwLTI4MzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcHVibGlzaGVyIiBjb250ZW50PSJPeGZvcmQgVW5pdmVyc2l0eSBQcmVzcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWNzIG9mIGh1bWFuIGdlbmUgZXhwcmVzc2lvbjogbWFwcGluZyBETkEgdmFyaWFudHMgdGhhdCBpbmZsdWVuY2UgZ2VuZSBleHByZXNzaW9uOyBOYXQuIFJldi4gR2VuZXQuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NTk1LTYwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UXVhbnRpdGF0aXZlIG1vbml0b3Jpbmcgb2YgZ2VuZSBleHByZXNzaW9uIHBhdHRlcm5zIHdpdGggYSBjb21wbGVtZW50YXJ5IEROQSBtaWNyb2FycmF5OyBTY2llbmNlOyAgY2l0YXRpb25feWVhcj0xOTk1OyAgY2l0YXRpb25fdm9sdW1lPTI3MDsgIGNpdGF0aW9uX3BhZ2VzPTQ2Ny03MDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BLXNlcTogYW4gYXNzZXNzbWVudCBvZiB0ZWNobmljYWwgcmVwcm9kdWNpYmlsaXR5IGFuZCBjb21wYXJpc29uIHdpdGggZ2VuZSBleHByZXNzaW9uIGFycmF5czsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xNTA5LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdmFsdWF0aW9uIG9mIGdlbmUgZXhwcmVzc2lvbiBtZWFzdXJlbWVudHMgZnJvbSBjb21tZXJjaWFsIG1pY3JvYXJyYXkgcGxhdGZvcm1zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDM7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz01Njc2LTg0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1MYXJnZSBzY2FsZSByZWFsLXRpbWUgUENSIHZhbGlkYXRpb24gb24gZ2VuZSBleHByZXNzaW9uIG1lYXN1cmVtZW50cyBmcm9tIHR3byBjb21tZXJjaWFsIGxvbmctb2xpZ29udWNsZW90aWRlIG1pY3JvYXJyYXlzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTU5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGV0aXRpdmUgUENSIGZvciBwcmVjaXNlIG51Y2xlaWMgYWNpZCBxdWFudGlmaWNhdGlvbjsgTmF0LiBQcm90b2MuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI7ICBjaXRhdGlvbl9wYWdlcz0yMDkyLTEwNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGV0aXRpdmUgUENSOyBOYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTE5OTI7ICBjaXRhdGlvbl92b2x1bWU9MzU5OyAgY2l0YXRpb25fcGFnZXM9NTU3LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgaGlnaC10aHJvdWdocHV0IGdlbmUgZXhwcmVzc2lvbiBhbmFseXNpcyB0ZWNobmlxdWUgdXNpbmcgY29tcGV0aXRpdmUgUENSIGFuZCBtYXRyaXgtYXNzaXN0ZWQgbGFzZXIgZGVzb3JwdGlvbiBpb25pemF0aW9uIHRpbWUtb2YtZmxpZ2h0IE1TOyBQcm9jLiBOYXRsLiBBY2FkLiBTY2kuIFVTQTsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0xMDA7ICBjaXRhdGlvbl9wYWdlcz0zMDU5LTY0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWdpdGFsIHNpbmdsZS1udWNsZW90aWRlIHBvbHltb3JwaGlzbSBhbmFseXNpcyBmb3IgYWxsZWxpYyBpbWJhbGFuY2U7IE1ldGhvZHMgTW9sLiBNZWQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTEwMzsgIGNpdGF0aW9uX3BhZ2VzPTEzNy00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWNjdXJhdGUgcXVhbnRpdGF0aW9uIG9mIGFsbGVsZS1zcGVjaWZpYyBleHByZXNzaW9uIHBhdHRlcm5zIGJ5IGFuYWx5c2lzIG9mIEROQSBtZWx0aW5nOyBHZW5vbWUgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT0xNzsgIGNpdGF0aW9uX3BhZ2VzPTEwOTMtMTAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SZWxhdGl2ZSBxdWFudGl0YXRpb24gb2YgcmVzdHJpY3Rpb24gZnJhZ21lbnQgbGVuZ3RoIHBvbHltb3JwaGljIEROQXMgdmlhIEROQSBtZWx0aW5nIGFuYWx5c2lzIHByb3ZpZGVzIGFuIGVmZmVjdGl2ZSB3YXkgdG8gZGV0ZXJtaW5lIGFsbGVsZSBmcmVxdWVuY2llczsgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9OTQ7ICBjaXRhdGlvbl9wYWdlcz0zNTUtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBmdW5jdGlvbmFsIGNvbnNlcXVlbmNlcyBvZiBhbHRlcm5hdGl2ZSBwcm9tb3RlciB1c2UgaW4gbWFtbWFsaWFuIGdlbm9tZXM7IFRyZW5kcyBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjQ7ICBjaXRhdGlvbl9wYWdlcz0xNjctNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRvcnkgbWVjaGFuaXNtcyBhdCB0aGUgbW91c2UgSWdmMi9IMTkgbG9jdXM7IE1vbC4gQ2VsbCBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTgxODktOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbiBvZiBpbnN1bGluLWxpa2UgZ3Jvd3RoIGZhY3RvciBJSTogY2hhcmFjdGVyaXphdGlvbiBvZiB0aGUgbW91c2UgSUdGLUlJIGdlbmUgYW5kIGlkZW50aWZpY2F0aW9uIG9mIHR3byBwc2V1ZG8tZXhvbnM7IEROQSBDZWxsIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTkwOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz03MjUtMzU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBlciBjZWxsIG5vcm1hbGl6YXRpb24gbWV0aG9kIGZvciBtUk5BIG1lYXN1cmVtZW50IGJ5IHF1YW50aXRhdGl2ZSBQQ1IgYW5kIG1pY3JvYXJyYXlzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTY0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWJzb2x1dGUgcXVhbnRpZmljYXRpb24gb2YgdGhlIGJ1ZGRpbmcgeWVhc3QgdHJhbnNjcmlwdG9tZSBieSBtZWFucyBvZiBjb21wZXRpdGl2ZSBQQ1IgYmV0d2VlbiBnZW5vbWljIGFuZCBjb21wbGVtZW50YXJ5IEROQXM7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9NTc0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TGFyZ2Utc2NhbGUgY29weSBudW1iZXIgdmFyaWFudHMgKENOVnMpOiBkaXN0cmlidXRpb24gaW4gbm9ybWFsIHN1YmplY3RzIGFuZCBGSVNIL3JlYWwtdGltZSBxUENSIGFuYWx5c2lzOyBCTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9ODsgIGNpdGF0aW9uX3BhZ2VzPTE2NyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgbm92ZWwgdGVjaG5pcXVlIGZvciBtZWFzdXJpbmcgdmFyaWF0aW9ucyBpbiBETkEgY29weS1udW1iZXI6IGNvbXBldGl0aXZlIGdlbm9taWMgcG9seW1lcmFzZSBjaGFpbiByZWFjdGlvbjsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0yMDYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTkvMy8yMDkvMTI4MzQ2Ny9kc3MwMDIucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEEgbW9sZWN1bGFyIHRvb2wgdGhhdCBjYW4gY29tcGFyZSB0aGUgYWJ1bmRhbmNlcyBvZiBkaWZmZXJlbnQgRE5BIHNlcXVlbmNlcyBpcyBuZWNlc3NhcnkgZm9yIGNvbXBhcmluZyBpbnRlcmdlbmljIG9yIGludGVyc3BlY2lmaWMgZ2VuZSBleHByZXNzaW9uLiBXIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00NzQ3MzMmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkYMnSABRcmPs35VyN5u+aqx6kcQk=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f209%2f474733" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f209%2f474733" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=18620" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/3"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/3/1/m_cover.gif?Expires=2147483647&amp;Signature=Fdyy0J9TWIOvyTsobWsaSe0-4dpdJpPal81dGnPphMlVPQ-~doKkHPlJp1UwhE6iBMO5EiCDjakz8lsMXANxWPYyveBKgVvoq77bYMGovqWIYBOCCELrdx0j6HVYZwX9Y74Ddg90XP77LfXNnNk59wdvjztMTMkN-3mdBevmXsNmZKVV5PmBqkKimC9vUcC6~0jrzJOVoAsfn74p43Pf6SY8Ru3oEUvpmOEuP9brDQjSX9QkiRbtOgqGZ2ApS-2m29LaARntNx2YmeI-Qi7~alsILkg5AbRepwVxxLytg4b29~~Z7IOxt-m0JR0e1LCuqAZKEhE4gn5lRwgzeExSjA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/3">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 3</div>

            </a>

        </div>
        <div class="ii-pub-date">
June 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="7156185">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7156185">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7156189">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7156189">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7156196">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7156196">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7156214">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7156214">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7156246">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7156246">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7156253">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7156253">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7156255">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7156255">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="7156257">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#7156257">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="empty arrow">&nbsp;</li>
            <li class="next arrow">
            <a href="/dnaresearch/article/19/3/219/474767">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Accurate Measurement of the Relative Abundance of Different DNA Species in Complex DNA Mixtures
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sangkyun Jeong</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Medical Research Division</div>, <div class="insititution">Korea Institute of Oriental Medicine</div>, <div class="addr-line">483 Exporo, Yusong-gu, Daejon 305-811</div>, <div class="country">Republic of</div><div class="country">Korea</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +82 42-868-9314. Fax. +82 42-868-9480. Email: <a href="mailto:skjeong@kiom.re.kr" target="_blank">skjeong@kiom.re.kr</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sangkyun+Jeong">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Jeong S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Jeong S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sangkyun Jeong</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hyunjoo Yu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">Medical Research Division</div>, <div class="insititution">Korea Institute of Oriental Medicine</div>, <div class="addr-line">483 Exporo, Yusong-gu, Daejon 305-811</div>, <div class="country">Republic of</div><div class="country">Korea</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hyunjoo+Yu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yu H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yu H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hyunjoo Yu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Karl Pfeifer</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Laboratory of Mammalian Genes and Development</div>, <div class="insititution">National Institutes of Child Health and Development, National Institute of Health</div>, <div class="addr-line">Bethesda, MD 20892</div>, <div class="country">USA</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Karl+Pfeifer">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Pfeifer K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Pfeifer K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Karl Pfeifer</a>
                                </span>

                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 3, 1 June 2012, Pages 209218, <a href='https://doi.org/10.1093/dnares/dss002'>https://doi.org/10.1093/dnares/dss002</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">14 February 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">26 October 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">06 January 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="474733" href="/dnaresearch/article-pdf/19/3/209/1283467/dss002.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Sangkyun Jeong, Hyunjoo Yu, Karl Pfeifer;  Accurate Measurement of the Relative Abundance of Different DNA Species in Complex DNA Mixtures, <em>DNA Research</em>, Volume 19, Issue 3, 1 June 2012, Pages 209218, <a href='https://doi.org/10.1093/dnares/dss002'>https://doi.org/10.1093/dnares/dss002</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/474733?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/474733?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/474733?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/474733?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/474733?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/474733?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/474733?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Accurate%20Measurement%20of%20the%20Relative%20Abundance%20of%20Different%20DNA%20Species%20in%20Complex%20DNA%20Mixtures&amp;publicationDate=2012-02-14&amp;volumeNum=19&amp;issueNum=3&amp;author=Jeong%2C%20Sangkyun%3B%20Yu%2C%20Hyunjoo&amp;startPage=209&amp;endPage=218&amp;contentId=10.1093%2Fdnares%2Fdss002&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f209%2f474733" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=7156185 id="7156185" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>A molecular tool that can compare the abundances of different DNA sequences is necessary for comparing intergenic or interspecific gene expression. We devised and verified such a tool using a quantitative competitive polymerase chain reaction approach. For this approach, we adapted a competitor array, an artificially made plasmid DNA in which all the competitor templates for the target DNAs are arranged with a defined ratio, and melting analysis for allele quantitation for accurate quantitation of the fractional ratios of competitively amplified DNAs. Assays on two sets of DNA mixtures with explicitly known compositional structures of the test sequences were performed. The resultant average relative errors of 0.059 and 0.021 emphasize the highly accurate nature of this method. Furthermore, the method's capability of obtaining biological data is demonstrated by the fact that it can illustrate the tissue-specific quantitative expression signatures of the three housekeeping genes <em>G6pdx</em>, <em>Ubc</em>, and <em>Rps27</em> by using the forms of the relative abundances of their transcripts, and the differential preferences of <em>Igf2</em> enhancers for each of the multiple <em>Igf2</em> promoters for the transcription.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;competitive PCR&quot;">competitive PCR</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;relative quantitation&quot;">relative quantitation</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="transcription">transcription</a></div>                            <h2 scrollto-destination=7156189 id="7156189" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>Gene transcription levels have been measured by methods based on Northern analysis or reverse transcription (RT)polymerase chain reaction (PCR) assays. These approaches are generally effective for comparing the RNA levels of any gene among many samples. However, these methods are unsuitable for comparing the quantitation data generated in different places or times, primarily due to the paucity of quantitation standardization. Comparing the expression levels of different genes is even more problematic, even though this task is important for understanding the coordinative nature of gene expressions and the relative strength of regulatory elements.<sup><a reveal-id="DSS002C1" data-open="DSS002C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Microarray technology and recent advancements in sequencing technology, such as RNAseq, have opened the way for a thorough examination of the expression of many RNAs; they also facilitate the visualization of relative expression levels of multiple genes.<sup><a reveal-id="DSS002C2" data-open="DSS002C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSS002C3" data-open="DSS002C3" class="link link-ref link-reveal xref-bibr">3</a></sup> Nonetheless, these assays are yet to be tailored to attain sufficient accuracy or reproducibility between different assay platforms; they are also unsuitable for examining a small number of genes in numerous samples.<sup><a reveal-id="DSS002C4" data-open="DSS002C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSS002C5" data-open="DSS002C5" class="link link-ref link-reveal xref-bibr">5</a></sup></p><p>The quantitative competitive PCR (qcPCR) method measures the amount of target DNA of interest relative to the amount of an artificially made DNA competitor. With this measurement, the absolute quantity of the target DNA can be determined if the amount of the corresponding competitor is known.<sup><a reveal-id="DSS002C6" data-open="DSS002C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS002C7" data-open="DSS002C7" class="link link-ref link-reveal xref-bibr">7</a></sup> A single-nucleotide difference between two otherwise identical DNAs is the best competition structure that warrants the equal amplification efficiency of these dimorphic DNAs in the PCR. The resultant PCR amplicons have been analysed for the allele ratios in the samples using a wide range of application platforms including Maldi-TOF, pyrosequencing, and real-time DNA melting.<sup><a reveal-id="DSS002C8  DSS002C11" data-open="DSS002C8  DSS002C11" class="link link-ref link-reveal xref-bibr">811</a></sup> However, qcPCR approach needs to deal with some inevitable uncertainties. The quantitation of the competitor templates or samples of interest is largely performed in a spectrophotometric manner; that is, the method takes into account the common measurement biases that occur between measurements and between laboratories.</p><p>In this study, we present an alternative method of nucleic acid quantitation based on a qcPCR approach. With this method, the amount of a given sequence in a mixture such as cDNA or genomic DNA (gDNA) is determined in relation to the amount of another sequence in the mixture. To accomplish this task, we introduced a competitor array as a novel reaction component in the qcPCR. The competitor array is an artificially made plasmid DNA in which the competitor templates for the target DNAs in question are arranged with a defined aspect ratio. This structure of the competitor array enables one to infer the quantitative relation between two or more DNA sequences if the relative quantity of each sequence to its respective competitor templates in the competitor array is determined. We also exploited high accuracy and precision of the melting analysis for allele quantitation (MAAQ) method in determining the relative quantity of the target DNA with respect to the competitor template. The MAAQ method utilizes the differential melting patterns of either the fluorescent resonance energy transfer (FRET) probe encompassing on a single-nucleotide variation or restriction fragment length polymorphic DNAs. This method mathematically analyses the melting curve of a sample to extract the constitutional fraction with which each allelic version contributes to the curve.<sup><a reveal-id="DSS002C10" data-open="DSS002C10" class="link link-ref link-reveal xref-bibr">10</a>,<a reveal-id="DSS002C11" data-open="DSS002C11" class="link link-ref link-reveal xref-bibr">11</a></sup> It offers a simple and rapid procedure as it does not require post-PCR purification or additional biochemical reactions while ensuring high accuracy and precision. In addition, the MAAQ analysis can be performed with a relatively less expensive device than other methods. Various exemplary assays are presented to emphasize the validity and versatility of this method in the field of nucleic acid quantitation.</p>                            <h2 scrollto-destination=7156196 id="7156196" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=7156198 id="7156198" class="section-title" data-legacy-id=s2a>2.1. Plasmids</h3>
<p>The plasmid DNAs containing a single target or competitor template were made by TA cloning of the corresponding PCR amplicons into pCR2.1 (Invitrogen) so that the inserted fragments could be liberated with <em>Eco</em>RI restriction. Competitor template was generated by PCR amplification by applying a mismatch-carrying primer. Plasmid DNAs containing multiple template sequences which were used for an equimolar mixture of the target and competitor template and for the competitor arrays were constructed by a means of a single or repeated subcloning process. For the subcloning, a <em>Spe</em>I/<em>Xba</em>I fragment of a plasmid was placed into the <em>Xba</em>I or <em>Spe</em>I site of another plasmid. Plasmids that contained any form of an inverted repeat were avoided. Further information on the plasmids used in this study is presented in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>.</p>                            <h3 scrollto-destination=7156202 id="7156202" class="section-title" data-legacy-id=s2b>2.2. RT and PCR</h3>
<p>RT was carried out on 1 g of the total RNA with an iScript cDNA synthesis kit (BioRad) in accordance with the manufacturer's instructions. PCR was performed with 0.2 l of the reverse transcribed cDNA template, 20 ng of gDNA or 0.1 pg of plasmid DNA for appropriate analyses and 0.5 M of corresponding primer pairs: G6pd-1F and G6pd-7R for <em>G6pdx</em>; Ubc-1F and Ubc-4R for <em>Ubc</em>; Rps-4F and Rps-2R for <em>Rps27</em>; Igf2-E1F1 and Igf2-E4R1 for the <em>Igf2</em> P1 transcript; Igf2-E2F1 and Igf2-E4R1 for the <em>Igf2</em> P2 transcript; Igf2-E3F1 and Igf2-E4R1 for the <em>Igf2</em> P3 transcript; and Igf2-E4F1(GTACCAATGGGGATCCCAGT) and Igf2-E4R1 for the <em>Igf2</em> total transcript. Information on the primer sequences is presented in <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>. The competitor arrays were prepared by cleaving the corresponding plasmid and subsequently diluting it in water containing phage DNA (1 ng/l). For the competitive PCR, each template (except for the references) was mixed with its appropriate competitor array and amplified by PCR by using 40 cycles at 94C for 10 s, 56C for 20 s, and 72C for 30 s.</p>                            <h3 scrollto-destination=7156206 id="7156206" class="section-title" data-legacy-id=s2c>2.3. Melting analysis</h3>
<p>The relative ratios of the target DNA to its competitor in the PCR amplicon were determined as previously described. Briefly, 5 l of PCR amplicons were mixed with 5 l of 10 mM ethylenediaminetetraacetic acid and with 0.4 M of FRET probes, Igf2-S1 and Igf2-A1 (see Fig.<a reveal-id="DSS002F1" data-open="DSS002F1" class="link link-reveal link-table xref-fig">1</a>b) for <em>Igf2</em>; G6pdx-S, Red640-5-GCACTG TTGGTGGAAGATGTCA-3, and G6pdx-A, 5-CACACG GATGACCAGCTCATTACG-3-FITC, for <em>G6pdx</em>; Ubc-S, Red640-5-TACGTCTGTCTTCCTGTTTGAC-3, and Ubc-A, 5-GGCTCTTTTTAGATACTGTGGTGAGG-3-FITC, for <em>Ubc</em>; and Rps27-A, 5-CCTTCTGATGA ATGTGGTGCTGGAG-3-FITC and Rps27-S, Red640-5-TCTGGGAAGCCACTTTGACAG-3, for <em>Rps27</em>. The DNA melting was performed with Roche LightCycler 2.0 under the following conditions: 30 s at 95C; and 8 min at either 45C (for <em>Igf2</em>, <em>G6pdx</em>, and <em>Ubc</em>) or 44C (for <em>Rps27</em>) at 0.1C stepwise increments in temperature until the probe's melting is completed. The background-subtracted fluorescence values (640/Back-530) from the Red640 dye were extracted with the aid of LightCycler Software 4.05 and then submitted for calculation as previously reported.<sup><a reveal-id="DSS002C11" data-open="DSS002C11" class="link link-ref link-reveal xref-bibr">11</a></sup> The relative abundance of two sequences in each sample was determined from the ratio of the T/C values of the two comparing sequences after reflecting the copy numbers of their corresponding competitor templates in the competitor array used. </p>                        <a id="7156211" scrollto-destination="7156211"></a>
<div id="DSS002F1" data-id="DSS002F1" class="fig fig-section"><div class="label fig-label">Figure1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss002/2/m_dss00201.jpeg?Expires=2147483647&Signature=QVTEtgkQJpRmsabru7FRaf58JhGZ3c8d62vcWstrp6iokRKEhXlKDcNrqUZx4mpVsOZnT~lESVzxZrxVs7vMrutRMN1Qya-9lAl5kJHR~w38xGwZIakJ51ZIq977DfbcupQGYqrBd5TQ1yxrSgdQzl1k7oqAvHJdtk5yjw6bZBGba0xwFdKmkDR3MfSDFm0u0CcAeczlIqKqkKmpFYiL-2FFDLQH4KTA2~QE0RnJCJlPbeRFeLmI6PkAvQUnstPXbH-sNZ~TjW08dsY4hj9h1PeiLc9GFEUeDhXPIN3OsSAfc2SU8tzcc9mIh0N3IQZk0dy-Aumc0cP1Ng2083EcMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Determination of relative abundances of different DNA sequences. (a). A schematic representation of the entire procedure for determining the relative abundances among three sequences (A, B, and C) in a cDNA sample. The respective competitor templates (A, B, and C) which are arranged in a competitor array with a given aspect ratio of :: are liberated by endonuclease restriction, mixed with cDNAs, and subjected to PCR reactions for each sequence of interest. The relative quantities of the target sequences with respect to the corresponding competitor in each amplicon, n, p, and q, as assessed by MAAQ, are fed into a calculation based on the aspect ratio of the competitor template in the competitor array to yield the relative abundances among the target sequences. (b). A set of artificial DNA mixtures constituted by the restriction of the plasmids (AJ) that contain cDNA fragments of three housekeeping genes with variable copy numbers as denoted under the name of the mixtures and fractionated on an agarose gel. The identity of each DNA is denoted on the left. (c). The relative abundances of two cDNA sequences were determined and scatter-plotted against the expected values on the log scale axes." path-from-xml="dss00201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7156211/dss00201.jpeg" path-from-xml="dss00201.jpeg" target="_blank">View large</a><a section="7156211" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss002/2/dss00201.jpeg?Expires=2147483647&Signature=ubGIk~dVN7g3VKWEYphrQzQZia~AeI4KpQi57UKHEeS67jdsZQuEtPBmb4mOK~ttVOE4VC3aTTz9yMDJ9mB3zG5E-3LypWIL4NJwkv6m6w3waE8Is0nKscgah6r8DO7GxFEX46uKB~T1GtfZYBocE8r-u6uQqBLhWiXD85zu1bzk1zQkyEH1qfbEtmu-jr35n-nDAk6g1dYlgOOkWWF1NiHuz7bJe85AksZStAORVxIo31dBKCWJ0gymIgLQc8CmYDaXcxWkMHf1u2TVBy6TAUveqkYyz-~pIuKzZ6Aq7iMQ-YFDNOw-O6KHdJtNSFEE8Og7AocKVoLijX6-43wylQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7156211&ar=474733&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Determination of relative abundances of different DNA sequences. (a). A schematic representation of the entire procedure for determining the relative abundances among three sequences (A, B, and C) in a cDNA sample. The respective competitor templates (A, B, and C) which are arranged in a competitor array with a given aspect ratio of :: are liberated by endonuclease restriction, mixed with cDNAs, and subjected to PCR reactions for each sequence of interest. The relative quantities of the target sequences with respect to the corresponding competitor in each amplicon, n, p, and q, as assessed by MAAQ, are fed into a calculation based on the aspect ratio of the competitor template in the competitor array to yield the relative abundances among the target sequences. (b). A set of artificial DNA mixtures constituted by the restriction of the plasmids (AJ) that contain cDNA fragments of three housekeeping genes with variable copy numbers as denoted under the name of the mixtures and fractionated on an agarose gel. The identity of each DNA is denoted on the left. (c). The relative abundances of two cDNA sequences were determined and scatter-plotted against the expected values on the log scale axes.</p></div></div><div content-id="DSS002F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss002/2/m_dss00201.jpeg?Expires=2147483647&Signature=QVTEtgkQJpRmsabru7FRaf58JhGZ3c8d62vcWstrp6iokRKEhXlKDcNrqUZx4mpVsOZnT~lESVzxZrxVs7vMrutRMN1Qya-9lAl5kJHR~w38xGwZIakJ51ZIq977DfbcupQGYqrBd5TQ1yxrSgdQzl1k7oqAvHJdtk5yjw6bZBGba0xwFdKmkDR3MfSDFm0u0CcAeczlIqKqkKmpFYiL-2FFDLQH4KTA2~QE0RnJCJlPbeRFeLmI6PkAvQUnstPXbH-sNZ~TjW08dsY4hj9h1PeiLc9GFEUeDhXPIN3OsSAfc2SU8tzcc9mIh0N3IQZk0dy-Aumc0cP1Ng2083EcMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Determination of relative abundances of different DNA sequences. (a). A schematic representation of the entire procedure for determining the relative abundances among three sequences (A, B, and C) in a cDNA sample. The respective competitor templates (A, B, and C) which are arranged in a competitor array with a given aspect ratio of :: are liberated by endonuclease restriction, mixed with cDNAs, and subjected to PCR reactions for each sequence of interest. The relative quantities of the target sequences with respect to the corresponding competitor in each amplicon, n, p, and q, as assessed by MAAQ, are fed into a calculation based on the aspect ratio of the competitor template in the competitor array to yield the relative abundances among the target sequences. (b). A set of artificial DNA mixtures constituted by the restriction of the plasmids (AJ) that contain cDNA fragments of three housekeeping genes with variable copy numbers as denoted under the name of the mixtures and fractionated on an agarose gel. The identity of each DNA is denoted on the left. (c). The relative abundances of two cDNA sequences were determined and scatter-plotted against the expected values on the log scale axes." path-from-xml="dss00201.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7156211/dss00201.jpeg" path-from-xml="dss00201.jpeg" target="_blank">View large</a><a section="7156211" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss002/2/dss00201.jpeg?Expires=2147483647&Signature=ubGIk~dVN7g3VKWEYphrQzQZia~AeI4KpQi57UKHEeS67jdsZQuEtPBmb4mOK~ttVOE4VC3aTTz9yMDJ9mB3zG5E-3LypWIL4NJwkv6m6w3waE8Is0nKscgah6r8DO7GxFEX46uKB~T1GtfZYBocE8r-u6uQqBLhWiXD85zu1bzk1zQkyEH1qfbEtmu-jr35n-nDAk6g1dYlgOOkWWF1NiHuz7bJe85AksZStAORVxIo31dBKCWJ0gymIgLQc8CmYDaXcxWkMHf1u2TVBy6TAUveqkYyz-~pIuKzZ6Aq7iMQ-YFDNOw-O6KHdJtNSFEE8Og7AocKVoLijX6-43wylQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7156211&ar=474733&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00201.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Determination of relative abundances of different DNA sequences. (a). A schematic representation of the entire procedure for determining the relative abundances among three sequences (A, B, and C) in a cDNA sample. The respective competitor templates (A, B, and C) which are arranged in a competitor array with a given aspect ratio of :: are liberated by endonuclease restriction, mixed with cDNAs, and subjected to PCR reactions for each sequence of interest. The relative quantities of the target sequences with respect to the corresponding competitor in each amplicon, n, p, and q, as assessed by MAAQ, are fed into a calculation based on the aspect ratio of the competitor template in the competitor array to yield the relative abundances among the target sequences. (b). A set of artificial DNA mixtures constituted by the restriction of the plasmids (AJ) that contain cDNA fragments of three housekeeping genes with variable copy numbers as denoted under the name of the mixtures and fractionated on an agarose gel. The identity of each DNA is denoted on the left. (c). The relative abundances of two cDNA sequences were determined and scatter-plotted against the expected values on the log scale axes.</p></div></div>                            <h2 scrollto-destination=7156214 id="7156214" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=7156216 id="7156216" class="section-title" data-legacy-id=s3a>3.1. Determining the relative abundance of DNAs in simple mixtures</h3>
<p>Figure<a reveal-id="DSS002F1" data-open="DSS002F1" class="link link-reveal link-table xref-fig">1</a>a depicts the typical procedure for determining the relative abundances of the sequences of interest in a mixture. To demonstrate the feasibility of this quantitation approach, we used it to determine the relative quantity of a cDNA sequence with respect to another sequence in the artificially made cDNA mixtures. A set of cDNA mixtures was composed through the construction of 10 plasmids (AJ); they contained variable numbers of each of the three cDNA fragments of mouse <em>Ubc</em>, <em>Rps27</em>, and <em>G6pdx</em> genes. Plasmids H and I were from independent clones with the same sequence composition. The relative abundances of the sequences in each plasmid are therefore explicitly known (Fig.<a reveal-id="DSS002F1" data-open="DSS002F1" class="link link-reveal link-table xref-fig">1</a>b). Another plasmid DNA, the competitor array CA1, was made to contain a single copy of each competitor template, namely a single-nucleotide substituted version of the respective cDNA sequence. These plasmid DNAs were structured to liberate each cDNA fragment as a single entity upon <em>Eco</em>RI digestion (Fig.<a reveal-id="DSS002F1" data-open="DSS002F1" class="link link-reveal link-table xref-fig">1</a>b). Each mixture generated by the <em>Eco</em>RI digestion of the corresponding plasmid was individually mixed with the <em>Eco</em>RI-restricted CA1 and subsequently subjected to competitive PCR reactions along with three standard templates, namely the target cDNA, the corresponding competitor template, and an equimolar mixture of the target cDNA and its competitor template. One of the fluorescently labelled FRET probes for each cDNA sequence was designed to span the nucleotide of variation between two competing templates so that the probe's melting from the complementary sequences are differentiated depending on the completeness of the base pairing. The melting curves that we generated from the PCR amplicons by applying the respective FRET probes were fed into the MAAQ process to determine the relative quantities of each target cDNA sequence with respect to the competitor template (T/C for short).</p><p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a> presents the T/C values of each cDNA sequence (G6pdx/C, Ubc/C, and Rps27/C) in the cDNA mixtures; the quantities were determined from the quadruplicate PCR reactions. The abundances of a cDNA sequence relative to another (<em>Rps27</em>/<em>G6pdx</em>, <em>G6pdx</em>/<em>Ubc</em>, and <em>Ubc</em>/<em>Rps27</em>) in each mixture were then obtained from the ratios of the respective T/C values of the two sequences in question. Figure<a reveal-id="DSS002F1" data-open="DSS002F1" class="link link-reveal link-table xref-fig">1</a>c shows a scattered plot of the observed versus expected values of the relative abundances; the plot highlights the accuracy of this assay. The average relative error for this assay is 0.059, demonstrating that our qcPCR approach can accurately assess the abundance of a sequence in relation to another sequence in a simple mixture.</p>                            <h3 scrollto-destination=7156222 id="7156222" class="section-title" data-legacy-id=s3b>3.2. Determining the relative abundance of different DNAs in complex mixtures</h3>
<p>We subsequently considered whether our approach would be reliable in the analysis of complex samples such as those that are likely to be encountered in a laboratory. To address this matter, we measured the copy number ratios of the autosomal <em>Ubc</em> and the X-linked <em>G6pdx</em> sequences in male and female gDNAs. We chose gDNA because it is a particularly complex mixture of sequences and provides a stringent test for our assay. The <em>Ubc</em>/<em>G6pdx</em> ratios were expected to be 1 and 2 for females and males, respectively. About 20 ng of each liver gDNA of five females and five males was mixed with 0.05 pg of CA1 and subjected to PCR reactions for two sequences. We then applied the MAAQ. Five XY samples have an average ratio of 1.978, and five XX samples have a ratio of 1.000 with an average relative error of 0.021. These results strongly confirm that this method can measure the relative abundances of the target DNAs, even in a highly complex mixture such as gDNA (Table<a reveal-id="DSS002TB1" data-open="DSS002TB1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="7156227" scrollto-destination="7156227"></a>
<div content-id="DSS002TB1" class="table-modal"><div class="table-wrap"><div id="DSS002TB1" class="table-wrap-title" data-id="DSS002TB1"><div class="label -label">Table1.</div><div class="caption"><p>The relative abundance of nucleic acid species can be reliably calculated in gDNA samples</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"></th><th colspan="5">Female<hr /></th><th colspan="5">Male<hr /></th></tr><tr><th>F1</th><th>F2</th><th>F3</th><th>F4</th><th>F5</th><th>M1</th><th>M2</th><th>M3</th><th>M4</th><th>M5</th></tr></thead><tbody><tr><td colspan="11">G6pdx</td></tr><tr><td>G6pdx</td><td>0.601</td><td>0.512</td><td>0.518</td><td>0.510</td><td>0.543</td><td>0.378</td><td>0.385</td><td>0.214</td><td>0.326</td><td>0.400</td></tr><tr><td>SD</td><td>0.008</td><td>0.007</td><td>0.010</td><td>0.013</td><td>0.010</td><td>0.020</td><td>0.015</td><td>0.011</td><td>0.012</td><td>0.015</td></tr><tr><td>G6pdx/CA1</td><td>1.504</td><td>1.048</td><td>1.074</td><td>1.039</td><td>1.187</td><td>0.607</td><td>0.625</td><td>0.273</td><td>0.484</td><td>0.666</td></tr><tr><td colspan="11">Ubc</td></tr><tr><td>Ubc</td><td>0.599</td><td>0.511</td><td>0.519</td><td>0.506</td><td>0.548</td><td>0.561</td><td>0.553</td><td>0.336</td><td>0.49</td><td>0.567</td></tr><tr><td>SD</td><td>0.008</td><td>0.007</td><td>0.017</td><td>0.008</td><td>0.006</td><td>0.006</td><td>0.010</td><td>0.023</td><td>0.011</td><td>0.014</td></tr><tr><td>Ubc/CA1</td><td>1.492</td><td>1.044</td><td>1.081</td><td>1.024</td><td>1.213</td><td>1.276</td><td>1.238</td><td>0.506</td><td>0.960</td><td>1.310</td></tr><tr><td>Ubc/G6pdx<sup>a</sup></td><td>0.992</td><td>0.996</td><td>1.006</td><td>0.985</td><td>1.022</td><td>2.102</td><td>1.981</td><td>1.857</td><td>1.985</td><td>1.967</td></tr><tr><td>Obs/Exp</td><td>0.992</td><td>0.996</td><td>1.006</td><td>0.985</td><td>1.022</td><td>1.051</td><td>0.990</td><td>0.929</td><td>0.992</td><td>0.984</td></tr></tbody></table></div><div content-id="DSS002TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"></th><th colspan="5">Female<hr /></th><th colspan="5">Male<hr /></th></tr><tr><th>F1</th><th>F2</th><th>F3</th><th>F4</th><th>F5</th><th>M1</th><th>M2</th><th>M3</th><th>M4</th><th>M5</th></tr></thead><tbody><tr><td colspan="11">G6pdx</td></tr><tr><td>G6pdx</td><td>0.601</td><td>0.512</td><td>0.518</td><td>0.510</td><td>0.543</td><td>0.378</td><td>0.385</td><td>0.214</td><td>0.326</td><td>0.400</td></tr><tr><td>SD</td><td>0.008</td><td>0.007</td><td>0.010</td><td>0.013</td><td>0.010</td><td>0.020</td><td>0.015</td><td>0.011</td><td>0.012</td><td>0.015</td></tr><tr><td>G6pdx/CA1</td><td>1.504</td><td>1.048</td><td>1.074</td><td>1.039</td><td>1.187</td><td>0.607</td><td>0.625</td><td>0.273</td><td>0.484</td><td>0.666</td></tr><tr><td colspan="11">Ubc</td></tr><tr><td>Ubc</td><td>0.599</td><td>0.511</td><td>0.519</td><td>0.506</td><td>0.548</td><td>0.561</td><td>0.553</td><td>0.336</td><td>0.49</td><td>0.567</td></tr><tr><td>SD</td><td>0.008</td><td>0.007</td><td>0.017</td><td>0.008</td><td>0.006</td><td>0.006</td><td>0.010</td><td>0.023</td><td>0.011</td><td>0.014</td></tr><tr><td>Ubc/CA1</td><td>1.492</td><td>1.044</td><td>1.081</td><td>1.024</td><td>1.213</td><td>1.276</td><td>1.238</td><td>0.506</td><td>0.960</td><td>1.310</td></tr><tr><td>Ubc/G6pdx<sup>a</sup></td><td>0.992</td><td>0.996</td><td>1.006</td><td>0.985</td><td>1.022</td><td>2.102</td><td>1.981</td><td>1.857</td><td>1.985</td><td>1.967</td></tr><tr><td>Obs/Exp</td><td>0.992</td><td>0.996</td><td>1.006</td><td>0.985</td><td>1.022</td><td>1.051</td><td>0.990</td><td>0.929</td><td>0.992</td><td>0.984</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>A sample of gDNA was prepared from the kidneys of 10 adult mice (five females: F1F5 and five males: M1M5). The relative abundance of the <em>Ubc</em> and <em>G6pdx</em> genes was then determined as described in the text. The terms G6pdx and Ubc are the proportions of the target sequences within the competitively amplified amplicons, and G6pdx/CA1and Ubc/CA1 are the relative ratios of the <em>G6pdx</em> and the <em>Ubc</em> sequences with respect to their corresponding competitors, respectively; the standard deviation (SD) is based on from three measurements; the term Obs/Exp refers to the ratios of the observed per expected values.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Average relative error, 0.021.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7156227" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS002TB1" class="table-wrap-title" data-id="DSS002TB1"><div class="label -label">Table1.</div><div class="caption"><p>The relative abundance of nucleic acid species can be reliably calculated in gDNA samples</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2"></th><th colspan="5">Female<hr /></th><th colspan="5">Male<hr /></th></tr><tr><th>F1</th><th>F2</th><th>F3</th><th>F4</th><th>F5</th><th>M1</th><th>M2</th><th>M3</th><th>M4</th><th>M5</th></tr></thead><tbody><tr><td colspan="11">G6pdx</td></tr><tr><td>G6pdx</td><td>0.601</td><td>0.512</td><td>0.518</td><td>0.510</td><td>0.543</td><td>0.378</td><td>0.385</td><td>0.214</td><td>0.326</td><td>0.400</td></tr><tr><td>SD</td><td>0.008</td><td>0.007</td><td>0.010</td><td>0.013</td><td>0.010</td><td>0.020</td><td>0.015</td><td>0.011</td><td>0.012</td><td>0.015</td></tr><tr><td>G6pdx/CA1</td><td>1.504</td><td>1.048</td><td>1.074</td><td>1.039</td><td>1.187</td><td>0.607</td><td>0.625</td><td>0.273</td><td>0.484</td><td>0.666</td></tr><tr><td colspan="11">Ubc</td></tr><tr><td>Ubc</td><td>0.599</td><td>0.511</td><td>0.519</td><td>0.506</td><td>0.548</td><td>0.561</td><td>0.553</td><td>0.336</td><td>0.49</td><td>0.567</td></tr><tr><td>SD</td><td>0.008</td><td>0.007</td><td>0.017</td><td>0.008</td><td>0.006</td><td>0.006</td><td>0.010</td><td>0.023</td><td>0.011</td><td>0.014</td></tr><tr><td>Ubc/CA1</td><td>1.492</td><td>1.044</td><td>1.081</td><td>1.024</td><td>1.213</td><td>1.276</td><td>1.238</td><td>0.506</td><td>0.960</td><td>1.310</td></tr><tr><td>Ubc/G6pdx<sup>a</sup></td><td>0.992</td><td>0.996</td><td>1.006</td><td>0.985</td><td>1.022</td><td>2.102</td><td>1.981</td><td>1.857</td><td>1.985</td><td>1.967</td></tr><tr><td>Obs/Exp</td><td>0.992</td><td>0.996</td><td>1.006</td><td>0.985</td><td>1.022</td><td>1.051</td><td>0.990</td><td>0.929</td><td>0.992</td><td>0.984</td></tr></tbody></table></div><div content-id="DSS002TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2"></th><th colspan="5">Female<hr /></th><th colspan="5">Male<hr /></th></tr><tr><th>F1</th><th>F2</th><th>F3</th><th>F4</th><th>F5</th><th>M1</th><th>M2</th><th>M3</th><th>M4</th><th>M5</th></tr></thead><tbody><tr><td colspan="11">G6pdx</td></tr><tr><td>G6pdx</td><td>0.601</td><td>0.512</td><td>0.518</td><td>0.510</td><td>0.543</td><td>0.378</td><td>0.385</td><td>0.214</td><td>0.326</td><td>0.400</td></tr><tr><td>SD</td><td>0.008</td><td>0.007</td><td>0.010</td><td>0.013</td><td>0.010</td><td>0.020</td><td>0.015</td><td>0.011</td><td>0.012</td><td>0.015</td></tr><tr><td>G6pdx/CA1</td><td>1.504</td><td>1.048</td><td>1.074</td><td>1.039</td><td>1.187</td><td>0.607</td><td>0.625</td><td>0.273</td><td>0.484</td><td>0.666</td></tr><tr><td colspan="11">Ubc</td></tr><tr><td>Ubc</td><td>0.599</td><td>0.511</td><td>0.519</td><td>0.506</td><td>0.548</td><td>0.561</td><td>0.553</td><td>0.336</td><td>0.49</td><td>0.567</td></tr><tr><td>SD</td><td>0.008</td><td>0.007</td><td>0.017</td><td>0.008</td><td>0.006</td><td>0.006</td><td>0.010</td><td>0.023</td><td>0.011</td><td>0.014</td></tr><tr><td>Ubc/CA1</td><td>1.492</td><td>1.044</td><td>1.081</td><td>1.024</td><td>1.213</td><td>1.276</td><td>1.238</td><td>0.506</td><td>0.960</td><td>1.310</td></tr><tr><td>Ubc/G6pdx<sup>a</sup></td><td>0.992</td><td>0.996</td><td>1.006</td><td>0.985</td><td>1.022</td><td>2.102</td><td>1.981</td><td>1.857</td><td>1.985</td><td>1.967</td></tr><tr><td>Obs/Exp</td><td>0.992</td><td>0.996</td><td>1.006</td><td>0.985</td><td>1.022</td><td>1.051</td><td>0.990</td><td>0.929</td><td>0.992</td><td>0.984</td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>A sample of gDNA was prepared from the kidneys of 10 adult mice (five females: F1F5 and five males: M1M5). The relative abundance of the <em>Ubc</em> and <em>G6pdx</em> genes was then determined as described in the text. The terms G6pdx and Ubc are the proportions of the target sequences within the competitively amplified amplicons, and G6pdx/CA1and Ubc/CA1 are the relative ratios of the <em>G6pdx</em> and the <em>Ubc</em> sequences with respect to their corresponding competitors, respectively; the standard deviation (SD) is based on from three measurements; the term Obs/Exp refers to the ratios of the observed per expected values.</p></div></div><div content-id=""><div class="fn"><p><sup>a</sup>Average relative error, 0.021.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/7156227" target="_blank">View Large</a></div></div>                            <h3 scrollto-destination=7156229 id="7156229" class="section-title" data-legacy-id=s3c>3.3. Determining the tissue-specific expression signatures via the forms of relative abundance of cDNAs</h3>
<p>Because of the proven accuracy and reliability of our method, we used it to obtain two types of biological data. The first type of data is the relative expression abundances among three housekeeping genes in different mouse tissues. A preliminary assessment using the serial dilutions of the CA1 on the tissue cDNAs indicates that the abundances of the <em>Ubc</em> and <em>Rps27</em> transcripts are in an equivalent range whereas the abundance of <em>G6pdx</em> transcript is far less than those of the two transcripts (more than a hundredfold in some cases, data not shown). To accurately cover this wide range of expression levels, we made an additional competitor array, CA2, to carry the copies of <em>G6pdx</em>, <em>Ubc</em>, and <em>Rps27</em>sequences at a ratio of 1:7:7. The PCR amplicons of each sequence were generated by using a mixture of each cDNA and CA2 as a template. Subsequent analysis revealed the T/C values and the relative abundances among the target sequences (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). Figure<a reveal-id="DSS002F2" data-open="DSS002F2" class="link link-reveal link-table xref-fig">2</a>a visualizes the relative abundances of two subjected sequences in the cDNAs of the liver, heart, and brain of three female adult mice. These results demonstrate the tissue-specific expression signatures of these three genes with forms of their relative transcript abundances; they also demonstrate the relatively small individual variations [which have a 0.125 coefficient of variation (CV)]. </p>                        <a id="7156233" scrollto-destination="7156233"></a>
<div id="DSS002F2" data-id="DSS002F2" class="fig fig-section"><div class="label fig-label">Figure2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss002/2/m_dss00202.jpeg?Expires=2147483647&Signature=A7mdfnJB-brOu7ZjW0nQ3XPKpQtyS~I8QuAvNPaSive8ROXudnScNb2~yuvuCJcBju-ldDLeGBbMqSiBfW0xwSddGujy-fT4kkzwFVTIrj7gR27emt1Y9IOy7Zb~cp4jE1y0MM947J3IENObYnvRj-FmfdeE3VhUf51IV1cMSANrucHrpX51aWB~Uyw-4rpTCpt-IiCyw2gJrqioOSOF73GVDhIXCB3TkANqT78782nHOA45WINjtwFX88~99pAsi~zl9z5mRAYTyiHBUqAKGgmiuMj10x13MBeLfiSWVLJ8kgbBN-nTDD-z4ZLf7uvQnd~3kAkuCJK~82iYLrKfLA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relative abundances among the transcripts of three housekeeping genes that display tissue-specific expression signatures. (a) A plot of the relative expression ratios of two of the three housekeeping genes in three adult tissue samples as determined from three individuals. The error bar denotes the standard deviations among the individuals. (b) A plot of the relative expression ratio as determined from multiple RT reactions of a single individual; the plot highlights the precision of the assay. The error bar denotes the standard deviations among multiple RT reactions." path-from-xml="dss00202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7156233/dss00202.jpeg" path-from-xml="dss00202.jpeg" target="_blank">View large</a><a section="7156233" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss002/2/dss00202.jpeg?Expires=2147483647&Signature=UKQRb~jcgWjPPszOHyB6VaxS6O1UKghOO1agjmHiEpZSAEw~DioMfg-t0d7FX~s4qnKjhNZ6zZ~OEH8ojiDXGsthl37YVOeNyMLy7GLt8wESCgUvPAQoYltMdeBBoS5wabQvYbdAHWfaHl27v59TxewnpztU63lJluXKsEDxlM~KMcKBt9FoG64U5gWJ34gOz81E3GPqUZkCUHyRMX1c0r~rNqityn2S0rfIuMUr8D~jGmzBia1rZ5s1~GPNc4KLKJr2UDyoxhPykai1hxH0BgK1qHmzWSSX56UNnoL~4Ri8raUmxN~J-CMAeGDBpEPsmnZ-MOrfCzULWeCP0AlZKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7156233&ar=474733&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relative abundances among the transcripts of three housekeeping genes that display tissue-specific expression signatures. (a) A plot of the relative expression ratios of two of the three housekeeping genes in three adult tissue samples as determined from three individuals. The error bar denotes the standard deviations among the individuals. (b) A plot of the relative expression ratio as determined from multiple RT reactions of a single individual; the plot highlights the precision of the assay. The error bar denotes the standard deviations among multiple RT reactions.</p></div></div><div content-id="DSS002F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss002/2/m_dss00202.jpeg?Expires=2147483647&Signature=A7mdfnJB-brOu7ZjW0nQ3XPKpQtyS~I8QuAvNPaSive8ROXudnScNb2~yuvuCJcBju-ldDLeGBbMqSiBfW0xwSddGujy-fT4kkzwFVTIrj7gR27emt1Y9IOy7Zb~cp4jE1y0MM947J3IENObYnvRj-FmfdeE3VhUf51IV1cMSANrucHrpX51aWB~Uyw-4rpTCpt-IiCyw2gJrqioOSOF73GVDhIXCB3TkANqT78782nHOA45WINjtwFX88~99pAsi~zl9z5mRAYTyiHBUqAKGgmiuMj10x13MBeLfiSWVLJ8kgbBN-nTDD-z4ZLf7uvQnd~3kAkuCJK~82iYLrKfLA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relative abundances among the transcripts of three housekeeping genes that display tissue-specific expression signatures. (a) A plot of the relative expression ratios of two of the three housekeeping genes in three adult tissue samples as determined from three individuals. The error bar denotes the standard deviations among the individuals. (b) A plot of the relative expression ratio as determined from multiple RT reactions of a single individual; the plot highlights the precision of the assay. The error bar denotes the standard deviations among multiple RT reactions." path-from-xml="dss00202.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7156233/dss00202.jpeg" path-from-xml="dss00202.jpeg" target="_blank">View large</a><a section="7156233" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss002/2/dss00202.jpeg?Expires=2147483647&Signature=UKQRb~jcgWjPPszOHyB6VaxS6O1UKghOO1agjmHiEpZSAEw~DioMfg-t0d7FX~s4qnKjhNZ6zZ~OEH8ojiDXGsthl37YVOeNyMLy7GLt8wESCgUvPAQoYltMdeBBoS5wabQvYbdAHWfaHl27v59TxewnpztU63lJluXKsEDxlM~KMcKBt9FoG64U5gWJ34gOz81E3GPqUZkCUHyRMX1c0r~rNqityn2S0rfIuMUr8D~jGmzBia1rZ5s1~GPNc4KLKJr2UDyoxhPykai1hxH0BgK1qHmzWSSX56UNnoL~4Ri8raUmxN~J-CMAeGDBpEPsmnZ-MOrfCzULWeCP0AlZKg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7156233&ar=474733&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00202.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relative abundances among the transcripts of three housekeeping genes that display tissue-specific expression signatures. (a) A plot of the relative expression ratios of two of the three housekeeping genes in three adult tissue samples as determined from three individuals. The error bar denotes the standard deviations among the individuals. (b) A plot of the relative expression ratio as determined from multiple RT reactions of a single individual; the plot highlights the precision of the assay. The error bar denotes the standard deviations among multiple RT reactions.</p></div></div><p>Figure<a reveal-id="DSS002F2" data-open="DSS002F2" class="link link-reveal link-table xref-fig">2</a>b shows the relative abundances among the transcripts of the three tissues of a single individual. The values were averaged from four measurements. We determined three of the measurements from three independent RT reactions by adapting another version of competitor array, CA3, with the respective copy ratio of 1:9:9 for the sequences of <em>G6pdx</em>, <em>Ubc</em>, and <em>Rps27</em>. The CV of these measurements (0.079) confirms that our quantitation method has a high level of precision.</p>                            <h3 scrollto-destination=7156239 id="7156239" class="section-title" data-legacy-id=s3d>3.4. Measuring the relative promoter strength of multiple promoters of a gene</h3>
<p>We tried to obtain the relative transcriptional activity of the multiple <em>Igf2</em> promoters in embryonic tissue. The utilization of multiple promoters for transcription is not rare in a eukaryotic system.<sup><a reveal-id="DSS002C12" data-open="DSS002C12" class="link link-ref link-reveal xref-bibr">12</a></sup> The usage rate of each promoter could be the subject of developmental or tissue-specific regulations. If the promoter strength is linearly correlated with the cellular level of the transcript it drives, our assay schemeis well suited for assessing the relative strength among the promoters. This type of assessment provides insight into the relative promoter activity of the given samples and confirms the potential of our approach as a versatile quantitation method. Transcriptional regulation of the murine <em>Igf2</em> gene is relatively well characterized with respect to the role and position of <em>cis</em>-regulatory elements, including promoters and enhancers.<sup><a reveal-id="DSS002C13" data-open="DSS002C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Fetal and neonatal <em>Igf2</em> transcription is derived from multiple promoters, P1, P2, and P3, through appropriate interactions between regulatory sequences; these types of interactions provide a good model system for comparing the transcriptions of multiple promoters and specific interactions.<sup><a reveal-id="DSS002C14" data-open="DSS002C14" class="link link-ref link-reveal xref-bibr">14</a></sup></p><p>We conducted an assay in which each promoter-specific transcription was compared with the total level of <em>Igf2</em> transcription in two embryonic tissues, the liver and muscle with endodermal and mesodermal origins, respectively. The primers were designed to span two consecutive exons to prevent any potential noise from gDNA contamination. The promoter specificity of the PCR amplicon was given by placing a part of the forward primers on the promoter-specific exons (Fig.<a reveal-id="DSS002F3" data-open="DSS002F3" class="link link-reveal link-table xref-fig">3</a>a). The competitor templates for the promoter-specific amplification were made by placing only a single-nucleotide substitution on exon E4; they were then arranged into a plasmid to constitute the competitor array CA4. Each promoter-specific competitor was also used as the competitor for the entire <em>Igf2</em> transcript by applying a new forward PCR primer placed on the 5-end of the shared exon E4 (Fig.<a reveal-id="DSS002F3" data-open="DSS002F3" class="link link-reveal link-table xref-fig">3</a>a). In this way, CA4 can be exploited in two different competitions with an aspect ratio of 1:3. One competition is for the promoter-specific transcript; the other is for the total transcript. Each cDNA was added to the CA4 and subsequently subjected to four separate PCR amplifications: three of the PCR amplifications were specific for the promoter-specific transcript and the other was used for the total transcript. After determining the T/C values of each PCR reaction, we obtained the relative abundance of each promoter-specific transcript with respect to the total <em>Igf2</em> transcript; the results are presented in Fig.<a reveal-id="DSS002F3" data-open="DSS002F3" class="link link-reveal link-table xref-fig">3</a>b and <a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>. </p>                        <a id="7156243" scrollto-destination="7156243"></a>
<div id="DSS002F3" data-id="DSS002F3" class="fig fig-section"><div class="label fig-label">Figure3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss002/2/m_dss00203.jpeg?Expires=2147483647&Signature=lg3NGIUCQjNK8m5OHUI5KrrMuqkJU~oJJarbVoq9Q~IqRTKbBKhfPPEWINypPLZWGxzFvtgonem~qN-isTDC0UkmbfuExIc~eiQLcW7ODCWvS1KwsO14W9Mg6I3mTovEANyX-AC2fjqROV31AjSxCEEwkcHceeH7HjbjiSTmeJETXifTS1o93d7O6dEOUXGtw66i9Bd8Vq1ralWKD8aae3kaJB24gc5LkemikOlYWcpVKfbuKwjNX3JRrt0J8obMFvt29nzRDDvxENKrrzss-ZZ3OxWnnKbcp-3NKax-7hehInaHabVkz3gjxHoPkd3k~5AkkhREY~SCBAT48ztniw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Promoter-specific transcriptions as measured in relation to the total transcription level. (a) The fetal Igf2 expression as derived from the promoters P1, P2, and P3. Each exon (E1, E2, and E3) constitutes a promoter-specific part of Igf2 mRNA that is spliced to the shared exon E4. The primers (arrows with dotted line in the middle) were designed to span two consecutive exons except for one forward primer, Igf2-E4F, which is placed on the 5-most part of exon E4 to account for the total transcription of Igf2 and the common reverse primer. The detailed structure of the exon E4exon E5 junction region regarding the positions of the reverse primers Igf2-E4R1 and Igf2-E4R2 and the FRET probes Igf2-A and Igf2-S is depicted in the lower part of the figure. (b) The relative quantities of the promoter-specific transcription with respect to the total transcription yield the relative abundance of each transcript in the samples. The relative transcript levels, which were measured from muscle tissue and liver tissue, represent the mesodermal and endodermal origins, respectively." path-from-xml="dss00203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7156243/dss00203.jpeg" path-from-xml="dss00203.jpeg" target="_blank">View large</a><a section="7156243" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss002/2/dss00203.jpeg?Expires=2147483647&Signature=BIYUz0x8rUSgPkbtLGlU89dtn1vxyRPkx-f6OPjQIl77M5fP5UayPRLhrgJrh1NgVJh4cEEBeAUur94dAJ-4EKYvht1segFnIYWTlQrxsXxosF78auyQIJZIha4bPn-Q0YbqUVMVQDIkuVJsYgBYtM7RjF2jPU9bxPaS~Y9OWbj2XTXTxLtZo21XhvP~ke7V83BQKPBjrS6dcCfSNmGLwofuSquBAor3uRrovatJ5XzArcrGtOfZ7b0tgn-JG1GPBIu9fo8xozAy1Pwp-GU~cVxZhaxsUs1qWQaFYMKDYK9UC1OTZOKYjn6MX7iNp5qZr95jWWzJRQTmAsIax-030Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7156243&ar=474733&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Promoter-specific transcriptions as measured in relation to the total transcription level. (a) The fetal <em>Igf2</em> expression as derived from the promoters P1, P2, and P3. Each exon (E1, E2, and E3) constitutes a promoter-specific part of <em>Igf2</em> mRNA that is spliced to the shared exon E4. The primers (arrows with dotted line in the middle) were designed to span two consecutive exons except for one forward primer, Igf2-E4F, which is placed on the 5-most part of exon E4 to account for the total transcription of <em>Igf2</em> and the common reverse primer. The detailed structure of the exon E4exon E5 junction region regarding the positions of the reverse primers Igf2-E4R1 and Igf2-E4R2 and the FRET probes Igf2-A and Igf2-S is depicted in the lower part of the figure. (b) The relative quantities of the promoter-specific transcription with respect to the total transcription yield the relative abundance of each transcript in the samples. The relative transcript levels, which were measured from muscle tissue and liver tissue, represent the mesodermal and endodermal origins, respectively.</p></div></div><div content-id="DSS002F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss002/2/m_dss00203.jpeg?Expires=2147483647&Signature=lg3NGIUCQjNK8m5OHUI5KrrMuqkJU~oJJarbVoq9Q~IqRTKbBKhfPPEWINypPLZWGxzFvtgonem~qN-isTDC0UkmbfuExIc~eiQLcW7ODCWvS1KwsO14W9Mg6I3mTovEANyX-AC2fjqROV31AjSxCEEwkcHceeH7HjbjiSTmeJETXifTS1o93d7O6dEOUXGtw66i9Bd8Vq1ralWKD8aae3kaJB24gc5LkemikOlYWcpVKfbuKwjNX3JRrt0J8obMFvt29nzRDDvxENKrrzss-ZZ3OxWnnKbcp-3NKax-7hehInaHabVkz3gjxHoPkd3k~5AkkhREY~SCBAT48ztniw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Promoter-specific transcriptions as measured in relation to the total transcription level. (a) The fetal Igf2 expression as derived from the promoters P1, P2, and P3. Each exon (E1, E2, and E3) constitutes a promoter-specific part of Igf2 mRNA that is spliced to the shared exon E4. The primers (arrows with dotted line in the middle) were designed to span two consecutive exons except for one forward primer, Igf2-E4F, which is placed on the 5-most part of exon E4 to account for the total transcription of Igf2 and the common reverse primer. The detailed structure of the exon E4exon E5 junction region regarding the positions of the reverse primers Igf2-E4R1 and Igf2-E4R2 and the FRET probes Igf2-A and Igf2-S is depicted in the lower part of the figure. (b) The relative quantities of the promoter-specific transcription with respect to the total transcription yield the relative abundance of each transcript in the samples. The relative transcript levels, which were measured from muscle tissue and liver tissue, represent the mesodermal and endodermal origins, respectively." path-from-xml="dss00203.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/7156243/dss00203.jpeg" path-from-xml="dss00203.jpeg" target="_blank">View large</a><a section="7156243" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss002/2/dss00203.jpeg?Expires=2147483647&Signature=BIYUz0x8rUSgPkbtLGlU89dtn1vxyRPkx-f6OPjQIl77M5fP5UayPRLhrgJrh1NgVJh4cEEBeAUur94dAJ-4EKYvht1segFnIYWTlQrxsXxosF78auyQIJZIha4bPn-Q0YbqUVMVQDIkuVJsYgBYtM7RjF2jPU9bxPaS~Y9OWbj2XTXTxLtZo21XhvP~ke7V83BQKPBjrS6dcCfSNmGLwofuSquBAor3uRrovatJ5XzArcrGtOfZ7b0tgn-JG1GPBIu9fo8xozAy1Pwp-GU~cVxZhaxsUs1qWQaFYMKDYK9UC1OTZOKYjn6MX7iNp5qZr95jWWzJRQTmAsIax-030Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=7156243&ar=474733&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00203.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Promoter-specific transcriptions as measured in relation to the total transcription level. (a) The fetal <em>Igf2</em> expression as derived from the promoters P1, P2, and P3. Each exon (E1, E2, and E3) constitutes a promoter-specific part of <em>Igf2</em> mRNA that is spliced to the shared exon E4. The primers (arrows with dotted line in the middle) were designed to span two consecutive exons except for one forward primer, Igf2-E4F, which is placed on the 5-most part of exon E4 to account for the total transcription of <em>Igf2</em> and the common reverse primer. The detailed structure of the exon E4exon E5 junction region regarding the positions of the reverse primers Igf2-E4R1 and Igf2-E4R2 and the FRET probes Igf2-A and Igf2-S is depicted in the lower part of the figure. (b) The relative quantities of the promoter-specific transcription with respect to the total transcription yield the relative abundance of each transcript in the samples. The relative transcript levels, which were measured from muscle tissue and liver tissue, represent the mesodermal and endodermal origins, respectively.</p></div></div><p>The <em>Igf2</em> transcription in the liver necessitates the involvement of an endoderm enhancer, whereas the muscle expression largely relies on a mesoderm enhancer. Our results confirm that minimal transcription activity occurs on promoter P1, regardless of which enhancer is involved; specifically, only 67% of the total transcript is attributed to promoter P1 activity in both the liver and muscle. On the other hand, the two types of tissue with developmentally different origins have different preferences with regard to the utilization of promoters P2 and P3: compared with P3, P2 has more transcription in the muscle but less in the liver. This difference indicates that the relative strength of the two promoters depends on the type of enhancers used. Moreover, the results confirm that this method is highly reproducible; the standard deviations of the measurements of three independent RT reactions are minute (ranging from 0.051 to 0.001, CV = 0.033).</p><p>If exon E4 is spliced solely for the transcripts derived from the three promoters addressed here, the relative transcription levels of the three promoters should add up to 1. The fact that the aggregate transcription levels for the muscle samples are 1.010 and 1.008 is consistent with this notion. On the other hand, the aggregate transcription levels for the liver samples are 0.920 and 0.938. This difference may be attributed to the possible existence of one or more transcripts that contain exon E4 in the liver.</p>                            <h2 scrollto-destination=7156246 id="7156246" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>In this study, we presented a qcPCR method that can be used to measure the relative abundances among the sequences of interest in any DNA sample. Because of the technical difficulty of obtaining a direct quantitative comparison between two unlinked sequences, we exploited the absolute relativity of their respective competitor templates which are arranged in the competitor array. In addition, the high accuracy of the MAAQ method was exploited for the quantitation of the fractional ratios of the competitively amplified DNAs. The relative abundances of two different sequences were consequently obtained in a series of plasmid constructs and gDNAs. These abundances turned out to be very accurate; the average relative errors of 0.059 and 0.021, respectively, confirm the reliability and practical feasibility of this approach.</p><p>The quantitation scheme was further validated by its successful acquisition of biological data. We compared three housekeeping genes in terms of the abundances of their transcripts for multiple types of mouse tissue and analysed <em>Igf2</em> expression in terms of the proportion of the promoter-specific transcript in the total <em>Igf2</em> transcripts. These results efficiently illustrate the tissue-specific expression signatures of the three housekeeping genes and the relative transcriptional strength of the <em>Igf2</em> promoters.</p><p>With an exploitation of the high level of accuracy and precision, our method can potentially be used as a form of standardized quantitation. That is, the expression level of any gene of interest can be determined relative to the expression of a single gene or a combination of several reference genes in the same sample. Thus, our method can produce a stand-alone value for the transcription level of a gene in a particular sample. Furthermore, this value can readily be compared with other results that are independently generated by the same method at any place or time. In this regard, <em>Ubc</em> and <em>Rps27</em> can be considered as reference genes because their relative transcript abundances were relatively constant in all the tested tissues. Our data also demonstrate that the assay remains robust across a wide range of expression levels (e.g. the <em>G6pdx</em>/<em>Ubc</em> value in liver is 0.005). This range can be further extended by applying a competitor array that contains the respective competitor templates with a larger aspect ratio.</p><p>For the purpose of sharing and exchanging the quantitation data of the transcripts, various applications have been developed to present the quantities on an absolute scale such as copy numbers per cell.<sup><a reveal-id="DSS002C15" data-open="DSS002C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSS002C16" data-open="DSS002C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Miura <em>et al.</em><sup><a reveal-id="DSS002C16" data-open="DSS002C16" class="link link-ref link-reveal xref-bibr">16</a></sup> reported a qcPCR approach with which the transcription level of thousands of yeast genes were described as copy numbers per cell. Kanno <em>et al.</em><sup><a reveal-id="DSS002C15" data-open="DSS002C15" class="link link-ref link-reveal xref-bibr">15</a></sup> also presented a method to convert the microarray data or quantitative PCR data to copy numbers per cell. Both approaches utilize a series of standard RNAs which is spiked into the sample of interest in order to provide a way of calibration of inter-experiment differences or normalization of quantity data in the experimental sets. Although these approaches are effective in providing the structural architecture of transcriptome abundances on an absolute scale, it still requires standardization of standard RNAs in order to cross-refer those data to others that are independently generated. In this circumstance, it would be more practical to present the transcription level of a gene in relation to that of another in the same sample as we did. Many approaches including above two examples and others that are focused on defining copy number variation in the genomes are indeed capable of delineating the abundances of multiple sequences in the view of their relativity.<sup><a reveal-id="DSS002C17" data-open="DSS002C17" class="link link-ref link-reveal xref-bibr">17</a>,<a reveal-id="DSS002C18" data-open="DSS002C18" class="link link-ref link-reveal xref-bibr">18</a></sup> However, the accuracy and the precision that these approaches can confer are not as high as those our method can, or were not addressed probably due to disregard of presenting the relative abundances of intergenic sequences.</p><p>On the other hand, application of our method for the standardized quantitation is yet to address certain problems that are potentially caused by the differences in the cDNA conversion efficiency of individual RNAs and between the segments of a given RNA. The different reaction conditions or types of enzymes used for RT consequently produce variations in the relative quantity of given sequences in the cDNA pool. Nonetheless, our approach provides a way of examining the extent to which the relative abundance of those sequences varies on account of the different reaction conditions; it also provides a way of determining the best conditions for minimizing such variations.</p><p>With simplicity, speed, and accuracy, our quantitation approach provides a straightforward paradigm for comparing gene expression levels across cell types, developmental stages, and even different species. Most importantly, with this approach, experiments performed in different laboratories or at different times can be correctly referenced to one another.</p>                            <h2 scrollto-destination=7156253 id="7156253" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at www.dnaresearch.oxfordjournals.org</a>.</p>                            <h2 scrollto-destination=7156255 id="7156255" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the National Research Foundation of Korea grant (NRF, No.20110027739) and a Korea Institute of Oriental Medicine grant (KIOM, No.K11070) funded by the Korea government (MEST).</p>                            <h2 scrollto-destination=7156257 id="7156257" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS002C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cheung</div> <div class="given-names">V.G.</div></div>, <div class="name"><div class="surname">Spielman</div> <div class="given-names">R.S.</div></div>.</div><div class="article-title">Genetics of human gene expression: mapping DNA variants that influence gene expression</div>,<div class="source">Nat. Rev. Genet.</div>,<div class="year">2009</div>,vol.<div class="volume">10</div>(pg.<div class="fpage">595</div>-<div class="lpage">604</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetics%20of%20human%20gene%20expression%3A%20mapping%20DNA%20variants%20that%20influence%20gene%20expression&amp;author=V.G.%20Cheung&amp;author=R.S.%20Spielman&amp;publication_year=2009&amp;journal=Nat.%20Rev.%20Genet.&amp;volume=10&amp;pages=595-604" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrg2630" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrg2630" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrg2630"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19636342" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Schena</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Shalon</div> <div class="given-names">D.</div></div>, <div class="name"><div class="surname">Davis</div> <div class="given-names">R.W.</div></div>, <div class="name"><div class="surname">Brown</div> <div class="given-names">P.O.</div></div>.</div><div class="article-title">Quantitative monitoring of gene expression patterns with a complementary DNA microarray</div>,<div class="source">Science</div>,<div class="year">1995</div>,vol.<div class="volume">270</div>(pg.<div class="fpage">467</div>-<div class="lpage">70</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Quantitative%20monitoring%20of%20gene%20expression%20patterns%20with%20a%20complementary%20DNA%20microarray&amp;author=M.%20Schena&amp;author=D.%20Shalon&amp;author=R.W.%20Davis&amp;author=P.O.%20Brown&amp;publication_year=1995&amp;journal=Science&amp;volume=270&amp;pages=467-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1126/science.270.5235.467" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1126%2Fscience.270.5235.467" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1126%2Fscience.270.5235.467"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7569999" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marioni</div> <div class="given-names">J.C.</div></div>, <div class="name"><div class="surname">Mason</div> <div class="given-names">C.E.</div></div>, <div class="name"><div class="surname">Mane</div> <div class="given-names">S.M.</div></div>, <div class="name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>, <div class="name"><div class="surname">Gilad</div> <div class="given-names">Y.</div></div>.</div><div class="article-title">RNA-seq: an assessment of technical reproducibility and comparison with gene expression arrays</div>,<div class="source">Genome Res.</div>,<div class="year">2008</div>,vol.<div class="volume">18</div>(pg.<div class="fpage">1509</div>-<div class="lpage">17</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-seq%3A%20an%20assessment%20of%20technical%20reproducibility%20and%20comparison%20with%20gene%20expression%20arrays&amp;author=J.C.%20Marioni&amp;author=C.E.%20Mason&amp;author=S.M.%20Mane&amp;author=M.%20Stephens&amp;author=Y.%20Gilad&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1509-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.079558.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.079558.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.079558.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18550803" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tan</div> <div class="given-names">P.K.</div></div>, <div class="name"><div class="surname">Downey</div> <div class="given-names">T.J.</div></div>, <div class="name"><div class="surname">Spitznagel</div> <div class="given-names">E.L.</div>Jr</div>,et al.</div><div class="article-title">Evaluation of gene expression measurements from commercial microarray platforms</div>,<div class="source">Nucleic Acids Res.</div>,<div class="year">2003</div>,vol.<div class="volume">31</div>(pg.<div class="fpage">5676</div>-<div class="lpage">84</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evaluation%20of%20gene%20expression%20measurements%20from%20commercial%20microarray%20platforms&amp;author=P.K.%20Tan&amp;author=T.J.%20Downey&amp;author=E.L.%20Spitznagel&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res.&amp;volume=31&amp;pages=5676-84" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg763" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg763" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg763"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14500831" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Barbacioru</div> <div class="given-names">C.</div></div>, <div class="name"><div class="surname">Hyland</div> <div class="given-names">F.</div></div>,et al.</div><div class="article-title">Large scale real-time PCR validation on gene expression measurements from two commercial long-oligonucleotide microarrays</div>,<div class="source">BMC Genomics</div>,<div class="year">2006</div>,vol.<div class="volume">7</div>pg.<div class="fpage">59</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large%20scale%20real-time%20PCR%20validation%20on%20gene%20expression%20measurements%20from%20two%20commercial%20long-oligonucleotide%20microarrays&amp;author=Y.%20Wang&amp;author=C.%20Barbacioru&amp;author=F.%20Hyland&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=59" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-59" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-59" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-59"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16551369" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zentilin</div> <div class="given-names">L.</div></div>, <div class="name"><div class="surname">Giacca</div> <div class="given-names">M.</div></div>.</div><div class="article-title">Competitive PCR for precise nucleic acid quantification</div>,<div class="source">Nat. Protoc.</div>,<div class="year">2007</div>,vol.<div class="volume">2</div>(pg.<div class="fpage">2092</div>-<div class="lpage">104</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Competitive%20PCR%20for%20precise%20nucleic%20acid%20quantification&amp;author=L.%20Zentilin&amp;author=M.%20Giacca&amp;publication_year=2007&amp;journal=Nat.%20Protoc.&amp;volume=2&amp;pages=2092-104" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2007.299" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2007.299" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2007.299"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17853864" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Siebert</div> <div class="given-names">P.D.</div></div>, <div class="name"><div class="surname">Larrick</div> <div class="given-names">J.W.</div></div>.</div><div class="article-title">Competitive PCR</div>,<div class="source">Nature</div>,<div class="year">1992</div>,vol.<div class="volume">359</div>(pg.<div class="fpage">557</div>-<div class="lpage">8</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Competitive%20PCR&amp;author=P.D.%20Siebert&amp;author=J.W.%20Larrick&amp;publication_year=1992&amp;journal=Nature&amp;volume=359&amp;pages=557-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/359557a0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2F359557a0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2F359557a0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1383831" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ding</div> <div class="given-names">C.</div></div>, <div class="name"><div class="surname">Cantor</div> <div class="given-names">C.R.</div></div>.</div><div class="article-title">A high-throughput gene expression analysis technique using competitive PCR and matrix-assisted laser desorption ionization time-of-flight MS</div>,<div class="source">Proc. Natl. Acad. Sci. USA</div>,<div class="year">2003</div>,vol.<div class="volume">100</div>(pg.<div class="fpage">3059</div>-<div class="lpage">64</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20high-throughput%20gene%20expression%20analysis%20technique%20using%20competitive%20PCR%20and%20matrix-assisted%20laser%20desorption%20ionization%20time-of-flight%20MS&amp;author=C.%20Ding&amp;author=C.R.%20Cantor&amp;publication_year=2003&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20USA&amp;volume=100&amp;pages=3059-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0630494100" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0630494100" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0630494100"> </span></div></div></div></div></div></div><div content-id="DSS002C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Chang</div> <div class="given-names">H.W.</div></div>, <div class="name"><div class="surname">Shih Ie</div> <div class="given-names">M.</div></div>.</div><div class="article-title">Digital single-nucleotide polymorphism analysis for allelic imbalance</div>,<div class="source">Methods Mol. Med.</div>,<div class="year">2005</div>,vol.<div class="volume">103</div>(pg.<div class="fpage">137</div>-<div class="lpage">41</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Digital%20single-nucleotide%20polymorphism%20analysis%20for%20allelic%20imbalance&amp;author=H.W.%20Chang&amp;author=M.%20Shih%20Ie&amp;publication_year=2005&amp;journal=Methods%20Mol.%20Med.&amp;volume=103&amp;pages=137-41" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15542903" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jeong</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Hahn</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Rong</div> <div class="given-names">Q.</div></div>, <div class="name"><div class="surname">Pfeifer</div> <div class="given-names">K.</div></div>.</div><div class="article-title">Accurate quantitation of allele-specific expression patterns by analysis of DNA melting</div>,<div class="source">Genome Res.</div>,<div class="year">2007</div>,vol.<div class="volume">17</div>(pg.<div class="fpage">1093</div>-<div class="lpage">100</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Accurate%20quantitation%20of%20allele-specific%20expression%20patterns%20by%20analysis%20of%20DNA%20melting&amp;author=S.%20Jeong&amp;author=Y.%20Hahn&amp;author=Q.%20Rong&amp;author=K.%20Pfeifer&amp;publication_year=2007&amp;journal=Genome%20Res.&amp;volume=17&amp;pages=1093-100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.6028507" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.6028507" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.6028507"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17545578" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Yu</div> <div class="given-names">H.</div></div>, <div class="name"><div class="surname">Koo</div> <div class="given-names">I.</div></div>, <div class="name"><div class="surname">Jeong</div> <div class="given-names">S.</div></div>.</div><div class="article-title">Relative quantitation of restriction fragment length polymorphic DNAs via DNA melting analysis provides an effective way to determine allele frequencies</div>,<div class="source">Genomics</div>,<div class="year">2009</div>,vol.<div class="volume">94</div>(pg.<div class="fpage">355</div>-<div class="lpage">61</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Relative%20quantitation%20of%20restriction%20fragment%20length%20polymorphic%20DNAs%20via%20DNA%20melting%20analysis%20provides%20an%20effective%20way%20to%20determine%20allele%20frequencies&amp;author=H.%20Yu&amp;author=I.%20Koo&amp;author=S.%20Jeong&amp;publication_year=2009&amp;journal=Genomics&amp;volume=94&amp;pages=355-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ygeno.2009.08.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ygeno.2009.08.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ygeno.2009.08.004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19686835" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Davuluri</div> <div class="given-names">R.V.</div></div>, <div class="name"><div class="surname">Suzuki</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Sugano</div> <div class="given-names">S.</div></div>, <div class="name"><div class="surname">Plass</div> <div class="given-names">C.</div></div>, <div class="name"><div class="surname">Huang</div> <div class="given-names">T.H.</div></div>.</div><div class="article-title">The functional consequences of alternative promoter use in mammalian genomes</div>,<div class="source">Trends Genet.</div>,<div class="year">2008</div>,vol.<div class="volume">24</div>(pg.<div class="fpage">167</div>-<div class="lpage">77</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20functional%20consequences%20of%20alternative%20promoter%20use%20in%20mammalian%20genomes&amp;author=R.V.%20Davuluri&amp;author=Y.%20Suzuki&amp;author=S.%20Sugano&amp;author=C.%20Plass&amp;author=T.H.%20Huang&amp;publication_year=2008&amp;journal=Trends%20Genet.&amp;volume=24&amp;pages=167-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.tig.2008.01.008" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.tig.2008.01.008" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.tig.2008.01.008"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18329129" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kaffer</div> <div class="given-names">C.R.</div></div>, <div class="name"><div class="surname">Grinberg</div> <div class="given-names">A.</div></div>, <div class="name"><div class="surname">Pfeifer</div> <div class="given-names">K.</div></div>.</div><div class="article-title">Regulatory mechanisms at the mouse Igf2/H19 locus</div>,<div class="source">Mol. Cell Biol.</div>,<div class="year">2001</div>,vol.<div class="volume">21</div>(pg.<div class="fpage">8189</div>-<div class="lpage">96</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulatory%20mechanisms%20at%20the%20mouse%20Igf2%2FH19%20locus&amp;author=C.R.%20Kaffer&amp;author=A.%20Grinberg&amp;author=K.%20Pfeifer&amp;publication_year=2001&amp;journal=Mol.%20Cell%20Biol.&amp;volume=21&amp;pages=8189-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MCB.21.23.8189-8196.2001" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMCB.21.23.8189-8196.2001" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMCB.21.23.8189-8196.2001"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11689707" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rotwein</div> <div class="given-names">P.</div></div>, <div class="name"><div class="surname">Hall</div> <div class="given-names">L.J.</div></div>.</div><div class="article-title">Evolution of insulin-like growth factor II: characterization of the mouse IGF-II gene and identification of two pseudo-exons</div>,<div class="source">DNA Cell Biol.</div>,<div class="year">1990</div>,vol.<div class="volume">9</div>(pg.<div class="fpage">725</div>-<div class="lpage">35</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolution%20of%20insulin-like%20growth%20factor%20II%3A%20characterization%20of%20the%20mouse%20IGF-II%20gene%20and%20identification%20of%20two%20pseudo-exons&amp;author=P.%20Rotwein&amp;author=L.J.%20Hall&amp;publication_year=1990&amp;journal=DNA%20Cell%20Biol.&amp;volume=9&amp;pages=725-35" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1089/dna.1990.9.725" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1089%2Fdna.1990.9.725" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1089%2Fdna.1990.9.725"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/1702294" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Kanno</div> <div class="given-names">J.</div></div>, <div class="name"><div class="surname">Aisaki</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Igarashi</div> <div class="given-names">K.</div></div>,et al.</div><div class="article-title">Per cell normalization method for mRNA measurement by quantitative PCR and microarrays</div>,<div class="source">BMC Genomics</div>,<div class="year">2006</div>,vol.<div class="volume">7</div>pg.<div class="fpage">64</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Per%20cell%20normalization%20method%20for%20mRNA%20measurement%20by%20quantitative%20PCR%20and%20microarrays&amp;author=J.%20Kanno&amp;author=K.%20Aisaki&amp;author=K.%20Igarashi&amp;publication_year=2006&amp;journal=BMC%20Genomics&amp;volume=7&amp;pages=64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-7-64" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-7-64" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-7-64"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16571132" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miura</div> <div class="given-names">F.</div></div>, <div class="name"><div class="surname">Kawaguchi</div> <div class="given-names">N.</div></div>, <div class="name"><div class="surname">Yoshida</div> <div class="given-names">M.</div></div>,et al.</div><div class="article-title">Absolute quantification of the budding yeast transcriptome by means of competitive PCR between genomic and complementary DNAs</div>,<div class="source">BMC Genomics</div>,<div class="year">2008</div>,vol.<div class="volume">9</div>pg.<div class="fpage">574</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Absolute%20quantification%20of%20the%20budding%20yeast%20transcriptome%20by%20means%20of%20competitive%20PCR%20between%20genomic%20and%20complementary%20DNAs&amp;author=F.%20Miura&amp;author=N.%20Kawaguchi&amp;author=M.%20Yoshida&amp;publication_year=2008&amp;journal=BMC%20Genomics&amp;volume=9&amp;pages=574" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-9-574" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-9-574" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-9-574"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19040753" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qiao</div> <div class="given-names">Y.</div></div>, <div class="name"><div class="surname">Liu</div> <div class="given-names">X.</div></div>, <div class="name"><div class="surname">Harvard</div> <div class="given-names">C.</div></div>,et al.</div><div class="article-title">Large-scale copy number variants (CNVs): distribution in normal subjects and FISH/real-time qPCR analysis</div>,<div class="source">BMC Genomics</div>,<div class="year">2007</div>,vol.<div class="volume">8</div>pg.<div class="fpage">167</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Large-scale%20copy%20number%20variants%20(CNVs)%3A%20distribution%20in%20normal%20subjects%20and%20FISH%2Freal-time%20qPCR%20analysis&amp;author=Y.%20Qiao&amp;author=X.%20Liu&amp;author=C.%20Harvard&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=167" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-167" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-167" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-167"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17565693" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div><div content-id="DSS002C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS002C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iwao-Koizumi</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Maekawa</div> <div class="given-names">K.</div></div>, <div class="name"><div class="surname">Nakamura</div> <div class="given-names">Y.</div></div>,et al.</div><div class="article-title">A novel technique for measuring variations in DNA copy-number: competitive genomic polymerase chain reaction</div>,<div class="source">BMC Genomics</div>,<div class="year">2007</div>,vol.<div class="volume">8</div>pg.<div class="fpage">206</div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20novel%20technique%20for%20measuring%20variations%20in%20DNA%20copy-number%3A%20competitive%20genomic%20polymerase%20chain%20reaction&amp;author=K.%20Iwao-Koizumi&amp;author=K.%20Maekawa&amp;author=Y.%20Nakamura&amp;publication_year=2007&amp;journal=BMC%20Genomics&amp;volume=8&amp;pages=206" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-8-206" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-8-206" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-8-206"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17601344" class="link link-pub-id" target="_blank">PubMed</a></div></div></div></div></div></div></div><div content-id=""><div class="fn"><div class="footnote-content">Edited by Naotake Ogasawara</div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement"> The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=474733&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/3/10.1093/dnares/dss002/2/dss002-dss002supp.doc?Expires=2147483647&amp;Signature=j0iEkhvFlW~CKGp75xivWr-jYkfLVXgsP49rhE6j0ZW4V5Ns9XPn3-T5FMmyuREMlNSFjP9gOWdPiPOBacYGZsMIdI4z1UQiE28ojFPhuFTQClJTlFL4zTp8hrvfdNP0CKjUf4EUzBa-ZErC3ns9yNbMJJk5OCn~VC6WtQn~dEVSNgKCG18GXqRtqRBvMrhjcemYw60pFoFSQhJ6t6aMxjmaeXZ2Jv1PC-K-qHpvULYKtFjIYdUV2EEjUYlsKZNgp5WpBVlxv6WO4KE5Dyd871PEl6NgB3ZI5zcbiZhkDwazhByuAMSbBCQviTT5pBZwGuXx-WYSfji24qaSKqxBiw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">181</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000305723200001&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss002"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="474733"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000305723200001&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f3%2f209%2f474733">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000305723200001&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F3%2F209%2F474733" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/474733'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transpositionduplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Accurate Measurement of the Relative Abundance of Different DNA Species in Complex DNA Mixtures" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss002");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f3%2f209%2f474733">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

