

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/25/2/207/4675083","@type":"ScholarlyArticle","name":"Small RNA repertoires and their intraspecies variation in  Aggregatibacter actinomycetemcomitans ","datePublished":"2017-12-02","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/25/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2018-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx050","keywords":["small RNA","RNA-seq","<italic>Aggregatibacter actinomycetemcomitans</italic>"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/25/2/207/4675083","author":[{"name":"Oogai, Yuichi","affiliation":"Department of Oral Microbiology, Kagoshima University Graduate School of Medical and Dental Sciences, 8-35-1, Sakuragaoka, Kagoshima 890-8544, Japan","@type":"Person"},{"name":"Gotoh, Yasuhiro","affiliation":"Department of Bacteriology, Faculty of Medical Sciences, Kyushu University, 3-1-1 Maidashi, Higashi-ku, Fukuoka 812-8582, Japan","@type":"Person"},{"name":"Ogura, Yoshitoshi","affiliation":"Department of Bacteriology, Faculty of Medical Sciences, Kyushu University, 3-1-1 Maidashi, Higashi-ku, Fukuoka 812-8582, Japan","@type":"Person"},{"name":"Kawada-Matsuo, Miki","affiliation":"Department of Oral Microbiology, Kagoshima University Graduate School of Medical and Dental Sciences, 8-35-1, Sakuragaoka, Kagoshima 890-8544, Japan","@type":"Person"},{"name":"Hayashi, Tetsuya","affiliation":"Department of Bacteriology, Faculty of Medical Sciences, Kyushu University, 3-1-1 Maidashi, Higashi-ku, Fukuoka 812-8582, Japan","@type":"Person"},{"name":"Komatsuzawa, Hitoshi","affiliation":"Department of Oral Microbiology, Kagoshima University Graduate School of Medical and Dental Sciences, 8-35-1, Sakuragaoka, Kagoshima 890-8544, Japan","@type":"Person"}],"description":"Abstract.  Aggregatibacter actinomycetemcomitans is a major periodontal pathogen that has several virulence factors such as leukotoxin and cytolethal distending","pageStart":"207","pageEnd":"215","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Small RNA repertoires and their intraspecies variation in  Aggregatibacter actinomycetemcomitans ","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Small RNA repertoires and their intraspecies variation in  Aggregatibacter actinomycetemcomitans " />
    <meta property="og:description" content="Abstract.  Aggregatibacter actinomycetemcomitans is a major periodontal pathogen that has several virulence factors such as leukotoxin and cytolethal distending" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/25/2/207/4675083" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-12-02" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Oogai, Yuichi" /><meta name="citation_author_institution" content="Department of Oral Microbiology, Kagoshima University Graduate School of Medical and Dental Sciences, 8-35-1, Sakuragaoka, Kagoshima 890-8544, Japan" /><meta name="citation_author" content="Gotoh, Yasuhiro" /><meta name="citation_author_institution" content="Department of Bacteriology, Faculty of Medical Sciences, Kyushu University, 3-1-1 Maidashi, Higashi-ku, Fukuoka 812-8582, Japan" /><meta name="citation_author" content="Ogura, Yoshitoshi" /><meta name="citation_author_institution" content="Department of Bacteriology, Faculty of Medical Sciences, Kyushu University, 3-1-1 Maidashi, Higashi-ku, Fukuoka 812-8582, Japan" /><meta name="citation_author" content="Kawada-Matsuo, Miki" /><meta name="citation_author_institution" content="Department of Oral Microbiology, Kagoshima University Graduate School of Medical and Dental Sciences, 8-35-1, Sakuragaoka, Kagoshima 890-8544, Japan" /><meta name="citation_author" content="Hayashi, Tetsuya" /><meta name="citation_author_institution" content="Department of Bacteriology, Faculty of Medical Sciences, Kyushu University, 3-1-1 Maidashi, Higashi-ku, Fukuoka 812-8582, Japan" /><meta name="citation_author" content="Komatsuzawa, Hitoshi" /><meta name="citation_author_institution" content="Department of Oral Microbiology, Kagoshima University Graduate School of Medical and Dental Sciences, 8-35-1, Sakuragaoka, Kagoshima 890-8544, Japan" /><meta name="citation_title" content="Small RNA repertoires and their intraspecies variation in Aggregatibacter actinomycetemcomitans" /><meta name="citation_firstpage" content="207" /><meta name="citation_lastpage" content="215" /><meta name="citation_doi" content="10.1093/dnares/dsx050" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="25" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2018/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Aggregatibacter actinomycetemcomitans and its relationship to initiation of localized aggressive periodontitis: longitudinal cohort study of initially healthy adolescents;  citation_journal_title=J. Clin. Microbiol;  citation_year=2007;  citation_volume=45;  citation_pages=3859-69; " /><meta name="citation_reference" content="citation_title=Aggregatibacter actinomycetemcomitans leukotoxin: from threat to therapy;  citation_journal_title=J. Dent. Res;  citation_year=2010;  citation_volume=89;  citation_pages=561-70; " /><meta name="citation_reference" content="citation_title=Cytolethal distending toxin of Actinobacillus actinomycetemcomitans. Occurrence and association with periodontal disease;  citation_journal_title=J. Periodont. Res;  citation_year=2002;  citation_volume=37;  citation_pages=268-72; " /><meta name="citation_reference" content="citation_title=Regulatory RNAs in bacteria;  citation_journal_title=Cell;  citation_year=2009;  citation_volume=136;  citation_pages=615-28; " /><meta name="citation_reference" content="citation_title=Small RNA binding to 5′ mRNA coding region inhibits translational initiation;  citation_journal_title=Mol. Cell;  citation_year=2008;  citation_volume=32;  citation_pages=827-37; " /><meta name="citation_reference" content="citation_title=Coupled degradation of a small regulatory RNA and its mRNA targets in Escherichia coli;  citation_journal_title=Genes Dev;  citation_year=2003;  citation_volume=17;  citation_pages=2374-83; " /><meta name="citation_reference" content="citation_title=Activation of alpha-toxin translation in Staphylococcus aureus by the trans-encoded antisense RNA, RNAIII;  citation_journal_title=EMBO J;  citation_year=1995;  citation_volume=14;  citation_pages=4569-77; " /><meta name="citation_reference" content="citation_title=cis-antisense RNA, another level of gene regulation in bacteria;  citation_journal_title=Microbiol. Mol. Biol. Rev;  citation_year=2011;  citation_volume=75;  citation_pages=286-300; " /><meta name="citation_reference" content="citation_title=An internal antisense RNA regulates expression of the photosynthesis gene isiA;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2006;  citation_volume=103;  citation_pages=7054-8; " /><meta name="citation_reference" content="citation_title=An antisense RNA controls synthesis of an SOS-induced toxin evolved from an antitoxin;  citation_journal_title=Mol. Microbiol;  citation_year=2007;  citation_volume=64;  citation_pages=738-54; " /><meta name="citation_reference" content="citation_title=Computational small RNA prediction in bacteria;  citation_journal_title=Bioinform. Biol. Insights;  citation_year=2013;  citation_volume=7;  citation_pages=83-95; " /><meta name="citation_reference" content="citation_title=Experimental approaches for the discovery and characterization of regulatory small RNA;  citation_journal_title=Curr. Opin. Microbiol;  citation_year=2009;  citation_volume=12;  citation_pages=536-46; " /><meta name="citation_reference" content="citation_title=A survey of small RNA-encoding genes in Escherichia coli;  citation_journal_title=Nucleic Acids Res;  citation_year=2003;  citation_volume=31;  citation_pages=1813-20; " /><meta name="citation_reference" content="citation_title=Deep sequencing reveals as-yet-undiscovered small RNAs in Escherichia coli;  citation_journal_title=BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=428" /><meta name="citation_reference" content="citation_title=Genome-wide identification of novel small RNAs in Pseudomonas aeruginosa;  citation_journal_title=Environ. Microbiol;  citation_year=2012;  citation_volume=14;  citation_pages=2006-16; " /><meta name="citation_reference" content="citation_title=Characterization of a novel riboswitch-regulated lysine transporter in Aggregatibacter actinomycetemcomitans;  citation_journal_title=J. Bacteriol;  citation_year=2010;  citation_volume=192;  citation_pages=6240-50; " /><meta name="citation_reference" content="citation_title=Probing bacterial metabolism during infection using high-resolution transcriptomics;  citation_journal_title=J. Bacteriol;  citation_year=2013;  citation_volume=195;  citation_pages=4991-8; " /><meta name="citation_reference" content="citation_title=Novel iron-regulated and Fur-regulated small regulatory RNAs in Aggregatibacter actinomycetemcomitans;  citation_journal_title=Mol. Oral Microbiol;  citation_year=2012;  citation_volume=27;  citation_pages=327-49; " /><meta name="citation_reference" content="citation_title=Microevolution and patterns of dissemination of the JP2 clone of Aggregatibacter (Actinobacillus) actinomycetemcomitans;  citation_journal_title=Infect. Immun;  citation_year=2007;  citation_volume=75;  citation_pages=3080-8; " /><meta name="citation_reference" content="citation_title=The isolation and fractionation of soluble ribonucleic acids;  citation_journal_title=J. Mol. Biol;  citation_year=1962;  citation_volume=4;  citation_pages=347-56; " /><meta name="citation_reference" content="citation_title=Fast and accurate long-read alignment with Burrows–Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=589-95; " /><meta name="citation_reference" content="citation_title=Integrative genomics viewer;  citation_journal_title=Nat. Biotechnol;  citation_year=2011;  citation_volume=29;  citation_pages=24-6; " /><meta name="citation_reference" content="citation_title=Intragenomic recombination in the highly leukotoxic JP2 clone of Actinobacillus actinomycetemcomitans;  citation_journal_title=Microbiology (Reading, Engl.);  citation_year=2005;  citation_volume=151;  citation_pages=3371-9; " /><meta name="citation_reference" content="citation_title=MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0;  citation_journal_title=Mol. Biol. Evol;  citation_year=2007;  citation_volume=24;  citation_pages=1596-9; " /><meta name="citation_reference" content="citation_title=IntaRNA: efficient prediction of bacterial sRNA targets incorporating target site accessibility and seed regions;  citation_journal_title=Bioinformatics;  citation_year=2008;  citation_volume=24;  citation_pages=2849-56; " /><meta name="citation_reference" content="citation_title=The primary transcriptome of the major human pathogen Helicobacter pylori;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=464;  citation_pages=250-5; " /><meta name="citation_reference" content="citation_title=PolyU tail of rho-independent terminator of bacterial small RNAs is essential for Hfq action;  citation_journal_title=Proc. Natl. Acad. Sci. U.S.A;  citation_year=2011;  citation_volume=108;  citation_pages=13059-64; " /><meta name="citation_reference" content="citation_title=Prediction of non-coding and antisense RNA genes in Escherichia coli with Gapped Markov Model;  citation_journal_title=Gene;  citation_year=2006;  citation_volume=372;  citation_pages=171-81; " /><meta name="citation_reference" content="citation_title=Evidence for a major role of antisense RNAs in cyanobacterial gene regulation;  citation_journal_title=Mol. Syst. Biol;  citation_year=2009;  citation_volume=5;  citation_pages=305" /><meta name="citation_reference" content="citation_title=Cartography of methicillin-resistant S. aureus transcripts: detection, orientation and temporal expression during growth phase and stress conditions;  citation_journal_title=PLoS One;  citation_year=2010;  citation_volume=5;  citation_pages=e10725" /><meta name="citation_reference" content="citation_title=Detection and quantitative analysis of small RNAs by PCR;  citation_journal_title=Methods Mol. Biol;  citation_year=2010;  citation_volume=629;  citation_pages=295-305; " /><meta name="citation_reference" content="citation_title=High-resolution transcriptome maps reveal strain-specific regulatory features of multiple Campylobacter jejuni isolates;  citation_journal_title=PLoS Genet;  citation_year=2013;  citation_volume=9;  citation_pages=e1003495" /><meta name="citation_reference" content="citation_title=Relationship of Actinobacillus actinomycetemcomitans serotypes to periodontal condition: prevalence and proportions in subgingival plaque;  citation_journal_title=Eur. J. Oral Sci;  citation_year=2005;  citation_volume=113;  citation_pages=28-33; " /><meta name="citation_reference" content="citation_title=Genetic diversity and toxic activity of Aggregatibacter actinomycetemcomitans isolates;  citation_journal_title=Oral Microbiol. Immunol;  citation_year=2009;  citation_volume=24;  citation_pages=493-501; " /><meta name="citation_reference" content="citation_title=Evolutionary divergence of Aggregatibacter actinomycetemcomitans;  citation_journal_title=J. Dent. Res;  citation_year=2016;  citation_volume=95;  citation_pages=94-101; " /><meta name="citation_reference" content="citation_title=GadY, a small-RNA regulator of acid response genes in Escherichia coli;  citation_journal_title=J. Bacteriol;  citation_year=2004;  citation_volume=186;  citation_pages=6698-705; " /><meta name="citation_reference" content="citation_title=Outer membrane components of the Tad (tight adherence) secretion of Aggregatibacter actinomycetemcomitans;  citation_journal_title=J. Bacteriol;  citation_year=2008;  citation_volume=190;  citation_pages=980-90; " /><meta name="citation_reference" content="citation_title=Toxin–antitoxin loci vapBC-1 and vapXD contribute to survival and virulence in nontypeable Haemophilus influenzae;  citation_journal_title=BMC Microbiol;  citation_year=2012;  citation_volume=12;  citation_pages=263" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/25/2/207/24662224/dsx050.pdf" /><meta name="description" content="Abstract.  Aggregatibacter actinomycetemcomitans is a major periodontal pathogen that has several virulence factors such as leukotoxin and cytolethal distending" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/25/2/207/4675083" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/25/2/207/4675083" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/25/2/207/4675083");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2796125&account_list=2796125,2777753&authnips=128.235.12.133&doi=10.1093/dnares/dsx050';
    </script>
<title>
	Small RNA repertoires and their intraspecies variation in Aggregatibacter actinomycetemcomitans | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./4675083" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BfZfPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPb2dhaSwgWXVpY2hpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBPcmFsIE1pY3JvYmlvbG9neSwgS2Fnb3NoaW1hIFVuaXZlcnNpdHkgR3JhZHVhdGUgU2Nob29sIG9mIE1lZGljYWwgYW5kIERlbnRhbCBTY2llbmNlcywgOC0zNS0xLCBTYWt1cmFnYW9rYSwgS2Fnb3NoaW1hIDg5MC04NTQ0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkdvdG9oLCBZYXN1aGlybyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmFjdGVyaW9sb2d5LCBGYWN1bHR5IG9mIE1lZGljYWwgU2NpZW5jZXMsIEt5dXNodSBVbml2ZXJzaXR5LCAzLTEtMSBNYWlkYXNoaSwgSGlnYXNoaS1rdSwgRnVrdW9rYSA4MTItODU4MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJPZ3VyYSwgWW9zaGl0b3NoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQmFjdGVyaW9sb2d5LCBGYWN1bHR5IG9mIE1lZGljYWwgU2NpZW5jZXMsIEt5dXNodSBVbml2ZXJzaXR5LCAzLTEtMSBNYWlkYXNoaSwgSGlnYXNoaS1rdSwgRnVrdW9rYSA4MTItODU4MiwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLYXdhZGEtTWF0c3VvLCBNaWtpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBPcmFsIE1pY3JvYmlvbG9neSwgS2Fnb3NoaW1hIFVuaXZlcnNpdHkgR3JhZHVhdGUgU2Nob29sIG9mIE1lZGljYWwgYW5kIERlbnRhbCBTY2llbmNlcywgOC0zNS0xLCBTYWt1cmFnYW9rYSwgS2Fnb3NoaW1hIDg5MC04NTQ0LCBKYXBhbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkhheWFzaGksIFRldHN1eWEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIEJhY3RlcmlvbG9neSwgRmFjdWx0eSBvZiBNZWRpY2FsIFNjaWVuY2VzLCBLeXVzaHUgVW5pdmVyc2l0eSwgMy0xLTEgTWFpZGFzaGksIEhpZ2FzaGkta3UsIEZ1a3Vva2EgODEyLTg1ODIsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iS29tYXRzdXphd2EsIEhpdG9zaGkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIE9yYWwgTWljcm9iaW9sb2d5LCBLYWdvc2hpbWEgVW5pdmVyc2l0eSBHcmFkdWF0ZSBTY2hvb2wgb2YgTWVkaWNhbCBhbmQgRGVudGFsIFNjaWVuY2VzLCA4LTM1LTEsIFNha3VyYWdhb2thLCBLYWdvc2hpbWEgODkwLTg1NDQsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJTbWFsbCBSTkEgcmVwZXJ0b2lyZXMgYW5kIHRoZWlyIGludHJhc3BlY2llcyB2YXJpYXRpb24gaW4gQWdncmVnYXRpYmFjdGVyIGFjdGlub215Y2V0ZW1jb21pdGFucyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjIwNyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjE1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN4MDUwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTgvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZ2dyZWdhdGliYWN0ZXIgYWN0aW5vbXljZXRlbWNvbWl0YW5zIGFuZCBpdHMgcmVsYXRpb25zaGlwIHRvIGluaXRpYXRpb24gb2YgbG9jYWxpemVkIGFnZ3Jlc3NpdmUgcGVyaW9kb250aXRpczogbG9uZ2l0dWRpbmFsIGNvaG9ydCBzdHVkeSBvZiBpbml0aWFsbHkgaGVhbHRoeSBhZG9sZXNjZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQ2xpbi4gTWljcm9iaW9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9Mzg1OS02OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWdncmVnYXRpYmFjdGVyIGFjdGlub215Y2V0ZW1jb21pdGFucyBsZXVrb3RveGluOiBmcm9tIHRocmVhdCB0byB0aGVyYXB5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBEZW50LiBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9ODk7ICBjaXRhdGlvbl9wYWdlcz01NjEtNzA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUN5dG9sZXRoYWwgZGlzdGVuZGluZyB0b3hpbiBvZiBBY3Rpbm9iYWNpbGx1cyBhY3Rpbm9teWNldGVtY29taXRhbnMuIE9jY3VycmVuY2UgYW5kIGFzc29jaWF0aW9uIHdpdGggcGVyaW9kb250YWwgZGlzZWFzZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gUGVyaW9kb250LiBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9Mzc7ICBjaXRhdGlvbl9wYWdlcz0yNjgtNzI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlZ3VsYXRvcnkgUk5BcyBpbiBiYWN0ZXJpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2VsbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0xMzY7ICBjaXRhdGlvbl9wYWdlcz02MTUtMjg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNtYWxsIFJOQSBiaW5kaW5nIHRvIDXigLIgbVJOQSBjb2RpbmcgcmVnaW9uIGluaGliaXRzIHRyYW5zbGF0aW9uYWwgaW5pdGlhdGlvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBDZWxsOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTMyOyAgY2l0YXRpb25fcGFnZXM9ODI3LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db3VwbGVkIGRlZ3JhZGF0aW9uIG9mIGEgc21hbGwgcmVndWxhdG9yeSBSTkEgYW5kIGl0cyBtUk5BIHRhcmdldHMgaW4gRXNjaGVyaWNoaWEgY29saTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZXMgRGV2OyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MjM3NC04MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWN0aXZhdGlvbiBvZiBhbHBoYS10b3hpbiB0cmFuc2xhdGlvbiBpbiBTdGFwaHlsb2NvY2N1cyBhdXJldXMgYnkgdGhlIHRyYW5zLWVuY29kZWQgYW50aXNlbnNlIFJOQSwgUk5BSUlJOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1FTUJPIEo7ICBjaXRhdGlvbl95ZWFyPTE5OTU7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz00NTY5LTc3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1jaXMtYW50aXNlbnNlIFJOQSwgYW5vdGhlciBsZXZlbCBvZiBnZW5lIHJlZ3VsYXRpb24gaW4gYmFjdGVyaWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU1pY3JvYmlvbC4gTW9sLiBCaW9sLiBSZXY7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9NzU7ICBjaXRhdGlvbl9wYWdlcz0yODYtMzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBpbnRlcm5hbCBhbnRpc2Vuc2UgUk5BIHJlZ3VsYXRlcyBleHByZXNzaW9uIG9mIHRoZSBwaG90b3N5bnRoZXNpcyBnZW5lIGlzaUE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS5TLkE7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MTAzOyAgY2l0YXRpb25fcGFnZXM9NzA1NC04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbiBhbnRpc2Vuc2UgUk5BIGNvbnRyb2xzIHN5bnRoZXNpcyBvZiBhbiBTT1MtaW5kdWNlZCB0b3hpbiBldm9sdmVkIGZyb20gYW4gYW50aXRveGluOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT02NDsgIGNpdGF0aW9uX3BhZ2VzPTczOC01NDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcHV0YXRpb25hbCBzbWFsbCBSTkEgcHJlZGljdGlvbiBpbiBiYWN0ZXJpYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtLiBCaW9sLiBJbnNpZ2h0czsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT03OyAgY2l0YXRpb25fcGFnZXM9ODMtOTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV4cGVyaW1lbnRhbCBhcHByb2FjaGVzIGZvciB0aGUgZGlzY292ZXJ5IGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIHJlZ3VsYXRvcnkgc21hbGwgUk5BOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DdXJyLiBPcGluLiBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz01MzYtNDY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUEgc3VydmV5IG9mIHNtYWxsIFJOQS1lbmNvZGluZyBnZW5lcyBpbiBFc2NoZXJpY2hpYSBjb2xpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OdWNsZWljIEFjaWRzIFJlczsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT0zMTsgIGNpdGF0aW9uX3BhZ2VzPTE4MTMtMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURlZXAgc2VxdWVuY2luZyByZXZlYWxzIGFzLXlldC11bmRpc2NvdmVyZWQgc21hbGwgUk5BcyBpbiBFc2NoZXJpY2hpYSBjb2xpOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CTUMgR2Vub21pY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz00MjgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HZW5vbWUtd2lkZSBpZGVudGlmaWNhdGlvbiBvZiBub3ZlbCBzbWFsbCBSTkFzIGluIFBzZXVkb21vbmFzIGFlcnVnaW5vc2E7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUVudmlyb24uIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xNDsgIGNpdGF0aW9uX3BhZ2VzPTIwMDYtMTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNoYXJhY3Rlcml6YXRpb24gb2YgYSBub3ZlbCByaWJvc3dpdGNoLXJlZ3VsYXRlZCBseXNpbmUgdHJhbnNwb3J0ZXIgaW4gQWdncmVnYXRpYmFjdGVyIGFjdGlub215Y2V0ZW1jb21pdGFuczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE5MjsgIGNpdGF0aW9uX3BhZ2VzPTYyNDAtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByb2JpbmcgYmFjdGVyaWFsIG1ldGFib2xpc20gZHVyaW5nIGluZmVjdGlvbiB1c2luZyBoaWdoLXJlc29sdXRpb24gdHJhbnNjcmlwdG9taWNzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBCYWN0ZXJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTk1OyAgY2l0YXRpb25fcGFnZXM9NDk5MS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ob3ZlbCBpcm9uLXJlZ3VsYXRlZCBhbmQgRnVyLXJlZ3VsYXRlZCBzbWFsbCByZWd1bGF0b3J5IFJOQXMgaW4gQWdncmVnYXRpYmFjdGVyIGFjdGlub215Y2V0ZW1jb21pdGFuczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBPcmFsIE1pY3JvYmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTMyNy00OTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9ldm9sdXRpb24gYW5kIHBhdHRlcm5zIG9mIGRpc3NlbWluYXRpb24gb2YgdGhlIEpQMiBjbG9uZSBvZiBBZ2dyZWdhdGliYWN0ZXIgKEFjdGlub2JhY2lsbHVzKSBhY3Rpbm9teWNldGVtY29taXRhbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUluZmVjdC4gSW1tdW47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NzU7ICBjaXRhdGlvbl9wYWdlcz0zMDgwLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBpc29sYXRpb24gYW5kIGZyYWN0aW9uYXRpb24gb2Ygc29sdWJsZSByaWJvbnVjbGVpYyBhY2lkczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sOyAgY2l0YXRpb25feWVhcj0xOTYyOyAgY2l0YXRpb25fdm9sdW1lPTQ7ICBjaXRhdGlvbl9wYWdlcz0zNDctNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgYW5kIGFjY3VyYXRlIGxvbmctcmVhZCBhbGlnbm1lbnQgd2l0aCBCdXJyb3dz4oCTV2hlZWxlciB0cmFuc2Zvcm07ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9NTg5LTk1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRlZ3JhdGl2ZSBnZW5vbWljcyB2aWV3ZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gQmlvdGVjaG5vbDsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yOTsgIGNpdGF0aW9uX3BhZ2VzPTI0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUludHJhZ2Vub21pYyByZWNvbWJpbmF0aW9uIGluIHRoZSBoaWdobHkgbGV1a290b3hpYyBKUDIgY2xvbmUgb2YgQWN0aW5vYmFjaWxsdXMgYWN0aW5vbXljZXRlbWNvbWl0YW5zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1NaWNyb2Jpb2xvZ3kgKFJlYWRpbmcsIEVuZ2wuKTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNTE7ICBjaXRhdGlvbl9wYWdlcz0zMzcxLTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1FR0E0OiBNb2xlY3VsYXIgRXZvbHV0aW9uYXJ5IEdlbmV0aWNzIEFuYWx5c2lzIChNRUdBKSBzb2Z0d2FyZSB2ZXJzaW9uIDQuMDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TW9sLiBCaW9sLiBFdm9sOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTI0OyAgY2l0YXRpb25fcGFnZXM9MTU5Ni05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbnRhUk5BOiBlZmZpY2llbnQgcHJlZGljdGlvbiBvZiBiYWN0ZXJpYWwgc1JOQSB0YXJnZXRzIGluY29ycG9yYXRpbmcgdGFyZ2V0IHNpdGUgYWNjZXNzaWJpbGl0eSBhbmQgc2VlZCByZWdpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTI4NDktNTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBwcmltYXJ5IHRyYW5zY3JpcHRvbWUgb2YgdGhlIG1ham9yIGh1bWFuIHBhdGhvZ2VuIEhlbGljb2JhY3RlciBweWxvcmk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT00NjQ7ICBjaXRhdGlvbl9wYWdlcz0yNTAtNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9seVUgdGFpbCBvZiByaG8taW5kZXBlbmRlbnQgdGVybWluYXRvciBvZiBiYWN0ZXJpYWwgc21hbGwgUk5BcyBpcyBlc3NlbnRpYWwgZm9yIEhmcSBhY3Rpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVByb2MuIE5hdGwuIEFjYWQuIFNjaS4gVS5TLkE7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MTA4OyAgY2l0YXRpb25fcGFnZXM9MTMwNTktNjQ7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVByZWRpY3Rpb24gb2Ygbm9uLWNvZGluZyBhbmQgYW50aXNlbnNlIFJOQSBnZW5lcyBpbiBFc2NoZXJpY2hpYSBjb2xpIHdpdGggR2FwcGVkIE1hcmtvdiBNb2RlbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2VuZTsgIGNpdGF0aW9uX3llYXI9MjAwNjsgIGNpdGF0aW9uX3ZvbHVtZT0zNzI7ICBjaXRhdGlvbl9wYWdlcz0xNzEtODE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2aWRlbmNlIGZvciBhIG1ham9yIHJvbGUgb2YgYW50aXNlbnNlIFJOQXMgaW4gY3lhbm9iYWN0ZXJpYWwgZ2VuZSByZWd1bGF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFN5c3QuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTMwNSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNhcnRvZ3JhcGh5IG9mIG1ldGhpY2lsbGluLXJlc2lzdGFudCBTLiBhdXJldXMgdHJhbnNjcmlwdHM6IGRldGVjdGlvbiwgb3JpZW50YXRpb24gYW5kIHRlbXBvcmFsIGV4cHJlc3Npb24gZHVyaW5nIGdyb3d0aCBwaGFzZSBhbmQgc3RyZXNzIGNvbmRpdGlvbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTU7ICBjaXRhdGlvbl9wYWdlcz1lMTA3MjUiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZXRlY3Rpb24gYW5kIHF1YW50aXRhdGl2ZSBhbmFseXNpcyBvZiBzbWFsbCBSTkFzIGJ5IFBDUjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TWV0aG9kcyBNb2wuIEJpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NjI5OyAgY2l0YXRpb25fcGFnZXM9Mjk1LTMwNTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC1yZXNvbHV0aW9uIHRyYW5zY3JpcHRvbWUgbWFwcyByZXZlYWwgc3RyYWluLXNwZWNpZmljIHJlZ3VsYXRvcnkgZmVhdHVyZXMgb2YgbXVsdGlwbGUgQ2FtcHlsb2JhY3RlciBqZWp1bmkgaXNvbGF0ZXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgR2VuZXQ7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPWUxMDAzNDk1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVsYXRpb25zaGlwIG9mIEFjdGlub2JhY2lsbHVzIGFjdGlub215Y2V0ZW1jb21pdGFucyBzZXJvdHlwZXMgdG8gcGVyaW9kb250YWwgY29uZGl0aW9uOiBwcmV2YWxlbmNlIGFuZCBwcm9wb3J0aW9ucyBpbiBzdWJnaW5naXZhbCBwbGFxdWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUV1ci4gSi4gT3JhbCBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTEzOyAgY2l0YXRpb25fcGFnZXM9MjgtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmV0aWMgZGl2ZXJzaXR5IGFuZCB0b3hpYyBhY3Rpdml0eSBvZiBBZ2dyZWdhdGliYWN0ZXIgYWN0aW5vbXljZXRlbWNvbWl0YW5zIGlzb2xhdGVzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1PcmFsIE1pY3JvYmlvbC4gSW1tdW5vbDsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNDsgIGNpdGF0aW9uX3BhZ2VzPTQ5My01MDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUV2b2x1dGlvbmFyeSBkaXZlcmdlbmNlIG9mIEFnZ3JlZ2F0aWJhY3RlciBhY3Rpbm9teWNldGVtY29taXRhbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIERlbnQuIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT05NTsgIGNpdGF0aW9uX3BhZ2VzPTk0LTEwMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2FkWSwgYSBzbWFsbC1STkEgcmVndWxhdG9yIG9mIGFjaWQgcmVzcG9uc2UgZ2VuZXMgaW4gRXNjaGVyaWNoaWEgY29saTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gQmFjdGVyaW9sOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTE4NjsgIGNpdGF0aW9uX3BhZ2VzPTY2OTgtNzA1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1PdXRlciBtZW1icmFuZSBjb21wb25lbnRzIG9mIHRoZSBUYWQgKHRpZ2h0IGFkaGVyZW5jZSkgc2VjcmV0aW9uIG9mIEFnZ3JlZ2F0aWJhY3RlciBhY3Rpbm9teWNldGVtY29taXRhbnM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUouIEJhY3RlcmlvbDsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xOTA7ICBjaXRhdGlvbl9wYWdlcz05ODAtOTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRveGlu4oCTYW50aXRveGluIGxvY2kgdmFwQkMtMSBhbmQgdmFwWEQgY29udHJpYnV0ZSB0byBzdXJ2aXZhbCBhbmQgdmlydWxlbmNlIGluIG5vbnR5cGVhYmxlIEhhZW1vcGhpbHVzIGluZmx1ZW56YWU7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJNQyBNaWNyb2Jpb2w7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0yNjMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjUvMi8yMDcvMjQ2NjIyMjQvZHN4MDUwLnBkZiIgLz48bWV0YSBuYW1lPSJkZXNjcmlwdGlvbiIgY29udGVudD0iQWJzdHJhY3QuICBBZ2dyZWdhdGliYWN0ZXIgYWN0aW5vbXljZXRlbWNvbWl0YW5zIGlzIGEgbWFqb3IgcGVyaW9kb250YWwgcGF0aG9nZW4gdGhhdCBoYXMgc2V2ZXJhbCB2aXJ1bGVuY2UgZmFjdG9ycyBzdWNoIGFzIGxldWtvdG94aW4gYW5kIGN5dG9sZXRoYWwgZGlzdGVuZGluZyIgLz5kAgQPZBYCZg9kFgJmDxYEHgRocmVmBXMvL291cC5zaWx2ZXJjaGFpci1jZG4uY29tL0Rvd25sb2FkRmlsZS9Eb3dubG9hZEltYWdlLmFzcHg/aW1hZ2U9JlBQVHR5cGU9U2xpZGVTZXQmYXI9NDY3NTA4MyZ4c2x0UGF0aD1+L1VJL2FwcC9YU0xUHglpbm5lcmh0bWwFFERvd25sb2FkIGFsbCBmaWd1cmVzZGT37UODdghB8hOKREz4tztZU6ohxw==" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">New Jersey Institute of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

New Jersey Institute of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f207%2f4675083" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f207%2f4675083" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=108668" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/25/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/2/1/m_dnares_25_2cover.png?Expires=2147483647&amp;Signature=yfSnSDuEb8Ru1iEiczh9QiVIRujFlv5eQUGA2PixxhIP2ifTZWTC6Dh4YwZbNdfMcc6NjI5nmXhBBnXl~MO-9TlTyrmfE~hJJOwCiRzc0uAKzzN1WszCYLPzg4YPUGBBBvmjvYjtWeuxzT6HMUtRCVQ1Lykx1-FxL9MPbgabfK3LfM0lfiO9VFwha2rRByA4174pANV1EEHkTLYoj9VZAqG9NluBapcroduGQ77QCNh6n6sU6cwRapmFfSHO~HcCIdSas62HDXNUy3sCegaV5-lnkgDK34Vs8bE~hWbIr-4dZ69GanJYFX-rhZ4a7s6UpCy38dZKnOnrMjKCPyIBeg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/25/2">
                        <div class="volume trailing-comma">Volume 25</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2018        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="115733896">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733896">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733898">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733898">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733904">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733904">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733923">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733923">3. Results and discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733945">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733945">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733947">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733947">Accession numbers</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733949">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733949">Conflict of interest</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733951">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733951">Supplementary data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="115733953">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#115733953">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/25/2/195/4638432">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/25/2/217/4683712">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Small RNA repertoires and their intraspecies variation in <em>Aggregatibacter actinomycetemcomitans</em>
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yuichi Oogai</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Oral Microbiology, Kagoshima University Graduate School of Medical and Dental Sciences, 8-35-1, Sakuragaoka, Kagoshima 890-8544, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yuichi+Oogai">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Oogai Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Oogai Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yuichi Oogai</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yasuhiro Gotoh</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Bacteriology, Faculty of Medical Sciences, Kyushu University, 3-1-1 Maidashi, Higashi-ku, Fukuoka 812-8582, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yasuhiro+Gotoh">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Gotoh Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Gotoh Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yasuhiro Gotoh</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yoshitoshi Ogura</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Bacteriology, Faculty of Medical Sciences, Kyushu University, 3-1-1 Maidashi, Higashi-ku, Fukuoka 812-8582, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yoshitoshi+Ogura">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Ogura Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Ogura Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yoshitoshi Ogura</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Miki Kawada-Matsuo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Oral Microbiology, Kagoshima University Graduate School of Medical and Dental Sciences, 8-35-1, Sakuragaoka, Kagoshima 890-8544, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Miki+Kawada-Matsuo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kawada-Matsuo M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kawada-Matsuo M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Miki Kawada-Matsuo</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Tetsuya Hayashi</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Bacteriology, Faculty of Medical Sciences, Kyushu University, 3-1-1 Maidashi, Higashi-ku, Fukuoka 812-8582, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Tetsuya+Hayashi">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hayashi T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hayashi T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Tetsuya Hayashi</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hitoshi Komatsuzawa</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Oral Microbiology, Kagoshima University Graduate School of Medical and Dental Sciences, 8-35-1, Sakuragaoka, Kagoshima 890-8544, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx050-cor1" data-open="dsx050-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed. Tel. +81-99-275-6150. Fax. +81-99-275-6158. Email: <a href="mailto:hkomatsu@dent.kagoshima-u.ac.jp" target="_blank">hkomatsu@dent.kagoshima-u.ac.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hitoshi+Komatsuzawa">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Komatsuzawa H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Komatsuzawa H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hitoshi Komatsuzawa</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx050-FM1"><div class="fn"><div class="footnote-content">Edited by Dr. Katsumi Isono</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 25, Issue 2, 1 April 2018, Pages 207–215, <a href='https://doi.org/10.1093/dnares/dsx050'>https://doi.org/10.1093/dnares/dsx050</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">02 December 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">06 April 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">15 November 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="4675083" href="/dnaresearch/article-pdf/25/2/207/24662224/dsx050.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Yuichi Oogai, Yasuhiro Gotoh, Yoshitoshi Ogura, Miki Kawada-Matsuo, Tetsuya Hayashi, Hitoshi Komatsuzawa;  Small RNA repertoires and their intraspecies variation in <em>Aggregatibacter actinomycetemcomitans</em>, <em>DNA Research</em>, Volume 25, Issue 2, 1 April 2018, Pages 207–215, <a href='https://doi.org/10.1093/dnares/dsx050'>https://doi.org/10.1093/dnares/dsx050</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/4675083?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/4675083?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/4675083?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/4675083?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/4675083?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/4675083?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/4675083?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Small%20RNA%20repertoires%20and%20their%20intraspecies%20variation%20in%20%26lt%3Bem%26gt%3BAggregatibacter%20actinomycetemcomitans%26lt%3B%2Fem%26gt%3B&amp;publicationDate=2017-12-02&amp;volumeNum=25&amp;issueNum=2&amp;author=Oogai%2C%20Yuichi%3B%20Gotoh%2C%20Yasuhiro&amp;startPage=207&amp;endPage=215&amp;contentId=10.1093%2Fdnares%2Fdsx050&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            New Jersey Institute of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

New Jersey Institute of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f207%2f4675083" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=115733896 id="115733896" class="abstract-title" >Abstract</h2>
<section class="abstract"><p><em>Aggregatibacter actinomycetemcomitans</em> is a major periodontal pathogen that has several virulence factors such as leukotoxin and cytolethal distending toxin. Although the genes responsible for virulence have been identified, little is known about their regulatory mechanisms. Small RNA (sRNA) has been recognized as an important factor for gene regulation. To identify new regulatory mechanisms via sRNA in <em>A. actinomycetemcomitans</em> HK1651, we performed a systematic search for sRNAs by RNA-seq and identified 90 intergenic region sRNAs and 30 antisense sRNAs. Of the 85 analysable sRNAs, we successfully detected and quantified 70 sRNAs by developing an RT-PCR system, and we identified 17 sRNAs that were differentially expressed during different growth phases. In addition, we found notable intraspecies variation in the sRNA repertoire of <em>A. actinomycetemcomitans</em>, thus suggesting that frequent acquisition or deletion of sRNAs occurred during the evolution of this species. The predicted target genes of the intergenic region sRNAs indicated the possibility of sRNA interaction with several virulence genes including leukotoxin and cytolethal distending toxin. Our results should serve as an important genomic and genetic basis for future studies to fully understand the regulatory network in <em>A. actinomycetemcomitans</em> and provide new insights into the intraspecies variation of the bacterial sRNA repertoire in bacteria.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;small RNA&quot;">small RNA</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="RNA-seq">RNA-seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Aggregatibacter actinomycetemcomitans&quot;"><em>Aggregatibacter actinomycetemcomitans</em></a></div>                            <h2 scrollto-destination=115733898 id="115733898" class="section-title" >1. Introduction</h2>
<p><em>Aggregatibacter actinomycetemcomitans</em> is a Gram-negative facultative anaerobe. It is a major periodontal pathogen that is often associated with aggressive periodontitis.<a reveal-id="dsx050-B1" data-open="dsx050-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> Two exotoxins, leukotoxin and cytolethal distending toxin (Cdt), have been reported as the major virulence factors of <em>A. actinomycetemcomitans</em>. Leukotoxin is a membrane-active toxin that specifically kills leukocytes and contributes to the evasion of the host immune response.<a reveal-id="dsx050-B2" data-open="dsx050-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a> Cdt inhibits the proliferation of host cells, including lymphocytes and gingival fibroblasts, by arresting the cell cycle, and it can also induce apoptosis.<a reveal-id="dsx050-B3" data-open="dsx050-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a></p><p>In the past decade, a group of non-coding RNAs known as small RNAs (sRNAs) have been identified as novel regulatory factors of gene expression in various bacteria, including several pathogens.<a reveal-id="dsx050-B4" data-open="dsx050-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> The regulatory functions of sRNAs can be categorized into two types: <em>trans</em>-acting and <em>cis</em>-acting. The <em>trans</em>-acting regulation is mediated by sRNAs that are encoded in the intergenic region (IGR) and have been termed IGR sRNAs. Most IGR sRNAs contain a stem–loop structure. Via short, imperfect base paring between the loop and the target RNAs, IGR sRNAs regulate the expression of multiple genes located at several genome positions. Both negative and positive regulations by IGR sRNA have been demonstrated.<a reveal-id="dsx050-B4" data-open="dsx050-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a> In negative regulation, IGR sRNA base pairs with a ribosome-binding site (RBS) in the target mRNA and inhibits translation by preventing the ribosome from recognizing an mRNA.<a reveal-id="dsx050-B5" data-open="dsx050-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> IGR sRNAs can also affect the stability of mRNA by binding to target mRNAs and recruiting endoribonucleases to accelerate the degradation of the target mRNAs.<a reveal-id="dsx050-B6" data-open="dsx050-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> Some IGR sRNAs can positively regulate gene expression via translational activation or RNA protection. For example, the <em>Staphylococcus aureus</em> sRNA RNAIII is a global regulator of virulence genes that enhances the translation of alpha-toxin (Hla). The binding of RNAIII to the 5′ untranslated region (UTR) of <em>hla</em> mRNA affects the secondary structure of the mRNA, thus enhancing ribosomal access and translation.<a reveal-id="dsx050-B7" data-open="dsx050-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> The <em>cis</em>-acting sRNA regulation is mediated by ‘antisense sRNAs (asRNAs)’, which are encoded on the opposite strand of the same loci as the target RNA. Several regulatory mechanisms have been reported for asRNAs including target RNA degradation and translational inhibition.<a reveal-id="dsx050-B8" data-open="dsx050-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> For example, the degradation of the <em>isrA</em> mRNA of <em>Synechocystis</em> PCC 6803 is enhanced by the binding of an asRNA (IsiR).<a reveal-id="dsx050-B9" data-open="dsx050-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> In the SymE–SymR toxin–antitoxin system of <em>Escherichia coli</em>, the SymR asRNA binds the RBS of <em>symE</em> mRNA and inhibits its translation.<a reveal-id="dsx050-B10" data-open="dsx050-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a></p><p>Systematic searches for bacterial sRNAs have recently been performed by using various comprehensive methods such as computational prediction, a tiling array analysis, and RNA sequencing (RNA-seq) with high-throughput next generation sequencers.<a reveal-id="dsx050-B11" data-open="dsx050-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a><sup>,</sup><a reveal-id="dsx050-B12" data-open="dsx050-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> In <em>E. coli</em>, more than 400 sRNAs have been identified, and some of them have been characterized.<a reveal-id="dsx050-B13" data-open="dsx050-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a><sup>,</sup><a reveal-id="dsx050-B14" data-open="dsx050-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> In <em>Pseudomonas aeruginosa</em>, more than 500 sRNAs have been identified by RNA-seq, nearly 90% of which have no orthologues in other bacterial species.<a reveal-id="dsx050-B15" data-open="dsx050-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a></p><p>Efforts to identify sRNAs in <em>A. actinomycetemcomitans</em> have also been undertaken. Jorth et al. have used northern blotting to identify 9 sRNAs from 35 sRNA candidates predicted by bioinformatics in strain VT1169.<a reveal-id="dsx050-B16" data-open="dsx050-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> The same group has performed a more systematic screening of sRNA by using RNA-seq in strain 624 and has identified 202 sRNAs.<a reveal-id="dsx050-B17" data-open="dsx050-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> In strain HK1651, four sRNAs that are expressed under iron-limiting conditions have been identified.<a reveal-id="dsx050-B18" data-open="dsx050-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a></p><p>In the present study, we systematically screened sRNAs by RNA-seq to determine the repertoire of sRNAs in <em>A. actinomycetemcomitans</em> strain HK1651, which is a clinical isolate from an aggressive periodontitis patient and a JP2 clonal isolate.<a reveal-id="dsx050-B19" data-open="dsx050-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> We subsequently validated the expression of the identified sRNAs by RT-PCR, analysed the distribution of sRNAs in four other <em>A. actinomycetemcomitans</em> strains and determined the expression pattern of the sRNAs during growth.</p>                            <h2 scrollto-destination=115733904 id="115733904" class="section-title" >2. Materials and methods</h2>
                            <h3 scrollto-destination=115733905 id="115733905" class="section-title" >2.1. Bacterial strains and growth conditions</h3>
<p>We analysed five <em>A. actinomycetemcomitans</em> strains: HK1651 (serotype b), ATCC29523 (serotype a), Y4 (serotype b), NCTC9710 (serotype c), and IDH781 (serotype d) in the present study. Of these, the genomes of HK1651 and IDH781 have been sequenced (CP008984.1 and CP016553.1). All strains were grown in Todd Hewitt broth (Becton, Dickinson and Company, Franklin Lakes, NJ, USA) supplemented with 1% (wt/vol) yeast extract (Nacalai Tesque, Kyoto, Japan) (THY broth) at 37°C under 5% CO<sub>2</sub> without shaking.</p>                            <h3 scrollto-destination=115733907 id="115733907" class="section-title" >2.2. RNA isolation</h3>
<p>A small aliquot (10<sup>8</sup> CFU) of an overnight culture of strain HK1651 was inoculated into 30 ml of THY broth and grown to early-exponential phase [optical density at 660 nm (OD<sub>660</sub>) of 0.1] or late-exponential phase (OD<sub>660</sub> of 0.2) (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_figure_s1_and_2.doc?Expires=2147483647&Signature=r9PcC7eogLonH7RRZLPVNtPMhCjocZSXVxe1YiQ4TYL5-~L7d-qEBTu6XPBLAh~ISZI05D7LIv-ZoVwvxYflq0mvGJtcj3rdggtYcchcNYE2XsX~nizmftPScVQ3UWJzK~Hsmj26ezTLKBP8gDLw6E9h5k4nGXAGlCwPOEp4Yvtsp1qkXrlP8FAkMSPc5d0y0DIU5~JBTjk60pyXjA950TVCRyYS32xhcyXrTTHSCoaDK-FZJwUiMQ25BNdZuj-QNIGuWj7q~Epwp5C-~sthW4Zid2F~TnIX8dwjf10cFPhl8o4cgNrlAx6KYlEGo8pojU~BjEKrQXm3jgTiSaUt~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). The total RNA was extracted using the acidic phenol method<a reveal-id="dsx050-B20" data-open="dsx050-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> with several modifications. Briefly, the bacterial culture was pelleted by centrifugation (3,000 × <em>g</em> for 10 min) and re-suspended in ice-cold resuspension buffer (50 mM sodium acetate, 10 mM magnesium acetate). Total RNA was extracted twice with acidic phenol (saturated with citrate buffer, pH 4.3) extraction, followed by a chloroform extraction and ethanol precipitation. The sample was treated with TURBO<sup>TM</sup> DNase (Thermo Fisher Scientific, Inc., Waltham, MA, USA) (20 units/ml) to remove the DNA. One microgram of RNA was electrophoresed on a TBE-Urea (9M)-acrylamide (8%) gel and visualized by ethidium bromide staining to confirm the RNA quality. </p>                            <h3 scrollto-destination=115733909 id="115733909" class="section-title" >2.3. RNA-seq analysis</h3>
<p>RNA-seq was performed on two independently prepared samples from early-exponential phase cells. Sequencing libraries were prepared from each RNA preparation without fragmentation by using the NEBNext<sup>®</sup> Multiplex Small RNA Library Prep Set for Illumina<sup>®</sup> (New England Biolabs, Ipswich, MA, USA) according to the manufacturer’s protocol, with a minor modification. This kit is specialized for sequencing eukaryotic sRNAs that contain a 5′ monophosphate, such as miRNA and siRNA. Therefore, to analyse sRNAs containing a 5′ monophosphate terminus or a triphosphate terminus, the total RNA was treated with RNA 5′ poly phosphatase (Epicentre, Madison, WI, USA) according to the manufacturer’s instructions before a sequencing library was prepared (<a reveal-id="dsx050-F1" data-open="dsx050-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>). Paired-end sequencing (2 × 300 bp) was performed on the MiSeq platform (Illumina, San Diego, CA, USA) according to the manufacturer’s protocol. After removal of the low-quality reads and trimming of the adaptor sequences using the FASTX-Toolkit (<a class="link link-uri" href="http://hannonlab.cshl.edu/fastx_toolkit/index.html" target="_blank">http://hannonlab.cshl.edu/fastx_toolkit/index.html</a>), the sequence reads were strand-specifically mapped to the HK1651 genome sequence by using the Burrows–Wheeler Aligner,<a reveal-id="dsx050-B21" data-open="dsx050-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> and the data were visualized with Integrative Genomics Viewer.<a reveal-id="dsx050-B22" data-open="dsx050-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> The sequence reads obtained have been deposited in the GenBank/EMBL/DDBJ database (DRA submission number, DRA005568; DRR accession number, DRR088289). </p>                        <a id="115733911" scrollto-destination="115733911"></a>
<div id="dsx050-F1" data-id="dsx050-F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/m_dsx050f1.png?Expires=2147483647&Signature=LfhBiyx6miUlOIm4tbBNYWutRB793jCbk1K17IrQS2gOXTUuCu~pIvVZ3p4~Q6Dg8usn59MlaW5XJ5FUFePzoUjUbOVc8g8JaYduxqtdxMu6peTXDIm7zKX5gq~44jVjSMEKJg3HCEtKlSISLtynZQLA3tsvwMEL5LFL4qwJj2vYQxSjDTWyX4Sdeub6RwUxo47bcA~8F0jjyracY~fpIfgftUypmRrXRtYwgwu-JRNxNoIZcsjwiY-YUVVfZGO9Vnke~7PkUmnJ0KBtlCGdmxTzuiNM1s-Ud3T4-FCK6EzHVBGzEW1~eWOxPVlQmIEv7UvZAobuTPsPqoAHK02uXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A flowchart illustrating sRNA identification in A. actinomycetemcomitans HK1651." path-from-xml="dsx050f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733911/dsx050f1.tif" path-from-xml="dsx050f1.tif" target="_blank">View large</a><a section="115733911" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050f1.png?Expires=2147483647&Signature=EwvHpQtNv01KtiPiexYJgKcw~KBCg0Od6l7xlTFlJ-3elM2q7VT4~BV9Y5cs81NTj7JaqYK1eZ-4lOxpV0SX3pBLx74GP7XSzJNbXRQLNHTVxywHD8qSC9d~AgRjWtl8owG-fSYSKeXM5lgPKn3aVlmCS8x25-t3kw-zJHJbC-G3qA2SW9mIQ9svPVYgelPNC4BQmpKJ9DB7xPDFZxzD4UyT0rC~-FrZsMKHlF9ivZBTfNGAQx08ujugn7p1L~K3b7z6T2WA8zZgauVSO21GJHwfI8fUYZgtI5MLQNmqmVsJCpKxUXhOJtaBLW4IPMSOrO0tiwZEmQ~o4DH4EYcOTA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733911&ar=4675083&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx050f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A flowchart illustrating sRNA identification in <em>A. actinomycetemcomitans</em> HK1651.</p></div></div><div content-id="dsx050-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/m_dsx050f1.png?Expires=2147483647&Signature=LfhBiyx6miUlOIm4tbBNYWutRB793jCbk1K17IrQS2gOXTUuCu~pIvVZ3p4~Q6Dg8usn59MlaW5XJ5FUFePzoUjUbOVc8g8JaYduxqtdxMu6peTXDIm7zKX5gq~44jVjSMEKJg3HCEtKlSISLtynZQLA3tsvwMEL5LFL4qwJj2vYQxSjDTWyX4Sdeub6RwUxo47bcA~8F0jjyracY~fpIfgftUypmRrXRtYwgwu-JRNxNoIZcsjwiY-YUVVfZGO9Vnke~7PkUmnJ0KBtlCGdmxTzuiNM1s-Ud3T4-FCK6EzHVBGzEW1~eWOxPVlQmIEv7UvZAobuTPsPqoAHK02uXA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="A flowchart illustrating sRNA identification in A. actinomycetemcomitans HK1651." path-from-xml="dsx050f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733911/dsx050f1.tif" path-from-xml="dsx050f1.tif" target="_blank">View large</a><a section="115733911" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050f1.png?Expires=2147483647&Signature=EwvHpQtNv01KtiPiexYJgKcw~KBCg0Od6l7xlTFlJ-3elM2q7VT4~BV9Y5cs81NTj7JaqYK1eZ-4lOxpV0SX3pBLx74GP7XSzJNbXRQLNHTVxywHD8qSC9d~AgRjWtl8owG-fSYSKeXM5lgPKn3aVlmCS8x25-t3kw-zJHJbC-G3qA2SW9mIQ9svPVYgelPNC4BQmpKJ9DB7xPDFZxzD4UyT0rC~-FrZsMKHlF9ivZBTfNGAQx08ujugn7p1L~K3b7z6T2WA8zZgauVSO21GJHwfI8fUYZgtI5MLQNmqmVsJCpKxUXhOJtaBLW4IPMSOrO0tiwZEmQ~o4DH4EYcOTA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733911&ar=4675083&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx050f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>A flowchart illustrating sRNA identification in <em>A. actinomycetemcomitans</em> HK1651.</p></div></div>                            <h3 scrollto-destination=115733912 id="115733912" class="section-title" >2.4. Northern blot analysis</h3>
<p>Twenty micrograms of total RNA extracted from HK1651 harvested during the early-exponential phases was electrophoresed on a TBE-Urea (9M)-acrylamide (8%) gel. The separated RNA was transferred onto a positively charged nylon membrane (Roche Diagnostics, Basel, Switzerland) in 20× SSC. The membrane was baked for 2 h at 80°C. After prehybridization using DIG Easy Hyb (Roche Diagnostics), the membrane was hybridized with DIG Easy Hyb containing a 25 pmol of single-stranded DNA probe (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s1-s4.doc?Expires=2147483647&Signature=KU8FqWWPovL5rUPUrBluwDtnWxfiisdSa~CodmdqLpr~k-ylhOwNe2BWcTdweHE19eyhqTRs0OOCJcqvy7MXXtMruhabhFn2fIyd~M9QN4XTyavqVjHlhddbDHOdESKsW7bsOWI2lOY5tprQ8caH9fTNcTSDmsTEA10I4pLGUwk9UF2H6-GBE6Htnrwh2WNvGOoRL7rqoV~m5tLciuc6S8Ag2qMsUPG409sR4H2Wjm1jBzOSsY88Au7p8EqTseGap-xznvlaTzR4a1ooSriCMpx6A6J9mpO6Ok4E7-scCvmTF5siNoGvYnkwO7jGP~BPERqIYyGldXgAf-M15k~39w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) labelled with digoxigenin using DIG Oligonucleotide Tailing Kit, 2nd generation (Roche Diagnostics) for 18 h at 45°C. After hybridization, the membrane was washed and blocked using DIG Wash and Block Buffer Set (Roche Diagnostics), and then the hybridization signals were detected using DIG Luminescent Detection Kit (Roche Diagnostics) and ChemiDoc™ XRS+ (Bio-Rad Laboratories, Inc., Hercules, CA, USA).</p>                            <h3 scrollto-destination=115733914 id="115733914" class="section-title" >2.5. RT-PCR and quantitative RT-PCR</h3>
<p>Total RNA extracted from five <em>A. actinomycetemcomitans</em> strains (HK1651, ATCC29523, Y4, NCTC9710, and IDH781) harvested during the early- and late-exponential phases was polyadenylated using <em>E. coli</em> poly (A) polymerase (New England Biolabs) at 37°C for 30 min. This was followed by phenol/chloroform extraction and ethanol precipitation. One microgram of the RNA was then converted to cDNA with a Transcriptor First Strand cDNA Synthesis Kit (Roche Diagnostics) with an oligo d(T) primer containing a sequence tag at the 5′-end (5′-AAGCAGTGGTATCAACGCAGAGTACTTTTTTTTTTTTTTTTTTTTVN-3′). The cDNA was amplified using FastStart Essential DNA Green Master reaction mix (Roche Diagnostics) with 5′-primers specific to each sRNA and a 3′-primer (5′-CAGTGGTATCAACGCAGAGT-3′) that targeted the previously mentioned sequence tag. The sRNA specific primers are listed in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s1-s4.doc?Expires=2147483647&Signature=KU8FqWWPovL5rUPUrBluwDtnWxfiisdSa~CodmdqLpr~k-ylhOwNe2BWcTdweHE19eyhqTRs0OOCJcqvy7MXXtMruhabhFn2fIyd~M9QN4XTyavqVjHlhddbDHOdESKsW7bsOWI2lOY5tprQ8caH9fTNcTSDmsTEA10I4pLGUwk9UF2H6-GBE6Htnrwh2WNvGOoRL7rqoV~m5tLciuc6S8Ag2qMsUPG409sR4H2Wjm1jBzOSsY88Au7p8EqTseGap-xznvlaTzR4a1ooSriCMpx6A6J9mpO6Ok4E7-scCvmTF5siNoGvYnkwO7jGP~BPERqIYyGldXgAf-M15k~39w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>. The specific primers were designed within 50 bp from the 5′ terminus of each sRNA to amplify the nearly full-length sRNAs. In cases in which the sRNAs had high homology to other loci, the primers were designed to contain two nucleotide mismatches at the 3′ termini to prevent non-specific amplification. The thermal cycling conditions used were a 10-min incubation at 95°C followed by 25–45 cycles of 10 s at 95°C and 30 s at 58–62°C. The number of amplification cycles and the annealing/elongation temperatures were altered according to the amplification efficiency of each sRNA. The RT-PCR products were electrophoresed on a 2% agarose gel and visualized with ethidium bromide staining. Each sRNA was analysed two or three times using independently prepared RNA samples.</p><p>Quantitative RT-PCR (qPCR) was performed using a LightCycler Nano (Roche Diagnostics) and the same experimental procedures described for the RT-PCR assay (polyadenylation, cDNA conversion, and PCR primers). The transcription level was normalized to 5S rRNA (5′-primer; 5′-TGAAACCATACCGAACTCAG-3′) in a 1,000-fold diluted cDNA sample. Each sRNA was analysed by qPCR three times using independently prepared RNA samples. The EZR version 1.32 software (<a class="link link-uri" href="http://www.jichi.ac.jp/saitama-sct/SaitamaHP.files/statmedEN.html" target="_blank">http://www.jichi.ac.jp/saitama-sct/SaitamaHP.files/statmedEN.html</a>) was used for statistical analyses.</p>                            <h3 scrollto-destination=115733917 id="115733917" class="section-title" >2.6. Genomic DNA PCR</h3>
<p><em>Aggregatibacter actinomycetemcomitans</em> strains were cultivated in 5 ml of THY broth at 37°C under 5% CO<sub>2</sub> without shaking for 24 h. The genomic DNA was extracted using a DNeasy Blood &amp; Tissue kit (Qiagen, Hilden, Germany), and 100 ng was used as a template for PCR. PCR was performed using BIOTAQ™ DNA Polymerase (Bioline Reagents Ltd, London, UK) with the 5′ primers listed in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s1-s4.doc?Expires=2147483647&Signature=KU8FqWWPovL5rUPUrBluwDtnWxfiisdSa~CodmdqLpr~k-ylhOwNe2BWcTdweHE19eyhqTRs0OOCJcqvy7MXXtMruhabhFn2fIyd~M9QN4XTyavqVjHlhddbDHOdESKsW7bsOWI2lOY5tprQ8caH9fTNcTSDmsTEA10I4pLGUwk9UF2H6-GBE6Htnrwh2WNvGOoRL7rqoV~m5tLciuc6S8Ag2qMsUPG409sR4H2Wjm1jBzOSsY88Au7p8EqTseGap-xznvlaTzR4a1ooSriCMpx6A6J9mpO6Ok4E7-scCvmTF5siNoGvYnkwO7jGP~BPERqIYyGldXgAf-M15k~39w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span> and the 3′ primers listed in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s1-s4.doc?Expires=2147483647&Signature=KU8FqWWPovL5rUPUrBluwDtnWxfiisdSa~CodmdqLpr~k-ylhOwNe2BWcTdweHE19eyhqTRs0OOCJcqvy7MXXtMruhabhFn2fIyd~M9QN4XTyavqVjHlhddbDHOdESKsW7bsOWI2lOY5tprQ8caH9fTNcTSDmsTEA10I4pLGUwk9UF2H6-GBE6Htnrwh2WNvGOoRL7rqoV~m5tLciuc6S8Ag2qMsUPG409sR4H2Wjm1jBzOSsY88Au7p8EqTseGap-xznvlaTzR4a1ooSriCMpx6A6J9mpO6Ok4E7-scCvmTF5siNoGvYnkwO7jGP~BPERqIYyGldXgAf-M15k~39w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>. The 3′ primers were designed ∼100 bp downstream of the 5′ primer for each target genomic region. The thermal cycling conditions were a 2-min incubation at 94°C followed by 30 cycles of 20 s at 94°C, 30 s at 60°C, and 30 s at 72°C. The PCR products were electrophoresed and visualized as described above.</p>                            <h3 scrollto-destination=115733919 id="115733919" class="section-title" >2.7. Multilocus sequence analysis</h3>
<p>Six housekeeping genes (<em>pgi</em>, <em>recA</em>, <em>adk</em>, <em>frdB</em>, <em>atpG</em>, and <em>mdh</em>) were chosen according to a previous study.<a reveal-id="dsx050-B23" data-open="dsx050-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a> Internal fragments of ∼500 bp were amplified by PCR with the primers listed in <span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s1-s4.doc?Expires=2147483647&Signature=KU8FqWWPovL5rUPUrBluwDtnWxfiisdSa~CodmdqLpr~k-ylhOwNe2BWcTdweHE19eyhqTRs0OOCJcqvy7MXXtMruhabhFn2fIyd~M9QN4XTyavqVjHlhddbDHOdESKsW7bsOWI2lOY5tprQ8caH9fTNcTSDmsTEA10I4pLGUwk9UF2H6-GBE6Htnrwh2WNvGOoRL7rqoV~m5tLciuc6S8Ag2qMsUPG409sR4H2Wjm1jBzOSsY88Au7p8EqTseGap-xznvlaTzR4a1ooSriCMpx6A6J9mpO6Ok4E7-scCvmTF5siNoGvYnkwO7jGP~BPERqIYyGldXgAf-M15k~39w__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>. PCR was performed using TaKaRa Ex Taq<sup>®</sup> (TAKARA BIO INC., Kusatsu, Shiga, Japan). The reactions contained genomic DNA (100 ng) that was purified from the three <em>A. actinomycetemcomitans</em> strains (Y4, ATCC29523, and NCTC9710) as described above. The thermal cycling conditions were 2 min of incubation at 94°C followed by 35 cycles of 20 s at 94°C, 30 s at 60°C, and 1 min at 72°C. The PCR products were purified using a Microcon YM-100 (Merck Millipore Corporation, Darmstadt, Germany) and sequenced using a BigDye<sup>®</sup> Terminator v3.1 Cycle Sequencing Kit (Thermo Fisher Scientific Inc.) and an ABI PRISM 3130xl Genetic Analyser (Thermo Fisher Scientific Inc.). The primers used for PCR amplification were used as sequencing primers. The sequences of the six genes (<em>pgi</em>; 514 bp, <em>recA</em>; 480 bp, <em>adk</em>; 432 bp, <em>frdB</em>; 538 bp, <em>atpG</em>; 516 bp, and <em>mdh</em>; 457 bp) were concatenated for each of the three strains, and the corresponding sequences of HK1651 (CP008984.1), IDH781 (CP016553.1), and <em>Aggregatibacter aphrophilus</em> NJ8700 (CP001607.1) were obtained from the GenBank/EMBL/DDBJ database. The concatenated sequences were aligned for a phylogenetic analysis by the maximum likelihood method using MEGA ver. 7.0.21.<a reveal-id="dsx050-B24" data-open="dsx050-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a></p>                            <h3 scrollto-destination=115733921 id="115733921" class="section-title" >2.8. Target prediction of sRNAs</h3>
<p>The target mRNAs of each IGR sRNA were predicted by IntaRNA,<a reveal-id="dsx050-B25" data-open="dsx050-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> using the sequences encompassing 75 bp upstream and downstream of the start codon of each protein coding sequence (CDS), which were extracted from the <em>A. actinomycetemcomitans</em> HK1651 RefSeq sequence (NZ_CP007502.1). The parameters were set to a minimum of seven base pairs in the seed region. We defined the threshold for target prediction at a maximal energy of −14.2 kcal/mol. The maximal energy was the sum of the hybridization free energy of the interacting subsequences and the free energies required to unfold the interaction sites in both RNA molecules.</p>                            <h2 scrollto-destination=115733923 id="115733923" class="section-title" >3. Results and discussion</h2>
                            <h3 scrollto-destination=115733924 id="115733924" class="section-title" >3.1. Identification of sRNAs in <em>A. actinomycetemcomitans</em> HK1651</h3>
<p>The sRNA identification process is shown in <a reveal-id="dsx050-F1" data-open="dsx050-F1" class="link link-reveal link-table xref-fig">Fig. 1</a>. The primary transcripts contained a triphosphate at the 5′ terminus, whereas the processed transcripts had a 5′ monophosphate. To identify both types of sRNA molecules,<a reveal-id="dsx050-B26" data-open="dsx050-B26" class="link link-ref link-reveal xref-bibr"><sup>26</sup></a> we converted the 5′ triphosphate RNA to a 5′ monophosphate RNA and then constructed a library for RNA-seq. Because most known sRNAs are 30–500 nucleotides long, we did not perform RNA fragmentation; thus, large transcripts were ignored. We obtained 1,055,983 (Experiment 1) and 844,585 (Experiment 2) pairs of sequence reads from the RNA-seq analyses of 2 independently prepared RNA samples from early exponentially growing HK1651 cells. Although many reads were derived from the <em>rrn</em> loci (mainly 5S rRNAs), 471,017 (Experiment 1) and 473,797 (experiments 2) read pairs were mapped to non-<em>rrn</em> loci. Using these reads, we searched for sRNAs on the basis of the following criteria: (i) transcribed from IGRs (defined as IGR sRNA) or opposite strands of CDSs (asRNA); (ii) with &gt;50 (Experiment 1) and &gt;40 (Experiment 2) mapping coverage to normalize for the different numbers of sequence reads between 2 samples (1,055,983 and 844,585 reads); and (iii) of identical size in the two samples. In addition, we excluded data with gradually decreased coverage towards the ends of transcripts, to remove false positives derived from degraded RNA. Through these criteria, we identified a total of 120 sRNAs that were 35–386 bp long (<a reveal-id="dsx050-F2" data-open="dsx050-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>) and broadly distributed throughout the entire chromosome (<a reveal-id="dsx050-F2" data-open="dsx050-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>). Of the 120 sRNAs, 90 were IGR sRNAs, including two riboswitches and one tmRNA, and 30 were asRNAs (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s5.xlsx?Expires=2147483647&amp;Signature=pm8yKbnMhi~iCDZRzGka5kosGL~2LVgDwxS8ettmyXni87t46NoW07GU4p6UOAaxYqTnKk3nt8-1VR6cvmkS1FVx62fQnYMRwz3SB~Qn-mRXkcTQr2syZQfW3wwTG0Ir4zSuChTS~azIfyfAwZk-WtSBM5~wWHk80~iBbRetBX88Q9XjVaFWnh0Y1XLPd-agglr1A6pirmFScbGmYUtCgvjfmkgHWFUu5DmjciS2qI7bhjYVDcdBjzjI5rKPrH12OuX7fES-ODDh9nra4RQvQ8NdOmF7XS76OqE5KJe0S1CPrWTV40EVIDv8WrnyUzqIs2fVQ~iPUZQ~DPOY6PnsCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S5</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s6.xlsx?Expires=2147483647&amp;Signature=tEmt2OX65UNJOMX~bps-0gODxnkmYp5Z4tTB517PanEleqSNhgR32csz6yNHirwNsvZW-0ZKrDzpuzqNCz4zFZc2ZAP2CrKj2-WK-oXsoVocpIDd-AzCtsXzJ6ocUa45~~Ff4ltZ9uNYKucm3rA7TVkKqmON7kMvARfeEUHxHjW-MnDSLlSdGFJ2dBRbjndhjCh-A9-uSSQILCAY1OeGOMtP0gEahMxh-wJbfJ15-7zjK1GRayBNGNYFkRtNFFFDWnVKf1DWoaf4dcRm15wsTA3wAABKrfD0Vq6TlrgJkX6Kci9d1T4g6~aJ1wcO~OUSpVVid6D1IzV2XG2sl2hy2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S6</a></span>). To confirm the expression of the sRNAs, we performed northern blot analysis. We selected 10 sRNAs based on the high expression by RNA-seq results, and found the bands of 10 sRNAs (<a reveal-id="dsx050-F3" data-open="dsx050-F3" class="link link-reveal link-table xref-fig">Fig. 3</a>). Most of their lengths were almost consistent with those of the RNA-seq, but the lengths of AaHKsRNA099 and 103 were slightly different between northern analysis and RNA seq. This difference may be due to the definition of sRNA with our criteria described above. We found that the three sets of IGR sRNAs (AaHKsRNA031 and 032, AaHKsRNA046 and 047, AaHKsRNA063 and 064) were located in both strands and were partially overlapping (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s5.xlsx?Expires=2147483647&amp;Signature=pm8yKbnMhi~iCDZRzGka5kosGL~2LVgDwxS8ettmyXni87t46NoW07GU4p6UOAaxYqTnKk3nt8-1VR6cvmkS1FVx62fQnYMRwz3SB~Qn-mRXkcTQr2syZQfW3wwTG0Ir4zSuChTS~azIfyfAwZk-WtSBM5~wWHk80~iBbRetBX88Q9XjVaFWnh0Y1XLPd-agglr1A6pirmFScbGmYUtCgvjfmkgHWFUu5DmjciS2qI7bhjYVDcdBjzjI5rKPrH12OuX7fES-ODDh9nra4RQvQ8NdOmF7XS76OqE5KJe0S1CPrWTV40EVIDv8WrnyUzqIs2fVQ~iPUZQ~DPOY6PnsCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). In these cases, it was possible for one of the transcripts to be a protein-encoding mRNA and the other one to be an asRNA, such as in a toxin–antitoxin system.<a reveal-id="dsx050-B10" data-open="dsx050-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a> However, we did not find a functional CDS in these sRNAs. Many IGR sRNAs have a Rho-independent terminator, which is an important structure for interaction with the RNA chaperone Hfq.<a reveal-id="dsx050-B27" data-open="dsx050-B27" class="link link-ref link-reveal xref-bibr"><sup>27</sup></a> Therefore, we assessed the presence of Rho-independent terminators in sRNAs by using ARNold (<a class="link link-uri" href="http://rna.igmors.u-psud.fr/toolbox/arnold" target="_blank">http://rna.igmors.u-psud.fr/toolbox/arnold/</a>) and found 34 Rho-independent terminators in 90 IGR sRNAs (<span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s5.xlsx?Expires=2147483647&amp;Signature=pm8yKbnMhi~iCDZRzGka5kosGL~2LVgDwxS8ettmyXni87t46NoW07GU4p6UOAaxYqTnKk3nt8-1VR6cvmkS1FVx62fQnYMRwz3SB~Qn-mRXkcTQr2syZQfW3wwTG0Ir4zSuChTS~azIfyfAwZk-WtSBM5~wWHk80~iBbRetBX88Q9XjVaFWnh0Y1XLPd-agglr1A6pirmFScbGmYUtCgvjfmkgHWFUu5DmjciS2qI7bhjYVDcdBjzjI5rKPrH12OuX7fES-ODDh9nra4RQvQ8NdOmF7XS76OqE5KJe0S1CPrWTV40EVIDv8WrnyUzqIs2fVQ~iPUZQ~DPOY6PnsCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S5</a></span>). </p>                        <a id="115733926" scrollto-destination="115733926"></a>
<div id="dsx050-F2" data-id="dsx050-F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/m_dsx050f2.png?Expires=2147483647&Signature=G5iGQoeXpZFUrHk8ei75zCRGxLqcz8yYvC-F2ErGiOekXDOQNfMZ8RAmoK03OGGpeSKfTWSBpzm02hU4TRStn9l6LI8b0TiYO3N5JO-76bmfI~Pd~n6H~gDKz3-EPPhAWPvzr6zu9mz3-YZ4XqmEW2e1JHQjNuVHt0~-h8N0oP7VlE89ijoET0UxAEIPndM151lRWWd9jPITVkSyK9MGDi62LWN585x2ljo1F3HcxEq58iG6e9F4VPBhMCtmH3oNT-5BAHbN4DA1KgbQ6AX2IgBl0SMmSSV2-D7t~Vwmur2Me211XWuNjM0OAJMde2QWNK7zGO6mYGpn3lTFTVcPog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="sRNAs identified in A. actinomycetemcomitans HK1651. (A) Length distribution of 120 sRNAs identified in the present study. (B) The circular genome mapping of the sRNAs identified. Outer and inner circles indicate the loci of the IGR sRNAs and the asRNAs, respectively. The outward and inward lines indicate the sRNAs located on the forward and reverse strands, respectively." path-from-xml="dsx050f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733926/dsx050f2.tif" path-from-xml="dsx050f2.tif" target="_blank">View large</a><a section="115733926" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050f2.png?Expires=2147483647&Signature=TFLHeCte8DJAJTB5WrOQA1mM~DX1TrEqgAZ0C46mm1sZwzybuIHJ36dcYvG~~v4t0rN6wTba8E53lwHsU67TU6P1LC0Xqbg20BOLXvgBp64SFkc8BEouzl5-JSReEQtsi5LiQ0jjMdM~N~GzxmENiJ7C3NsK4f686uAs-VyagQSwOObTi12AdtnfxAUlQQg4Ckj~OiJ8bNPwIdbvprfnpGJ08QfULTrUl6MoLlGBKzrUNOn2v2xebG6r2iM-XUGoJw0K3NlkkLtUXMMFmKHctKVr-KR-W7fP13XScM-rELeIuzo8NFkcEw59215ub-qNvsmd21VSWT2EeIpjjf-GTA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733926&ar=4675083&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx050f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>sRNAs identified in <em>A. actinomycetemcomitans</em> HK1651. (A) Length distribution of 120 sRNAs identified in the present study. (B) The circular genome mapping of the sRNAs identified. Outer and inner circles indicate the loci of the IGR sRNAs and the asRNAs, respectively. The outward and inward lines indicate the sRNAs located on the forward and reverse strands, respectively.</p></div></div><div content-id="dsx050-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/m_dsx050f2.png?Expires=2147483647&Signature=G5iGQoeXpZFUrHk8ei75zCRGxLqcz8yYvC-F2ErGiOekXDOQNfMZ8RAmoK03OGGpeSKfTWSBpzm02hU4TRStn9l6LI8b0TiYO3N5JO-76bmfI~Pd~n6H~gDKz3-EPPhAWPvzr6zu9mz3-YZ4XqmEW2e1JHQjNuVHt0~-h8N0oP7VlE89ijoET0UxAEIPndM151lRWWd9jPITVkSyK9MGDi62LWN585x2ljo1F3HcxEq58iG6e9F4VPBhMCtmH3oNT-5BAHbN4DA1KgbQ6AX2IgBl0SMmSSV2-D7t~Vwmur2Me211XWuNjM0OAJMde2QWNK7zGO6mYGpn3lTFTVcPog__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="sRNAs identified in A. actinomycetemcomitans HK1651. (A) Length distribution of 120 sRNAs identified in the present study. (B) The circular genome mapping of the sRNAs identified. Outer and inner circles indicate the loci of the IGR sRNAs and the asRNAs, respectively. The outward and inward lines indicate the sRNAs located on the forward and reverse strands, respectively." path-from-xml="dsx050f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733926/dsx050f2.tif" path-from-xml="dsx050f2.tif" target="_blank">View large</a><a section="115733926" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050f2.png?Expires=2147483647&Signature=TFLHeCte8DJAJTB5WrOQA1mM~DX1TrEqgAZ0C46mm1sZwzybuIHJ36dcYvG~~v4t0rN6wTba8E53lwHsU67TU6P1LC0Xqbg20BOLXvgBp64SFkc8BEouzl5-JSReEQtsi5LiQ0jjMdM~N~GzxmENiJ7C3NsK4f686uAs-VyagQSwOObTi12AdtnfxAUlQQg4Ckj~OiJ8bNPwIdbvprfnpGJ08QfULTrUl6MoLlGBKzrUNOn2v2xebG6r2iM-XUGoJw0K3NlkkLtUXMMFmKHctKVr-KR-W7fP13XScM-rELeIuzo8NFkcEw59215ub-qNvsmd21VSWT2EeIpjjf-GTA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733926&ar=4675083&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx050f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>sRNAs identified in <em>A. actinomycetemcomitans</em> HK1651. (A) Length distribution of 120 sRNAs identified in the present study. (B) The circular genome mapping of the sRNAs identified. Outer and inner circles indicate the loci of the IGR sRNAs and the asRNAs, respectively. The outward and inward lines indicate the sRNAs located on the forward and reverse strands, respectively.</p></div></div>                        <a id="115733927" scrollto-destination="115733927"></a>
<div id="dsx050-F3" data-id="dsx050-F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/m_dsx050f3.png?Expires=2147483647&Signature=MBjQH4DdI3JIIpzKZVyrBNp-1jb8QhkEWmnuwZuGMdrQbXNCW06mC-67IKYZD-y4zKc-igsrEE9llodIWP2JLyJY6Hviesl4i1MA9mzvD~hmSS3FrN8P0JLzwYRH-p-~5uz-BWrt77gflEmXbPU8mPfNME6I6BbxkM8S7lJXPF5QUQFtzaYifmD563E4J5EKAuzrE5s9gNGyED3B3Ut0P93X5fKX~aM6vdAChZGUz0Fnz8o9L2NMcqAJJ2ZkAz0M87tiasQL392n88mx-wE4dlvzYmjWtZdEhXaZtH5AAbEukaTAnfKK2MOz11U0XuPb4KiNkcGE70XpSG8ivF5XCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Northern blot analysis of sRNAs in A. actinomycetemcomitans HK1651. Ten sRNAs were validated by Northern blot analysis. The RNA sizes (bp) indicate on the left of the blot. Arrows indicate the predicted sRNAs." path-from-xml="dsx050f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733927/dsx050f3.tif" path-from-xml="dsx050f3.tif" target="_blank">View large</a><a section="115733927" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050f3.png?Expires=2147483647&Signature=YuxJu~ysf0~IB6LOvPsLXduANVjZ3A1fvRaiEKD~OdJpA6Yj3mNXExkKu72bQJbHj-JU2ew-8k-b8cV79YKITnH5yIzBvaytfXEKobt1fTXikv2no~OASDxaQP63NIW6lsyW~W69JTP2CXF-Ktlsfes9raFvCdQOaINcPX5SrYSFdLXfBcnUq1sxZs6nAc9mjVpwgOG-UyiZrORQMLKhOwvAT7Oj37UkTivjBK53rK0P1AshO419hhuymkrV2jrwnum3~LIKGSYDD-e481PSRVvqDOdrSy7NB4FtMVyEuZc3PfeUxz2gBiHQ4nVEif6kl7giph0qGTTjbTOUr2DtrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733927&ar=4675083&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx050f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Northern blot analysis of sRNAs in <em>A. actinomycetemcomitans</em> HK1651. Ten sRNAs were validated by Northern blot analysis. The RNA sizes (bp) indicate on the left of the blot. Arrows indicate the predicted sRNAs.</p></div></div><div content-id="dsx050-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/m_dsx050f3.png?Expires=2147483647&Signature=MBjQH4DdI3JIIpzKZVyrBNp-1jb8QhkEWmnuwZuGMdrQbXNCW06mC-67IKYZD-y4zKc-igsrEE9llodIWP2JLyJY6Hviesl4i1MA9mzvD~hmSS3FrN8P0JLzwYRH-p-~5uz-BWrt77gflEmXbPU8mPfNME6I6BbxkM8S7lJXPF5QUQFtzaYifmD563E4J5EKAuzrE5s9gNGyED3B3Ut0P93X5fKX~aM6vdAChZGUz0Fnz8o9L2NMcqAJJ2ZkAz0M87tiasQL392n88mx-wE4dlvzYmjWtZdEhXaZtH5AAbEukaTAnfKK2MOz11U0XuPb4KiNkcGE70XpSG8ivF5XCQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Northern blot analysis of sRNAs in A. actinomycetemcomitans HK1651. Ten sRNAs were validated by Northern blot analysis. The RNA sizes (bp) indicate on the left of the blot. Arrows indicate the predicted sRNAs." path-from-xml="dsx050f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733927/dsx050f3.tif" path-from-xml="dsx050f3.tif" target="_blank">View large</a><a section="115733927" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050f3.png?Expires=2147483647&Signature=YuxJu~ysf0~IB6LOvPsLXduANVjZ3A1fvRaiEKD~OdJpA6Yj3mNXExkKu72bQJbHj-JU2ew-8k-b8cV79YKITnH5yIzBvaytfXEKobt1fTXikv2no~OASDxaQP63NIW6lsyW~W69JTP2CXF-Ktlsfes9raFvCdQOaINcPX5SrYSFdLXfBcnUq1sxZs6nAc9mjVpwgOG-UyiZrORQMLKhOwvAT7Oj37UkTivjBK53rK0P1AshO419hhuymkrV2jrwnum3~LIKGSYDD-e481PSRVvqDOdrSy7NB4FtMVyEuZc3PfeUxz2gBiHQ4nVEif6kl7giph0qGTTjbTOUr2DtrA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733927&ar=4675083&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx050f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Northern blot analysis of sRNAs in <em>A. actinomycetemcomitans</em> HK1651. Ten sRNAs were validated by Northern blot analysis. The RNA sizes (bp) indicate on the left of the blot. Arrows indicate the predicted sRNAs.</p></div></div><p>We compared our results against the sRNAs identified in previous studies<a reveal-id="dsx050-B16" data-open="dsx050-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a><sup>,</sup><sup>–</sup><a reveal-id="dsx050-B18" data-open="dsx050-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> and found that 60 of the 90 IGR sRNAs were newly identified. Of the 75 IGR sRNA detected in strain 624 by Jorth et al.<a reveal-id="dsx050-B17" data-open="dsx050-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> under various conditions including an <em>in vivo</em> model, 49 were not found in our IGR sRNA set. An analysis of these 49 sRNAs revealed that the sequences corresponding to 19 sRNAs were absent in the HK1651 genome, 8 sRNAs were in CDSs, and 7 were located in repetitive sequences. The 13 IGR sRNAs were detected in our RNA-seq analysis, but their mapping coverage was lower than the study thresholds (below 50 in Experiment 1 and/or below 40 in Experiment 2). No reads were obtained for the remaining two IGR sRNAs in our RNA-seq analysis. Of the four Fur regulated IGR sRNAs (JA01, JA02, JA03, and JA04) identified in strain HK1651,<a reveal-id="dsx050-B18" data-open="dsx050-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> only JA04 was identified under our experimental conditions, and no reads were detected for the other three.</p><p>Of the 30 asRNAs identified in the present study, as many as 29 were newly identified. Although in the previously mentioned study of strain 624, the authors have reported 127 asRNAs,<a reveal-id="dsx050-B17" data-open="dsx050-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> only one of these (asRNA corresponding to D7S_0307.1) was identified in the present study (AaHKsRNA084). In addition, the lengths of D7S_0307.1 (1,067 bp) and AaHKsRNA084 (56 bp) differed significantly. Of the remaining 126 asRNAs identified in the previous study, 1 asRNA (D7S_2363.1) showed sequence homology to 2 tRNA genes, and sequences corresponding to 22 asRNAs were absent in the HK1651 genome. Although sequences corresponding to the remaining 103 asRNAs were conserved in the HK1651 genome, the mapping coverage of 29 asRNAs was below the threshold, and no reads were detected for 73 asRNAs in our RNA-seq analysis. A portion of one sRNA reported as an asRNA in strain 624 (D7S_2289.1; 295 bp) corresponded to IGR sRNA020 (77 bp) identified in the present study. Thus, the asRNA set identified in HK651 was significantly different from that of strain 624 (17). Two large asRNAs (D7S_1370.1; 996 bp and D7S_0307.1; 1,624 bp) identified in strain 624 were undetectable in our experimental analysis. A total of 32 asRNAs shorter than 30 bp were identified in strain 624 that were not identified in strain HK1651. Although systematic searches for asRNAs have been performed in several bacterial species,<a reveal-id="dsx050-B28 dsx050-B29 dsx050-B30" data-open="dsx050-B28 dsx050-B29 dsx050-B30" class="link link-ref link-reveal xref-bibr"><sup>28–30</sup></a> asRNAs shorter than 30 bp have rarely been reported, including in our study.</p>                            <h3 scrollto-destination=115733930 id="115733930" class="section-title" >3.2. Development of RT-PCR to detect sRNA and confirmation of the expression of the sRNAs identified in strain HK1651</h3>
<p>We attempted to develop an RT-PCR-based, simple detection system for sRNAs to confirm the expression of sRNAs identified by RNA-seq. However, it is impossible to distinguish an asRNA from its cognate mRNA by conventional RT-PCR. It is also difficult to design forward and reverse primers for many IGR sRNAs because of their short lengths. Therefore, we first polyadenylated the RNA molecules, converted them to cDNA by using a tagged oligo d(T) primer, and then detected sRNAs by PCR using a sequence tag-specific primer and a primer specific to each sRNA.<a reveal-id="dsx050-B31" data-open="dsx050-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> Because we designed specific primers within 50 bp from the 5′ terminus of each sRNA to amplify nearly full-length sRNAs, we were unable to design optimal primers for three sRNAs. To analyse these three sRNAs, the primers were designed slightly further downstream from the 5′ terminus (AaHKsRNA020, 40–58 bp from the 5′ terminus; AaHKsRNA037, 26–53 bp; and AaHKsRNA040, 34–60 bp). In addition, we were not able to design specific primers for 35 sRNAs, because identical or nearly identical sequences to any primer candidate sequences were found in other genomic loci. Therefore, we finally constructed an RT-PCR system to detect 85 of the 120 sRNAs identified by RNA-seq. Using this system, we confirmed the expression of 70 sRNAs (53 IGR sRNAs and 17 asRNAs) (<a reveal-id="dsx050-F4" data-open="dsx050-F4" class="link link-reveal link-table xref-fig">Fig. 4A;</a><span class="content-section supplementary-material"><a path-from-xml="sup3" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s5.xlsx?Expires=2147483647&amp;Signature=pm8yKbnMhi~iCDZRzGka5kosGL~2LVgDwxS8ettmyXni87t46NoW07GU4p6UOAaxYqTnKk3nt8-1VR6cvmkS1FVx62fQnYMRwz3SB~Qn-mRXkcTQr2syZQfW3wwTG0Ir4zSuChTS~azIfyfAwZk-WtSBM5~wWHk80~iBbRetBX88Q9XjVaFWnh0Y1XLPd-agglr1A6pirmFScbGmYUtCgvjfmkgHWFUu5DmjciS2qI7bhjYVDcdBjzjI5rKPrH12OuX7fES-ODDh9nra4RQvQ8NdOmF7XS76OqE5KJe0S1CPrWTV40EVIDv8WrnyUzqIs2fVQ~iPUZQ~DPOY6PnsCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S5</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup4" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s6.xlsx?Expires=2147483647&amp;Signature=tEmt2OX65UNJOMX~bps-0gODxnkmYp5Z4tTB517PanEleqSNhgR32csz6yNHirwNsvZW-0ZKrDzpuzqNCz4zFZc2ZAP2CrKj2-WK-oXsoVocpIDd-AzCtsXzJ6ocUa45~~Ff4ltZ9uNYKucm3rA7TVkKqmON7kMvARfeEUHxHjW-MnDSLlSdGFJ2dBRbjndhjCh-A9-uSSQILCAY1OeGOMtP0gEahMxh-wJbfJ15-7zjK1GRayBNGNYFkRtNFFFDWnVKf1DWoaf4dcRm15wsTA3wAABKrfD0Vq6TlrgJkX6Kci9d1T4g6~aJ1wcO~OUSpVVid6D1IzV2XG2sl2hy2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">S6</a></span>). Our inability to detect the expression of other 15 sRNAs might have been due to the lower sensitivity of this RT-PCR system and/or the incomplete optimization of the PCR primers and/or conditions. </p>                        <a id="115733932" scrollto-destination="115733932"></a>
<div id="dsx050-F4" data-id="dsx050-F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/m_dsx050f4.png?Expires=2147483647&Signature=aOrSlI59f5e7YaYoWVMqdCJ54mJirpfx9L01-R8bxAWNJNhd1rNdIXkbqWzj-F0YUB8Ul7c5F4464oLfLU0ZsFXbV7fxtH8~7VDr~m6bikbRx9yNXYnYxCYOT-S8n6OP-jTRCWSEOOpA6~ZQU2YZwEToZbcwPpO1h0eZtYF-qxl8bhPVVU6Gvn97d~NNDn2mo~EW4wR-3SKknkwSnOp9IG27MGum7vf4jStcXqG4E8cv6HVNXgOQmJ3Ny~IY8XtuQHqfunXESg1KGWmsUOCK4F-wg98WC9VsvZSMZMu4Y7luE~PfDu~-MjhlCvpvT2Dz3cSaQeDJJtKz-v9DrooJ9g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of the 70 sRNAs identified in strain HK1651 in other A. actinomycetemcomitans strains and their phylogenetic relationship with strain HK1651. (A) The RT-PCR results of 10 of the 70 representative sRNAs in A. actinomycetemcomitans strains. The 5S rRNA was used as a loading control. (B) A Venn diagram shows the distribution/conservation pattern of the 70 sRNAs in five A. actinomycetemcomitans strains. Five sRNAs were specific to strain HK1651. (C) A maximum likelihood tree for the five A. actinomycetemcomitans strains, which was constructed on the basis of the sequences of the six housekeeping genes pgi, recA, adk, frdB, atpG, and mdh. Aggregatibacter aphrophilus NJ8700 was used as an outgroup. The bootstrap values calculated from 1,000 replicates are indicated at the corresponding nodes. The scale bar corresponds to 0.02 estimated nucleotide substitutions per site." path-from-xml="dsx050f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733932/dsx050f4.tif" path-from-xml="dsx050f4.tif" target="_blank">View large</a><a section="115733932" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050f4.png?Expires=2147483647&Signature=JPGmTJZprCZF2C6lSFQ7dkzbXCZ443N-pXEuiNtqjxO5FUMksoMsFqKMfVh54Kt6GhymH324syADm2hWFnxiuOTrpp6SOL3bZMoK~-IQKOMLVA9q4AoqGIaqDZqvVigN07jcanVwHxakhx7OzSqhY7tvZ0HsGKD4pxJI2xHSQMtJyMyfi9z8ubWsoL4JEKzgVPe5F3EFMt3h-tLBx8yb8leBV8S~67Zp7pntxRGmygNgYkLG~IEOsGd6uao2oUZHjl1SstkZQphfKUmiytB5nYcGOmrFuVc9UER90ktORhydQJfABf8Ni-ZdPkMcfEBloqfminCurU-LjEZJ~QYHTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733932&ar=4675083&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx050f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of the 70 sRNAs identified in strain HK1651 in other <em>A. actinomycetemcomitans</em> strains and their phylogenetic relationship with strain HK1651. (A) The RT-PCR results of 10 of the 70 representative sRNAs in <em>A. actinomycetemcomitans</em> strains. The 5S rRNA was used as a loading control. (B) A Venn diagram shows the distribution/conservation pattern of the 70 sRNAs in five <em>A. actinomycetemcomitans</em> strains. Five sRNAs were specific to strain HK1651. (C) A maximum likelihood tree for the five <em>A. actinomycetemcomitans</em> strains, which was constructed on the basis of the sequences of the six housekeeping genes <em>pgi</em>, <em>recA</em>, <em>adk</em>, <em>frdB</em>, <em>atpG</em>, and <em>mdh</em>. <em>Aggregatibacter aphrophilus</em> NJ8700 was used as an outgroup. The bootstrap values calculated from 1,000 replicates are indicated at the corresponding nodes. The scale bar corresponds to 0.02 estimated nucleotide substitutions per site.</p></div></div><div content-id="dsx050-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/m_dsx050f4.png?Expires=2147483647&Signature=aOrSlI59f5e7YaYoWVMqdCJ54mJirpfx9L01-R8bxAWNJNhd1rNdIXkbqWzj-F0YUB8Ul7c5F4464oLfLU0ZsFXbV7fxtH8~7VDr~m6bikbRx9yNXYnYxCYOT-S8n6OP-jTRCWSEOOpA6~ZQU2YZwEToZbcwPpO1h0eZtYF-qxl8bhPVVU6Gvn97d~NNDn2mo~EW4wR-3SKknkwSnOp9IG27MGum7vf4jStcXqG4E8cv6HVNXgOQmJ3Ny~IY8XtuQHqfunXESg1KGWmsUOCK4F-wg98WC9VsvZSMZMu4Y7luE~PfDu~-MjhlCvpvT2Dz3cSaQeDJJtKz-v9DrooJ9g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Distribution of the 70 sRNAs identified in strain HK1651 in other A. actinomycetemcomitans strains and their phylogenetic relationship with strain HK1651. (A) The RT-PCR results of 10 of the 70 representative sRNAs in A. actinomycetemcomitans strains. The 5S rRNA was used as a loading control. (B) A Venn diagram shows the distribution/conservation pattern of the 70 sRNAs in five A. actinomycetemcomitans strains. Five sRNAs were specific to strain HK1651. (C) A maximum likelihood tree for the five A. actinomycetemcomitans strains, which was constructed on the basis of the sequences of the six housekeeping genes pgi, recA, adk, frdB, atpG, and mdh. Aggregatibacter aphrophilus NJ8700 was used as an outgroup. The bootstrap values calculated from 1,000 replicates are indicated at the corresponding nodes. The scale bar corresponds to 0.02 estimated nucleotide substitutions per site." path-from-xml="dsx050f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733932/dsx050f4.tif" path-from-xml="dsx050f4.tif" target="_blank">View large</a><a section="115733932" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050f4.png?Expires=2147483647&Signature=JPGmTJZprCZF2C6lSFQ7dkzbXCZ443N-pXEuiNtqjxO5FUMksoMsFqKMfVh54Kt6GhymH324syADm2hWFnxiuOTrpp6SOL3bZMoK~-IQKOMLVA9q4AoqGIaqDZqvVigN07jcanVwHxakhx7OzSqhY7tvZ0HsGKD4pxJI2xHSQMtJyMyfi9z8ubWsoL4JEKzgVPe5F3EFMt3h-tLBx8yb8leBV8S~67Zp7pntxRGmygNgYkLG~IEOsGd6uao2oUZHjl1SstkZQphfKUmiytB5nYcGOmrFuVc9UER90ktORhydQJfABf8Ni-ZdPkMcfEBloqfminCurU-LjEZJ~QYHTw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733932&ar=4675083&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx050f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Distribution of the 70 sRNAs identified in strain HK1651 in other <em>A. actinomycetemcomitans</em> strains and their phylogenetic relationship with strain HK1651. (A) The RT-PCR results of 10 of the 70 representative sRNAs in <em>A. actinomycetemcomitans</em> strains. The 5S rRNA was used as a loading control. (B) A Venn diagram shows the distribution/conservation pattern of the 70 sRNAs in five <em>A. actinomycetemcomitans</em> strains. Five sRNAs were specific to strain HK1651. (C) A maximum likelihood tree for the five <em>A. actinomycetemcomitans</em> strains, which was constructed on the basis of the sequences of the six housekeeping genes <em>pgi</em>, <em>recA</em>, <em>adk</em>, <em>frdB</em>, <em>atpG</em>, and <em>mdh</em>. <em>Aggregatibacter aphrophilus</em> NJ8700 was used as an outgroup. The bootstrap values calculated from 1,000 replicates are indicated at the corresponding nodes. The scale bar corresponds to 0.02 estimated nucleotide substitutions per site.</p></div></div>                            <h3 scrollto-destination=115733933 id="115733933" class="section-title" >3.3. Comparison of sRNA repertoires among <em>A. actinomycetemcomitans</em> strains</h3>
<p>A comparison of the sRNA sets identified in strains 624 and HK1651 suggested that a significant difference in sRNA repertoire between <em>A. actinomycetemcomitans</em> strains might exist. Therefore, we used the RT-PCR system described above to analyse the expression of 70 sRNAs in four <em>A. actinomycetemcomitans</em> strains (Y4, ATCC29523, NCTC9710, and IDH781). <a reveal-id="dsx050-F4" data-open="dsx050-F4" class="link link-reveal link-table xref-fig">Figure 4A and B</a> and <span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s7.xlsx?Expires=2147483647&Signature=I~qX87BzRRygt~3tUMDDZtFEaAksuKaYs1nSSgiWjmZv23Tn-v2-ofXazuZauCci5IM5dkHbghnzC3w4bT1VRRPPcigh6rV6hgccfwQPlDZBRLQM4jeTsehR9XyK00of2EfLhs7Yhvan7~PTs6ZqTJ5A8YPfE2ip62mjt31UBbHuOyN1QADnB4TiLC2MhLPkTqxXasnwkC5Orqcwgucr2taoqk-PpLgZ4Q8MnU-UeCtOWslyKVJboQy6Z~aL-ObR~71NyxWoDefGudfN-~CsXbBATjbXM9n8rY7OdQpv2~WR6vyXuBr7FO7epvf9n5TmpsSYGo-mq8QemWyibigMZw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span> show that 44 sRNAs (37 IGR sRNAs and seven asRNAs) were detected in all five <em>A. actinomycetemcomitans</em> strains, whereas 5 sRNAs were specifically detected in HK1651. The remaining 21 sRNAs were variably detected in the four strains, but the number of sRNAs detected in Y4 and NCTC9710 (64 and 60, respectively) was higher than in ATCC29523 and IDH781 (47 in both).</p><p>Because as much as 37% of the sRNAs analysed (26 of 70) was variably detected in the five strains, we investigated the existence of genomic loci encoding the 26 sRNAs in the strains by PCR. Of the six sRNAs not detected by RT-PCR in strain Y4, three were not detected by PCR of genomic DNA. Similarly, in the other strains, 5 of 10 (NCTC9710), 15 of 23 (ATCC29523), and 16 of 23 (IDH781) sRNAs were not detected by PCR of genomic DNA (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s7.xlsx?Expires=2147483647&amp;Signature=I~qX87BzRRygt~3tUMDDZtFEaAksuKaYs1nSSgiWjmZv23Tn-v2-ofXazuZauCci5IM5dkHbghnzC3w4bT1VRRPPcigh6rV6hgccfwQPlDZBRLQM4jeTsehR9XyK00of2EfLhs7Yhvan7~PTs6ZqTJ5A8YPfE2ip62mjt31UBbHuOyN1QADnB4TiLC2MhLPkTqxXasnwkC5Orqcwgucr2taoqk-PpLgZ4Q8MnU-UeCtOWslyKVJboQy6Z~aL-ObR~71NyxWoDefGudfN-~CsXbBATjbXM9n8rY7OdQpv2~WR6vyXuBr7FO7epvf9n5TmpsSYGo-mq8QemWyibigMZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). Because the complete genome sequence was available for IDH781, we further examined the IDH781 sequences corresponding to the 16 primers that did not yield a product in the genomic DNA PCR assays for this strain. This analysis revealed that whereas three primers did not work because of mismatches in the primer sequence (<span class="content-section supplementary-material"><a path-from-xml="sup2" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s1-s4.doc?Expires=2147483647&amp;Signature=KU8FqWWPovL5rUPUrBluwDtnWxfiisdSa~CodmdqLpr~k-ylhOwNe2BWcTdweHE19eyhqTRs0OOCJcqvy7MXXtMruhabhFn2fIyd~M9QN4XTyavqVjHlhddbDHOdESKsW7bsOWI2lOY5tprQ8caH9fTNcTSDmsTEA10I4pLGUwk9UF2H6-GBE6Htnrwh2WNvGOoRL7rqoV~m5tLciuc6S8Ag2qMsUPG409sR4H2Wjm1jBzOSsY88Au7p8EqTseGap-xznvlaTzR4a1ooSriCMpx6A6J9mpO6Ok4E7-scCvmTF5siNoGvYnkwO7jGP~BPERqIYyGldXgAf-M15k~39w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>), the genomic loci encoding 13 sRNAs were absent in IDH781 (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s7.xlsx?Expires=2147483647&amp;Signature=I~qX87BzRRygt~3tUMDDZtFEaAksuKaYs1nSSgiWjmZv23Tn-v2-ofXazuZauCci5IM5dkHbghnzC3w4bT1VRRPPcigh6rV6hgccfwQPlDZBRLQM4jeTsehR9XyK00of2EfLhs7Yhvan7~PTs6ZqTJ5A8YPfE2ip62mjt31UBbHuOyN1QADnB4TiLC2MhLPkTqxXasnwkC5Orqcwgucr2taoqk-PpLgZ4Q8MnU-UeCtOWslyKVJboQy6Z~aL-ObR~71NyxWoDefGudfN-~CsXbBATjbXM9n8rY7OdQpv2~WR6vyXuBr7FO7epvf9n5TmpsSYGo-mq8QemWyibigMZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). This result indicated that, although our inability to detect some sRNAs was due to a primer sequence mismatch, a significant number of sRNAs were not detected by RT-PCR because their genes were actually absent in the strain. Although seven sRNAs were found in the chromosome of IDH781, their expression was not detected by RT-PCR without a primer mismatch. We checked the upstream regions of the seven sRNAs and found that four sRNAs contained one or two single-nucleotide polymorphisms (SNPs) within the −50 position of each sRNA, when compared with the HK1651 sequence (AaHKsRNA019; −47C/T, AaHKsRNA028; −10C/T and −39G/A, AaHKsRNA054; −45C/T, and AaHKsRNA076; −3C/G), and AaHKsRNA097 showed no homology at the promoter region, when compared with the HK1651 sequence, whereas the other two sRNAs had no SNPs. These SNPs or the deletion found in the 5′ UTR region may be associated with the intraspecies variation of the expression of the sRNAs.</p><p>These data, together with those from the comparison of sRNAs between strains 624 and HK1651, indicated that the sRNA repertoire is notably different among <em>A. actinomycetemcomitans</em> strains. A phylogenetic tree based on five housekeeping genes showed that HK1651 was closely related to Y4 and NCTC9710 but not to ATCC29523 and IDH781 (<a reveal-id="dsx050-F4" data-open="dsx050-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). The distribution/conservation patterns of the sRNAs of the five strains were consistent with their phylogenetic relationships; strains more closely related to HK1651 (Y4 and NCTC9710) shared more sRNAs with HK1651 (<a reveal-id="dsx050-F4" data-open="dsx050-F4" class="link link-reveal link-table xref-fig">Fig. 4B and C</a>), thus suggesting the lineage-dependent acquisition or deletion of these sRNAs during the evolution of these <em>A. actinomycetemcomitans</em> strains. A similar result has been reported for other species such as <em>Campylobacter jejuni</em>, which shows a differential distribution pattern of sRNAs among strains.<a reveal-id="dsx050-B32" data-open="dsx050-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> Although intraspecies variation in the sRNA repertoire has not been well elucidated in other bacteria, similar strain-to-strain variation may be observed in many other bacterial species.</p><p><em>Aggregatibacter actinomycetemcomitans</em> have been classified into seven serotypes, and certain characteristics of their virulence properties differ among the serotypes. Specifically, serotype b strains have stronger virulence than other serotypes.<a reveal-id="dsx050-B1" data-open="dsx050-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a><sup>,</sup><a reveal-id="dsx050-B33" data-open="dsx050-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a><sup>,</sup><a reveal-id="dsx050-B34" data-open="dsx050-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> Recently, the phylogenetic distance of 31 strains of <em>A. actinomycetemcomitans</em> including HK1651 has been analysed using 397 core genes, and their phylogeny has been found to be divisible into five clades, one of which is constructed in the serotype b strains.<a reveal-id="dsx050-B35" data-open="dsx050-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> In a sRNAs distribution/conservation pattern analysis, HK1651 and Y4, which are serotype b, exhibited the closest pattern, when compared with the other three strains that showed other serotypes (NCTC9710; serotype c, ATCC29523; serotype a and IDH781; serotype d) (<a reveal-id="dsx050-F4" data-open="dsx050-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). Additionally, the phylogenetic distances of HK1651 and Y4, constructed using the five housekeeping genes, were the closest (<a reveal-id="dsx050-F4" data-open="dsx050-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). Therefore, the presence of sRNA and its regulatory mechanism may be a determinant of virulence among the different serotypes.</p>                            <h3 scrollto-destination=115733938 id="115733938" class="section-title" >3.4. Expression of sRNAs during growth</h3>
<p>The expression levels of the RT-PCR products of the 70 sRNAs detected by RT-PCR in HK1651 were further analysed in the early- and late-exponential phases. As shown in <span class="content-section supplementary-material"><a path-from-xml="sup6" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s8.doc?Expires=2147483647&Signature=Gc-60ydp63Op2GIXEAoynqa4VgVnmo02VKmfiWdK9YgbE-0Mw-i~Hae2UWtp6kLABYPmy1aNRQ6rDJnuhQlbrLeaWsDiYSo93I5EJXcr5DJb9rEeq4mLajLvinhL2lV4VXZvJ5FCiGFJ0OQUeB3EXjxMZ8Ec~wwbEAkN3AA9JkQXLArGqBGAlxaRrezl1w9GagtUzQttGU1B1domB3fMxvGX12Gt~Youk4ksjcEu8WGEQNSg7edDV2ro2YkXstpUKVycruVW8Rk4NQIM0Xf7FLjI-U5UPoXyvkz6SGVpSjb9hEd7aKDUL-Uj7V6hRppgc0AtpKGcxoY6XYISoeqT-A__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>, we identified 17 sRNAs exhibiting greater than 2-fold changes during late-exponential phase compared with early-exponential phase (nine were upregulated and eight were downregulated). A previous study has demonstrated that the expression of sRNA Aa35 in <em>A. actinomycetemcomitans</em> strain VT1169, which corresponds to AaHKsRNA003 in strain HK1651, is drastically decreased during the stationary phase compared with the exponential phase.<a reveal-id="dsx050-B16" data-open="dsx050-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> Consistent with this, the expression of AaHKsRNA003 (Aa35) exhibited a 3.5-fold decrease during late-exponential phase. To analyse the <em>cis</em>-acting regulation by asRNA, the transcriptional levels of genes located on opposite strands of four asRNAs (downregulated sRNAs; AaHKsRNA011, 066 and 070, upregulated sRNA; AaHKsRNA106) that showed altered expression during growth were analysed by qPCR. Three of the four genes (HK1651_09435, HK1651_05805, and HK1651_05685) showed slightly decreased transcription in the late-exponential phase (data not shown). Generally, many of asRNA negatively affects the gene expression.<a reveal-id="dsx050-B8" data-open="dsx050-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> However, some sRNAs were reported as positive regulator such as GadY. GadY asRNA stabilized <em>gadX</em> transcripts through interacting with 3′ UTR of <em>gadX</em> mRNA.<a reveal-id="dsx050-B36" data-open="dsx050-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> Although two asRNAs (AaHKsRNA066 and 070) may regulate the expression of the target gene positively, AaHKsRNA066 and 070 locate in a CDS, not in 3′ UTR, of the opposite strand transcripts, suggesting AaHKsRNA066 and 070 may have a different regulation mechanism with GadY asRNA.</p>                            <h3 scrollto-destination=115733940 id="115733940" class="section-title" >3.5. Target prediction of IGR RNAs</h3>
<p>Finally, the target mRNAs of the 83 IGR sRNAs identified in the present study (excluding 7 IGR sRNAs; tmRNA, 6S RNA, RNase P RNA, 2 riboswitches, and 2 signal recognition particle RNAs) were predicted by the IntaRNA programme.<a reveal-id="dsx050-B25" data-open="dsx050-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> Lists of the predicted targets are provided in <span class="content-section supplementary-material"><a path-from-xml="sup7" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s9.xlsx?Expires=2147483647&amp;Signature=bPUQEXIi0SkrT88dtwRYsJqmKQE0YfPihGnCJ-anh3vBQZbLjzNsjeATRB~i8n5Ahe1RplpNh-y4M6097O3ZN3VI20vlRQdxAswT1xR6HmIT3gbP9iOuV7Snpz16FccAVM498znJq5fV9-Dl9XgYYUwjulS6-invlRKLXnkoOt65iRMlEB9nYMb-uyzPSWlZvjP4nu9p6RsdqQiOvS2C806q1qTblpPypZOk~V6ClZWl4lNA-S-dWm4QQnF328Oxt1lCzx-eEx9uj0hh4HePbLWmgbSq6kYiuLoVWHvm6ip5mA89EsznzL~WP3TQ4FlizlDGk~7DslbzAGsMqLoZzg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9</a></span>. Although the predicted targets are involved in a wide range of biological functions, the most notable finding was that several IGR sRNAs were predicted to bind to mRNAs encoding known virulence factors. For example, AaHKsRNAs042 was predicted to interact with the mRNA of the leukotoxin export protein (LtxD) (CF65_RS09465) (<a reveal-id="dsx050-F5" data-open="dsx050-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>), AaHKsRNA093 was predicted to interact with CdtA (CF65_RS02275) (<a reveal-id="dsx050-F5" data-open="dsx050-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>), and AaHKsRNA051 was predicted to interact with the mRNA of fimbrial protein (Flp1) (CF65_RS02930)<a reveal-id="dsx050-B37" data-open="dsx050-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_figure_s1_and_2.doc?Expires=2147483647&amp;Signature=r9PcC7eogLonH7RRZLPVNtPMhCjocZSXVxe1YiQ4TYL5-~L7d-qEBTu6XPBLAh~ISZI05D7LIv-ZoVwvxYflq0mvGJtcj3rdggtYcchcNYE2XsX~nizmftPScVQ3UWJzK~Hsmj26ezTLKBP8gDLw6E9h5k4nGXAGlCwPOEp4Yvtsp1qkXrlP8FAkMSPc5d0y0DIU5~JBTjk60pyXjA950TVCRyYS32xhcyXrTTHSCoaDK-FZJwUiMQ25BNdZuj-QNIGuWj7q~Epwp5C-~sthW4Zid2F~TnIX8dwjf10cFPhl8o4cgNrlAx6KYlEGo8pojU~BjEKrQXm3jgTiSaUt~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2A</a></span>). In addition, eight sRNAs [AaHKsRNA004, 015, 025, 036, 046, 065, 068, and 105 (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_figure_s1_and_2.doc?Expires=2147483647&amp;Signature=r9PcC7eogLonH7RRZLPVNtPMhCjocZSXVxe1YiQ4TYL5-~L7d-qEBTu6XPBLAh~ISZI05D7LIv-ZoVwvxYflq0mvGJtcj3rdggtYcchcNYE2XsX~nizmftPScVQ3UWJzK~Hsmj26ezTLKBP8gDLw6E9h5k4nGXAGlCwPOEp4Yvtsp1qkXrlP8FAkMSPc5d0y0DIU5~JBTjk60pyXjA950TVCRyYS32xhcyXrTTHSCoaDK-FZJwUiMQ25BNdZuj-QNIGuWj7q~Epwp5C-~sthW4Zid2F~TnIX8dwjf10cFPhl8o4cgNrlAx6KYlEGo8pojU~BjEKrQXm3jgTiSaUt~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2B</a></span>)] were predicted to interact with mRNA of the virulence-associated protein D, which encodes a homologue of <em>Haemophilus influenza</em> VapD that is a part of a toxin–antitoxin system.<a reveal-id="dsx050-B38" data-open="dsx050-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a> Of these sRNAs, AaHKsRNA065 was expressed only in serotype b strains (HK1651 and Y4) (<span class="content-section supplementary-material"><a path-from-xml="sup5" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s7.xlsx?Expires=2147483647&amp;Signature=I~qX87BzRRygt~3tUMDDZtFEaAksuKaYs1nSSgiWjmZv23Tn-v2-ofXazuZauCci5IM5dkHbghnzC3w4bT1VRRPPcigh6rV6hgccfwQPlDZBRLQM4jeTsehR9XyK00of2EfLhs7Yhvan7~PTs6ZqTJ5A8YPfE2ip62mjt31UBbHuOyN1QADnB4TiLC2MhLPkTqxXasnwkC5Orqcwgucr2taoqk-PpLgZ4Q8MnU-UeCtOWslyKVJboQy6Z~aL-ObR~71NyxWoDefGudfN-~CsXbBATjbXM9n8rY7OdQpv2~WR6vyXuBr7FO7epvf9n5TmpsSYGo-mq8QemWyibigMZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). Therefore, the interaction of this sRNA and <em>vapD</em> mRNA may be one of the determinants of strong virulence in the serotype b. </p>                        <a id="115733942" scrollto-destination="115733942"></a>
<div id="dsx050-F5" data-id="dsx050-F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/m_dsx050f5.png?Expires=2147483647&Signature=KejqMFpa5~nvqQHTkloHEPOwB0YtsnhBeeMedu7lvbM~k0oS-A-1jG7bn2WZBCNcHerwUqrra-8Ub0kArI356hvTtVz6mDbvL-Pu---GCnxqbpz-SdOt9QuqdbDafSYcbXhMnrVEQiGqpoXD4wSmeCiPcO~BAbUFBb2GiSe8SpcD4BGTelDyqNjDXidAW5DCrYk48tYMUCi~rKZy8EqIs7mf3ie4yz8zGA6BtoLQyBBAwdDqlddc666h7cSLV3LlnXA6y6qBfiAIjGzmHDQJU~EqkAWyxhBkBBQhRAtyzy039gGiTZwLnI-kfSwixdAmM84pgr-NnObqizvgNMMSGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Putative sRNA interaction with mRNA encoding ltxD (A) and cdtA (B). Nucleotide sequences of sRNA (1) and the region of the target gene upstream and downstream of the start codon (2) were shown. Underlines indicate the region of sRNA–mRNA binding predicted by IntaRNA. Boxed nucleotides indicate start codon in target gene. Blue and red nucleotides indicate rho independent terminator in sRNA. (3) Base pairing of sRNA and target mRNA was predicted by IntaRNA. Binding energy of sRNA–mRNA is shown. (4) Secondary structure prediction of sRNA by mFold. The lowest free energy secondary structure is shown. " path-from-xml="dsx050f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733942/dsx050f5.tif" path-from-xml="dsx050f5.tif" target="_blank">View large</a><a section="115733942" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050f5.png?Expires=2147483647&Signature=oM2-SqqjJksI7YK559RkCpWPJkhZHTu8XQJP7JanBjBa9eXoXFdXQnnK2cXe-cruC0fm16JKVyBFXWKE1uLD1xXVIweCW7RhjIr~MBiKKs5weoThXuRG-F5~qQ~ndR-prlRvmTjZqbcivAZKO5fn4yZoi4z0NF-UinYI39ugdpUXoBTDeY4WUZGIDEmvyBvg1YXrz-~aE3hpXI6x3eoY29s4GZngFaibEhEXYig4uBYM9-jf~I78PsFCR4tsHQ58PjOIc85fpbrBZxlBJGn1ZrY6l64ONL6G5rU4awQcqp0h-0jXG2TdRzRlKpz7fY6WXYStxpq~F6fnoPx71QB7pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733942&ar=4675083&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx050f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Putative sRNA interaction with mRNA encoding <em>ltxD</em> (A) and <em>cdtA</em> (B). Nucleotide sequences of sRNA (1) and the region of the target gene upstream and downstream of the start codon (2) were shown. Underlines indicate the region of sRNA–mRNA binding predicted by IntaRNA. Boxed nucleotides indicate start codon in target gene. Blue and red nucleotides indicate rho independent terminator in sRNA. (3) Base pairing of sRNA and target mRNA was predicted by IntaRNA. Binding energy of sRNA–mRNA is shown. (4) Secondary structure prediction of sRNA by mFold. The lowest free energy secondary structure is shown. </p></div></div><div content-id="dsx050-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/m_dsx050f5.png?Expires=2147483647&Signature=KejqMFpa5~nvqQHTkloHEPOwB0YtsnhBeeMedu7lvbM~k0oS-A-1jG7bn2WZBCNcHerwUqrra-8Ub0kArI356hvTtVz6mDbvL-Pu---GCnxqbpz-SdOt9QuqdbDafSYcbXhMnrVEQiGqpoXD4wSmeCiPcO~BAbUFBb2GiSe8SpcD4BGTelDyqNjDXidAW5DCrYk48tYMUCi~rKZy8EqIs7mf3ie4yz8zGA6BtoLQyBBAwdDqlddc666h7cSLV3LlnXA6y6qBfiAIjGzmHDQJU~EqkAWyxhBkBBQhRAtyzy039gGiTZwLnI-kfSwixdAmM84pgr-NnObqizvgNMMSGw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Putative sRNA interaction with mRNA encoding ltxD (A) and cdtA (B). Nucleotide sequences of sRNA (1) and the region of the target gene upstream and downstream of the start codon (2) were shown. Underlines indicate the region of sRNA–mRNA binding predicted by IntaRNA. Boxed nucleotides indicate start codon in target gene. Blue and red nucleotides indicate rho independent terminator in sRNA. (3) Base pairing of sRNA and target mRNA was predicted by IntaRNA. Binding energy of sRNA–mRNA is shown. (4) Secondary structure prediction of sRNA by mFold. The lowest free energy secondary structure is shown. " path-from-xml="dsx050f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/115733942/dsx050f5.tif" path-from-xml="dsx050f5.tif" target="_blank">View large</a><a section="115733942" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050f5.png?Expires=2147483647&Signature=oM2-SqqjJksI7YK559RkCpWPJkhZHTu8XQJP7JanBjBa9eXoXFdXQnnK2cXe-cruC0fm16JKVyBFXWKE1uLD1xXVIweCW7RhjIr~MBiKKs5weoThXuRG-F5~qQ~ndR-prlRvmTjZqbcivAZKO5fn4yZoi4z0NF-UinYI39ugdpUXoBTDeY4WUZGIDEmvyBvg1YXrz-~aE3hpXI6x3eoY29s4GZngFaibEhEXYig4uBYM9-jf~I78PsFCR4tsHQ58PjOIc85fpbrBZxlBJGn1ZrY6l64ONL6G5rU4awQcqp0h-0jXG2TdRzRlKpz7fY6WXYStxpq~F6fnoPx71QB7pg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=115733942&ar=4675083&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx050f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Putative sRNA interaction with mRNA encoding <em>ltxD</em> (A) and <em>cdtA</em> (B). Nucleotide sequences of sRNA (1) and the region of the target gene upstream and downstream of the start codon (2) were shown. Underlines indicate the region of sRNA–mRNA binding predicted by IntaRNA. Boxed nucleotides indicate start codon in target gene. Blue and red nucleotides indicate rho independent terminator in sRNA. (3) Base pairing of sRNA and target mRNA was predicted by IntaRNA. Binding energy of sRNA–mRNA is shown. (4) Secondary structure prediction of sRNA by mFold. The lowest free energy secondary structure is shown. </p></div></div><p>In IDH781, the 34 detectable IGR sRNAs (excluding 7 IGR sRNAs; tmRNA, 6S RNA, RNase P RNA, 2 riboswitches, and 2 signal recognition particle RNAs) by our RT-PCR were compared with the corresponding sRNAs in HK1651. We found that the 18 sRNAs contained 1–7 SNPs (<span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s10.xlsx?Expires=2147483647&Signature=vof0Dz6N8S4V69NhKzvXpBTZOx0Cz87Wuw5MrOH~-66-cKILH1G50wKCMlJR33LuMYvwksIG0Bgb5VfqhYLFcmo9~4UjO8sqY19ly1-thsL4XQSf32B10LsE92L64p6ni9jHgKi2QSzQ0zhI8VWrC6LpO5adSDx16BKfjNg~0hvYLd9MXEuTXowjKvrHDA9-Qrb4x81pQuffeS2Kc50NXx2A-uu6PBqxrRANjdEZYOuKysg9Qzuqa3YLX9ZToBoa-0O~u2t9zVXRzeERmgFSacudDufqr-Lo6UC1rgbKWC-drkhMPjG3xuWYn9G5F6lKJsfUpyJgTJu~Jw9pDYKrAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>). We predicted the targets of the 18 sRNAs to analyse the switching of the target mRNAs due to the SNPs by using the HK1651 RefSeq sequence because the IDH781 RefSeq sequence was not supported in IntaRNA. Some of the prediction patterns were drastically altered by a few SNPs (<span class="content-section supplementary-material"><a path-from-xml="sup8" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s10.xlsx?Expires=2147483647&Signature=vof0Dz6N8S4V69NhKzvXpBTZOx0Cz87Wuw5MrOH~-66-cKILH1G50wKCMlJR33LuMYvwksIG0Bgb5VfqhYLFcmo9~4UjO8sqY19ly1-thsL4XQSf32B10LsE92L64p6ni9jHgKi2QSzQ0zhI8VWrC6LpO5adSDx16BKfjNg~0hvYLd9MXEuTXowjKvrHDA9-Qrb4x81pQuffeS2Kc50NXx2A-uu6PBqxrRANjdEZYOuKysg9Qzuqa3YLX9ZToBoa-0O~u2t9zVXRzeERmgFSacudDufqr-Lo6UC1rgbKWC-drkhMPjG3xuWYn9G5F6lKJsfUpyJgTJu~Jw9pDYKrAA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S10</a></span>), thus suggesting that SNPs in the sRNA may affect the regulatory systems among the strains.</p><p>These data, together with other data obtained in this study, should serve as an important genomic and genetic basis for future studies to fully understand the sRNA-involved regulatory network in <em>A. actinomycetemcomitans</em> and provide new insights into the intraspecies variation of the bacterial sRNA repertoire in bacteria. Functional analyses of each sRNA in different conditions and sRNA repertoire analyses of a wider range of strains, including those in closely related species, will enhance our understanding of the regulatory network in <em>A. actinomycetemcomitans</em>, especially that regulates the virulence of this important periodontal pathogen.</p>                            <h2 scrollto-destination=115733945 id="115733945" class="backsection-title" >Funding</h2>
<p>This study was supported in part by Grants-in-Aid for Scientific Research (C) (Grant No: 25462868) from the Ministry of Education, Culture, Sports, Sciences, and Technology of Japan.</p>                            <h2 scrollto-destination=115733947 id="115733947" class="backsection-title" >Accession numbers</h2>
<p>DRA submission number, DRA005568; DRR accession number, DRR088289.</p>                            <h2 scrollto-destination=115733949 id="115733949" class="backsection-title" >Conflict of interest</h2>
<p>None declared. </p>                            <h2 scrollto-destination=115733951 id="115733951" class="backsection-title" >Supplementary data</h2>
<p><span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_figure_s1_and_2.doc?Expires=2147483647&Signature=r9PcC7eogLonH7RRZLPVNtPMhCjocZSXVxe1YiQ4TYL5-~L7d-qEBTu6XPBLAh~ISZI05D7LIv-ZoVwvxYflq0mvGJtcj3rdggtYcchcNYE2XsX~nizmftPScVQ3UWJzK~Hsmj26ezTLKBP8gDLw6E9h5k4nGXAGlCwPOEp4Yvtsp1qkXrlP8FAkMSPc5d0y0DIU5~JBTjk60pyXjA950TVCRyYS32xhcyXrTTHSCoaDK-FZJwUiMQ25BNdZuj-QNIGuWj7q~Epwp5C-~sthW4Zid2F~TnIX8dwjf10cFPhl8o4cgNrlAx6KYlEGo8pojU~BjEKrQXm3jgTiSaUt~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span> are available at <em>DNARES</em> online.</p>                            <h2 scrollto-destination=115733953 id="115733953" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx050-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Fine</div> <div class="given-names">D.H.</div></div>, <div class="name string-name"><div class="surname">Markowitz</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Furgang</div> <div class="given-names">D.</div></div></div> <div class="year">2007</div>, 
<div class="article-title"><em>Aggregatibacter actinomycetemcomitans</em> and its relationship to initiation of localized aggressive periodontitis: longitudinal cohort study of initially healthy adolescents</div>, <div class="source">J. Clin. Microbiol</div> ., <div class="volume">45</div>, <div class="fpage">3859</div>–<div class="lpage">69</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aggregatibacter%20actinomycetemcomitans%20and%20its%20relationship%20to%20initiation%20of%20localized%20aggressive%20periodontitis%3A%20longitudinal%20cohort%20study%20of%20initially%20healthy%20adolescents&amp;author=D.H.%20Fine&amp;author=K.%20Markowitz&amp;author=D.%20Furgang&amp;publication_year=2007&amp;journal=J.%20Clin.%20Microbiol&amp;volume=45&amp;pages=3859-69" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JCM.00653-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJCM.00653-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJCM.00653-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17942658" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kachlany</div> <div class="given-names">S.C.</div></div></div> <div class="year">2010</div>, 
<div class="article-title"><em>Aggregatibacter actinomycetemcomitans</em> leukotoxin: from threat to therapy</div>, <div class="source">J. Dent. Res</div> ., <div class="volume">89</div>, <div class="fpage">561</div>–<div class="lpage">70</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aggregatibacter%20actinomycetemcomitans%20leukotoxin%3A%20from%20threat%20to%20therapy&amp;author=S.C.%20Kachlany&amp;publication_year=2010&amp;journal=J.%20Dent.%20Res&amp;volume=89&amp;pages=561-70" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1177/0022034510363682" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1177%2F0022034510363682" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1177%2F0022034510363682"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20200418" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tan</div> <div class="given-names">K.S.</div></div>, <div class="name string-name"><div class="surname">Song</div> <div class="given-names">K.P.</div></div>, <div class="name string-name"><div class="surname">Ong</div> <div class="given-names">G.</div></div></div> <div class="year">2002</div>, 
<div class="article-title">Cytolethal distending toxin of <em>Actinobacillus actinomycetemcomitans</em>. Occurrence and association with periodontal disease</div>, <div class="source">J. Periodont. Res</div> ., <div class="volume">37</div>, <div class="fpage">268</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cytolethal%20distending%20toxin%20of%20Actinobacillus%20actinomycetemcomitans.%20Occurrence%20and%20association%20with%20periodontal%20disease&amp;author=K.S.%20Tan&amp;author=K.P.%20Song&amp;author=G.%20Ong&amp;publication_year=2002&amp;journal=J.%20Periodont.%20Res&amp;volume=37&amp;pages=268-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1034/j.1600-0765.2002.01618.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1034%2Fj.1600-0765.2002.01618.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1034%2Fj.1600-0765.2002.01618.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12200970" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Waters</div> <div class="given-names">L.S.</div></div>, <div class="name string-name"><div class="surname">Storz</div> <div class="given-names">G.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Regulatory RNAs in bacteria</div>, <div class="source">Cell</div> , <div class="volume">136</div>, <div class="fpage">615</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Regulatory%20RNAs%20in%20bacteria&amp;author=L.S.%20Waters&amp;author=G.%20Storz&amp;publication_year=2009&amp;journal=Cell&amp;volume=136&amp;pages=615-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2009.01.043" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2009.01.043" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2009.01.043"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19239884" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Bouvier</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">C.M.</div></div>, <div class="name string-name"><div class="surname">Mika</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Nierhaus</div> <div class="given-names">K.H.</div></div>, <div class="name string-name"><div class="surname">Vogel</div> <div class="given-names">J.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Small RNA binding to 5′ mRNA coding region inhibits translational initiation</div>, <div class="source">Mol. Cell</div> ., <div class="volume">32</div>, <div class="fpage">827</div>–<div class="lpage">37</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Small%20RNA%20binding%20to%205%E2%80%B2%20mRNA%20coding%20region%20inhibits%20translational%20initiation&amp;author=M.%20Bouvier&amp;author=C.M.%20Sharma&amp;author=F.%20Mika&amp;author=K.H.%20Nierhaus&amp;author=J.%20Vogel&amp;publication_year=2008&amp;journal=Mol.%20Cell&amp;volume=32&amp;pages=827-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.molcel.2008.10.027" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.molcel.2008.10.027" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.molcel.2008.10.027"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19111662" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Massé</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Escorcia</div> <div class="given-names">F.E.</div></div>, <div class="name string-name"><div class="surname">Gottesman</div> <div class="given-names">S.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">Coupled degradation of a small regulatory RNA and its mRNA targets in <em>Escherichia coli</em></div>, <div class="source">Genes Dev</div> ., <div class="volume">17</div>, <div class="fpage">2374</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Coupled%20degradation%20of%20a%20small%20regulatory%20RNA%20and%20its%20mRNA%20targets%20in%20Escherichia%20coli&amp;author=E.%20Mass%C3%A9&amp;author=F.E.%20Escorcia&amp;author=S.%20Gottesman&amp;publication_year=2003&amp;journal=Genes%20Dev&amp;volume=17&amp;pages=2374-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gad.1127103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgad.1127103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgad.1127103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12975324" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Morfeldt</div> <div class="given-names">E.</div></div>, <div class="name string-name"><div class="surname">Taylor</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">von Gabain</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Arvidson</div> <div class="given-names">S.</div></div></div> <div class="year">1995</div>, 
<div class="article-title">Activation of alpha-toxin translation in <em>Staphylococcus aureus</em> by the <em>trans</em>-encoded antisense RNA, RNAIII</div>, <div class="source">EMBO J</div> ., <div class="volume">14</div>, <div class="fpage">4569</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Activation%20of%20alpha-toxin%20translation%20in%20Staphylococcus%20aureus%20by%20the%20trans-encoded%20antisense%20RNA%2C%20RNAIII&amp;author=E.%20Morfeldt&amp;author=D.%20Taylor&amp;author=A.%20von%20Gabain&amp;author=S.%20Arvidson&amp;publication_year=1995&amp;journal=EMBO%20J&amp;volume=14&amp;pages=4569-77" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/7556100" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Georg</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Hess</div> <div class="given-names">W.R.</div></div></div> <div class="year">2011</div>, 
<div class="article-title"><em>cis</em>-antisense RNA, another level of gene regulation in bacteria</div>, <div class="source">Microbiol. Mol. Biol. Rev</div> ., <div class="volume">75</div>, <div class="fpage">286</div>–<div class="lpage">300</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=cis-antisense%20RNA%2C%20another%20level%20of%20gene%20regulation%20in%20bacteria&amp;author=J.%20Georg&amp;author=W.R.%20Hess&amp;publication_year=2011&amp;journal=Microbiol.%20Mol.%20Biol.%20Rev&amp;volume=75&amp;pages=286-300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/MMBR.00032-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FMMBR.00032-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FMMBR.00032-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21646430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dühring</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Axmann</div> <div class="given-names">I.M.</div></div>, <div class="name string-name"><div class="surname">Hess</div> <div class="given-names">W.R.</div></div>, <div class="name string-name"><div class="surname">Wilde</div> <div class="given-names">A.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">An internal antisense RNA regulates expression of the photosynthesis gene <em>isiA</em></div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">103</div>, <div class="fpage">7054</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20internal%20antisense%20RNA%20regulates%20expression%20of%20the%20photosynthesis%20gene%20isiA&amp;author=U.%20D%C3%BChring&amp;author=I.M.%20Axmann&amp;author=W.R.%20Hess&amp;author=A.%20Wilde&amp;publication_year=2006&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=103&amp;pages=7054-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.0600927103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.0600927103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.0600927103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16636284" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kawano</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Aravind</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Storz</div> <div class="given-names">G.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">An antisense RNA controls synthesis of an SOS-induced toxin evolved from an antitoxin</div>, <div class="source">Mol. Microbiol</div> ., <div class="volume">64</div>, <div class="fpage">738</div>–<div class="lpage">54</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20antisense%20RNA%20controls%20synthesis%20of%20an%20SOS-induced%20toxin%20evolved%20from%20an%20antitoxin&amp;author=M.%20Kawano&amp;author=L.%20Aravind&amp;author=G.%20Storz&amp;publication_year=2007&amp;journal=Mol.%20Microbiol&amp;volume=64&amp;pages=738-54" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mmi.2007.64.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmmi.2007.64.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmmi.2007.64.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17462020" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sridhar</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Gunasekaran</div> <div class="given-names">P.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Computational small RNA prediction in bacteria</div>, <div class="source">Bioinform. Biol. Insights</div> , <div class="volume">7</div>, <div class="fpage">83</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20small%20RNA%20prediction%20in%20bacteria&amp;author=J.%20Sridhar&amp;author=P.%20Gunasekaran&amp;publication_year=2013&amp;journal=Bioinform.%20Biol.%20Insights&amp;volume=7&amp;pages=83-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4137/BBI.S11213" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4137%2FBBI.S11213" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4137%2FBBI.S11213"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23516022" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">C.M.</div></div>, <div class="name string-name"><div class="surname">Vogel</div> <div class="given-names">J.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Experimental approaches for the discovery and characterization of regulatory small RNA</div>, <div class="source">Curr. Opin. Microbiol</div> ., <div class="volume">12</div>, <div class="fpage">536</div>–<div class="lpage">46</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Experimental%20approaches%20for%20the%20discovery%20and%20characterization%20of%20regulatory%20small%20RNA&amp;author=C.M.%20Sharma&amp;author=J.%20Vogel&amp;publication_year=2009&amp;journal=Curr.%20Opin.%20Microbiol&amp;volume=12&amp;pages=536-46" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.mib.2009.07.006" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.mib.2009.07.006" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.mib.2009.07.006"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19758836" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hershberg</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Altuvia</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Margalit</div> <div class="given-names">H.</div></div></div> <div class="year">2003</div>, 
<div class="article-title">A survey of small RNA-encoding genes in <em>Escherichia coli</em></div>, <div class="source">Nucleic Acids Res</div> ., <div class="volume">31</div>, <div class="fpage">1813</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20survey%20of%20small%20RNA-encoding%20genes%20in%20Escherichia%20coli&amp;author=R.%20Hershberg&amp;author=S.%20Altuvia&amp;author=H.%20Margalit&amp;publication_year=2003&amp;journal=Nucleic%20Acids%20Res&amp;volume=31&amp;pages=1813-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkg297" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkg297" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkg297"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12654996" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shinhara</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Matsui</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hiraoka</div> <div class="given-names">K.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Deep sequencing reveals as-yet-undiscovered small RNAs in <em>Escherichia coli</em></div>, <div class="source">BMC Genomics</div> , <div class="volume">12</div>, <div class="fpage">428</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deep%20sequencing%20reveals%20as-yet-undiscovered%20small%20RNAs%20in%20Escherichia%20coli&amp;author=A.%20Shinhara&amp;author=M.%20Matsui&amp;author=K.%20Hiraoka&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=428" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-428" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-428" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-428"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21864382" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Gómez-Lozano</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Marvig</div> <div class="given-names">R.L.</div></div>, <div class="name string-name"><div class="surname">Molin</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Long</div> <div class="given-names">K.S.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Genome-wide identification of novel small RNAs in <em>Pseudomonas aeruginosa</em></div>, <div class="source">Environ. Microbiol</div> ., <div class="volume">14</div>, <div class="fpage">2006</div>–<div class="lpage">16</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20identification%20of%20novel%20small%20RNAs%20in%20Pseudomonas%20aeruginosa&amp;author=M.%20G%C3%B3mez-Lozano&amp;author=R.L.%20Marvig&amp;author=S.%20Molin&amp;author=K.S.%20Long&amp;publication_year=2012&amp;journal=Environ.%20Microbiol&amp;volume=14&amp;pages=2006-16" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/j.1462-2920.2012.02759.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fj.1462-2920.2012.02759.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fj.1462-2920.2012.02759.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22533370" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jorth</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Whiteley</div> <div class="given-names">M.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Characterization of a novel riboswitch-regulated lysine transporter in <em>Aggregatibacter actinomycetemcomitans</em></div>, <div class="source">J. Bacteriol</div> ., <div class="volume">192</div>, <div class="fpage">6240</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Characterization%20of%20a%20novel%20riboswitch-regulated%20lysine%20transporter%20in%20Aggregatibacter%20actinomycetemcomitans&amp;author=P.%20Jorth&amp;author=M.%20Whiteley&amp;publication_year=2010&amp;journal=J.%20Bacteriol&amp;volume=192&amp;pages=6240-50" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00935-10" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00935-10" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00935-10"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20889741" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jorth</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Trivedi</div> <div class="given-names">U.</div></div>, <div class="name string-name"><div class="surname">Rumbaugh</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Whiteley</div> <div class="given-names">M.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">Probing bacterial metabolism during infection using high-resolution transcriptomics</div>, <div class="source">J. Bacteriol</div> ., <div class="volume">195</div>, <div class="fpage">4991</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Probing%20bacterial%20metabolism%20during%20infection%20using%20high-resolution%20transcriptomics&amp;author=P.%20Jorth&amp;author=U.%20Trivedi&amp;author=K.%20Rumbaugh&amp;author=M.%20Whiteley&amp;publication_year=2013&amp;journal=J.%20Bacteriol&amp;volume=195&amp;pages=4991-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.00875-13" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.00875-13" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.00875-13"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23974023" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Amarasinghe</div> <div class="given-names">J.J.</div></div>, <div class="name string-name"><div class="surname">Connell</div> <div class="given-names">T.D.</div></div>, <div class="name string-name"><div class="surname">Scannapieco</div> <div class="given-names">F.A.</div></div>, <div class="name string-name"><div class="surname">Haase</div> <div class="given-names">E.M.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Novel iron-regulated and Fur-regulated small regulatory RNAs in <em>Aggregatibacter actinomycetemcomitans</em></div>, <div class="source">Mol. Oral Microbiol</div> ., <div class="volume">27</div>, <div class="fpage">327</div>–<div class="lpage">49</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Novel%20iron-regulated%20and%20Fur-regulated%20small%20regulatory%20RNAs%20in%20Aggregatibacter%20actinomycetemcomitans&amp;author=J.J.%20Amarasinghe&amp;author=T.D.%20Connell&amp;author=F.A.%20Scannapieco&amp;author=E.M.%20Haase&amp;publication_year=2012&amp;journal=Mol.%20Oral%20Microbiol&amp;volume=27&amp;pages=327-49" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/omi.2012.27.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fomi.2012.27.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fomi.2012.27.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22958383" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Haubek</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Poulsen</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Kilian</div> <div class="given-names">M.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Microevolution and patterns of dissemination of the JP2 clone of <em>Aggregatibacter</em> (<em>Actinobacillus</em>) <em>actinomycetemcomitans</em></div>, <div class="source">Infect. Immun</div> ., <div class="volume">75</div>, <div class="fpage">3080</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microevolution%20and%20patterns%20of%20dissemination%20of%20the%20JP2%20clone%20of%20Aggregatibacter%20(Actinobacillus)%20actinomycetemcomitans&amp;author=D.%20Haubek&amp;author=K.%20Poulsen&amp;author=M.%20Kilian&amp;publication_year=2007&amp;journal=Infect.%20Immun&amp;volume=75&amp;pages=3080-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/IAI.01734-06" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FIAI.01734-06" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FIAI.01734-06"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17353281" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Zubay</div> <div class="given-names">G.</div></div></div> <div class="year">1962</div>, 
<div class="article-title">The isolation and fractionation of soluble ribonucleic acids</div>, <div class="source">J. Mol. Biol</div> ., <div class="volume">4</div>, <div class="fpage">347</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20isolation%20and%20fractionation%20of%20soluble%20ribonucleic%20acids&amp;author=G.%20Zubay&amp;publication_year=1962&amp;journal=J.%20Mol.%20Biol&amp;volume=4&amp;pages=347-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(62)80015-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(62)80015-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(62)80015-1"> </span></div> </div></div></div></div></div><div content-id="dsx050-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Fast and accurate long-read alignment with Burrows–Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">589</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20long-read%20alignment%20with%20Burrows%E2%80%93Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=589-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp698" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp698" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp698"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20080505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Robinson</div> <div class="given-names">J.T.</div></div>, <div class="name string-name"><div class="surname">Thorvaldsdóttir</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Winckler</div> <div class="given-names">W.</div></div></div>, et al.   <div class="year">2011</div>, 
<div class="article-title">Integrative genomics viewer</div>, <div class="source">Nat. Biotechnol</div> ., <div class="volume">29</div>, <div class="fpage">24</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Integrative%20genomics%20viewer&amp;author=J.T.%20Robinson&amp;author=H.%20Thorvaldsd%C3%B3ttir&amp;author=W.%20Winckler&amp;publication_year=2011&amp;journal=Nat.%20Biotechnol&amp;volume=29&amp;pages=24-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1754" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1754" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1754"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21221095" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Eriksen</div> <div class="given-names">K.T.</div></div>, <div class="name string-name"><div class="surname">Haubek</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Poulsen</div> <div class="given-names">K.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Intragenomic recombination in the highly leukotoxic JP2 clone of <em>Actinobacillus actinomycetemcomitans</em></div>, <div class="source">Microbiology (Reading, Engl.)</div> , <div class="volume">151</div>, <div class="fpage">3371</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Intragenomic%20recombination%20in%20the%20highly%20leukotoxic%20JP2%20clone%20of%20Actinobacillus%20actinomycetemcomitans&amp;author=K.T.%20Eriksen&amp;author=D.%20Haubek&amp;author=K.%20Poulsen&amp;publication_year=2005&amp;journal=Microbiology%20(Reading%2C%20Engl.)&amp;volume=151&amp;pages=3371-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1099/mic.0.28193-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1099%2Fmic.0.28193-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1099%2Fmic.0.28193-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16207919" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Tamura</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Dudley</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Nei</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Kumar</div> <div class="given-names">S.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">MEGA4: Molecular Evolutionary Genetics Analysis (MEGA) software version 4.0</div>, <div class="source">Mol. Biol. Evol</div> ., <div class="volume">24</div>, <div class="fpage">1596</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MEGA4%3A%20Molecular%20Evolutionary%20Genetics%20Analysis%20(MEGA)%20software%20version%204.0&amp;author=K.%20Tamura&amp;author=J.%20Dudley&amp;author=M.%20Nei&amp;author=S.%20Kumar&amp;publication_year=2007&amp;journal=Mol.%20Biol.%20Evol&amp;volume=24&amp;pages=1596-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/molbev/msm092" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fmolbev%2Fmsm092" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fmolbev%2Fmsm092"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17488738" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Busch</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Richter</div> <div class="given-names">A.S.</div></div>, <div class="name string-name"><div class="surname">Backofen</div> <div class="given-names">R.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">IntaRNA: efficient prediction of bacterial sRNA targets incorporating target site accessibility and seed regions</div>, <div class="source">Bioinformatics</div> , <div class="volume">24</div>, <div class="fpage">2849</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=IntaRNA%3A%20efficient%20prediction%20of%20bacterial%20sRNA%20targets%20incorporating%20target%20site%20accessibility%20and%20seed%20regions&amp;author=A.%20Busch&amp;author=A.S.%20Richter&amp;author=R.%20Backofen&amp;publication_year=2008&amp;journal=Bioinformatics&amp;volume=24&amp;pages=2849-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btn544" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtn544" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtn544"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18940824" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">C.M.</div></div>, <div class="name string-name"><div class="surname">Hoffmann</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Darfeuille</div> <div class="given-names">F.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">The primary transcriptome of the major human pathogen <em>Helicobacter pylori</em></div>, <div class="source">Nature</div> , <div class="volume">464</div>, <div class="fpage">250</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20primary%20transcriptome%20of%20the%20major%20human%20pathogen%20Helicobacter%20pylori&amp;author=C.M.%20Sharma&amp;author=S.%20Hoffmann&amp;author=F.%20Darfeuille&amp;publication_year=2010&amp;journal=Nature&amp;volume=464&amp;pages=250-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08756" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08756" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08756"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20164839" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Otaka</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Ishikawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Morita</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Aiba</div> <div class="given-names">H.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">PolyU tail of rho-independent terminator of bacterial small RNAs is essential for Hfq action</div>, <div class="source">Proc. Natl. Acad. Sci. U.S.A</div> ., <div class="volume">108</div>, <div class="fpage">13059</div>–<div class="lpage">64</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=PolyU%20tail%20of%20rho-independent%20terminator%20of%20bacterial%20small%20RNAs%20is%20essential%20for%20Hfq%20action&amp;author=H.%20Otaka&amp;author=H.%20Ishikawa&amp;author=T.%20Morita&amp;author=H.%20Aiba&amp;publication_year=2011&amp;journal=Proc.%20Natl.%20Acad.%20Sci.%20U.S.A&amp;volume=108&amp;pages=13059-64" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1073/pnas.1107050108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1073%2Fpnas.1107050108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1073%2Fpnas.1107050108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21788484" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yachie</div> <div class="given-names">N.</div></div>, <div class="name string-name"><div class="surname">Numata</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Saito</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Kanai</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Tomita</div> <div class="given-names">M.</div></div></div> <div class="year">2006</div>, 
<div class="article-title">Prediction of non-coding and antisense RNA genes in <em>Escherichia coli</em> with Gapped Markov Model</div>, <div class="source">Gene</div> , <div class="volume">372</div>, <div class="fpage">171</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Prediction%20of%20non-coding%20and%20antisense%20RNA%20genes%20in%20Escherichia%20coli%20with%20Gapped%20Markov%20Model&amp;author=N.%20Yachie&amp;author=K.%20Numata&amp;author=R.%20Saito&amp;author=A.%20Kanai&amp;author=M.%20Tomita&amp;publication_year=2006&amp;journal=Gene&amp;volume=372&amp;pages=171-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.gene.2005.12.034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.gene.2005.12.034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.gene.2005.12.034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16564143" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Georg</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Voß</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Scholz</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Mitschke</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Wilde</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Hess</div> <div class="given-names">W.R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Evidence for a major role of antisense RNAs in cyanobacterial gene regulation</div>, <div class="source">Mol. Syst. Biol</div> ., <div class="volume">5</div>, <div class="fpage">305</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evidence%20for%20a%20major%20role%20of%20antisense%20RNAs%20in%20cyanobacterial%20gene%20regulation&amp;author=J.%20Georg&amp;author=B.%20Vo%C3%9F&amp;author=I.%20Scholz&amp;author=J.%20Mitschke&amp;author=A.%20Wilde&amp;author=W.R.%20Hess&amp;publication_year=2009&amp;journal=Mol.%20Syst.%20Biol&amp;volume=5&amp;pages=305" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2009.63" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2009.63" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb.2009.63"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19756044" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Beaume</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Hernandez</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Farinelli</div> <div class="given-names">L.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Cartography of methicillin-resistant <em>S. aureus</em> transcripts: detection, orientation and temporal expression during growth phase and stress conditions</div>, <div class="source">PLoS One</div> , <div class="volume">5</div>, <div class="fpage">e10725</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Cartography%20of%20methicillin-resistant%20S.%20aureus%20transcripts%3A%20detection%2C%20orientation%20and%20temporal%20expression%20during%20growth%20phase%20and%20stress%20conditions&amp;author=M.%20Beaume&amp;author=D.%20Hernandez&amp;author=L.%20Farinelli&amp;publication_year=2010&amp;journal=PLoS%20One&amp;volume=5&amp;pages=e10725" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0010725" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0010725" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0010725"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20505759" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ro</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yan</div> <div class="given-names">W.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Detection and quantitative analysis of small RNAs by PCR</div>, <div class="source">Methods Mol. Biol</div> ., <div class="volume">629</div>, <div class="fpage">295</div>–<div class="lpage">305</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Detection%20and%20quantitative%20analysis%20of%20small%20RNAs%20by%20PCR&amp;author=S.%20Ro&amp;author=W.%20Yan&amp;publication_year=2010&amp;journal=Methods%20Mol.%20Biol&amp;volume=629&amp;pages=295-305" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20387157" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Dugar</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Herbig</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Förstner</div> <div class="given-names">K.U.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">High-resolution transcriptome maps reveal strain-specific regulatory features of multiple <em>Campylobacter jejuni</em> isolates</div>, <div class="source">PLoS Genet</div> ., <div class="volume">9</div>, <div class="fpage">e1003495</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-resolution%20transcriptome%20maps%20reveal%20strain-specific%20regulatory%20features%20of%20multiple%20Campylobacter%20jejuni%20isolates&amp;author=G.%20Dugar&amp;author=A.%20Herbig&amp;author=K.U.%20F%C3%B6rstner&amp;publication_year=2013&amp;journal=PLoS%20Genet&amp;volume=9&amp;pages=e1003495" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pgen.1003495" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pgen.1003495" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pgen.1003495"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23696746" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">H.W.</div></div>, <div class="name string-name"><div class="surname">Huang</div> <div class="given-names">Y.F.</div></div>, <div class="name string-name"><div class="surname">Chan</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Chou</div> <div class="given-names">M.Y.</div></div></div> <div class="year">2005</div>, 
<div class="article-title">Relationship of <em>Actinobacillus actinomycetemcomitans</em> serotypes to periodontal condition: prevalence and proportions in subgingival plaque</div>, <div class="source">Eur. J. Oral Sci</div> ., <div class="volume">113</div>, <div class="fpage">28</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Relationship%20of%20Actinobacillus%20actinomycetemcomitans%20serotypes%20to%20periodontal%20condition%3A%20prevalence%20and%20proportions%20in%20subgingival%20plaque&amp;author=H.W.%20Yang&amp;author=Y.F.%20Huang&amp;author=Y.%20Chan&amp;author=M.Y.%20Chou&amp;publication_year=2005&amp;journal=Eur.%20J.%20Oral%20Sci&amp;volume=113&amp;pages=28-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/eos.2005.113.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Feos.2005.113.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Feos.2005.113.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15693826" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kawamoto</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Ando</div> <div class="given-names">E.S.</div></div>, <div class="name string-name"><div class="surname">Longo</div> <div class="given-names">P.L.</div></div>, <div class="name string-name"><div class="surname">Nunes</div> <div class="given-names">A.C.</div></div>, <div class="name string-name"><div class="surname">Wikström</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Mayer</div> <div class="given-names">M.P.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Genetic diversity and toxic activity of <em>Aggregatibacter actinomycetemcomitans</em> isolates</div>, <div class="source">Oral Microbiol. Immunol</div> ., <div class="volume">24</div>, <div class="fpage">493</div>–<div class="lpage">501</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genetic%20diversity%20and%20toxic%20activity%20of%20Aggregatibacter%20actinomycetemcomitans%20isolates&amp;author=D.%20Kawamoto&amp;author=E.S.%20Ando&amp;author=P.L.%20Longo&amp;author=A.C.%20Nunes&amp;author=M.%20Wikstr%C3%B6m&amp;author=M.P.%20Mayer&amp;publication_year=2009&amp;journal=Oral%20Microbiol.%20Immunol&amp;volume=24&amp;pages=493-501" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/omi.2009.24.issue-6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fomi.2009.24.issue-6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fomi.2009.24.issue-6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19832802" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kittichotirat</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Bumgarner</div> <div class="given-names">R.E.</div></div>, <div class="name string-name"><div class="surname">Chen</div> <div class="given-names">C.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Evolutionary divergence of <em>Aggregatibacter actinomycetemcomitans</em></div>, <div class="source">J. Dent. Res</div> ., <div class="volume">95</div>, <div class="fpage">94</div>–<div class="lpage">101</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Evolutionary%20divergence%20of%20Aggregatibacter%20actinomycetemcomitans&amp;author=W.%20Kittichotirat&amp;author=R.E.%20Bumgarner&amp;author=C.%20Chen&amp;publication_year=2016&amp;journal=J.%20Dent.%20Res&amp;volume=95&amp;pages=94-101" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1177/0022034515608163" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1177%2F0022034515608163" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1177%2F0022034515608163"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26420795" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Opdyke</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Kang</div> <div class="given-names">J.G.</div></div>, <div class="name string-name"><div class="surname">Storz</div> <div class="given-names">G.</div></div></div> <div class="year">2004</div>, 
<div class="article-title">GadY, a small-RNA regulator of acid response genes in <em>Escherichia coli</em></div>, <div class="source">J. Bacteriol</div> ., <div class="volume">186</div>, <div class="fpage">6698</div>–<div class="lpage">705</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=GadY%2C%20a%20small-RNA%20regulator%20of%20acid%20response%20genes%20in%20Escherichia%20coli&amp;author=J.A.%20Opdyke&amp;author=J.G.%20Kang&amp;author=G.%20Storz&amp;publication_year=2004&amp;journal=J.%20Bacteriol&amp;volume=186&amp;pages=6698-705" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.186.20.6698-6705.2004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.186.20.6698-6705.2004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.186.20.6698-6705.2004"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15466020" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Clock</div> <div class="given-names">S.A.</div></div>, <div class="name string-name"><div class="surname">Planet</div> <div class="given-names">P.J.</div></div>, <div class="name string-name"><div class="surname">Perez</div> <div class="given-names">B.A.</div></div>, <div class="name string-name"><div class="surname">Figurski</div> <div class="given-names">D.H.</div></div></div> <div class="year">2008</div>, 
<div class="article-title">Outer membrane components of the Tad (tight adherence) secretion of <em>Aggregatibacter actinomycetemcomitans</em></div>, <div class="source">J. Bacteriol</div> ., <div class="volume">190</div>, <div class="fpage">980</div>–<div class="lpage">90</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Outer%20membrane%20components%20of%20the%20Tad%20(tight%20adherence)%20secretion%20of%20Aggregatibacter%20actinomycetemcomitans&amp;author=S.A.%20Clock&amp;author=P.J.%20Planet&amp;author=B.A.%20Perez&amp;author=D.H.%20Figurski&amp;publication_year=2008&amp;journal=J.%20Bacteriol&amp;volume=190&amp;pages=980-90" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1128/JB.01347-07" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1128%2FJB.01347-07" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1128%2FJB.01347-07"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18055598" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx050-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx050-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ren</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Walker</div> <div class="given-names">A.N.</div></div>, <div class="name string-name"><div class="surname">Daines</div> <div class="given-names">D.A.</div></div></div> <div class="year">2012</div>, 
<div class="article-title">Toxin–antitoxin loci <em>vapBC-1</em> and <em>vapXD</em> contribute to survival and virulence in nontypeable <em>Haemophilus influenzae</em></div>, <div class="source">BMC Microbiol</div> ., <div class="volume">12</div>, <div class="fpage">263</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toxin%E2%80%93antitoxin%20loci%20vapBC-1%20and%20vapXD%20contribute%20to%20survival%20and%20virulence%20in%20nontypeable%20Haemophilus%20influenzae&amp;author=D.%20Ren&amp;author=A.N.%20Walker&amp;author=D.A.%20Daines&amp;publication_year=2012&amp;journal=BMC%20Microbiol&amp;volume=12&amp;pages=263" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2180-12-263" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2180-12-263" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2180-12-263"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23157645" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=4675083&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_figure_s1_and_2.doc?Expires=2147483647&amp;Signature=r9PcC7eogLonH7RRZLPVNtPMhCjocZSXVxe1YiQ4TYL5-~L7d-qEBTu6XPBLAh~ISZI05D7LIv-ZoVwvxYflq0mvGJtcj3rdggtYcchcNYE2XsX~nizmftPScVQ3UWJzK~Hsmj26ezTLKBP8gDLw6E9h5k4nGXAGlCwPOEp4Yvtsp1qkXrlP8FAkMSPc5d0y0DIU5~JBTjk60pyXjA950TVCRyYS32xhcyXrTTHSCoaDK-FZJwUiMQ25BNdZuj-QNIGuWj7q~Epwp5C-~sthW4Zid2F~TnIX8dwjf10cFPhl8o4cgNrlAx6KYlEGo8pojU~BjEKrQXm3jgTiSaUt~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figure S1 and 2</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s1-s4.doc?Expires=2147483647&amp;Signature=KU8FqWWPovL5rUPUrBluwDtnWxfiisdSa~CodmdqLpr~k-ylhOwNe2BWcTdweHE19eyhqTRs0OOCJcqvy7MXXtMruhabhFn2fIyd~M9QN4XTyavqVjHlhddbDHOdESKsW7bsOWI2lOY5tprQ8caH9fTNcTSDmsTEA10I4pLGUwk9UF2H6-GBE6Htnrwh2WNvGOoRL7rqoV~m5tLciuc6S8Ag2qMsUPG409sR4H2Wjm1jBzOSsY88Au7p8EqTseGap-xznvlaTzR4a1ooSriCMpx6A6J9mpO6Ok4E7-scCvmTF5siNoGvYnkwO7jGP~BPERqIYyGldXgAf-M15k~39w__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S1-S4</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s5.xlsx?Expires=2147483647&amp;Signature=pm8yKbnMhi~iCDZRzGka5kosGL~2LVgDwxS8ettmyXni87t46NoW07GU4p6UOAaxYqTnKk3nt8-1VR6cvmkS1FVx62fQnYMRwz3SB~Qn-mRXkcTQr2syZQfW3wwTG0Ir4zSuChTS~azIfyfAwZk-WtSBM5~wWHk80~iBbRetBX88Q9XjVaFWnh0Y1XLPd-agglr1A6pirmFScbGmYUtCgvjfmkgHWFUu5DmjciS2qI7bhjYVDcdBjzjI5rKPrH12OuX7fES-ODDh9nra4RQvQ8NdOmF7XS76OqE5KJe0S1CPrWTV40EVIDv8WrnyUzqIs2fVQ~iPUZQ~DPOY6PnsCA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S5</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s6.xlsx?Expires=2147483647&amp;Signature=tEmt2OX65UNJOMX~bps-0gODxnkmYp5Z4tTB517PanEleqSNhgR32csz6yNHirwNsvZW-0ZKrDzpuzqNCz4zFZc2ZAP2CrKj2-WK-oXsoVocpIDd-AzCtsXzJ6ocUa45~~Ff4ltZ9uNYKucm3rA7TVkKqmON7kMvARfeEUHxHjW-MnDSLlSdGFJ2dBRbjndhjCh-A9-uSSQILCAY1OeGOMtP0gEahMxh-wJbfJ15-7zjK1GRayBNGNYFkRtNFFFDWnVKf1DWoaf4dcRm15wsTA3wAABKrfD0Vq6TlrgJkX6Kci9d1T4g6~aJ1wcO~OUSpVVid6D1IzV2XG2sl2hy2A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S6</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s7.xlsx?Expires=2147483647&amp;Signature=I~qX87BzRRygt~3tUMDDZtFEaAksuKaYs1nSSgiWjmZv23Tn-v2-ofXazuZauCci5IM5dkHbghnzC3w4bT1VRRPPcigh6rV6hgccfwQPlDZBRLQM4jeTsehR9XyK00of2EfLhs7Yhvan7~PTs6ZqTJ5A8YPfE2ip62mjt31UBbHuOyN1QADnB4TiLC2MhLPkTqxXasnwkC5Orqcwgucr2taoqk-PpLgZ4Q8MnU-UeCtOWslyKVJboQy6Z~aL-ObR~71NyxWoDefGudfN-~CsXbBATjbXM9n8rY7OdQpv2~WR6vyXuBr7FO7epvf9n5TmpsSYGo-mq8QemWyibigMZw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S7</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s8.doc?Expires=2147483647&amp;Signature=Gc-60ydp63Op2GIXEAoynqa4VgVnmo02VKmfiWdK9YgbE-0Mw-i~Hae2UWtp6kLABYPmy1aNRQ6rDJnuhQlbrLeaWsDiYSo93I5EJXcr5DJb9rEeq4mLajLvinhL2lV4VXZvJ5FCiGFJ0OQUeB3EXjxMZ8Ec~wwbEAkN3AA9JkQXLArGqBGAlxaRrezl1w9GagtUzQttGU1B1domB3fMxvGX12Gt~Youk4ksjcEu8WGEQNSg7edDV2ro2YkXstpUKVycruVW8Rk4NQIM0Xf7FLjI-U5UPoXyvkz6SGVpSjb9hEd7aKDUL-Uj7V6hRppgc0AtpKGcxoY6XYISoeqT-A__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S8</div></a> - doc file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s9.xlsx?Expires=2147483647&amp;Signature=bPUQEXIi0SkrT88dtwRYsJqmKQE0YfPihGnCJ-anh3vBQZbLjzNsjeATRB~i8n5Ahe1RplpNh-y4M6097O3ZN3VI20vlRQdxAswT1xR6HmIT3gbP9iOuV7Snpz16FccAVM498znJq5fV9-Dl9XgYYUwjulS6-invlRKLXnkoOt65iRMlEB9nYMb-uyzPSWlZvjP4nu9p6RsdqQiOvS2C806q1qTblpPypZOk~V6ClZWl4lNA-S-dWm4QQnF328Oxt1lCzx-eEx9uj0hh4HePbLWmgbSq6kYiuLoVWHvm6ip5mA89EsznzL~WP3TQ4FlizlDGk~7DslbzAGsMqLoZzg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S9</div></a> - xlsx file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/25/2/10.1093_dnares_dsx050/1/dsx050_supplementary_table_s10.xlsx?Expires=2147483647&amp;Signature=vof0Dz6N8S4V69NhKzvXpBTZOx0Cz87Wuw5MrOH~-66-cKILH1G50wKCMlJR33LuMYvwksIG0Bgb5VfqhYLFcmo9~4UjO8sqY19ly1-thsL4XQSf32B10LsE92L64p6ni9jHgKi2QSzQ0zhI8VWrC6LpO5adSDx16BKfjNg~0hvYLd9MXEuTXowjKvrHDA9-Qrb4x81pQuffeS2Kc50NXx2A-uu6PBqxrRANjdEZYOuKysg9Qzuqa3YLX9ZToBoa-0O~u2t9zVXRzeERmgFSacudDufqr-Lo6UC1rgbKWC-drkhMPjG3xuWYn9G5F6lKJsfUpyJgTJu~Jw9pDYKrAA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Table S10</div></a> - xlsx file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">626</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000430686200009&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">1</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx050"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="4675083"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000430686200009&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f25%2f2%2f207%2f4675083">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000430686200009&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (1)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F25%2F2%2F207%2F4675083" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/4675083'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Small RNA repertoires and their intraspecies variation in Aggregatibacter actinomycetemcomitans" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx050");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f25%2f2%2f207%2f4675083">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

