

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/23/2/93/1745299","@type":"ScholarlyArticle","name":"Highly sensitive and ultrafast read mapping for RNA-seq analysis","datePublished":"2016-01-05","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/23/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2016-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsv039","keywords":["RNA-seq","mapping","Burrows-Wheeler Transform","high-performance computing"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/23/2/93/1745299","author":[{"name":"Medina, I.","affiliation":"1  HPC Service ,  UIS, University of Cambridge ,  Cambridge ,  UK","@type":"Person"},{"name":"Tárraga, J.","affiliation":"2  Computational Genomics Department ,  Centro de Investigación Príncipe Felipe (CIPF) ,  Valencia ,  Spain","@type":"Person"},{"name":"Martínez, H.","affiliation":"3  Departamento de Ingeniería y Ciencia de Computadores ,  Universitat Jaume I ,  Castellón de la Plana ,  Spain","@type":"Person"},{"name":"Barrachina, S.","affiliation":"3  Departamento de Ingeniería y Ciencia de Computadores ,  Universitat Jaume I ,  Castellón de la Plana ,  Spain","@type":"Person"},{"name":"Castillo, M. I.","affiliation":"3  Departamento de Ingeniería y Ciencia de Computadores ,  Universitat Jaume I ,  Castellón de la Plana ,  Spain","@type":"Person"},{"name":"Paschall, J.","affiliation":"4  European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge ,  UK","@type":"Person"},{"name":"Salavert-Torres, J.","affiliation":"5  Instituto de Instrumentación para Imagen Molecular ,  Universitat Politècnica de València ,  Valencia ,  Spain","@type":"Person"},{"name":"Blanquer-Espert, I.","affiliation":"5  Instituto de Instrumentación para Imagen Molecular ,  Universitat Politècnica de València ,  Valencia ,  Spain    6  Grupo de Investigación Biomédica de Imagen (GIBI 2^30) ,  La Fe Polytechnic University Hospital ,  Valencia ,  Spain","@type":"Person"},{"name":"Hernández-García, V.","affiliation":"5  Instituto de Instrumentación para Imagen Molecular ,  Universitat Politècnica de València ,  Valencia ,  Spain","@type":"Person"},{"name":"Quintana-Ortí, E. S.","affiliation":"3  Departamento de Ingeniería y Ciencia de Computadores ,  Universitat Jaume I ,  Castellón de la Plana ,  Spain","@type":"Person"},{"name":"Dopazo, J.","affiliation":"2  Computational Genomics Department ,  Centro de Investigación Príncipe Felipe (CIPF) ,  Valencia ,  Spain    7  Functional Genomics Node ,  (INB) at CIPF ,  Valencia ,  Spain    8  CIBER de Enfermedades Raras (CIBERER) ,  Valencia ,  Spain","@type":"Person"}],"description":"Abstract.  As sequencing technologies progress, the amount of data produced grows exponentially, shifting the bottleneck of discovery towards the data analysis ","pageStart":"93","pageEnd":"100","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Highly sensitive and ultrafast read mapping for RNA-seq analysis","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Highly sensitive and ultrafast read mapping for RNA-seq analysis" />
    <meta property="og:description" content="Abstract.  As sequencing technologies progress, the amount of data produced grows exponentially, shifting the bottleneck of discovery towards the data analysis " />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/23/2/93/1745299" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2016-01-05" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Medina, I." /><meta name="citation_author_institution" content="HPC Service, UIS, University of Cambridge, Cambridge, UK" /><meta name="citation_author" content="Tárraga, J." /><meta name="citation_author_institution" content="Computational Genomics Department, Centro de Investigación Príncipe Felipe (CIPF), Valencia, Spain" /><meta name="citation_author" content="Martínez, H." /><meta name="citation_author_institution" content="Departamento de Ingeniería y Ciencia de Computadores, Universitat Jaume I, Castellón de la Plana, Spain" /><meta name="citation_author" content="Barrachina, S." /><meta name="citation_author_institution" content="Departamento de Ingeniería y Ciencia de Computadores, Universitat Jaume I, Castellón de la Plana, Spain" /><meta name="citation_author" content="Castillo, M. I." /><meta name="citation_author_institution" content="Departamento de Ingeniería y Ciencia de Computadores, Universitat Jaume I, Castellón de la Plana, Spain" /><meta name="citation_author" content="Paschall, J." /><meta name="citation_author_institution" content="European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge, UK" /><meta name="citation_author" content="Salavert-Torres, J." /><meta name="citation_author_institution" content="Instituto de Instrumentación para Imagen Molecular, Universitat Politècnica de València, Valencia, Spain" /><meta name="citation_author" content="Blanquer-Espert, I." /><meta name="citation_author_institution" content="Instituto de Instrumentación para Imagen Molecular, Universitat Politècnica de València, Valencia, Spain" /><meta name="citation_author_institution" content="Grupo de Investigación Biomédica de Imagen (GIBI 2^30), La Fe Polytechnic University Hospital, Valencia, Spain" /><meta name="citation_author" content="Hernández-García, V." /><meta name="citation_author_institution" content="Instituto de Instrumentación para Imagen Molecular, Universitat Politècnica de València, Valencia, Spain" /><meta name="citation_author" content="Quintana-Ortí, E. S." /><meta name="citation_author_institution" content="Departamento de Ingeniería y Ciencia de Computadores, Universitat Jaume I, Castellón de la Plana, Spain" /><meta name="citation_author" content="Dopazo, J." /><meta name="citation_author_institution" content="Computational Genomics Department, Centro de Investigación Príncipe Felipe (CIPF), Valencia, Spain" /><meta name="citation_author_institution" content="Functional Genomics Node, (INB) at CIPF, Valencia, Spain" /><meta name="citation_author_institution" content="CIBER de Enfermedades Raras (CIBERER), Valencia, Spain" /><meta name="citation_title" content="Highly sensitive and ultrafast read mapping for RNA-seq analysis" /><meta name="citation_firstpage" content="93" /><meta name="citation_lastpage" content="100" /><meta name="citation_doi" content="10.1093/dnares/dsv039" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="23" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2016/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Computational methods for transcriptome annotation and quantification using RNA-seq;  citation_journal_title=Nat. Methods;  citation_year=2011;  citation_volume=8;  citation_pages=469-77; " /><meta name="citation_reference" content="citation_title=RNA-seq: an assessment of technical reproducibility and comparison with gene expression arrays;  citation_journal_title=Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1509-17; " /><meta name="citation_reference" content="citation_title=A survey of sequence alignment algorithms for next-generation sequencing;  citation_journal_title=Brief Bioinform.;  citation_year=2010;  citation_volume=11;  citation_pages=473-83; " /><meta name="citation_reference" content="citation_title=Ultrafast and memory-efficient alignment of short DNA sequences to the human genome;  citation_journal_title=Genome Biol.;  citation_year=2009;  citation_volume=10;  citation_pages=R25" /><meta name="citation_reference" content="citation_title=SOAP2: an improved ultrafast tool for short read alignment;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1966-7; " /><meta name="citation_reference" content="citation_title=BFAST: an alignment tool for large scale genome resequencing;  citation_journal_title=PLoS ONE;  citation_year=2009;  citation_volume=4;  citation_pages=e7767" /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=Fast and accurate long-read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=589-95; " /><meta name="citation_reference" content="citation_title=The GEM mapper: fast, accurate and versatile alignment by filtration;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=1185-8; " /><meta name="citation_reference" content="citation_title=Fast gapped-read alignment with Bowtie 2;  citation_journal_title=Nat. Methods;  citation_year=2012;  citation_volume=9;  citation_pages=357-9; " /><meta name="citation_reference" content="citation_title=Acceleration of short and long DNA read mapping without loss of accuracy using suffix array;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=3396-8; " /><meta name="citation_reference" content="citation_title=Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project;  citation_journal_title=Nature;  citation_year=2007;  citation_volume=447;  citation_pages=799-816; " /><meta name="citation_reference" content="citation_title=Global and unbiased detection of splice junctions from RNA-seq data;  citation_journal_title=Genome Biol.;  citation_year=2010;  citation_volume=11;  citation_pages=R34" /><meta name="citation_reference" content="citation_title=TopHat: discovering splice junctions with RNA-Seq;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1105-11; " /><meta name="citation_reference" content="citation_title=Fast and SNP-tolerant detection of complex variants and splicing in short reads;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=873-81; " /><meta name="citation_reference" content="citation_title=TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions;  citation_journal_title=Genome Biol.;  citation_year=2013;  citation_volume=14;  citation_pages=R36" /><meta name="citation_reference" content="citation_title=Comparative analysis of RNA-Seq alignment algorithms and the RNA-Seq unified mapper (RUM);  citation_journal_title=Bioinformatics;  citation_year=2011;  citation_volume=27;  citation_pages=2518-28; " /><meta name="citation_reference" content="citation_title=STAR: ultrafast universal RNA-seq aligner;  citation_journal_title=Bioinformatics;  citation_year=2013;  citation_volume=29;  citation_pages=15-21; " /><meta name="citation_reference" content="citation_title=MapSplice: accurate mapping of RNA-seq reads for splice junction discovery;  citation_journal_title=Nucleic Acids Res.;  citation_year=2010;  citation_volume=38;  citation_pages=e178" /><meta name="citation_reference" content="citation_title=Systematic evaluation of spliced alignment programs for RNA-seq data;  citation_journal_title=Nat. Methods;  citation_year=2013;  citation_volume=10;  citation_pages=1185-91; " /><meta name="citation_reference" content="citation_title=HISAT: a fast spliced aligner with low memory requirements;  citation_journal_title=Nat. Methods;  citation_year=2015;  citation_volume=12;  citation_pages=357-60; " /><meta name="citation_reference" content="citation_title=Differential expression in RNA-seq: a matter of depth;  citation_journal_title=Genome Res.;  citation_year=2011;  citation_volume=21;  citation_pages=2213-23; " /><meta name="citation_reference" content="citation_title=Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks;  citation_journal_title=Nat. Protoc.;  citation_year=2012;  citation_volume=7;  citation_pages=562-78; " /><meta name="citation_reference" content="citation_title=Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation;  citation_journal_title=Nat. Biotechnol.;  citation_year=2010;  citation_volume=28;  citation_pages=511-5; " /><meta name="citation_reference" content="citation_title=Basic local alignment search tool;  citation_journal_title=J. Mol. Biol.;  citation_year=1990;  citation_volume=215;  citation_pages=403-10; " /><meta name="citation_reference" content="citation_title=Using GPUs for the exact alignment of short-read genetic sequences by means of the Burrows-Wheeler transform;  citation_journal_title=IEEE/ACM Trans. Comput. Biol. Bioinform.;  citation_year=2012;  citation_volume=9;  citation_pages=1245-56; " /><meta name="citation_reference" content="citation_title=Microparallelism and high-performance protein matching. In: Proceedings of the ACM/IEEE Supercomputing Conference, San Diego, California, December 8, 1995;  citation_year=1995;  citation_pages=24" /><meta name="citation_reference" content="citation_title=EuroMPI ‘13 Proceedings of the 20th European MPI Users’ Group Meeting. In: A Dynamic Pipeline for RNA Sequencing on Multicore Processors, Madrid, Spain, September 15–18, 2013;  citation_year=2013;  citation_pages=235-40; " /><meta name="citation_reference" content="citation_title=Ensembl 2013;  citation_journal_title=Nucleic Acids Res.;  citation_year=2013;  citation_volume=41;  citation_pages=D48-55; " /><meta name="citation_reference" content="citation_title=High-throughput sequencing for biology and medicine;  citation_journal_title=Mol. Syst. Biol.;  citation_year=2013;  citation_volume=9;  citation_pages=640" /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/23/2/93/17462857/dsv039.pdf" /><meta name="description" content="Abstract.  As sequencing technologies progress, the amount of data produced grows exponentially, shifting the bottleneck of discovery towards the data analysis " />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/23/2/93/1745299" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/23/2/93/1745299" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/23/2/93/1745299");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsv039';
    </script>
<title>
	Highly sensitive and ultrafast read mapping for RNA-seq analysis | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./1745299" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0Bd9VPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNZWRpbmEsIEkuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iSFBDIFNlcnZpY2UsIFVJUywgVW5pdmVyc2l0eSBvZiBDYW1icmlkZ2UsIENhbWJyaWRnZSwgVUsiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUw6FycmFnYSwgSi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDb21wdXRhdGlvbmFsIEdlbm9taWNzIERlcGFydG1lbnQsIENlbnRybyBkZSBJbnZlc3RpZ2FjacOzbiBQcsOtbmNpcGUgRmVsaXBlIChDSVBGKSwgVmFsZW5jaWEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWFydMOtbmV6LCBILiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydGFtZW50byBkZSBJbmdlbmllcsOtYSB5IENpZW5jaWEgZGUgQ29tcHV0YWRvcmVzLCBVbml2ZXJzaXRhdCBKYXVtZSBJLCBDYXN0ZWxsw7NuIGRlIGxhIFBsYW5hLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkJhcnJhY2hpbmEsIFMuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0YW1lbnRvIGRlIEluZ2VuaWVyw61hIHkgQ2llbmNpYSBkZSBDb21wdXRhZG9yZXMsIFVuaXZlcnNpdGF0IEphdW1lIEksIENhc3RlbGzDs24gZGUgbGEgUGxhbmEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQ2FzdGlsbG8sIE0uIEkuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0YW1lbnRvIGRlIEluZ2VuaWVyw61hIHkgQ2llbmNpYSBkZSBDb21wdXRhZG9yZXMsIFVuaXZlcnNpdGF0IEphdW1lIEksIENhc3RlbGzDs24gZGUgbGEgUGxhbmEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iUGFzY2hhbGwsIEouIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRXVyb3BlYW4gTW9sZWN1bGFyIEJpb2xvZ3kgTGFib3JhdG9yeSwgRXVyb3BlYW4gQmlvaW5mb3JtYXRpY3MgSW5zdGl0dXRlLCBXZWxsY29tZSBUcnVzdCBHZW5vbWUgQ2FtcHVzLCBIaW54dG9uLCBDYW1icmlkZ2UsIFVLIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU2FsYXZlcnQtVG9ycmVzLCBKLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0byBkZSBJbnN0cnVtZW50YWNpw7NuIHBhcmEgSW1hZ2VuIE1vbGVjdWxhciwgVW5pdmVyc2l0YXQgUG9saXTDqGNuaWNhIGRlIFZhbMOobmNpYSwgVmFsZW5jaWEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iQmxhbnF1ZXItRXNwZXJ0LCBJLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9Ikluc3RpdHV0byBkZSBJbnN0cnVtZW50YWNpw7NuIHBhcmEgSW1hZ2VuIE1vbGVjdWxhciwgVW5pdmVyc2l0YXQgUG9saXTDqGNuaWNhIGRlIFZhbMOobmNpYSwgVmFsZW5jaWEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iR3J1cG8gZGUgSW52ZXN0aWdhY2nDs24gQmlvbcOpZGljYSBkZSBJbWFnZW4gKEdJQkkgMl4zMCksIExhIEZlIFBvbHl0ZWNobmljIFVuaXZlcnNpdHkgSG9zcGl0YWwsIFZhbGVuY2lhLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9Ikhlcm7DoW5kZXotR2FyY8OtYSwgVi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dG8gZGUgSW5zdHJ1bWVudGFjacOzbiBwYXJhIEltYWdlbiBNb2xlY3VsYXIsIFVuaXZlcnNpdGF0IFBvbGl0w6hjbmljYSBkZSBWYWzDqG5jaWEsIFZhbGVuY2lhLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IlF1aW50YW5hLU9ydMOtLCBFLiBTLiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydGFtZW50byBkZSBJbmdlbmllcsOtYSB5IENpZW5jaWEgZGUgQ29tcHV0YWRvcmVzLCBVbml2ZXJzaXRhdCBKYXVtZSBJLCBDYXN0ZWxsw7NuIGRlIGxhIFBsYW5hLCBTcGFpbiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IkRvcGF6bywgSi4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDb21wdXRhdGlvbmFsIEdlbm9taWNzIERlcGFydG1lbnQsIENlbnRybyBkZSBJbnZlc3RpZ2FjacOzbiBQcsOtbmNpcGUgRmVsaXBlIChDSVBGKSwgVmFsZW5jaWEsIFNwYWluIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRnVuY3Rpb25hbCBHZW5vbWljcyBOb2RlLCAoSU5CKSBhdCBDSVBGLCBWYWxlbmNpYSwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDSUJFUiBkZSBFbmZlcm1lZGFkZXMgUmFyYXMgKENJQkVSRVIpLCBWYWxlbmNpYSwgU3BhaW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IkhpZ2hseSBzZW5zaXRpdmUgYW5kIHVsdHJhZmFzdCByZWFkIG1hcHBpbmcgZm9yIFJOQS1zZXEgYW5hbHlzaXMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZmlyc3RwYWdlIiBjb250ZW50PSI5MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMTAwIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN2MDM5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjIzIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTYvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wdXRhdGlvbmFsIG1ldGhvZHMgZm9yIHRyYW5zY3JpcHRvbWUgYW5ub3RhdGlvbiBhbmQgcXVhbnRpZmljYXRpb24gdXNpbmcgUk5BLXNlcTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz00NjktNzc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJOQS1zZXE6IGFuIGFzc2Vzc21lbnQgb2YgdGVjaG5pY2FsIHJlcHJvZHVjaWJpbGl0eSBhbmQgY29tcGFyaXNvbiB3aXRoIGdlbmUgZXhwcmVzc2lvbiBhcnJheXM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBSZXMuOyAgY2l0YXRpb25feWVhcj0yMDA4OyAgY2l0YXRpb25fdm9sdW1lPTE4OyAgY2l0YXRpb25fcGFnZXM9MTUwOS0xNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QSBzdXJ2ZXkgb2Ygc2VxdWVuY2UgYWxpZ25tZW50IGFsZ29yaXRobXMgZm9yIG5leHQtZ2VuZXJhdGlvbiBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CcmllZiBCaW9pbmZvcm0uOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NDczLTgzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1VbHRyYWZhc3QgYW5kIG1lbW9yeS1lZmZpY2llbnQgYWxpZ25tZW50IG9mIHNob3J0IEROQSBzZXF1ZW5jZXMgdG8gdGhlIGh1bWFuIGdlbm9tZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9UjI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U09BUDI6IGFuIGltcHJvdmVkIHVsdHJhZmFzdCB0b29sIGZvciBzaG9ydCByZWFkIGFsaWdubWVudDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MjU7ICBjaXRhdGlvbl9wYWdlcz0xOTY2LTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJGQVNUOiBhbiBhbGlnbm1lbnQgdG9vbCBmb3IgbGFyZ2Ugc2NhbGUgZ2Vub21lIHJlc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBPTkU7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9NDsgIGNpdGF0aW9uX3BhZ2VzPWU3NzY3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBhbmQgYWNjdXJhdGUgc2hvcnQgcmVhZCBhbGlnbm1lbnQgd2l0aCBCdXJyb3dzLVdoZWVsZXIgdHJhbnNmb3JtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTE3NTQtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgYW5kIGFjY3VyYXRlIGxvbmctcmVhZCBhbGlnbm1lbnQgd2l0aCBCdXJyb3dzLVdoZWVsZXIgdHJhbnNmb3JtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTU4OS05NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VGhlIEdFTSBtYXBwZXI6IGZhc3QsIGFjY3VyYXRlIGFuZCB2ZXJzYXRpbGUgYWxpZ25tZW50IGJ5IGZpbHRyYXRpb247ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gTWV0aG9kczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTE4NS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGdhcHBlZC1yZWFkIGFsaWdubWVudCB3aXRoIEJvd3RpZSAyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9OTsgIGNpdGF0aW9uX3BhZ2VzPTM1Ny05OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY2NlbGVyYXRpb24gb2Ygc2hvcnQgYW5kIGxvbmcgRE5BIHJlYWQgbWFwcGluZyB3aXRob3V0IGxvc3Mgb2YgYWNjdXJhY3kgdXNpbmcgc3VmZml4IGFycmF5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTMzOTYtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gYW5kIGFuYWx5c2lzIG9mIGZ1bmN0aW9uYWwgZWxlbWVudHMgaW4gMSUgb2YgdGhlIGh1bWFuIGdlbm9tZSBieSB0aGUgRU5DT0RFIHBpbG90IHByb2plY3Q7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNzsgIGNpdGF0aW9uX3ZvbHVtZT00NDc7ICBjaXRhdGlvbl9wYWdlcz03OTktODE2OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1HbG9iYWwgYW5kIHVuYmlhc2VkIGRldGVjdGlvbiBvZiBzcGxpY2UganVuY3Rpb25zIGZyb20gUk5BLXNlcSBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MTE7ICBjaXRhdGlvbl9wYWdlcz1SMzQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3BIYXQ6IGRpc2NvdmVyaW5nIHNwbGljZSBqdW5jdGlvbnMgd2l0aCBSTkEtU2VxOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTExMDUtMTE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUZhc3QgYW5kIFNOUC10b2xlcmFudCBkZXRlY3Rpb24gb2YgY29tcGxleCB2YXJpYW50cyBhbmQgc3BsaWNpbmcgaW4gc2hvcnQgcmVhZHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI2OyAgY2l0YXRpb25fcGFnZXM9ODczLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Ub3BIYXQyOiBhY2N1cmF0ZSBhbGlnbm1lbnQgb2YgdHJhbnNjcmlwdG9tZXMgaW4gdGhlIHByZXNlbmNlIG9mIGluc2VydGlvbnMsIGRlbGV0aW9ucyBhbmQgZ2VuZSBmdXNpb25zOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTQ7ICBjaXRhdGlvbl9wYWdlcz1SMzYiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBhbmFseXNpcyBvZiBSTkEtU2VxIGFsaWdubWVudCBhbGdvcml0aG1zIGFuZCB0aGUgUk5BLVNlcSB1bmlmaWVkIG1hcHBlciAoUlVNKTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9Mjc7ICBjaXRhdGlvbl9wYWdlcz0yNTE4LTI4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TVEFSOiB1bHRyYWZhc3QgdW5pdmVyc2FsIFJOQS1zZXEgYWxpZ25lcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9QmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9Mjk7ICBjaXRhdGlvbl9wYWdlcz0xNS0yMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWFwU3BsaWNlOiBhY2N1cmF0ZSBtYXBwaW5nIG9mIFJOQS1zZXEgcmVhZHMgZm9yIHNwbGljZSBqdW5jdGlvbiBkaXNjb3Zlcnk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zODsgIGNpdGF0aW9uX3BhZ2VzPWUxNzgiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TeXN0ZW1hdGljIGV2YWx1YXRpb24gb2Ygc3BsaWNlZCBhbGlnbm1lbnQgcHJvZ3JhbXMgZm9yIFJOQS1zZXEgZGF0YTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9MTE4NS05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SElTQVQ6IGEgZmFzdCBzcGxpY2VkIGFsaWduZXIgd2l0aCBsb3cgbWVtb3J5IHJlcXVpcmVtZW50czsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZXRob2RzOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MzU3LTYwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnRpYWwgZXhwcmVzc2lvbiBpbiBSTkEtc2VxOiBhIG1hdHRlciBvZiBkZXB0aDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMTE7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0yMjEzLTIzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EaWZmZXJlbnRpYWwgZ2VuZSBhbmQgdHJhbnNjcmlwdCBleHByZXNzaW9uIGFuYWx5c2lzIG9mIFJOQS1zZXEgZXhwZXJpbWVudHMgd2l0aCBUb3BIYXQgYW5kIEN1ZmZsaW5rczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBQcm90b2MuOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz01NjItNzg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYW5zY3JpcHQgYXNzZW1ibHkgYW5kIHF1YW50aWZpY2F0aW9uIGJ5IFJOQS1TZXEgcmV2ZWFscyB1bmFubm90YXRlZCB0cmFuc2NyaXB0cyBhbmQgaXNvZm9ybSBzd2l0Y2hpbmcgZHVyaW5nIGNlbGwgZGlmZmVyZW50aWF0aW9uOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTI4OyAgY2l0YXRpb25fcGFnZXM9NTExLTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUJhc2ljIGxvY2FsIGFsaWdubWVudCBzZWFyY2ggdG9vbDsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Si4gTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MTk5MDsgIGNpdGF0aW9uX3ZvbHVtZT0yMTU7ICBjaXRhdGlvbl9wYWdlcz00MDMtMTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVVzaW5nIEdQVXMgZm9yIHRoZSBleGFjdCBhbGlnbm1lbnQgb2Ygc2hvcnQtcmVhZCBnZW5ldGljIHNlcXVlbmNlcyBieSBtZWFucyBvZiB0aGUgQnVycm93cy1XaGVlbGVyIHRyYW5zZm9ybTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SUVFRS9BQ00gVHJhbnMuIENvbXB1dC4gQmlvbC4gQmlvaW5mb3JtLjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT05OyAgY2l0YXRpb25fcGFnZXM9MTI0NS01NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWljcm9wYXJhbGxlbGlzbSBhbmQgaGlnaC1wZXJmb3JtYW5jZSBwcm90ZWluIG1hdGNoaW5nLiBJbjogUHJvY2VlZGluZ3Mgb2YgdGhlIEFDTS9JRUVFIFN1cGVyY29tcHV0aW5nIENvbmZlcmVuY2UsIFNhbiBEaWVnbywgQ2FsaWZvcm5pYSwgRGVjZW1iZXIgOCwgMTk5NTsgIGNpdGF0aW9uX3llYXI9MTk5NTsgIGNpdGF0aW9uX3BhZ2VzPTI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RXVyb01QSSDigJgxMyBQcm9jZWVkaW5ncyBvZiB0aGUgMjB0aCBFdXJvcGVhbiBNUEkgVXNlcnPigJkgR3JvdXAgTWVldGluZy4gSW46IEEgRHluYW1pYyBQaXBlbGluZSBmb3IgUk5BIFNlcXVlbmNpbmcgb24gTXVsdGljb3JlIFByb2Nlc3NvcnMsIE1hZHJpZCwgU3BhaW4sIFNlcHRlbWJlciAxNeKAkzE4LCAyMDEzOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fcGFnZXM9MjM1LTQwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FbnNlbWJsIDIwMTM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMzsgIGNpdGF0aW9uX3ZvbHVtZT00MTsgIGNpdGF0aW9uX3BhZ2VzPUQ0OC01NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaC10aHJvdWdocHV0IHNlcXVlbmNpbmcgZm9yIGJpb2xvZ3kgYW5kIG1lZGljaW5lOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1Nb2wuIFN5c3QuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTk7ICBjaXRhdGlvbl9wYWdlcz02NDAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMjMvMi85My8xNzQ2Mjg1Ny9kc3YwMzkucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEFzIHNlcXVlbmNpbmcgdGVjaG5vbG9naWVzIHByb2dyZXNzLCB0aGUgYW1vdW50IG9mIGRhdGEgcHJvZHVjZWQgZ3Jvd3MgZXhwb25lbnRpYWxseSwgc2hpZnRpbmcgdGhlIGJvdHRsZW5lY2sgb2YgZGlzY292ZXJ5IHRvd2FyZHMgdGhlIGRhdGEgYW5hbHlzaXMgIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0xNzQ1Mjk5JnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZKWRFVKRk0cEIW+Mc4lRJJxB7AM7" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f93%2f1745299" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f93%2f1745299" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=69060" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/23/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/23/2/1/m_cover.gif?Expires=2147483647&amp;Signature=H~-rl2t0~4icOqxuxQciOuaILaWLPqFGH3A~nroLlKomW3yjgtZ~qWZv6XD0n7MjtydtUqNoJUBHP0IS9FeSlJ7e824tplkVC9mcToG2lma9nT3MqHOrgXg-FmGm8R1Hko1D-xKC7m9goSRdTV0PpCgY514~J0A53r05NXU-LixEnWtvnkrSQlIKg2cNSOr5m2suFRYKdAKYSGWqTMps1eFybyDDmECiZJwPFJr9hkzFGuT0qtGzqG0CshylrK0bC2jCz4U60gjwvUfzNmkXc68xyAxGKYJwvJo89NhQiuWScVg0wHNDvMxjguS6KYmoQK6GthtPkW1VFz6XLn-4lQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/23/2">
                        <div class="volume trailing-comma">Volume 23</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2016        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="87197033">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197033">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197037">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197037">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197047">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197047">2. Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197079">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197079">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197096">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197096">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197101">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197101">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="87197103">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#87197103">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/23/2/81/1745287">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/23/2/101/1745328">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Highly sensitive and ultrafast read mapping for RNA-seq analysis
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                            <div class="meta-authors--limited">
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">I. Medina</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">HPC Service</div>, <div class="insititution">UIS, University of Cambridge</div>, <div class="addr-line">Cambridge</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=I.+Medina">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Medina I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Medina I&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">I. Medina</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">J. T&#225;rraga</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Computational Genomics Department</div>, <div class="insititution">Centro de Investigación Príncipe Felipe (CIPF)</div>, <div class="addr-line">Valencia</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=J.+T%c3%a1rraga">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=T&#225;rraga J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;T&#225;rraga J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">J. Tárraga</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">H. Mart&#237;nez</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Departamento de Ingeniería y Ciencia de Computadores</div>, <div class="insititution">Universitat Jaume I</div>, <div class="addr-line">Castellón de la Plana</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=H.+Mart%c3%adnez">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mart&#237;nez H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mart&#237;nez H&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">H. Martínez</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">S. Barrachina</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Departamento de Ingeniería y Ciencia de Computadores</div>, <div class="insititution">Universitat Jaume I</div>, <div class="addr-line">Castellón de la Plana</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=S.+Barrachina">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Barrachina S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Barrachina S&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">S. Barrachina</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">M. I. Castillo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Departamento de Ingeniería y Ciencia de Computadores</div>, <div class="insititution">Universitat Jaume I</div>, <div class="addr-line">Castellón de la Plana</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=M.+I.+Castillo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Castillo M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Castillo M&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">M. I. Castillo</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">J. Paschall</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">4</div><div class="addr-line">European Molecular Biology Laboratory, European Bioinformatics Institute, Wellcome Trust Genome Campus, Hinxton, Cambridge</div>, <div class="country">UK</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=J.+Paschall">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Paschall J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Paschall J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">J. Paschall</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">J. Salavert-Torres</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Instituto de Instrumentación para Imagen Molecular</div>, <div class="insititution">Universitat Politècnica de València</div>, <div class="addr-line">Valencia</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=J.+Salavert-Torres">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Salavert-Torres J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Salavert-Torres J&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">J. Salavert-Torres</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">I. Blanquer-Espert</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Instituto de Instrumentación para Imagen Molecular</div>, <div class="insititution">Universitat Politècnica de València</div>, <div class="addr-line">Valencia</div>, <div class="country">Spain</div></div><div class="aff"><div class="label -label">6</div><div class="addr-line">Grupo de Investigación Biomédica de Imagen (GIBI 2^30)</div>, <div class="insititution">La Fe Polytechnic University Hospital</div>, <div class="addr-line">Valencia</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=I.+Blanquer-Espert">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Blanquer-Espert I">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Blanquer-Espert I&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">I. Blanquer-Espert</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">V. Hern&#225;ndez-Garc&#237;a</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">5</div><div class="addr-line">Instituto de Instrumentación para Imagen Molecular</div>, <div class="insititution">Universitat Politècnica de València</div>, <div class="addr-line">Valencia</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=V.+Hern%c3%a1ndez-Garc%c3%ada">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Hern&#225;ndez-Garc&#237;a V">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Hern&#225;ndez-Garc&#237;a V&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">V. Hernández-García</a>
                                    </span>
                                    <span class="al-author-name">

                                        <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">E. S. Quintana-Ort&#237;</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Departamento de Ingeniería y Ciencia de Computadores</div>, <div class="insititution">Universitat Jaume I</div>, <div class="addr-line">Castellón de la Plana</div>, <div class="country">Spain</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=E.+S.+Quintana-Ort%c3%ad">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Quintana-Ort&#237; E">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Quintana-Ort&#237; E&quot;">Google Scholar</a>
            </div>

</div>
                                        </span>
                                        <a class="linked-name" href="javascript:;">E. S. Quintana-Ortí</a>
                                    </span>
                                <a class="meta-authors--etal is-active" id="show-meta-authors" aria-controls="remaining-authors" href="#">
                                    <span>...</span> Show more</a>
                                <div id="remaining-authors" class="meta-authors--remaining" aria-hidden="true" aria-labeledby="show-meta-authors">
                                        <span class="al-author-name-more">

                                            <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">J. Dopazo</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Computational Genomics Department</div>, <div class="insititution">Centro de Investigación Príncipe Felipe (CIPF)</div>, <div class="addr-line">Valencia</div>, <div class="country">Spain</div></div><div class="aff"><div class="label -label">7</div><div class="addr-line">Functional Genomics Node</div>, <div class="insititution">(INB) at CIPF</div>, <div class="addr-line">Valencia</div>, <div class="country">Spain</div></div><div class="aff"><div class="label -label">8</div><div class="insititution">CIBER de Enfermedades Raras (CIBERER)</div>, <div class="addr-line">Valencia</div>, <div class="country">Spain</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. E-mail: <a href="mailto:jdopazo@cipf.es" target="_blank">jdopazo@cipf.es</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=J.+Dopazo">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Dopazo J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Dopazo J&quot;">Google Scholar</a>
            </div>

</div>
                                            </span>
                                            <a class="linked-name" href="javascript:;">J. Dopazo</a>
                                        </span>
                                </div>
                            </div>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="footnote-content">Edited by Dr Mikita Suyama</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 23, Issue 2, 1 April 2016, Pages 93–100, <a href='https://doi.org/10.1093/dnares/dsv039'>https://doi.org/10.1093/dnares/dsv039</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">05 January 2016</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">13 March 2015</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">21 November 2015</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="1745299" href="/dnaresearch/article-pdf/23/2/93/17462857/dsv039.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>I. Medina, J. Tárraga, H. Martínez, S. Barrachina, M. I. Castillo, J. Paschall, J. Salavert-Torres, I. Blanquer-Espert, V. Hernández-García, E. S. Quintana-Ortí, J. Dopazo;  Highly sensitive and ultrafast read mapping for RNA-seq analysis, <em>DNA Research</em>, Volume 23, Issue 2, 1 April 2016, Pages 93–100, <a href='https://doi.org/10.1093/dnares/dsv039'>https://doi.org/10.1093/dnares/dsv039</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/1745299?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745299?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745299?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745299?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745299?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745299?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/1745299?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Highly%20sensitive%20and%20ultrafast%20read%20mapping%20for%20RNA-seq%20analysis&amp;publicationDate=2016-01-05&amp;volumeNum=23&amp;issueNum=2&amp;author=Medina%2C%20I.%3B%20T%C3%A1rraga%2C%20J.&amp;startPage=93&amp;endPage=100&amp;contentId=10.1093%2Fdnares%2Fdsv039&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f93%2f1745299" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=87197033 id="87197033" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>As sequencing technologies progress, the amount of data produced grows exponentially, shifting the bottleneck of discovery towards the data analysis phase. In particular, currently available mapping solutions for RNA-seq leave room for improvement in terms of sensitivity and performance, hindering an efficient analysis of transcriptomes by massive sequencing. Here, we present an innovative approach that combines re-engineering, optimization and parallelization. This solution results in a significant increase of mapping sensitivity over a wide range of read lengths and substantial shorter runtimes when compared with current RNA-seq mapping methods available.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="RNA-seq">RNA-seq</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="mapping">mapping</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Burrows-Wheeler Transform&quot;">Burrows-Wheeler Transform</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;high-performance computing&quot;">high-performance computing</a></div>                            <h2 scrollto-destination=87197037 id="87197037" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>The last generations of high-throughput sequencers produce data at an unprecedented scale with associated sequencing costs in a continuous decrease. In particular, RNA sequencing (RNA-seq) technology,<sup><a reveal-id="DSV039C1" data-open="DSV039C1" class="link link-ref link-reveal xref-bibr">1</a></sup> which provides a comprehensive profile of a transcriptome, is increasingly replacing conventional expression microarrays.<sup><a reveal-id="DSV039C2" data-open="DSV039C2" class="link link-ref link-reveal xref-bibr">2</a></sup> Primary data processing in RNA-seq (as well as in other massive sequencing experiments, including genome resequencing) involves mapping reads onto a reference genome. This step constitutes a computationally expensive process in which, in addition, sensitivity is a serious concern.<sup><a reveal-id="DSV039C3" data-open="DSV039C3" class="link link-ref link-reveal xref-bibr">3</a></sup> A variety of programs, many of them implementing the Burrows-Wheeler Transform (BWT) algorithm that is based on an indexing method which enormously speed up the searching process, have been developed.<sup><a reveal-id="DSV039C4  DSV039C8" data-open="DSV039C4  DSV039C8" class="link link-ref link-reveal xref-bibr">4–8</a></sup></p><p>Recently proposed algorithms for DNA read mapping map ∼98–99% of DNA reads (a gain of only 2% with respect to the 96% obtained by previously available mappers at a modest speedup).<sup><a reveal-id="DSV039C9  DSV039C11" data-open="DSV039C9  DSV039C11" class="link link-ref link-reveal xref-bibr">9–11</a></sup> Thus, the current software provides a reasonably quick and sensitive framework for read mapping in DNA resequencing experiments. However, the scenario in RNA-seq is far from these standards. Mapping sequencing reads in the context of transcripts is a much more complex problem than simply mapping reads onto the genomic sequence, as in the case of genome resequencing experiments. Eukaryotic transcriptomes are complex, with an average of more than nine transcripts per gene<sup><a reveal-id="DSV039C12" data-open="DSV039C12" class="link link-ref link-reveal xref-bibr">12</a></sup> with exons of 250 bp on the average that span over several hundreds of kilobases, and more new transcript isoforms are continuously being discovered.<sup><a reveal-id="DSV039C13" data-open="DSV039C13" class="link link-ref link-reveal xref-bibr">13</a></sup> It has been reported that widely used mappers, such as TopHat,<sup><a reveal-id="DSV039C14" data-open="DSV039C14" class="link link-ref link-reveal xref-bibr">14</a></sup> can hardly reach 70% of 100 bp reads correctly mapped in a realistic scenario with sequencing errors, while the best sensitivity (82%) is attained by GSNAP.<sup><a reveal-id="DSV039C15" data-open="DSV039C15" class="link link-ref link-reveal xref-bibr">15</a></sup> Although GSNAP seems to display a lower sensitivity in an ideal error-free scenario.<sup><a reveal-id="DSV039C16" data-open="DSV039C16" class="link link-ref link-reveal xref-bibr">16</a></sup> According to this last report, other RNA-seq mappers like RUM,<sup><a reveal-id="DSV039C17" data-open="DSV039C17" class="link link-ref link-reveal xref-bibr">17</a></sup> STAR,<sup><a reveal-id="DSV039C18" data-open="DSV039C18" class="link link-ref link-reveal xref-bibr">18</a></sup> or MapSplice<sup><a reveal-id="DSV039C19" data-open="DSV039C19" class="link link-ref link-reveal xref-bibr">19</a></sup> align sequencing reads with even lower accuracy. However, more recent reports<sup><a reveal-id="DSV039C20" data-open="DSV039C20" class="link link-ref link-reveal xref-bibr">20</a></sup> support a better performance of STAR<sup><a reveal-id="DSV039C18" data-open="DSV039C18" class="link link-ref link-reveal xref-bibr">18</a></sup> or MapSplice<sup><a reveal-id="DSV039C19" data-open="DSV039C19" class="link link-ref link-reveal xref-bibr">19</a></sup> with respect to the other methods, in agreement with earlier benchmarks.<sup><a reveal-id="DSV039C17" data-open="DSV039C17" class="link link-ref link-reveal xref-bibr">17</a></sup> Moreover, as read length increases (the natural trend of the continuous upgrades of sequencing instruments), the sensitivity of the methods drops down, causing serious problems in terms of sensitivity and runtime.<sup><a reveal-id="DSV039C16" data-open="DSV039C16" class="link link-ref link-reveal xref-bibr">16</a></sup> Mapping is actually problematic with most BWT-based mappers, because they often allow only a small number of mismatches (insufficient for the current read lengths even in scenarios of low variability). New aligners, such TopHat2,<sup><a reveal-id="DSV039C14" data-open="DSV039C14" class="link link-ref link-reveal xref-bibr">14</a></sup> which uses the new Bowtie2,<sup><a reveal-id="DSV039C10" data-open="DSV039C10" class="link link-ref link-reveal xref-bibr">10</a></sup> have significantly accelerated the mapping step (with a small trade-off in sensitivity), although the strategy for junction detection remains unchanged. While this study was under review, a new mapper was published, HISAT,<sup><a reveal-id="DSV039C21" data-open="DSV039C21" class="link link-ref link-reveal xref-bibr">21</a></sup> that combines the BWT and the Ferragina-Manzini index in its indexing scheme. Similarly, this speedup that shows this program seems to be attained at the exchange of a trade-off in sensitivity.</p><p>Despite the interest in the use of RNA-seq for transcriptome analysis and the advancements in algorithms either for quantifying reads<sup><a reveal-id="DSV039C22" data-open="DSV039C22" class="link link-ref link-reveal xref-bibr">22</a>,<a reveal-id="DSV039C23" data-open="DSV039C23" class="link link-ref link-reveal xref-bibr">23</a></sup> or for transcript discovery,<sup><a reveal-id="DSV039C24" data-open="DSV039C24" class="link link-ref link-reveal xref-bibr">24</a></sup> the basic problem of mapping reads in the context of transcripts is far from being solved.</p><p>Here, we propose an innovative solution for high-quality mapping of both short and long reads, based on a combination of mapping with BWT and local alignment with Smith-Waterman (SW), that drastically increases mapping accuracy (95 versus 60–85% by current mappers, in the most common scenarios) and substantially reduces runtimes (by about 15× when compared with TopHat2, 8× with MapSplice and 2× with STAR). In addition, the proposed strategy has also demonstrated to be quite robust against indels and mismatches. This proposal provides a simple, fast and elegant solution that maps almost all the reads, even those containing a high number of mismatches or indels. This solution also saves a substantial amount of time in the mapping step which, consequently, critically contributes to the acceleration of the current pipelines of sequencing data processing. This strategy, implemented in a program that makes use of different high-performance computing (HPC) technologies, HPG Aligner, shows an excellent performance with both, short and long reads, with runtimes presenting only a linear dependence with the number of reads.</p>                            <h2 scrollto-destination=87197047 id="87197047" class="section-title" data-legacy-id=s2>2. Methods</h2>
<p>Most modern mappers rely on the BWT indexing method to speed up the searching process. However, speedup by indexing is achieved at the exchange of sensitivity, which decreases as read length increases, since most common BWT implementations only allow a limited number of mismatches. On the other side of the spectrum, the well-known BLAST mapper<sup><a reveal-id="DSV039C25" data-open="DSV039C25" class="link link-ref link-reveal xref-bibr">25</a></sup> uses SW as a local alignment algorithm, which presents a high sensitivity, being able to map reads with many mismatches or indels. However, its performance in terms of runtimes makes BLAST unsuitable for mapping the huge number of reads produced in next-generation sequencing (NGS) experiments.</p><p>Here, we have brought together the best of both algorithms: the performance of BWT and the sensitivity of SW, by implementing a combined strategy that allows an ultrafast and high sensitivity read mapping, even in the presence of high proportions of mutations and indels. The algorithm proposed also deals very efficiently with the problem of introns, no matter that only a small fragment of the end of the read is contained in the next exon. This is achieved by using a highly efficient data structure, which we called ‘metaexon’, that learns from high-quality mapped reads and helps to map low-quality or more complex reads. This metaexon structure stores information of the genomic coordinates and the number of aligned reads with low memory requirements (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/dsv039_Supplementary_Data.zip?Expires=2147483647&Signature=O5zhXMbjdL9MfWtRHyZZoQtjwf5zaVXvYWYJtmT4rJBXhDGjpUt1cPrA598ZLSZ5pBQ6P6Lva2hSCiTy5Iub-1KISygLW8FTa3n-3Q0Hx-c6T6cnlY9n58wYv4xALGU6Fhnc4tm5sQRRSXH6LHC6xvCJn3GOzr9jmAz9AwCyLzaX14zCOGlfVslVT1AUEsVTh5IqKN1mCI0yqmSXKsVMgrqYyU95lTF6GY4a9BEbPphosWJ6-xflQndXzfUiDMr26ltDmvR6B2SLrM3AsMrURh-wZzNo2ZJj52bNBe23SttTfMf~nmqX1hQsgV5hvy8vbhYo4oeFS5mnqlhH9NS8ZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). By doing so, the proportion of hard and soft clipping is drastically reduced.</p><p>The aligning process is completed in two main steps: first, our BWT implementation, based on a previous version for exact alignment,<sup><a reveal-id="DSV039C26" data-open="DSV039C26" class="link link-ref link-reveal xref-bibr">26</a></sup> is used to map reads containing up to two mismatches (BWT index is built with a default factor of 8). The resulting high-quality alignment information is stored in the metaexon data structure. Secondly, reads containing more than two mismatches, indels or introns are mapped following a multi-stage process: (i) contiguous seeds are generated covering the whole read and also two more reads overlapping the beginning and end of the read are generated (Fig. <a reveal-id="DSV039F1" data-open="DSV039F1" class="link link-reveal link-table xref-fig">1</a>). These overlapping seeds will help to align multi-exon covering reads. A seed size of 16 nucleotides (nt) represents a good trade-off between performance and sensitivity. These seeds are mapped using BWT, with no mismatches allowed. (ii) Seeds mapping at distances closer than a read length and with the same strand orientation are brought together to generate candidate alignment locations (labelled as candidate alignment location in Fig. <a reveal-id="DSV039F1" data-open="DSV039F1" class="link link-reveal link-table xref-fig">1</a>); (iii) candidate alignment locations are clustered together to define transcripts providing they are closer than 500,000 nt and in the same strand orientation. Note that several clusters can be obtained from the candidate alignment locations; (iv) for each cluster, the gaps produced by either not aligned seeds or exon–intron boundaries are aligned using a HPC implementation of SW. This implementation uses Streaming SIMD Extensions (SSE) instructions, which allow mismatches and indels and present a high sensitivity to determine intron boundaries. For aligning seeds mapping at the ends of the exons, the metaexon data structure is queried to obtain candidate exon junctions. Using this simple strategy, HPG Aligner can provide a high-quality alignment for spliced reads; (v) finally, a score is obtained and the alignments are reported. </p>                        <a id="87197053" scrollto-destination="87197053"></a>
<div id="DSV039F1" data-id="DSV039F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/m_dsv03901.jpeg?Expires=2147483647&Signature=syUxeJXX-TQOmNLJ3OFufWhUNZqtFlGpTZSuikJjNkdF-Ih4Ni9bYaFNYxFkfDzIGyxa4rZL7Vep5tGwfywYNGbEQEKNScSeLukIKIIJLSQnx4mXnnlGyU5DJD49sIW7iUctJvRCyBP6zRMue6zZQWJlWP5sGcu2WFPIdEo8HWQ1hLCMJSK51Gdk1e784eopYja8Zqc2kkFybsdg4V8kbL-QaxzQvP5Hc2UU5NTdlTVqSz1TGq5ApTs2tIYbKBWFN3XTcL5Ey6ePmEX43yqmucQ5LgRrfSzQC7~ucD3wqGdrAOzIldTV55GI1KXzVgx90wTdbXy~4eymjMWHBrOqwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schema of the implementation of the mapping process. (Top) Contiguous seeds of size (16 bp) are taken covering the whole read. Also, two more overlapping seeds near the ends of the read are taken for anchoring the ends to the exons. (Middle) Seeds are mapped without allowing any mismatch. Seeds mapped closer than read size and, in the same strand orientation, constitute a candidate alignment location (CAL). (Bottom) CALs closer than 500,000 bp and, in the same strand orientation, are clustered to form candidate exons and transcripts. These are evaluated and scores based on SW are assigned to them. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197053/dsv03901.jpeg" path-from-xml="dsv03901.jpeg" target="_blank">View large</a><a section="87197053" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/dsv03901.jpeg?Expires=2147483647&Signature=I4b3RwH~JI4Fc-1zFcQKIStgQRPEIju8MlboN2yxJhjsW1gR9oESQEGloWjf3lW-Q2Ln~G6m4VYpP7SqiWarY~iKELD3k7DjVOON2gYy8gvoh-b-sus-FkawLF57xp0LyLnNObUGfG0PxS1KklZhfkOUu9yj3-7V9q3cNVGytEfoPU-J19BKAL4u65BczZnCfbHhr8A9ohh7nhQc3x3N-1HoNOpfN211nR2Qb24nghFk-fgljoNtSUf-oc00yq--RIRuBnU31qzmp5uubIjNDSmwx2nacLlnKEqK5ENz7AFMO3LnLATmz9Q6aNbAjLiwyBSX4moVUVzrGgWOW0qRKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197053&ar=1745299&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schema of the implementation of the mapping process. (Top) Contiguous seeds of size (16 bp) are taken covering the whole read. Also, two more overlapping seeds near the ends of the read are taken for anchoring the ends to the exons. (Middle) Seeds are mapped without allowing any mismatch. Seeds mapped closer than read size and, in the same strand orientation, constitute a candidate alignment location (CAL). (Bottom) CALs closer than 500,000 bp and, in the same strand orientation, are clustered to form candidate exons and transcripts. These are evaluated and scores based on SW are assigned to them. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><div content-id="DSV039F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/m_dsv03901.jpeg?Expires=2147483647&Signature=syUxeJXX-TQOmNLJ3OFufWhUNZqtFlGpTZSuikJjNkdF-Ih4Ni9bYaFNYxFkfDzIGyxa4rZL7Vep5tGwfywYNGbEQEKNScSeLukIKIIJLSQnx4mXnnlGyU5DJD49sIW7iUctJvRCyBP6zRMue6zZQWJlWP5sGcu2WFPIdEo8HWQ1hLCMJSK51Gdk1e784eopYja8Zqc2kkFybsdg4V8kbL-QaxzQvP5Hc2UU5NTdlTVqSz1TGq5ApTs2tIYbKBWFN3XTcL5Ey6ePmEX43yqmucQ5LgRrfSzQC7~ucD3wqGdrAOzIldTV55GI1KXzVgx90wTdbXy~4eymjMWHBrOqwg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Schema of the implementation of the mapping process. (Top) Contiguous seeds of size (16 bp) are taken covering the whole read. Also, two more overlapping seeds near the ends of the read are taken for anchoring the ends to the exons. (Middle) Seeds are mapped without allowing any mismatch. Seeds mapped closer than read size and, in the same strand orientation, constitute a candidate alignment location (CAL). (Bottom) CALs closer than 500,000 bp and, in the same strand orientation, are clustered to form candidate exons and transcripts. These are evaluated and scores based on SW are assigned to them. This figure is available in black and white in print and in colour at DNA Research online." path-from-xml="dsv03901.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197053/dsv03901.jpeg" path-from-xml="dsv03901.jpeg" target="_blank">View large</a><a section="87197053" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/dsv03901.jpeg?Expires=2147483647&Signature=I4b3RwH~JI4Fc-1zFcQKIStgQRPEIju8MlboN2yxJhjsW1gR9oESQEGloWjf3lW-Q2Ln~G6m4VYpP7SqiWarY~iKELD3k7DjVOON2gYy8gvoh-b-sus-FkawLF57xp0LyLnNObUGfG0PxS1KklZhfkOUu9yj3-7V9q3cNVGytEfoPU-J19BKAL4u65BczZnCfbHhr8A9ohh7nhQc3x3N-1HoNOpfN211nR2Qb24nghFk-fgljoNtSUf-oc00yq--RIRuBnU31qzmp5uubIjNDSmwx2nacLlnKEqK5ENz7AFMO3LnLATmz9Q6aNbAjLiwyBSX4moVUVzrGgWOW0qRKw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=87197053&ar=1745299&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsv03901.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Schema of the implementation of the mapping process. (Top) Contiguous seeds of size (16 bp) are taken covering the whole read. Also, two more overlapping seeds near the ends of the read are taken for anchoring the ends to the exons. (Middle) Seeds are mapped without allowing any mismatch. Seeds mapped closer than read size and, in the same strand orientation, constitute a candidate alignment location (CAL). (Bottom) CALs closer than 500,000 bp and, in the same strand orientation, are clustered to form candidate exons and transcripts. These are evaluated and scores based on SW are assigned to them. This figure is available in black and white in print and in colour at <em>DNA Research</em> online.</p></div></div><p>Due to the HPC implementation, we have devised a simple but powerful strategy in which the original high sensitivity of the algorithms is maintained without sacrificing performance. HPG Aligner can map reads containing many mismatches and indels, which can cover one or more introns. The algorithms have been implemented in the most convenient accelerator hardware. Thus, BWT has been implemented in two ways: in a multi-thread library that exploits the modern multicore CPUs and also using Nvidia CUDA library for GPUs. Both implementations can be executed simultaneously to obtain higher performance. On the other hand, SW benefits from the parallelism of SIMD registers present in each core of modern CPUs and has been implemented using SSE4 instructions.</p><p>The performance of HPG aligner was greatly improved with the implementation of some specific strategies to overcome the bottlenecks detected in the alignment process. Most significant implementation improvements include: (i) the use of a custom SW implementation based on multisequence vectors;<sup><a reveal-id="DSV039C27" data-open="DSV039C27" class="link link-ref link-reveal xref-bibr">27</a></sup> during the alignment, SW executions are buffered to execute them in parallel. (ii) Grouping seeds that map close in the genome (conceptually similar to clustering) was an expensive operation. This was easily solved by storing all the seeds from different chromosomes independently and sorted. (iii) BWT is a very fast algorithm and a significant fraction of the FASTQ file could be aligned with less than two mismatches. Therefore, reads are aligned with BWT before seeding. When the alignment fails (because there are more than two mismatches), the aligned part of the read is still stored as a seed (i.e. if after BWT there are 40 bp exact alignment, then this is considered a seed.). (iv) The metaexon data structure (see above and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/dsv039_Supplementary_Data.zip?Expires=2147483647&Signature=O5zhXMbjdL9MfWtRHyZZoQtjwf5zaVXvYWYJtmT4rJBXhDGjpUt1cPrA598ZLSZ5pBQ6P6Lva2hSCiTy5Iub-1KISygLW8FTa3n-3Q0Hx-c6T6cnlY9n58wYv4xALGU6Fhnc4tm5sQRRSXH6LHC6xvCJn3GOzr9jmAz9AwCyLzaX14zCOGlfVslVT1AUEsVTh5IqKN1mCI0yqmSXKsVMgrqYyU95lTF6GY4a9BEbPphosWJ6-xflQndXzfUiDMr26ltDmvR6B2SLrM3AsMrURh-wZzNo2ZJj52bNBe23SttTfMf~nmqX1hQsgV5hvy8vbhYo4oeFS5mnqlhH9NS8ZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>) reduces the computing time for some of the more difficult alignments.</p><p>To improve the overall performance of the implementation, several advanced tools for high-performance profiling were used during the development (in particular, Paraver <a class="link link-uri" href="http://www.bsc.es/computer-sciences/performance-tools/paraver">http://www.bsc.es/computer-sciences/performance-tools/paraver</a> and Extrae <a class="link link-uri" href="http://www.bsc.es/computer-sciences/extrae">http://www.bsc.es/computer-sciences/extrae</a>). These tools revealed that some steps of the internal pipeline were not taking fully advantage of the hardware resources. A dynamic runtime engine was developed and built inside HPG Aligner.<sup><a reveal-id="DSV039C28" data-open="DSV039C28" class="link link-ref link-reveal xref-bibr">28</a></sup> Each of the steps of the pipeline were considered as a task that needs some resource to be executed. Requested resources were assigned by the engine runtime in a dynamic and efficient way. In this way, the computing resources (such as cores) are assigned to the different tasks in an optimal way to avoid some cores to be waiting for other tasks to finish. This is a more complex implementation, which can usually be found in HPC applications. This allowed us to increase significantly the performance.</p><p>By leveraging the HPC implementation, the software runs many times faster than any other currently available solution and, in addition, it presents much higher sensitivity. Moreover, this implementation has proved to be technically much more efficient than current state-of-the art implementations since (i) the requirement of memory is much lower and it is kept below 10 GB, (ii) scalability in a multicore CPUs (up to 24 cores) is maintained and (iii) no large secondary indices or ‘temp’ files are generated during the execution, so no extra hard disk space is required.</p>                            <h3 scrollto-destination=87197068 id="87197068" class="section-title" data-legacy-id=s2a>2.1. Paired and unpaired RNA-seq simulated datasets comparison</h3>
<p>We have used two popular programs for the simulations of paired and unpaired reads from the human transcriptome: BEERS (<a class="link link-uri" href="http://www.cbil.upenn.edu/BEERS/">http://www.cbil.upenn.edu/BEERS/</a>), a simulator specifically devised to produce mRNA populations<sup><a reveal-id="DSV039C17" data-open="DSV039C17" class="link link-ref link-reveal xref-bibr">17</a></sup> and also the popular dwgsim 0.1.8 from SAMtools (<a class="link link-uri" href="http://sourceforge.net/apps/mediawiki/dnaa/index.php?title=Whole_Genome_Simulation">http://sourceforge.net/apps/mediawiki/dnaa/index.php?title=Whole_Genome_Simulation</a>).</p><p>Using BEERS, we generated 10 million reads for 10,000 genes in a duplicate of all the simulations, that is, FastQ files of 50, 75, 100, 150, 250 and 400 bp long, with different error ratios (0.001, 0.01 and 0.02 corresponding to 0.1, 1 and 2% mismatches, respectively). In all the configurations, the fraction of mutations that are indels was set to 10%.</p><p>The program dwgsim was run in ‘Illumina’ mode with option ‘-c 0’ for the six different length sizes tested. To study sensitivity against mutations and N's present in reads, three quality configurations were tested. First, a high-quality dataset containing 0.1% of mutations and a maximum of 2 N's per read was generated with options ‘-r 0.001 -n 2’. For the second and third datasets, higher proportion of mutations and N's were allowed to increase up to 1 and 2% of mutations, respectively, and 3 N's per read with options ‘-r 0.01 -n 3’ and ‘-r 0.02 -n 3’. In both configurations, the fraction of mutations that are indels was set to 10% with option ‘-R 0.1’.</p><p>The coordinates for cDNA sequences were taken from Ensembl 68 built upon GRCh37. For all combinations of length sizes and quality configurations, single-end and paired-end datasets were generated. For paired-end datasets, the inner distance between reads was set to 400 for reads smaller than 250 nt and a distance of 250 for the datasets of 250 and 400 length sizes.</p><p>Human assembly GRCh37.p8 was taken from Ensembl 68. HPG Aligner was run in ‘rna’ mode. The default command line parameters for the programs can be found in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/dsv039_Supplementary_Data.zip?Expires=2147483647&Signature=O5zhXMbjdL9MfWtRHyZZoQtjwf5zaVXvYWYJtmT4rJBXhDGjpUt1cPrA598ZLSZ5pBQ6P6Lva2hSCiTy5Iub-1KISygLW8FTa3n-3Q0Hx-c6T6cnlY9n58wYv4xALGU6Fhnc4tm5sQRRSXH6LHC6xvCJn3GOzr9jmAz9AwCyLzaX14zCOGlfVslVT1AUEsVTh5IqKN1mCI0yqmSXKsVMgrqYyU95lTF6GY4a9BEbPphosWJ6-xflQndXzfUiDMr26ltDmvR6B2SLrM3AsMrURh-wZzNo2ZJj52bNBe23SttTfMf~nmqX1hQsgV5hvy8vbhYo4oeFS5mnqlhH9NS8ZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary data</a></span>.</p><p>Benchmarks were performed in a high-end machine with two hexa-core Intel Xeon E5645 2.40 GHz CPUs and 48 GB of memory. All executions were done using the 12 cores available and memory use was monitored. HPG Aligner showed a memory peak of 10 GB.</p><p>All the FastQ file generated in the simulations and the BAM files generated in the benchmarking are available at: <a class="link link-uri" href="http://bioinfo.cipf.es/publications/supplementary_material">http://bioinfo.cipf.es/publications/supplementary_material</a>.</p>                            <h3 scrollto-destination=87197076 id="87197076" class="section-title" data-legacy-id=s2b>2.2. Real RNA-seq datasets comparison</h3>
<p>To test short- and long-read alignment with real data, two different datasets from Sequence Read Archive (SRA) were used, and the first dataset tested was SRR364003 (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sra?term=SRR364003">http://www.ncbi.nlm.nih.gov/sra?term=SRR364003</a>) sequenced with Illumina HiSeq 2000 platform and contained 81.6 million single and paired-end short reads of 100 nt length; the second dataset was built by combining the two runs of SRX025091 (<a class="link link-uri" href="http://www.ncbi.nlm.nih.gov/sra?term=SRX025091">http://www.ncbi.nlm.nih.gov/sra?term=SRX025091</a>) sequenced using the Roche 454 GS FLX platform; in total, a dataset of 1.26 million of single-end reads of about sequences, 600 nt long, was obtained.</p><p>The programs were run using default command line options in a single and paired-end mode (see above). Evaluation was performed by counting as correctly mapped the number of reads mapping within known transcript positions. The known transcripts are taken from Ensembl.<sup><a reveal-id="DSV039C29" data-open="DSV039C29" class="link link-ref link-reveal xref-bibr">29</a></sup> Benchmarks were performed in a high-end machine with two hexa-core Intel Xeon E5645 2.40 GHz CPUs and 48 GB of memory.</p>                            <h2 scrollto-destination=87197079 id="87197079" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=87197080 id="87197080" class="section-title" data-legacy-id=s3a>3.1. Performance of the method: speed and sensitivity</h3>
<p>We have compared the proposed aligner with the most extensively used RNA-seq mapper, TopHat2<sup><a reveal-id="DSV039C14" data-open="DSV039C14" class="link link-ref link-reveal xref-bibr">14</a></sup> (version 2.0.9), STAR<sup><a reveal-id="DSV039C18" data-open="DSV039C18" class="link link-ref link-reveal xref-bibr">18</a></sup> (version 2.3.0), MapSplice<sup><a reveal-id="DSV039C19" data-open="DSV039C19" class="link link-ref link-reveal xref-bibr">19</a></sup> (version 2.1.3) and HISAT<sup><a reveal-id="DSV039C21" data-open="DSV039C21" class="link link-ref link-reveal xref-bibr">21</a></sup> in simulated datasets with 10 million reads of lengths of 50, 75, 100, 150, 250 and 400 nt, in three scenarios of variability (0.1, 1 and 2% of mismatches, 10% of these being indels). Simulations were initially carried out with the BEERS program (see the Methods section).</p><p>Figure <a reveal-id="DSV039F2" data-open="DSV039F2" class="link link-reveal link-table xref-fig">2</a> shows the sensitivity of the proposed aligner, measured as the number of reads generated by BEERS correctly mapped (that is, mapping correctly the start and end and covering splice sites in between). HPG Aligner outperforms TopHat2 with Bowtie 2, HISAT (except in unrealistic scenarios of no error with read lengths of 150 and 250), STAR and MapSplice, even when mapping short reads (see details in Table <a reveal-id="DSV039TB1" data-open="DSV039TB1" class="link link-reveal link-table xref-fig">1</a>) and, as read length grows and the error rate increases, the difference in sensitivity increases even more. In current conventional lengths (100 bp) in an ideal scenario of low error, performances are quite similar (95–96%). However, if error is a bit higher (1%), HPG Aligner and MapSplice 2 maintain the sensitivity (94.50 and 95.34%, respectively). At this read length, TopHat2 and STAR sensitivities drop to a lower, but still reasonable value of ∼87%. For longer reads, the difference in sensitivity between HPG Aligner and STAR with respect to TopHat2 and MapSplice 2 grows considerably in scenarios of moderate error (see Table <a reveal-id="DSV039TB1" data-open="DSV039TB1" class="link link-reveal link-table xref-fig">1</a> reads 250 or 400 long at errors 1 or 2%). TopHat2 presents a peculiar behaviour: it keeps an extraordinarily low ratio of incorrect read mapping (below 2%) at the exchange of increasing the number of unmapped reads, which arrives to 27.13% for read lengths of 150 bp and 1% error and 53.29% for 250 bp (same error). It is remarkable that HPG Aligner, STAR and MapSplice keep the number of unmapped read below 5% across all the range of read lengths. Regarding runtimes, HPG Aligner runs much faster, ranging from 7× to almost than 10× for read lengths of 50–150, respectively, for TopHat2, 7× in the case of MapSplice and 2× in the case of STAR. HISAT runtimes are similar, although a bit slower, except in the case of very short reads (not produces anymore by modern sequencers), in which it is slightly faster than HPG Aligner. </p>                        <a id="87197083" scrollto-destination="87197083"></a>
<div content-id="DSV039TB1" class="table-modal"><div class="table-wrap"><div id="DSV039TB1" class="table-wrap-title" data-id="DSV039TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Benchmarking for the simulated dataset containing 10 million single-end reads simulated with the <em>BEERS</em> program</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">RL </th><th rowspan="2">MR (%) </th><th colspan="4">HPG Aligner 1<hr /> </th><th colspan="4">HISAT<hr /> </th><th colspan="4">STAR 2<hr /> </th><th colspan="4">TopHat 2 + Bowtie 2<hr /> </th><th colspan="4">MapSplice 2<hr /> </th></tr><tr><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th></tr></thead><tbody><tr><td> </td><td>0.1 </td><td>3.08 </td><td>0.42 </td><td>96.5 </td><td>4.8 </td><td>4.42 </td><td>3.22 </td><td>92.36 </td><td>4.45 </td><td>7.32 </td><td>1.46 </td><td>91.22 </td><td>8.28 </td><td>2.05 </td><td>0.93 </td><td>97.02 </td><td>36.35 </td><td>2.82 </td><td>1.16 </td><td>96.02 </td><td>26.93 </td></tr><tr><td>50 </td><td>1 </td><td>4.26 </td><td>0.84 </td><td>94.9 </td><td>4.9 </td><td>9.79 </td><td>8.61 </td><td>81.60 </td><td>4.80 </td><td>11.23 </td><td>1.72 </td><td>87.05 </td><td>8.73 </td><td>2.06 </td><td>3.04 </td><td>94.90 </td><td>79.65 </td><td>2.95 </td><td>2.03 </td><td>95.02 </td><td>26.13 </td></tr><tr><td> </td><td>2 </td><td>5.82 </td><td>2.26 </td><td>91.92 </td><td>5.2 </td><td>16.29 </td><td>17.18 </td><td>66.53 </td><td>4.58 </td><td>15.44 </td><td>3.16 </td><td>81.40 </td><td>8.85 </td><td>2.00 </td><td>10.18 </td><td>87.82 </td><td>374.9 </td><td>3.21 </td><td>4.81 </td><td>91.98 </td><td>26.08 </td></tr><tr><td> </td><td>0.1 </td><td>3.51 </td><td>0.24 </td><td>96.25 </td><td>5.8 </td><td>2.30 </td><td>1.13 </td><td>96.57 </td><td>6.21 </td><td>7.81 </td><td>0.46 </td><td>91.73 </td><td>9.52 </td><td>0.28 </td><td>2.98 </td><td>96.74 </td><td>48.22 </td><td>3.31 </td><td>0.24 </td><td>96.45 </td><td>37.32 </td></tr><tr><td>75 </td><td>1 </td><td>4.17 </td><td>0.94 </td><td>94.89 </td><td>6.0 </td><td>4.64 </td><td>3.48 </td><td>91.88 </td><td>6.40 </td><td>11.82 </td><td>1.18 </td><td>87.00 </td><td>9.97 </td><td>1.24 </td><td>6.90 </td><td>91.86 </td><td>50.78 </td><td>3.59 </td><td>1.16 </td><td>95.25 </td><td>35.05 </td></tr><tr><td> </td><td>2 </td><td>5.64 </td><td>1.27 </td><td>93.09 </td><td>6.4 </td><td>10.59 </td><td>10.46 </td><td>78.95 </td><td>6.46 </td><td>16.62 </td><td>1.55 </td><td>81.83 </td><td>11.25 </td><td>0.94 </td><td>22.18 </td><td>76.88 </td><td>58.93 </td><td>5.13 </td><td>2.20 </td><td>92.67 </td><td>34.28 </td></tr><tr><td> </td><td>0.1 </td><td>4.07 </td><td>0.24 </td><td>95.69 </td><td>7.4 </td><td>2.57 </td><td>1.24 </td><td>96.19 </td><td>7.86 </td><td>7.59 </td><td>0.64 </td><td>91.77 </td><td>11.77 </td><td>1.69 </td><td>1.94 </td><td>96.37 </td><td>65.08 </td><td>3.54 </td><td>0.26 </td><td>96.20 </td><td>51.17 </td></tr><tr><td>100 </td><td>1 </td><td>4.95 </td><td>0.55 </td><td>94.50 </td><td>7.5 </td><td>5.70 </td><td>4.51 </td><td>89.79 </td><td>8.15 </td><td>11.90 </td><td>0.62 </td><td>87.48 </td><td>12.08 </td><td>0.99 </td><td>11.70 </td><td>87.31 </td><td>74.02 </td><td>3.98 </td><td>0.68 </td><td>95.34 </td><td>46.03 </td></tr><tr><td> </td><td>2 </td><td>6.07 </td><td>0.93 </td><td>93.00 </td><td>7.8 </td><td>15.64 </td><td>17.19 </td><td>67.17 </td><td>8.40 </td><td>16.48 </td><td>1.14 </td><td>82.38 </td><td>12.35 </td><td>0.60 </td><td>36.14 </td><td>63.26 </td><td>85.65 </td><td>6.45 </td><td>1.50 </td><td>92.05 </td><td>45.6 </td></tr><tr><td> </td><td>0.1 </td><td>5.58 </td><td>0.31 </td><td>94.11 </td><td>10.5 </td><td>3.15 </td><td>1.56 </td><td>95.29 </td><td>11.75 </td><td>8.40 </td><td>0.57 </td><td>91.03 </td><td>15.07 </td><td>1.49 </td><td>3.55 </td><td>94.96 </td><td>97.9 </td><td>5.45 </td><td>0.21 </td><td>94.34 </td><td>56.88 </td></tr><tr><td>150 </td><td>1 </td><td>7.77 </td><td>1.95 </td><td>90.28 </td><td>10.6 </td><td>12.49 </td><td>13.03 </td><td>74.48 </td><td>11.41 </td><td>12.88 </td><td>2.73 </td><td>84.39 </td><td>15.08 </td><td>0.82 </td><td>27.13 </td><td>72.05 </td><td>121.55 </td><td>8.23 </td><td>2.67 </td><td>89.10 </td><td>53.53 </td></tr><tr><td> </td><td>2 </td><td>7.06 </td><td>0.78 </td><td>92.16 </td><td>10.9 </td><td>24.42 </td><td>39.76 </td><td>35.82 </td><td>11.91 </td><td>15.97 </td><td>0.73 </td><td>83.30 </td><td>15.73 </td><td>0.45 </td><td>61.84 </td><td>37.71 </td><td>140.28 </td><td>13.65 </td><td>1.57 </td><td>84.78 </td><td>56.73 </td></tr><tr><td> </td><td>0.1 </td><td>8.21 </td><td>0.66 </td><td>91.13 </td><td>16.55 </td><td>4.49 </td><td>2.47 </td><td>93.04 </td><td>17.80 </td><td>9.13 </td><td>0.62 </td><td>90.25 </td><td>21.58 </td><td>1.62 </td><td>6.70 </td><td>91.68 </td><td>185.78 </td><td>11.51 </td><td>0.23 </td><td>88.26 </td><td>75.55 </td></tr><tr><td>250 </td><td>1 </td><td>8.43 </td><td>0.81 </td><td>90.76 </td><td>16.5 </td><td>22.28 </td><td>31.6 </td><td>46.12 </td><td>19.28 </td><td>12.99 </td><td>0.71 </td><td>86.30 </td><td>22.3 </td><td>0.28 </td><td>53.29 </td><td>46.43 </td><td>256.83 </td><td>29.85 </td><td>0.75 </td><td>69.40 </td><td>79.28 </td></tr><tr><td> </td><td>2 </td><td>8.72 </td><td>1.62 </td><td>89.66 </td><td>17.45 </td><td>17.75 </td><td>77.04 </td><td>5.21 </td><td>17.46 </td><td>18.35 </td><td>1.43 </td><td>80.22 </td><td>23.35 </td><td>0.07 </td><td>89.62 </td><td>10.31 </td><td>309.85 </td><td>35.27 </td><td>1.24 </td><td>63.49 </td><td>76.37 </td></tr><tr><td> </td><td>0.1 </td><td>10.02 </td><td>1.08 </td><td>88.90 </td><td>25.45 </td><td>6.88 </td><td>5.04 </td><td>88.08 </td><td>28.5 </td><td>9.98 </td><td>0.56 </td><td>89.46 </td><td>32.12 </td><td>0.99 </td><td>12.32 </td><td>86.69 </td><td>369.58 </td><td>17.33 </td><td>0.03 </td><td>82.64 </td><td>106.67 </td></tr><tr><td>400 </td><td>1 </td><td>12.94 </td><td>3.35 </td><td>83.71 </td><td>25.6 </td><td>22.52 </td><td>66.17 </td><td>11.31 </td><td>29.7 </td><td>14.82 </td><td>4.26 </td><td>80.92 </td><td>33.83 </td><td>0.09 </td><td>82.40 </td><td>17.51 </td><td>573.18 </td><td>51.64 </td><td>1.65 </td><td>46.71 </td><td>103.98 </td></tr><tr><td> </td><td>2 </td><td>11.78 </td><td>2.84 </td><td>85.38 </td><td>26.0 </td><td>3.08 </td><td>96.82 </td><td>0.10 </td><td>27.10 </td><td>32.75 </td><td>2.75 </td><td>64.5 </td><td>24.5 </td><td>0.01 </td><td>99.02 </td><td>0.97 </td><td>603.28 </td><td>54.34 </td><td>0.51 </td><td>45.15 </td><td>106.53 </td></tr></tbody></table></div><div content-id="DSV039TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">RL </th><th rowspan="2">MR (%) </th><th colspan="4">HPG Aligner 1<hr /> </th><th colspan="4">HISAT<hr /> </th><th colspan="4">STAR 2<hr /> </th><th colspan="4">TopHat 2 + Bowtie 2<hr /> </th><th colspan="4">MapSplice 2<hr /> </th></tr><tr><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th></tr></thead><tbody><tr><td> </td><td>0.1 </td><td>3.08 </td><td>0.42 </td><td>96.5 </td><td>4.8 </td><td>4.42 </td><td>3.22 </td><td>92.36 </td><td>4.45 </td><td>7.32 </td><td>1.46 </td><td>91.22 </td><td>8.28 </td><td>2.05 </td><td>0.93 </td><td>97.02 </td><td>36.35 </td><td>2.82 </td><td>1.16 </td><td>96.02 </td><td>26.93 </td></tr><tr><td>50 </td><td>1 </td><td>4.26 </td><td>0.84 </td><td>94.9 </td><td>4.9 </td><td>9.79 </td><td>8.61 </td><td>81.60 </td><td>4.80 </td><td>11.23 </td><td>1.72 </td><td>87.05 </td><td>8.73 </td><td>2.06 </td><td>3.04 </td><td>94.90 </td><td>79.65 </td><td>2.95 </td><td>2.03 </td><td>95.02 </td><td>26.13 </td></tr><tr><td> </td><td>2 </td><td>5.82 </td><td>2.26 </td><td>91.92 </td><td>5.2 </td><td>16.29 </td><td>17.18 </td><td>66.53 </td><td>4.58 </td><td>15.44 </td><td>3.16 </td><td>81.40 </td><td>8.85 </td><td>2.00 </td><td>10.18 </td><td>87.82 </td><td>374.9 </td><td>3.21 </td><td>4.81 </td><td>91.98 </td><td>26.08 </td></tr><tr><td> </td><td>0.1 </td><td>3.51 </td><td>0.24 </td><td>96.25 </td><td>5.8 </td><td>2.30 </td><td>1.13 </td><td>96.57 </td><td>6.21 </td><td>7.81 </td><td>0.46 </td><td>91.73 </td><td>9.52 </td><td>0.28 </td><td>2.98 </td><td>96.74 </td><td>48.22 </td><td>3.31 </td><td>0.24 </td><td>96.45 </td><td>37.32 </td></tr><tr><td>75 </td><td>1 </td><td>4.17 </td><td>0.94 </td><td>94.89 </td><td>6.0 </td><td>4.64 </td><td>3.48 </td><td>91.88 </td><td>6.40 </td><td>11.82 </td><td>1.18 </td><td>87.00 </td><td>9.97 </td><td>1.24 </td><td>6.90 </td><td>91.86 </td><td>50.78 </td><td>3.59 </td><td>1.16 </td><td>95.25 </td><td>35.05 </td></tr><tr><td> </td><td>2 </td><td>5.64 </td><td>1.27 </td><td>93.09 </td><td>6.4 </td><td>10.59 </td><td>10.46 </td><td>78.95 </td><td>6.46 </td><td>16.62 </td><td>1.55 </td><td>81.83 </td><td>11.25 </td><td>0.94 </td><td>22.18 </td><td>76.88 </td><td>58.93 </td><td>5.13 </td><td>2.20 </td><td>92.67 </td><td>34.28 </td></tr><tr><td> </td><td>0.1 </td><td>4.07 </td><td>0.24 </td><td>95.69 </td><td>7.4 </td><td>2.57 </td><td>1.24 </td><td>96.19 </td><td>7.86 </td><td>7.59 </td><td>0.64 </td><td>91.77 </td><td>11.77 </td><td>1.69 </td><td>1.94 </td><td>96.37 </td><td>65.08 </td><td>3.54 </td><td>0.26 </td><td>96.20 </td><td>51.17 </td></tr><tr><td>100 </td><td>1 </td><td>4.95 </td><td>0.55 </td><td>94.50 </td><td>7.5 </td><td>5.70 </td><td>4.51 </td><td>89.79 </td><td>8.15 </td><td>11.90 </td><td>0.62 </td><td>87.48 </td><td>12.08 </td><td>0.99 </td><td>11.70 </td><td>87.31 </td><td>74.02 </td><td>3.98 </td><td>0.68 </td><td>95.34 </td><td>46.03 </td></tr><tr><td> </td><td>2 </td><td>6.07 </td><td>0.93 </td><td>93.00 </td><td>7.8 </td><td>15.64 </td><td>17.19 </td><td>67.17 </td><td>8.40 </td><td>16.48 </td><td>1.14 </td><td>82.38 </td><td>12.35 </td><td>0.60 </td><td>36.14 </td><td>63.26 </td><td>85.65 </td><td>6.45 </td><td>1.50 </td><td>92.05 </td><td>45.6 </td></tr><tr><td> </td><td>0.1 </td><td>5.58 </td><td>0.31 </td><td>94.11 </td><td>10.5 </td><td>3.15 </td><td>1.56 </td><td>95.29 </td><td>11.75 </td><td>8.40 </td><td>0.57 </td><td>91.03 </td><td>15.07 </td><td>1.49 </td><td>3.55 </td><td>94.96 </td><td>97.9 </td><td>5.45 </td><td>0.21 </td><td>94.34 </td><td>56.88 </td></tr><tr><td>150 </td><td>1 </td><td>7.77 </td><td>1.95 </td><td>90.28 </td><td>10.6 </td><td>12.49 </td><td>13.03 </td><td>74.48 </td><td>11.41 </td><td>12.88 </td><td>2.73 </td><td>84.39 </td><td>15.08 </td><td>0.82 </td><td>27.13 </td><td>72.05 </td><td>121.55 </td><td>8.23 </td><td>2.67 </td><td>89.10 </td><td>53.53 </td></tr><tr><td> </td><td>2 </td><td>7.06 </td><td>0.78 </td><td>92.16 </td><td>10.9 </td><td>24.42 </td><td>39.76 </td><td>35.82 </td><td>11.91 </td><td>15.97 </td><td>0.73 </td><td>83.30 </td><td>15.73 </td><td>0.45 </td><td>61.84 </td><td>37.71 </td><td>140.28 </td><td>13.65 </td><td>1.57 </td><td>84.78 </td><td>56.73 </td></tr><tr><td> </td><td>0.1 </td><td>8.21 </td><td>0.66 </td><td>91.13 </td><td>16.55 </td><td>4.49 </td><td>2.47 </td><td>93.04 </td><td>17.80 </td><td>9.13 </td><td>0.62 </td><td>90.25 </td><td>21.58 </td><td>1.62 </td><td>6.70 </td><td>91.68 </td><td>185.78 </td><td>11.51 </td><td>0.23 </td><td>88.26 </td><td>75.55 </td></tr><tr><td>250 </td><td>1 </td><td>8.43 </td><td>0.81 </td><td>90.76 </td><td>16.5 </td><td>22.28 </td><td>31.6 </td><td>46.12 </td><td>19.28 </td><td>12.99 </td><td>0.71 </td><td>86.30 </td><td>22.3 </td><td>0.28 </td><td>53.29 </td><td>46.43 </td><td>256.83 </td><td>29.85 </td><td>0.75 </td><td>69.40 </td><td>79.28 </td></tr><tr><td> </td><td>2 </td><td>8.72 </td><td>1.62 </td><td>89.66 </td><td>17.45 </td><td>17.75 </td><td>77.04 </td><td>5.21 </td><td>17.46 </td><td>18.35 </td><td>1.43 </td><td>80.22 </td><td>23.35 </td><td>0.07 </td><td>89.62 </td><td>10.31 </td><td>309.85 </td><td>35.27 </td><td>1.24 </td><td>63.49 </td><td>76.37 </td></tr><tr><td> </td><td>0.1 </td><td>10.02 </td><td>1.08 </td><td>88.90 </td><td>25.45 </td><td>6.88 </td><td>5.04 </td><td>88.08 </td><td>28.5 </td><td>9.98 </td><td>0.56 </td><td>89.46 </td><td>32.12 </td><td>0.99 </td><td>12.32 </td><td>86.69 </td><td>369.58 </td><td>17.33 </td><td>0.03 </td><td>82.64 </td><td>106.67 </td></tr><tr><td>400 </td><td>1 </td><td>12.94 </td><td>3.35 </td><td>83.71 </td><td>25.6 </td><td>22.52 </td><td>66.17 </td><td>11.31 </td><td>29.7 </td><td>14.82 </td><td>4.26 </td><td>80.92 </td><td>33.83 </td><td>0.09 </td><td>82.40 </td><td>17.51 </td><td>573.18 </td><td>51.64 </td><td>1.65 </td><td>46.71 </td><td>103.98 </td></tr><tr><td> </td><td>2 </td><td>11.78 </td><td>2.84 </td><td>85.38 </td><td>26.0 </td><td>3.08 </td><td>96.82 </td><td>0.10 </td><td>27.10 </td><td>32.75 </td><td>2.75 </td><td>64.5 </td><td>24.5 </td><td>0.01 </td><td>99.02 </td><td>0.97 </td><td>603.28 </td><td>54.34 </td><td>0.51 </td><td>45.15 </td><td>106.53 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>First column, RL, indicated read length in bp. Second column represents the mutation rate (MR). For each program, the table contains the following columns with the percentages of: IMR: incorrectly mapped reads; RNM: reads not mapped; CMR: correctly mapped reads covering the corresponding splice junctions. The last column, T, represents runtimes for producing a BAM file in min.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197083" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV039TB1" class="table-wrap-title" data-id="DSV039TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Benchmarking for the simulated dataset containing 10 million single-end reads simulated with the <em>BEERS</em> program</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">RL </th><th rowspan="2">MR (%) </th><th colspan="4">HPG Aligner 1<hr /> </th><th colspan="4">HISAT<hr /> </th><th colspan="4">STAR 2<hr /> </th><th colspan="4">TopHat 2 + Bowtie 2<hr /> </th><th colspan="4">MapSplice 2<hr /> </th></tr><tr><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th></tr></thead><tbody><tr><td> </td><td>0.1 </td><td>3.08 </td><td>0.42 </td><td>96.5 </td><td>4.8 </td><td>4.42 </td><td>3.22 </td><td>92.36 </td><td>4.45 </td><td>7.32 </td><td>1.46 </td><td>91.22 </td><td>8.28 </td><td>2.05 </td><td>0.93 </td><td>97.02 </td><td>36.35 </td><td>2.82 </td><td>1.16 </td><td>96.02 </td><td>26.93 </td></tr><tr><td>50 </td><td>1 </td><td>4.26 </td><td>0.84 </td><td>94.9 </td><td>4.9 </td><td>9.79 </td><td>8.61 </td><td>81.60 </td><td>4.80 </td><td>11.23 </td><td>1.72 </td><td>87.05 </td><td>8.73 </td><td>2.06 </td><td>3.04 </td><td>94.90 </td><td>79.65 </td><td>2.95 </td><td>2.03 </td><td>95.02 </td><td>26.13 </td></tr><tr><td> </td><td>2 </td><td>5.82 </td><td>2.26 </td><td>91.92 </td><td>5.2 </td><td>16.29 </td><td>17.18 </td><td>66.53 </td><td>4.58 </td><td>15.44 </td><td>3.16 </td><td>81.40 </td><td>8.85 </td><td>2.00 </td><td>10.18 </td><td>87.82 </td><td>374.9 </td><td>3.21 </td><td>4.81 </td><td>91.98 </td><td>26.08 </td></tr><tr><td> </td><td>0.1 </td><td>3.51 </td><td>0.24 </td><td>96.25 </td><td>5.8 </td><td>2.30 </td><td>1.13 </td><td>96.57 </td><td>6.21 </td><td>7.81 </td><td>0.46 </td><td>91.73 </td><td>9.52 </td><td>0.28 </td><td>2.98 </td><td>96.74 </td><td>48.22 </td><td>3.31 </td><td>0.24 </td><td>96.45 </td><td>37.32 </td></tr><tr><td>75 </td><td>1 </td><td>4.17 </td><td>0.94 </td><td>94.89 </td><td>6.0 </td><td>4.64 </td><td>3.48 </td><td>91.88 </td><td>6.40 </td><td>11.82 </td><td>1.18 </td><td>87.00 </td><td>9.97 </td><td>1.24 </td><td>6.90 </td><td>91.86 </td><td>50.78 </td><td>3.59 </td><td>1.16 </td><td>95.25 </td><td>35.05 </td></tr><tr><td> </td><td>2 </td><td>5.64 </td><td>1.27 </td><td>93.09 </td><td>6.4 </td><td>10.59 </td><td>10.46 </td><td>78.95 </td><td>6.46 </td><td>16.62 </td><td>1.55 </td><td>81.83 </td><td>11.25 </td><td>0.94 </td><td>22.18 </td><td>76.88 </td><td>58.93 </td><td>5.13 </td><td>2.20 </td><td>92.67 </td><td>34.28 </td></tr><tr><td> </td><td>0.1 </td><td>4.07 </td><td>0.24 </td><td>95.69 </td><td>7.4 </td><td>2.57 </td><td>1.24 </td><td>96.19 </td><td>7.86 </td><td>7.59 </td><td>0.64 </td><td>91.77 </td><td>11.77 </td><td>1.69 </td><td>1.94 </td><td>96.37 </td><td>65.08 </td><td>3.54 </td><td>0.26 </td><td>96.20 </td><td>51.17 </td></tr><tr><td>100 </td><td>1 </td><td>4.95 </td><td>0.55 </td><td>94.50 </td><td>7.5 </td><td>5.70 </td><td>4.51 </td><td>89.79 </td><td>8.15 </td><td>11.90 </td><td>0.62 </td><td>87.48 </td><td>12.08 </td><td>0.99 </td><td>11.70 </td><td>87.31 </td><td>74.02 </td><td>3.98 </td><td>0.68 </td><td>95.34 </td><td>46.03 </td></tr><tr><td> </td><td>2 </td><td>6.07 </td><td>0.93 </td><td>93.00 </td><td>7.8 </td><td>15.64 </td><td>17.19 </td><td>67.17 </td><td>8.40 </td><td>16.48 </td><td>1.14 </td><td>82.38 </td><td>12.35 </td><td>0.60 </td><td>36.14 </td><td>63.26 </td><td>85.65 </td><td>6.45 </td><td>1.50 </td><td>92.05 </td><td>45.6 </td></tr><tr><td> </td><td>0.1 </td><td>5.58 </td><td>0.31 </td><td>94.11 </td><td>10.5 </td><td>3.15 </td><td>1.56 </td><td>95.29 </td><td>11.75 </td><td>8.40 </td><td>0.57 </td><td>91.03 </td><td>15.07 </td><td>1.49 </td><td>3.55 </td><td>94.96 </td><td>97.9 </td><td>5.45 </td><td>0.21 </td><td>94.34 </td><td>56.88 </td></tr><tr><td>150 </td><td>1 </td><td>7.77 </td><td>1.95 </td><td>90.28 </td><td>10.6 </td><td>12.49 </td><td>13.03 </td><td>74.48 </td><td>11.41 </td><td>12.88 </td><td>2.73 </td><td>84.39 </td><td>15.08 </td><td>0.82 </td><td>27.13 </td><td>72.05 </td><td>121.55 </td><td>8.23 </td><td>2.67 </td><td>89.10 </td><td>53.53 </td></tr><tr><td> </td><td>2 </td><td>7.06 </td><td>0.78 </td><td>92.16 </td><td>10.9 </td><td>24.42 </td><td>39.76 </td><td>35.82 </td><td>11.91 </td><td>15.97 </td><td>0.73 </td><td>83.30 </td><td>15.73 </td><td>0.45 </td><td>61.84 </td><td>37.71 </td><td>140.28 </td><td>13.65 </td><td>1.57 </td><td>84.78 </td><td>56.73 </td></tr><tr><td> </td><td>0.1 </td><td>8.21 </td><td>0.66 </td><td>91.13 </td><td>16.55 </td><td>4.49 </td><td>2.47 </td><td>93.04 </td><td>17.80 </td><td>9.13 </td><td>0.62 </td><td>90.25 </td><td>21.58 </td><td>1.62 </td><td>6.70 </td><td>91.68 </td><td>185.78 </td><td>11.51 </td><td>0.23 </td><td>88.26 </td><td>75.55 </td></tr><tr><td>250 </td><td>1 </td><td>8.43 </td><td>0.81 </td><td>90.76 </td><td>16.5 </td><td>22.28 </td><td>31.6 </td><td>46.12 </td><td>19.28 </td><td>12.99 </td><td>0.71 </td><td>86.30 </td><td>22.3 </td><td>0.28 </td><td>53.29 </td><td>46.43 </td><td>256.83 </td><td>29.85 </td><td>0.75 </td><td>69.40 </td><td>79.28 </td></tr><tr><td> </td><td>2 </td><td>8.72 </td><td>1.62 </td><td>89.66 </td><td>17.45 </td><td>17.75 </td><td>77.04 </td><td>5.21 </td><td>17.46 </td><td>18.35 </td><td>1.43 </td><td>80.22 </td><td>23.35 </td><td>0.07 </td><td>89.62 </td><td>10.31 </td><td>309.85 </td><td>35.27 </td><td>1.24 </td><td>63.49 </td><td>76.37 </td></tr><tr><td> </td><td>0.1 </td><td>10.02 </td><td>1.08 </td><td>88.90 </td><td>25.45 </td><td>6.88 </td><td>5.04 </td><td>88.08 </td><td>28.5 </td><td>9.98 </td><td>0.56 </td><td>89.46 </td><td>32.12 </td><td>0.99 </td><td>12.32 </td><td>86.69 </td><td>369.58 </td><td>17.33 </td><td>0.03 </td><td>82.64 </td><td>106.67 </td></tr><tr><td>400 </td><td>1 </td><td>12.94 </td><td>3.35 </td><td>83.71 </td><td>25.6 </td><td>22.52 </td><td>66.17 </td><td>11.31 </td><td>29.7 </td><td>14.82 </td><td>4.26 </td><td>80.92 </td><td>33.83 </td><td>0.09 </td><td>82.40 </td><td>17.51 </td><td>573.18 </td><td>51.64 </td><td>1.65 </td><td>46.71 </td><td>103.98 </td></tr><tr><td> </td><td>2 </td><td>11.78 </td><td>2.84 </td><td>85.38 </td><td>26.0 </td><td>3.08 </td><td>96.82 </td><td>0.10 </td><td>27.10 </td><td>32.75 </td><td>2.75 </td><td>64.5 </td><td>24.5 </td><td>0.01 </td><td>99.02 </td><td>0.97 </td><td>603.28 </td><td>54.34 </td><td>0.51 </td><td>45.15 </td><td>106.53 </td></tr></tbody></table></div><div content-id="DSV039TB1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">RL </th><th rowspan="2">MR (%) </th><th colspan="4">HPG Aligner 1<hr /> </th><th colspan="4">HISAT<hr /> </th><th colspan="4">STAR 2<hr /> </th><th colspan="4">TopHat 2 + Bowtie 2<hr /> </th><th colspan="4">MapSplice 2<hr /> </th></tr><tr><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th><th>IMR </th><th>RNM </th><th>CMR </th><th>T </th></tr></thead><tbody><tr><td> </td><td>0.1 </td><td>3.08 </td><td>0.42 </td><td>96.5 </td><td>4.8 </td><td>4.42 </td><td>3.22 </td><td>92.36 </td><td>4.45 </td><td>7.32 </td><td>1.46 </td><td>91.22 </td><td>8.28 </td><td>2.05 </td><td>0.93 </td><td>97.02 </td><td>36.35 </td><td>2.82 </td><td>1.16 </td><td>96.02 </td><td>26.93 </td></tr><tr><td>50 </td><td>1 </td><td>4.26 </td><td>0.84 </td><td>94.9 </td><td>4.9 </td><td>9.79 </td><td>8.61 </td><td>81.60 </td><td>4.80 </td><td>11.23 </td><td>1.72 </td><td>87.05 </td><td>8.73 </td><td>2.06 </td><td>3.04 </td><td>94.90 </td><td>79.65 </td><td>2.95 </td><td>2.03 </td><td>95.02 </td><td>26.13 </td></tr><tr><td> </td><td>2 </td><td>5.82 </td><td>2.26 </td><td>91.92 </td><td>5.2 </td><td>16.29 </td><td>17.18 </td><td>66.53 </td><td>4.58 </td><td>15.44 </td><td>3.16 </td><td>81.40 </td><td>8.85 </td><td>2.00 </td><td>10.18 </td><td>87.82 </td><td>374.9 </td><td>3.21 </td><td>4.81 </td><td>91.98 </td><td>26.08 </td></tr><tr><td> </td><td>0.1 </td><td>3.51 </td><td>0.24 </td><td>96.25 </td><td>5.8 </td><td>2.30 </td><td>1.13 </td><td>96.57 </td><td>6.21 </td><td>7.81 </td><td>0.46 </td><td>91.73 </td><td>9.52 </td><td>0.28 </td><td>2.98 </td><td>96.74 </td><td>48.22 </td><td>3.31 </td><td>0.24 </td><td>96.45 </td><td>37.32 </td></tr><tr><td>75 </td><td>1 </td><td>4.17 </td><td>0.94 </td><td>94.89 </td><td>6.0 </td><td>4.64 </td><td>3.48 </td><td>91.88 </td><td>6.40 </td><td>11.82 </td><td>1.18 </td><td>87.00 </td><td>9.97 </td><td>1.24 </td><td>6.90 </td><td>91.86 </td><td>50.78 </td><td>3.59 </td><td>1.16 </td><td>95.25 </td><td>35.05 </td></tr><tr><td> </td><td>2 </td><td>5.64 </td><td>1.27 </td><td>93.09 </td><td>6.4 </td><td>10.59 </td><td>10.46 </td><td>78.95 </td><td>6.46 </td><td>16.62 </td><td>1.55 </td><td>81.83 </td><td>11.25 </td><td>0.94 </td><td>22.18 </td><td>76.88 </td><td>58.93 </td><td>5.13 </td><td>2.20 </td><td>92.67 </td><td>34.28 </td></tr><tr><td> </td><td>0.1 </td><td>4.07 </td><td>0.24 </td><td>95.69 </td><td>7.4 </td><td>2.57 </td><td>1.24 </td><td>96.19 </td><td>7.86 </td><td>7.59 </td><td>0.64 </td><td>91.77 </td><td>11.77 </td><td>1.69 </td><td>1.94 </td><td>96.37 </td><td>65.08 </td><td>3.54 </td><td>0.26 </td><td>96.20 </td><td>51.17 </td></tr><tr><td>100 </td><td>1 </td><td>4.95 </td><td>0.55 </td><td>94.50 </td><td>7.5 </td><td>5.70 </td><td>4.51 </td><td>89.79 </td><td>8.15 </td><td>11.90 </td><td>0.62 </td><td>87.48 </td><td>12.08 </td><td>0.99 </td><td>11.70 </td><td>87.31 </td><td>74.02 </td><td>3.98 </td><td>0.68 </td><td>95.34 </td><td>46.03 </td></tr><tr><td> </td><td>2 </td><td>6.07 </td><td>0.93 </td><td>93.00 </td><td>7.8 </td><td>15.64 </td><td>17.19 </td><td>67.17 </td><td>8.40 </td><td>16.48 </td><td>1.14 </td><td>82.38 </td><td>12.35 </td><td>0.60 </td><td>36.14 </td><td>63.26 </td><td>85.65 </td><td>6.45 </td><td>1.50 </td><td>92.05 </td><td>45.6 </td></tr><tr><td> </td><td>0.1 </td><td>5.58 </td><td>0.31 </td><td>94.11 </td><td>10.5 </td><td>3.15 </td><td>1.56 </td><td>95.29 </td><td>11.75 </td><td>8.40 </td><td>0.57 </td><td>91.03 </td><td>15.07 </td><td>1.49 </td><td>3.55 </td><td>94.96 </td><td>97.9 </td><td>5.45 </td><td>0.21 </td><td>94.34 </td><td>56.88 </td></tr><tr><td>150 </td><td>1 </td><td>7.77 </td><td>1.95 </td><td>90.28 </td><td>10.6 </td><td>12.49 </td><td>13.03 </td><td>74.48 </td><td>11.41 </td><td>12.88 </td><td>2.73 </td><td>84.39 </td><td>15.08 </td><td>0.82 </td><td>27.13 </td><td>72.05 </td><td>121.55 </td><td>8.23 </td><td>2.67 </td><td>89.10 </td><td>53.53 </td></tr><tr><td> </td><td>2 </td><td>7.06 </td><td>0.78 </td><td>92.16 </td><td>10.9 </td><td>24.42 </td><td>39.76 </td><td>35.82 </td><td>11.91 </td><td>15.97 </td><td>0.73 </td><td>83.30 </td><td>15.73 </td><td>0.45 </td><td>61.84 </td><td>37.71 </td><td>140.28 </td><td>13.65 </td><td>1.57 </td><td>84.78 </td><td>56.73 </td></tr><tr><td> </td><td>0.1 </td><td>8.21 </td><td>0.66 </td><td>91.13 </td><td>16.55 </td><td>4.49 </td><td>2.47 </td><td>93.04 </td><td>17.80 </td><td>9.13 </td><td>0.62 </td><td>90.25 </td><td>21.58 </td><td>1.62 </td><td>6.70 </td><td>91.68 </td><td>185.78 </td><td>11.51 </td><td>0.23 </td><td>88.26 </td><td>75.55 </td></tr><tr><td>250 </td><td>1 </td><td>8.43 </td><td>0.81 </td><td>90.76 </td><td>16.5 </td><td>22.28 </td><td>31.6 </td><td>46.12 </td><td>19.28 </td><td>12.99 </td><td>0.71 </td><td>86.30 </td><td>22.3 </td><td>0.28 </td><td>53.29 </td><td>46.43 </td><td>256.83 </td><td>29.85 </td><td>0.75 </td><td>69.40 </td><td>79.28 </td></tr><tr><td> </td><td>2 </td><td>8.72 </td><td>1.62 </td><td>89.66 </td><td>17.45 </td><td>17.75 </td><td>77.04 </td><td>5.21 </td><td>17.46 </td><td>18.35 </td><td>1.43 </td><td>80.22 </td><td>23.35 </td><td>0.07 </td><td>89.62 </td><td>10.31 </td><td>309.85 </td><td>35.27 </td><td>1.24 </td><td>63.49 </td><td>76.37 </td></tr><tr><td> </td><td>0.1 </td><td>10.02 </td><td>1.08 </td><td>88.90 </td><td>25.45 </td><td>6.88 </td><td>5.04 </td><td>88.08 </td><td>28.5 </td><td>9.98 </td><td>0.56 </td><td>89.46 </td><td>32.12 </td><td>0.99 </td><td>12.32 </td><td>86.69 </td><td>369.58 </td><td>17.33 </td><td>0.03 </td><td>82.64 </td><td>106.67 </td></tr><tr><td>400 </td><td>1 </td><td>12.94 </td><td>3.35 </td><td>83.71 </td><td>25.6 </td><td>22.52 </td><td>66.17 </td><td>11.31 </td><td>29.7 </td><td>14.82 </td><td>4.26 </td><td>80.92 </td><td>33.83 </td><td>0.09 </td><td>82.40 </td><td>17.51 </td><td>573.18 </td><td>51.64 </td><td>1.65 </td><td>46.71 </td><td>103.98 </td></tr><tr><td> </td><td>2 </td><td>11.78 </td><td>2.84 </td><td>85.38 </td><td>26.0 </td><td>3.08 </td><td>96.82 </td><td>0.10 </td><td>27.10 </td><td>32.75 </td><td>2.75 </td><td>64.5 </td><td>24.5 </td><td>0.01 </td><td>99.02 </td><td>0.97 </td><td>603.28 </td><td>54.34 </td><td>0.51 </td><td>45.15 </td><td>106.53 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>First column, RL, indicated read length in bp. Second column represents the mutation rate (MR). For each program, the table contains the following columns with the percentages of: IMR: incorrectly mapped reads; RNM: reads not mapped; CMR: correctly mapped reads covering the corresponding splice junctions. The last column, T, represents runtimes for producing a BAM file in min.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197083" target="_blank">View Large</a></div></div>                        <a id="87197084" scrollto-destination="87197084"></a>
<div id="DSV039F2" data-id="DSV039F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/m_dsv03902.jpeg?Expires=2147483647&amp;Signature=mQSF9TVKwuM~C9WvYqm~RhIReq43xttYKWkOheVdqB93jvf9dkPuvfb8IYxQzHUmY8vvumWahAdNyvYtK3Dg0RvyyscSagHOUop5PYchpqGnxnot2SqR9-zA-Hd4pDasmFl7acmc5Bg~olcPm0C-Y~tf4M7xFl9jio2Jr6hbgV~lvViQbZoT0MW6gHJUgs7pzxVkT2QM7RZXxK93PHkEtv~vpaoIhuZtuO5jub73b-yWftpCjj2OI9abDdlPZry3l3NnIRTOIrbT~BqHbCcDskQ4B4kreeq2bhWxr6zPBH6am3YSZStdhdXhRnL7FshNWl5PsUmN4sq5jAeiVD5IHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representation of mapping sensitivity (percentage of reads correctly mapped) versus runtimes (in min) for simulated datasets of different single read lengths (from 50 bp, upper left, to 400 bp, lower right) containing 10 million single-end reads in two scenarios of variability (low variability represented by circles and high variability represented by triangles). The colour code corresponds to the different programs. In all the cases, HPG Aligner sensibilities and runtimes were better than the ones shown by the rest of programs (Supplementary Table S1). These differences increase as read length and error increase." path-from-xml="dsv03902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197084/dsv03902.jpeg" path-from-xml="dsv03902.jpeg" target="_blank">View large</a><a section="87197084" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/dsv03902.jpeg?Expires=2147483647&amp;Signature=kdHoz9Z1Or-a3jnfHuYEJk7ZunHa7tx-s-mcDU1mRxqk5DJ7P~qM9jcfMEVAVj9k5y9b8oRfnJaT5wsk7FH~AjSOuy2Skx4EnX63IGaE2s1iZkMekqWTmnwTwZvYlVFvP7iLKgGY-rceRLDUYXAJmuBA7K6qzQpyAFyFxMtdx9asSwJ-CX5u9fj0xe-nMiXLjvPug23lxA-I8WjQorWl-BxMKORLj-jWjRQdIaeLo6lnSEkWtcqjHUNESxFYSu9edLt~6-kS1v7~Pu5wm9saVQ1SAhf3NaTjCsHjyTumwaJ0sJ4UmCDSzdQc0Fe88dnAJDGcE12XaaE3a~-LoremAg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197084&amp;ar=1745299&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsv03902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representation of mapping sensitivity (percentage of reads correctly mapped) versus runtimes (in min) for simulated datasets of different single read lengths (from 50 bp, upper left, to 400 bp, lower right) containing 10 million single-end reads in two scenarios of variability (low variability represented by circles and high variability represented by triangles). The colour code corresponds to the different programs. In all the cases, HPG Aligner sensibilities and runtimes were better than the ones shown by the rest of programs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/dsv039_Supplementary_Data.zip?Expires=2147483647&amp;Signature=O5zhXMbjdL9MfWtRHyZZoQtjwf5zaVXvYWYJtmT4rJBXhDGjpUt1cPrA598ZLSZ5pBQ6P6Lva2hSCiTy5Iub-1KISygLW8FTa3n-3Q0Hx-c6T6cnlY9n58wYv4xALGU6Fhnc4tm5sQRRSXH6LHC6xvCJn3GOzr9jmAz9AwCyLzaX14zCOGlfVslVT1AUEsVTh5IqKN1mCI0yqmSXKsVMgrqYyU95lTF6GY4a9BEbPphosWJ6-xflQndXzfUiDMr26ltDmvR6B2SLrM3AsMrURh-wZzNo2ZJj52bNBe23SttTfMf~nmqX1hQsgV5hvy8vbhYo4oeFS5mnqlhH9NS8ZA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). These differences increase as read length and error increase.</p></div></div><div content-id="DSV039F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/m_dsv03902.jpeg?Expires=2147483647&amp;Signature=mQSF9TVKwuM~C9WvYqm~RhIReq43xttYKWkOheVdqB93jvf9dkPuvfb8IYxQzHUmY8vvumWahAdNyvYtK3Dg0RvyyscSagHOUop5PYchpqGnxnot2SqR9-zA-Hd4pDasmFl7acmc5Bg~olcPm0C-Y~tf4M7xFl9jio2Jr6hbgV~lvViQbZoT0MW6gHJUgs7pzxVkT2QM7RZXxK93PHkEtv~vpaoIhuZtuO5jub73b-yWftpCjj2OI9abDdlPZry3l3NnIRTOIrbT~BqHbCcDskQ4B4kreeq2bhWxr6zPBH6am3YSZStdhdXhRnL7FshNWl5PsUmN4sq5jAeiVD5IHQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Representation of mapping sensitivity (percentage of reads correctly mapped) versus runtimes (in min) for simulated datasets of different single read lengths (from 50 bp, upper left, to 400 bp, lower right) containing 10 million single-end reads in two scenarios of variability (low variability represented by circles and high variability represented by triangles). The colour code corresponds to the different programs. In all the cases, HPG Aligner sensibilities and runtimes were better than the ones shown by the rest of programs (Supplementary Table S1). These differences increase as read length and error increase." path-from-xml="dsv03902.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/87197084/dsv03902.jpeg" path-from-xml="dsv03902.jpeg" target="_blank">View large</a><a section="87197084" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/dsv03902.jpeg?Expires=2147483647&amp;Signature=kdHoz9Z1Or-a3jnfHuYEJk7ZunHa7tx-s-mcDU1mRxqk5DJ7P~qM9jcfMEVAVj9k5y9b8oRfnJaT5wsk7FH~AjSOuy2Skx4EnX63IGaE2s1iZkMekqWTmnwTwZvYlVFvP7iLKgGY-rceRLDUYXAJmuBA7K6qzQpyAFyFxMtdx9asSwJ-CX5u9fj0xe-nMiXLjvPug23lxA-I8WjQorWl-BxMKORLj-jWjRQdIaeLo6lnSEkWtcqjHUNESxFYSu9edLt~6-kS1v7~Pu5wm9saVQ1SAhf3NaTjCsHjyTumwaJ0sJ4UmCDSzdQc0Fe88dnAJDGcE12XaaE3a~-LoremAg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=87197084&amp;ar=1745299&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsv03902.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Representation of mapping sensitivity (percentage of reads correctly mapped) versus runtimes (in min) for simulated datasets of different single read lengths (from 50 bp, upper left, to 400 bp, lower right) containing 10 million single-end reads in two scenarios of variability (low variability represented by circles and high variability represented by triangles). The colour code corresponds to the different programs. In all the cases, HPG Aligner sensibilities and runtimes were better than the ones shown by the rest of programs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/dsv039_Supplementary_Data.zip?Expires=2147483647&amp;Signature=O5zhXMbjdL9MfWtRHyZZoQtjwf5zaVXvYWYJtmT4rJBXhDGjpUt1cPrA598ZLSZ5pBQ6P6Lva2hSCiTy5Iub-1KISygLW8FTa3n-3Q0Hx-c6T6cnlY9n58wYv4xALGU6Fhnc4tm5sQRRSXH6LHC6xvCJn3GOzr9jmAz9AwCyLzaX14zCOGlfVslVT1AUEsVTh5IqKN1mCI0yqmSXKsVMgrqYyU95lTF6GY4a9BEbPphosWJ6-xflQndXzfUiDMr26ltDmvR6B2SLrM3AsMrURh-wZzNo2ZJj52bNBe23SttTfMf~nmqX1hQsgV5hvy8vbhYo4oeFS5mnqlhH9NS8ZA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). These differences increase as read length and error increase.</p></div></div><p>We repeated the simulations in the same conditions with the dwgsim simulator (see the Methods section). The results obtained in the transcriptomics simulated scenario provided by dwgsim are similar although the percentages of correct mapping decrease for all the programs across the wide range of lengths (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/dsv039_Supplementary_Data.zip?Expires=2147483647&Signature=O5zhXMbjdL9MfWtRHyZZoQtjwf5zaVXvYWYJtmT4rJBXhDGjpUt1cPrA598ZLSZ5pBQ6P6Lva2hSCiTy5Iub-1KISygLW8FTa3n-3Q0Hx-c6T6cnlY9n58wYv4xALGU6Fhnc4tm5sQRRSXH6LHC6xvCJn3GOzr9jmAz9AwCyLzaX14zCOGlfVslVT1AUEsVTh5IqKN1mCI0yqmSXKsVMgrqYyU95lTF6GY4a9BEbPphosWJ6-xflQndXzfUiDMr26ltDmvR6B2SLrM3AsMrURh-wZzNo2ZJj52bNBe23SttTfMf~nmqX1hQsgV5hvy8vbhYo4oeFS5mnqlhH9NS8ZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). In particular, the sensitivity of TopHat2 notably decreases in the low variability (0.1%). As noise and read length increase, the sensitivity falls down again. One of the reasons for this general decrease in the sensitivity can be the fact that dwgsim has an extra parameter that accounts for the error of the sequencing technology, which introduces an extra amount of mismatches per read. As expected, the results obtained in the case of paired reads were quite similar (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/dsv039_Supplementary_Data.zip?Expires=2147483647&Signature=O5zhXMbjdL9MfWtRHyZZoQtjwf5zaVXvYWYJtmT4rJBXhDGjpUt1cPrA598ZLSZ5pBQ6P6Lva2hSCiTy5Iub-1KISygLW8FTa3n-3Q0Hx-c6T6cnlY9n58wYv4xALGU6Fhnc4tm5sQRRSXH6LHC6xvCJn3GOzr9jmAz9AwCyLzaX14zCOGlfVslVT1AUEsVTh5IqKN1mCI0yqmSXKsVMgrqYyU95lTF6GY4a9BEbPphosWJ6-xflQndXzfUiDMr26ltDmvR6B2SLrM3AsMrURh-wZzNo2ZJj52bNBe23SttTfMf~nmqX1hQsgV5hvy8vbhYo4oeFS5mnqlhH9NS8ZA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2</a></span>).</p><p>Actually, sequencing technologies are progressively increasing the read length (currently, HiSeq 2500 or HiSeq 1500 in rapid run mode produces read lengths of 150 bp, 454 GS FLX+ Lifesciences average read length is of 700–800 bp and Pacific Biosciences is over 3 Kb). Therefore, a desirable property for aligners is robustness against increasingly large sequence lengths.</p><p>When the five aligners are tested against a real dataset of 20 million reads, 100 bp long, from an Illumina HiSeq (SRA accession SRR364003), HPG Aligner was able to map at known transcript positions almost 80% of the reads, whereas HISAT and MapSplice reached ∼76%, STAR 72% and TopHat2 only reached 63%. In terms of runtimes, HPG Aligner, HISAT and STAR are &gt;10 times faster than TopHat2 and MapSplice (∼10 versus over 100 min; Table <a reveal-id="DSV039TB2" data-open="DSV039TB2" class="link link-reveal link-table xref-fig">2</a>). When a dataset with longer reads, obtained with 454 GS FLX+ Lifesciences (average 600 bp), was analysed (SRA accession SRP003173), the number of reads mapped by HPG Aligner reduces to ∼50%, while STAR only reached 30%, HISAT and TopHat almost none and MapSplice did not get any result after 3 days running (Table <a reveal-id="DSV039TB2" data-open="DSV039TB2" class="link link-reveal link-table xref-fig">2</a>). </p>                        <a id="87197088" scrollto-destination="87197088"></a>
<div content-id="DSV039TB2" class="table-modal"><div class="table-wrap"><div id="DSV039TB2" class="table-wrap-title" data-id="DSV039TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Real RNA-seq datasets taken from sequence read archive (SRA)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">SRA Study </th><th rowspan="2">Run accession </th><th rowspan="2">Run description </th><th colspan="3">HPG Aligner<hr /> </th><th colspan="3">HISAT<hr /> </th><th colspan="3">STAR<hr /> </th><th colspan="3">TopHat2 + Bowtie2<hr /> </th><th colspan="3">MapSplice<hr /> </th></tr><tr><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th></tr></thead><tbody><tr><td>SRP009262 </td><td>SRR364003_1 </td><td>Illumina HiSeq 81 M 100 nt (single-end) </td><td>4.53 </td><td>79.72 </td><td>11.0 </td><td>15.16 </td><td>76.10 </td><td>10.43 </td><td>5.50 </td><td>75.00 </td><td>13.9 </td><td>25.73 </td><td>63.38 </td><td>115.9 </td><td>4.63 </td><td>76.83 </td><td>139.9 </td></tr><tr><td> </td><td>SRR364003_1<br />SRR364003_2 </td><td>Illumina HiSeq 81 M 100 nt (paired-end) </td><td>4.90 </td><td>78.20 </td><td>21.6 </td><td>15.00 </td><td>77.8 </td><td>21.46 </td><td>8.00 </td><td>72.02 </td><td>22.7 </td><td>29.26 </td><td>59.9 </td><td>230.6 </td><td>5.04 </td><td>77.5 </td><td>288.1 </td></tr><tr><td>SRP003173 </td><td>SRR063344<br />SRR063345 </td><td>Roche 454 GS FLX 1.26 M ∼600 nt (single-end) </td><td>11.40 </td><td>48.04 </td><td>6.9 </td><td>99.87 </td><td>0.10 </td><td>3.45 </td><td>41.6 </td><td>30.25 </td><td>6.6 </td><td>99.97 </td><td>0.02 </td><td>520.6 </td><td>NA </td><td>NA </td><td>NA </td></tr></tbody></table></div><div content-id="DSV039TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">SRA Study </th><th rowspan="2">Run accession </th><th rowspan="2">Run description </th><th colspan="3">HPG Aligner<hr /> </th><th colspan="3">HISAT<hr /> </th><th colspan="3">STAR<hr /> </th><th colspan="3">TopHat2 + Bowtie2<hr /> </th><th colspan="3">MapSplice<hr /> </th></tr><tr><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th></tr></thead><tbody><tr><td>SRP009262 </td><td>SRR364003_1 </td><td>Illumina HiSeq 81 M 100 nt (single-end) </td><td>4.53 </td><td>79.72 </td><td>11.0 </td><td>15.16 </td><td>76.10 </td><td>10.43 </td><td>5.50 </td><td>75.00 </td><td>13.9 </td><td>25.73 </td><td>63.38 </td><td>115.9 </td><td>4.63 </td><td>76.83 </td><td>139.9 </td></tr><tr><td> </td><td>SRR364003_1<br />SRR364003_2 </td><td>Illumina HiSeq 81 M 100 nt (paired-end) </td><td>4.90 </td><td>78.20 </td><td>21.6 </td><td>15.00 </td><td>77.8 </td><td>21.46 </td><td>8.00 </td><td>72.02 </td><td>22.7 </td><td>29.26 </td><td>59.9 </td><td>230.6 </td><td>5.04 </td><td>77.5 </td><td>288.1 </td></tr><tr><td>SRP003173 </td><td>SRR063344<br />SRR063345 </td><td>Roche 454 GS FLX 1.26 M ∼600 nt (single-end) </td><td>11.40 </td><td>48.04 </td><td>6.9 </td><td>99.87 </td><td>0.10 </td><td>3.45 </td><td>41.6 </td><td>30.25 </td><td>6.6 </td><td>99.97 </td><td>0.02 </td><td>520.6 </td><td>NA </td><td>NA </td><td>NA </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>SRA SRP009262 study was sequenced with Illumina HiSeq and contains 81.6 M reads (20 M reads are used for the benchmarking) of 100 nt length. Aligners were run in single and paired-end mode. SRA SRP003173 study was sequenced with Roche 454 GS FLX and contains 1.26 M reads (1,265,460) of ∼600 nt.</p></div></div><div content-id="tblfn3"><div class="fn"><p>For each program, the table contains the following columns with the percentages of: RNM: reads not mapped; CMR: correctly mapped reads (mapped reads that cover the corresponding to known transcript positions). The last column, T, represents runtimes for producing a BAM file in min.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197088" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV039TB2" class="table-wrap-title" data-id="DSV039TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Real RNA-seq datasets taken from sequence read archive (SRA)</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">SRA Study </th><th rowspan="2">Run accession </th><th rowspan="2">Run description </th><th colspan="3">HPG Aligner<hr /> </th><th colspan="3">HISAT<hr /> </th><th colspan="3">STAR<hr /> </th><th colspan="3">TopHat2 + Bowtie2<hr /> </th><th colspan="3">MapSplice<hr /> </th></tr><tr><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th></tr></thead><tbody><tr><td>SRP009262 </td><td>SRR364003_1 </td><td>Illumina HiSeq 81 M 100 nt (single-end) </td><td>4.53 </td><td>79.72 </td><td>11.0 </td><td>15.16 </td><td>76.10 </td><td>10.43 </td><td>5.50 </td><td>75.00 </td><td>13.9 </td><td>25.73 </td><td>63.38 </td><td>115.9 </td><td>4.63 </td><td>76.83 </td><td>139.9 </td></tr><tr><td> </td><td>SRR364003_1<br />SRR364003_2 </td><td>Illumina HiSeq 81 M 100 nt (paired-end) </td><td>4.90 </td><td>78.20 </td><td>21.6 </td><td>15.00 </td><td>77.8 </td><td>21.46 </td><td>8.00 </td><td>72.02 </td><td>22.7 </td><td>29.26 </td><td>59.9 </td><td>230.6 </td><td>5.04 </td><td>77.5 </td><td>288.1 </td></tr><tr><td>SRP003173 </td><td>SRR063344<br />SRR063345 </td><td>Roche 454 GS FLX 1.26 M ∼600 nt (single-end) </td><td>11.40 </td><td>48.04 </td><td>6.9 </td><td>99.87 </td><td>0.10 </td><td>3.45 </td><td>41.6 </td><td>30.25 </td><td>6.6 </td><td>99.97 </td><td>0.02 </td><td>520.6 </td><td>NA </td><td>NA </td><td>NA </td></tr></tbody></table></div><div content-id="DSV039TB2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">SRA Study </th><th rowspan="2">Run accession </th><th rowspan="2">Run description </th><th colspan="3">HPG Aligner<hr /> </th><th colspan="3">HISAT<hr /> </th><th colspan="3">STAR<hr /> </th><th colspan="3">TopHat2 + Bowtie2<hr /> </th><th colspan="3">MapSplice<hr /> </th></tr><tr><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th><th>RNM </th><th>CMR </th><th>Time </th></tr></thead><tbody><tr><td>SRP009262 </td><td>SRR364003_1 </td><td>Illumina HiSeq 81 M 100 nt (single-end) </td><td>4.53 </td><td>79.72 </td><td>11.0 </td><td>15.16 </td><td>76.10 </td><td>10.43 </td><td>5.50 </td><td>75.00 </td><td>13.9 </td><td>25.73 </td><td>63.38 </td><td>115.9 </td><td>4.63 </td><td>76.83 </td><td>139.9 </td></tr><tr><td> </td><td>SRR364003_1<br />SRR364003_2 </td><td>Illumina HiSeq 81 M 100 nt (paired-end) </td><td>4.90 </td><td>78.20 </td><td>21.6 </td><td>15.00 </td><td>77.8 </td><td>21.46 </td><td>8.00 </td><td>72.02 </td><td>22.7 </td><td>29.26 </td><td>59.9 </td><td>230.6 </td><td>5.04 </td><td>77.5 </td><td>288.1 </td></tr><tr><td>SRP003173 </td><td>SRR063344<br />SRR063345 </td><td>Roche 454 GS FLX 1.26 M ∼600 nt (single-end) </td><td>11.40 </td><td>48.04 </td><td>6.9 </td><td>99.87 </td><td>0.10 </td><td>3.45 </td><td>41.6 </td><td>30.25 </td><td>6.6 </td><td>99.97 </td><td>0.02 </td><td>520.6 </td><td>NA </td><td>NA </td><td>NA </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn2"><div class="fn"><p>SRA SRP009262 study was sequenced with Illumina HiSeq and contains 81.6 M reads (20 M reads are used for the benchmarking) of 100 nt length. Aligners were run in single and paired-end mode. SRA SRP003173 study was sequenced with Roche 454 GS FLX and contains 1.26 M reads (1,265,460) of ∼600 nt.</p></div></div><div content-id="tblfn3"><div class="fn"><p>For each program, the table contains the following columns with the percentages of: RNM: reads not mapped; CMR: correctly mapped reads (mapped reads that cover the corresponding to known transcript positions). The last column, T, represents runtimes for producing a BAM file in min.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197088" target="_blank">View Large</a></div></div><p>Apart from read length, another important consideration is the continuous increase in the number of reads. Table <a reveal-id="DSV039TB3" data-open="DSV039TB3" class="link link-reveal link-table xref-fig">3</a> summarizes the different behaviours of the four aligners tested. HPG Aligner and HISAT clearly render much better runtimes than the rest of programs. </p>                        <a id="87197090" scrollto-destination="87197090"></a>
<div content-id="DSV039TB3" class="table-modal"><div class="table-wrap"><div id="DSV039TB3" class="table-wrap-title" data-id="DSV039TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Scalability with the number of reads</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Million reads </th><th>HPG Aligner </th><th>HISAT </th><th>STAR </th><th>TopHat2+Bowtie2 </th><th>MapSplice </th></tr></thead><tbody><tr><td>2 </td><td>0.9 </td><td>0.95 </td><td>4.5 </td><td>38.7 </td><td>11.8 </td></tr><tr><td>5 </td><td>1.9 </td><td>1.7 </td><td>5.6 </td><td>40.5 </td><td>18.65 </td></tr><tr><td>10 </td><td>3.8 </td><td>4.2 </td><td>7.75 </td><td>75.2 </td><td>36.4 </td></tr><tr><td>20 </td><td>7.6 </td><td>7.28 </td><td>13.5 </td><td>112.2 </td><td>68.5 </td></tr></tbody></table></div><div content-id="DSV039TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Million reads </th><th>HPG Aligner </th><th>HISAT </th><th>STAR </th><th>TopHat2+Bowtie2 </th><th>MapSplice </th></tr></thead><tbody><tr><td>2 </td><td>0.9 </td><td>0.95 </td><td>4.5 </td><td>38.7 </td><td>11.8 </td></tr><tr><td>5 </td><td>1.9 </td><td>1.7 </td><td>5.6 </td><td>40.5 </td><td>18.65 </td></tr><tr><td>10 </td><td>3.8 </td><td>4.2 </td><td>7.75 </td><td>75.2 </td><td>36.4 </td></tr><tr><td>20 </td><td>7.6 </td><td>7.28 </td><td>13.5 </td><td>112.2 </td><td>68.5 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p>Runtimes for producing a BAM file (in min) of the different aligners in datasets of growing size.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197090" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSV039TB3" class="table-wrap-title" data-id="DSV039TB3"><div class="label -label">Table 3.</div><div class="caption"><p>Scalability with the number of reads</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Million reads </th><th>HPG Aligner </th><th>HISAT </th><th>STAR </th><th>TopHat2+Bowtie2 </th><th>MapSplice </th></tr></thead><tbody><tr><td>2 </td><td>0.9 </td><td>0.95 </td><td>4.5 </td><td>38.7 </td><td>11.8 </td></tr><tr><td>5 </td><td>1.9 </td><td>1.7 </td><td>5.6 </td><td>40.5 </td><td>18.65 </td></tr><tr><td>10 </td><td>3.8 </td><td>4.2 </td><td>7.75 </td><td>75.2 </td><td>36.4 </td></tr><tr><td>20 </td><td>7.6 </td><td>7.28 </td><td>13.5 </td><td>112.2 </td><td>68.5 </td></tr></tbody></table></div><div content-id="DSV039TB3" class="table-modal"><table><thead name="thead" align=""><tr><th>Million reads </th><th>HPG Aligner </th><th>HISAT </th><th>STAR </th><th>TopHat2+Bowtie2 </th><th>MapSplice </th></tr></thead><tbody><tr><td>2 </td><td>0.9 </td><td>0.95 </td><td>4.5 </td><td>38.7 </td><td>11.8 </td></tr><tr><td>5 </td><td>1.9 </td><td>1.7 </td><td>5.6 </td><td>40.5 </td><td>18.65 </td></tr><tr><td>10 </td><td>3.8 </td><td>4.2 </td><td>7.75 </td><td>75.2 </td><td>36.4 </td></tr><tr><td>20 </td><td>7.6 </td><td>7.28 </td><td>13.5 </td><td>112.2 </td><td>68.5 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn4"><div class="fn"><p>Runtimes for producing a BAM file (in min) of the different aligners in datasets of growing size.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/87197090" target="_blank">View Large</a></div></div><p>We have also assessed the performance of the method in specifically mapping reads containing splice junctions. Two sets containing 5 million reads 100 bp long, spliced and non-spliced, respectively, were constructed. Runtimes obtained were very similar (220.61 s non-spliced versus 222.45 s spliced). We repeated the simulation with longer reads (250 bp) obtaining a similar result (485.56 s non-spliced versus 488.74 s spliced).</p>                            <h3 scrollto-destination=87197092 id="87197092" class="section-title" data-legacy-id=s3b>3.2. Other technical advantages</h3>
<p>In addition, the implementation presented here has additional advantages. The program can directly read gzipped, FASTQ files saving in this way both disc space and the time required for the decompression. It is also able to directly generate BAM formats, saving the SAM to BAM conversion step. From a technical point of view, the use of memory is highly efficient and can be managed by the user. For example, HPG Aligner only needs 9 GB of RAM in the analysis of the simulations shown in Fig. <a reveal-id="DSV039F2" data-open="DSV039F2" class="link link-reveal link-table xref-fig">2</a>, whereas TopHat2 and STAR require 35 GB. HPC implementation guarantees that software will have a good runtime performance in most modern and future CPUs and GPUs. HPG Aligner scales quite efficiently with the number of cores (threads), which is a desirable property in a scenario in which the deep of sequencing (number of reads sequenced) reveals as a crucial factor to properly measure differential expression.<sup><a reveal-id="DSV039C22" data-open="DSV039C22" class="link link-ref link-reveal xref-bibr">22</a></sup> Source code and development process have been opened to the community and released in GitHub; biologist and computational researchers are encouraged to use and contribute to HPG Aligner.</p>                            <h3 scrollto-destination=87197094 id="87197094" class="section-title" data-legacy-id=s3c>3.3. Program availability</h3>
<p>HPG Aligner is free and open source. Documentation and software are available at: <a class="link link-uri" href="https://github.com/opencb/hpg-aligner/wiki">https://github.com/opencb/hpg-aligner/wiki</a>.</p>                            <h2 scrollto-destination=87197096 id="87197096" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Transcriptomic studies are nowadays essential for understanding biological mechanisms.<sup><a reveal-id="DSV039C30" data-open="DSV039C30" class="link link-ref link-reveal xref-bibr">30</a></sup> For most than one decade, expression microarrays have been the dominant technology for transcriptomic analysis. However, in a quick technological transition, RNA-seq is becoming today the mainstream technology. The continuous upgrades of NGS technologies result in an increase of both throughput and read lengths. This trend requires of algorithms able to efficiently and accurately process an increasing number of reads of increasing length. Current algorithms present serious limitations in both aspects, which preclude an optimal use of RNA-seq technologies.</p><p>Mapping sequencing reads in the context of transcripts is a problem of much higher complexity than simply mapping reads onto the genomic sequence, as in the case of genome resequencing experiments. Eukaryotic transcriptomes are complex, with an average of more than nine transcripts per gene<sup><a reveal-id="DSV039C12" data-open="DSV039C12" class="link link-ref link-reveal xref-bibr">12</a></sup> with exons of 250 bp on the average that span over several hundreds of kilobases, and more new transcript isoforms are continuously being discovered.<sup><a reveal-id="DSV039C13" data-open="DSV039C13" class="link link-ref link-reveal xref-bibr">13</a></sup> Particularly, splices near the ends of reads can be especially difficult to align, given that a minimum amount of sequence is needed to confidently identify exon boundaries.<sup><a reveal-id="DSV039C20" data-open="DSV039C20" class="link link-ref link-reveal xref-bibr">20</a></sup> Accurate mapping of both short and long reads as well as a precise detection of potential splice junctions are crucial for accurate transcript isoform definition, which is essential for a proper RNA-seq data analysis.</p><p>In addition, new strategies to speed up runtimes are needed. Brute-force scaling-up strategies that rely only in the use of more powerful computers are expensive, inefficient and unsustainable. Contrarily, algorithmic solutions that take advantage of the intrinsic parallelism of NGS data and exploit the different possibilities of parallelization in the hardware used (multicore CPUs, SIMD and GPUs) offer a promising solution to the problem of the ever-growing genomic data. Actually, given the trend of increasing the number of cores in the new CPUs, scalability becomes a desirable property for any algorithm that aims to cope with the future growing sequencing datasets.</p><p>Recent comparative analysis reported that STAR<sup><a reveal-id="DSV039C18" data-open="DSV039C18" class="link link-ref link-reveal xref-bibr">18</a></sup> or MapSplice<sup><a reveal-id="DSV039C19" data-open="DSV039C19" class="link link-ref link-reveal xref-bibr">19</a></sup> performed better than other methods. Here, we have shown how the proposed approach is faster and more accurate than these ones over a wide range of read lengths and errors. Summarizing, here we present a solution for RNA-seq short-read mapping, which is optimal in terms of speed and accuracy alignment. This solution is robust to growing read lengths, it is highly sensitive to indels, and from a technological point of view, it is quite effective in memory management and efficiently scalable to many cores or processors. All these properties make of HPG Aligner, a particularly suitable solution for the analysis of current and future sequencing datasets.</p>                            <h2 scrollto-destination=87197101 id="87197101" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work is supported by grants from the Spanish Ministry of Economy and Competitiveness (BIO2014-57291-R) and co-funded with European Regional Development Funds (ERDF), AECID (D/016099/08) and from the Conselleria d'Educacio of the Valencian Community (PROMETEOII/2014/025). This work has been carried out in the context of the HPC4G initiative (<a class="link link-uri" href="http://www.hpc4g.org">http://www.hpc4g.org</a>) and the Bull-CIPF Chair for Computational Genomics. Funding to pay the Open Access publication charges for this article was provided by grant BIO2014-57291-R from the Spanish Ministry of Economy and Competitiveness (MINECO), co-funded with European Regional Development Funds (ERDF).</p>                            <h2 scrollto-destination=87197103 id="87197103" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSV039C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Garber</div>
               <div class="given-names">M.</div>
            </div>, <div class="name string-name"><div class="surname">Grabherr</div> <div class="given-names">M.G.</div></div>, <div class="name string-name"><div class="surname">Guttman</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Computational methods for transcriptome annotation and quantification using RNA-seq</div>, <div class="source">Nat. Methods</div> , <div class="volume">8</div>, <div class="fpage">469</div>–<div class="lpage">77</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Computational%20methods%20for%20transcriptome%20annotation%20and%20quantification%20using%20RNA-seq&amp;author=M.%20Garber&amp;author=M.G.%20Grabherr&amp;author=M.%20Guttman&amp;author=C.%20Trapnell&amp;publication_year=2011&amp;journal=Nat.%20Methods&amp;volume=8&amp;pages=469-77" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1613" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1613" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1613"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21623353" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Marioni</div>
               <div class="given-names">J.C.</div>
            </div>, <div class="name string-name"><div class="surname">Mason</div> <div class="given-names">C.E.</div></div>, <div class="name string-name"><div class="surname">Mane</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Gilad</div> <div class="given-names">Y.</div></div></div>
          <div class="year">2008</div>, <div class="article-title">RNA-seq: an assessment of technical reproducibility and comparison with gene expression arrays</div>, <div class="source">Genome Res.</div> , <div class="volume">18</div>, <div class="fpage">1509</div>–<div class="lpage">17</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-seq%3A%20an%20assessment%20of%20technical%20reproducibility%20and%20comparison%20with%20gene%20expression%20arrays&amp;author=J.C.%20Marioni&amp;author=C.E.%20Mason&amp;author=S.M.%20Mane&amp;author=M.%20Stephens&amp;author=Y.%20Gilad&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1509-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.079558.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.079558.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.079558.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18550803" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Homer</div> <div class="given-names">N.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">A survey of sequence alignment algorithms for next-generation sequencing</div>, <div class="source">Brief Bioinform.</div> , <div class="volume">11</div>, <div class="fpage">473</div>–<div class="lpage">83</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20survey%20of%20sequence%20alignment%20algorithms%20for%20next-generation%20sequencing&amp;author=H.%20Li&amp;author=N.%20Homer&amp;publication_year=2010&amp;journal=Brief%20Bioinform.&amp;volume=11&amp;pages=473-83" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bib/bbq015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbib%2Fbbq015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbib%2Fbbq015"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20460430" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pop</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Ultrafast and memory-efficient alignment of short DNA sequences to the human genome</div>, <div class="source">Genome Biol.</div> , <div class="volume">10</div>, <div class="fpage">R25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ultrafast%20and%20memory-efficient%20alignment%20of%20short%20DNA%20sequences%20to%20the%20human%20genome&amp;author=B.%20Langmead&amp;author=C.%20Trapnell&amp;author=M.%20Pop&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Genome%20Biol.&amp;volume=10&amp;pages=R25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2009-10-3-r25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2009-10-3-r25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2009-10-3-r25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19261174" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">R.</div>
            </div>, <div class="name string-name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>et al.  </div>. <div class="year">2009</div>, <div class="article-title">SOAP2: an improved ultrafast tool for short read alignment</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1966</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAP2%3A%20an%20improved%20ultrafast%20tool%20for%20short%20read%20alignment&amp;author=R.%20Li&amp;author=C.%20Yu&amp;author=Y.%20Li&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1966-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp336" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp336" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp336"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19497933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Homer</div>
               <div class="given-names">N.</div>
            </div>, <div class="name string-name"><div class="surname">Merriman</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Nelson</div> <div class="given-names">S.F.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">BFAST: an alignment tool for large scale genome resequencing</div>, <div class="source">PLoS ONE</div> , <div class="volume">4</div>, <div class="fpage">e7767</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=BFAST%3A%20an%20alignment%20tool%20for%20large%20scale%20genome%20resequencing&amp;author=N.%20Homer&amp;author=B.%20Merriman&amp;author=S.F.%20Nelson&amp;publication_year=2009&amp;journal=PLoS%20ONE&amp;volume=4&amp;pages=e7767" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0007767" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0007767" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0007767"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19907642" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Li</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Fast and accurate long-read alignment with Burrows-Wheeler transform</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">589</div>–<div class="lpage">95</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20long-read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=589-95" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp698" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp698" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp698"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20080505" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Marco-Sola</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Sammeth</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Guigo</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Ribeca</div> <div class="given-names">P.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">The GEM mapper: fast, accurate and versatile alignment by filtration</div>, <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">1185</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20GEM%20mapper%3A%20fast%2C%20accurate%20and%20versatile%20alignment%20by%20filtration&amp;author=S.%20Marco-Sola&amp;author=M.%20Sammeth&amp;author=R.%20Guigo&amp;author=P.%20Ribeca&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=1185-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2221" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2221" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2221"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23103880" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Langmead</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2012</div>, <div class="article-title">Fast gapped-read alignment with Bowtie 2</div>, <div class="source">Nat. Methods</div> , <div class="volume">9</div>, <div class="fpage">357</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20gapped-read%20alignment%20with%20Bowtie%202&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2012&amp;journal=Nat.%20Methods&amp;volume=9&amp;pages=357-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1923" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1923" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1923"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22388286" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tarraga</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Arnau</div> <div class="given-names">V.</div></div>, <div class="name string-name"><div class="surname">Martinez</div> <div class="given-names">H.</div></div>et al.  </div>. <div class="year">2014</div>, <div class="article-title">Acceleration of short and long DNA read mapping without loss of accuracy using suffix array</div>, <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">3396</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Acceleration%20of%20short%20and%20long%20DNA%20read%20mapping%20without%20loss%20of%20accuracy%20using%20suffix%20array&amp;author=J.%20Tarraga&amp;author=V.%20Arnau&amp;author=H.%20Martinez&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=3396-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu553" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu553" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu553"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25143289" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Birney</div>
               <div class="given-names">E.</div>
            </div>, <div class="name string-name"><div class="surname">Stamatoyannopoulos</div> <div class="given-names">J.A.</div></div>, <div class="name string-name"><div class="surname">Dutta</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2007</div>, <div class="article-title">Identification and analysis of functional elements in 1% of the human genome by the ENCODE pilot project</div>, <div class="source">Nature</div> , <div class="volume">447</div>, <div class="fpage">799</div>–<div class="lpage">816</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20analysis%20of%20functional%20elements%20in%201%25%20of%20the%20human%20genome%20by%20the%20ENCODE%20pilot%20project&amp;author=E.%20Birney&amp;author=J.A.%20Stamatoyannopoulos&amp;author=A.%20Dutta&amp;publication_year=2007&amp;journal=Nature&amp;volume=447&amp;pages=799-816" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05874" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05874" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05874"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17571346" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Ameur</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Wetterbom</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Feuk</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Gyllensten</div> <div class="given-names">U.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Global and unbiased detection of splice junctions from RNA-seq data</div>, <div class="source">Genome Biol.</div> , <div class="volume">11</div>, <div class="fpage">R34</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Global%20and%20unbiased%20detection%20of%20splice%20junctions%20from%20RNA-seq%20data&amp;author=A.%20Ameur&amp;author=A.%20Wetterbom&amp;author=L.%20Feuk&amp;author=U.%20Gyllensten&amp;publication_year=2010&amp;journal=Genome%20Biol.&amp;volume=11&amp;pages=R34" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2010-11-3-r34" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2010-11-3-r34" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2010-11-3-r34"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20236510" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Trapnell</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Pachter</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2009</div>, <div class="article-title">TopHat: discovering splice junctions with RNA-Seq</div>, <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1105</div>–<div class="lpage">11</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat%3A%20discovering%20splice%20junctions%20with%20RNA-Seq&amp;author=C.%20Trapnell&amp;author=L.%20Pachter&amp;author=S.L.%20Salzberg&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1105-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp120" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp120" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp120"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19289445" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wu</div>
               <div class="given-names">T.D.</div>
            </div>, <div class="name string-name"><div class="surname">Nacu</div> <div class="given-names">S.</div></div></div>
          <div class="year">2010</div>, <div class="article-title">Fast and SNP-tolerant detection of complex variants and splicing in short reads</div>, <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">873</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20SNP-tolerant%20detection%20of%20complex%20variants%20and%20splicing%20in%20short%20reads&amp;author=T.D.%20Wu&amp;author=S.%20Nacu&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=873-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq057" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq057" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq057"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20147302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kim</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Trapnell</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Pimentel</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Kelley</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">TopHat2: accurate alignment of transcriptomes in the presence of insertions, deletions and gene fusions</div>, <div class="source">Genome Biol.</div> , <div class="volume">14</div>, <div class="fpage">R36</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TopHat2%3A%20accurate%20alignment%20of%20transcriptomes%20in%20the%20presence%20of%20insertions%2C%20deletions%20and%20gene%20fusions&amp;author=D.%20Kim&amp;author=G.%20Pertea&amp;author=C.%20Trapnell&amp;author=H.%20Pimentel&amp;author=R.%20Kelley&amp;author=S.L.%20Salzberg&amp;publication_year=2013&amp;journal=Genome%20Biol.&amp;volume=14&amp;pages=R36" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/gb-2013-14-4-r36" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fgb-2013-14-4-r36" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fgb-2013-14-4-r36"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23618408" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Grant</div>
               <div class="given-names">G.R.</div>
            </div>, <div class="name string-name"><div class="surname">Farkas</div> <div class="given-names">M.H.</div></div>, <div class="name string-name"><div class="surname">Pizarro</div> <div class="given-names">A.D.</div></div>et al.  </div>. <div class="year">2011</div>, <div class="article-title">Comparative analysis of RNA-Seq alignment algorithms and the RNA-Seq unified mapper (RUM)</div>, <div class="source">Bioinformatics</div> , <div class="volume">27</div>, <div class="fpage">2518</div>–<div class="lpage">28</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20analysis%20of%20RNA-Seq%20alignment%20algorithms%20and%20the%20RNA-Seq%20unified%20mapper%20(RUM)&amp;author=G.R.%20Grant&amp;author=M.H.%20Farkas&amp;author=A.D.%20Pizarro&amp;publication_year=2011&amp;journal=Bioinformatics&amp;volume=27&amp;pages=2518-28" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btr427" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtr427" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtr427"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21775302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Dobin</div>
               <div class="given-names">A.</div>
            </div>, <div class="name string-name"><div class="surname">Davis</div> <div class="given-names">C.A.</div></div>, <div class="name string-name"><div class="surname">Schlesinger</div> <div class="given-names">F.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">STAR: ultrafast universal RNA-seq aligner</div>, <div class="source">Bioinformatics</div> , <div class="volume">29</div>, <div class="fpage">15</div>–<div class="lpage">21</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=STAR%3A%20ultrafast%20universal%20RNA-seq%20aligner&amp;author=A.%20Dobin&amp;author=C.A.%20Davis&amp;author=F.%20Schlesinger&amp;publication_year=2013&amp;journal=Bioinformatics&amp;volume=29&amp;pages=15-21" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bts635" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbts635" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbts635"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23104886" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Wang</div>
               <div class="given-names">K.</div>
            </div>, <div class="name string-name"><div class="surname">Singh</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Zeng</div> <div class="given-names">Z.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">MapSplice: accurate mapping of RNA-seq reads for splice junction discovery</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">38</div>, <div class="fpage">e178</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MapSplice%3A%20accurate%20mapping%20of%20RNA-seq%20reads%20for%20splice%20junction%20discovery&amp;author=K.%20Wang&amp;author=D.%20Singh&amp;author=Z.%20Zeng&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res.&amp;volume=38&amp;pages=e178" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq622" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq622" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq622"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20802226" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Engstrom</div>
               <div class="given-names">P.G.</div>
            </div>, <div class="name string-name"><div class="surname">Steijger</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Sipos</div> <div class="given-names">B.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Systematic evaluation of spliced alignment programs for RNA-seq data</div>, <div class="source">Nat. Methods</div> , <div class="volume">10</div>, <div class="fpage">1185</div>–<div class="lpage">91</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Systematic%20evaluation%20of%20spliced%20alignment%20programs%20for%20RNA-seq%20data&amp;author=P.G.%20Engstrom&amp;author=T.%20Steijger&amp;author=B.%20Sipos&amp;publication_year=2013&amp;journal=Nat.%20Methods&amp;volume=10&amp;pages=1185-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.2722" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.2722" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.2722"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24185836" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Kim</div>
               <div class="given-names">D.</div>
            </div>, <div class="name string-name"><div class="surname">Langmead</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Salzberg</div> <div class="given-names">S.L.</div></div></div>
          <div class="year">2015</div>, <div class="article-title">HISAT: a fast spliced aligner with low memory requirements</div>, <div class="source">Nat. Methods</div> , <div class="volume">12</div>, <div class="fpage">357</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=HISAT%3A%20a%20fast%20spliced%20aligner%20with%20low%20memory%20requirements&amp;author=D.%20Kim&amp;author=B.%20Langmead&amp;author=S.L.%20Salzberg&amp;publication_year=2015&amp;journal=Nat.%20Methods&amp;volume=12&amp;pages=357-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.3317" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.3317" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.3317"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25751142" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Tarazona</div>
               <div class="given-names">S.</div>
            </div>, <div class="name string-name"><div class="surname">Garcia-Alcalde</div> <div class="given-names">F.</div></div>, <div class="name string-name"><div class="surname">Dopazo</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ferrer</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Conesa</div> <div class="given-names">A.</div></div></div>
          <div class="year">2011</div>, <div class="article-title">Differential expression in RNA-seq: a matter of depth</div>, <div class="source">Genome Res.</div> , <div class="volume">21</div>, <div class="fpage">2213</div>–<div class="lpage">23</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20expression%20in%20RNA-seq%3A%20a%20matter%20of%20depth&amp;author=S.%20Tarazona&amp;author=F.%20Garcia-Alcalde&amp;author=J.%20Dopazo&amp;author=A.%20Ferrer&amp;author=A.%20Conesa&amp;publication_year=2011&amp;journal=Genome%20Res.&amp;volume=21&amp;pages=2213-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.124321.111" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.124321.111" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.124321.111"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21903743" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Trapnell</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Roberts</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Goff</div> <div class="given-names">L.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Differential gene and transcript expression analysis of RNA-seq experiments with TopHat and Cufflinks</div>, <div class="source">Nat. Protoc.</div> , <div class="volume">7</div>, <div class="fpage">562</div>–<div class="lpage">78</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Differential%20gene%20and%20transcript%20expression%20analysis%20of%20RNA-seq%20experiments%20with%20TopHat%20and%20Cufflinks&amp;author=C.%20Trapnell&amp;author=A.%20Roberts&amp;author=L.%20Goff&amp;publication_year=2012&amp;journal=Nat.%20Protoc.&amp;volume=7&amp;pages=562-78" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nprot.2012.016" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnprot.2012.016" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnprot.2012.016"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22383036" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Trapnell</div>
               <div class="given-names">C.</div>
            </div>, <div class="name string-name"><div class="surname">Williams</div> <div class="given-names">B.A.</div></div>, <div class="name string-name"><div class="surname">Pertea</div> <div class="given-names">G.</div></div>et al.  </div>. <div class="year">2010</div>, <div class="article-title">Transcript assembly and quantification by RNA-Seq reveals unannotated transcripts and isoform switching during cell differentiation</div>, <div class="source">Nat. Biotechnol.</div> , <div class="volume">28</div>, <div class="fpage">511</div>–<div class="lpage">5</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcript%20assembly%20and%20quantification%20by%20RNA-Seq%20reveals%20unannotated%20transcripts%20and%20isoform%20switching%20during%20cell%20differentiation&amp;author=C.%20Trapnell&amp;author=B.A.%20Williams&amp;author=G.%20Pertea&amp;publication_year=2010&amp;journal=Nat.%20Biotechnol.&amp;volume=28&amp;pages=511-5" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.1621" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.1621" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.1621"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20436464" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Altschul</div>
               <div class="given-names">S.F.</div>
            </div>, <div class="name string-name"><div class="surname">Gish</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Myers</div> <div class="given-names">E.W.</div></div>, <div class="name string-name"><div class="surname">Lipman</div> <div class="given-names">D.J.</div></div></div>
          <div class="year">1990</div>, <div class="article-title">Basic local alignment search tool</div>, <div class="source">J. Mol. Biol.</div> , <div class="volume">215</div>, <div class="fpage">403</div>–<div class="lpage">10</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Basic%20local%20alignment%20search%20tool&amp;author=S.F.%20Altschul&amp;author=W.%20Gish&amp;author=W.%20Miller&amp;author=E.W.%20Myers&amp;author=D.J.%20Lipman&amp;publication_year=1990&amp;journal=J.%20Mol.%20Biol.&amp;volume=215&amp;pages=403-10" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S0022-2836(05)80360-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS0022-2836(05)80360-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS0022-2836(05)80360-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/2231712" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Salavert Torres</div>
               <div class="given-names">J.</div>
            </div>, <div class="name string-name"><div class="surname">Blanquer Espert</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Domínguez</div> <div class="given-names">A.</div></div>et al.  </div>. <div class="year">2012</div>, <div class="article-title">Using GPUs for the exact alignment of short-read genetic sequences by means of the Burrows-Wheeler transform</div>, <div class="source">IEEE/ACM Trans. Comput. Biol. Bioinform.</div> , <div class="volume">9</div>, <div class="fpage">1245</div>–<div class="lpage">56</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Using%20GPUs%20for%20the%20exact%20alignment%20of%20short-read%20genetic%20sequences%20by%20means%20of%20the%20Burrows-Wheeler%20transform&amp;author=J.%20Salavert%20Torres&amp;author=I.%20Blanquer%20Espert&amp;author=A.%20Dom%C3%ADnguez&amp;publication_year=2012&amp;journal=IEEE%2FACM%20Trans.%20Comput.%20Biol.%20Bioinform.&amp;volume=9&amp;pages=1245-56" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1109/TCBB.2012.49" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1109%2FTCBB.2012.49" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1109%2FTCBB.2012.49"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22450827" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Alpern</div>
               <div class="given-names">B.</div>
            </div>, <div class="name string-name"><div class="surname">Carter</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Gatlin</div> <div class="given-names">K.</div></div></div>
          <div class="year">1995</div>, <div class="article-title">Microparallelism and high-performance protein matching. In: <em>Proceedings of the ACM/IEEE Supercomputing Conference, San Diego, California</em>, December 8, 1995</div>, p. <div class="fpage">24</div>.</div></div></div></div><div content-id="DSV039C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Martinez</div>
               <div class="given-names">H.</div>
            </div>, <div class="name string-name"><div class="surname">Tarraga</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Medina</div> <div class="given-names">I.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">EuroMPI ‘13 Proceedings of the 20th European MPI Users’ Group Meeting. In: <em>A Dynamic Pipeline for RNA Sequencing on Multicore Processors, Madrid, Spain</em>, September 15–18, 2013</div>, pp. <div class="fpage">235</div>–<div class="lpage">40</div>.</div></div></div></div><div content-id="DSV039C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Flicek</div>
               <div class="given-names">P.</div>
            </div>, <div class="name string-name"><div class="surname">Ahmed</div> <div class="given-names">I.</div></div>, <div class="name string-name"><div class="surname">Amode</div> <div class="given-names">M.R.</div></div>et al.  </div>. <div class="year">2013</div>, <div class="article-title">Ensembl 2013</div>, <div class="source">Nucleic Acids Res.</div> , <div class="volume">41</div>, <div class="fpage">D48</div>–<div class="lpage">55</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ensembl%202013&amp;author=P.%20Flicek&amp;author=I.%20Ahmed&amp;author=M.R.%20Amode&amp;publication_year=2013&amp;journal=Nucleic%20Acids%20Res.&amp;volume=41&amp;pages=D48-55" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gks1236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgks1236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgks1236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23203987" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSV039C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSV039C30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation">
          <div class="person-group">
            <div class="name string-name">
              <div class="surname">Soon</div>
               <div class="given-names">W.W.</div>
            </div>, <div class="name string-name"><div class="surname">Hariharan</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Snyder</div> <div class="given-names">M.P.</div></div></div>
          <div class="year">2013</div>, <div class="article-title">High-throughput sequencing for biology and medicine</div>, <div class="source">Mol. Syst. Biol.</div> , <div class="volume">9</div>, <div class="fpage">640</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High-throughput%20sequencing%20for%20biology%20and%20medicine&amp;author=W.W.%20Soon&amp;author=M.%20Hariharan&amp;author=M.P.%20Snyder&amp;publication_year=2013&amp;journal=Mol.%20Syst.%20Biol.&amp;volume=9&amp;pages=640" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/msb.2012.61" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fmsb.2012.61" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fmsb.2012.61"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23340846" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2016. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=1745299&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/23/2/10.1093_dnares_dsv039/2/dsv039_Supplementary_Data.zip?Expires=2147483647&amp;Signature=O5zhXMbjdL9MfWtRHyZZoQtjwf5zaVXvYWYJtmT4rJBXhDGjpUt1cPrA598ZLSZ5pBQ6P6Lva2hSCiTy5Iub-1KISygLW8FTa3n-3Q0Hx-c6T6cnlY9n58wYv4xALGU6Fhnc4tm5sQRRSXH6LHC6xvCJn3GOzr9jmAz9AwCyLzaX14zCOGlfVslVT1AUEsVTh5IqKN1mCI0yqmSXKsVMgrqYyU95lTF6GY4a9BEbPphosWJ6-xflQndXzfUiDMr26ltDmvR6B2SLrM3AsMrURh-wZzNo2ZJj52bNBe23SttTfMf~nmqX1hQsgV5hvy8vbhYo4oeFS5mnqlhH9NS8ZA__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Data</div></a> - zip file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">1,436</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000375088800002&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">5</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsv039"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="1745299"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000375088800002&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f23%2f2%2f93%2f1745299">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000375088800002&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (5)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F23%2F2%2F93%2F1745299" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/1745299'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Highly sensitive and ultrafast read mapping for RNA-seq analysis" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsv039");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f23%2f2%2f93%2f1745299">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

