

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/24/6/585/3896153","@type":"ScholarlyArticle","name":"Sequencing and phasing cancer mutations in lung cancers using a long-read portable sequencer","datePublished":"2017-06-27","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/24/6","@type":"PublicationIssue","issueNumber":"6","datePublished":"2017-12-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dsx027","keywords":["MinION","lung cancer cell lines","cancer mutations","phasing"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/24/6/585/3896153","author":[{"name":"Suzuki, Ayako","affiliation":"Division of Translational Genomics, Exploratory Oncology Research and Clinical Trial Center, National Cancer Center, Kashiwa, Chiba, Japan","@type":"Person"},{"name":"Suzuki, Mizuto","affiliation":"Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan","@type":"Person"},{"name":"Mizushima-Sugano, Junko","affiliation":"Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan  Department of Chemistry and Life Science, Kogakuin University, Nishi-Shinjuku, Shinjuku-Ku, Tokyo, Japan","@type":"Person"},{"name":"Frith, Martin C","affiliation":"Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan  Computational Biology Research Center, The National Institute for Advanced Industrial Science and Technology, Aomi, Koto-Ku, Tokyo, Japan","@type":"Person"},{"name":"Makałowski, Wojciech","affiliation":"Institute of Bioinformatics, Faculty of Medicine, University of Muenster, Munster, Germany","@type":"Person"},{"name":"Kohno, Takashi","affiliation":"Division of Genome Biology, National Cancer Center Research Institute, Tsukiji, Chuo-Ku, Tokyo, Japan","@type":"Person"},{"name":"Sugano, Sumio","affiliation":"Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan","@type":"Person"},{"name":"Tsuchihara, Katsuya","affiliation":"Division of Translational Genomics, Exploratory Oncology Research and Clinical Trial Center, National Cancer Center, Kashiwa, Chiba, Japan","@type":"Person"},{"name":"Suzuki, Yutaka","affiliation":"Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan","@type":"Person"}],"description":"Abstract.  Here, we employed cDNA amplicon sequencing using a long-read portable sequencer, MinION, to characterize various types of mutations in cancer-related","pageStart":"585","pageEnd":"596","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Sequencing and phasing cancer mutations in lung cancers using a long-read portable sequencer","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Sequencing and phasing cancer mutations in lung cancers using a long-read portable sequencer" />
    <meta property="og:description" content="Abstract.  Here, we employed cDNA amplicon sequencing using a long-read portable sequencer, MinION, to characterize various types of mutations in cancer-related" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/24/6/585/3896153" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2017-06-27" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Suzuki, Ayako" /><meta name="citation_author_institution" content="Division of Translational Genomics, Exploratory Oncology Research and Clinical Trial Center, National Cancer Center, Kashiwa, Chiba, Japan" /><meta name="citation_author" content="Suzuki, Mizuto" /><meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan" /><meta name="citation_author" content="Mizushima-Sugano, Junko" /><meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan" /><meta name="citation_author_institution" content="Department of Chemistry and Life Science, Kogakuin University, Nishi-Shinjuku, Shinjuku-Ku, Tokyo, Japan" /><meta name="citation_author" content="Frith, Martin C" /><meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan" /><meta name="citation_author_institution" content="Computational Biology Research Center, The National Institute for Advanced Industrial Science and Technology, Aomi, Koto-Ku, Tokyo, Japan" /><meta name="citation_author" content="Makałowski, Wojciech" /><meta name="citation_author_institution" content="Institute of Bioinformatics, Faculty of Medicine, University of Muenster, Munster, Germany" /><meta name="citation_author" content="Kohno, Takashi" /><meta name="citation_author_institution" content="Division of Genome Biology, National Cancer Center Research Institute, Tsukiji, Chuo-Ku, Tokyo, Japan" /><meta name="citation_author" content="Sugano, Sumio" /><meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan" /><meta name="citation_author" content="Tsuchihara, Katsuya" /><meta name="citation_author_institution" content="Division of Translational Genomics, Exploratory Oncology Research and Clinical Trial Center, National Cancer Center, Kashiwa, Chiba, Japan" /><meta name="citation_author" content="Suzuki, Yutaka" /><meta name="citation_author_institution" content="Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan" /><meta name="citation_title" content="Sequencing and phasing cancer mutations in lung cancers using a long-read portable sequencer" /><meta name="citation_firstpage" content="585" /><meta name="citation_lastpage" content="596" /><meta name="citation_doi" content="10.1093/dnares/dsx027" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="24" /><meta name="citation_issue" content="6" /><meta name="citation_publication_date" content="2017/12/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=The Cancer Genome Atlas Pan-Cancer analysis project; citation_author=The Cancer Genome Atlas Research Network;  citation_journal_title=Nat. Genet;  citation_year=2013;  citation_volume=45;  citation_pages=1113-20; " /><meta name="citation_reference" content="citation_title=International network of cancer genome projects; citation_author=International Cancer Genome Consortium;  citation_journal_title=Nature;  citation_year=2010;  citation_volume=464;  citation_pages=993-8; " /><meta name="citation_reference" content="citation_title=Epidermal growth factor receptor mutations in lung cancer;  citation_journal_title=Nat. Rev. Cancer;  citation_year=2007;  citation_volume=7;  citation_pages=169-81; " /><meta name="citation_reference" content="citation_title=Identification of the transforming EML4-ALK fusion gene in non-small-cell lung cancer;  citation_journal_title=Nature;  citation_year=2007;  citation_volume=448;  citation_pages=561-6; " /><meta name="citation_reference" content="citation_title=Anaplastic lymphoma kinase inhibition in non-small-cell lung cancer;  citation_journal_title=N. Engl J. Med;  citation_year=2010;  citation_volume=363;  citation_pages=1693-1703; " /><meta name="citation_reference" content="citation_title=Improved data analysis for the MinION nanopore sequencer;  citation_journal_title=Nat. Methods;  citation_year=2015;  citation_volume=12;  citation_pages=351-6; " /><meta name="citation_reference" content="citation_title=MinION nanopore sequencing identifies the position and structure of a bacterial antibiotic resistance island;  citation_journal_title=Nat. Biotechnol;  citation_year=2015;  citation_volume=33;  citation_pages=296-300; " /><meta name="citation_reference" content="citation_title=Rapid draft sequencing and real-time nanopore sequencing in a hospital outbreak of Salmonella;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=114" /><meta name="citation_reference" content="citation_title=Real-time, portable genome sequencing for Ebola surveillance;  citation_journal_title=Nature;  citation_year=2016;  citation_volume=530;  citation_pages=228-32; " /><meta name="citation_reference" content="citation_title=Nanopore sequencing detects structural variants in cancer;  citation_journal_title=Cancer Biol. Ther;  citation_year=2016;  citation_volume=17;  citation_pages=246-53; " /><meta name="citation_reference" content="citation_title=TP53 gene mutation analysis in chronic lymphocytic leukemia by nanopore MinION sequencing;  citation_journal_title=Diagn. Pathol;  citation_year=2016;  citation_volume=11;  citation_pages=96" /><meta name="citation_reference" content="citation_title=Aberrant transcriptional regulations in cancers: genome, transcriptome and epigenome analysis of lung adenocarcinoma cell lines;  citation_journal_title=Nucleic Acids Res;  citation_year=2014;  citation_volume=42;  citation_pages=13557-72; " /><meta name="citation_reference" content="citation_title=KIF5B-RET fusions in lung adenocarcinoma;  citation_journal_title=Nat. Med;  citation_year=2012;  citation_volume=18;  citation_pages=375-7; " /><meta name="citation_reference" content="citation_title=Primer3Plus, an enhanced web interface to Primer3;  citation_journal_title=Nucleic Acids Res;  citation_year=2007;  citation_volume=35;  citation_pages=W71-4; " /><meta name="citation_reference" content="citation_title=Poretools: a toolkit for analyzing nanopore sequence data;  citation_journal_title=Bioinformatics;  citation_year=2014;  citation_volume=30;  citation_pages=3399-401; " /><meta name="citation_reference" content="citation_title=Adaptive seeds tame genomic sequence comparison;  citation_journal_title=Genome Res;  citation_year=2011;  citation_volume=21;  citation_pages=487-93; " /><meta name="citation_reference" content="citation_title=Fast and accurate short read alignment with Burrows-Wheeler transform;  citation_journal_title=Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1754-60; " /><meta name="citation_reference" content="citation_title=Training alignment parameters for arbitrary sequencers with LAST-TRAIN;  citation_journal_title=Bioinformatics;  citation_year=2016;  citation_volume=33;  citation_pages=926-8; " /><meta name="citation_reference" content="citation_title=Incorporating sequence quality data into alignment improves DNA read mapping;  citation_journal_title=Nucleic Acids Res;  citation_year=2010;  citation_volume=38;  citation_pages=e100" /><meta name="citation_reference" content="citation_title=The UCSC Genome Browser database: 2016 update;  citation_journal_title=Nucleic Acids Res;  citation_year=2016;  citation_volume=44;  citation_pages=D717-25; " /><meta name="citation_reference" content="citation_title=Fast and SNP-tolerant detection of complex variants and splicing in short reads;  citation_journal_title=Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=873-81; " /><meta name="citation_reference" content="citation_title=Split-alignment of genomes finds orthologies more accurately;  citation_journal_title=Genome Biol;  citation_year=2015;  citation_volume=16;  citation_pages=106" /><meta name="citation_reference" content="citation_title=Identification of CCDC6-RET fusion in the human lung adenocarcinoma cell line, LC-2/ad;  citation_journal_title=J. Thorac. Oncol;  citation_year=2012;  citation_volume=7;  citation_pages=1872-6; " /><meta name="citation_reference" content="citation_title=Identification of a lung adenocarcinoma cell line with CCDC6-RET fusion gene and the effect of RET inhibitors in vitro and in vivo;  citation_journal_title=Cancer Sci;  citation_year=2013;  citation_volume=104;  citation_pages=896-903; " /><meta name="citation_reference" content="citation_title=Discovery of ALK-PTPN3 gene fusion from human non-small cell lung carcinoma cell line using next generation RNA sequencing;  citation_journal_title=Genes Chromosomes Cancer;  citation_year=2012;  citation_volume=51;  citation_pages=590-7; " /><meta name="citation_reference" content="citation_title=Somatic mutations affect key pathways in lung adenocarcinoma;  citation_journal_title=Nature;  citation_year=2008;  citation_volume=455;  citation_pages=1069-75; " /><meta name="citation_reference" content="citation_title=Mapping the hallmarks of lung adenocarcinoma with massively parallel sequencing;  citation_journal_title=Cell;  citation_year=2012;  citation_volume=150;  citation_pages=1107-20; " /><meta name="citation_reference" content="citation_title=The transcriptional landscape and mutational profile of lung adenocarcinoma;  citation_journal_title=Genome Res;  citation_year=2012;  citation_volume=22;  citation_pages=2109-19; " /><meta name="citation_reference" content="citation_title=Identification and characterization of cancer mutations in Japanese lung adenocarcinoma without sequencing of normal tissue counterparts;  citation_journal_title=PLoS One;  citation_year=2013;  citation_volume=8;  citation_pages=e73484" /><meta name="citation_reference" content="citation_title=Comprehensive molecular profiling of lung adenocarcinoma; citation_author=The Cancer Genome Atlas Research Network;  citation_journal_title=Nature;  citation_year=2014;  citation_volume=511;  citation_pages=543-50; " /><meta name="citation_reference" content="citation_title=Acquired resistance of lung adenocarcinomas to gefitinib or erlotinib is associated with a second mutation in the EGFR kinase domain;  citation_journal_title=PLoS Med;  citation_year=2005;  citation_volume=2;  citation_pages=e73" /><meta name="citation_reference" content="citation_title=Acquired EGFR C797S mutation mediates resistance to AZD9291 in non-small cell lung cancer harboring EGFR T790M;  citation_journal_title=Nat. Med;  citation_year=2015;  citation_volume=21;  citation_pages=560-2; " /><meta name="citation_reference" content="citation_title=A prospective PCR-based screening for the EML4-ALK oncogene in non-small cell lung cancer;  citation_journal_title=Clin. Cancer Res;  citation_year=2012;  citation_volume=18;  citation_pages=5682-9; " /><meta name="citation_reference" content="citation_title=Comparison of gefitinib, erlotinib and afatinib in non-small cell lung cancer: A meta-analysis;  citation_journal_title=Int. J. Cancer;  citation_year=2017;  citation_volume=140;  citation_pages=2805-19; " /><meta name="citation_reference" content="citation_title=Two novel ALK mutations mediate acquired resistance to the next-generation ALK inhibitor alectinib;  citation_journal_title=Clin Cancer Res;  citation_year=2014;  citation_volume=20;  citation_pages=5686-96; " /><meta name="citation_reference" content="citation_title=RET fusion gene: translation to personalized lung cancer therapy;  citation_journal_title=Cancer Sci;  citation_year=2013;  citation_volume=104;  citation_pages=1396-1400; " /><meta name="citation_reference" content="citation_title=Acquired resistance to TKIs in solid tumours: learning from lung cancer;  citation_journal_title=Nat. Rev. Clin. Oncol;  citation_year=2014;  citation_volume=11;  citation_pages=473-81; " /><meta name="citation_reference" content="citation_title=The allelic context of the C797S mutation acquired upon treatment with third-generation EGFR inhibitors impacts sensitivity to subsequent treatment strategies;  citation_journal_title=Clin. Cancer Res;  citation_year=2015;  citation_volume=21;  citation_pages=3924-33; " /><meta name="citation_reference" content="citation_title=Resensitization to crizotinib by the lorlatinib ALK resistance mutation L1198F;  citation_journal_title=N. Engl J. Med;  citation_year=2016;  citation_volume=374;  citation_pages=54-61; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/24/6/585/22295764/dsx027.pdf" /><meta name="description" content="Abstract.  Here, we employed cDNA amplicon sequencing using a long-read portable sequencer, MinION, to characterize various types of mutations in cancer-related" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/24/6/585/3896153" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/24/6/585/3896153" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/24/6/585/3896153");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dsx027';
    </script>
<title>
	Sequencing and phasing cancer mutations in lung cancers using a long-read portable sequencer | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./3896153" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BbZoPG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJTdXp1a2ksIEF5YWtvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgVHJhbnNsYXRpb25hbCBHZW5vbWljcywgRXhwbG9yYXRvcnkgT25jb2xvZ3kgUmVzZWFyY2ggYW5kIENsaW5pY2FsIFRyaWFsIENlbnRlciwgTmF0aW9uYWwgQ2FuY2VyIENlbnRlciwgS2FzaGl3YSwgQ2hpYmEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3V6dWtpLCBNaXp1dG8iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENvbXB1dGF0aW9uYWwgQmlvbG9neSBhbmQgTWVkaWNhbCBTY2llbmNlcywgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgS2FzaGl3YSwgQ2hpYmEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iTWl6dXNoaW1hLVN1Z2FubywgSnVua28iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENvbXB1dGF0aW9uYWwgQmlvbG9neSBhbmQgTWVkaWNhbCBTY2llbmNlcywgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgS2FzaGl3YSwgQ2hpYmEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGVwYXJ0bWVudCBvZiBDaGVtaXN0cnkgYW5kIExpZmUgU2NpZW5jZSwgS29nYWt1aW4gVW5pdmVyc2l0eSwgTmlzaGktU2hpbmp1a3UsIFNoaW5qdWt1LUt1LCBUb2t5bywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJGcml0aCwgTWFydGluIEMiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENvbXB1dGF0aW9uYWwgQmlvbG9neSBhbmQgTWVkaWNhbCBTY2llbmNlcywgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgS2FzaGl3YSwgQ2hpYmEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29tcHV0YXRpb25hbCBCaW9sb2d5IFJlc2VhcmNoIENlbnRlciwgVGhlIE5hdGlvbmFsIEluc3RpdHV0ZSBmb3IgQWR2YW5jZWQgSW5kdXN0cmlhbCBTY2llbmNlIGFuZCBUZWNobm9sb2d5LCBBb21pLCBLb3RvLUt1LCBUb2t5bywgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJNYWthxYJvd3NraSwgV29qY2llY2giIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgQmlvaW5mb3JtYXRpY3MsIEZhY3VsdHkgb2YgTWVkaWNpbmUsIFVuaXZlcnNpdHkgb2YgTXVlbnN0ZXIsIE11bnN0ZXIsIEdlcm1hbnkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJLb2hubywgVGFrYXNoaSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRpdmlzaW9uIG9mIEdlbm9tZSBCaW9sb2d5LCBOYXRpb25hbCBDYW5jZXIgQ2VudGVyIFJlc2VhcmNoIEluc3RpdHV0ZSwgVHN1a2lqaSwgQ2h1by1LdSwgVG9reW8sIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3VnYW5vLCBTdW1pbyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3JfaW5zdGl0dXRpb24iIGNvbnRlbnQ9IkRlcGFydG1lbnQgb2YgQ29tcHV0YXRpb25hbCBCaW9sb2d5IGFuZCBNZWRpY2FsIFNjaWVuY2VzLCBHcmFkdWF0ZSBTY2hvb2wgb2YgRnJvbnRpZXIgU2NpZW5jZXMsIFRoZSBVbml2ZXJzaXR5IG9mIFRva3lvLCBLYXNoaXdhLCBDaGliYSwgSmFwYW4iIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJUc3VjaGloYXJhLCBLYXRzdXlhIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iRGl2aXNpb24gb2YgVHJhbnNsYXRpb25hbCBHZW5vbWljcywgRXhwbG9yYXRvcnkgT25jb2xvZ3kgUmVzZWFyY2ggYW5kIENsaW5pY2FsIFRyaWFsIENlbnRlciwgTmF0aW9uYWwgQ2FuY2VyIENlbnRlciwgS2FzaGl3YSwgQ2hpYmEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iU3V6dWtpLCBZdXRha2EiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJEZXBhcnRtZW50IG9mIENvbXB1dGF0aW9uYWwgQmlvbG9neSBhbmQgTWVkaWNhbCBTY2llbmNlcywgR3JhZHVhdGUgU2Nob29sIG9mIEZyb250aWVyIFNjaWVuY2VzLCBUaGUgVW5pdmVyc2l0eSBvZiBUb2t5bywgS2FzaGl3YSwgQ2hpYmEsIEphcGFuIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3RpdGxlIiBjb250ZW50PSJTZXF1ZW5jaW5nIGFuZCBwaGFzaW5nIGNhbmNlciBtdXRhdGlvbnMgaW4gbHVuZyBjYW5jZXJzIHVzaW5nIGEgbG9uZy1yZWFkIHBvcnRhYmxlIHNlcXVlbmNlciIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjU4NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iNTk2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHN4MDI3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjI0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSI2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTcvMTIvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgQ2FuY2VyIEdlbm9tZSBBdGxhcyBQYW4tQ2FuY2VyIGFuYWx5c2lzIHByb2plY3Q7IGNpdGF0aW9uX2F1dGhvcj1UaGUgQ2FuY2VyIEdlbm9tZSBBdGxhcyBSZXNlYXJjaCBOZXR3b3JrOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEdlbmV0OyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTQ1OyAgY2l0YXRpb25fcGFnZXM9MTExMy0yMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW50ZXJuYXRpb25hbCBuZXR3b3JrIG9mIGNhbmNlciBnZW5vbWUgcHJvamVjdHM7IGNpdGF0aW9uX2F1dGhvcj1JbnRlcm5hdGlvbmFsIENhbmNlciBHZW5vbWUgQ29uc29ydGl1bTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTQ2NDsgIGNpdGF0aW9uX3BhZ2VzPTk5My04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FcGlkZXJtYWwgZ3Jvd3RoIGZhY3RvciByZWNlcHRvciBtdXRhdGlvbnMgaW4gbHVuZyBjYW5jZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdC4gUmV2LiBDYW5jZXI7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NzsgIGNpdGF0aW9uX3BhZ2VzPTE2OS04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SWRlbnRpZmljYXRpb24gb2YgdGhlIHRyYW5zZm9ybWluZyBFTUw0LUFMSyBmdXNpb24gZ2VuZSBpbiBub24tc21hbGwtY2VsbCBsdW5nIGNhbmNlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0dXJlOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTQ0ODsgIGNpdGF0aW9uX3BhZ2VzPTU2MS02OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BbmFwbGFzdGljIGx5bXBob21hIGtpbmFzZSBpbmhpYml0aW9uIGluIG5vbi1zbWFsbC1jZWxsIGx1bmcgY2FuY2VyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OLiBFbmdsIEouIE1lZDsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0zNjM7ICBjaXRhdGlvbl9wYWdlcz0xNjkzLTE3MDM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUltcHJvdmVkIGRhdGEgYW5hbHlzaXMgZm9yIHRoZSBNaW5JT04gbmFub3BvcmUgc2VxdWVuY2VyOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MTI7ICBjaXRhdGlvbl9wYWdlcz0zNTEtNjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TWluSU9OIG5hbm9wb3JlIHNlcXVlbmNpbmcgaWRlbnRpZmllcyB0aGUgcG9zaXRpb24gYW5kIHN0cnVjdHVyZSBvZiBhIGJhY3RlcmlhbCBhbnRpYmlvdGljIHJlc2lzdGFuY2UgaXNsYW5kOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIEJpb3RlY2hub2w7ICBjaXRhdGlvbl95ZWFyPTIwMTU7ICBjaXRhdGlvbl92b2x1bWU9MzM7ICBjaXRhdGlvbl9wYWdlcz0yOTYtMzAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1SYXBpZCBkcmFmdCBzZXF1ZW5jaW5nIGFuZCByZWFsLXRpbWUgbmFub3BvcmUgc2VxdWVuY2luZyBpbiBhIGhvc3BpdGFsIG91dGJyZWFrIG9mIFNhbG1vbmVsbGE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbm9tZSBCaW9sOyAgY2l0YXRpb25feWVhcj0yMDE1OyAgY2l0YXRpb25fdm9sdW1lPTE2OyAgY2l0YXRpb25fcGFnZXM9MTE0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UmVhbC10aW1lLCBwb3J0YWJsZSBnZW5vbWUgc2VxdWVuY2luZyBmb3IgRWJvbGEgc3VydmVpbGxhbmNlOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXR1cmU7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NTMwOyAgY2l0YXRpb25fcGFnZXM9MjI4LTMyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1OYW5vcG9yZSBzZXF1ZW5jaW5nIGRldGVjdHMgc3RydWN0dXJhbCB2YXJpYW50cyBpbiBjYW5jZXI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNhbmNlciBCaW9sLiBUaGVyOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTE3OyAgY2l0YXRpb25fcGFnZXM9MjQ2LTUzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UUDUzIGdlbmUgbXV0YXRpb24gYW5hbHlzaXMgaW4gY2hyb25pYyBseW1waG9jeXRpYyBsZXVrZW1pYSBieSBuYW5vcG9yZSBNaW5JT04gc2VxdWVuY2luZzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9RGlhZ24uIFBhdGhvbDsgIGNpdGF0aW9uX3llYXI9MjAxNjsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTk2IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWJlcnJhbnQgdHJhbnNjcmlwdGlvbmFsIHJlZ3VsYXRpb25zIGluIGNhbmNlcnM6IGdlbm9tZSwgdHJhbnNjcmlwdG9tZSBhbmQgZXBpZ2Vub21lIGFuYWx5c2lzIG9mIGx1bmcgYWRlbm9jYXJjaW5vbWEgY2VsbCBsaW5lczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTQ7ICBjaXRhdGlvbl92b2x1bWU9NDI7ICBjaXRhdGlvbl9wYWdlcz0xMzU1Ny03MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9S0lGNUItUkVUIGZ1c2lvbnMgaW4gbHVuZyBhZGVub2NhcmNpbm9tYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBNZWQ7ICBjaXRhdGlvbl95ZWFyPTIwMTI7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0zNzUtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJpbWVyM1BsdXMsIGFuIGVuaGFuY2VkIHdlYiBpbnRlcmZhY2UgdG8gUHJpbWVyMzsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MzU7ICBjaXRhdGlvbl9wYWdlcz1XNzEtNDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9yZXRvb2xzOiBhIHRvb2xraXQgZm9yIGFuYWx5emluZyBuYW5vcG9yZSBzZXF1ZW5jZSBkYXRhOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0zMDsgIGNpdGF0aW9uX3BhZ2VzPTMzOTktNDAxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BZGFwdGl2ZSBzZWVkcyB0YW1lIGdlbm9taWMgc2VxdWVuY2UgY29tcGFyaXNvbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTQ4Ny05MzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RmFzdCBhbmQgYWNjdXJhdGUgc2hvcnQgcmVhZCBhbGlnbm1lbnQgd2l0aCBCdXJyb3dzLVdoZWVsZXIgdHJhbnNmb3JtOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTE3NTQtNjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRyYWluaW5nIGFsaWdubWVudCBwYXJhbWV0ZXJzIGZvciBhcmJpdHJhcnkgc2VxdWVuY2VycyB3aXRoIExBU1QtVFJBSU47ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDE2OyAgY2l0YXRpb25fdm9sdW1lPTMzOyAgY2l0YXRpb25fcGFnZXM9OTI2LTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUluY29ycG9yYXRpbmcgc2VxdWVuY2UgcXVhbGl0eSBkYXRhIGludG8gYWxpZ25tZW50IGltcHJvdmVzIEROQSByZWFkIG1hcHBpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU51Y2xlaWMgQWNpZHMgUmVzOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTM4OyAgY2l0YXRpb25fcGFnZXM9ZTEwMCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSBVQ1NDIEdlbm9tZSBCcm93c2VyIGRhdGFiYXNlOiAyMDE2IHVwZGF0ZTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TnVjbGVpYyBBY2lkcyBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz1ENzE3LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1GYXN0IGFuZCBTTlAtdG9sZXJhbnQgZGV0ZWN0aW9uIG9mIGNvbXBsZXggdmFyaWFudHMgYW5kIHNwbGljaW5nIGluIHNob3J0IHJlYWRzOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1CaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0yNjsgIGNpdGF0aW9uX3BhZ2VzPTg3My04MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9U3BsaXQtYWxpZ25tZW50IG9mIGdlbm9tZXMgZmluZHMgb3J0aG9sb2dpZXMgbW9yZSBhY2N1cmF0ZWx5OyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1HZW5vbWUgQmlvbDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0xNjsgIGNpdGF0aW9uX3BhZ2VzPTEwNiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIENDREM2LVJFVCBmdXNpb24gaW4gdGhlIGh1bWFuIGx1bmcgYWRlbm9jYXJjaW5vbWEgY2VsbCBsaW5lLCBMQy0yL2FkOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1KLiBUaG9yYWMuIE9uY29sOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTc7ICBjaXRhdGlvbl9wYWdlcz0xODcyLTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIG9mIGEgbHVuZyBhZGVub2NhcmNpbm9tYSBjZWxsIGxpbmUgd2l0aCBDQ0RDNi1SRVQgZnVzaW9uIGdlbmUgYW5kIHRoZSBlZmZlY3Qgb2YgUkVUIGluaGliaXRvcnMgaW4gdml0cm8gYW5kIGluIHZpdm87ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNhbmNlciBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9ODk2LTkwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGlzY292ZXJ5IG9mIEFMSy1QVFBOMyBnZW5lIGZ1c2lvbiBmcm9tIGh1bWFuIG5vbi1zbWFsbCBjZWxsIGx1bmcgY2FyY2lub21hIGNlbGwgbGluZSB1c2luZyBuZXh0IGdlbmVyYXRpb24gUk5BIHNlcXVlbmNpbmc7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUdlbmVzIENocm9tb3NvbWVzIENhbmNlcjsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT01MTsgIGNpdGF0aW9uX3BhZ2VzPTU5MC03OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Tb21hdGljIG11dGF0aW9ucyBhZmZlY3Qga2V5IHBhdGh3YXlzIGluIGx1bmcgYWRlbm9jYXJjaW5vbWE7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT00NTU7ICBjaXRhdGlvbl9wYWdlcz0xMDY5LTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1NYXBwaW5nIHRoZSBoYWxsbWFya3Mgb2YgbHVuZyBhZGVub2NhcmNpbm9tYSB3aXRoIG1hc3NpdmVseSBwYXJhbGxlbCBzZXF1ZW5jaW5nOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1DZWxsOyAgY2l0YXRpb25feWVhcj0yMDEyOyAgY2l0YXRpb25fdm9sdW1lPTE1MDsgIGNpdGF0aW9uX3BhZ2VzPTExMDctMjA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRoZSB0cmFuc2NyaXB0aW9uYWwgbGFuZHNjYXBlIGFuZCBtdXRhdGlvbmFsIHByb2ZpbGUgb2YgbHVuZyBhZGVub2NhcmNpbm9tYTsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9R2Vub21lIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0yMjsgIGNpdGF0aW9uX3BhZ2VzPTIxMDktMTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUlkZW50aWZpY2F0aW9uIGFuZCBjaGFyYWN0ZXJpemF0aW9uIG9mIGNhbmNlciBtdXRhdGlvbnMgaW4gSmFwYW5lc2UgbHVuZyBhZGVub2NhcmNpbm9tYSB3aXRob3V0IHNlcXVlbmNpbmcgb2Ygbm9ybWFsIHRpc3N1ZSBjb3VudGVycGFydHM7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPVBMb1MgT25lOyAgY2l0YXRpb25feWVhcj0yMDEzOyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz1lNzM0ODQiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wcmVoZW5zaXZlIG1vbGVjdWxhciBwcm9maWxpbmcgb2YgbHVuZyBhZGVub2NhcmNpbm9tYTsgY2l0YXRpb25fYXV0aG9yPVRoZSBDYW5jZXIgR2Vub21lIEF0bGFzIFJlc2VhcmNoIE5ldHdvcms7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPU5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT01MTE7ICBjaXRhdGlvbl9wYWdlcz01NDMtNTA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjcXVpcmVkIHJlc2lzdGFuY2Ugb2YgbHVuZyBhZGVub2NhcmNpbm9tYXMgdG8gZ2VmaXRpbmliIG9yIGVybG90aW5pYiBpcyBhc3NvY2lhdGVkIHdpdGggYSBzZWNvbmQgbXV0YXRpb24gaW4gdGhlIEVHRlIga2luYXNlIGRvbWFpbjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9UExvUyBNZWQ7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPWU3MyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFjcXVpcmVkIEVHRlIgQzc5N1MgbXV0YXRpb24gbWVkaWF0ZXMgcmVzaXN0YW5jZSB0byBBWkQ5MjkxIGluIG5vbi1zbWFsbCBjZWxsIGx1bmcgY2FuY2VyIGhhcmJvcmluZyBFR0ZSIFQ3OTBNOyAgY2l0YXRpb25fam91cm5hbF90aXRsZT1OYXQuIE1lZDsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTU2MC0yOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BIHByb3NwZWN0aXZlIFBDUi1iYXNlZCBzY3JlZW5pbmcgZm9yIHRoZSBFTUw0LUFMSyBvbmNvZ2VuZSBpbiBub24tc21hbGwgY2VsbCBsdW5nIGNhbmNlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2xpbi4gQ2FuY2VyIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxMjsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTU2ODItOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyaXNvbiBvZiBnZWZpdGluaWIsIGVybG90aW5pYiBhbmQgYWZhdGluaWIgaW4gbm9uLXNtYWxsIGNlbGwgbHVuZyBjYW5jZXI6IEEgbWV0YS1hbmFseXNpczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9SW50LiBKLiBDYW5jZXI7ICBjaXRhdGlvbl95ZWFyPTIwMTc7ICBjaXRhdGlvbl92b2x1bWU9MTQwOyAgY2l0YXRpb25fcGFnZXM9MjgwNS0xOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHdvIG5vdmVsIEFMSyBtdXRhdGlvbnMgbWVkaWF0ZSBhY3F1aXJlZCByZXNpc3RhbmNlIHRvIHRoZSBuZXh0LWdlbmVyYXRpb24gQUxLIGluaGliaXRvciBhbGVjdGluaWI7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNsaW4gQ2FuY2VyIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNDsgIGNpdGF0aW9uX3ZvbHVtZT0yMDsgIGNpdGF0aW9uX3BhZ2VzPTU2ODYtOTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJFVCBmdXNpb24gZ2VuZTogdHJhbnNsYXRpb24gdG8gcGVyc29uYWxpemVkIGx1bmcgY2FuY2VyIHRoZXJhcHk7ICBjaXRhdGlvbl9qb3VybmFsX3RpdGxlPUNhbmNlciBTY2k7ICBjaXRhdGlvbl95ZWFyPTIwMTM7ICBjaXRhdGlvbl92b2x1bWU9MTA0OyAgY2l0YXRpb25fcGFnZXM9MTM5Ni0xNDAwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1BY3F1aXJlZCByZXNpc3RhbmNlIHRvIFRLSXMgaW4gc29saWQgdHVtb3VyczogbGVhcm5pbmcgZnJvbSBsdW5nIGNhbmNlcjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9TmF0LiBSZXYuIENsaW4uIE9uY29sOyAgY2l0YXRpb25feWVhcj0yMDE0OyAgY2l0YXRpb25fdm9sdW1lPTExOyAgY2l0YXRpb25fcGFnZXM9NDczLTgxOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UaGUgYWxsZWxpYyBjb250ZXh0IG9mIHRoZSBDNzk3UyBtdXRhdGlvbiBhY3F1aXJlZCB1cG9uIHRyZWF0bWVudCB3aXRoIHRoaXJkLWdlbmVyYXRpb24gRUdGUiBpbmhpYml0b3JzIGltcGFjdHMgc2Vuc2l0aXZpdHkgdG8gc3Vic2VxdWVudCB0cmVhdG1lbnQgc3RyYXRlZ2llczsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Q2xpbi4gQ2FuY2VyIFJlczsgIGNpdGF0aW9uX3llYXI9MjAxNTsgIGNpdGF0aW9uX3ZvbHVtZT0yMTsgIGNpdGF0aW9uX3BhZ2VzPTM5MjQtMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVJlc2Vuc2l0aXphdGlvbiB0byBjcml6b3RpbmliIGJ5IHRoZSBsb3JsYXRpbmliIEFMSyByZXNpc3RhbmNlIG11dGF0aW9uIEwxMTk4RjsgIGNpdGF0aW9uX2pvdXJuYWxfdGl0bGU9Ti4gRW5nbCBKLiBNZWQ7ICBjaXRhdGlvbl95ZWFyPTIwMTY7ICBjaXRhdGlvbl92b2x1bWU9Mzc0OyAgY2l0YXRpb25fcGFnZXM9NTQtNjE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9mdWxsdGV4dF93b3JsZF9yZWFkYWJsZSIgY29udGVudD0iIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3BkZl91cmwiIGNvbnRlbnQ9Imh0dHBzOi8vYWNhZGVtaWMub3VwLmNvbS9kbmFyZXNlYXJjaC9hcnRpY2xlLXBkZi8yNC82LzU4NS8yMjI5NTc2NC9kc3gwMjcucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIEhlcmUsIHdlIGVtcGxveWVkIGNETkEgYW1wbGljb24gc2VxdWVuY2luZyB1c2luZyBhIGxvbmctcmVhZCBwb3J0YWJsZSBzZXF1ZW5jZXIsIE1pbklPTiwgdG8gY2hhcmFjdGVyaXplIHZhcmlvdXMgdHlwZXMgb2YgbXV0YXRpb25zIGluIGNhbmNlci1yZWxhdGVkIiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFcy8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj0zODk2MTUzJnhzbHRQYXRoPX4vVUkvYXBwL1hTTFQeCWlubmVyaHRtbAUURG93bmxvYWQgYWxsIGZpZ3VyZXNkZBMnCqxzctiGMLrhmv7MiwUMxDzq" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f585%2f3896153" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f585%2f3896153" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=103821" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/24/6"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/24/6/2/m_dnares_24_6cover.png?Expires=2147483647&amp;Signature=z4yCzT-6enr7Fof9QqkUJvtPQAXDRV5XYB7R7L2HBbQZ2iaySlHxBG~hDePAa-rY6Gn-PH9W8mIlJwAyG9R2RE6Vd3RTDct~I7biyNGgXk9CP54tGn91x8VYIMO24kwuQBqahsZQMv3NxRiQKhbGI0yEDanozvBgG~-9FpaC12JFLE6A4BT6rBzjsJLh9UJqvGv0F2hWW2jLvdITi~5PoATpRIsbIK-S1wcvqp49OSN~oRepZtr6iz~107ho8gZAxoaG8nb4DvXKrvfnCuhhmbRD41NP3K2zFyztHicWtTYb1xunX52RqodNq-M3zVvUit2rSw5-mRSvm-jXb-p6nw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/24/6">
                        <div class="volume trailing-comma">Volume 24</div>
                        <div class="issue">Issue 6</div>

            </a>

        </div>
        <div class="ii-pub-date">
December 2017        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="104086203">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086203">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086205">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086205">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086209">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086209">2. Materials and Methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086246">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086246">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086271">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086271">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086276">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086276">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="104086284">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#104086284">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/24/6/571/3868562">&lt; Previous</a>
        </li>        
            <li class="next arrow">
            <a href="/dnaresearch/article/24/6/597/3902984">Next &gt;</a>
        </li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Sequencing and phasing cancer mutations in lung cancers using a long-read portable sequencer
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Ayako Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Division of Translational Genomics, Exploratory Oncology Research and Clinical Trial Center, National Cancer Center, Kashiwa, Chiba, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Ayako+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki A">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki A&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Ayako Suzuki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Mizuto Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Mizuto+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Mizuto Suzuki</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Junko Mizushima-Sugano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan</div><div class="aff">Department of Chemistry and Life Science, Kogakuin University, Nishi-Shinjuku, Shinjuku-Ku, Tokyo, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Junko+Mizushima-Sugano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Mizushima-Sugano J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Mizushima-Sugano J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Junko Mizushima-Sugano</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Martin C Frith</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan</div><div class="aff">Computational Biology Research Center, The National Institute for Advanced Industrial Science and Technology, Aomi, Koto-Ku, Tokyo, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Martin+C+Frith">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Frith M">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Frith M&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Martin C Frith</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wojciech Makałowski</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Institute of Bioinformatics, Faculty of Medicine, University of Muenster, Munster, Germany</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wojciech+Maka%c5%82owski">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Makałowski W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Makałowski W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wojciech Makałowski</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Takashi Kohno</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Division of Genome Biology, National Cancer Center Research Institute, Tsukiji, Chuo-Ku, Tokyo, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Takashi+Kohno">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Kohno T">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Kohno T&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Takashi Kohno</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Sumio Sugano</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Sumio+Sugano">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Sugano S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Sugano S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Sumio Sugano</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Katsuya Tsuchihara</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Division of Translational Genomics, Exploratory Oncology Research and Clinical Trial Center, National Cancer Center, Kashiwa, Chiba, Japan</div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Katsuya+Tsuchihara">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Tsuchihara K">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Tsuchihara K&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Katsuya Tsuchihara</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yutaka Suzuki</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff">Department of Computational Biology and Medical Sciences, Graduate School of Frontier Sciences, The University of Tokyo, Kashiwa, Chiba, Japan</div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="dsx027-cor1" data-open="dsx027-cor1" class="link link-ref link-reveal xref-default"><!----></a>
                </div>

        <div class="info-author-correspondence">
            To whom correspondence should be addressed: Tel. +81 4 7136 3607. Fax. +81 4 7136 3607. Email: <a href="mailto:ysuzuki@hgc.jp" target="_blank">ysuzuki@hgc.jp</a>
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yutaka+Suzuki">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Suzuki Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Suzuki Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yutaka Suzuki</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx027-FM1"><div class="fn"><div class="footnote-content">Ayako Suzuki and Mizuto Suzuki contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="dsx027-FM2"><div class="fn"><div class="footnote-content">Edited by Dr. Mitsuo Oshimura</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 585–596, <a href='https://doi.org/10.1093/dnares/dsx027'>https://doi.org/10.1093/dnares/dsx027</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">27 June 2017</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">05 March 2017</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">29 May 2017</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="3896153" href="/dnaresearch/article-pdf/24/6/585/22295764/dsx027.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Ayako Suzuki, Mizuto Suzuki, Junko Mizushima-Sugano, Martin C Frith, Wojciech Makałowski, Takashi Kohno, Sumio Sugano, Katsuya Tsuchihara, Yutaka Suzuki;  Sequencing and phasing cancer mutations in lung cancers using a long-read portable sequencer, <em>DNA Research</em>, Volume 24, Issue 6, 1 December 2017, Pages 585–596, <a href='https://doi.org/10.1093/dnares/dsx027'>https://doi.org/10.1093/dnares/dsx027</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/3896153?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/3896153?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/3896153?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/3896153?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/3896153?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/3896153?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/3896153?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Sequencing%20and%20phasing%20cancer%20mutations%20in%20lung%20cancers%20using%20a%20long-read%20portable%20sequencer&amp;publicationDate=2017-06-27&amp;volumeNum=24&amp;issueNum=6&amp;author=Suzuki%2C%20Ayako%3B%20Suzuki%2C%20Mizuto&amp;startPage=585&amp;endPage=596&amp;contentId=10.1093%2Fdnares%2Fdsx027&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f585%2f3896153" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=104086203 id="104086203" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Here, we employed cDNA amplicon sequencing using a long-read portable sequencer, MinION, to characterize various types of mutations in cancer-related genes, namely, EGFR, KRAS, NRAS and NF1. For homozygous SNVs, the precision and recall rates were 87.5% and 91.3%, respectively. For previously reported hotspot mutations, the precision and recall rates reached 100%. The precise junctions of EML4-ALK, CCDC6-RET and five other gene fusions were also detected. Taking advantages of long-read sequencing, we conducted phasing of EGFR mutations and elucidated the mutational allelic backgrounds of anti-tumor drug-sensitive and resistant mutations, which could provide useful information for selecting therapeutic approaches. In the H1975 cells, 72% of the reads harbored both L858R and T790M mutations, and 22% of the reads harbored neither mutation. To ensure that the clinical requirements can be met in potentially low cancer cell populations, we further conducted a serial dilution analysis of the template for EGFR mutations. Several percent of the mutant alleles could be detected depending on the yield and quality of the sequencing data. Finally, we characterized the mutation genotypes in eight clinical samples. This method could be a convenient long-read sequencing-based analytical approach and thus may change the current approaches used for cancer genome sequencing.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="MinION">MinION</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;lung cancer cell lines&quot;">lung cancer cell lines</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;cancer mutations&quot;">cancer mutations</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="phasing">phasing</a></div>                            <h2 scrollto-destination=104086205 id="104086205" class="section-title" >1. Introduction</h2>
<p>Sequencing technologies have substantially advanced the study of cancer genomics. Large-scale studies, such as The Cancer Genome Atlas (TCGA)<a reveal-id="dsx027-B1" data-open="dsx027-B1" class="link link-ref link-reveal xref-bibr"><sup>1</sup></a> and the International Cancer Genome Consortium (ICGC)<a reveal-id="dsx027-B2" data-open="dsx027-B2" class="link link-ref link-reveal xref-bibr"><sup>2</sup></a>, have generated catalogs of various types of mutations in cancer cells, and this information is currently an invaluable resource for various purposes. In particular, these data collections provide critical information that can be used to identify the driver mutations in each cancer patient and inform initial therapeutic approaches, including molecular-targeted anti-cancer drugs. For example, several effective anti-cancer drugs are available for the treatment of lung adenocarcinoma. The EGFR tyrosine kinase inhibitors gefitinib and erlotinib have been used to treat EGFR-mutant tumors,<a reveal-id="dsx027-B3" data-open="dsx027-B3" class="link link-ref link-reveal xref-bibr"><sup>3</sup></a> while crizotinib and other ALK-kinase inhibitors can be a highly effective therapy for ALK-fusion positive patients.<a reveal-id="dsx027-B4" data-open="dsx027-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx027-B5" data-open="dsx027-B5" class="link link-ref link-reveal xref-bibr"><sup>5</sup></a> To determine whether these drugs should be administered, it is necessary to sequence the cancer nucleic acids and identify the driver mutation in each case. However, sequencing is limited by the following disadvantages: a large and very expensive instrument is needed; short-read sequencing occasionally overlooks the allelic background of the cancer mutations; and structural aberrations, such as fusion genes and splicing alterations, are typically difficult to detect.</p><p>MinION, which is a recently developed portable, disposable long-read sequencer,<a reveal-id="dsx027-B6" data-open="dsx027-B6" class="link link-ref link-reveal xref-bibr"><sup>6</sup></a> has the potential to address the drawbacks of the currently used cancer genome sequencing technologies. MinION is USB-sized and can be operated on a laptop PC. The template preparation takes only 2–3 h and does not require any specific experimental skills. In less than 48 h, hundreds of thousands of reads can be obtained, which are occasionally longer than 10 kb. Recent studies have applied MinION for several purposes focusing primarily on the detection and genotyping of pathogens. For example, one study reported the antibiotic genomic regions in Salmonella strains.<a reveal-id="dsx027-B7" data-open="dsx027-B7" class="link link-ref link-reveal xref-bibr"><sup>7</sup></a> In a more clinical setting, real-time sequencing using MinION and a phylogenic analysis of the obtained sequences revealed Salmonella outbreaks in a hospital.<a reveal-id="dsx027-B8" data-open="dsx027-B8" class="link link-ref link-reveal xref-bibr"><sup>8</sup></a> More recently, the first field application of MinION was used to diagnose the lethal virus Ebola in Africa.<a reveal-id="dsx027-B9" data-open="dsx027-B9" class="link link-ref link-reveal xref-bibr"><sup>9</sup></a> However, an analysis of genomic mutations in cancers using MinION has not been reported, except for a few pioneering studies.<a reveal-id="dsx027-B10" data-open="dsx027-B10" class="link link-ref link-reveal xref-bibr"><sup>10</sup></a><sup>,</sup><a reveal-id="dsx027-B11" data-open="dsx027-B11" class="link link-ref link-reveal xref-bibr"><sup>11</sup></a> To the best of our knowledge, the application of MinION for the identification and characterization of various genomic aberrations in cancer cells remains elusive.</p><p>In this study, we assessed the ability of MinION sequencing technology to characterize various types of cancer mutations. We sequenced cDNA amplicons from cancer-related genes and detected single nucleotide variants (SNVs) in EGFR, KRAS and NRAS; a short deletion in EGFR; aberrantly spliced RNAs in NF1; and CCDC6-RET, EML4-ALK, and other gene fusion transcripts in lung adenocarcinoma cell lines.<a reveal-id="dsx027-B12" data-open="dsx027-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> The phasing of the allelic information of the mutations was also examined using the long-read sequencing capability of MinION. We used cell lines as a model to apply a simple sequencing method to clinical cancer genomes.</p>                            <h2 scrollto-destination=104086209 id="104086209" class="section-title" >2. Materials and Methods</h2>
                            <h3 scrollto-destination=104086210 id="104086210" class="section-title" >2.1. Cell lines</h3>
<p>The lung adenocarcinoma cell lines PC-9, LC2/ad, PC-7, RERF-LC-Ad2, H1437, H1975, H2228, H2347, A549 and H322 were used in this study (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>) and have been previously described.<a reveal-id="dsx027-B12" data-open="dsx027-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> Total RNA was extracted from frozen cancer cell pellets using the RNeasy Maxi kit (Qiagen). The total RNA was assessed using an Agilent BioAnalyzer (Agilent), and the total RNA samples satisfied an RNA Integrity Number (RIN)&gt;9. For the H1975 and A549 cell lines, the genomic DNA was extracted from 1 × 10<sup>7</sup> cell pellets using the DNeasy Blood &amp; Tissue Kit (Qiagen). The extracted genomic DNA was eluted into 200 μl Buffer AE.</p>                            <h3 scrollto-destination=104086212 id="104086212" class="section-title" >2.2. Clinical samples</h3>
<p>The RNA samples were obtained from eight Japanese lung adenocarcinoma patients after obtaining their appreciated informed consent and the institutional review at the National Cancer Center Japan. The total RNA was extracted from frozen tissues using TRIzol (Invitrogen) as previously reported.<a reveal-id="dsx027-B13" data-open="dsx027-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> The RNA was assessed using an Agilent BioAnalyzer (Agilent). All samples satisfied RIN &gt; 7.</p>                            <h3 scrollto-destination=104086214 id="104086214" class="section-title" >2.3. RT-PCR</h3>
<p>Prior to the cDNA synthesis, the total RNA (approximately 10 μg from the cell lines and 500 ng from the clinical samples) was treated with DNase I (Takara) in a buffer containing 8 mM MgCl<sub>2</sub>, 40 mM Tris-HCl (pH 7.5), 5 mM DTT and RNasin Ribonuclease Inhibitor (Promega) for 10 min at 37 °C. After the phenol-chloroform extraction and ethanol precipitation for RNA purification, first-strand cDNA was synthesized using SuperScript II Reverse Transcriptase (Invitrogen) in First-Strand buffer with 0.8 mM dNTPs, 12 mM DTT, 2.5 μl dT primer (5’-GCGGCTGAAGACGGCCTATGTGGCCTTTTTTTTTTTTTTTTT-3’, 10 pmol/μl) and RNasin Ribonuclease Inhibitor (Promega) at 42 °C overnight. After the phenol-chloroform extraction of 100 μl cDNA sample, 2 μl 0.5 M EDTA (pH 8.0) and 15 μl 0.1 M NaOH were added. The samples were incubated at 65 °C for 40 min, and 20 μl 1 M Tris-HCl (pH 7.0–7.5) were added to the sample. After the ethanol precipitation, the samples were dissolved in ≥50 μl (for cell lines) or 20 μl (for clinical samples) H<sub>2</sub>O. The cDNA samples were PCR amplified using 1 cycle of 30 s at 98 °C; 40 cycles of 10 s at 98 °C, 30 s at 60 °C and 5 min at 72 °C; and a final 10 min cycle at 72 °C. For the PCR reaction, 25 μl 2× Phusion Master Mix (Finnzymes), 16-19 μl H<sub>2</sub>O and 5 μl 5 μM forward and reverse primers were added to 1–4 μl cDNA template. For the EGFR analysis in the clinical samples (<em>EGFR-iii</em>), a second (nested) PCR was conducted with 1/10-diluted PCR products using 1 cycle of 30 s at 98 °C; 30 cycles of 10 s at 98 °C, 30 s at 60 °C and 5 min at 72 °C; and a final 10 min cycle at 72 °C. The PCR primers and expected product lengths are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2A</a></span>. The primers were designed using Primer3plus.<a reveal-id="dsx027-B14" data-open="dsx027-B14" class="link link-ref link-reveal xref-bibr"><sup>14</sup></a> After the PCR reaction, the amplicons were purified using a QIAquick PCR purification kit (Qiagen).</p>                            <h3 scrollto-destination=104086216 id="104086216" class="section-title" >2.4. PCR of EGFR using genomic DNAs</h3>
<p>For the PCR reaction, 1μl genomic DNA templates were mixed with 25 μl 2× Phusion Master Mix (Finnzymes), 19 μl nuclease-free water, 2.5 μl forward primer (10 μM) and 2.5 μl reverse primer (10 μM). The mixed samples were amplified using 1 cycle of 40 s at 98 °C; 40 cycles of 10 s at 98 °C, 30 s at 60 °C and 1 min at 72 °C; and a final 10 min cycle at 72 °C. The PCR primers are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2B</a></span>.</p>                            <h3 scrollto-destination=104086218 id="104086218" class="section-title" >2.5. MinION library preparation for the cell lines</h3>
<p>Using a mixture of cDNA amplicons, MinION sequencing libraries were prepared using the Nanopore Sequencing Kit (SQK-MAP005, Oxford Nanopore Technologies) according to the manufacturer’s instructions. In total, 1 μg cDNA amplicons were prepared in 80 μl nuclease-free water and 5 μl DNA CS were added to the template. For the end-repair, the template was incubated at 20 °C for 20 min with 10 μl End-repair buffer and 5 μl End-repair enzyme mix (NEBNext End Repair Module, New England Biolabs). The template was purified using 100 μl Agencourt AMPure XP beads (Beckman Coulter, Inc.) and eluted into 25 μl H<sub>2</sub>O. To perform the dA tailing, 3 μl dA-tailing buffer and 2 μl dA-tailing enzyme (NEBNext dA-Taling Module) were added to the end-repaired DNA. The template was incubated at 37 °C for 10 min, purified by AMPure XP beads and eluted to 30 μl nuclease-free water. For the adapter ligation, the template was incubated for 10 min at room temperature with 10 μl Adapter Mix, 10 μl HP Adapter and 50 μl Blunt/TA adapter Ligase Master Mix (New England Biolabs). In total, 10 μl His-Tag beads (Dynabeads His-Tag Isolation and Pulldown, Life Technologies) were washed twice and resuspended with the Bead Binding Buffer. After the ligation, the adapter-ligated DNA was purified using the washed His-Tag beads and eluted in 25 μl Elution Buffer (‘pre-sequencing mix’). Before the sample loading, 325 μl priming buffer were mixed with 6.5 μl Fuel Mix, 162.5 μl 2× Running Buffer and 156 μl H<sub>2</sub>O. To prime the MinION R7.3 flow cell (FLO-MAP003, Oxford Nanopore technologies), 150 μl of the priming buffer were loaded twice with a 10-min waiting period before each loading. After preparing the flow cell, 75 μl 2× Running Buffer, 66 μl H<sub>2</sub>O, 3 μl Fuel Mix and 6 μl pre-sequencing mix were mixed, and then 150 μl of the resulting MinION sequencing library were loaded into the flow cell.</p>                            <h3 scrollto-destination=104086220 id="104086220" class="section-title" >2.6. MinION library preparation for the clinical samples and dilution series</h3>
<p>The Nanopore Sequencing Kit SQK-MAP006 (Oxford Nanopore Technologies) was used for the library preparation for the clinical samples and dilution series. Using a cDNA template (approximately 1 μg in 45 μl), 5 μl DNA CS were added. To perform the end-repair and dA tailing, 7 μl End-Prep buffer and 3 μl End-Prep enzyme mix (NEBNext Ultra II End-Repair/dA-tailing Module, New England Biolabs) were added. The template was incubated at 20 °C for 5 min and 65 °C for 5 min and was then purified using Agencourt AMPure XP beads (Beckman Coulter, Inc.). For the adapter ligation, 8 μl H<sub>2</sub>O, 10 μl Adapter Mix, 2 μl HP Adapter and 50 μl Blunt/TA adapter Ligase Master Mix (New England Biolabs) were added to the 30 μl DNA template. After incubating for 10 min at room temperature, 1 μl HP tether was added. The template was incubated for 10 min at room temperature. During the ligation, Dynabeads MyOne Streptavidin C1 beads (Life Technologies) were washed twice and suspended with 100 μl Bead Binding Buffer. After the ligation, the adapter-ligated DNA samples were purified using the washed beads and eluted using 25 μl Elution Buffer, followed by an incubation for 10 min at 37 °C. The eluted DNA sample was called the ‘pre-sequencing mix.’ Before loading the library, a mixture was prepared with 26.6 μl Fuel Mix, 500 μl 2× Running Buffer and 473.4 μl H<sub>2</sub>O. To prime the MinION R7.3 flow cell (FLO-MAP103, Oxford Nanopore Technologies), 500 μl of the mixture were loaded twice with a waiting period of 10 min before each loading. After preparing the flow cell, the MinION sequencing library was prepared with 75 μl 2× Running Buffer, 65 μl H<sub>2</sub>O, 4 μl Fuel Mix and 6 μl pre-sequencing mix and loaded to the flow cell.</p>                            <h3 scrollto-destination=104086222 id="104086222" class="section-title" >2.7. MinION sequencing</h3>
<p>Sequencing was performed for 48 h using MinKNOW. After the sequencing, base calling (2D Basecalling v1.24, SQK-MAP005 v1.34, SQK-MAP006 v1.62 and SQK-MAP006 v1.69) was performed via the ONT Metrichor (<a class="link link-uri" href="https://metrichor.com/s/index.shtml" target="_blank">https://metrichor.com/s/index.shtml</a>).</p>                            <h3 scrollto-destination=104086224 id="104086224" class="section-title" >2.8. Alignment of MinION sequencing data of cDNA amplicons</h3>
<p>The fastq files for the template, complement or 2D reads were converted from fast5 files using poretools version 0.5.1.<a reveal-id="dsx027-B15" data-open="dsx027-B15" class="link link-ref link-reveal xref-bibr"><sup>15</sup></a> Both ‘pass’ and ‘fail’ reads were used in this study. To compare the yield and qualities of the reads under different alignment conditions, LAST (version 658).<a reveal-id="dsx027-B16" data-open="dsx027-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> and BWA (version 0.7.12-r1039).<a reveal-id="dsx027-B17" data-open="dsx027-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> were used with the following sets of parameters; 1) BWA in ont2d mode; 2) LAST with the default parameter; 3) LAST with a match score of 1 (-r1), mismatch cost of 1 (-q1), gap existence cost of 1 (-a1) and gap extension cost of 1 (-b1); and 4) LAST with a gap existence cost of 12 (-a12), insertion existence cost of 15 (-A15), gap extension cost of 4 (-b4) and insertion extension cost of 4 (-B4) as determined according to the last-train.<a reveal-id="dsx027-B18" data-open="dsx027-B18" class="link link-ref link-reveal xref-bibr"><sup>18</sup></a> LAST uses the fastq quality data (with -Q1) to obtain more accurate alignments.<a reveal-id="dsx027-B19" data-open="dsx027-B19" class="link link-ref link-reveal xref-bibr"><sup>19</sup></a> The gap costs correspond to a statistical model of the specific probabilities of opening and extending insertions and deletions. The last-train determines the probabilities (and therefore costs) that fit the given sequence data. In these data, insertions are rarer than deletions; thus, their cost is higher.</p><p>Using LAST with the parameters -a12, -A15, -b4 and -B4, all 2D reads were mapped onto the 32,104 reference mRNA sequences based on the annotation file (refGene.txt) distributed by the UCSC Genome Browser (<a class="link link-uri" href="https://genome.ucsc.edu/" target="_blank">https://genome.ucsc.edu/</a>).<a reveal-id="dsx027-B20" data-open="dsx027-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a> The alignments with the best score in each query were extracted and used for further analysis. When the scores were identical for two hits, the first hit was selected. When the read was aligned to different isoforms of the same gene with identical scores, the representative isoform was prioritized (representative isoforms selected were NM_005528 in EGFR, NM_004985 in KRAS, NM_002524 in NRAS, NM_000267 in NF1, NM_019063 in EML4, NM_020630 in RET, NM_000855 in GUCY1A2, NM_025202 in EFHD1, NM_004198 in CHRNA6 and NM_022464 in SIL1 according to Illumina RNA-Seq data previously published in the DNA Data Bank of Japan (DDBJ) under the accession number DRA001846<a reveal-id="dsx027-B12" data-open="dsx027-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a>).</p>                            <h3 scrollto-destination=104086227 id="104086227" class="section-title" >2.9. Alignment of MinION sequencing data of genomic DNA amplicons</h3>
<p>Using LAST (version 658) with the parameters -a13, -A14, -b4 and -B3 which were determined by the last-train, all 2D reads were mapped onto chromosome 7 of the human genome UCSC hg38. The best alignments in each query were determined using last-split.</p>                            <h3 scrollto-destination=104086229 id="104086229" class="section-title" >2.10. Detection of SNVs</h3>
<p>Using the MinION reads, SNVs were detected as follows: 1) the reads aligned to the EGFR, KRAS, NRAS and NF1 target regions were extracted; 2) at each position, the depths of each base context were calculated considering only reads without errors ±3 bp of that position, and consensus sequences were constructed; 3) the consensus sequences were compared with the reference sequences and the SNVs were detected; and 4) the SNV candidates were verified using the Illumina whole-genome sequencing and RNA-Seq data obtained from the DDBJ under the accession numbers DRA001859 and DRA001846.<a reveal-id="dsx027-B12" data-open="dsx027-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a></p><p>Consensus sequences for the MinION reads were constructed as shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span> (step2). Positions with a read depth of less than 100 were defined as ‘Low depth.’ Positions in which the types of bases could not be determined were defined as ‘Unknown.’ A base (A, T, G or C) in a given position was called if the number of supporting reads for that base was more than twice the sum of the other bases. For heterozygous sites, a second base was similarly defined. Reads without mismatches ±3 bp of the same position were used.</p><p>To verify the SNVs detected by the MinION consensus, the SNVs were also identified using the Illumina RNA-Seq data.<a reveal-id="dsx027-B12" data-open="dsx027-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> The RNA-Seq data were mapped onto the human reference genome (UCSC hg19) using GSNAP.<a reveal-id="dsx027-B21" data-open="dsx027-B21" class="link link-ref link-reveal xref-bibr"><sup>21</sup></a> By scanning all the EGFR, KRAS, NRAS and NF1 target regions, 41 SNVs (variant allele frequency &gt;10%) were detected as a validation dataset. Among the 41 true SNVs, 39 SNVs harboring sufficient depths in the MinION reads were considered true positives.</p>                            <h3 scrollto-destination=104086233 id="104086233" class="section-title" >2.11. Detection of deletions and aberrant splicing</h3>
<p>To detect short deletions in EGFR, the deletion depths were considered as the consensus sequences were constructed. The deletions were detected similarly to the SNV detection. After comparing to a known driver EGFR deletion, the false positive detections were removed. For aberrant NF1 exon skipping, the reads aligned to NF1 were re-aligned by split alignment using LAST. After calculating the depth of the split alignment, NF1 skipping was observed as site with no or few aligned reads.</p>                            <h3 scrollto-destination=104086235 id="104086235" class="section-title" >2.12. Analysis of fusion transcripts</h3>
<p>To detect the fusion transcripts and junction points in the MinION reads, the MinION reads were aligned to 32,104 reference mRNA sequences with split alignment<a reveal-id="dsx027-B22" data-open="dsx027-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> using LAST (with the -m1 option to allow for multiple hits). Reads split between two different genes were extracted and counted. Gene pairs supported by more than 100 split reads were defined as fusion transcripts. A position at which the depth most significantly changed was considered a potential junction point. The junction points of known driver fusion transcripts (ALK and RET fusion transcripts) have been confirmed in previous studies.<a reveal-id="dsx027-B23 dsx027-B24 dsx027-B25" data-open="dsx027-B23 dsx027-B24 dsx027-B25" class="link link-ref link-reveal xref-bibr"><sup>23–25</sup></a></p><p>To assess the alignment accuracy and confirm the junction points, the MinION reads were also aligned to fused RNA sequences using LAST with the trained parameters described above. In addition, the MinION reads were aligned to 32,104 reference mRNA sequences to avoid a misalignment.</p>                            <h3 scrollto-destination=104086238 id="104086238" class="section-title" >2.13.Phasing</h3>
<p>To phase the two EGFR mutations (T790M and L858R) in the H1975 cells, the reads completely covering the EGFR kinase domain were extracted. Reads without any mismatches ±3 bp at both mutation sites were used for the phasing. The reads with deletions at the mutation sites were removed from this analysis.</p>                            <h3 scrollto-destination=104086240 id="104086240" class="section-title" >2.14. EGFR dilution analysis for mutation detection</h3>
<p>PCR amplicons of EGFR (<em>EGFR-i</em>, 3.4 kb) were prepared using cDNAs from H1975 (mutant) and RERF-LC-Ad2 (wild-type) cells. The H1975 amplicons were diluted with those from RERF-LC-Ad2 at the following ratios: 1:1, 1:4, 1:9, 1:19 and 1:99. The diluted samples were sequenced using MinION according to the manufacturer’s instructions. The MinION reads were processed for mutation phasing as described above.</p>                            <h3 scrollto-destination=104086242 id="104086242" class="section-title" >2.15. Validation using TA cloning and Sanger sequencing</h3>
<p>All driver mutations and junction points in the fusion transcripts were validated by direct Sanger sequencing. The results of the mutation phasing of EGFR in H1975 and junction sequences of EML4-ALK in H2228 were validated with TA cloning using the pMD20-T vector (Mighty TA-Cloning Kit, #6028, Takara). The PCR and sequencing primers are listed in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S3</a></span>.</p>                            <h3 scrollto-destination=104086244 id="104086244" class="section-title" >2.16. Data access</h3>
<p>The MinION sequencing data from the cell lines were deposited in the DDBJ under the accession numbers DRA004627 and DRA005767. The MinION sequencing data from the clinical samples were published in the National Bioscience Database Center (NBDC) and DDBJ in Japan with the accession number JGAS00000000065 under controlled access.</p>                            <h2 scrollto-destination=104086246 id="104086246" class="section-title" >3. Results</h2>
                            <h3 scrollto-destination=104086247 id="104086247" class="section-title" >3.1. Nanopore sequencing of cancer-related genes</h3>
<p>Using MinION, which was provided by Oxford Nanopore Technologies (ONT), we sequenced the reverse-transcription (RT)-PCR products from four genes, i.e., EGFR, KRAS, NRAS and NF1, as representative oncogenes and tumor-suppressor genes in lung cancers (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>; the primer sequences are shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S2A</a></span>).<a reveal-id="dsx027-B26 dsx027-B27 dsx027-B28 dsx027-B29 dsx027-B30" data-open="dsx027-B26 dsx027-B27 dsx027-B28 dsx027-B29 dsx027-B30" class="link link-ref link-reveal xref-bibr"><sup>26–30</sup></a> These genes represent various mutation patterns. Specifically, single base substitutions, short deletions and exon skipping have been reported in the EGFR, KRAS and NRAS genes; the EGFR gene; and the NF1 gene, respectively. We also analyzed the ALK and RET fusion transcripts and several novel fusion transcripts identified in previous Illumina RNA-Seq studies.<a reveal-id="dsx027-B12" data-open="dsx027-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S1</a></span>). In total, 33 PCR products were sequenced. We divided these products into three pools and performed five MinION sequencing runs. In each run, an average of 47,306, 25,948 and 21,812 reads were obtained as ‘template,’ ‘complement’ and ‘2D’ reads, respectively (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S4</a></span>). In the following analysis, we used a total of 109,060 2D reads from the five runs.</p><p>As shown in <a reveal-id="dsx027-F1" data-open="dsx027-F1" class="link link-reveal link-table xref-fig">Fig. 1A</a>, the average read length was 1,835 bases. We expected that such generally long reads might sufficiently cover entire target regions, which ranged from 640 to 3,402 bases in length (see below). The base quality values (QV) were approximately 10.5 on average (<a reveal-id="dsx027-F1" data-open="dsx027-F1" class="link link-reveal link-table xref-fig">Fig. 1B</a>, statistics of the template and complement reads are also shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S3</a></span>). We aligned the obtained reads to the reference human transcript sequences (32,104 transcripts; UCSC Genome Browser<a reveal-id="dsx027-B20" data-open="dsx027-B20" class="link link-ref link-reveal xref-bibr"><sup>20</sup></a>). In the alignment, we compared two commonly used alignment programs BWA (ont2d mode)<a reveal-id="dsx027-B17" data-open="dsx027-B17" class="link link-ref link-reveal xref-bibr"><sup>17</sup></a> and LAST (with three sets of parameters).<a reveal-id="dsx027-B16" data-open="dsx027-B16" class="link link-ref link-reveal xref-bibr"><sup>16</sup></a> These programs were generally consistent with the aligned reads, although the target cover rates and sequence identities were dependent on the respective parameters (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S4</a></span>, the parameters used in this analysis are shown in the Methods section and Supplementary Figure legends). Using LAST with the ‘trained’ parameters, on average 75% of the reads were aligned to the target mRNAs (<a reveal-id="dsx027-F1" data-open="dsx027-F1" class="link link-reveal link-table xref-fig">Fig. 1C</a>). Under these conditions, at least 150 reads were obtained for each of the bases in the target regions. For the EGFR, KRAS, NRAS and NF1 genes, the aligned reads showed 83% sequence identity on average, and 73% of the reads showed more than 80% sequence identity (<a reveal-id="dsx027-F1" data-open="dsx027-F1" class="link link-reveal link-table xref-fig">Fig. 1D</a>). We also examined the patterns of the sequencing errors. The error rates for the mismatches, deletions and insertions accounted for 7.3%, 7.8% and 2.5%, respectively, of the total error rate of 17% (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5A</a></span>). Cytosine or guanine bases were more likely to be miscalled. Deletions accumulated more at the homopolymer sites (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S5B</a></span> and C). We also examined the length of each read that aligned to the target sequence. The results showed that on average, each read covered 82% of the target region, and 66% of the aligned reads covered more than 90% of the target regions, indicating that nearly the full-length of these amplicons was sequenced (<a reveal-id="dsx027-F1" data-open="dsx027-F1" class="link link-reveal link-table xref-fig">Fig. 1E</a>). These results suggested that the MinION reads obtained in this study are worthy for further analyses, such as mutation detection and phasing. </p>                        <a id="104086250" scrollto-destination="104086250"></a>
<div id="dsx027-F1" data-id="dsx027-F1" class="fig fig-section"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/m_dsx027f1.png?Expires=2147483647&Signature=BFqqx10Tj4R~meorpC2vURsbDS-ZjQi~ecN7iZ9kmUHeseGIoiv6bdYLo0VCY7ghRCnqhuRNehx99z-wE2TW3CsM5M94DEzUQEUKVQo7YAY-nM~IN8C19MrtbfyNbRKoVxBCYhZeOLbDI~~bWVpppFailpN61tackGZI8-x3ZDs3utObxBYqwIqv0Lxbh8GaHf8sJ09-zIZCccagMt6~mLspPTDPcY23UYwZ~yqjFS-EGwsYp0QEA9cFopX76Nf7YQrzMSFhs72oZm4IBU6ctSnaRdQ~qkMKb3tTTgJm2Wb1aThimqhJsTHctX8NJvtR6aZFORr31mJL1FM9YJJm~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary of the amplicon sequencing and alignment statistics. (A, B) Distribution of the read lengths (A) and QVs (B) in all five sequencing runs. The average number is shown in the inset. (C) The number of MinION reads aligned to each of the PCR target regions. For the alignment, we used LAST with tuned parameters as described in the Methods section. (D, E) Distributions of the sequence identity (D) and target cover rate (E) in each read. The average number is shown in the inset." path-from-xml="dsx027f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086250/dsx027f1.tif" path-from-xml="dsx027f1.tif" target="_blank">View large</a><a section="104086250" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/dsx027f1.png?Expires=2147483647&Signature=Ty3gwQBMi6GZV9UqphX9px~J7M~Bl6iYTIWi55Wei1AuXJyPbQ2iRnIKtkBx724A29sRVD7lJLWjCOkrasM6tPAOe~I2CX-3ayj5uxVGWbcCqAvRF~70kqrwQEiWu84mG4yYokhiQm8zvqWJOtQBA~r9JErmud2IHjcAZC3LjJlVREzRFOi8OmhHhB6O2zqcmzY7xsf19YWaZlWIGJAZIyooobfaJGCwZSi0qtonWwn8MpVYK0M4wQZiJSiLYX1kbYeaEKBtGTSOvttrbP5WJcLCw0lRQWjUrffMHTq2Uqlz8G0XFfpnL5e601eTGEpyx5axguR0597OGw8x3dbKfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086250&ar=3896153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx027f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary of the amplicon sequencing and alignment statistics. (A, B) Distribution of the read lengths (A) and QVs (B) in all five sequencing runs. The average number is shown in the inset. (C) The number of MinION reads aligned to each of the PCR target regions. For the alignment, we used LAST with tuned parameters as described in the Methods section. (D, E) Distributions of the sequence identity (D) and target cover rate (E) in each read. The average number is shown in the inset.</p></div></div><div content-id="dsx027-F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/m_dsx027f1.png?Expires=2147483647&Signature=BFqqx10Tj4R~meorpC2vURsbDS-ZjQi~ecN7iZ9kmUHeseGIoiv6bdYLo0VCY7ghRCnqhuRNehx99z-wE2TW3CsM5M94DEzUQEUKVQo7YAY-nM~IN8C19MrtbfyNbRKoVxBCYhZeOLbDI~~bWVpppFailpN61tackGZI8-x3ZDs3utObxBYqwIqv0Lxbh8GaHf8sJ09-zIZCccagMt6~mLspPTDPcY23UYwZ~yqjFS-EGwsYp0QEA9cFopX76Nf7YQrzMSFhs72oZm4IBU6ctSnaRdQ~qkMKb3tTTgJm2Wb1aThimqhJsTHctX8NJvtR6aZFORr31mJL1FM9YJJm~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Summary of the amplicon sequencing and alignment statistics. (A, B) Distribution of the read lengths (A) and QVs (B) in all five sequencing runs. The average number is shown in the inset. (C) The number of MinION reads aligned to each of the PCR target regions. For the alignment, we used LAST with tuned parameters as described in the Methods section. (D, E) Distributions of the sequence identity (D) and target cover rate (E) in each read. The average number is shown in the inset." path-from-xml="dsx027f1.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086250/dsx027f1.tif" path-from-xml="dsx027f1.tif" target="_blank">View large</a><a section="104086250" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/dsx027f1.png?Expires=2147483647&Signature=Ty3gwQBMi6GZV9UqphX9px~J7M~Bl6iYTIWi55Wei1AuXJyPbQ2iRnIKtkBx724A29sRVD7lJLWjCOkrasM6tPAOe~I2CX-3ayj5uxVGWbcCqAvRF~70kqrwQEiWu84mG4yYokhiQm8zvqWJOtQBA~r9JErmud2IHjcAZC3LjJlVREzRFOi8OmhHhB6O2zqcmzY7xsf19YWaZlWIGJAZIyooobfaJGCwZSi0qtonWwn8MpVYK0M4wQZiJSiLYX1kbYeaEKBtGTSOvttrbP5WJcLCw0lRQWjUrffMHTq2Uqlz8G0XFfpnL5e601eTGEpyx5axguR0597OGw8x3dbKfw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086250&ar=3896153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx027f1.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Summary of the amplicon sequencing and alignment statistics. (A, B) Distribution of the read lengths (A) and QVs (B) in all five sequencing runs. The average number is shown in the inset. (C) The number of MinION reads aligned to each of the PCR target regions. For the alignment, we used LAST with tuned parameters as described in the Methods section. (D, E) Distributions of the sequence identity (D) and target cover rate (E) in each read. The average number is shown in the inset.</p></div></div>                            <h3 scrollto-destination=104086251 id="104086251" class="section-title" >3.2.Detection of SNVs and other types of cancerous mutations</h3>
<p>In the EGFR, KRAS, NRAS and NF1 genes, the expected types of mutations were detected, namely SNVs, short deletions and aberrantly spliced transcripts reflecting a splice site mutation. To detect the SNVs, we constructed consensus sequences for the MinION reads that were aligned to the target transcripts. As shown in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>, an SNV was considered a possible homozygous SNV if the read count of a particular base (A, T, G or C) at a given position was larger than twice the sum of the other bases. We also considered possible heterozygous SNVs similarly using varying thresholds (see below; also see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S2</a></span>). To further refine the detected SNVs, we only considered the reads with no mismatches in the ±3 bp region. This filter was useful, likely reflecting the base call scheme of MinION, based on ‘squiggle,’ a five-base window.</p><p>For the detected SNVs, we validated the correct identification rates by comparing these variants with the RNA-Seq data from Illumina HiSeq. The indicated precision and recall rates of the called SNVs were determined according to the category (putative homo, heterozygous SNVs or ‘minor’ SNVs reflecting minor cancer cell numbers or mutant alleles in the population). The detection depended on the threshold of the variant allele frequencies (VAFs) of the MinION reads, which is indicated as ‘X’ in <a reveal-id="dsx027-F2" data-open="dsx027-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a> (also see <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Tables S5 and S6</a></span>). Indeed, the sensitivity and precision of detecting the SNVs primarily depended on the extent of the coverage of the minor SNVs. SNVs with high variant allele frequencies (&gt;75% in the Illumina RNA-Seq data), including homogeneous SNPs and mutations, were detected with high precision and recall rates (87.5% and 91.3%, respectively, when X = 0.9; blue line; <a reveal-id="dsx027-F2" data-open="dsx027-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). However, the performance declined when minor SNPs (&gt;50% or &gt;10% in the Illumina RNA-Seq data) were also considered (precision and recall rates were 85.2% and 71.9%, respectively, when X = 0.5; orange line; 50.7% and 89.7%, respectively, when X = 0.3; black line; <a reveal-id="dsx027-F2" data-open="dsx027-F2" class="link link-reveal link-table xref-fig">Fig. 2A</a>). Intriguingly, the variant allele frequencies of the known heterozygous SNPs in the MinION reads were highly correlated with those in the Illumina RNA-Seq reads (<a reveal-id="dsx027-F2" data-open="dsx027-F2" class="link link-reveal link-table xref-fig">Fig. 2C</a>). Notably, most of the representative driver mutations, such as KRAS G12S in the A549 cells and NRAS Q61R in the H2347 cells (<a reveal-id="dsx027-F2" data-open="dsx027-F2" class="link link-reveal link-table xref-fig">Fig. 2B</a>), were detected. </p>                        <a id="104086254" scrollto-destination="104086254"></a>
<div id="dsx027-F2" data-id="dsx027-F2" class="fig fig-section"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/m_dsx027f2.png?Expires=2147483647&amp;Signature=JPbP7S94TqWdzxc641pyEX0UUNuKXfXND~5Iofb~hRhJQMrBjT5Gp5LtoghVjLrASgDWXJT0n-JJ4iNDWLzVeYeLV2H4PElj-4tCee60Sl~9ViBuZDnUT0T5jMFbXyAW1H-EDh4Ev~w8AYaeysgNuqkY~ewG56lNy0rRNk4oG19L3bWmsh3tqul8ozdZIeVOFbwDM3Kk0rdioCWPnKrykdsqpYfY9XwdPUXy0k8wYflN8ck69SpIsQLAtx5zJy2J2jzAXNX~kpKGAAbRuWnyi9q4kzDOszODvIhFWO6Z~TnnBdAG1DCFbSg6Tb5wFsBxHJc2R75ewMkW9I~1P9adMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of SNPs and mutations in the MinION reads. (A) Precision and recall rates of SNV detection using MinION. Blue, orange and black lines represent three datasets of SNVs corresponding to different variant allele frequencies (VAF) of Illumina standards, which are more than 75% (targets are only homozygous variants), 50% (targets include heterozygous variants) and 10% (targets include minor population variant). ‘X’ represents one of the parameters for the SNV detection, which is the threshold of the VAFs of the MinION reads. Additional details regarding the procedure are described in Supplementary Fig. 2. (B) The depths and base patterns of KRAS G12S in the A549 cells (left) and NRAS Q61R in the H2347 cells (right). The pre-cleaned data are shown in the upper panel. The cleaned data in which the MinION reads without mismatches ±3 bp of the SNVs were used are shown in the lower panel. The color key for the base patterns is represented in the margin. (C) VAFs for the Illumina RNA-Seq and MinION sequencing at the 41 SNVs. SNPs and somatic SNVs are shown as black circles and red crosses, respectively. (D) The depths and base patterns of the 15-base EGFR deletion in the PC-9 cells in the MinION reads. The pre-cleaned and cleaned data are shown in the upper and middle panel. IGV visualization of the Illumina RNA-Seq data is represented in the lower panel. The color key is the same as that shown in B. (E) Exon skipping in exon 19 of NF1 in the PC-7 cells. In the upper panel, the sequence depths of the MinION reads aligned to the NF1-ii region with split alignment using LAST are shown in the PC-7 (black) and LC2/ad (blue, wild-type) cells. The exon skipping in the Illumina RNA-Seq data is also shown in the lower panel." path-from-xml="dsx027f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086254/dsx027f2.tif" path-from-xml="dsx027f2.tif" target="_blank">View large</a><a section="104086254" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/dsx027f2.png?Expires=2147483647&amp;Signature=NfeedwmYMib8Q6A6~rEbW~E4Az3cGJ6R-ZCFjYiZK~DVjmyAiH5kTNMbYMfFkFznxWSdXn7cRvsP2QdkKNuPjT0f7ei7MRwhdT-ZT1hAV-NRhm7zD0yyceGwN1I5~C3O47XHF8SD5izK~X8h47J328S8gai1D3BUhoVhuPSBT2CktQP708U2G0YEu-1fq2i6F7NR7hqTxBFtc0igZo~1YZQZ8S-3QO7-pnGEgzsbb0-6Y5URbWsxGf1OqxBJ61~TRFPOO3oOcgkL1zFbbJbKbxgGPsaD4vdOugiUy5RD19Lm~3JyOmOAmUeL8Tt-K5hwQ4rRCPNS3MSOxlOlgY-cTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=104086254&amp;ar=3896153&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx027f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of SNPs and mutations in the MinION reads. (A) Precision and recall rates of SNV detection using MinION. Blue, orange and black lines represent three datasets of SNVs corresponding to different variant allele frequencies (VAF) of Illumina standards, which are more than 75% (targets are only homozygous variants), 50% (targets include heterozygous variants) and 10% (targets include minor population variant). ‘X’ represents one of the parameters for the SNV detection, which is the threshold of the VAFs of the MinION reads. Additional details regarding the procedure are described in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&amp;Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. 2</a></span>. (B) The depths and base patterns of KRAS G12S in the A549 cells (left) and NRAS Q61R in the H2347 cells (right). The pre-cleaned data are shown in the upper panel. The cleaned data in which the MinION reads without mismatches ±3 bp of the SNVs were used are shown in the lower panel. The color key for the base patterns is represented in the margin. (C) VAFs for the Illumina RNA-Seq and MinION sequencing at the 41 SNVs. SNPs and somatic SNVs are shown as black circles and red crosses, respectively. (D) The depths and base patterns of the 15-base EGFR deletion in the PC-9 cells in the MinION reads. The pre-cleaned and cleaned data are shown in the upper and middle panel. IGV visualization of the Illumina RNA-Seq data is represented in the lower panel. The color key is the same as that shown in B. (E) Exon skipping in exon 19 of NF1 in the PC-7 cells. In the upper panel, the sequence depths of the MinION reads aligned to the <em>NF1-ii</em> region with split alignment using LAST are shown in the PC-7 (black) and LC2/ad (blue, wild-type) cells. The exon skipping in the Illumina RNA-Seq data is also shown in the lower panel.</p></div></div><div content-id="dsx027-F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/m_dsx027f2.png?Expires=2147483647&amp;Signature=JPbP7S94TqWdzxc641pyEX0UUNuKXfXND~5Iofb~hRhJQMrBjT5Gp5LtoghVjLrASgDWXJT0n-JJ4iNDWLzVeYeLV2H4PElj-4tCee60Sl~9ViBuZDnUT0T5jMFbXyAW1H-EDh4Ev~w8AYaeysgNuqkY~ewG56lNy0rRNk4oG19L3bWmsh3tqul8ozdZIeVOFbwDM3Kk0rdioCWPnKrykdsqpYfY9XwdPUXy0k8wYflN8ck69SpIsQLAtx5zJy2J2jzAXNX~kpKGAAbRuWnyi9q4kzDOszODvIhFWO6Z~TnnBdAG1DCFbSg6Tb5wFsBxHJc2R75ewMkW9I~1P9adMg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of SNPs and mutations in the MinION reads. (A) Precision and recall rates of SNV detection using MinION. Blue, orange and black lines represent three datasets of SNVs corresponding to different variant allele frequencies (VAF) of Illumina standards, which are more than 75% (targets are only homozygous variants), 50% (targets include heterozygous variants) and 10% (targets include minor population variant). ‘X’ represents one of the parameters for the SNV detection, which is the threshold of the VAFs of the MinION reads. Additional details regarding the procedure are described in Supplementary Fig. 2. (B) The depths and base patterns of KRAS G12S in the A549 cells (left) and NRAS Q61R in the H2347 cells (right). The pre-cleaned data are shown in the upper panel. The cleaned data in which the MinION reads without mismatches ±3 bp of the SNVs were used are shown in the lower panel. The color key for the base patterns is represented in the margin. (C) VAFs for the Illumina RNA-Seq and MinION sequencing at the 41 SNVs. SNPs and somatic SNVs are shown as black circles and red crosses, respectively. (D) The depths and base patterns of the 15-base EGFR deletion in the PC-9 cells in the MinION reads. The pre-cleaned and cleaned data are shown in the upper and middle panel. IGV visualization of the Illumina RNA-Seq data is represented in the lower panel. The color key is the same as that shown in B. (E) Exon skipping in exon 19 of NF1 in the PC-7 cells. In the upper panel, the sequence depths of the MinION reads aligned to the NF1-ii region with split alignment using LAST are shown in the PC-7 (black) and LC2/ad (blue, wild-type) cells. The exon skipping in the Illumina RNA-Seq data is also shown in the lower panel." path-from-xml="dsx027f2.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086254/dsx027f2.tif" path-from-xml="dsx027f2.tif" target="_blank">View large</a><a section="104086254" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/dsx027f2.png?Expires=2147483647&amp;Signature=NfeedwmYMib8Q6A6~rEbW~E4Az3cGJ6R-ZCFjYiZK~DVjmyAiH5kTNMbYMfFkFznxWSdXn7cRvsP2QdkKNuPjT0f7ei7MRwhdT-ZT1hAV-NRhm7zD0yyceGwN1I5~C3O47XHF8SD5izK~X8h47J328S8gai1D3BUhoVhuPSBT2CktQP708U2G0YEu-1fq2i6F7NR7hqTxBFtc0igZo~1YZQZ8S-3QO7-pnGEgzsbb0-6Y5URbWsxGf1OqxBJ61~TRFPOO3oOcgkL1zFbbJbKbxgGPsaD4vdOugiUy5RD19Lm~3JyOmOAmUeL8Tt-K5hwQ4rRCPNS3MSOxlOlgY-cTg__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&amp;sec=104086254&amp;ar=3896153&amp;xsltPath=~/UI/app/XSLT&amp;imagename=" class="download-slide" path-from-xml="dsx027f2.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of SNPs and mutations in the MinION reads. (A) Precision and recall rates of SNV detection using MinION. Blue, orange and black lines represent three datasets of SNVs corresponding to different variant allele frequencies (VAF) of Illumina standards, which are more than 75% (targets are only homozygous variants), 50% (targets include heterozygous variants) and 10% (targets include minor population variant). ‘X’ represents one of the parameters for the SNV detection, which is the threshold of the VAFs of the MinION reads. Additional details regarding the procedure are described in <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&amp;Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. 2</a></span>. (B) The depths and base patterns of KRAS G12S in the A549 cells (left) and NRAS Q61R in the H2347 cells (right). The pre-cleaned data are shown in the upper panel. The cleaned data in which the MinION reads without mismatches ±3 bp of the SNVs were used are shown in the lower panel. The color key for the base patterns is represented in the margin. (C) VAFs for the Illumina RNA-Seq and MinION sequencing at the 41 SNVs. SNPs and somatic SNVs are shown as black circles and red crosses, respectively. (D) The depths and base patterns of the 15-base EGFR deletion in the PC-9 cells in the MinION reads. The pre-cleaned and cleaned data are shown in the upper and middle panel. IGV visualization of the Illumina RNA-Seq data is represented in the lower panel. The color key is the same as that shown in B. (E) Exon skipping in exon 19 of NF1 in the PC-7 cells. In the upper panel, the sequence depths of the MinION reads aligned to the <em>NF1-ii</em> region with split alignment using LAST are shown in the PC-7 (black) and LC2/ad (blue, wild-type) cells. The exon skipping in the Illumina RNA-Seq data is also shown in the lower panel.</p></div></div><p>We further examined the patterns in the false SNV detection. For example, one SNP in the 3’ UTR of KRAS (c.*264C &gt; T in A549 and H2228; rs4285970) was not detected using MinION. Upon further inspection, we observed that this SNP is sandwiched between a 4-base homopolymer GGGG and another 4-base homopolymer TTTT (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S6</a></span>), and the deletions were called in most of the reads at this locus. In total, 521 false positive SNVs were called when the threshold ‘X’ (to 0.1) was lowered to consider the minor SNVs (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7A</a></span>). Among these false positive SNVs, nearly half were located within or adjacent to ≥3-base homopolymer sequences. In addition, more than 70% of the false positives were miscalls to a surrounding base (±1 bp) and/or C &gt; G/G &gt; C errors (Supplementary Fig. S7B and C). These results suggested that, for a more precise SNV detection, we should consider the presence of homopolymer sites in the surrounding regions. We also found that these errors could be decreased by examining longer matches of the surrounding bases (right, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S7C</a></span>). A lower accuracy regarding homopolymers is one of the known problems with MinION sequencing. Base-callers occasionally call a wrong number of bases at homopolymer sites. Oxford Nanopore Technologies and several academic bioinformatics teams have been attempting to improve the accuracy by developing several options. Nanopolish (<a class="link link-uri" href="https://github.com/jts/nanopolish" target="_blank">https://github.com/jts/nanopolish</a>) is a software package for MinION data, which attempts to improve the read accuracy. One of the other options is a newly developed base-caller program, Scrappie. Using this program, a more precise estimation of the homopolymer lengths is enabled. Some other programs, which should be complementarily used for the original base-callers, are also under development. Indeed, the increasing noise-prone detection of SNVs with lower allelic frequencies is a concern that is not unique to MinION sequencing and is also an issue for Illumina sequencing, where one of the biggest advantages lies in its overwhelming sequencing depth. Further increasing the sequencing depth in MinION could, at least partially, address this issue.</p><p>In addition to the SNVs, other types of mutations were also detected. The PC-9 cells harbored a 15-base deletion in EGFR that was covered by 943 reads without mismatches of ±3 bp (<a reveal-id="dsx027-F2" data-open="dsx027-F2" class="link link-reveal link-table xref-fig">Fig. 2D</a>). No mutant reads were detected in the RERF-LC-Ad2 cells, in which the EGFR gene is not mutated. The precise identification and mutant allele frequency were validated using Illumina RNA-Seq (lower panel, <a reveal-id="dsx027-F2" data-open="dsx027-F2" class="link link-reveal link-table xref-fig">Fig. 2D</a>). We could also detect aberrantly spliced transcripts. We observed irregular alignments in the sequence reads of the NF1 gene (<em>NF1-ii</em>) in the PC-7 cells that covered a portion of the target length, leaving a long gap in the transcript. We further re-aligned the 3,755 reads to <em>NF1-ii</em> to obtain a split alignment.<a reveal-id="dsx027-B22" data-open="dsx027-B22" class="link link-ref link-reveal xref-bibr"><sup>22</sup></a> As shown in <a reveal-id="dsx027-F2" data-open="dsx027-F2" class="link link-reveal link-table xref-fig">Fig. 2E</a>, the split alignment revealed that aberrant exon skipping occurred in exon 19, which eliminated 74 bases from this gene.</p><p>These results showed that various categories of cancer mutations can be detected by MinION sequencing. All the driver aberrations were validated with Sanger sequencing (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S8</a></span>). Additionally, we conducted MinION sequencing of genomic DNA amplicons of the EGFR gene (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9</a></span> and Table S2). We could also detect mutations in genomic DNAs. However, the phasing analysis was challenging using the genomic DNA as a starting material because the mutations were occasionally separated by a long distance (e.g., 10.4 kb distance between EGFR T790M and L858R in the genome, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S9D</a></span>). Particularly, it is practically very difficult or almost impossible to detect the fusion transcripts in the genomic DNA rather compared to the cDNA amplicons. The junction points of the fusion genes are frequently located in a large segment of an intron. PCR should cover very long DNA fragments that include such junction points when genomic DNA was used as the starting material. The detailed analysis of the mutation phasing and fusion genes are described in a later section. In summary, we could detect the driver mutations in the cell lines as follows: four somatic SNVs (KRAS G12S in A549; NRAS Q61R in H2347 and EGFR T790M/L858R in H1975) (<a reveal-id="dsx027-T1" data-open="dsx027-T1" class="link link-reveal link-table xref-fig">Table 1</a>), a deletion (EGFR E746_A750del in PC-9) and splice aberration (NF1 exon 19 skipping in PC-7), which have also been reported in a previous short-read sequencing study.<a reveal-id="dsx027-B12" data-open="dsx027-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a><div content-id="dsx027-T1" class="table-modal"><div class="table-wrap"><div id="dsx027-T1" data-id="dsx027-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Summary of the driver mutations in the MinION sequencing data</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Mutation </th><th rowspan="2">Cell line </th><th rowspan="2">Status in Illumina RNA-Seq </th><th colspan="9">Thresholds of MinION; X (0.1 - 0.9)<hr /> </th></tr><tr><th>0.1 </th><th>0.2 </th><th>0.3 </th><th>0.4 </th><th>0.5 </th><th>0.6 </th><th>0.7 </th><th>0.8 </th><th>0.9 </th></tr></thead><tbody><tr><td rowspan="3">EGFR T790M </td><td>H1975 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>Unk </td><td>Unk </td></tr><tr><td>PC-9 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td>RERF-LC-Ad2 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td rowspan="3">EGFR L858R </td><td>H1975 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>Unk </td><td>Unk </td><td>Unk </td><td>Unk </td><td>Unk </td><td>Unk </td></tr><tr><td>PC-9 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td>RERF-LC-Ad2 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td rowspan="2">KRAS G12S </td><td>A549 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td></tr><tr><td>H2228 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td rowspan="2">NRAS Q61R </td><td>H2347 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td></tr><tr><td>H2228 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td colspan="2" rowspan="2"><strong>Concordance</strong> </td><td><strong>Precision</strong> </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td></tr><tr><td><strong>Recall</strong> </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>0.8 </td><td>0.8 </td><td>0.8 </td><td>0.8 </td><td>0.5 </td><td>0.5 </td></tr></tbody></table></div><div content-id="dsx027-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Mutation </th><th rowspan="2">Cell line </th><th rowspan="2">Status in Illumina RNA-Seq </th><th colspan="9">Thresholds of MinION; X (0.1 - 0.9)<hr /> </th></tr><tr><th>0.1 </th><th>0.2 </th><th>0.3 </th><th>0.4 </th><th>0.5 </th><th>0.6 </th><th>0.7 </th><th>0.8 </th><th>0.9 </th></tr></thead><tbody><tr><td rowspan="3">EGFR T790M </td><td>H1975 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>Unk </td><td>Unk </td></tr><tr><td>PC-9 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td>RERF-LC-Ad2 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td rowspan="3">EGFR L858R </td><td>H1975 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>Unk </td><td>Unk </td><td>Unk </td><td>Unk </td><td>Unk </td><td>Unk </td></tr><tr><td>PC-9 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td>RERF-LC-Ad2 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td rowspan="2">KRAS G12S </td><td>A549 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td></tr><tr><td>H2228 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td rowspan="2">NRAS Q61R </td><td>H2347 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td></tr><tr><td>H2228 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td colspan="2" rowspan="2"><strong>Concordance</strong> </td><td><strong>Precision</strong> </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td></tr><tr><td><strong>Recall</strong> </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>0.8 </td><td>0.8 </td><td>0.8 </td><td>0.8 </td><td>0.5 </td><td>0.5 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>By only considering the positions of the 4 driver mutations described above, there were no false positive detections.</p></div></div><div content-id="tblfn2"><div class="fn"><p>TP, true positive; TN, true negative; Unk, unknown (false negative).</p></div></div></div></div></div><div class="table-wrap"><div id="dsx027-T1" data-id="dsx027-T1" class="table-wrap-title"><div class="label -label">Table 1</div><div class="caption"><p>Summary of the driver mutations in the MinION sequencing data</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Mutation </th><th rowspan="2">Cell line </th><th rowspan="2">Status in Illumina RNA-Seq </th><th colspan="9">Thresholds of MinION; X (0.1 - 0.9)<hr /> </th></tr><tr><th>0.1 </th><th>0.2 </th><th>0.3 </th><th>0.4 </th><th>0.5 </th><th>0.6 </th><th>0.7 </th><th>0.8 </th><th>0.9 </th></tr></thead><tbody><tr><td rowspan="3">EGFR T790M </td><td>H1975 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>Unk </td><td>Unk </td></tr><tr><td>PC-9 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td>RERF-LC-Ad2 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td rowspan="3">EGFR L858R </td><td>H1975 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>Unk </td><td>Unk </td><td>Unk </td><td>Unk </td><td>Unk </td><td>Unk </td></tr><tr><td>PC-9 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td>RERF-LC-Ad2 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td rowspan="2">KRAS G12S </td><td>A549 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td></tr><tr><td>H2228 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td rowspan="2">NRAS Q61R </td><td>H2347 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td></tr><tr><td>H2228 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td colspan="2" rowspan="2"><strong>Concordance</strong> </td><td><strong>Precision</strong> </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td></tr><tr><td><strong>Recall</strong> </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>0.8 </td><td>0.8 </td><td>0.8 </td><td>0.8 </td><td>0.5 </td><td>0.5 </td></tr></tbody></table></div><div content-id="dsx027-T1" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Mutation </th><th rowspan="2">Cell line </th><th rowspan="2">Status in Illumina RNA-Seq </th><th colspan="9">Thresholds of MinION; X (0.1 - 0.9)<hr /> </th></tr><tr><th>0.1 </th><th>0.2 </th><th>0.3 </th><th>0.4 </th><th>0.5 </th><th>0.6 </th><th>0.7 </th><th>0.8 </th><th>0.9 </th></tr></thead><tbody><tr><td rowspan="3">EGFR T790M </td><td>H1975 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>Unk </td><td>Unk </td></tr><tr><td>PC-9 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td>RERF-LC-Ad2 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td rowspan="3">EGFR L858R </td><td>H1975 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>Unk </td><td>Unk </td><td>Unk </td><td>Unk </td><td>Unk </td><td>Unk </td></tr><tr><td>PC-9 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td>RERF-LC-Ad2 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td rowspan="2">KRAS G12S </td><td>A549 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td></tr><tr><td>H2228 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td rowspan="2">NRAS Q61R </td><td>H2347 </td><td>Positive </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td><td>TP </td></tr><tr><td>H2228 </td><td>Negative </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td><td>TN </td></tr><tr><td colspan="2" rowspan="2"><strong>Concordance</strong> </td><td><strong>Precision</strong> </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>1.0 </td></tr><tr><td><strong>Recall</strong> </td><td>1.0 </td><td>1.0 </td><td>1.0 </td><td>0.8 </td><td>0.8 </td><td>0.8 </td><td>0.8 </td><td>0.5 </td><td>0.5 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn1"><div class="fn"><p>By only considering the positions of the 4 driver mutations described above, there were no false positive detections.</p></div></div><div content-id="tblfn2"><div class="fn"><p>TP, true positive; TN, true negative; Unk, unknown (false negative).</p></div></div></div></div></p>                            <h3 scrollto-destination=104086258 id="104086258" class="section-title" >3.3. Sequencing junction points in fusion transcripts</h3>
<p>We also used MinION sequencing to identify fusion transcripts. First, we selected two representative oncogenic fusion transcripts, i.e., EML4-ALK in H2228 cells.<a reveal-id="dsx027-B25" data-open="dsx027-B25" class="link link-ref link-reveal xref-bibr"><sup>25</sup></a> and CCDC6-RET in LC2/ad cells.<a reveal-id="dsx027-B23" data-open="dsx027-B23" class="link link-ref link-reveal xref-bibr"><sup>23</sup></a><sup>,</sup><a reveal-id="dsx027-B24" data-open="dsx027-B24" class="link link-ref link-reveal xref-bibr"><sup>24</sup></a> We designed PCR primers allowing for the resulting amplicon to sufficiently cover the potential junction points in the fusion transcripts (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). A split alignment to the sequence reads was conducted using LAST, which generated 540 and 341 reads bridging EML4-ALK and CCDC6-RET, respectively (<a reveal-id="dsx027-F3" data-open="dsx027-F3" class="link link-reveal link-table xref-fig">Fig. 3A</a> and <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). We also sequenced five other fusion transcripts that were discovered in our previous RNA-Seq study of lung cancers<a reveal-id="dsx027-B12" data-open="dsx027-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S1</a></span>). Using only the short reads, we could not obtain any information other than their junction points. Using the long reads in MinION, we could analyze their entire structures and obtain the phasing information of the junction points and the neighboring mutations/variations. More than 150 reads bridged the fusion partners in all cases (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). These reads accounted for 63%, on average, of the reads aligned to either of the fusion partners. We further re-aligned the MinION reads to the detected fusion transcript sequences (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S7</a></span>). The aligned reads showed 82% sequence identity, and an average of 76% of the target regions were covered by the individual reads (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S10</a></span>). Among these, more than 100 reads covered ±50 bases of the junction point of each of the fusion genes. </p>                        <a id="104086260" scrollto-destination="104086260"></a>
<div id="dsx027-F3" data-id="dsx027-F3" class="fig fig-section"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/m_dsx027f3.png?Expires=2147483647&Signature=adEEXCJjWLP-qJKZWn2huPGGhZ7DiRpZATcHLj5QcU37xLUpvJ8YCiBbZCdmejPkn0S9r~kHTbJDe-rLHuqDojIkVbZT9ILClOWCRCEA4uu1IcM4ZQj6RN7ym8k1WstbclqkjGJiekV705bCVyE2kULS-YYig44Qr9N6FfaHr8ayKvCvzF5V8V2DtqEVs1F~92UEZiVpRkyPSvC1tQ0D4jY~FkOP37C2CrkYcMssuW5a1SZVbUMt2iD3jZkaoBjk0Ex3h~8BnXRE~uDJWhq45Pas9lIjrIAivonzherKxf8esathYChIJT73GZqRKB7SSz9rl-XjHUXKZa17CkS-0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequencing the fusion transcripts using the MinION reads.(A) Sequence depths of EML4-ALK and CCDC6-RET. The reads were split to both fusion partners with split alignment. (B) Allelic relevance between the SNP and the junction point of the EFHD1-UBR3 fusion transcript in the PC-9 cells. In the upper panel, the depths and base patterns of the MinION reads are shown in the EFHD1-UBR3 target region. The junction point is shown as a broken black line. One of the heterozygous SNPs in EFHD1, which is encircled with a red broken line and is referred to as G, co-occurred with the fusion junction in the MinION reads (left in the lower panel). This SNP was verified as heterozygous using Illumina RNA-Seq (right in the lower panel)." path-from-xml="dsx027f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086260/dsx027f3.tif" path-from-xml="dsx027f3.tif" target="_blank">View large</a><a section="104086260" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/dsx027f3.png?Expires=2147483647&Signature=tTDtyLl5YKAUhW2DmGTA1mmdlGCQKV3T9pfa4ChsKHaMKdvjWdQj9ptDa5QPa4xaayC8FILjbAnj3Fp-RoN7l6-rZfKlY5QyRpeClAI5Kyu~qPPKZLF4lcGG1SYN-9XJkcvLXRP1E7~i6nkbgWVayDjFseVSefdom1aJ-V2AlrGvDUfp9uuLLMeBDCm5ORRStTPYgjUbOjlkst57voe0L5RdMavXxlHafMwa1YNBR4n4Uk7THunrykirVOdHpcJQRM2Yrtk6G3X24UCZ40PJluDX~PFTllDRXa-A2G4ozkoX12rE07OQa5cX73nw3K7lQDQhvDNf8a9yqr6sWvggXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086260&ar=3896153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx027f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequencing the fusion transcripts using the MinION reads.(A) Sequence depths of EML4-ALK and CCDC6-RET. The reads were split to both fusion partners with split alignment. (B) Allelic relevance between the SNP and the junction point of the EFHD1-UBR3 fusion transcript in the PC-9 cells. In the upper panel, the depths and base patterns of the MinION reads are shown in the EFHD1-UBR3 target region. The junction point is shown as a broken black line. One of the heterozygous SNPs in EFHD1, which is encircled with a red broken line and is referred to as G, co-occurred with the fusion junction in the MinION reads (left in the lower panel). This SNP was verified as heterozygous using Illumina RNA-Seq (right in the lower panel).</p></div></div><div content-id="dsx027-F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/m_dsx027f3.png?Expires=2147483647&Signature=adEEXCJjWLP-qJKZWn2huPGGhZ7DiRpZATcHLj5QcU37xLUpvJ8YCiBbZCdmejPkn0S9r~kHTbJDe-rLHuqDojIkVbZT9ILClOWCRCEA4uu1IcM4ZQj6RN7ym8k1WstbclqkjGJiekV705bCVyE2kULS-YYig44Qr9N6FfaHr8ayKvCvzF5V8V2DtqEVs1F~92UEZiVpRkyPSvC1tQ0D4jY~FkOP37C2CrkYcMssuW5a1SZVbUMt2iD3jZkaoBjk0Ex3h~8BnXRE~uDJWhq45Pas9lIjrIAivonzherKxf8esathYChIJT73GZqRKB7SSz9rl-XjHUXKZa17CkS-0Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Sequencing the fusion transcripts using the MinION reads.(A) Sequence depths of EML4-ALK and CCDC6-RET. The reads were split to both fusion partners with split alignment. (B) Allelic relevance between the SNP and the junction point of the EFHD1-UBR3 fusion transcript in the PC-9 cells. In the upper panel, the depths and base patterns of the MinION reads are shown in the EFHD1-UBR3 target region. The junction point is shown as a broken black line. One of the heterozygous SNPs in EFHD1, which is encircled with a red broken line and is referred to as G, co-occurred with the fusion junction in the MinION reads (left in the lower panel). This SNP was verified as heterozygous using Illumina RNA-Seq (right in the lower panel)." path-from-xml="dsx027f3.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086260/dsx027f3.tif" path-from-xml="dsx027f3.tif" target="_blank">View large</a><a section="104086260" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/dsx027f3.png?Expires=2147483647&Signature=tTDtyLl5YKAUhW2DmGTA1mmdlGCQKV3T9pfa4ChsKHaMKdvjWdQj9ptDa5QPa4xaayC8FILjbAnj3Fp-RoN7l6-rZfKlY5QyRpeClAI5Kyu~qPPKZLF4lcGG1SYN-9XJkcvLXRP1E7~i6nkbgWVayDjFseVSefdom1aJ-V2AlrGvDUfp9uuLLMeBDCm5ORRStTPYgjUbOjlkst57voe0L5RdMavXxlHafMwa1YNBR4n4Uk7THunrykirVOdHpcJQRM2Yrtk6G3X24UCZ40PJluDX~PFTllDRXa-A2G4ozkoX12rE07OQa5cX73nw3K7lQDQhvDNf8a9yqr6sWvggXw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086260&ar=3896153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx027f3.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Sequencing the fusion transcripts using the MinION reads.(A) Sequence depths of EML4-ALK and CCDC6-RET. The reads were split to both fusion partners with split alignment. (B) Allelic relevance between the SNP and the junction point of the EFHD1-UBR3 fusion transcript in the PC-9 cells. In the upper panel, the depths and base patterns of the MinION reads are shown in the EFHD1-UBR3 target region. The junction point is shown as a broken black line. One of the heterozygous SNPs in EFHD1, which is encircled with a red broken line and is referred to as G, co-occurred with the fusion junction in the MinION reads (left in the lower panel). This SNP was verified as heterozygous using Illumina RNA-Seq (right in the lower panel).</p></div></div><p>RNA-Seq and whole-genome Illumina sequencing data published by a previous study<a reveal-id="dsx027-B12" data-open="dsx027-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> revealed that LC2/ad cells harbor a heterozygous SNP in CCDC6 near the junction point. This SNP was also detected as a homogeneous SNP using MinION, indicating that the gene fusion occurred in the same allele encoding this SNP. This SNP and the gene fusion appeared on the same Illumina reads because the distance between them was only seven bases; therefore, a particularly long MinION read was not necessary in this case. However, a heterozygous SNP (chr2:233498669, G/T) was identified in the fusion transcript EFHD1-UBR3 in PC-9 cells.<a reveal-id="dsx027-B12" data-open="dsx027-B12" class="link link-ref link-reveal xref-bibr"><sup>12</sup></a> In this case, the SNP was more than 300 bases from the fusion junction. The results of the MinION sequencing suggested that the gene fusion occurred at the G allele (reference allele of UCSC hg19) of this SNP. Such ‘phasing’ of multiple SNVs or SNVs with rearrangement points would be difficult using only Illumina short-read sequencing (<a reveal-id="dsx027-F3" data-open="dsx027-F3" class="link link-reveal link-table xref-fig">Fig. 3B</a>). All fusion junctions were validated by Sanger sequencing (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S11</a></span>).</p>                            <h3 scrollto-destination=104086262 id="104086262" class="section-title" >3.4. Phasing SNVs and evaluation of the detection limits</h3>
<p>Using the long-read sequencing of MinION, we attempted to phase multiple SNVs. We selected the targets as allelic associations of multiple SNVs, which should have direct clinical relevance such as sensitivity to the therapeutic anti-cancer drugs. The Illumina reads revealed that the H1975 cells harbored two mutations, i.e., T790M and L858R, in the EGFR kinase domain. While anti-cancer drugs targeting EGFR mutants, such as gefitinib and erlotinib, are effective for the L858R mutation, the T790M secondary mutation on the same allele, if exists, would suppress the effect of the drugs.<a reveal-id="dsx027-B31" data-open="dsx027-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a> The Illumina sequencing also showed that both mutations were heterozygous SNVs, and we sought to determine whether these SNVs were in the same allele. The MinION sequencing revealed that 71% of the aligned reads (4089/5754 in <em>EGFR-i</em>) covered the kinase domain. We used 677 reads without mismatches ±3 bp at both sites. As shown in <a reveal-id="dsx027-F4" data-open="dsx027-F4" class="link link-reveal link-table xref-fig">Fig. 4A</a>, 72% of the reads harbored both mutations and 22% of the reads showed neither of the mutations. These results suggested that these mutations occurred in the same allele and that the remaining allele was not mutated. These results were also validated using TA cloning followed by Sanger sequencing (<a reveal-id="dsx027-F4" data-open="dsx027-F4" class="link link-reveal link-table xref-fig">Fig. 4B</a>). Small fractions of both the MinION and Sanger reads suggested additional very minor and complicated mutation patterns. Notably, we did not detect any other positions, particularly, in the protein kinase domain. A recently reported mutation, i.e., C797S, is known to render cancers resistant to the third-generation EGFR tyrosine kinase inhibitor AZD9291, which is effective against cancers with double mutations at L858R and T790M.<a reveal-id="dsx027-B32" data-open="dsx027-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> For the prognosis of a possible novel drug resistance, it should be of remarkable clinical relevance to monitor the acquisition of novel mutations, which may occasionally scatter over a wide region beyond the reach of Illumina sequencing, in an allele-specific manner. </p>                        <a id="104086264" scrollto-destination="104086264"></a>
<div id="dsx027-F4" data-id="dsx027-F4" class="fig fig-section"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/m_dsx027f4.png?Expires=2147483647&Signature=HfNDosiyfWAYVkxACJdp2cJERLwMK1DJnmcZ7OS9Gf119CjavcQapSVBDT2W4KSjQQun6k~92IxNHOkyE1rMFCi051kwomB8u8BxFlIgTn0JA1O4psKY6cE3eKj6knZCDqP-0PVsOMi0yFdvVPxk6fCGcp5JGwL34yclTWByeIhHpXimDwJAM~GOBJOZ2S8ciBwOiM11ijM-kPkUZu6Ga~UTn8KOXZ6NMYHZJRW-7IwS0TqLbPjvXJ2Pa8EVBaAS6CPPxh2U4zaXsN2d9q~5SIN3FSciZ0Qi8kQSnd72xuDi3qZ45tiAJYS~oengsKfOMQ-v8CL0RZ9dV4kN-AIsBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phasing cancer mutations. (A) Phasing of the EGFR mutations T790M and L858R in H1975 cells. The number of MinION reads called for each of the SNV patterns. The MinION reads without mismatches ±3 bp of the positions of both SNVs. (B) T790M and L858R mutations using Illumina RNA-Seq and Sanger sequencing. Both SNVs were called as heterozygous mutations by Illumina RNA-Seq and direct Sanger sequencing in the upper and middle panels, respectively. The pattern and variant tag frequencies of both SNVs in each DNA molecule were validated with TA cloning, followed by Sanger sequencing (lower panel). (C) Variant allele frequencies (VAFs) in the five-point EGFR-mutant dilution series. The expected and observed VAFs of the phased double mutant (T790M/L858R) are shown in blue solid and dashed lines, respectively. The VAFs in the wild-type and other patterns are also represented in pink and gray, respectively." path-from-xml="dsx027f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086264/dsx027f4.tif" path-from-xml="dsx027f4.tif" target="_blank">View large</a><a section="104086264" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/dsx027f4.png?Expires=2147483647&Signature=u-YOdShlvMPfp1dCF51g1O6eGQHKIFrP~KkSDBabOKogQ2a3pOhNoHmz7lGwvpzfdP2S00EUc~o-Z8iQ3gRWiarewi~DZyZVe1f6oeJ1cldgol~-3cMPkxYI45QH8MVsbl9w~ZQsO4buhkcEDBcQukzo-l5nH6-AItwtf-MQiYkmzdVAAnyVLcxJC158sBQxuVpfafoNhbPz8wcGbimsaBDf3CnkHgyFyP~8RGdnV1WCvT3yLLe7aebh1y6Wl76X8-ytQ7Xb6Hez~fImJMbTbA0bf5hTPudWyMJLvExHjejdbRUt8LFLLRhUJJT4Ozq8ATZ1bTC7UDFmmM-vQvL6mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086264&ar=3896153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx027f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phasing cancer mutations. (A) Phasing of the EGFR mutations T790M and L858R in H1975 cells. The number of MinION reads called for each of the SNV patterns. The MinION reads without mismatches ±3 bp of the positions of both SNVs. (B) T790M and L858R mutations using Illumina RNA-Seq and Sanger sequencing. Both SNVs were called as heterozygous mutations by Illumina RNA-Seq and direct Sanger sequencing in the upper and middle panels, respectively. The pattern and variant tag frequencies of both SNVs in each DNA molecule were validated with TA cloning, followed by Sanger sequencing (lower panel). (C) Variant allele frequencies (VAFs) in the five-point EGFR-mutant dilution series. The expected and observed VAFs of the phased double mutant (T790M/L858R) are shown in blue solid and dashed lines, respectively. The VAFs in the wild-type and other patterns are also represented in pink and gray, respectively.</p></div></div><div content-id="dsx027-F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/m_dsx027f4.png?Expires=2147483647&Signature=HfNDosiyfWAYVkxACJdp2cJERLwMK1DJnmcZ7OS9Gf119CjavcQapSVBDT2W4KSjQQun6k~92IxNHOkyE1rMFCi051kwomB8u8BxFlIgTn0JA1O4psKY6cE3eKj6knZCDqP-0PVsOMi0yFdvVPxk6fCGcp5JGwL34yclTWByeIhHpXimDwJAM~GOBJOZ2S8ciBwOiM11ijM-kPkUZu6Ga~UTn8KOXZ6NMYHZJRW-7IwS0TqLbPjvXJ2Pa8EVBaAS6CPPxh2U4zaXsN2d9q~5SIN3FSciZ0Qi8kQSnd72xuDi3qZ45tiAJYS~oengsKfOMQ-v8CL0RZ9dV4kN-AIsBw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Phasing cancer mutations. (A) Phasing of the EGFR mutations T790M and L858R in H1975 cells. The number of MinION reads called for each of the SNV patterns. The MinION reads without mismatches ±3 bp of the positions of both SNVs. (B) T790M and L858R mutations using Illumina RNA-Seq and Sanger sequencing. Both SNVs were called as heterozygous mutations by Illumina RNA-Seq and direct Sanger sequencing in the upper and middle panels, respectively. The pattern and variant tag frequencies of both SNVs in each DNA molecule were validated with TA cloning, followed by Sanger sequencing (lower panel). (C) Variant allele frequencies (VAFs) in the five-point EGFR-mutant dilution series. The expected and observed VAFs of the phased double mutant (T790M/L858R) are shown in blue solid and dashed lines, respectively. The VAFs in the wild-type and other patterns are also represented in pink and gray, respectively." path-from-xml="dsx027f4.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086264/dsx027f4.tif" path-from-xml="dsx027f4.tif" target="_blank">View large</a><a section="104086264" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/dsx027f4.png?Expires=2147483647&Signature=u-YOdShlvMPfp1dCF51g1O6eGQHKIFrP~KkSDBabOKogQ2a3pOhNoHmz7lGwvpzfdP2S00EUc~o-Z8iQ3gRWiarewi~DZyZVe1f6oeJ1cldgol~-3cMPkxYI45QH8MVsbl9w~ZQsO4buhkcEDBcQukzo-l5nH6-AItwtf-MQiYkmzdVAAnyVLcxJC158sBQxuVpfafoNhbPz8wcGbimsaBDf3CnkHgyFyP~8RGdnV1WCvT3yLLe7aebh1y6Wl76X8-ytQ7Xb6Hez~fImJMbTbA0bf5hTPudWyMJLvExHjejdbRUt8LFLLRhUJJT4Ozq8ATZ1bTC7UDFmmM-vQvL6mA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086264&ar=3896153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx027f4.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Phasing cancer mutations. (A) Phasing of the EGFR mutations T790M and L858R in H1975 cells. The number of MinION reads called for each of the SNV patterns. The MinION reads without mismatches ±3 bp of the positions of both SNVs. (B) T790M and L858R mutations using Illumina RNA-Seq and Sanger sequencing. Both SNVs were called as heterozygous mutations by Illumina RNA-Seq and direct Sanger sequencing in the upper and middle panels, respectively. The pattern and variant tag frequencies of both SNVs in each DNA molecule were validated with TA cloning, followed by Sanger sequencing (lower panel). (C) Variant allele frequencies (VAFs) in the five-point EGFR-mutant dilution series. The expected and observed VAFs of the phased double mutant (T790M/L858R) are shown in blue solid and dashed lines, respectively. The VAFs in the wild-type and other patterns are also represented in pink and gray, respectively.</p></div></div><p>Before initiating clinical applications of MinION sequencing, we further investigated whether mutations present at low levels could be detected and phased. This analysis was performed to ensure that the clinical requirements can be met if the cancer cell population is occasionally low in a clinical sample. Using serially diluted samples containing mixtures of the EGFR mutant (H1975) and wild-type (RERF-LC-Ad2) cell lines (mutant:wild type = 1:99, 1:19, 1:9, 1:4 and 1:1), we assessed the number of reads required to obtain sufficient mutant reads and determined the precision at which mutations can be detected and phased in the dilution series (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S8</a></span>). The mutations with the respective frequencies could be detected and phased at almost the expected rates (<a reveal-id="dsx027-F4" data-open="dsx027-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>). Indeed, we found that several percent of the mutants were dropped in our dilution analysis. It may be still difficult to detect mutations in very minor population (less than several percent) using MinION. However, the sequencing yields and quality of the MinION data have been continuously increasing; therefore, this ‘drop’ problem, which is represented by the gray-colored area in <a reveal-id="dsx027-F4" data-open="dsx027-F4" class="link link-reveal link-table xref-fig">Fig. 4C</a>, will become less significant. Indeed, we found the error rates were significantly decreased in the newer version of the flow cell (R9). Using the R9 flow cell, 23,640 2D pass reads were aligned and the reads showed an average of 88.4% sequence identity (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S12</a></span>). More than 45% of the aligned reads harbored more than 90% sequence identity.</p>                            <h3 scrollto-destination=104086266 id="104086266" class="section-title" >3.5. Sequencing clinical samples using MinION</h3>
<p>Finally, for clinical applications, we performed MinION sequencing using cDNA amplicons prepared from eight Japanese lung adenocarcinoma patients (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>). Six patients harbored EGFR or KRAS mutations. To detect the driver mutations in the EGFR and KRAS genes, we amplified target regions of approximately 1 kb that covered the mutational hotspot. Three patients harbored KRAS G12 mutations (G12V in one patient and G12C in two patients, <a reveal-id="dsx027-F5" data-open="dsx027-F5" class="link link-reveal link-table xref-fig">Fig. 5A</a>) and three other patients harbored deletions in exon 19 of EGFR (L747_T751del in two patients and L747_P753delinsS in one patient, <a reveal-id="dsx027-F5" data-open="dsx027-F5" class="link link-reveal link-table xref-fig">Fig. 5B</a>). These mutations were further confirmed by Sanger sequencing (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13</a></span>). The variant allele frequencies were calculated by MinION using only reads without any mismatches in the 3-bp region surrounding the mutations, which is consistent with the cell line analysis. The results highly correlated with those obtained by the Sanger sequencing. </p>                        <a id="104086268" scrollto-destination="104086268"></a>
<div id="dsx027-F5" data-id="dsx027-F5" class="fig fig-section"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/m_dsx027f5.png?Expires=2147483647&Signature=niCchKsk94wkgk7in1bGVrkjmI7ZkzjfU91Trqz62nP2TnGrLdKQ9k3r8-QwaPkheCYY9amQMC3HPTDTe~pyZFN4vi6lllWXtLBuPx9TaSnOqNDGEjXnI34kuqSPVUBNeAA3dBESCyql3zYE217GZLV8bvndYVS6DrEtARORjePazrbHv3xhuox7jpvaTezUgBMYsscXmnvJ2ie7lGt6eq-UrR4AvunYaaFZLeIjOtxDdDjtGx~ZWkgFjREoI-tluIO9JQ-QQtf1cUKLZF9QQAGE2RfBW-lvcftGmusBNTPuIE9Nlzg5hKAo1oj7W-4qBoD--KT5kIeM4v2V~shIMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of driver alterations in clinical samples. (A, B) Detection of driver alterations in clinical samples using MinION sequencing. KRAS (A) and EGFR (B) mutations are shown for the mutation-positive patients. The PCR target regions are shown in the upper panel. The pre-cleaned and cleaned (without mismatches ±3 bp of the mutation) depths are shown in the middle and lower panel. (C, D) Sequence depths of split alignment for EML4-ALK (C) and KIF5B-RET (D)." path-from-xml="dsx027f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086268/dsx027f5.tif" path-from-xml="dsx027f5.tif" target="_blank">View large</a><a section="104086268" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/dsx027f5.png?Expires=2147483647&Signature=YGTpbzMBleAi2lagTgRbkyoBQvQBzzEufYgkqfU1vO1A~a88lbOLnwIjf1iIbT-n7KVbGqh-c18MPqFL0rFpKiBFt3q6D3vWwseTqGCSmjr5wiFSIC-kWPqWgCdt12Sdu4Ahm-UOZwTUycXfLTcsQCA~OymjD-IodqV-dUo5-KMXouCxfUh8sI~ZTG4g9Xb0fzt6oqJtp6gIY5LqXgusMmC5RZOHr7RS-suj0P~HdK2q8bAoACHrBEkdGSE19hMXsVTAkxZxlV1kAt6IJNjtn-YOBSH1sK7NZjhkr7mp8bCIW5rae~tJgrfLElBvv9zox5g9l-e3e4LzA5pYGox3xw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086268&ar=3896153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx027f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of driver alterations in clinical samples. (A, B) Detection of driver alterations in clinical samples using MinION sequencing. KRAS (A) and EGFR (B) mutations are shown for the mutation-positive patients. The PCR target regions are shown in the upper panel. The pre-cleaned and cleaned (without mismatches ±3 bp of the mutation) depths are shown in the middle and lower panel. (C, D) Sequence depths of split alignment for EML4-ALK (C) and KIF5B-RET (D).</p></div></div><div content-id="dsx027-F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/m_dsx027f5.png?Expires=2147483647&Signature=niCchKsk94wkgk7in1bGVrkjmI7ZkzjfU91Trqz62nP2TnGrLdKQ9k3r8-QwaPkheCYY9amQMC3HPTDTe~pyZFN4vi6lllWXtLBuPx9TaSnOqNDGEjXnI34kuqSPVUBNeAA3dBESCyql3zYE217GZLV8bvndYVS6DrEtARORjePazrbHv3xhuox7jpvaTezUgBMYsscXmnvJ2ie7lGt6eq-UrR4AvunYaaFZLeIjOtxDdDjtGx~ZWkgFjREoI-tluIO9JQ-QQtf1cUKLZF9QQAGE2RfBW-lvcftGmusBNTPuIE9Nlzg5hKAo1oj7W-4qBoD--KT5kIeM4v2V~shIMA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Detection of driver alterations in clinical samples. (A, B) Detection of driver alterations in clinical samples using MinION sequencing. KRAS (A) and EGFR (B) mutations are shown for the mutation-positive patients. The PCR target regions are shown in the upper panel. The pre-cleaned and cleaned (without mismatches ±3 bp of the mutation) depths are shown in the middle and lower panel. (C, D) Sequence depths of split alignment for EML4-ALK (C) and KIF5B-RET (D)." path-from-xml="dsx027f5.tif" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/104086268/dsx027f5.tif" path-from-xml="dsx027f5.tif" target="_blank">View large</a><a section="104086268" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/dsx027f5.png?Expires=2147483647&Signature=YGTpbzMBleAi2lagTgRbkyoBQvQBzzEufYgkqfU1vO1A~a88lbOLnwIjf1iIbT-n7KVbGqh-c18MPqFL0rFpKiBFt3q6D3vWwseTqGCSmjr5wiFSIC-kWPqWgCdt12Sdu4Ahm-UOZwTUycXfLTcsQCA~OymjD-IodqV-dUo5-KMXouCxfUh8sI~ZTG4g9Xb0fzt6oqJtp6gIY5LqXgusMmC5RZOHr7RS-suj0P~HdK2q8bAoACHrBEkdGSE19hMXsVTAkxZxlV1kAt6IJNjtn-YOBSH1sK7NZjhkr7mp8bCIW5rae~tJgrfLElBvv9zox5g9l-e3e4LzA5pYGox3xw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=104086268&ar=3896153&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dsx027f5.tif" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Detection of driver alterations in clinical samples. (A, B) Detection of driver alterations in clinical samples using MinION sequencing. KRAS (A) and EGFR (B) mutations are shown for the mutation-positive patients. The PCR target regions are shown in the upper panel. The pre-cleaned and cleaned (without mismatches ±3 bp of the mutation) depths are shown in the middle and lower panel. (C, D) Sequence depths of split alignment for EML4-ALK (C) and KIF5B-RET (D).</p></div></div><p>The other two patients carried the EML4-ALK and KIF5B-RET gene fusions. By split alignment using LAST, 1,219 and 26,023 reads were aligned to EML4-ALK and KIF5B-RET (<a reveal-id="dsx027-F5" data-open="dsx027-F5" class="link link-reveal link-table xref-fig">Fig. 5C and D</a>), respectively, in these patients. Using previously reported gene fusion variants as references, we determined that EML4-ALK is variant E18;A20,<a reveal-id="dsx027-B4" data-open="dsx027-B4" class="link link-ref link-reveal xref-bibr"><sup>4</sup></a><sup>,</sup><a reveal-id="dsx027-B33" data-open="dsx027-B33" class="link link-ref link-reveal xref-bibr"><sup>33</sup></a> and KIF5B-RET is variant K23;R12.<a reveal-id="dsx027-B13" data-open="dsx027-B13" class="link link-ref link-reveal xref-bibr"><sup>13</sup></a> The precise detection of the junction points was confirmed by Sanger sequencing (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S13D</a></span>). We also aligned the MinION reads to the fused transcripts to further examine the sequencing accuracy and junction coverage. The reads showed 84% sequence identity on average, and 79% and 69% of the reads, respectively, aligned to the ALK and RET fusions covered the junction points of these gene fusions (±50 bp, <span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9A</a></span>). We also compared the consensus sequences of the MinION reads with those of the Illumina RNA-Seq reads. The overall concordances were consistent with those obtained from the cell line analysis (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Table S9B</a></span>).</p><p>Furthermore, we conducted a phasing analysis of a short deletion and a heterozygous SNP in EGFR in one patient with EGFR mutations (<span class="content-section supplementary-material"><a path-from-xml="sup1" href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">Supplementary Fig. S14</a></span>). In total, 88% of the reads showed two major patterns of allelic associations. Based on these results, we concluded that the precise detection of driver mutations by sequencing was also achieved using the clinical samples. The MinION sequencing data of the clinical samples are summarized in <a reveal-id="dsx027-T2" data-open="dsx027-T2" class="link link-reveal link-table xref-fig">Table 2</a>. <div content-id="dsx027-T2" class="table-modal"><div class="table-wrap"><div id="dsx027-T2" data-id="dsx027-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Summary of the MinION sequencing data of clinical samples</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Run </th><th rowspan="2">Sample ID </th><th rowspan="2">Target gene </th><th rowspan="2">Avg. QV </th><th rowspan="2">Avg. read length </th><th colspan="4">Number of 2D reads<hr /> </th><th rowspan="2">Avg. identity </th><th rowspan="2">Avg. target cover rate </th><th rowspan="2">Driver mutation (%VAF) </th></tr><tr><th>Total </th><th>Aligned </th><th colspan="2">On-target </th></tr></thead><tbody><tr><td rowspan="2">#C1 </td><td>LUAD001 </td><td>EML4-ALK </td><td rowspan="2">10.6 </td><td rowspan="2">916 </td><td rowspan="2">59,810 </td><td rowspan="2">51,967 (87%) </td><td rowspan="2">47,290 (79%) </td><td>2,296 </td><td rowspan="2">84%<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="2">0.65<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>E18:A20 </td></tr><tr><td>LUAD002 </td><td>KIF5B-RET </td><td>44,994 </td><td>K23;R12 </td></tr><tr><td rowspan="2">#C2 </td><td rowspan="2">LUAD003 </td><td>EGFR </td><td rowspan="2">9.8 </td><td rowspan="2">754 </td><td rowspan="2">144,229 </td><td rowspan="2">126,038 (87%) </td><td rowspan="2">115,999 (80%) </td><td>88,751 </td><td rowspan="2">85% </td><td rowspan="2">0.74 </td><td>WT </td></tr><tr><td>KRAS </td><td>27,248 </td><td>G12V (61%) </td></tr><tr><td rowspan="2">#C3 </td><td rowspan="2">LUAD004 </td><td>EGFR </td><td rowspan="2">9.2 </td><td rowspan="2">600 </td><td rowspan="2">63,112 </td><td rowspan="2">50,881 (81%) </td><td rowspan="2">45,062 (71%) </td><td>33,177 </td><td rowspan="2">84% </td><td rowspan="2">0.59 </td><td>WT </td></tr><tr><td>KRAS </td><td>11,885 </td><td>G12C (40%) </td></tr><tr><td rowspan="2">#C4 </td><td rowspan="2">LUAD005 </td><td>EGFR </td><td rowspan="2">9.6 </td><td rowspan="2">672 </td><td rowspan="2">36,535 </td><td rowspan="2">28,359 (78%) </td><td rowspan="2">19,586 (54%) </td><td>5,895 </td><td rowspan="2">84% </td><td rowspan="2">0.76 </td><td>L747_T751del (42%) </td></tr><tr><td>KRAS </td><td>13,691 </td><td>WT </td></tr><tr><td rowspan="2">#C5 </td><td rowspan="2">LUAD006 </td><td>EGFR </td><td rowspan="2">9,3 </td><td rowspan="2">516 </td><td rowspan="2">64,566 </td><td rowspan="2">43,469 (67%) </td><td rowspan="2">31,768 (49%) </td><td>20,498 </td><td rowspan="2">85% </td><td rowspan="2">0.58 </td><td>L747_T751del (53%) </td></tr><tr><td>KRAS </td><td>11,270 </td><td>WT </td></tr><tr><td rowspan="2">#C6 </td><td rowspan="2">LUAD007 </td><td>EGFR </td><td rowspan="2">9.3 </td><td rowspan="2">631 </td><td rowspan="2">41,537 </td><td rowspan="2">32,207 (78%) </td><td rowspan="2">27,910 (67%) </td><td>21,141 </td><td rowspan="2">84% </td><td rowspan="2">0.63 </td><td>L747_P753delinsS (86%) </td></tr><tr><td>KRAS </td><td>6,769 </td><td>WT </td></tr><tr><td rowspan="2">#C7 </td><td rowspan="2">LUAD008 </td><td>EGFR </td><td rowspan="2">9.4 </td><td rowspan="2">827 </td><td rowspan="2">98,312 </td><td rowspan="2">77,847 (79%) </td><td rowspan="2">65,238 (66%) </td><td>51,805 </td><td rowspan="2">82% </td><td rowspan="2">0.82 </td><td>WT </td></tr><tr><td>KRAS </td><td>13,433 </td><td>G12C (69%) </td></tr></tbody></table></div><div content-id="dsx027-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Run </th><th rowspan="2">Sample ID </th><th rowspan="2">Target gene </th><th rowspan="2">Avg. QV </th><th rowspan="2">Avg. read length </th><th colspan="4">Number of 2D reads<hr /> </th><th rowspan="2">Avg. identity </th><th rowspan="2">Avg. target cover rate </th><th rowspan="2">Driver mutation (%VAF) </th></tr><tr><th>Total </th><th>Aligned </th><th colspan="2">On-target </th></tr></thead><tbody><tr><td rowspan="2">#C1 </td><td>LUAD001 </td><td>EML4-ALK </td><td rowspan="2">10.6 </td><td rowspan="2">916 </td><td rowspan="2">59,810 </td><td rowspan="2">51,967 (87%) </td><td rowspan="2">47,290 (79%) </td><td>2,296 </td><td rowspan="2">84%<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="2">0.65<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>E18:A20 </td></tr><tr><td>LUAD002 </td><td>KIF5B-RET </td><td>44,994 </td><td>K23;R12 </td></tr><tr><td rowspan="2">#C2 </td><td rowspan="2">LUAD003 </td><td>EGFR </td><td rowspan="2">9.8 </td><td rowspan="2">754 </td><td rowspan="2">144,229 </td><td rowspan="2">126,038 (87%) </td><td rowspan="2">115,999 (80%) </td><td>88,751 </td><td rowspan="2">85% </td><td rowspan="2">0.74 </td><td>WT </td></tr><tr><td>KRAS </td><td>27,248 </td><td>G12V (61%) </td></tr><tr><td rowspan="2">#C3 </td><td rowspan="2">LUAD004 </td><td>EGFR </td><td rowspan="2">9.2 </td><td rowspan="2">600 </td><td rowspan="2">63,112 </td><td rowspan="2">50,881 (81%) </td><td rowspan="2">45,062 (71%) </td><td>33,177 </td><td rowspan="2">84% </td><td rowspan="2">0.59 </td><td>WT </td></tr><tr><td>KRAS </td><td>11,885 </td><td>G12C (40%) </td></tr><tr><td rowspan="2">#C4 </td><td rowspan="2">LUAD005 </td><td>EGFR </td><td rowspan="2">9.6 </td><td rowspan="2">672 </td><td rowspan="2">36,535 </td><td rowspan="2">28,359 (78%) </td><td rowspan="2">19,586 (54%) </td><td>5,895 </td><td rowspan="2">84% </td><td rowspan="2">0.76 </td><td>L747_T751del (42%) </td></tr><tr><td>KRAS </td><td>13,691 </td><td>WT </td></tr><tr><td rowspan="2">#C5 </td><td rowspan="2">LUAD006 </td><td>EGFR </td><td rowspan="2">9,3 </td><td rowspan="2">516 </td><td rowspan="2">64,566 </td><td rowspan="2">43,469 (67%) </td><td rowspan="2">31,768 (49%) </td><td>20,498 </td><td rowspan="2">85% </td><td rowspan="2">0.58 </td><td>L747_T751del (53%) </td></tr><tr><td>KRAS </td><td>11,270 </td><td>WT </td></tr><tr><td rowspan="2">#C6 </td><td rowspan="2">LUAD007 </td><td>EGFR </td><td rowspan="2">9.3 </td><td rowspan="2">631 </td><td rowspan="2">41,537 </td><td rowspan="2">32,207 (78%) </td><td rowspan="2">27,910 (67%) </td><td>21,141 </td><td rowspan="2">84% </td><td rowspan="2">0.63 </td><td>L747_P753delinsS (86%) </td></tr><tr><td>KRAS </td><td>6,769 </td><td>WT </td></tr><tr><td rowspan="2">#C7 </td><td rowspan="2">LUAD008 </td><td>EGFR </td><td rowspan="2">9.4 </td><td rowspan="2">827 </td><td rowspan="2">98,312 </td><td rowspan="2">77,847 (79%) </td><td rowspan="2">65,238 (66%) </td><td>51,805 </td><td rowspan="2">82% </td><td rowspan="2">0.82 </td><td>WT </td></tr><tr><td>KRAS </td><td>13,433 </td><td>G12C (69%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p><sup>a</sup>Reads aligned to the fusion RNA were used in the calculations.</p></div></div></div></div></div><div class="table-wrap"><div id="dsx027-T2" data-id="dsx027-T2" class="table-wrap-title"><div class="label -label">Table 2</div><div class="caption"><p>Summary of the MinION sequencing data of clinical samples</p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th rowspan="2">Run </th><th rowspan="2">Sample ID </th><th rowspan="2">Target gene </th><th rowspan="2">Avg. QV </th><th rowspan="2">Avg. read length </th><th colspan="4">Number of 2D reads<hr /> </th><th rowspan="2">Avg. identity </th><th rowspan="2">Avg. target cover rate </th><th rowspan="2">Driver mutation (%VAF) </th></tr><tr><th>Total </th><th>Aligned </th><th colspan="2">On-target </th></tr></thead><tbody><tr><td rowspan="2">#C1 </td><td>LUAD001 </td><td>EML4-ALK </td><td rowspan="2">10.6 </td><td rowspan="2">916 </td><td rowspan="2">59,810 </td><td rowspan="2">51,967 (87%) </td><td rowspan="2">47,290 (79%) </td><td>2,296 </td><td rowspan="2">84%<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="2">0.65<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>E18:A20 </td></tr><tr><td>LUAD002 </td><td>KIF5B-RET </td><td>44,994 </td><td>K23;R12 </td></tr><tr><td rowspan="2">#C2 </td><td rowspan="2">LUAD003 </td><td>EGFR </td><td rowspan="2">9.8 </td><td rowspan="2">754 </td><td rowspan="2">144,229 </td><td rowspan="2">126,038 (87%) </td><td rowspan="2">115,999 (80%) </td><td>88,751 </td><td rowspan="2">85% </td><td rowspan="2">0.74 </td><td>WT </td></tr><tr><td>KRAS </td><td>27,248 </td><td>G12V (61%) </td></tr><tr><td rowspan="2">#C3 </td><td rowspan="2">LUAD004 </td><td>EGFR </td><td rowspan="2">9.2 </td><td rowspan="2">600 </td><td rowspan="2">63,112 </td><td rowspan="2">50,881 (81%) </td><td rowspan="2">45,062 (71%) </td><td>33,177 </td><td rowspan="2">84% </td><td rowspan="2">0.59 </td><td>WT </td></tr><tr><td>KRAS </td><td>11,885 </td><td>G12C (40%) </td></tr><tr><td rowspan="2">#C4 </td><td rowspan="2">LUAD005 </td><td>EGFR </td><td rowspan="2">9.6 </td><td rowspan="2">672 </td><td rowspan="2">36,535 </td><td rowspan="2">28,359 (78%) </td><td rowspan="2">19,586 (54%) </td><td>5,895 </td><td rowspan="2">84% </td><td rowspan="2">0.76 </td><td>L747_T751del (42%) </td></tr><tr><td>KRAS </td><td>13,691 </td><td>WT </td></tr><tr><td rowspan="2">#C5 </td><td rowspan="2">LUAD006 </td><td>EGFR </td><td rowspan="2">9,3 </td><td rowspan="2">516 </td><td rowspan="2">64,566 </td><td rowspan="2">43,469 (67%) </td><td rowspan="2">31,768 (49%) </td><td>20,498 </td><td rowspan="2">85% </td><td rowspan="2">0.58 </td><td>L747_T751del (53%) </td></tr><tr><td>KRAS </td><td>11,270 </td><td>WT </td></tr><tr><td rowspan="2">#C6 </td><td rowspan="2">LUAD007 </td><td>EGFR </td><td rowspan="2">9.3 </td><td rowspan="2">631 </td><td rowspan="2">41,537 </td><td rowspan="2">32,207 (78%) </td><td rowspan="2">27,910 (67%) </td><td>21,141 </td><td rowspan="2">84% </td><td rowspan="2">0.63 </td><td>L747_P753delinsS (86%) </td></tr><tr><td>KRAS </td><td>6,769 </td><td>WT </td></tr><tr><td rowspan="2">#C7 </td><td rowspan="2">LUAD008 </td><td>EGFR </td><td rowspan="2">9.4 </td><td rowspan="2">827 </td><td rowspan="2">98,312 </td><td rowspan="2">77,847 (79%) </td><td rowspan="2">65,238 (66%) </td><td>51,805 </td><td rowspan="2">82% </td><td rowspan="2">0.82 </td><td>WT </td></tr><tr><td>KRAS </td><td>13,433 </td><td>G12C (69%) </td></tr></tbody></table></div><div content-id="dsx027-T2" class="table-modal"><table><thead name="thead" align=""><tr><th rowspan="2">Run </th><th rowspan="2">Sample ID </th><th rowspan="2">Target gene </th><th rowspan="2">Avg. QV </th><th rowspan="2">Avg. read length </th><th colspan="4">Number of 2D reads<hr /> </th><th rowspan="2">Avg. identity </th><th rowspan="2">Avg. target cover rate </th><th rowspan="2">Driver mutation (%VAF) </th></tr><tr><th>Total </th><th>Aligned </th><th colspan="2">On-target </th></tr></thead><tbody><tr><td rowspan="2">#C1 </td><td>LUAD001 </td><td>EML4-ALK </td><td rowspan="2">10.6 </td><td rowspan="2">916 </td><td rowspan="2">59,810 </td><td rowspan="2">51,967 (87%) </td><td rowspan="2">47,290 (79%) </td><td>2,296 </td><td rowspan="2">84%<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td rowspan="2">0.65<a reveal-id="tblfn3" data-open="tblfn3" class="link link-ref link-reveal xref-fn"><sup>a</sup></a> </td><td>E18:A20 </td></tr><tr><td>LUAD002 </td><td>KIF5B-RET </td><td>44,994 </td><td>K23;R12 </td></tr><tr><td rowspan="2">#C2 </td><td rowspan="2">LUAD003 </td><td>EGFR </td><td rowspan="2">9.8 </td><td rowspan="2">754 </td><td rowspan="2">144,229 </td><td rowspan="2">126,038 (87%) </td><td rowspan="2">115,999 (80%) </td><td>88,751 </td><td rowspan="2">85% </td><td rowspan="2">0.74 </td><td>WT </td></tr><tr><td>KRAS </td><td>27,248 </td><td>G12V (61%) </td></tr><tr><td rowspan="2">#C3 </td><td rowspan="2">LUAD004 </td><td>EGFR </td><td rowspan="2">9.2 </td><td rowspan="2">600 </td><td rowspan="2">63,112 </td><td rowspan="2">50,881 (81%) </td><td rowspan="2">45,062 (71%) </td><td>33,177 </td><td rowspan="2">84% </td><td rowspan="2">0.59 </td><td>WT </td></tr><tr><td>KRAS </td><td>11,885 </td><td>G12C (40%) </td></tr><tr><td rowspan="2">#C4 </td><td rowspan="2">LUAD005 </td><td>EGFR </td><td rowspan="2">9.6 </td><td rowspan="2">672 </td><td rowspan="2">36,535 </td><td rowspan="2">28,359 (78%) </td><td rowspan="2">19,586 (54%) </td><td>5,895 </td><td rowspan="2">84% </td><td rowspan="2">0.76 </td><td>L747_T751del (42%) </td></tr><tr><td>KRAS </td><td>13,691 </td><td>WT </td></tr><tr><td rowspan="2">#C5 </td><td rowspan="2">LUAD006 </td><td>EGFR </td><td rowspan="2">9,3 </td><td rowspan="2">516 </td><td rowspan="2">64,566 </td><td rowspan="2">43,469 (67%) </td><td rowspan="2">31,768 (49%) </td><td>20,498 </td><td rowspan="2">85% </td><td rowspan="2">0.58 </td><td>L747_T751del (53%) </td></tr><tr><td>KRAS </td><td>11,270 </td><td>WT </td></tr><tr><td rowspan="2">#C6 </td><td rowspan="2">LUAD007 </td><td>EGFR </td><td rowspan="2">9.3 </td><td rowspan="2">631 </td><td rowspan="2">41,537 </td><td rowspan="2">32,207 (78%) </td><td rowspan="2">27,910 (67%) </td><td>21,141 </td><td rowspan="2">84% </td><td rowspan="2">0.63 </td><td>L747_P753delinsS (86%) </td></tr><tr><td>KRAS </td><td>6,769 </td><td>WT </td></tr><tr><td rowspan="2">#C7 </td><td rowspan="2">LUAD008 </td><td>EGFR </td><td rowspan="2">9.4 </td><td rowspan="2">827 </td><td rowspan="2">98,312 </td><td rowspan="2">77,847 (79%) </td><td rowspan="2">65,238 (66%) </td><td>51,805 </td><td rowspan="2">82% </td><td rowspan="2">0.82 </td><td>WT </td></tr><tr><td>KRAS </td><td>13,433 </td><td>G12C (69%) </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id="tblfn3"><div class="fn"><p><sup>a</sup>Reads aligned to the fusion RNA were used in the calculations.</p></div></div></div></div></p>                            <h2 scrollto-destination=104086271 id="104086271" class="section-title" >4. Discussion</h2>
<p>In this study, we used a convenient, long-read MinION sequencer for mutation detection and phasing in cancers. We successfully applied the developed approach to identify cancerous mutations in first cultured cell models and then in clinical samples. Despite the error-prone nature of the sequence data of MinION, in the case of homozygous mutant alleles, the cancerous mutations could be robustly detected. We observed that the minor mutant alleles were occasionally difficult to detect depending on the allele frequency. The detection of such a minor mutant allele is often important for many clinical cancers. Tumor cells are evolutionally diverse with genetic heterogeneity within the population and samples are occasionally mixed with normal cells. In the current study, we demonstrate that additional filtration of the data considering ±3 bp of the SNVs is useful for lowering the error rate and, thus, for increasing the detection limit. Further bioinformatics tools could address these concerns of accuracy and sensitivity. Moreover, we expect that the drawbacks of MinION should be lessened by rapid improvements in the sequencing technology.</p><p>One of the characteristics of MinION is its convenience for use, including its portability and easy settings for the library preparation and sequencing. Although second-generation sequencers have enabled us to easily conduct genotyping in clinical samples, we need to develop simple and cost-effective procedures to identify driver genes in each patient for personalized medicine for various kinds of cancers, particularly lung adenocarcinoma. In our study, we could detect the major driver genes, which have diverse patterns, including point mutations and fusions, using MinION. In lung adenocarcinoma, a number of molecular-targeting medicines are available, such as gefitinib, erlotinib and afatinib for EGFR;<a reveal-id="dsx027-B34" data-open="dsx027-B34" class="link link-ref link-reveal xref-bibr"><sup>34</sup></a> crizotinib, ceritinib and alectinib for ALK;<a reveal-id="dsx027-B35" data-open="dsx027-B35" class="link link-ref link-reveal xref-bibr"><sup>35</sup></a> and vandetanib and cabozantinib for RET.<a reveal-id="dsx027-B36" data-open="dsx027-B36" class="link link-ref link-reveal xref-bibr"><sup>36</sup></a> The detection of KRAS mutations is also required because a large portion of patients harbor these mutations, but there are no effective anti-cancer drugs targeting KRAS. The simple methods of MinION sequencing could possibly enable small/mid-scale research centers and hospitals to conduct research studies by genotyping driver genes and selecting suitable therapeutic approaches.</p><p>Following the initial success of this methodological development, the obvious next goal is to expand this approach to other cancers. Thus, additional primer sets should be designed for various genes that have clinical and diagnostic importance depending on the cancer types. Eventually, it would be particularly beneficial to design an array of primer sets covering the representative frequently mutated genes as depicted in some commercial cancer panels. Therefore, further technical developments are needed, such as designing accurate and robust PCR primers for each gene.</p><p>In addition to the detection of the cancerous mutations, using the long MinION reads, we could analyze the structural alterations and phasing of SNPs and mutations. It is important to elucidate the allelic background of mutations that are occasionally distantly located. For example, mutual associations between primary and secondary mutations, particularly mutations associated with drug resistance,<a reveal-id="dsx027-B31" data-open="dsx027-B31" class="link link-ref link-reveal xref-bibr"><sup>31</sup></a><sup>,</sup><a reveal-id="dsx027-B32" data-open="dsx027-B32" class="link link-ref link-reveal xref-bibr"><sup>32</sup></a> have important clinical relevance. In addition to the cases of EGFR, various types of associations between primary and secondary/drug-resistant mutations have been reported. For example, for ALK-fusion-positive lung adenocarcinoma patients, drugs that molecularly target the tyrosine kinases, such as crizotinib, have been used. Similar to other effective anti-cancer drugs, a relapse frequently occurs as resistant clones are developed. Patients move to the second line tyrosine kinase inhibitor, which could also eventually become ineffective.<a reveal-id="dsx027-B37" data-open="dsx027-B37" class="link link-ref link-reveal xref-bibr"><sup>37</sup></a> However, intriguingly, very recent reports indicated that, in some cases, the cancer recovers the sensitivity for the previous round of drugs depending on the patterns of the initial and secondary resistant mutations and their allelic backgrounds.<a reveal-id="dsx027-B38" data-open="dsx027-B38" class="link link-ref link-reveal xref-bibr"><sup>38</sup></a><sup>,</sup><a reveal-id="dsx027-B39" data-open="dsx027-B39" class="link link-ref link-reveal xref-bibr"><sup>39</sup></a> For these cases, long-read sequencing could provide indispensable information for selecting the next therapeutic approaches. This study is the first step toward a more widespread application of long-read sequence-based diagnosis in cancers in general clinical practice.</p>                            <h2 scrollto-destination=104086276 id="104086276" class="backacknowledgements-title" >Acknowledgements</h2>
<p>The authors would like to thank K. Imamura, T. Horiuchi, H. Wakaguri and K. Abe for their technical assistance in the library preparation, sequencing and data processing. The authors would also like to thank A. Kinase, A. Onai and Y. Ichinose for their assistance with the PCR experiments.</p>                            <h3 scrollto-destination=104086278 id="104086278" class="section-title" >Conflict of interest</h3>
<p>None declared.</p>                            <h3 scrollto-destination=104086280 id="104086280" class="section-title" >Supplementary data</h3>
<p>Supplementary data are available at DNARES online. </p>                            <h3 scrollto-destination=104086282 id="104086282" class="section-title" >Funding</h3>
<p>MEXT KAKENHI Grant Number 16H01582 and MEXT KAKENHI Grant Number 16H06279.</p>                            <h2 scrollto-destination=104086284 id="104086284" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="dsx027-B1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B1" class="ref-content "><div class="label -label">1</div><div class="citation mixed-citation"><div class="collab">The Cancer Genome Atlas Research Network</div>, <div class="person-group"><div class="name string-name"><div class="surname">Weinstein</div> <div class="given-names">J.N.</div></div>, <div class="name string-name"><div class="surname">Collisson</div> <div class="given-names">E.A.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">The Cancer Genome Atlas Pan-Cancer analysis project</div>. <div class="source">Nat. Genet</div> ., <div class="volume">45</div>, <div class="fpage">1113</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20Cancer%20Genome%20Atlas%20Pan-Cancer%20analysis%20project&amp;author=J.N.%20Weinstein&amp;author=E.A.%20Collisson&amp;publication_year=2013&amp;journal=Nat.%20Genet&amp;volume=45&amp;pages=1113-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/ng.2764" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fng.2764" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fng.2764"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24071849" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B2" class="ref-content "><div class="label -label">2</div><div class="citation mixed-citation"><div class="collab">International Cancer Genome Consortium</div> <div class="year">2010</div>, 
<div class="article-title">International network of cancer genome projects</div>. <div class="source">Nature</div> , <div class="volume">464</div>, <div class="fpage">993</div>–<div class="lpage">8</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature08987" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature08987" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature08987"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20393554" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B3" class="ref-content "><div class="label -label">3</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Sharma</div> <div class="given-names">S.V.</div></div>, <div class="name string-name"><div class="surname">Bell</div> <div class="given-names">D.W.</div></div>, <div class="name string-name"><div class="surname">Settleman</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Haber</div> <div class="given-names">D.A.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Epidermal growth factor receptor mutations in lung cancer</div>. <div class="source">Nat. Rev. Cancer</div> , <div class="volume">7</div>, <div class="fpage">169</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Epidermal%20growth%20factor%20receptor%20mutations%20in%20lung%20cancer&amp;author=S.V.%20Sharma&amp;author=D.W.%20Bell&amp;author=J.%20Settleman&amp;author=D.A.%20Haber&amp;publication_year=2007&amp;journal=Nat.%20Rev.%20Cancer&amp;volume=7&amp;pages=169-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrc2088" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrc2088" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrc2088"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17318210" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B4" class="ref-content "><div class="label -label">4</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Soda</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Choi</div> <div class="given-names">Y.L.</div></div>, <div class="name string-name"><div class="surname">Enomoto</div> <div class="given-names">M.</div></div></div>, et al.   <div class="year">2007</div>, 
<div class="article-title">Identification of the transforming EML4-ALK fusion gene in non-small-cell lung cancer</div>. <div class="source">Nature</div> , <div class="volume">448</div>, <div class="fpage">561</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20the%20transforming%20EML4-ALK%20fusion%20gene%20in%20non-small-cell%20lung%20cancer&amp;author=M.%20Soda&amp;author=Y.L.%20Choi&amp;author=M.%20Enomoto&amp;publication_year=2007&amp;journal=Nature&amp;volume=448&amp;pages=561-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature05945" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature05945" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature05945"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17625570" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B5" class="ref-content "><div class="label -label">5</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kwak</div> <div class="given-names">E.L.</div></div>, <div class="name string-name"><div class="surname">Bang</div> <div class="given-names">Y.J.</div></div>, <div class="name string-name"><div class="surname">Camidge</div> <div class="given-names">D.R.</div></div></div>, et al.   <div class="year">2010</div>, 
<div class="article-title">Anaplastic lymphoma kinase inhibition in non-small-cell lung cancer</div>. <div class="source">N. Engl J. Med</div> ., <div class="volume">363</div>, <div class="fpage">1693</div>-<div class="lpage">1703</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Anaplastic%20lymphoma%20kinase%20inhibition%20in%20non-small-cell%20lung%20cancer&amp;author=E.L.%20Kwak&amp;author=Y.J.%20Bang&amp;author=D.R.%20Camidge&amp;publication_year=2010&amp;journal=N.%20Engl%20J.%20Med&amp;volume=363&amp;pages=1693-1703" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1056/NEJMoa1006448" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1056%2FNEJMoa1006448" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1056%2FNEJMoa1006448"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20979469" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B6" class="ref-content "><div class="label -label">6</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Fiddes</div> <div class="given-names">I.T.</div></div>, <div class="name string-name"><div class="surname">Miga</div> <div class="given-names">K.H.</div></div>, <div class="name string-name"><div class="surname">Olsen</div> <div class="given-names">H.E.</div></div>, <div class="name string-name"><div class="surname">Paten</div> <div class="given-names">B.</div></div>, <div class="name string-name"><div class="surname">Akeson</div> <div class="given-names">M.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Improved data analysis for the MinION nanopore sequencer</div>. <div class="source">Nat. Methods</div> , <div class="volume">12</div>, <div class="fpage">351</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improved%20data%20analysis%20for%20the%20MinION%20nanopore%20sequencer&amp;author=M.%20Jain&amp;author=I.T.%20Fiddes&amp;author=K.H.%20Miga&amp;author=H.E.%20Olsen&amp;author=B.%20Paten&amp;author=M.%20Akeson&amp;publication_year=2015&amp;journal=Nat.%20Methods&amp;volume=12&amp;pages=351-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.3290" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.3290" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.3290"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25686389" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B7" class="ref-content "><div class="label -label">7</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ashton</div> <div class="given-names">P.M.</div></div>, <div class="name string-name"><div class="surname">Nair</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Dallman</div> <div class="given-names">T.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">MinION nanopore sequencing identifies the position and structure of a bacterial antibiotic resistance island</div>. <div class="source">Nat. Biotechnol</div> ., <div class="volume">33</div>, <div class="fpage">296</div>–<div class="lpage">300</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=MinION%20nanopore%20sequencing%20identifies%20the%20position%20and%20structure%20of%20a%20bacterial%20antibiotic%20resistance%20island&amp;author=P.M.%20Ashton&amp;author=S.%20Nair&amp;author=T.%20Dallman&amp;publication_year=2015&amp;journal=Nat.%20Biotechnol&amp;volume=33&amp;pages=296-300" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nbt.3103" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnbt.3103" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnbt.3103"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25485618" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B8" class="ref-content "><div class="label -label">8</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quick</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Ashton</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Calus</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Rapid draft sequencing and real-time nanopore sequencing in a hospital outbreak of Salmonella</div>. <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">114</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Rapid%20draft%20sequencing%20and%20real-time%20nanopore%20sequencing%20in%20a%20hospital%20outbreak%20of%20Salmonella&amp;author=J.%20Quick&amp;author=P.%20Ashton&amp;author=S.%20Calus&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=114" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0677-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0677-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0677-2"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26025440" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B9" class="ref-content "><div class="label -label">9</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Quick</div> <div class="given-names">J.</div></div>, <div class="name string-name"><div class="surname">Loman</div> <div class="given-names">N.J.</div></div>, <div class="name string-name"><div class="surname">Duraffour</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Real-time, portable genome sequencing for Ebola surveillance</div>. <div class="source">Nature</div> , <div class="volume">530</div>, <div class="fpage">228</div>–<div class="lpage">32</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Real-time%2C%20portable%20genome%20sequencing%20for%20Ebola%20surveillance&amp;author=J.%20Quick&amp;author=N.J.%20Loman&amp;author=S.%20Duraffour&amp;publication_year=2016&amp;journal=Nature&amp;volume=530&amp;pages=228-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature16996" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature16996" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature16996"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26840485" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B10" class="ref-content "><div class="label -label">10</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Norris</div> <div class="given-names">A.L.</div></div>, <div class="name string-name"><div class="surname">Workman</div> <div class="given-names">R.E.</div></div>, <div class="name string-name"><div class="surname">Fan</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Eshleman</div> <div class="given-names">J.R.</div></div>, <div class="name string-name"><div class="surname">Timp</div> <div class="given-names">W.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Nanopore sequencing detects structural variants in cancer</div>. <div class="source">Cancer Biol. Ther</div> ., <div class="volume">17</div>, <div class="fpage">246</div>–<div class="lpage">53</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Nanopore%20sequencing%20detects%20structural%20variants%20in%20cancer&amp;author=A.L.%20Norris&amp;author=R.E.%20Workman&amp;author=Y.%20Fan&amp;author=J.R.%20Eshleman&amp;author=W.%20Timp&amp;publication_year=2016&amp;journal=Cancer%20Biol.%20Ther&amp;volume=17&amp;pages=246-53" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1080/15384047.2016.1139236" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1080%2F15384047.2016.1139236" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1080%2F15384047.2016.1139236"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26787508" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B11" class="ref-content "><div class="label -label">11</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Minervini</div> <div class="given-names">C.F.</div></div>, <div class="name string-name"><div class="surname">Cumbo</div> <div class="given-names">C.</div></div>, <div class="name string-name"><div class="surname">Orsini</div> <div class="given-names">P.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">TP53 gene mutation analysis in chronic lymphocytic leukemia by nanopore MinION sequencing</div>. <div class="source">Diagn. Pathol</div> ., <div class="volume">11</div>, <div class="fpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TP53%20gene%20mutation%20analysis%20in%20chronic%20lymphocytic%20leukemia%20by%20nanopore%20MinION%20sequencing&amp;author=C.F.%20Minervini&amp;author=C.%20Cumbo&amp;author=P.%20Orsini&amp;publication_year=2016&amp;journal=Diagn.%20Pathol&amp;volume=11&amp;pages=96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13000-016-0550-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13000-016-0550-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13000-016-0550-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/27724982" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B12" class="ref-content "><div class="label -label">12</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Makinoshima</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Wakaguri</div> <div class="given-names">H.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Aberrant transcriptional regulations in cancers: genome, transcriptome and epigenome analysis of lung adenocarcinoma cell lines</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">42</div>, <div class="fpage">13557</div>–<div class="lpage">72</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Aberrant%20transcriptional%20regulations%20in%20cancers%3A%20genome%2C%20transcriptome%20and%20epigenome%20analysis%20of%20lung%20adenocarcinoma%20cell%20lines&amp;author=A.%20Suzuki&amp;author=H.%20Makinoshima&amp;author=H.%20Wakaguri&amp;publication_year=2014&amp;journal=Nucleic%20Acids%20Res&amp;volume=42&amp;pages=13557-72" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gku885" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgku885" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgku885"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25378332" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B13" class="ref-content "><div class="label -label">13</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kohno</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Ichikawa</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Totoki</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">KIF5B-RET fusions in lung adenocarcinoma</div>. <div class="source">Nat. Med</div> ., <div class="volume">18</div>, <div class="fpage">375</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=KIF5B-RET%20fusions%20in%20lung%20adenocarcinoma&amp;author=T.%20Kohno&amp;author=H.%20Ichikawa&amp;author=Y.%20Totoki&amp;publication_year=2012&amp;journal=Nat.%20Med&amp;volume=18&amp;pages=375-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nm.2644" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnm.2644" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnm.2644"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22327624" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B14" class="ref-content "><div class="label -label">14</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Untergasser</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Nijveen</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Rao</div> <div class="given-names">X.</div></div>, <div class="name string-name"><div class="surname">Bisseling</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Geurts</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Leunissen</div> <div class="given-names">J.A.</div></div></div> <div class="year">2007</div>, 
<div class="article-title">Primer3Plus, an enhanced web interface to Primer3</div>. <div class="source">Nucleic Acids Res</div> ., <div class="volume">35</div>, <div class="fpage">W71</div>–<div class="lpage">4</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Primer3Plus%2C%20an%20enhanced%20web%20interface%20to%20Primer3&amp;author=A.%20Untergasser&amp;author=H.%20Nijveen&amp;author=X.%20Rao&amp;author=T.%20Bisseling&amp;author=R.%20Geurts&amp;author=J.A.%20Leunissen&amp;publication_year=2007&amp;journal=Nucleic%20Acids%20Res&amp;volume=35&amp;pages=W71-4" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkm306" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkm306" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkm306"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17485472" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B15" class="ref-content "><div class="label -label">15</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Loman</div> <div class="given-names">N.J.</div></div>, <div class="name string-name"><div class="surname">Quinlan</div> <div class="given-names">A.R.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Poretools: a toolkit for analyzing nanopore sequence data</div>. <div class="source">Bioinformatics</div> , <div class="volume">30</div>, <div class="fpage">3399</div>–<div class="lpage">401</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Poretools%3A%20a%20toolkit%20for%20analyzing%20nanopore%20sequence%20data&amp;author=N.J.%20Loman&amp;author=A.R.%20Quinlan&amp;publication_year=2014&amp;journal=Bioinformatics&amp;volume=30&amp;pages=3399-401" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btu555" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtu555" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtu555"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25143291" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B16" class="ref-content "><div class="label -label">16</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kiełbasa</div> <div class="given-names">S.M.</div></div>, <div class="name string-name"><div class="surname">Wan</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Sato</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Horton</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Frith</div> <div class="given-names">M.C.</div></div></div> <div class="year">2011</div>, 
<div class="article-title">Adaptive seeds tame genomic sequence comparison</div>. <div class="source">Genome Res</div> ., <div class="volume">21</div>, <div class="fpage">487</div>–<div class="lpage">93</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptive%20seeds%20tame%20genomic%20sequence%20comparison&amp;author=S.M.%20Kie%C5%82basa&amp;author=R.%20Wan&amp;author=K.%20Sato&amp;author=P.%20Horton&amp;author=M.C.%20Frith&amp;publication_year=2011&amp;journal=Genome%20Res&amp;volume=21&amp;pages=487-93" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.113985.110" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.113985.110" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.113985.110"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21209072" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B17" class="ref-content "><div class="label -label">17</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Li</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Durbin</div> <div class="given-names">R.</div></div></div> <div class="year">2009</div>, 
<div class="article-title">Fast and accurate short read alignment with Burrows-Wheeler transform</div>. <div class="source">Bioinformatics</div> , <div class="volume">25</div>, <div class="fpage">1754</div>–<div class="lpage">60</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20accurate%20short%20read%20alignment%20with%20Burrows-Wheeler%20transform&amp;author=H.%20Li&amp;author=R.%20Durbin&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1754-60" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp324" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp324" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp324"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19451168" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B18" class="ref-content "><div class="label -label">18</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Hamada</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Ono</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Asai</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Frith</div> <div class="given-names">M.C.</div></div></div> <div class="year">2016</div>, 
<div class="article-title">Training alignment parameters for arbitrary sequencers with LAST-TRAIN</div>. <div class="source">Bioinformatics</div> , <div class="volume">33</div>, <div class="fpage">926</div>–<div class="lpage">8</div>.</div></div></div></div><div content-id="dsx027-B19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B19" class="ref-content "><div class="label -label">19</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Frith</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Wan</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Horton</div> <div class="given-names">P.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Incorporating sequence quality data into alignment improves DNA read mapping</div>. <div class="source">Nucleic Acids Res</div> , <div class="volume">38</div>, <div class="fpage">e100</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Incorporating%20sequence%20quality%20data%20into%20alignment%20improves%20DNA%20read%20mapping&amp;author=M.C.%20Frith&amp;author=R.%20Wan&amp;author=P.%20Horton&amp;publication_year=2010&amp;journal=Nucleic%20Acids%20Res&amp;volume=38&amp;pages=e100" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkq010" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkq010" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkq010"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20110255" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B20" class="ref-content "><div class="label -label">20</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Speir</div> <div class="given-names">M.L.</div></div>, <div class="name string-name"><div class="surname">Zweig</div> <div class="given-names">A.S.</div></div>, <div class="name string-name"><div class="surname">Rosenbloom</div> <div class="given-names">K.R.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">The UCSC Genome Browser database: 2016 update</div>. <div class="source">Nucleic Acids Res</div> , <div class="volume">44</div>, <div class="fpage">D717</div>–<div class="lpage">25</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20UCSC%20Genome%20Browser%20database%3A%202016%20update&amp;author=M.L.%20Speir&amp;author=A.S.%20Zweig&amp;author=K.R.%20Rosenbloom&amp;publication_year=2016&amp;journal=Nucleic%20Acids%20Res&amp;volume=44&amp;pages=D717-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkv1275" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkv1275" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkv1275"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26590259" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B21" class="ref-content "><div class="label -label">21</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Wu</div> <div class="given-names">T.D.</div></div>, <div class="name string-name"><div class="surname">Nacu</div> <div class="given-names">S.</div></div></div> <div class="year">2010</div>, 
<div class="article-title">Fast and SNP-tolerant detection of complex variants and splicing in short reads</div>. <div class="source">Bioinformatics</div> , <div class="volume">26</div>, <div class="fpage">873</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Fast%20and%20SNP-tolerant%20detection%20of%20complex%20variants%20and%20splicing%20in%20short%20reads&amp;author=T.D.%20Wu&amp;author=S.%20Nacu&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=873-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btq057" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtq057" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtq057"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20147302" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B22" class="ref-content "><div class="label -label">22</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Frith</div> <div class="given-names">M.C.</div></div>, <div class="name string-name"><div class="surname">Kawaguchi</div> <div class="given-names">R.</div></div></div> <div class="year">2015</div>, 
<div class="article-title">Split-alignment of genomes finds orthologies more accurately</div>. <div class="source">Genome Biol</div> ., <div class="volume">16</div>, <div class="fpage">106</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Split-alignment%20of%20genomes%20finds%20orthologies%20more%20accurately&amp;author=M.C.%20Frith&amp;author=R.%20Kawaguchi&amp;publication_year=2015&amp;journal=Genome%20Biol&amp;volume=16&amp;pages=106" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/s13059-015-0670-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2Fs13059-015-0670-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2Fs13059-015-0670-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25994148" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B23" class="ref-content "><div class="label -label">23</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Matsubara</div> <div class="given-names">D.</div></div>, <div class="name string-name"><div class="surname">Kanai</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Ishikawa</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Identification of CCDC6-RET fusion in the human lung adenocarcinoma cell line, LC-2/ad</div>. <div class="source">J. Thorac. Oncol</div> ., <div class="volume">7</div>, <div class="fpage">1872</div>–<div class="lpage">6</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20CCDC6-RET%20fusion%20in%20the%20human%20lung%20adenocarcinoma%20cell%20line%2C%20LC-2%2Fad&amp;author=D.%20Matsubara&amp;author=Y.%20Kanai&amp;author=S.%20Ishikawa&amp;publication_year=2012&amp;journal=J.%20Thorac.%20Oncol&amp;volume=7&amp;pages=1872-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1097/JTO.0b013e3182721ed1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1097%2FJTO.0b013e3182721ed1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1097%2FJTO.0b013e3182721ed1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23154560" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B24" class="ref-content "><div class="label -label">24</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Makinoshima</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Matsumoto</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Identification of a lung adenocarcinoma cell line with CCDC6-RET fusion gene and the effect of RET inhibitors in vitro and in vivo</div>. <div class="source">Cancer Sci</div> ., <div class="volume">104</div>, <div class="fpage">896</div>–<div class="lpage">903</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20of%20a%20lung%20adenocarcinoma%20cell%20line%20with%20CCDC6-RET%20fusion%20gene%20and%20the%20effect%20of%20RET%20inhibitors%20in%20vitro%20and%20in%20vivo&amp;author=M.%20Suzuki&amp;author=H.%20Makinoshima&amp;author=S.%20Matsumoto&amp;publication_year=2013&amp;journal=Cancer%20Sci&amp;volume=104&amp;pages=896-903" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/cas.12175" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fcas.12175" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fcas.12175"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23578175" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B25" class="ref-content "><div class="label -label">25</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Jung</div> <div class="given-names">Y.</div></div>, <div class="name string-name"><div class="surname">Kim</div> <div class="given-names">P.</div></div>, <div class="name string-name"><div class="surname">Keum</div> <div class="given-names">J.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Discovery of ALK-PTPN3 gene fusion from human non-small cell lung carcinoma cell line using next generation RNA sequencing</div>. <div class="source">Genes Chromosomes Cancer</div> , <div class="volume">51</div>, <div class="fpage">590</div>–<div class="lpage">7</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Discovery%20of%20ALK-PTPN3%20gene%20fusion%20from%20human%20non-small%20cell%20lung%20carcinoma%20cell%20line%20using%20next%20generation%20RNA%20sequencing&amp;author=Y.%20Jung&amp;author=P.%20Kim&amp;author=J.%20Keum&amp;publication_year=2012&amp;journal=Genes%20Chromosomes%20Cancer&amp;volume=51&amp;pages=590-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/gcc.v51.6" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fgcc.v51.6" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fgcc.v51.6"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22334442" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B26" class="ref-content "><div class="label -label">26</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Ding</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Getz</div> <div class="given-names">G.</div></div>, <div class="name string-name"><div class="surname">Wheeler</div> <div class="given-names">D.A.</div></div></div>, et al.   <div class="year">2008</div>, 
<div class="article-title">Somatic mutations affect key pathways in lung adenocarcinoma</div>. <div class="source">Nature</div> , <div class="volume">455</div>, <div class="fpage">1069</div>–<div class="lpage">75</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Somatic%20mutations%20affect%20key%20pathways%20in%20lung%20adenocarcinoma&amp;author=L.%20Ding&amp;author=G.%20Getz&amp;author=D.A.%20Wheeler&amp;publication_year=2008&amp;journal=Nature&amp;volume=455&amp;pages=1069-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature07423" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature07423" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature07423"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18948947" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B27" class="ref-content "><div class="label -label">27</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Imielinski</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Berger</div> <div class="given-names">A.H.</div></div>, <div class="name string-name"><div class="surname">Hammerman</div> <div class="given-names">P.S.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">Mapping the hallmarks of lung adenocarcinoma with massively parallel sequencing</div>. <div class="source">Cell</div> , <div class="volume">150</div>, <div class="fpage">1107</div>–<div class="lpage">20</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20the%20hallmarks%20of%20lung%20adenocarcinoma%20with%20massively%20parallel%20sequencing&amp;author=M.%20Imielinski&amp;author=A.H.%20Berger&amp;author=P.S.%20Hammerman&amp;publication_year=2012&amp;journal=Cell&amp;volume=150&amp;pages=1107-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.cell.2012.08.029" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.cell.2012.08.029" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.cell.2012.08.029"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22980975" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B28" class="ref-content "><div class="label -label">28</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Seo</div> <div class="given-names">J.S.</div></div>, <div class="name string-name"><div class="surname">Ju</div> <div class="given-names">Y.S.</div></div>, <div class="name string-name"><div class="surname">Lee</div> <div class="given-names">W.C.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">The transcriptional landscape and mutational profile of lung adenocarcinoma</div>. <div class="source">Genome Res</div> ., <div class="volume">22</div>, <div class="fpage">2109</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20transcriptional%20landscape%20and%20mutational%20profile%20of%20lung%20adenocarcinoma&amp;author=J.S.%20Seo&amp;author=Y.S.%20Ju&amp;author=W.C.%20Lee&amp;publication_year=2012&amp;journal=Genome%20Res&amp;volume=22&amp;pages=2109-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.145144.112" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.145144.112" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.145144.112"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22975805" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B29" class="ref-content "><div class="label -label">29</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Suzuki</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Mimaki</div> <div class="given-names">S.</div></div>, <div class="name string-name"><div class="surname">Yamane</div> <div class="given-names">Y.</div></div></div>, et al.   <div class="year">2013</div>, 
<div class="article-title">Identification and characterization of cancer mutations in Japanese lung adenocarcinoma without sequencing of normal tissue counterparts</div>. <div class="source">PLoS One</div> , <div class="volume">8</div>, <div class="fpage">e73484</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Identification%20and%20characterization%20of%20cancer%20mutations%20in%20Japanese%20lung%20adenocarcinoma%20without%20sequencing%20of%20normal%20tissue%20counterparts&amp;author=A.%20Suzuki&amp;author=S.%20Mimaki&amp;author=Y.%20Yamane&amp;publication_year=2013&amp;journal=PLoS%20One&amp;volume=8&amp;pages=e73484" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pone.0073484" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pone.0073484" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pone.0073484"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24069199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B30" class="ref-content "><div class="label -label">30</div><div class="citation mixed-citation"><div class="collab">The Cancer Genome Atlas Research Network</div> <div class="year">2014</div>, 
<div class="article-title">Comprehensive molecular profiling of lung adenocarcinoma</div>. <div class="source">Nature</div> , <div class="volume">511</div>, <div class="fpage">543</div>–<div class="lpage">50</div>.<div class="citation-links"><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nature13385" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnature13385" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnature13385"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25079552" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B31" class="ref-content "><div class="label -label">31</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Pao</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Miller</div> <div class="given-names">V.A.</div></div>, <div class="name string-name"><div class="surname">Politi</div> <div class="given-names">K.A.</div></div></div>, et al.   <div class="year">2005</div>, 
<div class="article-title">Acquired resistance of lung adenocarcinomas to gefitinib or erlotinib is associated with a second mutation in the EGFR kinase domain</div>. <div class="source">PLoS Med</div> ., <div class="volume">2</div>, <div class="fpage">e73</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Acquired%20resistance%20of%20lung%20adenocarcinomas%20to%20gefitinib%20or%20erlotinib%20is%20associated%20with%20a%20second%20mutation%20in%20the%20EGFR%20kinase%20domain&amp;author=W.%20Pao&amp;author=V.A.%20Miller&amp;author=K.A.%20Politi&amp;publication_year=2005&amp;journal=PLoS%20Med&amp;volume=2&amp;pages=e73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1371/journal.pmed.0020073" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1371%2Fjournal.pmed.0020073" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1371%2Fjournal.pmed.0020073"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15737014" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B32" class="ref-content "><div class="label -label">32</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Thress</div> <div class="given-names">K.S.</div></div>, <div class="name string-name"><div class="surname">Paweletz</div> <div class="given-names">C.P.</div></div>, <div class="name string-name"><div class="surname">Felip</div> <div class="given-names">E.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">Acquired EGFR C797S mutation mediates resistance to AZD9291 in non-small cell lung cancer harboring EGFR T790M</div>. <div class="source">Nat. Med</div> ., <div class="volume">21</div>, <div class="fpage">560</div>–<div class="lpage">2</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Acquired%20EGFR%20C797S%20mutation%20mediates%20resistance%20to%20AZD9291%20in%20non-small%20cell%20lung%20cancer%20harboring%20EGFR%20T790M&amp;author=K.S.%20Thress&amp;author=C.P.%20Paweletz&amp;author=E.%20Felip&amp;publication_year=2015&amp;journal=Nat.%20Med&amp;volume=21&amp;pages=560-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nm.3854" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnm.3854" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnm.3854"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25939061" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B33" class="ref-content "><div class="label -label">33</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Soda</div> <div class="given-names">M.</div></div>, <div class="name string-name"><div class="surname">Isobe</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Inoue</div> <div class="given-names">A.</div></div></div>, et al.   <div class="year">2012</div>, 
<div class="article-title">A prospective PCR-based screening for the EML4-ALK oncogene in non-small cell lung cancer</div>. <div class="source">Clin. Cancer Res</div> ., <div class="volume">18</div>, <div class="fpage">5682</div>–<div class="lpage">9</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=A%20prospective%20PCR-based%20screening%20for%20the%20EML4-ALK%20oncogene%20in%20non-small%20cell%20lung%20cancer&amp;author=M.%20Soda&amp;author=K.%20Isobe&amp;author=A.%20Inoue&amp;publication_year=2012&amp;journal=Clin.%20Cancer%20Res&amp;volume=18&amp;pages=5682-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/1078-0432.CCR-11-2947" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F1078-0432.CCR-11-2947" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F1078-0432.CCR-11-2947"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/22908099" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B34" class="ref-content "><div class="label -label">34</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Yang</div> <div class="given-names">Z.</div></div>, <div class="name string-name"><div class="surname">Hackshaw</div> <div class="given-names">A.</div></div>, <div class="name string-name"><div class="surname">Feng</div> <div class="given-names">Q.</div></div></div>, et al.   <div class="year">2017</div>, 
<div class="article-title">Comparison of gefitinib, erlotinib and afatinib in non-small cell lung cancer: A meta-analysis</div>. <div class="source">Int. J. Cancer</div> , <div class="volume">140</div>, <div class="fpage">2805</div>–<div class="lpage">19</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparison%20of%20gefitinib%2C%20erlotinib%20and%20afatinib%20in%20non-small%20cell%20lung%20cancer%3A%20A%20meta-analysis&amp;author=Z.%20Yang&amp;author=A.%20Hackshaw&amp;author=Q.%20Feng&amp;publication_year=2017&amp;journal=Int.%20J.%20Cancer&amp;volume=140&amp;pages=2805-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1002/ijc.v140.12" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1002%2Fijc.v140.12" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1002%2Fijc.v140.12"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/28295308" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B35" class="ref-content "><div class="label -label">35</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Katayama</div> <div class="given-names">R.</div></div>, <div class="name string-name"><div class="surname">Friboulet</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Koike</div> <div class="given-names">S.</div></div></div>, et al.   <div class="year">2014</div>, 
<div class="article-title">Two novel ALK mutations mediate acquired resistance to the next-generation ALK inhibitor alectinib</div>. <div class="source">Clin Cancer Res</div> ., <div class="volume">20</div>, <div class="fpage">5686</div>–<div class="lpage">96</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Two%20novel%20ALK%20mutations%20mediate%20acquired%20resistance%20to%20the%20next-generation%20ALK%20inhibitor%20alectinib&amp;author=R.%20Katayama&amp;author=L.%20Friboulet&amp;author=S.%20Koike&amp;publication_year=2014&amp;journal=Clin%20Cancer%20Res&amp;volume=20&amp;pages=5686-96" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/1078-0432.CCR-14-1511" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F1078-0432.CCR-14-1511" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F1078-0432.CCR-14-1511"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25228534" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B36" class="ref-content "><div class="label -label">36</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Kohno</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Tsuta</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Tsuchihara</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Nakaoku</div> <div class="given-names">T.</div></div>, <div class="name string-name"><div class="surname">Yoh</div> <div class="given-names">K.</div></div>, <div class="name string-name"><div class="surname">Goto</div> <div class="given-names">K.</div></div></div> <div class="year">2013</div>, 
<div class="article-title">RET fusion gene: translation to personalized lung cancer therapy</div>. <div class="source">Cancer Sci</div> ., <div class="volume">104</div>, <div class="fpage">1396</div>–<div class="lpage">1400</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RET%20fusion%20gene%3A%20translation%20to%20personalized%20lung%20cancer%20therapy&amp;author=T.%20Kohno&amp;author=K.%20Tsuta&amp;author=K.%20Tsuchihara&amp;author=T.%20Nakaoku&amp;author=K.%20Yoh&amp;author=K.%20Goto&amp;publication_year=2013&amp;journal=Cancer%20Sci&amp;volume=104&amp;pages=1396-1400" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/cas.2013.104.issue-11" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fcas.2013.104.issue-11" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fcas.2013.104.issue-11"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/23991695" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B37" class="ref-content "><div class="label -label">37</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Camidge</div> <div class="given-names">D.R.</div></div>, <div class="name string-name"><div class="surname">Pao</div> <div class="given-names">W.</div></div>, <div class="name string-name"><div class="surname">Sequist</div> <div class="given-names">L.V.</div></div></div> <div class="year">2014</div>, 
<div class="article-title">Acquired resistance to TKIs in solid tumours: learning from lung cancer</div>. <div class="source">Nat. Rev. Clin. Oncol</div> ., <div class="volume">11</div>, <div class="fpage">473</div>–<div class="lpage">81</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Acquired%20resistance%20to%20TKIs%20in%20solid%20tumours%3A%20learning%20from%20lung%20cancer&amp;author=D.R.%20Camidge&amp;author=W.%20Pao&amp;author=L.V.%20Sequist&amp;publication_year=2014&amp;journal=Nat.%20Rev.%20Clin.%20Oncol&amp;volume=11&amp;pages=473-81" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nrclinonc.2014.104" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnrclinonc.2014.104" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnrclinonc.2014.104"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/24981256" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B38" class="ref-content "><div class="label -label">38</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Niederst</div> <div class="given-names">M.J.</div></div>, <div class="name string-name"><div class="surname">Hu</div> <div class="given-names">H.</div></div>, <div class="name string-name"><div class="surname">Mulvey</div> <div class="given-names">H.E.</div></div></div>, et al.   <div class="year">2015</div>, 
<div class="article-title">The allelic context of the C797S mutation acquired upon treatment with third-generation EGFR inhibitors impacts sensitivity to subsequent treatment strategies</div>. <div class="source">Clin. Cancer Res</div> ., <div class="volume">21</div>, <div class="fpage">3924</div>–<div class="lpage">33</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=The%20allelic%20context%20of%20the%20C797S%20mutation%20acquired%20upon%20treatment%20with%20third-generation%20EGFR%20inhibitors%20impacts%20sensitivity%20to%20subsequent%20treatment%20strategies&amp;author=M.J.%20Niederst&amp;author=H.%20Hu&amp;author=H.E.%20Mulvey&amp;publication_year=2015&amp;journal=Clin.%20Cancer%20Res&amp;volume=21&amp;pages=3924-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1158/1078-0432.CCR-15-0560" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1158%2F1078-0432.CCR-15-0560" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1158%2F1078-0432.CCR-15-0560"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/25964297" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="dsx027-B39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="dsx027-B39" class="ref-content "><div class="label -label">39</div><div class="citation mixed-citation"><div class="person-group"><div class="name string-name"><div class="surname">Shaw</div> <div class="given-names">A.T.</div></div>, <div class="name string-name"><div class="surname">Friboulet</div> <div class="given-names">L.</div></div>, <div class="name string-name"><div class="surname">Leshchiner</div> <div class="given-names">I.</div></div></div>, et al.   <div class="year">2016</div>, 
<div class="article-title">Resensitization to crizotinib by the lorlatinib ALK resistance mutation L1198F</div>. <div class="source">N. Engl J. Med</div> ., <div class="volume">374</div>, <div class="fpage">54</div>–<div class="lpage">61</div>.<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Resensitization%20to%20crizotinib%20by%20the%20lorlatinib%20ALK%20resistance%20mutation%20L1198F&amp;author=A.T.%20Shaw&amp;author=L.%20Friboulet&amp;author=I.%20Leshchiner&amp;publication_year=2016&amp;journal=N.%20Engl%20J.%20Med&amp;volume=374&amp;pages=54-61" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1056/NEJMoa1508887" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1056%2FNEJMoa1508887" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1056%2FNEJMoa1508887"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/26698910" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2017. Published by Oxford University Press on behalf of Kazusa DNA Research Institute.</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (<a class="link link-uri" href="http://creativecommons.org/licenses/by-nc/4.0/" target="_blank">http://creativecommons.org/licenses/by-nc/4.0/</a>), which permits non-commercial re-use, distribution, and reproduction in any medium, provided the original work is properly cited. For commercial re-use, please contact journals.permissions@oup.com</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=3896153&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/24/6/10.1093_dnares_dsx027/1/supplementary_data_dsx027.pdf?Expires=2147483647&amp;Signature=pfJy9s5kTZBC-2LePWUYT0U5TfTfCQwmtMz4t4F~yGsxbWYobBaC5fafSrc8p3cTCanU-cax1GXYFhIZshRhJDvz4blNcofh~CTZBsZATUhpAskE3oshb8hlZMdcsjloP-iNnIKTIZ44MDMeQM8LudQj3NNsmM4W6VSfnmo5bu7kU-oV-I7Ld7FAFVoiI9xLAwLmVIC1d8Mt8Y9-tJmoZvz-HTtwn~7zscmM-jpSOLMcHF8QZ1YDyOSsy6NhSvdWEi0EekgsPGmjv9a8vbvqEc~DOn6v97DQ8vqeUJp61kaXNw0QdwoI-qaY1zATLxSGI0z39UauitaDtHioCTxOjQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"><div class="label -label">Supplementary Figures and Tables</div></a> - pdf file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">2,817</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000418420400004&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">4</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dsx027"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="3896153"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000418420400004&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f24%2f6%2f585%2f3896153">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000418420400004&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (4)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F24%2F6%2F585%2F3896153" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/3896153'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Sequencing and phasing cancer mutations in lung cancers using a long-read portable sequencer" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dsx027");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f24%2f6%2f585%2f3896153">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

