

<!DOCTYPE html>
<html lang="en">

<head>
    
    <!-- Responsive View Port -->
    <meta name="viewport" content="width=device-width, initial-scale=1, maximum-scale=1" />

    <!-- Meta -->
    <meta charset="utf-8" /><meta http-equiv="Content-Type" content="text/html; charset=utf-8" /><meta http-equiv="X-UA-Compatible" content="IE=Edge" />
    <!-- Bookmark Icons -->
    <link rel="apple-touch-icon" sizes="180x180" href="/UI/app/img/apple-touch-icon.png" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-32x32.png" sizes="32x32" /><link rel="icon" type="image/png" href="/UI/app/img/favicon-16x16.png" sizes="16x16" /><link rel="manifest" href="/UI/app/img/manifest.json" /><link rel="mask-icon" href="/UI/app/img/safari-pinned-tab.svg" color="#5bbad5" /><link rel="shortcut icon" href="/UI/app/img/favicon.ico" /><meta name="msapplication-config" content="/UI/app/img/browserconfig.xml" /><meta name="theme-color" content="#002f65" />

    <!-- Stylesheets -->
    
<link href="https://fonts.googleapis.com/css?family=Merriweather:300,400,400italic,700,700italic|Source+Sans+Pro:400,400italic,700,700italic" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/6157936185d10c38faf5cbb0c836ebbd86f778b4/globalCSS" type="text/css" rel="stylesheet"/>
<link href="//oup.silverchair-cdn.com/cassette.axd/stylesheet/637258fa5134e2f8ddc699bcb63f020450a21aa3/cookiePolicyCSS" type="text/css" rel="stylesheet"/>
<link href="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.css" type="text/css" rel="stylesheet"/>

        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssetsOriginals/Live/CSS/journals/global.css rel="stylesheet" type="text/css" />
    
        <link href=//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/CSS/dnaresearch/Site1025602322.css rel="stylesheet" type="text/css" />
    

    <!-- Scripts -->    
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/26e6b63b81813d8ad942c90d369df2673602b812/UI/app/vendor/jquery-2.2.4.js" type="text/javascript"></script>

<!--     

 -->    

    
            <!-- Google Tag Manager -->
         <script>(function(w,d,s,l,i){w[l]=w[l]||[];w[l].push({'gtm.start':new Date().getTime(),event:'gtm.js'});var f=d.getElementsByTagName(s)[0],j=d.createElement(s),dl=l!='dataLayer'?'&l='+l:'';j.async=true;j.src='https://www.googletagmanager.com/gtm.js?id='+i+dl;f.parentNode.insertBefore(j,f);})(window,document,'script','dataLayer','GTM-W6DD7HV');</script>
            <!-- End Google Tag Manager -->
    

    <script type="text/javascript">
        var App = App || {};
        App.LoginUserInfo = { 
            isInstLoggedIn: [1] 
        };

        App.CurrentSubdomain = 'dnaresearch';
    </script>

    <!-- Extra Head Content -->
    
    
        <script type="application/ld+json">
        {"@context":"http://schema.org","@id":"https://academic.oup.com/dnaresearch/article/19/2/195/420778","@type":"ScholarlyArticle","name":"Transcriptome Characterization and Sequencing-Based Identification of Salt-Responsive Genes in  Millettia pinnata , a Semi-Mangrove Plant","datePublished":"2012-02-19","isPartOf":{"@id":"https://academic.oup.com/dnaresearch/issue/19/2","@type":"PublicationIssue","issueNumber":"2","datePublished":"2012-04-01","isPartOf":{"@id":"https://academic.oup.com/dnaresearch","@type":"Periodical","name":"DNA Research","issn":["1756-1663"]}},"url":"http://dx.doi.org/10.1093/dnares/dss004","keywords":["transcriptome characterization","Illumina sequencing","salt-responsive genes"],"inLanguage":"en","copyrightHolder":"Kazusa DNA Research Institute","copyrightYear":"2018","publisher":"Oxford University Press","sameAs":"https://academic.oup.com/dnaresearch/article/19/2/195/420778","author":[{"name":"Huang, Jianzi","affiliation":"1  College of Life Science ,  Shenzhen University ,  Shenzhen 518060 ,  China","@type":"Person"},{"name":"Lu, Xiang","affiliation":"1  College of Life Science ,  Shenzhen University ,  Shenzhen 518060 ,  China","@type":"Person"},{"name":"Yan, Hao","affiliation":"1  College of Life Science ,  Shenzhen University ,  Shenzhen 518060 ,  China","@type":"Person"},{"name":"Chen, Shouyi","affiliation":"2  Institute of Genetics and Developmental Biology ,  Chinese Academy of Sciences ,  Beijing 100101 ,  China","@type":"Person"},{"name":"Zhang, Wanke","affiliation":"2  Institute of Genetics and Developmental Biology ,  Chinese Academy of Sciences ,  Beijing 100101 ,  China","@type":"Person"},{"name":"Huang, Rongfeng","affiliation":"3  Biotechnology Research Institute ,  Chinese Academy of Agricultural Sciences ,  Beijing 100081 ,  China","@type":"Person"},{"name":"Zheng, Yizhi","affiliation":"1  College of Life Science ,  Shenzhen University ,  Shenzhen 518060 ,  China","@type":"Person"}],"description":"Abstract.  Semi-mangroves form a group of transitional species between glycophytes and halophytes, and hold unique potential for learning molecular mechanisms u","pageStart":"195","pageEnd":"207","thumbnailURL":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/2/1/m_cover.gif?Expires=2147483647&Signature=EMTi8MiwskfL8whcuG7gPpMQVkoTywhsN9k2fcs1qMWyjLc9OmU3zzfeux1LcC8edMnaqIc6tYNqgsUJ8hCH0UQnGgbUVvjHa69ZDqc3rwPQg3T0QzS9XdepkqXV57OA92AfnyIdwEzEggsZ18zDGYx5fC~exu1TTRcnk2bBo6TdYsVgw52BF-63NakC5-nPvHQVSqNzjaFosams6g1DZ~UchQ1c7lmvgwn6cnAg013zUUb9XUUhijVu63SQWmhw0ByRocoE2U4pz7BLDSieaJZqPdVUaKjHeYkUlMHEhZhGv0eccckPuOqL2UhMZ0FzVdPiNB6hKa39ET8qYgw8~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA","headline":"Transcriptome Characterization and Sequencing-Based Identification of Salt-Responsive Genes in  Millettia pinnata , a Semi-Mangrove Plant","image":"https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/2/1/m_cover.gif?Expires=2147483647&Signature=EMTi8MiwskfL8whcuG7gPpMQVkoTywhsN9k2fcs1qMWyjLc9OmU3zzfeux1LcC8edMnaqIc6tYNqgsUJ8hCH0UQnGgbUVvjHa69ZDqc3rwPQg3T0QzS9XdepkqXV57OA92AfnyIdwEzEggsZ18zDGYx5fC~exu1TTRcnk2bBo6TdYsVgw52BF-63NakC5-nPvHQVSqNzjaFosams6g1DZ~UchQ1c7lmvgwn6cnAg013zUUb9XUUhijVu63SQWmhw0ByRocoE2U4pz7BLDSieaJZqPdVUaKjHeYkUlMHEhZhGv0eccckPuOqL2UhMZ0FzVdPiNB6hKa39ET8qYgw8~Q__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA"}
    </script>
    <meta property="og:type" content="article" />
    <meta property="og:title" content="Transcriptome Characterization and Sequencing-Based Identification of Salt-Responsive Genes in  Millettia pinnata , a Semi-Mangrove Plant" />
    <meta property="og:description" content="Abstract.  Semi-mangroves form a group of transitional species between glycophytes and halophytes, and hold unique potential for learning molecular mechanisms u" />
    <meta property="og:url" content="https://academic.oup.com/dnaresearch/article/19/2/195/420778" />
    <meta property="og:image" content="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/25/5/1/m_dnares_25_5cover.png?Expires=2147483647&amp;Signature=TAvyHl8yqYv2li5BYt7w2uw0ovclwcjxHsLyTqDx3ndYPVImDx6jnn0eGbe2u1UJ1xVuQvC5~b186g7~tvRh84ZJzsvJ5Z55oqwNn8CXXToo21ZaQo57YPdq3ahP1pzvVgCjKSlDXb2~TT1QzIT7qvGI2zrr9g4XGebQkKEP2dKcJcTkWnhmXzNIkDeR8vghaIsS0de7M5oWzyZ4rYrc2DGqFkGjrbXYu-AQBLCmaDBR3wOdgy7QhHC2OrR56uaI3ZcQ5i8JBfsTLTEaOgQGnv-A0s0jvKlEoqNEmugy6HV7Y75-jdxzx452QI3dYZzZsMH7mc0rPtGHwoNhJj2OOw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" />
    <meta property="og:updated_time" content="2012-02-19" />
    <meta property="og:site_name" content="OUP Academic" />
<meta name="citation_author" content="Huang, Jianzi" /><meta name="citation_author_institution" content="College of Life Science, Shenzhen University, Shenzhen 518060, China" /><meta name="citation_author" content="Lu, Xiang" /><meta name="citation_author_institution" content="College of Life Science, Shenzhen University, Shenzhen 518060, China" /><meta name="citation_author" content="Yan, Hao" /><meta name="citation_author_institution" content="College of Life Science, Shenzhen University, Shenzhen 518060, China" /><meta name="citation_author" content="Chen, Shouyi" /><meta name="citation_author_institution" content="Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing 100101, China" /><meta name="citation_author" content="Zhang, Wanke" /><meta name="citation_author_institution" content="Institute of Genetics and Developmental Biology, Chinese Academy of Sciences, Beijing 100101, China" /><meta name="citation_author" content="Huang, Rongfeng" /><meta name="citation_author_institution" content="Biotechnology Research Institute, Chinese Academy of Agricultural Sciences, Beijing 100081, China" /><meta name="citation_author" content="Zheng, Yizhi" /><meta name="citation_author_institution" content="College of Life Science, Shenzhen University, Shenzhen 518060, China" /><meta name="citation_title" content="Transcriptome Characterization and Sequencing-Based Identification of Salt-Responsive Genes in Millettia pinnata, a Semi-Mangrove Plant" /><meta name="citation_firstpage" content="195" /><meta name="citation_lastpage" content="207" /><meta name="citation_doi" content="10.1093/dnares/dss004" /><meta name="citation_journal_title" content="DNA Research" /><meta name="citation_journal_abbrev" content="DNA Res" /><meta name="citation_volume" content="19" /><meta name="citation_issue" content="2" /><meta name="citation_publication_date" content="2012/04/01" /><meta name="citation_issn" content="1340-2838" /><meta name="citation_publisher" content="Oxford University Press" /><meta name="citation_reference" content="citation_title=Improving crop salt tolerance; J. Exp. Bot.;  citation_year=2004;  citation_volume=55;  citation_pages=307-19; " /><meta name="citation_reference" content="citation_title=Plant cellular and molecular responses to high salinity; Annu. Rev. Plant Physiol. Plant Mol. Biol.;  citation_year=2000;  citation_volume=51;  citation_pages=463-99; " /><meta name="citation_reference" content="citation_title=Salt and drought stress signal transduction in plants; Annu. Rev. Plant Biol.;  citation_year=2002;  citation_volume=53;  citation_pages=247-73; " /><meta name="citation_reference" content="citation_title=Monitoring the expression profiles of 7000 Arabidopsis genes under drought, cold and high-salinity stresses using a full-length cDNA microarray; Plant J.;  citation_year=2002;  citation_volume=31;  citation_pages=279-92; " /><meta name="citation_reference" content="citation_title=Comprehensive transcriptional profiling of NaCl-stressed Arabidopsis roots reveals novel classes of responsive genes; BMC Plant Biol.;  citation_year=2006;  citation_volume=6;  citation_pages=25" /><meta name="citation_reference" content="citation_title=Monitoring expression profiles of rice genes under cold, drought, and high-salinity stresses and abscisic acid application using cDNA microarray and RNA get-blot analyses; Plant Physiol.;  citation_year=2003;  citation_volume=133;  citation_pages=1755-67; " /><meta name="citation_reference" content="citation_title=Genome-wide transcriptional analysis of salinity stressed japonica and indica rice genotypes during panicle initiation stage; Plant Mol. Biol.;  citation_year=2007;  citation_volume=63;  citation_pages=609-23; " /><meta name="citation_reference" content="citation_title=Temporal progression of gene expression responses to salt shock in maize roots; Plant Mol. Biol.;  citation_year=2003;  citation_volume=52;  citation_pages=873-91; " /><meta name="citation_reference" content="citation_title=Comparative profiles of gene expression in leaves and roots of maize seedlings under conditions of salt stress and the removal of salt stress; Plant Cell Physiol.;  citation_year=2009;  citation_volume=50;  citation_pages=889-903; " /><meta name="citation_reference" content="citation_title=Genes that are uniquely stress regulated in salt overly sensitive (SOS) mutants; Plant Physiol.;  citation_year=2001;  citation_volume=126;  citation_pages=363-75; " /><meta name="citation_reference" content="citation_title=ERF protein JERF1 that transcriptionally modulates the expression of abscisic acid biosynthesis-related gene enhances the tolerance under salinity and cold in tobacco; Planta;  citation_year=2007;  citation_volume=226;  citation_pages=815-25; " /><meta name="citation_reference" content="citation_title=Ethylene signaling regulates salt stress response: an overview; Plant Signal. Behav.;  citation_year=2008;  citation_volume=3;  citation_pages=761-3; " /><meta name="citation_reference" content="citation_title=Calcium-dependent protein kinases play an essential role in a plant defence response; Embo J.;  citation_year=2001;  citation_volume=20;  citation_pages=5556-67; " /><meta name="citation_reference" content="citation_title=Salt cress. A halophyte and cryophyte Arabidopsis relative model system and its applicability to molecular genetic analyses of growth and development of extremophiles; Plant Physiol.;  citation_year=2004;  citation_volume=135;  citation_pages=1718-37; " /><meta name="citation_reference" content="citation_title=Comparative genomics in salt tolerance between Arabidopsis and Arabidopsis-related halophyte salt cress using Arabidopsis microarray; Plant Physiol.;  citation_year=2004;  citation_volume=135;  citation_pages=1697-709; " /><meta name="citation_reference" content="citation_title=Salinity stress adaptation competence in the extremophile Thellungiella halophila in comparison with its relative Arabidopsis thaliana; Plant J.;  citation_year=2005;  citation_volume=44;  citation_pages=826-39; " /><meta name="citation_reference" content="citation_title=Plant salt tolerance; Trends Plant Sci.;  citation_year=2001;  citation_volume=6;  citation_pages=66-71; " /><meta name="citation_reference" content="citation_title=Differentiation between true mangroves and mangrove associates based on leaf traits and salt contents; J. Plant Ecol.;  citation_year=2010; " /><meta name="citation_reference" content="citation_title=Trees, shrubs and grasses on saline soils of Indo-Gangetic plains; Indian Forester;  citation_year=1993;  citation_volume=119;  citation_pages=630-47; " /><meta name="citation_reference" content="citation_title=Effect of soil salinity and sodicity on seedling growth and mineral compsition of Pongamia pinnata; Indian Forester;  citation_year=1999;  citation_volume=125;  citation_pages=618-22; " /><meta name="citation_reference" content="citation_title=Preparation of biodiesel from crude oil of Pongamia pinnata; Bioresour. Technol.;  citation_year=2005;  citation_volume=96;  citation_pages=1425-9; " /><meta name="citation_reference" content="citation_title=High yield and conversion of biodiesel from a nonedible feedstock (Pongamia pinnata); J. Agr. Food Chem.;  citation_year=2010;  citation_volume=58;  citation_pages=242-7; " /><meta name="citation_reference" content="citation_title=Pongamia pinnata: an untapped resource for the biofuels industry of the future; Bioenergy Res.;  citation_year=2008;  citation_volume=1;  citation_pages=2-11; " /><meta name="citation_reference" content="citation_title=Protective influence of Pongamia pinnata (Karanja) on blood ammonia and urea levels in ammonium chloride-induced hyperammonemia: antihyperammonemic effect of the leaf extract; J. Appl. Biomed.;  citation_year=2005;  citation_volume=3;  citation_pages=133-8; " /><meta name="citation_reference" content="citation_title=Investigation of antihyperglycaemic activity of aqueous and petroleum ether extract of stem bark of Pongamia pinnata on serum glucose level in diabetic mice; J. Ethnopharmacol.;  citation_year=2009;  citation_volume=123;  citation_pages=115-20; " /><meta name="citation_reference" content="citation_title=Monophyletic subgroups of the tribe Millettieae (Leguminosae) as revealed by phytochrome nucleotide sequence data; Am. J. Bot.;  citation_year=1998;  citation_volume=85;  citation_pages=412-33; " /><meta name="citation_reference" content="citation_title=Phylogenetic analysis of nuclear ribosomal ITS/5.8S sequences in the Tribe Millettieae (Fabaceae): Poecilanthe-Cyclolobium, the core Millettieae, and the Callerya group; Syst. Bot.;  citation_year=2002;  citation_volume=27;  citation_pages=722-33; " /><meta name="citation_reference" content="citation_title=Molecular phylogenetic analysis of mangroves: independent evolutionary origins of vivipary and salt secretion; Mol. Phylogenet. Evol.;  citation_year=2005;  citation_volume=34;  citation_pages=159-66; " /><meta name="citation_reference" content="citation_title=Toward the 1,000 dollars human genome; Pharmacogenomics.;  citation_year=2005;  citation_volume=6;  citation_pages=373-82; " /><meta name="citation_reference" content="citation_title=Genome sequencing in microfabricated high-density picolitre reactors; Nature;  citation_year=2005;  citation_volume=437;  citation_pages=376-80; " /><meta name="citation_reference" content="citation_title=An approach to transcriptome analysis of non-model organisms using short-read sequences; Genome Inform.;  citation_year=2008;  citation_volume=21;  citation_pages=3-14; " /><meta name="citation_reference" content="citation_title=De novo assembly of human genomes with massively parallel short read sequencing; citation_text=doi/10.1101/gr.097261.109; Genome Res;  citation_year=2009; " /><meta name="citation_reference" content="citation_title=TIGR Gene Indices clustering tools (TGICL): a software system for fast clustering of large EST datasets; Bioinformatics;  citation_year=2003;  citation_volume=19;  citation_pages=651-2; " /><meta name="citation_reference" content="citation_title=ESTScan: a program for detecting, evaluating, and reconstructing potential coding regions in EST sequences; Proc. Int. Conf. Intell. Syst. Mol. Biol.;  citation_year=1999;  citation_pages=138-48; " /><meta name="citation_reference" content="citation_title=Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research; Bioinformatics;  citation_year=2005;  citation_volume=21;  citation_pages=3674-6; " /><meta name="citation_reference" content="citation_title=WEGO: a web tool for plotting GO annotations; Nucleic Acids Res.;  citation_year=2006;  citation_volume=34;  citation_pages=W293-7; " /><meta name="citation_reference" content="citation_title=Mapping and quantifying mammalian transcriptomes by RNA-Seq; Nat. Methods;  citation_year=2008;  citation_volume=5;  citation_pages=621-8; " /><meta name="citation_reference" content="citation_title=SOAP2: an improved ultrafast tool for short read alignment; Bioinformatics;  citation_year=2009;  citation_volume=25;  citation_pages=1966-7; " /><meta name="citation_reference" content="citation_title=DEGseq: an R package for identifying differentially expressed genes from RNA-seq data; Bioinformatics;  citation_year=2010;  citation_volume=26;  citation_pages=136-8; " /><meta name="citation_reference" content="citation_title=Phytophthora sojae: root rot pathogen of soybean and model oomycete; Mol. Plant Pathol.;  citation_year=2007;  citation_volume=8;  citation_pages=1-8; " /><meta name="citation_reference" content="citation_title=Microarray analysis of 7029 gene expression patterns in burma mangrove under high-salinity stress; Plant Sci.;  citation_year=2007;  citation_volume=172;  citation_pages=948-57; " /><meta name="citation_reference" content="citation_title=Transcriptional and physiological study of the response of Burma mangrove (Bruguiera gymnorhiza) to salt and osmotic stress; Plant Mol. Biol.;  citation_year=2008;  citation_volume=68;  citation_pages=119-29; " /><meta name="citation_reference" content="citation_title=Linking the salt transcriptome with physiological responses of a salt-resistant Populus species as a strategy to identify genes important for stress acclimation; Plant Physiol.;  citation_year=2010;  citation_volume=154;  citation_pages=1697-709; " /><meta name="citation_reference" content="citation_publisher=Cambridge University Press, Cambridge; The Botany of Mangroves;  citation_year=1986; " /><meta name="citation_reference" content="citation_title=Generation and analysis of expressed sequence tags from the salt-tolerant mangrove species Avicennia marina (Forsk) Vierh; Theor. Appl. Genet.;  citation_year=2005;  citation_volume=110;  citation_pages=416-24; " /><meta name="citation_reference" content="citation_title=Sequencing and analysis of 14,842 expressed sequence tags of burma mangrove, Bruguiera gymnorrhiza; , Plant Sci.;  citation_year=2006;  citation_volume=171;  citation_pages=234-41; " /><meta name="citation_reference" content="citation_title=Generation and analysis of expressed sequence tags from the mangrove plant, Acanthus ebracteatus Vahl; Tree Genet. Genomes;  citation_year=2006;  citation_volume=2;  citation_pages=196-201; " /><meta name="citation_reference" content="citation_title=Shedding light on an extremophile lifestyle through transcriptomics; New Phytol.;  citation_year=2009;  citation_volume=183;  citation_pages=764-75; " /><meta name="citation_reference" content="citation_title=De novo assembly of chickpea transcriptome using short reads for gene discovery and marker identification; DNA Res.;  citation_year=2011;  citation_volume=18;  citation_pages=53-63; " /><meta name="citation_reference" content="citation_title=Deep sequencing of the Camellia sinensis transcriptome revealed candidate genes for major metabolic pathways of tea-specific compounds; BMC Genomics;  citation_year=2011;  citation_volume=12;  citation_pages=131" /><meta name="citation_reference" content="citation_title=Assembly, gene annotation and marker development using 454 floral transcriptome sequences in Ziziphus Celata (Rhamnaceae), a highly endangered, Florida endemic plant; DNA Res.;  citation_year=2011; " /><meta name="citation_reference" content="citation_title=Transcriptome sequencing in an ecologically important tree species: assembly, annotation, and marker discovery; BMC Genomics;  citation_year=2010;  citation_volume=11;  citation_pages=180" /><meta name="citation_reference" content="citation_title=Comparative transcriptomics for mangrove species: an expanding resource; Funct. Integr. Genomics;  citation_year=2010;  citation_volume=10;  citation_pages=523-32; " /><meta name="citation_reference" content="citation_title=Adaptation to salinity in mangroves: Implication on the evolution of salt-tolerance; Chinese Sci. Bull.;  citation_year=2008;  citation_volume=53;  citation_pages=1708-15; " /><meta name="citation_reference" content="citation_title=An expression database for roots of the model legume Medicago truncatula under salt stress; BMC Genomics;  citation_year=2009;  citation_volume=10;  citation_pages=517" /><meta name="citation_reference" content="citation_title=RNA-seq: an assessment of technical reproducibility and comparison with gene expression arrays; Genome Res.;  citation_year=2008;  citation_volume=18;  citation_pages=1509-17; " /><meta name="citation_fulltext_world_readable" content="" /><meta name="citation_pdf_url" content="https://academic.oup.com/dnaresearch/article-pdf/19/2/195/1165922/dss004.pdf" /><meta name="description" content="Abstract.  Semi-mangroves form a group of transitional species between glycophytes and halophytes, and hold unique potential for learning molecular mechanisms u" />
        <meta name="citation_xml_url" content="https://academic.oup.com/dnaresearch/article-xml/19/2/195/420778" />



    <link rel="canonical" href="https://academic.oup.com/dnaresearch/article/19/2/195/420778" />


        <!-- Start: GPT Async -->

<script async='async' src='https://www.googletagservices.com/tag/js/gpt.js'></script>
<script>
  var googletag = googletag || {};
  googletag.cmd = googletag.cmd || [];
</script>



<script type='text/javascript'>
  var gptAdSlots = [];
  googletag.cmd.push(function() {
    var pathChar = '/';
    
    var mapping_ad1 = googletag.sizeMapping().
    addSize([1024, 0], [[970, 90], [728, 90]]).
    addSize([768, 0], [728, 90]).
    addSize([0, 0], [320, 50]).build();
    gptAdSlots["ad1"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad1', [[970, 90], [728, 90], [320, 50]], 'adBlockHeader').
    defineSizeMapping(mapping_ad1).
    addService(googletag.pubads());
    
    var mapping_ad2 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad2"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad2', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyTop').
    defineSizeMapping(mapping_ad2).
    addService(googletag.pubads());
    
    var mapping_ad3 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [[300, 250], [300, 600], [160, 600]]).build();
    gptAdSlots["ad3"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad3', [[300, 250], [160, 600], [300, 600]], 'adBlockMainBodyBottom').
    defineSizeMapping(mapping_ad3).
    addService(googletag.pubads());
    
    var mapping_ad4 = googletag.sizeMapping().
	addSize([0, 0], []).
    addSize([768, 0], [728, 90]).build();
    gptAdSlots["ad4"] = googletag.defineSlot(pathChar+'116097782'+pathChar+'dnaresearch_Supplement_Ad4', [728, 90], 'adBlockFooter').
    defineSizeMapping(mapping_ad4).
    addService(googletag.pubads());
    
    googletag.pubads().setTargeting("jnlspage", "article");
    googletag.pubads().setTargeting("jnlsurl", "dnaresearch/article/19/2/195/420778");
    googletag.pubads().enableSingleRequest();
    googletag.pubads().collapseEmptyDivs();
    
  });
</script>


    <script>
        var oupcookiepolicy_siteid = "journals";;	  	// the website id	
        var oupcookiepolicy_messagetype = 'implied';			            // type of alert message e.g, implied / explicit
        var oupcookiepolicy_preferredlanguage = "en";		// preferred language of the website
        var oupcookiepolicy_impliedmessageclass = 'cookiepolicyimplied';	// the css class for implied alert message
        var oupcookiepolicy_documentroot = '/';	  			                // the document root the cookie is set to
    </script>
    <noscript>We use cookies to enhance your experience on our website. By continuing to use our website, you are agreeing to our use of cookies. You can change your cookie settings at any time. <a href="https://global.oup.com/cookiepolicy/">Find out more</a></noscript>

    <script type="text/javascript">
        var NTPT_PGEXTRA='event_type=full-text&supplier_tag=SC_Journals&siteid=dnares&authentication_method=IP&authzrequired=false&account_id=2777753&account_list=2777753,2796125&authnips=128.235.12.133&doi=10.1093/dnares/dss004';
    </script>
<title>
	Transcriptome Characterization and Sequencing-Based Identification of Salt-Responsive Genes in Millettia pinnata, a Semi-Mangrove Plant | DNA Research | Oxford Academic
</title></head>

<body data-sitename="dnaresearch" class="off-canvas pg_article " theme-dnaresearch>
    
            <!-- Google Tag Manager (noscript) -->
            <noscript><iframe src="https://www.googletagmanager.com/ns.html?id=GTM-W6DD7HV"height="0" width="0" style="display:none;visibility:hidden"></iframe></noscript>
            <!-- End Google Tag Manager (noscript) -->
    
    
    <a href="#main" class="skipnav">Skip to Main Content</a>
    <form method="post" action="./420778" id="webform">
<input type="hidden" name="__VIEWSTATE" id="__VIEWSTATE" value="/wEPDwUKMTAxNjEyOTA4OQ9kFgJmD2QWAmYPZBYEZg9kFgICDg9kFgICAQ9kFgJmDxYCHgRUZXh0BYV8PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIdWFuZywgSmlhbnppIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29sbGVnZSBvZiBMaWZlIFNjaWVuY2UsIFNoZW56aGVuIFVuaXZlcnNpdHksIFNoZW56aGVuIDUxODA2MCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJMdSwgWGlhbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJDb2xsZWdlIG9mIExpZmUgU2NpZW5jZSwgU2hlbnpoZW4gVW5pdmVyc2l0eSwgU2hlbnpoZW4gNTE4MDYwLCBDaGluYSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9hdXRob3IiIGNvbnRlbnQ9IllhbiwgSGFvIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29sbGVnZSBvZiBMaWZlIFNjaWVuY2UsIFNoZW56aGVuIFVuaXZlcnNpdHksIFNoZW56aGVuIDUxODA2MCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJDaGVuLCBTaG91eWkiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgR2VuZXRpY3MgYW5kIERldmVsb3BtZW50YWwgQmlvbG9neSwgQ2hpbmVzZSBBY2FkZW15IG9mIFNjaWVuY2VzLCBCZWlqaW5nIDEwMDEwMSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJaaGFuZywgV2Fua2UiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJJbnN0aXR1dGUgb2YgR2VuZXRpY3MgYW5kIERldmVsb3BtZW50YWwgQmlvbG9neSwgQ2hpbmVzZSBBY2FkZW15IG9mIFNjaWVuY2VzLCBCZWlqaW5nIDEwMDEwMSwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yIiBjb250ZW50PSJIdWFuZywgUm9uZ2ZlbmciIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fYXV0aG9yX2luc3RpdHV0aW9uIiBjb250ZW50PSJCaW90ZWNobm9sb2d5IFJlc2VhcmNoIEluc3RpdHV0ZSwgQ2hpbmVzZSBBY2FkZW15IG9mIEFncmljdWx0dXJhbCBTY2llbmNlcywgQmVpamluZyAxMDAwODEsIENoaW5hIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvciIgY29udGVudD0iWmhlbmcsIFlpemhpIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2F1dGhvcl9pbnN0aXR1dGlvbiIgY29udGVudD0iQ29sbGVnZSBvZiBMaWZlIFNjaWVuY2UsIFNoZW56aGVuIFVuaXZlcnNpdHksIFNoZW56aGVuIDUxODA2MCwgQ2hpbmEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fdGl0bGUiIGNvbnRlbnQ9IlRyYW5zY3JpcHRvbWUgQ2hhcmFjdGVyaXphdGlvbiBhbmQgU2VxdWVuY2luZy1CYXNlZCBJZGVudGlmaWNhdGlvbiBvZiBTYWx0LVJlc3BvbnNpdmUgR2VuZXMgaW4gTWlsbGV0dGlhIHBpbm5hdGEsIGEgU2VtaS1NYW5ncm92ZSBQbGFudCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9maXJzdHBhZ2UiIGNvbnRlbnQ9IjE5NSIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9sYXN0cGFnZSIgY29udGVudD0iMjA3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2RvaSIgY29udGVudD0iMTAuMTA5My9kbmFyZXMvZHNzMDA0IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2pvdXJuYWxfdGl0bGUiIGNvbnRlbnQ9IkROQSBSZXNlYXJjaCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9qb3VybmFsX2FiYnJldiIgY29udGVudD0iRE5BIFJlcyIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl92b2x1bWUiIGNvbnRlbnQ9IjE5IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX2lzc3VlIiBjb250ZW50PSIyIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1YmxpY2F0aW9uX2RhdGUiIGNvbnRlbnQ9IjIwMTIvMDQvMDEiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25faXNzbiIgY29udGVudD0iMTM0MC0yODM4IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3B1Ymxpc2hlciIgY29udGVudD0iT3hmb3JkIFVuaXZlcnNpdHkgUHJlc3MiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1JbXByb3ZpbmcgY3JvcCBzYWx0IHRvbGVyYW5jZTsgSi4gRXhwLiBCb3QuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTU1OyAgY2l0YXRpb25fcGFnZXM9MzA3LTE5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QbGFudCBjZWxsdWxhciBhbmQgbW9sZWN1bGFyIHJlc3BvbnNlcyB0byBoaWdoIHNhbGluaXR5OyBBbm51LiBSZXYuIFBsYW50IFBoeXNpb2wuIFBsYW50IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDA7ICBjaXRhdGlvbl92b2x1bWU9NTE7ICBjaXRhdGlvbl9wYWdlcz00NjMtOTk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNhbHQgYW5kIGRyb3VnaHQgc3RyZXNzIHNpZ25hbCB0cmFuc2R1Y3Rpb24gaW4gcGxhbnRzOyBBbm51LiBSZXYuIFBsYW50IEJpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAyOyAgY2l0YXRpb25fdm9sdW1lPTUzOyAgY2l0YXRpb25fcGFnZXM9MjQ3LTczOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb25pdG9yaW5nIHRoZSBleHByZXNzaW9uIHByb2ZpbGVzIG9mIDcwMDAgQXJhYmlkb3BzaXMgZ2VuZXMgdW5kZXIgZHJvdWdodCwgY29sZCBhbmQgaGlnaC1zYWxpbml0eSBzdHJlc3NlcyB1c2luZyBhIGZ1bGwtbGVuZ3RoIGNETkEgbWljcm9hcnJheTsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDI7ICBjaXRhdGlvbl92b2x1bWU9MzE7ICBjaXRhdGlvbl9wYWdlcz0yNzktOTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXByZWhlbnNpdmUgdHJhbnNjcmlwdGlvbmFsIHByb2ZpbGluZyBvZiBOYUNsLXN0cmVzc2VkIEFyYWJpZG9wc2lzIHJvb3RzIHJldmVhbHMgbm92ZWwgY2xhc3NlcyBvZiByZXNwb25zaXZlIGdlbmVzOyBCTUMgUGxhbnQgQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTI1IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9uaXRvcmluZyBleHByZXNzaW9uIHByb2ZpbGVzIG9mIHJpY2UgZ2VuZXMgdW5kZXIgY29sZCwgZHJvdWdodCwgYW5kIGhpZ2gtc2FsaW5pdHkgc3RyZXNzZXMgYW5kIGFic2Npc2ljIGFjaWQgYXBwbGljYXRpb24gdXNpbmcgY0ROQSBtaWNyb2FycmF5IGFuZCBSTkEgZ2V0LWJsb3QgYW5hbHlzZXM7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTEzMzsgIGNpdGF0aW9uX3BhZ2VzPTE3NTUtNjc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbm9tZS13aWRlIHRyYW5zY3JpcHRpb25hbCBhbmFseXNpcyBvZiBzYWxpbml0eSBzdHJlc3NlZCBqYXBvbmljYSBhbmQgaW5kaWNhIHJpY2UgZ2Vub3R5cGVzIGR1cmluZyBwYW5pY2xlIGluaXRpYXRpb24gc3RhZ2U7IFBsYW50IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9NjM7ICBjaXRhdGlvbl9wYWdlcz02MDktMjM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRlbXBvcmFsIHByb2dyZXNzaW9uIG9mIGdlbmUgZXhwcmVzc2lvbiByZXNwb25zZXMgdG8gc2FsdCBzaG9jayBpbiBtYWl6ZSByb290czsgUGxhbnQgTW9sLiBCaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwMzsgIGNpdGF0aW9uX3ZvbHVtZT01MjsgIGNpdGF0aW9uX3BhZ2VzPTg3My05MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Q29tcGFyYXRpdmUgcHJvZmlsZXMgb2YgZ2VuZSBleHByZXNzaW9uIGluIGxlYXZlcyBhbmQgcm9vdHMgb2YgbWFpemUgc2VlZGxpbmdzIHVuZGVyIGNvbmRpdGlvbnMgb2Ygc2FsdCBzdHJlc3MgYW5kIHRoZSByZW1vdmFsIG9mIHNhbHQgc3RyZXNzOyBQbGFudCBDZWxsIFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTUwOyAgY2l0YXRpb25fcGFnZXM9ODg5LTkwMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXMgdGhhdCBhcmUgdW5pcXVlbHkgc3RyZXNzIHJlZ3VsYXRlZCBpbiBzYWx0IG92ZXJseSBzZW5zaXRpdmUgKFNPUykgbXV0YW50czsgUGxhbnQgUGh5c2lvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MTI2OyAgY2l0YXRpb25fcGFnZXM9MzYzLTc1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FUkYgcHJvdGVpbiBKRVJGMSB0aGF0IHRyYW5zY3JpcHRpb25hbGx5IG1vZHVsYXRlcyB0aGUgZXhwcmVzc2lvbiBvZiBhYnNjaXNpYyBhY2lkIGJpb3N5bnRoZXNpcy1yZWxhdGVkIGdlbmUgZW5oYW5jZXMgdGhlIHRvbGVyYW5jZSB1bmRlciBzYWxpbml0eSBhbmQgY29sZCBpbiB0b2JhY2NvOyBQbGFudGE7ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MjI2OyAgY2l0YXRpb25fcGFnZXM9ODE1LTI1OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1FdGh5bGVuZSBzaWduYWxpbmcgcmVndWxhdGVzIHNhbHQgc3RyZXNzIHJlc3BvbnNlOiBhbiBvdmVydmlldzsgUGxhbnQgU2lnbmFsLiBCZWhhdi47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MzsgIGNpdGF0aW9uX3BhZ2VzPTc2MS0zOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1DYWxjaXVtLWRlcGVuZGVudCBwcm90ZWluIGtpbmFzZXMgcGxheSBhbiBlc3NlbnRpYWwgcm9sZSBpbiBhIHBsYW50IGRlZmVuY2UgcmVzcG9uc2U7IEVtYm8gSi47ICBjaXRhdGlvbl95ZWFyPTIwMDE7ICBjaXRhdGlvbl92b2x1bWU9MjA7ICBjaXRhdGlvbl9wYWdlcz01NTU2LTY3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TYWx0IGNyZXNzLiBBIGhhbG9waHl0ZSBhbmQgY3J5b3BoeXRlIEFyYWJpZG9wc2lzIHJlbGF0aXZlIG1vZGVsIHN5c3RlbSBhbmQgaXRzIGFwcGxpY2FiaWxpdHkgdG8gbW9sZWN1bGFyIGdlbmV0aWMgYW5hbHlzZXMgb2YgZ3Jvd3RoIGFuZCBkZXZlbG9wbWVudCBvZiBleHRyZW1vcGhpbGVzOyBQbGFudCBQaHlzaW9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNDsgIGNpdGF0aW9uX3ZvbHVtZT0xMzU7ICBjaXRhdGlvbl9wYWdlcz0xNzE4LTM3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Db21wYXJhdGl2ZSBnZW5vbWljcyBpbiBzYWx0IHRvbGVyYW5jZSBiZXR3ZWVuIEFyYWJpZG9wc2lzIGFuZCBBcmFiaWRvcHNpcy1yZWxhdGVkIGhhbG9waHl0ZSBzYWx0IGNyZXNzIHVzaW5nIEFyYWJpZG9wc2lzIG1pY3JvYXJyYXk7IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDA0OyAgY2l0YXRpb25fdm9sdW1lPTEzNTsgIGNpdGF0aW9uX3BhZ2VzPTE2OTctNzA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TYWxpbml0eSBzdHJlc3MgYWRhcHRhdGlvbiBjb21wZXRlbmNlIGluIHRoZSBleHRyZW1vcGhpbGUgVGhlbGx1bmdpZWxsYSBoYWxvcGhpbGEgaW4gY29tcGFyaXNvbiB3aXRoIGl0cyByZWxhdGl2ZSBBcmFiaWRvcHNpcyB0aGFsaWFuYTsgUGxhbnQgSi47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NDQ7ICBjaXRhdGlvbl9wYWdlcz04MjYtMzk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBsYW50IHNhbHQgdG9sZXJhbmNlOyBUcmVuZHMgUGxhbnQgU2NpLjsgIGNpdGF0aW9uX3llYXI9MjAwMTsgIGNpdGF0aW9uX3ZvbHVtZT02OyAgY2l0YXRpb25fcGFnZXM9NjYtNzE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPURpZmZlcmVudGlhdGlvbiBiZXR3ZWVuIHRydWUgbWFuZ3JvdmVzIGFuZCBtYW5ncm92ZSBhc3NvY2lhdGVzIGJhc2VkIG9uIGxlYWYgdHJhaXRzIGFuZCBzYWx0IGNvbnRlbnRzOyBKLiBQbGFudCBFY29sLjsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJlZXMsIHNocnVicyBhbmQgZ3Jhc3NlcyBvbiBzYWxpbmUgc29pbHMgb2YgSW5kby1HYW5nZXRpYyBwbGFpbnM7IEluZGlhbiBGb3Jlc3RlcjsgIGNpdGF0aW9uX3llYXI9MTk5MzsgIGNpdGF0aW9uX3ZvbHVtZT0xMTk7ICBjaXRhdGlvbl9wYWdlcz02MzAtNDc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVmZmVjdCBvZiBzb2lsIHNhbGluaXR5IGFuZCBzb2RpY2l0eSBvbiBzZWVkbGluZyBncm93dGggYW5kIG1pbmVyYWwgY29tcHNpdGlvbiBvZiBQb25nYW1pYSBwaW5uYXRhOyBJbmRpYW4gRm9yZXN0ZXI7ICBjaXRhdGlvbl95ZWFyPTE5OTk7ICBjaXRhdGlvbl92b2x1bWU9MTI1OyAgY2l0YXRpb25fcGFnZXM9NjE4LTIyOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1QcmVwYXJhdGlvbiBvZiBiaW9kaWVzZWwgZnJvbSBjcnVkZSBvaWwgb2YgUG9uZ2FtaWEgcGlubmF0YTsgQmlvcmVzb3VyLiBUZWNobm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT05NjsgIGNpdGF0aW9uX3BhZ2VzPTE0MjUtOTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SGlnaCB5aWVsZCBhbmQgY29udmVyc2lvbiBvZiBiaW9kaWVzZWwgZnJvbSBhIG5vbmVkaWJsZSBmZWVkc3RvY2sgKFBvbmdhbWlhIHBpbm5hdGEpOyBKLiBBZ3IuIEZvb2QgQ2hlbS47ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9NTg7ICBjaXRhdGlvbl9wYWdlcz0yNDItNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UG9uZ2FtaWEgcGlubmF0YTogYW4gdW50YXBwZWQgcmVzb3VyY2UgZm9yIHRoZSBiaW9mdWVscyBpbmR1c3RyeSBvZiB0aGUgZnV0dXJlOyBCaW9lbmVyZ3kgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT0xOyAgY2l0YXRpb25fcGFnZXM9Mi0xMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UHJvdGVjdGl2ZSBpbmZsdWVuY2Ugb2YgUG9uZ2FtaWEgcGlubmF0YSAoS2FyYW5qYSkgb24gYmxvb2QgYW1tb25pYSBhbmQgdXJlYSBsZXZlbHMgaW4gYW1tb25pdW0gY2hsb3JpZGUtaW5kdWNlZCBoeXBlcmFtbW9uZW1pYTogYW50aWh5cGVyYW1tb25lbWljIGVmZmVjdCBvZiB0aGUgbGVhZiBleHRyYWN0OyBKLiBBcHBsLiBCaW9tZWQuOyAgY2l0YXRpb25feWVhcj0yMDA1OyAgY2l0YXRpb25fdm9sdW1lPTM7ICBjaXRhdGlvbl9wYWdlcz0xMzMtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9SW52ZXN0aWdhdGlvbiBvZiBhbnRpaHlwZXJnbHljYWVtaWMgYWN0aXZpdHkgb2YgYXF1ZW91cyBhbmQgcGV0cm9sZXVtIGV0aGVyIGV4dHJhY3Qgb2Ygc3RlbSBiYXJrIG9mIFBvbmdhbWlhIHBpbm5hdGEgb24gc2VydW0gZ2x1Y29zZSBsZXZlbCBpbiBkaWFiZXRpYyBtaWNlOyBKLiBFdGhub3BoYXJtYWNvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICBjaXRhdGlvbl92b2x1bWU9MTIzOyAgY2l0YXRpb25fcGFnZXM9MTE1LTIwOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1Nb25vcGh5bGV0aWMgc3ViZ3JvdXBzIG9mIHRoZSB0cmliZSBNaWxsZXR0aWVhZSAoTGVndW1pbm9zYWUpIGFzIHJldmVhbGVkIGJ5IHBoeXRvY2hyb21lIG51Y2xlb3RpZGUgc2VxdWVuY2UgZGF0YTsgQW0uIEouIEJvdC47ICBjaXRhdGlvbl95ZWFyPTE5OTg7ICBjaXRhdGlvbl92b2x1bWU9ODU7ICBjaXRhdGlvbl9wYWdlcz00MTItMzM7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVBoeWxvZ2VuZXRpYyBhbmFseXNpcyBvZiBudWNsZWFyIHJpYm9zb21hbCBJVFMvNS44UyBzZXF1ZW5jZXMgaW4gdGhlIFRyaWJlIE1pbGxldHRpZWFlIChGYWJhY2VhZSk6IFBvZWNpbGFudGhlLUN5Y2xvbG9iaXVtLCB0aGUgY29yZSBNaWxsZXR0aWVhZSwgYW5kIHRoZSBDYWxsZXJ5YSBncm91cDsgU3lzdC4gQm90LjsgIGNpdGF0aW9uX3llYXI9MjAwMjsgIGNpdGF0aW9uX3ZvbHVtZT0yNzsgIGNpdGF0aW9uX3BhZ2VzPTcyMi0zMzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9TW9sZWN1bGFyIHBoeWxvZ2VuZXRpYyBhbmFseXNpcyBvZiBtYW5ncm92ZXM6IGluZGVwZW5kZW50IGV2b2x1dGlvbmFyeSBvcmlnaW5zIG9mIHZpdmlwYXJ5IGFuZCBzYWx0IHNlY3JldGlvbjsgTW9sLiBQaHlsb2dlbmV0LiBFdm9sLjsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT0zNDsgIGNpdGF0aW9uX3BhZ2VzPTE1OS02NjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VG93YXJkIHRoZSAxLDAwMCBkb2xsYXJzIGh1bWFuIGdlbm9tZTsgUGhhcm1hY29nZW5vbWljcy47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9NjsgIGNpdGF0aW9uX3BhZ2VzPTM3My04MjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2Vub21lIHNlcXVlbmNpbmcgaW4gbWljcm9mYWJyaWNhdGVkIGhpZ2gtZGVuc2l0eSBwaWNvbGl0cmUgcmVhY3RvcnM7IE5hdHVyZTsgIGNpdGF0aW9uX3llYXI9MjAwNTsgIGNpdGF0aW9uX3ZvbHVtZT00Mzc7ICBjaXRhdGlvbl9wYWdlcz0zNzYtODA7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGFwcHJvYWNoIHRvIHRyYW5zY3JpcHRvbWUgYW5hbHlzaXMgb2Ygbm9uLW1vZGVsIG9yZ2FuaXNtcyB1c2luZyBzaG9ydC1yZWFkIHNlcXVlbmNlczsgR2Vub21lIEluZm9ybS47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zLTE0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZSBub3ZvIGFzc2VtYmx5IG9mIGh1bWFuIGdlbm9tZXMgd2l0aCBtYXNzaXZlbHkgcGFyYWxsZWwgc2hvcnQgcmVhZCBzZXF1ZW5jaW5nOyBjaXRhdGlvbl90ZXh0PWRvaS8xMC4xMTAxL2dyLjA5NzI2MS4xMDk7IEdlbm9tZSBSZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVRJR1IgR2VuZSBJbmRpY2VzIGNsdXN0ZXJpbmcgdG9vbHMgKFRHSUNMKTogYSBzb2Z0d2FyZSBzeXN0ZW0gZm9yIGZhc3QgY2x1c3RlcmluZyBvZiBsYXJnZSBFU1QgZGF0YXNldHM7IEJpb2luZm9ybWF0aWNzOyAgY2l0YXRpb25feWVhcj0yMDAzOyAgY2l0YXRpb25fdm9sdW1lPTE5OyAgY2l0YXRpb25fcGFnZXM9NjUxLTI7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUVTVFNjYW46IGEgcHJvZ3JhbSBmb3IgZGV0ZWN0aW5nLCBldmFsdWF0aW5nLCBhbmQgcmVjb25zdHJ1Y3RpbmcgcG90ZW50aWFsIGNvZGluZyByZWdpb25zIGluIEVTVCBzZXF1ZW5jZXM7IFByb2MuIEludC4gQ29uZi4gSW50ZWxsLiBTeXN0LiBNb2wuIEJpb2wuOyAgY2l0YXRpb25feWVhcj0xOTk5OyAgY2l0YXRpb25fcGFnZXM9MTM4LTQ4OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1CbGFzdDJHTzogYSB1bml2ZXJzYWwgdG9vbCBmb3IgYW5ub3RhdGlvbiwgdmlzdWFsaXphdGlvbiBhbmQgYW5hbHlzaXMgaW4gZnVuY3Rpb25hbCBnZW5vbWljcyByZXNlYXJjaDsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MjE7ICBjaXRhdGlvbl9wYWdlcz0zNjc0LTY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVdFR086IGEgd2ViIHRvb2wgZm9yIHBsb3R0aW5nIEdPIGFubm90YXRpb25zOyBOdWNsZWljIEFjaWRzIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MzQ7ICBjaXRhdGlvbl9wYWdlcz1XMjkzLTc7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1hcHBpbmcgYW5kIHF1YW50aWZ5aW5nIG1hbW1hbGlhbiB0cmFuc2NyaXB0b21lcyBieSBSTkEtU2VxOyBOYXQuIE1ldGhvZHM7ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9NTsgIGNpdGF0aW9uX3BhZ2VzPTYyMS04OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TT0FQMjogYW4gaW1wcm92ZWQgdWx0cmFmYXN0IHRvb2wgZm9yIHNob3J0IHJlYWQgYWxpZ25tZW50OyBCaW9pbmZvcm1hdGljczsgIGNpdGF0aW9uX3llYXI9MjAwOTsgIGNpdGF0aW9uX3ZvbHVtZT0yNTsgIGNpdGF0aW9uX3BhZ2VzPTE5NjYtNzsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9REVHc2VxOiBhbiBSIHBhY2thZ2UgZm9yIGlkZW50aWZ5aW5nIGRpZmZlcmVudGlhbGx5IGV4cHJlc3NlZCBnZW5lcyBmcm9tIFJOQS1zZXEgZGF0YTsgQmlvaW5mb3JtYXRpY3M7ICBjaXRhdGlvbl95ZWFyPTIwMTA7ICBjaXRhdGlvbl92b2x1bWU9MjY7ICBjaXRhdGlvbl9wYWdlcz0xMzYtODsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9UGh5dG9waHRob3JhIHNvamFlOiByb290IHJvdCBwYXRob2dlbiBvZiBzb3liZWFuIGFuZCBtb2RlbCBvb215Y2V0ZTsgTW9sLiBQbGFudCBQYXRob2wuOyAgY2l0YXRpb25feWVhcj0yMDA3OyAgY2l0YXRpb25fdm9sdW1lPTg7ICBjaXRhdGlvbl9wYWdlcz0xLTg7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPU1pY3JvYXJyYXkgYW5hbHlzaXMgb2YgNzAyOSBnZW5lIGV4cHJlc3Npb24gcGF0dGVybnMgaW4gYnVybWEgbWFuZ3JvdmUgdW5kZXIgaGlnaC1zYWxpbml0eSBzdHJlc3M7IFBsYW50IFNjaS47ICBjaXRhdGlvbl95ZWFyPTIwMDc7ICBjaXRhdGlvbl92b2x1bWU9MTcyOyAgY2l0YXRpb25fcGFnZXM9OTQ4LTU3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1UcmFuc2NyaXB0aW9uYWwgYW5kIHBoeXNpb2xvZ2ljYWwgc3R1ZHkgb2YgdGhlIHJlc3BvbnNlIG9mIEJ1cm1hIG1hbmdyb3ZlIChCcnVndWllcmEgZ3ltbm9yaGl6YSkgdG8gc2FsdCBhbmQgb3Ntb3RpYyBzdHJlc3M7IFBsYW50IE1vbC4gQmlvbC47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9Njg7ICBjaXRhdGlvbl9wYWdlcz0xMTktMjk7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUxpbmtpbmcgdGhlIHNhbHQgdHJhbnNjcmlwdG9tZSB3aXRoIHBoeXNpb2xvZ2ljYWwgcmVzcG9uc2VzIG9mIGEgc2FsdC1yZXNpc3RhbnQgUG9wdWx1cyBzcGVjaWVzIGFzIGEgc3RyYXRlZ3kgdG8gaWRlbnRpZnkgZ2VuZXMgaW1wb3J0YW50IGZvciBzdHJlc3MgYWNjbGltYXRpb247IFBsYW50IFBoeXNpb2wuOyAgY2l0YXRpb25feWVhcj0yMDEwOyAgY2l0YXRpb25fdm9sdW1lPTE1NDsgIGNpdGF0aW9uX3BhZ2VzPTE2OTctNzA5OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl9wdWJsaXNoZXI9Q2FtYnJpZGdlIFVuaXZlcnNpdHkgUHJlc3MsIENhbWJyaWRnZTsgVGhlIEJvdGFueSBvZiBNYW5ncm92ZXM7ICBjaXRhdGlvbl95ZWFyPTE5ODY7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUdlbmVyYXRpb24gYW5kIGFuYWx5c2lzIG9mIGV4cHJlc3NlZCBzZXF1ZW5jZSB0YWdzIGZyb20gdGhlIHNhbHQtdG9sZXJhbnQgbWFuZ3JvdmUgc3BlY2llcyBBdmljZW5uaWEgbWFyaW5hIChGb3JzaykgVmllcmg7IFRoZW9yLiBBcHBsLiBHZW5ldC47ICBjaXRhdGlvbl95ZWFyPTIwMDU7ICBjaXRhdGlvbl92b2x1bWU9MTEwOyAgY2l0YXRpb25fcGFnZXM9NDE2LTI0OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1TZXF1ZW5jaW5nIGFuZCBhbmFseXNpcyBvZiAxNCw4NDIgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3Mgb2YgYnVybWEgbWFuZ3JvdmUsIEJydWd1aWVyYSBneW1ub3JyaGl6YTsgLCBQbGFudCBTY2kuOyAgY2l0YXRpb25feWVhcj0yMDA2OyAgY2l0YXRpb25fdm9sdW1lPTE3MTsgIGNpdGF0aW9uX3BhZ2VzPTIzNC00MTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9R2VuZXJhdGlvbiBhbmQgYW5hbHlzaXMgb2YgZXhwcmVzc2VkIHNlcXVlbmNlIHRhZ3MgZnJvbSB0aGUgbWFuZ3JvdmUgcGxhbnQsIEFjYW50aHVzIGVicmFjdGVhdHVzIFZhaGw7IFRyZWUgR2VuZXQuIEdlbm9tZXM7ICBjaXRhdGlvbl95ZWFyPTIwMDY7ICBjaXRhdGlvbl92b2x1bWU9MjsgIGNpdGF0aW9uX3BhZ2VzPTE5Ni0yMDE7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPVNoZWRkaW5nIGxpZ2h0IG9uIGFuIGV4dHJlbW9waGlsZSBsaWZlc3R5bGUgdGhyb3VnaCB0cmFuc2NyaXB0b21pY3M7IE5ldyBQaHl0b2wuOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTE4MzsgIGNpdGF0aW9uX3BhZ2VzPTc2NC03NTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9RGUgbm92byBhc3NlbWJseSBvZiBjaGlja3BlYSB0cmFuc2NyaXB0b21lIHVzaW5nIHNob3J0IHJlYWRzIGZvciBnZW5lIGRpc2NvdmVyeSBhbmQgbWFya2VyIGlkZW50aWZpY2F0aW9uOyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIGNpdGF0aW9uX3ZvbHVtZT0xODsgIGNpdGF0aW9uX3BhZ2VzPTUzLTYzOyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fcmVmZXJlbmNlIiBjb250ZW50PSJjaXRhdGlvbl90aXRsZT1EZWVwIHNlcXVlbmNpbmcgb2YgdGhlIENhbWVsbGlhIHNpbmVuc2lzIHRyYW5zY3JpcHRvbWUgcmV2ZWFsZWQgY2FuZGlkYXRlIGdlbmVzIGZvciBtYWpvciBtZXRhYm9saWMgcGF0aHdheXMgb2YgdGVhLXNwZWNpZmljIGNvbXBvdW5kczsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDExOyAgY2l0YXRpb25fdm9sdW1lPTEyOyAgY2l0YXRpb25fcGFnZXM9MTMxIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QXNzZW1ibHksIGdlbmUgYW5ub3RhdGlvbiBhbmQgbWFya2VyIGRldmVsb3BtZW50IHVzaW5nIDQ1NCBmbG9yYWwgdHJhbnNjcmlwdG9tZSBzZXF1ZW5jZXMgaW4gWml6aXBodXMgQ2VsYXRhIChSaGFtbmFjZWFlKSwgYSBoaWdobHkgZW5kYW5nZXJlZCwgRmxvcmlkYSBlbmRlbWljIHBsYW50OyBETkEgUmVzLjsgIGNpdGF0aW9uX3llYXI9MjAxMTsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9VHJhbnNjcmlwdG9tZSBzZXF1ZW5jaW5nIGluIGFuIGVjb2xvZ2ljYWxseSBpbXBvcnRhbnQgdHJlZSBzcGVjaWVzOiBhc3NlbWJseSwgYW5ub3RhdGlvbiwgYW5kIG1hcmtlciBkaXNjb3Zlcnk7IEJNQyBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMTsgIGNpdGF0aW9uX3BhZ2VzPTE4MCIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUNvbXBhcmF0aXZlIHRyYW5zY3JpcHRvbWljcyBmb3IgbWFuZ3JvdmUgc3BlY2llczogYW4gZXhwYW5kaW5nIHJlc291cmNlOyBGdW5jdC4gSW50ZWdyLiBHZW5vbWljczsgIGNpdGF0aW9uX3llYXI9MjAxMDsgIGNpdGF0aW9uX3ZvbHVtZT0xMDsgIGNpdGF0aW9uX3BhZ2VzPTUyMy0zMjsgIiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9QWRhcHRhdGlvbiB0byBzYWxpbml0eSBpbiBtYW5ncm92ZXM6IEltcGxpY2F0aW9uIG9uIHRoZSBldm9sdXRpb24gb2Ygc2FsdC10b2xlcmFuY2U7IENoaW5lc2UgU2NpLiBCdWxsLjsgIGNpdGF0aW9uX3llYXI9MjAwODsgIGNpdGF0aW9uX3ZvbHVtZT01MzsgIGNpdGF0aW9uX3BhZ2VzPTE3MDgtMTU7ICIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9yZWZlcmVuY2UiIGNvbnRlbnQ9ImNpdGF0aW9uX3RpdGxlPUFuIGV4cHJlc3Npb24gZGF0YWJhc2UgZm9yIHJvb3RzIG9mIHRoZSBtb2RlbCBsZWd1bWUgTWVkaWNhZ28gdHJ1bmNhdHVsYSB1bmRlciBzYWx0IHN0cmVzczsgQk1DIEdlbm9taWNzOyAgY2l0YXRpb25feWVhcj0yMDA5OyAgY2l0YXRpb25fdm9sdW1lPTEwOyAgY2l0YXRpb25fcGFnZXM9NTE3IiAvPjxtZXRhIG5hbWU9ImNpdGF0aW9uX3JlZmVyZW5jZSIgY29udGVudD0iY2l0YXRpb25fdGl0bGU9Uk5BLXNlcTogYW4gYXNzZXNzbWVudCBvZiB0ZWNobmljYWwgcmVwcm9kdWNpYmlsaXR5IGFuZCBjb21wYXJpc29uIHdpdGggZ2VuZSBleHByZXNzaW9uIGFycmF5czsgR2Vub21lIFJlcy47ICBjaXRhdGlvbl95ZWFyPTIwMDg7ICBjaXRhdGlvbl92b2x1bWU9MTg7ICBjaXRhdGlvbl9wYWdlcz0xNTA5LTE3OyAiIC8+PG1ldGEgbmFtZT0iY2l0YXRpb25fZnVsbHRleHRfd29ybGRfcmVhZGFibGUiIGNvbnRlbnQ9IiIgLz48bWV0YSBuYW1lPSJjaXRhdGlvbl9wZGZfdXJsIiBjb250ZW50PSJodHRwczovL2FjYWRlbWljLm91cC5jb20vZG5hcmVzZWFyY2gvYXJ0aWNsZS1wZGYvMTkvMi8xOTUvMTE2NTkyMi9kc3MwMDQucGRmIiAvPjxtZXRhIG5hbWU9ImRlc2NyaXB0aW9uIiBjb250ZW50PSJBYnN0cmFjdC4gIFNlbWktbWFuZ3JvdmVzIGZvcm0gYSBncm91cCBvZiB0cmFuc2l0aW9uYWwgc3BlY2llcyBiZXR3ZWVuIGdseWNvcGh5dGVzIGFuZCBoYWxvcGh5dGVzLCBhbmQgaG9sZCB1bmlxdWUgcG90ZW50aWFsIGZvciBsZWFybmluZyBtb2xlY3VsYXIgbWVjaGFuaXNtcyB1IiAvPmQCBA9kFgJmD2QWAmYPFgQeBGhyZWYFci8vb3VwLnNpbHZlcmNoYWlyLWNkbi5jb20vRG93bmxvYWRGaWxlL0Rvd25sb2FkSW1hZ2UuYXNweD9pbWFnZT0mUFBUdHlwZT1TbGlkZVNldCZhcj00MjA3NzgmeHNsdFBhdGg9fi9VSS9hcHAvWFNMVB4JaW5uZXJodG1sBRREb3dubG9hZCBhbGwgZmlndXJlc2RkqAVxaIX3sqD7JJTJ5CjZ1lrV6M0=" />

<input type="hidden" name="__VIEWSTATEGENERATOR" id="__VIEWSTATEGENERATOR" value="2173C2F0" />
        </form>
        
        <input id="hdnSiteID" type="hidden" value="5275" />
        <input id="hdnAdDelaySeconds" type="hidden" value="5000" />
        <input id="hdnAdConfigurationTop" type="hidden" value="default" />
        <input id="hdnAdConfigurationRightRail" type="hidden" value="sticky" />

        

        
    <div class="master-container">
        <section class="master-header row">
                <div class="widget widget-SitePageHeader widget-instance-SitePageHeader">
        
    <div class="ad-banner js-ad-banner-header">
    <div class="widget widget-AdBlock widget-instance-HeaderAd">
        

<div id="adBlockHeader"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockHeader'); });
    </script>
</div>

    </div>

    </div>

    <div class="oup-header">
        <div class="center-inner-row">

            <div class="oup-header-logo">
                <a href="/"><img src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oxford-academic-logo.svg" alt="Oxford University Press" class="oup-header-image"></a>
            </div><!-- /.oup-header-logo -->

            <ul class="oup-header-menu account-menu">

                <!-- MOBILE SEARCH  -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-search-toggle">
                        <i class="icon-menu_search"><span class="screenreader-text">Search</span></i>
                    </button>
                </li>

                <!-- MOBILE MENU -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-account-toggle">
                        <i class="icon-menu_account"><span class="screenreader-text">Account Menu</span></i>
                    </button>
                </li>

                <!-- MOBILE NAV -->
                <li class="oup-header-menu-item mobile">
                    <button href="" class="btn btn-as-icon mobile-dropdown-toggle mobile-nav-toggle">
                        <i class="icon-menu_hamburger"><span class="screenreader-text">Menu</span></i>
                    </button>
                </li>

                <!-- DESKTOP INSTITUTIONS -->
                    <li class="oup-header-menu-item desktop">
                        
                            <a href="" class="dropdown-toggle institution">NJ Inst of Technology <i class="icon-general-arrow-filled-down arrow-icon"></i></a>
                        <div class="dropdown-panel dropdown-panel-institution">
                            
                            <div class="dropdown-menu">
                                <ul>

                                        <li class="dropdown-institution-item">

NJ Inst of Technology
                                        </li>

                                </ul>

                            </div><!-- /.dropdown-menu -->
                        </div><!-- /.dropdown-panel .dropdown-panel-institution -->
                    </li><!-- /.oup-header-menu-item .desktop -->

                <!-- DESKTOP SIGN IN -->
                <li class="oup-header-menu-item desktop">
                    <a href="" class="dropdown-toggle signin" id="header-account-info-user-fullname">
                        
Sign In                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                    </a>
                    <div class="dropdown-panel dropdown-panel-signin dropdown-panel-form">
                        <div class="spinner"></div>
                    </div><!-- /.dropdown-panel -->
                </li>

                <!-- DESKTOP REGISTRATION -->
                    <li class="oup-header-menu-item desktop"><a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f195%2f420778" class="register">Register</a></li>

                <!-- LANGUAGE -->
                    <!-- <li class="oup-header-menu-item desktop">
                        <a href="" class="dropdown-toggle language">Language</a>
                        <div class="dropdown-panel">
                            <div class="dropdown-menu">
                                <ul>
                                    foreach (var lang in Model.LanguageOptions)
                                    {
                                        string selectedClass = (lang.SiteId == Model.SiteId ? " selected" : "");
                                        Html.Raw(String.Format("<li class=\"header-language-item{0}\"><a href=\"//{1}\">{2}</a></li>", selectedClass, lang.SiteUrl, lang.LanguageText))
                                        ;
                                    }
                                </ul>
                            </div>
                        </div>
                    </li> -->

            </ul><!-- /.oup-header-menu .account-menu -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-header -->

<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f195%2f420778" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->


    <div class="journal-header journal-bg">
        <div class="center-inner-row">
                <a href="/dnaresearch" class=" journal-logo-container">

            <img id="logo-DNAResearch" class="journal-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_title-2068071465.svg" alt="DNA Research" />

                </a>
            <div class="society-logo-block">
                    <div class="society-block-inner-wrap">
                            <a href="http://www.kazusa.or.jp/e/index.html" target="" class="society-logo-container">
                                <img id="logo-KazusaDNAResearchInstitute" class="society-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_h1-805551131.svg" alt="Kazusa DNA Research Institute" />
                            </a>
                                            </div><!-- /.society-block-inner-wrap - NOTE: used to vertically center one or more society logos -->
            </div> <!-- /.society-logo-block - NOTE: We want this element on the page, even if there are no society logos -->
        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-header -->

<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">


    <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-0">
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102797">
            <a href="/dnaresearch/issue" class="nav-link ">Issues</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102794">
            <a href="/dnaresearch/#" class="nav-link dummy-link">More Content</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102798">
            <a href="/dnaresearch/advance-articles" class="nav-link ">Advance articles</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102799">
            <a href="https://academic.oup.com/dnaresearch/pages/free_access_articles" class="nav-link ">Editor's Choice</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102795">
            <a href="/dnaresearch/#" class="nav-link dummy-link">Submit</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102800">
            <a href="/dnaresearch/pages/Instructions_To_Authors" class="nav-link ">Author Guidelines</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102801">
            <a href="http://mc.manuscriptcentral.com/dnar" class="nav-link ">Submission Site</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102802">
            <a href="http://www.oxfordjournals.org/en/authors/offprints.html" class="nav-link ">Order Offprints</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102803">
            <a href="http://www.oxfordjournals.org/en/oxford-open/index.html" class="nav-link ">Open Access Options</a>
            
        </li>
    </ul>

        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102804">
            <a href="/dnaresearch/subscribe" class="nav-link ">Purchase</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102805">
            <a href="/my-account/email-alerts" class="nav-link ">Alerts</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-0" id="site-menu-item-1102796">
            <a href="/dnaresearch/#" class="nav-link dummy-link">About</a>
                <i class="desktop-nav-arrow icon-general-arrow-filled-down arrow-icon"></i><i class="mobile-nav-arrow icon-general_arrow-down"></i>
    <ul class="site-menu site-menu-lvl-1">
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102806">
            <a href="/dnaresearch/pages/About" class="nav-link ">About DNA Research</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102807">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">About the Kazusa DNA Research Institute</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102808">
            <a href="/dnaresearch/pages/Editorial_Board" class="nav-link ">Editorial Board</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102809">
            <a href="http://www.oupmediainfo.com/" class="nav-link ">Advertising and Corporate Services</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102810">
            <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org" class="nav-link ">Journals Career Network</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102811">
            <a href="https://academic.oup.com/journals/pages/access_purchase/rights_and_permissions/self_archiving_policy_c" class="nav-link ">Self-Archiving Policy</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102812">
            <a href="http://www.oxfordjournals.org/en/access-purchase/dispatch-dates.html" class="nav-link ">Dispatch Dates</a>
            
        </li>
        <li class="site-menu-item site-menu-lvl-1" id="site-menu-item-1102813">
            <a href="http://www.kazusa.or.jp/e/index.html" class="nav-link ">Contact Us</a>
            
        </li>
    </ul>

        </li>
    </ul>
        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

    <input type="hidden" name="searchScope" id="hfSolrJournalID" value="" />
    <input type="hidden" name="parentSiteName" id="hfParentSiteName" value="Journals" />
    <input type="hidden" name="parentSiteUrl" id="hfparentSiteUrl" value="academic.oup.com/journals" />
    <input type="hidden" name="siteID" id="hfsiteID" value="5275" />
    <input type="hidden" name="parentSiteID" id="hfparentSiteID" value="5567" />
    <input type="hidden" id="hfSolrJournalName" value="" />
    <input type="hidden" id="hfSolrMaxAllowSearchChar" value="100" />
    <input type="hidden" id="hfJournalShortName" value="" />
    <input type="hidden" id="hfSearchPlaceholder" value="" />
    <input type="hidden" name="hfGlobalSearchSiteURL" id="hfGlobalSearchSiteURL" value="" />
    <input type="hidden" name="journalSiteScope" id="hfjournalSiteScope" value="Journals" />
    <input type="hidden" name="parentSiteScope" id="hfparentSiteScope" value="Parent" />
    <input type="hidden" name="defaultSearchURL" id="hfdefaultSearchURL" value="search-results?page=1&amp;q=" />
    <input type="hidden" name="issueSearch" id="hfIssueSearch" value="&amp;fl_IssueID=16354" />
    <input type="hidden" name="issueSiteScope" id="hfIssueSiteScope" value="Issue" />
<input id="hfDefaultAdvancedSearchUrl" name="hfDefaultAdvancedSearchUrl" type="hidden" value="advanced-search?page=1&amp;q=" />    <input type="hidden" name="hfSearchSiteURL" id="hfSiteURL" value="academic.oup.com/dnaresearch" />
    <script type="text/javascript">
        (function () {
            var hfSiteUrl = document.getElementById('hfSiteURL');
            var siteUrl = hfSiteUrl.value;
            var subdomainIndex = siteUrl.indexOf('/');

            hfSiteUrl.value = location.host + (subdomainIndex >= 0 ? siteUrl.substring(subdomainIndex) : '');
        })();
    </script>
<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

        </section>
        <main id="main" tabindex="-1">
            <section class="master-main row">
                <div class="center-inner-row no-overflow">
                    
    <div class="page-column-wrap">
        <div id="InfoColumn" class="page-column page-column--left">
            <div class="mobile-content-topbar hide">
                <button class="toggle-left-col toggle-left-col__article">Article Navigation</button>
            </div>
            <div class="info-inner-wrap">
                
                <button class="toggle-left-col__close btn-as-icon icon-general-close">
                    <span class="screenreader-text">Close mobile search navigation</span>
                </button>
                <div class="responsive-nav-title">Article navigation</div>

                    <div class="info-widget-wrap">
                        <div class="widget widget-IssueInfo widget-instance-OUP_IssueInfo_Article">
        

<div id="issueInfo-OUP_IssueInfo_Article" class="article-info-wrap clearfix">
    <i class="icon-general-close mobile-nav-btn nav-open"></i>
    <div class="article-issue-img">
            <a href="/dnaresearch/issue/19/2"><img id="issueImage" class="fb-featured-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/Issue/19/2/1/m_cover.gif?Expires=2147483647&amp;Signature=EMTi8MiwskfL8whcuG7gPpMQVkoTywhsN9k2fcs1qMWyjLc9OmU3zzfeux1LcC8edMnaqIc6tYNqgsUJ8hCH0UQnGgbUVvjHa69ZDqc3rwPQg3T0QzS9XdepkqXV57OA92AfnyIdwEzEggsZ18zDGYx5fC~exu1TTRcnk2bBo6TdYsVgw52BF-63NakC5-nPvHQVSqNzjaFosams6g1DZ~UchQ1c7lmvgwn6cnAg013zUUb9XUUhijVu63SQWmhw0ByRocoE2U4pz7BLDSieaJZqPdVUaKjHeYkUlMHEhZhGv0eccckPuOqL2UhMZ0FzVdPiNB6hKa39ET8qYgw8~Q__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Issue Cover" /></a>
    </div>
    <div class="article-issue-info">
        <div class="volume-issue__wrap">

            <a href="/dnaresearch/issue/19/2">
                        <div class="volume trailing-comma">Volume 19</div>
                        <div class="issue">Issue 2</div>

            </a>

        </div>
        <div class="ii-pub-date">
April 2012        </div>
        
    </div>

</div>
    </div>


                    <div class="content-nav">
                            <div class="widget widget-ArticleJumpLinks widget-instance-OUP_ArticleJumpLinks_Widget">
            <div class="contents-title">Article Contents</div>
    <ul class="jumplink-list">
            <li class="section-jump-link head-1" link-destination="6234549">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6234549">Abstract</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6234551">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6234551">1. Introduction</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6234558">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6234558">2. Materials and methods</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6234573">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6234573">3. Results</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6234607">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6234607">4. Discussion</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6234614">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6234614">Supplementary Data</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6234616">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6234616">Funding</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6234618">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6234618">Acknowledgements</a>
                </div>
            </li>
            <li class="section-jump-link head-1" link-destination="6234620">
                <div class="section-jump-link__link-wrap">

                    <a class="js-jumplink scrollTo" href="#6234620">References</a>
                </div>
            </li>
    </ul>

    </div>

	                </div>
            
                        <div class="widget widget-ArticleNavLinks widget-instance-OUP_ArticleNavLinks_Article">
        <ul class="inline-list">
        <li class="prev arrow">
            <a href="/dnaresearch/article/19/2/179/420473">&lt; Previous</a>
        </li>        
            <li class="empty arrow">&nbsp;</li>
</ul>
    </div>

                </div>
            </div>
        </div> 
    
        <div id="ContentColumn" class="page-column page-column--center">
            
             <div class="article-browse-top article-browse-mobile-nav">
                <div class="article-browse-mobile-nav-inner">
                    <button class="toggle-left-col toggle-left-col__article btn-as-link">
                        Article Navigation
                    </button>
                </div>
            </div>

            <div class="content-inner-wrap">
                    <div class="widget widget-ArticleTopInfo widget-instance-OUP_ArticleTop_Info_Widget">
        <div class="module-widget article-top-widget">

    

    
        <div class="access-state-logos all-viewports">
    
            </div>


    <div class="widget-items">
                <h1 class="wi-article-title article-title-main">Transcriptome Characterization and Sequencing-Based Identification of Salt-Responsive Genes in <em>Millettia pinnata</em>, a Semi-Mangrove Plant
<i class='icon-availability_open' title='Open Access' ></i>                </h1>
                <div class="wi-authors">
                    <div class="al-authors-list">
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Jianzi Huang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">College of Life Science</div>, <div class="insititution">Shenzhen University</div>, <div class="addr-line">Shenzhen 518060</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Jianzi+Huang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Huang J">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Huang J&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Jianzi Huang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Xiang Lu</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">College of Life Science</div>, <div class="insititution">Shenzhen University</div>, <div class="addr-line">Shenzhen 518060</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Xiang+Lu">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Lu X">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Lu X&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Xiang Lu</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Hao Yan</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">College of Life Science</div>, <div class="insititution">Shenzhen University</div>, <div class="addr-line">Shenzhen 518060</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Hao+Yan">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Yan H">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Yan H&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Hao Yan</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Shouyi Chen</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Institute of Genetics and Developmental Biology</div>, <div class="insititution">Chinese Academy of Sciences</div>, <div class="addr-line">Beijing 100101</div>, <div class="country">China</div></div>
        </div>





        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Shouyi+Chen">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Chen S">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Chen S&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Shouyi Chen</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Wanke Zhang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">2</div><div class="addr-line">Institute of Genetics and Developmental Biology</div>, <div class="insititution">Chinese Academy of Sciences</div>, <div class="addr-line">Beijing 100101</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 10-64886859 (W.Z.); +86 10-82106143 (R.H.); +86 755-26558941 (Y.Z.). Fax. +86 755-26534274 (Y.Z.). Email: <a href="mailto:wkzhang@genetics.ac.cn" target="_blank">wkzhang@genetics.ac.cn</a> (W.Z.); <a href="mailto:rfhuang@caas.net.cn" target="_blank">rfhuang@caas.net.cn</a> (R.H.); <a href="mailto:yzzheng@szu.edu.cn" target="_blank">yzzheng@szu.edu.cn</a> (Y.Z.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Wanke+Zhang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zhang W">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zhang W&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Wanke Zhang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Rongfeng Huang</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">3</div><div class="addr-line">Biotechnology Research Institute</div>, <div class="insititution">Chinese Academy of Agricultural Sciences</div>, <div class="addr-line">Beijing 100081</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 10-64886859 (W.Z.); +86 10-82106143 (R.H.); +86 755-26558941 (Y.Z.). Fax. +86 755-26534274 (Y.Z.). Email: <a href="mailto:wkzhang@genetics.ac.cn" target="_blank">wkzhang@genetics.ac.cn</a> (W.Z.); <a href="mailto:rfhuang@caas.net.cn" target="_blank">rfhuang@caas.net.cn</a> (R.H.); <a href="mailto:yzzheng@szu.edu.cn" target="_blank">yzzheng@szu.edu.cn</a> (Y.Z.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Rongfeng+Huang">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Huang R">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Huang R&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Rongfeng Huang</a>
                                </span>
                                <span class="al-author-name-more">

                                    <span class="al-author-info-wrap arrow-up">


<div id="authorInfo_OUP_ArticleTop_Info_Widget" class="info-card-author">
    <div class="name-role-wrap">
        <div class="info-card-name">Yizhi Zheng</div>
    </div>

        <div class="info-card-affilitation">
            <div class="aff"><div class="label -label">1</div><div class="addr-line">College of Life Science</div>, <div class="insititution">Shenzhen University</div>, <div class="addr-line">Shenzhen 518060</div>, <div class="country">China</div></div>
        </div>

                <div class="info-card-note">
                    <a reveal-id="cor1" data-open="cor1" class="link link-ref link-reveal xref-default">*</a>
                </div>

        <div class="info-author-correspondence">
            <div class="label -label">*</div>To whom correspondence should be addressed. Tel. +86 10-64886859 (W.Z.); +86 10-82106143 (R.H.); +86 755-26558941 (Y.Z.). Fax. +86 755-26534274 (Y.Z.). Email: <a href="mailto:wkzhang@genetics.ac.cn" target="_blank">wkzhang@genetics.ac.cn</a> (W.Z.); <a href="mailto:rfhuang@caas.net.cn" target="_blank">rfhuang@caas.net.cn</a> (R.H.); <a href="mailto:yzzheng@szu.edu.cn" target="_blank">yzzheng@szu.edu.cn</a> (Y.Z.)
        </div>



        <div class="info-card-search-label">
            Search for other works by this author on:
        </div>
            <div class="info-card-search info-card-search-oxford">
                <a href="/dnaresearch/search-results?f_Authors=Yizhi+Zheng">Oxford Academic</a>
            </div>
            <div class="info-card-search info-card-search-pubmed">
                <a href="http://www.ncbi.nlm.nih.gov/pubmed?cmd=search&amp;term=Zheng Y">PubMed</a>
            </div>
            <div class="info-card-search info-card-search-google">
                <a href="http://scholar.google.com/scholar?q=author:&quot;Zheng Y&quot;">Google Scholar</a>
            </div>

</div>
                                    </span>
                                    <a class="linked-name" href="javascript:;">Yizhi Zheng</a>
                                </span>

                                <div class="al-author-name al-author-footnotes">
                                    <div class="al-author-info-wrap arrow-up">
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id="AN1"><div class="fn"><div class="label -label">†</div><div class="footnote-content">These authors contributed equally to this work.</div></div></div>
    </div>
    <div class="widget widget-SingleSection widget-instance-OUP_FootnoteSection">
        <div content-id=""><div class="fn"><div class="footnote-content">Edited by Satoshi Tabata</div></div></div>
    </div>
                                    </div>
                                    <a class="js-linked-footnotes" href="javascript:;">Author Notes</a>
                                </div>
                    </div>
                </div>
            <div class="pub-history-wrap clearfix">

                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-primary"><em>DNA Research</em>, Volume 19, Issue 2, 1 April 2012, Pages 195–207, <a href='https://doi.org/10.1093/dnares/dss004'>https://doi.org/10.1093/dnares/dss004</a></div>
                    </div>
                    <div class="pub-history-row clearfix">
                        <div class="ww-citation-date-wrap">
                            <div class="citation-label">Published:</div>
                            <div class="citation-date">19 February 2012</div>
                        </div>
                            <div class="ww-citation-history-wrap">
                                <div class="citation-label">
                                    Article history<i class="icon-general-arrow-filled-down arrow-icon"></i>
                                </div>
                            </div>
                    </div>
                                    <div class="ww-history">
                            <div class="history-entry">
                                <div class="wi-state">Received:</div>
                                <div class="wi-date">19 November 2011</div>
                            </div>
                            <div class="history-entry">
                                <div class="wi-state">Accepted:</div>
                                <div class="wi-date">17 January 2012</div>
                            </div>

                        <div class="ww-history-close"><a>Close</a></div>
                    </div><!-- /.ww-history -->
            </div><!-- /.pub-history-wrap -->

    </div>
</div>

<script>
    $(document).ready(function () {
        $('.article-top-widget').on('click', '.ati-toggle-trigger', function () {
            $(this).find('.icon-general-add, .icon-minus').toggleClass('icon-minus icon-general-add');
            $(this).siblings('.ati-toggle-content').toggleClass('hide');
        });

        // In Chrome, an anchor tag with target="_blank" and a "mailto:" href opens a new tab/window as well as the email client
        // I suspect this behavior will be corrected in the future
        // Remove the target="_blank"
        $('ul.wi-affiliationList').find('a[href^="mailto:"]').each(function () {
            $(this).removeAttr('target');
        });
    });
</script>


    </div>
    <div class="widget widget-ArticleLinks widget-instance-OUP_Article_Links_Widget">
        


    </div>

        
                <div class="article-body">
                                    <div class="toolbar-wrap">
                    <div class="toolbar-inner-wrap"><!-- for mobile -->
                        <ul id="Toolbar" role="navigation">
                            
                            <li class="toolbar-item item-link">
                                <a class="split-view">
                                    <i class="icon-menu_split"></i>
                                    Split View
                                </a>
                            </li>
                            
                            <li class="toolbar-item item-with-dropdown item-views">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="FilterDrop" aria-haspopup="true">
                                    <i class="icon-menu_views"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Views</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ViewsDrop" class="f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li class="article-content-filter" data-content-filter="article-content"><a href="javascript:;"><span>Article contents</span></a></li>
                                    <li class="article-content-filter" data-content-filter="figures-tables"><a href="javascript:;"><span>Figures &amp; tables</span></a></li>
                                    <li class="article-content-filter" data-content-filter="video"><a href="javascript:;"><span>Video</span></a></li>
                                    <li class="article-content-filter" data-content-filter="audio"><a href="javascript:;"><span>Audio</span></a></li>
                                    <li class="article-content-filter" data-content-filter="supplementary-data"><a href="javascript:;"><span>Supplementary Data</span></a></li>
                                </ul>
                            </li>
                                <li class="toolbar-item item-pdf">
        <a class="al-link pdf article-pdfLink" data-article-id="420778" href="/dnaresearch/article-pdf/19/2/195/1165922/dss004.pdf">
            <i class="icon-menu_pdf"></i><span>PDF</span>
        </a>
    </li>

                            <li class="toolbar-item">
                                    <div class="widget widget-ToolboxGetCitation widget-instance-OUP_Get_Citation">
        
<a href="#" data-reveal-id="getCitation" data-reveal><i class="icon-read-more"></i><span>Cite</span></a>

<div id="getCitation" class="reveal-modal" data-reveal>
    <div class="modal-title">Citation</div>
    <p>Jianzi Huang, Xiang Lu, Hao Yan, Shouyi Chen, Wanke Zhang, Rongfeng Huang, Yizhi Zheng;  Transcriptome Characterization and Sequencing-Based Identification of Salt-Responsive Genes in <em>Millettia pinnata</em>, a Semi-Mangrove Plant, <em>DNA Research</em>, Volume 19, Issue 2, 1 April 2012, Pages 195–207, <a href='https://doi.org/10.1093/dnares/dss004'>https://doi.org/10.1093/dnares/dss004</a></p>
    <p class="citation-label">Download citation file:</p>  
    <ul>
        <li><a href="/dnaresearch/downloadcitation/420778?format=ris">RIS (Zotero)</a></li>
        <li><a href="/dnaresearch/downloadcitation/420778?format=ris">EndNote</a></li>
        <li><a href="/dnaresearch/downloadcitation/420778?format=bibtex">BibTex</a></li>
        <li><a href="/dnaresearch/downloadcitation/420778?format=txt">Medlars</a></li>
        <li><a href="/dnaresearch/downloadcitation/420778?format=ris">ProCite</a></li>
        <li><a href="/dnaresearch/downloadcitation/420778?format=txt">RefWorks</a></li>
        <li><a href="/dnaresearch/downloadcitation/420778?format=ris">Reference Manager</a></li>
    </ul>
    <hr/>
    
    <p>&#169; 2018 Oxford University Press</p>
    <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
</div>
    </div>
 
                            </li>
                            <li class="toolbar-item item-tools">
                                <i class="icon-menu_permissions"></i>
                                    <div class="widget widget-ToolboxPermissions widget-instance-">
            <div class="module-widget">
            <a href="https://s100.copyright.com/AppDispatchServlet?publisherName=OUP&amp;publication=1756-1663&amp;title=Transcriptome%20Characterization%20and%20Sequencing-Based%20Identification%20of%20Salt-Responsive%20Genes%20in%20%26lt%3Bem%26gt%3BMillettia%20pinnata%26lt%3B%2Fem%26gt%3B%2C%20a%20Semi-Mangrove%20Plant&amp;publicationDate=2012-02-19&amp;volumeNum=19&amp;issueNum=2&amp;author=Huang%2C%20Jianzi%3B%20Lu%2C%20Xiang&amp;startPage=195&amp;endPage=207&amp;contentId=10.1093%2Fdnares%2Fdss004&amp;oa=CC%20BY-NC&amp;copyright=Oxford%20University%20Press&amp;orderBeanReset=True" id="PermissionsLink" class="" target="_blank">Permissions</a>
    </div>

    </div>

                            </li>
                            <li class="toolbar-item item-with-dropdown item-share last">
                                <a class="drop-trigger" href="javascript:;" data-dropdown="ShareDrop" aria-haspopup="true">
                                    <i class="icon-menu_share"></i>
                                    <div class="toolbar-label">
                                        <div class="toolbar-text">Share</div>
                                        <i class="icon-general-arrow-filled-down arrow-icon"></i>
                                    </div>
                                </a>
                                <ul id="ShareDrop" class="addthis_toolbox addthis_default_style addthis_20x20_style f-dropdown" data-dropdown-content aria-label="submenu">
                                    <div class="arrow-up"></div>
                                    <li><a class="addthis_button_email"><span>Email</span></a></li>
                                    <li><a class="addthis_button_twitter"><span>Twitter</span></a></li>
                                    <li><a class="addthis_button_facebook"><span>Facebook</span></a></li>
                                </ul>
                            </li>
                        </ul>
                    
                        <div class="toolbar-search">
                                <div class="widget widget-SitePageHeader widget-instance-OUP_ArticleToolbarSearchBox">
        


<div class="dropdown-panel-wrap">

    
    <div class="dropdown-panel mobile-search-dropdown">
        <div class="mobile-search-inner-wrap">
            <div class="navbar-search">
                <div id="MobileMicrositeSearch" class="mobile-microsite-search">
                    <label for="mobile-navbar-search-filter" class="screenreader-text">Navbar Search Filter</label>
                    <select class="navbar-search-filter" id="mobile-navbar-search-filter">
                                                    <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue">This issue</option><option class="header-search-bar-filters-item" value="">All  DNA Research</option>
                            </optgroup>
                            <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals">All  Journals</option>                            </optgroup>
                    </select>





                    <label for="MobileMicrositeSearchTerm" class="screenreader-text">Mobile Microsite Search Term</label>
                    <input class="mobile-search-input mobile-microsite-search-term" type="text" maxlength="255" placeholder="Search" id="MobileMicrositeSearchTerm">

                    <span id="MobileMicrositeSearchIcon" class="mobile-microsite-search-icon mobile-search-submit icon-menu_search"></span>

                </div><!-- /#MobileMicrositeSearch -->
            </div><!-- /.navbar-search -->
        </div><!-- /.mobile-search-inner-wrap -->
    </div><!-- /.dropdown-panel .dropdown-panel-search -->
    
    <div class="dropdown-panel mobile-account-dropdown">
        <ul class="site-menu site-menu-lvl-0">
            <li class="site-menu-item site-menu-lvl-0">
                <a href="" class="nav-link mobile-account-signin dummy-link" id="header-account-info-user-fullname">
                    
Sign In                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                </a>
                <ul class="site-menu site-menu-lvl-1 individual-menu">
                    <li class="dropdown-signin-form">
                        <div class="spinner"></div>
                    </li>
                </ul>
            </li>
                <li class="site-menu-item site-menu-lvl-0">
                        <a class="nav-link mobile-account-institution dummy-link">
                            NJ Inst of Technology
                        </a>

                    <i class="mobile-nav-arrow icon-general_arrow-down"></i>
                    <ul class="site-menu site-menu-lvl-1">
                            <li class="dropdown-institution-item">

NJ Inst of Technology
                            </li>

                        <li class="dropdown-institution-signout">
                        </li>
                    </ul>
                </li>
                            <li class="site-menu-item">
                    <a href="/my-account/register?siteId=5275&amp;returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f195%2f420778" class="register nav-link">Register</a>
                </li>
        </ul>
    </div><!-- /.dropdown-panel .dropdown-panel-account -->
    
    <div class="dropdown-panel mobile-nav-dropdown">

    </div><!-- /.dropdown-panel .dropdown-panel-nav -->

</div><!-- /.dropdown-panel-wrap -->



<div class="navbar">
    <div class="center-inner-row">
        <nav class="navbar-menu">

        </nav>
            <div class="navbar-search-container">
                <div class="navbar-search-close js_close-navsearch">Close</div>
                <div class="navbar-search">
                    <div id="MicrositeSearch" class="microsite-search">
                        <label for="navbar-search-filter" class="screenreader-text">search filter</label>
                        <select class="navbar-search-filter" id="navbar-search-filter">
                                                            <optgroup label="Search within this journal">
<option class="header-search-bar-filters-item" value="Issue" >This issue</option><option class="header-search-bar-filters-item" value="" selected="selected">All  DNA Research</option>
                                </optgroup>
                                <optgroup label="Search beyond this journal">
<option class="header-search-bar-filters-item" value="Journals" >All  Journals</option>                                </optgroup>
                        </select>

                        <label for="MicrositeSearchTerm" class="screenreader-text">search input</label>
                        <input class="navbar-search-input microsite-search-term" type="text" maxlength="255" id="MicrositeSearchTerm" title="search input">

                        <span id="MicrositeSearchIcon" class="microsite-search-icon navbar-search-submit icon-menu_search"></span>

                    </div><!-- /#MicrositeSearch -->
                </div><!-- /.navbar-search -->
                <div class="navbar-search-advanced"><a href="/dnaresearch/advanced-search" class="advanced-search js-advanced-search">Advanced Search</a></div>
            </div><!-- /.navbar-search-container -->
            <div class="navbar-search-collapsed"><i class="icon-menu_search js_expand-navsearch"></i></div>
    </div><!-- /.center-inner-row -->
</div><!-- /.navbar -->

<input id="routename" name="RouteName" type="hidden" value="dnaresearch" />
    </div>

                        </div>
                    </div><!-- /.toolbar-inner-wrap -->                    
                </div>
                    <div id="ContentTab" class="content active">
                            <div class="widget widget-ArticleFulltext widget-instance-OUP_Article_FullText_Widget">
        <div class="module-widget">
    <div class="widget-items" data-widgetname="ArticleFulltext">





                            <h2 scrollto-destination=6234549 id="6234549" class="abstract-title" >Abstract</h2>
<section class="abstract"><p>Semi-mangroves form a group of transitional species between glycophytes and halophytes, and hold unique potential for learning molecular mechanisms underlying plant salt tolerance. <em>Millettia pinnata</em> is a semi-mangrove plant that can survive a wide range of saline conditions in the absence of specialized morphological and physiological traits. By employing the Illumina sequencing platform, we generated ∼192 million short reads from four cDNA libraries of <em>M. pinnata</em> and processed them into 108 598 unisequences with a high depth of coverage. The mean length and total length of these unisequences were 606 bp and 65.8 Mb, respectively. A total of 54 596 (50.3%) unisequences were assigned Nr annotations. Functional classification revealed the involvement of unisequences in various biological processes related to metabolism and environmental adaptation. We identified 23 815 candidate salt-responsive genes with significantly differential expression under seawater and freshwater treatments. Based on the reverse transcription–polymerase chain reaction (RT–PCR) and real-time PCR analyses, we verified the changes in expression levels for a number of candidate genes. The functional enrichment analyses for the candidate genes showed tissue-specific patterns of transcriptome remodelling upon salt stress in the roots and the leaves. The transcriptome of <em>M. pinnata</em> will provide valuable gene resources for future application in crop improvement. In addition, this study sets a good example for large-scale identification of salt-responsive genes in non-model organisms using the sequencing-based approach.</p></section>                        <div class="article-metadata-panel clearfix"></div>
<div class="kwd-group"><a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;transcriptome characterization&quot;">transcriptome characterization</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;Illumina sequencing&quot;">Illumina sequencing</a>, <a class="kwd-part kwd-main" href="javascript:;" data-keyword="&quot;salt-responsive genes&quot;">salt-responsive genes</a></div>                            <h2 scrollto-destination=6234551 id="6234551" class="section-title" data-legacy-id=s1>1. Introduction</h2>
<p>High salinity is one of the most severe environmental stresses causing crop yield reduction. The ability of plants to develop salt tolerance is a trait governed by multigenes and is a determining factor in their growth and productivity.<sup><a reveal-id="DSS004C1" data-open="DSS004C1" class="link link-ref link-reveal xref-bibr">1</a></sup> Among the intertwining physiological and molecular mechanisms, the regulation of gene expression plays a pivotal role in plant salt tolerance,<sup><a reveal-id="DSS004C2" data-open="DSS004C2" class="link link-ref link-reveal xref-bibr">2</a>,<a reveal-id="DSS004C3" data-open="DSS004C3" class="link link-ref link-reveal xref-bibr">3</a></sup> and it has therefore stimulated increasing efforts on global expression profiling for various plant species over the past decade.</p><p>The early insights into transcriptome responses were primarily obtained from the glycophytic model plants and major crops such as Arabidopsis,<sup><a reveal-id="DSS004C4" data-open="DSS004C4" class="link link-ref link-reveal xref-bibr">4</a>,<a reveal-id="DSS004C5" data-open="DSS004C5" class="link link-ref link-reveal xref-bibr">5</a></sup> rice,<sup><a reveal-id="DSS004C6" data-open="DSS004C6" class="link link-ref link-reveal xref-bibr">6</a>,<a reveal-id="DSS004C7" data-open="DSS004C7" class="link link-ref link-reveal xref-bibr">7</a></sup> and maize,<sup><a reveal-id="DSS004C8" data-open="DSS004C8" class="link link-ref link-reveal xref-bibr">8</a>,<a reveal-id="DSS004C9" data-open="DSS004C9" class="link link-ref link-reveal xref-bibr">9</a></sup> in which hundreds of potential salt-responsive genes were identified and analysed to determine their expression patterns under salt treatments. The studies on glycophytes also revealed the involvement of several signal transduction pathways in salt stress response, including the salt overly sensitive (SOS) pathway, the ABA signalling pathway, the ethylene signalling pathway, and the Ca<sup>2+</sup>-dependent signalling pathway.<sup><a reveal-id="DSS004C10  DSS004C13" data-open="DSS004C10  DSS004C13" class="link link-ref link-reveal xref-bibr">10–13</a></sup> These pathways constitute a dynamic network and work collectively to combat ionic disequilibrium, osmotic imbalance, and other deleterious effects of salt stress. However, most glycophytes can only endure relatively low concentrations of salt. For example, Arabidopsis cannot survive NaCl concentrations higher than 200 mM.<sup><a reveal-id="DSS004C14" data-open="DSS004C14" class="link link-ref link-reveal xref-bibr">14</a></sup> Therefore, the transcriptome analyses of these glycophytes would miss the information about some key genes or pathways that are related to salt tolerance.</p><p>To broaden the knowledge on the regulatory network for plant salt tolerance, the focus of recent transcriptomic research has been gradually shifting from the salt-sensitive glycophytes to the salt-tolerant halophytes. Salt cress (<em>Thellungiella halophila</em>) is the most commonly surveyed halophyte because its high nucleotide sequence identity with Arabidopsis cDNA facilitates the use of molecular and genetic tools that are available for Arabidopsis. Comparisons of the expression profiles between these two species demonstrated that they possessed both shared and divergent responses to salt stress, and the constitutive overexpression of many genes that were stress-inducible in Arabidopsis might contribute to the stress tolerance of salt cress.<sup><a reveal-id="DSS004C15" data-open="DSS004C15" class="link link-ref link-reveal xref-bibr">15</a>,<a reveal-id="DSS004C16" data-open="DSS004C16" class="link link-ref link-reveal xref-bibr">16</a></sup> These findings are partly in accordance with the hypothesis that salt tolerance mechanisms are largely conserved in halophytes and glycophytes, and that the large variations in their tolerance arise from subtle differences in the regulation of the same basic set of genes.<sup><a reveal-id="DSS004C17" data-open="DSS004C17" class="link link-ref link-reveal xref-bibr">17</a></sup> In other words, the formation of salt tolerance is a cumulative process with stepwise changes in gene regulation. Therefore, the salt-induced response of the transitional species between glycophytes and halophytes may provide new insights into the acquisition of salt tolerance. Nevertheless, the transcriptomic profiles of such transitional species are still not documented.</p><p><em>Millettia pinnata</em> is a semi-mangrove (also called ‘mangrove associate’) that can grow in either freshwater or seawater habitats. A recent study regarding the differentiation between true mangroves and semi-mangroves based on their leaf traits and salt contents proposed <em>M. pinnata</em> to be a glycophyte with moderate salt tolerance.<sup><a reveal-id="DSS004C18" data-open="DSS004C18" class="link link-ref link-reveal xref-bibr">18</a></sup> Unlike other mangrove species, <em>M. pinnata</em> does not exclude salt via ultra-filtration system at its root surface, excrete salt through glands on its leaf surface, or develop succulence leaf. However, this plant can still endure salinity up to 3%.<sup><a reveal-id="DSS004C19" data-open="DSS004C19" class="link link-ref link-reveal xref-bibr">19</a>,<a reveal-id="DSS004C20" data-open="DSS004C20" class="link link-ref link-reveal xref-bibr">20</a></sup> In the absence of specialized morphological and physiological traits, the adaptation of <em>M. pinnata</em> may be more exclusively attributed to the alternation in gene regulation, which makes <em>M. pinnata</em> extremely attractive for investigating on its transcriptomic adjustments under salt stress.</p><p>As an outbreeding diploid (2n = 2x = 22) legume, <em>M. pinnata</em> has high levels of seed oil and has received much attention on its applications in biodiesel.<sup><a reveal-id="DSS004C21  DSS004C23" data-open="DSS004C21  DSS004C23" class="link link-ref link-reveal xref-bibr">21–23</a></sup> In addition, its biomedical properties have also been investigated because it was traditionally known as a medicinal plant.<sup><a reveal-id="DSS004C24" data-open="DSS004C24" class="link link-ref link-reveal xref-bibr">24</a>,<a reveal-id="DSS004C25" data-open="DSS004C25" class="link link-ref link-reveal xref-bibr">25</a></sup> However, the genetic background of this non-model species remains largely unknown. To date, only a few genes, such as the phytochrome A (<em>PHYA</em>) gene, the maturase (<em>matR</em>) gene, and the ribosomal RNA gene, have been sequenced from <em>M. pinnata</em>, and none of these genes have been shown to be directly responsible for its salt tolerance.<sup><a reveal-id="DSS004C26  DSS004C28" data-open="DSS004C26  DSS004C28" class="link link-ref link-reveal xref-bibr">26–28</a></sup> This situation might now be changed with the introduction of next-generation massively parallel sequencing technologies developed by Roche/454 Life Sciences and Illumina.<sup><a reveal-id="DSS004C29" data-open="DSS004C29" class="link link-ref link-reveal xref-bibr">29</a>,<a reveal-id="DSS004C30" data-open="DSS004C30" class="link link-ref link-reveal xref-bibr">30</a></sup> These sequencing-based approaches do not rely on the prior knowledge of genomic sequences, hence they are particularly useful in quantifying transcriptomes for non-model organisms.<sup><a reveal-id="DSS004C31" data-open="DSS004C31" class="link link-ref link-reveal xref-bibr">31</a></sup></p><p>In this study, we initiated a comprehensive transcriptome analysis of <em>M. pinnata</em> based on the Illumina sequencing platform. Our goals were to (i) set up a data set with abundant transcript sequences from the root and leaf tissues of <em>M. pinnata</em>, (ii) screen out genes with differential expression under seawater and freshwater treatments as candidate salt-responsive genes, and (iii) generalize the common and tissue-specific patterns of transcriptomic responses to salt stress in <em>M. pinnata</em>. The characterization of the transcriptome changes in this transitional species may provide useful complements to the understandings of molecular mechanisms underlying plant salt tolerance. The candidate genes identified in this study can be applied to molecular breeding for plants with enhanced salt tolerance.</p>                            <h2 scrollto-destination=6234558 id="6234558" class="section-title" data-legacy-id=s2>2. Materials and methods</h2>
                            <h3 scrollto-destination=6234559 id="6234559" class="section-title" data-legacy-id=s2a>2.1. Plant growth and salt treatments</h3>
<p>The seeds of <em>M. pinnata</em> were obtained from the Dongzhaigang National Nature Reserve in Hainan, China. The field-collected seeds were surface-sterilized with a 75% ethanol solution, rinsed with sterilized distilled water, and sown in sterilized cover soil for germination. The seedlings were grown in plastic pots under a 12/12 h photoperiod at 25°C (day) and 18°C (night). The volume of each plant pot is ∼800 ml. For the salt treatment, a group of 3-week-old plants were watered by seawater with a salinity of 3% (w/v, ∼500 mM NaCl). The treatments were initiated at 8:00 a.m., at which time each individual plant was watered with 300 ml of seawater to field capacity. Meanwhile, a group of control plants were watered with an equal amount of freshwater (distilled water).</p>                            <h3 scrollto-destination=6234561 id="6234561" class="section-title" data-legacy-id=s2b>2.2. Sample collection and RNA preparation</h3>
<p>The roots and leaves were sampled at 2, 4, and 8 h after the application of seawater or freshwater. At each time point, the primary root with some lateral roots and three leaves at the shoot apex were simultaneously collected from each individual plant, and were separately frozen in liquid nitrogen and stored at −80°C prior to RNA extraction. The total RNA was isolated using TRIzol reagent (Invitrogen) following the manufacturer's instructions. The quality of RNAs was determined with a 2100 Bioanalyzer RNA Nanochip (Agilent Technologies). For each sample, at least 20 µg of total RNAs was sent to the Beijing Genomics Institute for Illumina sequencing (commercial service). For the reverse transcription–polymerase chain reaction (RT–PCR) and real-time PCR experiments, the total RNAs were prepared following the aforementioned procedures.</p>                            <h3 scrollto-destination=6234563 id="6234563" class="section-title" data-legacy-id=s2c>2.3. Sequencing and assembly</h3>
<p>For a given tissue, equal amounts of total RNAs from the plants treated for 2, 4, and 8 h were pooled in a combined sample. Totally, four combined samples were collected and denoted as <em>MpRs</em>, <em>MpRf</em>, <em>MpLs</em>, and <em>MpLf</em> according to the tissues (<em>R</em>oot or <em>L</em>eaf) and the treatments (<em>s</em>eawater or <em>f</em>reshwater) of their sampling sources. For each combined sample, mRNAs were purified using oligo(dT)-attached beads and fragmented into small pieces (100–400 bp). The cleaved RNA fragments were then primed with random hexamers and submitted to the synthesis of the first-strand and second-strand cDNAs. The synthesized cDNAs were ligated with paired-end adaptors. Then, the cDNA fragments with 200 bp (±20 bp) size were selected by agarose gel electrophoresis and enriched by PCR amplification. Finally, four cDNA libraries were constructed for sequencing on an Illumina GA IIx. All sequence data have been deposited in the Short Read Archive (SRA) at the NCBI database under the project accession number SRA046342.1.</p><p>After sequencing, the raw sequence data were first purified by trimming adapter sequences and removing low-quality sequences. The resulting clean reads were assembled using the SOAPdenovo program (version 1.04; <a class="link link-uri" href="http://soap.genomics.org.cn/soapdenovo.html">http://soap.genomics.org.cn/soapdenovo.html</a>) with K-mer size set to 29 bp.<sup><a reveal-id="DSS004C32" data-open="DSS004C32" class="link link-ref link-reveal xref-bibr">32</a></sup> The clean reads were split into smaller pieces, the K-mers, and were conjoined into contigs using the <em>de Bruijn</em> graph. Next, the contigs from the same transcript were identified by paired-end reads and were connected into scaffolds. Then the gap fillings were carried out using the pair-end information to retrieve read pairs with one read well aligned on the contigs and another read located in the gap region. The resulting scaffolds with least Ns were defined as unigenes. The unigenes assembled by short reads from four samples were further clustered into all-unigenes in a comprehensive transcriptome using TGI Clustering Tool (TGICL) with the parameters of 50 bp overlap and a minimum of 90% identity.<sup><a reveal-id="DSS004C33" data-open="DSS004C33" class="link link-ref link-reveal xref-bibr">33</a></sup></p>                            <h3 scrollto-destination=6234566 id="6234566" class="section-title" data-legacy-id=s2d>2.4. Data analysis</h3>
<p>The sequence orientations of the all-unigenes were determined by BLASTx against the NCBI non-redundant (Nr) protein database, the Swiss-Prot protein database, the Kyoto Encyclopedia of Genes and Genomes (KEGG) pathway database, and the Cluster of Orthologous Groups (COG) database. The incongruent results from different databases were settled under a priority order of Nr, Swiss-Prot, KEGG, and COG. For the rest all-unigenes that were unaligned to the above databases, ESTScan was used to predict their sequence orientations.<sup><a reveal-id="DSS004C34" data-open="DSS004C34" class="link link-ref link-reveal xref-bibr">34</a></sup></p><p>For assignments of gene descriptions, the all-unigenes were searched against the Nr database using BLASTx with an <em>E</em>-value cut-off of 1.0e–5. The BLAST results were parsed by a Perl script (available on request). The results of only the best hit were extracted. In practice, the <em>E</em>-values for more than 85% of the annotated sequences were ≤1.0e–10. Based on their Nr annotations, the all-unigenes were assigned GO annotations using Blast2GO (<a class="link link-uri" href="http://www.blast2go.rog/)">http://www.blast2go.rog/)</a>,<sup><a reveal-id="DSS004C35" data-open="DSS004C35" class="link link-ref link-reveal xref-bibr">35</a></sup> followed by functional classification using the WEGO software.<sup><a reveal-id="DSS004C36" data-open="DSS004C36" class="link link-ref link-reveal xref-bibr">36</a></sup> Besides, the putative metabolic pathways for the all-unigenes were assigned by performing BLASTx against the KEGG pathway database with the <em>E</em>-value cut-off of 1.0e–5.</p><p>The RPKM (reads per kilobase per million reads) values were applied to measure the gene expression levels.<sup><a reveal-id="DSS004C37" data-open="DSS004C37" class="link link-ref link-reveal xref-bibr">37</a></sup> For a given all-unigene, four RPKM values were generated by mapping the reads from the four subtranscriptomes to it using SOAP2 with a maximum of three mismatches.<sup><a reveal-id="DSS004C38" data-open="DSS004C38" class="link link-ref link-reveal xref-bibr">38</a></sup> Only the reads mapped to a single all-unigene were used to estimate the expression level. The differentially expressed genes (DEGs) between the seawater- and freshwater-treated samples were identified using an R package named ‘DEGseq’.<sup><a reveal-id="DSS004C39" data-open="DSS004C39" class="link link-ref link-reveal xref-bibr">39</a></sup> The GO enrichment analysis and KEGG pathway enrichment analysis for the DEGs were both performed by conducting hypergeometric tests with the whole <em>M. pinnata</em> transcriptome set as the background.</p>                            <h3 scrollto-destination=6234570 id="6234570" class="section-title" data-legacy-id=s2e>2.5. RT–PCR and real-time PCR analyses</h3>
<p>We conducted RT–PCR for 100 DEGs using the <em>M. pinnata actin</em> and <em>18S rRNA</em> genes as the controls. The first-strand cDNAs were synthesized from 1 µg of total RNAs using the PrimeScript<sup>®</sup> 1st Strand cDNA Synthesis Kit (TAKARA). The gene-specific primers were designed based on the sequencing results using the Primer Premier software (version 6.0). Then, the 20 µl of PCR samples containing 1 µl of first-strand cDNAs and 1 µl of primers were subjected to 35 cycles of 30 s denaturing at 94°C, 30 s annealing at the temperature that was altered for different DEGs, and 30 s extending at 72°C. The PCR products were electrophoresed on a 1% agarose gel.</p><p>The real-time PCR was performed on an ABI PRISM 7300 Sequence Detection System (Applied Biosystems) using 1 µl of first-strand cDNAs and SYBR<sup>®</sup> Premix Ex Taq™ (TAKARA). The thermal cycling conditions were 30 s at 95°C, followed by 40 cycles of 5 s at 95°C and 31 s at 60°C. All of the reactions were performed in triplicate. The relative expression levels for each gene were calculated using the 2<sup>−ΔΔCT</sup> method with normalization to the internal control gene. The LSD-<em>t</em> test of one-way analysis of variance was performed to determine the significant differences in expression levels between the treated samples at 1, 2, 4, and 8 h and those at 0 h using the SPSS software package (version 13.0).</p>                            <h2 scrollto-destination=6234573 id="6234573" class="section-title" data-legacy-id=s3>3. Results</h2>
                            <h3 scrollto-destination=6234574 id="6234574" class="section-title" data-legacy-id=s3a>3.1. Illumina sequencing and de novo assembly</h3>
<p>Our preliminary experiments showed that the young leaves of the <em>M. pinnata</em> plants treated with seawater began to wilt at 2 h, wilted to the greatest extent at 4 h, and almost completely recovered by 8 h after the treatments. Meanwhile, the control plants showed no symptoms during the course of the freshwater treatments. This result indicates that <em>M. pinnata</em> may suffer severe salt stress at 4 h, but it may overcome this stress by 8 h. We performed high-throughput sequencing of four samples, i.e. <em>MpRs</em> and <em>MpRf</em> from the roots, <em>MpLs</em> and <em>MpLf</em> from the leaves. For a broad survey of the salt-responsive genes, a combined cDNA library was constructed for each sample, which was represented by a transcript mixture with equal amounts of total RNA from 2, 4, and 8 h-treated plants. The Illumina sequencing was then performed separately for four cDNA libraries and generated four sub-transcriptomes with 75-bp raw reads. After filtration of low-quality and adapter sequences, nearly 192 million clean reads were remained for the four sub-transcriptomes. The percentage of Q20 bases for the clean reads in the four sub-transcriptomes were all ∼93% (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S1</a>). In total, these clean reads constitute ∼12 GB of sequence data.</p><p>The clean reads from the <em>MpRs</em>, <em>MpRf</em>, <em>MpLs</em>, and <em>MpLf</em> sub-transcriptomes were assembled into 248 011, 238 722, 194 806, and 194 020 contigs, respectively (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The average contig sizes of the four sub-transcriptomes were all above 200 bp. The following step used the paired-end reads to identify the contigs from the same transcript, and these contigs were merged into 130 027, 127 341, 101 737, and 104 087 scaffolds with an average size over 350 bp (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S2</a>). The scaffolds were further processed by a gap-filling step, which resulted in 78 062, 78 165, 63 081, and 64 436 unigenes for <em>MpRs</em>, <em>MpRf</em>, <em>MpLs</em>, and <em>MpLf</em>, respectively (Fig. <a reveal-id="DSS004F1" data-open="DSS004F1" class="link link-reveal link-table xref-fig">1</a>). The numbers of unigenes obtained from the root samples were larger than those from the leaf samples, whereas the mean lengths of the unigenes were longer in the latter samples than those in the former ones (Table <a reveal-id="DSS004TB1" data-open="DSS004TB1" class="link link-reveal link-table xref-fig">1</a>). </p>                        <a id="6234577" scrollto-destination="6234577"></a>
<div content-id="DSS004TB1" class="table-modal"><div class="table-wrap"><div id="DSS004TB1" class="table-wrap-title" data-id="DSS004TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics for the unigenes of <em>M. pinnata</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="5">Number of unigenes<hr /> </th></tr><tr><th>Length of unigenes (bp) </th><th><em>MpRs</em> </th><th><em>MpRf</em> </th><th><em>MpLs</em> </th><th><em>MpLf</em> </th><th>All </th></tr></thead><tbody><tr><td>200–500 </td><td>54 617 </td><td>53 061 </td><td>41 787 </td><td>41 700 </td><td>70 455 </td></tr><tr><td>500–1000 </td><td>16 231 </td><td>16 819 </td><td>13 930 </td><td>14 539 </td><td>21 028 </td></tr><tr><td>1000–1500 </td><td>4403 </td><td>4930 </td><td>4348 </td><td>4604 </td><td>8184 </td></tr><tr><td>1500–2000 </td><td>1677 </td><td>1914 </td><td>1800 </td><td>2036 </td><td>4356 </td></tr><tr><td>≥2000 </td><td>1134 </td><td>1441 </td><td>1216 </td><td>1557 </td><td>4575 </td></tr><tr><td>Total </td><td>78 062 </td><td>78 165 </td><td>63 081 </td><td>64 436 </td><td>108 598 </td></tr><tr><td>Mean length (bp) </td><td>497 </td><td>521 </td><td>537 </td><td>558 </td><td>606 </td></tr><tr><td>N50 (bp) </td><td>589 </td><td>634 </td><td>674 </td><td>708 </td><td>887 </td></tr><tr><td>Total length (bp) </td><td>38 794 023 </td><td>40 744 474 </td><td>33 850 579 </td><td>35 949 682 </td><td>65 814 388 </td></tr></tbody></table></div><div content-id="DSS004TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="5">Number of unigenes<hr /> </th></tr><tr><th>Length of unigenes (bp) </th><th><em>MpRs</em> </th><th><em>MpRf</em> </th><th><em>MpLs</em> </th><th><em>MpLf</em> </th><th>All </th></tr></thead><tbody><tr><td>200–500 </td><td>54 617 </td><td>53 061 </td><td>41 787 </td><td>41 700 </td><td>70 455 </td></tr><tr><td>500–1000 </td><td>16 231 </td><td>16 819 </td><td>13 930 </td><td>14 539 </td><td>21 028 </td></tr><tr><td>1000–1500 </td><td>4403 </td><td>4930 </td><td>4348 </td><td>4604 </td><td>8184 </td></tr><tr><td>1500–2000 </td><td>1677 </td><td>1914 </td><td>1800 </td><td>2036 </td><td>4356 </td></tr><tr><td>≥2000 </td><td>1134 </td><td>1441 </td><td>1216 </td><td>1557 </td><td>4575 </td></tr><tr><td>Total </td><td>78 062 </td><td>78 165 </td><td>63 081 </td><td>64 436 </td><td>108 598 </td></tr><tr><td>Mean length (bp) </td><td>497 </td><td>521 </td><td>537 </td><td>558 </td><td>606 </td></tr><tr><td>N50 (bp) </td><td>589 </td><td>634 </td><td>674 </td><td>708 </td><td>887 </td></tr><tr><td>Total length (bp) </td><td>38 794 023 </td><td>40 744 474 </td><td>33 850 579 </td><td>35 949 682 </td><td>65 814 388 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><em>MpRs</em>, <em>MpRf</em>, <em>MpLs</em>, and <em>MpLf</em> represent four sub-transcriptomes that were derived from two tissues (<em>R</em>oot or <em>L</em>eaf) under two treatments (<em>s</em>eawater or <em>f</em>reshwater). The last column presents the statistics for the all-unigenes clustered by the unigenes from four sub-transcriptomes.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6234577" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS004TB1" class="table-wrap-title" data-id="DSS004TB1"><div class="label -label">Table 1.</div><div class="caption"><p>Statistics for the unigenes of <em>M. pinnata</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th> </th><th colspan="5">Number of unigenes<hr /> </th></tr><tr><th>Length of unigenes (bp) </th><th><em>MpRs</em> </th><th><em>MpRf</em> </th><th><em>MpLs</em> </th><th><em>MpLf</em> </th><th>All </th></tr></thead><tbody><tr><td>200–500 </td><td>54 617 </td><td>53 061 </td><td>41 787 </td><td>41 700 </td><td>70 455 </td></tr><tr><td>500–1000 </td><td>16 231 </td><td>16 819 </td><td>13 930 </td><td>14 539 </td><td>21 028 </td></tr><tr><td>1000–1500 </td><td>4403 </td><td>4930 </td><td>4348 </td><td>4604 </td><td>8184 </td></tr><tr><td>1500–2000 </td><td>1677 </td><td>1914 </td><td>1800 </td><td>2036 </td><td>4356 </td></tr><tr><td>≥2000 </td><td>1134 </td><td>1441 </td><td>1216 </td><td>1557 </td><td>4575 </td></tr><tr><td>Total </td><td>78 062 </td><td>78 165 </td><td>63 081 </td><td>64 436 </td><td>108 598 </td></tr><tr><td>Mean length (bp) </td><td>497 </td><td>521 </td><td>537 </td><td>558 </td><td>606 </td></tr><tr><td>N50 (bp) </td><td>589 </td><td>634 </td><td>674 </td><td>708 </td><td>887 </td></tr><tr><td>Total length (bp) </td><td>38 794 023 </td><td>40 744 474 </td><td>33 850 579 </td><td>35 949 682 </td><td>65 814 388 </td></tr></tbody></table></div><div content-id="DSS004TB1" class="table-modal"><table><thead name="thead" align=""><tr><th> </th><th colspan="5">Number of unigenes<hr /> </th></tr><tr><th>Length of unigenes (bp) </th><th><em>MpRs</em> </th><th><em>MpRf</em> </th><th><em>MpLs</em> </th><th><em>MpLf</em> </th><th>All </th></tr></thead><tbody><tr><td>200–500 </td><td>54 617 </td><td>53 061 </td><td>41 787 </td><td>41 700 </td><td>70 455 </td></tr><tr><td>500–1000 </td><td>16 231 </td><td>16 819 </td><td>13 930 </td><td>14 539 </td><td>21 028 </td></tr><tr><td>1000–1500 </td><td>4403 </td><td>4930 </td><td>4348 </td><td>4604 </td><td>8184 </td></tr><tr><td>1500–2000 </td><td>1677 </td><td>1914 </td><td>1800 </td><td>2036 </td><td>4356 </td></tr><tr><td>≥2000 </td><td>1134 </td><td>1441 </td><td>1216 </td><td>1557 </td><td>4575 </td></tr><tr><td>Total </td><td>78 062 </td><td>78 165 </td><td>63 081 </td><td>64 436 </td><td>108 598 </td></tr><tr><td>Mean length (bp) </td><td>497 </td><td>521 </td><td>537 </td><td>558 </td><td>606 </td></tr><tr><td>N50 (bp) </td><td>589 </td><td>634 </td><td>674 </td><td>708 </td><td>887 </td></tr><tr><td>Total length (bp) </td><td>38 794 023 </td><td>40 744 474 </td><td>33 850 579 </td><td>35 949 682 </td><td>65 814 388 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p><em>MpRs</em>, <em>MpRf</em>, <em>MpLs</em>, and <em>MpLf</em> represent four sub-transcriptomes that were derived from two tissues (<em>R</em>oot or <em>L</em>eaf) under two treatments (<em>s</em>eawater or <em>f</em>reshwater). The last column presents the statistics for the all-unigenes clustered by the unigenes from four sub-transcriptomes.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6234577" target="_blank">View Large</a></div></div>                        <a id="6234578" scrollto-destination="6234578"></a>
<div id="DSS004F1" data-id="DSS004F1" class="fig fig-section"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/m_dss00401.jpeg?Expires=2147483647&Signature=OE-jRdIc3-f3YtMRqLfutPekMiy0t~WWR2leTsF5EYsnEouumUKDAf2sdJ5giALSDemkiK4iTf8XYC8kcHgOOpM7vogOiI~MKHXuhMkJZdk7XNn0vUreNjMq~5fQTfd8k-190aeVxxwsCie5b7sKj6stcZJGqAgvsazQk1IX-QzDVOvgfIFsEAMlXT3Z-dRn4hwmNIpjwAtkKrxk6A3hIV98RR0fzB1Y1uZ8wtBhUpX0SRpNP30ppc-hSQ9j42zFXkk95-btLb6OsHtDCfI3j9a7nSnv5hD90bOV7KCXS1Xu41xmArCCam~Yd6TZQodxxhKK6QyKHP1IgaG1NDtiHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flowcharts of the transcriptome analyses for M. pinnata. The steps include sequence assembly and clustering, Nr annotation, GO annotation, KEGG analysis, and identification of DEGs as candidate salt-responsive genes." path-from-xml="dss00401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6234578/dss00401.jpeg" path-from-xml="dss00401.jpeg" target="_blank">View large</a><a section="6234578" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/dss00401.jpeg?Expires=2147483647&Signature=ghtHdp-qnl63NGIl40bs~nL8AvJnk6ylAknLIOCl5LPa5YsrCC~N3c0kbP0CNTPsU45CLC1Dx6aQPZjQdflMXStAtyW0E0k~6TxVgSfPvDVVUfaadRHalDTqK5Ziup4ulVWXb7hTVu5E5t3~AWasO7g6vEFqPxsokUOxPAmWT-to86Bem-USdCqB1eQlWed3QT6ReBRnqee~jJsRUa6w8Qu68HJ5fj6AVAngSPZUfDoLN6NEvHFEt6vue6reuXqUsUpFiQJTLLYag0CPREa0K3vAwQrxkZ3yEaSdNvH38oajKKSaz-AqHN7y5Hx0Sbq9Rg1HbcWbtsqQgYQXw1a6Ag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6234578&ar=420778&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flowcharts of the transcriptome analyses for <em>M. pinnata</em>. The steps include sequence assembly and clustering, Nr annotation, GO annotation, KEGG analysis, and identification of DEGs as candidate salt-responsive genes.</p></div></div><div content-id="DSS004F1" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 1.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/m_dss00401.jpeg?Expires=2147483647&Signature=OE-jRdIc3-f3YtMRqLfutPekMiy0t~WWR2leTsF5EYsnEouumUKDAf2sdJ5giALSDemkiK4iTf8XYC8kcHgOOpM7vogOiI~MKHXuhMkJZdk7XNn0vUreNjMq~5fQTfd8k-190aeVxxwsCie5b7sKj6stcZJGqAgvsazQk1IX-QzDVOvgfIFsEAMlXT3Z-dRn4hwmNIpjwAtkKrxk6A3hIV98RR0fzB1Y1uZ8wtBhUpX0SRpNP30ppc-hSQ9j42zFXkk95-btLb6OsHtDCfI3j9a7nSnv5hD90bOV7KCXS1Xu41xmArCCam~Yd6TZQodxxhKK6QyKHP1IgaG1NDtiHg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Flowcharts of the transcriptome analyses for M. pinnata. The steps include sequence assembly and clustering, Nr annotation, GO annotation, KEGG analysis, and identification of DEGs as candidate salt-responsive genes." path-from-xml="dss00401.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6234578/dss00401.jpeg" path-from-xml="dss00401.jpeg" target="_blank">View large</a><a section="6234578" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/dss00401.jpeg?Expires=2147483647&Signature=ghtHdp-qnl63NGIl40bs~nL8AvJnk6ylAknLIOCl5LPa5YsrCC~N3c0kbP0CNTPsU45CLC1Dx6aQPZjQdflMXStAtyW0E0k~6TxVgSfPvDVVUfaadRHalDTqK5Ziup4ulVWXb7hTVu5E5t3~AWasO7g6vEFqPxsokUOxPAmWT-to86Bem-USdCqB1eQlWed3QT6ReBRnqee~jJsRUa6w8Qu68HJ5fj6AVAngSPZUfDoLN6NEvHFEt6vue6reuXqUsUpFiQJTLLYag0CPREa0K3vAwQrxkZ3yEaSdNvH38oajKKSaz-AqHN7y5Hx0Sbq9Rg1HbcWbtsqQgYQXw1a6Ag__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6234578&ar=420778&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00401.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Flowcharts of the transcriptome analyses for <em>M. pinnata</em>. The steps include sequence assembly and clustering, Nr annotation, GO annotation, KEGG analysis, and identification of DEGs as candidate salt-responsive genes.</p></div></div><p>Under the clustering criteria with a minimum of 50-bp overlap and 90% identity, the unigenes from all four sub-transcriptomes were further clustered into 108 598 all-unigenes to form a comprehensive transcriptome of <em>M. pinnata</em> (Fig. <a reveal-id="DSS004F1" data-open="DSS004F1" class="link link-reveal link-table xref-fig">1</a>). The gene number showed a substantial increase from the sub-transcriptome to the comprehensive transcriptome, and the increase was more prominent for the genes with larger sizes (Table <a reveal-id="DSS004TB1" data-open="DSS004TB1" class="link link-reveal link-table xref-fig">1</a>). The length of the all-unigenes ranged from 200 to 10 843 bp, with the mean length and the N50 value rises to 606 and 887 bp, respectively. As a result of the increase in both gene number and gene length, the total length of the all-unigenes extended to 65.8 Mb after clustering (Table <a reveal-id="DSS004TB1" data-open="DSS004TB1" class="link link-reveal link-table xref-fig">1</a>). A total of 49 836 (45.9%) all-unigenes were clustered by at least two unigenes, with a maximum of 31 unigenes per all-unigenes, whereas the rest 58 762 (54.1%) were corresponding to single unigene from any of the four sub-transcriptomes. The average sequencing depth calculated by realigning all usable sequencing reads to the all-unigenes were ∼30-folds for each sample. The quality of these all-unigenes was evaluated by two analyses: (i) the random distribution of reads in the all-unigenes showed that the all-unigenes were evenly covered by the reads from each sub-transcriptome with relatively fewer reads in the 3′ ends of them (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S1</a>), and (ii) the gap distribution of the all-unigenes indicated that the majority of them displayed a ratio of gap length to gene length &lt;5% (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S2</a>).</p>                            <h3 scrollto-destination=6234580 id="6234580" class="section-title" data-legacy-id=s3b>3.2. Gene annotation and functional classification</h3>
<p>First, the all-unigenes of <em>M. pinnata</em> were assigned putative gene descriptions based on the BLAST search against the NCBI non-redundant (Nr) protein database. Out of the 108 598 all-unigenes, 54 596 (50.3%) showing significant similarity with proteins in the Nr database were assigned Nr annotations (Fig. <a reveal-id="DSS004F1" data-open="DSS004F1" class="link link-reveal link-table xref-fig">1</a>). Overall, these all-unigenes matched 24 281 unique protein accessions, which indicated that our sequencing project had generated a substantial number of genes for <em>M. pinnata</em>. Only 33.2% of the all-unigenes with a length &lt;500 bp had BLAST hits in the Nr database. The proportion of all-unigenes with BLAST hits increased markedly for those with larger sizes (Fig. <a reveal-id="DSS004F2" data-open="DSS004F2" class="link link-reveal link-table xref-fig">2</a>). It seemed that longer all-unigenes were more likely to have Nr annotations. The <em>E</em>-value distribution of the top hits in the Nr database also revealed that a larger proportion of all-unigenes longer than 500 bp had strong homology (<em>E</em>-value ≤1.0e–50) (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S3</a>). </p>                        <a id="6234582" scrollto-destination="6234582"></a>
<div id="DSS004F2" data-id="DSS004F2" class="fig fig-section"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/m_dss00402.jpeg?Expires=2147483647&Signature=o~AztLz2pV24OnqNEKQka~7InmGCZumYL7RU713KzUv7hnM-ACQaE1iDPktiocr8QrFo5WZotF8a5qD8fyfRCkVhCdjDihEOYr6p55nQQc05I-BHjqPGJET4yhrmJNltsXFLGO3h~hRm8CjdQBSwI-HOm8H35gwjYTF0bILOL58nr6FFfloeTc0vo0OeRxSnYGjJ6g2RHayW1EENpB0xku3-VvF-p8CuT1EkowFD9f6cXQEpd31hEzlGebTfqsnkPdXdoFJPYxe2m1qgS~6jV0lmx5zQUi0q4M~h-LC79JNdi974yeJeGp6uwh~5aRDPhxAPxFpLKq-RrNtczXHyjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Length distribution of the all-unigenes with Nr annotations. In this study, only 33% of the all-unigenes shorter than 500 bp had BLAST hits in the Nr database, whereas more than 80% of the all-unigenes over 500 bp did." path-from-xml="dss00402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6234582/dss00402.jpeg" path-from-xml="dss00402.jpeg" target="_blank">View large</a><a section="6234582" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/dss00402.jpeg?Expires=2147483647&Signature=Od5V2AnLBPTcD8drg6AlRXUgiwQr5bCkNuJZUPnFVdZA8mnTyNsFFL1O7ymKO25o58-PibsSchNv2rB-o-XGoSzN-uIsSw5d1hCA6Ofo0YwR1c2MZz9O4oxEK1~bmqJF7Uyni3iAAvWDK9h-68peVfh1WcqS34njogEskShQlw7InPooC~QfDj8l~efV1W9oEB44-YYZRc5fXseNN2YEgxTB~TPriqiBTjMOMPbYcGlea4THny73dqG4EpsDE71PhZiwRMN6lGCjQFae61-0J9EWA9XGEMWHjvg02pLPRuA9kzvXv0W3WaPBhXfV9AZwA5yZBsm8DrguTQWvkEu17g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6234582&ar=420778&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Length distribution of the all-unigenes with Nr annotations. In this study, only 33% of the all-unigenes shorter than 500 bp had BLAST hits in the Nr database, whereas more than 80% of the all-unigenes over 500 bp did.</p></div></div><div content-id="DSS004F2" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 2.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/m_dss00402.jpeg?Expires=2147483647&Signature=o~AztLz2pV24OnqNEKQka~7InmGCZumYL7RU713KzUv7hnM-ACQaE1iDPktiocr8QrFo5WZotF8a5qD8fyfRCkVhCdjDihEOYr6p55nQQc05I-BHjqPGJET4yhrmJNltsXFLGO3h~hRm8CjdQBSwI-HOm8H35gwjYTF0bILOL58nr6FFfloeTc0vo0OeRxSnYGjJ6g2RHayW1EENpB0xku3-VvF-p8CuT1EkowFD9f6cXQEpd31hEzlGebTfqsnkPdXdoFJPYxe2m1qgS~6jV0lmx5zQUi0q4M~h-LC79JNdi974yeJeGp6uwh~5aRDPhxAPxFpLKq-RrNtczXHyjQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Length distribution of the all-unigenes with Nr annotations. In this study, only 33% of the all-unigenes shorter than 500 bp had BLAST hits in the Nr database, whereas more than 80% of the all-unigenes over 500 bp did." path-from-xml="dss00402.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6234582/dss00402.jpeg" path-from-xml="dss00402.jpeg" target="_blank">View large</a><a section="6234582" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/dss00402.jpeg?Expires=2147483647&Signature=Od5V2AnLBPTcD8drg6AlRXUgiwQr5bCkNuJZUPnFVdZA8mnTyNsFFL1O7ymKO25o58-PibsSchNv2rB-o-XGoSzN-uIsSw5d1hCA6Ofo0YwR1c2MZz9O4oxEK1~bmqJF7Uyni3iAAvWDK9h-68peVfh1WcqS34njogEskShQlw7InPooC~QfDj8l~efV1W9oEB44-YYZRc5fXseNN2YEgxTB~TPriqiBTjMOMPbYcGlea4THny73dqG4EpsDE71PhZiwRMN6lGCjQFae61-0J9EWA9XGEMWHjvg02pLPRuA9kzvXv0W3WaPBhXfV9AZwA5yZBsm8DrguTQWvkEu17g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6234582&ar=420778&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00402.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Length distribution of the all-unigenes with Nr annotations. In this study, only 33% of the all-unigenes shorter than 500 bp had BLAST hits in the Nr database, whereas more than 80% of the all-unigenes over 500 bp did.</p></div></div><p>The majority of these annotated all-unigenes (54 266 out of 54 596) displayed the highest homology to genes from plants (Table <a reveal-id="DSS004TB2" data-open="DSS004TB2" class="link link-reveal link-table xref-fig">2</a>). A quarter of them retrieved an annotation from Arabidopsis, partly due to the fact that the Arabidopsis genome annotation is the most advanced and complete for any higher plant to date in the Nr database. Besides, there were also a quarter of annotated all-unigenes with matched accessions from its legume relatives (Table <a reveal-id="DSS004TB2" data-open="DSS004TB2" class="link link-reveal link-table xref-fig">2</a>). The remaining &lt;1% of annotated all-unigenes were annotated with sequences from the non-plant sources. Interestingly, we found 12 all-unigenes with an Nr annotation from <em>Phytophthora sojae</em>, which is a plant pathogen that causes soybean stem and root rot.<sup><a reveal-id="DSS004C40" data-open="DSS004C40" class="link link-ref link-reveal xref-bibr">40</a></sup> Accordingly, we verified that all of these 12 all-unigenes were clustered by unigenes from the root-derived sub-transcriptome. This observation demonstrated a good representation of the comprehensive transcriptome in which even trivial amounts of pathogen transcripts were detected. </p>                        <a id="6234586" scrollto-destination="6234586"></a>
<div content-id="DSS004TB2" class="table-modal"><div class="table-wrap"><div id="DSS004TB2" class="table-wrap-title" data-id="DSS004TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Annotation sources for the all-unigenes of <em>M. pinnata</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Annotation sources </th><th>Number of matched all-unigenes </th></tr></thead><tbody><tr><td colspan="2">Ten species with the most abundant matches </td></tr><tr><td> <em>Arabidopsis thaliana</em> </td><td>15 308 </td></tr><tr><td> <em>Oryza sativa</em> </td><td>5739 </td></tr><tr><td> <em>Glycine max</em> </td><td>5198 </td></tr><tr><td> <em>Arabidopsis lyrata</em> </td><td>4551 </td></tr><tr><td> <em>Medicago truncatula</em> </td><td>4414 </td></tr><tr><td> <em>Populus trichocarpa</em> </td><td>3463 </td></tr><tr><td> <em>Vitis vinifera</em> </td><td>2491 </td></tr><tr><td> <em>Zea mays</em> </td><td>1067 </td></tr><tr><td> <em>Lotus japonicus</em> </td><td>705 </td></tr><tr><td> <em>Ricinus communis</em> </td><td>685 </td></tr><tr><td colspan="2">Ten legume genus with the most abundant matches </td></tr><tr><td> <em>Glycine</em> </td><td>5220 </td></tr><tr><td> <em>Medicago</em> </td><td>4722 </td></tr><tr><td> <em>Lotus</em> </td><td>722 </td></tr><tr><td> <em>Pisum</em> </td><td>683 </td></tr><tr><td> <em>Phaseolus</em> </td><td>550 </td></tr><tr><td> <em>Vigna</em> </td><td>312 </td></tr><tr><td> <em>Cicer</em> </td><td>197 </td></tr><tr><td> <em>Trifolium</em> </td><td>157 </td></tr><tr><td> <em>Arachis</em> </td><td>152 </td></tr><tr><td> <em>Vicia</em> </td><td>108 </td></tr><tr><td>nr_plants </td><td>54 266 </td></tr><tr><td>nr_nonplants </td><td>330 </td></tr></tbody></table></div><div content-id="DSS004TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Annotation sources </th><th>Number of matched all-unigenes </th></tr></thead><tbody><tr><td colspan="2">Ten species with the most abundant matches </td></tr><tr><td> <em>Arabidopsis thaliana</em> </td><td>15 308 </td></tr><tr><td> <em>Oryza sativa</em> </td><td>5739 </td></tr><tr><td> <em>Glycine max</em> </td><td>5198 </td></tr><tr><td> <em>Arabidopsis lyrata</em> </td><td>4551 </td></tr><tr><td> <em>Medicago truncatula</em> </td><td>4414 </td></tr><tr><td> <em>Populus trichocarpa</em> </td><td>3463 </td></tr><tr><td> <em>Vitis vinifera</em> </td><td>2491 </td></tr><tr><td> <em>Zea mays</em> </td><td>1067 </td></tr><tr><td> <em>Lotus japonicus</em> </td><td>705 </td></tr><tr><td> <em>Ricinus communis</em> </td><td>685 </td></tr><tr><td colspan="2">Ten legume genus with the most abundant matches </td></tr><tr><td> <em>Glycine</em> </td><td>5220 </td></tr><tr><td> <em>Medicago</em> </td><td>4722 </td></tr><tr><td> <em>Lotus</em> </td><td>722 </td></tr><tr><td> <em>Pisum</em> </td><td>683 </td></tr><tr><td> <em>Phaseolus</em> </td><td>550 </td></tr><tr><td> <em>Vigna</em> </td><td>312 </td></tr><tr><td> <em>Cicer</em> </td><td>197 </td></tr><tr><td> <em>Trifolium</em> </td><td>157 </td></tr><tr><td> <em>Arachis</em> </td><td>152 </td></tr><tr><td> <em>Vicia</em> </td><td>108 </td></tr><tr><td>nr_plants </td><td>54 266 </td></tr><tr><td>nr_nonplants </td><td>330 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>‘nr_plants’ and ‘nr_nonplants’ indicate the annotation sources from the NCBI non-redundant sequences for plants and non-plant species, respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6234586" target="_blank">View Large</a></div></div></div><div class="table-wrap"><div id="DSS004TB2" class="table-wrap-title" data-id="DSS004TB2"><div class="label -label">Table 2.</div><div class="caption"><p>Annotation sources for the all-unigenes of <em>M. pinnata</em></p></div> </div><div class="table-overflow"><table><thead name="thead" align=""><tr><th>Annotation sources </th><th>Number of matched all-unigenes </th></tr></thead><tbody><tr><td colspan="2">Ten species with the most abundant matches </td></tr><tr><td> <em>Arabidopsis thaliana</em> </td><td>15 308 </td></tr><tr><td> <em>Oryza sativa</em> </td><td>5739 </td></tr><tr><td> <em>Glycine max</em> </td><td>5198 </td></tr><tr><td> <em>Arabidopsis lyrata</em> </td><td>4551 </td></tr><tr><td> <em>Medicago truncatula</em> </td><td>4414 </td></tr><tr><td> <em>Populus trichocarpa</em> </td><td>3463 </td></tr><tr><td> <em>Vitis vinifera</em> </td><td>2491 </td></tr><tr><td> <em>Zea mays</em> </td><td>1067 </td></tr><tr><td> <em>Lotus japonicus</em> </td><td>705 </td></tr><tr><td> <em>Ricinus communis</em> </td><td>685 </td></tr><tr><td colspan="2">Ten legume genus with the most abundant matches </td></tr><tr><td> <em>Glycine</em> </td><td>5220 </td></tr><tr><td> <em>Medicago</em> </td><td>4722 </td></tr><tr><td> <em>Lotus</em> </td><td>722 </td></tr><tr><td> <em>Pisum</em> </td><td>683 </td></tr><tr><td> <em>Phaseolus</em> </td><td>550 </td></tr><tr><td> <em>Vigna</em> </td><td>312 </td></tr><tr><td> <em>Cicer</em> </td><td>197 </td></tr><tr><td> <em>Trifolium</em> </td><td>157 </td></tr><tr><td> <em>Arachis</em> </td><td>152 </td></tr><tr><td> <em>Vicia</em> </td><td>108 </td></tr><tr><td>nr_plants </td><td>54 266 </td></tr><tr><td>nr_nonplants </td><td>330 </td></tr></tbody></table></div><div content-id="DSS004TB2" class="table-modal"><table><thead name="thead" align=""><tr><th>Annotation sources </th><th>Number of matched all-unigenes </th></tr></thead><tbody><tr><td colspan="2">Ten species with the most abundant matches </td></tr><tr><td> <em>Arabidopsis thaliana</em> </td><td>15 308 </td></tr><tr><td> <em>Oryza sativa</em> </td><td>5739 </td></tr><tr><td> <em>Glycine max</em> </td><td>5198 </td></tr><tr><td> <em>Arabidopsis lyrata</em> </td><td>4551 </td></tr><tr><td> <em>Medicago truncatula</em> </td><td>4414 </td></tr><tr><td> <em>Populus trichocarpa</em> </td><td>3463 </td></tr><tr><td> <em>Vitis vinifera</em> </td><td>2491 </td></tr><tr><td> <em>Zea mays</em> </td><td>1067 </td></tr><tr><td> <em>Lotus japonicus</em> </td><td>705 </td></tr><tr><td> <em>Ricinus communis</em> </td><td>685 </td></tr><tr><td colspan="2">Ten legume genus with the most abundant matches </td></tr><tr><td> <em>Glycine</em> </td><td>5220 </td></tr><tr><td> <em>Medicago</em> </td><td>4722 </td></tr><tr><td> <em>Lotus</em> </td><td>722 </td></tr><tr><td> <em>Pisum</em> </td><td>683 </td></tr><tr><td> <em>Phaseolus</em> </td><td>550 </td></tr><tr><td> <em>Vigna</em> </td><td>312 </td></tr><tr><td> <em>Cicer</em> </td><td>197 </td></tr><tr><td> <em>Trifolium</em> </td><td>157 </td></tr><tr><td> <em>Arachis</em> </td><td>152 </td></tr><tr><td> <em>Vicia</em> </td><td>108 </td></tr><tr><td>nr_plants </td><td>54 266 </td></tr><tr><td>nr_nonplants </td><td>330 </td></tr></tbody></table></div><div class="table-wrap-foot"><div content-id=""><div class="fn"><p>‘nr_plants’ and ‘nr_nonplants’ indicate the annotation sources from the NCBI non-redundant sequences for plants and non-plant species, respectively.</p></div></div></div><div class="graphic-wrap"><a class="fig-view-orig openInAnotherWindow btn js-view-large" href="/view-large/6234586" target="_blank">View Large</a></div></div><p>Following their Nr annotations, we mapped the all-unigenes into the records of the GO database and retrieved GO annotations for 11 974 of them (Fig. <a reveal-id="DSS004F1" data-open="DSS004F1" class="link link-reveal link-table xref-fig">1</a>). These all-unigenes were assigned GO terms from the three main categories, including 6407 all-unigenes with terms from ‘Biological process’, 7095 all-unigenes with terms from ‘Cellular component’, and 8299 all-unigenes with terms from ‘Molecular function’ (Fig. <a reveal-id="DSS004F3" data-open="DSS004F3" class="link link-reveal link-table xref-fig">3</a>). Among them, 3228 all-unigenes had an assignment in all three categories. The remaining all-unigenes failed to obtain a GO term, largely due to their uninformative (e.g. ‘unknown’, ‘putative’, or ‘hypothetical’ protein) descriptions. Within the ‘Biological process’ category, the two most abundantly represented lineages were ‘metabolic process’ and ‘cellular process’. There were also a large number of genes involved in ‘biological regulation’, ‘pigmentation’, ‘localization’, and ‘response to stimulus’. In the ‘Molecular function’ category, ‘binding’ and ‘catalytic activity’ were predominantly represented. The former was mainly represented by genes for ‘nucleotide binding’ and ‘protein binding’, while the latter was mainly represented by genes with ‘transferase activity’, ‘hydrolase activity’, and ‘kinase activity’. In the ‘Cellular component’ category, most of all-unigenes were located in ‘cell part’. In contrast, rare all-unigenes were sorted into ‘extracellular region part’ or ‘virion part’. </p>                        <a id="6234592" scrollto-destination="6234592"></a>
<div id="DSS004F3" data-id="DSS004F3" class="fig fig-section"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/m_dss00403.jpeg?Expires=2147483647&Signature=JOP1WZ-in55JFaxt2BBn5w5JE1Mc77zmW9PfsV5E5CAZ1RdE3-GivJB3ZtCZFBKFF3lZ7c8oBcezBDJS8eI~W79NGldkTEHmnNzQQzwzLamlUB67ldR1hxH0aLK2iVCduITqz3HhcpXjxSNVQHFTQmkYFIimw2TRbYF70DBdH8BNav1Sv6o6MKGuAhla3T7ebURknbchkkho~zu1Mp6vwg3tmzlNwh3TIhC39BrmE4mTsH4LjIk97QI2OS1qLWBGUiNZZILsNl5g3vrhJSUWiT4lfXAHD0TyBG7oBmRQOeVm3OjX1~DGOwn8mniJ6nQrpP5LhCKyqcYtbD3BgDW9UA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GO classification of M. pinnata transcriptome. The results are summarized in three main categories as follows: biological process, cellular component, and molecular function. In total, 11 974 all-unigenes have been assigned GO terms. In some cases, one all-unigene has multiple terms." path-from-xml="dss00403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6234592/dss00403.jpeg" path-from-xml="dss00403.jpeg" target="_blank">View large</a><a section="6234592" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/dss00403.jpeg?Expires=2147483647&Signature=PFcjPNP9Fqczbz6N9JMi3AG8vEseVfVamucWGar8UfT6n~qOZm9r4gJ7DLyu~qtw7zQvAXP5aRlQnzCSe9TZlav2dwne-Wo7lyLWXwcWc844vqC7tNFB05yQkTt0~2lf4c9h~f0b24c2g32bgskgWgfqeemNGpueTseSLjrhBc2t3pmgn6xN~N74~0TI063lpvM9QlG0UaDvfnJibW5oHQv6vMl6ZLfc8apIf-x-3vsKgoqSF8jJBGFuBU6HpCRI3mI0VPk-FPJHyM1D1RZRpEtHFRfHBkpW7p0UWHO91tWsVgcqZ11bBf6qLpiml9trIHKBFwB2obqq2u2lbG5Jsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6234592&ar=420778&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GO classification of <em>M. pinnata</em> transcriptome. The results are summarized in three main categories as follows: biological process, cellular component, and molecular function. In total, 11 974 all-unigenes have been assigned GO terms. In some cases, one all-unigene has multiple terms.</p></div></div><div content-id="DSS004F3" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 3.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/m_dss00403.jpeg?Expires=2147483647&Signature=JOP1WZ-in55JFaxt2BBn5w5JE1Mc77zmW9PfsV5E5CAZ1RdE3-GivJB3ZtCZFBKFF3lZ7c8oBcezBDJS8eI~W79NGldkTEHmnNzQQzwzLamlUB67ldR1hxH0aLK2iVCduITqz3HhcpXjxSNVQHFTQmkYFIimw2TRbYF70DBdH8BNav1Sv6o6MKGuAhla3T7ebURknbchkkho~zu1Mp6vwg3tmzlNwh3TIhC39BrmE4mTsH4LjIk97QI2OS1qLWBGUiNZZILsNl5g3vrhJSUWiT4lfXAHD0TyBG7oBmRQOeVm3OjX1~DGOwn8mniJ6nQrpP5LhCKyqcYtbD3BgDW9UA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="GO classification of M. pinnata transcriptome. The results are summarized in three main categories as follows: biological process, cellular component, and molecular function. In total, 11 974 all-unigenes have been assigned GO terms. In some cases, one all-unigene has multiple terms." path-from-xml="dss00403.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6234592/dss00403.jpeg" path-from-xml="dss00403.jpeg" target="_blank">View large</a><a section="6234592" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/dss00403.jpeg?Expires=2147483647&Signature=PFcjPNP9Fqczbz6N9JMi3AG8vEseVfVamucWGar8UfT6n~qOZm9r4gJ7DLyu~qtw7zQvAXP5aRlQnzCSe9TZlav2dwne-Wo7lyLWXwcWc844vqC7tNFB05yQkTt0~2lf4c9h~f0b24c2g32bgskgWgfqeemNGpueTseSLjrhBc2t3pmgn6xN~N74~0TI063lpvM9QlG0UaDvfnJibW5oHQv6vMl6ZLfc8apIf-x-3vsKgoqSF8jJBGFuBU6HpCRI3mI0VPk-FPJHyM1D1RZRpEtHFRfHBkpW7p0UWHO91tWsVgcqZ11bBf6qLpiml9trIHKBFwB2obqq2u2lbG5Jsg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6234592&ar=420778&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00403.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>GO classification of <em>M. pinnata</em> transcriptome. The results are summarized in three main categories as follows: biological process, cellular component, and molecular function. In total, 11 974 all-unigenes have been assigned GO terms. In some cases, one all-unigene has multiple terms.</p></div></div><p>Finally, we performed the KEGG pathway analysis to assign biological pathways to the all-unigenes. In total, 23 697 out of 108 598 (21.8%) all-unigenes were assigned 124 KEGG pathways (Fig. <a reveal-id="DSS004F1" data-open="DSS004F1" class="link link-reveal link-table xref-fig">1</a>). These pathways belonged to 20 clades under five major KEGG categories, including ‘Metabolism’, ‘Genetic information processing’, ‘Environmental information processing’, ‘Cellular processes’, and ‘Organismal systems’ (Fig. <a reveal-id="DSS004F4" data-open="DSS004F4" class="link link-reveal link-table xref-fig">4</a>). Among them, ‘plant–pathogen interaction’, ‘spliceosome’, ‘biosynthesis of phenylpropanoids’, ‘biosynthesis of plant hormones’, and ‘ribosome’ were the top five pathways most represented by the all-unigenes. These results suggested that the living of <em>M. pinnata</em> was featured by dynamic metabolism as well as active environmental adaptation. </p>                        <a id="6234596" scrollto-destination="6234596"></a>
<div id="DSS004F4" data-id="DSS004F4" class="fig fig-section"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/m_dss00404.jpeg?Expires=2147483647&Signature=I5k8Gpd4Ry-QkGCZq~1r7gJk6no0WorhK93ECr8tAI9oijzQwpaPuD2G0rc07nhi7FCB9kTvF1jfdRKx5z84N1R9VGvce-G3pPH-ca4gE7gpMrpRWV44qLVj9U1Qpe5Wi8BPzheq~j3jwSFP-wpukbZDInHiJyDj7MrJgpvbjlK-~~7r7kjaNxpDKM3WFb9ur1DygUEfgamMN9N2N5Ewornhwj-q43gI1EJytleI8TGF5f4QnL2pIO2RIf4dO02Qk578yFnXkkhhWDVnl0u41zmjWbTvD5CYPf7-oDTNURZlDxp9jlSGWoTalv6qqZV6F7ewp7QymmtpdsJEp1y8Yg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of all-unigenes in each clade of the KEGG pathway maps. The all-unigenes were assigned 124 KEGG pathways within 20 clades under five major categories: Metabolism (I), Genetic information processing (II), Environmental information processing (III), Cellular processes (IV), and Organismal systems (V). In each category, the clades are listed according to their abundancies of all-unigenes." path-from-xml="dss00404.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6234596/dss00404.jpeg" path-from-xml="dss00404.jpeg" target="_blank">View large</a><a section="6234596" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/dss00404.jpeg?Expires=2147483647&Signature=VvCpVP-az7McvM1RcpeTf9uFEQazEOFEZs2-5cQFU5ZFdxLS4AV7x4cf8Ln4tpn6Vh-aTkSt8v2SkmBWuz2E~rRDKfMW7NBKJu6ye12DHRxtErykEY5MoctQ9uCnckPt1Lw9F~O3wEdH98iQsT3lMtEHrOaGi92vVjUbuJLu8Kfd~l-CahlO~gZmhSLJ-spOKSl3f-iLSXvRQQHqtRrvBWMBz1O4wCY58P30EBwfy4KxunuOqWee1s7jkk7NRb1DR9AISKpWHmtp~SD0NxeR2fsXy6fL526NpVbBIa~Wfa5QoWa6zl-6QnuDAjPl5tNolW8u1WltYQcs52iSpMipNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6234596&ar=420778&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00404.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of all-unigenes in each clade of the KEGG pathway maps. The all-unigenes were assigned 124 KEGG pathways within 20 clades under five major categories: Metabolism (I), Genetic information processing (II), Environmental information processing (III), Cellular processes (IV), and Organismal systems (V). In each category, the clades are listed according to their abundancies of all-unigenes.</p></div></div><div content-id="DSS004F4" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 4.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/m_dss00404.jpeg?Expires=2147483647&Signature=I5k8Gpd4Ry-QkGCZq~1r7gJk6no0WorhK93ECr8tAI9oijzQwpaPuD2G0rc07nhi7FCB9kTvF1jfdRKx5z84N1R9VGvce-G3pPH-ca4gE7gpMrpRWV44qLVj9U1Qpe5Wi8BPzheq~j3jwSFP-wpukbZDInHiJyDj7MrJgpvbjlK-~~7r7kjaNxpDKM3WFb9ur1DygUEfgamMN9N2N5Ewornhwj-q43gI1EJytleI8TGF5f4QnL2pIO2RIf4dO02Qk578yFnXkkhhWDVnl0u41zmjWbTvD5CYPf7-oDTNURZlDxp9jlSGWoTalv6qqZV6F7ewp7QymmtpdsJEp1y8Yg__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of all-unigenes in each clade of the KEGG pathway maps. The all-unigenes were assigned 124 KEGG pathways within 20 clades under five major categories: Metabolism (I), Genetic information processing (II), Environmental information processing (III), Cellular processes (IV), and Organismal systems (V). In each category, the clades are listed according to their abundancies of all-unigenes." path-from-xml="dss00404.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6234596/dss00404.jpeg" path-from-xml="dss00404.jpeg" target="_blank">View large</a><a section="6234596" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/dss00404.jpeg?Expires=2147483647&Signature=VvCpVP-az7McvM1RcpeTf9uFEQazEOFEZs2-5cQFU5ZFdxLS4AV7x4cf8Ln4tpn6Vh-aTkSt8v2SkmBWuz2E~rRDKfMW7NBKJu6ye12DHRxtErykEY5MoctQ9uCnckPt1Lw9F~O3wEdH98iQsT3lMtEHrOaGi92vVjUbuJLu8Kfd~l-CahlO~gZmhSLJ-spOKSl3f-iLSXvRQQHqtRrvBWMBz1O4wCY58P30EBwfy4KxunuOqWee1s7jkk7NRb1DR9AISKpWHmtp~SD0NxeR2fsXy6fL526NpVbBIa~Wfa5QoWa6zl-6QnuDAjPl5tNolW8u1WltYQcs52iSpMipNA__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6234596&ar=420778&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00404.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of all-unigenes in each clade of the KEGG pathway maps. The all-unigenes were assigned 124 KEGG pathways within 20 clades under five major categories: Metabolism (I), Genetic information processing (II), Environmental information processing (III), Cellular processes (IV), and Organismal systems (V). In each category, the clades are listed according to their abundancies of all-unigenes.</p></div></div>                            <h3 scrollto-destination=6234597 id="6234597" class="section-title" data-legacy-id=s3c>3.3. Identification and functional classification of DEGs</h3>
<p>We singled out the DEGs with two criteria for their expression profiles: (i) the average fold change in gene expression level was more than or equal to 2-fold between the seawater- and freshwater-treated samples and (ii) the false discovery rate of the single-sample <em>t</em>-test was &lt;0.1%. Under these criteria, 23 815 out of 108 598 (21.9%) all-unigenes were identified to be differentially expressed in at least one tissue following the salt treatment (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S3</a>; Fig. <a reveal-id="DSS004F1" data-open="DSS004F1" class="link link-reveal link-table xref-fig">1</a>). In this study, DEGs with higher expression levels in seawater-treated samples when compared with freshwater-treated samples were denoted as ‘up-regulated’, while those with lower expression levels in seawater-treated samples were denoted as ‘down-regulated’. Overall, there were much more DEGs in the roots than those in the leaves (Fig. <a reveal-id="DSS004F5" data-open="DSS004F5" class="link link-reveal link-table xref-fig">5</a>). In the root, the number of up-regulated DEGs was less than that of down-regulated DEGs. The situation was reversed in the leaf, in which more up-regulated DEGs than down-regulated DEGs were identified. Only a small portion of DEGs (1050 up-regulated and 622 down-regulated) shared common tendency of expression changes between the roots and the leaves. A still less of DEGs showed a completely opposite tendency of expression changes between the two tissues (577 DEGs up-regulated in the leaf but down-regulated in the root, and 157 DEGs down-regulated in the leaf but up-regulated in the root). The remaining majority of DEGs were exclusively up-regulated or down-regulated in either tissue. </p>                        <a id="6234599" scrollto-destination="6234599"></a>
<div id="DSS004F5" data-id="DSS004F5" class="fig fig-section"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/m_dss00405.jpeg?Expires=2147483647&Signature=lWJWisbkIhX18JMutLpKpyco3Hjg2FW0ptWOR7qG7p8myfjOclcc7s8L3EPSYsFldrapkYuAPNuzuBgxAcQemDBG40nBnzUxEXjiBal2sXQqHOlhnhCGss3EK-A-98yB22qpvJIUXsUbfPdEKBappXq-RLbgtytDs3FX0Oy3itGzc8G8FWRfrwRApwBYt~QyVbDORxZM9zFwmjcD2h3Tb06cvcgGCd4DduMJVhL0E1RZV2lf-MZQOBzO2kyWQ5eAsV5OQwEKgzBA5n9t4Z7tl~Kg0XyEtAbjublHa7EyUJtmfu3QJeFJhKiPzYzvlbGJIvAX7I7I90fhCPlWeBuEiw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of DEGs in the root and the leaf of M. pinnata. The numbers of DEGs that were exclusively up- or down-regulated in one tissue are shown in each circle. The numbers of DEGs with a common or opposite tendency of expression changes between the two tissues are shown in the overlapping regions. The total numbers of up- or down-regulated genes in each tissue are shown outside the circles." path-from-xml="dss00405.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6234599/dss00405.jpeg" path-from-xml="dss00405.jpeg" target="_blank">View large</a><a section="6234599" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/dss00405.jpeg?Expires=2147483647&Signature=gLumCIvvU5dNKJ-hzcbln39Xh9NLxtiK39ZABOEcZ-76SPI8aMXRHD-MfYogMOjcxWV-Nq5DlnhvopmJ9thQJf-UHhzeFcs1pkWEFBcBWerc~aGby6GxMJZgZ8SFtkZMvM2iRR0tmNv-J3j6bxaQ2jLBsPIUISNZFIlteAHuXYVXHX~IcFPDDxQslVNBQXfgu1IDhnEpIDW2ERduSE-tD3pGeEQF3lLgqRNSWcqXJt7bmRexRvU~bMGA5py8C55D9-xZOAcrOrjmzSpCgBe8Aow8uHOxkiX9puKNT5rD2xKrzoj4ZI~Mifdj2UrPd640gpDOiE-XxX6aTidREODILQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6234599&ar=420778&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00405.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of DEGs in the root and the leaf of <em>M. pinnata</em>. The numbers of DEGs that were exclusively up- or down-regulated in one tissue are shown in each circle. The numbers of DEGs with a common or opposite tendency of expression changes between the two tissues are shown in the overlapping regions. The total numbers of up- or down-regulated genes in each tissue are shown outside the circles.</p></div></div><div content-id="DSS004F5" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 5.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/m_dss00405.jpeg?Expires=2147483647&Signature=lWJWisbkIhX18JMutLpKpyco3Hjg2FW0ptWOR7qG7p8myfjOclcc7s8L3EPSYsFldrapkYuAPNuzuBgxAcQemDBG40nBnzUxEXjiBal2sXQqHOlhnhCGss3EK-A-98yB22qpvJIUXsUbfPdEKBappXq-RLbgtytDs3FX0Oy3itGzc8G8FWRfrwRApwBYt~QyVbDORxZM9zFwmjcD2h3Tb06cvcgGCd4DduMJVhL0E1RZV2lf-MZQOBzO2kyWQ5eAsV5OQwEKgzBA5n9t4Z7tl~Kg0XyEtAbjublHa7EyUJtmfu3QJeFJhKiPzYzvlbGJIvAX7I7I90fhCPlWeBuEiw__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Number of DEGs in the root and the leaf of M. pinnata. The numbers of DEGs that were exclusively up- or down-regulated in one tissue are shown in each circle. The numbers of DEGs with a common or opposite tendency of expression changes between the two tissues are shown in the overlapping regions. The total numbers of up- or down-regulated genes in each tissue are shown outside the circles." path-from-xml="dss00405.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6234599/dss00405.jpeg" path-from-xml="dss00405.jpeg" target="_blank">View large</a><a section="6234599" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/dss00405.jpeg?Expires=2147483647&Signature=gLumCIvvU5dNKJ-hzcbln39Xh9NLxtiK39ZABOEcZ-76SPI8aMXRHD-MfYogMOjcxWV-Nq5DlnhvopmJ9thQJf-UHhzeFcs1pkWEFBcBWerc~aGby6GxMJZgZ8SFtkZMvM2iRR0tmNv-J3j6bxaQ2jLBsPIUISNZFIlteAHuXYVXHX~IcFPDDxQslVNBQXfgu1IDhnEpIDW2ERduSE-tD3pGeEQF3lLgqRNSWcqXJt7bmRexRvU~bMGA5py8C55D9-xZOAcrOrjmzSpCgBe8Aow8uHOxkiX9puKNT5rD2xKrzoj4ZI~Mifdj2UrPd640gpDOiE-XxX6aTidREODILQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6234599&ar=420778&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00405.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Number of DEGs in the root and the leaf of <em>M. pinnata</em>. The numbers of DEGs that were exclusively up- or down-regulated in one tissue are shown in each circle. The numbers of DEGs with a common or opposite tendency of expression changes between the two tissues are shown in the overlapping regions. The total numbers of up- or down-regulated genes in each tissue are shown outside the circles.</p></div></div><p>To further characterize the expression changes in these two tissues, we conducted GO enrichment analysis for the DEGs with the whole transcriptome set as the background. In the root, the significantly overrepresented GO terms of ‘Biological process’ were ‘gene expression’, ‘sulphur metabolic process’, ‘response to oxidative stress’, ‘cellular amino acid derivative metabolic process’, ‘oxidation reduction’, and ‘secondary metabolic process’ (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S4</a>). In the leaf, the significantly overrepresented GO terms of ‘Biological process’ included ‘oxidation reduction’, ‘cellular amino acid derivative metabolic processes’, and ‘cellular aromatic compound metabolic process’. From these results, we noticed that both tissues of <em>M. pinnata</em> tend to mobilize genes related to oxidation reduction and cellular amino acid metabolism. Furthermore, as the salt-induced deleterious effects varied among different parts of a plant, the roots and the leaves also activated some distinct groups of genes (i.e. genes in ‘translation’, ‘sulphur metabolic process’, and ‘response to oxidative stress’ for the root, genes in ‘phenylpropanoid metabolic process’ for the leaf) to overcome them.</p><p>The KEGG pathway enrichment analysis for the DEGs also revealed both common and tissue-specific patterns of overrepresentations. The DEGs were enriched in 37 pathways in the root and 31 pathways in the leaf (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S5</a>). Among them, 19 pathways appeared in both tissues and were mostly related to amino acid metabolism, lipid metabolism, energy metabolism, biosynthesis of secondary metabolites, and environmental adaptation. Generally speaking, this finding demonstrated that both tissues of <em>M. pinnata</em> demand more energy and more biological substances to cope with salt stress. In addition to these common pathways, there were some tissue-specific pathways with overrepresented DEGs. For example, in the clade of ‘Energy metabolism’, the leaves of <em>M. pinnata</em> were enriched with DEGs in ‘oxidative phosphorylation’ and ‘photosynthesis’ pathways, and these processes were also employed by the leaves of other plant species under salt stress.<sup><a reveal-id="DSS004C41  DSS004C43" data-open="DSS004C41  DSS004C43" class="link link-ref link-reveal xref-bibr">41–43</a></sup> On the other hand, the roots of <em>M. pinnata</em> were enriched with DEGs in the ‘sulphur metabolism’ pathway, which was likely related to their sulphur-rich living conditions.<sup><a reveal-id="DSS004C44" data-open="DSS004C44" class="link link-ref link-reveal xref-bibr">44</a></sup></p>                            <h3 scrollto-destination=6234602 id="6234602" class="section-title" data-legacy-id=s3d>3.4. Experimental verification of DEGs</h3>
<p>To assess the reliability of our sequencing-based approach in identifying salt-responsive genes, we monitored the expression of candidate DEGs. In the first place, we performed RT–PCR for 100 randomly chosen candidates with specific primers (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S6</a>). We compared gene expression levels at the three time points separately between stressed and control samples. The expression profiles of 70 candidates were basically in agreement with the predictions from the Illumina sequencing results. Six candidates displayed incongruent expression profiles with the predictions. The remaining 24 candidates showed non-specifically amplified products or no products. In other words, 70–92% of the candidates were confirmed by RT–PCR analyses. The confirmed candidates were differentially expressed in at least one of the three time points (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Fig. S4</a>). In the root, the maximal number of confirmed DEGs occurred at the 4 h time point (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S7</a>). Conversely, the minimal number of confirmed DEGs in the leaf occurred at the 4 h time point, whereas the maximum occurred at the 8 h time point. From these results, we can infer a delay of transcriptional response to salt stress in the leaf, which may be explained by a later salt accumulation in this tissue.<sup><a reveal-id="DSS004C9" data-open="DSS004C9" class="link link-ref link-reveal xref-bibr">9</a></sup></p><p>In the second place, we employed real-time PCR to delineate the detailed expression profiles of 20 candidate DEGs. These candidates mainly included genes that were reported to be related to salt stress response in other plant species, such as genes involved in several signal transduction pathways, genes associated with osmotic adjustments or detoxification of reactive oxygen species (ROS), and genes encoding transcription factors (<a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Table S8</a>). Besides, there were three candidates with unknown functions, which represented a new resource for gene transfer from <em>M. pinnata</em> to its legume relatives or other crops.</p><p>All of these 20 candidates showed significantly differential expression with the original expression level in at least one time point following the seawater treatments (Fig. <a reveal-id="DSS004F6" data-open="DSS004F6" class="link link-reveal link-table xref-fig">6</a>). In other words, all candidates displayed an expression pattern roughly consistent with the prediction based on the Illumina sequencing results. In most cases, the expression changes were more dramatic in the roots than in the leaves. Besides, most of the candidates were up-regulated from the 0 to 4 h time point, and then they gradually restored to the original expression levels. The pace of the rise and fall in gene expression levels was roughly coincident with the wilting and recovering of the young leaves. This phenomenon was also observed in other mangroves,<sup><a reveal-id="DSS004C42" data-open="DSS004C42" class="link link-ref link-reveal xref-bibr">42</a></sup> and could be explained by referring to the different phases of salt stress response.<sup><a reveal-id="DSS004C43" data-open="DSS004C43" class="link link-ref link-reveal xref-bibr">43</a></sup> The up-regulation and restoration of the DEGs in both tissues may parallel the phases of ‘salt accumulation’ and ‘osmotic recovery’, and the delay in gene activation in the leaves may result from an extra phase of ‘dehydration’ preceding the above two phases. The leaves wilted during the ‘dehydration’ phase, and then revived gradually during the following two phases. In this sense, these results again demonstrated that a later salt accumulation was responsible for the delay in the transcriptional response in the leaves. </p>                        <a id="6234606" scrollto-destination="6234606"></a>
<div id="DSS004F6" data-id="DSS004F6" class="fig fig-section"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/m_dss00406.jpeg?Expires=2147483647&Signature=m7~o3MuJfd2QPjI-0rzM1Q~msD98wY1dzylwBshMNO4lfFu2SwZxWRvHEYJ~fvs2IMSZgnJ5OLE9xXRhWNfzcm22MRhMiE3x8LOXSC-Aclu1KZk-mIBKK-N4~DzjAKTEuWMwB0USbZijUUZ80~4N~xs3fbz5h1iml01saqBZkUv9XHGdwgNU~gVrdQKz-40k5uMf5clcMod2DR8RT6cAF6q~r-peMoto8RNXaJLr9zQE8FXrCQkmaj~Llk-2k~zM2n-tKwxHnV2MtvYC5pXMmV-VoR-QDPA3h2OYqnKYje42YCXbHifjAPQpj~mzBRA8tuaeH59pak9ikCqldfs8~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relative expression levels of 20 DEGs at a series of time points following the seawater treatments. The relative gene expression levels as expressed by 2−ΔΔCT were determined separately for each tissue and were presented as the mean ± SD. Significant differences (P &lt; 0.01, LSD-t test) between the expression levels at 1, 2, 4, and 8 h time points and those at 0 h are indicated by asterisks." path-from-xml="dss00406.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6234606/dss00406.jpeg" path-from-xml="dss00406.jpeg" target="_blank">View large</a><a section="6234606" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/dss00406.jpeg?Expires=2147483647&Signature=C4eOa1zwArkUpMXJA1uUFhXnVnF6MCbSESFbHhJ8ZSblVSK880Tv4OrQ1SrKUSFYH4w5hx0ca7aUFBTCPRtAhamX97Xx~z2xIZm1wdRc4N7uxSw5E9iaQnIp0D4mjEAazrTgKW91Z2Qvjb7XnlqUEshrII3iYSAVpe8nf-Nahw3IseAg~0PpI90XJ87LqxrBT-D-BTDhpva6gpN8DtZTkM6JhOym9zIJ1724DVL8o7E44TUG--P9~fxzwv6CbWvbSySgKiZbTKgvhXyqQRwSLCJnSdBRiNmhRwQTQ8EYjCd3iNa50crjWKZmWrW8tG0c2lwkjlChlpHJ8OFmjxEzKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6234606&ar=420778&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00406.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relative expression levels of 20 DEGs at a series of time points following the seawater treatments. The relative gene expression levels as expressed by 2<sup>−ΔΔCT</sup> were determined separately for each tissue and were presented as the mean ± SD. Significant differences (<em>P</em> &lt; 0.01, LSD-<em>t</em> test) between the expression levels at 1, 2, 4, and 8 h time points and those at 0 h are indicated by asterisks.</p></div></div><div content-id="DSS004F6" class="fig fig-modal reveal-modal"><div class="label fig-label">Figure 6.</div><div class="graphic-wrap"><img class="content-image" src="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/m_dss00406.jpeg?Expires=2147483647&Signature=m7~o3MuJfd2QPjI-0rzM1Q~msD98wY1dzylwBshMNO4lfFu2SwZxWRvHEYJ~fvs2IMSZgnJ5OLE9xXRhWNfzcm22MRhMiE3x8LOXSC-Aclu1KZk-mIBKK-N4~DzjAKTEuWMwB0USbZijUUZ80~4N~xs3fbz5h1iml01saqBZkUv9XHGdwgNU~gVrdQKz-40k5uMf5clcMod2DR8RT6cAF6q~r-peMoto8RNXaJLr9zQE8FXrCQkmaj~Llk-2k~zM2n-tKwxHnV2MtvYC5pXMmV-VoR-QDPA3h2OYqnKYje42YCXbHifjAPQpj~mzBRA8tuaeH59pak9ikCqldfs8~g__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA" alt="Relative expression levels of 20 DEGs at a series of time points following the seawater treatments. The relative gene expression levels as expressed by 2−ΔΔCT were determined separately for each tissue and were presented as the mean ± SD. Significant differences (P &lt; 0.01, LSD-t test) between the expression levels at 1, 2, 4, and 8 h time points and those at 0 h are indicated by asterisks." path-from-xml="dss00406.jpeg" /><div class="fig-orig original-slide"><a class="fig-view-orig js-view-large openInAnotherWindow" href="/view-large/figure/6234606/dss00406.jpeg" path-from-xml="dss00406.jpeg" target="_blank">View large</a><a section="6234606" href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/dss00406.jpeg?Expires=2147483647&Signature=C4eOa1zwArkUpMXJA1uUFhXnVnF6MCbSESFbHhJ8ZSblVSK880Tv4OrQ1SrKUSFYH4w5hx0ca7aUFBTCPRtAhamX97Xx~z2xIZm1wdRc4N7uxSw5E9iaQnIp0D4mjEAazrTgKW91Z2Qvjb7XnlqUEshrII3iYSAVpe8nf-Nahw3IseAg~0PpI90XJ87LqxrBT-D-BTDhpva6gpN8DtZTkM6JhOym9zIJ1724DVL8o7E44TUG--P9~fxzwv6CbWvbSySgKiZbTKgvhXyqQRwSLCJnSdBRiNmhRwQTQ8EYjCd3iNa50crjWKZmWrW8tG0c2lwkjlChlpHJ8OFmjxEzKQ__&Key-Pair-Id=APKAIE5G5CRDK6RD3PGA&sec=6234606&ar=420778&xsltPath=~/UI/app/XSLT&imagename=" class="download-slide" path-from-xml="dss00406.jpeg" target="_blank">Download slide</a></div></div><div class="caption fig-caption"><p>Relative expression levels of 20 DEGs at a series of time points following the seawater treatments. The relative gene expression levels as expressed by 2<sup>−ΔΔCT</sup> were determined separately for each tissue and were presented as the mean ± SD. Significant differences (<em>P</em> &lt; 0.01, LSD-<em>t</em> test) between the expression levels at 1, 2, 4, and 8 h time points and those at 0 h are indicated by asterisks.</p></div></div>                            <h2 scrollto-destination=6234607 id="6234607" class="section-title" data-legacy-id=s4>4. Discussion</h2>
<p>Mangroves are a group of plant species that live along the sea coast with the ability to tolerate high salinity.<sup><a reveal-id="DSS004C44" data-open="DSS004C44" class="link link-ref link-reveal xref-bibr">44</a></sup> The extremely high salinity within their living conditions confers mangroves unique resources for exploiting the dynamic nature of their gene regulation during salt exposure. While the collections of ESTs based on cDNA libraries from several mangrove species have been described,<sup><a reveal-id="DSS004C45  DSS004C47" data-open="DSS004C45  DSS004C47" class="link link-ref link-reveal xref-bibr">45–47</a></sup> the transcript data for such a plant group containing diverse species are still scarce. In a recent study using the 454 pyrosequencing platform, 67 375 and 96 989 unisequences with a mean length of 360 and 433 bp were produced for two taxonomically diverse mangroves.<sup><a reveal-id="DSS004C48" data-open="DSS004C48" class="link link-ref link-reveal xref-bibr">48</a></sup> Compared with these results, our transcriptome for <em>M. pinnata</em> exceed in both the total number and the mean length of the unisequences. The mean length of the all-unigenes in our transcriptome was also longer than those reported for other plant transcriptomics using either the Illumina sequencing technology<sup><a reveal-id="DSS004C49" data-open="DSS004C49" class="link link-ref link-reveal xref-bibr">49</a>,<a reveal-id="DSS004C50" data-open="DSS004C50" class="link link-ref link-reveal xref-bibr">50</a></sup> or the 454 pyrosequencing technology.<sup><a reveal-id="DSS004C51" data-open="DSS004C51" class="link link-ref link-reveal xref-bibr">51</a>,<a reveal-id="DSS004C52" data-open="DSS004C52" class="link link-ref link-reveal xref-bibr">52</a></sup></p><p>The enhancement in the gene length of our transcriptome may be attributed to various factors, such as the optimized assembly parameters (e.g. the K-mer size). One of the most important reasons for this enhancement might be that the final comprehensive transcriptome was indeed integrated by four sub-transcriptomes, each of which was comparable in gene number and gene length with those in the previous studies. The final unisequences possessed a relatively high depth of average coverage (∼30-fold) by reads from each sub-transcriptome. We also noticed an increase in the number of unisequences from the sub-transcriptome to the comprehensive transcriptome. The ‘extra’ unisequences represented either splice variants or genes with tissue-specific or stress-specific expression. So far, our transcriptome for <em>M. pinnata</em> has presented many more sequences than the sequencing efforts for any other mangrove species have produced.<sup><a reveal-id="DSS004C53" data-open="DSS004C53" class="link link-ref link-reveal xref-bibr">53</a></sup></p><p>As for the GO and KEGG annotations, the aforementioned two mangrove species displayed remarkable similarities to each other.<sup><a reveal-id="DSS004C48" data-open="DSS004C48" class="link link-ref link-reveal xref-bibr">48</a></sup> They both showed a lack of representation in the GO category ‘response to stimulus’, whereas the transcriptome of <em>M. pinnata</em> was well represented in this category. We suggest that this discrepancy may be partly attributed to the different status of adaptation for each mangrove species. It has been hypothesized that semi-mangroves represent an intermediate during the long-term evolution of mangroves in adaptation to the high-salinity environments.<sup><a reveal-id="DSS004C54" data-open="DSS004C54" class="link link-ref link-reveal xref-bibr">54</a></sup> Accordingly, while those two mangroves that were ‘completely’ adapted to the extreme conditions did not require so many transcripts for responding to stimulus, <em>M. pinnata</em> with a ‘half’ tolerance may still be in great need of these transcripts. With the exception of this discrepancy, the functional categories were similar between <em>M. pinnata</em> and those two mangroves. For example, all these three species possessed large quantities of transcripts related to energy metabolism.</p><p>Based on the comparison of gene expression levels between stressed and control samples using RPKM values, we screened out 23 815 all-unigenes as candidate salt-responsive genes, which took up ∼22% of the total all-unigenes. This percentage was higher than the percentages of salt-induced DEGs derived from other related species, such as <em>Bruguiera gymnorrhiza</em> (another mangrove) and <em>Medicago truncatula</em> (another legume), using the array-based platform.<sup><a reveal-id="DSS004C41" data-open="DSS004C41" class="link link-ref link-reveal xref-bibr">41</a>,<a reveal-id="DSS004C55" data-open="DSS004C55" class="link link-ref link-reveal xref-bibr">55</a></sup> First, the differences in DEG percentages might reflect the inherent differences between species in their transcriptome profiles. Secondly, it might reflect a greater sensitivity and higher resolution of the sequencing-based platform in identifying DEGs. The sequencing-based approaches offer several advantages over the existing hybridization-based approaches, which often suffer from background and cross-hybridization problems.<sup><a reveal-id="DSS004C56" data-open="DSS004C56" class="link link-ref link-reveal xref-bibr">56</a></sup> Thirdly, since each sample for Illumina sequencing was pooled by equal amounts of RNAs from 2, 4, and 8 h-treated plants, the RPKM values in each sub-transcriptome actually reflected the average gene expression levels for the three time points. Theoretically speaking, our methods would miss some genes with 2-fold difference in expression levels between stressed and control samples at less than two time points due to the diluting effect of the pooled samples. Nevertheless, it could help us to detect genes with more dramatic changes (4-fold or greater) in expression levels at even only one time point, which might be more closely related to salt stress response. Lastly, we still should not neglect the possibilities of falsely discovered DEGs through the sequence-based platform. In our study, both RT–PCR and real-time PCR analyses demonstrated a high reliability of the candidate salt-responsive genes identified by sequencing-based approach.</p><p>The DEGs identified in <em>M. pinnata</em> were enriched in the pathways related to protein synthesis and biosynthesis of secondary metabolites. A comparative study between glycophyte and halophyte showed that Arabidopsis adopted a global defence strategy that requires bulk protein synthesis, whereas salt cress mainly activated genes that functioned in protein modification and redistribution.<sup><a reveal-id="DSS004C16" data-open="DSS004C16" class="link link-ref link-reveal xref-bibr">16</a></sup> It seemed that protein modification might be a faster mechanism for stress response than protein synthesis. In this aspect, <em>M. pinnata</em> was more similar to glycophytes. On the other hand, salt cress showed a salt-induced increase in secondary metabolites,<sup><a reveal-id="DSS004C16" data-open="DSS004C16" class="link link-ref link-reveal xref-bibr">16</a></sup> which were also found to accumulate in <em>M. pinnata</em> but not in Arabidopsis. Taken together, these two results suggest that <em>M. pinnata</em> can utilize the adjustment mechanisms preferentially adopted by either glycophytes or halophytes.</p><p>In conclusion, we characterized a comprehensive transcriptome of a semi-mangrove plant <em>M. pinnata</em> using the Illumina sequencing technology. This transcriptome will not only provide plenty of informative data, but also expand the vision of the genetic basis underlying adaptation for the mangrove community. The candidate salt-responsive genes identified in <em>M. pinnata</em> contain both the previously reported salt-responsive genes and some species-specific genes which will be a new resource for molecular breeding in legumes or other crops. The functional enrichment analyses for the candidate genes showed both common and tissue-specific patterns of transcriptome remodelling under salt treatments. Moreover, the experimental analyses for some candidate genes provided some evidences for a delay in transcriptional responses in the leaves. Our work has demonstrated a high reliability of the sequencing-based approach in identifying stress-responsive genes in non-model organisms and established a connection between glycophytes and halophytes in the views of transcriptomic adjustments to salt stress.</p>                            <h2 scrollto-destination=6234614 id="6234614" class="section-title" data-legacy-id=s5>Supplementary Data</h2>
<p><a class="supplementary-data tab-trigger jumplink scrollTo" target="SupplementalTab" href="#supplementary-data" data-tab="#supplementary-data">Supplementary Data are available at</a><a class="link link-uri" href="http://www.dnaresearch.oxfordjournals.org">www.dnaresearch.oxfordjournals.org</a><a class="link link-uri" href="http://dnaresearch.oxfordjournals.org/lookup/suppl/doi:10.1093/dnares/dss004/-/DC1" target="_blank">.</a></p>                            <h2 scrollto-destination=6234616 id="6234616" class="section-title" data-legacy-id=s6>Funding</h2>
<p>This work was supported by the National Basic Research Program of China (2012CB114200), the Startup Project of Marine Biotechnology from Shenzhen University, and the National Transgenic Research Project (2009ZX08004-006B).</p>                            <h2 scrollto-destination=6234618 id="6234618" class="backacknowledgements-title" >Acknowledgements</h2>
<p>We appreciate Mr Shichuan Li from the Dongzhaigang National Nature Reserve for assistance in collecting the seeds of <em>M. pinnata</em>. We are also grateful to three anonymous editors from the Nature Publishing Group Language Editing for improving the text.</p>                            <h2 scrollto-destination=6234620 id="6234620" class="backreferences-title" >References</h2>
<div class="ref-list"><div content-id="DSS004C1" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C1" class="ref-content "><div class="label -label">1</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Flowers</div> <div class="given-names">T.J.</div></div>. </div><div class="article-title">Improving crop salt tolerance</div>, <div class="source">J. Exp. Bot.</div> , <div class="year">2004</div>, vol. <div class="volume">55</div> (pg. <div class="fpage">307</div>-<div class="lpage">19</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/jxb/erh003" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/jxb/erh003</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Improving%20crop%20salt%20tolerance&amp;author=T.J.%20Flowers&amp;publication_year=2004&amp;journal=J.%20Exp.%20Bot.&amp;volume=55&amp;pages=307-19" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/jxb/erh003" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fjxb%2Ferh003" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fjxb%2Ferh003"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14718494" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C2" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C2" class="ref-content "><div class="label -label">2</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hasegawa</div> <div class="given-names">P.M.</div></div>,  <div class="name"><div class="surname">Bressan</div> <div class="given-names">R.A.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">J.K.</div></div>,  <div class="name"><div class="surname">Bohnert</div> <div class="given-names">H. J.</div></div>. </div><div class="article-title">Plant cellular and molecular responses to high salinity</div>, <div class="source">Annu. Rev. Plant Physiol. Plant Mol. Biol.</div> , <div class="year">2000</div>, vol. <div class="volume">51</div> (pg. <div class="fpage">463</div>-<div class="lpage">99</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev.arplant.51.1.463" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1146/annurev.arplant.51.1.463</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20cellular%20and%20molecular%20responses%20to%20high%20salinity&amp;author=P.M.%20Hasegawa&amp;author=R.A.%20Bressan&amp;author=J.K.%20Zhu&amp;author=H.%20J.%20Bohnert&amp;publication_year=2000&amp;journal=Annu.%20Rev.%20Plant%20Physiol.%20Plant%20Mol.%20Biol.&amp;volume=51&amp;pages=463-99" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.51.1.463" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.51.1.463" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.51.1.463"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15012199" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C3" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C3" class="ref-content "><div class="label -label">3</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">J.K.</div></div>. </div><div class="article-title">Salt and drought stress signal transduction in plants</div>, <div class="source">Annu. Rev. Plant Biol.</div> , <div class="year">2002</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">247</div>-<div class="lpage">73</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1146/annurev.arplant.53.091401.143329" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1146/annurev.arplant.53.091401.143329</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Salt%20and%20drought%20stress%20signal%20transduction%20in%20plants&amp;author=J.K.%20Zhu&amp;publication_year=2002&amp;journal=Annu.%20Rev.%20Plant%20Biol.&amp;volume=53&amp;pages=247-73" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1146/annurev.arplant.53.091401.143329" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1146%2Fannurev.arplant.53.091401.143329" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1146%2Fannurev.arplant.53.091401.143329"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12221975" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C4" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C4" class="ref-content "><div class="label -label">4</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Seki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Narusaka</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Ishida</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">Monitoring the expression profiles of 7000 <em>Arabidopsis</em> genes under drought, cold and high-salinity stresses using a full-length cDNA microarray</div>, <div class="source">Plant J.</div> , <div class="year">2002</div>, vol. <div class="volume">31</div> (pg. <div class="fpage">279</div>-<div class="lpage">92</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1046/j.1365-313X.2002.01359.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1046/j.1365-313X.2002.01359.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Monitoring%20the%20expression%20profiles%20of%207000%20Arabidopsis%20genes%20under%20drought%2C%20cold%20and%20high-salinity%20stresses%20using%20a%20full-length%20cDNA%20microarray&amp;author=M.%20Seki&amp;author=M.%20Narusaka&amp;author=J.%20Ishida&amp;publication_year=2002&amp;journal=Plant%20J.&amp;volume=31&amp;pages=279-92" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1046/j.1365-313X.2002.01359.x" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1046%2Fj.1365-313X.2002.01359.x" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1046%2Fj.1365-313X.2002.01359.x"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12164808" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C5" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C5" class="ref-content "><div class="label -label">5</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Jiang</div> <div class="given-names">Y.Q.</div></div>,  <div class="name"><div class="surname">Deyholos</div> <div class="given-names">M.K.</div></div>. </div><div class="article-title">Comprehensive transcriptional profiling of NaCl-stressed <em>Arabidopsis</em> roots reveals novel classes of responsive genes</div>, <div class="source">BMC Plant Biol.</div> , <div class="year">2006</div>, vol. <div class="volume">6</div> pg. <div class="fpage">25</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2229-6-25" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2229-6-25</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comprehensive%20transcriptional%20profiling%20of%20NaCl-stressed%20Arabidopsis%20roots%20reveals%20novel%20classes%20of%20responsive%20genes&amp;author=Y.Q.%20Jiang&amp;author=M.K.%20Deyholos&amp;publication_year=2006&amp;journal=BMC%20Plant%20Biol.&amp;volume=6&amp;pages=25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2229-6-25" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2229-6-25" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2229-6-25"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17038189" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C6" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C6" class="ref-content "><div class="label -label">6</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Rabbani</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Maruyama</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Abe</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">Monitoring expression profiles of rice genes under cold, drought, and high-salinity stresses and abscisic acid application using cDNA microarray and RNA get-blot analyses</div>, <div class="source">Plant Physiol.</div> , <div class="year">2003</div>, vol. <div class="volume">133</div> (pg. <div class="fpage">1755</div>-<div class="lpage">67</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.103.025742" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.103.025742</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Monitoring%20expression%20profiles%20of%20rice%20genes%20under%20cold%2C%20drought%2C%20and%20high-salinity%20stresses%20and%20abscisic%20acid%20application%20using%20cDNA%20microarray%20and%20RNA%20get-blot%20analyses&amp;author=M.A.%20Rabbani&amp;author=K.%20Maruyama&amp;author=H.%20Abe&amp;publication_year=2003&amp;journal=Plant%20Physiol.&amp;volume=133&amp;pages=1755-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.103.025742" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.103.025742" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.103.025742"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/14645724" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C7" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C7" class="ref-content "><div class="label -label">7</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Walia</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wilson</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Zeng</div> <div class="given-names">L.H.</div></div>,  <div class="name"><div class="surname">Ismail</div> <div class="given-names">A.M.</div></div>,  <div class="name"><div class="surname">Condamine</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Close</div> <div class="given-names">T.J.</div></div>. </div><div class="article-title">Genome-wide transcriptional analysis of salinity stressed <em>japonica</em> and <em>indica</em> rice genotypes during panicle initiation stage</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2007</div>, vol. <div class="volume">63</div> (pg. <div class="fpage">609</div>-<div class="lpage">23</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11103-006-9112-0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11103-006-9112-0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome-wide%20transcriptional%20analysis%20of%20salinity%20stressed%20japonica%20and%20indica%20rice%20genotypes%20during%20panicle%20initiation%20stage&amp;author=H.%20Walia&amp;author=C.%20Wilson&amp;author=L.H.%20Zeng&amp;author=A.M.%20Ismail&amp;author=P.%20Condamine&amp;author=T.J.%20Close&amp;publication_year=2007&amp;journal=Plant%20Mol.%20Biol.&amp;volume=63&amp;pages=609-23" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-006-9112-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-006-9112-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-006-9112-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17160619" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C8" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C8" class="ref-content "><div class="label -label">8</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Miyazaki</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Kawai</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Deyholos</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Galbraith</div> <div class="given-names">D.W.</div></div>,  <div class="name"><div class="surname">Bohnert</div> <div class="given-names">H.J.</div></div>. </div><div class="article-title">Temporal progression of gene expression responses to salt shock in maize roots</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2003</div>, vol. <div class="volume">52</div> (pg. <div class="fpage">873</div>-<div class="lpage">91</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1023/A:1025029026375" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1023/A:1025029026375</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Temporal%20progression%20of%20gene%20expression%20responses%20to%20salt%20shock%20in%20maize%20roots&amp;author=H.%20Wang&amp;author=S.%20Miyazaki&amp;author=K.%20Kawai&amp;author=M.%20Deyholos&amp;author=D.W.%20Galbraith&amp;author=H.J.%20Bohnert&amp;publication_year=2003&amp;journal=Plant%20Mol.%20Biol.&amp;volume=52&amp;pages=873-91" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1023/A:1025029026375" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1023%2FA:1025029026375" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1023%2FA:1025029026375"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/13677474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C9" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C9" class="ref-content "><div class="label -label">9</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Qing</div> <div class="given-names">D.J.</div></div>,  <div class="name"><div class="surname">Lu</div> <div class="given-names">H.F.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Dong</div> <div class="given-names">H.T.</div></div>,  <div class="name"><div class="surname">Dong</div> <div class="given-names">D.F.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.Z.</div></div>. </div><div class="article-title">Comparative profiles of gene expression in leaves and roots of maize seedlings under conditions of salt stress and the removal of salt stress</div>, <div class="source">Plant Cell Physiol.</div> , <div class="year">2009</div>, vol. <div class="volume">50</div> (pg. <div class="fpage">889</div>-<div class="lpage">903</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/pcp/pcp038" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/pcp/pcp038</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20profiles%20of%20gene%20expression%20in%20leaves%20and%20roots%20of%20maize%20seedlings%20under%20conditions%20of%20salt%20stress%20and%20the%20removal%20of%20salt%20stress&amp;author=D.J.%20Qing&amp;author=H.F.%20Lu&amp;author=N.%20Li&amp;author=H.T.%20Dong&amp;author=D.F.%20Dong&amp;author=Y.Z.%20Li&amp;publication_year=2009&amp;journal=Plant%20Cell%20Physiol.&amp;volume=50&amp;pages=889-903" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/pcp/pcp038" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fpcp%2Fpcp038" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fpcp%2Fpcp038"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19264788" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C10" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C10" class="ref-content "><div class="label -label">10</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gong</div> <div class="given-names">Z.Z.</div></div>,  <div class="name"><div class="surname">Koiwa</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Cushman</div> <div class="given-names">M.A.</div></div>, et al. </div><div class="article-title">Genes that are uniquely stress regulated in salt overly sensitive (SOS) mutants</div>, <div class="source">Plant Physiol.</div> , <div class="year">2001</div>, vol. <div class="volume">126</div> (pg. <div class="fpage">363</div>-<div class="lpage">75</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.126.1.363" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.126.1.363</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genes%20that%20are%20uniquely%20stress%20regulated%20in%20salt%20overly%20sensitive%20(SOS)%20mutants&amp;author=Z.Z.%20Gong&amp;author=H.%20Koiwa&amp;author=M.A.%20Cushman&amp;publication_year=2001&amp;journal=Plant%20Physiol.&amp;volume=126&amp;pages=363-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.126.1.363" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.126.1.363" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.126.1.363"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11351099" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C11" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C11" class="ref-content "><div class="label -label">11</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wu</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">X.L.</div></div>,  <div class="name"><div class="surname">Ren</div> <div class="given-names">H.Y.</div></div>, et al. </div><div class="article-title">ERF protein JERF1 that transcriptionally modulates the expression of abscisic acid biosynthesis-related gene enhances the tolerance under salinity and cold in tobacco</div>, <div class="source">Planta</div> , <div class="year">2007</div>, vol. <div class="volume">226</div> (pg. <div class="fpage">815</div>-<div class="lpage">25</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00425-007-0528-9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00425-007-0528-9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=ERF%20protein%20JERF1%20that%20transcriptionally%20modulates%20the%20expression%20of%20abscisic%20acid%20biosynthesis-related%20gene%20enhances%20the%20tolerance%20under%20salinity%20and%20cold%20in%20tobacco&amp;author=L.J.%20Wu&amp;author=X.L.%20Chen&amp;author=H.Y.%20Ren&amp;publication_year=2007&amp;journal=Planta&amp;volume=226&amp;pages=815-25" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00425-007-0528-9" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00425-007-0528-9" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00425-007-0528-9"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/17479283" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C12" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C12" class="ref-content "><div class="label -label">12</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Cao</div> <div class="given-names">Y.R.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">S.Y.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">J.S.</div></div>. </div><div class="article-title">Ethylene signaling regulates salt stress response: an overview</div>, <div class="source">Plant Signal. Behav.</div> , <div class="year">2008</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">761</div>-<div class="lpage">3</div>)<div class="pub-id-doi"><a href="https://doi.org/10.4161/psb.3.10.5934" class="link link-pub-id-doi" target="_blank">https://doi.org/10.4161/psb.3.10.5934</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Ethylene%20signaling%20regulates%20salt%20stress%20response%3A%20an%20overview&amp;author=Y.R.%20Cao&amp;author=S.Y.%20Chen&amp;author=J.S.%20Zhang&amp;publication_year=2008&amp;journal=Plant%20Signal.%20Behav.&amp;volume=3&amp;pages=761-3" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.4161/psb.3.10.5934" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.4161%2Fpsb.3.10.5934" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.4161%2Fpsb.3.10.5934"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19513226" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C13" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C13" class="ref-content "><div class="label -label">13</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Romeis</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Ludwig</div> <div class="given-names">A.A.</div></div>,  <div class="name"><div class="surname">Martin</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Jones</div> <div class="given-names">J.D.G.</div></div>. </div><div class="article-title">Calcium-dependent protein kinases play an essential role in a plant defence response</div>, <div class="source">Embo J.</div> , <div class="year">2001</div>, vol. <div class="volume">20</div> (pg. <div class="fpage">5556</div>-<div class="lpage">67</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/emboj/20.20.5556" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/emboj/20.20.5556</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Calcium-dependent%20protein%20kinases%20play%20an%20essential%20role%20in%20a%20plant%20defence%20response&amp;author=T.%20Romeis&amp;author=A.A.%20Ludwig&amp;author=R.%20Martin&amp;author=J.D.G.%20Jones&amp;publication_year=2001&amp;journal=Embo%20J.&amp;volume=20&amp;pages=5556-67" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/emboj/20.20.5556" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Femboj%2F20.20.5556" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Femboj%2F20.20.5556"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11597999" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C14" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C14" class="ref-content "><div class="label -label">14</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Inan</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">Q.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">P.</div></div>, et al. </div><div class="article-title">Salt cress. A halophyte and cryophyte <em>Arabidopsis</em> relative model system and its applicability to molecular genetic analyses of growth and development of extremophiles</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">135</div> (pg. <div class="fpage">1718</div>-<div class="lpage">37</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.104.041723" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.104.041723</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Salt%20cress.%20A%20halophyte%20and%20cryophyte%20Arabidopsis%20relative%20model%20system%20and%20its%20applicability%20to%20molecular%20genetic%20analyses%20of%20growth%20and%20development%20of%20extremophiles&amp;author=G.%20Inan&amp;author=Q.%20Zhang&amp;author=P.%20Li&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=135&amp;pages=1718-37" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.041723" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.041723" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.041723"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15247369" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C15" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C15" class="ref-content "><div class="label -label">15</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Taji</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Seki</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Satou</div> <div class="given-names">M.</div></div>, et al. </div><div class="article-title">Comparative genomics in salt tolerance between <em>Arabidopsis</em> and <em>Arabidopsis</em>-related halophyte salt cress using <em>Arabidopsis</em> microarray</div>, <div class="source">Plant Physiol.</div> , <div class="year">2004</div>, vol. <div class="volume">135</div> (pg. <div class="fpage">1697</div>-<div class="lpage">709</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.104.039909" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.104.039909</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20genomics%20in%20salt%20tolerance%20between%20Arabidopsis%20and%20Arabidopsis-related%20halophyte%20salt%20cress%20using%20Arabidopsis%20microarray&amp;author=T.%20Taji&amp;author=M.%20Seki&amp;author=M.%20Satou&amp;publication_year=2004&amp;journal=Plant%20Physiol.&amp;volume=135&amp;pages=1697-709" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.104.039909" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.104.039909" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.104.039909"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15247402" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C16" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C16" class="ref-content "><div class="label -label">16</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Gong</div> <div class="given-names">Q.Q.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">P.H.</div></div>,  <div class="name"><div class="surname">Ma</div> <div class="given-names">S.S.</div></div>,  <div class="name"><div class="surname">Rupassara</div> <div class="given-names">S.I.</div></div>,  <div class="name"><div class="surname">Bohnert</div> <div class="given-names">H.J.</div></div>. </div><div class="article-title">Salinity stress adaptation competence in the extremophile <em>Thellungiella halophila</em> in comparison with its relative <em>Arabidopsis thaliana</em></div>, <div class="source">Plant J.</div> , <div class="year">2005</div>, vol. <div class="volume">44</div> (pg. <div class="fpage">826</div>-<div class="lpage">39</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1365-313X.2005.02587.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1365-313X.2005.02587.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Salinity%20stress%20adaptation%20competence%20in%20the%20extremophile%20Thellungiella%20halophila%20in%20comparison%20with%20its%20relative%20Arabidopsis%20thaliana&amp;author=Q.Q.%20Gong&amp;author=P.H.%20Li&amp;author=S.S.%20Ma&amp;author=S.I.%20Rupassara&amp;author=H.J.%20Bohnert&amp;publication_year=2005&amp;journal=Plant%20J.&amp;volume=44&amp;pages=826-39" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/tpj.2005.44.issue-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Ftpj.2005.44.issue-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Ftpj.2005.44.issue-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16297073" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C17" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C17" class="ref-content "><div class="label -label">17</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Zhu</div> <div class="given-names">J.K.</div></div>. </div><div class="article-title">Plant salt tolerance</div>, <div class="source">Trends Plant Sci.</div> , <div class="year">2001</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">66</div>-<div class="lpage">71</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/S1360-1385(00)01838-0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/S1360-1385(00)01838-0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Plant%20salt%20tolerance&amp;author=J.K.%20Zhu&amp;publication_year=2001&amp;journal=Trends%20Plant%20Sci.&amp;volume=6&amp;pages=66-71" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/S1360-1385(00)01838-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2FS1360-1385(00)01838-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2FS1360-1385(00)01838-0"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/11173290" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C18" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C18" class="ref-content "><div class="label -label">18</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">L.M.</div></div>,  <div class="name"><div class="surname">Mu</div> <div class="given-names">M.R.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">X.F.</div></div>,  <div class="name"><div class="surname">Lin</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">W.Q.</div></div>. </div><div class="article-title">Differentiation between true mangroves and mangrove associates based on leaf traits and salt contents</div>, <div class="source">J. Plant Ecol.</div> , <div class="year">2010</div><div class="pub-id-doi"><a href="https://doi.org/10.1093/jpe/rtq1008" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/jpe/rtq1008</a></div></div></div></div></div><div content-id="DSS004C19" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C19" class="ref-content "><div class="label -label">19</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singh</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Jha</div> <div class="given-names">M.N.</div></div>. </div><div class="article-title">Trees, shrubs and grasses on saline soils of Indo-Gangetic plains</div>, <div class="source">Indian Forester</div> , <div class="year">1993</div>, vol. <div class="volume">119</div> (pg. <div class="fpage">630</div>-<div class="lpage">47</div>)</div></div></div></div><div content-id="DSS004C20" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C20" class="ref-content "><div class="label -label">20</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Singh</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Yadav</div> <div class="given-names">J.</div></div>. </div><div class="article-title">Effect of soil salinity and sodicity on seedling growth and mineral compsition of <em>Pongamia pinnata</em></div>, <div class="source">Indian Forester</div> , <div class="year">1999</div>, vol. <div class="volume">125</div> (pg. <div class="fpage">618</div>-<div class="lpage">22</div>)</div></div></div></div><div content-id="DSS004C21" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C21" class="ref-content "><div class="label -label">21</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Karmee</div> <div class="given-names">S.K.</div></div>,  <div class="name"><div class="surname">Chadha</div> <div class="given-names">A.</div></div>. </div><div class="article-title">Preparation of biodiesel from crude oil of <em>Pongamia pinnata</em></div>, <div class="source">Bioresour. Technol.</div> , <div class="year">2005</div>, vol. <div class="volume">96</div> (pg. <div class="fpage">1425</div>-<div class="lpage">9</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.biortech.2004.12.011" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.biortech.2004.12.011</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Preparation%20of%20biodiesel%20from%20crude%20oil%20of%20Pongamia%20pinnata&amp;author=S.K.%20Karmee&amp;author=A.%20Chadha&amp;publication_year=2005&amp;journal=Bioresour.%20Technol.&amp;volume=96&amp;pages=1425-9" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.biortech.2004.12.011" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.biortech.2004.12.011" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.biortech.2004.12.011"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15939268" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C22" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C22" class="ref-content "><div class="label -label">22</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Sharma</div> <div class="given-names">Y.C.</div></div>,  <div class="name"><div class="surname">Singh</div> <div class="given-names">B.</div></div>,  <div class="name"><div class="surname">Korstad</div> <div class="given-names">J.</div></div>. </div><div class="article-title">High yield and conversion of biodiesel from a nonedible feedstock (<em>Pongamia pinnata</em>)</div>, <div class="source">J. Agr. Food Chem.</div> , <div class="year">2010</div>, vol. <div class="volume">58</div> (pg. <div class="fpage">242</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1021/jf903227e" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1021/jf903227e</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=High%20yield%20and%20conversion%20of%20biodiesel%20from%20a%20nonedible%20feedstock%20(Pongamia%20pinnata)&amp;author=Y.C.%20Sharma&amp;author=B.%20Singh&amp;author=J.%20Korstad&amp;publication_year=2010&amp;journal=J.%20Agr.%20Food%20Chem.&amp;volume=58&amp;pages=242-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1021/jf903227e" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1021%2Fjf903227e" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1021%2Fjf903227e"> </span></div> </div></div></div></div></div><div content-id="DSS004C23" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C23" class="ref-content "><div class="label -label">23</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Scott</div> <div class="given-names">P.T.</div></div>,  <div class="name"><div class="surname">Pregelj</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Chen</div> <div class="given-names">N.</div></div>,  <div class="name"><div class="surname">Hadler</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Djordjevic</div> <div class="given-names">M.A.</div></div>,  <div class="name"><div class="surname">Gresshoff</div> <div class="given-names">P.M.</div></div>. </div><div class="article-title"><em>Pongamia pinnata</em>: an untapped resource for the biofuels industry of the future</div>, <div class="source">Bioenergy Res.</div> , <div class="year">2008</div>, vol. <div class="volume">1</div> (pg. <div class="fpage">2</div>-<div class="lpage">11</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s12155-008-9003-0" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s12155-008-9003-0</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Pongamia%20pinnata%3A%20an%20untapped%20resource%20for%20the%20biofuels%20industry%20of%20the%20future&amp;author=P.T.%20Scott&amp;author=L.%20Pregelj&amp;author=N.%20Chen&amp;author=J.S.%20Hadler&amp;author=M.A.%20Djordjevic&amp;author=P.M.%20Gresshoff&amp;publication_year=2008&amp;journal=Bioenergy%20Res.&amp;volume=1&amp;pages=2-11" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s12155-008-9003-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs12155-008-9003-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs12155-008-9003-0"> </span></div> </div></div></div></div></div><div content-id="DSS004C24" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C24" class="ref-content "><div class="label -label">24</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Essa</div> <div class="given-names">M.M.</div></div>,  <div class="name"><div class="surname">Subramanian</div> <div class="given-names">P.</div></div>,  <div class="name"><div class="surname">Suthakar</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Manivasagam</div> <div class="given-names">T.</div></div>,  <div class="name"><div class="surname">Dakshayani</div> <div class="given-names">K.B.</div></div>. </div><div class="article-title">Protective influence of <em>Pongamia pinnata</em> (<em>Karanja</em>) on blood ammonia and urea levels in ammonium chloride-induced hyperammonemia: antihyperammonemic effect of the leaf extract</div>, <div class="source">J. Appl. Biomed.</div> , <div class="year">2005</div>, vol. <div class="volume">3</div> (pg. <div class="fpage">133</div>-<div class="lpage">8</div>)</div></div></div></div><div content-id="DSS004C25" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C25" class="ref-content "><div class="label -label">25</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Badole</div> <div class="given-names">S.L.</div></div>,  <div class="name"><div class="surname">Bodhankar</div> <div class="given-names">S.L.</div></div>. </div><div class="article-title">Investigation of antihyperglycaemic activity of aqueous and petroleum ether extract of stem bark of <em>Pongamia pinnata</em> on serum glucose level in diabetic mice</div>, <div class="source">J. Ethnopharmacol.</div> , <div class="year">2009</div>, vol. <div class="volume">123</div> (pg. <div class="fpage">115</div>-<div class="lpage">20</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.jep.2009.02.018" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.jep.2009.02.018</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Investigation%20of%20antihyperglycaemic%20activity%20of%20aqueous%20and%20petroleum%20ether%20extract%20of%20stem%20bark%20of%20Pongamia%20pinnata%20on%20serum%20glucose%20level%20in%20diabetic%20mice&amp;author=S.L.%20Badole&amp;author=S.L.%20Bodhankar&amp;publication_year=2009&amp;journal=J.%20Ethnopharmacol.&amp;volume=123&amp;pages=115-20" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.jep.2009.02.018" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.jep.2009.02.018" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.jep.2009.02.018"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19429349" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C26" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C26" class="ref-content "><div class="label -label">26</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Lavin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Eshbaugh</div> <div class="given-names">E.</div></div>,  <div class="name"><div class="surname">Hu</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Mathews</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Sharrock</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Monophyletic subgroups of the tribe Millettieae (Leguminosae) as revealed by phytochrome nucleotide sequence data</div>, <div class="source">Am. J. Bot.</div> , <div class="year">1998</div>, vol. <div class="volume">85</div> (pg. <div class="fpage">412</div>-<div class="lpage">33</div>)<div class="pub-id-doi"><a href="https://doi.org/10.2307/2446334" class="link link-pub-id-doi" target="_blank">https://doi.org/10.2307/2446334</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Monophyletic%20subgroups%20of%20the%20tribe%20Millettieae%20(Leguminosae)%20as%20revealed%20by%20phytochrome%20nucleotide%20sequence%20data&amp;author=M.%20Lavin&amp;author=E.%20Eshbaugh&amp;author=J.M.%20Hu&amp;author=S.%20Mathews&amp;author=R.A.%20Sharrock&amp;publication_year=1998&amp;journal=Am.%20J.%20Bot.&amp;volume=85&amp;pages=412-33" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.2307/2446334" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.2307%2F2446334" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.2307%2F2446334"> </span></div> </div></div></div></div></div><div content-id="DSS004C27" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C27" class="ref-content "><div class="label -label">27</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Hu</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Lavin</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Wojciechowski</div> <div class="given-names">M.F.</div></div>,  <div class="name"><div class="surname">Sanderson</div> <div class="given-names">M.J.</div></div>. </div><div class="article-title">Phylogenetic analysis of nuclear ribosomal ITS/5.8S sequences in the Tribe Millettieae (Fabaceae): <em>Poecilanthe-Cyclolobium</em>, the core Millettieae, and the Callerya group</div>, <div class="source">Syst. Bot.</div> , <div class="year">2002</div>, vol. <div class="volume">27</div> (pg. <div class="fpage">722</div>-<div class="lpage">33</div>)</div></div></div></div><div content-id="DSS004C28" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C28" class="ref-content "><div class="label -label">28</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shi</div> <div class="given-names">S.H.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">Y.L.</div></div>,  <div class="name"><div class="surname">Zeng</div> <div class="given-names">K.</div></div>, et al. </div><div class="article-title">Molecular phylogenetic analysis of mangroves: independent evolutionary origins of vivipary and salt secretion</div>, <div class="source">Mol. Phylogenet. Evol.</div> , <div class="year">2005</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">159</div>-<div class="lpage">66</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.ympev.2004.09.002" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.ympev.2004.09.002</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Molecular%20phylogenetic%20analysis%20of%20mangroves%3A%20independent%20evolutionary%20origins%20of%20vivipary%20and%20salt%20secretion&amp;author=S.H.%20Shi&amp;author=Y.L.%20Huang&amp;author=K.%20Zeng&amp;publication_year=2005&amp;journal=Mol.%20Phylogenet.%20Evol.&amp;volume=34&amp;pages=159-66" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.ympev.2004.09.002" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.ympev.2004.09.002" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.ympev.2004.09.002"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15579389" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C29" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C29" class="ref-content "><div class="label -label">29</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Bennett</div> <div class="given-names">S.T.</div></div>,  <div class="name"><div class="surname">Barnes</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Cox</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Davies</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Brown</div> <div class="given-names">C.</div></div>. </div><div class="article-title">Toward the 1,000 dollars human genome</div>, <div class="source">Pharmacogenomics.</div> , <div class="year">2005</div>, vol. <div class="volume">6</div> (pg. <div class="fpage">373</div>-<div class="lpage">82</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1517/14622416.6.4.373" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1517/14622416.6.4.373</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Toward%20the%201%2C000%20dollars%20human%20genome&amp;author=S.T.%20Bennett&amp;author=C.%20Barnes&amp;author=A.%20Cox&amp;author=L.%20Davies&amp;author=C.%20Brown&amp;publication_year=2005&amp;journal=Pharmacogenomics.&amp;volume=6&amp;pages=373-82" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1517/14622416.6.4.373" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1517%2F14622416.6.4.373" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1517%2F14622416.6.4.373"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16004555" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C30" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C30" class="ref-content "><div class="label -label">30</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Margulies</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Egholm</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Altman</div> <div class="given-names">W.E.</div></div>, et al. </div><div class="article-title">Genome sequencing in microfabricated high-density picolitre reactors</div>, <div class="source">Nature</div> , <div class="year">2005</div>, vol. <div class="volume">437</div> (pg. <div class="fpage">376</div>-<div class="lpage">80</div>)<div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Genome%20sequencing%20in%20microfabricated%20high-density%20picolitre%20reactors&amp;author=M.%20Margulies&amp;author=M.%20Egholm&amp;author=W.E.%20Altman&amp;publication_year=2005&amp;journal=Nature&amp;volume=437&amp;pages=376-80" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16056220" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C31" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C31" class="ref-content "><div class="label -label">31</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Collins</div> <div class="given-names">L.J.</div></div>,  <div class="name"><div class="surname">Biggs</div> <div class="given-names">P.J.</div></div>,  <div class="name"><div class="surname">Voelckel</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Joly</div> <div class="given-names">S.</div></div>. </div><div class="article-title">An approach to transcriptome analysis of non-model organisms using short-read sequences</div>, <div class="source">Genome Inform.</div> , <div class="year">2008</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">3</div>-<div class="lpage">14</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1142/9781848163324_0001" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1142/9781848163324_0001</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20approach%20to%20transcriptome%20analysis%20of%20non-model%20organisms%20using%20short-read%20sequences&amp;author=L.J.%20Collins&amp;author=P.J.%20Biggs&amp;author=C.%20Voelckel&amp;author=S.%20Joly&amp;publication_year=2008&amp;journal=Genome%20Inform.&amp;volume=21&amp;pages=3-14" target="_blank">Google Scholar</a></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19425143" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C32" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C32" class="ref-content "><div class="label -label">32</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">R.Q.</div></div>,  <div class="name"><div class="surname">Zhu</div> <div class="given-names">H.M.</div></div>,  <div class="name"><div class="surname">Ruan</div> <div class="given-names">J.</div></div>, et al. </div><div class="article-title">De novo assembly of human genomes with massively parallel short read sequencing</div>, <div class="source">Genome Res</div> , <div class="year">2009</div> <div class="comment">doi/10.1101/gr.097261.109</div></div></div></div></div><div content-id="DSS004C33" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C33" class="ref-content "><div class="label -label">33</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Pertea</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Huang</div> <div class="given-names">X.Q.</div></div>,  <div class="name"><div class="surname">Liang</div> <div class="given-names">F.</div></div>, et al. </div><div class="article-title">TIGR Gene Indices clustering tools (TGICL): a software system for fast clustering of large EST datasets</div>, <div class="source">Bioinformatics</div> , <div class="year">2003</div>, vol. <div class="volume">19</div> (pg. <div class="fpage">651</div>-<div class="lpage">2</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btg034" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btg034</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=TIGR%20Gene%20Indices%20clustering%20tools%20(TGICL)%3A%20a%20software%20system%20for%20fast%20clustering%20of%20large%20EST%20datasets&amp;author=G.%20Pertea&amp;author=X.Q.%20Huang&amp;author=F.%20Liang&amp;publication_year=2003&amp;journal=Bioinformatics&amp;volume=19&amp;pages=651-2" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btg034" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtg034" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtg034"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/12651724" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C34" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C34" class="ref-content "><div class="label -label">34</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Iseli</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Jongeneel</div> <div class="given-names">C.V.</div></div>,  <div class="name"><div class="surname">Bucher</div> <div class="given-names">P.</div></div>. </div><div class="article-title">ESTScan: a program for detecting, evaluating, and reconstructing potential coding regions in EST sequences</div>, <div class="source">Proc. Int. Conf. Intell. Syst. Mol. Biol.</div> , <div class="year">1999</div>(pg. <div class="fpage">138</div>-<div class="lpage">48</div>)</div></div></div></div><div content-id="DSS004C35" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C35" class="ref-content "><div class="label -label">35</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Conesa</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Gotz</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Garcia-Gomez</div> <div class="given-names">J.M.</div></div>,  <div class="name"><div class="surname">Terol</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Talon</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Robles</div> <div class="given-names">M.</div></div>. </div><div class="article-title">Blast2GO: a universal tool for annotation, visualization and analysis in functional genomics research</div>, <div class="source">Bioinformatics</div> , <div class="year">2005</div>, vol. <div class="volume">21</div> (pg. <div class="fpage">3674</div>-<div class="lpage">6</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/bti610" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/bti610</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Blast2GO%3A%20a%20universal%20tool%20for%20annotation%2C%20visualization%20and%20analysis%20in%20functional%20genomics%20research&amp;author=A.%20Conesa&amp;author=S.%20Gotz&amp;author=J.M.%20Garcia-Gomez&amp;author=J.%20Terol&amp;author=M.%20Talon&amp;author=M.%20Robles&amp;publication_year=2005&amp;journal=Bioinformatics&amp;volume=21&amp;pages=3674-6" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/bti610" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbti610" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbti610"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16081474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C36" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C36" class="ref-content "><div class="label -label">36</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Ye</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Fang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Zheng</div> <div class="given-names">H.</div></div>, et al. </div><div class="article-title">WEGO: a web tool for plotting GO annotations</div>, <div class="source">Nucleic Acids Res.</div> , <div class="year">2006</div>, vol. <div class="volume">34</div> (pg. <div class="fpage">W293</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/nar/gkl031" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/nar/gkl031</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=WEGO%3A%20a%20web%20tool%20for%20plotting%20GO%20annotations&amp;author=J.%20Ye&amp;author=L.%20Fang&amp;author=H.%20Zheng&amp;publication_year=2006&amp;journal=Nucleic%20Acids%20Res.&amp;volume=34&amp;pages=W293-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/nar/gkl031" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fnar%2Fgkl031" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fnar%2Fgkl031"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/16845012" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C37" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C37" class="ref-content "><div class="label -label">37</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mortazavi</div> <div class="given-names">A.</div></div>,  <div class="name"><div class="surname">Williams</div> <div class="given-names">B. A.</div></div>,  <div class="name"><div class="surname">McCue</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Schaeffer</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Wold</div> <div class="given-names">B.</div></div>. </div><div class="article-title">Mapping and quantifying mammalian transcriptomes by RNA-Seq</div>, <div class="source">Nat. Methods</div> , <div class="year">2008</div>, vol. <div class="volume">5</div> (pg. <div class="fpage">621</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1038/nmeth.1226" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1038/nmeth.1226</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Mapping%20and%20quantifying%20mammalian%20transcriptomes%20by%20RNA-Seq&amp;author=A.%20Mortazavi&amp;author=B.%20A.%20Williams&amp;author=K.%20McCue&amp;author=L.%20Schaeffer&amp;author=B.%20Wold&amp;publication_year=2008&amp;journal=Nat.%20Methods&amp;volume=5&amp;pages=621-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1038/nmeth.1226" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1038%2Fnmeth.1226" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1038%2Fnmeth.1226"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18516045" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C38" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C38" class="ref-content "><div class="label -label">38</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Yu</div> <div class="given-names">C.</div></div>,  <div class="name"><div class="surname">Li</div> <div class="given-names">Y.</div></div>, et al. </div><div class="article-title">SOAP2: an improved ultrafast tool for short read alignment</div>, <div class="source">Bioinformatics</div> , <div class="year">2009</div>, vol. <div class="volume">25</div> (pg. <div class="fpage">1966</div>-<div class="lpage">7</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btp336" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btp336</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=SOAP2%3A%20an%20improved%20ultrafast%20tool%20for%20short%20read%20alignment&amp;author=R.%20Li&amp;author=C.%20Yu&amp;author=Y.%20Li&amp;publication_year=2009&amp;journal=Bioinformatics&amp;volume=25&amp;pages=1966-7" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp336" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp336" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp336"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19497933" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C39" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C39" class="ref-content "><div class="label -label">39</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Wang</div> <div class="given-names">L.</div></div>,  <div class="name"><div class="surname">Feng</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">X.</div></div>,  <div class="name"><div class="surname">Zhang</div> <div class="given-names">X.</div></div>. </div><div class="article-title">DEGseq: an R package for identifying differentially expressed genes from RNA-seq data</div>, <div class="source">Bioinformatics</div> , <div class="year">2010</div>, vol. <div class="volume">26</div> (pg. <div class="fpage">136</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/bioinformatics/btp612" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/bioinformatics/btp612</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=DEGseq%3A%20an%20R%20package%20for%20identifying%20differentially%20expressed%20genes%20from%20RNA-seq%20data&amp;author=L.%20Wang&amp;author=Z.%20Feng&amp;author=X.%20Wang&amp;author=X.%20Zhang&amp;publication_year=2010&amp;journal=Bioinformatics&amp;volume=26&amp;pages=136-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/bioinformatics/btp612" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fbioinformatics%2Fbtp612" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fbioinformatics%2Fbtp612"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19855105" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C40" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C40" class="ref-content "><div class="label -label">40</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tyler</div> <div class="given-names">B.M.</div></div>. </div><div class="article-title"><em>Phytophthora sojae</em>: root rot pathogen of soybean and model oomycete</div>, <div class="source">Mol. Plant Pathol.</div> , <div class="year">2007</div>, vol. <div class="volume">8</div> (pg. <div class="fpage">1</div>-<div class="lpage">8</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1364-3703.2006.00373.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1364-3703.2006.00373.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Phytophthora%20sojae%3A%20root%20rot%20pathogen%20of%20soybean%20and%20model%20oomycete&amp;author=B.M.%20Tyler&amp;publication_year=2007&amp;journal=Mol.%20Plant%20Pathol.&amp;volume=8&amp;pages=1-8" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/mpp.2007.8.issue-1" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fmpp.2007.8.issue-1" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fmpp.2007.8.issue-1"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20507474" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C41" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C41" class="ref-content "><div class="label -label">41</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miyama</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hanagata</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Microarray analysis of 7029 gene expression patterns in burma mangrove under high-salinity stress</div>, <div class="source">Plant Sci.</div> , <div class="year">2007</div>, vol. <div class="volume">172</div> (pg. <div class="fpage">948</div>-<div class="lpage">57</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.plantsci.2007.01.004" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.plantsci.2007.01.004</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Microarray%20analysis%20of%207029%20gene%20expression%20patterns%20in%20burma%20mangrove%20under%20high-salinity%20stress&amp;author=M.%20Miyama&amp;author=N.%20Hanagata&amp;publication_year=2007&amp;journal=Plant%20Sci.&amp;volume=172&amp;pages=948-57" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plantsci.2007.01.004" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2007.01.004" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plantsci.2007.01.004"> </span></div> </div></div></div></div></div><div content-id="DSS004C42" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C42" class="ref-content "><div class="label -label">42</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miyama</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Tada</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">Transcriptional and physiological study of the response of Burma mangrove (<em>Bruguiera gymnorhiza</em>) to salt and osmotic stress</div>, <div class="source">Plant Mol. Biol.</div> , <div class="year">2008</div>, vol. <div class="volume">68</div> (pg. <div class="fpage">119</div>-<div class="lpage">29</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11103-008-9356-y" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11103-008-9356-y</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptional%20and%20physiological%20study%20of%20the%20response%20of%20Burma%20mangrove%20(Bruguiera%20gymnorhiza)%20to%20salt%20and%20osmotic%20stress&amp;author=M.%20Miyama&amp;author=Y.%20Tada&amp;publication_year=2008&amp;journal=Plant%20Mol.%20Biol.&amp;volume=68&amp;pages=119-29" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11103-008-9356-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11103-008-9356-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11103-008-9356-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18568404" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C43" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C43" class="ref-content "><div class="label -label">43</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Brinker</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Brosche</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Vinocur</div> <div class="given-names">B.</div></div>, et al. </div><div class="article-title">Linking the salt transcriptome with physiological responses of a salt-resistant <em>Populus</em> species as a strategy to identify genes important for stress acclimation</div>, <div class="source">Plant Physiol.</div> , <div class="year">2010</div>, vol. <div class="volume">154</div> (pg. <div class="fpage">1697</div>-<div class="lpage">709</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1104/pp.110.164152" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1104/pp.110.164152</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Linking%20the%20salt%20transcriptome%20with%20physiological%20responses%20of%20a%20salt-resistant%20Populus%20species%20as%20a%20strategy%20to%20identify%20genes%20important%20for%20stress%20acclimation&amp;author=M.%20Brinker&amp;author=M.%20Brosche&amp;author=B.%20Vinocur&amp;publication_year=2010&amp;journal=Plant%20Physiol.&amp;volume=154&amp;pages=1697-709" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1104/pp.110.164152" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1104%2Fpp.110.164152" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1104%2Fpp.110.164152"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20959419" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C44" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C44" class="ref-content "><div class="label -label">44</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Tomlinson</div> <div class="given-names">P.B.</div></div>. </div>, <div class="source">The Botany of Mangroves</div> , <div class="year">1986</div><div class="publisher-loc">Cambridge</div><div class="publisher-name">Cambridge University Press</div></div></div></div></div><div content-id="DSS004C45" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C45" class="ref-content "><div class="label -label">45</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Mehta</div> <div class="given-names">P.A.</div></div>,  <div class="name"><div class="surname">Sivaprakash</div> <div class="given-names">K.</div></div>,  <div class="name"><div class="surname">Parani</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Venkataraman</div> <div class="given-names">G.</div></div>,  <div class="name"><div class="surname">Parida</div> <div class="given-names">A.K.</div></div>. </div><div class="article-title">Generation and analysis of expressed sequence tags from the salt-tolerant mangrove species <em>Avicennia marina</em> (Forsk) Vierh</div>, <div class="source">Theor. Appl. Genet.</div> , <div class="year">2005</div>, vol. <div class="volume">110</div> (pg. <div class="fpage">416</div>-<div class="lpage">24</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s00122-004-1801-y" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s00122-004-1801-y</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20and%20analysis%20of%20expressed%20sequence%20tags%20from%20the%20salt-tolerant%20mangrove%20species%20Avicennia%20marina%20(Forsk)%20Vierh&amp;author=P.A.%20Mehta&amp;author=K.%20Sivaprakash&amp;author=M.%20Parani&amp;author=G.%20Venkataraman&amp;author=A.K.%20Parida&amp;publication_year=2005&amp;journal=Theor.%20Appl.%20Genet.&amp;volume=110&amp;pages=416-24" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s00122-004-1801-y" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs00122-004-1801-y" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs00122-004-1801-y"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/15609053" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C46" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C46" class="ref-content "><div class="label -label">46</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Miyama</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Shimizu</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Sugiyama</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Hanagata</div> <div class="given-names">N.</div></div>. </div><div class="article-title">Sequencing and analysis of 14,842 expressed sequence tags of burma mangrove, <em>Bruguiera gymnorrhiza</em></div>, <div class="source">, <em>Plant Sci</em>.</div> , <div class="year">2006</div>, vol. <div class="volume">171</div> (pg. <div class="fpage">234</div>-<div class="lpage">41</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1016/j.plantsci.2006.03.015" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1016/j.plantsci.2006.03.015</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Sequencing%20and%20analysis%20of%2014%2C842%20expressed%20sequence%20tags%20of%20burma%20mangrove%2C%20Bruguiera%20gymnorrhiza&amp;author=M.%20Miyama&amp;author=H.%20Shimizu&amp;author=M.%20Sugiyama&amp;author=N.%20Hanagata&amp;publication_year=2006&amp;journal=%2C%20Plant%20Sci.&amp;volume=171&amp;pages=234-41" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1016/j.plantsci.2006.03.015" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1016%2Fj.plantsci.2006.03.015" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1016%2Fj.plantsci.2006.03.015"> </span></div> </div></div></div></div></div><div content-id="DSS004C47" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C47" class="ref-content "><div class="label -label">47</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Nguyen</div> <div class="given-names">P.D.</div></div>,  <div class="name"><div class="surname">Ho</div> <div class="given-names">C.L.</div></div>,  <div class="name"><div class="surname">Harikrishna</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Wong</div> <div class="given-names">M.C.V.L.</div></div>,  <div class="name"><div class="surname">Rahim</div> <div class="given-names">R.A.</div></div>. </div><div class="article-title">Generation and analysis of expressed sequence tags from the mangrove plant, <em>Acanthus ebracteatus</em> Vahl</div>, <div class="source">Tree Genet. Genomes</div> , <div class="year">2006</div>, vol. <div class="volume">2</div> (pg. <div class="fpage">196</div>-<div class="lpage">201</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11295-006-0044-2" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11295-006-0044-2</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Generation%20and%20analysis%20of%20expressed%20sequence%20tags%20from%20the%20mangrove%20plant%2C%20Acanthus%20ebracteatus%20Vahl&amp;author=P.D.%20Nguyen&amp;author=C.L.%20Ho&amp;author=J.A.%20Harikrishna&amp;author=M.C.V.L.%20Wong&amp;author=R.A.%20Rahim&amp;publication_year=2006&amp;journal=Tree%20Genet.%20Genomes&amp;volume=2&amp;pages=196-201" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11295-006-0044-2" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11295-006-0044-2" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11295-006-0044-2"> </span></div> </div></div></div></div></div><div content-id="DSS004C48" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C48" class="ref-content "><div class="label -label">48</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dassanayake</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Haas</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Bohnert</div> <div class="given-names">H.J.</div></div>,  <div class="name"><div class="surname">Cheeseman</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Shedding light on an extremophile lifestyle through transcriptomics</div>, <div class="source">New Phytol.</div> , <div class="year">2009</div>, vol. <div class="volume">183</div> (pg. <div class="fpage">764</div>-<div class="lpage">75</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1111/j.1469-8137.2009.02913.x" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1111/j.1469-8137.2009.02913.x</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Shedding%20light%20on%20an%20extremophile%20lifestyle%20through%20transcriptomics&amp;author=M.%20Dassanayake&amp;author=J.S.%20Haas&amp;author=H.J.%20Bohnert&amp;author=J.M.%20Cheeseman&amp;publication_year=2009&amp;journal=New%20Phytol.&amp;volume=183&amp;pages=764-75" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1111/nph.2009.183.issue-3" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1111%2Fnph.2009.183.issue-3" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1111%2Fnph.2009.183.issue-3"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19549131" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C49" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C49" class="ref-content "><div class="label -label">49</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Garg</div> <div class="given-names">R.</div></div>,  <div class="name"><div class="surname">Patel</div> <div class="given-names">R.K.</div></div>,  <div class="name"><div class="surname">Tyagi</div> <div class="given-names">A.K.</div></div>,  <div class="name"><div class="surname">Jain</div> <div class="given-names">M.</div></div>. </div><div class="article-title">De novo assembly of chickpea transcriptome using short reads for gene discovery and marker identification</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">53</div>-<div class="lpage">63</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsq028" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsq028</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=De%20novo%20assembly%20of%20chickpea%20transcriptome%20using%20short%20reads%20for%20gene%20discovery%20and%20marker%20identification&amp;author=R.%20Garg&amp;author=R.K.%20Patel&amp;author=A.K.%20Tyagi&amp;author=M.%20Jain&amp;publication_year=2011&amp;journal=DNA%20Res.&amp;volume=18&amp;pages=53-63" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1093/dnares/dsq028" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1093%2Fdnares%2Fdsq028" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1093%2Fdnares%2Fdsq028"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21217129" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C50" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C50" class="ref-content "><div class="label -label">50</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Shi</div> <div class="given-names">C.Y.</div></div>,  <div class="name"><div class="surname">Yang</div> <div class="given-names">H.</div></div>,  <div class="name"><div class="surname">Wei</div> <div class="given-names">C.L.</div></div>, et al. </div><div class="article-title">Deep sequencing of the <em>Camellia sinensis</em> transcriptome revealed candidate genes for major metabolic pathways of tea-specific compounds</div>, <div class="source">BMC Genomics</div> , <div class="year">2011</div>, vol. <div class="volume">12</div> pg. <div class="fpage">131</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-12-131" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-12-131</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Deep%20sequencing%20of%20the%20Camellia%20sinensis%20transcriptome%20revealed%20candidate%20genes%20for%20major%20metabolic%20pathways%20of%20tea-specific%20compounds&amp;author=C.Y.%20Shi&amp;author=H.%20Yang&amp;author=C.L.%20Wei&amp;publication_year=2011&amp;journal=BMC%20Genomics&amp;volume=12&amp;pages=131" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-12-131" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-12-131" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-12-131"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/21356090" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C51" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C51" class="ref-content "><div class="label -label">51</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Edwards</div> <div class="given-names">C.E.</div></div>,  <div class="name"><div class="surname">Parchman</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Weekley</div> <div class="given-names">C.W.</div></div>. </div><div class="article-title">Assembly, gene annotation and marker development using 454 floral transcriptome sequences in <em>Ziziphus Celata</em> (Rhamnaceae), a highly endangered, Florida endemic plant</div>, <div class="source">DNA Res.</div> , <div class="year">2011</div><div class="pub-id-doi"><a href="https://doi.org/10.1093/dnares/dsr1037" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1093/dnares/dsr1037</a></div></div></div></div></div><div content-id="DSS004C52" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C52" class="ref-content "><div class="label -label">52</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Parchman</div> <div class="given-names">T.L.</div></div>,  <div class="name"><div class="surname">Geist</div> <div class="given-names">K.S.</div></div>,  <div class="name"><div class="surname">Grahnen</div> <div class="given-names">J.A.</div></div>,  <div class="name"><div class="surname">Benkman</div> <div class="given-names">C.W.</div></div>,  <div class="name"><div class="surname">Buerkle</div> <div class="given-names">C.A.</div></div>. </div><div class="article-title">Transcriptome sequencing in an ecologically important tree species: assembly, annotation, and marker discovery</div>, <div class="source">BMC Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">11</div> pg. <div class="fpage">180</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-11-180" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-11-180</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Transcriptome%20sequencing%20in%20an%20ecologically%20important%20tree%20species%3A%20assembly%2C%20annotation%2C%20and%20marker%20discovery&amp;author=T.L.%20Parchman&amp;author=K.S.%20Geist&amp;author=J.A.%20Grahnen&amp;author=C.W.%20Benkman&amp;author=C.A.%20Buerkle&amp;publication_year=2010&amp;journal=BMC%20Genomics&amp;volume=11&amp;pages=180" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-11-180" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-11-180" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-11-180"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20233449" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C53" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C53" class="ref-content "><div class="label -label">53</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Dassanayake</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Haas</div> <div class="given-names">J.S.</div></div>,  <div class="name"><div class="surname">Bohnert</div> <div class="given-names">H.J.</div></div>,  <div class="name"><div class="surname">Cheeseman</div> <div class="given-names">J.M.</div></div>. </div><div class="article-title">Comparative transcriptomics for mangrove species: an expanding resource</div>, <div class="source">Funct. Integr. Genomics</div> , <div class="year">2010</div>, vol. <div class="volume">10</div> (pg. <div class="fpage">523</div>-<div class="lpage">32</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s10142-009-0156-5" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s10142-009-0156-5</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Comparative%20transcriptomics%20for%20mangrove%20species%3A%20an%20expanding%20resource&amp;author=M.%20Dassanayake&amp;author=J.S.%20Haas&amp;author=H.J.%20Bohnert&amp;author=J.M.%20Cheeseman&amp;publication_year=2010&amp;journal=Funct.%20Integr.%20Genomics&amp;volume=10&amp;pages=523-32" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s10142-009-0156-5" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs10142-009-0156-5" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs10142-009-0156-5"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/20107865" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C54" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C54" class="ref-content "><div class="label -label">54</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Liang</div> <div class="given-names">S.</div></div>,  <div class="name"><div class="surname">Zhou</div> <div class="given-names">R.C.</div></div>,  <div class="name"><div class="surname">Dong</div> <div class="given-names">S.S.</div></div>,  <div class="name"><div class="surname">Shi</div> <div class="given-names">S.H.</div></div>. </div><div class="article-title">Adaptation to salinity in mangroves: Implication on the evolution of salt-tolerance</div>, <div class="source">Chinese Sci. Bull.</div> , <div class="year">2008</div>, vol. <div class="volume">53</div> (pg. <div class="fpage">1708</div>-<div class="lpage">15</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1007/s11434-008-0221-9" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1007/s11434-008-0221-9</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=Adaptation%20to%20salinity%20in%20mangroves%3A%20Implication%20on%20the%20evolution%20of%20salt-tolerance&amp;author=S.%20Liang&amp;author=R.C.%20Zhou&amp;author=S.S.%20Dong&amp;author=S.H.%20Shi&amp;publication_year=2008&amp;journal=Chinese%20Sci.%20Bull.&amp;volume=53&amp;pages=1708-15" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1007/s11434-008-0149-0" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1007%2Fs11434-008-0149-0" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1007%2Fs11434-008-0149-0"> </span></div> </div></div></div></div></div><div content-id="DSS004C55" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C55" class="ref-content "><div class="label -label">55</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Li</div> <div class="given-names">D.</div></div>,  <div class="name"><div class="surname">Su</div> <div class="given-names">Z.</div></div>,  <div class="name"><div class="surname">Dong</div> <div class="given-names">J.</div></div>,  <div class="name"><div class="surname">Wang</div> <div class="given-names">T.</div></div>. </div><div class="article-title">An expression database for roots of the model legume <em>Medicago truncatula</em> under salt stress</div>, <div class="source">BMC Genomics</div> , <div class="year">2009</div>, vol. <div class="volume">10</div> pg. <div class="fpage">517</div> <div class="pub-id-doi"><a href="https://doi.org/10.1186/1471-2164-10-517" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1186/1471-2164-10-517</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=An%20expression%20database%20for%20roots%20of%20the%20model%20legume%20Medicago%20truncatula%20under%20salt%20stress&amp;author=D.%20Li&amp;author=Z.%20Su&amp;author=J.%20Dong&amp;author=T.%20Wang&amp;publication_year=2009&amp;journal=BMC%20Genomics&amp;volume=10&amp;pages=517" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1186/1471-2164-10-517" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1186%2F1471-2164-10-517" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1186%2F1471-2164-10-517"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/19906315" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div><div content-id="DSS004C56" xmlns:helper="urn:XsltStringHelper"><div class="ref false"><div role="DSS004C56" class="ref-content "><div class="label -label">56</div><div class="citation element-citation"><div class="person-group"><div class="name"><div class="surname">Marioni</div> <div class="given-names">J.C.</div></div>,  <div class="name"><div class="surname">Mason</div> <div class="given-names">C.E.</div></div>,  <div class="name"><div class="surname">Mane</div> <div class="given-names">S.M.</div></div>,  <div class="name"><div class="surname">Stephens</div> <div class="given-names">M.</div></div>,  <div class="name"><div class="surname">Gilad</div> <div class="given-names">Y.</div></div>. </div><div class="article-title">RNA-seq: an assessment of technical reproducibility and comparison with gene expression arrays</div>, <div class="source">Genome Res.</div> , <div class="year">2008</div>, vol. <div class="volume">18</div> (pg. <div class="fpage">1509</div>-<div class="lpage">17</div>)<div class="pub-id-doi"><a href="https://doi.org/10.1101/gr.079558.108" class="link link-pub-id-doi" target="_blank">https://doi.org/10.1101/gr.079558.108</a></div><div class="citation-links"><div class="google-scholar-ref-link"><a href="https://scholar.google.com/scholar_lookup?title=RNA-seq%3A%20an%20assessment%20of%20technical%20reproducibility%20and%20comparison%20with%20gene%20expression%20arrays&amp;author=J.C.%20Marioni&amp;author=C.E.%20Mason&amp;author=S.M.%20Mane&amp;author=M.%20Stephens&amp;author=Y.%20Gilad&amp;publication_year=2008&amp;journal=Genome%20Res.&amp;volume=18&amp;pages=1509-17" target="_blank">Google Scholar</a></div><div class="crossref-doi"><a href="http://dx.doi.org/10.1101/gr.079558.108" target="_blank">Crossref</a></div><div class="adsDoiReference hide"><a href="http://adsabs.harvard.edu/cgi-bin/basic_connect?qsearch=10.1101%2Fgr.079558.108" target="_blank">Search ADS</a></div><div class="xslopenurl" style="display:none"><span class="inst-open-url-holders" data-targetId="10.1101%2Fgr.079558.108"> </span></div><div class="pub-id"><a href="http://www.ncbi.nlm.nih.gov/pubmed/18550803" class="link link-pub-id" target="_blank">PubMed</a></div> </div></div></div></div></div></div>        <!-- /foreach -->



        <div class="article-metadata-standalone-panel clearfix"></div>

        
<div class="copyright copyright-statement">© The Author 2012. Published by Oxford University Press on behalf of Kazusa DNA Research Institute</div><div class="license"><div class="license-p">This is an Open Access article distributed under the terms of the Creative Commons Attribution Non-Commercial License (http://creativecommons.org/licenses/by-nc/3.0/), which permits unrestricted non-commercial use, distribution, and reproduction in any medium, provided the original work is properly cited.</div></div><!-- /foreach -->

        <span id="UserHasAccess" data-userHasAccess="True"></span>

    </div><!-- /.widget-items -->
</div><!-- /.module-widget -->
    </div>
    <div class="widget widget-SolrResourceMetadata widget-instance-OUP_Article_ResourceMetadata_Widget">
        
        <div class="article-metadata-tocSections">
            <div class="article-metadata-tocSections-title">Issue Section:</div>
                <a href="/dnaresearch/search-results?f_TocHeadingTitle=Full Papers">Full Papers</a>
        </div>


    </div>
    <div class="widget widget-EditorInformation widget-instance-OUP_Article_EditorInformation_Widget">
        

    </div>


                        <div id="ContentTabFilteredView"></div>
                        <div class="downloadImagesppt">
                            <a href="//oup.silverchair-cdn.com/DownloadFile/DownloadImage.aspx?image=&PPTtype=SlideSet&ar=420778&xsltPath=~/UI/app/XSLT" id="BodyContent_PageContent_lnkDownloadAllImages">Download all figures</a>
                        </div>
                        <div></div>
                            <div class="widget widget-ArticleDataSupplements widget-instance-OUP_Data_Supplements_Tab">
            <a name="SupplementalTab" id="SupplementalTab"></a>
        <li class="section-jump-link head-1 dataSupplements-jumplink">
            <div class="section-jump-link__link-wrap">
                <a class="jumplink js-jumplink scrollTo" href="#supplementary-data">Supplementary data</a>
            </div>
        </li>
        <h2 scrollto-destination="supplementary-data"  id="supplementary-data" class="section-title">Supplementary data</h2>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/dss004-dss004supp_table4.xls?Expires=2147483647&amp;Signature=xmt15VNTy9b25RznsiqGzqT8m4ExdAcpv1SDcdnVoWsT-pYTFxUC3A7QAy6zT1wHjXIvdR6LO2SC7OiY44WxNQGbTFdCerH2I4CdSE3Bd8EuBrO~8VPhIBJLcZUa1c0HGfXi7wq6QSCJ0a6YcK4FZ988eEuVFWHROjxvXWjDO7pSZIHFlJQMbufa~vunHbI4A77w3I9qccHgU4pUi7G7F4j2toIB0c4eOJhqDUZXej-ZrzMWmxB8fZ7u2GNmMtK1VzXWi20XKSIgl4c8jKiUl0r9L~gOIqwsTDR2DxQa1BNH9lO5KLCme9ptDBXgVhlqgGFRrfMzcRi4gNJ6mO8Axw__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-3</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/dss004-dss004supp_table3.xls?Expires=2147483647&amp;Signature=uv2zIATqlIYaD6A2BbIViMmAk06tf95Fsx5vJkt83VT63-8FS9XKhcDgxESUg5YAhOnUcLEboK07~8QnIiOMvAVaOkFK2aY68OYdBXHLQeJnu7DMyMZn~wRyO~-jecP9oh6B5TuPv8llJ5gbFHKi42LDvqFQuBGq4XYmNoxCqV8QM7ROV3Sgv5bDEOWTuZmuAXx7betw~0wizbUHw5p8j6mXmYlogQO1qkuprNPxrJ2OAy0rnX4F1LS6Yky~jc5x6pWEfBGvPz~zK2X3mg3P2FsI3aEetMRVQ24oLMI17CCEvf1QsvWsLM-V-M4HerRfYm0Pzby7R4fYzCNCbuZtCQ__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-2</a> - xls file</div>
        <div class="dataSuppLink"><a href="https://oup.silverchair-cdn.com/oup/backfile/Content_public/Journal/dnaresearch/19/2/10.1093/dnares/dss004/2/dss004-dss004supp.doc?Expires=2147483647&amp;Signature=lIvYVY8fOHrDvFHSA27gM~LeYVlhWu7ih50M3yI-kDE35R9e3rGm1fRb0wX3xtbzmniw0cC~4lu~WbwJbqItNHotyBJopQlFpIOQsqpnC14My8NlgiqcsNEOHZ0Y3ZZL7VD~NaY5F9KP6Kj2nCJqO7BeFSBiHyV7ePWv3HVgrlGlE-UZsfrYzB9yIF~18waXN6tBTQZilaDsN4M-dDl3EEyl3rmSZgKuvqB3olV-USZX4CP4HEvkb4UtyfKBIo8P1BIV05B-lKgNBbHDTNUVj3boo0Hkfj0~auB4282BC3nqoeNqsdxsTBlCpJFUqRccpwj2BAIQx4IztqU8KQkB2g__&amp;Key-Pair-Id=APKAIE5G5CRDK6RD3PGA">AddSuppFiles-1</a> - doc file</div>

    </div>
    <div class="widget widget-ArticleDataRepositories widget-instance-Article_DryadLink">
        
    </div>

                         <div class="comments">
                                <div class="widget widget-UserCommentBody widget-instance-UserCommentBody_Article">
        


    </div>
    <div class="widget widget-UserComment widget-instance-OUP_UserComment_Article">
        
    </div>

                        </div>
                    </div>
                   
                </div> 

                     


            </div><!-- /.content-inner-wrap -->
        </div> 

        <div id="Sidebar" class="page-column page-column--right">
                <div class="widget widget-AdBlock widget-instance-ArticlePageTopSidebar">
        

<div id="adBlockMainBodyTop"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyTop'); });
    </script>
</div>

    </div>
<div class="widget widget-dynamic " data-count="1"> 

    <div class="widget-dynamic-inner-wrap">

<div class="widget widget-dynamic " data-count="10"> 

    <div class="widget-dynamic-inner-wrap">

    <div class="widget widget-ArticleLevelMetrics widget-instance-Article_RightRailB0Article_RightRail_ArticleLevelMetrics">
        





    <div class="artmet-condensed-wrap clearfix">
        <div class="artmet-condensed-stats clearfix">
            <div class="artmet-item artmet-views">
                <span class="artmet-number">422</span>
                <span class="artmet-text">Views</span>
            </div>

            <div class="artmet-item artmet-citations">
                <span class="artmet-number">
                     <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000303290100008&amp;DestLinkType=CitingArticles&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">46</a>
                </span>
                <span class="artmet-text">Citations</span>
            </div>
    
            <div class="artmet-item artmet-altmetric">
    <div class="widget widget-AltmetricLink widget-instance-ArticleLevelMetrics_AltmetricLinkSummary">
            <!-- Altmetrics -->
    <div id="altmetricEmbedId"
         runat="server"
         class='altmetric-embed'
         data-badge-type="donut"
         data-hide-no-mentions="false"
         data-doi="10.1093/dnares/dss004"

        

        ></div>
         <script type='text/javascript' src='https://d1bxh8uas1mnw7.cloudfront.net/assets/embed.js'></script>

    </div>

            </div>
        </div>

            <div class="artmet-modal-trigger-wrap clearfix">
                <a class="artmet-modal-trigger" data-article-id="420778"><i class="icon-metrics"></i><span>View Metrics</span></a>
            </div>
    </div>
        <div class="artmet-modal" id="MetricsModal">
            <div class="artmet-modal-contents">
                <a class="artmet-close-modal">&#215;</a>
            </div>
        </div>


    </div>
    <div class="widget widget-Alerts widget-instance-Article_RightRailB0Article_RightRail_Alerts">
        
    <div class="alertsWidget">
        <h3>Email alerts</h3>
                <div class="userAlert alertType-5">
                    <a role="link" aria-label="New issue alert"
                       data-userLoggedIn="False"
                       data-alertType="5">New issue alert</a>
                </div>
                <div class="userAlert alertType-3">
                    <a role="link" aria-label="Advance article alerts"
                       data-userLoggedIn="False"
                       data-alertType="3">Advance article alerts</a>
                </div>
                <div class="userAlert alertType-1">
                    <a role="link" aria-label="Article activity alert"
                       data-userLoggedIn="False"
                       data-alertType="1">Article activity alert</a>
                </div>
                    <div class="userAlert alertType-MarketingLink">
                <a role="link" aria-label="Receive exclusive offers and updates from Oxford Academic"
                   data-userLoggedIn="False"
                   data-additionalUrl="/my-account/communication-preferences">Receive exclusive offers and updates from Oxford Academic</a>
            </div>
        <div class="userAlertSignUpModal reveal-modal small" data-reveal>
            <div class="userAlertSignUp"></div>
            <label for="alert-signup-modal-close" class="screenreader-text">Close</label>
            <a role="button" id="alert-signup-modal-close" class="close-reveal-modal" href="javascript:;">
                <i class="icon-general-close"></i>
            </a>
        </div>
    </div>

    </div>
    <div class="widget widget-ArticleLinks widget-instance-Article_RightRailB0Article_RightRail_ArticleLinks">
        


    </div>
    <div class="widget widget-RelatedContent widget-instance-Article_RightRailB0Article_RightRail_RelatedContent">
        
    </div>
    <div class="widget widget-RelatedArticleIn widget-instance-Article_RightRailB0Article_RightRail_RelatedArticleIn">
            <h3 class="relatedArticleIn-title">Related articles in</h3>
    <div class="relatedArticleIn-content">
        <ul>
                <li>
                    <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&amp;SrcApp=PARTNER_APP&amp;SrcAuth=LinksAMR&amp;KeyUT=WOS:000303290100008&amp;DestLinkType=RelatedRecords&amp;DestApp=ALL_WOS&amp;UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a">Web of Science</a>
                </li>
                <li>
                    <a href="https://scholar.google.com/scholar?q=related:https%3a%2f%2facademic.oup.com%2fdnaresearch%2farticle%2f19%2f2%2f195%2f420778">Google Scholar</a>
                </li>
        </ul>
    </div>

    </div>
    <div class="widget widget-ArticleCitedBy widget-instance-Article_RightRailB0Article_RightRail_ArticleCitedBy">
        <h3 class="article-cited-title">Citing articles via</h3>
<div class="widget-links_wrap">
        <div class="article-cited-link-wrap web-of-science">
            <a href="http://gateway.webofknowledge.com/gateway/Gateway.cgi?GWVersion=2&SrcApp=PARTNER_APP&SrcAuth=LinksAMR&KeyUT=WOS:000303290100008&DestLinkType=CitingArticles&DestApp=ALL_WOS&UsrCustomerID=61f30d8ae69c46f86624c5f98a3bc13a" target="_blank">Web of Science (46)</a>
        </div>
            <div class="article-cited-link-wrap google-scholar-url">
            <a href="http://scholar.google.com/scholar?q=link:https%3A%2F%2Facademic.oup.com%2Fdnaresearch%2Farticle%2F19%2F2%2F195%2F420778" target="_blank">Google Scholar</a>
        </div>
            <div class="article-cited-link-wrap crossref-citedby">
            <a href='https://academic.oup.com/dnaresearch/CrossRef-CitedBy/420778'>CrossRef</a>
        </div>
</div>
    </div>
    <div class="widget widget-ArticleListNewAndPopular widget-instance-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined">
            <ul class="articleListNewAndPopularCombinedView">
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode active" data-mode="MostRecent">Latest</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostRead">Most Read</a>
            </li>
            <li>
                <a href="javascript:;" class="articleListNewAndPopular-mode " data-mode="MostCited">Most Cited</a>
            </li>
    </ul>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRecent hasContent">




<div id="newPopularList-Article_RightRailB0Article_RightRail_ArticleNewPopularCombined" class="fb">

    







<div class="widget-layout widget-layout--vert">






        <div class="widget-columns widget-col-1">
            <div class="col">

                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">Landscape of gene transposition–duplication within the Brassicaceae family</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy035/5151297">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">Multi-tissue transcriptomes of caecilian amphibians highlight incomplete knowledge of vertebrate gene families</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy034/5142401">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">Overcoming challenges in variant calling: exploring sequence diversity in candidate genes for plant development in perennial ryegrass ( Lolium perenne )</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy033/5133005">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
                       <div class="widget-dynamic-entry js-widget-dynamic-entry" data-resource-url="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">                           
                        



                        
                            <div class="widget-dynamic-journal-title">
                                <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">Genome-wide screening and analysis of imprinted genes in rapeseed ( Brassica napus  L.) endosperm</a>
                            </div>
                                <div class="widget-dynamic-journal-image-synopsis">
                                        <div class="widget-dynamic-journal-synopsis">
                                            <a href="/dnaresearch/advance-article/doi/10.1093/dnares/dsy030/5110093">
                                                
                                            </a>
                                        </div> <!-- /.widget-dynamic-journal-synopsis -->
                                </div>

                    </div>
            </div>
        </div>
</div><!-- /.widget-layout-[orientation] --></div>
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostRead hide">
        </section>
        <section class="articleListNewPopContent articleListNewAndPopular-ContentView-MostCited hide">
        </section>

    </div>

    </div>

</div>
    </div>

</div>    <div class="widget widget-AdBlock widget-instance-ArticlePageTopMainBodyBottom">
        

<div id="adBlockMainBodyBottom"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockMainBodyBottom'); });
    </script>
</div>

    </div>

        </div> 
    </div><!-- /.page-column-wrap -->
    <input name="ctl00$ctl00$BodyContent$PageContent$hfArticleTitle" type="hidden" id="hfArticleTitle" value="Transcriptome Characterization and Sequencing-Based Identification of Salt-Responsive Genes in Millettia pinnata, a Semi-Mangrove Plant" />

                </div><!-- /.center-inner-row -->
            </section>
        </main>
    </div>
    

        
    
    <script type="text/javascript">
        $(document).ready(function () {
            SCM.ScholarlyiQ.trackArticlePageEvents();
        });
        googletag.cmd.push(function () {
            ////////////////////////////////////// IMPORTANT: //////////////////////////////////////
            // Article DOIs are passed to GPT to target specific articles. Because the DOI cannot //
            // be accessed from Site.Master, put off calling enableServices() until the targeting //
            // for DOI is set. Only call enableServices() if it is not going to be called later   //
            // in Article.aspx. See Site.Master                                                   //
            ////////////////////////////////////////////////////////////////////////////////////////
            googletag.pubads().setTargeting("jnlsdoi", "10.1093/dnares/dss004");
            googletag.enableServices();
        });
    </script>

        <div class="widget widget-SitePageFooter widget-instance-SitePageFooter">
        <div class="ad-banner ad-banner-footer">
    <div class="widget widget-AdBlock widget-instance-FooterAd">
        

<div id="adBlockFooter"
     style=" ">
    <script>
        googletag.cmd.push(function () { googletag.display('adBlockFooter'); });
    </script>
</div>

    </div>

</div>

    <div class="journal-footer journal-bg">
        <div class="center-inner-row">

                <div class="journal-footer-menu">

                    <ul>
                            <li class="link-1">
                                <a href="/dnaresearch/pages/About">About DNA Research</a>
                            </li>
                            <li class="link-2">
                                <a href="/dnaresearch/pages/Editorial_Board">Editorial Board</a>
                            </li>
                            <li class="link-3">
                                <a href="/dnaresearch/pages/Instructions_To_Authors">Author Guidelines</a>
                            </li>
                            <li class="link-4">
                                <a href="http://www.kazusa.or.jp/e/index.html">Contact Us</a>
                            </li>
                            <li class="link-5">
                                <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
                            </li>
</ul><ul>                            <li class="link-1">
                                <a href="https://twitter.com/OxfordJournals">Twitter</a>
                            </li>
                            <li class="link-2">
                                <a href="http://www.oupmediainfo.com/">Advertising and Corporate Services</a>
                            </li>
                            <li class="link-3">
                                <a href="http://science-and-mathematics-careernetwork.oxfordjournals.org">Journals Career Network</a>
                            </li>

                    </ul>


                </div><!-- /.journal-footer-menu -->

            <div class="journal-footer-affiliations">
                <!-- <h3>Affiliations</h3> -->
            <a href="https://academic.oup.com/dh" target="">
                <img id="footer-logo-DNAResearch" class="journal-footer-affiliations-logo" src="//oup.silverchair-cdn.com/data/SiteBuilderAssets/Live/Images/dnaresearch/dnaresearch_f1-2068071465.svg" alt="DNA Research" />
            </a>
            </div><!-- /.journal-footer-affiliations -->

            <div class="journal-footer-colophon">
                <ul>
                        <li>Online ISSN 1756-1663</li>
                                            <li>Print ISSN 1340-2838</li>
                    <li>Copyright &#169; 2018 Kazusa DNA Research Institute</li>
                </ul>
            </div><!-- /.journal-footer-colophon -->


        </div><!-- /.center-inner-row -->
    </div><!-- /.journal-footer -->

    </div>

    <div class="oup-footer">
        <div class="center-inner-row">
                <div class="widget widget-SelfServeContent widget-instance-OupUmbrellaFooterSelfServe">
        
<div class="oup-footer-row journal-links">
<div class="global-footer selfservelinks">
<ul>
    <li>
    <a href="/journals/pages/about_us">About Us</a>
    </li>
    <li>
    <a href="/journals/pages/contact_us">Contact Us</a>
    </li>
    <li>
    <a href="http://global.oup.com/jobs">Careers</a>
    </li>
    <li>
    <a href="/journals/pages/help">Help</a>
    </li>
    <li>
    <a href="/journals/pages/access_purchase">Access &amp; Purchase</a></li>
    <li>
    <a href="/journals/pages/access_purchase/rights_and_permissions">Rights &amp; Permissions</a>
    </li>
    <li>
    <a href="/journals/pages/open_access">Open Access</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4 id="global-footer-customer services" class="title footerheader">Connect</h4>
<ul>
    <li>
    <a href="/my-account/communication-preferences">Join Our Mailing List</a>
    </li>
    <li>
    <a href="http://blog.oup.com/">OUPblog</a>
    </li>
    <li>
    <a href="http://twitter.com/oupacademic">Twitter</a>
    </li>
    <li>
    <a href="https://www.facebook.com/OUPAcademic">Facebook</a>
    </li>
    <li>
    <a href="http://www.youtube.com/playlist?list=PL3MAPgqN8JWioLLTkU4mlFM4-SNt-f1Xs">YouTube</a>
    </li>
    <li>
    <a href="http://oupacademic.tumblr.com/">Tumblr</a>
    </li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Resources</h4>
<ul>
    <li><a href="/journals/pages/authors">Authors</a></li>
    <li><a href="/journals/pages/librarians">Librarians</a></li>
    <li><a href="/journals/pages/societies">Societies</a></li>
    <li><a href="https://academic.oup.com/journals/pages/sponsorship_and_advertising">Sponsors &amp; Advertisers</a></li>
    <li><a href="/journals/pages/press">Press &amp; Media</a></li>
    <li><a href="/journals/pages/agents">Agents</a></li>
</ul>
</div>
<div class="global-footer selfservelinks">
<h4>Explore</h4>
<ul>
    <li><a href="http://global.oup.com/academic">Shop OUP Academic</a></li>
    <li><a href="http://www.oxforddictionaries.com/">Oxford Dictionaries</a></li>
    <li><a href="http://oxfordindex.oup.com/">Oxford Index</a></li>
    <li><a href="https://www.epigeum.com/">Epigeum</a></li>
    <li><a href="http://global.oup.com/">OUP Worldwide</a></li>
    <li><a href="http://www.ox.ac.uk/">University of Oxford</a></li>
</ul>
</div>
<div class="OUP-mission">
<p>Oxford University Press is a department of the University of Oxford. It furthers the University's objective of excellence in research, scholarship, and education by publishing worldwide</p>
<img class="journal-footer-logo" src="//oup.silverchair-cdn.com/UI/app/svg/umbrella/oup-logo.svg" alt="Oxford University Press" width="150" height="42" />
</div>
</div>
<div class="oup-footer-row">
<div class="oup-footer-row-links">
<ul>
    <li>Copyright &copy; 2018 Oxford University Press</li>
    <li><a href="https://global.oup.com/cookiepolicy/">Cookie Policy</a></li>
    <li><a href="https://global.oup.com/privacy">Privacy Policy</a></li>
    <li><a href="/journals/pages/about_us/legal/notices">Legal Notice</a></li>
    <li><a href="/journals/pages/sitemap">Site Map</a></li>
    <li><a href="/journals/pages/about_us/legal/accessibility">Accessibility</a></li>
    <li><a href="https://get.adobe.com/reader/">Get Adobe Reader</a></li>
</ul>
</div>
</div>
<style type="text/css">
    /* Issue right column fix for tablet/mobile */
    @media (max-width: 1200px) {
    .pg_issue .widget-instance-OUP_Issue {
    width: 100%;
    }
    }
</style>
    
    
    
    </div>

        </div><!-- /.center-inner-row -->
    </div><!-- /.oup-footer -->
    <div class="ss-ui-only">
        
    </div>


        
        <div id="revealModal" class="reveal-modal" data-reveal>
            <div id="revealContent"></div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        <div id="NeedSubscription" class="reveal-modal small" data-reveal>
            <div class="subscription-needed">
                <h5>This Feature Is Available To Subscribers Only</h5>
                <p><a href="/sign-in">Sign In</a> or <a href="/my-account/register?siteId=5275&returnUrl=%2fdnaresearch%2farticle%2f19%2f2%2f195%2f420778">Create an Account</a></p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        

        <div id="noAccessReveal" class="reveal-modal tiny" data-reveal>
            <p>This PDF is available to Subscribers Only</p>
            <a class="hide-for-article-page" id="articleLinkToPurchase" data-reveal><span>View Article Abstract & Purchase Options</span></a>
            <div class="issue-purchase-modal">
                <p>For full access to this pdf, sign in to an existing account, or purchase an annual subscription.</p>
            </div>
            <a class="close-reveal-modal" href="javascript:;"><i class="icon-general-close"></i><span class="screenreader-text">Close</span></a>
        </div>
        
        
 
    
<script src="//oup.silverchair-cdn.com/cassette.axd/script/b5563c8146708c035780ef68ed935e3872d56f24/footerJS" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/610f1e5df562bc74d41ca3c9dfd22a8e7083ae6a/OupCookiePolicyJS" type="text/javascript"></script>
<script src="https://cdnjs.cloudflare.com/ajax/libs/mathjax/2.7.1/MathJax.js?config=TeX-AMS-MML_HTMLorMML" type="text/javascript"></script>
<script src="//cdn.jsdelivr.net/chartist.js/latest/chartist.min.js" type="text/javascript"></script>
<script src="//oup.silverchair-cdn.com/cassette.axd/script/92bdf416fbca406a9665b4984b883e6c0faf1f20/ArticlePageJS" type="text/javascript"></script>
<script src="https://s7.addthis.com/js/300/addthis_widget.js#pubid=xa-5265518246c10183&domready=1" type="text/javascript"></script>
<script src="https://www.google.com/recaptcha/api.js?onload=onloadCallback&render=explicit" type="text/javascript"></script>

    
    <!-- Enter Google Analytics or Tag Manager here -->
    
     
 

    <!-- BEGIN: Scholarly iQ Page Tag -->
    <!-- Copyright 2001-2010, IBM Corporation All rights reserved. -->
    <script language="JavaScript" src="//ouptag.scholarlyiq.com/ntpagetag.js"></script>
    <noscript>
       <img src="//ouptag.scholarlyiq.com/ntpagetag.gif?js=0" height="1" width="1" border="0" hspace="0" vspace="0" alt="Scholarly IQ">
    </noscript>
    <!-- END: Scholarly iQ Page Tag -->
</body>
</html>

